
<html lang="en"     class="pb-page"  data-request-id="1fae1b4f-5a47-4493-b920-be60ad7ef24d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00510;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-16;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Development of a Series of Pyrazolo[3,4-d]pyridazinone Compounds as the Novel Covalent Fibroblast Growth Factor Receptor Inhibitors by the Rational Drug Design" /></meta><meta name="dc.Creator" content="Yulan  Wang" /></meta><meta name="dc.Creator" content="Yang  Dai" /></meta><meta name="dc.Creator" content="Xiaowei  Wu" /></meta><meta name="dc.Creator" content="Fei  Li" /></meta><meta name="dc.Creator" content="Bo  Liu" /></meta><meta name="dc.Creator" content="Chunpu  Li" /></meta><meta name="dc.Creator" content="Qiufeng  Liu" /></meta><meta name="dc.Creator" content="Yuanyang  Zhou" /></meta><meta name="dc.Creator" content="Bao  Wang" /></meta><meta name="dc.Creator" content="Mingrui  Zhu" /></meta><meta name="dc.Creator" content="Rongrong  Cui" /></meta><meta name="dc.Creator" content="Xiaoqin  Tan" /></meta><meta name="dc.Creator" content="Zhaoping  Xiong" /></meta><meta name="dc.Creator" content="Jia  Liu" /></meta><meta name="dc.Creator" content="Minjia  Tan" /></meta><meta name="dc.Creator" content="Yechun  Xu" /></meta><meta name="dc.Creator" content="Meiyu  Geng" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Hong  Liu" /></meta><meta name="dc.Creator" content="Jing  Ai" /></meta><meta name="dc.Creator" content="Mingyue  Zheng" /></meta><meta name="dc.Description" content="Alterations of fibroblast growth factor receptors (FGFRs) play key roles in numerous cancer progression and development, which makes FGFRs attractive targets in the cancer therapy. In the present s..." /></meta><meta name="Description" content="Alterations of fibroblast growth factor receptors (FGFRs) play key roles in numerous cancer progression and development, which makes FGFRs attractive targets in the cancer therapy. In the present s..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 23, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00510" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00510" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00510" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00510" /></link>
        
    
    

<title>Discovery and Development of a Series of Pyrazolo[3,4-d]pyridazinone Compounds as the Novel Covalent Fibroblast Growth Factor Receptor Inhibitors by the Rational Drug Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00510" /></meta><meta property="og:title" content="Discovery and Development of a Series of Pyrazolo[3,4-d]pyridazinone Compounds as the Novel Covalent Fibroblast Growth Factor Receptor Inhibitors by the Rational Drug Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0012.jpeg" /></meta><meta property="og:description" content="Alterations of fibroblast growth factor receptors (FGFRs) play key roles in numerous cancer progression and development, which makes FGFRs attractive targets in the cancer therapy. In the present study, based on a newly devised FGFR target-specific scoring function, a novel FGFR inhibitor hit was identified through virtual screening. Hit-to-lead optimization was then performed by integrating molecular docking and site-of-metabolism predictions with an array of in vitro evaluations and X-ray cocrystal structure determination, leading to a covalent FGFR inhibitor 15, which showed a highly selective and potent FGFR inhibition profile. Pharmacokinetic assessment, protein kinase profiling, and hERG inhibition evaluation were also conducted, and they confirmed the value of 15 as a lead for further investigation. Overall, this study exemplifies the importance of the integrative use of computational methods and experimental techniques in drug discovery." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00510"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00510">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00510&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00510&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00510&amp;href=/doi/10.1021/acs.jmedchem.9b00510" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7473-7488</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00394" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00625" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Development of a Series of Pyrazolo[3,4-<i>d</i>]pyridazinone Compounds as the Novel Covalent Fibroblast Growth Factor Receptor Inhibitors by the Rational Drug Design</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yulan Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yulan Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yulan++Wang">Yulan Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yang Dai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yang Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yang++Dai">Yang Dai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaowei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaowei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaowei++Wu">Xiaowei Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fei Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fei Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Chemistry, Shanghai University, 99 ShangDa Road, Shanghai 200444, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Li">Fei Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bo Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Liu">Bo Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunpu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunpu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica,  Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunpu++Li">Chunpu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiufeng Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiufeng Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiufeng++Liu">Qiufeng Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuanyang Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuanyang Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuanyang++Zhou">Yuanyang Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bao Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bao Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Middle Huaxia Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bao++Wang">Bao Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mingrui Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingrui Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingrui++Zhu">Mingrui Zhu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rongrong Cui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rongrong Cui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rongrong++Cui">Rongrong Cui</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoqin Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoqin Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoqin++Tan">Xiaoqin Tan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhaoping Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaoping Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Middle Huaxia Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaoping++Xiong">Zhaoping Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jia Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jia Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jia++Liu">Jia Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Minjia Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Minjia Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Minjia++Tan">Minjia Tan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6784-9653" title="Orcid link">http://orcid.org/0000-0002-6784-9653</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yechun Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yechun Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yechun++Xu">Yechun Xu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1581-6155" title="Orcid link">http://orcid.org/0000-0002-1581-6155</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Meiyu Geng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Meiyu Geng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Meiyu++Geng">Meiyu Geng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hualiang Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hualiang Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Middle Huaxia Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#741c181d0134071d19195a15175a171a"><span class="__cf_email__" data-cfemail="375f5b5e4277445e5a5a195654195459">[email protected]</span></a>. Phone: +86-21-50807042 (H.L.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Liu">Hong Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0863-7853" title="Orcid link">http://orcid.org/0000-0002-0863-7853</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Ai</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Ai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#016b60684172686c6c2f60622f626f"><span class="__cf_email__" data-cfemail="7d171c143d0e141010531c1e531e13">[email protected]</span></a>. Phone: +86-21-50806600-2413 (J.A.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Ai">Jing Ai</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Mingyue Zheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingyue Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#2b465251434e454c6b58424646054a48054845"><span class="__cf_email__" data-cfemail="d5b8acafbdb0bbb295a6bcb8b8fbb4b6fbb6bb">[email protected]</span></a>. Phone: +86-21-50806600-1308 (M.Z.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingyue++Zheng">Mingyue Zheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3323-3092" title="Orcid link">http://orcid.org/0000-0002-3323-3092</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00510&amp;href=/doi/10.1021%2Facs.jmedchem.9b00510" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7473–7488</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 23, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 March 2019</li><li><span class="item_label"><b>Published</b> online</span>23 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00510" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00510</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7473%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYulan%2BWang%252C%2BYang%2BDai%252C%2BXiaowei%2BWu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D16%26contentID%3Dacs.jmedchem.9b00510%26title%3DDiscovery%2Band%2BDevelopment%2Bof%2Ba%2BSeries%2Bof%2BPyrazolo%255B3%252C4-d%255Dpyridazinone%2BCompounds%2Bas%2Bthe%2BNovel%2BCovalent%2BFibroblast%2BGrowth%2BFactor%2BReceptor%2BInhibitors%2Bby%2Bthe%2BRational%2BDrug%2BDesign%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7488%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00510"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3156</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00510" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Development of a Series of Pyrazolo[3,4-d]pyridazinone Compounds as the Novel Covalent Fibroblast Growth Factor Receptor Inhibitors by the Rational Drug Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yulan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yang&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Xiaowei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Chunpu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Qiufeng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yuanyang&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Bao&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Mingrui&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Rongrong&quot;,&quot;last_name&quot;:&quot;Cui&quot;},{&quot;first_name&quot;:&quot;Xiaoqin&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Zhaoping&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Jia&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Minjia&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Yechun&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Meiyu&quot;,&quot;last_name&quot;:&quot;Geng&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Ai&quot;},{&quot;first_name&quot;:&quot;Mingyue&quot;,&quot;last_name&quot;:&quot;Zheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7473-7488&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00510&quot;},&quot;abstract&quot;:&quot;Alterations of fibroblast growth factor receptors (FGFRs) play key roles in numerous cancer progression and development, which makes FGFRs attractive targets in the cancer therapy. In the present study, based on a newly devised FGFR target-specific scoring function, a novel FGFR inhibitor hit was identified through virtual screening. Hit-to-lead optimization was then performed by integrating molecular docking and site-of-metabolism predictions with an array of in vitro evaluations and X-ray cocrystal structure determination, leading to a covalent FGFR inhibitor 15, which showed a highly selective and potent FGFR inhibition profile. Pharmacokinetic assessment, protein kinase profiling, and hERG inhibition evaluation were also conducted, and they confirmed the value of 15 as a lead for further investigation. Overall, this study exemplifies the importance of the integrative use of computational methods and experimental techniques in drug discovery.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00510&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00510" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00510&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00510" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00510&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00510" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00510&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00510&amp;href=/doi/10.1021/acs.jmedchem.9b00510" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00510" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00510" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00510%26sid%3Dliteratum%253Aachs%26pmid%3D31335138%26genre%3Darticle%26aulast%3DWang%26date%3D2019%26atitle%3DDiscovery%2Band%2BDevelopment%2Bof%2Ba%2BSeries%2Bof%2BPyrazolo%255B3%252C4-d%255Dpyridazinone%2BCompounds%2Bas%2Bthe%2BNovel%2BCovalent%2BFibroblast%2BGrowth%2BFactor%2BReceptor%2BInhibitors%2Bby%2Bthe%2BRational%2BDrug%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D16%26spage%3D7473%26epage%3D7488%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/jmcmar.2019.62.issue-16/20190822/jmcmar.2019.62.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Alterations of fibroblast growth factor receptors (FGFRs) play key roles in numerous cancer progression and development, which makes FGFRs attractive targets in the cancer therapy. In the present study, based on a newly devised FGFR target-specific scoring function, a novel FGFR inhibitor hit was identified through virtual screening. Hit-to-lead optimization was then performed by integrating molecular docking and site-of-metabolism predictions with an array of in vitro evaluations and X-ray cocrystal structure determination, leading to a covalent FGFR inhibitor <b>15</b>, which showed a highly selective and potent FGFR inhibition profile. Pharmacokinetic assessment, protein kinase profiling, and hERG inhibition evaluation were also conducted, and they confirmed the value of <b>15</b> as a lead for further investigation. Overall, this study exemplifies the importance of the integrative use of computational methods and experimental techniques in drug discovery.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The fibroblast growth factor receptor (FGFR) family is a subfamily of receptor tyrosine kinases (RTKs), which comprises four highly conserved transmembrane RTKs, FGFR1, FGFR2, FGFR3, and FGFR4.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> FGFRs play important roles in a variety of cell functions, such as cell proliferation and differentiation, and biological processes, including development, angiogenesis, homeostasis, and wound repair.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Alterations of FGFRs are associated with the progression and development of several cancers, and they contribute to carcinogenesis in three main situations, namely, driver mutations, neoangiogenesis, and resistance to anticancer agents. Extensive FGFR alterations in cancers have been observed, such as FGFR1 amplifications in 6% of small cell lung carcinomas, 20% of squamous nonsmall cell lung carcinomas, 17% of osteosarcomas, 10–15% of breast cancers, 5% of ovarian cancers, and 9% of esophageal cancers; FGFR2 translocations in 14% of intrahepatic cholangiocarcinomas (CCAs); FGFR2 mutations in 12–14% of endometrial cancers and 5% of squamous nonsmall cell lung carcinomas; FGFR2 amplifications in 12–14% of gastric cancers and 4% of breast cancers; FGFR3 translocations in 3–6% of bladder carcinomas, 3% of glioblastomas, and 15–20% of myelomas; FGFR3 mutations in 60–80% of non-muscle-invasive bladder carcinomas; FGFR4 mutations in 6–8% of rhabdomyosarcomas; and so forth.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Therefore, targeting FGFRs with small-molecule inhibitors represents a promising therapeutic strategy for treating the above FGFR-related cancers.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">A large amount of effort has been devoted to developing FGF/FGFR inhibitors as anticancer treatments.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Several classes of small-molecule FGFR inhibitors have been reported, and they can be clustered into two classes based on their inhibition profiles. The first class of inhibitors is characterized by “multitarget” tyrosine kinase inhibitors (TKIs), which have broad targets, including FGFR. In addition to FGFR, these “multitarget” TKIs may also target vascular endothelial growth receptors (VEGFRs), platelet-derived growth factor receptors (PDGFRs), and other tyrosine kinases, resulting in serious adverse effects in clinical studies, particularly in VEGFR2-based dose-limiting toxicities.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> The second class is inhibitors with highly selective and potent inhibitory activities against FGFR. These compounds can specifically target FGFR, reducing the risks of side effects in clinical usage and making the mechanism of action (MOA) easier to study. According to their MOA, FGFR-selective inhibitors fall into two categories: noncovalent inhibitors and covalent inhibitors. To date, there are several selective inhibitors, including both noncovalent and covalent inhibitors [shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>; noncovalent inhibitors: AZD4547<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (<b>1</b>), NVP-BGJ398<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (<b>2</b>), E7090<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (<b>3</b>), LY2874455<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (<b>4</b>), JNJ-42756493<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (<b>5</b>), and CH5183284<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (<b>6</b>); covalent inhibitors: TAS-120<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> (<b>7</b>)] are in clinical development, and some have demonstrated clinically meaningful benefits with manageable toxicity profiles in clinical trials. For example, <b>1</b>, a selective reversible FGFR inhibitor, has demonstrated a well-tolerable safety profile and modest antitumor activity in advanced squamous cell lung cancers in a phase Ib study.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In a phase Ib/IIa study, a combination of <b>1</b> with either anastrozole or letrozole showed anti-tumor activity in advanced estrogen receptor-positive breast cancers resistant to aromatase inhibitors.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In addition, a selective covalent FGFR inhibitor, <b>7</b>, has shown a tolerable safety profile and antitumor activity in patients with CCA harboring FGFR2 gene fusions and CCA patients with FGFR2 gene fusion resistance prior to the FGFR inhibitor <b>2</b>.<a onclick="showRef(event, 'ref15 ref16 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref19">(15,16,19)</a> A phase II study of <b>7</b> with patients with intrahepatic CCA harboring FGFR2 gene fusions has been initiated. Despite the advances made in FGFR inhibitors, the road to clinical treatment of various diseases related to alterations of FGFR remains long and challenging. The structural diversification of highly selective and highly bioactive FGFR inhibitors is still in high demand, and the options for the clinical treatment of diverse FGFR aberrations in a wide variety of cancers are still very limited, highlighting the importance of developing FGFR inhibitors of novel chemotypes.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. FGFR-selective inhibitors in clinical development.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The current study reports the discovery and development of a series of pyrazolo[3,4-<i>d</i>]pyridazinone derivatives as a novel type of FGFR inhibitors, and it emphasizes the synergetic use of the computer-aided drug design (CADD) with experimental evaluations to address different issues in our hit identification and hit-to-lead optimization stages. Moreover, a variety of chemical, biophysical, and bio-pharmacological methodologies were used for the full characterization of the anticancer activity, physicochemical and PK properties, and MOA of the discovered FGFR inhibitor <b>15</b>, which may serve as a promising new lead for developing potential anticancer treatments.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Hit Discovery by Target Specific Scoring Function and Virtual Screening</h3><div class="NLM_p">The target-specific scoring function has proven effective in enhancing the enrichment of virtual screening (VS) for novel hit discovery and for interpreting the subtle commonalities and differences in the ligands binding to the target binding site of interest. Previously, we built a target-specific scoring function for methyltransferases, SAMscore, based on the crystal structures of methyltransferase and their ligand activity.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The SAMscore demonstrated improved scoring ability and has been successfully used to identify novel disruptor of silencing telomeric 1-like inhibitors.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> To date, the crystal structures of many FGFR proteins and FGFR inhibitors have been reported, providing the basis for the development of FGFR target-specific scoring functions. In this study, using a workflow similar to that of SAMscore, we developed a target-specific scoring function for the FGFR, RTKscore. First, we selected protein crystal structures for FGFR1, FGFR2, and FGFR3, and the PDB codes were <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PVF">2PVF</a>,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K33">4K33</a>,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> respectively. In addition, 1090 FGFR ligands with known chemical structural information and activity data were collected from BindingDB.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The ligands with reported inhibitory activity data such as >10<sup>9</sup> nM were marked as inactive. Second, the molecular docking program Glide<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was used to generate the binding modes of the FGFR proteins and their ligands. The iterative potential of mean force (iPMF)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> features of the binding mode of each ligand with the corresponding protein were calculated to characterize the interaction between the ligand and the protein as the input data for modeling building the model. Third, original features were processed and selected with the machine learning methods, epsilon support vector regression (ε-SVR)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and recursive feature elimination (RFE),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> in which the features were removed stepwise according to their importance to the model. Here, the importance of each feature was evaluated by calculating the square of its Pearson correlation coefficient (<i>R</i><sup>2</sup>) with the ligands’ affinity data. Similar to our previous report,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> iPMF features were calculated to characterize the protein–ligand interactions, and these features included pairwise interaction features involving 17 types of atoms in the protein, 30 types of atoms in the ligands, and 11 distance ranges. In this study, a total of 1125 features were selected for further model building. A five-fold cross validation (CV) process was used to evaluate the performance of the model. Finally, we obtained a model called RTKscore with reasonable prediction ability; <i>R</i><sup>2</sup> of its 5-fold CV was 0.46. In addition, we compared RTKscore with Glide on a same test set, and RTKscore exhibited a more significant correlation with the pIC<sub>50</sub> values of these ligands (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The <i>R</i><sup>2</sup> of RTKscore was 0.59, whereas the <i>R</i><sup>2</sup> of Glide was near zero.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Workflow of RTKscore development and comparison of RTKscore with Glide on a test set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As described above, the RTKscore showed better predictive performance for FGFR ligands than a general-purpose scoring function. We therefore applied the RTKscore in a VS project for discovering novel FGFR1 inhibitors. The SPECS small-molecule database was used as the ligand databases, and compounds containing any PAINS substructures,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> inorganic atoms, unwanted functionalities, and reactive groups were filtered out first. There were 190 038 ligands that remained and were prepared for VS, and Glide was used to generate the potential binding modes of each ligand. The RTKscore was used to rescore the poses of each ligand, which included calculating the iPMF features for each pose, the feature subset extraction and rescoring with the RTKscore. According to the highest score of each ligand, as scored by RTKscore, we kept the top 1000 ligands for further evaluation. To ensure structural diversity among the ligands for biochemical testing, we clustered ligands into 100 clusters on the basis of their chemical similarities as measured by molecular fingerprints ECFP4,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and only one or two ligands from each cluster were selected. Finally, 112 ligands were selected and purchased for FGFR1 inhibitory activity evaluation. We used enzyme-linked immunosorbent assay (ELISA) to test the inhibitory activities of the ligands against FGFR1 at 50 μM and found that compound <b>8</b> showed high inhibitory activity against FGFR1, with an IC<sub>50</sub> of 114.5 ± 15.3 nM. Its inhibitory activity against FGFR1-depended KG1 cell proliferation was tested, and the results showed that this compound could inhibit the proliferation of the KG1 cell with IC<sub>50</sub> = 1107.8 ± 264.7 nM.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Hit-to-Lead Optimization by Covalent Inhibitor Design</h3><div class="NLM_p">To better investigate structure–activity relationships, we used molecular docking to simulate the binding mode of <b>8</b> with FGFR1. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, the putative binding mode of compound <b>8</b> follows the typical pattern of most reported FGFR inhibitors. It may also form hydrogen bonds with the hinge area of FGFR1, in which the pyridazine moiety of compound <b>8</b> forms three hydrogen bonds with the residues Ala564 and Glu562. The benzofuran moiety of compound <b>8</b> was located in the hydrophobic pocket of the protein active site, and the <i>N</i>-phenyl substituent of the ligand reached the solvent-exposed area of the loop part of the protein. Based on the putative binding mode of <b>8</b> with the FGFR1 protein, we found that the hydrophobic pocket of FGFR1 has additional space (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). To better accommodate the hydrophobic pocket, we first explored benzofuran substituents. After testing a few kinds of substituents, we found that compound <b>9</b> with a 3,5-dimethyl-2-benzofuran moiety shows more potent inhibitory activities, with FGFR1 IC<sub>50</sub> = 10.8 ± 1.3 nM and KG1 cell IC<sub>50</sub> = 415.7 ± 8.7 nM.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Structures of compounds <b>8</b>, <b>9</b>, <b>10</b>, and <b>11</b>; (B) putative binding mode of compound <b>8</b> with the FGFR1(PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>); (C) active site pocket of FGFR1 depicted as the mesh surface. (D) putative binding modes of compound <b>10</b> (blue) and compound <b>11</b> (orange) with FGFR1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>) simulated by covalent docking. (E) FGFR1, FGFR2, FGFR3, and FGFR4 P-loop sequence alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Recently, with the resurgence of covalent inhibitors, more and more covalent small molecule kinase inhibitors have been approved by FDA.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a> A large number of kinase targets have a cysteine located in the vicinity of the adenosine 5′-triphosphate (ATP) pocket that could be targeted by irreversible inhibitors.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> As for FGFR, covalent targeting the Cys residue in the P-loop of FGFRs is also an effective strategy for developing selective inhibitors.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As reported by the Gray Laboratory, the covalent inhibitor showed gratifying inhibitory activity against FGFR1 at both the molecular level and cellular level.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In this study, the simulated binding mode of <b>8</b> implied that its <i>N</i>-phenyl substituent extends out the P-loop of FGFR1, which contains a conserved Cys residue within the FGFR family (shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E marked with a green box, i.e., Cys 488 in FGFR1). The distance from the phenyl ring of <b>8</b> to this Cys residue was 11.1 Å, within the appropriate distance range for a reactive Michael acceptor (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Based on this observation, two covalent inhibitors <b>10</b> and <b>11</b> were designed by introducing Michael acceptor substituents with different distances to the benzene ring of compound <b>9</b>. We used covalent molecular docking to simulate the binding mode of <b>10</b> and <b>11</b> with FGFR1, and the simulated results showed that both compounds could form a covalent interaction with Cys488 of the FGFR1 P-loop without a significant change in their overall binding mode (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.3.  In Vitro Activities of Compounds <b>10</b> and <b>11</b></h3><div class="NLM_p">Compounds <b>10</b> and <b>11</b> were synthesized and tested for their inhibitory activity against FGFRs and a diverse panel of representative human kinases. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, both <b>10</b> and <b>11</b> exhibited great inhibitory activity to FGFR1, FGFR2, and FGFR3 in vitro (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Meanwhile compound <b>10</b> also showed potent inhibitory activity against FGFR4 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compounds <b>10</b> and <b>11</b> showed excellent selectivity for the FGFR were observed across typical angiogenesis-regulating kinases, including PDGFRα, PDGFRβ, VEGFR1, and VEGFR2, and other tested human kinases.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Molecular and Cellular Inhibitory Activity Evaluation of Compounds <b>10</b> and <b>11</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center"><b>10</b></th><th class="colsep0 rowsep0" align="center"><b>11</b></th><th class="colsep0 rowsep0" align="center" char="±"><b>1</b></th><th class="colsep0 rowsep0" align="center">cell</th><th class="colsep0 rowsep0" align="center"><b>10</b></th><th class="colsep0 rowsep0" align="center"><b>11</b></th><th class="colsep0 rowsep0" align="center" char="±"><b>1</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR1</td><td class="colsep0 rowsep0" align="left">4.8 ± 1.6</td><td class="colsep0 rowsep0" align="left">16.3 ± 1.1</td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.1</td><td class="colsep0 rowsep0" align="left">KG1</td><td class="colsep0 rowsep0" align="left"><0.3</td><td class="colsep0 rowsep0" align="left">0.6 ± 0.0</td><td class="colsep0 rowsep0" align="char" char="±">3.7 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR2</td><td class="colsep0 rowsep0" align="left">2.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">6.6 ± 1.6</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.0</td><td class="colsep0 rowsep0" align="left">SNU16</td><td class="colsep0 rowsep0" align="left"><0.3</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">4.9 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR3</td><td class="colsep0 rowsep0" align="left">15.0 ± 1.1</td><td class="colsep0 rowsep0" align="left">12.5 ± 1.2</td><td class="colsep0 rowsep0" align="char" char="±">3.4 ± 0.4</td><td class="colsep0 rowsep0" align="left">UMUC14</td><td class="colsep0 rowsep0" align="left">6.4 ± 2.1</td><td class="colsep0 rowsep0" align="left">36.4 ± 5.3</td><td class="colsep0 rowsep0" align="char" char="±">10.3 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR4</td><td class="colsep0 rowsep0" align="left">23.1 ± 3.6</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="char" char="±">49.8 ± 7.6</td><td class="colsep0 rowsep0" align="left">BaF3/TEL-FGFR1</td><td class="colsep0 rowsep0" align="left"><0.3</td><td class="colsep0 rowsep0" align="left">9.8 ± 2.8</td><td class="colsep0 rowsep0" align="char" char="±">0.5 ± 0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VEGFR2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±">57.9 ± 22.1</td><td class="colsep0 rowsep0" align="left">BaF3/TEL-VEGFR2</td><td class="colsep0 rowsep0" align="left">6119.8 ± 1040.6</td><td class="colsep0 rowsep0" align="left">>10 000</td><td class="colsep0 rowsep0" align="char" char="±">416.9 ± 10.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VEGFR1</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">67.6%@1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFRβ</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">58.8%@1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ret</td><td class="colsep0 rowsep0" align="left">78.4%@1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">c-Src</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">c-Met</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ALK</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ErbB2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ErbB4</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EPH-A2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ABL</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div></div><div class="NLM_p">Increased FGFR signaling promotes cancer cell proliferation. To elucidate the impact of <b>10</b> and <b>11</b> on FGFR-mediated cancer cell proliferation, five FGFR-driven cell lines harboring frequently occurring oncogenic forms of FGFRs were chosen: FGFR1-translocated KG1 cells, FGFR2-amplified H716 cells, FGFR2-amplified SNU16 cells, FGFR3-mutant UMUC14 cells, and BaF3/TEL-FGFR1 cells. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>10</b> strongly inhibited cell proliferation of KG1, H716, SNU16, UMUC14, and BaF3/TEL-FGFR1, and most of its IC<sub>50</sub> values were less than 10 nM, making it more potent than AZD4547. The model cell line BaF3/TEL-VEGFR2, which stably expressed the constitutively active form of TEL-VEGFR2, was used to further assess the cellular selectivity of compound <b>10</b> for FGFR against VEGFR2. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compared with its high potency against cell proliferation of BaF3/TEL-FGFR1 (IC<sub>50</sub> < 0.3 nM), compound <b>10</b> had no significant inhibitory effect on VEGFR2-mediated cell proliferation (IC<sub>50</sub> = 6119.8 ± 1040.6 nM), confirming the high selectivity of compound <b>10</b> for FGFRs against VEGFR2 at the cellular level.</div><div class="NLM_p">To further evaluate the cellular activity of compound <b>10</b> for targeting FGFR kinase, we analyzed its effects on the phosphorylation of FGFR and its major downstream signaling molecule, Erk. Two representative human cancer cell lines with FGFR aberrations were used, namely, KG1 and SNU16. We found that <b>10</b> showed significant inhibition of the phosphorylation of FGFR1 and FGFR2 in the individual cancer cell lines. The phosphorylation of Erk was also inhibited (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Thus, at the cellular level, <b>10</b> potently inhibits FGFR signaling.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of compound <b>10</b> on the phosphorylation of the FGFR and the downstream effector Erk in the KG1 and SNU16 cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.4.  Investigation of the Covalent Binding Mechanism Investigation</h3><div class="NLM_p">To elucidate the interaction mechanism of this series of compounds, we solved the cocrystal structures of compound <b>9</b> in complex with FGFR1 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ">6ITJ</a>), compound <b>10</b> with FGFR4 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP">6IUP</a>), and compound <b>11</b> with FGFR4 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO">6IUO</a>). Overall, these cocrystal structures provided a solid structural basis for understanding the binding mode of this series of compounds and also confirmed our molecular design concepts. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A shows the crystal structure overlaid with the predicted structure, where the binding mode of <b>9</b> determined in the crystal structure is highly consistent with the binding mode simulated by molecular docking. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B compares the crystal structures of complexed <b>10</b> and <b>11</b>, which differ from each other by only a −CH<sub>2</sub>– group in the acrylamide moiety. As expected, compounds <b>10</b> and <b>11</b> have binding modes highly similar to that of compound <b>9</b>, and the acrylamide moiety of these two compounds reached the targeted Cys residue in the P-loop. The P-loop is highly flexible, and the minor structural differences between the compounds significantly influenced the spatial arrangement of the loop. For compound <b>11</b>, a covalent bond with the Cys477 residue of FGFR4 can be observed. For compound <b>10</b>, the distance from the carbon atom of the Michael acceptor to the sulfur atom of Cys477 is 2.6 Å, which is slightly longer than the C–S bond length. Because the crystallographic structure is static and represents the spatial average over the whole crystal, we cannot rule out a potential covalent bonding interaction between <b>10</b> and FGFR4 given such a short C–S distance and the highly flexibility of the binding site.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Cocrystal structure (shown in green) of compound <b>9</b> and the FGFR1 complex (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ">6ITJ</a>) and the putative binding mode (shown in pink) of compound <b>9</b> and FGFR1 simulated by molecular docking. (B) Crystal structures of the complex of compound <b>10</b> (shown in magenta) with FGFR4 (shown in salmon) (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP">6IUP</a>) and the complex of compound <b>11</b> (shown in lemon) with FGFR4 (shown in cyan) (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO">6IUO</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To investigate the potential covalent bond between <b>10</b> and FGFR4, we used protein mass spectrometry (MS) to analyze the amino acid residues modified by compound <b>10</b> in the FGFR4 kinase domain. The FGFR4 kinase domain protein was incubated with compound <b>10</b> and digested with trypsin. Then, the tryptic peptides were subjected to liquid chromatography–MS (LC–MS)/MS analysis. The MASCOT program was used to identify protein modification by searching the Uniport database. The target peptide, LVLGKPLGEGCFGQVVR (the peptide contains the Cys residue that may be covalently bound), has been identified (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Figure S1</a>). Further analysis showed that the Cys residue of the peptide, LVLGKPLGEGCFGQVVR, was modified by 440.46 Da, which exactly matches the molecular weight of compound <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Figure S2</a>). These results indicated that compound <b>10</b> could bind to the FGFR4 protein and form specifically a covalent bond with the residue of Cys477.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.5.  Optimization of PK Properties</h3><div class="NLM_p">The pharmacokinetics (PK) properties of compound <b>10</b> in rats were determined, and it showed low oral bioavailability in rats (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). To improve the PK profile of <b>10</b>, we first used site-of-metabolism (SOM) prediction programs to identify the reactive sites of <b>10</b>. Three SOM prediction programs were used: SOME-UGT,<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> SMARTCyp,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and the metabolism prediction module<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> of Maestro (Schrödinger LLC, New York, NY, 2015). The 5-methyl group on the benzofuran moiety was predicted to be a reactive site by all three programs (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). In addition, the Michael acceptor moiety that is prone to nucleophilic reactions. Overall, we considered that the Michael acceptor of <b>10</b> and the 5-methyl group on the benzofuran are high priorities for site modification.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. SOM prediction of compound <b>10</b> by three programs. SMARTCyp (orange circles, metabolism probability ordered by the size of the circle), SOME (blue triangles) and SOME-UGT (grey diamonds), and Schrödinger (green pentagons, metabolism probability ordered by the size).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The optimization of the covalent reactive groups is a key challenge in the covalent drug design. An ideal reactive group should have sufficient reaction activity to form the desired bond with the target protein, but indiscriminate and high reactivity may lead to insurmountable PK problems and toxicity in the late stage of drug development.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> For compound <b>10</b>, we aim to slightly weaken the reactivity of the Michael acceptor, to reduce its binding to off-target nucleophiles, for example, glutathione (GSH) and proteins with exposed nucleophilic centers such as serum albumin. These unspecific interactions may account for the fast clearance of <b>10</b>. Several studies have reported that a useful optimization strategy is to add an <i>N</i>,<i>N</i>-dimethylaminomethyl substituent or analogues to the end of the Michael acceptor,<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> which would not only reduce the electrophilicity but also enhance the steric hindrance of the Michael acceptor. Moreover, a basic functional group onto the Michael acceptor of the ligand would improve the reactivity between the ligand and the target protein by catalyzing the intramolecular Michael addition and/or having an inductive effect from the protonated basic group.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Thus, we substituted the Michael acceptor moiety of compound <b>10</b> with several different amino methyl substituents.</div><div class="NLM_p">Regarding the 5-methyl group on the benzofuran moiety of <b>10</b>, the methyl group can be replaced with halogen to improve metabolic stability in a straight forward manner. In this study, we replaced the 5-methyl group with two different halogen atoms: chlorine and bromine. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, five derivatives were synthesized for biochemical and metabolic evaluation. First, we tested the inhibitory activities of these derivatives against FGFR1 and FGFR2, and their inhibitory activities on the proliferation of KG1 cells and SUN16 cells. The results suggested that these structural modifications retained the in vitro activities of these compounds. All of these derivatives potently inhibit FGFR1 and FGFR2 enzymatic activity and effectively inhibit the proliferation of FGFR1-depended cancer cells KG1. Second, we compared the solubility of these compounds using a kinetic solubility determination assay. Among them, compounds <b>10</b> and <b>12</b> showed poor solubility (20 μM) and compounds <b>13</b>, <b>14</b>, and <b>16</b> were more soluble, with solubilities comparable to that of ibrutinib<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (<b>17</b>) (an anticancer drug with low solubility<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>), and compound <b>15</b> showed the highest solubility (200 μM, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The metabolic stability of compounds <b>10</b>, <b>12</b>, <b>13</b>, and <b>15</b> in mouse liver microsomes were evaluated. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, in vitro <i>T</i><sub>1/2</sub> values for the four compounds ranged from 29 min for compound <b>12</b> and 31 and 63 min for compound <b>10</b> and compound <b>13</b>, respectively, up to 103 min for compound <b>15</b> which showed the best stability in mouse liver microsomes because of the longest half-life.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Derivatives of Compound <b>10</b> and Their Biochemical and Metabolic Evaluation</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0011.gif" alt="" id="GRAPHIC-d7e1593-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound id</th><th class="colsep0 rowsep0" align="center" char="±">FGFR1 (nM)</th><th class="colsep0 rowsep0" align="center" char="±">FGFR2 (nM)</th><th class="colsep0 rowsep0" align="center">KG1 cell (nM)</th><th class="colsep0 rowsep0" align="center" char=".">solubility at pH 7.4 (μM)</th><th class="colsep0 rowsep0" align="center" char=".">microsomal stability in vitro <i>T</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">1.5 ± 0.1</td><td class="colsep0 rowsep0" align="left"><0.3</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char="±">0.8 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">1.1 ± 0.0</td><td class="colsep0 rowsep0" align="left"><0.3</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char="±">2.4 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">1.9 ± 0.7</td><td class="colsep0 rowsep0" align="left"><0.3</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char="±">30.9 ± 7.3</td><td class="colsep0 rowsep0" align="char" char="±">8.0 ± 0.7</td><td class="colsep0 rowsep0" align="left">2.7 ± 1.4</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char="±">20.4 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="±">7.2 ± 3.6</td><td class="colsep0 rowsep0" align="left">3.7 ± 2.5</td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char=".">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">2.0 ± 0.0</td><td class="colsep0 rowsep0" align="left">0.4 ± 0.1</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.0</td><td class="colsep0 rowsep0" align="left">8.3 ± 2.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div></div><div class="NLM_p">To cost-effectively evaluate the reactivity of the designed covalent inhibitors, their undesirable reactions with GSH protein were evaluated in vitro. GSH protein was incubated separately with compounds <b>10</b>, <b>13</b>, <b>15</b>, and <b>16</b> separately, and then their GSH binding metabolites were determined with LC–MS/MS, and covalent drugs <b>17</b> and afatinib<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> (<b>18</b>) were chosen as controls. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, the GSH adduct of <b>10</b> was the only quickly generated metabolite, and other compounds demonstrated comparable stabilities toward off-target thiols of GSH. Thus, we supposed that the poor PK profile of compound <b>10</b> is mainly caused by its high intrinsic chemical reactivity profile and the modified acrylamide functional groups in compounds <b>13</b>, <b>15</b>, and <b>16</b> have acceptable reactivity profiles compared with the covalent drugs on the market. This argument is further supported by the following rat whole blood stability assay,<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> in which we incubated compounds <b>13</b> and <b>15</b> in whole blood for 5 h and detected the remaining parent compounds at different time points. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B, the prototype of compound <b>10</b> decreased quickly, and compound <b>15</b> decreased relatively slowly. Moreover, after 5 h, with <b>10</b>, the parent compound tended to be depleted, while 20% of compound <b>15</b> remained. In general, compound <b>15</b> showed good solubility, a weaker binding affinity for GSH, and improved stability in whole blood. Therefore, we chose <b>15</b> as the lead compound for further characterization.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) GSH affinity evaluation of compounds <b>10</b>, <b>13</b>, <b>15</b>, and <b>16</b>; (B) blood stability evaluation of compounds <b>10</b> and <b>15</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">2.6.  Characterization of Lead Compound <b>15</b></h3><div id="sec2_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12">2.6.1.  Biochemical Kinetic Profiling</h4><div class="NLM_p">We further did the kinetic assessment of compounds <b>10</b> and <b>15</b>. FGFR1 was used as the representative target kinase. At first, a classic diluting assay was conducted to determine whether compounds <b>10</b> and <b>15</b> inhibited FGFR via irreversible binding. As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, in contrast to the known reversible FGFR inhibitor <b>1</b> that dissociate quickly allowing recovery of enzyme activity, compounds <b>10</b> and <b>15</b> markedly prevent recovery of enzyme activity, strongly supporting the irreversible binding mode of compounds <b>10</b> and <b>15</b> to FGFR1.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound <b>10</b> (A) and <b>15</b> (B) irreversibly binds to FGFR1, compared with compound <b>1</b> (C) that is a known reversible FGFR inhibitor. Enzyme activity of FGFR1 was assayed by a Caliper EZ Reader under three different conditions: without the enzyme (background), without the compound (nonpre-incubation control), and pre-incubated with the compound. “Conversion” here represents the enzyme activity and means “the percent of conversion of the substrate peptide”.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Then, we conducted traditional kinetics analysis measuring <i>K</i><sub>inact</sub>/<i>K</i><sub>i</sub> which defines the second order rate constant for covalent binding to the target protein.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The efficient covalent target engagement is apparent for compound <b>10</b> and <b>15</b> to FGFR1 with a calculated <i>K</i><sub>inact</sub>/<i>K</i><sub>i</sub> ratio of 0.38, 0.070 (μM<sup>–1</sup> s<sup>–1</sup>), respectively (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This <i>K</i><sub>inact</sub>/<i>K</i><sub>i</sub> value was comparable to some EGFR covalent inhibitors or the FGFR covalent inhibitor.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> In addition, compared with compound <b>10</b> (<i>K</i><sub>i</sub> = 1.9 nM), weaker affinity for compound <b>15</b> to FGFR1 (<i>K</i><sub>i</sub> = 10 nM) was observed, which is consistent with the biochemical and cellular IC<sub>50</sub> data (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The kinetics studies also suggested covalent irreversible binding of the compound <b>10</b> and <b>15</b> to FGFR.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Kinetics Values of Compounds <b>10</b> and <b>15</b> Binding to FGFR1</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="×" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound id</th><th class="colsep0 rowsep0" align="center" char="×"><i>K</i><sub>inact</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>inact</sub>/<i>K</i><sub>i</sub> (μM<sup>–1</sup> s<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char="×">7.2 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char="×">7.1 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.070</td></tr></tbody></table></div></div></div><div id="sec2_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14">2.6.2.  Protein Kinase Profiling</h4><div class="NLM_p">To better understand the general kinase target selectivity trends and potential toxicity poisonousness of compound <b>15</b>, the commercial KinaseProfiler Service (Eurofins Scientific, Inc.) was utilized to evaluate the selectivity of <b>15</b> against a panel of 405 kinases at three fixed concentrations (10, 100, and 1000 nM). The KinaseProfiler assay protocols measure the percent inhibition of phosphorylation of a peptide substrate in the presence of a fixed concentration of ATP (10 μM). Remarkably, compound <b>15</b> showed high specificity for FGFR kinases over all other kinases tested, and no appreciable inhibition of the other kinases was observed in the presence of 10 nM, with most maintaining >70% of their control activity. At a concentration of 100 nM, compound <b>15</b> inhibited a small number of kinases other than FGFRs, including c-RAF, SIK, DDR1, Arg, Yes, Flt1, and DNA-PK. The location of the profiled kinases and active interactions depicted in a kinase dendrogram tree is shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. Details of profiles including all the data from the Eurofins KinaseProfiler (405 targets) are available in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Supporting Information</a>. This experiment further supported that compound <b>15</b> has high selectivity for the FGFR family.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Kinase profile of compound <b>15</b> drawn with KinMap.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Kinase inhibition ratios above 50% at corresponding concentrations are marked as red circles.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15">2.6.3.  In Vivo PK Evaluation</h4><div class="NLM_p">The in vivo PK evaluation of compound <b>15</b> was performed in rats via IV (10 mg/kg), PO (20 mg/kg) and IP (20 mg/kg). Compared with compound <b>10</b>, the PK profile of compound <b>15</b> shows a significant improvement. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, when dosing via IV, the plasma exposure of compound <b>15</b> was AUC<sub>0–<i>t</i></sub> = 4412.8 ng·h/mL that was four times of compound <b>10</b>, and the half-life period of compound <b>15</b> was 8.55 h. When dosing via IP, plasma exposure was AUC<sub>0–<i>t</i></sub> = 16790.8 ng·h/mL and the half-life period was 4.63 h. Unfortunately, when dosing via PO, the bioavailability of <b>15</b> was almost zero. The results revealed that <b>15</b> has an acceptable PK profile only when dosing via IV and IP.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pharmacokinetic Properties of Compounds <b>10</b> and <b>15</b> in SD Rats</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">dose (mg/kg)</th><th class="rowsep1 colsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="rowsep1 colsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="rowsep1 colsep0" align="center" char=".">AUC<sub>0–<i>t</i></sub> (ng·h/mL)</th><th class="rowsep1 colsep0" align="center" char=".">AUC<sub>0–∞</sub> (ng·h/mL)</th><th class="rowsep1 colsep0" align="center" char=".">MRT (h)</th><th class="rowsep1 colsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th><th class="rowsep1 colsep0" align="center" char=".">CL (L/h/kg)</th><th class="rowsep1 colsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="rowsep1 colsep0" align="center">F (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char=".">0.083</td><td class="colsep0 rowsep0" align="char" char=".">1878.0</td><td class="colsep0 rowsep0" align="char" char=".">1093.9</td><td class="colsep0 rowsep0" align="char" char=".">1272.4</td><td class="colsep0 rowsep0" align="char" char=".">5.04</td><td class="colsep0 rowsep0" align="char" char=".">6.25</td><td class="colsep0 rowsep0" align="char" char=".">8.60</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char=".">0.861</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">2.17</td><td class="colsep0 rowsep0" align="char" char=".">1.77</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">4308.7</td><td class="colsep0 rowsep0" align="char" char=".">4412.8</td><td class="colsep0 rowsep0" align="char" char=".">1.64</td><td class="colsep0 rowsep0" align="char" char=".">8.55</td><td class="colsep0 rowsep0" align="char" char=".">2.27</td><td class="colsep0 rowsep0" align="char" char=".">28.0</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">32.6</td><td class="colsep0 rowsep0" align="char" char=".">51.5</td><td class="colsep0 rowsep0" align="char" char=".">7.33</td><td class="colsep0 rowsep0" align="char" char=".">21.64</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">∼0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">IP</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">5042.5</td><td class="colsep0 rowsep0" align="char" char=".">16790.8</td><td class="colsep0 rowsep0" align="char" char=".">16984.3</td><td class="colsep0 rowsep0" align="char" char=".">3.30</td><td class="colsep0 rowsep0" align="char" char=".">4.63</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div></div></div><div id="sec2_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17">2.6.4.  Species Difference Study</h4><div class="NLM_p last">To understand the metabolic species difference of <b>15</b>, we used an in vitro model of hepatic subcellular fractions to incubate compounds and analyzed the metabolites with UPLC-UV/Q-TOF MS. The metabolites were ranked in ascending order by their mass-to-charge ratio or their LC retention time when the metabolites had the same mass-to-charge ratio. The results of the metabolism of <b>15</b> by hepatic subcellular fractions (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Table S4</a>) show that <b>15</b> was comparatively stable that after 180 min of incubation, and only 20% of the parent compounds was metabolized. The main metabolic pathways of <b>15</b> are amino oxidation, GSH binding, and amino demethylation. <b>15</b> is more likely to bind GSH in human and monkey hepatic subcellular fractions than it is in dogs, rats, and mice. The proposed metabolic pathways of <b>15</b> in the hepatics subcellular fractions are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Figure S3</a>. Species differences suggested that there are few variations among hepatic metabolic pathways of <b>15</b> in dogs, rats, mice, monkeys, and humans. In addition, the metabolism of <b>15</b> in human hepatic subcellular fractions is simpler than those of other species.</div></div><div id="sec2_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">2.6.5.  hERG Inhibitory Activity</h4><div class="NLM_p">A blockade of the hERG channel may cause drug-induced prolongation of the QT interval, which has become a major concern in drug discovery and development. To evaluate the cardiotoxicities of the candidates, we used a patch-clamp experiment to determine the hERG inhibitory activities of compounds <b>9</b>, <b>10</b>, <b>13</b>, and <b>15</b>, with cisapride<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> (<b>19</b>) as the positive control. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, the hERG IC<sub>50</sub> values of most of the compounds are higher than 10 μM, and compounds <b>10</b> and <b>15</b> exhibited hERG IC<sub>50</sub> values higher than 40 μM. In general, a safe drug candidate is expected to have an hERG IC<sub>50</sub> value greater than 10 μM,<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> so compound <b>15</b> has a relatively low risk of hERG toxicity.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. hERG Inhibitory Activity Evaluation</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound id</th><th class="colsep0 rowsep0" align="center"><b>9</b></th><th class="colsep0 rowsep0" align="center"><b>10</b></th><th class="colsep0 rowsep0" align="center"><b>13</b></th><th class="colsep0 rowsep0" align="center"><b>15</b></th><th class="colsep0 rowsep0" align="center"><b>19</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hERG IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">19.53</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">32.85</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">0.03</td></tr></tbody></table></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">CADD has the potential to improve the efficiency of drug research and discovery to assist or accelerate decision making throughout the entire development process. Although an increasing number of successful drug discovery cases involving CADD methods are being reported,<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> the shortcomings of CADD methods are also obvious. For instance, because of inaccurate or inapplicable computational models,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> the false positive ratio in VSs remains high, and it is still of challenge to reliably prioritize synthetic efforts to focus on the optimization of compounds with favorable activities as well as absorption, distribution, metabolism, excretion, and toxicity properties. The rational application of CADD methods requires experimental researchers to better understand the limitations of computational models and choose the appropriate models by comprehensively considering their computation costs and the expected accuracy level. In this study, we first devised an FGFR-targeted scoring function, the RTKscore, to address the inaccuracy associated with activity rank-ordering of VSs. Based on the RTKscore, we successfully identified compound <b>8</b> from a commercial chemical library, and this compound has a novel structure and exhibits subnanomolar inhibitory activity (FGFR1 IC<sub>50</sub> = 114.5 ± 15.3 nM) against FGFR. Structure-guided optimization then led to compound <b>9</b> with improved molecular and whole-cell activities against the FGFR (FGFR1 IC<sub>50</sub> = 10.8 ± 1.3 nM, KG1 IC<sub>50</sub> = 554.4 ± 447.2 nM) and covalent compound <b>10</b> with subnanomolar inhibitory activity toward FGFRs and FGFR-dependent cancer cell lines. Molecular biology experiments, X-ray crystallography, and protein MS studies showed that <b>10</b> could selectively target the FGFR and form a covalent bond with the Cys residue in the loop of FGFRs. Despite its excellent in vitro activity, compound <b>10</b> showed a poor PK profile because of its low solubility in water and metabolic instability issues. To address these problems, we further used SOM prediction to identify the metabolically labile sites on compound <b>10</b> and successfully obtained a more soluble and stable compound, <b>15</b>. In vivo PK experiments indeed showed that <b>15</b> has high plasma exposure and an acceptable <i>t</i><sub>1/2</sub> when dosing via IP and IV, despite there being no improvement in oral exposure. Moreover, the hERG inhibitory activity evaluation experiment indicated that <b>15</b> has a relatively low risk of hERG toxicity. These results indicated that compound <b>15</b> could be a promising FGFR lead compound. Taken together, this study describes our efforts in the rational design of novel FGFR inhibitors. Accordingly, further medicinal chemistry exploration around compound <b>15</b> is currently ongoing with the aim to provide validated candidates for the novel anticancer treatments.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41317" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41317" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22">4.1.  RTKscore Development</h3><div class="NLM_p">The protocol for the development of the RTKscore is similar to that of SAMscore development. First, we collected the crystal structures of FGFR1 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>), FGFR2 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PVF">2PVF</a>), and FGFR3 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K33">4K33</a>) from PDB, and the ligand information for these proteins from the BindingDB, and these data point included the ligand structure and activity information with the corresponding target protein. A total of 1090 ligands were collected. The activity data of these ligands were converted to pIC<sub>50</sub> (pIC<sub>50</sub> = 9 – log<sub>10</sub>(IC<sub>50</sub>)), and the ligands without the explicit data value were marked as inactive with pIC<sub>50</sub> = 0. We divided all the ligands into two groups, the training set and the test data in a ratio of 5:1, in which the training data set had 908 ligands and the test data set had 182 ligands.</div><div class="NLM_p">The molecular docking program Glide was used to generate the binding modes of each ligand with the corresponding target protein. Before molecular docking, the ligands were prepared by the LigPrep module in Schrödinger software (LigPrep, version 3.4; Schrödinger, LLC: New York, NY, 2015), and the protein structures were optimized by the module of Protein Preparation Wizard module in the Maestro program (Maestro, version 10; Schrödinger, LLC: New York, NY, 2015). Based on the optimized protein structures, protein grid files were generated with the center of the ATP binding site by the Receptor Grid Generation module in the Glide program (Schrödinger, LLC: New York, NY, 2015). The prepared ligand conformations were docked to the corresponding target protein grid files by Glide with the SP precision mode. All parameters for the above processes were the default parameters. The conformation of the lowest Glide Emodel score for each ligand was selected as the pose of the ligand for further study.</div><div class="NLM_p last">Based on the binding mode of the ligands, the iPMF features for each ligand were calculated with in-house Python scripts for each ligand. Seventeen types of atoms in the proteins, 30 types of atoms in the ligand, and 11 distance ranges (0.0–2.0, 0.0–3.0, ..., and 0.0–12.0 Å) were considered. Overall, there were 5610 iPMF features for each ligand. Then, ε-SVR and RFE methods were used to perform feature selection, and the unimportant features were eliminated step by step until the optimal feature sub set of features was identified. The importance of each feature was evaluated by the square of the Pearson correlation coefficient. Fifty unimportant features were eliminated every time, and the data set with sub features were trained with the ε-SVR method and evaluated by 5-fold CV. Finally, the model with 1125 features, the RTKscore, was obtained. The RTKscore was compared with Glide on the test data set, and the squares of the correlation efficient of the ligands’ pIC<sub>50</sub> values with the score predicted RTKscore or Glide was calculated.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23">4.2.  Virtual Screening</h3><div class="NLM_p last">The SPECS database, containing 207, 163 compounds, was used as the ligand database for VS. First, we removed compounds containing inorganic atoms, PAINS substructures, low draggability, or reactive substructures. After filtering, the remaining 190 038 compounds were prepared using the same protocol used for RTKscore development for VS. The prepared protein structure of FGFR1 (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>) from RTKscore development was chosen as the receptor structure for VS. The prepared ligands were docked to the receptor protein structure with Glide, and five conformations of each ligand were output. Next, we used the RTKscore to evaluate each ligand pose, which included iPMF features calculation, feature selection, and score prediction. According to ligands’ scores predicted by the RTKscore, the top 1000 ligands were kept for further analysis. We clustered the ligands into 100 clusters based on fingerprint ECFP4 with software Pipeline Pilot v7.5 software. To ensure selected ligands were structurally diverse, we selected one or two compounds from each cluster. Finally, there were 112 compounds were selected and purchased for biochemical evaluations.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24">4.3.  Molecular Modeling</h3><div class="NLM_p">For noncovalent inhibitors, the binding mode of the ligand and protein was simulated with the molecular docking program Glide. The protocol of the molecular docking was similar to that mentioned above, and we outputted all conformations of the ligand and checked the conformations manually.</div><div class="NLM_p last">For covalent inhibitors, we simulated their binding mode with CovDock,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> a covalent docking program from Schrödinger (Schrödinger, LLC: New York, NY, 2015). As the residue Cys488 of the FGFR1 crystal structure (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>) was mutated to an alanine, we first mutated the residue 488 back to cystine, and then the protein structure and ligands were prepared with the same protocol as described above. After choosing Cys488 of FGFR1 as the reaction residues of the target protein, and the reaction type was set as a Michael addition reaction. Other parameters were left at the default settings.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25">4.4.  General Chemistry Information</h3><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, we developed an efficient route to obtain derivatives <b>8–16</b>. Herein, we use 2′-hydroxyacetophenone derivatives <b>17a–d</b> as the starting materials (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) and treated them with ethyl bromoacetate and K<sub>2</sub>CO<sub>3</sub> in dimethylformamide (DMF) to obtain the intermediates <b>19a–d</b>. Subsequent treatment with <b>21a–c</b> to afford the pyrazole intermediates <b>22a–f</b>. Compounds <b>8</b>, <b>9</b>, and <b>23a–d</b> were synthesized by the reaction of intermediates <b>22a–f</b> and hydrazine hydrate in EtOH under one-pot conditions. The reductions of compounds <b>23a–c</b> with Fe powder afforded the aniline derivatives, which were ideally poised to undergo acylation with the corresponding acyl chlorides to afford compounds <b>10</b> and <b>12–16</b>. The Cbz group of <b>23d</b> was deprotected by Pd(OH)<sub>2</sub>/H<sub>2</sub> in MeOH to generate the amine intermediate, which could then undergo acylation with acryloyl chloride to afford compound <b>11</b>.</div><figure id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl bromoacetate, K<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C; (b) CH<sub>3</sub>CN, NaH, THF, 50 °C; (c) ethyl 2-chloroacetoacetate, NaNO<sub>2</sub>, NaOAc, EtOH, H<sub>2</sub>O/HCl (3:1, v/v), 0 °C-rt; (d) Et<sub>3</sub>N, DCM, rt; (e) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, HCl (conc.), EtOH, 100 °C, MW; (f) for compounds <b>23a–c</b>, Fe, HCl (conc.), MeOH/H<sub>2</sub>O (6:1, v/v), 80 °C, 1 h; (g) acryloyl chloride or acryloyl chloride amino derivatives, K<sub>2</sub>CO<sub>3</sub>, THF, rt; (h) for compound <b>23d</b>, Pd(OH)<sub>2</sub>/C, HCl (conc.), MeOH, 40 °C, 8 h.</p></p></figure></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26">4.5.  General Methods</h3><div class="NLM_p last">The reagents (chemicals) were purchased and used without further purification. Nuclear magnetic resonance (NMR) spectroscopy was performed on a Bruker AMX-400 and AMX-300 NMR (TMS as the IS). Chemical shifts are reported in parts per million (ppm, δ) downfield from TMS. Proton coupling patterns are described as singlet (s), doublet (d), triplet (t), quartet (q), and multiplet (m). Low- and high-resolution mass spectra (LRMS and HRMS) were acquired with electron impact, electrospray, and matrix-assisted laser desorption ionization (EI, ESI, and MALDI) analyses on a Finnigan MAT-95, LCQ-DECA spectrometer and an IonSpec 4.7 T instrument. High-performance LC (HPLC) analyses of all final compounds for biological testing were carried out on an Agilent 1260 Series HPLC with an Agilent Extend-C18 column (150 × 4.6 mm, 5 μm) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Table S5</a>). All final compounds achieved a minimum of 95% purity. The known acryloyl chloride amino derivatives were already prepared.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27">4.6.  4-Amino-3-(3-methylbenzofuran-2-yl)-2-phenyl-2,6-dihydro-7<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-7-one (<b>8</b>)</h3><div class="NLM_p last">To a solution of <b>22a</b> (500 mg, 1.35 mmol) in EtOH (5 mL) was added hydrazine hydrate (472 mg, 9.42 mmol). The mixture was stirred at 60 °C overnight. The reaction mixture was diluted with water (20 mL). The reaction mixture was then extracted with DCM (3 × 30 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was dissolved in EtOH (5 mL) and added few drops HCl (conc.). The mixture was stirred at 100 °C for 2 h under microwave irradiation. The reaction mixture was then concentrated under reduced pressure and purified by column chromatography using silica with 10% MeOH in DCM as the eluent to give <b>8</b> (210 mg, 32% yield), as a yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.48 (s, 1H), 7.66 (d, <i>J</i> = 7.5 Hz, 1H), 7.59 (d, <i>J</i> = 8.3 Hz, 1H), 7.50–7.39 (m, 6H), 7.36–7.30 (m, 1H), 5.20 (s, 2H), 1.92 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO): δ 155.00, 154.47, 143.19, 143.11, 138.92, 137.37, 129.44, 128.61, 127.63, 126.19, 124.65, 123.22, 120.64, 119.85, 114.82, 111.54, 8.12. LRMS (ESI, <i>m</i>/<i>z</i>): 358.0 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>15</sub>N<sub>5</sub>O<sub>2</sub> ([M + H]<sup>+</sup>), 358.1299; found, 358.1290.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28">4.7.  4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-2-phenyl-2,6-dihydro-7<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-7-one (<b>9</b>)</h3><div class="NLM_p last">Compound <b>9</b> was prepared in a similar manner as described for compound <b>8</b>. It was obtained in 31% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.49 (s, 1H), 7.52–7.36 (m, 7H), 7.26–7.17 (m, 1H), 5.19 (s, 2H), 3.07 (s, 3H), 1.87 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.02, 152.94, 143.21, 143.10, 138.92, 137.43, 132.44, 129.46, 129.41, 128.67, 127.74, 127.39, 124.60, 120.25, 119.59, 114.76, 111.12, 20.86, 8.11. LRMS (ESI, <i>m</i>/<i>z</i>): 372.0 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>17</sub>N<sub>5</sub>O<sub>2</sub> ([M + Na]<sup>+</sup>), 394.1274; found, 394.1272.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29">4.8.  <i>N</i>-(4-(4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)phenyl)acrylamide (<b>10</b>)</h3><div class="NLM_p last">To a solution of <b>23a</b> (500 mg, 1.2 mmol) in MeOH/H<sub>2</sub>O (6:1, 14 mL) was added HCl (0.5 mL) and Fe powder (268 mg, 4.8 mmol). The mixture was stirred at 80 °C for 6 h. The reaction mixture was filtered and the filtrate was concentrated under reduced pressure. The residue was dissolved in tetrahydrofuran (THF) (5 mL) and K<sub>2</sub>CO<sub>3</sub> was added (248.8 mg, 1.8 mmol). Then, a solution of acryloyl chloride (217.2 mg, 2.4 mmol) in THF (5 mL) was added to the reaction mixture dropwise at rt. Until thin-layer chromatography (TLC) showed the reaction completed, the reaction mixture was then concentrated under reduced pressure and purified by column chromatography using silica with 10% MeOH in DCM as the eluent to give <b>10</b> (120 mg, 23% yield), as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.48 (s, 1H), 10.95 (s, 1H), 7.82 (d, <i>J</i> = 8.9 Hz, 2H), 7.55–7.32 (m, 4H), 7.22 (d, <i>J</i> = 8.9 Hz, 1H), 6.59 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.25 (d, <i>J</i> = 17.0 Hz, 1H), 5.75 (d, <i>J</i> = 10.2 Hz, 1H), 5.19 (s, 2H), 2.41 (s, 3H), 1.89 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.55, 155.00, 152.93, 143.17, 142.91, 140.04, 137.48, 133.82, 132.40, 131.78, 128.67, 127.58, 127.34, 127.15, 125.05, 120.25, 119.47, 114.61, 111.11, 20.86, 8.14. LRMS (ESI, <i>m</i>/<i>z</i>): 441.9 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>21</sub>N<sub>6</sub>O<sub>3</sub> ([M + H]<sup>+</sup>), 441.1670; found, 441.1681.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30">4.9.  <i>N</i>-(4-(4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)benzyl)acrylamide (<b>11</b>)</h3><div class="NLM_p last">To a solution of <b>23d</b> (540 mg, 1.0 mmol) in MeOH (5 mL) was added HCl (1 mL) and Pd(OH)<sub>2</sub>/C (20%, 35 mg, 0.05 mmol). The mixture was stirred at 40 °C overnight under the H<sub>2</sub> atmosphere. The reaction mixture was filtered and the filtrate was concentrated under reduced pressure. The residue was dissolved in THF (5 mL) and added K<sub>2</sub>CO<sub>3</sub> (207.3 mg, 1.5 mmol). Then, a solution of acryloyl chloride (181 mg, 2 mmol) in THF (5 mL) was added to the reaction mixture dropwise at rt. Until TLC showed the reaction completed, the reaction mixture was then concentrated under reduced pressure and purified by column chromatography using silica with 10% MeOH in DCM as the eluent to give <b>11</b> (99 mg, 21% yield), as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.51 (s, 1H), 8.82 (t, <i>J</i> = 5.9 Hz, 1H), 7.48 (d, <i>J</i> = 8.4 Hz, 1H), 7.45–7.38 (m, 3H), 7.32 (d, <i>J</i> = 8.5 Hz, 2H), 7.22 (d, <i>J</i> = 8.6 Hz, 1H), 6.30 (dd, <i>J</i> = 17.1, 10.2 Hz, 1H), 6.12 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.61 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 5.19 (s, 2H), 4.37 (d, <i>J</i> = 6.0 Hz, 2H), 2.40 (s, 3H), 1.89 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.88, 155.09, 153.01, 143.26, 143.06, 140.81, 137.66, 137.42, 132.52, 131.52, 128.70, 128.06, 127.70, 127.46, 125.78, 124.56, 120.32, 119.69, 114.86, 111.22, 41.54, 20.91, 8.19. LRMS (ESI, <i>m</i>/<i>z</i>): 456.0 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>23</sub>N<sub>6</sub>O<sub>3</sub> ([M + H]<sup>+</sup>), 455.1826; found, 455.1837.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31">4.10.  <i>N</i>-(4-(4-Amino-3-(5-chloro-3-methylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)phenyl)acrylamide (<b>12</b>)</h3><div class="NLM_p last">Compound <b>12</b> was prepared in a similar manner as described for compound <b>10</b>. It is obtained in 20% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.50 (s, 1H), 10.43 (s, 1H), 7.82–7.69 (m, 3H), 7.64 (d, <i>J</i> = 8.8 Hz, 1H), 7.45–7.38 (m, 3H), 6.43 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.26 (d, <i>J</i> = 17.0 Hz, 1H), 5.78 (d, <i>J</i> = 11.2 Hz, 1H), 5.23 (s, 2H), 1.95 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.45, 154.95, 153.03, 143.17, 142.99, 139.85, 139.00, 133.88, 131.53, 130.38, 127.71, 127.56, 126.98, 125.98, 125.23, 120.29, 119.79, 119.57, 114.86, 113.23, 8.17. LRMS (ESI, <i>m</i>/<i>z</i>): 461.0 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>17</sub>N<sub>6</sub>O<sub>3</sub>Cl ([M + Na]<sup>+</sup>), 483.0943; found, 483.0952.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32">4.11.  (<i>E</i>)-<i>N</i>-(4-(4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)phenyl)-4-(dimethylamino)but-2-enamide (<b>13</b>)</h3><div class="NLM_p last">Compound <b>13</b> was prepared in a similar manner as described for compound <b>10</b>. It is obtained in 19% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.48 (s, 1H), 10.61 (s, 1H), 7.80–7.71 (m, 2H), 7.49–7.37 (m, 4H), 7.23 (dd, <i>J</i> = 8.5, 1.6 Hz, 1H), 6.76 (d, <i>J</i> = 15.4 Hz, 1H), 6.38 (d, <i>J</i> = 15.4 Hz, 1H), 5.18 (s, 2H), 2.41 (s, 3H), 2.36 (s, 6H), 1.89 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.21, 155.08, 152.99, 143.25, 142.97, 139.95, 137.51, 133.94, 132.49, 128.72, 127.67, 127.41, 125.21, 120.29, 119.55, 114.67, 111.17, 58.66, 44.04, 20.91, 8.20. HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>26</sub>N<sub>7</sub>O<sub>3</sub> ([M – H]<sup>−</sup>), 496.2103; found, 496.2109.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33">4.12.  (<i>E</i>)-<i>N</i>-(4-(4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)phenyl)-4-morpholinobut-2-enamide (<b>14</b>)</h3><div class="NLM_p last">Compound <b>14</b> was prepared in a similar manner as described for compound <b>10</b>. It is obtained in 9% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.48 (s, 1H), 10.41 (s, 1H), 7.74 (d, <i>J</i> = 9.0 Hz, 2H), 7.49 (d, <i>J</i> = 8.4 Hz, 1H), 7.44 (s, 1H), 7.41–7.36 (m, 2H), 7.23 (dd, <i>J</i> = 8.5, 1.6 Hz, 1H), 6.79–6.70 (m, 1H), 6.33 (d, <i>J</i> = 15.1 Hz, 1H), 5.19 (s, 2H), 3.72–3.57 (m, 4H), 3.29–3.11 (m, 2H), 2.49–2.30 (m, 7H), 1.90 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.32, 155.00, 152.93, 143.16, 142.93, 139.88, 137.48, 133.87, 132.40, 128.69, 127.59, 127.33, 125.18, 120.23, 119.51, 119.46, 114.62, 111.12, 66.00, 52.99, 45.66, 20.86, 8.15. HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>28</sub>N<sub>7</sub>O<sub>4</sub> ([M – H]<sup>−</sup>), 538.2208; found, 538.2211.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34">4.13.  (<i>E</i>)-<i>N</i>-(4-(4-Amino-3-(5-chloro-3-methylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)phenyl)-4-(dimethylamino)but-2-enamide (<b>15</b>)</h3><div class="NLM_p last">Compound <b>15</b> was prepared in a similar manner as described for compound <b>10</b>. It is obtained in 25% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.49 (s, 1H), 10.70 (s, 1H), 7.80–7.73 (m, 3H), 7.64 (d, <i>J</i> = 8.8 Hz, 1H), 7.49–7.31 (m, 3H), 6.83–6.73 (m, 1H), 6.47 (d, <i>J</i> = 15.4 Hz, 1H), 5.23 (s, 2H), 3.92 (d, <i>J</i> = 6.8 Hz, 2H), 2.75 (s, 6H), 1.95 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.32, 154.92, 153.03, 143.14, 142.99, 139.67, 138.97, 134.05, 131.54, 130.38, 127.69, 126.98, 125.97, 125.27, 120.28, 119.79, 119.68, 114.85, 113.22, 56.64, 41.92, 8.16. HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>25</sub>ClN<sub>7</sub>O<sub>3</sub> ([M + H]<sup>+</sup>), 518.1702; found, 518.1707.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35">4.14.  (<i>E</i>)-<i>N</i>-(4-(4-Amino-3-(5-bromo-3-methylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)phenyl)-4-(dimethylamino)but-2-enamide (<b>16</b>)</h3><div class="NLM_p last">Compound <b>16</b> was prepared in a similar manner as described for compound <b>10</b>. It is obtained in 14% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.50 (s, 1H), 11.02 (s, 1H), 7.92 (d, <i>J</i> = 5.1 Hz, 1H), 7.82 (d, <i>J</i> = 9.0 Hz, 2H), 7.58–7.52 (m, 2H), 7.42–7.34 (m, 2H), 6.86–6.77 (m, 1H), 6.55 (d, <i>J</i> = 15.4 Hz, 1H), 5.23 (s, 3H), 3.73 (d, <i>J</i> = 5.7 Hz, 2H), 2.61 (s, 6H), 1.94 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.77, 155.01, 153.42, 143.23, 143.00, 140.00, 138.80, 133.90, 130.99, 128.71, 127.01, 125.20, 124.80, 123.33, 121.59, 119.72, 119.64, 115.59, 114.88, 113.72, 57.27, 42.51, 8.19. HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>25</sub>N<sub>7</sub>O<sub>3</sub>Br ([M + H]<sup>+</sup>), 562.1197; found, 562.1210.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36">4.15.  Ethyl 3-Methylbenzofuran-2-carboxylate (<b>18a</b>)</h3><div class="NLM_p last">To a solution of 2′-hydroxyacetophenone <b>17a</b> (2.0 g, 14.4 mmol) in dry DMF (20 mL) was added K<sub>2</sub>CO<sub>3</sub> (2.98 g, 21.6 mmol) and ethyl bromoacetate (2.88 g, 17.3 mmol), respectively. The mixture was stirred at 100 °C overnight under Ar. The reaction mixture was then cooled to rt. The suspension was filtered through a Celite cartridge and the cartridge was rinsed with EA. The filtrate was diluted with EA and water. The reaction mixture was then extracted with EA (3 × 40 mL) and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography on silica gel with using 5% EA in hexane as the eluent to give <b>18a</b> (1.8 g, 61% yield), as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.64–7.62 (m, 1H), 7.56–7.52 (m, 1H), 7.46–7.42 (m, 1H), 7.32–7.26 (m, 1H), 4.45 (q, <i>J</i> = 7.1 Hz, 2H), 2.59 (s, 3H), 1.44 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37">4.16.  Ethyl 3,5-Dimethylbenzofuran-2-carboxylate (<b>18b</b>)</h3><div class="NLM_p last">Compound <b>12b</b> was prepared in a similar manner as described for compound <b>11a</b>. It is obtained in 41% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.45–7.34 (m, 2H), 7.27–7.21 (m, 1H), 4.44 (q, <i>J</i> = 7.1 Hz, 2H), 2.56 (s, 3H), 2.46 (s, 3H), 1.44 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38">4.17.  Ethyl 5-Chloro-3-methylbenzofuran-2-carboxylate (<b>18c</b>)</h3><div class="NLM_p last">Compound <b>12c</b> was prepared in a similar manner as described for compound <b>11a</b>. It is obtained in 43% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.60 (d, <i>J</i> = 2.1 Hz, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.39 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 4.45 (q, <i>J</i> = 7.1 Hz, 2H), 2.56 (s, 3H), 1.44 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39">4.18.  Ethyl 5-Bromo-3-methylbenzofuran-2-carboxylate (<b>18d</b>)</h3><div class="NLM_p last">Compound <b>12d</b> was prepared in a similar manner as described for compound <b>11a</b>. It is obtained in 45% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.76 (d, <i>J</i> = 1.8 Hz, 1H), 7.52 (dd, <i>J</i> = 8.8, 1.8 Hz, 1H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 4.45 (q, <i>J</i> = 7.1 Hz, 2H), 2.55 (s, 3H), 1.44 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40">4.19.  3-(3-Methylbenzofuran-2-yl)-3-oxopropanenitrile (<b>19a</b>)</h3><div class="NLM_p last">To a solution of ethyl 3-methyl-2-benzofurancarboxylate <b>11a</b> (2 g, 9.79 mmol) and acetonitrile (1.0 mL, 19.6 mmol) in dry THF (25 mL) was added sodium hydride (60%, 0.51 g, 12.7 mmol) portionwise. The resulting mixture was stirred at 50 °C for 8 h under Ar and then allowed to cool to room temperature. The mixture was quenched with saturated ammonium chloride solution (50 mL) and then was acidified with 1 N HCl solution until pH 5–6. The resulting precipitate was filtered, washed with water, and dried to afford <b>12a</b> (1.3 g) as a pale yellow solid. The compound was used without further purification. LRMS (ESI, <i>m</i>/<i>z</i>): 198.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41">4.20.  3-(3,5-Dimethylbenzofuran-2-yl)-3-oxopropanenitrile (<b>19b</b>)</h3><div class="NLM_p last">Compound <b>19b</b> was prepared in a similar manner as described for compound <b>19a</b>. It is obtained in 73% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 212.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42">4.21.  3-(5-Chloro-3-methylbenzofuran-2-yl)-3-oxopropanenitrile (<b>19c</b>)</h3><div class="NLM_p last">Compound <b>19c</b> was prepared in a similar manner as described for compound <b>19a</b>. It is obtained in 45% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 233.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43">4.22.  3-(5-Bromo-3-methylbenzofuran-2-yl)-3-oxopropanenitrile (<b>19d</b>)</h3><div class="NLM_p last">Compound <b>19d</b> was prepared in a similar manner as described for compound <b>19a</b>. It is obtained in 38% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 277.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44">4.23.  (<i>Z</i>)-Ethyl 2-Chloro-2-(2-phenylhydrazono)acetate (<b>21a</b>)</h3><div class="NLM_p last">To a solution of aniline (5.0 g, 52.08 mmol) in 3 M HCl (40 mL) at 0 °C was added a solution of sodium nitrite (3.95 g, 57.3 mmol) in water (10 mL) dropwise. The mixture was stirred at 0 °C for 1 h, and a solution of NaOAc (4.7 g, 57.3 mmol) and ethyl 2-chloroacetoacetate (9.43 g, 57.3 mmol) in EtOH (30 mL) was added and the mixture was stirred at rt for 4 h. The reaction was quenched with water (50 mL), then extracted with EA (3 × 40 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography using silica with 10% EtOAc in hexane as the eluent to give <b>21a</b> (9 g, 76% yield), as a yellow solid.<sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (s, 1H), 7.37–7.31 (m, 2H), 7.25–7.21 (m, 2H), 7.08–7.01 (m, 1H), 4.39 (q, <i>J</i> = 7.1 Hz, 2H), 1.41 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45">4.24.  (<i>Z</i>)-Ethyl 2-Chloro-2-(2-(4-nitrophenyl)hydrazono)acetate (<b>21b</b>)</h3><div class="NLM_p last">Compound <b>21b</b> was prepared in a similar manner as described for compound <b>21a</b>. It is obtained in 82% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.62 (s, 1H), 8.30–8.17 (m, 2H), 7.35–7.28 (m, 2H), 4.41 (q, <i>J</i> = 7.1 Hz, 2H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46">4.25.  (<i>Z</i>)-Ethyl 2-(2-(4-((((Benzyloxy)carbonyl)amino)methyl)phenyl)hydrazono)-2-chloroacetate (<b>21c</b>)</h3><div class="NLM_p last">Compound <b>21c</b> was prepared in a similar manner as described for compound <b>21a</b>. It is obtained in 60% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 413.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47">4.26.  Ethyl 4-Cyano-5-(3-methylbenzofuran-2-yl)-1-phenyl-1<i>H</i>-pyrazole-3-carboxylate (<b>22a</b>)</h3><div class="NLM_p last">To a solution of <b>19a</b> (1.0 g, 4.6 mmol) in DCM (10 mL) was added NEt<sub>3</sub> (1.86 g, 18.4 mmol) and <b>21a</b> (1.25 g, 5.5 mmol). The mixture was stirred at rt for 5 h. The reaction mixture was diluted with water (30 mL). The reaction mixture was then extracted with EA (3 × 30 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography using silica with 15% EA in hexane as the eluent to give <b>22a</b> (1.0 g, 56% yield), as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.51 (dd, <i>J</i> = 8.5, 5.3 Hz, 1H), 7.39 (s, 5H), 7.11–7.03 (m, 2H), 4.54 (q, <i>J</i> = 7.1 Hz, 2H), 2.26 (s, 3H), 1.48 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48">4.27.  Ethyl 4-Cyano-5-(3,5-dimethylbenzofuran-2-yl)-1-phenyl-1<i>H</i>-pyrazole-3-carboxylate (<b>22b</b>)</h3><div class="NLM_p last">Compound <b>22b</b> was prepared in a similar manner as described for compound <b>22a</b>. It is obtained in 43% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.41–7.35 (m, 5H), 7.24–7.15 (m, 3H), 4.54 (q, <i>J</i> = 7.1 Hz, 2H), 2.46 (s, 3H), 2.23 (s, 3H), 1.48 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49">4.28.  Ethyl 4-Cyano-5-(3,5-dimethylbenzofuran-2-yl)-1-(4-nitrophenyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>22c</b>)</h3><div class="NLM_p last">Compound <b>22c</b> was prepared in a similar manner as described for compound <b>22a</b>. It is obtained in 41% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.39–8.30 (m, 2H), 7.79–7.69 (m, 2H), 7.57–7.52 (m, 1H), 7.38 (d, <i>J</i> = 8.5 Hz, 1H), 7.25 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 4.45 (q, <i>J</i> = 7.1 Hz, 2H), 2.42 (s, 3H), 2.21 (s, 3H), 1.37 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50">4.29.  Ethyl 5-(5-Chloro-3-methylbenzofuran-2-yl)-4-cyano-1-(4-nitrophenyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>22d</b>)</h3><div class="NLM_p last">Compound <b>22d</b> was prepared in a similar manner as described for compound <b>22a</b>. It is obtained in 46% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 450.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51">4.30.  Ethyl 5-(5-Bromo-3-methylbenzofuran-2-yl)-4-cyano-1-(4-nitrophenyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>22e</b>)</h3><div class="NLM_p last">Compound <b>22e</b> was prepared in a similar manner as described for compound <b>22a</b>. It is obtained in 36% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.52–7.47 (m, 2H), 7.40–7.37 (m, 1H), 7.30–7.26 (m, 2H), 7.23 (s, 1H), 7.22–7.18 (m, 1H), 4.54 (q, <i>J</i> = 7.1 Hz, 2H), 2.47 (s, 3H), 2.29 (s, 3H), 1.48 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52">4.31.  Ethyl 1-(4-((((Benzyloxy)carbonyl)amino)methyl)phenyl)-4-cyano-5-(3,5-dimethylbenzofuran-2-yl)-1<i>H</i>-pyrazole-3-carboxylate (<b>22f</b>)</h3><div class="NLM_p last">Compound <b>22f</b> was prepared in a similar manner as described for compound <b>22a</b>. It is obtained in 55% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.41–7.27 (m, 11H), 7.24–7.16 (m, 2H), 5.12 (s, 2H), 4.54 (q, <i>J</i> = 7.1 Hz, 2H), 4.40 (d, <i>J</i> = 6.0 Hz, 2H), 2.47 (s, 3H), 2.24 (s, 3H), 1.48 (t, 3H).</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53">4.32.  4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-2-(4-nitrophenyl)-2,6-dihydro-7<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-7-one (<b>23a</b>)</h3><div class="NLM_p last">Compound <b>23a</b> was prepared in a similar manner as described for compound <b>8</b>. It is obtained in 32% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.55 (s, 1H), 8.35–8.28 (m, 2H), 7.76–7.69 (m, 2H), 7.52–7.44 (m, 2H), 7.25 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 5.27 (s, 2H), 2.42 (s, 3H), 1.94 (s, 3H).</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54">4.33.  4-Amino-3-(5-chloro-3-methylbenzofuran-2-yl)-2-(4-nitrophenyl)-2,6-dihydro-7<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-7-one (<b>23b</b>)</h3><div class="NLM_p last">Compound <b>23b</b> was prepared in a similar manner as described for compound <b>8</b>. It is obtained in 44% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 458.8 [M + Na]<sup>+</sup>.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55">4.34.  4-Amino-3-(5-bromo-3-methylbenzofuran-2-yl)-2-(4-nitrophenyl)-2,6-dihydro-7<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-7-one (<b>23c</b>)</h3><div class="NLM_p last">Compound <b>23c</b> was prepared in a similar manner as described for compound <b>8</b>. Yield: It is obtained in 26% yield. LRMS (ESI, <i>m</i>/<i>z</i>): 480.8, 482.7 [M, M + 2]<sup>+</sup>.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56">4.35.  Benzyl (4-(4-Amino-3-(3,5-dimethylbenzofuran-2-yl)-7-oxo-6,7-dihydro-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-2-yl)benzyl)carbamate (<b>23d</b>)</h3><div class="NLM_p last">Compound <b>23d</b> was prepared in a similar manner as described for compound <b>8</b>. It is obtained in 28% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.49 (s, 1H), 7.87 (t, <i>J</i> = 6.1 Hz, 1H), 7.49 (d, <i>J</i> = 8.4 Hz, 1H), 7.45–7.37 (m, 3H), 7.35–7.28 (m, 6H), 7.23 (d, <i>J</i> = 8.5 Hz, 1H), 5.17 (s, 2H), 5.03 (s, 2H), 4.23 (d, <i>J</i> = 6.1 Hz, 2H), 2.41 (s, 3H), 1.88 (s, 3H).</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57">4.36.  Kinase Inhibition Assay</h3><div class="NLM_p last">FGFR1, FGFR2, FGFR3, FGFR4, VEGFR2, VEGFR1, PDGFRβ, Ret, c-Src, and c-Met active proteins were purchased from Eurofins. The kinase activities were assessed using ELISA assay. Briefly, 96-well plates were precoated with 20 μg/mL poly (Glu, Tyr)<sub>4:1</sub> (Sigma, St Louis, MO, USA) as a substrate. A 50 μL aliquot of 10 μmol/L ATP solution diluted in kinase reaction buffer [50 mmol/L <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES) (pH 7.4), 50 mmol/L MgCl<sub>2</sub>, 0.5 mmol/L MnCl<sub>2</sub>, 0.2 mmol/L Na<sub>3</sub>VO<sub>4</sub>, and 1 mmol/L dithiothreitol (DTT)] was added to each well; 1 μL of the indicated compound diluted in 1% dimethyl sulfoxide (DMSO) (v/v) (Sigma, St. Louis, MO, USA) was then added to each reaction well. DMSO (1%, v/v) was used as the negative control. The kinase reaction was initiated by the addition of purified tyrosine kinase proteins diluted in 49 μL of kinase reaction buffer. After incubation for 60 min at 37 °C, the plate was washed three times with phosphate-buffered saline (PBS) containing 0.1% Tween 20 (T-PBS). Antiphosphotyrosine (PY99) antibody (100 μL; 1:500, diluted in 5 mg/mL bovine serum albumin (BSA) T-PBS) was then added. After a 30 min incubation at 37 °C, the plate was washed three times, and 100 μL of horseradish peroxidase-conjugated goat anti-mouse IgG (1:1000, diluted in 5 mg/mL BSA T-PBS) was added. The plate was then incubated at 37 °C for 30 min and washed three times. A 100 μL aliquot of a solution containing 0.03% H<sub>2</sub>O<sub>2</sub> and 2 mg/mL <i>o</i>-phenylenediamine in 0.1 mol/L citrate buffer (pH 5.5) was then added. The reaction was terminated by the addition of 50 μL of 2 mol/L H<sub>2</sub>SO<sub>4</sub>, causing a color, and the plate was then analyzed using a multiwell spectrophotometer (SpectraMAX190, from Molecular Devices, Palo Alto, CA, USA) at 490 nm. The inhibition rate (%) was calculated using the following equation: [1 – (<i>A</i><sub>490</sub>/<i>A</i><sub>490 control</sub>)] × 100%. The IC<sub>50</sub> values were calculated from the inhibition curves in two separate experiments.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58">4.37.  Kinase Kinetic Inhibition Assessment</h3><div class="NLM_p last">Kinetics of inhibition assays for irreversible inhibition as well as <i>K</i><sub>i</sub>, <i>K</i><sub>inact</sub>, and <i>K</i><sub>inact</sub>/<i>K</i><sub>i</sub> calculation was conducted in Shanghai ChemPartner Co., Ltd (China).</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59">4.38.  Cell Culture</h3><div class="NLM_p last">Unless otherwise mentioned, the cells were purchased from American Type Culture Collection (ATCC). RT112 was obtained from Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSMZ). UMUC14 was obtained from the European Collection of Cell Cultures (ECACC). SUM52PE was obtained from Asterand Company. All the cell lines used in this study were obtained between 2000 and 2017 and cultured according to the suppliers’ instructions. Cells were checked to be confirmed to be mycoplasma free, and the cells were passaged no more than 25–30 times after thawing. Cell lines were characterized by the Genesky Biopharma Technology using short tandem repeat markers (latest tested in 2017).</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60">4.39.  Western Blot Analysis</h3><div class="NLM_p last">Cells were treated with the indicated dose of the test compounds for 2 h at 37 °C and then lysed in 1× sodium dodecyl sulfate (SDS) sample buffer. The cell lysates were subsequently resolved by 10% SDS-polyacrylamide gel electrophoresis (PAGE) and transferred to nitrocellulose membranes. The membranes were probed with the appropriate primary antibodies and then with horseradish peroxidase-conjugated anti-rabbit or anti-mouse IgG. The immune reactive proteins were detected using an enhanced chemiluminescence detection reagent (Thermo Fisher Scientific, Rockford, IL, USA).</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61">4.40.  Cell Proliferation Assays</h3><div class="NLM_p last">Cells were seeded in 96-well plates at a low density in growth medium. The next day, the appropriate controls or designated concentrations of test compounds were added to each well, and the cells were incubated for 72 h. Finally, cell proliferation was determined using a sulforhodamine B assay or a cell counting kit (CCK-8) assay. The IC<sub>50</sub> values were calculated by fitting concentration–response curves fitting using a SoftMax pro-based four-parameter method.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62">4.41.  Protein Purification and crystallization</h3><div class="NLM_p">The kinase domain (residues 458–756, with mutagenesis of C488A and C584S) of recombinant human FGFR1 was produced following the protocol we have published previously.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Crystallization of the FGFR1 kinase domain was carried out by mixing a solution of the protein with an equal volume of precipitation solution (0.1 M bis–tris pH 6.5, 0.3 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 5% glycerol, 15–20% PEG10K). Crystallization utilized the vapor-diffusion method in hanging drops at 4 °C. Crystals appeared in one week. The crystals of the protein–ligand complex were obtained by soaking the apo crystals in a buffer (0.1 M bis–tris, pH 6.5, 0.3 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 5% glycerol, 25% PEG10K) containing 1 mM inhibitor compound <b>9</b>. Crystals of the complex were then flash-frozen in liquid nitrogen in the presence of soaking buffer.</div><div class="NLM_p last">The human FGFR4 kinase domain (residues 445–753, R664E) was expressed and purified as follows. The cDNA fragment was cloned into the pET28a vector at NcoI/XhoI restriction sites. Then, the construct was coexpressed with catYopH subcloned in pET15b (164–468 aa).<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> The expressed protein was passed through a Ni-NTA column (Qiagen). The kinase domain was further purified on a Q HP ion exchange column (GE) which was eluted with 20 mM Tris at pH 7.5 with 150 mM NaCl, 10% glycerol, 1 mM DTT. The eluted protein was buffer exchanged into 10 mM HEPES at pH 8.0 with 100 mM NaCl, 3% glycerol, 1 mM TCEP using a HiLoad 16/60 Superdex 75 column (GE Health). The protein was concentrated to ∼10–20 mg/mL for further crystallization. Cocrystal of the FGFR4 kinase domain with different compounds were obtained by vapor diffusion in a reservoir solution of 0.1 M MES at pH 5.5 with 0.2 M Li<sub>2</sub>SO<sub>4</sub>, 18% PEG3350 for compound <b>10</b> and 0.1 M HEPES at pH 7.5 with 1.3 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> for compound <b>11</b>, respectively.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63">4.42.  Structure Determination and Refinement</h3><div class="NLM_p last">Data were collected at 100 K at the Shanghai Synchrotron Radiation Facility (SSRF) and were processed with the XDS<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> and HKL3000<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> software packages. The structures were solved by molecular replacement using the program PHASER<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> with the search model of PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Z0S">5Z0S</a><a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> for FGFR1 and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQT">4QQT</a><a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> for FGFR4. The structures were refined with the simulated-annealing protocol implemented in the program PHENIX.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> With the aid of the program Coot,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> the compound, as well as water molecules, were fitted into an initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map. The refined structures were deposited in Protein Data Bank with accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ">6ITJ</a> for compound <b>9</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP">6IUP</a> for compound <b>10</b>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO">6IUO</a> for compound <b>11</b>. The complete statistics, as well as the qualities of the solved structures, are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">Table S2</a>.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64">4.43.  HPLC–MS/MS Analysis</h3><div class="NLM_p last">The human FGFR4 kinase domain was produced by following the protocol described above and was included in the buffer of 10 mM HEPES at pH 8.0 with 100 mM NaCl and 3% glycerol. The FGFR4 kinase domain protein was incubated with 1 mg/mL compound <b>10</b> at 4 °C for 24 h, and then preseparated by SDS-PAGE and cut off from the PAGE gel. Next, the protein was incubated with trypsin [trypsin/protein = 1:50 (w/w)] at 37 °C for 20 h. The tryptic peptides were desalted and dried in a Speed-Vac. The peak lists from HPLC–MS/MS data were generated by Proteome Discoverer software (version 1.4, Thermo Fisher) and searched against the UniProt Human database by Mascot (v2.3, Matrix Science Ltd., London, UK). Two kinds of MS, higher energy collision dissociation MS and collision-induced dissociation MS, were used to identify the modification of the human FGFR4 kinase domain.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65">4.44.  Compound Solubility Studying</h3><div class="NLM_p last">The solubility of each of the test compounds in PBS was measured by nephelometry using a NEPHELOstar plus apparatus (BMG Lab Technologies). This detection gave the amount of the particles in suspension by measuring the light that is diffused by the particles.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66">4.45.  Liver Microsomes Stability Evaluation</h3><div class="NLM_p last">Microsomal incubations were carried out in 96-well plates. Briefly, reaction mixtures were created that contained mouse liver microsomes (0.33 mg/mL final protein concentration), 5 mM MgCl<sub>2</sub>, 0.1 μM test compound, 1 mM reduced nicotinamide adenine dinucleotide phosphate (NADPH), and 0.1 M Tris-HCl buffer, pH 7.4. The test compounds were dissolved in DMSO and diluted with 0.01% BSA, and the final DMSO concentration used in the assay was 0.01% (v/v). Reactions commenced with the addition of NADPH after prewarming at 37 °C for 10 min. Following incubation, an aliquot of 50 μL of the incubation sample was removed at 0, 7, 17, 30, and 60 min and quenched with the same volume of methanol. The mixture was vortexed and centrifuged. Then, the supernatant was subjected to LC–MS/MS analysis. All samples were prepared and analyzed in duplicates. Finally, the first-order rate of consumption of the compound (−<i>k</i>) and the in vitro <i>T</i><sub>1/2</sub> values (<i>T</i><sub>1/2</sub> = −0.693/<i>k</i>) were measured.<a onclick="showRef(event, 'ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref70 ref71">(70,71)</a></div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67">4.46.  GSH Binding Affinity Assay</h3><div class="NLM_p last">The test compound (500 nM) was incubated with 2 mM GSH in methanol/PBS (15/85, v/v) solution at 37 °C. Aliquots were sampled at 0, 0.5, 1.0, 1.5, 2.1, and 19 h of incubation, and the reaction was stopped by the addition of 20 mM <i>N</i>-ethylmaleimide solution. The samples were determined by LC–MS/MS.</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68">4.47.  Blood Stability Evaluation</h3><div class="NLM_p last">The test compound (5 μM) was incubated with fresh rat blood at 37 °C. Aliquots were sampled at 0, 0.5, 1.0, 2.0, and 5.0 h of incubation, and the reaction was stopped by protein precipitation in methanol. Then, the percentage of the residual compound was determined by LC–MS/MS.</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69">4.48.  In Vivo PK</h3><div class="NLM_p last">The pharmacokinetic profiles were determined in SD rats, and the experiments were approved by the Bioethics Committee of the Shanghai Institute of Materia Medica, Chinese Academy of Sciences (IACUC: 2018-03-TW-05). Compound <b>15</b> (5% DMSO + 5% Tween-80 in 90% saline) was subjected to PK studies in SD rats. The test compound was administered via IV at 10 mg/kg, PO at 20 mg/kg and IP at 20 mg/kg. After administration, blood samples were collected. The blood samples were centrifuged to obtain the plasma fraction. The plasma samples were deproteinized with methanol containing an internal standard. After centrifugation, the supernatant was diluted with methanol and centrifuged again. The compound concentrations of the target compound in the supernatant were measured by LC–MS/MS.</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70">4.49.  Species Difference Study</h3><div class="NLM_p last">Liver microsomal incubations were conducted in a total volume of 200 μL (liver microsomal incubation of human was 100 μL) containing human, mouse, rat, dog, and monkey liver microsomes (1.0 × 10<sup>6</sup> cells/mL), William’s medium E formulation (pH 7.4), and substrate (3.0 μM). The corresponding mixture was preincubated in a water bath at 37 °C. After 180 min, the reaction was terminated by adding an equal volume of ice-cold acetonitrile and stored at −70 °C until later analysis. All experiments were conducted in duplicate. For qualitative analysis, duplicate samples of each species were pooled. The combined samples were vortex-mixed and centrifuged at 14 000 rpm for 5 min. The supernatants were transferred into a glass tube, concentrated to dryness under a stream of nitrogen at 40 °C, and then reconstituted in 120 μL of acetonitrile with water (10:90, v/v). After centrifugation at 14 000 rpm for 5 min, a 7 μL aliquot of each reconstituted solutions was analyzed by UPLC-UV/Q-TOF MS. Then, data were collected and processed by the software Analyst TF V1.6 (AB SCIEX, Framingham, MA, USA) and Masslynx V4.1 (Waters Corp., Milford, MA, USA) and analyzed with PeakView V1.2 and MetabolitePilot V1.5 from AB SCIEX.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00510" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32216" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32216" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00510" class="ext-link">10.1021/acs.jmedchem.9b00510</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Structure, scores, and FGFR1 inhibitory activities of the 112 compounds selected from the VS; inhibitory activities were determined by ELISA; X-ray crystal structure analysis: data collection and refinement statistics; sequence coverage of the FGFR4 kinase domain by protein HPLC–MS/MS analysis; residue Cys477 in FGFR4 covalently modified by compound <b>10</b>; inhibition ratios of kinases treated with 0.01, 0.1, and 1 μM compound <b>15</b>; UPLC-UV/Q-TOF MS analysis results of the metabolites of compound <b>15</b> in five different species liver cells from five different species, namely, human, monkey, dog, rat, and mouse; putative metabolic pathway of compound <b>15</b> in liver cells; and HPLC analysis data of all the tested compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf">jm9b00510_si_001.pdf (1.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_002.csv">jm9b00510_si_002.csv (10.52 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ">6ITJ</a> (FGFR1/<b>9</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP">6IUP</a> (FGFR4/<b>10</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO">6IUO</a> (FGFR4/<b>11</b>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00510" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77626" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77626" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Liu</span> - <span class="hlFld-Affiliation affiliation">Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0863-7853" title="Orcid link">http://orcid.org/0000-0002-0863-7853</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fe9692978bbe8d979393d09f9dd09d90"><span class="__cf_email__" data-cfemail="bad2d6d3cffac9d3d7d794dbd994d9d4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Ai</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#59333830192a30343477383a773a37"><span class="__cf_email__" data-cfemail="4d272c240d3e242020632c2e632e23">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingyue Zheng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3323-3092" title="Orcid link">http://orcid.org/0000-0002-3323-3092</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#04697d7e6c616a6344776d69692a65672a676a"><span class="__cf_email__" data-cfemail="69041013010c070e291a00040447080a470a07">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yulan Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yang Dai</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaowei Wu</span> - <span class="hlFld-Affiliation affiliation">Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Chemistry, Shanghai University, 99 ShangDa Road, Shanghai 200444, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Liu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunpu Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiufeng Liu</span> - <span class="hlFld-Affiliation affiliation">Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanyang Zhou</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bao Wang</span> - <span class="hlFld-Affiliation affiliation">Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Middle Huaxia Road, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingrui Zhu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rongrong Cui</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoqin Tan</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaoping Xiong</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Middle Huaxia Road, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jia Liu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Minjia Tan</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6784-9653" title="Orcid link">http://orcid.org/0000-0002-6784-9653</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yechun Xu</span> - <span class="hlFld-Affiliation affiliation">Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1581-6155" title="Orcid link">http://orcid.org/0000-0002-1581-6155</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meiyu Geng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia
Medica, 
    , and , Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Middle Huaxia Road, Shanghai 200031, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.W., Y.D. and X.W. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14471" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We gratefully acknowledge financial support from the National Natural Science Foundation of China (81773634 to M.Z., 81773762 to J.A., 81620108027 and 21632008 to H.L.), National Science & Technology Major Project “Key New Drug Creation and Manufacturing Program”, China (2018ZX09711002), the Major Project of Chinese National Programs for Fundamental Research and Development (2015CB910304 to H.L.), “Personalized Medicines—Molecular Signature-based Drug Discovery and Development”, Strategic Priority Research Pro-gram of the Chinese Academy of Sciences (XDA12050201 to M.Z. and XDA12050401 To H.J.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">FGFR</td><td class="NLM_def"><p class="first last">fibroblast growth factor receptor</p></td></tr><tr><td class="NLM_term">RTKs</td><td class="NLM_def"><p class="first last">receptor tyrosine kinases</p></td></tr><tr><td class="NLM_term">TKIs</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitors</p></td></tr><tr><td class="NLM_term">VEGFRs</td><td class="NLM_def"><p class="first last">vascular endothelial growth receptors</p></td></tr><tr><td class="NLM_term">PDGFRs</td><td class="NLM_def"><p class="first last">platelet-derived growth factor receptors</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">CCA</td><td class="NLM_def"><p class="first last">cholangiocarcinoma</p></td></tr><tr><td class="NLM_term">CADD</td><td class="NLM_def"><p class="first last">computer-aided drug design</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screening</p></td></tr><tr><td class="NLM_term"><i>R</i><sup>2</sup></td><td class="NLM_def"><p class="first last">the square of Pearson correlation coefficient</p></td></tr><tr><td class="NLM_term">SOM</td><td class="NLM_def"><p class="first last">site-of-metabolism</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86055" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86055" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 71 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiong, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, C.-O.</span></span> <span> </span><span class="NLM_article-title">Functional roles of fibroblast growth factor receptors (FGFRs) signaling in human cancers</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1447</span>– <span class="NLM_lpage">1468</span>, <span class="refDoi"> DOI: 10.1007/s10495-013-0886-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1007%2Fs10495-013-0886-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23900974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslOkur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1447-1468&author=K.+H.+Tiongauthor=L.+Y.+Mahauthor=C.-O.+Leong&title=Functional+roles+of+fibroblast+growth+factor+receptors+%28FGFRs%29+signaling+in+human+cancers&doi=10.1007%2Fs10495-013-0886-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Functional roles of fibroblast growth factor receptors (FGFRs) signaling in human cancers</span></div><div class="casAuthors">Tiong, Kai Hung; Mah, Li Yen; Leong, Chee-Onn</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1447-1468</span>CODEN:
                <span class="NLM_cas:coden">APOPFN</span>;
        ISSN:<span class="NLM_cas:issn">1360-8185</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The fibroblast growth factor receptors (FGFRs) regulate important biol. processes including cell proliferation and differentiation during development and tissue repair.  Over the past decades, numerous pathol. conditions and developmental syndromes have emerged as a consequence of deregulation in the FGFRs signaling network.  This review aims to provide an overview of FGFR family, their complex signaling pathways in tumorigenesis, and the current development and application of therapeutics targeting the FGFRs signaling for treatment of refractory human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4FLwi8M_8RLVg90H21EOLACvtfcHk0li1-zRCASpH6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslOkur7J&md5=3e1180c8cdfc5903fa11f0a9dfeab904</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs10495-013-0886-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-013-0886-7%26sid%3Dliteratum%253Aachs%26aulast%3DTiong%26aufirst%3DK.%2BH.%26aulast%3DMah%26aufirst%3DL.%2BY.%26aulast%3DLeong%26aufirst%3DC.-O.%26atitle%3DFunctional%2520roles%2520of%2520fibroblast%2520growth%2520factor%2520receptors%2520%2528FGFRs%2529%2520signaling%2520in%2520human%2520cancers%26jtitle%3DApoptosis%26date%3D2013%26volume%3D18%26spage%3D1447%26epage%3D1468%26doi%3D10.1007%2Fs10495-013-0886-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thisse, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thisse, C.</span></span> <span> </span><span class="NLM_article-title">Functions and regulations of fibroblast growth factor signaling during embryonic development</span>. <i>Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">390</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1016/j.ydbio.2005.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.ydbio.2005.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=16216232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1GhtL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2005&pages=390-402&author=B.+Thisseauthor=C.+Thisse&title=Functions+and+regulations+of+fibroblast+growth+factor+signaling+during+embryonic+development&doi=10.1016%2Fj.ydbio.2005.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Functions and regulations of fibroblast growth factor signaling during embryonic development</span></div><div class="casAuthors">Thisse, Bernard; Thisse, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Biology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">390-402</span>CODEN:
                <span class="NLM_cas:coden">DEBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0012-1606</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fibroblast growth factors (FGF) are secreted mols. which function through the activation of specific tyrosine kinases receptors, the FGF receptors that transduce the signal by activating different pathways including the Ras/MAP kinase and the phospholipase-Cγ pathways.  FGFs are involved in the regulation of many developmental processes including patterning, morphogenesis, differentiation, cell proliferation, or migration.  Such a diverse set of activities requires a tight control of the transduction signal which is achieved through the induction of different feedback inhibitors such as the Sproutys, Sef, and MAP kinase phosphatase 3 which are responsible for the attenuation of FGF signals, limiting FGF activities in time and space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqQ6hbl6aVdbVg90H21EOLACvtfcHk0ljv1vbxaGSCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1GhtL%252FF&md5=7a2a1199001aff1a05836d6ff19f20ee</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ydbio.2005.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ydbio.2005.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DThisse%26aufirst%3DB.%26aulast%3DThisse%26aufirst%3DC.%26atitle%3DFunctions%2520and%2520regulations%2520of%2520fibroblast%2520growth%2520factor%2520signaling%2520during%2520embryonic%2520development%26jtitle%3DDev.%2520Biol.%26date%3D2005%26volume%3D287%26spage%3D390%26epage%3D402%26doi%3D10.1016%2Fj.ydbio.2005.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dienstmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Garcia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iafrate, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuciforo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span> <span> </span><span class="NLM_article-title">Genomic aberrations in the FGFR pathway: opportunities for targeted therapies in solid tumors</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdt419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdt419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24265351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BC2c7ptFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=552-563&author=R.+Dienstmannauthor=J.+Rodonauthor=A.+Pratauthor=J.+Perez-Garciaauthor=B.+Adamoauthor=E.+Felipauthor=J.+Cortesauthor=A.+J.+Iafrateauthor=P.+Nuciforoauthor=J.+Tabernero&title=Genomic+aberrations+in+the+FGFR+pathway%3A+opportunities+for+targeted+therapies+in+solid+tumors&doi=10.1093%2Fannonc%2Fmdt419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic aberrations in the FGFR pathway: opportunities for targeted therapies in solid tumors</span></div><div class="casAuthors">Dienstmann R; Rodon J; Prat A; Perez-Garcia J; Adamo B; Felip E; Cortes J; Iafrate A J; Nuciforo P; Tabernero J</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">552-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The fibroblast growth factor receptor (FGFR) cascade plays crucial roles in tumor cell proliferation, angiogenesis, migration and survival.  Accumulating evidence suggests that in some tumor types, FGFRs are bona fide oncogenes to which cancer cells are addicted.  Because FGFR inhibition can reduce proliferation and induce cell death in a variety of in vitro and in vivo tumor models harboring FGFR aberrations, a growing number of research groups have selected FGFRs as targets for anticancer drug development.  Multikinase FGFR/vascular endothelial growth factor receptor (VEGFR) inhibitors have shown promising activity in breast cancer patients with FGFR1 and/or FGF3 amplification.  Early clinical trials with selective FGFR inhibitors, which may overcome the toxicity constraints raised by multitarget kinase inhibition, are recruiting patients with known FGFR(1-4) status based on genomic screens.  Preliminary signs of antitumor activity have been demonstrated in some tumor types, including squamous cell lung carcinomas.  Rational combination of targeted therapies is expected to further increase the efficacy of selective FGFR inhibitors.  Herein, we discuss unsolved questions in the clinical development of these agents and suggest guidelines for management of hyperphosphatemia, a class-specific mechanism-based toxicity.  In addition, we propose standardized definitions for FGFR1 and FGFR2 gene amplification based on in situ hybridization methods.  Extended access to next-generation sequencing platforms will facilitate the identification of diseases in which somatic FGFR(1-4) mutations, amplifications and fusions are potentially driving cancer cell viability, further strengthening the role of FGFR signaling in cancer biology and providing more possibilities for the therapeutic application of FGFR inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqiWGpXuCcvmx8TEN9K18QfW6udTcc2ebsr8zjS0f9jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7ptFWmsA%253D%253D&md5=633eb08ba5e38f239a73dfbfff04064c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdt419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdt419%26sid%3Dliteratum%253Aachs%26aulast%3DDienstmann%26aufirst%3DR.%26aulast%3DRodon%26aufirst%3DJ.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DPerez-Garcia%26aufirst%3DJ.%26aulast%3DAdamo%26aufirst%3DB.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DNuciforo%26aufirst%3DP.%26aulast%3DTabernero%26aufirst%3DJ.%26atitle%3DGenomic%2520aberrations%2520in%2520the%2520FGFR%2520pathway%253A%2520opportunities%2520for%2520targeted%2520therapies%2520in%2520solid%2520tumors%26jtitle%3DAnn.%2520Oncol.%26date%3D2014%26volume%3D25%26spage%3D552%26epage%3D563%26doi%3D10.1093%2Fannonc%2Fmdt419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araújo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reclusa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franchina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Steen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Dam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naing, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfo, C.</span></span> <span> </span><span class="NLM_article-title">FGFR a promising druggable target in cancer: molecular biology and new drugs</span>. <i>Crit. Rev. Oncol. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2017.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.critrevonc.2017.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28427515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BC1cvps1ajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2017&pages=256-267&author=R.+Portaauthor=R.+Boreaauthor=A.+Coelhoauthor=S.+Khanauthor=A.+Ara%C3%BAjoauthor=P.+Reclusaauthor=T.+Franchinaauthor=N.+Van+Der+Steenauthor=P.+Van+Damauthor=J.+Ferriauthor=R.+Sireraauthor=A.+Naingauthor=D.+Hongauthor=C.+Rolfo&title=FGFR+a+promising+druggable+target+in+cancer%3A+molecular+biology+and+new+drugs&doi=10.1016%2Fj.critrevonc.2017.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">FGFR a promising druggable target in cancer: Molecular biology and new drugs</span></div><div class="casAuthors">Porta Rut; Borea Roberto; Coelho Andreia; Reclusa Pablo; Van Dam Peter; Ferri Jose; Sirera Rafael; Khan Shahanavaj; Araujo Antonio; Franchina Tindara; Van Der Steen Nele; Naing Aung; Hong David; Rolfo Christian</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">256-267</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  The Fibroblast Growth Factor Receptor (FGFR) family consists of Tyrosine Kinase Receptors (TKR) involved in several biological functions.  Recently, alterations of FGFR have been reported to be important for progression and development of several cancers.  In this setting, different studies are trying to evaluate the efficacy of different therapies targeting FGFR.  AREAS COVERED:  This review summarizes the current status of treatments targeting FGFR, focusing on the trials that are evaluating the FGFR profile as inclusion criteria: Multi-Target, Pan-FGFR Inhibitors and anti-FGF (Fibroblast Growth Factor)/FGFR Monoclonal Antibodies.  EXPERT OPINION:  Most of the TKR share intracellular signaling pathways; therefore, cancer cells tend to overcome the inhibition of one tyrosine kinase receptor by activating another.  The future of TKI (Tyrosine Kinase Inhibitor) therapy will potentially come from multi-targeted TKIs that target different TKR simultaneously.  It is crucial to understand the interaction of the FGF-FGFR axis with other known driver TKRs.  Based on this, it is possible to develop therapeutic strategies targeting multiple connected TKRs at once.  One correct step in this direction is the reassessment of multi target inhibitors considering the FGFR status of the tumor.  Another opportunity arises from assessing the use of FGFR TKI on patients harboring FGFR alterations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFLmRJD6qzpJcXNb62YGmbfW6udTcc2ebsr8zjS0f9jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvps1ajsA%253D%253D&md5=d03086e258264a12c8c5cc7f2c5de8e3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2017.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2017.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DPorta%26aufirst%3DR.%26aulast%3DBorea%26aufirst%3DR.%26aulast%3DCoelho%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DAra%25C3%25BAjo%26aufirst%3DA.%26aulast%3DReclusa%26aufirst%3DP.%26aulast%3DFranchina%26aufirst%3DT.%26aulast%3DVan%2BDer%2BSteen%26aufirst%3DN.%26aulast%3DVan%2BDam%26aufirst%3DP.%26aulast%3DFerri%26aufirst%3DJ.%26aulast%3DSirera%26aufirst%3DR.%26aulast%3DNaing%26aufirst%3DA.%26aulast%3DHong%26aufirst%3DD.%26aulast%3DRolfo%26aufirst%3DC.%26atitle%3DFGFR%2520a%2520promising%2520druggable%2520target%2520in%2520cancer%253A%2520molecular%2520biology%2520and%2520new%2520drugs%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2017%26volume%3D113%26spage%3D256%26epage%3D267%26doi%3D10.1016%2Fj.critrevonc.2017.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babina, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span> <span> </span><span class="NLM_article-title">Advances and challenges in targeting FGFR signalling in cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnrc.2017.8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28303906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Ont7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=318-332&author=I.+S.+Babinaauthor=N.+C.+Turner&title=Advances+and+challenges+in+targeting+FGFR+signalling+in+cancer&doi=10.1038%2Fnrc.2017.8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Advances and challenges in targeting FGFR signalling in cancer</span></div><div class="casAuthors">Babina, Irina S.; Turner, Nicholas C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">318-332</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fibroblast growth factors (FGFs) and their receptors (FGFRs) regulate numerous cellular processes.  Deregulation of FGFR signalling is obsd. in a subset of many cancers, making activated FGFRs a highly promising potential therapeutic target supported by multiple preclin. studies.  However, early-phase clin. trials have produced mixed results with FGFR-targeted cancer therapies, revealing substantial complexity to targeting aberrant FGFR signalling.  In this Review, we discuss the increasing understanding of the differences between diverse mechanisms of oncogenic activation of FGFR, and the factors that det. response and resistance to FGFR targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKkSMehfBbyLVg90H21EOLACvtfcHk0liZkWlUopSowA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Ont7g%253D&md5=ed8f0ad426859a07930f7c0c2d2a9b71</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.8%26sid%3Dliteratum%253Aachs%26aulast%3DBabina%26aufirst%3DI.%2BS.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26atitle%3DAdvances%2520and%2520challenges%2520in%2520targeting%2520FGFR%2520signalling%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D318%26epage%3D332%26doi%3D10.1038%2Fnrc.2017.8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Izzedine, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ederhy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldwasser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khayat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spano, J. P.</span></span> <span> </span><span class="NLM_article-title">Management of hypertension in angiogenesis inhibitor-treated patients</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdn713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19150949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BD1MzgvVyhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=807-815&author=H.+Izzedineauthor=S.+Ederhyauthor=F.+Goldwasserauthor=J.+C.+Soriaauthor=G.+Milanoauthor=A.+Cohenauthor=D.+Khayatauthor=J.+P.+Spano&title=Management+of+hypertension+in+angiogenesis+inhibitor-treated+patients&doi=10.1093%2Fannonc%2Fmdn713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Management of hypertension in angiogenesis inhibitor-treated patients</span></div><div class="casAuthors">Izzedine H; Ederhy S; Goldwasser F; Soria J C; Milano G; Cohen A; Khayat D; Spano J P</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">807-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Hypertension (HTN) is one of the most frequent side-effects of systemic inhibition of vascular endothelial growth factor (VEGF) signaling.  Its incidence and severity are dependent on the type of drugs, dose, and schedule used.  The recognition of this side-effect is an important issue because poorly controlled HTN could lead to serious cardiovascular events.  On another hand, HTN induced by anti-VEGF agents maybe a predictive factor of oncologic response.  Knowledge of this clinical toxicity and/or therapeutic target or novel biomarker of drug activity can aid clinicians choosing the optimal and least toxic regimen suitable for an individual patient.  METHODS:  A Medline search was carried out using the following criteria: (i) all Medline listings as of 1 January 2000 with abstracts, (ii) English language, and (iii) Humans.  The following phrases were used to query the database: ('hypertension', OR 'blood pressure') AND ('anti-VEGF' OR 'VEGF inhibition' OR 'bevacizumab' OR 'sunitinib' OR 'sorafenib' OR 'VEGF Trap').  The references of each article identified were carefully reviewed for additional reference.  RESULTS:  Lifestyle modification should be encouraged.  However, these nonpharmacologic strategies are not always suitable to patients with altered performance status related to metastatic cancer necessitating early drug intervention.  Only one randomized study showed a beneficial effect of a calcium channel blocker use to prevent or minimize HTN secondary to antiangiogenic therapy.  Nitrates looks as effective in controlling such side-effect.  CONCLUSIONS:  No clear recommendation for an antihypertensive agent can be made in this context because there is a lack of controlled studies addressing the subject.  Blood pressure-lowering drugs should be individualized to the patient's clinical circumstances and angiogenic inhibitors should be withheld only from patients who experienced hypertensive crisis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSwDYVxbcpKCw8IxFYvehhofW6udTcc2eb-O8i6hbFDKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzgvVyhsg%253D%253D&md5=6d056114baad0ac979d647806a410738</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn713%26sid%3Dliteratum%253Aachs%26aulast%3DIzzedine%26aufirst%3DH.%26aulast%3DEderhy%26aufirst%3DS.%26aulast%3DGoldwasser%26aufirst%3DF.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DMilano%26aufirst%3DG.%26aulast%3DCohen%26aufirst%3DA.%26aulast%3DKhayat%26aufirst%3DD.%26aulast%3DSpano%26aufirst%3DJ.%2BP.%26atitle%3DManagement%2520of%2520hypertension%2520in%2520angiogenesis%2520inhibitor-treated%2520patients%26jtitle%3DAnn.%2520Oncol.%26date%3D2009%26volume%3D20%26spage%3D807%26epage%3D815%26doi%3D10.1093%2Fannonc%2Fmdn713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricciardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Marinis, F.</span></span> <span> </span><span class="NLM_article-title">Toxicity of targeted therapy in non-small-cell lung cancer management</span>. <i>Clin. Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.3816/clc.2009.n.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.3816%2Fclc.2009.n.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19289369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslalu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=28-35&author=S.+Ricciardiauthor=S.+Tomaoauthor=F.+de+Marinis&title=Toxicity+of+targeted+therapy+in+non-small-cell+lung+cancer+management&doi=10.3816%2Fclc.2009.n.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicity of targeted therapy in non-small-cell lung cancer management</span></div><div class="casAuthors">Ricciardi, Serena; Tomao, Silverio; de Marinis, Filippo</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Lung Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-35</span>CODEN:
                <span class="NLM_cas:coden">CLCLCA</span>;
        ISSN:<span class="NLM_cas:issn">1525-7304</span>.
    
            (<span class="NLM_cas:orgname">CIG Media Group, LP</span>)
        </div><div class="casAbstract">A review.  Lung cancer is the leading cause of cancer-related death worldwide.  Despite several chemotherapeutic agents, a survival plateau has been reached, so new treatment strategies are clearly needed.  A strong interest is now focused on the use of targeted therapies for the management of non-small-cell lung cancer.  Monoclonal antibodies against the epidermal growth factor receptor (EGFR; cetuximab) or vascular endothelial growth factor receptor (VEGFR; bevacizumab) and EGFR tyrosine kinase inhibitors (gefitinib, erlotinib) are generally well tolerated and do not have the severe systemic side effects usually seen with cytotoxic drugs.  A considerable no. of treated patients develop dermatol. side effects, such as acneiform eruption, xerosis, and eczema, and unfortunately, this is often one cause of neg. impact on a patient's quality of life.  No controlled clin. trials have been performed to manage rash, so it is necessary to provide suggestions for managing this frequent side effect.  The main problems related to the class of angiogenesis inhibitors affecting VEGFRs are the exclusion of patients with brain metastases and/or squamous histol., and vascular adverse effects, such as hypertension, proteinuria, thrombosis, and hemorrhage.  There are other new agents in clin. development, such as sorafenib, sunitinib, vorinostat, vandetanib, everolimus, panobinostat, and ASA404.  They are all assocd. with a spectrum of toxicities, often reversible with interruption of dosing.  Further research is required to clarify the role of targeted therapies and toxicities management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMqY7hrfU1gbVg90H21EOLACvtfcHk0lhOejPovcjljA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslalu70%253D&md5=9ab0230026d6e4993fd796e5b4c33375</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3816%2Fclc.2009.n.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3816%252Fclc.2009.n.004%26sid%3Dliteratum%253Aachs%26aulast%3DRicciardi%26aufirst%3DS.%26aulast%3DTomao%26aufirst%3DS.%26aulast%3Dde%2BMarinis%26aufirst%3DF.%26atitle%3DToxicity%2520of%2520targeted%2520therapy%2520in%2520non-small-cell%2520lung%2520cancer%2520management%26jtitle%3DClin.%2520Lung%2520Cancer%26date%3D2009%26volume%3D10%26spage%3D28%26epage%3D35%26doi%3D10.3816%2Fclc.2009.n.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptors as treatment targets in clinical oncology</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0115-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fs41571-018-0115-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=30367139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=105-122&author=M.+Katoh&title=Fibroblast+growth+factor+receptors+as+treatment+targets+in+clinical+oncology&doi=10.1038%2Fs41571-018-0115-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor receptors as treatment targets in clinical oncology</span></div><div class="casAuthors">Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-122</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  FGFRs are receptor tyrosine kinases with a role in several biol. processes, such as the regulation of development and tissue repair.  However, alterations in FGFRs 1-4, such as amplifications, fusions and mutations, as well as aberrant epigenetic or transcriptional regulation and changes in tumor-stromal interactions in the tumor microenvironment, can lead to the development and/or progression of cancer.  Similar to other kinase alterations, such alterations are targetable using small mols. or antibodies, and the benefits of FGFR inhibitors have been demonstrated in clin. trials involving subsets of patients with solid tumors harbouring FGFR alterations.  However, the response rates in patients with FGFR alterations were relatively low, and responses in patients without detectable FGFR alterations were also obsd.  In this Review, the author describes the clin. experience with FGFR inhibitors to date, and highlights key aspects that might lead to improved response rates and/or the avoidance of acquired resistance, including the selection of patients who are most likely to benefit from treatment, and the use of FGFR inhibitors in combination regimens with other agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8oCjTdYY2DLVg90H21EOLACvtfcHk0lhOejPovcjljA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO&md5=41440ace3cb26bac6d7cdb0e18b46c39</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0115-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0115-y%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DFibroblast%2520growth%2520factor%2520receptors%2520as%2520treatment%2520targets%2520in%2520clinical%2520oncology%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D105%26epage%3D122%26doi%3D10.1038%2Fs41571-018-0115-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mooney, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgour, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Kadhimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span> <span> </span><span class="NLM_article-title">AZD4547: an orally bioavailable, potent, and selective inhibitor of the fibroblast growth factor receptor tyrosine kinase family</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2045</span>– <span class="NLM_lpage">2056</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-11-3034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F0008-5472.CAN-11-3034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=22369928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2045-2056&author=P.+R.+Gavineauthor=L.+Mooneyauthor=E.+Kilgourauthor=A.+P.+Thomasauthor=K.+Al-Kadhimiauthor=S.+Beckauthor=C.+Rooneyauthor=T.+Colemanauthor=D.+Bakerauthor=M.+J.+Mellorauthor=A.+N.+Brooksauthor=T.+Klinowska&title=AZD4547%3A+an+orally+bioavailable%2C+potent%2C+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+tyrosine+kinase+family&doi=10.1158%2F0008-5472.can-11-3034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">AZD4547: An Orally Bioavailable, Potent, and Selective Inhibitor of the Fibroblast Growth Factor Receptor Tyrosine Kinase Family</span></div><div class="casAuthors">Gavine, Paul R.; Mooney, Lorraine; Kilgour, Elaine; Thomas, Andrew P.; Al-Kadhimi, Katherine; Beck, Sarah; Rooney, Claire; Coleman, Tanya; Baker, Dawn; Mellor, Martine J.; Brooks, A. Nigel; Klinowska, Teresa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2045-2056</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The fibroblast growth factor (FGF) signaling axis is increasingly implicated in tumorigenesis and chemoresistance.  Several small-mol. FGF receptor (FGFR) kinase inhibitors are currently in clin. development; however, the predominant activity of the most advanced of these agents is against the kinase insert domain receptor (KDR), which compromises the FGFR selectivity.  Here, we report the pharmacol. profile of AZD4547, a novel and selective inhibitor of the FGFR1, 2, and 3 tyrosine kinases.  AZD4547 inhibited recombinant FGFR kinase activity in vitro and suppressed FGFR signaling and growth in tumor cell lines with deregulated FGFR expression.  In a representative FGFR-driven human tumor xenograft model, oral administration of AZD4547 was well tolerated and resulted in potent dose-dependent antitumor activity, consistent with plasma exposure and pharmacodynamic modulation of tumor FGFR.  Importantly, at efficacious doses, no evidence of anti-KDR-related effects were obsd., confirming the in vivo FGFR selectivity of AZD4547.  Taken together, our findings show that AZD4547 is a novel selective small-mol. inhibitor of FGFR with potent antitumor activity against FGFR-deregulated tumors in preclin. models.  AZD4547 is under clin. investigation for the treatment of FGFR-dependent tumors.  Cancer Res; 72(8); 2045-56.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpExryJc9xbVg90H21EOLACvtfcHk0lhOejPovcjljA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D&md5=cba90ad8f9b9e51cd10288868bbcc85e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-3034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-3034%26sid%3Dliteratum%253Aachs%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DMooney%26aufirst%3DL.%26aulast%3DKilgour%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DA.%2BP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DColeman%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DKlinowska%26aufirst%3DT.%26atitle%3DAZD4547%253A%2520an%2520orally%2520bioavailable%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520family%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2045%26epage%3D2056%26doi%3D10.1158%2F0008-5472.can-11-3034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Douget, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-(2,6-dichloro-3,5-dimethoxy-phenyl)-1--1-methyl-urea (NVP-BGJ398), a potent and selective inhibitor of the fibroblast growth factor receptor family of receptor tyrosine kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7066</span>– <span class="NLM_lpage">7083</span>, <span class="refDoi"> DOI: 10.1021/jm2006222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2006222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7066-7083&author=V.+Guagnanoauthor=P.+Furetauthor=C.+Spankaauthor=V.+Bordasauthor=M.+Le+Dougetauthor=C.+Stammauthor=J.+Brueggenauthor=M.+R.+Jensenauthor=C.+Schnellauthor=H.+Schmidauthor=M.+Wartmannauthor=J.+Berghausenauthor=P.+Drueckesauthor=A.+Zimmerlinauthor=D.+Bussiereauthor=J.+Murrayauthor=D.+Graus+Porta&title=Discovery+of+3-%282%2C6-dichloro-3%2C5-dimethoxy-phenyl%29-1%2D%2D1-methyl-urea+%28NVP-BGJ398%29%2C+a+potent+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+family+of+receptor+tyrosine+kinase&doi=10.1021%2Fjm2006222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-(2,6-Dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), A Potent and Selective Inhibitor of the Fibroblast Growth Factor Receptor Family of Receptor Tyrosine Kinase</span></div><div class="casAuthors">Guagnano, Vito; Furet, Pascal; Spanka, Carsten; Bordas, Vincent; Le Douget, Mickael; Stamm, Christelle; Brueggen, Josef; Jensen, Michael R.; Schnell, Christian; Schmid, Herbert; Wartmann, Markus; Berghausen, Joerg; Drueckes, Peter; Zimmerlin, Alfred; Bussiere, Dirksen; Murray, Jeremy; Graus Porta, Diana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7066-7083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of N-aryl-N'-pyrimidin-4-yl ureas has been optimized to afford potent and selective inhibitors of the fibroblast growth factor receptor tyrosine kinases 1, 2, and 3 by rationally designing the substitution pattern of the aryl ring.  On the basis of its in vitro profile, compd. 1h (NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3.  These results support the potential therapeutic use of 1h as a new anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSC1v0A_6I_bVg90H21EOLACvtfcHk0liu8G2MK4HndQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK&md5=f9a2d742294cd3814e872bdd8add1ffa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm2006222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2006222%26sid%3Dliteratum%253Aachs%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBordas%26aufirst%3DV.%26aulast%3DLe%2BDouget%26aufirst%3DM.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DH.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DDiscovery%2520of%25203-%25282%252C6-dichloro-3%252C5-dimethoxy-phenyl%2529-1--1-methyl-urea%2520%2528NVP-BGJ398%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520family%2520of%2520receptor%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7066%26epage%3D7083%26doi%3D10.1021%2Fjm2006222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe
Miyano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyajima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuramochi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funasaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugi, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uenaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruoka, A.</span></span> <span> </span><span class="NLM_article-title">E7090, a novel selective inhibitor of fibroblast growth factor receptors, displays potent antitumor activity and prolongs survival in preclinical models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2630</span>– <span class="NLM_lpage">2639</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-16-0261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.mct-16-0261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27535969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BC2szhsV2itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2630-2639&author=S.+Watanabe%0AMiyanoauthor=Y.+Yamamotoauthor=K.+Kodamaauthor=Y.+Miyajimaauthor=M.+Mikamotoauthor=T.+Nakagawaauthor=H.+Kuramochiauthor=S.+Funasakaauthor=S.+Nagaoauthor=N.+H.+Sugiauthor=K.+Okamotoauthor=Y.+Minoshimaauthor=Y.+Nakataniauthor=Y.+Karojiauthor=I.+Ohashiauthor=Y.+Yamaneauthor=T.+Okadaauthor=T.+Matsushimaauthor=J.+Matsuiauthor=M.+Iwataauthor=T.+Uenakaauthor=A.+Tsuruoka&title=E7090%2C+a+novel+selective+inhibitor+of+fibroblast+growth+factor+receptors%2C+displays+potent+antitumor+activity+and+prolongs+survival+in+preclinical+models&doi=10.1158%2F1535-7163.mct-16-0261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">E7090, a Novel Selective Inhibitor of Fibroblast Growth Factor Receptors, Displays Potent Antitumor Activity and Prolongs Survival in Preclinical Models</span></div><div class="casAuthors">Watanabe Miyano Saori; Yamamoto Yuji; Kodama Kotaro; Miyajima Yukiko; Mikamoto Masaki; Nakagawa Takayuki; Kuramochi Hiroko; Funasaka Setsuo; Nagao Satoshi; Sugi Naoko Hata; Okamoto Kiyoshi; Minoshima Yukinori; Nakatani Yusuke; Karoji Yuki; Ohashi Isao; Yamane Yoshinobu; Okada Toshimi; Matsushima Tomohiro; Matsui Junji; Iwata Masao; Uenaka Toshimitsu; Tsuruoka Akihiko; Watanabe Miyano Saori</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2630-2639</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The FGFR signaling pathway has a crucial role in proliferation, survival, and migration of cancer cells, tumor angiogenesis, and drug resistance.  FGFR genetic abnormalities, such as gene fusion, mutation, and amplification, have been implicated in several types of cancer.  Therefore, FGFRs are considered potential targets for cancer therapy.  E7090 is an orally available and selective inhibitor of the tyrosine kinase activities of FGFR1, -2, and -3.  In kinetic analyses of the interaction between E7090 and FGFR1 tyrosine kinase, E7090 associated more rapidly with FGFR1 than did the type II FGFR1 inhibitor ponatinib, and E7090 dissociated more slowly from FGFR1, with a relatively longer residence time, than did the type I FGFR1 inhibitor AZD4547, suggesting that its kinetics are more similar to the type V inhibitors, such as lenvatinib.  E7090 showed selective antiproliferative activity against cancer cell lines harboring FGFR genetic abnormalities and decreased tumor size in a mouse xenograft model using cell lines with dysregulated FGFR Furthermore, E7090 administration significantly prolonged the survival of mice with metastasized tumors in the lung.  Our results suggest that E7090 is a promising candidate as a therapeutic agent for the treatment of tumors harboring FGFR genetic abnormalities.  It is currently being investigated in a phase I clinical trial.  Mol Cancer Ther; 15(11); 2630-9. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTFDNVVHGDPezslzEEsg76ufW6udTcc2ebKo07ZoS0-W7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szhsV2itA%253D%253D&md5=e0fbe235b75cd03d13dc8f6e9fc9ecbe</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-16-0261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-16-0261%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%2BMiyano%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DY.%26aulast%3DKodama%26aufirst%3DK.%26aulast%3DMiyajima%26aufirst%3DY.%26aulast%3DMikamoto%26aufirst%3DM.%26aulast%3DNakagawa%26aufirst%3DT.%26aulast%3DKuramochi%26aufirst%3DH.%26aulast%3DFunasaka%26aufirst%3DS.%26aulast%3DNagao%26aufirst%3DS.%26aulast%3DSugi%26aufirst%3DN.%2BH.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DKaroji%26aufirst%3DY.%26aulast%3DOhashi%26aufirst%3DI.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DMatsushima%26aufirst%3DT.%26aulast%3DMatsui%26aufirst%3DJ.%26aulast%3DIwata%26aufirst%3DM.%26aulast%3DUenaka%26aufirst%3DT.%26aulast%3DTsuruoka%26aufirst%3DA.%26atitle%3DE7090%252C%2520a%2520novel%2520selective%2520inhibitor%2520of%2520fibroblast%2520growth%2520factor%2520receptors%252C%2520displays%2520potent%2520antitumor%2520activity%2520and%2520prolongs%2520survival%2520in%2520preclinical%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2630%26epage%3D2639%26doi%3D10.1158%2F1535-7163.mct-16-0261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zia-Ebrahimi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloem, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huss, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.-b.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, D. J.</span></span> <span> </span><span class="NLM_article-title">A novel, selective inhibitor of fibroblast growth factor receptors that shows a potent broad spectrum of antitumor activity in several tumor xenograft models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2200</span>– <span class="NLM_lpage">2210</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-11-0306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.MCT-11-0306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=21900693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVeltb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=2200-2210&author=G.+Zhaoauthor=W.-y.+Liauthor=D.+Chenauthor=J.+R.+Henryauthor=H.-Y.+Liauthor=Z.+Chenauthor=M.+Zia-Ebrahimiauthor=L.+Bloemauthor=Y.+Zhaiauthor=K.+Hussauthor=S.-b.+Pengauthor=D.+J.+McCann&title=A+novel%2C+selective+inhibitor+of+fibroblast+growth+factor+receptors+that+shows+a+potent+broad+spectrum+of+antitumor+activity+in+several+tumor+xenograft+models&doi=10.1158%2F1535-7163.mct-11-0306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel, Selective Inhibitor of Fibroblast Growth Factor Receptors That Shows a Potent Broad Spectrum of Antitumor Activity in Several Tumor Xenograft Models</span></div><div class="casAuthors">Zhao, Genshi; Li, Wei-ying; Chen, Daohong; Henry, James R.; Li, Hong-Yu; Chen, Zhaogen; Zia-Ebrahimi, Mohammad; Bloem, Laura; Zhai, Yan; Huss, Karen; Peng, Sheng-bin; McCann, Denis J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2200-2210</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The fibroblast growth factor receptors (FGFR) are tyrosine kinases that are present in many types of endothelial and tumor cells and play an important role in tumor cell growth, survival, and migration as well as in maintaining tumor angiogenesis.  Overexpression of FGFRs or aberrant regulation of their activities has been implicated in many forms of human malignancies.  Therefore, targeting FGFRs represents an attractive strategy for development of cancer treatment options by simultaneously inhibiting tumor cell growth, survival, and migration as well as tumor angiogenesis.  Here, we describe a potent, selective, small-mol. FGFR inhibitor, (R)-(E)-2-(4-(2-(5-(1-(3,5-Dichloropyridin-4-yl)ethoxy)-1H-indazol-3yl)vinyl)-1H-pyrazol-1-yl)ethanol, designated as LY2874455.  This mol. is active against all 4 FGFRs, with a similar potency in biochem. assays.  It exhibits a potent activity against FGF/FGFR-mediated signaling in several cancer cell lines and shows an excellent broad spectrum of antitumor activity in several tumor xenograft models representing the major FGF/FGFR relevant tumor histologies including lung, gastric, and bladder cancers and multiple myeloma, and with a well-defined pharmacokinetic/pharmacodynamic relationship.  LY2874455 also exhibits a 6- to 9-fold in vitro and in vivo selectivity on inhibition of FGF- over VEGF-mediated target signaling in mice.  Furthermore, LY2874455 did not show VEGF receptor 2-mediated toxicities such as hypertension at efficacious doses.  Currently, this mol. is being evaluated for its potential use in the clinic.  Mol Cancer Ther; 10(11); 2200-10.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLrzAyZQK_grVg90H21EOLACvtfcHk0lgLeMagmt7wkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVeltb7N&md5=eaf08fd213a836f48da2055e01bd9c29</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-11-0306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-11-0306%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DW.-y.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DHenry%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DH.-Y.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZia-Ebrahimi%26aufirst%3DM.%26aulast%3DBloem%26aufirst%3DL.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DHuss%26aufirst%3DK.%26aulast%3DPeng%26aufirst%3DS.-b.%26aulast%3DMcCann%26aufirst%3DD.%2BJ.%26atitle%3DA%2520novel%252C%2520selective%2520inhibitor%2520of%2520fibroblast%2520growth%2520factor%2520receptors%2520that%2520shows%2520a%2520potent%2520broad%2520spectrum%2520of%2520antitumor%2520activity%2520in%2520several%2520tumor%2520xenograft%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2011%26volume%3D10%26spage%3D2200%26epage%3D2210%26doi%3D10.1158%2F1535-7163.mct-11-0306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, T. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovcheva, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mevellec, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vialard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Lange, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhulst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulussen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van De Ven, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freyne, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squires, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilissen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platero, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkera, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angibaud, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span> <span> </span><span class="NLM_article-title">Discovery and pharmacological characterization of JNJ-42756493 (Erdafitinib), a functionally selective small-molecule FGFR family inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1020</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-16-0589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.MCT-16-0589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28341788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1010-1020&author=T.+P.+S.+Pereraauthor=E.+Jovchevaauthor=L.+Mevellecauthor=J.+Vialardauthor=D.+De+Langeauthor=T.+Verhulstauthor=C.+Paulussenauthor=K.+Van+De+Venauthor=P.+Kingauthor=E.+Freyneauthor=D.+C.+Reesauthor=M.+Squiresauthor=G.+Saxtyauthor=M.+Pageauthor=C.+W.+Murrayauthor=R.+Gilissenauthor=G.+Wardauthor=N.+T.+Thompsonauthor=D.+R.+Newellauthor=N.+Chengauthor=L.+Xieauthor=J.+Yangauthor=S.+J.+Plateroauthor=J.+D.+Karkeraauthor=C.+Moyauthor=P.+Angibaudauthor=S.+Laquerreauthor=M.+V.+Lorenzi&title=Discovery+and+pharmacological+characterization+of+JNJ-42756493+%28Erdafitinib%29%2C+a+functionally+selective+small-molecule+FGFR+family+inhibitor&doi=10.1158%2F1535-7163.mct-16-0589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Pharmacological Characterization of JNJ-42756493 (Erdafitinib), a Functionally Selective Small-Molecule FGFR Family Inhibitor</span></div><div class="casAuthors">Perera, Timothy P. S.; Jovcheva, Eleonora; Mevellec, Laurence; Vialard, Jorge; De Lange, Desiree; Verhulst, Tinne; Paulussen, Caroline; Van De Ven, Kelly; King, Peter; Freyne, Eddy; Rees, David C.; Squires, Matthew; Saxty, Gordon; Page, Martin; Murray, Christopher W.; Gilissen, Ron; Ward, George; Thompson, Neil T.; Newell, David R.; Cheng, Na; Xie, Liang; Yang, Jennifer; Platero, Suso J.; Karkera, Jayaprakash D.; Moy, Christopher; Angibaud, Patrick; Laquerre, Sylvie; Lorenzi, Matthew V.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1010-1020</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Fibroblast growth factor (FGF) signaling plays crit. roles in key biol. processes ranging from embryogenesis to wound healing and has strong links to several hallmarks of cancer.  Genetic alterations in FGF receptor (FGFR) family members are assocd. with increased tumor growth, metastasis, angiogenesis, and decreased survival.  JNJ-42756493, erdafitinib, is an orally active small mol. with potent tyrosine kinase inhibitory activity against all four FGFR family members and selectivity vs. other highly related kinases.  JNJ-42756493 shows rapid uptake into the lysosomal compartment of cells in culture, which is assocd. with prolonged inhibition of FGFR signaling, possibly due to sustained release of the inhibitor.  In xenografts from human tumor cell lines or patient-derived tumor tissue with activating FGFR alterations, JNJ-42756493 administration results in potent and dose-dependent antitumor activity accompanied by pharmacodynamic modulation of phospho-FGFR and phospho-ERK in tumors.  The results of the current study provide a strong rationale for the clin. investigation of JNJ-42756493 in patients with tumors harboring FGFR pathway alterations.  Mol Cancer Ther; 16(6); 1010-20. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhJZ2AgFcI6LVg90H21EOLACvtfcHk0ljdPLSPuUJscA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sht7g%253D&md5=2cae21f09ad519925ab2a3c760284b02</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0589%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DT.%2BP.%2BS.%26aulast%3DJovcheva%26aufirst%3DE.%26aulast%3DMevellec%26aufirst%3DL.%26aulast%3DVialard%26aufirst%3DJ.%26aulast%3DDe%2BLange%26aufirst%3DD.%26aulast%3DVerhulst%26aufirst%3DT.%26aulast%3DPaulussen%26aufirst%3DC.%26aulast%3DVan%2BDe%2BVen%26aufirst%3DK.%26aulast%3DKing%26aufirst%3DP.%26aulast%3DFreyne%26aufirst%3DE.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DSquires%26aufirst%3DM.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DPage%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DGilissen%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DG.%26aulast%3DThompson%26aufirst%3DN.%2BT.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPlatero%26aufirst%3DS.%2BJ.%26aulast%3DKarkera%26aufirst%3DJ.%2BD.%26aulast%3DMoy%26aufirst%3DC.%26aulast%3DAngibaud%26aufirst%3DP.%26aulast%3DLaquerre%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26atitle%3DDiscovery%2520and%2520pharmacological%2520characterization%2520of%2520JNJ-42756493%2520%2528Erdafitinib%2529%252C%2520a%2520functionally%2520selective%2520small-molecule%2520FGFR%2520family%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D1010%26epage%3D1020%26doi%3D10.1158%2F1535-7163.mct-16-0589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isobe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morikami, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindoh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebiike, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, N.</span></span> <span> </span><span class="NLM_article-title">The fibroblast growth factor receptor genetic status as a potential predictor of the sensitivity to CH5183284/Debio 1347, a novel selective FGFR inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2547</span>– <span class="NLM_lpage">2558</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-14-0248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.mct-14-0248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=25169980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=2547-2558&author=Y.+Nakanishiauthor=N.+Akiyamaauthor=T.+Tsukaguchiauthor=T.+Fujiiauthor=K.+Sakataauthor=H.+Saseauthor=T.+Isobeauthor=K.+Morikamiauthor=H.+Shindohauthor=T.+Mioauthor=H.+Ebiikeauthor=N.+Takaauthor=Y.+Aokiauthor=N.+Ishii&title=The+fibroblast+growth+factor+receptor+genetic+status+as+a+potential+predictor+of+the+sensitivity+to+CH5183284%2FDebio+1347%2C+a+novel+selective+FGFR+inhibitor&doi=10.1158%2F1535-7163.mct-14-0248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Fibroblast Growth Factor Receptor Genetic Status as a Potential Predictor of the Sensitivity to CH5183284/Debio 1347, a Novel Selective FGFR Inhibitor</span></div><div class="casAuthors">Nakanishi, Yoshito; Akiyama, Nukinori; Tsukaguchi, Toshiyuki; Fujii, Toshihiko; Sakata, Kiyoaki; Sase, Hitoshi; Isobe, Takehito; Morikami, Kenji; Shindoh, Hidetoshi; Mio, Toshiyuki; Ebiike, Hirosato; Taka, Naoki; Aoki, Yuko; Ishii, Nobuya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2547-2558</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The FGF receptors (FGFR) are tyrosine kinases that are constitutively activated in a subset of tumors by genetic alterations such as gene amplifications, point mutations, or chromosomal translocations/rearrangements.  Recently, small-mol. inhibitors that can inhibit the FGFR family as well as the VEGF receptor (VEGFR) or platelet-derived growth factor receptor (PDGFR) family displayed clin. benefits in cohorts of patients with FGFR genetic alterations.  However, to achieve more potent and prolonged activity in such populations, a selective FGFR inhibitor is still needed.  Here, we report the identification of CH5183284/Debio 1347, a selective and orally available FGFR1, FGFR2, and FGFR3 inhibitor that has a unique chem. scaffold.  By interacting with unique residues in the ATP-binding site of FGFR1, FGFR2, or FGFR3, CH5183284/Debio 1347 selectively inhibits FGFR1, FGFR2, and FGFR3 but does not inhibit kinase insert domain receptor (KDR) or other kinases.  Consistent with its high selectivity for FGFR enzymes, CH5183284/Debio 1347 displayed preferential antitumor activity against cancer cells with various FGFR genetic alterations in a panel of 327 cancer cell lines and in xenograft models.  Because of its unique binding mode, CH5183284/Debio 1347 can inhibit FGFR2 harboring one type of the gatekeeper mutation that causes resistance to other FGFR inhibitors and block FGFR2 V564F-driven tumor growth.  CH5183284/Debio 1347 is under clin. investigation for the treatment of patients harboring FGFR genetic alterations.  Mol Cancer Ther; 13(11); 2547-58. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGIBEnK21337Vg90H21EOLACvtfcHk0ljdPLSPuUJscA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73F&md5=35629d891a421582022b0f6ae0496479</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-14-0248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-14-0248%26sid%3Dliteratum%253Aachs%26aulast%3DNakanishi%26aufirst%3DY.%26aulast%3DAkiyama%26aufirst%3DN.%26aulast%3DTsukaguchi%26aufirst%3DT.%26aulast%3DFujii%26aufirst%3DT.%26aulast%3DSakata%26aufirst%3DK.%26aulast%3DSase%26aufirst%3DH.%26aulast%3DIsobe%26aufirst%3DT.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DShindoh%26aufirst%3DH.%26aulast%3DMio%26aufirst%3DT.%26aulast%3DEbiike%26aufirst%3DH.%26aulast%3DTaka%26aufirst%3DN.%26aulast%3DAoki%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DN.%26atitle%3DThe%2520fibroblast%2520growth%2520factor%2520receptor%2520genetic%2520status%2520as%2520a%2520potential%2520predictor%2520of%2520the%2520sensitivity%2520to%2520CH5183284%252FDebio%25201347%252C%2520a%2520novel%2520selective%2520FGFR%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D2547%26epage%3D2558%26doi%3D10.1158%2F1535-7163.mct-14-0248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, L.</span>; <span class="NLM_string-name">Arkenau, H.-T.</span>; <span class="NLM_string-name">Tran, B.</span>; <span class="NLM_string-name">Soria, J.-C.</span>; <span class="NLM_string-name">Bahleda, R.</span>; <span class="NLM_string-name">Mak, G.</span>; <span class="NLM_string-name">Zhu, A.</span>; <span class="NLM_string-name">Javle, M.</span>; <span class="NLM_string-name">Hiroshi, H.</span>; <span class="NLM_string-name">Benedetti, F.</span>; <span class="NLM_string-name">Huang, J.</span>; <span class="NLM_string-name">Winkler, R.</span>; <span class="NLM_string-name">Meric-Bernstam, F.</span></span>. <span> </span><span class="NLM_article-title">O-020 Early clinical efficacy of TAS-120, a covalently bound FGFR inhibitor, in patients with cholangiocarcinoma</span>.  <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume">28</span>, suppl_3. <span class="refDoi"> DOI: 10.1093/annonc/mdx262.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdx262.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&author=L.+Goyal&author=H.-T.+Arkenau&author=B.+Tran&author=J.-C.+Soria&author=R.+Bahleda&author=G.+Mak&author=A.+Zhu&author=M.+Javle&author=H.+Hiroshi&author=F.+Benedetti&author=J.+Huang&author=R.+Winkler&author=F.+Meric-Bernstam&title=O-020+Early+clinical+efficacy+of+TAS-120%2C+a+covalently+bound+FGFR+inhibitor%2C+in+patients+with+cholangiocarcinoma&doi=10.1093%2Fannonc%2Fmdx262.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx262.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx262.019%26sid%3Dliteratum%253Aachs%26aulast%3DGoyal%26aufirst%3DL.%26atitle%3DO-020%2520Early%2520clinical%2520efficacy%2520of%2520TAS-120%252C%2520a%2520covalently%2520bound%2520FGFR%2520inhibitor%252C%2520in%2520patients%2520with%2520cholangiocarcinoma%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26doi%3D10.1093%2Fannonc%2Fmdx262.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span>; <span class="NLM_string-name">Arkenau, H.</span>; <span class="NLM_string-name">Tran, B.</span>; <span class="NLM_string-name">Bahleda, R.</span>; <span class="NLM_string-name">Kelley, R.</span>; <span class="NLM_string-name">Hierro, C.</span>; <span class="NLM_string-name">Ahn, D.</span>; <span class="NLM_string-name">Zhu, A.</span>; <span class="NLM_string-name">Javle, M.</span>; <span class="NLM_string-name">Winkler, R.</span>; <span class="NLM_string-name">He, H.</span>; <span class="NLM_string-name">Huang, J.</span>; <span class="NLM_string-name">Goyal, L.</span></span> <span> </span><span class="NLM_article-title">O-001 Efficacy of TAS-120, an irreversible Fibroblast Growth Factor Receptor (FGFR) inhibitor, in cholangiocarcinoma patients with FGFR pathway alterations who were previously treated with chemotherapy and other FGFR inhibitors</span>.  <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">29</span>, suppl_5.<span class="refDoi"> DOI: 10.1093/annonc/mdy149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdy149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=29635542" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&author=F.+Meric-Bernstam&author=H.+Arkenau&author=B.+Tran&author=R.+Bahleda&author=R.+Kelley&author=C.+Hierro&author=D.+Ahn&author=A.+Zhu&author=M.+Javle&author=R.+Winkler&author=H.+He&author=J.+Huang&author=L.+Goyal&title=O-001+Efficacy+of+TAS-120%2C+an+irreversible+Fibroblast+Growth+Factor+Receptor+%28FGFR%29+inhibitor%2C+in+cholangiocarcinoma+patients+with+FGFR+pathway+alterations+who+were+previously+treated+with+chemotherapy+and+other+FGFR+inhibitors&doi=10.1093%2Fannonc%2Fmdy149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdy149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdy149%26sid%3Dliteratum%253Aachs%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26atitle%3DO-001%2520Efficacy%2520of%2520TAS-120%252C%2520an%2520irreversible%2520Fibroblast%2520Growth%2520Factor%2520Receptor%2520%2528FGFR%2529%2520inhibitor%252C%2520in%2520cholangiocarcinoma%2520patients%2520with%2520FGFR%2520pathway%2520alterations%2520who%2520were%2520previously%2520treated%2520with%2520chemotherapy%2520and%2520other%2520FGFR%2520inhibitors%26jtitle%3DAnn.%2520Oncol.%26date%3D2018%26volume%3D29%26doi%3D10.1093%2Fannonc%2Fmdy149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paik, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathewson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgour, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frewer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">André, F.</span></span> <span> </span><span class="NLM_article-title">A Phase Ib open-label multicenter study of AZD4547 in patients with advanced squamous cell lung cancers</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5366</span>– <span class="NLM_lpage">5373</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-17-0645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1078-0432.ccr-17-0645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28615371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=5366-5373&author=P.+K.+Paikauthor=R.+Shenauthor=M.+F.+Bergerauthor=D.+Ferryauthor=J.-C.+Soriaauthor=A.+Mathewsonauthor=C.+Rooneyauthor=N.+R.+Smithauthor=M.+Cullbergauthor=E.+Kilgourauthor=D.+Landersauthor=P.+Frewerauthor=N.+Brooksauthor=F.+Andr%C3%A9&title=A+Phase+Ib+open-label+multicenter+study+of+AZD4547+in+patients+with+advanced+squamous+cell+lung+cancers&doi=10.1158%2F1078-0432.ccr-17-0645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Phase Ib Open-Label Multicenter Study of AZD4547 in Patients with Advanced Squamous Cell Lung Cancers</span></div><div class="casAuthors">Paik, Paul K.; Shen, Ronglai; Berger, Michael F.; Ferry, David; Soria, Jean-Charles; Mathewson, Alastair; Rooney, Claire; Smith, Neil R.; Cullberg, Marie; Kilgour, Elaine; Landers, Donal; Frewer, Paul; Brooks, Nigel; Andre, Fabrice</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5366-5373</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Squamous cell lung cancers (SQCLC) account for 25% of all NSCLCs, yet the prognosis of these patients is poor and treatment options are limited.  Amplified FGFR1 is one of the most common oncogenic events in SQCLCs, occurring in approx. 20% of cases.  AZD4547 is a potent and selective FGFR1-3 inhibitor with antitumor activity in FGFR1-amplified SQCLC cell lines and patient-derived xenografts.  Exptl. Design: On the basis of these data, we performed a phase I study of AZD4547 in patients with previously treated stage IV FGFR1-amplified SQCLCs (NCT00979134).  FGFR1 amplification (FGFR1:CEP8 ≥ 2) was detd. by FISH.  The primary endpoint was safety/tolerability.  Secondary endpoints included antitumor activity, pharmacokinetics, pharmacodynamics, and mol. analyses.  Results: Fifteen FGFR1-amplified patients were treated.  The most common related adverse events (AE) were gastrointestinal and dermatol.  Grade ≥3-related AEs occurred in 3 patients (23%).  Thirteen patients were evaluable for radiog. response assessment.  The overall response rate was 8% (1 PR).  Two of 15 patients (13.3%) were progression-free at 12 wk, and the median overall survival was 4.9 mo.  Mol. tests, including next-generation sequencing, gene expression anal., and FGFR1 immunohistochem., showed poor correlation between gene amplification and expression, potential genomic modifiers of efficacy, and heterogeneity in 8p11 amplicon.  Conclusions: AZD4547 was tolerable at a dosage of 80 mg oral twice a day, with modest antitumor activity.  Detailed mol. studies show that these tumors are heterogeneous, with a range of mutational covariates and stark differences in gene expression of the 8p11 amplicon that likely explain the modest efficacy of FGFR inhibition in this disease.  Clin Cancer Res; 23(18); 5366-73. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtbUEY_Z929rVg90H21EOLACvtfcHk0lgzdgpWtgF4Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7fP&md5=17d6065f9f30ea9dd61ee4415fb72b33</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-17-0645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-17-0645%26sid%3Dliteratum%253Aachs%26aulast%3DPaik%26aufirst%3DP.%2BK.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DFerry%26aufirst%3DD.%26aulast%3DSoria%26aufirst%3DJ.-C.%26aulast%3DMathewson%26aufirst%3DA.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DN.%2BR.%26aulast%3DCullberg%26aufirst%3DM.%26aulast%3DKilgour%26aufirst%3DE.%26aulast%3DLanders%26aufirst%3DD.%26aulast%3DFrewer%26aufirst%3DP.%26aulast%3DBrooks%26aufirst%3DN.%26aulast%3DAndr%25C3%25A9%26aufirst%3DF.%26atitle%3DA%2520Phase%2520Ib%2520open-label%2520multicenter%2520study%2520of%2520AZD4547%2520in%2520patients%2520with%2520advanced%2520squamous%2520cell%2520lung%2520cancers%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D5366%26epage%3D5373%26doi%3D10.1158%2F1078-0432.ccr-17-0645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seckl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badman, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zubairi, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Corbacho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cresti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombes, C.</span></span> <span> </span><span class="NLM_article-title">RADICAL trial: a phase Ib/IIa study to assess the safety and efficacy of AZD4547 in combination with either anastrozole or letrozole in ER positive breast cancer patients progressing on these aromatase inhibitors (AIs)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1059</span>, <span class="refDoi"> DOI: 10.1200/jco.2017.35.15_suppl.1059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1200%2Fjco.2017.35.15_suppl.1059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1059&author=M.+Secklauthor=P.+D.+Badmanauthor=X.+Liuauthor=I.+R.+MacPhersonauthor=I.+H.+Zubairiauthor=R.+D.+Bairdauthor=J.+Garcia-Corbachoauthor=N.+Crestiauthor=E.+R.+Plummerauthor=A.+C.+Armstrongauthor=R.+Allertonauthor=D.+Landersauthor=H.+Nicholasauthor=L.+McLellanauthor=A.+K.+Limauthor=C.+Coombes&title=RADICAL+trial%3A+a+phase+Ib%2FIIa+study+to+assess+the+safety+and+efficacy+of+AZD4547+in+combination+with+either+anastrozole+or+letrozole+in+ER+positive+breast+cancer+patients+progressing+on+these+aromatase+inhibitors+%28AIs%29&doi=10.1200%2Fjco.2017.35.15_suppl.1059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1200%2Fjco.2017.35.15_suppl.1059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2017.35.15_suppl.1059%26sid%3Dliteratum%253Aachs%26aulast%3DSeckl%26aufirst%3DM.%26aulast%3DBadman%26aufirst%3DP.%2BD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMacPherson%26aufirst%3DI.%2BR.%26aulast%3DZubairi%26aufirst%3DI.%2BH.%26aulast%3DBaird%26aufirst%3DR.%2BD.%26aulast%3DGarcia-Corbacho%26aufirst%3DJ.%26aulast%3DCresti%26aufirst%3DN.%26aulast%3DPlummer%26aufirst%3DE.%2BR.%26aulast%3DArmstrong%26aufirst%3DA.%2BC.%26aulast%3DAllerton%26aufirst%3DR.%26aulast%3DLanders%26aufirst%3DD.%26aulast%3DNicholas%26aufirst%3DH.%26aulast%3DMcLellan%26aufirst%3DL.%26aulast%3DLim%26aufirst%3DA.%2BK.%26aulast%3DCoombes%26aufirst%3DC.%26atitle%3DRADICAL%2520trial%253A%2520a%2520phase%2520Ib%252FIIa%2520study%2520to%2520assess%2520the%2520safety%2520and%2520efficacy%2520of%2520AZD4547%2520in%2520combination%2520with%2520either%2520anastrozole%2520or%2520letrozole%2520in%2520ER%2520positive%2520breast%2520cancer%2520patients%2520progressing%2520on%2520these%2520aromatase%2520inhibitors%2520%2528AIs%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D35%26spage%3D1059%26doi%3D10.1200%2Fjco.2017.35.15_suppl.1059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, L.</span>; <span class="NLM_string-name">Liu, L.
Y.</span>; <span class="NLM_string-name">Lennerz, J. K.</span>; <span class="NLM_string-name">Harding, J. J.</span>; <span class="NLM_string-name">Huang, J.</span>; <span class="NLM_string-name">Winkler, R.</span>; <span class="NLM_string-name">Hiroshi, H.</span>; <span class="NLM_string-name">Ting, D.
T.</span>; <span class="NLM_string-name">Juric, D.</span>; <span class="NLM_string-name">Corcoran, R. B.</span>; <span class="NLM_string-name">El-Bardeesy, N.</span>; <span class="NLM_string-name">Zhu, A. X.</span></span> <span> </span><span class="NLM_article-title">Abstract LB-092: TAS120, a covalently-binding FGFR inhibitor (FGFRi), overcomes resistance to BGJ398 in patients with FGFR2 fusion positive cholangiocarcinoma</span>.  <i>Cancer Research</i>, <span class="NLM_year">2018</span>; Vol.  <span class="NLM_volume">78</span>, LB-092-LB-092.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=L.+Goyal&author=L.%0AY.+Liu&author=J.+K.+Lennerz&author=J.+J.+Harding&author=J.+Huang&author=R.+Winkler&author=H.+Hiroshi&author=D.%0AT.+Ting&author=D.+Juric&author=R.+B.+Corcoran&author=N.+El-Bardeesy&author=A.+X.+Zhu&title=Abstract+LB-092%3A+TAS120%2C+a+covalently-binding+FGFR+inhibitor+%28FGFRi%29%2C+overcomes+resistance+to+BGJ398+in+patients+with+FGFR2+fusion+positive+cholangiocarcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DGoyal%26aufirst%3DL.%26atitle%3DAbstract%2520LB-092%253A%2520TAS120%252C%2520a%2520covalently-binding%2520FGFR%2520inhibitor%2520%2528FGFRi%2529%252C%2520overcomes%2520resistance%2520to%2520BGJ398%2520in%2520patients%2520with%2520FGFR2%2520fusion%2520positive%2520cholangiocarcinoma%26jtitle%3DCancer%2520Research%26date%3D2018%26volume%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel disruptor of silencing telomeric 1-like (DOT1L) inhibitors using a target-specific scoring function for the (S)-Adenosyl-l-methionine (SAM)-dependent methyltransferase family</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2026</span>– <span class="NLM_lpage">2036</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01785</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01785" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1SjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2026-2036&author=Y.+Wangauthor=L.+Liauthor=B.+Zhangauthor=J.+Xingauthor=S.+Chenauthor=W.+Wanauthor=Y.+Songauthor=H.+Jiangauthor=H.+Jiangauthor=C.+Luoauthor=M.+Zheng&title=Discovery+of+novel+disruptor+of+silencing+telomeric+1-like+%28DOT1L%29+inhibitors+using+a+target-specific+scoring+function+for+the+%28S%29-Adenosyl-l-methionine+%28SAM%29-dependent+methyltransferase+family&doi=10.1021%2Facs.jmedchem.6b01785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Disruptor of Silencing Telomeric 1-Like (DOT1L) Inhibitors using a Target-Specific Scoring Function for the (S)-Adenosyl-L-methionine (SAM)-Dependent Methyltransferase Family</span></div><div class="casAuthors">Wang, Yulan; Li, Linjuan; Zhang, Bidong; Xing, Jing; Chen, Shijie; Wan, Wei; Song, Yakai; Jiang, Hao; Jiang, Hualiang; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2026-2036</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The disruptor of telomeric silencing 1-like (DOT1L) protein is a histone H3K79 methyltransferase that plays a key role in transcriptional elongation and cell cycle regulation and is required for the development and maintenance of MLL-rearranged mixed lineage leukemia.  Much effort has been dedicated towards discovering novel scaffold DOT1L inhibitors using different strategies.  Here, the authors report the development and application of a target-specific scoring function, the SAM score, for S-adenosyl-L-methionine (SAM)-dependent methyltransferase, for the discovery of novel DOT1L inhibitors.  Based on the SAM score, the authors successfully identified a novel class of DOT1L inhibitors with a scaffold of [1,2,4]-triazolo-[3,4-b][1,3,4]-thiadiazole, in which compd. (I) exhibits an IC50 value of 8.3 μM with selectivity vs. other tested SAM-dependent methyltransferases.  In cellular studies, I selectively targets DOT1L, blocks the proliferation of mixed lineage leukemia cell lines, and causes cell cycle arrest and apoptosis.  Moreover, the authors analyzed the putative binding modes of 6 and its analogs obtained by mol. docking, which may assist with the future development of DOT1L inhibitors with improved potency and selectivity profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpphV6NpZDlrVg90H21EOLACvtfcHk0lhgtmaPbhBZrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1SjtLo%253D&md5=b1df8ae062762c717ca467014cd93de6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01785%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWan%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520novel%2520disruptor%2520of%2520silencing%2520telomeric%25201-like%2520%2528DOT1L%2529%2520inhibitors%2520using%2520a%2520target-specific%2520scoring%2520function%2520for%2520the%2520%2528S%2529-Adenosyl-l-methionine%2520%2528SAM%2529-dependent%2520methyltransferase%2520family%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2026%26epage%3D2036%26doi%3D10.1021%2Facs.jmedchem.6b01785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliseenkova, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.molcel.2007.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=17803937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=717-730&author=H.+Chenauthor=J.+Maauthor=W.+Liauthor=A.+V.+Eliseenkovaauthor=C.+Xuauthor=T.+A.+Neubertauthor=W.+T.+Millerauthor=M.+Mohammadi&title=A+molecular+brake+in+the+kinase+hinge+region+regulates+the+activity+of+receptor+tyrosine+kinases&doi=10.1016%2Fj.molcel.2007.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Huaibin; Ma, Jinghong; Li, Wanqing; Eliseenkova, Anna V.; Xu, Chongfeng; Neubert, Thomas A.; Miller, W. Todd; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">717-730</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Activating mutations in the tyrosine kinase domain of receptor tyrosine kinases (RTKs) cause cancer and skeletal disorders.  Comparison of the crystal structures of unphosphorylated and phosphorylated wild-type FGFR2 kinase domains (FGFR2Ks) with those of seven unphosphorylated pathogenic mutants reveals an autoinhibitory "mol. brake" mediated by a triad of residues in the kinase hinge region of all FGFRs.  Structural anal. shows that many other RTKs, including PDGFRs, VEGFRs, KIT, CSF1R, FLT3, TEK, and TIE, are also subject to regulation by this brake.  Pathogenic mutations activate FGFRs and other RTKs by disengaging the brake either directly or indirectly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYTkvrN7lYrVg90H21EOLACvtfcHk0lgMxODOUN85VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I&md5=dba8a6b97b9d1826b2223146be137963</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DA%2520molecular%2520brake%2520in%2520the%2520kinase%2520hinge%2520region%2520regulates%2520the%2520activity%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D717%26epage%3D730%26doi%3D10.1016%2Fj.molcel.2007.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">Structural mimicry of a-loop tyrosine phosphorylation by a pathogenic FGF receptor 3 mutation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1889</span>– <span class="NLM_lpage">1896</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.str.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23972473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Slu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1889-1896&author=Z.+Huangauthor=H.+Chenauthor=S.+Blaisauthor=T.+A.+Neubertauthor=X.+Liauthor=M.+Mohammadi&title=Structural+mimicry+of+a-loop+tyrosine+phosphorylation+by+a+pathogenic+FGF+receptor+3+mutation&doi=10.1016%2Fj.str.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Mimicry of A-Loop Tyrosine Phosphorylation by a Pathogenic FGF Receptor 3 Mutation</span></div><div class="casAuthors">Huang, Zhifeng; Chen, Huaibin; Blais, Steven; Neubert, Thomas A.; Li, Xiaokun; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1889-1896</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The K650E gain-of-function mutation in the tyrosine kinase domain of FGF receptor 3 (FGFR3) causes Thanatophoric Dysplasia type II, a neonatal lethal congenital dwarfism syndrome, and when acquired somatically, it contributes to carcinogenesis.  In this report, we det. the crystal structure of the FGFR3 kinase domain harboring this pathogenic mutation and show that the mutation introduces a network of intramol. hydrogen bonds to stabilize the active-state conformation.  In the crystal, the mutant FGFR3 kinases are caught in the act of trans-phosphorylation on a kinase insert autophosphorylation site, emphasizing the fact that the K650E mutation circumvents the requirement for A-loop tyrosine phosphorylation in kinase activation.  Anal. of this trans-phosphorylation complex sheds light onto the determinants of tyrosine trans-phosphorylation specificity.  We propose that the targeted inhibition of this pathogenic FGFR3 kinase may be achievable by small mol. kinase inhibitors that selectively bind the active-state conformation of FGFR3 kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNzI66rWB_urVg90H21EOLACvtfcHk0lgMxODOUN85VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Slu7rL&md5=f555b8b61c93500f436f1cee396b1aba</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DBlais%26aufirst%3DS.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DStructural%2520mimicry%2520of%2520a-loop%2520tyrosine%2520phosphorylation%2520by%2520a%2520pathogenic%2520FGF%2520receptor%25203%2520mutation%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1889%26epage%3D1896%26doi%3D10.1016%2Fj.str.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorissen, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">D198</span>– <span class="NLM_lpage">D201</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=17145705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+a+web-accessible+database+of+experimentally+determined+protein-ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span></div><div class="casAuthors">Liu, Tiqing; Lin, Yuhmei; Wen, Xin; Jorissen, Robert N.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D198-D201</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB is a publicly accessible database currently contg. ∼20 000 exptl. detd. binding affinities of protein-ligand complexes, for 110 protein targets including isoforms and mutational variants, and ∼11 000 small mol. ligands.  The data are extd. from the scientific literature, data collection focusing on proteins that are drug-targets or candidate drug-targets and for which structural data are present in the Protein Data Bank.  The BindingDB website supports a range of query types, including searches by chem. structure, substructure and similarity; protein sequence; ligand and protein names; affinity ranges and mol. wt.  Data sets generated by BindingDB queries can be downloaded in the form of annotated SDfiles for further anal., or used as the basis for virtual screening of a compd. database uploaded by the user.  The data in BindingDB are linked both to structural data in the PDB via PDB IDs and chem. and sequence searches, and to the literature in PubMed via PubMed IDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymy3G_LuG-rVg90H21EOLACvtfcHk0lhb5wlhtCVQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D&md5=0ccb20d9b9178a624d4829b5909e7ff8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein-ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.%0AA.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhb5wlhtCVQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Knowledge-based scoring functions in drug design: 2. can the knowledge base be enriched?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">386</span>– <span class="NLM_lpage">397</span>, <span class="refDoi"> DOI: 10.1021/ci100343j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100343j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFSm" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=386-397&author=Q.+Shenauthor=B.+Xiongauthor=M.+Zhengauthor=X.+Luoauthor=C.+Luoauthor=X.+Liuauthor=Y.+Duauthor=J.+Liauthor=W.+Zhuauthor=J.+Shenauthor=H.+Jiang&title=Knowledge-based+scoring+functions+in+drug+design%3A+2.+can+the+knowledge+base+be+enriched%3F&doi=10.1021%2Fci100343j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-Based Scoring Functions in Drug Design: 2. Can the Knowledge Base Be Enriched?</span></div><div class="casAuthors">Shen, Qiancheng; Xiong, Bing; Zheng, Mingyue; Luo, Xiaomin; Luo, Cheng; Liu, Xian; Du, Yun; Li, Jing; Zhu, Weiliang; Shen, Jingkang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">386-397</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fast and accurate predicting of the binding affinities of large sets of diverse protein-ligand complexes is an important, yet extremely challenging, task in drug discovery.  The development of knowledge-based scoring functions exploiting structural information of known protein-ligand complexes represents a valuable contribution to such a computational prediction.  In this study, we report a scoring function named IPMF that integrates addnl. exptl. binding affinity information into the extd. potentials, on the assumption that a scoring function with the "enriched" knowledge base may achieve increased accuracy in binding affinity prediction.  In our approach, the functions and atom types of PMF04 were inherited to implicitly capture binding effects that are hard to model explicitly, and a novel iteration device was designed to gradually tailor the initial potentials.  We evaluated the performance of the resultant IPMF with a diverse set of 219 protein-ligand complexes and compared it with seven scoring functions commonly used in computer-aided drug design, including GLIDE, AutoDock4, VINA, PLP, LUDI, PMF, and PMF04.  While the IPMF is only moderately successful in ranking native or near native conformations, it yields the lowest mean error of 1.41 log Ki/Kd units from measured inhibition affinities and the highest Pearson's correlation coeff. of Rp2 0.40 for the test set.  These results corroborate our initial supposition about the role of "enriched" knowledge base.  With the rapid growing vol. of high-quality structural and interaction data in the public domain, this work marks a pos. step toward improving the accuracy of knowledge-based scoring functions in binding affinity prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVKikMRxKKibVg90H21EOLACvtfcHk0ljlTMhZaIqW7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFSm&md5=5e2c7f13dd7fdcb1fcca5527a6e70f7e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci100343j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100343j%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DKnowledge-based%2520scoring%2520functions%2520in%2520drug%2520design%253A%25202.%2520can%2520the%2520knowledge%2520base%2520be%2520enriched%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D386%26epage%3D397%26doi%3D10.1021%2Fci100343j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. J.</span></span> <span> </span><span class="NLM_article-title">LIBSVM: a library for support vector machines</span>. <i>ACM Trans. Intell. Syst. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1145/1961189.1961199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1145%2F1961189.1961199" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=1-27&author=C.-C.+Changauthor=C.+J.+Lin&title=LIBSVM%3A+a+library+for+support+vector+machines&doi=10.1145%2F1961189.1961199"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1145%2F1961189.1961199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F1961189.1961199%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.-C.%26aulast%3DLin%26aufirst%3DC.%2BJ.%26atitle%3DLIBSVM%253A%2520a%2520library%2520for%2520support%2520vector%2520machines%26jtitle%3DACM%2520Trans.%2520Intell.%2520Syst.%2520Technol.%26date%3D2011%26volume%3D2%26spage%3D1%26epage%3D27%26doi%3D10.1145%2F1961189.1961199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guyon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnhill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vapnik, V.</span></span> <span> </span><span class="NLM_article-title">Gene selection for cancer classification using support vector machines</span>. <i>Mach. Learn.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1023/a:1012487302797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1023%2FA%3A1012487302797" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=389-422&author=I.+Guyonauthor=J.+Westonauthor=S.+Barnhillauthor=V.+Vapnik&title=Gene+selection+for+cancer+classification+using+support+vector+machines&doi=10.1023%2Fa%3A1012487302797"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1023%2FA%3A1012487302797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1012487302797%26sid%3Dliteratum%253Aachs%26aulast%3DGuyon%26aufirst%3DI.%26aulast%3DWeston%26aufirst%3DJ.%26aulast%3DBarnhill%26aufirst%3DS.%26aulast%3DVapnik%26aufirst%3DV.%26atitle%3DGene%2520selection%2520for%2520cancer%2520classification%2520using%2520support%2520vector%2520machines%26jtitle%3DMach.%2520Learn.%26date%3D2002%26volume%3D46%26spage%3D389%26epage%3D422%26doi%3D10.1023%2Fa%3A1012487302797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljlTMhZaIqW7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0li1Q-AQAOYrIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0li1Q-AQAOYrIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Progress with covalent small-molecule kinase inhibitors</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">735</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.01.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.drudis.2018.01.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=29337202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=727-735&author=Z.+Zhaoauthor=P.+E.+Bourne&title=Progress+with+covalent+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.drudis.2018.01.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Progress with covalent small-molecule kinase inhibitors</span></div><div class="casAuthors">Zhao, Zheng; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">With reduced risk of toxicity and high selectivity, covalent small-mol. kinase inhibitors (CSKIs) have emerged rapidly.  Through the lens of structural system pharmacol., here we review this rapid progress by considering design strategies and the challenges and opportunities offered by current CSKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog6l6qMAopYLVg90H21EOLACvtfcHk0ljHmDLCiHgpOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D&md5=f97a19f53a72044812a778ee9351fe94</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProgress%2520with%2520covalent%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2018%26volume%3D23%26spage%3D727%26epage%3D735%26doi%3D10.1016%2Fj.drudis.2018.01.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Properties of FDA-approved small molecule protein kinase inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.phrs.2019.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=30877063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=19-50&author=R.+Roskoski&title=Properties+of+FDA-approved+small+molecule+protein+kinase+inhibitors&doi=10.1016%2Fj.phrs.2019.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of FDA-approved small molecule protein kinase inhibitors</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19-50</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Because mutations, overexpression, and dysregulation of protein kinases play essential roles in the pathogenesis of many illnesses, this enzyme family has become one of the most important drug targets in the past 20 years.  The US FDA has approved 48 small mol. protein kinase inhibitors, nearly all of which are orally effective with the exceptions of netarsudil (which is given as an eye drop) and temsirolimus (which is given i.v.).  Of the 48 approved drugs, the majority (25) target receptor protein-tyrosine kinases, ten target non-receptor protein-tyrosine kinases, and 13 target protein-serine/threonine protein kinases.  The data indicate that 43 of these drugs are used in the treatment of malignancies (36 against solid tumors including lymphomas and seven against non-solid tumors, e.g., leukemias).  Seven drugs are used in the treatment of non-malignancies: baricitinib, rheumatoid arthritis; fostamatinib, chronic immune thrombocytopenia; ruxolitinib, myelofibrosis and polycythemia vera; nintedanib, idiopathic pulmonary fibrosis; sirolimus, renal graft vs. host disease; netarsudil, glaucoma; tofacitinib, rheumatoid arthritis, Crohn disease, and ulcerative colitis.  Moreover, ibrutinib and sirolimus are used for the treatment of both malignant and non-malignant diseases.  The most common drug targets include ALK, B-Raf, BCR-Abl, epidermal growth factor receptor (EGFR), and vascular endothelial growth factor receptor (VEGFR).  Most of the small mol. inhibitors (45) interact directly with the protein kinase domain.  In contrast, sirolimus, temsirolimus, and everolimus are larger mols. (MW ≈ 1000) that bind to FKBP-12 to generate a complex that inhibits mTOR (mammalian target of rapamycin).  This review presents the available drug-enzyme X-ray crystal structures for 27 of the approved drugs as well as the chem. structures and physicochem. properties of all of the FDA-approved small mol. protein kinase antagonists.  Six of the drugs bind covalently and irreversibly to their target.  Twenty of the 48 drugs have mol. wts. greater than 500, exceeding a Lipinski rule of five criterion.  Excluding the macrolides (everolimus, sirolimus, temsirolimus), the av. mol. wt. of drugs is 480 with a range of 306 (ruxolitinib) to 615 (trametinib).  Nearly half of the antagonists (23) have a lipophilic efficiency with values of less than five while the recommended optima range from 5-10.  One of the vexing problems is the near universal development of resistance that is assocd. with the use of small mol. protein kinase inhibitors for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLum_hDK00hbVg90H21EOLACvtfcHk0ljHmDLCiHgpOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D&md5=a24dee87603bc4ea12bd34616e46f53d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DProperties%2520of%2520FDA-approved%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D144%26spage%3D19%26epage%3D50%26doi%3D10.1016%2Fj.phrs.2019.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lgLFLufbzv1Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bliven, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Determining cysteines available for covalent inhibition across the human kinome</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2879</span>– <span class="NLM_lpage">2889</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2879-2889&author=Z.+Zhaoauthor=Q.+Liuauthor=S.+Blivenauthor=L.+Xieauthor=P.+E.+Bourne&title=Determining+cysteines+available+for+covalent+inhibition+across+the+human+kinome&doi=10.1021%2Facs.jmedchem.6b01815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Determining Cysteines Available for Covalent Inhibition Across the Human Kinome</span></div><div class="casAuthors">Zhao, Zheng; Liu, Qingsong; Bliven, Spencer; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2879-2889</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalently bound protein kinase inhibitors have been frequently designed to target non-catalytic Cys residues at the ATP binding site.  Thus, it is important to know if a given Cys residue can form a covalent bond.  Here, we combined a function-site interaction fingerprint method and DFT calcns. to det. the potential of Cys residues to form a covalent interaction with an inhibitor.  By harnessing the human structural kinome, a comprehensive structure-based binding site Cys dataset was assembled.  The orientation of the Cys SH group indicated which Cys residues could potentially form covalent bonds.  These covalent inhibitor accessible Cys residues were located within 5 kinase regions (P-loop, roof of pocket, front pocket, catalytic-2 of the catalytic loop and DFG-3 close to the DFG peptide).  In an independent test set, these Cys residues covered 95% of covalent kinase inhibitors.  Thus, this study provides new insights into Cys reactivity and preference which is important for the prospective development of covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKJnGGcKhC5rVg90H21EOLACvtfcHk0lgLFLufbzv1Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVGjsLg%253D&md5=22ea95ce38eb0836d32c1332dba54fad</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01815%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DBliven%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetermining%2520cysteines%2520available%2520for%2520covalent%2520inhibition%2520across%2520the%2520human%2520kinome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2879%26epage%3D2889%26doi%3D10.1021%2Facs.jmedchem.6b01815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A structure-guided approach to creating covalent FGFR inhibitors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.chembiol.2010.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20338520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=285-295&author=W.+Zhouauthor=W.+Hurauthor=U.+McDermottauthor=A.+Duttauthor=W.+Xianauthor=S.+B.+Ficarroauthor=J.+Zhangauthor=S.+V.+Sharmaauthor=J.+Bruggeauthor=M.+Meyersonauthor=J.+Settlemanauthor=N.+S.+Gray&title=A+structure-guided+approach+to+creating+covalent+FGFR+inhibitors&doi=10.1016%2Fj.chembiol.2010.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Structure-Guided Approach to Creating Covalent FGFR Inhibitors</span></div><div class="casAuthors">Zhou, Wenjun; Hur, Wooyoung; McDermott, Ultan; Dutt, Amit; Xian, Wa; Ficarro, Scott B.; Zhang, Jianming; Sharma, Sreenath V.; Brugge, Joan; Meyerson, Matthew; Settleman, Jeffrey; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">285-295</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The fibroblast growth factor receptor tyrosine kinases (FGFR1, 2, 3, and 4) represent promising therapeutic targets in a no. of cancers.  We have developed the first potent and selective irreversible inhibitor of FGFR1, 2, 3, and 4, which we named FIIN-1 that forms a covalent bond with cysteine 486 located in the P loop of the FGFR1 ATP binding site.  We demonstrated that the inhibitor potently inhibits Tel-FGFR1-transformed Ba/F3 cells (EC50 = 14 nM) as well as numerous FGFR-dependent cancer cell lines.  A biotin-derivatized version of the inhibitor, FIIN-1-biotin, was shown to covalently label FGFR1 at Cys486.  FIIN-1 is a useful probe of FGFR-dependent cellular phenomena and may provide a starting point of the development of therapeutically relevant irreversible inhibitors of wild-type and drug-resistant forms of FGFR kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqxMZh8QQP47Vg90H21EOLACvtfcHk0lgLFLufbzv1Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKhu7s%253D&md5=0b7b084a073b1c6f30f38565a0e4db81</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DDutt%26aufirst%3DA.%26aulast%3DXian%26aufirst%3DW.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DBrugge%26aufirst%3DJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520structure-guided%2520approach%2520to%2520creating%2520covalent%2520FGFR%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D285%26epage%3D295%26doi%3D10.1016%2Fj.chembiol.2010.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Site of metabolism prediction for six biotransformations mediated by cytochromes P450</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1251</span>– <span class="NLM_lpage">1258</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fbioinformatics%2Fbtp140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19286831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlslGgtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1251-1258&author=M.+Zhengauthor=X.+Luoauthor=Q.+Shenauthor=Y.+Wangauthor=Y.+Duauthor=W.+Zhuauthor=H.+Jiang&title=Site+of+metabolism+prediction+for+six+biotransformations+mediated+by+cytochromes+P450&doi=10.1093%2Fbioinformatics%2Fbtp140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Site of metabolism prediction for six biotransformations mediated by cytochromes P450</span></div><div class="casAuthors">Zheng, Mingyue; Luo, Xiaomin; Shen, Qiancheng; Wang, Yong; Du, Yun; Zhu, Weiliang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1251-1258</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">One goal of metabolomics is to define and monitor the entire metabolite complement of a cell, while it is still far from reach since systematic and rapid approaches for detg. the biotransformations of newly discovered metabolites are lacking.  For drug development, such metabolic biotransformation of a new chem. entity (NCE) is of more interest because it may profoundly affect its bioavailability, activity and toxicity profile.  The use of in silico methods to predict the site of metab. (SOM) in phase I cytochromes P 450-mediated reactions is usually a starting point of metabolic pathway studies, which may also assist in the process of drug/lead optimization.  This article reports the Cytochromes P 450 (CYP450)-mediated SOM prediction for the six most important metabolic reactions by incorporating the use of machine learning and semi-empirical quantum chem. calcns.  Non-local models were developed on the basis of a large dataset comprising 1858 metabolic reactions extd. from 1034 heterogeneous chems.  For validation, the overall accuracies of all six reaction types are higher than 0.81, four of which exceed 0.90.  In further receiver operating characteristic (ROC) analyses, each of the SOM model gave a significant area under curve (AUC) value over 0.86, indicating a good predicting power.  An external test was made on a previously published dataset, of which 80% of the exptl. obsd. SOMs can be correctly identified by applying the full set of our SOM models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7cfpdfTECvbVg90H21EOLACvtfcHk0li4syjOaV9GOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlslGgtL0%253D&md5=aaab21cdbc9d8c625d3a101d05e1d959</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp140%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DSite%2520of%2520metabolism%2520prediction%2520for%2520six%2520biotransformations%2520mediated%2520by%2520cytochromes%2520P450%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D1251%26epage%3D1258%26doi%3D10.1093%2Fbioinformatics%2Fbtp140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span> <span> </span><span class="NLM_article-title"><i>In silico</i> site of metabolism prediction for human UGT-catalyzed reactions</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">398</span>– <span class="NLM_lpage">405</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fbioinformatics%2Fbtt681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24273240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=398-405&author=J.+Pengauthor=J.+Luauthor=Q.+Shenauthor=M.+Zhengauthor=X.+Luoauthor=W.+Zhuauthor=H.+Jiangauthor=K.+Chen&title=In+silico+site+of+metabolism+prediction+for+human+UGT-catalyzed+reactions&doi=10.1093%2Fbioinformatics%2Fbtt681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">In silico site of metabolism prediction for human UGT-catalyzed reactions</span></div><div class="casAuthors">Peng, Jianlong; Lu, Jing; Shen, Qiancheng; Zheng, Mingyue; Luo, Xiaomin; Zhu, Weiliang; Jiang, Hualiang; Chen, Kaixian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">398-405</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The human uridine diphosphate-glucuronosyltransferase enzyme family catalyzes the glucuronidation of the glycosyl group of a nucleotide sugar to an acceptor compd. (substrate), which is the most common conjugation pathway that serves to protect the organism from the potential toxicity of xenobiotics.  Moreover, it could affect the pharmacol. profile of a drug.  Therefore, it is important to identify the metabolically labile sites for glucuronidation.  Results: In the present study, we developed four in silico models to predict sites of glucuronidation, for four major sites of metab. functional groups, i.e. aliph. hydroxyl, arom. hydroxyl, carboxylic acid or amino nitrogen, resp.  According to the mechanism of glucuronidation, a series of 'local' and 'global' mol. descriptors characterizing the at. reactivity, bonding strength and phys.-chem. properties were calcd. and selected with a genetic algorithm-based feature selection approach.  The constructed support vector machine classification models show good prediction performance, with the balanced accuracy ranging from 0.88 to 0.96 on test set.  For further validation, our models can successfully identify 84% of exptl. obsd. sites of metabs. for an external test set contg. 54 mols.  Availability and implementation: The software somugt based on our models is available at www.dddc.ac.cn/adme/jlpeng/somugt_win32.zip.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="344c5958415b745d59591a55571a575a">[email protected]</a> or <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="016c787b69646f664160686d2f7269626f622f60622f626f">[email protected]</a> Supplementary information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTnU5JiJGKPrVg90H21EOLACvtfcHk0lglRBWw80Ig2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D&md5=c99109a95f1bb99af1c93a3946a3dd44</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt681%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26atitle%3DIn%2520silico%2520site%2520of%2520metabolism%2520prediction%2520for%2520human%2520UGT-catalyzed%2520reactions%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26spage%3D398%26epage%3D405%26doi%3D10.1093%2Fbioinformatics%2Fbtt681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rydberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaretzki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breneman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, L.</span></span> <span> </span><span class="NLM_article-title">SMARTCyp: A 2D method for prediction of Cytochrome P450-mediated drug metabolism</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1021/ml100016x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100016x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFOqsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=96-100&author=P.+Rydbergauthor=D.+E.+Gloriamauthor=J.+Zaretzkiauthor=C.+Brenemanauthor=L.+Olsen&title=SMARTCyp%3A+A+2D+method+for+prediction+of+Cytochrome+P450-mediated+drug+metabolism&doi=10.1021%2Fml100016x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">SMARTCyp: A 2D Method for Prediction of Cytochrome P450-Mediated Drug Metabolism</span></div><div class="casAuthors">Rydberg, Patrik; Gloriam, David E.; Zaretzki, Jed; Breneman, Curt; Olsen, Lars</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">96-100</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMARTCyp is an in silico method that predicts the sites of cytochrome P 450-mediated metab. of druglike mols.  The method is foremost a reactivity model, and as such, it shows a preference for predicting sites that are metabolized by the cytochrome P 450 3A4 isoform.  SMARTCyp predicts the site of metab. directly from the 2D structure of a mol., without requiring calcn. of electronic properties or generation of 3D structures.  This is a major advantage, because it makes SMARTCyp very fast.  Other advantages are that exptl. data are not a prerequisite to create the model, and it can easily be integrated with other methods to create models for other cytochrome P 450 isoforms.  Benchmarking tests on a database of 394 3A4 substrates show that SMARTCyp successfully identifies at least one metabolic site in the top two ranked positions 76% of the time.  SMARTCyp is available for download at http://www.farma.ku.dk/P 450.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1mh5MwR6547Vg90H21EOLACvtfcHk0lglRBWw80Ig2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFOqsL0%253D&md5=4761de933c2c3cef5da2ffb29ac076d0</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fml100016x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100016x%26sid%3Dliteratum%253Aachs%26aulast%3DRydberg%26aufirst%3DP.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26aulast%3DZaretzki%26aufirst%3DJ.%26aulast%3DBreneman%26aufirst%3DC.%26aulast%3DOlsen%26aufirst%3DL.%26atitle%3DSMARTCyp%253A%2520A%25202D%2520method%2520for%2520prediction%2520of%2520Cytochrome%2520P450-mediated%2520drug%2520metabolism%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D96%26epage%3D100%26doi%3D10.1021%2Fml100016x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneebeli, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">IDSite: an accurate approach to predict P450-mediated drug metabolism</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3829</span>– <span class="NLM_lpage">3845</span>, <span class="refDoi"> DOI: 10.1021/ct200462q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200462q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1CqtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=3829-3845&author=J.+Liauthor=S.+T.+Schneebeliauthor=J.+Bylundauthor=R.+Faridauthor=R.+A.+Friesner&title=IDSite%3A+an+accurate+approach+to+predict+P450-mediated+drug+metabolism&doi=10.1021%2Fct200462q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">IDSite: An Accurate Approach to Predict P450-Mediated Drug Metabolism</span></div><div class="casAuthors">Li, Jianing; Schneebeli, Severin T.; Bylund, Joseph; Farid, Ramy; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3829-3845</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate prediction of drug metab. is crucial for drug design.  Since a large majority of drugs' metab. involves P 450 enzymes, we herein describe a computational approach, IDSite, to predict P 450-mediated drug metab.  To model induced-fit effects, IDSite samples the conformational space with flexible docking in Glide followed by two refinement stages using the Protein Local Optimization Program (PLOP).  Sites of metab. (SOMs) are predicted according to a phys.-based score that evaluates the potential of atoms to react with the catalytic iron center.  As a preliminary test, we present in this paper the prediction of hydroxylation and O-dealkylation sites mediated by CYP2D6 using two different models: a phys.-based simulation model and a modification of this model in which a small no. of parameters are fit to a training set.  Without fitting any parameters to exptl. data, the phys. IDSite scoring recovers 83% of the exptl. observations for 56 compds. with a very low false pos. rate.  With only four fitted parameters, the fitted IDSite was trained with a subset of 36 compds. and successfully applied to the other 20 compds., recovering 94% of the exptl. observations with high sensitivity and specificity for both sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgmJGUujXp57Vg90H21EOLACvtfcHk0ljR0ZNmksOTOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1CqtbzP&md5=ab64a2c42b5c5d8bc46a7885c53e4eaf</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct200462q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200462q%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSchneebeli%26aufirst%3DS.%2BT.%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DIDSite%253A%2520an%2520accurate%2520approach%2520to%2520predict%2520P450-mediated%2520drug%2520metabolism%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D3829%26epage%3D3845%26doi%3D10.1021%2Fct200462q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T.
A.</span></span> <span> </span><span class="NLM_article-title">Targeted covalent inhibitors for drug design</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">13408</span>– <span class="NLM_lpage">13421</span>, <span class="refDoi"> DOI: 10.1002/anie.201601091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1002%2Fanie.201601091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=13408-13421&author=T.%0AA.+Baillie&title=Targeted+covalent+inhibitors+for+drug+design&doi=10.1002%2Fanie.201601091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Covalent Inhibitors for Drug Design</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">13408-13421</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In contrast to the traditional mechanism of drug action that relies on the reversible, noncovalent interaction of a ligand with its biol. target, a targeted covalent inhibitor (TCI) is designed such that the initial, reversible assocn. is followed by the formation of a covalent bond between an electrophile on the ligand and a nucleophilic center in the protein.  Although this approach offers a variety of potential benefits (high potency and extended duration of action), concerns over the possible toxicol. consequences of protein haptenization have hindered the development of the TCI concept.  Recently, approaches to mitigate the risk of serious adverse reactions to this new class of agent have emerged, thus stimulating interest in the field and leading to authorization of the first cadre of TCIs to be marketed.  The covalent inhibitor approach is rapidly gaining acceptance as a valuable tool in drug discovery, and is poised to make a major impact on the design of enzyme inhibitors and receptor modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEbjdFrctvNrVg90H21EOLACvtfcHk0ljR0ZNmksOTOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN&md5=18bed2ae08dcb4245a42b3070abc6354</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.201601091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201601091%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DTargeted%2520covalent%2520inhibitors%2520for%2520drug%2520design%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D13408%26epage%3D13421%26doi%3D10.1002%2Fanie.201601091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0ljR0ZNmksOTOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafer, C. M.</span></span> <span> </span><span class="NLM_article-title">Drug discovery considerations in the development of covalent inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.bmcl.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24314671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=33-39&author=R.+Mahauthor=J.+R.+Thomasauthor=C.+M.+Shafer&title=Drug+discovery+considerations+in+the+development+of+covalent+inhibitors&doi=10.1016%2Fj.bmcl.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery considerations in the development of covalent inhibitors</span></div><div class="casAuthors">Mah, Robert; Thomas, Jason R.; Shafer, Cynthia M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In recent years, the no. of drug candidates with a covalent mechanism of action progressing through clin. trials or being approved by the FDA has increased significantly.  And as interest in covalent inhibitors has increased, the tech. challenges for characterizing and optimizing these inhibitors have become evident.  A no. of new tools have been developed to aid this process, but these have not gained wide-spread use.  This review will highlight a no. of methods and tools useful for prosecuting covalent inhibitor drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7dY63LHbEhLVg90H21EOLACvtfcHk0ljIv-MNZypa7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsr3O&md5=9407bcb76e83065f5a56a97b63547ede</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DMah%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DShafer%26aufirst%3DC.%2BM.%26atitle%3DDrug%2520discovery%2520considerations%2520in%2520the%2520development%2520of%2520covalent%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D33%26epage%3D39%26doi%3D10.1016%2Fj.bmcl.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Cesco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufresne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittermaier, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors design and discovery</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.ejmech.2017.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28651155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=96-114&author=S.+De%0ACescoauthor=J.+Kurianauthor=C.+Dufresneauthor=A.+K.+Mittermaierauthor=N.+Moitessier&title=Covalent+inhibitors+design+and+discovery&doi=10.1016%2Fj.ejmech.2017.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors design and discovery</span></div><div class="casAuthors">De Cesco, Stephane; Kurian, Jerry; Dufresne, Caroline; Mittermaier, Anthony K.; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-114</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In the history of therapeutics, covalent drugs occupy a very distinct category.  While representing a significant fraction of the drugs on the market, very few have been deliberately designed to interact covalently with their biol. target.  In this review, the prevalence of covalent drugs will first be briefly covered, followed by an introduction to their mechanisms of action and more detailed discussions of their discovery and the development of safe and efficient covalent enzyme inhibitors.  All stages of a drug discovery program will be covered, from target considerations to lead optimization, strategies to tune reactivity and computational methods.  The goal of this article is to provide an overview of the field and to outline good practices that are needed for the proper assessment and development of covalent inhibitors as well as a good understanding of the potential and limitations of current computational methods for the design of covalent drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeLNDmLtBdB7Vg90H21EOLACvtfcHk0ljIv-MNZypa7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI&md5=551c9a9c4bd69d4ad4d9156c315d5a94</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BCesco%26aufirst%3DS.%26aulast%3DKurian%26aufirst%3DJ.%26aulast%3DDufresne%26aufirst%3DC.%26aulast%3DMittermaier%26aufirst%3DA.%2BK.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DCovalent%2520inhibitors%2520design%2520and%2520discovery%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D96%26epage%3D114%26doi%3D10.1016%2Fj.ejmech.2017.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamuya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBlanc, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehn, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberger, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">6-Substituted-4-(3-bromophenylamino)quinazolines as putative irreversible inhibitors of the epidermal growth factor receptor (EGFR) and human epidermal growth factor receptor (HER-2) tyrosine kinases with enhanced antitumor activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2734</span>, <span class="refDoi"> DOI: 10.1021/jm0005555</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0005555" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=2719-2734&author=H.-R.+Tsouauthor=N.+Mamuyaauthor=B.+D.+Johnsonauthor=M.+F.+Reichauthor=B.+C.+Gruberauthor=F.+Yeauthor=R.+Nilakantanauthor=R.+Shenauthor=C.+Discafaniauthor=R.+DeBlancauthor=R.+Davisauthor=F.+E.+Koehnauthor=L.+M.+Greenbergerauthor=Y.-F.+Wangauthor=A.+Wissner&title=6-Substituted-4-%283-bromophenylamino%29quinazolines+as+putative+irreversible+inhibitors+of+the+epidermal+growth+factor+receptor+%28EGFR%29+and+human+epidermal+growth+factor+receptor+%28HER-2%29+tyrosine+kinases+with+enhanced+antitumor+activity&doi=10.1021%2Fjm0005555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">6-Substituted-4-(3-bromophenylamino)quinazolines as Putative Irreversible Inhibitors of the Epidermal Growth Factor Receptor (EGFR) and Human Epidermal Growth Factor Receptor (HER-2) Tyrosine Kinases with Enhanced Antitumor Activity</span></div><div class="casAuthors">Tsou, Hwei-Ru; Mamuya, Nellie; Johnson, Bernard D.; Reich, Marvin F.; Gruber, Brian C.; Ye, Fei; Nilakantan, Ramaswamy; Shen, Ru; Discafani, Carolyn; DeBlanc, Ronald; Davis, Rachel; Koehn, Frank E.; Greenberger, Lee M.; Wang, Yu-Fen; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2719-2734</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 6-substituted-4-(3-bromophenylamino)quinazoline derivs. that may function as irreversible inhibitors of epidermal growth factor receptor (EGFR) and human epidermal growth factor receptor (HER-2) tyrosine kinases have been prepd.  These inhibitors have, at the C-6 position, butynamide, crotonamide, and methacrylamide Michael acceptors bearing water-solubilizing substituents.  These compds. were prepd. by acylation of 6-amino-4-(3-bromophenylamino)quinazoline with unsatd. acid chlorides or mixed anhydrides.  We show that attaching a basic functional group onto the Michael acceptor results in greater reactivity, due to intramol. catalysis of the Michael addn. and/or an inductive effect of the protonated basic group. This, along with improved water soly., results in compds. with enhanced biol. properties.  We present mol. modeling and exptl. evidence that these inhibitors interact covalently with the target enzymes.  One compd., (I) was shown to have excellent oral activity in a human epidermoid carcinoma (A431) xenograft model in nude mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVl-zt4kiGDrVg90H21EOLACvtfcHk0liNGw3HxaLafA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFChsb8%253D&md5=989e6e77195324d92e70ca13f532e1cf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm0005555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0005555%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DMamuya%26aufirst%3DN.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DGruber%26aufirst%3DB.%2BC.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DDeBlanc%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DKoehn%26aufirst%3DF.%2BE.%26aulast%3DGreenberger%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3D6-Substituted-4-%25283-bromophenylamino%2529quinazolines%2520as%2520putative%2520irreversible%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520and%2520human%2520epidermal%2520growth%2520factor%2520receptor%2520%2528HER-2%2529%2520tyrosine%2520kinases%2520with%2520enhanced%2520antitumor%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D2719%26epage%3D2734%26doi%3D10.1021%2Fjm0005555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furman, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutre, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burger, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wierda, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerema, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukbuntherng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clow, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedrick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buggy, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, S.</span></span> <span> </span><span class="NLM_article-title">Targeting BTK with ibrutinib in relapsed chronic lymphocytic leukemia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1215637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1056%2Fnejmoa1215637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23782158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFygu7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2013&pages=32-42&author=J.+C.+Byrdauthor=R.+R.+Furmanauthor=S.+E.+Coutreauthor=I.+W.+Flinnauthor=J.+A.+Burgerauthor=K.+A.+Blumauthor=B.+Grantauthor=J.+P.+Sharmanauthor=M.+Colemanauthor=W.+G.+Wierdaauthor=J.+A.+Jonesauthor=W.+Zhaoauthor=N.+A.+Heeremaauthor=A.+J.+Johnsonauthor=J.+Sukbuntherngauthor=B.+Y.+Changauthor=F.+Clowauthor=E.+Hedrickauthor=J.+J.+Buggyauthor=D.+F.+Jamesauthor=S.+O%E2%80%99Brien&title=Targeting+BTK+with+ibrutinib+in+relapsed+chronic+lymphocytic+leukemia&doi=10.1056%2Fnejmoa1215637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BTK with ibrutinib in relapsed chronic lymphocytic leukemia</span></div><div class="casAuthors">Byrd, John C.; Furman, Richard R.; Coutre, Steven E.; Flinn, Ian W.; Burger, Jan A.; Blum, Kristie A.; Grant, Barbara; Sharman, Jeff P.; Coleman, Morton; Wierda, William G.; Jones, Jeffrey A.; Zhao, Weiqiang; Heerema, Nyla A.; Johnson, Amy J.; Sukbuntherng, Juthamas; Chang, Betty Y.; Clow, Fong; Hedrick, Eric; Buggy, Joseph J.; James, Danelle F.; O'Brien, Susan</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-42</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background The treatment of relapsed chronic lymphocytic leukemia (CLL) has resulted in few durable remissions.  Bruton's tyrosine kinase (BTK), an essential component of B-cell-receptor signaling, mediates interactions with the tumor microenvironment and promotes the survival and proliferation of CLL cells.  Methods We conducted a phase 1b-2 multicenter study to assess the safety, efficacy, pharmacokinetics, and pharmacodynamics of ibrutinib (PCI-32765), a first-in-class, oral covalent inhibitor of BTK designed for treatment of B-cell cancers, in patients with relapsed or refractory CLL or small lymphocytic lymphoma.  A total of 85 patients, the majority of whom were considered to have high-risk disease, received ibrutinib orally once daily; 51 received 420 mg, and 34 received 840 mg.  Results Toxic effects were predominantly grade 1 or 2 and included transient diarrhea, fatigue, and upper respiratory tract infection; thus, patients could receive extended treatment with minimal hematol. toxic effects.  The overall response rate was the same in the group that received 420 mg and the group that received 840 mg (71%), and an addnl. 20% and 15% of patients in the resp. groups had a partial response with lymphocytosis.  The response was independent of clin. and genomic risk factors present before treatment, including advanced-stage disease, the no. of previous therapies, and the 17p13.1 deletion.  At 26 mo, the estd. progression-free survival rate was 75% and the rate of overall survival was 83%.  Conclusions Ibrutinib was assocd. with a high frequency of durable remissions in patients with relapsed or refractory CLL and small lymphocytic lymphoma, including patients with high-risk genetic lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1EhPQxMRz7rVg90H21EOLACvtfcHk0liNGw3HxaLafA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFygu7zN&md5=32a5eb6ab06beeecf64d9ada84139925</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1056%2Fnejmoa1215637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmoa1215637%26sid%3Dliteratum%253Aachs%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DFurman%26aufirst%3DR.%2BR.%26aulast%3DCoutre%26aufirst%3DS.%2BE.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DBurger%26aufirst%3DJ.%2BA.%26aulast%3DBlum%26aufirst%3DK.%2BA.%26aulast%3DGrant%26aufirst%3DB.%26aulast%3DSharman%26aufirst%3DJ.%2BP.%26aulast%3DColeman%26aufirst%3DM.%26aulast%3DWierda%26aufirst%3DW.%2BG.%26aulast%3DJones%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DHeerema%26aufirst%3DN.%2BA.%26aulast%3DJohnson%26aufirst%3DA.%2BJ.%26aulast%3DSukbuntherng%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DB.%2BY.%26aulast%3DClow%26aufirst%3DF.%26aulast%3DHedrick%26aufirst%3DE.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26aulast%3DJames%26aufirst%3DD.%2BF.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%26atitle%3DTargeting%2520BTK%2520with%2520ibrutinib%2520in%2520relapsed%2520chronic%2520lymphocytic%2520leukemia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D369%26spage%3D32%26epage%3D42%26doi%3D10.1056%2Fnejmoa1215637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span> <i>FDA Imbruvica Review</i>; <span class="NLM_publisher-name">U.S. Food&Drug
Administration</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+FDA+Imbruvica+Review%3B+U.S.+Food%26Drug%0AAdministration%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DFDA%2520Imbruvica%2520Review%26pub%3DU.S.%2520Food%2526Drug%250AAdministration%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dungo, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, G. M.</span></span> <span> </span><span class="NLM_article-title">Afatinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1007/s40265-013-0111-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1007%2Fs40265-013-0111-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23982599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1503-1515&author=R.+T.+Dungoauthor=G.+M.+Keating&title=Afatinib%3A+first+global+approval&doi=10.1007%2Fs40265-013-0111-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib: First Global Approval</span></div><div class="casAuthors">Dungo, Rosselle T.; Keating, Gillian M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1503-1515</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Afatinib, an irreversible inhibitor of the ErbB family of tyrosine kinases, is under development with Boehringer Ingelheim for the once-daily, oral treatment of cancer.  Afatinib downregulates ErbB signalling by covalently binding to epidermal growth factor receptor (EGFR), human epidermal growth factor receptor (HER) 2 and HER4, irreversibly inhibiting tyrosine kinase autophosphorylation.  It also inhibits transphosphorylation of HER3.  Oral afatinib (Gilotrif®) has been approved in the US for the first-line treatment of patients with metastatic non-small-cell lung cancer (NSCLC) who have tumors with EGFR exon 19 deletions or exon 21 (L858R) substitution mutations as detected by a US FDA-approved test.  Afatinib has also been approved in Taiwan for the first-line treatment of patients with EGFR mutation-pos. NSCLC.  In addn., the European Medicines Agency's Committee for Medicinal Products for Human Use has recommended the approval of afatinib (Giotrif) for the treatment of patients with locally advanced or metastatic NSCLC with activating EGFR mutations who are EGFR tyrosine kinase inhibitor naive.  Afatinib is also under regulatory review in Canada, Japan and other Asian countries.  This article summarizes the milestones in the development of afatinib, leading to this first approval in patients with metastatic NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdPcU5aIqrY7Vg90H21EOLACvtfcHk0ljyhgOpRZGwAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM&md5=21851cbfac56e2fffcba6c731f123cff</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0111-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0111-6%26sid%3Dliteratum%253Aachs%26aulast%3DDungo%26aufirst%3DR.%2BT.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1503%26epage%3D1515%26doi%3D10.1007%2Fs40265-013-0111-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strelevitz, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zientek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfield, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowty, M. E.</span></span> <span> </span><span class="NLM_article-title">Clearance prediction of targeted covalent inhibitors by <i>in vitro-in vivo</i> extrapolation of hepatic and extrahepatic clearance mechanisms</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1124/dmd.116.072983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1124%2Fdmd.116.072983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27784718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlSksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=1-7&author=L.+Leungauthor=X.+Yangauthor=T.+J.+Strelevitzauthor=J.+Montgomeryauthor=M.+F.+Brownauthor=M.+A.+Zientekauthor=C.+Banfieldauthor=A.+M.+Gilbertauthor=A.+Thorarensenauthor=M.+E.+Dowty&title=Clearance+prediction+of+targeted+covalent+inhibitors+by+in+vitro-in+vivo+extrapolation+of+hepatic+and+extrahepatic+clearance+mechanisms&doi=10.1124%2Fdmd.116.072983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Clearance prediction of targeted covalent inhibitors by in vitro-in vivo extrapolation of hepatic and extrahepatic clearance mechanisms</span></div><div class="casAuthors">Leung, Louis; Yang, Xin; Strelevitz, Timothy J.; Montgomery, Justin; Brown, Matthew F.; Zientek, Michael A.; Banfield, Christopher; Gilbert, Adam M.; Thorarensen, Atli; Dowty, Martin E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The concept of target-specific covalent enzyme inhibitors appears attractive from both an efficacy and a selectivity viewpoint considering the potential for enhanced biochem. efficiency assocd. with an irreversible mechanism.  Aside from potential safety concerns, clearance prediction of covalent inhibitors represents a unique challenge due to the inclusion of nontraditional metabolic pathways of direct conjugation with glutathione (GSH) or via GSH S-transferase-mediated processes.  In this article, a novel pharmacokinetic algorithm was developed using a series of Pfizer kinase selective acrylamide covalent inhibitors based on their in vitro-in vivo extrapolation of systemic clearance in rats.  The algorithmencompasses the use of hepatocytes as an in vitro model for hepatic clearance due to oxidative metab. and GSH conjugation, and the use of whole blood as an in vitro surrogate for GSH conjugation in extrahepatic tissues.  Initial evaluations with clin. covalent inhibitors suggested that the scaling algorithm developed from rats may also be useful for human clearance prediction when species-specific parameters, such as hepatocyte and blood stability and blood binding, were considered.  With careful consideration of clearance mechanisms, the described in vitro-in vivo extrapolation approach may be useful to facilitate candidate optimization, selection, and prediction of human pharmacokinetic clearance during the discovery and development of targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvIMBCrCxzNLVg90H21EOLACvtfcHk0ljyhgOpRZGwAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlSksL4%253D&md5=8c8b918b087278b2025e0ae56a7abe37</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.072983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.072983%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DStrelevitz%26aufirst%3DT.%2BJ.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DZientek%26aufirst%3DM.%2BA.%26aulast%3DBanfield%26aufirst%3DC.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DDowty%26aufirst%3DM.%2BE.%26atitle%3DClearance%2520prediction%2520of%2520targeted%2520covalent%2520inhibitors%2520by%2520in%2520vitro-in%2520vivo%2520extrapolation%2520of%2520hepatic%2520and%2520extrahepatic%2520clearance%2520mechanisms%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2017%26volume%3D45%26spage%3D1%26epage%3D7%26doi%3D10.1124%2Fdmd.116.072983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacolod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturedi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apuy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, A.</span></span> <span> </span><span class="NLM_article-title">A proposed screening paradigm for discovery of covalent inhibitor drugs</span>. <i>Drug Metab. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.2174/1872312808666140317151735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.2174%2F1872312808666140317151735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24628405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGqur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=19-30&author=M.+Moghaddamauthor=Y.+Tangauthor=Z.+O%E2%80%99Brienauthor=S.+Richardsonauthor=M.+Bacolodauthor=P.+Chaturediauthor=J.+Apuyauthor=A.+Kulkarni&title=A+proposed+screening+paradigm+for+discovery+of+covalent+inhibitor+drugs&doi=10.2174%2F1872312808666140317151735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A Proposed Screening Paradigm for Discovery of Covalent Inhibitor Drugs</span></div><div class="casAuthors">Moghaddam, Mehran F.; Tang, Yang; O'Brien, Zhihong; Richardson, Samantha J.; Bacolod, Maria; Chaturedi, Prasoon; Apuy, Julius; Kulkarni, Ashutosh</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-30</span>CODEN:
                <span class="NLM_cas:coden">DMLRBM</span>;
        ISSN:<span class="NLM_cas:issn">1872-3128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The in vitro and in vivo preclin. ADME properties of 10 clin. late stage or marketed covalent inhibitors were evaluated in order to define advancement criteria for discovery of future drugs in this arena.  Our studies revealed the following: After incubating with S9 fractions for 30 min, the rat and human in vitro stability for these compds. ranged from 1% to 100%.  The blood stability ranged from 30% to 100%.  There was a broad range of CYP inhibition with prevalence for time-dependent inhibition of at least one enzyme.  The Caco-2 permeability (A→B) ranged from negligible (0.6 x 10-6 cm/s) to highly permeable (31 x 10-6 cm/s) and the efflux ratio also varied widely (0.2-30).  Most of the compds. were highly protein bound in both rat and human with binding ≥ 90%.  Rat plasma clearance for the 10 compds. ranged from slow (11 mL/min/kg) to very rapid (350 mL/min/kg).  The Vss ranged from low (0.67 L/kg) to very high (115 L/kg).  MRT's also ranged from short (0.5 h) to long (7.4 h).  The oral exposures also showed a very broad range with Cmax's ranging from 0.01-77 μM and exposure levels ranging from 0.03-106 μM.hr.  In conclusion, the wide range in in vitro and in vivo ADME data makes these particular ADME assays non-discriminatory in the selection of promising compds.  In our opinion, non-traditional assays such as target mass modification, target confirmation by amino acid sequencing, cellular target occupancy, and target turnover rate data in combination with the pharmacokinetic profiles are the crit. considerations for progression of irreversible compds. in early discovery. ; <b>Biog.</b>: Mehran Moghaddam, PhD, is.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobrQbTwxt6f7Vg90H21EOLACvtfcHk0lhogXGDSYChHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGqur%252FF&md5=97ff5ba47e9547cbae3e5d10ec35112e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.2174%2F1872312808666140317151735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1872312808666140317151735%26sid%3Dliteratum%253Aachs%26aulast%3DMoghaddam%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DZ.%26aulast%3DRichardson%26aufirst%3DS.%26aulast%3DBacolod%26aufirst%3DM.%26aulast%3DChaturedi%26aufirst%3DP.%26aulast%3DApuy%26aufirst%3DJ.%26aulast%3DKulkarni%26aufirst%3DA.%26atitle%3DA%2520proposed%2520screening%2520paradigm%2520for%2520discovery%2520of%2520covalent%2520inhibitor%2520drugs%26jtitle%3DDrug%2520Metab.%2520Lett.%26date%3D2014%26volume%3D8%26spage%3D19%26epage%3D30%26doi%3D10.2174%2F1872312808666140317151735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J. M.</span></span> <span> </span><span class="NLM_article-title">A perspective on the kinetics of covalent and irreversible inhibition</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&author=J.+M.+Strelow&title=A+perspective+on+the+kinetics+of+covalent+and+irreversible+inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0lhogXGDSYChHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520perspective%2520on%2520the%2520kinetics%2520of%2520covalent%2520and%2520irreversible%2520inhibition%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venetsanakos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaStant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughhead, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, J. O.</span></span> <span> </span><span class="NLM_article-title">Discovery of the irreversible covalent FGFR inhibitor 8-(3-(4-Acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (PRN1371) for the Treatment of Solid Tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6516</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6516-6527&author=K.+A.+Brameldauthor=T.+D.+Owensauthor=E.+Vernerauthor=E.+Venetsanakosauthor=J.+M.+Bradshawauthor=V.+T.+Phanauthor=D.+Tamauthor=K.+Leungauthor=J.+Shuauthor=J.+LaStantauthor=D.+G.+Loughheadauthor=T.+Tonauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=D.+M.+Goldsteinauthor=J.+O.+Funk&title=Discovery+of+the+irreversible+covalent+FGFR+inhibitor+8-%283-%284-Acryloylpiperazin-1-yl%29propyl%29-6-%282%2C6-dichloro-3%2C5-dimethoxyphenyl%29-2-%28methylamino%29pyrido%5B2%2C3-d%5Dpyrimidin-7%288H%29-one+%28PRN1371%29+for+the+Treatment+of+Solid+Tumors&doi=10.1021%2Facs.jmedchem.7b00360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the Irreversible Covalent FGFR Inhibitor 8-(3-(4-Acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-d]pyrimidin-7(8H)-one (PRN1371) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Brameld, Ken A.; Owens, Timothy D.; Verner, Erik; Venetsanakos, Eleni; Bradshaw, J. Michael; Phan, Vernon T.; Tam, Danny; Leung, Kwan; Shu, Jin; LaStant, Jacob; Loughhead, David G.; Ton, Tony; Karr, Dane E.; Gerritsen, Mary E.; Goldstein, David M.; Funk, Jens Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6516-6527</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant signaling of the FGF/FGFR pathway occurs frequently in cancers and is an oncogenic driver in many solid tumors.  Clin. validation of FGFR as a therapeutic target has been demonstrated in bladder, liver, lung, breast, and gastric cancers.  Our goal was to develop an irreversible covalent inhibitor of FGFR1-4 for use in oncol. indications.  An irreversible covalent binding mechanism imparts many desirable pharmacol. benefits including high potency, selectivity, and prolonged target inhibition.  Herein we report the structure-based design, medicinal chem. optimization, and unique ADME assays of our irreversible covalent drug discovery program which culminated in the discovery of compd. 34 (PRN1371), a highly selective and potent FGFR1-4 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklTuvm1niy7Vg90H21EOLACvtfcHk0lgT3brod3HWRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK&md5=987069b645d5e0e36ccda5e5c81520b9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00360%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DVenetsanakos%26aufirst%3DE.%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DLaStant%26aufirst%3DJ.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26atitle%3DDiscovery%2520of%2520the%2520irreversible%2520covalent%2520FGFR%2520inhibitor%25208-%25283-%25284-Acryloylpiperazin-1-yl%2529propyl%2529-6-%25282%252C6-dichloro-3%252C5-dimethoxyphenyl%2529-2-%2528methylamino%2529pyrido%255B2%252C3-d%255Dpyrimidin-7%25288H%2529-one%2520%2528PRN1371%2529%2520for%2520the%2520Treatment%2520of%2520Solid%2520Tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6516%26epage%3D6527%26doi%3D10.1021%2Facs.jmedchem.7b00360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolanos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almaden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span> <span> </span><span class="NLM_article-title">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173-178&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+EGFR+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact ≤ 2.1 × 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0ljhoxdRWJt6Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520EGFR%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D173%26epage%3D178%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">KinMap: a web-based tool for interactive navigation through human kinome data</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">16</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-1433-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1186%2Fs12859-016-1433-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28056780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=16&author=S.+Eidauthor=S.+Turkauthor=A.+Volkamerauthor=F.+Rippmannauthor=S.+Fulle&title=KinMap%3A+a+web-based+tool+for+interactive+navigation+through+human+kinome+data&doi=10.1186%2Fs12859-016-1433-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">KinMap: a web-based tool for interactive navigation through human kinome data</span></div><div class="casAuthors">Eid, Sameh; Turk, Samo; Volkamer, Andrea; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16/1-16/6</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Annotations of the phylogenetic tree of the human kinome is an intuitive way to visualize compd. profiling data, structural features of kinases or functional relationships within this important class of proteins.  The increasing vol. and complexity of kinase-related data underlines the need for a tool that enables complex queries pertaining to kinase disease involvement and potential therapeutic uses of kinase inhibitors.  Results: Here, we present KinMap, a user-friendly online tool that facilitates the interactive navigation through kinase knowledge by linking biochem., structural, and disease assocn. data to the human kinome tree.  To this end, preprocessed data from freely-available sources, such as ChEMBL, the Protein Data Bank, and the Center for Therapeutic Target Validation platform are integrated into KinMap and can easily be complemented by proprietary data.  The value of KinMap will be exemplarily demonstrated for uncovering new therapeutic indications of known kinase inhibitors and for prioritizing kinases for drug development efforts.  Conclusions: KinMap represents a new generation of kinome tree viewers which facilitates interactive exploration of the human kinome.  KinMap enables generation of high-quality annotated images of the human kinome tree as well as exchange of kinome-related data in scientific communications.  Furthermore, KinMap supports multiple input and output formats and recognizes alternative kinase names and links them to a unified naming scheme, which makes it a useful tool across different disciplines and applications.  A web-service of KinMap is freely available at http://www.kinhub.org/kinmap/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1M2aa8fuKALVg90H21EOLACvtfcHk0ljhoxdRWJt6Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D&md5=29737c7cda8d9caf75018e0d29e9a459</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-1433-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-1433-7%26sid%3Dliteratum%253Aachs%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DKinMap%253A%2520a%2520web-based%2520tool%2520for%2520interactive%2520navigation%2520through%2520human%2520kinome%2520data%26jtitle%3DBMC%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D16%26doi%3D10.1186%2Fs12859-016-1433-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">January, C. T.</span></span> <span> </span><span class="NLM_article-title">Blockage of the hERG human cardiac K<sup>+</sup> channel by the gastrointestinal prokinetic agent cisapride</span>. <i>Am. J. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">H2534</span>– <span class="NLM_lpage">H2538</span>, <span class="refDoi"> DOI: 10.1152/ajpheart.1997.273.5.h2534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1152%2Fajpheart.1997.273.5.h2534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=9374794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1997&pages=H2534-H2538&author=S.+Mohammadauthor=Z.+Zhouauthor=Q.+Gongauthor=C.+T.+January&title=Blockage+of+the+hERG+human+cardiac+K%2B+channel+by+the+gastrointestinal+prokinetic+agent+cisapride&doi=10.1152%2Fajpheart.1997.273.5.h2534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Blockage of the HERG human cardiac K+ channel by the gastrointestinal prokinetic agent cisapride</span></div><div class="casAuthors">Mohammad, Saeed; Zhou, Zhengfeng; Gong, Qiuming; January, Craig T.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Physiology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5, Pt. 2</span>),
    <span class="NLM_cas:pages">H2534-H2538</span>CODEN:
                <span class="NLM_cas:coden">AJPHAP</span>;
        ISSN:<span class="NLM_cas:issn">0002-9513</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Cisapride, a gastrointestinal prokinetic agent, is known to caused long Q-T syndrome and ventricular arrhythmias.  The cellular mechanism is not known.  The human ether-a-go-go-related gene (HERG), which encodes the rapidly activating delayed rectifier K+ current and is important in cardiac repolarization, may serve as a target for the action of cisapride.  The authors tested the hypothesis that cisapride blocks HERG.  The whole cell path-clamp recording technique was used to study HERG channels stably expressed heterologously in HEK293 cells.  Under voltage-clamp conditions, cisapride block of HERG is dose dependent with a half-maximal inhibitory concn. of 6.5 nM at 22° (cells).  Currents rapidly recovered with drug washout.  The onset of block by cisapride required channel activation indicative of open or inactivated state blockage.  Block of HERG with cisapride after channel activation was voltage dependent.  At -20 mV, 10 nM cisapride reduced HERG tail-current amplitude by 5%, whereas, at +20 mV, the tail-current amplitude was reduced by 45% (cells).  At -20 and +20 mV, 100 nM cisapride reduced tail-current amplitude by 66 and 90%, resp.  The authors conclude that cisapride is a potent blocker of HERG channels expressed in HEK293 cells.  This effect may account for the clin. occurrence of Q-T prolongation and ventricular arrhythmias obsd. with cisapride.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNimFBPcs477Vg90H21EOLACvtfcHk0ljZGgVmUw_c5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D&md5=291fcfb4f2d9efd92120735a91d27b59</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1152%2Fajpheart.1997.273.5.h2534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpheart.1997.273.5.h2534%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DQ.%26aulast%3DJanuary%26aufirst%3DC.%2BT.%26atitle%3DBlockage%2520of%2520the%2520hERG%2520human%2520cardiac%2520K%252B%2520channel%2520by%2520the%2520gastrointestinal%2520prokinetic%2520agent%2520cisapride%26jtitle%3DAm.%2520J.%2520Physiol.%26date%3D1997%26volume%3D273%26spage%3DH2534%26epage%3DH2538%26doi%3D10.1152%2Fajpheart.1997.273.5.h2534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazic, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dota, C.</span></span> <span> </span><span class="NLM_article-title">An analysis of the relationship between preclinical and clinical QT interval-related data</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">101</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfx125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Ftoxsci%2Fkfx125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28903488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2017&pages=94-101&author=C.+E.+Pollardauthor=M.+Skinnerauthor=S.+E.+Lazicauthor=H.+M.+Priorauthor=K.+M.+Conlonauthor=J.-P.+Valentinauthor=C.+Dota&title=An+analysis+of+the+relationship+between+preclinical+and+clinical+QT+interval-related+data&doi=10.1093%2Ftoxsci%2Fkfx125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of the relationship between preclinical and clinical QT interval-related data</span></div><div class="casAuthors">Pollard, Christopher E.; Skinner, Matthew; Lazic, Stanley E.; Prior, Helen M.; Conlon, Kelly M.; Valentin, Jean-Pierre; Dota, Corina</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-101</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">There has been significant focus on drug-induced QT interval prolongation caused by block of the human ether-a-go-corelated gene (hERG)-encoded potassium channel.  Regulatory guidance has been implemented to assess QT interval prolongation risk: preclin. guidance requires a candidate drug potency as a hERG channel blocker to be defined and also its effect on QT interval in a non-rodent species; clin. guidance requires a "Thorough QT Study" during development, although some QT prolonging compds. are identified earlier via a Phase I study.  Clin., heart rate-cor. QT interval (QTc) data on 24 compds. (13 positives; 11 negatives) were compared with their effect on dog QTc and the concn. of compd. causing 50% inhibition (IC50) of hERG current.  Concordance was assessed by calcg. sensitivity and specificity across a range of decision thresholds, thus yielding receiver operating characteristic curves of sensitivity vs. (1-specificity).  The area under the curve of ROC curves (for which 0.5 and 1 indicate chance and perfect concordance, resp.) was used to summarize concordance.  Three aspects of preclin. data were compared with the clin. outcome (receiver operating characteristic area under the curve values shown in brackets): abs. hERG IC50 (0.78); safety margin between hERG IC50 and clin. peak free plasma exposure (0.80); safety margin between QTc effects in dogs and clin. peak free plasma exposure (0.81).  Pos. and neg. predictive values of abs. hERG IIC50 indicated that from an early drug discovery perspective, low potency compds. can be progressed on the basis of a low risk of causing a QTc increase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzer6VhKiwIbVg90H21EOLACvtfcHk0ljZGgVmUw_c5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D&md5=442a63490fb478512d52828a9d8d0a08</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfx125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfx125%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DC.%2BE.%26aulast%3DSkinner%26aufirst%3DM.%26aulast%3DLazic%26aufirst%3DS.%2BE.%26aulast%3DPrior%26aufirst%3DH.%2BM.%26aulast%3DConlon%26aufirst%3DK.%2BM.%26aulast%3DValentin%26aufirst%3DJ.-P.%26aulast%3DDota%26aufirst%3DC.%26atitle%3DAn%2520analysis%2520of%2520the%2520relationship%2520between%2520preclinical%2520and%2520clinical%2520QT%2520interval-related%2520data%26jtitle%3DToxicol.%2520Sci.%26date%3D2017%26volume%3D159%26spage%3D94%26epage%3D101%26doi%3D10.1093%2Ftoxsci%2Fkfx125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, B. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheelan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergquist, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becherer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashim, M. A.</span></span> <span> </span><span class="NLM_article-title">Predicting QT prolongation in humans during early drug development using hERG inhibition and an anaesthetized guinea-pig model</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>154</i></span>,  <span class="NLM_fpage">1446</span>– <span class="NLM_lpage">1456</span>, <span class="refDoi"> DOI: 10.1038/bjp.2008.267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fbjp.2008.267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=18587422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVajsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2008&pages=1446-1456&author=X.+Yaoauthor=D.+L.+Andersonauthor=S.+A.+Rossauthor=D.+G.+Langauthor=B.+Z.+Desaiauthor=D.+C.+Cooperauthor=P.+Wheelanauthor=M.+S.+McIntyreauthor=M.+L.+Bergquistauthor=K.+I.+MacKenzieauthor=J.+D.+Bechererauthor=M.+A.+Hashim&title=Predicting+QT+prolongation+in+humans+during+early+drug+development+using+hERG+inhibition+and+an+anaesthetized+guinea-pig+model&doi=10.1038%2Fbjp.2008.267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting QT prolongation in humans during early drug development using hERG inhibition and an anaesthetized guinea-pig model</span></div><div class="casAuthors">Yao, X.; Anderson, D. L.; Ross, S. A.; Lang, D. G.; Desai, B. Z.; Cooper, D. C.; Wheelan, P.; McIntyre, M. S.; Bergquist, M. L.; MacKenzie, K. I.; Becherer, J. D.; Hashim, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1446-1456</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug-induced prolongation of the QT interval can lead to torsade de pointes, a life-threatening ventricular arrhythmia.  Finding appropriate assays from among the plethora of options available to predict reliably this serious adverse effect in humans remains a challenging issue for the discovery and development of drugs.  The purpose of the present study was to develop and verify a reliable and relatively simple approach for assessing, during preclin. development, the propensity of drugs to prolong the QT interval in humans.  Sixteen marketed drugs from various pharmacol. classes with a known incidence-or lack thereof-of QT prolongation in humans were examd. in hERG (human ether a-go-go-related gene) patch-clamp assay and an anesthetized guinea-pig assay for QT prolongation using specific protocols.  Drug concns. in perfusates from hERG assays and plasma samples from guinea-pigs were detd. using liq. chromatog.-mass spectrometry.  Various pharmacol. agents that inhibit hERG currents prolong the QT interval in anesthetized guinea-pigs in a manner similar to that seen in humans and at comparable drug exposures.  Several compds. not assocd. with QT prolongation in humans failed to prolong the QT interval in this model.  Anal. of hERG inhibitory potency in conjunction with drug exposures and QT interval measurements in anesthetized guinea-pigs can reliably predict, during preclin. drug development, the risk of human QT prolongation.  A strategy is proposed for mitigating the risk of QT prolongation of new chem. entities during early lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRuiQ1CyjedrVg90H21EOLACvtfcHk0ljZGgVmUw_c5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVajsLw%253D&md5=1dbf4f50957b26496c4efa213e3f7e5c</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fbjp.2008.267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjp.2008.267%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DD.%2BL.%26aulast%3DRoss%26aufirst%3DS.%2BA.%26aulast%3DLang%26aufirst%3DD.%2BG.%26aulast%3DDesai%26aufirst%3DB.%2BZ.%26aulast%3DCooper%26aufirst%3DD.%2BC.%26aulast%3DWheelan%26aufirst%3DP.%26aulast%3DMcIntyre%26aufirst%3DM.%2BS.%26aulast%3DBergquist%26aufirst%3DM.%2BL.%26aulast%3DMacKenzie%26aufirst%3DK.%2BI.%26aulast%3DBecherer%26aufirst%3DJ.%2BD.%26aulast%3DHashim%26aufirst%3DM.%2BA.%26atitle%3DPredicting%2520QT%2520prolongation%2520in%2520humans%2520during%2520early%2520drug%2520development%2520using%2520hERG%2520inhibition%2520and%2520an%2520anaesthetized%2520guinea-pig%2520model%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2008%26volume%3D154%26spage%3D1446%26epage%3D1456%26doi%3D10.1038%2Fbjp.2008.267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ban, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBlanc, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennie, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherkasov, A.</span></span> <span> </span><span class="NLM_article-title">Best practices of computer-aided drug discovery: lessons learned from the development of a preclinical candidate for prostate cancer with a new mechanism of action</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1028</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00137</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00137" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXms1Wgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1018-1028&author=F.+Banauthor=K.+Dalalauthor=H.+Liauthor=E.+LeBlancauthor=P.+S.+Rennieauthor=A.+Cherkasov&title=Best+practices+of+computer-aided+drug+discovery%3A+lessons+learned+from+the+development+of+a+preclinical+candidate+for+prostate+cancer+with+a+new+mechanism+of+action&doi=10.1021%2Facs.jcim.7b00137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Best Practices of Computer-Aided Drug Discovery: Lessons Learned from the Development of a Preclinical Candidate for Prostate Cancer with a New Mechanism of Action</span></div><div class="casAuthors">Ban, Fuqiang; Dalal, Kush; Li, Huifang; LeBlanc, Eric; Rennie, Paul S.; Cherkasov, Artem</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1018-1028</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. drug design is a complex and iterative decision-making process relying on pre-existing knowledge and driven by exptl. data.  Low mol. wt. chems. represent an attractive therapeutic option as they are readily accessible to org. synthesis and can easily be characterized. T.  Their potency, as well as pharmacokinetic and pharmacodynamic properties can be systematically and rationally investigated, and ultimately optimized via expert science behind medicinal chem. and methods of computer-aided drug design (CADD).  In recent years, significant advances in mol. modeling techniques afforded a variety of tools to effectively identify potential binding pockets on prospective targets, to map key interactions between ligands and their binding sites, to construct and assess energetics of the resulting complexes, to predict ADMET properties of candidate compds., and to systematically analyze exptl. and computational data to derive meaningful structure-activity relationships leading to the creation of a drug candidate.  This perspective paper describes a real case of a drug discovery campaign accomplished in a relatively short time with limited resources.  The study integrated an arsenal of available mol. modeling techniques with an array of exptl. tools to successfully develop a novel class of potent and selective androgen receptor inhibitors with a novel mode of action.  It resulted in the largest academic licensing deal in Canadian history totaling $142M.  This project exemplifies the importance of team science, integrative approach to drug discovery, and the use of best practices in CADD.  We posit that lessons learned and best practices for executing an effective CADD project can be applied, with similar success, to many drug discover projects both in academia and industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL6gpFVHyq0rVg90H21EOLACvtfcHk0lgHmZa4QNbojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXms1Wgu78%253D&md5=2bfa1b93976eb0721aac6a422ec52d01</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00137%26sid%3Dliteratum%253Aachs%26aulast%3DBan%26aufirst%3DF.%26aulast%3DDalal%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLeBlanc%26aufirst%3DE.%26aulast%3DRennie%26aufirst%3DP.%2BS.%26aulast%3DCherkasov%26aufirst%3DA.%26atitle%3DBest%2520practices%2520of%2520computer-aided%2520drug%2520discovery%253A%2520lessons%2520learned%2520from%2520the%2520development%2520of%2520a%2520preclinical%2520candidate%2520for%2520prostate%2520cancer%2520with%2520a%2520new%2520mechanism%2520of%2520action%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1018%26epage%3D1028%26doi%3D10.1021%2Facs.jcim.7b00137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aryal, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dengler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguère, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Duan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherrer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of opioid analgesics with reduced side effects</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1038/nature19112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnature19112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27533032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2016&pages=185-190&author=A.+Manglikauthor=H.+Linauthor=D.+K.+Aryalauthor=J.+D.+McCorvyauthor=D.+Denglerauthor=G.+Corderauthor=A.+Levitauthor=R.+C.+Klingauthor=V.+Bernatauthor=H.+H%C3%BCbnerauthor=X.-P.+Huangauthor=M.+F.+Sassanoauthor=P.+M.+Gigu%C3%A8reauthor=S.+L%C3%B6berauthor=D.+Da+Duanauthor=G.+Scherrerauthor=B.+K.+Kobilkaauthor=P.+Gmeinerauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+opioid+analgesics+with+reduced+side+effects&doi=10.1038%2Fnature19112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of opioid analgesics with reduced side effects</span></div><div class="casAuthors">Manglik, Aashish; Lin, Henry; Aryal, Dipendra K.; McCorvy, John D.; Dengler, Daniela; Corder, Gregory; Levit, Anat; Kling, Ralf C.; Bernat, Viachaslau; Hubner, Harald; Huang, Xi-Ping; Sassano, Maria F.; Giguere, Patrick M.; Lober, Stefan; Da Duan; Scherrer, Gregory; Kobilka, Brian K.; Gmeiner, Peter; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue">7619</span>),
    <span class="NLM_cas:pages">185-190</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Morphine is an alkaloid from the opium poppy used to treat pain.  The potentially lethal side effects of morphine and related opioids-which include fatal respiratory depression-are thought to be mediated by μ-opioid-receptor (μOR) signalling through the β-arrestin pathway or by actions at other receptors.  Conversely, G-protein μOR signalling is thought to confer analgesia.  Here we computationally dock over 3 million mols. against the μOR structure and identify new scaffolds unrelated to known opioids.  Structure-based optimization yields PZM21-a potent Gi activator with exceptional selectivity for μOR and minimal β-arrestin-2 recruitment.  Unlike morphine, PZM21 is more efficacious for the affective component of analgesia vs. the reflexive component and is devoid of both respiratory depression and morphine-like reinforcing activity in mice at equi-analgesic doses.  PZM21 thus serves as both a probe to disentangle μOR signalling and a therapeutic lead that is devoid of many of the side effects of current opioids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiCff2w3U8iLVg90H21EOLACvtfcHk0lgHmZa4QNbojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI&md5=b8bb2d8575dadf4a6dc74d94c26d2c1a</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature19112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19112%26sid%3Dliteratum%253Aachs%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DAryal%26aufirst%3DD.%2BK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DDengler%26aufirst%3DD.%26aulast%3DCorder%26aufirst%3DG.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DGigu%25C3%25A8re%26aufirst%3DP.%2BM.%26aulast%3DL%25C3%25B6ber%26aufirst%3DS.%26aulast%3DDa%2BDuan%26aufirst%3DD.%26aulast%3DScherrer%26aufirst%3DG.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520opioid%2520analgesics%2520with%2520reduced%2520side%2520effects%26jtitle%3DNature%26date%3D2016%26volume%3D537%26spage%3D185%26epage%3D190%26doi%3D10.1038%2Fnature19112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capelli, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaLonde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semus, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedesco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peishoff, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, M. S.</span></span> <span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">5912</span>– <span class="NLM_lpage">5931</span>, <span class="refDoi"> DOI: 10.1021/jm050362n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.-M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions&doi=10.1021%2Fjm050362n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0lir6ZRGoAxWBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.-M.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931%26doi%3D10.1021%2Fjm050362n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span> <span> </span><span class="NLM_article-title">Docking covalent inhibitors: a parameter free approach to pose prediction and scoring</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1932</span>– <span class="NLM_lpage">1940</span>, <span class="refDoi"> DOI: 10.1021/ci500118s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500118s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1932-1940&author=K.+Zhuauthor=K.+W.+Borrelliauthor=J.+R.+Greenwoodauthor=T.+Dayauthor=R.+Abelauthor=R.+S.+Faridauthor=E.+Harder&title=Docking+covalent+inhibitors%3A+a+parameter+free+approach+to+pose+prediction+and+scoring&doi=10.1021%2Fci500118s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span></div><div class="casAuthors">Zhu, Kai; Borrelli, Kenneth W.; Greenwood, Jeremy R.; Day, Tyler; Abel, Robert; Farid, Ramy S.; Harder, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1932-1940</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although many popular docking programs include a facility to account for covalent ligands, large-scale systematic docking validation studies of covalent inhibitors have been sparse.  In this paper, we present the development and validation of a novel approach for docking and scoring covalent inhibitors, which consists of conventional noncovalent docking, heuristic formation of the covalent attachment point, and structural refinement of the protein-ligand complex.  This approach combines the strengths of the docking program Glide and the protein structure modeling program Prime and does not require any parameter fitting for the study of addnl. covalent reaction types.  We first test this method by predicting the native binding geometry of 38 covalently bound complexes.  The av. RMSD of the predicted poses is 1.52 Å, and 76% of test set inhibitors have an RMSD of less than 2.0 Å.  In addn., the apparent affinity score constructed herein is tested on a virtual screening study and the characterization of the SAR properties of two different series of congeneric compds. with satisfactory success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbtjEb3EqzQ7Vg90H21EOLACvtfcHk0lgLRX3njzebSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D&md5=3633c29fa5dc29817b178fd10a4c7b15</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci500118s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500118s%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DHarder%26aufirst%3DE.%26atitle%3DDocking%2520covalent%2520inhibitors%253A%2520a%2520parameter%2520free%2520approach%2520to%2520pose%2520prediction%2520and%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1932%26epage%3D1940%26doi%3D10.1021%2Fci500118s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek-Klumpers, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upeslacis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles as orally active, irreversible inhibitors of human epidermal growth factor receptor-2 kinase activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1021/jm040159c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040159c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1107-1131&author=H.-R.+Tsouauthor=E.+G.+Overbeek-Klumpersauthor=W.+A.+Hallettauthor=M.+F.+Reichauthor=M.+B.+Floydauthor=B.+D.+Johnsonauthor=R.+S.+Michalakauthor=R.+Nilakantanauthor=C.+Discafaniauthor=J.+Golasauthor=S.+K.+Rabindranauthor=R.+Shenauthor=X.+Shiauthor=Y.-F.+Wangauthor=J.+Upeslacisauthor=A.+Wissner&title=Optimization+of+6%2C7-disubstituted-4-%28arylamino%29quinoline-3-carbonitriles+as+orally+active%2C+irreversible+inhibitors+of+human+epidermal+growth+factor+receptor-2+kinase+activity&doi=10.1021%2Fjm040159c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 6,7-Disubstituted-4-(arylamino)quinoline-3-carbonitriles as Orally Active, Irreversible Inhibitors of Human Epidermal Growth Factor Receptor-2 Kinase Activity</span></div><div class="casAuthors">Tsou, Hwei-Ru; Overbeek-Klumpers, Elsebe G.; Hallett, William A.; Reich, Marvin F.; Floyd, M. Brawner; Johnson, Bernard D.; Michalak, Ronald S.; Nilakantan, Ramaswamy; Discafani, Carolyn; Golas, Jonathan; Rabindran, Sridhar K.; Shen, Ru; Shi, Xiaoqing; Wang, Yu-Fen; Upeslacis, Janis; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1107-1131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles, e.g. I (R1 = H, Cl; R2 = PhCH2O, 1-imidazolyl, 2-furylmethoxy, etc.; R3 = Cl, CN, PhCH2O; R4 = Me, Et; R5 = Me, R6 = Me, HOCH2CH2; R5R6N = azetidinyl, piperidinyl, thiomorpholinyl, etc.) that function as irreversible inhibitors of human epidermal growth factor receptor-2 (HER-2) and epidermal growth factor receptor (EGFR) kinases have been prepd.  These compds. demonstrated enhanced activities for inhibiting HER-2 kinase and the growth of HER-2 pos. cells compared to the EGFR kinase inhibitor I [R1 = H; R2 = F; R3 = Cl; R4 = Et; R5 = R6 = Me; (EKB-569)].  Three synthetic routes were used to prep. these compds.  They were prepd. mostly by acylation of 6-amino-4-(arylamino)quinoline-3-carbonitriles with unsatd. acid chlorides or by amination of 4-chloro-6-(crotonamido)quinoline-3-carbonitriles with monocyclic or bicyclic anilines.  The third route was developed to prep. a key intermediate, 6-acetamido-4-chloroquinoline-3-carbonitrile, that involved a safer cyclization step.  It was shown that attaching a large lipophilic group at the para position of the 4-(arylamino) ring results in improved potency for inhibiting HER-2 kinase.  The importance of a basic dialkylamino group at the end of the Michael acceptor for activity, due to intramol. catalysis of the Michael addn. has also been demonstrated.  This, along with improved water soly., resulted in compds. with enhanced biol. properties.  The mol. modeling results consistent with the proposed mechanism of inhibition are presented.  Binding studies of one compd., I [R1 = H; R2 = 2-pyridylmethoxy; R3 = Cl; R4 = Et; R5 = R6 = Me; (HKI-272)] (C-14 radiolabeled), showed that it binds irreversibly to HER-2 protein in BT474 cells.  Furthermore, it demonstrated excellent oral activity, esp. in HER-2 overexpressing xenografts.  Compd. HKI-272 was selected for further studies and is currently in phase I clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD0QL4-qQZCbVg90H21EOLACvtfcHk0lgLRX3njzebSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D&md5=55f77e41bf920f738ae2750e7ca0eb22</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm040159c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040159c%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DOverbeek-Klumpers%26aufirst%3DE.%2BG.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DMichalak%26aufirst%3DR.%2BS.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DGolas%26aufirst%3DJ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DUpeslacis%26aufirst%3DJ.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3DOptimization%2520of%25206%252C7-disubstituted-4-%2528arylamino%2529quinoline-3-carbonitriles%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520human%2520epidermal%2520growth%2520factor%2520receptor-2%2520kinase%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1107%26epage%3D1131%26doi%3D10.1021%2Fjm040159c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of a series of 5H-Pyrrolo[2,3-b]pyrazine FGFR kinase inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">698</span>, <span class="refDoi"> DOI: 10.3390/molecules23030698</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.3390%2Fmolecules23030698" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12gtLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=698&author=A.+Jiangauthor=Q.+Liuauthor=R.+Wangauthor=P.+Weiauthor=Y.+Daiauthor=X.+Wangauthor=Y.+Xuauthor=Y.+Maauthor=J.+Aiauthor=J.+Shenauthor=J.+Dingauthor=B.+Xiong&title=Structure-based+discovery+of+a+series+of+5H-Pyrrolo%5B2%2C3-b%5Dpyrazine+FGFR+kinase+inhibitors&doi=10.3390%2Fmolecules23030698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of a series of 5H-pyrrolo[2,3-b]pyrazine FGFR kinase inhibitors</span></div><div class="casAuthors">Jiang, Alan; Liu, Qiufeng; Wang, Ruifeng; Wei, Peng; Dai, Yang; Wang, Xin; Xu, Yechun; Ma, Yuchi; Ai, Jing; Shen, Jingkang; Ding, Jian; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">698/1-698/19</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFRs), a subfamily of receptor tyrosine kinases, are aberrant in various cancer types, and considered to be promising targets for cancer therapy.  We started with a weak-active compd. that was identified from our internal hepatocyte growth factor receptor (also called c-Met) inhibitor project, and optimized it with the guidance of a co-crystal structure of compd. 8 with FGFR1.  Through rational design, synthesis, and the biol. evaluation of a series of 5H-pyrrolo[2,3-b]pyrazine derivs., we discovered several potent FGFR kinase inhibitors.  Among them, compd. 13 displayed high selectivity and favorable metabolic properties, demonstrating a promising lead for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGeqBkLkHhrVg90H21EOLACvtfcHk0liJuxEmlYmpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12gtLbE&md5=98c0f1e1b49800f46e2c5c5fa439c35a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23030698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23030698%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DP.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520series%2520of%25205H-Pyrrolo%255B2%252C3-b%255Dpyrazine%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D698%26doi%3D10.3390%2Fmolecules23030698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicena, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">High yield bacterial expression of active c-Abl and c-Src tyrosine kinases</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3135</span>– <span class="NLM_lpage">3139</span>, <span class="refDoi"> DOI: 10.1110/ps.051750905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1110%2Fps.051750905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=16260764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlWlsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=3135-3139&author=M.+A.+Seeligerauthor=M.+Youngauthor=M.+N.+Hendersonauthor=P.+Pellicenaauthor=D.+S.+Kingauthor=A.+M.+Falickauthor=J.+Kuriyan&title=High+yield+bacterial+expression+of+active+c-Abl+and+c-Src+tyrosine+kinases&doi=10.1110%2Fps.051750905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">High yield bacterial expression of active c-Abl and c-Src tyrosine kinases</span></div><div class="casAuthors">Seeliger, Markus A.; Young, Matthew; Henderson, M. Nidanie; Pellicena, Patricia; King, David S.; Falick, Arnold M.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3135-3139</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The Abl and Src tyrosine kinases are key signaling proteins that are of considerable interest as drug targets in cancer and many other diseases.  The regulatory mechanisms that control the activity of these proteins are complex, and involve large-scale conformational changes in response to phosphorylation and other modulatory signals.  The success of the Abl inhibitor imatinib in the treatment of chronic myelogenous leukemia has shown the potential of kinase inhibitors, but the rise of drug resistance in patients has also shown that drugs with alternative modes of binding to the kinase are needed.  The detailed understanding of mechanisms of protein-drug interaction and drug resistance through biophys. methods demands a method for the prodn. of active protein on the milligram scale.  We have developed a bacterial expression system for the kinase domains of c-Abl and c-Src, which allows for the quick expression and purifn. of active wild-type and mutant kinase domains by coexpression with the YopH tyrosine phosphatase.  This method makes practical the use of isotopic labeling of c-Abl and c-Src for NMR studies, and is also applicable for constructs contg. the SH2 and SH3 domains of the kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis8a3O54lbVg90H21EOLACvtfcHk0liJuxEmlYmpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlWlsL3F&md5=62a85fa3b28dc95098dc3e2ec39690ff</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1110%2Fps.051750905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.051750905%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DM.%2BN.%26aulast%3DPellicena%26aufirst%3DP.%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DFalick%26aufirst%3DA.%2BM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DHigh%2520yield%2520bacterial%2520expression%2520of%2520active%2520c-Abl%2520and%2520c-Src%2520tyrosine%2520kinases%26jtitle%3DProtein%2520Sci.%26date%3D2005%26volume%3D14%26spage%3D3135%26epage%3D3139%26doi%3D10.1110%2Fps.051750905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg59wPdQOkGNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution--from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/s0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%2D%2Dfrom+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2Fs0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lg59wPdQOkGNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution--from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2Fs0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lg59wPdQOkGNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">DFG-out mode of inhibition by an irreversible type-1 inhibitor capable of overcoming gate-keeper mutations in FGF receptors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1021/cb500674s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500674s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=299-309&author=Z.+Huangauthor=L.+Tanauthor=H.+Wangauthor=Y.+Liuauthor=S.+Blaisauthor=J.+Dengauthor=T.+A.+Neubertauthor=N.+S.+Grayauthor=X.+Liauthor=M.+Mohammadi&title=DFG-out+mode+of+inhibition+by+an+irreversible+type-1+inhibitor+capable+of+overcoming+gate-keeper+mutations+in+FGF+receptors&doi=10.1021%2Fcb500674s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">DFG-out Mode of Inhibition by an Irreversible Type-1 Inhibitor Capable of Overcoming Gate-Keeper Mutations in FGF Receptors</span></div><div class="casAuthors">Huang, Zhifeng; Tan, Li; Wang, Huiyan; Liu, Yang; Blais, Steven; Deng, Jingjing; Neubert, Thomas A.; Gray, Nathanael S.; Li, Xiaokun; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">299-309</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-resistance acquisition through kinase gate-keeper mutations is a major hurdle in the clinic.  Here, we detd. the first crystal structures of the human FGFR4 kinase domain (FGFR4K) alone and complexed with ponatinib, a promiscuous type-2 (DFG-out) kinase inhibitor, and an oncogenic FGFR4K harboring the V550L gate-keeper mutation bound to FIIN-2, a new type-1 irreversible inhibitor.  Remarkably, like ponatinib, FIIN-2 also binds in the DFG-out mode despite lacking a functional group necessary to occupy the pocket vacated upon the DFG-out flip.  Structural anal. reveals that the covalent bond between FIIN-2 and a cysteine, uniquely present in the glycine-rich loop of FGFR kinases, facilitates the DFG-out conformation, which together with the internal flexibility of FIIN-2 enables FIIN-2 to avoid the steric clash with the gate-keeper mutation that causes the ponatinib resistance.  The structural data provide a blueprint for the development of next generation anticancer inhibitors through combining the salient inhibitory mechanisms of ponatinib and FIIN-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf97N01ZUSpbVg90H21EOLACvtfcHk0lgxBNkF75gWYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM&md5=0c741cc5492596f9ea9323e4d45e274b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fcb500674s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500674s%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBlais%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DDFG-out%2520mode%2520of%2520inhibition%2520by%2520an%2520irreversible%2520type-1%2520inhibitor%2520capable%2520of%2520overcoming%2520gate-keeper%2520mutations%2520in%2520FGF%2520receptors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D299%26epage%3D309%26doi%3D10.1021%2Fcb500674s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgxBNkF75gWYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lg3VL5EbEvOSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of <i>in vitro</i> half-life approach and nonspecific binding to microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+an+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0lg3VL5EbEvOSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520an%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agababa, D.</span></span> <span> </span><span class="NLM_article-title">Prediction of hepatic microsomal intrinsic clearance and human clearance values for drugs</span>. <i>J. Mol. Graph. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2009.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.jmgm.2009.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19713138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WrtbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=245-252&author=K.+Nikolicauthor=D.+Agababa&title=Prediction+of+hepatic+microsomal+intrinsic+clearance+and+human+clearance+values+for+drugs&doi=10.1016%2Fj.jmgm.2009.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of hepatic microsomal intrinsic clearance and human clearance values for drugs</span></div><div class="casAuthors">Nikolic, Katarina; Agababa, Danica</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-252</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Twenty-nine drugs of different structures were used in theor. QSAR anal. of human hepatic microsomal intrinsic clearance (in vitro T 1/2 and in vitro C L ' int ) and whole body clearance (CLblood).  The examd. compds. demonstrated a wide range of scaled intrinsic clearance values.  Constitutional, geometrical, physico-chem. and electronic descriptors were computed for the examd. structures by use of the Marvin Sketch 5.1.32, the Chem3D Ultra 7.0.0 and the Dragon 5.4 program.  Partial least squares regression (PLSR), has been applied for selection of the most relevant mol. descriptors and development of quant. structure-activity relationship (QSAR) model for human hepatic microsomal intrinsic clearance (in vitro T 1/2).  Optimal QSAR models with nine and ten variables, R 2 > 0.808 and cross-validation parameter q p r e 2 > 0.623 , were selected and compared.  Since the microsomal in vitro T 1/2 data can be used for calcn. of in vitro C L ' int and in vivo CLblood, the developed QSAR model will enable one to analyze the kinetics of cytochrome P 450-mediated reactions in term of intrinsic clearance and whole body clearance.  A comparison is made between predictions produced from the QSAR anal. and exptl. data, and there appears to be generally satisfactory correlations with the literature values for intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcONn29BFARbVg90H21EOLACvtfcHk0lj1cNJ9pcfHUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WrtbjE&md5=80f9d2bef510e20f3fb7738238d18c06</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2009.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2009.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DNikolic%26aufirst%3DK.%26aulast%3DAgababa%26aufirst%3DD.%26atitle%3DPrediction%2520of%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520and%2520human%2520clearance%2520values%2520for%2520drugs%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model.%26date%3D2009%26volume%3D28%26spage%3D245%26epage%3D252%26doi%3D10.1016%2Fj.jmgm.2009.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yujiao Wei, Yanting Tang, Yunyun Zhou, Yuyu Yang, Yetong Cui, Xuan Wang, Yubo Wang, Yulin Liu, Ning Liu, Qianqian Wang, Chong Li, Hao Ruan, Honggang Zhou, Mingming Wei, Guang Yang, <span class="NLM_string-name hlFld-ContribAuthor">Cheng Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of a Novel 2H-Pyrazolo[3,4-d]pyrimidine Derivative as a Potent Irreversible Pan-Fibroblast Growth Factor Receptor Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9078-9099. <a href="https://doi.org/10.1021/acs.jmedchem.1c00174" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00174%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252Ba%252BNovel%252B2H-Pyrazolo%25255B3%25252C4-d%25255Dpyrimidine%252BDerivative%252Bas%252Ba%252BPotent%252BIrreversible%252BPan-Fibroblast%252BGrowth%252BFactor%252BReceptor%252BInhibitor%26aulast%3DWei%26aufirst%3DYujiao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D31012021%26date%3D15062021%26volume%3D64%26issue%3D13%26spage%3D9078%26epage%3D9099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peng Zhang, Zhihui Min, Yang Gao, Jiang Bian, Xin Lin, Jie He, Deyong Ye, Yilin Li, Chao Peng, Yunfeng Cheng, <span class="NLM_string-name hlFld-ContribAuthor">Yong Chu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Benzothiazepinones as Irreversible Covalent Glycogen Synthase Kinase 3β Inhibitors for the Treatment of Acute Promyelocytic Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7341-7358. <a href="https://doi.org/10.1021/acs.jmedchem.0c02254" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02254%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BBenzothiazepinones%252Bas%252BIrreversible%252BCovalent%252BGlycogen%252BSynthase%252BKinase%252B3%2525CE%2525B2%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BAcute%252BPromyelocytic%252BLeukemia%26aulast%3DZhang%26aufirst%3DPeng%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29122020%26date%3D24052021%26volume%3D64%26issue%3D11%26spage%3D7341%26epage%3D7358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingping  Su</span>, <span class="hlFld-ContribAuthor ">Zhihao  Liu</span>, <span class="hlFld-ContribAuthor ">Lin  Yue</span>, <span class="hlFld-ContribAuthor ">Xiuli  Wu</span>, <span class="hlFld-ContribAuthor ">Wei  Wei</span>, <span class="hlFld-ContribAuthor ">Hanyun  Que</span>, <span class="hlFld-ContribAuthor ">Tinghong  Ye</span>, <span class="hlFld-ContribAuthor ">Yi  Luo</span>, <span class="hlFld-ContribAuthor ">Yiwen  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of 1
              H
              -pyrrolo[2,3-
              b
              ]pyridine derivatives as potent fibroblast growth factor receptor inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (34)
                                     , 20651-20661. <a href="https://doi.org/10.1039/D1RA02660G" title="DOI URL">https://doi.org/10.1039/D1RA02660G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1RA02660G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1RA02660G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252B1%252BH%252B-pyrrolo%25255B2%25252C3-%252Bb%252B%25255Dpyridine%252Bderivatives%252Bas%252Bpotent%252Bfibroblast%252Bgrowth%252Bfactor%252Breceptor%252Binhibitors%26aulast%3DSu%26aufirst%3DXingping%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D34%26spage%3D20651%26epage%3D20661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Zhongjie  Liang</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Drug design targeting active posttranslational modification protein isoforms. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2021,</strong> <em>41 </em>
                                    (3)
                                     , 1701-1750. <a href="https://doi.org/10.1002/med.21774" title="DOI URL">https://doi.org/10.1002/med.21774</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21774%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DDrug%252Bdesign%252Btargeting%252Bactive%252Bposttranslational%252Bmodification%252Bprotein%252Bisoforms%26aulast%3DMeng%26aufirst%3DFanwang%26date%3D2021%26date%3D2020%26volume%3D41%26issue%3D3%26spage%3D1701%26epage%3D1750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaowei  Wu</span>, <span class="hlFld-ContribAuthor ">Mengdi  Dai</span>, <span class="hlFld-ContribAuthor ">Rongrong  Cui</span>, <span class="hlFld-ContribAuthor ">Yulan  Wang</span>, <span class="hlFld-ContribAuthor ">Chunpu  Li</span>, <span class="hlFld-ContribAuthor ">Xia  Peng</span>, <span class="hlFld-ContribAuthor ">Jihui  Zhao</span>, <span class="hlFld-ContribAuthor ">Bao  Wang</span>, <span class="hlFld-ContribAuthor ">Yang  Dai</span>, <span class="hlFld-ContribAuthor ">Dan  Feng</span>, <span class="hlFld-ContribAuthor ">Tianbiao  Yang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Meiyu  Geng</span>, <span class="hlFld-ContribAuthor ">Jing  Ai</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of pyrazolo[3,4-d]pyridazinone derivatives as covalent FGFR inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (3)
                                     , 781-794. <a href="https://doi.org/10.1016/j.apsb.2020.09.002" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.09.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.09.002%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bpyrazolo%25255B3%25252C4-d%25255Dpyridazinone%252Bderivatives%252Bas%252Bcovalent%252BFGFR%252Binhibitors%26aulast%3DWu%26aufirst%3DXiaowei%26date%3D2021%26volume%3D11%26issue%3D3%26spage%3D781%26epage%3D794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oya Unsal  Tan</span>, <span class="hlFld-ContribAuthor ">Keriman  Ozadali Sari</span>. </span><span class="cited-content_cbyCitation_article-title">Azolo[
              d
              ]pyridazinones in medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (3)
                                     , 287-311. <a href="https://doi.org/10.4155/fmc-2020-0234" title="DOI URL">https://doi.org/10.4155/fmc-2020-0234</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0234%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAzolo%25255B%252Bd%252B%25255Dpyridazinones%252Bin%252Bmedicinal%252Bchemistry%26aulast%3DTan%26aufirst%3DOya%2BUnsal%26date%3D2021%26volume%3D13%26issue%3D3%26spage%3D287%26epage%3D311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiaomei  Jin</span>, <span class="hlFld-ContribAuthor ">Dongjian  Zhang</span>, <span class="hlFld-ContribAuthor ">Meng  Gao</span>, <span class="hlFld-ContribAuthor ">Cuihua  Jiang</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrrolo[2,3-b]pyridine-3-one derivatives as novel fibroblast growth factor receptor 4 inhibitors for the treatment of hepatocellular carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>29 </em>, 115862. <a href="https://doi.org/10.1016/j.bmc.2020.115862" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115862</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115862%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DPyrrolo%25255B2%25252C3-b%25255Dpyridine-3-one%252Bderivatives%252Bas%252Bnovel%252Bfibroblast%252Bgrowth%252Bfactor%252Breceptor%252B4%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Bhepatocellular%252Bcarcinoma%26aulast%3DJin%26aufirst%3DQiaomei%26date%3D2021%26volume%3D29%26spage%3D115862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhang-Xu  He</span>, <span class="hlFld-ContribAuthor ">Yun-Peng  Gong</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Pyridazine as a privileged structure: An updated review on anticancer activity of pyridazine containing bioactive molecules. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112946. <a href="https://doi.org/10.1016/j.ejmech.2020.112946" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112946</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112946%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPyridazine%252Bas%252Ba%252Bprivileged%252Bstructure%25253A%252BAn%252Bupdated%252Breview%252Bon%252Banticancer%252Bactivity%252Bof%252Bpyridazine%252Bcontaining%252Bbioactive%252Bmolecules%26aulast%3DHe%26aufirst%3DZhang-Xu%26date%3D2021%26volume%3D209%26spage%3D112946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antti  Poso</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Modeling of Protein Kinases: Current Status and Challenges. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 25-41. <a href="https://doi.org/10.1007/7355_2020_106" title="DOI URL">https://doi.org/10.1007/7355_2020_106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2020_106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2020_106%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BModeling%252Bof%252BProtein%252BKinases%25253A%252BCurrent%252BStatus%252Band%252BChallenges%26aulast%3DPoso%26aufirst%3DAntti%26date%3D2020%26date%3D2020%26spage%3D25%26epage%3D41%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DProteinkinase%252BInhibitors%26aulast%3DLaufer%26aufirst%3DStefan%26date%3D2021%26volume%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Marseglia</span>, <span class="hlFld-ContribAuthor ">Alessio  Lodola</span>, <span class="hlFld-ContribAuthor ">Marco  Mor</span>, <span class="hlFld-ContribAuthor ">Riccardo  Castelli</span>. </span><span class="cited-content_cbyCitation_article-title">Fibroblast growth factor receptor inhibitors: patent review (2015–2019). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2019,</strong> <em>29 </em>
                                    (12)
                                     , 965-977. <a href="https://doi.org/10.1080/13543776.2019.1688300" title="DOI URL">https://doi.org/10.1080/13543776.2019.1688300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2019.1688300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2019.1688300%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DFibroblast%252Bgrowth%252Bfactor%252Breceptor%252Binhibitors%25253A%252Bpatent%252Breview%252B%2525282015%2525E2%252580%2525932019%252529%26aulast%3DMarseglia%26aufirst%3DGiuseppe%26date%3D2019%26date%3D2019%26volume%3D29%26issue%3D12%26spage%3D965%26epage%3D977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. FGFR-selective inhibitors in clinical development.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Workflow of RTKscore development and comparison of RTKscore with Glide on a test set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Structures of compounds <b>8</b>, <b>9</b>, <b>10</b>, and <b>11</b>; (B) putative binding mode of compound <b>8</b> with the FGFR1(PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>); (C) active site pocket of FGFR1 depicted as the mesh surface. (D) putative binding modes of compound <b>10</b> (blue) and compound <b>11</b> (orange) with FGFR1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0">3TT0</a>) simulated by covalent docking. (E) FGFR1, FGFR2, FGFR3, and FGFR4 P-loop sequence alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of compound <b>10</b> on the phosphorylation of the FGFR and the downstream effector Erk in the KG1 and SNU16 cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Cocrystal structure (shown in green) of compound <b>9</b> and the FGFR1 complex (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ">6ITJ</a>) and the putative binding mode (shown in pink) of compound <b>9</b> and FGFR1 simulated by molecular docking. (B) Crystal structures of the complex of compound <b>10</b> (shown in magenta) with FGFR4 (shown in salmon) (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP">6IUP</a>) and the complex of compound <b>11</b> (shown in lemon) with FGFR4 (shown in cyan) (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO">6IUO</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. SOM prediction of compound <b>10</b> by three programs. SMARTCyp (orange circles, metabolism probability ordered by the size of the circle), SOME (blue triangles) and SOME-UGT (grey diamonds), and Schrödinger (green pentagons, metabolism probability ordered by the size).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) GSH affinity evaluation of compounds <b>10</b>, <b>13</b>, <b>15</b>, and <b>16</b>; (B) blood stability evaluation of compounds <b>10</b> and <b>15</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound <b>10</b> (A) and <b>15</b> (B) irreversibly binds to FGFR1, compared with compound <b>1</b> (C) that is a known reversible FGFR inhibitor. Enzyme activity of FGFR1 was assayed by a Caliper EZ Reader under three different conditions: without the enzyme (background), without the compound (nonpre-incubation control), and pre-incubated with the compound. “Conversion” here represents the enzyme activity and means “the percent of conversion of the substrate peptide”.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Kinase profile of compound <b>15</b> drawn with KinMap.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Kinase inhibition ratios above 50% at corresponding concentrations are marked as red circles.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/medium/jm9b00510_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00510/20190815/images/large/jm9b00510_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00510&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl bromoacetate, K<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C; (b) CH<sub>3</sub>CN, NaH, THF, 50 °C; (c) ethyl 2-chloroacetoacetate, NaNO<sub>2</sub>, NaOAc, EtOH, H<sub>2</sub>O/HCl (3:1, v/v), 0 °C-rt; (d) Et<sub>3</sub>N, DCM, rt; (e) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, HCl (conc.), EtOH, 100 °C, MW; (f) for compounds <b>23a–c</b>, Fe, HCl (conc.), MeOH/H<sub>2</sub>O (6:1, v/v), 80 °C, 1 h; (g) acryloyl chloride or acryloyl chloride amino derivatives, K<sub>2</sub>CO<sub>3</sub>, THF, rt; (h) for compound <b>23d</b>, Pd(OH)<sub>2</sub>/C, HCl (conc.), MeOH, 40 °C, 8 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12874" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12874" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 71 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiong, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, C.-O.</span></span> <span> </span><span class="NLM_article-title">Functional roles of fibroblast growth factor receptors (FGFRs) signaling in human cancers</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1447</span>– <span class="NLM_lpage">1468</span>, <span class="refDoi"> DOI: 10.1007/s10495-013-0886-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1007%2Fs10495-013-0886-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23900974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslOkur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1447-1468&author=K.+H.+Tiongauthor=L.+Y.+Mahauthor=C.-O.+Leong&title=Functional+roles+of+fibroblast+growth+factor+receptors+%28FGFRs%29+signaling+in+human+cancers&doi=10.1007%2Fs10495-013-0886-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Functional roles of fibroblast growth factor receptors (FGFRs) signaling in human cancers</span></div><div class="casAuthors">Tiong, Kai Hung; Mah, Li Yen; Leong, Chee-Onn</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1447-1468</span>CODEN:
                <span class="NLM_cas:coden">APOPFN</span>;
        ISSN:<span class="NLM_cas:issn">1360-8185</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The fibroblast growth factor receptors (FGFRs) regulate important biol. processes including cell proliferation and differentiation during development and tissue repair.  Over the past decades, numerous pathol. conditions and developmental syndromes have emerged as a consequence of deregulation in the FGFRs signaling network.  This review aims to provide an overview of FGFR family, their complex signaling pathways in tumorigenesis, and the current development and application of therapeutics targeting the FGFRs signaling for treatment of refractory human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4FLwi8M_8RLVg90H21EOLACvtfcHk0lhBZtkbJSPRTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslOkur7J&md5=3e1180c8cdfc5903fa11f0a9dfeab904</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs10495-013-0886-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-013-0886-7%26sid%3Dliteratum%253Aachs%26aulast%3DTiong%26aufirst%3DK.%2BH.%26aulast%3DMah%26aufirst%3DL.%2BY.%26aulast%3DLeong%26aufirst%3DC.-O.%26atitle%3DFunctional%2520roles%2520of%2520fibroblast%2520growth%2520factor%2520receptors%2520%2528FGFRs%2529%2520signaling%2520in%2520human%2520cancers%26jtitle%3DApoptosis%26date%3D2013%26volume%3D18%26spage%3D1447%26epage%3D1468%26doi%3D10.1007%2Fs10495-013-0886-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thisse, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thisse, C.</span></span> <span> </span><span class="NLM_article-title">Functions and regulations of fibroblast growth factor signaling during embryonic development</span>. <i>Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">390</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1016/j.ydbio.2005.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.ydbio.2005.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=16216232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1GhtL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2005&pages=390-402&author=B.+Thisseauthor=C.+Thisse&title=Functions+and+regulations+of+fibroblast+growth+factor+signaling+during+embryonic+development&doi=10.1016%2Fj.ydbio.2005.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Functions and regulations of fibroblast growth factor signaling during embryonic development</span></div><div class="casAuthors">Thisse, Bernard; Thisse, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Biology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">390-402</span>CODEN:
                <span class="NLM_cas:coden">DEBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0012-1606</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fibroblast growth factors (FGF) are secreted mols. which function through the activation of specific tyrosine kinases receptors, the FGF receptors that transduce the signal by activating different pathways including the Ras/MAP kinase and the phospholipase-Cγ pathways.  FGFs are involved in the regulation of many developmental processes including patterning, morphogenesis, differentiation, cell proliferation, or migration.  Such a diverse set of activities requires a tight control of the transduction signal which is achieved through the induction of different feedback inhibitors such as the Sproutys, Sef, and MAP kinase phosphatase 3 which are responsible for the attenuation of FGF signals, limiting FGF activities in time and space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqQ6hbl6aVdbVg90H21EOLACvtfcHk0ljL7QRY66kk2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1GhtL%252FF&md5=7a2a1199001aff1a05836d6ff19f20ee</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ydbio.2005.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ydbio.2005.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DThisse%26aufirst%3DB.%26aulast%3DThisse%26aufirst%3DC.%26atitle%3DFunctions%2520and%2520regulations%2520of%2520fibroblast%2520growth%2520factor%2520signaling%2520during%2520embryonic%2520development%26jtitle%3DDev.%2520Biol.%26date%3D2005%26volume%3D287%26spage%3D390%26epage%3D402%26doi%3D10.1016%2Fj.ydbio.2005.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dienstmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Garcia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iafrate, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuciforo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span> <span> </span><span class="NLM_article-title">Genomic aberrations in the FGFR pathway: opportunities for targeted therapies in solid tumors</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdt419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdt419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24265351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BC2c7ptFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=552-563&author=R.+Dienstmannauthor=J.+Rodonauthor=A.+Pratauthor=J.+Perez-Garciaauthor=B.+Adamoauthor=E.+Felipauthor=J.+Cortesauthor=A.+J.+Iafrateauthor=P.+Nuciforoauthor=J.+Tabernero&title=Genomic+aberrations+in+the+FGFR+pathway%3A+opportunities+for+targeted+therapies+in+solid+tumors&doi=10.1093%2Fannonc%2Fmdt419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic aberrations in the FGFR pathway: opportunities for targeted therapies in solid tumors</span></div><div class="casAuthors">Dienstmann R; Rodon J; Prat A; Perez-Garcia J; Adamo B; Felip E; Cortes J; Iafrate A J; Nuciforo P; Tabernero J</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">552-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The fibroblast growth factor receptor (FGFR) cascade plays crucial roles in tumor cell proliferation, angiogenesis, migration and survival.  Accumulating evidence suggests that in some tumor types, FGFRs are bona fide oncogenes to which cancer cells are addicted.  Because FGFR inhibition can reduce proliferation and induce cell death in a variety of in vitro and in vivo tumor models harboring FGFR aberrations, a growing number of research groups have selected FGFRs as targets for anticancer drug development.  Multikinase FGFR/vascular endothelial growth factor receptor (VEGFR) inhibitors have shown promising activity in breast cancer patients with FGFR1 and/or FGF3 amplification.  Early clinical trials with selective FGFR inhibitors, which may overcome the toxicity constraints raised by multitarget kinase inhibition, are recruiting patients with known FGFR(1-4) status based on genomic screens.  Preliminary signs of antitumor activity have been demonstrated in some tumor types, including squamous cell lung carcinomas.  Rational combination of targeted therapies is expected to further increase the efficacy of selective FGFR inhibitors.  Herein, we discuss unsolved questions in the clinical development of these agents and suggest guidelines for management of hyperphosphatemia, a class-specific mechanism-based toxicity.  In addition, we propose standardized definitions for FGFR1 and FGFR2 gene amplification based on in situ hybridization methods.  Extended access to next-generation sequencing platforms will facilitate the identification of diseases in which somatic FGFR(1-4) mutations, amplifications and fusions are potentially driving cancer cell viability, further strengthening the role of FGFR signaling in cancer biology and providing more possibilities for the therapeutic application of FGFR inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqiWGpXuCcvmx8TEN9K18QfW6udTcc2ebW5MiGlz5ZgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7ptFWmsA%253D%253D&md5=633eb08ba5e38f239a73dfbfff04064c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdt419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdt419%26sid%3Dliteratum%253Aachs%26aulast%3DDienstmann%26aufirst%3DR.%26aulast%3DRodon%26aufirst%3DJ.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DPerez-Garcia%26aufirst%3DJ.%26aulast%3DAdamo%26aufirst%3DB.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DNuciforo%26aufirst%3DP.%26aulast%3DTabernero%26aufirst%3DJ.%26atitle%3DGenomic%2520aberrations%2520in%2520the%2520FGFR%2520pathway%253A%2520opportunities%2520for%2520targeted%2520therapies%2520in%2520solid%2520tumors%26jtitle%3DAnn.%2520Oncol.%26date%3D2014%26volume%3D25%26spage%3D552%26epage%3D563%26doi%3D10.1093%2Fannonc%2Fmdt419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araújo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reclusa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franchina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Steen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Dam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naing, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfo, C.</span></span> <span> </span><span class="NLM_article-title">FGFR a promising druggable target in cancer: molecular biology and new drugs</span>. <i>Crit. Rev. Oncol. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2017.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.critrevonc.2017.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28427515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BC1cvps1ajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2017&pages=256-267&author=R.+Portaauthor=R.+Boreaauthor=A.+Coelhoauthor=S.+Khanauthor=A.+Ara%C3%BAjoauthor=P.+Reclusaauthor=T.+Franchinaauthor=N.+Van+Der+Steenauthor=P.+Van+Damauthor=J.+Ferriauthor=R.+Sireraauthor=A.+Naingauthor=D.+Hongauthor=C.+Rolfo&title=FGFR+a+promising+druggable+target+in+cancer%3A+molecular+biology+and+new+drugs&doi=10.1016%2Fj.critrevonc.2017.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">FGFR a promising druggable target in cancer: Molecular biology and new drugs</span></div><div class="casAuthors">Porta Rut; Borea Roberto; Coelho Andreia; Reclusa Pablo; Van Dam Peter; Ferri Jose; Sirera Rafael; Khan Shahanavaj; Araujo Antonio; Franchina Tindara; Van Der Steen Nele; Naing Aung; Hong David; Rolfo Christian</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">256-267</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  The Fibroblast Growth Factor Receptor (FGFR) family consists of Tyrosine Kinase Receptors (TKR) involved in several biological functions.  Recently, alterations of FGFR have been reported to be important for progression and development of several cancers.  In this setting, different studies are trying to evaluate the efficacy of different therapies targeting FGFR.  AREAS COVERED:  This review summarizes the current status of treatments targeting FGFR, focusing on the trials that are evaluating the FGFR profile as inclusion criteria: Multi-Target, Pan-FGFR Inhibitors and anti-FGF (Fibroblast Growth Factor)/FGFR Monoclonal Antibodies.  EXPERT OPINION:  Most of the TKR share intracellular signaling pathways; therefore, cancer cells tend to overcome the inhibition of one tyrosine kinase receptor by activating another.  The future of TKI (Tyrosine Kinase Inhibitor) therapy will potentially come from multi-targeted TKIs that target different TKR simultaneously.  It is crucial to understand the interaction of the FGF-FGFR axis with other known driver TKRs.  Based on this, it is possible to develop therapeutic strategies targeting multiple connected TKRs at once.  One correct step in this direction is the reassessment of multi target inhibitors considering the FGFR status of the tumor.  Another opportunity arises from assessing the use of FGFR TKI on patients harboring FGFR alterations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFLmRJD6qzpJcXNb62YGmbfW6udTcc2eZVlRj4SdTRPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvps1ajsA%253D%253D&md5=d03086e258264a12c8c5cc7f2c5de8e3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2017.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2017.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DPorta%26aufirst%3DR.%26aulast%3DBorea%26aufirst%3DR.%26aulast%3DCoelho%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DAra%25C3%25BAjo%26aufirst%3DA.%26aulast%3DReclusa%26aufirst%3DP.%26aulast%3DFranchina%26aufirst%3DT.%26aulast%3DVan%2BDer%2BSteen%26aufirst%3DN.%26aulast%3DVan%2BDam%26aufirst%3DP.%26aulast%3DFerri%26aufirst%3DJ.%26aulast%3DSirera%26aufirst%3DR.%26aulast%3DNaing%26aufirst%3DA.%26aulast%3DHong%26aufirst%3DD.%26aulast%3DRolfo%26aufirst%3DC.%26atitle%3DFGFR%2520a%2520promising%2520druggable%2520target%2520in%2520cancer%253A%2520molecular%2520biology%2520and%2520new%2520drugs%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2017%26volume%3D113%26spage%3D256%26epage%3D267%26doi%3D10.1016%2Fj.critrevonc.2017.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babina, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span> <span> </span><span class="NLM_article-title">Advances and challenges in targeting FGFR signalling in cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnrc.2017.8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28303906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Ont7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=318-332&author=I.+S.+Babinaauthor=N.+C.+Turner&title=Advances+and+challenges+in+targeting+FGFR+signalling+in+cancer&doi=10.1038%2Fnrc.2017.8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Advances and challenges in targeting FGFR signalling in cancer</span></div><div class="casAuthors">Babina, Irina S.; Turner, Nicholas C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">318-332</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fibroblast growth factors (FGFs) and their receptors (FGFRs) regulate numerous cellular processes.  Deregulation of FGFR signalling is obsd. in a subset of many cancers, making activated FGFRs a highly promising potential therapeutic target supported by multiple preclin. studies.  However, early-phase clin. trials have produced mixed results with FGFR-targeted cancer therapies, revealing substantial complexity to targeting aberrant FGFR signalling.  In this Review, we discuss the increasing understanding of the differences between diverse mechanisms of oncogenic activation of FGFR, and the factors that det. response and resistance to FGFR targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKkSMehfBbyLVg90H21EOLACvtfcHk0liNJwBIkx001A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Ont7g%253D&md5=ed8f0ad426859a07930f7c0c2d2a9b71</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.8%26sid%3Dliteratum%253Aachs%26aulast%3DBabina%26aufirst%3DI.%2BS.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26atitle%3DAdvances%2520and%2520challenges%2520in%2520targeting%2520FGFR%2520signalling%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D318%26epage%3D332%26doi%3D10.1038%2Fnrc.2017.8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Izzedine, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ederhy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldwasser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khayat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spano, J. P.</span></span> <span> </span><span class="NLM_article-title">Management of hypertension in angiogenesis inhibitor-treated patients</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdn713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19150949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BD1MzgvVyhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=807-815&author=H.+Izzedineauthor=S.+Ederhyauthor=F.+Goldwasserauthor=J.+C.+Soriaauthor=G.+Milanoauthor=A.+Cohenauthor=D.+Khayatauthor=J.+P.+Spano&title=Management+of+hypertension+in+angiogenesis+inhibitor-treated+patients&doi=10.1093%2Fannonc%2Fmdn713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Management of hypertension in angiogenesis inhibitor-treated patients</span></div><div class="casAuthors">Izzedine H; Ederhy S; Goldwasser F; Soria J C; Milano G; Cohen A; Khayat D; Spano J P</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">807-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Hypertension (HTN) is one of the most frequent side-effects of systemic inhibition of vascular endothelial growth factor (VEGF) signaling.  Its incidence and severity are dependent on the type of drugs, dose, and schedule used.  The recognition of this side-effect is an important issue because poorly controlled HTN could lead to serious cardiovascular events.  On another hand, HTN induced by anti-VEGF agents maybe a predictive factor of oncologic response.  Knowledge of this clinical toxicity and/or therapeutic target or novel biomarker of drug activity can aid clinicians choosing the optimal and least toxic regimen suitable for an individual patient.  METHODS:  A Medline search was carried out using the following criteria: (i) all Medline listings as of 1 January 2000 with abstracts, (ii) English language, and (iii) Humans.  The following phrases were used to query the database: ('hypertension', OR 'blood pressure') AND ('anti-VEGF' OR 'VEGF inhibition' OR 'bevacizumab' OR 'sunitinib' OR 'sorafenib' OR 'VEGF Trap').  The references of each article identified were carefully reviewed for additional reference.  RESULTS:  Lifestyle modification should be encouraged.  However, these nonpharmacologic strategies are not always suitable to patients with altered performance status related to metastatic cancer necessitating early drug intervention.  Only one randomized study showed a beneficial effect of a calcium channel blocker use to prevent or minimize HTN secondary to antiangiogenic therapy.  Nitrates looks as effective in controlling such side-effect.  CONCLUSIONS:  No clear recommendation for an antihypertensive agent can be made in this context because there is a lack of controlled studies addressing the subject.  Blood pressure-lowering drugs should be individualized to the patient's clinical circumstances and angiogenic inhibitors should be withheld only from patients who experienced hypertensive crisis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSwDYVxbcpKCw8IxFYvehhofW6udTcc2eZVlRj4SdTRPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzgvVyhsg%253D%253D&md5=6d056114baad0ac979d647806a410738</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn713%26sid%3Dliteratum%253Aachs%26aulast%3DIzzedine%26aufirst%3DH.%26aulast%3DEderhy%26aufirst%3DS.%26aulast%3DGoldwasser%26aufirst%3DF.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DMilano%26aufirst%3DG.%26aulast%3DCohen%26aufirst%3DA.%26aulast%3DKhayat%26aufirst%3DD.%26aulast%3DSpano%26aufirst%3DJ.%2BP.%26atitle%3DManagement%2520of%2520hypertension%2520in%2520angiogenesis%2520inhibitor-treated%2520patients%26jtitle%3DAnn.%2520Oncol.%26date%3D2009%26volume%3D20%26spage%3D807%26epage%3D815%26doi%3D10.1093%2Fannonc%2Fmdn713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricciardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Marinis, F.</span></span> <span> </span><span class="NLM_article-title">Toxicity of targeted therapy in non-small-cell lung cancer management</span>. <i>Clin. Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.3816/clc.2009.n.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.3816%2Fclc.2009.n.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19289369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslalu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=28-35&author=S.+Ricciardiauthor=S.+Tomaoauthor=F.+de+Marinis&title=Toxicity+of+targeted+therapy+in+non-small-cell+lung+cancer+management&doi=10.3816%2Fclc.2009.n.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicity of targeted therapy in non-small-cell lung cancer management</span></div><div class="casAuthors">Ricciardi, Serena; Tomao, Silverio; de Marinis, Filippo</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Lung Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-35</span>CODEN:
                <span class="NLM_cas:coden">CLCLCA</span>;
        ISSN:<span class="NLM_cas:issn">1525-7304</span>.
    
            (<span class="NLM_cas:orgname">CIG Media Group, LP</span>)
        </div><div class="casAbstract">A review.  Lung cancer is the leading cause of cancer-related death worldwide.  Despite several chemotherapeutic agents, a survival plateau has been reached, so new treatment strategies are clearly needed.  A strong interest is now focused on the use of targeted therapies for the management of non-small-cell lung cancer.  Monoclonal antibodies against the epidermal growth factor receptor (EGFR; cetuximab) or vascular endothelial growth factor receptor (VEGFR; bevacizumab) and EGFR tyrosine kinase inhibitors (gefitinib, erlotinib) are generally well tolerated and do not have the severe systemic side effects usually seen with cytotoxic drugs.  A considerable no. of treated patients develop dermatol. side effects, such as acneiform eruption, xerosis, and eczema, and unfortunately, this is often one cause of neg. impact on a patient's quality of life.  No controlled clin. trials have been performed to manage rash, so it is necessary to provide suggestions for managing this frequent side effect.  The main problems related to the class of angiogenesis inhibitors affecting VEGFRs are the exclusion of patients with brain metastases and/or squamous histol., and vascular adverse effects, such as hypertension, proteinuria, thrombosis, and hemorrhage.  There are other new agents in clin. development, such as sorafenib, sunitinib, vorinostat, vandetanib, everolimus, panobinostat, and ASA404.  They are all assocd. with a spectrum of toxicities, often reversible with interruption of dosing.  Further research is required to clarify the role of targeted therapies and toxicities management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMqY7hrfU1gbVg90H21EOLACvtfcHk0lhQeOQIwe1uXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslalu70%253D&md5=9ab0230026d6e4993fd796e5b4c33375</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3816%2Fclc.2009.n.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3816%252Fclc.2009.n.004%26sid%3Dliteratum%253Aachs%26aulast%3DRicciardi%26aufirst%3DS.%26aulast%3DTomao%26aufirst%3DS.%26aulast%3Dde%2BMarinis%26aufirst%3DF.%26atitle%3DToxicity%2520of%2520targeted%2520therapy%2520in%2520non-small-cell%2520lung%2520cancer%2520management%26jtitle%3DClin.%2520Lung%2520Cancer%26date%3D2009%26volume%3D10%26spage%3D28%26epage%3D35%26doi%3D10.3816%2Fclc.2009.n.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptors as treatment targets in clinical oncology</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0115-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fs41571-018-0115-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=30367139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=105-122&author=M.+Katoh&title=Fibroblast+growth+factor+receptors+as+treatment+targets+in+clinical+oncology&doi=10.1038%2Fs41571-018-0115-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor receptors as treatment targets in clinical oncology</span></div><div class="casAuthors">Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-122</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  FGFRs are receptor tyrosine kinases with a role in several biol. processes, such as the regulation of development and tissue repair.  However, alterations in FGFRs 1-4, such as amplifications, fusions and mutations, as well as aberrant epigenetic or transcriptional regulation and changes in tumor-stromal interactions in the tumor microenvironment, can lead to the development and/or progression of cancer.  Similar to other kinase alterations, such alterations are targetable using small mols. or antibodies, and the benefits of FGFR inhibitors have been demonstrated in clin. trials involving subsets of patients with solid tumors harbouring FGFR alterations.  However, the response rates in patients with FGFR alterations were relatively low, and responses in patients without detectable FGFR alterations were also obsd.  In this Review, the author describes the clin. experience with FGFR inhibitors to date, and highlights key aspects that might lead to improved response rates and/or the avoidance of acquired resistance, including the selection of patients who are most likely to benefit from treatment, and the use of FGFR inhibitors in combination regimens with other agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8oCjTdYY2DLVg90H21EOLACvtfcHk0lhQeOQIwe1uXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO&md5=41440ace3cb26bac6d7cdb0e18b46c39</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0115-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0115-y%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DFibroblast%2520growth%2520factor%2520receptors%2520as%2520treatment%2520targets%2520in%2520clinical%2520oncology%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D105%26epage%3D122%26doi%3D10.1038%2Fs41571-018-0115-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mooney, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgour, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Kadhimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span> <span> </span><span class="NLM_article-title">AZD4547: an orally bioavailable, potent, and selective inhibitor of the fibroblast growth factor receptor tyrosine kinase family</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2045</span>– <span class="NLM_lpage">2056</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-11-3034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F0008-5472.CAN-11-3034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=22369928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2045-2056&author=P.+R.+Gavineauthor=L.+Mooneyauthor=E.+Kilgourauthor=A.+P.+Thomasauthor=K.+Al-Kadhimiauthor=S.+Beckauthor=C.+Rooneyauthor=T.+Colemanauthor=D.+Bakerauthor=M.+J.+Mellorauthor=A.+N.+Brooksauthor=T.+Klinowska&title=AZD4547%3A+an+orally+bioavailable%2C+potent%2C+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+tyrosine+kinase+family&doi=10.1158%2F0008-5472.can-11-3034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">AZD4547: An Orally Bioavailable, Potent, and Selective Inhibitor of the Fibroblast Growth Factor Receptor Tyrosine Kinase Family</span></div><div class="casAuthors">Gavine, Paul R.; Mooney, Lorraine; Kilgour, Elaine; Thomas, Andrew P.; Al-Kadhimi, Katherine; Beck, Sarah; Rooney, Claire; Coleman, Tanya; Baker, Dawn; Mellor, Martine J.; Brooks, A. Nigel; Klinowska, Teresa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2045-2056</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The fibroblast growth factor (FGF) signaling axis is increasingly implicated in tumorigenesis and chemoresistance.  Several small-mol. FGF receptor (FGFR) kinase inhibitors are currently in clin. development; however, the predominant activity of the most advanced of these agents is against the kinase insert domain receptor (KDR), which compromises the FGFR selectivity.  Here, we report the pharmacol. profile of AZD4547, a novel and selective inhibitor of the FGFR1, 2, and 3 tyrosine kinases.  AZD4547 inhibited recombinant FGFR kinase activity in vitro and suppressed FGFR signaling and growth in tumor cell lines with deregulated FGFR expression.  In a representative FGFR-driven human tumor xenograft model, oral administration of AZD4547 was well tolerated and resulted in potent dose-dependent antitumor activity, consistent with plasma exposure and pharmacodynamic modulation of tumor FGFR.  Importantly, at efficacious doses, no evidence of anti-KDR-related effects were obsd., confirming the in vivo FGFR selectivity of AZD4547.  Taken together, our findings show that AZD4547 is a novel selective small-mol. inhibitor of FGFR with potent antitumor activity against FGFR-deregulated tumors in preclin. models.  AZD4547 is under clin. investigation for the treatment of FGFR-dependent tumors.  Cancer Res; 72(8); 2045-56.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpExryJc9xbVg90H21EOLACvtfcHk0lhQeOQIwe1uXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D&md5=cba90ad8f9b9e51cd10288868bbcc85e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-3034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-3034%26sid%3Dliteratum%253Aachs%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DMooney%26aufirst%3DL.%26aulast%3DKilgour%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DA.%2BP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DColeman%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DKlinowska%26aufirst%3DT.%26atitle%3DAZD4547%253A%2520an%2520orally%2520bioavailable%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520family%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2045%26epage%3D2056%26doi%3D10.1158%2F0008-5472.can-11-3034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Douget, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-(2,6-dichloro-3,5-dimethoxy-phenyl)-1--1-methyl-urea (NVP-BGJ398), a potent and selective inhibitor of the fibroblast growth factor receptor family of receptor tyrosine kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7066</span>– <span class="NLM_lpage">7083</span>, <span class="refDoi"> DOI: 10.1021/jm2006222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2006222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7066-7083&author=V.+Guagnanoauthor=P.+Furetauthor=C.+Spankaauthor=V.+Bordasauthor=M.+Le+Dougetauthor=C.+Stammauthor=J.+Brueggenauthor=M.+R.+Jensenauthor=C.+Schnellauthor=H.+Schmidauthor=M.+Wartmannauthor=J.+Berghausenauthor=P.+Drueckesauthor=A.+Zimmerlinauthor=D.+Bussiereauthor=J.+Murrayauthor=D.+Graus+Porta&title=Discovery+of+3-%282%2C6-dichloro-3%2C5-dimethoxy-phenyl%29-1%2D%2D1-methyl-urea+%28NVP-BGJ398%29%2C+a+potent+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+family+of+receptor+tyrosine+kinase&doi=10.1021%2Fjm2006222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-(2,6-Dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), A Potent and Selective Inhibitor of the Fibroblast Growth Factor Receptor Family of Receptor Tyrosine Kinase</span></div><div class="casAuthors">Guagnano, Vito; Furet, Pascal; Spanka, Carsten; Bordas, Vincent; Le Douget, Mickael; Stamm, Christelle; Brueggen, Josef; Jensen, Michael R.; Schnell, Christian; Schmid, Herbert; Wartmann, Markus; Berghausen, Joerg; Drueckes, Peter; Zimmerlin, Alfred; Bussiere, Dirksen; Murray, Jeremy; Graus Porta, Diana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7066-7083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of N-aryl-N'-pyrimidin-4-yl ureas has been optimized to afford potent and selective inhibitors of the fibroblast growth factor receptor tyrosine kinases 1, 2, and 3 by rationally designing the substitution pattern of the aryl ring.  On the basis of its in vitro profile, compd. 1h (NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3.  These results support the potential therapeutic use of 1h as a new anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSC1v0A_6I_bVg90H21EOLACvtfcHk0liWID7R1d9O7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK&md5=f9a2d742294cd3814e872bdd8add1ffa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm2006222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2006222%26sid%3Dliteratum%253Aachs%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBordas%26aufirst%3DV.%26aulast%3DLe%2BDouget%26aufirst%3DM.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DH.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DDiscovery%2520of%25203-%25282%252C6-dichloro-3%252C5-dimethoxy-phenyl%2529-1--1-methyl-urea%2520%2528NVP-BGJ398%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520family%2520of%2520receptor%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7066%26epage%3D7083%26doi%3D10.1021%2Fjm2006222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe
Miyano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyajima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuramochi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funasaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugi, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uenaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruoka, A.</span></span> <span> </span><span class="NLM_article-title">E7090, a novel selective inhibitor of fibroblast growth factor receptors, displays potent antitumor activity and prolongs survival in preclinical models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2630</span>– <span class="NLM_lpage">2639</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-16-0261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.mct-16-0261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27535969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A280%3ADC%252BC2szhsV2itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2630-2639&author=S.+Watanabe%0AMiyanoauthor=Y.+Yamamotoauthor=K.+Kodamaauthor=Y.+Miyajimaauthor=M.+Mikamotoauthor=T.+Nakagawaauthor=H.+Kuramochiauthor=S.+Funasakaauthor=S.+Nagaoauthor=N.+H.+Sugiauthor=K.+Okamotoauthor=Y.+Minoshimaauthor=Y.+Nakataniauthor=Y.+Karojiauthor=I.+Ohashiauthor=Y.+Yamaneauthor=T.+Okadaauthor=T.+Matsushimaauthor=J.+Matsuiauthor=M.+Iwataauthor=T.+Uenakaauthor=A.+Tsuruoka&title=E7090%2C+a+novel+selective+inhibitor+of+fibroblast+growth+factor+receptors%2C+displays+potent+antitumor+activity+and+prolongs+survival+in+preclinical+models&doi=10.1158%2F1535-7163.mct-16-0261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">E7090, a Novel Selective Inhibitor of Fibroblast Growth Factor Receptors, Displays Potent Antitumor Activity and Prolongs Survival in Preclinical Models</span></div><div class="casAuthors">Watanabe Miyano Saori; Yamamoto Yuji; Kodama Kotaro; Miyajima Yukiko; Mikamoto Masaki; Nakagawa Takayuki; Kuramochi Hiroko; Funasaka Setsuo; Nagao Satoshi; Sugi Naoko Hata; Okamoto Kiyoshi; Minoshima Yukinori; Nakatani Yusuke; Karoji Yuki; Ohashi Isao; Yamane Yoshinobu; Okada Toshimi; Matsushima Tomohiro; Matsui Junji; Iwata Masao; Uenaka Toshimitsu; Tsuruoka Akihiko; Watanabe Miyano Saori</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2630-2639</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The FGFR signaling pathway has a crucial role in proliferation, survival, and migration of cancer cells, tumor angiogenesis, and drug resistance.  FGFR genetic abnormalities, such as gene fusion, mutation, and amplification, have been implicated in several types of cancer.  Therefore, FGFRs are considered potential targets for cancer therapy.  E7090 is an orally available and selective inhibitor of the tyrosine kinase activities of FGFR1, -2, and -3.  In kinetic analyses of the interaction between E7090 and FGFR1 tyrosine kinase, E7090 associated more rapidly with FGFR1 than did the type II FGFR1 inhibitor ponatinib, and E7090 dissociated more slowly from FGFR1, with a relatively longer residence time, than did the type I FGFR1 inhibitor AZD4547, suggesting that its kinetics are more similar to the type V inhibitors, such as lenvatinib.  E7090 showed selective antiproliferative activity against cancer cell lines harboring FGFR genetic abnormalities and decreased tumor size in a mouse xenograft model using cell lines with dysregulated FGFR Furthermore, E7090 administration significantly prolonged the survival of mice with metastasized tumors in the lung.  Our results suggest that E7090 is a promising candidate as a therapeutic agent for the treatment of tumors harboring FGFR genetic abnormalities.  It is currently being investigated in a phase I clinical trial.  Mol Cancer Ther; 15(11); 2630-9. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTFDNVVHGDPezslzEEsg76ufW6udTcc2eb3PLSn0Nt5M7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szhsV2itA%253D%253D&md5=e0fbe235b75cd03d13dc8f6e9fc9ecbe</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-16-0261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-16-0261%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%2BMiyano%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DY.%26aulast%3DKodama%26aufirst%3DK.%26aulast%3DMiyajima%26aufirst%3DY.%26aulast%3DMikamoto%26aufirst%3DM.%26aulast%3DNakagawa%26aufirst%3DT.%26aulast%3DKuramochi%26aufirst%3DH.%26aulast%3DFunasaka%26aufirst%3DS.%26aulast%3DNagao%26aufirst%3DS.%26aulast%3DSugi%26aufirst%3DN.%2BH.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DKaroji%26aufirst%3DY.%26aulast%3DOhashi%26aufirst%3DI.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DMatsushima%26aufirst%3DT.%26aulast%3DMatsui%26aufirst%3DJ.%26aulast%3DIwata%26aufirst%3DM.%26aulast%3DUenaka%26aufirst%3DT.%26aulast%3DTsuruoka%26aufirst%3DA.%26atitle%3DE7090%252C%2520a%2520novel%2520selective%2520inhibitor%2520of%2520fibroblast%2520growth%2520factor%2520receptors%252C%2520displays%2520potent%2520antitumor%2520activity%2520and%2520prolongs%2520survival%2520in%2520preclinical%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2630%26epage%3D2639%26doi%3D10.1158%2F1535-7163.mct-16-0261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zia-Ebrahimi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloem, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huss, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.-b.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, D. J.</span></span> <span> </span><span class="NLM_article-title">A novel, selective inhibitor of fibroblast growth factor receptors that shows a potent broad spectrum of antitumor activity in several tumor xenograft models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2200</span>– <span class="NLM_lpage">2210</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-11-0306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.MCT-11-0306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=21900693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVeltb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=2200-2210&author=G.+Zhaoauthor=W.-y.+Liauthor=D.+Chenauthor=J.+R.+Henryauthor=H.-Y.+Liauthor=Z.+Chenauthor=M.+Zia-Ebrahimiauthor=L.+Bloemauthor=Y.+Zhaiauthor=K.+Hussauthor=S.-b.+Pengauthor=D.+J.+McCann&title=A+novel%2C+selective+inhibitor+of+fibroblast+growth+factor+receptors+that+shows+a+potent+broad+spectrum+of+antitumor+activity+in+several+tumor+xenograft+models&doi=10.1158%2F1535-7163.mct-11-0306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel, Selective Inhibitor of Fibroblast Growth Factor Receptors That Shows a Potent Broad Spectrum of Antitumor Activity in Several Tumor Xenograft Models</span></div><div class="casAuthors">Zhao, Genshi; Li, Wei-ying; Chen, Daohong; Henry, James R.; Li, Hong-Yu; Chen, Zhaogen; Zia-Ebrahimi, Mohammad; Bloem, Laura; Zhai, Yan; Huss, Karen; Peng, Sheng-bin; McCann, Denis J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2200-2210</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The fibroblast growth factor receptors (FGFR) are tyrosine kinases that are present in many types of endothelial and tumor cells and play an important role in tumor cell growth, survival, and migration as well as in maintaining tumor angiogenesis.  Overexpression of FGFRs or aberrant regulation of their activities has been implicated in many forms of human malignancies.  Therefore, targeting FGFRs represents an attractive strategy for development of cancer treatment options by simultaneously inhibiting tumor cell growth, survival, and migration as well as tumor angiogenesis.  Here, we describe a potent, selective, small-mol. FGFR inhibitor, (R)-(E)-2-(4-(2-(5-(1-(3,5-Dichloropyridin-4-yl)ethoxy)-1H-indazol-3yl)vinyl)-1H-pyrazol-1-yl)ethanol, designated as LY2874455.  This mol. is active against all 4 FGFRs, with a similar potency in biochem. assays.  It exhibits a potent activity against FGF/FGFR-mediated signaling in several cancer cell lines and shows an excellent broad spectrum of antitumor activity in several tumor xenograft models representing the major FGF/FGFR relevant tumor histologies including lung, gastric, and bladder cancers and multiple myeloma, and with a well-defined pharmacokinetic/pharmacodynamic relationship.  LY2874455 also exhibits a 6- to 9-fold in vitro and in vivo selectivity on inhibition of FGF- over VEGF-mediated target signaling in mice.  Furthermore, LY2874455 did not show VEGF receptor 2-mediated toxicities such as hypertension at efficacious doses.  Currently, this mol. is being evaluated for its potential use in the clinic.  Mol Cancer Ther; 10(11); 2200-10.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLrzAyZQK_grVg90H21EOLACvtfcHk0lgqx2yodWRDkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVeltb7N&md5=eaf08fd213a836f48da2055e01bd9c29</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-11-0306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-11-0306%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DW.-y.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DHenry%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DH.-Y.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZia-Ebrahimi%26aufirst%3DM.%26aulast%3DBloem%26aufirst%3DL.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DHuss%26aufirst%3DK.%26aulast%3DPeng%26aufirst%3DS.-b.%26aulast%3DMcCann%26aufirst%3DD.%2BJ.%26atitle%3DA%2520novel%252C%2520selective%2520inhibitor%2520of%2520fibroblast%2520growth%2520factor%2520receptors%2520that%2520shows%2520a%2520potent%2520broad%2520spectrum%2520of%2520antitumor%2520activity%2520in%2520several%2520tumor%2520xenograft%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2011%26volume%3D10%26spage%3D2200%26epage%3D2210%26doi%3D10.1158%2F1535-7163.mct-11-0306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, T. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovcheva, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mevellec, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vialard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Lange, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhulst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulussen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van De Ven, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freyne, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squires, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilissen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platero, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkera, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angibaud, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span> <span> </span><span class="NLM_article-title">Discovery and pharmacological characterization of JNJ-42756493 (Erdafitinib), a functionally selective small-molecule FGFR family inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1020</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-16-0589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.MCT-16-0589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28341788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1010-1020&author=T.+P.+S.+Pereraauthor=E.+Jovchevaauthor=L.+Mevellecauthor=J.+Vialardauthor=D.+De+Langeauthor=T.+Verhulstauthor=C.+Paulussenauthor=K.+Van+De+Venauthor=P.+Kingauthor=E.+Freyneauthor=D.+C.+Reesauthor=M.+Squiresauthor=G.+Saxtyauthor=M.+Pageauthor=C.+W.+Murrayauthor=R.+Gilissenauthor=G.+Wardauthor=N.+T.+Thompsonauthor=D.+R.+Newellauthor=N.+Chengauthor=L.+Xieauthor=J.+Yangauthor=S.+J.+Plateroauthor=J.+D.+Karkeraauthor=C.+Moyauthor=P.+Angibaudauthor=S.+Laquerreauthor=M.+V.+Lorenzi&title=Discovery+and+pharmacological+characterization+of+JNJ-42756493+%28Erdafitinib%29%2C+a+functionally+selective+small-molecule+FGFR+family+inhibitor&doi=10.1158%2F1535-7163.mct-16-0589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Pharmacological Characterization of JNJ-42756493 (Erdafitinib), a Functionally Selective Small-Molecule FGFR Family Inhibitor</span></div><div class="casAuthors">Perera, Timothy P. S.; Jovcheva, Eleonora; Mevellec, Laurence; Vialard, Jorge; De Lange, Desiree; Verhulst, Tinne; Paulussen, Caroline; Van De Ven, Kelly; King, Peter; Freyne, Eddy; Rees, David C.; Squires, Matthew; Saxty, Gordon; Page, Martin; Murray, Christopher W.; Gilissen, Ron; Ward, George; Thompson, Neil T.; Newell, David R.; Cheng, Na; Xie, Liang; Yang, Jennifer; Platero, Suso J.; Karkera, Jayaprakash D.; Moy, Christopher; Angibaud, Patrick; Laquerre, Sylvie; Lorenzi, Matthew V.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1010-1020</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Fibroblast growth factor (FGF) signaling plays crit. roles in key biol. processes ranging from embryogenesis to wound healing and has strong links to several hallmarks of cancer.  Genetic alterations in FGF receptor (FGFR) family members are assocd. with increased tumor growth, metastasis, angiogenesis, and decreased survival.  JNJ-42756493, erdafitinib, is an orally active small mol. with potent tyrosine kinase inhibitory activity against all four FGFR family members and selectivity vs. other highly related kinases.  JNJ-42756493 shows rapid uptake into the lysosomal compartment of cells in culture, which is assocd. with prolonged inhibition of FGFR signaling, possibly due to sustained release of the inhibitor.  In xenografts from human tumor cell lines or patient-derived tumor tissue with activating FGFR alterations, JNJ-42756493 administration results in potent and dose-dependent antitumor activity accompanied by pharmacodynamic modulation of phospho-FGFR and phospho-ERK in tumors.  The results of the current study provide a strong rationale for the clin. investigation of JNJ-42756493 in patients with tumors harboring FGFR pathway alterations.  Mol Cancer Ther; 16(6); 1010-20. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhJZ2AgFcI6LVg90H21EOLACvtfcHk0lgqx2yodWRDkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sht7g%253D&md5=2cae21f09ad519925ab2a3c760284b02</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0589%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DT.%2BP.%2BS.%26aulast%3DJovcheva%26aufirst%3DE.%26aulast%3DMevellec%26aufirst%3DL.%26aulast%3DVialard%26aufirst%3DJ.%26aulast%3DDe%2BLange%26aufirst%3DD.%26aulast%3DVerhulst%26aufirst%3DT.%26aulast%3DPaulussen%26aufirst%3DC.%26aulast%3DVan%2BDe%2BVen%26aufirst%3DK.%26aulast%3DKing%26aufirst%3DP.%26aulast%3DFreyne%26aufirst%3DE.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DSquires%26aufirst%3DM.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DPage%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DGilissen%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DG.%26aulast%3DThompson%26aufirst%3DN.%2BT.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPlatero%26aufirst%3DS.%2BJ.%26aulast%3DKarkera%26aufirst%3DJ.%2BD.%26aulast%3DMoy%26aufirst%3DC.%26aulast%3DAngibaud%26aufirst%3DP.%26aulast%3DLaquerre%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26atitle%3DDiscovery%2520and%2520pharmacological%2520characterization%2520of%2520JNJ-42756493%2520%2528Erdafitinib%2529%252C%2520a%2520functionally%2520selective%2520small-molecule%2520FGFR%2520family%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D1010%26epage%3D1020%26doi%3D10.1158%2F1535-7163.mct-16-0589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isobe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morikami, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindoh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebiike, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, N.</span></span> <span> </span><span class="NLM_article-title">The fibroblast growth factor receptor genetic status as a potential predictor of the sensitivity to CH5183284/Debio 1347, a novel selective FGFR inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2547</span>– <span class="NLM_lpage">2558</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-14-0248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1535-7163.mct-14-0248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=25169980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=2547-2558&author=Y.+Nakanishiauthor=N.+Akiyamaauthor=T.+Tsukaguchiauthor=T.+Fujiiauthor=K.+Sakataauthor=H.+Saseauthor=T.+Isobeauthor=K.+Morikamiauthor=H.+Shindohauthor=T.+Mioauthor=H.+Ebiikeauthor=N.+Takaauthor=Y.+Aokiauthor=N.+Ishii&title=The+fibroblast+growth+factor+receptor+genetic+status+as+a+potential+predictor+of+the+sensitivity+to+CH5183284%2FDebio+1347%2C+a+novel+selective+FGFR+inhibitor&doi=10.1158%2F1535-7163.mct-14-0248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Fibroblast Growth Factor Receptor Genetic Status as a Potential Predictor of the Sensitivity to CH5183284/Debio 1347, a Novel Selective FGFR Inhibitor</span></div><div class="casAuthors">Nakanishi, Yoshito; Akiyama, Nukinori; Tsukaguchi, Toshiyuki; Fujii, Toshihiko; Sakata, Kiyoaki; Sase, Hitoshi; Isobe, Takehito; Morikami, Kenji; Shindoh, Hidetoshi; Mio, Toshiyuki; Ebiike, Hirosato; Taka, Naoki; Aoki, Yuko; Ishii, Nobuya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2547-2558</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The FGF receptors (FGFR) are tyrosine kinases that are constitutively activated in a subset of tumors by genetic alterations such as gene amplifications, point mutations, or chromosomal translocations/rearrangements.  Recently, small-mol. inhibitors that can inhibit the FGFR family as well as the VEGF receptor (VEGFR) or platelet-derived growth factor receptor (PDGFR) family displayed clin. benefits in cohorts of patients with FGFR genetic alterations.  However, to achieve more potent and prolonged activity in such populations, a selective FGFR inhibitor is still needed.  Here, we report the identification of CH5183284/Debio 1347, a selective and orally available FGFR1, FGFR2, and FGFR3 inhibitor that has a unique chem. scaffold.  By interacting with unique residues in the ATP-binding site of FGFR1, FGFR2, or FGFR3, CH5183284/Debio 1347 selectively inhibits FGFR1, FGFR2, and FGFR3 but does not inhibit kinase insert domain receptor (KDR) or other kinases.  Consistent with its high selectivity for FGFR enzymes, CH5183284/Debio 1347 displayed preferential antitumor activity against cancer cells with various FGFR genetic alterations in a panel of 327 cancer cell lines and in xenograft models.  Because of its unique binding mode, CH5183284/Debio 1347 can inhibit FGFR2 harboring one type of the gatekeeper mutation that causes resistance to other FGFR inhibitors and block FGFR2 V564F-driven tumor growth.  CH5183284/Debio 1347 is under clin. investigation for the treatment of patients harboring FGFR genetic alterations.  Mol Cancer Ther; 13(11); 2547-58. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGIBEnK21337Vg90H21EOLACvtfcHk0lhudHXMoB5Cow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73F&md5=35629d891a421582022b0f6ae0496479</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-14-0248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-14-0248%26sid%3Dliteratum%253Aachs%26aulast%3DNakanishi%26aufirst%3DY.%26aulast%3DAkiyama%26aufirst%3DN.%26aulast%3DTsukaguchi%26aufirst%3DT.%26aulast%3DFujii%26aufirst%3DT.%26aulast%3DSakata%26aufirst%3DK.%26aulast%3DSase%26aufirst%3DH.%26aulast%3DIsobe%26aufirst%3DT.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DShindoh%26aufirst%3DH.%26aulast%3DMio%26aufirst%3DT.%26aulast%3DEbiike%26aufirst%3DH.%26aulast%3DTaka%26aufirst%3DN.%26aulast%3DAoki%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DN.%26atitle%3DThe%2520fibroblast%2520growth%2520factor%2520receptor%2520genetic%2520status%2520as%2520a%2520potential%2520predictor%2520of%2520the%2520sensitivity%2520to%2520CH5183284%252FDebio%25201347%252C%2520a%2520novel%2520selective%2520FGFR%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D2547%26epage%3D2558%26doi%3D10.1158%2F1535-7163.mct-14-0248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, L.</span>; <span class="NLM_string-name">Arkenau, H.-T.</span>; <span class="NLM_string-name">Tran, B.</span>; <span class="NLM_string-name">Soria, J.-C.</span>; <span class="NLM_string-name">Bahleda, R.</span>; <span class="NLM_string-name">Mak, G.</span>; <span class="NLM_string-name">Zhu, A.</span>; <span class="NLM_string-name">Javle, M.</span>; <span class="NLM_string-name">Hiroshi, H.</span>; <span class="NLM_string-name">Benedetti, F.</span>; <span class="NLM_string-name">Huang, J.</span>; <span class="NLM_string-name">Winkler, R.</span>; <span class="NLM_string-name">Meric-Bernstam, F.</span></span>. <span> </span><span class="NLM_article-title">O-020 Early clinical efficacy of TAS-120, a covalently bound FGFR inhibitor, in patients with cholangiocarcinoma</span>.  <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume">28</span>, suppl_3. <span class="refDoi"> DOI: 10.1093/annonc/mdx262.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdx262.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&author=L.+Goyal&author=H.-T.+Arkenau&author=B.+Tran&author=J.-C.+Soria&author=R.+Bahleda&author=G.+Mak&author=A.+Zhu&author=M.+Javle&author=H.+Hiroshi&author=F.+Benedetti&author=J.+Huang&author=R.+Winkler&author=F.+Meric-Bernstam&title=O-020+Early+clinical+efficacy+of+TAS-120%2C+a+covalently+bound+FGFR+inhibitor%2C+in+patients+with+cholangiocarcinoma&doi=10.1093%2Fannonc%2Fmdx262.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx262.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx262.019%26sid%3Dliteratum%253Aachs%26aulast%3DGoyal%26aufirst%3DL.%26atitle%3DO-020%2520Early%2520clinical%2520efficacy%2520of%2520TAS-120%252C%2520a%2520covalently%2520bound%2520FGFR%2520inhibitor%252C%2520in%2520patients%2520with%2520cholangiocarcinoma%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26doi%3D10.1093%2Fannonc%2Fmdx262.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span>; <span class="NLM_string-name">Arkenau, H.</span>; <span class="NLM_string-name">Tran, B.</span>; <span class="NLM_string-name">Bahleda, R.</span>; <span class="NLM_string-name">Kelley, R.</span>; <span class="NLM_string-name">Hierro, C.</span>; <span class="NLM_string-name">Ahn, D.</span>; <span class="NLM_string-name">Zhu, A.</span>; <span class="NLM_string-name">Javle, M.</span>; <span class="NLM_string-name">Winkler, R.</span>; <span class="NLM_string-name">He, H.</span>; <span class="NLM_string-name">Huang, J.</span>; <span class="NLM_string-name">Goyal, L.</span></span> <span> </span><span class="NLM_article-title">O-001 Efficacy of TAS-120, an irreversible Fibroblast Growth Factor Receptor (FGFR) inhibitor, in cholangiocarcinoma patients with FGFR pathway alterations who were previously treated with chemotherapy and other FGFR inhibitors</span>.  <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">29</span>, suppl_5.<span class="refDoi"> DOI: 10.1093/annonc/mdy149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fannonc%2Fmdy149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=29635542" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&author=F.+Meric-Bernstam&author=H.+Arkenau&author=B.+Tran&author=R.+Bahleda&author=R.+Kelley&author=C.+Hierro&author=D.+Ahn&author=A.+Zhu&author=M.+Javle&author=R.+Winkler&author=H.+He&author=J.+Huang&author=L.+Goyal&title=O-001+Efficacy+of+TAS-120%2C+an+irreversible+Fibroblast+Growth+Factor+Receptor+%28FGFR%29+inhibitor%2C+in+cholangiocarcinoma+patients+with+FGFR+pathway+alterations+who+were+previously+treated+with+chemotherapy+and+other+FGFR+inhibitors&doi=10.1093%2Fannonc%2Fmdy149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdy149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdy149%26sid%3Dliteratum%253Aachs%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26atitle%3DO-001%2520Efficacy%2520of%2520TAS-120%252C%2520an%2520irreversible%2520Fibroblast%2520Growth%2520Factor%2520Receptor%2520%2528FGFR%2529%2520inhibitor%252C%2520in%2520cholangiocarcinoma%2520patients%2520with%2520FGFR%2520pathway%2520alterations%2520who%2520were%2520previously%2520treated%2520with%2520chemotherapy%2520and%2520other%2520FGFR%2520inhibitors%26jtitle%3DAnn.%2520Oncol.%26date%3D2018%26volume%3D29%26doi%3D10.1093%2Fannonc%2Fmdy149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paik, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathewson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgour, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frewer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">André, F.</span></span> <span> </span><span class="NLM_article-title">A Phase Ib open-label multicenter study of AZD4547 in patients with advanced squamous cell lung cancers</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5366</span>– <span class="NLM_lpage">5373</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-17-0645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1158%2F1078-0432.ccr-17-0645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28615371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=5366-5373&author=P.+K.+Paikauthor=R.+Shenauthor=M.+F.+Bergerauthor=D.+Ferryauthor=J.-C.+Soriaauthor=A.+Mathewsonauthor=C.+Rooneyauthor=N.+R.+Smithauthor=M.+Cullbergauthor=E.+Kilgourauthor=D.+Landersauthor=P.+Frewerauthor=N.+Brooksauthor=F.+Andr%C3%A9&title=A+Phase+Ib+open-label+multicenter+study+of+AZD4547+in+patients+with+advanced+squamous+cell+lung+cancers&doi=10.1158%2F1078-0432.ccr-17-0645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Phase Ib Open-Label Multicenter Study of AZD4547 in Patients with Advanced Squamous Cell Lung Cancers</span></div><div class="casAuthors">Paik, Paul K.; Shen, Ronglai; Berger, Michael F.; Ferry, David; Soria, Jean-Charles; Mathewson, Alastair; Rooney, Claire; Smith, Neil R.; Cullberg, Marie; Kilgour, Elaine; Landers, Donal; Frewer, Paul; Brooks, Nigel; Andre, Fabrice</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5366-5373</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Squamous cell lung cancers (SQCLC) account for 25% of all NSCLCs, yet the prognosis of these patients is poor and treatment options are limited.  Amplified FGFR1 is one of the most common oncogenic events in SQCLCs, occurring in approx. 20% of cases.  AZD4547 is a potent and selective FGFR1-3 inhibitor with antitumor activity in FGFR1-amplified SQCLC cell lines and patient-derived xenografts.  Exptl. Design: On the basis of these data, we performed a phase I study of AZD4547 in patients with previously treated stage IV FGFR1-amplified SQCLCs (NCT00979134).  FGFR1 amplification (FGFR1:CEP8 ≥ 2) was detd. by FISH.  The primary endpoint was safety/tolerability.  Secondary endpoints included antitumor activity, pharmacokinetics, pharmacodynamics, and mol. analyses.  Results: Fifteen FGFR1-amplified patients were treated.  The most common related adverse events (AE) were gastrointestinal and dermatol.  Grade ≥3-related AEs occurred in 3 patients (23%).  Thirteen patients were evaluable for radiog. response assessment.  The overall response rate was 8% (1 PR).  Two of 15 patients (13.3%) were progression-free at 12 wk, and the median overall survival was 4.9 mo.  Mol. tests, including next-generation sequencing, gene expression anal., and FGFR1 immunohistochem., showed poor correlation between gene amplification and expression, potential genomic modifiers of efficacy, and heterogeneity in 8p11 amplicon.  Conclusions: AZD4547 was tolerable at a dosage of 80 mg oral twice a day, with modest antitumor activity.  Detailed mol. studies show that these tumors are heterogeneous, with a range of mutational covariates and stark differences in gene expression of the 8p11 amplicon that likely explain the modest efficacy of FGFR inhibition in this disease.  Clin Cancer Res; 23(18); 5366-73. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtbUEY_Z929rVg90H21EOLACvtfcHk0lhudHXMoB5Cow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2mu7fP&md5=17d6065f9f30ea9dd61ee4415fb72b33</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-17-0645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-17-0645%26sid%3Dliteratum%253Aachs%26aulast%3DPaik%26aufirst%3DP.%2BK.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DFerry%26aufirst%3DD.%26aulast%3DSoria%26aufirst%3DJ.-C.%26aulast%3DMathewson%26aufirst%3DA.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DN.%2BR.%26aulast%3DCullberg%26aufirst%3DM.%26aulast%3DKilgour%26aufirst%3DE.%26aulast%3DLanders%26aufirst%3DD.%26aulast%3DFrewer%26aufirst%3DP.%26aulast%3DBrooks%26aufirst%3DN.%26aulast%3DAndr%25C3%25A9%26aufirst%3DF.%26atitle%3DA%2520Phase%2520Ib%2520open-label%2520multicenter%2520study%2520of%2520AZD4547%2520in%2520patients%2520with%2520advanced%2520squamous%2520cell%2520lung%2520cancers%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D5366%26epage%3D5373%26doi%3D10.1158%2F1078-0432.ccr-17-0645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seckl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badman, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zubairi, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Corbacho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cresti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombes, C.</span></span> <span> </span><span class="NLM_article-title">RADICAL trial: a phase Ib/IIa study to assess the safety and efficacy of AZD4547 in combination with either anastrozole or letrozole in ER positive breast cancer patients progressing on these aromatase inhibitors (AIs)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1059</span>, <span class="refDoi"> DOI: 10.1200/jco.2017.35.15_suppl.1059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1200%2Fjco.2017.35.15_suppl.1059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1059&author=M.+Secklauthor=P.+D.+Badmanauthor=X.+Liuauthor=I.+R.+MacPhersonauthor=I.+H.+Zubairiauthor=R.+D.+Bairdauthor=J.+Garcia-Corbachoauthor=N.+Crestiauthor=E.+R.+Plummerauthor=A.+C.+Armstrongauthor=R.+Allertonauthor=D.+Landersauthor=H.+Nicholasauthor=L.+McLellanauthor=A.+K.+Limauthor=C.+Coombes&title=RADICAL+trial%3A+a+phase+Ib%2FIIa+study+to+assess+the+safety+and+efficacy+of+AZD4547+in+combination+with+either+anastrozole+or+letrozole+in+ER+positive+breast+cancer+patients+progressing+on+these+aromatase+inhibitors+%28AIs%29&doi=10.1200%2Fjco.2017.35.15_suppl.1059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1200%2Fjco.2017.35.15_suppl.1059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2017.35.15_suppl.1059%26sid%3Dliteratum%253Aachs%26aulast%3DSeckl%26aufirst%3DM.%26aulast%3DBadman%26aufirst%3DP.%2BD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMacPherson%26aufirst%3DI.%2BR.%26aulast%3DZubairi%26aufirst%3DI.%2BH.%26aulast%3DBaird%26aufirst%3DR.%2BD.%26aulast%3DGarcia-Corbacho%26aufirst%3DJ.%26aulast%3DCresti%26aufirst%3DN.%26aulast%3DPlummer%26aufirst%3DE.%2BR.%26aulast%3DArmstrong%26aufirst%3DA.%2BC.%26aulast%3DAllerton%26aufirst%3DR.%26aulast%3DLanders%26aufirst%3DD.%26aulast%3DNicholas%26aufirst%3DH.%26aulast%3DMcLellan%26aufirst%3DL.%26aulast%3DLim%26aufirst%3DA.%2BK.%26aulast%3DCoombes%26aufirst%3DC.%26atitle%3DRADICAL%2520trial%253A%2520a%2520phase%2520Ib%252FIIa%2520study%2520to%2520assess%2520the%2520safety%2520and%2520efficacy%2520of%2520AZD4547%2520in%2520combination%2520with%2520either%2520anastrozole%2520or%2520letrozole%2520in%2520ER%2520positive%2520breast%2520cancer%2520patients%2520progressing%2520on%2520these%2520aromatase%2520inhibitors%2520%2528AIs%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D35%26spage%3D1059%26doi%3D10.1200%2Fjco.2017.35.15_suppl.1059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, L.</span>; <span class="NLM_string-name">Liu, L.
Y.</span>; <span class="NLM_string-name">Lennerz, J. K.</span>; <span class="NLM_string-name">Harding, J. J.</span>; <span class="NLM_string-name">Huang, J.</span>; <span class="NLM_string-name">Winkler, R.</span>; <span class="NLM_string-name">Hiroshi, H.</span>; <span class="NLM_string-name">Ting, D.
T.</span>; <span class="NLM_string-name">Juric, D.</span>; <span class="NLM_string-name">Corcoran, R. B.</span>; <span class="NLM_string-name">El-Bardeesy, N.</span>; <span class="NLM_string-name">Zhu, A. X.</span></span> <span> </span><span class="NLM_article-title">Abstract LB-092: TAS120, a covalently-binding FGFR inhibitor (FGFRi), overcomes resistance to BGJ398 in patients with FGFR2 fusion positive cholangiocarcinoma</span>.  <i>Cancer Research</i>, <span class="NLM_year">2018</span>; Vol.  <span class="NLM_volume">78</span>, LB-092-LB-092.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=L.+Goyal&author=L.%0AY.+Liu&author=J.+K.+Lennerz&author=J.+J.+Harding&author=J.+Huang&author=R.+Winkler&author=H.+Hiroshi&author=D.%0AT.+Ting&author=D.+Juric&author=R.+B.+Corcoran&author=N.+El-Bardeesy&author=A.+X.+Zhu&title=Abstract+LB-092%3A+TAS120%2C+a+covalently-binding+FGFR+inhibitor+%28FGFRi%29%2C+overcomes+resistance+to+BGJ398+in+patients+with+FGFR2+fusion+positive+cholangiocarcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DGoyal%26aufirst%3DL.%26atitle%3DAbstract%2520LB-092%253A%2520TAS120%252C%2520a%2520covalently-binding%2520FGFR%2520inhibitor%2520%2528FGFRi%2529%252C%2520overcomes%2520resistance%2520to%2520BGJ398%2520in%2520patients%2520with%2520FGFR2%2520fusion%2520positive%2520cholangiocarcinoma%26jtitle%3DCancer%2520Research%26date%3D2018%26volume%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel disruptor of silencing telomeric 1-like (DOT1L) inhibitors using a target-specific scoring function for the (S)-Adenosyl-l-methionine (SAM)-dependent methyltransferase family</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2026</span>– <span class="NLM_lpage">2036</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01785</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01785" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1SjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2026-2036&author=Y.+Wangauthor=L.+Liauthor=B.+Zhangauthor=J.+Xingauthor=S.+Chenauthor=W.+Wanauthor=Y.+Songauthor=H.+Jiangauthor=H.+Jiangauthor=C.+Luoauthor=M.+Zheng&title=Discovery+of+novel+disruptor+of+silencing+telomeric+1-like+%28DOT1L%29+inhibitors+using+a+target-specific+scoring+function+for+the+%28S%29-Adenosyl-l-methionine+%28SAM%29-dependent+methyltransferase+family&doi=10.1021%2Facs.jmedchem.6b01785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Disruptor of Silencing Telomeric 1-Like (DOT1L) Inhibitors using a Target-Specific Scoring Function for the (S)-Adenosyl-L-methionine (SAM)-Dependent Methyltransferase Family</span></div><div class="casAuthors">Wang, Yulan; Li, Linjuan; Zhang, Bidong; Xing, Jing; Chen, Shijie; Wan, Wei; Song, Yakai; Jiang, Hao; Jiang, Hualiang; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2026-2036</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The disruptor of telomeric silencing 1-like (DOT1L) protein is a histone H3K79 methyltransferase that plays a key role in transcriptional elongation and cell cycle regulation and is required for the development and maintenance of MLL-rearranged mixed lineage leukemia.  Much effort has been dedicated towards discovering novel scaffold DOT1L inhibitors using different strategies.  Here, the authors report the development and application of a target-specific scoring function, the SAM score, for S-adenosyl-L-methionine (SAM)-dependent methyltransferase, for the discovery of novel DOT1L inhibitors.  Based on the SAM score, the authors successfully identified a novel class of DOT1L inhibitors with a scaffold of [1,2,4]-triazolo-[3,4-b][1,3,4]-thiadiazole, in which compd. (I) exhibits an IC50 value of 8.3 μM with selectivity vs. other tested SAM-dependent methyltransferases.  In cellular studies, I selectively targets DOT1L, blocks the proliferation of mixed lineage leukemia cell lines, and causes cell cycle arrest and apoptosis.  Moreover, the authors analyzed the putative binding modes of 6 and its analogs obtained by mol. docking, which may assist with the future development of DOT1L inhibitors with improved potency and selectivity profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpphV6NpZDlrVg90H21EOLACvtfcHk0limrxdFZUm4Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1SjtLo%253D&md5=b1df8ae062762c717ca467014cd93de6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01785%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWan%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520novel%2520disruptor%2520of%2520silencing%2520telomeric%25201-like%2520%2528DOT1L%2529%2520inhibitors%2520using%2520a%2520target-specific%2520scoring%2520function%2520for%2520the%2520%2528S%2529-Adenosyl-l-methionine%2520%2528SAM%2529-dependent%2520methyltransferase%2520family%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2026%26epage%3D2036%26doi%3D10.1021%2Facs.jmedchem.6b01785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliseenkova, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.molcel.2007.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=17803937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=717-730&author=H.+Chenauthor=J.+Maauthor=W.+Liauthor=A.+V.+Eliseenkovaauthor=C.+Xuauthor=T.+A.+Neubertauthor=W.+T.+Millerauthor=M.+Mohammadi&title=A+molecular+brake+in+the+kinase+hinge+region+regulates+the+activity+of+receptor+tyrosine+kinases&doi=10.1016%2Fj.molcel.2007.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Huaibin; Ma, Jinghong; Li, Wanqing; Eliseenkova, Anna V.; Xu, Chongfeng; Neubert, Thomas A.; Miller, W. Todd; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">717-730</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Activating mutations in the tyrosine kinase domain of receptor tyrosine kinases (RTKs) cause cancer and skeletal disorders.  Comparison of the crystal structures of unphosphorylated and phosphorylated wild-type FGFR2 kinase domains (FGFR2Ks) with those of seven unphosphorylated pathogenic mutants reveals an autoinhibitory "mol. brake" mediated by a triad of residues in the kinase hinge region of all FGFRs.  Structural anal. shows that many other RTKs, including PDGFRs, VEGFRs, KIT, CSF1R, FLT3, TEK, and TIE, are also subject to regulation by this brake.  Pathogenic mutations activate FGFRs and other RTKs by disengaging the brake either directly or indirectly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYTkvrN7lYrVg90H21EOLACvtfcHk0limrxdFZUm4Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I&md5=dba8a6b97b9d1826b2223146be137963</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DA%2520molecular%2520brake%2520in%2520the%2520kinase%2520hinge%2520region%2520regulates%2520the%2520activity%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D717%26epage%3D730%26doi%3D10.1016%2Fj.molcel.2007.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">Structural mimicry of a-loop tyrosine phosphorylation by a pathogenic FGF receptor 3 mutation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1889</span>– <span class="NLM_lpage">1896</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.str.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23972473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Slu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1889-1896&author=Z.+Huangauthor=H.+Chenauthor=S.+Blaisauthor=T.+A.+Neubertauthor=X.+Liauthor=M.+Mohammadi&title=Structural+mimicry+of+a-loop+tyrosine+phosphorylation+by+a+pathogenic+FGF+receptor+3+mutation&doi=10.1016%2Fj.str.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Mimicry of A-Loop Tyrosine Phosphorylation by a Pathogenic FGF Receptor 3 Mutation</span></div><div class="casAuthors">Huang, Zhifeng; Chen, Huaibin; Blais, Steven; Neubert, Thomas A.; Li, Xiaokun; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1889-1896</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The K650E gain-of-function mutation in the tyrosine kinase domain of FGF receptor 3 (FGFR3) causes Thanatophoric Dysplasia type II, a neonatal lethal congenital dwarfism syndrome, and when acquired somatically, it contributes to carcinogenesis.  In this report, we det. the crystal structure of the FGFR3 kinase domain harboring this pathogenic mutation and show that the mutation introduces a network of intramol. hydrogen bonds to stabilize the active-state conformation.  In the crystal, the mutant FGFR3 kinases are caught in the act of trans-phosphorylation on a kinase insert autophosphorylation site, emphasizing the fact that the K650E mutation circumvents the requirement for A-loop tyrosine phosphorylation in kinase activation.  Anal. of this trans-phosphorylation complex sheds light onto the determinants of tyrosine trans-phosphorylation specificity.  We propose that the targeted inhibition of this pathogenic FGFR3 kinase may be achievable by small mol. kinase inhibitors that selectively bind the active-state conformation of FGFR3 kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNzI66rWB_urVg90H21EOLACvtfcHk0liGzgjiqIyFGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Slu7rL&md5=f555b8b61c93500f436f1cee396b1aba</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DBlais%26aufirst%3DS.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DStructural%2520mimicry%2520of%2520a-loop%2520tyrosine%2520phosphorylation%2520by%2520a%2520pathogenic%2520FGF%2520receptor%25203%2520mutation%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1889%26epage%3D1896%26doi%3D10.1016%2Fj.str.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorissen, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">D198</span>– <span class="NLM_lpage">D201</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=17145705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+a+web-accessible+database+of+experimentally+determined+protein-ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span></div><div class="casAuthors">Liu, Tiqing; Lin, Yuhmei; Wen, Xin; Jorissen, Robert N.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D198-D201</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB is a publicly accessible database currently contg. ∼20 000 exptl. detd. binding affinities of protein-ligand complexes, for 110 protein targets including isoforms and mutational variants, and ∼11 000 small mol. ligands.  The data are extd. from the scientific literature, data collection focusing on proteins that are drug-targets or candidate drug-targets and for which structural data are present in the Protein Data Bank.  The BindingDB website supports a range of query types, including searches by chem. structure, substructure and similarity; protein sequence; ligand and protein names; affinity ranges and mol. wt.  Data sets generated by BindingDB queries can be downloaded in the form of annotated SDfiles for further anal., or used as the basis for virtual screening of a compd. database uploaded by the user.  The data in BindingDB are linked both to structural data in the PDB via PDB IDs and chem. and sequence searches, and to the literature in PubMed via PubMed IDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymy3G_LuG-rVg90H21EOLACvtfcHk0liGzgjiqIyFGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D&md5=0ccb20d9b9178a624d4829b5909e7ff8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein-ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.%0AA.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0liGzgjiqIyFGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Knowledge-based scoring functions in drug design: 2. can the knowledge base be enriched?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">386</span>– <span class="NLM_lpage">397</span>, <span class="refDoi"> DOI: 10.1021/ci100343j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100343j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFSm" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=386-397&author=Q.+Shenauthor=B.+Xiongauthor=M.+Zhengauthor=X.+Luoauthor=C.+Luoauthor=X.+Liuauthor=Y.+Duauthor=J.+Liauthor=W.+Zhuauthor=J.+Shenauthor=H.+Jiang&title=Knowledge-based+scoring+functions+in+drug+design%3A+2.+can+the+knowledge+base+be+enriched%3F&doi=10.1021%2Fci100343j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-Based Scoring Functions in Drug Design: 2. Can the Knowledge Base Be Enriched?</span></div><div class="casAuthors">Shen, Qiancheng; Xiong, Bing; Zheng, Mingyue; Luo, Xiaomin; Luo, Cheng; Liu, Xian; Du, Yun; Li, Jing; Zhu, Weiliang; Shen, Jingkang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">386-397</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fast and accurate predicting of the binding affinities of large sets of diverse protein-ligand complexes is an important, yet extremely challenging, task in drug discovery.  The development of knowledge-based scoring functions exploiting structural information of known protein-ligand complexes represents a valuable contribution to such a computational prediction.  In this study, we report a scoring function named IPMF that integrates addnl. exptl. binding affinity information into the extd. potentials, on the assumption that a scoring function with the "enriched" knowledge base may achieve increased accuracy in binding affinity prediction.  In our approach, the functions and atom types of PMF04 were inherited to implicitly capture binding effects that are hard to model explicitly, and a novel iteration device was designed to gradually tailor the initial potentials.  We evaluated the performance of the resultant IPMF with a diverse set of 219 protein-ligand complexes and compared it with seven scoring functions commonly used in computer-aided drug design, including GLIDE, AutoDock4, VINA, PLP, LUDI, PMF, and PMF04.  While the IPMF is only moderately successful in ranking native or near native conformations, it yields the lowest mean error of 1.41 log Ki/Kd units from measured inhibition affinities and the highest Pearson's correlation coeff. of Rp2 0.40 for the test set.  These results corroborate our initial supposition about the role of "enriched" knowledge base.  With the rapid growing vol. of high-quality structural and interaction data in the public domain, this work marks a pos. step toward improving the accuracy of knowledge-based scoring functions in binding affinity prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVKikMRxKKibVg90H21EOLACvtfcHk0lgDeW6n8nG82w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFSm&md5=5e2c7f13dd7fdcb1fcca5527a6e70f7e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci100343j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100343j%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DKnowledge-based%2520scoring%2520functions%2520in%2520drug%2520design%253A%25202.%2520can%2520the%2520knowledge%2520base%2520be%2520enriched%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D386%26epage%3D397%26doi%3D10.1021%2Fci100343j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. J.</span></span> <span> </span><span class="NLM_article-title">LIBSVM: a library for support vector machines</span>. <i>ACM Trans. Intell. Syst. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1145/1961189.1961199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1145%2F1961189.1961199" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=1-27&author=C.-C.+Changauthor=C.+J.+Lin&title=LIBSVM%3A+a+library+for+support+vector+machines&doi=10.1145%2F1961189.1961199"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1145%2F1961189.1961199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F1961189.1961199%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.-C.%26aulast%3DLin%26aufirst%3DC.%2BJ.%26atitle%3DLIBSVM%253A%2520a%2520library%2520for%2520support%2520vector%2520machines%26jtitle%3DACM%2520Trans.%2520Intell.%2520Syst.%2520Technol.%26date%3D2011%26volume%3D2%26spage%3D1%26epage%3D27%26doi%3D10.1145%2F1961189.1961199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guyon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnhill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vapnik, V.</span></span> <span> </span><span class="NLM_article-title">Gene selection for cancer classification using support vector machines</span>. <i>Mach. Learn.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1023/a:1012487302797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1023%2FA%3A1012487302797" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=389-422&author=I.+Guyonauthor=J.+Westonauthor=S.+Barnhillauthor=V.+Vapnik&title=Gene+selection+for+cancer+classification+using+support+vector+machines&doi=10.1023%2Fa%3A1012487302797"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1023%2FA%3A1012487302797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1012487302797%26sid%3Dliteratum%253Aachs%26aulast%3DGuyon%26aufirst%3DI.%26aulast%3DWeston%26aufirst%3DJ.%26aulast%3DBarnhill%26aufirst%3DS.%26aulast%3DVapnik%26aufirst%3DV.%26atitle%3DGene%2520selection%2520for%2520cancer%2520classification%2520using%2520support%2520vector%2520machines%26jtitle%3DMach.%2520Learn.%26date%3D2002%26volume%3D46%26spage%3D389%26epage%3D422%26doi%3D10.1023%2Fa%3A1012487302797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgDeW6n8nG82w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lhFYPmh7Ak5IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lhFYPmh7Ak5IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Progress with covalent small-molecule kinase inhibitors</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">735</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.01.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.drudis.2018.01.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=29337202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=727-735&author=Z.+Zhaoauthor=P.+E.+Bourne&title=Progress+with+covalent+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.drudis.2018.01.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Progress with covalent small-molecule kinase inhibitors</span></div><div class="casAuthors">Zhao, Zheng; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">With reduced risk of toxicity and high selectivity, covalent small-mol. kinase inhibitors (CSKIs) have emerged rapidly.  Through the lens of structural system pharmacol., here we review this rapid progress by considering design strategies and the challenges and opportunities offered by current CSKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog6l6qMAopYLVg90H21EOLACvtfcHk0lhFYPmh7Ak5IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D&md5=f97a19f53a72044812a778ee9351fe94</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProgress%2520with%2520covalent%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2018%26volume%3D23%26spage%3D727%26epage%3D735%26doi%3D10.1016%2Fj.drudis.2018.01.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Properties of FDA-approved small molecule protein kinase inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.phrs.2019.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=30877063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=19-50&author=R.+Roskoski&title=Properties+of+FDA-approved+small+molecule+protein+kinase+inhibitors&doi=10.1016%2Fj.phrs.2019.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of FDA-approved small molecule protein kinase inhibitors</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19-50</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Because mutations, overexpression, and dysregulation of protein kinases play essential roles in the pathogenesis of many illnesses, this enzyme family has become one of the most important drug targets in the past 20 years.  The US FDA has approved 48 small mol. protein kinase inhibitors, nearly all of which are orally effective with the exceptions of netarsudil (which is given as an eye drop) and temsirolimus (which is given i.v.).  Of the 48 approved drugs, the majority (25) target receptor protein-tyrosine kinases, ten target non-receptor protein-tyrosine kinases, and 13 target protein-serine/threonine protein kinases.  The data indicate that 43 of these drugs are used in the treatment of malignancies (36 against solid tumors including lymphomas and seven against non-solid tumors, e.g., leukemias).  Seven drugs are used in the treatment of non-malignancies: baricitinib, rheumatoid arthritis; fostamatinib, chronic immune thrombocytopenia; ruxolitinib, myelofibrosis and polycythemia vera; nintedanib, idiopathic pulmonary fibrosis; sirolimus, renal graft vs. host disease; netarsudil, glaucoma; tofacitinib, rheumatoid arthritis, Crohn disease, and ulcerative colitis.  Moreover, ibrutinib and sirolimus are used for the treatment of both malignant and non-malignant diseases.  The most common drug targets include ALK, B-Raf, BCR-Abl, epidermal growth factor receptor (EGFR), and vascular endothelial growth factor receptor (VEGFR).  Most of the small mol. inhibitors (45) interact directly with the protein kinase domain.  In contrast, sirolimus, temsirolimus, and everolimus are larger mols. (MW ≈ 1000) that bind to FKBP-12 to generate a complex that inhibits mTOR (mammalian target of rapamycin).  This review presents the available drug-enzyme X-ray crystal structures for 27 of the approved drugs as well as the chem. structures and physicochem. properties of all of the FDA-approved small mol. protein kinase antagonists.  Six of the drugs bind covalently and irreversibly to their target.  Twenty of the 48 drugs have mol. wts. greater than 500, exceeding a Lipinski rule of five criterion.  Excluding the macrolides (everolimus, sirolimus, temsirolimus), the av. mol. wt. of drugs is 480 with a range of 306 (ruxolitinib) to 615 (trametinib).  Nearly half of the antagonists (23) have a lipophilic efficiency with values of less than five while the recommended optima range from 5-10.  One of the vexing problems is the near universal development of resistance that is assocd. with the use of small mol. protein kinase inhibitors for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLum_hDK00hbVg90H21EOLACvtfcHk0lir_0SGKglt3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D&md5=a24dee87603bc4ea12bd34616e46f53d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DProperties%2520of%2520FDA-approved%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D144%26spage%3D19%26epage%3D50%26doi%3D10.1016%2Fj.phrs.2019.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lir_0SGKglt3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bliven, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Determining cysteines available for covalent inhibition across the human kinome</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2879</span>– <span class="NLM_lpage">2889</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2879-2889&author=Z.+Zhaoauthor=Q.+Liuauthor=S.+Blivenauthor=L.+Xieauthor=P.+E.+Bourne&title=Determining+cysteines+available+for+covalent+inhibition+across+the+human+kinome&doi=10.1021%2Facs.jmedchem.6b01815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Determining Cysteines Available for Covalent Inhibition Across the Human Kinome</span></div><div class="casAuthors">Zhao, Zheng; Liu, Qingsong; Bliven, Spencer; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2879-2889</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalently bound protein kinase inhibitors have been frequently designed to target non-catalytic Cys residues at the ATP binding site.  Thus, it is important to know if a given Cys residue can form a covalent bond.  Here, we combined a function-site interaction fingerprint method and DFT calcns. to det. the potential of Cys residues to form a covalent interaction with an inhibitor.  By harnessing the human structural kinome, a comprehensive structure-based binding site Cys dataset was assembled.  The orientation of the Cys SH group indicated which Cys residues could potentially form covalent bonds.  These covalent inhibitor accessible Cys residues were located within 5 kinase regions (P-loop, roof of pocket, front pocket, catalytic-2 of the catalytic loop and DFG-3 close to the DFG peptide).  In an independent test set, these Cys residues covered 95% of covalent kinase inhibitors.  Thus, this study provides new insights into Cys reactivity and preference which is important for the prospective development of covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKJnGGcKhC5rVg90H21EOLACvtfcHk0lir_0SGKglt3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVGjsLg%253D&md5=22ea95ce38eb0836d32c1332dba54fad</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01815%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DBliven%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetermining%2520cysteines%2520available%2520for%2520covalent%2520inhibition%2520across%2520the%2520human%2520kinome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2879%26epage%3D2889%26doi%3D10.1021%2Facs.jmedchem.6b01815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A structure-guided approach to creating covalent FGFR inhibitors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.chembiol.2010.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20338520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=285-295&author=W.+Zhouauthor=W.+Hurauthor=U.+McDermottauthor=A.+Duttauthor=W.+Xianauthor=S.+B.+Ficarroauthor=J.+Zhangauthor=S.+V.+Sharmaauthor=J.+Bruggeauthor=M.+Meyersonauthor=J.+Settlemanauthor=N.+S.+Gray&title=A+structure-guided+approach+to+creating+covalent+FGFR+inhibitors&doi=10.1016%2Fj.chembiol.2010.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Structure-Guided Approach to Creating Covalent FGFR Inhibitors</span></div><div class="casAuthors">Zhou, Wenjun; Hur, Wooyoung; McDermott, Ultan; Dutt, Amit; Xian, Wa; Ficarro, Scott B.; Zhang, Jianming; Sharma, Sreenath V.; Brugge, Joan; Meyerson, Matthew; Settleman, Jeffrey; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">285-295</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The fibroblast growth factor receptor tyrosine kinases (FGFR1, 2, 3, and 4) represent promising therapeutic targets in a no. of cancers.  We have developed the first potent and selective irreversible inhibitor of FGFR1, 2, 3, and 4, which we named FIIN-1 that forms a covalent bond with cysteine 486 located in the P loop of the FGFR1 ATP binding site.  We demonstrated that the inhibitor potently inhibits Tel-FGFR1-transformed Ba/F3 cells (EC50 = 14 nM) as well as numerous FGFR-dependent cancer cell lines.  A biotin-derivatized version of the inhibitor, FIIN-1-biotin, was shown to covalently label FGFR1 at Cys486.  FIIN-1 is a useful probe of FGFR-dependent cellular phenomena and may provide a starting point of the development of therapeutically relevant irreversible inhibitors of wild-type and drug-resistant forms of FGFR kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqxMZh8QQP47Vg90H21EOLACvtfcHk0lir_0SGKglt3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKhu7s%253D&md5=0b7b084a073b1c6f30f38565a0e4db81</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DDutt%26aufirst%3DA.%26aulast%3DXian%26aufirst%3DW.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DBrugge%26aufirst%3DJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520structure-guided%2520approach%2520to%2520creating%2520covalent%2520FGFR%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D285%26epage%3D295%26doi%3D10.1016%2Fj.chembiol.2010.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Site of metabolism prediction for six biotransformations mediated by cytochromes P450</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1251</span>– <span class="NLM_lpage">1258</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fbioinformatics%2Fbtp140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19286831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlslGgtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1251-1258&author=M.+Zhengauthor=X.+Luoauthor=Q.+Shenauthor=Y.+Wangauthor=Y.+Duauthor=W.+Zhuauthor=H.+Jiang&title=Site+of+metabolism+prediction+for+six+biotransformations+mediated+by+cytochromes+P450&doi=10.1093%2Fbioinformatics%2Fbtp140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Site of metabolism prediction for six biotransformations mediated by cytochromes P450</span></div><div class="casAuthors">Zheng, Mingyue; Luo, Xiaomin; Shen, Qiancheng; Wang, Yong; Du, Yun; Zhu, Weiliang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1251-1258</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">One goal of metabolomics is to define and monitor the entire metabolite complement of a cell, while it is still far from reach since systematic and rapid approaches for detg. the biotransformations of newly discovered metabolites are lacking.  For drug development, such metabolic biotransformation of a new chem. entity (NCE) is of more interest because it may profoundly affect its bioavailability, activity and toxicity profile.  The use of in silico methods to predict the site of metab. (SOM) in phase I cytochromes P 450-mediated reactions is usually a starting point of metabolic pathway studies, which may also assist in the process of drug/lead optimization.  This article reports the Cytochromes P 450 (CYP450)-mediated SOM prediction for the six most important metabolic reactions by incorporating the use of machine learning and semi-empirical quantum chem. calcns.  Non-local models were developed on the basis of a large dataset comprising 1858 metabolic reactions extd. from 1034 heterogeneous chems.  For validation, the overall accuracies of all six reaction types are higher than 0.81, four of which exceed 0.90.  In further receiver operating characteristic (ROC) analyses, each of the SOM model gave a significant area under curve (AUC) value over 0.86, indicating a good predicting power.  An external test was made on a previously published dataset, of which 80% of the exptl. obsd. SOMs can be correctly identified by applying the full set of our SOM models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7cfpdfTECvbVg90H21EOLACvtfcHk0ljKWe3zjX8X4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlslGgtL0%253D&md5=aaab21cdbc9d8c625d3a101d05e1d959</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp140%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DSite%2520of%2520metabolism%2520prediction%2520for%2520six%2520biotransformations%2520mediated%2520by%2520cytochromes%2520P450%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D1251%26epage%3D1258%26doi%3D10.1093%2Fbioinformatics%2Fbtp140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span> <span> </span><span class="NLM_article-title"><i>In silico</i> site of metabolism prediction for human UGT-catalyzed reactions</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">398</span>– <span class="NLM_lpage">405</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Fbioinformatics%2Fbtt681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24273240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=398-405&author=J.+Pengauthor=J.+Luauthor=Q.+Shenauthor=M.+Zhengauthor=X.+Luoauthor=W.+Zhuauthor=H.+Jiangauthor=K.+Chen&title=In+silico+site+of+metabolism+prediction+for+human+UGT-catalyzed+reactions&doi=10.1093%2Fbioinformatics%2Fbtt681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">In silico site of metabolism prediction for human UGT-catalyzed reactions</span></div><div class="casAuthors">Peng, Jianlong; Lu, Jing; Shen, Qiancheng; Zheng, Mingyue; Luo, Xiaomin; Zhu, Weiliang; Jiang, Hualiang; Chen, Kaixian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">398-405</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The human uridine diphosphate-glucuronosyltransferase enzyme family catalyzes the glucuronidation of the glycosyl group of a nucleotide sugar to an acceptor compd. (substrate), which is the most common conjugation pathway that serves to protect the organism from the potential toxicity of xenobiotics.  Moreover, it could affect the pharmacol. profile of a drug.  Therefore, it is important to identify the metabolically labile sites for glucuronidation.  Results: In the present study, we developed four in silico models to predict sites of glucuronidation, for four major sites of metab. functional groups, i.e. aliph. hydroxyl, arom. hydroxyl, carboxylic acid or amino nitrogen, resp.  According to the mechanism of glucuronidation, a series of 'local' and 'global' mol. descriptors characterizing the at. reactivity, bonding strength and phys.-chem. properties were calcd. and selected with a genetic algorithm-based feature selection approach.  The constructed support vector machine classification models show good prediction performance, with the balanced accuracy ranging from 0.88 to 0.96 on test set.  For further validation, our models can successfully identify 84% of exptl. obsd. sites of metabs. for an external test set contg. 54 mols.  Availability and implementation: The software somugt based on our models is available at www.dddc.ac.cn/adme/jlpeng/somugt_win32.zip.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e39b8e8f968ca38a8e8ecd8280cd808d">[email protected]</a> or <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="8ce1f5f6e4e9e2ebccede5e0a2ffe4efe2efa2edefa2efe2">[email protected]</a> Supplementary information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTnU5JiJGKPrVg90H21EOLACvtfcHk0ljKWe3zjX8X4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D&md5=c99109a95f1bb99af1c93a3946a3dd44</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt681%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26atitle%3DIn%2520silico%2520site%2520of%2520metabolism%2520prediction%2520for%2520human%2520UGT-catalyzed%2520reactions%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26spage%3D398%26epage%3D405%26doi%3D10.1093%2Fbioinformatics%2Fbtt681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rydberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaretzki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breneman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, L.</span></span> <span> </span><span class="NLM_article-title">SMARTCyp: A 2D method for prediction of Cytochrome P450-mediated drug metabolism</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1021/ml100016x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100016x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFOqsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=96-100&author=P.+Rydbergauthor=D.+E.+Gloriamauthor=J.+Zaretzkiauthor=C.+Brenemanauthor=L.+Olsen&title=SMARTCyp%3A+A+2D+method+for+prediction+of+Cytochrome+P450-mediated+drug+metabolism&doi=10.1021%2Fml100016x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">SMARTCyp: A 2D Method for Prediction of Cytochrome P450-Mediated Drug Metabolism</span></div><div class="casAuthors">Rydberg, Patrik; Gloriam, David E.; Zaretzki, Jed; Breneman, Curt; Olsen, Lars</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">96-100</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMARTCyp is an in silico method that predicts the sites of cytochrome P 450-mediated metab. of druglike mols.  The method is foremost a reactivity model, and as such, it shows a preference for predicting sites that are metabolized by the cytochrome P 450 3A4 isoform.  SMARTCyp predicts the site of metab. directly from the 2D structure of a mol., without requiring calcn. of electronic properties or generation of 3D structures.  This is a major advantage, because it makes SMARTCyp very fast.  Other advantages are that exptl. data are not a prerequisite to create the model, and it can easily be integrated with other methods to create models for other cytochrome P 450 isoforms.  Benchmarking tests on a database of 394 3A4 substrates show that SMARTCyp successfully identifies at least one metabolic site in the top two ranked positions 76% of the time.  SMARTCyp is available for download at http://www.farma.ku.dk/P 450.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1mh5MwR6547Vg90H21EOLACvtfcHk0likqm7sKk8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFOqsL0%253D&md5=4761de933c2c3cef5da2ffb29ac076d0</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fml100016x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100016x%26sid%3Dliteratum%253Aachs%26aulast%3DRydberg%26aufirst%3DP.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26aulast%3DZaretzki%26aufirst%3DJ.%26aulast%3DBreneman%26aufirst%3DC.%26aulast%3DOlsen%26aufirst%3DL.%26atitle%3DSMARTCyp%253A%2520A%25202D%2520method%2520for%2520prediction%2520of%2520Cytochrome%2520P450-mediated%2520drug%2520metabolism%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D96%26epage%3D100%26doi%3D10.1021%2Fml100016x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneebeli, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">IDSite: an accurate approach to predict P450-mediated drug metabolism</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3829</span>– <span class="NLM_lpage">3845</span>, <span class="refDoi"> DOI: 10.1021/ct200462q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200462q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1CqtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=3829-3845&author=J.+Liauthor=S.+T.+Schneebeliauthor=J.+Bylundauthor=R.+Faridauthor=R.+A.+Friesner&title=IDSite%3A+an+accurate+approach+to+predict+P450-mediated+drug+metabolism&doi=10.1021%2Fct200462q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">IDSite: An Accurate Approach to Predict P450-Mediated Drug Metabolism</span></div><div class="casAuthors">Li, Jianing; Schneebeli, Severin T.; Bylund, Joseph; Farid, Ramy; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3829-3845</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate prediction of drug metab. is crucial for drug design.  Since a large majority of drugs' metab. involves P 450 enzymes, we herein describe a computational approach, IDSite, to predict P 450-mediated drug metab.  To model induced-fit effects, IDSite samples the conformational space with flexible docking in Glide followed by two refinement stages using the Protein Local Optimization Program (PLOP).  Sites of metab. (SOMs) are predicted according to a phys.-based score that evaluates the potential of atoms to react with the catalytic iron center.  As a preliminary test, we present in this paper the prediction of hydroxylation and O-dealkylation sites mediated by CYP2D6 using two different models: a phys.-based simulation model and a modification of this model in which a small no. of parameters are fit to a training set.  Without fitting any parameters to exptl. data, the phys. IDSite scoring recovers 83% of the exptl. observations for 56 compds. with a very low false pos. rate.  With only four fitted parameters, the fitted IDSite was trained with a subset of 36 compds. and successfully applied to the other 20 compds., recovering 94% of the exptl. observations with high sensitivity and specificity for both sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgmJGUujXp57Vg90H21EOLACvtfcHk0likqm7sKk8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1CqtbzP&md5=ab64a2c42b5c5d8bc46a7885c53e4eaf</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct200462q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200462q%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSchneebeli%26aufirst%3DS.%2BT.%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DIDSite%253A%2520an%2520accurate%2520approach%2520to%2520predict%2520P450-mediated%2520drug%2520metabolism%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D3829%26epage%3D3845%26doi%3D10.1021%2Fct200462q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T.
A.</span></span> <span> </span><span class="NLM_article-title">Targeted covalent inhibitors for drug design</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">13408</span>– <span class="NLM_lpage">13421</span>, <span class="refDoi"> DOI: 10.1002/anie.201601091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1002%2Fanie.201601091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=13408-13421&author=T.%0AA.+Baillie&title=Targeted+covalent+inhibitors+for+drug+design&doi=10.1002%2Fanie.201601091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Covalent Inhibitors for Drug Design</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">13408-13421</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In contrast to the traditional mechanism of drug action that relies on the reversible, noncovalent interaction of a ligand with its biol. target, a targeted covalent inhibitor (TCI) is designed such that the initial, reversible assocn. is followed by the formation of a covalent bond between an electrophile on the ligand and a nucleophilic center in the protein.  Although this approach offers a variety of potential benefits (high potency and extended duration of action), concerns over the possible toxicol. consequences of protein haptenization have hindered the development of the TCI concept.  Recently, approaches to mitigate the risk of serious adverse reactions to this new class of agent have emerged, thus stimulating interest in the field and leading to authorization of the first cadre of TCIs to be marketed.  The covalent inhibitor approach is rapidly gaining acceptance as a valuable tool in drug discovery, and is poised to make a major impact on the design of enzyme inhibitors and receptor modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEbjdFrctvNrVg90H21EOLACvtfcHk0likqm7sKk8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN&md5=18bed2ae08dcb4245a42b3070abc6354</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.201601091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201601091%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DTargeted%2520covalent%2520inhibitors%2520for%2520drug%2520design%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D13408%26epage%3D13421%26doi%3D10.1002%2Fanie.201601091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0likqm7sKk8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafer, C. M.</span></span> <span> </span><span class="NLM_article-title">Drug discovery considerations in the development of covalent inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.bmcl.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24314671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=33-39&author=R.+Mahauthor=J.+R.+Thomasauthor=C.+M.+Shafer&title=Drug+discovery+considerations+in+the+development+of+covalent+inhibitors&doi=10.1016%2Fj.bmcl.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery considerations in the development of covalent inhibitors</span></div><div class="casAuthors">Mah, Robert; Thomas, Jason R.; Shafer, Cynthia M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In recent years, the no. of drug candidates with a covalent mechanism of action progressing through clin. trials or being approved by the FDA has increased significantly.  And as interest in covalent inhibitors has increased, the tech. challenges for characterizing and optimizing these inhibitors have become evident.  A no. of new tools have been developed to aid this process, but these have not gained wide-spread use.  This review will highlight a no. of methods and tools useful for prosecuting covalent inhibitor drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7dY63LHbEhLVg90H21EOLACvtfcHk0lhQTC7ZRsUW_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsr3O&md5=9407bcb76e83065f5a56a97b63547ede</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DMah%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DShafer%26aufirst%3DC.%2BM.%26atitle%3DDrug%2520discovery%2520considerations%2520in%2520the%2520development%2520of%2520covalent%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D33%26epage%3D39%26doi%3D10.1016%2Fj.bmcl.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Cesco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufresne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittermaier, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors design and discovery</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.ejmech.2017.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28651155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=96-114&author=S.+De%0ACescoauthor=J.+Kurianauthor=C.+Dufresneauthor=A.+K.+Mittermaierauthor=N.+Moitessier&title=Covalent+inhibitors+design+and+discovery&doi=10.1016%2Fj.ejmech.2017.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors design and discovery</span></div><div class="casAuthors">De Cesco, Stephane; Kurian, Jerry; Dufresne, Caroline; Mittermaier, Anthony K.; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-114</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In the history of therapeutics, covalent drugs occupy a very distinct category.  While representing a significant fraction of the drugs on the market, very few have been deliberately designed to interact covalently with their biol. target.  In this review, the prevalence of covalent drugs will first be briefly covered, followed by an introduction to their mechanisms of action and more detailed discussions of their discovery and the development of safe and efficient covalent enzyme inhibitors.  All stages of a drug discovery program will be covered, from target considerations to lead optimization, strategies to tune reactivity and computational methods.  The goal of this article is to provide an overview of the field and to outline good practices that are needed for the proper assessment and development of covalent inhibitors as well as a good understanding of the potential and limitations of current computational methods for the design of covalent drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeLNDmLtBdB7Vg90H21EOLACvtfcHk0lhQTC7ZRsUW_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI&md5=551c9a9c4bd69d4ad4d9156c315d5a94</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BCesco%26aufirst%3DS.%26aulast%3DKurian%26aufirst%3DJ.%26aulast%3DDufresne%26aufirst%3DC.%26aulast%3DMittermaier%26aufirst%3DA.%2BK.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DCovalent%2520inhibitors%2520design%2520and%2520discovery%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D96%26epage%3D114%26doi%3D10.1016%2Fj.ejmech.2017.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamuya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBlanc, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehn, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberger, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">6-Substituted-4-(3-bromophenylamino)quinazolines as putative irreversible inhibitors of the epidermal growth factor receptor (EGFR) and human epidermal growth factor receptor (HER-2) tyrosine kinases with enhanced antitumor activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2734</span>, <span class="refDoi"> DOI: 10.1021/jm0005555</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0005555" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=2719-2734&author=H.-R.+Tsouauthor=N.+Mamuyaauthor=B.+D.+Johnsonauthor=M.+F.+Reichauthor=B.+C.+Gruberauthor=F.+Yeauthor=R.+Nilakantanauthor=R.+Shenauthor=C.+Discafaniauthor=R.+DeBlancauthor=R.+Davisauthor=F.+E.+Koehnauthor=L.+M.+Greenbergerauthor=Y.-F.+Wangauthor=A.+Wissner&title=6-Substituted-4-%283-bromophenylamino%29quinazolines+as+putative+irreversible+inhibitors+of+the+epidermal+growth+factor+receptor+%28EGFR%29+and+human+epidermal+growth+factor+receptor+%28HER-2%29+tyrosine+kinases+with+enhanced+antitumor+activity&doi=10.1021%2Fjm0005555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">6-Substituted-4-(3-bromophenylamino)quinazolines as Putative Irreversible Inhibitors of the Epidermal Growth Factor Receptor (EGFR) and Human Epidermal Growth Factor Receptor (HER-2) Tyrosine Kinases with Enhanced Antitumor Activity</span></div><div class="casAuthors">Tsou, Hwei-Ru; Mamuya, Nellie; Johnson, Bernard D.; Reich, Marvin F.; Gruber, Brian C.; Ye, Fei; Nilakantan, Ramaswamy; Shen, Ru; Discafani, Carolyn; DeBlanc, Ronald; Davis, Rachel; Koehn, Frank E.; Greenberger, Lee M.; Wang, Yu-Fen; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2719-2734</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 6-substituted-4-(3-bromophenylamino)quinazoline derivs. that may function as irreversible inhibitors of epidermal growth factor receptor (EGFR) and human epidermal growth factor receptor (HER-2) tyrosine kinases have been prepd.  These inhibitors have, at the C-6 position, butynamide, crotonamide, and methacrylamide Michael acceptors bearing water-solubilizing substituents.  These compds. were prepd. by acylation of 6-amino-4-(3-bromophenylamino)quinazoline with unsatd. acid chlorides or mixed anhydrides.  We show that attaching a basic functional group onto the Michael acceptor results in greater reactivity, due to intramol. catalysis of the Michael addn. and/or an inductive effect of the protonated basic group. This, along with improved water soly., results in compds. with enhanced biol. properties.  We present mol. modeling and exptl. evidence that these inhibitors interact covalently with the target enzymes.  One compd., (I) was shown to have excellent oral activity in a human epidermoid carcinoma (A431) xenograft model in nude mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVl-zt4kiGDrVg90H21EOLACvtfcHk0lhQTC7ZRsUW_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFChsb8%253D&md5=989e6e77195324d92e70ca13f532e1cf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm0005555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0005555%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DMamuya%26aufirst%3DN.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DGruber%26aufirst%3DB.%2BC.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DDeBlanc%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DKoehn%26aufirst%3DF.%2BE.%26aulast%3DGreenberger%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3D6-Substituted-4-%25283-bromophenylamino%2529quinazolines%2520as%2520putative%2520irreversible%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520and%2520human%2520epidermal%2520growth%2520factor%2520receptor%2520%2528HER-2%2529%2520tyrosine%2520kinases%2520with%2520enhanced%2520antitumor%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D2719%26epage%3D2734%26doi%3D10.1021%2Fjm0005555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furman, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutre, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burger, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wierda, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerema, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukbuntherng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clow, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedrick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buggy, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, S.</span></span> <span> </span><span class="NLM_article-title">Targeting BTK with ibrutinib in relapsed chronic lymphocytic leukemia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1215637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1056%2Fnejmoa1215637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23782158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFygu7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2013&pages=32-42&author=J.+C.+Byrdauthor=R.+R.+Furmanauthor=S.+E.+Coutreauthor=I.+W.+Flinnauthor=J.+A.+Burgerauthor=K.+A.+Blumauthor=B.+Grantauthor=J.+P.+Sharmanauthor=M.+Colemanauthor=W.+G.+Wierdaauthor=J.+A.+Jonesauthor=W.+Zhaoauthor=N.+A.+Heeremaauthor=A.+J.+Johnsonauthor=J.+Sukbuntherngauthor=B.+Y.+Changauthor=F.+Clowauthor=E.+Hedrickauthor=J.+J.+Buggyauthor=D.+F.+Jamesauthor=S.+O%E2%80%99Brien&title=Targeting+BTK+with+ibrutinib+in+relapsed+chronic+lymphocytic+leukemia&doi=10.1056%2Fnejmoa1215637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BTK with ibrutinib in relapsed chronic lymphocytic leukemia</span></div><div class="casAuthors">Byrd, John C.; Furman, Richard R.; Coutre, Steven E.; Flinn, Ian W.; Burger, Jan A.; Blum, Kristie A.; Grant, Barbara; Sharman, Jeff P.; Coleman, Morton; Wierda, William G.; Jones, Jeffrey A.; Zhao, Weiqiang; Heerema, Nyla A.; Johnson, Amy J.; Sukbuntherng, Juthamas; Chang, Betty Y.; Clow, Fong; Hedrick, Eric; Buggy, Joseph J.; James, Danelle F.; O'Brien, Susan</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-42</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background The treatment of relapsed chronic lymphocytic leukemia (CLL) has resulted in few durable remissions.  Bruton's tyrosine kinase (BTK), an essential component of B-cell-receptor signaling, mediates interactions with the tumor microenvironment and promotes the survival and proliferation of CLL cells.  Methods We conducted a phase 1b-2 multicenter study to assess the safety, efficacy, pharmacokinetics, and pharmacodynamics of ibrutinib (PCI-32765), a first-in-class, oral covalent inhibitor of BTK designed for treatment of B-cell cancers, in patients with relapsed or refractory CLL or small lymphocytic lymphoma.  A total of 85 patients, the majority of whom were considered to have high-risk disease, received ibrutinib orally once daily; 51 received 420 mg, and 34 received 840 mg.  Results Toxic effects were predominantly grade 1 or 2 and included transient diarrhea, fatigue, and upper respiratory tract infection; thus, patients could receive extended treatment with minimal hematol. toxic effects.  The overall response rate was the same in the group that received 420 mg and the group that received 840 mg (71%), and an addnl. 20% and 15% of patients in the resp. groups had a partial response with lymphocytosis.  The response was independent of clin. and genomic risk factors present before treatment, including advanced-stage disease, the no. of previous therapies, and the 17p13.1 deletion.  At 26 mo, the estd. progression-free survival rate was 75% and the rate of overall survival was 83%.  Conclusions Ibrutinib was assocd. with a high frequency of durable remissions in patients with relapsed or refractory CLL and small lymphocytic lymphoma, including patients with high-risk genetic lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1EhPQxMRz7rVg90H21EOLACvtfcHk0liVYFpzLi6Ylg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFygu7zN&md5=32a5eb6ab06beeecf64d9ada84139925</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1056%2Fnejmoa1215637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmoa1215637%26sid%3Dliteratum%253Aachs%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DFurman%26aufirst%3DR.%2BR.%26aulast%3DCoutre%26aufirst%3DS.%2BE.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DBurger%26aufirst%3DJ.%2BA.%26aulast%3DBlum%26aufirst%3DK.%2BA.%26aulast%3DGrant%26aufirst%3DB.%26aulast%3DSharman%26aufirst%3DJ.%2BP.%26aulast%3DColeman%26aufirst%3DM.%26aulast%3DWierda%26aufirst%3DW.%2BG.%26aulast%3DJones%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DHeerema%26aufirst%3DN.%2BA.%26aulast%3DJohnson%26aufirst%3DA.%2BJ.%26aulast%3DSukbuntherng%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DB.%2BY.%26aulast%3DClow%26aufirst%3DF.%26aulast%3DHedrick%26aufirst%3DE.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26aulast%3DJames%26aufirst%3DD.%2BF.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%26atitle%3DTargeting%2520BTK%2520with%2520ibrutinib%2520in%2520relapsed%2520chronic%2520lymphocytic%2520leukemia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D369%26spage%3D32%26epage%3D42%26doi%3D10.1056%2Fnejmoa1215637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span> <i>FDA Imbruvica Review</i>; <span class="NLM_publisher-name">U.S. Food&Drug
Administration</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+FDA+Imbruvica+Review%3B+U.S.+Food%26Drug%0AAdministration%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DFDA%2520Imbruvica%2520Review%26pub%3DU.S.%2520Food%2526Drug%250AAdministration%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dungo, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, G. M.</span></span> <span> </span><span class="NLM_article-title">Afatinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1007/s40265-013-0111-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1007%2Fs40265-013-0111-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=23982599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1503-1515&author=R.+T.+Dungoauthor=G.+M.+Keating&title=Afatinib%3A+first+global+approval&doi=10.1007%2Fs40265-013-0111-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib: First Global Approval</span></div><div class="casAuthors">Dungo, Rosselle T.; Keating, Gillian M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1503-1515</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Afatinib, an irreversible inhibitor of the ErbB family of tyrosine kinases, is under development with Boehringer Ingelheim for the once-daily, oral treatment of cancer.  Afatinib downregulates ErbB signalling by covalently binding to epidermal growth factor receptor (EGFR), human epidermal growth factor receptor (HER) 2 and HER4, irreversibly inhibiting tyrosine kinase autophosphorylation.  It also inhibits transphosphorylation of HER3.  Oral afatinib (Gilotrif®) has been approved in the US for the first-line treatment of patients with metastatic non-small-cell lung cancer (NSCLC) who have tumors with EGFR exon 19 deletions or exon 21 (L858R) substitution mutations as detected by a US FDA-approved test.  Afatinib has also been approved in Taiwan for the first-line treatment of patients with EGFR mutation-pos. NSCLC.  In addn., the European Medicines Agency's Committee for Medicinal Products for Human Use has recommended the approval of afatinib (Giotrif) for the treatment of patients with locally advanced or metastatic NSCLC with activating EGFR mutations who are EGFR tyrosine kinase inhibitor naive.  Afatinib is also under regulatory review in Canada, Japan and other Asian countries.  This article summarizes the milestones in the development of afatinib, leading to this first approval in patients with metastatic NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdPcU5aIqrY7Vg90H21EOLACvtfcHk0liVYFpzLi6Ylg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM&md5=21851cbfac56e2fffcba6c731f123cff</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0111-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0111-6%26sid%3Dliteratum%253Aachs%26aulast%3DDungo%26aufirst%3DR.%2BT.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1503%26epage%3D1515%26doi%3D10.1007%2Fs40265-013-0111-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strelevitz, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zientek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfield, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowty, M. E.</span></span> <span> </span><span class="NLM_article-title">Clearance prediction of targeted covalent inhibitors by <i>in vitro-in vivo</i> extrapolation of hepatic and extrahepatic clearance mechanisms</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1124/dmd.116.072983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1124%2Fdmd.116.072983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27784718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlSksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=1-7&author=L.+Leungauthor=X.+Yangauthor=T.+J.+Strelevitzauthor=J.+Montgomeryauthor=M.+F.+Brownauthor=M.+A.+Zientekauthor=C.+Banfieldauthor=A.+M.+Gilbertauthor=A.+Thorarensenauthor=M.+E.+Dowty&title=Clearance+prediction+of+targeted+covalent+inhibitors+by+in+vitro-in+vivo+extrapolation+of+hepatic+and+extrahepatic+clearance+mechanisms&doi=10.1124%2Fdmd.116.072983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Clearance prediction of targeted covalent inhibitors by in vitro-in vivo extrapolation of hepatic and extrahepatic clearance mechanisms</span></div><div class="casAuthors">Leung, Louis; Yang, Xin; Strelevitz, Timothy J.; Montgomery, Justin; Brown, Matthew F.; Zientek, Michael A.; Banfield, Christopher; Gilbert, Adam M.; Thorarensen, Atli; Dowty, Martin E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The concept of target-specific covalent enzyme inhibitors appears attractive from both an efficacy and a selectivity viewpoint considering the potential for enhanced biochem. efficiency assocd. with an irreversible mechanism.  Aside from potential safety concerns, clearance prediction of covalent inhibitors represents a unique challenge due to the inclusion of nontraditional metabolic pathways of direct conjugation with glutathione (GSH) or via GSH S-transferase-mediated processes.  In this article, a novel pharmacokinetic algorithm was developed using a series of Pfizer kinase selective acrylamide covalent inhibitors based on their in vitro-in vivo extrapolation of systemic clearance in rats.  The algorithmencompasses the use of hepatocytes as an in vitro model for hepatic clearance due to oxidative metab. and GSH conjugation, and the use of whole blood as an in vitro surrogate for GSH conjugation in extrahepatic tissues.  Initial evaluations with clin. covalent inhibitors suggested that the scaling algorithm developed from rats may also be useful for human clearance prediction when species-specific parameters, such as hepatocyte and blood stability and blood binding, were considered.  With careful consideration of clearance mechanisms, the described in vitro-in vivo extrapolation approach may be useful to facilitate candidate optimization, selection, and prediction of human pharmacokinetic clearance during the discovery and development of targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvIMBCrCxzNLVg90H21EOLACvtfcHk0lgrj37Z46icJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlSksL4%253D&md5=8c8b918b087278b2025e0ae56a7abe37</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.072983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.072983%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DStrelevitz%26aufirst%3DT.%2BJ.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DZientek%26aufirst%3DM.%2BA.%26aulast%3DBanfield%26aufirst%3DC.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DDowty%26aufirst%3DM.%2BE.%26atitle%3DClearance%2520prediction%2520of%2520targeted%2520covalent%2520inhibitors%2520by%2520in%2520vitro-in%2520vivo%2520extrapolation%2520of%2520hepatic%2520and%2520extrahepatic%2520clearance%2520mechanisms%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2017%26volume%3D45%26spage%3D1%26epage%3D7%26doi%3D10.1124%2Fdmd.116.072983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacolod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturedi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apuy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, A.</span></span> <span> </span><span class="NLM_article-title">A proposed screening paradigm for discovery of covalent inhibitor drugs</span>. <i>Drug Metab. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.2174/1872312808666140317151735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.2174%2F1872312808666140317151735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24628405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGqur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=19-30&author=M.+Moghaddamauthor=Y.+Tangauthor=Z.+O%E2%80%99Brienauthor=S.+Richardsonauthor=M.+Bacolodauthor=P.+Chaturediauthor=J.+Apuyauthor=A.+Kulkarni&title=A+proposed+screening+paradigm+for+discovery+of+covalent+inhibitor+drugs&doi=10.2174%2F1872312808666140317151735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A Proposed Screening Paradigm for Discovery of Covalent Inhibitor Drugs</span></div><div class="casAuthors">Moghaddam, Mehran F.; Tang, Yang; O'Brien, Zhihong; Richardson, Samantha J.; Bacolod, Maria; Chaturedi, Prasoon; Apuy, Julius; Kulkarni, Ashutosh</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-30</span>CODEN:
                <span class="NLM_cas:coden">DMLRBM</span>;
        ISSN:<span class="NLM_cas:issn">1872-3128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The in vitro and in vivo preclin. ADME properties of 10 clin. late stage or marketed covalent inhibitors were evaluated in order to define advancement criteria for discovery of future drugs in this arena.  Our studies revealed the following: After incubating with S9 fractions for 30 min, the rat and human in vitro stability for these compds. ranged from 1% to 100%.  The blood stability ranged from 30% to 100%.  There was a broad range of CYP inhibition with prevalence for time-dependent inhibition of at least one enzyme.  The Caco-2 permeability (A→B) ranged from negligible (0.6 x 10-6 cm/s) to highly permeable (31 x 10-6 cm/s) and the efflux ratio also varied widely (0.2-30).  Most of the compds. were highly protein bound in both rat and human with binding ≥ 90%.  Rat plasma clearance for the 10 compds. ranged from slow (11 mL/min/kg) to very rapid (350 mL/min/kg).  The Vss ranged from low (0.67 L/kg) to very high (115 L/kg).  MRT's also ranged from short (0.5 h) to long (7.4 h).  The oral exposures also showed a very broad range with Cmax's ranging from 0.01-77 μM and exposure levels ranging from 0.03-106 μM.hr.  In conclusion, the wide range in in vitro and in vivo ADME data makes these particular ADME assays non-discriminatory in the selection of promising compds.  In our opinion, non-traditional assays such as target mass modification, target confirmation by amino acid sequencing, cellular target occupancy, and target turnover rate data in combination with the pharmacokinetic profiles are the crit. considerations for progression of irreversible compds. in early discovery. ; <b>Biog.</b>: Mehran Moghaddam, PhD, is.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobrQbTwxt6f7Vg90H21EOLACvtfcHk0lgrj37Z46icJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGqur%252FF&md5=97ff5ba47e9547cbae3e5d10ec35112e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.2174%2F1872312808666140317151735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1872312808666140317151735%26sid%3Dliteratum%253Aachs%26aulast%3DMoghaddam%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DZ.%26aulast%3DRichardson%26aufirst%3DS.%26aulast%3DBacolod%26aufirst%3DM.%26aulast%3DChaturedi%26aufirst%3DP.%26aulast%3DApuy%26aufirst%3DJ.%26aulast%3DKulkarni%26aufirst%3DA.%26atitle%3DA%2520proposed%2520screening%2520paradigm%2520for%2520discovery%2520of%2520covalent%2520inhibitor%2520drugs%26jtitle%3DDrug%2520Metab.%2520Lett.%26date%3D2014%26volume%3D8%26spage%3D19%26epage%3D30%26doi%3D10.2174%2F1872312808666140317151735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J. M.</span></span> <span> </span><span class="NLM_article-title">A perspective on the kinetics of covalent and irreversible inhibition</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&author=J.+M.+Strelow&title=A+perspective+on+the+kinetics+of+covalent+and+irreversible+inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0lgrj37Z46icJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520perspective%2520on%2520the%2520kinetics%2520of%2520covalent%2520and%2520irreversible%2520inhibition%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venetsanakos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaStant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughhead, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, J. O.</span></span> <span> </span><span class="NLM_article-title">Discovery of the irreversible covalent FGFR inhibitor 8-(3-(4-Acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (PRN1371) for the Treatment of Solid Tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6516</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6516-6527&author=K.+A.+Brameldauthor=T.+D.+Owensauthor=E.+Vernerauthor=E.+Venetsanakosauthor=J.+M.+Bradshawauthor=V.+T.+Phanauthor=D.+Tamauthor=K.+Leungauthor=J.+Shuauthor=J.+LaStantauthor=D.+G.+Loughheadauthor=T.+Tonauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=D.+M.+Goldsteinauthor=J.+O.+Funk&title=Discovery+of+the+irreversible+covalent+FGFR+inhibitor+8-%283-%284-Acryloylpiperazin-1-yl%29propyl%29-6-%282%2C6-dichloro-3%2C5-dimethoxyphenyl%29-2-%28methylamino%29pyrido%5B2%2C3-d%5Dpyrimidin-7%288H%29-one+%28PRN1371%29+for+the+Treatment+of+Solid+Tumors&doi=10.1021%2Facs.jmedchem.7b00360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the Irreversible Covalent FGFR Inhibitor 8-(3-(4-Acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-d]pyrimidin-7(8H)-one (PRN1371) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Brameld, Ken A.; Owens, Timothy D.; Verner, Erik; Venetsanakos, Eleni; Bradshaw, J. Michael; Phan, Vernon T.; Tam, Danny; Leung, Kwan; Shu, Jin; LaStant, Jacob; Loughhead, David G.; Ton, Tony; Karr, Dane E.; Gerritsen, Mary E.; Goldstein, David M.; Funk, Jens Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6516-6527</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant signaling of the FGF/FGFR pathway occurs frequently in cancers and is an oncogenic driver in many solid tumors.  Clin. validation of FGFR as a therapeutic target has been demonstrated in bladder, liver, lung, breast, and gastric cancers.  Our goal was to develop an irreversible covalent inhibitor of FGFR1-4 for use in oncol. indications.  An irreversible covalent binding mechanism imparts many desirable pharmacol. benefits including high potency, selectivity, and prolonged target inhibition.  Herein we report the structure-based design, medicinal chem. optimization, and unique ADME assays of our irreversible covalent drug discovery program which culminated in the discovery of compd. 34 (PRN1371), a highly selective and potent FGFR1-4 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklTuvm1niy7Vg90H21EOLACvtfcHk0lgH6SIOchkJIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK&md5=987069b645d5e0e36ccda5e5c81520b9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00360%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DVenetsanakos%26aufirst%3DE.%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DLaStant%26aufirst%3DJ.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26atitle%3DDiscovery%2520of%2520the%2520irreversible%2520covalent%2520FGFR%2520inhibitor%25208-%25283-%25284-Acryloylpiperazin-1-yl%2529propyl%2529-6-%25282%252C6-dichloro-3%252C5-dimethoxyphenyl%2529-2-%2528methylamino%2529pyrido%255B2%252C3-d%255Dpyrimidin-7%25288H%2529-one%2520%2528PRN1371%2529%2520for%2520the%2520Treatment%2520of%2520Solid%2520Tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6516%26epage%3D6527%26doi%3D10.1021%2Facs.jmedchem.7b00360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolanos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almaden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span> <span> </span><span class="NLM_article-title">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173-178&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+EGFR+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact ≤ 2.1 × 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0lgH6SIOchkJIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520EGFR%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D173%26epage%3D178%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">KinMap: a web-based tool for interactive navigation through human kinome data</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">16</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-1433-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1186%2Fs12859-016-1433-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28056780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=16&author=S.+Eidauthor=S.+Turkauthor=A.+Volkamerauthor=F.+Rippmannauthor=S.+Fulle&title=KinMap%3A+a+web-based+tool+for+interactive+navigation+through+human+kinome+data&doi=10.1186%2Fs12859-016-1433-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">KinMap: a web-based tool for interactive navigation through human kinome data</span></div><div class="casAuthors">Eid, Sameh; Turk, Samo; Volkamer, Andrea; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16/1-16/6</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Annotations of the phylogenetic tree of the human kinome is an intuitive way to visualize compd. profiling data, structural features of kinases or functional relationships within this important class of proteins.  The increasing vol. and complexity of kinase-related data underlines the need for a tool that enables complex queries pertaining to kinase disease involvement and potential therapeutic uses of kinase inhibitors.  Results: Here, we present KinMap, a user-friendly online tool that facilitates the interactive navigation through kinase knowledge by linking biochem., structural, and disease assocn. data to the human kinome tree.  To this end, preprocessed data from freely-available sources, such as ChEMBL, the Protein Data Bank, and the Center for Therapeutic Target Validation platform are integrated into KinMap and can easily be complemented by proprietary data.  The value of KinMap will be exemplarily demonstrated for uncovering new therapeutic indications of known kinase inhibitors and for prioritizing kinases for drug development efforts.  Conclusions: KinMap represents a new generation of kinome tree viewers which facilitates interactive exploration of the human kinome.  KinMap enables generation of high-quality annotated images of the human kinome tree as well as exchange of kinome-related data in scientific communications.  Furthermore, KinMap supports multiple input and output formats and recognizes alternative kinase names and links them to a unified naming scheme, which makes it a useful tool across different disciplines and applications.  A web-service of KinMap is freely available at http://www.kinhub.org/kinmap/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1M2aa8fuKALVg90H21EOLACvtfcHk0lgH6SIOchkJIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D&md5=29737c7cda8d9caf75018e0d29e9a459</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-1433-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-1433-7%26sid%3Dliteratum%253Aachs%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DKinMap%253A%2520a%2520web-based%2520tool%2520for%2520interactive%2520navigation%2520through%2520human%2520kinome%2520data%26jtitle%3DBMC%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D16%26doi%3D10.1186%2Fs12859-016-1433-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">January, C. T.</span></span> <span> </span><span class="NLM_article-title">Blockage of the hERG human cardiac K<sup>+</sup> channel by the gastrointestinal prokinetic agent cisapride</span>. <i>Am. J. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">H2534</span>– <span class="NLM_lpage">H2538</span>, <span class="refDoi"> DOI: 10.1152/ajpheart.1997.273.5.h2534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1152%2Fajpheart.1997.273.5.h2534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=9374794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1997&pages=H2534-H2538&author=S.+Mohammadauthor=Z.+Zhouauthor=Q.+Gongauthor=C.+T.+January&title=Blockage+of+the+hERG+human+cardiac+K%2B+channel+by+the+gastrointestinal+prokinetic+agent+cisapride&doi=10.1152%2Fajpheart.1997.273.5.h2534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Blockage of the HERG human cardiac K+ channel by the gastrointestinal prokinetic agent cisapride</span></div><div class="casAuthors">Mohammad, Saeed; Zhou, Zhengfeng; Gong, Qiuming; January, Craig T.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Physiology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5, Pt. 2</span>),
    <span class="NLM_cas:pages">H2534-H2538</span>CODEN:
                <span class="NLM_cas:coden">AJPHAP</span>;
        ISSN:<span class="NLM_cas:issn">0002-9513</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Cisapride, a gastrointestinal prokinetic agent, is known to caused long Q-T syndrome and ventricular arrhythmias.  The cellular mechanism is not known.  The human ether-a-go-go-related gene (HERG), which encodes the rapidly activating delayed rectifier K+ current and is important in cardiac repolarization, may serve as a target for the action of cisapride.  The authors tested the hypothesis that cisapride blocks HERG.  The whole cell path-clamp recording technique was used to study HERG channels stably expressed heterologously in HEK293 cells.  Under voltage-clamp conditions, cisapride block of HERG is dose dependent with a half-maximal inhibitory concn. of 6.5 nM at 22° (cells).  Currents rapidly recovered with drug washout.  The onset of block by cisapride required channel activation indicative of open or inactivated state blockage.  Block of HERG with cisapride after channel activation was voltage dependent.  At -20 mV, 10 nM cisapride reduced HERG tail-current amplitude by 5%, whereas, at +20 mV, the tail-current amplitude was reduced by 45% (cells).  At -20 and +20 mV, 100 nM cisapride reduced tail-current amplitude by 66 and 90%, resp.  The authors conclude that cisapride is a potent blocker of HERG channels expressed in HEK293 cells.  This effect may account for the clin. occurrence of Q-T prolongation and ventricular arrhythmias obsd. with cisapride.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNimFBPcs477Vg90H21EOLACvtfcHk0lj3BGDQJctjmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D&md5=291fcfb4f2d9efd92120735a91d27b59</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1152%2Fajpheart.1997.273.5.h2534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpheart.1997.273.5.h2534%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DQ.%26aulast%3DJanuary%26aufirst%3DC.%2BT.%26atitle%3DBlockage%2520of%2520the%2520hERG%2520human%2520cardiac%2520K%252B%2520channel%2520by%2520the%2520gastrointestinal%2520prokinetic%2520agent%2520cisapride%26jtitle%3DAm.%2520J.%2520Physiol.%26date%3D1997%26volume%3D273%26spage%3DH2534%26epage%3DH2538%26doi%3D10.1152%2Fajpheart.1997.273.5.h2534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazic, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dota, C.</span></span> <span> </span><span class="NLM_article-title">An analysis of the relationship between preclinical and clinical QT interval-related data</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">101</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfx125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1093%2Ftoxsci%2Fkfx125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=28903488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2017&pages=94-101&author=C.+E.+Pollardauthor=M.+Skinnerauthor=S.+E.+Lazicauthor=H.+M.+Priorauthor=K.+M.+Conlonauthor=J.-P.+Valentinauthor=C.+Dota&title=An+analysis+of+the+relationship+between+preclinical+and+clinical+QT+interval-related+data&doi=10.1093%2Ftoxsci%2Fkfx125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of the relationship between preclinical and clinical QT interval-related data</span></div><div class="casAuthors">Pollard, Christopher E.; Skinner, Matthew; Lazic, Stanley E.; Prior, Helen M.; Conlon, Kelly M.; Valentin, Jean-Pierre; Dota, Corina</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-101</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">There has been significant focus on drug-induced QT interval prolongation caused by block of the human ether-a-go-corelated gene (hERG)-encoded potassium channel.  Regulatory guidance has been implemented to assess QT interval prolongation risk: preclin. guidance requires a candidate drug potency as a hERG channel blocker to be defined and also its effect on QT interval in a non-rodent species; clin. guidance requires a "Thorough QT Study" during development, although some QT prolonging compds. are identified earlier via a Phase I study.  Clin., heart rate-cor. QT interval (QTc) data on 24 compds. (13 positives; 11 negatives) were compared with their effect on dog QTc and the concn. of compd. causing 50% inhibition (IC50) of hERG current.  Concordance was assessed by calcg. sensitivity and specificity across a range of decision thresholds, thus yielding receiver operating characteristic curves of sensitivity vs. (1-specificity).  The area under the curve of ROC curves (for which 0.5 and 1 indicate chance and perfect concordance, resp.) was used to summarize concordance.  Three aspects of preclin. data were compared with the clin. outcome (receiver operating characteristic area under the curve values shown in brackets): abs. hERG IC50 (0.78); safety margin between hERG IC50 and clin. peak free plasma exposure (0.80); safety margin between QTc effects in dogs and clin. peak free plasma exposure (0.81).  Pos. and neg. predictive values of abs. hERG IIC50 indicated that from an early drug discovery perspective, low potency compds. can be progressed on the basis of a low risk of causing a QTc increase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzer6VhKiwIbVg90H21EOLACvtfcHk0lj3BGDQJctjmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D&md5=442a63490fb478512d52828a9d8d0a08</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfx125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfx125%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DC.%2BE.%26aulast%3DSkinner%26aufirst%3DM.%26aulast%3DLazic%26aufirst%3DS.%2BE.%26aulast%3DPrior%26aufirst%3DH.%2BM.%26aulast%3DConlon%26aufirst%3DK.%2BM.%26aulast%3DValentin%26aufirst%3DJ.-P.%26aulast%3DDota%26aufirst%3DC.%26atitle%3DAn%2520analysis%2520of%2520the%2520relationship%2520between%2520preclinical%2520and%2520clinical%2520QT%2520interval-related%2520data%26jtitle%3DToxicol.%2520Sci.%26date%3D2017%26volume%3D159%26spage%3D94%26epage%3D101%26doi%3D10.1093%2Ftoxsci%2Fkfx125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, B. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheelan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergquist, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becherer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashim, M. A.</span></span> <span> </span><span class="NLM_article-title">Predicting QT prolongation in humans during early drug development using hERG inhibition and an anaesthetized guinea-pig model</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>154</i></span>,  <span class="NLM_fpage">1446</span>– <span class="NLM_lpage">1456</span>, <span class="refDoi"> DOI: 10.1038/bjp.2008.267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fbjp.2008.267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=18587422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVajsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2008&pages=1446-1456&author=X.+Yaoauthor=D.+L.+Andersonauthor=S.+A.+Rossauthor=D.+G.+Langauthor=B.+Z.+Desaiauthor=D.+C.+Cooperauthor=P.+Wheelanauthor=M.+S.+McIntyreauthor=M.+L.+Bergquistauthor=K.+I.+MacKenzieauthor=J.+D.+Bechererauthor=M.+A.+Hashim&title=Predicting+QT+prolongation+in+humans+during+early+drug+development+using+hERG+inhibition+and+an+anaesthetized+guinea-pig+model&doi=10.1038%2Fbjp.2008.267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting QT prolongation in humans during early drug development using hERG inhibition and an anaesthetized guinea-pig model</span></div><div class="casAuthors">Yao, X.; Anderson, D. L.; Ross, S. A.; Lang, D. G.; Desai, B. Z.; Cooper, D. C.; Wheelan, P.; McIntyre, M. S.; Bergquist, M. L.; MacKenzie, K. I.; Becherer, J. D.; Hashim, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1446-1456</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug-induced prolongation of the QT interval can lead to torsade de pointes, a life-threatening ventricular arrhythmia.  Finding appropriate assays from among the plethora of options available to predict reliably this serious adverse effect in humans remains a challenging issue for the discovery and development of drugs.  The purpose of the present study was to develop and verify a reliable and relatively simple approach for assessing, during preclin. development, the propensity of drugs to prolong the QT interval in humans.  Sixteen marketed drugs from various pharmacol. classes with a known incidence-or lack thereof-of QT prolongation in humans were examd. in hERG (human ether a-go-go-related gene) patch-clamp assay and an anesthetized guinea-pig assay for QT prolongation using specific protocols.  Drug concns. in perfusates from hERG assays and plasma samples from guinea-pigs were detd. using liq. chromatog.-mass spectrometry.  Various pharmacol. agents that inhibit hERG currents prolong the QT interval in anesthetized guinea-pigs in a manner similar to that seen in humans and at comparable drug exposures.  Several compds. not assocd. with QT prolongation in humans failed to prolong the QT interval in this model.  Anal. of hERG inhibitory potency in conjunction with drug exposures and QT interval measurements in anesthetized guinea-pigs can reliably predict, during preclin. drug development, the risk of human QT prolongation.  A strategy is proposed for mitigating the risk of QT prolongation of new chem. entities during early lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRuiQ1CyjedrVg90H21EOLACvtfcHk0lj3BGDQJctjmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVajsLw%253D&md5=1dbf4f50957b26496c4efa213e3f7e5c</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fbjp.2008.267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjp.2008.267%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DD.%2BL.%26aulast%3DRoss%26aufirst%3DS.%2BA.%26aulast%3DLang%26aufirst%3DD.%2BG.%26aulast%3DDesai%26aufirst%3DB.%2BZ.%26aulast%3DCooper%26aufirst%3DD.%2BC.%26aulast%3DWheelan%26aufirst%3DP.%26aulast%3DMcIntyre%26aufirst%3DM.%2BS.%26aulast%3DBergquist%26aufirst%3DM.%2BL.%26aulast%3DMacKenzie%26aufirst%3DK.%2BI.%26aulast%3DBecherer%26aufirst%3DJ.%2BD.%26aulast%3DHashim%26aufirst%3DM.%2BA.%26atitle%3DPredicting%2520QT%2520prolongation%2520in%2520humans%2520during%2520early%2520drug%2520development%2520using%2520hERG%2520inhibition%2520and%2520an%2520anaesthetized%2520guinea-pig%2520model%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2008%26volume%3D154%26spage%3D1446%26epage%3D1456%26doi%3D10.1038%2Fbjp.2008.267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ban, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBlanc, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennie, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherkasov, A.</span></span> <span> </span><span class="NLM_article-title">Best practices of computer-aided drug discovery: lessons learned from the development of a preclinical candidate for prostate cancer with a new mechanism of action</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1028</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00137</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00137" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2sXms1Wgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1018-1028&author=F.+Banauthor=K.+Dalalauthor=H.+Liauthor=E.+LeBlancauthor=P.+S.+Rennieauthor=A.+Cherkasov&title=Best+practices+of+computer-aided+drug+discovery%3A+lessons+learned+from+the+development+of+a+preclinical+candidate+for+prostate+cancer+with+a+new+mechanism+of+action&doi=10.1021%2Facs.jcim.7b00137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Best Practices of Computer-Aided Drug Discovery: Lessons Learned from the Development of a Preclinical Candidate for Prostate Cancer with a New Mechanism of Action</span></div><div class="casAuthors">Ban, Fuqiang; Dalal, Kush; Li, Huifang; LeBlanc, Eric; Rennie, Paul S.; Cherkasov, Artem</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1018-1028</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. drug design is a complex and iterative decision-making process relying on pre-existing knowledge and driven by exptl. data.  Low mol. wt. chems. represent an attractive therapeutic option as they are readily accessible to org. synthesis and can easily be characterized. T.  Their potency, as well as pharmacokinetic and pharmacodynamic properties can be systematically and rationally investigated, and ultimately optimized via expert science behind medicinal chem. and methods of computer-aided drug design (CADD).  In recent years, significant advances in mol. modeling techniques afforded a variety of tools to effectively identify potential binding pockets on prospective targets, to map key interactions between ligands and their binding sites, to construct and assess energetics of the resulting complexes, to predict ADMET properties of candidate compds., and to systematically analyze exptl. and computational data to derive meaningful structure-activity relationships leading to the creation of a drug candidate.  This perspective paper describes a real case of a drug discovery campaign accomplished in a relatively short time with limited resources.  The study integrated an arsenal of available mol. modeling techniques with an array of exptl. tools to successfully develop a novel class of potent and selective androgen receptor inhibitors with a novel mode of action.  It resulted in the largest academic licensing deal in Canadian history totaling $142M.  This project exemplifies the importance of team science, integrative approach to drug discovery, and the use of best practices in CADD.  We posit that lessons learned and best practices for executing an effective CADD project can be applied, with similar success, to many drug discover projects both in academia and industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL6gpFVHyq0rVg90H21EOLACvtfcHk0lgHXRJW7_Fchw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXms1Wgu78%253D&md5=2bfa1b93976eb0721aac6a422ec52d01</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00137%26sid%3Dliteratum%253Aachs%26aulast%3DBan%26aufirst%3DF.%26aulast%3DDalal%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLeBlanc%26aufirst%3DE.%26aulast%3DRennie%26aufirst%3DP.%2BS.%26aulast%3DCherkasov%26aufirst%3DA.%26atitle%3DBest%2520practices%2520of%2520computer-aided%2520drug%2520discovery%253A%2520lessons%2520learned%2520from%2520the%2520development%2520of%2520a%2520preclinical%2520candidate%2520for%2520prostate%2520cancer%2520with%2520a%2520new%2520mechanism%2520of%2520action%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1018%26epage%3D1028%26doi%3D10.1021%2Facs.jcim.7b00137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aryal, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dengler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguère, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Duan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherrer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of opioid analgesics with reduced side effects</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1038/nature19112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1038%2Fnature19112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=27533032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2016&pages=185-190&author=A.+Manglikauthor=H.+Linauthor=D.+K.+Aryalauthor=J.+D.+McCorvyauthor=D.+Denglerauthor=G.+Corderauthor=A.+Levitauthor=R.+C.+Klingauthor=V.+Bernatauthor=H.+H%C3%BCbnerauthor=X.-P.+Huangauthor=M.+F.+Sassanoauthor=P.+M.+Gigu%C3%A8reauthor=S.+L%C3%B6berauthor=D.+Da+Duanauthor=G.+Scherrerauthor=B.+K.+Kobilkaauthor=P.+Gmeinerauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+opioid+analgesics+with+reduced+side+effects&doi=10.1038%2Fnature19112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of opioid analgesics with reduced side effects</span></div><div class="casAuthors">Manglik, Aashish; Lin, Henry; Aryal, Dipendra K.; McCorvy, John D.; Dengler, Daniela; Corder, Gregory; Levit, Anat; Kling, Ralf C.; Bernat, Viachaslau; Hubner, Harald; Huang, Xi-Ping; Sassano, Maria F.; Giguere, Patrick M.; Lober, Stefan; Da Duan; Scherrer, Gregory; Kobilka, Brian K.; Gmeiner, Peter; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue">7619</span>),
    <span class="NLM_cas:pages">185-190</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Morphine is an alkaloid from the opium poppy used to treat pain.  The potentially lethal side effects of morphine and related opioids-which include fatal respiratory depression-are thought to be mediated by μ-opioid-receptor (μOR) signalling through the β-arrestin pathway or by actions at other receptors.  Conversely, G-protein μOR signalling is thought to confer analgesia.  Here we computationally dock over 3 million mols. against the μOR structure and identify new scaffolds unrelated to known opioids.  Structure-based optimization yields PZM21-a potent Gi activator with exceptional selectivity for μOR and minimal β-arrestin-2 recruitment.  Unlike morphine, PZM21 is more efficacious for the affective component of analgesia vs. the reflexive component and is devoid of both respiratory depression and morphine-like reinforcing activity in mice at equi-analgesic doses.  PZM21 thus serves as both a probe to disentangle μOR signalling and a therapeutic lead that is devoid of many of the side effects of current opioids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiCff2w3U8iLVg90H21EOLACvtfcHk0lgHXRJW7_Fchw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI&md5=b8bb2d8575dadf4a6dc74d94c26d2c1a</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature19112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19112%26sid%3Dliteratum%253Aachs%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DAryal%26aufirst%3DD.%2BK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DDengler%26aufirst%3DD.%26aulast%3DCorder%26aufirst%3DG.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DGigu%25C3%25A8re%26aufirst%3DP.%2BM.%26aulast%3DL%25C3%25B6ber%26aufirst%3DS.%26aulast%3DDa%2BDuan%26aufirst%3DD.%26aulast%3DScherrer%26aufirst%3DG.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520opioid%2520analgesics%2520with%2520reduced%2520side%2520effects%26jtitle%3DNature%26date%3D2016%26volume%3D537%26spage%3D185%26epage%3D190%26doi%3D10.1038%2Fnature19112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capelli, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaLonde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semus, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedesco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peishoff, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, M. S.</span></span> <span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">5912</span>– <span class="NLM_lpage">5931</span>, <span class="refDoi"> DOI: 10.1021/jm050362n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.-M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions&doi=10.1021%2Fjm050362n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0lj7zWqvWquBOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.-M.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931%26doi%3D10.1021%2Fjm050362n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span> <span> </span><span class="NLM_article-title">Docking covalent inhibitors: a parameter free approach to pose prediction and scoring</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1932</span>– <span class="NLM_lpage">1940</span>, <span class="refDoi"> DOI: 10.1021/ci500118s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500118s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1932-1940&author=K.+Zhuauthor=K.+W.+Borrelliauthor=J.+R.+Greenwoodauthor=T.+Dayauthor=R.+Abelauthor=R.+S.+Faridauthor=E.+Harder&title=Docking+covalent+inhibitors%3A+a+parameter+free+approach+to+pose+prediction+and+scoring&doi=10.1021%2Fci500118s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span></div><div class="casAuthors">Zhu, Kai; Borrelli, Kenneth W.; Greenwood, Jeremy R.; Day, Tyler; Abel, Robert; Farid, Ramy S.; Harder, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1932-1940</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although many popular docking programs include a facility to account for covalent ligands, large-scale systematic docking validation studies of covalent inhibitors have been sparse.  In this paper, we present the development and validation of a novel approach for docking and scoring covalent inhibitors, which consists of conventional noncovalent docking, heuristic formation of the covalent attachment point, and structural refinement of the protein-ligand complex.  This approach combines the strengths of the docking program Glide and the protein structure modeling program Prime and does not require any parameter fitting for the study of addnl. covalent reaction types.  We first test this method by predicting the native binding geometry of 38 covalently bound complexes.  The av. RMSD of the predicted poses is 1.52 Å, and 76% of test set inhibitors have an RMSD of less than 2.0 Å.  In addn., the apparent affinity score constructed herein is tested on a virtual screening study and the characterization of the SAR properties of two different series of congeneric compds. with satisfactory success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbtjEb3EqzQ7Vg90H21EOLACvtfcHk0lj7zWqvWquBOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D&md5=3633c29fa5dc29817b178fd10a4c7b15</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci500118s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500118s%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DHarder%26aufirst%3DE.%26atitle%3DDocking%2520covalent%2520inhibitors%253A%2520a%2520parameter%2520free%2520approach%2520to%2520pose%2520prediction%2520and%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1932%26epage%3D1940%26doi%3D10.1021%2Fci500118s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek-Klumpers, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upeslacis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles as orally active, irreversible inhibitors of human epidermal growth factor receptor-2 kinase activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1021/jm040159c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040159c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1107-1131&author=H.-R.+Tsouauthor=E.+G.+Overbeek-Klumpersauthor=W.+A.+Hallettauthor=M.+F.+Reichauthor=M.+B.+Floydauthor=B.+D.+Johnsonauthor=R.+S.+Michalakauthor=R.+Nilakantanauthor=C.+Discafaniauthor=J.+Golasauthor=S.+K.+Rabindranauthor=R.+Shenauthor=X.+Shiauthor=Y.-F.+Wangauthor=J.+Upeslacisauthor=A.+Wissner&title=Optimization+of+6%2C7-disubstituted-4-%28arylamino%29quinoline-3-carbonitriles+as+orally+active%2C+irreversible+inhibitors+of+human+epidermal+growth+factor+receptor-2+kinase+activity&doi=10.1021%2Fjm040159c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 6,7-Disubstituted-4-(arylamino)quinoline-3-carbonitriles as Orally Active, Irreversible Inhibitors of Human Epidermal Growth Factor Receptor-2 Kinase Activity</span></div><div class="casAuthors">Tsou, Hwei-Ru; Overbeek-Klumpers, Elsebe G.; Hallett, William A.; Reich, Marvin F.; Floyd, M. Brawner; Johnson, Bernard D.; Michalak, Ronald S.; Nilakantan, Ramaswamy; Discafani, Carolyn; Golas, Jonathan; Rabindran, Sridhar K.; Shen, Ru; Shi, Xiaoqing; Wang, Yu-Fen; Upeslacis, Janis; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1107-1131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles, e.g. I (R1 = H, Cl; R2 = PhCH2O, 1-imidazolyl, 2-furylmethoxy, etc.; R3 = Cl, CN, PhCH2O; R4 = Me, Et; R5 = Me, R6 = Me, HOCH2CH2; R5R6N = azetidinyl, piperidinyl, thiomorpholinyl, etc.) that function as irreversible inhibitors of human epidermal growth factor receptor-2 (HER-2) and epidermal growth factor receptor (EGFR) kinases have been prepd.  These compds. demonstrated enhanced activities for inhibiting HER-2 kinase and the growth of HER-2 pos. cells compared to the EGFR kinase inhibitor I [R1 = H; R2 = F; R3 = Cl; R4 = Et; R5 = R6 = Me; (EKB-569)].  Three synthetic routes were used to prep. these compds.  They were prepd. mostly by acylation of 6-amino-4-(arylamino)quinoline-3-carbonitriles with unsatd. acid chlorides or by amination of 4-chloro-6-(crotonamido)quinoline-3-carbonitriles with monocyclic or bicyclic anilines.  The third route was developed to prep. a key intermediate, 6-acetamido-4-chloroquinoline-3-carbonitrile, that involved a safer cyclization step.  It was shown that attaching a large lipophilic group at the para position of the 4-(arylamino) ring results in improved potency for inhibiting HER-2 kinase.  The importance of a basic dialkylamino group at the end of the Michael acceptor for activity, due to intramol. catalysis of the Michael addn. has also been demonstrated.  This, along with improved water soly., resulted in compds. with enhanced biol. properties.  The mol. modeling results consistent with the proposed mechanism of inhibition are presented.  Binding studies of one compd., I [R1 = H; R2 = 2-pyridylmethoxy; R3 = Cl; R4 = Et; R5 = R6 = Me; (HKI-272)] (C-14 radiolabeled), showed that it binds irreversibly to HER-2 protein in BT474 cells.  Furthermore, it demonstrated excellent oral activity, esp. in HER-2 overexpressing xenografts.  Compd. HKI-272 was selected for further studies and is currently in phase I clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD0QL4-qQZCbVg90H21EOLACvtfcHk0ljOhfdDkzy3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D&md5=55f77e41bf920f738ae2750e7ca0eb22</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm040159c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040159c%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DOverbeek-Klumpers%26aufirst%3DE.%2BG.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DMichalak%26aufirst%3DR.%2BS.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DGolas%26aufirst%3DJ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DUpeslacis%26aufirst%3DJ.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3DOptimization%2520of%25206%252C7-disubstituted-4-%2528arylamino%2529quinoline-3-carbonitriles%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520human%2520epidermal%2520growth%2520factor%2520receptor-2%2520kinase%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1107%26epage%3D1131%26doi%3D10.1021%2Fjm040159c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of a series of 5H-Pyrrolo[2,3-b]pyrazine FGFR kinase inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">698</span>, <span class="refDoi"> DOI: 10.3390/molecules23030698</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.3390%2Fmolecules23030698" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12gtLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=698&author=A.+Jiangauthor=Q.+Liuauthor=R.+Wangauthor=P.+Weiauthor=Y.+Daiauthor=X.+Wangauthor=Y.+Xuauthor=Y.+Maauthor=J.+Aiauthor=J.+Shenauthor=J.+Dingauthor=B.+Xiong&title=Structure-based+discovery+of+a+series+of+5H-Pyrrolo%5B2%2C3-b%5Dpyrazine+FGFR+kinase+inhibitors&doi=10.3390%2Fmolecules23030698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of a series of 5H-pyrrolo[2,3-b]pyrazine FGFR kinase inhibitors</span></div><div class="casAuthors">Jiang, Alan; Liu, Qiufeng; Wang, Ruifeng; Wei, Peng; Dai, Yang; Wang, Xin; Xu, Yechun; Ma, Yuchi; Ai, Jing; Shen, Jingkang; Ding, Jian; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">698/1-698/19</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFRs), a subfamily of receptor tyrosine kinases, are aberrant in various cancer types, and considered to be promising targets for cancer therapy.  We started with a weak-active compd. that was identified from our internal hepatocyte growth factor receptor (also called c-Met) inhibitor project, and optimized it with the guidance of a co-crystal structure of compd. 8 with FGFR1.  Through rational design, synthesis, and the biol. evaluation of a series of 5H-pyrrolo[2,3-b]pyrazine derivs., we discovered several potent FGFR kinase inhibitors.  Among them, compd. 13 displayed high selectivity and favorable metabolic properties, demonstrating a promising lead for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGeqBkLkHhrVg90H21EOLACvtfcHk0ljOhfdDkzy3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12gtLbE&md5=98c0f1e1b49800f46e2c5c5fa439c35a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23030698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23030698%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DP.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520series%2520of%25205H-Pyrrolo%255B2%252C3-b%255Dpyrazine%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D698%26doi%3D10.3390%2Fmolecules23030698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicena, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">High yield bacterial expression of active c-Abl and c-Src tyrosine kinases</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3135</span>– <span class="NLM_lpage">3139</span>, <span class="refDoi"> DOI: 10.1110/ps.051750905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1110%2Fps.051750905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=16260764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlWlsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=3135-3139&author=M.+A.+Seeligerauthor=M.+Youngauthor=M.+N.+Hendersonauthor=P.+Pellicenaauthor=D.+S.+Kingauthor=A.+M.+Falickauthor=J.+Kuriyan&title=High+yield+bacterial+expression+of+active+c-Abl+and+c-Src+tyrosine+kinases&doi=10.1110%2Fps.051750905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">High yield bacterial expression of active c-Abl and c-Src tyrosine kinases</span></div><div class="casAuthors">Seeliger, Markus A.; Young, Matthew; Henderson, M. Nidanie; Pellicena, Patricia; King, David S.; Falick, Arnold M.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3135-3139</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The Abl and Src tyrosine kinases are key signaling proteins that are of considerable interest as drug targets in cancer and many other diseases.  The regulatory mechanisms that control the activity of these proteins are complex, and involve large-scale conformational changes in response to phosphorylation and other modulatory signals.  The success of the Abl inhibitor imatinib in the treatment of chronic myelogenous leukemia has shown the potential of kinase inhibitors, but the rise of drug resistance in patients has also shown that drugs with alternative modes of binding to the kinase are needed.  The detailed understanding of mechanisms of protein-drug interaction and drug resistance through biophys. methods demands a method for the prodn. of active protein on the milligram scale.  We have developed a bacterial expression system for the kinase domains of c-Abl and c-Src, which allows for the quick expression and purifn. of active wild-type and mutant kinase domains by coexpression with the YopH tyrosine phosphatase.  This method makes practical the use of isotopic labeling of c-Abl and c-Src for NMR studies, and is also applicable for constructs contg. the SH2 and SH3 domains of the kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis8a3O54lbVg90H21EOLACvtfcHk0lg4rvoMwRbU7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlWlsL3F&md5=62a85fa3b28dc95098dc3e2ec39690ff</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1110%2Fps.051750905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.051750905%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DM.%2BN.%26aulast%3DPellicena%26aufirst%3DP.%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DFalick%26aufirst%3DA.%2BM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DHigh%2520yield%2520bacterial%2520expression%2520of%2520active%2520c-Abl%2520and%2520c-Src%2520tyrosine%2520kinases%26jtitle%3DProtein%2520Sci.%26date%3D2005%26volume%3D14%26spage%3D3135%26epage%3D3139%26doi%3D10.1110%2Fps.051750905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg4rvoMwRbU7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution--from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/s0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%2D%2Dfrom+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2Fs0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lgomM2wDTC7yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution--from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2Fs0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgomM2wDTC7yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">DFG-out mode of inhibition by an irreversible type-1 inhibitor capable of overcoming gate-keeper mutations in FGF receptors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1021/cb500674s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500674s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=299-309&author=Z.+Huangauthor=L.+Tanauthor=H.+Wangauthor=Y.+Liuauthor=S.+Blaisauthor=J.+Dengauthor=T.+A.+Neubertauthor=N.+S.+Grayauthor=X.+Liauthor=M.+Mohammadi&title=DFG-out+mode+of+inhibition+by+an+irreversible+type-1+inhibitor+capable+of+overcoming+gate-keeper+mutations+in+FGF+receptors&doi=10.1021%2Fcb500674s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">DFG-out Mode of Inhibition by an Irreversible Type-1 Inhibitor Capable of Overcoming Gate-Keeper Mutations in FGF Receptors</span></div><div class="casAuthors">Huang, Zhifeng; Tan, Li; Wang, Huiyan; Liu, Yang; Blais, Steven; Deng, Jingjing; Neubert, Thomas A.; Gray, Nathanael S.; Li, Xiaokun; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">299-309</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-resistance acquisition through kinase gate-keeper mutations is a major hurdle in the clinic.  Here, we detd. the first crystal structures of the human FGFR4 kinase domain (FGFR4K) alone and complexed with ponatinib, a promiscuous type-2 (DFG-out) kinase inhibitor, and an oncogenic FGFR4K harboring the V550L gate-keeper mutation bound to FIIN-2, a new type-1 irreversible inhibitor.  Remarkably, like ponatinib, FIIN-2 also binds in the DFG-out mode despite lacking a functional group necessary to occupy the pocket vacated upon the DFG-out flip.  Structural anal. reveals that the covalent bond between FIIN-2 and a cysteine, uniquely present in the glycine-rich loop of FGFR kinases, facilitates the DFG-out conformation, which together with the internal flexibility of FIIN-2 enables FIIN-2 to avoid the steric clash with the gate-keeper mutation that causes the ponatinib resistance.  The structural data provide a blueprint for the development of next generation anticancer inhibitors through combining the salient inhibitory mechanisms of ponatinib and FIIN-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf97N01ZUSpbVg90H21EOLACvtfcHk0ljoIvYmMAlCLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM&md5=0c741cc5492596f9ea9323e4d45e274b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fcb500674s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500674s%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBlais%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DDFG-out%2520mode%2520of%2520inhibition%2520by%2520an%2520irreversible%2520type-1%2520inhibitor%2520capable%2520of%2520overcoming%2520gate-keeper%2520mutations%2520in%2520FGF%2520receptors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D299%26epage%3D309%26doi%3D10.1021%2Fcb500674s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljoIvYmMAlCLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhJcX3cSxCX4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of <i>in vitro</i> half-life approach and nonspecific binding to microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+an+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0lhJcX3cSxCX4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520an%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agababa, D.</span></span> <span> </span><span class="NLM_article-title">Prediction of hepatic microsomal intrinsic clearance and human clearance values for drugs</span>. <i>J. Mol. Graph. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2009.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=10.1016%2Fj.jmgm.2009.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=19713138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WrtbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=245-252&author=K.+Nikolicauthor=D.+Agababa&title=Prediction+of+hepatic+microsomal+intrinsic+clearance+and+human+clearance+values+for+drugs&doi=10.1016%2Fj.jmgm.2009.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of hepatic microsomal intrinsic clearance and human clearance values for drugs</span></div><div class="casAuthors">Nikolic, Katarina; Agababa, Danica</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-252</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Twenty-nine drugs of different structures were used in theor. QSAR anal. of human hepatic microsomal intrinsic clearance (in vitro T 1/2 and in vitro C L ' int ) and whole body clearance (CLblood).  The examd. compds. demonstrated a wide range of scaled intrinsic clearance values.  Constitutional, geometrical, physico-chem. and electronic descriptors were computed for the examd. structures by use of the Marvin Sketch 5.1.32, the Chem3D Ultra 7.0.0 and the Dragon 5.4 program.  Partial least squares regression (PLSR), has been applied for selection of the most relevant mol. descriptors and development of quant. structure-activity relationship (QSAR) model for human hepatic microsomal intrinsic clearance (in vitro T 1/2).  Optimal QSAR models with nine and ten variables, R 2 > 0.808 and cross-validation parameter q p r e 2 > 0.623 , were selected and compared.  Since the microsomal in vitro T 1/2 data can be used for calcn. of in vitro C L ' int and in vivo CLblood, the developed QSAR model will enable one to analyze the kinetics of cytochrome P 450-mediated reactions in term of intrinsic clearance and whole body clearance.  A comparison is made between predictions produced from the QSAR anal. and exptl. data, and there appears to be generally satisfactory correlations with the literature values for intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcONn29BFARbVg90H21EOLACvtfcHk0li2g6roagIA0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WrtbjE&md5=80f9d2bef510e20f3fb7738238d18c06</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2009.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2009.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DNikolic%26aufirst%3DK.%26aulast%3DAgababa%26aufirst%3DD.%26atitle%3DPrediction%2520of%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520and%2520human%2520clearance%2520values%2520for%2520drugs%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model.%26date%3D2009%26volume%3D28%26spage%3D245%26epage%3D252%26doi%3D10.1016%2Fj.jmgm.2009.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TT0','PDB','3TT0'); return false;">PDB: 3TT0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PVF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PVF','PDB','2PVF'); return false;">PDB: 2PVF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K33" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K33','PDB','4K33'); return false;">PDB: 4K33</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ','PDB','6ITJ'); return false;">PDB: 6ITJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP','PDB','6IUP'); return false;">PDB: 6IUP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO','PDB','6IUO'); return false;">PDB: 6IUO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Z0S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Z0S','PDB','5Z0S'); return false;">PDB: 5Z0S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQT','PDB','4QQT'); return false;">PDB: 4QQT</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00510">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_25300"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00510">10.1021/acs.jmedchem.9b00510</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Structure, scores, and FGFR1 inhibitory activities of the 112 compounds selected from the VS; inhibitory activities were determined by ELISA; X-ray crystal structure analysis: data collection and refinement statistics; sequence coverage of the FGFR4 kinase domain by protein HPLC–MS/MS analysis; residue Cys477 in FGFR4 covalently modified by compound <b>10</b>; inhibition ratios of kinases treated with 0.01, 0.1, and 1 μM compound <b>15</b>; UPLC-UV/Q-TOF MS analysis results of the metabolites of compound <b>15</b> in five different species liver cells from five different species, namely, human, monkey, dog, rat, and mouse; putative metabolic pathway of compound <b>15</b> in liver cells; and HPLC analysis data of all the tested compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_001.pdf">jm9b00510_si_001.pdf (1.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00510/suppl_file/jm9b00510_si_002.csv">jm9b00510_si_002.csv (10.52 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ITJ">6ITJ</a> (FGFR1/<b>9</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUP">6IUP</a> (FGFR4/<b>10</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6IUO">6IUO</a> (FGFR4/<b>11</b>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00510&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00510%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-16%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00510" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677af9186a463dca","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
