
<html lang="en"     class="pb-page"  data-request-id="b04d2ee8-cea2-4697-80c5-6ea1a4823f90"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00089;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-14;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases" /></meta><meta name="dc.Creator" content="Christine  Yueh" /></meta><meta name="dc.Creator" content="Justin  Rettenmaier" /></meta><meta name="dc.Creator" content="Bing  Xia" /></meta><meta name="dc.Creator" content="David R.  Hall" /></meta><meta name="dc.Creator" content="Andrey  Alekseenko" /></meta><meta name="dc.Creator" content="Kathryn A.  Porter" /></meta><meta name="dc.Creator" content="Krister  Barkovich" /></meta><meta name="dc.Creator" content="Gyorgy  Keseru" /></meta><meta name="dc.Creator" content="Adrian  Whitty" /></meta><meta name="dc.Creator" content="James A.  Wells" /></meta><meta name="dc.Creator" content="Sandor  Vajda" /></meta><meta name="dc.Creator" content="Dima  Kozakov" /></meta><meta name="dc.Description" content="The inhibition of kinases has been pursued by the pharmaceutical industry for over 20 years. While the locations of the sites that bind type II and III inhibitors at or near the adenosine 5′-tripho..." /></meta><meta name="Description" content="The inhibition of kinases has been pursued by the pharmaceutical industry for over 20 years. While the locations of the sites that bind type II and III inhibitors at or near the adenosine 5′-tripho..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 4, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00089" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00089" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00089" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00089" /></link>
        
    
    

<title>Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00089" /></meta><meta property="og:title" content="Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0008.jpeg" /></meta><meta property="og:description" content="The inhibition of kinases has been pursued by the pharmaceutical industry for over 20 years. While the locations of the sites that bind type II and III inhibitors at or near the adenosine 5′-triphosphate binding sites are well defined, the literature describes 10 different regions that were reported as regulatory hot spots in some kinases and thus are potential target sites for type IV inhibitors. Kinase Atlas is a systematic collection of binding hot spots located at the above ten sites in 4910 structures of 376 distinct kinases available in the Protein Data Bank. The hot spots are identified by FTMap, a computational analogue of experimental fragment screening. Users of Kinase Atlas (https://kinase-atlas.bu.edu) may view summarized results for all structures of a particular kinase, such as which binding sites are present and how druggable they are, or they may view hot spot information for a particular kinase structure of interest." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00089"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00089">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00089&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00089&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00089&amp;href=/doi/10.1021/acs.jmedchem.9b00089" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6512-6524</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00061" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00171" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Christine Yueh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine Yueh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Yueh">Christine Yueh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justin Rettenmaier</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justin Rettenmaier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Pharmaceutical Chemistry and Cellular and Molecular Pharmacology, University of California, 1700 Fourth Street, San Francisco, California 9415, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justin++Rettenmaier">Justin Rettenmaier</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bing Xia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Xia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Xia">Bing Xia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David R. Hall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David R. Hall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Acpharis Incorporated, Holliston, Massachusetts 01746, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+R.++Hall">David R. Hall</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrey Alekseenko</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrey Alekseenko</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Applied Mathematics and Statistics,  Laufer Center for Physical and Quantitative Biology, Stony Brook University, New York 11794, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrey++Alekseenko">Andrey Alekseenko</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kathryn A. Porter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kathryn A. Porter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kathryn+A.++Porter">Kathryn A. Porter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Krister Barkovich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krister Barkovich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Pharmaceutical Chemistry and Cellular and Molecular Pharmacology, University of California, 1700 Fourth Street, San Francisco, California 9415, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krister++Barkovich">Krister Barkovich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gyorgy Keseru</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gyorgy Keseru</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry Research Group, Research Center for Natural Sciences, Magyar tudósok krt. 2, H-1117 Budapest, Hungary</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gyorgy++Keseru">Gyorgy Keseru</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adrian Whitty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adrian Whitty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adrian++Whitty">Adrian Whitty</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0331-4369" title="Orcid link">http://orcid.org/0000-0002-0331-4369</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James A. Wells</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James A. Wells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Pharmaceutical Chemistry and Cellular and Molecular Pharmacology, University of California, 1700 Fourth Street, San Francisco, California 9415, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+A.++Wells">James A. Wells</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sandor Vajda</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sandor Vajda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering,  Department of Chemistry, Boston University, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#1b6d7a717f7a5b796e357e7f6e"><span class="__cf_email__" data-cfemail="5422353e30351436217a313021">[email protected]</span></a>. Phone: 1-617-353-4757 (S.V.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sandor++Vajda">Sandor Vajda</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1540-8220" title="Orcid link">http://orcid.org/0000-0003-1540-8220</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Dima Kozakov</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dima Kozakov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Applied Mathematics and Statistics,  Laufer Center for Physical and Quantitative Biology, Stony Brook University, New York 11794, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#ddb0b4b9bcae9db1bca8bbb8afbeb8b3a9b8aff3b2afba"><span class="__cf_email__" data-cfemail="5934303d382a1935382c3f3c2b3a3c372d3c2b77362b3e">[email protected]</span></a>. Phone: 1-617-319-7952 (D.K.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dima++Kozakov">Dima Kozakov</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0464-4500" title="Orcid link">http://orcid.org/0000-0003-0464-4500</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00089&amp;href=/doi/10.1021%2Facs.jmedchem.9b00089" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6512–6524</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 4, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 January 2019</li><li><span class="item_label"><b>Published</b> online</span>4 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 July 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00089" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00089</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6512%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChristine%2BYueh%252C%2BJustin%2BRettenmaier%252C%2BBing%2BXia%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D14%26contentID%3Dacs.jmedchem.9b00089%26title%3DKinase%2BAtlas%253A%2BDruggability%2BAnalysis%2Bof%2BPotential%2BAllosteric%2BSites%2Bin%2BKinases%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6524%26publicationDate%3DJuly%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00089"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3268</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">18</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00089" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Yueh&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;Rettenmaier&quot;},{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Xia&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;R. Hall&quot;},{&quot;first_name&quot;:&quot;Andrey&quot;,&quot;last_name&quot;:&quot;Alekseenko&quot;},{&quot;first_name&quot;:&quot;Kathryn&quot;,&quot;last_name&quot;:&quot;A. Porter&quot;},{&quot;first_name&quot;:&quot;Krister&quot;,&quot;last_name&quot;:&quot;Barkovich&quot;},{&quot;first_name&quot;:&quot;Gyorgy&quot;,&quot;last_name&quot;:&quot;Keseru&quot;},{&quot;first_name&quot;:&quot;Adrian&quot;,&quot;last_name&quot;:&quot;Whitty&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;A. Wells&quot;},{&quot;first_name&quot;:&quot;Sandor&quot;,&quot;last_name&quot;:&quot;Vajda&quot;},{&quot;first_name&quot;:&quot;Dima&quot;,&quot;last_name&quot;:&quot;Kozakov&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;6512-6524&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00089&quot;},&quot;abstract&quot;:&quot;The inhibition of kinases has been pursued by the pharmaceutical industry for over 20 years. While the locations of the sites that bind type II and III inhibitors at or near the adenosine 5′-triphosphate binding sites are well defined, the literature describes 10 different regions that were reported as regulatory hot spots in some kinases and thus are potential target sites for type IV inhibitors. Kinase Atlas is a systematic collection of binding hot spots located at the above ten sites in 4910 structures of 376 distinct kinases available in the Protein Data Bank. The hot spots are identified by FTMap, a computational analogue of experimental fragment screening. Users of Kinase Atlas (https://kinase-atlas.bu.edu) may view summarized results for all structures of a particular kinase, such as which binding sites are present and how druggable they are, or they may view hot spot information for a particular kinase structure of interest.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00089&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00089" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00089&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00089" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00089&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00089" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00089&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00089&amp;href=/doi/10.1021/acs.jmedchem.9b00089" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00089" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00089" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00089%26sid%3Dliteratum%253Aachs%26pmid%3D31274316%26genre%3Darticle%26aulast%3DYueh%26date%3D2019%26atitle%3DKinase%2BAtlas%253A%2BDruggability%2BAnalysis%2Bof%2BPotential%2BAllosteric%2BSites%2Bin%2BKinases%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D14%26spage%3D6512%26epage%3D6524%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/jmcmar.2019.62.issue-14/20190725/jmcmar.2019.62.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The inhibition of kinases has been pursued by the pharmaceutical industry for over 20 years. While the locations of the sites that bind type II and III inhibitors at or near the adenosine 5′-triphosphate binding sites are well defined, the literature describes 10 different regions that were reported as regulatory hot spots in some kinases and thus are potential target sites for type IV inhibitors. Kinase Atlas is a systematic collection of binding hot spots located at the above ten sites in 4910 structures of 376 distinct kinases available in the Protein Data Bank. The hot spots are identified by FTMap, a computational analogue of experimental fragment screening. Users of Kinase Atlas (<a href="https://kinase-atlas.bu.edu" class="extLink">https://kinase-atlas.bu.edu</a>) may view summarized results for all structures of a particular kinase, such as which binding sites are present and how druggable they are, or they may view hot spot information for a particular kinase structure of interest.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Members of the protein kinase family play vital roles in cellular physiology and are major drug targets,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> as they have been implicated in many types of diseases such as cancer, diabetes, neurodegeneration, and inflammation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Their association with a wide variety of ailments stems from their involvement in nearly all cellular processes because they are responsible for regulating the activity of other proteins.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Although the development of new kinase inhibitors is a major focus in pharmaceutical research, a large number of kinases remained so far unexplored in drug discovery projects.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Currently, 37 small molecules have been approved by the FDA as kinase inhibitors,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and the vast majority of these target the active site, which is located between the N- and C-terminal domains and binds adenosine 5′-triphosphate (ATP).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Inhibitors that bind to the ATP pocket in the active form of the kinase are known as “type I” kinase inhibitors,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and despite their popularity, the development process presents two major challenges: first, type I inhibitors must bind with enough potency to overcome high physiological concentrations of ATP, and second, the ATP site is highly conserved between all kinases, making it difficult to design inhibitors with enough selectivity to bind only to their intended targets.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Alternatively, kinase inhibitors can be classified as “type II” if they target the ATP site but bind to an inactive conformation known as “DFG-out”, in which a conserved DFG (Asp-Phe-Gly) motif partially blocks the ATP pocket.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Type II inhibitors, which make up about a quarter of approved kinase inhibitors,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> often extend into the hydrophobic back pocket formed in the “DFG-out” state. As this pocket is less conserved than the active site, binding here offers the potential to achieve sufficient selectivity more easily.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Allosteric kinase inhibitors, which do not target the ATP site at all, are classified by where they bind: type III inhibitors bind to the active kinase conformation in a pocket adjacent to the ATP site, whereas type IV inhibitors bind away from the ATP site entirely.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Although type III and IV inhibitors would potentially face fewer issues with potency and/or selectivity than ATP-competitive inhibitors, they remain far less common: only two of the FDA-approved kinase inhibitors (trametinib and cobimetinib) would be considered type III,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and no inhibitors that bind to the kinase catalytic domain would be considered type IV.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">Here, we have explored the potential of kinase allosteric sites as targets for inhibition because they have been relatively underutilized for kinase drug development. To detect and assess the druggability potential of these sites, we used FTMap, a computational analogue of experimental fragment screening,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> to identify binding hot spots on all available kinase structures in the Protein Data Bank.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Hot spots are small regions within a binding site that contribute disproportionately to the binding free energy, and they can be detected in a protein structure even without ligand binding information or an obvious pocket because they tend to bind a large number and variety of small molecules even in an unliganded state. FTMap finds consensus sites, which are regions on the protein surface that bind a large number of small-molecule probe clusters; thus, we can use FTMap to identify binding hot spots even in unliganded structures, as well as to estimate the potency with which they could bind potential ligands because consensus site strength relates to potential binding affinity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">In addition to the pockets associated with type II and III inhibitors, we have identified ten sites on the kinase catalytic domain that have been described in the literature as being involved in either regulating the activity of a kinase, or the ability of a kinase to regulate the activity of its substrates. Seven of these sites are known to bind compounds that exhibit IC<sub>50</sub> values in the micromolar or even nanomolar range, whereas the other three should be considered more speculative. Here, we do not investigate the signal transduction mechanisms from the allosteric binding site to the kinase active site that may require extensive simulations<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> but focus on the detection of sites that can bind ligands with some measure of druggability.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> We consider a site druggable if it includes a primary hot spot with at least 16 probe clusters and at least one additional hot spot nearby. As shown in our earlier work,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> these conditions guarantee that the site is potentially capable of binding a drug-size molecule with at least micromolar affinity, although this affinity may not be sufficient for developing a successful drug.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In addition, we emphasize that this notion of druggability does not imply that the binding of any ligand at the site will modulate the behavior of the target protein.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> However, we assume that because the structure of the kinase catalytic domain is fairly conserved,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> an allosteric site found on one kinase may be present in the same location on other kinases, although the structures and sequences of these analogous pockets would differ between kinases—unlike the ATP site, which is relatively similar between all kinases. This is already known to be true of the DFG-out pocket, for example, which is found in many different kinases,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and many of the other potential allosteric sites we have identified are also associated with more than one kinase.</div><div class="NLM_p last">Our FTMap results for all kinase structures have been made available online as the Kinase Atlas (<a href="https://kinase-atlas.bu.edu" class="extLink">https://kinase-atlas.bu.edu</a>), intended as a resource for researchers interested in kinases. Users may view summarized results for all structures of a particular kinase, such as which allosteric sites are present on the kinase and how druggable they are, or they may view or download FTMap results for a particular kinase structure of interest. The Kinase Atlas can thus be used either to discover where to target a kinase of interest or whether a kinase is worth targeting at all, as some kinases do not have any structures with strong enough consensus sites to be considered druggable. In comparison to the active site, allosteric sites are rarely considered as targets for inhibitor development, but the Kinase Atlas shows that many of these pockets have the potential to be druggable and may be worth pursuing. In particular, we present experimental data to show that the PIF pocket region, an allosteric site associated with PDK1,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> but also found by FTMap in CDK2, is able to bind ligands leading to the discovery of novel micromolar type IV inhibitors. We also demonstrate that our results can be used to find sites where binding has validated regulatory impact as reported in the literature.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68105" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68105" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The Kinase Atlas contains FTMap results for 3887 unique PDB IDs, corresponding to 4910 total kinase structures from 376 different kinases. The number of kinase structures is higher than the number of PDB IDs because some of the PDB files include multiple structures (multiple chains) in the unit cell. The consensus sites of probe clusters are determined by the protein mapping program FTMap and are assigned to 12 different putative binding sites that are potentially allosteric as defined on the basis of kinase literature (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Each of these 12 sites was found in, or is most widely associated with, a particular kinase defined as the “source” kinase for the site (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), although most sites can also be found in a variety of kinases. FTMap was able to identify hot spots located at each site in an unliganded structure of the “source” kinase for all sites. Further information on naming and selection of putative allosteric sites is given in the <a class="ref internalNav" href="#sec3" aria-label="Discussion">Discussion</a> section and in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">Figure S1</a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Positions of all allosteric sites discussed in this paper. ATP is represented as yellow sticks, the N-terminal domain is shown in green, and the C-terminal domain is shown in gray. From top to bottom: MPP (cyan), PIF (purple), MT3 (orange), DFG (red), DRS (blue), PMP (pink), PDIG (dark green), AAS (light yellow), CMP (brown), DEF (teal), LBP (magenta), and EDI (olive).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Definitions and Examples of Allosteric Sites</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">site</th><th class="colsep0 rowsep0" align="center">site name origin</th><th class="colsep0 rowsep0" align="center">inhibitor type</th><th class="colsep0 rowsep0" align="center">source kinase</th><th class="colsep0 rowsep0" align="center">example PDB</th><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center">binding data</th><th class="colsep0 rowsep0" align="center">mapped kinase</th><th class="colsep0 rowsep0" align="center">mapped PDB</th><th class="colsep0 rowsep0" align="center">consensus sites</th><th class="colsep0 rowsep0" align="center">pocket description</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">DFG</td><td class="colsep0 rowsep0" align="center">DFG motif</td><td class="colsep0 rowsep0" align="center">II</td><td class="colsep0 rowsep0" align="center">many</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IEP">1IEP</a></td><td class="colsep0 rowsep0" align="center">imatinib</td><td class="colsep0 rowsep0" align="center">IC<sub>50</sub> = 10.8 nM</td><td class="colsep0 rowsep0" align="center">Tie2</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVR">1FVR</a>_B</td><td class="colsep0 rowsep0" align="center">1(16)</td><td class="colsep0 rowsep0" align="center">hydrophobic pocket that opens up when DFG motif switches to inactive “DFG-out” conformation; binding here may stabilize inactive kinase conformation</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">MT3</td><td class="colsep0 rowsep0" align="center">MEK1/2 type III inhibitor</td><td class="colsep0 rowsep0" align="center">III</td><td class="colsep0 rowsep0" align="center">MEK1/2</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2">4AN2</a></td><td class="colsep0 rowsep0" align="center">cobimetinib</td><td class="colsep0 rowsep0" align="center">IC<sub>50</sub> = 0.9 nM</td><td class="colsep0 rowsep0" align="center">MEK1</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EQF">3EQF</a>_A</td><td class="colsep0 rowsep0" align="center">0(20)</td><td class="colsep0 rowsep0" align="center">adjacent to ATP and DFG-out pockets; binding disrupts salt bridge required for kinase activity</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">PIF</td><td class="colsep0 rowsep0" align="center">PDK1 interacting fragment</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">PDK1</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RQK">4RQK</a></td><td class="colsep0 rowsep0" align="center">RS1</td><td class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> = 1.5 μM</td><td class="colsep0 rowsep0" align="center">PDK1</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IOP">3IOP</a>_A</td><td class="colsep0 rowsep0" align="center">0(21), 8(3)</td><td class="colsep0 rowsep0" align="center">PDK1 regulates other AGC kinases by recruiting them through this site</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">CMP</td><td class="colsep0 rowsep0" align="center">c-Abl myristoyl pocket</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">c-Abl</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5V">3K5V</a></td><td class="colsep0 rowsep0" align="center">GNF-2</td><td class="colsep0 rowsep0" align="center">IC<sub>50</sub> = 267 nM</td><td class="colsep0 rowsep0" align="center">c-Abl</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QRJ">3QRJ</a>_B</td><td class="colsep0 rowsep0" align="center">3(12), 8(2)</td><td class="colsep0 rowsep0" align="center">binding here leads to active or inactive state in c-Abl (depending on ligand size) by affecting SH domain binding</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">DRS</td><td class="colsep0 rowsep0" align="center">D-recruitment site</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">MAPK8</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1UKI">1UKI</a></td><td class="colsep0 rowsep0" align="center">pepJIP1</td><td class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> = 0.42 μM</td><td class="colsep0 rowsep0" align="center">JNK3</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z9L">4Z9L</a>_A</td><td class="colsep0 rowsep0" align="center">2(13)</td><td class="colsep0 rowsep0" align="center">substrate docking site present in all MAP kinases</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">DEF</td><td class="colsep0 rowsep0" align="center">docking site for ERK FXF</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">MAPK8</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O2M">3O2M</a></td><td class="colsep0 rowsep0" align="center">A-82118</td><td class="colsep0 rowsep0" align="center">IC<sub>50</sub> = 7.7 μM</td><td class="colsep0 rowsep0" align="center">JNK1</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3V">3V3V</a>_A</td><td class="colsep0 rowsep0" align="center">0(22), 1(14)</td><td class="colsep0 rowsep0" align="center">substrate docking site present in some MAP kinases; located near MAPK insert</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">LBP</td><td class="colsep0 rowsep0" align="center">lipid binding pocket</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">p38a MAPK</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW">3NEW</a></td><td class="colsep0 rowsep0" align="center">compound 10</td><td class="colsep0 rowsep0" align="center">IC<sub>50</sub> = 1.2 μM</td><td class="colsep0 rowsep0" align="center">p38a MAPK</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S4Q">3S4Q</a>_A</td><td class="colsep0 rowsep0" align="center">0(21), 3(15)</td><td class="colsep0 rowsep0" align="center">binding of different lipids here affects p38a MAPK’s preference and activity for different substrates</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">PDIG</td><td class="colsep0 rowsep0" align="center">PDIG motif</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">Chk1</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS">3JVS</a></td><td class="colsep0 rowsep0" align="center">compound 3</td><td class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> = 146 nM</td><td class="colsep0 rowsep0" align="center">Chk1</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RVK">4RVK</a>_A</td><td class="colsep0 rowsep0" align="center">0(17), 9(1)</td><td class="colsep0 rowsep0" align="center">substrate recognition site located near PDIG motif in Chk1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">EDI</td><td class="colsep0 rowsep0" align="center">EGFR dimerization interface</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">EGFR</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RFE">2RFE</a></td><td class="colsep0 rowsep0" align="center">Mig6</td><td class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> = 13 μM</td><td class="colsep0 rowsep0" align="center">EGFR</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RJ5">4RJ5</a>_A</td><td class="colsep0 rowsep0" align="center">1(16)</td><td class="colsep0 rowsep0" align="center">an EGFR monomer activates another by binding at this interface on the C-terminal domain</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">PMP</td><td class="colsep0 rowsep0" align="center">PKA myristoyl pocket</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">PKA</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CMK">1CMK</a></td><td class="colsep0 rowsep0" align="center">myristoyl</td><td class="colsep0 rowsep0" align="center">n/a</td><td class="colsep0 rowsep0" align="center">PKA</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AE9">4AE9</a>_A</td><td class="colsep0 rowsep0" align="center">1(24), 4(08)</td><td class="colsep0 rowsep0" align="center">myristoyl binding here activates membrane binding in PKA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">AAS</td><td class="colsep0 rowsep0" align="center">Aurora A activation segment</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">Aurora A</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P">4C3P</a></td><td class="colsep0 rowsep0" align="center">Aurora A</td><td class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> > 300 μM</td><td class="colsep0 rowsep0" align="center">Aurora A</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O51">3O51</a>_A</td><td class="colsep0 rowsep0" align="center">0(22), 1(17)</td><td class="colsep0 rowsep0" align="center">an Aurora A monomer activates another through binding of its activation segment to this site</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">MPP</td><td class="colsep0 rowsep0" align="center">MKK4 p38a peptide</td><td class="colsep0 rowsep0" align="center">IV</td><td class="colsep0 rowsep0" align="center">MKK4</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALO">3ALO</a></td><td class="colsep0 rowsep0" align="center">p38a peptide</td><td class="colsep0 rowsep0" align="center">n/a</td><td class="colsep0 rowsep0" align="center">MKK4</td><td class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALN">3ALN</a>_A<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">1(20), 3(11)</td><td class="colsep0 rowsep0" align="center">p38a peptide binding inhibits MKK4 by inducing conformational changes that lead to auto-inhibition</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALN">3ALN</a>_A does have a ligand at the MPP site, but MKK4 does not have any unliganded structures that contain a complete N-terminal domain.</p></div></div></div><div class="NLM_p">As mentioned, the number of probe clusters in a ligand binding site predicts the druggability of the site. Not every binding site with druggable potential will have a liganded structure or binding data available, so being able to detect binding hot spots in unliganded structures is often useful in determining whether a protein is likely to be druggable as well as which regions to target.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> We note that some of the strongest binding hot spots in all kinases are at the ATP binding site between the N and C lobes of the kinase domain. Because we are interested in allosteric sites, the two lobes are mapped separately, thereby decreasing the fraction of the FTMap probes that cluster in the ATP binding pocket and increasing the relative importance of potential allosteric sites in other surface regions of the protein.</div><div class="NLM_p">Each binding site and its FTMap results are described briefly in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, and in more detail in the following sections. Most of them were found to be druggable in at least one unliganded structure (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>); the structure with the strongest consensus site(s) for each allosteric site is listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The number of kinases that were found to be druggable at each allosteric site varied widely, with some sites being found in as few as 6 kinases, and others in well over 100 kinases, as seen in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. An allosteric site being “common” does not necessarily render it unsuitable as a target, however, as allosteric sites are not as conserved as the ATP site is between different kinases. Below we describe the 12 sites that are examined for the existence of hot spots. The first nine of these sites have known inhibitors in some kinases, whereas the last three sites have been reported to have some allosteric function.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Number of kinases found to have consensus sites at each putative allosteric site. ATP is not an allosteric site but is included here as a reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Mapping results for unliganded structures (except MPP) of kinases associated with each allosteric site, with superimposed ligands from bound structures shown as yellow sticks. Each probe cluster in the consensus cluster is represented by a single probe shown in line representation. Structural, ligand, and mapping details for each site are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Number of Kinases Found to be Druggable at Each Allosteric Site (Out of Total Number of Kinases Containing Each Site)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">all vs human kinases</th><th class="rowsep1 colsep0" colspan="8" align="center">human kinases by family</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">site</th><th class="colsep0 rowsep0" align="center">all (376)</th><th class="colsep0 rowsep0" align="center">human (239)</th><th class="colsep0 rowsep0" align="center">AGC</th><th class="colsep0 rowsep0" align="center">CAMK</th><th class="colsep0 rowsep0" align="center">CK1</th><th class="colsep0 rowsep0" align="center">CMGC</th><th class="colsep0 rowsep0" align="center">other</th><th class="colsep0 rowsep0" align="center">STE</th><th class="colsep0 rowsep0" align="center">TKL</th><th class="colsep0 rowsep0" align="center">Tyr</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATP</td><td class="colsep0 rowsep0" align="left">298/373</td><td class="colsep0 rowsep0" align="left">193/238</td><td class="colsep0 rowsep0" align="left">21/22</td><td class="colsep0 rowsep0" align="left">23/27</td><td class="colsep0 rowsep0" align="left">8/10</td><td class="colsep0 rowsep0" align="left">27/31</td><td class="colsep0 rowsep0" align="left">26/27</td><td class="colsep0 rowsep0" align="left">22/27</td><td class="colsep0 rowsep0" align="left">14/15</td><td class="colsep0 rowsep0" align="left">44/49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DFG</td><td class="colsep0 rowsep0" align="left">93/272</td><td class="colsep0 rowsep0" align="left">71/190</td><td class="colsep0 rowsep0" align="left">5/13</td><td class="colsep0 rowsep0" align="left">8/18</td><td class="colsep0 rowsep0" align="left">2/7</td><td class="colsep0 rowsep0" align="left">10/28</td><td class="colsep0 rowsep0" align="left">6/20</td><td class="colsep0 rowsep0" align="left">7/21</td><td class="colsep0 rowsep0" align="left">10/13</td><td class="colsep0 rowsep0" align="left">22/46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">182/304</td><td class="colsep0 rowsep0" align="left">125/200</td><td class="colsep0 rowsep0" align="left">7/16</td><td class="colsep0 rowsep0" align="left">17/26</td><td class="colsep0 rowsep0" align="left">2/6</td><td class="colsep0 rowsep0" align="left">20/27</td><td class="colsep0 rowsep0" align="left">14/22</td><td class="colsep0 rowsep0" align="left">19/26</td><td class="colsep0 rowsep0" align="left">5/14</td><td class="colsep0 rowsep0" align="left">38/46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">116/199</td><td class="colsep0 rowsep0" align="left">87/141</td><td class="colsep0 rowsep0" align="left">10/11</td><td class="colsep0 rowsep0" align="left">10/15</td><td class="colsep0 rowsep0" align="left">3/6</td><td class="colsep0 rowsep0" align="left">16/20</td><td class="colsep0 rowsep0" align="left">12/15</td><td class="colsep0 rowsep0" align="left">10/17</td><td class="colsep0 rowsep0" align="left">3/7</td><td class="colsep0 rowsep0" align="left">21/35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">76/200</td><td class="colsep0 rowsep0" align="left">61/145</td><td class="colsep0 rowsep0" align="left">0/12</td><td class="colsep0 rowsep0" align="left">13/17</td><td class="colsep0 rowsep0" align="left">2/7</td><td class="colsep0 rowsep0" align="left">8/17</td><td class="colsep0 rowsep0" align="left">7/13</td><td class="colsep0 rowsep0" align="left">11/20</td><td class="colsep0 rowsep0" align="left">1/9</td><td class="colsep0 rowsep0" align="left">15/36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">103/231</td><td class="colsep0 rowsep0" align="left">70/151</td><td class="colsep0 rowsep0" align="left">15/20</td><td class="colsep0 rowsep0" align="left">7/13</td><td class="colsep0 rowsep0" align="left">5/10</td><td class="colsep0 rowsep0" align="left">2/16</td><td class="colsep0 rowsep0" align="left">5/15</td><td class="colsep0 rowsep0" align="left">12/19</td><td class="colsep0 rowsep0" align="left">0/6</td><td class="colsep0 rowsep0" align="left">24/33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DEF</td><td class="colsep0 rowsep0" align="left">18/65</td><td class="colsep0 rowsep0" align="left">16/48</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">0/1</td><td class="colsep0 rowsep0" align="left">2/2</td><td class="colsep0 rowsep0" align="left">7/12</td><td class="colsep0 rowsep0" align="left">1/2</td><td class="colsep0 rowsep0" align="left">0/6</td><td class="colsep0 rowsep0" align="left">1/2</td><td class="colsep0 rowsep0" align="left">5/16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LBP</td><td class="colsep0 rowsep0" align="left">22/92</td><td class="colsep0 rowsep0" align="left">16/69</td><td class="colsep0 rowsep0" align="left">1/8</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">3/6</td><td class="colsep0 rowsep0" align="left">10/17</td><td class="colsep0 rowsep0" align="left">2/8</td><td class="colsep0 rowsep0" align="left">0/4</td><td class="colsep0 rowsep0" align="left">0/3</td><td class="colsep0 rowsep0" align="left">0/13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIG</td><td class="colsep0 rowsep0" align="left">114/252</td><td class="colsep0 rowsep0" align="left">80/166</td><td class="colsep0 rowsep0" align="left">10/20</td><td class="colsep0 rowsep0" align="left">15/21</td><td class="colsep0 rowsep0" align="left">2/5</td><td class="colsep0 rowsep0" align="left">13/24</td><td class="colsep0 rowsep0" align="left">11/17</td><td class="colsep0 rowsep0" align="left">9/21</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">17/36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">109/232</td><td class="colsep0 rowsep0" align="left">69/148</td><td class="colsep0 rowsep0" align="left">6/12</td><td class="colsep0 rowsep0" align="left">12/19</td><td class="colsep0 rowsep0" align="left">2/7</td><td class="colsep0 rowsep0" align="left">26/29</td><td class="colsep0 rowsep0" align="left">6/13</td><td class="colsep0 rowsep0" align="left">0/4</td><td class="colsep0 rowsep0" align="left">6/11</td><td class="colsep0 rowsep0" align="left">9/32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PMP</td><td class="colsep0 rowsep0" align="left">6/21</td><td class="colsep0 rowsep0" align="left">2/13</td><td class="colsep0 rowsep0" align="left">2/5</td><td class="colsep0 rowsep0" align="left">0/4</td><td class="colsep0 rowsep0" align="left">0/0</td><td class="colsep0 rowsep0" align="left">0/1</td><td class="colsep0 rowsep0" align="left">0/1</td><td class="colsep0 rowsep0" align="left">0/0</td><td class="colsep0 rowsep0" align="left">0/0</td><td class="colsep0 rowsep0" align="left">0/2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AAS</td><td class="colsep0 rowsep0" align="left">128/234</td><td class="colsep0 rowsep0" align="left">86/158</td><td class="colsep0 rowsep0" align="left">8/17</td><td class="colsep0 rowsep0" align="left">16/22</td><td class="colsep0 rowsep0" align="left">2/7</td><td class="colsep0 rowsep0" align="left">5/16</td><td class="colsep0 rowsep0" align="left">12/16</td><td class="colsep0 rowsep0" align="left">13/17</td><td class="colsep0 rowsep0" align="left">7/11</td><td class="colsep0 rowsep0" align="left">20/33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">128/249</td><td class="colsep0 rowsep0" align="left">102/171</td><td class="colsep0 rowsep0" align="left">6/9</td><td class="colsep0 rowsep0" align="left">7/17</td><td class="colsep0 rowsep0" align="left">2/4</td><td class="colsep0 rowsep0" align="left">8/21</td><td class="colsep0 rowsep0" align="left">9/17</td><td class="colsep0 rowsep0" align="left">21/25</td><td class="colsep0 rowsep0" align="left">6/10</td><td class="colsep0 rowsep0" align="left">40/47</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">For human kinases, these numbers are further divided by family. ATP is not an allosteric site but is included here as a reference.</p></div></div></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> DFG-Out Pocket (DFG)</h3><div class="NLM_p last">The DFG-out pocket is a hydrophobic pocket that opens up when the conserved DFG motif changes conformation as a kinase switches from an active to an inactive state. In the active, DFG-in state, this site is occupied by the Phe residue in the DFG motif, and ATP can bind to the active site; in the inactive, DFG-out state, however, Phe instead partially occupies the ATP site, preventing ATP from binding and exposing the less conserved DFG-out pocket.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Type II inhibitors, which bind to the DFG-out conformation, are ATP-competitive, but they frequently extend into the DFG-out pocket.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Because such inhibitors stabilize the DFG-out conformation of the DFG loop and lead to inhibition, the DFG-out pocket is sometimes considered allosteric,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> in spite of the non-allosteric inhibition mechanism. As this site differs more between kinases than the ATP site, binding here may allow inhibitors to be more selective, although existing type II inhibitors do not necessarily have improved selectivity.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> FTMap results for known targets of type II inhibitors suggest that the DFG-out pocket may not be particularly strong; many structures were shown to be only borderline druggable at this site, although some kinases had apo structures with strong enough consensus sites for this site to be considered druggable. The ability of type II inhibitors to bind with high affinity (e.g., imatinib exhibits an IC<sub>50</sub> value of 10.8 nM for c-Abl)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> may be due mostly to the ATP pocket, with the DFG-out pocket playing a supporting role. In fact, because type II inhibitors bind to both the ATP and DFG-out pockets, it is difficult to separate the contributions of the corresponding inhibitor moieties to the binding free energy. However, in most kinases that bind both type I and type II inhibitors, high binding affinity is already achieved by type I inhibitors that do not expand into the DFG-out pocket. Thus, while type II inhibitors generally do not further improve binding, they can provide improved selectivity.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> MEK1/2 Type III Inhibitor Site (MT3)</h3><div class="NLM_p last">This site is based on the location where type III kinase inhibitors bind; adjacent to the ATP and DFG-out sites and between the N- and C-terminal domains. Binding to this pocket in MEK1/2 disrupts the salt bridge between the conserved lysine residue K97/101 on the β3-strand and the conserved glutamate E114/118 on the αC-helix that is required for kinase activity.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Unlike the DFG-out pocket, this site is present in the active form of the kinase, and ligands can bind here even if the kinase is already bound to ATP.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> FDA-approved type III inhibitors include trametinib and cobimetinib, both of which target MAPK/ERK kinase (MEK1/2) and have been used to treat melanoma.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Mapping results show that this site is highly druggable, with strong consensus sites present even in unliganded structures of MEK1/2, which is consistent with experiments showing that type III inhibitors can be highly potent (IC<sub>50</sub> = 0.9 nM for cobimetinib/MEK1).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> An analogous pocket appears to be present in the epidermal growth receptor (EGFR), which binds the inhibitor EAI045 with IC<sub>50</sub> values as low as 3 nM.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> PDK1 Interacting Fragment (PIF)</h3><div class="NLM_p last">The PIF pocket is a hydrophobic groove found on the N-terminal domain of phosphoinositide-dependent kinase 1 (PDK1), which uses this pocket to recruit the C-terminal hydrophobic motif (HM) on other members of the AGC kinase family and thereby regulate their activity through phosphorylation.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> PDK1 activity could thus be modulated through inhibitor binding at this site, as this would disrupt its interactions with its substrates and prevent PDK1 from either activating or inhibiting other kinases. Both activators<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and inhibitors<a onclick="showRef(event, 'ref18 ref24'); return false;" href="javascript:void(0);" class="ref ref18 ref24">(18,24)</a> have been developed for the PIF pocket, with many in the low affinity range (as low as <i>K</i><sub>d</sub> = 1.5 μM).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> These compounds could likely be optimized for stronger binding, as mapping results indicate that this site is likely to be capable of binding compounds with higher affinity. Mapping showed that this was also true for other AGC kinases, many of which are known to possess a similar pocket.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> c-Abl Myristoyl Pocket (CMP)</h3><div class="NLM_p last">The myristoyl pocket in Abelson tyrosine-protein kinase 1 (c-Abl) binds the myristoyl group from the N-terminal cap of the kinase, allowing SH3 and SH2 domains to associate and induce an autoinhibited state. This N-terminal cap is not present, however, in the fusion BCR-Abl oncogene, which results from a chromosomal translocation that fuses the breakpoint cluster region (BCR) with c-Abl; BCR-Abl thus cannot be autoinhibited by myristoylation, and its elevated activity leads to disorders such as chronic myelogenous leukemia (CML).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> CML has been treated with imatinib and other ATP-competitive inhibitors, but mutations near the ATP pocket in BCR-Abl are common and frequently lead to drug resistance.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> As the myristoyl pocket is located in the C-terminal domain, far from the ATP site, these mutations would be less likely to affect binding there. GNF-2 has been found to bind at the myristoyl pocket in BCR-Abl, and similar to the myristoyl group, it encourages SH2 and SH3 domain binding; it also induces conformational changes that promote binding of ATP-competitive inhibitors.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Alternatively, c-Abl can be activated by the binding of bulkier groups to the myristoyl pocket, such as DPH, which leads to conformational changes that prevent SH domains from associating with and inhibiting the kinase.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Although FTMap places a hot spot at the myristoyl pocket in most Abl kinase structures, the site is predicted to be at best borderline druggable. Nevertheless, a series of small molecules was shown to bind to the BCR-Abl myristoyl pocket and inhibit kinase activity via an allosteric mechanism,<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29−31)</a> in the case of ABL001 with high affinity.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Even better results have been achieved by combining allosteric inhibitors with ATP-competitive inhibitors imatinib or nilotinib as such combinations can overcome resistance to either agent alone.<a onclick="showRef(event, 'ref27 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref27 ref30 ref31 ref32">(27,30−32)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> D-Recruitment Site (DRS)</h3><div class="NLM_p">The D-recruitment site is found in the C-terminal domain of all mitogen-activated protein (MAP) kinases, a family that includes extracellular signal-regulated kinases (ERKs), c-Jun N-terminal kinases (JNKs), and p38 MAPKs. Its name comes from the “D-motif” sequences found in MAPK binders, and it contains two subsites to which D-motifs bind, an acidic patch and a hydrophobic pocket.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> MAP kinases control intracellular responses to extracellular stimuli, phosphorylating their substrates within the cell after being activated by their upstream regulators, MAPK kinases (MKKs).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Their interactions with other proteins in the MAPK signal transduction pathway are thus critical to the regulatory role MAPKs play in most cellular processes, and the D-recruitment site is a major docking site for these interactions.</div><div class="NLM_p last">Inhibitors have been developed to target the D-recruitment site, including peptides such as pepJIP1<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and small molecules such as BI-78D3.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> These inhibitors are mimetics of JIP1, a scaffold protein that enhances signaling in JNK and compete with JIP1 to inhibit JNK activity. A small-molecule natural product, rooperol, has also been shown to inhibit p38a MAP kinase.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> None of these bind with the affinity that would be required for an appropriate drug candidate, however, with BI-78D3 being the most potent (IC<sub>50</sub> = 280 nM).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Mapping results for MAPK structures suggest that developing compounds with greater affinity may not be likely, as the D-recruitment site was shown to be borderline druggable at best at the hydrophobic pocket and not druggable at all at the acidic patch. Stronger binders that bind at the D-recruitment site with high affinity (IC<sub>50</sub> = 18 nM) have been reported, but these are long molecules that also extend into the ATP site, rather than targeting the D-recruitment site alone.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The D-recruitment site is thus not a promising pocket to be targeted on its own for MAP kinases, but this region was shown to have the potential to bind compounds with higher affinity in related kinases, such as members of the MKK family.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Docking Site for ERK FXF (DEF)</h3><div class="NLM_p last">The DEF site is found in the C-terminal domains of several members of the MAP kinase family, such as ERK1/2, p38a MAPK, and JNK1.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> It is also known as the FXF site because it binds to the FXF motif found on several MAPK substrates, and it is located near the MAP kinase insert, where it appears after phosphorylation activates and induces conformational changes in ERK.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Compounds that bind to MAP kinases at the DEF site have been reported, such as biaryl tetrazoles identified by Comess et al. that inhibit JNK1 activation by MKK7 (IC<sub>50</sub> = 7.7 μM),<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and mapping results suggest that the DEF site has the potential to be highly druggable, unlike the other major MAPK docking site, the D-recruitment site.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Lipid Binding Pocket (LBP)</h3><div class="NLM_p last">The lipid binding pocket is located in the C-terminal domain of p38a MAP kinases near the MAP kinase insert. The biological relevance of this pocket has not been verified conclusively, but it appears to be able to accommodate a variety of lipids, and it has been suggested that binding different lipids may affect p38a MAPK’s catalytic activity and preference for specific substrates, particularly in cellular processes that involve lipids.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Several lipid-based molecules have been found to activate p38a MAPK upon binding to this site—such as phosphatidylinositol ether lipid analogues (PIAs) and perifosine, a phase II AKT/PKB inhibitor structurally similar to PIAs—by inducing conformational changes that lead to autophosphorylation.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> These lipids bind with low affinity to p38a MAPK (IC<sub>50</sub> = 1.2 μM), but mapping results indicate that this site would likely be able to bind ligands with higher affinity because many structures of p38a MAPK had strong consensus sites at the lipid binding pocket.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> PDIG Motif Site (PDIG)</h3><div class="NLM_p last">This pocket is located near the PDIG motif in the C-terminal domain of checkpoint kinase 1 (Chk1), an important regulator in the DNA damage response pathway. It appears to act as a substrate recognition site, and so Chk1 activity could potentially be inhibited by compounds that target this site and compete with substrate binding.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Several inhibitors have been developed that bind to this site with low micromolar affinity (as low as <i>K</i><sub>i</sub> = 146 nM), such as thioquinazolinones,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> carbamates, and semicarbazides,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and mapping results for Chk1 structures suggest that the site has the potential to bind compounds with even greater affinity. A similar pocket appears to be present on PIM1 kinase, which was found to bind mitoxantrone, an FDA-approved chemotherapy drug that targets type II topoisomerase.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Mitoxantrone binds to two locations on PIM1, the substrate binding site (analogous to the one on Chk1) and the ATP site, with high affinity; this is in agreement with mapped structures of PIM1, which have strong consensus sites in this pocket.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> EGFR Dimerization Interface (EDI)</h3><div class="NLM_p last">Members of the EGFR family are activated upon formation of an asymmetric dimer between two EGFR kinases, in which the C-terminal domain of one kinase interacts with the N-terminal domain of the second kinase. This dimerization is considered analogous to the interaction between cyclin and cyclin-dependent kinase (CDK), which activates CDK, and the inactive forms of CDK and EGFR are considered to be similar to each other.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Binding at the EGFR dimerization interface could thus be used to inhibit EGFR activity because it would interfere with the ability of an EGFR monomer to be activated by another monomer. A peptide derived from the MIG6 (mitogen-inducible gene 6) protein was found to inhibit EGFR by binding at and blocking the dimerization interface, exhibiting a <i>K</i><sub>d</sub> value of 13 μM.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> CDK2 appears to possess a similar pocket, as D-luciferin was found to bind in the same location and inhibit CDK2 (although it binds in two locations, with the other being the ATP site, so its contribution is less clear).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Mapping showed that this site appears to be druggable in both EGFR and CDK2, with strong consensus sites present in this pocket in structures of both kinases, although it appears to be stronger in CDK2.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> PKA Myristoyl Pocket (PMP)</h3><div class="NLM_p last">The myristoyl pocket in protein kinase A (PKA) is located in the C-terminal domain in a different area from the c-Abl myristoyl pocket, and myristate binding here appears to activate membrane association with PKA.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> This pocket has received less attention than the myristoyl pocket in c-Abl and does not appear to be targeted by any known inhibitors, but based on the mapping results of PKA structures, the PKA myristoyl pocket appears to be highly druggable.</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Aurora A Autophosphorylation Site (AAS)</h3><div class="NLM_p last">This pocket is present in Aurora A kinases in the C-terminal domain between the PDIG and DEF sites. Autophosphorylation of Aurora A occurs when the activation segment on one monomer binds to this region on a second monomer, activating the second monomer.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Inhibitors do not appear to have been developed for this site, but mapping indicates that it is likely to be highly druggable, with multiple strong consensus sites located there in some mapped structures of Aurora A.</div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> MKK4 p38 Peptide Site (MPP)</h3><div class="NLM_p last">This site is located on the N-terminal domain of MAP kinase kinase 4 (MKK4), which phosphorylates and activates members of the MAP kinase family, such as JNKs and p38 kinases. A p38a peptide was found to inhibit MKK4 by binding at this site and inducing an auto-inhibition state.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Few structures of MKK4 are available, and none of them are unliganded at this site, but the available structures were found to have strong consensus sites there.</div></div><div id="sec2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Experimental Validation of CDK2 Inhibitors Binding at the PIF Pocket Identified by FTMap</h3><div class="NLM_p">FTMap predicts a strong hot spot in the “PIF pocket” of CDK2 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). To experimentally identify ligands that bind to that region, we used tethering, a disulfide-based fragment screening approach<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> described in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">Figure S4</a>). Briefly, a cysteine residue was introduced adjacent to the PIF pocket by mutation of Lys56 to Cys, and the mutant protein was used to screen a library of 1280 small-molecule disulfides for potential ligands. Hits were selected by measuring the formation of a protein-fragment mixed disulfide by intact protein mass spectrometry. The relative “affinities” of the top 12 hits were assessed by measuring the concentration of β-mercaptoethanol required to displace half of the fragment from the protein (βME<sub>50</sub>). Higher βME<sub>50</sub> values indicate a stronger noncovalent interaction between the fragment and the protein. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> shows the two most potent compounds bound to CDK2 as identified by tethering and the percent inhibition of disulfide labeling versus the βME<sub>50</sub> concentration. We note that these results are fairly preliminary, as the concentration–response curves did not sufficiently flatten out to a maximum value, and there was only one replicate at each concentration. Nevertheless, we further studied the 12 most promising compounds and used a radioactive peptide phosphorylation assay to measure whether any of the compounds could modulate CDK2 activity in the presence of Cyclin A2. Compounds <b>1</b> and <b>2</b> shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> were found to have the greatest inhibitory effect, reducing the phosphorylation of the Histone H1 peptide by CDK2–Cyclin by 42 and 81%, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Additionally, at the reductant concentration used in the functional assay, there was no detectable labeling of CDK2 lacking the mutant K56C; this demonstrated that the modulation of CDK2 was due to ligand binding in the PIF pocket region, likely through disruption of the CDK2–Cyclin interaction.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. FTMap finds a strong hot spot in the PIF pocket of CDK2. Only the N-lobe of CDK2 is shown, and each probe cluster placed by FTMap is represented by a single probe at its center (gray lines). Residues interacting with the probes are shown in magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds bound to CDK2 as identified by tethering. (A) Two most potent compounds. (B) Preliminary potency measurements for compounds <b>1</b> and <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Radioactive peptide phosphorylation assay results demonstrating the inhibitory effects of compounds <b>1</b> and <b>2</b> upon CDK2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> How Frequently are Druggable Sites Present in Different Structures of a Kinase?</h3><div class="NLM_p">One strength of Kinase Atlas is that it can simultaneously display the binding hot spots predicted in all structures of a target kinase, thus providing statistics to assess the robustness of a particular binding site. Here, we use this option to explore 11 sites of the kinases we have studied in the paper, followed by the analysis of four kinases with validated allosteric sites, and one kinase that is the target of current drug discovery efforts. Results are presented in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. We did not include the DFG-out site here, primarily because most structures, either unliganded or bound to a type I inhibitor, are in the DFG-in conformation even for kinases that have a validated type II inhibitor, and thus the statistics of site distribution is not very informative. In addition, as we already noted, it is questionable whether the DFG-out pocket is a genuine allosteric site.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Statistics of Druggable (and Weaker but Still Observable) Allosteric Sites in Various Structures of Specific Kinases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center">site 1</th><th class="rowsep1 colsep0" colspan="2" align="center">site 2</th><th class="rowsep1 colsep0" colspan="2" align="center">site 3</th><th class="rowsep1 colsep0" colspan="2" align="center">site 4</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">site</th><th class="colsep0 rowsep0" align="center">site name</th><th class="colsep0 rowsep0" align="center">inhibitor type</th><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center">example PDB</th><th class="colsep0 rowsep0" align="center" char=".">number of structures</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">number</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">number</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">number</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">number</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">MEK1/2 type III inhibitor</td><td class="colsep0 rowsep0" align="left">III</td><td class="colsep0 rowsep0" align="left">MEK1/2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2">4AN2</a></td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="left"><b>MT3</b></td><td class="colsep0 rowsep0" align="left"><b>33 (36)</b></td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">22 (25)</td><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">5 (19)</td><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">2 (16)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">PDK1 interacting fragment</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">PDK1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RQK">4RQK</a></td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="left"><b>PIF</b></td><td class="colsep0 rowsep0" align="left"><b>43 (53)</b></td><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">23 (47)</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">10 (14)</td><td class="colsep0 rowsep0" align="left">AAS</td><td class="colsep0 rowsep0" align="left">5 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">c-Abl myristoyl pocket</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">c-Abl</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FO0">2FO0</a></td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">22 (23)</td><td class="colsep0 rowsep0" align="left">AAS</td><td class="colsep0 rowsep0" align="left">6 (12)</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">3 (6)</td><td class="colsep0 rowsep0" align="left"><b>CMP</b></td><td class="colsep0 rowsep0" align="left"><b>1 (6)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">D-recruitment site</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">MAPK8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1UKI">1UKI</a></td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">28 (30)</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">23 (28)</td><td class="colsep0 rowsep0" align="left">DEF</td><td class="colsep0 rowsep0" align="left">10 (22)</td><td class="colsep0 rowsep0" align="left"><b>DRS</b></td><td class="colsep0 rowsep0" align="left"><b>0 (6)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DEF</td><td class="colsep0 rowsep0" align="left">docking site for ERK FXF</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">MAPK8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O2M">3O2M</a></td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">28 (30)</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">23 (28)</td><td class="colsep0 rowsep0" align="left"><b>DEF</b></td><td class="colsep0 rowsep0" align="left"><b>10 (22)</b></td><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">0 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LBP</td><td class="colsep0 rowsep0" align="left">lipid binding pocket</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">p38a MAPK</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW">3NEW</a></td><td class="colsep0 rowsep0" align="char" char=".">203</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">181 (202)</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">62 (109)</td><td class="colsep0 rowsep0" align="left"><b>LBP</b></td><td class="colsep0 rowsep0" align="left"><b>35 (70)</b></td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">14 (72)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIG</td><td class="colsep0 rowsep0" align="left">PDIG motif</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">Chk1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS">3JVS</a></td><td class="colsep0 rowsep0" align="char" char=".">113</td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">80 (111)</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">79 (108)</td><td class="colsep0 rowsep0" align="left">AAS</td><td class="colsep0 rowsep0" align="left">28 (47)</td><td class="colsep0 rowsep0" align="left"><b>PDIG</b></td><td class="colsep0 rowsep0" align="left"><b>19 (30)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">EGFR dimerization interface</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RFE">2RFE</a></td><td class="colsep0 rowsep0" align="char" char=".">110</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">33 (35)</td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">30 (46)</td><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">3 (20)</td><td class="colsep0 rowsep0" align="left"><b>EDI</b></td><td class="colsep0 rowsep0" align="left"><b>2 (54)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PMP</td><td class="colsep0 rowsep0" align="left">PKA myristoyl pocket</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">PKA</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CMK">1CMK</a></td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">16 (25)</td><td class="colsep0 rowsep0" align="left"><b>PMP</b></td><td class="colsep0 rowsep0" align="left"><b>14 (28)</b></td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">8 (31)</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">1 (1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AAS</td><td class="colsep0 rowsep0" align="left">Aurora A activation segment</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">Aurora A</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P">4C3P</a></td><td class="colsep0 rowsep0" align="char" char=".">111</td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">101 (108)</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">52 (67)</td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">49 (72)</td><td class="colsep0 rowsep0" align="left"><b>AAS</b></td><td class="colsep0 rowsep0" align="left"><b>12 (27)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">MKK4 p38a peptide</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">MKK4</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALO">3ALO</a></td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">AAS</td><td class="colsep0 rowsep0" align="left">3 (3)</td><td class="colsep0 rowsep0" align="left"><b>MPP</b></td><td class="colsep0 rowsep0" align="left"><b>2 (2)</b></td><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">1(2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">MT3 in EGFR</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3W2R">3W2R</a></td><td class="colsep0 rowsep0" align="char" char=".">110</td><td class="colsep0 rowsep0" align="left"><b>MT3</b></td><td class="colsep0 rowsep0" align="left"><b>33 (35)</b></td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">30 (46)</td><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">3 (20)</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">2 (54)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">PIF in CDK2</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">CDK2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QTU">3QTU</a></td><td class="colsep0 rowsep0" align="char" char=".">371</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">185 (259)</td><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">130 (201)</td><td class="colsep0 rowsep0" align="left"><b>PIF</b></td><td class="colsep0 rowsep0" align="left"><b>96 (109)</b></td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">80 (122)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIG</td><td class="colsep0 rowsep0" align="left">PDIG in JNK3</td><td class="colsep0 rowsep0" align="left">III/IV</td><td class="colsep0 rowsep0" align="left">JNK3</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V6R">3V6R</a></td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">41 (45)</td><td class="colsep0 rowsep0" align="left">EDI</td><td class="colsep0 rowsep0" align="left">40 (47)</td><td class="colsep0 rowsep0" align="left"><b>PDIG</b></td><td class="colsep0 rowsep0" align="left"><b>19 (27)</b></td><td class="colsep0 rowsep0" align="left">DEF</td><td class="colsep0 rowsep0" align="left">14 (42)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIG</td><td class="colsep0 rowsep0" align="left">PDIG in DAPK</td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="left">DAPK</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AV4">5AV4</a></td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="left"><b>PDIG</b></td><td class="colsep0 rowsep0" align="left"><b>25 (43)</b></td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">10 (18)</td><td class="colsep0 rowsep0" align="left">CMP</td><td class="colsep0 rowsep0" align="left">7 (9)</td><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">5 (12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIG</td><td class="colsep0 rowsep0" align="left">PDIG in PKCθ</td><td class="colsep0 rowsep0" align="left">?</td><td class="colsep0 rowsep0" align="left">PDIG</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RA5">4RA5</a></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left"><b>PDIG</b></td><td class="colsep0 rowsep0" align="left"><b>4 (5)</b></td><td class="colsep0 rowsep0" align="left">MPP</td><td class="colsep0 rowsep0" align="left">2 (4)</td><td class="colsep0 rowsep0" align="left">PIF</td><td class="colsep0 rowsep0" align="left">2 (3)</td><td class="colsep0 rowsep0" align="left">DRS</td><td class="colsep0 rowsep0" align="left">0 (2)</td></tr></tbody></table></div></div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> shows that the MT3 site is druggable (includes over 16 probe clusters) in 33 of the 39 MEK1/2 structures in the PDB. The number in parenthesis (36) shows the number of structures with more than 10 probe clusters at the site. The only other site that is predicted to be druggable in many MEK1/2 structures is the MPP pocket (note that the ATP site, while detected in all structures, is not shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The MT3 pocket is also druggable in 33 of the 110 EGFR structures in the PDB, and high affinity allosteric inhibitors targeting the site have been developed.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The PIF pocket is predicted to be druggable in a large fraction of the PDK1 structures, and we have just shown that this site is also important in CDK2. PIF is the most druggable site in MAPK8, although the MAPK8 structures in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> were used to define the DRS and DEF pockets that are weaker sites than PIF. Based on the FTMap results, apart from the MT3 and PIF pockets, druggable sires are predicted in smaller fractions of all structures, demonstrating that kinases are flexible molecules and druggability heavily depends on the conformation of the kinase. While sites that are present in many structures are candidates for inhibitor development, a site can be important even when it is open only in specific structures. For example, the c-Abl myristoyl pocket is generally a weak site, but it is druggable in some structures, and it was recently shown to bind the inhibitor ABL001 with low nanomolar affinity in the interface of SH2 and SH3 binding domains.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> We also note that while some of the sites occur infrequently in the “source” kinases they are primarily associated with, they can be found in many other kinases (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><div class="NLM_p">The last five proteins in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> are not among the 12 “source” kinases considered in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and the results are shown to demonstrate further applications of Kinase Atlas using all structure of a target kinase. First, we already mentioned that the MT3 pocket is a promising site for development of allosteric EGFR inhibitors. The site, which is druggable in many EGFR structures (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), was reported to bind an allosteric inhibitor, EAI045, that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The crystal structure shows that the allosteric site is created by the displacement of the regulatory C-helix in an inactive conformation of the kinase, thus it is clear why the pocket is druggable only in a fraction of structures. The application demonstrates the use of an allosteric site for improved selectivity, as the compound inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochemical assays. As a single agent, EAI045 is still not effective in blocking EGFR-driven proliferation in cells, but it works in synergy with cetuximab, an antibody therapeutic that blocks EGFR dimerization.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p last">The next row in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> shows that the PIF pocket is druggable in many CDK2 structures, in agreement with the experimental validation presented in the previous section. However, the MT3 and CMP sites of CDK2 also show high levels of druggability. The PIF pocket is the most frequently occurring druggable site also in JNK3. However, here we focus on the PDIG pocket, which is druggable in some JNK structures, and it was shown to bind irreversible inhibitors of JNK1/2/3. The co-crystal structure of one of the compounds in JNK3 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V6R">3V6R</a>) shows a long molecule with one end in the PDIG pocket and the other end reaching the ATP site.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> In addition, the compound, JNK-IN-8, forms a covalent bond with a conserved cysteine residue, and it is a selective JNK inhibitor that inhibits phosphorylation of c-Jun, a direct substrate of JNK, in cells exposed to submicromolar drugs.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The PDIG pocket is an allosteric substrate recognition site in DAPK,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and it consistently includes strong hot spots (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). We also studied PKCθ, which was reported as a target for drug discovery by HotSpot Therapeutics. PKCθ is a well-validated immunokinase in autoimmune and rare diseases, but its active-site inhibitors lack selectivity.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The protein has only five structures in the PDB, but the PDIG pocket is druggable in four of these structures, and there is no competing site that could offer comparable druggability (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Existing Kinase Databases</h3><div class="NLM_p last">The Kinase Atlas is the first database to focus on allosteric sites in kinases and their ability to bind potential ligands. The most similar existing database would likely be KLIFS, a structural kinase–ligand interaction database that covers the region of the kinase between the N- and C-terminal domains—which would include the DFG-out and MT3 sites—and provides detailed structural and ligand-binding information for all human and mouse kinase structures available in the PDB.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> KLIFS provides a consistent numbering scheme for kinase residues and descriptions of subpockets within the catalytic cleft, which makes it simpler to compare how known ligands bind to kinases and identify potential patterns in kinase–ligand interactions. The goal of the Kinase Atlas, on the other hand, is to identify which allosteric sites on each kinase might be suited for inhibitor development, even without ligand-binding data for that site—which is likely to be the case for many kinases of future interest.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> How to Use the Kinase Atlas</h3><div class="NLM_p">The Kinase Atlas is available at <a href="https://kinase-atlas.bu.edu" class="extLink">https://kinase-atlas.bu.edu</a> and contains FTMap results for all kinase structures available in the PDB. Each structure has its own page, where users may access consensus site data (including whether consensus sites corresponded to any allosteric sites), or they may download or visualize mapping results for that structure; downloaded results are available as PyMOL session files containing the protein structure that was mapped and the resulting consensus sites, which indicate the regions on the surface that would be most likely to bind ligands. For users interested in a particular kinase, summarized mapping results for the structures associated with each kinase (based on their UniProt accessions) are also available, listing the strongest (if any) consensus sites associated with each allosteric site on each structure.</div><div class="NLM_p">As an example, the serine/threonine kinase PKR (protein kinase R) has been linked to breast cancer,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> hepatocellular carcinoma,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and Huntington’s disease,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> but it has just three structures of the full catalytic domain in the PDB (2A19, 2A1A, and 3UIU) and 271 bioactivities listed in ChEMBL (many other kinases have thousands of associated bioactivities). FTMap found that the AAS and PDIG sites were likely to be druggable in PKR, with both of them having strong consensus sites; several smaller consensus sites were also located near these pockets, fulfilling the criteria described by Kozakov et al. for being druggable by traditional druglike compounds.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a><a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A shows the strong hot spots predicted by FTMap for the apo structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UIU">3UIU</a>_B of PKR. The strongest hot spot, 0(19), shown in magenta, is located at the AAS site. The second strongest hot spot, 1(18), shown in coral, is at the PDIG site. The latter also accommodates the secondary consensus sites 7(5) (in green) and 8(4) (in blue) indicating more minor hot spots. For orientation, we superimposed the Chk1 inhibitor (compound <b>3</b>, shown in yellow) from the Chk1 structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS">3JVS</a>. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B shows a close-up of the PDIG site with PKR in surface representation to show that at this site, PKR has a well-defined pocket that could accommodate small compounds.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. FTMap results for apo structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UIU">3UIU</a>_B of PKR. (A) Strongest hot spot, 0(19), shown in magenta, located at the AAS site. The second strongest hot spot, 1(18), shown in coral, is located at the PDIG site. The PDIG site also accommodates the secondary consensus sites 7(5) (in green) and 8(4) (in blue). For orientation, we superimposed the Chk1 inhibitor (shown in yellow) from the Chk1 structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS">3JVS</a>. (B) Close-up of the PDIG site with PKR in surface representation to show that at this site, PKR has a well-defined pocket that could accommodate small compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Selection and Naming of Allosteric Sites</h3><div class="NLM_p last">The two most well-known kinase allosteric sites would be the DFG-out and MT3 pockets, both of which have FDA-approved inhibitors, and a literature search turned up descriptions of six other sites (PIF, CMP, DRS, PDIG, DEF, and LBP). The remaining four sites (EDI, MPP, PMP, and AAS) were identified after mapping was performed on all kinase structures, and several regions were frequently found to have strong consensus sites that were not associated with any of the previously identified allosteric sites. Kinases that were found to have these hot spots then had their structures aligned with all other kinase structures to identify those that had either a protein–protein interaction site or a bound ligand/peptide in the same location as the consensus sites, and some of these structures were associated with publications that described an allosteric site in the regions of interest (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">Figure S1</a>). Some kinase allosteric sites have established names; these tend to be based on either the “source” kinase (PIF, DEF), a motif located near the site (DFG), or a ligand/peptide that binds to the site (DRS, DEF). The remaining eight sites were thus named similarly, with all of them referencing the “source” kinase and ligand aside from PDIG (based on a motif) and LBP (based on the ligand only).</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63138" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63138" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The existing FDA-approved kinase inhibitors are overwhelmingly ATP-competitive, and they are also disproportionate in which kinases and diseases they target. As of 2015, 18 out of the 27 protein kinase inhibitors targeted tyrosine kinases (that comprise only 90 out of the 518 human kinases),<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and 26 out of 28 kinase inhibitors were intended to treat cancer, even though many other diseases are associated with kinases.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Similarly, the amount of structural and bioactivity data available for different kinases is also uneven, with over a quarter of the kinase bioactivity data from ChEMBL covering just 18 kinases, and the most popular 10 kinases accounting for over 40% of kinase structures.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> However, a number of kinases that are well-validated targets for the treatment of various diseases do not have efficient active-site modulators because of the limited selectivity of their ATP binding site or because of the high mutation rate in the region leading to resistance. While developing allosteric modulators to avoid the problem could be a promising approach, only a few allosteric inhibitors have been approved by the FDA. Thus, for the majority of kinases without much ligand-binding data available, the Kinase Atlas could be a valuable resource for determining which regions are likely to be druggable because FTMap can detect binding sites even in unliganded structures. We have shown examples of sites detected by Kinase Atlas that bind allosteric modulators reported in the literature. Although such modulators generally have moderate affinity, their combination with ATP competitive inhibitors can provide improved selectivity and resistance to mutations. Another avenue toward improved affinity can be the development of covalent inhibitors if the target includes a cystine residue at suitable location. Thus, we hope that Kinase Atlas will be used and results will be reported in the literature.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37161" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37161" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this section, we describe both the computational tools used for developing the Kinase Atlas database and the experimental methods applied for the validation of the predicted PIF pocket in CDK2.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Computational Solvent Mapping by FTMap</h3><div class="NLM_p last">FTMap identifies binding hot spots on the surface of a protein by finding the most favorable regions for 16 small molecular probes of various shapes and polarities (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">Figure S2</a>).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Given a protein structure, for each probe, the algorithm places the tens of thousands of copies all over the surface on the basis of dense rotational and translational grids, retains the most favorable probe positions by energy and refines their orientations, and then clusters the probe molecules by location and ranks them by their average energy. The lowest energy probe clusters of each probe type are retained, and clustering is performed once more on clusters of all probe types to form the consensus sites, which are ranked by their population of probe clusters. Consensus sites identify the locations of binding hot spots on the protein surface, and their rank corresponds to the relative strength and importance of the associated hot spot.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Kinase Structure Selection</h3><div class="NLM_p last">To obtain a list of kinase catalytic domain structures available for mapping, two resources were used: Pfam,<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> a database for protein families, which groups proteins by sequence and matches them to PDB structures through UniProt,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> and the Gene Ontology (GO) project,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> which describes gene products by their biological processes, molecular functions, and cellular components. A structure had to be classified as a “protein kinase domain” (Pkinase) by Pfam and having “protein kinase activity” as a molecular function by GO to be included for mapping. The final list contained 4910 structures of 376 distinct kinases available in the Protein Data Bank; 239 of these were human kinases.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Mapping Preparation</h3><div class="NLM_p last">After each kinase structure was downloaded from the Protein Data Bank,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> it was split into its N- and C-terminal domains before mapping (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">Figure S3</a>). The active site in kinases is located between the domains and binds with high affinity to ATP, so separating the domains before mapping is required to break up the ATP site and allow potential allosteric sites to be detected. CATH, a database that classifies protein domains by a secondary structure, was used to identify the domains in each structure.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> For structures without an entry in CATH, the classification from a similar structure (identified using BLAST) was applied.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Assignment of Mapping Results to Allosteric Sites</h3><div class="NLM_p">Each allosteric site was assigned a representative structure from the kinase of origin with a ligand (small molecule/inhibitor or peptide) known to bind at the allosteric site.</div><div class="NLM_p">After using PyMOL to align each representative structure to each mapped structure, consensus sites were assigned to allosteric sites based on whether they overlapped with a representative ligand. A consensus site that overlapped with multiple representative ligands was assigned to the allosteric site with which it had the greatest overlap. Overlap between a consensus site and representative ligand was found by using SciPy to calculate the convex hull of the ligand and determining whether any consensus site atoms were located within the convex hull.</div><div class="NLM_p last">We note that the splitting and separate mapping of N and C domains may result in a false positive hot spot for some kinases. This hot spot is on the C-lobe in a pocket normally occupied by the side chains of residues from the loop between the C-end of the αC helix and the next β-strand on the N-lobe. However, because there is no putative allosteric site in the vicinity, the probes binding to this region do not impact any of the 12 sites shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Druggability Assessment</h3><div class="NLM_p last">The strength of a consensus site (based on its population) can be considered as a measure of the potential binding affinity at that site. A kinase was considered druggable at a particular allosteric site if at least one of its structures had a strong consensus site (at least 16 probe clusters) assigned to that allosteric site. For a site to be druggable using conventional small-molecule drugs, the positions of other nearby consensus sites would also need to be considered, but the main factor in determining whether a site would be an appropriate drug target is its potential to bind ligands with high affinity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Pockets with a slightly weaker consensus site (at least 13 probe clusters) would be considered borderline druggable, but in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, we have listed even the number of structures with 10 or more probe clusters at the considered site.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Experimental Validation of the PIF Pocket in CDK2</h3><div class="NLM_p">Full-length human CDK2 was cloned into the pMCSG10 bacterial expression vector, which appends a 6xHis-GST purification tag to the N-terminus of CDK2, which can be later removed by TEV protease. To enable tethering at the PIF pocket of CDK2, Cys118 and Cys177 were mutated by alanine to prevent off-target labeling at these sites and Lys56 was mutated to Cys to enable targeting of fragments to the adjacent PIF pocket. CDK2 was expressed in the Rosetta <i>Escherichia coli</i> strain by induction with IPTG at 16 °C and growth overnight. Cells were collected by centrifugation, resuspended in lysis buffer [50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid pH 7.5, 150 mM NaCl, 10 mM MgCl2, 2 mM dithiothreitol, 5% glycerol] with protease inhibitors, and lysed using a microfluidizer. The clarified lysate was batch-loaded onto Ni-NTA resin, washed, and eluted with imidazole. The sample was dialyzed overnight in lysis buffer with 10 mM imidazole and 1:20 w/w TEV protease to cleave the purification tag. The dialyzed mixture was passed over a Ni-NTA column and the flow-through was collected. Finally, the sample was purified by size exclusion chromatography on a Superdex 200 16/60 column in lysis buffer. The protein was >95% pure by sodium dodecyl sulfate-polyacrylamide gel electrophoresis, and its identity was confirmed by intact protein mass spectrometry. Although it was not validated that the mutations did affect structural integrity, the tethering technology has been successfully used for exploring suspected and orphan allosteric sites in a variety of proteins. In particular, the method was used to study ligand binding to the PIF pocket of PDK1 by introducing individual Cys mutation at six positions around the pocket.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> It was reported that the mutants were somewhat less catalytically active than the WT PDK1. Nevertheless, the study discovered a variety of small-molecule fragment disulfides that could either activate or inhibit PDK1 by conjugation to the PIF pocket.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Considering the success of this and a variety of other applications, the tethering method remains an important tool in the UCSF Small Molecule Discovery Center.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></div><div class="NLM_p last">Screening for fragment binding to CDK2 was performed by monitoring mixed disulfide formation between the protein and the fragment using intact protein mass spectrometry on a Waters LCT Premier (LC-TOF) with in-line C4 desalting column (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">Figure S4</a>). Each tethering reaction was run in Tris-buffered saline containing 2 μM CDK2, 100 μM of a small-molecule disulfide, and 1 mM βME as a reductant. From the primary screen of 1280 fragments, the top 60 hits were cherry-picked and retested to confirm. The relative potency of the top 12 confirmed hits was measured by determining the concentration of β-mercaptoethanol required to displace half of the fragment from the protein (βME<sub>50</sub>), with higher values indicative of a stronger noncovalent interaction between the fragment and the protein. The impact of the top 12 hits on CDK2 function was assessed by first labeling CDK2 with 100 μM of each fragment in the presence of 800 μM βME and then measuring CDK2 catalytic activity in the absence or presence of its activator Cyclin A2 using a radioactive kinase assay with [γ-<sup>32</sup>P]-ATP and Histone H1 peptide as the substrates.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00089" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38545" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38545" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00089" class="ext-link">10.1021/acs.jmedchem.9b00089</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Diagram representing the definition of the 12 putative allosteric sites; probe molecules used by FTMap; flowchart of developing Kinase Atlas; and tethering, a disulfide-based fragment screening method (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf">jm9b00089_si_001.pdf (340.33 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00089" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86048" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86048" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandor Vajda</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, 
    Department of Chemistry, 
     and , Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1540-8220" title="Orcid link">http://orcid.org/0000-0003-1540-8220</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4234232826230220376c272637"><span class="__cf_email__" data-cfemail="d0a6b1bab4b190b2a5feb5b4a5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dima Kozakov</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics and Statistics, 
    Laufer Center for Physical and Quantitative
Biology, 
     and , Stony Brook University, New York 11794, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0464-4500" title="Orcid link">http://orcid.org/0000-0003-0464-4500</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5f32363b3e2c1f333e2a393a2d3c3a312b3a2d71302d38"><span class="__cf_email__" data-cfemail="234e4a474250634f425645465140464d5746510d4c5144">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Yueh</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering and , Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin Rettenmaier</span> - <span class="hlFld-Affiliation affiliation">Departments of Pharmaceutical Chemistry
and Cellular and Molecular Pharmacology, University of California, 1700 Fourth Street, San Francisco, California 9415, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Xia</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering and , Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David R. Hall</span> - <span class="hlFld-Affiliation affiliation">Acpharis Incorporated, Holliston, Massachusetts 01746, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrey Alekseenko</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics and Statistics, 
    Laufer Center for Physical and Quantitative
Biology, 
     and , Stony Brook University, New York 11794, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathryn A. Porter</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering and , Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krister Barkovich</span> - <span class="hlFld-Affiliation affiliation">Departments of Pharmaceutical Chemistry
and Cellular and Molecular Pharmacology, University of California, 1700 Fourth Street, San Francisco, California 9415, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gyorgy Keseru</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry Research Group, Research Center
for Natural Sciences, Magyar tudósok krt. 2, H-1117 Budapest, Hungary</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adrian Whitty</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry and , Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0331-4369" title="Orcid link">http://orcid.org/0000-0002-0331-4369</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James A. Wells</span> - <span class="hlFld-Affiliation affiliation">Departments of Pharmaceutical Chemistry
and Cellular and Molecular Pharmacology, University of California, 1700 Fourth Street, San Francisco, California 9415, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.Y. and B.X. performed the analysis of kinases and built the Kinase Atlas database. J.R. and K.B. performed CDK2 tethering experiments. J.A.W. planned and supervised the experimental work. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): D.R.H. is a full-time employee of Acpharis, Inc.<br /></br><p class="inlineNote">Acpharis Inc. offers commercial licenses to ATLAS, a software product in function similar to FTMap. Dima Kozakov and Sandor Vajda owns stock in the company. However, the FTMap program and the use of the FTMap server and the Kinase Atlas database are free for academic and governmental use.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This investigation was supported by grant R35-GM118078 from the National Institute of General Medical Sciences, grant AF 1816314 from the National Science Foundation, and grant K-116904 from National Research, Development and Innovation Office of Hungary.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AAS</td><td class="NLM_def"><p class="first last">Aurora A activation segment</p></td></tr><tr><td class="NLM_term">AGC</td><td class="NLM_def"><p class="first last">protein kinase AGC</p></td></tr><tr><td class="NLM_term">BCR</td><td class="NLM_def"><p class="first last">breakpoint cluster region</p></td></tr><tr><td class="NLM_term">CML</td><td class="NLM_def"><p class="first last">chronic myelogenous leukemia</p></td></tr><tr><td class="NLM_term">CMP</td><td class="NLM_def"><p class="first last">c-Abl myristoyl pocket</p></td></tr><tr><td class="NLM_term">DEF</td><td class="NLM_def"><p class="first last">docking site for ERK FXF</p></td></tr><tr><td class="NLM_term">DFG</td><td class="NLM_def"><p class="first last">Asp-Phe-Glu</p></td></tr><tr><td class="NLM_term">EDI</td><td class="NLM_def"><p class="first last">EGFR dimerization interface</p></td></tr><tr><td class="NLM_term">HM</td><td class="NLM_def"><p class="first last">hydrophobic motif</p></td></tr><tr><td class="NLM_term">JIP1</td><td class="NLM_def"><p class="first last">JNK-interacting protein 1</p></td></tr><tr><td class="NLM_term">JNK</td><td class="NLM_def"><p class="first last">c-Jun N-terminal kinase</p></td></tr><tr><td class="NLM_term">KLIFS</td><td class="NLM_def"><p class="first last">Kinase–Ligand Interaction Fingerprints and Structures database</p></td></tr><tr><td class="NLM_term">LBP</td><td class="NLM_def"><p class="first last">lipid binding pocket</p></td></tr><tr><td class="NLM_term">MEK1/2</td><td class="NLM_def"><p class="first last">dual specificity mitogen-activated protein kinase kinase 1/2</p></td></tr><tr><td class="NLM_term">MKK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinase</p></td></tr><tr><td class="NLM_term">MPP</td><td class="NLM_def"><p class="first last">MKK4 p38a peptide</p></td></tr><tr><td class="NLM_term">MT3</td><td class="NLM_def"><p class="first last">MEK1/2 type III inhibitor</p></td></tr><tr><td class="NLM_term">PDIG</td><td class="NLM_def"><p class="first last">Pro-Asp-Ile-Gly motif</p></td></tr><tr><td class="NLM_term">PDK1</td><td class="NLM_def"><p class="first last">phosphoinositide-dependent kinase 1</p></td></tr><tr><td class="NLM_term">PIA</td><td class="NLM_def"><p class="first last">phosphatidylinositol ether lipid analogue</p></td></tr><tr><td class="NLM_term">PIF</td><td class="NLM_def"><p class="first last">PDK1 interacting fragment</p></td></tr><tr><td class="NLM_term">PKR</td><td class="NLM_def"><p class="first last">protein kinase R</p></td></tr><tr><td class="NLM_term">PMP</td><td class="NLM_def"><p class="first last">PKA myristoyl pocket</p></td></tr><tr><td class="NLM_term">SH2/3</td><td class="NLM_def"><p class="first last">Src homology 2/3</p></td></tr><tr><td class="NLM_term">TEV</td><td class="NLM_def"><p class="first last">tobacco etch virus</p></td></tr><tr><td class="NLM_term">TIE2</td><td class="NLM_def"><p class="first last">tunica interna endothelial cell kinase</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 69 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lh2YuTDZuIJ7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span> </span><span class="NLM_article-title">Human Protein Kinase Knowledgebase</span>. <a href="http://www.kinasenet.ca" class="extLink">http://www.kinasenet.ca</a> (accessed May 13, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Human+Protein+Kinase+Knowledgebase.+http%3A%2F%2Fwww.kinasenet.ca+%28accessed+May+13%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DHuman%2520Protein%2520Kinase%2520Knowledgebase%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manning, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudarsanam, S.</span></span> <span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1912</span>– <span class="NLM_lpage">1934</span>, <span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lh2YuTDZuIJ7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brylinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skolnick, J.</span></span> <span> </span><span class="NLM_article-title">Comprehensive structural and functional characterization of the human kinome by protein structure modeling and ligand virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1839</span>– <span class="NLM_lpage">1854</span>, <span class="refDoi"> DOI: 10.1021/ci100235n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100235n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1839-1854&author=M.+Brylinskiauthor=J.+Skolnick&title=Comprehensive+structural+and+functional+characterization+of+the+human+kinome+by+protein+structure+modeling+and+ligand+virtual+screening&doi=10.1021%2Fci100235n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening</span></div><div class="casAuthors">Brylinski, Michal; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1839-1854</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The growing interest in the identification of kinase inhibitors, promising therapeutics in the treatment of many diseases, has created a demand for the structural characterization of the entire human kinome.  At the outset of the drug development process, the lead-finding stage, approaches that enrich the screening library with bioactive compds. are needed.  Here, protein structure based methods can play an important role, but despite structural genomics efforts, it is unlikely that the three-dimensional structures of the entire kinome will be available soon.  Therefore, at the proteome level, structure-based approaches must rely on predicted models, with a key issue being their utility in virtual ligand screening.  In this study, we employ the recently developed FINDSITE/Q-Dock ligand homol. modeling approach, which is well-suited for proteome-scale applications using predicted structures, to provide extensive structural and functional characterization of the human kinome.  Specifically, we construct structure models for the human kinome; these are subsequently subject to virtual screening against a library of more than 2 million compds.  To rank the compds., we employ a hierarchical approach that combines ligand- and structure-based filters.  Modeling accuracy is carefully validated using available exptl. data with particularly encouraging results found for the ability to identify, without prior knowledge, specific kinase inhibitors.  More generally, the modeling procedure results in a large no. of predicted mol. interactions between kinases and small ligands that should be of practical use in the development of novel inhibitors.  The data set is freely available to the academic community via a user-friendly Web interface at http://cssb.biol.gatech.edu/kinomelhm/ as well as at the ZINC Web site (http://zinc.docking.org/applications/2010Apr/Brylinski-2010.tar.gz).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsNAiYM1dJuLVg90H21EOLACvtfcHk0lh3uGpcq6naKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL&md5=4dd283d41804d7f05138f5acf02ad404</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fci100235n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100235n%26sid%3Dliteratum%253Aachs%26aulast%3DBrylinski%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DComprehensive%2520structural%2520and%2520functional%2520characterization%2520of%2520the%2520human%2520kinome%2520by%2520protein%2520structure%2520modeling%2520and%2520ligand%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1839%26epage%3D1854%26doi%3D10.1021%2Fci100235n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Blue
Ridge Institute for Medical Research</span>. <span> </span><span class="NLM_article-title">FDA-approved protein kinase inhibitors</span>. <a href="http://www.brimr.org/PKI/PKIs.htm" class="extLink">http://www.brimr.org/PKI/PKIs.htm</a> (accessed May 13, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Blue%0ARidge+Institute+for+Medical+Research&title=FDA-approved+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DFDA-approved%2520protein%2520kinase%2520inhibitors%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span> <span> </span><span class="NLM_article-title">Structure-based virtual screening approaches in kinase-directed drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2235</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.2174/1568026617666170224121313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.2174%2F1568026617666170224121313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28240180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Smt7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=2235-2259&author=D.+Bajuszauthor=G.+G.+Ferenczyauthor=G.+M.+Keseru&title=Structure-based+virtual+screening+approaches+in+kinase-directed+drug+discovery&doi=10.2174%2F1568026617666170224121313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Virtual Screening Approaches in Kinase-directed Drug Discovery</span></div><div class="casAuthors">Bajusz, David; Ferenczy, Gyorgy G.; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2235-2259</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Protein kinases are one of the most targeted protein families in current drug discovery pipelines.  They are implicated in many oncol., inflammatory, CNS-related and other clin. indications.  Virtual screening is a computational technique with a diverse set of available tools that has been shown many times to provide novel starting points for kinase-directed drug discovery.  This review starts with a concise overview of the function, structural features and inhibitory mechanisms of protein kinases.  In addn. to briefly reviewing the practical aspects of structure-based virtual screenings, we discuss several case studies to illustrate the state of the art in the virtual screening for type I, type II, allosteric (type III-V) and covalent (type VI) kinase inhibitors.  With this review, we strive to provide a summary of the latest advances in the structure-based discovery of novel kinase inhibitors, as well as a practical tool to anyone who wishes to embark on such an endeavor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoANpN3hgHTtLVg90H21EOLACvtfcHk0lh3uGpcq6naKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Smt7nF&md5=5b85e22ae4fbf7b25cbb70fc14d147a3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170224121313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170224121313%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DStructure-based%2520virtual%2520screening%2520approaches%2520in%2520kinase-directed%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D2235%26epage%3D2259%26doi%3D10.2174%2F1568026617666170224121313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Strategies for the selective regulation of kinases with allosteric modulators: Exploiting exclusive structural features</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1021/cb300663j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300663j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCls7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=58-70&author=Z.+Fangauthor=C.+Gr%C3%BCtterauthor=D.+Rauh&title=Strategies+for+the+selective+regulation+of+kinases+with+allosteric+modulators%3A+Exploiting+exclusive+structural+features&doi=10.1021%2Fcb300663j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the Selective Regulation of Kinases with Allosteric Modulators: Exploiting Exclusive Structural Features</span></div><div class="casAuthors">Fang, Zhizhou; Gruetter, Christian; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-70</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The modulation of kinase function has become an important goal in modern drug discovery and chem. biol. research.  In cancer-targeted therapies, kinase inhibitors have been experiencing an upsurge, which can be measured by the increasing no. of kinase inhibitors approved by the FDA in recent years.  However, lack of efficacy, limited selectivity, and the emergence of acquired drug resistance still represent major bottlenecks in the clinic and challenge inhibitor development.  Most known kinase inhibitors target the active kinase and are ATP competitive.  A second class of small org. mols., which address remote sites of the kinase and stabilize enzymically inactive conformations, is rapidly moving to the forefront of kinase inhibitor research.  Such allosteric modulators bind to sites that are less conserved across the kinome and only accessible upon conformational changes.  These mols. are therefore thought to provide various advantages such as higher selectivity and extended drug target residence times.  This review highlights various strategies that have been developed to utilizing exclusive structural features of kinases and thereby modulating their activity allosterically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSjxxztmczXrVg90H21EOLACvtfcHk0lh3uGpcq6naKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCls7bN&md5=50864096bf66229ff74482a9b56847d6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb300663j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300663j%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStrategies%2520for%2520the%2520selective%2520regulation%2520of%2520kinases%2520with%2520allosteric%2520modulators%253A%2520Exploiting%2520exclusive%2520structural%2520features%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D58%26epage%3D70%26doi%3D10.1021%2Fcb300663j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vijayan, R. S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong-Ly, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbrack, R. L.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. M.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of the DFG-out kinase motif and biochemical profiling of structurally validated type II inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">466</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1021/jm501603h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501603h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=466-479&author=R.+S.+K.+Vijayanauthor=P.+Heauthor=V.+Modiauthor=K.+C.+Duong-Lyauthor=H.+Maauthor=J.+R.+Petersonauthor=R.+L.+Dunbrackauthor=R.+M.+Levy&title=Conformational+analysis+of+the+DFG-out+kinase+motif+and+biochemical+profiling+of+structurally+validated+type+II+inhibitors&doi=10.1021%2Fjm501603h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of the DFG-Out Kinase Motif and Biochemical Profiling of Structurally Validated Type II Inhibitors</span></div><div class="casAuthors">Vijayan, R. S. K.; He, Peng; Modi, Vivek; Duong-Ly, Krisna C.; Ma, Haiching; Peterson, Jeffrey R.; Dunbrack, Roland L.; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">466-479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural coverage of the human kinome has been steadily increasing over time.  The structures provide valuable insights into the mol. basis of kinase function and also provide a foundation for understanding the mechanisms of kinase inhibitors.  There are a large no. of kinase structures in the PDB for which the Asp and Phe of the DFG motif on the activation loop swap positions, resulting in the formation of a new allosteric pocket.  The authors refer to these structures as "classical DFG-out" conformations in order to distinguish them from conformations that have also been referred to as DFG-out in the literature but that do not have a fully formed allosteric pocket.  The authors have completed a structural anal. of almost 200 small mol. inhibitors bound to classical DFG-out conformations; the authors find that they are recognized by both type I and type II inhibitors.  In contrast, the authors find that nonclassical DFG-out conformations strongly select against type II inhibitors because these structures have not formed a large enough allosteric pocket to accommodate this type of binding mode.  In the course of this study the authors discovered that the no. of structurally validated type II inhibitors that can be found in the PDB and that are also represented in publicly available biochem. profiling studies of kinase inhibitors is very small.  The authors have obtained new profiling results for several addnl. structurally validated type II inhibitors identified through the conformational anal.  Although the available profiling data for type II inhibitors is still much smaller than for type I inhibitors, a comparison of the two data sets supports the conclusion that type II inhibitors are more selective than type I.  The authors comment on the possible contribution of the DFG-in to DFG-out conformational reorganization to the selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb9E5JKOmDZ7Vg90H21EOLACvtfcHk0lhjNZ4chzVN5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP&md5=6d729997f242d582a4adff9463fd585c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm501603h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501603h%26sid%3Dliteratum%253Aachs%26aulast%3DVijayan%26aufirst%3DR.%2BS.%2BK.%26aulast%3DHe%26aufirst%3DP.%26aulast%3DModi%26aufirst%3DV.%26aulast%3DDuong-Ly%26aufirst%3DK.%2BC.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DConformational%2520analysis%2520of%2520the%2520DFG-out%2520kinase%2520motif%2520and%2520biochemical%2520profiling%2520of%2520structurally%2520validated%2520type%2520II%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D466%26epage%3D479%26doi%3D10.1021%2Fjm501603h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Allosteric MEK1/2 inhibitors including cobimetanib and trametinib in the treatment of cutaneous melanomas</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2016.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.phrs.2016.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27956260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSnsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=20-31&author=R.+Roskoski&title=Allosteric+MEK1%2F2+inhibitors+including+cobimetanib+and+trametinib+in+the+treatment+of+cutaneous+melanomas&doi=10.1016%2Fj.phrs.2016.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric MEK1/2 inhibitors including cobimetanib and trametinib in the treatment of cutaneous melanomas</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20-31</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Ras-Raf-MEK-ERK (Map kinase) cellular pathway is a highly conserved eukaryotic signaling module that transduces extracellular signals from growth factors and cytokines into intracellular regulatory events that are involved in cell growth and proliferation or the contrary pathway of cell differentiation.  Dysregulation of this pathway occurs in more than one-third of all malignancies, a process that has fostered the development of targeted Map kinase pathway inhibitors.  Cutaneous melanomas, which arise from skin melanocytes, are the most aggressive form of skin cancer.  Mutations that activate the Map kinase pathway occur in more than 90% of these melanomas.  This has led to the development of the combination of dabrafenib and trametinib or vemurafenib and cobimetanib for the treatment of BRAF V600E mutant melanomas.  Dabrafenib and vemurafenib target V600E/K BRAF mutants while trametinib and cobimetanib target MEK1/2.  The latter two agents bind to MEK1/2 at a site that is adjacent to, but sep. from, the ATP-binding site and are therefore classified as type III allosteric protein kinase inhibitors.  These agents form a hydrogen bond with a conserved β3-lysine and they make numerous hydrophobic contacts with residues within the αC-helix, the β5 strand, and within the activation segment, regions of the protein kinase domain that exhibit greater diversity than those found within the ATP-binding site.  One advantage of such allosteric inhibitors is that they do not have to compete with millimolar concns. of cellular ATP, which most FDA-approved small mol. competitive inhibitors such as imatinib must do.  Owing to the wide spread activation of this pathway in numerous neoplasms, trametinib and cobimetinib are being studied in combination with other targeted and cytotoxic drugs in a variety of clin. situations.  Except for BRAF and NRAS mutations, there are no other biomarkers correlated with treatment responses following MEK1/2 inhibition and the discovery of such biomarkers would represent an important therapeutic breakthrough.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJSWK7CmuK_7Vg90H21EOLACvtfcHk0lhjNZ4chzVN5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSnsbnO&md5=db7220ba32cfe520a99ec59e4ba33c6f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2016.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2016.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DAllosteric%2520MEK1%252F2%2520inhibitors%2520including%2520cobimetanib%2520and%2520trametinib%2520in%2520the%2520treatment%2520of%2520cutaneous%2520melanomas%26jtitle%3DPharmacol.%2520Res.%26date%3D2017%26volume%3D117%26spage%3D20%26epage%3D31%26doi%3D10.1016%2Fj.phrs.2016.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohnuud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mottarella, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">755</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lhjNZ4chzVN5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhjNZ4chzVN5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">New frontiers in druggability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9063</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9063-9088&author=D.+Kozakovauthor=D.+R.+Hallauthor=R.+L.+Napoleonauthor=C.+Yuehauthor=A.+Whittyauthor=S.+Vajda&title=New+frontiers+in+druggability&doi=10.1021%2Facs.jmedchem.5b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New Frontiers in Druggability</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Napoleon, Raeanne L.; Yueh, Christine; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9063-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A powerful early approach to evaluating the druggability of proteins involved detg. the hit rate in NMR-based screening of a library of small compds.  Here, we show that a computational analog of this method, based on mapping proteins using small mols. as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree.  We apply the method to a large set of proteins.  The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compds. beyond those resembling validated targets and conventionally druglike ligands.  In particular, the method identifies targets that, while not druggable by druglike compds., may become druggable using compd. classes such as macrocycles or other large mols. beyond the rule-of-five limit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWoMwMpuM2bVg90H21EOLACvtfcHk0ljbZhOHCyAOQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF&md5=77fd532a1f0129d096eb7a1fbfb1d422</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNapoleon%26aufirst%3DR.%2BL.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DNew%2520frontiers%2520in%2520druggability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9063%26epage%3D9088%26doi%3D10.1021%2Facs.jmedchem.5b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foda, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span> <span> </span><span class="NLM_article-title">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">5939</span>, <span class="refDoi"> DOI: 10.1038/ncomms6939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fncomms6939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25600932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5939&author=Z.+H.+Fodaauthor=Y.+Shanauthor=E.+T.+Kimauthor=D.+E.+Shawauthor=M.+A.+Seeliger&title=A+dynamically+coupled+allosteric+network+underlies+binding+cooperativity+in+Src+kinase&doi=10.1038%2Fncomms6939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span></div><div class="casAuthors">Foda, Zachariah H.; Shan, Yibing; Kim, Eric T.; Shaw, David E.; Seeliger, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5939</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein tyrosine kinases are attractive drug targets because many human diseases are assocd. with the deregulation of kinase activity.  However, how the catalytic kinase domain integrates different signals and switches from an active to an inactive conformation remains incompletely understood.  Here we identify an allosteric network of dynamically coupled amino acids in Src kinase that connects regulatory sites to the ATP- and substrate-binding sites.  Surprisingly, reactants (ATP and peptide substrates) bind with neg. cooperativity to Src kinase while products (ADP and phosphopeptide) bind with pos. cooperativity.  We confirm the mol. details of the signal relay through the allosteric network by biochem. studies.  Expts. on two addnl. protein tyrosine kinases indicate that the allosteric network may be largely conserved among these enzymes.  Our work provides new insights into the regulation of protein tyrosine kinases and establishes a potential conduit by which resistance mutations to ATP-competitive kinase inhibitors can affect their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGfRZJun_T1rVg90H21EOLACvtfcHk0ljbZhOHCyAOQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ&md5=cc24df9830ccb1525f66cc554795965c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fncomms6939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6939%26sid%3Dliteratum%253Aachs%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26atitle%3DA%2520dynamically%2520coupled%2520allosteric%2520network%2520underlies%2520binding%2520cooperativity%2520in%2520Src%2520kinase%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D5939%26doi%3D10.1038%2Fncomms6939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornev, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span> <span> </span><span class="NLM_article-title">Allostery through the computational microscope: cAMP activation of a canonical signalling domain</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7588</span>, <span class="refDoi"> DOI: 10.1038/ncomms8588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fncomms8588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26145448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FgvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7588&author=R.+D.+Malmstromauthor=A.+P.+Kornevauthor=S.+S.+Taylorauthor=R.+E.+Amaro&title=Allostery+through+the+computational+microscope%3A+cAMP+activation+of+a+canonical+signalling+domain&doi=10.1038%2Fncomms8588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery through the computational microscope: cAMP activation of a canonical signalling domain</span></div><div class="casAuthors">Malmstrom Robert D; Amaro Rommie E; Kornev Alexandr P; Taylor Susan S</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7588</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ligand-induced protein allostery plays a central role in modulating cellular signalling pathways.  Here using the conserved cyclic nucleotide-binding domain of protein kinase A's (PKA) regulatory subunit as a prototype signalling unit, we combine long-timescale, all-atom molecular dynamics simulations with Markov state models to elucidate the conformational ensembles of PKA's cyclic nucleotide-binding domain A for the cAMP-free (apo) and cAMP-bound states.  We find that both systems exhibit shallow free-energy landscapes that link functional states through multiple transition pathways.  This observation suggests conformational selection as the general mechanism of allostery in this canonical signalling domain.  Further, we expose the propagation of the allosteric signal through key structural motifs in the cyclic nucleotide-binding domain and explore the role of kinetics in its function.  Our approach integrates disparate lines of experimental data into one cohesive framework to understand structure, dynamics and function in complex biological systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNy5_7QVOqG9yzdcP07UmufW6udTcc2eZJ2qcwSGyT5Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FgvFWrsA%253D%253D&md5=fdc93fe9e6e29b9dc69e1e007b08233a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms8588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8588%26sid%3Dliteratum%253Aachs%26aulast%3DMalmstrom%26aufirst%3DR.%2BD.%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26atitle%3DAllostery%2520through%2520the%2520computational%2520microscope%253A%2520cAMP%2520activation%2520of%2520a%2520canonical%2520signalling%2520domain%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7588%26doi%3D10.1038%2Fncomms8588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J. E.</span></span> <span> </span><span class="NLM_article-title">Predicting protein dynamics and allostery using multi-protein atomic distance constraints</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1016/j.str.2017.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.str.2017.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28190781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=546-558&author=J.+G.+Greenerauthor=I.+Filippisauthor=M.+J.+E.+Sternberg&title=Predicting+protein+dynamics+and+allostery+using+multi-protein+atomic+distance+constraints&doi=10.1016%2Fj.str.2017.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Protein Dynamics and Allostery Using Multi-Protein Atomic Distance Constraints</span></div><div class="casAuthors">Greener, Joe G.; Filippis, Ioannis; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">546-558</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The related concepts of protein dynamics, conformational ensembles and allostery are often difficult to study with mol. dynamics (MD) due to the timescales involved.  We present ExProSE (Exploration of Protein Structural Ensembles), a distance geometry-based method that generates an ensemble of protein structures from two input structures.  ExProSE provides a unified framework for the exploration of protein structure and dynamics in a fast and accessible way.  Using a dataset of apo/holo pairs it is shown that existing coarse-grained methods often cannot span large conformational changes.  For T4-lysozyme, ExProSE is able to generate ensembles that are more native-like than tCONCOORD and NMSim, and comparable with targeted MD.  By adding addnl. constraints representing potential modulators, ExProSE can predict allosteric sites.  ExProSE ranks an allosteric pocket first or second for 27 out of 58 allosteric proteins, which is similar and complementary to existing methods.  The ExProSE source code is freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF34-q1BCDrVg90H21EOLACvtfcHk0lhcBUokzhMMgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D&md5=2d2ea675b3b29ff52f9915ddecabe032</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DFilippis%26aufirst%3DI.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%2BE.%26atitle%3DPredicting%2520protein%2520dynamics%2520and%2520allostery%2520using%2520multi-protein%2520atomic%2520distance%2520constraints%26jtitle%3DStructure%26date%3D2017%26volume%3D25%26spage%3D546%26epage%3D558%26doi%3D10.1016%2Fj.str.2017.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caliman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Mapping the allosteric sites of the A2A adenosine receptor</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1111/cbdd.13053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1111%2Fcbdd.13053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28639411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFynsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2018&pages=5-16&author=A.+D.+Calimanauthor=Y.+Miaoauthor=J.+A.+McCammon&title=Mapping+the+allosteric+sites+of+the+A2A+adenosine+receptor&doi=10.1111%2Fcbdd.13053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the allosteric sites of the A2A adenosine receptor</span></div><div class="casAuthors">Caliman, Alisha D.; Miao, Yinglong; McCammon, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-16</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The A2A adenosine receptor (A2AAR) is a G protein-coupled receptor that is pharmacol. targeted for the treatment of inflammation, sepsis, cancer, neurodegeneration, and Parkinson's disease.  Recently, we applied long-time-scale mol. dynamics simulations on 2 ligand-free receptor conformations, starting from the agonist-bound (PDB ID: 3QAK) and antagonist-bound (PDB ID: 3EML) x-ray structures.  This anal. revealed 4 distinct conformers of the A2AAR: active, intermediate 1, intermediate 2, and inactive conformers.  Here, we applied the fragment-based mapping algorithm (FTMap) on these receptor conformations to uncover 5 non-orthosteric sites on the A2AAR.  Two sites that were identified in the active conformation were located in the intracellular region of the transmembrane helixes (TM) 3/TM4 and the G protein-binding site in the intracellular region between TM2/TM3/TM6/TM7.  Three sites are identified in the intermediate 1 and intermediate 2 conformations, annexing a site in the lipid interface of TM5/TM6.  Five sites were identified in the inactive conformation, comprising a site in the intracellular region of TM1/TM7 and in the extracellular region of TM3/TM4 of the A2AAR.  We postulate that these sites on the A2AAR be screened for allosteric modulators for the treatment of inflammatory and neurol. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkjjgfTQct9bVg90H21EOLACvtfcHk0lhcBUokzhMMgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFynsbnF&md5=0a7d738c2f9906177b0dc34ac55a9d98</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.13053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.13053%26sid%3Dliteratum%253Aachs%26aulast%3DCaliman%26aufirst%3DA.%2BD.%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DMapping%2520the%2520allosteric%2520sites%2520of%2520the%2520A2A%2520adenosine%2520receptor%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2018%26volume%3D91%26spage%3D5%26epage%3D16%26doi%3D10.1111%2Fcbdd.13053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Lessons from hot spot analysis for fragment-based drug discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.tips.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26538314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=724-736&author=D.+R.+Hallauthor=D.+Kozakovauthor=A.+Whittyauthor=S.+Vajda&title=Lessons+from+hot+spot+analysis+for+fragment-based+drug+discovery&doi=10.1016%2Fj.tips.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from hot spot analysis for fragment-based drug discovery</span></div><div class="casAuthors">Hall, David R.; Kozakov, Dima; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">724-736</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Anal. of binding energy hot spots at protein surfaces can provide crucial insights into the prospects for successful application of fragment-based drug discovery (FBDD), and whether a fragment hit can be advanced into a high-affinity, drug-like ligand.  The key factor is the strength of the top ranking hot spot, and how well a given fragment complements it.  We show that published data are sufficient to provide a sophisticated and quant. understanding of how hot spots derive from a protein 3D structure, and how their strength, no., and spatial arrangement govern the potential for a surface site to bind to fragment-sized and larger ligands.  This improved understanding provides important guidance for the effective application of FBDD in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQmPulmhPx7Vg90H21EOLACvtfcHk0lhcBUokzhMMgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF&md5=3e682f28482d162590826f8f2e321b96</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLessons%2520from%2520hot%2520spot%2520analysis%2520for%2520fragment-based%2520drug%2520discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D724%26epage%3D736%26doi%3D10.1016%2Fj.tips.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rettenmaier, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">A small-molecule mimic of a peptide docking motif inhibits the protein kinase PDK1</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">18590</span>– <span class="NLM_lpage">18595</span>, <span class="refDoi"> DOI: 10.1073/pnas.1415365112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.1415365112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25518860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18590-18595&author=T.+J.+Rettenmaierauthor=J.+D.+Sadowskyauthor=N.+D.+Thomsenauthor=S.+C.+Chenauthor=A.+K.+Doakauthor=M.+R.+Arkinauthor=J.+A.+Wells&title=A+small-molecule+mimic+of+a+peptide+docking+motif+inhibits+the+protein+kinase+PDK1&doi=10.1073%2Fpnas.1415365112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule mimic of a peptide docking motif inhibits the protein kinase PDK1</span></div><div class="casAuthors">Rettenmaier, T. Justin; Sadowsky, Jack D.; Thomsen, Nathan D.; Chen, Steven C.; Doak, Allison K.; Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">18590-18595</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">There is great interest in developing selective protein kinase inhibitors by targeting allosteric sites, but these sites often involve protein-protein or protein-peptide interfaces that are very challenging to target with small mols.  Here we present a systematic approach to targeting a functionally conserved allosteric site on the protein kinase PDK1 called the PDK1-interacting fragment (PIF)tide-binding site, or PIF pocket.  More than two dozen prosurvival and progrowth kinases dock a conserved peptide tail into this binding site, which recruits them to PDK1 to become activated.  Using a site-directed chem. screen, we identified and chem. optimized ligand-efficient, selective, and cell-penetrant small mols. (mol. wt. ∼380 Da) that compete with the peptide docking motif for binding to PDK1.  We solved the first high-resoln. structure of a peptide docking motif (PIFtide) bound to PDK1 and mapped binding energy hot spots using mutational anal.  We then solved structures of PDK1 bound to the allosteric small mols., which revealed a binding mode that remarkably mimics three of five hot-spot residues in PIFtide.  These allosteric small mols. are substrate-selective PDK1 inhibitors when used as single agents, but when combined with an ATP-competitive inhibitor, they completely suppress the activation of the downstream kinases.  This work provides a promising new scaffold for the development of high-affinity PIF pocket ligands, which may be used to enhance the anticancer activity of existing PDK1 inhibitors.  Moreover, our results provide further impetus for exploring the helix αC patches of other protein kinases as potential therapeutic targets even though they involve protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeMoENY8xXbVg90H21EOLACvtfcHk0lhcBUokzhMMgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ&md5=4a1d88395865677d5a9c8d3dee58eca2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1415365112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1415365112%26sid%3Dliteratum%253Aachs%26aulast%3DRettenmaier%26aufirst%3DT.%2BJ.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DThomsen%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520small-molecule%2520mimic%2520of%2520a%2520peptide%2520docking%2520motif%2520inhibits%2520the%2520protein%2520kinase%2520PDK1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D18590%26epage%3D18595%26doi%3D10.1073%2Fpnas.1415365112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Exploration of type II binding mode: A privileged approach for kinase inhibitor focused drug discovery?</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+A+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lhZcn-XbZ0kCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520A%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The design, synthesis, and evaluation of 8 hybrid DFG-out allosteric kinase inhibitors: A structural analysis of the binding interactions of Gleevec, Nexavar, and BIRB-796</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5748</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bmc.2010.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20621496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlGhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=5738-5748&author=J.+Dietrichauthor=C.+Hulmeauthor=L.+H.+Hurley&title=The+design%2C+synthesis%2C+and+evaluation+of+8+hybrid+DFG-out+allosteric+kinase+inhibitors%3A+A+structural+analysis+of+the+binding+interactions+of+Gleevec%2C+Nexavar%2C+and+BIRB-796&doi=10.1016%2Fj.bmc.2010.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The design, synthesis, and evaluation of 8 hybrid DFG-out allosteric kinase inhibitors: A structural analysis of the binding interactions of Gleevec, Nexavar, and BIRB-796</span></div><div class="casAuthors">Dietrich, Justin; Hulme, Christopher; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5738-5748</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The majority of kinase inhibitors developed to date are competitive inhibitors that target the ATP binding site; however, recent crystal structures of Gleevec (imatinib mesylate, STI571, PDB: 1IEP), Nexavar (Sorafenib tosylate, BAY 43-9006, PDB: 1UWJ), and BIRB-796 (PDB: 1KV2) have revealed a secondary binding site adjacent to the ATP binding site known as the DFG-out allosteric binding site.  The recent successes of Gleevec and Nexavar for the treatment of chronic myeloid leukemia and renal cell carcinoma has generated great interest in the development of other kinase inhibitors that target this secondary binding site.  Here, we present a structural comparison of the important and similar interactions necessary for Gleevec, Nexavar, and BIRB-796 to bind to their resp. DFG-out allosteric binding pockets and the selectivity of each with respect to c-Abl, B-Raf, and p38α.  A structural anal. of their selectivity profiles has been generated from the synthesis and evaluation of 8 addnl. DFG-out allosteric inhibitors that were developed directly from fragments of these successful scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuTXCEZCwd0bVg90H21EOLACvtfcHk0lhZcn-XbZ0kCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlGhu7s%253D&md5=430e5d3c95213441b17030a931f6e2ad</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DDietrich%26aufirst%3DJ.%26aulast%3DHulme%26aufirst%3DC.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520design%252C%2520synthesis%252C%2520and%2520evaluation%2520of%25208%2520hybrid%2520DFG-out%2520allosteric%2520kinase%2520inhibitors%253A%2520A%2520structural%2520analysis%2520of%2520the%2520binding%2520interactions%2520of%2520Gleevec%252C%2520Nexavar%252C%2520and%2520BIRB-796%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D5738%26epage%3D5748%26doi%3D10.1016%2Fj.bmc.2010.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aay, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blazey, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowles, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussenius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Defina, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubenko, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engst, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koltun, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lougheed, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manalo, J.-C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuss, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peto, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S.</span></span> <span> </span><span class="NLM_article-title">Novel carboxamide-based allosteric MEK inhibitors: Discovery and optimization efforts toward XL518 (GDC-0973)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">416</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1021/ml300049d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300049d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1eit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=416-421&author=K.+D.+Riceauthor=N.+Aayauthor=N.+K.+Anandauthor=C.+M.+Blazeyauthor=O.+J.+Bowlesauthor=J.+Busseniusauthor=S.+Costanzoauthor=J.+K.+Curtisauthor=S.+C.+Definaauthor=L.+Dubenkoauthor=S.+Engstauthor=A.+A.+Joshiauthor=A.+R.+Kennedyauthor=A.+I.+Kimauthor=E.+S.+Koltunauthor=J.+C.+Lougheedauthor=J.-C.+L.+Manaloauthor=J.-F.+Martiniauthor=J.+M.+Nussauthor=C.+J.+Petoauthor=T.+H.+Tsangauthor=P.+Yuauthor=S.+Johnston&title=Novel+carboxamide-based+allosteric+MEK+inhibitors%3A+Discovery+and+optimization+efforts+toward+XL518+%28GDC-0973%29&doi=10.1021%2Fml300049d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Carboxamide-Based Allosteric MEK Inhibitors: Discovery and Optimization Efforts toward XL518 (GDC-0973)</span></div><div class="casAuthors">Rice, Kenneth D.; Aay, Naing; Anand, Neel K.; Blazey, Charles M.; Bowles, Owen J.; Bussenius, Joerg; Costanzo, Simona; Curtis, Jeffry K.; Defina, Steven C.; Dubenko, Larisa; Engst, Stefan; Joshi, Anagha A.; Kennedy, Abigail R.; Kim, Angie I.; Koltun, Elena S.; Lougheed, Julie C.; Manalo, Jean-Claire L.; Martini, Jean-Francois; Nuss, John M.; Peto, Csaba J.; Tsang, Tsze H.; Yu, Peiwen; Johnston, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">416-421</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ERK/MAP kinase cascade is a key mechanism subject to dysregulation in cancer and is constitutively activated or highly upregulated in many tumor types.  Mutations assocd. with upstream pathway components RAS and Raf occur frequently and contribute to the oncogenic phenotype through activation of MEK and then ERK.  Inhibitors of MEK have been shown to effectively block upregulated ERK/MAPK signaling in a range of cancer cell lines and have further demonstrated early evidence of efficacy in the clinic for the treatment of cancer.  Guided by structural insight, a strategy aimed at the identification of an optimal diphenylamine-based MEK inhibitor with an improved metab. and safety profile vs. PD-0325901 led to the discovery of development candidate 1-({3,4-difluoro-2-[(2-fluoro-4-iodophenyl)amino]phenyl}carbonyl)-3-[(2S)-piperidin-2-yl]azetidin-3-ol (XL518, GDC-0973) (I).  I exhibits robust in vitro and in vivo potency and efficacy in preclin. models with sustained duration of action and is currently in early stage clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOx-Upr00b8LVg90H21EOLACvtfcHk0lhV1isbn4flnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1eit7k%253D&md5=04aa82afd24efe83a3b888140781bfc6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fml300049d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300049d%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DK.%2BD.%26aulast%3DAay%26aufirst%3DN.%26aulast%3DAnand%26aufirst%3DN.%2BK.%26aulast%3DBlazey%26aufirst%3DC.%2BM.%26aulast%3DBowles%26aufirst%3DO.%2BJ.%26aulast%3DBussenius%26aufirst%3DJ.%26aulast%3DCostanzo%26aufirst%3DS.%26aulast%3DCurtis%26aufirst%3DJ.%2BK.%26aulast%3DDefina%26aufirst%3DS.%2BC.%26aulast%3DDubenko%26aufirst%3DL.%26aulast%3DEngst%26aufirst%3DS.%26aulast%3DJoshi%26aufirst%3DA.%2BA.%26aulast%3DKennedy%26aufirst%3DA.%2BR.%26aulast%3DKim%26aufirst%3DA.%2BI.%26aulast%3DKoltun%26aufirst%3DE.%2BS.%26aulast%3DLougheed%26aufirst%3DJ.%2BC.%26aulast%3DManalo%26aufirst%3DJ.-C.%2BL.%26aulast%3DMartini%26aufirst%3DJ.-F.%26aulast%3DNuss%26aufirst%3DJ.%2BM.%26aulast%3DPeto%26aufirst%3DC.%2BJ.%26aulast%3DTsang%26aufirst%3DT.%2BH.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DJohnston%26aufirst%3DS.%26atitle%3DNovel%2520carboxamide-based%2520allosteric%2520MEK%2520inhibitors%253A%2520Discovery%2520and%2520optimization%2520efforts%2520toward%2520XL518%2520%2528GDC-0973%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D416%26epage%3D421%26doi%3D10.1021%2Fml300049d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juarez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palakurthi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelais, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michellys, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epple, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsilje, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">Overcoming EGFR(T790M) and EGFR(c797s) resistance with mutant-selective allosteric inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature17960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature17960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27251290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=129-132&author=Y.+Jiaauthor=C.-H.+Yunauthor=E.+Parkauthor=D.+Ercanauthor=M.+Manuiaauthor=J.+Juarezauthor=C.+Xuauthor=K.+Rheeauthor=T.+Chenauthor=H.+Zhangauthor=S.+Palakurthiauthor=J.+Jangauthor=G.+Lelaisauthor=M.+DiDonatoauthor=B.+Bursulayaauthor=P.-Y.+Michellysauthor=R.+Eppleauthor=T.+H.+Marsiljeauthor=M.+McNeillauthor=W.+Luauthor=J.+Harrisauthor=S.+Benderauthor=K.-K.+Wongauthor=P.+A.+J%C3%A4nneauthor=M.+J.+Eck&title=Overcoming+EGFR%28T790M%29+and+EGFR%28c797s%29+resistance+with+mutant-selective+allosteric+inhibitors&doi=10.1038%2Fnature17960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span></div><div class="casAuthors">Jia, Yong; Yun, Cai-Hong; Park, Eunyoung; Ercan, Dalia; Manuia, Mari; Juarez, Jose; Xu, Chunxiao; Rhee, Kevin; Chen, Ting; Zhang, Haikuo; Palakurthi, Sangeetha; Jang, Jaebong; Lelais, Gerald; DiDonato, Michael; Bursulaya, Badry; Michellys, Pierre-Yves; Epple, Robert; Marsilje, Thomas H.; McNeill, Matthew; Lu, Wenshuo; Harris, Jennifer; Bender, Steven; Wong, Kwok-Kin; Janne, Pasi A.; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7605</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR)-directed tyrosine kinase inhibitors (TKIs) gefitinib, erlotinib and afatinib are approved treatments for non-small cell lung cancers harbouring activating mutations in the EGFR kinase, but resistance arises rapidly, most frequently owing to the secondary T790M mutation within the ATP site of the receptor.  Recently developed mutant-selective irreversible inhibitors are highly active against the T790M mutant, but their efficacy can be compromised by acquired mutation of C797, the cysteine residue with which they form a key covalent bond.  All current EGFR TKIs target the ATP-site of the kinase, highlighting the need for therapeutic agents with alternative mechanisms of action.  Here we describe the rational discovery of EAI045, an allosteric inhibitor that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.  The crystal structure shows that the compd. binds an allosteric site created by the displacement of the regulatory C-helix in an inactive conformation of the kinase.  The compd. inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochem. assays.  However, as a single agent it is not effective in blocking EGFR-driven proliferation in cells owing to differential potency on the two subunits of the dimeric receptor, which interact in an asym. manner in the active state.  We observe marked synergy of EAI045 with cetuximab, an antibody therapeutic that blocks EGFR dimerization, rendering the kinase uniformly susceptible to the allosteric agent.  EAI045 in combination with cetuximab is effective in mouse models of lung cancer driven by EGFR(L858R/T790M) and by EGFR(L858R/T790M/C797S), a mutant that is resistant to all currently available EGFR TKIs.  More generally, our findings illustrate the utility of purposefully targeting allosteric sites to obtain mutant-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQkr363w1dMLVg90H21EOLACvtfcHk0lhV1isbn4flnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D&md5=f4c2d2362eb900c64bad8550b9da0872</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature17960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17960%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%26aulast%3DYun%26aufirst%3DC.-H.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DManuia%26aufirst%3DM.%26aulast%3DJuarez%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DRhee%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DPalakurthi%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DLelais%26aufirst%3DG.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMichellys%26aufirst%3DP.-Y.%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DMarsilje%26aufirst%3DT.%2BH.%26aulast%3DMcNeill%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DOvercoming%2520EGFR%2528T790M%2529%2520and%2520EGFR%2528c797s%2529%2520resistance%2520with%2520mutant-selective%2520allosteric%2520inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D129%26epage%3D132%26doi%3D10.1038%2Fnature17960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hindie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garcia, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idrissova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeuzem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, T. J.
D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzari, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span> <span> </span><span class="NLM_article-title">Structure and allosteric effects of low-molecular-weight activators on the protein kinase PDK1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">758</span>– <span class="NLM_lpage">764</span>, <span class="refDoi"> DOI: 10.1038/nchembio.208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnchembio.208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=19718043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=758-764&author=V.+Hindieauthor=A.+Strobaauthor=H.+Zhangauthor=L.+A.+Lopez-Garciaauthor=L.+Idrissovaauthor=S.+Zeuzemauthor=D.+Hirschbergauthor=F.+Schaefferauthor=T.+J.%0AD.+J%C3%B8rgensenauthor=M.+Engelauthor=P.+M.+Alzariauthor=R.+M.+Biondi&title=Structure+and+allosteric+effects+of+low-molecular-weight+activators+on+the+protein+kinase+PDK1&doi=10.1038%2Fnchembio.208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and allosteric effects of low-molecular-weight activators on the protein kinase PDK1</span></div><div class="casAuthors">Hindie, Valerie; Stroba, Adriana; Zhang, Hua; Lopez-Garcia, Laura A.; Idrissova, Leila; Zeuzem, Stefan; Hirschberg, Daniel; Schaeffer, Francis; Jorgensen, Thomas J. D.; Engel, Matthias; Alzari, Pedro M.; Biondi, Ricardo M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">758-764</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein phosphorylation transduces a large set of intracellular signals.  One mechanism by which phosphorylation mediates signal transduction is by prompting conformational changes in the target protein or interacting proteins.  Previous work described an allosteric site mediating phosphorylation-dependent activation of AGC kinases.  AGC kinase PDK1 is activated by the docking of a phosphorylated motif from substrates.  Here, the authors present the crystallog. of PDK1 bound to a rationally developed low-mol.-wt. activator and describe the conformational changes induced by small compds. in the crystal and in soln. using a fluorescence-based assay and deuterium exchange expts.  The results indicate that the binding of the compd. produces local changes at the target site, the PIF-binding pocket, and also allosteric changes at the ATP-binding site and the activation loop.  Altogether, the authors present mol. details of the allosteric changes induced by small compds. that trigger the activation of PDK1 through mimicry of phosphorylation-dependent conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBilNGbn-krbVg90H21EOLACvtfcHk0lhV1isbn4flnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F&md5=c9afd03c1d2c954fbcf04709570fdb76</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.208%26sid%3Dliteratum%253Aachs%26aulast%3DHindie%26aufirst%3DV.%26aulast%3DStroba%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLopez-Garcia%26aufirst%3DL.%2BA.%26aulast%3DIdrissova%26aufirst%3DL.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DHirschberg%26aufirst%3DD.%26aulast%3DSchaeffer%26aufirst%3DF.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DT.%2BJ.%2BD.%26aulast%3DEngel%26aufirst%3DM.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26atitle%3DStructure%2520and%2520allosteric%2520effects%2520of%2520low-molecular-weight%2520activators%2520on%2520the%2520protein%2520kinase%2520PDK1%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D758%26epage%3D764%26doi%3D10.1038%2Fnchembio.208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bobkova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blume-Jensen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrup, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunapuli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariv, I.</span></span> <span> </span><span class="NLM_article-title">Discovery of PDK1 kinase inhibitors with a novel mechanism of action by ultrahigh throughput screening</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">18838</span>– <span class="NLM_lpage">18846</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.089946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1074%2Fjbc.m109.089946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20385558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvFaqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=18838-18846&author=E.+V.+Bobkovaauthor=M.+J.+Weberauthor=Z.+Xuauthor=Y.-L.+Zhangauthor=J.+Jungauthor=P.+Blume-Jensenauthor=A.+Northrupauthor=P.+Kunapuliauthor=J.+N.+Andersenauthor=I.+Kariv&title=Discovery+of+PDK1+kinase+inhibitors+with+a+novel+mechanism+of+action+by+ultrahigh+throughput+screening&doi=10.1074%2Fjbc.m109.089946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of PDK1 Kinase Inhibitors with a Novel Mechanism of Action by Ultrahigh Throughput Screening</span></div><div class="casAuthors">Bobkova, Ekaterina V.; Weber, Michael J.; Xu, Zangwei; Zhang, Yan-Ling; Jung, Joon; Blume-Jensen, Peter; Northrup, Alan; Kunapuli, Priya; Andersen, Jannik N.; Kariv, Ilona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">18838-18846</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The phosphoinositide 3-kinase/AKT signaling pathway plays a key role in cancer cell growth, survival, and angiogenesis.  Phosphoinositide-dependent protein kinase-1 (PDK1) acts at a focal point in this pathway immediately downstream of phosphoinositide 3-kinase and PTEN, where it phosphorylates numerous AGC kinases.  The PDK1 kinase domain has at least three ligand-binding sites: the ATP-binding pocket, the peptide substrate-binding site, and a groove in the N-terminal lobe that binds the C-terminal hydrophobic motif of its kinase substrates.  Based on the unique PDK1 substrate recognition system, ultrahigh throughput TR-FRET and Alphascreen screening assays were developed using a biotinylated version of the PDK1-tide substrate contg. the activation loop of AKT fused to a pseudo-activated hydrophobic motif peptide.  Using full-length PDK1, Km values were detd. as 5.6 μM for ATP and 40 nM for the fusion peptide, revealing 50-fold higher affinity compared with the classical AKT(Thr-308)-tide.  Kinetic and biophys. studies confirmed the PDK1 catalytic mechanism as a rapid equil. random bireactant reaction.  Following an ultrahigh throughput screen of a large library, 2,000 compds. were selected from the reconfirmed hits by computational anal. with a focus on novel scaffolds.  ATP-competitive hits were deconvoluted by dose-response studies at 1x and 10x Km concns. of ATP, and specificity of binding was assessed in thermal shift assay.  Inhibition studies using fusion PDK1-tide1 substrate vs. AKT(Thr-308)-tide and kinase selectivity profiling revealed a novel selective alkaloid scaffold that evidently binds to the PDK1-interacting fragment pocket.  Mol. modeling suggests a structural paradigm for the design of inhibitory vs. activating allosteric ligands of PDK1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtz2tgwFgbXrVg90H21EOLACvtfcHk0lhcEOb8rPagxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvFaqtrk%253D&md5=9fc38810a123771bd4bccade64a7a528</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m109.089946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m109.089946%26sid%3Dliteratum%253Aachs%26aulast%3DBobkova%26aufirst%3DE.%2BV.%26aulast%3DWeber%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DJung%26aufirst%3DJ.%26aulast%3DBlume-Jensen%26aufirst%3DP.%26aulast%3DNorthrup%26aufirst%3DA.%26aulast%3DKunapuli%26aufirst%3DP.%26aulast%3DAndersen%26aufirst%3DJ.%2BN.%26aulast%3DKariv%26aufirst%3DI.%26atitle%3DDiscovery%2520of%2520PDK1%2520kinase%2520inhibitors%2520with%2520a%2520novel%2520mechanism%2520of%2520action%2520by%2520ultrahigh%2520throughput%2520screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D18838%26epage%3D18846%26doi%3D10.1074%2Fjbc.m109.089946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheffzek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarkson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the autoinhibition of c-ABL tyrosine kinase</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(03)00194-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fs0092-8674%2803%2900194-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12654251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2003&pages=859-871&author=B.+Nagarauthor=O.+Hantschelauthor=M.+A.+Youngauthor=K.+Scheffzekauthor=D.+Veachauthor=W.+Bornmannauthor=B.+Clarksonauthor=G.+Superti-Furgaauthor=J.+Kuriyan&title=Structural+basis+for+the+autoinhibition+of+c-ABL+tyrosine+kinase&doi=10.1016%2Fs0092-8674%2803%2900194-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the autoinhibition of c-Abl tyrosine kinase</span></div><div class="casAuthors">Nagar, Bhushan; Hantschel, Oliver; Young, Matthew A.; Scheffzek, Klaus; Veach, Darren; Bornmann, William; Clarkson, Bayard; Superti-Furga, Giulio; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-871</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">C-Abl is normally regulated by an autoinhibitory mechanism, the disruption of which leads to chronic myelogenous leukemia.  The details of this mechanism have been elusive because c-Abl lacks a phosphotyrosine residue that triggers the assembly of the autoinhibited form of the closely related Src kinases by internally engaging the SH2 domain.  Crystal structures of c-Abl show that the N-terminal myristoyl modification of c-Abl 1b binds to the kinase domain and induces conformational changes that allow the SH2 and SH3 domains to dock onto it.  Autoinhibited c-Abl forms an assembly that is strikingly similar to that of inactive Src kinases but with specific differences that explain the differential ability of the drug STI-571/Gleevec/imatinib (STI-571) to inhibit the catalytic activity of Abl, but not that of c-Src.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaks_bu7ZlC7Vg90H21EOLACvtfcHk0lhcEOb8rPagxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D&md5=19245c92b1e36779e2024bc071aa723b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fs0092-8674%2803%2900194-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0092-8674%252803%252900194-6%26sid%3Dliteratum%253Aachs%26aulast%3DNagar%26aufirst%3DB.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DYoung%26aufirst%3DM.%2BA.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DVeach%26aufirst%3DD.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520the%2520autoinhibition%2520of%2520c-ABL%2520tyrosine%2520kinase%26jtitle%3DCell%26date%3D2003%26volume%3D112%26spage%3D859%26epage%3D871%26doi%3D10.1016%2Fs0092-8674%2803%2900194-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woessner, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W.</span></span> <span> </span><span class="NLM_article-title">Development of an effective therapy for chronic Myelogenous Leukemia</span>. <i>Cancer J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1097/ppo.0b013e318237e5b7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1097%2Fppo.0b013e318237e5b7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=22157291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XitVOjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=477-486&author=D.+W.+Woessnerauthor=C.+S.+Limauthor=M.+W.+Deininger&title=Development+of+an+effective+therapy+for+chronic+Myelogenous+Leukemia&doi=10.1097%2Fppo.0b013e318237e5b7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an effective therapy for chronic myelogenous leukemia</span></div><div class="casAuthors">Woessner, David W.; Lim, Carol S.; Deininger, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Journal (Hagerstown, MD, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">477-486</span>CODEN:
                <span class="NLM_cas:coden">CAJOCB</span>;
        ISSN:<span class="NLM_cas:issn">1528-9117</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  Targeted small-mol. drugs have revolutionized treatment of chronic myeloid leukemia (CML) during the last decade.  These agents interrupt a constitutively active BCR-ABL, the causative agent for CML, by interfering with adenosine 5' triphosphate-dependent ABL tyrosine kinase.  Although the efficacy of tyrosine kinase inhibitors (TKIs) has resulted in overall survival of greater than 90%, TKIs are not curative.  Moreover, no currently approved TKIs are effective against the T315I BCR-ABL variant.  However, a new generation of TKIs with activity against T315I is on the horizon.  We will highlight the clin. utility of historical CML therapeutics, those used today (first- and second-generation TKIs), and discuss treatment modalities that are under development.  Recent advances have illuminated the complexity of CML, esp. within the marrow microenvironment.  We contend that the key to curing CML will involve strategies beyond targeting BCR-ABL because primitive human CML stem cells are not dependent on BCR-ABL.  Ultimately, drug combinations or exploiting synthetic lethality may transform responses into definitive cures for CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPF8KXNet0xrVg90H21EOLACvtfcHk0li8qk6g5SMLUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitVOjtrc%253D&md5=81ca4b49e136803cf0731c9f1a4e95cc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1097%2Fppo.0b013e318237e5b7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252Fppo.0b013e318237e5b7%26sid%3Dliteratum%253Aachs%26aulast%3DWoessner%26aufirst%3DD.%2BW.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26atitle%3DDevelopment%2520of%2520an%2520effective%2520therapy%2520for%2520chronic%2520Myelogenous%2520Leukemia%26jtitle%3DCancer%2520J.%26date%3D2011%26volume%3D17%26spage%3D477%26epage%3D486%26doi%3D10.1097%2Fppo.0b013e318237e5b7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacob, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierks, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, G.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okram, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojciechowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendrich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strauss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajpai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting BCR-ABL by combining allosteric with ATP-binding-site inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1038/nature08675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature08675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20072125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=501-506&author=J.+Zhangauthor=F.+J.+Adri%C3%A1nauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+G.+Liauthor=R.+E.+Iacobauthor=T.+Simauthor=J.+Powersauthor=C.+Dierksauthor=F.+Sunauthor=G.-R.+Guoauthor=Q.+Dingauthor=B.+Okramauthor=Y.+Choiauthor=A.+Wojciechowskiauthor=X.+Dengauthor=G.+Liuauthor=G.+Fendrichauthor=A.+Straussauthor=N.+Vajpaiauthor=S.+Grzesiekauthor=T.+Tuntlandauthor=Y.+Liuauthor=B.+Bursulayaauthor=M.+Azamauthor=P.+W.+Manleyauthor=J.+R.+Engenauthor=G.+Q.+Daleyauthor=M.+Warmuthauthor=N.+S.+Gray&title=Targeting+BCR-ABL+by+combining+allosteric+with+ATP-binding-site+inhibitors&doi=10.1038%2Fnature08675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Bcr-Abl by combining allosteric with ATP-binding-site inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Adrian, Francisco J.; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Li, Allen G.; Iacob, Roxana E.; Sim, Taebo; Powers, John; Dierks, Christine; Sun, Fangxian; Guo, Gui-Rong; Ding, Qiang; Okram, Barun; Choi, Yongmun; Wojciechowski, Amy; Deng, Xianming; Liu, Guoxun; Fendrich, Gabriele; Strauss, Andre; Vajpai, Navratna; Grzesiek, Stephan; Tuntland, Tove; Liu, Yi; Bursulaya, Badry; Azam, Mohammad; Manley, Paul W.; Engen, John R.; Daley, George Q.; Warmuth, Markus; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7280</span>),
    <span class="NLM_cas:pages">501-506</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In an effort to find new pharmacol. modalities to overcome resistance to ATP-binding-site inhibitors of Bcr-Abl, we recently reported the discovery of GNF-2, a selective allosteric Bcr-Abl inhibitor.  Here, using soln. NMR, X-ray crystallog., mutagenesis and hydrogen exchange mass spectrometry, we show that GNF-2 binds to the myristate-binding site of Abl, leading to changes in the structural dynamics of the ATP-binding site.  GNF-5, an analog of GNF-2 with improved pharmacokinetic properties, when used in combination with the ATP-competitive inhibitors imatinib or nilotinib, suppressed the emergence of resistance mutations in vitro, displayed additive inhibitory activity in biochem. and cellular assays against T315I mutant human Bcr-Abl and displayed in vivo efficacy against this recalcitrant mutant in a murine bone-marrow transplantation model.  These results show that therapeutically relevant inhibition of Bcr-Abl activity can be achieved with inhibitors that bind to the myristate-binding site and that combining allosteric and ATP-competitive inhibitors can overcome resistance to either agent alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUtkKFuTQx1LVg90H21EOLACvtfcHk0li8qk6g5SMLUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D&md5=46c122384fa004d27865c77f4a9165b7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature08675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08675%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%2BJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLi%26aufirst%3DA.%2BG.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DDierks%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DG.-R.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DOkram%26aufirst%3DB.%26aulast%3DChoi%26aufirst%3DY.%26aulast%3DWojciechowski%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFendrich%26aufirst%3DG.%26aulast%3DStrauss%26aufirst%3DA.%26aulast%3DVajpai%26aufirst%3DN.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DAzam%26aufirst%3DM.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DDaley%26aufirst%3DG.%2BQ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520BCR-ABL%2520by%2520combining%2520allosteric%2520with%2520ATP-binding-site%2520inhibitors%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D501%26epage%3D506%26doi%3D10.1038%2Fnature08675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuntini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furutani-Seiki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishima, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagby, S.</span></span> <span> </span><span class="NLM_article-title">Structural features and ligand binding properties of tandem WW domains from YAP and TAZ, nuclear effectors of the Hippo pathway</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">3300</span>– <span class="NLM_lpage">3309</span>, <span class="refDoi"> DOI: 10.1021/bi2001888</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2001888" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVCnt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=3300-3309&author=C.+Webbauthor=A.+Upadhyayauthor=F.+Giuntiniauthor=I.+Egglestonauthor=M.+Furutani-Seikiauthor=R.+Ishimaauthor=S.+Bagby&title=Structural+features+and+ligand+binding+properties+of+tandem+WW+domains+from+YAP+and+TAZ%2C+nuclear+effectors+of+the+Hippo+pathway&doi=10.1021%2Fbi2001888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Features and Ligand Binding Properties of Tandem WW Domains from YAP and TAZ, Nuclear Effectors of the Hippo Pathway</span></div><div class="casAuthors">Webb, Claire; Upadhyay, Abhishek; Giuntini, Francesca; Eggleston, Ian; Furutani-Seiki, Makoto; Ishima, Rieko; Bagby, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3300-3309</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The paralogous multifunctional adaptor proteins YAP and TAZ are the nuclear effectors of the Hippo pathway, a central mechanism of organ size control and stem cell self-renewal.  WW domains, mediators of protein-protein interactions, are essential for YAP and TAZ function, enabling interactions with PPxY motifs of numerous partner proteins.  YAP has single and double WW domain isoforms (YAP1 and YAP2) whereas only a single WW domain isoform of TAZ has been described to date.  Here we identify the first example of a double WW domain isoform of TAZ.  Using NMR, we have characterized conformational features and peptide binding of YAP and TAZ tandem WW domains (WW1-WW2).  The soln. structure of YAP WW2 confirms that it has a canonical three-stranded antiparallel β-sheet WW domain fold.  While chem. shift-based anal. indicates that the WW domains in the tandem WW pairs retain the characteristic WW domain fold, 15N relaxation data show that, within the resp. WW pairs, YAP WW1 and both WW1 and WW2 of TAZ undergo conformational exchange.  15N relaxation data also indicate that the linker between the WW domains is flexible in both YAP and TAZ.  Within both YAP and TAZ tandem WW pairs, WW1 and WW2 bind single PPxY-contg. peptide ligand concurrently and noncooperatively with sub-mM affinity.  YAP and TAZ WW1-WW2 bind a dual PPxY-contg. peptide with approx. 6-fold higher affinity.  Our results indicate that both WW domains in YAP and TAZ are functional and capable of enhanced affinity binding to multi-PPxY partner proteins such as LATS1, ErbB4, and AMOT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvTiUSYhJGc7Vg90H21EOLACvtfcHk0lgmPmD7ubiIPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVCnt7g%253D&md5=994139bb21116a57b7216a04d1b2485f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fbi2001888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2001888%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DA.%26aulast%3DGiuntini%26aufirst%3DF.%26aulast%3DEggleston%26aufirst%3DI.%26aulast%3DFurutani-Seiki%26aufirst%3DM.%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DBagby%26aufirst%3DS.%26atitle%3DStructural%2520features%2520and%2520ligand%2520binding%2520properties%2520of%2520tandem%2520WW%2520domains%2520from%2520YAP%2520and%2520TAZ%252C%2520nuclear%2520effectors%2520of%2520the%2520Hippo%2520pathway%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D3300%26epage%3D3309%26doi%3D10.1021%2Fbi2001888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velentza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sloan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mestan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibitors of BCR-ABL-dependent cell proliferation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">102</span>, <span class="refDoi"> DOI: 10.1038/nchembio760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnchembio760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=16415863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvF2rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=95-102&author=F.+J.+Adri%C3%A1nauthor=Q.+Dingauthor=T.+Simauthor=A.+Velentzaauthor=C.+Sloanauthor=Y.+Liuauthor=G.+Zhangauthor=W.+Hurauthor=S.+Dingauthor=P.+Manleyauthor=J.+Mestanauthor=D.+Fabbroauthor=N.+S.+Gray&title=Allosteric+inhibitors+of+BCR-ABL-dependent+cell+proliferation&doi=10.1038%2Fnchembio760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric inhibitors of Bcr-abl-dependent cell proliferation</span></div><div class="casAuthors">Adrian, Francisco J.; Ding, Qiang; Sim, Taebo; Velentza, Anastasia; Sloan, Christine; Liu, Yi; Zhang, Guobao; Hur, Wooyoung; Ding, Sheng; Manley, Paul; Mestan, Juergen; Fabbro, Doriano; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">95-102</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myelogenous leukemia (CML) is a myeloproliferative disorder characterized at the mol. level by the expression of Bcr-abl, a 210-kDa fusion protein with deregulated tyrosine kinase activity.  Encouraged by the clin. validation of Bcr-abl as the target for the treatment of CML by imatinib, we sought to identify pharmacol. agents that could target this kinase by a distinct mechanism.  We report the discovery of a new class of Bcr-abl inhibitors using an unbiased differential cytotoxicity screen of a combinatorial kinase-directed heterocycle library.  Compds. in this class (exemplified by GNF-2) show exclusive antiproliferative activity toward Bcr-abl-transformed cells, with potencies similar to imatinib, while showing no inhibition of the kinase activity of full-length or catalytic domain of c-abl.  We propose that this new class of compds. inhibits Bcr-abl kinase activity through an allosteric non-ATP competitive mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqjXG0lHqHurVg90H21EOLACvtfcHk0lgmPmD7ubiIPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvF2rsA%253D%253D&md5=c8617fecb6092d71ae163e7cda1e8f94</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnchembio760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio760%26sid%3Dliteratum%253Aachs%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%2BJ.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DVelentza%26aufirst%3DA.%26aulast%3DSloan%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DManley%26aufirst%3DP.%26aulast%3DMestan%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DAllosteric%2520inhibitors%2520of%2520BCR-ABL-dependent%2520cell%2520proliferation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D95%26epage%3D102%26doi%3D10.1038%2Fnchembio760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotesta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groell, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellé, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rummel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Asciminib (ABL001), an allosteric inhibitor of the tyrosine kinase activity of BCR-ABL1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8120</span>– <span class="NLM_lpage">8135</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01040</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01040" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8120-8135&author=J.+Schoepferauthor=W.+Jahnkeauthor=G.+Berelliniauthor=S.+Buonamiciauthor=S.+Cotestaauthor=S.+W.+Cowan-Jacobauthor=S.+Doddauthor=P.+Drueckesauthor=D.+Fabbroauthor=T.+Gabrielauthor=J.-M.+Groellauthor=R.+M.+Grotzfeldauthor=A.+Q.+Hassanauthor=C.+Henryauthor=V.+Iyerauthor=D.+Jonesauthor=F.+Lombardoauthor=A.+Looauthor=P.+W.+Manleyauthor=X.+Pell%C3%A9author=G.+Rummelauthor=B.+Salemauthor=M.+Warmuthauthor=A.+A.+Wylieauthor=T.+Zollerauthor=A.+L.+Marzinzikauthor=P.+Furet&title=Discovery+of+Asciminib+%28ABL001%29%2C+an+allosteric+inhibitor+of+the+tyrosine+kinase+activity+of+BCR-ABL1&doi=10.1021%2Facs.jmedchem.8b01040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1</span></div><div class="casAuthors">Schoepfer, Joseph; Jahnke, Wolfgang; Berellini, Giuliano; Buonamici, Silvia; Cotesta, Simona; Cowan-Jacob, Sandra W.; Dodd, Stephanie; Drueckes, Peter; Fabbro, Doriano; Gabriel, Tobias; Groell, Jean-Marc; Grotzfeld, Robert M.; Hassan, A. Quamrul; Henry, Chrystele; Iyer, Varsha; Jones, Darryl; Lombardo, Franco; Loo, Alice; Manley, Paul W.; Pelle, Xavier; Rummel, Gabriele; Salem, Bahaa; Warmuth, Markus; Wylie, Andrew A.; Zoller, Thomas; Marzinzik, Andreas L.; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8120-8135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of asciminib I, an allosteric inhibitor of the BCR-ABL1 kinase for use in treating chronic myelogenous leukemia (CML), is delineated.  I binds to the myristate pocket of BCR-ABL1 and maintains activity against cancer cells possessing ATP-site mutations in BCR-ABL1 that render them resistant to tyrosine kinase inhibitors; cancer cells resistant to myristate-site mutations are likely to be susceptible to ATP-competitive inhibitors so that combinations of I and tyrosine kinase inhibitors should suppress the emergence of resistance.  Fragment-based screening using NMR and X-ray yielded ligands for the myristate pocket, while an NMR-based conformational assay guided the transformation of these inactive ligands into ABL1 inhibitors.  Further structure-based optimization for potency, physicochem., pharmacokinetic, and drug-like properties, culminated in I which is currently undergoing clin. studies in CML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIQbEIXXgRgbVg90H21EOLACvtfcHk0lgmPmD7ubiIPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN&md5=023a566cdf4faba6dfc05501b8b6a6ed</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01040%26sid%3Dliteratum%253Aachs%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DCotesta%26aufirst%3DS.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGroell%26aufirst%3DJ.-M.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DHenry%26aufirst%3DC.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DPell%25C3%25A9%26aufirst%3DX.%26aulast%3DRummel%26aufirst%3DG.%26aulast%3DSalem%26aufirst%3DB.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DZoller%26aufirst%3DT.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Asciminib%2520%2528ABL001%2529%252C%2520an%2520allosteric%2520inhibitor%2520of%2520the%2520tyrosine%2520kinase%2520activity%2520of%2520BCR-ABL1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8120%26epage%3D8135%26doi%3D10.1021%2Facs.jmedchem.8b01040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzzelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanasse, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nature21702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature21702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28329763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=733-737&author=A.+A.+Wylieauthor=J.+Schoepferauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+Looauthor=P.+Furetauthor=A.+L.+Marzinzikauthor=X.+Pelleauthor=J.+Donovanauthor=W.+Zhuauthor=S.+Buonamiciauthor=A.+Q.+Hassanauthor=F.+Lombardoauthor=V.+Iyerauthor=M.+Palmerauthor=G.+Berelliniauthor=S.+Doddauthor=S.+Thohanauthor=H.+Bitterauthor=S.+Branfordauthor=D.+M.+Rossauthor=T.+P.+Hughesauthor=L.+Petruzzelliauthor=K.+G.+Vanasseauthor=M.+Warmuthauthor=F.+Hofmannauthor=N.+J.+Keenauthor=W.+R.+Sellers&title=The+allosteric+inhibitor+ABL001+enables+dual+targeting+of+BCR-ABL1&doi=10.1038%2Fnature21702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span></div><div class="casAuthors">Wylie, Andrew A.; Schoepfer, Joseph; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Loo, Alice; Furet, Pascal; Marzinzik, Andreas L.; Pelle, Xavier; Donovan, Jerry; Zhu, Wenjing; Buonamici, Silvia; Hassan, A. Quamrul; Lombardo, Franco; Iyer, Varsha; Palmer, Michael; Berellini, Giuliano; Dodd, Stephanie; Thohan, Sanjeev; Bitter, Hans; Branford, Susan; Ross, David M.; Hughes, Timothy P.; Petruzzelli, Lilli; Vanasse, K. Gary; Warmuth, Markus; Hofmann, Francesco; Keen, Nicholas J.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7647</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myeloid leukemia (CML) is driven by the activity of the BCR-ABL1 fusion oncoprotein.  ABL1 kinase inhibitors have improved the clin. outcomes for patients with CML, with over 80% of patients treated with imatinib surviving for more than 10 years.  Second-generation ABL1 kinase inhibitors induce more potent mol. responses in both previously untreated and imatinib-resistant patients with CML.  Studies in patients with chronic-phase CML have shown that around 50% of patients who achieve and maintain undetectable BCR-ABL1 transcript levels for at least 2 years remain disease-free after the withdrawal of treatment.  Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clin. development testing in patients with CML and Philadelphia chromosome-pos. (Ph+) acute lymphoblastic leukemia.  In contrast to catalytic-site ABL1 kinase inhibitors, ABL001 binds to the myristoyl pocket of ABL1 and induces the formation of an inactive kinase conformation.  ABL001 and second-generation catalytic inhibitors have similar cellular potencies but distinct patterns of resistance mutations, with genetic barcoding studies revealing pre-existing clonal populations with no shared resistance between ABL001 and the catalytic inhibitor nilotinib.  Consistent with this profile, acquired resistance was obsd. with single-agent therapy in mice; however, the combination of ABL001 and nilotinib led to complete disease control and eradicated CML xenograft tumors without recurrence after the cessation of treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreJ5YSS_wiS7Vg90H21EOLACvtfcHk0lhN38kBzQeQBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D&md5=20d00f222ab8d9387cb340a9531b8ba5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature21702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21702%26sid%3Dliteratum%253Aachs%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DPelle%26aufirst%3DX.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DBranford%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DT.%2BP.%26aulast%3DPetruzzelli%26aufirst%3DL.%26aulast%3DVanasse%26aufirst%3DK.%2BG.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DThe%2520allosteric%2520inhibitor%2520ABL001%2520enables%2520dual%2520targeting%2520of%2520BCR-ABL1%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnature21702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khateb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruimi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khamisie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najajreh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metodieva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajna, J.</span></span> <span> </span><span class="NLM_article-title">Overcoming BCR-ABL T315I mutation by combination of GNF-2 and ATP competitors in an ABL-independent mechanism</span>. <i>BMC Canc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">563</span>, <span class="refDoi"> DOI: 10.1186/1471-2407-12-563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1186%2F1471-2407-12-563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23186157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVKiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=563&author=M.+Khatebauthor=N.+Ruimiauthor=H.+Khamisieauthor=Y.+Najajrehauthor=A.+Mianauthor=A.+Metodievaauthor=M.+Ruthardtauthor=J.+Mahajna&title=Overcoming+BCR-ABL+T315I+mutation+by+combination+of+GNF-2+and+ATP+competitors+in+an+ABL-independent+mechanism&doi=10.1186%2F1471-2407-12-563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Bcr-Abl T315I mutation by combination of GNF-2 and ATP competitors in an Abl-independent mechanism</span></div><div class="casAuthors">Khateb, Mamduh; Ruimi, Nili; Khamisie, Hazem; Najajreh, Yousef; Mian, Afsar; Metodieva, Anna; Ruthardt, Martin; Mahajna, Jamal</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">563</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Philadelphia pos. leukemias are characterized by the presence of Bcr-Abl fusion protein which exhibits an abnormal kinase activity.  Selective Abl kinase inhibitors have been successfully established for the treatment of Ph(+) leukemias.  Despite high rates of clin. response, Ph(+) patients can develop resistance against these kinase inhibitors mainly due to point mutations within the Abl protein.  Of special interest is the 'gatekeeper' T315I mutation, which confers complete resistance to Abl kinase inhibitors.  Recently, GNF-2, Abl allosteric kinase inhibitor, was demonstrated to possess cellular activity against Bcr-Abl transformed cells.  Similarly to Abl kinase inhibitors (AKIs), GNF-2 failed to inhibit activity of mutated Bcr-Abl carrying the T315I mutation.  Methods: Ba/F3 cells harboring native or T315I mutated Bcr-Abl constructs were treated with GNF-2 and AKIs.  We monitored the effect of GNF-2 with AKIs on the proliferation and clonigenicity of the different Ba/F3 cells.  In addn., we monitored the auto-phosphorylation activity of Bcr-Abl and JAK2 in cells treated with GNF-2 and AKIs.  Results: In this study, we report a cooperation between AKIs and GNF-2 in inhibiting proliferation and clonigenicity of Ba/F3 cells carrying T315I mutated Bcr-Abl.  Interestingly, cooperation was most evident between Dasatinib and GNF-2.  Furthermore, we showed that GNF-2 was moderately active in inhibiting the activity of JAK2 kinase, and presence of AKIs augmented GNF-2 activity.  Conclusions: Our data illustrated the ability of allosteric inhibitors such as GNF-2 to cooperate with AKIs to overcome T315I mutation by Bcr-Abl-independent mechanisms, providing a possibility of enhancing AKIs efficacy and overcoming resistance in Ph+ leukemia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVaaSokbndBLVg90H21EOLACvtfcHk0lhN38kBzQeQBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVKiu70%253D&md5=0f1c71dda807ee8d7c539012f5c00f87</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-12-563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-12-563%26sid%3Dliteratum%253Aachs%26aulast%3DKhateb%26aufirst%3DM.%26aulast%3DRuimi%26aufirst%3DN.%26aulast%3DKhamisie%26aufirst%3DH.%26aulast%3DNajajreh%26aufirst%3DY.%26aulast%3DMian%26aufirst%3DA.%26aulast%3DMetodieva%26aufirst%3DA.%26aulast%3DRuthardt%26aufirst%3DM.%26aulast%3DMahajna%26aufirst%3DJ.%26atitle%3DOvercoming%2520BCR-ABL%2520T315I%2520mutation%2520by%2520combination%2520of%2520GNF-2%2520and%2520ATP%2520competitors%2520in%2520an%2520ABL-independent%2520mechanism%26jtitle%3DBMC%2520Canc.%26date%3D2012%26volume%3D12%26spage%3D563%26doi%3D10.1186%2F1471-2407-12-563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith, E. J.</span></span> <span> </span><span class="NLM_article-title">Unique map kinase binding sites</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1784</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2007.09.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bbapap.2007.09.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=18068683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlCjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1784&publication_year=2008&pages=48-55&author=R.+Akellaauthor=T.+M.+Moonauthor=E.+J.+Goldsmith&title=Unique+map+kinase+binding+sites&doi=10.1016%2Fj.bbapap.2007.09.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Unique MAP Kinase binding sites</span></div><div class="casAuthors">Akella, Radha; Moon, Thomas M.; Goldsmith, Elizabeth J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1784</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-55</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Map kinases are drug targets for autoimmune disease, cancer, and apoptosis-related diseases.  Drug discovery efforts have developed MAP kinase inhibitors directed toward the ATP binding site and neighboring "DFG-out" site, both of which are targets for inhibitors of other protein kinases.  On the other hand, MAP kinases have unique substrate and small mol. binding sites that could serve as inhibition sites.  The substrate and processing enzyme D-motif binding site is present in all MAP kinases, and has many features of a good small mol. binding site.  Further, the MAP kinase p38α has a binding site near its C-terminus discovered in crystallog. studies.  Finally, the MAP kinases ERK2 and p38α have a second substrate binding site, the FXFP binding site that is exposed in active ERK2 and the D-motif peptide induced conformation of MAP kinases.  Crystallog. evidence of these latter two binding sites is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo1xtxFBIntbVg90H21EOLACvtfcHk0lhN38kBzQeQBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlCjsw%253D%253D&md5=729bec92a896aec49b59182677563e62</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2007.09.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2007.09.016%26sid%3Dliteratum%253Aachs%26aulast%3DAkella%26aufirst%3DR.%26aulast%3DMoon%26aufirst%3DT.%2BM.%26aulast%3DGoldsmith%26aufirst%3DE.%2BJ.%26atitle%3DUnique%2520map%2520kinase%2520binding%2520sites%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2008%26volume%3D1784%26spage%3D48%26epage%3D55%26doi%3D10.1016%2Fj.bbapap.2007.09.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapadat, R.</span></span> <span> </span><span class="NLM_article-title">Mitogen-activated protein kinase pathways mediated by ERK, JNK, AND P38 protein kinases</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1911</span>– <span class="NLM_lpage">1912</span>, <span class="refDoi"> DOI: 10.1126/science.1072682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1126%2Fscience.1072682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12471242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1911-1912&author=G.+L.+Johnsonauthor=R.+Lapadat&title=Mitogen-activated+protein+kinase+pathways+mediated+by+ERK%2C+JNK%2C+AND+P38+protein+kinases&doi=10.1126%2Fscience.1072682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Mitogen-activated protein kinase pathways mediated by ERK, JNK, and p38 protein kinases</span></div><div class="casAuthors">Johnson, Gary L.; Lapadat, Razvan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1911-1912</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Multicellular organisms have 3 well-characterized subfamilies of MAP kinases (MAPKs) that control a vast array of physiol. processes.  These enzymes are regulated by a characteristic phospho-relay system in which a series of 3 protein kinases phosphorylate and activate one another.  The extracellular signal-regulated kinases (ERK1/2) function in the control of cell division, and inhibitors of these enzymes are being explored as anticancer agents.  The c-Jun N-terminal kinases (JNKs) are crit. regulators of transcription, and JNK inhibitors may be effective in control of rheumatoid arthritis.  The p38 MAPKs are activated by inflammatory cytokines and environmental stresses and may contribute to diseases like asthma and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmioiOeBgw-bVg90H21EOLACvtfcHk0likyv4Nf5dElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisbc%253D&md5=6788f1e6b9be24756ccec658f6c1c6b6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1126%2Fscience.1072682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1072682%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DG.%2BL.%26aulast%3DLapadat%26aufirst%3DR.%26atitle%3DMitogen-activated%2520protein%2520kinase%2520pathways%2520mediated%2520by%2520ERK%252C%2520JNK%252C%2520AND%2520P38%2520protein%2520kinases%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1911%26epage%3D1912%26doi%3D10.1126%2Fscience.1072682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heo, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, B.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyun, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-H.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the selective inhibition of JNK1 by the scaffolding protein JIP1 and SP600125</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2185</span>– <span class="NLM_lpage">2195</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fsj.emboj.7600212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=15141161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksV2hsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2185-2195&author=Y.-S.+Heoauthor=S.-K.+Kimauthor=C.+I.+Seoauthor=Y.+K.+Kimauthor=B.-J.+Sungauthor=H.+S.+Leeauthor=J.+I.+Leeauthor=S.-Y.+Parkauthor=J.+H.+Kimauthor=K.+Y.+Hwangauthor=Y.-L.+Hyunauthor=Y.+H.+Jeonauthor=S.+Roauthor=J.+M.+Choauthor=T.+G.+Leeauthor=C.-H.+Yang&title=Structural+basis+for+the+selective+inhibition+of+JNK1+by+the+scaffolding+protein+JIP1+and+SP600125&doi=10.1038%2Fsj.emboj.7600212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the selective inhibition of JNK1 by the scaffolding protein JIP1 and SP600125</span></div><div class="casAuthors">Heo, Yong-Seok; Kim, Su-Kyoung; Seo, Chang Il; Kim, Young Kwan; Sung, Byung-Je; Lee, Hye Shin; Lee, Jae Il; Park, Sam-Yong; Kim, Jin Hwan; Hwang, Kwang Yeon; Hyun, Young-Lan; Jeon, Young Ho; Ro, Seonggu; Cho, Joong Myung; Lee, Tae Gyu; Yang, Chul-Hak</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2185-2195</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The c-jun N-terminal kinase (JNK) signaling pathway is regulated by JNK-interacting protein-1 (JIP1), which is a scaffolding protein assembling the components of the JNK cascade.  Overexpression of JIP1 deactivates the JNK pathway selectively by cytoplasmic retention of JNK and thereby inhibits gene expression mediated by JNK, which occurs in the nucleus.  Here, we report the crystal structure of human JNK1 complexed with pepJIP1, the peptide fragment of JIP1, revealing its selectivity for JNK1 over other MAPKs and the allosteric inhibition mechanism.  The van der Waals contacts by the three residues (Pro157, Leu160, and Leu162) of pepJIP1 and the hydrogen bonding between Glu329 of JNK1 and Arg156 of pepJIP1 are crit. for the selective binding.  Binding of the peptide also induces a hinge motion between the N- and C-terminal domains of JNK1 and distorts the ATP-binding cleft, reducing the affinity of the kinase for ATP.  In addn., we also detd. the ternary complex structure of pepJIP1-bound JNK1 complexed with SP600125, an ATP-competitive inhibitor of JNK, providing the basis for the JNK specificity of the compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJlRXjZD8QM7Vg90H21EOLACvtfcHk0likyv4Nf5dElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksV2hsbw%253D&md5=6353695c402b21d1d0b5089107e03ee0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600212%26sid%3Dliteratum%253Aachs%26aulast%3DHeo%26aufirst%3DY.-S.%26aulast%3DKim%26aufirst%3DS.-K.%26aulast%3DSeo%26aufirst%3DC.%2BI.%26aulast%3DKim%26aufirst%3DY.%2BK.%26aulast%3DSung%26aufirst%3DB.-J.%26aulast%3DLee%26aufirst%3DH.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BI.%26aulast%3DPark%26aufirst%3DS.-Y.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DHwang%26aufirst%3DK.%2BY.%26aulast%3DHyun%26aufirst%3DY.-L.%26aulast%3DJeon%26aufirst%3DY.%2BH.%26aulast%3DRo%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DT.%2BG.%26aulast%3DYang%26aufirst%3DC.-H.%26atitle%3DStructural%2520basis%2520for%2520the%2520selective%2520inhibition%2520of%2520JNK1%2520by%2520the%2520scaffolding%2520protein%2520JIP1%2520and%2520SP600125%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2185%26epage%3D2195%26doi%3D10.1038%2Fsj.emboj.7600212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stebbins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becattini, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntzen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cellitti, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riel-Mehan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emdadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solinas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span> <span> </span><span class="NLM_article-title">Identification of a new JNK inhibitor targeting the JNK-JIP interaction site</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">16809</span>– <span class="NLM_lpage">16813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0805677105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.0805677105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=18922779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlertbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=16809-16813&author=J.+L.+Stebbinsauthor=S.+K.+Deauthor=T.+Machleidtauthor=B.+Becattiniauthor=J.+Vazquezauthor=C.+Kuntzenauthor=L.-H.+Chenauthor=J.+F.+Cellittiauthor=M.+Riel-Mehanauthor=A.+Emdadiauthor=G.+Solinasauthor=M.+Karinauthor=M.+Pellecchia&title=Identification+of+a+new+JNK+inhibitor+targeting+the+JNK-JIP+interaction+site&doi=10.1073%2Fpnas.0805677105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a new JNK inhibitor targeting the JNK-JIP interaction site</span></div><div class="casAuthors">Stebbins, John L.; De, Surya K.; Machleidt, Thomas; Becattini, Barbara; Vazquez, Jesus; Kuntzen, Christian; Chen, Li-Hsing; Cellitti, Jason F.; Riel-Mehan, Megan; Emdadi, Aras; Solinas, Giovanni; Karin, Michael; Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">16809-16813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">JNK is a stress-activated protein kinase that modulates pathways implicated in a variety of disease states.  JNK-interacting protein-1 (JIP1) is a scaffolding protein that enhances JNK signaling by creating a proximity effect between JNK and upstream kinases.  A minimal peptide region derived from JIP1 is able to inhibit JNK activity both in vitro and in cell.  We report here a series of small mols. JIP1 mimics that function as substrate competitive inhibitors of JNK.  One such compd., BI-78D3, dose-dependently inhibits the phosphorylation of JNK substrates both in vitro and in cell.  In animal studies, BI-78D3 not only blocks JNK dependent Con A-induced liver damage but also restores insulin sensitivity in mouse models of type 2 diabetes.  Our findings open the way for the development of protein kinase inhibitors targeting substrate specific docking sites, rather than the highly conserved ATP binding sites.  In view of its favorable inhibition profile, selectivity, and ability to function in the cellular milieu and in vivo, BI-78D3 represents not only a JNK inhibitor, but also a promising stepping stone toward the development of an innovative class of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5qVRg8EpCvbVg90H21EOLACvtfcHk0likyv4Nf5dElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlertbvM&md5=8fef92335594787de91361c5b1683f5e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0805677105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0805677105%26sid%3Dliteratum%253Aachs%26aulast%3DStebbins%26aufirst%3DJ.%2BL.%26aulast%3DDe%26aufirst%3DS.%2BK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DBecattini%26aufirst%3DB.%26aulast%3DVazquez%26aufirst%3DJ.%26aulast%3DKuntzen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.-H.%26aulast%3DCellitti%26aufirst%3DJ.%2BF.%26aulast%3DRiel-Mehan%26aufirst%3DM.%26aulast%3DEmdadi%26aufirst%3DA.%26aulast%3DSolinas%26aufirst%3DG.%26aulast%3DKarin%26aufirst%3DM.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520new%2520JNK%2520inhibitor%2520targeting%2520the%2520JNK-JIP%2520interaction%2520site%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D16809%26epage%3D16813%26doi%3D10.1073%2Fpnas.0805677105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaoud, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeVieux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbreath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moharana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalby, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerwin, S. M.</span></span> <span> </span><span class="NLM_article-title">A fluorescence-based assay for p38α recruitment site binders: Identification of Rooperol as a novel p38α kinase inhibitor</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1002%2Fcbic.201200529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23225637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslymsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=66-71&author=J.+Liauthor=T.+S.+Kaoudauthor=J.+LeVieuxauthor=B.+Gilbreathauthor=S.+Moharanaauthor=K.+N.+Dalbyauthor=S.+M.+Kerwin&title=A+fluorescence-based+assay+for+p38%CE%B1+recruitment+site+binders%3A+Identification+of+Rooperol+as+a+novel+p38%CE%B1+kinase+inhibitor&doi=10.1002%2Fcbic.201200529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A Fluorescence-Based Assay for p38α Recruitment Site Binders: Identification of Rooperol as a Novel p38α Kinase Inhibitor</span></div><div class="casAuthors">Li, Jing; Kaoud, Tamer S.; LeVieux, Jake; Gilbreath, Brad; Moharana, Swapna; Dalby, Kevin N.; Kerwin, Sean M.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">66-71</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This article presents a new class of N-alkynylimidazole small mols. that can serve as probes of the D-recruitment site (DRS) of p38α.  These novel p38α DRS probes have been identified through covalent interaction with Cys119.  Using alkyne-azide click chem., one such probe can be used to fluorescently label p38α in vitro and in cells.  The adduction of p38α by this probe is specifically blocked by the p38α-cognate MKK3 D-site peptide.  Using this probe to identify small mols. that recognize the p38α DRS, the anti-inflammatory natural product rooperol was identified as a novel p38α inhibitor.  This study demonstrates the utility of targeting specific cysteine residues assocd. with kinase recruitment sites for the discovery of novel inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6cWdbk0Gwj7Vg90H21EOLACvtfcHk0lhmg0zRnmA9RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslymsbzE&md5=b0edaf37ef7cd27747d84141ddae45bc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200529%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKaoud%26aufirst%3DT.%2BS.%26aulast%3DLeVieux%26aufirst%3DJ.%26aulast%3DGilbreath%26aufirst%3DB.%26aulast%3DMoharana%26aufirst%3DS.%26aulast%3DDalby%26aufirst%3DK.%2BN.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26atitle%3DA%2520fluorescence-based%2520assay%2520for%2520p38%25CE%25B1%2520recruitment%2520site%2520binders%253A%2520Identification%2520of%2520Rooperol%2520as%2520a%2520novel%2520p38%25CE%25B1%2520kinase%2520inhibitor%26jtitle%3DChemBioChem%26date%3D2013%26volume%3D14%26spage%3D66%26epage%3D71%26doi%3D10.1002%2Fcbic.201200529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stebbins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlickova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntzen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of a potent and selective dual ATP- and substrate-competitive subnanomolar bidentate C-JUN N-terminal kinase (JNK) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6206</span>– <span class="NLM_lpage">6214</span>, <span class="refDoi"> DOI: 10.1021/jm200479c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200479c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6206-6214&author=J.+L.+Stebbinsauthor=S.+K.+Deauthor=P.+Pavlickovaauthor=V.+Chenauthor=T.+Machleidtauthor=L.-H.+Chenauthor=C.+Kuntzenauthor=S.+Kitadaauthor=M.+Karinauthor=M.+Pellecchia&title=Design+and+characterization+of+a+potent+and+selective+dual+ATP-+and+substrate-competitive+subnanomolar+bidentate+C-JUN+N-terminal+kinase+%28JNK%29+inhibitor&doi=10.1021%2Fjm200479c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of a Potent and Selective Dual ATP- and Substrate-Competitive Subnanomolar Bidentate c-Jun N-Terminal Kinase (JNK) Inhibitor</span></div><div class="casAuthors">Stebbins, John L.; De, Surya K.; Pavlickova, Petra; Chen, Vida; Machleidt, Thomas; Chen, Li-Hsing; Kuntzen, Christian; Kitada, Shinichi; Karin, Michael; Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6206-6214</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">C-Jun N-terminal kinases (JNKs) represent valuable targets in the development of new therapies.  Present on the surface of JNK is a binding pocket for substrates and the scaffolding protein JIP1 in close proximity to the ATP binding pocket.  We propose that bidentate compds. linking the binding energies of weakly interacting ATP and substrate mimetics could result in potent and selective JNK inhibitors.  We describe here a bidentate mol., (I), designed against JNK.  I inhibits JNK kinase activity (IC50 = 18 nM; Ki = 1.5 nM) and JNK/substrate assocn. in a displacement assay (IC50 = 46 nM; Ki = 2 nM).  Our data demonstrate that I targets for the ATP and substrate-binding sites on JNK concurrently.  Finally, compd. I successfully inhibits JNK in a variety of cell-based expts., as well as in vivo where it is shown to protect against Jo-2 induced liver damage and improve glucose tolerance in diabetic mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorJZfTAV7pgrVg90H21EOLACvtfcHk0lhmg0zRnmA9RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7jL&md5=088d1dc1c124ab58b7597a541ba60947</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm200479c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200479c%26sid%3Dliteratum%253Aachs%26aulast%3DStebbins%26aufirst%3DJ.%2BL.%26aulast%3DDe%26aufirst%3DS.%2BK.%26aulast%3DPavlickova%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DV.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.-H.%26aulast%3DKuntzen%26aufirst%3DC.%26aulast%3DKitada%26aufirst%3DS.%26aulast%3DKarin%26aufirst%3DM.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DDesign%2520and%2520characterization%2520of%2520a%2520potent%2520and%2520selective%2520dual%2520ATP-%2520and%2520substrate-competitive%2520subnanomolar%2520bidentate%2520C-JUN%2520N-terminal%2520kinase%2520%2528JNK%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6206%26epage%3D6214%26doi%3D10.1021%2Fjm200479c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzarum, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komornik, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben Chetrit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">DEF pocket in p38α facilitates substrate selectivity and mediates autophosphorylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">19537</span>– <span class="NLM_lpage">19547</span>, <span class="refDoi"> DOI: 10.1074/jbc.m113.464511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1074%2Fjbc.m113.464511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23671282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=19537-19547&author=N.+Tzarumauthor=N.+Komornikauthor=D.+Ben+Chetritauthor=D.+Engelbergauthor=O.+Livnah&title=DEF+pocket+in+p38%CE%B1+facilitates+substrate+selectivity+and+mediates+autophosphorylation&doi=10.1074%2Fjbc.m113.464511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">DEF pocket in p38α facilitates substrate selectivity and mediates autophosphorylation</span></div><div class="casAuthors">Tzarum, Netanel; Komornik, Nadav; Ben Chetrit, Dorin; Engelberg, David; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">19537-19547</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Signaling processes are primarily promoted by mol. recognition and corresponding protein-protein interactions.  One of the key eukaryotic signaling pathways is the MAP kinase cascade involved in vital cellular processes such as cell proliferation, differentiation, apoptosis, and stress response.  The principle recognition site of MAP kinases, the common docking (CD) region, forms selective interactions with substrates, upstream activators, and phosphatases.  A second docking site, defined as the DEF site interaction pocket (DEF pocket), is formed subsequent to ERK2 and p38α activation.  Both crystal structures of p38α in its dually phosphorylated form and of intrinsically active mutants showed the DEF pocket, giving motivation for studying its role in substrate activation and selectivity.  Mutating selected DEF pocket residues significantly decreased the phosphorylation levels of three p38α substrates (ATFII, Elk-1, and MBP) with no apparent effect on the phosphorylation of MK2 kinase.  Conversely, mutating the CD region gave the opposite effect, suggesting p38α substrates can be classified into DEF-dependent and DEF-independent substrates.  In addn., mutating DEF pocket residues decreased the autophosphorylation capability of intrinsically active p38α mutants, suggesting DEF-mediated trans-autophosphorylation in p38α.  These results could contribute to understanding substrate selectivity of p38α and serve as a platform for designing p38α-selective DEF site blockers, which partially inhibit p38α binding DEF-dependent substrates, whereas maintaining its other functions intact.  In this context, preliminary results using synthetic peptides reveal significant inhibition of substrate phosphorylation by activated p38α.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqntKfmOsxz1bVg90H21EOLACvtfcHk0lhmg0zRnmA9RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7%252FF&md5=7a6b23bf517f038fef6420a570f7f740</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m113.464511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m113.464511%26sid%3Dliteratum%253Aachs%26aulast%3DTzarum%26aufirst%3DN.%26aulast%3DKomornik%26aufirst%3DN.%26aulast%3DBen%2BChetrit%26aufirst%3DD.%26aulast%3DEngelberg%26aufirst%3DD.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DDEF%2520pocket%2520in%2520p38%25CE%25B1%2520facilitates%2520substrate%2520selectivity%2520and%2520mediates%2520autophosphorylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D19537%26epage%3D19547%26doi%3D10.1074%2Fjbc.m113.464511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. X.</span></span> <span> </span><span class="NLM_article-title">A conserved motif in JNK/p38-specific mapk phosphatases as a determinant for JNK1 recognition and inactivation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10879</span>, <span class="refDoi"> DOI: 10.1038/ncomms10879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fncomms10879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26988444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1elsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10879&author=X.+Liuauthor=C.+S.+Zhangauthor=C.+Luauthor=S.+C.+Linauthor=J.+W.+Wuauthor=Z.+X.+Wang&title=A+conserved+motif+in+JNK%2Fp38-specific+mapk+phosphatases+as+a+determinant+for+JNK1+recognition+and+inactivation&doi=10.1038%2Fncomms10879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved motif in JNK/p38-specific MAPK phosphatases as a determinant for JNK1 recognition and inactivation</span></div><div class="casAuthors">Liu, Xin; Zhang, Chen-Song; Lu, Chang; Lin, Sheng-Cai; Wu, Jia-Wei; Wang, Zhi-Xin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10879</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mitogen-activated protein kinases (MAPKs), important in a large array of signalling pathways, are tightly controlled by a cascade of protein kinases and by MAPK phosphatases (MKPs).  MAPK signalling efficiency and specificity is modulated by protein-protein interactions between individual MAPKs and the docking motifs in cognate binding partners.  Two types of docking interactions have been identified: D-motif-mediated interaction and FXF-docking interaction.  Here we report the crystal structure of JNK1 bound to the catalytic domain of MKP7 at 2.4-Å resoln., providing high-resoln. structural insight into the FXF-docking interaction.  The 285FNFL288 segment in MKP7 directly binds to a hydrophobic site on JNK1 that is near the MAPK insertion and helix αG.  Biochem. studies further reveal that this highly conserved structural motif is present in all members of the MKP family, and the interaction mode is universal and crit. for the MKP-MAPK recognition and biol. function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2QX0w69JOtbVg90H21EOLACvtfcHk0lgzQgTixkHm5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1elsL0%253D&md5=9a7b84c5075d024d50e7bd4b12f4b68a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms10879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10879%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DC.%2BS.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DS.%2BC.%26aulast%3DWu%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DZ.%2BX.%26atitle%3DA%2520conserved%2520motif%2520in%2520JNK%252Fp38-specific%2520mapk%2520phosphatases%2520as%2520a%2520determinant%2520for%2520JNK1%2520recognition%2520and%2520inactivation%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D10879%26doi%3D10.1038%2Fncomms10879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoofnagle, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabuyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resing, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, N. G.</span></span> <span> </span><span class="NLM_article-title">Docking motif interactions in MAP kinases revealed by hydrogen exchange mass spectrometry</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(04)00161-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fs1097-2765%2804%2900161-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=15068802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvVaqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=43-55&author=T.+Leeauthor=A.+N.+Hoofnagleauthor=Y.+Kabuyamaauthor=J.+Stroudauthor=X.+Minauthor=E.+J.+Goldsmithauthor=L.+Chenauthor=K.+A.+Resingauthor=N.+G.+Ahn&title=Docking+motif+interactions+in+MAP+kinases+revealed+by+hydrogen+exchange+mass+spectrometry&doi=10.1016%2Fs1097-2765%2804%2900161-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Docking motif interactions in MAP kinases revealed by hydrogen exchange mass spectrometry</span></div><div class="casAuthors">Lee, Thomas; Hoofnagle, Andrew N.; Kabuyama, Yukihito; Stroud, James; Min, Xiaoshan; Goldsmith, Elizabeth J.; Chen, Lin; Resing, Katheryn A.; Ahn, Natalie G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-55</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein interactions between MAP kinases and substrates, activators, and scaffolding proteins are regulated by docking site motifs, one contg. basic residues proximal to Leu-X-Leu (DEJL) and a second contg. Phe-X-Phe (DEF).  Hydrogen exchange mass spectrometry was used to identify regions in MAP kinases protected from solvent by docking motif interactions.  Protection by DEJL peptide binding was obsd. in loops spanning β7-β8 and αD-αE in p38α and ERK2.  In contrast, protection by DEF binding to ERK2 revealed a distinct hydrophobic pocket for Phe-X-Phe binding formed between the P+1 site, αF helix, and the MAP kinase insert.  In inactive ERK2, this pocket is occluded by intramol. interactions with residues in the activation lip.  In vitro assays confirm the dependence of Elk1 and nucleoporin binding on ERK2 phosphorylation, and provide a structural basis for preferential involvement of active ERK in substrate binding and nuclear pore protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNNYWwgunLSrVg90H21EOLACvtfcHk0lgzQgTixkHm5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvVaqsbY%253D&md5=7e7f935ccd5d0b503c82b3ea1525dbbb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2804%2900161-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252804%252900161-3%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%26aulast%3DHoofnagle%26aufirst%3DA.%2BN.%26aulast%3DKabuyama%26aufirst%3DY.%26aulast%3DStroud%26aufirst%3DJ.%26aulast%3DMin%26aufirst%3DX.%26aulast%3DGoldsmith%26aufirst%3DE.%2BJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DResing%26aufirst%3DK.%2BA.%26aulast%3DAhn%26aufirst%3DN.%2BG.%26atitle%3DDocking%2520motif%2520interactions%2520in%2520MAP%2520kinases%2520revealed%2520by%2520hydrogen%2520exchange%2520mass%2520spectrometry%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D14%26spage%3D43%26epage%3D55%26doi%3D10.1016%2Fs1097-2765%2804%2900161-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Zapatero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gum, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groebe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clampit, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haasch, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoll, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderwood, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of non-ATP site inhibitors of the mitogen activated protein (MAP) kinases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1021/cb1002619</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb1002619" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=234-244&author=K.+M.+Comessauthor=C.+Sunauthor=C.+Abad-Zapateroauthor=E.+R.+Goedkenauthor=R.+J.+Gumauthor=D.+W.+Borhaniauthor=M.+Argiriadiauthor=D.+R.+Groebeauthor=Y.+Jiaauthor=J.+E.+Clampitauthor=D.+L.+Haaschauthor=H.+T.+Smithauthor=S.+Wangauthor=D.+Songauthor=M.+L.+Coenauthor=T.+E.+Cloutierauthor=H.+Tangauthor=X.+Chengauthor=C.+Quinnauthor=B.+Liuauthor=Z.+Xinauthor=G.+Liuauthor=E.+H.+Fryauthor=V.+Stollauthor=T.+I.+Ngauthor=D.+Banachauthor=D.+Marcotteauthor=D.+J.+Burnsauthor=D.+J.+Calderwoodauthor=P.+J.+Hajduk&title=Discovery+and+characterization+of+non-ATP+site+inhibitors+of+the+mitogen+activated+protein+%28MAP%29+kinases&doi=10.1021%2Fcb1002619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of Non-ATP Site Inhibitors of the Mitogen Activated Protein (MAP) Kinases</span></div><div class="casAuthors">Comess, Kenneth M.; Sun, Chaohong; Abad-Zapatero, Cele; Goedken, Eric R.; Gum, Rebecca J.; Borhani, David W.; Argiriadi, Maria; Groebe, Duncan R.; Jia, Yong; Clampit, Jill E.; Haasch, Deanna L.; Smith, Harriet T.; Wang, Sanyi; Song, Danying; Coen, Michael L.; Cloutier, Timothy E.; Tang, Hua; Cheng, Xueheng; Quinn, Christopher; Liu, Bo; Xin, Zhili; Liu, Gang; Fry, Elizabeth H.; Stoll, Vincent; Ng, Teresa I.; Banach, David; Marcotte, Doug; Burns, David J.; Calderwood, David J.; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-244</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of protein kinases has validated therapeutic utility for cancer, with at least seven kinase inhibitor drugs on the market.  Protein kinase inhibition also has significant potential for a variety of other diseases, including diabetes, pain, cognition, and chronic inflammatory and immunol. diseases.  However, as the vast majority of current approaches to kinase inhibition target the highly conserved ATP-binding site, the use of kinase inhibitors in treating nononcol. diseases may require great selectivity for the target kinase.  As protein kinases are signal transducers that are involved in binding to a variety of other proteins, targeting alternative, less conserved sites on the protein may provide an avenue for greater selectivity.  Here we report an affinity-based, high-throughput screening technique that allows nonbiased interrogation of small mol. libraries for binding to all exposed sites on a protein surface.  This approach was used to screen both the c-Jun N-terminal protein kinase Jnk-1 (involved in insulin signaling) and p38α (involved in the formation of TNFα and other cytokines).  In addn. to canonical ATP-site ligands, compds. were identified that bind to novel allosteric sites.  The nature, biol. relevance, and mode of binding of these ligands were extensively characterized using two-dimensional 1H/13C NMR spectroscopy, protein X-ray crystallog., surface plasmon resonance, and direct enzymic activity and activation cascade assays.  Jnk-1 and p38α both belong to the MAP kinase family, and the allosteric ligands for both targets bind similarly on a ledge of the protein surface exposed by the MAP insertion present in the CMGC family of protein kinases and distant from the active site.  Medicinal chem. studies resulted in an improved Jnk-1 ligand able to increase adiponectin secretion in human adipocytes and increase insulin-induced protein kinase PKB phosphorylation in human hepatocytes, in similar fashion to Jnk-1 siRNA and to rosiglitazone treatment.  Together, the data suggest that these new ligand series bind to a novel, allosteric, and physiol. relevant site and therefore represent a unique approach to identify kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyEwNo6QURxLVg90H21EOLACvtfcHk0ljvI6cG3Zk-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D&md5=857035713020ac66110b7ef6114232b9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fcb1002619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb1002619%26sid%3Dliteratum%253Aachs%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DGum%26aufirst%3DR.%2BJ.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DArgiriadi%26aufirst%3DM.%26aulast%3DGroebe%26aufirst%3DD.%2BR.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DClampit%26aufirst%3DJ.%2BE.%26aulast%3DHaasch%26aufirst%3DD.%2BL.%26aulast%3DSmith%26aufirst%3DH.%2BT.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DCoen%26aufirst%3DM.%2BL.%26aulast%3DCloutier%26aufirst%3DT.%2BE.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFry%26aufirst%3DE.%2BH.%26aulast%3DStoll%26aufirst%3DV.%26aulast%3DNg%26aufirst%3DT.%2BI.%26aulast%3DBanach%26aufirst%3DD.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCalderwood%26aufirst%3DD.%2BJ.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520non-ATP%2520site%2520inhibitors%2520of%2520the%2520mitogen%2520activated%2520protein%2520%2528MAP%2529%2520kinases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D234%26epage%3D244%26doi%3D10.1021%2Fcb1002619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diskin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">A novel lipid binding site formed by the map kinase insert in p38α</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>375</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.jmb.2007.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=17999933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=375&publication_year=2008&pages=70-79&author=R.+Diskinauthor=D.+Engelbergauthor=O.+Livnah&title=A+novel+lipid+binding+site+formed+by+the+map+kinase+insert+in+p38%CE%B1&doi=10.1016%2Fj.jmb.2007.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Lipid Binding Site Formed by the MAP Kinase Insert in p38α</span></div><div class="casAuthors">Diskin, Ron; Engelberg, David; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The p38 mitogen-activated protein (MAP) kinases function as signaling mols. essential for many cellular processes, particularly mediating stress response.  The activity of p38 MAP kinases is meticulously regulated to reach the desired cellular phenotype.  Several alternative activation and attenuation mechanisms have been characterized recently which include new phosphorylation sites.  Here we present the crystal structure of p38α MAP kinase in complex with n-octyl-β-glucopyranoside (β-OG) detergent.  The complex unveils a novel lipid-binding site formed by a local conformational change of the MAP kinase insert.  This binding is the first evidence of a possible role for the MAP kinase insert in p38.  The binding site can accommodate a large selection of lipidic mols.  In addn., we also show via biophys. methods that arachidonic acid (AA) and its derivs. bind p38α in vitro.  Based on our anal. we propose that the binding of lipids could fine-tune p38α catalytic activity towards a preferred phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6J5fd2Jqe77Vg90H21EOLACvtfcHk0ljvI6cG3Zk-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI&md5=88b97152503182d41fa0ea7b53271229</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DDiskin%26aufirst%3DR.%26aulast%3DEngelberg%26aufirst%3DD.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DA%2520novel%2520lipid%2520binding%2520site%2520formed%2520by%2520the%2520map%2520kinase%2520insert%2520in%2520p38%25CE%25B1%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D375%26spage%3D70%26epage%3D79%26doi%3D10.1016%2Fj.jmb.2007.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzarum, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg-Domovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gills, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">Lipid molecules induce p38α activation via a novel molecular switch</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.jmb.2012.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23079240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2012&pages=339-353&author=N.+Tzarumauthor=Y.+Eisenberg-Domovichauthor=J.+J.+Gillsauthor=P.+A.+Dennisauthor=O.+Livnah&title=Lipid+molecules+induce+p38%CE%B1+activation+via+a+novel+molecular+switch&doi=10.1016%2Fj.jmb.2012.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid Molecules Induce p38α Activation via a Novel Molecular Switch</span></div><div class="casAuthors">Tzarum, Netanel; Eisenberg-Domovich, Yael; Gills, Joell J.; Dennis, Phillip A.; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">339-353</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">P38α mitogen-activated protein kinase (MAPK) is generally activated by dual phosphorylation but has also been shown to exhibit alternative activation modes.  One of these modes included a direct interaction with phosphatidylinositol ether lipid analogs (PIA), inducing p38α autoactivation and apoptosis.  Perifosine, an Akt inhibitor in phase II clin. trials, also showed p38α activation properties similar to those of PIAs.  The crystal structures of p38α in complex with PIA23, PIA24 and perifosine provide insights into this unique activation mode.  The activating mols. bind a unique hydrophobic binding site in the kinase C'-lobe formed in part by the MAPK insert region.  In addn., there are conformational changes in the short αEF/αF loop region that acts as an activation switch, inducing autophosphorylation.  Structural and biochem. characterization of the αEF/αF loop identified Trp197 as a key residue in the lipid binding and in p38α catalytic activity.  The lipid binding site also accommodates hydrophobic inhibitor mols., and thus can serve as a novel p38α-target for specific activation or inhibition, with novel therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ5MaNgn-kFbVg90H21EOLACvtfcHk0ljvI6cG3Zk-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I&md5=4341b3620e35b8a54f68448817ca157b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTzarum%26aufirst%3DN.%26aulast%3DEisenberg-Domovich%26aufirst%3DY.%26aulast%3DGills%26aufirst%3DJ.%2BJ.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DLipid%2520molecules%2520induce%2520p38%25CE%25B1%2520activation%2520via%2520a%2520novel%2520molecular%2520switch%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D424%26spage%3D339%26epage%3D353%26doi%3D10.1016%2Fj.jmb.2012.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanderpool, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ping, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phonephaly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margosiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermolieff, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the CHK1 allosteric inhibitor binding site</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9823</span>– <span class="NLM_lpage">9830</span>, <span class="refDoi"> DOI: 10.1021/bi900258v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi900258v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9823-9830&author=D.+Vanderpoolauthor=T.+O.+Johnsonauthor=C.+Pingauthor=S.+Bergqvistauthor=G.+Altonauthor=S.+Phonephalyauthor=E.+Ruiauthor=C.+Luoauthor=Y.-L.+Dengauthor=S.+Grantauthor=T.+Quenzerauthor=S.+Margosiakauthor=J.+Registerauthor=E.+Brownauthor=J.+Ermolieff&title=Characterization+of+the+CHK1+allosteric+inhibitor+binding+site&doi=10.1021%2Fbi900258v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the CHK1 Allosteric Inhibitor Binding Site</span></div><div class="casAuthors">Vanderpool, Darin; Johnson, Ted O.; Ping, Chen; Bergqvist, Simon; Alton, Gordon; Phonephaly, Soneprasith; Rui, Eugene; Luo, Chun; Deng, Ya-Li; Grant, Stephan; Quenzer, Terri; Margosiak, Steve; Register, James; Brown, Ed; Ermolieff, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9823-9830</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Checkpoint kinase 1 (CHK1) is a key element in the DNA damage response pathway and plays a crucial role in the S-G2-phase checkpoint.  Inhibiting CHK1 is a therapeutic strategy involving abrogation of the G2/M mitotic checkpoint defense of tumor cells toward lethal damage induced by DNA-directed chemotherapeutic agents.  To date, most CHK1 inhibition approaches have involved targeting the ATP site of this kinase.  In this study, we provide crystallog. and kinetic characterization of two small mol. inhibitors that bind to an allosteric site in the proximity of the CHK1 substrate site.  Anal. of kinetic and biophys. data has led to the conclusion that these small mol. allosteric site inhibitors of CHK1 are reversible and are neither ATP- nor peptide substrate-competitive.  Ki values of 1.89 and 0.15 μM, resp., have been detd. for these compds. using a mixed inhibitor kinetic anal.  Cocrystal structures of the inhibitors bound to CHK1 reveal an allosteric site, unique to CHK1, located in the C-terminal domain and consisting of a shallow groove linked to a small hydrophobic pocket.  The pocket displays induced fit characteristics in the presence of the two inhibitors.  These findings establish the potential for the development of highly selective CHK1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMx2Syic38ULVg90H21EOLACvtfcHk0ljaf03CdgStBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO&md5=463e6df560e0cc8b3e8fed96be567284</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi900258v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi900258v%26sid%3Dliteratum%253Aachs%26aulast%3DVanderpool%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DPing%26aufirst%3DC.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DPhonephaly%26aufirst%3DS.%26aulast%3DRui%26aufirst%3DE.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DY.-L.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DMargosiak%26aufirst%3DS.%26aulast%3DRegister%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DE.%26aulast%3DErmolieff%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520the%2520CHK1%2520allosteric%2520inhibitor%2520binding%2520site%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D9823%26epage%3D9830%26doi%3D10.1021%2Fbi900258v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Converso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartingh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreatsoulas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stirdivant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drakas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepp-Lorenzino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jezequel-Sur, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span> <span> </span><span class="NLM_article-title">Development of thioquinazolinones, allosteric CHK1 kinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1240</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.12.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bmcl.2008.12.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=19155174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1240-1244&author=A.+Conversoauthor=T.+Hartinghauthor=R.+M.+Garbaccioauthor=E.+Tasberauthor=K.+Rickertauthor=M.+E.+Fraleyauthor=Y.+Yanauthor=C.+Kreatsoulasauthor=S.+Stirdivantauthor=B.+Drakasauthor=E.+S.+Walshauthor=K.+Hamiltonauthor=C.+A.+Buserauthor=X.+Maoauthor=M.+T.+Abramsauthor=S.+C.+Beckauthor=W.+Taoauthor=R.+Lobellauthor=L.+Sepp-Lorenzinoauthor=J.+Zugay-Murphyauthor=V.+Sardanaauthor=S.+K.+Munshiauthor=S.+M.+Jezequel-Surauthor=P.+D.+Zuckauthor=G.+D.+Hartman&title=Development+of+thioquinazolinones%2C+allosteric+CHK1+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2008.12.076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Development of thioquinazolinones, allosteric Chk1 kinase inhibitors</span></div><div class="casAuthors">Converso, Antonella; Hartingh, Timothy; Garbaccio, Robert M.; Tasber, Edward; Rickert, Keith; Fraley, Mark E.; Yan, Youwei; Kreatsoulas, Constantine; Stirdivant, Steve; Drakas, Bob; Walsh, Eileen S.; Hamilton, Kelly; Buser, Carolyn A.; Mao, Xianzhi; Abrams, Marc T.; Beck, Stephen C.; Tao, Weikang; Lobell, Rob; Sepp-Lorenzino, Laura; Zugay-Murphy, Joan; Sardana, Vinod; Munshi, Sanjeev K.; Jezequel-Sur, Sylvie Marie; Zuck, Paul D.; Hartman, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1240-1244</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high throughput screening campaign was designed to identify allosteric inhibitors of Chk1 kinase by testing compds. at high concn.  Activity was then obsd. at Km for ATP and at near-physiol. concns. of ATP.  This strategy led to the discovery of a non-ATP competitive thioquinazolinone series which was optimized for potency and stability.  An X-ray crystal structure for the complex of our best inhibitor bound to Chk1 was solved, indicating that it binds to an allosteric site ∼13 Å from the ATP binding site.  Preliminary data is presented for several of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bIRNnirJ4bVg90H21EOLACvtfcHk0ljaf03CdgStBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D&md5=e91440930e976b9aafcfbb1597679511</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.12.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.12.076%26sid%3Dliteratum%253Aachs%26aulast%3DConverso%26aufirst%3DA.%26aulast%3DHartingh%26aufirst%3DT.%26aulast%3DGarbaccio%26aufirst%3DR.%2BM.%26aulast%3DTasber%26aufirst%3DE.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DFraley%26aufirst%3DM.%2BE.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DKreatsoulas%26aufirst%3DC.%26aulast%3DStirdivant%26aufirst%3DS.%26aulast%3DDrakas%26aufirst%3DB.%26aulast%3DWalsh%26aufirst%3DE.%2BS.%26aulast%3DHamilton%26aufirst%3DK.%26aulast%3DBuser%26aufirst%3DC.%2BA.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DAbrams%26aufirst%3DM.%2BT.%26aulast%3DBeck%26aufirst%3DS.%2BC.%26aulast%3DTao%26aufirst%3DW.%26aulast%3DLobell%26aufirst%3DR.%26aulast%3DSepp-Lorenzino%26aufirst%3DL.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DSardana%26aufirst%3DV.%26aulast%3DMunshi%26aufirst%3DS.%2BK.%26aulast%3DJezequel-Sur%26aufirst%3DS.%2BM.%26aulast%3DZuck%26aufirst%3DP.%2BD.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26atitle%3DDevelopment%2520of%2520thioquinazolinones%252C%2520allosteric%2520CHK1%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1240%26epage%3D1244%26doi%3D10.1016%2Fj.bmcl.2008.12.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, N.</span></span> <span> </span><span class="NLM_article-title">A new target for an old drug: Identifying mitoxantrone as a nanomolar inhibitor of PIM1 kinase via kinome-wide selectivity modeling</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2619</span>– <span class="NLM_lpage">2629</span>, <span class="refDoi"> DOI: 10.1021/jm400045y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400045y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2619-2629&author=X.+Wanauthor=W.+Zhangauthor=L.+Liauthor=Y.+Xieauthor=W.+Liauthor=N.+Huang&title=A+new+target+for+an+old+drug%3A+Identifying+mitoxantrone+as+a+nanomolar+inhibitor+of+PIM1+kinase+via+kinome-wide+selectivity+modeling&doi=10.1021%2Fjm400045y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling</span></div><div class="casAuthors">Wan, Xiaobo; Zhang, Wei; Li, Li; Xie, Yuting; Li, Wei; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2619-2629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rational design of selective kinase inhibitors remains a great challenge.  Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile.  We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases.  Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays.  Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells.  We further detd. the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition.  Although mitoxantrone's mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute to mitoxantrone's therapeutic efficacy and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCdODeUj1frVg90H21EOLACvtfcHk0ljaf03CdgStBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D&md5=ff53cfe69b80b0b28821a4bfb4583378</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm400045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400045y%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DA%2520new%2520target%2520for%2520an%2520old%2520drug%253A%2520Identifying%2520mitoxantrone%2520as%2520a%2520nanomolar%2520inhibitor%2520of%2520PIM1%2520kinase%2520via%2520kinome-wide%2520selectivity%2520modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2619%26epage%3D2629%26doi%3D10.1021%2Fjm400045y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gureasko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">An allosteric mechanism for activation of the kinase domain of epidermal growth factor receptor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1149</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.cell.2006.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=16777603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=1137-1149&author=X.+Zhangauthor=J.+Gureaskoauthor=K.+Shenauthor=P.+A.+Coleauthor=J.+Kuriyan&title=An+allosteric+mechanism+for+activation+of+the+kinase+domain+of+epidermal+growth+factor+receptor&doi=10.1016%2Fj.cell.2006.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric mechanism for activation of the kinase domain of epidermal growth factor receptor</span></div><div class="casAuthors">Zhang, Xuewu; Gureasko, Jodi; Shen, Kui; Cole, Philip A.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1137-1149</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mechanism by which the epidermal growth factor receptor (EGFR) is activated upon dimerization has eluded definition.  We find that the EGFR kinase domain can be activated by increasing its local concn. or by mutating a leucine (L834R) in the activation loop, the phosphorylation of which is not required for activation.  This suggests that the kinase domain is intrinsically auto-inhibited, and an intermol. interaction promotes its activation.  Using further mutational anal. and crystallog. we demonstrate that the auto-inhibited conformation of the EGFR kinase domain resembles that of Src and cyclin-dependent kinases (CDKs).  EGFR activation results from the formation of an asym. dimer in which the C-terminal lobe of one kinase domain plays a role analogous to that of cyclin in activated CDK/cyclin complexes.  The CDK/cyclin-like complex formed by two kinase domains thus explains the activation of EGFR-family receptors by homo- or heterodimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQsRAzLF-wK7Vg90H21EOLACvtfcHk0lg-jXnOMBn2OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D&md5=f7b6795851425346f588cbd682125c93</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGureasko%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DAn%2520allosteric%2520mechanism%2520for%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D1137%26epage%3D1149%26doi%3D10.1016%2Fj.cell.2006.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickin, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the egf receptor by binding of MIG6 to an activating kinase domain interface</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nature05998</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature05998" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=18046415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=741-744&author=X.+Zhangauthor=K.+A.+Pickinauthor=R.+Boseauthor=N.+Juraauthor=P.+A.+Coleauthor=J.+Kuriyan&title=Inhibition+of+the+egf+receptor+by+binding+of+MIG6+to+an+activating+kinase+domain+interface&doi=10.1038%2Fnature05998"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the EGF receptor by binding of MIG6 to an activating kinase domain interface</span></div><div class="casAuthors">Zhang, Xuewu; Pickin, Kerry A.; Bose, Ron; Jura, Natalia; Cole, Philip A.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7170</span>),
    <span class="NLM_cas:pages">741-744</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the epidermal growth factor receptor family (EGFR/ERBB1, ERBB2/HER2, ERBB3/HER3 and ERBB4/HER4) are key targets for inhibition in cancer therapy.  Crit. for activation is the formation of an asym. dimer by the intracellular kinase domains, in which the carboxy-terminal lobe (C lobe) of one kinase domain induces an active conformation in the other.  The cytoplasmic protein MIG6 (mitogen-induced gene 6; also known as ERRFI1) interacts with and inhibits the kinase domains of EGFR and ERBB2 (refs).  Crystal structures of complexes between the EGFR kinase domain and a fragment of MIG6 show that a ∼25-residue epitope (segment 1) from MIG6 binds to the distal surface of the C lobe of the kinase domain.  Biochem. and cell-based analyses confirm that this interaction contributes to EGFR inhibition by blocking the formation of the activating dimer interface.  A longer MIG6 peptide that is extended C terminal to segment 1 has increased potency as an inhibitor of the activated EGFR kinase domain, while retaining a crit. dependence on segment 1.  We show that signaling by EGFR mols. that contain constitutively active kinase domains still requires formation of the asym. dimer, underscoring the importance of dimer interface blockage in MIG6-mediated inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2aM4ZGKQTr7Vg90H21EOLACvtfcHk0lg-jXnOMBn2OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrP&md5=788822e6303a35eddea3e032875c57d7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature05998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05998%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DPickin%26aufirst%3DK.%2BA.%26aulast%3DBose%26aufirst%3DR.%26aulast%3DJura%26aufirst%3DN.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520the%2520egf%2520receptor%2520by%2520binding%2520of%2520MIG6%2520to%2520an%2520activating%2520kinase%2520domain%2520interface%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D741%26epage%3D744%26doi%3D10.1038%2Fnature05998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rothweiler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stensen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. S.</span></span> <span> </span><span class="NLM_article-title">Luciferin and derivatives as a DYRK selective scaffold for the design of protein kinase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.ejmech.2015.02.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25768698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFGjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=140-148&author=U.+Rothweilerauthor=J.+Erikssonauthor=W.+Stensenauthor=F.+Leesonauthor=R.+A.+Enghauthor=J.+S.+Svendsen&title=Luciferin+and+derivatives+as+a+DYRK+selective+scaffold+for+the+design+of+protein+kinase+inhibitors&doi=10.1016%2Fj.ejmech.2015.02.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Luciferin and derivatives as a DYRK selective scaffold for the design of protein kinase inhibitors</span></div><div class="casAuthors">Rothweiler, Ulli; Eriksson, Jonas; Stensen, Wenche; Leeson, Frederick; Engh, Richard A.; Svendsen, John S.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">140-148</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">D-Luciferin is widely used as a substrate in luciferase catalyzed bioluminescence assays for in vitro studies.  However, little is known about cross reactivity and potential interference of D-luciferin with other enzymes.  The authors serendipitously found that firefly luciferin inhibited the CDK2/Cyclin A protein kinase.  Inhibition profiling of D-luciferin over a 103-protein kinase panel showed significant inhibition of a small set of protein kinases, in particular the DYRK-family, but also other members of the CMGC-group, including ERK8 and CK2.  Inhibition profiling on a 16-member focused library derived from D-luciferin confirms that D-luciferin represents a DYRK-selective chemotype of fragment-like mol. wt.  Thus, observation of its inhibitory activity and the initial SAR information reported here promise to be useful for further design of protein kinase inhibitors with related scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqEtt2EfYajrVg90H21EOLACvtfcHk0lg-jXnOMBn2OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFGjsL4%253D&md5=5d3fd3650bf11958c7e60a5656b28dfd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DRothweiler%26aufirst%3DU.%26aulast%3DEriksson%26aufirst%3DJ.%26aulast%3DStensen%26aufirst%3DW.%26aulast%3DLeeson%26aufirst%3DF.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DSvendsen%26aufirst%3DJ.%2BS.%26atitle%3DLuciferin%2520and%2520derivatives%2520as%2520a%2520DYRK%2520selective%2520scaffold%2520for%2520the%2520design%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D140%26epage%3D148%26doi%3D10.1016%2Fj.ejmech.2015.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaffarogullari, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masterson, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalfe, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traaseth, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balatri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musa, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distefano, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veglia, G.</span></span> <span> </span><span class="NLM_article-title">A myristoyl/phosphoserine switch controls cAMP-dependent protein kinase association to membranes</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>411</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.06.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.jmb.2011.06.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=21740913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVaitLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=411&publication_year=2011&pages=823-836&author=E.+C.+Gaffarogullariauthor=L.+R.+Mastersonauthor=E.+E.+Metcalfeauthor=N.+J.+Traasethauthor=E.+Balatriauthor=M.+M.+Musaauthor=D.+Mullenauthor=M.+D.+Distefanoauthor=G.+Veglia&title=A+myristoyl%2Fphosphoserine+switch+controls+cAMP-dependent+protein+kinase+association+to+membranes&doi=10.1016%2Fj.jmb.2011.06.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A Myristoyl/Phosphoserine Switch Controls cAMP-Dependent Protein Kinase Association to Membranes</span></div><div class="casAuthors">Gaffarogullari, Ece C.; Masterson, Larry R.; Metcalfe, Emily E.; Traaseth, Nathaniel J.; Balatri, Erica; Musa, Musa M.; Mullen, Daniel; Distefano, Mark D.; Veglia, Gianluigi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">411</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-836</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The cAMP-dependent protein kinase [protein kinase A (PKA)] mediates a myriad of cellular signaling events, and its activity is tightly regulated in both space and time.  Among these regulatory mechanisms is N-myristoylation, whose biol. role has been elusive.  Using a combination of thermodn., kinetics, and spectroscopic methods, we analyzed the effects of N-myristoylation and phosphorylation at Ser10 on the interactions of PKA with model membranes.  We found that in the absence of lipids, the myristoyl group is tucked into the hydrophobic binding pocket of the enzyme (myr-in state).  Upon assocn. with lipid bilayers, the myristoyl group is extruded and inserts into the hydrocarbon region of the lipid bilayer (myr-out state).  NMR data indicate that the enzyme undergoes conformational equil. between myr-in and myr-out states, which can be shifted by either interaction with membranes and/or phosphorylation at Ser10.  Our results provide evidence that the membrane binding motif of the myristoylated C-subunit of PKA (PKA-C) steers the enzyme toward lipids independent of its regulatory subunit or an A-kinase anchoring protein, providing an addnl. mechanism to localize the enzyme near membrane-bound substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRbg7PQNbdnbVg90H21EOLACvtfcHk0lipPrK-xL_2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVaitLzL&md5=3e039c8d0ef13a83d98b453f838a387a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.06.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.06.034%26sid%3Dliteratum%253Aachs%26aulast%3DGaffarogullari%26aufirst%3DE.%2BC.%26aulast%3DMasterson%26aufirst%3DL.%2BR.%26aulast%3DMetcalfe%26aufirst%3DE.%2BE.%26aulast%3DTraaseth%26aufirst%3DN.%2BJ.%26aulast%3DBalatri%26aufirst%3DE.%26aulast%3DMusa%26aufirst%3DM.%2BM.%26aulast%3DMullen%26aufirst%3DD.%26aulast%3DDistefano%26aufirst%3DM.%2BD.%26aulast%3DVeglia%26aufirst%3DG.%26atitle%3DA%2520myristoyl%252Fphosphoserine%2520switch%2520controls%2520cAMP-dependent%2520protein%2520kinase%2520association%2520to%2520membranes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D411%26spage%3D823%26epage%3D836%26doi%3D10.1016%2Fj.jmb.2011.06.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buosi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontiggia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanism of Aurora A kinase autophosphorylation and its allosteric activation by TPX2</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e02667</span> <span class="refDoi"> DOI: 10.7554/elife.02667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.7554%2Felife.02667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=24867643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Smtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&author=A.+Zorbaauthor=V.+Buosiauthor=S.+Kutterauthor=N.+Kernauthor=F.+Pontiggiaauthor=Y.-J.+Choauthor=D.+Kern&title=Molecular+mechanism+of+Aurora+A+kinase+autophosphorylation+and+its+allosteric+activation+by+TPX2&doi=10.7554%2Felife.02667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanism of Aurora A kinase autophosphorylation and its allosteric activation by TPX2</span></div><div class="casAuthors">Zorba, Adelajda; Buosi, Vanessa; Kutter, Steffen; Kern, Nadja; Pontiggia, Francesco; Cho, Young-Jin; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e02667/1-e02667/24, 24</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">We elucidate the mol. mechanisms of two distinct activation strategies (autophosphorylation and TPX2-mediated activation) in human Aurora A kinase.  Classic allosteric activation is in play where either activation loop phosphorylation or TPX2 binding to a conserved hydrophobic groove shifts the equil. far towards the active conformation.  We resolve the controversy about the mechanism of autophosphorylation by demonstrating intermol. autophosphorylation in a long-lived dimer by combining X-ray crystallog. with functional assays.  We then address the allosteric activation by TPX2 through activity assays and the crystal structure of a domain-swapped dimer of dephosphorylated Aurora A and TPX21-25.  While autophosphorylation is the key regulatory mechanism in the centrosomes in the early stages of mitosis, allosteric activation by TPX2 of dephosphorylated Aurora A could be at play in the spindle microtubules.  The mechanistic insights into autophosphorylation and allosteric activation by TPX2 binding proposed here, may have implications for understanding regulation of other protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLzzlNNQcyXrVg90H21EOLACvtfcHk0lipPrK-xL_2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Smtb7E&md5=fadf30228b273acd1d60fbc6be1e1098</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.7554%2Felife.02667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252Felife.02667%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DBuosi%26aufirst%3DV.%26aulast%3DKutter%26aufirst%3DS.%26aulast%3DKern%26aufirst%3DN.%26aulast%3DPontiggia%26aufirst%3DF.%26aulast%3DCho%26aufirst%3DY.-J.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DMolecular%2520mechanism%2520of%2520Aurora%2520A%2520kinase%2520autophosphorylation%2520and%2520its%2520allosteric%2520activation%2520by%2520TPX2%26jtitle%3DeLife%26date%3D2014%26volume%3D3%26doi%3D10.7554%2Felife.02667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tada, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of MKK4 kinase domain reveal that substrate peptide binds to an allosteric site and induces an auto-inhibition state</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>400</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2010.08.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bbrc.2010.08.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20732303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1WltrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=369-373&author=T.+Matsumotoauthor=T.+Kinoshitaauthor=Y.+Kiriiauthor=K.+Yokotaauthor=K.+Hamadaauthor=T.+Tada&title=Crystal+structures+of+MKK4+kinase+domain+reveal+that+substrate+peptide+binds+to+an+allosteric+site+and+induces+an+auto-inhibition+state&doi=10.1016%2Fj.bbrc.2010.08.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of MKK4 kinase domain reveal that substrate peptide binds to an allosteric site and induces an auto-inhibition state</span></div><div class="casAuthors">Matsumoto, Takashi; Kinoshita, Takayoshi; Kirii, Yasuyuki; Yokota, Koichi; Hamada, Kensaku; Tada, Toshiji</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">369-373</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MKK4 kinase activates both JNK and p38 kinases.  Here, the authors detd. the crystal structures of the human non-phosphorylated MKK4 kinase domain (npMKK4) complexed with AMP-PNP (npMKK4/AMP) and a ternary complex of npMKK4, AMP-PNP and p38α peptide (npMKK4/AMP/p38).  These crystal structures revealed that the p38α peptide-bound npMKK4 at the allosteric site rather than at the putative substrate binding site and induced an auto-inhibition state.  While the activation loop of the npMKK4/AMP complex was disordered, in the npMKK4/AMP/p38 complex it configured a long α-helix, which prevented substrate access to the active site and αC-helix movement to the active configuration of MKK4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-U_oOxzk9frVg90H21EOLACvtfcHk0lipPrK-xL_2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1WltrjK&md5=17398782a2a8508f07bf6b1f5745c676</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2010.08.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2010.08.071%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DT.%26aulast%3DKinoshita%26aufirst%3DT.%26aulast%3DKirii%26aufirst%3DY.%26aulast%3DYokota%26aufirst%3DK.%26aulast%3DHamada%26aufirst%3DK.%26aulast%3DTada%26aufirst%3DT.%26atitle%3DCrystal%2520structures%2520of%2520MKK4%2520kinase%2520domain%2520reveal%2520that%2520substrate%2520peptide%2520binds%2520to%2520an%2520allosteric%2520site%2520and%2520induces%2520an%2520auto-inhibition%2520state%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D400%26spage%3D369%26epage%3D373%26doi%3D10.1016%2Fj.bbrc.2010.08.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braisted, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raphael, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Site-directed ligand discovery</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">9367</span>– <span class="NLM_lpage">9372</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.17.9367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.97.17.9367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10944209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtVensr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=9367-9372&author=D.+A.+Erlansonauthor=A.+C.+Braistedauthor=D.+R.+Raphaelauthor=M.+Randalauthor=R.+M.+Stroudauthor=E.+M.+Gordonauthor=J.+A.+Wells&title=Site-directed+ligand+discovery&doi=10.1073%2Fpnas.97.17.9367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Site-directed ligand discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Braisted, Andrew C.; Raphael, Darren R.; Randal, Mike; Stroud, Robert M.; Gordon, Eric M.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9367-9372</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report a strategy (called "tethering") to discover low mol. wt. ligands (≈250 Da) that bind weakly to targeted sites on proteins through an intermediary disulfide tether.  A native or engineered cysteine in a protein is allowed to react reversibly with a small library of disulfide-contg. mols. (≈1,200 compds.) at concns. typically used in drug screening (10 to 200 μM).  The cysteine-captured ligands, which are readily identified by MS, are among the most stable complexes, even though in the absence of the covalent tether the ligands may bind very weakly.  This method was applied to generate a potent inhibitor for thymidylate synthase, an essential enzyme in pyrimidine metab. with therapeutic applications in cancer and infectious diseases.  The affinity of the untethered ligand (Ki≈1 mM) was improved 3,000-fold by synthesis of a small set of analogs with the aid of crystallog. structures of the tethered complex.  Such site-directed ligand discovery allows one to nucleate drug design from a spatially targeted lead fragment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4pxPEoDE3FrVg90H21EOLACvtfcHk0lipPrK-xL_2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtVensr8%253D&md5=fef57a0d21af9bc7f6f1f4bf8c023ae5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.17.9367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.17.9367%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DBraisted%26aufirst%3DA.%2BC.%26aulast%3DRaphael%26aufirst%3DD.%2BR.%26aulast%3DRandal%26aufirst%3DM.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DGordon%26aufirst%3DE.%2BM.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSite-directed%2520ligand%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D9367%26epage%3D9372%26doi%3D10.1073%2Fpnas.97.17.9367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inesta-Vaquera, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughlin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoGrasso, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent inhibitors of JNK</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.chembiol.2011.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=22284361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=140-154&author=T.+Zhangauthor=F.+Inesta-Vaqueraauthor=M.+Niepelauthor=J.+Zhangauthor=S.+B.+Ficarroauthor=T.+Machleidtauthor=T.+Xieauthor=J.+A.+Martoauthor=N.+Kimauthor=T.+Simauthor=J.+D.+Laughlinauthor=H.+Parkauthor=P.+V.+LoGrassoauthor=M.+Patricelliauthor=T.+K.+Nomanbhoyauthor=P.+K.+Sorgerauthor=D.+R.+Alessiauthor=N.+S.+Gray&title=Discovery+of+potent+and+selective+covalent+inhibitors+of+JNK&doi=10.1016%2Fj.chembiol.2011.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Inhibitors of JNK</span></div><div class="casAuthors">Zhang, Tinghu; Inesta-Vaquera, Francisco; Niepel, Mario; Zhang, Jianming; Ficarro, Scott B.; Machleidt, Thomas; Xie, Ting; Marto, Jarrod A.; Kim, Nam Doo; Sim, Taebo; Laughlin, John D.; Park, Hajeung; LoGrasso, Philip V.; Patricelli, Matt; Nomanbhoy, Tyzoon K.; Sorger, Peter K.; Alessi, Dario R.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-154</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The mitogen-activated kinases JNK1/2/3 are key enzymes in signaling modules that transduce and integrate extracellular stimuli into coordinated cellular response.  Here, we report the discovery of irreversible inhibitors of JNK1/2/3.  We describe two JNK3 cocrystal structures at 2.60 and 2.97 Å resoln. that show the compds. form covalent bonds with a conserved cysteine residue.  JNK-IN-8 is a selective JNK inhibitor that inhibits phosphorylation of c-Jun, a direct substrate of JNK, in cells exposed to submicromolar drug in a manner that depends on covalent modification of the conserved cysteine residue.  Extensive biochem., cellular, and pathway-based profiling establish the selectivity of JNK-IN-8 for JNK and suggests that the compd. will be broadly useful as a pharmacol. probe of JNK-dependent signal transduction.  Potential lead compds. have also been identified for kinases, including IRAK1, PIK3C3, PIP4K2C, and PIP5K3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXs0FY7FSmt7Vg90H21EOLACvtfcHk0lgKcmhqo120jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghurw%253D&md5=f7d9647fb2be162c637b1d31a778ceb4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DInesta-Vaquera%26aufirst%3DF.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DLaughlin%26aufirst%3DJ.%2BD.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLoGrasso%26aufirst%3DP.%2BV.%26aulast%3DPatricelli%26aufirst%3DM.%26aulast%3DNomanbhoy%26aufirst%3DT.%2BK.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520inhibitors%2520of%2520JNK%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D140%26epage%3D154%26doi%3D10.1016%2Fj.chembiol.2011.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Temmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Diego, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogenberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonko, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilmanns, M.</span></span> <span> </span><span class="NLM_article-title">A PEF/Y substrate recognition and signature motif plays a critical role in DAPK-related kinase activity</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.chembiol.2013.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=24440081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFChsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=264-273&author=K.+Temmermanauthor=I.+de+Diegoauthor=V.+Pogenbergauthor=B.+Simonauthor=W.+Jonkoauthor=X.+Liauthor=M.+Wilmanns&title=A+PEF%2FY+substrate+recognition+and+signature+motif+plays+a+critical+role+in+DAPK-related+kinase+activity&doi=10.1016%2Fj.chembiol.2013.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A PEF/Y Substrate Recognition and Signature Motif Plays a Critical Role in DAPK-Related Kinase Activity</span></div><div class="casAuthors">Temmerman, Koen; de Diego, Inaki; Pogenberg, Vivian; Simon, Bertrand; Jonko, Weronika; Li, Xun; Wilmanns, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-273</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Knowledge about protein kinase substrate preferences is biased toward residues immediately adjacent to the site of phosphorylation.  By a combined structural, biochem., and cellular approach, we have discovered an unexpected substrate recognition element with the consensus sequence PEF/Y in the tumor suppressor death-assocd. protein kinase 1.  This motif can be effectively blocked by a specific pseudosubstrate-type interaction with an autoregulatory domain of this kinase.  In this arrangement, the central PEF/Y glutamate interacts with a conserved arginine distant to the phosphorylation site in sequence and structure.  We also demonstrate that the element is crucial for kinase activity regulation and substrate recognition.  The PEF/Y motif distinguishes close death-assocd. protein kinase relatives from canonical calcium/calmodulin-dependent protein kinases.  Insight into this signature and mode of action offers new opportunities to identify specific small mol. inhibitors in PEF/Y-contg. protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZr7KNdPm2LVg90H21EOLACvtfcHk0lgKcmhqo120jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFChsLY%253D&md5=d7d49078ecdffd22051d28f8f820c19e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DTemmerman%26aufirst%3DK.%26aulast%3Dde%2BDiego%26aufirst%3DI.%26aulast%3DPogenberg%26aufirst%3DV.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DJonko%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWilmanns%26aufirst%3DM.%26atitle%3DA%2520PEF%252FY%2520substrate%2520recognition%2520and%2520signature%2520motif%2520plays%2520a%2520critical%2520role%2520in%2520DAPK-related%2520kinase%2520activity%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D264%26epage%3D273%26doi%3D10.1016%2Fj.chembiol.2013.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh Bahia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silakari, O.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C-theta inhibitors: A novel therapy for inflammatory disorders</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4725</span>– <span class="NLM_lpage">4746</span>, <span class="refDoi"> DOI: 10.2174/138161212802651625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.2174%2F138161212802651625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=22830352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2nsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4725-4746&author=S.+Chandauthor=N.+Mehtaauthor=M.+Singh+Bahiaauthor=A.+Dixitauthor=O.+Silakari&title=Protein+kinase+C-theta+inhibitors%3A+A+novel+therapy+for+inflammatory+disorders&doi=10.2174%2F138161212802651625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C-theta inhibitors: a novel therapy for inflammatory disorders</span></div><div class="casAuthors">Chand, Sukhvir; Mehta, Nisha; Bahia, Malkeet Singh; Dixit, Anshuman; Silakari, Om</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4725-4746</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  PKC-θ is a serine/threonine specific protein kinase and its activation depends upon the concn. of diacylglycerol (DAG) and phospholipids (phosphatidylserine).  PKC-θ phosphorylates a variety of proteins that are known to be involved in the diverse cellular signaling pathways.  It is predominantly expressed in the T-cells and localized in the center of immunol. synapse upon T-cell receptor (TCR) and CD28 signaling.  Activation of PKC-θ leads to the activation of various transcription factors in the nuclei of T-cells, e.g. NF-κB, NFAT, c-Jun, c-Fos and AP-1 that further control the proliferation and differentiation of T-cells.  Defective T-cell activation in turn leads to the aberrant expression of apoptosis related proteins that cause the poor T-cell survival.  Researchers have found that T-cells deficient in PKC-θ exhibit reduced interleukin-2 (IL-2) prodn.  Apart from this role on IL-2 expression, it also plays crucial roles in the proliferation, differentiation and survival of the T-cells, which make it an attractive therapeutic target for a variety of immunol. and T-cell mediated diseases.  Hence, new mols. capable of modulating the expression or biol. activity of PKC-θ are being developed and tested for their potential as novel therapy for several T-cells mediated disease conditions such as multiple sclerosis, rheumatoid arthritis, asthma, inflammatory bowel disease and organ transplantation, etc.  In the present review, we tried to integrate the recent discoveries on PKC-θ including its pharmacol. and therapeutic potential, along with brief update on its inhibitor mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIN1yWUu7f4rVg90H21EOLACvtfcHk0lgKcmhqo120jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2nsr7O&md5=ccb3ca06a596973d1e9e3c262fce05e8</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.2174%2F138161212802651625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212802651625%26sid%3Dliteratum%253Aachs%26aulast%3DChand%26aufirst%3DS.%26aulast%3DMehta%26aufirst%3DN.%26aulast%3DSingh%2BBahia%26aufirst%3DM.%26aulast%3DDixit%26aufirst%3DA.%26aulast%3DSilakari%26aufirst%3DO.%26atitle%3DProtein%2520kinase%2520C-theta%2520inhibitors%253A%2520A%2520novel%2520therapy%2520for%2520inflammatory%2520disorders%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D4725%26epage%3D4746%26doi%3D10.2174%2F138161212802651625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanev, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Linden, O. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">KLIFS: a structural kinase-ligand interaction database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D365</span>– <span class="NLM_lpage">D371</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkv1082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26496949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D365-D371&author=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=O.+P.+J.+van+Lindenauthor=R.+Leursauthor=I.+J.+P.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+structural+kinase-ligand+interaction+database&doi=10.1093%2Fnar%2Fgkv1082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: a structural kinase-ligand interaction database</span></div><div class="casAuthors">Kooistra, Albert J.; Kanev, Georgi K.; van Linden, Oscar P. J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D365-D371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein kinases play a crucial role in cell signaling and are important drug targets in several therapeutic areas.  The KLIFS database contains detailed structural kinase-ligand interaction information derived from all (>2900) structures of catalytic domains of human and mouse protein kinases deposited in the Protein Data Bank in order to provide insights into the structural determinants of kinase-ligand binding and selectivity.  The kinase structures have been processed in a consistent manner by systematically analyzing the structural features and mol. interaction fingerprints (IFPs) of a predefined set of 85 binding site residues with bound ligands.  KLIFS has been completely rebuilt and extended (>65% more structures) since its first release as a data set, including: novel automated annotation methods for (i) the assessment of ligand-targeted subpockets and the anal. of (ii) DFG and (iii) αC-helix conformations; improved and automated protocols for (iv) the generation of sequence/structure alignments, (v) the curation of ligand atom and bond typing for accurate IFP anal. and (vi) weekly database updates.  KLIFS is now accessible via a website (http://klifs.vucompmedchem. nl) that provides a comprehensive visual presentation of different types of chem., biol. and structural chemogenomics data, and allows the user to easily access, compare, search and download the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocfCaKwwgOlbVg90H21EOLACvtfcHk0ljZV3djU4O1UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP&md5=bc093b02225987fe791a2d41738e4d99</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1082%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520structural%2520kinase-ligand%2520interaction%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD365%26epage%3DD371%26doi%3D10.1093%2Fnar%2Fgkv1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunnery, S.</span></span> <span> </span><span class="NLM_article-title">Human breast cancer cells contain elevated levels and activity of the protein kinase, PKR</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3086</span>– <span class="NLM_lpage">3094</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fsj.onc.1203632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10871861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkvVylurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=3086-3094&author=S.+H.+Kimauthor=A.+P.+Formanauthor=M.+B.+Mathewsauthor=S.+Gunnery&title=Human+breast+cancer+cells+contain+elevated+levels+and+activity+of+the+protein+kinase%2C+PKR&doi=10.1038%2Fsj.onc.1203632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Human breast cancer cells contain elevated levels and activity of the protein kinase, PKR</span></div><div class="casAuthors">Kim, Steve H.; Forman, Adam P.; Mathews, Michael B.; Gunnery, Shobha</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3086-3094</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PKR is a double-stranded (ds) RNA activated protein kinase whose expression is induced by interferon.  Activated PKR phosphorylates its cellular substrate, eIF2, an essential initiation factor of translation.  Prior evidence from a murine model system suggested that PKR may act as a tumor suppressor, but the evidence from human tumors is equivocal.  To study PKR function in human breast cancer, PKR activity was measured in mammary carcinoma cell lines and nontransformed mammary epithelial cell lines.  If PKR functioned as a tumor suppressor in this system, its activity would be higher in nontransformed cells than in carcinoma cells.  On the contrary, PKR autophosphorylation and the phosphorylation of its substrate, the α-subunit of eIF2, is 7-40-fold higher in lysates prepd. from breast carcinoma cell lines than in those from nontransformed epithelial cell lines.  Correspondingly, a larger proportion of eIF2α is present in a phosphorylated state in carcinoma cell lines than in nontransformed cell lines.  Protein synthesis is not inhibited by the high eIF2α phosphorylation in carcinoma cells, probably because they contain higher levels of eIF2B, the initiation factor that is inhibited by eIF2α phosphorylation.  The dramatically lower PKR activity in nontransformed cell lines is partially due to lower PKR protein levels (2-4-fold) as well as to the presence of a PKR inhibitor.  The nontransformed cells contain P58, a known cellular inhibitor of PKR that phys. interacts with PKR and may be responsible for the low PKR activity in these cells.  Taken together, these observations call into question the role of PKR as a tumor suppressor and suggest a pos. regulatory role of PKR in growth control of breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOok8IF4WDkrVg90H21EOLACvtfcHk0ljZV3djU4O1UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkvVylurs%253D&md5=5d897a1b807a1a25343970c494b194c0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203632%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DForman%26aufirst%3DA.%2BP.%26aulast%3DMathews%26aufirst%3DM.%2BB.%26aulast%3DGunnery%26aufirst%3DS.%26atitle%3DHuman%2520breast%2520cancer%2520cells%2520contain%2520elevated%2520levels%2520and%2520activity%2520of%2520the%2520protein%2520kinase%252C%2520PKR%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D3086%26epage%3D3094%26doi%3D10.1038%2Fsj.onc.1203632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delhem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabile, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajardo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podevin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abadie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremsdorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beretta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brechot, C.</span></span> <span> </span><span class="NLM_article-title">Activation of the interferon-inducible protein kinase PKR by hepatocellular carcinoma derived-hepatitis C virus core protein</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5836</span>– <span class="NLM_lpage">5845</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fsj.onc.1204744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=11593389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlSgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=5836-5845&author=N.+Delhemauthor=A.+Sabileauthor=R.+Gajardoauthor=P.+Podevinauthor=A.+Abadieauthor=M.+A.+Blatonauthor=D.+Kremsdorfauthor=L.+Berettaauthor=C.+Brechot&title=Activation+of+the+interferon-inducible+protein+kinase+PKR+by+hepatocellular+carcinoma+derived-hepatitis+C+virus+core+protein&doi=10.1038%2Fsj.onc.1204744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the interferon-inducible protein kinase PKR by hepatocellular carcinoma derived-hepatitis C virus core protein</span></div><div class="casAuthors">Delhem, Nadvia; Sabile, Abdelmajid; Gajardo, Rodrigo; Podevin, Philippe; Abadie, Annie; Blaton, Maria Agnes; Kremsdorf, Dina; Beretta, Laura; Brechot, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">5836-5845</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) is a major etiol. agent of chronic liver disease and hepatocellular carcinoma (HCC).  The authors demonstrate herewith that HCV core proteins encoded by sequences isolated from HCC tumor tissues, but not those derived from their non-tumor counterparts in the same liver, co-localize in vitro and in vivo and co-immunoppt. with PKR in hepatocytic Huh7 cells.  The authors show that this assocn. in fact augments the autophosphorylation of PKR and the phosphorylation of the translation initiation factor eIF2α, which are two markers of PKR activity.  The present study therefore identifies a novel model of virus-cell interactions whereby a viral protein, the HCV core, activates PKR activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHlIAsJKbGhLVg90H21EOLACvtfcHk0ljZV3djU4O1UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlSgtrc%253D&md5=15531a174ead64a0b70dc714b943431e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204744%26sid%3Dliteratum%253Aachs%26aulast%3DDelhem%26aufirst%3DN.%26aulast%3DSabile%26aufirst%3DA.%26aulast%3DGajardo%26aufirst%3DR.%26aulast%3DPodevin%26aufirst%3DP.%26aulast%3DAbadie%26aufirst%3DA.%26aulast%3DBlaton%26aufirst%3DM.%2BA.%26aulast%3DKremsdorf%26aufirst%3DD.%26aulast%3DBeretta%26aufirst%3DL.%26aulast%3DBrechot%26aufirst%3DC.%26atitle%3DActivation%2520of%2520the%2520interferon-inducible%2520protein%2520kinase%2520PKR%2520by%2520hepatocellular%2520carcinoma%2520derived-hepatitis%2520C%2520virus%2520core%2520protein%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D5836%26epage%3D5845%26doi%3D10.1038%2Fsj.onc.1204744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peel, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottrell, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellerby, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bredesen, D. E.</span></span> <span> </span><span class="NLM_article-title">Double-stranded RNA-dependent protein kinase, PKR, binds preferentially to Huntington’s disease (HD) transcripts and is activated in HD tissue</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1531</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1093/hmg/10.15.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fhmg%2F10.15.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=11468270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvFOitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2001&pages=1531-1538&author=A.+L.+Peelauthor=R.+V.+Raoauthor=B.+A.+Cottrellauthor=M.+R.+Haydenauthor=L.+M.+Ellerbyauthor=D.+E.+Bredesen&title=Double-stranded+RNA-dependent+protein+kinase%2C+PKR%2C+binds+preferentially+to+Huntington%E2%80%99s+disease+%28HD%29+transcripts+and+is+activated+in+HD+tissue&doi=10.1093%2Fhmg%2F10.15.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Double-stranded RNA-dependent protein kinase, PKR, binds preferentially to Huntington's disease (HD) transcripts and is activated in HD tissue</span></div><div class="casAuthors">Peel, Alyson L.; Rao, Ram V.; Cottrell, Barbara A.; Hayden, Michael R.; Ellerby, Lisa M.; Bredesen, Dale E.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1531-1538</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Fourteen neurol. diseases have been assocd. with the expansion of trinucleotide repeat regions.  These diseases have been categorized into those that give rise to the translation of toxic polyglutamine proteins and those that are untranslated.  Thus far, compelling evidence has not surfaced for the inclusion of a model in which a common mechanism may participate in the pathobiol. of both translated and untranslated trinucleotide diseases.  In these studies we show that a double-stranded RNA-binding protein, PKR, which has previously been linked to virally-induced and stress-mediated apoptosis, preferentially binds mutant huntingtin RNA transcripts immobilized on streptavidin columns that have been incubated with human brain exts.  These studies also show, by immunodetection in tissue slices, that PKR is present in its activated form in both human Huntington autopsy material and brain tissue derived from Huntington yeast artificial chromosome transgenic mice.  The increased immunolocalization of the activated kinase is more pronounced in areas most affected by the disease and, coupled with the RNA binding results, suggests a role for PKR activation in the disease process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonZ2uuXE1HwbVg90H21EOLACvtfcHk0lgILboW_hj-fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvFOitb0%253D&md5=80bcd3ad64f855e415bcbf5b8b42e2cd</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F10.15.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F10.15.1531%26sid%3Dliteratum%253Aachs%26aulast%3DPeel%26aufirst%3DA.%2BL.%26aulast%3DRao%26aufirst%3DR.%2BV.%26aulast%3DCottrell%26aufirst%3DB.%2BA.%26aulast%3DHayden%26aufirst%3DM.%2BR.%26aulast%3DEllerby%26aufirst%3DL.%2BM.%26aulast%3DBredesen%26aufirst%3DD.%2BE.%26atitle%3DDouble-stranded%2520RNA-dependent%2520protein%2520kinase%252C%2520PKR%252C%2520binds%2520preferentially%2520to%2520Huntington%25E2%2580%2599s%2520disease%2520%2528HD%2529%2520transcripts%2520and%2520is%2520activated%2520in%2520HD%2520tissue%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2001%26volume%3D10%26spage%3D1531%26epage%3D1538%26doi%3D10.1093%2Fhmg%2F10.15.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lgILboW_hj-fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggill, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhardt, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Punta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangrador-Vegas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salazar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, A.</span></span> <span> </span><span class="NLM_article-title">The PFAM protein families database: Towards a more sustainable future</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D279</span>– <span class="NLM_lpage">D285</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkv1344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26673716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D279-D285&author=R.+D.+Finnauthor=P.+Coggillauthor=R.+Y.+Eberhardtauthor=S.+R.+Eddyauthor=J.+Mistryauthor=A.+L.+Mitchellauthor=S.+C.+Potterauthor=M.+Puntaauthor=M.+Qureshiauthor=A.+Sangrador-Vegasauthor=G.+A.+Salazarauthor=J.+Tateauthor=A.+Bateman&title=The+PFAM+protein+families+database%3A+Towards+a+more+sustainable+future&doi=10.1093%2Fnar%2Fgkv1344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfam protein families database: towards a more sustainable future</span></div><div class="casAuthors">Finn, Robert D.; Coggill, Penelope; Eberhardt, Ruth Y.; Eddy, Sean R.; Mistry, Jaina; Mitchell, Alex L.; Potter, Simon C.; Punta, Marco; Qureshi, Matloob; Sangrador-Vegas, Amaia; Salazar, Gustavo A.; Tate, John; Bateman, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D279-D285</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In the last two years the Pfam database (http://pfam. xfam.org) has undergone a substantial reorganization to reduce the effort involved in making a release, thereby permitting more frequent releases.  Arguably the most significant of these changes is that Pfam is now primarily based on the UniProtKB ref. proteomes, with the counts of matched sequences and species reported on the website restricted to this smaller set.  Building families on ref. proteomes sequences brings greater stability, which decreases the amt. of manual curation required to maintain them.  It also reduces the no. of sequences displayed on the website, while still providing access to many important model organisms.  Matches to the full UniProtKB database are, however, still available and Pfam annotations for individual UniProtKB sequences can still be retrieved.  Some Pfam entries (1.6%) which have no matches to ref. proteomes remain; we are working with UniProt to see if sequences from them can be incorporated into ref. proteomes.  Pfam-B, the automatically-generated supplement to Pfam, has been removed.  The current release (Pfam 29.0) includes 16 295 entries and 559 clans.  The facility to view the relationship between families within a clan has been improved by the introduction of a new tool.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzdSs831JgtbVg90H21EOLACvtfcHk0lgILboW_hj-fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7nF&md5=8c52b94dbbb0adab77938456c64cfc49</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1344%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DR.%2BD.%26aulast%3DCoggill%26aufirst%3DP.%26aulast%3DEberhardt%26aufirst%3DR.%2BY.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DMistry%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DA.%2BL.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DPunta%26aufirst%3DM.%26aulast%3DQureshi%26aufirst%3DM.%26aulast%3DSangrador-Vegas%26aufirst%3DA.%26aulast%3DSalazar%26aufirst%3DG.%2BA.%26aulast%3DTate%26aufirst%3DJ.%26aulast%3DBateman%26aufirst%3DA.%26atitle%3DThe%2520PFAM%2520protein%2520families%2520database%253A%2520Towards%2520a%2520more%2520sustainable%2520future%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD279%26epage%3DD285%26doi%3D10.1093%2Fnar%2Fgkv1344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">The UniProt Consortium</span> <span> </span><span class="NLM_article-title">UniProt: The universal protein knowledgebase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The+UniProt+Consortium&title=UniProt%3A+The+universal+protein+knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%2520The%2520universal%2520protein%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">The Gene Ontology
Consortium</span> <span> </span><span class="NLM_article-title">Expansion of the
gene ontology knowledgebase and resources</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D331</span>– <span class="NLM_lpage">D338</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkw1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27899567" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D331-D338&author=The+Gene+Ontology%0AConsortium&title=Expansion+of+the%0Agene+ontology+knowledgebase+and+resources&doi=10.1093%2Fnar%2Fgkw1108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1108%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DExpansion%2520of%2520the%250Agene%2520ontology%2520knowledgebase%2520and%2520resources%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD331%26epage%3DD338%26doi%3D10.1093%2Fnar%2Fgkw1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battistuz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluhm, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iype, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marvin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.1107/s0907444902003451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1107%2Fs0907444902003451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12037327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=899-907&author=H.+M.+Bermanauthor=T.+Battistuzauthor=T.+N.+Bhatauthor=W.+F.+Bluhmauthor=P.+E.+Bourneauthor=K.+Burkhardtauthor=Z.+Fengauthor=G.+L.+Gillilandauthor=L.+Iypeauthor=S.+Jainauthor=P.+Faganauthor=J.+Marvinauthor=D.+Padillaauthor=V.+Ravichandranauthor=B.+Schneiderauthor=N.+Thankiauthor=H.+Weissigauthor=J.+D.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1107%2Fs0907444902003451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Battistuz, Tammy; Bhat, T. N.; Bluhm, Wolfgang F.; Bourne, Philip E.; Burkhardt, Kyle; Feng, Zukang; Gilliland, Gary L.; Iype, Lisa; Jain, Shri; Fagan, Phoebe; Marvin, Jessica; Padilla, David; Ravichandran, Veerasamy; Schneider, Bohdan; Thanki, Narmada; Weissig, Helge; Westbrook, John D.; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">6, No. 1</span>),
    <span class="NLM_cas:pages">899-907</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Protein Data Bank [PDB; Berman, Westbrook et al. (2000), Nucleic Acids Res. 28, 235-242; http://www.pdb.org/] is the single worldwide archive of primary structural data of biol. macromols.  Many secondary sources of information are derived from PDB data.  It is the starting point for studies in structural bioinformatics.  This article describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information and plans for the future development of the resource.  The reader should come away with an understanding of the scope of the PDB and what is provided by the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUFcMyys-PrVg90H21EOLACvtfcHk0lgfKW0SbnqsSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D&md5=d7594ad00ec8e5bacc413d38bceaa360</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2Fs0907444902003451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444902003451%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBattistuz%26aufirst%3DT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBluhm%26aufirst%3DW.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBurkhardt%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%2BL.%26aulast%3DIype%26aufirst%3DL.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DFagan%26aufirst%3DP.%26aulast%3DMarvin%26aufirst%3DJ.%26aulast%3DPadilla%26aufirst%3DD.%26aulast%3DRavichandran%26aufirst%3DV.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DThanki%26aufirst%3DN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D899%26epage%3D907%26doi%3D10.1107%2Fs0907444902003451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashford, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orengo, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sillitoe, I.</span></span> <span> </span><span class="NLM_article-title">CATH: An expanded resource to predict protein function through structure and sequence</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D289</span>– <span class="NLM_lpage">D295</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkw1098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27899584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D289-D295&author=N.+L.+Dawsonauthor=T.+E.+Lewisauthor=S.+Dasauthor=J.+G.+Leesauthor=D.+Leeauthor=P.+Ashfordauthor=C.+A.+Orengoauthor=I.+Sillitoe&title=CATH%3A+An+expanded+resource+to+predict+protein+function+through+structure+and+sequence&doi=10.1093%2Fnar%2Fgkw1098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">CATH: an expanded resource to predict protein function through structure and sequence</span></div><div class="casAuthors">Dawson, Natalie L.; Lewis, Tony E.; Das, Sayoni; Lees, Jonathan G.; Lee, David; Ashford, Paul; Orengo, Christine A.; Sillitoe, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D289-D295</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The latest version of the CATH-Gene3D protein structure classification database has recently been released (version 4.1, http://www.cathdb.info).  The resource comprises over 300 000 domain structures and over 53 million protein domains classified into 2737 homologous superfamilies, doubling the no. of predicted protein domains in the previous version.  The daily-updated CATH-B, which contains our very latest domain assignment data, provides putative classifications for over 100 000 addnl. protein domains.  This article describes developments to the CATH-Gene3D resource over the last two years since the publication in 2015, including: significant increases to our structural and sequence coverage; expansion of the functional families in CATH; building a support vector machine (SVM) to automatically assign domains to superfamilies; improved search facilities to return alignments of query sequences against multiple sequence alignments; the redesign of the web pages and download site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsFexQmzV7KbVg90H21EOLACvtfcHk0lgfKW0SbnqsSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhu7k%253D&md5=0f0181b39280024e816e8baa299859c1</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1098%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DN.%2BL.%26aulast%3DLewis%26aufirst%3DT.%2BE.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DLees%26aufirst%3DJ.%2BG.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DAshford%26aufirst%3DP.%26aulast%3DOrengo%26aufirst%3DC.%2BA.%26aulast%3DSillitoe%26aufirst%3DI.%26atitle%3DCATH%253A%2520An%2520expanded%2520resource%2520to%2520predict%2520protein%2520function%2520through%2520structure%2520and%2520sequence%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD289%26epage%3DD295%26doi%3D10.1093%2Fnar%2Fgkw1098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlingame, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolan, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Turning a protein kinase on or off from a single allosteric site via disulfide trapping</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">6056</span>– <span class="NLM_lpage">6061</span>, <span class="refDoi"> DOI: 10.1073/pnas.1102376108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.1102376108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=21430264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVymtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=6056-6061&author=J.+D.+Sadowskyauthor=M.+A.+Burlingameauthor=D.+W.+Wolanauthor=C.+L.+McClendonauthor=M.+P.+Jacobsonauthor=J.+A.+Wells&title=Turning+a+protein+kinase+on+or+off+from+a+single+allosteric+site+via+disulfide+trapping&doi=10.1073%2Fpnas.1102376108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Turning a protein kinase on or off from a single allosteric site via disulfide trapping</span></div><div class="casAuthors">Sadowsky, Jack D.; Burlingame, Mark A.; Wolan, Dennis W.; McClendon, Christopher L.; Jacobson, Matthew P.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6056-6061, S6056/1-S6056/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">There is significant interest in identifying and characterizing allosteric sites in enzymes such as protein kinases both for understanding allosteric mechanisms as well as for drug discovery.  Here, we apply a site-directed technol., disulfide trapping, to interrogate structurally and functionally how an allosteric site on the Ser/Thr kinase, 3-phosphoinositide-dependent kinase 1 (PDK1) - the PDK1-interacting-fragment (PIF) pocket - is engaged by an activating peptide motif on downstream substrate kinases (PIFtides) and by small mol. fragments.  By monitoring pairwise disulfide conjugation between PIFtide and PDK1 cysteine mutants, we defined the PIFtide binding orientation in the PIF pocket of PDK1 and assessed subtle relationships between PIFtide positioning and kinase activation.  We also discovered a variety of small mol. fragment disulfides (<300 Da) that could either activate or inhibit PDK1 by conjugation to the PIF pocket, thus displaying greater functional diversity than is displayed by PIFtides conjugated to the same sites.  Biochem. data and three crystal structures provided insight into the mechanism of action of the best fragment activators and inhibitors.  These studies show that disulfide trapping is useful for characterizing allosteric sites on kinases and that a single allosteric site on a protein kinase can be exploited for both activation and inhibition by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfPsWN5mhicLVg90H21EOLACvtfcHk0liRzE5xXdZXqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVymtb8%253D&md5=dab9d4429539ffc25153e9938a7d958c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102376108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102376108%26sid%3Dliteratum%253Aachs%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DBurlingame%26aufirst%3DM.%2BA.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DTurning%2520a%2520protein%2520kinase%2520on%2520or%2520off%2520from%2520a%2520single%2520allosteric%2520site%2520via%2520disulfide%2520trapping%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D6056%26epage%3D6061%26doi%3D10.1073%2Fpnas.1102376108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A.</span></span> <span> </span><span class="NLM_article-title">UCSF Small Molecule Discovery Center: Innovation, collaboration and chemical biology in the Bay area</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.2174/1386207317666140323133841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.2174%2F1386207317666140323133841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=24661212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1elsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2014&pages=333-342&author=M.+Arkinauthor=K.+Angauthor=S.+Chenauthor=J.+Daviesauthor=C.+Merronauthor=Y.+Tangauthor=C.+Wilsonauthor=A.+Renslo&title=UCSF+Small+Molecule+Discovery+Center%3A+Innovation%2C+collaboration+and+chemical+biology+in+the+Bay+area&doi=10.2174%2F1386207317666140323133841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Small Molecule Discovery Center: Innovation, Collaboration and Chemical Biology in the Bay Area</span></div><div class="casAuthors">Arkin, Michelle R.; Ang, Kenny K. H.; Chen, Steven; Davies, Julia; Merron, Connie; Tang, Yinyan; Wilson, Christopher G. M.; Renslo, Adam R.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">333-342</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The Small Mol. Discovery Center (SMDC) at the University of California, San Francisco, works collaboratively with the scientific community to solve challenging problems in chem. biol. and drug discovery.  The SMDC includes a high throughput screening facility, medicinal chem., and research labs focused on fundamental problems in biochem. and targeted drug delivery.  Here, we outline our HTS program and provide examples of chem. tools developed through SMDC collaborations.  We have an active research program in developing quant. cell-based screens for primary cells and whole organisms; here, we describe whole-organism screens to find drugs against parasites that cause neglected tropical diseases.  We are also very interested in target-based approaches for so-called "undruggable", protein classes and fragment-based lead discovery.  This expertise has led to several pharmaceutical collaborations; addnl., the SMDC works with start-up companies to enable their early-stage research.  The SMDC, located in the biotech-focused Mission Bay neighborhood in San Francisco, is a hub for innovative small-mol. discovery research at UCSF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFwj4Qi0lusrVg90H21EOLACvtfcHk0liRzE5xXdZXqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1elsr8%253D&md5=c0d76cd40afc12119d6a5b888bf7e229</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.2174%2F1386207317666140323133841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207317666140323133841%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%26aulast%3DAng%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DMerron%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DRenslo%26aufirst%3DA.%26atitle%3DUCSF%2520Small%2520Molecule%2520Discovery%2520Center%253A%2520Innovation%252C%2520collaboration%2520and%2520chemical%2520biology%2520in%2520the%2520Bay%2520area%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2014%26volume%3D17%26spage%3D333%26epage%3D342%26doi%3D10.2174%2F1386207317666140323133841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 18 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Wei-Ven Tee, Zhen Wah Tan, Keene Lee, Enrico Guarnera, <span class="NLM_string-name hlFld-ContribAuthor">Igor N. Berezovsky</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the Allosteric Territory of Protein Function. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (15)
                                     , 3763-3780. <a href="https://doi.org/10.1021/acs.jpcb.1c00540" title="DOI URL">https://doi.org/10.1021/acs.jpcb.1c00540</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.1c00540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.1c00540%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DExploring%252Bthe%252BAllosteric%252BTerritory%252Bof%252BProtein%252BFunction%26aulast%3DTee%26aufirst%3DWei-Ven%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D21012021%26date%3D29032021%26date%3D12042021%26volume%3D125%26issue%3D15%26spage%3D3763%26epage%3D3780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gennady M.  Verkhivker</span>. </span><span class="cited-content_cbyCitation_article-title">Making the invisible visible: Toward structural characterization of allosteric states, interaction networks, and allosteric regulatory mechanisms in protein kinases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2021,</strong> <em>71 </em>, 71-78. <a href="https://doi.org/10.1016/j.sbi.2021.06.002" title="DOI URL">https://doi.org/10.1016/j.sbi.2021.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2021.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2021.06.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DMaking%252Bthe%252Binvisible%252Bvisible%25253A%252BToward%2525C2%2525A0structural%252Bcharacterization%252Bof%252Ballosteric%252Bstates%25252C%252Binteraction%252Bnetworks%25252C%252Band%252Ballosteric%252Bregulatory%252Bmechanisms%252Bin%252Bprotein%252Bkinases%26aulast%3DVerkhivker%26aufirst%3DGennady%2BM.%26date%3D2021%26volume%3D71%26spage%3D71%26epage%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominika Natalia  Wojewska</span>, <span class="hlFld-ContribAuthor ">Arjan  Kortholt</span>. </span><span class="cited-content_cbyCitation_article-title">LRRK2 Targeting Strategies as Potential Treatment of Parkinson’s Disease. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2021,</strong> <em>11 </em>
                                    (8)
                                     , 1101. <a href="https://doi.org/10.3390/biom11081101" title="DOI URL">https://doi.org/10.3390/biom11081101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom11081101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom11081101%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DLRRK2%252BTargeting%252BStrategies%252Bas%252BPotential%252BTreatment%252Bof%252BParkinson%2525E2%252580%252599s%252BDisease%26aulast%3DWojewska%26aufirst%3DDominika%2BNatalia%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D8%26spage%3D1101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paweł  Łukasik</span>, <span class="hlFld-ContribAuthor ">Irena  Baranowska-Bosiacka</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kulczycka</span>, <span class="hlFld-ContribAuthor ">Izabela  Gutowska</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Cyclin-Dependent Kinases: Types and Their Mechanism of Action. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (6)
                                     , 2806. <a href="https://doi.org/10.3390/ijms22062806" title="DOI URL">https://doi.org/10.3390/ijms22062806</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22062806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22062806%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DInhibitors%252Bof%252BCyclin-Dependent%252BKinases%25253A%252BTypes%252Band%252BTheir%252BMechanism%252Bof%252BAction%26aulast%3D%25C5%2581ukasik%26aufirst%3DPawe%25C5%2582%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D6%26spage%3D2806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Koji  Umezawa</span>, <span class="hlFld-ContribAuthor ">Isao  Kii</span>. </span><span class="cited-content_cbyCitation_article-title">Druggable Transient Pockets in Protein Kinases. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (3)
                                     , 651. <a href="https://doi.org/10.3390/molecules26030651" title="DOI URL">https://doi.org/10.3390/molecules26030651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26030651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26030651%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDruggable%252BTransient%252BPockets%252Bin%252BProtein%252BKinases%26aulast%3DUmezawa%26aufirst%3DKoji%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D3%26spage%3D651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arne Christian  Rufer</span>. </span><span class="cited-content_cbyCitation_article-title">Drug discovery for enzymes. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>34 </em><a href="https://doi.org/10.1016/j.drudis.2021.01.006" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.01.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.01.006%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DDrug%252Bdiscovery%252Bfor%252Benzymes%26aulast%3DRufer%26aufirst%3DArne%2BChristian%26date%3D2021%26volume%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan  Egbert</span>, <span class="hlFld-ContribAuthor ">Kathryn A.  Porter</span>, <span class="hlFld-ContribAuthor ">Usman  Ghani</span>, <span class="hlFld-ContribAuthor ">Sergei  Kotelnikov</span>, <span class="hlFld-ContribAuthor ">Thu  Nguyen</span>, <span class="hlFld-ContribAuthor ">Ryota  Ashizawa</span>, <span class="hlFld-ContribAuthor ">Dima  Kozakov</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Conservation of binding properties in protein models. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2021,</strong> <em>19 </em>, 2549-2566. <a href="https://doi.org/10.1016/j.csbj.2021.04.048" title="DOI URL">https://doi.org/10.1016/j.csbj.2021.04.048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2021.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2021.04.048%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DConservation%252Bof%252Bbinding%252Bproperties%252Bin%252Bprotein%252Bmodels%26aulast%3DEgbert%26aufirst%3DMegan%26date%3D2021%26volume%3D19%26spage%3D2549%26epage%3D2566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arramshetti  Venkanna</span>, <span class="hlFld-ContribAuthor ">Lalita  Subedi</span>, <span class="hlFld-ContribAuthor ">Mahesh K.  Teli</span>, <span class="hlFld-ContribAuthor ">Prema Dhorma  Lama</span>, <span class="hlFld-ContribAuthor ">Bhargav Gupta  Nangunuri</span>, <span class="hlFld-ContribAuthor ">Sang-Yoon  Lee</span>, <span class="hlFld-ContribAuthor ">Sun Yeou  Kim</span>, <span class="hlFld-ContribAuthor ">Mi-hyun  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Positioning of an unprecedented spiro[5.5]undeca ring system into kinase inhibitor space. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-78158-9" title="DOI URL">https://doi.org/10.1038/s41598-020-78158-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-78158-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-78158-9%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DPositioning%252Bof%252Ban%252Bunprecedented%252Bspiro%25255B5.5%25255Dundeca%252Bring%252Bsystem%252Binto%252Bkinase%252Binhibitor%252Bspace%26aulast%3DVenkanna%26aufirst%3DArramshetti%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Carofiglio</span>, <span class="hlFld-ContribAuthor ">Daniela  Trisciuzzi</span>, <span class="hlFld-ContribAuthor ">Nicola  Gambacorta</span>, <span class="hlFld-ContribAuthor ">Francesco  Leonetti</span>, <span class="hlFld-ContribAuthor ">Angela  Stefanachi</span>, <span class="hlFld-ContribAuthor ">Orazio  Nicolotti</span>. </span><span class="cited-content_cbyCitation_article-title">Bcr-Abl Allosteric Inhibitors: Where We Are and Where We Are Going to. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (18)
                                     , 4210. <a href="https://doi.org/10.3390/molecules25184210" title="DOI URL">https://doi.org/10.3390/molecules25184210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25184210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25184210%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DBcr-Abl%252BAllosteric%252BInhibitors%25253A%252BWhere%252BWe%252BAre%252Band%252BWhere%252BWe%252BAre%252BGoing%252Bto%26aulast%3DCarofiglio%26aufirst%3DFrancesca%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D18%26spage%3D4210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyun  Lu</span>, <span class="hlFld-ContribAuthor ">Jeff B.  Smaill</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Allosterische Kinaseinhibitoren – Erwartungen und Chancen. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (33)
                                     , 13868-13881. <a href="https://doi.org/10.1002/ange.201914525" title="DOI URL">https://doi.org/10.1002/ange.201914525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201914525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201914525%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DAllosterische%252BKinaseinhibitoren%252B%2525E2%252580%252593%252BErwartungen%252Bund%252BChancen%26aulast%3DLu%26aufirst%3DXiaoyun%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D33%26spage%3D13868%26epage%3D13881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyun  Lu</span>, <span class="hlFld-ContribAuthor ">Jeff B.  Smaill</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">New Promise and Opportunities for Allosteric Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (33)
                                     , 13764-13776. <a href="https://doi.org/10.1002/anie.201914525" title="DOI URL">https://doi.org/10.1002/anie.201914525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201914525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201914525%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DNew%252BPromise%252Band%252BOpportunities%252Bfor%252BAllosteric%252BKinase%252BInhibitors%26aulast%3DLu%26aufirst%3DXiaoyun%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D33%26spage%3D13764%26epage%3D13776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sang Bae  Lee</span>, <span class="hlFld-ContribAuthor ">Aram  Ko</span>, <span class="hlFld-ContribAuthor ">Young Taek  Oh</span>, <span class="hlFld-ContribAuthor ">Peiguo  Shi</span>, <span class="hlFld-ContribAuthor ">Fulvio  D’Angelo</span>, <span class="hlFld-ContribAuthor ">Brulinda  Frangaj</span>, <span class="hlFld-ContribAuthor ">Antonius  Koller</span>, <span class="hlFld-ContribAuthor ">Emily I.  Chen</span>, <span class="hlFld-ContribAuthor ">Timothy  Cardozo</span>, <span class="hlFld-ContribAuthor ">Antonio  Iavarone</span>, <span class="hlFld-ContribAuthor ">Anna  Lasorella</span>. </span><span class="cited-content_cbyCitation_article-title">Proline Hydroxylation Primes Protein Kinases for Autophosphorylation and Activation. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2020,</strong> <em>79 </em>
                                    (3)
                                     , 376-389.e8. <a href="https://doi.org/10.1016/j.molcel.2020.06.021" title="DOI URL">https://doi.org/10.1016/j.molcel.2020.06.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2020.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2020.06.021%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DProline%252BHydroxylation%252BPrimes%252BProtein%252BKinases%252Bfor%252BAutophosphorylation%252Band%252BActivation%26aulast%3DLee%26aufirst%3DSang%2BBae%26date%3D2020%26volume%3D79%26issue%3D3%26spage%3D376%26epage%3D389.e8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jintong  Du</span>, <span class="hlFld-ContribAuthor ">Jing  Guo</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Zhihong  Li</span>, <span class="hlFld-ContribAuthor ">Guan  Wang</span>, <span class="hlFld-ContribAuthor ">Jianbing  Wu</span>, <span class="hlFld-ContribAuthor ">Zhen  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Fang</span>, <span class="hlFld-ContribAuthor ">Xuben  Hou</span>, <span class="hlFld-ContribAuthor ">Zhangjian  Huang</span>, <span class="hlFld-ContribAuthor ">Guobo  Li</span>, <span class="hlFld-ContribAuthor ">Xiaoyun  Lu</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>, <span class="hlFld-ContribAuthor ">Li  Rao</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>, <span class="hlFld-ContribAuthor ">Xiaojin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yihua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">New techniques and strategies in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2020,</strong> <em>31 </em>
                                    (7)
                                     , 1695-1708. <a href="https://doi.org/10.1016/j.cclet.2020.03.028" title="DOI URL">https://doi.org/10.1016/j.cclet.2020.03.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2020.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2020.03.028%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DNew%252Btechniques%252Band%252Bstrategies%252Bin%252Bdrug%252Bdiscovery%26aulast%3DDu%26aufirst%3DJintong%26date%3D2020%26volume%3D31%26issue%3D7%26spage%3D1695%26epage%3D1708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Enrico  Guarnera</span>, <span class="hlFld-ContribAuthor ">Igor N  Berezovsky</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric drugs and mutations: chances, challenges, and necessity. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2020,</strong> <em>62 </em>, 149-157. <a href="https://doi.org/10.1016/j.sbi.2020.01.010" title="DOI URL">https://doi.org/10.1016/j.sbi.2020.01.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2020.01.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DAllosteric%252Bdrugs%252Band%252Bmutations%25253A%252Bchances%25252C%252Bchallenges%25252C%252Band%252Bnecessity%26aulast%3DGuarnera%26aufirst%3DEnrico%26date%3D2020%26volume%3D62%26spage%3D149%26epage%3D157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maurice  Michel</span>, <span class="hlFld-ContribAuthor ">Evert J.  Homan</span>, <span class="hlFld-ContribAuthor ">Elisée  Wiita</span>, <span class="hlFld-ContribAuthor ">Kia  Pedersen</span>, <span class="hlFld-ContribAuthor ">Ingrid  Almlöf</span>, <span class="hlFld-ContribAuthor ">Anna-Lena  Gustavsson</span>, <span class="hlFld-ContribAuthor ">Thomas  Lundbäck</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Druggability Assessment of the NUDIX Hydrolase Protein Family as a Workflow for Target Prioritization. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.00443" title="DOI URL">https://doi.org/10.3389/fchem.2020.00443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.00443%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BDruggability%252BAssessment%252Bof%252Bthe%252BNUDIX%252BHydrolase%252BProtein%252BFamily%252Bas%252Ba%252BWorkflow%252Bfor%252BTarget%252BPrioritization%26aulast%3DMichel%26aufirst%3DMaurice%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junhong  Qin</span>, <span class="hlFld-ContribAuthor ">Xiaofei  Shen</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Da  Jia</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric inhibitors of the STAT3 signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>190 </em>, 112122. <a href="https://doi.org/10.1016/j.ejmech.2020.112122" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112122%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252Binhibitors%252Bof%252Bthe%252BSTAT3%252Bsignaling%252Bpathway%26aulast%3DQin%26aufirst%3DJunhong%26date%3D2020%26volume%3D190%26spage%3D112122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zarko  Gagic</span>, <span class="hlFld-ContribAuthor ">Dusan  Ruzic</span>, <span class="hlFld-ContribAuthor ">Nemanja  Djokovic</span>, <span class="hlFld-ContribAuthor ">Teodora  Djikic</span>, <span class="hlFld-ContribAuthor ">Katarina  Nikolic</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Methods for Design of Kinase Inhibitors as Anticancer Drugs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00873" title="DOI URL">https://doi.org/10.3389/fchem.2019.00873</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00873%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BMethods%252Bfor%252BDesign%252Bof%252BKinase%252BInhibitors%252Bas%252BAnticancer%252BDrugs%26aulast%3DGagic%26aufirst%3DZarko%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramon  Martinez</span>, <span class="hlFld-ContribAuthor ">Amy  Defnet</span>, <span class="hlFld-ContribAuthor ">Paul  Shapiro</span>. </span><span class="cited-content_cbyCitation_article-title">Avoiding or Co-Opting ATP Inhibition: Overview of Type III, IV, V, and VI Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 29-59. <a href="https://doi.org/10.1007/978-3-030-48283-1_3" title="DOI URL">https://doi.org/10.1007/978-3-030-48283-1_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-48283-1_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-48283-1_3%26sid%3Dliteratum%253Aachs%26atitle%3DAvoiding%252Bor%252BCo-Opting%252BATP%252BInhibition%25253A%252BOverview%252Bof%252BType%252BIII%25252C%252BIV%25252C%252BV%25252C%252Band%252BVI%252BKinase%252BInhibitors%26aulast%3DMartinez%26aufirst%3DRamon%26date%3D2020%26date%3D2020%26spage%3D29%26epage%3D59%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DNext%252BGeneration%252BKinase%252BInhibitors%26aulast%3DShapiro%26aufirst%3DPaul%26date%3D2020%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Positions of all allosteric sites discussed in this paper. ATP is represented as yellow sticks, the N-terminal domain is shown in green, and the C-terminal domain is shown in gray. From top to bottom: MPP (cyan), PIF (purple), MT3 (orange), DFG (red), DRS (blue), PMP (pink), PDIG (dark green), AAS (light yellow), CMP (brown), DEF (teal), LBP (magenta), and EDI (olive).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Number of kinases found to have consensus sites at each putative allosteric site. ATP is not an allosteric site but is included here as a reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Mapping results for unliganded structures (except MPP) of kinases associated with each allosteric site, with superimposed ligands from bound structures shown as yellow sticks. Each probe cluster in the consensus cluster is represented by a single probe shown in line representation. Structural, ligand, and mapping details for each site are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. FTMap finds a strong hot spot in the PIF pocket of CDK2. Only the N-lobe of CDK2 is shown, and each probe cluster placed by FTMap is represented by a single probe at its center (gray lines). Residues interacting with the probes are shown in magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds bound to CDK2 as identified by tethering. (A) Two most potent compounds. (B) Preliminary potency measurements for compounds <b>1</b> and <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Radioactive peptide phosphorylation assay results demonstrating the inhibitory effects of compounds <b>1</b> and <b>2</b> upon CDK2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/medium/jm-2019-00089c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. FTMap results for apo structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UIU">3UIU</a>_B of PKR. (A) Strongest hot spot, 0(19), shown in magenta, located at the AAS site. The second strongest hot spot, 1(18), shown in coral, is located at the PDIG site. The PDIG site also accommodates the secondary consensus sites 7(5) (in green) and 8(4) (in blue). For orientation, we superimposed the Chk1 inhibitor (shown in yellow) from the Chk1 structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS">3JVS</a>. (B) Close-up of the PDIG site with PKR in surface representation to show that at this site, PKR has a well-defined pocket that could accommodate small compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00089/20190719/images/large/jm-2019-00089c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00089&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63825" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63825" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 69 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lgGgAVWgI-SEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span> </span><span class="NLM_article-title">Human Protein Kinase Knowledgebase</span>. <a href="http://www.kinasenet.ca" class="extLink">http://www.kinasenet.ca</a> (accessed May 13, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Human+Protein+Kinase+Knowledgebase.+http%3A%2F%2Fwww.kinasenet.ca+%28accessed+May+13%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DHuman%2520Protein%2520Kinase%2520Knowledgebase%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manning, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudarsanam, S.</span></span> <span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1912</span>– <span class="NLM_lpage">1934</span>, <span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0ljXCm6t32jNvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brylinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skolnick, J.</span></span> <span> </span><span class="NLM_article-title">Comprehensive structural and functional characterization of the human kinome by protein structure modeling and ligand virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1839</span>– <span class="NLM_lpage">1854</span>, <span class="refDoi"> DOI: 10.1021/ci100235n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100235n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1839-1854&author=M.+Brylinskiauthor=J.+Skolnick&title=Comprehensive+structural+and+functional+characterization+of+the+human+kinome+by+protein+structure+modeling+and+ligand+virtual+screening&doi=10.1021%2Fci100235n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening</span></div><div class="casAuthors">Brylinski, Michal; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1839-1854</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The growing interest in the identification of kinase inhibitors, promising therapeutics in the treatment of many diseases, has created a demand for the structural characterization of the entire human kinome.  At the outset of the drug development process, the lead-finding stage, approaches that enrich the screening library with bioactive compds. are needed.  Here, protein structure based methods can play an important role, but despite structural genomics efforts, it is unlikely that the three-dimensional structures of the entire kinome will be available soon.  Therefore, at the proteome level, structure-based approaches must rely on predicted models, with a key issue being their utility in virtual ligand screening.  In this study, we employ the recently developed FINDSITE/Q-Dock ligand homol. modeling approach, which is well-suited for proteome-scale applications using predicted structures, to provide extensive structural and functional characterization of the human kinome.  Specifically, we construct structure models for the human kinome; these are subsequently subject to virtual screening against a library of more than 2 million compds.  To rank the compds., we employ a hierarchical approach that combines ligand- and structure-based filters.  Modeling accuracy is carefully validated using available exptl. data with particularly encouraging results found for the ability to identify, without prior knowledge, specific kinase inhibitors.  More generally, the modeling procedure results in a large no. of predicted mol. interactions between kinases and small ligands that should be of practical use in the development of novel inhibitors.  The data set is freely available to the academic community via a user-friendly Web interface at http://cssb.biol.gatech.edu/kinomelhm/ as well as at the ZINC Web site (http://zinc.docking.org/applications/2010Apr/Brylinski-2010.tar.gz).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsNAiYM1dJuLVg90H21EOLACvtfcHk0ljXCm6t32jNvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL&md5=4dd283d41804d7f05138f5acf02ad404</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fci100235n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100235n%26sid%3Dliteratum%253Aachs%26aulast%3DBrylinski%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DComprehensive%2520structural%2520and%2520functional%2520characterization%2520of%2520the%2520human%2520kinome%2520by%2520protein%2520structure%2520modeling%2520and%2520ligand%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1839%26epage%3D1854%26doi%3D10.1021%2Fci100235n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Blue
Ridge Institute for Medical Research</span>. <span> </span><span class="NLM_article-title">FDA-approved protein kinase inhibitors</span>. <a href="http://www.brimr.org/PKI/PKIs.htm" class="extLink">http://www.brimr.org/PKI/PKIs.htm</a> (accessed May 13, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Blue%0ARidge+Institute+for+Medical+Research&title=FDA-approved+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DFDA-approved%2520protein%2520kinase%2520inhibitors%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span> <span> </span><span class="NLM_article-title">Structure-based virtual screening approaches in kinase-directed drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2235</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.2174/1568026617666170224121313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.2174%2F1568026617666170224121313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28240180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Smt7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=2235-2259&author=D.+Bajuszauthor=G.+G.+Ferenczyauthor=G.+M.+Keseru&title=Structure-based+virtual+screening+approaches+in+kinase-directed+drug+discovery&doi=10.2174%2F1568026617666170224121313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Virtual Screening Approaches in Kinase-directed Drug Discovery</span></div><div class="casAuthors">Bajusz, David; Ferenczy, Gyorgy G.; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2235-2259</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Protein kinases are one of the most targeted protein families in current drug discovery pipelines.  They are implicated in many oncol., inflammatory, CNS-related and other clin. indications.  Virtual screening is a computational technique with a diverse set of available tools that has been shown many times to provide novel starting points for kinase-directed drug discovery.  This review starts with a concise overview of the function, structural features and inhibitory mechanisms of protein kinases.  In addn. to briefly reviewing the practical aspects of structure-based virtual screenings, we discuss several case studies to illustrate the state of the art in the virtual screening for type I, type II, allosteric (type III-V) and covalent (type VI) kinase inhibitors.  With this review, we strive to provide a summary of the latest advances in the structure-based discovery of novel kinase inhibitors, as well as a practical tool to anyone who wishes to embark on such an endeavor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoANpN3hgHTtLVg90H21EOLACvtfcHk0ljXCm6t32jNvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Smt7nF&md5=5b85e22ae4fbf7b25cbb70fc14d147a3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170224121313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170224121313%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DStructure-based%2520virtual%2520screening%2520approaches%2520in%2520kinase-directed%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D2235%26epage%3D2259%26doi%3D10.2174%2F1568026617666170224121313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Strategies for the selective regulation of kinases with allosteric modulators: Exploiting exclusive structural features</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1021/cb300663j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300663j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCls7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=58-70&author=Z.+Fangauthor=C.+Gr%C3%BCtterauthor=D.+Rauh&title=Strategies+for+the+selective+regulation+of+kinases+with+allosteric+modulators%3A+Exploiting+exclusive+structural+features&doi=10.1021%2Fcb300663j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the Selective Regulation of Kinases with Allosteric Modulators: Exploiting Exclusive Structural Features</span></div><div class="casAuthors">Fang, Zhizhou; Gruetter, Christian; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-70</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The modulation of kinase function has become an important goal in modern drug discovery and chem. biol. research.  In cancer-targeted therapies, kinase inhibitors have been experiencing an upsurge, which can be measured by the increasing no. of kinase inhibitors approved by the FDA in recent years.  However, lack of efficacy, limited selectivity, and the emergence of acquired drug resistance still represent major bottlenecks in the clinic and challenge inhibitor development.  Most known kinase inhibitors target the active kinase and are ATP competitive.  A second class of small org. mols., which address remote sites of the kinase and stabilize enzymically inactive conformations, is rapidly moving to the forefront of kinase inhibitor research.  Such allosteric modulators bind to sites that are less conserved across the kinome and only accessible upon conformational changes.  These mols. are therefore thought to provide various advantages such as higher selectivity and extended drug target residence times.  This review highlights various strategies that have been developed to utilizing exclusive structural features of kinases and thereby modulating their activity allosterically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSjxxztmczXrVg90H21EOLACvtfcHk0lg-cmbGQBm9tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCls7bN&md5=50864096bf66229ff74482a9b56847d6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb300663j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300663j%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStrategies%2520for%2520the%2520selective%2520regulation%2520of%2520kinases%2520with%2520allosteric%2520modulators%253A%2520Exploiting%2520exclusive%2520structural%2520features%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D58%26epage%3D70%26doi%3D10.1021%2Fcb300663j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vijayan, R. S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong-Ly, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbrack, R. L.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. M.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of the DFG-out kinase motif and biochemical profiling of structurally validated type II inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">466</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1021/jm501603h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501603h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=466-479&author=R.+S.+K.+Vijayanauthor=P.+Heauthor=V.+Modiauthor=K.+C.+Duong-Lyauthor=H.+Maauthor=J.+R.+Petersonauthor=R.+L.+Dunbrackauthor=R.+M.+Levy&title=Conformational+analysis+of+the+DFG-out+kinase+motif+and+biochemical+profiling+of+structurally+validated+type+II+inhibitors&doi=10.1021%2Fjm501603h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of the DFG-Out Kinase Motif and Biochemical Profiling of Structurally Validated Type II Inhibitors</span></div><div class="casAuthors">Vijayan, R. S. K.; He, Peng; Modi, Vivek; Duong-Ly, Krisna C.; Ma, Haiching; Peterson, Jeffrey R.; Dunbrack, Roland L.; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">466-479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural coverage of the human kinome has been steadily increasing over time.  The structures provide valuable insights into the mol. basis of kinase function and also provide a foundation for understanding the mechanisms of kinase inhibitors.  There are a large no. of kinase structures in the PDB for which the Asp and Phe of the DFG motif on the activation loop swap positions, resulting in the formation of a new allosteric pocket.  The authors refer to these structures as "classical DFG-out" conformations in order to distinguish them from conformations that have also been referred to as DFG-out in the literature but that do not have a fully formed allosteric pocket.  The authors have completed a structural anal. of almost 200 small mol. inhibitors bound to classical DFG-out conformations; the authors find that they are recognized by both type I and type II inhibitors.  In contrast, the authors find that nonclassical DFG-out conformations strongly select against type II inhibitors because these structures have not formed a large enough allosteric pocket to accommodate this type of binding mode.  In the course of this study the authors discovered that the no. of structurally validated type II inhibitors that can be found in the PDB and that are also represented in publicly available biochem. profiling studies of kinase inhibitors is very small.  The authors have obtained new profiling results for several addnl. structurally validated type II inhibitors identified through the conformational anal.  Although the available profiling data for type II inhibitors is still much smaller than for type I inhibitors, a comparison of the two data sets supports the conclusion that type II inhibitors are more selective than type I.  The authors comment on the possible contribution of the DFG-in to DFG-out conformational reorganization to the selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb9E5JKOmDZ7Vg90H21EOLACvtfcHk0lg-cmbGQBm9tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP&md5=6d729997f242d582a4adff9463fd585c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm501603h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501603h%26sid%3Dliteratum%253Aachs%26aulast%3DVijayan%26aufirst%3DR.%2BS.%2BK.%26aulast%3DHe%26aufirst%3DP.%26aulast%3DModi%26aufirst%3DV.%26aulast%3DDuong-Ly%26aufirst%3DK.%2BC.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DConformational%2520analysis%2520of%2520the%2520DFG-out%2520kinase%2520motif%2520and%2520biochemical%2520profiling%2520of%2520structurally%2520validated%2520type%2520II%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D466%26epage%3D479%26doi%3D10.1021%2Fjm501603h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Allosteric MEK1/2 inhibitors including cobimetanib and trametinib in the treatment of cutaneous melanomas</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2016.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.phrs.2016.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27956260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSnsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=20-31&author=R.+Roskoski&title=Allosteric+MEK1%2F2+inhibitors+including+cobimetanib+and+trametinib+in+the+treatment+of+cutaneous+melanomas&doi=10.1016%2Fj.phrs.2016.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric MEK1/2 inhibitors including cobimetanib and trametinib in the treatment of cutaneous melanomas</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20-31</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Ras-Raf-MEK-ERK (Map kinase) cellular pathway is a highly conserved eukaryotic signaling module that transduces extracellular signals from growth factors and cytokines into intracellular regulatory events that are involved in cell growth and proliferation or the contrary pathway of cell differentiation.  Dysregulation of this pathway occurs in more than one-third of all malignancies, a process that has fostered the development of targeted Map kinase pathway inhibitors.  Cutaneous melanomas, which arise from skin melanocytes, are the most aggressive form of skin cancer.  Mutations that activate the Map kinase pathway occur in more than 90% of these melanomas.  This has led to the development of the combination of dabrafenib and trametinib or vemurafenib and cobimetanib for the treatment of BRAF V600E mutant melanomas.  Dabrafenib and vemurafenib target V600E/K BRAF mutants while trametinib and cobimetanib target MEK1/2.  The latter two agents bind to MEK1/2 at a site that is adjacent to, but sep. from, the ATP-binding site and are therefore classified as type III allosteric protein kinase inhibitors.  These agents form a hydrogen bond with a conserved β3-lysine and they make numerous hydrophobic contacts with residues within the αC-helix, the β5 strand, and within the activation segment, regions of the protein kinase domain that exhibit greater diversity than those found within the ATP-binding site.  One advantage of such allosteric inhibitors is that they do not have to compete with millimolar concns. of cellular ATP, which most FDA-approved small mol. competitive inhibitors such as imatinib must do.  Owing to the wide spread activation of this pathway in numerous neoplasms, trametinib and cobimetinib are being studied in combination with other targeted and cytotoxic drugs in a variety of clin. situations.  Except for BRAF and NRAS mutations, there are no other biomarkers correlated with treatment responses following MEK1/2 inhibition and the discovery of such biomarkers would represent an important therapeutic breakthrough.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJSWK7CmuK_7Vg90H21EOLACvtfcHk0lg-cmbGQBm9tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSnsbnO&md5=db7220ba32cfe520a99ec59e4ba33c6f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2016.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2016.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DAllosteric%2520MEK1%252F2%2520inhibitors%2520including%2520cobimetanib%2520and%2520trametinib%2520in%2520the%2520treatment%2520of%2520cutaneous%2520melanomas%26jtitle%3DPharmacol.%2520Res.%26date%3D2017%26volume%3D117%26spage%3D20%26epage%3D31%26doi%3D10.1016%2Fj.phrs.2016.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohnuud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mottarella, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">755</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0ljIiupcxr4NAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljIiupcxr4NAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">New frontiers in druggability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9063</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9063-9088&author=D.+Kozakovauthor=D.+R.+Hallauthor=R.+L.+Napoleonauthor=C.+Yuehauthor=A.+Whittyauthor=S.+Vajda&title=New+frontiers+in+druggability&doi=10.1021%2Facs.jmedchem.5b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New Frontiers in Druggability</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Napoleon, Raeanne L.; Yueh, Christine; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9063-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A powerful early approach to evaluating the druggability of proteins involved detg. the hit rate in NMR-based screening of a library of small compds.  Here, we show that a computational analog of this method, based on mapping proteins using small mols. as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree.  We apply the method to a large set of proteins.  The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compds. beyond those resembling validated targets and conventionally druglike ligands.  In particular, the method identifies targets that, while not druggable by druglike compds., may become druggable using compd. classes such as macrocycles or other large mols. beyond the rule-of-five limit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWoMwMpuM2bVg90H21EOLACvtfcHk0ljIiupcxr4NAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF&md5=77fd532a1f0129d096eb7a1fbfb1d422</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNapoleon%26aufirst%3DR.%2BL.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DNew%2520frontiers%2520in%2520druggability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9063%26epage%3D9088%26doi%3D10.1021%2Facs.jmedchem.5b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foda, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span> <span> </span><span class="NLM_article-title">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">5939</span>, <span class="refDoi"> DOI: 10.1038/ncomms6939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fncomms6939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25600932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5939&author=Z.+H.+Fodaauthor=Y.+Shanauthor=E.+T.+Kimauthor=D.+E.+Shawauthor=M.+A.+Seeliger&title=A+dynamically+coupled+allosteric+network+underlies+binding+cooperativity+in+Src+kinase&doi=10.1038%2Fncomms6939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span></div><div class="casAuthors">Foda, Zachariah H.; Shan, Yibing; Kim, Eric T.; Shaw, David E.; Seeliger, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5939</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein tyrosine kinases are attractive drug targets because many human diseases are assocd. with the deregulation of kinase activity.  However, how the catalytic kinase domain integrates different signals and switches from an active to an inactive conformation remains incompletely understood.  Here we identify an allosteric network of dynamically coupled amino acids in Src kinase that connects regulatory sites to the ATP- and substrate-binding sites.  Surprisingly, reactants (ATP and peptide substrates) bind with neg. cooperativity to Src kinase while products (ADP and phosphopeptide) bind with pos. cooperativity.  We confirm the mol. details of the signal relay through the allosteric network by biochem. studies.  Expts. on two addnl. protein tyrosine kinases indicate that the allosteric network may be largely conserved among these enzymes.  Our work provides new insights into the regulation of protein tyrosine kinases and establishes a potential conduit by which resistance mutations to ATP-competitive kinase inhibitors can affect their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGfRZJun_T1rVg90H21EOLACvtfcHk0lgsugankoZI-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ&md5=cc24df9830ccb1525f66cc554795965c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fncomms6939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6939%26sid%3Dliteratum%253Aachs%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26atitle%3DA%2520dynamically%2520coupled%2520allosteric%2520network%2520underlies%2520binding%2520cooperativity%2520in%2520Src%2520kinase%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D5939%26doi%3D10.1038%2Fncomms6939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornev, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span> <span> </span><span class="NLM_article-title">Allostery through the computational microscope: cAMP activation of a canonical signalling domain</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7588</span>, <span class="refDoi"> DOI: 10.1038/ncomms8588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fncomms8588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26145448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FgvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7588&author=R.+D.+Malmstromauthor=A.+P.+Kornevauthor=S.+S.+Taylorauthor=R.+E.+Amaro&title=Allostery+through+the+computational+microscope%3A+cAMP+activation+of+a+canonical+signalling+domain&doi=10.1038%2Fncomms8588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery through the computational microscope: cAMP activation of a canonical signalling domain</span></div><div class="casAuthors">Malmstrom Robert D; Amaro Rommie E; Kornev Alexandr P; Taylor Susan S</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7588</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ligand-induced protein allostery plays a central role in modulating cellular signalling pathways.  Here using the conserved cyclic nucleotide-binding domain of protein kinase A's (PKA) regulatory subunit as a prototype signalling unit, we combine long-timescale, all-atom molecular dynamics simulations with Markov state models to elucidate the conformational ensembles of PKA's cyclic nucleotide-binding domain A for the cAMP-free (apo) and cAMP-bound states.  We find that both systems exhibit shallow free-energy landscapes that link functional states through multiple transition pathways.  This observation suggests conformational selection as the general mechanism of allostery in this canonical signalling domain.  Further, we expose the propagation of the allosteric signal through key structural motifs in the cyclic nucleotide-binding domain and explore the role of kinetics in its function.  Our approach integrates disparate lines of experimental data into one cohesive framework to understand structure, dynamics and function in complex biological systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNy5_7QVOqG9yzdcP07UmufW6udTcc2eZLhjypw1XaNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FgvFWrsA%253D%253D&md5=fdc93fe9e6e29b9dc69e1e007b08233a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms8588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8588%26sid%3Dliteratum%253Aachs%26aulast%3DMalmstrom%26aufirst%3DR.%2BD.%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26atitle%3DAllostery%2520through%2520the%2520computational%2520microscope%253A%2520cAMP%2520activation%2520of%2520a%2520canonical%2520signalling%2520domain%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7588%26doi%3D10.1038%2Fncomms8588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J. E.</span></span> <span> </span><span class="NLM_article-title">Predicting protein dynamics and allostery using multi-protein atomic distance constraints</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1016/j.str.2017.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.str.2017.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28190781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=546-558&author=J.+G.+Greenerauthor=I.+Filippisauthor=M.+J.+E.+Sternberg&title=Predicting+protein+dynamics+and+allostery+using+multi-protein+atomic+distance+constraints&doi=10.1016%2Fj.str.2017.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Protein Dynamics and Allostery Using Multi-Protein Atomic Distance Constraints</span></div><div class="casAuthors">Greener, Joe G.; Filippis, Ioannis; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">546-558</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The related concepts of protein dynamics, conformational ensembles and allostery are often difficult to study with mol. dynamics (MD) due to the timescales involved.  We present ExProSE (Exploration of Protein Structural Ensembles), a distance geometry-based method that generates an ensemble of protein structures from two input structures.  ExProSE provides a unified framework for the exploration of protein structure and dynamics in a fast and accessible way.  Using a dataset of apo/holo pairs it is shown that existing coarse-grained methods often cannot span large conformational changes.  For T4-lysozyme, ExProSE is able to generate ensembles that are more native-like than tCONCOORD and NMSim, and comparable with targeted MD.  By adding addnl. constraints representing potential modulators, ExProSE can predict allosteric sites.  ExProSE ranks an allosteric pocket first or second for 27 out of 58 allosteric proteins, which is similar and complementary to existing methods.  The ExProSE source code is freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF34-q1BCDrVg90H21EOLACvtfcHk0lgsugankoZI-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D&md5=2d2ea675b3b29ff52f9915ddecabe032</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DFilippis%26aufirst%3DI.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%2BE.%26atitle%3DPredicting%2520protein%2520dynamics%2520and%2520allostery%2520using%2520multi-protein%2520atomic%2520distance%2520constraints%26jtitle%3DStructure%26date%3D2017%26volume%3D25%26spage%3D546%26epage%3D558%26doi%3D10.1016%2Fj.str.2017.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caliman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Mapping the allosteric sites of the A2A adenosine receptor</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1111/cbdd.13053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1111%2Fcbdd.13053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28639411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFynsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2018&pages=5-16&author=A.+D.+Calimanauthor=Y.+Miaoauthor=J.+A.+McCammon&title=Mapping+the+allosteric+sites+of+the+A2A+adenosine+receptor&doi=10.1111%2Fcbdd.13053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the allosteric sites of the A2A adenosine receptor</span></div><div class="casAuthors">Caliman, Alisha D.; Miao, Yinglong; McCammon, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-16</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The A2A adenosine receptor (A2AAR) is a G protein-coupled receptor that is pharmacol. targeted for the treatment of inflammation, sepsis, cancer, neurodegeneration, and Parkinson's disease.  Recently, we applied long-time-scale mol. dynamics simulations on 2 ligand-free receptor conformations, starting from the agonist-bound (PDB ID: 3QAK) and antagonist-bound (PDB ID: 3EML) x-ray structures.  This anal. revealed 4 distinct conformers of the A2AAR: active, intermediate 1, intermediate 2, and inactive conformers.  Here, we applied the fragment-based mapping algorithm (FTMap) on these receptor conformations to uncover 5 non-orthosteric sites on the A2AAR.  Two sites that were identified in the active conformation were located in the intracellular region of the transmembrane helixes (TM) 3/TM4 and the G protein-binding site in the intracellular region between TM2/TM3/TM6/TM7.  Three sites are identified in the intermediate 1 and intermediate 2 conformations, annexing a site in the lipid interface of TM5/TM6.  Five sites were identified in the inactive conformation, comprising a site in the intracellular region of TM1/TM7 and in the extracellular region of TM3/TM4 of the A2AAR.  We postulate that these sites on the A2AAR be screened for allosteric modulators for the treatment of inflammatory and neurol. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkjjgfTQct9bVg90H21EOLACvtfcHk0lgsugankoZI-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFynsbnF&md5=0a7d738c2f9906177b0dc34ac55a9d98</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.13053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.13053%26sid%3Dliteratum%253Aachs%26aulast%3DCaliman%26aufirst%3DA.%2BD.%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DMapping%2520the%2520allosteric%2520sites%2520of%2520the%2520A2A%2520adenosine%2520receptor%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2018%26volume%3D91%26spage%3D5%26epage%3D16%26doi%3D10.1111%2Fcbdd.13053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Lessons from hot spot analysis for fragment-based drug discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.tips.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26538314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=724-736&author=D.+R.+Hallauthor=D.+Kozakovauthor=A.+Whittyauthor=S.+Vajda&title=Lessons+from+hot+spot+analysis+for+fragment-based+drug+discovery&doi=10.1016%2Fj.tips.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from hot spot analysis for fragment-based drug discovery</span></div><div class="casAuthors">Hall, David R.; Kozakov, Dima; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">724-736</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Anal. of binding energy hot spots at protein surfaces can provide crucial insights into the prospects for successful application of fragment-based drug discovery (FBDD), and whether a fragment hit can be advanced into a high-affinity, drug-like ligand.  The key factor is the strength of the top ranking hot spot, and how well a given fragment complements it.  We show that published data are sufficient to provide a sophisticated and quant. understanding of how hot spots derive from a protein 3D structure, and how their strength, no., and spatial arrangement govern the potential for a surface site to bind to fragment-sized and larger ligands.  This improved understanding provides important guidance for the effective application of FBDD in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQmPulmhPx7Vg90H21EOLACvtfcHk0lg_U2tLP3Ho9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF&md5=3e682f28482d162590826f8f2e321b96</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLessons%2520from%2520hot%2520spot%2520analysis%2520for%2520fragment-based%2520drug%2520discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D724%26epage%3D736%26doi%3D10.1016%2Fj.tips.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rettenmaier, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">A small-molecule mimic of a peptide docking motif inhibits the protein kinase PDK1</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">18590</span>– <span class="NLM_lpage">18595</span>, <span class="refDoi"> DOI: 10.1073/pnas.1415365112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.1415365112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25518860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18590-18595&author=T.+J.+Rettenmaierauthor=J.+D.+Sadowskyauthor=N.+D.+Thomsenauthor=S.+C.+Chenauthor=A.+K.+Doakauthor=M.+R.+Arkinauthor=J.+A.+Wells&title=A+small-molecule+mimic+of+a+peptide+docking+motif+inhibits+the+protein+kinase+PDK1&doi=10.1073%2Fpnas.1415365112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule mimic of a peptide docking motif inhibits the protein kinase PDK1</span></div><div class="casAuthors">Rettenmaier, T. Justin; Sadowsky, Jack D.; Thomsen, Nathan D.; Chen, Steven C.; Doak, Allison K.; Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">18590-18595</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">There is great interest in developing selective protein kinase inhibitors by targeting allosteric sites, but these sites often involve protein-protein or protein-peptide interfaces that are very challenging to target with small mols.  Here we present a systematic approach to targeting a functionally conserved allosteric site on the protein kinase PDK1 called the PDK1-interacting fragment (PIF)tide-binding site, or PIF pocket.  More than two dozen prosurvival and progrowth kinases dock a conserved peptide tail into this binding site, which recruits them to PDK1 to become activated.  Using a site-directed chem. screen, we identified and chem. optimized ligand-efficient, selective, and cell-penetrant small mols. (mol. wt. ∼380 Da) that compete with the peptide docking motif for binding to PDK1.  We solved the first high-resoln. structure of a peptide docking motif (PIFtide) bound to PDK1 and mapped binding energy hot spots using mutational anal.  We then solved structures of PDK1 bound to the allosteric small mols., which revealed a binding mode that remarkably mimics three of five hot-spot residues in PIFtide.  These allosteric small mols. are substrate-selective PDK1 inhibitors when used as single agents, but when combined with an ATP-competitive inhibitor, they completely suppress the activation of the downstream kinases.  This work provides a promising new scaffold for the development of high-affinity PIF pocket ligands, which may be used to enhance the anticancer activity of existing PDK1 inhibitors.  Moreover, our results provide further impetus for exploring the helix αC patches of other protein kinases as potential therapeutic targets even though they involve protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeMoENY8xXbVg90H21EOLACvtfcHk0lg_U2tLP3Ho9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ&md5=4a1d88395865677d5a9c8d3dee58eca2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1415365112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1415365112%26sid%3Dliteratum%253Aachs%26aulast%3DRettenmaier%26aufirst%3DT.%2BJ.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DThomsen%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520small-molecule%2520mimic%2520of%2520a%2520peptide%2520docking%2520motif%2520inhibits%2520the%2520protein%2520kinase%2520PDK1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D18590%26epage%3D18595%26doi%3D10.1073%2Fpnas.1415365112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Exploration of type II binding mode: A privileged approach for kinase inhibitor focused drug discovery?</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+A+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lg_U2tLP3Ho9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520A%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The design, synthesis, and evaluation of 8 hybrid DFG-out allosteric kinase inhibitors: A structural analysis of the binding interactions of Gleevec, Nexavar, and BIRB-796</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5748</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bmc.2010.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20621496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlGhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=5738-5748&author=J.+Dietrichauthor=C.+Hulmeauthor=L.+H.+Hurley&title=The+design%2C+synthesis%2C+and+evaluation+of+8+hybrid+DFG-out+allosteric+kinase+inhibitors%3A+A+structural+analysis+of+the+binding+interactions+of+Gleevec%2C+Nexavar%2C+and+BIRB-796&doi=10.1016%2Fj.bmc.2010.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The design, synthesis, and evaluation of 8 hybrid DFG-out allosteric kinase inhibitors: A structural analysis of the binding interactions of Gleevec, Nexavar, and BIRB-796</span></div><div class="casAuthors">Dietrich, Justin; Hulme, Christopher; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5738-5748</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The majority of kinase inhibitors developed to date are competitive inhibitors that target the ATP binding site; however, recent crystal structures of Gleevec (imatinib mesylate, STI571, PDB: 1IEP), Nexavar (Sorafenib tosylate, BAY 43-9006, PDB: 1UWJ), and BIRB-796 (PDB: 1KV2) have revealed a secondary binding site adjacent to the ATP binding site known as the DFG-out allosteric binding site.  The recent successes of Gleevec and Nexavar for the treatment of chronic myeloid leukemia and renal cell carcinoma has generated great interest in the development of other kinase inhibitors that target this secondary binding site.  Here, we present a structural comparison of the important and similar interactions necessary for Gleevec, Nexavar, and BIRB-796 to bind to their resp. DFG-out allosteric binding pockets and the selectivity of each with respect to c-Abl, B-Raf, and p38α.  A structural anal. of their selectivity profiles has been generated from the synthesis and evaluation of 8 addnl. DFG-out allosteric inhibitors that were developed directly from fragments of these successful scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuTXCEZCwd0bVg90H21EOLACvtfcHk0ljnZRCfcnN7Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlGhu7s%253D&md5=430e5d3c95213441b17030a931f6e2ad</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DDietrich%26aufirst%3DJ.%26aulast%3DHulme%26aufirst%3DC.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520design%252C%2520synthesis%252C%2520and%2520evaluation%2520of%25208%2520hybrid%2520DFG-out%2520allosteric%2520kinase%2520inhibitors%253A%2520A%2520structural%2520analysis%2520of%2520the%2520binding%2520interactions%2520of%2520Gleevec%252C%2520Nexavar%252C%2520and%2520BIRB-796%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D5738%26epage%3D5748%26doi%3D10.1016%2Fj.bmc.2010.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aay, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blazey, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowles, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussenius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Defina, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubenko, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engst, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koltun, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lougheed, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manalo, J.-C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuss, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peto, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S.</span></span> <span> </span><span class="NLM_article-title">Novel carboxamide-based allosteric MEK inhibitors: Discovery and optimization efforts toward XL518 (GDC-0973)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">416</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1021/ml300049d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300049d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1eit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=416-421&author=K.+D.+Riceauthor=N.+Aayauthor=N.+K.+Anandauthor=C.+M.+Blazeyauthor=O.+J.+Bowlesauthor=J.+Busseniusauthor=S.+Costanzoauthor=J.+K.+Curtisauthor=S.+C.+Definaauthor=L.+Dubenkoauthor=S.+Engstauthor=A.+A.+Joshiauthor=A.+R.+Kennedyauthor=A.+I.+Kimauthor=E.+S.+Koltunauthor=J.+C.+Lougheedauthor=J.-C.+L.+Manaloauthor=J.-F.+Martiniauthor=J.+M.+Nussauthor=C.+J.+Petoauthor=T.+H.+Tsangauthor=P.+Yuauthor=S.+Johnston&title=Novel+carboxamide-based+allosteric+MEK+inhibitors%3A+Discovery+and+optimization+efforts+toward+XL518+%28GDC-0973%29&doi=10.1021%2Fml300049d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Carboxamide-Based Allosteric MEK Inhibitors: Discovery and Optimization Efforts toward XL518 (GDC-0973)</span></div><div class="casAuthors">Rice, Kenneth D.; Aay, Naing; Anand, Neel K.; Blazey, Charles M.; Bowles, Owen J.; Bussenius, Joerg; Costanzo, Simona; Curtis, Jeffry K.; Defina, Steven C.; Dubenko, Larisa; Engst, Stefan; Joshi, Anagha A.; Kennedy, Abigail R.; Kim, Angie I.; Koltun, Elena S.; Lougheed, Julie C.; Manalo, Jean-Claire L.; Martini, Jean-Francois; Nuss, John M.; Peto, Csaba J.; Tsang, Tsze H.; Yu, Peiwen; Johnston, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">416-421</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ERK/MAP kinase cascade is a key mechanism subject to dysregulation in cancer and is constitutively activated or highly upregulated in many tumor types.  Mutations assocd. with upstream pathway components RAS and Raf occur frequently and contribute to the oncogenic phenotype through activation of MEK and then ERK.  Inhibitors of MEK have been shown to effectively block upregulated ERK/MAPK signaling in a range of cancer cell lines and have further demonstrated early evidence of efficacy in the clinic for the treatment of cancer.  Guided by structural insight, a strategy aimed at the identification of an optimal diphenylamine-based MEK inhibitor with an improved metab. and safety profile vs. PD-0325901 led to the discovery of development candidate 1-({3,4-difluoro-2-[(2-fluoro-4-iodophenyl)amino]phenyl}carbonyl)-3-[(2S)-piperidin-2-yl]azetidin-3-ol (XL518, GDC-0973) (I).  I exhibits robust in vitro and in vivo potency and efficacy in preclin. models with sustained duration of action and is currently in early stage clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOx-Upr00b8LVg90H21EOLACvtfcHk0ljnZRCfcnN7Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1eit7k%253D&md5=04aa82afd24efe83a3b888140781bfc6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fml300049d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300049d%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DK.%2BD.%26aulast%3DAay%26aufirst%3DN.%26aulast%3DAnand%26aufirst%3DN.%2BK.%26aulast%3DBlazey%26aufirst%3DC.%2BM.%26aulast%3DBowles%26aufirst%3DO.%2BJ.%26aulast%3DBussenius%26aufirst%3DJ.%26aulast%3DCostanzo%26aufirst%3DS.%26aulast%3DCurtis%26aufirst%3DJ.%2BK.%26aulast%3DDefina%26aufirst%3DS.%2BC.%26aulast%3DDubenko%26aufirst%3DL.%26aulast%3DEngst%26aufirst%3DS.%26aulast%3DJoshi%26aufirst%3DA.%2BA.%26aulast%3DKennedy%26aufirst%3DA.%2BR.%26aulast%3DKim%26aufirst%3DA.%2BI.%26aulast%3DKoltun%26aufirst%3DE.%2BS.%26aulast%3DLougheed%26aufirst%3DJ.%2BC.%26aulast%3DManalo%26aufirst%3DJ.-C.%2BL.%26aulast%3DMartini%26aufirst%3DJ.-F.%26aulast%3DNuss%26aufirst%3DJ.%2BM.%26aulast%3DPeto%26aufirst%3DC.%2BJ.%26aulast%3DTsang%26aufirst%3DT.%2BH.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DJohnston%26aufirst%3DS.%26atitle%3DNovel%2520carboxamide-based%2520allosteric%2520MEK%2520inhibitors%253A%2520Discovery%2520and%2520optimization%2520efforts%2520toward%2520XL518%2520%2528GDC-0973%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D416%26epage%3D421%26doi%3D10.1021%2Fml300049d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juarez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palakurthi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelais, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michellys, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epple, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsilje, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">Overcoming EGFR(T790M) and EGFR(c797s) resistance with mutant-selective allosteric inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature17960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature17960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27251290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=129-132&author=Y.+Jiaauthor=C.-H.+Yunauthor=E.+Parkauthor=D.+Ercanauthor=M.+Manuiaauthor=J.+Juarezauthor=C.+Xuauthor=K.+Rheeauthor=T.+Chenauthor=H.+Zhangauthor=S.+Palakurthiauthor=J.+Jangauthor=G.+Lelaisauthor=M.+DiDonatoauthor=B.+Bursulayaauthor=P.-Y.+Michellysauthor=R.+Eppleauthor=T.+H.+Marsiljeauthor=M.+McNeillauthor=W.+Luauthor=J.+Harrisauthor=S.+Benderauthor=K.-K.+Wongauthor=P.+A.+J%C3%A4nneauthor=M.+J.+Eck&title=Overcoming+EGFR%28T790M%29+and+EGFR%28c797s%29+resistance+with+mutant-selective+allosteric+inhibitors&doi=10.1038%2Fnature17960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span></div><div class="casAuthors">Jia, Yong; Yun, Cai-Hong; Park, Eunyoung; Ercan, Dalia; Manuia, Mari; Juarez, Jose; Xu, Chunxiao; Rhee, Kevin; Chen, Ting; Zhang, Haikuo; Palakurthi, Sangeetha; Jang, Jaebong; Lelais, Gerald; DiDonato, Michael; Bursulaya, Badry; Michellys, Pierre-Yves; Epple, Robert; Marsilje, Thomas H.; McNeill, Matthew; Lu, Wenshuo; Harris, Jennifer; Bender, Steven; Wong, Kwok-Kin; Janne, Pasi A.; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7605</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR)-directed tyrosine kinase inhibitors (TKIs) gefitinib, erlotinib and afatinib are approved treatments for non-small cell lung cancers harbouring activating mutations in the EGFR kinase, but resistance arises rapidly, most frequently owing to the secondary T790M mutation within the ATP site of the receptor.  Recently developed mutant-selective irreversible inhibitors are highly active against the T790M mutant, but their efficacy can be compromised by acquired mutation of C797, the cysteine residue with which they form a key covalent bond.  All current EGFR TKIs target the ATP-site of the kinase, highlighting the need for therapeutic agents with alternative mechanisms of action.  Here we describe the rational discovery of EAI045, an allosteric inhibitor that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.  The crystal structure shows that the compd. binds an allosteric site created by the displacement of the regulatory C-helix in an inactive conformation of the kinase.  The compd. inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochem. assays.  However, as a single agent it is not effective in blocking EGFR-driven proliferation in cells owing to differential potency on the two subunits of the dimeric receptor, which interact in an asym. manner in the active state.  We observe marked synergy of EAI045 with cetuximab, an antibody therapeutic that blocks EGFR dimerization, rendering the kinase uniformly susceptible to the allosteric agent.  EAI045 in combination with cetuximab is effective in mouse models of lung cancer driven by EGFR(L858R/T790M) and by EGFR(L858R/T790M/C797S), a mutant that is resistant to all currently available EGFR TKIs.  More generally, our findings illustrate the utility of purposefully targeting allosteric sites to obtain mutant-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQkr363w1dMLVg90H21EOLACvtfcHk0ljnZRCfcnN7Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D&md5=f4c2d2362eb900c64bad8550b9da0872</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature17960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17960%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%26aulast%3DYun%26aufirst%3DC.-H.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DManuia%26aufirst%3DM.%26aulast%3DJuarez%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DRhee%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DPalakurthi%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DLelais%26aufirst%3DG.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMichellys%26aufirst%3DP.-Y.%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DMarsilje%26aufirst%3DT.%2BH.%26aulast%3DMcNeill%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DOvercoming%2520EGFR%2528T790M%2529%2520and%2520EGFR%2528c797s%2529%2520resistance%2520with%2520mutant-selective%2520allosteric%2520inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D129%26epage%3D132%26doi%3D10.1038%2Fnature17960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hindie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garcia, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idrissova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeuzem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, T. J.
D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzari, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span> <span> </span><span class="NLM_article-title">Structure and allosteric effects of low-molecular-weight activators on the protein kinase PDK1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">758</span>– <span class="NLM_lpage">764</span>, <span class="refDoi"> DOI: 10.1038/nchembio.208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnchembio.208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=19718043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=758-764&author=V.+Hindieauthor=A.+Strobaauthor=H.+Zhangauthor=L.+A.+Lopez-Garciaauthor=L.+Idrissovaauthor=S.+Zeuzemauthor=D.+Hirschbergauthor=F.+Schaefferauthor=T.+J.%0AD.+J%C3%B8rgensenauthor=M.+Engelauthor=P.+M.+Alzariauthor=R.+M.+Biondi&title=Structure+and+allosteric+effects+of+low-molecular-weight+activators+on+the+protein+kinase+PDK1&doi=10.1038%2Fnchembio.208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and allosteric effects of low-molecular-weight activators on the protein kinase PDK1</span></div><div class="casAuthors">Hindie, Valerie; Stroba, Adriana; Zhang, Hua; Lopez-Garcia, Laura A.; Idrissova, Leila; Zeuzem, Stefan; Hirschberg, Daniel; Schaeffer, Francis; Jorgensen, Thomas J. D.; Engel, Matthias; Alzari, Pedro M.; Biondi, Ricardo M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">758-764</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein phosphorylation transduces a large set of intracellular signals.  One mechanism by which phosphorylation mediates signal transduction is by prompting conformational changes in the target protein or interacting proteins.  Previous work described an allosteric site mediating phosphorylation-dependent activation of AGC kinases.  AGC kinase PDK1 is activated by the docking of a phosphorylated motif from substrates.  Here, the authors present the crystallog. of PDK1 bound to a rationally developed low-mol.-wt. activator and describe the conformational changes induced by small compds. in the crystal and in soln. using a fluorescence-based assay and deuterium exchange expts.  The results indicate that the binding of the compd. produces local changes at the target site, the PIF-binding pocket, and also allosteric changes at the ATP-binding site and the activation loop.  Altogether, the authors present mol. details of the allosteric changes induced by small compds. that trigger the activation of PDK1 through mimicry of phosphorylation-dependent conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBilNGbn-krbVg90H21EOLACvtfcHk0lg760hIBlwhqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F&md5=c9afd03c1d2c954fbcf04709570fdb76</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.208%26sid%3Dliteratum%253Aachs%26aulast%3DHindie%26aufirst%3DV.%26aulast%3DStroba%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLopez-Garcia%26aufirst%3DL.%2BA.%26aulast%3DIdrissova%26aufirst%3DL.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DHirschberg%26aufirst%3DD.%26aulast%3DSchaeffer%26aufirst%3DF.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DT.%2BJ.%2BD.%26aulast%3DEngel%26aufirst%3DM.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26atitle%3DStructure%2520and%2520allosteric%2520effects%2520of%2520low-molecular-weight%2520activators%2520on%2520the%2520protein%2520kinase%2520PDK1%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D758%26epage%3D764%26doi%3D10.1038%2Fnchembio.208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bobkova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blume-Jensen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrup, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunapuli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariv, I.</span></span> <span> </span><span class="NLM_article-title">Discovery of PDK1 kinase inhibitors with a novel mechanism of action by ultrahigh throughput screening</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">18838</span>– <span class="NLM_lpage">18846</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.089946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1074%2Fjbc.m109.089946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20385558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvFaqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=18838-18846&author=E.+V.+Bobkovaauthor=M.+J.+Weberauthor=Z.+Xuauthor=Y.-L.+Zhangauthor=J.+Jungauthor=P.+Blume-Jensenauthor=A.+Northrupauthor=P.+Kunapuliauthor=J.+N.+Andersenauthor=I.+Kariv&title=Discovery+of+PDK1+kinase+inhibitors+with+a+novel+mechanism+of+action+by+ultrahigh+throughput+screening&doi=10.1074%2Fjbc.m109.089946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of PDK1 Kinase Inhibitors with a Novel Mechanism of Action by Ultrahigh Throughput Screening</span></div><div class="casAuthors">Bobkova, Ekaterina V.; Weber, Michael J.; Xu, Zangwei; Zhang, Yan-Ling; Jung, Joon; Blume-Jensen, Peter; Northrup, Alan; Kunapuli, Priya; Andersen, Jannik N.; Kariv, Ilona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">18838-18846</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The phosphoinositide 3-kinase/AKT signaling pathway plays a key role in cancer cell growth, survival, and angiogenesis.  Phosphoinositide-dependent protein kinase-1 (PDK1) acts at a focal point in this pathway immediately downstream of phosphoinositide 3-kinase and PTEN, where it phosphorylates numerous AGC kinases.  The PDK1 kinase domain has at least three ligand-binding sites: the ATP-binding pocket, the peptide substrate-binding site, and a groove in the N-terminal lobe that binds the C-terminal hydrophobic motif of its kinase substrates.  Based on the unique PDK1 substrate recognition system, ultrahigh throughput TR-FRET and Alphascreen screening assays were developed using a biotinylated version of the PDK1-tide substrate contg. the activation loop of AKT fused to a pseudo-activated hydrophobic motif peptide.  Using full-length PDK1, Km values were detd. as 5.6 μM for ATP and 40 nM for the fusion peptide, revealing 50-fold higher affinity compared with the classical AKT(Thr-308)-tide.  Kinetic and biophys. studies confirmed the PDK1 catalytic mechanism as a rapid equil. random bireactant reaction.  Following an ultrahigh throughput screen of a large library, 2,000 compds. were selected from the reconfirmed hits by computational anal. with a focus on novel scaffolds.  ATP-competitive hits were deconvoluted by dose-response studies at 1x and 10x Km concns. of ATP, and specificity of binding was assessed in thermal shift assay.  Inhibition studies using fusion PDK1-tide1 substrate vs. AKT(Thr-308)-tide and kinase selectivity profiling revealed a novel selective alkaloid scaffold that evidently binds to the PDK1-interacting fragment pocket.  Mol. modeling suggests a structural paradigm for the design of inhibitory vs. activating allosteric ligands of PDK1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtz2tgwFgbXrVg90H21EOLACvtfcHk0lg760hIBlwhqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvFaqtrk%253D&md5=9fc38810a123771bd4bccade64a7a528</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m109.089946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m109.089946%26sid%3Dliteratum%253Aachs%26aulast%3DBobkova%26aufirst%3DE.%2BV.%26aulast%3DWeber%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DJung%26aufirst%3DJ.%26aulast%3DBlume-Jensen%26aufirst%3DP.%26aulast%3DNorthrup%26aufirst%3DA.%26aulast%3DKunapuli%26aufirst%3DP.%26aulast%3DAndersen%26aufirst%3DJ.%2BN.%26aulast%3DKariv%26aufirst%3DI.%26atitle%3DDiscovery%2520of%2520PDK1%2520kinase%2520inhibitors%2520with%2520a%2520novel%2520mechanism%2520of%2520action%2520by%2520ultrahigh%2520throughput%2520screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D18838%26epage%3D18846%26doi%3D10.1074%2Fjbc.m109.089946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheffzek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarkson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the autoinhibition of c-ABL tyrosine kinase</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(03)00194-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fs0092-8674%2803%2900194-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12654251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2003&pages=859-871&author=B.+Nagarauthor=O.+Hantschelauthor=M.+A.+Youngauthor=K.+Scheffzekauthor=D.+Veachauthor=W.+Bornmannauthor=B.+Clarksonauthor=G.+Superti-Furgaauthor=J.+Kuriyan&title=Structural+basis+for+the+autoinhibition+of+c-ABL+tyrosine+kinase&doi=10.1016%2Fs0092-8674%2803%2900194-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the autoinhibition of c-Abl tyrosine kinase</span></div><div class="casAuthors">Nagar, Bhushan; Hantschel, Oliver; Young, Matthew A.; Scheffzek, Klaus; Veach, Darren; Bornmann, William; Clarkson, Bayard; Superti-Furga, Giulio; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-871</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">C-Abl is normally regulated by an autoinhibitory mechanism, the disruption of which leads to chronic myelogenous leukemia.  The details of this mechanism have been elusive because c-Abl lacks a phosphotyrosine residue that triggers the assembly of the autoinhibited form of the closely related Src kinases by internally engaging the SH2 domain.  Crystal structures of c-Abl show that the N-terminal myristoyl modification of c-Abl 1b binds to the kinase domain and induces conformational changes that allow the SH2 and SH3 domains to dock onto it.  Autoinhibited c-Abl forms an assembly that is strikingly similar to that of inactive Src kinases but with specific differences that explain the differential ability of the drug STI-571/Gleevec/imatinib (STI-571) to inhibit the catalytic activity of Abl, but not that of c-Src.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaks_bu7ZlC7Vg90H21EOLACvtfcHk0lg760hIBlwhqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D&md5=19245c92b1e36779e2024bc071aa723b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fs0092-8674%2803%2900194-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0092-8674%252803%252900194-6%26sid%3Dliteratum%253Aachs%26aulast%3DNagar%26aufirst%3DB.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DYoung%26aufirst%3DM.%2BA.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DVeach%26aufirst%3DD.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520the%2520autoinhibition%2520of%2520c-ABL%2520tyrosine%2520kinase%26jtitle%3DCell%26date%3D2003%26volume%3D112%26spage%3D859%26epage%3D871%26doi%3D10.1016%2Fs0092-8674%2803%2900194-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woessner, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W.</span></span> <span> </span><span class="NLM_article-title">Development of an effective therapy for chronic Myelogenous Leukemia</span>. <i>Cancer J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1097/ppo.0b013e318237e5b7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1097%2Fppo.0b013e318237e5b7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=22157291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XitVOjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=477-486&author=D.+W.+Woessnerauthor=C.+S.+Limauthor=M.+W.+Deininger&title=Development+of+an+effective+therapy+for+chronic+Myelogenous+Leukemia&doi=10.1097%2Fppo.0b013e318237e5b7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an effective therapy for chronic myelogenous leukemia</span></div><div class="casAuthors">Woessner, David W.; Lim, Carol S.; Deininger, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Journal (Hagerstown, MD, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">477-486</span>CODEN:
                <span class="NLM_cas:coden">CAJOCB</span>;
        ISSN:<span class="NLM_cas:issn">1528-9117</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  Targeted small-mol. drugs have revolutionized treatment of chronic myeloid leukemia (CML) during the last decade.  These agents interrupt a constitutively active BCR-ABL, the causative agent for CML, by interfering with adenosine 5' triphosphate-dependent ABL tyrosine kinase.  Although the efficacy of tyrosine kinase inhibitors (TKIs) has resulted in overall survival of greater than 90%, TKIs are not curative.  Moreover, no currently approved TKIs are effective against the T315I BCR-ABL variant.  However, a new generation of TKIs with activity against T315I is on the horizon.  We will highlight the clin. utility of historical CML therapeutics, those used today (first- and second-generation TKIs), and discuss treatment modalities that are under development.  Recent advances have illuminated the complexity of CML, esp. within the marrow microenvironment.  We contend that the key to curing CML will involve strategies beyond targeting BCR-ABL because primitive human CML stem cells are not dependent on BCR-ABL.  Ultimately, drug combinations or exploiting synthetic lethality may transform responses into definitive cures for CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPF8KXNet0xrVg90H21EOLACvtfcHk0lgxH_hcmE-plg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitVOjtrc%253D&md5=81ca4b49e136803cf0731c9f1a4e95cc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1097%2Fppo.0b013e318237e5b7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252Fppo.0b013e318237e5b7%26sid%3Dliteratum%253Aachs%26aulast%3DWoessner%26aufirst%3DD.%2BW.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26atitle%3DDevelopment%2520of%2520an%2520effective%2520therapy%2520for%2520chronic%2520Myelogenous%2520Leukemia%26jtitle%3DCancer%2520J.%26date%3D2011%26volume%3D17%26spage%3D477%26epage%3D486%26doi%3D10.1097%2Fppo.0b013e318237e5b7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacob, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierks, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, G.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okram, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojciechowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendrich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strauss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajpai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting BCR-ABL by combining allosteric with ATP-binding-site inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1038/nature08675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature08675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20072125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=501-506&author=J.+Zhangauthor=F.+J.+Adri%C3%A1nauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+G.+Liauthor=R.+E.+Iacobauthor=T.+Simauthor=J.+Powersauthor=C.+Dierksauthor=F.+Sunauthor=G.-R.+Guoauthor=Q.+Dingauthor=B.+Okramauthor=Y.+Choiauthor=A.+Wojciechowskiauthor=X.+Dengauthor=G.+Liuauthor=G.+Fendrichauthor=A.+Straussauthor=N.+Vajpaiauthor=S.+Grzesiekauthor=T.+Tuntlandauthor=Y.+Liuauthor=B.+Bursulayaauthor=M.+Azamauthor=P.+W.+Manleyauthor=J.+R.+Engenauthor=G.+Q.+Daleyauthor=M.+Warmuthauthor=N.+S.+Gray&title=Targeting+BCR-ABL+by+combining+allosteric+with+ATP-binding-site+inhibitors&doi=10.1038%2Fnature08675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Bcr-Abl by combining allosteric with ATP-binding-site inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Adrian, Francisco J.; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Li, Allen G.; Iacob, Roxana E.; Sim, Taebo; Powers, John; Dierks, Christine; Sun, Fangxian; Guo, Gui-Rong; Ding, Qiang; Okram, Barun; Choi, Yongmun; Wojciechowski, Amy; Deng, Xianming; Liu, Guoxun; Fendrich, Gabriele; Strauss, Andre; Vajpai, Navratna; Grzesiek, Stephan; Tuntland, Tove; Liu, Yi; Bursulaya, Badry; Azam, Mohammad; Manley, Paul W.; Engen, John R.; Daley, George Q.; Warmuth, Markus; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7280</span>),
    <span class="NLM_cas:pages">501-506</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In an effort to find new pharmacol. modalities to overcome resistance to ATP-binding-site inhibitors of Bcr-Abl, we recently reported the discovery of GNF-2, a selective allosteric Bcr-Abl inhibitor.  Here, using soln. NMR, X-ray crystallog., mutagenesis and hydrogen exchange mass spectrometry, we show that GNF-2 binds to the myristate-binding site of Abl, leading to changes in the structural dynamics of the ATP-binding site.  GNF-5, an analog of GNF-2 with improved pharmacokinetic properties, when used in combination with the ATP-competitive inhibitors imatinib or nilotinib, suppressed the emergence of resistance mutations in vitro, displayed additive inhibitory activity in biochem. and cellular assays against T315I mutant human Bcr-Abl and displayed in vivo efficacy against this recalcitrant mutant in a murine bone-marrow transplantation model.  These results show that therapeutically relevant inhibition of Bcr-Abl activity can be achieved with inhibitors that bind to the myristate-binding site and that combining allosteric and ATP-competitive inhibitors can overcome resistance to either agent alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUtkKFuTQx1LVg90H21EOLACvtfcHk0lgxH_hcmE-plg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D&md5=46c122384fa004d27865c77f4a9165b7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature08675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08675%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%2BJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLi%26aufirst%3DA.%2BG.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DDierks%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DG.-R.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DOkram%26aufirst%3DB.%26aulast%3DChoi%26aufirst%3DY.%26aulast%3DWojciechowski%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFendrich%26aufirst%3DG.%26aulast%3DStrauss%26aufirst%3DA.%26aulast%3DVajpai%26aufirst%3DN.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DAzam%26aufirst%3DM.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DDaley%26aufirst%3DG.%2BQ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520BCR-ABL%2520by%2520combining%2520allosteric%2520with%2520ATP-binding-site%2520inhibitors%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D501%26epage%3D506%26doi%3D10.1038%2Fnature08675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuntini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furutani-Seiki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishima, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagby, S.</span></span> <span> </span><span class="NLM_article-title">Structural features and ligand binding properties of tandem WW domains from YAP and TAZ, nuclear effectors of the Hippo pathway</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">3300</span>– <span class="NLM_lpage">3309</span>, <span class="refDoi"> DOI: 10.1021/bi2001888</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2001888" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVCnt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=3300-3309&author=C.+Webbauthor=A.+Upadhyayauthor=F.+Giuntiniauthor=I.+Egglestonauthor=M.+Furutani-Seikiauthor=R.+Ishimaauthor=S.+Bagby&title=Structural+features+and+ligand+binding+properties+of+tandem+WW+domains+from+YAP+and+TAZ%2C+nuclear+effectors+of+the+Hippo+pathway&doi=10.1021%2Fbi2001888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Features and Ligand Binding Properties of Tandem WW Domains from YAP and TAZ, Nuclear Effectors of the Hippo Pathway</span></div><div class="casAuthors">Webb, Claire; Upadhyay, Abhishek; Giuntini, Francesca; Eggleston, Ian; Furutani-Seiki, Makoto; Ishima, Rieko; Bagby, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3300-3309</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The paralogous multifunctional adaptor proteins YAP and TAZ are the nuclear effectors of the Hippo pathway, a central mechanism of organ size control and stem cell self-renewal.  WW domains, mediators of protein-protein interactions, are essential for YAP and TAZ function, enabling interactions with PPxY motifs of numerous partner proteins.  YAP has single and double WW domain isoforms (YAP1 and YAP2) whereas only a single WW domain isoform of TAZ has been described to date.  Here we identify the first example of a double WW domain isoform of TAZ.  Using NMR, we have characterized conformational features and peptide binding of YAP and TAZ tandem WW domains (WW1-WW2).  The soln. structure of YAP WW2 confirms that it has a canonical three-stranded antiparallel β-sheet WW domain fold.  While chem. shift-based anal. indicates that the WW domains in the tandem WW pairs retain the characteristic WW domain fold, 15N relaxation data show that, within the resp. WW pairs, YAP WW1 and both WW1 and WW2 of TAZ undergo conformational exchange.  15N relaxation data also indicate that the linker between the WW domains is flexible in both YAP and TAZ.  Within both YAP and TAZ tandem WW pairs, WW1 and WW2 bind single PPxY-contg. peptide ligand concurrently and noncooperatively with sub-mM affinity.  YAP and TAZ WW1-WW2 bind a dual PPxY-contg. peptide with approx. 6-fold higher affinity.  Our results indicate that both WW domains in YAP and TAZ are functional and capable of enhanced affinity binding to multi-PPxY partner proteins such as LATS1, ErbB4, and AMOT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvTiUSYhJGc7Vg90H21EOLACvtfcHk0lgxH_hcmE-plg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVCnt7g%253D&md5=994139bb21116a57b7216a04d1b2485f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fbi2001888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2001888%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DA.%26aulast%3DGiuntini%26aufirst%3DF.%26aulast%3DEggleston%26aufirst%3DI.%26aulast%3DFurutani-Seiki%26aufirst%3DM.%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DBagby%26aufirst%3DS.%26atitle%3DStructural%2520features%2520and%2520ligand%2520binding%2520properties%2520of%2520tandem%2520WW%2520domains%2520from%2520YAP%2520and%2520TAZ%252C%2520nuclear%2520effectors%2520of%2520the%2520Hippo%2520pathway%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D3300%26epage%3D3309%26doi%3D10.1021%2Fbi2001888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velentza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sloan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mestan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibitors of BCR-ABL-dependent cell proliferation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">102</span>, <span class="refDoi"> DOI: 10.1038/nchembio760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnchembio760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=16415863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvF2rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=95-102&author=F.+J.+Adri%C3%A1nauthor=Q.+Dingauthor=T.+Simauthor=A.+Velentzaauthor=C.+Sloanauthor=Y.+Liuauthor=G.+Zhangauthor=W.+Hurauthor=S.+Dingauthor=P.+Manleyauthor=J.+Mestanauthor=D.+Fabbroauthor=N.+S.+Gray&title=Allosteric+inhibitors+of+BCR-ABL-dependent+cell+proliferation&doi=10.1038%2Fnchembio760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric inhibitors of Bcr-abl-dependent cell proliferation</span></div><div class="casAuthors">Adrian, Francisco J.; Ding, Qiang; Sim, Taebo; Velentza, Anastasia; Sloan, Christine; Liu, Yi; Zhang, Guobao; Hur, Wooyoung; Ding, Sheng; Manley, Paul; Mestan, Juergen; Fabbro, Doriano; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">95-102</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myelogenous leukemia (CML) is a myeloproliferative disorder characterized at the mol. level by the expression of Bcr-abl, a 210-kDa fusion protein with deregulated tyrosine kinase activity.  Encouraged by the clin. validation of Bcr-abl as the target for the treatment of CML by imatinib, we sought to identify pharmacol. agents that could target this kinase by a distinct mechanism.  We report the discovery of a new class of Bcr-abl inhibitors using an unbiased differential cytotoxicity screen of a combinatorial kinase-directed heterocycle library.  Compds. in this class (exemplified by GNF-2) show exclusive antiproliferative activity toward Bcr-abl-transformed cells, with potencies similar to imatinib, while showing no inhibition of the kinase activity of full-length or catalytic domain of c-abl.  We propose that this new class of compds. inhibits Bcr-abl kinase activity through an allosteric non-ATP competitive mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqjXG0lHqHurVg90H21EOLACvtfcHk0ljWRHjsTq6QwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvF2rsA%253D%253D&md5=c8617fecb6092d71ae163e7cda1e8f94</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnchembio760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio760%26sid%3Dliteratum%253Aachs%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%2BJ.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DVelentza%26aufirst%3DA.%26aulast%3DSloan%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DManley%26aufirst%3DP.%26aulast%3DMestan%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DAllosteric%2520inhibitors%2520of%2520BCR-ABL-dependent%2520cell%2520proliferation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D95%26epage%3D102%26doi%3D10.1038%2Fnchembio760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotesta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groell, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellé, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rummel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Asciminib (ABL001), an allosteric inhibitor of the tyrosine kinase activity of BCR-ABL1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8120</span>– <span class="NLM_lpage">8135</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01040</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01040" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8120-8135&author=J.+Schoepferauthor=W.+Jahnkeauthor=G.+Berelliniauthor=S.+Buonamiciauthor=S.+Cotestaauthor=S.+W.+Cowan-Jacobauthor=S.+Doddauthor=P.+Drueckesauthor=D.+Fabbroauthor=T.+Gabrielauthor=J.-M.+Groellauthor=R.+M.+Grotzfeldauthor=A.+Q.+Hassanauthor=C.+Henryauthor=V.+Iyerauthor=D.+Jonesauthor=F.+Lombardoauthor=A.+Looauthor=P.+W.+Manleyauthor=X.+Pell%C3%A9author=G.+Rummelauthor=B.+Salemauthor=M.+Warmuthauthor=A.+A.+Wylieauthor=T.+Zollerauthor=A.+L.+Marzinzikauthor=P.+Furet&title=Discovery+of+Asciminib+%28ABL001%29%2C+an+allosteric+inhibitor+of+the+tyrosine+kinase+activity+of+BCR-ABL1&doi=10.1021%2Facs.jmedchem.8b01040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1</span></div><div class="casAuthors">Schoepfer, Joseph; Jahnke, Wolfgang; Berellini, Giuliano; Buonamici, Silvia; Cotesta, Simona; Cowan-Jacob, Sandra W.; Dodd, Stephanie; Drueckes, Peter; Fabbro, Doriano; Gabriel, Tobias; Groell, Jean-Marc; Grotzfeld, Robert M.; Hassan, A. Quamrul; Henry, Chrystele; Iyer, Varsha; Jones, Darryl; Lombardo, Franco; Loo, Alice; Manley, Paul W.; Pelle, Xavier; Rummel, Gabriele; Salem, Bahaa; Warmuth, Markus; Wylie, Andrew A.; Zoller, Thomas; Marzinzik, Andreas L.; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8120-8135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of asciminib I, an allosteric inhibitor of the BCR-ABL1 kinase for use in treating chronic myelogenous leukemia (CML), is delineated.  I binds to the myristate pocket of BCR-ABL1 and maintains activity against cancer cells possessing ATP-site mutations in BCR-ABL1 that render them resistant to tyrosine kinase inhibitors; cancer cells resistant to myristate-site mutations are likely to be susceptible to ATP-competitive inhibitors so that combinations of I and tyrosine kinase inhibitors should suppress the emergence of resistance.  Fragment-based screening using NMR and X-ray yielded ligands for the myristate pocket, while an NMR-based conformational assay guided the transformation of these inactive ligands into ABL1 inhibitors.  Further structure-based optimization for potency, physicochem., pharmacokinetic, and drug-like properties, culminated in I which is currently undergoing clin. studies in CML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIQbEIXXgRgbVg90H21EOLACvtfcHk0ljWRHjsTq6QwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN&md5=023a566cdf4faba6dfc05501b8b6a6ed</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01040%26sid%3Dliteratum%253Aachs%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DCotesta%26aufirst%3DS.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGroell%26aufirst%3DJ.-M.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DHenry%26aufirst%3DC.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DPell%25C3%25A9%26aufirst%3DX.%26aulast%3DRummel%26aufirst%3DG.%26aulast%3DSalem%26aufirst%3DB.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DZoller%26aufirst%3DT.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Asciminib%2520%2528ABL001%2529%252C%2520an%2520allosteric%2520inhibitor%2520of%2520the%2520tyrosine%2520kinase%2520activity%2520of%2520BCR-ABL1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8120%26epage%3D8135%26doi%3D10.1021%2Facs.jmedchem.8b01040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzzelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanasse, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nature21702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature21702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=28329763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=733-737&author=A.+A.+Wylieauthor=J.+Schoepferauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+Looauthor=P.+Furetauthor=A.+L.+Marzinzikauthor=X.+Pelleauthor=J.+Donovanauthor=W.+Zhuauthor=S.+Buonamiciauthor=A.+Q.+Hassanauthor=F.+Lombardoauthor=V.+Iyerauthor=M.+Palmerauthor=G.+Berelliniauthor=S.+Doddauthor=S.+Thohanauthor=H.+Bitterauthor=S.+Branfordauthor=D.+M.+Rossauthor=T.+P.+Hughesauthor=L.+Petruzzelliauthor=K.+G.+Vanasseauthor=M.+Warmuthauthor=F.+Hofmannauthor=N.+J.+Keenauthor=W.+R.+Sellers&title=The+allosteric+inhibitor+ABL001+enables+dual+targeting+of+BCR-ABL1&doi=10.1038%2Fnature21702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span></div><div class="casAuthors">Wylie, Andrew A.; Schoepfer, Joseph; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Loo, Alice; Furet, Pascal; Marzinzik, Andreas L.; Pelle, Xavier; Donovan, Jerry; Zhu, Wenjing; Buonamici, Silvia; Hassan, A. Quamrul; Lombardo, Franco; Iyer, Varsha; Palmer, Michael; Berellini, Giuliano; Dodd, Stephanie; Thohan, Sanjeev; Bitter, Hans; Branford, Susan; Ross, David M.; Hughes, Timothy P.; Petruzzelli, Lilli; Vanasse, K. Gary; Warmuth, Markus; Hofmann, Francesco; Keen, Nicholas J.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7647</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myeloid leukemia (CML) is driven by the activity of the BCR-ABL1 fusion oncoprotein.  ABL1 kinase inhibitors have improved the clin. outcomes for patients with CML, with over 80% of patients treated with imatinib surviving for more than 10 years.  Second-generation ABL1 kinase inhibitors induce more potent mol. responses in both previously untreated and imatinib-resistant patients with CML.  Studies in patients with chronic-phase CML have shown that around 50% of patients who achieve and maintain undetectable BCR-ABL1 transcript levels for at least 2 years remain disease-free after the withdrawal of treatment.  Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clin. development testing in patients with CML and Philadelphia chromosome-pos. (Ph+) acute lymphoblastic leukemia.  In contrast to catalytic-site ABL1 kinase inhibitors, ABL001 binds to the myristoyl pocket of ABL1 and induces the formation of an inactive kinase conformation.  ABL001 and second-generation catalytic inhibitors have similar cellular potencies but distinct patterns of resistance mutations, with genetic barcoding studies revealing pre-existing clonal populations with no shared resistance between ABL001 and the catalytic inhibitor nilotinib.  Consistent with this profile, acquired resistance was obsd. with single-agent therapy in mice; however, the combination of ABL001 and nilotinib led to complete disease control and eradicated CML xenograft tumors without recurrence after the cessation of treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreJ5YSS_wiS7Vg90H21EOLACvtfcHk0ljNfMScAEtDhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D&md5=20d00f222ab8d9387cb340a9531b8ba5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature21702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21702%26sid%3Dliteratum%253Aachs%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DPelle%26aufirst%3DX.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DBranford%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DT.%2BP.%26aulast%3DPetruzzelli%26aufirst%3DL.%26aulast%3DVanasse%26aufirst%3DK.%2BG.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DThe%2520allosteric%2520inhibitor%2520ABL001%2520enables%2520dual%2520targeting%2520of%2520BCR-ABL1%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnature21702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khateb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruimi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khamisie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najajreh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metodieva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajna, J.</span></span> <span> </span><span class="NLM_article-title">Overcoming BCR-ABL T315I mutation by combination of GNF-2 and ATP competitors in an ABL-independent mechanism</span>. <i>BMC Canc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">563</span>, <span class="refDoi"> DOI: 10.1186/1471-2407-12-563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1186%2F1471-2407-12-563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23186157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVKiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=563&author=M.+Khatebauthor=N.+Ruimiauthor=H.+Khamisieauthor=Y.+Najajrehauthor=A.+Mianauthor=A.+Metodievaauthor=M.+Ruthardtauthor=J.+Mahajna&title=Overcoming+BCR-ABL+T315I+mutation+by+combination+of+GNF-2+and+ATP+competitors+in+an+ABL-independent+mechanism&doi=10.1186%2F1471-2407-12-563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Bcr-Abl T315I mutation by combination of GNF-2 and ATP competitors in an Abl-independent mechanism</span></div><div class="casAuthors">Khateb, Mamduh; Ruimi, Nili; Khamisie, Hazem; Najajreh, Yousef; Mian, Afsar; Metodieva, Anna; Ruthardt, Martin; Mahajna, Jamal</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">563</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Philadelphia pos. leukemias are characterized by the presence of Bcr-Abl fusion protein which exhibits an abnormal kinase activity.  Selective Abl kinase inhibitors have been successfully established for the treatment of Ph(+) leukemias.  Despite high rates of clin. response, Ph(+) patients can develop resistance against these kinase inhibitors mainly due to point mutations within the Abl protein.  Of special interest is the 'gatekeeper' T315I mutation, which confers complete resistance to Abl kinase inhibitors.  Recently, GNF-2, Abl allosteric kinase inhibitor, was demonstrated to possess cellular activity against Bcr-Abl transformed cells.  Similarly to Abl kinase inhibitors (AKIs), GNF-2 failed to inhibit activity of mutated Bcr-Abl carrying the T315I mutation.  Methods: Ba/F3 cells harboring native or T315I mutated Bcr-Abl constructs were treated with GNF-2 and AKIs.  We monitored the effect of GNF-2 with AKIs on the proliferation and clonigenicity of the different Ba/F3 cells.  In addn., we monitored the auto-phosphorylation activity of Bcr-Abl and JAK2 in cells treated with GNF-2 and AKIs.  Results: In this study, we report a cooperation between AKIs and GNF-2 in inhibiting proliferation and clonigenicity of Ba/F3 cells carrying T315I mutated Bcr-Abl.  Interestingly, cooperation was most evident between Dasatinib and GNF-2.  Furthermore, we showed that GNF-2 was moderately active in inhibiting the activity of JAK2 kinase, and presence of AKIs augmented GNF-2 activity.  Conclusions: Our data illustrated the ability of allosteric inhibitors such as GNF-2 to cooperate with AKIs to overcome T315I mutation by Bcr-Abl-independent mechanisms, providing a possibility of enhancing AKIs efficacy and overcoming resistance in Ph+ leukemia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVaaSokbndBLVg90H21EOLACvtfcHk0ljNfMScAEtDhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVKiu70%253D&md5=0f1c71dda807ee8d7c539012f5c00f87</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-12-563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-12-563%26sid%3Dliteratum%253Aachs%26aulast%3DKhateb%26aufirst%3DM.%26aulast%3DRuimi%26aufirst%3DN.%26aulast%3DKhamisie%26aufirst%3DH.%26aulast%3DNajajreh%26aufirst%3DY.%26aulast%3DMian%26aufirst%3DA.%26aulast%3DMetodieva%26aufirst%3DA.%26aulast%3DRuthardt%26aufirst%3DM.%26aulast%3DMahajna%26aufirst%3DJ.%26atitle%3DOvercoming%2520BCR-ABL%2520T315I%2520mutation%2520by%2520combination%2520of%2520GNF-2%2520and%2520ATP%2520competitors%2520in%2520an%2520ABL-independent%2520mechanism%26jtitle%3DBMC%2520Canc.%26date%3D2012%26volume%3D12%26spage%3D563%26doi%3D10.1186%2F1471-2407-12-563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith, E. J.</span></span> <span> </span><span class="NLM_article-title">Unique map kinase binding sites</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1784</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2007.09.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bbapap.2007.09.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=18068683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlCjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1784&publication_year=2008&pages=48-55&author=R.+Akellaauthor=T.+M.+Moonauthor=E.+J.+Goldsmith&title=Unique+map+kinase+binding+sites&doi=10.1016%2Fj.bbapap.2007.09.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Unique MAP Kinase binding sites</span></div><div class="casAuthors">Akella, Radha; Moon, Thomas M.; Goldsmith, Elizabeth J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1784</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-55</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Map kinases are drug targets for autoimmune disease, cancer, and apoptosis-related diseases.  Drug discovery efforts have developed MAP kinase inhibitors directed toward the ATP binding site and neighboring "DFG-out" site, both of which are targets for inhibitors of other protein kinases.  On the other hand, MAP kinases have unique substrate and small mol. binding sites that could serve as inhibition sites.  The substrate and processing enzyme D-motif binding site is present in all MAP kinases, and has many features of a good small mol. binding site.  Further, the MAP kinase p38α has a binding site near its C-terminus discovered in crystallog. studies.  Finally, the MAP kinases ERK2 and p38α have a second substrate binding site, the FXFP binding site that is exposed in active ERK2 and the D-motif peptide induced conformation of MAP kinases.  Crystallog. evidence of these latter two binding sites is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo1xtxFBIntbVg90H21EOLACvtfcHk0ljNfMScAEtDhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlCjsw%253D%253D&md5=729bec92a896aec49b59182677563e62</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2007.09.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2007.09.016%26sid%3Dliteratum%253Aachs%26aulast%3DAkella%26aufirst%3DR.%26aulast%3DMoon%26aufirst%3DT.%2BM.%26aulast%3DGoldsmith%26aufirst%3DE.%2BJ.%26atitle%3DUnique%2520map%2520kinase%2520binding%2520sites%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2008%26volume%3D1784%26spage%3D48%26epage%3D55%26doi%3D10.1016%2Fj.bbapap.2007.09.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapadat, R.</span></span> <span> </span><span class="NLM_article-title">Mitogen-activated protein kinase pathways mediated by ERK, JNK, AND P38 protein kinases</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1911</span>– <span class="NLM_lpage">1912</span>, <span class="refDoi"> DOI: 10.1126/science.1072682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1126%2Fscience.1072682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12471242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1911-1912&author=G.+L.+Johnsonauthor=R.+Lapadat&title=Mitogen-activated+protein+kinase+pathways+mediated+by+ERK%2C+JNK%2C+AND+P38+protein+kinases&doi=10.1126%2Fscience.1072682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Mitogen-activated protein kinase pathways mediated by ERK, JNK, and p38 protein kinases</span></div><div class="casAuthors">Johnson, Gary L.; Lapadat, Razvan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1911-1912</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Multicellular organisms have 3 well-characterized subfamilies of MAP kinases (MAPKs) that control a vast array of physiol. processes.  These enzymes are regulated by a characteristic phospho-relay system in which a series of 3 protein kinases phosphorylate and activate one another.  The extracellular signal-regulated kinases (ERK1/2) function in the control of cell division, and inhibitors of these enzymes are being explored as anticancer agents.  The c-Jun N-terminal kinases (JNKs) are crit. regulators of transcription, and JNK inhibitors may be effective in control of rheumatoid arthritis.  The p38 MAPKs are activated by inflammatory cytokines and environmental stresses and may contribute to diseases like asthma and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmioiOeBgw-bVg90H21EOLACvtfcHk0ljdDgKeLK_IIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisbc%253D&md5=6788f1e6b9be24756ccec658f6c1c6b6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1126%2Fscience.1072682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1072682%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DG.%2BL.%26aulast%3DLapadat%26aufirst%3DR.%26atitle%3DMitogen-activated%2520protein%2520kinase%2520pathways%2520mediated%2520by%2520ERK%252C%2520JNK%252C%2520AND%2520P38%2520protein%2520kinases%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1911%26epage%3D1912%26doi%3D10.1126%2Fscience.1072682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heo, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, B.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyun, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-H.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the selective inhibition of JNK1 by the scaffolding protein JIP1 and SP600125</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2185</span>– <span class="NLM_lpage">2195</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fsj.emboj.7600212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=15141161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksV2hsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2185-2195&author=Y.-S.+Heoauthor=S.-K.+Kimauthor=C.+I.+Seoauthor=Y.+K.+Kimauthor=B.-J.+Sungauthor=H.+S.+Leeauthor=J.+I.+Leeauthor=S.-Y.+Parkauthor=J.+H.+Kimauthor=K.+Y.+Hwangauthor=Y.-L.+Hyunauthor=Y.+H.+Jeonauthor=S.+Roauthor=J.+M.+Choauthor=T.+G.+Leeauthor=C.-H.+Yang&title=Structural+basis+for+the+selective+inhibition+of+JNK1+by+the+scaffolding+protein+JIP1+and+SP600125&doi=10.1038%2Fsj.emboj.7600212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the selective inhibition of JNK1 by the scaffolding protein JIP1 and SP600125</span></div><div class="casAuthors">Heo, Yong-Seok; Kim, Su-Kyoung; Seo, Chang Il; Kim, Young Kwan; Sung, Byung-Je; Lee, Hye Shin; Lee, Jae Il; Park, Sam-Yong; Kim, Jin Hwan; Hwang, Kwang Yeon; Hyun, Young-Lan; Jeon, Young Ho; Ro, Seonggu; Cho, Joong Myung; Lee, Tae Gyu; Yang, Chul-Hak</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2185-2195</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The c-jun N-terminal kinase (JNK) signaling pathway is regulated by JNK-interacting protein-1 (JIP1), which is a scaffolding protein assembling the components of the JNK cascade.  Overexpression of JIP1 deactivates the JNK pathway selectively by cytoplasmic retention of JNK and thereby inhibits gene expression mediated by JNK, which occurs in the nucleus.  Here, we report the crystal structure of human JNK1 complexed with pepJIP1, the peptide fragment of JIP1, revealing its selectivity for JNK1 over other MAPKs and the allosteric inhibition mechanism.  The van der Waals contacts by the three residues (Pro157, Leu160, and Leu162) of pepJIP1 and the hydrogen bonding between Glu329 of JNK1 and Arg156 of pepJIP1 are crit. for the selective binding.  Binding of the peptide also induces a hinge motion between the N- and C-terminal domains of JNK1 and distorts the ATP-binding cleft, reducing the affinity of the kinase for ATP.  In addn., we also detd. the ternary complex structure of pepJIP1-bound JNK1 complexed with SP600125, an ATP-competitive inhibitor of JNK, providing the basis for the JNK specificity of the compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJlRXjZD8QM7Vg90H21EOLACvtfcHk0ljdDgKeLK_IIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksV2hsbw%253D&md5=6353695c402b21d1d0b5089107e03ee0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600212%26sid%3Dliteratum%253Aachs%26aulast%3DHeo%26aufirst%3DY.-S.%26aulast%3DKim%26aufirst%3DS.-K.%26aulast%3DSeo%26aufirst%3DC.%2BI.%26aulast%3DKim%26aufirst%3DY.%2BK.%26aulast%3DSung%26aufirst%3DB.-J.%26aulast%3DLee%26aufirst%3DH.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BI.%26aulast%3DPark%26aufirst%3DS.-Y.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DHwang%26aufirst%3DK.%2BY.%26aulast%3DHyun%26aufirst%3DY.-L.%26aulast%3DJeon%26aufirst%3DY.%2BH.%26aulast%3DRo%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DT.%2BG.%26aulast%3DYang%26aufirst%3DC.-H.%26atitle%3DStructural%2520basis%2520for%2520the%2520selective%2520inhibition%2520of%2520JNK1%2520by%2520the%2520scaffolding%2520protein%2520JIP1%2520and%2520SP600125%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2185%26epage%3D2195%26doi%3D10.1038%2Fsj.emboj.7600212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stebbins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becattini, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntzen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cellitti, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riel-Mehan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emdadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solinas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span> <span> </span><span class="NLM_article-title">Identification of a new JNK inhibitor targeting the JNK-JIP interaction site</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">16809</span>– <span class="NLM_lpage">16813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0805677105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.0805677105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=18922779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlertbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=16809-16813&author=J.+L.+Stebbinsauthor=S.+K.+Deauthor=T.+Machleidtauthor=B.+Becattiniauthor=J.+Vazquezauthor=C.+Kuntzenauthor=L.-H.+Chenauthor=J.+F.+Cellittiauthor=M.+Riel-Mehanauthor=A.+Emdadiauthor=G.+Solinasauthor=M.+Karinauthor=M.+Pellecchia&title=Identification+of+a+new+JNK+inhibitor+targeting+the+JNK-JIP+interaction+site&doi=10.1073%2Fpnas.0805677105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a new JNK inhibitor targeting the JNK-JIP interaction site</span></div><div class="casAuthors">Stebbins, John L.; De, Surya K.; Machleidt, Thomas; Becattini, Barbara; Vazquez, Jesus; Kuntzen, Christian; Chen, Li-Hsing; Cellitti, Jason F.; Riel-Mehan, Megan; Emdadi, Aras; Solinas, Giovanni; Karin, Michael; Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">16809-16813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">JNK is a stress-activated protein kinase that modulates pathways implicated in a variety of disease states.  JNK-interacting protein-1 (JIP1) is a scaffolding protein that enhances JNK signaling by creating a proximity effect between JNK and upstream kinases.  A minimal peptide region derived from JIP1 is able to inhibit JNK activity both in vitro and in cell.  We report here a series of small mols. JIP1 mimics that function as substrate competitive inhibitors of JNK.  One such compd., BI-78D3, dose-dependently inhibits the phosphorylation of JNK substrates both in vitro and in cell.  In animal studies, BI-78D3 not only blocks JNK dependent Con A-induced liver damage but also restores insulin sensitivity in mouse models of type 2 diabetes.  Our findings open the way for the development of protein kinase inhibitors targeting substrate specific docking sites, rather than the highly conserved ATP binding sites.  In view of its favorable inhibition profile, selectivity, and ability to function in the cellular milieu and in vivo, BI-78D3 represents not only a JNK inhibitor, but also a promising stepping stone toward the development of an innovative class of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5qVRg8EpCvbVg90H21EOLACvtfcHk0ljdDgKeLK_IIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlertbvM&md5=8fef92335594787de91361c5b1683f5e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0805677105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0805677105%26sid%3Dliteratum%253Aachs%26aulast%3DStebbins%26aufirst%3DJ.%2BL.%26aulast%3DDe%26aufirst%3DS.%2BK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DBecattini%26aufirst%3DB.%26aulast%3DVazquez%26aufirst%3DJ.%26aulast%3DKuntzen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.-H.%26aulast%3DCellitti%26aufirst%3DJ.%2BF.%26aulast%3DRiel-Mehan%26aufirst%3DM.%26aulast%3DEmdadi%26aufirst%3DA.%26aulast%3DSolinas%26aufirst%3DG.%26aulast%3DKarin%26aufirst%3DM.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520new%2520JNK%2520inhibitor%2520targeting%2520the%2520JNK-JIP%2520interaction%2520site%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D16809%26epage%3D16813%26doi%3D10.1073%2Fpnas.0805677105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaoud, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeVieux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbreath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moharana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalby, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerwin, S. M.</span></span> <span> </span><span class="NLM_article-title">A fluorescence-based assay for p38α recruitment site binders: Identification of Rooperol as a novel p38α kinase inhibitor</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1002%2Fcbic.201200529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23225637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslymsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=66-71&author=J.+Liauthor=T.+S.+Kaoudauthor=J.+LeVieuxauthor=B.+Gilbreathauthor=S.+Moharanaauthor=K.+N.+Dalbyauthor=S.+M.+Kerwin&title=A+fluorescence-based+assay+for+p38%CE%B1+recruitment+site+binders%3A+Identification+of+Rooperol+as+a+novel+p38%CE%B1+kinase+inhibitor&doi=10.1002%2Fcbic.201200529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A Fluorescence-Based Assay for p38α Recruitment Site Binders: Identification of Rooperol as a Novel p38α Kinase Inhibitor</span></div><div class="casAuthors">Li, Jing; Kaoud, Tamer S.; LeVieux, Jake; Gilbreath, Brad; Moharana, Swapna; Dalby, Kevin N.; Kerwin, Sean M.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">66-71</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This article presents a new class of N-alkynylimidazole small mols. that can serve as probes of the D-recruitment site (DRS) of p38α.  These novel p38α DRS probes have been identified through covalent interaction with Cys119.  Using alkyne-azide click chem., one such probe can be used to fluorescently label p38α in vitro and in cells.  The adduction of p38α by this probe is specifically blocked by the p38α-cognate MKK3 D-site peptide.  Using this probe to identify small mols. that recognize the p38α DRS, the anti-inflammatory natural product rooperol was identified as a novel p38α inhibitor.  This study demonstrates the utility of targeting specific cysteine residues assocd. with kinase recruitment sites for the discovery of novel inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6cWdbk0Gwj7Vg90H21EOLACvtfcHk0ljdDgKeLK_IIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslymsbzE&md5=b0edaf37ef7cd27747d84141ddae45bc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200529%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKaoud%26aufirst%3DT.%2BS.%26aulast%3DLeVieux%26aufirst%3DJ.%26aulast%3DGilbreath%26aufirst%3DB.%26aulast%3DMoharana%26aufirst%3DS.%26aulast%3DDalby%26aufirst%3DK.%2BN.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26atitle%3DA%2520fluorescence-based%2520assay%2520for%2520p38%25CE%25B1%2520recruitment%2520site%2520binders%253A%2520Identification%2520of%2520Rooperol%2520as%2520a%2520novel%2520p38%25CE%25B1%2520kinase%2520inhibitor%26jtitle%3DChemBioChem%26date%3D2013%26volume%3D14%26spage%3D66%26epage%3D71%26doi%3D10.1002%2Fcbic.201200529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stebbins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlickova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntzen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of a potent and selective dual ATP- and substrate-competitive subnanomolar bidentate C-JUN N-terminal kinase (JNK) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6206</span>– <span class="NLM_lpage">6214</span>, <span class="refDoi"> DOI: 10.1021/jm200479c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200479c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6206-6214&author=J.+L.+Stebbinsauthor=S.+K.+Deauthor=P.+Pavlickovaauthor=V.+Chenauthor=T.+Machleidtauthor=L.-H.+Chenauthor=C.+Kuntzenauthor=S.+Kitadaauthor=M.+Karinauthor=M.+Pellecchia&title=Design+and+characterization+of+a+potent+and+selective+dual+ATP-+and+substrate-competitive+subnanomolar+bidentate+C-JUN+N-terminal+kinase+%28JNK%29+inhibitor&doi=10.1021%2Fjm200479c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of a Potent and Selective Dual ATP- and Substrate-Competitive Subnanomolar Bidentate c-Jun N-Terminal Kinase (JNK) Inhibitor</span></div><div class="casAuthors">Stebbins, John L.; De, Surya K.; Pavlickova, Petra; Chen, Vida; Machleidt, Thomas; Chen, Li-Hsing; Kuntzen, Christian; Kitada, Shinichi; Karin, Michael; Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6206-6214</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">C-Jun N-terminal kinases (JNKs) represent valuable targets in the development of new therapies.  Present on the surface of JNK is a binding pocket for substrates and the scaffolding protein JIP1 in close proximity to the ATP binding pocket.  We propose that bidentate compds. linking the binding energies of weakly interacting ATP and substrate mimetics could result in potent and selective JNK inhibitors.  We describe here a bidentate mol., (I), designed against JNK.  I inhibits JNK kinase activity (IC50 = 18 nM; Ki = 1.5 nM) and JNK/substrate assocn. in a displacement assay (IC50 = 46 nM; Ki = 2 nM).  Our data demonstrate that I targets for the ATP and substrate-binding sites on JNK concurrently.  Finally, compd. I successfully inhibits JNK in a variety of cell-based expts., as well as in vivo where it is shown to protect against Jo-2 induced liver damage and improve glucose tolerance in diabetic mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorJZfTAV7pgrVg90H21EOLACvtfcHk0lgWExQ2XM9EQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7jL&md5=088d1dc1c124ab58b7597a541ba60947</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm200479c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200479c%26sid%3Dliteratum%253Aachs%26aulast%3DStebbins%26aufirst%3DJ.%2BL.%26aulast%3DDe%26aufirst%3DS.%2BK.%26aulast%3DPavlickova%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DV.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.-H.%26aulast%3DKuntzen%26aufirst%3DC.%26aulast%3DKitada%26aufirst%3DS.%26aulast%3DKarin%26aufirst%3DM.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DDesign%2520and%2520characterization%2520of%2520a%2520potent%2520and%2520selective%2520dual%2520ATP-%2520and%2520substrate-competitive%2520subnanomolar%2520bidentate%2520C-JUN%2520N-terminal%2520kinase%2520%2528JNK%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6206%26epage%3D6214%26doi%3D10.1021%2Fjm200479c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzarum, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komornik, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben Chetrit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">DEF pocket in p38α facilitates substrate selectivity and mediates autophosphorylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">19537</span>– <span class="NLM_lpage">19547</span>, <span class="refDoi"> DOI: 10.1074/jbc.m113.464511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1074%2Fjbc.m113.464511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23671282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=19537-19547&author=N.+Tzarumauthor=N.+Komornikauthor=D.+Ben+Chetritauthor=D.+Engelbergauthor=O.+Livnah&title=DEF+pocket+in+p38%CE%B1+facilitates+substrate+selectivity+and+mediates+autophosphorylation&doi=10.1074%2Fjbc.m113.464511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">DEF pocket in p38α facilitates substrate selectivity and mediates autophosphorylation</span></div><div class="casAuthors">Tzarum, Netanel; Komornik, Nadav; Ben Chetrit, Dorin; Engelberg, David; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">19537-19547</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Signaling processes are primarily promoted by mol. recognition and corresponding protein-protein interactions.  One of the key eukaryotic signaling pathways is the MAP kinase cascade involved in vital cellular processes such as cell proliferation, differentiation, apoptosis, and stress response.  The principle recognition site of MAP kinases, the common docking (CD) region, forms selective interactions with substrates, upstream activators, and phosphatases.  A second docking site, defined as the DEF site interaction pocket (DEF pocket), is formed subsequent to ERK2 and p38α activation.  Both crystal structures of p38α in its dually phosphorylated form and of intrinsically active mutants showed the DEF pocket, giving motivation for studying its role in substrate activation and selectivity.  Mutating selected DEF pocket residues significantly decreased the phosphorylation levels of three p38α substrates (ATFII, Elk-1, and MBP) with no apparent effect on the phosphorylation of MK2 kinase.  Conversely, mutating the CD region gave the opposite effect, suggesting p38α substrates can be classified into DEF-dependent and DEF-independent substrates.  In addn., mutating DEF pocket residues decreased the autophosphorylation capability of intrinsically active p38α mutants, suggesting DEF-mediated trans-autophosphorylation in p38α.  These results could contribute to understanding substrate selectivity of p38α and serve as a platform for designing p38α-selective DEF site blockers, which partially inhibit p38α binding DEF-dependent substrates, whereas maintaining its other functions intact.  In this context, preliminary results using synthetic peptides reveal significant inhibition of substrate phosphorylation by activated p38α.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqntKfmOsxz1bVg90H21EOLACvtfcHk0lgWExQ2XM9EQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7%252FF&md5=7a6b23bf517f038fef6420a570f7f740</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m113.464511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m113.464511%26sid%3Dliteratum%253Aachs%26aulast%3DTzarum%26aufirst%3DN.%26aulast%3DKomornik%26aufirst%3DN.%26aulast%3DBen%2BChetrit%26aufirst%3DD.%26aulast%3DEngelberg%26aufirst%3DD.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DDEF%2520pocket%2520in%2520p38%25CE%25B1%2520facilitates%2520substrate%2520selectivity%2520and%2520mediates%2520autophosphorylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D19537%26epage%3D19547%26doi%3D10.1074%2Fjbc.m113.464511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. X.</span></span> <span> </span><span class="NLM_article-title">A conserved motif in JNK/p38-specific mapk phosphatases as a determinant for JNK1 recognition and inactivation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10879</span>, <span class="refDoi"> DOI: 10.1038/ncomms10879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fncomms10879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26988444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1elsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10879&author=X.+Liuauthor=C.+S.+Zhangauthor=C.+Luauthor=S.+C.+Linauthor=J.+W.+Wuauthor=Z.+X.+Wang&title=A+conserved+motif+in+JNK%2Fp38-specific+mapk+phosphatases+as+a+determinant+for+JNK1+recognition+and+inactivation&doi=10.1038%2Fncomms10879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved motif in JNK/p38-specific MAPK phosphatases as a determinant for JNK1 recognition and inactivation</span></div><div class="casAuthors">Liu, Xin; Zhang, Chen-Song; Lu, Chang; Lin, Sheng-Cai; Wu, Jia-Wei; Wang, Zhi-Xin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10879</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mitogen-activated protein kinases (MAPKs), important in a large array of signalling pathways, are tightly controlled by a cascade of protein kinases and by MAPK phosphatases (MKPs).  MAPK signalling efficiency and specificity is modulated by protein-protein interactions between individual MAPKs and the docking motifs in cognate binding partners.  Two types of docking interactions have been identified: D-motif-mediated interaction and FXF-docking interaction.  Here we report the crystal structure of JNK1 bound to the catalytic domain of MKP7 at 2.4-Å resoln., providing high-resoln. structural insight into the FXF-docking interaction.  The 285FNFL288 segment in MKP7 directly binds to a hydrophobic site on JNK1 that is near the MAPK insertion and helix αG.  Biochem. studies further reveal that this highly conserved structural motif is present in all members of the MKP family, and the interaction mode is universal and crit. for the MKP-MAPK recognition and biol. function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2QX0w69JOtbVg90H21EOLACvtfcHk0lgWExQ2XM9EQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1elsL0%253D&md5=9a7b84c5075d024d50e7bd4b12f4b68a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms10879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10879%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DC.%2BS.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DS.%2BC.%26aulast%3DWu%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DZ.%2BX.%26atitle%3DA%2520conserved%2520motif%2520in%2520JNK%252Fp38-specific%2520mapk%2520phosphatases%2520as%2520a%2520determinant%2520for%2520JNK1%2520recognition%2520and%2520inactivation%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D10879%26doi%3D10.1038%2Fncomms10879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoofnagle, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabuyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resing, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, N. G.</span></span> <span> </span><span class="NLM_article-title">Docking motif interactions in MAP kinases revealed by hydrogen exchange mass spectrometry</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(04)00161-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fs1097-2765%2804%2900161-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=15068802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvVaqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=43-55&author=T.+Leeauthor=A.+N.+Hoofnagleauthor=Y.+Kabuyamaauthor=J.+Stroudauthor=X.+Minauthor=E.+J.+Goldsmithauthor=L.+Chenauthor=K.+A.+Resingauthor=N.+G.+Ahn&title=Docking+motif+interactions+in+MAP+kinases+revealed+by+hydrogen+exchange+mass+spectrometry&doi=10.1016%2Fs1097-2765%2804%2900161-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Docking motif interactions in MAP kinases revealed by hydrogen exchange mass spectrometry</span></div><div class="casAuthors">Lee, Thomas; Hoofnagle, Andrew N.; Kabuyama, Yukihito; Stroud, James; Min, Xiaoshan; Goldsmith, Elizabeth J.; Chen, Lin; Resing, Katheryn A.; Ahn, Natalie G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-55</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein interactions between MAP kinases and substrates, activators, and scaffolding proteins are regulated by docking site motifs, one contg. basic residues proximal to Leu-X-Leu (DEJL) and a second contg. Phe-X-Phe (DEF).  Hydrogen exchange mass spectrometry was used to identify regions in MAP kinases protected from solvent by docking motif interactions.  Protection by DEJL peptide binding was obsd. in loops spanning β7-β8 and αD-αE in p38α and ERK2.  In contrast, protection by DEF binding to ERK2 revealed a distinct hydrophobic pocket for Phe-X-Phe binding formed between the P+1 site, αF helix, and the MAP kinase insert.  In inactive ERK2, this pocket is occluded by intramol. interactions with residues in the activation lip.  In vitro assays confirm the dependence of Elk1 and nucleoporin binding on ERK2 phosphorylation, and provide a structural basis for preferential involvement of active ERK in substrate binding and nuclear pore protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNNYWwgunLSrVg90H21EOLACvtfcHk0ljxBUcGi3VKMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvVaqsbY%253D&md5=7e7f935ccd5d0b503c82b3ea1525dbbb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2804%2900161-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252804%252900161-3%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%26aulast%3DHoofnagle%26aufirst%3DA.%2BN.%26aulast%3DKabuyama%26aufirst%3DY.%26aulast%3DStroud%26aufirst%3DJ.%26aulast%3DMin%26aufirst%3DX.%26aulast%3DGoldsmith%26aufirst%3DE.%2BJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DResing%26aufirst%3DK.%2BA.%26aulast%3DAhn%26aufirst%3DN.%2BG.%26atitle%3DDocking%2520motif%2520interactions%2520in%2520MAP%2520kinases%2520revealed%2520by%2520hydrogen%2520exchange%2520mass%2520spectrometry%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D14%26spage%3D43%26epage%3D55%26doi%3D10.1016%2Fs1097-2765%2804%2900161-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Zapatero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gum, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groebe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clampit, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haasch, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoll, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderwood, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of non-ATP site inhibitors of the mitogen activated protein (MAP) kinases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1021/cb1002619</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb1002619" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=234-244&author=K.+M.+Comessauthor=C.+Sunauthor=C.+Abad-Zapateroauthor=E.+R.+Goedkenauthor=R.+J.+Gumauthor=D.+W.+Borhaniauthor=M.+Argiriadiauthor=D.+R.+Groebeauthor=Y.+Jiaauthor=J.+E.+Clampitauthor=D.+L.+Haaschauthor=H.+T.+Smithauthor=S.+Wangauthor=D.+Songauthor=M.+L.+Coenauthor=T.+E.+Cloutierauthor=H.+Tangauthor=X.+Chengauthor=C.+Quinnauthor=B.+Liuauthor=Z.+Xinauthor=G.+Liuauthor=E.+H.+Fryauthor=V.+Stollauthor=T.+I.+Ngauthor=D.+Banachauthor=D.+Marcotteauthor=D.+J.+Burnsauthor=D.+J.+Calderwoodauthor=P.+J.+Hajduk&title=Discovery+and+characterization+of+non-ATP+site+inhibitors+of+the+mitogen+activated+protein+%28MAP%29+kinases&doi=10.1021%2Fcb1002619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of Non-ATP Site Inhibitors of the Mitogen Activated Protein (MAP) Kinases</span></div><div class="casAuthors">Comess, Kenneth M.; Sun, Chaohong; Abad-Zapatero, Cele; Goedken, Eric R.; Gum, Rebecca J.; Borhani, David W.; Argiriadi, Maria; Groebe, Duncan R.; Jia, Yong; Clampit, Jill E.; Haasch, Deanna L.; Smith, Harriet T.; Wang, Sanyi; Song, Danying; Coen, Michael L.; Cloutier, Timothy E.; Tang, Hua; Cheng, Xueheng; Quinn, Christopher; Liu, Bo; Xin, Zhili; Liu, Gang; Fry, Elizabeth H.; Stoll, Vincent; Ng, Teresa I.; Banach, David; Marcotte, Doug; Burns, David J.; Calderwood, David J.; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-244</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of protein kinases has validated therapeutic utility for cancer, with at least seven kinase inhibitor drugs on the market.  Protein kinase inhibition also has significant potential for a variety of other diseases, including diabetes, pain, cognition, and chronic inflammatory and immunol. diseases.  However, as the vast majority of current approaches to kinase inhibition target the highly conserved ATP-binding site, the use of kinase inhibitors in treating nononcol. diseases may require great selectivity for the target kinase.  As protein kinases are signal transducers that are involved in binding to a variety of other proteins, targeting alternative, less conserved sites on the protein may provide an avenue for greater selectivity.  Here we report an affinity-based, high-throughput screening technique that allows nonbiased interrogation of small mol. libraries for binding to all exposed sites on a protein surface.  This approach was used to screen both the c-Jun N-terminal protein kinase Jnk-1 (involved in insulin signaling) and p38α (involved in the formation of TNFα and other cytokines).  In addn. to canonical ATP-site ligands, compds. were identified that bind to novel allosteric sites.  The nature, biol. relevance, and mode of binding of these ligands were extensively characterized using two-dimensional 1H/13C NMR spectroscopy, protein X-ray crystallog., surface plasmon resonance, and direct enzymic activity and activation cascade assays.  Jnk-1 and p38α both belong to the MAP kinase family, and the allosteric ligands for both targets bind similarly on a ledge of the protein surface exposed by the MAP insertion present in the CMGC family of protein kinases and distant from the active site.  Medicinal chem. studies resulted in an improved Jnk-1 ligand able to increase adiponectin secretion in human adipocytes and increase insulin-induced protein kinase PKB phosphorylation in human hepatocytes, in similar fashion to Jnk-1 siRNA and to rosiglitazone treatment.  Together, the data suggest that these new ligand series bind to a novel, allosteric, and physiol. relevant site and therefore represent a unique approach to identify kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyEwNo6QURxLVg90H21EOLACvtfcHk0ljxBUcGi3VKMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D&md5=857035713020ac66110b7ef6114232b9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fcb1002619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb1002619%26sid%3Dliteratum%253Aachs%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DGum%26aufirst%3DR.%2BJ.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DArgiriadi%26aufirst%3DM.%26aulast%3DGroebe%26aufirst%3DD.%2BR.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DClampit%26aufirst%3DJ.%2BE.%26aulast%3DHaasch%26aufirst%3DD.%2BL.%26aulast%3DSmith%26aufirst%3DH.%2BT.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DCoen%26aufirst%3DM.%2BL.%26aulast%3DCloutier%26aufirst%3DT.%2BE.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFry%26aufirst%3DE.%2BH.%26aulast%3DStoll%26aufirst%3DV.%26aulast%3DNg%26aufirst%3DT.%2BI.%26aulast%3DBanach%26aufirst%3DD.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCalderwood%26aufirst%3DD.%2BJ.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520non-ATP%2520site%2520inhibitors%2520of%2520the%2520mitogen%2520activated%2520protein%2520%2528MAP%2529%2520kinases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D234%26epage%3D244%26doi%3D10.1021%2Fcb1002619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diskin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">A novel lipid binding site formed by the map kinase insert in p38α</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>375</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.jmb.2007.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=17999933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=375&publication_year=2008&pages=70-79&author=R.+Diskinauthor=D.+Engelbergauthor=O.+Livnah&title=A+novel+lipid+binding+site+formed+by+the+map+kinase+insert+in+p38%CE%B1&doi=10.1016%2Fj.jmb.2007.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Lipid Binding Site Formed by the MAP Kinase Insert in p38α</span></div><div class="casAuthors">Diskin, Ron; Engelberg, David; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The p38 mitogen-activated protein (MAP) kinases function as signaling mols. essential for many cellular processes, particularly mediating stress response.  The activity of p38 MAP kinases is meticulously regulated to reach the desired cellular phenotype.  Several alternative activation and attenuation mechanisms have been characterized recently which include new phosphorylation sites.  Here we present the crystal structure of p38α MAP kinase in complex with n-octyl-β-glucopyranoside (β-OG) detergent.  The complex unveils a novel lipid-binding site formed by a local conformational change of the MAP kinase insert.  This binding is the first evidence of a possible role for the MAP kinase insert in p38.  The binding site can accommodate a large selection of lipidic mols.  In addn., we also show via biophys. methods that arachidonic acid (AA) and its derivs. bind p38α in vitro.  Based on our anal. we propose that the binding of lipids could fine-tune p38α catalytic activity towards a preferred phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6J5fd2Jqe77Vg90H21EOLACvtfcHk0ljxBUcGi3VKMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI&md5=88b97152503182d41fa0ea7b53271229</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DDiskin%26aufirst%3DR.%26aulast%3DEngelberg%26aufirst%3DD.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DA%2520novel%2520lipid%2520binding%2520site%2520formed%2520by%2520the%2520map%2520kinase%2520insert%2520in%2520p38%25CE%25B1%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D375%26spage%3D70%26epage%3D79%26doi%3D10.1016%2Fj.jmb.2007.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzarum, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg-Domovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gills, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">Lipid molecules induce p38α activation via a novel molecular switch</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.jmb.2012.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=23079240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2012&pages=339-353&author=N.+Tzarumauthor=Y.+Eisenberg-Domovichauthor=J.+J.+Gillsauthor=P.+A.+Dennisauthor=O.+Livnah&title=Lipid+molecules+induce+p38%CE%B1+activation+via+a+novel+molecular+switch&doi=10.1016%2Fj.jmb.2012.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid Molecules Induce p38α Activation via a Novel Molecular Switch</span></div><div class="casAuthors">Tzarum, Netanel; Eisenberg-Domovich, Yael; Gills, Joell J.; Dennis, Phillip A.; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">339-353</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">P38α mitogen-activated protein kinase (MAPK) is generally activated by dual phosphorylation but has also been shown to exhibit alternative activation modes.  One of these modes included a direct interaction with phosphatidylinositol ether lipid analogs (PIA), inducing p38α autoactivation and apoptosis.  Perifosine, an Akt inhibitor in phase II clin. trials, also showed p38α activation properties similar to those of PIAs.  The crystal structures of p38α in complex with PIA23, PIA24 and perifosine provide insights into this unique activation mode.  The activating mols. bind a unique hydrophobic binding site in the kinase C'-lobe formed in part by the MAPK insert region.  In addn., there are conformational changes in the short αEF/αF loop region that acts as an activation switch, inducing autophosphorylation.  Structural and biochem. characterization of the αEF/αF loop identified Trp197 as a key residue in the lipid binding and in p38α catalytic activity.  The lipid binding site also accommodates hydrophobic inhibitor mols., and thus can serve as a novel p38α-target for specific activation or inhibition, with novel therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ5MaNgn-kFbVg90H21EOLACvtfcHk0ljgGPc33Iy0OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I&md5=4341b3620e35b8a54f68448817ca157b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTzarum%26aufirst%3DN.%26aulast%3DEisenberg-Domovich%26aufirst%3DY.%26aulast%3DGills%26aufirst%3DJ.%2BJ.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DLipid%2520molecules%2520induce%2520p38%25CE%25B1%2520activation%2520via%2520a%2520novel%2520molecular%2520switch%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D424%26spage%3D339%26epage%3D353%26doi%3D10.1016%2Fj.jmb.2012.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanderpool, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ping, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phonephaly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margosiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermolieff, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the CHK1 allosteric inhibitor binding site</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9823</span>– <span class="NLM_lpage">9830</span>, <span class="refDoi"> DOI: 10.1021/bi900258v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi900258v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9823-9830&author=D.+Vanderpoolauthor=T.+O.+Johnsonauthor=C.+Pingauthor=S.+Bergqvistauthor=G.+Altonauthor=S.+Phonephalyauthor=E.+Ruiauthor=C.+Luoauthor=Y.-L.+Dengauthor=S.+Grantauthor=T.+Quenzerauthor=S.+Margosiakauthor=J.+Registerauthor=E.+Brownauthor=J.+Ermolieff&title=Characterization+of+the+CHK1+allosteric+inhibitor+binding+site&doi=10.1021%2Fbi900258v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the CHK1 Allosteric Inhibitor Binding Site</span></div><div class="casAuthors">Vanderpool, Darin; Johnson, Ted O.; Ping, Chen; Bergqvist, Simon; Alton, Gordon; Phonephaly, Soneprasith; Rui, Eugene; Luo, Chun; Deng, Ya-Li; Grant, Stephan; Quenzer, Terri; Margosiak, Steve; Register, James; Brown, Ed; Ermolieff, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9823-9830</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Checkpoint kinase 1 (CHK1) is a key element in the DNA damage response pathway and plays a crucial role in the S-G2-phase checkpoint.  Inhibiting CHK1 is a therapeutic strategy involving abrogation of the G2/M mitotic checkpoint defense of tumor cells toward lethal damage induced by DNA-directed chemotherapeutic agents.  To date, most CHK1 inhibition approaches have involved targeting the ATP site of this kinase.  In this study, we provide crystallog. and kinetic characterization of two small mol. inhibitors that bind to an allosteric site in the proximity of the CHK1 substrate site.  Anal. of kinetic and biophys. data has led to the conclusion that these small mol. allosteric site inhibitors of CHK1 are reversible and are neither ATP- nor peptide substrate-competitive.  Ki values of 1.89 and 0.15 μM, resp., have been detd. for these compds. using a mixed inhibitor kinetic anal.  Cocrystal structures of the inhibitors bound to CHK1 reveal an allosteric site, unique to CHK1, located in the C-terminal domain and consisting of a shallow groove linked to a small hydrophobic pocket.  The pocket displays induced fit characteristics in the presence of the two inhibitors.  These findings establish the potential for the development of highly selective CHK1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMx2Syic38ULVg90H21EOLACvtfcHk0ljgGPc33Iy0OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO&md5=463e6df560e0cc8b3e8fed96be567284</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi900258v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi900258v%26sid%3Dliteratum%253Aachs%26aulast%3DVanderpool%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DPing%26aufirst%3DC.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DPhonephaly%26aufirst%3DS.%26aulast%3DRui%26aufirst%3DE.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DY.-L.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DMargosiak%26aufirst%3DS.%26aulast%3DRegister%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DE.%26aulast%3DErmolieff%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520the%2520CHK1%2520allosteric%2520inhibitor%2520binding%2520site%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D9823%26epage%3D9830%26doi%3D10.1021%2Fbi900258v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Converso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartingh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreatsoulas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stirdivant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drakas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepp-Lorenzino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jezequel-Sur, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span> <span> </span><span class="NLM_article-title">Development of thioquinazolinones, allosteric CHK1 kinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1240</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.12.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bmcl.2008.12.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=19155174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1240-1244&author=A.+Conversoauthor=T.+Hartinghauthor=R.+M.+Garbaccioauthor=E.+Tasberauthor=K.+Rickertauthor=M.+E.+Fraleyauthor=Y.+Yanauthor=C.+Kreatsoulasauthor=S.+Stirdivantauthor=B.+Drakasauthor=E.+S.+Walshauthor=K.+Hamiltonauthor=C.+A.+Buserauthor=X.+Maoauthor=M.+T.+Abramsauthor=S.+C.+Beckauthor=W.+Taoauthor=R.+Lobellauthor=L.+Sepp-Lorenzinoauthor=J.+Zugay-Murphyauthor=V.+Sardanaauthor=S.+K.+Munshiauthor=S.+M.+Jezequel-Surauthor=P.+D.+Zuckauthor=G.+D.+Hartman&title=Development+of+thioquinazolinones%2C+allosteric+CHK1+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2008.12.076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Development of thioquinazolinones, allosteric Chk1 kinase inhibitors</span></div><div class="casAuthors">Converso, Antonella; Hartingh, Timothy; Garbaccio, Robert M.; Tasber, Edward; Rickert, Keith; Fraley, Mark E.; Yan, Youwei; Kreatsoulas, Constantine; Stirdivant, Steve; Drakas, Bob; Walsh, Eileen S.; Hamilton, Kelly; Buser, Carolyn A.; Mao, Xianzhi; Abrams, Marc T.; Beck, Stephen C.; Tao, Weikang; Lobell, Rob; Sepp-Lorenzino, Laura; Zugay-Murphy, Joan; Sardana, Vinod; Munshi, Sanjeev K.; Jezequel-Sur, Sylvie Marie; Zuck, Paul D.; Hartman, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1240-1244</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high throughput screening campaign was designed to identify allosteric inhibitors of Chk1 kinase by testing compds. at high concn.  Activity was then obsd. at Km for ATP and at near-physiol. concns. of ATP.  This strategy led to the discovery of a non-ATP competitive thioquinazolinone series which was optimized for potency and stability.  An X-ray crystal structure for the complex of our best inhibitor bound to Chk1 was solved, indicating that it binds to an allosteric site ∼13 Å from the ATP binding site.  Preliminary data is presented for several of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bIRNnirJ4bVg90H21EOLACvtfcHk0ljgGPc33Iy0OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D&md5=e91440930e976b9aafcfbb1597679511</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.12.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.12.076%26sid%3Dliteratum%253Aachs%26aulast%3DConverso%26aufirst%3DA.%26aulast%3DHartingh%26aufirst%3DT.%26aulast%3DGarbaccio%26aufirst%3DR.%2BM.%26aulast%3DTasber%26aufirst%3DE.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DFraley%26aufirst%3DM.%2BE.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DKreatsoulas%26aufirst%3DC.%26aulast%3DStirdivant%26aufirst%3DS.%26aulast%3DDrakas%26aufirst%3DB.%26aulast%3DWalsh%26aufirst%3DE.%2BS.%26aulast%3DHamilton%26aufirst%3DK.%26aulast%3DBuser%26aufirst%3DC.%2BA.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DAbrams%26aufirst%3DM.%2BT.%26aulast%3DBeck%26aufirst%3DS.%2BC.%26aulast%3DTao%26aufirst%3DW.%26aulast%3DLobell%26aufirst%3DR.%26aulast%3DSepp-Lorenzino%26aufirst%3DL.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DSardana%26aufirst%3DV.%26aulast%3DMunshi%26aufirst%3DS.%2BK.%26aulast%3DJezequel-Sur%26aufirst%3DS.%2BM.%26aulast%3DZuck%26aufirst%3DP.%2BD.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26atitle%3DDevelopment%2520of%2520thioquinazolinones%252C%2520allosteric%2520CHK1%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1240%26epage%3D1244%26doi%3D10.1016%2Fj.bmcl.2008.12.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, N.</span></span> <span> </span><span class="NLM_article-title">A new target for an old drug: Identifying mitoxantrone as a nanomolar inhibitor of PIM1 kinase via kinome-wide selectivity modeling</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2619</span>– <span class="NLM_lpage">2629</span>, <span class="refDoi"> DOI: 10.1021/jm400045y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400045y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2619-2629&author=X.+Wanauthor=W.+Zhangauthor=L.+Liauthor=Y.+Xieauthor=W.+Liauthor=N.+Huang&title=A+new+target+for+an+old+drug%3A+Identifying+mitoxantrone+as+a+nanomolar+inhibitor+of+PIM1+kinase+via+kinome-wide+selectivity+modeling&doi=10.1021%2Fjm400045y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling</span></div><div class="casAuthors">Wan, Xiaobo; Zhang, Wei; Li, Li; Xie, Yuting; Li, Wei; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2619-2629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rational design of selective kinase inhibitors remains a great challenge.  Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile.  We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases.  Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays.  Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells.  We further detd. the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition.  Although mitoxantrone's mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute to mitoxantrone's therapeutic efficacy and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCdODeUj1frVg90H21EOLACvtfcHk0lijUEWUXqr5cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D&md5=ff53cfe69b80b0b28821a4bfb4583378</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm400045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400045y%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DA%2520new%2520target%2520for%2520an%2520old%2520drug%253A%2520Identifying%2520mitoxantrone%2520as%2520a%2520nanomolar%2520inhibitor%2520of%2520PIM1%2520kinase%2520via%2520kinome-wide%2520selectivity%2520modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2619%26epage%3D2629%26doi%3D10.1021%2Fjm400045y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gureasko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">An allosteric mechanism for activation of the kinase domain of epidermal growth factor receptor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1149</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.cell.2006.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=16777603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=1137-1149&author=X.+Zhangauthor=J.+Gureaskoauthor=K.+Shenauthor=P.+A.+Coleauthor=J.+Kuriyan&title=An+allosteric+mechanism+for+activation+of+the+kinase+domain+of+epidermal+growth+factor+receptor&doi=10.1016%2Fj.cell.2006.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric mechanism for activation of the kinase domain of epidermal growth factor receptor</span></div><div class="casAuthors">Zhang, Xuewu; Gureasko, Jodi; Shen, Kui; Cole, Philip A.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1137-1149</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mechanism by which the epidermal growth factor receptor (EGFR) is activated upon dimerization has eluded definition.  We find that the EGFR kinase domain can be activated by increasing its local concn. or by mutating a leucine (L834R) in the activation loop, the phosphorylation of which is not required for activation.  This suggests that the kinase domain is intrinsically auto-inhibited, and an intermol. interaction promotes its activation.  Using further mutational anal. and crystallog. we demonstrate that the auto-inhibited conformation of the EGFR kinase domain resembles that of Src and cyclin-dependent kinases (CDKs).  EGFR activation results from the formation of an asym. dimer in which the C-terminal lobe of one kinase domain plays a role analogous to that of cyclin in activated CDK/cyclin complexes.  The CDK/cyclin-like complex formed by two kinase domains thus explains the activation of EGFR-family receptors by homo- or heterodimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQsRAzLF-wK7Vg90H21EOLACvtfcHk0lijUEWUXqr5cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D&md5=f7b6795851425346f588cbd682125c93</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGureasko%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DAn%2520allosteric%2520mechanism%2520for%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D1137%26epage%3D1149%26doi%3D10.1016%2Fj.cell.2006.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickin, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the egf receptor by binding of MIG6 to an activating kinase domain interface</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nature05998</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fnature05998" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=18046415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=741-744&author=X.+Zhangauthor=K.+A.+Pickinauthor=R.+Boseauthor=N.+Juraauthor=P.+A.+Coleauthor=J.+Kuriyan&title=Inhibition+of+the+egf+receptor+by+binding+of+MIG6+to+an+activating+kinase+domain+interface&doi=10.1038%2Fnature05998"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the EGF receptor by binding of MIG6 to an activating kinase domain interface</span></div><div class="casAuthors">Zhang, Xuewu; Pickin, Kerry A.; Bose, Ron; Jura, Natalia; Cole, Philip A.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7170</span>),
    <span class="NLM_cas:pages">741-744</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the epidermal growth factor receptor family (EGFR/ERBB1, ERBB2/HER2, ERBB3/HER3 and ERBB4/HER4) are key targets for inhibition in cancer therapy.  Crit. for activation is the formation of an asym. dimer by the intracellular kinase domains, in which the carboxy-terminal lobe (C lobe) of one kinase domain induces an active conformation in the other.  The cytoplasmic protein MIG6 (mitogen-induced gene 6; also known as ERRFI1) interacts with and inhibits the kinase domains of EGFR and ERBB2 (refs).  Crystal structures of complexes between the EGFR kinase domain and a fragment of MIG6 show that a ∼25-residue epitope (segment 1) from MIG6 binds to the distal surface of the C lobe of the kinase domain.  Biochem. and cell-based analyses confirm that this interaction contributes to EGFR inhibition by blocking the formation of the activating dimer interface.  A longer MIG6 peptide that is extended C terminal to segment 1 has increased potency as an inhibitor of the activated EGFR kinase domain, while retaining a crit. dependence on segment 1.  We show that signaling by EGFR mols. that contain constitutively active kinase domains still requires formation of the asym. dimer, underscoring the importance of dimer interface blockage in MIG6-mediated inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2aM4ZGKQTr7Vg90H21EOLACvtfcHk0lijUEWUXqr5cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrP&md5=788822e6303a35eddea3e032875c57d7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature05998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05998%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DPickin%26aufirst%3DK.%2BA.%26aulast%3DBose%26aufirst%3DR.%26aulast%3DJura%26aufirst%3DN.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520the%2520egf%2520receptor%2520by%2520binding%2520of%2520MIG6%2520to%2520an%2520activating%2520kinase%2520domain%2520interface%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D741%26epage%3D744%26doi%3D10.1038%2Fnature05998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rothweiler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stensen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. S.</span></span> <span> </span><span class="NLM_article-title">Luciferin and derivatives as a DYRK selective scaffold for the design of protein kinase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.ejmech.2015.02.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25768698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFGjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=140-148&author=U.+Rothweilerauthor=J.+Erikssonauthor=W.+Stensenauthor=F.+Leesonauthor=R.+A.+Enghauthor=J.+S.+Svendsen&title=Luciferin+and+derivatives+as+a+DYRK+selective+scaffold+for+the+design+of+protein+kinase+inhibitors&doi=10.1016%2Fj.ejmech.2015.02.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Luciferin and derivatives as a DYRK selective scaffold for the design of protein kinase inhibitors</span></div><div class="casAuthors">Rothweiler, Ulli; Eriksson, Jonas; Stensen, Wenche; Leeson, Frederick; Engh, Richard A.; Svendsen, John S.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">140-148</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">D-Luciferin is widely used as a substrate in luciferase catalyzed bioluminescence assays for in vitro studies.  However, little is known about cross reactivity and potential interference of D-luciferin with other enzymes.  The authors serendipitously found that firefly luciferin inhibited the CDK2/Cyclin A protein kinase.  Inhibition profiling of D-luciferin over a 103-protein kinase panel showed significant inhibition of a small set of protein kinases, in particular the DYRK-family, but also other members of the CMGC-group, including ERK8 and CK2.  Inhibition profiling on a 16-member focused library derived from D-luciferin confirms that D-luciferin represents a DYRK-selective chemotype of fragment-like mol. wt.  Thus, observation of its inhibitory activity and the initial SAR information reported here promise to be useful for further design of protein kinase inhibitors with related scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqEtt2EfYajrVg90H21EOLACvtfcHk0lhC1RLfyvBu4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFGjsL4%253D&md5=5d3fd3650bf11958c7e60a5656b28dfd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DRothweiler%26aufirst%3DU.%26aulast%3DEriksson%26aufirst%3DJ.%26aulast%3DStensen%26aufirst%3DW.%26aulast%3DLeeson%26aufirst%3DF.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DSvendsen%26aufirst%3DJ.%2BS.%26atitle%3DLuciferin%2520and%2520derivatives%2520as%2520a%2520DYRK%2520selective%2520scaffold%2520for%2520the%2520design%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D140%26epage%3D148%26doi%3D10.1016%2Fj.ejmech.2015.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaffarogullari, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masterson, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalfe, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traaseth, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balatri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musa, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distefano, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veglia, G.</span></span> <span> </span><span class="NLM_article-title">A myristoyl/phosphoserine switch controls cAMP-dependent protein kinase association to membranes</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>411</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.06.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.jmb.2011.06.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=21740913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVaitLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=411&publication_year=2011&pages=823-836&author=E.+C.+Gaffarogullariauthor=L.+R.+Mastersonauthor=E.+E.+Metcalfeauthor=N.+J.+Traasethauthor=E.+Balatriauthor=M.+M.+Musaauthor=D.+Mullenauthor=M.+D.+Distefanoauthor=G.+Veglia&title=A+myristoyl%2Fphosphoserine+switch+controls+cAMP-dependent+protein+kinase+association+to+membranes&doi=10.1016%2Fj.jmb.2011.06.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A Myristoyl/Phosphoserine Switch Controls cAMP-Dependent Protein Kinase Association to Membranes</span></div><div class="casAuthors">Gaffarogullari, Ece C.; Masterson, Larry R.; Metcalfe, Emily E.; Traaseth, Nathaniel J.; Balatri, Erica; Musa, Musa M.; Mullen, Daniel; Distefano, Mark D.; Veglia, Gianluigi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">411</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-836</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The cAMP-dependent protein kinase [protein kinase A (PKA)] mediates a myriad of cellular signaling events, and its activity is tightly regulated in both space and time.  Among these regulatory mechanisms is N-myristoylation, whose biol. role has been elusive.  Using a combination of thermodn., kinetics, and spectroscopic methods, we analyzed the effects of N-myristoylation and phosphorylation at Ser10 on the interactions of PKA with model membranes.  We found that in the absence of lipids, the myristoyl group is tucked into the hydrophobic binding pocket of the enzyme (myr-in state).  Upon assocn. with lipid bilayers, the myristoyl group is extruded and inserts into the hydrocarbon region of the lipid bilayer (myr-out state).  NMR data indicate that the enzyme undergoes conformational equil. between myr-in and myr-out states, which can be shifted by either interaction with membranes and/or phosphorylation at Ser10.  Our results provide evidence that the membrane binding motif of the myristoylated C-subunit of PKA (PKA-C) steers the enzyme toward lipids independent of its regulatory subunit or an A-kinase anchoring protein, providing an addnl. mechanism to localize the enzyme near membrane-bound substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRbg7PQNbdnbVg90H21EOLACvtfcHk0lhC1RLfyvBu4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVaitLzL&md5=3e039c8d0ef13a83d98b453f838a387a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.06.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.06.034%26sid%3Dliteratum%253Aachs%26aulast%3DGaffarogullari%26aufirst%3DE.%2BC.%26aulast%3DMasterson%26aufirst%3DL.%2BR.%26aulast%3DMetcalfe%26aufirst%3DE.%2BE.%26aulast%3DTraaseth%26aufirst%3DN.%2BJ.%26aulast%3DBalatri%26aufirst%3DE.%26aulast%3DMusa%26aufirst%3DM.%2BM.%26aulast%3DMullen%26aufirst%3DD.%26aulast%3DDistefano%26aufirst%3DM.%2BD.%26aulast%3DVeglia%26aufirst%3DG.%26atitle%3DA%2520myristoyl%252Fphosphoserine%2520switch%2520controls%2520cAMP-dependent%2520protein%2520kinase%2520association%2520to%2520membranes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D411%26spage%3D823%26epage%3D836%26doi%3D10.1016%2Fj.jmb.2011.06.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buosi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontiggia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanism of Aurora A kinase autophosphorylation and its allosteric activation by TPX2</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e02667</span> <span class="refDoi"> DOI: 10.7554/elife.02667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.7554%2Felife.02667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=24867643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Smtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&author=A.+Zorbaauthor=V.+Buosiauthor=S.+Kutterauthor=N.+Kernauthor=F.+Pontiggiaauthor=Y.-J.+Choauthor=D.+Kern&title=Molecular+mechanism+of+Aurora+A+kinase+autophosphorylation+and+its+allosteric+activation+by+TPX2&doi=10.7554%2Felife.02667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanism of Aurora A kinase autophosphorylation and its allosteric activation by TPX2</span></div><div class="casAuthors">Zorba, Adelajda; Buosi, Vanessa; Kutter, Steffen; Kern, Nadja; Pontiggia, Francesco; Cho, Young-Jin; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e02667/1-e02667/24, 24</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">We elucidate the mol. mechanisms of two distinct activation strategies (autophosphorylation and TPX2-mediated activation) in human Aurora A kinase.  Classic allosteric activation is in play where either activation loop phosphorylation or TPX2 binding to a conserved hydrophobic groove shifts the equil. far towards the active conformation.  We resolve the controversy about the mechanism of autophosphorylation by demonstrating intermol. autophosphorylation in a long-lived dimer by combining X-ray crystallog. with functional assays.  We then address the allosteric activation by TPX2 through activity assays and the crystal structure of a domain-swapped dimer of dephosphorylated Aurora A and TPX21-25.  While autophosphorylation is the key regulatory mechanism in the centrosomes in the early stages of mitosis, allosteric activation by TPX2 of dephosphorylated Aurora A could be at play in the spindle microtubules.  The mechanistic insights into autophosphorylation and allosteric activation by TPX2 binding proposed here, may have implications for understanding regulation of other protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLzzlNNQcyXrVg90H21EOLACvtfcHk0lhC1RLfyvBu4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Smtb7E&md5=fadf30228b273acd1d60fbc6be1e1098</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.7554%2Felife.02667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252Felife.02667%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DBuosi%26aufirst%3DV.%26aulast%3DKutter%26aufirst%3DS.%26aulast%3DKern%26aufirst%3DN.%26aulast%3DPontiggia%26aufirst%3DF.%26aulast%3DCho%26aufirst%3DY.-J.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DMolecular%2520mechanism%2520of%2520Aurora%2520A%2520kinase%2520autophosphorylation%2520and%2520its%2520allosteric%2520activation%2520by%2520TPX2%26jtitle%3DeLife%26date%3D2014%26volume%3D3%26doi%3D10.7554%2Felife.02667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tada, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of MKK4 kinase domain reveal that substrate peptide binds to an allosteric site and induces an auto-inhibition state</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>400</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2010.08.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.bbrc.2010.08.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=20732303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1WltrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=369-373&author=T.+Matsumotoauthor=T.+Kinoshitaauthor=Y.+Kiriiauthor=K.+Yokotaauthor=K.+Hamadaauthor=T.+Tada&title=Crystal+structures+of+MKK4+kinase+domain+reveal+that+substrate+peptide+binds+to+an+allosteric+site+and+induces+an+auto-inhibition+state&doi=10.1016%2Fj.bbrc.2010.08.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of MKK4 kinase domain reveal that substrate peptide binds to an allosteric site and induces an auto-inhibition state</span></div><div class="casAuthors">Matsumoto, Takashi; Kinoshita, Takayoshi; Kirii, Yasuyuki; Yokota, Koichi; Hamada, Kensaku; Tada, Toshiji</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">369-373</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MKK4 kinase activates both JNK and p38 kinases.  Here, the authors detd. the crystal structures of the human non-phosphorylated MKK4 kinase domain (npMKK4) complexed with AMP-PNP (npMKK4/AMP) and a ternary complex of npMKK4, AMP-PNP and p38α peptide (npMKK4/AMP/p38).  These crystal structures revealed that the p38α peptide-bound npMKK4 at the allosteric site rather than at the putative substrate binding site and induced an auto-inhibition state.  While the activation loop of the npMKK4/AMP complex was disordered, in the npMKK4/AMP/p38 complex it configured a long α-helix, which prevented substrate access to the active site and αC-helix movement to the active configuration of MKK4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-U_oOxzk9frVg90H21EOLACvtfcHk0lih-kwuRGMb4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1WltrjK&md5=17398782a2a8508f07bf6b1f5745c676</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2010.08.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2010.08.071%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DT.%26aulast%3DKinoshita%26aufirst%3DT.%26aulast%3DKirii%26aufirst%3DY.%26aulast%3DYokota%26aufirst%3DK.%26aulast%3DHamada%26aufirst%3DK.%26aulast%3DTada%26aufirst%3DT.%26atitle%3DCrystal%2520structures%2520of%2520MKK4%2520kinase%2520domain%2520reveal%2520that%2520substrate%2520peptide%2520binds%2520to%2520an%2520allosteric%2520site%2520and%2520induces%2520an%2520auto-inhibition%2520state%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D400%26spage%3D369%26epage%3D373%26doi%3D10.1016%2Fj.bbrc.2010.08.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braisted, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raphael, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Site-directed ligand discovery</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">9367</span>– <span class="NLM_lpage">9372</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.17.9367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.97.17.9367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10944209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtVensr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=9367-9372&author=D.+A.+Erlansonauthor=A.+C.+Braistedauthor=D.+R.+Raphaelauthor=M.+Randalauthor=R.+M.+Stroudauthor=E.+M.+Gordonauthor=J.+A.+Wells&title=Site-directed+ligand+discovery&doi=10.1073%2Fpnas.97.17.9367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Site-directed ligand discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Braisted, Andrew C.; Raphael, Darren R.; Randal, Mike; Stroud, Robert M.; Gordon, Eric M.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9367-9372</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report a strategy (called "tethering") to discover low mol. wt. ligands (≈250 Da) that bind weakly to targeted sites on proteins through an intermediary disulfide tether.  A native or engineered cysteine in a protein is allowed to react reversibly with a small library of disulfide-contg. mols. (≈1,200 compds.) at concns. typically used in drug screening (10 to 200 μM).  The cysteine-captured ligands, which are readily identified by MS, are among the most stable complexes, even though in the absence of the covalent tether the ligands may bind very weakly.  This method was applied to generate a potent inhibitor for thymidylate synthase, an essential enzyme in pyrimidine metab. with therapeutic applications in cancer and infectious diseases.  The affinity of the untethered ligand (Ki≈1 mM) was improved 3,000-fold by synthesis of a small set of analogs with the aid of crystallog. structures of the tethered complex.  Such site-directed ligand discovery allows one to nucleate drug design from a spatially targeted lead fragment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4pxPEoDE3FrVg90H21EOLACvtfcHk0lih-kwuRGMb4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtVensr8%253D&md5=fef57a0d21af9bc7f6f1f4bf8c023ae5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.17.9367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.17.9367%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DBraisted%26aufirst%3DA.%2BC.%26aulast%3DRaphael%26aufirst%3DD.%2BR.%26aulast%3DRandal%26aufirst%3DM.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DGordon%26aufirst%3DE.%2BM.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSite-directed%2520ligand%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D9367%26epage%3D9372%26doi%3D10.1073%2Fpnas.97.17.9367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inesta-Vaquera, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughlin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoGrasso, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent inhibitors of JNK</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.chembiol.2011.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=22284361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=140-154&author=T.+Zhangauthor=F.+Inesta-Vaqueraauthor=M.+Niepelauthor=J.+Zhangauthor=S.+B.+Ficarroauthor=T.+Machleidtauthor=T.+Xieauthor=J.+A.+Martoauthor=N.+Kimauthor=T.+Simauthor=J.+D.+Laughlinauthor=H.+Parkauthor=P.+V.+LoGrassoauthor=M.+Patricelliauthor=T.+K.+Nomanbhoyauthor=P.+K.+Sorgerauthor=D.+R.+Alessiauthor=N.+S.+Gray&title=Discovery+of+potent+and+selective+covalent+inhibitors+of+JNK&doi=10.1016%2Fj.chembiol.2011.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Inhibitors of JNK</span></div><div class="casAuthors">Zhang, Tinghu; Inesta-Vaquera, Francisco; Niepel, Mario; Zhang, Jianming; Ficarro, Scott B.; Machleidt, Thomas; Xie, Ting; Marto, Jarrod A.; Kim, Nam Doo; Sim, Taebo; Laughlin, John D.; Park, Hajeung; LoGrasso, Philip V.; Patricelli, Matt; Nomanbhoy, Tyzoon K.; Sorger, Peter K.; Alessi, Dario R.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-154</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The mitogen-activated kinases JNK1/2/3 are key enzymes in signaling modules that transduce and integrate extracellular stimuli into coordinated cellular response.  Here, we report the discovery of irreversible inhibitors of JNK1/2/3.  We describe two JNK3 cocrystal structures at 2.60 and 2.97 Å resoln. that show the compds. form covalent bonds with a conserved cysteine residue.  JNK-IN-8 is a selective JNK inhibitor that inhibits phosphorylation of c-Jun, a direct substrate of JNK, in cells exposed to submicromolar drug in a manner that depends on covalent modification of the conserved cysteine residue.  Extensive biochem., cellular, and pathway-based profiling establish the selectivity of JNK-IN-8 for JNK and suggests that the compd. will be broadly useful as a pharmacol. probe of JNK-dependent signal transduction.  Potential lead compds. have also been identified for kinases, including IRAK1, PIK3C3, PIP4K2C, and PIP5K3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXs0FY7FSmt7Vg90H21EOLACvtfcHk0lih-kwuRGMb4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghurw%253D&md5=f7d9647fb2be162c637b1d31a778ceb4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DInesta-Vaquera%26aufirst%3DF.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DLaughlin%26aufirst%3DJ.%2BD.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLoGrasso%26aufirst%3DP.%2BV.%26aulast%3DPatricelli%26aufirst%3DM.%26aulast%3DNomanbhoy%26aufirst%3DT.%2BK.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520inhibitors%2520of%2520JNK%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D140%26epage%3D154%26doi%3D10.1016%2Fj.chembiol.2011.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Temmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Diego, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogenberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonko, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilmanns, M.</span></span> <span> </span><span class="NLM_article-title">A PEF/Y substrate recognition and signature motif plays a critical role in DAPK-related kinase activity</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.chembiol.2013.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=24440081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFChsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=264-273&author=K.+Temmermanauthor=I.+de+Diegoauthor=V.+Pogenbergauthor=B.+Simonauthor=W.+Jonkoauthor=X.+Liauthor=M.+Wilmanns&title=A+PEF%2FY+substrate+recognition+and+signature+motif+plays+a+critical+role+in+DAPK-related+kinase+activity&doi=10.1016%2Fj.chembiol.2013.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A PEF/Y Substrate Recognition and Signature Motif Plays a Critical Role in DAPK-Related Kinase Activity</span></div><div class="casAuthors">Temmerman, Koen; de Diego, Inaki; Pogenberg, Vivian; Simon, Bertrand; Jonko, Weronika; Li, Xun; Wilmanns, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-273</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Knowledge about protein kinase substrate preferences is biased toward residues immediately adjacent to the site of phosphorylation.  By a combined structural, biochem., and cellular approach, we have discovered an unexpected substrate recognition element with the consensus sequence PEF/Y in the tumor suppressor death-assocd. protein kinase 1.  This motif can be effectively blocked by a specific pseudosubstrate-type interaction with an autoregulatory domain of this kinase.  In this arrangement, the central PEF/Y glutamate interacts with a conserved arginine distant to the phosphorylation site in sequence and structure.  We also demonstrate that the element is crucial for kinase activity regulation and substrate recognition.  The PEF/Y motif distinguishes close death-assocd. protein kinase relatives from canonical calcium/calmodulin-dependent protein kinases.  Insight into this signature and mode of action offers new opportunities to identify specific small mol. inhibitors in PEF/Y-contg. protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZr7KNdPm2LVg90H21EOLACvtfcHk0lih-kwuRGMb4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFChsLY%253D&md5=d7d49078ecdffd22051d28f8f820c19e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DTemmerman%26aufirst%3DK.%26aulast%3Dde%2BDiego%26aufirst%3DI.%26aulast%3DPogenberg%26aufirst%3DV.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DJonko%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWilmanns%26aufirst%3DM.%26atitle%3DA%2520PEF%252FY%2520substrate%2520recognition%2520and%2520signature%2520motif%2520plays%2520a%2520critical%2520role%2520in%2520DAPK-related%2520kinase%2520activity%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D264%26epage%3D273%26doi%3D10.1016%2Fj.chembiol.2013.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh Bahia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silakari, O.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C-theta inhibitors: A novel therapy for inflammatory disorders</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4725</span>– <span class="NLM_lpage">4746</span>, <span class="refDoi"> DOI: 10.2174/138161212802651625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.2174%2F138161212802651625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=22830352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2nsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4725-4746&author=S.+Chandauthor=N.+Mehtaauthor=M.+Singh+Bahiaauthor=A.+Dixitauthor=O.+Silakari&title=Protein+kinase+C-theta+inhibitors%3A+A+novel+therapy+for+inflammatory+disorders&doi=10.2174%2F138161212802651625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C-theta inhibitors: a novel therapy for inflammatory disorders</span></div><div class="casAuthors">Chand, Sukhvir; Mehta, Nisha; Bahia, Malkeet Singh; Dixit, Anshuman; Silakari, Om</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4725-4746</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  PKC-θ is a serine/threonine specific protein kinase and its activation depends upon the concn. of diacylglycerol (DAG) and phospholipids (phosphatidylserine).  PKC-θ phosphorylates a variety of proteins that are known to be involved in the diverse cellular signaling pathways.  It is predominantly expressed in the T-cells and localized in the center of immunol. synapse upon T-cell receptor (TCR) and CD28 signaling.  Activation of PKC-θ leads to the activation of various transcription factors in the nuclei of T-cells, e.g. NF-κB, NFAT, c-Jun, c-Fos and AP-1 that further control the proliferation and differentiation of T-cells.  Defective T-cell activation in turn leads to the aberrant expression of apoptosis related proteins that cause the poor T-cell survival.  Researchers have found that T-cells deficient in PKC-θ exhibit reduced interleukin-2 (IL-2) prodn.  Apart from this role on IL-2 expression, it also plays crucial roles in the proliferation, differentiation and survival of the T-cells, which make it an attractive therapeutic target for a variety of immunol. and T-cell mediated diseases.  Hence, new mols. capable of modulating the expression or biol. activity of PKC-θ are being developed and tested for their potential as novel therapy for several T-cells mediated disease conditions such as multiple sclerosis, rheumatoid arthritis, asthma, inflammatory bowel disease and organ transplantation, etc.  In the present review, we tried to integrate the recent discoveries on PKC-θ including its pharmacol. and therapeutic potential, along with brief update on its inhibitor mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIN1yWUu7f4rVg90H21EOLACvtfcHk0lh29xX1Ibuf9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2nsr7O&md5=ccb3ca06a596973d1e9e3c262fce05e8</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.2174%2F138161212802651625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212802651625%26sid%3Dliteratum%253Aachs%26aulast%3DChand%26aufirst%3DS.%26aulast%3DMehta%26aufirst%3DN.%26aulast%3DSingh%2BBahia%26aufirst%3DM.%26aulast%3DDixit%26aufirst%3DA.%26aulast%3DSilakari%26aufirst%3DO.%26atitle%3DProtein%2520kinase%2520C-theta%2520inhibitors%253A%2520A%2520novel%2520therapy%2520for%2520inflammatory%2520disorders%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D4725%26epage%3D4746%26doi%3D10.2174%2F138161212802651625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanev, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Linden, O. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">KLIFS: a structural kinase-ligand interaction database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D365</span>– <span class="NLM_lpage">D371</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkv1082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26496949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D365-D371&author=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=O.+P.+J.+van+Lindenauthor=R.+Leursauthor=I.+J.+P.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+structural+kinase-ligand+interaction+database&doi=10.1093%2Fnar%2Fgkv1082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: a structural kinase-ligand interaction database</span></div><div class="casAuthors">Kooistra, Albert J.; Kanev, Georgi K.; van Linden, Oscar P. J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D365-D371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein kinases play a crucial role in cell signaling and are important drug targets in several therapeutic areas.  The KLIFS database contains detailed structural kinase-ligand interaction information derived from all (>2900) structures of catalytic domains of human and mouse protein kinases deposited in the Protein Data Bank in order to provide insights into the structural determinants of kinase-ligand binding and selectivity.  The kinase structures have been processed in a consistent manner by systematically analyzing the structural features and mol. interaction fingerprints (IFPs) of a predefined set of 85 binding site residues with bound ligands.  KLIFS has been completely rebuilt and extended (>65% more structures) since its first release as a data set, including: novel automated annotation methods for (i) the assessment of ligand-targeted subpockets and the anal. of (ii) DFG and (iii) αC-helix conformations; improved and automated protocols for (iv) the generation of sequence/structure alignments, (v) the curation of ligand atom and bond typing for accurate IFP anal. and (vi) weekly database updates.  KLIFS is now accessible via a website (http://klifs.vucompmedchem. nl) that provides a comprehensive visual presentation of different types of chem., biol. and structural chemogenomics data, and allows the user to easily access, compare, search and download the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocfCaKwwgOlbVg90H21EOLACvtfcHk0lh29xX1Ibuf9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP&md5=bc093b02225987fe791a2d41738e4d99</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1082%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520structural%2520kinase-ligand%2520interaction%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD365%26epage%3DD371%26doi%3D10.1093%2Fnar%2Fgkv1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunnery, S.</span></span> <span> </span><span class="NLM_article-title">Human breast cancer cells contain elevated levels and activity of the protein kinase, PKR</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3086</span>– <span class="NLM_lpage">3094</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fsj.onc.1203632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10871861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkvVylurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=3086-3094&author=S.+H.+Kimauthor=A.+P.+Formanauthor=M.+B.+Mathewsauthor=S.+Gunnery&title=Human+breast+cancer+cells+contain+elevated+levels+and+activity+of+the+protein+kinase%2C+PKR&doi=10.1038%2Fsj.onc.1203632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Human breast cancer cells contain elevated levels and activity of the protein kinase, PKR</span></div><div class="casAuthors">Kim, Steve H.; Forman, Adam P.; Mathews, Michael B.; Gunnery, Shobha</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3086-3094</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PKR is a double-stranded (ds) RNA activated protein kinase whose expression is induced by interferon.  Activated PKR phosphorylates its cellular substrate, eIF2, an essential initiation factor of translation.  Prior evidence from a murine model system suggested that PKR may act as a tumor suppressor, but the evidence from human tumors is equivocal.  To study PKR function in human breast cancer, PKR activity was measured in mammary carcinoma cell lines and nontransformed mammary epithelial cell lines.  If PKR functioned as a tumor suppressor in this system, its activity would be higher in nontransformed cells than in carcinoma cells.  On the contrary, PKR autophosphorylation and the phosphorylation of its substrate, the α-subunit of eIF2, is 7-40-fold higher in lysates prepd. from breast carcinoma cell lines than in those from nontransformed epithelial cell lines.  Correspondingly, a larger proportion of eIF2α is present in a phosphorylated state in carcinoma cell lines than in nontransformed cell lines.  Protein synthesis is not inhibited by the high eIF2α phosphorylation in carcinoma cells, probably because they contain higher levels of eIF2B, the initiation factor that is inhibited by eIF2α phosphorylation.  The dramatically lower PKR activity in nontransformed cell lines is partially due to lower PKR protein levels (2-4-fold) as well as to the presence of a PKR inhibitor.  The nontransformed cells contain P58, a known cellular inhibitor of PKR that phys. interacts with PKR and may be responsible for the low PKR activity in these cells.  Taken together, these observations call into question the role of PKR as a tumor suppressor and suggest a pos. regulatory role of PKR in growth control of breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOok8IF4WDkrVg90H21EOLACvtfcHk0lh29xX1Ibuf9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkvVylurs%253D&md5=5d897a1b807a1a25343970c494b194c0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203632%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DForman%26aufirst%3DA.%2BP.%26aulast%3DMathews%26aufirst%3DM.%2BB.%26aulast%3DGunnery%26aufirst%3DS.%26atitle%3DHuman%2520breast%2520cancer%2520cells%2520contain%2520elevated%2520levels%2520and%2520activity%2520of%2520the%2520protein%2520kinase%252C%2520PKR%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D3086%26epage%3D3094%26doi%3D10.1038%2Fsj.onc.1203632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delhem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabile, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajardo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podevin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abadie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremsdorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beretta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brechot, C.</span></span> <span> </span><span class="NLM_article-title">Activation of the interferon-inducible protein kinase PKR by hepatocellular carcinoma derived-hepatitis C virus core protein</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5836</span>– <span class="NLM_lpage">5845</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1038%2Fsj.onc.1204744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=11593389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlSgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=5836-5845&author=N.+Delhemauthor=A.+Sabileauthor=R.+Gajardoauthor=P.+Podevinauthor=A.+Abadieauthor=M.+A.+Blatonauthor=D.+Kremsdorfauthor=L.+Berettaauthor=C.+Brechot&title=Activation+of+the+interferon-inducible+protein+kinase+PKR+by+hepatocellular+carcinoma+derived-hepatitis+C+virus+core+protein&doi=10.1038%2Fsj.onc.1204744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the interferon-inducible protein kinase PKR by hepatocellular carcinoma derived-hepatitis C virus core protein</span></div><div class="casAuthors">Delhem, Nadvia; Sabile, Abdelmajid; Gajardo, Rodrigo; Podevin, Philippe; Abadie, Annie; Blaton, Maria Agnes; Kremsdorf, Dina; Beretta, Laura; Brechot, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">5836-5845</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) is a major etiol. agent of chronic liver disease and hepatocellular carcinoma (HCC).  The authors demonstrate herewith that HCV core proteins encoded by sequences isolated from HCC tumor tissues, but not those derived from their non-tumor counterparts in the same liver, co-localize in vitro and in vivo and co-immunoppt. with PKR in hepatocytic Huh7 cells.  The authors show that this assocn. in fact augments the autophosphorylation of PKR and the phosphorylation of the translation initiation factor eIF2α, which are two markers of PKR activity.  The present study therefore identifies a novel model of virus-cell interactions whereby a viral protein, the HCV core, activates PKR activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHlIAsJKbGhLVg90H21EOLACvtfcHk0livFzMKz8CVmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlSgtrc%253D&md5=15531a174ead64a0b70dc714b943431e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204744%26sid%3Dliteratum%253Aachs%26aulast%3DDelhem%26aufirst%3DN.%26aulast%3DSabile%26aufirst%3DA.%26aulast%3DGajardo%26aufirst%3DR.%26aulast%3DPodevin%26aufirst%3DP.%26aulast%3DAbadie%26aufirst%3DA.%26aulast%3DBlaton%26aufirst%3DM.%2BA.%26aulast%3DKremsdorf%26aufirst%3DD.%26aulast%3DBeretta%26aufirst%3DL.%26aulast%3DBrechot%26aufirst%3DC.%26atitle%3DActivation%2520of%2520the%2520interferon-inducible%2520protein%2520kinase%2520PKR%2520by%2520hepatocellular%2520carcinoma%2520derived-hepatitis%2520C%2520virus%2520core%2520protein%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D5836%26epage%3D5845%26doi%3D10.1038%2Fsj.onc.1204744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peel, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottrell, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellerby, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bredesen, D. E.</span></span> <span> </span><span class="NLM_article-title">Double-stranded RNA-dependent protein kinase, PKR, binds preferentially to Huntington’s disease (HD) transcripts and is activated in HD tissue</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1531</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1093/hmg/10.15.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fhmg%2F10.15.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=11468270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvFOitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2001&pages=1531-1538&author=A.+L.+Peelauthor=R.+V.+Raoauthor=B.+A.+Cottrellauthor=M.+R.+Haydenauthor=L.+M.+Ellerbyauthor=D.+E.+Bredesen&title=Double-stranded+RNA-dependent+protein+kinase%2C+PKR%2C+binds+preferentially+to+Huntington%E2%80%99s+disease+%28HD%29+transcripts+and+is+activated+in+HD+tissue&doi=10.1093%2Fhmg%2F10.15.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Double-stranded RNA-dependent protein kinase, PKR, binds preferentially to Huntington's disease (HD) transcripts and is activated in HD tissue</span></div><div class="casAuthors">Peel, Alyson L.; Rao, Ram V.; Cottrell, Barbara A.; Hayden, Michael R.; Ellerby, Lisa M.; Bredesen, Dale E.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1531-1538</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Fourteen neurol. diseases have been assocd. with the expansion of trinucleotide repeat regions.  These diseases have been categorized into those that give rise to the translation of toxic polyglutamine proteins and those that are untranslated.  Thus far, compelling evidence has not surfaced for the inclusion of a model in which a common mechanism may participate in the pathobiol. of both translated and untranslated trinucleotide diseases.  In these studies we show that a double-stranded RNA-binding protein, PKR, which has previously been linked to virally-induced and stress-mediated apoptosis, preferentially binds mutant huntingtin RNA transcripts immobilized on streptavidin columns that have been incubated with human brain exts.  These studies also show, by immunodetection in tissue slices, that PKR is present in its activated form in both human Huntington autopsy material and brain tissue derived from Huntington yeast artificial chromosome transgenic mice.  The increased immunolocalization of the activated kinase is more pronounced in areas most affected by the disease and, coupled with the RNA binding results, suggests a role for PKR activation in the disease process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonZ2uuXE1HwbVg90H21EOLACvtfcHk0livFzMKz8CVmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvFOitb0%253D&md5=80bcd3ad64f855e415bcbf5b8b42e2cd</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F10.15.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F10.15.1531%26sid%3Dliteratum%253Aachs%26aulast%3DPeel%26aufirst%3DA.%2BL.%26aulast%3DRao%26aufirst%3DR.%2BV.%26aulast%3DCottrell%26aufirst%3DB.%2BA.%26aulast%3DHayden%26aufirst%3DM.%2BR.%26aulast%3DEllerby%26aufirst%3DL.%2BM.%26aulast%3DBredesen%26aufirst%3DD.%2BE.%26atitle%3DDouble-stranded%2520RNA-dependent%2520protein%2520kinase%252C%2520PKR%252C%2520binds%2520preferentially%2520to%2520Huntington%25E2%2580%2599s%2520disease%2520%2528HD%2529%2520transcripts%2520and%2520is%2520activated%2520in%2520HD%2520tissue%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2001%26volume%3D10%26spage%3D1531%26epage%3D1538%26doi%3D10.1093%2Fhmg%2F10.15.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0livFzMKz8CVmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggill, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhardt, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Punta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangrador-Vegas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salazar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, A.</span></span> <span> </span><span class="NLM_article-title">The PFAM protein families database: Towards a more sustainable future</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D279</span>– <span class="NLM_lpage">D285</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkv1344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=26673716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D279-D285&author=R.+D.+Finnauthor=P.+Coggillauthor=R.+Y.+Eberhardtauthor=S.+R.+Eddyauthor=J.+Mistryauthor=A.+L.+Mitchellauthor=S.+C.+Potterauthor=M.+Puntaauthor=M.+Qureshiauthor=A.+Sangrador-Vegasauthor=G.+A.+Salazarauthor=J.+Tateauthor=A.+Bateman&title=The+PFAM+protein+families+database%3A+Towards+a+more+sustainable+future&doi=10.1093%2Fnar%2Fgkv1344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfam protein families database: towards a more sustainable future</span></div><div class="casAuthors">Finn, Robert D.; Coggill, Penelope; Eberhardt, Ruth Y.; Eddy, Sean R.; Mistry, Jaina; Mitchell, Alex L.; Potter, Simon C.; Punta, Marco; Qureshi, Matloob; Sangrador-Vegas, Amaia; Salazar, Gustavo A.; Tate, John; Bateman, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D279-D285</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In the last two years the Pfam database (http://pfam. xfam.org) has undergone a substantial reorganization to reduce the effort involved in making a release, thereby permitting more frequent releases.  Arguably the most significant of these changes is that Pfam is now primarily based on the UniProtKB ref. proteomes, with the counts of matched sequences and species reported on the website restricted to this smaller set.  Building families on ref. proteomes sequences brings greater stability, which decreases the amt. of manual curation required to maintain them.  It also reduces the no. of sequences displayed on the website, while still providing access to many important model organisms.  Matches to the full UniProtKB database are, however, still available and Pfam annotations for individual UniProtKB sequences can still be retrieved.  Some Pfam entries (1.6%) which have no matches to ref. proteomes remain; we are working with UniProt to see if sequences from them can be incorporated into ref. proteomes.  Pfam-B, the automatically-generated supplement to Pfam, has been removed.  The current release (Pfam 29.0) includes 16 295 entries and 559 clans.  The facility to view the relationship between families within a clan has been improved by the introduction of a new tool.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzdSs831JgtbVg90H21EOLACvtfcHk0livFzMKz8CVmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7nF&md5=8c52b94dbbb0adab77938456c64cfc49</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1344%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DR.%2BD.%26aulast%3DCoggill%26aufirst%3DP.%26aulast%3DEberhardt%26aufirst%3DR.%2BY.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DMistry%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DA.%2BL.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DPunta%26aufirst%3DM.%26aulast%3DQureshi%26aufirst%3DM.%26aulast%3DSangrador-Vegas%26aufirst%3DA.%26aulast%3DSalazar%26aufirst%3DG.%2BA.%26aulast%3DTate%26aufirst%3DJ.%26aulast%3DBateman%26aufirst%3DA.%26atitle%3DThe%2520PFAM%2520protein%2520families%2520database%253A%2520Towards%2520a%2520more%2520sustainable%2520future%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD279%26epage%3DD285%26doi%3D10.1093%2Fnar%2Fgkv1344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">The UniProt Consortium</span> <span> </span><span class="NLM_article-title">UniProt: The universal protein knowledgebase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The+UniProt+Consortium&title=UniProt%3A+The+universal+protein+knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%2520The%2520universal%2520protein%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">The Gene Ontology
Consortium</span> <span> </span><span class="NLM_article-title">Expansion of the
gene ontology knowledgebase and resources</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D331</span>– <span class="NLM_lpage">D338</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkw1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27899567" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D331-D338&author=The+Gene+Ontology%0AConsortium&title=Expansion+of+the%0Agene+ontology+knowledgebase+and+resources&doi=10.1093%2Fnar%2Fgkw1108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1108%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DExpansion%2520of%2520the%250Agene%2520ontology%2520knowledgebase%2520and%2520resources%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD331%26epage%3DD338%26doi%3D10.1093%2Fnar%2Fgkw1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battistuz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluhm, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iype, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marvin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.1107/s0907444902003451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1107%2Fs0907444902003451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=12037327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=899-907&author=H.+M.+Bermanauthor=T.+Battistuzauthor=T.+N.+Bhatauthor=W.+F.+Bluhmauthor=P.+E.+Bourneauthor=K.+Burkhardtauthor=Z.+Fengauthor=G.+L.+Gillilandauthor=L.+Iypeauthor=S.+Jainauthor=P.+Faganauthor=J.+Marvinauthor=D.+Padillaauthor=V.+Ravichandranauthor=B.+Schneiderauthor=N.+Thankiauthor=H.+Weissigauthor=J.+D.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1107%2Fs0907444902003451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Battistuz, Tammy; Bhat, T. N.; Bluhm, Wolfgang F.; Bourne, Philip E.; Burkhardt, Kyle; Feng, Zukang; Gilliland, Gary L.; Iype, Lisa; Jain, Shri; Fagan, Phoebe; Marvin, Jessica; Padilla, David; Ravichandran, Veerasamy; Schneider, Bohdan; Thanki, Narmada; Weissig, Helge; Westbrook, John D.; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">6, No. 1</span>),
    <span class="NLM_cas:pages">899-907</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Protein Data Bank [PDB; Berman, Westbrook et al. (2000), Nucleic Acids Res. 28, 235-242; http://www.pdb.org/] is the single worldwide archive of primary structural data of biol. macromols.  Many secondary sources of information are derived from PDB data.  It is the starting point for studies in structural bioinformatics.  This article describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information and plans for the future development of the resource.  The reader should come away with an understanding of the scope of the PDB and what is provided by the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUFcMyys-PrVg90H21EOLACvtfcHk0lgCLoFpcfYbGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D&md5=d7594ad00ec8e5bacc413d38bceaa360</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2Fs0907444902003451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444902003451%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBattistuz%26aufirst%3DT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBluhm%26aufirst%3DW.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBurkhardt%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%2BL.%26aulast%3DIype%26aufirst%3DL.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DFagan%26aufirst%3DP.%26aulast%3DMarvin%26aufirst%3DJ.%26aulast%3DPadilla%26aufirst%3DD.%26aulast%3DRavichandran%26aufirst%3DV.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DThanki%26aufirst%3DN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D899%26epage%3D907%26doi%3D10.1107%2Fs0907444902003451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashford, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orengo, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sillitoe, I.</span></span> <span> </span><span class="NLM_article-title">CATH: An expanded resource to predict protein function through structure and sequence</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D289</span>– <span class="NLM_lpage">D295</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1093%2Fnar%2Fgkw1098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=27899584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D289-D295&author=N.+L.+Dawsonauthor=T.+E.+Lewisauthor=S.+Dasauthor=J.+G.+Leesauthor=D.+Leeauthor=P.+Ashfordauthor=C.+A.+Orengoauthor=I.+Sillitoe&title=CATH%3A+An+expanded+resource+to+predict+protein+function+through+structure+and+sequence&doi=10.1093%2Fnar%2Fgkw1098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">CATH: an expanded resource to predict protein function through structure and sequence</span></div><div class="casAuthors">Dawson, Natalie L.; Lewis, Tony E.; Das, Sayoni; Lees, Jonathan G.; Lee, David; Ashford, Paul; Orengo, Christine A.; Sillitoe, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D289-D295</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The latest version of the CATH-Gene3D protein structure classification database has recently been released (version 4.1, http://www.cathdb.info).  The resource comprises over 300 000 domain structures and over 53 million protein domains classified into 2737 homologous superfamilies, doubling the no. of predicted protein domains in the previous version.  The daily-updated CATH-B, which contains our very latest domain assignment data, provides putative classifications for over 100 000 addnl. protein domains.  This article describes developments to the CATH-Gene3D resource over the last two years since the publication in 2015, including: significant increases to our structural and sequence coverage; expansion of the functional families in CATH; building a support vector machine (SVM) to automatically assign domains to superfamilies; improved search facilities to return alignments of query sequences against multiple sequence alignments; the redesign of the web pages and download site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsFexQmzV7KbVg90H21EOLACvtfcHk0lgCLoFpcfYbGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhu7k%253D&md5=0f0181b39280024e816e8baa299859c1</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1098%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DN.%2BL.%26aulast%3DLewis%26aufirst%3DT.%2BE.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DLees%26aufirst%3DJ.%2BG.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DAshford%26aufirst%3DP.%26aulast%3DOrengo%26aufirst%3DC.%2BA.%26aulast%3DSillitoe%26aufirst%3DI.%26atitle%3DCATH%253A%2520An%2520expanded%2520resource%2520to%2520predict%2520protein%2520function%2520through%2520structure%2520and%2520sequence%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD289%26epage%3DD295%26doi%3D10.1093%2Fnar%2Fgkw1098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burlingame, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolan, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Turning a protein kinase on or off from a single allosteric site via disulfide trapping</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">6056</span>– <span class="NLM_lpage">6061</span>, <span class="refDoi"> DOI: 10.1073/pnas.1102376108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.1073%2Fpnas.1102376108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=21430264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVymtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=6056-6061&author=J.+D.+Sadowskyauthor=M.+A.+Burlingameauthor=D.+W.+Wolanauthor=C.+L.+McClendonauthor=M.+P.+Jacobsonauthor=J.+A.+Wells&title=Turning+a+protein+kinase+on+or+off+from+a+single+allosteric+site+via+disulfide+trapping&doi=10.1073%2Fpnas.1102376108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Turning a protein kinase on or off from a single allosteric site via disulfide trapping</span></div><div class="casAuthors">Sadowsky, Jack D.; Burlingame, Mark A.; Wolan, Dennis W.; McClendon, Christopher L.; Jacobson, Matthew P.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6056-6061, S6056/1-S6056/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">There is significant interest in identifying and characterizing allosteric sites in enzymes such as protein kinases both for understanding allosteric mechanisms as well as for drug discovery.  Here, we apply a site-directed technol., disulfide trapping, to interrogate structurally and functionally how an allosteric site on the Ser/Thr kinase, 3-phosphoinositide-dependent kinase 1 (PDK1) - the PDK1-interacting-fragment (PIF) pocket - is engaged by an activating peptide motif on downstream substrate kinases (PIFtides) and by small mol. fragments.  By monitoring pairwise disulfide conjugation between PIFtide and PDK1 cysteine mutants, we defined the PIFtide binding orientation in the PIF pocket of PDK1 and assessed subtle relationships between PIFtide positioning and kinase activation.  We also discovered a variety of small mol. fragment disulfides (<300 Da) that could either activate or inhibit PDK1 by conjugation to the PIF pocket, thus displaying greater functional diversity than is displayed by PIFtides conjugated to the same sites.  Biochem. data and three crystal structures provided insight into the mechanism of action of the best fragment activators and inhibitors.  These studies show that disulfide trapping is useful for characterizing allosteric sites on kinases and that a single allosteric site on a protein kinase can be exploited for both activation and inhibition by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfPsWN5mhicLVg90H21EOLACvtfcHk0ljGk3-Oj_J5GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVymtb8%253D&md5=dab9d4429539ffc25153e9938a7d958c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102376108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102376108%26sid%3Dliteratum%253Aachs%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DBurlingame%26aufirst%3DM.%2BA.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DTurning%2520a%2520protein%2520kinase%2520on%2520or%2520off%2520from%2520a%2520single%2520allosteric%2520site%2520via%2520disulfide%2520trapping%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D6056%26epage%3D6061%26doi%3D10.1073%2Fpnas.1102376108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A.</span></span> <span> </span><span class="NLM_article-title">UCSF Small Molecule Discovery Center: Innovation, collaboration and chemical biology in the Bay area</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.2174/1386207317666140323133841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=10.2174%2F1386207317666140323133841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=24661212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1elsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2014&pages=333-342&author=M.+Arkinauthor=K.+Angauthor=S.+Chenauthor=J.+Daviesauthor=C.+Merronauthor=Y.+Tangauthor=C.+Wilsonauthor=A.+Renslo&title=UCSF+Small+Molecule+Discovery+Center%3A+Innovation%2C+collaboration+and+chemical+biology+in+the+Bay+area&doi=10.2174%2F1386207317666140323133841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Small Molecule Discovery Center: Innovation, Collaboration and Chemical Biology in the Bay Area</span></div><div class="casAuthors">Arkin, Michelle R.; Ang, Kenny K. H.; Chen, Steven; Davies, Julia; Merron, Connie; Tang, Yinyan; Wilson, Christopher G. M.; Renslo, Adam R.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">333-342</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The Small Mol. Discovery Center (SMDC) at the University of California, San Francisco, works collaboratively with the scientific community to solve challenging problems in chem. biol. and drug discovery.  The SMDC includes a high throughput screening facility, medicinal chem., and research labs focused on fundamental problems in biochem. and targeted drug delivery.  Here, we outline our HTS program and provide examples of chem. tools developed through SMDC collaborations.  We have an active research program in developing quant. cell-based screens for primary cells and whole organisms; here, we describe whole-organism screens to find drugs against parasites that cause neglected tropical diseases.  We are also very interested in target-based approaches for so-called "undruggable", protein classes and fragment-based lead discovery.  This expertise has led to several pharmaceutical collaborations; addnl., the SMDC works with start-up companies to enable their early-stage research.  The SMDC, located in the biotech-focused Mission Bay neighborhood in San Francisco, is a hub for innovative small-mol. discovery research at UCSF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFwj4Qi0lusrVg90H21EOLACvtfcHk0ljGk3-Oj_J5GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1elsr8%253D&md5=c0d76cd40afc12119d6a5b888bf7e229</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.2174%2F1386207317666140323133841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207317666140323133841%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%26aulast%3DAng%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DMerron%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DRenslo%26aufirst%3DA.%26atitle%3DUCSF%2520Small%2520Molecule%2520Discovery%2520Center%253A%2520Innovation%252C%2520collaboration%2520and%2520chemical%2520biology%2520in%2520the%2520Bay%2520area%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2014%26volume%3D17%26spage%3D333%26epage%3D342%26doi%3D10.2174%2F1386207317666140323133841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IEP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IEP','PDB','1IEP'); return false;">PDB: 1IEP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FVR','PDB','1FVR'); return false;">PDB: 1FVR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2','PDB','4AN2'); return false;">PDB: 4AN2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EQF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EQF','PDB','3EQF'); return false;">PDB: 3EQF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RQK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RQK','PDB','4RQK'); return false;">PDB: 4RQK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IOP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IOP','PDB','3IOP'); return false;">PDB: 3IOP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5V','PDB','3K5V'); return false;">PDB: 3K5V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QRJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QRJ','PDB','3QRJ'); return false;">PDB: 3QRJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1UKI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1UKI','PDB','1UKI'); return false;">PDB: 1UKI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z9L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z9L','PDB','4Z9L'); return false;">PDB: 4Z9L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O2M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O2M','PDB','3O2M'); return false;">PDB: 3O2M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3V','PDB','3V3V'); return false;">PDB: 3V3V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW','PDB','3NEW'); return false;">PDB: 3NEW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S4Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S4Q','PDB','3S4Q'); return false;">PDB: 3S4Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVS','PDB','3JVS'); return false;">PDB: 3JVS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RVK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RVK','PDB','4RVK'); return false;">PDB: 4RVK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RFE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RFE','PDB','2RFE'); return false;">PDB: 2RFE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RJ5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RJ5','PDB','4RJ5'); return false;">PDB: 4RJ5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CMK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CMK','PDB','1CMK'); return false;">PDB: 1CMK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AE9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AE9','PDB','4AE9'); return false;">PDB: 4AE9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P','PDB','4C3P'); return false;">PDB: 4C3P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O51" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O51','PDB','3O51'); return false;">PDB: 3O51</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALO','PDB','3ALO'); return false;">PDB: 3ALO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ALN','PDB','3ALN'); return false;">PDB: 3ALN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FO0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FO0','PDB','2FO0'); return false;">PDB: 2FO0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3W2R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3W2R','PDB','3W2R'); return false;">PDB: 3W2R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QTU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QTU','PDB','3QTU'); return false;">PDB: 3QTU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V6R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V6R','PDB','3V6R'); return false;">PDB: 3V6R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AV4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AV4','PDB','5AV4'); return false;">PDB: 5AV4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RA5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RA5','PDB','4RA5'); return false;">PDB: 4RA5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UIU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UIU','PDB','3UIU'); return false;">PDB: 3UIU</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i33"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00089">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_46200"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00089">10.1021/acs.jmedchem.9b00089</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Diagram representing the definition of the 12 putative allosteric sites; probe molecules used by FTMap; flowchart of developing Kinase Atlas; and tethering, a disulfide-based fragment screening method (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00089/suppl_file/jm9b00089_si_001.pdf">jm9b00089_si_001.pdf (340.33 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00089&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00089%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-14%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00089" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a7e346ac93c16","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
