
<html lang="en"     class="pb-page"  data-request-id="cbd1f22d-adaa-420c-b84c-18277c826b6d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.5b01684;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2016.59.issue-9;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Role of Molecular Dynamics and Related Methods in Drug Discovery" /></meta><meta name="dc.Creator" content="Marco  De Vivo" /></meta><meta name="dc.Creator" content="Matteo  Masetti" /></meta><meta name="dc.Creator" content="Giovanni  Bottegoni" /></meta><meta name="dc.Creator" content="Andrea  Cavalli" /></meta><meta name="dc.Description" content="Molecular dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery. Their main advantage is in explicitly treating structural flexibility and entropic ..." /></meta><meta name="Description" content="Molecular dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery. Their main advantage is in explicitly treating structural flexibility and entropic ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 8, 2016" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01684" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01684" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01684" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01684" /></link>
        
    
    

<title>Role of Molecular Dynamics and Related Methods in Drug Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01684" /></meta><meta property="og:title" content="Role of Molecular Dynamics and Related Methods in Drug Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0007.jpeg" /></meta><meta property="og:description" content="Molecular dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery. Their main advantage is in explicitly treating structural flexibility and entropic effects. This allows a more accurate estimate of the thermodynamics and kinetics associated with drug–target recognition and binding, as better algorithms and hardware architectures increase their use. Here, we review the theoretical background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug–target binding. We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand–target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy. Further issues discussed include allosteric modulation and the role of water molecules in ligand binding and optimization. We conclude by calling for more prospective studies to attest to these methods’ utility in discovering novel drug candidates." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01684"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01684">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01684&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01684&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01684&amp;href=/doi/10.1021/acs.jmedchem.5b01684" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4035-4061</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00511" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.5b01746" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marco++De+Vivo">Marco De Vivo</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matteo++Masetti">Matteo Masetti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giovanni++Bottegoni">Giovanni Bottegoni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Cavalli">Andrea Cavalli</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Laboratory of Molecular Modeling and Drug Discovery, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genova, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, I-40126 Bologna, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">CompuNet, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genova, Italy</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">IAS-5/INM-9 Computational Biomedicine Forschungszentrum Jülich, Wilhelm-Johnen-Straße, 52428 Jülich, Germany</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">BiKi Technologies srl, Via XX Settembre 33/10, 16121 Genova, Italy</span></div><div class="corresp-info"><strong>*</strong>M.D.V.: e-mail, <a href="/cdn-cgi/l/email-protection#a7cac6d5c4c889c3c2d1ced1c8e7ceced389ced3"><span class="__cf_email__" data-cfemail="cea3afbcada1e0aaabb8a7b8a18ea7a7bae0a7ba">[email protected]</span></a>; phone, +39 010 71781577.</div><div class="corresp-info"><strong>*</strong>A.C.: e-mail, <a href="/cdn-cgi/l/email-protection#33525d574156521d505245525f5f5a73465d5a515c1d5a47"><span class="__cf_email__" data-cfemail="0766696375626629646671666b6b6e4772696e6568296e73">[email protected]</span></a>; phone, +39 051 2099735.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01684&amp;href=/doi/10.1021%2Facs.jmedchem.5b01684" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4035–4061</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 25, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 October 2015</li><li><span class="item_label"><b>Published</b> online</span>8 February 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01684" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01684</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01684"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">24100</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">389</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01684" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Role of Molecular Dynamics and Related Methods in Drug Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Marco&quot;,&quot;last_name&quot;:&quot;De Vivo&quot;},{&quot;first_name&quot;:&quot;Matteo&quot;,&quot;last_name&quot;:&quot;Masetti&quot;},{&quot;first_name&quot;:&quot;Giovanni&quot;,&quot;last_name&quot;:&quot;Bottegoni&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Cavalli&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4035-4061&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01684&quot;},&quot;abstract&quot;:&quot;Molecular dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery. Their main advantage is in explicitly treating structural flexibility and entropic effects. This allows a more accurate estimate of the thermodynamics and kinetics associated with drug–target recognition and binding, as better algorithms and hardware architectures increase their use. Here, we review the theoretical background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug–target binding. We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand–target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy. Further issues discussed include allosteric modulation and the role of water molecules in ligand binding and optimization. We conclude by calling for more prospective studies to atte&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01684&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01684" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01684&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01684" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01684&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01684" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01684&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01684&amp;href=/doi/10.1021/acs.jmedchem.5b01684" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01684" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01684" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26pmid%3D26807648%26genre%3Darticle%26aulast%3DDe%2BVivo%26date%3D2016%26atitle%3DRole%2Bof%2BMolecular%2BDynamics%2Band%2BRelated%2BMethods%2Bin%2BDrug%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D9%26spage%3D4035%26epage%3D4061%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290976" title="Free energy">Free energy</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=290905" title="Kinetics">Kinetics</a>,</li><li><a href="/action/doSearch?ConceptID=291707" title="Computer simulations">Computer simulations</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/jmcmar.2016.59.issue-9/20160512/jmcmar.2016.59.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Molecular dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery. Their main advantage is in explicitly treating structural flexibility and entropic effects. This allows a more accurate estimate of the thermodynamics and kinetics associated with drug–target recognition and binding, as better algorithms and hardware architectures increase their use. Here, we review the theoretical background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug–target binding. We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand–target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy. Further issues discussed include allosteric modulation and the role of water molecules in ligand binding and optimization. We conclude by calling for more prospective studies to attest to these methods’ utility in discovering novel drug candidates.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/9"><issue-title>Computational Methods for Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01430" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01430" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Computational drug discovery can accelerate the challenging process of designing and optimizing a new drug candidate.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The impact of computational structure-based drug design (SBDD) on drug discovery has intensified in the past decade because of the rapid development of faster architectures and better algorithms for high-level computations in a time-affordable manner.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Classical molecular dynamics (MD) simulations nowadays allow implementation of SBDD strategies that fully account for structural flexibility of the overall drug–target model system.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Indeed, it is now widely accepted that the two major drug-binding paradigms (induced-fit and conformational selection) have superseded Emil Fischer’s rigid lock-and-key binding paradigm.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5-7)</a> Receptor and ligand flexibility are crucial for correctly predicting drug binding and related thermodynamic and kinetic properties.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> As a result, classical MD is no longer considered prohibitive for effective drug design. Instead, it is pushing the frontiers of computationally driven drug discovery in both academia and industry.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">Classical MD is a physical method for studying the interaction and motion of atoms and molecules according to Newton’s physics. A force field is used to estimate the forces between interacting atoms and calculate the overall energy of the system. Then, during MD simulations, the integration of Newton’s laws of motions generates successive configurations of the evolving system, providing trajectories that specify positions and velocities of the particles over time. From these MD trajectories, a variety of properties can be calculated, including free energy, kinetics measures, and other macroscopic quantities, which can be compared with experimental observables. The method was originally conceived within theoretical physics in the late 1950s but is now applied in chemical physics, materials science, modeling of biomolecules, and more recently, drug discovery.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a></div><div class="NLM_p">Pioneering studies by Karplus and McCammon<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and by Warshel and Levitt<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> showed the crucial role of classical MD simulations in studying biological systems. They used MD simulations to obtain different conformations of proteins and nucleic acids, including early attempts to simulate spontaneously complex phenomena, such as protein folding.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> In recent decades, researchers have increasingly recognized that MD can also overcome the major limitations of static structure-based drug design, such as those limitations that characterize routinely applied ligand docking calculations, which do not sample the major protein conformational rearrangements often observed during ligand binding.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a></div><div class="NLM_p">The issue of structural flexibility in SBDD was first addressed using advance molecular docking protocols, which permitted partial flexibility of the receptor when screening compound libraries.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In the past decade, for example, using multiple pre-existing conformations of a target has become widely accepted as a way to partially account for protein flexibility in SBDD.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> One notable example is ensemble docking, where several independent protein conformations are targeted. The results are eventually merged together and averaged, leading to improved outcomes.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In addition to using experimental techniques to structurally characterize the receptor in ensemble docking,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> MD simulations were an obvious way to obtain multiple conformations of macromolecular targets.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> As early as 1994, Pang and Kozikowski extracted multiple conformations of the acetylcholinesterase enzyme from a 40 ps trajectory. They used these to successfully predict, through rigid docking, the bound pose of huperzine A.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Since then, numerous research groups have used snapshots extracted from MD trajectories to provide a discrete representation of the target plasticity.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The relaxed complex method developed by McCammon et al.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> is a particularly notable example. In the past decade, MD protocols for ensemble docking have shown great potential in handling targets governed by significant structural flexibility. These include protein kinases<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and G-protein-coupled receptors,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> whose function is characterized by remarkable conformational plasticity.</div><div class="NLM_p">Moving beyond protocols that use MD to incorporate target flexibility into standard docking calculations, it is now possible to run MD simulations for long enough to explore the free-energy landscape and kinetic profile associated with the overall drug-binding process (i.e., from the drug fully solvated in water to the drug–target bound state).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Today, a full dynamical description of the protein–ligand binding event can be obtained, with various degrees of accuracy. This is due to increasing computer power, the advent of graphical processor unit (GPU) architectures, and MD software that can efficiently run on these innovative hardware infrastructures. In pioneering studies by Buch et al. and Shan et al.,<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> for example, a ligand binding to a target protein was investigated by multiple replicas of microsecond-long MD simulations. More recently, Decherchi et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> used a similar protocol in combination with machine learning algorithms to identify different routes to detect binding and to generate accurate free-energy profiles associated with these routes.</div><div class="NLM_p">Although simulations lasting up to a few milliseconds (corresponding to 10<sup>12</sup> time steps) are now possible,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> several different trajectories are recommended to obtain adequate statistics and exhaustive sample of the conformational space. Thus, even when studying a single lead compound, the process is demanding. It is also common to find druglike molecules with long unbinding kinetics (in the order of several minutes).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Conventional MD methods, even if running on specialized hardware, still cannot describe such slow unbinding events. In fast-paced drug discovery programs, this is the major issue limiting the use of MD-based simulations for kinetic prediction.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, the sampling issue has led to the development of many innovative algorithms that form the basis of “enhanced sampling methods”. These speed up the description of <i>slow</i> processes, accelerating the rare events characterized by high-in-free-energy states.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Notable enhanced sampling methods (developed over the past 2 decades and now widely used) include free-energy perturbation,<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> umbrella sampling,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> replica exchange,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> metadynamics,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> steered MD,<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> accelerated MD,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> milestoning,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> transition-path sampling,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and their many possible combinations. Researchers have recently demonstrated the power of these methods for studying protein–ligand binding and estimating the associated free energy and kinetics.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Recent studies have also shown how various implementations of MD can effectively engage targets<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> that are druggable but that do not appear to be very ligandable.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In these challenging cases, MD has aided the discovery of entirely new (usually allosteric) pockets.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> These pockets are only transiently formed and are often too short-lived to be mapped on the protein surface by conventional experimental techniques.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">Given the ever-increasing role played by MD in the evolution of computationally driven drug discovery, we will here discuss a few theoretical concepts that form the basis of classical MD, focusing on those most related to drug discovery. We review some of the most recent and effective applications of plain MD for studying protein–ligand binding and unbinding and for retrieving major thermodynamic data such as binding free energy. We move on to examples where enhanced sampling methods combine with MD to characterize ligand–target conformational and energetic landscapes. Again, we focus on those methods most widely applied in drug discovery, such as free-energy perturbation, metadynamics, and steered MD.</div><div class="NLM_p">This Perspective will also touch on the emerging and critical role of MD simulations and related methods in predicting the binding and unbinding kinetics of drugs, since these physicochemical observables play an increasingly central role in drug design and optimization. We close by outlining how MD simulations can help in understanding and using allosteric mechanisms for drug design and in examining the thermodynamic properties of the water molecules that solvate protein-binding sites, which may help in designing new and more potent lead compounds.</div><div class="NLM_p last">Overall, we provide an overview of the state of the art of classical MD in pharmaceutical research, highlighting key applications to drug design from recent years. We envisage a not-too-distant future wherein MD and related approaches are routinely used for the in silico screening of large libraries of small molecules, accelerating drug candidate identification and optimization. More generally, we anticipate the wider utilization of MD-based methods in pharmaceutical endeavors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">2 Theoretical Background</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23125" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23125" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3">2.1 Molecular Dynamics with Classical Potentials</h3><div class="NLM_p">The central idea behind MD simulations is to study the time-dependent behavior of microscopic systems. This is obtained by solving the second-order differential equations represented by Newton’s second law:<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>where <b>f</b><sub><i>i</i></sub>(<i>t</i>) is the net force acting on the <i>i</i>th atom of the system at a given point in time <i>t</i>, <b>a</b><sub><i>i</i></sub>(<i>t</i>) is the corresponding acceleration, and <i>m</i><sub><i>i</i></sub> is the mass. In <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>, the instantaneous configuration of the system is represented by the vector <b>x</b>(<i>t</i>), which describes the position of the <i>N</i> interacting atoms in the Cartesian space (<b>x</b> = {<i>x</i><sub>1</sub>, <i>y</i><sub>1</sub>, <i>z</i><sub>1</sub>, <i>x</i><sub>2</sub>, <i>y</i><sub>2</sub>, <i>z</i><sub>2</sub>, ..., <i>x</i><sub><i>N</i></sub>, <i>y</i><sub><i>N</i></sub>, <i>z</i><sub><i>N</i></sub>}). Usually, in computational drug discovery, we adopt a classical mechanics description of the forces. Notably, this approximation holds for massive particles such as nuclei, while the electron motions must be averaged out. To achieve this, an empirical potential energy function is introduced (<i>V</i>(<b>x</b>) in <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>), and the model arising from this simplified representation is referred to as the force field (FF), or molecular mechanics (MM):<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p">In <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>, the first three terms represent intramolecular interactions of the atoms. They describe variations in potential energy as a function of bond stretching, bending, and torsions between atoms directly involved in bonding relationships. They are represented by summations over bond lengths (<i>l</i>), angles (α), and dihedral angles (θ), respectively. Bond stretching and bending contributions share the same functional form, as they are both described by harmonic potentials with reference values <i>l</i><sub>0</sub> and α<sub>0</sub> and force constants <i>k</i><sub><i>l</i></sub> and <i>k</i><sub>α</sub>, respectively. However, because of their intrinsic periodicity, torsional terms are naturally defined by a cosine series of <i>M</i> terms for each dihedral angle. Thus, <i>n</i><sub><i>ik</i></sub> is a parameter describing the multiplicity for the <i>k</i>th term of the series, θ<sub>0,<i>ik</i></sub> is the corresponding phase angle, and <i>V</i><sub><i>ik</i></sub> is the energy barrier. Taken as a whole, this group of contributions is usually referred to as the “bonded” terms of the FF.</div><div class="NLM_p">The fourth and fifth terms in <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a> represent van der Waals and electrostatic interactions between atoms, respectively, and are denoted as “nonbonded” terms. These contributions act on every pair of atoms in the system that is not already covered by the bonded counterpart. The energy is expressed as an inverse power function of the distance between the considered atoms, <i>r</i><sub><i>ij</i></sub>. The van der Waals interactions are generally treated with a 12–6 Lennard-Jones potential, where ε<sub><i>ij</i></sub> is a parameter defining the depth of the energy well, whereas <i>r</i><sub>0,<i>ij</i></sub> is the minimum energy distance that equals the sum of the van der Waals radii of the two interacting atoms. Finally, the electrostatic energy is described with the Coulomb potential, where <i>q</i><sub><i>i</i></sub> and <i>q</i><sub><i>j</i></sub> are the partial charges of a pair of atoms, ε<sub>0</sub> stands for the permittivity of free space, and ε<sub>r</sub> is the relative permittivity (or dielectric constant), which takes a value of 1 in vacuum.</div><div class="NLM_p">The major advantage of FFs is that they speed up calculations considerably, especially compared to quantum mechanics (QM). QM would be a more accurate theory for treating molecular-sized systems,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> but it is intrinsically difficult to solve the Schrödinger equation for many interacting particles. This prevents QM from being applied practically to studying very large systems of several thousand atoms. Lately, however, accurate QM-based approaches have found increasing applicability in drug discovery.<a onclick="showRef(event, 'ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53">(51-53)</a></div><div class="NLM_p">In this context, FFs have a long history of success, permitting extended MD simulations of large biomolecular systems, despite some intrinsic limitations (see below). FFs widely used nowadays for biomolecular simulations (condensed-phase FFs) include AMBER,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> CHARMM,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and OPLS.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> These have attained such a high standard of quality that the preference for one over the other is often dictated by practical considerations only, related to their implementation with the MD engine of choice. In addition to amino acids, parameters for nucleic acids, lipids, carbohydrates, and several ionic species have been included in the parent FFs in recent years by suitably extending the original parametrization. However, the structural variability of small molecules (i.e., ligands) has seriously challenged the condensed-phase FFs. To get around this, the user must often supply specific parametrizations. Much effort has been devoted to simplifying and automating this time-consuming and error-prone procedure, including the development of general force field sets for organic compounds (e.g., GAFF<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> for AMBER, and CGenFF<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> for CHARMM) and specific parametrization toolkits. Among the latter we mention the Antechamber program,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Paramfit,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and Hopkins and Roitberg’s proposed procedure for AMBER,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> while the general methodology GAAMP can be used to improve parametrizations relying on both GAFF and CGenFF libraries, with the added value of identifying, scanning, and optimizing all soft dihedral parameters of small molecules according to QM data.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Another promising general strategy is the systematic parametrization method provided by ForceBalance, which is able to derive parameters by combining theoretical and experimental data in a flexible way.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> But QM-derived partial atomic charges and dihedral angle potentials should still be checked carefully when dealing with nonstandard small molecules.</div><div class="NLM_p">Although FFs can reduce the computational burden, <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a> can still be analytically solved for only a few atoms. For biological macromolecule simulations, numerical methods must be used to split the integration of the equations of motion into discrete time intervals, called time-steps, δ<i>t</i>. In doing so, forces are assumed to be constants during each integration step. However, as reported in <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>, forces depend upon atomic positions that change over time. Thus, a small δ<i>t</i> guarantees reliable forces over time. In practice, with appropriate workarounds, a time-step of 1 or 2 fs can be safely considered as a good compromise between calculation accuracy and efficiency. This is commonly achieved, for example, with the SHAKE and related algorithms.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Here, the equations of motions are solved while still satisfying the geometric constraint for the fastest degrees of freedom (i.e., the stretching and bending of bonds involving hydrogen atoms). Other strategies envision, for example, repartitioning the mass of heavy atoms onto the neighboring hydrogen atoms, allowing for a time-step increase of up to 4 fs.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> As an example of an integrator, the velocity-Verlet<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> is a simple and widely used algorithm in MD codes. Within such an integrator, positions at time (<i>t</i> + δ<i>t</i>) are calculated by current positions, velocities (<b>v</b>(<i>t</i>)), and accelerations according to<span class="NLM_disp-formula" id="eq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m003.gif" alt="" /></img><span class="labelSpan">(3)</span></span></div><div class="NLM_p last">Accelerations are in turn calculated by the forces acting on each atom, that is, by taking the first derivative of the potential energy with respect to positions with the opposite sign as shown in <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>. Then, velocities are propagated as follows:<span class="NLM_disp-formula" id="eq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m004.gif" alt="" /></img><span class="labelSpan">(4)</span></span>As seen in <a class="ref internalNav" href="#eq4" aria-label="eq 4">eq 4</a>, the second term on the right-hand side corresponds to the arithmetic mean of the accelerations taken at time <i>t</i> and (<i>t</i> + δ<i>t</i>). Moreover, to calculate accelerations at the next step, positions must be updated <i>before</i> advancing velocities. As a matter of fact, to be safely used in an MD simulation, any numerical integrator must satisfy some practical and theoretical requirements. They must follow the analytical trajectory as closely as possible and must preserve the physical properties of the equations of motion, with the total energy that is constant of motion, given by the sum of the potential and the kinetic energy (<i>K</i>(<b>p</b>)):<span class="NLM_disp-formula" id="eq5"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m005.gif" alt="" /></img><span class="labelSpan">(5)</span></span>where <i>H</i>(<b>x</b>,<b>p</b>) is the classical Hamiltonian of the system that depends upon the coordinates and momenta of the particles and returns the total energy (<i>E</i>) of the system. From these considerations, it should be clear that as long as the integrator works properly, MD naturally follows the motion of a microscopic isolated system, where neither matter nor energy are exchanged with the surroundings. In other words, Newtonian dynamics sample a statistical ensemble of microstates characterized by a constant number of particles (<i>N</i>), volume (<i>V</i>), and energy (<i>NVE</i> = const, or microcanonical ensemble). Nonetheless, it is possible to better mimic actual macroscopic behavior by also controlling the system’s temperature and pressure during simulation. Constant temperature is maintained through algorithms, called thermostats,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> that allow fluctuations in kinetic energy as if the simulated system were immersed in a thermostatic bath (<i>NVT</i> = const, canonical ensemble). However, the analogy with a thermostatic bath should not be taken too literally, since heat flow is not simulated. Instead, the system temperature is forced to attain, on average, the desired macroscopic value through proper alterations of the equations of motion. Stochastic dynamics (SD) can also be used for this purpose.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Very similar considerations apply for the so-called barostat algorithms,<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> by which the pressure is controlled by opportunely scaling (isotropically or not) the system volume (<i>NPT</i> = const, isothermal–isobaric ensemble). In order to better describe bulk properties with finite size systems, periodic boundary conditions (PBC) are also commonly employed in MD. With the use of PBC, the model system is placed in a unit cell that is replicated in all directions to form an infinite lattice of image atoms. In this way, coordinates and velocities are stored and propagated for the unit cell only, although the evaluation of nonbonded terms must in principle be extended to every pair of atoms in the unit cell and periodic images.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> For this reason, a spherical cutoff scheme with a radius of at least 10 Å can be used to contain the computational cost and, as such, it is employed in the calculation of the short-ranged van der Waals terms. Conversely, because of the intrinsically long-ranged nature of electrostatics interactions (decay as <i>r</i><sup>–1</sup>), the evaluation of the Coulomb energy for the entire periodic lattice is required. The calculation of full electrostatics is achieved through the Ewald sum methods, which treats differently the rapidly varying electrostatic interactions at short-range (in the real space) and the slowly decaying potential at large distances (in the reciprocal space). With the use of spherical cutoffs and an appropriate splitting parameter between short and long-range interactions, one can use the particle-mesh Ewald (PME) method<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> to greatly improve the efficiency of the computation. In this way, the point charges in reciprocal space are smeared on a regular grid, and the calculation is performed by taking advantage of fast Fourier transform.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The interested reader is encouraged to delve into the review articles and books that focus on PME.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> By use of these algorithms, MD simulations nowadays can simulate the evolution of a protein in realistic conditions for hundreds of microseconds and more. This allows researchers to observe events such as a drug binding to its target and to estimate the kinetics and free energy associated with binding.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.2 Major Limitations of MD with Classical Potentials</h3><div class="NLM_p">Here, we mention some key approximations of FF-based simulations. First, the atomic charge of each atom is fixed, disallowing any charge polarizability over time. To overcome this limitation, polarizable FFs have been developed, which can mimic, to a certain extent, the electronic redistribution in response to an external electric field (for recent reviews, see Masetti et al.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> and Shi et al.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a>). This effect may play a non-negligible role in the energetics of certain protein–ligand complexes.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Polarizable FFs are very promising; however, they remain quite computationally demanding, with their use and parametrization being less user-friendly than that of their fixed-charge counterparts. Advances in polarizable FFs will certainly widen their use in the coming decades.</div><div class="NLM_p">Another major limitation of FF-based simulations is that they cannot be used to study chemical reactivity, since chemical bonds cannot be broken or formed during MD. To address this, researchers can use methods that exploit a dual-resolution description. For example, QM/MM simulations<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> treat a very limited portion of the system with higher accuracy. This is typically the binding site of a protein where chemical reactions occur. The majority of the system (the remainder of the protein and solvent) is then treated at a less accurate level of theory. These methods have increasingly impacted drug discovery. One example is in the study of covalent inhibitors, which block the target by forming a chemical bond. During drug design, QM/MM calculations can be used to generate QM-based electrostatic potential maps of the receptor’s binding site, to determine the protonation states of key residues of the binding pocket, and to dissect reaction mechanisms of enzymes that are drug discovery targets.<a onclick="showRef(event, 'ref53 ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref53 ref74 ref75">(53, 74, 75)</a> The ReaxFF<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> is one emerging approach to studying chemical reactivity using classical MD. This FF allows chemical reactions to be studied through a geometry-dependent parametrization of reactants and products. Unfortunately, this parametrization is not simple and has only covered limited classes of chemical species to date.</div><div class="NLM_p">Because chemical reactivity is not allowed in conventional FF-based simulations, specific protonation and tautomeric states must be preassigned to all system residues (amino acids, nucleic acids, ligands, etc.). This is often referred to as the system’s topology, which is specified by the user during the setup phase and maintained throughout the simulation. Usually, assuming a neutral solution, acidic amino acids such as aspartate and glutamate can be safely modeled in their deprotonated state (negatively charged), while basic residues like lysine and arginine are modeled in their protonated state (positively charged). However, cysteine and histidine, by virtue of their p<i>K</i><sub>a</sub> (around 8–9 and 6–7, respectively), sample a population of states at physiological pH rather than being a single dominant form. For histidine, the issue is further complicated by the fact that two tautomeric forms (N<sup>ε</sup> and N<sup>δ</sup> protonated) are at equilibrium in the neutral state. In this regard, the so-called constant-pH simulations allow dynamical changes in topology during dynamics, mimicking more realistic conditions for biological model systems. Interestingly, the AMBER and CHARMM engines have recently started to support explicit solvent–constant-pH implementations to be used in conjunction with their own native FF,<a onclick="showRef(event, 'ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref77 ref78">(77, 78)</a> while an in-house version of GROMACS is available to run this kind of simulation.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> However, constant-pH simulations are generally much more expensive than ordinary setups, and a significant increase in computational power is probably required before they achieve widespread use.</div><div class="NLM_p last">In summary, the inherent approximations of FFs unavoidably introduce a systematic error into simulations. This error can always be reduced using a better (yet more computationally demanding) description of the model system. In addition, MD simulations are always affected by another approximation, related to the limited sampling allowed (far from being exhaustive). This creates a random error, which can be decreased by extending the length of the simulations. In particular, just as experiments must be replicated to assess the reliability of results, it is becoming computationally affordable to adopt this good practice when running MD simulations too. Indeed, the easiest and most general way to evaluate the statistical uncertainty is to repeat independent runs with different (uncorrelated) initial conditions and/or random seeds (e.g., different velocity distributions). More specific strategies can also be used in the context of single runs.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> However, certain types of calculations (especially free-energy estimates, see below) are often so time-consuming that, until recently, they excluded repeated simulations. Continued increases in computational power will lead to better statistics, thus improving the reliability of MD-derived findings. With these considerations in mind, we describe here how classical MD simulations can be used to make informed choices in drug discovery programs.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.3 MD-Derived Observables for Drug Discovery</h3><div class="NLM_p">Many observables can be obtained from MD runs. The principles of statistical mechanics allow quantitative estimates of important thermodynamic observables to be computed from MD trajectories. These include internal energy, pressure, and heat capacity.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> However, the key thermodynamic quantity in drug discovery is the protein–ligand binding free energy. The noncovalent association between a protein P and a ligand L in solution, to form the complex PL,<span class="NLM_disp-formula" id="eq6"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m006.gif" alt="" /></img><span class="labelSpan">(6)</span></span>is described with the equilibrium (or association) constant <i>K</i><sub>a</sub>, which is defined as<span class="NLM_disp-formula" id="eq7"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m007.gif" alt="" /></img><span class="labelSpan">(7)</span></span>where the square brackets indicate a concentration, usually expressed in molar units. The reciprocal of <a class="ref internalNav" href="#eq7" aria-label="eq 7">eq 7</a> is simply the equilibrium constant for the opposite reaction, namely, the dissociation constant (<i>K</i><sub>d</sub> = 1/<i>K</i><sub>a</sub>). Because <i>K</i><sub>d</sub> corresponds to the ligand concentration for which an equal probability of bound and unbound protein is achieved, this is used more than <i>K</i><sub>a</sub> in pharmaceutical research. Moreover, for enzyme inhibitor assays, <i>K</i><sub>d</sub> is usually replaced by the inhibition constant <i>K</i><sub>i</sub>, which has the very same chemical interpretation, so long as competitive inhibition is considered. The link between the experimentally accessible equilibrium constants and thermodynamics is provided by the standard free-energy change for binding at constant temperature and pressure:<span class="NLM_disp-formula" id="eq8"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m008.gif" alt="" /></img><span class="labelSpan">(8)</span></span>where <i>C</i>° is a constant defining the standard concentration, which is 1 M by convention (1 molecule in the volume of 1663 Å<sup>3</sup>). Importantly, multiplying the association constant by <i>C</i>°, or equivalently dividing the dissociation constant by the same amount, makes the argument of the logarithm in <a class="ref internalNav" href="#eq8" aria-label="eq 8">eq 8</a> a dimensionless number, as it should be. Since the standard binding free energy (sometimes also referred to as “absolute” binding free energy) depends on the equilibrium constant and the reference value of <i>C</i>° (see <a class="ref internalNav" href="#eq8" aria-label="eq 8">eq 8</a>), this conventional concentration must be properly considered to make meaningful comparisons between computational free-energy estimates and absolute experimental data (see below).<a onclick="showRef(event, 'ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref81 ref82">(81, 82)</a></div><div class="NLM_p last">As mentioned previously, the calculation of free-energy differences is not simple. Together with entropy-related quantities, free energy is the thermodynamic observable whose estimation suffers the most from sampling limitations. A rigorous formulation of the problem would require statistical mechanics arguments that are outside the scope of this Perspective. Therefore, we limit ourselves here to recalling that the probability of visiting microstates in the canonical ensemble is proportional to the Boltzmann factor:<span class="NLM_disp-formula" id="eq9"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m009.gif" alt="" /></img><span class="labelSpan">(9)</span></span>Because of the exponential relationship, high-in-energy configurations are much less frequently visited than low-energy states, and barriers larger than a few <i>k</i><sub>B</sub><i>T</i> units can severely hamper the efficient exploration of space, which is required to achieve reliable (or converging) free-energy estimates. For this reason, transitions between states underlying significant free-energy barriers can be viewed as <i>rare events</i> relative to the time scales accessible through MD simulations. From a statistical mechanics standpoint, <a class="ref internalNav" href="#eq8" aria-label="eq 8">eq 8</a> can be reformulated as<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a><span class="NLM_disp-formula" id="eq10"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m010.gif" alt="" /></img><span class="labelSpan">(10)</span></span>The first term in <a class="ref internalNav" href="#eq10" aria-label="eq 10">eq 10</a> represents the computational free-energy difference expressed as a probability ratio. Here, <i>p</i><sub>bound</sub>(<b>x</b>) and <i>p</i><sub>unbound</sub>(<b>x</b>) are the probabilities of observing the protein in the bound and unbound states, respectively, provided that a clear distinction between them can be achieved by defining an appropriate reaction coordinate (or collective variable, see below). However, the second term in <a class="ref internalNav" href="#eq10" aria-label="eq 10">eq 10</a> is a correction that must be introduced to retrieve the standard binding free energy. This term takes into account both the reference concentration of the standard state (<i>C</i>°) and the concentration of the interacting partners in the simulation box (<i>C</i><sup>box</sup>). This correction is only relevant for absolute comparisons with experimental data, as it cancels out when computing relative free-energy differences (ΔΔ<i>G</i><sub>b</sub>). However, reliable binding free energies through the direct use of <a class="ref internalNav" href="#eq10" aria-label="eq 10">eq 10</a> require extremely long (and possibly repeated) simulations. In principle, many binding and unbinding events must be observed to assess a statistically meaningful probability ratio between states. To date, this approach has been successful for weakly interacting solutes (solvent mapping).<a onclick="showRef(event, 'ref84 ref85'); return false;" href="javascript:void(0);" class="ref ref84 ref85">(84, 85)</a> If stronger binding is considered (approximately larger than 3–4 kcal/mol), unbiased MD is typically unable to recover transition rates, and methods based on enhanced sampling procedures can be applied.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.4 Enhanced Sampling in MD</h3><div class="NLM_p">To overcome limitations inherent to Boltzmann sampling, several theoretical methodologies have been developed. These are generally referred to as “enhanced sampling methods”, since they attempt to escape Boltzmann statistics while retaining the correct distribution of states in the given statistical ensemble.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> A large group of these methodologies exploit the fact that the free energy is a state function; thus, differences in free energy do not depend upon the path from state A to B. This means that we can transform state A into state B without worrying about the physical transition (i.e., the binding/unbinding path). From a practical standpoint, this is achieved by introducing a hybrid (or mixed) Hamiltonian, which is a function of the potential energy of states A and B through the coupling parameter λ:<span class="NLM_disp-formula" id="eq11"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m011.gif" alt="" /></img><span class="labelSpan">(11)</span></span>In <a class="ref internalNav" href="#eq11" aria-label="eq 11">eq 11</a>, the general case of linear interpolation between the two potential energy functions is shown. This is not always the best possible solution, and more sophisticated and effective mixing can be used (nonlinear interpolation schemes, the use of soft-core potentials, and the separated coupling of electrostatics and van der Waals terms). For relative free-energy differences, this can be thought of as performing a smooth and progressive transformation of a ligand A into a ligand B, in both the protein-bound and unbound states (bulk solvent). Then, the free-energy changes for the two transformations are calculated with appropriate theoretical estimators and practical precautions (stratification, forward and backward sampling, etc.), and the relative contributions are finally combined through a suitable thermodynamic cycle to recover the ΔΔ<i>G</i><sub>b</sub> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Because of the unphysical path used for the free-energy calculation, this class of enhanced sampling methods is sometimes referred to as “alchemical transformations”, although in drug design they are mostly known as “free-energy perturbation” (FEP) methods.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> FEP is commonly used to calculate the free-energy contribution associated with the vertical legs of the cycle in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. But, in principle, other approaches can be used, such as thermodynamic integration (TI). In fact, these methods can be grouped into a more general class of hybrid Hamiltonian techniques.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Thermodynamic cycle used to calculate relative binding free energies (ΔΔ<i>G</i><sub>b</sub> = Δ<i>G</i>°<sub arrange="stagger">bA</sub> – Δ<i>G</i>°<sub arrange="stagger">bB</sub>) between congeneric ligands. The horizontal legs correspond to the physical binding process, whereas vertical legs indicate the unphysical transformation of ligand <i>A</i> (blue) into ligand <i>B</i> (green) performed in bulk solvent (left) and in the protein binding site (right). FEP is used to compute the free energy associated with the vertical legs of the cycle, and the relative binding free energy is calculated as the difference between the free energy required to transform the ligands in the binding site (Δ<i>G</i><sup>p</sup>) and in solution (Δ<i>G</i><sup>w</sup>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Standard binding free energy can also be recovered using complex thermodynamic cycles and more challenging simulations (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). For example, during MD simulations, the ligand is reversibly changed into a fictitious noninteracting particle. In other words, it is decoupled from the environment (the protein and the solvent). For this reason, this theoretical framework is called the double decoupling method (DDM).<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> The method was first introduced by Jorgensen in 1988 as the double annihilation method (DAM)<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> and later formalized by Gilson et al.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> and others.<a onclick="showRef(event, 'ref87 ref88 ref89'); return false;" href="javascript:void(0);" class="ref ref87 ref88 ref89">(87-89)</a> A critical aspect of the procedure is that to avoid the “ligand wandering” problem associated with highly decoupled states, suitable restraining potentials must be introduced to keep the ligand in place in the binding site during the whole transformation (see <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). These restraints must be applied in such a way as to allow an exact recovery for the free-energy penalty due to their introduction. Moreover, during ligand decoupling, a cavity is created in the binding site that must be filled by an appropriate number of water molecules. However, this process can be slow enough to challenge the effectiveness of the whole procedure. To address this, Helms and Wade introduced a complex hybrid Hamiltonian scheme to replace the ligand with a certain amount of water molecules.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> With a more general purpose, Woods et al.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> proposed the water-swap reaction coordinate, where an equivalent volume of bulk water molecules replaces the ligand in the binding pocket. This method is better suited to describing both buried and superficial binding cavities.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Typical thermodynamic cycle used to compute absolute binding free energy differences (double decoupling method, DDM). In the simplest implementation, the cycle involves the introduction of rototranslational restraints to the bound ligand in order to avoid the “wandering ligand” problem. The restraints are depicted as a ring fastening the ligand to the protein, and their associated free-energy penalty is Δ<i>G</i><sup>p</sup><sub>rstr</sub>. Eventually, conformational restraints might also be envisioned (not shown in the picture for simplicity). Then the ligand is reversibly decoupled from the protein (or the solvent), usually, in two steps: first, electrostatics is turned off (Δ<i>G</i><sup>p</sup><sub>ele</sub> or Δ<i>G</i><sup>w</sup><sub>ele</sub>), then the van der Waals contributions are switched off (Δ<i>G</i><sup>p</sup><sub>vdW</sub> or Δ<i>G</i><sup>w</sup><sub>vdW</sub>). The “uncharged” state is represented by the ligand colored in white, whereas the transparent ligand depicts the fully decoupled state. The free-energy penalties associated with confining the decoupled ligand in the binding-site volume are calculated analytically (horizontal leg at the bottom of the cycle), and the standard state correction (Δ<i>G</i>°<sub>rstr</sub>) is usually added to this contribution.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Enhanced sampling methods based on unphysical pathways can be effectively used to compute free-energy differences, providing valuable insight into the binding thermodynamics. However, these approaches do not reveal kinetic aspects of the drug-binding process that are often of interest. Accounting for the binding kinetics requires methods that use physical pathways for free-energy calculations, through which free-energy barriers and possible intermediates along the (un)binding path can be determined.</div><div class="NLM_p">Enhanced sampling methods, which rely on physical pathways, can reconstruct the free energy of the investigated event as a function of a few reaction coordinates, usually called collective variables (CVs), able to account for relevant degrees of freedom of protein–ligand binding and unbinding. The projection of the free energy along the relevant degrees of freedom is called the potential of mean force (PMF). In principle, it can be used to extract both thermodynamic and kinetic information (see <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> In fact, we can distinguish between those methods that explicitly use CVs during sampling and those that do not. The first method class allows the exploration of rare events by biasing the MD simulations along the chosen CVs. This can be achieved in several ways, such as acting on the forces or introducing external potentials that may or may not change over time (leading to equilibrium or nonequilibrium approaches, respectively). Thus, CV-based methods require an a priori definition of the reaction coordinate to sample. In most cases, it is not simple to define a proper CV (or an ensemble thereof). Much chemical intuition and/or trial simulation is required to achieve a satisfying description.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Umbrella sampling (US)<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> is one notable equilibrium-CV-based enhanced sampling method, while steered MD<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> and metadynamics<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> are certainly the most popular nonequilibrium ones. The second method class, which does not explicitly use CV, acts by heating all the degrees of freedom of the system (or a fraction of them) at once. Here, we further underline that temperature must be understood figuratively, since the same enhancement in sampling can be obtained by actually heating the system or by opportunely scaling the Hamiltonian. This is analogous to what takes place for the enhanced sampling methods based on unphysical pathways. Temperature replica exchange MD (T-REMD, also known as “parallel tempering”)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and Hamiltonian replica exchange MD (H-REMD, also known as “solute tempering”)<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> are examples of these methodologies. The recently proposed scaled MD<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> can also be grouped in this class of techniques. These methods are devoid of any preconceived notion of the rare event and CVs. For this reason, however, they are also less specific and often less efficient than CV-based methods. Interestingly, because of their complementarity, CV-based and replica exchange methods are sometimes coupled to fully exploit the potential of both approaches (e.g., US or metadynamics combined with T/H-REMD).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Moreover, replica exchange methods can also be coupled to enhanced sampling techniques based on unphysical pathways, such as FEP.<a onclick="showRef(event, 'ref93 ref95 ref96'); return false;" href="javascript:void(0);" class="ref ref93 ref95 ref96">(93, 95, 96)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Simplified free-energy profile for the (un)binding process. The drug–receptor complex is shown as a deep free-energy minimum on the left (bound state), and the dissociated complex is represented as a higher-in-energy minimum on the right (unbound state). The thermodynamics of binding is quantified (in first approximation) by the free-energy difference between these minima (Δ<i>G</i><sub>b</sub>), and the kinetics of (un)binding is determined by the dissociation and association rate constants, <i>k</i><sub>off</sub> and <i>k</i><sub>on</sub>. These quantities are related to the free-energy differences between minima and the transition state, Δ<i>G</i><sup>⧧</sup><sub>off</sub> and Δ<i>G</i><sup>⧧</sup><sub>on</sub>, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">3 Combining Molecular Dynamics with Ligand Docking and Virtual Screening</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Ligand docking has been successfully used for drug discovery in recent years.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> It attempts to predict the three-dimensional structure and binding free energy of the complex formed by a receptor, usually a protein, and a small ligand. When applied iteratively to a library of small molecules, each member of the library is docked into the receptor, assigned a predicted binding energy, and ranked accordingly. This computational approach is called “structure-based virtual screening”.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> The topmost ranking compounds are then prioritized for further in silico studies or experimental testing.</div><div class="NLM_p">Despite its widespread and successful use,<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> docking suffers from one main drawback: the ability to handle proteins’ intrinsic flexibility in docking is either absent or limited.<a onclick="showRef(event, 'ref15 ref99'); return false;" href="javascript:void(0);" class="ref ref15 ref99">(15, 99)</a> This downside has greatly limited this technique’s prospective applicability. To address this, researchers have implemented several strategies, including soft docking,<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> rotamer libraries,<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> and local optimization of side chains.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> However, these relatively simple heuristics do not allow extended rearrangements of the protein structure, including significant conformational changes at the backbone level.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a></div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">3.1 MD for Generating Ensembles of Receptor Conformations</h3><div class="NLM_p">One practical alternative to on-the-fly modeling of receptor plasticity is the use of pre-existing multiple receptor conformations (MRC) of the binding pocket. This is facilitated by the steadily increasing number of X-ray and NMR protein structures.<a onclick="showRef(event, 'ref17 ref104'); return false;" href="javascript:void(0);" class="ref ref17 ref104">(17, 104)</a> Different experimental structures of the same receptor, if available, are used to populate the conformational ensemble used in MRC docking.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> However, the number of targets whose three-dimensional structure has been experimentally solved is limited, considering the estimated extent of the druggable genome.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> Moreover, the amount of conformational space explored by experimental structures is usually quite limited and biased toward a few known ligand–receptor complexes. While several computational techniques, including homology modeling<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> and normal modes analysis,<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> can complement experimental structures for the MRC ensemble, MD is an obvious way to generate receptor conformations.<a onclick="showRef(event, 'ref23 ref29'); return false;" href="javascript:void(0);" class="ref ref23 ref29">(23, 29)</a></div><div class="NLM_p"><a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> reports a general framework for combining molecular dynamics and MRC virtual ligand screening. While others have applied a similar workflow,<a onclick="showRef(event, 'ref28 ref109'); return false;" href="javascript:void(0);" class="ref ref28 ref109">(28, 109)</a> the relaxed complex method developed by McCammon’s group is one of the most well-known examples of this strategy for drug design.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> First, MD simulations of the selected target are used to generate a diverse set of receptor conformations, which also characterize the structural flexibility of most relevant regions of the receptor for ligand binding. Enhanced sampling methods can be used, allowing a most efficient exploration of the conformational space.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> For example, in their studies on PARP-1, Antolin et al. used temperature replica exchange MD.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Then, at regular intervals, multiple snapshots are extracted from the trajectories (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). Each snapshot represents a conformational variant of the binding pocket. In principle, every snapshot could be used for MRC virtual screening.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fundamental steps in a virtual screening workflow combining docking and MD simulations. (A) An MD trajectory is used to explore the receptor conformational space. (B) From the trajectory, several snapshots are extracted and redundancy is eliminated by means of cluster analysis. (C) From each cluster, a representative structure (e.g., medoid) is selected. (D) Virtual ligand screening is independently carried out at each representative conformation. (E) Activity predictions returned by independent runs are combined together in a global ranking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Notably, a selection of the MD-generated snapshots is normally needed to avoid redundant structures. This procedure has demonstrated that some conformers perform better than others in enriching active compounds.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Nevertheless, if no a priori knowledge indicates the single best performing conformation, some general guidelines exist for building a representative ensemble of targeted structures.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> From a survey of the literature, the most important concept is probably that a combination of two or three conformational variants of the target usually performs better than random single-conformation rigid docking. A limited conformational ensemble can improve both the final enrichment and the chemical diversity of the resulting hits. In fact, if the set of target conformations for MRC is too large and diverse, this often generates an overwhelming noise that deteriorates the virtual screening performance.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> For these reasons, a limited (yet hopefully significant) number of conformations is advisible for efficient MRC virtual screening. This can be selected by means of cluster analysis, as one example.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> Ideally, the selected snapshots should capture the entire structural diversity of the target, sampled along the trajectory, with the minimum number of significant conformers (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). For instance, a nonredundant set of conformations could be obtained using rmsd-based hierarchical-agglomerative cluster analysis protocols and clusterization methods based on QR-factorization.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> In a comparative study, Nichols et al.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> demonstrated how MD-generated receptor variants can match and possibly outperform crystal structures in retrospective virtual screening experiments. Each variant is used in an independent run to generate an individual set of results (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D). These separate rankings are eventually joined together (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>E).<a onclick="showRef(event, 'ref118 ref119'); return false;" href="javascript:void(0);" class="ref ref118 ref119">(118, 119)</a> Importantly, the relaxed complex method was successfully used to prospectively identify several modulators of relevant pharmaceutical targets, as reviewed in detail by Ivetac and McCammon.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">3.2 MD for Postprocessing of Docked Protein–Ligand Complexes</h3><div class="NLM_p">Because sampling and scoring algorithms for docking suffer from several limitations,<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> MD can be used as a postprocessing tool to validate and/or refine docking solutions. Here, the underlying assumption is that a bad docking pose will generate an unstable MD trajectory, during which the ligand could even leave the binding site. Conversely, a meaningful docking pose will display stable and specific interactions with the target, showing a low rmsd over time, with respect to the starting conformation. Furthermore, MD could provide evidence of so-called “induced-fit effects”, in which the binding site adapts to the starting pose of the ligand, strengthening the interactions already captured by docking and establishing additional interactions during the MD runs. Finally, MD in explicit solvent can suggest the role of structural water molecules within the binding site, which is important for correctly predicting ligand binding.<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a></div><div class="NLM_p last">Our work in 2004 is one example of how docking and MD simulations can be fruitfully coupled. Docking and cluster analysis allowed us to identify several binding modes of propidium at the peripheral anionic site of the human acetylcholinesterase enzyme.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> We investigated these initial poses using MD and eventually identified two alternative binding modes. Both binding modes were compatible with the electron density map, with one resembling the crystallographic pose and the other being flipped by 180°. This study showed that even a few nanoseconds of MD simulations could discriminate good docking poses from bad, when surface solvent-exposed binding sites were considered. In another study, Kacker et al.<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> used a combination of QM calculations, docking, and MD to define the protonation state of BACE-1’s catalytic machinery in complex with different ligands. MD was crucial in discriminating stable hydrogen bonds established between ligands and the catalytic dyad from more transient ones and in assessing the role of bridging water molecules. Complementing docking and MD, Rastelli et al.<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> devised an automatic pipeline, in which they rescored docking-generated conformations using MM-PB/GBSA free-energy calculations. With an acceptable increase in calculation time, they improved hit enrichment in retrospective virtual-screening experiments. Focusing on GPCR, particularly the human A<sub>2A</sub> adenosine receptor, Sabbadin et al.<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> proposed a sequential approach to studying stability in an explicit lipid–water environment of GPCR–ligand complexes generated by docking. Trajectories were analyzed in terms of individual electrostatic and hydrophobic contributions to the binding energy of each residue contacting the ligand. The bioactive conformation was expected to display a persistent interaction network along the trajectory. On the other hand, it is worth mentioning the recent study of Lauro  et al.,<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> which represents a counterexample where MD, employed to perform linear interaction energy calculations, while efficient, could not significantly improve ligand-ranking accuracy with respect to results generated by high quality docking scoring functions. For several other applications of MD as a ligand-docking postprocessing tool, the reader may refer to the review article by Alonso and colleagues.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">4 Unbiased MD for Investigating Drug Binding</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53778" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53778" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">During unbiased MD simulations, the model system evolves freely over time, without any (biasing) force acting on it. Thus, this seems the simplest way to simulate and observe the spontaneous evolution of a drug binding from the bulk of the solvent into the biological target. That is, one can run simulations for long enough to let the drug diffuse into the solvation waters until it finds its way into the target binding pocket, leading to the final bimolecular complex. However, for slow binding events (i.e., weak binders), very long simulations may be required, which can easily exceed today’s affordable simulation time (up to hundreds of microseconds and milliseconds).</div><div class="NLM_p">Until recently, the simulation of drug-binding events was far beyond the reach of atomistic MD simulations. Great progress has been made since the very first simulations of 9 ps in vacuo of the small bovine pancreatic trypsin inhibitor protein, reported in 1977’s landmark study by McCammon, Gelin, and Karplus.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Specialized supercomputers designed for MD simulations, such as the Anton machine created by the DE Shaw lab, can run millisecond-long continuous single-trajectory MD simulations of small globular proteins.<a onclick="showRef(event, 'ref29 ref33'); return false;" href="javascript:void(0);" class="ref ref29 ref33">(29, 33)</a> Even longer aggregated sampling times have been reported using <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="b4f2dbd8d0dddad3f4fcdbd9d1">[email protected]</a> from the Pande lab.<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> While these highly efficient computers and MD systems are available to just a few researchers, the growing distribution of graphics processor units (GPUs) is increasing the MD community’s ability to run long simulations at an affordable cost. Nowadays, researchers can quite routinely generate tens of microseconds of simulated time for medium-size models (50–100K atoms).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Only 10 years ago, it was hard to envisage such a rapid increase in MD time scales. Remarkably, this has mostly been achieved because of technological progress rather than the development of novel theories.</div><div class="NLM_p">For drug discovery, typical model systems only include the solvated target and ligand(s), which do not usually exceed a few hundred thousand atoms in size. In the past few years, unbiased MD simulations of these models have been used quite extensively to describe the process by which drugs bind to targets in several systems. Hundreds of ∼100 ns runs of unbiased MD were used to identify possible pathways for the spontaneous binding of benzamidine to the trypsin.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Tens of microsecond-long simulations were run to detect the spontaneous binding of G-protein-coupled receptor (GPCR) agonists and antagonists, as well as protein kinase inhibitors.<a onclick="showRef(event, 'ref31 ref127 ref128 ref129'); return false;" href="javascript:void(0);" class="ref ref31 ref127 ref128 ref129">(31, 127-129)</a> More recently, Decherchi et al. ran extensive unbiased MD simulations (about 1 μs each) to investigate the tight binding event of a transition state analog (DADMe-immucilin H) into the pharmaceutical target, purine nucleoside phosphorylase.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Despite observing spontaneous binding through different routes, the authors used the pathways obtained by MD simulations to determine the free-energy profiles associated with the diverse binding mechanisms. By combining MD with machine learning and enhanced sampling methods (see below), they could observe the binding and estimate the associated kinetics and thermodynamics. Overall, these unbiased MD simulations have validated the computational methodology, reproducing the known crystallographic pose of the drug quite well (with an rmsd that was usually within 2 Å). In addition, the free-energy difference between the bound and unbound states can be calculated in many formally equivalent ways and compared to the experimental <i>K</i><sub>i</sub>, when available. Kinetic quantities, such as <i>k</i><sub>on</sub> and <i>k</i><sub>off</sub>, are further important observables that can be retrieved from these long MD simulations. They must also reproduce the experimental data to further validate the simulated drug-binding process (see below for further details on binding kinetics). In this way, the accuracy of the computational approach to simulating the drug-binding process is endorsed and, to some extent, benchmarked. Only then can MD trajectories be trusted, analyzed, and interpreted in a meaningful way. Doing so reveals a wealth of potentially useful insights into how the drug can reach the final bound conformation, highlighting key transient interactions that drive drug binding. For example, these simulations can reveal mesostates and high-energy intermediates formed along the path to the final binding into the cavity. It is easy to raise concerns about the real existence of these metastable conformational states. However, these concerns can be addressed by providing rational hypotheses and corroborating data retrieved from the MD simulations. This can explain, for example, the experimental drug–target affinity, which the MD simulations must reproduce well.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p last">Indeed, these first successful applications of unbiased MD to reconstructing spontaneous drug-binding processes, and the related thermodynamics constants, are a great achievement. Their success demonstrates the potential of unbiased MD to predict (and not just reproduce) plausible binding poses of new compounds in the not-too-distant future. Simulations can reveal key drug–target interactions that must be formed and/or disrupted to achieve tight drug binding. This can suggest hot spots (on both the ligand and target) that could in principle be modified to modulate their activity. It can also help prioritize compounds in medicinal chemistry campaigns.<a onclick="showRef(event, 'ref130 ref131'); return false;" href="javascript:void(0);" class="ref ref130 ref131">(130, 131)</a> As mentioned above, these unbiased trajectories can also provide quite realistic initial configurations and guess paths for subsequent simulations.<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> These can be used to improve sampling along that initial path and to estimate the kinetics and thermodynamics of drug–target binding.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">5 MD Coupled to Enhanced Sampling Methods for Investigating Ligand Binding</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64161" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64161" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">While unbiased MD is a straightforward but costly way of simulating drug-binding processes, enhanced sampling methods are often coupled to MD to accelerate this process and to retrieve useful thermodynamic and kinetic data.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> Indeed, enhanced sampling methods (mostly steered MD and metadynamics) were used for the first simulations of drug-binding events, with the ligand moving into and/or out of the binding pocket.<a onclick="showRef(event, 'ref43 ref133 ref134'); return false;" href="javascript:void(0);" class="ref ref43 ref133 ref134">(43, 133, 134)</a> Researchers can now rank compounds according to the relative free energy of binding, calculated using the enhanced sampling scheme. Of the enhanced sampling techniques developed in recent years, we focus here on FEP,<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> umbrella sampling,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> steered MD,<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> and metadynamics.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> These methods have appealed most broadly to the drug discovery community, with demonstrated potential for impacting structure-based drug design. Other existing methods will be discussed in the final paragraph.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">5.1 Free-Energy Perturbation (FEP)</h3><div class="NLM_p">FEP theory and calculations relate the free energy of an initial reference state to the final target state of a system, using Monte Carlo (MC) or atomistic MD as a sampling technique.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> In FEP calculations for drug design, perturbations of the initial lead compound are made using a coupling parameter during the simulations, driving a smooth mutation of the starting molecule to another close one. This cycle is then repeated a number of times, evaluating the relative binding free energies of several possible changes of the initial lead compound, which can be transformed into several close analogs. The relative free energies of binding of each compound are calculated according to the thermodynamic cycle in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. If needed, the standard binding free energy can be retrieved by appropriately modifying the cycle (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).<a onclick="showRef(event, 'ref37 ref81 ref135'); return false;" href="javascript:void(0);" class="ref ref37 ref81 ref135">(37, 81, 135)</a> The final free-energy estimate suffers from the usual issues of FF-based calculations. These include the quality of the FF, missing polarization effects, and limited sampling. Notably, thermodynamic integration (TI) calculations could be used to compute free-energy changes in a way that is as formally rigorous as using FEP. Nevertheless, FEP-based calculations appeal more broadly to the drug discovery community.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> Jorgensen et al. recently provided a thorough overview of the history, main contributors, and methodological development of FEP theory since its inception.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p">In 1985, Jorgensen et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> became the first to use FEP calculations to convert one molecule of ethane into methanol, in water. Just 1 year later, McCammon et al.<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> used FEP for protein–ligand binding calculations. More recently, however, increased computational power has allowed researchers to robustly validate FEP calculations for the rational design of small molecules. This rigorous approach is now endorsed for screening small libraries of close analogs for enzyme inhibition.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> In the past, FEP calculations of protein–ligand systems were primarily used to reproduce known experimental data for a few small molecule inhibitors. Remarkably, prospective FEP calculations, within an MD framework (FEP/MD) for drug design, were already reported in 1989 by Merz and Kollman, who correctly predicted the binding free energy of a previously unreported inhibitor of the thermolysin endopeptidase.<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> Similarly, few years later, FEP/MD was again successfully employed to predict the affinity of a novel HIV-1 peptide inhibitor.<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> Interestingly, the design of HIV-1 inhibitors became an excellent testing ground for the whole FEP methodology. Several groups utilized such model system to rationalize structure–activity relationships and to suggest viable modifications of known compounds, using simulations of increasing complexity. Among these, we mention free energy calculations between two classes of HIV-1 protease inhibitors (amide and ester analogs) by Rao et al.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a> and the calculations of the free energy difference caused by the deletion of an entire valine residue from a peptidomimetic compound,<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> which represented a significant alchemical transformation relative to the computational resources available at the time.</div><div class="NLM_p">Today, FEP calculations have demonstrated astonishing potential for driving lead optimization campaigns.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> Those first early studies are now recognized as crucial in advancing this effective methodology for rational lead optimization. Coincidentally, in their initial prospective applications of FEP to drug design, Jorgensen’s group mostly coupled FEP to MC (FEP/MC).<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> If the pose of a lead compound within the binding pocket is known with confidence (possibly based on the X-ray structure of the ligand–target complex), rigorous FEP/MC calculations have indeed proven to be highly accurate and informative in predicting what type of (small) modifications could generate a boost in ligand-binding affinity for the target.</div><div class="NLM_p">One exceptionally successful application of FEP/MC-guided lead optimization is the rational design of what are currently the most potent non-nucleoside inhibitors of HIV-1 reverse transcriptase (NNRTIs). The FEP/MC-guided modifications of a few promising scaffolds have led to several potent NNRTIs inhibitors, as reported by Jorgensen et al.<a onclick="showRef(event, 'ref143 ref144'); return false;" href="javascript:void(0);" class="ref ref143 ref144">(143, 144)</a> These modifications include the optimization of a 5 μM docking hit into a 55 pM catechol diether NNRTIs inhibitor,<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> which was further optimized to improve druglikeness and resistance to mutations (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(145)</a> Other successful examples include inhibitors of the tautomerase activity of human macrophage migration inhibitory factor (MIF)<a onclick="showRef(event, 'ref146 ref147 ref148'); return false;" href="javascript:void(0);" class="ref ref146 ref147 ref148">(146-148)</a> and others.<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> Taken together, these paradigmatic examples attest the impact of FEP calculations on lead optimization for drug discovery, with a clear demonstration that this approach, nowadays, offers great advantages and progress.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. From the crystal structure of compound R221239 (compound <b>3</b>), the modest anti-HIV activity of the initial docking hit (compound <b>1</b>), 5 μM potency toward WT HIV-1 reverse transcriptase, was efficiently evolved into highly potent catechol diethers such as compound <b>42</b>, a 55 pM non-nucleoside inhibitor of HIV-1 reverse transcriptase (NNRTIs) discovered with the aid of the computational analyses guided by FEP results.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">These numerous examples have also provided a way to test and standardize protocols for running FEP calculations efficaciously, via MC- or MD-based simulations.<a onclick="showRef(event, 'ref150'); return false;" href="javascript:void(0);" class="ref ref150">(150)</a> It is only recently, however, that researchers designed tight-binding ligands using massive FEP/MD calculations.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> Wang et al.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> used over 200 ligands and 10 targets to validate, with retrospective studies, the high accuracy of FEP calculations (with an average error of ∼1 kcal/mol). They demonstrated the efficiency of these FEP/MD calculations, obtained using an improved force field (OPLS2.1 and 3), an improved enhanced sampling technique (i.e., FEP coupled to replica exchange with solute tempering or REST; see more on this in the below sections), and importantly, an automated user-friendly workflow running on graphics-processing units (GPUs).<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a></div><div class="NLM_p last">In accounting for the widespread applicability of the FEP approach in drug design, key factors include the improved accuracy of FEP results and the accelerated input setup and output generation. FEP/MD is now proposed for screening medium-to-large libraries of potential new compounds, where modifications of the initial compound can be quite substantial. FEP/MD can now generate the type of information that can pragmatically drive a drug design project, as Wang et al.’s encouraging study<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> suggests. In the coming years, we expect new prospective studies to further demonstrate FEP/MD’s impact on the hit-to-lead and lead optimization phases of drug discovery. FEP/MD could gradually become a routinely applied method for guiding key medicinal chemistry decisions and ultimately accelerating the generation of promising lead and drug candidates.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">5.2 Umbrella Sampling (US)</h3><div class="NLM_p">Umbrella sampling (US) can be considered the forerunner of all the CV-based enhanced sampling methods. US was conceived by Torrie and Valleau in 1977 in the context of MC sampling.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Soon after, it was adapted to the framework of MD simulations. The idea is to enforce sampling along the chosen CV by performing staged simulations subject to an energetic bias, which traditionally takes the form of a harmonic potential (umbrella). In this way, high-energy regions along the reaction coordinate can be sampled exhaustively, and the unbiased distribution of states required to compute the potential of mean force (PMF) is recovered through suitable postprocessing methods. The weighted histogram analysis method (WHAM) is the most popular postprocessing method,<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> but other strategies, such as the umbrella integration, can also be used.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a> The key parameters for US simulations are (i) the force constants chosen for the umbrellas, which must be high enough to ensure uniform sampling along the entire CV space, and (ii) the separation in the CV space between different simulations (usually called “windows”), which must be small enough to provide sufficient overlap between neighboring windows. A more detailed introduction to the theory of US can be found elsewhere.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a></div><div class="NLM_p">Although US is one of the most accurate techniques for free-energy calculations, its practical application in drug design is mostly limited by its elevated computational cost. For example, many overlapping windows are needed to examine the free energy of a ligand unbinding event from the protein, where each window must be prepared, sufficiently equilibrated, and finally sampled. Moreover, simulations of ligand unbinding often require a few CVs, leading to a PMF reconstruction that is exponentially slower as the number of CV increases. Other enhanced sampling methods, such as metadynamics and adaptive biasing force (ABF),<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a> are more efficient in exploring the free-energy space using multiple CVs, returning a multidimensional PMF. For this reason, they are more appealing for practical drug design (see below).</div><div class="NLM_p">In 2006, Woo and Roux reported a practical US-based methodology for successfully computing the binding affinity of a peptide to the SH2 domain of human Lck kinase.<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> This approach describes the ligand (un)docking, using a one-dimensional PMF. This PMF is calculated from US simulations, using a well-defined axis as the CV to connect the protein and the ligand. Importantly, restraining potentials are used to appropriately control the transverse degrees of freedom that might change upon ligand (un)binding. These include ligand conformation, orientation, and radial translations along the CV. The absolute binding free energy is recovered by the equilibrium-binding constant, which is in turn obtained by integrating the PMF. This methodology was later successfully applied to other pharmaceutically relevant problems.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a> Interestingly, this approach has been thoughtfully discussed and compared with the double decoupling method (DDM),<a onclick="showRef(event, 'ref81 ref89'); return false;" href="javascript:void(0);" class="ref ref81 ref89">(81, 89)</a> with which it shares some resemblances.<a onclick="showRef(event, 'ref156'); return false;" href="javascript:void(0);" class="ref ref156">(156)</a></div><div class="NLM_p last">More recently, similar US-based strategies have been devised by Lee and Olson<a onclick="showRef(event, 'ref157'); return false;" href="javascript:void(0);" class="ref ref157">(157)</a> and, later, by Doudou et al.,<a onclick="showRef(event, 'ref158'); return false;" href="javascript:void(0);" class="ref ref158">(158)</a> who used these free-energy simulations to estimate ligand binding. The overall procedure, however, remains so elaborate that US-based methods for drug discovery are not yet routinely or widely used. We expect this to change once the procedure and related corrections for US-based simulations to relate the PMF to the binding affinity become less cumbersome. In particular, automated procedures, such as self-learning approaches and adaptive sampling, may be a viable solution for reducing computational costs and improving the method’s general applicability.<a onclick="showRef(event, 'ref159'); return false;" href="javascript:void(0);" class="ref ref159">(159)</a></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">5.3 Steered MD</h3><div class="NLM_p">With this method, a time-dependent external potential is applied to the ligand to facilitate its unbinding from the target protein. Ligand undocking is therefore accelerated by acting on a descriptor (or CV), which is usually the protein–ligand distance or a vector describing the ligand exit pathway. To achieve this, the ligand is attached to a spring with a given force constant, and the center of the harmonic restraint is moved to a finite velocity along the descriptor so as to drive a smooth exit of the ligand from the pocket.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> Steered MD is actually a nonequilibrium method, in which the pulling velocity and spring constant are key parameters for these simulations. Conceptually, steered MD is similar to performing atomic force microscope (AFM) experiments, although pulling in steered MD is typically performed at much higher velocities.</div><div class="NLM_p">Importantly, researchers can use steered MD to calculate the exerted force and the external work performed on the system for the unbinding of each ligand considered. Grubmüller et al.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> first demonstrated this in their pioneering investigation of the streptavidin–biotin complex. If the force constant is high enough (stiff-spring regime), the rupture force scales linearly with the amount of irreversible work (i.e., with the applied velocity). This provides important qualitative and semiquantitative insights into the unbinding process. However, in 1997, Jarzynski<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a> defined a fundamental relationship between irreversible and reversible work (i.e., the free-energy difference). The so-called “Jarzynski equality” (also known as “nonequilibrium work relation”) was a milestone that allowed the PMF for the investigated process to be obtained through a series of pulling simulations, regardless of the speed applied. The equality is exact only in the limit of an infinite number of realizations. Therefore, Park et al.<a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a> devised an approximate formula, which holds in the limit of the stiff-spring regime. This provides better statistical results than the naive application of the Jarzynski equality. A theoretical description of steered MD can be found elsewhere.<a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a></div><div class="NLM_p">In the context of MD-based rational drug design, one prospective study investigated the binding of five structurally related flavonoids to the protein FabZ. This enzyme from the malaria-causing parasite <i>Plasmodium falciparum</i> is a potential target for antimalarial drugs.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> Steered MD simulations were used in a similar way to single-molecule pulling experiments, where the force required to pull out each of the studied ligands from the target protein was computed and used to individuate the tighter binders (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). Strongly bound inhibitors gave profiles with higher peak forces than weakly bound ligands, which gave a flatter force profile. A new flavonoid was designed and predicted via steered MD, with experimental validation of the approach’s reliability for computational drug discovery. This paradigmatic study clearly demonstrated that steered MD could distinguish active from inactive inhibitors, although the computed observables (forces required to extract each inhibitor from the targeted enzyme) remained only qualitative. This is mostly due to the poor efficiency in computing fully converged free-energy profiles during steered MD simulations. Later, this approach was used to examine different series of CDK5 inhibitors, confirming steered MD’s ability to correctly discriminate binders from nonbinders, although the method failed to correctly rank series of inhibitors with similar inhibiting potencies.<a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a> Researchers may use the unbinding force profile to improve docking scoring function or alternatively as a postprocessing tool in structure-based virtual screening campaigns.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison of the undocking force profiles of different flavonoid ligands. (A) Chemical structures of ligands under investigation. (B) Force profiles derived from pulling the ligands along the unbinding reaction coordinate. For each ligand, the plots show the resulting mean values from averaging the force profiles from five different SMD runs.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of these examples, steered MD seems particularly suited to discriminating active and inactive binders, which interact with the target through H-bonds that must be disrupted during the unbinding process. This was true for steered MD simulations used to characterize the unbinding of the clinical candidate F14512, which bears a spermine chain forming a complex H-bond network with the targeted topoisomerase enzyme.<a onclick="showRef(event, 'ref164'); return false;" href="javascript:void(0);" class="ref ref164">(164)</a> The polyamine chain in F14512 generates high peak forces during the steered MD for drug unbinding, when compared to close analogs with no ability to form such a strong H-bond network. Conversely, hydrophobic interactions do not provide clear rupture forces, marginally affecting the resulting work profile and hampering a proper ranking of ligands with different potencies.<a onclick="showRef(event, 'ref164'); return false;" href="javascript:void(0);" class="ref ref164">(164)</a> Notably, this is a further example where MD simulations have been used in a prospective manner to design analogs of F14512, which have been synthesized and tested to validate the MD-based results.</div><div class="NLM_p last">Patel et al. reported a different approach, devising a steered-MD-based protocol to capture relevant biological information from local work profiles obtained by protein–ligand pulling simulations in an unsupervised way. Interestingly, this protocol could in principle be exploited to tune affinity and kinetic observables.<a onclick="showRef(event, 'ref165'); return false;" href="javascript:void(0);" class="ref ref165">(165)</a> However, steered MD is somewhat limited to qualitatively estimating the tightness of drug binding rather than accurately calculating the free energy of binding (as with FEP calculations). This suggests steered MD as suitable for postprocessing ligand screening during hit identification rather than for lead optimization. Steered-MD-based screening of large and diverse data sets of compounds would increase our understanding of this methodology’s true potential and practical location within the drug discovery pipeline.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">5.4 Metadynamics-Based Methods</h3><div class="NLM_p">These include a broad family of enhanced sampling techniques, which allow fast exploration of the underlying free-energy landscape of rare events. They use a set of order parameters, usually referred to as collective variables (CVs), that approximate the true reaction coordinate of the process. These CVs have a broad range, including coordination numbers, the number of hydrogen bonds, relative molecule orientation/rotation, and bond lengths, angles, or torsions. Once the most suitable CVs have been identified, the central idea in metadynamics is to bias the dynamics of the system along those CVs, using a history-dependent repulsive potential. To achieve this, at regular time intervals, a relatively small Gaussian-shaped potential is added to the bias at the current position of the CVs. This discourages the system from revisiting already explored regions of space.<a onclick="showRef(event, 'ref41 ref166'); return false;" href="javascript:void(0);" class="ref ref41 ref166">(41, 166)</a> The history-dependent potential thus builds up until it counterbalances the projection of the free energy along the chosen CVs. This allows the system to escape via a saddle point to a nearby local minimum, where the procedure is repeated. When all minima are “filled” with Gaussians, the system moves in a barrier-free manner among the different states. The bias potential can then be used as an unbiased estimator of the PMF by simply changing sign (see <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>). Since the bias potential changes over time during the simulations, metadynamics is considered to be a nonequilibrium enhanced sampling method.<a onclick="showRef(event, 'ref41 ref167'); return false;" href="javascript:void(0);" class="ref ref41 ref167">(41, 167)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Pictorial representation of a metadynamics simulation. The underlying free energy, which is unknown beforehand, consists of three minima, A, B, and C, separated by two transition states. It is depicted as a black thick curve. The bias introduced by metadynamics is represented by filled curves with a dark-gray to red color gradient indicating the progression of the simulation. Starting the calculation in minimum A, the Gaussian bias released in time by metadynamics gently pushes the system toward the first transition state (TS<sub>1</sub>) and leads to the identification of the global minimum B. Then, by continuation of the simulation, TS<sub>2</sub> and the metastable state C are also visited. Finally, when the underlying free energy is completely filled by the Gaussian potentials, the system evolves with diffusive motion along the collective variable, and the simulation can be stopped. At that time, with conventional metadynamics, the free energy can be recovered from the bias by simply changing the sign.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In metadynamics, the reconstruction of the free-energy surface can easily exceed two dimensions. This key feature distinguishes it from other CV-based algorithms. In addition, unlike umbrella sampling, metadynamics requires no additional postprocessing analysis step such as WHAM. The crucial parameters for reliably reconstructing the free energy in metadynamics are (i) the Gaussians height, (ii) the Gaussians width, and (iii) the deposition time. Briefly, the error approximately scales with the square root of the ratio between the Gaussians height and the deposition time. This implies that to achieve a reasonably accurate free-energy profile, the height of the Gaussians must be sufficiently small compared to the main free-energy barrier. They should also not be added too frequently in time. Conversely, the choice of the Gaussians width is much less straightforward. It should be small enough to provide a good resolution of the PMF.<a onclick="showRef(event, 'ref168'); return false;" href="javascript:void(0);" class="ref ref168">(168)</a> A number of recent review articles provide a more thorough description of this method.<a onclick="showRef(event, 'ref166 ref169 ref170'); return false;" href="javascript:void(0);" class="ref ref166 ref169 ref170">(166, 169, 170)</a></div><div class="NLM_p">Metadynamics has significantly evolved since its introduction, with several modifications aimed at improving its convergence behavior or efficiency. Researchers have also focused on better describing the investigated event and have occasionally extended the conventional metadynamics formulation to achieve this. The multiple walkers metadynamics is one notable variation, wherein multiple simulations share the same bias potential, which improves the parallel performances of sampling. However, the most important improvement is the so-called “well-tempered metadynamics” (WTmetaD). In WTmetaD, the underlying bias and a given energy threshold govern the height of the added Gaussians, generating a more efficient and unbiased estimate of the free energy of the system.<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a> In contrast to conventional metadynamics, the bias in WTmetaD does not fully compensate the underlying PMF, but it converges to a well-defined value. In a recent variation of metadynamics, researchers explored the possibility of using adaptive Gaussians with on-the-fly modified width and orientation with respect to the CV space. This variation showed better accuracy and convergence properties than previous formulations.<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> Eventually, combining WTmetaD and adaptive hills may relieve the user of the delicate choice of parameters, controlling the shape of the Gaussians.</div><div class="NLM_p">Branduardi et al.<a onclick="showRef(event, 'ref173'); return false;" href="javascript:void(0);" class="ref ref173">(173)</a> developed the path collective variables (PCVs) method to improve the definition of CVs. The PCVs method allows a nonlocal exploration of complex multidimensional processes, using a predefined pathway on a low-dimensional (2D) space. Notably, several metrics can be used to define the path, including the rmsd or the contact map distance (CMAP).<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a> So far, PCVs have been successfully used to address several biological problems ranging from large protein conformational transitions to ion permeation. Their use is not strictly limited to metadynamics sampling.<a onclick="showRef(event, 'ref175 ref176'); return false;" href="javascript:void(0);" class="ref ref175 ref176">(175, 176)</a> More recently, to improve the unsupervised definition of CVs, significant effort has been devoted to developing “smarter” variables based on dimensional reduction techniques, such as nonlinear multidimensional scaling.<a onclick="showRef(event, 'ref177'); return false;" href="javascript:void(0);" class="ref ref177">(177)</a> From a different perspective, when a suboptimal choice of CVs adversely affects the reconstructed free energy, this can be mitigated by coupling metadynamics to replica-exchange-like methods. This can effectively improve the sampling along transverse degrees of freedom.<a onclick="showRef(event, 'ref178'); return false;" href="javascript:void(0);" class="ref ref178">(178)</a> In this context, another metadynamics formulation used the potential energy as a CV, thus enhancing the energy fluctuations required for an efficient replica exchange setup (well-tempered ensemble).<a onclick="showRef(event, 'ref179'); return false;" href="javascript:void(0);" class="ref ref179">(179)</a> Bias-exchange metadynamics (BEmetaD)<a onclick="showRef(event, 'ref180'); return false;" href="javascript:void(0);" class="ref ref180">(180)</a> is a popular extension of the conventional metadynamics method, which still relies on a replica-exchange-like scheme. In this case, a series of metadynamics simulations, biasing different CVs, are run in parallel and exchanged at fixed times according to a Metropolis-like criterion. This reduces the problem of choosing a priori a small number of CVs for the investigated event. Along the same lines, reconnaissance metadynamics is a sophisticated scheme for automatically detecting a set of CVs through machine-learning techniques. It has been introduced and successfully applied to biologically related problems.<a onclick="showRef(event, 'ref181'); return false;" href="javascript:void(0);" class="ref ref181">(181)</a></div><div class="NLM_p last">In the past few years, metadynamics, in its various implementations, has been applied to a number of ligand–target complexes, demonstrating its ability to characterize binding and unbinding paths, to treat conformation flexibility, and to compute free-energy profiles. Since Gervasio et al.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> first applied metadynamics to ligand–target complexes, several other informative studies have been reported. Two representative examples are the recent investigation of the (un)binding of a nonsteroidal anti-inflammatory ligand 4-[5-(4-bromophenyl)-3-(trifluoromethyl)-1<i>H</i>-pyrazol-1-yl]benzenesulfonamide (SC-558) to COX-1 and COX-2 isoforms,<a onclick="showRef(event, 'ref182'); return false;" href="javascript:void(0);" class="ref ref182">(182)</a> and the recognition of cortisone from the 11β-hydroxysteroid dehydrogenase enzyme obtained by combining steered MD and metadynamics.<a onclick="showRef(event, 'ref183'); return false;" href="javascript:void(0);" class="ref ref183">(183)</a> Both studies used PCVs to describe the (un)binding process, with the former using the CMAP metric and the latter using the rmsd metric. In a paper investigating the recognition mechanisms of naloxone to the δ opioid receptor, Provasi et al.<a onclick="showRef(event, 'ref184'); return false;" href="javascript:void(0);" class="ref ref184">(184)</a> provided a notably elegant way of recovering the standard binding free energy from (un)binding metadynamics simulations. This was analogous to the methods discussed above in the context of US. Another reported strategy more closely resembled Woo and Roux’s method,<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> using a reverse funnel-shaped potential that limits the space available to the ligand once it has undocked (“funnel metadynamics”).<a onclick="showRef(event, 'ref185'); return false;" href="javascript:void(0);" class="ref ref185">(185)</a> Other studies have described drug binding to protein kinases, or the association mechanism of molecules binding to DNA and RNA, such as the interaction of ligands with DNA G-quadruplexes and the unbinding of the anticancer drug distamycin from the DNA minor groove.<a onclick="showRef(event, 'ref170 ref186'); return false;" href="javascript:void(0);" class="ref ref170 ref186">(170, 186)</a> “Coarse metadynamics” uses a combination of docking, cluster analysis, and metadynamics confined within the internal binding cavity. When applied to drug design and compound screening, it has shown the best performance in conventional structure-based endeavors.<a onclick="showRef(event, 'ref187'); return false;" href="javascript:void(0);" class="ref ref187">(187)</a> Nevertheless, if one is interested in the relative free energy of a set of congeneric compounds with a similar binding pose, enhanced sampling methods based on unphysical pathways (i.e., methods such as FEP and TI, which compute the difference of free energy between the bound and the unbound state) are often more efficient and rigorous than methods that compute the free energy along a hypothetical association pathway. For this reason, metadynamics has so far mostly been used in retrospective studies of ligand binding. These informative applications of metadynamics-based simulations suggest that the time has now come to use metadynamics in prospective drug design efforts, with a particular focus on lead optimization. If these studies were successful, then metadynamics simulations would offer an intrinsic added value over FEP and TI. This is because the entire physical path of protein–ligand binding and/or unbinding could be accurately computed and revealed, together with the associated thermodynamic and kinetic profiles.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">5.5 Other Approaches and Method Combinations</h3><div class="NLM_p">Several other MD-based enhanced sampling techniques exist and have recently been used to estimate the free energy of drug binding and to rationalize the experimental drug–target affinity. These include the adaptive biasing force (ABF)<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a> and accelerated MD.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> While these approaches can correctly reproduce experimental data and rationalize target affinity for different drugs/ligands, their predictive applicability for practical drug design is in its infancy. Their high computational cost is the most likely hindrance. Nonstandardized protocols and parameters, which can affect the final results, may also be a factor at times.</div><div class="NLM_p">Of these many methods, the replica exchange class of methods is gaining popularity for drug design. This is mostly due to their flexibility and the embarrassingly parallel nature of computations, which makes them ideal for coupling to several enhanced sampling techniques. In addition, they almost linearly scale on multicore supercomputers, making them the methods of choice for CPU-intensive or GPU-intensive simulations. Replica exchange methods use several copies of the system evolving, in parallel, under different simulating conditions, such as the temperature (T-REMD).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Then, at regular intervals, swapping between a pair of replicas is attempted according to a Metropolis MC acceptance criterion. Thus, a random walk along the temperature space provides the required enhancement in sampling, while the coldest (or physical) replica ensures the correct distribution of states in the reference statistical ensemble. However, a large number of replicas are required for the method to be effective. This is because the probability of accepting MC moves depends exponentially on the difference in potential energy between replicas. Moreover, because the potential energy grows with the number of degrees of freedom, this issue is more pronounced in large systems. A valid alternative is the H-REMD variant (or REST),<a onclick="showRef(event, 'ref93 ref95'); return false;" href="javascript:void(0);" class="ref ref93 ref95">(93, 95)</a> in which, rather than increasing temperature, the potential energy function is gradually scaled down along the progression of the replicas. By doing so, it is possible to affect only selected (relevant) degrees of freedom (i.e., the binding site and the ligand), greatly improving the acceptance probability and, in turn, the efficiency compared to the conventional temperature version. Interestingly, H-REMD has been used to accelerate the sampling to obtain the correct protein–ligand binding modes for implicit or explicit solvent simulations.<a onclick="showRef(event, 'ref188 ref189'); return false;" href="javascript:void(0);" class="ref ref188 ref189">(188, 189)</a> This mimics the search algorithms of docking programs within an MD framework.</div><div class="NLM_p">Actually, the combination of MD sampling with MC has a long tradition in theoretical physics, which goes back to the 80s when the hybrid Monte Carlo (HMC) method consisted of evolving the equations of motions, accepting or rejecting the advancement in positions according to the Metropolis criterion.<a onclick="showRef(event, 'ref190'); return false;" href="javascript:void(0);" class="ref ref190">(190)</a> Later, this method was further developed, like in the case of adaptive temperature HMC by Fischer at al.,<a onclick="showRef(event, 'ref191'); return false;" href="javascript:void(0);" class="ref ref191">(191)</a> where transitions to higher temperatures were used to overcome relevant energy barriers. Then, an appropriate reweighting scheme was employed to assess the correct populations of states at the target temperature. Interestingly, the same method was utilized as sampling engine for one of the earliest applications of Markov state model (MSM) to biomolecular relevant problems (see below), such as the conformational dynamics of <i>n</i>-pentane and a triribonucleotide fragment.<a onclick="showRef(event, 'ref192'); return false;" href="javascript:void(0);" class="ref ref192">(192)</a> Concerning more drug-design-oriented methods, we also mention the mixed MC/SD scheme proposed by Guarnieri and Still.<a onclick="showRef(event, 'ref193'); return false;" href="javascript:void(0);" class="ref ref193">(193)</a> Here, different from HMC implementations, SD sampling is periodically interrupted by ordinary MC moves performed in the dihedral space of molecules. As such, the MC/SD approach samples effectively the conformational space of relatively small druglike molecules in implicit solvent.</div><div class="NLM_p">Recently, replica exchange methods have been efficaciously coupled to conventional FEP methods to improve the sampling. Jiang et al.<a onclick="showRef(event, 'ref194'); return false;" href="javascript:void(0);" class="ref ref194">(194)</a> reported one way of coupling replica exchange to FEP, by including swaps along the staged alchemical transformation (FEP/REMD scheme, see <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A). This improved the convergence properties of the free-energy calculations. In this context, the Roux group took full advantage of the H-REMD potentiality only later, when a second ladder of replicas controlling the torsional energy of selected rotamers on the binding site was added, thus leading to a two-dimensional space of exchanges (FEP/H-REMD scheme, <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B).<a onclick="showRef(event, 'ref195'); return false;" href="javascript:void(0);" class="ref ref195">(195)</a> However, the large number of replicas required by the FEP/H-REMD scheme probably counterbalances its effective benefits. Recently, researchers proposed a very efficient implementation of FEP augmented with replica exchange, named FEP/REST.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> As shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>C, FEP/REST uses a one-dimensional ladder of replicas, where both the alchemical transformation on the ligand and the potential energy scaling of the binding site (amino acids as well as ligand torsions) occur. Thus, intermediate points along the progression represent unphysical states of the ligand bound to “hot” conformations of the protein–ligand complex, while the end points correspond to physical states of ligand analogs, each bound to the corresponding “cold” protein conformation.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison of three strategies to couple FEP with replica exchange. In panel A, the FEP/REMD method implements replica exchanges between the staged alchemical transformation in the context of the double decoupling method (DDM). Here, the unphysical transformation is performed in three steps: charges, dispersive, and repulsive van der Waals interactions are turned off separately and in sequence. For simplicity, the inclusion of rototranslational and conformational restraints, as envisioned in the original paper, is not considered. The axis controlling the alchemical transformation (λ-axis) is retained in the FEP/H-REMD method (B), but it is coupled to a boosting axis where the potential energy of selected dihedrals is scaled down, mimicking the effect of high temperature. In the context of relative binding free-energy calculations, a fictitious high temperature is also used in the FEP/REST method (C). In this case, the alchemical transformation is performed together with the potential energy rescaling. Thus, the “hottest” replicas are located in the middle of the ladder, and physical states are only found at the end points.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While FEP has been coupled to enhanced sampling techniques other than replica exchange,<a onclick="showRef(event, 'ref196'); return false;" href="javascript:void(0);" class="ref ref196">(196)</a> the coupling of replica-exchange-like methods is not limited to alchemical transformations only. Indeed, it is possible to use a replica exchange framework to alleviate convergence issues related to CV-based enhanced sampling methods, such as US or metadynamics. For example, the Roux group adopted the previously reported FEP/H-REMD methodology for exchanges along US windows on a two-dimensional CV space (US/H-REMD).<a onclick="showRef(event, 'ref197'); return false;" href="javascript:void(0);" class="ref ref197">(197)</a> Furthermore, in the context of metadynamics, Sutto and Gervasio<a onclick="showRef(event, 'ref198'); return false;" href="javascript:void(0);" class="ref ref198">(198)</a> studied the conformational free energy of EGFR kinase, using WTmetaD coupled to T-REMD. In this case, the replica exchange strategy helps in relaxing all the transverse degrees of freedom that are not explicitly accounted for in the definition of CVs. Moreover, the energy overlap among replicas, required for an optimal usage of resources, was greatly improved using the bias obtained in preliminary metadynamics simulations where the potential energy was used as an additional CV (well-tempered ensemble sampling).</div><div class="NLM_p last">Finally, we mention two other methods that inherit the idea of alchemical transformations augmented with a replica exchange framework, even though they are rather different in formulation from all the techniques discussed above. The first is Gallicchio et al.’s BEDAM (binding energy distribution analysis method) approach,<a onclick="showRef(event, 'ref199'); return false;" href="javascript:void(0);" class="ref ref199">(199)</a> which uses the AGBNP2 solvation model to perform replica exchanges along the staged alchemical transformation. By virtue of the implicit solvent model used, the alchemical transformation in BEDAM involves the effective ligand–environment interactions. Thus, the end points of the progression of replicas consist of physical states representing the ligand interacting with the solvent and the protein (uncoupled and coupled states, respectively). The absolute binding free energy is then recovered from the sampled energy distributions by exploiting statistical mechanics arguments. From this standpoint, BEDAM shares some resemblance with the MM-PB/GBSA method and with DDM. Procacci et al. have recently proposed another interesting approach (called EDU-HREM<a onclick="showRef(event, 'ref200'); return false;" href="javascript:void(0);" class="ref ref200">(200)</a>) to compute the standard binding free energy. This approach can be thought of as an explicit-solvent variant of BEDAM. During EDU-HREM, an unphysical transformation is achieved with an H-REMD strategy whereby protein–ligand interactions as well as torsional and intramolecular nonbonded potentials are progressively <i>decreased</i> along replicas, while, at the same time, the protein–solvent and ligand–solvent interactions are gradually <i>increased</i>. This scheme undocks the ligand from the binding site without the need to specify any CV in advance, so the binding free energy can be recovered accordingly. Taken together, it is clear that the field of MD coupled to enhanced sampling schemes has evolved remarkably over the past 10–15 years. This is particularly true with reference to protein–ligand binding, opening up novel avenues and scenarios for computational medicinal chemists.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">6 MD-Derived Kinetics of Drug Binding and Unbinding</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Kinetics represents the physicochemical description of association and dissociation rates of a drug binding to and unbinding from its target. While high affinity for a target is the basic requirement for any potential drug candidate, thermodynamics alone is not enough to comprehensively characterize drug–target binding. Association and dissociation rates depend on transient interactions between the ligand and the surroundings (i.e., protein and solvent), which cannot be captured by a state function such as the binding free energy. In addition, high-affinity ligands can sometimes show unexpectedly poor pharmacological efficacy in vivo, where the equilibrium conditions underlying binding potency are not necessarily met.<a onclick="showRef(event, 'ref201'); return false;" href="javascript:void(0);" class="ref ref201">(201)</a> In this context, researchers recently found MD-based methods to be potentially useful in evaluating ligand-binding association (<i>k</i><sub>on</sub>) and dissociation (<i>k</i><sub>off</sub>) rates.<a onclick="showRef(event, 'ref202'); return false;" href="javascript:void(0);" class="ref ref202">(202)</a> These constants (particularly the <i>k</i><sub>off</sub>) return a direct measure of how long a ligand is likely to remain bound to its receptor, thus generating the desired pharmacological effects. Ligands that remain bound to their receptor for a longer time are pharmacologically more appealing than those characterized by a short-lived complex. The use of MD to estimate <i>k</i><sub>on</sub> and <i>k</i><sub>off</sub> is at the forefront of computational drug discovery. Several different MD-based approaches have been developed to calculate these key observables.<a onclick="showRef(event, 'ref202'); return false;" href="javascript:void(0);" class="ref ref202">(202)</a></div><div class="NLM_p">Considering the reaction scheme for the noncovalent association shown in <a class="ref internalNav" href="#eq6" aria-label="eq 6">eq 6</a>, we see that the rate of forward reaction is of second order in reactant concentrations, whereas the reverse process (i.e., the dissociation of the protein–ligand complex) is of first order. When these rates are combined, we obtain the phenomenological rate equation for the protein–ligand complex:<span class="NLM_disp-formula" id="eq12"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m012.gif" alt="" /></img><span class="labelSpan">(12)</span></span><a class="ref internalNav" href="#eq12" aria-label="Equation 12">Equation 12</a> represents the law of mass action; at the equilibrium (i.e., when d[PL]/d<i>t</i> = 0), the binding constant can be expressed as the ratio of <i>k</i><sub>off</sub> over <i>k</i><sub>on</sub>. Thus, the following expression connects the thermodynamic observable <i>K</i><sub>d</sub> with the kinetics observables <i>k</i><sub>off</sub> and <i>k</i><sub>on</sub> at equilibrium conditions:<span class="NLM_disp-formula" id="eq13"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m013.gif" alt="" /></img><span class="labelSpan">(13)</span></span></div><div class="NLM_p">From a microscopic point of view, the (un)binding process can be described as a double-welled one-dimensional PMF (see <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). The barrier separating the two minima is assumed to be high enough that the transitions from one basin (i.e., the bound state) to the other (i.e., the unbound state) can be considered as rare events compared to the intrabasin dynamics. According to transition state theory, the rate constant for these processes is expressed as<a onclick="showRef(event, 'ref203'); return false;" href="javascript:void(0);" class="ref ref203">(203)</a><span class="NLM_disp-formula" id="eq14"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m014.gif" alt="" /></img><span class="labelSpan">(14)</span></span>where Δ<i>G</i><sup>⧧</sup> is the activation free energy (or free-energy barrier), whereas <i>k</i><sub>0</sub> is a proportionality constant (i.e., the pre-exponential factor, or Arrhenius constant), which takes into account the frequency of transition attempts as well as the probability of recrossing events from the transition state. <a class="ref internalNav" href="#eq14" aria-label="Equation 14">Equation 14</a> shows the exponential relationship between kinetic constant and activation free energy. This makes any computational prediction of kinetics particularly challenging.</div><div class="NLM_p">Shan et al.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and Buch et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> conducted the first computational studies wherein the entire binding process was revealed and analyzed at the atomic level from MD simulations. Notably, Buch et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> characterized the trypsin–benzamidine complex formation. They applied Markov state model (MSM)<a onclick="showRef(event, 'ref204'); return false;" href="javascript:void(0);" class="ref ref204">(204)</a> analysis, a mathematical approach borrowed from protein-folding studies, to calculate both the thermodynamics and kinetics observables related to drug–target complex formation. The key idea is to discretize the configurational space of the system under investigation using some structural metric (usually the rmsd) and traditional clustering techniques. Then, the stochastic jumps between states are modeled by counting the number of transitions observed in the simulation trajectories during a certain lag time. This generates the transition probabilities matrix, which includes both the structural transitions (eigenvectors) of the investigated event and the corresponding time scale (eigenvalues). This indicates possible pathways between initial and final states and reactive fluxes between them. Thus, MSM can predict the equilibrium distribution of states and kinetic quantities for events occurring on time scales longer than those reached through the ensemble of the MD simulations that are actually performed. This returns an understandable picture of the investigated event through a simplified kinetic model, which does not rely on physical reaction coordinates. Several informative reviews have provided a more thorough and exhaustive introduction to MSM theory.<a onclick="showRef(event, 'ref204 ref205'); return false;" href="javascript:void(0);" class="ref ref204 ref205">(204, 205)</a></div><div class="NLM_p">Recently, researchers using MSM to resolve ligand-binding kinetics have begun to include the role played by protein conformational transitions in the entire ligand recognition process. By studying the binding of choline to ChoX and using a flux analysis, Gu et al. quantified that a contribution of about 90% of conformational selection over induced fit is expected to operate under experimental conditions.<a onclick="showRef(event, 'ref206'); return false;" href="javascript:void(0);" class="ref ref206">(206)</a> Platter and Noé investigated the trypsin–benzamidine system, identifying a combination of conformational selection and induced-fit mechanisms in the binding kinetics.<a onclick="showRef(event, 'ref207'); return false;" href="javascript:void(0);" class="ref ref207">(207)</a> Notably, an advantage of MSM over standard analysis of MD trajectories is the possibility of identifying undersampled states along the (un)binding process and thus of lowering the statistical uncertainty on the calculated observables by rerunning simulations starting from the more relevant configurations (seeding).<a onclick="showRef(event, 'ref204'); return false;" href="javascript:void(0);" class="ref ref204">(204)</a> This strategy, which is opposed to the brute force extension of previous simulations, is usually referred to as adaptive sampling. To this regard, the on-the-fly learning method devised by Doerr and De Fabritiis seems to be particularly appealing not only because the adaptive sampling is totally unsupervised but also because the iterative seeding allows one to converge thermodynamic quantities at about 1 order of magnitude faster than conventional approaches.<a onclick="showRef(event, 'ref208'); return false;" href="javascript:void(0);" class="ref ref208">(208)</a> There is a growing number of software packages aimed at constructing and visualizing MSM-derived kinetic models, such as MSMBuilder and pyEMMA. These will most likely increase the popularity of this technology in more application-oriented investigations too.</div><div class="NLM_p">Still relying on extensive and unbiased MD simulations, Decherchi et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> provided a novel approach to calculating <i>k</i><sub>on</sub>. Using available experimental thermodynamic (<i>K</i><sub>i</sub>) and kinetic (<i>k</i><sub>off</sub>) data, they estimated the time needed for the very first molecule of a ligand to move from the bulk of the solvent into the target binding pocket (the “experimental time for first binding”). Under their simulation conditions, the experimental time for first binding was estimated to be about 250 ns. Then, using all the trajectories that led to a binding event (11 out of 13 MD simulations of about 1 μs each), they obtained the “calculated time for first binding”. This was about 220 ns, in very good agreement with the experimental estimate. For prospective studies, this may offer a further way of estimating <i>k</i><sub>on</sub> and comparing computational predictions with experimental data.</div><div class="NLM_p">However, from a drug discovery standpoint, it is more relevant to estimate and optimize the unbinding kinetics, <i>k</i><sub>off</sub>. This is because the dissociation of protein–ligand complexes, which follows an exponential decay with the characteristic time τ<sub>R</sub> = 1/<i>k</i><sub>off</sub>, is quite a reliable indicator of in vivo drug efficacy.<a onclick="showRef(event, 'ref201 ref209'); return false;" href="javascript:void(0);" class="ref ref201 ref209">(201, 209)</a> This is the average time required for the complex to dissociate and is usually referred to as “residence time”. The long time scales involved in the dissociation of protein–ligand complexes can last from milliseconds to seconds (or even more), which makes it remarkably difficult to simulate dissociation events through brute force MD. In this respect, MSM applied to long and unbiased MD trajectories has been reported to provide estimations of unbinding kinetics rates.<a onclick="showRef(event, 'ref205'); return false;" href="javascript:void(0);" class="ref ref205">(205)</a> However, in several notable examples reported, the error on the <i>k</i><sub>off</sub> estimation was quite high. This is most likely because unbiased MD simulations can perform very limited sampling in the region around the unbinding transition states.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> As mentioned above, the exponential relationship between <i>k</i><sub>off</sub> and activation free energy amplifies the error in the <i>k</i><sub>off</sub> prediction. This is mainly because sampling around transition states remains poor and the unbinding activation free energies are rather approximate.</div><div class="NLM_p">Mollica et al. recently suggested a practical solution for screening the residence time of a congeneric series of compounds.<a onclick="showRef(event, 'ref210'); return false;" href="javascript:void(0);" class="ref ref210">(210)</a> The authors used scaled MD, an enhanced sampling method with which the potential energy of the whole system is reduced by an arbitrary λ factor. In doing so, they observed several unbinding events and acquired enough statistics to correctly rank the dissociation constants in pharmacologically relevant case studies. Notably, this methodology is CV-free, requiring no preidentification or predefinition of a reaction coordinate or of collective variables. Therefore, the protocol is fully unsupervised, as no (or just a little) a priori information about the unbinding path must be known. In addition, the good correlation between computed residence times and experimental <i>k</i><sub>off</sub> values favors this approach for kinetics prediction. But prospective application studies are needed to definitively assess its robustness and actual applicability for drug discovery. Indeed, unbinding kinetics is emerging as a crucial parameter for fine-tuning during lead optimization for drug discovery so that compounds can be prioritized during chemical synthesis campaigns. Conversely, even fast approaches applied to binding kinetics estimation can be much slower than conventional virtual screening methodologies, which limits their applicability to hit identification for drug discovery.</div><div class="NLM_p last">Researchers have proposed and successfully applied other approaches to evaluating ligand-binding kinetics within the framework of CV-based enhanced sampling methods. Once the free-energy barriers are determined, the kinetic constant can in principle be estimated through <a class="ref internalNav" href="#eq14" aria-label="eq 14">eq 14</a> only after assessing the pre-exponential factor. For example, in their seminal work, Bui et al.<a onclick="showRef(event, 'ref211'); return false;" href="javascript:void(0);" class="ref ref211">(211)</a> estimated rate constants for tetramethylammonium (un)binding to AChE using US to reconstruct the PMF. They then derived the pre-exponential factor through transition state theory arguments. Despite this important result, to the best of our knowledge, this and related approaches are mostly effective for simpler computational problems. This is mostly due to difficulties with estimating the pre-exponential factor with reasonable accuracy for complex processes such as protein–ligand associations/dissociations. Marinelli et al.<a onclick="showRef(event, 'ref212'); return false;" href="javascript:void(0);" class="ref ref212">(212)</a> conceived an elegant procedure to address this. They first used BEmetaD<a onclick="showRef(event, 'ref180'); return false;" href="javascript:void(0);" class="ref ref180">(180)</a> to reconstruct a multidimensional free-energy landscape for the investigated process. They then used a discrete-state kinetic-MC simulation to build a consistent kinetic model.<a onclick="showRef(event, 'ref212'); return false;" href="javascript:void(0);" class="ref ref212">(212)</a> They later used this approach to successfully compute the binding free energy and corresponding association/dissociation rates for a peptide substrate to the HIV-1 protease.<a onclick="showRef(event, 'ref213'); return false;" href="javascript:void(0);" class="ref ref213">(213)</a> Conversely, Tiwary and Parrinello<a onclick="showRef(event, 'ref214'); return false;" href="javascript:void(0);" class="ref ref214">(214)</a> devised a way to recover unbiased rates from metadynamics-biased simulations, bypassing the problems related to the pre-exponential factor.<a onclick="showRef(event, 'ref215'); return false;" href="javascript:void(0);" class="ref ref215">(215)</a> Their approach assumed that as long as the Gaussian deposition frequency is sufficiently small, the well-to-well dynamics is not affected by the bias potential. Unbiased rate constants could therefore be estimated after assessing the acceleration factor introduced by metadynamics. Notably, the authors showed a faster convergence of rates rather than the full free-energy reconstruction, which makes this approach particularly appealing for drug discovery.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">7 Other Challenges for MD in SBDD</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27428" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27428" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Above, we discussed how MD-based methods can be used to investigate and understand binding affinity and kinetics for rational drug design. Now, we briefly touch upon the use of MD to tackle two additional topics: allosteric mechanisms and modulation, and the role of water for ligand binding and optimization. Both topics have been actively investigated using MD-based approaches over the past decade, as they have become major research areas for computationally driven drug discovery. This has led to several methodological advances, increasing our comprehension of these complex biological phenomena, which could play a role in drug discovery.</div><div id="sec7_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">7.1 MD and Allosteric Modulation in Drug Design</h3><div class="NLM_p">Allostery occurs when distant binding pockets of biological macromolecules, mostly proteins, communicate and so modulate their activity. Interfering with this allosteric process can thus regulate target function acting far from the catalytic (orthosteric) site of proteins. This offers a new strategy for tuning target activity through allosteric ligands (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Allosteric modulation in BCR-ABL (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f4j">2f4j</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5v">3k5v</a>). The ligand GNF-2 binds to the myristate pocket (red circle, allosteric site) concomitantly to the bending of the αI′-helix and induces a c-ABL-like autoinhibited conformation of BCR-ABL that has reduced kinase activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Since Monod and Jacob first introduced the term “allostery” in 1961, two major conceptual models have emerged for describing this communication between distinct binding sites: the Monod, Wyman, and Changeux model (MWC) and the Koshland, Nemethy, and Filmer model (KNF).<a onclick="showRef(event, 'ref216'); return false;" href="javascript:void(0);" class="ref ref216">(216)</a> These phenomenological descriptions mirror the more familiar concepts of population shift (or conformational selection) and induced fit, respectively. Allostery is usually associated with long-range propagation of large conformational movements (domain motions, hinge-bending movements, etc.). Notably, however, it can sometimes be related to alterations in dynamics between distinct binding sites with no major conformational changes. Indeed, while structural changes are largely driven by enthalpy, alterations in dynamics due to an allosteric binding (such as changes in frequencies and amplitudes of thermal fluctuations) are primarily entropic in nature.<a onclick="showRef(event, 'ref217'); return false;" href="javascript:void(0);" class="ref ref217">(217)</a></div><div class="NLM_p">For drug design, targeting allosteric binding sites offers several potential advantages over traditional orthosteric sites. These can be summarized as (i) improved selectivity, (ii) improved druggability, (iii) activity rescuing, and (iv) activity potentiation.<a onclick="showRef(event, 'ref218'); return false;" href="javascript:void(0);" class="ref ref218">(218)</a> The potential for improved selectivity is because orthosteric inhibition is often associated with poor specificity toward a given target, such as protein kinases or GPCRs. This is mainly due to different targets within the same family sharing similarities in their orthosteric sites. Allosteric ligands thus offer a viable strategy for targeting binding sites that are topologically diverse and often less conserved, which could in principle allow better target selectivity. In addition, allosteric modulation expands the druggability of a given target to include different pockets of the same target protein. Furthermore, it offers the possibility of rescuing the activity of dysregulated proteins caused by disease-related mutations, which often lead to drug-resistance issues. Mutation of key amino acids can prevent the binding and efficacy of drugs targeting the orthosteric site. In this case, the allosteric effector can restore the protein function by canceling or surmounting the conformational change associated with the dysregulated activity of mutated/resistant forms of the targeted proteins. Finally, allosteric ligands could potentiate the target activity, with examples including benzodiazepines for GABAA<a onclick="showRef(event, 'ref219'); return false;" href="javascript:void(0);" class="ref ref219">(219)</a> and galantamine for nAChRs.<a onclick="showRef(event, 'ref220'); return false;" href="javascript:void(0);" class="ref ref220">(220)</a></div><div class="NLM_p">The potential advantages are, however, counterbalanced by some serious challenges in effectively discovering and optimizing small molecule allosteric ligands. These are mainly related to the structural and dynamic nature of allosteric pockets, which are usually rather shallow, superficial, and highly flexible or even transient. These difficult features are then reflected in allosteric ligands with poorer affinity (in the low μM range) for the target than orthosteric inhibitors, which usually reach low nM potency. It can also be particularly challenging to experimentally verify allosteric modulation by new compounds. Sophisticated functional assays must be put in place to verify and confirm the allosteric mode of action of new ligands. Finally, poor target affinity makes it more difficult to structurally resolve allosteric ligand/target complexes by X-ray crystallography.</div><div class="NLM_p">In this context, MD-based approaches could be used to detect and characterize allosteric binding sites. The current time scales of MD simulations allow the formation of transient pockets to be observed.<a onclick="showRef(event, 'ref221 ref222'); return false;" href="javascript:void(0);" class="ref ref221 ref222">(221, 222)</a> Enhanced sampling techniques for MD-based conformational sampling further increase the ability of MD simulations to detect allosteric pockets, as recently reported for accelerated MD.<a onclick="showRef(event, 'ref223'); return false;" href="javascript:void(0);" class="ref ref223">(223)</a> Through MD simulation runs, researchers can also identify communication pathways between putative allosteric sites and the active site of the targeted protein. This challenging task has been addressed by new methods, which borrow from the concepts on which the MWC allosteric model is based. That is, most MD data analysis methods rely on identifying correlated motions through which the structural and/or dynamic information can be transmitted between distal pockets. Well-established methods are based on calculating the covariance matrix of atomic fluctuations:<a onclick="showRef(event, 'ref224'); return false;" href="javascript:void(0);" class="ref ref224">(224)</a><span class="NLM_disp-formula" id="eq15"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m015.gif" alt="" /></img><span class="labelSpan">(15)</span></span>where <b>r</b><sub><i>i</i></sub> and <b>r</b><sub><i>j</i></sub> are the instantaneous position vectors of atoms <i>i</i> and <i>j</i>, respectively, in the reference frame of the protein, and the brackets stand for ensemble average. The diagonal elements of the matrix correspond to mean-squared fluctuation of atoms and are related to their <i>B</i>-factors. Upon diagonalization of the covariance matrix (or principal component analysis, PCA), one can obtain a set of orthogonal modes of motions that maximize the fluctuation amplitudes along each mode and that represent a quasi-harmonic approximation of the free-energy surface of the protein.<a onclick="showRef(event, 'ref224'); return false;" href="javascript:void(0);" class="ref ref224">(224)</a> The “essential subspace” of protein dynamics can also be used to visualize correlated motions with common graphical software. Moreover, achieving the essential subspace corresponds to extracting relevant CVs, along which the conformational free energy of the protein can be projected and, if needed, resampled. Similarly, the cross-correlation matrix can be calculated by normalizing the elements of the covariance matrix:<span class="NLM_disp-formula" id="eq16"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/jm-2015-016843_m016.gif" alt="" /></img><span class="labelSpan">(16)</span></span>The values of the elements of the cross-correlation matrix range from −1 for fully anticorrelated motions to +1 for fully correlated motions. They correspond to the Pearson correlation coefficient. In contrast to the elements of covariance matrix, however, the magnitude of the correlated motion is lost.</div><div class="NLM_p">Both analyses, however, suffer from two major drawbacks. The first limitation is because atomic correlation is only detected for parallel motions. Due to the form of <a class="ref internalNav" href="#eq15" aria-label="eq 15">eq 15</a>, fully correlated motions of two atoms oscillating along perpendicular directions will return a null correlation. Moreover, the Pearson coefficient only treats linear correlations, excluding nonlinear or higher-order correlations. To overcome this, Lange and Grubmüller devised a generalized metric, called mutual information (MI),<a onclick="showRef(event, 'ref225'); return false;" href="javascript:void(0);" class="ref ref225">(225)</a> which quantifies the correlation between random variables as the deviation of their joint probability distributions from the hypothetical distributions of independent random variables. Thus, MI returns a null value only in the case of fully uncorrelated motions but properly detects any kind of correlation. In analogy to PCA, the same authors developed a consistent way to extract collective degrees of freedom based on the MI metric (full correlation analysis, FCA).<a onclick="showRef(event, 'ref226'); return false;" href="javascript:void(0);" class="ref ref226">(226)</a></div><div class="NLM_p">The second limitation that affects metrics based on atomic fluctuations (including the previously reported implementation of MI) is that they are based on Cartesian coordinates. Hence, they are appropriate for detecting major conformational changes that are typical of classical allosteric sites. Conversely, the MutInf approach developed by McClendon et al.<a onclick="showRef(event, 'ref227'); return false;" href="javascript:void(0);" class="ref ref227">(227)</a> is related to MI, but the analysis is performed on the internal degrees of freedom (backbone and side chains). Moreover, relying on an entropy-based metric (as well as MI), this approach seems to be particularly suited to detecting minor or subtle alterations in dynamics, such as those involved in entropic-driven allosteric modulations. A robust statistical analysis based on Bayesian filtering and sampling penalties further strengthens an unambiguous detection of relevant correlated motions. Notably, these correlation methods are often complemented by graph-theory and community network analysis in order to characterize communication channels between distal binding sites. For example, Sethi et al.<a onclick="showRef(event, 'ref228'); return false;" href="javascript:void(0);" class="ref ref228">(228)</a> identified signaling pathways in different tRNA–protein complexes by building a dynamical network where the nodes represent amino acids or nucleotides, whereas the weight of the edges (<i>w</i><sub><i>ij</i></sub>) is proportional to the information transfer between the two nodes <i>i</i> and <i>j</i> quantified in terms of the Pearson coefficient (<i>w</i><sub><i>ij</i></sub> = −log(|<i>C</i><sub><i>ij</i></sub>|)). Community network analysis<a onclick="showRef(event, 'ref229'); return false;" href="javascript:void(0);" class="ref ref229">(229)</a> is thus used to identify loosely interconnected but locally densely intraconnected substructures. The allosteric signal is then detected as a function of the number of shortest paths between critical nodes belonging to different communities. Similarly, Rivalta et al.<a onclick="showRef(event, 'ref230'); return false;" href="javascript:void(0);" class="ref ref230">(230)</a> identified the allosteric pathways in imidazole glycerol phosphate synthase, using the same theoretical framework, although operating with MI as a measure of the information transfer between nodes. There are also complementary approaches that focus on energy couplings rather than on correlated motions alone. One group recently proposed an elegant combined approach based on structural fluctuation analysis and pairwise energy decomposition to characterize the allosteric mechanism for the homologous PDZ2 and PDZ3 proteins.<a onclick="showRef(event, 'ref231'); return false;" href="javascript:void(0);" class="ref ref231">(231)</a></div><div class="NLM_p last">Nowadays, extended MD simulations can be combined with a growing number of methods for analyzing correlated motions. This has generated several informative MD-based studies of allosteric modulation in different target families. These include Foda et al.’s study on allosteric network and binding cooperativity in Src protein kinases<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> and Morra et al.’s integrated MD/dynamic pharmacophore approach for identifying and targeting allosteric hot-spots on the N-terminal domain of the molecular chaperone Hsp90.<a onclick="showRef(event, 'ref232'); return false;" href="javascript:void(0);" class="ref ref232">(232)</a> Here too, the field is in its infancy. Future MD applications and analysis methods are needed to definitively assess the reliability of these protocols for prospective drug discovery, in particular for contributing to the search for novel allosteric modulators as drug candidates in different therapeutic areas.</div></div><div id="sec7_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">7.2 MD To Investigate the Role of Waters in Drug Binding</h3><div class="NLM_p">Water molecules can influence the binding affinity of a ligand to its targeted biomolecule in different ways (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>). First, interfacial waters can mediate the initial approach of the ligand to the pocket, with an active role in determining ligand–protein binding or rejection. Desolvation of the binding pocket is then necessary for drug binding. During this process, a network of waters could affect ligand binding, raising complex considerations of whether or not the displacement of ordered waters can produce an entropy gain, ultimately aiding binding affinity.<a onclick="showRef(event, 'ref233 ref234'); return false;" href="javascript:void(0);" class="ref ref233 ref234">(233, 234)</a> For example, it can be very challenging to determine from structural data alone whether a key bridging water, which mediates the ligand–target interaction in the crystal structure of the complex, should be maintained or displaced during ligand optimization.<a onclick="showRef(event, 'ref235 ref236'); return false;" href="javascript:void(0);" class="ref ref235 ref236">(235, 236)</a> The entropic reward derived from releasing an ordered water molecule into the bulk should be coupled to an enthalpic gain generated by forming tight protein–ligand interactions. Clearly, this area presents major scientific challenges related to better understanding of the fundamental principles that govern drug binding. Thus, rational drug design could be greatly impacted by methods and approaches to understand and quantify the crucial entropic/enthalpic balance due to water displacement or solvent reorganization upon ligand binding.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Water-mediated protein–ligand interactions from crystallographic structures. (A) Bound pose of deoxythymidine at the binding site of thymidine kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM">1KIM</a>) and (B) bound pose of a benzisoxazole-based derivative at the binding site of HSP90 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BMY">3BMY</a>). Ligands, relevant protein residues, and bridging water molecules are reported in ball and stick representation. Water molecules are highlighted by red spheres. The boundaries of the binding pocket are represented by a transparent white mesh. Dotted red lines represent the most relevant hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Over the past decade, MD simulations have been extensively used to characterize waters located in the binding sites, with a focus on those buried waters with long residence times in protein structures. These often raise questions as to whether or not water-mediated interactions should be retained to improve lead compound potency. Major progress has been made in locating water molecules in the targeted binding pocket and identifying and classifying specific water molecules that should be displaced and retained to improve binding affinity. Thus, there are now several computational methods for better quantifying the enthalpy–entropy compensation in protein–ligand binding.<a onclick="showRef(event, 'ref237'); return false;" href="javascript:void(0);" class="ref ref237">(237)</a></div><div class="NLM_p last">Nowadays, long MD trajectories can serve to sample the solvation network of binding sites, revealing hydration patterns within the binding pocket that can complement or support structural data. For example, MD-based approaches have been used to characterize the role of the interfacial waters during ligand approach and binding.<a onclick="showRef(event, 'ref238 ref239 ref240'); return false;" href="javascript:void(0);" class="ref ref238 ref239 ref240">(238-240)</a> From MD simulations, one can identify and analyze water sites by the peaks in water density or averaging over water molecule locations during the collected trajectories. Also, from short MD runs, Lazaradis et al.’s inhomogeneous fluid solvation theory (IFST)<a onclick="showRef(event, 'ref241'); return false;" href="javascript:void(0);" class="ref ref241">(241)</a> conveniently allows quantification of the entropic and enthalpic contributions and overall thermodynamics of water molecules in the binding site. More recently, researchers have developed methods like WaterMap, which uses IFST, to map the locations and thermodynamic properties of water molecules that solvate protein binding sites, indicating which should be removed or retained to improve ligand binding affinity.<a onclick="showRef(event, 'ref242'); return false;" href="javascript:void(0);" class="ref ref242">(242)</a> Similarly, researchers have reported, several other methods, such as waterFLAP, for characterizing waters in the binding sites. These work by coupling (short) MD trajectories to other methods/approaches such as docking and scoring, or MM-PB/GBSA calculations.<a onclick="showRef(event, 'ref243'); return false;" href="javascript:void(0);" class="ref ref243">(243)</a> This field is vast and increasing, with the potential to more directly link the in-depth description of thermodynamics of binding site waters with the rational design of new, more efficient ligands.</div></div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">8 Summary and Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56774" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56774" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As extensive molecular dynamics (MD) becomes ever more affordable, it promises to impact fast-paced drug discovery programs. Mainly because of the advent of GPUs and software codes that can fully exploit these innovative hardware architectures, it is nowadays possible to run MD simulations in the time frame of microseconds up to a few milliseconds. This allows a thorough sampling of the conformational space, including that of large biomolecules. This can include, for example, the complete description of the pathway of the ligand binding to its target protein. These long MD trajectories can then be coupled to free-energy methods to provide the free-energy profile of protein–ligand binding, with the thermodynamic and kinetic data being crucial for drug discovery.</div><div class="NLM_p">Although brute force MD-based approaches can be quite powerful, they are computationally very demanding. This limits their practical use for drug discovery to just a small number of ligands. In the past few decades, researchers have reported several different strategies for overcoming this by enhancing the sampling of relevant regions of the free-energy surface. Of these, free-energy perturbation (FEP) has best demonstrated its great potential to impact drug discovery. FEP is ready for prime time. We expect that an increasing number of MD-based FEP prospective studies will be published, ultimately proving FEP to be an efficient tool for optimizing a variety of new leads. The very successful MC-based FEP studies reported recently are one such example.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> However, FEP does not describe the route of binding and unbinding, which instead can be depicted by other pathway-dependent enhanced sampling methods. Here, for example, we have extensively covered metadynamics and steered MD, which are among the first methods used to dissect routes for protein–ligand binding and unbinding. Nevertheless, metadynamics depends on the correct definition of the collective variables used to describe the chemical process under investigation. The CVs must be properly identified in order for metadynamics to accurately predict the free-energy landscape related to the ligand-binding process and so aid lead optimization campaigns. This is particularly true given its recent evolution (e.g., WTmetaD). Alternatively, more accurate and expensive protocols of metadynamics could be used to thoroughly investigate binding routes, providing thermodynamic and kinetic profiles associated with drug binding and unbinding. In analogy, when using a single distance-dependent reaction coordinate, steered MD can evaluate binding affinity quickly but only in a qualitative manner. For this reason, we can envision a scenario where steered MD could be used as a postprocessing tool for virtual ligand screening in order to improve the enrichment factor of active hits from among the best-ranked compounds.</div><div class="NLM_p">Binding kinetics, and in particular residence time (i.e., the inverse of <i>k</i><sub>off</sub>), is nowadays considered to be one of the key parameters in lead optimization for potency and efficacy in vivo. In this context, other methods (including scaled MD and similar smoothed-potential approaches) are emerging as suitable for unbinding-kinetics investigations and <i>k</i><sub>off</sub> predictions. Prospective applications will likely appear in the literature in the near future, further testing the actual applicability of this quite novel approach to kinetics predictions.</div><div class="NLM_p">From the above, it is clear that MD-based methods can nowadays help in several key drug discovery steps. The aforementioned methods are just a few of the many MD-based approaches to studying ligand binding. Each deserves attention. Rational drug design will be majorly impacted by the inclusion of full flexibility and entropic effects in studying protein–ligand recognition processes, allosteric modulation, and the thermodynamics and kinetics of binding-site waters. This will ultimately increase understanding of ligand binding, returning a more accurate and quantitative description of this crucial event for drug discovery and development.</div><div class="NLM_p">Although the recent results from MD-based drug discovery studies are very encouraging, we should nevertheless remember that major challenges must be overcome to deepen the impact of MD-based methods on drug design. Improvements in the current force field are expected (and most probably needed) to further progress the accuracy of free-energy predictions. The currently available molecular mechanics force fields partially or fully neglect polarization effects, charge transfer, and many electronic-based interactions (π–π, cation−π, halogen bonds, etc.). In this respect, polarizable force fields or quantum mechanical calculations might be used in future to help refine free-energy estimations for increasingly accurate predictions. The limits of force field and MD-based methods should be pushed to correctly treat other challenging target families (such as metalloproteins), which include many drug discovery targets that can still only be studied with limited accuracy.</div><div class="NLM_p">One additional key challenge in drug discovery comes from the fact that potency, although essential, is only the very first step toward the discovery of a promising drug candidate. Once a potent inhibitor is discovered, this must be tuned into a druglike compound with a favorable ADMET profile, during the lead optimization phase. Most of the time, this is the very key step, and the real bottleneck, in drug discovery. The ability of predicting thermodynamics and kinetics of binding through MD-based methods has the potential to impact lead optimization as well, indicating which compounds and modifications are the most favorable ones. If this will be demonstrated by prospective studies, MD-based methods will take more standard, and static, SBDD approaches to a new level. Toward this goal, we expect and call for research efforts proving that thermodynamics and kinetics of binding, retrieved from MD-based methods, can pragmatically impact the overall complex lead optimization phase of a drug discovery project.</div><div class="NLM_p last">We conclude that the time has come for an operational use of MD and related methods for fast-paced drug discovery, which would offer savings in time and money. Novel methods, novel software, and novel hardware have boosted the widespread diffusion of MD-based methods within the pharmaceutical community. Indeed, MD is on the verge of joining the already large arsenal of computational tools routinely applied in biopharmaceutical drug discovery. Despite the limitations and major challenges in using MD simulations for drug discovery, we therefore conclude by calling for more MD-based prospective studies. Prospective studies will serve as the ultimate proof that MD can indeed be used to assist the costly and highly challenging drug discovery process.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01684" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marco De Vivo</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Molecular Modeling and Drug Discovery, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genova, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">IAS-5/INM-9 Computational
Biomedicine Forschungszentrum Jülich, Wilhelm-Johnen-Straße, 52428 Jülich, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6805091a0b07460c0d1e011e072801011c46011c"><span class="__cf_email__" data-cfemail="96fbf7e4f5f9b8f2f3e0ffe0f9d6ffffe2b8ffe2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Cavalli</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro
6, I-40126 Bologna, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">CompuNet, Istituto
Italiano di Tecnologia, Via Morego
30, 16163 Genova, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#15747b716770743b7674637479797c55607b7c777a3b7c61"><span class="__cf_email__" data-cfemail="98f9f6fceafdf9b6fbf9eef9f4f4f1d8edf6f1faf7b6f1ec">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matteo Masetti</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro
6, I-40126 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giovanni Bottegoni</span> - <span class="hlFld-Affiliation affiliation">CompuNet, Istituto
Italiano di Tecnologia, Via Morego
30, 16163 Genova, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">BiKi Technologies
srl, Via XX Settembre 33/10, 16121 Genova, Italy</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Andrea Cavalli and Giovanni Bottegoni are co-founders of BiKi Technologies, a startup company that develops tools for drug discovery based on molecular dynamics. Giovanni Bottegoni is CEO of the same company.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17410" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17410" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Marco De Vivo</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=BIO-d647e3536-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Marco De Vivo</b> obtained an M.Sc. in Chemistry and a Ph.D. in Pharmaceutical Chemistry in 2004 from the University of Bologna. Then, for 3 years, he was a postdoctoral researcher in the group of Prof. M. L. Klein at the University of Pennsylvania, before joining the structure-based drug design group at Rib-X Pharmaceuticals. In 2009, he joined the Istituto Italiano di Tecnologia (IIT), where he leads the Laboratory of Molecular Modeling and Drug Discovery. He is also a Research Associate at the IAS-5/INM-9 Computational Biomedicine Institute, Forschungszentrum Jülich. His research interests include the computational investigation of pharmaceutically relevant enzymes. In his group, computational insights are fully integrated with experiments to discover promising new compounds potentially endowed with the desired beneficial effect.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Matteo Masetti</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=BIO-d647e3541-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Matteo Masetti</b> received his Ph.D. in Medicinal Chemistry in 2007 from the University of Bologna under the supervision of Maurizio Recanatini. He is now a postdoctoral researcher at the Department of Pharmacy and Biotechology at the same university. His research interests include the study of pharmacotoxicologically relevant membrane proteins and the computational prediction of free energies related to biomolecular interactions.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Giovanni Bottegoni</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=BIO-d647e3546-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Giovanni Bottegoni</b> obtained his Doctorate in Medicinal Chemistry from the University of Bologna in 2006. After a postdoctoral experience at the Scripps Research Institute in La Jolla, CA, U.S., he joined the Istituto Italiano di Tecnologia (IIT), where he was team leader at CompuNet. In 2015, he completed a Master program in International Health Care Management, Economics and Policy (MIHMEP) offered by SDA Bocconi (Milan, IT), graduating cum laude. Currently, he is cofounder and CEO of a start-up company, BiKi Technologies. His main research interests are in in silico polypharmacology and structure-based drug design. In 2015, he was awarded the DCF prize for medicinal chemistry.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Andrea Cavalli</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=BIO-d647e3551-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Andrea Cavalli</b> is a Professor in Medicinal Chemistry at the University of Bologna and Head of CompuNet at the Istituto Italiano di Tecnologia. His research interests are in the field of computational drug discovery. In particular, he has developed and applied computational approaches to accelerate the discovery of druglike compounds in several therapeutic areas, including cancer, neurodegenerative diseases, and neglected tropical diseases. Prof. Cavalli is the author of more than 170 publications in high-ranked journals, co-inventor of several PCT international patents and has delivered numerous invited lectures and seminars. In 2003, he was awarded the “Farmindustria Prize for Pharmaceutical Research”. He is co-founder of the high-tech startup company, BiKi Technologies.</p></figure></div><div class="ack" id="ACK-d647e3556-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77528" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77528" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Istituto Italiano di Tecnologia, the University of Bologna, the PRIN2011 project (A.C. and M.M.), and the Italian Association for Cancer Research (AIRC) through the “MFAG No. 14140” Grant (M.D.V.) for financial support. We thank Dr. Roberto Gaspari for useful discussions, Giuseppina La Sala for technical assistance, and Grace Fox for editing and proofreading the manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i28" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i28"> Abbreviations Used</h2><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure-based drug design</p></td></tr><tr><td class="NLM_term">GPU</td><td class="NLM_def"><p class="first last">graphical processor unit</p></td></tr><tr><td class="NLM_term">MM</td><td class="NLM_def"><p class="first last">molecular mechanics</p></td></tr><tr><td class="NLM_term">FF</td><td class="NLM_def"><p class="first last">force field</p></td></tr><tr><td class="NLM_term">QM</td><td class="NLM_def"><p class="first last">quantum mechanics</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">stochastic dynamics</p></td></tr><tr><td class="NLM_term">PBC</td><td class="NLM_def"><p class="first last">periodic boundary conditions</p></td></tr><tr><td class="NLM_term">PME</td><td class="NLM_def"><p class="first last">particle-mesh Ewald</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free-energy perturbation</p></td></tr><tr><td class="NLM_term">TI</td><td class="NLM_def"><p class="first last">thermodynamic integration</p></td></tr><tr><td class="NLM_term">DDM</td><td class="NLM_def"><p class="first last">double decoupling method</p></td></tr><tr><td class="NLM_term">DAM</td><td class="NLM_def"><p class="first last">double annihilation method</p></td></tr><tr><td class="NLM_term">CV</td><td class="NLM_def"><p class="first last">collective variable</p></td></tr><tr><td class="NLM_term">PMF</td><td class="NLM_def"><p class="first last">potential of mean force</p></td></tr><tr><td class="NLM_term">US</td><td class="NLM_def"><p class="first last">umbrella sampling</p></td></tr><tr><td class="NLM_term">T-REMD</td><td class="NLM_def"><p class="first last">temperature replica exchange MD</p></td></tr><tr><td class="NLM_term">H-REMD</td><td class="NLM_def"><p class="first last">Hamiltonian replica exchange MD</p></td></tr><tr><td class="NLM_term">MRC</td><td class="NLM_def"><p class="first last">multiple receptor conformations</p></td></tr><tr><td class="NLM_term">PARP-1</td><td class="NLM_def"><p class="first last">poly (ADP-ribose) polymerase 1</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">MM-PB</td><td class="NLM_def"><p class="first last">molecular mechanics Poisson–Boltzmann</p></td></tr><tr><td class="NLM_term">GBSA</td><td class="NLM_def"><p class="first last">generalized Born surface area</p></td></tr><tr><td class="NLM_term">MC</td><td class="NLM_def"><p class="first last">Monte Carlo</p></td></tr><tr><td class="NLM_term">NNRTI</td><td class="NLM_def"><p class="first last">non-nucleoside inhibitor of HIV-1 reverse transcriptase</p></td></tr><tr><td class="NLM_term">MIF</td><td class="NLM_def"><p class="first last">migration inhibitory factor MIF</p></td></tr><tr><td class="NLM_term">REST</td><td class="NLM_def"><p class="first last">replica exchange with solute tempering</p></td></tr><tr><td class="NLM_term">WHAM</td><td class="NLM_def"><p class="first last">weighted histogram analysis method</p></td></tr><tr><td class="NLM_term">ABF</td><td class="NLM_def"><p class="first last">adaptive biasing force</p></td></tr><tr><td class="NLM_term">AFM</td><td class="NLM_def"><p class="first last">atomic force microscope</p></td></tr><tr><td class="NLM_term">FabZ</td><td class="NLM_def"><p class="first last">β-hydroxyacylacyl carrier protein dehydratase</p></td></tr><tr><td class="NLM_term">WTmetaD</td><td class="NLM_def"><p class="first last">well-tempered metadynamics</p></td></tr><tr><td class="NLM_term">PCV</td><td class="NLM_def"><p class="first last">path collective variable</p></td></tr><tr><td class="NLM_term">CMAP</td><td class="NLM_def"><p class="first last">contact map distance</p></td></tr><tr><td class="NLM_term">BEmetaD</td><td class="NLM_def"><p class="first last">bias-exchange metadynamics</p></td></tr><tr><td class="NLM_term">HMC</td><td class="NLM_def"><p class="first last">hybrid Monte Carlo</p></td></tr><tr><td class="NLM_term">MSM</td><td class="NLM_def"><p class="first last">Markov state model</p></td></tr><tr><td class="NLM_term">BEDAM</td><td class="NLM_def"><p class="first last">binding energy distribution analysis method</p></td></tr><tr><td class="NLM_term">PCA</td><td class="NLM_def"><p class="first last">principal component analysis</p></td></tr><tr><td class="NLM_term">MWC</td><td class="NLM_def"><p class="first last">Monod, Wyman, and Changeux</p></td></tr><tr><td class="NLM_term">KNF</td><td class="NLM_def"><p class="first last">Koshland, Nemethy, and Filmer</p></td></tr><tr><td class="NLM_term">MI</td><td class="NLM_def"><p class="first last">mutual information</p></td></tr><tr><td class="NLM_term">FCA</td><td class="NLM_def"><p class="first last">full correlation analysis</p></td></tr><tr><td class="NLM_term">PDZ</td><td class="NLM_def"><p class="first last">PSD-95/SAP-90, disc large, zonulla occludens-1</p></td></tr><tr><td class="NLM_term">IFST</td><td class="NLM_def"><p class="first last">inhomogeneous fluid solvation theory</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47644" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47644" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 243 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">The many roles of computation in drug discovery</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1818</span><span class="refDoi"> DOI: 10.1126/science.1096361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.1096361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15031495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1813-1818&author=W.+L.+Jorgensen&title=The+many+roles+of+computation+in+drug+discovery&doi=10.1126%2Fscience.1096361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Many Roles of Computation in Drug Discovery</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1813-1818</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  An overview is given on the diverse uses of computational chem. in drug discovery.  Particular emphasis is placed on virtual screening, de novo design, evaluation of drug-likeness, and advanced methods for detg. protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPr2xRlx_bxbVg90H21EOLACvtfcHk0liG4e12k829dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D&md5=77b2fac63e750ca7cdc7e4f35654549a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1096361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1096361%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DThe%2520many%2520roles%2520of%2520computation%2520in%2520drug%2520discovery%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1813%26epage%3D1818%26doi%3D10.1126%2Fscience.1096361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Bridging quantum mechanics and structure-based drug design</span> <span class="citation_source-journal">Front. Biosci., Landmark Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1619</span><span class="NLM_x">–</span> <span class="NLM_lpage">1633</span><span class="refDoi"> DOI: 10.2741/3809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2741%2F3809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21196252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXislCks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=1619-1633&author=M.+De+Vivo&title=Bridging+quantum+mechanics+and+structure-based+drug+design&doi=10.2741%2F3809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Bridging quantum mechanics and structure-based drug design</span></div><div class="casAuthors">De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1619-1633</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  The last decade has seen great advances in the use of quantum mechanics (QM) to solve biol. problems of pharmaceutical relevance.  For instance, enzymic catalysis is often investigated by means of the so-called QM/MM approach, which uses QM and mol. mechanics (MM) methods to det. the (free) energy landscape of the enzymic reaction mechanism.  Here, I will discuss a few representative examples of QM and QM/MM studies of important metalloenzymes of pharmaceutical interest (i.e. metallophosphatases and metallo-beta-lactamases).  This review article aims to show how QM-based methods can be used to elucidate ligand-receptor interactions.  The challenge is then to exploit this knowledge for the structure-based design of new and potent inhibitors, such as transition state (TS) analogs that resemble the structure and physicochem. properties of the enzymic TS.  Given the results and potential expressed to date by QM-based methods in studying biol. problems, the application of QM in structure-based drug design will likely increase, making of these once-prohibitive computations a routinely used tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop9buFmXM2srVg90H21EOLACvtfcHk0lhGIgxnsFiPVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXislCks7c%253D&md5=0db260d6c1d81db9caf0b45f519d3e85</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2741%2F3809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3809%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DBridging%2520quantum%2520mechanics%2520and%2520structure-based%2520drug%2520design%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2011%26volume%3D16%26spage%3D1619%26epage%3D1633%26doi%3D10.2741%2F3809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Durrant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations and drug discovery</span> <span class="citation_source-journal">BMC Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="refDoi"> DOI: 10.1186/1741-7007-9-71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1186%2F1741-7007-9-71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22035460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=71&author=J.+Durrantauthor=J.+A.+McCammon&title=Molecular+dynamics+simulations+and+drug+discovery&doi=10.1186%2F1741-7007-9-71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations and drug discovery</span></div><div class="casAuthors">Durrant, Jacob D.; Andrew McCammon, J.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71</span>CODEN:
                <span class="NLM_cas:coden">BBMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1741-7007</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">This review discusses the many roles atomistic computer simulations of macromol. (for example, protein) receptors and their assocd. small-mol. ligands can play in drug discovery, including the identification of cryptic or allosteric binding sites, the enhancement of traditional virtual-screening methodologies, and the direct prediction of small-mol. binding energies.  The limitations of current simulation methodologies, including the high computational costs and approxns. of mol. forces required, are also discussed.  With const. improvements in both computer power and algorithm design, the future of computer-aided drug design is promising; mol. dynamics simulations are likely to play an increasingly important role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVpt_0gt58jbVg90H21EOLACvtfcHk0lhGIgxnsFiPVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu73L&md5=34c37576ca19c71701afdb593c370a5f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2F1741-7007-9-71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1741-7007-9-71%26sid%3Dliteratum%253Aachs%26aulast%3DDurrant%26aufirst%3DJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520and%2520drug%2520discovery%26jtitle%3DBMC%2520Biol.%26date%3D2011%26volume%3D9%26spage%3D71%26doi%3D10.1186%2F1741-7007-9-71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Harvey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">High-throughput molecular dynamics: the powerful new tool for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1059</span><span class="NLM_x">–</span> <span class="NLM_lpage">1062</span><span class="refDoi"> DOI: 10.1016/j.drudis.2012.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2012.03.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1059-1062&author=M.+J.+Harveyauthor=G.+De+Fabritiis&title=High-throughput+molecular+dynamics%3A+the+powerful+new+tool+for+drug+discovery&doi=10.1016%2Fj.drudis.2012.03.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DHarvey%26aufirst%3DM.%2BJ.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DHigh-throughput%2520molecular%2520dynamics%253A%2520the%2520powerful%2520new%2520tool%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D1059%26epage%3D1062%26doi%3D10.1016%2Fj.drudis.2012.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Boehr, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, P. E.</span><span> </span><span class="NLM_article-title">The role of dynamic conformational ensembles in biomolecular recognition</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">789</span><span class="NLM_x">–</span> <span class="NLM_lpage">796</span><span class="refDoi"> DOI: 10.1038/nchembio.232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnchembio.232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19841628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OnsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=789-796&author=D.+D.+Boehrauthor=R.+Nussinovauthor=P.+E.+Wright&title=The+role+of+dynamic+conformational+ensembles+in+biomolecular+recognition&doi=10.1038%2Fnchembio.232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The role of dynamic conformational ensembles in biomolecular recognition</span></div><div class="casAuthors">Boehr, David D.; Nussinov, Ruth; Wright, Peter E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">789-796</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mol. recognition by biomols. is central to all biol. processes.  For the past 50 yr, the 'induced fit' hypothesis of D. E. Koshland (1958) has been the textbook explanation for mol. recognition events.  However, recent exptl. evidence supports an alternative mechanism.  The 'conformational selection' model postulates that all conformations pre-exist, and that the ligand selects the most favored conformation.  Following binding, the ensemble undergoes a population shift, redistributing the conformational states.  Both conformational selection and induced fit appear to play roles.  Following binding by a primary conformational selection event, optimization of side-chain and backbone interactions is likely to proceed by an induced fit mechanism.  Conformational selection has been obsd. for protein-ligand, protein-protein, protein-DNA, protein-RNA and RNA-ligand interactions.  These data support a new mol. recognition paradigm for processes as diverse as signaling, catalysis, gene regulation, and protein aggregation in disease, which has the potential to significantly impact researchers' views and strategies in drug design, biomol. engineering, and mol. evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGputCrvx3HqSLVg90H21EOLACvtfcHk0lhqdSnoad059Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OnsLrJ&md5=984e03a4380d9dafd19048a4b45589c6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.232%26sid%3Dliteratum%253Aachs%26aulast%3DBoehr%26aufirst%3DD.%2BD.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DP.%2BE.%26atitle%3DThe%2520role%2520of%2520dynamic%2520conformational%2520ensembles%2520in%2520biomolecular%2520recognition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D789%26epage%3D796%26doi%3D10.1038%2Fnchembio.232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Changeux, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelstein, S.</span><span> </span><span class="NLM_article-title">Conformational selection or induced fit? 50 years of debate resolved</span> <span class="citation_source-journal">F1000 Biol. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="refDoi"> DOI: 10.3410/B3-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.3410%2FB3-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21941598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC3MflvVWisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=19&author=J.-P.+Changeuxauthor=S.+Edelstein&title=Conformational+selection+or+induced+fit%3F+50+years+of+debate+resolved&doi=10.3410%2FB3-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational selection or induced fit? 50 years of debate resolved</span></div><div class="casAuthors">Changeux Jean-Pierre; Edelstein Stuart</div><div class="citationInfo"><span class="NLM_cas:title">F1000 biology reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Exactly 50 years ago, biochemists raised the question of the mechanism of the conformational change that mediates "allosteric" interactions between regulatory sites and biologically active sites in regulatory/receptor proteins.  Do the different conformations involved already exist spontaneously in the absence of the regulatory ligands (Monod-Wyman-Changeux), such that the complementary protein conformation would be selected to mediate signal transduction, or do particular ligands induce the receptor to adopt the conformation best suited to them (Koshland-Nemethy-Filmer-induced fit)? This is not just a central question for biophysics, it also has enormous importance for drug design.  Recent advances in techniques have allowed detailed experimental and theoretical comparisons with the formal models of both scenarios.  Also, it has been shown that mutated receptors can adopt constitutively active confirmations in the absence of ligand.  There have also been demonstrations that the atomic resolution structures of the same protein are essentially the same whether ligand is bound or not.  These and other advances in past decades have produced a situation where the vast majority of the data using different categories of regulatory proteins (including regulatory enzymes, ligand-gated ion channels, G protein-coupled receptors, and nuclear receptors) support the conformational selection scheme of signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTRVYQlbx4kqkloaUqgkFHkfW6udTcc2eY5YfB3wpY3Qbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MflvVWisw%253D%253D&md5=01de9b3981bdbfda147e056f8b223e08</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3410%2FB3-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3410%252FB3-19%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26aulast%3DEdelstein%26aufirst%3DS.%26atitle%3DConformational%2520selection%2520or%2520induced%2520fit%253F%252050%2520years%2520of%2520debate%2520resolved%26jtitle%3DF1000%2520Biol.%2520Rep.%26date%3D2011%26volume%3D3%26spage%3D19%26doi%3D10.3410%2FB3-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Vogt, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Cera, E.</span><span> </span><span class="NLM_article-title">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5894</span><span class="NLM_x">–</span> <span class="NLM_lpage">5902</span><span class="refDoi"> DOI: 10.1021/bi3006913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3006913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5894-5902&author=A.+D.+Vogtauthor=E.+Di+Cera&title=Conformational+selection+or+induced+fit%3F+A+critical+appraisal+of+the+kinetic+mechanism&doi=10.1021%2Fbi3006913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span></div><div class="casAuthors">Vogt, Austin D.; Di Cera, Enrico</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5894-5902</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">For almost 5 decades, 2 competing mechanisms of ligand recognition, conformational selection and induced fit, have dominated the interpretation of ligand binding in biol. macromols.  When binding-dissocn. events are fast compared to conformational transitions, the rate of approach to equil., kobs, becomes diagnostic of conformational selection or induced fit based on whether it decreases or increases, resp., with the ligand concn. [L].  However, this simple conclusion based on the rapid equil. approxn. is not valid in general.  Here, the authors show that conformational selection is assocd. with a rich repertoire of kinetic properties, with kobs decreasing or increasing with [L] depending on the relative magnitude of the rate of ligand dissocn., koff, and the rate of conformational isomerization, kr.  The authors prove that, even for the simplest 2-step mechanism of ligand binding, a decrease in kobs with [L] is unequivocal evidence of conformational selection, but an increase in kobs with [L] is not unequivocal evidence of induced fit.  Ligand binding to glucokinase, thrombin, and its precursor, prethrombin-2, are used as relevant examples.  It is concluded that conformational selection as a mechanism for ligand binding to its target may be far more common than currently believed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdButaqMoNzLVg90H21EOLACvtfcHk0lhqdSnoad059Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D&md5=0f7c66027467c390967075c3be8e5254</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbi3006913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3006913%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DA.%2BD.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DConformational%2520selection%2520or%2520induced%2520fit%253F%2520A%2520critical%2520appraisal%2520of%2520the%2520kinetic%2520mechanism%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5894%26epage%3D5902%26doi%3D10.1021%2Fbi3006913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1038/nchem.1954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnchem.1954" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24950326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=575-583&author=M.+Fischerauthor=R.+G.+Colemanauthor=J.+S.+Fraserauthor=B.+K.+Shoichet&title=Incorporation+of+protein+flexibility+and+conformational+energy+penalties+in+docking+screens+to+improve+ligand+discovery&doi=10.1038%2Fnchem.1954"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span></div><div class="casAuthors">Fischer, Marcus; Coleman, Ryan G.; Fraser, James S.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">575-583</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins fluctuate between alternative conformations, which presents a challenge for ligand discovery because such flexibility is difficult to treat computationally owing to problems with conformational sampling and energy weighting.  Here we describe a flexible docking method that samples and wts. protein conformations using exptl. derived conformations as a guide.  The crystallog. refined occupancies of these conformations, which are observable in an apo receptor structure, define energy penalties for docking.  In a large prospective library screen, we identified new ligands that target specific receptor conformations of a cavity in cytochrome c peroxidase, and we confirm both ligand pose and assocd. receptor conformation predictions by crystallog.  The inclusion of receptor flexibility led to ligands with new chemotypes and phys. properties.  By exploiting exptl. measures of loop and side-chain flexibility, this method can be extended to the discovery of new ligands for hundreds of targets in the Protein Data Bank for which similar exptl. information is available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9q4phmwMLHbVg90H21EOLACvtfcHk0ljHBSL3-N_5_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D&md5=141d6f5238664dabb0890cc5f764b99d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1954%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DIncorporation%2520of%2520protein%2520flexibility%2520and%2520conformational%2520energy%2520penalties%2520in%2520docking%2520screens%2520to%2520improve%2520ligand%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D575%26epage%3D583%26doi%3D10.1038%2Fnchem.1954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">High-throughput docking for lead generation</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">382</span><span class="refDoi"> DOI: 10.1016/S1367-5931(00)00217-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1367-5931%2800%2900217-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11470599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=375-382&author=R.+Abagyanauthor=M.+Totrov&title=High-throughput+docking+for+lead+generation&doi=10.1016%2FS1367-5931%2800%2900217-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput docking for lead generation</span></div><div class="casAuthors">Abagyan, R.; Totrov, M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-382</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Recent improvements in flexible docking technol. may lead to a bigger role for computational methods in lead discovery.  Although fast and accurate computational prediction of binding affinities for an arbitrary mol. is still beyond the limits of current methods, the docking and screening procedures can select small sets of likely lead candidates from large libraries of either com. or synthetically available compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9_FjizU9XdbVg90H21EOLACvtfcHk0ljHBSL3-N_5_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D&md5=c9bfaa493becdadb4386522865b9e7fb</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900217-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900217-9%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DHigh-throughput%2520docking%2520for%2520lead%2520generation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D375%26epage%3D382%26doi%3D10.1016%2FS1367-5931%2800%2900217-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">The future of molecular dynamics simulations in drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1007/s10822-011-9517-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-011-9517-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22183577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2ltb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=15-26&author=D.+W.+Borhaniauthor=D.+E.+Shaw&title=The+future+of+molecular+dynamics+simulations+in+drug+discovery&doi=10.1007%2Fs10822-011-9517-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The future of molecular dynamics simulations in drug discovery</span></div><div class="casAuthors">Borhani, David W.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-26</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mol. dynamics simulations can now track rapid processes-those occurring in less than about a millisecond-at at. resoln. for many biol. relevant systems.  These simulations appear poised to exert a significant impact on how new drugs are found, perhaps even transforming the very process of drug discovery.  We predict here future results we can expect from, and enhancements we need to make in, mol. dynamics simulations over the coming 25 years, and in so doing set out several Grand Challenges for the field.  In the context of the problems now facing the pharmaceutical industry, we ask how we can best address drug discovery needs of the next quarter century using mol. dynamics simulations, and we suggest some possible approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeWiHHq3tLC7Vg90H21EOLACvtfcHk0ljHBSL3-N_5_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2ltb0%253D&md5=78388465d8bb54e99917d8da3d1e0824</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9517-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9517-y%26sid%3Dliteratum%253Aachs%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DThe%2520future%2520of%2520molecular%2520dynamics%2520simulations%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D15%26epage%3D26%26doi%3D10.1007%2Fs10822-011-9517-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Frenkel, D.; Smit, B.</span> <span class="citation_source-book">Understanding Molecular Simulation</span>; <span class="NLM_publisher-name">Academic Press, Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2001</span><span class="NLM_x">; </span>p  <span class="NLM_fpage">638</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=638&author=D.+Frenkel&author=B.+Smit&title=Understanding+Molecular+Simulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFrenkel%26aufirst%3DD.%26btitle%3DUnderstanding%2520Molecular%2520Simulation%26pub%3DAcademic%2520Press%252C%2520Inc%26date%3D2001%26spage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Allen, M. P.; Tildesley, D. J.</span> <span class="citation_source-book">Computer Simulation of Liquids</span>; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">1989</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&author=M.+P.+Allen&author=D.+J.+Tildesley&title=Computer+Simulation+of+Liquids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DM.%2BP.%26btitle%3DComputer%2520Simulation%2520of%2520Liquids%26pub%3DOxford%2520University%2520Press%26date%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelin, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Dynamics of folded proteins</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">585</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span><span class="refDoi"> DOI: 10.1038/267585a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2F267585a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=301613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE2sXlsVOgsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1977&pages=585-590&author=J.+A.+McCammonauthor=B.+R.+Gelinauthor=M.+Karplus&title=Dynamics+of+folded+proteins&doi=10.1038%2F267585a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics of folded proteins</span></div><div class="casAuthors">McCammon, J. Andrew; Gelin, Bruce R.; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5612</span>),
    <span class="NLM_cas:pages">585-90</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The dynamics of a folded globular protein (bovine pancreatic trypsin inhibitor) have been studied by solving the equations of motions for the atoms with an empirical potential energy function.  They provide the magnitude, correlations, and decay of fluctuations about the av. structure.  The protein interior may therefore be fluidlike in that the local atom motions have a diffusional character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZp3fNflIAd7Vg90H21EOLACvtfcHk0ljbzBP5jV_q7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXlsVOgsbg%253D&md5=825f061d862ff7de635fcbaac6e9afbb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2F267585a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F267585a0%26sid%3Dliteratum%253Aachs%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DGelin%26aufirst%3DB.%2BR.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DDynamics%2520of%2520folded%2520proteins%26jtitle%3DNature%26date%3D1977%26volume%3D267%26spage%3D585%26epage%3D590%26doi%3D10.1038%2F267585a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Levitt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warshel, A.</span><span> </span><span class="NLM_article-title">Computer simulation of protein folding</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1975</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">694</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1038/253694a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2F253694a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1167625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE2MXkt1eksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1975&pages=694-698&author=M.+Levittauthor=A.+Warshel&title=Computer+simulation+of+protein+folding&doi=10.1038%2F253694a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Computer simulation of protein folding</span></div><div class="casAuthors">Levitt, Michael; Warshel, Arieh</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5494</span>),
    <span class="NLM_cas:pages">694-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Computer simulation of protein folding was obtained by combining a simple representation of protein conformation, which averaged over the fine details, with convergent energy minimization and normal mode thermalization.  Under certain conditions, the procedure reproduced the correct overall folding of bovine pancreatic trypsin inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNCNUZcQtSobVg90H21EOLACvtfcHk0ljbzBP5jV_q7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXkt1eksbw%253D&md5=fb6bbeb114d0c1a4d95d12a082e01ccf</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F253694a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F253694a0%26sid%3Dliteratum%253Aachs%26aulast%3DLevitt%26aufirst%3DM.%26aulast%3DWarshel%26aufirst%3DA.%26atitle%3DComputer%2520simulation%2520of%2520protein%2520folding%26jtitle%3DNature%26date%3D1975%26volume%3D253%26spage%3D694%26epage%3D698%26doi%3D10.1038%2F253694a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Protein flexibility and drug design: how to hit a moving target</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">447</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span><span class="refDoi"> DOI: 10.1016/S1367-5931(02)00341-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1367-5931%2802%2900341-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12133719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2002&pages=447-452&author=H.+A.+Carlson&title=Protein+flexibility+and+drug+design%3A+how+to+hit+a+moving+target&doi=10.1016%2FS1367-5931%2802%2900341-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Protein flexibility and drug design: how to hit a moving target</span></div><div class="casAuthors">Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-452</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  The most advanced methods for computer-aided drug design and database mining incorporate protein flexibility.  Such techniques are not only needed to obtain proper results; they are also crit. for dealing with the growing body of information from structural genomics.  Advanced methods for incorporating protein flexibility into database searching and computer-aided drug design are not only needed to obtain proper results; they are crit. for dealing with the growing body of information from structural genomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrATwQbEeJJUbVg90H21EOLACvtfcHk0lgyvSDubF_fsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjtbg%253D&md5=eb8ff234381f16086fb8caf2dc88486d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2802%2900341-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252802%252900341-1%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DProtein%2520flexibility%2520and%2520drug%2520design%253A%2520how%2520to%2520hit%2520a%2520moving%2520target%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2002%26volume%3D6%26spage%3D447%26epage%3D452%26doi%3D10.1016%2FS1367-5931%2802%2900341-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Cozzini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellogg, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spyrakis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emerson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fanelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orozco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pertinhez, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotriffer, C.</span><span> </span><span class="NLM_article-title">Target Flexibility: An emerging consideration in drug discovery and design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6237</span><span class="NLM_x">–</span> <span class="NLM_lpage">6255</span><span class="refDoi"> DOI: 10.1021/jm800562d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800562d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFaqtLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6237-6255&author=P.+Cozziniauthor=G.+E.+Kelloggauthor=F.+Spyrakisauthor=D.+J.+Abrahamauthor=G.+Costantinoauthor=A.+Emersonauthor=F.+Fanelliauthor=H.+Gohlkeauthor=L.+A.+Kuhnauthor=G.+M.+Morrisauthor=M.+Orozcoauthor=T.+A.+Pertinhezauthor=M.+Rizziauthor=C.+Sotriffer&title=Target+Flexibility%3A+An+emerging+consideration+in+drug+discovery+and+design&doi=10.1021%2Fjm800562d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Target Flexibility: An Emerging Consideration in Drug Discovery and Design</span></div><div class="casAuthors">Cozzini, Pietro; Kellogg, Glen E.; Spyrakis, Francesca; Abraham, Donald J.; Costantino, Gabriele; Emerson, Andrew; Fanelli, Francesca; Gohlke, Holger; Kuhn, Leslie A.; Morris, Garrett M.; Orozco, Modesto; Pertinhez, Thelma A.; Rizzi, Menico; Sotriffer, Christoph A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6237-6255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5h-wKT3wh6bVg90H21EOLACvtfcHk0lgyvSDubF_fsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFaqtLvK&md5=8b9a64cc09fe04fb43cbbc45463abf1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm800562d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800562d%26sid%3Dliteratum%253Aachs%26aulast%3DCozzini%26aufirst%3DP.%26aulast%3DKellogg%26aufirst%3DG.%2BE.%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DAbraham%26aufirst%3DD.%2BJ.%26aulast%3DCostantino%26aufirst%3DG.%26aulast%3DEmerson%26aufirst%3DA.%26aulast%3DFanelli%26aufirst%3DF.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DKuhn%26aufirst%3DL.%2BA.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DOrozco%26aufirst%3DM.%26aulast%3DPertinhez%26aufirst%3DT.%2BA.%26aulast%3DRizzi%26aufirst%3DM.%26aulast%3DSotriffer%26aufirst%3DC.%26atitle%3DTarget%2520Flexibility%253A%2520An%2520emerging%2520consideration%2520in%2520drug%2520discovery%2520and%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6237%26epage%3D6255%26doi%3D10.1021%2Fjm800562d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Flexible ligand docking to multiple receptor conformations: a practical alternative</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">178</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1016/j.sbi.2008.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2008.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18302984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=178-184&author=M.+Totrovauthor=R.+Abagyan&title=Flexible+ligand+docking+to+multiple+receptor+conformations%3A+a+practical+alternative&doi=10.1016%2Fj.sbi.2008.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible ligand docking to multiple receptor conformations: a practical alternative</span></div><div class="casAuthors">Totrov, Maxim; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-184</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  State of the art docking algorithms predict an incorrect binding pose for about 50-70% of all ligands when only a single fixed receptor conformation is considered.  In many more cases, lack of receptor flexibility results in meaningless ligand binding scores, even when the correct pose is obtained.  Incorporating conformational rearrangements of the receptor binding pocket into predictions of both ligand binding pose and binding score is crucial for improving structure-based drug design and virtual ligand screening methodologies.  However, direct modeling of protein binding site flexibility remains challenging because of the large conformational space that must be sampled, and difficulties remain in constructing a suitably accurate energy function.  Here the authors show that using multiple fixed receptor conformations, either exptl. detd. by crystallog. or NMR, or computationally generated, is a practical shortcut that may improve docking calcns.  In several cases, such an approach has led to exptl. validated predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppaNNJ60eASbVg90H21EOLACvtfcHk0ljV2iUehS-zMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D&md5=61b273c1621cfd64832bdda28898b439</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2008.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2008.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DFlexible%2520ligand%2520docking%2520to%2520multiple%2520receptor%2520conformations%253A%2520a%2520practical%2520alternative%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D18%26spage%3D178%26epage%3D184%26doi%3D10.1016%2Fj.sbi.2008.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Four-dimensional docking: a fast and accurate account of discrete receptor flexibility in ligand docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1021/jm8009958</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8009958" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=397-406&author=G.+Bottegoniauthor=I.+Kufarevaauthor=M.+Totrovauthor=R.+Abagyan&title=Four-dimensional+docking%3A+a+fast+and+accurate+account+of+discrete+receptor+flexibility+in+ligand+docking&doi=10.1021%2Fjm8009958"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm8009958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8009958%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DFour-dimensional%2520docking%253A%2520a%2520fast%2520and%2520accurate%2520account%2520of%2520discrete%2520receptor%2520flexibility%2520in%2520ligand%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D397%26epage%3D406%26doi%3D10.1021%2Fjm8009958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span> </span><span class="NLM_article-title">Protein-ligand docking</span> <span class="citation_source-journal">Front. Biosci., Landmark Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">2289</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.2741/3854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2741%2F3854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21622177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVagsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=2289-306&author=G.+Bottegoni&title=Protein-ligand+docking&doi=10.2741%2F3854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-ligand docking</span></div><div class="casAuthors">Bottegoni, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2289-2306</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Ligand-docking is an established computational technique universally applied in structure-based drug design.  Since the first attempts carried out in the early '80s to predict the three-dimensional conformation of a protein-ligand bound complex, this methodol. has evolved constantly and it is presently implemented in many different ways.  The present study aims at explaining the std. protein-ligand docking protocol, together with its main advantages and drawbacks.  Milestone reports and future directions are reported and discussed as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvKVbh630jHrVg90H21EOLACvtfcHk0ljV2iUehS-zMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVagsrk%253D&md5=efd9553ea8dc75c60b348e75387e77cf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2741%2F3854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3854%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DProtein-ligand%2520docking%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2011%26volume%3D16%26spage%3D2289%26epage%3D306%26doi%3D10.2741%2F3854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. T.</span><span> </span><span class="NLM_article-title">Understanding protein-ligand interactions: the price of protein flexibility</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">335</span><span class="NLM_x">, </span> <span class="NLM_fpage">1325</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1016/j.jmb.2003.11.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmb.2003.11.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2004&pages=1325-41&author=D.+Rauhauthor=G.+Klebeauthor=M.+T.+Stubbs&title=Understanding+protein-ligand+interactions%3A+the+price+of+protein+flexibility&doi=10.1016%2Fj.jmb.2003.11.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.11.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.11.041%26sid%3Dliteratum%253Aachs%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26atitle%3DUnderstanding%2520protein-ligand%2520interactions%253A%2520the%2520price%2520of%2520protein%2520flexibility%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D335%26spage%3D1325%26epage%3D41%26doi%3D10.1016%2Fj.jmb.2003.11.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Damm, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Exploring experimental sources of multiple protein conformations in structure-based drug design</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">8225</span><span class="NLM_x">–</span> <span class="NLM_lpage">8235</span><span class="refDoi"> DOI: 10.1021/ja0709728</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0709728" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=8225-8235&author=K.+L.+Dammauthor=H.+A.+Carlson&title=Exploring+experimental+sources+of+multiple+protein+conformations+in+structure-based+drug+design&doi=10.1021%2Fja0709728"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja0709728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0709728%26sid%3Dliteratum%253Aachs%26aulast%3DDamm%26aufirst%3DK.%2BL.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DExploring%2520experimental%2520sources%2520of%2520multiple%2520protein%2520conformations%2520in%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D8225%26epage%3D8235%26doi%3D10.1021%2Fja0709728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Alonso, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bliznyuk, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gready, J. E.</span><span> </span><span class="NLM_article-title">Combining docking and molecular dynamic simulations in drug design</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">568</span><span class="refDoi"> DOI: 10.1002/med.20067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fmed.20067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16758486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsFGru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=531-568&author=H.+Alonsoauthor=A.+A.+Bliznyukauthor=J.+E.+Gready&title=Combining+docking+and+molecular+dynamic+simulations+in+drug+design&doi=10.1002%2Fmed.20067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Combining docking and molecular dynamic simulations in drug design</span></div><div class="casAuthors">Alonso, Hernan; Bliznyuk, Andrey A.; Gready, Jill E.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">531-568</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  A rational approach is needed to maximize the chances of finding new drugs, and to exploit the opportunities of potential new drug targets emerging from genomic and proteomic initiatives, and from the large libraries of small compds. now readily available through combinatorial chem.  Despite a shaky early history, computer-aided drug design techniques can now be effective in reducing costs and speeding up drug discovery.  This happy outcome results from development of more accurate and reliable algorithms, use of more thoughtfully planned strategies to apply them, and greatly increased computer power to allow studies with the necessary reliability to be performed.  Our review focuses on applications and protocols, with the main emphasis on crit. anal. of recent studies where docking calcns. and mol. dynamics (MD) simulations were combined to dock small mols. into protein receptors.  We highlight successes to demonstrate what is possible now, but also point out drawbacks and future directions.  The review is structured to lead the reader from the simpler to more compute-intensive methods.  Thus, while inexpensive and fast docking algorithms can be used to scan large compd. libraries and reduce their size, more accurate but expensive MD simulations can be applied when a few selected ligand candidates remain.  MD simulations can be used: during the prepn. of the protein receptor before docking, to optimize its structure and account for protein flexibility; for the refinement of docked complexes, to include solvent effects and account for induced fit; to calc. binding free energies, to provide an accurate ranking of the potential ligands; and in the latest developments, during the docking process itself to find the binding site and correctly dock the ligand a priori.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfwZFHimixV7Vg90H21EOLACvtfcHk0lgacQqAVdvpeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsFGru7o%253D&md5=f20210fea1ee020f95228846307e8a92</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fmed.20067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20067%26sid%3Dliteratum%253Aachs%26aulast%3DAlonso%26aufirst%3DH.%26aulast%3DBliznyuk%26aufirst%3DA.%2BA.%26aulast%3DGready%26aufirst%3DJ.%2BE.%26atitle%3DCombining%2520docking%2520and%2520molecular%2520dynamic%2520simulations%2520in%2520drug%2520design%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D531%26epage%3D568%26doi%3D10.1002%2Fmed.20067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Morra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genoni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neves, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span> </span><span class="NLM_article-title">Molecular recognition and drug-lead identification: what can molecular simulations tell us?</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.2174/092986710789957797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F092986710789957797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19941480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=25-41&author=G.+Morraauthor=A.+Genoniauthor=M.+Nevesauthor=K.+Merzauthor=G.+Colombo&title=Molecular+recognition+and+drug-lead+identification%3A+what+can+molecular+simulations+tell+us%3F&doi=10.2174%2F092986710789957797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition and drug-lead identification: what can molecular simulations tell us?</span></div><div class="casAuthors">Morra, G.; Genoni, A.; Neves, M. A. C.; Merz, K. M., Jr.; Colombo, G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-41</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Mol. recognition and ligand binding involving proteins underlie the most important life processes within the cell, such as substrate transport, catalysis, signal transmission, receptor trafficking, gene regulation, switching on and off of biochem. pathways.  Despite recent successes in predicting the structures of many protein-substrate complexes, the dynamic aspects of binding have been largely neglected by computational/theor. investigations.  Recently, several groups have started tackling these problems with the use of exptl. and simulation methods and developed models describing the variation of protein dynamics upon complex formation, shedding light on how substrate or inhibitor binding can alter protein flexibility and function.  The study of ligand-induced dynamic variations has also been exploited to review the concept of allosteric changes, in the absence of major conformational changes.  In this context, the study of the influence of protein motions on signal transduction and on catalytic activities has been used to develop pharmacophore models based on ensembles of protein conformations.  These models, taking flexibility explicitly into account, are able to distinguish active inhibitors vs. nonactive drug-like compds., to define new mol. motifs and to preferentially identify specific ligands for a certain protein target.  The application of these methods holds great promise in advancing structure-based drug discovery and medicinal chem. in general, opening up the possibility to explore broader chem. spaces than is normally done in an efficient way.  In this review, examples illustrating the extent to which simulations can be used to understand these phenomena will be presented along with examples of methodol. developments to increase phys. understanding of the processes and improve the possibility to rationally design new mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNRcMp8Hm0HLVg90H21EOLACvtfcHk0ljoOYJ3fOIE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSmt70%253D&md5=c695d9184a4bb4655dee00f6a2643f4d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.2174%2F092986710789957797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986710789957797%26sid%3Dliteratum%253Aachs%26aulast%3DMorra%26aufirst%3DG.%26aulast%3DGenoni%26aufirst%3DA.%26aulast%3DNeves%26aufirst%3DM.%26aulast%3DMerz%26aufirst%3DK.%26aulast%3DColombo%26aufirst%3DG.%26atitle%3DMolecular%2520recognition%2520and%2520drug-lead%2520identification%253A%2520what%2520can%2520molecular%2520simulations%2520tell%2520us%253F%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2010%26volume%3D17%26spage%3D25%26epage%3D41%26doi%3D10.2174%2F092986710789957797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Pang, Y.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozikowski, A.</span><span> </span><span class="NLM_article-title">Prediction of the binding sites of huperzine A in acetylcholinesterase by docking studies</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span><span class="refDoi"> DOI: 10.1007/BF00124014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2FBF00124014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=669-681&author=Y.-P.+Pangauthor=A.+Kozikowski&title=Prediction+of+the+binding+sites+of+huperzine+A+in+acetylcholinesterase+by+docking+studies&doi=10.1007%2FBF00124014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2FBF00124014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124014%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DY.-P.%26aulast%3DKozikowski%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520the%2520binding%2520sites%2520of%2520huperzine%2520A%2520in%2520acetylcholinesterase%2520by%2520docking%2520studies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D669%26epage%3D681%26doi%3D10.1007%2FBF00124014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Ivetac, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Molecular recognition in the case of flexible targets</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1663</span><span class="NLM_x">–</span> <span class="NLM_lpage">1671</span><span class="refDoi"> DOI: 10.2174/138161211796355056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F138161211796355056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21619526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVyjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=1663-1671&author=A.+Ivetacauthor=J.+A.+McCammon&title=Molecular+recognition+in+the+case+of+flexible+targets&doi=10.2174%2F138161211796355056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition in the case of flexible targets</span></div><div class="casAuthors">Ivetac, Anthony; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1663-1671</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  A protein's flexibility is well recognized to underlie its capacity to engage in crit. functions, such as signal transduction, biomol. transport and biochem. reactivity.  Mol. recognition is also tightly linked to the dynamics of the binding partners, yet protein flexibility has largely been ignored by the growing field of structure-based drug design (SBDD).  In combination with exptl. detd. structures, a no. of computational methods have been proposed to model protein movements, which may be important for small mol. binding.  Such techniques have the ability to expose new binding site conformations, which may in turn recognize and lead to the discovery of more potent and selective drugs through mol. docking.  In this article, we discuss various methods and focus on the Relaxed Complex Scheme (RCS), which uses Mol. Dynamics (MD) simulations to model full protein flexibility and enhance virtual screening programs.  We review practical applications of the RCS and use a recent study of the HIV-1 reverse transcriptase to illustrate the various phases of the scheme.  We also discuss some encouraging developments, aimed at addressing current weaknesses of the RCS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFY1LH59mtWLVg90H21EOLACvtfcHk0ljoOYJ3fOIE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVyjt7k%253D&md5=f8d1e0b1963435d0c40dfa96c8dcf711</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.2174%2F138161211796355056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161211796355056%26sid%3Dliteratum%253Aachs%26aulast%3DIvetac%26aufirst%3DA.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DMolecular%2520recognition%2520in%2520the%2520case%2520of%2520flexible%2520targets%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2011%26volume%3D17%26spage%3D1663%26epage%3D1671%26doi%3D10.2174%2F138161211796355056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Lin, J.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schames, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The relaxed complex method: accommodating receptor flexibility for drug design with an improved scoring scheme</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">62</span><span class="refDoi"> DOI: 10.1002/bip.10218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fbip.10218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12579579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlyrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=47-62&author=J.-H.+Linauthor=A.+L.+Perrymanauthor=J.+R.+Schamesauthor=J.+A.+McCammon&title=The+relaxed+complex+method%3A+accommodating+receptor+flexibility+for+drug+design+with+an+improved+scoring+scheme&doi=10.1002%2Fbip.10218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The relaxed complex method: accommodating receptor flexibility for drug design with an improved scoring scheme</span></div><div class="casAuthors">Lin, Jung-Hsin; Perryman, Alexander L.; Schames, Julie R.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-62</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">An extension of the new computational methodol. for drug design, the "relaxed complex" method, which accommodates receptor flexibility, is described.  This relaxed complex method recognizes that ligand may bind to conformations that occur only rarely in the dynamics of the receptor.  We have shown that the ligand-enzyme binding modes are very sensitive to the enzyme conformations, and our approach is capable of finding the best ligand enzyme complexes.  Rapid docking serves as an efficient initial filtering method to screen a myriad of docking modes to a limited set, and it is then followed by more accurate scoring with the MM/PBSA (Mol. Mechanics/Poisson Boltzmann Surface Area) approach to find the best ligand-receptor complexes.  The MM/PBSA scorings consistently indicate that the calcd. binding modes that are most similar to those obsd. in the x-ray crystallog. complexes are the ones with the lowest free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3skpENhME5rVg90H21EOLACvtfcHk0lhARqaMQ9hVYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlyrtQ%253D%253D&md5=f81abcd665d2bd0b6dab9962adff68c2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fbip.10218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.10218%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.-H.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DSchames%26aufirst%3DJ.%2BR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520relaxed%2520complex%2520method%253A%2520accommodating%2520receptor%2520flexibility%2520for%2520drug%2520design%2520with%2520an%2520improved%2520scoring%2520scheme%26jtitle%3DBiopolymers%26date%3D2003%26volume%3D68%26spage%3D47%26epage%3D62%26doi%3D10.1002%2Fbip.10218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Tian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, T.</span><span> </span><span class="NLM_article-title">Assessing an ensemble docking-based virtual screening strategy for kinase targets by considering protein flexibility</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2664</span><span class="NLM_x">–</span> <span class="NLM_lpage">2679</span><span class="refDoi"> DOI: 10.1021/ci500414b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500414b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2664-2679&author=S.+Tianauthor=H.+Sunauthor=P.+Panauthor=D.+Liauthor=X.+Zhenauthor=Y.+Liauthor=T.+Hou&title=Assessing+an+ensemble+docking-based+virtual+screening+strategy+for+kinase+targets+by+considering+protein+flexibility&doi=10.1021%2Fci500414b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing an Ensemble Docking-Based Virtual Screening Strategy for Kinase Targets by Considering Protein Flexibility</span></div><div class="casAuthors">Tian, Sheng; Sun, Huiyong; Pan, Peichen; Li, Dan; Zhen, Xuechu; Li, Youyong; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2664-2679</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, to accommodate receptor flexibility, based on multiple receptor conformations, a novel ensemble docking protocol was developed by using the naive Bayesian classification technique, and it was evaluated in terms of the prediction accuracy of docking-based virtual screening (VS) of three important targets in the kinase family: ALK, CDK2, and VEGFR2.  First, for each target, the representative crystal structures were selected by structural clustering, and the capability of mol. docking based on each representative structure to discriminate inhibitors from non-inhibitors was examd.  Then, for each target, 50 ns mol. dynamics (MD) simulations were carried out to generate an ensemble of the conformations, and multiple representative structures/snapshots were extd. from each MD trajectory by structural clustering.  On av., the representative crystal structures outperform the representative structures extd. from MD simulations in terms of the capabilities to sep. inhibitors from non-inhibitors.  Finally, by using the naive Bayesian classification technique, an integrated VS strategy was developed to combine the prediction results of mol. docking based on different representative conformations chosen from crystal structures and MD trajectories.  It was encouraging to observe that the integrated VS strategy yields better performance than the docking-based VS based on any single rigid conformation.  This novel protocol may provide an improvement over existing strategies to search for more diverse and promising active compds. for a target of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMFhXu4lCyd7Vg90H21EOLACvtfcHk0lhARqaMQ9hVYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbnJ&md5=d674a8d7bd48d44c71ecb3935c7a6ef7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci500414b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500414b%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZhen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DAssessing%2520an%2520ensemble%2520docking-based%2520virtual%2520screening%2520strategy%2520for%2520kinase%2520targets%2520by%2520considering%2520protein%2520flexibility%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2664%26epage%3D2679%26doi%3D10.1021%2Fci500414b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Tarcsay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paragi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vass, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jojart, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bogar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">The impact of molecular dynamics sampling on the performance of virtual screening against GPCRs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2990</span><span class="NLM_x">–</span> <span class="NLM_lpage">2999</span><span class="refDoi"> DOI: 10.1021/ci400087b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400087b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SgsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=2990-2999&author=A.+Tarcsayauthor=G.+Paragiauthor=M.+Vassauthor=B.+Jojartauthor=F.+Bogarauthor=G.+M.+Keseru&title=The+impact+of+molecular+dynamics+sampling+on+the+performance+of+virtual+screening+against+GPCRs&doi=10.1021%2Fci400087b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The Impact of Molecular Dynamics Sampling on the Performance of Virtual Screening against GPCRs</span></div><div class="casAuthors">Tarcsay, Akos; Paragi, Gabor; Vass, Marton; Jojart, Balazs; Bogar, Ferenc; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2990-2999</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of ligand-protein complexes requires simultaneous adaptation of the binding partners.  In structure based virtual screening, high throughput docking approaches typically consider the ligand flexibility, but the conformational freedom of the protein is usually taken into account in a limited way.  The goal of this study is to elaborate a methodol. for incorporating protein flexibility to improve the virtual screening enrichments on GPCRs.  Explicit-solvated mol. dynamics simulations (MD) were carried out in lipid bilayers to generate an ensemble of protein conformations for the X-ray structures and homol. models of both aminergic and peptidergic GPCRs including the chemokine CXCR4, dopamine D3, histamine H4, and serotonin 5HT6holo receptor complexes.  The quality of the receptor models was assessed by enrichment studies to compare X-ray structures, homol. models, and snapshots from the MD trajectory.  According to our results, selected frames from the MD trajectory can outperform X-ray structures and homol. models in terms of enrichment factor and AUC values.  Significant changes were obsd. considering EF1% values: comparing the original CXCR4, D3, and H4 targets and the addnl. 5HT6 initial models to that of the best MD frame resulted in 0 to 6.7, 0.32 to 3.5 (10×), 13.3 to 26.7 (2×), and 0 to 14.1 improvements, resp.  It is worth noting that rank-av. based ensemble evaluation calcd. for different ensemble sizes could not improve the results further.  We propose here that MD simulation can capture protein conformations representing the key interacting points of the receptor but less biased toward one specific chemotype.  These conformations are useful for the identification of a consensus binding site with improved performance in virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWkcgOyiLlm7Vg90H21EOLACvtfcHk0lggPhH2P_EjBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SgsLjO&md5=30e28d71e92c45ed5883a3d984645be7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci400087b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400087b%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DParagi%26aufirst%3DG.%26aulast%3DVass%26aufirst%3DM.%26aulast%3DJojart%26aufirst%3DB.%26aulast%3DBogar%26aufirst%3DF.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DThe%2520impact%2520of%2520molecular%2520dynamics%2520sampling%2520on%2520the%2520performance%2520of%2520virtual%2520screening%2520against%2520GPCRs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D2990%26epage%3D2999%26doi%3D10.1021%2Fci400087b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindorff-Larsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Long-timescale molecular dynamics simulations of protein structure and function</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">127</span><span class="refDoi"> DOI: 10.1016/j.sbi.2009.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2009.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19361980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvVSqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=120-127&author=J.+L.+Klepeisauthor=K.+Lindorff-Larsenauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Long-timescale+molecular+dynamics+simulations+of+protein+structure+and+function&doi=10.1016%2Fj.sbi.2009.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Long-timescale molecular dynamics simulations of protein structure and function</span></div><div class="casAuthors">Klepeis, John L.; Lindorff-Larsen, Kresten; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics simulations allow for at.-level characterization of biomol. processes such as the conformational transitions assocd. with protein function.  The computational demands of such simulations, however, have historically prevented them from reaching the microsecond and greater timescales on which these events often occur.  Recent advances in algorithms, software, and computer hardware have made microsecond-timescale simulations with tens of thousands of atoms practical, with millisecond-timescale simulations on the horizon.  This review outlines these advances in high-performance mol. dynamics simulation and discusses recent applications to studies of protein dynamics and function as well as exptl. validation of the underlying computational models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtFlQ4IqPPIbVg90H21EOLACvtfcHk0lggPhH2P_EjBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvVSqt70%253D&md5=7f40af4e683cc9aa24c49ef8eb7ae61f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2009.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2009.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DLindorff-Larsen%26aufirst%3DK.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DLong-timescale%2520molecular%2520dynamics%2520simulations%2520of%2520protein%2520structure%2520and%2520function%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.sbi.2009.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Buch, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giorgino, T.</span><span> </span><span class="NLM_article-title">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x"> (</span><span class="NLM_issue">25</span><span class="NLM_x">) </span> <span class="NLM_fpage">10184</span><span class="NLM_x">–</span> <span class="NLM_lpage">10189</span><span class="refDoi"> DOI: 10.1073/pnas.1103547108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1103547108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21646537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=10184-10189&issue=25&author=I.+Buchauthor=T.+Giorgino&title=Complete+reconstruction+of+an+enzyme-inhibitor+binding+process+by+molecular+dynamics+simulations&doi=10.1073%2Fpnas.1103547108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span></div><div class="casAuthors">Buch, Ignasi; Giorgino, Toni; De Fabritiis, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10184-10189, S10184/1-S10184/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The understanding of protein-ligand binding is of crit. importance for biomedical research, yet the process itself has been very difficult to study because of its intrinsically dynamic character.  Here, we have been able to quant. reconstruct the complete binding process of the enzyme-inhibitor complex trypsin-benzamidine by performing 495 mol. dynamics simulations of free ligand binding of 100 ns each, 187 of which produced binding events with an rmsd less than 2 Å compared to the crystal structure.  The binding paths obtained are able to capture the kinetic pathway of the inhibitor diffusing from solvent (SO) to the bound (S4) state passing through two metastable intermediate states S2 and S3.  Rather than directly entering the binding pocket the inhibitor appears to roll on the surface of the protein in its transition between S3 and the final binding pocket, whereas the transition between S2 and the bound pose requires rediffusion to S3.  An estn. of the std. free energy of binding gives ΔG° = -5.2 0.4 kcal/mol (cf. the exptl. value -6.2 kcal/mol), and a two-states kinetic model kon = (1.5 ± 0.2) × 108 M-1 s-1 and koff = (9.5 ± 3.3) × 104 s-1 for unbound to bound transitions.  The ability to reconstruct by simple diffusion the binding pathway of an enzyme-inhibitor binding process demonstrates the predictive power of unconventional high-throughput mol. simulations.  Moreover, the methodol. is directly applicable to other mol. systems and thus of general interest in biomedical and pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEgdoM19tFbbVg90H21EOLACvtfcHk0lggPhH2P_EjBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D&md5=40fdd0ec62da71c0f11d57731ee518a1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1103547108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1103547108%26sid%3Dliteratum%253Aachs%26aulast%3DBuch%26aufirst%3DI.%26aulast%3DGiorgino%26aufirst%3DT.%26atitle%3DComplete%2520reconstruction%2520of%2520an%2520enzyme-inhibitor%2520binding%2520process%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D25%26spage%3D10184%26epage%3D10189%26doi%3D10.1073%2Fpnas.1103547108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">How does a drug molecule find its target binding site?</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">9181</span><span class="NLM_x">–</span> <span class="NLM_lpage">9183</span><span class="refDoi"> DOI: 10.1021/ja202726y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja202726y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=9181-9183&author=Y.+Shanauthor=E.+T.+Kimauthor=M.+P.+Eastwoodauthor=R.+O.+Drorauthor=M.+A.+Seeligerauthor=D.+E.+Shaw&title=How+does+a+drug+molecule+find+its+target+binding+site%3F&doi=10.1021%2Fja202726y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">How Does a Drug Molecule Find Its Target Binding Site?</span></div><div class="casAuthors">Shan, Yibing; Kim, Eric T.; Eastwood, Michael P.; Dror, Ron O.; Seeliger, Markus A.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9181-9183</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although the thermodn. principles that control the binding of drug mols. to their protein targets are well understood, detailed exptl. characterization of the process by which such binding occurs has proven challenging.  We conducted relatively long, unguided mol. dynamics simulations in which a ligand (the cancer drug dasatinib or the kinase inhibitor PP1) was initially placed at a random location within a box that also contained a protein (Src kinase) to which that ligand was known to bind.  In several of these simulations, the ligand correctly identified its target binding site, forming a complex virtually identical to the crystallog. detd. bound structure.  The simulated trajectories provide a continuous, at.-level view of the entire binding process, revealing persistent and noteworthy intermediate conformations and shedding light on the role of water mols.  The technique we employed, which does not assume any prior knowledge of the binding site's location, may prove particularly useful in the development of allosteric inhibitors that target previously undiscovered binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfc8mwES5csLVg90H21EOLACvtfcHk0ljPdsPbEjAVBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWgtb8%253D&md5=11847733dfeb27ed5ec1a565327b258c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja202726y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja202726y%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DEastwood%26aufirst%3DM.%2BP.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DHow%2520does%2520a%2520drug%2520molecule%2520find%2520its%2520target%2520binding%2520site%253F%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D9181%26epage%3D9183%26doi%3D10.1021%2Fja202726y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Decherchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berteotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">The ligand binding mechanism to purine nucleoside phosphorylase elucidated via molecular dynamics and machine learning</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">6155</span><span class="refDoi"> DOI: 10.1038/ncomms7155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fncomms7155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25625196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlaitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6155&author=S.+Decherchiauthor=A.+Berteottiauthor=G.+Bottegoniauthor=W.+Rocchiaauthor=A.+Cavalli&title=The+ligand+binding+mechanism+to+purine+nucleoside+phosphorylase+elucidated+via+molecular+dynamics+and+machine+learning&doi=10.1038%2Fncomms7155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The ligand binding mechanism to purine nucleoside phosphorylase elucidated via molecular dynamics and machine learning</span></div><div class="casAuthors">Decherchi, Sergio; Berteotti, Anna; Bottegoni, Giovanni; Rocchia, Walter; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6155</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The study of biomol. interactions between a drug and its biol. target is of paramount importance for the design of novel bioactive compds.  In this paper, we report on the use of mol. dynamics (MD) simulations and machine learning to study the binding mechanism of a transition state analog (DADMe-immucillin-H) to the purine nucleoside phosphorylase (PNP) enzyme.  Microsecond-long MD simulations allow us to observe several binding events, following different dynamical routes and reaching diverse binding configurations.  These simulations are used to est. kinetic and thermodn. quantities, such as kon and binding free energy, obtaining a good agreement with available exptl. data.  In addn., we advance a hypothesis for the slow-onset inhibition mechanism of DADMe-immucillin-H against PNP.  Combining extensive MD simulations with machine learning algorithms could therefore be a fruitful approach for capturing key aspects of drug-target recognition and binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYXSG2uDGjJbVg90H21EOLACvtfcHk0ljPdsPbEjAVBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlaitbo%253D&md5=205b8ecc6a37f3caf9953c5393c2826f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncomms7155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7155%26sid%3Dliteratum%253Aachs%26aulast%3DDecherchi%26aufirst%3DS.%26aulast%3DBerteotti%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DThe%2520ligand%2520binding%2520mechanism%2520to%2520purine%2520nucleoside%2520phosphorylase%2520elucidated%2520via%2520molecular%2520dynamics%2520and%2520machine%2520learning%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6155%26doi%3D10.1038%2Fncomms7155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Exploring atomic resolution physiology on a femtosecond to millisecond timescale using molecular dynamics simulations</span> <span class="citation_source-journal">J. Gen. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">555</span><span class="NLM_x">–</span> <span class="NLM_lpage">562</span><span class="refDoi"> DOI: 10.1085/jgp.200910373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1085%2Fjgp.200910373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20513757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFensr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2010&pages=555-562&author=R.+O.+Drorauthor=M.+O.+Jensenauthor=D.+W.+Borhaniauthor=D.+E.+Shaw&title=Exploring+atomic+resolution+physiology+on+a+femtosecond+to+millisecond+timescale+using+molecular+dynamics+simulations&doi=10.1085%2Fjgp.200910373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring atomic resolution physiology on a femtosecond to millisecond timescale using molecular dynamics simulations</span></div><div class="casAuthors">Dror, Ron O.; Jensen, Morten Oe.; Borhani, David W.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Physiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">555-562</span>CODEN:
                <span class="NLM_cas:coden">JGPLAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-1295</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">A review.  This study deals with the mechanistic insights that physiologists might garner by complementing expts. with atomistic mol. dynamics (MD) simulations.  It discusses the strengths and limitations of MD as a tool for physiologists.  MD simulations are important among physiologists because these allow access to a spatiotemporal domain that is difficult to probe exptl.  Simulations can be particularly valuable for membrane proteins, for which exptl. characterization of structural dynamics tends to be challenging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4vfNL81MfN7Vg90H21EOLACvtfcHk0ljPdsPbEjAVBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFensr8%253D&md5=75e55def06d3831daab98511452be286</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1085%2Fjgp.200910373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1085%252Fjgp.200910373%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DJensen%26aufirst%3DM.%2BO.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DExploring%2520atomic%2520resolution%2520physiology%2520on%2520a%2520femtosecond%2520to%2520millisecond%2520timescale%2520using%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Gen.%2520Physiol.%26date%3D2010%26volume%3D135%26spage%3D555%26epage%3D562%26doi%3D10.1085%2Fjgp.200910373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Lane, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">To milliseconds and beyond: challenges in the simulation of protein folding</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.sbi.2012.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2012.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23237705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=58-65&author=T.+J.+Laneauthor=D.+Shuklaauthor=K.+A.+Beauchampauthor=V.+S.+Pande&title=To+milliseconds+and+beyond%3A+challenges+in+the+simulation+of+protein+folding&doi=10.1016%2Fj.sbi.2012.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">To milliseconds and beyond: challenges in the simulation of protein folding</span></div><div class="casAuthors">Lane, Thomas J.; Shukla, Diwakar; Beauchamp, Kyle A.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-65</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Quant. accurate all-atom mol. dynamics (MD) simulations of protein folding have long been considered a holy grail of computational biol.  Due to the large system sizes and long timescales involved, such a pursuit was for many years computationally intractable.  Further, sufficiently accurate forcefields needed to be developed in order to realistically model folding.  This decade, however, saw the first reports of folding simulations describing kinetics on the order of milliseconds, placing many proteins firmly within reach of these methods.  Progress in sampling and forcefield accuracy, however, presents a new challenge: how to turn huge MD datasets into scientific understanding.  Here, we review recent progress in MD simulation techniques and show how the vast datasets generated by such techniques present new challenges for anal.  We critically discuss the state of the art, including reaction coordinate and Markov state model (MSM) methods, and provide a perspective for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphYpDSTwX8JrVg90H21EOLACvtfcHk0lj8YJIZhcKHBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLjF&md5=76f625b813b7c85cfbcabac6659155cd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2012.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2012.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DLane%26aufirst%3DT.%2BJ.%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DTo%2520milliseconds%2520and%2520beyond%253A%2520challenges%2520in%2520the%2520simulation%2520of%2520protein%2520folding%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2013%26volume%3D23%26spage%3D58%26epage%3D65%26doi%3D10.1016%2Fj.sbi.2012.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Lu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span> </span><span class="NLM_article-title">Drug-target residence time: critical information for lead optimization</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.06.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.cbpa.2010.06.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20663707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=467-474&author=H.+Luauthor=P.+J.+Tonge&title=Drug-target+residence+time%3A+critical+information+for+lead+optimization&doi=10.1016%2Fj.cbpa.2010.06.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time: Critical information for lead optimization</span></div><div class="casAuthors">Lu, Hao; Tonge, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Failure due to poor in vivo efficacy is a primary contributor to attrition during the development of new chemotherapeutics.  Lead optimization programs that in their quest for efficacy focus solely on improving the affinity of drug-target binding are flawed, since this approach ignores the fluctuations in drug concn. that occur in vivo.  Instead the lifetime of the drug-target complex must also be considered, since drugs only act when they are bound to their targets.  Consequently, to improve the correlation between the in vitro and in vivo activity of drugs, measurements of drug-target residence time must be incorporated into the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk2kN_txuMLVg90H21EOLACvtfcHk0lj8YJIZhcKHBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D&md5=3189f56032106db858cfb4269d3fc11f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.06.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.06.176%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DDrug-target%2520residence%2520time%253A%2520critical%2520information%2520for%2520lead%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D467%26epage%3D474%26doi%3D10.1016%2Fj.cbpa.2010.06.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Abrams, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussi, G.</span><span> </span><span class="NLM_article-title">Enhanced sampling in molecular dynamics using metadynamics, replica-exchange, and temperature-acceleration</span> <span class="citation_source-journal">Entropy</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.3390/e16010163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.3390%2Fe16010163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptV2htbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=163-199&author=C.+Abramsauthor=G.+Bussi&title=Enhanced+sampling+in+molecular+dynamics+using+metadynamics%2C+replica-exchange%2C+and+temperature-acceleration&doi=10.3390%2Fe16010163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced sampling in molecular dynamics using metadynamics, replica-exchange, and temperature-acceleration</span></div><div class="casAuthors">Abrams, Cameron; Bussi, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Entropy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">163-199, 37 pp.</span>CODEN:
                <span class="NLM_cas:coden">ENTRFG</span>;
        ISSN:<span class="NLM_cas:issn">1099-4300</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">We review a selection of methods for performing enhanced sampling in mol. dynamics simulations.  We consider methods based on collective variable biasing and on tempering, and offer both historical and contemporary perspectives.  In collective-variable biasing, we first discuss methods stemming from thermodn. integration that use mean force biasing, including the adaptive biasing force algorithm and temp. acceleration.  We then turn to methods that use bias potentials, including umbrella sampling and metadynamics.  We next consider parallel tempering and replica-exchange methods.  We conclude with a brief presentation of some combination methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgwp9LGvawa7Vg90H21EOLACvtfcHk0lj8YJIZhcKHBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptV2htbg%253D&md5=7e27e14ffdacdd6de1e89f512b0f96ce</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.3390%2Fe16010163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fe16010163%26sid%3Dliteratum%253Aachs%26aulast%3DAbrams%26aufirst%3DC.%26aulast%3DBussi%26aufirst%3DG.%26atitle%3DEnhanced%2520sampling%2520in%2520molecular%2520dynamics%2520using%2520metadynamics%252C%2520replica-exchange%252C%2520and%2520temperature-acceleration%26jtitle%3DEntropy%26date%3D2014%26volume%3D16%26spage%3D163%26epage%3D199%26doi%3D10.3390%2Fe16010163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravimohan, C.</span><span> </span><span class="NLM_article-title">Monte-carlo simulation of differences in free-energies of hydration</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">3050</span><span class="NLM_x">–</span> <span class="NLM_lpage">3054</span><span class="refDoi"> DOI: 10.1063/1.449208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.449208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL2MXls1Oiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1985&pages=3050-3054&author=W.+L.+Jorgensenauthor=C.+Ravimohan&title=Monte-carlo+simulation+of+differences+in+free-energies+of+hydration&doi=10.1063%2F1.449208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Monte Carlo simulation of differences in free energies of hydration</span></div><div class="casAuthors">Jorgensen, William L.; Ravimohan, C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3050-4</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Perturbation theory was applied to calc. the relative free energies of hydration of methanol and ethane in dil. soln..  Only 2 or 3 Monte Carlo simulations using double-wide sampling are necessary to obtain results with high precision.  The small statistical uncertainty in the computed change in free energy of hydration and the agreement with exptl. thermodn. data are most encouraging for application of the procedure to a wide range of problems.  Structural effects accompanying the mutation of methanol to ethane in water are also discussed; H bonding to the solute is essentially eliminated by only a 25% redn. in the at. charges of MeOH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUM9poULbA5rVg90H21EOLACvtfcHk0lj8YJIZhcKHBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXls1Oiu7w%253D&md5=1d2760883bcd23620d10e98a2295d2e5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1063%2F1.449208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.449208%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DRavimohan%26aufirst%3DC.%26atitle%3DMonte-carlo%2520simulation%2520of%2520differences%2520in%2520free-energies%2520of%2520hydration%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1985%26volume%3D83%26spage%3D3050%26epage%3D3054%26doi%3D10.1063%2F1.449208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, L. L.</span><span> </span><span class="NLM_article-title">Perspective on free-energy perturbation calculations for chemical equilibria</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1021/ct800011m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct800011m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=869-876&author=W.+L.+Jorgensenauthor=L.+L.+Thomas&title=Perspective+on+free-energy+perturbation+calculations+for+chemical+equilibria&doi=10.1021%2Fct800011m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective on Free-Energy Perturbation Calculations for Chemical Equilibria</span></div><div class="casAuthors">Jorgensen, William L.; Thomas, Laura L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-876</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An overview has been provided on the history of free-energy perturbation (FEP) calcns. and their application to computation of relative free energies of solvation.  In view of the expanding role of FEP calcns. for pursuing a wide range of problems in chem. and biochem., including mol. design, it is important to establish reliable protocols for their execution.  For 30 interconversions of substituted benzenes, which represent prototypical cases for mol. design, alternative sampling procedures and nos. of FEP windows were considered.  Simple overlap sampling with 11 or 15 windows and double-wide sampling with 14 windows emerged as viable procedures for such routine FEP calcns.; the present protocols with these sampling methods yield statistical uncertainties of 0.2 kcal/mol for free-energy differences.  The slightly greater accuracy obtainable with 21 to 25 windows does not clearly justify the increased costs, while overlap sampling with 6 or 8 windows may be useful for preliminary exploration of large nos. of mol. changes.  Application of the power-series expansion of the Zwanzig equation was also tested.  Use of the first-order term is generally reliable, while inclusion of the slowly convergent, second-order fluctuation term causes deterioration in the results for strongly hydrogen-bonded solutes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqukDgvYtLny7Vg90H21EOLACvtfcHk0liZtNw-aeanCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D&md5=4f6cd47142dbbcd51e94aedd2d878538</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fct800011m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct800011m%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DThomas%26aufirst%3DL.%2BL.%26atitle%3DPerspective%2520on%2520free-energy%2520perturbation%2520calculations%2520for%2520chemical%2520equilibria%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D869%26epage%3D876%26doi%3D10.1021%2Fct800011m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Torrie, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valleau, J. P.</span><span> </span><span class="NLM_article-title">Nonphysical sampling distributions in monte carlo free-energy estimation: umbrella sampling</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90121-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0021-9991%2877%2990121-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=187-199&author=G.+M.+Torrieauthor=J.+P.+Valleau&title=Nonphysical+sampling+distributions+in+monte+carlo+free-energy+estimation%3A+umbrella+sampling&doi=10.1016%2F0021-9991%2877%2990121-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990121-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990121-8%26sid%3Dliteratum%253Aachs%26aulast%3DTorrie%26aufirst%3DG.%2BM.%26aulast%3DValleau%26aufirst%3DJ.%2BP.%26atitle%3DNonphysical%2520sampling%2520distributions%2520in%2520monte%2520carlo%2520free-energy%2520estimation%253A%2520umbrella%2520sampling%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D187%26epage%3D199%26doi%3D10.1016%2F0021-9991%2877%2990121-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Sugita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, Y.</span><span> </span><span class="NLM_article-title">Replica-exchange molecular dynamics method for protein folding</span> <span class="citation_source-journal">Chem. Phys. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">314</span><span class="NLM_x">, </span> <span class="NLM_fpage">141</span><span class="NLM_x">–</span> <span class="NLM_lpage">151</span><span class="refDoi"> DOI: 10.1016/S0009-2614(99)01123-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0009-2614%2899%2901123-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1MXotVKrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=1999&pages=141-151&author=Y.+Sugitaauthor=Y.+Okamoto&title=Replica-exchange+molecular+dynamics+method+for+protein+folding&doi=10.1016%2FS0009-2614%2899%2901123-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Replica-exchange molecular dynamics method for protein folding</span></div><div class="casAuthors">Sugita, Y.; Okamoto, Y.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">141-151</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We have developed a formulation for mol. dynamics algorithm for the replica-exchange method.  The effectiveness of the method for the protein-folding problem is tested with the penta-peptide Met-enkephalin.  The method can overcome the multiple-min. problem by exchanging non-interacting replicas of the system at several temps.  From only one simulation run, one can obtain probability distributions in canonical ensemble for a wide temp. range using multiple-histogram re-weighting techniques, which allows the calcn. of any thermodn. quantity as a function of temp. in that range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4vjM65fDLULVg90H21EOLACvtfcHk0liZtNw-aeanCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXotVKrsLc%253D&md5=0fec0ff81ca7806c1e1ac29e5f50ce19</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2901123-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252901123-9%26sid%3Dliteratum%253Aachs%26aulast%3DSugita%26aufirst%3DY.%26aulast%3DOkamoto%26aufirst%3DY.%26atitle%3DReplica-exchange%2520molecular%2520dynamics%2520method%2520for%2520protein%2520folding%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D314%26spage%3D141%26epage%3D151%26doi%3D10.1016%2FS0009-2614%2899%2901123-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Escaping free-energy minima</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">12562</span><span class="NLM_x">–</span> <span class="NLM_lpage">12566</span><span class="refDoi"> DOI: 10.1073/pnas.202427399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.202427399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12271136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvFGiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=12562-12566&author=A.+Laioauthor=M.+Parrinello&title=Escaping+free-energy+minima&doi=10.1073%2Fpnas.202427399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Escaping free-energy minima</span></div><div class="casAuthors">Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">12562-12566</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We introduce a powerful method for exploring the properties of the multidimensional free energy surfaces (FESs) of complex many-body systems by means of coarse-grained non-Markovian dynamics in the space defined by a few collective coordinates.  A characteristic feature of these dynamics is the presence of a history-dependent potential term that, in time, fills the min. in the FES, allowing the efficient exploration and accurate detn. of the FES as a function of the collective coordinates.  We demonstrate the usefulness of this approach in the case of the dissocn. of a NaCl mol. in water and in the study of the conformational changes of a dialanine in soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoTAY5TRy70LVg90H21EOLACvtfcHk0liZtNw-aeanCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvFGiurc%253D&md5=48d5bc7436f3ef9d78369671e70fa608</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202427399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202427399%26sid%3Dliteratum%253Aachs%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DEscaping%2520free-energy%2520minima%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D12562%26epage%3D12566%26doi%3D10.1073%2Fpnas.202427399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Isralewitz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">Steered molecular dynamics and mechanical functions of proteins</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span><span class="refDoi"> DOI: 10.1016/S0959-440X(00)00194-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0959-440X%2800%2900194-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11297932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsVyisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=224-230&author=B.+Isralewitzauthor=M.+Gaoauthor=K.+Schulten&title=Steered+molecular+dynamics+and+mechanical+functions+of+proteins&doi=10.1016%2FS0959-440X%2800%2900194-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Steered molecular dynamics and mechanical functions of proteins</span></div><div class="casAuthors">Isralewitz, Barry; Gao, Mu; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">224-230</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  At. force microscopy of single mols., steered mol. dynamics and the theory of stochastic processes have established a new field that investigates mech. functions of proteins, such as ligand-receptor binding/unbinding and elasticity of muscle proteins during stretching.  The combination of these methods yields information on the energy landscape that controls mech. function and on the force-bearing components of proteins, as well as on the underlying phys. mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMPwb3lv1-ibVg90H21EOLACvtfcHk0lj3ZxUlDhyhIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsVyisro%253D&md5=09b7a191f8ffa19d316ec0f4421c3731</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2800%2900194-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252800%252900194-9%26sid%3Dliteratum%253Aachs%26aulast%3DIsralewitz%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DSteered%2520molecular%2520dynamics%2520and%2520mechanical%2520functions%2520of%2520proteins%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D224%26epage%3D230%26doi%3D10.1016%2FS0959-440X%2800%2900194-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Grubmuller, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heymann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavan, P.</span><span> </span><span class="NLM_article-title">Ligand binding: molecular mechanics calculation of the streptavidin biotin rupture force</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">997</span><span class="NLM_x">–</span> <span class="NLM_lpage">999</span><span class="refDoi"> DOI: 10.1126/science.271.5251.997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.271.5251.997" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=997-999&author=H.+Grubmullerauthor=B.+Heymannauthor=P.+Tavan&title=Ligand+binding%3A+molecular+mechanics+calculation+of+the+streptavidin+biotin+rupture+force&doi=10.1126%2Fscience.271.5251.997"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fscience.271.5251.997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.271.5251.997%26sid%3Dliteratum%253Aachs%26aulast%3DGrubmuller%26aufirst%3DH.%26aulast%3DHeymann%26aufirst%3DB.%26aulast%3DTavan%26aufirst%3DP.%26atitle%3DLigand%2520binding%253A%2520molecular%2520mechanics%2520calculation%2520of%2520the%2520streptavidin%2520biotin%2520rupture%2520force%26jtitle%3DScience%26date%3D1996%26volume%3D271%26spage%3D997%26epage%3D999%26doi%3D10.1126%2Fscience.271.5251.997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Hamelberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mongan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">11919</span><span class="NLM_x">–</span> <span class="NLM_lpage">11929</span><span class="refDoi"> DOI: 10.1063/1.1755656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.1755656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15268227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvVant7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2004&pages=11919-11929&author=D.+Hamelbergauthor=J.+Monganauthor=J.+A.+McCammon&title=Accelerated+molecular+dynamics%3A+a+promising+and+efficient+simulation+method+for+biomolecules&doi=10.1063%2F1.1755656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules</span></div><div class="casAuthors">Hamelberg, Donald; Mongan, John; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11919-11929</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Many interesting dynamic properties of biol. mols. cannot be simulated directly using mol. dynamics because of nanosecond time scale limitations.  These systems are trapped in potential energy min. with high free energy barriers for large nos. of computational steps.  The dynamic evolution of many mol. systems occurs through a series of rare events as the system moves from one potential energy basin to another.  Therefore, we have proposed a robust bias potential function that can be used in an efficient accelerated mol. dynamics approach to simulate the transition of high energy barriers without any advance knowledge of the location of either the potential energy wells or saddle points.  In this method, the potential energy landscape is altered by adding a bias potential to the true potential such that the escape rates from potential wells are enhanced, which accelerates and extends the time scale in mol. dynamics simulations.  Our definition of the bias potential echoes the underlying shape of the potential energy landscape on the modified surface, thus allowing for the potential energy min. to be well defined, and hence properly sampled during the simulation.  We have shown that our approach, which can be extended to biomols., samples the conformational space more efficiently than normal mol. dynamics simulations, and converges to the correct canonical distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFSt4zsfWGVbVg90H21EOLACvtfcHk0lj3ZxUlDhyhIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvVant7w%253D&md5=f7ee967e10493f27a944bed8cd17c640</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1063%2F1.1755656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1755656%26sid%3Dliteratum%253Aachs%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DMongan%26aufirst%3DJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAccelerated%2520molecular%2520dynamics%253A%2520a%2520promising%2520and%2520efficient%2520simulation%2520method%2520for%2520biomolecules%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2004%26volume%3D120%26spage%3D11919%26epage%3D11929%26doi%3D10.1063%2F1.1755656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Faradjian, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elber, R.</span><span> </span><span class="NLM_article-title">Computing time scales from reaction coordinates by milestoning</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">10880</span><span class="NLM_x">–</span> <span class="NLM_lpage">10889</span><span class="refDoi"> DOI: 10.1063/1.1738640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.1738640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15268118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVCqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2004&pages=10880-10889&author=A.+K.+Faradjianauthor=R.+Elber&title=Computing+time+scales+from+reaction+coordinates+by+milestoning&doi=10.1063%2F1.1738640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Computing time scales from reaction coordinates by milestoning</span></div><div class="casAuthors">Faradjian, Anton K.; Elber, Ron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10880-10889</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">An algorithm is presented to compute time scales of complex processes following predetd. milestones along a reaction coordinate.  A non-Markovian hopping mechanism is assumed and constructed from underlying microscopic dynamics.  General anal. anal., a pedagogical example, and numerical solns. of the non-Markovian model are presented.  No assumption is made in the theor. derivation on the type of microscopic dynamics along the reaction coordinate.  However, the detailed calcns. are for Brownian dynamics in which the velocities are uncorrelated in time (but spatial memory remains).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdLFuyCZqyWrVg90H21EOLACvtfcHk0lj3ZxUlDhyhIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVCqt74%253D&md5=cfa2d2ed0611f22293240622883a4925</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1063%2F1.1738640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1738640%26sid%3Dliteratum%253Aachs%26aulast%3DFaradjian%26aufirst%3DA.%2BK.%26aulast%3DElber%26aufirst%3DR.%26atitle%3DComputing%2520time%2520scales%2520from%2520reaction%2520coordinates%2520by%2520milestoning%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2004%26volume%3D120%26spage%3D10880%26epage%3D10889%26doi%3D10.1063%2F1.1738640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Bolhuis, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dellago, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geissler, P. L.</span><span> </span><span class="NLM_article-title">Transition path sampling: Throwing ropes over rough mountain passes, in the dark</span> <span class="citation_source-journal">Annu. Rev. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">318</span><span class="refDoi"> DOI: 10.1146/annurev.physchem.53.082301.113146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1146%2Fannurev.physchem.53.082301.113146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11972010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Ois7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2002&pages=291-318&author=P.+G.+Bolhuisauthor=D.+Chandlerauthor=C.+Dellagoauthor=P.+L.+Geissler&title=Transition+path+sampling%3A+Throwing+ropes+over+rough+mountain+passes%2C+in+the+dark&doi=10.1146%2Fannurev.physchem.53.082301.113146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Transition path sampling: throwing ropes over rough mountain passes, in the dark</span></div><div class="casAuthors">Bolhuis, Peter G.; Chandler, David; Dellago, Christoph; Geissler, Phillip L.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Physical Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">291-318</span>CODEN:
                <span class="NLM_cas:coden">ARPLAP</span>;
        ISSN:<span class="NLM_cas:issn">0066-426X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review is given of the concepts and methods of transition path sampling.  These methods allow computational studies of rare events without requiring prior knowledge of mechanisms, reaction coordinates, and transition states.  Based upon a statistical mechanics of trajectory space, they provide a perspective with which time dependent phenomena, even for systems driven far from equil., can be examd. with the same types of importance sampling tools that in the past have been applied so successfully to static equil. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_Z72WTpw67Vg90H21EOLACvtfcHk0lglw0xIXqo5eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Ois7g%253D&md5=fbf79b52e751dfd87a73c345b9581898</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1146%2Fannurev.physchem.53.082301.113146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.physchem.53.082301.113146%26sid%3Dliteratum%253Aachs%26aulast%3DBolhuis%26aufirst%3DP.%2BG.%26aulast%3DChandler%26aufirst%3DD.%26aulast%3DDellago%26aufirst%3DC.%26aulast%3DGeissler%26aufirst%3DP.%2BL.%26atitle%3DTransition%2520path%2520sampling%253A%2520Throwing%2520ropes%2520over%2520rough%2520mountain%2520passes%252C%2520in%2520the%2520dark%26jtitle%3DAnnu.%2520Rev.%2520Phys.%2520Chem.%26date%3D2002%26volume%3D53%26spage%3D291%26epage%3D318%26doi%3D10.1146%2Fannurev.physchem.53.082301.113146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Ellingson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baudry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span> </span><span class="NLM_article-title">Multi-conformer ensemble docking to difficult protein targets</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1021/jp506511p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp506511p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWjsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=1026-1034&author=S.+R.+Ellingsonauthor=Y.+Miaoauthor=J.+Baudryauthor=J.+C.+Smith&title=Multi-conformer+ensemble+docking+to+difficult+protein+targets&doi=10.1021%2Fjp506511p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Conformer Ensemble Docking to Difficult Protein Targets</span></div><div class="casAuthors">Ellingson, Sally R.; Miao, Yinglong; Baudry, Jerome; Smith, Jeremy C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1026-1034</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large-scale ensemble docking was studied using five proteins from the Directory of Useful Decoys (DUD, dud.docking.org) for which docking to crystal structures proved difficult.  Mol. dynamics trajectories were produced for each protein and an ensemble of representative conformational structures extd. from the trajectories.  Docking calcns. were performed on these selected simulation structures and ensemble-based enrichment factors compared with those obtained using docking in crystal structures of the same protein targets or random selection of compds.  Simulation-derived snapshots are found with improved enrichment factors that increase the chem. diversity of docking hits for four of the five selected proteins.  A combination of all the docking results obtained from mol. dynamics simulation followed by selection of top-ranking compds. appears to be an effective strategy for increasing the no. and diversity of hits when using docking to screen large libraries of chems. against difficult protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8lJ2vsbEsz7Vg90H21EOLACvtfcHk0lglw0xIXqo5eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWjsb3E&md5=b421fbb1a3b7fc57d06e4645476aabb8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjp506511p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp506511p%26sid%3Dliteratum%253Aachs%26aulast%3DEllingson%26aufirst%3DS.%2BR.%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DBaudry%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26atitle%3DMulti-conformer%2520ensemble%2520docking%2520to%2520difficult%2520protein%2520targets%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2015%26volume%3D119%26spage%3D1026%26epage%3D1034%26doi%3D10.1021%2Fjp506511p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structural biology and drug discovery of difficult targets: the limits of ligandability</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.chembiol.2011.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22284353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=42-50&author=S.+Suradeauthor=T.+L.+Blundell&title=Structural+biology+and+drug+discovery+of+difficult+targets%3A+the+limits+of+ligandability&doi=10.1016%2Fj.chembiol.2011.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology and Drug Discovery of Difficult Targets: The Limits of Ligandability</span></div><div class="casAuthors">Surade, Sachin; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, researchers in the pharmaceutical industry and academia have made retrospective analyses of successful drug campaigns in order to establish "rules" to guide the selection of new target proteins.  They have identified features that are considered undesirable and some that make targets "unligandable.".  This review focuses on the factors that make targets difficult: featureless binding sites, the lack of hydrogen-bond donors and acceptors, the presence of metal ions, the need for adaptive changes in conformation, and the lipophilicity of residues at the protein-ligand interface.  Protein-protein interfaces of multiprotein assemblies share many of these undesirable features, although those that involve concerted binding and folding in their assembly have better defined pockets or grooves, and these can provide opportunities for identifying hits and for lead optimization.  In some protein-protein interfaces conformational changes-often involving rearrangement of large side chains such as those of tyrosine, tryptophan, or arginine-are required to configure an appropriate binding site, and this may require tethering of the ligands until higher affinity is achieved.  In many enzymes, larger conformational rearrangements are required to form the binding site, and these can make fragment-based approaches particularly difficult.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqCA_OH6VRoLVg90H21EOLACvtfcHk0lglw0xIXqo5eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D&md5=646b6cad327b44785848375383ca90ea</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520and%2520drug%2520discovery%2520of%2520difficult%2520targets%253A%2520the%2520limits%2520of%2520ligandability%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D42%26epage%3D50%26doi%3D10.1016%2Fj.chembiol.2011.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Bowman, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolin, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maguire, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marqusee, S.</span><span> </span><span class="NLM_article-title">Discovery of multiple hidden allosteric sites by combining markov state models and experiments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">2734</span><span class="NLM_x">–</span> <span class="NLM_lpage">2739</span><span class="refDoi"> DOI: 10.1073/pnas.1417811112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1417811112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25730859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=2734-2739&author=G.+R.+Bowmanauthor=E.+R.+Bolinauthor=K.+M.+Hartauthor=B.+C.+Maguireauthor=S.+Marqusee&title=Discovery+of+multiple+hidden+allosteric+sites+by+combining+markov+state+models+and+experiments&doi=10.1073%2Fpnas.1417811112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of multiple hidden allosteric sites by combining Markov state models and experiments</span></div><div class="casAuthors">Bowman, Gregory R.; Bolin, Eric R.; Hart, Kathryn M.; Maguire, Brendan C.; Marqusee, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2734-2739</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The discovery of drug-like mols. that bind pockets in proteins that are not present in crystallog. structures yet exert allosteric control over activity has generated great interest in designing pharmaceuticals that exploit allosteric effects.  However, there have only been a small no. of successes, so the therapeutic potential of these pockets--called hidden allosteric sites--remains unclear.  One challenge for assessing their utility is that rational drug design approaches require foreknowledge of the target site, but most hidden allosteric sites are only discovered when a small mol. is found to stabilize them.  The authors present a means of decoupling the identification of hidden allosteric sites from the discovery of drugs that bind them by drawing on new developments in Markov state modeling that provide unprecedented access to microsecond- to millisecond-timescale fluctuations of a protein's structure.  Visualizing these fluctuations allows the authors to identify potential hidden allosteric sites, which the authors then test via thiol labeling expts.  Application of these methods reveals multiple hidden allosteric sites in an important antibiotic target-TEM-1 β-lactamase.  This result supports the hypothesis that there are many as yet undiscovered hidden allosteric sites and suggests the authors' methodol. can identify such sites, providing a starting point for future drug design efforts.  More generally, the authors' results demonstrate the power of using Markov state models to guide expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY4EqjsensDLVg90H21EOLACvtfcHk0lhtTeM5aZnOXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D&md5=f20db19a2f0f13679bc64f4616c856e7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417811112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417811112%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DBolin%26aufirst%3DE.%2BR.%26aulast%3DHart%26aufirst%3DK.%2BM.%26aulast%3DMaguire%26aufirst%3DB.%2BC.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520multiple%2520hidden%2520allosteric%2520sites%2520by%2520combining%2520markov%2520state%2520models%2520and%2520experiments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D2734%26epage%3D2739%26doi%3D10.1073%2Fpnas.1417811112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span> </span><span class="NLM_article-title">Recent computational advances in the identification of allosteric sites in proteins</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1595</span><span class="NLM_x">–</span> <span class="NLM_lpage">1600</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2014.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25107670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1595-1600&author=S.+Luauthor=W.+Huangauthor=J.+Zhang&title=Recent+computational+advances+in+the+identification+of+allosteric+sites+in+proteins&doi=10.1016%2Fj.drudis.2014.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Recent computational advances in the identification of allosteric sites in proteins</span></div><div class="casAuthors">Lu, Shaoyong; Huang, Wenkang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1595-1600</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allosteric modulators have the potential to fine-tune protein functional activity.  Therefore, the targeting of allosteric sites, as a strategy in drug design, is gaining increasing attention.  Currently, it is not trivial to find and characterize new allosteric sites by exptl. approaches.  Alternatively, computational approaches are useful in helping researchers analyze and select potential allosteric sites for drug discovery.  Here, we review state-of-the-art computational approaches directed at predicting putative allosteric sites in proteins, along with examples of successes in identifying allosteric sites utilizing these methods.  We also discuss the challenges in developing reliable methods for predicting allosteric sites and tactics to resolve demanding tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaspE0Qbe47Vg90H21EOLACvtfcHk0lhtTeM5aZnOXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F&md5=fbaefacf51bee99d30fb882f0f04a47d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRecent%2520computational%2520advances%2520in%2520the%2520identification%2520of%2520allosteric%2520sites%2520in%2520proteins%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1595%26epage%3D1600%26doi%3D10.1016%2Fj.drudis.2014.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Lodola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">The increasing role of QM/MM in drug discovery</span> <span class="citation_source-journal">Adv. Protein Chem. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">362</span><span class="refDoi"> DOI: 10.1016/B978-0-12-398312-1.00011-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FB978-0-12-398312-1.00011-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22607760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Kmu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2012&pages=337-362&author=A.+Lodolaauthor=M.+De+Vivo&title=The+increasing+role+of+QM%2FMM+in+drug+discovery&doi=10.1016%2FB978-0-12-398312-1.00011-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">The increasing role of QM/MM in drug discovery</span></div><div class="casAuthors">Lodola, Alessio; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Protein Chemistry and Structural Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">Structural and Mechanistic Enzymology</span>),
    <span class="NLM_cas:pages">337-362</span>CODEN:
                <span class="NLM_cas:coden">APCSG7</span>;
        ISSN:<span class="NLM_cas:issn">1876-1623</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Since its first appearance in 1976, the quantum mechanics/mol. mechanics (QM/MM) approach has mostly been used to study the chem. reactions of enzymes, which are frequently the target of drug discovery programs.  In principle, a detailed understanding of the enzymic mechanism should help researchers to design a potent enzyme inhibitor or new drug.  However, QM/MM has not yet had a widespread impact on structure-based drug design.  This is mostly due to its high computational cost.  We expect this to change with the recent and extraordinary increases in computational power, and with the availability of more efficient algorithms for QM/MM calcns.  Here, we report on some representative examples of QM/MM studies, including our own research, of pharmaceutically relevant enzymes, such as RNase H and fatty acid amide hydrolase (FAAH).  We aim to show how QM/MM has traditionally been used to study enzymic catalysis.  In this regard, we discuss its potential to become a routinely used drug design tool.  To support this, we also discuss selected computational studies where QM/MM insights have been helpful in improving the potency of covalent inhibitors of FAAH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8JGGBuNrFw7Vg90H21EOLACvtfcHk0lhtTeM5aZnOXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Kmu7nL&md5=f3712a42a86ad8c784d03933b0306aff</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-398312-1.00011-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-398312-1.00011-1%26sid%3Dliteratum%253Aachs%26aulast%3DLodola%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DThe%2520increasing%2520role%2520of%2520QM%252FMM%2520in%2520drug%2520discovery%26jtitle%3DAdv.%2520Protein%2520Chem.%2520Struct.%2520Biol.%26date%3D2012%26volume%3D87%26spage%3D337%26epage%3D362%26doi%3D10.1016%2FB978-0-12-398312-1.00011-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothlisberger, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Computational insights into function and inhibition of fatty acid amide hydrolase</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.ejmech.2014.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25240419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2015&pages=15-26&author=G.+Palermoauthor=U.+Rothlisbergerauthor=A.+Cavalliauthor=M.+De+Vivo&title=Computational+insights+into+function+and+inhibition+of+fatty+acid+amide+hydrolase&doi=10.1016%2Fj.ejmech.2014.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Computational insights into function and inhibition of fatty acid amide hydrolase</span></div><div class="casAuthors">Palermo, Giulia; Rothlisberger, Ursula; Cavalli, Andrea; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-26</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The Fatty Acid Amide Hydrolase (FAAH) enzyme is a membrane-bound serine hydrolase responsible for the deactivating hydrolysis of a family of naturally occurring fatty acid amides.  FAAH is a crit. enzyme of the endocannabinoid system, being mainly responsible for regulating the level of its main cannabinoid substrate anandamide.  For this reason, pharmacol. inhibition of FAAH, which increases the level of endogenous anandamide, is a promising strategy to cure a variety of diseases including pain, inflammation, and cancer.Much structural, mutagenesis, and kinetic data on FAAH has been generated over the last couple of decades.  This has prompted several informative computational investigations to elucidate, at the at.-level, mechanistic details on catalysis and inhibition of this pharmaceutically relevant enzyme.  Here, we review how these computational studies - based on classical mol. dynamics, full quantum mechanics, and hybrid QM/MM methods - have clarified the binding and reactivity of some relevant substrates and inhibitors of FAAH.  We also discuss the exptl. implications of these computational insights, which have provided a thoughtful elucidation of the complex phys. and chem. steps of the enzymic mechanism of FAAH.  Finally, we discuss how computations have been helpful for building structure-activity relationships of potent FAAH inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-xquSauxYo7Vg90H21EOLACvtfcHk0lgwADaqCwJntg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOls7jJ&md5=5dd76bb3eda21ebdfe66e956a46f7858</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DRothlisberger%26aufirst%3DU.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DComputational%2520insights%2520into%2520function%2520and%2520inhibition%2520of%2520fatty%2520acid%2520amide%2520hydrolase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D91%26spage%3D15%26epage%3D26%26doi%3D10.1016%2Fj.ejmech.2014.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carloni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span> </span><span class="NLM_article-title">Target-related applications of first principles quantum chemical methods in drug design</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">3497</span><span class="NLM_x">–</span> <span class="NLM_lpage">3519</span><span class="refDoi"> DOI: 10.1021/cr050579p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr050579p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=3497-3519&author=A.+Cavalliauthor=P.+Carloniauthor=M.+Recanatini&title=Target-related+applications+of+first+principles+quantum+chemical+methods+in+drug+design&doi=10.1021%2Fcr050579p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fcr050579p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050579p%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DCarloni%26aufirst%3DP.%26aulast%3DRecanatini%26aufirst%3DM.%26atitle%3DTarget-related%2520applications%2520of%2520first%2520principles%2520quantum%2520chemical%2520methods%2520in%2520drug%2520design%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D3497%26epage%3D3519%26doi%3D10.1021%2Fcr050579p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Cornell, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cieplak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gould, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spellmeyer, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. a.</span><span> </span><span class="NLM_article-title">A 2nd generation force-field for the simulation of proteins, nucleic-acids, and organic-molecules</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">5179</span><span class="NLM_x">–</span> <span class="NLM_lpage">5197</span><span class="refDoi"> DOI: 10.1021/ja00124a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00124a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=5179-5197&author=W.+D.+Cornellauthor=P.+Cieplakauthor=C.+I.+Baylyauthor=I.+R.+Gouldauthor=K.+M.+Merzauthor=D.+M.+Fergusonauthor=D.+C.+Spellmeyerauthor=T.+Foxauthor=J.+W.+Caldwellauthor=P.+a.+Kollman&title=A+2nd+generation+force-field+for+the+simulation+of+proteins%2C+nucleic-acids%2C+and+organic-molecules&doi=10.1021%2Fja00124a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A Second Generation Force Field for the Simulation of Proteins, Nucleic Acids, and Organic Molecules</span></div><div class="casAuthors">Cornell, Wendy D.; Cieplak, Piotr; Bayly, Christopher I.; Gould, Ian R.; Merz, Kenneth M., Jr.; Ferguson, David M.; Spellmeyer, David C.; Fox, Thomas; Caldwell, James W.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5179-97</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors present the derivation of a new mol. mech. force field for simulating the structures, conformational energies, and interaction energies of proteins, nucleic acids, and many related org. mols. in condensed phases.  This effective two-body force field is the successor to the Weiner et al. force field and was developed with some of the same philosophies, such as the use of a simple diagonal potential function and electrostatic potential fit atom centered charges.  The need for a 10-12 function for representing hydrogen bonds is no longer necessary due to the improved performance of the new charge model and new van der Waals parameters.  These new charges are detd. using a 6-31G* basis set and restrained electrostatic potential (RESP) fitting and have been shown to reproduce interaction energies, free energies of solvation, and conformational energies of simple small mols. to a good degree of accuracy.  Furthermore, the new RESP charges exhibit less variability as a function of the mol. conformation used in the charge detn.  The new van der Waals parameters have been derived from liq. simulations and include hydrogen parameters which take into account the effects of any geminal electroneg. atoms.  The bonded parameters developed by Weiner et al. were modified as necessary to reproduce exptl. vibrational frequencies and structures.  Most of the simple dihedral parameters have been retained from Weiner et al., but a complex set of .vphi. and ψ parameters which do a good job of reproducing the energies of the low-energy conformations of glycyl and alanyl dipeptides was developed for the peptide backbone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_vSNiuEezGrVg90H21EOLACvtfcHk0lgwADaqCwJntg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D&md5=2c8901ed614d543db51d3a28f5b6053c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fja00124a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00124a002%26sid%3Dliteratum%253Aachs%26aulast%3DCornell%26aufirst%3DW.%2BD.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DGould%26aufirst%3DI.%2BR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26aulast%3DSpellmeyer%26aufirst%3DD.%2BC.%26aulast%3DFox%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2Ba.%26atitle%3DA%25202nd%2520generation%2520force-field%2520for%2520the%2520simulation%2520of%2520proteins%252C%2520nucleic-acids%252C%2520and%2520organic-molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5179%26epage%3D5197%26doi%3D10.1021%2Fja00124a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">MacKerell, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evanseck, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchnir, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuczera, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, F. T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michnick, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prodhom, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiher, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlenkrich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stote, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Straub, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiorkiewicz-Kuczera, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">All-atom empirical potential for molecular modeling and dynamics studies of proteins</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3586</span><span class="NLM_x">–</span> <span class="NLM_lpage">3616</span><span class="refDoi"> DOI: 10.1021/jp973084f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp973084f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3586-3616&author=A.+D.+MacKerellauthor=D.+Bashfordauthor=M.+Bellottauthor=R.+L.+Dunbrackauthor=J.+D.+Evanseckauthor=M.+J.+Fieldauthor=S.+Fischerauthor=J.+Gaoauthor=H.+Guoauthor=S.+Haauthor=D.+Joseph-McCarthyauthor=L.+Kuchnirauthor=K.+Kuczeraauthor=F.+T.+K.+Lauauthor=C.+Mattosauthor=S.+Michnickauthor=T.+Ngoauthor=D.+T.+Nguyenauthor=B.+Prodhomauthor=W.+E.+Reiherauthor=B.+Rouxauthor=M.+Schlenkrichauthor=J.+C.+Smithauthor=R.+Stoteauthor=J.+Straubauthor=M.+Watanabeauthor=J.+Wiorkiewicz-Kuczeraauthor=D.+Yinauthor=M.+Karplus&title=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins&doi=10.1021%2Fjp973084f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">All-Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins</span></div><div class="casAuthors">MacKerell, A. D., Jr.; Bashford, D.; Bellott, M.; Dunbrack, R. L.; Evanseck, J. D.; Field, M. J.; Fischer, S.; Gao, J.; Guo, H.; Ha, S.; Joseph-McCarthy, D.; Kuchnir, L.; Kuczera, K.; Lau, F. T. K.; Mattos, C.; Michnick, S.; Ngo, T.; Nguyen, D. T.; Prodhom, B.; Reiher, W. E., III; Roux, B.; Schlenkrich, M.; Smith, J. C.; Stote, R.; Straub, J.; Watanabe, M.; Wiorkiewicz-Kuczera, J.; Yin, D.; Karplus, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3586-3616</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New protein parameters are reported for the all-atom empirical energy function in the CHARMM program.  The parameter evaluation was based on a self-consistent approach designed to achieve a balance between the internal (bonding) and interaction (nonbonding) terms of the force field and among the solvent-solvent, solvent-solute, and solute-solute interactions.  Optimization of the internal parameters used exptl. gas-phase geometries, vibrational spectra, and torsional energy surfaces supplemented with ab initio results.  The peptide backbone bonding parameters were optimized with respect to data for N-methylacetamide and the alanine dipeptide.  The interaction parameters, particularly the at. charges, were detd. by fitting ab initio interaction energies and geometries of complexes between water and model compds. that represented the backbone and the various side chains.  In addn., dipole moments, exptl. heats and free energies of vaporization, solvation and sublimation, mol. vols., and crystal pressures and structures were used in the optimization.  The resulting protein parameters were tested by applying them to noncyclic tripeptide crystals, cyclic peptide crystals, and the proteins crambin, bovine pancreatic trypsin inhibitor, and carbonmonoxy myoglobin in vacuo and in a crystal.  A detailed anal. of the relationship between the alanine dipeptide potential energy surface and calcd. protein φ, χ angles was made and used in optimizing the peptide group torsional parameters.  The results demonstrate that use of ab initio structural and energetic data by themselves are not sufficient to obtain an adequate backbone representation for peptides and proteins in soln. and in crystals.  Extensive comparisons between mol. dynamics simulation and exptl. data for polypeptides and proteins were performed for both structural and dynamic properties.  Calcd. data from energy minimization and dynamics simulations for crystals demonstrate that the latter are needed to obtain meaningful comparisons with exptl. crystal structures.  The presented parameters, in combination with the previously published CHARMM all-atom parameters for nucleic acids and lipids, provide a consistent set for condensed-phase simulations of a wide variety of mols. of biol. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor1-3zB5qPwrVg90H21EOLACvtfcHk0lgwADaqCwJntg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D&md5=ebb5100dafd0daeee60ca2fa66c1324a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjp973084f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp973084f%26sid%3Dliteratum%253Aachs%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DBashford%26aufirst%3DD.%26aulast%3DBellott%26aufirst%3DM.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DEvanseck%26aufirst%3DJ.%2BD.%26aulast%3DField%26aufirst%3DM.%2BJ.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DKuchnir%26aufirst%3DL.%26aulast%3DKuczera%26aufirst%3DK.%26aulast%3DLau%26aufirst%3DF.%2BT.%2BK.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DMichnick%26aufirst%3DS.%26aulast%3DNgo%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DD.%2BT.%26aulast%3DProdhom%26aufirst%3DB.%26aulast%3DReiher%26aufirst%3DW.%2BE.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DSchlenkrich%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DStote%26aufirst%3DR.%26aulast%3DStraub%26aufirst%3DJ.%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DWiorkiewicz-Kuczera%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAll-atom%2520empirical%2520potential%2520for%2520molecular%2520modeling%2520and%2520dynamics%2520studies%2520of%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3586%26epage%3D3616%26doi%3D10.1021%2Fjp973084f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiradorives, J.</span><span> </span><span class="NLM_article-title">The Opls Potential Functions for proteins - energy minimizations for crystals of cyclic-peptides and crambin</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">1657</span><span class="NLM_x">–</span> <span class="NLM_lpage">1666</span><span class="refDoi"> DOI: 10.1021/ja00214a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=1657-1666&author=W.+L.+Jorgensenauthor=J.+Tiradorives&title=The+Opls+Potential+Functions+for+proteins+-+energy+minimizations+for+crystals+of+cyclic-peptides+and+crambin&doi=10.1021%2Fja00214a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1657-66</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A complete set of intermol. potential functions was developed for use in computer simulations of proteins in their native environment.  Parameters are reported for 25 peptide residues as well as the common neutral and charged terminal groups.  The potential functions have the simple Coulomb plus Lennard-Jones form and are compatible with the widely used models for water, TIP4P, TIP3P, and SPC.  The parameters were obtained and tested primarily in conjunction with Monte Carlo statistical mechanics simulations of 36 pure org. liqs. and numerous aq. solns. of org. ions representative of subunits in the side chains and backbones of proteins.  Bond stretch, angle bend, and torsional terms were adopted from the AMBER united-atom force field.  As reported here, further testing involved studies of conformational energy surfaces and optimizations of the crystal structures for 4 cyclic hexapeptides and a cyclic pentapeptide.  The av. root mean square deviation from the x-ray structures of the crystals is only 0.17 Å for the at. positions and 3% for the unit cell vols.  A more crit. test was then provided by performing energy minimizations for the complete crystal of the protein crambin, including 182 water mols. that were initially placed via a Monte Carlo simulation.  The resultant root mean square deviation for the non-H atoms is still only 0.17 Å and the variation in the errors for charged, polar, and nonpolar residues is small.  Significant improvement is apparent over the AMBER united-atom force field which was previously demonstrated to be superior to many alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6an38QWApbVg90H21EOLACvtfcHk0lgfUIaphGVXSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D&md5=b6c901d8c295b3b37329a7faef527e12</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fja00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTiradorives%26aufirst%3DJ.%26atitle%3DThe%2520Opls%2520Potential%2520Functions%2520for%2520proteins%2520-%2520energy%2520minimizations%2520for%2520crystals%2520of%2520cyclic-peptides%2520and%2520crambin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D1657%26epage%3D1666%26doi%3D10.1021%2Fja00214a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lgfUIaphGVXSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatcher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darian, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobyov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.</span><span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+MacKerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0lgjqHTVK_FoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lgjqHTVK_FoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Betz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Paramfit: automated optimization of force field parameters for molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1002/jcc.23775</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.23775" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25413259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=79-87&author=R.+M.+Betzauthor=R.+C.+Walker&title=Paramfit%3A+automated+optimization+of+force+field+parameters+for+molecular+dynamics+simulations&doi=10.1002%2Fjcc.23775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Paramfit: Automated optimization of force field parameters for molecular dynamics simulations</span></div><div class="casAuthors">Betz, Robin M.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">79-87</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The generation of bond, angle, and torsion parameters for classical mol. dynamics force fields typically requires fitting parameters such that classical properties such as energies and gradients match precalcd. quantum data for structures that scan the value of interest.  We present a program, Paramfit, distributed as part of the AmberTools software package that automates and extends this fitting process, allowing for simplified parameter generation for applications ranging from single mols. to entire force fields.  Paramfit implements a novel combination of a genetic and simplex algorithm to find the optimal set of parameters that replicate either quantum energy or force data.  The program allows for the derivation of multiple parameters simultaneously using significantly fewer quantum calcns. than previous methods, and can also fit parameters across multiple mols. with applications to force field development.  Paramfit has been applied successfully to systems with a sparse no. of structures, and has already proven crucial in the development of the Assisted Model Building with Energy Refinement Lipid14 force field. © 2014 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt7_grqBnCgLVg90H21EOLACvtfcHk0lgjqHTVK_FoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtLbM&md5=be5df82613ad9916e567795a54f4f3b3</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23775%26sid%3Dliteratum%253Aachs%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DParamfit%253A%2520automated%2520optimization%2520of%2520force%2520field%2520parameters%2520for%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D79%26epage%3D87%26doi%3D10.1002%2Fjcc.23775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Hopkins, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Fitting of dihedral terms in classical force fields as an analytic linear least-squares problem</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1978</span><span class="NLM_x">–</span> <span class="NLM_lpage">1986</span><span class="refDoi"> DOI: 10.1021/ci500112w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500112w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaksrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1978-1986&author=C.+W.+Hopkinsauthor=A.+E.+Roitberg&title=Fitting+of+dihedral+terms+in+classical+force+fields+as+an+analytic+linear+least-squares+problem&doi=10.1021%2Fci500112w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Fitting of Dihedral Terms in Classical Force Fields as an Analytic Linear Least-Squares Problem</span></div><div class="casAuthors">Hopkins, Chad W.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1978-1986</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The derivation and optimization of most energy terms in modern force fields are aided by automated computational tools.  It is therefore important to have algorithms to rapidly and precisely train large nos. of interconnected parameters to allow investigators to make better decisions about the content of mol. models.  In particular, the traditional approach to deriving dihedral parameters has been a least-squares fit to target conformational energies through variational optimization strategies.  We present a computational approach for simultaneously fitting force field dihedral amplitudes and phase consts. which is analytic within the scope of the data set.  This approach completes the optimal mol. mechanics representation of a quantum mech. potential energy surface in a single linear least-squares fit by recasting the dihedral potential into a linear function in the parameters.  We compare the resulting method to a genetic algorithm in terms of computational time and quality of fit for two simple mols.  As suggested in previous studies, arbitrary dihedral phases are only necessary when modeling chiral mols., which include more than half of drugs currently in use, so we also examd. a dihedral parametrization case for the drug amoxicillin and one of its stereoisomers where the target dihedral includes a chiral center.  Asym. dihedral phases are needed in these types of cases to properly represent the quantum mech. energy surface and to differentiate between stereoisomers about the chiral center.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprIgbV7-frvrVg90H21EOLACvtfcHk0lhRCt90S6XUbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaksrvE&md5=70c162c18e1dcbfa462da31d1d554b8c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci500112w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500112w%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DC.%2BW.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DFitting%2520of%2520dihedral%2520terms%2520in%2520classical%2520force%2520fields%2520as%2520an%2520analytic%2520linear%2520least-squares%2520problem%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1978%26epage%3D1986%26doi%3D10.1021%2Fci500112w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Automated force field parameterization for non-polarizable and polarizable atomic models based on target data</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3543</span><span class="NLM_x">–</span> <span class="NLM_lpage">3556</span><span class="refDoi"> DOI: 10.1021/ct4003477</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct4003477" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSns7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3543-3556&author=L.+Huangauthor=B.+Roux&title=Automated+force+field+parameterization+for+non-polarizable+and+polarizable+atomic+models+based+on+target+data&doi=10.1021%2Fct4003477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Force Field Parameterization for Nonpolarizable and Polarizable Atomic Models Based on Ab Initio Target Data</span></div><div class="casAuthors">Huang, Lei; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3543-3556</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Classical mol. dynamics (MD) simulations based on atomistic models are increasingly used to study a wide range of biol. systems.  A prerequisite for meaningful results from such simulations is an accurate mol. mech. force field.  Most biomol. simulations are currently based on the widely used AMBER and CHARMM force fields, which were parametrized and optimized to cover a small set of basic compds. corresponding to the natural amino acids and nucleic acid bases.  Atomic models of addnl. compds. are commonly generated by analogy to the parameter set of a given force field.  While this procedure yields models that are internally consistent, the accuracy of the resulting models can be limited.  The authors propose a method, general automated at. model parameterization (GAAMP), for generating automatically the parameters of at. models of small mols. using the results from ab initio quantum mech. (QM) calcns. as target data.  Force fields that were previously developed for a wide range of model compds. serve as initial guesses, although any of the final parameter can be optimized.  The electrostatic parameters (partial charges, polarizabilities, and shielding) are optimized on the basis of QM electrostatic potential (ESP) and, if applicable, the interaction energies between the compd. and water mols.  The soft dihedrals are automatically identified and parametrized by targeting QM dihedral scans as well as the energies of stable conformers.  To validate the approach, the solvation free energy is calcd. for >200 small mols. and MD simulations of three different proteins were carried out.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjMGhS8IcFx7Vg90H21EOLACvtfcHk0lhRCt90S6XUbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSns7rP&md5=8f7c3d2fd1e29afe685a4e94a5e0943a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fct4003477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct4003477%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAutomated%2520force%2520field%2520parameterization%2520for%2520non-polarizable%2520and%2520polarizable%2520atomic%2520models%2520based%2520on%2520target%2520data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3543%26epage%3D3556%26doi%3D10.1021%2Fct4003477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Wang, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Building force fields: an automatic, systematic, and reproducible approach</span> <span class="citation_source-journal">J. Phys. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1885</span><span class="NLM_x">–</span> <span class="NLM_lpage">1891</span><span class="refDoi"> DOI: 10.1021/jz500737m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jz500737m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvV2htrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1885-1891&author=L.+P.+Wangauthor=T.+J.+Martinezauthor=V.+S.+Pande&title=Building+force+fields%3A+an+automatic%2C+systematic%2C+and+reproducible+approach&doi=10.1021%2Fjz500737m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Building Force Fields: An Automatic, Systematic, and Reproducible Approach</span></div><div class="casAuthors">Wang, Lee-Ping; Martinez, Todd J.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1885-1891</span>CODEN:
                <span class="NLM_cas:coden">JPCLCD</span>;
        ISSN:<span class="NLM_cas:issn">1948-7185</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of accurate mol. mechanics force fields is a significant challenge that must be addressed for the continued success of mol. simulation.  We developed the ForceBalance method to automatically derive accurate force field parameters using flexible combinations of exptl. and theor. ref. data.  The method is demonstrated in the parametrization of two rigid water models, yielding new parameter sets (TIP3P-FB and TIP4P-FB) that accurately describe many phys. properties of water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFqC-UGvhNiLVg90H21EOLACvtfcHk0lhRCt90S6XUbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvV2htrs%253D&md5=0b901b4ef8949839df5b5e3e36e38424</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjz500737m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjz500737m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DMartinez%26aufirst%3DT.%2BJ.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DBuilding%2520force%2520fields%253A%2520an%2520automatic%252C%2520systematic%252C%2520and%2520reproducible%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1885%26epage%3D1891%26doi%3D10.1021%2Fjz500737m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Ryckaert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciccotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lhRCt90S6XUbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Hopkins, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Long-time-step molecular dynamics through hydrogen mass repartitioning</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1864</span><span class="NLM_x">–</span> <span class="NLM_lpage">1874</span><span class="refDoi"> DOI: 10.1021/ct5010406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5010406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1864-1874&author=C.+W.+Hopkinsauthor=S.+Le+Grandauthor=R.+C.+Walkerauthor=A.+E.+Roitberg&title=Long-time-step+molecular+dynamics+through+hydrogen+mass+repartitioning&doi=10.1021%2Fct5010406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Long-Time-Step Molecular Dynamics through Hydrogen Mass Repartitioning</span></div><div class="casAuthors">Hopkins, Chad W.; Le Grand, Scott; Walker, Ross C.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1864-1874</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies have shown that the method of hydrogen mass repartitioning (HMR) is a potentially useful tool for accelerating mol. dynamics (MD) simulations.  By repartitioning the mass of heavy atoms into the bonded hydrogen atoms, it is possible to slow the highest-frequency motions of the macromol. under study, thus allowing the time step of the simulation to be increased by up to a factor of 2.  In this communication, we investigate further how this mass repartitioning allows the simulation time step to be increased in a stable fashion without significantly increasing discretization error.  To this end, we ran a set of simulations with different time steps and mass distributions on a three-residue peptide to get a comprehensive view of the effect of mass repartitioning and time step increase on a system whose accessible phase space is fully explored in a relatively short amt. of time.  We next studied a 129-residue protein, hen egg white lysozyme (HEWL), to verify that the obsd. behavior extends to a larger, more-realistic, system.  Results for the protein include structural comparisons from MD trajectories, as well as comparisons of pKa calcns. via const.-pH MD.  We also calcd. a potential of mean force (PMF) of a dihedral rotation for the MTS [(1-oxyl-2,2,5,5-tetramethyl-pyrroline-3-methyl)methanethiosulfonate] spin label via umbrella sampling with a set of regular MD trajectories, as well as a set of mass-repartitioned trajectories with a time step of 4 fs.  Since no significant difference in kinetics or thermodn. is obsd. by the use of fast HMR trajectories, further evidence is provided that long-time-step HMR MD simulations are a viable tool for accelerating MD simulations for mols. of biochem. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfG1v1aGj1NrVg90H21EOLACvtfcHk0libkJVGIlA4Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D&md5=6ca75b544b63523481de7836d11a3c1b</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fct5010406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5010406%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DC.%2BW.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DLong-time-step%2520molecular%2520dynamics%2520through%2520hydrogen%2520mass%2520repartitioning%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D1864%26epage%3D1874%26doi%3D10.1021%2Fct5010406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Hünenberger, P.</span><span> </span><span class="NLM_article-title">Thermostat algorithms for molecular dynamics simulations</span>. In  <span class="citation_source-book">Advanced Computer Simulation</span>; <span class="NLM_contrib-group">Holm, C.; Kremer, K.</span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">173</span>, pp  <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">149</span>, DOI: <span class="refDoi"> DOI: 10.1007/b99427</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fb99427" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=105-149&author=P.+H%C3%BCnenbergerauthor=C.+Holm&author=K.+Kremer&title=Advanced+Computer+Simulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fb99427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb99427%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%26atitle%3DThermostat%2520algorithms%2520for%2520molecular%2520dynamics%2520simulations%26btitle%3DAdvanced%2520Computer%2520Simulation%26aulast%3DHolm%26aufirst%3DC.%26pub%3DSpringer%26date%3D2005%26volume%3D173%26spage%3D105%26epage%3D149%26doi%3D10.1007%2Fb99427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L.</span><span> </span><span class="NLM_article-title">Particle mesh Ewald: An N·log(N) method for ewald sums in large systems</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10089</span><span class="NLM_x">–</span> <span class="NLM_lpage">10092</span><span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N%C2%B7log%28N%29+method+for+ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0libkJVGIlA4Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Essmann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berkowitz, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L. G.</span><span> </span><span class="NLM_article-title">A smooth particle mesh ewald method</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">8577</span><span class="NLM_x">–</span> <span class="NLM_lpage">8593</span><span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0liGf1CfyI7zhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Sagui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T. A.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations of biomolecules: long-range electrostatic effects</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1146/annurev.biophys.28.1.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1146%2Fannurev.biophys.28.1.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10410799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1MXkt1ertbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1999&pages=155-179&author=C.+Saguiauthor=T.+A.+Darden&title=Molecular+dynamics+simulations+of+biomolecules%3A+long-range+electrostatic+effects&doi=10.1146%2Fannurev.biophys.28.1.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations of biomolecules: long-range electrostatic effects</span></div><div class="casAuthors">Sagui, Celeste; Darden, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-179</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 72 refs.  Current computer simulations of biomols. typically make use of classical mol. dynamics methods, as a very large no. (tens to hundreds of thousands) of atoms are involved over timescales of many nanoseconds.  The methodol. for treating short-range bonded and van der Waals interactions has matured.  However, long-range electrostatic interactions still represent a bottleneck in simulations.  In this article, we introduce the basic issues for an accurate representation of the relevant electrostatic interactions.  In spite of the huge computational time demanded by most biomol. systems, it is no longer necessary to resort to uncontrolled approxns. such as the use of cutoffs.  In particular, we discuss the Ewald summation methods, the fast particle mesh methods, and the fast multipole methods.  We also review recent efforts to understand the role of boundary conditions in systems with long-range interactions and conclude with a short perspective on future trends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoNMaeS53ux7Vg90H21EOLACvtfcHk0liGf1CfyI7zhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXkt1ertbc%253D&md5=1da8a35730741e9430fe29dbfe33e84f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.28.1.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.28.1.155%26sid%3Dliteratum%253Aachs%26aulast%3DSagui%26aufirst%3DC.%26aulast%3DDarden%26aufirst%3DT.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520of%2520biomolecules%253A%2520long-range%2520electrostatic%2520effects%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D1999%26volume%3D28%26spage%3D155%26epage%3D179%26doi%3D10.1146%2Fannurev.biophys.28.1.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span> </span><span class="NLM_article-title">Molecular mechanics and dynamics: numerical tools to sample the configuration space</span> <span class="citation_source-journal">Front. Biosci., Landmark Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">578</span><span class="NLM_x">–</span> <span class="NLM_lpage">604</span><span class="refDoi"> DOI: 10.2741/4229</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2741%2F4229" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=578-604&issue=19&author=M.+Masettiauthor=W.+Rocchia&title=Molecular+mechanics+and+dynamics%3A+numerical+tools+to+sample+the+configuration+space&doi=10.2741%2F4229"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.2741%2F4229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4229%26sid%3Dliteratum%253Aachs%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DMolecular%2520mechanics%2520and%2520dynamics%253A%2520numerical%2520tools%2520to%2520sample%2520the%2520configuration%2520space%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2014%26volume%3D1%26issue%3D19%26spage%3D578%26epage%3D604%26doi%3D10.2741%2F4229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Shi, Y.; Ren, P.; Schnieders, M.; Piquemal, J.-P.</span><span> </span><span class="NLM_article-title">Polarizable force fields for biomolecular modeling</span>. In  <span class="citation_source-book">Reviews in Computational Chemistry</span>; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">28</span>, pp  <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span>, DOI: <span class="refDoi"> DOI: 10.1002/9781118889886.ch2</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2F9781118889886.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=51-86&author=Y.+Shi&author=P.+Ren&author=M.+Schnieders&author=J.-P.+Piquemal&title=Reviews+in+Computational+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2F9781118889886.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118889886.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26atitle%3DPolarizable%2520force%2520fields%2520for%2520biomolecular%2520modeling%26btitle%3DReviews%2520in%2520Computational%2520Chemistry%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26volume%3D28%26spage%3D51%26epage%3D86%26doi%3D10.1002%2F9781118889886.ch2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Jiao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golubkov, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, P.</span><span> </span><span class="NLM_article-title">Calculation of protein-ligand binding free energy by using a polarizable potential</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6290</span><span class="NLM_x">–</span> <span class="NLM_lpage">6295</span><span class="refDoi"> DOI: 10.1073/pnas.0711686105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0711686105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18427113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6290-6295&author=D.+Jiaoauthor=P.+A.+Golubkovauthor=T.+A.+Dardenauthor=P.+Ren&title=Calculation+of+protein-ligand+binding+free+energy+by+using+a+polarizable+potential&doi=10.1073%2Fpnas.0711686105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of protein-ligand binding free energy by using a polarizable potential</span></div><div class="casAuthors">Jiao, Dian; Golubkov, Pavel A.; Darden, Thomas A.; Ren, Pengyu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6290-6295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The binding of charged ligands benzamidine and diazamidine to trypsin was investigated by using a polarizable potential energy function and explicit-water mol. dynamics simulations.  The binding free energies were computed from the difference between the free energies of decoupling the ligand from water and protein environments.  Both the abs. and the relative free energies from the perturbation simulations agree with exptl. measurements to within 0.5 kcal-mol-1.  Comparison of free-energy components sampled from different thermodn. paths indicates that electrostatics is the main driving force behind benzamidine recognition of trypsin.  The contribution of electronic polarization to binding appears to be crucial.  By computing the free-energy contribution caused by the polarization between the ligand and its surroundings, we found that polarization has the opposite effect in dissimilar environments.  Although polarization favors ligand solvation in water, it weakens the protein-ligand attraction by screening the electrostatic interaction between trypsin and benzamidine.  We also examd. the relative binding free energies of a benzamidine analog diazamidine to trypsin.  The changes in free energy on benzamidine-diazamidine substitution were tens of kilocalories in both water and trypsin environments; however, the change in the total binding free energy is <2 kcal-mol-1 because of cancellation, consistent with the exptl. results.  Overall, our results suggest that the use of a polarizable force field, given adequate sampling, is capable of achieving chem. accuracy in mol. simulations of protein-ligand recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc3_p806L5GLVg90H21EOLACvtfcHk0liGf1CfyI7zhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyht7Y%253D&md5=ca74cfaca7f521a315e2ee85440591f9</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711686105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711686105%26sid%3Dliteratum%253Aachs%26aulast%3DJiao%26aufirst%3DD.%26aulast%3DGolubkov%26aufirst%3DP.%2BA.%26aulast%3DDarden%26aufirst%3DT.%2BA.%26aulast%3DRen%26aufirst%3DP.%26atitle%3DCalculation%2520of%2520protein-ligand%2520binding%2520free%2520energy%2520by%2520using%2520a%2520polarizable%2520potential%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D6290%26epage%3D6295%26doi%3D10.1073%2Fpnas.0711686105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Warshel, a.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levitt, M.</span><span> </span><span class="NLM_article-title">Theoretical studies of enzymic reactions - dielectric, electrostatic and steric stabilization of carbonium-ion in reaction of lysozyme</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1016/0022-2836(76)90311-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0022-2836%2876%2990311-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=985660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE28XktFKhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1976&pages=227-249&author=a.+Warshelauthor=M.+Levitt&title=Theoretical+studies+of+enzymic+reactions+-+dielectric%2C+electrostatic+and+steric+stabilization+of+carbonium-ion+in+reaction+of+lysozyme&doi=10.1016%2F0022-2836%2876%2990311-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Theoretical studies of enzymic reactions:  dielectric, electrostatic and steric stabilization of the carbonium ion in the reaction of lysozyme</span></div><div class="casAuthors">Warshel, A.; Levitt, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">227-49</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A general method for detailed study of enzymic reactions is presented.  The method considers the complete enzyme-substrate complex together with the surrounding solvent and evaluates all the different quantum mech. and classical energy factors that can affect the reaction pathway.  These factors include the quantum mech. energies assocd. with bond cleavage and charge redistribution of the substrate and the classical energies of steric and electrostatic interactions between the substrate and the enzyme.  The electrostatic polarization of the enzyme atoms and the orientation of the dipoles of the surrounding H2O mols. is simulated by a microscopic dielec. model.  The solvation energy resulting from this polarization is considerable and must be included in any realistic calcn. of chem. reactions involving anything more than an isolated mol. in vacuo.  Without it, acidic groups can never become ionized and the charge distribution on the substrate will not be reasonable.  The same dielec. model can also be used to study the reaction of the substrate in soln.  In this way the reaction in soln. can be compared with the enzymic reaction.  The stability of the carbonium ion intermediate formed in the cleavage of a glycosidic bond by lysozyme was studied.  Electrostatic stabilization is an important factor in increasing the rate of the reaction step that leads to the formation of the carbonium ion intermediate.  Steric factors, such as the strain of the substrate on binding to lysozyme, do not seem to contribute significantly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB3Es44qCfVrVg90H21EOLACvtfcHk0ljtnVH_T3Grsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XktFKhtr0%253D&md5=f34df33b5971b6b02bd03be95dcd7ba5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2876%2990311-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252876%252990311-9%26sid%3Dliteratum%253Aachs%26aulast%3DWarshel%26aufirst%3Da.%26aulast%3DLevitt%26aufirst%3DM.%26atitle%3DTheoretical%2520studies%2520of%2520enzymic%2520reactions%2520-%2520dielectric%252C%2520electrostatic%2520and%2520steric%2520stabilization%2520of%2520carbonium-ion%2520in%2520reaction%2520of%2520lysozyme%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1976%26volume%3D103%26spage%3D227%26epage%3D249%26doi%3D10.1016%2F0022-2836%2876%2990311-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Raha, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">WollaCott, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westerhoff, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.</span><span> </span><span class="NLM_article-title">The role of quantum mechanics in structure-based drug design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span><span class="refDoi"> DOI: 10.1016/j.drudis.2007.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2007.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=17826685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=725-731&author=K.+Rahaauthor=M.+B.+Petersauthor=B.+Wangauthor=N.+Yuauthor=A.+M.+WollaCottauthor=L.+M.+Westerhoffauthor=K.+M.+Merz&title=The+role+of+quantum+mechanics+in+structure-based+drug+design&doi=10.1016%2Fj.drudis.2007.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">The role of quantum mechanics in structure-based drug design</span></div><div class="casAuthors">Raha, Kaushik; Peters, Martin B.; Wang, Bing; Yu, Ning; Wollacott, Andrew M.; Westerhoff, Lance M.; Merz, Kenneth M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">17&18</span>),
    <span class="NLM_cas:pages">725-731</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Herein we will focus on the use of quantum mechanics (QM) in drug design (DD) to solve disparate problems from scoring protein-ligand poses to building QM QSAR models.  Through the variational principle of QM we know that we can obtain a more accurate representation of mol. systems than classical models, and while this is not a matter of debate, it still has not been shown that the expense of QM approaches is offset by improved accuracy in DD applications.  Objectively validating the improved applicability and performance of QM over classical-based models in DD will be the focus of research in the coming years along with research on the conformational sampling problem as it relates to protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ISn2DWdN9LVg90H21EOLACvtfcHk0lhEOsSBYEindQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK&md5=67fecc13843c7961ff820026c85a9290</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DRaha%26aufirst%3DK.%26aulast%3DPeters%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DN.%26aulast%3DWollaCott%26aufirst%3DA.%2BM.%26aulast%3DWesterhoff%26aufirst%3DL.%2BM.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26atitle%3DThe%2520role%2520of%2520quantum%2520mechanics%2520in%2520structure-based%2520drug%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D725%26epage%3D731%26doi%3D10.1016%2Fj.drudis.2007.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Quantum mechanical methods for drug design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.2174/156802610790232242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802610790232242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19929831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=33-45&author=T.+Zhouauthor=D.+Huangauthor=A.+Caflisch&title=Quantum+mechanical+methods+for+drug+design&doi=10.2174%2F156802610790232242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum mechanical methods for drug design</span></div><div class="casAuthors">Zhou, Ting; Huang, Danzhi; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-45</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Quantum mech. (QM) methods are becoming popular in computational drug design and development mainly because high accuracy is required to est. (relative) binding affinities.  For low- to medium-throughput in silico screening, (e.g., scoring and prioritizing a series of inhibitors sharing the same mol. scaffold) efficient approxns. have been developed in the past decade, like linear scaling QM in which the computation time scales almost linearly with the no. of basis functions.  Furthermore, QM-based procedures have been used recently for detg. protonation states of ionizable groups, evaluating energies, and optimizing mol. structures.  For high-throughput in silico screening QM approaches have been employed to derive robust quant. structure-activity relation models.  It is expected that the use of QM methods will keep growing in all phases of computer-aided drug design and development.  However, extensive sampling of conformational space and treatment of soln. of macromols. are still limiting factors for the broad application of QM in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocoQjA2ielFbVg90H21EOLACvtfcHk0lhEOsSBYEindQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D&md5=735ac208504a7e563e6ca0eb3c1cd17b</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.2174%2F156802610790232242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232242%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DQuantum%2520mechanical%2520methods%2520for%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D33%26epage%3D45%26doi%3D10.2174%2F156802610790232242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">van Duin, A. C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorant, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, W. A.</span><span> </span><span class="NLM_article-title">ReaxFF: A reactive force field for hydrocarbons</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">9396</span><span class="NLM_x">–</span> <span class="NLM_lpage">9409</span><span class="refDoi"> DOI: 10.1021/jp004368u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp004368u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFChu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=9396-9409&author=A.+C.+T.+van+Duinauthor=S.+Dasguptaauthor=F.+Lorantauthor=W.+A.+Goddard&title=ReaxFF%3A+A+reactive+force+field+for+hydrocarbons&doi=10.1021%2Fjp004368u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">ReaxFF: A Reactive Force Field for Hydrocarbons</span></div><div class="casAuthors">van Duin, Adri C. T.; Dasgupta, Siddharth; Lorant, Francois; Goddard, William A., III</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9396-9409</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To make practical the mol. dynamics simulation of large scale reactive chem. systems (1000 s of atoms), the authors developed ReaxFF, a force field for reactive systems. ReaxFF uses a general relation between bond distance and bond order on one hand and between bond order and bond energy however, that leads to proper dissocn. of bonds to sepd. atoms.  Other valence terms present in the force field (angle and torsion) are defined in terms of the same bond orders so that all these terms go to zero smoothly as bonds break.  In addn., ReaxFF has Coulomb and Morse (van der Waals) potentials to describe nonbond interactions between all atoms (no exclusions).  These nonbond interactions are shielded at short range so that the Coulomb and van der Waals interactions become const. as Rij → 0.  The authors report here the ReaxFF for hydrocarbons.  The parameters were derived from quantum chem. calcns. on bond dissocn. and reactions of small mols. plus heat of formation and geometry data for a no. of stable hydrocarbon compds.  The ReaxFF provides a good description of these data. Generally, the results are of an accuracy similar or better than PM3, while ReaxFF is ∼100 times faster.  In turn, the PM3 is ∼100 times faster than the QC calcns. Thus, with ReaxFF the authors hope to be able to study complex reactions in hydrocarbons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZfMcUTM2NorVg90H21EOLACvtfcHk0lhEOsSBYEindQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFChu78%253D&md5=ea59efc08d5e135745df988f2006a7fd</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjp004368u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp004368u%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BDuin%26aufirst%3DA.%2BC.%2BT.%26aulast%3DDasgupta%26aufirst%3DS.%26aulast%3DLorant%26aufirst%3DF.%26aulast%3DGoddard%26aufirst%3DW.%2BA.%26atitle%3DReaxFF%253A%2520A%2520reactive%2520force%2520field%2520for%2520hydrocarbons%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2001%26volume%3D105%26spage%3D9396%26epage%3D9409%26doi%3D10.1021%2Fjp004368u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Goh, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hulbert, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.</span><span> </span><span class="NLM_article-title">Constant pH molecular dynamics of proteins in explicit solvent with proton tautomerism</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">1319</span><span class="NLM_x">–</span> <span class="NLM_lpage">1331</span><span class="refDoi"> DOI: 10.1002/prot.24499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.24499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24375620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt12ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2014&pages=1319-1331&author=G.+B.+Gohauthor=B.+S.+Hulbertauthor=H.+Q.+Zhouauthor=C.+L.+Brooks&title=Constant+pH+molecular+dynamics+of+proteins+in+explicit+solvent+with+proton+tautomerism&doi=10.1002%2Fprot.24499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH molecular dynamics of proteins in explicit solvent with proton tautomerism</span></div><div class="casAuthors">Goh, Garrett B.; Hulbert, Benjamin S.; Zhou, Huiqing; Brooks, Charles L., III</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1319-1331</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">PH is a ubiquitous regulator of biol. activity, including protein-folding, protein-protein interactions, and enzymic activity.  Existing const. pH mol. dynamics (CPHMD) models that were developed to address questions related to the pH-dependent properties of proteins are largely based on implicit solvent models.  However, implicit solvent models are known to underestimate the desolvation energy of buried charged residues, increasing the error assocd. with predictions that involve internal ionizable residue that are important in processes like hydrogen transport and electron transfer.  Furthermore, discrete water and ions cannot be modeled in implicit solvent, which are important in systems like membrane proteins and ion channels.  We report on an explicit solvent const. pH mol. dynamics framework based on multi-site λ-dynamics (CPHMDMSλD).  In the CPHMDMSλD framework, we performed seamless alchem. transitions between protonation and tautomeric states using multi-site λ-dynamics, and designed novel biasing potentials to ensure that the phys. end-states are predominantly sampled.  We show that explicit solvent CPHMDMSλD simulations model realistic pH-dependent properties of proteins such as the Hen-Egg White Lysozyme (HEWL), binding domain of 2-oxoglutarate dehydrogenase (BBL) and N-terminal domain of ribosomal protein L9 (NTL9), and the pKa predictions are in excellent agreement with exptl. values, with a RMSE ranging from 0.72 to 0.84 pKa units.  With the recent development of the explicit solvent CPHMDMSλD framework for nucleic acids, accurate modeling of pH-dependent properties of both major class of biomols.-proteins and nucleic acids is now possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Ai09dtRsLbVg90H21EOLACvtfcHk0lgHPrD1Q6CiLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt12ntw%253D%253D&md5=4e99b314f79489155546b3966614f832</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fprot.24499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24499%26sid%3Dliteratum%253Aachs%26aulast%3DGoh%26aufirst%3DG.%2BB.%26aulast%3DHulbert%26aufirst%3DB.%2BS.%26aulast%3DZhou%26aufirst%3DH.%2BQ.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26atitle%3DConstant%2520pH%2520molecular%2520dynamics%2520of%2520proteins%2520in%2520explicit%2520solvent%2520with%2520proton%2520tautomerism%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2014%26volume%3D82%26spage%3D1319%26epage%3D1331%26doi%3D10.1002%2Fprot.24499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Constant pH replica exchange molecular dynamics in explicit solvent using discrete protonation states: implementation, testing, and validation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1341</span><span class="NLM_x">–</span> <span class="NLM_lpage">1352</span><span class="refDoi"> DOI: 10.1021/ct401042b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct401042b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1341-1352&author=J.+M.+Swailsauthor=D.+M.+Yorkauthor=A.+E.+Roitberg&title=Constant+pH+replica+exchange+molecular+dynamics+in+explicit+solvent+using+discrete+protonation+states%3A+implementation%2C+testing%2C+and+validation&doi=10.1021%2Fct401042b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH Replica Exchange Molecular Dynamics in Explicit Solvent Using Discrete Protonation States: Implementation, Testing, and Validation</span></div><div class="casAuthors">Swails, Jason M.; York, Darrin M.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1341-1352</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By utilizing Graphics Processing Units, we show that const. pH mol. dynamics simulations (CpHMD) run in Generalized Born (GB) implicit solvent for long time scales can yield poor pKa predictions as a result of sampling unrealistic conformations.  To address this shortcoming, we present a method for performing const. pH mol. dynamics simulations (CpHMD) in explicit solvent using a discrete protonation state model.  The method involves std. mol. dynamics (MD) being propagated in explicit solvent followed by protonation state changes being attempted in GB implicit solvent at fixed intervals.  Replica exchange along the pH-dimension (pH-REMD) helps to obtain acceptable titrn. behavior with the proposed method.  We analyzed the effects of various parameters and settings on the titrn. behavior of CpHMD and pH-REMD in explicit solvent, including the size of the simulation unit cell and the length of the relaxation dynamics following protonation state changes.  We tested the method with the amino acid model compds., a small pentapeptide with two titratable sites, and hen egg white lysozyme (HEWL).  The proposed method yields superior predicted pKa values for HEWL over hundreds of nanoseconds of simulation relative to corresponding predicted values from simulations run in implicit solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlQ-zHy36SV7Vg90H21EOLACvtfcHk0lgHPrD1Q6CiLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyiu7g%253D&md5=7f320339f40c5308035b25cc1aaea09c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fct401042b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct401042b%26sid%3Dliteratum%253Aachs%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DYork%26aufirst%3DD.%2BM.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DConstant%2520pH%2520replica%2520exchange%2520molecular%2520dynamics%2520in%2520explicit%2520solvent%2520using%2520discrete%2520protonation%2520states%253A%2520implementation%252C%2520testing%252C%2520and%2520validation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D1341%26epage%3D1352%26doi%3D10.1021%2Fct401042b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Donnini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegeler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Constant pH molecular dynamics in explicit solvent with lambda-dynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1962</span><span class="NLM_x">–</span> <span class="NLM_lpage">1978</span><span class="refDoi"> DOI: 10.1021/ct200061r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200061r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=1962-1978&author=S.+Donniniauthor=F.+Tegelerauthor=G.+Groenhofauthor=H.+Grubmuller&title=Constant+pH+molecular+dynamics+in+explicit+solvent+with+lambda-dynamics&doi=10.1021%2Fct200061r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH Molecular Dynamics in Explicit Solvent with λ-Dynamics</span></div><div class="casAuthors">Donnini, Serena; Tegeler, Florian; Groenhof, Gerrit; Grubmuller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1962-1978</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PH is an important parameter in condensed-phase systems, because it dets. the protonation state of titratable groups and thus influences the structure, dynamics, and function of mols. in soln.  In most force field simulation protocols, however, the protonation state of a system (rather than its pH) is kept fixed and cannot adapt to changes of the local environment.  Here, we present a method, implemented within the MD package GROMACS, for const. pH mol. dynamics simulations in explicit solvent that is based on the λ-dynamics approach.  In the latter, the dynamics of the titrn. coordinate λ, which interpolates between the protonated and deprotonated states, is driven by generalized forces between the protonated and deprotonated states.  The hydration free energy, as a function of pH, is included to facilitate const. pH simulations.  The protonation states of titratable groups are allowed to change dynamically during a simulation, thus reproducing av. protonation probabilities at a certain pH.  The accuracy of the method is tested against titrn. curves of single amino acids and a dipeptide in explicit solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIp1iVOhE1DLVg90H21EOLACvtfcHk0lgHPrD1Q6CiLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D&md5=0637c4d334ea58836db9e465401624c3</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fct200061r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200061r%26sid%3Dliteratum%253Aachs%26aulast%3DDonnini%26aufirst%3DS.%26aulast%3DTegeler%26aufirst%3DF.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DConstant%2520pH%2520molecular%2520dynamics%2520in%2520explicit%2520solvent%2520with%2520lambda-dynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D1962%26epage%3D1978%26doi%3D10.1021%2Fct200061r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Grossfield, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuckerman, D. M.</span><span> </span><span class="NLM_article-title">Quantifying uncertainty and sampling quality in biomolecular simulations</span> <span class="citation_source-journal">Annu. Rep. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">48</span><span class="refDoi"> DOI: 10.1016/S1574-1400(09)00502-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1574-1400%2809%2900502-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20454547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1aiu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=23-48&author=A.+Grossfieldauthor=D.+M.+Zuckerman&title=Quantifying+uncertainty+and+sampling+quality+in+biomolecular+simulations&doi=10.1016%2FS1574-1400%2809%2900502-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying uncertainty and sampling quality in biomolecular simulations</span></div><div class="casAuthors">Grossfield, Alan; Zuckerman, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-48</span>CODEN:
                <span class="NLM_cas:coden">ARCCC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-1400</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Growing computing capacity and algorithmic advances have facilitated the study of increasingly large biomol. systems at longer timescales.  However, with these larger, more complex systems come questions about the quality of sampling and statistical convergence.  What size systems can be sampled fully.  If a system is not fully sampled, can certain "fast variables" be considered well converged How can one det. the statistical significance of obsd. results.  The present review describes statistical tools and the underlying phys. ideas necessary to address these questions.  Basic definitions and ready-to-use analyses are provided, along with explicit recommendations.  Such statistical analyses are of paramount importance in establishing the reliability of simulation data in any given study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkELYsWb9OZLVg90H21EOLACvtfcHk0ljeBmxMiQkR2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1aiu7rM&md5=59be756cd4b6a617debb6da01b46fcc9</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2FS1574-1400%2809%2900502-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1574-1400%252809%252900502-7%26sid%3Dliteratum%253Aachs%26aulast%3DGrossfield%26aufirst%3DA.%26aulast%3DZuckerman%26aufirst%3DD.%2BM.%26atitle%3DQuantifying%2520uncertainty%2520and%2520sampling%2520quality%2520in%2520biomolecular%2520simulations%26jtitle%3DAnnu.%2520Rep.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D5%26spage%3D23%26epage%3D48%26doi%3D10.1016%2FS1574-1400%2809%2900502-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Given, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">1047</span><span class="NLM_x">–</span> <span class="NLM_lpage">1069</span><span class="refDoi"> DOI: 10.1016/S0006-3495(97)78756-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0006-3495%2897%2978756-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=9138555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1997&pages=1047-1069&author=M.+K.+Gilsonauthor=J.+A.+Givenauthor=B.+L.+Bushauthor=J.+A.+McCammon&title=The+statistical-thermodynamic+basis+for+computation+of+binding+affinities%3A+a+critical+review&doi=10.1016%2FS0006-3495%2897%2978756-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span></div><div class="casAuthors">Gilson, Michael K.; Given, James A.; Bush, Bruce L.; Mccammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1047-1069</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A review with many refs.  Although the statistical thermodn. of noncovalent binding has been considered in a no. of theor. papers, few methods of computing binding affinities are derived explicitly from this underlying theory.  This has contributed to uncertainty and controversy in certain areas.  This article therefore reviews and extends the connections of some important computational methods with the underlying statistical thermodn.  A derivation of the std. free energy of binding forms the basis of this review.  This derivation should be useful in formulating novel computational methods for predicting binding affinities.  It also permits several important points to be established.  For example, it is found that the double annihilation method of computing binding energy does not yield the std. free energy of binding, but can be modified to yield this quantity.  The derivation also makes it possible to define clearly the changes in translational, rotational, configurational, and solvent entropy upon binding.  It is argued that mol. mass has a negligible effect upon the std. free energy of binding for biomol. systems, and that the cratic entropy defined by Gurney is not a useful concept.  In addn., the use of continuum models of the solvent in binding calcns. is reviewed, and a formalism is presented for incorporating a limited no. of solvent mols. explicitly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZwKHjQ8gW7Vg90H21EOLACvtfcHk0ljeBmxMiQkR2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D&md5=2bf97b219b9d15fccab6cb50dc7686aa</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2897%2978756-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252897%252978756-3%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DGiven%26aufirst%3DJ.%2BA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520statistical-thermodynamic%2520basis%2520for%2520computation%2520of%2520binding%2520affinities%253A%2520a%2520critical%2520review%26jtitle%3DBiophys.%2520J.%26date%3D1997%26volume%3D72%26spage%3D1047%26epage%3D1069%26doi%3D10.1016%2FS0006-3495%2897%2978756-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">General, I. J.</span><span> </span><span class="NLM_article-title">A note on the standard state’s binding free energy</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2520</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span><span class="refDoi"> DOI: 10.1021/ct100255z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100255z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovFemsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2520-2524&author=I.+J.+General&title=A+note+on+the+standard+state%E2%80%99s+binding+free+energy&doi=10.1021%2Fct100255z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">A Note on the Standard State's Binding Free Energy</span></div><div class="casAuthors">General, Ignacio J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2520-2524</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The relation between the equil. const. of a given chem. reaction and the assocd. free energy is an issue well studied in chem. books, but when the reaction involves changes in the no. of components in the system, as is the case in binding, things become a little more obscure since one needs to define the so-called std. state.  This is reflected in the literature, esp. in computational studies of binding, where contradicting approaches are followed when treating this problem.  In this work, we present a detailed and unifying explanation of the concepts involved and derive the necessary relations to convert a binding free energy from an arbitrary state to some given std. state.  This is done in three independent ways, from the point of view of (1) the dimensions of the quantities involved, (2) the energy and entropy of the mols., and (3) their chem. potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDFgcYRUjt_bVg90H21EOLACvtfcHk0ljeBmxMiQkR2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovFemsrw%253D&md5=90323fe06cf72991d4e521f0c70c8875</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fct100255z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100255z%26sid%3Dliteratum%253Aachs%26aulast%3DGeneral%26aufirst%3DI.%2BJ.%26atitle%3DA%2520note%2520on%2520the%2520standard%2520state%25E2%2580%2599s%2520binding%2520free%2520energy%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2520%26epage%3D2524%26doi%3D10.1021%2Fct100255z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">De Jong, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vries, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrink, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Determining equilibrium constants for dimerization reactions from molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">1919</span><span class="NLM_x">–</span> <span class="NLM_lpage">1928</span><span class="refDoi"> DOI: 10.1002/jcc.21776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.21776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21469160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVGnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=1919-1928&author=D.+H.+De+Jongauthor=L.+V.+Schaferauthor=A.+H.+De+Vriesauthor=S.+J.+Marrinkauthor=H.+J.+C.+Berendsenauthor=H.+Grubmuller&title=Determining+equilibrium+constants+for+dimerization+reactions+from+molecular+dynamics+simulations&doi=10.1002%2Fjcc.21776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Determining equilibrium constants for dimerization reactions from molecular dynamics simulations</span></div><div class="casAuthors">De Jong, Djurre H.; Schaefer, Lars V.; De Vries, Alex H.; Marrink, Siewert J.; Berendsen, Herman J. C.; Grubmueller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1919-1928</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">With today's available computer power, free energy calcns. from equil. mol. dynamics simulations "via counting" become feasible for an increasing no. of reactions.  An example is the dimerization reaction of transmembrane alpha-helixes.  If an extended simulation of the two helixes covers sufficiently many dimerization and dissocn. events, their binding free energy is readily derived from the fraction of time during which the two helixes are obsd. in dimeric form.  Exactly how the correct value for the free energy is to be calcd., however, is unclear, and indeed several different and contradictory approaches have been used.  In particular, results obtained via Boltzmann statistics differ from those detd. via the law of mass action.  Here, the authors develop a theory that resolves this discrepancy.  The authors show that for simulation systems contg. two mols., the dimerization free energy is given by a formula of the form ΔG .varies. ln(P1/P0).  The authors' theory is also applicable to high concns. that typically have to be used in mol. dynamics simulations to keep the simulation system small, where the textbook dil. approxns. fail.  It also covers simulations with an arbitrary no. of monomers and dimers and provides rigorous error ests.  Comparison with test simulations of a simple Lennard Jones system with various particle nos. as well as with ref. free energy values obtained from radial distribution functions show full agreement for both binding free energies and dimerization statistics.  © 2011 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou9ZX2izyj9LVg90H21EOLACvtfcHk0liOGGSlJI7e2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVGnurk%253D&md5=4197a85271c389d786af2a286dd6d129</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21776%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BJong%26aufirst%3DD.%2BH.%26aulast%3DSchafer%26aufirst%3DL.%2BV.%26aulast%3DDe%2BVries%26aufirst%3DA.%2BH.%26aulast%3DMarrink%26aufirst%3DS.%2BJ.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DDetermining%2520equilibrium%2520constants%2520for%2520dimerization%2520reactions%2520from%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D1919%26epage%3D1928%26doi%3D10.1002%2Fjcc.21776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-2371&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0liOGGSlJI7e2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Bakan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lakdawala, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahar, I.</span><span> </span><span class="NLM_article-title">Druggability assessment of allosteric proteins by dynamics simulations in the presence of probe molecules</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2435</span><span class="NLM_x">–</span> <span class="NLM_lpage">2447</span><span class="refDoi"> DOI: 10.1021/ct300117j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300117j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XotFGlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2435-2447&author=A.+Bakanauthor=N.+Nevinsauthor=A.+S.+Lakdawalaauthor=I.+Bahar&title=Druggability+assessment+of+allosteric+proteins+by+dynamics+simulations+in+the+presence+of+probe+molecules&doi=10.1021%2Fct300117j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Assessment of Allosteric Proteins by Dynamics Simulations in the Presence of Probe Molecules</span></div><div class="casAuthors">Bakan, Ahmet; Nevins, Neysa; Lakdawala, Ami S.; Bahar, Ivet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2435-2447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Druggability assessment of a target protein has emerged in recent years as an important concept in hit-to-lead optimization.  A reliable and phys. relevant measure of druggability would allow informed decisions on the risk of investing in a particular target.  Here, we define "druggability" as a quant. est. of binding sites and affinities for a potential drug acting on a specific protein target.  In the present study, we describe a new methodol. that successfully predicts the druggability and maximal binding affinity for a series of challenging targets, including those that function through allosteric mechanisms.  Two distinguishing features of the methodol. are (i) simulation of the binding dynamics of a diversity of probe mols. selected on the basis of an anal. of approved drugs and (ii) identification of druggable sites and estn. of corresponding binding affinities on the basis of an evaluation of the geometry and energetics of bound probe clusters.  The use of the methodol. for a variety of targets such as murine double mutant-2, protein tyrosine phosphatase 1B (PTP1B), lymphocyte function-assocd. antigen 1, vertebrate kinesin-5 (Eg5), and p38 mitogen-activated protein kinase provides examples for which the method correctly captures the location and binding affinities of known drugs.  It also provides insights into novel druggable sites and the target's structural changes that would accommodate, if not promote and stabilize, drug binding.  Notably, the ability to identify high affinity spots even in challenging cases such as PTP1B or Eg5 shows promise as a rational tool for assessing the druggability of protein targets and identifying allosteric or novel sites for drug binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgPQ7A70lFJLVg90H21EOLACvtfcHk0liOGGSlJI7e2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotFGlsLo%253D&md5=328391521ec9e1c32ec8448546f1d3ea</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fct300117j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300117j%26sid%3Dliteratum%253Aachs%26aulast%3DBakan%26aufirst%3DA.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DBahar%26aufirst%3DI.%26atitle%3DDruggability%2520assessment%2520of%2520allosteric%2520proteins%2520by%2520dynamics%2520simulations%2520in%2520the%2520presence%2520of%2520probe%2520molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2435%26epage%3D2447%26doi%3D10.1021%2Fct300117j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckner, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boudon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiradorives, J.</span><span> </span><span class="NLM_article-title">Efficient computation of absolute free-energies of binding by computer-simulations - application to the methane dimer in water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">3742</span><span class="NLM_x">–</span> <span class="NLM_lpage">3746</span><span class="refDoi"> DOI: 10.1063/1.454895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.454895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL1cXlslOkt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1988&pages=3742-3746&author=W.+L.+Jorgensenauthor=J.+K.+Bucknerauthor=S.+Boudonauthor=J.+Tiradorives&title=Efficient+computation+of+absolute+free-energies+of+binding+by+computer-simulations+-+application+to+the+methane+dimer+in+water&doi=10.1063%2F1.454895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient computation of absolute free energies of binding by computer simulations.  Application to the methane dimer in water</span></div><div class="casAuthors">Jorgensen, William L.; Buckner, J. Kathleen; Boudon, Stephane; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3742-6</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An efficient procedure is noted for computing abs. free energies of binding for complexes in soln.  Two series of computer simulations are required in which the substrate is annihilated in the solvent by itself and in the solvated complex.  For illustration, the free energy of binding for two methane-like particles at their contact sepn. of 4 Å was computed in TIP4P water.  Though several alternatives are possible, in this case, Monte Carlo simulations were employed with statistical perturbation theory in the NPT ensemble at 25° and 1 atm.  The results for the free energy of binding as well as for the potential of mean force are consistent with prior findings from the integral equation theory of Pratt and Chandler.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfqADFbgq60LVg90H21EOLACvtfcHk0liOGGSlJI7e2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXlslOkt7g%253D&md5=29dc47048d4a91d6aea0e73c16fcaff8</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1063%2F1.454895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.454895%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBuckner%26aufirst%3DJ.%2BK.%26aulast%3DBoudon%26aufirst%3DS.%26aulast%3DTiradorives%26aufirst%3DJ.%26atitle%3DEfficient%2520computation%2520of%2520absolute%2520free-energies%2520of%2520binding%2520by%2520computer-simulations%2520-%2520application%2520to%2520the%2520methane%2520dimer%2520in%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1988%26volume%3D89%26spage%3D3742%26epage%3D3746%26doi%3D10.1063%2F1.454895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Hermans, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span> </span><span class="NLM_article-title">Inclusion of loss of translational and rotational freedom in theoretical estimates of free energies of binding. Application to a complex of benzene and mutant T4 lysozyme</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">2707</span><span class="NLM_x">–</span> <span class="NLM_lpage">2714</span><span class="refDoi"> DOI: 10.1021/ja963568+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja963568%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXhvV2qtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=2707-2714&author=J.+Hermansauthor=L.+Wang&title=Inclusion+of+loss+of+translational+and+rotational+freedom+in+theoretical+estimates+of+free+energies+of+binding.+Application+to+a+complex+of+benzene+and+mutant+T4+lysozyme&doi=10.1021%2Fja963568%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Inclusion of Loss of Translational and Rotational Freedom in Theoretical Estimates of Free Energies of Binding. Application to a Complex of Benzene and Mutant T4 Lysozyme</span></div><div class="casAuthors">Hermans, Jan; Wang, Lu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2707-2714</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the first complete treatment for calcg. theor. ests. of free energies of formation of macromol.-ligand complexes with mol. dynamics simulations, as the free energy for transforming the ligand into a non-interacting state by gradually diminishing the forces between macromol. (plus solvent) and ligand.  The calcns. become possible due to the introduction of a specially designed potential ("mol. tweezers") which restrains the spatial position and orientation of the ligand mol. and is gradually applied as the transformation proceeds from complexed to non-interacting components.  The binding of benzene to a mutant T4 lysozyme (Morton et al. Biochem. 1995, 34, 8564-8575) has been used as a test case.  The simulations reproduce the value of the free energy of binding (-5.19 kcal/mol if the std. state of benzene is a 1 M aq. soln.) within the sum of exptl. and statistical error.  Another series of such simulations with rigid protein models provides an est. of the dependence of the free energy of binding on the protein conformation.  The free energy of binding is found to decrease in the series: energy-minimized ligand-free protein (-3.5 kcal/mol), energy-minimized ligand-contg. protein (-6.3 kcal/mol), and crystal structure (-8.5 kcal/mol).  The free energy of binding to a series of snapshots from a protein-ligand dynamics trajectory varies between -7 and -9 kcal/mol.  The "cratic" free energy contribution, which corresponds to the loss of translational and rotational freedom of the ligand mol., was estd. at 7 kcal/mol.  It has proved possible to decomp. this into translational and rotational components and, from these free energies, est. the remaining freedom of the benzene in the binding pocket, at 0.6 Å for positional range and 10-15° for angular range, in excellent agreement with the motion obsd. in a dynamics trajectory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1E9cnzjwo4rVg90H21EOLACvtfcHk0ljCH-QsSD4a1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvV2qtbg%253D&md5=1af691d300438687118b5ed434cb5180</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fja963568%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja963568%252B%26sid%3Dliteratum%253Aachs%26aulast%3DHermans%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DInclusion%2520of%2520loss%2520of%2520translational%2520and%2520rotational%2520freedom%2520in%2520theoretical%2520estimates%2520of%2520free%2520energies%2520of%2520binding.%2520Application%2520to%2520a%2520complex%2520of%2520benzene%2520and%2520mutant%2520T4%2520lysozyme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D2707%26epage%3D2714%26doi%3D10.1021%2Fja963568%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Hamelberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Standard free energy of releasing a localized water molecule from the binding pockets of proteins: Double-decoupling method</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">7683</span><span class="NLM_x">–</span> <span class="NLM_lpage">7689</span><span class="refDoi"> DOI: 10.1021/ja0377908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0377908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVejtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=7683-7689&author=D.+Hamelbergauthor=J.+A.+McCammon&title=Standard+free+energy+of+releasing+a+localized+water+molecule+from+the+binding+pockets+of+proteins%3A+Double-decoupling+method&doi=10.1021%2Fja0377908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Standard Free Energy of Releasing a Localized Water Molecule from the Binding Pockets of Proteins: Double-Decoupling Method</span></div><div class="casAuthors">Hamelberg, Donald; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7683-7689</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Localized water mols. in the binding pockets of proteins play an important role in noncovalent assocn. of proteins and small drug compds.  At times, the dominant contribution to the binding free energy comes from the release of localized water mols. in the binding pockets of biomols.  Therefore, to quantify the energetic importance of these water mols. for drug design purposes, the authors have used the double-decoupling approach to calc. the std. free energy of tying up a water mol. in the binding pockets of two protein complexes.  The double-decoupling approach is based on the underlying principle of statistical thermodn.  The authors have calcd. the std. free energies of tying up the water mol. in the binding pockets of these complexes to be favorable.  These water mols. stabilize the protein-drug complexes by interacting with the ligands and binding pockets.  The results offer ideas that could be used in optimizing protein-drug interactions, by designing ligands that are capable of targeting localized water mols. in protein binding sites.  The resulting free energy of ligand binding could benefit from the potential free energy gain accompanying the release of these water mols.  Furthermore, the authors have examd. the theor. background of the double-decoupling method and its connection to the mol. dynamics thermodn. integration techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsMbAJlCZuqrVg90H21EOLACvtfcHk0ljCH-QsSD4a1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVejtb0%253D&md5=a0949ecffa5cfddf1f2128b1660d04f7</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Fja0377908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0377908%26sid%3Dliteratum%253Aachs%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DStandard%2520free%2520energy%2520of%2520releasing%2520a%2520localized%2520water%2520molecule%2520from%2520the%2520binding%2520pockets%2520of%2520proteins%253A%2520Double-decoupling%2520method%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D7683%26epage%3D7689%26doi%3D10.1021%2Fja0377908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Wang, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2814</span><span class="refDoi"> DOI: 10.1529/biophysj.106.084301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1529%2Fbiophysj.106.084301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16844742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2006&pages=2798-2814&author=J.+Y.+Wangauthor=Y.+Q.+Dengauthor=B.+Roux&title=Absolute+binding+free+energy+calculations+using+molecular+dynamics+simulations+with+restraining+potentials&doi=10.1529%2Fbiophysj.106.084301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span></div><div class="casAuthors">Wang, Jiyao; Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2798-2814</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The abs. (std.) binding free energy of eight FK506-related ligands to FKBP12 is calcd. using free energy perturbation mol. dynamics (FEP/MD) simulations with explicit solvent.  A no. of features are implemented to improve the accuracy and enhance the convergence of the calcns.  First, the abs. binding free energy is decompd. into sequential steps during which the ligand-surrounding interactions as well as various biasing potentials restraining the translation, orientation, and conformation of the ligand are turned "on" and "off.".  Second, sampling of the ligand conformation is enforced by a restraining potential based on the root mean-square deviation relative to the bound state conformation.  The effect of all the restraining potentials is rigorously unbiased, and it is shown explicitly that the final results are independent of all artificial restraints.  Third, the repulsive and dispersive free energy contribution arising from the Lennard-Jones interactions of the ligand with its surrounding (protein and solvent) is calcd. using the Weeks-Chandler-Andersen sepn.  This sepn. also improves convergence of the FEP/MD calcns.  Fourth, to decrease the computational cost, only a small no. of atoms in the vicinity of the binding site are simulated explicitly, while all the influence of the remaining atoms is incorporated implicitly using the generalized solvent boundary potential (GSBP) method.  With GSBP, the size of the simulated FKBP12/ligand systems is significantly reduced, from ∼25,000 to 2500.  The computations are very efficient and the statistical error is small (∼1 kcal/mol).  The calcd. binding free energies are generally in good agreement with available exptl. data and previous calcns. (within ∼2 kcal/mol).  The present results indicate that a strategy based on FEP/MD simulations of a reduced GSBP at. model sampled with conformational, translational, and orientational restraining potentials can be computationally inexpensive and accurate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsDftyP7DXobVg90H21EOLACvtfcHk0ljCH-QsSD4a1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP&md5=010f74f61048d03fee5dce41172dac70</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.084301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.084301%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BY.%26aulast%3DDeng%26aufirst%3DY.%2BQ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculations%2520using%2520molecular%2520dynamics%2520simulations%2520with%2520restraining%2520potentials%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D91%26spage%3D2798%26epage%3D2814%26doi%3D10.1529%2Fbiophysj.106.084301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Helms, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R. C.</span><span> </span><span class="NLM_article-title">Computational alchemy to calculate absolute protein-ligand binding free energy</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">2710</span><span class="NLM_x">–</span> <span class="NLM_lpage">2713</span><span class="refDoi"> DOI: 10.1021/ja9738539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9738539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1cXis12itr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=2710-2713&author=V.+Helmsauthor=R.+C.+Wade&title=Computational+alchemy+to+calculate+absolute+protein-ligand+binding+free+energy&doi=10.1021%2Fja9738539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Alchemy To Calculate Absolute Protein-Ligand Binding Free Energy</span></div><div class="casAuthors">Helms, Volkhard; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2710-2713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability to reliably compute accurate protein-ligand binding affinities is crucial to understanding protein-ligand recognition and to structure-based drug design.  A ligand's binding affinity is specified by its abs. binding free energy, ΔGbind, the free energy difference between the bound and unbound states.  To compute accurate free energy differences by free energy perturbation (FEP), "alchem." rather than phys. processes are usually simulated by mol. dynamics simulations so as to minimize the perturbation to the system.  Here, we report a novel "alchemistic" application of the FEP methodol. involving a large perturbation.  By mutating a ligand with 11 non-hydrogen atoms into six water mols. in the binding site of a protein, we computed a ΔGbind within 3 kJ/mol of the exptl. value.  This is the first successful example of the computation of ΔGbind for a protein:ligand pair with full treatment of the solvent degrees of freedom.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcFHdnjc43ZrVg90H21EOLACvtfcHk0lhpPFjVfnB9mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis12itr0%253D&md5=ec7b248dd479a1538ae74f89561bc19e</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fja9738539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9738539%26sid%3Dliteratum%253Aachs%26aulast%3DHelms%26aufirst%3DV.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DComputational%2520alchemy%2520to%2520calculate%2520absolute%2520protein-ligand%2520binding%2520free%2520energy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D2710%26epage%3D2713%26doi%3D10.1021%2Fja9738539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Woods, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malaisree, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannongbua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, A. J.</span><span> </span><span class="NLM_article-title">A water-swap reaction coordinate for the calculation of absolute protein-ligand binding free energies</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">054114</span><span class="refDoi"> DOI: 10.1063/1.3519057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.3519057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21303099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2011&pages=054114&author=C.+J.+Woodsauthor=M.+Malaisreeauthor=S.+Hannongbuaauthor=A.+J.+Mulholland&title=A+water-swap+reaction+coordinate+for+the+calculation+of+absolute+protein-ligand+binding+free+energies&doi=10.1063%2F1.3519057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">A water-swap reaction coordinate for the calculation of absolute protein-ligand binding free energies</span></div><div class="casAuthors">Woods, Christopher J.; Malaisree, Maturos; Hannongbua, Supot; Mulholland, Adrian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">054114/1-054114/13</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The accurate prediction of abs. protein-ligand binding free energies is one of the grand challenge problems of computational science.  Binding free energy measures the strength of binding between a ligand and a protein, and an algorithm that would allow its accurate prediction would be a powerful tool for rational drug design.  Here we present the development of a new method that allows for the abs. binding free energy of a protein-ligand complex to be calcd. from first principles, using a single simulation.  Our method involves the use of a novel reaction coordinate that swaps a ligand bound to a protein with an equiv. vol. of bulk water.  This water-swap reaction coordinate is built using an identity constraint, which identifies a cluster of water mols. from bulk water that occupies the same vol. as the ligand in the protein active site.  A dual topol. algorithm is then used to swap the ligand from the active site with the identified water cluster from bulk water.  The free energy is then calcd. using replica exchange thermodn. integration.  This returns the free energy change of simultaneously transferring the ligand to bulk water, as an equiv. vol. of bulk water is transferred back to the protein active site.  This, directly, is the abs. binding free energy.  It should be noted that while this reaction coordinate models the binding process directly, an accurate force field and sufficient sampling are still required to allow for the binding free energy to be predicted correctly.  In this paper we present the details and development of this method, and demonstrate how the potential of mean force along the water-swap coordinate can be improved by calibrating the soft-core Coulomb and Lennard-Jones parameters used for the dual topol. calcn.  The optimal parameters were applied to calcns. of protein-ligand binding free energies of a neuraminidase inhibitor (oseltamivir), with these results compared to expt.  These results demonstrate that the water-swap coordinate provides a viable and potentially powerful new route for the prediction of protein-ligand binding free energies.  (c) 2011 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGYbA6KMDz7Vg90H21EOLACvtfcHk0lhpPFjVfnB9mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSqsbo%253D&md5=a83aaffa025f46f3f4248d792bfc0d31</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1063%2F1.3519057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3519057%26sid%3Dliteratum%253Aachs%26aulast%3DWoods%26aufirst%3DC.%2BJ.%26aulast%3DMalaisree%26aufirst%3DM.%26aulast%3DHannongbua%26aufirst%3DS.%26aulast%3DMulholland%26aufirst%3DA.%2BJ.%26atitle%3DA%2520water-swap%2520reaction%2520coordinate%2520for%2520the%2520calculation%2520of%2520absolute%2520protein-ligand%2520binding%2520free%2520energies%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2011%26volume%3D134%26spage%3D054114%26doi%3D10.1063%2F1.3519057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">Rocchia, W.; Masetti, M.; Cavalli, A.</span><span> </span><span class="NLM_article-title">Enhanced sampling methods in drug design</span>. In  <span class="citation_source-book">Physico-Chemical and Computational Approaches to Drug Discovery</span>; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>Chapter 11, pp  <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span>, DOI: <span class="refDoi"> DOI: 10.1039/9781849735377-00273</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1039%2F9781849735377-00273" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=273-301&author=W.+Rocchia&author=M.+Masetti&author=A.+Cavalli&title=Physico-Chemical+and+Computational+Approaches+to+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1039%2F9781849735377-00273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849735377-00273%26sid%3Dliteratum%253Aachs%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DEnhanced%2520sampling%2520methods%2520in%2520drug%2520design%26btitle%3DPhysico-Chemical%2520and%2520Computational%2520Approaches%2520to%2520Drug%2520Discovery%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2012%26spage%3D273%26epage%3D301%26doi%3D10.1039%2F9781849735377-00273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Liu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span> </span><span class="NLM_article-title">Replica exchange with solute tempering: A method for sampling biological systems in explicit water</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">13749</span><span class="NLM_x">–</span> <span class="NLM_lpage">13754</span><span class="refDoi"> DOI: 10.1073/pnas.0506346102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0506346102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16172406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVOqsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=13749-13754&author=P.+Liuauthor=B.+Kimauthor=R.+A.+Friesnerauthor=B.+J.+Berne&title=Replica+exchange+with+solute+tempering%3A+A+method+for+sampling+biological+systems+in+explicit+water&doi=10.1073%2Fpnas.0506346102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Replica exchange with solute tempering: A method for sampling biological systems in explicit water</span></div><div class="casAuthors">Liu, Pu; Kim, Byungchan; Friesner, Richard A.; Berne, B. J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">13749-13754</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">An innovative replica exchange (parallel tempering) method called replica exchange with solute tempering (REST) for the efficient sampling of aq. protein solns. is presented here.  The method bypasses the poor scaling with system size of std. replica exchange and thus reduces the no. of replicas (parallel processes) that must be used.  This redn. is accomplished by deforming the Hamiltonian function for each replica in such a way that the acceptance probability for the exchange of replica configurations does not depend on the no. of explicit water mols. in the system.  For proof of concept, REST is compared with std. replica exchange for an alanine dipeptide mol. in water.  The comparisons confirm that REST greatly reduces the no. of CPUs required by regular replica exchange and increases the sampling efficiency.  This method reduces the CPU time required for calcg. thermodn. avs. and for the ab initio folding of proteins in explicit water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDjppdHbYZJLVg90H21EOLACvtfcHk0lhpPFjVfnB9mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVOqsb3P&md5=7ebecadb3ad2037a35bdf92a361e5daf</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506346102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506346102%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26atitle%3DReplica%2520exchange%2520with%2520solute%2520tempering%253A%2520A%2520method%2520for%2520sampling%2520biological%2520systems%2520in%2520explicit%2520water%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D13749%26epage%3D13754%26doi%3D10.1073%2Fpnas.0506346102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Sinko, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Oliveira, C. A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Population based reweighting of scaled molecular dynamics</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">12759</span><span class="NLM_x">–</span> <span class="NLM_lpage">12768</span><span class="refDoi"> DOI: 10.1021/jp401587e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp401587e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVOhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=12759-12768&author=W.+Sinkoauthor=Y.+L.+Miaoauthor=C.+A.+F.+de+Oliveiraauthor=J.+A.+McCammon&title=Population+based+reweighting+of+scaled+molecular+dynamics&doi=10.1021%2Fjp401587e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Population Based Reweighting of Scaled Molecular Dynamics</span></div><div class="casAuthors">Sinko, William; Miao, Yinglong; de Oliveira, Cesar Augusto F.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">12759-12768</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics simulation using enhanced sampling methods is one of the powerful computational tools used to explore protein conformations and free energy landscapes.  Enhanced sampling methods often employ either an increase in temp. or a flattening of the potential energy surface to rapidly sample phase space, and a corresponding reweighting algorithm is used to recover the Boltzmann statistics.  However, potential energies of complex biomols. usually involve large fluctuations on a magnitude of hundreds of kcal/mol despite minimal structural changes during simulation.  This leads to noisy reweighting statistics and complicates the obtainment of accurate final results.  To overcome this common issue in enhanced conformational sampling, we propose a scaled mol. dynamics method, which modifies the biomol. potential energy surface and employs a reweighting scheme based on configurational populations.  Statistical mech. theory is applied to derive the reweighting formula, and the canonical ensemble of simulated structures is recovered accordingly.  Test simulations on alanine dipeptide and the fast folding polypeptide Chignolin exhibit sufficiently enhanced conformational sampling and accurate recovery of free energy surfaces and thermodn. properties.  The results are comparable to long conventional mol. dynamics simulations and exhibit better recovery of canonical statistics over methods which employ a potential energy term in reweighting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogoBM_fh_axLVg90H21EOLACvtfcHk0lgIsUplxNwWMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVOhtrs%253D&md5=99939d7a061f0eb25339b626c1fb7a54</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Fjp401587e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp401587e%26sid%3Dliteratum%253Aachs%26aulast%3DSinko%26aufirst%3DW.%26aulast%3DMiao%26aufirst%3DY.%2BL.%26aulast%3Dde%2BOliveira%26aufirst%3DC.%2BA.%2BF.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DPopulation%2520based%2520reweighting%2520of%2520scaled%2520molecular%2520dynamics%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D12759%26epage%3D12768%26doi%3D10.1021%2Fjp401587e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span> </span><span class="NLM_article-title">Correction to “Replica exchange with solute scaling: a more efficient version of replica exchange with solute tempering (REST2)”</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">11305</span><span class="NLM_x">–</span> <span class="NLM_lpage">11305</span><span class="refDoi"> DOI: 10.1021/jp207931v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp207931v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFSmtr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2011&pages=11305-11305&author=L.+Wangauthor=R.+A.+Friesnerauthor=B.+J.+Berne&title=Correction+to+%E2%80%9CReplica+exchange+with+solute+scaling%3A+a+more+efficient+version+of+replica+exchange+with+solute+tempering+%28REST2%29%E2%80%9D&doi=10.1021%2Fjp207931v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Replica Exchange with Solute Scaling: A More Efficient Version of Replica Exchange with Solute Tempering (REST2) [Erratum to document cited in CA155:147308]</span></div><div class="casAuthors">Wang, Lingle; Friesner, Richard A.; Berne, B. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11305</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">On page 9437, information was omitted from the Acknowledgments section; the omitted information is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq31kSV7snPlLVg90H21EOLACvtfcHk0lgIsUplxNwWMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFSmtr%252FI&md5=f973782faa627fb89e637b07f29286b2</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fjp207931v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp207931v%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26atitle%3DCorrection%2520to%2520%25E2%2580%259CReplica%2520exchange%2520with%2520solute%2520scaling%253A%2520a%2520more%2520efficient%2520version%2520of%2520replica%2520exchange%2520with%2520solute%2520tempering%2520%2528REST2%2529%25E2%2580%259D%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2011%26volume%3D115%26spage%3D11305%26epage%3D11305%26doi%3D10.1021%2Fjp207931v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">On achieving high accuracy and reliability in the calculation of relative protein-ligand binding affinities</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">1937</span><span class="NLM_x">–</span> <span class="NLM_lpage">1942</span><span class="refDoi"> DOI: 10.1073/pnas.1114017109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1114017109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22308365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVSht74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=1937-1942&author=L.+Wangauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=On+achieving+high+accuracy+and+reliability+in+the+calculation+of+relative+protein-ligand+binding+affinities&doi=10.1073%2Fpnas.1114017109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">On achieving high accuracy and reliability in the calculation of relative protein-ligand binding affinities</span></div><div class="casAuthors">Wang, Lingle; Berne, B. J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1937-1942, S1937/1-S1937/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We apply a free energy perturbation simulation method, free energy perturbation/replica exchange with solute tempering, to two modifications of protein-ligand complexes that lead to significant conformational changes, the first in the protein and the second in the ligand.  The approach is shown to facilitate sampling in these challenging cases where high free energy barriers sep. the initial and final conformations and leads to superior convergence of the free energy as demonstrated both by consistency of the results (independence from the starting conformation) and agreement with exptl. binding affinity data.  The second case, consisting of two neutral thrombin ligands that are taken from a recent medicinal chem. program for this interesting pharmaceutical target, is of particular significance in that it demonstrates that good results can be obtained for large, complex ligands, as opposed to relatively simple model systems.  To achieve quant. agreement with expt. in the thrombin case, a next generation force field, Optimized Potentials for Liq. Simulations 2.0, is required, which provides superior charges and torsional parameters as compared to earlier alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6CESSaIXBJbVg90H21EOLACvtfcHk0lgIsUplxNwWMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVSht74%253D&md5=23db7d18e624d5e1214bbd6702619a87</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1114017109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1114017109%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOn%2520achieving%2520high%2520accuracy%2520and%2520reliability%2520in%2520the%2520calculation%2520of%2520relative%2520protein-ligand%2520binding%2520affinities%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D1937%26epage%3D1942%26doi%3D10.1073%2Fpnas.1114017109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Lionta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spyrou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassilatis, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournia, Z.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening for drug discovery: principles, applications and recent advances</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1923</span><span class="NLM_x">–</span> <span class="NLM_lpage">1938</span><span class="refDoi"> DOI: 10.2174/1568026614666140929124445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F1568026614666140929124445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25262799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1923-1938&author=E.+Liontaauthor=G.+Spyrouauthor=D.+K.+Vassilatisauthor=Z.+Cournia&title=Structure-based+virtual+screening+for+drug+discovery%3A+principles%2C+applications+and+recent+advances&doi=10.2174%2F1568026614666140929124445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Virtual Screening for Drug Discovery: Principles, Applications and Recent Advances</span></div><div class="casAuthors">Lionta, Evanthia; Spyrou, George; Vassilatis, Demetrios K.; Cournia, Zoe</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1923-1938</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug discovery (SBDD) is becoming an essential tool in assisting fast and cost-efficient lead discovery and optimization.  The application of rational, structure-based drug design is proven to be more efficient than the traditional way of drug discovery since it aims to understand the mol. basis of a disease and utilizes the knowledge of the three-dimensional structure of the biol. target in the process.  In this review, we focus on the principles and applications of Virtual Screening (VS) within the context of SBDD and examine different procedures ranging from the initial stages of the process that include receptor and library pre-processing, to docking, scoring and post-processing of top scoring hits.  Recent improvements in structure-based virtual screening (SBVS) efficiency through ensemble docking, induced fit and consensus docking are also discussed.  The review highlights advances in the field within the framework of several success studies that have led to nM inhibition directly from VS and provides recent trends in library design as well as discusses limitations of the method.  Applications of SBVS in the design of substrates for engineered proteins that enable the discovery of new metabolic and signal transduction pathways and the design of inhibitors of multifunctional proteins are also reviewed.  Finally, we contribute two promising VS protocols recently developed by us that aim to increase inhibitor selectivity.  In the first protocol, we describe the discovery of micromolar inhibitors through SBVS designed to inhibit the mutant H1047R PI3Kα kinase.  Second, we discuss a strategy for the identification of selective binders for the RXRα nuclear receptor.  In this protocol, a set of target structures is constructed for ensemble docking based on binding site shape characterization and clustering, aiming to enhance the hit rate of selective inhibitors for the desired protein target through the SBVS process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps80ptfisjzLVg90H21EOLACvtfcHk0lgIsUplxNwWMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ&md5=0550a396dd011adf52699b091548345b</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.2174%2F1568026614666140929124445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026614666140929124445%26sid%3Dliteratum%253Aachs%26aulast%3DLionta%26aufirst%3DE.%26aulast%3DSpyrou%26aufirst%3DG.%26aulast%3DVassilatis%26aufirst%3DD.%2BK.%26aulast%3DCournia%26aufirst%3DZ.%26atitle%3DStructure-based%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520principles%252C%2520applications%2520and%2520recent%2520advances%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2014%26volume%3D14%26spage%3D1923%26epage%3D1938%26doi%3D10.2174%2F1568026614666140929124445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Matter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotriffer, C.</span><span> </span><span class="NLM_article-title">Applications and success stories in virtual screening</span> <span class="citation_source-journal">Virtual screening: principles, challenges, and practical guidelines</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">358</span><span class="refDoi"> DOI: 10.1002/9783527633326.ch12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2F9783527633326.ch12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2011&pages=319-358&author=H.+Matterauthor=C.+Sotriffer&title=Applications+and+success+stories+in+virtual+screening&doi=10.1002%2F9783527633326.ch12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2F9783527633326.ch12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9783527633326.ch12%26sid%3Dliteratum%253Aachs%26aulast%3DMatter%26aufirst%3DH.%26aulast%3DSotriffer%26aufirst%3DC.%26atitle%3DApplications%2520and%2520success%2520stories%2520in%2520virtual%2520screening%26jtitle%3DVirtual%2520screening%253A%2520principles%252C%2520challenges%252C%2520and%2520practical%2520guidelines%26date%3D2011%26volume%3D48%26spage%3D319%26epage%3D358%26doi%3D10.1002%2F9783527633326.ch12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and scoring in virtual screening for drug discovery: methods and applications</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">949</span><span class="refDoi"> DOI: 10.1038/nrd1549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd1549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15520816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+scoring+in+virtual+screening+for+drug+discovery%3A+methods+and+applications&doi=10.1038%2Fnrd1549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring in virtual screening for drug discovery: methods and applications</span></div><div class="casAuthors">Kitchen, Douglas B.; Decornez, Helene; Furr, John R.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">935-949</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Computational approaches that 'dock' small mols. into the structures of macromol. targets and 'score' their potential complementarity to binding sites are widely used in hit identification and lead optimization.  Indeed, there are now a no. of drugs whose development was heavily influenced by or based on structure-based design and screening strategies, such as HIV protease inhibitors.  Nevertheless, there remain significant challenges in the application of these approaches, in particular in relation to current scoring schemes.  Here, we review key concepts and specific features of small-mol.-protein docking methods, highlight selected applications and discuss recent advances that aim to address the acknowledged limitations of established approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMLibgFe3mfLVg90H21EOLACvtfcHk0lgKgy5NkwDUMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D&md5=875a2b37a4299181509a1922b11dbd2f</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fnrd1549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1549%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520scoring%2520in%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520methods%2520and%2520applications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949%26doi%3D10.1038%2Fnrd1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Jiang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span> </span><span class="NLM_article-title">Soft Docking - Matching of molecular-surface cubes</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">219</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span><span class="refDoi"> DOI: 10.1016/0022-2836(91)90859-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0022-2836%2891%2990859-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=219&publication_year=1991&pages=79-102&author=F.+Jiangauthor=S.+H.+Kim&title=Soft+Docking+-+Matching+of+molecular-surface+cubes&doi=10.1016%2F0022-2836%2891%2990859-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2891%2990859-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252891%252990859-5%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DS.%2BH.%26atitle%3DSoft%2520Docking%2520-%2520Matching%2520of%2520molecular-surface%2520cubes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1991%26volume%3D219%26spage%3D79%26epage%3D102%26doi%3D10.1016%2F0022-2836%2891%2990859-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Leach, A. R.</span><span> </span><span class="NLM_article-title">Ligand docking to proteins with discrete side-chain flexibility</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1016/S0022-2836(05)80038-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0022-2836%2805%2980038-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=8289255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2cXhvVKlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1994&pages=345-356&author=A.+R.+Leach&title=Ligand+docking+to+proteins+with+discrete+side-chain+flexibility&doi=10.1016%2FS0022-2836%2805%2980038-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand docking to proteins with discrete side-chain flexibility</span></div><div class="casAuthors">Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">345-56</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">An algorithm is described that explores the conformational degrees of freedom of the amino acid side-chains and of the ligand when docking a putative ligand into a receptor site.  For a given orientation of the ligand relative to the protein, the method can find the lowest energy combination of amino acid side-chains and ligand conformations as well as all other combinations in order of increasing energy within a specified energy cutoff.  The amino acid side-chains and the ligand are restricted discrete low-energy conformations, detd. for the former by analyzing high-resoln. protein structures and in the latter case from a conformational anal.  Coupled to an algorithm for exploring the six degrees of orientational freedom of the ligand with respect to the receptor, the method can be used to perform conformationally flexible ligand docking.  A combination of two search methods is employed to explore the conformational degrees of freedom: Dead End Elimination and the A* algorithm.  When no ligand is present the approach can be used to predict the lowest energy combinations of side-chain conformations for a given protein backbone structure.  The approach is employed to illustrate how such a procedure can be used to est. the conformational entropy change that accompanies the formation of an intermol. complex between a protein and a ligand and to demonstrate that the protein's conformational entropy may in some cases increase on binding the ligand.  This is due to a modification of the protein's energy hypersurface that makes more conformational states accessible.  The authors' results highlight the need for appropriate methods to est. the strength of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVjujR5VxjuLVg90H21EOLACvtfcHk0lgKgy5NkwDUMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhvVKlu7c%253D&md5=2bf871e7b7b977acf4df1e03371cdeb8</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980038-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980038-5%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DLigand%2520docking%2520to%2520proteins%2520with%2520discrete%2520side-chain%2520flexibility%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D235%26spage%3D345%26epage%3D356%26doi%3D10.1016%2FS0022-2836%2805%2980038-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Meiler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, D.</span><span> </span><span class="NLM_article-title">Rosettaligand: protein-small molecule docking with full side-chain flexibility</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">548</span><span class="refDoi"> DOI: 10.1002/prot.21086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.21086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16972285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=538-548&author=J.+Meilerauthor=D.+Baker&title=Rosettaligand%3A+protein-small+molecule+docking+with+full+side-chain+flexibility&doi=10.1002%2Fprot.21086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility</span></div><div class="casAuthors">Meiler, Jens; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-548</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Protein-small mol. docking algorithms provide a means to model the structure of protein-small mol. complexes in structural detail and play an important role in drug development.  In recent years the necessity of simulating protein side-chain flexibility for an accurate prediction of the protein-small mol. interfaces has become apparent, and an increasing no. of docking algorithms probe different approaches to include protein flexibility.  Here the authors describe a new method for docking small mols. into protein binding sites employing a Monte Carlo minimization procedure in which the rigid body position and orientation of the small mol. and the protein side-chain conformations are optimized simultaneously.  The energy function comprises van der Waals (VDW) interactions, an implicit solvation model, an explicit orientation hydrogen bonding potential, and an electrostatics model.  In an evaluation of the scoring function the computed energy correlated with exptl. small mol. binding energy with a correlation coeff. of 0.63 across a diverse set of 229 protein-small mol. complexes.  The docking method produced lowest energy models with a root mean square deviation (RMSD) smaller than 2 Å in 71 out of 100 protein-small mol. crystal structure complexes (self-docking).  In cross-docking calcns. in which both protein side-chain and small mol. internal degrees of freedom were varied the lowest energy predictions had RMSDs less than 2 Å in 14 of 20 test cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb1x4K4yjDk7Vg90H21EOLACvtfcHk0ljIwE1eDdkKcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ&md5=0e5580cc490c11a7e9c6265c27a14408</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1002%2Fprot.21086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21086%26sid%3Dliteratum%253Aachs%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DRosettaligand%253A%2520protein-small%2520molecule%2520docking%2520with%2520full%2520side-chain%2520flexibility%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26spage%3D538%26epage%3D548%26doi%3D10.1002%2Fprot.21086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Teague, S. J.</span><span> </span><span class="NLM_article-title">Implications of protein flexibility for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">527</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nrd1129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd1129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12838268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=527-541&author=S.+J.+Teague&title=Implications+of+protein+flexibility+for+drug+discovery&doi=10.1038%2Fnrd1129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of protein flexibility for drug discovery</span></div><div class="casAuthors">Teague, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">527-541</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins are in const. motion between different conformational states with similar energies.  This has often been ignored in drug design.  However, protein flexibility is fundamental to understanding the ways in which drugs exert biol. effects, their binding-site location, binding orientation, binding kinetics, metab. and transport.  Protein flexibility allows increased affinity to be achieved between a drug and its target.  This is crucial, because the lipophilicity and no. of polar interactions allowed for an oral drug is limited by absorption, distribution, metab. and toxicol. considerations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8-I4F-CgSbVg90H21EOLACvtfcHk0ljIwE1eDdkKcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D&md5=850b8838a78c1e00493da3089a0c62e9</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fnrd1129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1129%26sid%3Dliteratum%253Aachs%26aulast%3DTeague%26aufirst%3DS.%2BJ.%26atitle%3DImplications%2520of%2520protein%2520flexibility%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D527%26epage%3D541%26doi%3D10.1038%2Fnrd1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Computational chemistry in 25 years</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">9</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1007/s10822-011-9516-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-011-9516-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=9-10&author=R.+Abagyan&title=Computational+chemistry+in+25+years&doi=10.1007%2Fs10822-011-9516-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9516-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9516-z%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DComputational%2520chemistry%2520in%252025%2520years%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D9%26epage%3D10%26doi%3D10.1007%2Fs10822-011-9516-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group">Gane, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dean, P. M.</span><span> </span><span class="NLM_article-title">Recent advances in structure-based rational drug design</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span><span class="refDoi"> DOI: 10.1016/S0959-440X(00)00105-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0959-440X%2800%2900105-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10981625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlvFOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=401-404&author=P.+J.+Ganeauthor=P.+M.+Dean&title=Recent+advances+in+structure-based+rational+drug+design&doi=10.1016%2FS0959-440X%2800%2900105-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in structure-based rational drug design</span></div><div class="casAuthors">Gane, Paul J.; Dean, Philip M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">401-404</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 33 refs.  Two approaches to structure-based drug design, i.e., the docking of known compds. into a target protein and mol. assembly in situ, are seen to be merging technologies.  The need for structural information about drug-protein complexes is now fundamental for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKbI0wbFM0ArVg90H21EOLACvtfcHk0lj7TlFL5IJ2Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlvFOmtrs%253D&md5=7cd3881e867d9151557dcdd5f7f0e4bb</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2800%2900105-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252800%252900105-6%26sid%3Dliteratum%253Aachs%26aulast%3DGane%26aufirst%3DP.%2BJ.%26aulast%3DDean%26aufirst%3DP.%2BM.%26atitle%3DRecent%2520advances%2520in%2520structure-based%2520rational%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D10%26spage%3D401%26epage%3D404%26doi%3D10.1016%2FS0959-440X%2800%2900105-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lj7TlFL5IJ2Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of prescription drugs that interact with the norepinephrine transporter, NET</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">15810</span><span class="NLM_x">–</span> <span class="NLM_lpage">15815</span><span class="refDoi"> DOI: 10.1073/pnas.1106030108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1106030108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21885739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1emtr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=15810-15815&author=A.+Schlessingerauthor=E.+Geierauthor=H.+Fanauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+M.+Giacominiauthor=A.+Sali&title=Structure-based+discovery+of+prescription+drugs+that+interact+with+the+norepinephrine+transporter%2C+NET&doi=10.1073%2Fpnas.1106030108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of prescription drugs that interact with the norepinephrine transporter, NET</span></div><div class="casAuthors">Schlessinger, Avner; Geier, Ethan; Fan, Hao; Irwin, John J.; Shoichet, Brian K.; Giacomini, Kathleen M.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">15810-15815, S15810/1-S15810/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The norepinephrine transporter (NET) transports norepinephrine from the synapse into presynaptic neurons, where norepinephrine regulates signaling pathways assocd. with cardiovascular effects and behavioral traits via binding to various receptors (e.g., β2-adrenergic receptor).  NET is a known target for a variety of prescription drugs, including antidepressants and psychostimulants, and may mediate off-target effects of other prescription drugs.  Here, we identify prescription drugs that bind NET, using virtual ligand screening followed by exptl. validation of predicted ligands.  We began by constructing a comparative structural model of NET based on its alignment to the at. structure of a prokaryotic NET homolog, the leucine transporter LeuT.  The modeled binding site was validated by confirming that known NET ligands can be docked favorably compared to nonbinding mols.  We then computationally screened 6,436 drugs from the Kyoto Encyclopedia of Genes and Genomes (KEGG DRUG) against the NET model.  Ten of the 18 high-scoring drugs tested exptl. were found to be NET inhibitors; five of these were chem. novel ligands of NET.  These results may rationalize the efficacy of several sympathetic (tuaminoheptane) and antidepressant (tranylcypromine) drugs, as well as side effects of diabetes (phenformin) and 'Alzheimer's (talsaclidine) drugs.  The observations highlight the utility of virtual screening against a comparative model, even when the target shares less than 30% sequence identity with its template structure and no known ligands in the primary binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrClJE5OOI_VrVg90H21EOLACvtfcHk0lj7TlFL5IJ2Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1emtr3O&md5=980b33b8f54b1a57396645edd35450e4</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1106030108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1106030108%26sid%3Dliteratum%253Aachs%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DGeier%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStructure-based%2520discovery%2520of%2520prescription%2520drugs%2520that%2520interact%2520with%2520the%2520norepinephrine%2520transporter%252C%2520NET%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D15810%26epage%3D15815%26doi%3D10.1073%2Fpnas.1106030108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Rueda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Consistent improvement of cross-docking results using binding site ensembles generated with elastic network normal modes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">716</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1021/ci8003732</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci8003732" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFGmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=716-725&author=M.+Ruedaauthor=G.+Bottegoniauthor=R.+Abagyan&title=Consistent+improvement+of+cross-docking+results+using+binding+site+ensembles+generated+with+elastic+network+normal+modes&doi=10.1021%2Fci8003732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Consistent Improvement of Cross-Docking Results Using Binding Site Ensembles Generated with Elastic Network Normal Modes</span></div><div class="casAuthors">Rueda, Manuel; Bottegoni, Giovanni; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">716-725</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The representation of protein flexibility is still a challenge for the state-of-the-art flexible ligand docking protocols.  In this article, we use a large and diverse benchmark to prove that is possible to improve consistently the cross-docking performance against a single receptor conformation, using an equil. ensemble of binding site conformers.  The benchmark contained 28 proteins, and our method predicted the top-ranked near native ligand poses 20% more efficiently than using a single receptor.  The multiple conformations were derived from the collective variable space defined by all heavy-atom elastic network normal modes, including backbone and side chains.  We have found that the binding site displacements for best positioning of the ligand seem rather independent from the global collective motions of the protein.  We also found that the no. of binding site conformations needed to represent nonredundant flexibility was <100.  The ensemble of receptor conformations can be generated at our Web site at http://abagyan.scripps.edu/MRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKq-bZloG-zrVg90H21EOLACvtfcHk0lh25ENaTdF6HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFGmsrk%253D&md5=4b0c4c58c2d66df41acea3155d6ddd6c</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fci8003732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003732%26sid%3Dliteratum%253Aachs%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DConsistent%2520improvement%2520of%2520cross-docking%2520results%2520using%2520binding%2520site%2520ensembles%2520generated%2520with%2520elastic%2520network%2520normal%2520modes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D716%26epage%3D725%26doi%3D10.1021%2Fci8003732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Campbell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span> </span><span class="NLM_article-title">Ensemble-based docking using biased molecular dynamics</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2127</span><span class="NLM_x">–</span> <span class="NLM_lpage">2138</span><span class="refDoi"> DOI: 10.1021/ci400729j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400729j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovVGmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2127-2138&author=A.+J.+Campbellauthor=M.+L.+Lambauthor=D.+Joseph-McCarthy&title=Ensemble-based+docking+using+biased+molecular+dynamics&doi=10.1021%2Fci400729j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Ensemble-Based Docking Using Biased Molecular Dynamics</span></div><div class="casAuthors">Campbell, Arthur J.; Lamb, Michelle L.; Joseph-McCarthy, Diane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2127-2138</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins are dynamic mols., and understanding their movements, esp. as they relate to mol. recognition and protein-ligand interactions, poses a significant challenge to structure-based drug discovery.  In most instances, protein flexibility is underrepresented in computer-aided drug design due to uncertainties on how it should be accurately modeled as well as the computational cost assocd. with attempting to incorporate flexibility in the calcns.  One approach that aims to address these issues is ensemble-based docking.  With this technique, ligands are docked to an ensemble of rigid protein conformations.  Mol. dynamics (MD) simulations can be used to generate the ensemble of protein conformations for the subsequent docking.  Here we present a novel approach that uses biased-MD simulations to generate the docking ensemble.  The MD simulations are biased toward an initial protein-ligand X-ray complex structure.  The biasing maintains some of the original crystallog. pocket-ligand information and thereby enhances sampling of the more relevant conformational space of the protein.  Resulting trajectories are clustered to select a representative set of protein conformations, and ligands are docked to that reduced set of conformations.  Cross-docking to this ensemble and then selecting the lowest scoring pose enables reliable identification of the correct binding mode.  Various levels of biasing are investigated, and the method is validated for cyclin-dependent kinase 2 and factor Xa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94p-kbdFG87Vg90H21EOLACvtfcHk0lh25ENaTdF6HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovVGmt7c%253D&md5=5789c563c368e3d8720d62f3e3749dfc</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Fci400729j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400729j%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26atitle%3DEnsemble-based%2520docking%2520using%2520biased%2520molecular%2520dynamics%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2127%26epage%3D2138%26doi%3D10.1021%2Fci400729j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group">Lin, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schames, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Computational drug design accommodating receptor flexibility: The relaxed complex scheme</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">5632</span><span class="NLM_x">–</span> <span class="NLM_lpage">5633</span><span class="refDoi"> DOI: 10.1021/ja0260162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0260162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtFags7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2002&pages=5632-5633&author=J.+H.+Linauthor=A.+L.+Perrymanauthor=J.+R.+Schamesauthor=J.+A.+McCammon&title=Computational+drug+design+accommodating+receptor+flexibility%3A+The+relaxed+complex+scheme&doi=10.1021%2Fja0260162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Drug Design Accommodating Receptor Flexibility: The Relaxed Complex Scheme</span></div><div class="casAuthors">Lin, Jung-Hsin; Perryman, Alexander L.; Schames, Julie R.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5632-5633</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel computational methodol. for drug design that accommodates receptor flexibility is described.  This "relaxed-complex" method recognizes that ligand may bind to conformations that occur only rarely in the dynamics of the receptor.  We have shown that the ligand-enzyme binding modes are very sensitive to the enzyme conformations, and our approach is capable of finding the best ligand-enzyme complexes.  This new method serves as the computational analog of the exptl. "SAR by NMR" and "tether" methods, which permit a building block approach for constructing a very potent drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcdlKmeE9wgbVg90H21EOLACvtfcHk0lieQ79QF1HUbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtFags7s%253D&md5=3877fe5dad2dd6bdf63a71c5aa84e2dc</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Fja0260162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0260162%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%2BH.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DSchames%26aufirst%3DJ.%2BR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DComputational%2520drug%2520design%2520accommodating%2520receptor%2520flexibility%253A%2520The%2520relaxed%2520complex%2520scheme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D5632%26epage%3D5633%26doi%3D10.1021%2Fja0260162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group">Spyrakis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">BidonChanal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span> </span><span class="NLM_article-title">Protein flexibility and ligand recognition: challenges for molecular modeling</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.2174/156802611794863571</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802611794863571" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20939788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslahsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=192-210&author=F.+Spyrakisauthor=A.+BidonChanalauthor=X.+Barrilauthor=F.+J.+Luque&title=Protein+flexibility+and+ligand+recognition%3A+challenges+for+molecular+modeling&doi=10.2174%2F156802611794863571"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Protein flexibility and ligand recognition: challenges for molecular modeling</span></div><div class="casAuthors">Spyrakis, Francesca; Chanal, Axel Bidon; Barril, Xavier; Luque, F. Javier</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">192-210</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The intrinsic dynamics of macromols. is an essential property to relate the structure of biomol. systems with their function in the cell.  In the field of ligand-receptor recognition, numerous evidences have revealed the limitations of the lock-and-key theory, and the need to elaborate models that take into account the inherent plasticity of biomols., such as the induced-fit model or the existence of an ensemble of pre-equilibrated conformations.  Depending on the nature of the target system, ligand binding can be assocd. with small local adjustments in side chains or even the backbone to large-scale motions of structural fragments, domains or even subunits.  Reproducing the inherent flexibility of biomols. has thus become one of the most challenging issues in mol. modeling and simulation studies, as it has direct implications in our understanding of the structure-function relationships, but even in areas such as virtual screening and structure-based drug discovery.  Given the intrinsic limitation of conventional simulation tools, only events occurring in short time scales can be reproduced at a high accuracy level through all-atom techniques such as Mol. Dynamics simulations.  However, larger structural rearrangements demand the use of enhanced sampling methods relying on modified descriptions of the biomol. system or the potential surface.  This review illustrates the crucial role that structural plasticity plays in mediating ligand recognition through representative examples.  In addn., it discusses some of the most powerful computational tools developed to characterize the conformational flexibility in ligand-receptor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEZna2C0ZlSLVg90H21EOLACvtfcHk0lieQ79QF1HUbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslahsrc%253D&md5=9d6b8673c6dc5cc6db4f6d0943373826</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.2174%2F156802611794863571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794863571%26sid%3Dliteratum%253Aachs%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DBidonChanal%26aufirst%3DA.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26atitle%3DProtein%2520flexibility%2520and%2520ligand%2520recognition%253A%2520challenges%2520for%2520molecular%2520modeling%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D192%26epage%3D210%26doi%3D10.2174%2F156802611794863571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group">Antolin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hakkaya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaioni, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicciari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macchiarulo, A.</span><span> </span><span class="NLM_article-title">Exploring the effect of PARP-1 flexibility in docking studies</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">201</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2013.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmgm.2013.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Wks7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=192-201&author=A.+A.+Antolinauthor=A.+Carottiauthor=R.+Nutiauthor=A.+Hakkayaauthor=E.+Camaioniauthor=J.+Mestresauthor=R.+Pellicciariauthor=A.+Macchiarulo&title=Exploring+the+effect+of+PARP-1+flexibility+in+docking+studies&doi=10.1016%2Fj.jmgm.2013.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the effect of PARP-1 flexibility in docking studies</span></div><div class="casAuthors">Antolin, Albert A.; Carotti, Andrea; Nuti, Roberto; Hakkaya, Aydie; Camaioni, Emidio; Mestres, Jordi; Pellicciari, Roberto; Macchiarulo, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">192-201</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose)polymerase-1 (PARP-1) is an enzyme belonging to the ADP-ribosyltransferase family.  A large body of works has validated PARP-1 as an attractive drug target for different therapeutic areas, including cancers and ischemia.  Accordingly, sampling the conformational space of the enzyme is pivotal to understand its functions and improve structure-based drug discovery approaches.  In the first part of this study we apply replica exchange mol. dynamic (REMD) simulations to sample the conformational space of the catalytic domain of PARP-1 in the ligand-bound and unbound forms.  In the second part, we assess how and to what extend the emerging enzyme flexibility affects the performance of docking expts. of a library of PARP-1 inhibitors.  This study pinpoints a putative key role of conformational shifts of Leu324, Tyr325 and Lys242 in opening an addnl. binding site pocket that affects the binding of ligands to the catalytic cleft of PARP-1.  Furthermore, it highlights the improvement of the enrichment factor of active ligands obtained in docking expts. when using conformations generated with REMD simulations of ligand-bound PARP-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDVOOBC0BALbVg90H21EOLACvtfcHk0lieQ79QF1HUbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Wks7%252FI&md5=eaa9dc18eb6e3db0bc9d68afc0714ddb</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2013.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2013.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DAntolin%26aufirst%3DA.%2BA.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DNuti%26aufirst%3DR.%26aulast%3DHakkaya%26aufirst%3DA.%26aulast%3DCamaioni%26aufirst%3DE.%26aulast%3DMestres%26aufirst%3DJ.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DMacchiarulo%26aufirst%3DA.%26atitle%3DExploring%2520the%2520effect%2520of%2520PARP-1%2520flexibility%2520in%2520docking%2520studies%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2013%26volume%3D45%26spage%3D192%26epage%3D201%26doi%3D10.1016%2Fj.jmgm.2013.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Information decay in molecular docking screens against holo, apo, and modeled conformations of enzymes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2895</span><span class="NLM_x">–</span> <span class="NLM_lpage">2907</span><span class="refDoi"> DOI: 10.1021/jm0300330</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0300330" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2895-2907&author=S.+L.+McGovernauthor=B.+K.+Shoichet&title=Information+decay+in+molecular+docking+screens+against+holo%2C+apo%2C+and+modeled+conformations+of+enzymes&doi=10.1021%2Fjm0300330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes</span></div><div class="casAuthors">McGovern, Susan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2895-2907</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking uses the three-dimensional structure of a receptor to screen a small mol. database for potential ligands.  The dependence of docking screens on the conformation of the binding site remains an open question.  To evaluate the information loss that occurs as the active site conformation becomes less defined, a small mol. database was docked against the holo (ligand bound), apo, and homol. modeled structures of 10 different enzyme binding sites.  The holo and apo representations were crystallog. structures taken from the Protein Data Bank (PDB), and the homol.-modeled structures were taken from the publicly available resource ModBase.  The database docked was the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small mols. that contained at least 35 ligands for each of the 10 systems.  In all sites, at least 99% of the mols. in the MDDR were treated as nonbinding decoys.  For each system, the holo, apo, and modeled structures were used to screen the MDDR, and the ability of each structure to enrich the known ligands for that system over random selection was evaluated.  The best overall enrichment was produced by the holo structure in seven systems, the apo structure in two systems, and the modeled structure in one system.  These results suggest that the performance of the docking calcn. is affected by the particular representation of the receptor used in the screen, and that the holo structure is the one most likely to yield the best discrimination between known ligands and decoy mols., but important exceptions to this rule also emerge from this study.  Although each of the holo, apo, and modeled conformations led to enrichment of known ligands in all systems, the enrichment did not always rise to a level judged to be sufficient to justify the effort of a docking screen.  Using a 20-fold enrichment of known ligands over random selection as a rough guideline for what might be enough to justify a docking screen, the holo conformation of the enzyme met this criterion in eight of 10 sites, whereas the apo conformation met this criterion in only two sites and the modeled conformation in three.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo248QTuvleNrVg90H21EOLACvtfcHk0ljQNTIwE5urWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsFSju7c%253D&md5=bd3145db4e1fbc1288f5dd65d8c52291</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Fjm0300330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0300330%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DInformation%2520decay%2520in%2520molecular%2520docking%2520screens%2520against%2520holo%252C%2520apo%252C%2520and%2520modeled%2520conformations%2520of%2520enzymes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2895%26epage%3D2907%26doi%3D10.1021%2Fjm0300330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group">Rueda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Recipes for the selection of experimental protein conformations for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1021/ci9003943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=186-193&author=M.+Ruedaauthor=G.+Bottegoniauthor=R.+Abagyan&title=Recipes+for+the+selection+of+experimental+protein+conformations+for+virtual+screening&doi=10.1021%2Fci9003943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Recipes for the Selection of Experimental Protein Conformations for Virtual Screening</span></div><div class="casAuthors">Rueda, Manuel; Bottegoni, Giovanni; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-193</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of multiple x-ray protein structures has been reported to be an efficient alternative for the representation of the binding pocket flexibility needed for accurate small mols. docking.  However, the docking performance of the individual single conformations varies widely, and adding certain conformations to an ensemble is even counterproductive.  Here the authors used a very large and diverse benchmark of 1068 x-ray protein conformations of 99 therapeutically relevant proteins, first, to compare the performance of the ensemble and single-conformation docking and, second, to find the properties of the best-performing conformers that can be used to select a smaller set of conformers for ensemble docking.  The conformer selection has been validated through retrospective virtual screening expts. aimed at sepg. known ligand binders from decoys.  The authors found that the conformers cocrystd. with the largest ligands displayed high selectivity for binders, and when combined in ensembles they consistently provided better results than randomly chosen protein conformations.  The use of ensembles encompassing between 3 and 5 exptl. conformations consistently improved the docking accuracy and binders vs. decoys sepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgQPiTcnZy7Vg90H21EOLACvtfcHk0ljQNTIwE5urWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I&md5=159915d0fb8aa304a49592a0bbb6401f</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fci9003943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003943%26sid%3Dliteratum%253Aachs%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DRecipes%2520for%2520the%2520selection%2520of%2520experimental%2520protein%2520conformations%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D186%26epage%3D193%26doi%3D10.1021%2Fci9003943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, S. D.</span><span> </span><span class="NLM_article-title">Unveiling the full potential of flexible receptor docking using multiple crystallographic structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4432</span><span class="NLM_x">–</span> <span class="NLM_lpage">4443</span><span class="refDoi"> DOI: 10.1021/jm048972v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm048972v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlKisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4432-4443&author=X.+Barrilauthor=S.+D.+Morley&title=Unveiling+the+full+potential+of+flexible+receptor+docking+using+multiple+crystallographic+structures&doi=10.1021%2Fjm048972v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the Full Potential of Flexible Receptor Docking Using Multiple Crystallographic Structures</span></div><div class="casAuthors">Barril, Xavier; Morley, S. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4432-4443</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the current challenges in docking studies is the inclusion of receptor flexibility.  This is crucial because the binding sites of many therapeutic targets sample a wide range of conformational states, which has major consequences on mol. recognition.  In this paper, we make use of very large sets of x-ray structures of cyclin dependent kinase 2 (CDK2) and heat shock protein 90 (HSP90) to assess the performance of flexible receptor docking in binding-mode prediction and virtual screening expts.  Flexible receptor docking performs much better than rigid receptor docking in the former application.  Regarding the latter, we observe a significant improvement in the prediction of binding affinities, but owing to an increase in the no. of false positives, this is not translated into better hit rates.  A simple scoring scheme to correct this limitation is presented.  More importantly, pitfalls inherent to flexible receptor docking have been identified and guidelines are presented to avoid them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmHIP2n9ybSrVg90H21EOLACvtfcHk0liA3PatJY97Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktlKisbk%253D&md5=9a5b15df10784020470cd2972d737995</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1021%2Fjm048972v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm048972v%26sid%3Dliteratum%253Aachs%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DMorley%26aufirst%3DS.%2BD.%26atitle%3DUnveiling%2520the%2520full%2520potential%2520of%2520flexible%2520receptor%2520docking%2520using%2520multiple%2520crystallographic%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4432%26epage%3D4443%26doi%3D10.1021%2Fjm048972v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group">Nichols, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">On the Use of molecular dynamics receptor conformations for virtual screening</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">819</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1007/978-1-61779-465-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2F978-1-61779-465-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22183532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=819&publication_year=2012&pages=93-103&author=S.+E.+Nicholsauthor=R.+Baronauthor=J.+A.+McCammon&title=On+the+Use+of+molecular+dynamics+receptor+conformations+for+virtual+screening&doi=10.1007%2F978-1-61779-465-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">On the use of molecular dynamics receptor conformations for virtual screening</span></div><div class="casAuthors">Nichols Sara E; Baron Riccardo; McCammon J Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Methods in molecular biology (Clifton, N.J.)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">819</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">93-103</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Receptors are inherently dynamic and this flexibility is important to consider when constructing a model of molecular association.  Conformations from molecular dynamics simulations, a well-established method for examining protein dynamics, can be used in virtual screening to account for flexibility in structure-based drug discovery.  Different receptor configurations influence docking results.  Molecular dynamics simulations can provide snapshots that improve virtual screening predictive power over known crystal structures, most likely as a result of sampling more relevant receptor conformations.  Here we highlight some details and nuances of using such snapshots and evaluating them for predictive performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3KPL-W_yq5LP3FVUi_gf8fW6udTcc2eZMZxsgQVYH6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjug%253D%253D&md5=8cff07626bfa72238d00170ce4d23ee6</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DNichols%26aufirst%3DS.%2BE.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DOn%2520the%2520Use%2520of%2520molecular%2520dynamics%2520receptor%2520conformations%2520for%2520virtual%2520screening%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D819%26spage%3D93%26epage%3D103%26doi%3D10.1007%2F978-1-61779-465-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group">Amaro, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">An improved relaxed complex scheme for receptor flexibility in computer-aided drug design</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span><span class="refDoi"> DOI: 10.1007/s10822-007-9159-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-007-9159-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18196463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCqur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=693-705&author=R.+E.+Amaroauthor=R.+Baronauthor=J.+A.+McCammon&title=An+improved+relaxed+complex+scheme+for+receptor+flexibility+in+computer-aided+drug+design&doi=10.1007%2Fs10822-007-9159-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">An improved relaxed complex scheme for receptor flexibility in computer-aided drug design</span></div><div class="casAuthors">Amaro, Rommie E.; Baron, Riccardo; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The interactions among assocg. (macro)mols. are dynamic, which adds to the complexity of mol. recognition.  While ligand flexibility is well accounted for in computational drug design, the effective inclusion of receptor flexibility remains an important challenge.  The relaxed complex scheme (RCS) is a promising computational methodol. that combines the advantages of docking algorithms with dynamic structural information provided by mol. dynamics (MD) simulations, therefore explicitly accounting for the flexibility of both the receptor and the docked ligands.  Here, the authors briefly review the RCS and discuss new extensions and improvements of this methodol. in the context of ligand binding to 2 example targets: kinetoplastid RNA editing ligase 1 and the W191G cavity mutant of cytochrome c peroxidase.  The RCS improvements include its extension to virtual screening, more rigorous characterization of local and global binding effects, and methods to improve its computational efficiency by reducing the receptor ensemble to a representative set of configurations.  The choice of receptor ensemble, its influence on the predictive power of RCS, and the current limitations for an accurate treatment of the solvent contributions are also briefly discussed.  Finally, the authors outline potential methodol. improvements that they anticipate will assist future development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouc8v4XJw69rVg90H21EOLACvtfcHk0liA3PatJY97Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCqur7F&md5=e3abac702a530277df56bb155802a649</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9159-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9159-2%26sid%3Dliteratum%253Aachs%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAn%2520improved%2520relaxed%2520complex%2520scheme%2520for%2520receptor%2520flexibility%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D693%26epage%3D705%26doi%3D10.1007%2Fs10822-007-9159-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rueda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Systematic exploitation of multiple receptor conformations for virtual ligand screening</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e18845</span><span class="refDoi"> DOI: 10.1371/journal.pone.0018845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pone.0018845" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e18845&author=G.+Bottegoniauthor=W.+Rocchiaauthor=M.+Ruedaauthor=R.+Abagyanauthor=A.+Cavalli&title=Systematic+exploitation+of+multiple+receptor+conformations+for+virtual+ligand+screening&doi=10.1371%2Fjournal.pone.0018845"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0018845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0018845%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSystematic%2520exploitation%2520of%2520multiple%2520receptor%2520conformations%2520for%2520virtual%2520ligand%2520screening%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De18845%26doi%3D10.1371%2Fjournal.pone.0018845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group">Swanson, J. M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Revisiting free energy calculations: One step closer to rigorous scoring functions and one step beyond MM/PBSA</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span><span class="refDoi"> DOI: 10.1016/S0006-3495(04)74084-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0006-3495%2804%2974084-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2004&pages=67-74&author=J.+M.+J.+Swansonauthor=R.+Henchmanauthor=J.+A.+McCammon&title=Revisiting+free+energy+calculations%3A+One+step+closer+to+rigorous+scoring+functions+and+one+step+beyond+MM%2FPBSA&doi=10.1016%2FS0006-3495%2804%2974084-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2804%2974084-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252804%252974084-9%26sid%3Dliteratum%253Aachs%26aulast%3DSwanson%26aufirst%3DJ.%2BM.%2BJ.%26aulast%3DHenchman%26aufirst%3DR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DRevisiting%2520free%2520energy%2520calculations%253A%2520One%2520step%2520closer%2520to%2520rigorous%2520scoring%2520functions%2520and%2520one%2520step%2520beyond%2520MM%252FPBSA%26jtitle%3DBiophys.%2520J.%26date%3D2004%26volume%3D86%26spage%3D67%26epage%3D74%26doi%3D10.1016%2FS0006-3495%2804%2974084-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group">de Beer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P. E.</span><span> </span><span class="NLM_article-title">The role of water molecules in computational drug design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.2174/156802610790232288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802610790232288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19929830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=55-66&author=S.+de+Beerauthor=N.+P.+E.+Vermeulen&title=The+role+of+water+molecules+in+computational+drug+design&doi=10.2174%2F156802610790232288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">The role of water molecules in computational drug design</span></div><div class="casAuthors">de Beer, Stephanie B. A.; Vermeulen, Nico P. E.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-66</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Although water mols. are small and only consist of two different atom types, they play various roles in cellular systems.  This review discusses their influence on the binding process between biomacromol. targets and small mol. ligands and how this influence can be modeled in computational drug design approaches.  Both the structure and the thermodn. of active site waters will be discussed as these influence the binding process significantly.  Structurally conserved waters cannot always be detd. exptl. and if obsd., it is not clear if they will be replaced upon ligand binding, even if sufficient space is available.  Methods to predict the presence of water in protein-ligand complexes will be reviewed.  Subsequently, we will discuss methods to include water in computational drug research.  Either as an addnl. factor in automated docking expts., or explicitly in detailed mol. dynamics simulations, the effect of water on the quality of the simulations is significant, but not easily predicted.  The most detailed calcns. involve ests. of the free energy contribution of water mols. to protein-ligand complexes.  These calcns. are computationally demanding, but give insight in the versatility and importance of water in ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhA2h-JAqzhbVg90H21EOLACvtfcHk0ljoDVsQ4jOD_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D&md5=5767b8a3f82a33845fae3941a1084067</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.2174%2F156802610790232288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232288%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBeer%26aufirst%3DS.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%2BE.%26atitle%3DThe%2520role%2520of%2520water%2520molecules%2520in%2520computational%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D55%26epage%3D66%26doi%3D10.2174%2F156802610790232288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raco, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span> </span><span class="NLM_article-title">A computational study of the binding of propidium to the peripheral anionic site of human acetylcholinesterase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3991</span><span class="NLM_x">–</span> <span class="NLM_lpage">3999</span><span class="refDoi"> DOI: 10.1021/jm040787u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040787u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFyhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3991-3999&author=A.+Cavalliauthor=G.+Bottegoniauthor=C.+Racoauthor=M.+De+Vivoauthor=M.+Recanatini&title=A+computational+study+of+the+binding+of+propidium+to+the+peripheral+anionic+site+of+human+acetylcholinesterase&doi=10.1021%2Fjm040787u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">A Computational Study of the Binding of Propidium to the Peripheral Anionic Site of Human Acetylcholinesterase</span></div><div class="casAuthors">Cavalli, Andrea; Bottegoni, Giovanni; Raco, Caterina; De Vivo, Marco; Recanatini, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3991-3999</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Combined docking and mol. dynamics (MD) simulations were carried out to investigate the binding mode of propidium at the human acetylcholinesterase (HuAChE) peripheral site.  Two different docking protocols followed by cluster analyses were performed, allowing the identification of five high-populated and low-energy configuration families.  To dynamically explore the behavior of the ligand at the peripheral HuAChE binding site, six complexes (five low-energy and one high-energy) were submitted to 2.5 ns of MD simulations.  The representative propidium/HuAChE binding modes were chosen on the basis of both the docking energy score and the dynamic stability of the complexes throughout the MD simulations.  The most stable poses of propidium at HuAChE PAS were similar to those exptl. detd. with the murine enzyme.  The authors therefore suggest that the present modeling protocol might be used in the dynamic investigation of the interactions of a small-mol. inhibitor with a surface-like binding site of a protein.  Finally, because of the biol. relevance of the target studied here, the present results can be of interest for the rational design of mols. potentially useful in the treatment of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqceHTIQSi_7rVg90H21EOLACvtfcHk0ljoDVsQ4jOD_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFyhtL8%253D&md5=ce40924ee4d40405e890566142fa7d46</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1021%2Fjm040787u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040787u%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRaco%26aufirst%3DC.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DRecanatini%26aufirst%3DM.%26atitle%3DA%2520computational%2520study%2520of%2520the%2520binding%2520of%2520propidium%2520to%2520the%2520peripheral%2520anionic%2520site%2520of%2520human%2520acetylcholinesterase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3991%26epage%3D3999%26doi%3D10.1021%2Fjm040787u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group">Kacker, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangold, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Combining dyad protonation and active site plasticity in BACE-1 structure-based drug design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1079</span><span class="NLM_x">–</span> <span class="NLM_lpage">1085</span><span class="refDoi"> DOI: 10.1021/ci200366z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200366z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVOjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1079-1085&author=P.+Kackerauthor=M.+Masettiauthor=M.+Mangoldauthor=G.+Bottegoniauthor=A.+Cavalli&title=Combining+dyad+protonation+and+active+site+plasticity+in+BACE-1+structure-based+drug+design&doi=10.1021%2Fci200366z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Dyad Protonation and Active Site Plasticity in BACE-1 Structure-Based Drug Design</span></div><div class="casAuthors">Kacker, Puneet; Masetti, Matteo; Mangold, Martina; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1079-1085</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of the BACE-1 catalytic dyad to adopt multiple protonation states and the conformational flexibility of the active site have hampered the reliability of computational screening campaigns carried out on this drug target for Alzheimer's disease.  Here, we propose a protocol that, for the first time, combining quantum mech. calcns., mol. dynamics, and conformational ensemble virtual ligand screening addresses these issues simultaneously.  The encouraging results prefigure this approach as a valuable tool for future drug discovery campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKRhdkbh-ZUbVg90H21EOLACvtfcHk0ljoDVsQ4jOD_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVOjtro%253D&md5=6e718d38ed3c4fa24820b33161183b78</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Fci200366z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200366z%26sid%3Dliteratum%253Aachs%26aulast%3DKacker%26aufirst%3DP.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DMangold%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DCombining%2520dyad%2520protonation%2520and%2520active%2520site%2520plasticity%2520in%2520BACE-1%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1079%26epage%3D1085%26doi%3D10.1021%2Fci200366z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degliesposti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Rio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sgobba, M.</span><span> </span><span class="NLM_article-title">Binding estimation after refinement, a new automated procedure for the refinement and rescoring of docked ligands in virtual screening</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">286</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2009.00780.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1111%2Fj.1747-0285.2009.00780.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19207463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1WhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2009&pages=283-286&author=G.+Rastelliauthor=G.+Degliespostiauthor=A.+Del+Rioauthor=M.+Sgobba&title=Binding+estimation+after+refinement%2C+a+new+automated+procedure+for+the+refinement+and+rescoring+of+docked+ligands+in+virtual+screening&doi=10.1111%2Fj.1747-0285.2009.00780.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Binding estimation after refinement, a new automated procedure for the refinement and rescoring of docked ligands in virtual screening</span></div><div class="casAuthors">Rastelli, Giulio; Degliesposti, Gianluca; Del Rio, Alberto; Sgobba, Miriam</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-286</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Binding estn. after refinement (BEAR) is a novel automated computational procedure suitable for correcting and overcoming limitations of docking procedures such as poor scoring function and the generation of unreasonable ligand conformations.  BEAR makes use of mol. dynamics simulation followed by MM-PBSA and MM-GBSA binding free energy ests. as tools to refine and rescore the structures obtained from docking virtual screenings.  As binding estn. after refinement relies on mol. dynamics, the entire procedure can be tailored to the needs of the end-user in terms of computational time and the desired accuracy of the results.  In a validation test, binding estn. after refinement and rescoring resulted in a significant enrichment of known ligands among top scoring compds. compared with the original docking results.  Binding estn. after refinement has direct and straightforward application in virtual screening for correcting both false-pos. and false-neg. hits, and should facilitate more reliable selection of biol. active mols. from compd. databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_CIA8Wshju7Vg90H21EOLACvtfcHk0ljiB54cPEtNQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1WhtL8%253D&md5=8a113fb4e3ae908b3d70eb97a4c964f5</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2009.00780.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2009.00780.x%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DDegliesposti%26aufirst%3DG.%26aulast%3DDel%2BRio%26aufirst%3DA.%26aulast%3DSgobba%26aufirst%3DM.%26atitle%3DBinding%2520estimation%2520after%2520refinement%252C%2520a%2520new%2520automated%2520procedure%2520for%2520the%2520refinement%2520and%2520rescoring%2520of%2520docked%2520ligands%2520in%2520virtual%2520screening%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2009%26volume%3D73%26spage%3D283%26epage%3D286%26doi%3D10.1111%2Fj.1747-0285.2009.00780.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group">Sabbadin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciancetta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moro, S.</span><span> </span><span class="NLM_article-title">Bridging molecular docking to membrane molecular dynamics to investigate GPCR-ligand recognition: the human A(2)A adenosine receptor as a key study</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span><span class="refDoi"> DOI: 10.1021/ci400532b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400532b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=169-183&author=D.+Sabbadinauthor=A.+Ciancettaauthor=S.+Moro&title=Bridging+molecular+docking+to+membrane+molecular+dynamics+to+investigate+GPCR-ligand+recognition%3A+the+human+A%282%29A+adenosine+receptor+as+a+key+study&doi=10.1021%2Fci400532b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Bridging Molecular Docking to Membrane Molecular Dynamics To Investigate GPCR-Ligand Recognition: The Human A2A Adenosine Receptor as a Key Study</span></div><div class="casAuthors">Sabbadin, Davide; Ciancetta, Antonella; Moro, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">169-183</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) represent the largest family of cell-surface receptors and about one-third of the actual targets of clin. used drugs.  Following the progress made in the field of GPCRs structural detn., docking-based screening for novel potent and selective ligands is becoming an increasingly adopted strategy in the drug discovery process.  However, this methodol. is not yet able to anticipate the "bioactive" binding mode and discern it among other conformations.  The authors present a novel approach consisting in the integration of mol. docking and membrane MD simulations with the aim to merge the rapid sampling of ligand poses into the binding site, typical of docking algorithms, with the thermodn. accuracy of MD simulations in describing, at the mol. level, the stability a GPCR-ligand complex embedded into explicit lipid-water environment.  To validate the authors' approach, the authors have chosen as a key study the human A2A adenosine receptor (hA2A AR) and selected four receptor-antagonist complexes and one receptor-agonist complex that have been recently crystd.  In light of the obtained results, the authors believe that the authors' novel strategy can be extended to other GPCRs and might represent a valuable tool to anticipate the "bioactive" conformation of high-affinity ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpddvdZooeg-LVg90H21EOLACvtfcHk0ljiB54cPEtNQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurvF&md5=eec1db7115955775558cfff9fbaaed9e</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Fci400532b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400532b%26sid%3Dliteratum%253Aachs%26aulast%3DSabbadin%26aufirst%3DD.%26aulast%3DCiancetta%26aufirst%3DA.%26aulast%3DMoro%26aufirst%3DS.%26atitle%3DBridging%2520molecular%2520docking%2520to%2520membrane%2520molecular%2520dynamics%2520to%2520investigate%2520GPCR-ligand%2520recognition%253A%2520the%2520human%2520A%25282%2529A%2520adenosine%2520receptor%2520as%2520a%2520key%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D169%26epage%3D183%26doi%3D10.1021%2Fci400532b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group">Lauro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferruz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">Reranking docking poses using molecular simulations and approximate free energy methods</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2185</span><span class="NLM_x">–</span> <span class="NLM_lpage">2189</span><span class="refDoi"> DOI: 10.1021/ci500309a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500309a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyns7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2185-2189&author=G.+Lauroauthor=N.+Ferruzauthor=S.+Fulleauthor=M.+J.+Harveyauthor=P.+W.+Finnauthor=G.+De+Fabritiis&title=Reranking+docking+poses+using+molecular+simulations+and+approximate+free+energy+methods&doi=10.1021%2Fci500309a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Reranking Docking Poses Using Molecular Simulations and Approximate Free Energy Methods</span></div><div class="casAuthors">Lauro, G.; Ferruz, N.; Fulle, S.; Harvey, M. J.; Finn, P. W.; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2185-2189</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fast and accurate identification of active compds. is essential for effective use of virtual screening workflows.  Here, the authors have compared the ligand-ranking efficiency of the linear interaction energy (LIE) method against std. docking approaches.  Using a trypsin set of 1549 compds., the authors performed 12,250 mol dynamics simulations.  The LIE method proved effective but did not yield results significantly better than those obtained with docking codes.  The entire database of simulations is released.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW19HnT0xTwrVg90H21EOLACvtfcHk0ljM8iTj7gyNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyns7bM&md5=9ab6c274c82f8eec7bb97a8223503d36</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Fci500309a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500309a%26sid%3Dliteratum%253Aachs%26aulast%3DLauro%26aufirst%3DG.%26aulast%3DFerruz%26aufirst%3DN.%26aulast%3DFulle%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DM.%2BJ.%26aulast%3DFinn%26aufirst%3DP.%2BW.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DReranking%2520docking%2520poses%2520using%2520molecular%2520simulations%2520and%2520approximate%2520free%2520energy%2520methods%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2185%26epage%3D2189%26doi%3D10.1021%2Fci500309a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group">Shirts, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Computing: screen savers of the world unite!</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">1903</span><span class="NLM_x">–</span> <span class="NLM_lpage">1904</span><span class="refDoi"> DOI: 10.1126/science.290.5498.1903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.290.5498.1903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=17742054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslKitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2000&pages=1903-1904&author=M.+Shirtsauthor=V.+S.+Pande&title=Computing%3A+screen+savers+of+the+world+unite%21&doi=10.1126%2Fscience.290.5498.1903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Perspectives: Computing: Screen savers of the World unite!</span></div><div class="casAuthors">Shirts, Michael; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">5498</span>),
    <span class="NLM_cas:pages">1903-1904</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW_KxVt3dZRrVg90H21EOLACvtfcHk0ljM8iTj7gyNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslKitrw%253D&md5=920269eabb61d86cd45aa74cfa89a246</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1126%2Fscience.290.5498.1903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.290.5498.1903%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DComputing%253A%2520screen%2520savers%2520of%2520the%2520world%2520unite%2521%26jtitle%3DScience%26date%3D2000%26volume%3D290%26spage%3D1903%26epage%3D1904%26doi%3D10.1126%2Fscience.290.5498.1903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlow, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Pathway and mechanism of drug binding to G-protein-coupled receptors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">13118</span><span class="NLM_x">–</span> <span class="NLM_lpage">13123</span><span class="refDoi"> DOI: 10.1073/pnas.1104614108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1104614108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21778406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13118-13123&author=R.+O.+Drorauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+W.+Borhaniauthor=P.+Maragakisauthor=Y.+Shanauthor=H.+Xuauthor=D.+E.+Shaw&title=Pathway+and+mechanism+of+drug+binding+to+G-protein-coupled+receptors&doi=10.1073%2Fpnas.1104614108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Pathway and mechanism of drug binding to G-protein-coupled receptors</span></div><div class="casAuthors">Dror, Ron O.; Pan, Albert C.; Arlow, Daniel H.; Borhani, David W.; Maragakis, Paul; Shan, Yibing; Xu, Huafeng; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">13118-13123, S13118/1-S13118/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">How drugs bind to their receptors-from initial assocn., through drug entry into the binding pocket, to adoption of the final bound conformation, or "pose"-has remained unknown, even for G-protein-coupled receptor modulators, which constitute 1/3 of all marketed drugs.  Here, the authors captured this pharmaceutically crit. process in at. detail using the 1st unbiased mol. dynamics simulations in which drug mols. spontaneously assoc. with G-protein-coupled receptors to achieve final poses matching those detd. crystallog.  The authors found that several beta blockers and a beta agonist all traversed the same well-defined, dominant pathway as they bound to the β1- and β2-adrenergic receptors, initially making contact with a vestibule on each receptor's extracellular surface.  Surprisingly, assocn. with this vestibule, at a distance of 15 Å from the binding pocket, often presented the largest energetic barrier to binding, despite the fact that subsequent entry into the binding pocket required the receptor to deform and the drug to squeeze through a narrow passage.  The early barrier appeared to reflect the substantial dehydration that takes place as the drug assocs. with the vestibule.  The at.-level description of the binding process suggests opportunities for allosteric modulation and provides a structural foundation for future optimization of drug-receptor binding and unbinding rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKUYFtkTGwf7Vg90H21EOLACvtfcHk0ljM8iTj7gyNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE&md5=9cfa8673217931cacd586321309fa72e</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1104614108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1104614108%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DPathway%2520and%2520mechanism%2520of%2520drug%2520binding%2520to%2520G-protein-coupled%2520receptors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13118%26epage%3D13123%26doi%3D10.1073%2Fpnas.1104614108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group">Foda, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, M. A.</span><span> </span><span class="NLM_article-title">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">5939</span><span class="refDoi"> DOI: 10.1038/ncomms6939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fncomms6939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25600932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5939&author=Z.+H.+Fodaauthor=Y.+Shanauthor=E.+T.+Kimauthor=D.+E.+Shawauthor=M.+A.+Seeliger&title=A+dynamically+coupled+allosteric+network+underlies+binding+cooperativity+in+Src+kinase&doi=10.1038%2Fncomms6939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span></div><div class="casAuthors">Foda, Zachariah H.; Shan, Yibing; Kim, Eric T.; Shaw, David E.; Seeliger, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5939</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein tyrosine kinases are attractive drug targets because many human diseases are assocd. with the deregulation of kinase activity.  However, how the catalytic kinase domain integrates different signals and switches from an active to an inactive conformation remains incompletely understood.  Here we identify an allosteric network of dynamically coupled amino acids in Src kinase that connects regulatory sites to the ATP- and substrate-binding sites.  Surprisingly, reactants (ATP and peptide substrates) bind with neg. cooperativity to Src kinase while products (ADP and phosphopeptide) bind with pos. cooperativity.  We confirm the mol. details of the signal relay through the allosteric network by biochem. studies.  Expts. on two addnl. protein tyrosine kinases indicate that the allosteric network may be largely conserved among these enzymes.  Our work provides new insights into the regulation of protein tyrosine kinases and establishes a potential conduit by which resistance mutations to ATP-competitive kinase inhibitors can affect their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGfRZJun_T1rVg90H21EOLACvtfcHk0lg4q9_txtgSHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ&md5=cc24df9830ccb1525f66cc554795965c</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fncomms6939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6939%26sid%3Dliteratum%253Aachs%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26atitle%3DA%2520dynamically%2520coupled%2520allosteric%2520network%2520underlies%2520binding%2520cooperativity%2520in%2520Src%2520kinase%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D5939%26doi%3D10.1038%2Fncomms6939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkhipov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jumper, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Oncogenic mutations counteract intrinsic disorder in the EGFR kinase and promote receptor dimerization</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">860</span><span class="NLM_x">–</span> <span class="NLM_lpage">870</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.02.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.cell.2012.02.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22579287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFaju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=860-870&author=Y.+Shanauthor=M.+P.+Eastwoodauthor=X.+Zhangauthor=E.+T.+Kimauthor=A.+Arkhipovauthor=R.+O.+Drorauthor=J.+Jumperauthor=J.+Kuriyanauthor=D.+E.+Shaw&title=Oncogenic+mutations+counteract+intrinsic+disorder+in+the+EGFR+kinase+and+promote+receptor+dimerization&doi=10.1016%2Fj.cell.2012.02.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic mutations counteract intrinsic disorder in the EGFR kinase and promote receptor dimerization</span></div><div class="casAuthors">Shan, Yibing; Eastwood, Michael P.; Zhang, Xuewu; Kim, Eric T.; Arkhipov, Anton; Dror, Ron O.; Jumper, John; Kuriyan, John; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">860-870</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mutation and overexpression of the epidermal growth factor receptor (EGFR) are assocd. with the development of a variety of cancers, making this prototypical dimerization-activated receptor tyrosine kinase a prominent target of cancer drugs.  Using long-timescale mol. dynamics simulations, we find that the N lobe dimerization interface of the wild-type EGFR kinase domain is intrinsically disordered and that it becomes ordered only upon dimerization.  Our simulations suggest, moreover, that some cancer-linked mutations distal to the dimerization interface, particularly the widespread L834R mutation (also referred to as L858R), facilitate EGFR dimerization by suppressing this local disorder.  Corroborating these findings, our biophys. expts. and kinase enzymic assays indicate that the L834R mutation causes abnormally high activity primarily by promoting EGFR dimerization rather than by allowing activation without dimerization.  We also find that phosphorylation of EGFR kinase domain at Tyr845 may suppress the intrinsic disorder, suggesting a mol. mechanism for autonomous EGFR signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhEY912uUTrVg90H21EOLACvtfcHk0lg4q9_txtgSHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFaju7k%253D&md5=8ab973e5a5626ac495d176322f30d2b7</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.063%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DY.%26aulast%3DEastwood%26aufirst%3DM.%2BP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DArkhipov%26aufirst%3DA.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DJumper%26aufirst%3DJ.%26aulast%3DKuriyan%26aufirst%3DJ.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DOncogenic%2520mutations%2520counteract%2520intrinsic%2520disorder%2520in%2520the%2520EGFR%2520kinase%2520and%2520promote%2520receptor%2520dimerization%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D860%26epage%3D870%26doi%3D10.1016%2Fj.cell.2012.02.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favia, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Convertino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">The molecular basis for dual fatty acid amide hydrolase (FAAH)/cyclooxygenase (COX) inhibition</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1002/cmdc.201500507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fcmdc.201500507" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=G.+Palermoauthor=A.+D.+Faviaauthor=M.+Convertinoauthor=M.+De+Vivo&title=The+molecular+basis+for+dual+fatty+acid+amide+hydrolase+%28FAAH%29%2Fcyclooxygenase+%28COX%29+inhibition&doi=10.1002%2Fcmdc.201500507"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500507%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DFavia%26aufirst%3DA.%2BD.%26aulast%3DConvertino%26aufirst%3DM.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DThe%2520molecular%2520basis%2520for%2520dual%2520fatty%2520acid%2520amide%2520hydrolase%2520%2528FAAH%2529%252Fcyclooxygenase%2520%2528COX%2529%2520inhibition%26jtitle%3DChemMedChem%26date%3D2015%26doi%3D10.1002%2Fcmdc.201500507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piomelli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Covalent inhibitors of fatty acid amide hydrolase: a rationale for the activity of piperidine and piperazine aryl ureas</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6612</span><span class="NLM_x">–</span> <span class="NLM_lpage">6623</span><span class="refDoi"> DOI: 10.1021/jm2004283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2004283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6612-6623&author=G.+Palermoauthor=D.+Branduardiauthor=M.+Masettiauthor=A.+Lodolaauthor=M.+Morauthor=D.+Piomelliauthor=A.+Cavalliauthor=M.+De+Vivo&title=Covalent+inhibitors+of+fatty+acid+amide+hydrolase%3A+a+rationale+for+the+activity+of+piperidine+and+piperazine+aryl+ureas&doi=10.1021%2Fjm2004283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibitors of Fatty Acid Amide Hydrolase: A Rationale for the Activity of Piperidine and Piperazine Aryl Ureas</span></div><div class="casAuthors">Palermo, Giulia; Branduardi, Davide; Masetti, Matteo; Lodola, Alessio; Mor, Marco; Piomelli, Daniele; Cavalli, Andrea; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6612-6623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, covalent drugs have attracted great interest in the drug discovery community, with successful examples that have demonstrated their therapeutic effects.  Here, we focus on the covalent inhibition of the fatty acid amide hydrolase (FAAH), which is a promising strategy in the treatment of pain and inflammation.  Among the most recent and potent FAAH inhibitors (FAAHi), there are the cyclic piperidine and piperazine aryl ureas.  FAAH hydrolyzes efficiently the amide bond of these compds., forming a covalent enzyme-inhibitor adduct.  To rationalize this exptl. evidence, we performed an extensive computational anal. centered on piperidine-based PF750 (1) and piperazine-based JNJ1661010 (2), two potent lead compds. used to generate covalent inhibitors as clin. candidates.  We found that FAAH induces a distortion of the amide bond of the piperidine and piperazine aryl ureas.  Quantum mechanics/mol. mechanics ΔELUMO-HOMO energies indicate that the obsd. enzyme-induced distortion of the amide bond favors the formation of a covalent FAAH-inhibitor adduct.  These findings could help in the rational structure-based design of novel covalent FAAHi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXM0Vnrifb87Vg90H21EOLACvtfcHk0lhQ2y2NicYcRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsb3O&md5=3130e9ef929a52cc0900ad77b38a7453</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Fjm2004283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2004283%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DLodola%26aufirst%3DA.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DCovalent%2520inhibitors%2520of%2520fatty%2520acid%2520amide%2520hydrolase%253A%2520a%2520rationale%2520for%2520the%2520activity%2520of%2520piperidine%2520and%2520piperazine%2520aryl%2520ureas%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6612%26epage%3D6623%26doi%3D10.1021%2Fjm2004283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stenta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Peraro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Molecular simulations highlight the role of metals in catalysis and inhibition of type II topoisomerase</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">857</span><span class="NLM_x">–</span> <span class="NLM_lpage">862</span><span class="refDoi"> DOI: 10.1021/ct300691u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300691u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1Wiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=857-862&author=G.+Palermoauthor=M.+Stentaauthor=A.+Cavalliauthor=M.+Dal+Peraroauthor=M.+De+Vivo&title=Molecular+simulations+highlight+the+role+of+metals+in+catalysis+and+inhibition+of+type+II+topoisomerase&doi=10.1021%2Fct300691u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Simulations Highlight the Role of Metals in Catalysis and Inhibition of Type II Topoisomerase</span></div><div class="casAuthors">Palermo, Giulia; Stenta, Marco; Cavalli, Andrea; Dal Peraro, Matteo; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">857-862</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Type II topoisomerase (topoII) is a metalloenzyme targeted by clin. antibiotics and anticancer agents.  Here, we integrate existing structural data with mol. simulation and propose a model for the yet uncharacterized structure of the reactant state of topoII.  This model describes a canonical two-metal-ion mechanism and suggests how the metals could rearrange at the catalytic pocket during enzymic turnover, explaining also exptl. evidence for topoII inhibition.  These results call for further exptl. validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqWdNp-7sLNbVg90H21EOLACvtfcHk0lhQ2y2NicYcRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1Wiug%253D%253D&md5=3cd42ed1cc8929e2a3ca387751382460</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1021%2Fct300691u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300691u%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DStenta%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DDal%2BPeraro%26aufirst%3DM.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DMolecular%2520simulations%2520highlight%2520the%2520role%2520of%2520metals%2520in%2520catalysis%2520and%2520inhibition%2520of%2520type%2520II%2520topoisomerase%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D857%26epage%3D862%26doi%3D10.1021%2Fct300691u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Flexible docking in solution using metadynamics</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">2600</span><span class="NLM_x">–</span> <span class="NLM_lpage">2607</span><span class="refDoi"> DOI: 10.1021/ja0445950</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0445950" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVSntbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=2600-2607&author=F.+L.+Gervasioauthor=A.+Laioauthor=M.+Parrinello&title=Flexible+docking+in+solution+using+metadynamics&doi=10.1021%2Fja0445950"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible Docking in Solution Using Metadynamics</span></div><div class="casAuthors">Gervasio, Francesco Luigi; Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2600-2607</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We apply our recently developed metadynamics method to the docking of ligands on flexible receptors in water soln.  This method mimics the real dynamics of a ligand exiting or entering an enzyme and in so doing reconstructs the free energy surface.  We apply it to four docking cases: β-trypsin/benzamidine, β-trypsin/chlorobenzamidine, Ig McPC-603/phosphocholine, and cyclin-dependent kinase 2/staurosporine.  In every case studied, the method is able to predict the docked geometry and the free energy of docking.  Its added value with respect to many other available methods is that it reconstructs the complete free energy surface, including all the relevant min. and the barriers between them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbeUwtpDaXtrVg90H21EOLACvtfcHk0lhQ2y2NicYcRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVSntbw%253D&md5=b2e17e2ebb518d0754d8f6f693557820</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Fja0445950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0445950%26sid%3Dliteratum%253Aachs%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFlexible%2520docking%2520in%2520solution%2520using%2520metadynamics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D2600%26epage%3D2607%26doi%3D10.1021%2Fja0445950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group">Colizzi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perozzo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scapozza, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Single-molecule pulling simulations can discern active from inactive enzyme inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">7361</span><span class="NLM_x">–</span> <span class="NLM_lpage">7371</span><span class="refDoi"> DOI: 10.1021/ja100259r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja100259r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=7361-7371&author=F.+Colizziauthor=R.+Perozzoauthor=L.+Scapozzaauthor=M.+Recanatiniauthor=A.+Cavalli&title=Single-molecule+pulling+simulations+can+discern+active+from+inactive+enzyme+inhibitors&doi=10.1021%2Fja100259r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Single-Molecule Pulling Simulations Can Discern Active from Inactive Enzyme Inhibitors</span></div><div class="casAuthors">Colizzi, Francesco; Perozzo, Remo; Scapozza, Leonardo; Recanatini, Maurizio; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7361-7371</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding ligand-protein recognition and interaction processes is of primary importance for structure-based drug design.  Traditionally, several approaches combining docking and mol. dynamics (MD) simulations have been exploited to investigate the physicochem. properties of complexes of pharmaceutical interest.  Even if the geometric properties of a modeled protein-ligand complex can be well predicted by computational methods, it is challenging to rank a series of analogs in a consistent fashion with biol. data.  In the unique β-hydroxyacyl-ACP dehydratase of Plasmodium falciparum (PfFabZ), the application of std. mol. docking and MD simulations was partially sufficient to shed light on the activity of previously discovered inhibitors.  Complementing docking results with atomistic simulations in the steered mol. dynamics (SMD) framework, we devised an in silico approach to study mol. interactions and to compare the binding characteristics of ligand analogs.  We hypothesized an interaction model that both explained the biol. activity of known ligands, and provided insight into designing novel enzyme inhibitors.  Mimicking single-mol. pulling expts., we used SMD-derived force profiles to discern active from inactive compds. for the first time.  A new compd. was designed and its biol. activity toward the PfFabZ enzyme predicted.  Finally, the computational predictions were exptl. confirmed, highlighting the robustness of the drug design approach presented herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3wn1eZl92ErVg90H21EOLACvtfcHk0lhQ2y2NicYcRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtro%253D&md5=6c52680599d889ee992b5445bc37b498</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fja100259r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja100259r%26sid%3Dliteratum%253Aachs%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DPerozzo%26aufirst%3DR.%26aulast%3DScapozza%26aufirst%3DL.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSingle-molecule%2520pulling%2520simulations%2520can%2520discern%2520active%2520from%2520inactive%2520enzyme%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D7361%26epage%3D7371%26doi%3D10.1021%2Fja100259r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group">Bash, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, U. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, F. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Calculation of the relative change in binding free energy of a protein-inhibitor complex</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1126/science.3810157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.3810157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=3810157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL2sXitV2lt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=574-6&author=P.+A.+Bashauthor=U.+C.+Singhauthor=F.+K.+Brownauthor=R.+Langridgeauthor=P.+A.+Kollman&title=Calculation+of+the+relative+change+in+binding+free+energy+of+a+protein-inhibitor+complex&doi=10.1126%2Fscience.3810157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of the relative change in binding free energy of a protein-inhibitor complex</span></div><div class="casAuthors">Bash, Paul A.; Singh, U. Chandra; Brown, Frank K.; Langridge, Robert; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">4788</span>),
    <span class="NLM_cas:pages">574-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The relative free energy of binding was calcd. for thermolysin complexed with a pair of phosphonamidate and phosphonate ester inhibitors by a thermodn. perturbation method implemented with mol. dynamics.  The calcd. difference in free energy of binding was 4.21 kcal/mol.  This compares well with the exptl. value of 4.1 kcal/mol.  The method is general and can be used to det. a change or mutation in any system that can be suitably represented.  It is likely to prove useful for protein and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrco-HX_Uxaf7Vg90H21EOLACvtfcHk0li0TnYAGNKxrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXitV2lt74%253D&md5=159b4f27c4ded3b388e1ead2174981b1</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1126%2Fscience.3810157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.3810157%26sid%3Dliteratum%253Aachs%26aulast%3DBash%26aufirst%3DP.%2BA.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26aulast%3DBrown%26aufirst%3DF.%2BK.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DCalculation%2520of%2520the%2520relative%2520change%2520in%2520binding%2520free%2520energy%2520of%2520a%2520protein-inhibitor%2520complex%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D574%26epage%3D6%26doi%3D10.1126%2Fscience.3810157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Efficient drug lead discovery and optimization</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">733</span><span class="refDoi"> DOI: 10.1021/ar800236t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800236t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=724-733&author=W.+L.+Jorgensen&title=Efficient+drug+lead+discovery+and+optimization&doi=10.1021%2Far800236t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Drug Lead Discovery and Optimization</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">724-733</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  During the 1980s, advances in the abilities to perform computer simulations of chem. and biomol. systems and to calc. free energy changes led to the expectation that such methodol. would soon show great utility for guiding mol. design.  Important potential applications included design of selective receptors, catalysts, and regulators of biol. function including enzyme inhibitors.  This time also saw the rise of high-throughput screening and combinatorial chem. along with complementary computational methods for de novo design and virtual screening including docking.  These technologies appeared poised to deliver diverse lead compds. for any biol. target.  As with many technol. advances, realization of the expectations required significant addnl. effort and time.  However, as summarized here, striking success has now been achieved for computer-aided drug lead generation and optimization.  De novo design using both mol. growing and docking are illustrated for lead generation, and lead optimization features free energy perturbation calcns. in conjunction with Monte Carlo statistical mechanics simulations for protein-inhibitor complexes in aq. soln.  The specific applications are to the discovery of non-nucleoside inhibitors of HIV reverse transcriptase (HIV-RT) and inhibitors of the binding of the proinflammatory cytokine MIF to its receptor CD74.  A std. protocol is presented that includes scans for possible addns. of small substituents to a mol. core, interchange of heterocycles, and focused optimization of substituents at one site.  Initial leads with activities at low-micromolar concns. have been advanced rapidly to low-nanomolar inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0YpWuo1jCLVg90H21EOLACvtfcHk0li0TnYAGNKxrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D&md5=9641587769701c6541b38e18fa05538a</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1021%2Far800236t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800236t%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEfficient%2520drug%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2009%26volume%3D42%26spage%3D724%26epage%3D733%26doi%3D10.1021%2Far800236t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group">Wong, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccammon, J. A.</span><span> </span><span class="NLM_article-title">Dynamics and design of enzymes and inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">3830</span><span class="NLM_x">–</span> <span class="NLM_lpage">3832</span><span class="refDoi"> DOI: 10.1021/ja00273a048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00273a048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL28Xkt1Sru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=1986&pages=3830-3832&author=C.+F.+Wongauthor=J.+A.+Mccammon&title=Dynamics+and+design+of+enzymes+and+inhibitors&doi=10.1021%2Fja00273a048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics and design of enzymes and inhibitors</span></div><div class="casAuthors">Wong, Chung F.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3830-2</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Mol. dynamics computer simulations are analyzed by a new approach to predict the effects of structural modifications on enzyme-inhibitor binding in water.  The relative free energy of binding 2 different benzamidines to trypsin has been calcd., as has the relative free energy of binding benzamidine to native and mutant trypsin.  The agreement with exptl. results is encouraging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQCyux77gvDrVg90H21EOLACvtfcHk0li0TnYAGNKxrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28Xkt1Sru70%253D&md5=36009cf12837f48fe1c92224bcdfd763</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1021%2Fja00273a048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00273a048%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.%2BF.%26aulast%3DMccammon%26aufirst%3DJ.%2BA.%26atitle%3DDynamics%2520and%2520design%2520of%2520enzymes%2520and%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1986%26volume%3D108%26spage%3D3830%26epage%3D3832%26doi%3D10.1021%2Fja00273a048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group">Merz, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Free energy perturbation simulations of the inhibition of thermolysin: prediction of the free energy of binding of a new inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">5649</span><span class="NLM_x">–</span> <span class="NLM_lpage">5658</span><span class="refDoi"> DOI: 10.1021/ja00197a022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00197a022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL1MXkslelsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1989&pages=5649-5658&author=K.+M.+Merzauthor=P.+A.+Kollman&title=Free+energy+perturbation+simulations+of+the+inhibition+of+thermolysin%3A+prediction+of+the+free+energy+of+binding+of+a+new+inhibitor&doi=10.1021%2Fja00197a022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy perturbation simulations of the inhibition of thermolysin:  prediction of the free energy of binding of a new inhibitor</span></div><div class="casAuthors">Merz, Kenneth M., Jr.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5649-58</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Free energy perturbation simulations on a series of thermolysin inhibitors are reported.  The 3 compds. examd. were transition-state analogs and had the general structure, carbobenzoxy-Glyp(X)-L-Leu-L-Leu, where X = NH, O, and CH2.  Whereas the NH and O compds. have had their crystal structure solved as well as having their Ki detd., this is not true for the CH2 compd.  The simulations predicted the CH2 compd. to have a Ki similar to that of the NH compd. (ΔΔGbind = -0.3 kcal/mol favoring the NH compd.), whereas both had Ki greater than the O compd.  The reason why the CH2 compd. was a potent inhibitor could be traced to 2 factors: (1) the solvation free energy for the CH2 compd. was less than that for the NH and O compds. (hydrophobic effect) and (2) the CH2 compd. did not have a repulsive interaction between the main-chain carbonyl group, of alanine-113 and the O atom.  The NH → O simulations reported earlier were also redone using a new set of charges based on a more complete quantum mech. geometry optimization of models of the inhibitors.  These new charges led to a calcd. ΔΔG for NH → O of 5.9 kcal/mol, in contrast to the 4.2 kcal/mol calcd. earlier.  By mutating the new charges into the old, it was possible to more closely reproduce (ΔΔGcalc = 4.05 kcal/mol) previously calcd. results, albeit with different relative contributions of solvation and binding free energies.  Simulations were also carried out in which the force field parameter used for the neutral carboxylic acid X-C-O-H torsion was modified.  This was done because glutamate-143 interacts with the perturbation site, and hence it was expected that the smaller torsion barrier used previously (3.6 kcal/mol) might bias the calcn. considering that the exptl. value for this torsion was ∼10 kcal/mol.  In these simulations, which also used the new charges, the free energy value for the CH2 → NH perturbation changed very slightly (-0.3 vs. 0.0 kcal/mol), whereas for the NH → O perturbation there was a much larger change (5.9 vs. 3.3 kcal/mol).  Concomitantly, a significant change in the active site geometry between the 2 parameter sets was obsd., with the newer set matching the exptl. structure more closely.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJa5DNWccvwbVg90H21EOLACvtfcHk0ljboX2LFW0wCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkslelsLg%253D&md5=86d607d69ce4bd2a288edf6f2edd21ed</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1021%2Fja00197a022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00197a022%26sid%3Dliteratum%253Aachs%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DFree%2520energy%2520perturbation%2520simulations%2520of%2520the%2520inhibition%2520of%2520thermolysin%253A%2520prediction%2520of%2520the%2520free%2520energy%2520of%2520binding%2520of%2520a%2520new%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1989%26volume%3D111%26spage%3D5649%26epage%3D5658%26doi%3D10.1021%2Fja00197a022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group">Ferguson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radmer, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Determination of the relative binding free energies of peptide inhibitors to the HIV-1 protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2654</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1021/jm00112a048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00112a048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK3MXkvVCru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=2654-2549&author=D.+M.+Fergusonauthor=R.+J.+Radmerauthor=P.+A.+Kollman&title=Determination+of+the+relative+binding+free+energies+of+peptide+inhibitors+to+the+HIV-1+protease&doi=10.1021%2Fjm00112a048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of the relative binding free energies of peptide inhibitors to the HIV-1 protease</span></div><div class="casAuthors">Ferguson, David M.; Radmer, Randall J.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2654-9</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Mol. dynamics and free energy perturbation methods are used to det. the stereoselective binding of a potent modified peptide inhibitor to the HIV-1 protease.  The inhibitor, reported by (D. H. Rich et al., 1990) contains a key hydroxyl group that interacts with the catalytic aspartyl residues, mimicking the tetrahedral intermediate in hydrolysis.  The binding affinity, however, is dependent on the configuration of the chiral center to which this hydroxyl group is attached.  The relative binding free energy of the 2 diastereomers favor the S-configured hydroxyl inhibitor over the R form by 2.8 kcal/mol.  This is in agreement with the exptl. value of 2.6 kcal/mol that was measured simultaneously and independently of this study.  To further validate this model, the relative binding free energy of a third yet unreported inhibitor lacking the hydroxyl group was also detd.  The results predict a disease in binding affinity for this inhibitor in accord with other peptide inhibitors that do not contain the key hydroxyl group.  The calcns. also suggest that one of the two aspartyl residues (Asp125) is protonated when the inhibitors are bound.  The results indicate that the theor. model developed can be used reliably for the prediction of relative binding affinities and should be a useful tool for the design of viable anti-AIDS therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM53opFhNW5rVg90H21EOLACvtfcHk0ljboX2LFW0wCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXkvVCru78%253D&md5=4e55bb75bf3e6ff2c77ed9e635cd6fcc</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1021%2Fjm00112a048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00112a048%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26aulast%3DRadmer%26aufirst%3DR.%2BJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DDetermination%2520of%2520the%2520relative%2520binding%2520free%2520energies%2520of%2520peptide%2520inhibitors%2520to%2520the%2520HIV-1%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D2654%26epage%3D2549%26doi%3D10.1021%2Fjm00112a048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group">Rao, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilton, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, U. C.</span><span> </span><span class="NLM_article-title">Free energy perturbation studies on inhibitor binding to HIV-1 proteinase</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">4447</span><span class="NLM_x">–</span> <span class="NLM_lpage">4452</span><span class="refDoi"> DOI: 10.1021/ja00038a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00038a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1992&pages=4447-4452&author=B.+G.+Raoauthor=R.+F.+Tiltonauthor=U.+C.+Singh&title=Free+energy+perturbation+studies+on+inhibitor+binding+to+HIV-1+proteinase&doi=10.1021%2Fja00038a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1021%2Fja00038a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00038a001%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DB.%2BG.%26aulast%3DTilton%26aufirst%3DR.%2BF.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26atitle%3DFree%2520energy%2520perturbation%2520studies%2520on%2520inhibitor%2520binding%2520to%2520HIV-1%2520proteinase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1992%26volume%3D114%26spage%3D4447%26epage%3D4452%26doi%3D10.1021%2Fja00038a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group">Reddy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viswanadhan, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J. N.</span><span> </span><span class="NLM_article-title">Relative differences in the binding free energies of human immunodeficiency virus 1 protease inhibitors: a thermodynamic cycle-perturbation approach</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">10287</span><span class="NLM_x">–</span> <span class="NLM_lpage">10291</span><span class="refDoi"> DOI: 10.1073/pnas.88.22.10287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.88.22.10287" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=10287-10291&author=M.+R.+Reddyauthor=V.+N.+Viswanadhanauthor=J.+N.+Weinstein&title=Relative+differences+in+the+binding+free+energies+of+human+immunodeficiency+virus+1+protease+inhibitors%3A+a+thermodynamic+cycle-perturbation+approach&doi=10.1073%2Fpnas.88.22.10287"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1073%2Fpnas.88.22.10287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.88.22.10287%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DViswanadhan%26aufirst%3DV.%2BN.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26atitle%3DRelative%2520differences%2520in%2520the%2520binding%2520free%2520energies%2520of%2520human%2520immunodeficiency%2520virus%25201%2520protease%2520inhibitors%253A%2520a%2520thermodynamic%2520cycle-perturbation%2520approach%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1991%26volume%3D88%26spage%3D10287%26epage%3D10291%26doi%3D10.1073%2Fpnas.88.22.10287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupyan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wester, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">–</span> <span class="NLM_lpage">2703</span><span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+reliable+prediction+of+relative+ligand+binding+potency+in+prospective+drug+discovery+by+way+of+a+modern+free-energy+calculation+protocol+and+force+field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0ljboX2LFW0wCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520relative%2520ligand%2520binding%2520potency%2520in%2520prospective%2520drug%2520discovery%2520by%2520way%2520of%2520a%2520modern%2520free-energy%2520calculation%2520protocol%2520and%2520force%2520field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domaoal, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallardo-Macias, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Computationally-guided optimization of a docking hit to yield catechol diethers as potent anti-HIV agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8582</span><span class="NLM_x">–</span> <span class="NLM_lpage">8591</span><span class="refDoi"> DOI: 10.1021/jm201134m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201134m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8582-8591&author=M.+Bolliniauthor=R.+A.+Domaoalauthor=V.+V.+Thakurauthor=R.+Gallardo-Maciasauthor=K.+A.+Spasovauthor=K.+S.+Andersonauthor=W.+L.+Jorgensen&title=Computationally-guided+optimization+of+a+docking+hit+to+yield+catechol+diethers+as+potent+anti-HIV+agents&doi=10.1021%2Fjm201134m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally-Guided Optimization of a Docking Hit to Yield Catechol Diethers as Potent Anti-HIV Agents</span></div><div class="casAuthors">Bollini, Mariela; Domaoal, Robert A.; Thakur, Vinay V.; Gallardo-Macias, Ricardo; Spasov, Krasimir A.; Anderson, Karen S.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8582-8591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A 5-μM docking hit has been optimized to an extraordinarily potent (55 pM) non-nucleoside inhibitor of HIV reverse transcriptase.  Use of free energy perturbation (FEP) calcns. to predict relative free energies of binding aided the optimizations by identifying optimal substitution patterns for Ph rings and a linker.  The most potent resultant catechol diethers feature terminal uracil and cyanovinylphenyl groups.  A halogen bond with Pro95 likely contributes to the extreme potency of compd. 42 (I).  In addn., several examples are provided illustrating failures of attempted grafting of a substructure from a very active compd. onto a seemingly related scaffold to improve its activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjuw2rS4-u1LVg90H21EOLACvtfcHk0lhQCYtj2YOj3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7rE&md5=f06e165987ffa547f6262b4547bd9af6</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fjm201134m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201134m%26sid%3Dliteratum%253Aachs%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DDomaoal%26aufirst%3DR.%2BA.%26aulast%3DThakur%26aufirst%3DV.%2BV.%26aulast%3DGallardo-Macias%26aufirst%3DR.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DComputationally-guided%2520optimization%2520of%2520a%2520docking%2520hit%2520to%2520yield%2520catechol%2520diethers%2520as%2520potent%2520anti-HIV%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8582%26epage%3D8591%26doi%3D10.1021%2Fjm201134m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group">Lee, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallardo-Macias, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Picomolar inhibitors of HIV reverse transcriptase featuring bicyclic replacement of a cyanovinylphenyl group</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">16705</span><span class="NLM_x">–</span> <span class="NLM_lpage">16713</span><span class="refDoi"> DOI: 10.1021/ja408917n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja408917n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kktb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=16705-16713&author=W.+G.+Leeauthor=R.+Gallardo-Maciasauthor=K.+M.+Freyauthor=K.+A.+Spasovauthor=M.+Bolliniauthor=K.+S.+Andersonauthor=W.+L.+Jorgensen&title=Picomolar+inhibitors+of+HIV+reverse+transcriptase+featuring+bicyclic+replacement+of+a+cyanovinylphenyl+group&doi=10.1021%2Fja408917n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Picomolar Inhibitors of HIV Reverse Transcriptase Featuring Bicyclic Replacement of a Cyanovinylphenyl Group</span></div><div class="casAuthors">Lee, Won-Gil; Gallardo-Macias, Ricardo; Frey, Kathleen M.; Spasov, Krasimir A.; Bollini, Mariela; Anderson, Karen S.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">16705-16713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the catechol diether class are highly potent non-nucleoside inhibitors of HIV-1 reverse transcriptase (NNRTIs).  The most active compds. yield EC50 values below 0.5 nM in assays using human T-cells infected by wild-type HIV-1.  However, these compds. such as rilpivirine, the most recently FDA-approved NNRTI, bear a cyanovinylphenyl (CVP) group.  This is an uncommon substructure in drugs that gives reactivity concerns.  In the present work, computer simulations were used to design bicyclic replacements for the CVP group.  The predicted viability of a 2-cyanoindolizinyl alternative was confirmed exptl. and provided compds. with 0.4 nM activity against the wild-type virus.  The compds. also performed well with EC50 values of 10 nM against the challenging HIV-1 variant that contains the Lys103Asn/Tyr181Cys double mutation in the RT enzyme.  Indolyl and benzofuranyl analogs were also investigated; the most potent compds. in these cases have EC50 values toward wild-type HIV-1 near 10 nM and high-nanomolar activities toward the double-variant.  The structural expectations from the modeling were much enhanced by obtaining an X-ray crystal structure at 2.88 Å resoln. for the complex of the parent 2-cyanoindolizine I and HIV-1 RT.  The aq. solubilities of the most potent indolizine analogs were also measured to be ∼40 μg/mL, which is similar to that for the approved drug efavirenz and ∼1000-fold greater than for rilpivirine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2eJNNTzKRp7Vg90H21EOLACvtfcHk0lhQCYtj2YOj3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kktb7L&md5=2a92d73a9d550ab0338cbd98f81faff0</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fja408917n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja408917n%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%2BG.%26aulast%3DGallardo-Macias%26aufirst%3DR.%26aulast%3DFrey%26aufirst%3DK.%2BM.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DPicomolar%2520inhibitors%2520of%2520HIV%2520reverse%2520transcriptase%2520featuring%2520bicyclic%2520replacement%2520of%2520a%2520cyanovinylphenyl%2520group%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D16705%26epage%3D16713%26doi%3D10.1021%2Fja408917n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group">Frey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puleo, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span> </span><span class="NLM_article-title">Structure-based evaluation of non-nucleoside inhibitors with improved potency and solubility that target HIV reverse transcriptase variants</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2737</span><span class="NLM_x">–</span> <span class="NLM_lpage">2745</span><span class="refDoi"> DOI: 10.1021/jm501908a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501908a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2737-2745&author=K.+M.+Freyauthor=D.+E.+Puleoauthor=K.+A.+Spasovauthor=M.+Bolliniauthor=W.+L.+Jorgensenauthor=K.+S.+Anderson&title=Structure-based+evaluation+of+non-nucleoside+inhibitors+with+improved+potency+and+solubility+that+target+HIV+reverse+transcriptase+variants&doi=10.1021%2Fjm501908a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fjm501908a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501908a%26sid%3Dliteratum%253Aachs%26aulast%3DFrey%26aufirst%3DK.%2BM.%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26atitle%3DStructure-based%2520evaluation%2520of%2520non-nucleoside%2520inhibitors%2520with%2520improved%2520potency%2520and%2520solubility%2520that%2520target%2520HIV%2520reverse%2520transcriptase%2520variants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2737%26epage%3D2745%26doi%3D10.1021%2Fjm501908a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group">Dziedzic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cisneros, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danford, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, R. H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Design, Synthesis, and Protein Crystallography of Biaryltriazoles as Potent Tautomerase Inhibitors of Macrophage Migration Inhibitory Factor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2996</span><span class="NLM_x">–</span> <span class="NLM_lpage">3003</span><span class="refDoi"> DOI: 10.1021/ja512112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2996-3003&author=P.+Dziedzicauthor=J.+A.+Cisnerosauthor=M.+J.+Robertsonauthor=A.+A.+Hareauthor=N.+E.+Danfordauthor=R.+H.+G.+Baxterauthor=W.+L.+Jorgensen&title=Design%2C+Synthesis%2C+and+Protein+Crystallography+of+Biaryltriazoles+as+Potent+Tautomerase+Inhibitors+of+Macrophage+Migration+Inhibitory+Factor&doi=10.1021%2Fja512112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and protein crystallography of biaryltriazoles as potent tautomerase inhibitors of macrophage migration inhibitory factor</span></div><div class="casAuthors">Dziedzic, Pawel; Cisneros, Jose A.; Robertson, Michael J.; Hare, Alissa A.; Danford, Nadia E.; Baxter, Richard H. G.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2996-3003</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Optimization is reported for biaryltriazoles as inhibitors of the tautomerase activity of human macrophage migration inhibitory factor (MIF), a proinflammatory cytokine assocd. with numerous inflammatory diseases and cancer.  A combined approach was taken featuring org. synthesis, enzymic assaying, crystallog., and modeling including free-energy perturbation (FEP) calcns.  X-ray crystal structures for two inhibitors bound to MIF are reported and provided a basis for the modeling efforts.  The accommodation of the inhibitors in the binding site is striking with multiple hydrogen bonds and aryl-aryl interactions.  Addnl. modeling encouraged pursuit of 5-phenoxyquinolinyl analogs, which led to the very potent compd.  Activity was further enhanced by addn. of a fluorine atom adjacent to the phenolic hydroxyl group to strengthen a key hydrogen bond.  It is also shown that phys. properties of the compds. can be modulated by variation of solvent-exposed substituents.  Several of the compds. are likely the most potent known MIF tautomerase inhibitors; the most active ones are more than 1000-fold more active than the well-studied (R)-ISO-1 and more than 200-fold more active than the chromen-4-one Orita-13.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYFcj_0uhyz7Vg90H21EOLACvtfcHk0lhIlf2IpWktjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D&md5=2b5fea70b6db3cb39f7217c39995bd06</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1021%2Fja512112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512112j%26sid%3Dliteratum%253Aachs%26aulast%3DDziedzic%26aufirst%3DP.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DHare%26aufirst%3DA.%2BA.%26aulast%3DDanford%26aufirst%3DN.%2BE.%26aulast%3DBaxter%26aufirst%3DR.%2BH.%2BG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Protein%2520Crystallography%2520of%2520Biaryltriazoles%2520as%2520Potent%2520Tautomerase%2520Inhibitors%2520of%2520Macrophage%2520Migration%2520Inhibitory%2520Factor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2996%26epage%3D3003%26doi%3D10.1021%2Fja512112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group">Hare, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandavadi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournia, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Optimization of N-benzyl-benzoxazol-2-ones as receptor antagonists of macrophage migration inhibitory factor (MIF)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">5811</span><span class="NLM_x">–</span> <span class="NLM_lpage">5814</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.07.129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.bmcl.2010.07.129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20728358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGnt7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=5811-5814&author=A.+A.+Hareauthor=L.+Lengauthor=S.+Gandavadiauthor=X.+Duauthor=Z.+Courniaauthor=R.+Bucalaauthor=W.+L.+Jorgensen&title=Optimization+of+N-benzyl-benzoxazol-2-ones+as+receptor+antagonists+of+macrophage+migration+inhibitory+factor+%28MIF%29&doi=10.1016%2Fj.bmcl.2010.07.129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of N-benzyl-benzoxazol-2-ones as receptor antagonists of macrophage migration inhibitory factor (MIF)</span></div><div class="casAuthors">Hare, Alissa A.; Leng, Lin; Gandavadi, Sunilkumar; Du, Xin; Cournia, Zoe; Bucala, Richard; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5811-5814</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The cytokine MIF is involved in inflammation and cell proliferation via pathways initiated by its binding to the transmembrane receptor CD74.  MIF also exhibits keto-enol tautomerase activity, believed to be vestigial in mammals.  Starting from a 1 μM hit from virtual screening, substituted benzoxazol-2-ones have been discovered as antagonists with IC50 values as low as 7.5 nM in a tautomerase assay and 80 nM in a MIF-CD74 binding assay.  Addnl. studies for one of the potent inhibitors demonstrated that it is not a covalent inhibitor of MIF and that it attenuates MIF-dependent ERK1/2 phosphorylation in human synovial fibroblasts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_iLpA0sOWJ7Vg90H21EOLACvtfcHk0lhIlf2IpWktjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGnt7%252FP&md5=44b40f72065a7cd52582196dd640fecb</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.07.129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.07.129%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DA.%2BA.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DGandavadi%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DOptimization%2520of%2520N-benzyl-benzoxazol-2-ones%2520as%2520receptor%2520antagonists%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D5811%26epage%3D5814%26doi%3D10.1016%2Fj.bmcl.2010.07.129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group">Cournia, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandavadi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Discovery of human macrophage migration inhibitory factor (MIF)-CD74 Aantagonists via virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">424</span><span class="refDoi"> DOI: 10.1021/jm801100v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801100v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=416-424&author=Z.+Courniaauthor=L.+Lengauthor=S.+Gandavadiauthor=X.+Duauthor=R.+Bucalaauthor=W.+L.+Jorgensen&title=Discovery+of+human+macrophage+migration+inhibitory+factor+%28MIF%29-CD74+Aantagonists+via+virtual+screening&doi=10.1021%2Fjm801100v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1021%2Fjm801100v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801100v%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DGandavadi%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDiscovery%2520of%2520human%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529-CD74%2520Aantagonists%2520via%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D416%26epage%3D424%26doi%3D10.1021%2Fjm801100v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group">Acevedo, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrose, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aamer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sambasivarao, S. V.</span><span> </span><span class="NLM_article-title">Identification of HIV inhibitors guided by free energy perturbation calculations</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1216</span><span class="refDoi"> DOI: 10.2174/138161212799436421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F138161212799436421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22316150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1199-1216&author=O.+Acevedoauthor=Z.+Ambroseauthor=P.+T.+Flahertyauthor=H.+Aamerauthor=P.+Jainauthor=S.+V.+Sambasivarao&title=Identification+of+HIV+inhibitors+guided+by+free+energy+perturbation+calculations&doi=10.2174%2F138161212799436421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of HIV inhibitors guided by free energy perturbation calculations</span></div><div class="casAuthors">Acevedo, Orlando; Ambrose, Zandrea; Flaherty, Patrick T.; Aamer, Hadega; Jain, Prashi; Sambasivarao, Somisetti V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1199-1216</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Free energy perturbation (FEP) theory coupled to mol. dynamics (MD) or Monte Carlo (MC) statistical mechanics offers a theor. precise method for detg. the free energy differences of related biol. inhibitors.  Traditionally requiring extensive computational resources and expertise, it is only recently that its impact is being felt in drug discovery.  A review of computer-aided anti-HIV efforts employing FEP calcns. is provided here that describes early and recent successes in the design of human immunodeficiency virus type 1 (HIV-1) protease and non-nucleoside reverse transcriptase inhibitors.  In addn., our ongoing work developing and optimizing leads for small mol. inhibitors of cyclophilin A (CypA) is highlighted as an update on the current capabilities of the field.  CypA has been shown to aid HIV-1 replication by catalyzing the cis/trans isomerization of a conserved Gly-Pro motif in the N-terminal domain of HIV-1 capsid (CA) protein.  In the absence of a functional CypA, e.g., by the addn. of an inhibitor such as cyclosporine A (CsA), HIV-1 has reduced infectivity.  Our simulations of acylurea-based and 1-indanylketone-based CypA inhibitors have detd. that their nanomolar and micromolar binding affinities, resp., are tied to their ability to stabilize Arg55 and Asn102.  A structurally novel 1-(2,6-dichlorobenzamido) indole core was proposed to maximize these interactions.  FEP-guided optimization, exptl. synthesis, and biol. testing of lead compds. for toxicity and inhibition of wild-type HIV-1 and CA mutants have demonstrated a dose-dependent inhibition of HIV-1 infection in two cell lines.  While the inhibition is modest compared to CsA, the results are encouraging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuuljqaTqDGrVg90H21EOLACvtfcHk0ljcP5YrH-JRdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbw%253D&md5=b992dc0cfe8d4874e1f5fea9cb2faa5b</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.2174%2F138161212799436421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212799436421%26sid%3Dliteratum%253Aachs%26aulast%3DAcevedo%26aufirst%3DO.%26aulast%3DAmbrose%26aufirst%3DZ.%26aulast%3DFlaherty%26aufirst%3DP.%2BT.%26aulast%3DAamer%26aufirst%3DH.%26aulast%3DJain%26aufirst%3DP.%26aulast%3DSambasivarao%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520of%2520HIV%2520inhibitors%2520guided%2520by%2520free%2520energy%2520perturbation%2520calculations%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D1199%26epage%3D1216%26doi%3D10.2174%2F138161212799436421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group">Cole, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">966</span><span class="NLM_x">–</span> <span class="NLM_lpage">971</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2014.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.bbagen.2014.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25196360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=966-971&author=D.+J.+Coleauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Molecular+dynamics+and+Monte+Carlo+simulations+for+protein-ligand+binding+and+inhibitor+design&doi=10.1016%2Fj.bbagen.2014.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span></div><div class="casAuthors">Cole, Daniel J.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">966-971</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Non-nucleoside inhibitors of HIV reverse transcriptase are an important component of treatment against HIV infection.  Novel inhibitors are sought that increase potency against variants that contain the Tyr181Cys mutation.  Mol. dynamics based free energy perturbation simulations have been run to study factors that contribute to protein-ligand binding, and the results are compared with those from previous Monte Carlo based simulations and activity data.  Predictions of protein-ligand binding modes are very consistent for the two simulation methods; the accord is attributed to the use of an enhanced sampling protocol.  The Tyr181Cys binding pocket supports large, hydrophobic substituents, which is in good agreement with expt.  Although some discrepancies exist between the results of the two simulation methods and expt., free energy perturbation simulations can be used to rapidly test small mols. for gains in binding affinity.  Free energy perturbation methods show promise in providing fast, reliable and accurate data that can be used to complement expt. in lead optimization projects.  This article is part of a Special Issue entitled "Recent developments of mol. dynamics".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1gmI-9_FKtrVg90H21EOLACvtfcHk0ljcP5YrH-JRdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK&md5=2fd7025d2402b77ba6fb2e091733bc12</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DD.%2BJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DMolecular%2520dynamics%2520and%2520Monte%2520Carlo%2520simulations%2520for%2520protein-ligand%2520binding%2520and%2520inhibitor%2520design%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D966%26epage%3D971%26doi%3D10.1016%2Fj.bbagen.2014.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouzida, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swendsen, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, J. M.</span><span> </span><span class="NLM_article-title">The weighted histogram analysis method for free-energy calculations on biomolecules 0.1. The Method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1021</span><span class="refDoi"> DOI: 10.1002/jcc.540130812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.540130812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=1011-1021&author=S.+Kumarauthor=D.+Bouzidaauthor=R.+H.+Swendsenauthor=P.+A.+Kollmanauthor=J.+M.+Rosenberg&title=The+weighted+histogram+analysis+method+for+free-energy+calculations+on+biomolecules+0.1.+The+Method&doi=10.1002%2Fjcc.540130812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">The weighted histogram analysis method for free-energy calculations on biomolecules.  I.  The method</span></div><div class="casAuthors">Kumar, Shankar; Bouzida, Djamal; Swendsen, Robert H.; Kollman, Peter A.; Rosenberg, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1011-21</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The Weighted Histogram Anal. Method (WHAM), an extension of Ferrenberg and Swendsen's Multiple Histogram Technique, has been applied for the first time on complex biomol. Hamiltonians.  The method is presented here as an extension of the Umbrella Sampling method for free-energy and Potential of Mean Force calcns.  This algorithm possesses the following advantages over methods that are currently employed: (1) it provides a built-in est. of sampling errors thereby yielding objective ests. of the optimal location and length of addnl. simulations needed to achieve a desired level of precision; (2) it yields the "best" value of free energies by taking into account all the simulations so as to minimize the statistical errors; (3) in addn. to optimizing the links between simulations, it also allows multiple overlaps of probability distributions for obtaining better ests. of the free-energy differences.  By recasting the Ferrenberg-Swendsen Multiple Histogram equations in a form suitable for mol. mechanics type Hamiltonians, we have demonstrated the feasibility and robustness of this method by applying it to a test problem of the generation of the Potential of Mean Force profile of the pseudorotation phase angle of the sugar ring in deoxyadenosine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLayYt_QpVO7Vg90H21EOLACvtfcHk0ljcP5YrH-JRdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D&md5=5b2ad7410198f03025708a37c0fbe89d</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130812%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBouzida%26aufirst%3DD.%26aulast%3DSwendsen%26aufirst%3DR.%2BH.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520weighted%2520histogram%2520analysis%2520method%2520for%2520free-energy%2520calculations%2520on%2520biomolecules%25200.1.%2520The%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D1011%26epage%3D1021%26doi%3D10.1002%2Fjcc.540130812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group">Kastner, J.</span><span> </span><span class="NLM_article-title">Umbrella sampling</span> <span class="citation_source-journal">WIREs Comput. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">932</span><span class="NLM_x">–</span> <span class="NLM_lpage">942</span><span class="refDoi"> DOI: 10.1002/wcms.66</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fwcms.66" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=932-942&author=J.+Kastner&title=Umbrella+sampling&doi=10.1002%2Fwcms.66"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1002%2Fwcms.66&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.66%26sid%3Dliteratum%253Aachs%26aulast%3DKastner%26aufirst%3DJ.%26atitle%3DUmbrella%2520sampling%26jtitle%3DWIREs%2520Comput.%2520Mol.%2520Sci.%26date%3D2011%26volume%3D1%26spage%3D932%26epage%3D942%26doi%3D10.1002%2Fwcms.66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group">Comer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gumbart, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelievre, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohorille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipot, C.</span><span> </span><span class="NLM_article-title">The adaptive biasing force method: everything you always wanted to know but were afraid to ask</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1151</span><span class="refDoi"> DOI: 10.1021/jp506633n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp506633n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsF2nurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=1129-1151&author=J.+Comerauthor=J.+C.+Gumbartauthor=J.+Heninauthor=T.+Lelievreauthor=A.+Pohorilleauthor=C.+Chipot&title=The+adaptive+biasing+force+method%3A+everything+you+always+wanted+to+know+but+were+afraid+to+ask&doi=10.1021%2Fjp506633n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">The Adaptive Biasing Force Method: Everything You Always Wanted To Know but Were Afraid To Ask</span></div><div class="casAuthors">Comer, Jeffrey; Gumbart, James C.; Henin, Jerome; Lelievre, Tony; Pohorille, Andrew; Chipot, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1129-1151</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the host of numerical schemes devised to calc. free energy differences by way of geometric transformations, the adaptive biasing force algorithm has emerged as a promising route to map complex free-energy landscapes.  It relies upon the simple concept that as a simulation progresses, a continuously updated biasing force is added to the equations of motion, such that in the long-time limit it yields a Hamiltonian devoid of an av. force acting along the transition coordinate of interest.  This means that sampling proceeds uniformly on a flat free-energy surface, thus providing reliable free-energy ests.  Much of the appeal of the algorithm to the practitioner is in its phys. intuitive underlying ideas and the absence of any requirements for prior knowledge about free-energy landscapes.  Since its inception in 2001, the adaptive biasing force scheme has been the subject of considerable attention, from in-depth math. anal. of convergence properties to novel developments and extensions.  The method has also been successfully applied to many challenging problems in chem. and biol.  In this contribution, the method is presented in a comprehensive, self-contained fashion, discussing with a crit. eye its properties, applicability, and inherent limitations, as well as introducing novel extensions.  Through free-energy calcns. of prototypical mol. systems, many methodol. aspects are examd., from stratification strategies to overcoming the so-called hidden barriers in orthogonal space, relevant not only to the adaptive biasing force algorithm but also to other importance-sampling schemes.  On the basis of the discussions in this paper, a no. of good practices for improving the efficiency and reliability of the computed free-energy differences are proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUPvI90eVb-LVg90H21EOLACvtfcHk0ljZ6akMgYozIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsF2nurvP&md5=2ea7fa53559bf7355d24630fc126fcc9</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1021%2Fjp506633n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp506633n%26sid%3Dliteratum%253Aachs%26aulast%3DComer%26aufirst%3DJ.%26aulast%3DGumbart%26aufirst%3DJ.%2BC.%26aulast%3DHenin%26aufirst%3DJ.%26aulast%3DLelievre%26aufirst%3DT.%26aulast%3DPohorille%26aufirst%3DA.%26aulast%3DChipot%26aufirst%3DC.%26atitle%3DThe%2520adaptive%2520biasing%2520force%2520method%253A%2520everything%2520you%2520always%2520wanted%2520to%2520know%2520but%2520were%2520afraid%2520to%2520ask%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2015%26volume%3D119%26spage%3D1129%26epage%3D1151%26doi%3D10.1021%2Fjp506633n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group">Woo, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding free energy from computer simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">6825</span><span class="NLM_x">–</span> <span class="NLM_lpage">6830</span><span class="refDoi"> DOI: 10.1073/pnas.0409005102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0409005102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15867154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6825-6830&author=H.+J.+Wooauthor=B.+Roux&title=Calculation+of+absolute+protein-ligand+binding+free+energy+from+computer+simulations&doi=10.1073%2Fpnas.0409005102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding free energy from computer simulations</span></div><div class="casAuthors">Woo, Hyung-June; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6825-6830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A general methodol. for calcg. the equil. binding const. of a flexible ligand to a protein receptor is formulated on the basis of potentials of mean force.  The overall process is decompd. into several stages that can be computed sep.: the free ligand in the bulk is first restrained into the conformation it adopts in the bound state, position, and orientation by applying biasing potentials, then it is translated into the binding site, where it is released completely.  The conformational restraining potential is based on the root-mean-square deviation of the peptide coordinates relative to its av. conformation in the bound complex.  Free energy contributions from each stage are calcd. by free energy perturbation potential of mean force techniques by using appropriate order parameters.  The present approach avoids the need to decouple the ligand from its surrounding (bulk solvent and receptor protein) as is traditionally performed in the double-decoupling scheme.  It is believed that the present formulation will be particularly useful when the solvation free energy of the ligand is very large.  As an application, the equil. binding const. of the phosphotyrosine peptide pYEEI to the Src homol. 2 domain of human Lck has been calcd.  The results are in good agreement with exptl. values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktN0FgqFW_LVg90H21EOLACvtfcHk0ljZ6akMgYozIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D&md5=23be06c80a875efaff0207ea8dc2b08d</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409005102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409005102%26sid%3Dliteratum%253Aachs%26aulast%3DWoo%26aufirst%3DH.%2BJ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520free%2520energy%2520from%2520computer%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D6825%26epage%3D6830%26doi%3D10.1073%2Fpnas.0409005102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group">Ge, X. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute Binding Free Energy Calculations of Sparsomycin Analogs to the Bacterial Ribosome</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">9525</span><span class="NLM_x">–</span> <span class="NLM_lpage">9539</span><span class="refDoi"> DOI: 10.1021/jp100579y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp100579y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2010&pages=9525-9539&author=X.+X.+Geauthor=B.+Roux&title=Absolute+Binding+Free+Energy+Calculations+of+Sparsomycin+Analogs+to+the+Bacterial+Ribosome&doi=10.1021%2Fjp100579y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1021%2Fjp100579y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp100579y%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DX.%2BX.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520Binding%2520Free%2520Energy%2520Calculations%2520of%2520Sparsomycin%2520Analogs%2520to%2520the%2520Bacterial%2520Ribosome%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D9525%26epage%3D9539%26doi%3D10.1021%2Fjp100579y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group">Deng, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Computations of standard binding free energies with molecular dynamics simulations</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">2234</span><span class="NLM_x">–</span> <span class="NLM_lpage">2246</span><span class="refDoi"> DOI: 10.1021/jp807701h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp807701h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=2234-2246&author=Y.+Q.+Dengauthor=B.+Roux&title=Computations+of+standard+binding+free+energies+with+molecular+dynamics+simulations&doi=10.1021%2Fjp807701h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">Computations of Standard Binding Free Energies with Molecular Dynamics Simulations</span></div><div class="casAuthors">Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2234-2246</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An increasing no. of studies have reported computations of the std. (abs.) binding free energy of small ligands to proteins using mol. dynamics (MD) simulations and explicit solvent mols. that are in good agreement with expts.  This encouraging progress suggests that physics-based approaches hold the promise of making important contributions to the process of drug discovery and optimization in the near future.  Two types of approaches are principally used to compute binding free energies with MD simulations.  The most widely known is the alchem. double decoupling method, in which the interaction of the ligand with its surroundings are progressively switched off.  It is also possible to use a potential of mean force (PMF) method, in which the ligand is phys. sepd. from the protein receptor.  For both of these computational approaches, restraining potentials may be activated and released during the simulation for sampling efficiently the changes in translational, rotational, and conformational freedom of the ligand and protein upon binding.  Because such restraining potentials add bias to the simulations, it is important that their effects be rigorously removed to yield a binding free energy that is properly unbiased with respect to the std. state.  A review of recent results is presented, and differences in computational methods are discussed.  Examples of computations with T4-lysozyme mutants, FKBP12, SH2 domain, and cytochrome P 450 are discussed and compared.  Remaining difficulties and challenges are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG7gDgd4q9KrVg90H21EOLACvtfcHk0liOdjzNolb6gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFOqtA%253D%253D&md5=8a6759d9ce2a909c92d198fb8660f378</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Fjp807701h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp807701h%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DY.%2BQ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DComputations%2520of%2520standard%2520binding%2520free%2520energies%2520with%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D2234%26epage%3D2246%26doi%3D10.1021%2Fjp807701h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, M. A.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">864</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1529/biophysj.105.071589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1529%2Fbiophysj.105.071589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16284269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2006&pages=864-877&author=M.+S.+Leeauthor=M.+A.+Olson&title=Calculation+of+absolute+protein-ligand+binding+affinity+using+path+and+endpoint+approaches&doi=10.1529%2Fbiophysj.105.071589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span></div><div class="casAuthors">Lee, Michael S.; Olson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">864-877</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A comparative anal. is provided of rigorous and approx. methods for calcg. abs. binding affinities of two protein-ligand complexes: the FKBP protein bound with small mols. 4-hydroxy-2-butanone and FK506.  The authors' rigorous approach is an umbrella sampling technique where a potential of mean force is detd. by pulling the ligand out of the protein active site over several simulation windows.  The results of this approach agree well with exptl. obsd. binding affinities.  Also assessed is a commonly used approx. endpoint approach, which sep. ests. enthalpy, solvation free energy, and entropy.  The authors show that this endpoint approach has numerous variations, all of which are prone to crit. shortcomings.  For example, conventional harmonic and quasiharmonic entropy estn. procedures produce disparate results for the relatively simple protein-ligand systems studied in this work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAEk5VeXijpLVg90H21EOLACvtfcHk0liOdjzNolb6gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D&md5=3d25be5ba65e123249743538bbbf9e01</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.105.071589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.105.071589%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DOlson%26aufirst%3DM.%2BA.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520affinity%2520using%2520path%2520and%2520endpoint%2520approaches%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D90%26spage%3D864%26epage%3D877%26doi%3D10.1529%2Fbiophysj.105.071589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group">Doudou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R. H.</span><span> </span><span class="NLM_article-title">Standard free energy of binding from a one-dimensional potential of mean force</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">909</span><span class="NLM_x">–</span> <span class="NLM_lpage">918</span><span class="refDoi"> DOI: 10.1021/ct8002354</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct8002354" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2qt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=909-918&author=S.+Doudouauthor=N.+A.+Burtonauthor=R.+H.+Henchman&title=Standard+free+energy+of+binding+from+a+one-dimensional+potential+of+mean+force&doi=10.1021%2Fct8002354"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Standard Free Energy of Binding from a One-Dimensional Potential of Mean Force</span></div><div class="casAuthors">Doudou, Slimane; Burton, Neil A.; Henchman, Richard H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">909-918</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A practical approach that enables one to calc. the std. free energy of binding from a one-dimensional potential of mean force (PMF) is proposed.  Umbrella sampling and the weighted histogram anal. method are used to generate a PMF along the reaction coordinate of binding.  At each point, a restraint is applied orthogonal to the reaction coordinate to make possible the detn. of the vol. sampled by the ligand.  The free energy of binding from an arbitrary unbound vol. to the restrained bound form is calcd. from the ratio of the PMF integrated over the bound region to that of the unbound.  Adding the free energy changes from the std.-state vol. to the unbound vol. and from the restrained to the unrestrained bound state gives the std. free energy of binding.  Exploration of the best choice of binding paths is also made.  This approach is first demonstrated on a model binding system and then tested on the benzamidine-trypsin system for which reasonable agreement with expt. is found.  A comparison is made with other methods to obtain the std. free energy of binding from the PMF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-crj3Z8z_FLVg90H21EOLACvtfcHk0liOdjzNolb6gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2qt70%253D&md5=91aad6d99bd33da268c2b65a90220069</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1021%2Fct8002354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct8002354%26sid%3Dliteratum%253Aachs%26aulast%3DDoudou%26aufirst%3DS.%26aulast%3DBurton%26aufirst%3DN.%2BA.%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26atitle%3DStandard%2520free%2520energy%2520of%2520binding%2520from%2520a%2520one-dimensional%2520potential%2520of%2520mean%2520force%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D909%26epage%3D918%26doi%3D10.1021%2Fct8002354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group">Wojtas-Niziurski, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berneche, S.</span><span> </span><span class="NLM_article-title">Self-learning adaptive umbrella sampling method for the determination of free energy landscapes in multiple dimensions</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1885</span><span class="NLM_x">–</span> <span class="NLM_lpage">1895</span><span class="refDoi"> DOI: 10.1021/ct300978b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300978b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslGgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1885-1895&author=W.+Wojtas-Niziurskiauthor=Y.+L.+Mengauthor=B.+Rouxauthor=S.+Berneche&title=Self-learning+adaptive+umbrella+sampling+method+for+the+determination+of+free+energy+landscapes+in+multiple+dimensions&doi=10.1021%2Fct300978b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Learning Adaptive Umbrella Sampling Method for the Determination of Free Energy Landscapes in Multiple Dimensions</span></div><div class="casAuthors">Wojtas-Niziurski, Wojciech; Meng, Yilin; Roux, Benoit; Berneche, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1885-1895</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The potential of mean force describing conformational changes of biomols. is a central quantity for understanding the function of biomol. systems.  Calcg. an energy landscape of a process that depends on three or more reaction coordinates requires extensive computational power, making some multidimensional calcns. practically impossible.  Here, we present an efficient automatized umbrella sampling strategy for calcg. a multidimensional potential of mean force.  The method progressively learns by itself, through a feedback mechanism, which regions of a multidimensional space are worth exploring and automatically generates a set of umbrella sampling windows that is adapted to the system.  The self-learning adaptive umbrella sampling method is first explained with illustrative examples based on simplified reduced model systems and then applied to two nontrivial situations: the conformational equil. of the pentapeptide Met-enkephalin in soln. and ion permeation in the KcsA potassium channel.  With this method, it is demonstrated that a significant smaller no. of umbrella windows needs to be employed to characterize the free energy landscape over the most relevant regions without any loss in accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhBNxaUDWvq7Vg90H21EOLACvtfcHk0liOdjzNolb6gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslGgurw%253D&md5=cfccc621d18674ed5c3075a2a8ef633e</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1021%2Fct300978b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300978b%26sid%3Dliteratum%253Aachs%26aulast%3DWojtas-Niziurski%26aufirst%3DW.%26aulast%3DMeng%26aufirst%3DY.%2BL.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DBerneche%26aufirst%3DS.%26atitle%3DSelf-learning%2520adaptive%2520umbrella%2520sampling%2520method%2520for%2520the%2520determination%2520of%2520free%2520energy%2520landscapes%2520in%2520multiple%2520dimensions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1885%26epage%3D1895%26doi%3D10.1021%2Fct300978b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group">Jarzynski, C.</span><span> </span><span class="NLM_article-title">Nonequilibrium equality for free energy differences</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">2690</span><span class="NLM_x">–</span> <span class="NLM_lpage">2693</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.78.2690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.78.2690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1997&pages=2690-2693&author=C.+Jarzynski&title=Nonequilibrium+equality+for+free+energy+differences&doi=10.1103%2FPhysRevLett.78.2690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Nonequilibrium equality for free energy differences</span></div><div class="casAuthors">Jarzynski, C.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2690-2693</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">An expression is derived for the equil. free energy difference between two configurations of a system, in terms of an ensemble of finite-time measurements of the work performed in parametrically switching from one configuration to the other.  Two well-known identities emerge as limiting cases of this result.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl50isjLbR7Vg90H21EOLACvtfcHk0liPpmJGVlzuTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D&md5=5e572f64b7e9044fff5af96867602877</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.78.2690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.78.2690%26sid%3Dliteratum%253Aachs%26aulast%3DJarzynski%26aufirst%3DC.%26atitle%3DNonequilibrium%2520equality%2520for%2520free%2520energy%2520differences%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1997%26volume%3D78%26spage%3D2690%26epage%3D2693%26doi%3D10.1103%2FPhysRevLett.78.2690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khalili-Araghi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tajkhorshid, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">Free energy calculation from steered molecular dynamics simulations using Jarzynski’s equality</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">3559</span><span class="NLM_x">–</span> <span class="NLM_lpage">3566</span><span class="refDoi"> DOI: 10.1063/1.1590311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.1590311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvVWktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2003&pages=3559-3566&author=S.+Parkauthor=F.+Khalili-Araghiauthor=E.+Tajkhorshidauthor=K.+Schulten&title=Free+energy+calculation+from+steered+molecular+dynamics+simulations+using+Jarzynski%E2%80%99s+equality&doi=10.1063%2F1.1590311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculation from steered molecular dynamics simulations using Jarzynski's equality</span></div><div class="casAuthors">Park, Sanghyun; Khalili-Araghi, Fatemeh; Tajkhorshid, Emad; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3559-3566</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Jarzynski's equality is applied to free energy calcns. from steered mol. dynamics simulations of biomols.  The helix-coil transition of deca-alanine in vacuum is used as an example.  With about ten trajectories sampled, the second order cumulant expansion, among the various averaging schemes examd., yields the most accurate ests.  We compare umbrella sampling and the present method, and find that their efficiencies are comparable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8HS8jTFZiF7Vg90H21EOLACvtfcHk0liPpmJGVlzuTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvVWktr0%253D&md5=936e78b8aef1173f864b055ee4030d9c</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1063%2F1.1590311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1590311%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.%26aulast%3DKhalili-Araghi%26aufirst%3DF.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DFree%2520energy%2520calculation%2520from%2520steered%2520molecular%2520dynamics%2520simulations%2520using%2520Jarzynski%25E2%2580%2599s%2520equality%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2003%26volume%3D119%26spage%3D3559%26epage%3D3566%26doi%3D10.1063%2F1.1590311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group">Xiong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crespo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estrin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Free energy calculations with non-equilibrium methods: applications of the Jarzynski relationship</span> <span class="citation_source-journal">Theor. Chem. Acc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1007/s00214-005-0072-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs00214-005-0072-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28Xns1Sqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2006&pages=338-346&author=H.+Xiongauthor=A.+Crespoauthor=M.+Martiauthor=D.+Estrinauthor=A.+E.+Roitberg&title=Free+energy+calculations+with+non-equilibrium+methods%3A+applications+of+the+Jarzynski+relationship&doi=10.1007%2Fs00214-005-0072-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">Free Energy Calculations with Non-Equilibrium Methods: Applications of the Jarzynski Relationship</span></div><div class="casAuthors">Xiong, Hui; Crespo, Alejandro; Marti, Marcelo; Estrin, Dario; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Chemistry Accounts</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">338-346</span>CODEN:
                <span class="NLM_cas:coden">TCACFW</span>;
        ISSN:<span class="NLM_cas:issn">1432-881X</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  We present an introduction to the Jarzynski relationship that makes a strong connection, for a thermodn. transformation, between the distribution of non-equil. work values and the corresponding equil. free energy differences.  The relationship is discussed in the context of sampling issues, high level parallel computing, and convergence criteria.  We discuss three different applications by our group: mech. unfolding of peptides, mixed quantum/classical free energy calcns. in enzymes, and ligand escape pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiA4kHcfLKxrVg90H21EOLACvtfcHk0liPpmJGVlzuTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xns1Sqtbo%253D&md5=737d6e658c90905c93a42784fd861e60</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1007%2Fs00214-005-0072-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00214-005-0072-2%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DCrespo%26aufirst%3DA.%26aulast%3DMarti%26aufirst%3DM.%26aulast%3DEstrin%26aufirst%3DD.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DFree%2520energy%2520calculations%2520with%2520non-equilibrium%2520methods%253A%2520applications%2520of%2520the%2520Jarzynski%2520relationship%26jtitle%3DTheor.%2520Chem.%2520Acc.%26date%3D2006%26volume%3D116%26spage%3D338%26epage%3D346%26doi%3D10.1007%2Fs00214-005-0072-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group">Patel, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berteotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ronsisvalle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Steered molecular dynamics simulations for studying protein-ligand interaction in cyclin-dependent Kinase 5</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span><span class="refDoi"> DOI: 10.1021/ci4003574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4003574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt1yrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=470-480&author=J.+S.+Patelauthor=A.+Berteottiauthor=S.+Ronsisvalleauthor=W.+Rocchiaauthor=A.+Cavalli&title=Steered+molecular+dynamics+simulations+for+studying+protein-ligand+interaction+in+cyclin-dependent+Kinase+5&doi=10.1021%2Fci4003574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">Steered molecular dynamics simulations for studying protein-ligand interaction in cyclin-dependent kinase 5</span></div><div class="casAuthors">Patel, Jagdish Suresh; Berteotti, Anna; Ronsisvalle, Simone; Rocchia, Walter; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-480</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, the authors applied steered mol. dynamics (SMD) simulations to investigate the unbinding mechanism of 9 inhibitors of cyclin-dependent kinase 5 (CDK5).  The study had 2 major objectives: (1) to create a correlation between the unbinding force profiles and the inhibition activities of these compds. expressed as IC50 values; and (2) to investigate the unbinding mechanism and to reveal atomistic insights, which could help identify accessory binding sites and transient interactions.  Overall, the authors carried out 1.35 μs of cumulative SMD simulations.  The authors showed that SMD could qual. discriminate binders from nonbinders, while it failed to properly rank series of inhibitors, particularly when the IC50 values were too similar.  From a mechanistic standpoint, SMD provided useful insights related to transient and dynamical interactions, which could complement static description obtained by X-ray crystallog. expts.  Thus, the present study represents a further step toward a systematic exploitation of SMD and other dynamical approaches in structure-based drug design and computational medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdkn6WGwb0P7Vg90H21EOLACvtfcHk0liPpmJGVlzuTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt1yrug%253D%253D&md5=0977c6b933d23f5a33abb7a5c60f6b31</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fci4003574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4003574%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DJ.%2BS.%26aulast%3DBerteotti%26aufirst%3DA.%26aulast%3DRonsisvalle%26aufirst%3DS.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSteered%2520molecular%2520dynamics%2520simulations%2520for%2520studying%2520protein-ligand%2520interaction%2520in%2520cyclin-dependent%2520Kinase%25205%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D470%26epage%3D480%26doi%3D10.1021%2Fci4003574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minniti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greco, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riccardi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simoni, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Convertino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchetti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minarini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">An optimized polyamine moiety boosts the potency of human type II topoisomerase poisons as quantified by comparative analysis centered on the clinical candidate F14512</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">14310</span><span class="NLM_x">–</span> <span class="NLM_lpage">14313</span><span class="refDoi"> DOI: 10.1039/C5CC05065K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1039%2FC5CC05065K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26234198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=14310-14313&author=G.+Palermoauthor=E.+Minnitiauthor=M.+L.+Grecoauthor=L.+Riccardiauthor=E.+Simoniauthor=M.+Convertinoauthor=C.+Marchettiauthor=M.+Rosiniauthor=C.+Sissiauthor=A.+Minariniauthor=M.+De+Vivo&title=An+optimized+polyamine+moiety+boosts+the+potency+of+human+type+II+topoisomerase+poisons+as+quantified+by+comparative+analysis+centered+on+the+clinical+candidate+F14512&doi=10.1039%2FC5CC05065K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">An optimized polyamine moiety boosts the potency of human type II topoisomerase poisons as quantified by comparative analysis centered on the clinical candidate F14512</span></div><div class="casAuthors">Palermo, Giulia; Minniti, Elirosa; Greco, Maria Laura; Riccardi, Laura; Simoni, Elena; Convertino, Marino; Marchetti, Chiara; Rosini, Michela; Sissi, Claudia; Minarini, Anna; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">76</span>),
    <span class="NLM_cas:pages">14310-14313</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Combined computational-exptl. analyses explain and quantify the spermine-vectorized F14512's boosted potency as a topoII poison.  We found that an optimized polyamine moiety boosts drug binding to the topoII/DNA cleavage complex, rather than to the DNA alone.  These results provide new structural bases and key ref. data for designing new human topoII poisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBiw33UPksALVg90H21EOLACvtfcHk0lhHwxY9iFVXkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3K&md5=7a45fe86d3796d99a63e996206e509ca</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1039%2FC5CC05065K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC05065K%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DMinniti%26aufirst%3DE.%26aulast%3DGreco%26aufirst%3DM.%2BL.%26aulast%3DRiccardi%26aufirst%3DL.%26aulast%3DSimoni%26aufirst%3DE.%26aulast%3DConvertino%26aufirst%3DM.%26aulast%3DMarchetti%26aufirst%3DC.%26aulast%3DRosini%26aufirst%3DM.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DMinarini%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DAn%2520optimized%2520polyamine%2520moiety%2520boosts%2520the%2520potency%2520of%2520human%2520type%2520II%2520topoisomerase%2520poisons%2520as%2520quantified%2520by%2520comparative%2520analysis%2520centered%2520on%2520the%2520clinical%2520candidate%2520F14512%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D14310%26epage%3D14313%26doi%3D10.1039%2FC5CC05065K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group">Patel, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Insights into ligand-protein binding from local mechanical response</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3368</span><span class="NLM_x">–</span> <span class="NLM_lpage">3378</span><span class="refDoi"> DOI: 10.1021/ct200324j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200324j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOit7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=3368-3378&author=J.+S.+Patelauthor=D.+Branduardiauthor=M.+Masettiauthor=W.+Rocchiaauthor=A.+Cavalli&title=Insights+into+ligand-protein+binding+from+local+mechanical+response&doi=10.1021%2Fct200324j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into Ligand-Protein Binding from Local Mechanical Response</span></div><div class="casAuthors">Patel, Jagdish Suresh; Branduardi, Davide; Masetti, Matteo; Rocchia, Walter; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3368-3378</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational studies of ligand-protein binding are crucial for properly designing novel compds. of potential pharmacol. interest.  In this respect, researchers are increasingly interested in steered mol. dynamics for ligand-protein binding and unbinding studies.  In particular, it has been suggested that analyzing the work profiles along the ligand-protein undocking paths could be fruitful.  Here, the authors propose that small portions of work profiles, termed "local mech. responses" of the system to a steering force, could serve as a universal measure for capturing relevant information about the system under investigation.  Specifically, the authors first collected a high no. of steering trajectories using two biol. systems of increasing complexity (i.e., alanine dipeptide and (R)-roscovitine/CDK5 complex).  Then, the authors devised a novel postprocessing tool to be applied to the local mech. responses, to ext. structural information related to the biol. processes under investigation.  Despite the out-of-equil. character of the trajectories, the anal. carried out on the work profiles provided pivotal information about the investigated biol. processes.  This could eventually be applied to drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtLwG--YFzZLVg90H21EOLACvtfcHk0lhHwxY9iFVXkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOit7vO&md5=298c6a91bdeb995a8259a8e037fea02c</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1021%2Fct200324j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200324j%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DJ.%2BS.%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DInsights%2520into%2520ligand-protein%2520binding%2520from%2520local%2520mechanical%2520response%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D3368%26epage%3D3378%26doi%3D10.1021%2Fct200324j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group">Ensing, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Metadynamics as a tool for exploring free energy landscapes of chemical reactions</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1021/ar040198i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar040198i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1emsLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=73-81&author=B.+Ensingauthor=M.+De+Vivoauthor=Z.+W.+Liuauthor=P.+Mooreauthor=M.+L.+Klein&title=Metadynamics+as+a+tool+for+exploring+free+energy+landscapes+of+chemical+reactions&doi=10.1021%2Far040198i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">Metadynamics as a Tool for Exploring Free Energy Landscapes of Chemical Reactions</span></div><div class="casAuthors">Ensing, Bernd; De Vivo, Marco; Liu, Zhiwei; Moore, Preston; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">73-81</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The metadynamics or hills method is a relatively new mol. dynamics technique aimed to enhance the sampling of sepd. regions in phase space and map out the underlying free energy landscape as a function of a small no. of order parameters or collective variables.  The high efficiency allows for the application of metadynamics in combination with first principles dynamics methods, in particular with Car-Parrinello mol. dynamics, to study processes in which changes in the electronic structure play a dominant role, such as chem. reactions.  The option to choose several independent collective variables is important to tackle complex and concerted transformations that lack an obvious a priori choice for a single reaction coordinate.  In this Account, we discuss the role of metadynamics in the search of transition states, local min., reaction paths, free energy profiles, and reaction coordinates among a growing list of alternative methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvHygCAl8Vm7Vg90H21EOLACvtfcHk0litt45KD1Zypw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1emsLbI&md5=74760985582071d13d2cfeef9c25b777</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1021%2Far040198i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far040198i%26sid%3Dliteratum%253Aachs%26aulast%3DEnsing%26aufirst%3DB.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%2BW.%26aulast%3DMoore%26aufirst%3DP.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DMetadynamics%2520as%2520a%2520tool%2520for%2520exploring%2520free%2520energy%2520landscapes%2520of%2520chemical%2520reactions%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2006%26volume%3D39%26spage%3D73%26epage%3D81%26doi%3D10.1021%2Far040198i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Equilibrium free energies from nonequilibrium metadynamics</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">090601</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.96.090601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.96.090601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16606249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlSis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2006&pages=090601&author=G.+Bussiauthor=A.+Laioauthor=M.+Parrinello&title=Equilibrium+free+energies+from+nonequilibrium+metadynamics&doi=10.1103%2FPhysRevLett.96.090601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrium Free Energies from Nonequilibrium Metadynamics</span></div><div class="casAuthors">Bussi, Giovanni; Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">090601/1-090601/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We propose a new formalism to map history-dependent metadynamics in a Markovian process.  We apply this formalism to model Langevin dynamics and det. the equil. distribution of a collection of simulations.  We demonstrate that the reconstructed free energy is an unbiased est. of the underlying free energy and anal. derive an expression for the error.  The present results can be applied to other history-dependent stochastic processes, such as Wang-Landau sampling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe2rBjNl6IJrVg90H21EOLACvtfcHk0litt45KD1Zypw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlSis7o%253D&md5=b4cf2dd6399f8cd58be36692776c38e5</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.96.090601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.96.090601%26sid%3Dliteratum%253Aachs%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DEquilibrium%2520free%2520energies%2520from%2520nonequilibrium%2520metadynamics%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2006%26volume%3D96%26spage%3D090601%26doi%3D10.1103%2FPhysRevLett.96.090601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez-Fortea, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccarelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Assessing the accuracy of metadynamics</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">6714</span><span class="NLM_x">–</span> <span class="NLM_lpage">6721</span><span class="refDoi"> DOI: 10.1021/jp045424k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp045424k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ygurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2005&pages=6714-6721&author=A.+Laioauthor=A.+Rodriguez-Forteaauthor=F.+L.+Gervasioauthor=M.+Ceccarelliauthor=M.+Parrinello&title=Assessing+the+accuracy+of+metadynamics&doi=10.1021%2Fjp045424k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Accuracy of Metadynamics</span></div><div class="casAuthors">Laio, Alessandro; Rodriguez-Fortea, Antonio; Gervasio, Francesco Luigi; Ceccarelli, Matteo; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6714-6721</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metadynamics is a powerful technique that has been successfully exploited to explore the multidimensional free energy surface of complex polyat. systems and predict transition mechanisms in very different fields, ranging from chem. and solid-state physics to biophysics.  We here derive an explicit expression for the accuracy of the methodol. and provide a way to choose the parameters of the method in order to optimize its performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp1Nu73t30sbVg90H21EOLACvtfcHk0litt45KD1Zypw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ygurs%253D&md5=f0946da733889a7e05d4f97f6608468b</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1021%2Fjp045424k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp045424k%26sid%3Dliteratum%253Aachs%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DRodriguez-Fortea%26aufirst%3DA.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DCeccarelli%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DAssessing%2520the%2520accuracy%2520of%2520metadynamics%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2005%26volume%3D109%26spage%3D6714%26epage%3D6721%26doi%3D10.1021%2Fjp045424k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Metadynamics: a method to simulate rare events and reconstruct the free energy in biophysics, chemistry and material science</span> <span class="citation_source-journal">Rep. Prog. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">126601</span><span class="refDoi"> DOI: 10.1088/0034-4885/71/12/126601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1088%2F0034-4885%2F71%2F12%2F126601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=126601&author=A.+Laioauthor=F.+L.+Gervasio&title=Metadynamics%3A+a+method+to+simulate+rare+events+and+reconstruct+the+free+energy+in+biophysics%2C+chemistry+and+material+science&doi=10.1088%2F0034-4885%2F71%2F12%2F126601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Metadynamics: a method to stimulate rare events and reconstruct the free energy in biophysics, chemistry and material science</span></div><div class="casAuthors">Laio, Alessandro; Gervasio, Francesco L.</div><div class="citationInfo"><span class="NLM_cas:title">Reports on Progress in Physics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">126601/1-126601/22</span>CODEN:
                <span class="NLM_cas:coden">RPPHAG</span>;
        ISSN:<span class="NLM_cas:issn">0034-4885</span>.
    
            (<span class="NLM_cas:orgname">Institute of Physics Publishing</span>)
        </div><div class="casAbstract">A review.  Metadynamics is a powerful algorithm that can be used both for reconstructing the free energy and for accelerating rare events in systems described by complex Hamiltonians, at the classical or at the quantum level.  In the algorithm the normal evolution of the system is biased by a history-dependent potential constructed as a sum of Gaussians centered along the trajectory followed by a suitably chosen set of collective variables.  The sum of Gaussians is exploited for reconstructing iteratively an estimator of the free energy and forcing the system to escape from local min.  This review is intended to provide a comprehensive description of the algorithm, with a focus on the practical aspects that need to be addressed when one attempts to apply metadynamics to a new system: (i) the choice of the appropriate set of collective variables; (ii) the optimal choice of the metadynamics parameters and (iii) how to control the error and ensure convergence of the algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTaRxtUYS9_7Vg90H21EOLACvtfcHk0lg7WMobStk8Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyntrk%253D&md5=cd84cfc103f97c7d7ccf09fc434e2478</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1088%2F0034-4885%2F71%2F12%2F126601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1088%252F0034-4885%252F71%252F12%252F126601%26sid%3Dliteratum%253Aachs%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DMetadynamics%253A%2520a%2520method%2520to%2520simulate%2520rare%2520events%2520and%2520reconstruct%2520the%2520free%2520energy%2520in%2520biophysics%252C%2520chemistry%2520and%2520material%2520science%26jtitle%3DRep.%2520Prog.%2520Phys.%26date%3D2008%26volume%3D71%26spage%3D126601%26doi%3D10.1088%2F0034-4885%2F71%2F12%2F126601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitaleri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saladino, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Investigating Drug-Target Association and dissociation mechanisms using metadynamics-based algorithms</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span><span class="refDoi"> DOI: 10.1021/ar500356n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500356n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nt7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=277-285&author=A.+Cavalliauthor=A.+Spitaleriauthor=G.+Saladinoauthor=F.+L.+Gervasio&title=Investigating+Drug-Target+Association+and+dissociation+mechanisms+using+metadynamics-based+algorithms&doi=10.1021%2Far500356n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Investigating Drug-Target Association and Dissociation Mechanisms Using Metadynamics-Based Algorithms</span></div><div class="casAuthors">Cavalli, Andrea; Spitaleri, Andrea; Saladino, Giorgio; Gervasio, Francesco L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">277-285</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This Account highlights recent advances and discusses major challenges in the field of drug-target recognition, binding, and unbinding studied using metadynamics-based approaches, with particular emphasis on their role in structure-based design.  Computational chem. has significantly contributed to drug design and optimization in an extremely broad range of areas, including prediction of target druggability and drug likeness, de novo design, fragment screening, ligand docking, estn. of binding affinity, and modulation of ADMET (absorption, distribution, metab., excretion, toxicity) properties.  Computationally driven drug discovery must continuously adapt to keep pace with the evolving knowledge of the factors that modulate the pharmacol. action of drugs.  There is thus an urgent need for novel computational approaches that integrate the vast amt. of complex information currently available for small (bio)org. compds., biol. relevant targets and their complexes, while also accounting accurately for the thermodn. and kinetics of drug-target assocn., the intrinsic dynamical behavior of biomol. systems, and the complexity of protein-protein networks.  Understanding the mechanism of drug binding to and unbinding from biol. targets is fundamental for optimizing lead compds. and designing novel biol. active ones.  One major challenge is the accurate description of the conformational complexity prior to and upon formation of drug-target complexes.  Recently, enhanced sampling methods, including metadynamics and related approaches, have been successfully applied to investigate complex mechanisms of drugs binding to flexible targets.  Metadynamics is a family of enhanced sampling techniques aimed at enhancing the rare events and reconstructing the underlying free energy landscape as a function of a set of order parameters, usually referred to as collective variables.  Studies of drug binding mechanisms have predicted the most probable assocn. and dissocn. pathways and the related binding free energy profile.  In addn., the availability of an efficient open-source implementation, running on cost-effective GPU (i.e., graphical processor unit) architectures, has considerably decreased the learning curve and the computational costs of the methods, and increased their adoption by the community.  Here, we review the recent contributions of metadynamics and other enhanced sampling methods to the field of drug-target recognition and binding.  We discuss how metadynamics has been used to search for transition states, to predict binding and unbinding paths, to treat conformational flexibility, and to compute free energy profiles.  We highlight the importance of such predictions in drug discovery.  Major challenges in the field and possible solns. will finally be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_m41iVArELVg90H21EOLACvtfcHk0lg7WMobStk8Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nt7zL&md5=e3b79a23aca3a58914eb0bbf4d73306e</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1021%2Far500356n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500356n%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DSpitaleri%26aufirst%3DA.%26aulast%3DSaladino%26aufirst%3DG.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DInvestigating%2520Drug-Target%2520Association%2520and%2520dissociation%2520mechanisms%2520using%2520metadynamics-based%2520algorithms%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D277%26epage%3D285%26doi%3D10.1021%2Far500356n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group">Barducci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Well-tempered metadynamics: A smoothly converging and tunable free-energy method</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">020603</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.100.020603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.100.020603" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18232845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2008&pages=020603&author=A.+Barducciauthor=G.+Bussiauthor=M.+Parrinello&title=Well-tempered+metadynamics%3A+A+smoothly+converging+and+tunable+free-energy+method&doi=10.1103%2FPhysRevLett.100.020603"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">Well-Tempered Metadynamics: A Smoothly Converging and Tunable Free-Energy Method</span></div><div class="casAuthors">Barducci, Alessandro; Bussi, Giovanni; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">020603/1-020603/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We present a method for detg. the free-energy dependence on a selected no. of collective variables using an adaptive bias.  The formalism provides a unified description which has metadynamics and canonical sampling as limiting cases.  Convergence and errors can be rigorously and easily controlled.  The parameters of the simulation can be tuned so as to focus the computational effort only on the phys. relevant regions of the order parameter space.  The algorithm is tested on the reconstruction of an alanine dipeptide free-energy landscape.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrul1Eh_nSvy7Vg90H21EOLACvtfcHk0lg7WMobStk8Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFensQ%253D%253D&md5=701ccfeee476c2e9a5d1e5a6b0e82197</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.100.020603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.100.020603%26sid%3Dliteratum%253Aachs%26aulast%3DBarducci%26aufirst%3DA.%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DWell-tempered%2520metadynamics%253A%2520A%2520smoothly%2520converging%2520and%2520tunable%2520free-energy%2520method%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2008%26volume%3D100%26spage%3D020603%26doi%3D10.1103%2FPhysRevLett.100.020603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Metadynamics with Adaptive Gaussians</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2247</span><span class="NLM_x">–</span> <span class="NLM_lpage">2254</span><span class="refDoi"> DOI: 10.1021/ct3002464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct3002464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2247-2254&author=D.+Branduardiauthor=G.+Bussiauthor=M.+Parrinello&title=Metadynamics+with+Adaptive+Gaussians&doi=10.1021%2Fct3002464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Metadynamics with Adaptive Gaussians</span></div><div class="casAuthors">Branduardi, Davide; Bussi, Giovanni; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2247-2254</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metadynamics is an established sampling method aimed at reconstructing the free-energy surface relative to a set of appropriately chosen collective variables.  In std. metadynamics, the free-energy surface is filled by the addn. of Gaussian potentials of preassigned and typically diagonal covariance.  Asymptotically the free-energy surface is proportional to the bias deposited.  Here, we consider the possibility of using Gaussians whose variance is adjusted on the fly to the local properties of the free-energy surface.  We suggest two different prescriptions: one is based on the local diffusivity and the other on the local geometrical properties.  We further examine the problem of extg. the free-energy surface when using adaptive Gaussians.  We show that the std. relation between the bias and the free energy does not hold.  In the limit of narrow Gaussians an explicit correction can be evaluated.  In the general case, we propose to use instead a relation between bias and free energy borrowed from umbrella sampling.  This relation holds for all kinds of incrementally deposited bias.  We illustrate on the case of alanine dipeptide the advantage of using adaptive Gaussians in conjunction with the new free-energy estimator both in terms of accuracy and speed of convergence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM59Gkb4Mc8LVg90H21EOLACvtfcHk0lg7WMobStk8Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrsLc%253D&md5=abed7a6d34ff4797d7cbdc3167ad9060</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1021%2Fct3002464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct3002464%26sid%3Dliteratum%253Aachs%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DMetadynamics%2520with%2520Adaptive%2520Gaussians%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2247%26epage%3D2254%26doi%3D10.1021%2Fct3002464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref173"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref173'); return false;" data-citation="" class="refNumLink">173</a></strong><div class="NLM_citation" id="cit173"><span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">From A to B in free energy space</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">054103</span><span class="refDoi"> DOI: 10.1063/1.2432340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.2432340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=17302470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2007&pages=054103&author=D.+Branduardiauthor=F.+L.+Gervasioauthor=M.+Parrinello&title=From+A+to+B+in+free+energy+space&doi=10.1063%2F1.2432340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">From A to B in free energy space</span></div><div class="casAuthors">Branduardi, Davide; Gervasio, Francesco Luigi; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">054103/1-054103/10</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The authors present a new method for searching low free energy paths in complex mol. systems at finite temp.  They introduce two variables that are able to describe the position of a point in configurational space relative to a preassigned path.  With the help of these two variables the authors combine features of approaches such as metadynamics or umbrella sampling with those of path based methods.  This allows global searches in the space of paths to be performed and a new variational principle for the detn. of low free energy paths to be established.  Contrary to metadynamics or umbrella sampling the path can be described by an arbitrary large no. of variables, still the energy profile along the path can be calcd.  The authors exemplify the method numerically by studying the conformational changes of alanine dipeptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEGhhA71jj2rVg90H21EOLACvtfcHk0ljR0p_MlIylSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVarurk%253D&md5=c77c94b6d208f45a08ea2269894010f1</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1063%2F1.2432340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2432340%26sid%3Dliteratum%253Aachs%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFrom%2520A%2520to%2520B%2520in%2520free%2520energy%2520space%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2007%26volume%3D126%26spage%3D054103%26doi%3D10.1063%2F1.2432340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref174"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref174'); return false;" data-citation="" class="refNumLink">174</a></strong><div class="NLM_citation" id="cit174"><span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiana, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provasi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broglia, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Insight into the folding inhibition of the HIV-1 protease by a small peptide</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">2813</span><span class="NLM_x">–</span> <span class="NLM_lpage">2821</span><span class="refDoi"> DOI: 10.1529/biophysj.107.106369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1529%2Fbiophysj.107.106369" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2007&pages=2813-2821&author=M.+Bonomiauthor=F.+L.+Gervasioauthor=G.+Tianaauthor=D.+Provasiauthor=R.+A.+Brogliaauthor=M.+Parrinello&title=Insight+into+the+folding+inhibition+of+the+HIV-1+protease+by+a+small+peptide&doi=10.1529%2Fbiophysj.107.106369"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.107.106369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.107.106369%26sid%3Dliteratum%253Aachs%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DTiana%26aufirst%3DG.%26aulast%3DProvasi%26aufirst%3DD.%26aulast%3DBroglia%26aufirst%3DR.%2BA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DInsight%2520into%2520the%2520folding%2520inhibition%2520of%2520the%2520HIV-1%2520protease%2520by%2520a%2520small%2520peptide%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D93%26spage%3D2813%26epage%3D2821%26doi%3D10.1529%2Fbiophysj.107.106369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref175"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref175'); return false;" data-citation="" class="refNumLink">175</a></strong><div class="NLM_citation" id="cit175"><span><span class="NLM_contrib-group">Berteotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Protein conformational transitions: the closure mechanism of a kinase explored by atomistic simulations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">244</span><span class="NLM_x">–</span> <span class="NLM_lpage">250</span><span class="refDoi"> DOI: 10.1021/ja806846q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja806846q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVOqt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=244-250&author=A.+Berteottiauthor=A.+Cavalliauthor=D.+Branduardiauthor=F.+L.+Gervasioauthor=M.+Recanatiniauthor=M.+Parrinello&title=Protein+conformational+transitions%3A+the+closure+mechanism+of+a+kinase+explored+by+atomistic+simulations&doi=10.1021%2Fja806846q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Conformational Transitions: The Closure Mechanism of a Kinase Explored by Atomistic Simulations</span></div><div class="casAuthors">Berteotti, Anna; Cavalli, Andrea; Branduardi, Davide; Gervasio, Francesco Luigi; Recanatini, Maurizio; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">244-250</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase large-scale conformational rearrangement is an issue of enormous biol. and pharmacol. relevance.  Atomistic simulations able to capture the dynamics and the energetics of kinase large-scale motions are still in their infancy.  Here, we present a computational study in which the atomistic dynamics of the "open-to-closed" movement of the cyclin-dependent kinase 5 (CDK5) have been simulated.  Simulations were carried out using a new sampling method that is able to find the lowest free-energy channel between an initial state and a final state.  This large-scale movement has a two-step mechanism: first, the αC-helix rotates by ∼45°, allowing the interaction between Glu51 and Arg149; then the CDK5 activation loop (T-loop) refolds to assume the closed conformation.  We have also estd. the free-energy profile assocd. with the global motion and identified a CDK5 intermediate, which could be exploited for drug-design purposes.  Our new sampling method turned out to be well-suited for investigating at an atomistic level the energetics and dynamics of kinase large-scale conformational motions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWrIGb0X4d0bVg90H21EOLACvtfcHk0ljR0p_MlIylSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVOqt7zP&md5=64a08f8125ae7db038a6127f6d80612a</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1021%2Fja806846q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja806846q%26sid%3Dliteratum%253Aachs%26aulast%3DBerteotti%26aufirst%3DA.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DProtein%2520conformational%2520transitions%253A%2520the%2520closure%2520mechanism%2520of%2520a%2520kinase%2520explored%2520by%2520atomistic%2520simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D244%26epage%3D250%26doi%3D10.1021%2Fja806846q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref176"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref176'); return false;" data-citation="" class="refNumLink">176</a></strong><div class="NLM_citation" id="cit176"><span><span class="NLM_contrib-group">Ceccarini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span> </span><span class="NLM_article-title">Ion Conduction through the hERG potassium channel</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e49017</span><span class="refDoi"> DOI: 10.1371/journal.pone.0049017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pone.0049017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23133669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslajtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e49017&author=L.+Ceccariniauthor=M.+Masettiauthor=A.+Cavalliauthor=M.+Recanatini&title=Ion+Conduction+through+the+hERG+potassium+channel&doi=10.1371%2Fjournal.pone.0049017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Ion conduction through the hERG potassium channel</span></div><div class="casAuthors">Ceccarini, Luisa; Masetti, Matteo; Cavalli, Andrea; Recanatini, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e49017</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The inward rectifier voltage-gated potassium channel hERG is of primary importance for the regulation of the membrane potential of cardiomyocytes.  Unlike most voltage-gated K+-channels, hERG shows a low elementary conductance at physiol. voltage and potassium concn.  To investigate the mol. features underlying this unusual behavior, we simulated the ion conduction through the selectivity filter at a fully atomistic level by means of mol. dynamics-based methods, using a homol.-derived model.  According to our calcns., permeation of potassium ions can occur along two pathways, one involving site vacancies inside the filter (showing an energy barrier of about 6 kcal mol-1) and the other characterized by the presence of a knock-on intermediate (about 8 kcal mol-1).  These barriers are indeed in accordance with a low conductance behavior and can be explained in terms of a series of distinctive structural features displayed by the hERG ion permeation pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqob-gIww-yO7Vg90H21EOLACvtfcHk0ljR0p_MlIylSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslajtLvN&md5=00f2558e937c4d22eb21977b160143db</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0049017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0049017%26sid%3Dliteratum%253Aachs%26aulast%3DCeccarini%26aufirst%3DL.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRecanatini%26aufirst%3DM.%26atitle%3DIon%2520Conduction%2520through%2520the%2520hERG%2520potassium%2520channel%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De49017%26doi%3D10.1371%2Fjournal.pone.0049017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref177"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref177'); return false;" data-citation="" class="refNumLink">177</a></strong><div class="NLM_citation" id="cit177"><span><span class="NLM_contrib-group">Tribello, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceriotti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Using sketch-map coordinates to analyze and bias molecular dynamics simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5196</span><span class="NLM_x">–</span> <span class="NLM_lpage">5201</span><span class="refDoi"> DOI: 10.1073/pnas.1201152109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1201152109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22427357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslKqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5196-5201&author=G.+A.+Tribelloauthor=M.+Ceriottiauthor=M.+Parrinello&title=Using+sketch-map+coordinates+to+analyze+and+bias+molecular+dynamics+simulations&doi=10.1073%2Fpnas.1201152109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Using sketch-map coordinates to analyze and bias molecular dynamics simulations</span></div><div class="casAuthors">Tribello, Gareth A.; Ceriotti, Michele; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5196-5201, S5196/1-S5196/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">When examg. complex problems, such as the folding of proteins, coarse grained descriptions of the system drive our investigation and help us to rationalize the results.  Oftentimes collective variables (CVs), derived through some chem. intuition about the process of interest, serve this purpose.  Because finding these CVs is the most difficult part of any investigation, the authors recently developed a dimensionality redn. algorithm, sketch-map, that can be used to build a low-dimensional map of a phase space of high-dimensionality.  In this paper we discuss how these machine-generated CVs can be used to accelerate the exploration of phase space and to reconstruct free-energy landscapes.  To do so, we develop a formalism in which high-dimensional configurations are no longer represented by low-dimensional position vectors.  Instead, for each configuration we calc. a probability distribution, which has a domain that encompasses the entirety of the low-dimensional space.  To construct a biasing potential, we exploit an analogy with metadynamics and use the trajectory to adaptively construct a repulsive, history-dependent bias from the distributions that correspond to the previously visited configurations.  This potential forces the system to explore more of phase space by making it desirable to adopt configurations whose distributions do not overlap with the bias.  We apply this algorithm to a small model protein and succeed in reproducing the free-energy surface that we obtain from a parallel tempering calcn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxdjYCoMTairVg90H21EOLACvtfcHk0lgUCo0hR0zfig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslKqu78%253D&md5=67fb6e0b72cbacef5fdbd575df9a9d61</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1201152109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1201152109%26sid%3Dliteratum%253Aachs%26aulast%3DTribello%26aufirst%3DG.%2BA.%26aulast%3DCeriotti%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DUsing%2520sketch-map%2520coordinates%2520to%2520analyze%2520and%2520bias%2520molecular%2520dynamics%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5196%26epage%3D5201%26doi%3D10.1073%2Fpnas.1201152109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref178"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref178'); return false;" data-citation="" class="refNumLink">178</a></strong><div class="NLM_citation" id="cit178"><span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Free-energy landscape for beta hairpin folding from combined parallel tempering and metadynamics</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">13435</span><span class="NLM_x">–</span> <span class="NLM_lpage">13441</span><span class="refDoi"> DOI: 10.1021/ja062463w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja062463w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVCitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=13435-13441&author=G.+Bussiauthor=F.+L.+Gervasioauthor=A.+Laioauthor=M.+Parrinello&title=Free-energy+landscape+for+beta+hairpin+folding+from+combined+parallel+tempering+and+metadynamics&doi=10.1021%2Fja062463w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">Free-Energy Landscape for β Hairpin Folding from Combined Parallel Tempering and Metadynamics</span></div><div class="casAuthors">Bussi, Giovanni; Gervasio, Francesco Luigi; Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13435-13441</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We develop a new free-energy method, based on the combination of parallel tempering and metadynamics, and apply this method to the calcn. of the free-energy landscape of the folding β hairpin in explicit water.  We show that the combined method greatly improves the performance of both parallel tempering and metadynamics.  In particular, we are able to sample the high free-energy regions, which are not accessible with conventional parallel tempering.  We use our results to calc. the difference in entropy and enthalpy between the folded and the unfolded state and to characterize the most populated configurations in the relevant free-energy basins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodH4winUNaNLVg90H21EOLACvtfcHk0lgUCo0hR0zfig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVCitbY%253D&md5=a48316e8acd40c53e00319a169181f8b</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1021%2Fja062463w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja062463w%26sid%3Dliteratum%253Aachs%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFree-energy%2520landscape%2520for%2520beta%2520hairpin%2520folding%2520from%2520combined%2520parallel%2520tempering%2520and%2520metadynamics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D13435%26epage%3D13441%26doi%3D10.1021%2Fja062463w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref179"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref179'); return false;" data-citation="" class="refNumLink">179</a></strong><div class="NLM_citation" id="cit179"><span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Enhanced sampling in the well-tempered ensemble</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">190601</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.104.190601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.104.190601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20866953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVKlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2010&pages=190601&author=M.+Bonomiauthor=M.+Parrinello&title=Enhanced+sampling+in+the+well-tempered+ensemble&doi=10.1103%2FPhysRevLett.104.190601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced Sampling in the Well-Tempered Ensemble</span></div><div class="casAuthors">Bonomi, M.; Parrinello, M.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">190601/1-190601/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We introduce the well-tempered ensemble (WTE) which is the biased ensemble sampled by well-tempered metadynamics when the energy is used as collective variable.  WTE can be designed so as to have approx. the same av. energy as the canonical ensemble but much larger fluctuations.  These two properties lead to an extremely fast exploration of phase space.  An even greater efficiency is obtained when WTE is combined with parallel tempering.  Unbiased Boltzmann avs. are computed on the fly by a recently developed reweighting method.  We apply WTE and its parallel tempering variant to the 2d Ising model and to a Go model of HIV protease, demonstrating in these two representative cases that convergence is accelerated by orders of magnitude.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6bxid23q8KbVg90H21EOLACvtfcHk0lgUCo0hR0zfig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVKlt7w%253D&md5=c00b17d1b1be22d497d366789ba450eb</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.104.190601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.104.190601%26sid%3Dliteratum%253Aachs%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DEnhanced%2520sampling%2520in%2520the%2520well-tempered%2520ensemble%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2010%26volume%3D104%26spage%3D190601%26doi%3D10.1103%2FPhysRevLett.104.190601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref180"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref180'); return false;" data-citation="" class="refNumLink">180</a></strong><div class="NLM_citation" id="cit180"><span><span class="NLM_contrib-group">Piana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span> </span><span class="NLM_article-title">A bias-exchange approach to protein folding</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">4553</span><span class="NLM_x">–</span> <span class="NLM_lpage">4559</span><span class="refDoi"> DOI: 10.1021/jp067873l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp067873l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvFCht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=4553-4559&author=S.+Pianaauthor=A.+Laio&title=A+bias-exchange+approach+to+protein+folding&doi=10.1021%2Fjp067873l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">A Bias-Exchange Approach to Protein Folding</span></div><div class="casAuthors">Piana, Stefano; Laio, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4553-4559</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By suitably extending a recent approach [Bussi, G., et al., 2006] the authors introduce a powerful methodol. that allows the parallel reconstruction of the free energy of a system in a virtually unlimited no. of variables.  Multiple metadynamics simulations of the same system at the same temp. are performed, biasing each replica with a time-dependent potential constructed in a different set of collective variables.  Exchanges between the bias potentials in the different variables are periodically allowed according to a replica exchange scheme.  Due to the efficaciously multidimensional nature of the bias the method allows exploring complex free energy landscapes with high efficiency.  The usefulness of the method is demonstrated by performing an atomistic simulation in explicit solvent of the folding of a Triptophane cage miniprotein.  It is shown that the folding free energy landscape can be fully characterized starting from an extended conformation with use of only 40 ns of simulation on 8 replicas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1apQMKRwl57Vg90H21EOLACvtfcHk0lgLexv6rKCJ8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvFCht7Y%253D&md5=7f64fd37737745fa58c24f00f839207b</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1021%2Fjp067873l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp067873l%26sid%3Dliteratum%253Aachs%26aulast%3DPiana%26aufirst%3DS.%26aulast%3DLaio%26aufirst%3DA.%26atitle%3DA%2520bias-exchange%2520approach%2520to%2520protein%2520folding%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2007%26volume%3D111%26spage%3D4553%26epage%3D4559%26doi%3D10.1021%2Fjp067873l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref181"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref181'); return false;" data-citation="" class="refNumLink">181</a></strong><div class="NLM_citation" id="cit181"><span><span class="NLM_contrib-group">Tribello, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceriotti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">A self-learning algorithm for biased molecular dynamics</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">17509</span><span class="NLM_x">–</span> <span class="NLM_lpage">17514</span><span class="refDoi"> DOI: 10.1073/pnas.1011511107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1011511107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20876135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlartbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=17509-17514&author=G.+A.+Tribelloauthor=M.+Ceriottiauthor=M.+Parrinello&title=A+self-learning+algorithm+for+biased+molecular+dynamics&doi=10.1073%2Fpnas.1011511107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">A self-learning algorithm for biased molecular dynamics</span></div><div class="casAuthors">Tribello, Gareth A.; Ceriotti, Michele; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">17509-17514, S17509/1-S17509/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A new self-learning algorithm for accelerated dynamics, reconnaissance meta-dynamics, is proposed that is able to work with a very large no. of collective coordinates.  Acceleration of the dynamics is achieved by constructing a bias potential in terms of a patchwork of one-dimensional, locally valid collective coordinates.  These collective coordinates are obtained from trajectory analyses so that they adapt to any new features encountered during the simulation.  We show how this methodol. can be used to enhance sampling in real chem. systems citing examples both from the physics of clusters and from the biol. sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVviEhtXC5YbVg90H21EOLACvtfcHk0lgLexv6rKCJ8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlartbzP&md5=38891e4390eb9041341c4c301625c8be</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1011511107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1011511107%26sid%3Dliteratum%253Aachs%26aulast%3DTribello%26aufirst%3DG.%2BA.%26aulast%3DCeriotti%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DA%2520self-learning%2520algorithm%2520for%2520biased%2520molecular%2520dynamics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D17509%26epage%3D17514%26doi%3D10.1073%2Fpnas.1011511107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref182"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref182'); return false;" data-citation="" class="refNumLink">182</a></strong><div class="NLM_citation" id="cit182"><span><span class="NLM_contrib-group">Limongelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marinelli, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Molecular basis of cyclooxygenase enzymes (COXs) selective inhibition</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">5411</span><span class="NLM_x">–</span> <span class="NLM_lpage">5416</span><span class="refDoi"> DOI: 10.1073/pnas.0913377107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0913377107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20215464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFKhsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=5411-5416&author=V.+Limongelliauthor=M.+Bonomiauthor=L.+Marinelliauthor=F.+L.+Gervasioauthor=A.+Cavalliauthor=E.+Novellinoauthor=M.+Parrinello&title=Molecular+basis+of+cyclooxygenase+enzymes+%28COXs%29+selective+inhibition&doi=10.1073%2Fpnas.0913377107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of cyclooxygenase enzymes (COXs) selective inhibition</span></div><div class="casAuthors">Limongelli, Vittorio; Bonomi, Massimilliano; Marinelli, Luciana; Gervasio, Francesco Luigi; Cavalli, Andrea; Novellino, Ettore; Parinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5411-5416, S5411/1-S5411/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The widely used nonsteroidal anti-inflammatory drugs block the cyclooxygenase enzymes (COXs) and are clin. used for the treatment of inflammation, pain, and cancers.  A selective inhibition of the different isoforms, particularly COX-2, is desirable, and consequently a deeper understanding of the mol. basis of selective inhibition is of great demand.  Using an advanced computational technique we have simulated the full dissocn. process of a highly potent and selective inhibitor, SC-558, in both COX-1 and COX-2.  We have found a previously unreported alternative binding mode in COX-2 explaining the time-dependent inhibition exhibited by this class of inhibitors and consequently their long residence time inside this isoform.  Our metadynamics-based approach allows us to illuminate the highly dynamical character of the ligand/protein recognition process, thus explaining a wealth of exptl. data and paving the way to an innovative strategy for designing new COX inhibitors with tuned selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKbCarszxULVg90H21EOLACvtfcHk0lgLexv6rKCJ8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFKhsrs%253D&md5=ef116a356063dd9068293daded6e5207</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0913377107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0913377107%26sid%3Dliteratum%253Aachs%26aulast%3DLimongelli%26aufirst%3DV.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DMolecular%2520basis%2520of%2520cyclooxygenase%2520enzymes%2520%2528COXs%2529%2520selective%2520inhibition%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D5411%26epage%3D5416%26doi%3D10.1073%2Fpnas.0913377107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref183"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref183'); return false;" data-citation="" class="refNumLink">183</a></strong><div class="NLM_citation" id="cit183"><span><span class="NLM_contrib-group">Favia, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Substrate binding process and mechanistic functioning of type 1 11 beta-hydroxysteroid dehydrogenase from enhanced sampling methods</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e25375</span><span class="refDoi"> DOI: 10.1371/journal.pone.0025375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pone.0025375" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25375&author=A.+D.+Faviaauthor=M.+Masettiauthor=M.+Recanatiniauthor=A.+Cavalli&title=Substrate+binding+process+and+mechanistic+functioning+of+type+1+11+beta-hydroxysteroid+dehydrogenase+from+enhanced+sampling+methods&doi=10.1371%2Fjournal.pone.0025375"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025375%26sid%3Dliteratum%253Aachs%26aulast%3DFavia%26aufirst%3DA.%2BD.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSubstrate%2520binding%2520process%2520and%2520mechanistic%2520functioning%2520of%2520type%25201%252011%2520beta-hydroxysteroid%2520dehydrogenase%2520from%2520enhanced%2520sampling%2520methods%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25375%26doi%3D10.1371%2Fjournal.pone.0025375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref184"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref184'); return false;" data-citation="" class="refNumLink">184</a></strong><div class="NLM_citation" id="cit184"><span><span class="NLM_contrib-group">Provasi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bortolato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filizola, M.</span><span> </span><span class="NLM_article-title">Exploring molecular mechanisms of ligand recognition by opioid receptors with metadynamics</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">10020</span><span class="NLM_x">–</span> <span class="NLM_lpage">10029</span><span class="refDoi"> DOI: 10.1021/bi901494n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901494n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WktbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=10020-10029&author=D.+Provasiauthor=A.+Bortolatoauthor=M.+Filizola&title=Exploring+molecular+mechanisms+of+ligand+recognition+by+opioid+receptors+with+metadynamics&doi=10.1021%2Fbi901494n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Molecular Mechanisms of Ligand Recognition by Opioid Receptors with Metadynamics</span></div><div class="casAuthors">Provasi, Davide; Bortolato, Andrea; Filizola, Marta</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">10020-10029</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Opioid receptors are G protein-coupled receptors (GPCRs) of utmost significance in the development of potent analgesic drugs for the treatment of severe pain.  An accurate evaluation at the mol. level of the ligand binding pathways into these receptors may play a key role in the design of new mols. with more desirable properties and reduced side effects.  The recent characterization of high-resoln. x-ray crystal structures of non-rhodopsin GPCRs for diffusible hormones and neurotransmitters presents an unprecedented opportunity to build improved homol. models of opioid receptors, and to study in more detail their mol. mechanisms of ligand recognition.  In this study, possible pathways for entry of the nonselective antagonist naloxone (NLX) from the water environment into the well-accepted alkaloid binding pocket of a delta opioid receptor (DOR) mol. model based on the β2-adrenergic receptor crystal structure are explored using microsecond-scale well-tempered metadynamics simulations.  Using as collective variables distances that account for the position of NLX and of the receptor extracellular loop 2 in relation to the DOR binding pocket, the authors were able to distinguish between the different states visited by the ligand (i.e., docked, undocked, and metastable bound intermediates) and to predict a free energy of binding close to exptl. values after correcting for possible drawbacks of the sampling approach.  The strategy employed herein holds promise for its application to the docking of diverse ligands to the opioid receptors as well as to other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4pkSAtQ5Mq7Vg90H21EOLACvtfcHk0lgeieDrEqibmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WktbbM&md5=eca8e33e8ca8eef73302070b80cb8a6f</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1021%2Fbi901494n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901494n%26sid%3Dliteratum%253Aachs%26aulast%3DProvasi%26aufirst%3DD.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DFilizola%26aufirst%3DM.%26atitle%3DExploring%2520molecular%2520mechanisms%2520of%2520ligand%2520recognition%2520by%2520opioid%2520receptors%2520with%2520metadynamics%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D10020%26epage%3D10029%26doi%3D10.1021%2Fbi901494n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref185"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref185'); return false;" data-citation="" class="refNumLink">185</a></strong><div class="NLM_citation" id="cit185"><span><span class="NLM_contrib-group">Limongelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Funnel metadynamics as accurate binding free-energy method</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">6358</span><span class="NLM_x">–</span> <span class="NLM_lpage">6363</span><span class="refDoi"> DOI: 10.1073/pnas.1303186110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1303186110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23553839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVOlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=6358-6363&author=V.+Limongelliauthor=M.+Bonomiauthor=M.+Parrinello&title=Funnel+metadynamics+as+accurate+binding+free-energy+method&doi=10.1073%2Fpnas.1303186110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Funnel metadynamics as accurate binding free-energy method</span></div><div class="casAuthors">Limongelli, Vittorio; Bonomi, Massimiliano; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6358-6363, S6358/1-S6358/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A detailed description of the events ruling ligand/protein interaction and an accurate estn. of the drug affinity to its target is of great help in speeding drug discovery strategies.  We have developed a metadynamics-based approach, named funnel metadynamics, that allows the ligand to enhance the sampling of the target binding sites and its solvated states.  This method leads to an efficient characterization of the binding free-energy surface and an accurate calcn. of the abs. protein-ligand binding free energy.  We illustrate our protocol in two systems, benzamidine/trypsin and SC-558/cyclooxygenase 2.  In both cases, the x-ray conformation has been found as the lowest free-energy pose, and the computed protein-ligand binding free energy in good agreement with expts.  Furthermore, funnel metadynamics unveils important information about the binding process, such as the presence of alternative binding modes and the role of waters.  The results achieved at an affordable computational cost make funnel metadynamics a valuable method for drug discovery and for dealing with a variety of problems in chem., physics, and material science.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6TRPMM3o257Vg90H21EOLACvtfcHk0liNtilKMtm_Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVOlurw%253D&md5=39d87aa2f8e06572ee8044a899ade3fb</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303186110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303186110%26sid%3Dliteratum%253Aachs%26aulast%3DLimongelli%26aufirst%3DV.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFunnel%2520metadynamics%2520as%2520accurate%2520binding%2520free-energy%2520method%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D6358%26epage%3D6363%26doi%3D10.1073%2Fpnas.1303186110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref186"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref186'); return false;" data-citation="" class="refNumLink">186</a></strong><div class="NLM_citation" id="cit186"><span><span class="NLM_contrib-group">Di Leva, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limongelli, V.</span><span> </span><span class="NLM_article-title">Mechanistic insight into ligand binding to G-quadruplex DNA</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">5447</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1093/nar/gku247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1093%2Fnar%2Fgku247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24753420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFGhur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=5447-55&author=F.+S.+Di+Levaauthor=E.+Novellinoauthor=A.+Cavalliauthor=M.+Parrinelloauthor=V.+Limongelli&title=Mechanistic+insight+into+ligand+binding+to+G-quadruplex+DNA&doi=10.1093%2Fnar%2Fgku247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insight into ligand binding to G-quadruplex DNA</span></div><div class="casAuthors">Di Leva, Francesco Saverio; Novellino, Ettore; Cavalli, Andrea; Parrinello, Michele; Limongelli, Vittorio</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5447-5455</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Specific guanine-rich regions in human genome can form higher-order DNA structures called G-quadruplexes, which regulate many relevant biol. processes.  For instance, the formation of G-quadruplex at telomeres can alter cellular functions, inducing apoptosis.  Thus, developing small mols. that are able to bind and stabilize the telomeric G-quadruplexes represents an attractive strategy for antitumor therapy.  An example is 3-(benzo[d]thiazol-2-yl)-7-hydroxy-8-((4-(2-hydroxyethyl)piperazin-1-yl)methyl)-2H-chromen-2-one (I), recently identified as a potent ligand of G-quadruplex [d(TGGGGT)]4 with promising in vitro antitumor activity.  The exptl. observations were suggestive of a complex binding mechanism that, despite efforts, has defied full characterization.  Here, the authors provide through metadynamics simulations a comprehensive understanding of the binding mechanism of I to G-quadruplex [d(TGGGGT)]4.  In the authors' calcns., the ligand explored all of the available binding sites on the DNA structure and the free-energy landscape of the whole binding process was computed.  The authors thus disclosed a peculiar hopping binding mechanism whereas I was able to bind both to the groove and to the 3' end of the G-quadruplex.  These results fully explain the available exptl. data, rendering the authors' approach of great value for further ligand/DNA studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpAZDnr8FxILVg90H21EOLACvtfcHk0lhGvNuMeUGpcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFGhur8%253D&md5=18fb1fcb9543ca972b2553ca994d9271</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku247%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26aulast%3DLimongelli%26aufirst%3DV.%26atitle%3DMechanistic%2520insight%2520into%2520ligand%2520binding%2520to%2520G-quadruplex%2520DNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D5447%26epage%3D55%26doi%3D10.1093%2Fnar%2Fgku247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref187"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref187'); return false;" data-citation="" class="refNumLink">187</a></strong><div class="NLM_citation" id="cit187"><span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Exploring complex protein-ligand recognition mechanisms with coarse metadynamics</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">4807</span><span class="NLM_x">–</span> <span class="NLM_lpage">4816</span><span class="refDoi"> DOI: 10.1021/jp803936q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp803936q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtl2qtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=4807-4816&author=M.+Masettiauthor=A.+Cavalliauthor=M.+Recanatiniauthor=F.+L.+Gervasio&title=Exploring+complex+protein-ligand+recognition+mechanisms+with+coarse+metadynamics&doi=10.1021%2Fjp803936q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit187R"><div class="casContent"><span class="casTitleNuber">187</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Complex Protein-Ligand Recognition Mechanisms with Coarse Metadynamics</span></div><div class="casAuthors">Masetti, Matteo; Cavalli, Andrea; Recanatini, Maurizio; Gervasio, Francesco Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4807-4816</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The metadynamics method has been shown to be a valuable tool to study the mechanism of mol. recognition in atomistic detail.  However, it requires an a priori knowledge of all slow degrees of freedom relevant to the docking/undocking mechanism.  Here we investigate a combination of docking/clustering with metadynamics performed with a subset of the necessary degrees of freedom (coarse metadynamics), and show that it provides a full mechanistic insight on the protein-ligand docking mechanism.  Moreover, the proposed protocol is able to clearly distinguish between crystallog. and noncrystallog. poses of protein-ligand complexes, and also to find the transition state of the full undocking mechanism, thus giving an indication on the binding free energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR9b4f41EsHrVg90H21EOLACvtfcHk0lhGvNuMeUGpcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtl2qtr4%253D&md5=a4debf330970a1ae2e6122265349ea76</span></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1021%2Fjp803936q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp803936q%26sid%3Dliteratum%253Aachs%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DExploring%2520complex%2520protein-ligand%2520recognition%2520mechanisms%2520with%2520coarse%2520metadynamics%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D4807%26epage%3D4816%26doi%3D10.1021%2Fjp803936q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref188"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref188'); return false;" data-citation="" class="refNumLink">188</a></strong><div class="NLM_citation" id="cit188"><span><span class="NLM_contrib-group">Wang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span> </span><span class="NLM_article-title">Identifying ligand binding sites and poses using GPU-accelerated hamiltonian replica exchange molecular dynamics</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">989</span><span class="NLM_x">–</span> <span class="NLM_lpage">1007</span><span class="refDoi"> DOI: 10.1007/s10822-013-9689-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-013-9689-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24297454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWqurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=989-1007&author=K.+Wangauthor=J.+D.+Choderaauthor=Y.+Z.+Yangauthor=M.+R.+Shirts&title=Identifying+ligand+binding+sites+and+poses+using+GPU-accelerated+hamiltonian+replica+exchange+molecular+dynamics&doi=10.1007%2Fs10822-013-9689-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit188R"><div class="casContent"><span class="casTitleNuber">188</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying ligand binding sites and poses using GPU-accelerated Hamiltonian replica exchange molecular dynamics</span></div><div class="casAuthors">Wang, Kai; Chodera, John D.; Yang, Yanzhi; Shirts, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">989-1007</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">We present a method to identify small mol. ligand binding sites and poses within a given protein crystal structure using GPU-accelerated Hamiltonian replica exchange mol. dynamics simulations.  The Hamiltonians used vary from the phys. end state of protein interacting with the ligand to an unphys. end state where the ligand does not interact with the protein.  As replicas explore the space of Hamiltonians interpolating between these states, the ligand can rapidly escape local min. and explore potential binding sites.  Geometric restraints keep the ligands from leaving the vicinity of the protein and an alchem. pathway designed to increase phase space overlap between intermediates ensures good mixing.  Because of the rigorous statistical mech. nature of the Hamiltonian exchange framework, we can also ext. binding free energy ests. for all putative binding sites.  We present results of this methodol. applied to the T4 lysozyme L99A model system for three known ligands and one non-binder as a control, using an implicit solvent.  We find that our methodol. identifies known crystallog. binding sites consistently and accurately for the small no. of ligands considered here and gives free energies consistent with expt.  We are also able to analyze the contribution of individual binding sites to the overall binding affinity.  Our methodol. points to near term potential applications in early-stage structure-guided drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9FKdHS8YIirVg90H21EOLACvtfcHk0lhGvNuMeUGpcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWqurzO&md5=9fd70948a1f8a1feac6cbcf32eb27fb8</span></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9689-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9689-8%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DK.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DIdentifying%2520ligand%2520binding%2520sites%2520and%2520poses%2520using%2520GPU-accelerated%2520hamiltonian%2520replica%2520exchange%2520molecular%2520dynamics%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D989%26epage%3D1007%26doi%3D10.1007%2Fs10822-013-9689-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref189"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref189'); return false;" data-citation="" class="refNumLink">189</a></strong><div class="NLM_citation" id="cit189"><span><span class="NLM_contrib-group">Luitz, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zacharias, M.</span><span> </span><span class="NLM_article-title">Protein-ligand docking using hamiltonian replica exchange simulations with soft core potentials</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1669</span><span class="NLM_x">–</span> <span class="NLM_lpage">1675</span><span class="refDoi"> DOI: 10.1021/ci500296f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500296f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFaitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1669-1675&author=M.+P.+Luitzauthor=M.+Zacharias&title=Protein-ligand+docking+using+hamiltonian+replica+exchange+simulations+with+soft+core+potentials&doi=10.1021%2Fci500296f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Ligand Docking Using Hamiltonian Replica Exchange Simulations with Soft Core Potentials</span></div><div class="casAuthors">Luitz, Manuel P.; Zacharias, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1669-1675</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations in explicit solvent allow studying receptor-ligand binding processes including full flexibility of the binding partners and an explicit inclusion of solvation effects.  However, in MD simulations, the search for an optimal ligand-receptor complex geometry is frequently trapped in locally stable non-native binding geometries.  A Hamiltonian replica-exchange (H-REMD)-based protocol has been designed to enhance the sampling of putative ligand-receptor complexes.  It is based on softening nonbonded ligand-receptor interactions along the replicas and one ref. replica under the control of the original force field.  The efficiency of the method has been evaluated on two receptor-ligand systems and one protein-peptide complex.  Starting from misplaced initial docking geometries, the H-REMD method reached in each case the known binding geometry significantly faster than a std. MD simulation.  The approach could also be useful to identify and evaluate alternative binding geometries in a given binding region with small relative differences in binding free energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6eowStElIMbVg90H21EOLACvtfcHk0lhGvNuMeUGpcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFaitro%253D&md5=1e54a0f00f1a80f878c2e2cbb7e34da0</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1021%2Fci500296f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500296f%26sid%3Dliteratum%253Aachs%26aulast%3DLuitz%26aufirst%3DM.%2BP.%26aulast%3DZacharias%26aufirst%3DM.%26atitle%3DProtein-ligand%2520docking%2520using%2520hamiltonian%2520replica%2520exchange%2520simulations%2520with%2520soft%2520core%2520potentials%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1669%26epage%3D1675%26doi%3D10.1021%2Fci500296f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref190"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref190'); return false;" data-citation="" class="refNumLink">190</a></strong><div class="NLM_citation" id="cit190"><span><span class="NLM_contrib-group">Duane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendleton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roweth, D.</span><span> </span><span class="NLM_article-title">Hybrid monte carlo</span> <span class="citation_source-journal">Phys. Lett. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">195</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1016/0370-2693(87)91197-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0370-2693%2887%2991197-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL2sXls1egsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=1987&pages=216-222&author=S.+Duaneauthor=A.+Kennedyauthor=B.+Pendletonauthor=D.+Roweth&title=Hybrid+monte+carlo&doi=10.1016%2F0370-2693%2887%2991197-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit190R"><div class="casContent"><span class="casTitleNuber">190</span><div class="casTitle"><span class="NLM_cas:atitle">Hybrid Monte Carlo</span></div><div class="casAuthors">Duane, Simon; Kennedy, A. D.; Pendleton, Brian J.; Roweth, Duncan</div><div class="citationInfo"><span class="NLM_cas:title">Physics Letters B</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">216-22</span>CODEN:
                <span class="NLM_cas:coden">PYLBAJ</span>;
        ISSN:<span class="NLM_cas:issn">0370-2693</span>.
    </div><div class="casAbstract">A new method is presented for the numerical simulation of lattice field theory.  A hybrid (mol. dynamics/Langevin) algorithm is used to guide a Monte Carlo simulation.  There are no discretization errors even for large step sizes.  The method is esp. efficient for systems such as quantum chromodynamics which contain fermionic degrees of freedom.  Detailed results are presented for 4-dimensional compact quantum electrodynamics including the dynamical effects of e.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXNgJcTdHflrVg90H21EOLACvtfcHk0lginlg2VSWbpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXls1egsbY%253D&md5=3f40fdc8cfc804b67b9c5673f811aba2</span></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1016%2F0370-2693%2887%2991197-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0370-2693%252887%252991197-X%26sid%3Dliteratum%253Aachs%26aulast%3DDuane%26aufirst%3DS.%26aulast%3DKennedy%26aufirst%3DA.%26aulast%3DPendleton%26aufirst%3DB.%26aulast%3DRoweth%26aufirst%3DD.%26atitle%3DHybrid%2520monte%2520carlo%26jtitle%3DPhys.%2520Lett.%2520B%26date%3D1987%26volume%3D195%26spage%3D216%26epage%3D222%26doi%3D10.1016%2F0370-2693%2887%2991197-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref191"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref191'); return false;" data-citation="" class="refNumLink">191</a></strong><div class="NLM_citation" id="cit191"><span><span class="NLM_contrib-group">Fischer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cordes, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schütte, C.</span><span> </span><span class="NLM_article-title">Hybrid monte carlo with adaptive temperature in mixed-canonical ensemble: efficient conformational analysis of RNA</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1697</span><span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(19981130)19:15<1689::AID-JCC2>3.3.CO;2-R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1689-1697&author=A.+Fischerauthor=F.+Cordesauthor=C.+Sch%C3%BCtte&title=Hybrid+monte+carlo+with+adaptive+temperature+in+mixed-canonical+ensemble%3A+efficient+conformational+analysis+of+RNA&doi=10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252819981130%252919%253A15%253C1689%253A%253AAID-JCC2%253E3.3.CO%253B2-R%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DCordes%26aufirst%3DF.%26aulast%3DSch%25C3%25BCtte%26aufirst%3DC.%26atitle%3DHybrid%2520monte%2520carlo%2520with%2520adaptive%2520temperature%2520in%2520mixed-canonical%2520ensemble%253A%2520efficient%2520conformational%2520analysis%2520of%2520RNA%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1998%26volume%3D19%26spage%3D1689%26epage%3D1697%26doi%3D10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref192"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref192'); return false;" data-citation="" class="refNumLink">192</a></strong><div class="NLM_citation" id="cit192"><span><span class="NLM_contrib-group">Schütte, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huisinga, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deuflhard, P.</span><span> </span><span class="NLM_article-title">A Direct Approach to conformational dynamics based on hybrid monte carlo</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">151</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span><span class="refDoi"> DOI: 10.1006/jcph.1999.6231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1006%2Fjcph.1999.6231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1MXivFejtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=1999&pages=146-168&author=C.+Sch%C3%BCtteauthor=A.+Fischerauthor=W.+Huisingaauthor=P.+Deuflhard&title=A+Direct+Approach+to+conformational+dynamics+based+on+hybrid+monte+carlo&doi=10.1006%2Fjcph.1999.6231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">A Direct Approach to Conformational Dynamics Based on Hybrid Monte Carlo</span></div><div class="casAuthors">Schutte, Ch.; Fischer, A.; Huisinga, W.; Deuflhard, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">146-168</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Recently, a novel concept for the computation of essential features of the dynamics of Hamiltonian systems (such as mol. dynamics) has been proposed.  The realization of this concept had been based on subdivision techniques applied to the Frobenius-Perron operator for the dynamical system.  The present paper suggests an alternative but related concept that merges the conceptual advantages of the dynamical systems approach with the appropriate statistical physics framework.  This approach allows us to define the phrase "conformation" in terms of the dynamical behavior of the mol. system and to characterize the dynamical stability of conformations.  In a first step, the frequency of conformational changes is characterized in statistical terms leading to the definition of some Markov operator T that describes the corresponding transition probabilities within the canonical ensemble.  In a second step, a discretization of T via specific hybrid Monte Carlo techniques is shown to lead to a stochastic matrix P. With these theor. prepns., an identification algorithm for conformations (to be presented in a later paper) is applicable.  It is demonstrated that the discretization of T can be restricted to few essential degrees of freedom so that the combinatorial explosion of discretization boxes is prevented and biomol. systems can be attacked.  Numerical results for the n-pentane mol. and the triribonucleotide adenylyl(3'-5')cytidylyl(3'-5')cytidin are given and interpreted.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBQXKX7uekjbVg90H21EOLACvtfcHk0lginlg2VSWbpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivFejtL0%253D&md5=5eae420b000b63b4a4f6f66c77b74ec9</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1006%2Fjcph.1999.6231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjcph.1999.6231%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCtte%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26aulast%3DDeuflhard%26aufirst%3DP.%26atitle%3DA%2520Direct%2520Approach%2520to%2520conformational%2520dynamics%2520based%2520on%2520hybrid%2520monte%2520carlo%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1999%26volume%3D151%26spage%3D146%26epage%3D168%26doi%3D10.1006%2Fjcph.1999.6231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref193"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref193'); return false;" data-citation="" class="refNumLink">193</a></strong><div class="NLM_citation" id="cit193"><span><span class="NLM_contrib-group">Guarnieri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Still, W. C.</span><span> </span><span class="NLM_article-title">A rapidly convergent simulation method: mixed monte carlo/stochastic dynamics</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1302</span><span class="NLM_x">–</span> <span class="NLM_lpage">1310</span><span class="refDoi"> DOI: 10.1002/jcc.540151111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.540151111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=1302-1310&author=F.+Guarnieriauthor=W.+C.+Still&title=A+rapidly+convergent+simulation+method%3A+mixed+monte+carlo%2Fstochastic+dynamics&doi=10.1002%2Fjcc.540151111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540151111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540151111%26sid%3Dliteratum%253Aachs%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DStill%26aufirst%3DW.%2BC.%26atitle%3DA%2520rapidly%2520convergent%2520simulation%2520method%253A%2520mixed%2520monte%2520carlo%252Fstochastic%2520dynamics%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D1302%26epage%3D1310%26doi%3D10.1002%2Fjcc.540151111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref194"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref194'); return false;" data-citation="" class="refNumLink">194</a></strong><div class="NLM_citation" id="cit194"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodoscek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Computation of absolute hydration and binding free energy with free energy perturbation distributed replica-exchange molecular dynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2583</span><span class="NLM_x">–</span> <span class="NLM_lpage">2588</span><span class="refDoi"> DOI: 10.1021/ct900223z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900223z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=2583-2588&author=W.+Jiangauthor=M.+Hodoscekauthor=B.+Roux&title=Computation+of+absolute+hydration+and+binding+free+energy+with+free+energy+perturbation+distributed+replica-exchange+molecular+dynamics&doi=10.1021%2Fct900223z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">Computation of Absolute Hydration and Binding Free Energy with Free Energy Perturbation Distributed Replica-Exchange Molecular Dynamics</span></div><div class="casAuthors">Jiang, Wei; Hodoscek, Milan; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2583-2588</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Distributed Replica (REPDSTR) is a powerful parallelization technique enabling simulations of a group of replicas in a parallel/parallel fashion, where each replica is distributed to different nodes of a large cluster [Theor. Chem. Acc. 2003, 109, 140].  Here, we use the framework provided by REPDSTR to combine a staged free energy perturbation protocol with replica-exchange mol. dynamics (FEP/REMD).  The structure of REPDSTR, which allows multiple parallel input/output (I/O), facilitates the treatment of replica-exchange to couple the N window simulations corresponding to different values of the thermodn. coupling parameters.  As a result, each of the N synchronous window simulations benefit from the sampling carried out by the N-1 others.  As illustrative examples of the FEP/REMD strategy, calcns. of the abs. hydration and binding free energy of small mols. were performed using the biomol. simulation program CHARMM adapted for the IBM Blue Gene/P platform.  The computations show that a FEP/REMD strategy significantly improves the sampling and accelerates the convergence of abs. free energy computations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQnhjw95RCz7Vg90H21EOLACvtfcHk0liwO2DE-sw82g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu7rE&md5=ca830c52bfe67e33eef10281cd891e8e</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1021%2Fct900223z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900223z%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DHodoscek%26aufirst%3DM.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DComputation%2520of%2520absolute%2520hydration%2520and%2520binding%2520free%2520energy%2520with%2520free%2520energy%2520perturbation%2520distributed%2520replica-exchange%2520molecular%2520dynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D2583%26epage%3D2588%26doi%3D10.1021%2Fct900223z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref195"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref195'); return false;" data-citation="" class="refNumLink">195</a></strong><div class="NLM_citation" id="cit195"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Free energy perturbation hamiltonian replica-exchange molecular dynamics (FEP/H-REMD) for absolute ligand binding free energy calculations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2559</span><span class="NLM_x">–</span> <span class="NLM_lpage">2565</span><span class="refDoi"> DOI: 10.1021/ct1001768</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct1001768" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsV2lsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2559-2565&author=W.+Jiangauthor=B.+Roux&title=Free+energy+perturbation+hamiltonian+replica-exchange+molecular+dynamics+%28FEP%2FH-REMD%29+for+absolute+ligand+binding+free+energy+calculations&doi=10.1021%2Fct1001768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit195R"><div class="casContent"><span class="casTitleNuber">195</span><div class="casTitle"><span class="NLM_cas:atitle">Free Energy Perturbation Hamiltonian Replica-Exchange Molecular Dynamics (FEP/H-REMD) for Absolute Ligand Binding Free Energy Calculations</span></div><div class="casAuthors">Jiang, Wei; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2559-2565</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free Energy Perturbation with Replica Exchange Mol. Dynamics (FEP/REMD) offers a powerful strategy to improve the convergence of free energy computations.  In particular, it has been shown previously that a FEP/REMD scheme allowing random moves within an extended replica ensemble of thermodn. coupling parameters "λ" can improve the statistical convergence in calcns. of abs. binding free energy of ligands to proteins.  In the present study, FEP/REMD is extended and combined with an accelerated MD simulations method based on Hamiltonian replica-exchange MD (H-REMD) to overcome the addnl. problems arising from the existence of kinetically trapped conformations within the protein receptor.  In the combined strategy, each system with a given thermodn. coupling factor λ in the extended ensemble is further coupled with a set of replicas evolving on a biased energy surface with boosting potentials used to accelerate the interconversion among different rotameric states of the side chains in the neighborhood of the binding site.  Exchanges are allowed to occur alternatively along the axes corresponding to the thermodn. coupling parameter λ and the boosting potential, in an extended dual array of coupled λ- and H-REMD simulations.  The method is implemented on the basis of new extensions to the REPDSTR module of the biomol. simulation program CHARMM.  As an illustrative example, the abs. binding free energy of p-xylene to the nonpolar cavity of the L99A mutant of the T4 lysozyme was calcd.  The tests demonstrate that the dual λ-REMD and H-REMD simulation scheme greatly accelerates the configurational sampling of the rotameric states of the side chains around the binding pocket, thereby improving the convergence of the FEP computations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphmWym-fGS6rVg90H21EOLACvtfcHk0liwO2DE-sw82g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsV2lsrY%253D&md5=4517b8d4fc1eaa08f3a000d6724ec7d5</span></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1021%2Fct1001768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct1001768%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DFree%2520energy%2520perturbation%2520hamiltonian%2520replica-exchange%2520molecular%2520dynamics%2520%2528FEP%252FH-REMD%2529%2520for%2520absolute%2520ligand%2520binding%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2559%26epage%3D2565%26doi%3D10.1021%2Fct1001768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref196"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref196'); return false;" data-citation="" class="refNumLink">196</a></strong><div class="NLM_citation" id="cit196"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">The confine-and-release method: obtaining correct binding free energies in the presence of protein conformational change</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">–</span> <span class="NLM_lpage">1235</span><span class="refDoi"> DOI: 10.1021/ct700032n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700032n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=1231-1235&author=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=The+confine-and-release+method%3A+obtaining+correct+binding+free+energies+in+the+presence+of+protein+conformational+change&doi=10.1021%2Fct700032n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">Confine-and-Release Method: Obtaining Correct Binding Free Energies in the Presence of Protein Conformational Change</span></div><div class="casAuthors">Mobley, David L.; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1231-1235</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free energy calcns. are increasingly being used to est. abs. and relative binding free energies of ligands to proteins.  However, computed free energies often appear to depend on the initial protein conformation, indicating incomplete sampling.  This is esp. true when proteins can change conformation on ligand binding, as free energies assocd. with these conformational changes are either ignored or assumed to be included by virtue of the sampling performed in the calcn.  Here, the authors show that, in a model protein system (a designed binding site in T4 lysozyme), conformational changes can make a difference of several kcal/mol in computed binding free energies and that they are neglected in computed binding free energies if the system remains kinetically trapped in a particular metastable state on simulation timescales.  The authors introduce a general "confine-and-release" framework for free energy calcns. that accounts for these free energies of conformational change.  The authors illustrate its use in this model system by demonstrating that an umbrella sampling protocol can obtain converged binding free energies that are independent of the starting protein structure and include these conformational change free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHdRFLUwdd7LVg90H21EOLACvtfcHk0liwO2DE-sw82g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D&md5=fa953a85593d81ee192ff6c5d7f525c2</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1021%2Fct700032n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700032n%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DThe%2520confine-and-release%2520method%253A%2520obtaining%2520correct%2520binding%2520free%2520energies%2520in%2520the%2520presence%2520of%2520protein%2520conformational%2520change%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D1231%26epage%3D1235%26doi%3D10.1021%2Fct700032n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref197"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref197'); return false;" data-citation="" class="refNumLink">197</a></strong><div class="NLM_citation" id="cit197"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragliano, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Calculation of free energy landscape in multi-dimensions with hamiltonian-exchange umbrella sampling on petascale supercomputer</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">4672</span><span class="NLM_x">–</span> <span class="NLM_lpage">4680</span><span class="refDoi"> DOI: 10.1021/ct300468g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300468g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2iu7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=4672-4680&author=W.+Jiangauthor=Y.+Luoauthor=L.+Maraglianoauthor=B.+Roux&title=Calculation+of+free+energy+landscape+in+multi-dimensions+with+hamiltonian-exchange+umbrella+sampling+on+petascale+supercomputer&doi=10.1021%2Fct300468g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit197R"><div class="casContent"><span class="casTitleNuber">197</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of Free Energy Landscape in Multi-Dimensions with Hamiltonian-Exchange Umbrella Sampling on Petascale Supercomputer</span></div><div class="casAuthors">Jiang, Wei; Luo, Yun; Maragliano, Luca; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4672-4680</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An extremely scalable computational strategy is described for calcns. of the potential of mean force (PMF) in multidimensions on massively distributed supercomputers.  The approach involves coupling thousands of umbrella sampling (US) simulation windows distributed to cover the space of order parameters with a Hamiltonian mol. dynamics replica-exchange (H-REMD) algorithm to enhance the sampling of each simulation.  In the present application, US/H-REMD is carried out in a two-dimensional (2D) space and exchanges are attempted alternatively along the two axes corresponding to the two order parameters.  The US/H-REMD strategy is implemented on the basis of parallel/parallel multiple copy protocol at the MPI level, and therefore can fully exploit computing power of large-scale supercomputers.  Here the novel technique is illustrated using the leadership supercomputer IBM Blue Gene/P with an application to a typical biomol. calcn. of general interest, namely the binding of calcium ions to the small protein Calbindin D9k.  The free energy landscape assocd. with two order parameters, the distance between the ion and its binding pocket and the root-mean-square deviation (rmsd) of the binding pocket relative the crystal structure, was calcd. using the US/H-REMD method.  The results are then used to est. the abs. binding free energy of calcium ion to Calbindin D9k.  The tests demonstrate that the 2D US/H-REMD scheme greatly accelerates the configurational sampling of the binding pocket, thereby improving the convergence of the potential of mean force calcn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0t1ZtE0yTQ7Vg90H21EOLACvtfcHk0liaheL_SbjSQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2iu7bE&md5=b0bb012cc6564e67cc030a8b7669cd05</span></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1021%2Fct300468g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300468g%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DMaragliano%26aufirst%3DL.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DCalculation%2520of%2520free%2520energy%2520landscape%2520in%2520multi-dimensions%2520with%2520hamiltonian-exchange%2520umbrella%2520sampling%2520on%2520petascale%2520supercomputer%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D4672%26epage%3D4680%26doi%3D10.1021%2Fct300468g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref198"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref198'); return false;" data-citation="" class="refNumLink">198</a></strong><div class="NLM_citation" id="cit198"><span><span class="NLM_contrib-group">Sutto, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Effects of oncogenic mutations on the conformational free-energy landscape of EGFR kinase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">10616</span><span class="NLM_x">–</span> <span class="NLM_lpage">10621</span><span class="refDoi"> DOI: 10.1073/pnas.1221953110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1221953110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23754386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFejs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=10616-10621&author=L.+Suttoauthor=F.+L.+Gervasio&title=Effects+of+oncogenic+mutations+on+the+conformational+free-energy+landscape+of+EGFR+kinase&doi=10.1073%2Fpnas.1221953110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of oncogenic mutations on the conformational free-energy landscape of EGFR kinase</span></div><div class="casAuthors">Sutto, Ludovico; Gervasio, Francesco Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10616-10621,S10616/1-S10616/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Activating mutations in the epidermal growth factor receptor (EGFR) tyrosine kinase are frequently found in many cancers.  It has been suggested that changes in the equil. between its active and inactive conformations are linked to its oncogenic potential.  Here, we quantify the effects of some of the most common single (L858R and T790M) and double (T790M-L858R) oncogenic mutations on the conformational free energy landscape of the EGFR kinase domain by using massive mol. dynamics simulations together with parallel tempering, metadynamics, and one of the best force-fields available.  Whereas the wild-type EGFR catalytic domain monomer is mostly found in an inactive conformation, our results show a clear shift toward the active conformation for all of the mutants.  The L858R mutation stabilizes the active conformation at the expense of the inactive conformation and rigidifies the αC-helix.  The T790M gatekeeper mutant favors activation by stabilizing a hydrophobic cluster.  Finally, T790M with L858R shows a significant pos. epistasis effect.  This combination not only stabilizes the active conformation, but in nontrivial ways changes the free-energy landscape lowering the transition barriers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXpgTGgqXQwbVg90H21EOLACvtfcHk0liaheL_SbjSQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFejs73F&md5=4f582e8bcf5fc22c9499fbc6163ca725</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221953110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221953110%26sid%3Dliteratum%253Aachs%26aulast%3DSutto%26aufirst%3DL.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DEffects%2520of%2520oncogenic%2520mutations%2520on%2520the%2520conformational%2520free-energy%2520landscape%2520of%2520EGFR%2520kinase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D10616%26epage%3D10621%26doi%3D10.1073%2Fpnas.1221953110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref199"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref199'); return false;" data-citation="" class="refNumLink">199</a></strong><div class="NLM_citation" id="cit199"><span><span class="NLM_contrib-group">Lapelosa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallicchio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">Conformational transitions and convergence of absolute binding free energy calculations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1021/ct200684b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200684b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKhurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=47-60&author=M.+Lapelosaauthor=E.+Gallicchioauthor=R.+M.+Levy&title=Conformational+transitions+and+convergence+of+absolute+binding+free+energy+calculations&doi=10.1021%2Fct200684b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit199R"><div class="casContent"><span class="casTitleNuber">199</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Transitions and Convergence of Absolute Binding Free Energy Calculations</span></div><div class="casAuthors">Lapelosa, Mauro; Gallicchio, Emilio; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-60</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Binding energy distribution anal. method (BEDAM) is employed to compute the std. binding free energies of a series of ligands to a FK506 binding protein (FKBP12) with implicit solvation.  Binding free energy ests. are in reasonably good agreement with exptl. affinities.  The conformations of the complexes identified by the simulations are in good agreement with crystallog. data, which were not used to restrain ligand orientations.  The BEDAM method is based on λ-hopping Hamiltonian parallel replica exchange (HREM) mol. dynamics conformational sampling, the OPLS-AA/AGBNP2 effective potential, and multistate free energy estimators (MBAR).  Achieving converged and accurate results depends on all of these elements of the calcn.  Convergence of the binding free energy is tied to the level of convergence of binding energy distributions at crit. intermediate states where bound and unbound states are at equil., and where the rate of binding/unbinding conformational transitions is maximal.  This finding mirrors similar observations in the context of order/disorder transitions as for example in protein folding.  Insights concerning the phys. mechanism of ligand binding and unbinding are obtained.  Convergence for the largest FK506 ligand is achieved only after imposing strict conformational restraints, which however require accurate prior structural knowledge of the structure of the complex.  The anal. AGBNP2 model is found to underestimate the magnitude of the hydrophobic driving force toward binding in these systems characterized by loosely packed protein-ligand binding interfaces.  Rescoring of the binding energies using a numerical surface area model corrects this deficiency.  This study illustrates the complex interplay between energy models, exploration of conformational space, and free energy estimators needed to obtain robust ests. from binding free energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdq9cmSAvLWbVg90H21EOLACvtfcHk0liaheL_SbjSQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKhurnF&md5=36d9b9a68595d5bd28cea4cf79222960</span></div><a href="/servlet/linkout?suffix=cit199&amp;dbid=16384&amp;doi=10.1021%2Fct200684b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200684b%26sid%3Dliteratum%253Aachs%26aulast%3DLapelosa%26aufirst%3DM.%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DConformational%2520transitions%2520and%2520convergence%2520of%2520absolute%2520binding%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D47%26epage%3D60%26doi%3D10.1021%2Fct200684b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref200"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref200'); return false;" data-citation="" class="refNumLink">200</a></strong><div class="NLM_citation" id="cit200"><span><span class="NLM_contrib-group">Procacci, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bizzarri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsili, S.</span><span> </span><span class="NLM_article-title">Energy-driven undocking (EDU-HREM) in solute tempering replica exchange simulations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span><span class="refDoi"> DOI: 10.1021/ct400809n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400809n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslymsrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=439-450&author=P.+Procacciauthor=M.+Bizzarriauthor=S.+Marsili&title=Energy-driven+undocking+%28EDU-HREM%29+in+solute+tempering+replica+exchange+simulations&doi=10.1021%2Fct400809n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit200R"><div class="casContent"><span class="casTitleNuber">200</span><div class="casTitle"><span class="NLM_cas:atitle">Energy-Driven Undocking (EDU-HREM) in Solute Tempering Replica Exchange Simulations</span></div><div class="casAuthors">Procacci, Piero; Bizzarri, Marco; Marsili, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new computational strategy for calcg. the abs. binding free energy for protein ligand assocn. in the context of atomistic simulation in explicit solvent.  The method is based on an appropriate definition of a solute tempering scheme enforced via Hamilton replica exchange method (HREM).  The definition of "solute" includes both the ligand and the active site, with the remainder of the systems defined as "solvent".  The hydrophilicity of the solute and the solute torsional plus nonbonded intrasolute interactions are increased and decreased, resp., along the replica progression, thus favoring the extrusion of the drug form the active site in the scaled states of the generalized ensemble.  The proposed technique, named "Energy Driven Undocking" (EDU-HREM), completely bypasses the need for defining and/or identifying the relevant reaction coordinates in a ligand receptor interactions and allows the calcn. of the abs. binding free energy in one single generalized simulation of the drug-receptor system.  The methodol. is applied, with encouraging results, to the calcn. of the abs. binding free energy of some FK506-related ligands of the peptidyl prolyl cis-trans isomerase protein (FKBP12) with known dissocn. consts.  Aspects of the binding/inhibition mechanism in FKBP12 are also analyzed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5AA0dW54SJrVg90H21EOLACvtfcHk0liaheL_SbjSQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslymsrrI&md5=58271181ad379a332b75901b9775b50f</span></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1021%2Fct400809n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400809n%26sid%3Dliteratum%253Aachs%26aulast%3DProcacci%26aufirst%3DP.%26aulast%3DBizzarri%26aufirst%3DM.%26aulast%3DMarsili%26aufirst%3DS.%26atitle%3DEnergy-driven%2520undocking%2520%2528EDU-HREM%2529%2520in%2520solute%2520tempering%2520replica%2520exchange%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D439%26epage%3D450%26doi%3D10.1021%2Fct400809n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref201"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref201'); return false;" data-citation="" class="refNumLink">201</a></strong><div class="NLM_citation" id="cit201"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Opinion - Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">739</span><span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Opinion+-+Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit201&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DOpinion%2520-%2520Drug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref202"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref202'); return false;" data-citation="" class="refNumLink">202</a></strong><div class="NLM_citation" id="cit202"><span><span class="NLM_contrib-group">Pan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Molecular determinants of drug-receptor binding kinetics</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">673</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2013.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23454741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=667-673&author=A.+C.+Panauthor=D.+W.+Borhaniauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Molecular+determinants+of+drug-receptor+binding+kinetics&doi=10.1016%2Fj.drudis.2013.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit202R"><div class="casContent"><span class="casTitleNuber">202</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of drug-receptor binding kinetics</span></div><div class="casAuthors">Pan, Albert C.; Borhani, David W.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">667-673</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  It is increasingly appreciated that the rates at which drugs assoc. with and dissoc. from receptors - the binding kinetics - directly impact drug efficacy and safety.  The mol. determinants of drug-receptor binding kinetics remain poorly understood, however, esp. when compared with the well-known factors that affect binding affinity.  The rational modulation of kinetics during lead optimization thus remains challenging.  We review some of the key factors thought to control drug-receptor binding kinetics at the mol. level - mol. size, conformational fluctuations, electrostatic interactions and hydrophobic effects - and discuss several possible approaches for the rational design of drugs with desired binding kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvEt3tx06Z7Vg90H21EOLACvtfcHk0lizApiAuRcWNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D&md5=df094bb86b1e280b9ddf5a90031b3f32</span></div><a href="/servlet/linkout?suffix=cit202&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DMolecular%2520determinants%2520of%2520drug-receptor%2520binding%2520kinetics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D667%26epage%3D673%26doi%3D10.1016%2Fj.drudis.2013.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref203"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref203'); return false;" data-citation="" class="refNumLink">203</a></strong><div class="NLM_citation" id="cit203"><span><span class="NLM_contrib-group">Zhou, H. X.</span><span> </span><span class="NLM_article-title">Rate theories for biologists</span> <span class="citation_source-journal">Q. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">219</span><span class="NLM_x">–</span> <span class="NLM_lpage">293</span><span class="refDoi"> DOI: 10.1017/S0033583510000120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1017%2FS0033583510000120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20691138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC3cjpt1alug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=219-293&author=H.+X.+Zhou&title=Rate+theories+for+biologists&doi=10.1017%2FS0033583510000120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit203R"><div class="casContent"><span class="casTitleNuber">203</span><div class="casTitle"><span class="NLM_cas:atitle">Rate theories for biologists</span></div><div class="casAuthors">Zhou Huan-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly reviews of biophysics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">219-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Some of the rate theories that are most useful for modeling biological processes are reviewed.  By delving into some of the details and subtleties in the development of the theories, the review will hopefully help the reader gain a more than superficial perspective.  Examples are presented to illustrate how rate theories can be used to generate insight at the microscopic level into biomolecular behaviors.  An attempt is made to clear up a number of misconceptions in the literature regarding popular rate theories, including the appearance of Planck's constant in the transition-state theory and the Smoluchowski result as an upper limit for protein-protein and protein-DNA association rate constants.  Future work in combining the implementation of rate theories through computer simulations with experimental probes of rate processes, and in modeling effects of intracellular environments so that theories can be used for generating rate constants for systems biology studies is particularly exciting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRI95WOrBhtK9FxYDaSSP8XfW6udTcc2ebwbLusciszkbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjpt1alug%253D%253D&md5=6af727b8039583014ca9f70e6c6cfee8</span></div><a href="/servlet/linkout?suffix=cit203&amp;dbid=16384&amp;doi=10.1017%2FS0033583510000120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583510000120%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%2BX.%26atitle%3DRate%2520theories%2520for%2520biologists%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2010%26volume%3D43%26spage%3D219%26epage%3D293%26doi%3D10.1017%2FS0033583510000120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref204"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref204'); return false;" data-citation="" class="refNumLink">204</a></strong><div class="NLM_citation" id="cit204"><span><span class="NLM_contrib-group">Pande, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, G. R.</span><span> </span><span class="NLM_article-title">Everything you wanted to know about markov state models but were afraid to ask</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2010.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.ymeth.2010.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20570730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2010&pages=99-105&author=V.+S.+Pandeauthor=K.+Beauchampauthor=G.+R.+Bowman&title=Everything+you+wanted+to+know+about+markov+state+models+but+were+afraid+to+ask&doi=10.1016%2Fj.ymeth.2010.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit204R"><div class="casContent"><span class="casTitleNuber">204</span><div class="casTitle"><span class="NLM_cas:atitle">Everything you wanted to know about Markov State Models but were afraid to ask</span></div><div class="casAuthors">Pande, Vijay S.; Beauchamp, Kyle; Bowman, Gregory R.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-105</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Simulating protein folding has been a challenging problem for decades due to the long timescales involved (compared with what is possible to simulate) and the challenges of gaining insight from the complex nature of the resulting simulation data.  Markov State Models (MSMs) present a means to tackle both of these challenges, yielding simulations on exptl. relevant timescales, statistical significance, and coarse grained representations that are readily humanly understandable.  Here, we review this method with the intended audience of non-experts, in order to introduce the method to a broader audience.  We review the motivations, methods, and caveats of MSMs, as well as some recent highlights of applications of the method.  We conclude by discussing how this approach is part of a paradigm shift in how one uses simulations, away from anecdotal single-trajectory approaches to a more comprehensive statistical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFtkKipM46Z7Vg90H21EOLACvtfcHk0lizApiAuRcWNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsLfK&md5=2afa98b7af7c65fb39a8cb61b5233f61</span></div><a href="/servlet/linkout?suffix=cit204&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2010.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2010.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DPande%26aufirst%3DV.%2BS.%26aulast%3DBeauchamp%26aufirst%3DK.%26aulast%3DBowman%26aufirst%3DG.%2BR.%26atitle%3DEverything%2520you%2520wanted%2520to%2520know%2520about%2520markov%2520state%2520models%2520but%2520were%2520afraid%2520to%2520ask%26jtitle%3DMethods%26date%3D2010%26volume%3D52%26spage%3D99%26epage%3D105%26doi%3D10.1016%2Fj.ymeth.2010.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref205"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref205'); return false;" data-citation="" class="refNumLink">205</a></strong><div class="NLM_citation" id="cit205"><span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noe, F.</span><span> </span><span class="NLM_article-title">Markov state models of biomolecular conformational dynamics</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">135</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span><span class="refDoi"> DOI: 10.1016/j.sbi.2014.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2014.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24836551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovFChu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=135-144&author=J.+D.+Choderaauthor=F.+Noe&title=Markov+state+models+of+biomolecular+conformational+dynamics&doi=10.1016%2Fj.sbi.2014.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit205R"><div class="casContent"><span class="casTitleNuber">205</span><div class="casTitle"><span class="NLM_cas:atitle">Markov state models of biomolecular conformational dynamics</span></div><div class="casAuthors">Chodera, John D.; Noe, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">135-144</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  It has recently become practical to construct Markov state models (MSMs) that reproduce the long-time statistical conformational dynamics of biomols. using data from mol. dynamics simulations.  MSMs can predict both stationary and kinetic quantities on long timescales (e.g. milliseconds) using a set of atomistic mol. dynamics simulations that are individually much shorter, thus addressing the well-known sampling problem in mol. dynamics simulation.  In addn. to providing predictive quant. models, MSMs greatly facilitate both the extn. of insight into biomol. mechanism (such as folding and functional dynamics) and quant. comparison with single-mol. and ensemble kinetics expts.  A variety of methodol. advances and software packages now bring the construction of these models closer to routine practice.  Here, we review recent progress in this field, considering theor. and methodol. advances, new software tools, and recent applications of these approaches in several domains of biochem. and biophysics, commenting on remaining challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyht1KHY5Z7Vg90H21EOLACvtfcHk0lghGEMH2jQq_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovFChu7s%253D&md5=4aab299d1bec5b92acc554341bc89b16</span></div><a href="/servlet/linkout?suffix=cit205&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2014.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2014.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DNoe%26aufirst%3DF.%26atitle%3DMarkov%2520state%2520models%2520of%2520biomolecular%2520conformational%2520dynamics%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D25%26spage%3D135%26epage%3D144%26doi%3D10.1016%2Fj.sbi.2014.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref206"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref206'); return false;" data-citation="" class="refNumLink">206</a></strong><div class="NLM_citation" id="cit206"><span><span class="NLM_contrib-group">Gu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span> </span><span class="NLM_article-title">Quantitatively characterizing the ligand binding mechanisms of choline binding protein using Markov state model analysis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003767</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pcbi.1003767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25101697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ars73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003767&author=S.+Guauthor=D.+A.+Silvaauthor=L.+Mengauthor=A.+Yueauthor=X.+Huang&title=Quantitatively+characterizing+the+ligand+binding+mechanisms+of+choline+binding+protein+using+Markov+state+model+analysis&doi=10.1371%2Fjournal.pcbi.1003767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit206R"><div class="casContent"><span class="casTitleNuber">206</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitatively characterizing the ligand binding mechanisms of choline binding protein using markov state model analysis</span></div><div class="casAuthors">Gu, Shuo; Silva, Daniel-Adriano; Meng, Luming; Yue, Alexander; Huang, Xuhui</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1003767/1-e1003767/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein-ligand recognition plays key roles in many biol. processes.  One of the most fascinating questions about protein-ligand recognition is to understand its underlying mechanism, which often results from a combination of induced fit and conformational selection.  In this study, we have developed a three-pronged approach of Markov State Models, Mol. Dynamics simulations, and flux anal. to det. the contribution of each model.  Using this approach, we have quantified the recognition mechanism of the choline binding protein (ChoX) to be ∼90% conformational selection dominant under exptl. conditions.  This is achieved by recovering all the necessary parameters for the flux anal. in combination with available exptl. data.  Our results also suggest that ChoX has several metastable conformational states, of which an apo-closed state is dominant, consistent with previous exptl. findings.  Our methodol. holds great potential to be widely applied to understand recognition mechanisms underlining many fundamental biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosNNPrkzNMfbVg90H21EOLACvtfcHk0lghGEMH2jQq_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ars73E&md5=a59ce9737dc4afa610544e0e18f5edcc</span></div><a href="/servlet/linkout?suffix=cit206&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003767%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DS.%26aulast%3DSilva%26aufirst%3DD.%2BA.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DQuantitatively%2520characterizing%2520the%2520ligand%2520binding%2520mechanisms%2520of%2520choline%2520binding%2520protein%2520using%2520Markov%2520state%2520model%2520analysis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003767%26doi%3D10.1371%2Fjournal.pcbi.1003767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref207"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref207'); return false;" data-citation="" class="refNumLink">207</a></strong><div class="NLM_citation" id="cit207"><span><span class="NLM_contrib-group">Plattner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noe, F.</span><span> </span><span class="NLM_article-title">Protein conformational plasticity and complex ligand-binding kinetics explored by atomistic simulations and Markov models</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">7653</span><span class="refDoi"> DOI: 10.1038/ncomms8653</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fncomms8653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26134632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC28%252Fgs1ajug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7653&author=N.+Plattnerauthor=F.+Noe&title=Protein+conformational+plasticity+and+complex+ligand-binding+kinetics+explored+by+atomistic+simulations+and+Markov+models&doi=10.1038%2Fncomms8653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit207R"><div class="casContent"><span class="casTitleNuber">207</span><div class="casTitle"><span class="NLM_cas:atitle">Protein conformational plasticity and complex ligand-binding kinetics explored by atomistic simulations and Markov models</span></div><div class="casAuthors">Plattner Nuria; Noe Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7653</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Understanding the structural mechanisms of protein-ligand binding and their dependence on protein sequence and conformation is of fundamental importance for biomedical research.  Here we investigate the interplay of conformational change and ligand-binding kinetics for the serine protease Trypsin and its competitive inhibitor Benzamidine with an extensive set of 150 μs molecular dynamics simulation data, analysed using a Markov state model.  Seven metastable conformations with different binding pocket structures are found that interconvert at timescales of tens of microseconds.  These conformations differ in their substrate-binding affinities and binding/dissociation rates.  For each metastable state, corresponding solved structures of Trypsin mutants or similar serine proteases are contained in the protein data bank.  Thus, our wild-type simulations explore a space of conformations that can be individually stabilized by adding ligands or making suitable changes in protein sequence.  These findings provide direct evidence of conformational plasticity in receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUuRoXD3S0w5ZoeNcnHOQ9fW6udTcc2eY9gaHV9jwMY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252Fgs1ajug%253D%253D&md5=4d2c200f1a134d8448e1ae1588dce791</span></div><a href="/servlet/linkout?suffix=cit207&amp;dbid=16384&amp;doi=10.1038%2Fncomms8653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8653%26sid%3Dliteratum%253Aachs%26aulast%3DPlattner%26aufirst%3DN.%26aulast%3DNoe%26aufirst%3DF.%26atitle%3DProtein%2520conformational%2520plasticity%2520and%2520complex%2520ligand-binding%2520kinetics%2520explored%2520by%2520atomistic%2520simulations%2520and%2520Markov%2520models%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7653%26doi%3D10.1038%2Fncomms8653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref208"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref208'); return false;" data-citation="" class="refNumLink">208</a></strong><div class="NLM_citation" id="cit208"><span><span class="NLM_contrib-group">Doerr, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">On-the-Fly Learning and sampling of ligand binding by high-throughput molecular simulations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2064</span><span class="NLM_x">–</span> <span class="NLM_lpage">2069</span><span class="refDoi"> DOI: 10.1021/ct400919u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400919u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvVyrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=2064-2069&author=S.+Doerrauthor=G.+De+Fabritiis&title=On-the-Fly+Learning+and+sampling+of+ligand+binding+by+high-throughput+molecular+simulations&doi=10.1021%2Fct400919u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit208R"><div class="casContent"><span class="casTitleNuber">208</span><div class="casTitle"><span class="NLM_cas:atitle">On-the-Fly Learning and Sampling of Ligand Binding by High-Throughput Molecular Simulations</span></div><div class="casAuthors">Doerr, S.; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2064-2069</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput mol. dynamics (MD) simulations are a computational method consisting of using multiple short trajectories, instead of few long ones, to cover slow biol. time scales.  Compared to long trajectories this method offers the possibility to start the simulations in successive batches, building a knowledgeable model of the available data to inform subsequent new simulations iteratively.  Here, we demonstrate an automatic, iterative, on-the-fly method for learning and sampling mol. simulations in the context of ligand binding for the case of trypsin-benzamidine binding.  The method uses Markov state models to learn a simplified model of the simulations and decide where best to sample from, achieving a converged binding affinity in approx. one microsecond, 1 order of magnitude faster than classical sampling.  This method demonstrates for the first time the potential of adaptive sampling schemes in the case of ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5z81HNCApn7Vg90H21EOLACvtfcHk0lhmWxM1EXgdkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvVyrtrg%253D&md5=7ff57bf52252341784fe709f8906e214</span></div><a href="/servlet/linkout?suffix=cit208&amp;dbid=16384&amp;doi=10.1021%2Fct400919u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400919u%26sid%3Dliteratum%253Aachs%26aulast%3DDoerr%26aufirst%3DS.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DOn-the-Fly%2520Learning%2520and%2520sampling%2520of%2520ligand%2520binding%2520by%2520high-throughput%2520molecular%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D2064%26epage%3D2069%26doi%3D10.1021%2Fct400919u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref209"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref209'); return false;" data-citation="" class="refNumLink">209</a></strong><div class="NLM_citation" id="cit209"><span><span class="NLM_contrib-group">Lu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span> </span><span class="NLM_article-title">Drug-target residence time: critical information for lead optimization</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.06.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.cbpa.2010.06.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20663707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=467-474&author=H.+Luauthor=P.+J.+Tonge&title=Drug-target+residence+time%3A+critical+information+for+lead+optimization&doi=10.1016%2Fj.cbpa.2010.06.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit209R"><div class="casContent"><span class="casTitleNuber">209</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time: Critical information for lead optimization</span></div><div class="casAuthors">Lu, Hao; Tonge, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Failure due to poor in vivo efficacy is a primary contributor to attrition during the development of new chemotherapeutics.  Lead optimization programs that in their quest for efficacy focus solely on improving the affinity of drug-target binding are flawed, since this approach ignores the fluctuations in drug concn. that occur in vivo.  Instead the lifetime of the drug-target complex must also be considered, since drugs only act when they are bound to their targets.  Consequently, to improve the correlation between the in vitro and in vivo activity of drugs, measurements of drug-target residence time must be incorporated into the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk2kN_txuMLVg90H21EOLACvtfcHk0lhmWxM1EXgdkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D&md5=3189f56032106db858cfb4269d3fc11f</span></div><a href="/servlet/linkout?suffix=cit209&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.06.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.06.176%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DDrug-target%2520residence%2520time%253A%2520critical%2520information%2520for%2520lead%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D467%26epage%3D474%26doi%3D10.1016%2Fj.cbpa.2010.06.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref210"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref210'); return false;" data-citation="" class="refNumLink">210</a></strong><div class="NLM_citation" id="cit210"><span><span class="NLM_contrib-group">Mollica, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decherchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zia, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaspari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span> </span><span class="NLM_article-title">Kinetics of protein-ligand unbinding via smoothed potential molecular dynamics simulations</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">11539</span><span class="refDoi"> DOI: 10.1038/srep11539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fsrep11539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26103621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11539&author=L.+Mollicaauthor=S.+Decherchiauthor=S.+R.+Ziaauthor=R.+Gaspariauthor=A.+Cavalliauthor=W.+Rocchia&title=Kinetics+of+protein-ligand+unbinding+via+smoothed+potential+molecular+dynamics+simulations&doi=10.1038%2Fsrep11539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit210R"><div class="casContent"><span class="casTitleNuber">210</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of protein-ligand unbinding via smoothed potential molecular dynamics simulations</span></div><div class="casAuthors">Mollica Luca; Decherchi Sergio; Zia Syeda Rehana; Gaspari Roberto; Rocchia Walter; Cavalli Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug discovery is expensive and high-risk.  Its main reasons of failure are lack of efficacy and toxicity of a drug candidate.  Binding affinity for the biological target has been usually considered one of the most relevant figures of merit to judge a drug candidate along with bioavailability, selectivity and metabolic properties, which could depend on off-target interactions.  Nevertheless, affinity does not always satisfactorily correlate with in vivo drug efficacy.  It is indeed becoming increasingly evident that the time a drug spends in contact with its target (aka residence time) can be a more reliable figure of merit.  Experimental kinetic measurements are operatively limited by the cost and the time needed to synthesize compounds to be tested, to express and purify the target, and to setup the assays.  We present here a simple and efficient molecular-dynamics-based computational approach to prioritize compounds according to their residence time.  We devised a multiple-replica scaled molecular dynamics protocol with suitably defined harmonic restraints to accelerate the unbinding events while preserving the native fold.  Ligands are ranked according to the mean observed scaled unbinding time.  The approach, trivially parallel and easily implementable, was validated against experimental information available on biological systems of pharmacological relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxJkC0jZN2SEH3Rlp_k91zfW6udTcc2eahYQ5A6TG987ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D&md5=8e3963bce57cf49954ea50fc004283d6</span></div><a href="/servlet/linkout?suffix=cit210&amp;dbid=16384&amp;doi=10.1038%2Fsrep11539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11539%26sid%3Dliteratum%253Aachs%26aulast%3DMollica%26aufirst%3DL.%26aulast%3DDecherchi%26aufirst%3DS.%26aulast%3DZia%26aufirst%3DS.%2BR.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DKinetics%2520of%2520protein-ligand%2520unbinding%2520via%2520smoothed%2520potential%2520molecular%2520dynamics%2520simulations%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11539%26doi%3D10.1038%2Fsrep11539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref211"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref211'); return false;" data-citation="" class="refNumLink">211</a></strong><div class="NLM_citation" id="cit211"><span><span class="NLM_contrib-group">Bui, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The dynamics of ligand barrier crossing inside the acetylcholinesterase gorge</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2267</span><span class="NLM_x">–</span> <span class="NLM_lpage">2272</span><span class="refDoi"> DOI: 10.1016/S0006-3495(03)74651-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0006-3495%2803%2974651-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2003&pages=2267-2272&author=J.+M.+Buiauthor=R.+H.+Henchmanauthor=J.+A.+McCammon&title=The+dynamics+of+ligand+barrier+crossing+inside+the+acetylcholinesterase+gorge&doi=10.1016%2FS0006-3495%2803%2974651-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit211&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2803%2974651-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252803%252974651-7%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DJ.%2BM.%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520dynamics%2520of%2520ligand%2520barrier%2520crossing%2520inside%2520the%2520acetylcholinesterase%2520gorge%26jtitle%3DBiophys.%2520J.%26date%3D2003%26volume%3D85%26spage%3D2267%26epage%3D2272%26doi%3D10.1016%2FS0006-3495%2803%2974651-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref212"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref212'); return false;" data-citation="" class="refNumLink">212</a></strong><div class="NLM_citation" id="cit212"><span><span class="NLM_contrib-group">Marinelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietrucci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piana, S.</span><span> </span><span class="NLM_article-title">A kinetic model of trp-cage folding from multiple biased molecular dynamics simulations</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000452</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pcbi.1000452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19662155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BD1MrktFSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000452&author=F.+Marinelliauthor=F.+Pietrucciauthor=A.+Laioauthor=S.+Piana&title=A+kinetic+model+of+trp-cage+folding+from+multiple+biased+molecular+dynamics+simulations&doi=10.1371%2Fjournal.pcbi.1000452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit212R"><div class="casContent"><span class="casTitleNuber">212</span><div class="casTitle"><span class="NLM_cas:atitle">A kinetic model of trp-cage folding from multiple biased molecular dynamics simulations</span></div><div class="casAuthors">Marinelli Fabrizio; Pietrucci Fabio; Laio Alessandro; Piana Stefano</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1000452</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Trp-cage is a designed 20-residue polypeptide that, in spite of its size, shares several features with larger globular proteins.Although the system has been intensively investigated experimentally and theoretically, its folding mechanism is not yet fully understood.  Indeed, some experiments suggest a two-state behavior, while others point to the presence of intermediates.  In this work we show that the results of a bias-exchange metadynamics simulation can be used for constructing a detailed thermodynamic and kinetic model of the system.  The model, although constructed from a biased simulation, has a quality similar to those extracted from the analysis of long unbiased molecular dynamics trajectories.  This is demonstrated by a careful benchmark of the approach on a smaller system, the solvated Ace-Ala3-Nme peptide.  For theTrp-cage folding, the model predicts that the relaxation time of 3100 ns observed experimentally is due to the presence of a compact molten globule-like conformation.  This state has an occupancy of only 3% at 300 K, but acts as a kinetic trap.Instead, non-compact structures relax to the folded state on the sub-microsecond timescale.  The model also predicts the presence of a state at Calpha-RMSD of 4.4 A from the NMR structure in which the Trp strongly interacts with Pro12.  This state can explain the abnormal temperature dependence of the Pro12-delta3 and Gly11-alpha3 chemical shifts.  The structures of the two most stable misfolded intermediates are in agreement with NMR experiments on the unfolded protein.  Our work shows that, using biased molecular dynamics trajectories, it is possible to construct a model describing in detail the Trp-cage folding kinetics and thermodynamics in agreement with experimental data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZmzIeeqUN7TMsgLhIoICsfW6udTcc2eahYQ5A6TG987ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrktFSitg%253D%253D&md5=42e8c3d2eeb63936b027aac05072325e</span></div><a href="/servlet/linkout?suffix=cit212&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000452%26sid%3Dliteratum%253Aachs%26aulast%3DMarinelli%26aufirst%3DF.%26aulast%3DPietrucci%26aufirst%3DF.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DPiana%26aufirst%3DS.%26atitle%3DA%2520kinetic%2520model%2520of%2520trp-cage%2520folding%2520from%2520multiple%2520biased%2520molecular%2520dynamics%2520simulations%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000452%26doi%3D10.1371%2Fjournal.pcbi.1000452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref213"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref213'); return false;" data-citation="" class="refNumLink">213</a></strong><div class="NLM_citation" id="cit213"><span><span class="NLM_contrib-group">Pietrucci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marinelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carloni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span> </span><span class="NLM_article-title">Substrate binding mechanism of hiv-1 protease from explicit-solvent atomistic simulations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">11811</span><span class="NLM_x">–</span> <span class="NLM_lpage">11818</span><span class="refDoi"> DOI: 10.1021/ja903045y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja903045y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1GqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=11811-11818&author=F.+Pietrucciauthor=F.+Marinelliauthor=P.+Carloniauthor=A.+Laio&title=Substrate+binding+mechanism+of+hiv-1+protease+from+explicit-solvent+atomistic+simulations&doi=10.1021%2Fja903045y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit213R"><div class="casContent"><span class="casTitleNuber">213</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Binding Mechanism of HIV-1 Protease from Explicit-Solvent Atomistic Simulations</span></div><div class="casAuthors">Pietrucci, Fabio; Marinelli, Fabrizio; Carloni, Paolo; Laio, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11811-11818</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding mechanism of a peptide substrate (Thr-Ile-Met-Met-Gln-Arg, cleavage site p2-NC of the viral polyprotein) to wild-type HIV-1 protease has been investigated by 1.6 μs biased all-atom mol. dynamics (MD) simulations in explicit water.  The configuration space has been explored biasing seven reaction coordinates by the bias-exchange metadynamics technique.  The structure of the Michaelis complex is obtained starting from the substrate outside the enzyme within a backbone rmsd of 0.9 Å.  The calcd. free energy of binding is -6 kcal/mol, and the kinetic consts. for assocn. and dissocn. are 1.3 × 106 M-1 s-1 and 57 s-1, resp., consistent with expts.  In the main binding pathway, the flaps of the protease do not open sizably.  The substrate slides inside the enzyme cavity from the tight lateral channel.  This may contrast with the natural polyprotein substrate which is expected to bind by opening the flaps.  Thus, mutations might influence differently the binding kinetics of peptidomimetic ligands and of the natural substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXYaLQMq4PG7Vg90H21EOLACvtfcHk0liAWXmWg9K0zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1GqtLs%253D&md5=0a0ff8c4a988a4518566258855cb56fe</span></div><a href="/servlet/linkout?suffix=cit213&amp;dbid=16384&amp;doi=10.1021%2Fja903045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja903045y%26sid%3Dliteratum%253Aachs%26aulast%3DPietrucci%26aufirst%3DF.%26aulast%3DMarinelli%26aufirst%3DF.%26aulast%3DCarloni%26aufirst%3DP.%26aulast%3DLaio%26aufirst%3DA.%26atitle%3DSubstrate%2520binding%2520mechanism%2520of%2520hiv-1%2520protease%2520from%2520explicit-solvent%2520atomistic%2520simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D11811%26epage%3D11818%26doi%3D10.1021%2Fja903045y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref214"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref214'); return false;" data-citation="" class="refNumLink">214</a></strong><div class="NLM_citation" id="cit214"><span><span class="NLM_contrib-group">Tiwary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">From Metadynamics to Dynamics</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">230602</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.111.230602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.111.230602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24476246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVSrs7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2013&pages=230602&author=P.+Tiwaryauthor=M.+Parrinello&title=From+Metadynamics+to+Dynamics&doi=10.1103%2FPhysRevLett.111.230602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit214R"><div class="casContent"><span class="casTitleNuber">214</span><div class="casTitle"><span class="NLM_cas:atitle">From metadynamics to dynamics</span></div><div class="casAuthors">Tiwary, Pratyush; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">230602/1-230602/5</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">Metadynamics is a commonly used and successful enhanced sampling method.  By the introduction of a history dependent bias which depends on a restricted no. of collective variables it can explore complex free energy surfaces characterized by several metastable states sepd. by large free energy barriers.  Here we extend its scope by introducing a simple yet powerful method for calcg. the rates of transition between different metastable states.  The method does not rely on a previous knowledge of the transition states or reaction coordinates, as long as collective variables are known that can distinguish between the various stable min. in free energy space.  We demonstrate that our method recovers the correct escape rates out of these stable states and also preserves the correct sequence of state-to-state transitions, with minimal extra computational effort needed over ordinary metadynamics.  We apply the formalism to three different problems and in each case find excellent agreement with the results of long unbiased mol. dynamics runs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOKDPbiMlopbVg90H21EOLACvtfcHk0liAWXmWg9K0zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVSrs7nF&md5=3ccbf6ea725bd13d42f5bfb04cef1c51</span></div><a href="/servlet/linkout?suffix=cit214&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.111.230602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.111.230602%26sid%3Dliteratum%253Aachs%26aulast%3DTiwary%26aufirst%3DP.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFrom%2520Metadynamics%2520to%2520Dynamics%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2013%26volume%3D111%26spage%3D230602%26doi%3D10.1103%2FPhysRevLett.111.230602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref215"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref215'); return false;" data-citation="" class="refNumLink">215</a></strong><div class="NLM_citation" id="cit215"><span><span class="NLM_contrib-group">Salvalaglio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Assessing the Reliability of the dynamics reconstructed from metadynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1420</span><span class="NLM_x">–</span> <span class="NLM_lpage">1425</span><span class="refDoi"> DOI: 10.1021/ct500040r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500040r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref215/cit215&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtV2rurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1420-1425&author=M.+Salvalaglioauthor=P.+Tiwaryauthor=M.+Parrinello&title=Assessing+the+Reliability+of+the+dynamics+reconstructed+from+metadynamics&doi=10.1021%2Fct500040r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit215R"><div class="casContent"><span class="casTitleNuber">215</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Reliability of the Dynamics Reconstructed from Metadynamics</span></div><div class="casAuthors">Salvalaglio, Matteo; Tiwary, Pratyush; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1420-1425</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sampling a mol. process characterized by an activation free energy significantly larger than kBT is a well-known challenge in mol. dynamics simulations.  In a recent work, we have demonstrated that the transition times of activated mol. transformations can be computed from well-tempered metadynamics provided that no bias is deposited in the transition state region and that the set of collective variables chosen to enhance sampling does not display hysteresis.  Ensuring though that these two criteria are met may not always be simple.  Here we build on the fact that the times of escape from a long-lived metastable state obey Poisson statistics.  This allows us to identify quant. measures of trustworthiness of our calcn.  We test our method on a few paradigmatic examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF3cPWRv1ldbVg90H21EOLACvtfcHk0lifVeuC9ph9kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtV2rurg%253D&md5=04b0084a6f6ff1ea2ad30bf746553464</span></div><a href="/servlet/linkout?suffix=cit215&amp;dbid=16384&amp;doi=10.1021%2Fct500040r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500040r%26sid%3Dliteratum%253Aachs%26aulast%3DSalvalaglio%26aufirst%3DM.%26aulast%3DTiwary%26aufirst%3DP.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DAssessing%2520the%2520Reliability%2520of%2520the%2520dynamics%2520reconstructed%2520from%2520metadynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D1420%26epage%3D1425%26doi%3D10.1021%2Fct500040r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref216"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref216'); return false;" data-citation="" class="refNumLink">216</a></strong><div class="NLM_citation" id="cit216"><span><span class="NLM_contrib-group">Changeux, J. P.</span><span> </span><span class="NLM_article-title">Allostery and the monod-wyman-changeux model after 50 years</span> <span class="citation_source-journal">Annu. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1146/annurev-biophys-050511-102222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1146%2Fannurev-biophys-050511-102222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22224598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1yitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=103-133&author=J.+P.+Changeux&title=Allostery+and+the+monod-wyman-changeux+model+after+50+years&doi=10.1146%2Fannurev-biophys-050511-102222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit216R"><div class="casContent"><span class="casTitleNuber">216</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery and the Monod-Wyman-Changeux model after 50 years</span></div><div class="casAuthors">Changeux, Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-133</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The Monod-Wyman-Changeux (MWC) model was conceived in 1965 to account for the signal transduction and cooperative properties of bacterial regulatory enzymes and Hb.  It was soon extended to pharmacol. receptors for neurotransmitters and other macromol. entities involved in intracellular and intercellular communications.  Five decades later, the 2 main hypotheses of the model are reexamd. on the basis of a variety of regulatory proteins with known x-ray structures: (1) regulatory proteins possess an oligomeric structure with symmetry properties, and (2) the allosteric interactions between topog. distinct sites are mediated by a conformational transition established between a few preestablished states with conservation of symmetry and ligand-directed conformational selection.  Several well-documented examples are adequately represented by the MWC model, yet a few possible exceptions are noted.  New questions are raised concerning the dynamics of the allosteric transitions and more complex supramol. ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMZolzBzmVNLVg90H21EOLACvtfcHk0lifVeuC9ph9kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1yitbc%253D&md5=0df027444211c5471de1236271ffa76d</span></div><a href="/servlet/linkout?suffix=cit216&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-050511-102222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-050511-102222%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.%2BP.%26atitle%3DAllostery%2520and%2520the%2520monod-wyman-changeux%2520model%2520after%252050%2520years%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D41%26spage%3D103%26epage%3D133%26doi%3D10.1146%2Fannurev-biophys-050511-102222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref217"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref217'); return false;" data-citation="" class="refNumLink">217</a></strong><div class="NLM_citation" id="cit217"><span><span class="NLM_contrib-group">Tsai, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span> </span><span class="NLM_article-title">A Unified view of “how allostery works”</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003394</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pcbi.1003394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24516370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003394&author=C.+J.+Tsaiauthor=R.+Nussinov&title=A+Unified+view+of+%E2%80%9Chow+allostery+works%E2%80%9D&doi=10.1371%2Fjournal.pcbi.1003394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit217R"><div class="casContent"><span class="casTitleNuber">217</span><div class="casTitle"><span class="NLM_cas:atitle">A unified view of "how allostery works"</span></div><div class="casAuthors">Tsai, Chung-Jung; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1003394/1-e1003394/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The question of how allostery works was posed almost 50 years ago.  Since then it has been the focus of much effort.  This is for two reasons: first, the intellectual curiosity of basic science and the desire to understand fundamental phenomena, and second, its vast practical importance.  Allostery is at play in all processes in the living cell, and increasingly in drug discovery.  Many models have been successfully formulated, and are able to describe allostery even in the absence of a detailed structural mechanism.  However, conceptual schemes designed to qual. explain allosteric mechanisms usually lack a quant. math. model, and are unable to link its thermodn. and structural foundations.  This hampers insight into oncogenic mutations in cancer progression and biased agonists' actions.  Here, we describe how allostery works from three different standpoints: thermodn., free energy landscape of population shift, and structure; all with exactly the same allosteric descriptors.  This results in a unified view which not only clarifies the elusive allosteric mechanism but also provides structural grasp of agonist-mediated signaling pathways, and guides allosteric drug discovery.  Of note, the unified view reasons that allosteric coupling (or communication) does not det. the allosteric efficacy; however, a communication channel is what makes potential binding sites allosteric.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEgA5AKySrFbVg90H21EOLACvtfcHk0lifVeuC9ph9kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D&md5=3912bf89813eeed254b3e3e4e1937f99</span></div><a href="/servlet/linkout?suffix=cit217&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003394%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DA%2520Unified%2520view%2520of%2520%25E2%2580%259Chow%2520allostery%2520works%25E2%2580%259D%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003394%26doi%3D10.1371%2Fjournal.pcbi.1003394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref218"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref218'); return false;" data-citation="" class="refNumLink">218</a></strong><div class="NLM_citation" id="cit218"><span><span class="NLM_contrib-group">Lee, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span> </span><span class="NLM_article-title">Trapping moving targets with small molecules</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">324</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">215</span><span class="refDoi"> DOI: 10.1126/science.1169378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.1169378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19359579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFalsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=213-215&author=G.+M.+Leeauthor=C.+S.+Craik&title=Trapping+moving+targets+with+small+molecules&doi=10.1126%2Fscience.1169378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit218R"><div class="casContent"><span class="casTitleNuber">218</span><div class="casTitle"><span class="NLM_cas:atitle">Trapping Moving Targets with Small Molecules</span></div><div class="casAuthors">Lee, Gregory M.; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5924</span>),
    <span class="NLM_cas:pages">213-215</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review and discussion.  Structure-based drug design traditionally uses static protein models as inspirations for focusing on "active" site targets.  Allosteric regulation of biol. macromols., however, is affected by both conformational and dynamic properties of the protein or protein complex and can potentially lead to more avenues for therapeutic development.  We discuss the advantages of searching for mols. that conformationally trap a macromol. in its inactive state.  Although multiple methodologies exist to probe protein dynamics and ligand binding, our current discussion highlights the use of NMR spectroscopy in the drug discovery and design process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCoLaWvdf9x7Vg90H21EOLACvtfcHk0lifVeuC9ph9kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFalsro%253D&md5=69579d00b8d0e8dff1a8bfbf3c6632df</span></div><a href="/servlet/linkout?suffix=cit218&amp;dbid=16384&amp;doi=10.1126%2Fscience.1169378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1169378%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DG.%2BM.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DTrapping%2520moving%2520targets%2520with%2520small%2520molecules%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D213%26epage%3D215%26doi%3D10.1126%2Fscience.1169378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref219"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref219'); return false;" data-citation="" class="refNumLink">219</a></strong><div class="NLM_citation" id="cit219"><span><span class="NLM_contrib-group">Conn, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christopoulos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindsley, C. W.</span><span> </span><span class="NLM_article-title">Allosteric modulators of GPCRs: a novel approach for the treatment of CNS disorders</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span><span class="refDoi"> DOI: 10.1038/nrd2760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd2760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19116626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFCkurfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=41-54&author=P.+J.+Connauthor=A.+Christopoulosauthor=C.+W.+Lindsley&title=Allosteric+modulators+of+GPCRs%3A+a+novel+approach+for+the+treatment+of+CNS+disorders&doi=10.1038%2Fnrd2760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit219R"><div class="casContent"><span class="casTitleNuber">219</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulators of GPCRs: a novel approach for the treatment of CNS disorders</span></div><div class="casAuthors">Conn, P. Jeffrey; Christopoulos, Arthur; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-54</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite G-protein-coupled receptors (GPCRs) being among the most fruitful targets for marketed drugs, intense discovery efforts for several GPCR subtypes have failed to deliver selective drug candidates.  Historically, drug discovery programs for GPCR ligands have been dominated by efforts to develop agonists and antagonists that act at orthosteric sites for endogenous ligands.  However, in recent years, there have been tremendous advances in the discovery of novel ligands for GPCRs that act at allosteric sites to regulate receptor function.  These compds. provide high selectivity, novel modes of efficacy and may lead to novel therapeutic agents for the treatment of multiple psychiatric and neurol. human disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmn5h5XCcPKbVg90H21EOLACvtfcHk0lgcD3AgypH-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFCkurfF&md5=e109e94fe3e2c1fcfd480dc84c6609f7</span></div><a href="/servlet/linkout?suffix=cit219&amp;dbid=16384&amp;doi=10.1038%2Fnrd2760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2760%26sid%3Dliteratum%253Aachs%26aulast%3DConn%26aufirst%3DP.%2BJ.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DAllosteric%2520modulators%2520of%2520GPCRs%253A%2520a%2520novel%2520approach%2520for%2520the%2520treatment%2520of%2520CNS%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D41%26epage%3D54%26doi%3D10.1038%2Fnrd2760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref220"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref220'); return false;" data-citation="" class="refNumLink">220</a></strong><div class="NLM_citation" id="cit220"><span><span class="NLM_contrib-group">Chatzidaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Millar, N. S.</span><span> </span><span class="NLM_article-title">Allosteric modulation of nicotinic acetylcholine receptors</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span><span class="refDoi"> DOI: 10.1016/j.bcp.2015.07.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.bcp.2015.07.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26231943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1GrsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2015&pages=408-417&author=A.+Chatzidakiauthor=N.+S.+Millar&title=Allosteric+modulation+of+nicotinic+acetylcholine+receptors&doi=10.1016%2Fj.bcp.2015.07.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit220R"><div class="casContent"><span class="casTitleNuber">220</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of nicotinic acetylcholine receptors</span></div><div class="casAuthors">Chatzidaki, Anna; Millar, Neil S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-417</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nicotinic acetylcholine receptors (nAChRs) are receptors for the neurotransmitter acetylcholine and are members of the 'Cys-loop' family of pentameric ligand-gated ion channels (LGICs).  Acetylcholine binds in the receptor extracellular domain at the interface between two subunits and research has identified a large no. of nAChR-selective ligands, including agonists and competitive antagonists, that bind at the same site as acetylcholine (commonly referred to as the orthosteric binding site).  In addn., more recent research has identified ligands that are able to modulate nAChR function by binding to sites that are distinct from the binding site for acetylcholine, including sites located in the transmembrane domain.  These include pos. allosteric modulators (PAMs), neg. allosteric modulators (NAMs), silent allosteric modulators (SAMs) and compds. that are able to activate nAChRs via an allosteric binding site (allosteric agonists).  Our aim in this article is to review important aspects of the pharmacol. diversity of nAChR allosteric modulators and to describe recent evidence aimed at identifying binding sites for allosteric modulators on nAChRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomoO8Pu1gYwbVg90H21EOLACvtfcHk0lgcD3AgypH-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1GrsL3F&md5=3c321adc918921f2ac7db2a56fe20af8</span></div><a href="/servlet/linkout?suffix=cit220&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.07.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.07.028%26sid%3Dliteratum%253Aachs%26aulast%3DChatzidaki%26aufirst%3DA.%26aulast%3DMillar%26aufirst%3DN.%2BS.%26atitle%3DAllosteric%2520modulation%2520of%2520nicotinic%2520acetylcholine%2520receptors%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2015%26volume%3D97%26spage%3D408%26epage%3D417%26doi%3D10.1016%2Fj.bcp.2015.07.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref221"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref221'); return false;" data-citation="" class="refNumLink">221</a></strong><div class="NLM_citation" id="cit221"><span><span class="NLM_contrib-group">Desdouits, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blondel, A.</span><span> </span><span class="NLM_article-title">Principal component analysis reveals correlation of cavities evolution and functional motions in proteins</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2014.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmgm.2014.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25424655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=13-24&author=N.+Desdouitsauthor=M.+Nilgesauthor=A.+Blondel&title=Principal+component+analysis+reveals+correlation+of+cavities+evolution+and+functional+motions+in+proteins&doi=10.1016%2Fj.jmgm.2014.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit221R"><div class="casContent"><span class="casTitleNuber">221</span><div class="casTitle"><span class="NLM_cas:atitle">Principal Component Analysis reveals correlation of cavities evolution and functional motions in proteins</span></div><div class="casAuthors">Desdouits, Nathan; Nilges, Michael; Blondel, Arnaud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-24</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein conformation has been recognized as the key feature detg. biol. function, as it dets. the position of the essential groups specifically interacting with substrates.  Hence, the shape of the cavities or grooves at the protein surface appears to drive those functions.  However, only a few studies describe the geometrical evolution of protein cavities during mol. dynamics simulations (MD), usually with a crude representation.  To unveil the dynamics of cavity geometry evolution, we developed an approach combining cavity detection and Principal Component Anal. (PCA).  This approach was applied to four systems subjected to MD (lysozyme, sperm whale myoglobin, Dengue envelope protein and EF-CaM complex).  PCA on cavities allows us to perform efficient anal. and classification of the geometry diversity explored by a cavity.  Addnl., it reveals correlations between the evolutions of the cavities and structures, and can even suggest how to modify the protein conformation to induce a given cavity geometry.  It also helps to perform fast and consensual clustering of conformations according to cavity geometry.  Finally, using this approach, we show that both carbon monoxide (CO) location and transfer among the different xenon sites of myoglobin are correlated with few cavity evolution modes of high amplitude.  This correlation illustrates the link between ligand diffusion and the dynamic network of internal cavities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsuVyGLV0O7Vg90H21EOLACvtfcHk0lgcD3AgypH-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvE&md5=d7634d896c02f34c2e7b937e4d6d1607</span></div><a href="/servlet/linkout?suffix=cit221&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2014.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2014.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DDesdouits%26aufirst%3DN.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DBlondel%26aufirst%3DA.%26atitle%3DPrincipal%2520component%2520analysis%2520reveals%2520correlation%2520of%2520cavities%2520evolution%2520and%2520functional%2520motions%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D55%26spage%3D13%26epage%3D24%26doi%3D10.1016%2Fj.jmgm.2014.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref222"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref222'); return false;" data-citation="" class="refNumLink">222</a></strong><div class="NLM_citation" id="cit222"><span><span class="NLM_contrib-group">Kokh, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czodrowski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R. C.</span><span> </span><span class="NLM_article-title">TRAPP: a tool for analysis of transient binding pockets in proteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1235</span><span class="NLM_x">–</span> <span class="NLM_lpage">1252</span><span class="refDoi"> DOI: 10.1021/ci4000294</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4000294" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1eitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1235-1252&author=D.+B.+Kokhauthor=S.+Richterauthor=S.+Henrichauthor=P.+Czodrowskiauthor=F.+Rippmannauthor=R.+C.+Wade&title=TRAPP%3A+a+tool+for+analysis+of+transient+binding+pockets+in+proteins&doi=10.1021%2Fci4000294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit222R"><div class="casContent"><span class="casTitleNuber">222</span><div class="casTitle"><span class="NLM_cas:atitle">TRAPP: A tool for analysis of transient binding pockets in proteins</span></div><div class="casAuthors">Kokh, Daria B.; Richter, Stefan; Henrich, Stefan; Czodrowski, Paul; Rippmann, Friedrich; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1235-1252</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors present TRAPP (TRAnsient Pockets in Proteins), a new automated software platform for tracking, anal., and visualization of binding pocket variations along a protein motion trajectory or within an ensemble of protein structures that may encompass conformational changes ranging from local side chain fluctuations to global backbone motions.  TRAPP performs accurate grid-based calcns. of the shape and physicochem. characteristics of a binding pocket for each structure and detects the conserved and transient regions of the pocket in an ensemble of protein conformations.  It also provides tools for tracing the opening of a particular subpocket and residues that contribute to the binding site.  TRAPP thus enables an assessment of the drugability of a disease-related target protein taking its flexibility into account.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3HpnnZZfvd7Vg90H21EOLACvtfcHk0lgcD3AgypH-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1eitb8%253D&md5=ad45c4a383da31f88a4d0a6844f76391</span></div><a href="/servlet/linkout?suffix=cit222&amp;dbid=16384&amp;doi=10.1021%2Fci4000294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4000294%26sid%3Dliteratum%253Aachs%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DRichter%26aufirst%3DS.%26aulast%3DHenrich%26aufirst%3DS.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DTRAPP%253A%2520a%2520tool%2520for%2520analysis%2520of%2520transient%2520binding%2520pockets%2520in%2520proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1235%26epage%3D1252%26doi%3D10.1021%2Fci4000294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref223"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref223'); return false;" data-citation="" class="refNumLink">223</a></strong><div class="NLM_citation" id="cit223"><span><span class="NLM_contrib-group">Kappel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wereszczynski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clubb, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The binding mechanism, multiple binding modes, and allosteric regulation of staphylococcus aureus sortase a probed by molecular dynamics simulations</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1858</span><span class="NLM_x">–</span> <span class="NLM_lpage">1871</span><span class="refDoi"> DOI: 10.1002/pro.2168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fpro.2168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23023444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslelsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=1858-1871&author=K.+Kappelauthor=J.+Wereszczynskiauthor=R.+T.+Clubbauthor=J.+A.+McCammon&title=The+binding+mechanism%2C+multiple+binding+modes%2C+and+allosteric+regulation+of+staphylococcus+aureus+sortase+a+probed+by+molecular+dynamics+simulations&doi=10.1002%2Fpro.2168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit223R"><div class="casContent"><span class="casTitleNuber">223</span><div class="casTitle"><span class="NLM_cas:atitle">The binding mechanism, multiple binding modes, and allosteric regulation of Staphylococcus aureus Sortase A probed by molecular dynamics simulations</span></div><div class="casAuthors">Kappel, Kalli; Wereszczynski, Jeff; Clubb, Robert T.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1858-1871</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Sortase enzymes are vitally important for the virulence of gram-pos. bacteria as they play a key role in the attachment of surface proteins to the cell wall.  These enzymes recognize a specific sorting sequence in proteins destined to be displayed on the surface of the bacteria and catalyze the transpeptidation reaction that links it to a cell wall precursor mol.  Because of their role in establishing pathogenicity, and in light of the recent rise of antibiotic-resistant bacterial strains, sortase enzymes are novel drug targets.  Here, we present a study of the prototypical sortase protein Staphylococcus aureus Sortase A (SrtA).  Both conventional and accelerated mol. dynamics simulations of S. aureus SrtA in its apo state and when bound to an LPATG sorting signal (SS) were performed.  Results support a binding mechanism that may be characterized as conformational selection followed by induced fit.  Addnl., the SS was found to adopt multiple metastable states, thus resolving discrepancies between binding conformations in previously reported exptl. structures.  Finally, correlation anal. reveals that the SS actively affects allosteric pathways throughout the protein that connect the first and the second substrate binding sites, which are proposed to be located on opposing faces of the protein.  Overall, these calcns. shed new light on the role of dynamics in the binding mechanism and function of sortase enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhL1c8Gb4WsLVg90H21EOLACvtfcHk0lguENH0ArGsLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslelsr3F&md5=3c0785e3dfef30dda49be2798282e951</span></div><a href="/servlet/linkout?suffix=cit223&amp;dbid=16384&amp;doi=10.1002%2Fpro.2168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2168%26sid%3Dliteratum%253Aachs%26aulast%3DKappel%26aufirst%3DK.%26aulast%3DWereszczynski%26aufirst%3DJ.%26aulast%3DClubb%26aufirst%3DR.%2BT.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520binding%2520mechanism%252C%2520multiple%2520binding%2520modes%252C%2520and%2520allosteric%2520regulation%2520of%2520staphylococcus%2520aureus%2520sortase%2520a%2520probed%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProtein%2520Sci.%26date%3D2012%26volume%3D21%26spage%3D1858%26epage%3D1871%26doi%3D10.1002%2Fpro.2168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref224"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref224'); return false;" data-citation="" class="refNumLink">224</a></strong><div class="NLM_citation" id="cit224"><span><span class="NLM_contrib-group">Amadei, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linssen, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">Essential dynamics of proteins</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">425</span><span class="refDoi"> DOI: 10.1002/prot.340170408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.340170408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=8108382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2cXis1Ghsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1993&pages=412-425&author=A.+Amadeiauthor=A.+B.+Linssenauthor=H.+J.+Berendsen&title=Essential+dynamics+of+proteins&doi=10.1002%2Fprot.340170408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit224R"><div class="casContent"><span class="casTitleNuber">224</span><div class="casTitle"><span class="NLM_cas:atitle">Essential dynamics of proteins</span></div><div class="casAuthors">Amadei, Andrea; Linssen, Antonius B.M.; Berendsen, Herman J.C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">412-25</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">Anal. of extended mol. dynamics (MD) simulations of lysozyme in vacuo and in aq. soln. reveals that it is possible to sep. the configurational space into two subspaces: (1) an "essential" subspace contg. only a few degrees of freedom in which anharmonic motion occurs that comprises most of the positional fluctuations and (2) the remaining space in which the motion has a narrow Gaussian distribution and which can be considered as "phys. constrained.".  If overall translation and rotation are eliminated, the two spaces can be constructed by a simple linear transformation in Cartesian coordinate space, which remains valid over several hundred picoseconds.  The transformation follows from the covariance matrix of the positional deviations.  The essential degrees of freedom seem to describe motions which are relevant for the function of the protein, while the phys. constrained subspace merely describes irrelevant local fluctuations.  The near-constraint behavior of the latter subspace allows the sepn. of equations of motion and promises the possibility of investigating independently the essential space and performing dynamic simulations only in this reduced space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo69NnvqF5j17Vg90H21EOLACvtfcHk0lguENH0ArGsLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXis1Ghsbc%253D&md5=e99ea5db083d315c00c85c5fe81d87f7</span></div><a href="/servlet/linkout?suffix=cit224&amp;dbid=16384&amp;doi=10.1002%2Fprot.340170408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340170408%26sid%3Dliteratum%253Aachs%26aulast%3DAmadei%26aufirst%3DA.%26aulast%3DLinssen%26aufirst%3DA.%2BB.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DEssential%2520dynamics%2520of%2520proteins%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1993%26volume%3D17%26spage%3D412%26epage%3D425%26doi%3D10.1002%2Fprot.340170408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref225"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref225'); return false;" data-citation="" class="refNumLink">225</a></strong><div class="NLM_citation" id="cit225"><span><span class="NLM_contrib-group">Lange, O. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Generalized correlation for biomolecular dynamics</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">1053</span><span class="NLM_x">–</span> <span class="NLM_lpage">1061</span><span class="refDoi"> DOI: 10.1002/prot.20784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.20784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16355416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVWnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=1053-1061&author=O.+F.+Langeauthor=H.+Grubmuller&title=Generalized+correlation+for+biomolecular+dynamics&doi=10.1002%2Fprot.20784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit225R"><div class="casContent"><span class="casTitleNuber">225</span><div class="casTitle"><span class="NLM_cas:atitle">Generalized correlation for biomolecular dynamics</span></div><div class="casAuthors">Lange, Oliver F.; Grubmueller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1053-1061</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Correlated motions in biomols. are often essential for their function, e.g., allosteric signal transduction or mech./thermodn. energy transport.  Because correlated motions in biomols. remain difficult to access exptl., mol. dynamics (MD) simulations are particular useful for their anal.  The established method to quantify correlations from MD simulations via calcn. of the covariance matrix, however, is restricted to linear correlations and therefore misses part of the correlations in the at. fluctuations.  Herein, the authors propose a general statistical mechanics approach to detect and quantify any correlated motion from MD trajectories.  This generalized correlation measure is contrasted with correlations obtained from covariance matrixes for the B1 domain of protein G and T4 lysozyme.  The new method successfully quantifies correlations and provides a valuable global overview over the functionally relevant collective motions of lysozyme.  In particular, correlated motions of helix 1 together with the two main lobes of lysozyme are detected, which are not seen by the conventional covariance matrix.  Overall, the established method misses more than 50% of the correlation.  This failure is attributed to both, an interfering and unnecessary dependence on mutual orientations of the at. fluctuations and, to a lesser extent, attributed to nonlinear correlations.  The authors' generalized correlation measure overcomes these problems and, moreover, allows for an improved understanding of the conformational dynamics by sepg. linear and nonlinear contributions of the correlation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFG8QqiRvT97Vg90H21EOLACvtfcHk0lguENH0ArGsLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVWnsr4%253D&md5=c5f5cb619a4fca1649328371ea8fe9ec</span></div><a href="/servlet/linkout?suffix=cit225&amp;dbid=16384&amp;doi=10.1002%2Fprot.20784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20784%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DO.%2BF.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DGeneralized%2520correlation%2520for%2520biomolecular%2520dynamics%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D62%26spage%3D1053%26epage%3D1061%26doi%3D10.1002%2Fprot.20784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref226"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref226'); return false;" data-citation="" class="refNumLink">226</a></strong><div class="NLM_citation" id="cit226"><span><span class="NLM_contrib-group">Lange, O. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Full correlation analysis of conformational protein dynamics</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">1294</span><span class="NLM_x">–</span> <span class="NLM_lpage">1312</span><span class="refDoi"> DOI: 10.1002/prot.21618</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.21618" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2008&pages=1294-1312&author=O.+F.+Langeauthor=H.+Grubmuller&title=Full+correlation+analysis+of+conformational+protein+dynamics&doi=10.1002%2Fprot.21618"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit226&amp;dbid=16384&amp;doi=10.1002%2Fprot.21618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21618%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DO.%2BF.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DFull%2520correlation%2520analysis%2520of%2520conformational%2520protein%2520dynamics%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D70%26spage%3D1294%26epage%3D1312%26doi%3D10.1002%2Fprot.21618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref227"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref227'); return false;" data-citation="" class="refNumLink">227</a></strong><div class="NLM_citation" id="cit227"><span><span class="NLM_contrib-group">McClendon, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amirkhani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Quantifying correlations between allosteric sites in thermodynamic ensembles</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2486</span><span class="NLM_x">–</span> <span class="NLM_lpage">2502</span><span class="refDoi"> DOI: 10.1021/ct9001812</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct9001812" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpslahsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=2486-2502&author=C.+L.+McClendonauthor=G.+Friedlandauthor=D.+L.+Mobleyauthor=H.+Amirkhaniauthor=M.+P.+Jacobson&title=Quantifying+correlations+between+allosteric+sites+in+thermodynamic+ensembles&doi=10.1021%2Fct9001812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit227R"><div class="casContent"><span class="casTitleNuber">227</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Correlations Between Allosteric Sites in Thermodynamic Ensembles</span></div><div class="casAuthors">McClendon, Christopher L.; Friedland, Gregory; Mobley, David L.; Amirkhani, Homeira; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2486-2502</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery describes altered protein function at one site due to a perturbation at another site.  One mechanism of allostery involves correlated motions, which can occur even in the absence of substantial conformational change.  We present a novel method, "MutInf", to identify statistically significant correlated motions from equil. mol. dynamics simulations.  Our approach analyzes both backbone and side chain motions using internal coordinates to account for the gear-like twists that can take place even in the absence of the large conformational changes typical of traditional allosteric proteins.  We quantify correlated motions using a mutual information metric, which we extend to incorporate data from multiple short simulations and to filter out correlations that are not statistically significant.  Applying our approach to uncover mechanisms of cooperative small mol. binding in human interleukin-2, we identify clusters of correlated residues from 50 ns of mol. dynamics simulations.  Interestingly, two of the clusters with the strongest correlations highlight known cooperative small-mol. binding sites and show substantial correlations between these sites.  These cooperative binding sites on interleukin-2 are correlated not only through the hydrophobic core of the protein but also through a dynamic polar network of hydrogen bonding and electrostatic interactions.  Since this approach identifies correlated conformations in an unbiased, statistically robust manner, it should be a useful tool for finding novel or "orphan" allosteric sites in proteins of biol. and therapeutic importance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9xO0p5xauKrVg90H21EOLACvtfcHk0li3Gdm34Rp4IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpslahsrc%253D&md5=158a76d26856825d4f19c8515f40ffc4</span></div><a href="/servlet/linkout?suffix=cit227&amp;dbid=16384&amp;doi=10.1021%2Fct9001812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct9001812%26sid%3Dliteratum%253Aachs%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26aulast%3DFriedland%26aufirst%3DG.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DAmirkhani%26aufirst%3DH.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DQuantifying%2520correlations%2520between%2520allosteric%2520sites%2520in%2520thermodynamic%2520ensembles%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D2486%26epage%3D2502%26doi%3D10.1021%2Fct9001812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref228"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref228'); return false;" data-citation="" class="refNumLink">228</a></strong><div class="NLM_citation" id="cit228"><span><span class="NLM_contrib-group">Sethi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eargle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthey-Schulten, Z.</span><span> </span><span class="NLM_article-title">Dynamical networks in tRNA: protein complexes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6620</span><span class="NLM_x">–</span> <span class="NLM_lpage">6625</span><span class="refDoi"> DOI: 10.1073/pnas.0810961106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0810961106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19351898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsVOlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6620-6625&author=A.+Sethiauthor=J.+Eargleauthor=A.+A.+Blackauthor=Z.+Luthey-Schulten&title=Dynamical+networks+in+tRNA%3A+protein+complexes&doi=10.1073%2Fpnas.0810961106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit228R"><div class="casContent"><span class="casTitleNuber">228</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamical networks in tRNA:protein complexes</span></div><div class="casAuthors">Sethi, Anurag; Eargle, John; Black, Alexis A.; Luthey-Schulten, Zaida</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6620-6625</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Community network anal. derived from mol. dynamics simulations is used to identify and compare the signaling pathways in a bacterial glutamyl-tRNA synthetase (GluRS):tRNAGlu and an archaeal leucyl-tRNA synthetase (LeuRS):tRNALeu complex.  Although the 2 class I synthetases have remarkably different interactions with their cognate tRNAs, the allosteric networks for charging tRNA with the correct amino acid display considerable similarities.  A dynamic contact map defines the edges connecting nodes (amino acids and nucleotides) in the phys. network whose overall topol. is presented as a network of communities, local substructures that are highly intraconnected, but loosely interconnected.  Whereas nodes within a single community can communicate through many alternate pathways, the communication between monomers in different communities has to take place through a smaller no. of crit. edges or interactions.  Consistent with this anal., there are a large no. of suboptimal paths that can be used for communication between the identity elements on the tRNAs and the catalytic site in the aaRS:tRNA complexes.  Residues and nucleotides in the majority of pathways for intercommunity signal transmission are evolutionarily conserved and are predicted to be important for allosteric signaling.  The same monomers are also found in a majority of the suboptimal paths.  Modifying these residues or nucleotides has a large effect on the communication pathways in the protein:RNA complex consistent with kinetic data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy2zkXI--Lx7Vg90H21EOLACvtfcHk0li3Gdm34Rp4IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsVOlu7o%253D&md5=5e3cfd80bf85ffe46c9c84960d176713</span></div><a href="/servlet/linkout?suffix=cit228&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0810961106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0810961106%26sid%3Dliteratum%253Aachs%26aulast%3DSethi%26aufirst%3DA.%26aulast%3DEargle%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DA.%2BA.%26aulast%3DLuthey-Schulten%26aufirst%3DZ.%26atitle%3DDynamical%2520networks%2520in%2520tRNA%253A%2520protein%2520complexes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6620%26epage%3D6625%26doi%3D10.1073%2Fpnas.0810961106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref229"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref229'); return false;" data-citation="" class="refNumLink">229</a></strong><div class="NLM_citation" id="cit229"><span><span class="NLM_contrib-group">Newman, M. E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girvan, M.</span><span> </span><span class="NLM_article-title">Finding and evaluating community structure in networks</span> <span class="citation_source-journal">Phys. Rev. E</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">026113</span><span class="refDoi"> DOI: 10.1103/PhysRevE.69.026113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref229/cit229&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevE.69.026113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2004&pages=026113&author=M.+E.+J.+Newmanauthor=M.+Girvan&title=Finding+and+evaluating+community+structure+in+networks&doi=10.1103%2FPhysRevE.69.026113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit229&amp;dbid=16384&amp;doi=10.1103%2FPhysRevE.69.026113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevE.69.026113%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DM.%2BE.%2BJ.%26aulast%3DGirvan%26aufirst%3DM.%26atitle%3DFinding%2520and%2520evaluating%2520community%2520structure%2520in%2520networks%26jtitle%3DPhys.%2520Rev.%2520E%26date%3D2004%26volume%3D69%26spage%3D026113%26doi%3D10.1103%2FPhysRevE.69.026113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref230"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref230'); return false;" data-citation="" class="refNumLink">230</a></strong><div class="NLM_citation" id="cit230"><span><span class="NLM_contrib-group">Rivalta, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sultan, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loria, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batista, V. S.</span><span> </span><span class="NLM_article-title">Allosteric pathways in imidazole glycerol phosphate synthase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">E1428</span><span class="NLM_x">–</span> <span class="NLM_lpage">E1436</span><span class="refDoi"> DOI: 10.1073/pnas.1120536109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1120536109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22586084" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=E1428-E1436&author=I.+Rivaltaauthor=M.+M.+Sultanauthor=N.+S.+Leeauthor=G.+A.+Manleyauthor=J.+P.+Loriaauthor=V.+S.+Batista&title=Allosteric+pathways+in+imidazole+glycerol+phosphate+synthase&doi=10.1073%2Fpnas.1120536109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit230&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120536109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120536109%26sid%3Dliteratum%253Aachs%26aulast%3DRivalta%26aufirst%3DI.%26aulast%3DSultan%26aufirst%3DM.%2BM.%26aulast%3DLee%26aufirst%3DN.%2BS.%26aulast%3DManley%26aufirst%3DG.%2BA.%26aulast%3DLoria%26aufirst%3DJ.%2BP.%26aulast%3DBatista%26aufirst%3DV.%2BS.%26atitle%3DAllosteric%2520pathways%2520in%2520imidazole%2520glycerol%2520phosphate%2520synthase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3DE1428%26epage%3DE1436%26doi%3D10.1073%2Fpnas.1120536109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref231"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref231'); return false;" data-citation="" class="refNumLink">231</a></strong><div class="NLM_citation" id="cit231"><span><span class="NLM_contrib-group">Morra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genoni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span> </span><span class="NLM_article-title">Mechanisms of differential allosteric modulation in homologous proteins: insights from the analysis of internal dynamics and energetics of pdz domains</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">5677</span><span class="NLM_x">–</span> <span class="NLM_lpage">5689</span><span class="refDoi"> DOI: 10.1021/ct500326g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500326g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=5677-5689&author=G.+Morraauthor=A.+Genoniauthor=G.+Colombo&title=Mechanisms+of+differential+allosteric+modulation+in+homologous+proteins%3A+insights+from+the+analysis+of+internal+dynamics+and+energetics+of+pdz+domains&doi=10.1021%2Fct500326g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit231R"><div class="casContent"><span class="casTitleNuber">231</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Differential Allosteric Modulation in Homologous Proteins: Insights from the Analysis of Internal Dynamics and Energetics of PDZ Domains</span></div><div class="casAuthors">Morra, Giulia; Genoni, Alessandro; Colombo, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5677-5689</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is a general phenomenon in proteins whereby a perturbation at one site reverberates into a functional change at another one, through modulation of its conformational dynamics.  Herein, we address the problem of how the mol. signal encoded by a ligand is differentially transmitted through the structures of two homologous PDZ proteins: PDZ2, which responds to binding with structural and dynamical changes in regions distal from the ligand site, and PDZ3, which is characterized by less-intense dynamical variations.  We use novel methods of anal. of MD simulations in the unbound and bound states to investigate the determinants of the differential allosteric behavior of the two proteins.  The anal. of the correlations between the redistribution of stabilization energy and local fluctuation patterns highlights the nucleus of residues responsible for the stabilization of the 3D fold, the stability core, as the substructure that defines the difference in the allosteric response: in PDZ2, it undergoes a consistent dynamic and energetic reorganization, whereas in PDZ3, it remains largely unperturbed.  Specifically, we observe for PDZ2 a significant anticorrelation between the motions of distal loops and residues of the stability core and differences in the correlation patterns between the bound and unbound states.  Such variation is not obsd. in PDZ3, indicating that its energetics and internal dynamics are less affected by the presence/absence of the ligand.  Finally, we propose a model with a direct link between the modulation of the structural, energetic and dynamic properties of a protein, and its allosteric response to a perturbation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE0MzwLruY77Vg90H21EOLACvtfcHk0liFjyJfhjDbgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsrnF&md5=8908c2566d34fbcbb5863714f4d07313</span></div><a href="/servlet/linkout?suffix=cit231&amp;dbid=16384&amp;doi=10.1021%2Fct500326g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500326g%26sid%3Dliteratum%253Aachs%26aulast%3DMorra%26aufirst%3DG.%26aulast%3DGenoni%26aufirst%3DA.%26aulast%3DColombo%26aufirst%3DG.%26atitle%3DMechanisms%2520of%2520differential%2520allosteric%2520modulation%2520in%2520homologous%2520proteins%253A%2520insights%2520from%2520the%2520analysis%2520of%2520internal%2520dynamics%2520and%2520energetics%2520of%2520pdz%2520domains%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D5677%26epage%3D5689%26doi%3D10.1021%2Fct500326g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref232"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref232'); return false;" data-citation="" class="refNumLink">232</a></strong><div class="NLM_citation" id="cit232"><span><span class="NLM_contrib-group">Morra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neves, M. A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plescia, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsustsumi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neckers, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verkhivker, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altieri, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span> </span><span class="NLM_article-title">Dynamics-based discovery of allosteric inhibitors: selection of new ligands for the c-terminal domain of hsp90</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2978</span><span class="NLM_x">–</span> <span class="NLM_lpage">2989</span><span class="refDoi"> DOI: 10.1021/ct100334n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100334n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVyrsbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2978-2989&author=G.+Morraauthor=M.+A.+C.+Nevesauthor=C.+J.+Plesciaauthor=S.+Tsustsumiauthor=L.+Neckersauthor=G.+Verkhivkerauthor=D.+C.+Altieriauthor=G.+Colombo&title=Dynamics-based+discovery+of+allosteric+inhibitors%3A+selection+of+new+ligands+for+the+c-terminal+domain+of+hsp90&doi=10.1021%2Fct100334n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit232R"><div class="casContent"><span class="casTitleNuber">232</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics-Based Discovery of Allosteric Inhibitors: Selection of New Ligands for the C-terminal Domain of Hsp90</span></div><div class="casAuthors">Morra, Giulia; Neves, Marco A. C.; Plescia, Christopher J.; Tsustsumi, Shinji; Neckers, Len; Verkhivker, Gennady; Altieri, Dario C.; Colombo, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2978-2989</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The study of allosteric functional modulation in dynamic proteins is attracting increasing attention.  In particular, the discovery of new allosteric sites may generate novel opportunities and strategies for drug development, overcoming the limits of classical active-site oriented drug design.  In this paper, we report on the results of a novel, ab initio, fully computational approach for the discovery of allosteric inhibitors based on the phys. characterization of signal propagation mechanisms in proteins and apply it to the important mol. chaperone Hsp90.  We first characterize the allosteric "hot spots" involved in interdomain communication pathways from the nucleotide-binding site in the N-domain to the distal C-domain.  On this basis, we develop dynamic pharmacophore models to screen drug libraries in the search for small mols. with the functional and conformational properties necessary to bind these "hot spot" allosteric sites.  Exptl. tests show that the selected mols. bind the Hsp90 C-domain, exhibit antiproliferative activity in different tumor cell lines, while not affecting proliferation of normal human cells, destabilize Hsp90 client proteins, and disrupt assocn. with several cochaperones known to bind the N- and M-domains of Hsp90.  These results prove that the hits alter Hsp90 function by affecting its conformational dynamics and recognition properties through an allosteric mechanism.  These findings provide us with new insights on the discovery and development of new allosteric inhibitors that are active on important cellular pathways through computational biol.  Though based on the specific case of Hsp90, our approach is general and can readily be extended to other target proteins and pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtIOnRuY95mrVg90H21EOLACvtfcHk0liFjyJfhjDbgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVyrsbfE&md5=2eef3fded78c9607e9e707e091a6716b</span></div><a href="/servlet/linkout?suffix=cit232&amp;dbid=16384&amp;doi=10.1021%2Fct100334n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100334n%26sid%3Dliteratum%253Aachs%26aulast%3DMorra%26aufirst%3DG.%26aulast%3DNeves%26aufirst%3DM.%2BA.%2BC.%26aulast%3DPlescia%26aufirst%3DC.%2BJ.%26aulast%3DTsustsumi%26aufirst%3DS.%26aulast%3DNeckers%26aufirst%3DL.%26aulast%3DVerkhivker%26aufirst%3DG.%26aulast%3DAltieri%26aufirst%3DD.%2BC.%26aulast%3DColombo%26aufirst%3DG.%26atitle%3DDynamics-based%2520discovery%2520of%2520allosteric%2520inhibitors%253A%2520selection%2520of%2520new%2520ligands%2520for%2520the%2520c-terminal%2520domain%2520of%2520hsp90%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2978%26epage%3D2989%26doi%3D10.1021%2Fct100334n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref233"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref233'); return false;" data-citation="" class="refNumLink">233</a></strong><div class="NLM_citation" id="cit233"><span><span class="NLM_contrib-group">de Beer, S. B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">The role of water molecules in computational drug design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.2174/156802610790232288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802610790232288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19929830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=55-66&author=S.+B.+A.+de+Beerauthor=N.+P.+E.+Vermeulenauthor=C.+Oostenbrink&title=The+role+of+water+molecules+in+computational+drug+design&doi=10.2174%2F156802610790232288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit233R"><div class="casContent"><span class="casTitleNuber">233</span><div class="casTitle"><span class="NLM_cas:atitle">The role of water molecules in computational drug design</span></div><div class="casAuthors">de Beer, Stephanie B. A.; Vermeulen, Nico P. E.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-66</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Although water mols. are small and only consist of two different atom types, they play various roles in cellular systems.  This review discusses their influence on the binding process between biomacromol. targets and small mol. ligands and how this influence can be modeled in computational drug design approaches.  Both the structure and the thermodn. of active site waters will be discussed as these influence the binding process significantly.  Structurally conserved waters cannot always be detd. exptl. and if obsd., it is not clear if they will be replaced upon ligand binding, even if sufficient space is available.  Methods to predict the presence of water in protein-ligand complexes will be reviewed.  Subsequently, we will discuss methods to include water in computational drug research.  Either as an addnl. factor in automated docking expts., or explicitly in detailed mol. dynamics simulations, the effect of water on the quality of the simulations is significant, but not easily predicted.  The most detailed calcns. involve ests. of the free energy contribution of water mols. to protein-ligand complexes.  These calcns. are computationally demanding, but give insight in the versatility and importance of water in ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhA2h-JAqzhbVg90H21EOLACvtfcHk0liFjyJfhjDbgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D&md5=5767b8a3f82a33845fae3941a1084067</span></div><a href="/servlet/linkout?suffix=cit233&amp;dbid=16384&amp;doi=10.2174%2F156802610790232288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232288%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBeer%26aufirst%3DS.%2BB.%2BA.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%2BE.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DThe%2520role%2520of%2520water%2520molecules%2520in%2520computational%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D55%26epage%3D66%26doi%3D10.2174%2F156802610790232288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref234"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref234'); return false;" data-citation="" class="refNumLink">234</a></strong><div class="NLM_citation" id="cit234"><span><span class="NLM_contrib-group">Athri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, W. D.</span><span> </span><span class="NLM_article-title">Molecular dynamics of water-mediated interactions of a linear benzimidazole-biphenyl diamidine with the dna minor groove</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">7618</span><span class="NLM_x">–</span> <span class="NLM_lpage">7625</span><span class="refDoi"> DOI: 10.1021/ja809249h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja809249h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=7618-7625&author=P.+Athriauthor=W.+D.+Wilson&title=Molecular+dynamics+of+water-mediated+interactions+of+a+linear+benzimidazole-biphenyl+diamidine+with+the+dna+minor+groove&doi=10.1021%2Fja809249h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit234&amp;dbid=16384&amp;doi=10.1021%2Fja809249h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja809249h%26sid%3Dliteratum%253Aachs%26aulast%3DAthri%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DMolecular%2520dynamics%2520of%2520water-mediated%2520interactions%2520of%2520a%2520linear%2520benzimidazole-biphenyl%2520diamidine%2520with%2520the%2520dna%2520minor%2520groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D7618%26epage%3D7625%26doi%3D10.1021%2Fja809249h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref235"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref235'); return false;" data-citation="" class="refNumLink">235</a></strong><div class="NLM_citation" id="cit235"><span><span class="NLM_contrib-group">Biela, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Water Makes the Difference: Rearrangement of Water Solvation Layer Triggers Non-additivity of functional group contributions in protein-ligand binding</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1423</span><span class="NLM_x">–</span> <span class="NLM_lpage">1434</span><span class="refDoi"> DOI: 10.1002/cmdc.201200206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fcmdc.201200206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22733601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVajsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1423-1434&author=A.+Bielaauthor=M.+Betzauthor=A.+Heineauthor=G.+Klebe&title=Water+Makes+the+Difference%3A+Rearrangement+of+Water+Solvation+Layer+Triggers+Non-additivity+of+functional+group+contributions+in+protein-ligand+binding&doi=10.1002%2Fcmdc.201200206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit235R"><div class="casContent"><span class="casTitleNuber">235</span><div class="casTitle"><span class="NLM_cas:atitle">Water Makes the Difference: Rearrangement of Water Solvation Layer Triggers Non-additivity of Functional Group Contributions in Protein-Ligand Binding</span></div><div class="casAuthors">Biela, Adam; Betz, Michael; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1423-1434, S1423/1-S1423/2</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The binding of four congeneric peptide-like thermolysin inhibitors has been studied by high-resoln. crystal structure anal. and isothermal titrn. calorimetry.  The ligands differ only by a terminal carboxylate and/or Me group.  A surprising non-additivity of functional group contributions for the carboxylate and/or Me groups is detected.  Adding the Me first and then the carboxylate group results in a small Gibbs free energy increase and minor enthalpy/entropy partitioning for the first modification, whereas the second involves a strong affinity increase combined with large enthalpy/entropy changes.  However, first adding the carboxylate and then the Me group yields reverse effects: the acidic group attachment now causes minor effects, whereas the added Me group provokes large changes.  As all crystal structures show virtually identical binding modes, affinity changes are related to rearrangements of the first solvation layer next to the S2' pocket.  About 20-25 water mols. are visible next to the studied complexes.  The added COO- groups perturb the local water network in both carboxylated complexes, and the attached Me groups provide favorable interaction sites for water mols.  Apart from one example, a contiguously connected water network between protein and ligand functional groups is obsd. in all complexes.  In the complex with the carboxylated ligand, which still lacks the terminal Me group, the water network is unfavorably ruptured.  This results in a surprising thermodn. signature showing only a minor affinity increase upon COO- group attachment.  Because the further added Me group provides a favorable interaction site for water, the network can be reestablished, and a strong affinity increase with a large enthalpy/entropy signature is then detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgp9MxUfIJirVg90H21EOLACvtfcHk0lhwk_4jEQ1eyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVajsrc%253D&md5=1a6eb12403cdf82cae631384230f3d29</span></div><a href="/servlet/linkout?suffix=cit235&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200206%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DWater%2520Makes%2520the%2520Difference%253A%2520Rearrangement%2520of%2520Water%2520Solvation%2520Layer%2520Triggers%2520Non-additivity%2520of%2520functional%2520group%2520contributions%2520in%2520protein-ligand%2520binding%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D1423%26epage%3D1434%26doi%3D10.1002%2Fcmdc.201200206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref236"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref236'); return false;" data-citation="" class="refNumLink">236</a></strong><div class="NLM_citation" id="cit236"><span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Prediction of the water content in protein binding sites</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">13337</span><span class="NLM_x">–</span> <span class="NLM_lpage">13346</span><span class="refDoi"> DOI: 10.1021/jp9047456</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9047456" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref236/cit236&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=13337-13346&author=J.+Michelauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Prediction+of+the+water+content+in+protein+binding+sites&doi=10.1021%2Fjp9047456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit236R"><div class="casContent"><span class="casTitleNuber">236</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of the Water Content in Protein Binding Sites</span></div><div class="casAuthors">Michel, Julien; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13337-13346</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient mol. simulation methodol. has been developed to det. the positioning of water mols. in the binding site of a protein or protein-ligand complex.  Occupancies and abs. binding free energies of water mols. are computed using a statistical thermodn. approach.  The methodol., referred to as Just Add Water Mols. (JAWS), features "θ-water" mols. that can appear and disappear on a binding-site grid.  Key approxns. render the technique far more efficient than conventional free energy simulations.  Testing of JAWS on five diverse examples (neuraminidase, scytalone dehydratase, major urinary protein 1, β-lactoglobulin, and COX-2) demonstrates its accuracy in locating hydration sites in comparison to results from high-resoln. crystal structures.  Possible applications include aid in refinement of protein crystal structures, drug lead optimization, setup of docking calcns., and simulations of protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6L_AGV7e7LVg90H21EOLACvtfcHk0lhwk_4jEQ1eyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I&md5=bc36a74922d2b1ce1d15e53b9e144afe</span></div><a href="/servlet/linkout?suffix=cit236&amp;dbid=16384&amp;doi=10.1021%2Fjp9047456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9047456%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DPrediction%2520of%2520the%2520water%2520content%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D13337%26epage%3D13346%26doi%3D10.1021%2Fjp9047456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref237"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref237'); return false;" data-citation="" class="refNumLink">237</a></strong><div class="NLM_citation" id="cit237"><span><span class="NLM_contrib-group">Henchman, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Extracting hydration sites around proteins from explicit water simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">869</span><span class="refDoi"> DOI: 10.1002/jcc.10074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.10074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11984847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVyksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=861-869&author=R.+H.+Henchmanauthor=J.+A.+McCammon&title=Extracting+hydration+sites+around+proteins+from+explicit+water+simulations&doi=10.1002%2Fjcc.10074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit237R"><div class="casContent"><span class="casTitleNuber">237</span><div class="casTitle"><span class="NLM_cas:atitle">Extracting hydration sites around proteins from explicit water simulations</span></div><div class="casAuthors">Henchman, Richard H.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">861-869</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Two new methods are assessed for detg. the location of hydration sites around proteins from computer simulation.  Current methods ext. hydration sites from peaks in the water d. constructed in the protein frame.  However, the dynamic nature of the water mols., the nearby protein residues, and the protein ref. frame as a whole tend to smear out the water d., making it more difficult to resolve sites.  Two techniques are introduced to better resolve the water d.  The first is to construct the water d. from the time-averaged position of each water mol. in the protein frame while the water remains within a given distance of this averaged position.  The second technique is to construct the water d. from the time-averaged position of each water in the ref. frame only of the nearby residues.  Criteria for detg. hydration sites from the water d. are examd.  Both techniques are found to significantly improve the detail in the water d. and the no. of hydration sites detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQkWNcVsa6drVg90H21EOLACvtfcHk0lhwk_4jEQ1eyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksVyksL4%253D&md5=96ab8a8e32f13def3b9668486d5ad562</span></div><a href="/servlet/linkout?suffix=cit237&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10074%26sid%3Dliteratum%253Aachs%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DExtracting%2520hydration%2520sites%2520around%2520proteins%2520from%2520explicit%2520water%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D861%26epage%3D869%26doi%3D10.1002%2Fjcc.10074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref238"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref238'); return false;" data-citation="" class="refNumLink">238</a></strong><div class="NLM_citation" id="cit238"><span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradley, P.</span><span> </span><span class="NLM_article-title">Probing the role of interfacial waters in protein-DNA recognition using a hybrid implicit/explicit solvation model</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">1318</span><span class="NLM_x">–</span> <span class="NLM_lpage">1329</span><span class="refDoi"> DOI: 10.1002/prot.24272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.24272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23444044" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=1318-1329&author=S.+Liauthor=P.+Bradley&title=Probing+the+role+of+interfacial+waters+in+protein-DNA+recognition+using+a+hybrid+implicit%2Fexplicit+solvation+model&doi=10.1002%2Fprot.24272"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit238&amp;dbid=16384&amp;doi=10.1002%2Fprot.24272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24272%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DBradley%26aufirst%3DP.%26atitle%3DProbing%2520the%2520role%2520of%2520interfacial%2520waters%2520in%2520protein-DNA%2520recognition%2520using%2520a%2520hybrid%2520implicit%252Fexplicit%2520solvation%2520model%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2013%26volume%3D81%26spage%3D1318%26epage%3D1329%26doi%3D10.1002%2Fprot.24272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref239"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref239'); return false;" data-citation="" class="refNumLink">239</a></strong><div class="NLM_citation" id="cit239"><span><span class="NLM_contrib-group">Ladbury, J. E.</span><span> </span><span class="NLM_article-title">Just add water! The effect of water on the specificity of protein-ligand binding sites and its potential application to drug design</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">973</span><span class="NLM_x">–</span> <span class="NLM_lpage">980</span><span class="refDoi"> DOI: 10.1016/S1074-5521(96)90164-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1074-5521%2896%2990164-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=9000013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=973-980&author=J.+E.+Ladbury&title=Just+add+water%21+The+effect+of+water+on+the+specificity+of+protein-ligand+binding+sites+and+its+potential+application+to+drug+design&doi=10.1016%2FS1074-5521%2896%2990164-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit239R"><div class="casContent"><span class="casTitleNuber">239</span><div class="casTitle"><span class="NLM_cas:atitle">Just add water! The effect of water on the specificity of protein-ligand binding sites and its potential application to drug design</span></div><div class="casAuthors">Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">973-980</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">A review with 39 refs.  Recent data have highlighted the enigmatic role that water plays in biomol. complexes.  Water at the interface of a complex can increase the promiscuity of an interaction, yet it can also provide increased specificity and affinity.  The ability to engineer water-binding sites into the interface between a drug and its target might prove useful in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJqyhDhRviwbVg90H21EOLACvtfcHk0ljvphpyPorZxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D&md5=8eb3242b84897de4bb61e20ec6bc8d54</span></div><a href="/servlet/linkout?suffix=cit239&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2896%2990164-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252896%252990164-7%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DJust%2520add%2520water%2521%2520The%2520effect%2520of%2520water%2520on%2520the%2520specificity%2520of%2520protein-ligand%2520binding%2520sites%2520and%2520its%2520potential%2520application%2520to%2520drug%2520design%26jtitle%3DChem.%2520Biol.%26date%3D1996%26volume%3D3%26spage%3D973%26epage%3D980%26doi%3D10.1016%2FS1074-5521%2896%2990164-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref240"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref240'); return false;" data-citation="" class="refNumLink">240</a></strong><div class="NLM_citation" id="cit240"><span><span class="NLM_contrib-group">Huang, D. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossini, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Structured water molecules in the binding site of bromodomains can be displaced by cosolvent</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">573</span><span class="NLM_x">–</span> <span class="NLM_lpage">579</span><span class="refDoi"> DOI: 10.1002/cmdc.201300156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fcmdc.201300156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23804246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVSht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=573-579&author=D.+Z.+Huangauthor=E.+Rossiniauthor=S.+Steinerauthor=A.+Caflisch&title=Structured+water+molecules+in+the+binding+site+of+bromodomains+can+be+displaced+by+cosolvent&doi=10.1002%2Fcmdc.201300156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit240R"><div class="casContent"><span class="casTitleNuber">240</span><div class="casTitle"><span class="NLM_cas:atitle">Structured Water Molecules in the Binding Site of Bromodomains Can Be Displaced by Cosolvent</span></div><div class="casAuthors">Huang, Danzhi; Rossini, Emanuele; Steiner, Sandra; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">573-579</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomains are α-helical bundles of approx. 110 residues that recognize acetylated lysine side chains mainly on histone tails.  Bromodomains are known to play an important role in cancer and inflammation, and as such, significant efforts are being made to identify small-mol. inhibitors of these epigenetic reader proteins.  Here, explicit solvent mol. dynamics (MD) simulations of two bromodomains (BAZ2B and CREBBP) are used to analyze the water mols. that seem to be conserved at the bottom of the acetyl-lysine binding site in most crystal structures of bromodomains.  The MD runs suggest that the occupancy of the structured water mols. is influenced by conformational transitions of the loop that connects helixes Z and A.  Addnl. simulations in the presence of 50 mols. of cosolvent (i.e., 440 mM of dimethylsulfoxide, methanol, or ethanol) indicate that some of the structured water mols. can be displaced transiently.  The residence time in the acetyl-lysine binding site is calcd. to be about 1 ns, 2-5 ns, and 10-30 ns for methanol, ethanol, and dimethylsulfoxide, resp., while the affinity of the three cosolvents for BAZ2B and CREBBP is in the range of 50-500 mM.  The results described have implications for ligand design, suggesting that only structured water mols. that do not exchange with cosolvent should be maintained in crystal structures used for docking campaigns, and that hydroxy substituents should be incorporated in the ligand so as to map the structured water mols. replaced by (m)ethanol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoapdiGvm2PUrVg90H21EOLACvtfcHk0ljvphpyPorZxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVSht7fF&md5=6f742480234fc072a61a4a051dfec948</span></div><a href="/servlet/linkout?suffix=cit240&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300156%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%2BZ.%26aulast%3DRossini%26aufirst%3DE.%26aulast%3DSteiner%26aufirst%3DS.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DStructured%2520water%2520molecules%2520in%2520the%2520binding%2520site%2520of%2520bromodomains%2520can%2520be%2520displaced%2520by%2520cosolvent%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D573%26epage%3D579%26doi%3D10.1002%2Fcmdc.201300156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref241"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref241'); return false;" data-citation="" class="refNumLink">241</a></strong><div class="NLM_citation" id="cit241"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span> </span><span class="NLM_article-title">Inhomogeneous fluid approach to solvation thermodynamics. 1. Theory</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3531</span><span class="NLM_x">–</span> <span class="NLM_lpage">3541</span><span class="refDoi"> DOI: 10.1021/jp9723574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9723574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3531-3541&author=T.+Lazaridis&title=Inhomogeneous+fluid+approach+to+solvation+thermodynamics.+1.+Theory&doi=10.1021%2Fjp9723574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit241R"><div class="casContent"><span class="casTitleNuber">241</span><div class="casTitle"><span class="NLM_cas:atitle">Inhomogeneous Fluid Approach to Solvation Thermodynamics. 1. Theory</span></div><div class="casAuthors">Lazaridis, Themis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3531-3541</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The solvation energy and entropy at infinite diln. consist of a solute-solvent term and a solvent reorganization term representing the contributions of changes in solvent structure upon solute insertion.  In the std., homogeneous treatment of solns., changes in solvent structure are expressed through derivs. of the homogeneous pair correlation function, which are very difficult to obtain by simulation.  Tractable expressions for the solvation energy and entropy are here derived by viewing the soln. as an inhomogeneous system with the solute fixed at a certain point.  The solvent reorganization terms in the inhomogeneous view contain two contributions: the local, "correlation" contributions, which are due to correlations between the solute and the solvent and dominate at high densities, and the nonlocal, "liberation" contributions, which are due to the effective diln. of the solvent caused by the thermal motion of the solute and dominate at low densities.  The liberation contributions are independent of the nature or size of the solute and depend only on the properties of the solvent.  For common liq. solvents the nonlocal terms are negligible and the solvation properties arise almost entirely from effects localized around the solute.  The new expressions are tested by calcns. of the solvent reorganization energy and entropy in ideal hard-sphere and Lennard-Jones mixts. (solute identical to solvent).  The solvent reorganization energy and entropy make distinct and significant contributions to the solvation free energy.  The theory can be applied to truly inhomogeneous systems as well as small solute solvation, thus providing a connection between interfacial phenomena and microscopic solvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxEN_nWI8WfbVg90H21EOLACvtfcHk0ljvphpyPorZxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D&md5=99755c87ffc33abd5e7df89c9996a665</span></div><a href="/servlet/linkout?suffix=cit241&amp;dbid=16384&amp;doi=10.1021%2Fjp9723574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9723574%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26atitle%3DInhomogeneous%2520fluid%2520approach%2520to%2520solvation%2520thermodynamics.%25201.%2520Theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3531%26epage%3D3541%26doi%3D10.1021%2Fjp9723574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref242"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref242'); return false;" data-citation="" class="refNumLink">242</a></strong><div class="NLM_citation" id="cit242"><span><span class="NLM_contrib-group">Breiten, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockett, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Sayah, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heroux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesides, G. M.</span><span> </span><span class="NLM_article-title">Water Networks contribute to enthalpy/entropy compensation in protein-ligand binding</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">15579</span><span class="NLM_x">–</span> <span class="NLM_lpage">15584</span><span class="refDoi"> DOI: 10.1021/ja4075776</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4075776" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref242/cit242&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVGjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=15579-15584&author=B.+Breitenauthor=M.+R.+Lockettauthor=W.+Shermanauthor=S.+Fujitaauthor=M.+Al-Sayahauthor=H.+Langeauthor=C.+M.+Bowersauthor=A.+Herouxauthor=G.+Krilovauthor=G.+M.+Whitesides&title=Water+Networks+contribute+to+enthalpy%2Fentropy+compensation+in+protein-ligand+binding&doi=10.1021%2Fja4075776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit242R"><div class="casContent"><span class="casTitleNuber">242</span><div class="casTitle"><span class="NLM_cas:atitle">Water Networks Contribute to Enthalpy/Entropy Compensation in Protein-Ligand Binding</span></div><div class="casAuthors">Breiten, Benjamin; Lockett, Matthew R.; Sherman, Woody; Fujita, Shuji; Al-Sayah, Mohammad; Lange, Heiko; Bowers, Carleen M.; Heroux, Annie; Krilov, Goran; Whitesides, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">15579-15584</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mechanism (or mechanisms) of enthalpy-entropy (H/S) compensation in protein-ligand binding remains controversial, and there are still no predictive models (theor. or exptl.) in which hypotheses of ligand binding can be readily tested.  Here we describe a particularly well-defined system of protein and ligands-human carbonic anhydrase (HCA) and a series of benzothiazole sulfonamide ligands with different patterns of fluorination-that we use to define enthalpy/entropy (H/S) compensation in this system thermodynamically and structurally.  The binding affinities of these ligands (with the exception of one ligand, in which the deviation is understood) to HCA are, despite differences in fluorination pattern, indistinguishable; they nonetheless reflect significant and compensating changes in enthalpy and entropy of binding.  Anal. reveals that differences in the structure and thermodn. properties of the waters surrounding the bound ligands are an important contributor to the obsd. H/S compensation.  These results support the hypothesis that the mols. of water filling the active site of a protein, and surrounding the ligand, are as important as the contact interactions between the protein and the ligand for biomol. recognition, and in detg. the thermodn. of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2cIrBILFS8bVg90H21EOLACvtfcHk0lgYp__q4aA9qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVGjtLbM&md5=8326ee85b1b188d1588a8558870816d5</span></div><a href="/servlet/linkout?suffix=cit242&amp;dbid=16384&amp;doi=10.1021%2Fja4075776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4075776%26sid%3Dliteratum%253Aachs%26aulast%3DBreiten%26aufirst%3DB.%26aulast%3DLockett%26aufirst%3DM.%2BR.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DFujita%26aufirst%3DS.%26aulast%3DAl-Sayah%26aufirst%3DM.%26aulast%3DLange%26aufirst%3DH.%26aulast%3DBowers%26aufirst%3DC.%2BM.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26atitle%3DWater%2520Networks%2520contribute%2520to%2520enthalpy%252Fentropy%2520compensation%2520in%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D15579%26epage%3D15584%26doi%3D10.1021%2Fja4075776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref243"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref243'); return false;" data-citation="" class="refNumLink">243</a></strong><div class="NLM_citation" id="cit243"><span><span class="NLM_contrib-group">Maffucci, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contini, A.</span><span> </span><span class="NLM_article-title">Explicit ligand hydration shells improve the correlation between MM-PB/GBSA binding energies and experimental activities</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">2706</span><span class="NLM_x">–</span> <span class="NLM_lpage">2717</span><span class="refDoi"> DOI: 10.1021/ct400045d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400045d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFWgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=2706-2717&author=I.+Maffucciauthor=A.+Contini&title=Explicit+ligand+hydration+shells+improve+the+correlation+between+MM-PB%2FGBSA+binding+energies+and+experimental+activities&doi=10.1021%2Fct400045d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit243R"><div class="casContent"><span class="casTitleNuber">243</span><div class="casTitle"><span class="NLM_cas:atitle">Explicit Ligand Hydration Shells Improve the Correlation between MM-PB/GBSA Binding Energies and Experimental Activities</span></div><div class="casAuthors">Maffucci, Irene; Contini, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2706-2717</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. Mechanics Poisson-Boltzmann Surface Area (MM-PBSA) and Mol. Mechanics Generalized Born Surface Area (MM-GBSA) methods are widely used for drug design/discovery purposes.  However, it is not clear if the correlation between predicted and exptl. binding affinities can be improved by explicitly considering selected water mols. in the calcn. of binding energies, since different and sometimes diverging opinions are found in the literature.  The authors evaluated how variably populated hydration shells explicitly considered around the ligands may affect the correlation between MM-PB/GBSA computed binding energy and biol. activities (IC50 and ΔGbind, depending on the available exptl. data).  Four different systems--namely, the DNA-topoisomerase complex, α-thrombin, penicillopepsin, and avidin--were considered and ligand hydration shells populated by 10-70 water mols. were systematically evaluated.  The consideration of a hydration shell populated by a no. of water residues (Nwat) between 30 and 70 provided, in all of the considered examples, a pos. effect on correlation between MM-PB/GBSA calcd. binding affinities and exptl. activities, with a negligible increment of computational cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaTvCz5Lu7MLVg90H21EOLACvtfcHk0lgYp__q4aA9qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFWgsLg%253D&md5=5e1e7777a576e16567e92319631ff394</span></div><a href="/servlet/linkout?suffix=cit243&amp;dbid=16384&amp;doi=10.1021%2Fct400045d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400045d%26sid%3Dliteratum%253Aachs%26aulast%3DMaffucci%26aufirst%3DI.%26aulast%3DContini%26aufirst%3DA.%26atitle%3DExplicit%2520ligand%2520hydration%2520shells%2520improve%2520the%2520correlation%2520between%2520MM-PB%252FGBSA%2520binding%2520energies%2520and%2520experimental%2520activities%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D2706%26epage%3D2717%26doi%3D10.1021%2Fct400045d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172'],'ref173':['cit173'],'ref174':['cit174'],'ref175':['cit175'],'ref176':['cit176'],'ref177':['cit177'],'ref178':['cit178'],'ref179':['cit179'],'ref180':['cit180'],'ref181':['cit181'],'ref182':['cit182'],'ref183':['cit183'],'ref184':['cit184'],'ref185':['cit185'],'ref186':['cit186'],'ref187':['cit187'],'ref188':['cit188'],'ref189':['cit189'],'ref190':['cit190'],'ref191':['cit191'],'ref192':['cit192'],'ref193':['cit193'],'ref194':['cit194'],'ref195':['cit195'],'ref196':['cit196'],'ref197':['cit197'],'ref198':['cit198'],'ref199':['cit199'],'ref200':['cit200'],'ref201':['cit201'],'ref202':['cit202'],'ref203':['cit203'],'ref204':['cit204'],'ref205':['cit205'],'ref206':['cit206'],'ref207':['cit207'],'ref208':['cit208'],'ref209':['cit209'],'ref210':['cit210'],'ref211':['cit211'],'ref212':['cit212'],'ref213':['cit213'],'ref214':['cit214'],'ref215':['cit215'],'ref216':['cit216'],'ref217':['cit217'],'ref218':['cit218'],'ref219':['cit219'],'ref220':['cit220'],'ref221':['cit221'],'ref222':['cit222'],'ref223':['cit223'],'ref224':['cit224'],'ref225':['cit225'],'ref226':['cit226'],'ref227':['cit227'],'ref228':['cit228'],'ref229':['cit229'],'ref230':['cit230'],'ref231':['cit231'],'ref232':['cit232'],'ref233':['cit233'],'ref234':['cit234'],'ref235':['cit235'],'ref236':['cit236'],'ref237':['cit237'],'ref238':['cit238'],'ref239':['cit239'],'ref240':['cit240'],'ref241':['cit241'],'ref242':['cit242'],'ref243':['cit243']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 389 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Lara Callea, Laura Bonati, <span class="NLM_string-name hlFld-ContribAuthor">Stefano Motta</span>. </span><span class="cited-content_cbyCitation_article-title">Metadynamics-Based Approaches for Modeling the Hypoxia-Inducible Factor 2α Ligand Binding Process. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (7)
                                     , 3841-3851. <a href="https://doi.org/10.1021/acs.jctc.1c00114" title="DOI URL">https://doi.org/10.1021/acs.jctc.1c00114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.1c00114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.1c00114%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DMetadynamics-Based%252BApproaches%252Bfor%252BModeling%252Bthe%252BHypoxia-Inducible%252BFactor%252B2%2525CE%2525B1%252BLigand%252BBinding%252BProcess%26aulast%3DCallea%26aufirst%3DLara%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01022021%26date%3D04062021%26volume%3D17%26issue%3D7%26spage%3D3841%26epage%3D3851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Luan Carvalho Martins, Elio A. Cino, <span class="NLM_string-name hlFld-ContribAuthor">Rafaela Salgado Ferreira</span>. </span><span class="cited-content_cbyCitation_article-title">PyAutoFEP: An Automated Free Energy Perturbation Workflow for GROMACS Integrating Enhanced Sampling Methods. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (7)
                                     , 4262-4273. <a href="https://doi.org/10.1021/acs.jctc.1c00194" title="DOI URL">https://doi.org/10.1021/acs.jctc.1c00194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.1c00194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.1c00194%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DPyAutoFEP%25253A%252BAn%252BAutomated%252BFree%252BEnergy%252BPerturbation%252BWorkflow%252Bfor%252BGROMACS%252BIntegrating%252BEnhanced%252BSampling%252BMethods%26aulast%3DCarvalho%2BMartins%26aufirst%3DLuan%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D23022021%26date%3D18062021%26volume%3D17%26issue%3D7%26spage%3D4262%26epage%3D4273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marco Marcia, Jacopo Manigrasso, <span class="NLM_string-name hlFld-ContribAuthor">Marco De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">Finding the Ion in the RNA-Stack: Can Computational Models Accurately Predict Key Functional Elements in Large Macromolecular Complexes?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (6)
                                     , 2511-2515. <a href="https://doi.org/10.1021/acs.jcim.1c00572" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00572</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00572%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFinding%252Bthe%252BIon%252Bin%252Bthe%252BRNA-Stack%25253A%252BCan%252BComputational%252BModels%252BAccurately%252BPredict%252BKey%252BFunctional%252BElements%252Bin%252BLarge%252BMacromolecular%252BComplexes%25253F%26aulast%3DMarcia%26aufirst%3DMarco%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D22052021%26date%3D16062021%26volume%3D61%26issue%3D6%26spage%3D2511%26epage%3D2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maria Galvez-Llompart, Riccardo Ocello, Laura Rullo, Serena Stamatakos, Irene Alessandrini, Riccardo Zanni, Iñaki Tuñón, Andrea Cavalli, Sanzio Candeletti, Matteo Masetti, Patrizia Romualdi, <span class="NLM_string-name hlFld-ContribAuthor">Maurizio Recanatini</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the JAK/STAT Pathway: A Combined Ligand- and Target-Based Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (6)
                                     , 3091-3108. <a href="https://doi.org/10.1021/acs.jcim.0c01468" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01468</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01468%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTargeting%252Bthe%252BJAK%25252FSTAT%252BPathway%25253A%252BA%252BCombined%252BLigand-%252Band%252BTarget-Based%252BApproach%26aulast%3DGalvez-Llompart%26aufirst%3DMaria%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22122020%26date%3D17052021%26volume%3D61%26issue%3D6%26spage%3D3091%26epage%3D3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">André Fischer, Martin Smieško, Manuel Sellner, <span class="NLM_string-name hlFld-ContribAuthor">Markus A. Lill</span>. </span><span class="cited-content_cbyCitation_article-title">Decision Making in Structure-Based Drug Discovery: Visual Inspection of Docking Results. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2489-2500. <a href="https://doi.org/10.1021/acs.jmedchem.0c02227" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02227</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02227%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDecision%252BMaking%252Bin%252BStructure-Based%252BDrug%252BDiscovery%25253A%252BVisual%252BInspection%252Bof%252BDocking%252BResults%26aulast%3DFischer%26aufirst%3DAndr%25C3%25A9%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D23122020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2489%26epage%3D2500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Giuseppe Deganutti, Kerry Barkan, Barbara Preti, Michele Leuenberger, Mark Wall, Bruno G. Frenguelli, Martin Lochner, Graham Ladds, <span class="NLM_string-name hlFld-ContribAuthor">Christopher A. Reynolds</span>. </span><span class="cited-content_cbyCitation_article-title">Deciphering the Agonist Binding Mechanism to the Adenosine A1 Receptor. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2021,</strong> <em>4 </em>
                                    (1)
                                     , 314-326. <a href="https://doi.org/10.1021/acsptsci.0c00195" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00195</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00195%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DDeciphering%252Bthe%252BAgonist%252BBinding%252BMechanism%252Bto%252Bthe%252BAdenosine%252BA1%252BReceptor%26aulast%3DDeganutti%26aufirst%3DGiuseppe%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D13112020%26date%3D21012021%26volume%3D4%26issue%3D1%26spage%3D314%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mariarosaria Ferraro, Elisabetta Moroni, Emiliano Ippoliti, Silvia Rinaldi, Carlos Sanchez-Martin, Andrea Rasola, Luca F. Pavarino, <span class="NLM_string-name hlFld-ContribAuthor">Giorgio Colombo</span>. </span><span class="cited-content_cbyCitation_article-title">Machine Learning of Allosteric Effects: The Analysis of Ligand-Induced Dynamics to Predict Functional Effects in TRAP1. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (1)
                                     , 101-114. <a href="https://doi.org/10.1021/acs.jpcb.0c09742" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c09742</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c09742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c09742%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DMachine%252BLearning%252Bof%252BAllosteric%252BEffects%25253A%252BThe%252BAnalysis%252Bof%252BLigand-Induced%252BDynamics%252Bto%252BPredict%252BFunctional%252BEffects%252Bin%252BTRAP1%26aulast%3DFerraro%26aufirst%3DMariarosaria%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28102020%26date%3D07122020%26date%3D28122020%26volume%3D125%26issue%3D1%26spage%3D101%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jordi Juárez-Jiménez, Philip Tew, Michael O′Connor, Salomé Llabrés, Rebecca Sage, David Glowacki, <span class="NLM_string-name hlFld-ContribAuthor">Julien Michel</span>. </span><span class="cited-content_cbyCitation_article-title">Combining Virtual Reality Visualization with Ensemble Molecular Dynamics to Study Complex Protein Conformational Changes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6344-6354. <a href="https://doi.org/10.1021/acs.jcim.0c00221" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00221%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCombining%252BVirtual%252BReality%252BVisualization%252Bwith%252BEnsemble%252BMolecular%252BDynamics%252Bto%252BStudy%252BComplex%252BProtein%252BConformational%252BChanges%26aulast%3DJu%25C3%25A1rez-Jim%25C3%25A9nez%26aufirst%3DJordi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24042020%26date%3D12112020%26volume%3D60%26issue%3D12%26spage%3D6344%26epage%3D6354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Levente M. Mihalovits, György G. Ferenczy, <span class="NLM_string-name hlFld-ContribAuthor">György M. Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Affinity and Selectivity Assessment of Covalent Inhibitors by Free Energy Calculations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6579-6594. <a href="https://doi.org/10.1021/acs.jcim.0c00834" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00834</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00834%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAffinity%252Band%252BSelectivity%252BAssessment%252Bof%252BCovalent%252BInhibitors%252Bby%252BFree%252BEnergy%252BCalculations%26aulast%3DMihalovits%26aufirst%3DLevente%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22072020%26date%3D09122020%26volume%3D60%26issue%3D12%26spage%3D6579%26epage%3D6594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sergio Decherchi, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">Thermodynamics and Kinetics of Drug-Target Binding by Molecular Simulation. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2020,</strong> <em>120 </em>
                                    (23)
                                     , 12788-12833. <a href="https://doi.org/10.1021/acs.chemrev.0c00534" title="DOI URL">https://doi.org/10.1021/acs.chemrev.0c00534</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.0c00534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.0c00534%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DThermodynamics%252Band%252BKinetics%252Bof%252BDrug-Target%252BBinding%252Bby%252BMolecular%252BSimulation%26aulast%3DDecherchi%26aufirst%3DSergio%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28052020%26date%3D02102020%26volume%3D120%26issue%3D23%26spage%3D12788%26epage%3D12833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pietro Vidossich, Luis Eduardo Castañeda Moreno, Cristiano Mota, Daniele de Sanctis, Gian Pietro Miscione, <span class="NLM_string-name hlFld-ContribAuthor">Marco De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">Functional Implications of Second-Shell Basic Residues for dUTPase DR2231 Enzymatic Specificity. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2020,</strong> <em>10 </em>
                                    (23)
                                     , 13825-13833. <a href="https://doi.org/10.1021/acscatal.0c04148" title="DOI URL">https://doi.org/10.1021/acscatal.0c04148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.0c04148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.0c04148%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DFunctional%252BImplications%252Bof%252BSecond-Shell%252BBasic%252BResidues%252Bfor%252BdUTPase%252BDR2231%252BEnzymatic%252BSpecificity%26aulast%3DVidossich%26aufirst%3DPietro%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22092020%26date%3D28102020%26date%3D12112020%26volume%3D10%26issue%3D23%26spage%3D13825%26epage%3D13833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jan Walther Perthold, Dražen Petrov, <span class="NLM_string-name hlFld-ContribAuthor">Chris Oostenbrink</span>. </span><span class="cited-content_cbyCitation_article-title">Toward Automated Free Energy Calculation with Accelerated Enveloping Distribution Sampling (A-EDS). </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (11)
                                     , 5395-5406. <a href="https://doi.org/10.1021/acs.jcim.0c00456" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00456%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DToward%252BAutomated%252BFree%252BEnergy%252BCalculation%252Bwith%252BAccelerated%252BEnveloping%252BDistribution%252BSampling%252B%252528A-EDS%252529%26aulast%3DPerthold%26aufirst%3DJan%2BWalther%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30042020%26date%3D23062020%26date%3D03062020%26volume%3D60%26issue%3D11%26spage%3D5395%26epage%3D5406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mariia Radaeva, Xuesen Dong, <span class="NLM_string-name hlFld-ContribAuthor">Artem Cherkasov</span>. </span><span class="cited-content_cbyCitation_article-title">The Use of Methods of Computer-Aided Drug Discovery in the Development of Topoisomerase II Inhibitors: Applications and Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3703-3721. <a href="https://doi.org/10.1021/acs.jcim.0c00325" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00325%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BUse%252Bof%252BMethods%252Bof%252BComputer-Aided%252BDrug%252BDiscovery%252Bin%252Bthe%252BDevelopment%252Bof%252BTopoisomerase%252BII%252BInhibitors%25253A%252BApplications%252Band%252BFuture%252BDirections%26aulast%3DRadaeva%26aufirst%3DMariia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03042020%26date%3D03082020%26date%3D20072020%26volume%3D60%26issue%3D8%26spage%3D3703%26epage%3D3721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Benjamin R. Jagger, Anupam A. Ojha, <span class="NLM_string-name hlFld-ContribAuthor">Rommie E. Amaro</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Ligand Binding Kinetics Using a Markovian Milestoning with Voronoi Tessellations Multiscale Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (8)
                                     , 5348-5357. <a href="https://doi.org/10.1021/acs.jctc.0c00495" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c00495</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c00495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c00495%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DPredicting%252BLigand%252BBinding%252BKinetics%252BUsing%252Ba%252BMarkovian%252BMilestoning%252Bwith%252BVoronoi%252BTessellations%252BMultiscale%252BApproach%26aulast%3DJagger%26aufirst%3DBenjamin%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D13052020%26date%3D09072020%26date%3D24062020%26volume%3D16%26issue%3D8%26spage%3D5348%26epage%3D5357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mohammed Benabderrahmane, Ronan Bureau, Anne Sophie Voisin-Chiret, <span class="NLM_string-name hlFld-ContribAuthor">Jana Sopkova-de Oliveira Santos</span>. </span><span class="cited-content_cbyCitation_article-title">Insights into Mcl-1 Conformational States and Allosteric Inhibition Mechanism from Molecular Dynamics Simulations, Enhanced Sampling, and Pocket Crosstalk Analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 3172-3187. <a href="https://doi.org/10.1021/acs.jcim.0c00315" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00315</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00315%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInsights%252Binto%252BMcl-1%252BConformational%252BStates%252Band%252BAllosteric%252BInhibition%252BMechanism%252Bfrom%252BMolecular%252BDynamics%252BSimulations%25252C%252BEnhanced%252BSampling%25252C%252Band%252BPocket%252BCrosstalk%252BAnalysis%26aulast%3DBenabderrahmane%26aufirst%3DMohammed%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D28052020%26date%3D11052020%26volume%3D60%26issue%3D6%26spage%3D3172%26epage%3D3187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kevin R. Hinkle, <span class="NLM_string-name hlFld-ContribAuthor">Frederick R. Phelan, Jr.</span>. </span><span class="cited-content_cbyCitation_article-title">Solvation Free Energy of Self-Assembled Complexes: Using Molecular Dynamics to Understand the Separation of ssDNA-Wrapped Single-Walled Carbon Nanotubes. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry C</span><span> <strong>2020,</strong> <em>124 </em>
                                    (24)
                                     , 13127-13140. <a href="https://doi.org/10.1021/acs.jpcc.0c00983" title="DOI URL">https://doi.org/10.1021/acs.jpcc.0c00983</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcc.0c00983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcc.0c00983%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520C%26atitle%3DSolvation%252BFree%252BEnergy%252Bof%252BSelf-Assembled%252BComplexes%25253A%252BUsing%252BMolecular%252BDynamics%252Bto%252BUnderstand%252Bthe%252BSeparation%252Bof%252BssDNA-Wrapped%252BSingle-Walled%252BCarbon%252BNanotubes%26aulast%3DHinkle%26aufirst%3DKevin%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D04022020%26date%3D14052020%26date%3D04062020%26date%3D20052020%26volume%3D124%26issue%3D24%26spage%3D13127%26epage%3D13140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Reed E. S. Harrison, Nehemiah T. Zewde, Yogesh B. Narkhede, Rohaine V. Hsu, Dimitrios Morikis, Valentine I. Vullev, <span class="NLM_string-name hlFld-ContribAuthor">Giulia Palermo</span>. </span><span class="cited-content_cbyCitation_article-title">Factor H-Inspired Design of Peptide Biomarkers of the Complement C3d Protein. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 1054-1059. <a href="https://doi.org/10.1021/acsmedchemlett.9b00663" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00663</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00663%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DFactor%252BH-Inspired%252BDesign%252Bof%252BPeptide%252BBiomarkers%252Bof%252Bthe%252BComplement%252BC3d%252BProtein%26aulast%3DHarrison%26aufirst%3DReed%2BE.%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D30122019%26date%3D28022020%26date%3D05032020%26date%3D28022020%26volume%3D11%26issue%3D5%26spage%3D1054%26epage%3D1059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Colin K. Egan, Brandon B. Bizzarro, Marc Riera, <span class="NLM_string-name hlFld-ContribAuthor">Francesco Paesani</span>. </span><span class="cited-content_cbyCitation_article-title">Nature of Alkali Ion–Water Interactions: Insights from Many-Body Representations and Density Functional Theory. II. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (5)
                                     , 3055-3072. <a href="https://doi.org/10.1021/acs.jctc.0c00082" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c00082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c00082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c00082%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D2983%26doi%3D10.1021%2Facs.jctc.9b00064%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DNature%252Bof%252BAlkali%252BIon%2525E2%252580%252593Water%252BInteractions%25253A%252BInsights%252Bfrom%252BMany-Body%252BRepresentations%252Band%252BDensity%252BFunctional%252BTheory.%252BII%26aulast%3DEgan%26aufirst%3DColin%2BK.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D23012020%26date%3D16042020%26date%3D06042020%26volume%3D16%26issue%3D5%26spage%3D3055%26epage%3D3072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuzhe Wang, Jagna Witek, Gregory A. Landrum, <span class="NLM_string-name hlFld-ContribAuthor">Sereina Riniker</span>. </span><span class="cited-content_cbyCitation_article-title">Improving Conformer Generation for Small Rings and Macrocycles Based on Distance Geometry and Experimental Torsional-Angle Preferences. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 2044-2058. <a href="https://doi.org/10.1021/acs.jcim.0c00025" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproving%252BConformer%252BGeneration%252Bfor%252BSmall%252BRings%252Band%252BMacrocycles%252BBased%252Bon%252BDistance%252BGeometry%252Band%252BExperimental%252BTorsional-Angle%252BPreferences%26aulast%3DWang%26aufirst%3DShuzhe%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09012020%26date%3D20032020%26date%3D10032020%26volume%3D60%26issue%3D4%26spage%3D2044%26epage%3D2058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Linn S. M. Evenseth, Riccardo Ocello, Mari Gabrielsen, Matteo Masetti, Maurizio Recanatini, Ingebrigt Sylte, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Conformational Dynamics of the Extracellular Venus flytrap Domain of the GABAB Receptor: A Path-Metadynamics Study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 2294-2303. <a href="https://doi.org/10.1021/acs.jcim.0c00163" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00163</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00163%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252BConformational%252BDynamics%252Bof%252Bthe%252BExtracellular%252BVenus%252Bflytrap%252BDomain%252Bof%252Bthe%252BGABAB%252BReceptor%25253A%252BA%252BPath-Metadynamics%252BStudy%26aulast%3DEvenseth%26aufirst%3DLinn%2BS.%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14022020%26date%3D15042020%26date%3D01042020%26volume%3D60%26issue%3D4%26spage%3D2294%26epage%3D2303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jose M. Arencibia, Nicoletta Brindani, Sebastian Franco-Ulloa, Michela Nigro, Jissy Akkarapattiakal Kuriappan, Giuliana Ottonello, Sine Mandrup Bertozzi, Maria Summa, Stefania Girotto, Rosalia Bertorelli, Andrea Armirotti, <span class="NLM_string-name hlFld-ContribAuthor">Marco De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, Dynamic Docking, Biochemical Characterization, and in Vivo Pharmacokinetics Studies of Novel Topoisomerase II Poisons with Promising Antiproliferative Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (7)
                                     , 3508-3521. <a href="https://doi.org/10.1021/acs.jmedchem.9b01760" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01760</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01760%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252BDynamic%252BDocking%25252C%252BBiochemical%252BCharacterization%25252C%252Band%252Bin%252BVivo%252BPharmacokinetics%252BStudies%252Bof%252BNovel%252BTopoisomerase%252BII%252BPoisons%252Bwith%252BPromising%252BAntiproliferative%252BActivity%26aulast%3DArencibia%26aufirst%3DJose%2BM.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D30102019%26date%3D30032020%26date%3D20032020%26volume%3D63%26issue%3D7%26spage%3D3508%26epage%3D3521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Giuseppe Deganutti, Stefano Moro, <span class="NLM_string-name hlFld-ContribAuthor">Christopher A. Reynolds</span>. </span><span class="cited-content_cbyCitation_article-title">A Supervised Molecular Dynamics Approach to Unbiased Ligand–Protein Unbinding. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1804-1817. <a href="https://doi.org/10.1021/acs.jcim.9b01094" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01094%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DA%252BSupervised%252BMolecular%252BDynamics%252BApproach%252Bto%252BUnbiased%252BLigand%2525E2%252580%252593Protein%252BUnbinding%26aulast%3DDeganutti%26aufirst%3DGiuseppe%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D26112019%26date%3D09032020%26date%3D03032020%26volume%3D60%26issue%3D3%26spage%3D1804%26epage%3D1817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor "> David    Ban </span>. </span><span class="cited-content_cbyCitation_article-title">Evolving Role of Conformational Dynamics in Understanding Fundamental Biomolecular Behavior. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 57-81. <a href="https://doi.org/10.1021/bk-2020-1357.ch004" title="DOI URL">https://doi.org/10.1021/bk-2020-1357.ch004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/bk-2020-1357.ch004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fbk-2020-1357.ch004%26sid%3Dliteratum%253Aachs%26atitle%3DEvolving%252BRole%252Bof%252BConformational%252BDynamics%252Bin%252BUnderstanding%252BFundamental%252BBiomolecular%252BBehavior%26aulast%3DBan%26aufirst%3DDavid%26date%3D2020%26date%3D2020%26spage%3D57%26epage%3D81%26atitle%3DMechanistic%252BEnzymology%25253A%252BBridging%252BStructure%252Band%252BFunction%26aulast%3DMiller%26aufirst%3DJustin%2BM.%26date%3D2020%26date%3D2020%26volume%3D1357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marta Ruiz-Garcia, Pilar Pérez-Lozano, Débora Mercadé-Frutós, Anna Nardi-Ricart, Marc Suñé-Pou, Mary Cano-Sarabia, Sonia Garcia-Jimeno, Josep M. Suñé-Negre, Daniel Maspoch, <span class="NLM_string-name hlFld-ContribAuthor">Encarna García-Montoya</span>. </span><span class="cited-content_cbyCitation_article-title">Development and Validation of a New High-Performance Liquid Chromatography Method for the Simultaneous Quantification of Coenzyme Q10, Phosphatidylserine, and Vitamin C from a Cutting-Edge Liposomal Vehiculization. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (22)
                                     , 19710-19715. <a href="https://doi.org/10.1021/acsomega.9b02456" title="DOI URL">https://doi.org/10.1021/acsomega.9b02456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b02456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b02456%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDevelopment%252Band%252BValidation%252Bof%252Ba%252BNew%252BHigh-Performance%252BLiquid%252BChromatography%252BMethod%252Bfor%252Bthe%252BSimultaneous%252BQuantification%252Bof%252BCoenzyme%252BQ10%25252C%252BPhosphatidylserine%25252C%252Band%252BVitamin%252BC%252Bfrom%252Ba%252BCutting-Edge%252BLiposomal%252BVehiculization%26aulast%3DRuiz-Garcia%26aufirst%3DMarta%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D02082019%26date%3D10102019%26date%3D11112019%26volume%3D4%26issue%3D22%26spage%3D19710%26epage%3D19715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marko Hanževački, Radha Dilip Banhatti, Karmen Čondić-Jurkić, Ana-Sunčana Smith, <span class="NLM_string-name hlFld-ContribAuthor">David M. Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Reactive Conformations of Coenzyme A during Binding and Unbinding to Pyruvate Formate–Lyase. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry A</span><span> <strong>2019,</strong> <em>123 </em>
                                    (43)
                                     , 9345-9356. <a href="https://doi.org/10.1021/acs.jpca.9b06913" title="DOI URL">https://doi.org/10.1021/acs.jpca.9b06913</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpca.9b06913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpca.9b06913%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520A%26atitle%3DExploring%252BReactive%252BConformations%252Bof%252BCoenzyme%252BA%252Bduring%252BBinding%252Band%252BUnbinding%252Bto%252BPyruvate%252BFormate%2525E2%252580%252593Lyase%26aulast%3DHan%25C5%25BEeva%25C4%258Dki%26aufirst%3DMarko%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20072019%26date%3D03102019%26date%3D17102019%26date%3D03102019%26volume%3D123%26issue%3D43%26spage%3D9345%26epage%3D9356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hui Wang, Zhuodong Tang, Yan Wang, Guangzhong Ma, <span class="NLM_string-name hlFld-ContribAuthor">Nongjian Tao</span>. </span><span class="cited-content_cbyCitation_article-title">Probing Single Molecule Binding and Free Energy Profile with Plasmonic Imaging of Nanoparticles. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (40)
                                     , 16071-16078. <a href="https://doi.org/10.1021/jacs.9b08405" title="DOI URL">https://doi.org/10.1021/jacs.9b08405</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b08405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b08405%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DProbing%252BSingle%252BMolecule%252BBinding%252Band%252BFree%252BEnergy%252BProfile%252Bwith%252BPlasmonic%252BImaging%252Bof%252BNanoparticles%26aulast%3DWang%26aufirst%3DHui%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04082019%26date%3D25092019%26date%3D16092019%26volume%3D141%26issue%3D40%26spage%3D16071%26epage%3D16078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mattia Bernetti, Matteo Masetti, Maurizio Recanatini, Rommie E. Amaro, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">An Integrated Markov State Model and Path Metadynamics Approach To Characterize Drug Binding Processes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (10)
                                     , 5689-5702. <a href="https://doi.org/10.1021/acs.jctc.9b00450" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b00450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b00450%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DAn%252BIntegrated%252BMarkov%252BState%252BModel%252Band%252BPath%252BMetadynamics%252BApproach%252BTo%252BCharacterize%252BDrug%252BBinding%252BProcesses%26aulast%3DBernetti%26aufirst%3DMattia%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08052019%26date%3D05092019%26date%3D22082019%26volume%3D15%26issue%3D10%26spage%3D5689%26epage%3D5702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jissy A. Kuriappan, Neil Osheroff, <span class="NLM_string-name hlFld-ContribAuthor">Marco De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">Smoothed Potential MD Simulations for Dissociation Kinetics of Etoposide To Unravel Isoform Specificity in Targeting Human Topoisomerase II. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 4007-4017. <a href="https://doi.org/10.1021/acs.jcim.9b00605" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00605</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00605%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSmoothed%252BPotential%252BMD%252BSimulations%252Bfor%252BDissociation%252BKinetics%252Bof%252BEtoposide%252BTo%252BUnravel%252BIsoform%252BSpecificity%252Bin%252BTargeting%252BHuman%252BTopoisomerase%252BII%26aulast%3DKuriappan%26aufirst%3DJissy%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23072019%26date%3D09092019%26date%3D26082019%26volume%3D59%26issue%3D9%26spage%3D4007%26epage%3D4017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Akshaya
K. Das, Lars Urban, Itai Leven, Matthias Loipersberger, Abdulrahman Aldossary, Martin Head-Gordon, <span class="NLM_string-name hlFld-ContribAuthor">Teresa Head-Gordon</span>. </span><span class="cited-content_cbyCitation_article-title">Development of an Advanced Force Field for Water Using Variational Energy Decomposition Analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (9)
                                     , 5001-5013. <a href="https://doi.org/10.1021/acs.jctc.9b00478" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b00478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b00478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b00478%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DDevelopment%252Bof%252Ban%252BAdvanced%252BForce%252BField%252Bfor%252BWater%252BUsing%252BVariational%252BEnergy%252BDecomposition%252BAnalysis%26aulast%3DDas%26aufirst%3DAkshaya%2BK.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D16052019%26date%3D26082019%26date%3D13082019%26volume%3D15%26issue%3D9%26spage%3D5001%26epage%3D5013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vladimir
P. Berishvili, Valentin O. Perkin, Andrew E. Voronkov, Eugene V. Radchenko, Riyaz Syed, Chittireddy Venkata Ramana Reddy, Viness Pillay, Pradeep Kumar, Yahya E. Choonara, Ahmed Kamal, <span class="NLM_string-name hlFld-ContribAuthor">Vladimir A. Palyulin</span>. </span><span class="cited-content_cbyCitation_article-title">Time-Domain Analysis of Molecular Dynamics Trajectories Using Deep Neural Networks: Application to Activity Ranking of Tankyrase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (8)
                                     , 3519-3532. <a href="https://doi.org/10.1021/acs.jcim.9b00135" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00135</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00135%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTime-Domain%252BAnalysis%252Bof%252BMolecular%252BDynamics%252BTrajectories%252BUsing%252BDeep%252BNeural%252BNetworks%25253A%252BApplication%252Bto%252BActivity%252BRanking%252Bof%252BTankyrase%252BInhibitors%26aulast%3DBerishvili%26aufirst%3DVladimir%2BP.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D14022019%26date%3D19072019%26date%3D05072019%26volume%3D59%26issue%3D8%26spage%3D3519%26epage%3D3532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tana Tandarić, <span class="NLM_string-name hlFld-ContribAuthor">Robert Vianello</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Insight into the Mechanism of the Irreversible Inhibition of Monoamine Oxidase Enzymes by the Antiparkinsonian Propargylamine Inhibitors Rasagiline and Selegiline. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (8)
                                     , 3532-3542. <a href="https://doi.org/10.1021/acschemneuro.9b00147" title="DOI URL">https://doi.org/10.1021/acschemneuro.9b00147</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.9b00147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.9b00147%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DComputational%252BInsight%252Binto%252Bthe%252BMechanism%252Bof%252Bthe%252BIrreversible%252BInhibition%252Bof%252BMonoamine%252BOxidase%252BEnzymes%252Bby%252Bthe%252BAntiparkinsonian%252BPropargylamine%252BInhibitors%252BRasagiline%252Band%252BSelegiline%26aulast%3DTandari%25C4%2587%26aufirst%3DTana%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D13032019%26date%3D17062019%26date%3D02072019%26date%3D17062019%26volume%3D10%26issue%3D8%26spage%3D3532%26epage%3D3542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Agostino Bruno, Elisabetta Barresi, Nicola Simola, Eleonora Da Pozzo, Barbara Costa, Ettore Novellino, Federico Da Settimo, Claudia Martini, Sabrina Taliani, <span class="NLM_string-name hlFld-ContribAuthor">Sandro Cosconati</span>. </span><span class="cited-content_cbyCitation_article-title">Unbinding of Translocator Protein 18 kDa (TSPO) Ligands: From in Vitro Residence Time to in Vivo Efficacy via in Silico Simulations. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (8)
                                     , 3805-3814. <a href="https://doi.org/10.1021/acschemneuro.9b00300" title="DOI URL">https://doi.org/10.1021/acschemneuro.9b00300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.9b00300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.9b00300%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DUnbinding%252Bof%252BTranslocator%252BProtein%252B18%252BkDa%252B%252528TSPO%252529%252BLigands%25253A%252BFrom%252Bin%252BVitro%252BResidence%252BTime%252Bto%252Bin%252BVivo%252BEfficacy%252Bvia%252Bin%252BSilico%252BSimulations%26aulast%3DBruno%26aufirst%3DAgostino%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22052019%26date%3D21062019%26date%3D03072019%26date%3D21062019%26volume%3D10%26issue%3D8%26spage%3D3805%26epage%3D3814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shaoyong Lu, Xinheng He, Duan Ni, <span class="NLM_string-name hlFld-ContribAuthor">Jian Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modulator Discovery: From Serendipity to Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6405-6421. <a href="https://doi.org/10.1021/acs.jmedchem.8b01749" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01749%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252BModulator%252BDiscovery%25253A%252BFrom%252BSerendipity%252Bto%252BStructure-Based%252BDesign%26aulast%3DLu%26aufirst%3DShaoyong%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D11112018%26date%3D07032019%26date%3D28022019%26volume%3D62%26issue%3D14%26spage%3D6405%26epage%3D6421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Emelie Flood, Céline Boiteux, Bogdan Lev, Igor Vorobyov, <span class="NLM_string-name hlFld-ContribAuthor">Toby W. Allen</span>. </span><span class="cited-content_cbyCitation_article-title">Atomistic Simulations of Membrane Ion Channel Conduction, Gating, and Modulation. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (13)
                                     , 7737-7832. <a href="https://doi.org/10.1021/acs.chemrev.8b00630" title="DOI URL">https://doi.org/10.1021/acs.chemrev.8b00630</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.8b00630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.8b00630%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DAtomistic%252BSimulations%252Bof%252BMembrane%252BIon%252BChannel%252BConduction%25252C%252BGating%25252C%252Band%252BModulation%26aulast%3DFlood%26aufirst%3DEmelie%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18102018%26date%3D27062019%26volume%3D119%26issue%3D13%26spage%3D7737%26epage%3D7832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Israel Cabeza de Vaca, Ricardo Zarzuela, Julian Tirado-Rives, <span class="NLM_string-name hlFld-ContribAuthor">William L. Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Robust Free Energy Perturbation Protocols for Creating Molecules in Solution. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (7)
                                     , 3941-3948. <a href="https://doi.org/10.1021/acs.jctc.9b00213" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b00213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b00213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b00213%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DRobust%252BFree%252BEnergy%252BPerturbation%252BProtocols%252Bfor%252BCreating%252BMolecules%252Bin%252BSolution%26aulast%3DCabeza%2Bde%2BVaca%26aufirst%3DIsrael%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D01032019%26date%3D24062019%26date%3D11062019%26volume%3D15%26issue%3D7%26spage%3D3941%26epage%3D3948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">E. Prabhu Raman</span>. </span><span class="cited-content_cbyCitation_article-title">Template-Based Method for Conformation Generation and Scoring for Congeneric Series of Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2690-2701. <a href="https://doi.org/10.1021/acs.jcim.9b00032" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00032%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTemplate-Based%252BMethod%252Bfor%252BConformation%252BGeneration%252Band%252BScoring%252Bfor%252BCongeneric%252BSeries%252Bof%252BLigands%26aulast%3DRaman%26aufirst%3DE.%2BPrabhu%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07012019%26date%3D09052019%26date%3D02052019%26volume%3D59%26issue%3D6%26spage%3D2690%26epage%3D2701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Srinivasaraghavan Kannan, Stephen J. Fox, <span class="NLM_string-name hlFld-ContribAuthor">Chandra S. Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Gatekeeper Mutations in EGFR through Computer Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2850-2858. <a href="https://doi.org/10.1021/acs.jcim.9b00361" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00361</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00361%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252BGatekeeper%252BMutations%252Bin%252BEGFR%252Bthrough%252BComputer%252BSimulations%26aulast%3DKannan%26aufirst%3DSrinivasaraghavan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29042019%26date%3D30052019%26date%3D17052019%26volume%3D59%26issue%3D6%26spage%3D2850%26epage%3D2858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Claudia Llinas del Torrent, Nil Casajuana-Martin, Leonardo Pardo, Gary Tresadern, <span class="NLM_string-name hlFld-ContribAuthor">Laura Pérez-Benito</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms Underlying Allosteric Molecular Switches of Metabotropic Glutamate Receptor 5. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (5)
                                     , 2456-2466. <a href="https://doi.org/10.1021/acs.jcim.8b00924" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00924</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00924%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMechanisms%252BUnderlying%252BAllosteric%252BMolecular%252BSwitches%252Bof%252BMetabotropic%252BGlutamate%252BReceptor%252B5%26aulast%3DLlinas%2Bdel%2BTorrent%26aufirst%3DClaudia%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D12122018%26date%3D12032019%26date%3D27022019%26volume%3D59%26issue%3D5%26spage%3D2456%26epage%3D2466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brandon
B. Bizzarro, Colin K. Egan, <span class="NLM_string-name hlFld-ContribAuthor">Francesco Paesani</span>. </span><span class="cited-content_cbyCitation_article-title">Nature of Halide–Water Interactions: Insights from Many-Body Representations and Density Functional Theory. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (5)
                                     , 2983-2995. <a href="https://doi.org/10.1021/acs.jctc.9b00064" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b00064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b00064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b00064%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DNature%252Bof%252BHalide%2525E2%252580%252593Water%252BInteractions%25253A%252BInsights%252Bfrom%252BMany-Body%252BRepresentations%252Band%252BDensity%252BFunctional%252BTheory%26aulast%3DBizzarro%26aufirst%3DBrandon%2BB.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23012019%26date%3D04042019%26date%3D26032019%26volume%3D15%26issue%3D5%26spage%3D2983%26epage%3D2995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Natalie
J. Tatum, Fernanda Duarte, Shina C. L. Kamerlin, <span class="NLM_string-name hlFld-ContribAuthor">Ehmke Pohl</span>. </span><span class="cited-content_cbyCitation_article-title">Relative Binding Energies Predict Crystallographic Binding Modes of Ethionamide Booster Lead Compounds. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 2244-2249. <a href="https://doi.org/10.1021/acs.jpclett.9b00741" title="DOI URL">https://doi.org/10.1021/acs.jpclett.9b00741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.9b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.9b00741%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DRelative%252BBinding%252BEnergies%252BPredict%252BCrystallographic%252BBinding%252BModes%252Bof%252BEthionamide%252BBooster%252BLead%252BCompounds%26aulast%3DTatum%26aufirst%3DNatalie%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D14032019%26date%3D09042019%26date%3D23042019%26date%3D09042019%26volume%3D10%26issue%3D9%26spage%3D2244%26epage%3D2249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Aria Gheeraert, Lorenza Pacini, Victor S. Batista, Laurent Vuillon, Claire Lesieur, <span class="NLM_string-name hlFld-ContribAuthor">Ivan Rivalta</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Allosteric Pathways of a V-Type Enzyme with Dynamical Perturbation Networks. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2019,</strong> <em>123 </em>
                                    (16)
                                     , 3452-3461. <a href="https://doi.org/10.1021/acs.jpcb.9b01294" title="DOI URL">https://doi.org/10.1021/acs.jpcb.9b01294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.9b01294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.9b01294%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DExploring%252BAllosteric%252BPathways%252Bof%252Ba%252BV-Type%252BEnzyme%252Bwith%252BDynamical%252BPerturbation%252BNetworks%26aulast%3DGheeraert%26aufirst%3DAria%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08022019%26date%3D31032019%26date%3D15042019%26date%3D03042019%26volume%3D123%26issue%3D16%26spage%3D3452%26epage%3D3461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrea Basciu, Giuliano Malloci, Fabio Pietrucci, Alexandre M. J.
J. Bonvin, <span class="NLM_string-name hlFld-ContribAuthor">Attilio V. Vargiu</span>. </span><span class="cited-content_cbyCitation_article-title">Holo-like and Druggable Protein Conformations from Enhanced Sampling of Binding Pocket Volume and Shape. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (4)
                                     , 1515-1528. <a href="https://doi.org/10.1021/acs.jcim.8b00730" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00730</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00730%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHolo-like%252Band%252BDruggable%252BProtein%252BConformations%252Bfrom%252BEnhanced%252BSampling%252Bof%252BBinding%252BPocket%252BVolume%252Band%252BShape%26aulast%3DBasciu%26aufirst%3DAndrea%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D19102018%26date%3D28032019%26date%3D18032019%26volume%3D59%26issue%3D4%26spage%3D1515%26epage%3D1528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joshua
T. Horton, Alice E. A. Allen, Leela S. Dodda, <span class="NLM_string-name hlFld-ContribAuthor">Daniel J. Cole</span>. </span><span class="cited-content_cbyCitation_article-title">QUBEKit: Automating the Derivation of Force Field Parameters from Quantum Mechanics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (4)
                                     , 1366-1381. <a href="https://doi.org/10.1021/acs.jcim.8b00767" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00767%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DQUBEKit%25253A%252BAutomating%252Bthe%252BDerivation%252Bof%252BForce%252BField%252BParameters%252Bfrom%252BQuantum%252BMechanics%26aulast%3DHorton%26aufirst%3DJoshua%2BT.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31102018%26date%3D22022019%26date%3D11022019%26volume%3D59%26issue%3D4%26spage%3D1366%26epage%3D1381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Duy Phuoc Tran, <span class="NLM_string-name hlFld-ContribAuthor">Akio Kitao</span>. </span><span class="cited-content_cbyCitation_article-title">Dissociation Process of a MDM2/p53 Complex Investigated by Parallel Cascade Selection Molecular Dynamics and the Markov State Model. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2019,</strong> <em>123 </em>
                                    (11)
                                     , 2469-2478. <a href="https://doi.org/10.1021/acs.jpcb.8b10309" title="DOI URL">https://doi.org/10.1021/acs.jpcb.8b10309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.8b10309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.8b10309%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DDissociation%252BProcess%252Bof%252Ba%252BMDM2%25252Fp53%252BComplex%252BInvestigated%252Bby%252BParallel%252BCascade%252BSelection%252BMolecular%252BDynamics%252Band%252Bthe%252BMarkov%252BState%252BModel%26aulast%3DTran%26aufirst%3DDuy%2BPhuoc%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D23102018%26date%3D26122018%26date%3D30012019%26date%3D15012019%26volume%3D123%26issue%3D11%26spage%3D2469%26epage%3D2478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yunyi Wang, <span class="NLM_string-name hlFld-ContribAuthor">Christian Hilty</span>. </span><span class="cited-content_cbyCitation_article-title">Determination of Ligand Binding Epitope Structures Using Polarization Transfer from Hyperpolarized Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2419-2427. <a href="https://doi.org/10.1021/acs.jmedchem.8b01711" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01711</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01711%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDetermination%252Bof%252BLigand%252BBinding%252BEpitope%252BStructures%252BUsing%252BPolarization%252BTransfer%252Bfrom%252BHyperpolarized%252BLigands%26aulast%3DWang%26aufirst%3DYunyi%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D02112018%26date%3D20022019%26date%3D04022019%26volume%3D62%26issue%3D5%26spage%3D2419%26epage%3D2427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dario Vassetti, Marco Pagliai, <span class="NLM_string-name hlFld-ContribAuthor">Piero Procacci</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of GAFF2 and OPLS-AA General Force Fields in Combination with the Water Models TIP3P, SPCE, and OPC3 for the Solvation Free Energy of Druglike Organic Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (3)
                                     , 1983-1995. <a href="https://doi.org/10.1021/acs.jctc.8b01039" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b01039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b01039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b01039%26sid%3Dliteratum%253Aachs%26jtitle%3DNucl.%2520Acids%2520Res.%26date%3D2017%26volume%3D121%26spage%3D3864%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1240%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D2813%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DAssessment%252Bof%252BGAFF2%252Band%252BOPLS-AA%252BGeneral%252BForce%252BFields%252Bin%252BCombination%252Bwith%252Bthe%252BWater%252BModels%252BTIP3P%25252C%252BSPCE%25252C%252Band%252BOPC3%252Bfor%252Bthe%252BSolvation%252BFree%252BEnergy%252Bof%252BDruglike%252BOrganic%252BMolecules%26aulast%3DVassetti%26aufirst%3DDario%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D22102018%26date%3D14022019%26date%3D29012019%26volume%3D15%26issue%3D3%26spage%3D1983%26epage%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Laura Pérez-Benito, Nil Casajuana-Martin, Mireia Jiménez-Rosés, Herman van Vlijmen, <span class="NLM_string-name hlFld-ContribAuthor">Gary Tresadern</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Activity Cliffs with Free-Energy Perturbation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (3)
                                     , 1884-1895. <a href="https://doi.org/10.1021/acs.jctc.8b01290" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b01290</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b01290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b01290%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DPredicting%252BActivity%252BCliffs%252Bwith%252BFree-Energy%252BPerturbation%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DLaura%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D21122018%26date%3D01032019%26date%3D18022019%26volume%3D15%26issue%3D3%26spage%3D1884%26epage%3D1895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sebastian Franco-Ulloa, Laura Riccardi, Federico Rimembrana, Mattia Pini, <span class="NLM_string-name hlFld-ContribAuthor">Marco De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">NanoModeler: A Webserver for Molecular Simulations and Engineering of Nanoparticles. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (3)
                                     , 2022-2032. <a href="https://doi.org/10.1021/acs.jctc.8b01304" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b01304</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b01304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b01304%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DNanoModeler%25253A%252BA%252BWebserver%252Bfor%252BMolecular%252BSimulations%252Band%252BEngineering%252Bof%252BNanoparticles%26aulast%3DFranco-Ulloa%26aufirst%3DSebastian%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D28122018%26date%3D25022019%26date%3D13022019%26volume%3D15%26issue%3D3%26spage%3D2022%26epage%3D2032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Leela
S. Dodda, Julian Tirado-Rives, <span class="NLM_string-name hlFld-ContribAuthor">William L. Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Unbinding Dynamics of Non-Nucleoside Inhibitors from HIV-1 Reverse Transcriptase. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2019,</strong> <em>123 </em>
                                    (8)
                                     , 1741-1748. <a href="https://doi.org/10.1021/acs.jpcb.8b10341" title="DOI URL">https://doi.org/10.1021/acs.jpcb.8b10341</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.8b10341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.8b10341%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DUnbinding%252BDynamics%252Bof%252BNon-Nucleoside%252BInhibitors%252Bfrom%252BHIV-1%252BReverse%252BTranscriptase%26aulast%3DDodda%26aufirst%3DLeela%2BS.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D23102018%26date%3D09122018%26date%3D03012019%26date%3D20122018%26volume%3D123%26issue%3D8%26spage%3D1741%26epage%3D1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pablo
R. Arantes, Marcelo D. Polêto, Elisa B. O. John, Conrado Pedebos, Bruno I. Grisci, Marcio Dorn, <span class="NLM_string-name hlFld-ContribAuthor">Hugo Verli</span>. </span><span class="cited-content_cbyCitation_article-title">Development of GROMOS-Compatible Parameter Set for Simulations of Chalcones and Flavonoids. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2019,</strong> <em>123 </em>
                                    (5)
                                     , 994-1008. <a href="https://doi.org/10.1021/acs.jpcb.8b10139" title="DOI URL">https://doi.org/10.1021/acs.jpcb.8b10139</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.8b10139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.8b10139%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DDevelopment%252Bof%252BGROMOS-Compatible%252BParameter%252BSet%252Bfor%252BSimulations%252Bof%252BChalcones%252Band%252BFlavonoids%26aulast%3DArantes%26aufirst%3DPablo%2BR.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D2019%26date%3D17102018%26date%3D09012019%26date%3D25012019%26date%3D09012019%26volume%3D123%26issue%3D5%26spage%3D994%26epage%3D1008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Doris
A. Schuetz, Mattia Bernetti, Martina Bertazzo, Djordje Musil, Hans-Michael Eggenweiler, Maurizio Recanatini, Matteo Masetti, Gerhard F. Ecker, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Residence Time and Drug Unbinding Pathway through Scaled Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 535-549. <a href="https://doi.org/10.1021/acs.jcim.8b00614" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredicting%252BResidence%252BTime%252Band%252BDrug%252BUnbinding%252BPathway%252Bthrough%252BScaled%252BMolecular%252BDynamics%26aulast%3DSchuetz%26aufirst%3DDoris%2BA.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D11092018%26date%3D13122018%26date%3D30112018%26volume%3D59%26issue%3D1%26spage%3D535%26epage%3D549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gregor
N. Simm, Alain C. Vaucher, <span class="NLM_string-name hlFld-ContribAuthor">Markus Reiher</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of Reaction Pathways and Chemical Transformation Networks. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry A</span><span> <strong>2019,</strong> <em>123 </em>
                                    (2)
                                     , 385-399. <a href="https://doi.org/10.1021/acs.jpca.8b10007" title="DOI URL">https://doi.org/10.1021/acs.jpca.8b10007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpca.8b10007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpca.8b10007%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520A%26atitle%3DExploration%252Bof%252BReaction%252BPathways%252Band%252BChemical%252BTransformation%252BNetworks%26aulast%3DSimm%26aufirst%3DGregor%2BN.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D13102018%26date%3D08112018%26date%3D09122018%26date%3D13112018%26volume%3D123%26issue%3D2%26spage%3D385%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Geetmani
Singh Nongthombam, Kasmika Borah, Thingreila Muinao, Yumnam Silla, Mintu Pal, Hari Prasanna Deka Boruah, <span class="NLM_string-name hlFld-ContribAuthor">Romesh Chandra Boruah</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of D-Ring Annulated Pyridosteroids from β-Formyl Enamides and Their Biological Evaluations. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2019,</strong> <em>21 </em>
                                    (1)
                                     , 11-27. <a href="https://doi.org/10.1021/acscombsci.8b00140" title="DOI URL">https://doi.org/10.1021/acscombsci.8b00140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.8b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.8b00140%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DSynthesis%252Bof%252BD-Ring%252BAnnulated%252BPyridosteroids%252Bfrom%252B%2525CE%2525B2-Formyl%252BEnamides%252Band%252BTheir%252BBiological%252BEvaluations%26aulast%3DNongthombam%26aufirst%3DGeetmani%2BSingh%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17092018%26date%3D19122018%26date%3D02012019%26date%3D21122018%26volume%3D21%26issue%3D1%26spage%3D11%26epage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Toru Ekimoto, Tsutomu Yamane, <span class="NLM_string-name hlFld-ContribAuthor">Mitsunori Ikeguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Elimination of Finite-Size Effects on Binding Free Energies via the Warp-Drive Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (12)
                                     , 6544-6559. <a href="https://doi.org/10.1021/acs.jctc.8b00280" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00280%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DElimination%252Bof%252BFinite-Size%252BEffects%252Bon%252BBinding%252BFree%252BEnergies%252Bvia%252Bthe%252BWarp-Drive%252BMethod%26aulast%3DEkimoto%26aufirst%3DToru%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D20032018%26date%3D21112018%26date%3D07112018%26volume%3D14%26issue%3D12%26spage%3D6544%26epage%3D6559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vito Genna, Elisa Donati, <span class="NLM_string-name hlFld-ContribAuthor">Marco De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">The Catalytic Mechanism of DNA and RNA Polymerases. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2018,</strong> <em>8 </em>
                                    (12)
                                     , 11103-11118. <a href="https://doi.org/10.1021/acscatal.8b03363" title="DOI URL">https://doi.org/10.1021/acscatal.8b03363</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.8b03363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.8b03363%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DThe%252BCatalytic%252BMechanism%252Bof%252BDNA%252Band%252BRNA%252BPolymerases%26aulast%3DGenna%26aufirst%3DVito%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22082018%26date%3D10102018%26date%3D30102018%26date%3D18102018%26volume%3D8%26issue%3D12%26spage%3D11103%26epage%3D11118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mattia Bernetti, Elena Rosini, Luca Mollica, Matteo Masetti, Loredano Pollegioni, Maurizio Recanatini, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Residence Time through Scaled Molecular Dynamics: A Prospective Application to hDAAO Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (11)
                                     , 2255-2265. <a href="https://doi.org/10.1021/acs.jcim.8b00518" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00518</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00518%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBinding%252BResidence%252BTime%252Bthrough%252BScaled%252BMolecular%252BDynamics%25253A%252BA%252BProspective%252BApplication%252Bto%252BhDAAO%252BInhibitors%26aulast%3DBernetti%26aufirst%3DMattia%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01082018%26date%3D05112018%26date%3D19102018%26volume%3D58%26issue%3D11%26spage%3D2255%26epage%3D2265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ajay Abisheck Paul George, Pascal Heimer, Astrid Maaß, Jan Hamaekers, Martin Hofmann-Apitius, Arijit Biswas, <span class="NLM_string-name hlFld-ContribAuthor">Diana Imhof</span>. </span><span class="cited-content_cbyCitation_article-title">Insights into the Folding of Disulfide-Rich μ-Conotoxins. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (10)
                                     , 12330-12340. <a href="https://doi.org/10.1021/acsomega.8b01465" title="DOI URL">https://doi.org/10.1021/acsomega.8b01465</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b01465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b01465%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DInsights%252Binto%252Bthe%252BFolding%252Bof%252BDisulfide-Rich%252B%2525CE%2525BC-Conotoxins%26aulast%3DPaul%2BGeorge%26aufirst%3DAjay%2BAbisheck%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D27062018%26date%3D12092018%26volume%3D3%26issue%3D10%26spage%3D12330%26epage%3D12340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Benjamin
R. Jagger, Christopher T. Lee, <span class="NLM_string-name hlFld-ContribAuthor">Rommie E. Amaro</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative Ranking of Ligand Binding Kinetics with a Multiscale Milestoning Simulation Approach. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (17)
                                     , 4941-4948. <a href="https://doi.org/10.1021/acs.jpclett.8b02047" title="DOI URL">https://doi.org/10.1021/acs.jpclett.8b02047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.8b02047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.8b02047%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DQuantitative%252BRanking%252Bof%252BLigand%252BBinding%252BKinetics%252Bwith%252Ba%252BMultiscale%252BMilestoning%252BSimulation%252BApproach%26aulast%3DJagger%26aufirst%3DBenjamin%2BR.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30062018%26date%3D02082018%26date%3D16082018%26date%3D02082018%26volume%3D9%26issue%3D17%26spage%3D4941%26epage%3D4948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daria B. Kokh, Marta Amaral, Joerg Bomke, Ulrich Grädler, Djordje Musil, Hans-Peter Buchstaller, Matthias K. Dreyer, Matthias Frech, Maryse Lowinski, Francois Vallee, Marc Bianciotto, Alexey Rak, <span class="NLM_string-name hlFld-ContribAuthor">Rebecca C. Wade</span>. </span><span class="cited-content_cbyCitation_article-title">Estimation of Drug-Target Residence Times by τ-Random Acceleration Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (7)
                                     , 3859-3869. <a href="https://doi.org/10.1021/acs.jctc.8b00230" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00230%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DEstimation%252Bof%252BDrug-Target%252BResidence%252BTimes%252Bby%252B%2525CF%252584-Random%252BAcceleration%252BMolecular%252BDynamics%252BSimulations%26aulast%3DKokh%26aufirst%3DDaria%2BB.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01032018%26date%3D04062018%26date%3D16052018%26volume%3D14%26issue%3D7%26spage%3D3859%26epage%3D3869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">J. Rydzewski, R. Jakubowski, W. Nowak, <span class="NLM_string-name hlFld-ContribAuthor">H. Grubmüller</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetics of Huperzine A Dissociation from Acetylcholinesterase via Multiple Unbinding Pathways. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (6)
                                     , 2843-2851. <a href="https://doi.org/10.1021/acs.jctc.8b00173" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00173</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00173%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DKinetics%252Bof%252BHuperzine%252BA%252BDissociation%252Bfrom%252BAcetylcholinesterase%252Bvia%252BMultiple%252BUnbinding%252BPathways%26aulast%3DRydzewski%26aufirst%3DJ.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16022018%26date%3D24052018%26date%3D01052018%26volume%3D14%26issue%3D6%26spage%3D2843%26epage%3D2851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wanli You, <span class="NLM_string-name hlFld-ContribAuthor">Chia-en A. Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Molecular Interactions and Protein Rearrangement in the Dissociation Kinetics of p38α MAP Kinase Type-I/II/III Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 968-981. <a href="https://doi.org/10.1021/acs.jcim.7b00640" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00640</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00640%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRole%252Bof%252BMolecular%252BInteractions%252Band%252BProtein%252BRearrangement%252Bin%252Bthe%252BDissociation%252BKinetics%252Bof%252Bp38%2525CE%2525B1%252BMAP%252BKinase%252BType-I%25252FII%25252FIII%252BInhibitors%26aulast%3DYou%26aufirst%3DWanli%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D01112017%26date%3D16042018%26date%3D05042018%26volume%3D58%26issue%3D5%26spage%3D968%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Doris A. Schuetz, Lars Richter, Marta Amaral, Melanie Grandits, Ulrich Grädler, Djordje Musil, Hans-Peter Buchstaller, Hans-Michael Eggenweiler, Matthias Frech, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard F. Ecker</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand Desolvation Steers On-Rate and Impacts Drug Residence Time of Heat Shock Protein 90 (Hsp90) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (10)
                                     , 4397-4411. <a href="https://doi.org/10.1021/acs.jmedchem.8b00080" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00080</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00080%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLigand%252BDesolvation%252BSteers%252BOn-Rate%252Band%252BImpacts%252BDrug%252BResidence%252BTime%252Bof%252BHeat%252BShock%252BProtein%252B90%252B%252528Hsp90%252529%252BInhibitors%26aulast%3DSchuetz%26aufirst%3DDoris%2BA.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17012018%26date%3D10052018%26date%3D27042018%26volume%3D61%26issue%3D10%26spage%3D4397%26epage%3D4411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Weiwei Xue, Fengyuan Yang, Panpan Wang, Guoxun Zheng, Yuzong Chen, Xiaojun Yao, <span class="NLM_string-name hlFld-ContribAuthor">Feng Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">What Contributes to Serotonin–Norepinephrine Reuptake Inhibitors’ Dual-Targeting Mechanism? The Key Role of Transmembrane Domain 6 in Human Serotonin and Norepinephrine Transporters Revealed by Molecular Dynamics Simulation. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2018,</strong> <em>9 </em>
                                    (5)
                                     , 1128-1140. <a href="https://doi.org/10.1021/acschemneuro.7b00490" title="DOI URL">https://doi.org/10.1021/acschemneuro.7b00490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.7b00490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.7b00490%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DWhat%252BContributes%252Bto%252BSerotonin%2525E2%252580%252593Norepinephrine%252BReuptake%252BInhibitors%2525E2%252580%252599%252BDual-Targeting%252BMechanism%25253F%252BThe%252BKey%252BRole%252Bof%252BTransmembrane%252BDomain%252B6%252Bin%252BHuman%252BSerotonin%252Band%252BNorepinephrine%252BTransporters%252BRevealed%252Bby%252BMolecular%252BDynamics%252BSimulation%26aulast%3DXue%26aufirst%3DWeiwei%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D07122017%26date%3D04012018%26date%3D24012018%26date%3D04012018%26volume%3D9%26issue%3D5%26spage%3D1128%26epage%3D1140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amirhossein Mafi, Srinivas Abbina, Manu Thomas Kalathottukaren, James H. Morrissey, Charles Haynes, Jayachandran N. Kizhakkedathu, Jim Pfaendtner, <span class="NLM_string-name hlFld-ContribAuthor">Keng C. Chou</span>. </span><span class="cited-content_cbyCitation_article-title">Design of Polyphosphate Inhibitors: A Molecular Dynamics Investigation on Polyethylene Glycol-Linked Cationic Binding Groups. </span><span class="cited-content_cbyCitation_journal-name">Biomacromolecules</span><span> <strong>2018,</strong> <em>19 </em>
                                    (4)
                                     , 1358-1367. <a href="https://doi.org/10.1021/acs.biomac.8b00327" title="DOI URL">https://doi.org/10.1021/acs.biomac.8b00327</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biomac.8b00327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biomac.8b00327%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomacromolecules%26atitle%3DDesign%252Bof%252BPolyphosphate%252BInhibitors%25253A%252BA%252BMolecular%252BDynamics%252BInvestigation%252Bon%252BPolyethylene%252BGlycol-Linked%252BCationic%252BBinding%252BGroups%26aulast%3DMafi%26aufirst%3DAmirhossein%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D23022018%26date%3D23032018%26date%3D14032018%26volume%3D19%26issue%3D4%26spage%3D1358%26epage%3D1367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joana Magalhães, Giannamaria Annunziato, Nina Franko, Marco Pieroni, Barbara Campanini, Agostino Bruno, <span class="NLM_string-name hlFld-ContribAuthor">Gabriele Costantino</span>. </span><span class="cited-content_cbyCitation_article-title">Integration of Enhanced Sampling Methods with Saturation Transfer Difference Experiments to Identify Protein Druggable Pockets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (3)
                                     , 710-723. <a href="https://doi.org/10.1021/acs.jcim.7b00733" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00733</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00733%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIntegration%252Bof%252BEnhanced%252BSampling%252BMethods%252Bwith%252BSaturation%252BTransfer%252BDifference%252BExperiments%252Bto%252BIdentify%252BProtein%252BDruggable%252BPockets%26aulast%3DMagalh%25C3%25A3es%26aufirst%3DJoana%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D21122017%26date%3D06032018%26date%3D26022018%26volume%3D58%26issue%3D3%26spage%3D710%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrea Spitaleri, Sergio Decherchi, Andrea Cavalli, <span class="NLM_string-name hlFld-ContribAuthor">Walter Rocchia</span>. </span><span class="cited-content_cbyCitation_article-title">Fast Dynamic Docking Guided by Adaptive Electrostatic Bias: The MD-Binding Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (3)
                                     , 1727-1736. <a href="https://doi.org/10.1021/acs.jctc.7b01088" title="DOI URL">https://doi.org/10.1021/acs.jctc.7b01088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.7b01088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.7b01088%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DFast%252BDynamic%252BDocking%252BGuided%252Bby%252BAdaptive%252BElectrostatic%252BBias%25253A%252BThe%252BMD-Binding%252BApproach%26aulast%3DSpitaleri%26aufirst%3DAndrea%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D28102017%26date%3D09022018%26date%3D19012018%26volume%3D14%26issue%3D3%26spage%3D1727%26epage%3D1736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Bertazzo</span>, <span class="hlFld-ContribAuthor ">Mattia  Bernetti</span>, <span class="hlFld-ContribAuthor ">Maurizio  Recanatini</span>, <span class="hlFld-ContribAuthor ">Matteo  Masetti</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">Fully Flexible Docking via Reaction-Coordinate-Independent Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (2)
                                     , 490-500. <a href="https://doi.org/10.1021/acs.jcim.7b00674" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00674%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFully%252BFlexible%252BDocking%252Bvia%252BReaction-Coordinate-Independent%252BMolecular%252BDynamics%252BSimulations%26aulast%3DBertazzo%26aufirst%3DMartina%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D12122017%26date%3D09022018%26date%3D26022018%26date%3D29012018%26volume%3D58%26issue%3D2%26spage%3D490%26epage%3D500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sergio Decherchi, Giovanni Bottegoni, Andrea Spitaleri, Walter Rocchia, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">BiKi Life Sciences: A New Suite for Molecular Dynamics and Related Methods in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (2)
                                     , 219-224. <a href="https://doi.org/10.1021/acs.jcim.7b00680" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00680</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00680%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBiKi%252BLife%252BSciences%25253A%252BA%252BNew%252BSuite%252Bfor%252BMolecular%252BDynamics%252Band%252BRelated%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DDecherchi%26aufirst%3DSergio%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D28112017%26date%3D31012018%26date%3D17012018%26volume%3D58%26issue%3D2%26spage%3D219%26epage%3D224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  Prati</span>, <span class="hlFld-ContribAuthor ">Giovanni  Bottegoni</span>, <span class="hlFld-ContribAuthor ">Maria Laura  Bolognesi</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">BACE-1 Inhibitors: From Recent Single-Target Molecules to Multitarget Compounds for Alzheimer’s Disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 619-637. <a href="https://doi.org/10.1021/acs.jmedchem.7b00393" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00393</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00393%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBACE-1%252BInhibitors%25253A%252BFrom%252BRecent%252BSingle-Target%252BMolecules%252Bto%252BMultitarget%252BCompounds%252Bfor%252BAlzheimer%2525E2%252580%252599s%252BDisease%26aulast%3DPrati%26aufirst%3DFederica%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D13032017%26date%3D08082017%26date%3D08022018%26date%3D27072017%26volume%3D61%26issue%3D3%26spage%3D619%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Billy J.  Williams-Noonan</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Yuriev</span>, and <span class="hlFld-ContribAuthor ">David K.  Chalmers</span>  . </span><span class="cited-content_cbyCitation_article-title">Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 638-649. <a href="https://doi.org/10.1021/acs.jmedchem.7b00681" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00681%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFree%252BEnergy%252BMethods%252Bin%252BDrug%252BDesign%25253A%252BProspects%252Bof%252B%2525E2%252580%25259CAlchemical%252BPerturbation%2525E2%252580%25259D%252Bin%252BMedicinal%252BChemistry%26aulast%3DWilliams-Noonan%26aufirst%3DBilly%2BJ.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D07052017%26date%3D04082017%26date%3D08022018%26date%3D26072017%26volume%3D61%26issue%3D3%26spage%3D638%26epage%3D649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jose Antonio  Ortega</span>, <span class="hlFld-ContribAuthor ">Laura  Riccardi</span>, <span class="hlFld-ContribAuthor ">Elirosa  Minniti</span>, <span class="hlFld-ContribAuthor ">Marco  Borgogno</span>, <span class="hlFld-ContribAuthor ">Jose M.  Arencibia</span>, <span class="hlFld-ContribAuthor ">Maria L.  Greco</span>, <span class="hlFld-ContribAuthor ">Anna  Minarini</span>, <span class="hlFld-ContribAuthor ">Claudia  Sissi</span>, and <span class="hlFld-ContribAuthor ">Marco  De Vivo</span>  . </span><span class="cited-content_cbyCitation_article-title">Pharmacophore Hybridization To Discover Novel Topoisomerase II Poisons with Promising Antiproliferative Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 1375-1379. <a href="https://doi.org/10.1021/acs.jmedchem.7b01388" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01388</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01388%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPharmacophore%252BHybridization%252BTo%252BDiscover%252BNovel%252BTopoisomerase%252BII%252BPoisons%252Bwith%252BPromising%252BAntiproliferative%252BActivity%26aulast%3DOrtega%26aufirst%3DJose%2BAntonio%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D19092017%26date%3D09112017%26date%3D08022018%26date%3D27102017%26volume%3D61%26issue%3D3%26spage%3D1375%26epage%3D1379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jigneshkumar Dahyabhai  Prajapati</span>, <span class="hlFld-ContribAuthor ">Carlos José Fernández  Solano</span>, <span class="hlFld-ContribAuthor ">Mathias  Winterhalter</span>, and <span class="hlFld-ContribAuthor ">Ulrich  Kleinekathöfer</span>  . </span><span class="cited-content_cbyCitation_article-title">Enrofloxacin Permeation Pathways across the Porin OmpC. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2018,</strong> <em>122 </em>
                                    (4)
                                     , 1417-1426. <a href="https://doi.org/10.1021/acs.jpcb.7b12568" title="DOI URL">https://doi.org/10.1021/acs.jpcb.7b12568</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.7b12568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.7b12568%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D4553%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DEnrofloxacin%252BPermeation%252BPathways%252Bacross%252Bthe%252BPorin%252BOmpC%26aulast%3DPrajapati%26aufirst%3DJigneshkumar%2BDahyabhai%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2018%26date%3D21122017%26date%3D05012018%26date%3D22012018%26date%3D01022018%26date%3D06012018%26volume%3D122%26issue%3D4%26spage%3D1417%26epage%3D1426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zoe  Cournia</span>, <span class="hlFld-ContribAuthor ">Bryce  Allen</span>, and <span class="hlFld-ContribAuthor ">Woody  Sherman</span>  . </span><span class="cited-content_cbyCitation_article-title">Relative Binding Free Energy Calculations in Drug Discovery: Recent Advances and Practical Considerations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 2911-2937. <a href="https://doi.org/10.1021/acs.jcim.7b00564" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00564</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00564%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRelative%252BBinding%252BFree%252BEnergy%252BCalculations%252Bin%252BDrug%252BDiscovery%25253A%252BRecent%252BAdvances%252Band%252BPractical%252BConsiderations%26aulast%3DCournia%26aufirst%3DZoe%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D19092017%26date%3D15122017%26date%3D26122017%26volume%3D57%26issue%3D12%26spage%3D2911%26epage%3D2937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edoardo  Giovannelli</span>, <span class="hlFld-ContribAuthor ">Piero  Procacci</span>, <span class="hlFld-ContribAuthor ">Gianni  Cardini</span>, <span class="hlFld-ContribAuthor ">Marco  Pagliai</span>, <span class="hlFld-ContribAuthor ">Victor  Volkov</span>, and <span class="hlFld-ContribAuthor ">Riccardo  Chelli</span>  . </span><span class="cited-content_cbyCitation_article-title">Binding Free Energies of Host–Guest Systems by Nonequilibrium Alchemical Simulations with Constrained Dynamics: Theoretical Framework. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2017,</strong> <em>13 </em>
                                    (12)
                                     , 5874-5886. <a href="https://doi.org/10.1021/acs.jctc.7b00594" title="DOI URL">https://doi.org/10.1021/acs.jctc.7b00594</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.7b00594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.7b00594%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DBinding%252BFree%252BEnergies%252Bof%252BHost%2525E2%252580%252593Guest%252BSystems%252Bby%252BNonequilibrium%252BAlchemical%252BSimulations%252Bwith%252BConstrained%252BDynamics%25253A%252BTheoretical%252BFramework%26aulast%3DGiovannelli%26aufirst%3DEdoardo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D07062017%26date%3D30112017%26date%3D12122017%26date%3D09102017%26volume%3D13%26issue%3D12%26spage%3D5874%26epage%3D5886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas  Tarenzi</span>, <span class="hlFld-ContribAuthor ">Vania  Calandrini</span>, <span class="hlFld-ContribAuthor ">Raffaello  Potestio</span>, <span class="hlFld-ContribAuthor ">Alejandro  Giorgetti</span>, and <span class="hlFld-ContribAuthor ">Paolo  Carloni</span>  . </span><span class="cited-content_cbyCitation_article-title">Open Boundary Simulations of Proteins and Their Hydration Shells by Hamiltonian Adaptive Resolution Scheme. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2017,</strong> <em>13 </em>
                                    (11)
                                     , 5647-5657. <a href="https://doi.org/10.1021/acs.jctc.7b00508" title="DOI URL">https://doi.org/10.1021/acs.jctc.7b00508</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.7b00508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.7b00508%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DOpen%252BBoundary%252BSimulations%252Bof%252BProteins%252Band%252BTheir%252BHydration%252BShells%252Bby%252BHamiltonian%252BAdaptive%252BResolution%252BScheme%26aulast%3DTarenzi%26aufirst%3DThomas%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D16052017%26date%3D27102017%26date%3D14112017%26date%3D09102017%26volume%3D13%26issue%3D11%26spage%3D5647%26epage%3D5657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mattia  Bernetti</span>, <span class="hlFld-ContribAuthor ">Matteo  Masetti</span>, <span class="hlFld-ContribAuthor ">Fabio  Pietrucci</span>, <span class="hlFld-ContribAuthor ">Martin  Blackledge</span>, <span class="hlFld-ContribAuthor ">Malene Ringkjobing  Jensen</span>, <span class="hlFld-ContribAuthor ">Maurizio  Recanatini</span>, <span class="hlFld-ContribAuthor ">Luca  Mollica</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">Structural and Kinetic Characterization of the Intrinsically Disordered Protein SeV NTAIL through Enhanced Sampling Simulations. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2017,</strong> <em>121 </em>
                                    (41)
                                     , 9572-9582. <a href="https://doi.org/10.1021/acs.jpcb.7b08925" title="DOI URL">https://doi.org/10.1021/acs.jpcb.7b08925</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.7b08925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.7b08925%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DStructural%252Band%252BKinetic%252BCharacterization%252Bof%252Bthe%252BIntrinsically%252BDisordered%252BProtein%252BSeV%252BNTAIL%252Bthrough%252BEnhanced%252BSampling%252BSimulations%26aulast%3DBernetti%26aufirst%3DMattia%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D07092017%26date%3D18092017%26date%3D04102017%26date%3D19102017%26date%3D19092017%26volume%3D121%26issue%3D41%26spage%3D9572%26epage%3D9582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anders S.  Larsen</span>, <span class="hlFld-ContribAuthor ">Michael T.  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Kristoffer E.  Johansson</span>, <span class="hlFld-ContribAuthor ">J. Axel  Zeitler</span>, and <span class="hlFld-ContribAuthor ">Jukka  Rantanen</span>  . </span><span class="cited-content_cbyCitation_article-title">Tracking Dehydration Mechanisms in Crystalline Hydrates with Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Crystal Growth & Design</span><span> <strong>2017,</strong> <em>17 </em>
                                    (10)
                                     , 5017-5022. <a href="https://doi.org/10.1021/acs.cgd.7b00889" title="DOI URL">https://doi.org/10.1021/acs.cgd.7b00889</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.cgd.7b00889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.cgd.7b00889%26sid%3Dliteratum%253Aachs%26jtitle%3DCrystal%2520Growth%2520%2526%2520Design%26atitle%3DTracking%252BDehydration%252BMechanisms%252Bin%252BCrystalline%252BHydrates%252Bwith%252BMolecular%252BDynamics%252BSimulations%26aulast%3DLarsen%26aufirst%3DAnders%2BS.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D24062017%26date%3D14082017%26date%3D31082017%26date%3D04102017%26date%3D24082017%26volume%3D17%26issue%3D10%26spage%3D5017%26epage%3D5022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppina  La Sala</span>, <span class="hlFld-ContribAuthor ">Sergio  Decherchi</span>, <span class="hlFld-ContribAuthor ">Marco  De Vivo</span>, and <span class="hlFld-ContribAuthor ">Walter  Rocchia</span>  . </span><span class="cited-content_cbyCitation_article-title">Allosteric Communication Networks in Proteins Revealed through Pocket Crosstalk Analysis. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2017,</strong> <em>3 </em>
                                    (9)
                                     , 949-960. <a href="https://doi.org/10.1021/acscentsci.7b00211" title="DOI URL">https://doi.org/10.1021/acscentsci.7b00211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.7b00211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.7b00211%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DAllosteric%252BCommunication%252BNetworks%252Bin%252BProteins%252BRevealed%252Bthrough%252BPocket%252BCrosstalk%252BAnalysis%26aulast%3DLa%2BSala%26aufirst%3DGiuseppina%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D18052017%26date%3D10082017%26date%3D27092017%26volume%3D3%26issue%3D9%26spage%3D949%26epage%3D960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Doerr</span>, <span class="hlFld-ContribAuthor ">Toni  Giorgino</span>, <span class="hlFld-ContribAuthor ">Gerard  Martínez-Rosell</span>, <span class="hlFld-ContribAuthor ">João M.  Damas</span>, and <span class="hlFld-ContribAuthor ">Gianni  De Fabritiis</span>  . </span><span class="cited-content_cbyCitation_article-title">High-Throughput Automated Preparation and Simulation of Membrane Proteins with HTMD. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2017,</strong> <em>13 </em>
                                    (9)
                                     , 4003-4011. <a href="https://doi.org/10.1021/acs.jctc.7b00480" title="DOI URL">https://doi.org/10.1021/acs.jctc.7b00480</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.7b00480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.7b00480%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DHigh-Throughput%252BAutomated%252BPreparation%252Band%252BSimulation%252Bof%252BMembrane%252BProteins%252Bwith%252BHTMD%26aulast%3DDoerr%26aufirst%3DStefan%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D09052017%26date%3D04082017%26date%3D12092017%26date%3D19072017%26volume%3D13%26issue%3D9%26spage%3D4003%26epage%3D4011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul C. D.  Hawkins</span>  . </span><span class="cited-content_cbyCitation_article-title">Conformation Generation: The State of the Art. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (8)
                                     , 1747-1756. <a href="https://doi.org/10.1021/acs.jcim.7b00221" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00221%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DConformation%252BGeneration%25253A%252BThe%252BState%252Bof%252Bthe%252BArt%26aulast%3DHawkins%26aufirst%3DPaul%2BC.%2BD.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D19042017%26date%3D31072017%26date%3D28082017%26date%3D06072017%26volume%3D57%26issue%3D8%26spage%3D1747%26epage%3D1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefano  Motta</span> and <span class="hlFld-ContribAuthor ">Laura  Bonati</span>  . </span><span class="cited-content_cbyCitation_article-title">Modeling Binding with Large Conformational Changes: Key Points in Ensemble-Docking Approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (7)
                                     , 1563-1578. <a href="https://doi.org/10.1021/acs.jcim.7b00125" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00125</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00125%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DModeling%252BBinding%252Bwith%252BLarge%252BConformational%252BChanges%25253A%252BKey%252BPoints%252Bin%252BEnsemble-Docking%252BApproaches%26aulast%3DMotta%26aufirst%3DStefano%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D28022017%26date%3D30062017%26date%3D24072017%26date%3D15062017%26volume%3D57%26issue%3D7%26spage%3D1563%26epage%3D1578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Abel</span>, <span class="hlFld-ContribAuthor ">Lingle  Wang</span>, <span class="hlFld-ContribAuthor ">Edward D.  Harder</span>, <span class="hlFld-ContribAuthor ">B. J.  Berne</span>, and <span class="hlFld-ContribAuthor ">Richard A.  Friesner</span>  . </span><span class="cited-content_cbyCitation_article-title">Advancing Drug Discovery through Enhanced Free Energy Calculations. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2017,</strong> <em>50 </em>
                                    (7)
                                     , 1625-1632. <a href="https://doi.org/10.1021/acs.accounts.7b00083" title="DOI URL">https://doi.org/10.1021/acs.accounts.7b00083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.7b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.7b00083%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DAdvancing%252BDrug%252BDiscovery%252Bthrough%252BEnhanced%252BFree%252BEnergy%252BCalculations%26aulast%3DAbel%26aufirst%3DRobert%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D10022017%26date%3D05072017%26date%3D18072017%26volume%3D50%26issue%3D7%26spage%3D1625%26epage%3D1632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tai-Sung  Lee</span>, <span class="hlFld-ContribAuthor ">Yuan  Hu</span>, <span class="hlFld-ContribAuthor ">Brad  Sherborne</span>, <span class="hlFld-ContribAuthor ">Zhuyan  Guo</span>, and <span class="hlFld-ContribAuthor ">Darrin M.  York</span>  . </span><span class="cited-content_cbyCitation_article-title">Toward Fast and Accurate Binding Affinity Prediction with pmemdGTI: An Efficient Implementation of GPU-Accelerated Thermodynamic Integration. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2017,</strong> <em>13 </em>
                                    (7)
                                     , 3077-3084. <a href="https://doi.org/10.1021/acs.jctc.7b00102" title="DOI URL">https://doi.org/10.1021/acs.jctc.7b00102</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.7b00102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.7b00102%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DToward%252BFast%252Band%252BAccurate%252BBinding%252BAffinity%252BPrediction%252Bwith%252BpmemdGTI%25253A%252BAn%252BEfficient%252BImplementation%252Bof%252BGPU-Accelerated%252BThermodynamic%252BIntegration%26aulast%3DLee%26aufirst%3DTai-Sung%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D31012017%26date%3D23062017%26date%3D11072017%26date%3D15062017%26volume%3D13%26issue%3D7%26spage%3D3077%26epage%3D3084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessio  Lodola</span>, <span class="hlFld-ContribAuthor ">Serena  Bertolini</span>, <span class="hlFld-ContribAuthor ">Matteo  Biagetti</span>, <span class="hlFld-ContribAuthor ">Silvia  Capacchi</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Facchinetti</span>, <span class="hlFld-ContribAuthor ">Paola Maria  Gallo</span>, <span class="hlFld-ContribAuthor ">Alice  Pappani</span>, <span class="hlFld-ContribAuthor ">Marco  Mor</span>, <span class="hlFld-ContribAuthor ">Daniele  Pala</span>, <span class="hlFld-ContribAuthor ">Silvia  Rivara</span>, <span class="hlFld-ContribAuthor ">Filippo  Visentini</span>, <span class="hlFld-ContribAuthor ">Mauro  Corsi</span>, and <span class="hlFld-ContribAuthor ">Anna Maria  Capelli</span>  . </span><span class="cited-content_cbyCitation_article-title">Atropisomerism and Conformational Equilibria: Impact on PI3Kδ Inhibition of 2-((6-Amino-9H-purin-9-yl)methyl)-5-methyl-3-(o-tolyl)quinazolin-4(3H)-one (IC87114) and Its Conformationally Restricted Analogs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (10)
                                     , 4304-4315. <a href="https://doi.org/10.1021/acs.jmedchem.7b00247" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00247</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00247%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAtropisomerism%252Band%252BConformational%252BEquilibria%25253A%252BImpact%252Bon%252BPI3K%2525CE%2525B4%252BInhibition%252Bof%252B2-%252528%2525286-Amino-9H-purin-9-yl%252529methyl%252529-5-methyl-3-%252528o-tolyl%252529quinazolin-4%2525283H%252529-one%252B%252528IC87114%252529%252Band%252BIts%252BConformationally%252BRestricted%252BAnalogs%26aulast%3DLodola%26aufirst%3DAlessio%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D14022017%26date%3D15052017%26date%3D25052017%26date%3D10052017%26volume%3D60%26issue%3D10%26spage%3D4304%26epage%3D4315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haleh  Abdizadeh</span>, <span class="hlFld-ContribAuthor ">Ali Rana  Atilgan</span>, and <span class="hlFld-ContribAuthor ">Canan  Atilgan</span>  . </span><span class="cited-content_cbyCitation_article-title">Mechanisms by Which Salt Concentration Moderates the Dynamics of Human Serum Transferrin. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2017,</strong> <em>121 </em>
                                    (18)
                                     , 4778-4789. <a href="https://doi.org/10.1021/acs.jpcb.7b02380" title="DOI URL">https://doi.org/10.1021/acs.jpcb.7b02380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.7b02380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.7b02380%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DMechanisms%252Bby%252BWhich%252BSalt%252BConcentration%252BModerates%252Bthe%252BDynamics%252Bof%252BHuman%252BSerum%252BTransferrin%26aulast%3DAbdizadeh%26aufirst%3DHaleh%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D13032017%26date%3D26042017%26date%3D02052017%26date%3D11052017%26date%3D26042017%26volume%3D121%26issue%3D18%26spage%3D4778%26epage%3D4789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Henrik  Keränen</span>, <span class="hlFld-ContribAuthor ">Laura  Pérez-Benito</span>, <span class="hlFld-ContribAuthor ">Myriam  Ciordia</span>, <span class="hlFld-ContribAuthor ">Francisca  Delgado</span>, <span class="hlFld-ContribAuthor ">Thomas B.  Steinbrecher</span>, <span class="hlFld-ContribAuthor ">Daniel  Oehlrich</span>, <span class="hlFld-ContribAuthor ">Herman W. T.  van Vlijmen</span>, <span class="hlFld-ContribAuthor ">Andrés A.  Trabanco</span>, and <span class="hlFld-ContribAuthor ">Gary  Tresadern</span>  . </span><span class="cited-content_cbyCitation_article-title">Acylguanidine Beta Secretase 1 Inhibitors: A Combined Experimental and Free Energy Perturbation Study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2017,</strong> <em>13 </em>
                                    (3)
                                     , 1439-1453. <a href="https://doi.org/10.1021/acs.jctc.6b01141" title="DOI URL">https://doi.org/10.1021/acs.jctc.6b01141</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.6b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.6b01141%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DAcylguanidine%252BBeta%252BSecretase%252B1%252BInhibitors%25253A%252BA%252BCombined%252BExperimental%252Band%252BFree%252BEnergy%252BPerturbation%252BStudy%26aulast%3DKer%25C3%25A4nen%26aufirst%3DHenrik%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D22112016%26date%3D03022017%26date%3D14032017%26date%3D19012017%26volume%3D13%26issue%3D3%26spage%3D1439%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Donatella  Callegari</span>, <span class="hlFld-ContribAuthor ">Alessio  Lodola</span>, <span class="hlFld-ContribAuthor ">Daniele  Pala</span>, <span class="hlFld-ContribAuthor ">Silvia  Rivara</span>, <span class="hlFld-ContribAuthor ">Marco  Mor</span>, <span class="hlFld-ContribAuthor ">Andrea  Rizzi</span>, and <span class="hlFld-ContribAuthor ">Anna Maria  Capelli</span>  . </span><span class="cited-content_cbyCitation_article-title">Metadynamics Simulations Distinguish Short- and Long-Residence-Time Inhibitors of Cyclin-Dependent Kinase 8. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (2)
                                     , 159-169. <a href="https://doi.org/10.1021/acs.jcim.6b00679" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00679</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00679%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMetadynamics%252BSimulations%252BDistinguish%252BShort-%252Band%252BLong-Residence-Time%252BInhibitors%252Bof%252BCyclin-Dependent%252BKinase%252B8%26aulast%3DCallegari%26aufirst%3DDonatella%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D04112016%26date%3D12012017%26date%3D27022017%26volume%3D57%26issue%3D2%26spage%3D159%26epage%3D169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elvira  Tarasova</span>, <span class="hlFld-ContribAuthor ">Vladimir  Farafonov</span>, <span class="hlFld-ContribAuthor ">Reza  Khayat</span>, <span class="hlFld-ContribAuthor ">Noriaki  Okimoto</span>, <span class="hlFld-ContribAuthor ">Teruhisa S.  Komatsu</span>, <span class="hlFld-ContribAuthor ">Makoto  Taiji</span>, and <span class="hlFld-ContribAuthor ">Dmitry  Nerukh</span>  . </span><span class="cited-content_cbyCitation_article-title">All-Atom Molecular Dynamics Simulations of Entire Virus Capsid Reveal the Role of Ion Distribution in Capsid’s Stability. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (4)
                                     , 779-784. <a href="https://doi.org/10.1021/acs.jpclett.6b02759" title="DOI URL">https://doi.org/10.1021/acs.jpclett.6b02759</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.6b02759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.6b02759%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DAll-Atom%252BMolecular%252BDynamics%252BSimulations%252Bof%252BEntire%252BVirus%252BCapsid%252BReveal%252Bthe%252BRole%252Bof%252BIon%252BDistribution%252Bin%252BCapsid%2525E2%252580%252599s%252BStability%26aulast%3DTarasova%26aufirst%3DElvira%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D24112016%26date%3D27012017%26date%3D01022017%26date%3D16022017%26date%3D27012017%26volume%3D8%26issue%3D4%26spage%3D779%26epage%3D784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matteo  Incerti</span>, <span class="hlFld-ContribAuthor ">Simonetta  Russo</span>, <span class="hlFld-ContribAuthor ">Donatella  Callegari</span>, <span class="hlFld-ContribAuthor ">Daniele  Pala</span>, <span class="hlFld-ContribAuthor ">Carmine  Giorgio</span>, <span class="hlFld-ContribAuthor ">Ilaria  Zanotti</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Barocelli</span>, <span class="hlFld-ContribAuthor ">Paola  Vicini</span>, <span class="hlFld-ContribAuthor ">Federica  Vacondio</span>, <span class="hlFld-ContribAuthor ">Silvia  Rivara</span>, <span class="hlFld-ContribAuthor ">Riccardo  Castelli</span>, <span class="hlFld-ContribAuthor ">Massimiliano  Tognolini</span>, and <span class="hlFld-ContribAuthor ">Alessio  Lodola</span>  . </span><span class="cited-content_cbyCitation_article-title">Metadynamics for Perspective Drug Design: Computationally Driven Synthesis of New Protein–Protein Interaction Inhibitors Targeting the EphA2 Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (2)
                                     , 787-796. <a href="https://doi.org/10.1021/acs.jmedchem.6b01642" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01642</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01642%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMetadynamics%252Bfor%252BPerspective%252BDrug%252BDesign%25253A%252BComputationally%252BDriven%252BSynthesis%252Bof%252BNew%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BInhibitors%252BTargeting%252Bthe%252BEphA2%252BReceptor%26aulast%3DIncerti%26aufirst%3DMatteo%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D08112016%26date%3D05012017%26date%3D26012017%26date%3D22122016%26volume%3D60%26issue%3D2%26spage%3D787%26epage%3D796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mitsugu  Araki</span>, <span class="hlFld-ContribAuthor ">Narutoshi  Kamiya</span>, <span class="hlFld-ContribAuthor ">Miwa  Sato</span>, <span class="hlFld-ContribAuthor ">Masahiko  Nakatsui</span>, <span class="hlFld-ContribAuthor ">Takatsugu  Hirokawa</span>, and <span class="hlFld-ContribAuthor ">Yasushi  Okuno</span>  . </span><span class="cited-content_cbyCitation_article-title">The Effect of Conformational Flexibility on Binding Free Energy Estimation between Kinases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (12)
                                     , 2445-2456. <a href="https://doi.org/10.1021/acs.jcim.6b00398" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00398</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00398%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BEffect%252Bof%252BConformational%252BFlexibility%252Bon%252BBinding%252BFree%252BEnergy%252BEstimation%252Bbetween%252BKinases%252Band%252BTheir%252BInhibitors%26aulast%3DAraki%26aufirst%3DMitsugu%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D12072016%26date%3D06122016%26date%3D27122016%26volume%3D56%26issue%3D12%26spage%3D2445%26epage%3D2456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Syeda Rehana  Zia</span>, <span class="hlFld-ContribAuthor ">Roberto  Gaspari</span>, <span class="hlFld-ContribAuthor ">Sergio  Decherchi</span>, and <span class="hlFld-ContribAuthor ">Walter  Rocchia</span>  . </span><span class="cited-content_cbyCitation_article-title">Probing Hydration Patterns in Class-A GPCRs via Biased MD: The A2A Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2016,</strong> <em>12 </em>
                                    (12)
                                     , 6049-6061. <a href="https://doi.org/10.1021/acs.jctc.6b00475" title="DOI URL">https://doi.org/10.1021/acs.jctc.6b00475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.6b00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.6b00475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DProbing%252BHydration%252BPatterns%252Bin%252BClass-A%252BGPCRs%252Bvia%252BBiased%252BMD%25253A%252BThe%252BA2A%252BReceptor%26aulast%3DZia%26aufirst%3DSyeda%2BRehana%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D10052016%26date%3D10112016%26date%3D13122016%26volume%3D12%26issue%3D12%26spage%3D6049%26epage%3D6061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Salomé  Llabrés</span>, <span class="hlFld-ContribAuthor ">Jordi  Juárez-Jiménez</span>, <span class="hlFld-ContribAuthor ">Matteo  Masetti</span>, <span class="hlFld-ContribAuthor ">Rosana  Leiva</span>, <span class="hlFld-ContribAuthor ">Santiago  Vázquez</span>, <span class="hlFld-ContribAuthor ">Sabrina  Gazzarrini</span>, <span class="hlFld-ContribAuthor ">Anna  Moroni</span>, <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>, and <span class="hlFld-ContribAuthor ">F. Javier  Luque</span>  . </span><span class="cited-content_cbyCitation_article-title">Mechanism of the Pseudoirreversible Binding of Amantadine to the M2 Proton Channel. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2016,</strong> <em>138 </em>
                                    (47)
                                     , 15345-15358. <a href="https://doi.org/10.1021/jacs.6b07096" title="DOI URL">https://doi.org/10.1021/jacs.6b07096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.6b07096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.6b07096%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DMechanism%252Bof%252Bthe%252BPseudoirreversible%252BBinding%252Bof%252BAmantadine%252Bto%252Bthe%252BM2%252BProton%252BChannel%26aulast%3DLlabr%25C3%25A9s%26aufirst%3DSalom%25C3%25A9%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D09072016%26date%3D16112016%26date%3D30112016%26date%3D07112016%26volume%3D138%26issue%3D47%26spage%3D15345%26epage%3D15358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppina  La Sala</span>, <span class="hlFld-ContribAuthor ">Laura  Riccardi</span>, <span class="hlFld-ContribAuthor ">Roberto  Gaspari</span>, <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>, <span class="hlFld-ContribAuthor ">Oliver  Hantschel</span>, and <span class="hlFld-ContribAuthor ">Marco  De Vivo</span>  . </span><span class="cited-content_cbyCitation_article-title">HRD Motif as the Central Hub of the Signaling Network for Activation Loop Autophosphorylation in Abl Kinase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2016,</strong> <em>12 </em>
                                    (11)
                                     , 5563-5574. <a href="https://doi.org/10.1021/acs.jctc.6b00600" title="DOI URL">https://doi.org/10.1021/acs.jctc.6b00600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.6b00600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.6b00600%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DHRD%252BMotif%252Bas%252Bthe%252BCentral%252BHub%252Bof%252Bthe%252BSignaling%252BNetwork%252Bfor%252BActivation%252BLoop%252BAutophosphorylation%252Bin%252BAbl%252BKinase%26aulast%3DLa%2BSala%26aufirst%3DGiuseppina%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D13062016%26date%3D11102016%26date%3D08112016%26date%3D28092016%26volume%3D12%26issue%3D11%26spage%3D5563%26epage%3D5574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Govindan  Subramanian</span>, <span class="hlFld-ContribAuthor ">Bharath  Ramsundar</span>, <span class="hlFld-ContribAuthor ">Vijay  Pande</span>, and <span class="hlFld-ContribAuthor ">Rajiah Aldrin  Denny</span>  . </span><span class="cited-content_cbyCitation_article-title">Computational Modeling of β-Secretase 1 (BACE-1) Inhibitors Using Ligand Based Approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (10)
                                     , 1936-1949. <a href="https://doi.org/10.1021/acs.jcim.6b00290" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00290</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00290%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComputational%252BModeling%252Bof%252B%2525CE%2525B2-Secretase%252B1%252B%252528BACE-1%252529%252BInhibitors%252BUsing%252BLigand%252BBased%252BApproaches%26aulast%3DSubramanian%26aufirst%3DGovindan%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D23052016%26date%3D10102016%26date%3D24102016%26date%3D30092016%26volume%3D56%26issue%3D10%26spage%3D1936%26epage%3D1949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Mollica</span>, <span class="hlFld-ContribAuthor ">Isabelle  Theret</span>, <span class="hlFld-ContribAuthor ">Mathias  Antoine</span>, <span class="hlFld-ContribAuthor ">Françoise  Perron-Sierra</span>, <span class="hlFld-ContribAuthor ">Yves  Charton</span>, <span class="hlFld-ContribAuthor ">Jean-Marie  Fourquez</span>, <span class="hlFld-ContribAuthor ">Michel  Wierzbicki</span>, <span class="hlFld-ContribAuthor ">Jean A.  Boutin</span>, <span class="hlFld-ContribAuthor ">Gilles  Ferry</span>, <span class="hlFld-ContribAuthor ">Sergio  Decherchi</span>, <span class="hlFld-ContribAuthor ">Giovanni  Bottegoni</span>, <span class="hlFld-ContribAuthor ">Pierre  Ducrot</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">Molecular Dynamics Simulations and Kinetic Measurements to Estimate and Predict Protein–Ligand Residence Times. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (15)
                                     , 7167-7176. <a href="https://doi.org/10.1021/acs.jmedchem.6b00632" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00632</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00632%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252BDynamics%252BSimulations%252Band%252BKinetic%252BMeasurements%252Bto%252BEstimate%252Band%252BPredict%252BProtein%2525E2%252580%252593Ligand%252BResidence%252BTimes%26aulast%3DMollica%26aufirst%3DLuca%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D25042016%26date%3D22072016%26date%3D11082016%26date%3D08072016%26volume%3D59%26issue%3D15%26spage%3D7167%26epage%3D7176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christoph  Grebner</span>, <span class="hlFld-ContribAuthor ">Jessica  Iegre</span>, <span class="hlFld-ContribAuthor ">Johan  Ulander</span>, <span class="hlFld-ContribAuthor ">Karl  Edman</span>, <span class="hlFld-ContribAuthor ">Anders  Hogner</span>, and <span class="hlFld-ContribAuthor ">Christian  Tyrchan</span>  . </span><span class="cited-content_cbyCitation_article-title">Binding Mode and Induced Fit Predictions for Prospective Computational Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (4)
                                     , 774-787. <a href="https://doi.org/10.1021/acs.jcim.5b00744" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00744</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00744%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBinding%252BMode%252Band%252BInduced%252BFit%252BPredictions%252Bfor%252BProspective%252BComputational%252BDrug%252BDesign%26aulast%3DGrebner%26aufirst%3DChristoph%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D21122015%26date%3D25032016%26date%3D25042016%26date%3D14032016%26volume%3D56%26issue%3D4%26spage%3D774%26epage%3D787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kento  Kasahara</span>, <span class="hlFld-ContribAuthor ">Suyong  Re</span>, <span class="hlFld-ContribAuthor ">Grzegorz  Nawrocki</span>, <span class="hlFld-ContribAuthor ">Hiraku  Oshima</span>, <span class="hlFld-ContribAuthor ">Chiemi  Mishima-Tsumagari</span>, <span class="hlFld-ContribAuthor ">Yukako  Miyata-Yabuki</span>, <span class="hlFld-ContribAuthor ">Mutsuko  Kukimoto-Niino</span>, <span class="hlFld-ContribAuthor ">Isseki  Yu</span>, <span class="hlFld-ContribAuthor ">Mikako  Shirouzu</span>, <span class="hlFld-ContribAuthor ">Michael  Feig</span>, <span class="hlFld-ContribAuthor ">Yuji  Sugita</span>. </span><span class="cited-content_cbyCitation_article-title">Reduced efficacy of a Src kinase inhibitor in crowded protein solution. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-021-24349-5" title="DOI URL">https://doi.org/10.1038/s41467-021-24349-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-021-24349-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-021-24349-5%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DReduced%252Befficacy%252Bof%252Ba%252BSrc%252Bkinase%252Binhibitor%252Bin%252Bcrowded%252Bprotein%252Bsolution%26aulast%3DKasahara%26aufirst%3DKento%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel M.  Shadrack</span>, <span class="hlFld-ContribAuthor ">Geradius  Deogratias</span>, <span class="hlFld-ContribAuthor ">Lucy W.  Kiruri</span>, <span class="hlFld-ContribAuthor ">Isaac  Onoka</span>, <span class="hlFld-ContribAuthor ">John-Mary  Vianney</span>, <span class="hlFld-ContribAuthor ">Hulda  Swai</span>, <span class="hlFld-ContribAuthor ">Stephen S.  Nyandoro</span>. </span><span class="cited-content_cbyCitation_article-title">Luteolin: a blocker of SARS-CoV-2 cell entry based on relaxed complex scheme, molecular dynamics simulation, and metadynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2021,</strong> <em>27 </em>
                                    (8)
                                     <a href="https://doi.org/10.1007/s00894-021-04833-x" title="DOI URL">https://doi.org/10.1007/s00894-021-04833-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-021-04833-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-021-04833-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DLuteolin%25253A%252Ba%252Bblocker%252Bof%252BSARS-CoV-2%252Bcell%252Bentry%252Bbased%252Bon%252Brelaxed%252Bcomplex%252Bscheme%25252C%252Bmolecular%252Bdynamics%252Bsimulation%25252C%252Band%252Bmetadynamics%26aulast%3DShadrack%26aufirst%3DDaniel%2BM.%26date%3D2021%26date%3D2021%26volume%3D27%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pauric  Bannigan</span>, <span class="hlFld-ContribAuthor ">Matteo  Aldeghi</span>, <span class="hlFld-ContribAuthor ">Zeqing  Bao</span>, <span class="hlFld-ContribAuthor ">Florian  Häse</span>, <span class="hlFld-ContribAuthor ">Alán  Aspuru-Guzik</span>, <span class="hlFld-ContribAuthor ">Christine  Allen</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning directed drug formulation development. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2021,</strong> <em>175 </em>, 113806. <a href="https://doi.org/10.1016/j.addr.2021.05.016" title="DOI URL">https://doi.org/10.1016/j.addr.2021.05.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2021.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2021.05.016%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DMachine%252Blearning%252Bdirected%252Bdrug%252Bformulation%252Bdevelopment%26aulast%3DBannigan%26aufirst%3DPauric%26date%3D2021%26volume%3D175%26spage%3D113806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francis A.  Acquah</span>, <span class="hlFld-ContribAuthor ">Matthew  Paramel</span>, <span class="hlFld-ContribAuthor ">Adama  Kuta</span>, <span class="hlFld-ContribAuthor ">Syed R.  Hussaini</span>, <span class="hlFld-ContribAuthor ">David R.  Wallace</span>, <span class="hlFld-ContribAuthor ">Blaine H. M.  Mooers</span>. </span><span class="cited-content_cbyCitation_article-title">Simulations of Promising Indolizidine—α6-β2 Nicotinic Acetylcholine Receptor Complexes. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (15)
                                     , 7934. <a href="https://doi.org/10.3390/ijms22157934" title="DOI URL">https://doi.org/10.3390/ijms22157934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22157934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22157934%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DSimulations%252Bof%252BPromising%252BIndolizidine%2525E2%252580%252594%2525CE%2525B16-%2525CE%2525B22%252BNicotinic%252BAcetylcholine%252BReceptor%252BComplexes%26aulast%3DAcquah%26aufirst%3DFrancis%2BA.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D15%26spage%3D7934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Facs.jmedchem.5b01684&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01684%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=389&amp;pagesCount=4&amp;pageNumber=2" title="Load more citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load more citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Thermodynamic cycle used to calculate relative binding free energies (ΔΔ<i>G</i><sub>b</sub> = Δ<i>G</i>°<sub arrange="stagger">bA</sub> – Δ<i>G</i>°<sub arrange="stagger">bB</sub>) between congeneric ligands. The horizontal legs correspond to the physical binding process, whereas vertical legs indicate the unphysical transformation of ligand <i>A</i> (blue) into ligand <i>B</i> (green) performed in bulk solvent (left) and in the protein binding site (right). FEP is used to compute the free energy associated with the vertical legs of the cycle, and the relative binding free energy is calculated as the difference between the free energy required to transform the ligands in the binding site (Δ<i>G</i><sup>p</sup>) and in solution (Δ<i>G</i><sup>w</sup>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Typical thermodynamic cycle used to compute absolute binding free energy differences (double decoupling method, DDM). In the simplest implementation, the cycle involves the introduction of rototranslational restraints to the bound ligand in order to avoid the “wandering ligand” problem. The restraints are depicted as a ring fastening the ligand to the protein, and their associated free-energy penalty is Δ<i>G</i><sup>p</sup><sub>rstr</sub>. Eventually, conformational restraints might also be envisioned (not shown in the picture for simplicity). Then the ligand is reversibly decoupled from the protein (or the solvent), usually, in two steps: first, electrostatics is turned off (Δ<i>G</i><sup>p</sup><sub>ele</sub> or Δ<i>G</i><sup>w</sup><sub>ele</sub>), then the van der Waals contributions are switched off (Δ<i>G</i><sup>p</sup><sub>vdW</sub> or Δ<i>G</i><sup>w</sup><sub>vdW</sub>). The “uncharged” state is represented by the ligand colored in white, whereas the transparent ligand depicts the fully decoupled state. The free-energy penalties associated with confining the decoupled ligand in the binding-site volume are calculated analytically (horizontal leg at the bottom of the cycle), and the standard state correction (Δ<i>G</i>°<sub>rstr</sub>) is usually added to this contribution.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Simplified free-energy profile for the (un)binding process. The drug–receptor complex is shown as a deep free-energy minimum on the left (bound state), and the dissociated complex is represented as a higher-in-energy minimum on the right (unbound state). The thermodynamics of binding is quantified (in first approximation) by the free-energy difference between these minima (Δ<i>G</i><sub>b</sub>), and the kinetics of (un)binding is determined by the dissociation and association rate constants, <i>k</i><sub>off</sub> and <i>k</i><sub>on</sub>. These quantities are related to the free-energy differences between minima and the transition state, Δ<i>G</i><sup>⧧</sup><sub>off</sub> and Δ<i>G</i><sup>⧧</sup><sub>on</sub>, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fundamental steps in a virtual screening workflow combining docking and MD simulations. (A) An MD trajectory is used to explore the receptor conformational space. (B) From the trajectory, several snapshots are extracted and redundancy is eliminated by means of cluster analysis. (C) From each cluster, a representative structure (e.g., medoid) is selected. (D) Virtual ligand screening is independently carried out at each representative conformation. (E) Activity predictions returned by independent runs are combined together in a global ranking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. From the crystal structure of compound R221239 (compound <b>3</b>), the modest anti-HIV activity of the initial docking hit (compound <b>1</b>), 5 μM potency toward WT HIV-1 reverse transcriptase, was efficiently evolved into highly potent catechol diethers such as compound <b>42</b>, a 55 pM non-nucleoside inhibitor of HIV-1 reverse transcriptase (NNRTIs) discovered with the aid of the computational analyses guided by FEP results.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison of the undocking force profiles of different flavonoid ligands. (A) Chemical structures of ligands under investigation. (B) Force profiles derived from pulling the ligands along the unbinding reaction coordinate. For each ligand, the plots show the resulting mean values from averaging the force profiles from five different SMD runs.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Pictorial representation of a metadynamics simulation. The underlying free energy, which is unknown beforehand, consists of three minima, A, B, and C, separated by two transition states. It is depicted as a black thick curve. The bias introduced by metadynamics is represented by filled curves with a dark-gray to red color gradient indicating the progression of the simulation. Starting the calculation in minimum A, the Gaussian bias released in time by metadynamics gently pushes the system toward the first transition state (TS<sub>1</sub>) and leads to the identification of the global minimum B. Then, by continuation of the simulation, TS<sub>2</sub> and the metastable state C are also visited. Finally, when the underlying free energy is completely filled by the Gaussian potentials, the system evolves with diffusive motion along the collective variable, and the simulation can be stopped. At that time, with conventional metadynamics, the free energy can be recovered from the bias by simply changing the sign.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison of three strategies to couple FEP with replica exchange. In panel A, the FEP/REMD method implements replica exchanges between the staged alchemical transformation in the context of the double decoupling method (DDM). Here, the unphysical transformation is performed in three steps: charges, dispersive, and repulsive van der Waals interactions are turned off separately and in sequence. For simplicity, the inclusion of rototranslational and conformational restraints, as envisioned in the original paper, is not considered. The axis controlling the alchemical transformation (λ-axis) is retained in the FEP/H-REMD method (B), but it is coupled to a boosting axis where the potential energy of selected dihedrals is scaled down, mimicking the effect of high temperature. In the context of relative binding free-energy calculations, a fictitious high temperature is also used in the FEP/REST method (C). In this case, the alchemical transformation is performed together with the potential energy rescaling. Thus, the “hottest” replicas are located in the middle of the ladder, and physical states are only found at the end points.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Allosteric modulation in BCR-ABL (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f4j">2f4j</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5v">3k5v</a>). The ligand GNF-2 binds to the myristate pocket (red circle, allosteric site) concomitantly to the bending of the αI′-helix and induces a c-ABL-like autoinhibited conformation of BCR-ABL that has reduced kinase activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/medium/jm-2015-016843_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Water-mediated protein–ligand interactions from crystallographic structures. (A) Bound pose of deoxythymidine at the binding site of thymidine kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM">1KIM</a>) and (B) bound pose of a benzisoxazole-based derivative at the binding site of HSP90 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BMY">3BMY</a>). Ligands, relevant protein residues, and bridging water molecules are reported in ball and stick representation. Water molecules are highlighted by red spheres. The boundaries of the binding pocket are represented by a transparent white mesh. Dotted red lines represent the most relevant hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01684/20161107/images/large/jm-2015-016843_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01684&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31518" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31518" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 243 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">The many roles of computation in drug discovery</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1818</span><span class="refDoi"> DOI: 10.1126/science.1096361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.1096361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15031495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1813-1818&author=W.+L.+Jorgensen&title=The+many+roles+of+computation+in+drug+discovery&doi=10.1126%2Fscience.1096361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Many Roles of Computation in Drug Discovery</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1813-1818</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  An overview is given on the diverse uses of computational chem. in drug discovery.  Particular emphasis is placed on virtual screening, de novo design, evaluation of drug-likeness, and advanced methods for detg. protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPr2xRlx_bxbVg90H21EOLACvtfcHk0lh0bodePNKj5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D&md5=77b2fac63e750ca7cdc7e4f35654549a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1096361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1096361%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DThe%2520many%2520roles%2520of%2520computation%2520in%2520drug%2520discovery%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1813%26epage%3D1818%26doi%3D10.1126%2Fscience.1096361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Bridging quantum mechanics and structure-based drug design</span> <span class="citation_source-journal">Front. Biosci., Landmark Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1619</span><span class="NLM_x">–</span> <span class="NLM_lpage">1633</span><span class="refDoi"> DOI: 10.2741/3809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2741%2F3809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21196252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXislCks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=1619-1633&author=M.+De+Vivo&title=Bridging+quantum+mechanics+and+structure-based+drug+design&doi=10.2741%2F3809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Bridging quantum mechanics and structure-based drug design</span></div><div class="casAuthors">De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1619-1633</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  The last decade has seen great advances in the use of quantum mechanics (QM) to solve biol. problems of pharmaceutical relevance.  For instance, enzymic catalysis is often investigated by means of the so-called QM/MM approach, which uses QM and mol. mechanics (MM) methods to det. the (free) energy landscape of the enzymic reaction mechanism.  Here, I will discuss a few representative examples of QM and QM/MM studies of important metalloenzymes of pharmaceutical interest (i.e. metallophosphatases and metallo-beta-lactamases).  This review article aims to show how QM-based methods can be used to elucidate ligand-receptor interactions.  The challenge is then to exploit this knowledge for the structure-based design of new and potent inhibitors, such as transition state (TS) analogs that resemble the structure and physicochem. properties of the enzymic TS.  Given the results and potential expressed to date by QM-based methods in studying biol. problems, the application of QM in structure-based drug design will likely increase, making of these once-prohibitive computations a routinely used tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop9buFmXM2srVg90H21EOLACvtfcHk0lhWZKAMFM0k_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXislCks7c%253D&md5=0db260d6c1d81db9caf0b45f519d3e85</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2741%2F3809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3809%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DBridging%2520quantum%2520mechanics%2520and%2520structure-based%2520drug%2520design%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2011%26volume%3D16%26spage%3D1619%26epage%3D1633%26doi%3D10.2741%2F3809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Durrant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations and drug discovery</span> <span class="citation_source-journal">BMC Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="refDoi"> DOI: 10.1186/1741-7007-9-71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1186%2F1741-7007-9-71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22035460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=71&author=J.+Durrantauthor=J.+A.+McCammon&title=Molecular+dynamics+simulations+and+drug+discovery&doi=10.1186%2F1741-7007-9-71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations and drug discovery</span></div><div class="casAuthors">Durrant, Jacob D.; Andrew McCammon, J.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71</span>CODEN:
                <span class="NLM_cas:coden">BBMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1741-7007</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">This review discusses the many roles atomistic computer simulations of macromol. (for example, protein) receptors and their assocd. small-mol. ligands can play in drug discovery, including the identification of cryptic or allosteric binding sites, the enhancement of traditional virtual-screening methodologies, and the direct prediction of small-mol. binding energies.  The limitations of current simulation methodologies, including the high computational costs and approxns. of mol. forces required, are also discussed.  With const. improvements in both computer power and algorithm design, the future of computer-aided drug design is promising; mol. dynamics simulations are likely to play an increasingly important role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVpt_0gt58jbVg90H21EOLACvtfcHk0lhWZKAMFM0k_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu73L&md5=34c37576ca19c71701afdb593c370a5f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2F1741-7007-9-71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1741-7007-9-71%26sid%3Dliteratum%253Aachs%26aulast%3DDurrant%26aufirst%3DJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520and%2520drug%2520discovery%26jtitle%3DBMC%2520Biol.%26date%3D2011%26volume%3D9%26spage%3D71%26doi%3D10.1186%2F1741-7007-9-71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Harvey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">High-throughput molecular dynamics: the powerful new tool for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1059</span><span class="NLM_x">–</span> <span class="NLM_lpage">1062</span><span class="refDoi"> DOI: 10.1016/j.drudis.2012.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2012.03.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1059-1062&author=M.+J.+Harveyauthor=G.+De+Fabritiis&title=High-throughput+molecular+dynamics%3A+the+powerful+new+tool+for+drug+discovery&doi=10.1016%2Fj.drudis.2012.03.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DHarvey%26aufirst%3DM.%2BJ.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DHigh-throughput%2520molecular%2520dynamics%253A%2520the%2520powerful%2520new%2520tool%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D1059%26epage%3D1062%26doi%3D10.1016%2Fj.drudis.2012.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Boehr, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, P. E.</span><span> </span><span class="NLM_article-title">The role of dynamic conformational ensembles in biomolecular recognition</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">789</span><span class="NLM_x">–</span> <span class="NLM_lpage">796</span><span class="refDoi"> DOI: 10.1038/nchembio.232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnchembio.232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19841628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OnsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=789-796&author=D.+D.+Boehrauthor=R.+Nussinovauthor=P.+E.+Wright&title=The+role+of+dynamic+conformational+ensembles+in+biomolecular+recognition&doi=10.1038%2Fnchembio.232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The role of dynamic conformational ensembles in biomolecular recognition</span></div><div class="casAuthors">Boehr, David D.; Nussinov, Ruth; Wright, Peter E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">789-796</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mol. recognition by biomols. is central to all biol. processes.  For the past 50 yr, the 'induced fit' hypothesis of D. E. Koshland (1958) has been the textbook explanation for mol. recognition events.  However, recent exptl. evidence supports an alternative mechanism.  The 'conformational selection' model postulates that all conformations pre-exist, and that the ligand selects the most favored conformation.  Following binding, the ensemble undergoes a population shift, redistributing the conformational states.  Both conformational selection and induced fit appear to play roles.  Following binding by a primary conformational selection event, optimization of side-chain and backbone interactions is likely to proceed by an induced fit mechanism.  Conformational selection has been obsd. for protein-ligand, protein-protein, protein-DNA, protein-RNA and RNA-ligand interactions.  These data support a new mol. recognition paradigm for processes as diverse as signaling, catalysis, gene regulation, and protein aggregation in disease, which has the potential to significantly impact researchers' views and strategies in drug design, biomol. engineering, and mol. evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGputCrvx3HqSLVg90H21EOLACvtfcHk0lhWZKAMFM0k_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OnsLrJ&md5=984e03a4380d9dafd19048a4b45589c6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.232%26sid%3Dliteratum%253Aachs%26aulast%3DBoehr%26aufirst%3DD.%2BD.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DP.%2BE.%26atitle%3DThe%2520role%2520of%2520dynamic%2520conformational%2520ensembles%2520in%2520biomolecular%2520recognition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D789%26epage%3D796%26doi%3D10.1038%2Fnchembio.232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Changeux, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelstein, S.</span><span> </span><span class="NLM_article-title">Conformational selection or induced fit? 50 years of debate resolved</span> <span class="citation_source-journal">F1000 Biol. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="refDoi"> DOI: 10.3410/B3-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.3410%2FB3-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21941598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC3MflvVWisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=19&author=J.-P.+Changeuxauthor=S.+Edelstein&title=Conformational+selection+or+induced+fit%3F+50+years+of+debate+resolved&doi=10.3410%2FB3-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational selection or induced fit? 50 years of debate resolved</span></div><div class="casAuthors">Changeux Jean-Pierre; Edelstein Stuart</div><div class="citationInfo"><span class="NLM_cas:title">F1000 biology reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Exactly 50 years ago, biochemists raised the question of the mechanism of the conformational change that mediates "allosteric" interactions between regulatory sites and biologically active sites in regulatory/receptor proteins.  Do the different conformations involved already exist spontaneously in the absence of the regulatory ligands (Monod-Wyman-Changeux), such that the complementary protein conformation would be selected to mediate signal transduction, or do particular ligands induce the receptor to adopt the conformation best suited to them (Koshland-Nemethy-Filmer-induced fit)? This is not just a central question for biophysics, it also has enormous importance for drug design.  Recent advances in techniques have allowed detailed experimental and theoretical comparisons with the formal models of both scenarios.  Also, it has been shown that mutated receptors can adopt constitutively active confirmations in the absence of ligand.  There have also been demonstrations that the atomic resolution structures of the same protein are essentially the same whether ligand is bound or not.  These and other advances in past decades have produced a situation where the vast majority of the data using different categories of regulatory proteins (including regulatory enzymes, ligand-gated ion channels, G protein-coupled receptors, and nuclear receptors) support the conformational selection scheme of signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTRVYQlbx4kqkloaUqgkFHkfW6udTcc2eb0-vNIXqHAYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MflvVWisw%253D%253D&md5=01de9b3981bdbfda147e056f8b223e08</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3410%2FB3-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3410%252FB3-19%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26aulast%3DEdelstein%26aufirst%3DS.%26atitle%3DConformational%2520selection%2520or%2520induced%2520fit%253F%252050%2520years%2520of%2520debate%2520resolved%26jtitle%3DF1000%2520Biol.%2520Rep.%26date%3D2011%26volume%3D3%26spage%3D19%26doi%3D10.3410%2FB3-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Vogt, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Cera, E.</span><span> </span><span class="NLM_article-title">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5894</span><span class="NLM_x">–</span> <span class="NLM_lpage">5902</span><span class="refDoi"> DOI: 10.1021/bi3006913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3006913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5894-5902&author=A.+D.+Vogtauthor=E.+Di+Cera&title=Conformational+selection+or+induced+fit%3F+A+critical+appraisal+of+the+kinetic+mechanism&doi=10.1021%2Fbi3006913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span></div><div class="casAuthors">Vogt, Austin D.; Di Cera, Enrico</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5894-5902</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">For almost 5 decades, 2 competing mechanisms of ligand recognition, conformational selection and induced fit, have dominated the interpretation of ligand binding in biol. macromols.  When binding-dissocn. events are fast compared to conformational transitions, the rate of approach to equil., kobs, becomes diagnostic of conformational selection or induced fit based on whether it decreases or increases, resp., with the ligand concn. [L].  However, this simple conclusion based on the rapid equil. approxn. is not valid in general.  Here, the authors show that conformational selection is assocd. with a rich repertoire of kinetic properties, with kobs decreasing or increasing with [L] depending on the relative magnitude of the rate of ligand dissocn., koff, and the rate of conformational isomerization, kr.  The authors prove that, even for the simplest 2-step mechanism of ligand binding, a decrease in kobs with [L] is unequivocal evidence of conformational selection, but an increase in kobs with [L] is not unequivocal evidence of induced fit.  Ligand binding to glucokinase, thrombin, and its precursor, prethrombin-2, are used as relevant examples.  It is concluded that conformational selection as a mechanism for ligand binding to its target may be far more common than currently believed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdButaqMoNzLVg90H21EOLACvtfcHk0lgLuj6UpmqjLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D&md5=0f7c66027467c390967075c3be8e5254</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbi3006913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3006913%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DA.%2BD.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DConformational%2520selection%2520or%2520induced%2520fit%253F%2520A%2520critical%2520appraisal%2520of%2520the%2520kinetic%2520mechanism%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5894%26epage%3D5902%26doi%3D10.1021%2Fbi3006913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1038/nchem.1954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnchem.1954" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24950326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=575-583&author=M.+Fischerauthor=R.+G.+Colemanauthor=J.+S.+Fraserauthor=B.+K.+Shoichet&title=Incorporation+of+protein+flexibility+and+conformational+energy+penalties+in+docking+screens+to+improve+ligand+discovery&doi=10.1038%2Fnchem.1954"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span></div><div class="casAuthors">Fischer, Marcus; Coleman, Ryan G.; Fraser, James S.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">575-583</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins fluctuate between alternative conformations, which presents a challenge for ligand discovery because such flexibility is difficult to treat computationally owing to problems with conformational sampling and energy weighting.  Here we describe a flexible docking method that samples and wts. protein conformations using exptl. derived conformations as a guide.  The crystallog. refined occupancies of these conformations, which are observable in an apo receptor structure, define energy penalties for docking.  In a large prospective library screen, we identified new ligands that target specific receptor conformations of a cavity in cytochrome c peroxidase, and we confirm both ligand pose and assocd. receptor conformation predictions by crystallog.  The inclusion of receptor flexibility led to ligands with new chemotypes and phys. properties.  By exploiting exptl. measures of loop and side-chain flexibility, this method can be extended to the discovery of new ligands for hundreds of targets in the Protein Data Bank for which similar exptl. information is available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9q4phmwMLHbVg90H21EOLACvtfcHk0lgLuj6UpmqjLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D&md5=141d6f5238664dabb0890cc5f764b99d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1954%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DIncorporation%2520of%2520protein%2520flexibility%2520and%2520conformational%2520energy%2520penalties%2520in%2520docking%2520screens%2520to%2520improve%2520ligand%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D575%26epage%3D583%26doi%3D10.1038%2Fnchem.1954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">High-throughput docking for lead generation</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">382</span><span class="refDoi"> DOI: 10.1016/S1367-5931(00)00217-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1367-5931%2800%2900217-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11470599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=375-382&author=R.+Abagyanauthor=M.+Totrov&title=High-throughput+docking+for+lead+generation&doi=10.1016%2FS1367-5931%2800%2900217-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput docking for lead generation</span></div><div class="casAuthors">Abagyan, R.; Totrov, M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-382</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Recent improvements in flexible docking technol. may lead to a bigger role for computational methods in lead discovery.  Although fast and accurate computational prediction of binding affinities for an arbitrary mol. is still beyond the limits of current methods, the docking and screening procedures can select small sets of likely lead candidates from large libraries of either com. or synthetically available compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9_FjizU9XdbVg90H21EOLACvtfcHk0lgLuj6UpmqjLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D&md5=c9bfaa493becdadb4386522865b9e7fb</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900217-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900217-9%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DHigh-throughput%2520docking%2520for%2520lead%2520generation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D375%26epage%3D382%26doi%3D10.1016%2FS1367-5931%2800%2900217-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">The future of molecular dynamics simulations in drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1007/s10822-011-9517-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-011-9517-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22183577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2ltb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=15-26&author=D.+W.+Borhaniauthor=D.+E.+Shaw&title=The+future+of+molecular+dynamics+simulations+in+drug+discovery&doi=10.1007%2Fs10822-011-9517-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The future of molecular dynamics simulations in drug discovery</span></div><div class="casAuthors">Borhani, David W.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-26</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mol. dynamics simulations can now track rapid processes-those occurring in less than about a millisecond-at at. resoln. for many biol. relevant systems.  These simulations appear poised to exert a significant impact on how new drugs are found, perhaps even transforming the very process of drug discovery.  We predict here future results we can expect from, and enhancements we need to make in, mol. dynamics simulations over the coming 25 years, and in so doing set out several Grand Challenges for the field.  In the context of the problems now facing the pharmaceutical industry, we ask how we can best address drug discovery needs of the next quarter century using mol. dynamics simulations, and we suggest some possible approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeWiHHq3tLC7Vg90H21EOLACvtfcHk0lhFN12DT39h7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2ltb0%253D&md5=78388465d8bb54e99917d8da3d1e0824</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9517-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9517-y%26sid%3Dliteratum%253Aachs%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DThe%2520future%2520of%2520molecular%2520dynamics%2520simulations%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D15%26epage%3D26%26doi%3D10.1007%2Fs10822-011-9517-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Frenkel, D.; Smit, B.</span> <span class="citation_source-book">Understanding Molecular Simulation</span>; <span class="NLM_publisher-name">Academic Press, Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2001</span><span class="NLM_x">; </span>p  <span class="NLM_fpage">638</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=638&author=D.+Frenkel&author=B.+Smit&title=Understanding+Molecular+Simulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFrenkel%26aufirst%3DD.%26btitle%3DUnderstanding%2520Molecular%2520Simulation%26pub%3DAcademic%2520Press%252C%2520Inc%26date%3D2001%26spage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Allen, M. P.; Tildesley, D. J.</span> <span class="citation_source-book">Computer Simulation of Liquids</span>; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">1989</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&author=M.+P.+Allen&author=D.+J.+Tildesley&title=Computer+Simulation+of+Liquids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DM.%2BP.%26btitle%3DComputer%2520Simulation%2520of%2520Liquids%26pub%3DOxford%2520University%2520Press%26date%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelin, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Dynamics of folded proteins</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">585</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span><span class="refDoi"> DOI: 10.1038/267585a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2F267585a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=301613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE2sXlsVOgsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1977&pages=585-590&author=J.+A.+McCammonauthor=B.+R.+Gelinauthor=M.+Karplus&title=Dynamics+of+folded+proteins&doi=10.1038%2F267585a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics of folded proteins</span></div><div class="casAuthors">McCammon, J. Andrew; Gelin, Bruce R.; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5612</span>),
    <span class="NLM_cas:pages">585-90</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The dynamics of a folded globular protein (bovine pancreatic trypsin inhibitor) have been studied by solving the equations of motions for the atoms with an empirical potential energy function.  They provide the magnitude, correlations, and decay of fluctuations about the av. structure.  The protein interior may therefore be fluidlike in that the local atom motions have a diffusional character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZp3fNflIAd7Vg90H21EOLACvtfcHk0lhFN12DT39h7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXlsVOgsbg%253D&md5=825f061d862ff7de635fcbaac6e9afbb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2F267585a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F267585a0%26sid%3Dliteratum%253Aachs%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DGelin%26aufirst%3DB.%2BR.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DDynamics%2520of%2520folded%2520proteins%26jtitle%3DNature%26date%3D1977%26volume%3D267%26spage%3D585%26epage%3D590%26doi%3D10.1038%2F267585a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Levitt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warshel, A.</span><span> </span><span class="NLM_article-title">Computer simulation of protein folding</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1975</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">694</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1038/253694a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2F253694a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1167625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE2MXkt1eksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1975&pages=694-698&author=M.+Levittauthor=A.+Warshel&title=Computer+simulation+of+protein+folding&doi=10.1038%2F253694a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Computer simulation of protein folding</span></div><div class="casAuthors">Levitt, Michael; Warshel, Arieh</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5494</span>),
    <span class="NLM_cas:pages">694-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Computer simulation of protein folding was obtained by combining a simple representation of protein conformation, which averaged over the fine details, with convergent energy minimization and normal mode thermalization.  Under certain conditions, the procedure reproduced the correct overall folding of bovine pancreatic trypsin inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNCNUZcQtSobVg90H21EOLACvtfcHk0lhFN12DT39h7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXkt1eksbw%253D&md5=fb6bbeb114d0c1a4d95d12a082e01ccf</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F253694a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F253694a0%26sid%3Dliteratum%253Aachs%26aulast%3DLevitt%26aufirst%3DM.%26aulast%3DWarshel%26aufirst%3DA.%26atitle%3DComputer%2520simulation%2520of%2520protein%2520folding%26jtitle%3DNature%26date%3D1975%26volume%3D253%26spage%3D694%26epage%3D698%26doi%3D10.1038%2F253694a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Protein flexibility and drug design: how to hit a moving target</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">447</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span><span class="refDoi"> DOI: 10.1016/S1367-5931(02)00341-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1367-5931%2802%2900341-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12133719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2002&pages=447-452&author=H.+A.+Carlson&title=Protein+flexibility+and+drug+design%3A+how+to+hit+a+moving+target&doi=10.1016%2FS1367-5931%2802%2900341-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Protein flexibility and drug design: how to hit a moving target</span></div><div class="casAuthors">Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-452</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  The most advanced methods for computer-aided drug design and database mining incorporate protein flexibility.  Such techniques are not only needed to obtain proper results; they are also crit. for dealing with the growing body of information from structural genomics.  Advanced methods for incorporating protein flexibility into database searching and computer-aided drug design are not only needed to obtain proper results; they are crit. for dealing with the growing body of information from structural genomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrATwQbEeJJUbVg90H21EOLACvtfcHk0ljHxH2djmrtug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjtbg%253D&md5=eb8ff234381f16086fb8caf2dc88486d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2802%2900341-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252802%252900341-1%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DProtein%2520flexibility%2520and%2520drug%2520design%253A%2520how%2520to%2520hit%2520a%2520moving%2520target%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2002%26volume%3D6%26spage%3D447%26epage%3D452%26doi%3D10.1016%2FS1367-5931%2802%2900341-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Cozzini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellogg, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spyrakis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emerson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fanelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orozco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pertinhez, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotriffer, C.</span><span> </span><span class="NLM_article-title">Target Flexibility: An emerging consideration in drug discovery and design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6237</span><span class="NLM_x">–</span> <span class="NLM_lpage">6255</span><span class="refDoi"> DOI: 10.1021/jm800562d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800562d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFaqtLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6237-6255&author=P.+Cozziniauthor=G.+E.+Kelloggauthor=F.+Spyrakisauthor=D.+J.+Abrahamauthor=G.+Costantinoauthor=A.+Emersonauthor=F.+Fanelliauthor=H.+Gohlkeauthor=L.+A.+Kuhnauthor=G.+M.+Morrisauthor=M.+Orozcoauthor=T.+A.+Pertinhezauthor=M.+Rizziauthor=C.+Sotriffer&title=Target+Flexibility%3A+An+emerging+consideration+in+drug+discovery+and+design&doi=10.1021%2Fjm800562d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Target Flexibility: An Emerging Consideration in Drug Discovery and Design</span></div><div class="casAuthors">Cozzini, Pietro; Kellogg, Glen E.; Spyrakis, Francesca; Abraham, Donald J.; Costantino, Gabriele; Emerson, Andrew; Fanelli, Francesca; Gohlke, Holger; Kuhn, Leslie A.; Morris, Garrett M.; Orozco, Modesto; Pertinhez, Thelma A.; Rizzi, Menico; Sotriffer, Christoph A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6237-6255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5h-wKT3wh6bVg90H21EOLACvtfcHk0ljHxH2djmrtug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFaqtLvK&md5=8b9a64cc09fe04fb43cbbc45463abf1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm800562d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800562d%26sid%3Dliteratum%253Aachs%26aulast%3DCozzini%26aufirst%3DP.%26aulast%3DKellogg%26aufirst%3DG.%2BE.%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DAbraham%26aufirst%3DD.%2BJ.%26aulast%3DCostantino%26aufirst%3DG.%26aulast%3DEmerson%26aufirst%3DA.%26aulast%3DFanelli%26aufirst%3DF.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DKuhn%26aufirst%3DL.%2BA.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DOrozco%26aufirst%3DM.%26aulast%3DPertinhez%26aufirst%3DT.%2BA.%26aulast%3DRizzi%26aufirst%3DM.%26aulast%3DSotriffer%26aufirst%3DC.%26atitle%3DTarget%2520Flexibility%253A%2520An%2520emerging%2520consideration%2520in%2520drug%2520discovery%2520and%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6237%26epage%3D6255%26doi%3D10.1021%2Fjm800562d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Flexible ligand docking to multiple receptor conformations: a practical alternative</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">178</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1016/j.sbi.2008.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2008.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18302984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=178-184&author=M.+Totrovauthor=R.+Abagyan&title=Flexible+ligand+docking+to+multiple+receptor+conformations%3A+a+practical+alternative&doi=10.1016%2Fj.sbi.2008.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible ligand docking to multiple receptor conformations: a practical alternative</span></div><div class="casAuthors">Totrov, Maxim; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-184</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  State of the art docking algorithms predict an incorrect binding pose for about 50-70% of all ligands when only a single fixed receptor conformation is considered.  In many more cases, lack of receptor flexibility results in meaningless ligand binding scores, even when the correct pose is obtained.  Incorporating conformational rearrangements of the receptor binding pocket into predictions of both ligand binding pose and binding score is crucial for improving structure-based drug design and virtual ligand screening methodologies.  However, direct modeling of protein binding site flexibility remains challenging because of the large conformational space that must be sampled, and difficulties remain in constructing a suitably accurate energy function.  Here the authors show that using multiple fixed receptor conformations, either exptl. detd. by crystallog. or NMR, or computationally generated, is a practical shortcut that may improve docking calcns.  In several cases, such an approach has led to exptl. validated predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppaNNJ60eASbVg90H21EOLACvtfcHk0ljHxH2djmrtug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D&md5=61b273c1621cfd64832bdda28898b439</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2008.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2008.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DFlexible%2520ligand%2520docking%2520to%2520multiple%2520receptor%2520conformations%253A%2520a%2520practical%2520alternative%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D18%26spage%3D178%26epage%3D184%26doi%3D10.1016%2Fj.sbi.2008.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Four-dimensional docking: a fast and accurate account of discrete receptor flexibility in ligand docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1021/jm8009958</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8009958" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=397-406&author=G.+Bottegoniauthor=I.+Kufarevaauthor=M.+Totrovauthor=R.+Abagyan&title=Four-dimensional+docking%3A+a+fast+and+accurate+account+of+discrete+receptor+flexibility+in+ligand+docking&doi=10.1021%2Fjm8009958"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm8009958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8009958%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DFour-dimensional%2520docking%253A%2520a%2520fast%2520and%2520accurate%2520account%2520of%2520discrete%2520receptor%2520flexibility%2520in%2520ligand%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D397%26epage%3D406%26doi%3D10.1021%2Fjm8009958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span> </span><span class="NLM_article-title">Protein-ligand docking</span> <span class="citation_source-journal">Front. Biosci., Landmark Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">2289</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.2741/3854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2741%2F3854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21622177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVagsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=2289-306&author=G.+Bottegoni&title=Protein-ligand+docking&doi=10.2741%2F3854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-ligand docking</span></div><div class="casAuthors">Bottegoni, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2289-2306</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Ligand-docking is an established computational technique universally applied in structure-based drug design.  Since the first attempts carried out in the early '80s to predict the three-dimensional conformation of a protein-ligand bound complex, this methodol. has evolved constantly and it is presently implemented in many different ways.  The present study aims at explaining the std. protein-ligand docking protocol, together with its main advantages and drawbacks.  Milestone reports and future directions are reported and discussed as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvKVbh630jHrVg90H21EOLACvtfcHk0lgyvitgAZUH4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVagsrk%253D&md5=efd9553ea8dc75c60b348e75387e77cf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2741%2F3854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3854%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DProtein-ligand%2520docking%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2011%26volume%3D16%26spage%3D2289%26epage%3D306%26doi%3D10.2741%2F3854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. T.</span><span> </span><span class="NLM_article-title">Understanding protein-ligand interactions: the price of protein flexibility</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">335</span><span class="NLM_x">, </span> <span class="NLM_fpage">1325</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1016/j.jmb.2003.11.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmb.2003.11.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2004&pages=1325-41&author=D.+Rauhauthor=G.+Klebeauthor=M.+T.+Stubbs&title=Understanding+protein-ligand+interactions%3A+the+price+of+protein+flexibility&doi=10.1016%2Fj.jmb.2003.11.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.11.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.11.041%26sid%3Dliteratum%253Aachs%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26atitle%3DUnderstanding%2520protein-ligand%2520interactions%253A%2520the%2520price%2520of%2520protein%2520flexibility%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D335%26spage%3D1325%26epage%3D41%26doi%3D10.1016%2Fj.jmb.2003.11.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Damm, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Exploring experimental sources of multiple protein conformations in structure-based drug design</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">8225</span><span class="NLM_x">–</span> <span class="NLM_lpage">8235</span><span class="refDoi"> DOI: 10.1021/ja0709728</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0709728" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=8225-8235&author=K.+L.+Dammauthor=H.+A.+Carlson&title=Exploring+experimental+sources+of+multiple+protein+conformations+in+structure-based+drug+design&doi=10.1021%2Fja0709728"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja0709728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0709728%26sid%3Dliteratum%253Aachs%26aulast%3DDamm%26aufirst%3DK.%2BL.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DExploring%2520experimental%2520sources%2520of%2520multiple%2520protein%2520conformations%2520in%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D8225%26epage%3D8235%26doi%3D10.1021%2Fja0709728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Alonso, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bliznyuk, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gready, J. E.</span><span> </span><span class="NLM_article-title">Combining docking and molecular dynamic simulations in drug design</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">568</span><span class="refDoi"> DOI: 10.1002/med.20067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fmed.20067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16758486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsFGru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=531-568&author=H.+Alonsoauthor=A.+A.+Bliznyukauthor=J.+E.+Gready&title=Combining+docking+and+molecular+dynamic+simulations+in+drug+design&doi=10.1002%2Fmed.20067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Combining docking and molecular dynamic simulations in drug design</span></div><div class="casAuthors">Alonso, Hernan; Bliznyuk, Andrey A.; Gready, Jill E.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">531-568</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  A rational approach is needed to maximize the chances of finding new drugs, and to exploit the opportunities of potential new drug targets emerging from genomic and proteomic initiatives, and from the large libraries of small compds. now readily available through combinatorial chem.  Despite a shaky early history, computer-aided drug design techniques can now be effective in reducing costs and speeding up drug discovery.  This happy outcome results from development of more accurate and reliable algorithms, use of more thoughtfully planned strategies to apply them, and greatly increased computer power to allow studies with the necessary reliability to be performed.  Our review focuses on applications and protocols, with the main emphasis on crit. anal. of recent studies where docking calcns. and mol. dynamics (MD) simulations were combined to dock small mols. into protein receptors.  We highlight successes to demonstrate what is possible now, but also point out drawbacks and future directions.  The review is structured to lead the reader from the simpler to more compute-intensive methods.  Thus, while inexpensive and fast docking algorithms can be used to scan large compd. libraries and reduce their size, more accurate but expensive MD simulations can be applied when a few selected ligand candidates remain.  MD simulations can be used: during the prepn. of the protein receptor before docking, to optimize its structure and account for protein flexibility; for the refinement of docked complexes, to include solvent effects and account for induced fit; to calc. binding free energies, to provide an accurate ranking of the potential ligands; and in the latest developments, during the docking process itself to find the binding site and correctly dock the ligand a priori.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfwZFHimixV7Vg90H21EOLACvtfcHk0lgyvitgAZUH4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsFGru7o%253D&md5=f20210fea1ee020f95228846307e8a92</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fmed.20067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20067%26sid%3Dliteratum%253Aachs%26aulast%3DAlonso%26aufirst%3DH.%26aulast%3DBliznyuk%26aufirst%3DA.%2BA.%26aulast%3DGready%26aufirst%3DJ.%2BE.%26atitle%3DCombining%2520docking%2520and%2520molecular%2520dynamic%2520simulations%2520in%2520drug%2520design%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D531%26epage%3D568%26doi%3D10.1002%2Fmed.20067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Morra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genoni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neves, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span> </span><span class="NLM_article-title">Molecular recognition and drug-lead identification: what can molecular simulations tell us?</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.2174/092986710789957797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F092986710789957797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19941480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=25-41&author=G.+Morraauthor=A.+Genoniauthor=M.+Nevesauthor=K.+Merzauthor=G.+Colombo&title=Molecular+recognition+and+drug-lead+identification%3A+what+can+molecular+simulations+tell+us%3F&doi=10.2174%2F092986710789957797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition and drug-lead identification: what can molecular simulations tell us?</span></div><div class="casAuthors">Morra, G.; Genoni, A.; Neves, M. A. C.; Merz, K. M., Jr.; Colombo, G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-41</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Mol. recognition and ligand binding involving proteins underlie the most important life processes within the cell, such as substrate transport, catalysis, signal transmission, receptor trafficking, gene regulation, switching on and off of biochem. pathways.  Despite recent successes in predicting the structures of many protein-substrate complexes, the dynamic aspects of binding have been largely neglected by computational/theor. investigations.  Recently, several groups have started tackling these problems with the use of exptl. and simulation methods and developed models describing the variation of protein dynamics upon complex formation, shedding light on how substrate or inhibitor binding can alter protein flexibility and function.  The study of ligand-induced dynamic variations has also been exploited to review the concept of allosteric changes, in the absence of major conformational changes.  In this context, the study of the influence of protein motions on signal transduction and on catalytic activities has been used to develop pharmacophore models based on ensembles of protein conformations.  These models, taking flexibility explicitly into account, are able to distinguish active inhibitors vs. nonactive drug-like compds., to define new mol. motifs and to preferentially identify specific ligands for a certain protein target.  The application of these methods holds great promise in advancing structure-based drug discovery and medicinal chem. in general, opening up the possibility to explore broader chem. spaces than is normally done in an efficient way.  In this review, examples illustrating the extent to which simulations can be used to understand these phenomena will be presented along with examples of methodol. developments to increase phys. understanding of the processes and improve the possibility to rationally design new mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNRcMp8Hm0HLVg90H21EOLACvtfcHk0lia6vxEgAh5qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSmt70%253D&md5=c695d9184a4bb4655dee00f6a2643f4d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.2174%2F092986710789957797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986710789957797%26sid%3Dliteratum%253Aachs%26aulast%3DMorra%26aufirst%3DG.%26aulast%3DGenoni%26aufirst%3DA.%26aulast%3DNeves%26aufirst%3DM.%26aulast%3DMerz%26aufirst%3DK.%26aulast%3DColombo%26aufirst%3DG.%26atitle%3DMolecular%2520recognition%2520and%2520drug-lead%2520identification%253A%2520what%2520can%2520molecular%2520simulations%2520tell%2520us%253F%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2010%26volume%3D17%26spage%3D25%26epage%3D41%26doi%3D10.2174%2F092986710789957797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Pang, Y.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozikowski, A.</span><span> </span><span class="NLM_article-title">Prediction of the binding sites of huperzine A in acetylcholinesterase by docking studies</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span><span class="refDoi"> DOI: 10.1007/BF00124014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2FBF00124014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=669-681&author=Y.-P.+Pangauthor=A.+Kozikowski&title=Prediction+of+the+binding+sites+of+huperzine+A+in+acetylcholinesterase+by+docking+studies&doi=10.1007%2FBF00124014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2FBF00124014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124014%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DY.-P.%26aulast%3DKozikowski%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520the%2520binding%2520sites%2520of%2520huperzine%2520A%2520in%2520acetylcholinesterase%2520by%2520docking%2520studies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D669%26epage%3D681%26doi%3D10.1007%2FBF00124014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Ivetac, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Molecular recognition in the case of flexible targets</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1663</span><span class="NLM_x">–</span> <span class="NLM_lpage">1671</span><span class="refDoi"> DOI: 10.2174/138161211796355056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F138161211796355056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21619526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVyjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=1663-1671&author=A.+Ivetacauthor=J.+A.+McCammon&title=Molecular+recognition+in+the+case+of+flexible+targets&doi=10.2174%2F138161211796355056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition in the case of flexible targets</span></div><div class="casAuthors">Ivetac, Anthony; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1663-1671</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  A protein's flexibility is well recognized to underlie its capacity to engage in crit. functions, such as signal transduction, biomol. transport and biochem. reactivity.  Mol. recognition is also tightly linked to the dynamics of the binding partners, yet protein flexibility has largely been ignored by the growing field of structure-based drug design (SBDD).  In combination with exptl. detd. structures, a no. of computational methods have been proposed to model protein movements, which may be important for small mol. binding.  Such techniques have the ability to expose new binding site conformations, which may in turn recognize and lead to the discovery of more potent and selective drugs through mol. docking.  In this article, we discuss various methods and focus on the Relaxed Complex Scheme (RCS), which uses Mol. Dynamics (MD) simulations to model full protein flexibility and enhance virtual screening programs.  We review practical applications of the RCS and use a recent study of the HIV-1 reverse transcriptase to illustrate the various phases of the scheme.  We also discuss some encouraging developments, aimed at addressing current weaknesses of the RCS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFY1LH59mtWLVg90H21EOLACvtfcHk0lia6vxEgAh5qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVyjt7k%253D&md5=f8d1e0b1963435d0c40dfa96c8dcf711</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.2174%2F138161211796355056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161211796355056%26sid%3Dliteratum%253Aachs%26aulast%3DIvetac%26aufirst%3DA.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DMolecular%2520recognition%2520in%2520the%2520case%2520of%2520flexible%2520targets%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2011%26volume%3D17%26spage%3D1663%26epage%3D1671%26doi%3D10.2174%2F138161211796355056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Lin, J.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schames, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The relaxed complex method: accommodating receptor flexibility for drug design with an improved scoring scheme</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">62</span><span class="refDoi"> DOI: 10.1002/bip.10218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fbip.10218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12579579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlyrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=47-62&author=J.-H.+Linauthor=A.+L.+Perrymanauthor=J.+R.+Schamesauthor=J.+A.+McCammon&title=The+relaxed+complex+method%3A+accommodating+receptor+flexibility+for+drug+design+with+an+improved+scoring+scheme&doi=10.1002%2Fbip.10218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The relaxed complex method: accommodating receptor flexibility for drug design with an improved scoring scheme</span></div><div class="casAuthors">Lin, Jung-Hsin; Perryman, Alexander L.; Schames, Julie R.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-62</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">An extension of the new computational methodol. for drug design, the "relaxed complex" method, which accommodates receptor flexibility, is described.  This relaxed complex method recognizes that ligand may bind to conformations that occur only rarely in the dynamics of the receptor.  We have shown that the ligand-enzyme binding modes are very sensitive to the enzyme conformations, and our approach is capable of finding the best ligand enzyme complexes.  Rapid docking serves as an efficient initial filtering method to screen a myriad of docking modes to a limited set, and it is then followed by more accurate scoring with the MM/PBSA (Mol. Mechanics/Poisson Boltzmann Surface Area) approach to find the best ligand-receptor complexes.  The MM/PBSA scorings consistently indicate that the calcd. binding modes that are most similar to those obsd. in the x-ray crystallog. complexes are the ones with the lowest free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3skpENhME5rVg90H21EOLACvtfcHk0lia6vxEgAh5qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlyrtQ%253D%253D&md5=f81abcd665d2bd0b6dab9962adff68c2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fbip.10218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.10218%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.-H.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DSchames%26aufirst%3DJ.%2BR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520relaxed%2520complex%2520method%253A%2520accommodating%2520receptor%2520flexibility%2520for%2520drug%2520design%2520with%2520an%2520improved%2520scoring%2520scheme%26jtitle%3DBiopolymers%26date%3D2003%26volume%3D68%26spage%3D47%26epage%3D62%26doi%3D10.1002%2Fbip.10218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Tian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, T.</span><span> </span><span class="NLM_article-title">Assessing an ensemble docking-based virtual screening strategy for kinase targets by considering protein flexibility</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2664</span><span class="NLM_x">–</span> <span class="NLM_lpage">2679</span><span class="refDoi"> DOI: 10.1021/ci500414b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500414b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2664-2679&author=S.+Tianauthor=H.+Sunauthor=P.+Panauthor=D.+Liauthor=X.+Zhenauthor=Y.+Liauthor=T.+Hou&title=Assessing+an+ensemble+docking-based+virtual+screening+strategy+for+kinase+targets+by+considering+protein+flexibility&doi=10.1021%2Fci500414b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing an Ensemble Docking-Based Virtual Screening Strategy for Kinase Targets by Considering Protein Flexibility</span></div><div class="casAuthors">Tian, Sheng; Sun, Huiyong; Pan, Peichen; Li, Dan; Zhen, Xuechu; Li, Youyong; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2664-2679</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, to accommodate receptor flexibility, based on multiple receptor conformations, a novel ensemble docking protocol was developed by using the naive Bayesian classification technique, and it was evaluated in terms of the prediction accuracy of docking-based virtual screening (VS) of three important targets in the kinase family: ALK, CDK2, and VEGFR2.  First, for each target, the representative crystal structures were selected by structural clustering, and the capability of mol. docking based on each representative structure to discriminate inhibitors from non-inhibitors was examd.  Then, for each target, 50 ns mol. dynamics (MD) simulations were carried out to generate an ensemble of the conformations, and multiple representative structures/snapshots were extd. from each MD trajectory by structural clustering.  On av., the representative crystal structures outperform the representative structures extd. from MD simulations in terms of the capabilities to sep. inhibitors from non-inhibitors.  Finally, by using the naive Bayesian classification technique, an integrated VS strategy was developed to combine the prediction results of mol. docking based on different representative conformations chosen from crystal structures and MD trajectories.  It was encouraging to observe that the integrated VS strategy yields better performance than the docking-based VS based on any single rigid conformation.  This novel protocol may provide an improvement over existing strategies to search for more diverse and promising active compds. for a target of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMFhXu4lCyd7Vg90H21EOLACvtfcHk0lhdnZ7Lemcy8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbnJ&md5=d674a8d7bd48d44c71ecb3935c7a6ef7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci500414b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500414b%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZhen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DAssessing%2520an%2520ensemble%2520docking-based%2520virtual%2520screening%2520strategy%2520for%2520kinase%2520targets%2520by%2520considering%2520protein%2520flexibility%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2664%26epage%3D2679%26doi%3D10.1021%2Fci500414b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Tarcsay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paragi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vass, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jojart, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bogar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">The impact of molecular dynamics sampling on the performance of virtual screening against GPCRs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2990</span><span class="NLM_x">–</span> <span class="NLM_lpage">2999</span><span class="refDoi"> DOI: 10.1021/ci400087b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400087b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SgsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=2990-2999&author=A.+Tarcsayauthor=G.+Paragiauthor=M.+Vassauthor=B.+Jojartauthor=F.+Bogarauthor=G.+M.+Keseru&title=The+impact+of+molecular+dynamics+sampling+on+the+performance+of+virtual+screening+against+GPCRs&doi=10.1021%2Fci400087b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The Impact of Molecular Dynamics Sampling on the Performance of Virtual Screening against GPCRs</span></div><div class="casAuthors">Tarcsay, Akos; Paragi, Gabor; Vass, Marton; Jojart, Balazs; Bogar, Ferenc; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2990-2999</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of ligand-protein complexes requires simultaneous adaptation of the binding partners.  In structure based virtual screening, high throughput docking approaches typically consider the ligand flexibility, but the conformational freedom of the protein is usually taken into account in a limited way.  The goal of this study is to elaborate a methodol. for incorporating protein flexibility to improve the virtual screening enrichments on GPCRs.  Explicit-solvated mol. dynamics simulations (MD) were carried out in lipid bilayers to generate an ensemble of protein conformations for the X-ray structures and homol. models of both aminergic and peptidergic GPCRs including the chemokine CXCR4, dopamine D3, histamine H4, and serotonin 5HT6holo receptor complexes.  The quality of the receptor models was assessed by enrichment studies to compare X-ray structures, homol. models, and snapshots from the MD trajectory.  According to our results, selected frames from the MD trajectory can outperform X-ray structures and homol. models in terms of enrichment factor and AUC values.  Significant changes were obsd. considering EF1% values: comparing the original CXCR4, D3, and H4 targets and the addnl. 5HT6 initial models to that of the best MD frame resulted in 0 to 6.7, 0.32 to 3.5 (10×), 13.3 to 26.7 (2×), and 0 to 14.1 improvements, resp.  It is worth noting that rank-av. based ensemble evaluation calcd. for different ensemble sizes could not improve the results further.  We propose here that MD simulation can capture protein conformations representing the key interacting points of the receptor but less biased toward one specific chemotype.  These conformations are useful for the identification of a consensus binding site with improved performance in virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWkcgOyiLlm7Vg90H21EOLACvtfcHk0lhdnZ7Lemcy8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SgsLjO&md5=30e28d71e92c45ed5883a3d984645be7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci400087b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400087b%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DParagi%26aufirst%3DG.%26aulast%3DVass%26aufirst%3DM.%26aulast%3DJojart%26aufirst%3DB.%26aulast%3DBogar%26aufirst%3DF.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DThe%2520impact%2520of%2520molecular%2520dynamics%2520sampling%2520on%2520the%2520performance%2520of%2520virtual%2520screening%2520against%2520GPCRs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D2990%26epage%3D2999%26doi%3D10.1021%2Fci400087b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindorff-Larsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Long-timescale molecular dynamics simulations of protein structure and function</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">127</span><span class="refDoi"> DOI: 10.1016/j.sbi.2009.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2009.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19361980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvVSqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=120-127&author=J.+L.+Klepeisauthor=K.+Lindorff-Larsenauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Long-timescale+molecular+dynamics+simulations+of+protein+structure+and+function&doi=10.1016%2Fj.sbi.2009.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Long-timescale molecular dynamics simulations of protein structure and function</span></div><div class="casAuthors">Klepeis, John L.; Lindorff-Larsen, Kresten; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics simulations allow for at.-level characterization of biomol. processes such as the conformational transitions assocd. with protein function.  The computational demands of such simulations, however, have historically prevented them from reaching the microsecond and greater timescales on which these events often occur.  Recent advances in algorithms, software, and computer hardware have made microsecond-timescale simulations with tens of thousands of atoms practical, with millisecond-timescale simulations on the horizon.  This review outlines these advances in high-performance mol. dynamics simulation and discusses recent applications to studies of protein dynamics and function as well as exptl. validation of the underlying computational models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtFlQ4IqPPIbVg90H21EOLACvtfcHk0lhdnZ7Lemcy8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvVSqt70%253D&md5=7f40af4e683cc9aa24c49ef8eb7ae61f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2009.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2009.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DLindorff-Larsen%26aufirst%3DK.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DLong-timescale%2520molecular%2520dynamics%2520simulations%2520of%2520protein%2520structure%2520and%2520function%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.sbi.2009.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Buch, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giorgino, T.</span><span> </span><span class="NLM_article-title">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x"> (</span><span class="NLM_issue">25</span><span class="NLM_x">) </span> <span class="NLM_fpage">10184</span><span class="NLM_x">–</span> <span class="NLM_lpage">10189</span><span class="refDoi"> DOI: 10.1073/pnas.1103547108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1103547108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21646537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=10184-10189&issue=25&author=I.+Buchauthor=T.+Giorgino&title=Complete+reconstruction+of+an+enzyme-inhibitor+binding+process+by+molecular+dynamics+simulations&doi=10.1073%2Fpnas.1103547108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span></div><div class="casAuthors">Buch, Ignasi; Giorgino, Toni; De Fabritiis, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10184-10189, S10184/1-S10184/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The understanding of protein-ligand binding is of crit. importance for biomedical research, yet the process itself has been very difficult to study because of its intrinsically dynamic character.  Here, we have been able to quant. reconstruct the complete binding process of the enzyme-inhibitor complex trypsin-benzamidine by performing 495 mol. dynamics simulations of free ligand binding of 100 ns each, 187 of which produced binding events with an rmsd less than 2 Å compared to the crystal structure.  The binding paths obtained are able to capture the kinetic pathway of the inhibitor diffusing from solvent (SO) to the bound (S4) state passing through two metastable intermediate states S2 and S3.  Rather than directly entering the binding pocket the inhibitor appears to roll on the surface of the protein in its transition between S3 and the final binding pocket, whereas the transition between S2 and the bound pose requires rediffusion to S3.  An estn. of the std. free energy of binding gives ΔG° = -5.2 0.4 kcal/mol (cf. the exptl. value -6.2 kcal/mol), and a two-states kinetic model kon = (1.5 ± 0.2) × 108 M-1 s-1 and koff = (9.5 ± 3.3) × 104 s-1 for unbound to bound transitions.  The ability to reconstruct by simple diffusion the binding pathway of an enzyme-inhibitor binding process demonstrates the predictive power of unconventional high-throughput mol. simulations.  Moreover, the methodol. is directly applicable to other mol. systems and thus of general interest in biomedical and pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEgdoM19tFbbVg90H21EOLACvtfcHk0lhdnZ7Lemcy8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D&md5=40fdd0ec62da71c0f11d57731ee518a1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1103547108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1103547108%26sid%3Dliteratum%253Aachs%26aulast%3DBuch%26aufirst%3DI.%26aulast%3DGiorgino%26aufirst%3DT.%26atitle%3DComplete%2520reconstruction%2520of%2520an%2520enzyme-inhibitor%2520binding%2520process%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D25%26spage%3D10184%26epage%3D10189%26doi%3D10.1073%2Fpnas.1103547108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">How does a drug molecule find its target binding site?</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">9181</span><span class="NLM_x">–</span> <span class="NLM_lpage">9183</span><span class="refDoi"> DOI: 10.1021/ja202726y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja202726y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=9181-9183&author=Y.+Shanauthor=E.+T.+Kimauthor=M.+P.+Eastwoodauthor=R.+O.+Drorauthor=M.+A.+Seeligerauthor=D.+E.+Shaw&title=How+does+a+drug+molecule+find+its+target+binding+site%3F&doi=10.1021%2Fja202726y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">How Does a Drug Molecule Find Its Target Binding Site?</span></div><div class="casAuthors">Shan, Yibing; Kim, Eric T.; Eastwood, Michael P.; Dror, Ron O.; Seeliger, Markus A.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9181-9183</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although the thermodn. principles that control the binding of drug mols. to their protein targets are well understood, detailed exptl. characterization of the process by which such binding occurs has proven challenging.  We conducted relatively long, unguided mol. dynamics simulations in which a ligand (the cancer drug dasatinib or the kinase inhibitor PP1) was initially placed at a random location within a box that also contained a protein (Src kinase) to which that ligand was known to bind.  In several of these simulations, the ligand correctly identified its target binding site, forming a complex virtually identical to the crystallog. detd. bound structure.  The simulated trajectories provide a continuous, at.-level view of the entire binding process, revealing persistent and noteworthy intermediate conformations and shedding light on the role of water mols.  The technique we employed, which does not assume any prior knowledge of the binding site's location, may prove particularly useful in the development of allosteric inhibitors that target previously undiscovered binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfc8mwES5csLVg90H21EOLACvtfcHk0ljLtLihfbjVsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWgtb8%253D&md5=11847733dfeb27ed5ec1a565327b258c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja202726y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja202726y%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DEastwood%26aufirst%3DM.%2BP.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DHow%2520does%2520a%2520drug%2520molecule%2520find%2520its%2520target%2520binding%2520site%253F%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D9181%26epage%3D9183%26doi%3D10.1021%2Fja202726y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Decherchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berteotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">The ligand binding mechanism to purine nucleoside phosphorylase elucidated via molecular dynamics and machine learning</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">6155</span><span class="refDoi"> DOI: 10.1038/ncomms7155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fncomms7155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25625196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlaitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6155&author=S.+Decherchiauthor=A.+Berteottiauthor=G.+Bottegoniauthor=W.+Rocchiaauthor=A.+Cavalli&title=The+ligand+binding+mechanism+to+purine+nucleoside+phosphorylase+elucidated+via+molecular+dynamics+and+machine+learning&doi=10.1038%2Fncomms7155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The ligand binding mechanism to purine nucleoside phosphorylase elucidated via molecular dynamics and machine learning</span></div><div class="casAuthors">Decherchi, Sergio; Berteotti, Anna; Bottegoni, Giovanni; Rocchia, Walter; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6155</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The study of biomol. interactions between a drug and its biol. target is of paramount importance for the design of novel bioactive compds.  In this paper, we report on the use of mol. dynamics (MD) simulations and machine learning to study the binding mechanism of a transition state analog (DADMe-immucillin-H) to the purine nucleoside phosphorylase (PNP) enzyme.  Microsecond-long MD simulations allow us to observe several binding events, following different dynamical routes and reaching diverse binding configurations.  These simulations are used to est. kinetic and thermodn. quantities, such as kon and binding free energy, obtaining a good agreement with available exptl. data.  In addn., we advance a hypothesis for the slow-onset inhibition mechanism of DADMe-immucillin-H against PNP.  Combining extensive MD simulations with machine learning algorithms could therefore be a fruitful approach for capturing key aspects of drug-target recognition and binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYXSG2uDGjJbVg90H21EOLACvtfcHk0ljLtLihfbjVsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlaitbo%253D&md5=205b8ecc6a37f3caf9953c5393c2826f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncomms7155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7155%26sid%3Dliteratum%253Aachs%26aulast%3DDecherchi%26aufirst%3DS.%26aulast%3DBerteotti%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DThe%2520ligand%2520binding%2520mechanism%2520to%2520purine%2520nucleoside%2520phosphorylase%2520elucidated%2520via%2520molecular%2520dynamics%2520and%2520machine%2520learning%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6155%26doi%3D10.1038%2Fncomms7155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Exploring atomic resolution physiology on a femtosecond to millisecond timescale using molecular dynamics simulations</span> <span class="citation_source-journal">J. Gen. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">555</span><span class="NLM_x">–</span> <span class="NLM_lpage">562</span><span class="refDoi"> DOI: 10.1085/jgp.200910373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1085%2Fjgp.200910373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20513757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFensr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2010&pages=555-562&author=R.+O.+Drorauthor=M.+O.+Jensenauthor=D.+W.+Borhaniauthor=D.+E.+Shaw&title=Exploring+atomic+resolution+physiology+on+a+femtosecond+to+millisecond+timescale+using+molecular+dynamics+simulations&doi=10.1085%2Fjgp.200910373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring atomic resolution physiology on a femtosecond to millisecond timescale using molecular dynamics simulations</span></div><div class="casAuthors">Dror, Ron O.; Jensen, Morten Oe.; Borhani, David W.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Physiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">555-562</span>CODEN:
                <span class="NLM_cas:coden">JGPLAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-1295</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">A review.  This study deals with the mechanistic insights that physiologists might garner by complementing expts. with atomistic mol. dynamics (MD) simulations.  It discusses the strengths and limitations of MD as a tool for physiologists.  MD simulations are important among physiologists because these allow access to a spatiotemporal domain that is difficult to probe exptl.  Simulations can be particularly valuable for membrane proteins, for which exptl. characterization of structural dynamics tends to be challenging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4vfNL81MfN7Vg90H21EOLACvtfcHk0ljLtLihfbjVsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFensr8%253D&md5=75e55def06d3831daab98511452be286</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1085%2Fjgp.200910373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1085%252Fjgp.200910373%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DJensen%26aufirst%3DM.%2BO.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DExploring%2520atomic%2520resolution%2520physiology%2520on%2520a%2520femtosecond%2520to%2520millisecond%2520timescale%2520using%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Gen.%2520Physiol.%26date%3D2010%26volume%3D135%26spage%3D555%26epage%3D562%26doi%3D10.1085%2Fjgp.200910373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Lane, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">To milliseconds and beyond: challenges in the simulation of protein folding</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.sbi.2012.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2012.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23237705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=58-65&author=T.+J.+Laneauthor=D.+Shuklaauthor=K.+A.+Beauchampauthor=V.+S.+Pande&title=To+milliseconds+and+beyond%3A+challenges+in+the+simulation+of+protein+folding&doi=10.1016%2Fj.sbi.2012.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">To milliseconds and beyond: challenges in the simulation of protein folding</span></div><div class="casAuthors">Lane, Thomas J.; Shukla, Diwakar; Beauchamp, Kyle A.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-65</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Quant. accurate all-atom mol. dynamics (MD) simulations of protein folding have long been considered a holy grail of computational biol.  Due to the large system sizes and long timescales involved, such a pursuit was for many years computationally intractable.  Further, sufficiently accurate forcefields needed to be developed in order to realistically model folding.  This decade, however, saw the first reports of folding simulations describing kinetics on the order of milliseconds, placing many proteins firmly within reach of these methods.  Progress in sampling and forcefield accuracy, however, presents a new challenge: how to turn huge MD datasets into scientific understanding.  Here, we review recent progress in MD simulation techniques and show how the vast datasets generated by such techniques present new challenges for anal.  We critically discuss the state of the art, including reaction coordinate and Markov state model (MSM) methods, and provide a perspective for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphYpDSTwX8JrVg90H21EOLACvtfcHk0lhP0uwMOqOI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLjF&md5=76f625b813b7c85cfbcabac6659155cd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2012.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2012.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DLane%26aufirst%3DT.%2BJ.%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DTo%2520milliseconds%2520and%2520beyond%253A%2520challenges%2520in%2520the%2520simulation%2520of%2520protein%2520folding%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2013%26volume%3D23%26spage%3D58%26epage%3D65%26doi%3D10.1016%2Fj.sbi.2012.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Lu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span> </span><span class="NLM_article-title">Drug-target residence time: critical information for lead optimization</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.06.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.cbpa.2010.06.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20663707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=467-474&author=H.+Luauthor=P.+J.+Tonge&title=Drug-target+residence+time%3A+critical+information+for+lead+optimization&doi=10.1016%2Fj.cbpa.2010.06.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time: Critical information for lead optimization</span></div><div class="casAuthors">Lu, Hao; Tonge, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Failure due to poor in vivo efficacy is a primary contributor to attrition during the development of new chemotherapeutics.  Lead optimization programs that in their quest for efficacy focus solely on improving the affinity of drug-target binding are flawed, since this approach ignores the fluctuations in drug concn. that occur in vivo.  Instead the lifetime of the drug-target complex must also be considered, since drugs only act when they are bound to their targets.  Consequently, to improve the correlation between the in vitro and in vivo activity of drugs, measurements of drug-target residence time must be incorporated into the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk2kN_txuMLVg90H21EOLACvtfcHk0lhP0uwMOqOI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D&md5=3189f56032106db858cfb4269d3fc11f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.06.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.06.176%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DDrug-target%2520residence%2520time%253A%2520critical%2520information%2520for%2520lead%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D467%26epage%3D474%26doi%3D10.1016%2Fj.cbpa.2010.06.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Abrams, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussi, G.</span><span> </span><span class="NLM_article-title">Enhanced sampling in molecular dynamics using metadynamics, replica-exchange, and temperature-acceleration</span> <span class="citation_source-journal">Entropy</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.3390/e16010163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.3390%2Fe16010163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptV2htbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=163-199&author=C.+Abramsauthor=G.+Bussi&title=Enhanced+sampling+in+molecular+dynamics+using+metadynamics%2C+replica-exchange%2C+and+temperature-acceleration&doi=10.3390%2Fe16010163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced sampling in molecular dynamics using metadynamics, replica-exchange, and temperature-acceleration</span></div><div class="casAuthors">Abrams, Cameron; Bussi, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Entropy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">163-199, 37 pp.</span>CODEN:
                <span class="NLM_cas:coden">ENTRFG</span>;
        ISSN:<span class="NLM_cas:issn">1099-4300</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">We review a selection of methods for performing enhanced sampling in mol. dynamics simulations.  We consider methods based on collective variable biasing and on tempering, and offer both historical and contemporary perspectives.  In collective-variable biasing, we first discuss methods stemming from thermodn. integration that use mean force biasing, including the adaptive biasing force algorithm and temp. acceleration.  We then turn to methods that use bias potentials, including umbrella sampling and metadynamics.  We next consider parallel tempering and replica-exchange methods.  We conclude with a brief presentation of some combination methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgwp9LGvawa7Vg90H21EOLACvtfcHk0lhP0uwMOqOI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptV2htbg%253D&md5=7e27e14ffdacdd6de1e89f512b0f96ce</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.3390%2Fe16010163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fe16010163%26sid%3Dliteratum%253Aachs%26aulast%3DAbrams%26aufirst%3DC.%26aulast%3DBussi%26aufirst%3DG.%26atitle%3DEnhanced%2520sampling%2520in%2520molecular%2520dynamics%2520using%2520metadynamics%252C%2520replica-exchange%252C%2520and%2520temperature-acceleration%26jtitle%3DEntropy%26date%3D2014%26volume%3D16%26spage%3D163%26epage%3D199%26doi%3D10.3390%2Fe16010163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravimohan, C.</span><span> </span><span class="NLM_article-title">Monte-carlo simulation of differences in free-energies of hydration</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">3050</span><span class="NLM_x">–</span> <span class="NLM_lpage">3054</span><span class="refDoi"> DOI: 10.1063/1.449208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.449208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL2MXls1Oiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1985&pages=3050-3054&author=W.+L.+Jorgensenauthor=C.+Ravimohan&title=Monte-carlo+simulation+of+differences+in+free-energies+of+hydration&doi=10.1063%2F1.449208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Monte Carlo simulation of differences in free energies of hydration</span></div><div class="casAuthors">Jorgensen, William L.; Ravimohan, C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3050-4</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Perturbation theory was applied to calc. the relative free energies of hydration of methanol and ethane in dil. soln..  Only 2 or 3 Monte Carlo simulations using double-wide sampling are necessary to obtain results with high precision.  The small statistical uncertainty in the computed change in free energy of hydration and the agreement with exptl. thermodn. data are most encouraging for application of the procedure to a wide range of problems.  Structural effects accompanying the mutation of methanol to ethane in water are also discussed; H bonding to the solute is essentially eliminated by only a 25% redn. in the at. charges of MeOH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUM9poULbA5rVg90H21EOLACvtfcHk0lhP0uwMOqOI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXls1Oiu7w%253D&md5=1d2760883bcd23620d10e98a2295d2e5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1063%2F1.449208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.449208%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DRavimohan%26aufirst%3DC.%26atitle%3DMonte-carlo%2520simulation%2520of%2520differences%2520in%2520free-energies%2520of%2520hydration%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1985%26volume%3D83%26spage%3D3050%26epage%3D3054%26doi%3D10.1063%2F1.449208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, L. L.</span><span> </span><span class="NLM_article-title">Perspective on free-energy perturbation calculations for chemical equilibria</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1021/ct800011m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct800011m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=869-876&author=W.+L.+Jorgensenauthor=L.+L.+Thomas&title=Perspective+on+free-energy+perturbation+calculations+for+chemical+equilibria&doi=10.1021%2Fct800011m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective on Free-Energy Perturbation Calculations for Chemical Equilibria</span></div><div class="casAuthors">Jorgensen, William L.; Thomas, Laura L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-876</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An overview has been provided on the history of free-energy perturbation (FEP) calcns. and their application to computation of relative free energies of solvation.  In view of the expanding role of FEP calcns. for pursuing a wide range of problems in chem. and biochem., including mol. design, it is important to establish reliable protocols for their execution.  For 30 interconversions of substituted benzenes, which represent prototypical cases for mol. design, alternative sampling procedures and nos. of FEP windows were considered.  Simple overlap sampling with 11 or 15 windows and double-wide sampling with 14 windows emerged as viable procedures for such routine FEP calcns.; the present protocols with these sampling methods yield statistical uncertainties of 0.2 kcal/mol for free-energy differences.  The slightly greater accuracy obtainable with 21 to 25 windows does not clearly justify the increased costs, while overlap sampling with 6 or 8 windows may be useful for preliminary exploration of large nos. of mol. changes.  Application of the power-series expansion of the Zwanzig equation was also tested.  Use of the first-order term is generally reliable, while inclusion of the slowly convergent, second-order fluctuation term causes deterioration in the results for strongly hydrogen-bonded solutes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqukDgvYtLny7Vg90H21EOLACvtfcHk0lg_-bmTcvj8Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D&md5=4f6cd47142dbbcd51e94aedd2d878538</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fct800011m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct800011m%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DThomas%26aufirst%3DL.%2BL.%26atitle%3DPerspective%2520on%2520free-energy%2520perturbation%2520calculations%2520for%2520chemical%2520equilibria%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D869%26epage%3D876%26doi%3D10.1021%2Fct800011m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Torrie, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valleau, J. P.</span><span> </span><span class="NLM_article-title">Nonphysical sampling distributions in monte carlo free-energy estimation: umbrella sampling</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90121-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0021-9991%2877%2990121-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=187-199&author=G.+M.+Torrieauthor=J.+P.+Valleau&title=Nonphysical+sampling+distributions+in+monte+carlo+free-energy+estimation%3A+umbrella+sampling&doi=10.1016%2F0021-9991%2877%2990121-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990121-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990121-8%26sid%3Dliteratum%253Aachs%26aulast%3DTorrie%26aufirst%3DG.%2BM.%26aulast%3DValleau%26aufirst%3DJ.%2BP.%26atitle%3DNonphysical%2520sampling%2520distributions%2520in%2520monte%2520carlo%2520free-energy%2520estimation%253A%2520umbrella%2520sampling%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D187%26epage%3D199%26doi%3D10.1016%2F0021-9991%2877%2990121-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Sugita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, Y.</span><span> </span><span class="NLM_article-title">Replica-exchange molecular dynamics method for protein folding</span> <span class="citation_source-journal">Chem. Phys. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">314</span><span class="NLM_x">, </span> <span class="NLM_fpage">141</span><span class="NLM_x">–</span> <span class="NLM_lpage">151</span><span class="refDoi"> DOI: 10.1016/S0009-2614(99)01123-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0009-2614%2899%2901123-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1MXotVKrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=1999&pages=141-151&author=Y.+Sugitaauthor=Y.+Okamoto&title=Replica-exchange+molecular+dynamics+method+for+protein+folding&doi=10.1016%2FS0009-2614%2899%2901123-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Replica-exchange molecular dynamics method for protein folding</span></div><div class="casAuthors">Sugita, Y.; Okamoto, Y.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">141-151</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We have developed a formulation for mol. dynamics algorithm for the replica-exchange method.  The effectiveness of the method for the protein-folding problem is tested with the penta-peptide Met-enkephalin.  The method can overcome the multiple-min. problem by exchanging non-interacting replicas of the system at several temps.  From only one simulation run, one can obtain probability distributions in canonical ensemble for a wide temp. range using multiple-histogram re-weighting techniques, which allows the calcn. of any thermodn. quantity as a function of temp. in that range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4vjM65fDLULVg90H21EOLACvtfcHk0lg_-bmTcvj8Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXotVKrsLc%253D&md5=0fec0ff81ca7806c1e1ac29e5f50ce19</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2901123-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252901123-9%26sid%3Dliteratum%253Aachs%26aulast%3DSugita%26aufirst%3DY.%26aulast%3DOkamoto%26aufirst%3DY.%26atitle%3DReplica-exchange%2520molecular%2520dynamics%2520method%2520for%2520protein%2520folding%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D314%26spage%3D141%26epage%3D151%26doi%3D10.1016%2FS0009-2614%2899%2901123-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Escaping free-energy minima</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">12562</span><span class="NLM_x">–</span> <span class="NLM_lpage">12566</span><span class="refDoi"> DOI: 10.1073/pnas.202427399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.202427399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12271136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvFGiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=12562-12566&author=A.+Laioauthor=M.+Parrinello&title=Escaping+free-energy+minima&doi=10.1073%2Fpnas.202427399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Escaping free-energy minima</span></div><div class="casAuthors">Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">12562-12566</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We introduce a powerful method for exploring the properties of the multidimensional free energy surfaces (FESs) of complex many-body systems by means of coarse-grained non-Markovian dynamics in the space defined by a few collective coordinates.  A characteristic feature of these dynamics is the presence of a history-dependent potential term that, in time, fills the min. in the FES, allowing the efficient exploration and accurate detn. of the FES as a function of the collective coordinates.  We demonstrate the usefulness of this approach in the case of the dissocn. of a NaCl mol. in water and in the study of the conformational changes of a dialanine in soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoTAY5TRy70LVg90H21EOLACvtfcHk0lg_-bmTcvj8Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvFGiurc%253D&md5=48d5bc7436f3ef9d78369671e70fa608</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202427399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202427399%26sid%3Dliteratum%253Aachs%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DEscaping%2520free-energy%2520minima%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D12562%26epage%3D12566%26doi%3D10.1073%2Fpnas.202427399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Isralewitz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">Steered molecular dynamics and mechanical functions of proteins</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span><span class="refDoi"> DOI: 10.1016/S0959-440X(00)00194-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0959-440X%2800%2900194-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11297932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsVyisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=224-230&author=B.+Isralewitzauthor=M.+Gaoauthor=K.+Schulten&title=Steered+molecular+dynamics+and+mechanical+functions+of+proteins&doi=10.1016%2FS0959-440X%2800%2900194-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Steered molecular dynamics and mechanical functions of proteins</span></div><div class="casAuthors">Isralewitz, Barry; Gao, Mu; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">224-230</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  At. force microscopy of single mols., steered mol. dynamics and the theory of stochastic processes have established a new field that investigates mech. functions of proteins, such as ligand-receptor binding/unbinding and elasticity of muscle proteins during stretching.  The combination of these methods yields information on the energy landscape that controls mech. function and on the force-bearing components of proteins, as well as on the underlying phys. mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMPwb3lv1-ibVg90H21EOLACvtfcHk0lgj42OWclySfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsVyisro%253D&md5=09b7a191f8ffa19d316ec0f4421c3731</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2800%2900194-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252800%252900194-9%26sid%3Dliteratum%253Aachs%26aulast%3DIsralewitz%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DSteered%2520molecular%2520dynamics%2520and%2520mechanical%2520functions%2520of%2520proteins%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D224%26epage%3D230%26doi%3D10.1016%2FS0959-440X%2800%2900194-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Grubmuller, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heymann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavan, P.</span><span> </span><span class="NLM_article-title">Ligand binding: molecular mechanics calculation of the streptavidin biotin rupture force</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">997</span><span class="NLM_x">–</span> <span class="NLM_lpage">999</span><span class="refDoi"> DOI: 10.1126/science.271.5251.997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.271.5251.997" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=997-999&author=H.+Grubmullerauthor=B.+Heymannauthor=P.+Tavan&title=Ligand+binding%3A+molecular+mechanics+calculation+of+the+streptavidin+biotin+rupture+force&doi=10.1126%2Fscience.271.5251.997"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fscience.271.5251.997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.271.5251.997%26sid%3Dliteratum%253Aachs%26aulast%3DGrubmuller%26aufirst%3DH.%26aulast%3DHeymann%26aufirst%3DB.%26aulast%3DTavan%26aufirst%3DP.%26atitle%3DLigand%2520binding%253A%2520molecular%2520mechanics%2520calculation%2520of%2520the%2520streptavidin%2520biotin%2520rupture%2520force%26jtitle%3DScience%26date%3D1996%26volume%3D271%26spage%3D997%26epage%3D999%26doi%3D10.1126%2Fscience.271.5251.997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Hamelberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mongan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">11919</span><span class="NLM_x">–</span> <span class="NLM_lpage">11929</span><span class="refDoi"> DOI: 10.1063/1.1755656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.1755656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15268227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvVant7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2004&pages=11919-11929&author=D.+Hamelbergauthor=J.+Monganauthor=J.+A.+McCammon&title=Accelerated+molecular+dynamics%3A+a+promising+and+efficient+simulation+method+for+biomolecules&doi=10.1063%2F1.1755656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules</span></div><div class="casAuthors">Hamelberg, Donald; Mongan, John; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11919-11929</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Many interesting dynamic properties of biol. mols. cannot be simulated directly using mol. dynamics because of nanosecond time scale limitations.  These systems are trapped in potential energy min. with high free energy barriers for large nos. of computational steps.  The dynamic evolution of many mol. systems occurs through a series of rare events as the system moves from one potential energy basin to another.  Therefore, we have proposed a robust bias potential function that can be used in an efficient accelerated mol. dynamics approach to simulate the transition of high energy barriers without any advance knowledge of the location of either the potential energy wells or saddle points.  In this method, the potential energy landscape is altered by adding a bias potential to the true potential such that the escape rates from potential wells are enhanced, which accelerates and extends the time scale in mol. dynamics simulations.  Our definition of the bias potential echoes the underlying shape of the potential energy landscape on the modified surface, thus allowing for the potential energy min. to be well defined, and hence properly sampled during the simulation.  We have shown that our approach, which can be extended to biomols., samples the conformational space more efficiently than normal mol. dynamics simulations, and converges to the correct canonical distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFSt4zsfWGVbVg90H21EOLACvtfcHk0lgj42OWclySfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvVant7w%253D&md5=f7ee967e10493f27a944bed8cd17c640</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1063%2F1.1755656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1755656%26sid%3Dliteratum%253Aachs%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DMongan%26aufirst%3DJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAccelerated%2520molecular%2520dynamics%253A%2520a%2520promising%2520and%2520efficient%2520simulation%2520method%2520for%2520biomolecules%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2004%26volume%3D120%26spage%3D11919%26epage%3D11929%26doi%3D10.1063%2F1.1755656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Faradjian, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elber, R.</span><span> </span><span class="NLM_article-title">Computing time scales from reaction coordinates by milestoning</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">10880</span><span class="NLM_x">–</span> <span class="NLM_lpage">10889</span><span class="refDoi"> DOI: 10.1063/1.1738640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.1738640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15268118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVCqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2004&pages=10880-10889&author=A.+K.+Faradjianauthor=R.+Elber&title=Computing+time+scales+from+reaction+coordinates+by+milestoning&doi=10.1063%2F1.1738640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Computing time scales from reaction coordinates by milestoning</span></div><div class="casAuthors">Faradjian, Anton K.; Elber, Ron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10880-10889</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">An algorithm is presented to compute time scales of complex processes following predetd. milestones along a reaction coordinate.  A non-Markovian hopping mechanism is assumed and constructed from underlying microscopic dynamics.  General anal. anal., a pedagogical example, and numerical solns. of the non-Markovian model are presented.  No assumption is made in the theor. derivation on the type of microscopic dynamics along the reaction coordinate.  However, the detailed calcns. are for Brownian dynamics in which the velocities are uncorrelated in time (but spatial memory remains).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdLFuyCZqyWrVg90H21EOLACvtfcHk0lgj42OWclySfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVCqt74%253D&md5=cfa2d2ed0611f22293240622883a4925</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1063%2F1.1738640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1738640%26sid%3Dliteratum%253Aachs%26aulast%3DFaradjian%26aufirst%3DA.%2BK.%26aulast%3DElber%26aufirst%3DR.%26atitle%3DComputing%2520time%2520scales%2520from%2520reaction%2520coordinates%2520by%2520milestoning%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2004%26volume%3D120%26spage%3D10880%26epage%3D10889%26doi%3D10.1063%2F1.1738640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Bolhuis, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dellago, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geissler, P. L.</span><span> </span><span class="NLM_article-title">Transition path sampling: Throwing ropes over rough mountain passes, in the dark</span> <span class="citation_source-journal">Annu. Rev. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">318</span><span class="refDoi"> DOI: 10.1146/annurev.physchem.53.082301.113146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1146%2Fannurev.physchem.53.082301.113146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11972010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Ois7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2002&pages=291-318&author=P.+G.+Bolhuisauthor=D.+Chandlerauthor=C.+Dellagoauthor=P.+L.+Geissler&title=Transition+path+sampling%3A+Throwing+ropes+over+rough+mountain+passes%2C+in+the+dark&doi=10.1146%2Fannurev.physchem.53.082301.113146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Transition path sampling: throwing ropes over rough mountain passes, in the dark</span></div><div class="casAuthors">Bolhuis, Peter G.; Chandler, David; Dellago, Christoph; Geissler, Phillip L.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Physical Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">291-318</span>CODEN:
                <span class="NLM_cas:coden">ARPLAP</span>;
        ISSN:<span class="NLM_cas:issn">0066-426X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review is given of the concepts and methods of transition path sampling.  These methods allow computational studies of rare events without requiring prior knowledge of mechanisms, reaction coordinates, and transition states.  Based upon a statistical mechanics of trajectory space, they provide a perspective with which time dependent phenomena, even for systems driven far from equil., can be examd. with the same types of importance sampling tools that in the past have been applied so successfully to static equil. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_Z72WTpw67Vg90H21EOLACvtfcHk0lgzQD7JSrvnmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Ois7g%253D&md5=fbf79b52e751dfd87a73c345b9581898</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1146%2Fannurev.physchem.53.082301.113146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.physchem.53.082301.113146%26sid%3Dliteratum%253Aachs%26aulast%3DBolhuis%26aufirst%3DP.%2BG.%26aulast%3DChandler%26aufirst%3DD.%26aulast%3DDellago%26aufirst%3DC.%26aulast%3DGeissler%26aufirst%3DP.%2BL.%26atitle%3DTransition%2520path%2520sampling%253A%2520Throwing%2520ropes%2520over%2520rough%2520mountain%2520passes%252C%2520in%2520the%2520dark%26jtitle%3DAnnu.%2520Rev.%2520Phys.%2520Chem.%26date%3D2002%26volume%3D53%26spage%3D291%26epage%3D318%26doi%3D10.1146%2Fannurev.physchem.53.082301.113146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Ellingson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baudry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span> </span><span class="NLM_article-title">Multi-conformer ensemble docking to difficult protein targets</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1021/jp506511p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp506511p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWjsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=1026-1034&author=S.+R.+Ellingsonauthor=Y.+Miaoauthor=J.+Baudryauthor=J.+C.+Smith&title=Multi-conformer+ensemble+docking+to+difficult+protein+targets&doi=10.1021%2Fjp506511p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Conformer Ensemble Docking to Difficult Protein Targets</span></div><div class="casAuthors">Ellingson, Sally R.; Miao, Yinglong; Baudry, Jerome; Smith, Jeremy C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1026-1034</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large-scale ensemble docking was studied using five proteins from the Directory of Useful Decoys (DUD, dud.docking.org) for which docking to crystal structures proved difficult.  Mol. dynamics trajectories were produced for each protein and an ensemble of representative conformational structures extd. from the trajectories.  Docking calcns. were performed on these selected simulation structures and ensemble-based enrichment factors compared with those obtained using docking in crystal structures of the same protein targets or random selection of compds.  Simulation-derived snapshots are found with improved enrichment factors that increase the chem. diversity of docking hits for four of the five selected proteins.  A combination of all the docking results obtained from mol. dynamics simulation followed by selection of top-ranking compds. appears to be an effective strategy for increasing the no. and diversity of hits when using docking to screen large libraries of chems. against difficult protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8lJ2vsbEsz7Vg90H21EOLACvtfcHk0lgzQD7JSrvnmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWjsb3E&md5=b421fbb1a3b7fc57d06e4645476aabb8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjp506511p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp506511p%26sid%3Dliteratum%253Aachs%26aulast%3DEllingson%26aufirst%3DS.%2BR.%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DBaudry%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26atitle%3DMulti-conformer%2520ensemble%2520docking%2520to%2520difficult%2520protein%2520targets%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2015%26volume%3D119%26spage%3D1026%26epage%3D1034%26doi%3D10.1021%2Fjp506511p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Surade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structural biology and drug discovery of difficult targets: the limits of ligandability</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.chembiol.2011.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22284353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=42-50&author=S.+Suradeauthor=T.+L.+Blundell&title=Structural+biology+and+drug+discovery+of+difficult+targets%3A+the+limits+of+ligandability&doi=10.1016%2Fj.chembiol.2011.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology and Drug Discovery of Difficult Targets: The Limits of Ligandability</span></div><div class="casAuthors">Surade, Sachin; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, researchers in the pharmaceutical industry and academia have made retrospective analyses of successful drug campaigns in order to establish "rules" to guide the selection of new target proteins.  They have identified features that are considered undesirable and some that make targets "unligandable.".  This review focuses on the factors that make targets difficult: featureless binding sites, the lack of hydrogen-bond donors and acceptors, the presence of metal ions, the need for adaptive changes in conformation, and the lipophilicity of residues at the protein-ligand interface.  Protein-protein interfaces of multiprotein assemblies share many of these undesirable features, although those that involve concerted binding and folding in their assembly have better defined pockets or grooves, and these can provide opportunities for identifying hits and for lead optimization.  In some protein-protein interfaces conformational changes-often involving rearrangement of large side chains such as those of tyrosine, tryptophan, or arginine-are required to configure an appropriate binding site, and this may require tethering of the ligands until higher affinity is achieved.  In many enzymes, larger conformational rearrangements are required to form the binding site, and these can make fragment-based approaches particularly difficult.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqCA_OH6VRoLVg90H21EOLACvtfcHk0lgzQD7JSrvnmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D&md5=646b6cad327b44785848375383ca90ea</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520and%2520drug%2520discovery%2520of%2520difficult%2520targets%253A%2520the%2520limits%2520of%2520ligandability%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D42%26epage%3D50%26doi%3D10.1016%2Fj.chembiol.2011.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Bowman, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolin, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maguire, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marqusee, S.</span><span> </span><span class="NLM_article-title">Discovery of multiple hidden allosteric sites by combining markov state models and experiments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">2734</span><span class="NLM_x">–</span> <span class="NLM_lpage">2739</span><span class="refDoi"> DOI: 10.1073/pnas.1417811112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1417811112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25730859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=2734-2739&author=G.+R.+Bowmanauthor=E.+R.+Bolinauthor=K.+M.+Hartauthor=B.+C.+Maguireauthor=S.+Marqusee&title=Discovery+of+multiple+hidden+allosteric+sites+by+combining+markov+state+models+and+experiments&doi=10.1073%2Fpnas.1417811112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of multiple hidden allosteric sites by combining Markov state models and experiments</span></div><div class="casAuthors">Bowman, Gregory R.; Bolin, Eric R.; Hart, Kathryn M.; Maguire, Brendan C.; Marqusee, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2734-2739</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The discovery of drug-like mols. that bind pockets in proteins that are not present in crystallog. structures yet exert allosteric control over activity has generated great interest in designing pharmaceuticals that exploit allosteric effects.  However, there have only been a small no. of successes, so the therapeutic potential of these pockets--called hidden allosteric sites--remains unclear.  One challenge for assessing their utility is that rational drug design approaches require foreknowledge of the target site, but most hidden allosteric sites are only discovered when a small mol. is found to stabilize them.  The authors present a means of decoupling the identification of hidden allosteric sites from the discovery of drugs that bind them by drawing on new developments in Markov state modeling that provide unprecedented access to microsecond- to millisecond-timescale fluctuations of a protein's structure.  Visualizing these fluctuations allows the authors to identify potential hidden allosteric sites, which the authors then test via thiol labeling expts.  Application of these methods reveals multiple hidden allosteric sites in an important antibiotic target-TEM-1 β-lactamase.  This result supports the hypothesis that there are many as yet undiscovered hidden allosteric sites and suggests the authors' methodol. can identify such sites, providing a starting point for future drug design efforts.  More generally, the authors' results demonstrate the power of using Markov state models to guide expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY4EqjsensDLVg90H21EOLACvtfcHk0ljDyD-pD18MfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D&md5=f20db19a2f0f13679bc64f4616c856e7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417811112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417811112%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DBolin%26aufirst%3DE.%2BR.%26aulast%3DHart%26aufirst%3DK.%2BM.%26aulast%3DMaguire%26aufirst%3DB.%2BC.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520multiple%2520hidden%2520allosteric%2520sites%2520by%2520combining%2520markov%2520state%2520models%2520and%2520experiments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D2734%26epage%3D2739%26doi%3D10.1073%2Fpnas.1417811112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span> </span><span class="NLM_article-title">Recent computational advances in the identification of allosteric sites in proteins</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1595</span><span class="NLM_x">–</span> <span class="NLM_lpage">1600</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2014.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25107670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1595-1600&author=S.+Luauthor=W.+Huangauthor=J.+Zhang&title=Recent+computational+advances+in+the+identification+of+allosteric+sites+in+proteins&doi=10.1016%2Fj.drudis.2014.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Recent computational advances in the identification of allosteric sites in proteins</span></div><div class="casAuthors">Lu, Shaoyong; Huang, Wenkang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1595-1600</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allosteric modulators have the potential to fine-tune protein functional activity.  Therefore, the targeting of allosteric sites, as a strategy in drug design, is gaining increasing attention.  Currently, it is not trivial to find and characterize new allosteric sites by exptl. approaches.  Alternatively, computational approaches are useful in helping researchers analyze and select potential allosteric sites for drug discovery.  Here, we review state-of-the-art computational approaches directed at predicting putative allosteric sites in proteins, along with examples of successes in identifying allosteric sites utilizing these methods.  We also discuss the challenges in developing reliable methods for predicting allosteric sites and tactics to resolve demanding tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaspE0Qbe47Vg90H21EOLACvtfcHk0ljDyD-pD18MfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F&md5=fbaefacf51bee99d30fb882f0f04a47d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRecent%2520computational%2520advances%2520in%2520the%2520identification%2520of%2520allosteric%2520sites%2520in%2520proteins%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1595%26epage%3D1600%26doi%3D10.1016%2Fj.drudis.2014.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Lodola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">The increasing role of QM/MM in drug discovery</span> <span class="citation_source-journal">Adv. Protein Chem. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">362</span><span class="refDoi"> DOI: 10.1016/B978-0-12-398312-1.00011-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FB978-0-12-398312-1.00011-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22607760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Kmu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2012&pages=337-362&author=A.+Lodolaauthor=M.+De+Vivo&title=The+increasing+role+of+QM%2FMM+in+drug+discovery&doi=10.1016%2FB978-0-12-398312-1.00011-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">The increasing role of QM/MM in drug discovery</span></div><div class="casAuthors">Lodola, Alessio; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Protein Chemistry and Structural Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">Structural and Mechanistic Enzymology</span>),
    <span class="NLM_cas:pages">337-362</span>CODEN:
                <span class="NLM_cas:coden">APCSG7</span>;
        ISSN:<span class="NLM_cas:issn">1876-1623</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Since its first appearance in 1976, the quantum mechanics/mol. mechanics (QM/MM) approach has mostly been used to study the chem. reactions of enzymes, which are frequently the target of drug discovery programs.  In principle, a detailed understanding of the enzymic mechanism should help researchers to design a potent enzyme inhibitor or new drug.  However, QM/MM has not yet had a widespread impact on structure-based drug design.  This is mostly due to its high computational cost.  We expect this to change with the recent and extraordinary increases in computational power, and with the availability of more efficient algorithms for QM/MM calcns.  Here, we report on some representative examples of QM/MM studies, including our own research, of pharmaceutically relevant enzymes, such as RNase H and fatty acid amide hydrolase (FAAH).  We aim to show how QM/MM has traditionally been used to study enzymic catalysis.  In this regard, we discuss its potential to become a routinely used drug design tool.  To support this, we also discuss selected computational studies where QM/MM insights have been helpful in improving the potency of covalent inhibitors of FAAH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8JGGBuNrFw7Vg90H21EOLACvtfcHk0ljDyD-pD18MfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Kmu7nL&md5=f3712a42a86ad8c784d03933b0306aff</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-398312-1.00011-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-398312-1.00011-1%26sid%3Dliteratum%253Aachs%26aulast%3DLodola%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DThe%2520increasing%2520role%2520of%2520QM%252FMM%2520in%2520drug%2520discovery%26jtitle%3DAdv.%2520Protein%2520Chem.%2520Struct.%2520Biol.%26date%3D2012%26volume%3D87%26spage%3D337%26epage%3D362%26doi%3D10.1016%2FB978-0-12-398312-1.00011-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothlisberger, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Computational insights into function and inhibition of fatty acid amide hydrolase</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.ejmech.2014.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25240419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2015&pages=15-26&author=G.+Palermoauthor=U.+Rothlisbergerauthor=A.+Cavalliauthor=M.+De+Vivo&title=Computational+insights+into+function+and+inhibition+of+fatty+acid+amide+hydrolase&doi=10.1016%2Fj.ejmech.2014.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Computational insights into function and inhibition of fatty acid amide hydrolase</span></div><div class="casAuthors">Palermo, Giulia; Rothlisberger, Ursula; Cavalli, Andrea; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-26</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The Fatty Acid Amide Hydrolase (FAAH) enzyme is a membrane-bound serine hydrolase responsible for the deactivating hydrolysis of a family of naturally occurring fatty acid amides.  FAAH is a crit. enzyme of the endocannabinoid system, being mainly responsible for regulating the level of its main cannabinoid substrate anandamide.  For this reason, pharmacol. inhibition of FAAH, which increases the level of endogenous anandamide, is a promising strategy to cure a variety of diseases including pain, inflammation, and cancer.Much structural, mutagenesis, and kinetic data on FAAH has been generated over the last couple of decades.  This has prompted several informative computational investigations to elucidate, at the at.-level, mechanistic details on catalysis and inhibition of this pharmaceutically relevant enzyme.  Here, we review how these computational studies - based on classical mol. dynamics, full quantum mechanics, and hybrid QM/MM methods - have clarified the binding and reactivity of some relevant substrates and inhibitors of FAAH.  We also discuss the exptl. implications of these computational insights, which have provided a thoughtful elucidation of the complex phys. and chem. steps of the enzymic mechanism of FAAH.  Finally, we discuss how computations have been helpful for building structure-activity relationships of potent FAAH inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-xquSauxYo7Vg90H21EOLACvtfcHk0ljDyD-pD18MfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOls7jJ&md5=5dd76bb3eda21ebdfe66e956a46f7858</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DRothlisberger%26aufirst%3DU.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DComputational%2520insights%2520into%2520function%2520and%2520inhibition%2520of%2520fatty%2520acid%2520amide%2520hydrolase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D91%26spage%3D15%26epage%3D26%26doi%3D10.1016%2Fj.ejmech.2014.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carloni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span> </span><span class="NLM_article-title">Target-related applications of first principles quantum chemical methods in drug design</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">3497</span><span class="NLM_x">–</span> <span class="NLM_lpage">3519</span><span class="refDoi"> DOI: 10.1021/cr050579p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr050579p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=3497-3519&author=A.+Cavalliauthor=P.+Carloniauthor=M.+Recanatini&title=Target-related+applications+of+first+principles+quantum+chemical+methods+in+drug+design&doi=10.1021%2Fcr050579p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fcr050579p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050579p%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DCarloni%26aufirst%3DP.%26aulast%3DRecanatini%26aufirst%3DM.%26atitle%3DTarget-related%2520applications%2520of%2520first%2520principles%2520quantum%2520chemical%2520methods%2520in%2520drug%2520design%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D3497%26epage%3D3519%26doi%3D10.1021%2Fcr050579p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Cornell, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cieplak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gould, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spellmeyer, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. a.</span><span> </span><span class="NLM_article-title">A 2nd generation force-field for the simulation of proteins, nucleic-acids, and organic-molecules</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">5179</span><span class="NLM_x">–</span> <span class="NLM_lpage">5197</span><span class="refDoi"> DOI: 10.1021/ja00124a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00124a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=5179-5197&author=W.+D.+Cornellauthor=P.+Cieplakauthor=C.+I.+Baylyauthor=I.+R.+Gouldauthor=K.+M.+Merzauthor=D.+M.+Fergusonauthor=D.+C.+Spellmeyerauthor=T.+Foxauthor=J.+W.+Caldwellauthor=P.+a.+Kollman&title=A+2nd+generation+force-field+for+the+simulation+of+proteins%2C+nucleic-acids%2C+and+organic-molecules&doi=10.1021%2Fja00124a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A Second Generation Force Field for the Simulation of Proteins, Nucleic Acids, and Organic Molecules</span></div><div class="casAuthors">Cornell, Wendy D.; Cieplak, Piotr; Bayly, Christopher I.; Gould, Ian R.; Merz, Kenneth M., Jr.; Ferguson, David M.; Spellmeyer, David C.; Fox, Thomas; Caldwell, James W.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5179-97</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors present the derivation of a new mol. mech. force field for simulating the structures, conformational energies, and interaction energies of proteins, nucleic acids, and many related org. mols. in condensed phases.  This effective two-body force field is the successor to the Weiner et al. force field and was developed with some of the same philosophies, such as the use of a simple diagonal potential function and electrostatic potential fit atom centered charges.  The need for a 10-12 function for representing hydrogen bonds is no longer necessary due to the improved performance of the new charge model and new van der Waals parameters.  These new charges are detd. using a 6-31G* basis set and restrained electrostatic potential (RESP) fitting and have been shown to reproduce interaction energies, free energies of solvation, and conformational energies of simple small mols. to a good degree of accuracy.  Furthermore, the new RESP charges exhibit less variability as a function of the mol. conformation used in the charge detn.  The new van der Waals parameters have been derived from liq. simulations and include hydrogen parameters which take into account the effects of any geminal electroneg. atoms.  The bonded parameters developed by Weiner et al. were modified as necessary to reproduce exptl. vibrational frequencies and structures.  Most of the simple dihedral parameters have been retained from Weiner et al., but a complex set of .vphi. and ψ parameters which do a good job of reproducing the energies of the low-energy conformations of glycyl and alanyl dipeptides was developed for the peptide backbone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_vSNiuEezGrVg90H21EOLACvtfcHk0lg0EtBNX0Xcbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D&md5=2c8901ed614d543db51d3a28f5b6053c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fja00124a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00124a002%26sid%3Dliteratum%253Aachs%26aulast%3DCornell%26aufirst%3DW.%2BD.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DGould%26aufirst%3DI.%2BR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26aulast%3DSpellmeyer%26aufirst%3DD.%2BC.%26aulast%3DFox%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2Ba.%26atitle%3DA%25202nd%2520generation%2520force-field%2520for%2520the%2520simulation%2520of%2520proteins%252C%2520nucleic-acids%252C%2520and%2520organic-molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5179%26epage%3D5197%26doi%3D10.1021%2Fja00124a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">MacKerell, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evanseck, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchnir, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuczera, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, F. T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michnick, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prodhom, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiher, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlenkrich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stote, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Straub, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiorkiewicz-Kuczera, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">All-atom empirical potential for molecular modeling and dynamics studies of proteins</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3586</span><span class="NLM_x">–</span> <span class="NLM_lpage">3616</span><span class="refDoi"> DOI: 10.1021/jp973084f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp973084f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3586-3616&author=A.+D.+MacKerellauthor=D.+Bashfordauthor=M.+Bellottauthor=R.+L.+Dunbrackauthor=J.+D.+Evanseckauthor=M.+J.+Fieldauthor=S.+Fischerauthor=J.+Gaoauthor=H.+Guoauthor=S.+Haauthor=D.+Joseph-McCarthyauthor=L.+Kuchnirauthor=K.+Kuczeraauthor=F.+T.+K.+Lauauthor=C.+Mattosauthor=S.+Michnickauthor=T.+Ngoauthor=D.+T.+Nguyenauthor=B.+Prodhomauthor=W.+E.+Reiherauthor=B.+Rouxauthor=M.+Schlenkrichauthor=J.+C.+Smithauthor=R.+Stoteauthor=J.+Straubauthor=M.+Watanabeauthor=J.+Wiorkiewicz-Kuczeraauthor=D.+Yinauthor=M.+Karplus&title=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins&doi=10.1021%2Fjp973084f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">All-Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins</span></div><div class="casAuthors">MacKerell, A. D., Jr.; Bashford, D.; Bellott, M.; Dunbrack, R. L.; Evanseck, J. D.; Field, M. J.; Fischer, S.; Gao, J.; Guo, H.; Ha, S.; Joseph-McCarthy, D.; Kuchnir, L.; Kuczera, K.; Lau, F. T. K.; Mattos, C.; Michnick, S.; Ngo, T.; Nguyen, D. T.; Prodhom, B.; Reiher, W. E., III; Roux, B.; Schlenkrich, M.; Smith, J. C.; Stote, R.; Straub, J.; Watanabe, M.; Wiorkiewicz-Kuczera, J.; Yin, D.; Karplus, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3586-3616</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New protein parameters are reported for the all-atom empirical energy function in the CHARMM program.  The parameter evaluation was based on a self-consistent approach designed to achieve a balance between the internal (bonding) and interaction (nonbonding) terms of the force field and among the solvent-solvent, solvent-solute, and solute-solute interactions.  Optimization of the internal parameters used exptl. gas-phase geometries, vibrational spectra, and torsional energy surfaces supplemented with ab initio results.  The peptide backbone bonding parameters were optimized with respect to data for N-methylacetamide and the alanine dipeptide.  The interaction parameters, particularly the at. charges, were detd. by fitting ab initio interaction energies and geometries of complexes between water and model compds. that represented the backbone and the various side chains.  In addn., dipole moments, exptl. heats and free energies of vaporization, solvation and sublimation, mol. vols., and crystal pressures and structures were used in the optimization.  The resulting protein parameters were tested by applying them to noncyclic tripeptide crystals, cyclic peptide crystals, and the proteins crambin, bovine pancreatic trypsin inhibitor, and carbonmonoxy myoglobin in vacuo and in a crystal.  A detailed anal. of the relationship between the alanine dipeptide potential energy surface and calcd. protein φ, χ angles was made and used in optimizing the peptide group torsional parameters.  The results demonstrate that use of ab initio structural and energetic data by themselves are not sufficient to obtain an adequate backbone representation for peptides and proteins in soln. and in crystals.  Extensive comparisons between mol. dynamics simulation and exptl. data for polypeptides and proteins were performed for both structural and dynamic properties.  Calcd. data from energy minimization and dynamics simulations for crystals demonstrate that the latter are needed to obtain meaningful comparisons with exptl. crystal structures.  The presented parameters, in combination with the previously published CHARMM all-atom parameters for nucleic acids and lipids, provide a consistent set for condensed-phase simulations of a wide variety of mols. of biol. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor1-3zB5qPwrVg90H21EOLACvtfcHk0lg0EtBNX0Xcbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D&md5=ebb5100dafd0daeee60ca2fa66c1324a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjp973084f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp973084f%26sid%3Dliteratum%253Aachs%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DBashford%26aufirst%3DD.%26aulast%3DBellott%26aufirst%3DM.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DEvanseck%26aufirst%3DJ.%2BD.%26aulast%3DField%26aufirst%3DM.%2BJ.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DKuchnir%26aufirst%3DL.%26aulast%3DKuczera%26aufirst%3DK.%26aulast%3DLau%26aufirst%3DF.%2BT.%2BK.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DMichnick%26aufirst%3DS.%26aulast%3DNgo%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DD.%2BT.%26aulast%3DProdhom%26aufirst%3DB.%26aulast%3DReiher%26aufirst%3DW.%2BE.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DSchlenkrich%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DStote%26aufirst%3DR.%26aulast%3DStraub%26aufirst%3DJ.%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DWiorkiewicz-Kuczera%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAll-atom%2520empirical%2520potential%2520for%2520molecular%2520modeling%2520and%2520dynamics%2520studies%2520of%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3586%26epage%3D3616%26doi%3D10.1021%2Fjp973084f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiradorives, J.</span><span> </span><span class="NLM_article-title">The Opls Potential Functions for proteins - energy minimizations for crystals of cyclic-peptides and crambin</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">1657</span><span class="NLM_x">–</span> <span class="NLM_lpage">1666</span><span class="refDoi"> DOI: 10.1021/ja00214a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=1657-1666&author=W.+L.+Jorgensenauthor=J.+Tiradorives&title=The+Opls+Potential+Functions+for+proteins+-+energy+minimizations+for+crystals+of+cyclic-peptides+and+crambin&doi=10.1021%2Fja00214a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1657-66</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A complete set of intermol. potential functions was developed for use in computer simulations of proteins in their native environment.  Parameters are reported for 25 peptide residues as well as the common neutral and charged terminal groups.  The potential functions have the simple Coulomb plus Lennard-Jones form and are compatible with the widely used models for water, TIP4P, TIP3P, and SPC.  The parameters were obtained and tested primarily in conjunction with Monte Carlo statistical mechanics simulations of 36 pure org. liqs. and numerous aq. solns. of org. ions representative of subunits in the side chains and backbones of proteins.  Bond stretch, angle bend, and torsional terms were adopted from the AMBER united-atom force field.  As reported here, further testing involved studies of conformational energy surfaces and optimizations of the crystal structures for 4 cyclic hexapeptides and a cyclic pentapeptide.  The av. root mean square deviation from the x-ray structures of the crystals is only 0.17 Å for the at. positions and 3% for the unit cell vols.  A more crit. test was then provided by performing energy minimizations for the complete crystal of the protein crambin, including 182 water mols. that were initially placed via a Monte Carlo simulation.  The resultant root mean square deviation for the non-H atoms is still only 0.17 Å and the variation in the errors for charged, polar, and nonpolar residues is small.  Significant improvement is apparent over the AMBER united-atom force field which was previously demonstrated to be superior to many alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6an38QWApbVg90H21EOLACvtfcHk0lhOank8qnzjfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D&md5=b6c901d8c295b3b37329a7faef527e12</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fja00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTiradorives%26aufirst%3DJ.%26atitle%3DThe%2520Opls%2520Potential%2520Functions%2520for%2520proteins%2520-%2520energy%2520minimizations%2520for%2520crystals%2520of%2520cyclic-peptides%2520and%2520crambin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D1657%26epage%3D1666%26doi%3D10.1021%2Fja00214a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lhOank8qnzjfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatcher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darian, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobyov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.</span><span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+MacKerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0lhOank8qnzjfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0ljbj_YoIDEnGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Betz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Paramfit: automated optimization of force field parameters for molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1002/jcc.23775</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.23775" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25413259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=79-87&author=R.+M.+Betzauthor=R.+C.+Walker&title=Paramfit%3A+automated+optimization+of+force+field+parameters+for+molecular+dynamics+simulations&doi=10.1002%2Fjcc.23775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Paramfit: Automated optimization of force field parameters for molecular dynamics simulations</span></div><div class="casAuthors">Betz, Robin M.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">79-87</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The generation of bond, angle, and torsion parameters for classical mol. dynamics force fields typically requires fitting parameters such that classical properties such as energies and gradients match precalcd. quantum data for structures that scan the value of interest.  We present a program, Paramfit, distributed as part of the AmberTools software package that automates and extends this fitting process, allowing for simplified parameter generation for applications ranging from single mols. to entire force fields.  Paramfit implements a novel combination of a genetic and simplex algorithm to find the optimal set of parameters that replicate either quantum energy or force data.  The program allows for the derivation of multiple parameters simultaneously using significantly fewer quantum calcns. than previous methods, and can also fit parameters across multiple mols. with applications to force field development.  Paramfit has been applied successfully to systems with a sparse no. of structures, and has already proven crucial in the development of the Assisted Model Building with Energy Refinement Lipid14 force field. © 2014 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt7_grqBnCgLVg90H21EOLACvtfcHk0ljbj_YoIDEnGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtLbM&md5=be5df82613ad9916e567795a54f4f3b3</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23775%26sid%3Dliteratum%253Aachs%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DParamfit%253A%2520automated%2520optimization%2520of%2520force%2520field%2520parameters%2520for%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D79%26epage%3D87%26doi%3D10.1002%2Fjcc.23775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Hopkins, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Fitting of dihedral terms in classical force fields as an analytic linear least-squares problem</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1978</span><span class="NLM_x">–</span> <span class="NLM_lpage">1986</span><span class="refDoi"> DOI: 10.1021/ci500112w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500112w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaksrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1978-1986&author=C.+W.+Hopkinsauthor=A.+E.+Roitberg&title=Fitting+of+dihedral+terms+in+classical+force+fields+as+an+analytic+linear+least-squares+problem&doi=10.1021%2Fci500112w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Fitting of Dihedral Terms in Classical Force Fields as an Analytic Linear Least-Squares Problem</span></div><div class="casAuthors">Hopkins, Chad W.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1978-1986</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The derivation and optimization of most energy terms in modern force fields are aided by automated computational tools.  It is therefore important to have algorithms to rapidly and precisely train large nos. of interconnected parameters to allow investigators to make better decisions about the content of mol. models.  In particular, the traditional approach to deriving dihedral parameters has been a least-squares fit to target conformational energies through variational optimization strategies.  We present a computational approach for simultaneously fitting force field dihedral amplitudes and phase consts. which is analytic within the scope of the data set.  This approach completes the optimal mol. mechanics representation of a quantum mech. potential energy surface in a single linear least-squares fit by recasting the dihedral potential into a linear function in the parameters.  We compare the resulting method to a genetic algorithm in terms of computational time and quality of fit for two simple mols.  As suggested in previous studies, arbitrary dihedral phases are only necessary when modeling chiral mols., which include more than half of drugs currently in use, so we also examd. a dihedral parametrization case for the drug amoxicillin and one of its stereoisomers where the target dihedral includes a chiral center.  Asym. dihedral phases are needed in these types of cases to properly represent the quantum mech. energy surface and to differentiate between stereoisomers about the chiral center.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprIgbV7-frvrVg90H21EOLACvtfcHk0ljbj_YoIDEnGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaksrvE&md5=70c162c18e1dcbfa462da31d1d554b8c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci500112w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500112w%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DC.%2BW.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DFitting%2520of%2520dihedral%2520terms%2520in%2520classical%2520force%2520fields%2520as%2520an%2520analytic%2520linear%2520least-squares%2520problem%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1978%26epage%3D1986%26doi%3D10.1021%2Fci500112w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Automated force field parameterization for non-polarizable and polarizable atomic models based on target data</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3543</span><span class="NLM_x">–</span> <span class="NLM_lpage">3556</span><span class="refDoi"> DOI: 10.1021/ct4003477</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct4003477" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSns7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3543-3556&author=L.+Huangauthor=B.+Roux&title=Automated+force+field+parameterization+for+non-polarizable+and+polarizable+atomic+models+based+on+target+data&doi=10.1021%2Fct4003477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Force Field Parameterization for Nonpolarizable and Polarizable Atomic Models Based on Ab Initio Target Data</span></div><div class="casAuthors">Huang, Lei; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3543-3556</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Classical mol. dynamics (MD) simulations based on atomistic models are increasingly used to study a wide range of biol. systems.  A prerequisite for meaningful results from such simulations is an accurate mol. mech. force field.  Most biomol. simulations are currently based on the widely used AMBER and CHARMM force fields, which were parametrized and optimized to cover a small set of basic compds. corresponding to the natural amino acids and nucleic acid bases.  Atomic models of addnl. compds. are commonly generated by analogy to the parameter set of a given force field.  While this procedure yields models that are internally consistent, the accuracy of the resulting models can be limited.  The authors propose a method, general automated at. model parameterization (GAAMP), for generating automatically the parameters of at. models of small mols. using the results from ab initio quantum mech. (QM) calcns. as target data.  Force fields that were previously developed for a wide range of model compds. serve as initial guesses, although any of the final parameter can be optimized.  The electrostatic parameters (partial charges, polarizabilities, and shielding) are optimized on the basis of QM electrostatic potential (ESP) and, if applicable, the interaction energies between the compd. and water mols.  The soft dihedrals are automatically identified and parametrized by targeting QM dihedral scans as well as the energies of stable conformers.  To validate the approach, the solvation free energy is calcd. for >200 small mols. and MD simulations of three different proteins were carried out.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjMGhS8IcFx7Vg90H21EOLACvtfcHk0li_fhe67fc97g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSns7rP&md5=8f7c3d2fd1e29afe685a4e94a5e0943a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fct4003477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct4003477%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAutomated%2520force%2520field%2520parameterization%2520for%2520non-polarizable%2520and%2520polarizable%2520atomic%2520models%2520based%2520on%2520target%2520data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3543%26epage%3D3556%26doi%3D10.1021%2Fct4003477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Wang, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Building force fields: an automatic, systematic, and reproducible approach</span> <span class="citation_source-journal">J. Phys. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1885</span><span class="NLM_x">–</span> <span class="NLM_lpage">1891</span><span class="refDoi"> DOI: 10.1021/jz500737m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jz500737m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvV2htrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1885-1891&author=L.+P.+Wangauthor=T.+J.+Martinezauthor=V.+S.+Pande&title=Building+force+fields%3A+an+automatic%2C+systematic%2C+and+reproducible+approach&doi=10.1021%2Fjz500737m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Building Force Fields: An Automatic, Systematic, and Reproducible Approach</span></div><div class="casAuthors">Wang, Lee-Ping; Martinez, Todd J.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1885-1891</span>CODEN:
                <span class="NLM_cas:coden">JPCLCD</span>;
        ISSN:<span class="NLM_cas:issn">1948-7185</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of accurate mol. mechanics force fields is a significant challenge that must be addressed for the continued success of mol. simulation.  We developed the ForceBalance method to automatically derive accurate force field parameters using flexible combinations of exptl. and theor. ref. data.  The method is demonstrated in the parametrization of two rigid water models, yielding new parameter sets (TIP3P-FB and TIP4P-FB) that accurately describe many phys. properties of water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFqC-UGvhNiLVg90H21EOLACvtfcHk0li_fhe67fc97g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvV2htrs%253D&md5=0b901b4ef8949839df5b5e3e36e38424</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjz500737m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjz500737m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DMartinez%26aufirst%3DT.%2BJ.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DBuilding%2520force%2520fields%253A%2520an%2520automatic%252C%2520systematic%252C%2520and%2520reproducible%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1885%26epage%3D1891%26doi%3D10.1021%2Fjz500737m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Ryckaert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciccotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0ljdSUmU-qL9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Hopkins, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Long-time-step molecular dynamics through hydrogen mass repartitioning</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1864</span><span class="NLM_x">–</span> <span class="NLM_lpage">1874</span><span class="refDoi"> DOI: 10.1021/ct5010406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5010406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1864-1874&author=C.+W.+Hopkinsauthor=S.+Le+Grandauthor=R.+C.+Walkerauthor=A.+E.+Roitberg&title=Long-time-step+molecular+dynamics+through+hydrogen+mass+repartitioning&doi=10.1021%2Fct5010406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Long-Time-Step Molecular Dynamics through Hydrogen Mass Repartitioning</span></div><div class="casAuthors">Hopkins, Chad W.; Le Grand, Scott; Walker, Ross C.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1864-1874</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies have shown that the method of hydrogen mass repartitioning (HMR) is a potentially useful tool for accelerating mol. dynamics (MD) simulations.  By repartitioning the mass of heavy atoms into the bonded hydrogen atoms, it is possible to slow the highest-frequency motions of the macromol. under study, thus allowing the time step of the simulation to be increased by up to a factor of 2.  In this communication, we investigate further how this mass repartitioning allows the simulation time step to be increased in a stable fashion without significantly increasing discretization error.  To this end, we ran a set of simulations with different time steps and mass distributions on a three-residue peptide to get a comprehensive view of the effect of mass repartitioning and time step increase on a system whose accessible phase space is fully explored in a relatively short amt. of time.  We next studied a 129-residue protein, hen egg white lysozyme (HEWL), to verify that the obsd. behavior extends to a larger, more-realistic, system.  Results for the protein include structural comparisons from MD trajectories, as well as comparisons of pKa calcns. via const.-pH MD.  We also calcd. a potential of mean force (PMF) of a dihedral rotation for the MTS [(1-oxyl-2,2,5,5-tetramethyl-pyrroline-3-methyl)methanethiosulfonate] spin label via umbrella sampling with a set of regular MD trajectories, as well as a set of mass-repartitioned trajectories with a time step of 4 fs.  Since no significant difference in kinetics or thermodn. is obsd. by the use of fast HMR trajectories, further evidence is provided that long-time-step HMR MD simulations are a viable tool for accelerating MD simulations for mols. of biochem. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfG1v1aGj1NrVg90H21EOLACvtfcHk0ljdSUmU-qL9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D&md5=6ca75b544b63523481de7836d11a3c1b</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fct5010406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5010406%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DC.%2BW.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DLong-time-step%2520molecular%2520dynamics%2520through%2520hydrogen%2520mass%2520repartitioning%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D1864%26epage%3D1874%26doi%3D10.1021%2Fct5010406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Hünenberger, P.</span><span> </span><span class="NLM_article-title">Thermostat algorithms for molecular dynamics simulations</span>. In  <span class="citation_source-book">Advanced Computer Simulation</span>; <span class="NLM_contrib-group">Holm, C.; Kremer, K.</span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">173</span>, pp  <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">149</span>, DOI: <span class="refDoi"> DOI: 10.1007/b99427</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fb99427" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=105-149&author=P.+H%C3%BCnenbergerauthor=C.+Holm&author=K.+Kremer&title=Advanced+Computer+Simulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fb99427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb99427%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%26atitle%3DThermostat%2520algorithms%2520for%2520molecular%2520dynamics%2520simulations%26btitle%3DAdvanced%2520Computer%2520Simulation%26aulast%3DHolm%26aufirst%3DC.%26pub%3DSpringer%26date%3D2005%26volume%3D173%26spage%3D105%26epage%3D149%26doi%3D10.1007%2Fb99427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L.</span><span> </span><span class="NLM_article-title">Particle mesh Ewald: An N·log(N) method for ewald sums in large systems</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10089</span><span class="NLM_x">–</span> <span class="NLM_lpage">10092</span><span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N%C2%B7log%28N%29+method+for+ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0ljdSUmU-qL9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Essmann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berkowitz, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L. G.</span><span> </span><span class="NLM_article-title">A smooth particle mesh ewald method</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">8577</span><span class="NLM_x">–</span> <span class="NLM_lpage">8593</span><span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0lhQ0zpUlueSMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Sagui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T. A.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations of biomolecules: long-range electrostatic effects</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1146/annurev.biophys.28.1.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1146%2Fannurev.biophys.28.1.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10410799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1MXkt1ertbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1999&pages=155-179&author=C.+Saguiauthor=T.+A.+Darden&title=Molecular+dynamics+simulations+of+biomolecules%3A+long-range+electrostatic+effects&doi=10.1146%2Fannurev.biophys.28.1.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations of biomolecules: long-range electrostatic effects</span></div><div class="casAuthors">Sagui, Celeste; Darden, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-179</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 72 refs.  Current computer simulations of biomols. typically make use of classical mol. dynamics methods, as a very large no. (tens to hundreds of thousands) of atoms are involved over timescales of many nanoseconds.  The methodol. for treating short-range bonded and van der Waals interactions has matured.  However, long-range electrostatic interactions still represent a bottleneck in simulations.  In this article, we introduce the basic issues for an accurate representation of the relevant electrostatic interactions.  In spite of the huge computational time demanded by most biomol. systems, it is no longer necessary to resort to uncontrolled approxns. such as the use of cutoffs.  In particular, we discuss the Ewald summation methods, the fast particle mesh methods, and the fast multipole methods.  We also review recent efforts to understand the role of boundary conditions in systems with long-range interactions and conclude with a short perspective on future trends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoNMaeS53ux7Vg90H21EOLACvtfcHk0lhQ0zpUlueSMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXkt1ertbc%253D&md5=1da8a35730741e9430fe29dbfe33e84f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.28.1.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.28.1.155%26sid%3Dliteratum%253Aachs%26aulast%3DSagui%26aufirst%3DC.%26aulast%3DDarden%26aufirst%3DT.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520of%2520biomolecules%253A%2520long-range%2520electrostatic%2520effects%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D1999%26volume%3D28%26spage%3D155%26epage%3D179%26doi%3D10.1146%2Fannurev.biophys.28.1.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span> </span><span class="NLM_article-title">Molecular mechanics and dynamics: numerical tools to sample the configuration space</span> <span class="citation_source-journal">Front. Biosci., Landmark Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">578</span><span class="NLM_x">–</span> <span class="NLM_lpage">604</span><span class="refDoi"> DOI: 10.2741/4229</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2741%2F4229" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=578-604&issue=19&author=M.+Masettiauthor=W.+Rocchia&title=Molecular+mechanics+and+dynamics%3A+numerical+tools+to+sample+the+configuration+space&doi=10.2741%2F4229"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.2741%2F4229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4229%26sid%3Dliteratum%253Aachs%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DMolecular%2520mechanics%2520and%2520dynamics%253A%2520numerical%2520tools%2520to%2520sample%2520the%2520configuration%2520space%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2014%26volume%3D1%26issue%3D19%26spage%3D578%26epage%3D604%26doi%3D10.2741%2F4229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Shi, Y.; Ren, P.; Schnieders, M.; Piquemal, J.-P.</span><span> </span><span class="NLM_article-title">Polarizable force fields for biomolecular modeling</span>. In  <span class="citation_source-book">Reviews in Computational Chemistry</span>; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">28</span>, pp  <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span>, DOI: <span class="refDoi"> DOI: 10.1002/9781118889886.ch2</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2F9781118889886.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=51-86&author=Y.+Shi&author=P.+Ren&author=M.+Schnieders&author=J.-P.+Piquemal&title=Reviews+in+Computational+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2F9781118889886.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118889886.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26atitle%3DPolarizable%2520force%2520fields%2520for%2520biomolecular%2520modeling%26btitle%3DReviews%2520in%2520Computational%2520Chemistry%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26volume%3D28%26spage%3D51%26epage%3D86%26doi%3D10.1002%2F9781118889886.ch2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Jiao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golubkov, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, P.</span><span> </span><span class="NLM_article-title">Calculation of protein-ligand binding free energy by using a polarizable potential</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6290</span><span class="NLM_x">–</span> <span class="NLM_lpage">6295</span><span class="refDoi"> DOI: 10.1073/pnas.0711686105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0711686105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18427113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6290-6295&author=D.+Jiaoauthor=P.+A.+Golubkovauthor=T.+A.+Dardenauthor=P.+Ren&title=Calculation+of+protein-ligand+binding+free+energy+by+using+a+polarizable+potential&doi=10.1073%2Fpnas.0711686105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of protein-ligand binding free energy by using a polarizable potential</span></div><div class="casAuthors">Jiao, Dian; Golubkov, Pavel A.; Darden, Thomas A.; Ren, Pengyu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6290-6295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The binding of charged ligands benzamidine and diazamidine to trypsin was investigated by using a polarizable potential energy function and explicit-water mol. dynamics simulations.  The binding free energies were computed from the difference between the free energies of decoupling the ligand from water and protein environments.  Both the abs. and the relative free energies from the perturbation simulations agree with exptl. measurements to within 0.5 kcal-mol-1.  Comparison of free-energy components sampled from different thermodn. paths indicates that electrostatics is the main driving force behind benzamidine recognition of trypsin.  The contribution of electronic polarization to binding appears to be crucial.  By computing the free-energy contribution caused by the polarization between the ligand and its surroundings, we found that polarization has the opposite effect in dissimilar environments.  Although polarization favors ligand solvation in water, it weakens the protein-ligand attraction by screening the electrostatic interaction between trypsin and benzamidine.  We also examd. the relative binding free energies of a benzamidine analog diazamidine to trypsin.  The changes in free energy on benzamidine-diazamidine substitution were tens of kilocalories in both water and trypsin environments; however, the change in the total binding free energy is <2 kcal-mol-1 because of cancellation, consistent with the exptl. results.  Overall, our results suggest that the use of a polarizable force field, given adequate sampling, is capable of achieving chem. accuracy in mol. simulations of protein-ligand recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc3_p806L5GLVg90H21EOLACvtfcHk0lhQ0zpUlueSMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyht7Y%253D&md5=ca74cfaca7f521a315e2ee85440591f9</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711686105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711686105%26sid%3Dliteratum%253Aachs%26aulast%3DJiao%26aufirst%3DD.%26aulast%3DGolubkov%26aufirst%3DP.%2BA.%26aulast%3DDarden%26aufirst%3DT.%2BA.%26aulast%3DRen%26aufirst%3DP.%26atitle%3DCalculation%2520of%2520protein-ligand%2520binding%2520free%2520energy%2520by%2520using%2520a%2520polarizable%2520potential%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D6290%26epage%3D6295%26doi%3D10.1073%2Fpnas.0711686105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Warshel, a.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levitt, M.</span><span> </span><span class="NLM_article-title">Theoretical studies of enzymic reactions - dielectric, electrostatic and steric stabilization of carbonium-ion in reaction of lysozyme</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1016/0022-2836(76)90311-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0022-2836%2876%2990311-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=985660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaE28XktFKhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1976&pages=227-249&author=a.+Warshelauthor=M.+Levitt&title=Theoretical+studies+of+enzymic+reactions+-+dielectric%2C+electrostatic+and+steric+stabilization+of+carbonium-ion+in+reaction+of+lysozyme&doi=10.1016%2F0022-2836%2876%2990311-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Theoretical studies of enzymic reactions:  dielectric, electrostatic and steric stabilization of the carbonium ion in the reaction of lysozyme</span></div><div class="casAuthors">Warshel, A.; Levitt, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">227-49</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A general method for detailed study of enzymic reactions is presented.  The method considers the complete enzyme-substrate complex together with the surrounding solvent and evaluates all the different quantum mech. and classical energy factors that can affect the reaction pathway.  These factors include the quantum mech. energies assocd. with bond cleavage and charge redistribution of the substrate and the classical energies of steric and electrostatic interactions between the substrate and the enzyme.  The electrostatic polarization of the enzyme atoms and the orientation of the dipoles of the surrounding H2O mols. is simulated by a microscopic dielec. model.  The solvation energy resulting from this polarization is considerable and must be included in any realistic calcn. of chem. reactions involving anything more than an isolated mol. in vacuo.  Without it, acidic groups can never become ionized and the charge distribution on the substrate will not be reasonable.  The same dielec. model can also be used to study the reaction of the substrate in soln.  In this way the reaction in soln. can be compared with the enzymic reaction.  The stability of the carbonium ion intermediate formed in the cleavage of a glycosidic bond by lysozyme was studied.  Electrostatic stabilization is an important factor in increasing the rate of the reaction step that leads to the formation of the carbonium ion intermediate.  Steric factors, such as the strain of the substrate on binding to lysozyme, do not seem to contribute significantly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB3Es44qCfVrVg90H21EOLACvtfcHk0ljuiGj3S_QThg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XktFKhtr0%253D&md5=f34df33b5971b6b02bd03be95dcd7ba5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2876%2990311-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252876%252990311-9%26sid%3Dliteratum%253Aachs%26aulast%3DWarshel%26aufirst%3Da.%26aulast%3DLevitt%26aufirst%3DM.%26atitle%3DTheoretical%2520studies%2520of%2520enzymic%2520reactions%2520-%2520dielectric%252C%2520electrostatic%2520and%2520steric%2520stabilization%2520of%2520carbonium-ion%2520in%2520reaction%2520of%2520lysozyme%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1976%26volume%3D103%26spage%3D227%26epage%3D249%26doi%3D10.1016%2F0022-2836%2876%2990311-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Raha, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">WollaCott, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westerhoff, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.</span><span> </span><span class="NLM_article-title">The role of quantum mechanics in structure-based drug design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span><span class="refDoi"> DOI: 10.1016/j.drudis.2007.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2007.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=17826685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=725-731&author=K.+Rahaauthor=M.+B.+Petersauthor=B.+Wangauthor=N.+Yuauthor=A.+M.+WollaCottauthor=L.+M.+Westerhoffauthor=K.+M.+Merz&title=The+role+of+quantum+mechanics+in+structure-based+drug+design&doi=10.1016%2Fj.drudis.2007.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">The role of quantum mechanics in structure-based drug design</span></div><div class="casAuthors">Raha, Kaushik; Peters, Martin B.; Wang, Bing; Yu, Ning; Wollacott, Andrew M.; Westerhoff, Lance M.; Merz, Kenneth M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">17&18</span>),
    <span class="NLM_cas:pages">725-731</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Herein we will focus on the use of quantum mechanics (QM) in drug design (DD) to solve disparate problems from scoring protein-ligand poses to building QM QSAR models.  Through the variational principle of QM we know that we can obtain a more accurate representation of mol. systems than classical models, and while this is not a matter of debate, it still has not been shown that the expense of QM approaches is offset by improved accuracy in DD applications.  Objectively validating the improved applicability and performance of QM over classical-based models in DD will be the focus of research in the coming years along with research on the conformational sampling problem as it relates to protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ISn2DWdN9LVg90H21EOLACvtfcHk0ljuiGj3S_QThg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsrvK&md5=67fecc13843c7961ff820026c85a9290</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DRaha%26aufirst%3DK.%26aulast%3DPeters%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DN.%26aulast%3DWollaCott%26aufirst%3DA.%2BM.%26aulast%3DWesterhoff%26aufirst%3DL.%2BM.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26atitle%3DThe%2520role%2520of%2520quantum%2520mechanics%2520in%2520structure-based%2520drug%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D725%26epage%3D731%26doi%3D10.1016%2Fj.drudis.2007.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Quantum mechanical methods for drug design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.2174/156802610790232242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802610790232242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19929831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=33-45&author=T.+Zhouauthor=D.+Huangauthor=A.+Caflisch&title=Quantum+mechanical+methods+for+drug+design&doi=10.2174%2F156802610790232242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum mechanical methods for drug design</span></div><div class="casAuthors">Zhou, Ting; Huang, Danzhi; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-45</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Quantum mech. (QM) methods are becoming popular in computational drug design and development mainly because high accuracy is required to est. (relative) binding affinities.  For low- to medium-throughput in silico screening, (e.g., scoring and prioritizing a series of inhibitors sharing the same mol. scaffold) efficient approxns. have been developed in the past decade, like linear scaling QM in which the computation time scales almost linearly with the no. of basis functions.  Furthermore, QM-based procedures have been used recently for detg. protonation states of ionizable groups, evaluating energies, and optimizing mol. structures.  For high-throughput in silico screening QM approaches have been employed to derive robust quant. structure-activity relation models.  It is expected that the use of QM methods will keep growing in all phases of computer-aided drug design and development.  However, extensive sampling of conformational space and treatment of soln. of macromols. are still limiting factors for the broad application of QM in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocoQjA2ielFbVg90H21EOLACvtfcHk0ljuiGj3S_QThg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D&md5=735ac208504a7e563e6ca0eb3c1cd17b</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.2174%2F156802610790232242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232242%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DQuantum%2520mechanical%2520methods%2520for%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D33%26epage%3D45%26doi%3D10.2174%2F156802610790232242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">van Duin, A. C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorant, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, W. A.</span><span> </span><span class="NLM_article-title">ReaxFF: A reactive force field for hydrocarbons</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">9396</span><span class="NLM_x">–</span> <span class="NLM_lpage">9409</span><span class="refDoi"> DOI: 10.1021/jp004368u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp004368u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFChu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=9396-9409&author=A.+C.+T.+van+Duinauthor=S.+Dasguptaauthor=F.+Lorantauthor=W.+A.+Goddard&title=ReaxFF%3A+A+reactive+force+field+for+hydrocarbons&doi=10.1021%2Fjp004368u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">ReaxFF: A Reactive Force Field for Hydrocarbons</span></div><div class="casAuthors">van Duin, Adri C. T.; Dasgupta, Siddharth; Lorant, Francois; Goddard, William A., III</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9396-9409</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To make practical the mol. dynamics simulation of large scale reactive chem. systems (1000 s of atoms), the authors developed ReaxFF, a force field for reactive systems. ReaxFF uses a general relation between bond distance and bond order on one hand and between bond order and bond energy however, that leads to proper dissocn. of bonds to sepd. atoms.  Other valence terms present in the force field (angle and torsion) are defined in terms of the same bond orders so that all these terms go to zero smoothly as bonds break.  In addn., ReaxFF has Coulomb and Morse (van der Waals) potentials to describe nonbond interactions between all atoms (no exclusions).  These nonbond interactions are shielded at short range so that the Coulomb and van der Waals interactions become const. as Rij → 0.  The authors report here the ReaxFF for hydrocarbons.  The parameters were derived from quantum chem. calcns. on bond dissocn. and reactions of small mols. plus heat of formation and geometry data for a no. of stable hydrocarbon compds.  The ReaxFF provides a good description of these data. Generally, the results are of an accuracy similar or better than PM3, while ReaxFF is ∼100 times faster.  In turn, the PM3 is ∼100 times faster than the QC calcns. Thus, with ReaxFF the authors hope to be able to study complex reactions in hydrocarbons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZfMcUTM2NorVg90H21EOLACvtfcHk0ljPR_jhwawioA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFChu78%253D&md5=ea59efc08d5e135745df988f2006a7fd</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjp004368u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp004368u%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BDuin%26aufirst%3DA.%2BC.%2BT.%26aulast%3DDasgupta%26aufirst%3DS.%26aulast%3DLorant%26aufirst%3DF.%26aulast%3DGoddard%26aufirst%3DW.%2BA.%26atitle%3DReaxFF%253A%2520A%2520reactive%2520force%2520field%2520for%2520hydrocarbons%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2001%26volume%3D105%26spage%3D9396%26epage%3D9409%26doi%3D10.1021%2Fjp004368u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Goh, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hulbert, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.</span><span> </span><span class="NLM_article-title">Constant pH molecular dynamics of proteins in explicit solvent with proton tautomerism</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">1319</span><span class="NLM_x">–</span> <span class="NLM_lpage">1331</span><span class="refDoi"> DOI: 10.1002/prot.24499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.24499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24375620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt12ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2014&pages=1319-1331&author=G.+B.+Gohauthor=B.+S.+Hulbertauthor=H.+Q.+Zhouauthor=C.+L.+Brooks&title=Constant+pH+molecular+dynamics+of+proteins+in+explicit+solvent+with+proton+tautomerism&doi=10.1002%2Fprot.24499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH molecular dynamics of proteins in explicit solvent with proton tautomerism</span></div><div class="casAuthors">Goh, Garrett B.; Hulbert, Benjamin S.; Zhou, Huiqing; Brooks, Charles L., III</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1319-1331</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">PH is a ubiquitous regulator of biol. activity, including protein-folding, protein-protein interactions, and enzymic activity.  Existing const. pH mol. dynamics (CPHMD) models that were developed to address questions related to the pH-dependent properties of proteins are largely based on implicit solvent models.  However, implicit solvent models are known to underestimate the desolvation energy of buried charged residues, increasing the error assocd. with predictions that involve internal ionizable residue that are important in processes like hydrogen transport and electron transfer.  Furthermore, discrete water and ions cannot be modeled in implicit solvent, which are important in systems like membrane proteins and ion channels.  We report on an explicit solvent const. pH mol. dynamics framework based on multi-site λ-dynamics (CPHMDMSλD).  In the CPHMDMSλD framework, we performed seamless alchem. transitions between protonation and tautomeric states using multi-site λ-dynamics, and designed novel biasing potentials to ensure that the phys. end-states are predominantly sampled.  We show that explicit solvent CPHMDMSλD simulations model realistic pH-dependent properties of proteins such as the Hen-Egg White Lysozyme (HEWL), binding domain of 2-oxoglutarate dehydrogenase (BBL) and N-terminal domain of ribosomal protein L9 (NTL9), and the pKa predictions are in excellent agreement with exptl. values, with a RMSE ranging from 0.72 to 0.84 pKa units.  With the recent development of the explicit solvent CPHMDMSλD framework for nucleic acids, accurate modeling of pH-dependent properties of both major class of biomols.-proteins and nucleic acids is now possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Ai09dtRsLbVg90H21EOLACvtfcHk0ljPR_jhwawioA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt12ntw%253D%253D&md5=4e99b314f79489155546b3966614f832</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fprot.24499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24499%26sid%3Dliteratum%253Aachs%26aulast%3DGoh%26aufirst%3DG.%2BB.%26aulast%3DHulbert%26aufirst%3DB.%2BS.%26aulast%3DZhou%26aufirst%3DH.%2BQ.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26atitle%3DConstant%2520pH%2520molecular%2520dynamics%2520of%2520proteins%2520in%2520explicit%2520solvent%2520with%2520proton%2520tautomerism%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2014%26volume%3D82%26spage%3D1319%26epage%3D1331%26doi%3D10.1002%2Fprot.24499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Constant pH replica exchange molecular dynamics in explicit solvent using discrete protonation states: implementation, testing, and validation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1341</span><span class="NLM_x">–</span> <span class="NLM_lpage">1352</span><span class="refDoi"> DOI: 10.1021/ct401042b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct401042b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1341-1352&author=J.+M.+Swailsauthor=D.+M.+Yorkauthor=A.+E.+Roitberg&title=Constant+pH+replica+exchange+molecular+dynamics+in+explicit+solvent+using+discrete+protonation+states%3A+implementation%2C+testing%2C+and+validation&doi=10.1021%2Fct401042b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH Replica Exchange Molecular Dynamics in Explicit Solvent Using Discrete Protonation States: Implementation, Testing, and Validation</span></div><div class="casAuthors">Swails, Jason M.; York, Darrin M.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1341-1352</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By utilizing Graphics Processing Units, we show that const. pH mol. dynamics simulations (CpHMD) run in Generalized Born (GB) implicit solvent for long time scales can yield poor pKa predictions as a result of sampling unrealistic conformations.  To address this shortcoming, we present a method for performing const. pH mol. dynamics simulations (CpHMD) in explicit solvent using a discrete protonation state model.  The method involves std. mol. dynamics (MD) being propagated in explicit solvent followed by protonation state changes being attempted in GB implicit solvent at fixed intervals.  Replica exchange along the pH-dimension (pH-REMD) helps to obtain acceptable titrn. behavior with the proposed method.  We analyzed the effects of various parameters and settings on the titrn. behavior of CpHMD and pH-REMD in explicit solvent, including the size of the simulation unit cell and the length of the relaxation dynamics following protonation state changes.  We tested the method with the amino acid model compds., a small pentapeptide with two titratable sites, and hen egg white lysozyme (HEWL).  The proposed method yields superior predicted pKa values for HEWL over hundreds of nanoseconds of simulation relative to corresponding predicted values from simulations run in implicit solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlQ-zHy36SV7Vg90H21EOLACvtfcHk0ljPR_jhwawioA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyiu7g%253D&md5=7f320339f40c5308035b25cc1aaea09c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fct401042b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct401042b%26sid%3Dliteratum%253Aachs%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DYork%26aufirst%3DD.%2BM.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DConstant%2520pH%2520replica%2520exchange%2520molecular%2520dynamics%2520in%2520explicit%2520solvent%2520using%2520discrete%2520protonation%2520states%253A%2520implementation%252C%2520testing%252C%2520and%2520validation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D1341%26epage%3D1352%26doi%3D10.1021%2Fct401042b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Donnini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegeler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Constant pH molecular dynamics in explicit solvent with lambda-dynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1962</span><span class="NLM_x">–</span> <span class="NLM_lpage">1978</span><span class="refDoi"> DOI: 10.1021/ct200061r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200061r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=1962-1978&author=S.+Donniniauthor=F.+Tegelerauthor=G.+Groenhofauthor=H.+Grubmuller&title=Constant+pH+molecular+dynamics+in+explicit+solvent+with+lambda-dynamics&doi=10.1021%2Fct200061r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pH Molecular Dynamics in Explicit Solvent with λ-Dynamics</span></div><div class="casAuthors">Donnini, Serena; Tegeler, Florian; Groenhof, Gerrit; Grubmuller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1962-1978</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PH is an important parameter in condensed-phase systems, because it dets. the protonation state of titratable groups and thus influences the structure, dynamics, and function of mols. in soln.  In most force field simulation protocols, however, the protonation state of a system (rather than its pH) is kept fixed and cannot adapt to changes of the local environment.  Here, we present a method, implemented within the MD package GROMACS, for const. pH mol. dynamics simulations in explicit solvent that is based on the λ-dynamics approach.  In the latter, the dynamics of the titrn. coordinate λ, which interpolates between the protonated and deprotonated states, is driven by generalized forces between the protonated and deprotonated states.  The hydration free energy, as a function of pH, is included to facilitate const. pH simulations.  The protonation states of titratable groups are allowed to change dynamically during a simulation, thus reproducing av. protonation probabilities at a certain pH.  The accuracy of the method is tested against titrn. curves of single amino acids and a dipeptide in explicit solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIp1iVOhE1DLVg90H21EOLACvtfcHk0lh1A48ETAvwtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFShsL8%253D&md5=0637c4d334ea58836db9e465401624c3</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fct200061r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200061r%26sid%3Dliteratum%253Aachs%26aulast%3DDonnini%26aufirst%3DS.%26aulast%3DTegeler%26aufirst%3DF.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DConstant%2520pH%2520molecular%2520dynamics%2520in%2520explicit%2520solvent%2520with%2520lambda-dynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D1962%26epage%3D1978%26doi%3D10.1021%2Fct200061r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Grossfield, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuckerman, D. M.</span><span> </span><span class="NLM_article-title">Quantifying uncertainty and sampling quality in biomolecular simulations</span> <span class="citation_source-journal">Annu. Rep. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">48</span><span class="refDoi"> DOI: 10.1016/S1574-1400(09)00502-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1574-1400%2809%2900502-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20454547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1aiu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=23-48&author=A.+Grossfieldauthor=D.+M.+Zuckerman&title=Quantifying+uncertainty+and+sampling+quality+in+biomolecular+simulations&doi=10.1016%2FS1574-1400%2809%2900502-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying uncertainty and sampling quality in biomolecular simulations</span></div><div class="casAuthors">Grossfield, Alan; Zuckerman, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-48</span>CODEN:
                <span class="NLM_cas:coden">ARCCC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-1400</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Growing computing capacity and algorithmic advances have facilitated the study of increasingly large biomol. systems at longer timescales.  However, with these larger, more complex systems come questions about the quality of sampling and statistical convergence.  What size systems can be sampled fully.  If a system is not fully sampled, can certain "fast variables" be considered well converged How can one det. the statistical significance of obsd. results.  The present review describes statistical tools and the underlying phys. ideas necessary to address these questions.  Basic definitions and ready-to-use analyses are provided, along with explicit recommendations.  Such statistical analyses are of paramount importance in establishing the reliability of simulation data in any given study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkELYsWb9OZLVg90H21EOLACvtfcHk0lh1A48ETAvwtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1aiu7rM&md5=59be756cd4b6a617debb6da01b46fcc9</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2FS1574-1400%2809%2900502-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1574-1400%252809%252900502-7%26sid%3Dliteratum%253Aachs%26aulast%3DGrossfield%26aufirst%3DA.%26aulast%3DZuckerman%26aufirst%3DD.%2BM.%26atitle%3DQuantifying%2520uncertainty%2520and%2520sampling%2520quality%2520in%2520biomolecular%2520simulations%26jtitle%3DAnnu.%2520Rep.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D5%26spage%3D23%26epage%3D48%26doi%3D10.1016%2FS1574-1400%2809%2900502-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Given, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">1047</span><span class="NLM_x">–</span> <span class="NLM_lpage">1069</span><span class="refDoi"> DOI: 10.1016/S0006-3495(97)78756-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0006-3495%2897%2978756-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=9138555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1997&pages=1047-1069&author=M.+K.+Gilsonauthor=J.+A.+Givenauthor=B.+L.+Bushauthor=J.+A.+McCammon&title=The+statistical-thermodynamic+basis+for+computation+of+binding+affinities%3A+a+critical+review&doi=10.1016%2FS0006-3495%2897%2978756-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span></div><div class="casAuthors">Gilson, Michael K.; Given, James A.; Bush, Bruce L.; Mccammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1047-1069</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A review with many refs.  Although the statistical thermodn. of noncovalent binding has been considered in a no. of theor. papers, few methods of computing binding affinities are derived explicitly from this underlying theory.  This has contributed to uncertainty and controversy in certain areas.  This article therefore reviews and extends the connections of some important computational methods with the underlying statistical thermodn.  A derivation of the std. free energy of binding forms the basis of this review.  This derivation should be useful in formulating novel computational methods for predicting binding affinities.  It also permits several important points to be established.  For example, it is found that the double annihilation method of computing binding energy does not yield the std. free energy of binding, but can be modified to yield this quantity.  The derivation also makes it possible to define clearly the changes in translational, rotational, configurational, and solvent entropy upon binding.  It is argued that mol. mass has a negligible effect upon the std. free energy of binding for biomol. systems, and that the cratic entropy defined by Gurney is not a useful concept.  In addn., the use of continuum models of the solvent in binding calcns. is reviewed, and a formalism is presented for incorporating a limited no. of solvent mols. explicitly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZwKHjQ8gW7Vg90H21EOLACvtfcHk0lh1A48ETAvwtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D&md5=2bf97b219b9d15fccab6cb50dc7686aa</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2897%2978756-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252897%252978756-3%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DGiven%26aufirst%3DJ.%2BA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520statistical-thermodynamic%2520basis%2520for%2520computation%2520of%2520binding%2520affinities%253A%2520a%2520critical%2520review%26jtitle%3DBiophys.%2520J.%26date%3D1997%26volume%3D72%26spage%3D1047%26epage%3D1069%26doi%3D10.1016%2FS0006-3495%2897%2978756-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">General, I. J.</span><span> </span><span class="NLM_article-title">A note on the standard state’s binding free energy</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2520</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span><span class="refDoi"> DOI: 10.1021/ct100255z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100255z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovFemsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2520-2524&author=I.+J.+General&title=A+note+on+the+standard+state%E2%80%99s+binding+free+energy&doi=10.1021%2Fct100255z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">A Note on the Standard State's Binding Free Energy</span></div><div class="casAuthors">General, Ignacio J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2520-2524</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The relation between the equil. const. of a given chem. reaction and the assocd. free energy is an issue well studied in chem. books, but when the reaction involves changes in the no. of components in the system, as is the case in binding, things become a little more obscure since one needs to define the so-called std. state.  This is reflected in the literature, esp. in computational studies of binding, where contradicting approaches are followed when treating this problem.  In this work, we present a detailed and unifying explanation of the concepts involved and derive the necessary relations to convert a binding free energy from an arbitrary state to some given std. state.  This is done in three independent ways, from the point of view of (1) the dimensions of the quantities involved, (2) the energy and entropy of the mols., and (3) their chem. potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDFgcYRUjt_bVg90H21EOLACvtfcHk0ljtMRLBaw4I2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovFemsrw%253D&md5=90323fe06cf72991d4e521f0c70c8875</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fct100255z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100255z%26sid%3Dliteratum%253Aachs%26aulast%3DGeneral%26aufirst%3DI.%2BJ.%26atitle%3DA%2520note%2520on%2520the%2520standard%2520state%25E2%2580%2599s%2520binding%2520free%2520energy%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2520%26epage%3D2524%26doi%3D10.1021%2Fct100255z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">De Jong, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vries, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrink, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Determining equilibrium constants for dimerization reactions from molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">1919</span><span class="NLM_x">–</span> <span class="NLM_lpage">1928</span><span class="refDoi"> DOI: 10.1002/jcc.21776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.21776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21469160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVGnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=1919-1928&author=D.+H.+De+Jongauthor=L.+V.+Schaferauthor=A.+H.+De+Vriesauthor=S.+J.+Marrinkauthor=H.+J.+C.+Berendsenauthor=H.+Grubmuller&title=Determining+equilibrium+constants+for+dimerization+reactions+from+molecular+dynamics+simulations&doi=10.1002%2Fjcc.21776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Determining equilibrium constants for dimerization reactions from molecular dynamics simulations</span></div><div class="casAuthors">De Jong, Djurre H.; Schaefer, Lars V.; De Vries, Alex H.; Marrink, Siewert J.; Berendsen, Herman J. C.; Grubmueller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1919-1928</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">With today's available computer power, free energy calcns. from equil. mol. dynamics simulations "via counting" become feasible for an increasing no. of reactions.  An example is the dimerization reaction of transmembrane alpha-helixes.  If an extended simulation of the two helixes covers sufficiently many dimerization and dissocn. events, their binding free energy is readily derived from the fraction of time during which the two helixes are obsd. in dimeric form.  Exactly how the correct value for the free energy is to be calcd., however, is unclear, and indeed several different and contradictory approaches have been used.  In particular, results obtained via Boltzmann statistics differ from those detd. via the law of mass action.  Here, the authors develop a theory that resolves this discrepancy.  The authors show that for simulation systems contg. two mols., the dimerization free energy is given by a formula of the form ΔG .varies. ln(P1/P0).  The authors' theory is also applicable to high concns. that typically have to be used in mol. dynamics simulations to keep the simulation system small, where the textbook dil. approxns. fail.  It also covers simulations with an arbitrary no. of monomers and dimers and provides rigorous error ests.  Comparison with test simulations of a simple Lennard Jones system with various particle nos. as well as with ref. free energy values obtained from radial distribution functions show full agreement for both binding free energies and dimerization statistics.  © 2011 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou9ZX2izyj9LVg90H21EOLACvtfcHk0ljtMRLBaw4I2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVGnurk%253D&md5=4197a85271c389d786af2a286dd6d129</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21776%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BJong%26aufirst%3DD.%2BH.%26aulast%3DSchafer%26aufirst%3DL.%2BV.%26aulast%3DDe%2BVries%26aufirst%3DA.%2BH.%26aulast%3DMarrink%26aufirst%3DS.%2BJ.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DDetermining%2520equilibrium%2520constants%2520for%2520dimerization%2520reactions%2520from%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D1919%26epage%3D1928%26doi%3D10.1002%2Fjcc.21776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-2371&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0ljtMRLBaw4I2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Bakan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lakdawala, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahar, I.</span><span> </span><span class="NLM_article-title">Druggability assessment of allosteric proteins by dynamics simulations in the presence of probe molecules</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2435</span><span class="NLM_x">–</span> <span class="NLM_lpage">2447</span><span class="refDoi"> DOI: 10.1021/ct300117j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300117j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XotFGlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2435-2447&author=A.+Bakanauthor=N.+Nevinsauthor=A.+S.+Lakdawalaauthor=I.+Bahar&title=Druggability+assessment+of+allosteric+proteins+by+dynamics+simulations+in+the+presence+of+probe+molecules&doi=10.1021%2Fct300117j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Assessment of Allosteric Proteins by Dynamics Simulations in the Presence of Probe Molecules</span></div><div class="casAuthors">Bakan, Ahmet; Nevins, Neysa; Lakdawala, Ami S.; Bahar, Ivet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2435-2447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Druggability assessment of a target protein has emerged in recent years as an important concept in hit-to-lead optimization.  A reliable and phys. relevant measure of druggability would allow informed decisions on the risk of investing in a particular target.  Here, we define "druggability" as a quant. est. of binding sites and affinities for a potential drug acting on a specific protein target.  In the present study, we describe a new methodol. that successfully predicts the druggability and maximal binding affinity for a series of challenging targets, including those that function through allosteric mechanisms.  Two distinguishing features of the methodol. are (i) simulation of the binding dynamics of a diversity of probe mols. selected on the basis of an anal. of approved drugs and (ii) identification of druggable sites and estn. of corresponding binding affinities on the basis of an evaluation of the geometry and energetics of bound probe clusters.  The use of the methodol. for a variety of targets such as murine double mutant-2, protein tyrosine phosphatase 1B (PTP1B), lymphocyte function-assocd. antigen 1, vertebrate kinesin-5 (Eg5), and p38 mitogen-activated protein kinase provides examples for which the method correctly captures the location and binding affinities of known drugs.  It also provides insights into novel druggable sites and the target's structural changes that would accommodate, if not promote and stabilize, drug binding.  Notably, the ability to identify high affinity spots even in challenging cases such as PTP1B or Eg5 shows promise as a rational tool for assessing the druggability of protein targets and identifying allosteric or novel sites for drug binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgPQ7A70lFJLVg90H21EOLACvtfcHk0lioJS-LvBfxow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotFGlsLo%253D&md5=328391521ec9e1c32ec8448546f1d3ea</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fct300117j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300117j%26sid%3Dliteratum%253Aachs%26aulast%3DBakan%26aufirst%3DA.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DBahar%26aufirst%3DI.%26atitle%3DDruggability%2520assessment%2520of%2520allosteric%2520proteins%2520by%2520dynamics%2520simulations%2520in%2520the%2520presence%2520of%2520probe%2520molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2435%26epage%3D2447%26doi%3D10.1021%2Fct300117j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckner, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boudon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiradorives, J.</span><span> </span><span class="NLM_article-title">Efficient computation of absolute free-energies of binding by computer-simulations - application to the methane dimer in water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">3742</span><span class="NLM_x">–</span> <span class="NLM_lpage">3746</span><span class="refDoi"> DOI: 10.1063/1.454895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.454895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL1cXlslOkt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1988&pages=3742-3746&author=W.+L.+Jorgensenauthor=J.+K.+Bucknerauthor=S.+Boudonauthor=J.+Tiradorives&title=Efficient+computation+of+absolute+free-energies+of+binding+by+computer-simulations+-+application+to+the+methane+dimer+in+water&doi=10.1063%2F1.454895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient computation of absolute free energies of binding by computer simulations.  Application to the methane dimer in water</span></div><div class="casAuthors">Jorgensen, William L.; Buckner, J. Kathleen; Boudon, Stephane; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3742-6</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An efficient procedure is noted for computing abs. free energies of binding for complexes in soln.  Two series of computer simulations are required in which the substrate is annihilated in the solvent by itself and in the solvated complex.  For illustration, the free energy of binding for two methane-like particles at their contact sepn. of 4 Å was computed in TIP4P water.  Though several alternatives are possible, in this case, Monte Carlo simulations were employed with statistical perturbation theory in the NPT ensemble at 25° and 1 atm.  The results for the free energy of binding as well as for the potential of mean force are consistent with prior findings from the integral equation theory of Pratt and Chandler.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfqADFbgq60LVg90H21EOLACvtfcHk0lioJS-LvBfxow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXlslOkt7g%253D&md5=29dc47048d4a91d6aea0e73c16fcaff8</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1063%2F1.454895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.454895%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBuckner%26aufirst%3DJ.%2BK.%26aulast%3DBoudon%26aufirst%3DS.%26aulast%3DTiradorives%26aufirst%3DJ.%26atitle%3DEfficient%2520computation%2520of%2520absolute%2520free-energies%2520of%2520binding%2520by%2520computer-simulations%2520-%2520application%2520to%2520the%2520methane%2520dimer%2520in%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1988%26volume%3D89%26spage%3D3742%26epage%3D3746%26doi%3D10.1063%2F1.454895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Hermans, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span> </span><span class="NLM_article-title">Inclusion of loss of translational and rotational freedom in theoretical estimates of free energies of binding. Application to a complex of benzene and mutant T4 lysozyme</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">2707</span><span class="NLM_x">–</span> <span class="NLM_lpage">2714</span><span class="refDoi"> DOI: 10.1021/ja963568+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja963568%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXhvV2qtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=2707-2714&author=J.+Hermansauthor=L.+Wang&title=Inclusion+of+loss+of+translational+and+rotational+freedom+in+theoretical+estimates+of+free+energies+of+binding.+Application+to+a+complex+of+benzene+and+mutant+T4+lysozyme&doi=10.1021%2Fja963568%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Inclusion of Loss of Translational and Rotational Freedom in Theoretical Estimates of Free Energies of Binding. Application to a Complex of Benzene and Mutant T4 Lysozyme</span></div><div class="casAuthors">Hermans, Jan; Wang, Lu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2707-2714</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the first complete treatment for calcg. theor. ests. of free energies of formation of macromol.-ligand complexes with mol. dynamics simulations, as the free energy for transforming the ligand into a non-interacting state by gradually diminishing the forces between macromol. (plus solvent) and ligand.  The calcns. become possible due to the introduction of a specially designed potential ("mol. tweezers") which restrains the spatial position and orientation of the ligand mol. and is gradually applied as the transformation proceeds from complexed to non-interacting components.  The binding of benzene to a mutant T4 lysozyme (Morton et al. Biochem. 1995, 34, 8564-8575) has been used as a test case.  The simulations reproduce the value of the free energy of binding (-5.19 kcal/mol if the std. state of benzene is a 1 M aq. soln.) within the sum of exptl. and statistical error.  Another series of such simulations with rigid protein models provides an est. of the dependence of the free energy of binding on the protein conformation.  The free energy of binding is found to decrease in the series: energy-minimized ligand-free protein (-3.5 kcal/mol), energy-minimized ligand-contg. protein (-6.3 kcal/mol), and crystal structure (-8.5 kcal/mol).  The free energy of binding to a series of snapshots from a protein-ligand dynamics trajectory varies between -7 and -9 kcal/mol.  The "cratic" free energy contribution, which corresponds to the loss of translational and rotational freedom of the ligand mol., was estd. at 7 kcal/mol.  It has proved possible to decomp. this into translational and rotational components and, from these free energies, est. the remaining freedom of the benzene in the binding pocket, at 0.6 Å for positional range and 10-15° for angular range, in excellent agreement with the motion obsd. in a dynamics trajectory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1E9cnzjwo4rVg90H21EOLACvtfcHk0lioJS-LvBfxow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvV2qtbg%253D&md5=1af691d300438687118b5ed434cb5180</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fja963568%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja963568%252B%26sid%3Dliteratum%253Aachs%26aulast%3DHermans%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DInclusion%2520of%2520loss%2520of%2520translational%2520and%2520rotational%2520freedom%2520in%2520theoretical%2520estimates%2520of%2520free%2520energies%2520of%2520binding.%2520Application%2520to%2520a%2520complex%2520of%2520benzene%2520and%2520mutant%2520T4%2520lysozyme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D2707%26epage%3D2714%26doi%3D10.1021%2Fja963568%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Hamelberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Standard free energy of releasing a localized water molecule from the binding pockets of proteins: Double-decoupling method</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">7683</span><span class="NLM_x">–</span> <span class="NLM_lpage">7689</span><span class="refDoi"> DOI: 10.1021/ja0377908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0377908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVejtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=7683-7689&author=D.+Hamelbergauthor=J.+A.+McCammon&title=Standard+free+energy+of+releasing+a+localized+water+molecule+from+the+binding+pockets+of+proteins%3A+Double-decoupling+method&doi=10.1021%2Fja0377908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Standard Free Energy of Releasing a Localized Water Molecule from the Binding Pockets of Proteins: Double-Decoupling Method</span></div><div class="casAuthors">Hamelberg, Donald; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7683-7689</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Localized water mols. in the binding pockets of proteins play an important role in noncovalent assocn. of proteins and small drug compds.  At times, the dominant contribution to the binding free energy comes from the release of localized water mols. in the binding pockets of biomols.  Therefore, to quantify the energetic importance of these water mols. for drug design purposes, the authors have used the double-decoupling approach to calc. the std. free energy of tying up a water mol. in the binding pockets of two protein complexes.  The double-decoupling approach is based on the underlying principle of statistical thermodn.  The authors have calcd. the std. free energies of tying up the water mol. in the binding pockets of these complexes to be favorable.  These water mols. stabilize the protein-drug complexes by interacting with the ligands and binding pockets.  The results offer ideas that could be used in optimizing protein-drug interactions, by designing ligands that are capable of targeting localized water mols. in protein binding sites.  The resulting free energy of ligand binding could benefit from the potential free energy gain accompanying the release of these water mols.  Furthermore, the authors have examd. the theor. background of the double-decoupling method and its connection to the mol. dynamics thermodn. integration techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsMbAJlCZuqrVg90H21EOLACvtfcHk0lgrj8ZqAJGFMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVejtb0%253D&md5=a0949ecffa5cfddf1f2128b1660d04f7</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Fja0377908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0377908%26sid%3Dliteratum%253Aachs%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DStandard%2520free%2520energy%2520of%2520releasing%2520a%2520localized%2520water%2520molecule%2520from%2520the%2520binding%2520pockets%2520of%2520proteins%253A%2520Double-decoupling%2520method%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D7683%26epage%3D7689%26doi%3D10.1021%2Fja0377908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Wang, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2814</span><span class="refDoi"> DOI: 10.1529/biophysj.106.084301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1529%2Fbiophysj.106.084301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16844742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2006&pages=2798-2814&author=J.+Y.+Wangauthor=Y.+Q.+Dengauthor=B.+Roux&title=Absolute+binding+free+energy+calculations+using+molecular+dynamics+simulations+with+restraining+potentials&doi=10.1529%2Fbiophysj.106.084301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span></div><div class="casAuthors">Wang, Jiyao; Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2798-2814</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The abs. (std.) binding free energy of eight FK506-related ligands to FKBP12 is calcd. using free energy perturbation mol. dynamics (FEP/MD) simulations with explicit solvent.  A no. of features are implemented to improve the accuracy and enhance the convergence of the calcns.  First, the abs. binding free energy is decompd. into sequential steps during which the ligand-surrounding interactions as well as various biasing potentials restraining the translation, orientation, and conformation of the ligand are turned "on" and "off.".  Second, sampling of the ligand conformation is enforced by a restraining potential based on the root mean-square deviation relative to the bound state conformation.  The effect of all the restraining potentials is rigorously unbiased, and it is shown explicitly that the final results are independent of all artificial restraints.  Third, the repulsive and dispersive free energy contribution arising from the Lennard-Jones interactions of the ligand with its surrounding (protein and solvent) is calcd. using the Weeks-Chandler-Andersen sepn.  This sepn. also improves convergence of the FEP/MD calcns.  Fourth, to decrease the computational cost, only a small no. of atoms in the vicinity of the binding site are simulated explicitly, while all the influence of the remaining atoms is incorporated implicitly using the generalized solvent boundary potential (GSBP) method.  With GSBP, the size of the simulated FKBP12/ligand systems is significantly reduced, from ∼25,000 to 2500.  The computations are very efficient and the statistical error is small (∼1 kcal/mol).  The calcd. binding free energies are generally in good agreement with available exptl. data and previous calcns. (within ∼2 kcal/mol).  The present results indicate that a strategy based on FEP/MD simulations of a reduced GSBP at. model sampled with conformational, translational, and orientational restraining potentials can be computationally inexpensive and accurate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsDftyP7DXobVg90H21EOLACvtfcHk0lgrj8ZqAJGFMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP&md5=010f74f61048d03fee5dce41172dac70</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.084301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.084301%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BY.%26aulast%3DDeng%26aufirst%3DY.%2BQ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculations%2520using%2520molecular%2520dynamics%2520simulations%2520with%2520restraining%2520potentials%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D91%26spage%3D2798%26epage%3D2814%26doi%3D10.1529%2Fbiophysj.106.084301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Helms, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R. C.</span><span> </span><span class="NLM_article-title">Computational alchemy to calculate absolute protein-ligand binding free energy</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">2710</span><span class="NLM_x">–</span> <span class="NLM_lpage">2713</span><span class="refDoi"> DOI: 10.1021/ja9738539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9738539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1cXis12itr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=2710-2713&author=V.+Helmsauthor=R.+C.+Wade&title=Computational+alchemy+to+calculate+absolute+protein-ligand+binding+free+energy&doi=10.1021%2Fja9738539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Alchemy To Calculate Absolute Protein-Ligand Binding Free Energy</span></div><div class="casAuthors">Helms, Volkhard; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2710-2713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability to reliably compute accurate protein-ligand binding affinities is crucial to understanding protein-ligand recognition and to structure-based drug design.  A ligand's binding affinity is specified by its abs. binding free energy, ΔGbind, the free energy difference between the bound and unbound states.  To compute accurate free energy differences by free energy perturbation (FEP), "alchem." rather than phys. processes are usually simulated by mol. dynamics simulations so as to minimize the perturbation to the system.  Here, we report a novel "alchemistic" application of the FEP methodol. involving a large perturbation.  By mutating a ligand with 11 non-hydrogen atoms into six water mols. in the binding site of a protein, we computed a ΔGbind within 3 kJ/mol of the exptl. value.  This is the first successful example of the computation of ΔGbind for a protein:ligand pair with full treatment of the solvent degrees of freedom.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcFHdnjc43ZrVg90H21EOLACvtfcHk0lgkO7qKRpoiwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis12itr0%253D&md5=ec7b248dd479a1538ae74f89561bc19e</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fja9738539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9738539%26sid%3Dliteratum%253Aachs%26aulast%3DHelms%26aufirst%3DV.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DComputational%2520alchemy%2520to%2520calculate%2520absolute%2520protein-ligand%2520binding%2520free%2520energy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D2710%26epage%3D2713%26doi%3D10.1021%2Fja9738539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Woods, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malaisree, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannongbua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, A. J.</span><span> </span><span class="NLM_article-title">A water-swap reaction coordinate for the calculation of absolute protein-ligand binding free energies</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">054114</span><span class="refDoi"> DOI: 10.1063/1.3519057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.3519057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21303099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2011&pages=054114&author=C.+J.+Woodsauthor=M.+Malaisreeauthor=S.+Hannongbuaauthor=A.+J.+Mulholland&title=A+water-swap+reaction+coordinate+for+the+calculation+of+absolute+protein-ligand+binding+free+energies&doi=10.1063%2F1.3519057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">A water-swap reaction coordinate for the calculation of absolute protein-ligand binding free energies</span></div><div class="casAuthors">Woods, Christopher J.; Malaisree, Maturos; Hannongbua, Supot; Mulholland, Adrian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">054114/1-054114/13</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The accurate prediction of abs. protein-ligand binding free energies is one of the grand challenge problems of computational science.  Binding free energy measures the strength of binding between a ligand and a protein, and an algorithm that would allow its accurate prediction would be a powerful tool for rational drug design.  Here we present the development of a new method that allows for the abs. binding free energy of a protein-ligand complex to be calcd. from first principles, using a single simulation.  Our method involves the use of a novel reaction coordinate that swaps a ligand bound to a protein with an equiv. vol. of bulk water.  This water-swap reaction coordinate is built using an identity constraint, which identifies a cluster of water mols. from bulk water that occupies the same vol. as the ligand in the protein active site.  A dual topol. algorithm is then used to swap the ligand from the active site with the identified water cluster from bulk water.  The free energy is then calcd. using replica exchange thermodn. integration.  This returns the free energy change of simultaneously transferring the ligand to bulk water, as an equiv. vol. of bulk water is transferred back to the protein active site.  This, directly, is the abs. binding free energy.  It should be noted that while this reaction coordinate models the binding process directly, an accurate force field and sufficient sampling are still required to allow for the binding free energy to be predicted correctly.  In this paper we present the details and development of this method, and demonstrate how the potential of mean force along the water-swap coordinate can be improved by calibrating the soft-core Coulomb and Lennard-Jones parameters used for the dual topol. calcn.  The optimal parameters were applied to calcns. of protein-ligand binding free energies of a neuraminidase inhibitor (oseltamivir), with these results compared to expt.  These results demonstrate that the water-swap coordinate provides a viable and potentially powerful new route for the prediction of protein-ligand binding free energies.  (c) 2011 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGYbA6KMDz7Vg90H21EOLACvtfcHk0lgkO7qKRpoiwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSqsbo%253D&md5=a83aaffa025f46f3f4248d792bfc0d31</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1063%2F1.3519057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3519057%26sid%3Dliteratum%253Aachs%26aulast%3DWoods%26aufirst%3DC.%2BJ.%26aulast%3DMalaisree%26aufirst%3DM.%26aulast%3DHannongbua%26aufirst%3DS.%26aulast%3DMulholland%26aufirst%3DA.%2BJ.%26atitle%3DA%2520water-swap%2520reaction%2520coordinate%2520for%2520the%2520calculation%2520of%2520absolute%2520protein-ligand%2520binding%2520free%2520energies%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2011%26volume%3D134%26spage%3D054114%26doi%3D10.1063%2F1.3519057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">Rocchia, W.; Masetti, M.; Cavalli, A.</span><span> </span><span class="NLM_article-title">Enhanced sampling methods in drug design</span>. In  <span class="citation_source-book">Physico-Chemical and Computational Approaches to Drug Discovery</span>; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>Chapter 11, pp  <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span>, DOI: <span class="refDoi"> DOI: 10.1039/9781849735377-00273</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1039%2F9781849735377-00273" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=273-301&author=W.+Rocchia&author=M.+Masetti&author=A.+Cavalli&title=Physico-Chemical+and+Computational+Approaches+to+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1039%2F9781849735377-00273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849735377-00273%26sid%3Dliteratum%253Aachs%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DEnhanced%2520sampling%2520methods%2520in%2520drug%2520design%26btitle%3DPhysico-Chemical%2520and%2520Computational%2520Approaches%2520to%2520Drug%2520Discovery%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2012%26spage%3D273%26epage%3D301%26doi%3D10.1039%2F9781849735377-00273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Liu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span> </span><span class="NLM_article-title">Replica exchange with solute tempering: A method for sampling biological systems in explicit water</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">13749</span><span class="NLM_x">–</span> <span class="NLM_lpage">13754</span><span class="refDoi"> DOI: 10.1073/pnas.0506346102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0506346102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16172406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVOqsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=13749-13754&author=P.+Liuauthor=B.+Kimauthor=R.+A.+Friesnerauthor=B.+J.+Berne&title=Replica+exchange+with+solute+tempering%3A+A+method+for+sampling+biological+systems+in+explicit+water&doi=10.1073%2Fpnas.0506346102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Replica exchange with solute tempering: A method for sampling biological systems in explicit water</span></div><div class="casAuthors">Liu, Pu; Kim, Byungchan; Friesner, Richard A.; Berne, B. J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">13749-13754</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">An innovative replica exchange (parallel tempering) method called replica exchange with solute tempering (REST) for the efficient sampling of aq. protein solns. is presented here.  The method bypasses the poor scaling with system size of std. replica exchange and thus reduces the no. of replicas (parallel processes) that must be used.  This redn. is accomplished by deforming the Hamiltonian function for each replica in such a way that the acceptance probability for the exchange of replica configurations does not depend on the no. of explicit water mols. in the system.  For proof of concept, REST is compared with std. replica exchange for an alanine dipeptide mol. in water.  The comparisons confirm that REST greatly reduces the no. of CPUs required by regular replica exchange and increases the sampling efficiency.  This method reduces the CPU time required for calcg. thermodn. avs. and for the ab initio folding of proteins in explicit water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDjppdHbYZJLVg90H21EOLACvtfcHk0liBTrKEvCcaNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVOqsb3P&md5=7ebecadb3ad2037a35bdf92a361e5daf</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506346102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506346102%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26atitle%3DReplica%2520exchange%2520with%2520solute%2520tempering%253A%2520A%2520method%2520for%2520sampling%2520biological%2520systems%2520in%2520explicit%2520water%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D13749%26epage%3D13754%26doi%3D10.1073%2Fpnas.0506346102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Sinko, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Oliveira, C. A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Population based reweighting of scaled molecular dynamics</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">12759</span><span class="NLM_x">–</span> <span class="NLM_lpage">12768</span><span class="refDoi"> DOI: 10.1021/jp401587e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp401587e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVOhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=12759-12768&author=W.+Sinkoauthor=Y.+L.+Miaoauthor=C.+A.+F.+de+Oliveiraauthor=J.+A.+McCammon&title=Population+based+reweighting+of+scaled+molecular+dynamics&doi=10.1021%2Fjp401587e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Population Based Reweighting of Scaled Molecular Dynamics</span></div><div class="casAuthors">Sinko, William; Miao, Yinglong; de Oliveira, Cesar Augusto F.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">12759-12768</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics simulation using enhanced sampling methods is one of the powerful computational tools used to explore protein conformations and free energy landscapes.  Enhanced sampling methods often employ either an increase in temp. or a flattening of the potential energy surface to rapidly sample phase space, and a corresponding reweighting algorithm is used to recover the Boltzmann statistics.  However, potential energies of complex biomols. usually involve large fluctuations on a magnitude of hundreds of kcal/mol despite minimal structural changes during simulation.  This leads to noisy reweighting statistics and complicates the obtainment of accurate final results.  To overcome this common issue in enhanced conformational sampling, we propose a scaled mol. dynamics method, which modifies the biomol. potential energy surface and employs a reweighting scheme based on configurational populations.  Statistical mech. theory is applied to derive the reweighting formula, and the canonical ensemble of simulated structures is recovered accordingly.  Test simulations on alanine dipeptide and the fast folding polypeptide Chignolin exhibit sufficiently enhanced conformational sampling and accurate recovery of free energy surfaces and thermodn. properties.  The results are comparable to long conventional mol. dynamics simulations and exhibit better recovery of canonical statistics over methods which employ a potential energy term in reweighting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogoBM_fh_axLVg90H21EOLACvtfcHk0liBTrKEvCcaNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVOhtrs%253D&md5=99939d7a061f0eb25339b626c1fb7a54</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Fjp401587e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp401587e%26sid%3Dliteratum%253Aachs%26aulast%3DSinko%26aufirst%3DW.%26aulast%3DMiao%26aufirst%3DY.%2BL.%26aulast%3Dde%2BOliveira%26aufirst%3DC.%2BA.%2BF.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DPopulation%2520based%2520reweighting%2520of%2520scaled%2520molecular%2520dynamics%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D12759%26epage%3D12768%26doi%3D10.1021%2Fjp401587e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span> </span><span class="NLM_article-title">Correction to “Replica exchange with solute scaling: a more efficient version of replica exchange with solute tempering (REST2)”</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">11305</span><span class="NLM_x">–</span> <span class="NLM_lpage">11305</span><span class="refDoi"> DOI: 10.1021/jp207931v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp207931v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFSmtr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2011&pages=11305-11305&author=L.+Wangauthor=R.+A.+Friesnerauthor=B.+J.+Berne&title=Correction+to+%E2%80%9CReplica+exchange+with+solute+scaling%3A+a+more+efficient+version+of+replica+exchange+with+solute+tempering+%28REST2%29%E2%80%9D&doi=10.1021%2Fjp207931v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Replica Exchange with Solute Scaling: A More Efficient Version of Replica Exchange with Solute Tempering (REST2) [Erratum to document cited in CA155:147308]</span></div><div class="casAuthors">Wang, Lingle; Friesner, Richard A.; Berne, B. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11305</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">On page 9437, information was omitted from the Acknowledgments section; the omitted information is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq31kSV7snPlLVg90H21EOLACvtfcHk0liBTrKEvCcaNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFSmtr%252FI&md5=f973782faa627fb89e637b07f29286b2</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fjp207931v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp207931v%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26atitle%3DCorrection%2520to%2520%25E2%2580%259CReplica%2520exchange%2520with%2520solute%2520scaling%253A%2520a%2520more%2520efficient%2520version%2520of%2520replica%2520exchange%2520with%2520solute%2520tempering%2520%2528REST2%2529%25E2%2580%259D%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2011%26volume%3D115%26spage%3D11305%26epage%3D11305%26doi%3D10.1021%2Fjp207931v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">On achieving high accuracy and reliability in the calculation of relative protein-ligand binding affinities</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">1937</span><span class="NLM_x">–</span> <span class="NLM_lpage">1942</span><span class="refDoi"> DOI: 10.1073/pnas.1114017109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1114017109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22308365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVSht74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=1937-1942&author=L.+Wangauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=On+achieving+high+accuracy+and+reliability+in+the+calculation+of+relative+protein-ligand+binding+affinities&doi=10.1073%2Fpnas.1114017109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">On achieving high accuracy and reliability in the calculation of relative protein-ligand binding affinities</span></div><div class="casAuthors">Wang, Lingle; Berne, B. J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1937-1942, S1937/1-S1937/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We apply a free energy perturbation simulation method, free energy perturbation/replica exchange with solute tempering, to two modifications of protein-ligand complexes that lead to significant conformational changes, the first in the protein and the second in the ligand.  The approach is shown to facilitate sampling in these challenging cases where high free energy barriers sep. the initial and final conformations and leads to superior convergence of the free energy as demonstrated both by consistency of the results (independence from the starting conformation) and agreement with exptl. binding affinity data.  The second case, consisting of two neutral thrombin ligands that are taken from a recent medicinal chem. program for this interesting pharmaceutical target, is of particular significance in that it demonstrates that good results can be obtained for large, complex ligands, as opposed to relatively simple model systems.  To achieve quant. agreement with expt. in the thrombin case, a next generation force field, Optimized Potentials for Liq. Simulations 2.0, is required, which provides superior charges and torsional parameters as compared to earlier alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6CESSaIXBJbVg90H21EOLACvtfcHk0lgTjhCyeaz3DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVSht74%253D&md5=23db7d18e624d5e1214bbd6702619a87</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1114017109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1114017109%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOn%2520achieving%2520high%2520accuracy%2520and%2520reliability%2520in%2520the%2520calculation%2520of%2520relative%2520protein-ligand%2520binding%2520affinities%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D1937%26epage%3D1942%26doi%3D10.1073%2Fpnas.1114017109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Lionta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spyrou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassilatis, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournia, Z.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening for drug discovery: principles, applications and recent advances</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1923</span><span class="NLM_x">–</span> <span class="NLM_lpage">1938</span><span class="refDoi"> DOI: 10.2174/1568026614666140929124445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F1568026614666140929124445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25262799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1923-1938&author=E.+Liontaauthor=G.+Spyrouauthor=D.+K.+Vassilatisauthor=Z.+Cournia&title=Structure-based+virtual+screening+for+drug+discovery%3A+principles%2C+applications+and+recent+advances&doi=10.2174%2F1568026614666140929124445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Virtual Screening for Drug Discovery: Principles, Applications and Recent Advances</span></div><div class="casAuthors">Lionta, Evanthia; Spyrou, George; Vassilatis, Demetrios K.; Cournia, Zoe</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1923-1938</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug discovery (SBDD) is becoming an essential tool in assisting fast and cost-efficient lead discovery and optimization.  The application of rational, structure-based drug design is proven to be more efficient than the traditional way of drug discovery since it aims to understand the mol. basis of a disease and utilizes the knowledge of the three-dimensional structure of the biol. target in the process.  In this review, we focus on the principles and applications of Virtual Screening (VS) within the context of SBDD and examine different procedures ranging from the initial stages of the process that include receptor and library pre-processing, to docking, scoring and post-processing of top scoring hits.  Recent improvements in structure-based virtual screening (SBVS) efficiency through ensemble docking, induced fit and consensus docking are also discussed.  The review highlights advances in the field within the framework of several success studies that have led to nM inhibition directly from VS and provides recent trends in library design as well as discusses limitations of the method.  Applications of SBVS in the design of substrates for engineered proteins that enable the discovery of new metabolic and signal transduction pathways and the design of inhibitors of multifunctional proteins are also reviewed.  Finally, we contribute two promising VS protocols recently developed by us that aim to increase inhibitor selectivity.  In the first protocol, we describe the discovery of micromolar inhibitors through SBVS designed to inhibit the mutant H1047R PI3Kα kinase.  Second, we discuss a strategy for the identification of selective binders for the RXRα nuclear receptor.  In this protocol, a set of target structures is constructed for ensemble docking based on binding site shape characterization and clustering, aiming to enhance the hit rate of selective inhibitors for the desired protein target through the SBVS process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps80ptfisjzLVg90H21EOLACvtfcHk0lgTjhCyeaz3DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ&md5=0550a396dd011adf52699b091548345b</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.2174%2F1568026614666140929124445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026614666140929124445%26sid%3Dliteratum%253Aachs%26aulast%3DLionta%26aufirst%3DE.%26aulast%3DSpyrou%26aufirst%3DG.%26aulast%3DVassilatis%26aufirst%3DD.%2BK.%26aulast%3DCournia%26aufirst%3DZ.%26atitle%3DStructure-based%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520principles%252C%2520applications%2520and%2520recent%2520advances%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2014%26volume%3D14%26spage%3D1923%26epage%3D1938%26doi%3D10.2174%2F1568026614666140929124445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Matter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotriffer, C.</span><span> </span><span class="NLM_article-title">Applications and success stories in virtual screening</span> <span class="citation_source-journal">Virtual screening: principles, challenges, and practical guidelines</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">358</span><span class="refDoi"> DOI: 10.1002/9783527633326.ch12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2F9783527633326.ch12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2011&pages=319-358&author=H.+Matterauthor=C.+Sotriffer&title=Applications+and+success+stories+in+virtual+screening&doi=10.1002%2F9783527633326.ch12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2F9783527633326.ch12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9783527633326.ch12%26sid%3Dliteratum%253Aachs%26aulast%3DMatter%26aufirst%3DH.%26aulast%3DSotriffer%26aufirst%3DC.%26atitle%3DApplications%2520and%2520success%2520stories%2520in%2520virtual%2520screening%26jtitle%3DVirtual%2520screening%253A%2520principles%252C%2520challenges%252C%2520and%2520practical%2520guidelines%26date%3D2011%26volume%3D48%26spage%3D319%26epage%3D358%26doi%3D10.1002%2F9783527633326.ch12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and scoring in virtual screening for drug discovery: methods and applications</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">949</span><span class="refDoi"> DOI: 10.1038/nrd1549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd1549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15520816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+scoring+in+virtual+screening+for+drug+discovery%3A+methods+and+applications&doi=10.1038%2Fnrd1549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring in virtual screening for drug discovery: methods and applications</span></div><div class="casAuthors">Kitchen, Douglas B.; Decornez, Helene; Furr, John R.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">935-949</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Computational approaches that 'dock' small mols. into the structures of macromol. targets and 'score' their potential complementarity to binding sites are widely used in hit identification and lead optimization.  Indeed, there are now a no. of drugs whose development was heavily influenced by or based on structure-based design and screening strategies, such as HIV protease inhibitors.  Nevertheless, there remain significant challenges in the application of these approaches, in particular in relation to current scoring schemes.  Here, we review key concepts and specific features of small-mol.-protein docking methods, highlight selected applications and discuss recent advances that aim to address the acknowledged limitations of established approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMLibgFe3mfLVg90H21EOLACvtfcHk0lgTjhCyeaz3DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D&md5=875a2b37a4299181509a1922b11dbd2f</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fnrd1549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1549%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520scoring%2520in%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520methods%2520and%2520applications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949%26doi%3D10.1038%2Fnrd1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Jiang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span> </span><span class="NLM_article-title">Soft Docking - Matching of molecular-surface cubes</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">219</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span><span class="refDoi"> DOI: 10.1016/0022-2836(91)90859-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0022-2836%2891%2990859-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=219&publication_year=1991&pages=79-102&author=F.+Jiangauthor=S.+H.+Kim&title=Soft+Docking+-+Matching+of+molecular-surface+cubes&doi=10.1016%2F0022-2836%2891%2990859-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2891%2990859-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252891%252990859-5%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DS.%2BH.%26atitle%3DSoft%2520Docking%2520-%2520Matching%2520of%2520molecular-surface%2520cubes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1991%26volume%3D219%26spage%3D79%26epage%3D102%26doi%3D10.1016%2F0022-2836%2891%2990859-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Leach, A. R.</span><span> </span><span class="NLM_article-title">Ligand docking to proteins with discrete side-chain flexibility</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1016/S0022-2836(05)80038-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0022-2836%2805%2980038-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=8289255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2cXhvVKlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1994&pages=345-356&author=A.+R.+Leach&title=Ligand+docking+to+proteins+with+discrete+side-chain+flexibility&doi=10.1016%2FS0022-2836%2805%2980038-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand docking to proteins with discrete side-chain flexibility</span></div><div class="casAuthors">Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">345-56</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">An algorithm is described that explores the conformational degrees of freedom of the amino acid side-chains and of the ligand when docking a putative ligand into a receptor site.  For a given orientation of the ligand relative to the protein, the method can find the lowest energy combination of amino acid side-chains and ligand conformations as well as all other combinations in order of increasing energy within a specified energy cutoff.  The amino acid side-chains and the ligand are restricted discrete low-energy conformations, detd. for the former by analyzing high-resoln. protein structures and in the latter case from a conformational anal.  Coupled to an algorithm for exploring the six degrees of orientational freedom of the ligand with respect to the receptor, the method can be used to perform conformationally flexible ligand docking.  A combination of two search methods is employed to explore the conformational degrees of freedom: Dead End Elimination and the A* algorithm.  When no ligand is present the approach can be used to predict the lowest energy combinations of side-chain conformations for a given protein backbone structure.  The approach is employed to illustrate how such a procedure can be used to est. the conformational entropy change that accompanies the formation of an intermol. complex between a protein and a ligand and to demonstrate that the protein's conformational entropy may in some cases increase on binding the ligand.  This is due to a modification of the protein's energy hypersurface that makes more conformational states accessible.  The authors' results highlight the need for appropriate methods to est. the strength of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVjujR5VxjuLVg90H21EOLACvtfcHk0lhQBX8LDqYZ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhvVKlu7c%253D&md5=2bf871e7b7b977acf4df1e03371cdeb8</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980038-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980038-5%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DLigand%2520docking%2520to%2520proteins%2520with%2520discrete%2520side-chain%2520flexibility%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D235%26spage%3D345%26epage%3D356%26doi%3D10.1016%2FS0022-2836%2805%2980038-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Meiler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, D.</span><span> </span><span class="NLM_article-title">Rosettaligand: protein-small molecule docking with full side-chain flexibility</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">548</span><span class="refDoi"> DOI: 10.1002/prot.21086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.21086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16972285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=538-548&author=J.+Meilerauthor=D.+Baker&title=Rosettaligand%3A+protein-small+molecule+docking+with+full+side-chain+flexibility&doi=10.1002%2Fprot.21086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility</span></div><div class="casAuthors">Meiler, Jens; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-548</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Protein-small mol. docking algorithms provide a means to model the structure of protein-small mol. complexes in structural detail and play an important role in drug development.  In recent years the necessity of simulating protein side-chain flexibility for an accurate prediction of the protein-small mol. interfaces has become apparent, and an increasing no. of docking algorithms probe different approaches to include protein flexibility.  Here the authors describe a new method for docking small mols. into protein binding sites employing a Monte Carlo minimization procedure in which the rigid body position and orientation of the small mol. and the protein side-chain conformations are optimized simultaneously.  The energy function comprises van der Waals (VDW) interactions, an implicit solvation model, an explicit orientation hydrogen bonding potential, and an electrostatics model.  In an evaluation of the scoring function the computed energy correlated with exptl. small mol. binding energy with a correlation coeff. of 0.63 across a diverse set of 229 protein-small mol. complexes.  The docking method produced lowest energy models with a root mean square deviation (RMSD) smaller than 2 Å in 71 out of 100 protein-small mol. crystal structure complexes (self-docking).  In cross-docking calcns. in which both protein side-chain and small mol. internal degrees of freedom were varied the lowest energy predictions had RMSDs less than 2 Å in 14 of 20 test cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb1x4K4yjDk7Vg90H21EOLACvtfcHk0lhQBX8LDqYZ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ&md5=0e5580cc490c11a7e9c6265c27a14408</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1002%2Fprot.21086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21086%26sid%3Dliteratum%253Aachs%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DRosettaligand%253A%2520protein-small%2520molecule%2520docking%2520with%2520full%2520side-chain%2520flexibility%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26spage%3D538%26epage%3D548%26doi%3D10.1002%2Fprot.21086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Teague, S. J.</span><span> </span><span class="NLM_article-title">Implications of protein flexibility for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">527</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nrd1129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd1129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12838268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=527-541&author=S.+J.+Teague&title=Implications+of+protein+flexibility+for+drug+discovery&doi=10.1038%2Fnrd1129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of protein flexibility for drug discovery</span></div><div class="casAuthors">Teague, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">527-541</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins are in const. motion between different conformational states with similar energies.  This has often been ignored in drug design.  However, protein flexibility is fundamental to understanding the ways in which drugs exert biol. effects, their binding-site location, binding orientation, binding kinetics, metab. and transport.  Protein flexibility allows increased affinity to be achieved between a drug and its target.  This is crucial, because the lipophilicity and no. of polar interactions allowed for an oral drug is limited by absorption, distribution, metab. and toxicol. considerations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8-I4F-CgSbVg90H21EOLACvtfcHk0lhQBX8LDqYZ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D&md5=850b8838a78c1e00493da3089a0c62e9</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fnrd1129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1129%26sid%3Dliteratum%253Aachs%26aulast%3DTeague%26aufirst%3DS.%2BJ.%26atitle%3DImplications%2520of%2520protein%2520flexibility%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D527%26epage%3D541%26doi%3D10.1038%2Fnrd1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Computational chemistry in 25 years</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">9</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1007/s10822-011-9516-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-011-9516-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=9-10&author=R.+Abagyan&title=Computational+chemistry+in+25+years&doi=10.1007%2Fs10822-011-9516-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9516-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9516-z%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DComputational%2520chemistry%2520in%252025%2520years%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D9%26epage%3D10%26doi%3D10.1007%2Fs10822-011-9516-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group">Gane, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dean, P. M.</span><span> </span><span class="NLM_article-title">Recent advances in structure-based rational drug design</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span><span class="refDoi"> DOI: 10.1016/S0959-440X(00)00105-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0959-440X%2800%2900105-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10981625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlvFOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=401-404&author=P.+J.+Ganeauthor=P.+M.+Dean&title=Recent+advances+in+structure-based+rational+drug+design&doi=10.1016%2FS0959-440X%2800%2900105-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in structure-based rational drug design</span></div><div class="casAuthors">Gane, Paul J.; Dean, Philip M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">401-404</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 33 refs.  Two approaches to structure-based drug design, i.e., the docking of known compds. into a target protein and mol. assembly in situ, are seen to be merging technologies.  The need for structural information about drug-protein complexes is now fundamental for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKbI0wbFM0ArVg90H21EOLACvtfcHk0lhQBX8LDqYZ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlvFOmtrs%253D&md5=7cd3881e867d9151557dcdd5f7f0e4bb</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2800%2900105-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252800%252900105-6%26sid%3Dliteratum%253Aachs%26aulast%3DGane%26aufirst%3DP.%2BJ.%26aulast%3DDean%26aufirst%3DP.%2BM.%26atitle%3DRecent%2520advances%2520in%2520structure-based%2520rational%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D10%26spage%3D401%26epage%3D404%26doi%3D10.1016%2FS0959-440X%2800%2900105-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0liiYCHS61DjyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of prescription drugs that interact with the norepinephrine transporter, NET</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">15810</span><span class="NLM_x">–</span> <span class="NLM_lpage">15815</span><span class="refDoi"> DOI: 10.1073/pnas.1106030108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1106030108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21885739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1emtr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=15810-15815&author=A.+Schlessingerauthor=E.+Geierauthor=H.+Fanauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+M.+Giacominiauthor=A.+Sali&title=Structure-based+discovery+of+prescription+drugs+that+interact+with+the+norepinephrine+transporter%2C+NET&doi=10.1073%2Fpnas.1106030108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of prescription drugs that interact with the norepinephrine transporter, NET</span></div><div class="casAuthors">Schlessinger, Avner; Geier, Ethan; Fan, Hao; Irwin, John J.; Shoichet, Brian K.; Giacomini, Kathleen M.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">15810-15815, S15810/1-S15810/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The norepinephrine transporter (NET) transports norepinephrine from the synapse into presynaptic neurons, where norepinephrine regulates signaling pathways assocd. with cardiovascular effects and behavioral traits via binding to various receptors (e.g., β2-adrenergic receptor).  NET is a known target for a variety of prescription drugs, including antidepressants and psychostimulants, and may mediate off-target effects of other prescription drugs.  Here, we identify prescription drugs that bind NET, using virtual ligand screening followed by exptl. validation of predicted ligands.  We began by constructing a comparative structural model of NET based on its alignment to the at. structure of a prokaryotic NET homolog, the leucine transporter LeuT.  The modeled binding site was validated by confirming that known NET ligands can be docked favorably compared to nonbinding mols.  We then computationally screened 6,436 drugs from the Kyoto Encyclopedia of Genes and Genomes (KEGG DRUG) against the NET model.  Ten of the 18 high-scoring drugs tested exptl. were found to be NET inhibitors; five of these were chem. novel ligands of NET.  These results may rationalize the efficacy of several sympathetic (tuaminoheptane) and antidepressant (tranylcypromine) drugs, as well as side effects of diabetes (phenformin) and 'Alzheimer's (talsaclidine) drugs.  The observations highlight the utility of virtual screening against a comparative model, even when the target shares less than 30% sequence identity with its template structure and no known ligands in the primary binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrClJE5OOI_VrVg90H21EOLACvtfcHk0liiYCHS61DjyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1emtr3O&md5=980b33b8f54b1a57396645edd35450e4</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1106030108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1106030108%26sid%3Dliteratum%253Aachs%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DGeier%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStructure-based%2520discovery%2520of%2520prescription%2520drugs%2520that%2520interact%2520with%2520the%2520norepinephrine%2520transporter%252C%2520NET%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D15810%26epage%3D15815%26doi%3D10.1073%2Fpnas.1106030108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Rueda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Consistent improvement of cross-docking results using binding site ensembles generated with elastic network normal modes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">716</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1021/ci8003732</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci8003732" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFGmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=716-725&author=M.+Ruedaauthor=G.+Bottegoniauthor=R.+Abagyan&title=Consistent+improvement+of+cross-docking+results+using+binding+site+ensembles+generated+with+elastic+network+normal+modes&doi=10.1021%2Fci8003732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Consistent Improvement of Cross-Docking Results Using Binding Site Ensembles Generated with Elastic Network Normal Modes</span></div><div class="casAuthors">Rueda, Manuel; Bottegoni, Giovanni; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">716-725</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The representation of protein flexibility is still a challenge for the state-of-the-art flexible ligand docking protocols.  In this article, we use a large and diverse benchmark to prove that is possible to improve consistently the cross-docking performance against a single receptor conformation, using an equil. ensemble of binding site conformers.  The benchmark contained 28 proteins, and our method predicted the top-ranked near native ligand poses 20% more efficiently than using a single receptor.  The multiple conformations were derived from the collective variable space defined by all heavy-atom elastic network normal modes, including backbone and side chains.  We have found that the binding site displacements for best positioning of the ligand seem rather independent from the global collective motions of the protein.  We also found that the no. of binding site conformations needed to represent nonredundant flexibility was <100.  The ensemble of receptor conformations can be generated at our Web site at http://abagyan.scripps.edu/MRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKq-bZloG-zrVg90H21EOLACvtfcHk0liiYCHS61DjyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFGmsrk%253D&md5=4b0c4c58c2d66df41acea3155d6ddd6c</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fci8003732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003732%26sid%3Dliteratum%253Aachs%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DConsistent%2520improvement%2520of%2520cross-docking%2520results%2520using%2520binding%2520site%2520ensembles%2520generated%2520with%2520elastic%2520network%2520normal%2520modes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D716%26epage%3D725%26doi%3D10.1021%2Fci8003732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Campbell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span> </span><span class="NLM_article-title">Ensemble-based docking using biased molecular dynamics</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2127</span><span class="NLM_x">–</span> <span class="NLM_lpage">2138</span><span class="refDoi"> DOI: 10.1021/ci400729j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400729j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovVGmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2127-2138&author=A.+J.+Campbellauthor=M.+L.+Lambauthor=D.+Joseph-McCarthy&title=Ensemble-based+docking+using+biased+molecular+dynamics&doi=10.1021%2Fci400729j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Ensemble-Based Docking Using Biased Molecular Dynamics</span></div><div class="casAuthors">Campbell, Arthur J.; Lamb, Michelle L.; Joseph-McCarthy, Diane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2127-2138</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins are dynamic mols., and understanding their movements, esp. as they relate to mol. recognition and protein-ligand interactions, poses a significant challenge to structure-based drug discovery.  In most instances, protein flexibility is underrepresented in computer-aided drug design due to uncertainties on how it should be accurately modeled as well as the computational cost assocd. with attempting to incorporate flexibility in the calcns.  One approach that aims to address these issues is ensemble-based docking.  With this technique, ligands are docked to an ensemble of rigid protein conformations.  Mol. dynamics (MD) simulations can be used to generate the ensemble of protein conformations for the subsequent docking.  Here we present a novel approach that uses biased-MD simulations to generate the docking ensemble.  The MD simulations are biased toward an initial protein-ligand X-ray complex structure.  The biasing maintains some of the original crystallog. pocket-ligand information and thereby enhances sampling of the more relevant conformational space of the protein.  Resulting trajectories are clustered to select a representative set of protein conformations, and ligands are docked to that reduced set of conformations.  Cross-docking to this ensemble and then selecting the lowest scoring pose enables reliable identification of the correct binding mode.  Various levels of biasing are investigated, and the method is validated for cyclin-dependent kinase 2 and factor Xa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94p-kbdFG87Vg90H21EOLACvtfcHk0ljGxPgtfDsvbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovVGmt7c%253D&md5=5789c563c368e3d8720d62f3e3749dfc</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Fci400729j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400729j%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26atitle%3DEnsemble-based%2520docking%2520using%2520biased%2520molecular%2520dynamics%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2127%26epage%3D2138%26doi%3D10.1021%2Fci400729j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group">Lin, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schames, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Computational drug design accommodating receptor flexibility: The relaxed complex scheme</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">5632</span><span class="NLM_x">–</span> <span class="NLM_lpage">5633</span><span class="refDoi"> DOI: 10.1021/ja0260162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0260162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtFags7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2002&pages=5632-5633&author=J.+H.+Linauthor=A.+L.+Perrymanauthor=J.+R.+Schamesauthor=J.+A.+McCammon&title=Computational+drug+design+accommodating+receptor+flexibility%3A+The+relaxed+complex+scheme&doi=10.1021%2Fja0260162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Drug Design Accommodating Receptor Flexibility: The Relaxed Complex Scheme</span></div><div class="casAuthors">Lin, Jung-Hsin; Perryman, Alexander L.; Schames, Julie R.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5632-5633</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel computational methodol. for drug design that accommodates receptor flexibility is described.  This "relaxed-complex" method recognizes that ligand may bind to conformations that occur only rarely in the dynamics of the receptor.  We have shown that the ligand-enzyme binding modes are very sensitive to the enzyme conformations, and our approach is capable of finding the best ligand-enzyme complexes.  This new method serves as the computational analog of the exptl. "SAR by NMR" and "tether" methods, which permit a building block approach for constructing a very potent drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcdlKmeE9wgbVg90H21EOLACvtfcHk0ljGxPgtfDsvbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtFags7s%253D&md5=3877fe5dad2dd6bdf63a71c5aa84e2dc</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Fja0260162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0260162%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%2BH.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DSchames%26aufirst%3DJ.%2BR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DComputational%2520drug%2520design%2520accommodating%2520receptor%2520flexibility%253A%2520The%2520relaxed%2520complex%2520scheme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D5632%26epage%3D5633%26doi%3D10.1021%2Fja0260162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group">Spyrakis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">BidonChanal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span> </span><span class="NLM_article-title">Protein flexibility and ligand recognition: challenges for molecular modeling</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.2174/156802611794863571</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802611794863571" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20939788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslahsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=192-210&author=F.+Spyrakisauthor=A.+BidonChanalauthor=X.+Barrilauthor=F.+J.+Luque&title=Protein+flexibility+and+ligand+recognition%3A+challenges+for+molecular+modeling&doi=10.2174%2F156802611794863571"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Protein flexibility and ligand recognition: challenges for molecular modeling</span></div><div class="casAuthors">Spyrakis, Francesca; Chanal, Axel Bidon; Barril, Xavier; Luque, F. Javier</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">192-210</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The intrinsic dynamics of macromols. is an essential property to relate the structure of biomol. systems with their function in the cell.  In the field of ligand-receptor recognition, numerous evidences have revealed the limitations of the lock-and-key theory, and the need to elaborate models that take into account the inherent plasticity of biomols., such as the induced-fit model or the existence of an ensemble of pre-equilibrated conformations.  Depending on the nature of the target system, ligand binding can be assocd. with small local adjustments in side chains or even the backbone to large-scale motions of structural fragments, domains or even subunits.  Reproducing the inherent flexibility of biomols. has thus become one of the most challenging issues in mol. modeling and simulation studies, as it has direct implications in our understanding of the structure-function relationships, but even in areas such as virtual screening and structure-based drug discovery.  Given the intrinsic limitation of conventional simulation tools, only events occurring in short time scales can be reproduced at a high accuracy level through all-atom techniques such as Mol. Dynamics simulations.  However, larger structural rearrangements demand the use of enhanced sampling methods relying on modified descriptions of the biomol. system or the potential surface.  This review illustrates the crucial role that structural plasticity plays in mediating ligand recognition through representative examples.  In addn., it discusses some of the most powerful computational tools developed to characterize the conformational flexibility in ligand-receptor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEZna2C0ZlSLVg90H21EOLACvtfcHk0ljGxPgtfDsvbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslahsrc%253D&md5=9d6b8673c6dc5cc6db4f6d0943373826</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.2174%2F156802611794863571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794863571%26sid%3Dliteratum%253Aachs%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DBidonChanal%26aufirst%3DA.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26atitle%3DProtein%2520flexibility%2520and%2520ligand%2520recognition%253A%2520challenges%2520for%2520molecular%2520modeling%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D192%26epage%3D210%26doi%3D10.2174%2F156802611794863571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group">Antolin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hakkaya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaioni, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicciari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macchiarulo, A.</span><span> </span><span class="NLM_article-title">Exploring the effect of PARP-1 flexibility in docking studies</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">201</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2013.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmgm.2013.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Wks7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=192-201&author=A.+A.+Antolinauthor=A.+Carottiauthor=R.+Nutiauthor=A.+Hakkayaauthor=E.+Camaioniauthor=J.+Mestresauthor=R.+Pellicciariauthor=A.+Macchiarulo&title=Exploring+the+effect+of+PARP-1+flexibility+in+docking+studies&doi=10.1016%2Fj.jmgm.2013.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the effect of PARP-1 flexibility in docking studies</span></div><div class="casAuthors">Antolin, Albert A.; Carotti, Andrea; Nuti, Roberto; Hakkaya, Aydie; Camaioni, Emidio; Mestres, Jordi; Pellicciari, Roberto; Macchiarulo, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">192-201</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose)polymerase-1 (PARP-1) is an enzyme belonging to the ADP-ribosyltransferase family.  A large body of works has validated PARP-1 as an attractive drug target for different therapeutic areas, including cancers and ischemia.  Accordingly, sampling the conformational space of the enzyme is pivotal to understand its functions and improve structure-based drug discovery approaches.  In the first part of this study we apply replica exchange mol. dynamic (REMD) simulations to sample the conformational space of the catalytic domain of PARP-1 in the ligand-bound and unbound forms.  In the second part, we assess how and to what extend the emerging enzyme flexibility affects the performance of docking expts. of a library of PARP-1 inhibitors.  This study pinpoints a putative key role of conformational shifts of Leu324, Tyr325 and Lys242 in opening an addnl. binding site pocket that affects the binding of ligands to the catalytic cleft of PARP-1.  Furthermore, it highlights the improvement of the enrichment factor of active ligands obtained in docking expts. when using conformations generated with REMD simulations of ligand-bound PARP-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDVOOBC0BALbVg90H21EOLACvtfcHk0ljGxPgtfDsvbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Wks7%252FI&md5=eaa9dc18eb6e3db0bc9d68afc0714ddb</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2013.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2013.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DAntolin%26aufirst%3DA.%2BA.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DNuti%26aufirst%3DR.%26aulast%3DHakkaya%26aufirst%3DA.%26aulast%3DCamaioni%26aufirst%3DE.%26aulast%3DMestres%26aufirst%3DJ.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DMacchiarulo%26aufirst%3DA.%26atitle%3DExploring%2520the%2520effect%2520of%2520PARP-1%2520flexibility%2520in%2520docking%2520studies%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2013%26volume%3D45%26spage%3D192%26epage%3D201%26doi%3D10.1016%2Fj.jmgm.2013.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Information decay in molecular docking screens against holo, apo, and modeled conformations of enzymes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2895</span><span class="NLM_x">–</span> <span class="NLM_lpage">2907</span><span class="refDoi"> DOI: 10.1021/jm0300330</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0300330" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2895-2907&author=S.+L.+McGovernauthor=B.+K.+Shoichet&title=Information+decay+in+molecular+docking+screens+against+holo%2C+apo%2C+and+modeled+conformations+of+enzymes&doi=10.1021%2Fjm0300330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes</span></div><div class="casAuthors">McGovern, Susan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2895-2907</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking uses the three-dimensional structure of a receptor to screen a small mol. database for potential ligands.  The dependence of docking screens on the conformation of the binding site remains an open question.  To evaluate the information loss that occurs as the active site conformation becomes less defined, a small mol. database was docked against the holo (ligand bound), apo, and homol. modeled structures of 10 different enzyme binding sites.  The holo and apo representations were crystallog. structures taken from the Protein Data Bank (PDB), and the homol.-modeled structures were taken from the publicly available resource ModBase.  The database docked was the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small mols. that contained at least 35 ligands for each of the 10 systems.  In all sites, at least 99% of the mols. in the MDDR were treated as nonbinding decoys.  For each system, the holo, apo, and modeled structures were used to screen the MDDR, and the ability of each structure to enrich the known ligands for that system over random selection was evaluated.  The best overall enrichment was produced by the holo structure in seven systems, the apo structure in two systems, and the modeled structure in one system.  These results suggest that the performance of the docking calcn. is affected by the particular representation of the receptor used in the screen, and that the holo structure is the one most likely to yield the best discrimination between known ligands and decoy mols., but important exceptions to this rule also emerge from this study.  Although each of the holo, apo, and modeled conformations led to enrichment of known ligands in all systems, the enrichment did not always rise to a level judged to be sufficient to justify the effort of a docking screen.  Using a 20-fold enrichment of known ligands over random selection as a rough guideline for what might be enough to justify a docking screen, the holo conformation of the enzyme met this criterion in eight of 10 sites, whereas the apo conformation met this criterion in only two sites and the modeled conformation in three.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo248QTuvleNrVg90H21EOLACvtfcHk0lj2IjOgNF4aFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsFSju7c%253D&md5=bd3145db4e1fbc1288f5dd65d8c52291</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Fjm0300330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0300330%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DInformation%2520decay%2520in%2520molecular%2520docking%2520screens%2520against%2520holo%252C%2520apo%252C%2520and%2520modeled%2520conformations%2520of%2520enzymes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2895%26epage%3D2907%26doi%3D10.1021%2Fjm0300330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group">Rueda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Recipes for the selection of experimental protein conformations for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1021/ci9003943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=186-193&author=M.+Ruedaauthor=G.+Bottegoniauthor=R.+Abagyan&title=Recipes+for+the+selection+of+experimental+protein+conformations+for+virtual+screening&doi=10.1021%2Fci9003943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Recipes for the Selection of Experimental Protein Conformations for Virtual Screening</span></div><div class="casAuthors">Rueda, Manuel; Bottegoni, Giovanni; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-193</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of multiple x-ray protein structures has been reported to be an efficient alternative for the representation of the binding pocket flexibility needed for accurate small mols. docking.  However, the docking performance of the individual single conformations varies widely, and adding certain conformations to an ensemble is even counterproductive.  Here the authors used a very large and diverse benchmark of 1068 x-ray protein conformations of 99 therapeutically relevant proteins, first, to compare the performance of the ensemble and single-conformation docking and, second, to find the properties of the best-performing conformers that can be used to select a smaller set of conformers for ensemble docking.  The conformer selection has been validated through retrospective virtual screening expts. aimed at sepg. known ligand binders from decoys.  The authors found that the conformers cocrystd. with the largest ligands displayed high selectivity for binders, and when combined in ensembles they consistently provided better results than randomly chosen protein conformations.  The use of ensembles encompassing between 3 and 5 exptl. conformations consistently improved the docking accuracy and binders vs. decoys sepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgQPiTcnZy7Vg90H21EOLACvtfcHk0lj2IjOgNF4aFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I&md5=159915d0fb8aa304a49592a0bbb6401f</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fci9003943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003943%26sid%3Dliteratum%253Aachs%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DRecipes%2520for%2520the%2520selection%2520of%2520experimental%2520protein%2520conformations%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D186%26epage%3D193%26doi%3D10.1021%2Fci9003943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, S. D.</span><span> </span><span class="NLM_article-title">Unveiling the full potential of flexible receptor docking using multiple crystallographic structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4432</span><span class="NLM_x">–</span> <span class="NLM_lpage">4443</span><span class="refDoi"> DOI: 10.1021/jm048972v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm048972v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlKisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4432-4443&author=X.+Barrilauthor=S.+D.+Morley&title=Unveiling+the+full+potential+of+flexible+receptor+docking+using+multiple+crystallographic+structures&doi=10.1021%2Fjm048972v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the Full Potential of Flexible Receptor Docking Using Multiple Crystallographic Structures</span></div><div class="casAuthors">Barril, Xavier; Morley, S. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4432-4443</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the current challenges in docking studies is the inclusion of receptor flexibility.  This is crucial because the binding sites of many therapeutic targets sample a wide range of conformational states, which has major consequences on mol. recognition.  In this paper, we make use of very large sets of x-ray structures of cyclin dependent kinase 2 (CDK2) and heat shock protein 90 (HSP90) to assess the performance of flexible receptor docking in binding-mode prediction and virtual screening expts.  Flexible receptor docking performs much better than rigid receptor docking in the former application.  Regarding the latter, we observe a significant improvement in the prediction of binding affinities, but owing to an increase in the no. of false positives, this is not translated into better hit rates.  A simple scoring scheme to correct this limitation is presented.  More importantly, pitfalls inherent to flexible receptor docking have been identified and guidelines are presented to avoid them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmHIP2n9ybSrVg90H21EOLACvtfcHk0lj2IjOgNF4aFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktlKisbk%253D&md5=9a5b15df10784020470cd2972d737995</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1021%2Fjm048972v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm048972v%26sid%3Dliteratum%253Aachs%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DMorley%26aufirst%3DS.%2BD.%26atitle%3DUnveiling%2520the%2520full%2520potential%2520of%2520flexible%2520receptor%2520docking%2520using%2520multiple%2520crystallographic%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4432%26epage%3D4443%26doi%3D10.1021%2Fjm048972v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group">Nichols, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">On the Use of molecular dynamics receptor conformations for virtual screening</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">819</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1007/978-1-61779-465-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2F978-1-61779-465-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22183532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=819&publication_year=2012&pages=93-103&author=S.+E.+Nicholsauthor=R.+Baronauthor=J.+A.+McCammon&title=On+the+Use+of+molecular+dynamics+receptor+conformations+for+virtual+screening&doi=10.1007%2F978-1-61779-465-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">On the use of molecular dynamics receptor conformations for virtual screening</span></div><div class="casAuthors">Nichols Sara E; Baron Riccardo; McCammon J Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Methods in molecular biology (Clifton, N.J.)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">819</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">93-103</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Receptors are inherently dynamic and this flexibility is important to consider when constructing a model of molecular association.  Conformations from molecular dynamics simulations, a well-established method for examining protein dynamics, can be used in virtual screening to account for flexibility in structure-based drug discovery.  Different receptor configurations influence docking results.  Molecular dynamics simulations can provide snapshots that improve virtual screening predictive power over known crystal structures, most likely as a result of sampling more relevant receptor conformations.  Here we highlight some details and nuances of using such snapshots and evaluating them for predictive performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3KPL-W_yq5LP3FVUi_gf8fW6udTcc2eakBrIgF5YHcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjug%253D%253D&md5=8cff07626bfa72238d00170ce4d23ee6</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DNichols%26aufirst%3DS.%2BE.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DOn%2520the%2520Use%2520of%2520molecular%2520dynamics%2520receptor%2520conformations%2520for%2520virtual%2520screening%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D819%26spage%3D93%26epage%3D103%26doi%3D10.1007%2F978-1-61779-465-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group">Amaro, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">An improved relaxed complex scheme for receptor flexibility in computer-aided drug design</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span><span class="refDoi"> DOI: 10.1007/s10822-007-9159-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-007-9159-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18196463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCqur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=693-705&author=R.+E.+Amaroauthor=R.+Baronauthor=J.+A.+McCammon&title=An+improved+relaxed+complex+scheme+for+receptor+flexibility+in+computer-aided+drug+design&doi=10.1007%2Fs10822-007-9159-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">An improved relaxed complex scheme for receptor flexibility in computer-aided drug design</span></div><div class="casAuthors">Amaro, Rommie E.; Baron, Riccardo; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The interactions among assocg. (macro)mols. are dynamic, which adds to the complexity of mol. recognition.  While ligand flexibility is well accounted for in computational drug design, the effective inclusion of receptor flexibility remains an important challenge.  The relaxed complex scheme (RCS) is a promising computational methodol. that combines the advantages of docking algorithms with dynamic structural information provided by mol. dynamics (MD) simulations, therefore explicitly accounting for the flexibility of both the receptor and the docked ligands.  Here, the authors briefly review the RCS and discuss new extensions and improvements of this methodol. in the context of ligand binding to 2 example targets: kinetoplastid RNA editing ligase 1 and the W191G cavity mutant of cytochrome c peroxidase.  The RCS improvements include its extension to virtual screening, more rigorous characterization of local and global binding effects, and methods to improve its computational efficiency by reducing the receptor ensemble to a representative set of configurations.  The choice of receptor ensemble, its influence on the predictive power of RCS, and the current limitations for an accurate treatment of the solvent contributions are also briefly discussed.  Finally, the authors outline potential methodol. improvements that they anticipate will assist future development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouc8v4XJw69rVg90H21EOLACvtfcHk0lg-oMKwLg_6kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCqur7F&md5=e3abac702a530277df56bb155802a649</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9159-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9159-2%26sid%3Dliteratum%253Aachs%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAn%2520improved%2520relaxed%2520complex%2520scheme%2520for%2520receptor%2520flexibility%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D693%26epage%3D705%26doi%3D10.1007%2Fs10822-007-9159-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rueda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Systematic exploitation of multiple receptor conformations for virtual ligand screening</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e18845</span><span class="refDoi"> DOI: 10.1371/journal.pone.0018845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pone.0018845" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e18845&author=G.+Bottegoniauthor=W.+Rocchiaauthor=M.+Ruedaauthor=R.+Abagyanauthor=A.+Cavalli&title=Systematic+exploitation+of+multiple+receptor+conformations+for+virtual+ligand+screening&doi=10.1371%2Fjournal.pone.0018845"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0018845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0018845%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSystematic%2520exploitation%2520of%2520multiple%2520receptor%2520conformations%2520for%2520virtual%2520ligand%2520screening%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De18845%26doi%3D10.1371%2Fjournal.pone.0018845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group">Swanson, J. M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Revisiting free energy calculations: One step closer to rigorous scoring functions and one step beyond MM/PBSA</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span><span class="refDoi"> DOI: 10.1016/S0006-3495(04)74084-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0006-3495%2804%2974084-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2004&pages=67-74&author=J.+M.+J.+Swansonauthor=R.+Henchmanauthor=J.+A.+McCammon&title=Revisiting+free+energy+calculations%3A+One+step+closer+to+rigorous+scoring+functions+and+one+step+beyond+MM%2FPBSA&doi=10.1016%2FS0006-3495%2804%2974084-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2804%2974084-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252804%252974084-9%26sid%3Dliteratum%253Aachs%26aulast%3DSwanson%26aufirst%3DJ.%2BM.%2BJ.%26aulast%3DHenchman%26aufirst%3DR.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DRevisiting%2520free%2520energy%2520calculations%253A%2520One%2520step%2520closer%2520to%2520rigorous%2520scoring%2520functions%2520and%2520one%2520step%2520beyond%2520MM%252FPBSA%26jtitle%3DBiophys.%2520J.%26date%3D2004%26volume%3D86%26spage%3D67%26epage%3D74%26doi%3D10.1016%2FS0006-3495%2804%2974084-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group">de Beer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P. E.</span><span> </span><span class="NLM_article-title">The role of water molecules in computational drug design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.2174/156802610790232288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802610790232288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19929830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=55-66&author=S.+de+Beerauthor=N.+P.+E.+Vermeulen&title=The+role+of+water+molecules+in+computational+drug+design&doi=10.2174%2F156802610790232288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">The role of water molecules in computational drug design</span></div><div class="casAuthors">de Beer, Stephanie B. A.; Vermeulen, Nico P. E.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-66</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Although water mols. are small and only consist of two different atom types, they play various roles in cellular systems.  This review discusses their influence on the binding process between biomacromol. targets and small mol. ligands and how this influence can be modeled in computational drug design approaches.  Both the structure and the thermodn. of active site waters will be discussed as these influence the binding process significantly.  Structurally conserved waters cannot always be detd. exptl. and if obsd., it is not clear if they will be replaced upon ligand binding, even if sufficient space is available.  Methods to predict the presence of water in protein-ligand complexes will be reviewed.  Subsequently, we will discuss methods to include water in computational drug research.  Either as an addnl. factor in automated docking expts., or explicitly in detailed mol. dynamics simulations, the effect of water on the quality of the simulations is significant, but not easily predicted.  The most detailed calcns. involve ests. of the free energy contribution of water mols. to protein-ligand complexes.  These calcns. are computationally demanding, but give insight in the versatility and importance of water in ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhA2h-JAqzhbVg90H21EOLACvtfcHk0lg-oMKwLg_6kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D&md5=5767b8a3f82a33845fae3941a1084067</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.2174%2F156802610790232288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232288%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBeer%26aufirst%3DS.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%2BE.%26atitle%3DThe%2520role%2520of%2520water%2520molecules%2520in%2520computational%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D55%26epage%3D66%26doi%3D10.2174%2F156802610790232288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raco, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span> </span><span class="NLM_article-title">A computational study of the binding of propidium to the peripheral anionic site of human acetylcholinesterase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3991</span><span class="NLM_x">–</span> <span class="NLM_lpage">3999</span><span class="refDoi"> DOI: 10.1021/jm040787u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040787u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFyhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3991-3999&author=A.+Cavalliauthor=G.+Bottegoniauthor=C.+Racoauthor=M.+De+Vivoauthor=M.+Recanatini&title=A+computational+study+of+the+binding+of+propidium+to+the+peripheral+anionic+site+of+human+acetylcholinesterase&doi=10.1021%2Fjm040787u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">A Computational Study of the Binding of Propidium to the Peripheral Anionic Site of Human Acetylcholinesterase</span></div><div class="casAuthors">Cavalli, Andrea; Bottegoni, Giovanni; Raco, Caterina; De Vivo, Marco; Recanatini, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3991-3999</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Combined docking and mol. dynamics (MD) simulations were carried out to investigate the binding mode of propidium at the human acetylcholinesterase (HuAChE) peripheral site.  Two different docking protocols followed by cluster analyses were performed, allowing the identification of five high-populated and low-energy configuration families.  To dynamically explore the behavior of the ligand at the peripheral HuAChE binding site, six complexes (five low-energy and one high-energy) were submitted to 2.5 ns of MD simulations.  The representative propidium/HuAChE binding modes were chosen on the basis of both the docking energy score and the dynamic stability of the complexes throughout the MD simulations.  The most stable poses of propidium at HuAChE PAS were similar to those exptl. detd. with the murine enzyme.  The authors therefore suggest that the present modeling protocol might be used in the dynamic investigation of the interactions of a small-mol. inhibitor with a surface-like binding site of a protein.  Finally, because of the biol. relevance of the target studied here, the present results can be of interest for the rational design of mols. potentially useful in the treatment of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqceHTIQSi_7rVg90H21EOLACvtfcHk0lgNqFtdRTgViw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFyhtL8%253D&md5=ce40924ee4d40405e890566142fa7d46</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1021%2Fjm040787u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040787u%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRaco%26aufirst%3DC.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DRecanatini%26aufirst%3DM.%26atitle%3DA%2520computational%2520study%2520of%2520the%2520binding%2520of%2520propidium%2520to%2520the%2520peripheral%2520anionic%2520site%2520of%2520human%2520acetylcholinesterase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3991%26epage%3D3999%26doi%3D10.1021%2Fjm040787u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group">Kacker, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangold, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Combining dyad protonation and active site plasticity in BACE-1 structure-based drug design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1079</span><span class="NLM_x">–</span> <span class="NLM_lpage">1085</span><span class="refDoi"> DOI: 10.1021/ci200366z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200366z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVOjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1079-1085&author=P.+Kackerauthor=M.+Masettiauthor=M.+Mangoldauthor=G.+Bottegoniauthor=A.+Cavalli&title=Combining+dyad+protonation+and+active+site+plasticity+in+BACE-1+structure-based+drug+design&doi=10.1021%2Fci200366z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Dyad Protonation and Active Site Plasticity in BACE-1 Structure-Based Drug Design</span></div><div class="casAuthors">Kacker, Puneet; Masetti, Matteo; Mangold, Martina; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1079-1085</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of the BACE-1 catalytic dyad to adopt multiple protonation states and the conformational flexibility of the active site have hampered the reliability of computational screening campaigns carried out on this drug target for Alzheimer's disease.  Here, we propose a protocol that, for the first time, combining quantum mech. calcns., mol. dynamics, and conformational ensemble virtual ligand screening addresses these issues simultaneously.  The encouraging results prefigure this approach as a valuable tool for future drug discovery campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKRhdkbh-ZUbVg90H21EOLACvtfcHk0lgNqFtdRTgViw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVOjtro%253D&md5=6e718d38ed3c4fa24820b33161183b78</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Fci200366z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200366z%26sid%3Dliteratum%253Aachs%26aulast%3DKacker%26aufirst%3DP.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DMangold%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DCombining%2520dyad%2520protonation%2520and%2520active%2520site%2520plasticity%2520in%2520BACE-1%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1079%26epage%3D1085%26doi%3D10.1021%2Fci200366z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degliesposti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Rio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sgobba, M.</span><span> </span><span class="NLM_article-title">Binding estimation after refinement, a new automated procedure for the refinement and rescoring of docked ligands in virtual screening</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">286</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2009.00780.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1111%2Fj.1747-0285.2009.00780.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19207463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1WhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2009&pages=283-286&author=G.+Rastelliauthor=G.+Degliespostiauthor=A.+Del+Rioauthor=M.+Sgobba&title=Binding+estimation+after+refinement%2C+a+new+automated+procedure+for+the+refinement+and+rescoring+of+docked+ligands+in+virtual+screening&doi=10.1111%2Fj.1747-0285.2009.00780.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Binding estimation after refinement, a new automated procedure for the refinement and rescoring of docked ligands in virtual screening</span></div><div class="casAuthors">Rastelli, Giulio; Degliesposti, Gianluca; Del Rio, Alberto; Sgobba, Miriam</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-286</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Binding estn. after refinement (BEAR) is a novel automated computational procedure suitable for correcting and overcoming limitations of docking procedures such as poor scoring function and the generation of unreasonable ligand conformations.  BEAR makes use of mol. dynamics simulation followed by MM-PBSA and MM-GBSA binding free energy ests. as tools to refine and rescore the structures obtained from docking virtual screenings.  As binding estn. after refinement relies on mol. dynamics, the entire procedure can be tailored to the needs of the end-user in terms of computational time and the desired accuracy of the results.  In a validation test, binding estn. after refinement and rescoring resulted in a significant enrichment of known ligands among top scoring compds. compared with the original docking results.  Binding estn. after refinement has direct and straightforward application in virtual screening for correcting both false-pos. and false-neg. hits, and should facilitate more reliable selection of biol. active mols. from compd. databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_CIA8Wshju7Vg90H21EOLACvtfcHk0lgNqFtdRTgViw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1WhtL8%253D&md5=8a113fb4e3ae908b3d70eb97a4c964f5</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2009.00780.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2009.00780.x%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DDegliesposti%26aufirst%3DG.%26aulast%3DDel%2BRio%26aufirst%3DA.%26aulast%3DSgobba%26aufirst%3DM.%26atitle%3DBinding%2520estimation%2520after%2520refinement%252C%2520a%2520new%2520automated%2520procedure%2520for%2520the%2520refinement%2520and%2520rescoring%2520of%2520docked%2520ligands%2520in%2520virtual%2520screening%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2009%26volume%3D73%26spage%3D283%26epage%3D286%26doi%3D10.1111%2Fj.1747-0285.2009.00780.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group">Sabbadin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciancetta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moro, S.</span><span> </span><span class="NLM_article-title">Bridging molecular docking to membrane molecular dynamics to investigate GPCR-ligand recognition: the human A(2)A adenosine receptor as a key study</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span><span class="refDoi"> DOI: 10.1021/ci400532b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400532b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=169-183&author=D.+Sabbadinauthor=A.+Ciancettaauthor=S.+Moro&title=Bridging+molecular+docking+to+membrane+molecular+dynamics+to+investigate+GPCR-ligand+recognition%3A+the+human+A%282%29A+adenosine+receptor+as+a+key+study&doi=10.1021%2Fci400532b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Bridging Molecular Docking to Membrane Molecular Dynamics To Investigate GPCR-Ligand Recognition: The Human A2A Adenosine Receptor as a Key Study</span></div><div class="casAuthors">Sabbadin, Davide; Ciancetta, Antonella; Moro, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">169-183</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) represent the largest family of cell-surface receptors and about one-third of the actual targets of clin. used drugs.  Following the progress made in the field of GPCRs structural detn., docking-based screening for novel potent and selective ligands is becoming an increasingly adopted strategy in the drug discovery process.  However, this methodol. is not yet able to anticipate the "bioactive" binding mode and discern it among other conformations.  The authors present a novel approach consisting in the integration of mol. docking and membrane MD simulations with the aim to merge the rapid sampling of ligand poses into the binding site, typical of docking algorithms, with the thermodn. accuracy of MD simulations in describing, at the mol. level, the stability a GPCR-ligand complex embedded into explicit lipid-water environment.  To validate the authors' approach, the authors have chosen as a key study the human A2A adenosine receptor (hA2A AR) and selected four receptor-antagonist complexes and one receptor-agonist complex that have been recently crystd.  In light of the obtained results, the authors believe that the authors' novel strategy can be extended to other GPCRs and might represent a valuable tool to anticipate the "bioactive" conformation of high-affinity ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpddvdZooeg-LVg90H21EOLACvtfcHk0liAopSmi1Yvpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurvF&md5=eec1db7115955775558cfff9fbaaed9e</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Fci400532b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400532b%26sid%3Dliteratum%253Aachs%26aulast%3DSabbadin%26aufirst%3DD.%26aulast%3DCiancetta%26aufirst%3DA.%26aulast%3DMoro%26aufirst%3DS.%26atitle%3DBridging%2520molecular%2520docking%2520to%2520membrane%2520molecular%2520dynamics%2520to%2520investigate%2520GPCR-ligand%2520recognition%253A%2520the%2520human%2520A%25282%2529A%2520adenosine%2520receptor%2520as%2520a%2520key%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D169%26epage%3D183%26doi%3D10.1021%2Fci400532b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group">Lauro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferruz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">Reranking docking poses using molecular simulations and approximate free energy methods</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2185</span><span class="NLM_x">–</span> <span class="NLM_lpage">2189</span><span class="refDoi"> DOI: 10.1021/ci500309a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500309a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyns7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2185-2189&author=G.+Lauroauthor=N.+Ferruzauthor=S.+Fulleauthor=M.+J.+Harveyauthor=P.+W.+Finnauthor=G.+De+Fabritiis&title=Reranking+docking+poses+using+molecular+simulations+and+approximate+free+energy+methods&doi=10.1021%2Fci500309a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Reranking Docking Poses Using Molecular Simulations and Approximate Free Energy Methods</span></div><div class="casAuthors">Lauro, G.; Ferruz, N.; Fulle, S.; Harvey, M. J.; Finn, P. W.; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2185-2189</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fast and accurate identification of active compds. is essential for effective use of virtual screening workflows.  Here, the authors have compared the ligand-ranking efficiency of the linear interaction energy (LIE) method against std. docking approaches.  Using a trypsin set of 1549 compds., the authors performed 12,250 mol dynamics simulations.  The LIE method proved effective but did not yield results significantly better than those obtained with docking codes.  The entire database of simulations is released.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW19HnT0xTwrVg90H21EOLACvtfcHk0liAopSmi1Yvpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyns7bM&md5=9ab6c274c82f8eec7bb97a8223503d36</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Fci500309a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500309a%26sid%3Dliteratum%253Aachs%26aulast%3DLauro%26aufirst%3DG.%26aulast%3DFerruz%26aufirst%3DN.%26aulast%3DFulle%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DM.%2BJ.%26aulast%3DFinn%26aufirst%3DP.%2BW.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DReranking%2520docking%2520poses%2520using%2520molecular%2520simulations%2520and%2520approximate%2520free%2520energy%2520methods%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2185%26epage%3D2189%26doi%3D10.1021%2Fci500309a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group">Shirts, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Computing: screen savers of the world unite!</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">1903</span><span class="NLM_x">–</span> <span class="NLM_lpage">1904</span><span class="refDoi"> DOI: 10.1126/science.290.5498.1903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.290.5498.1903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=17742054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslKitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2000&pages=1903-1904&author=M.+Shirtsauthor=V.+S.+Pande&title=Computing%3A+screen+savers+of+the+world+unite%21&doi=10.1126%2Fscience.290.5498.1903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Perspectives: Computing: Screen savers of the World unite!</span></div><div class="casAuthors">Shirts, Michael; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">5498</span>),
    <span class="NLM_cas:pages">1903-1904</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW_KxVt3dZRrVg90H21EOLACvtfcHk0liAopSmi1Yvpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslKitrw%253D&md5=920269eabb61d86cd45aa74cfa89a246</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1126%2Fscience.290.5498.1903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.290.5498.1903%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DComputing%253A%2520screen%2520savers%2520of%2520the%2520world%2520unite%2521%26jtitle%3DScience%26date%3D2000%26volume%3D290%26spage%3D1903%26epage%3D1904%26doi%3D10.1126%2Fscience.290.5498.1903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlow, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Pathway and mechanism of drug binding to G-protein-coupled receptors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">13118</span><span class="NLM_x">–</span> <span class="NLM_lpage">13123</span><span class="refDoi"> DOI: 10.1073/pnas.1104614108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1104614108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=21778406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13118-13123&author=R.+O.+Drorauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+W.+Borhaniauthor=P.+Maragakisauthor=Y.+Shanauthor=H.+Xuauthor=D.+E.+Shaw&title=Pathway+and+mechanism+of+drug+binding+to+G-protein-coupled+receptors&doi=10.1073%2Fpnas.1104614108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Pathway and mechanism of drug binding to G-protein-coupled receptors</span></div><div class="casAuthors">Dror, Ron O.; Pan, Albert C.; Arlow, Daniel H.; Borhani, David W.; Maragakis, Paul; Shan, Yibing; Xu, Huafeng; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">13118-13123, S13118/1-S13118/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">How drugs bind to their receptors-from initial assocn., through drug entry into the binding pocket, to adoption of the final bound conformation, or "pose"-has remained unknown, even for G-protein-coupled receptor modulators, which constitute 1/3 of all marketed drugs.  Here, the authors captured this pharmaceutically crit. process in at. detail using the 1st unbiased mol. dynamics simulations in which drug mols. spontaneously assoc. with G-protein-coupled receptors to achieve final poses matching those detd. crystallog.  The authors found that several beta blockers and a beta agonist all traversed the same well-defined, dominant pathway as they bound to the β1- and β2-adrenergic receptors, initially making contact with a vestibule on each receptor's extracellular surface.  Surprisingly, assocn. with this vestibule, at a distance of 15 Å from the binding pocket, often presented the largest energetic barrier to binding, despite the fact that subsequent entry into the binding pocket required the receptor to deform and the drug to squeeze through a narrow passage.  The early barrier appeared to reflect the substantial dehydration that takes place as the drug assocs. with the vestibule.  The at.-level description of the binding process suggests opportunities for allosteric modulation and provides a structural foundation for future optimization of drug-receptor binding and unbinding rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKUYFtkTGwf7Vg90H21EOLACvtfcHk0liAopSmi1Yvpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE&md5=9cfa8673217931cacd586321309fa72e</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1104614108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1104614108%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DPathway%2520and%2520mechanism%2520of%2520drug%2520binding%2520to%2520G-protein-coupled%2520receptors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13118%26epage%3D13123%26doi%3D10.1073%2Fpnas.1104614108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group">Foda, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, M. A.</span><span> </span><span class="NLM_article-title">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">5939</span><span class="refDoi"> DOI: 10.1038/ncomms6939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fncomms6939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25600932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5939&author=Z.+H.+Fodaauthor=Y.+Shanauthor=E.+T.+Kimauthor=D.+E.+Shawauthor=M.+A.+Seeliger&title=A+dynamically+coupled+allosteric+network+underlies+binding+cooperativity+in+Src+kinase&doi=10.1038%2Fncomms6939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">A dynamically coupled allosteric network underlies binding cooperativity in Src kinase</span></div><div class="casAuthors">Foda, Zachariah H.; Shan, Yibing; Kim, Eric T.; Shaw, David E.; Seeliger, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5939</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein tyrosine kinases are attractive drug targets because many human diseases are assocd. with the deregulation of kinase activity.  However, how the catalytic kinase domain integrates different signals and switches from an active to an inactive conformation remains incompletely understood.  Here we identify an allosteric network of dynamically coupled amino acids in Src kinase that connects regulatory sites to the ATP- and substrate-binding sites.  Surprisingly, reactants (ATP and peptide substrates) bind with neg. cooperativity to Src kinase while products (ADP and phosphopeptide) bind with pos. cooperativity.  We confirm the mol. details of the signal relay through the allosteric network by biochem. studies.  Expts. on two addnl. protein tyrosine kinases indicate that the allosteric network may be largely conserved among these enzymes.  Our work provides new insights into the regulation of protein tyrosine kinases and establishes a potential conduit by which resistance mutations to ATP-competitive kinase inhibitors can affect their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGfRZJun_T1rVg90H21EOLACvtfcHk0ljUYVYqKFvmpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lu7bJ&md5=cc24df9830ccb1525f66cc554795965c</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fncomms6939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6939%26sid%3Dliteratum%253Aachs%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26atitle%3DA%2520dynamically%2520coupled%2520allosteric%2520network%2520underlies%2520binding%2520cooperativity%2520in%2520Src%2520kinase%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D5939%26doi%3D10.1038%2Fncomms6939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group">Shan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkhipov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jumper, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Oncogenic mutations counteract intrinsic disorder in the EGFR kinase and promote receptor dimerization</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">860</span><span class="NLM_x">–</span> <span class="NLM_lpage">870</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.02.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.cell.2012.02.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22579287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFaju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=860-870&author=Y.+Shanauthor=M.+P.+Eastwoodauthor=X.+Zhangauthor=E.+T.+Kimauthor=A.+Arkhipovauthor=R.+O.+Drorauthor=J.+Jumperauthor=J.+Kuriyanauthor=D.+E.+Shaw&title=Oncogenic+mutations+counteract+intrinsic+disorder+in+the+EGFR+kinase+and+promote+receptor+dimerization&doi=10.1016%2Fj.cell.2012.02.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic mutations counteract intrinsic disorder in the EGFR kinase and promote receptor dimerization</span></div><div class="casAuthors">Shan, Yibing; Eastwood, Michael P.; Zhang, Xuewu; Kim, Eric T.; Arkhipov, Anton; Dror, Ron O.; Jumper, John; Kuriyan, John; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">860-870</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mutation and overexpression of the epidermal growth factor receptor (EGFR) are assocd. with the development of a variety of cancers, making this prototypical dimerization-activated receptor tyrosine kinase a prominent target of cancer drugs.  Using long-timescale mol. dynamics simulations, we find that the N lobe dimerization interface of the wild-type EGFR kinase domain is intrinsically disordered and that it becomes ordered only upon dimerization.  Our simulations suggest, moreover, that some cancer-linked mutations distal to the dimerization interface, particularly the widespread L834R mutation (also referred to as L858R), facilitate EGFR dimerization by suppressing this local disorder.  Corroborating these findings, our biophys. expts. and kinase enzymic assays indicate that the L834R mutation causes abnormally high activity primarily by promoting EGFR dimerization rather than by allowing activation without dimerization.  We also find that phosphorylation of EGFR kinase domain at Tyr845 may suppress the intrinsic disorder, suggesting a mol. mechanism for autonomous EGFR signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhEY912uUTrVg90H21EOLACvtfcHk0ljUYVYqKFvmpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFaju7k%253D&md5=8ab973e5a5626ac495d176322f30d2b7</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.063%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DY.%26aulast%3DEastwood%26aufirst%3DM.%2BP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DE.%2BT.%26aulast%3DArkhipov%26aufirst%3DA.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DJumper%26aufirst%3DJ.%26aulast%3DKuriyan%26aufirst%3DJ.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DOncogenic%2520mutations%2520counteract%2520intrinsic%2520disorder%2520in%2520the%2520EGFR%2520kinase%2520and%2520promote%2520receptor%2520dimerization%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D860%26epage%3D870%26doi%3D10.1016%2Fj.cell.2012.02.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favia, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Convertino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">The molecular basis for dual fatty acid amide hydrolase (FAAH)/cyclooxygenase (COX) inhibition</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1002/cmdc.201500507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fcmdc.201500507" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=G.+Palermoauthor=A.+D.+Faviaauthor=M.+Convertinoauthor=M.+De+Vivo&title=The+molecular+basis+for+dual+fatty+acid+amide+hydrolase+%28FAAH%29%2Fcyclooxygenase+%28COX%29+inhibition&doi=10.1002%2Fcmdc.201500507"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500507%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DFavia%26aufirst%3DA.%2BD.%26aulast%3DConvertino%26aufirst%3DM.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DThe%2520molecular%2520basis%2520for%2520dual%2520fatty%2520acid%2520amide%2520hydrolase%2520%2528FAAH%2529%252Fcyclooxygenase%2520%2528COX%2529%2520inhibition%26jtitle%3DChemMedChem%26date%3D2015%26doi%3D10.1002%2Fcmdc.201500507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piomelli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Covalent inhibitors of fatty acid amide hydrolase: a rationale for the activity of piperidine and piperazine aryl ureas</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6612</span><span class="NLM_x">–</span> <span class="NLM_lpage">6623</span><span class="refDoi"> DOI: 10.1021/jm2004283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2004283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6612-6623&author=G.+Palermoauthor=D.+Branduardiauthor=M.+Masettiauthor=A.+Lodolaauthor=M.+Morauthor=D.+Piomelliauthor=A.+Cavalliauthor=M.+De+Vivo&title=Covalent+inhibitors+of+fatty+acid+amide+hydrolase%3A+a+rationale+for+the+activity+of+piperidine+and+piperazine+aryl+ureas&doi=10.1021%2Fjm2004283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibitors of Fatty Acid Amide Hydrolase: A Rationale for the Activity of Piperidine and Piperazine Aryl Ureas</span></div><div class="casAuthors">Palermo, Giulia; Branduardi, Davide; Masetti, Matteo; Lodola, Alessio; Mor, Marco; Piomelli, Daniele; Cavalli, Andrea; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6612-6623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, covalent drugs have attracted great interest in the drug discovery community, with successful examples that have demonstrated their therapeutic effects.  Here, we focus on the covalent inhibition of the fatty acid amide hydrolase (FAAH), which is a promising strategy in the treatment of pain and inflammation.  Among the most recent and potent FAAH inhibitors (FAAHi), there are the cyclic piperidine and piperazine aryl ureas.  FAAH hydrolyzes efficiently the amide bond of these compds., forming a covalent enzyme-inhibitor adduct.  To rationalize this exptl. evidence, we performed an extensive computational anal. centered on piperidine-based PF750 (1) and piperazine-based JNJ1661010 (2), two potent lead compds. used to generate covalent inhibitors as clin. candidates.  We found that FAAH induces a distortion of the amide bond of the piperidine and piperazine aryl ureas.  Quantum mechanics/mol. mechanics ΔELUMO-HOMO energies indicate that the obsd. enzyme-induced distortion of the amide bond favors the formation of a covalent FAAH-inhibitor adduct.  These findings could help in the rational structure-based design of novel covalent FAAHi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXM0Vnrifb87Vg90H21EOLACvtfcHk0lji-b08pQQFNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsb3O&md5=3130e9ef929a52cc0900ad77b38a7453</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Fjm2004283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2004283%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DLodola%26aufirst%3DA.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DCovalent%2520inhibitors%2520of%2520fatty%2520acid%2520amide%2520hydrolase%253A%2520a%2520rationale%2520for%2520the%2520activity%2520of%2520piperidine%2520and%2520piperazine%2520aryl%2520ureas%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6612%26epage%3D6623%26doi%3D10.1021%2Fjm2004283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stenta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Peraro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">Molecular simulations highlight the role of metals in catalysis and inhibition of type II topoisomerase</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">857</span><span class="NLM_x">–</span> <span class="NLM_lpage">862</span><span class="refDoi"> DOI: 10.1021/ct300691u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300691u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1Wiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=857-862&author=G.+Palermoauthor=M.+Stentaauthor=A.+Cavalliauthor=M.+Dal+Peraroauthor=M.+De+Vivo&title=Molecular+simulations+highlight+the+role+of+metals+in+catalysis+and+inhibition+of+type+II+topoisomerase&doi=10.1021%2Fct300691u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Simulations Highlight the Role of Metals in Catalysis and Inhibition of Type II Topoisomerase</span></div><div class="casAuthors">Palermo, Giulia; Stenta, Marco; Cavalli, Andrea; Dal Peraro, Matteo; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">857-862</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Type II topoisomerase (topoII) is a metalloenzyme targeted by clin. antibiotics and anticancer agents.  Here, we integrate existing structural data with mol. simulation and propose a model for the yet uncharacterized structure of the reactant state of topoII.  This model describes a canonical two-metal-ion mechanism and suggests how the metals could rearrange at the catalytic pocket during enzymic turnover, explaining also exptl. evidence for topoII inhibition.  These results call for further exptl. validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqWdNp-7sLNbVg90H21EOLACvtfcHk0lji-b08pQQFNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1Wiug%253D%253D&md5=3cd42ed1cc8929e2a3ca387751382460</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1021%2Fct300691u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300691u%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DStenta%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DDal%2BPeraro%26aufirst%3DM.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DMolecular%2520simulations%2520highlight%2520the%2520role%2520of%2520metals%2520in%2520catalysis%2520and%2520inhibition%2520of%2520type%2520II%2520topoisomerase%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D857%26epage%3D862%26doi%3D10.1021%2Fct300691u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Flexible docking in solution using metadynamics</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">2600</span><span class="NLM_x">–</span> <span class="NLM_lpage">2607</span><span class="refDoi"> DOI: 10.1021/ja0445950</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0445950" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVSntbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=2600-2607&author=F.+L.+Gervasioauthor=A.+Laioauthor=M.+Parrinello&title=Flexible+docking+in+solution+using+metadynamics&doi=10.1021%2Fja0445950"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible Docking in Solution Using Metadynamics</span></div><div class="casAuthors">Gervasio, Francesco Luigi; Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2600-2607</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We apply our recently developed metadynamics method to the docking of ligands on flexible receptors in water soln.  This method mimics the real dynamics of a ligand exiting or entering an enzyme and in so doing reconstructs the free energy surface.  We apply it to four docking cases: β-trypsin/benzamidine, β-trypsin/chlorobenzamidine, Ig McPC-603/phosphocholine, and cyclin-dependent kinase 2/staurosporine.  In every case studied, the method is able to predict the docked geometry and the free energy of docking.  Its added value with respect to many other available methods is that it reconstructs the complete free energy surface, including all the relevant min. and the barriers between them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbeUwtpDaXtrVg90H21EOLACvtfcHk0liMKLhZDgOWug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVSntbw%253D&md5=b2e17e2ebb518d0754d8f6f693557820</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Fja0445950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0445950%26sid%3Dliteratum%253Aachs%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFlexible%2520docking%2520in%2520solution%2520using%2520metadynamics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D2600%26epage%3D2607%26doi%3D10.1021%2Fja0445950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group">Colizzi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perozzo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scapozza, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Single-molecule pulling simulations can discern active from inactive enzyme inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">7361</span><span class="NLM_x">–</span> <span class="NLM_lpage">7371</span><span class="refDoi"> DOI: 10.1021/ja100259r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja100259r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=7361-7371&author=F.+Colizziauthor=R.+Perozzoauthor=L.+Scapozzaauthor=M.+Recanatiniauthor=A.+Cavalli&title=Single-molecule+pulling+simulations+can+discern+active+from+inactive+enzyme+inhibitors&doi=10.1021%2Fja100259r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Single-Molecule Pulling Simulations Can Discern Active from Inactive Enzyme Inhibitors</span></div><div class="casAuthors">Colizzi, Francesco; Perozzo, Remo; Scapozza, Leonardo; Recanatini, Maurizio; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7361-7371</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding ligand-protein recognition and interaction processes is of primary importance for structure-based drug design.  Traditionally, several approaches combining docking and mol. dynamics (MD) simulations have been exploited to investigate the physicochem. properties of complexes of pharmaceutical interest.  Even if the geometric properties of a modeled protein-ligand complex can be well predicted by computational methods, it is challenging to rank a series of analogs in a consistent fashion with biol. data.  In the unique β-hydroxyacyl-ACP dehydratase of Plasmodium falciparum (PfFabZ), the application of std. mol. docking and MD simulations was partially sufficient to shed light on the activity of previously discovered inhibitors.  Complementing docking results with atomistic simulations in the steered mol. dynamics (SMD) framework, we devised an in silico approach to study mol. interactions and to compare the binding characteristics of ligand analogs.  We hypothesized an interaction model that both explained the biol. activity of known ligands, and provided insight into designing novel enzyme inhibitors.  Mimicking single-mol. pulling expts., we used SMD-derived force profiles to discern active from inactive compds. for the first time.  A new compd. was designed and its biol. activity toward the PfFabZ enzyme predicted.  Finally, the computational predictions were exptl. confirmed, highlighting the robustness of the drug design approach presented herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3wn1eZl92ErVg90H21EOLACvtfcHk0liMKLhZDgOWug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtro%253D&md5=6c52680599d889ee992b5445bc37b498</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fja100259r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja100259r%26sid%3Dliteratum%253Aachs%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DPerozzo%26aufirst%3DR.%26aulast%3DScapozza%26aufirst%3DL.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSingle-molecule%2520pulling%2520simulations%2520can%2520discern%2520active%2520from%2520inactive%2520enzyme%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D7361%26epage%3D7371%26doi%3D10.1021%2Fja100259r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group">Bash, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, U. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, F. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Calculation of the relative change in binding free energy of a protein-inhibitor complex</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1126/science.3810157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.3810157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=3810157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL2sXitV2lt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=574-6&author=P.+A.+Bashauthor=U.+C.+Singhauthor=F.+K.+Brownauthor=R.+Langridgeauthor=P.+A.+Kollman&title=Calculation+of+the+relative+change+in+binding+free+energy+of+a+protein-inhibitor+complex&doi=10.1126%2Fscience.3810157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of the relative change in binding free energy of a protein-inhibitor complex</span></div><div class="casAuthors">Bash, Paul A.; Singh, U. Chandra; Brown, Frank K.; Langridge, Robert; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">4788</span>),
    <span class="NLM_cas:pages">574-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The relative free energy of binding was calcd. for thermolysin complexed with a pair of phosphonamidate and phosphonate ester inhibitors by a thermodn. perturbation method implemented with mol. dynamics.  The calcd. difference in free energy of binding was 4.21 kcal/mol.  This compares well with the exptl. value of 4.1 kcal/mol.  The method is general and can be used to det. a change or mutation in any system that can be suitably represented.  It is likely to prove useful for protein and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrco-HX_Uxaf7Vg90H21EOLACvtfcHk0liMKLhZDgOWug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXitV2lt74%253D&md5=159b4f27c4ded3b388e1ead2174981b1</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1126%2Fscience.3810157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.3810157%26sid%3Dliteratum%253Aachs%26aulast%3DBash%26aufirst%3DP.%2BA.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26aulast%3DBrown%26aufirst%3DF.%2BK.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DCalculation%2520of%2520the%2520relative%2520change%2520in%2520binding%2520free%2520energy%2520of%2520a%2520protein-inhibitor%2520complex%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D574%26epage%3D6%26doi%3D10.1126%2Fscience.3810157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Efficient drug lead discovery and optimization</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">733</span><span class="refDoi"> DOI: 10.1021/ar800236t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800236t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=724-733&author=W.+L.+Jorgensen&title=Efficient+drug+lead+discovery+and+optimization&doi=10.1021%2Far800236t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Drug Lead Discovery and Optimization</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">724-733</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  During the 1980s, advances in the abilities to perform computer simulations of chem. and biomol. systems and to calc. free energy changes led to the expectation that such methodol. would soon show great utility for guiding mol. design.  Important potential applications included design of selective receptors, catalysts, and regulators of biol. function including enzyme inhibitors.  This time also saw the rise of high-throughput screening and combinatorial chem. along with complementary computational methods for de novo design and virtual screening including docking.  These technologies appeared poised to deliver diverse lead compds. for any biol. target.  As with many technol. advances, realization of the expectations required significant addnl. effort and time.  However, as summarized here, striking success has now been achieved for computer-aided drug lead generation and optimization.  De novo design using both mol. growing and docking are illustrated for lead generation, and lead optimization features free energy perturbation calcns. in conjunction with Monte Carlo statistical mechanics simulations for protein-inhibitor complexes in aq. soln.  The specific applications are to the discovery of non-nucleoside inhibitors of HIV reverse transcriptase (HIV-RT) and inhibitors of the binding of the proinflammatory cytokine MIF to its receptor CD74.  A std. protocol is presented that includes scans for possible addns. of small substituents to a mol. core, interchange of heterocycles, and focused optimization of substituents at one site.  Initial leads with activities at low-micromolar concns. have been advanced rapidly to low-nanomolar inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0YpWuo1jCLVg90H21EOLACvtfcHk0liMKLhZDgOWug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D&md5=9641587769701c6541b38e18fa05538a</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1021%2Far800236t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800236t%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEfficient%2520drug%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2009%26volume%3D42%26spage%3D724%26epage%3D733%26doi%3D10.1021%2Far800236t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group">Wong, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccammon, J. A.</span><span> </span><span class="NLM_article-title">Dynamics and design of enzymes and inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">3830</span><span class="NLM_x">–</span> <span class="NLM_lpage">3832</span><span class="refDoi"> DOI: 10.1021/ja00273a048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00273a048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL28Xkt1Sru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=1986&pages=3830-3832&author=C.+F.+Wongauthor=J.+A.+Mccammon&title=Dynamics+and+design+of+enzymes+and+inhibitors&doi=10.1021%2Fja00273a048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics and design of enzymes and inhibitors</span></div><div class="casAuthors">Wong, Chung F.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3830-2</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Mol. dynamics computer simulations are analyzed by a new approach to predict the effects of structural modifications on enzyme-inhibitor binding in water.  The relative free energy of binding 2 different benzamidines to trypsin has been calcd., as has the relative free energy of binding benzamidine to native and mutant trypsin.  The agreement with exptl. results is encouraging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQCyux77gvDrVg90H21EOLACvtfcHk0lhLqDp4OINLkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28Xkt1Sru70%253D&md5=36009cf12837f48fe1c92224bcdfd763</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1021%2Fja00273a048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00273a048%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.%2BF.%26aulast%3DMccammon%26aufirst%3DJ.%2BA.%26atitle%3DDynamics%2520and%2520design%2520of%2520enzymes%2520and%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1986%26volume%3D108%26spage%3D3830%26epage%3D3832%26doi%3D10.1021%2Fja00273a048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group">Merz, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Free energy perturbation simulations of the inhibition of thermolysin: prediction of the free energy of binding of a new inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">5649</span><span class="NLM_x">–</span> <span class="NLM_lpage">5658</span><span class="refDoi"> DOI: 10.1021/ja00197a022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00197a022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL1MXkslelsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1989&pages=5649-5658&author=K.+M.+Merzauthor=P.+A.+Kollman&title=Free+energy+perturbation+simulations+of+the+inhibition+of+thermolysin%3A+prediction+of+the+free+energy+of+binding+of+a+new+inhibitor&doi=10.1021%2Fja00197a022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy perturbation simulations of the inhibition of thermolysin:  prediction of the free energy of binding of a new inhibitor</span></div><div class="casAuthors">Merz, Kenneth M., Jr.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5649-58</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Free energy perturbation simulations on a series of thermolysin inhibitors are reported.  The 3 compds. examd. were transition-state analogs and had the general structure, carbobenzoxy-Glyp(X)-L-Leu-L-Leu, where X = NH, O, and CH2.  Whereas the NH and O compds. have had their crystal structure solved as well as having their Ki detd., this is not true for the CH2 compd.  The simulations predicted the CH2 compd. to have a Ki similar to that of the NH compd. (ΔΔGbind = -0.3 kcal/mol favoring the NH compd.), whereas both had Ki greater than the O compd.  The reason why the CH2 compd. was a potent inhibitor could be traced to 2 factors: (1) the solvation free energy for the CH2 compd. was less than that for the NH and O compds. (hydrophobic effect) and (2) the CH2 compd. did not have a repulsive interaction between the main-chain carbonyl group, of alanine-113 and the O atom.  The NH → O simulations reported earlier were also redone using a new set of charges based on a more complete quantum mech. geometry optimization of models of the inhibitors.  These new charges led to a calcd. ΔΔG for NH → O of 5.9 kcal/mol, in contrast to the 4.2 kcal/mol calcd. earlier.  By mutating the new charges into the old, it was possible to more closely reproduce (ΔΔGcalc = 4.05 kcal/mol) previously calcd. results, albeit with different relative contributions of solvation and binding free energies.  Simulations were also carried out in which the force field parameter used for the neutral carboxylic acid X-C-O-H torsion was modified.  This was done because glutamate-143 interacts with the perturbation site, and hence it was expected that the smaller torsion barrier used previously (3.6 kcal/mol) might bias the calcn. considering that the exptl. value for this torsion was ∼10 kcal/mol.  In these simulations, which also used the new charges, the free energy value for the CH2 → NH perturbation changed very slightly (-0.3 vs. 0.0 kcal/mol), whereas for the NH → O perturbation there was a much larger change (5.9 vs. 3.3 kcal/mol).  Concomitantly, a significant change in the active site geometry between the 2 parameter sets was obsd., with the newer set matching the exptl. structure more closely.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJa5DNWccvwbVg90H21EOLACvtfcHk0lhLqDp4OINLkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkslelsLg%253D&md5=86d607d69ce4bd2a288edf6f2edd21ed</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1021%2Fja00197a022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00197a022%26sid%3Dliteratum%253Aachs%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DFree%2520energy%2520perturbation%2520simulations%2520of%2520the%2520inhibition%2520of%2520thermolysin%253A%2520prediction%2520of%2520the%2520free%2520energy%2520of%2520binding%2520of%2520a%2520new%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1989%26volume%3D111%26spage%3D5649%26epage%3D5658%26doi%3D10.1021%2Fja00197a022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group">Ferguson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radmer, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Determination of the relative binding free energies of peptide inhibitors to the HIV-1 protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2654</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1021/jm00112a048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00112a048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK3MXkvVCru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=2654-2549&author=D.+M.+Fergusonauthor=R.+J.+Radmerauthor=P.+A.+Kollman&title=Determination+of+the+relative+binding+free+energies+of+peptide+inhibitors+to+the+HIV-1+protease&doi=10.1021%2Fjm00112a048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of the relative binding free energies of peptide inhibitors to the HIV-1 protease</span></div><div class="casAuthors">Ferguson, David M.; Radmer, Randall J.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2654-9</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Mol. dynamics and free energy perturbation methods are used to det. the stereoselective binding of a potent modified peptide inhibitor to the HIV-1 protease.  The inhibitor, reported by (D. H. Rich et al., 1990) contains a key hydroxyl group that interacts with the catalytic aspartyl residues, mimicking the tetrahedral intermediate in hydrolysis.  The binding affinity, however, is dependent on the configuration of the chiral center to which this hydroxyl group is attached.  The relative binding free energy of the 2 diastereomers favor the S-configured hydroxyl inhibitor over the R form by 2.8 kcal/mol.  This is in agreement with the exptl. value of 2.6 kcal/mol that was measured simultaneously and independently of this study.  To further validate this model, the relative binding free energy of a third yet unreported inhibitor lacking the hydroxyl group was also detd.  The results predict a disease in binding affinity for this inhibitor in accord with other peptide inhibitors that do not contain the key hydroxyl group.  The calcns. also suggest that one of the two aspartyl residues (Asp125) is protonated when the inhibitors are bound.  The results indicate that the theor. model developed can be used reliably for the prediction of relative binding affinities and should be a useful tool for the design of viable anti-AIDS therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM53opFhNW5rVg90H21EOLACvtfcHk0lhLqDp4OINLkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXkvVCru78%253D&md5=4e55bb75bf3e6ff2c77ed9e635cd6fcc</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1021%2Fjm00112a048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00112a048%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26aulast%3DRadmer%26aufirst%3DR.%2BJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DDetermination%2520of%2520the%2520relative%2520binding%2520free%2520energies%2520of%2520peptide%2520inhibitors%2520to%2520the%2520HIV-1%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D2654%26epage%3D2549%26doi%3D10.1021%2Fjm00112a048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group">Rao, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilton, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, U. C.</span><span> </span><span class="NLM_article-title">Free energy perturbation studies on inhibitor binding to HIV-1 proteinase</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">4447</span><span class="NLM_x">–</span> <span class="NLM_lpage">4452</span><span class="refDoi"> DOI: 10.1021/ja00038a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00038a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1992&pages=4447-4452&author=B.+G.+Raoauthor=R.+F.+Tiltonauthor=U.+C.+Singh&title=Free+energy+perturbation+studies+on+inhibitor+binding+to+HIV-1+proteinase&doi=10.1021%2Fja00038a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1021%2Fja00038a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00038a001%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DB.%2BG.%26aulast%3DTilton%26aufirst%3DR.%2BF.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26atitle%3DFree%2520energy%2520perturbation%2520studies%2520on%2520inhibitor%2520binding%2520to%2520HIV-1%2520proteinase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1992%26volume%3D114%26spage%3D4447%26epage%3D4452%26doi%3D10.1021%2Fja00038a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group">Reddy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viswanadhan, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J. N.</span><span> </span><span class="NLM_article-title">Relative differences in the binding free energies of human immunodeficiency virus 1 protease inhibitors: a thermodynamic cycle-perturbation approach</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">10287</span><span class="NLM_x">–</span> <span class="NLM_lpage">10291</span><span class="refDoi"> DOI: 10.1073/pnas.88.22.10287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.88.22.10287" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=10287-10291&author=M.+R.+Reddyauthor=V.+N.+Viswanadhanauthor=J.+N.+Weinstein&title=Relative+differences+in+the+binding+free+energies+of+human+immunodeficiency+virus+1+protease+inhibitors%3A+a+thermodynamic+cycle-perturbation+approach&doi=10.1073%2Fpnas.88.22.10287"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1073%2Fpnas.88.22.10287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.88.22.10287%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DViswanadhan%26aufirst%3DV.%2BN.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26atitle%3DRelative%2520differences%2520in%2520the%2520binding%2520free%2520energies%2520of%2520human%2520immunodeficiency%2520virus%25201%2520protease%2520inhibitors%253A%2520a%2520thermodynamic%2520cycle-perturbation%2520approach%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1991%26volume%3D88%26spage%3D10287%26epage%3D10291%26doi%3D10.1073%2Fpnas.88.22.10287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupyan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wester, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">–</span> <span class="NLM_lpage">2703</span><span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+reliable+prediction+of+relative+ligand+binding+potency+in+prospective+drug+discovery+by+way+of+a+modern+free-energy+calculation+protocol+and+force+field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0lhDnUok3VR1nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520relative%2520ligand%2520binding%2520potency%2520in%2520prospective%2520drug%2520discovery%2520by%2520way%2520of%2520a%2520modern%2520free-energy%2520calculation%2520protocol%2520and%2520force%2520field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domaoal, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallardo-Macias, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Computationally-guided optimization of a docking hit to yield catechol diethers as potent anti-HIV agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8582</span><span class="NLM_x">–</span> <span class="NLM_lpage">8591</span><span class="refDoi"> DOI: 10.1021/jm201134m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201134m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8582-8591&author=M.+Bolliniauthor=R.+A.+Domaoalauthor=V.+V.+Thakurauthor=R.+Gallardo-Maciasauthor=K.+A.+Spasovauthor=K.+S.+Andersonauthor=W.+L.+Jorgensen&title=Computationally-guided+optimization+of+a+docking+hit+to+yield+catechol+diethers+as+potent+anti-HIV+agents&doi=10.1021%2Fjm201134m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally-Guided Optimization of a Docking Hit to Yield Catechol Diethers as Potent Anti-HIV Agents</span></div><div class="casAuthors">Bollini, Mariela; Domaoal, Robert A.; Thakur, Vinay V.; Gallardo-Macias, Ricardo; Spasov, Krasimir A.; Anderson, Karen S.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8582-8591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A 5-μM docking hit has been optimized to an extraordinarily potent (55 pM) non-nucleoside inhibitor of HIV reverse transcriptase.  Use of free energy perturbation (FEP) calcns. to predict relative free energies of binding aided the optimizations by identifying optimal substitution patterns for Ph rings and a linker.  The most potent resultant catechol diethers feature terminal uracil and cyanovinylphenyl groups.  A halogen bond with Pro95 likely contributes to the extreme potency of compd. 42 (I).  In addn., several examples are provided illustrating failures of attempted grafting of a substructure from a very active compd. onto a seemingly related scaffold to improve its activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjuw2rS4-u1LVg90H21EOLACvtfcHk0liPXbximU9TOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7rE&md5=f06e165987ffa547f6262b4547bd9af6</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fjm201134m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201134m%26sid%3Dliteratum%253Aachs%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DDomaoal%26aufirst%3DR.%2BA.%26aulast%3DThakur%26aufirst%3DV.%2BV.%26aulast%3DGallardo-Macias%26aufirst%3DR.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DComputationally-guided%2520optimization%2520of%2520a%2520docking%2520hit%2520to%2520yield%2520catechol%2520diethers%2520as%2520potent%2520anti-HIV%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8582%26epage%3D8591%26doi%3D10.1021%2Fjm201134m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group">Lee, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallardo-Macias, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Picomolar inhibitors of HIV reverse transcriptase featuring bicyclic replacement of a cyanovinylphenyl group</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">16705</span><span class="NLM_x">–</span> <span class="NLM_lpage">16713</span><span class="refDoi"> DOI: 10.1021/ja408917n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja408917n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kktb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=16705-16713&author=W.+G.+Leeauthor=R.+Gallardo-Maciasauthor=K.+M.+Freyauthor=K.+A.+Spasovauthor=M.+Bolliniauthor=K.+S.+Andersonauthor=W.+L.+Jorgensen&title=Picomolar+inhibitors+of+HIV+reverse+transcriptase+featuring+bicyclic+replacement+of+a+cyanovinylphenyl+group&doi=10.1021%2Fja408917n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Picomolar Inhibitors of HIV Reverse Transcriptase Featuring Bicyclic Replacement of a Cyanovinylphenyl Group</span></div><div class="casAuthors">Lee, Won-Gil; Gallardo-Macias, Ricardo; Frey, Kathleen M.; Spasov, Krasimir A.; Bollini, Mariela; Anderson, Karen S.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">16705-16713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the catechol diether class are highly potent non-nucleoside inhibitors of HIV-1 reverse transcriptase (NNRTIs).  The most active compds. yield EC50 values below 0.5 nM in assays using human T-cells infected by wild-type HIV-1.  However, these compds. such as rilpivirine, the most recently FDA-approved NNRTI, bear a cyanovinylphenyl (CVP) group.  This is an uncommon substructure in drugs that gives reactivity concerns.  In the present work, computer simulations were used to design bicyclic replacements for the CVP group.  The predicted viability of a 2-cyanoindolizinyl alternative was confirmed exptl. and provided compds. with 0.4 nM activity against the wild-type virus.  The compds. also performed well with EC50 values of 10 nM against the challenging HIV-1 variant that contains the Lys103Asn/Tyr181Cys double mutation in the RT enzyme.  Indolyl and benzofuranyl analogs were also investigated; the most potent compds. in these cases have EC50 values toward wild-type HIV-1 near 10 nM and high-nanomolar activities toward the double-variant.  The structural expectations from the modeling were much enhanced by obtaining an X-ray crystal structure at 2.88 Å resoln. for the complex of the parent 2-cyanoindolizine I and HIV-1 RT.  The aq. solubilities of the most potent indolizine analogs were also measured to be ∼40 μg/mL, which is similar to that for the approved drug efavirenz and ∼1000-fold greater than for rilpivirine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2eJNNTzKRp7Vg90H21EOLACvtfcHk0liPXbximU9TOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kktb7L&md5=2a92d73a9d550ab0338cbd98f81faff0</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fja408917n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja408917n%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%2BG.%26aulast%3DGallardo-Macias%26aufirst%3DR.%26aulast%3DFrey%26aufirst%3DK.%2BM.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DPicomolar%2520inhibitors%2520of%2520HIV%2520reverse%2520transcriptase%2520featuring%2520bicyclic%2520replacement%2520of%2520a%2520cyanovinylphenyl%2520group%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D16705%26epage%3D16713%26doi%3D10.1021%2Fja408917n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group">Frey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puleo, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span> </span><span class="NLM_article-title">Structure-based evaluation of non-nucleoside inhibitors with improved potency and solubility that target HIV reverse transcriptase variants</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2737</span><span class="NLM_x">–</span> <span class="NLM_lpage">2745</span><span class="refDoi"> DOI: 10.1021/jm501908a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501908a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2737-2745&author=K.+M.+Freyauthor=D.+E.+Puleoauthor=K.+A.+Spasovauthor=M.+Bolliniauthor=W.+L.+Jorgensenauthor=K.+S.+Anderson&title=Structure-based+evaluation+of+non-nucleoside+inhibitors+with+improved+potency+and+solubility+that+target+HIV+reverse+transcriptase+variants&doi=10.1021%2Fjm501908a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fjm501908a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501908a%26sid%3Dliteratum%253Aachs%26aulast%3DFrey%26aufirst%3DK.%2BM.%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26atitle%3DStructure-based%2520evaluation%2520of%2520non-nucleoside%2520inhibitors%2520with%2520improved%2520potency%2520and%2520solubility%2520that%2520target%2520HIV%2520reverse%2520transcriptase%2520variants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2737%26epage%3D2745%26doi%3D10.1021%2Fjm501908a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group">Dziedzic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cisneros, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danford, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, R. H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Design, Synthesis, and Protein Crystallography of Biaryltriazoles as Potent Tautomerase Inhibitors of Macrophage Migration Inhibitory Factor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2996</span><span class="NLM_x">–</span> <span class="NLM_lpage">3003</span><span class="refDoi"> DOI: 10.1021/ja512112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2996-3003&author=P.+Dziedzicauthor=J.+A.+Cisnerosauthor=M.+J.+Robertsonauthor=A.+A.+Hareauthor=N.+E.+Danfordauthor=R.+H.+G.+Baxterauthor=W.+L.+Jorgensen&title=Design%2C+Synthesis%2C+and+Protein+Crystallography+of+Biaryltriazoles+as+Potent+Tautomerase+Inhibitors+of+Macrophage+Migration+Inhibitory+Factor&doi=10.1021%2Fja512112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and protein crystallography of biaryltriazoles as potent tautomerase inhibitors of macrophage migration inhibitory factor</span></div><div class="casAuthors">Dziedzic, Pawel; Cisneros, Jose A.; Robertson, Michael J.; Hare, Alissa A.; Danford, Nadia E.; Baxter, Richard H. G.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2996-3003</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Optimization is reported for biaryltriazoles as inhibitors of the tautomerase activity of human macrophage migration inhibitory factor (MIF), a proinflammatory cytokine assocd. with numerous inflammatory diseases and cancer.  A combined approach was taken featuring org. synthesis, enzymic assaying, crystallog., and modeling including free-energy perturbation (FEP) calcns.  X-ray crystal structures for two inhibitors bound to MIF are reported and provided a basis for the modeling efforts.  The accommodation of the inhibitors in the binding site is striking with multiple hydrogen bonds and aryl-aryl interactions.  Addnl. modeling encouraged pursuit of 5-phenoxyquinolinyl analogs, which led to the very potent compd.  Activity was further enhanced by addn. of a fluorine atom adjacent to the phenolic hydroxyl group to strengthen a key hydrogen bond.  It is also shown that phys. properties of the compds. can be modulated by variation of solvent-exposed substituents.  Several of the compds. are likely the most potent known MIF tautomerase inhibitors; the most active ones are more than 1000-fold more active than the well-studied (R)-ISO-1 and more than 200-fold more active than the chromen-4-one Orita-13.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYFcj_0uhyz7Vg90H21EOLACvtfcHk0lj5wYNaNKGoTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D&md5=2b5fea70b6db3cb39f7217c39995bd06</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1021%2Fja512112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512112j%26sid%3Dliteratum%253Aachs%26aulast%3DDziedzic%26aufirst%3DP.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DHare%26aufirst%3DA.%2BA.%26aulast%3DDanford%26aufirst%3DN.%2BE.%26aulast%3DBaxter%26aufirst%3DR.%2BH.%2BG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Protein%2520Crystallography%2520of%2520Biaryltriazoles%2520as%2520Potent%2520Tautomerase%2520Inhibitors%2520of%2520Macrophage%2520Migration%2520Inhibitory%2520Factor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2996%26epage%3D3003%26doi%3D10.1021%2Fja512112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group">Hare, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandavadi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournia, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Optimization of N-benzyl-benzoxazol-2-ones as receptor antagonists of macrophage migration inhibitory factor (MIF)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">5811</span><span class="NLM_x">–</span> <span class="NLM_lpage">5814</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.07.129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.bmcl.2010.07.129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20728358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGnt7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=5811-5814&author=A.+A.+Hareauthor=L.+Lengauthor=S.+Gandavadiauthor=X.+Duauthor=Z.+Courniaauthor=R.+Bucalaauthor=W.+L.+Jorgensen&title=Optimization+of+N-benzyl-benzoxazol-2-ones+as+receptor+antagonists+of+macrophage+migration+inhibitory+factor+%28MIF%29&doi=10.1016%2Fj.bmcl.2010.07.129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of N-benzyl-benzoxazol-2-ones as receptor antagonists of macrophage migration inhibitory factor (MIF)</span></div><div class="casAuthors">Hare, Alissa A.; Leng, Lin; Gandavadi, Sunilkumar; Du, Xin; Cournia, Zoe; Bucala, Richard; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5811-5814</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The cytokine MIF is involved in inflammation and cell proliferation via pathways initiated by its binding to the transmembrane receptor CD74.  MIF also exhibits keto-enol tautomerase activity, believed to be vestigial in mammals.  Starting from a 1 μM hit from virtual screening, substituted benzoxazol-2-ones have been discovered as antagonists with IC50 values as low as 7.5 nM in a tautomerase assay and 80 nM in a MIF-CD74 binding assay.  Addnl. studies for one of the potent inhibitors demonstrated that it is not a covalent inhibitor of MIF and that it attenuates MIF-dependent ERK1/2 phosphorylation in human synovial fibroblasts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_iLpA0sOWJ7Vg90H21EOLACvtfcHk0lj5wYNaNKGoTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGnt7%252FP&md5=44b40f72065a7cd52582196dd640fecb</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.07.129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.07.129%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DA.%2BA.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DGandavadi%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DOptimization%2520of%2520N-benzyl-benzoxazol-2-ones%2520as%2520receptor%2520antagonists%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D5811%26epage%3D5814%26doi%3D10.1016%2Fj.bmcl.2010.07.129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group">Cournia, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandavadi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Discovery of human macrophage migration inhibitory factor (MIF)-CD74 Aantagonists via virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">424</span><span class="refDoi"> DOI: 10.1021/jm801100v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801100v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=416-424&author=Z.+Courniaauthor=L.+Lengauthor=S.+Gandavadiauthor=X.+Duauthor=R.+Bucalaauthor=W.+L.+Jorgensen&title=Discovery+of+human+macrophage+migration+inhibitory+factor+%28MIF%29-CD74+Aantagonists+via+virtual+screening&doi=10.1021%2Fjm801100v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1021%2Fjm801100v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801100v%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DGandavadi%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDiscovery%2520of%2520human%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529-CD74%2520Aantagonists%2520via%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D416%26epage%3D424%26doi%3D10.1021%2Fjm801100v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group">Acevedo, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrose, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aamer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sambasivarao, S. V.</span><span> </span><span class="NLM_article-title">Identification of HIV inhibitors guided by free energy perturbation calculations</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1216</span><span class="refDoi"> DOI: 10.2174/138161212799436421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F138161212799436421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22316150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1199-1216&author=O.+Acevedoauthor=Z.+Ambroseauthor=P.+T.+Flahertyauthor=H.+Aamerauthor=P.+Jainauthor=S.+V.+Sambasivarao&title=Identification+of+HIV+inhibitors+guided+by+free+energy+perturbation+calculations&doi=10.2174%2F138161212799436421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of HIV inhibitors guided by free energy perturbation calculations</span></div><div class="casAuthors">Acevedo, Orlando; Ambrose, Zandrea; Flaherty, Patrick T.; Aamer, Hadega; Jain, Prashi; Sambasivarao, Somisetti V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1199-1216</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Free energy perturbation (FEP) theory coupled to mol. dynamics (MD) or Monte Carlo (MC) statistical mechanics offers a theor. precise method for detg. the free energy differences of related biol. inhibitors.  Traditionally requiring extensive computational resources and expertise, it is only recently that its impact is being felt in drug discovery.  A review of computer-aided anti-HIV efforts employing FEP calcns. is provided here that describes early and recent successes in the design of human immunodeficiency virus type 1 (HIV-1) protease and non-nucleoside reverse transcriptase inhibitors.  In addn., our ongoing work developing and optimizing leads for small mol. inhibitors of cyclophilin A (CypA) is highlighted as an update on the current capabilities of the field.  CypA has been shown to aid HIV-1 replication by catalyzing the cis/trans isomerization of a conserved Gly-Pro motif in the N-terminal domain of HIV-1 capsid (CA) protein.  In the absence of a functional CypA, e.g., by the addn. of an inhibitor such as cyclosporine A (CsA), HIV-1 has reduced infectivity.  Our simulations of acylurea-based and 1-indanylketone-based CypA inhibitors have detd. that their nanomolar and micromolar binding affinities, resp., are tied to their ability to stabilize Arg55 and Asn102.  A structurally novel 1-(2,6-dichlorobenzamido) indole core was proposed to maximize these interactions.  FEP-guided optimization, exptl. synthesis, and biol. testing of lead compds. for toxicity and inhibition of wild-type HIV-1 and CA mutants have demonstrated a dose-dependent inhibition of HIV-1 infection in two cell lines.  While the inhibition is modest compared to CsA, the results are encouraging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuuljqaTqDGrVg90H21EOLACvtfcHk0lgn49UyFP2gXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbw%253D&md5=b992dc0cfe8d4874e1f5fea9cb2faa5b</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.2174%2F138161212799436421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212799436421%26sid%3Dliteratum%253Aachs%26aulast%3DAcevedo%26aufirst%3DO.%26aulast%3DAmbrose%26aufirst%3DZ.%26aulast%3DFlaherty%26aufirst%3DP.%2BT.%26aulast%3DAamer%26aufirst%3DH.%26aulast%3DJain%26aufirst%3DP.%26aulast%3DSambasivarao%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520of%2520HIV%2520inhibitors%2520guided%2520by%2520free%2520energy%2520perturbation%2520calculations%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D1199%26epage%3D1216%26doi%3D10.2174%2F138161212799436421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group">Cole, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">966</span><span class="NLM_x">–</span> <span class="NLM_lpage">971</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2014.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.bbagen.2014.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25196360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=966-971&author=D.+J.+Coleauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Molecular+dynamics+and+Monte+Carlo+simulations+for+protein-ligand+binding+and+inhibitor+design&doi=10.1016%2Fj.bbagen.2014.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span></div><div class="casAuthors">Cole, Daniel J.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">966-971</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Non-nucleoside inhibitors of HIV reverse transcriptase are an important component of treatment against HIV infection.  Novel inhibitors are sought that increase potency against variants that contain the Tyr181Cys mutation.  Mol. dynamics based free energy perturbation simulations have been run to study factors that contribute to protein-ligand binding, and the results are compared with those from previous Monte Carlo based simulations and activity data.  Predictions of protein-ligand binding modes are very consistent for the two simulation methods; the accord is attributed to the use of an enhanced sampling protocol.  The Tyr181Cys binding pocket supports large, hydrophobic substituents, which is in good agreement with expt.  Although some discrepancies exist between the results of the two simulation methods and expt., free energy perturbation simulations can be used to rapidly test small mols. for gains in binding affinity.  Free energy perturbation methods show promise in providing fast, reliable and accurate data that can be used to complement expt. in lead optimization projects.  This article is part of a Special Issue entitled "Recent developments of mol. dynamics".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1gmI-9_FKtrVg90H21EOLACvtfcHk0lhlRELFzfHxdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK&md5=2fd7025d2402b77ba6fb2e091733bc12</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DD.%2BJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DMolecular%2520dynamics%2520and%2520Monte%2520Carlo%2520simulations%2520for%2520protein-ligand%2520binding%2520and%2520inhibitor%2520design%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D966%26epage%3D971%26doi%3D10.1016%2Fj.bbagen.2014.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouzida, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swendsen, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, J. M.</span><span> </span><span class="NLM_article-title">The weighted histogram analysis method for free-energy calculations on biomolecules 0.1. The Method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1021</span><span class="refDoi"> DOI: 10.1002/jcc.540130812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.540130812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=1011-1021&author=S.+Kumarauthor=D.+Bouzidaauthor=R.+H.+Swendsenauthor=P.+A.+Kollmanauthor=J.+M.+Rosenberg&title=The+weighted+histogram+analysis+method+for+free-energy+calculations+on+biomolecules+0.1.+The+Method&doi=10.1002%2Fjcc.540130812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">The weighted histogram analysis method for free-energy calculations on biomolecules.  I.  The method</span></div><div class="casAuthors">Kumar, Shankar; Bouzida, Djamal; Swendsen, Robert H.; Kollman, Peter A.; Rosenberg, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1011-21</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The Weighted Histogram Anal. Method (WHAM), an extension of Ferrenberg and Swendsen's Multiple Histogram Technique, has been applied for the first time on complex biomol. Hamiltonians.  The method is presented here as an extension of the Umbrella Sampling method for free-energy and Potential of Mean Force calcns.  This algorithm possesses the following advantages over methods that are currently employed: (1) it provides a built-in est. of sampling errors thereby yielding objective ests. of the optimal location and length of addnl. simulations needed to achieve a desired level of precision; (2) it yields the "best" value of free energies by taking into account all the simulations so as to minimize the statistical errors; (3) in addn. to optimizing the links between simulations, it also allows multiple overlaps of probability distributions for obtaining better ests. of the free-energy differences.  By recasting the Ferrenberg-Swendsen Multiple Histogram equations in a form suitable for mol. mechanics type Hamiltonians, we have demonstrated the feasibility and robustness of this method by applying it to a test problem of the generation of the Potential of Mean Force profile of the pseudorotation phase angle of the sugar ring in deoxyadenosine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLayYt_QpVO7Vg90H21EOLACvtfcHk0lhlRELFzfHxdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D&md5=5b2ad7410198f03025708a37c0fbe89d</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130812%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBouzida%26aufirst%3DD.%26aulast%3DSwendsen%26aufirst%3DR.%2BH.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520weighted%2520histogram%2520analysis%2520method%2520for%2520free-energy%2520calculations%2520on%2520biomolecules%25200.1.%2520The%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D1011%26epage%3D1021%26doi%3D10.1002%2Fjcc.540130812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group">Kastner, J.</span><span> </span><span class="NLM_article-title">Umbrella sampling</span> <span class="citation_source-journal">WIREs Comput. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">932</span><span class="NLM_x">–</span> <span class="NLM_lpage">942</span><span class="refDoi"> DOI: 10.1002/wcms.66</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fwcms.66" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=932-942&author=J.+Kastner&title=Umbrella+sampling&doi=10.1002%2Fwcms.66"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1002%2Fwcms.66&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.66%26sid%3Dliteratum%253Aachs%26aulast%3DKastner%26aufirst%3DJ.%26atitle%3DUmbrella%2520sampling%26jtitle%3DWIREs%2520Comput.%2520Mol.%2520Sci.%26date%3D2011%26volume%3D1%26spage%3D932%26epage%3D942%26doi%3D10.1002%2Fwcms.66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group">Comer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gumbart, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelievre, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohorille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipot, C.</span><span> </span><span class="NLM_article-title">The adaptive biasing force method: everything you always wanted to know but were afraid to ask</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1151</span><span class="refDoi"> DOI: 10.1021/jp506633n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp506633n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsF2nurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=1129-1151&author=J.+Comerauthor=J.+C.+Gumbartauthor=J.+Heninauthor=T.+Lelievreauthor=A.+Pohorilleauthor=C.+Chipot&title=The+adaptive+biasing+force+method%3A+everything+you+always+wanted+to+know+but+were+afraid+to+ask&doi=10.1021%2Fjp506633n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">The Adaptive Biasing Force Method: Everything You Always Wanted To Know but Were Afraid To Ask</span></div><div class="casAuthors">Comer, Jeffrey; Gumbart, James C.; Henin, Jerome; Lelievre, Tony; Pohorille, Andrew; Chipot, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1129-1151</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the host of numerical schemes devised to calc. free energy differences by way of geometric transformations, the adaptive biasing force algorithm has emerged as a promising route to map complex free-energy landscapes.  It relies upon the simple concept that as a simulation progresses, a continuously updated biasing force is added to the equations of motion, such that in the long-time limit it yields a Hamiltonian devoid of an av. force acting along the transition coordinate of interest.  This means that sampling proceeds uniformly on a flat free-energy surface, thus providing reliable free-energy ests.  Much of the appeal of the algorithm to the practitioner is in its phys. intuitive underlying ideas and the absence of any requirements for prior knowledge about free-energy landscapes.  Since its inception in 2001, the adaptive biasing force scheme has been the subject of considerable attention, from in-depth math. anal. of convergence properties to novel developments and extensions.  The method has also been successfully applied to many challenging problems in chem. and biol.  In this contribution, the method is presented in a comprehensive, self-contained fashion, discussing with a crit. eye its properties, applicability, and inherent limitations, as well as introducing novel extensions.  Through free-energy calcns. of prototypical mol. systems, many methodol. aspects are examd., from stratification strategies to overcoming the so-called hidden barriers in orthogonal space, relevant not only to the adaptive biasing force algorithm but also to other importance-sampling schemes.  On the basis of the discussions in this paper, a no. of good practices for improving the efficiency and reliability of the computed free-energy differences are proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUPvI90eVb-LVg90H21EOLACvtfcHk0ljca_y7fKmGAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsF2nurvP&md5=2ea7fa53559bf7355d24630fc126fcc9</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1021%2Fjp506633n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp506633n%26sid%3Dliteratum%253Aachs%26aulast%3DComer%26aufirst%3DJ.%26aulast%3DGumbart%26aufirst%3DJ.%2BC.%26aulast%3DHenin%26aufirst%3DJ.%26aulast%3DLelievre%26aufirst%3DT.%26aulast%3DPohorille%26aufirst%3DA.%26aulast%3DChipot%26aufirst%3DC.%26atitle%3DThe%2520adaptive%2520biasing%2520force%2520method%253A%2520everything%2520you%2520always%2520wanted%2520to%2520know%2520but%2520were%2520afraid%2520to%2520ask%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2015%26volume%3D119%26spage%3D1129%26epage%3D1151%26doi%3D10.1021%2Fjp506633n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group">Woo, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding free energy from computer simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">6825</span><span class="NLM_x">–</span> <span class="NLM_lpage">6830</span><span class="refDoi"> DOI: 10.1073/pnas.0409005102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0409005102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=15867154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6825-6830&author=H.+J.+Wooauthor=B.+Roux&title=Calculation+of+absolute+protein-ligand+binding+free+energy+from+computer+simulations&doi=10.1073%2Fpnas.0409005102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding free energy from computer simulations</span></div><div class="casAuthors">Woo, Hyung-June; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6825-6830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A general methodol. for calcg. the equil. binding const. of a flexible ligand to a protein receptor is formulated on the basis of potentials of mean force.  The overall process is decompd. into several stages that can be computed sep.: the free ligand in the bulk is first restrained into the conformation it adopts in the bound state, position, and orientation by applying biasing potentials, then it is translated into the binding site, where it is released completely.  The conformational restraining potential is based on the root-mean-square deviation of the peptide coordinates relative to its av. conformation in the bound complex.  Free energy contributions from each stage are calcd. by free energy perturbation potential of mean force techniques by using appropriate order parameters.  The present approach avoids the need to decouple the ligand from its surrounding (bulk solvent and receptor protein) as is traditionally performed in the double-decoupling scheme.  It is believed that the present formulation will be particularly useful when the solvation free energy of the ligand is very large.  As an application, the equil. binding const. of the phosphotyrosine peptide pYEEI to the Src homol. 2 domain of human Lck has been calcd.  The results are in good agreement with exptl. values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktN0FgqFW_LVg90H21EOLACvtfcHk0ljca_y7fKmGAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D&md5=23be06c80a875efaff0207ea8dc2b08d</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409005102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409005102%26sid%3Dliteratum%253Aachs%26aulast%3DWoo%26aufirst%3DH.%2BJ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520free%2520energy%2520from%2520computer%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D6825%26epage%3D6830%26doi%3D10.1073%2Fpnas.0409005102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group">Ge, X. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute Binding Free Energy Calculations of Sparsomycin Analogs to the Bacterial Ribosome</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">9525</span><span class="NLM_x">–</span> <span class="NLM_lpage">9539</span><span class="refDoi"> DOI: 10.1021/jp100579y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp100579y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2010&pages=9525-9539&author=X.+X.+Geauthor=B.+Roux&title=Absolute+Binding+Free+Energy+Calculations+of+Sparsomycin+Analogs+to+the+Bacterial+Ribosome&doi=10.1021%2Fjp100579y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1021%2Fjp100579y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp100579y%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DX.%2BX.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520Binding%2520Free%2520Energy%2520Calculations%2520of%2520Sparsomycin%2520Analogs%2520to%2520the%2520Bacterial%2520Ribosome%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D9525%26epage%3D9539%26doi%3D10.1021%2Fjp100579y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group">Deng, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Computations of standard binding free energies with molecular dynamics simulations</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">2234</span><span class="NLM_x">–</span> <span class="NLM_lpage">2246</span><span class="refDoi"> DOI: 10.1021/jp807701h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp807701h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=2234-2246&author=Y.+Q.+Dengauthor=B.+Roux&title=Computations+of+standard+binding+free+energies+with+molecular+dynamics+simulations&doi=10.1021%2Fjp807701h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">Computations of Standard Binding Free Energies with Molecular Dynamics Simulations</span></div><div class="casAuthors">Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2234-2246</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An increasing no. of studies have reported computations of the std. (abs.) binding free energy of small ligands to proteins using mol. dynamics (MD) simulations and explicit solvent mols. that are in good agreement with expts.  This encouraging progress suggests that physics-based approaches hold the promise of making important contributions to the process of drug discovery and optimization in the near future.  Two types of approaches are principally used to compute binding free energies with MD simulations.  The most widely known is the alchem. double decoupling method, in which the interaction of the ligand with its surroundings are progressively switched off.  It is also possible to use a potential of mean force (PMF) method, in which the ligand is phys. sepd. from the protein receptor.  For both of these computational approaches, restraining potentials may be activated and released during the simulation for sampling efficiently the changes in translational, rotational, and conformational freedom of the ligand and protein upon binding.  Because such restraining potentials add bias to the simulations, it is important that their effects be rigorously removed to yield a binding free energy that is properly unbiased with respect to the std. state.  A review of recent results is presented, and differences in computational methods are discussed.  Examples of computations with T4-lysozyme mutants, FKBP12, SH2 domain, and cytochrome P 450 are discussed and compared.  Remaining difficulties and challenges are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG7gDgd4q9KrVg90H21EOLACvtfcHk0lhVBGheRGFugg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFOqtA%253D%253D&md5=8a6759d9ce2a909c92d198fb8660f378</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Fjp807701h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp807701h%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DY.%2BQ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DComputations%2520of%2520standard%2520binding%2520free%2520energies%2520with%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D2234%26epage%3D2246%26doi%3D10.1021%2Fjp807701h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, M. A.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">864</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1529/biophysj.105.071589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1529%2Fbiophysj.105.071589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16284269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2006&pages=864-877&author=M.+S.+Leeauthor=M.+A.+Olson&title=Calculation+of+absolute+protein-ligand+binding+affinity+using+path+and+endpoint+approaches&doi=10.1529%2Fbiophysj.105.071589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span></div><div class="casAuthors">Lee, Michael S.; Olson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">864-877</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A comparative anal. is provided of rigorous and approx. methods for calcg. abs. binding affinities of two protein-ligand complexes: the FKBP protein bound with small mols. 4-hydroxy-2-butanone and FK506.  The authors' rigorous approach is an umbrella sampling technique where a potential of mean force is detd. by pulling the ligand out of the protein active site over several simulation windows.  The results of this approach agree well with exptl. obsd. binding affinities.  Also assessed is a commonly used approx. endpoint approach, which sep. ests. enthalpy, solvation free energy, and entropy.  The authors show that this endpoint approach has numerous variations, all of which are prone to crit. shortcomings.  For example, conventional harmonic and quasiharmonic entropy estn. procedures produce disparate results for the relatively simple protein-ligand systems studied in this work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAEk5VeXijpLVg90H21EOLACvtfcHk0lhVBGheRGFugg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D&md5=3d25be5ba65e123249743538bbbf9e01</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.105.071589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.105.071589%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DOlson%26aufirst%3DM.%2BA.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520affinity%2520using%2520path%2520and%2520endpoint%2520approaches%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D90%26spage%3D864%26epage%3D877%26doi%3D10.1529%2Fbiophysj.105.071589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group">Doudou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R. H.</span><span> </span><span class="NLM_article-title">Standard free energy of binding from a one-dimensional potential of mean force</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">909</span><span class="NLM_x">–</span> <span class="NLM_lpage">918</span><span class="refDoi"> DOI: 10.1021/ct8002354</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct8002354" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2qt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=909-918&author=S.+Doudouauthor=N.+A.+Burtonauthor=R.+H.+Henchman&title=Standard+free+energy+of+binding+from+a+one-dimensional+potential+of+mean+force&doi=10.1021%2Fct8002354"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Standard Free Energy of Binding from a One-Dimensional Potential of Mean Force</span></div><div class="casAuthors">Doudou, Slimane; Burton, Neil A.; Henchman, Richard H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">909-918</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A practical approach that enables one to calc. the std. free energy of binding from a one-dimensional potential of mean force (PMF) is proposed.  Umbrella sampling and the weighted histogram anal. method are used to generate a PMF along the reaction coordinate of binding.  At each point, a restraint is applied orthogonal to the reaction coordinate to make possible the detn. of the vol. sampled by the ligand.  The free energy of binding from an arbitrary unbound vol. to the restrained bound form is calcd. from the ratio of the PMF integrated over the bound region to that of the unbound.  Adding the free energy changes from the std.-state vol. to the unbound vol. and from the restrained to the unrestrained bound state gives the std. free energy of binding.  Exploration of the best choice of binding paths is also made.  This approach is first demonstrated on a model binding system and then tested on the benzamidine-trypsin system for which reasonable agreement with expt. is found.  A comparison is made with other methods to obtain the std. free energy of binding from the PMF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-crj3Z8z_FLVg90H21EOLACvtfcHk0lh4weZkjk1uVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2qt70%253D&md5=91aad6d99bd33da268c2b65a90220069</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1021%2Fct8002354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct8002354%26sid%3Dliteratum%253Aachs%26aulast%3DDoudou%26aufirst%3DS.%26aulast%3DBurton%26aufirst%3DN.%2BA.%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26atitle%3DStandard%2520free%2520energy%2520of%2520binding%2520from%2520a%2520one-dimensional%2520potential%2520of%2520mean%2520force%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D909%26epage%3D918%26doi%3D10.1021%2Fct8002354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group">Wojtas-Niziurski, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berneche, S.</span><span> </span><span class="NLM_article-title">Self-learning adaptive umbrella sampling method for the determination of free energy landscapes in multiple dimensions</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1885</span><span class="NLM_x">–</span> <span class="NLM_lpage">1895</span><span class="refDoi"> DOI: 10.1021/ct300978b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300978b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslGgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1885-1895&author=W.+Wojtas-Niziurskiauthor=Y.+L.+Mengauthor=B.+Rouxauthor=S.+Berneche&title=Self-learning+adaptive+umbrella+sampling+method+for+the+determination+of+free+energy+landscapes+in+multiple+dimensions&doi=10.1021%2Fct300978b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Learning Adaptive Umbrella Sampling Method for the Determination of Free Energy Landscapes in Multiple Dimensions</span></div><div class="casAuthors">Wojtas-Niziurski, Wojciech; Meng, Yilin; Roux, Benoit; Berneche, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1885-1895</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The potential of mean force describing conformational changes of biomols. is a central quantity for understanding the function of biomol. systems.  Calcg. an energy landscape of a process that depends on three or more reaction coordinates requires extensive computational power, making some multidimensional calcns. practically impossible.  Here, we present an efficient automatized umbrella sampling strategy for calcg. a multidimensional potential of mean force.  The method progressively learns by itself, through a feedback mechanism, which regions of a multidimensional space are worth exploring and automatically generates a set of umbrella sampling windows that is adapted to the system.  The self-learning adaptive umbrella sampling method is first explained with illustrative examples based on simplified reduced model systems and then applied to two nontrivial situations: the conformational equil. of the pentapeptide Met-enkephalin in soln. and ion permeation in the KcsA potassium channel.  With this method, it is demonstrated that a significant smaller no. of umbrella windows needs to be employed to characterize the free energy landscape over the most relevant regions without any loss in accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhBNxaUDWvq7Vg90H21EOLACvtfcHk0lh4weZkjk1uVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslGgurw%253D&md5=cfccc621d18674ed5c3075a2a8ef633e</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1021%2Fct300978b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300978b%26sid%3Dliteratum%253Aachs%26aulast%3DWojtas-Niziurski%26aufirst%3DW.%26aulast%3DMeng%26aufirst%3DY.%2BL.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DBerneche%26aufirst%3DS.%26atitle%3DSelf-learning%2520adaptive%2520umbrella%2520sampling%2520method%2520for%2520the%2520determination%2520of%2520free%2520energy%2520landscapes%2520in%2520multiple%2520dimensions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1885%26epage%3D1895%26doi%3D10.1021%2Fct300978b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group">Jarzynski, C.</span><span> </span><span class="NLM_article-title">Nonequilibrium equality for free energy differences</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">2690</span><span class="NLM_x">–</span> <span class="NLM_lpage">2693</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.78.2690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.78.2690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1997&pages=2690-2693&author=C.+Jarzynski&title=Nonequilibrium+equality+for+free+energy+differences&doi=10.1103%2FPhysRevLett.78.2690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Nonequilibrium equality for free energy differences</span></div><div class="casAuthors">Jarzynski, C.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2690-2693</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">An expression is derived for the equil. free energy difference between two configurations of a system, in terms of an ensemble of finite-time measurements of the work performed in parametrically switching from one configuration to the other.  Two well-known identities emerge as limiting cases of this result.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl50isjLbR7Vg90H21EOLACvtfcHk0lh4weZkjk1uVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D&md5=5e572f64b7e9044fff5af96867602877</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.78.2690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.78.2690%26sid%3Dliteratum%253Aachs%26aulast%3DJarzynski%26aufirst%3DC.%26atitle%3DNonequilibrium%2520equality%2520for%2520free%2520energy%2520differences%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1997%26volume%3D78%26spage%3D2690%26epage%3D2693%26doi%3D10.1103%2FPhysRevLett.78.2690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khalili-Araghi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tajkhorshid, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">Free energy calculation from steered molecular dynamics simulations using Jarzynski’s equality</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">3559</span><span class="NLM_x">–</span> <span class="NLM_lpage">3566</span><span class="refDoi"> DOI: 10.1063/1.1590311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.1590311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvVWktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2003&pages=3559-3566&author=S.+Parkauthor=F.+Khalili-Araghiauthor=E.+Tajkhorshidauthor=K.+Schulten&title=Free+energy+calculation+from+steered+molecular+dynamics+simulations+using+Jarzynski%E2%80%99s+equality&doi=10.1063%2F1.1590311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculation from steered molecular dynamics simulations using Jarzynski's equality</span></div><div class="casAuthors">Park, Sanghyun; Khalili-Araghi, Fatemeh; Tajkhorshid, Emad; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3559-3566</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Jarzynski's equality is applied to free energy calcns. from steered mol. dynamics simulations of biomols.  The helix-coil transition of deca-alanine in vacuum is used as an example.  With about ten trajectories sampled, the second order cumulant expansion, among the various averaging schemes examd., yields the most accurate ests.  We compare umbrella sampling and the present method, and find that their efficiencies are comparable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8HS8jTFZiF7Vg90H21EOLACvtfcHk0lj-F8D-rkUhDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvVWktr0%253D&md5=936e78b8aef1173f864b055ee4030d9c</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1063%2F1.1590311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1590311%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.%26aulast%3DKhalili-Araghi%26aufirst%3DF.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DFree%2520energy%2520calculation%2520from%2520steered%2520molecular%2520dynamics%2520simulations%2520using%2520Jarzynski%25E2%2580%2599s%2520equality%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2003%26volume%3D119%26spage%3D3559%26epage%3D3566%26doi%3D10.1063%2F1.1590311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group">Xiong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crespo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estrin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">Free energy calculations with non-equilibrium methods: applications of the Jarzynski relationship</span> <span class="citation_source-journal">Theor. Chem. Acc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1007/s00214-005-0072-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs00214-005-0072-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28Xns1Sqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2006&pages=338-346&author=H.+Xiongauthor=A.+Crespoauthor=M.+Martiauthor=D.+Estrinauthor=A.+E.+Roitberg&title=Free+energy+calculations+with+non-equilibrium+methods%3A+applications+of+the+Jarzynski+relationship&doi=10.1007%2Fs00214-005-0072-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">Free Energy Calculations with Non-Equilibrium Methods: Applications of the Jarzynski Relationship</span></div><div class="casAuthors">Xiong, Hui; Crespo, Alejandro; Marti, Marcelo; Estrin, Dario; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Chemistry Accounts</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">338-346</span>CODEN:
                <span class="NLM_cas:coden">TCACFW</span>;
        ISSN:<span class="NLM_cas:issn">1432-881X</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  We present an introduction to the Jarzynski relationship that makes a strong connection, for a thermodn. transformation, between the distribution of non-equil. work values and the corresponding equil. free energy differences.  The relationship is discussed in the context of sampling issues, high level parallel computing, and convergence criteria.  We discuss three different applications by our group: mech. unfolding of peptides, mixed quantum/classical free energy calcns. in enzymes, and ligand escape pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiA4kHcfLKxrVg90H21EOLACvtfcHk0lj-F8D-rkUhDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xns1Sqtbo%253D&md5=737d6e658c90905c93a42784fd861e60</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1007%2Fs00214-005-0072-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00214-005-0072-2%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DCrespo%26aufirst%3DA.%26aulast%3DMarti%26aufirst%3DM.%26aulast%3DEstrin%26aufirst%3DD.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DFree%2520energy%2520calculations%2520with%2520non-equilibrium%2520methods%253A%2520applications%2520of%2520the%2520Jarzynski%2520relationship%26jtitle%3DTheor.%2520Chem.%2520Acc.%26date%3D2006%26volume%3D116%26spage%3D338%26epage%3D346%26doi%3D10.1007%2Fs00214-005-0072-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group">Patel, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berteotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ronsisvalle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Steered molecular dynamics simulations for studying protein-ligand interaction in cyclin-dependent Kinase 5</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span><span class="refDoi"> DOI: 10.1021/ci4003574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4003574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt1yrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=470-480&author=J.+S.+Patelauthor=A.+Berteottiauthor=S.+Ronsisvalleauthor=W.+Rocchiaauthor=A.+Cavalli&title=Steered+molecular+dynamics+simulations+for+studying+protein-ligand+interaction+in+cyclin-dependent+Kinase+5&doi=10.1021%2Fci4003574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">Steered molecular dynamics simulations for studying protein-ligand interaction in cyclin-dependent kinase 5</span></div><div class="casAuthors">Patel, Jagdish Suresh; Berteotti, Anna; Ronsisvalle, Simone; Rocchia, Walter; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-480</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, the authors applied steered mol. dynamics (SMD) simulations to investigate the unbinding mechanism of 9 inhibitors of cyclin-dependent kinase 5 (CDK5).  The study had 2 major objectives: (1) to create a correlation between the unbinding force profiles and the inhibition activities of these compds. expressed as IC50 values; and (2) to investigate the unbinding mechanism and to reveal atomistic insights, which could help identify accessory binding sites and transient interactions.  Overall, the authors carried out 1.35 μs of cumulative SMD simulations.  The authors showed that SMD could qual. discriminate binders from nonbinders, while it failed to properly rank series of inhibitors, particularly when the IC50 values were too similar.  From a mechanistic standpoint, SMD provided useful insights related to transient and dynamical interactions, which could complement static description obtained by X-ray crystallog. expts.  Thus, the present study represents a further step toward a systematic exploitation of SMD and other dynamical approaches in structure-based drug design and computational medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdkn6WGwb0P7Vg90H21EOLACvtfcHk0lijRLffimUrqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt1yrug%253D%253D&md5=0977c6b933d23f5a33abb7a5c60f6b31</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fci4003574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4003574%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DJ.%2BS.%26aulast%3DBerteotti%26aufirst%3DA.%26aulast%3DRonsisvalle%26aufirst%3DS.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSteered%2520molecular%2520dynamics%2520simulations%2520for%2520studying%2520protein-ligand%2520interaction%2520in%2520cyclin-dependent%2520Kinase%25205%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D470%26epage%3D480%26doi%3D10.1021%2Fci4003574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group">Palermo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minniti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greco, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riccardi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simoni, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Convertino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchetti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minarini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span> </span><span class="NLM_article-title">An optimized polyamine moiety boosts the potency of human type II topoisomerase poisons as quantified by comparative analysis centered on the clinical candidate F14512</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">14310</span><span class="NLM_x">–</span> <span class="NLM_lpage">14313</span><span class="refDoi"> DOI: 10.1039/C5CC05065K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1039%2FC5CC05065K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26234198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=14310-14313&author=G.+Palermoauthor=E.+Minnitiauthor=M.+L.+Grecoauthor=L.+Riccardiauthor=E.+Simoniauthor=M.+Convertinoauthor=C.+Marchettiauthor=M.+Rosiniauthor=C.+Sissiauthor=A.+Minariniauthor=M.+De+Vivo&title=An+optimized+polyamine+moiety+boosts+the+potency+of+human+type+II+topoisomerase+poisons+as+quantified+by+comparative+analysis+centered+on+the+clinical+candidate+F14512&doi=10.1039%2FC5CC05065K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">An optimized polyamine moiety boosts the potency of human type II topoisomerase poisons as quantified by comparative analysis centered on the clinical candidate F14512</span></div><div class="casAuthors">Palermo, Giulia; Minniti, Elirosa; Greco, Maria Laura; Riccardi, Laura; Simoni, Elena; Convertino, Marino; Marchetti, Chiara; Rosini, Michela; Sissi, Claudia; Minarini, Anna; De Vivo, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">76</span>),
    <span class="NLM_cas:pages">14310-14313</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Combined computational-exptl. analyses explain and quantify the spermine-vectorized F14512's boosted potency as a topoII poison.  We found that an optimized polyamine moiety boosts drug binding to the topoII/DNA cleavage complex, rather than to the DNA alone.  These results provide new structural bases and key ref. data for designing new human topoII poisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBiw33UPksALVg90H21EOLACvtfcHk0lijRLffimUrqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3K&md5=7a45fe86d3796d99a63e996206e509ca</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1039%2FC5CC05065K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC05065K%26sid%3Dliteratum%253Aachs%26aulast%3DPalermo%26aufirst%3DG.%26aulast%3DMinniti%26aufirst%3DE.%26aulast%3DGreco%26aufirst%3DM.%2BL.%26aulast%3DRiccardi%26aufirst%3DL.%26aulast%3DSimoni%26aufirst%3DE.%26aulast%3DConvertino%26aufirst%3DM.%26aulast%3DMarchetti%26aufirst%3DC.%26aulast%3DRosini%26aufirst%3DM.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DMinarini%26aufirst%3DA.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26atitle%3DAn%2520optimized%2520polyamine%2520moiety%2520boosts%2520the%2520potency%2520of%2520human%2520type%2520II%2520topoisomerase%2520poisons%2520as%2520quantified%2520by%2520comparative%2520analysis%2520centered%2520on%2520the%2520clinical%2520candidate%2520F14512%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D14310%26epage%3D14313%26doi%3D10.1039%2FC5CC05065K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group">Patel, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Insights into ligand-protein binding from local mechanical response</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3368</span><span class="NLM_x">–</span> <span class="NLM_lpage">3378</span><span class="refDoi"> DOI: 10.1021/ct200324j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200324j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOit7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=3368-3378&author=J.+S.+Patelauthor=D.+Branduardiauthor=M.+Masettiauthor=W.+Rocchiaauthor=A.+Cavalli&title=Insights+into+ligand-protein+binding+from+local+mechanical+response&doi=10.1021%2Fct200324j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into Ligand-Protein Binding from Local Mechanical Response</span></div><div class="casAuthors">Patel, Jagdish Suresh; Branduardi, Davide; Masetti, Matteo; Rocchia, Walter; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3368-3378</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational studies of ligand-protein binding are crucial for properly designing novel compds. of potential pharmacol. interest.  In this respect, researchers are increasingly interested in steered mol. dynamics for ligand-protein binding and unbinding studies.  In particular, it has been suggested that analyzing the work profiles along the ligand-protein undocking paths could be fruitful.  Here, the authors propose that small portions of work profiles, termed "local mech. responses" of the system to a steering force, could serve as a universal measure for capturing relevant information about the system under investigation.  Specifically, the authors first collected a high no. of steering trajectories using two biol. systems of increasing complexity (i.e., alanine dipeptide and (R)-roscovitine/CDK5 complex).  Then, the authors devised a novel postprocessing tool to be applied to the local mech. responses, to ext. structural information related to the biol. processes under investigation.  Despite the out-of-equil. character of the trajectories, the anal. carried out on the work profiles provided pivotal information about the investigated biol. processes.  This could eventually be applied to drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtLwG--YFzZLVg90H21EOLACvtfcHk0lhf9p9-TlOTow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOit7vO&md5=298c6a91bdeb995a8259a8e037fea02c</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1021%2Fct200324j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200324j%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DJ.%2BS.%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DInsights%2520into%2520ligand-protein%2520binding%2520from%2520local%2520mechanical%2520response%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D3368%26epage%3D3378%26doi%3D10.1021%2Fct200324j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group">Ensing, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vivo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Metadynamics as a tool for exploring free energy landscapes of chemical reactions</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1021/ar040198i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar040198i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1emsLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=73-81&author=B.+Ensingauthor=M.+De+Vivoauthor=Z.+W.+Liuauthor=P.+Mooreauthor=M.+L.+Klein&title=Metadynamics+as+a+tool+for+exploring+free+energy+landscapes+of+chemical+reactions&doi=10.1021%2Far040198i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">Metadynamics as a Tool for Exploring Free Energy Landscapes of Chemical Reactions</span></div><div class="casAuthors">Ensing, Bernd; De Vivo, Marco; Liu, Zhiwei; Moore, Preston; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">73-81</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The metadynamics or hills method is a relatively new mol. dynamics technique aimed to enhance the sampling of sepd. regions in phase space and map out the underlying free energy landscape as a function of a small no. of order parameters or collective variables.  The high efficiency allows for the application of metadynamics in combination with first principles dynamics methods, in particular with Car-Parrinello mol. dynamics, to study processes in which changes in the electronic structure play a dominant role, such as chem. reactions.  The option to choose several independent collective variables is important to tackle complex and concerted transformations that lack an obvious a priori choice for a single reaction coordinate.  In this Account, we discuss the role of metadynamics in the search of transition states, local min., reaction paths, free energy profiles, and reaction coordinates among a growing list of alternative methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvHygCAl8Vm7Vg90H21EOLACvtfcHk0lhf9p9-TlOTow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1emsLbI&md5=74760985582071d13d2cfeef9c25b777</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1021%2Far040198i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far040198i%26sid%3Dliteratum%253Aachs%26aulast%3DEnsing%26aufirst%3DB.%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%2BW.%26aulast%3DMoore%26aufirst%3DP.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DMetadynamics%2520as%2520a%2520tool%2520for%2520exploring%2520free%2520energy%2520landscapes%2520of%2520chemical%2520reactions%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2006%26volume%3D39%26spage%3D73%26epage%3D81%26doi%3D10.1021%2Far040198i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Equilibrium free energies from nonequilibrium metadynamics</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">090601</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.96.090601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.96.090601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16606249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlSis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2006&pages=090601&author=G.+Bussiauthor=A.+Laioauthor=M.+Parrinello&title=Equilibrium+free+energies+from+nonequilibrium+metadynamics&doi=10.1103%2FPhysRevLett.96.090601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrium Free Energies from Nonequilibrium Metadynamics</span></div><div class="casAuthors">Bussi, Giovanni; Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">090601/1-090601/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We propose a new formalism to map history-dependent metadynamics in a Markovian process.  We apply this formalism to model Langevin dynamics and det. the equil. distribution of a collection of simulations.  We demonstrate that the reconstructed free energy is an unbiased est. of the underlying free energy and anal. derive an expression for the error.  The present results can be applied to other history-dependent stochastic processes, such as Wang-Landau sampling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe2rBjNl6IJrVg90H21EOLACvtfcHk0ljxeGYIbS9VFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlSis7o%253D&md5=b4cf2dd6399f8cd58be36692776c38e5</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.96.090601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.96.090601%26sid%3Dliteratum%253Aachs%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DEquilibrium%2520free%2520energies%2520from%2520nonequilibrium%2520metadynamics%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2006%26volume%3D96%26spage%3D090601%26doi%3D10.1103%2FPhysRevLett.96.090601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez-Fortea, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccarelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Assessing the accuracy of metadynamics</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">6714</span><span class="NLM_x">–</span> <span class="NLM_lpage">6721</span><span class="refDoi"> DOI: 10.1021/jp045424k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp045424k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ygurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2005&pages=6714-6721&author=A.+Laioauthor=A.+Rodriguez-Forteaauthor=F.+L.+Gervasioauthor=M.+Ceccarelliauthor=M.+Parrinello&title=Assessing+the+accuracy+of+metadynamics&doi=10.1021%2Fjp045424k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Accuracy of Metadynamics</span></div><div class="casAuthors">Laio, Alessandro; Rodriguez-Fortea, Antonio; Gervasio, Francesco Luigi; Ceccarelli, Matteo; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6714-6721</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metadynamics is a powerful technique that has been successfully exploited to explore the multidimensional free energy surface of complex polyat. systems and predict transition mechanisms in very different fields, ranging from chem. and solid-state physics to biophysics.  We here derive an explicit expression for the accuracy of the methodol. and provide a way to choose the parameters of the method in order to optimize its performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp1Nu73t30sbVg90H21EOLACvtfcHk0ljxeGYIbS9VFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ygurs%253D&md5=f0946da733889a7e05d4f97f6608468b</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1021%2Fjp045424k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp045424k%26sid%3Dliteratum%253Aachs%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DRodriguez-Fortea%26aufirst%3DA.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DCeccarelli%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DAssessing%2520the%2520accuracy%2520of%2520metadynamics%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2005%26volume%3D109%26spage%3D6714%26epage%3D6721%26doi%3D10.1021%2Fjp045424k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Metadynamics: a method to simulate rare events and reconstruct the free energy in biophysics, chemistry and material science</span> <span class="citation_source-journal">Rep. Prog. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">126601</span><span class="refDoi"> DOI: 10.1088/0034-4885/71/12/126601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1088%2F0034-4885%2F71%2F12%2F126601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=126601&author=A.+Laioauthor=F.+L.+Gervasio&title=Metadynamics%3A+a+method+to+simulate+rare+events+and+reconstruct+the+free+energy+in+biophysics%2C+chemistry+and+material+science&doi=10.1088%2F0034-4885%2F71%2F12%2F126601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Metadynamics: a method to stimulate rare events and reconstruct the free energy in biophysics, chemistry and material science</span></div><div class="casAuthors">Laio, Alessandro; Gervasio, Francesco L.</div><div class="citationInfo"><span class="NLM_cas:title">Reports on Progress in Physics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">126601/1-126601/22</span>CODEN:
                <span class="NLM_cas:coden">RPPHAG</span>;
        ISSN:<span class="NLM_cas:issn">0034-4885</span>.
    
            (<span class="NLM_cas:orgname">Institute of Physics Publishing</span>)
        </div><div class="casAbstract">A review.  Metadynamics is a powerful algorithm that can be used both for reconstructing the free energy and for accelerating rare events in systems described by complex Hamiltonians, at the classical or at the quantum level.  In the algorithm the normal evolution of the system is biased by a history-dependent potential constructed as a sum of Gaussians centered along the trajectory followed by a suitably chosen set of collective variables.  The sum of Gaussians is exploited for reconstructing iteratively an estimator of the free energy and forcing the system to escape from local min.  This review is intended to provide a comprehensive description of the algorithm, with a focus on the practical aspects that need to be addressed when one attempts to apply metadynamics to a new system: (i) the choice of the appropriate set of collective variables; (ii) the optimal choice of the metadynamics parameters and (iii) how to control the error and ensure convergence of the algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTaRxtUYS9_7Vg90H21EOLACvtfcHk0ljxeGYIbS9VFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyntrk%253D&md5=cd84cfc103f97c7d7ccf09fc434e2478</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1088%2F0034-4885%2F71%2F12%2F126601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1088%252F0034-4885%252F71%252F12%252F126601%26sid%3Dliteratum%253Aachs%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DMetadynamics%253A%2520a%2520method%2520to%2520simulate%2520rare%2520events%2520and%2520reconstruct%2520the%2520free%2520energy%2520in%2520biophysics%252C%2520chemistry%2520and%2520material%2520science%26jtitle%3DRep.%2520Prog.%2520Phys.%26date%3D2008%26volume%3D71%26spage%3D126601%26doi%3D10.1088%2F0034-4885%2F71%2F12%2F126601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitaleri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saladino, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Investigating Drug-Target Association and dissociation mechanisms using metadynamics-based algorithms</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span><span class="refDoi"> DOI: 10.1021/ar500356n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500356n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nt7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=277-285&author=A.+Cavalliauthor=A.+Spitaleriauthor=G.+Saladinoauthor=F.+L.+Gervasio&title=Investigating+Drug-Target+Association+and+dissociation+mechanisms+using+metadynamics-based+algorithms&doi=10.1021%2Far500356n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Investigating Drug-Target Association and Dissociation Mechanisms Using Metadynamics-Based Algorithms</span></div><div class="casAuthors">Cavalli, Andrea; Spitaleri, Andrea; Saladino, Giorgio; Gervasio, Francesco L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">277-285</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This Account highlights recent advances and discusses major challenges in the field of drug-target recognition, binding, and unbinding studied using metadynamics-based approaches, with particular emphasis on their role in structure-based design.  Computational chem. has significantly contributed to drug design and optimization in an extremely broad range of areas, including prediction of target druggability and drug likeness, de novo design, fragment screening, ligand docking, estn. of binding affinity, and modulation of ADMET (absorption, distribution, metab., excretion, toxicity) properties.  Computationally driven drug discovery must continuously adapt to keep pace with the evolving knowledge of the factors that modulate the pharmacol. action of drugs.  There is thus an urgent need for novel computational approaches that integrate the vast amt. of complex information currently available for small (bio)org. compds., biol. relevant targets and their complexes, while also accounting accurately for the thermodn. and kinetics of drug-target assocn., the intrinsic dynamical behavior of biomol. systems, and the complexity of protein-protein networks.  Understanding the mechanism of drug binding to and unbinding from biol. targets is fundamental for optimizing lead compds. and designing novel biol. active ones.  One major challenge is the accurate description of the conformational complexity prior to and upon formation of drug-target complexes.  Recently, enhanced sampling methods, including metadynamics and related approaches, have been successfully applied to investigate complex mechanisms of drugs binding to flexible targets.  Metadynamics is a family of enhanced sampling techniques aimed at enhancing the rare events and reconstructing the underlying free energy landscape as a function of a set of order parameters, usually referred to as collective variables.  Studies of drug binding mechanisms have predicted the most probable assocn. and dissocn. pathways and the related binding free energy profile.  In addn., the availability of an efficient open-source implementation, running on cost-effective GPU (i.e., graphical processor unit) architectures, has considerably decreased the learning curve and the computational costs of the methods, and increased their adoption by the community.  Here, we review the recent contributions of metadynamics and other enhanced sampling methods to the field of drug-target recognition and binding.  We discuss how metadynamics has been used to search for transition states, to predict binding and unbinding paths, to treat conformational flexibility, and to compute free energy profiles.  We highlight the importance of such predictions in drug discovery.  Major challenges in the field and possible solns. will finally be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_m41iVArELVg90H21EOLACvtfcHk0ljHaWhE7BmzbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nt7zL&md5=e3b79a23aca3a58914eb0bbf4d73306e</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1021%2Far500356n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500356n%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DSpitaleri%26aufirst%3DA.%26aulast%3DSaladino%26aufirst%3DG.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DInvestigating%2520Drug-Target%2520Association%2520and%2520dissociation%2520mechanisms%2520using%2520metadynamics-based%2520algorithms%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D277%26epage%3D285%26doi%3D10.1021%2Far500356n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group">Barducci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Well-tempered metadynamics: A smoothly converging and tunable free-energy method</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">020603</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.100.020603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.100.020603" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=18232845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2008&pages=020603&author=A.+Barducciauthor=G.+Bussiauthor=M.+Parrinello&title=Well-tempered+metadynamics%3A+A+smoothly+converging+and+tunable+free-energy+method&doi=10.1103%2FPhysRevLett.100.020603"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">Well-Tempered Metadynamics: A Smoothly Converging and Tunable Free-Energy Method</span></div><div class="casAuthors">Barducci, Alessandro; Bussi, Giovanni; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">020603/1-020603/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We present a method for detg. the free-energy dependence on a selected no. of collective variables using an adaptive bias.  The formalism provides a unified description which has metadynamics and canonical sampling as limiting cases.  Convergence and errors can be rigorously and easily controlled.  The parameters of the simulation can be tuned so as to focus the computational effort only on the phys. relevant regions of the order parameter space.  The algorithm is tested on the reconstruction of an alanine dipeptide free-energy landscape.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrul1Eh_nSvy7Vg90H21EOLACvtfcHk0ljHaWhE7BmzbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFensQ%253D%253D&md5=701ccfeee476c2e9a5d1e5a6b0e82197</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.100.020603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.100.020603%26sid%3Dliteratum%253Aachs%26aulast%3DBarducci%26aufirst%3DA.%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DWell-tempered%2520metadynamics%253A%2520A%2520smoothly%2520converging%2520and%2520tunable%2520free-energy%2520method%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2008%26volume%3D100%26spage%3D020603%26doi%3D10.1103%2FPhysRevLett.100.020603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Metadynamics with Adaptive Gaussians</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2247</span><span class="NLM_x">–</span> <span class="NLM_lpage">2254</span><span class="refDoi"> DOI: 10.1021/ct3002464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct3002464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2247-2254&author=D.+Branduardiauthor=G.+Bussiauthor=M.+Parrinello&title=Metadynamics+with+Adaptive+Gaussians&doi=10.1021%2Fct3002464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Metadynamics with Adaptive Gaussians</span></div><div class="casAuthors">Branduardi, Davide; Bussi, Giovanni; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2247-2254</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metadynamics is an established sampling method aimed at reconstructing the free-energy surface relative to a set of appropriately chosen collective variables.  In std. metadynamics, the free-energy surface is filled by the addn. of Gaussian potentials of preassigned and typically diagonal covariance.  Asymptotically the free-energy surface is proportional to the bias deposited.  Here, we consider the possibility of using Gaussians whose variance is adjusted on the fly to the local properties of the free-energy surface.  We suggest two different prescriptions: one is based on the local diffusivity and the other on the local geometrical properties.  We further examine the problem of extg. the free-energy surface when using adaptive Gaussians.  We show that the std. relation between the bias and the free energy does not hold.  In the limit of narrow Gaussians an explicit correction can be evaluated.  In the general case, we propose to use instead a relation between bias and free energy borrowed from umbrella sampling.  This relation holds for all kinds of incrementally deposited bias.  We illustrate on the case of alanine dipeptide the advantage of using adaptive Gaussians in conjunction with the new free-energy estimator both in terms of accuracy and speed of convergence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM59Gkb4Mc8LVg90H21EOLACvtfcHk0ljs_2YltP2m6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrsLc%253D&md5=abed7a6d34ff4797d7cbdc3167ad9060</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1021%2Fct3002464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct3002464%26sid%3Dliteratum%253Aachs%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DMetadynamics%2520with%2520Adaptive%2520Gaussians%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2247%26epage%3D2254%26doi%3D10.1021%2Fct3002464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref173"><div class="reference"><strong class="refLabel"><a href="#ref173" class="rightTabRefNumLink">173</a></strong><div class="NLM_citation" id="rightTab-cit173"><span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">From A to B in free energy space</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">054103</span><span class="refDoi"> DOI: 10.1063/1.2432340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1063%2F1.2432340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=17302470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2007&pages=054103&author=D.+Branduardiauthor=F.+L.+Gervasioauthor=M.+Parrinello&title=From+A+to+B+in+free+energy+space&doi=10.1063%2F1.2432340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">From A to B in free energy space</span></div><div class="casAuthors">Branduardi, Davide; Gervasio, Francesco Luigi; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">054103/1-054103/10</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The authors present a new method for searching low free energy paths in complex mol. systems at finite temp.  They introduce two variables that are able to describe the position of a point in configurational space relative to a preassigned path.  With the help of these two variables the authors combine features of approaches such as metadynamics or umbrella sampling with those of path based methods.  This allows global searches in the space of paths to be performed and a new variational principle for the detn. of low free energy paths to be established.  Contrary to metadynamics or umbrella sampling the path can be described by an arbitrary large no. of variables, still the energy profile along the path can be calcd.  The authors exemplify the method numerically by studying the conformational changes of alanine dipeptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEGhhA71jj2rVg90H21EOLACvtfcHk0ljs_2YltP2m6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVarurk%253D&md5=c77c94b6d208f45a08ea2269894010f1</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1063%2F1.2432340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2432340%26sid%3Dliteratum%253Aachs%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFrom%2520A%2520to%2520B%2520in%2520free%2520energy%2520space%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2007%26volume%3D126%26spage%3D054103%26doi%3D10.1063%2F1.2432340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref174"><div class="reference"><strong class="refLabel"><a href="#ref174" class="rightTabRefNumLink">174</a></strong><div class="NLM_citation" id="rightTab-cit174"><span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiana, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provasi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broglia, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Insight into the folding inhibition of the HIV-1 protease by a small peptide</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">2813</span><span class="NLM_x">–</span> <span class="NLM_lpage">2821</span><span class="refDoi"> DOI: 10.1529/biophysj.107.106369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1529%2Fbiophysj.107.106369" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2007&pages=2813-2821&author=M.+Bonomiauthor=F.+L.+Gervasioauthor=G.+Tianaauthor=D.+Provasiauthor=R.+A.+Brogliaauthor=M.+Parrinello&title=Insight+into+the+folding+inhibition+of+the+HIV-1+protease+by+a+small+peptide&doi=10.1529%2Fbiophysj.107.106369"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.107.106369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.107.106369%26sid%3Dliteratum%253Aachs%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DTiana%26aufirst%3DG.%26aulast%3DProvasi%26aufirst%3DD.%26aulast%3DBroglia%26aufirst%3DR.%2BA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DInsight%2520into%2520the%2520folding%2520inhibition%2520of%2520the%2520HIV-1%2520protease%2520by%2520a%2520small%2520peptide%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D93%26spage%3D2813%26epage%3D2821%26doi%3D10.1529%2Fbiophysj.107.106369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref175"><div class="reference"><strong class="refLabel"><a href="#ref175" class="rightTabRefNumLink">175</a></strong><div class="NLM_citation" id="rightTab-cit175"><span><span class="NLM_contrib-group">Berteotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branduardi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Protein conformational transitions: the closure mechanism of a kinase explored by atomistic simulations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">244</span><span class="NLM_x">–</span> <span class="NLM_lpage">250</span><span class="refDoi"> DOI: 10.1021/ja806846q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja806846q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVOqt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=244-250&author=A.+Berteottiauthor=A.+Cavalliauthor=D.+Branduardiauthor=F.+L.+Gervasioauthor=M.+Recanatiniauthor=M.+Parrinello&title=Protein+conformational+transitions%3A+the+closure+mechanism+of+a+kinase+explored+by+atomistic+simulations&doi=10.1021%2Fja806846q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Conformational Transitions: The Closure Mechanism of a Kinase Explored by Atomistic Simulations</span></div><div class="casAuthors">Berteotti, Anna; Cavalli, Andrea; Branduardi, Davide; Gervasio, Francesco Luigi; Recanatini, Maurizio; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">244-250</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase large-scale conformational rearrangement is an issue of enormous biol. and pharmacol. relevance.  Atomistic simulations able to capture the dynamics and the energetics of kinase large-scale motions are still in their infancy.  Here, we present a computational study in which the atomistic dynamics of the "open-to-closed" movement of the cyclin-dependent kinase 5 (CDK5) have been simulated.  Simulations were carried out using a new sampling method that is able to find the lowest free-energy channel between an initial state and a final state.  This large-scale movement has a two-step mechanism: first, the αC-helix rotates by ∼45°, allowing the interaction between Glu51 and Arg149; then the CDK5 activation loop (T-loop) refolds to assume the closed conformation.  We have also estd. the free-energy profile assocd. with the global motion and identified a CDK5 intermediate, which could be exploited for drug-design purposes.  Our new sampling method turned out to be well-suited for investigating at an atomistic level the energetics and dynamics of kinase large-scale conformational motions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWrIGb0X4d0bVg90H21EOLACvtfcHk0ljtWjadGhZs_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVOqt7zP&md5=64a08f8125ae7db038a6127f6d80612a</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1021%2Fja806846q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja806846q%26sid%3Dliteratum%253Aachs%26aulast%3DBerteotti%26aufirst%3DA.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBranduardi%26aufirst%3DD.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DProtein%2520conformational%2520transitions%253A%2520the%2520closure%2520mechanism%2520of%2520a%2520kinase%2520explored%2520by%2520atomistic%2520simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D244%26epage%3D250%26doi%3D10.1021%2Fja806846q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref176"><div class="reference"><strong class="refLabel"><a href="#ref176" class="rightTabRefNumLink">176</a></strong><div class="NLM_citation" id="rightTab-cit176"><span><span class="NLM_contrib-group">Ceccarini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span> </span><span class="NLM_article-title">Ion Conduction through the hERG potassium channel</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e49017</span><span class="refDoi"> DOI: 10.1371/journal.pone.0049017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pone.0049017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23133669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslajtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e49017&author=L.+Ceccariniauthor=M.+Masettiauthor=A.+Cavalliauthor=M.+Recanatini&title=Ion+Conduction+through+the+hERG+potassium+channel&doi=10.1371%2Fjournal.pone.0049017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Ion conduction through the hERG potassium channel</span></div><div class="casAuthors">Ceccarini, Luisa; Masetti, Matteo; Cavalli, Andrea; Recanatini, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e49017</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The inward rectifier voltage-gated potassium channel hERG is of primary importance for the regulation of the membrane potential of cardiomyocytes.  Unlike most voltage-gated K+-channels, hERG shows a low elementary conductance at physiol. voltage and potassium concn.  To investigate the mol. features underlying this unusual behavior, we simulated the ion conduction through the selectivity filter at a fully atomistic level by means of mol. dynamics-based methods, using a homol.-derived model.  According to our calcns., permeation of potassium ions can occur along two pathways, one involving site vacancies inside the filter (showing an energy barrier of about 6 kcal mol-1) and the other characterized by the presence of a knock-on intermediate (about 8 kcal mol-1).  These barriers are indeed in accordance with a low conductance behavior and can be explained in terms of a series of distinctive structural features displayed by the hERG ion permeation pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqob-gIww-yO7Vg90H21EOLACvtfcHk0ljtWjadGhZs_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslajtLvN&md5=00f2558e937c4d22eb21977b160143db</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0049017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0049017%26sid%3Dliteratum%253Aachs%26aulast%3DCeccarini%26aufirst%3DL.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRecanatini%26aufirst%3DM.%26atitle%3DIon%2520Conduction%2520through%2520the%2520hERG%2520potassium%2520channel%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De49017%26doi%3D10.1371%2Fjournal.pone.0049017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref177"><div class="reference"><strong class="refLabel"><a href="#ref177" class="rightTabRefNumLink">177</a></strong><div class="NLM_citation" id="rightTab-cit177"><span><span class="NLM_contrib-group">Tribello, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceriotti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Using sketch-map coordinates to analyze and bias molecular dynamics simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5196</span><span class="NLM_x">–</span> <span class="NLM_lpage">5201</span><span class="refDoi"> DOI: 10.1073/pnas.1201152109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1201152109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22427357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslKqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5196-5201&author=G.+A.+Tribelloauthor=M.+Ceriottiauthor=M.+Parrinello&title=Using+sketch-map+coordinates+to+analyze+and+bias+molecular+dynamics+simulations&doi=10.1073%2Fpnas.1201152109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Using sketch-map coordinates to analyze and bias molecular dynamics simulations</span></div><div class="casAuthors">Tribello, Gareth A.; Ceriotti, Michele; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5196-5201, S5196/1-S5196/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">When examg. complex problems, such as the folding of proteins, coarse grained descriptions of the system drive our investigation and help us to rationalize the results.  Oftentimes collective variables (CVs), derived through some chem. intuition about the process of interest, serve this purpose.  Because finding these CVs is the most difficult part of any investigation, the authors recently developed a dimensionality redn. algorithm, sketch-map, that can be used to build a low-dimensional map of a phase space of high-dimensionality.  In this paper we discuss how these machine-generated CVs can be used to accelerate the exploration of phase space and to reconstruct free-energy landscapes.  To do so, we develop a formalism in which high-dimensional configurations are no longer represented by low-dimensional position vectors.  Instead, for each configuration we calc. a probability distribution, which has a domain that encompasses the entirety of the low-dimensional space.  To construct a biasing potential, we exploit an analogy with metadynamics and use the trajectory to adaptively construct a repulsive, history-dependent bias from the distributions that correspond to the previously visited configurations.  This potential forces the system to explore more of phase space by making it desirable to adopt configurations whose distributions do not overlap with the bias.  We apply this algorithm to a small model protein and succeed in reproducing the free-energy surface that we obtain from a parallel tempering calcn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxdjYCoMTairVg90H21EOLACvtfcHk0lggeLkPQWipmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslKqu78%253D&md5=67fb6e0b72cbacef5fdbd575df9a9d61</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1201152109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1201152109%26sid%3Dliteratum%253Aachs%26aulast%3DTribello%26aufirst%3DG.%2BA.%26aulast%3DCeriotti%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DUsing%2520sketch-map%2520coordinates%2520to%2520analyze%2520and%2520bias%2520molecular%2520dynamics%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5196%26epage%3D5201%26doi%3D10.1073%2Fpnas.1201152109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref178"><div class="reference"><strong class="refLabel"><a href="#ref178" class="rightTabRefNumLink">178</a></strong><div class="NLM_citation" id="rightTab-cit178"><span><span class="NLM_contrib-group">Bussi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Free-energy landscape for beta hairpin folding from combined parallel tempering and metadynamics</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">13435</span><span class="NLM_x">–</span> <span class="NLM_lpage">13441</span><span class="refDoi"> DOI: 10.1021/ja062463w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja062463w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVCitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=13435-13441&author=G.+Bussiauthor=F.+L.+Gervasioauthor=A.+Laioauthor=M.+Parrinello&title=Free-energy+landscape+for+beta+hairpin+folding+from+combined+parallel+tempering+and+metadynamics&doi=10.1021%2Fja062463w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">Free-Energy Landscape for β Hairpin Folding from Combined Parallel Tempering and Metadynamics</span></div><div class="casAuthors">Bussi, Giovanni; Gervasio, Francesco Luigi; Laio, Alessandro; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13435-13441</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We develop a new free-energy method, based on the combination of parallel tempering and metadynamics, and apply this method to the calcn. of the free-energy landscape of the folding β hairpin in explicit water.  We show that the combined method greatly improves the performance of both parallel tempering and metadynamics.  In particular, we are able to sample the high free-energy regions, which are not accessible with conventional parallel tempering.  We use our results to calc. the difference in entropy and enthalpy between the folded and the unfolded state and to characterize the most populated configurations in the relevant free-energy basins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodH4winUNaNLVg90H21EOLACvtfcHk0lggeLkPQWipmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVCitbY%253D&md5=a48316e8acd40c53e00319a169181f8b</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1021%2Fja062463w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja062463w%26sid%3Dliteratum%253Aachs%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFree-energy%2520landscape%2520for%2520beta%2520hairpin%2520folding%2520from%2520combined%2520parallel%2520tempering%2520and%2520metadynamics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D13435%26epage%3D13441%26doi%3D10.1021%2Fja062463w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref179"><div class="reference"><strong class="refLabel"><a href="#ref179" class="rightTabRefNumLink">179</a></strong><div class="NLM_citation" id="rightTab-cit179"><span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Enhanced sampling in the well-tempered ensemble</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">190601</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.104.190601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.104.190601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20866953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVKlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2010&pages=190601&author=M.+Bonomiauthor=M.+Parrinello&title=Enhanced+sampling+in+the+well-tempered+ensemble&doi=10.1103%2FPhysRevLett.104.190601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced Sampling in the Well-Tempered Ensemble</span></div><div class="casAuthors">Bonomi, M.; Parrinello, M.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">190601/1-190601/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We introduce the well-tempered ensemble (WTE) which is the biased ensemble sampled by well-tempered metadynamics when the energy is used as collective variable.  WTE can be designed so as to have approx. the same av. energy as the canonical ensemble but much larger fluctuations.  These two properties lead to an extremely fast exploration of phase space.  An even greater efficiency is obtained when WTE is combined with parallel tempering.  Unbiased Boltzmann avs. are computed on the fly by a recently developed reweighting method.  We apply WTE and its parallel tempering variant to the 2d Ising model and to a Go model of HIV protease, demonstrating in these two representative cases that convergence is accelerated by orders of magnitude.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6bxid23q8KbVg90H21EOLACvtfcHk0liy5bPM1_idIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVKlt7w%253D&md5=c00b17d1b1be22d497d366789ba450eb</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.104.190601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.104.190601%26sid%3Dliteratum%253Aachs%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DEnhanced%2520sampling%2520in%2520the%2520well-tempered%2520ensemble%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2010%26volume%3D104%26spage%3D190601%26doi%3D10.1103%2FPhysRevLett.104.190601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref180"><div class="reference"><strong class="refLabel"><a href="#ref180" class="rightTabRefNumLink">180</a></strong><div class="NLM_citation" id="rightTab-cit180"><span><span class="NLM_contrib-group">Piana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span> </span><span class="NLM_article-title">A bias-exchange approach to protein folding</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">4553</span><span class="NLM_x">–</span> <span class="NLM_lpage">4559</span><span class="refDoi"> DOI: 10.1021/jp067873l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp067873l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvFCht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=4553-4559&author=S.+Pianaauthor=A.+Laio&title=A+bias-exchange+approach+to+protein+folding&doi=10.1021%2Fjp067873l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">A Bias-Exchange Approach to Protein Folding</span></div><div class="casAuthors">Piana, Stefano; Laio, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4553-4559</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By suitably extending a recent approach [Bussi, G., et al., 2006] the authors introduce a powerful methodol. that allows the parallel reconstruction of the free energy of a system in a virtually unlimited no. of variables.  Multiple metadynamics simulations of the same system at the same temp. are performed, biasing each replica with a time-dependent potential constructed in a different set of collective variables.  Exchanges between the bias potentials in the different variables are periodically allowed according to a replica exchange scheme.  Due to the efficaciously multidimensional nature of the bias the method allows exploring complex free energy landscapes with high efficiency.  The usefulness of the method is demonstrated by performing an atomistic simulation in explicit solvent of the folding of a Triptophane cage miniprotein.  It is shown that the folding free energy landscape can be fully characterized starting from an extended conformation with use of only 40 ns of simulation on 8 replicas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1apQMKRwl57Vg90H21EOLACvtfcHk0liy5bPM1_idIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvFCht7Y%253D&md5=7f64fd37737745fa58c24f00f839207b</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1021%2Fjp067873l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp067873l%26sid%3Dliteratum%253Aachs%26aulast%3DPiana%26aufirst%3DS.%26aulast%3DLaio%26aufirst%3DA.%26atitle%3DA%2520bias-exchange%2520approach%2520to%2520protein%2520folding%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2007%26volume%3D111%26spage%3D4553%26epage%3D4559%26doi%3D10.1021%2Fjp067873l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref181"><div class="reference"><strong class="refLabel"><a href="#ref181" class="rightTabRefNumLink">181</a></strong><div class="NLM_citation" id="rightTab-cit181"><span><span class="NLM_contrib-group">Tribello, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceriotti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">A self-learning algorithm for biased molecular dynamics</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">17509</span><span class="NLM_x">–</span> <span class="NLM_lpage">17514</span><span class="refDoi"> DOI: 10.1073/pnas.1011511107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1011511107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20876135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlartbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=17509-17514&author=G.+A.+Tribelloauthor=M.+Ceriottiauthor=M.+Parrinello&title=A+self-learning+algorithm+for+biased+molecular+dynamics&doi=10.1073%2Fpnas.1011511107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">A self-learning algorithm for biased molecular dynamics</span></div><div class="casAuthors">Tribello, Gareth A.; Ceriotti, Michele; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">17509-17514, S17509/1-S17509/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A new self-learning algorithm for accelerated dynamics, reconnaissance meta-dynamics, is proposed that is able to work with a very large no. of collective coordinates.  Acceleration of the dynamics is achieved by constructing a bias potential in terms of a patchwork of one-dimensional, locally valid collective coordinates.  These collective coordinates are obtained from trajectory analyses so that they adapt to any new features encountered during the simulation.  We show how this methodol. can be used to enhance sampling in real chem. systems citing examples both from the physics of clusters and from the biol. sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVviEhtXC5YbVg90H21EOLACvtfcHk0liy5bPM1_idIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlartbzP&md5=38891e4390eb9041341c4c301625c8be</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1011511107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1011511107%26sid%3Dliteratum%253Aachs%26aulast%3DTribello%26aufirst%3DG.%2BA.%26aulast%3DCeriotti%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DA%2520self-learning%2520algorithm%2520for%2520biased%2520molecular%2520dynamics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D17509%26epage%3D17514%26doi%3D10.1073%2Fpnas.1011511107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref182"><div class="reference"><strong class="refLabel"><a href="#ref182" class="rightTabRefNumLink">182</a></strong><div class="NLM_citation" id="rightTab-cit182"><span><span class="NLM_contrib-group">Limongelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marinelli, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Molecular basis of cyclooxygenase enzymes (COXs) selective inhibition</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">5411</span><span class="NLM_x">–</span> <span class="NLM_lpage">5416</span><span class="refDoi"> DOI: 10.1073/pnas.0913377107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0913377107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20215464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFKhsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=5411-5416&author=V.+Limongelliauthor=M.+Bonomiauthor=L.+Marinelliauthor=F.+L.+Gervasioauthor=A.+Cavalliauthor=E.+Novellinoauthor=M.+Parrinello&title=Molecular+basis+of+cyclooxygenase+enzymes+%28COXs%29+selective+inhibition&doi=10.1073%2Fpnas.0913377107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of cyclooxygenase enzymes (COXs) selective inhibition</span></div><div class="casAuthors">Limongelli, Vittorio; Bonomi, Massimilliano; Marinelli, Luciana; Gervasio, Francesco Luigi; Cavalli, Andrea; Novellino, Ettore; Parinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5411-5416, S5411/1-S5411/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The widely used nonsteroidal anti-inflammatory drugs block the cyclooxygenase enzymes (COXs) and are clin. used for the treatment of inflammation, pain, and cancers.  A selective inhibition of the different isoforms, particularly COX-2, is desirable, and consequently a deeper understanding of the mol. basis of selective inhibition is of great demand.  Using an advanced computational technique we have simulated the full dissocn. process of a highly potent and selective inhibitor, SC-558, in both COX-1 and COX-2.  We have found a previously unreported alternative binding mode in COX-2 explaining the time-dependent inhibition exhibited by this class of inhibitors and consequently their long residence time inside this isoform.  Our metadynamics-based approach allows us to illuminate the highly dynamical character of the ligand/protein recognition process, thus explaining a wealth of exptl. data and paving the way to an innovative strategy for designing new COX inhibitors with tuned selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKbCarszxULVg90H21EOLACvtfcHk0lhE-wWnhHGiwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFKhsrs%253D&md5=ef116a356063dd9068293daded6e5207</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0913377107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0913377107%26sid%3Dliteratum%253Aachs%26aulast%3DLimongelli%26aufirst%3DV.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DMolecular%2520basis%2520of%2520cyclooxygenase%2520enzymes%2520%2528COXs%2529%2520selective%2520inhibition%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D5411%26epage%3D5416%26doi%3D10.1073%2Fpnas.0913377107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref183"><div class="reference"><strong class="refLabel"><a href="#ref183" class="rightTabRefNumLink">183</a></strong><div class="NLM_citation" id="rightTab-cit183"><span><span class="NLM_contrib-group">Favia, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Substrate binding process and mechanistic functioning of type 1 11 beta-hydroxysteroid dehydrogenase from enhanced sampling methods</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e25375</span><span class="refDoi"> DOI: 10.1371/journal.pone.0025375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pone.0025375" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25375&author=A.+D.+Faviaauthor=M.+Masettiauthor=M.+Recanatiniauthor=A.+Cavalli&title=Substrate+binding+process+and+mechanistic+functioning+of+type+1+11+beta-hydroxysteroid+dehydrogenase+from+enhanced+sampling+methods&doi=10.1371%2Fjournal.pone.0025375"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025375%26sid%3Dliteratum%253Aachs%26aulast%3DFavia%26aufirst%3DA.%2BD.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSubstrate%2520binding%2520process%2520and%2520mechanistic%2520functioning%2520of%2520type%25201%252011%2520beta-hydroxysteroid%2520dehydrogenase%2520from%2520enhanced%2520sampling%2520methods%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25375%26doi%3D10.1371%2Fjournal.pone.0025375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref184"><div class="reference"><strong class="refLabel"><a href="#ref184" class="rightTabRefNumLink">184</a></strong><div class="NLM_citation" id="rightTab-cit184"><span><span class="NLM_contrib-group">Provasi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bortolato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filizola, M.</span><span> </span><span class="NLM_article-title">Exploring molecular mechanisms of ligand recognition by opioid receptors with metadynamics</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">10020</span><span class="NLM_x">–</span> <span class="NLM_lpage">10029</span><span class="refDoi"> DOI: 10.1021/bi901494n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901494n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WktbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=10020-10029&author=D.+Provasiauthor=A.+Bortolatoauthor=M.+Filizola&title=Exploring+molecular+mechanisms+of+ligand+recognition+by+opioid+receptors+with+metadynamics&doi=10.1021%2Fbi901494n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Molecular Mechanisms of Ligand Recognition by Opioid Receptors with Metadynamics</span></div><div class="casAuthors">Provasi, Davide; Bortolato, Andrea; Filizola, Marta</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">10020-10029</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Opioid receptors are G protein-coupled receptors (GPCRs) of utmost significance in the development of potent analgesic drugs for the treatment of severe pain.  An accurate evaluation at the mol. level of the ligand binding pathways into these receptors may play a key role in the design of new mols. with more desirable properties and reduced side effects.  The recent characterization of high-resoln. x-ray crystal structures of non-rhodopsin GPCRs for diffusible hormones and neurotransmitters presents an unprecedented opportunity to build improved homol. models of opioid receptors, and to study in more detail their mol. mechanisms of ligand recognition.  In this study, possible pathways for entry of the nonselective antagonist naloxone (NLX) from the water environment into the well-accepted alkaloid binding pocket of a delta opioid receptor (DOR) mol. model based on the β2-adrenergic receptor crystal structure are explored using microsecond-scale well-tempered metadynamics simulations.  Using as collective variables distances that account for the position of NLX and of the receptor extracellular loop 2 in relation to the DOR binding pocket, the authors were able to distinguish between the different states visited by the ligand (i.e., docked, undocked, and metastable bound intermediates) and to predict a free energy of binding close to exptl. values after correcting for possible drawbacks of the sampling approach.  The strategy employed herein holds promise for its application to the docking of diverse ligands to the opioid receptors as well as to other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4pkSAtQ5Mq7Vg90H21EOLACvtfcHk0ljANuX25snmMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WktbbM&md5=eca8e33e8ca8eef73302070b80cb8a6f</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1021%2Fbi901494n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901494n%26sid%3Dliteratum%253Aachs%26aulast%3DProvasi%26aufirst%3DD.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DFilizola%26aufirst%3DM.%26atitle%3DExploring%2520molecular%2520mechanisms%2520of%2520ligand%2520recognition%2520by%2520opioid%2520receptors%2520with%2520metadynamics%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D10020%26epage%3D10029%26doi%3D10.1021%2Fbi901494n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref185"><div class="reference"><strong class="refLabel"><a href="#ref185" class="rightTabRefNumLink">185</a></strong><div class="NLM_citation" id="rightTab-cit185"><span><span class="NLM_contrib-group">Limongelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Funnel metadynamics as accurate binding free-energy method</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">6358</span><span class="NLM_x">–</span> <span class="NLM_lpage">6363</span><span class="refDoi"> DOI: 10.1073/pnas.1303186110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1303186110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23553839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVOlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=6358-6363&author=V.+Limongelliauthor=M.+Bonomiauthor=M.+Parrinello&title=Funnel+metadynamics+as+accurate+binding+free-energy+method&doi=10.1073%2Fpnas.1303186110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Funnel metadynamics as accurate binding free-energy method</span></div><div class="casAuthors">Limongelli, Vittorio; Bonomi, Massimiliano; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6358-6363, S6358/1-S6358/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A detailed description of the events ruling ligand/protein interaction and an accurate estn. of the drug affinity to its target is of great help in speeding drug discovery strategies.  We have developed a metadynamics-based approach, named funnel metadynamics, that allows the ligand to enhance the sampling of the target binding sites and its solvated states.  This method leads to an efficient characterization of the binding free-energy surface and an accurate calcn. of the abs. protein-ligand binding free energy.  We illustrate our protocol in two systems, benzamidine/trypsin and SC-558/cyclooxygenase 2.  In both cases, the x-ray conformation has been found as the lowest free-energy pose, and the computed protein-ligand binding free energy in good agreement with expts.  Furthermore, funnel metadynamics unveils important information about the binding process, such as the presence of alternative binding modes and the role of waters.  The results achieved at an affordable computational cost make funnel metadynamics a valuable method for drug discovery and for dealing with a variety of problems in chem., physics, and material science.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6TRPMM3o257Vg90H21EOLACvtfcHk0ljANuX25snmMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVOlurw%253D&md5=39d87aa2f8e06572ee8044a899ade3fb</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303186110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303186110%26sid%3Dliteratum%253Aachs%26aulast%3DLimongelli%26aufirst%3DV.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFunnel%2520metadynamics%2520as%2520accurate%2520binding%2520free-energy%2520method%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D6358%26epage%3D6363%26doi%3D10.1073%2Fpnas.1303186110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref186"><div class="reference"><strong class="refLabel"><a href="#ref186" class="rightTabRefNumLink">186</a></strong><div class="NLM_citation" id="rightTab-cit186"><span><span class="NLM_contrib-group">Di Leva, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limongelli, V.</span><span> </span><span class="NLM_article-title">Mechanistic insight into ligand binding to G-quadruplex DNA</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">5447</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1093/nar/gku247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1093%2Fnar%2Fgku247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24753420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFGhur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=5447-55&author=F.+S.+Di+Levaauthor=E.+Novellinoauthor=A.+Cavalliauthor=M.+Parrinelloauthor=V.+Limongelli&title=Mechanistic+insight+into+ligand+binding+to+G-quadruplex+DNA&doi=10.1093%2Fnar%2Fgku247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insight into ligand binding to G-quadruplex DNA</span></div><div class="casAuthors">Di Leva, Francesco Saverio; Novellino, Ettore; Cavalli, Andrea; Parrinello, Michele; Limongelli, Vittorio</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5447-5455</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Specific guanine-rich regions in human genome can form higher-order DNA structures called G-quadruplexes, which regulate many relevant biol. processes.  For instance, the formation of G-quadruplex at telomeres can alter cellular functions, inducing apoptosis.  Thus, developing small mols. that are able to bind and stabilize the telomeric G-quadruplexes represents an attractive strategy for antitumor therapy.  An example is 3-(benzo[d]thiazol-2-yl)-7-hydroxy-8-((4-(2-hydroxyethyl)piperazin-1-yl)methyl)-2H-chromen-2-one (I), recently identified as a potent ligand of G-quadruplex [d(TGGGGT)]4 with promising in vitro antitumor activity.  The exptl. observations were suggestive of a complex binding mechanism that, despite efforts, has defied full characterization.  Here, the authors provide through metadynamics simulations a comprehensive understanding of the binding mechanism of I to G-quadruplex [d(TGGGGT)]4.  In the authors' calcns., the ligand explored all of the available binding sites on the DNA structure and the free-energy landscape of the whole binding process was computed.  The authors thus disclosed a peculiar hopping binding mechanism whereas I was able to bind both to the groove and to the 3' end of the G-quadruplex.  These results fully explain the available exptl. data, rendering the authors' approach of great value for further ligand/DNA studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpAZDnr8FxILVg90H21EOLACvtfcHk0ljANuX25snmMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFGhur8%253D&md5=18fb1fcb9543ca972b2553ca994d9271</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku247%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DParrinello%26aufirst%3DM.%26aulast%3DLimongelli%26aufirst%3DV.%26atitle%3DMechanistic%2520insight%2520into%2520ligand%2520binding%2520to%2520G-quadruplex%2520DNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D5447%26epage%3D55%26doi%3D10.1093%2Fnar%2Fgku247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref187"><div class="reference"><strong class="refLabel"><a href="#ref187" class="rightTabRefNumLink">187</a></strong><div class="NLM_citation" id="rightTab-cit187"><span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Exploring complex protein-ligand recognition mechanisms with coarse metadynamics</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">4807</span><span class="NLM_x">–</span> <span class="NLM_lpage">4816</span><span class="refDoi"> DOI: 10.1021/jp803936q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp803936q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtl2qtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=4807-4816&author=M.+Masettiauthor=A.+Cavalliauthor=M.+Recanatiniauthor=F.+L.+Gervasio&title=Exploring+complex+protein-ligand+recognition+mechanisms+with+coarse+metadynamics&doi=10.1021%2Fjp803936q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit187R"><div class="casContent"><span class="casTitleNuber">187</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Complex Protein-Ligand Recognition Mechanisms with Coarse Metadynamics</span></div><div class="casAuthors">Masetti, Matteo; Cavalli, Andrea; Recanatini, Maurizio; Gervasio, Francesco Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4807-4816</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The metadynamics method has been shown to be a valuable tool to study the mechanism of mol. recognition in atomistic detail.  However, it requires an a priori knowledge of all slow degrees of freedom relevant to the docking/undocking mechanism.  Here we investigate a combination of docking/clustering with metadynamics performed with a subset of the necessary degrees of freedom (coarse metadynamics), and show that it provides a full mechanistic insight on the protein-ligand docking mechanism.  Moreover, the proposed protocol is able to clearly distinguish between crystallog. and noncrystallog. poses of protein-ligand complexes, and also to find the transition state of the full undocking mechanism, thus giving an indication on the binding free energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR9b4f41EsHrVg90H21EOLACvtfcHk0lgT71CwtXuKug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtl2qtr4%253D&md5=a4debf330970a1ae2e6122265349ea76</span></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1021%2Fjp803936q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp803936q%26sid%3Dliteratum%253Aachs%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DExploring%2520complex%2520protein-ligand%2520recognition%2520mechanisms%2520with%2520coarse%2520metadynamics%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D4807%26epage%3D4816%26doi%3D10.1021%2Fjp803936q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref188"><div class="reference"><strong class="refLabel"><a href="#ref188" class="rightTabRefNumLink">188</a></strong><div class="NLM_citation" id="rightTab-cit188"><span><span class="NLM_contrib-group">Wang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span> </span><span class="NLM_article-title">Identifying ligand binding sites and poses using GPU-accelerated hamiltonian replica exchange molecular dynamics</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">989</span><span class="NLM_x">–</span> <span class="NLM_lpage">1007</span><span class="refDoi"> DOI: 10.1007/s10822-013-9689-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1007%2Fs10822-013-9689-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24297454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWqurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=989-1007&author=K.+Wangauthor=J.+D.+Choderaauthor=Y.+Z.+Yangauthor=M.+R.+Shirts&title=Identifying+ligand+binding+sites+and+poses+using+GPU-accelerated+hamiltonian+replica+exchange+molecular+dynamics&doi=10.1007%2Fs10822-013-9689-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit188R"><div class="casContent"><span class="casTitleNuber">188</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying ligand binding sites and poses using GPU-accelerated Hamiltonian replica exchange molecular dynamics</span></div><div class="casAuthors">Wang, Kai; Chodera, John D.; Yang, Yanzhi; Shirts, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">989-1007</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">We present a method to identify small mol. ligand binding sites and poses within a given protein crystal structure using GPU-accelerated Hamiltonian replica exchange mol. dynamics simulations.  The Hamiltonians used vary from the phys. end state of protein interacting with the ligand to an unphys. end state where the ligand does not interact with the protein.  As replicas explore the space of Hamiltonians interpolating between these states, the ligand can rapidly escape local min. and explore potential binding sites.  Geometric restraints keep the ligands from leaving the vicinity of the protein and an alchem. pathway designed to increase phase space overlap between intermediates ensures good mixing.  Because of the rigorous statistical mech. nature of the Hamiltonian exchange framework, we can also ext. binding free energy ests. for all putative binding sites.  We present results of this methodol. applied to the T4 lysozyme L99A model system for three known ligands and one non-binder as a control, using an implicit solvent.  We find that our methodol. identifies known crystallog. binding sites consistently and accurately for the small no. of ligands considered here and gives free energies consistent with expt.  We are also able to analyze the contribution of individual binding sites to the overall binding affinity.  Our methodol. points to near term potential applications in early-stage structure-guided drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9FKdHS8YIirVg90H21EOLACvtfcHk0lgT71CwtXuKug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWqurzO&md5=9fd70948a1f8a1feac6cbcf32eb27fb8</span></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9689-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9689-8%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DK.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DIdentifying%2520ligand%2520binding%2520sites%2520and%2520poses%2520using%2520GPU-accelerated%2520hamiltonian%2520replica%2520exchange%2520molecular%2520dynamics%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D989%26epage%3D1007%26doi%3D10.1007%2Fs10822-013-9689-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref189"><div class="reference"><strong class="refLabel"><a href="#ref189" class="rightTabRefNumLink">189</a></strong><div class="NLM_citation" id="rightTab-cit189"><span><span class="NLM_contrib-group">Luitz, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zacharias, M.</span><span> </span><span class="NLM_article-title">Protein-ligand docking using hamiltonian replica exchange simulations with soft core potentials</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1669</span><span class="NLM_x">–</span> <span class="NLM_lpage">1675</span><span class="refDoi"> DOI: 10.1021/ci500296f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500296f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFaitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1669-1675&author=M.+P.+Luitzauthor=M.+Zacharias&title=Protein-ligand+docking+using+hamiltonian+replica+exchange+simulations+with+soft+core+potentials&doi=10.1021%2Fci500296f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Ligand Docking Using Hamiltonian Replica Exchange Simulations with Soft Core Potentials</span></div><div class="casAuthors">Luitz, Manuel P.; Zacharias, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1669-1675</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations in explicit solvent allow studying receptor-ligand binding processes including full flexibility of the binding partners and an explicit inclusion of solvation effects.  However, in MD simulations, the search for an optimal ligand-receptor complex geometry is frequently trapped in locally stable non-native binding geometries.  A Hamiltonian replica-exchange (H-REMD)-based protocol has been designed to enhance the sampling of putative ligand-receptor complexes.  It is based on softening nonbonded ligand-receptor interactions along the replicas and one ref. replica under the control of the original force field.  The efficiency of the method has been evaluated on two receptor-ligand systems and one protein-peptide complex.  Starting from misplaced initial docking geometries, the H-REMD method reached in each case the known binding geometry significantly faster than a std. MD simulation.  The approach could also be useful to identify and evaluate alternative binding geometries in a given binding region with small relative differences in binding free energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6eowStElIMbVg90H21EOLACvtfcHk0lhyp_PM6oMHUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFaitro%253D&md5=1e54a0f00f1a80f878c2e2cbb7e34da0</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1021%2Fci500296f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500296f%26sid%3Dliteratum%253Aachs%26aulast%3DLuitz%26aufirst%3DM.%2BP.%26aulast%3DZacharias%26aufirst%3DM.%26atitle%3DProtein-ligand%2520docking%2520using%2520hamiltonian%2520replica%2520exchange%2520simulations%2520with%2520soft%2520core%2520potentials%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1669%26epage%3D1675%26doi%3D10.1021%2Fci500296f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref190"><div class="reference"><strong class="refLabel"><a href="#ref190" class="rightTabRefNumLink">190</a></strong><div class="NLM_citation" id="rightTab-cit190"><span><span class="NLM_contrib-group">Duane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendleton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roweth, D.</span><span> </span><span class="NLM_article-title">Hybrid monte carlo</span> <span class="citation_source-journal">Phys. Lett. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">195</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1016/0370-2693(87)91197-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2F0370-2693%2887%2991197-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaL2sXls1egsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=1987&pages=216-222&author=S.+Duaneauthor=A.+Kennedyauthor=B.+Pendletonauthor=D.+Roweth&title=Hybrid+monte+carlo&doi=10.1016%2F0370-2693%2887%2991197-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit190R"><div class="casContent"><span class="casTitleNuber">190</span><div class="casTitle"><span class="NLM_cas:atitle">Hybrid Monte Carlo</span></div><div class="casAuthors">Duane, Simon; Kennedy, A. D.; Pendleton, Brian J.; Roweth, Duncan</div><div class="citationInfo"><span class="NLM_cas:title">Physics Letters B</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">216-22</span>CODEN:
                <span class="NLM_cas:coden">PYLBAJ</span>;
        ISSN:<span class="NLM_cas:issn">0370-2693</span>.
    </div><div class="casAbstract">A new method is presented for the numerical simulation of lattice field theory.  A hybrid (mol. dynamics/Langevin) algorithm is used to guide a Monte Carlo simulation.  There are no discretization errors even for large step sizes.  The method is esp. efficient for systems such as quantum chromodynamics which contain fermionic degrees of freedom.  Detailed results are presented for 4-dimensional compact quantum electrodynamics including the dynamical effects of e.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXNgJcTdHflrVg90H21EOLACvtfcHk0lhyp_PM6oMHUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXls1egsbY%253D&md5=3f40fdc8cfc804b67b9c5673f811aba2</span></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1016%2F0370-2693%2887%2991197-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0370-2693%252887%252991197-X%26sid%3Dliteratum%253Aachs%26aulast%3DDuane%26aufirst%3DS.%26aulast%3DKennedy%26aufirst%3DA.%26aulast%3DPendleton%26aufirst%3DB.%26aulast%3DRoweth%26aufirst%3DD.%26atitle%3DHybrid%2520monte%2520carlo%26jtitle%3DPhys.%2520Lett.%2520B%26date%3D1987%26volume%3D195%26spage%3D216%26epage%3D222%26doi%3D10.1016%2F0370-2693%2887%2991197-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref191"><div class="reference"><strong class="refLabel"><a href="#ref191" class="rightTabRefNumLink">191</a></strong><div class="NLM_citation" id="rightTab-cit191"><span><span class="NLM_contrib-group">Fischer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cordes, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schütte, C.</span><span> </span><span class="NLM_article-title">Hybrid monte carlo with adaptive temperature in mixed-canonical ensemble: efficient conformational analysis of RNA</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1697</span><span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(19981130)19:15<1689::AID-JCC2>3.3.CO;2-R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1689-1697&author=A.+Fischerauthor=F.+Cordesauthor=C.+Sch%C3%BCtte&title=Hybrid+monte+carlo+with+adaptive+temperature+in+mixed-canonical+ensemble%3A+efficient+conformational+analysis+of+RNA&doi=10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252819981130%252919%253A15%253C1689%253A%253AAID-JCC2%253E3.3.CO%253B2-R%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DCordes%26aufirst%3DF.%26aulast%3DSch%25C3%25BCtte%26aufirst%3DC.%26atitle%3DHybrid%2520monte%2520carlo%2520with%2520adaptive%2520temperature%2520in%2520mixed-canonical%2520ensemble%253A%2520efficient%2520conformational%2520analysis%2520of%2520RNA%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1998%26volume%3D19%26spage%3D1689%26epage%3D1697%26doi%3D10.1002%2F%28SICI%291096-987X%2819981130%2919%3A15%3C1689%3A%3AAID-JCC2%3E3.3.CO%3B2-R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref192"><div class="reference"><strong class="refLabel"><a href="#ref192" class="rightTabRefNumLink">192</a></strong><div class="NLM_citation" id="rightTab-cit192"><span><span class="NLM_contrib-group">Schütte, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huisinga, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deuflhard, P.</span><span> </span><span class="NLM_article-title">A Direct Approach to conformational dynamics based on hybrid monte carlo</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">151</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span><span class="refDoi"> DOI: 10.1006/jcph.1999.6231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1006%2Fjcph.1999.6231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1MXivFejtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=1999&pages=146-168&author=C.+Sch%C3%BCtteauthor=A.+Fischerauthor=W.+Huisingaauthor=P.+Deuflhard&title=A+Direct+Approach+to+conformational+dynamics+based+on+hybrid+monte+carlo&doi=10.1006%2Fjcph.1999.6231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">A Direct Approach to Conformational Dynamics Based on Hybrid Monte Carlo</span></div><div class="casAuthors">Schutte, Ch.; Fischer, A.; Huisinga, W.; Deuflhard, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">146-168</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Recently, a novel concept for the computation of essential features of the dynamics of Hamiltonian systems (such as mol. dynamics) has been proposed.  The realization of this concept had been based on subdivision techniques applied to the Frobenius-Perron operator for the dynamical system.  The present paper suggests an alternative but related concept that merges the conceptual advantages of the dynamical systems approach with the appropriate statistical physics framework.  This approach allows us to define the phrase "conformation" in terms of the dynamical behavior of the mol. system and to characterize the dynamical stability of conformations.  In a first step, the frequency of conformational changes is characterized in statistical terms leading to the definition of some Markov operator T that describes the corresponding transition probabilities within the canonical ensemble.  In a second step, a discretization of T via specific hybrid Monte Carlo techniques is shown to lead to a stochastic matrix P. With these theor. prepns., an identification algorithm for conformations (to be presented in a later paper) is applicable.  It is demonstrated that the discretization of T can be restricted to few essential degrees of freedom so that the combinatorial explosion of discretization boxes is prevented and biomol. systems can be attacked.  Numerical results for the n-pentane mol. and the triribonucleotide adenylyl(3'-5')cytidylyl(3'-5')cytidin are given and interpreted.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBQXKX7uekjbVg90H21EOLACvtfcHk0lj7yBJY63PXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivFejtL0%253D&md5=5eae420b000b63b4a4f6f66c77b74ec9</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1006%2Fjcph.1999.6231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjcph.1999.6231%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCtte%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26aulast%3DDeuflhard%26aufirst%3DP.%26atitle%3DA%2520Direct%2520Approach%2520to%2520conformational%2520dynamics%2520based%2520on%2520hybrid%2520monte%2520carlo%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1999%26volume%3D151%26spage%3D146%26epage%3D168%26doi%3D10.1006%2Fjcph.1999.6231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref193"><div class="reference"><strong class="refLabel"><a href="#ref193" class="rightTabRefNumLink">193</a></strong><div class="NLM_citation" id="rightTab-cit193"><span><span class="NLM_contrib-group">Guarnieri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Still, W. C.</span><span> </span><span class="NLM_article-title">A rapidly convergent simulation method: mixed monte carlo/stochastic dynamics</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1302</span><span class="NLM_x">–</span> <span class="NLM_lpage">1310</span><span class="refDoi"> DOI: 10.1002/jcc.540151111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.540151111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=1302-1310&author=F.+Guarnieriauthor=W.+C.+Still&title=A+rapidly+convergent+simulation+method%3A+mixed+monte+carlo%2Fstochastic+dynamics&doi=10.1002%2Fjcc.540151111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540151111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540151111%26sid%3Dliteratum%253Aachs%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DStill%26aufirst%3DW.%2BC.%26atitle%3DA%2520rapidly%2520convergent%2520simulation%2520method%253A%2520mixed%2520monte%2520carlo%252Fstochastic%2520dynamics%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D1302%26epage%3D1310%26doi%3D10.1002%2Fjcc.540151111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref194"><div class="reference"><strong class="refLabel"><a href="#ref194" class="rightTabRefNumLink">194</a></strong><div class="NLM_citation" id="rightTab-cit194"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodoscek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Computation of absolute hydration and binding free energy with free energy perturbation distributed replica-exchange molecular dynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2583</span><span class="NLM_x">–</span> <span class="NLM_lpage">2588</span><span class="refDoi"> DOI: 10.1021/ct900223z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900223z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=2583-2588&author=W.+Jiangauthor=M.+Hodoscekauthor=B.+Roux&title=Computation+of+absolute+hydration+and+binding+free+energy+with+free+energy+perturbation+distributed+replica-exchange+molecular+dynamics&doi=10.1021%2Fct900223z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">Computation of Absolute Hydration and Binding Free Energy with Free Energy Perturbation Distributed Replica-Exchange Molecular Dynamics</span></div><div class="casAuthors">Jiang, Wei; Hodoscek, Milan; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2583-2588</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Distributed Replica (REPDSTR) is a powerful parallelization technique enabling simulations of a group of replicas in a parallel/parallel fashion, where each replica is distributed to different nodes of a large cluster [Theor. Chem. Acc. 2003, 109, 140].  Here, we use the framework provided by REPDSTR to combine a staged free energy perturbation protocol with replica-exchange mol. dynamics (FEP/REMD).  The structure of REPDSTR, which allows multiple parallel input/output (I/O), facilitates the treatment of replica-exchange to couple the N window simulations corresponding to different values of the thermodn. coupling parameters.  As a result, each of the N synchronous window simulations benefit from the sampling carried out by the N-1 others.  As illustrative examples of the FEP/REMD strategy, calcns. of the abs. hydration and binding free energy of small mols. were performed using the biomol. simulation program CHARMM adapted for the IBM Blue Gene/P platform.  The computations show that a FEP/REMD strategy significantly improves the sampling and accelerates the convergence of abs. free energy computations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQnhjw95RCz7Vg90H21EOLACvtfcHk0lj7yBJY63PXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgu7rE&md5=ca830c52bfe67e33eef10281cd891e8e</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1021%2Fct900223z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900223z%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DHodoscek%26aufirst%3DM.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DComputation%2520of%2520absolute%2520hydration%2520and%2520binding%2520free%2520energy%2520with%2520free%2520energy%2520perturbation%2520distributed%2520replica-exchange%2520molecular%2520dynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D2583%26epage%3D2588%26doi%3D10.1021%2Fct900223z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref195"><div class="reference"><strong class="refLabel"><a href="#ref195" class="rightTabRefNumLink">195</a></strong><div class="NLM_citation" id="rightTab-cit195"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Free energy perturbation hamiltonian replica-exchange molecular dynamics (FEP/H-REMD) for absolute ligand binding free energy calculations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2559</span><span class="NLM_x">–</span> <span class="NLM_lpage">2565</span><span class="refDoi"> DOI: 10.1021/ct1001768</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct1001768" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsV2lsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2559-2565&author=W.+Jiangauthor=B.+Roux&title=Free+energy+perturbation+hamiltonian+replica-exchange+molecular+dynamics+%28FEP%2FH-REMD%29+for+absolute+ligand+binding+free+energy+calculations&doi=10.1021%2Fct1001768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit195R"><div class="casContent"><span class="casTitleNuber">195</span><div class="casTitle"><span class="NLM_cas:atitle">Free Energy Perturbation Hamiltonian Replica-Exchange Molecular Dynamics (FEP/H-REMD) for Absolute Ligand Binding Free Energy Calculations</span></div><div class="casAuthors">Jiang, Wei; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2559-2565</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free Energy Perturbation with Replica Exchange Mol. Dynamics (FEP/REMD) offers a powerful strategy to improve the convergence of free energy computations.  In particular, it has been shown previously that a FEP/REMD scheme allowing random moves within an extended replica ensemble of thermodn. coupling parameters "λ" can improve the statistical convergence in calcns. of abs. binding free energy of ligands to proteins.  In the present study, FEP/REMD is extended and combined with an accelerated MD simulations method based on Hamiltonian replica-exchange MD (H-REMD) to overcome the addnl. problems arising from the existence of kinetically trapped conformations within the protein receptor.  In the combined strategy, each system with a given thermodn. coupling factor λ in the extended ensemble is further coupled with a set of replicas evolving on a biased energy surface with boosting potentials used to accelerate the interconversion among different rotameric states of the side chains in the neighborhood of the binding site.  Exchanges are allowed to occur alternatively along the axes corresponding to the thermodn. coupling parameter λ and the boosting potential, in an extended dual array of coupled λ- and H-REMD simulations.  The method is implemented on the basis of new extensions to the REPDSTR module of the biomol. simulation program CHARMM.  As an illustrative example, the abs. binding free energy of p-xylene to the nonpolar cavity of the L99A mutant of the T4 lysozyme was calcd.  The tests demonstrate that the dual λ-REMD and H-REMD simulation scheme greatly accelerates the configurational sampling of the rotameric states of the side chains around the binding pocket, thereby improving the convergence of the FEP computations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphmWym-fGS6rVg90H21EOLACvtfcHk0lifLS6KyQ8cgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsV2lsrY%253D&md5=4517b8d4fc1eaa08f3a000d6724ec7d5</span></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1021%2Fct1001768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct1001768%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DFree%2520energy%2520perturbation%2520hamiltonian%2520replica-exchange%2520molecular%2520dynamics%2520%2528FEP%252FH-REMD%2529%2520for%2520absolute%2520ligand%2520binding%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2559%26epage%3D2565%26doi%3D10.1021%2Fct1001768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref196"><div class="reference"><strong class="refLabel"><a href="#ref196" class="rightTabRefNumLink">196</a></strong><div class="NLM_citation" id="rightTab-cit196"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">The confine-and-release method: obtaining correct binding free energies in the presence of protein conformational change</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">–</span> <span class="NLM_lpage">1235</span><span class="refDoi"> DOI: 10.1021/ct700032n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700032n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=1231-1235&author=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=The+confine-and-release+method%3A+obtaining+correct+binding+free+energies+in+the+presence+of+protein+conformational+change&doi=10.1021%2Fct700032n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">Confine-and-Release Method: Obtaining Correct Binding Free Energies in the Presence of Protein Conformational Change</span></div><div class="casAuthors">Mobley, David L.; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1231-1235</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free energy calcns. are increasingly being used to est. abs. and relative binding free energies of ligands to proteins.  However, computed free energies often appear to depend on the initial protein conformation, indicating incomplete sampling.  This is esp. true when proteins can change conformation on ligand binding, as free energies assocd. with these conformational changes are either ignored or assumed to be included by virtue of the sampling performed in the calcn.  Here, the authors show that, in a model protein system (a designed binding site in T4 lysozyme), conformational changes can make a difference of several kcal/mol in computed binding free energies and that they are neglected in computed binding free energies if the system remains kinetically trapped in a particular metastable state on simulation timescales.  The authors introduce a general "confine-and-release" framework for free energy calcns. that accounts for these free energies of conformational change.  The authors illustrate its use in this model system by demonstrating that an umbrella sampling protocol can obtain converged binding free energies that are independent of the starting protein structure and include these conformational change free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHdRFLUwdd7LVg90H21EOLACvtfcHk0lifLS6KyQ8cgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D&md5=fa953a85593d81ee192ff6c5d7f525c2</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1021%2Fct700032n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700032n%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DThe%2520confine-and-release%2520method%253A%2520obtaining%2520correct%2520binding%2520free%2520energies%2520in%2520the%2520presence%2520of%2520protein%2520conformational%2520change%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D1231%26epage%3D1235%26doi%3D10.1021%2Fct700032n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref197"><div class="reference"><strong class="refLabel"><a href="#ref197" class="rightTabRefNumLink">197</a></strong><div class="NLM_citation" id="rightTab-cit197"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragliano, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Calculation of free energy landscape in multi-dimensions with hamiltonian-exchange umbrella sampling on petascale supercomputer</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">4672</span><span class="NLM_x">–</span> <span class="NLM_lpage">4680</span><span class="refDoi"> DOI: 10.1021/ct300468g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300468g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2iu7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=4672-4680&author=W.+Jiangauthor=Y.+Luoauthor=L.+Maraglianoauthor=B.+Roux&title=Calculation+of+free+energy+landscape+in+multi-dimensions+with+hamiltonian-exchange+umbrella+sampling+on+petascale+supercomputer&doi=10.1021%2Fct300468g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit197R"><div class="casContent"><span class="casTitleNuber">197</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of Free Energy Landscape in Multi-Dimensions with Hamiltonian-Exchange Umbrella Sampling on Petascale Supercomputer</span></div><div class="casAuthors">Jiang, Wei; Luo, Yun; Maragliano, Luca; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4672-4680</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An extremely scalable computational strategy is described for calcns. of the potential of mean force (PMF) in multidimensions on massively distributed supercomputers.  The approach involves coupling thousands of umbrella sampling (US) simulation windows distributed to cover the space of order parameters with a Hamiltonian mol. dynamics replica-exchange (H-REMD) algorithm to enhance the sampling of each simulation.  In the present application, US/H-REMD is carried out in a two-dimensional (2D) space and exchanges are attempted alternatively along the two axes corresponding to the two order parameters.  The US/H-REMD strategy is implemented on the basis of parallel/parallel multiple copy protocol at the MPI level, and therefore can fully exploit computing power of large-scale supercomputers.  Here the novel technique is illustrated using the leadership supercomputer IBM Blue Gene/P with an application to a typical biomol. calcn. of general interest, namely the binding of calcium ions to the small protein Calbindin D9k.  The free energy landscape assocd. with two order parameters, the distance between the ion and its binding pocket and the root-mean-square deviation (rmsd) of the binding pocket relative the crystal structure, was calcd. using the US/H-REMD method.  The results are then used to est. the abs. binding free energy of calcium ion to Calbindin D9k.  The tests demonstrate that the 2D US/H-REMD scheme greatly accelerates the configurational sampling of the binding pocket, thereby improving the convergence of the potential of mean force calcn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0t1ZtE0yTQ7Vg90H21EOLACvtfcHk0lgmkHMip6TEpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2iu7bE&md5=b0bb012cc6564e67cc030a8b7669cd05</span></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1021%2Fct300468g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300468g%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DMaragliano%26aufirst%3DL.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DCalculation%2520of%2520free%2520energy%2520landscape%2520in%2520multi-dimensions%2520with%2520hamiltonian-exchange%2520umbrella%2520sampling%2520on%2520petascale%2520supercomputer%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D4672%26epage%3D4680%26doi%3D10.1021%2Fct300468g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref198"><div class="reference"><strong class="refLabel"><a href="#ref198" class="rightTabRefNumLink">198</a></strong><div class="NLM_citation" id="rightTab-cit198"><span><span class="NLM_contrib-group">Sutto, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervasio, F. L.</span><span> </span><span class="NLM_article-title">Effects of oncogenic mutations on the conformational free-energy landscape of EGFR kinase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">10616</span><span class="NLM_x">–</span> <span class="NLM_lpage">10621</span><span class="refDoi"> DOI: 10.1073/pnas.1221953110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1221953110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23754386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFejs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=10616-10621&author=L.+Suttoauthor=F.+L.+Gervasio&title=Effects+of+oncogenic+mutations+on+the+conformational+free-energy+landscape+of+EGFR+kinase&doi=10.1073%2Fpnas.1221953110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of oncogenic mutations on the conformational free-energy landscape of EGFR kinase</span></div><div class="casAuthors">Sutto, Ludovico; Gervasio, Francesco Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10616-10621,S10616/1-S10616/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Activating mutations in the epidermal growth factor receptor (EGFR) tyrosine kinase are frequently found in many cancers.  It has been suggested that changes in the equil. between its active and inactive conformations are linked to its oncogenic potential.  Here, we quantify the effects of some of the most common single (L858R and T790M) and double (T790M-L858R) oncogenic mutations on the conformational free energy landscape of the EGFR kinase domain by using massive mol. dynamics simulations together with parallel tempering, metadynamics, and one of the best force-fields available.  Whereas the wild-type EGFR catalytic domain monomer is mostly found in an inactive conformation, our results show a clear shift toward the active conformation for all of the mutants.  The L858R mutation stabilizes the active conformation at the expense of the inactive conformation and rigidifies the αC-helix.  The T790M gatekeeper mutant favors activation by stabilizing a hydrophobic cluster.  Finally, T790M with L858R shows a significant pos. epistasis effect.  This combination not only stabilizes the active conformation, but in nontrivial ways changes the free-energy landscape lowering the transition barriers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXpgTGgqXQwbVg90H21EOLACvtfcHk0lgmkHMip6TEpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFejs73F&md5=4f582e8bcf5fc22c9499fbc6163ca725</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221953110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221953110%26sid%3Dliteratum%253Aachs%26aulast%3DSutto%26aufirst%3DL.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26atitle%3DEffects%2520of%2520oncogenic%2520mutations%2520on%2520the%2520conformational%2520free-energy%2520landscape%2520of%2520EGFR%2520kinase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D10616%26epage%3D10621%26doi%3D10.1073%2Fpnas.1221953110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref199"><div class="reference"><strong class="refLabel"><a href="#ref199" class="rightTabRefNumLink">199</a></strong><div class="NLM_citation" id="rightTab-cit199"><span><span class="NLM_contrib-group">Lapelosa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallicchio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">Conformational transitions and convergence of absolute binding free energy calculations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1021/ct200684b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200684b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKhurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=47-60&author=M.+Lapelosaauthor=E.+Gallicchioauthor=R.+M.+Levy&title=Conformational+transitions+and+convergence+of+absolute+binding+free+energy+calculations&doi=10.1021%2Fct200684b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit199R"><div class="casContent"><span class="casTitleNuber">199</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Transitions and Convergence of Absolute Binding Free Energy Calculations</span></div><div class="casAuthors">Lapelosa, Mauro; Gallicchio, Emilio; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-60</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Binding energy distribution anal. method (BEDAM) is employed to compute the std. binding free energies of a series of ligands to a FK506 binding protein (FKBP12) with implicit solvation.  Binding free energy ests. are in reasonably good agreement with exptl. affinities.  The conformations of the complexes identified by the simulations are in good agreement with crystallog. data, which were not used to restrain ligand orientations.  The BEDAM method is based on λ-hopping Hamiltonian parallel replica exchange (HREM) mol. dynamics conformational sampling, the OPLS-AA/AGBNP2 effective potential, and multistate free energy estimators (MBAR).  Achieving converged and accurate results depends on all of these elements of the calcn.  Convergence of the binding free energy is tied to the level of convergence of binding energy distributions at crit. intermediate states where bound and unbound states are at equil., and where the rate of binding/unbinding conformational transitions is maximal.  This finding mirrors similar observations in the context of order/disorder transitions as for example in protein folding.  Insights concerning the phys. mechanism of ligand binding and unbinding are obtained.  Convergence for the largest FK506 ligand is achieved only after imposing strict conformational restraints, which however require accurate prior structural knowledge of the structure of the complex.  The anal. AGBNP2 model is found to underestimate the magnitude of the hydrophobic driving force toward binding in these systems characterized by loosely packed protein-ligand binding interfaces.  Rescoring of the binding energies using a numerical surface area model corrects this deficiency.  This study illustrates the complex interplay between energy models, exploration of conformational space, and free energy estimators needed to obtain robust ests. from binding free energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdq9cmSAvLWbVg90H21EOLACvtfcHk0liM4hoZp0tvdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKhurnF&md5=36d9b9a68595d5bd28cea4cf79222960</span></div><a href="/servlet/linkout?suffix=cit199&amp;dbid=16384&amp;doi=10.1021%2Fct200684b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200684b%26sid%3Dliteratum%253Aachs%26aulast%3DLapelosa%26aufirst%3DM.%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DConformational%2520transitions%2520and%2520convergence%2520of%2520absolute%2520binding%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D47%26epage%3D60%26doi%3D10.1021%2Fct200684b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref200"><div class="reference"><strong class="refLabel"><a href="#ref200" class="rightTabRefNumLink">200</a></strong><div class="NLM_citation" id="rightTab-cit200"><span><span class="NLM_contrib-group">Procacci, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bizzarri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsili, S.</span><span> </span><span class="NLM_article-title">Energy-driven undocking (EDU-HREM) in solute tempering replica exchange simulations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span><span class="refDoi"> DOI: 10.1021/ct400809n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400809n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslymsrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=439-450&author=P.+Procacciauthor=M.+Bizzarriauthor=S.+Marsili&title=Energy-driven+undocking+%28EDU-HREM%29+in+solute+tempering+replica+exchange+simulations&doi=10.1021%2Fct400809n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit200R"><div class="casContent"><span class="casTitleNuber">200</span><div class="casTitle"><span class="NLM_cas:atitle">Energy-Driven Undocking (EDU-HREM) in Solute Tempering Replica Exchange Simulations</span></div><div class="casAuthors">Procacci, Piero; Bizzarri, Marco; Marsili, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new computational strategy for calcg. the abs. binding free energy for protein ligand assocn. in the context of atomistic simulation in explicit solvent.  The method is based on an appropriate definition of a solute tempering scheme enforced via Hamilton replica exchange method (HREM).  The definition of "solute" includes both the ligand and the active site, with the remainder of the systems defined as "solvent".  The hydrophilicity of the solute and the solute torsional plus nonbonded intrasolute interactions are increased and decreased, resp., along the replica progression, thus favoring the extrusion of the drug form the active site in the scaled states of the generalized ensemble.  The proposed technique, named "Energy Driven Undocking" (EDU-HREM), completely bypasses the need for defining and/or identifying the relevant reaction coordinates in a ligand receptor interactions and allows the calcn. of the abs. binding free energy in one single generalized simulation of the drug-receptor system.  The methodol. is applied, with encouraging results, to the calcn. of the abs. binding free energy of some FK506-related ligands of the peptidyl prolyl cis-trans isomerase protein (FKBP12) with known dissocn. consts.  Aspects of the binding/inhibition mechanism in FKBP12 are also analyzed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5AA0dW54SJrVg90H21EOLACvtfcHk0liM4hoZp0tvdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslymsrrI&md5=58271181ad379a332b75901b9775b50f</span></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1021%2Fct400809n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400809n%26sid%3Dliteratum%253Aachs%26aulast%3DProcacci%26aufirst%3DP.%26aulast%3DBizzarri%26aufirst%3DM.%26aulast%3DMarsili%26aufirst%3DS.%26atitle%3DEnergy-driven%2520undocking%2520%2528EDU-HREM%2529%2520in%2520solute%2520tempering%2520replica%2520exchange%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D439%26epage%3D450%26doi%3D10.1021%2Fct400809n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref201"><div class="reference"><strong class="refLabel"><a href="#ref201" class="rightTabRefNumLink">201</a></strong><div class="NLM_citation" id="rightTab-cit201"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Opinion - Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">739</span><span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Opinion+-+Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit201&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DOpinion%2520-%2520Drug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref202"><div class="reference"><strong class="refLabel"><a href="#ref202" class="rightTabRefNumLink">202</a></strong><div class="NLM_citation" id="rightTab-cit202"><span><span class="NLM_contrib-group">Pan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Molecular determinants of drug-receptor binding kinetics</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">673</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.drudis.2013.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23454741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=667-673&author=A.+C.+Panauthor=D.+W.+Borhaniauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Molecular+determinants+of+drug-receptor+binding+kinetics&doi=10.1016%2Fj.drudis.2013.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit202R"><div class="casContent"><span class="casTitleNuber">202</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of drug-receptor binding kinetics</span></div><div class="casAuthors">Pan, Albert C.; Borhani, David W.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">667-673</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  It is increasingly appreciated that the rates at which drugs assoc. with and dissoc. from receptors - the binding kinetics - directly impact drug efficacy and safety.  The mol. determinants of drug-receptor binding kinetics remain poorly understood, however, esp. when compared with the well-known factors that affect binding affinity.  The rational modulation of kinetics during lead optimization thus remains challenging.  We review some of the key factors thought to control drug-receptor binding kinetics at the mol. level - mol. size, conformational fluctuations, electrostatic interactions and hydrophobic effects - and discuss several possible approaches for the rational design of drugs with desired binding kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvEt3tx06Z7Vg90H21EOLACvtfcHk0lgqO_gHzGqK0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D&md5=df094bb86b1e280b9ddf5a90031b3f32</span></div><a href="/servlet/linkout?suffix=cit202&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DMolecular%2520determinants%2520of%2520drug-receptor%2520binding%2520kinetics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D667%26epage%3D673%26doi%3D10.1016%2Fj.drudis.2013.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref203"><div class="reference"><strong class="refLabel"><a href="#ref203" class="rightTabRefNumLink">203</a></strong><div class="NLM_citation" id="rightTab-cit203"><span><span class="NLM_contrib-group">Zhou, H. X.</span><span> </span><span class="NLM_article-title">Rate theories for biologists</span> <span class="citation_source-journal">Q. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">219</span><span class="NLM_x">–</span> <span class="NLM_lpage">293</span><span class="refDoi"> DOI: 10.1017/S0033583510000120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1017%2FS0033583510000120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20691138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC3cjpt1alug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=219-293&author=H.+X.+Zhou&title=Rate+theories+for+biologists&doi=10.1017%2FS0033583510000120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit203R"><div class="casContent"><span class="casTitleNuber">203</span><div class="casTitle"><span class="NLM_cas:atitle">Rate theories for biologists</span></div><div class="casAuthors">Zhou Huan-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly reviews of biophysics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">219-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Some of the rate theories that are most useful for modeling biological processes are reviewed.  By delving into some of the details and subtleties in the development of the theories, the review will hopefully help the reader gain a more than superficial perspective.  Examples are presented to illustrate how rate theories can be used to generate insight at the microscopic level into biomolecular behaviors.  An attempt is made to clear up a number of misconceptions in the literature regarding popular rate theories, including the appearance of Planck's constant in the transition-state theory and the Smoluchowski result as an upper limit for protein-protein and protein-DNA association rate constants.  Future work in combining the implementation of rate theories through computer simulations with experimental probes of rate processes, and in modeling effects of intracellular environments so that theories can be used for generating rate constants for systems biology studies is particularly exciting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRI95WOrBhtK9FxYDaSSP8XfW6udTcc2ebU1YB_vGAHObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjpt1alug%253D%253D&md5=6af727b8039583014ca9f70e6c6cfee8</span></div><a href="/servlet/linkout?suffix=cit203&amp;dbid=16384&amp;doi=10.1017%2FS0033583510000120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583510000120%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%2BX.%26atitle%3DRate%2520theories%2520for%2520biologists%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2010%26volume%3D43%26spage%3D219%26epage%3D293%26doi%3D10.1017%2FS0033583510000120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref204"><div class="reference"><strong class="refLabel"><a href="#ref204" class="rightTabRefNumLink">204</a></strong><div class="NLM_citation" id="rightTab-cit204"><span><span class="NLM_contrib-group">Pande, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, G. R.</span><span> </span><span class="NLM_article-title">Everything you wanted to know about markov state models but were afraid to ask</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2010.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.ymeth.2010.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20570730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2010&pages=99-105&author=V.+S.+Pandeauthor=K.+Beauchampauthor=G.+R.+Bowman&title=Everything+you+wanted+to+know+about+markov+state+models+but+were+afraid+to+ask&doi=10.1016%2Fj.ymeth.2010.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit204R"><div class="casContent"><span class="casTitleNuber">204</span><div class="casTitle"><span class="NLM_cas:atitle">Everything you wanted to know about Markov State Models but were afraid to ask</span></div><div class="casAuthors">Pande, Vijay S.; Beauchamp, Kyle; Bowman, Gregory R.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-105</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Simulating protein folding has been a challenging problem for decades due to the long timescales involved (compared with what is possible to simulate) and the challenges of gaining insight from the complex nature of the resulting simulation data.  Markov State Models (MSMs) present a means to tackle both of these challenges, yielding simulations on exptl. relevant timescales, statistical significance, and coarse grained representations that are readily humanly understandable.  Here, we review this method with the intended audience of non-experts, in order to introduce the method to a broader audience.  We review the motivations, methods, and caveats of MSMs, as well as some recent highlights of applications of the method.  We conclude by discussing how this approach is part of a paradigm shift in how one uses simulations, away from anecdotal single-trajectory approaches to a more comprehensive statistical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFtkKipM46Z7Vg90H21EOLACvtfcHk0lgqO_gHzGqK0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsLfK&md5=2afa98b7af7c65fb39a8cb61b5233f61</span></div><a href="/servlet/linkout?suffix=cit204&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2010.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2010.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DPande%26aufirst%3DV.%2BS.%26aulast%3DBeauchamp%26aufirst%3DK.%26aulast%3DBowman%26aufirst%3DG.%2BR.%26atitle%3DEverything%2520you%2520wanted%2520to%2520know%2520about%2520markov%2520state%2520models%2520but%2520were%2520afraid%2520to%2520ask%26jtitle%3DMethods%26date%3D2010%26volume%3D52%26spage%3D99%26epage%3D105%26doi%3D10.1016%2Fj.ymeth.2010.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref205"><div class="reference"><strong class="refLabel"><a href="#ref205" class="rightTabRefNumLink">205</a></strong><div class="NLM_citation" id="rightTab-cit205"><span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noe, F.</span><span> </span><span class="NLM_article-title">Markov state models of biomolecular conformational dynamics</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">135</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span><span class="refDoi"> DOI: 10.1016/j.sbi.2014.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.sbi.2014.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24836551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovFChu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=135-144&author=J.+D.+Choderaauthor=F.+Noe&title=Markov+state+models+of+biomolecular+conformational+dynamics&doi=10.1016%2Fj.sbi.2014.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit205R"><div class="casContent"><span class="casTitleNuber">205</span><div class="casTitle"><span class="NLM_cas:atitle">Markov state models of biomolecular conformational dynamics</span></div><div class="casAuthors">Chodera, John D.; Noe, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">135-144</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  It has recently become practical to construct Markov state models (MSMs) that reproduce the long-time statistical conformational dynamics of biomols. using data from mol. dynamics simulations.  MSMs can predict both stationary and kinetic quantities on long timescales (e.g. milliseconds) using a set of atomistic mol. dynamics simulations that are individually much shorter, thus addressing the well-known sampling problem in mol. dynamics simulation.  In addn. to providing predictive quant. models, MSMs greatly facilitate both the extn. of insight into biomol. mechanism (such as folding and functional dynamics) and quant. comparison with single-mol. and ensemble kinetics expts.  A variety of methodol. advances and software packages now bring the construction of these models closer to routine practice.  Here, we review recent progress in this field, considering theor. and methodol. advances, new software tools, and recent applications of these approaches in several domains of biochem. and biophysics, commenting on remaining challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyht1KHY5Z7Vg90H21EOLACvtfcHk0lh8NdRV-Ctw9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovFChu7s%253D&md5=4aab299d1bec5b92acc554341bc89b16</span></div><a href="/servlet/linkout?suffix=cit205&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2014.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2014.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DNoe%26aufirst%3DF.%26atitle%3DMarkov%2520state%2520models%2520of%2520biomolecular%2520conformational%2520dynamics%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D25%26spage%3D135%26epage%3D144%26doi%3D10.1016%2Fj.sbi.2014.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref206"><div class="reference"><strong class="refLabel"><a href="#ref206" class="rightTabRefNumLink">206</a></strong><div class="NLM_citation" id="rightTab-cit206"><span><span class="NLM_contrib-group">Gu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span> </span><span class="NLM_article-title">Quantitatively characterizing the ligand binding mechanisms of choline binding protein using Markov state model analysis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003767</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pcbi.1003767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25101697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ars73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003767&author=S.+Guauthor=D.+A.+Silvaauthor=L.+Mengauthor=A.+Yueauthor=X.+Huang&title=Quantitatively+characterizing+the+ligand+binding+mechanisms+of+choline+binding+protein+using+Markov+state+model+analysis&doi=10.1371%2Fjournal.pcbi.1003767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit206R"><div class="casContent"><span class="casTitleNuber">206</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitatively characterizing the ligand binding mechanisms of choline binding protein using markov state model analysis</span></div><div class="casAuthors">Gu, Shuo; Silva, Daniel-Adriano; Meng, Luming; Yue, Alexander; Huang, Xuhui</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1003767/1-e1003767/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein-ligand recognition plays key roles in many biol. processes.  One of the most fascinating questions about protein-ligand recognition is to understand its underlying mechanism, which often results from a combination of induced fit and conformational selection.  In this study, we have developed a three-pronged approach of Markov State Models, Mol. Dynamics simulations, and flux anal. to det. the contribution of each model.  Using this approach, we have quantified the recognition mechanism of the choline binding protein (ChoX) to be ∼90% conformational selection dominant under exptl. conditions.  This is achieved by recovering all the necessary parameters for the flux anal. in combination with available exptl. data.  Our results also suggest that ChoX has several metastable conformational states, of which an apo-closed state is dominant, consistent with previous exptl. findings.  Our methodol. holds great potential to be widely applied to understand recognition mechanisms underlining many fundamental biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosNNPrkzNMfbVg90H21EOLACvtfcHk0lh8NdRV-Ctw9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ars73E&md5=a59ce9737dc4afa610544e0e18f5edcc</span></div><a href="/servlet/linkout?suffix=cit206&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003767%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DS.%26aulast%3DSilva%26aufirst%3DD.%2BA.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DQuantitatively%2520characterizing%2520the%2520ligand%2520binding%2520mechanisms%2520of%2520choline%2520binding%2520protein%2520using%2520Markov%2520state%2520model%2520analysis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003767%26doi%3D10.1371%2Fjournal.pcbi.1003767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref207"><div class="reference"><strong class="refLabel"><a href="#ref207" class="rightTabRefNumLink">207</a></strong><div class="NLM_citation" id="rightTab-cit207"><span><span class="NLM_contrib-group">Plattner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noe, F.</span><span> </span><span class="NLM_article-title">Protein conformational plasticity and complex ligand-binding kinetics explored by atomistic simulations and Markov models</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">7653</span><span class="refDoi"> DOI: 10.1038/ncomms8653</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fncomms8653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26134632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC28%252Fgs1ajug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7653&author=N.+Plattnerauthor=F.+Noe&title=Protein+conformational+plasticity+and+complex+ligand-binding+kinetics+explored+by+atomistic+simulations+and+Markov+models&doi=10.1038%2Fncomms8653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit207R"><div class="casContent"><span class="casTitleNuber">207</span><div class="casTitle"><span class="NLM_cas:atitle">Protein conformational plasticity and complex ligand-binding kinetics explored by atomistic simulations and Markov models</span></div><div class="casAuthors">Plattner Nuria; Noe Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7653</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Understanding the structural mechanisms of protein-ligand binding and their dependence on protein sequence and conformation is of fundamental importance for biomedical research.  Here we investigate the interplay of conformational change and ligand-binding kinetics for the serine protease Trypsin and its competitive inhibitor Benzamidine with an extensive set of 150 μs molecular dynamics simulation data, analysed using a Markov state model.  Seven metastable conformations with different binding pocket structures are found that interconvert at timescales of tens of microseconds.  These conformations differ in their substrate-binding affinities and binding/dissociation rates.  For each metastable state, corresponding solved structures of Trypsin mutants or similar serine proteases are contained in the protein data bank.  Thus, our wild-type simulations explore a space of conformations that can be individually stabilized by adding ligands or making suitable changes in protein sequence.  These findings provide direct evidence of conformational plasticity in receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUuRoXD3S0w5ZoeNcnHOQ9fW6udTcc2ebuLdagkbNiU7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252Fgs1ajug%253D%253D&md5=4d2c200f1a134d8448e1ae1588dce791</span></div><a href="/servlet/linkout?suffix=cit207&amp;dbid=16384&amp;doi=10.1038%2Fncomms8653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8653%26sid%3Dliteratum%253Aachs%26aulast%3DPlattner%26aufirst%3DN.%26aulast%3DNoe%26aufirst%3DF.%26atitle%3DProtein%2520conformational%2520plasticity%2520and%2520complex%2520ligand-binding%2520kinetics%2520explored%2520by%2520atomistic%2520simulations%2520and%2520Markov%2520models%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7653%26doi%3D10.1038%2Fncomms8653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref208"><div class="reference"><strong class="refLabel"><a href="#ref208" class="rightTabRefNumLink">208</a></strong><div class="NLM_citation" id="rightTab-cit208"><span><span class="NLM_contrib-group">Doerr, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">On-the-Fly Learning and sampling of ligand binding by high-throughput molecular simulations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2064</span><span class="NLM_x">–</span> <span class="NLM_lpage">2069</span><span class="refDoi"> DOI: 10.1021/ct400919u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400919u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvVyrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=2064-2069&author=S.+Doerrauthor=G.+De+Fabritiis&title=On-the-Fly+Learning+and+sampling+of+ligand+binding+by+high-throughput+molecular+simulations&doi=10.1021%2Fct400919u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit208R"><div class="casContent"><span class="casTitleNuber">208</span><div class="casTitle"><span class="NLM_cas:atitle">On-the-Fly Learning and Sampling of Ligand Binding by High-Throughput Molecular Simulations</span></div><div class="casAuthors">Doerr, S.; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2064-2069</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput mol. dynamics (MD) simulations are a computational method consisting of using multiple short trajectories, instead of few long ones, to cover slow biol. time scales.  Compared to long trajectories this method offers the possibility to start the simulations in successive batches, building a knowledgeable model of the available data to inform subsequent new simulations iteratively.  Here, we demonstrate an automatic, iterative, on-the-fly method for learning and sampling mol. simulations in the context of ligand binding for the case of trypsin-benzamidine binding.  The method uses Markov state models to learn a simplified model of the simulations and decide where best to sample from, achieving a converged binding affinity in approx. one microsecond, 1 order of magnitude faster than classical sampling.  This method demonstrates for the first time the potential of adaptive sampling schemes in the case of ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5z81HNCApn7Vg90H21EOLACvtfcHk0lgW1in2on9L6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvVyrtrg%253D&md5=7ff57bf52252341784fe709f8906e214</span></div><a href="/servlet/linkout?suffix=cit208&amp;dbid=16384&amp;doi=10.1021%2Fct400919u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400919u%26sid%3Dliteratum%253Aachs%26aulast%3DDoerr%26aufirst%3DS.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DOn-the-Fly%2520Learning%2520and%2520sampling%2520of%2520ligand%2520binding%2520by%2520high-throughput%2520molecular%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D2064%26epage%3D2069%26doi%3D10.1021%2Fct400919u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref209"><div class="reference"><strong class="refLabel"><a href="#ref209" class="rightTabRefNumLink">209</a></strong><div class="NLM_citation" id="rightTab-cit209"><span><span class="NLM_contrib-group">Lu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span> </span><span class="NLM_article-title">Drug-target residence time: critical information for lead optimization</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.06.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.cbpa.2010.06.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=20663707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=467-474&author=H.+Luauthor=P.+J.+Tonge&title=Drug-target+residence+time%3A+critical+information+for+lead+optimization&doi=10.1016%2Fj.cbpa.2010.06.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit209R"><div class="casContent"><span class="casTitleNuber">209</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time: Critical information for lead optimization</span></div><div class="casAuthors">Lu, Hao; Tonge, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Failure due to poor in vivo efficacy is a primary contributor to attrition during the development of new chemotherapeutics.  Lead optimization programs that in their quest for efficacy focus solely on improving the affinity of drug-target binding are flawed, since this approach ignores the fluctuations in drug concn. that occur in vivo.  Instead the lifetime of the drug-target complex must also be considered, since drugs only act when they are bound to their targets.  Consequently, to improve the correlation between the in vitro and in vivo activity of drugs, measurements of drug-target residence time must be incorporated into the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk2kN_txuMLVg90H21EOLACvtfcHk0lgW1in2on9L6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D&md5=3189f56032106db858cfb4269d3fc11f</span></div><a href="/servlet/linkout?suffix=cit209&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.06.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.06.176%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DDrug-target%2520residence%2520time%253A%2520critical%2520information%2520for%2520lead%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D467%26epage%3D474%26doi%3D10.1016%2Fj.cbpa.2010.06.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref210"><div class="reference"><strong class="refLabel"><a href="#ref210" class="rightTabRefNumLink">210</a></strong><div class="NLM_citation" id="rightTab-cit210"><span><span class="NLM_contrib-group">Mollica, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decherchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zia, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaspari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span> </span><span class="NLM_article-title">Kinetics of protein-ligand unbinding via smoothed potential molecular dynamics simulations</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">11539</span><span class="refDoi"> DOI: 10.1038/srep11539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fsrep11539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26103621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11539&author=L.+Mollicaauthor=S.+Decherchiauthor=S.+R.+Ziaauthor=R.+Gaspariauthor=A.+Cavalliauthor=W.+Rocchia&title=Kinetics+of+protein-ligand+unbinding+via+smoothed+potential+molecular+dynamics+simulations&doi=10.1038%2Fsrep11539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit210R"><div class="casContent"><span class="casTitleNuber">210</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of protein-ligand unbinding via smoothed potential molecular dynamics simulations</span></div><div class="casAuthors">Mollica Luca; Decherchi Sergio; Zia Syeda Rehana; Gaspari Roberto; Rocchia Walter; Cavalli Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug discovery is expensive and high-risk.  Its main reasons of failure are lack of efficacy and toxicity of a drug candidate.  Binding affinity for the biological target has been usually considered one of the most relevant figures of merit to judge a drug candidate along with bioavailability, selectivity and metabolic properties, which could depend on off-target interactions.  Nevertheless, affinity does not always satisfactorily correlate with in vivo drug efficacy.  It is indeed becoming increasingly evident that the time a drug spends in contact with its target (aka residence time) can be a more reliable figure of merit.  Experimental kinetic measurements are operatively limited by the cost and the time needed to synthesize compounds to be tested, to express and purify the target, and to setup the assays.  We present here a simple and efficient molecular-dynamics-based computational approach to prioritize compounds according to their residence time.  We devised a multiple-replica scaled molecular dynamics protocol with suitably defined harmonic restraints to accelerate the unbinding events while preserving the native fold.  Ligands are ranked according to the mean observed scaled unbinding time.  The approach, trivially parallel and easily implementable, was validated against experimental information available on biological systems of pharmacological relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxJkC0jZN2SEH3Rlp_k91zfW6udTcc2eYn7tOb33vSPrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D&md5=8e3963bce57cf49954ea50fc004283d6</span></div><a href="/servlet/linkout?suffix=cit210&amp;dbid=16384&amp;doi=10.1038%2Fsrep11539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11539%26sid%3Dliteratum%253Aachs%26aulast%3DMollica%26aufirst%3DL.%26aulast%3DDecherchi%26aufirst%3DS.%26aulast%3DZia%26aufirst%3DS.%2BR.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DKinetics%2520of%2520protein-ligand%2520unbinding%2520via%2520smoothed%2520potential%2520molecular%2520dynamics%2520simulations%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11539%26doi%3D10.1038%2Fsrep11539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref211"><div class="reference"><strong class="refLabel"><a href="#ref211" class="rightTabRefNumLink">211</a></strong><div class="NLM_citation" id="rightTab-cit211"><span><span class="NLM_contrib-group">Bui, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The dynamics of ligand barrier crossing inside the acetylcholinesterase gorge</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2267</span><span class="NLM_x">–</span> <span class="NLM_lpage">2272</span><span class="refDoi"> DOI: 10.1016/S0006-3495(03)74651-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS0006-3495%2803%2974651-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2003&pages=2267-2272&author=J.+M.+Buiauthor=R.+H.+Henchmanauthor=J.+A.+McCammon&title=The+dynamics+of+ligand+barrier+crossing+inside+the+acetylcholinesterase+gorge&doi=10.1016%2FS0006-3495%2803%2974651-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit211&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2803%2974651-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252803%252974651-7%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DJ.%2BM.%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520dynamics%2520of%2520ligand%2520barrier%2520crossing%2520inside%2520the%2520acetylcholinesterase%2520gorge%26jtitle%3DBiophys.%2520J.%26date%3D2003%26volume%3D85%26spage%3D2267%26epage%3D2272%26doi%3D10.1016%2FS0006-3495%2803%2974651-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref212"><div class="reference"><strong class="refLabel"><a href="#ref212" class="rightTabRefNumLink">212</a></strong><div class="NLM_citation" id="rightTab-cit212"><span><span class="NLM_contrib-group">Marinelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietrucci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piana, S.</span><span> </span><span class="NLM_article-title">A kinetic model of trp-cage folding from multiple biased molecular dynamics simulations</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000452</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pcbi.1000452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19662155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A280%3ADC%252BD1MrktFSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000452&author=F.+Marinelliauthor=F.+Pietrucciauthor=A.+Laioauthor=S.+Piana&title=A+kinetic+model+of+trp-cage+folding+from+multiple+biased+molecular+dynamics+simulations&doi=10.1371%2Fjournal.pcbi.1000452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit212R"><div class="casContent"><span class="casTitleNuber">212</span><div class="casTitle"><span class="NLM_cas:atitle">A kinetic model of trp-cage folding from multiple biased molecular dynamics simulations</span></div><div class="casAuthors">Marinelli Fabrizio; Pietrucci Fabio; Laio Alessandro; Piana Stefano</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1000452</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Trp-cage is a designed 20-residue polypeptide that, in spite of its size, shares several features with larger globular proteins.Although the system has been intensively investigated experimentally and theoretically, its folding mechanism is not yet fully understood.  Indeed, some experiments suggest a two-state behavior, while others point to the presence of intermediates.  In this work we show that the results of a bias-exchange metadynamics simulation can be used for constructing a detailed thermodynamic and kinetic model of the system.  The model, although constructed from a biased simulation, has a quality similar to those extracted from the analysis of long unbiased molecular dynamics trajectories.  This is demonstrated by a careful benchmark of the approach on a smaller system, the solvated Ace-Ala3-Nme peptide.  For theTrp-cage folding, the model predicts that the relaxation time of 3100 ns observed experimentally is due to the presence of a compact molten globule-like conformation.  This state has an occupancy of only 3% at 300 K, but acts as a kinetic trap.Instead, non-compact structures relax to the folded state on the sub-microsecond timescale.  The model also predicts the presence of a state at Calpha-RMSD of 4.4 A from the NMR structure in which the Trp strongly interacts with Pro12.  This state can explain the abnormal temperature dependence of the Pro12-delta3 and Gly11-alpha3 chemical shifts.  The structures of the two most stable misfolded intermediates are in agreement with NMR experiments on the unfolded protein.  Our work shows that, using biased molecular dynamics trajectories, it is possible to construct a model describing in detail the Trp-cage folding kinetics and thermodynamics in agreement with experimental data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRZmzIeeqUN7TMsgLhIoICsfW6udTcc2eYn7tOb33vSPrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrktFSitg%253D%253D&md5=42e8c3d2eeb63936b027aac05072325e</span></div><a href="/servlet/linkout?suffix=cit212&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000452%26sid%3Dliteratum%253Aachs%26aulast%3DMarinelli%26aufirst%3DF.%26aulast%3DPietrucci%26aufirst%3DF.%26aulast%3DLaio%26aufirst%3DA.%26aulast%3DPiana%26aufirst%3DS.%26atitle%3DA%2520kinetic%2520model%2520of%2520trp-cage%2520folding%2520from%2520multiple%2520biased%2520molecular%2520dynamics%2520simulations%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000452%26doi%3D10.1371%2Fjournal.pcbi.1000452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref213"><div class="reference"><strong class="refLabel"><a href="#ref213" class="rightTabRefNumLink">213</a></strong><div class="NLM_citation" id="rightTab-cit213"><span><span class="NLM_contrib-group">Pietrucci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marinelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carloni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laio, A.</span><span> </span><span class="NLM_article-title">Substrate binding mechanism of hiv-1 protease from explicit-solvent atomistic simulations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">11811</span><span class="NLM_x">–</span> <span class="NLM_lpage">11818</span><span class="refDoi"> DOI: 10.1021/ja903045y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja903045y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1GqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=11811-11818&author=F.+Pietrucciauthor=F.+Marinelliauthor=P.+Carloniauthor=A.+Laio&title=Substrate+binding+mechanism+of+hiv-1+protease+from+explicit-solvent+atomistic+simulations&doi=10.1021%2Fja903045y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit213R"><div class="casContent"><span class="casTitleNuber">213</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Binding Mechanism of HIV-1 Protease from Explicit-Solvent Atomistic Simulations</span></div><div class="casAuthors">Pietrucci, Fabio; Marinelli, Fabrizio; Carloni, Paolo; Laio, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11811-11818</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding mechanism of a peptide substrate (Thr-Ile-Met-Met-Gln-Arg, cleavage site p2-NC of the viral polyprotein) to wild-type HIV-1 protease has been investigated by 1.6 μs biased all-atom mol. dynamics (MD) simulations in explicit water.  The configuration space has been explored biasing seven reaction coordinates by the bias-exchange metadynamics technique.  The structure of the Michaelis complex is obtained starting from the substrate outside the enzyme within a backbone rmsd of 0.9 Å.  The calcd. free energy of binding is -6 kcal/mol, and the kinetic consts. for assocn. and dissocn. are 1.3 × 106 M-1 s-1 and 57 s-1, resp., consistent with expts.  In the main binding pathway, the flaps of the protease do not open sizably.  The substrate slides inside the enzyme cavity from the tight lateral channel.  This may contrast with the natural polyprotein substrate which is expected to bind by opening the flaps.  Thus, mutations might influence differently the binding kinetics of peptidomimetic ligands and of the natural substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXYaLQMq4PG7Vg90H21EOLACvtfcHk0lg-xCFxT00pTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1GqtLs%253D&md5=0a0ff8c4a988a4518566258855cb56fe</span></div><a href="/servlet/linkout?suffix=cit213&amp;dbid=16384&amp;doi=10.1021%2Fja903045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja903045y%26sid%3Dliteratum%253Aachs%26aulast%3DPietrucci%26aufirst%3DF.%26aulast%3DMarinelli%26aufirst%3DF.%26aulast%3DCarloni%26aufirst%3DP.%26aulast%3DLaio%26aufirst%3DA.%26atitle%3DSubstrate%2520binding%2520mechanism%2520of%2520hiv-1%2520protease%2520from%2520explicit-solvent%2520atomistic%2520simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D11811%26epage%3D11818%26doi%3D10.1021%2Fja903045y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref214"><div class="reference"><strong class="refLabel"><a href="#ref214" class="rightTabRefNumLink">214</a></strong><div class="NLM_citation" id="rightTab-cit214"><span><span class="NLM_contrib-group">Tiwary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">From Metadynamics to Dynamics</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">230602</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.111.230602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevLett.111.230602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24476246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVSrs7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2013&pages=230602&author=P.+Tiwaryauthor=M.+Parrinello&title=From+Metadynamics+to+Dynamics&doi=10.1103%2FPhysRevLett.111.230602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit214R"><div class="casContent"><span class="casTitleNuber">214</span><div class="casTitle"><span class="NLM_cas:atitle">From metadynamics to dynamics</span></div><div class="casAuthors">Tiwary, Pratyush; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">230602/1-230602/5</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">Metadynamics is a commonly used and successful enhanced sampling method.  By the introduction of a history dependent bias which depends on a restricted no. of collective variables it can explore complex free energy surfaces characterized by several metastable states sepd. by large free energy barriers.  Here we extend its scope by introducing a simple yet powerful method for calcg. the rates of transition between different metastable states.  The method does not rely on a previous knowledge of the transition states or reaction coordinates, as long as collective variables are known that can distinguish between the various stable min. in free energy space.  We demonstrate that our method recovers the correct escape rates out of these stable states and also preserves the correct sequence of state-to-state transitions, with minimal extra computational effort needed over ordinary metadynamics.  We apply the formalism to three different problems and in each case find excellent agreement with the results of long unbiased mol. dynamics runs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOKDPbiMlopbVg90H21EOLACvtfcHk0lg-xCFxT00pTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVSrs7nF&md5=3ccbf6ea725bd13d42f5bfb04cef1c51</span></div><a href="/servlet/linkout?suffix=cit214&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.111.230602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.111.230602%26sid%3Dliteratum%253Aachs%26aulast%3DTiwary%26aufirst%3DP.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DFrom%2520Metadynamics%2520to%2520Dynamics%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2013%26volume%3D111%26spage%3D230602%26doi%3D10.1103%2FPhysRevLett.111.230602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref215"><div class="reference"><strong class="refLabel"><a href="#ref215" class="rightTabRefNumLink">215</a></strong><div class="NLM_citation" id="rightTab-cit215"><span><span class="NLM_contrib-group">Salvalaglio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrinello, M.</span><span> </span><span class="NLM_article-title">Assessing the Reliability of the dynamics reconstructed from metadynamics</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1420</span><span class="NLM_x">–</span> <span class="NLM_lpage">1425</span><span class="refDoi"> DOI: 10.1021/ct500040r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500040r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref215/cit215&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtV2rurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1420-1425&author=M.+Salvalaglioauthor=P.+Tiwaryauthor=M.+Parrinello&title=Assessing+the+Reliability+of+the+dynamics+reconstructed+from+metadynamics&doi=10.1021%2Fct500040r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit215R"><div class="casContent"><span class="casTitleNuber">215</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Reliability of the Dynamics Reconstructed from Metadynamics</span></div><div class="casAuthors">Salvalaglio, Matteo; Tiwary, Pratyush; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1420-1425</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sampling a mol. process characterized by an activation free energy significantly larger than kBT is a well-known challenge in mol. dynamics simulations.  In a recent work, we have demonstrated that the transition times of activated mol. transformations can be computed from well-tempered metadynamics provided that no bias is deposited in the transition state region and that the set of collective variables chosen to enhance sampling does not display hysteresis.  Ensuring though that these two criteria are met may not always be simple.  Here we build on the fact that the times of escape from a long-lived metastable state obey Poisson statistics.  This allows us to identify quant. measures of trustworthiness of our calcn.  We test our method on a few paradigmatic examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF3cPWRv1ldbVg90H21EOLACvtfcHk0lg-xCFxT00pTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtV2rurg%253D&md5=04b0084a6f6ff1ea2ad30bf746553464</span></div><a href="/servlet/linkout?suffix=cit215&amp;dbid=16384&amp;doi=10.1021%2Fct500040r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500040r%26sid%3Dliteratum%253Aachs%26aulast%3DSalvalaglio%26aufirst%3DM.%26aulast%3DTiwary%26aufirst%3DP.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DAssessing%2520the%2520Reliability%2520of%2520the%2520dynamics%2520reconstructed%2520from%2520metadynamics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D1420%26epage%3D1425%26doi%3D10.1021%2Fct500040r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref216"><div class="reference"><strong class="refLabel"><a href="#ref216" class="rightTabRefNumLink">216</a></strong><div class="NLM_citation" id="rightTab-cit216"><span><span class="NLM_contrib-group">Changeux, J. P.</span><span> </span><span class="NLM_article-title">Allostery and the monod-wyman-changeux model after 50 years</span> <span class="citation_source-journal">Annu. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1146/annurev-biophys-050511-102222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1146%2Fannurev-biophys-050511-102222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22224598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1yitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=103-133&author=J.+P.+Changeux&title=Allostery+and+the+monod-wyman-changeux+model+after+50+years&doi=10.1146%2Fannurev-biophys-050511-102222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit216R"><div class="casContent"><span class="casTitleNuber">216</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery and the Monod-Wyman-Changeux model after 50 years</span></div><div class="casAuthors">Changeux, Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-133</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The Monod-Wyman-Changeux (MWC) model was conceived in 1965 to account for the signal transduction and cooperative properties of bacterial regulatory enzymes and Hb.  It was soon extended to pharmacol. receptors for neurotransmitters and other macromol. entities involved in intracellular and intercellular communications.  Five decades later, the 2 main hypotheses of the model are reexamd. on the basis of a variety of regulatory proteins with known x-ray structures: (1) regulatory proteins possess an oligomeric structure with symmetry properties, and (2) the allosteric interactions between topog. distinct sites are mediated by a conformational transition established between a few preestablished states with conservation of symmetry and ligand-directed conformational selection.  Several well-documented examples are adequately represented by the MWC model, yet a few possible exceptions are noted.  New questions are raised concerning the dynamics of the allosteric transitions and more complex supramol. ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMZolzBzmVNLVg90H21EOLACvtfcHk0lg-xCFxT00pTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1yitbc%253D&md5=0df027444211c5471de1236271ffa76d</span></div><a href="/servlet/linkout?suffix=cit216&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-050511-102222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-050511-102222%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.%2BP.%26atitle%3DAllostery%2520and%2520the%2520monod-wyman-changeux%2520model%2520after%252050%2520years%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D41%26spage%3D103%26epage%3D133%26doi%3D10.1146%2Fannurev-biophys-050511-102222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref217"><div class="reference"><strong class="refLabel"><a href="#ref217" class="rightTabRefNumLink">217</a></strong><div class="NLM_citation" id="rightTab-cit217"><span><span class="NLM_contrib-group">Tsai, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span> </span><span class="NLM_article-title">A Unified view of “how allostery works”</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003394</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1371%2Fjournal.pcbi.1003394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=24516370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003394&author=C.+J.+Tsaiauthor=R.+Nussinov&title=A+Unified+view+of+%E2%80%9Chow+allostery+works%E2%80%9D&doi=10.1371%2Fjournal.pcbi.1003394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit217R"><div class="casContent"><span class="casTitleNuber">217</span><div class="casTitle"><span class="NLM_cas:atitle">A unified view of "how allostery works"</span></div><div class="casAuthors">Tsai, Chung-Jung; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1003394/1-e1003394/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The question of how allostery works was posed almost 50 years ago.  Since then it has been the focus of much effort.  This is for two reasons: first, the intellectual curiosity of basic science and the desire to understand fundamental phenomena, and second, its vast practical importance.  Allostery is at play in all processes in the living cell, and increasingly in drug discovery.  Many models have been successfully formulated, and are able to describe allostery even in the absence of a detailed structural mechanism.  However, conceptual schemes designed to qual. explain allosteric mechanisms usually lack a quant. math. model, and are unable to link its thermodn. and structural foundations.  This hampers insight into oncogenic mutations in cancer progression and biased agonists' actions.  Here, we describe how allostery works from three different standpoints: thermodn., free energy landscape of population shift, and structure; all with exactly the same allosteric descriptors.  This results in a unified view which not only clarifies the elusive allosteric mechanism but also provides structural grasp of agonist-mediated signaling pathways, and guides allosteric drug discovery.  Of note, the unified view reasons that allosteric coupling (or communication) does not det. the allosteric efficacy; however, a communication channel is what makes potential binding sites allosteric.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEgA5AKySrFbVg90H21EOLACvtfcHk0li6KoHvsm9HZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D&md5=3912bf89813eeed254b3e3e4e1937f99</span></div><a href="/servlet/linkout?suffix=cit217&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003394%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DA%2520Unified%2520view%2520of%2520%25E2%2580%259Chow%2520allostery%2520works%25E2%2580%259D%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003394%26doi%3D10.1371%2Fjournal.pcbi.1003394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref218"><div class="reference"><strong class="refLabel"><a href="#ref218" class="rightTabRefNumLink">218</a></strong><div class="NLM_citation" id="rightTab-cit218"><span><span class="NLM_contrib-group">Lee, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span> </span><span class="NLM_article-title">Trapping moving targets with small molecules</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">324</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">215</span><span class="refDoi"> DOI: 10.1126/science.1169378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1126%2Fscience.1169378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19359579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFalsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=213-215&author=G.+M.+Leeauthor=C.+S.+Craik&title=Trapping+moving+targets+with+small+molecules&doi=10.1126%2Fscience.1169378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit218R"><div class="casContent"><span class="casTitleNuber">218</span><div class="casTitle"><span class="NLM_cas:atitle">Trapping Moving Targets with Small Molecules</span></div><div class="casAuthors">Lee, Gregory M.; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5924</span>),
    <span class="NLM_cas:pages">213-215</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review and discussion.  Structure-based drug design traditionally uses static protein models as inspirations for focusing on "active" site targets.  Allosteric regulation of biol. macromols., however, is affected by both conformational and dynamic properties of the protein or protein complex and can potentially lead to more avenues for therapeutic development.  We discuss the advantages of searching for mols. that conformationally trap a macromol. in its inactive state.  Although multiple methodologies exist to probe protein dynamics and ligand binding, our current discussion highlights the use of NMR spectroscopy in the drug discovery and design process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCoLaWvdf9x7Vg90H21EOLACvtfcHk0li6KoHvsm9HZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFalsro%253D&md5=69579d00b8d0e8dff1a8bfbf3c6632df</span></div><a href="/servlet/linkout?suffix=cit218&amp;dbid=16384&amp;doi=10.1126%2Fscience.1169378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1169378%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DG.%2BM.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DTrapping%2520moving%2520targets%2520with%2520small%2520molecules%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D213%26epage%3D215%26doi%3D10.1126%2Fscience.1169378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref219"><div class="reference"><strong class="refLabel"><a href="#ref219" class="rightTabRefNumLink">219</a></strong><div class="NLM_citation" id="rightTab-cit219"><span><span class="NLM_contrib-group">Conn, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christopoulos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindsley, C. W.</span><span> </span><span class="NLM_article-title">Allosteric modulators of GPCRs: a novel approach for the treatment of CNS disorders</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span><span class="refDoi"> DOI: 10.1038/nrd2760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1038%2Fnrd2760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19116626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFCkurfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=41-54&author=P.+J.+Connauthor=A.+Christopoulosauthor=C.+W.+Lindsley&title=Allosteric+modulators+of+GPCRs%3A+a+novel+approach+for+the+treatment+of+CNS+disorders&doi=10.1038%2Fnrd2760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit219R"><div class="casContent"><span class="casTitleNuber">219</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulators of GPCRs: a novel approach for the treatment of CNS disorders</span></div><div class="casAuthors">Conn, P. Jeffrey; Christopoulos, Arthur; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-54</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite G-protein-coupled receptors (GPCRs) being among the most fruitful targets for marketed drugs, intense discovery efforts for several GPCR subtypes have failed to deliver selective drug candidates.  Historically, drug discovery programs for GPCR ligands have been dominated by efforts to develop agonists and antagonists that act at orthosteric sites for endogenous ligands.  However, in recent years, there have been tremendous advances in the discovery of novel ligands for GPCRs that act at allosteric sites to regulate receptor function.  These compds. provide high selectivity, novel modes of efficacy and may lead to novel therapeutic agents for the treatment of multiple psychiatric and neurol. human disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmn5h5XCcPKbVg90H21EOLACvtfcHk0li6KoHvsm9HZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFCkurfF&md5=e109e94fe3e2c1fcfd480dc84c6609f7</span></div><a href="/servlet/linkout?suffix=cit219&amp;dbid=16384&amp;doi=10.1038%2Fnrd2760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2760%26sid%3Dliteratum%253Aachs%26aulast%3DConn%26aufirst%3DP.%2BJ.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DAllosteric%2520modulators%2520of%2520GPCRs%253A%2520a%2520novel%2520approach%2520for%2520the%2520treatment%2520of%2520CNS%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D41%26epage%3D54%26doi%3D10.1038%2Fnrd2760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref220"><div class="reference"><strong class="refLabel"><a href="#ref220" class="rightTabRefNumLink">220</a></strong><div class="NLM_citation" id="rightTab-cit220"><span><span class="NLM_contrib-group">Chatzidaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Millar, N. S.</span><span> </span><span class="NLM_article-title">Allosteric modulation of nicotinic acetylcholine receptors</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span><span class="refDoi"> DOI: 10.1016/j.bcp.2015.07.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.bcp.2015.07.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=26231943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1GrsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2015&pages=408-417&author=A.+Chatzidakiauthor=N.+S.+Millar&title=Allosteric+modulation+of+nicotinic+acetylcholine+receptors&doi=10.1016%2Fj.bcp.2015.07.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit220R"><div class="casContent"><span class="casTitleNuber">220</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of nicotinic acetylcholine receptors</span></div><div class="casAuthors">Chatzidaki, Anna; Millar, Neil S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-417</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nicotinic acetylcholine receptors (nAChRs) are receptors for the neurotransmitter acetylcholine and are members of the 'Cys-loop' family of pentameric ligand-gated ion channels (LGICs).  Acetylcholine binds in the receptor extracellular domain at the interface between two subunits and research has identified a large no. of nAChR-selective ligands, including agonists and competitive antagonists, that bind at the same site as acetylcholine (commonly referred to as the orthosteric binding site).  In addn., more recent research has identified ligands that are able to modulate nAChR function by binding to sites that are distinct from the binding site for acetylcholine, including sites located in the transmembrane domain.  These include pos. allosteric modulators (PAMs), neg. allosteric modulators (NAMs), silent allosteric modulators (SAMs) and compds. that are able to activate nAChRs via an allosteric binding site (allosteric agonists).  Our aim in this article is to review important aspects of the pharmacol. diversity of nAChR allosteric modulators and to describe recent evidence aimed at identifying binding sites for allosteric modulators on nAChRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomoO8Pu1gYwbVg90H21EOLACvtfcHk0li4xBwlNuv4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1GrsL3F&md5=3c321adc918921f2ac7db2a56fe20af8</span></div><a href="/servlet/linkout?suffix=cit220&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.07.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.07.028%26sid%3Dliteratum%253Aachs%26aulast%3DChatzidaki%26aufirst%3DA.%26aulast%3DMillar%26aufirst%3DN.%2BS.%26atitle%3DAllosteric%2520modulation%2520of%2520nicotinic%2520acetylcholine%2520receptors%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2015%26volume%3D97%26spage%3D408%26epage%3D417%26doi%3D10.1016%2Fj.bcp.2015.07.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref221"><div class="reference"><strong class="refLabel"><a href="#ref221" class="rightTabRefNumLink">221</a></strong><div class="NLM_citation" id="rightTab-cit221"><span><span class="NLM_contrib-group">Desdouits, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blondel, A.</span><span> </span><span class="NLM_article-title">Principal component analysis reveals correlation of cavities evolution and functional motions in proteins</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2014.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2Fj.jmgm.2014.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=25424655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=13-24&author=N.+Desdouitsauthor=M.+Nilgesauthor=A.+Blondel&title=Principal+component+analysis+reveals+correlation+of+cavities+evolution+and+functional+motions+in+proteins&doi=10.1016%2Fj.jmgm.2014.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit221R"><div class="casContent"><span class="casTitleNuber">221</span><div class="casTitle"><span class="NLM_cas:atitle">Principal Component Analysis reveals correlation of cavities evolution and functional motions in proteins</span></div><div class="casAuthors">Desdouits, Nathan; Nilges, Michael; Blondel, Arnaud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-24</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein conformation has been recognized as the key feature detg. biol. function, as it dets. the position of the essential groups specifically interacting with substrates.  Hence, the shape of the cavities or grooves at the protein surface appears to drive those functions.  However, only a few studies describe the geometrical evolution of protein cavities during mol. dynamics simulations (MD), usually with a crude representation.  To unveil the dynamics of cavity geometry evolution, we developed an approach combining cavity detection and Principal Component Anal. (PCA).  This approach was applied to four systems subjected to MD (lysozyme, sperm whale myoglobin, Dengue envelope protein and EF-CaM complex).  PCA on cavities allows us to perform efficient anal. and classification of the geometry diversity explored by a cavity.  Addnl., it reveals correlations between the evolutions of the cavities and structures, and can even suggest how to modify the protein conformation to induce a given cavity geometry.  It also helps to perform fast and consensual clustering of conformations according to cavity geometry.  Finally, using this approach, we show that both carbon monoxide (CO) location and transfer among the different xenon sites of myoglobin are correlated with few cavity evolution modes of high amplitude.  This correlation illustrates the link between ligand diffusion and the dynamic network of internal cavities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsuVyGLV0O7Vg90H21EOLACvtfcHk0li4xBwlNuv4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvE&md5=d7634d896c02f34c2e7b937e4d6d1607</span></div><a href="/servlet/linkout?suffix=cit221&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2014.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2014.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DDesdouits%26aufirst%3DN.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DBlondel%26aufirst%3DA.%26atitle%3DPrincipal%2520component%2520analysis%2520reveals%2520correlation%2520of%2520cavities%2520evolution%2520and%2520functional%2520motions%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D55%26spage%3D13%26epage%3D24%26doi%3D10.1016%2Fj.jmgm.2014.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref222"><div class="reference"><strong class="refLabel"><a href="#ref222" class="rightTabRefNumLink">222</a></strong><div class="NLM_citation" id="rightTab-cit222"><span><span class="NLM_contrib-group">Kokh, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czodrowski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R. C.</span><span> </span><span class="NLM_article-title">TRAPP: a tool for analysis of transient binding pockets in proteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1235</span><span class="NLM_x">–</span> <span class="NLM_lpage">1252</span><span class="refDoi"> DOI: 10.1021/ci4000294</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4000294" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1eitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1235-1252&author=D.+B.+Kokhauthor=S.+Richterauthor=S.+Henrichauthor=P.+Czodrowskiauthor=F.+Rippmannauthor=R.+C.+Wade&title=TRAPP%3A+a+tool+for+analysis+of+transient+binding+pockets+in+proteins&doi=10.1021%2Fci4000294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit222R"><div class="casContent"><span class="casTitleNuber">222</span><div class="casTitle"><span class="NLM_cas:atitle">TRAPP: A tool for analysis of transient binding pockets in proteins</span></div><div class="casAuthors">Kokh, Daria B.; Richter, Stefan; Henrich, Stefan; Czodrowski, Paul; Rippmann, Friedrich; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1235-1252</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors present TRAPP (TRAnsient Pockets in Proteins), a new automated software platform for tracking, anal., and visualization of binding pocket variations along a protein motion trajectory or within an ensemble of protein structures that may encompass conformational changes ranging from local side chain fluctuations to global backbone motions.  TRAPP performs accurate grid-based calcns. of the shape and physicochem. characteristics of a binding pocket for each structure and detects the conserved and transient regions of the pocket in an ensemble of protein conformations.  It also provides tools for tracing the opening of a particular subpocket and residues that contribute to the binding site.  TRAPP thus enables an assessment of the drugability of a disease-related target protein taking its flexibility into account.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3HpnnZZfvd7Vg90H21EOLACvtfcHk0li4xBwlNuv4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1eitb8%253D&md5=ad45c4a383da31f88a4d0a6844f76391</span></div><a href="/servlet/linkout?suffix=cit222&amp;dbid=16384&amp;doi=10.1021%2Fci4000294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4000294%26sid%3Dliteratum%253Aachs%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DRichter%26aufirst%3DS.%26aulast%3DHenrich%26aufirst%3DS.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DTRAPP%253A%2520a%2520tool%2520for%2520analysis%2520of%2520transient%2520binding%2520pockets%2520in%2520proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1235%26epage%3D1252%26doi%3D10.1021%2Fci4000294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref223"><div class="reference"><strong class="refLabel"><a href="#ref223" class="rightTabRefNumLink">223</a></strong><div class="NLM_citation" id="rightTab-cit223"><span><span class="NLM_contrib-group">Kappel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wereszczynski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clubb, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The binding mechanism, multiple binding modes, and allosteric regulation of staphylococcus aureus sortase a probed by molecular dynamics simulations</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1858</span><span class="NLM_x">–</span> <span class="NLM_lpage">1871</span><span class="refDoi"> DOI: 10.1002/pro.2168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fpro.2168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23023444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslelsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=1858-1871&author=K.+Kappelauthor=J.+Wereszczynskiauthor=R.+T.+Clubbauthor=J.+A.+McCammon&title=The+binding+mechanism%2C+multiple+binding+modes%2C+and+allosteric+regulation+of+staphylococcus+aureus+sortase+a+probed+by+molecular+dynamics+simulations&doi=10.1002%2Fpro.2168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit223R"><div class="casContent"><span class="casTitleNuber">223</span><div class="casTitle"><span class="NLM_cas:atitle">The binding mechanism, multiple binding modes, and allosteric regulation of Staphylococcus aureus Sortase A probed by molecular dynamics simulations</span></div><div class="casAuthors">Kappel, Kalli; Wereszczynski, Jeff; Clubb, Robert T.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1858-1871</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Sortase enzymes are vitally important for the virulence of gram-pos. bacteria as they play a key role in the attachment of surface proteins to the cell wall.  These enzymes recognize a specific sorting sequence in proteins destined to be displayed on the surface of the bacteria and catalyze the transpeptidation reaction that links it to a cell wall precursor mol.  Because of their role in establishing pathogenicity, and in light of the recent rise of antibiotic-resistant bacterial strains, sortase enzymes are novel drug targets.  Here, we present a study of the prototypical sortase protein Staphylococcus aureus Sortase A (SrtA).  Both conventional and accelerated mol. dynamics simulations of S. aureus SrtA in its apo state and when bound to an LPATG sorting signal (SS) were performed.  Results support a binding mechanism that may be characterized as conformational selection followed by induced fit.  Addnl., the SS was found to adopt multiple metastable states, thus resolving discrepancies between binding conformations in previously reported exptl. structures.  Finally, correlation anal. reveals that the SS actively affects allosteric pathways throughout the protein that connect the first and the second substrate binding sites, which are proposed to be located on opposing faces of the protein.  Overall, these calcns. shed new light on the role of dynamics in the binding mechanism and function of sortase enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhL1c8Gb4WsLVg90H21EOLACvtfcHk0li4xBwlNuv4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslelsr3F&md5=3c0785e3dfef30dda49be2798282e951</span></div><a href="/servlet/linkout?suffix=cit223&amp;dbid=16384&amp;doi=10.1002%2Fpro.2168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2168%26sid%3Dliteratum%253Aachs%26aulast%3DKappel%26aufirst%3DK.%26aulast%3DWereszczynski%26aufirst%3DJ.%26aulast%3DClubb%26aufirst%3DR.%2BT.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520binding%2520mechanism%252C%2520multiple%2520binding%2520modes%252C%2520and%2520allosteric%2520regulation%2520of%2520staphylococcus%2520aureus%2520sortase%2520a%2520probed%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProtein%2520Sci.%26date%3D2012%26volume%3D21%26spage%3D1858%26epage%3D1871%26doi%3D10.1002%2Fpro.2168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref224"><div class="reference"><strong class="refLabel"><a href="#ref224" class="rightTabRefNumLink">224</a></strong><div class="NLM_citation" id="rightTab-cit224"><span><span class="NLM_contrib-group">Amadei, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linssen, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">Essential dynamics of proteins</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">425</span><span class="refDoi"> DOI: 10.1002/prot.340170408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.340170408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=8108382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2cXis1Ghsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1993&pages=412-425&author=A.+Amadeiauthor=A.+B.+Linssenauthor=H.+J.+Berendsen&title=Essential+dynamics+of+proteins&doi=10.1002%2Fprot.340170408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit224R"><div class="casContent"><span class="casTitleNuber">224</span><div class="casTitle"><span class="NLM_cas:atitle">Essential dynamics of proteins</span></div><div class="casAuthors">Amadei, Andrea; Linssen, Antonius B.M.; Berendsen, Herman J.C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">412-25</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">Anal. of extended mol. dynamics (MD) simulations of lysozyme in vacuo and in aq. soln. reveals that it is possible to sep. the configurational space into two subspaces: (1) an "essential" subspace contg. only a few degrees of freedom in which anharmonic motion occurs that comprises most of the positional fluctuations and (2) the remaining space in which the motion has a narrow Gaussian distribution and which can be considered as "phys. constrained.".  If overall translation and rotation are eliminated, the two spaces can be constructed by a simple linear transformation in Cartesian coordinate space, which remains valid over several hundred picoseconds.  The transformation follows from the covariance matrix of the positional deviations.  The essential degrees of freedom seem to describe motions which are relevant for the function of the protein, while the phys. constrained subspace merely describes irrelevant local fluctuations.  The near-constraint behavior of the latter subspace allows the sepn. of equations of motion and promises the possibility of investigating independently the essential space and performing dynamic simulations only in this reduced space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo69NnvqF5j17Vg90H21EOLACvtfcHk0lhz4ooV3_dOXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXis1Ghsbc%253D&md5=e99ea5db083d315c00c85c5fe81d87f7</span></div><a href="/servlet/linkout?suffix=cit224&amp;dbid=16384&amp;doi=10.1002%2Fprot.340170408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340170408%26sid%3Dliteratum%253Aachs%26aulast%3DAmadei%26aufirst%3DA.%26aulast%3DLinssen%26aufirst%3DA.%2BB.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DEssential%2520dynamics%2520of%2520proteins%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1993%26volume%3D17%26spage%3D412%26epage%3D425%26doi%3D10.1002%2Fprot.340170408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref225"><div class="reference"><strong class="refLabel"><a href="#ref225" class="rightTabRefNumLink">225</a></strong><div class="NLM_citation" id="rightTab-cit225"><span><span class="NLM_contrib-group">Lange, O. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Generalized correlation for biomolecular dynamics</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">1053</span><span class="NLM_x">–</span> <span class="NLM_lpage">1061</span><span class="refDoi"> DOI: 10.1002/prot.20784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.20784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=16355416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVWnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=1053-1061&author=O.+F.+Langeauthor=H.+Grubmuller&title=Generalized+correlation+for+biomolecular+dynamics&doi=10.1002%2Fprot.20784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit225R"><div class="casContent"><span class="casTitleNuber">225</span><div class="casTitle"><span class="NLM_cas:atitle">Generalized correlation for biomolecular dynamics</span></div><div class="casAuthors">Lange, Oliver F.; Grubmueller, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1053-1061</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Correlated motions in biomols. are often essential for their function, e.g., allosteric signal transduction or mech./thermodn. energy transport.  Because correlated motions in biomols. remain difficult to access exptl., mol. dynamics (MD) simulations are particular useful for their anal.  The established method to quantify correlations from MD simulations via calcn. of the covariance matrix, however, is restricted to linear correlations and therefore misses part of the correlations in the at. fluctuations.  Herein, the authors propose a general statistical mechanics approach to detect and quantify any correlated motion from MD trajectories.  This generalized correlation measure is contrasted with correlations obtained from covariance matrixes for the B1 domain of protein G and T4 lysozyme.  The new method successfully quantifies correlations and provides a valuable global overview over the functionally relevant collective motions of lysozyme.  In particular, correlated motions of helix 1 together with the two main lobes of lysozyme are detected, which are not seen by the conventional covariance matrix.  Overall, the established method misses more than 50% of the correlation.  This failure is attributed to both, an interfering and unnecessary dependence on mutual orientations of the at. fluctuations and, to a lesser extent, attributed to nonlinear correlations.  The authors' generalized correlation measure overcomes these problems and, moreover, allows for an improved understanding of the conformational dynamics by sepg. linear and nonlinear contributions of the correlation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFG8QqiRvT97Vg90H21EOLACvtfcHk0lhz4ooV3_dOXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVWnsr4%253D&md5=c5f5cb619a4fca1649328371ea8fe9ec</span></div><a href="/servlet/linkout?suffix=cit225&amp;dbid=16384&amp;doi=10.1002%2Fprot.20784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20784%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DO.%2BF.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DGeneralized%2520correlation%2520for%2520biomolecular%2520dynamics%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D62%26spage%3D1053%26epage%3D1061%26doi%3D10.1002%2Fprot.20784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref226"><div class="reference"><strong class="refLabel"><a href="#ref226" class="rightTabRefNumLink">226</a></strong><div class="NLM_citation" id="rightTab-cit226"><span><span class="NLM_contrib-group">Lange, O. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grubmuller, H.</span><span> </span><span class="NLM_article-title">Full correlation analysis of conformational protein dynamics</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">1294</span><span class="NLM_x">–</span> <span class="NLM_lpage">1312</span><span class="refDoi"> DOI: 10.1002/prot.21618</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.21618" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2008&pages=1294-1312&author=O.+F.+Langeauthor=H.+Grubmuller&title=Full+correlation+analysis+of+conformational+protein+dynamics&doi=10.1002%2Fprot.21618"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit226&amp;dbid=16384&amp;doi=10.1002%2Fprot.21618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21618%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DO.%2BF.%26aulast%3DGrubmuller%26aufirst%3DH.%26atitle%3DFull%2520correlation%2520analysis%2520of%2520conformational%2520protein%2520dynamics%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D70%26spage%3D1294%26epage%3D1312%26doi%3D10.1002%2Fprot.21618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref227"><div class="reference"><strong class="refLabel"><a href="#ref227" class="rightTabRefNumLink">227</a></strong><div class="NLM_citation" id="rightTab-cit227"><span><span class="NLM_contrib-group">McClendon, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amirkhani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Quantifying correlations between allosteric sites in thermodynamic ensembles</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2486</span><span class="NLM_x">–</span> <span class="NLM_lpage">2502</span><span class="refDoi"> DOI: 10.1021/ct9001812</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct9001812" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpslahsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=2486-2502&author=C.+L.+McClendonauthor=G.+Friedlandauthor=D.+L.+Mobleyauthor=H.+Amirkhaniauthor=M.+P.+Jacobson&title=Quantifying+correlations+between+allosteric+sites+in+thermodynamic+ensembles&doi=10.1021%2Fct9001812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit227R"><div class="casContent"><span class="casTitleNuber">227</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Correlations Between Allosteric Sites in Thermodynamic Ensembles</span></div><div class="casAuthors">McClendon, Christopher L.; Friedland, Gregory; Mobley, David L.; Amirkhani, Homeira; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2486-2502</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery describes altered protein function at one site due to a perturbation at another site.  One mechanism of allostery involves correlated motions, which can occur even in the absence of substantial conformational change.  We present a novel method, "MutInf", to identify statistically significant correlated motions from equil. mol. dynamics simulations.  Our approach analyzes both backbone and side chain motions using internal coordinates to account for the gear-like twists that can take place even in the absence of the large conformational changes typical of traditional allosteric proteins.  We quantify correlated motions using a mutual information metric, which we extend to incorporate data from multiple short simulations and to filter out correlations that are not statistically significant.  Applying our approach to uncover mechanisms of cooperative small mol. binding in human interleukin-2, we identify clusters of correlated residues from 50 ns of mol. dynamics simulations.  Interestingly, two of the clusters with the strongest correlations highlight known cooperative small-mol. binding sites and show substantial correlations between these sites.  These cooperative binding sites on interleukin-2 are correlated not only through the hydrophobic core of the protein but also through a dynamic polar network of hydrogen bonding and electrostatic interactions.  Since this approach identifies correlated conformations in an unbiased, statistically robust manner, it should be a useful tool for finding novel or "orphan" allosteric sites in proteins of biol. and therapeutic importance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9xO0p5xauKrVg90H21EOLACvtfcHk0lhz4ooV3_dOXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpslahsrc%253D&md5=158a76d26856825d4f19c8515f40ffc4</span></div><a href="/servlet/linkout?suffix=cit227&amp;dbid=16384&amp;doi=10.1021%2Fct9001812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct9001812%26sid%3Dliteratum%253Aachs%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26aulast%3DFriedland%26aufirst%3DG.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DAmirkhani%26aufirst%3DH.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DQuantifying%2520correlations%2520between%2520allosteric%2520sites%2520in%2520thermodynamic%2520ensembles%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D2486%26epage%3D2502%26doi%3D10.1021%2Fct9001812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref228"><div class="reference"><strong class="refLabel"><a href="#ref228" class="rightTabRefNumLink">228</a></strong><div class="NLM_citation" id="rightTab-cit228"><span><span class="NLM_contrib-group">Sethi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eargle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthey-Schulten, Z.</span><span> </span><span class="NLM_article-title">Dynamical networks in tRNA: protein complexes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6620</span><span class="NLM_x">–</span> <span class="NLM_lpage">6625</span><span class="refDoi"> DOI: 10.1073/pnas.0810961106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.0810961106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19351898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsVOlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6620-6625&author=A.+Sethiauthor=J.+Eargleauthor=A.+A.+Blackauthor=Z.+Luthey-Schulten&title=Dynamical+networks+in+tRNA%3A+protein+complexes&doi=10.1073%2Fpnas.0810961106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit228R"><div class="casContent"><span class="casTitleNuber">228</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamical networks in tRNA:protein complexes</span></div><div class="casAuthors">Sethi, Anurag; Eargle, John; Black, Alexis A.; Luthey-Schulten, Zaida</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6620-6625</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Community network anal. derived from mol. dynamics simulations is used to identify and compare the signaling pathways in a bacterial glutamyl-tRNA synthetase (GluRS):tRNAGlu and an archaeal leucyl-tRNA synthetase (LeuRS):tRNALeu complex.  Although the 2 class I synthetases have remarkably different interactions with their cognate tRNAs, the allosteric networks for charging tRNA with the correct amino acid display considerable similarities.  A dynamic contact map defines the edges connecting nodes (amino acids and nucleotides) in the phys. network whose overall topol. is presented as a network of communities, local substructures that are highly intraconnected, but loosely interconnected.  Whereas nodes within a single community can communicate through many alternate pathways, the communication between monomers in different communities has to take place through a smaller no. of crit. edges or interactions.  Consistent with this anal., there are a large no. of suboptimal paths that can be used for communication between the identity elements on the tRNAs and the catalytic site in the aaRS:tRNA complexes.  Residues and nucleotides in the majority of pathways for intercommunity signal transmission are evolutionarily conserved and are predicted to be important for allosteric signaling.  The same monomers are also found in a majority of the suboptimal paths.  Modifying these residues or nucleotides has a large effect on the communication pathways in the protein:RNA complex consistent with kinetic data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy2zkXI--Lx7Vg90H21EOLACvtfcHk0lj5gElc8fr_7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsVOlu7o%253D&md5=5e3cfd80bf85ffe46c9c84960d176713</span></div><a href="/servlet/linkout?suffix=cit228&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0810961106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0810961106%26sid%3Dliteratum%253Aachs%26aulast%3DSethi%26aufirst%3DA.%26aulast%3DEargle%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DA.%2BA.%26aulast%3DLuthey-Schulten%26aufirst%3DZ.%26atitle%3DDynamical%2520networks%2520in%2520tRNA%253A%2520protein%2520complexes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6620%26epage%3D6625%26doi%3D10.1073%2Fpnas.0810961106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref229"><div class="reference"><strong class="refLabel"><a href="#ref229" class="rightTabRefNumLink">229</a></strong><div class="NLM_citation" id="rightTab-cit229"><span><span class="NLM_contrib-group">Newman, M. E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girvan, M.</span><span> </span><span class="NLM_article-title">Finding and evaluating community structure in networks</span> <span class="citation_source-journal">Phys. Rev. E</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">026113</span><span class="refDoi"> DOI: 10.1103/PhysRevE.69.026113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref229/cit229&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1103%2FPhysRevE.69.026113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2004&pages=026113&author=M.+E.+J.+Newmanauthor=M.+Girvan&title=Finding+and+evaluating+community+structure+in+networks&doi=10.1103%2FPhysRevE.69.026113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit229&amp;dbid=16384&amp;doi=10.1103%2FPhysRevE.69.026113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevE.69.026113%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DM.%2BE.%2BJ.%26aulast%3DGirvan%26aufirst%3DM.%26atitle%3DFinding%2520and%2520evaluating%2520community%2520structure%2520in%2520networks%26jtitle%3DPhys.%2520Rev.%2520E%26date%3D2004%26volume%3D69%26spage%3D026113%26doi%3D10.1103%2FPhysRevE.69.026113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref230"><div class="reference"><strong class="refLabel"><a href="#ref230" class="rightTabRefNumLink">230</a></strong><div class="NLM_citation" id="rightTab-cit230"><span><span class="NLM_contrib-group">Rivalta, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sultan, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loria, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batista, V. S.</span><span> </span><span class="NLM_article-title">Allosteric pathways in imidazole glycerol phosphate synthase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">E1428</span><span class="NLM_x">–</span> <span class="NLM_lpage">E1436</span><span class="refDoi"> DOI: 10.1073/pnas.1120536109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1073%2Fpnas.1120536109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22586084" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=E1428-E1436&author=I.+Rivaltaauthor=M.+M.+Sultanauthor=N.+S.+Leeauthor=G.+A.+Manleyauthor=J.+P.+Loriaauthor=V.+S.+Batista&title=Allosteric+pathways+in+imidazole+glycerol+phosphate+synthase&doi=10.1073%2Fpnas.1120536109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit230&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120536109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120536109%26sid%3Dliteratum%253Aachs%26aulast%3DRivalta%26aufirst%3DI.%26aulast%3DSultan%26aufirst%3DM.%2BM.%26aulast%3DLee%26aufirst%3DN.%2BS.%26aulast%3DManley%26aufirst%3DG.%2BA.%26aulast%3DLoria%26aufirst%3DJ.%2BP.%26aulast%3DBatista%26aufirst%3DV.%2BS.%26atitle%3DAllosteric%2520pathways%2520in%2520imidazole%2520glycerol%2520phosphate%2520synthase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3DE1428%26epage%3DE1436%26doi%3D10.1073%2Fpnas.1120536109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref231"><div class="reference"><strong class="refLabel"><a href="#ref231" class="rightTabRefNumLink">231</a></strong><div class="NLM_citation" id="rightTab-cit231"><span><span class="NLM_contrib-group">Morra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genoni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span> </span><span class="NLM_article-title">Mechanisms of differential allosteric modulation in homologous proteins: insights from the analysis of internal dynamics and energetics of pdz domains</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">5677</span><span class="NLM_x">–</span> <span class="NLM_lpage">5689</span><span class="refDoi"> DOI: 10.1021/ct500326g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500326g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=5677-5689&author=G.+Morraauthor=A.+Genoniauthor=G.+Colombo&title=Mechanisms+of+differential+allosteric+modulation+in+homologous+proteins%3A+insights+from+the+analysis+of+internal+dynamics+and+energetics+of+pdz+domains&doi=10.1021%2Fct500326g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit231R"><div class="casContent"><span class="casTitleNuber">231</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Differential Allosteric Modulation in Homologous Proteins: Insights from the Analysis of Internal Dynamics and Energetics of PDZ Domains</span></div><div class="casAuthors">Morra, Giulia; Genoni, Alessandro; Colombo, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5677-5689</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is a general phenomenon in proteins whereby a perturbation at one site reverberates into a functional change at another one, through modulation of its conformational dynamics.  Herein, we address the problem of how the mol. signal encoded by a ligand is differentially transmitted through the structures of two homologous PDZ proteins: PDZ2, which responds to binding with structural and dynamical changes in regions distal from the ligand site, and PDZ3, which is characterized by less-intense dynamical variations.  We use novel methods of anal. of MD simulations in the unbound and bound states to investigate the determinants of the differential allosteric behavior of the two proteins.  The anal. of the correlations between the redistribution of stabilization energy and local fluctuation patterns highlights the nucleus of residues responsible for the stabilization of the 3D fold, the stability core, as the substructure that defines the difference in the allosteric response: in PDZ2, it undergoes a consistent dynamic and energetic reorganization, whereas in PDZ3, it remains largely unperturbed.  Specifically, we observe for PDZ2 a significant anticorrelation between the motions of distal loops and residues of the stability core and differences in the correlation patterns between the bound and unbound states.  Such variation is not obsd. in PDZ3, indicating that its energetics and internal dynamics are less affected by the presence/absence of the ligand.  Finally, we propose a model with a direct link between the modulation of the structural, energetic and dynamic properties of a protein, and its allosteric response to a perturbation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE0MzwLruY77Vg90H21EOLACvtfcHk0lj5gElc8fr_7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsrnF&md5=8908c2566d34fbcbb5863714f4d07313</span></div><a href="/servlet/linkout?suffix=cit231&amp;dbid=16384&amp;doi=10.1021%2Fct500326g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500326g%26sid%3Dliteratum%253Aachs%26aulast%3DMorra%26aufirst%3DG.%26aulast%3DGenoni%26aufirst%3DA.%26aulast%3DColombo%26aufirst%3DG.%26atitle%3DMechanisms%2520of%2520differential%2520allosteric%2520modulation%2520in%2520homologous%2520proteins%253A%2520insights%2520from%2520the%2520analysis%2520of%2520internal%2520dynamics%2520and%2520energetics%2520of%2520pdz%2520domains%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D5677%26epage%3D5689%26doi%3D10.1021%2Fct500326g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref232"><div class="reference"><strong class="refLabel"><a href="#ref232" class="rightTabRefNumLink">232</a></strong><div class="NLM_citation" id="rightTab-cit232"><span><span class="NLM_contrib-group">Morra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neves, M. A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plescia, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsustsumi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neckers, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verkhivker, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altieri, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span> </span><span class="NLM_article-title">Dynamics-based discovery of allosteric inhibitors: selection of new ligands for the c-terminal domain of hsp90</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2978</span><span class="NLM_x">–</span> <span class="NLM_lpage">2989</span><span class="refDoi"> DOI: 10.1021/ct100334n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100334n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVyrsbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2978-2989&author=G.+Morraauthor=M.+A.+C.+Nevesauthor=C.+J.+Plesciaauthor=S.+Tsustsumiauthor=L.+Neckersauthor=G.+Verkhivkerauthor=D.+C.+Altieriauthor=G.+Colombo&title=Dynamics-based+discovery+of+allosteric+inhibitors%3A+selection+of+new+ligands+for+the+c-terminal+domain+of+hsp90&doi=10.1021%2Fct100334n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit232R"><div class="casContent"><span class="casTitleNuber">232</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics-Based Discovery of Allosteric Inhibitors: Selection of New Ligands for the C-terminal Domain of Hsp90</span></div><div class="casAuthors">Morra, Giulia; Neves, Marco A. C.; Plescia, Christopher J.; Tsustsumi, Shinji; Neckers, Len; Verkhivker, Gennady; Altieri, Dario C.; Colombo, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2978-2989</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The study of allosteric functional modulation in dynamic proteins is attracting increasing attention.  In particular, the discovery of new allosteric sites may generate novel opportunities and strategies for drug development, overcoming the limits of classical active-site oriented drug design.  In this paper, we report on the results of a novel, ab initio, fully computational approach for the discovery of allosteric inhibitors based on the phys. characterization of signal propagation mechanisms in proteins and apply it to the important mol. chaperone Hsp90.  We first characterize the allosteric "hot spots" involved in interdomain communication pathways from the nucleotide-binding site in the N-domain to the distal C-domain.  On this basis, we develop dynamic pharmacophore models to screen drug libraries in the search for small mols. with the functional and conformational properties necessary to bind these "hot spot" allosteric sites.  Exptl. tests show that the selected mols. bind the Hsp90 C-domain, exhibit antiproliferative activity in different tumor cell lines, while not affecting proliferation of normal human cells, destabilize Hsp90 client proteins, and disrupt assocn. with several cochaperones known to bind the N- and M-domains of Hsp90.  These results prove that the hits alter Hsp90 function by affecting its conformational dynamics and recognition properties through an allosteric mechanism.  These findings provide us with new insights on the discovery and development of new allosteric inhibitors that are active on important cellular pathways through computational biol.  Though based on the specific case of Hsp90, our approach is general and can readily be extended to other target proteins and pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtIOnRuY95mrVg90H21EOLACvtfcHk0lhE0GnIS32njA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVyrsbfE&md5=2eef3fded78c9607e9e707e091a6716b</span></div><a href="/servlet/linkout?suffix=cit232&amp;dbid=16384&amp;doi=10.1021%2Fct100334n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100334n%26sid%3Dliteratum%253Aachs%26aulast%3DMorra%26aufirst%3DG.%26aulast%3DNeves%26aufirst%3DM.%2BA.%2BC.%26aulast%3DPlescia%26aufirst%3DC.%2BJ.%26aulast%3DTsustsumi%26aufirst%3DS.%26aulast%3DNeckers%26aufirst%3DL.%26aulast%3DVerkhivker%26aufirst%3DG.%26aulast%3DAltieri%26aufirst%3DD.%2BC.%26aulast%3DColombo%26aufirst%3DG.%26atitle%3DDynamics-based%2520discovery%2520of%2520allosteric%2520inhibitors%253A%2520selection%2520of%2520new%2520ligands%2520for%2520the%2520c-terminal%2520domain%2520of%2520hsp90%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2978%26epage%3D2989%26doi%3D10.1021%2Fct100334n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref233"><div class="reference"><strong class="refLabel"><a href="#ref233" class="rightTabRefNumLink">233</a></strong><div class="NLM_citation" id="rightTab-cit233"><span><span class="NLM_contrib-group">de Beer, S. B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">The role of water molecules in computational drug design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.2174/156802610790232288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.2174%2F156802610790232288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=19929830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=55-66&author=S.+B.+A.+de+Beerauthor=N.+P.+E.+Vermeulenauthor=C.+Oostenbrink&title=The+role+of+water+molecules+in+computational+drug+design&doi=10.2174%2F156802610790232288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit233R"><div class="casContent"><span class="casTitleNuber">233</span><div class="casTitle"><span class="NLM_cas:atitle">The role of water molecules in computational drug design</span></div><div class="casAuthors">de Beer, Stephanie B. A.; Vermeulen, Nico P. E.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-66</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Although water mols. are small and only consist of two different atom types, they play various roles in cellular systems.  This review discusses their influence on the binding process between biomacromol. targets and small mol. ligands and how this influence can be modeled in computational drug design approaches.  Both the structure and the thermodn. of active site waters will be discussed as these influence the binding process significantly.  Structurally conserved waters cannot always be detd. exptl. and if obsd., it is not clear if they will be replaced upon ligand binding, even if sufficient space is available.  Methods to predict the presence of water in protein-ligand complexes will be reviewed.  Subsequently, we will discuss methods to include water in computational drug research.  Either as an addnl. factor in automated docking expts., or explicitly in detailed mol. dynamics simulations, the effect of water on the quality of the simulations is significant, but not easily predicted.  The most detailed calcns. involve ests. of the free energy contribution of water mols. to protein-ligand complexes.  These calcns. are computationally demanding, but give insight in the versatility and importance of water in ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhA2h-JAqzhbVg90H21EOLACvtfcHk0lhE0GnIS32njA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7g%253D&md5=5767b8a3f82a33845fae3941a1084067</span></div><a href="/servlet/linkout?suffix=cit233&amp;dbid=16384&amp;doi=10.2174%2F156802610790232288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232288%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBeer%26aufirst%3DS.%2BB.%2BA.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%2BE.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DThe%2520role%2520of%2520water%2520molecules%2520in%2520computational%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D55%26epage%3D66%26doi%3D10.2174%2F156802610790232288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref234"><div class="reference"><strong class="refLabel"><a href="#ref234" class="rightTabRefNumLink">234</a></strong><div class="NLM_citation" id="rightTab-cit234"><span><span class="NLM_contrib-group">Athri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, W. D.</span><span> </span><span class="NLM_article-title">Molecular dynamics of water-mediated interactions of a linear benzimidazole-biphenyl diamidine with the dna minor groove</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">7618</span><span class="NLM_x">–</span> <span class="NLM_lpage">7625</span><span class="refDoi"> DOI: 10.1021/ja809249h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja809249h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=7618-7625&author=P.+Athriauthor=W.+D.+Wilson&title=Molecular+dynamics+of+water-mediated+interactions+of+a+linear+benzimidazole-biphenyl+diamidine+with+the+dna+minor+groove&doi=10.1021%2Fja809249h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit234&amp;dbid=16384&amp;doi=10.1021%2Fja809249h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja809249h%26sid%3Dliteratum%253Aachs%26aulast%3DAthri%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DMolecular%2520dynamics%2520of%2520water-mediated%2520interactions%2520of%2520a%2520linear%2520benzimidazole-biphenyl%2520diamidine%2520with%2520the%2520dna%2520minor%2520groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D7618%26epage%3D7625%26doi%3D10.1021%2Fja809249h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref235"><div class="reference"><strong class="refLabel"><a href="#ref235" class="rightTabRefNumLink">235</a></strong><div class="NLM_citation" id="rightTab-cit235"><span><span class="NLM_contrib-group">Biela, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Water Makes the Difference: Rearrangement of Water Solvation Layer Triggers Non-additivity of functional group contributions in protein-ligand binding</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1423</span><span class="NLM_x">–</span> <span class="NLM_lpage">1434</span><span class="refDoi"> DOI: 10.1002/cmdc.201200206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fcmdc.201200206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=22733601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVajsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1423-1434&author=A.+Bielaauthor=M.+Betzauthor=A.+Heineauthor=G.+Klebe&title=Water+Makes+the+Difference%3A+Rearrangement+of+Water+Solvation+Layer+Triggers+Non-additivity+of+functional+group+contributions+in+protein-ligand+binding&doi=10.1002%2Fcmdc.201200206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit235R"><div class="casContent"><span class="casTitleNuber">235</span><div class="casTitle"><span class="NLM_cas:atitle">Water Makes the Difference: Rearrangement of Water Solvation Layer Triggers Non-additivity of Functional Group Contributions in Protein-Ligand Binding</span></div><div class="casAuthors">Biela, Adam; Betz, Michael; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1423-1434, S1423/1-S1423/2</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The binding of four congeneric peptide-like thermolysin inhibitors has been studied by high-resoln. crystal structure anal. and isothermal titrn. calorimetry.  The ligands differ only by a terminal carboxylate and/or Me group.  A surprising non-additivity of functional group contributions for the carboxylate and/or Me groups is detected.  Adding the Me first and then the carboxylate group results in a small Gibbs free energy increase and minor enthalpy/entropy partitioning for the first modification, whereas the second involves a strong affinity increase combined with large enthalpy/entropy changes.  However, first adding the carboxylate and then the Me group yields reverse effects: the acidic group attachment now causes minor effects, whereas the added Me group provokes large changes.  As all crystal structures show virtually identical binding modes, affinity changes are related to rearrangements of the first solvation layer next to the S2' pocket.  About 20-25 water mols. are visible next to the studied complexes.  The added COO- groups perturb the local water network in both carboxylated complexes, and the attached Me groups provide favorable interaction sites for water mols.  Apart from one example, a contiguously connected water network between protein and ligand functional groups is obsd. in all complexes.  In the complex with the carboxylated ligand, which still lacks the terminal Me group, the water network is unfavorably ruptured.  This results in a surprising thermodn. signature showing only a minor affinity increase upon COO- group attachment.  Because the further added Me group provides a favorable interaction site for water, the network can be reestablished, and a strong affinity increase with a large enthalpy/entropy signature is then detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgp9MxUfIJirVg90H21EOLACvtfcHk0lhE0GnIS32njA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVajsrc%253D&md5=1a6eb12403cdf82cae631384230f3d29</span></div><a href="/servlet/linkout?suffix=cit235&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200206%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DWater%2520Makes%2520the%2520Difference%253A%2520Rearrangement%2520of%2520Water%2520Solvation%2520Layer%2520Triggers%2520Non-additivity%2520of%2520functional%2520group%2520contributions%2520in%2520protein-ligand%2520binding%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D1423%26epage%3D1434%26doi%3D10.1002%2Fcmdc.201200206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref236"><div class="reference"><strong class="refLabel"><a href="#ref236" class="rightTabRefNumLink">236</a></strong><div class="NLM_citation" id="rightTab-cit236"><span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Prediction of the water content in protein binding sites</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">13337</span><span class="NLM_x">–</span> <span class="NLM_lpage">13346</span><span class="refDoi"> DOI: 10.1021/jp9047456</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9047456" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref236/cit236&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=13337-13346&author=J.+Michelauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Prediction+of+the+water+content+in+protein+binding+sites&doi=10.1021%2Fjp9047456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit236R"><div class="casContent"><span class="casTitleNuber">236</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of the Water Content in Protein Binding Sites</span></div><div class="casAuthors">Michel, Julien; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13337-13346</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient mol. simulation methodol. has been developed to det. the positioning of water mols. in the binding site of a protein or protein-ligand complex.  Occupancies and abs. binding free energies of water mols. are computed using a statistical thermodn. approach.  The methodol., referred to as Just Add Water Mols. (JAWS), features "θ-water" mols. that can appear and disappear on a binding-site grid.  Key approxns. render the technique far more efficient than conventional free energy simulations.  Testing of JAWS on five diverse examples (neuraminidase, scytalone dehydratase, major urinary protein 1, β-lactoglobulin, and COX-2) demonstrates its accuracy in locating hydration sites in comparison to results from high-resoln. crystal structures.  Possible applications include aid in refinement of protein crystal structures, drug lead optimization, setup of docking calcns., and simulations of protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6L_AGV7e7LVg90H21EOLACvtfcHk0liLGUROfZ5-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I&md5=bc36a74922d2b1ce1d15e53b9e144afe</span></div><a href="/servlet/linkout?suffix=cit236&amp;dbid=16384&amp;doi=10.1021%2Fjp9047456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9047456%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DPrediction%2520of%2520the%2520water%2520content%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D13337%26epage%3D13346%26doi%3D10.1021%2Fjp9047456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref237"><div class="reference"><strong class="refLabel"><a href="#ref237" class="rightTabRefNumLink">237</a></strong><div class="NLM_citation" id="rightTab-cit237"><span><span class="NLM_contrib-group">Henchman, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Extracting hydration sites around proteins from explicit water simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">869</span><span class="refDoi"> DOI: 10.1002/jcc.10074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fjcc.10074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=11984847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVyksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=861-869&author=R.+H.+Henchmanauthor=J.+A.+McCammon&title=Extracting+hydration+sites+around+proteins+from+explicit+water+simulations&doi=10.1002%2Fjcc.10074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit237R"><div class="casContent"><span class="casTitleNuber">237</span><div class="casTitle"><span class="NLM_cas:atitle">Extracting hydration sites around proteins from explicit water simulations</span></div><div class="casAuthors">Henchman, Richard H.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">861-869</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Two new methods are assessed for detg. the location of hydration sites around proteins from computer simulation.  Current methods ext. hydration sites from peaks in the water d. constructed in the protein frame.  However, the dynamic nature of the water mols., the nearby protein residues, and the protein ref. frame as a whole tend to smear out the water d., making it more difficult to resolve sites.  Two techniques are introduced to better resolve the water d.  The first is to construct the water d. from the time-averaged position of each water mol. in the protein frame while the water remains within a given distance of this averaged position.  The second technique is to construct the water d. from the time-averaged position of each water in the ref. frame only of the nearby residues.  Criteria for detg. hydration sites from the water d. are examd.  Both techniques are found to significantly improve the detail in the water d. and the no. of hydration sites detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQkWNcVsa6drVg90H21EOLACvtfcHk0liLGUROfZ5-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksVyksL4%253D&md5=96ab8a8e32f13def3b9668486d5ad562</span></div><a href="/servlet/linkout?suffix=cit237&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10074%26sid%3Dliteratum%253Aachs%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DExtracting%2520hydration%2520sites%2520around%2520proteins%2520from%2520explicit%2520water%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D861%26epage%3D869%26doi%3D10.1002%2Fjcc.10074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref238"><div class="reference"><strong class="refLabel"><a href="#ref238" class="rightTabRefNumLink">238</a></strong><div class="NLM_citation" id="rightTab-cit238"><span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradley, P.</span><span> </span><span class="NLM_article-title">Probing the role of interfacial waters in protein-DNA recognition using a hybrid implicit/explicit solvation model</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">1318</span><span class="NLM_x">–</span> <span class="NLM_lpage">1329</span><span class="refDoi"> DOI: 10.1002/prot.24272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fprot.24272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23444044" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=1318-1329&author=S.+Liauthor=P.+Bradley&title=Probing+the+role+of+interfacial+waters+in+protein-DNA+recognition+using+a+hybrid+implicit%2Fexplicit+solvation+model&doi=10.1002%2Fprot.24272"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit238&amp;dbid=16384&amp;doi=10.1002%2Fprot.24272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24272%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DBradley%26aufirst%3DP.%26atitle%3DProbing%2520the%2520role%2520of%2520interfacial%2520waters%2520in%2520protein-DNA%2520recognition%2520using%2520a%2520hybrid%2520implicit%252Fexplicit%2520solvation%2520model%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2013%26volume%3D81%26spage%3D1318%26epage%3D1329%26doi%3D10.1002%2Fprot.24272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref239"><div class="reference"><strong class="refLabel"><a href="#ref239" class="rightTabRefNumLink">239</a></strong><div class="NLM_citation" id="rightTab-cit239"><span><span class="NLM_contrib-group">Ladbury, J. E.</span><span> </span><span class="NLM_article-title">Just add water! The effect of water on the specificity of protein-ligand binding sites and its potential application to drug design</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">973</span><span class="NLM_x">–</span> <span class="NLM_lpage">980</span><span class="refDoi"> DOI: 10.1016/S1074-5521(96)90164-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1016%2FS1074-5521%2896%2990164-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=9000013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=973-980&author=J.+E.+Ladbury&title=Just+add+water%21+The+effect+of+water+on+the+specificity+of+protein-ligand+binding+sites+and+its+potential+application+to+drug+design&doi=10.1016%2FS1074-5521%2896%2990164-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit239R"><div class="casContent"><span class="casTitleNuber">239</span><div class="casTitle"><span class="NLM_cas:atitle">Just add water! The effect of water on the specificity of protein-ligand binding sites and its potential application to drug design</span></div><div class="casAuthors">Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">973-980</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">A review with 39 refs.  Recent data have highlighted the enigmatic role that water plays in biomol. complexes.  Water at the interface of a complex can increase the promiscuity of an interaction, yet it can also provide increased specificity and affinity.  The ability to engineer water-binding sites into the interface between a drug and its target might prove useful in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJqyhDhRviwbVg90H21EOLACvtfcHk0liLGUROfZ5-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D&md5=8eb3242b84897de4bb61e20ec6bc8d54</span></div><a href="/servlet/linkout?suffix=cit239&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2896%2990164-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252896%252990164-7%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DJust%2520add%2520water%2521%2520The%2520effect%2520of%2520water%2520on%2520the%2520specificity%2520of%2520protein-ligand%2520binding%2520sites%2520and%2520its%2520potential%2520application%2520to%2520drug%2520design%26jtitle%3DChem.%2520Biol.%26date%3D1996%26volume%3D3%26spage%3D973%26epage%3D980%26doi%3D10.1016%2FS1074-5521%2896%2990164-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref240"><div class="reference"><strong class="refLabel"><a href="#ref240" class="rightTabRefNumLink">240</a></strong><div class="NLM_citation" id="rightTab-cit240"><span><span class="NLM_contrib-group">Huang, D. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossini, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Structured water molecules in the binding site of bromodomains can be displaced by cosolvent</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">573</span><span class="NLM_x">–</span> <span class="NLM_lpage">579</span><span class="refDoi"> DOI: 10.1002/cmdc.201300156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=10.1002%2Fcmdc.201300156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=23804246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVSht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=573-579&author=D.+Z.+Huangauthor=E.+Rossiniauthor=S.+Steinerauthor=A.+Caflisch&title=Structured+water+molecules+in+the+binding+site+of+bromodomains+can+be+displaced+by+cosolvent&doi=10.1002%2Fcmdc.201300156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit240R"><div class="casContent"><span class="casTitleNuber">240</span><div class="casTitle"><span class="NLM_cas:atitle">Structured Water Molecules in the Binding Site of Bromodomains Can Be Displaced by Cosolvent</span></div><div class="casAuthors">Huang, Danzhi; Rossini, Emanuele; Steiner, Sandra; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">573-579</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomains are α-helical bundles of approx. 110 residues that recognize acetylated lysine side chains mainly on histone tails.  Bromodomains are known to play an important role in cancer and inflammation, and as such, significant efforts are being made to identify small-mol. inhibitors of these epigenetic reader proteins.  Here, explicit solvent mol. dynamics (MD) simulations of two bromodomains (BAZ2B and CREBBP) are used to analyze the water mols. that seem to be conserved at the bottom of the acetyl-lysine binding site in most crystal structures of bromodomains.  The MD runs suggest that the occupancy of the structured water mols. is influenced by conformational transitions of the loop that connects helixes Z and A.  Addnl. simulations in the presence of 50 mols. of cosolvent (i.e., 440 mM of dimethylsulfoxide, methanol, or ethanol) indicate that some of the structured water mols. can be displaced transiently.  The residence time in the acetyl-lysine binding site is calcd. to be about 1 ns, 2-5 ns, and 10-30 ns for methanol, ethanol, and dimethylsulfoxide, resp., while the affinity of the three cosolvents for BAZ2B and CREBBP is in the range of 50-500 mM.  The results described have implications for ligand design, suggesting that only structured water mols. that do not exchange with cosolvent should be maintained in crystal structures used for docking campaigns, and that hydroxy substituents should be incorporated in the ligand so as to map the structured water mols. replaced by (m)ethanol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoapdiGvm2PUrVg90H21EOLACvtfcHk0liuJslhAGYi8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVSht7fF&md5=6f742480234fc072a61a4a051dfec948</span></div><a href="/servlet/linkout?suffix=cit240&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300156%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%2BZ.%26aulast%3DRossini%26aufirst%3DE.%26aulast%3DSteiner%26aufirst%3DS.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DStructured%2520water%2520molecules%2520in%2520the%2520binding%2520site%2520of%2520bromodomains%2520can%2520be%2520displaced%2520by%2520cosolvent%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D573%26epage%3D579%26doi%3D10.1002%2Fcmdc.201300156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref241"><div class="reference"><strong class="refLabel"><a href="#ref241" class="rightTabRefNumLink">241</a></strong><div class="NLM_citation" id="rightTab-cit241"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span> </span><span class="NLM_article-title">Inhomogeneous fluid approach to solvation thermodynamics. 1. Theory</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3531</span><span class="NLM_x">–</span> <span class="NLM_lpage">3541</span><span class="refDoi"> DOI: 10.1021/jp9723574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9723574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3531-3541&author=T.+Lazaridis&title=Inhomogeneous+fluid+approach+to+solvation+thermodynamics.+1.+Theory&doi=10.1021%2Fjp9723574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit241R"><div class="casContent"><span class="casTitleNuber">241</span><div class="casTitle"><span class="NLM_cas:atitle">Inhomogeneous Fluid Approach to Solvation Thermodynamics. 1. Theory</span></div><div class="casAuthors">Lazaridis, Themis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3531-3541</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The solvation energy and entropy at infinite diln. consist of a solute-solvent term and a solvent reorganization term representing the contributions of changes in solvent structure upon solute insertion.  In the std., homogeneous treatment of solns., changes in solvent structure are expressed through derivs. of the homogeneous pair correlation function, which are very difficult to obtain by simulation.  Tractable expressions for the solvation energy and entropy are here derived by viewing the soln. as an inhomogeneous system with the solute fixed at a certain point.  The solvent reorganization terms in the inhomogeneous view contain two contributions: the local, "correlation" contributions, which are due to correlations between the solute and the solvent and dominate at high densities, and the nonlocal, "liberation" contributions, which are due to the effective diln. of the solvent caused by the thermal motion of the solute and dominate at low densities.  The liberation contributions are independent of the nature or size of the solute and depend only on the properties of the solvent.  For common liq. solvents the nonlocal terms are negligible and the solvation properties arise almost entirely from effects localized around the solute.  The new expressions are tested by calcns. of the solvent reorganization energy and entropy in ideal hard-sphere and Lennard-Jones mixts. (solute identical to solvent).  The solvent reorganization energy and entropy make distinct and significant contributions to the solvation free energy.  The theory can be applied to truly inhomogeneous systems as well as small solute solvation, thus providing a connection between interfacial phenomena and microscopic solvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxEN_nWI8WfbVg90H21EOLACvtfcHk0liuJslhAGYi8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D&md5=99755c87ffc33abd5e7df89c9996a665</span></div><a href="/servlet/linkout?suffix=cit241&amp;dbid=16384&amp;doi=10.1021%2Fjp9723574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9723574%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26atitle%3DInhomogeneous%2520fluid%2520approach%2520to%2520solvation%2520thermodynamics.%25201.%2520Theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3531%26epage%3D3541%26doi%3D10.1021%2Fjp9723574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref242"><div class="reference"><strong class="refLabel"><a href="#ref242" class="rightTabRefNumLink">242</a></strong><div class="NLM_citation" id="rightTab-cit242"><span><span class="NLM_contrib-group">Breiten, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockett, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Sayah, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heroux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesides, G. M.</span><span> </span><span class="NLM_article-title">Water Networks contribute to enthalpy/entropy compensation in protein-ligand binding</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">15579</span><span class="NLM_x">–</span> <span class="NLM_lpage">15584</span><span class="refDoi"> DOI: 10.1021/ja4075776</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4075776" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref242/cit242&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVGjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=15579-15584&author=B.+Breitenauthor=M.+R.+Lockettauthor=W.+Shermanauthor=S.+Fujitaauthor=M.+Al-Sayahauthor=H.+Langeauthor=C.+M.+Bowersauthor=A.+Herouxauthor=G.+Krilovauthor=G.+M.+Whitesides&title=Water+Networks+contribute+to+enthalpy%2Fentropy+compensation+in+protein-ligand+binding&doi=10.1021%2Fja4075776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit242R"><div class="casContent"><span class="casTitleNuber">242</span><div class="casTitle"><span class="NLM_cas:atitle">Water Networks Contribute to Enthalpy/Entropy Compensation in Protein-Ligand Binding</span></div><div class="casAuthors">Breiten, Benjamin; Lockett, Matthew R.; Sherman, Woody; Fujita, Shuji; Al-Sayah, Mohammad; Lange, Heiko; Bowers, Carleen M.; Heroux, Annie; Krilov, Goran; Whitesides, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">15579-15584</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mechanism (or mechanisms) of enthalpy-entropy (H/S) compensation in protein-ligand binding remains controversial, and there are still no predictive models (theor. or exptl.) in which hypotheses of ligand binding can be readily tested.  Here we describe a particularly well-defined system of protein and ligands-human carbonic anhydrase (HCA) and a series of benzothiazole sulfonamide ligands with different patterns of fluorination-that we use to define enthalpy/entropy (H/S) compensation in this system thermodynamically and structurally.  The binding affinities of these ligands (with the exception of one ligand, in which the deviation is understood) to HCA are, despite differences in fluorination pattern, indistinguishable; they nonetheless reflect significant and compensating changes in enthalpy and entropy of binding.  Anal. reveals that differences in the structure and thermodn. properties of the waters surrounding the bound ligands are an important contributor to the obsd. H/S compensation.  These results support the hypothesis that the mols. of water filling the active site of a protein, and surrounding the ligand, are as important as the contact interactions between the protein and the ligand for biomol. recognition, and in detg. the thermodn. of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2cIrBILFS8bVg90H21EOLACvtfcHk0liuJslhAGYi8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVGjtLbM&md5=8326ee85b1b188d1588a8558870816d5</span></div><a href="/servlet/linkout?suffix=cit242&amp;dbid=16384&amp;doi=10.1021%2Fja4075776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4075776%26sid%3Dliteratum%253Aachs%26aulast%3DBreiten%26aufirst%3DB.%26aulast%3DLockett%26aufirst%3DM.%2BR.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DFujita%26aufirst%3DS.%26aulast%3DAl-Sayah%26aufirst%3DM.%26aulast%3DLange%26aufirst%3DH.%26aulast%3DBowers%26aufirst%3DC.%2BM.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26atitle%3DWater%2520Networks%2520contribute%2520to%2520enthalpy%252Fentropy%2520compensation%2520in%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D15579%26epage%3D15584%26doi%3D10.1021%2Fja4075776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref243"><div class="reference"><strong class="refLabel"><a href="#ref243" class="rightTabRefNumLink">243</a></strong><div class="NLM_citation" id="rightTab-cit243"><span><span class="NLM_contrib-group">Maffucci, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contini, A.</span><span> </span><span class="NLM_article-title">Explicit ligand hydration shells improve the correlation between MM-PB/GBSA binding energies and experimental activities</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">2706</span><span class="NLM_x">–</span> <span class="NLM_lpage">2717</span><span class="refDoi"> DOI: 10.1021/ct400045d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400045d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFWgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=2706-2717&author=I.+Maffucciauthor=A.+Contini&title=Explicit+ligand+hydration+shells+improve+the+correlation+between+MM-PB%2FGBSA+binding+energies+and+experimental+activities&doi=10.1021%2Fct400045d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit243R"><div class="casContent"><span class="casTitleNuber">243</span><div class="casTitle"><span class="NLM_cas:atitle">Explicit Ligand Hydration Shells Improve the Correlation between MM-PB/GBSA Binding Energies and Experimental Activities</span></div><div class="casAuthors">Maffucci, Irene; Contini, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2706-2717</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. Mechanics Poisson-Boltzmann Surface Area (MM-PBSA) and Mol. Mechanics Generalized Born Surface Area (MM-GBSA) methods are widely used for drug design/discovery purposes.  However, it is not clear if the correlation between predicted and exptl. binding affinities can be improved by explicitly considering selected water mols. in the calcn. of binding energies, since different and sometimes diverging opinions are found in the literature.  The authors evaluated how variably populated hydration shells explicitly considered around the ligands may affect the correlation between MM-PB/GBSA computed binding energy and biol. activities (IC50 and ΔGbind, depending on the available exptl. data).  Four different systems--namely, the DNA-topoisomerase complex, α-thrombin, penicillopepsin, and avidin--were considered and ligand hydration shells populated by 10-70 water mols. were systematically evaluated.  The consideration of a hydration shell populated by a no. of water residues (Nwat) between 30 and 70 provided, in all of the considered examples, a pos. effect on correlation between MM-PB/GBSA calcd. binding affinities and exptl. activities, with a negligible increment of computational cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaTvCz5Lu7MLVg90H21EOLACvtfcHk0liuJslhAGYi8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFWgsLg%253D&md5=5e1e7777a576e16567e92319631ff394</span></div><a href="/servlet/linkout?suffix=cit243&amp;dbid=16384&amp;doi=10.1021%2Fct400045d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400045d%26sid%3Dliteratum%253Aachs%26aulast%3DMaffucci%26aufirst%3DI.%26aulast%3DContini%26aufirst%3DA.%26atitle%3DExplicit%2520ligand%2520hydration%2520shells%2520improve%2520the%2520correlation%2520between%2520MM-PB%252FGBSA%2520binding%2520energies%2520and%2520experimental%2520activities%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D2706%26epage%3D2717%26doi%3D10.1021%2Fct400045d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f4j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f4j','PDB','2f4j'); return false;">PDB: 2f4j</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5v" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5v','PDB','3k5v'); return false;">PDB: 3k5v</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM','PDB','1KIM'); return false;">PDB: 1KIM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BMY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BMY','PDB','3BMY'); return false;">PDB: 3BMY</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01684%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01684" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b72c4d9961975","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
