
<html lang="en"     class="pb-page"  data-request-id="dd84d4d2-bec5-4bac-9196-30fb5b184c67"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-15;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.8b01673"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Deciphering the Rules of in Silico Autophagy Methods for Expediting Medicinal Research" /></meta><meta name="dc.Creator" content="Yi  Chen" /></meta><meta name="dc.Creator" content="Guan  Wang" /></meta><meta name="dc.Creator" content="Haoyang  Cai" /></meta><meta name="dc.Creator" content="Yang  Sun" /></meta><meta name="dc.Creator" content="Liang  Ouyang" /></meta><meta name="dc.Creator" content="Bo  Liu" /></meta><meta name="dc.Description" content="Autophagy is a highly evolutionarily conserved cellular catabolic process responsible for degradation of damaged organelles and long-lived proteins. It is not surprising that scientific interest in..." /></meta><meta name="Description" content="Autophagy is a highly evolutionarily conserved cellular catabolic process responsible for degradation of damaged organelles and long-lived proteins. It is not surprising that scientific interest in..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 19, 2019" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01673" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01673" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01673" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01673" /></link>
        
    
    

<title>Deciphering the Rules of in Silico Autophagy Methods for Expediting Medicinal Research | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01673" /></meta><meta property="og:title" content="Deciphering the Rules of in Silico Autophagy Methods for Expediting Medicinal Research" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0003.jpeg" /></meta><meta property="og:description" content="Autophagy is a highly evolutionarily conserved cellular catabolic process responsible for degradation of damaged organelles and long-lived proteins. It is not surprising that scientific interest in the connection of autophagy and diseases has been growing. Over the past 2 decades, many new experimental approaches have been emerging in the field of targeting autophagy for medicinal research. Interestingly, in addition to experimental methods, a number of databases, Web servers, mathematics models, omics approaches, and systems biology network approaches related to autophagy have become available online, which may be collectively considered to be “in silico autophagy methods” for expediting current medicinal research. Thus, we have summarized a series of relevant in silico autophagy approaches for promoting the most appropriate usage of these resources for potential therapeutic purposes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01673"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01673">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01673&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01673&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01673&amp;href=/doi/10.1021/acs.jmedchem.8b01673" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 6831-6842</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.8b01985" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Deciphering the Rules of in Silico Autophagy Methods for Expediting Medicinal Research</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yi Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center and Department of Gastrointestinal Surgery, West China Hospital, Sichuan University, and Collaborative Innovation Center for Biotherapy, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi++Chen">Yi Chen</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guan Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guan Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center and Department of Gastrointestinal Surgery, West China Hospital, Sichuan University, and Collaborative Innovation Center for Biotherapy, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guan++Wang">Guan Wang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haoyang Cai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haoyang Cai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center of Growth, Metabolism, and Aging, Key Laboratory of Bio-Resources and Eco-Environment, College of Life Sciences, Sichuan University, Chengdu 610064, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haoyang++Cai">Haoyang Cai</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yang Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yang Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Pharmaceutical Biotechnology, Department of Biotechnology and Pharmaceutical Sciences, School of Life Sciences, Nanjing University, Nanjing 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yang++Sun">Yang Sun</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liang Ouyang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liang Ouyang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center and Department of Gastrointestinal Surgery, West China Hospital, Sichuan University, and Collaborative Innovation Center for Biotherapy, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liang++Ouyang">Liang Ouyang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5537-8834" title="Orcid link">http://orcid.org/0000-0001-5537-8834</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Bo Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center and Department of Gastrointestinal Surgery, West China Hospital, Sichuan University, and Collaborative Innovation Center for Biotherapy, Chengdu 610041, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#e4888d91868bd6d0d4d4a4d5d2d7ca878b89"><span class="__cf_email__" data-cfemail="7814110d1a174a4c484838494e4b561b1715">[email protected]</span></a>. Phone/fax: (+86) 28-85164063.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Liu">Bo Liu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath6">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3900-9486" title="Orcid link">http://orcid.org/0000-0003-3900-9486</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01673&amp;href=/doi/10.1021%2Facs.jmedchem.8b01673" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 6831–6842</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 19, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 October 2018</li><li><span class="item_label"><b>Published</b> online</span>19 March 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01673" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01673</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6831%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYi%2BChen%252C%2BGuan%2BWang%252C%2BHaoyang%2BCai%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D15%26contentID%3Dacs.jmedchem.8b01673%26title%3DDeciphering%2Bthe%2BRules%2Bof%2Bin%2BSilico%2BAutophagy%2BMethods%2Bfor%2BExpediting%2BMedicinal%2BResearch%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6842%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01673"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1171</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01673" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Deciphering the Rules of in Silico Autophagy Methods for Expediting Medicinal Research&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yi&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Guan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Haoyang&quot;,&quot;last_name&quot;:&quot;Cai&quot;},{&quot;first_name&quot;:&quot;Yang&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Liang&quot;,&quot;last_name&quot;:&quot;Ouyang&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;6831-6842&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01673&quot;},&quot;abstract&quot;:&quot;Autophagy is a highly evolutionarily conserved cellular catabolic process responsible for degradation of damaged organelles and long-lived proteins. It is not surprising that scientific interest in the connection of autophagy and diseases has been growing. Over the past 2 decades, many new experimental approaches have been emerging in the field of targeting autophagy for medicinal research. Interestingly, in addition to experimental methods, a number of databases, Web servers, mathematics models, omics approaches, and systems biology network approaches related to autophagy have become available online, which may be collectively considered to be “in silico autophagy methods” for expediting current medicinal research. Thus, we have summarized a series of relevant in silico autophagy approaches for promoting the most appropriate usage of these resources for potential therapeutic purposes.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01673&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01673" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01673&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01673" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01673&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01673" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01673&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01673&amp;href=/doi/10.1021/acs.jmedchem.8b01673" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01673" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01673" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01673%26sid%3Dliteratum%253Aachs%26pmid%3D30888814%26genre%3Darticle%26aulast%3DChen%26date%3D2019%26atitle%3DDeciphering%2Bthe%2BRules%2Bof%2Bin%2BSilico%2BAutophagy%2BMethods%2Bfor%2BExpediting%2BMedicinal%2BResearch%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D15%26spage%3D6831%26epage%3D6842%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292527" title="Autophagy">Autophagy</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=291324" title="Proteomics">Proteomics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/jmcmar.2019.62.issue-15/20190808/jmcmar.2019.62.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/medium/jm-2018-01673b_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Autophagy is a highly evolutionarily conserved cellular catabolic process responsible for degradation of damaged organelles and long-lived proteins. It is not surprising that scientific interest in the connection of autophagy and diseases has been growing. Over the past 2 decades, many new experimental approaches have been emerging in the field of targeting autophagy for medicinal research. Interestingly, in addition to experimental methods, a number of databases, Web servers, mathematics models, omics approaches, and systems biology network approaches related to autophagy have become available online, which may be collectively considered to be “in silico autophagy methods” for expediting current medicinal research. Thus, we have summarized a series of relevant in silico autophagy approaches for promoting the most appropriate usage of these resources for potential therapeutic purposes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22703" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22703" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Autophagy is a physiological degradation process in lysosomes targeting superfluous proteins and damaged organelles.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> In human diseases, autophagy is believed to be a physiological mechanism to maintain cell homeostasis and survival temporarily.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> On the other hand, continuous and excessive autophagy induced by cellular stress will lead to cell death.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Accumulating experimental evidence has revealed that autophagy plays various pivotal roles by regulating several key mediators as potential targets in medicinal research.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> More recently, a number of small-molecule compounds (e.g., LYN-1604, rapamycin, and IPI-3063) targeting some key autophagy-related proteins (e.g., ULK1, mTOR, and PI3K) or even the autophagic process have been widely reported to be potential candidate drugs to treat disease, such as cancer, neurodegenerative disease, and immunity disease<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Accordingly, targeting autophagy by “experimental methods” may be an encouraging therapeutic strategy in different model organisms and relevant diseases.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/medium/jm-2018-01673b_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic model of the representative small-molecule compounds targeting autophagic proteins or autophagic process in different types of diseases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">However, there are still some limitations regarding the use of experimental methods to analyze the complicated process of autophagy. Thus, several “in silico” autophagy methods have recently been utilized to investigate autophagy. To date, there are more than 30 000 articles available when using “autophagy” as a keyword to search PubMed. These studies provide a wealth of information for further collection, classification, analysis, and even knowledge-based prediction in the field of autophagy research. To better utilize the available information on autophagy, there are several databases, Web servers, mathematics models, omics methods, and systems-biology network approaches that can collectively be considered to be key supplements to experimental methods to expedite the current medicinal research. In this review, we focus on summarizing relevant in silico autophagy approaches that will promote the most appropriate usage of the above-mentioned resources for the future therapeutic purposes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Databases and Web Servers for Autophagy</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31706" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Hitherto, there have been a number of new databases and Web servers related to in silico autophagy methods that provide a wealth of useful information for subsequent medicinal research. In this section, we discuss the categories of databases or Web servers according to different types of autophagy-related data they provide, including (i) autophagy-related proteins, (ii) autophagy-related ribonucleic acids (RNAs), (iii) autophagy-related compounds, (iv) orthologue analyses, and (v) predicted autophagic targets (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Databases and Web Servers Related to Autophagy</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">feature</th><th class="colsep0 rowsep0" align="center">data resource</th><th class="colsep0 rowsep0" align="center">data statistics</th><th class="colsep0 rowsep0" align="center">Web link</th><th class="colsep0 rowsep0" align="center">latest update</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ACDB</td><td class="colsep0 rowsep0" align="left">Autophagy-related comound</td><td class="colsep0 rowsep0" align="left">1. It includes a great deal of information on autophagy-related compounds. 2. For users to access to more than 300 curated compounds correlated to autopahgy.</td><td class="colsep0 rowsep0" align="left">text-mining, Pubmed, Scifinder Scholar, ChemcalBook, Reaxys</td><td class="colsep0 rowsep0" align="left">357 compounds; 96 pathways; 68 potential targets; 443 cell types</td><td class="colsep0 rowsep0" align="left"><a href="http://www.acdbliulab.com/" class="extLink">http://www.acdbliulab.com/</a></td><td class="colsep0 rowsep0" align="left">September 30, 2017</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ACTP</td><td class="colsep0 rowsep0" align="left">Predict autophagic target</td><td class="colsep0 rowsep0" align="left">1. ACTP can predict autophagic targets and relevant pathways. 2. Flexible docking and back end reverse docking can be provided.</td><td class="colsep0 rowsep0" align="left">Protein Data Bank database, UniProt database, DAVID database, PrePPI database</td><td class="colsep0 rowsep0" align="left">199 reviewed proteins, 231 unreviewed proteins, 24035 human protein accession numbers</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">January 13, 2016</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ANCHOR</td><td class="colsep0 rowsep0" align="left">Predict autophagic target</td><td class="colsep0 rowsep0" align="left">1. It can provide the predicting protein binding regions in disordered proteins. 2. Incorporates the result of a general disorder prediction method, IUPred, and can carry out simple motif searches.</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">6 samples were provided on the Web server</td><td class="colsep0 rowsep0" align="left"><a href="http://anchor.enzim.hu" class="extLink">http://anchor.enzim.hu</a></td><td class="colsep0 rowsep0" align="left">August 7, 2009</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AutomiRDB</td><td class="colsep0 rowsep0" align="left">Autophagy-related RNA</td><td class="colsep0 rowsep0" align="left">1. It includes human miRNAs, autophagic target genes/proteins, different types of cancer.</td><td class="colsep0 rowsep0" align="left">text-mining, Gene Ontology, OMIM, miRBase, miTarBase, TargetScan, MiRanda, PicTar</td><td class="colsep0 rowsep0" align="left">493 autopahgy-related miRNAs; 90 targeted autophagic genes/proteins; 18 types of cancer</td><td class="colsep0 rowsep0" align="left"><a href="http://www.chen-lab.com/index.php" class="extLink">http://www.chen-lab.com/index.php</a></td><td class="colsep0 rowsep0" align="left">May 1, 2015</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Autophagy database</td><td class="colsep0 rowsep0" align="left">Autophagy-related protein</td><td class="colsep0 rowsep0" align="left">1. It collected autophagy-related proteins identified in each species. 2. Users can search for proteins to obtain functional, structureal data and find possible functional homologs of proteins.</td><td class="colsep0 rowsep0" align="left">text-mining, mainly from 3 literatures</td><td class="colsep0 rowsep0" align="left">41 eukaryotes; 2163 proteins that are either known to be involved in autophagy or homologous to autophagy-related proteins</td><td class="colsep0 rowsep0" align="left"><a href="http://www.tanpaku.org/autophagy/overview.html" class="extLink">http://www.tanpaku.org/autophagy/overview.html</a></td><td class="colsep0 rowsep0" align="left">October 6, 2010</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AutophagySMDB</td><td class="colsep0 rowsep0" align="left">Autophagy-related compound</td><td class="colsep0 rowsep0" align="left">1. It collected curated information on the konwn autophagy targets and related molecules. 2. Users can search by text, structure, and advanced content. 3. Various computational tools were provided.</td><td class="colsep0 rowsep0" align="left">Text-mining, PubChem, Protein Data Bank (PDB)</td><td class="colsep0 rowsep0" align="left">71 targets, ∼10 000 small molecule modulators</td><td class="colsep0 rowsep0" align="left"><a href="http://www.autophagysmdb.org/" class="extLink">http://www.autophagysmdb.org/</a></td><td class="colsep0 rowsep0" align="left">January 14, 2019</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GAMDB</td><td class="colsep0 rowsep0" align="left">Autophagy-related RNA</td><td class="colsep0 rowsep0" align="left">1. The corresponding information about the relations between miRNAs and autophagy was collected.</td><td class="colsep0 rowsep0" align="left">text-mining, Gene Ontology, OMIM, DianaTools, miRBase, UniProt, DIANA-TarBase</td><td class="colsep0 rowsep0" align="left">836 autophagy-related miRNAs; 197 targeted genes/proteins; 56 aging-related diseases</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">January 29, 2016</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HADb</td><td class="colsep0 rowsep0" align="left">Autophagy-related protein</td><td class="colsep0 rowsep0" align="left">1. The features and sequences of autophagy genes, tracscripts, exons and proteins were described. 2. HADb integrates autophgy-related genes and proteins in research reports, thereby constituting the first exhaustive database dedicated to autophagy.</td><td class="colsep0 rowsep0" align="left">text-mining, NCBI RefSeq, GenBank, Gene Ontology, UniProt, Ingenuity Pathway Analysis, DAVID, FatiGO</td><td class="colsep0 rowsep0" align="left">234 autophagy-related genes</td><td class="colsep0 rowsep0" align="left"><a href="http://www.autophagy.lu" class="extLink">www.autophagy.lu</a></td><td class="colsep0 rowsep0" align="left">March 14, 2011</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HAMdb</td><td class="colsep0 rowsep0" align="left">Autophagy-related compound</td><td class="colsep0 rowsep0" align="left">1. 796 proteins, 841 chemicals, and 132 microRNAs from 871 literatures and 286 cell lines were collected. 2. A lot of external links were available for more information. 3. The user-friendly Web site allows readers without computational backgroud to search and download. 4. It can give hints for discovery of new modulators and targets</td><td class="colsep0 rowsep0" align="left">text-mining, search by databases such as HADb, autophagy database, autophagy regulatory network database, MedChem express, Selleck, APExBIO, ncRDeathDB</td><td class="colsep0 rowsep0" align="left">796 autophagy-related proteins; 841 chemicals; 132 microRNAs</td><td class="colsep0 rowsep0" align="left"><a href="http://hamdb.scbdd.com" class="extLink">http://hamdb.scbdd.com</a></td><td class="colsep0 rowsep0" align="left">July 31, 2018</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hfAIM</td><td class="colsep0 rowsep0" align="left">Autophagy-related protein</td><td class="colsep0 rowsep0" align="left">1. To reliably identify Atg8 interacting motifs in proteins in various organisms. 2. Nine peroxisomal PEX proteins were identified. 3. AtPEX6 and AtPEX10 could interact with Atg8 in plants. 4. Mutations occurring within or near hfAIMs in PEX1, PEX6, and PEX10 caused defects in various organisms.</td><td class="colsep0 rowsep0" align="left">text-mining, PeroxisomeDB</td><td class="colsep0 rowsep0" align="left">36 verified functional AIMs; 26 verfied <i>Arabidopsis</i> Atg8-interacting proteins</td><td class="colsep0 rowsep0" align="left"><a href="http://bioinformatics.psb.ugent.be/hfAIM/" class="extLink">http://bioinformatics.psb.ugent.be/hfAIM/</a></td><td class="colsep0 rowsep0" align="left">February 2, 2016</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iLIR</td><td class="colsep0 rowsep0" align="left">Autophagy-related protein</td><td class="colsep0 rowsep0" align="left">1. Listed all the putative canonical LIRCPs identified in silico in the proteomes of 8 model organisms. 2. The results were combined with the Gene Ontology term analysis. 3. Users can identify novel putative LICRPs in mammals.</td><td class="colsep0 rowsep0" align="left">UniProt database, GO database</td><td class="colsep0 rowsep0" align="left">8 model organisms: <i>Arabidopsis thaliana</i>, <i>Caenorhabditis elegans</i>, Danio rerio, <i>Gallus gallus</i>, <i>Homo sapiens</i>, <i>Mus musculus</i>, <i>Rattus norvegicus</i>, and <i>Saccharomyces cerevisiae</i></td><td class="colsep0 rowsep0" align="left"><a href="https://ilir.warwick.ac.uk" class="extLink">https://ilir.warwick.ac.uk</a></td><td class="colsep0 rowsep0" align="left">February 24, 2016</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">miRDeathDB</td><td class="colsep0 rowsep0" align="left">Autophagy-related RNA</td><td class="colsep0 rowsep0" align="left">1. Integrates information for experimentally identified miRNAs and their targets in PCD network. 2. Users can search the PCD-associated miRNAs and their targets. 3. Users can retrieve plenty of genomic, proteomic, and disease-associated data.</td><td class="colsep0 rowsep0" align="left">text-mining</td><td class="colsep0 rowsep0" align="left">92 miRNAs; 106 proteins; 233 entries</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">October 1, 2012</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ncRDeathDB</td><td class="colsep0 rowsep0" align="left">Autophagy-related RNA</td><td class="colsep0 rowsep0" align="left">1. Users can query, browse, and manipulate ncRNA-associated cell death interactions. 2. The database can help to visualize and navigate knowledge of the ncRNA of cell death and autophagy.</td><td class="colsep0 rowsep0" align="left">test-mining, 1495 references, 12 species</td><td class="colsep0 rowsep0" align="left">4615 ncRNA-mediated PCD entries in 12 species, 2403 apoptosis-associated entries, 2205 autophagy-associated entries, 7 necrosis-associated entries</td><td class="colsep0 rowsep0" align="left"><a href="http://www.rna-society.org/ncrdeathdb" class="extLink">http://www.rna-society.org/ncrdeathdb</a></td><td class="colsep0 rowsep0" align="left">May 26, 2015</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ProSelection</td><td class="colsep0 rowsep0" align="left">Predict autophagic target</td><td class="colsep0 rowsep0" align="left">1. A computational and statistical algorithm for picking the proper protein structure subsets. 2. It could accelerate the computational work in protein structure selection. 3. A useful tool for molecular docking, target prediction, and protein–chemical database establishment research.</td><td class="colsep0 rowsep0" align="left">collect from literature and public databases, including PubMed, PubChem, DrugBank, SciFinder, UniProt, ChEMBL, PDB</td><td class="colsep0 rowsep0" align="left">249 protein structures of 14 autophagy-related targets</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">October 10, 2010</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TBC2health database</td><td class="colsep0 rowsep0" align="left">Autophagy-related comound</td><td class="colsep0 rowsep0" align="left">1. Provided compound, disease or phenotype, evidence and reference information. 2. It provides bipartite network and topological analyses. 3. A submission page and useful tools are provided. 4. TBC2health can serve for the exploration of beneficial effects of tea on human health.</td><td class="colsep0 rowsep0" align="left">test-mining, over 300 published articles</td><td class="colsep0 rowsep0" align="left">1338 relationships between 497 tea bioactive compounds and 206 disease or phenotypes</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">July 5, 2016</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">THANATOS</td><td class="colsep0 rowsep0" align="left">Ortholog analysis</td><td class="colsep0 rowsep0" align="left">1. The collection, curation, and integration of regulators and post-translational modifications will be helpful for understanding the molecular mechanisms of autophagy at a systems level and provide highly useful information for further experimental consideration</td><td class="colsep0 rowsep0" align="left">Proteins from PubMed, ARN, DeathBase, the autophagy census; proteome sets from Ensembl, Ensembl Metazoa, Ensembl Fungi; PPI pairs from the I2D, IntAct, MINT; PTM sites from previous studies and CPLM; cancer genes from COSMIC; drug targets from DrugBank</td><td class="colsep0 rowsep0" align="left">4237 known proteins; 191 543 potential proteins in 164 eukaryotes; 93 222 PPIs; 65 015 known sites of 11 types of post-translational modifications; 559 well-curated cancer genes; 2247 human drug targets</td><td class="colsep0 rowsep0" align="left"><a href="http://thanatos.biocuckoo.org" class="extLink">http://thanatos.biocuckoo.org</a></td><td class="colsep0 rowsep0" align="left">May 7, 2017</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr></tbody></table></div></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Autophagy-Related Protein Databases</h3><div class="NLM_p last">The autophagy database, which is known to be an all-inclusive information resource for autophagy, collects data and identifies homologs in 41 eukaryotes.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> By searching of the keywords or sequences in this database, corresponding data including functional and structural information and possible functional homologs can be obtained. In addition, the microtubule-associated protein 1 light chain 3 (LC3) interacting region (iLIR) database is a Web-based resource for LIR motif-containing proteins in eukaryotes. It is used for the identification of LC3-interacting region-containing proteins (LIRCPs) in various organisms.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> This database lists all of the putative canonical LIRCPs in the proteomes of eight model organisms identified in silico and also provides a way to identify novel putative LICRPs in mammals by curated text-mining analysis of the literature. Similarly, the high fidelity Atg8 interacting motif (hfAIM) represents a bioinformatics methodology of in silico genome-wide identification, which is used to identify autophagy-associated AIMs in organisms.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The hfAIM is an effective tool for identifying proteins that are governed by certain types of selective autophagy by performing genome-wide in silico screening. Human autophagy database (HADb) is another comprehensive database for autophagy-related gene and protein. Over 200 genes and proteins in this database, collected from online resources and literature, provide general, genomic, and functional information. HADb can help analyze and search for autophagy-related gene and protein regulation in cellular models and diseases.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Autophagy-Related RNA Databases</h3><div class="NLM_p last">The gerontology-autophagic-microRNA database (GAMDB) is a Web-based resource that analyzes microRNAs (miRNAs) and autophagy in gerontology.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This database contains 56 aging-related diseases, 197 targeted genes/proteins, and 836 autophagy-related miRNAs. In addition, autophagy-microRNA database (AutomiRDB) is another Web-based resource for microRNAs involved in autophagy in cancer.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In AutomiRDB, information on experimentally identified microRNAs is integrated and autophagic target genes of these microRNAs in different types of cancer are provided. Eighteen types of cancer, 90 targeted autophagic genes/proteins, and 493 autophagy-related miRNAs are included in AutomiRDB. Moreover, miRDeathDB is another well-known database related to microRNAs and programmed cell death (PCD).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This database integrates useful information on the miRNAs that were identified in experiments and lists their targets in the PCD network. There are 210 records including 86 miRNAs and 95 protein coding genes in five model species in total. Similarly, ncRDeathDB comprehensively deciphered the network organization of cell death system mediated by noncoding RNA (ncRNA).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> More than 4600 ncRNA-mediated PCD entries, which were collected from 12 species, are documented in this database.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Autophagy-Related Compound Databases</h3><div class="NLM_p last">The autophagic compound database (ACDB) is an online resource providing information on autophagy-modulating compounds, their targets, and relevant diseases.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> This database covers 357 compounds with their potential targets and 164 corresponding signaling pathways in various human diseases. Valuably, ACDB allows its users to access more than 300 curated small-molecule autophagic compounds. Additionally, TBC2health is another database describing experimentally identified tea bioactive compounds with health-beneficial effects.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This database currently records 1338 relationships among 497 tea bioactive compounds and 206 relevant phenotypes or diseases. Recently, several autophagy databases as sources for structural and functional information have been generated. However, a special database covering information on autophagy modulator (microRNAs, chemicals, and proteins)-related targets, signaling pathways, and diseases has not been developed yet. Therefore, the human autophagy modulator database (HAMdb) is developed by our research group, which can be searched in the Web set of <a href="http://hamdb.scbdd.com" class="extLink">http://hamdb.scbdd.com</a>. It provides information on related pathways and disease to researchers. HAMdb contains 132 microRNAs, 841 chemicals, and 796 proteins. We manually collected and compiled these entities information on biological, physicochemical, disease, and their specific effects on autophagy. HAMdb is a user-friendly interface for users to search, browse, and query autophagy modulators and to obtain comprehensive relevant information. It provides detailed information about autophagy related diseases and promotes the understanding of autophagy process. Additionally, HAMdb provides hints to the users to identify new diagnostic and therapeutic targets and discover new autophagy modulators. Therefore, it is reasonable to believe that HAMdb can be a potential platform to facilitate autophagy research.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Moreover, autophagy small molecule database (AutophagySMDB) has been reported to be a curated database containing about 10 000 small molecules that regulate 71 protein targets in autophagy.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> AutophagySMDB may help the discovery of more potential small-molecule drug candidates for autophagy in human diseases.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Orthologue Analysis Database</h3><div class="NLM_p last">The autophagy necrosis, apoptosis orchestrators (THANATOS) is an orthologue analysis database resource for proteins in regulating autophagy and post-translational modifications. This database contains 4237 experimentally identified proteins that are involved in the process of autophagy and cell death.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Several potential orthologues of known proteins have been identified and stored in THANATOS database. THANATOS contains 191 543 proteins which show potential to regulate autophagy and cell death process in 164 eukaryotes.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Predicted Autophagic Target Web Servers</h3><div class="NLM_p last">Autophagic compound-target prediction (ACTP) is a Web server providing a basis for rapid predicting potential autophagic targets and pathways of possible autophagy-inhibiting or autophagy-activating small-molecule compounds/candidate drugs.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> ACTP sheds light on identifying future therapeutic implications of novel autophagy-inhibiting or autophagy-activating compounds. Moreover, protein structure selection (ProSelection) is a novel algorithm for in silico drug target identification to screen appropriate protein structure subsets.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> ProSelection is another helpful algorithm of docking preferred-protein selection and can be used to generate appropriate structure subset(s), including 249 protein structures and 14 autophagic targets. Furthermore, the performance of ProSelection was verified using 43 small-molecule antineoplastic drugs approved by Food and Drug Administration (FDA), which predicted the potential off-targets of drugs successfully. In addition, ANCHOR is a Web server to predict binding regions of protein in disordered proteins.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The minimum input of ANCHOR is a single amino acid sequence. Besides, ANCHOR can predict disordered protein binding regions in isolation. Taken together, the aforementioned autophagy-related databases and Web servers can help us to take advantage of Web-based resources to achieve a better understanding of in silico autophagy methods for potential medicinal applications.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Mathematical Models of Autophagy</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63306" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63306" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">With the rapid progress of autophagy research, an increasing number of mathematical models have been introduced in the field of in silico autophagy research. In this section, we mainly focus on highlighting some interesting mathematical models for autophagy research, including autophagy/autophagic mechanisms and autophagy-related diseases (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Mathematical Models of Autophagy</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">title</th><th class="colsep0 rowsep0" align="center">purpose</th><th class="colsep0 rowsep0" align="center">method</th><th class="colsep0 rowsep0" align="center">feature</th><th class="colsep0 rowsep0" align="center">application</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A comparative in silico analysis of autophagy proteins in ciliates.</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Integrated databases and resources</td><td class="colsep0 rowsep0" align="left">1. Atg proteins were partially conserved in ciliates. 2. Provided a better understanding for the autophagic destruction of the parental macronucleus.</td><td class="colsep0 rowsep0" align="left">The role of autophagy proteins in ciliates.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">In silico knockout studies of xenophagic capturing of Salmonella.</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Based on models and analysis methods</td><td class="colsep0 rowsep0" align="left">1. Petri net model of xenopahgy in epithelial cells were presented. 2.The in silico knockout analyses provided a basis for investigations of the Salmonella xenopahgy pathway.</td><td class="colsep0 rowsep0" align="left">Mechanism of xenophagic capturing of Salmonella.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Mutation–structure–function relationship based integrated strategy reveals the potential impact of deleterious missense mutations in autophagy related proteins on hepatocellular carcinoma (HCC): a comprehensive informatics approach</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Integrated databases and resources</td><td class="colsep0 rowsep0" align="left">1. LC3B Y113C, BECN1 I403T, SCD1 R126S, and SCD1 Y218C were highly deleterious HCC-associated mutations. 2. Provided a valuable resource for mechanistic insight into investigations of pathological missense SNPs.</td><td class="colsep0 rowsep0" align="left">Deleterious missense mutations in autophagy related proteins.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Computational model for autophagic vesicle dynamics in single cells</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Based on algorithms</td><td class="colsep0 rowsep0" align="left">1. The computational model describing autophagic vesicle dynamics in a mammalian system was developed.</td><td class="colsep0 rowsep0" align="left">Autophagic vesicle dynamics in single cells.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Agent-based modeling of autophagy reveals emergent regulatory behavior of spatiotemporal autophagy dynamics</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Based on algorithms</td><td class="colsep0 rowsep0" align="left">1. An agent-based model was developed to represent individual components of the autophagy pathway. 2. The model could reproduce short-term autophagic flux measurements and measure the degree of cell-to-cell variability.</td><td class="colsep0 rowsep0" align="left">The emergent regulatory behavior of spatiotemporal autophagy dynamics.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A multimethod computational simulation approach for investigating mitochondrial dynamics and dysfunction in degenerative aging</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Based on software</td><td class="colsep0 rowsep0" align="left">1. A multilevel hybrid-modeling paradigm was developed to simulate aging mitochondrial phenotypes. 2. The model supported insights into molecular determinants of aging as well as cytoprotective agents that may improve neurological or muscular healthspan.</td><td class="colsep0 rowsep0" align="left">Mitochondrial dynamics and dysfunction in degenerative aging.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Dynamic modeling of the interaction between autophagy and apoptosis in mammalian cells</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Based on algorithms</td><td class="colsep0 rowsep0" align="left">1. The mathematical model of cell fate decisions mediated by autophagy was established. 2. The model was consistent with existing quantitative measurements of autophagy and apoptosis in rat kidney proximal tubular cells responding to cisplatin-induced stress.</td><td class="colsep0 rowsep0" align="left">The interaction between autophagy and apoptosis.</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td></tr></tbody></table></div></div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> In Silico Models for Autophagy/Autophagic Mechanisms</h3><div class="NLM_p last">In silico comparative analysis of autophagy proteins has been reported in ciliates, demonstrating that autophagy-related (Atg) proteins are partially conserved and thus provide a better understanding of the autophagic destruction of parental macronucleus.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Additionally, Salmonella xenophagy as a key defense mechanism in epithelial cells has been investigated using the Petri net model.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The performance of in silico knockout analyses displayed a consistent result with the published experimental data, which provided guidance to researchers for the future study of Salmonella xenophagy pathway. Additionally, a recent study has carried out an extensive in silico mutational analysis of Atg proteins, including SCD1, BECN1, LC3A, and LC3B, and indicated that high-risk missense mutations damaged the structure and function of Atg proteins. This study provided an insight into pathological missense single-nucleotide polymorphisms (SNPs).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Moreover, a recent study has shown an in silico model that has been used in a mammalian system to describe autophagic vesicle dynamics. This accurate model is proposed to be a useful approach for quantitative characterization of autophagy.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Agent-based model (ABM) of autophagy has been reported to reveal the emergent regulation of spatiotemporal autophagy dynamics. In ABM, components of autophagy pathways, metabolic feedbacks with the cellular environment, and subcellular vesicle dynamics can be represented individually. ABM sheds new light on elucidation of spatiotemporal autophagy regulation and dynamic behavior.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> In Silico Models for Autophagy-Related Diseases</h3><div class="NLM_p last">In silico simulation approaches have been reported to investigate the mitochondrial dynamics and dysfunction linked to mitochondrial autophagy (mitophagy) in degenerative aging. In this case, a variety of genetic and pharmacological schemes is applied to virtually perturb the in silico system. Then, the alterations specific mechanistic targets in the transient alterations are quantitated, and the cytoprotective agents and molecular determinants of aging are supported to improve muscular or neurological health.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> More recently, dynamic modeling of the interaction has been reported between autophagy and apoptosis in cells of mammal.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Since altered level of autophagy is observed in many diseases, including cancer, it is necessary to have a comprehensive understanding of the crucial process from autophagy to apoptosis. As mentioned above, these mathematical models have been widely used to solve a variety of autophagy-related scientific problems, some of which can be also utilized in future medicinal research. However, these models are still in their infancy and require further in-depth investigations.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Omics-Based Analyses of Autophagy</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To date, researchers have performed numerous studies of “omics” technology covering genomics, transcriptomics, proteomics, and so on. Virtually, omics-based analyses have been validated to be powerful tools for understanding the underlying mechanisms of autophagy, especially in medicinal research (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Omics-Based Analyses of Autophagy</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">title</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">method and resource</th><th class="colsep0 rowsep0" align="center">feature</th><th class="colsep0 rowsep0" align="center">application</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Gene network exploration of crosstalk between apoptosis and autophagy in chronic myelogenous leukemia.</td><td class="colsep0 rowsep0" align="left">Data analysis</td><td class="colsep0 rowsep0" align="left">Integrated data from GEO database.</td><td class="colsep0 rowsep0" align="left">1. Exploring the functional linkages and the potential coordinate regulations. 2. Mcl1, Bcl2, Atg, Beclin-1, Bax, BAK, E2F,cMYC, PI3K, AKT, BAD, and LC3 were extracted for the crosstalk between apoptosis and autophagy.</td><td class="colsep0 rowsep0" align="left">Chronic myeloid leukemia</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Discovery of a small molecule targeting ULK1-modulated cell death of triple negative breast cancer in vitro and in vivo.</td><td class="colsep0 rowsep0" align="left">Gene microarray</td><td class="colsep0 rowsep0" align="left">The samples were obtained from cells.</td><td class="colsep0 rowsep0" align="left">1. ULK1 was remarkably downregulated in breast cancer. 2. A novel ULK1 agonist was designed.</td><td class="colsep0 rowsep0" align="left">Triple negative breast cancer</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Evalution of genetic variants in autophagy pathway genes as prognostic biomarkers for breast cancer.</td><td class="colsep0 rowsep0" align="left">Gene microarray</td><td class="colsep0 rowsep0" align="left">Tissue samples were recruited from hospitals.</td><td class="colsep0 rowsep0" align="left">1. Atg7 variant rs8154 represents a novel prognostic marker. 2. Discovered the effect of variants in core Atgs on breast cancer survival.</td><td class="colsep0 rowsep0" align="left">Breast cancer</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Transcriptomic analysis of the autophagy machinery in crustaceans.</td><td class="colsep0 rowsep0" align="left">Transcriptomic analysis</td><td class="colsep0 rowsep0" align="left">Tissue samples were obtained from <i>Macrobrachium rosenbergii</i>.</td><td class="colsep0 rowsep0" align="left">1. Five autophagy markers (Beclin-1, Vps34, MAP1LC3B, SQSTM1, and Lamp1) were identified that were conserved in crustaceans comparable in mammals.</td><td class="colsep0 rowsep0" align="left">In crustaceans</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Independent regulation of gene expression level and noise by histone modifications.</td><td class="colsep0 rowsep0" align="left">Data analysis</td><td class="colsep0 rowsep0" align="left">Integrated data from database and reported articles.</td><td class="colsep0 rowsep0" align="left">1. The “division of labor” among histone modifications facilitates the independent regulation of expression level and noise; 2. Expression noise had increased when the H3K79 methylation was knocked out in yeast.</td><td class="colsep0 rowsep0" align="left">Human embryonic cells</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Integrative bioinformatics and proteomics-based discovery of an eEF2K inhibitor (cefatrizine) with ER stress modultation in breast cancer cells.</td><td class="colsep0 rowsep0" align="left">Proteomics analysis</td><td class="colsep0 rowsep0" align="left">Performed by iTRAQ-based proteomics analysis.</td><td class="colsep0 rowsep0" align="left">1. Eukaryotic elongation factor-2 kinase (eEF2K) could be regarded as a promising therapeutic taret. 2. A novel eEF2K inhibitor was discovered.</td><td class="colsep0 rowsep0" align="left">Breast cancer</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Phosphoproteome-based kinase activity profiling reveals the critical role of MAP2K2 and PLK1 in neuronal autophagy.</td><td class="colsep0 rowsep0" align="left">Proteomics analysis</td><td class="colsep0 rowsep0" align="left">Performed by quantitative phosphoproteomic profiling.</td><td class="colsep0 rowsep0" align="left">1. Developed a powerful method for the identification of important regulators. 2. MAP2K2 and PLK1 play an important role in neuronal autophagy by phosphoproteomic analysis.</td><td class="colsep0 rowsep0" align="left">Neuronal autophagy</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Network organization of the human autophagy system.</td><td class="colsep0 rowsep0" align="left">Proteomics analysis</td><td class="colsep0 rowsep0" align="left">Performed by proteomics analysis.</td><td class="colsep0 rowsep0" align="left">1. Revealed a network of 751 interactions among 409 candidate interacting proteins with extensive connectivity among subnetworks. 2. Provided a global view of the mammalian autophagy interaction landscape. 3. Provided a resource for mechanistic analysis of this pathway.</td><td class="colsep0 rowsep0" align="left">Human autophagy system</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RAB7A phosphorylation by TBK1 promotes mitophagy via the PINK-PARKIN pathway.</td><td class="colsep0 rowsep0" align="left">Proteomics analysis</td><td class="colsep0 rowsep0" align="left">Performed by phosphoproteomics analysis.</td><td class="colsep0 rowsep0" align="left">1. TBK1 could promote downstream steps in capture of damaged mitochondria for mitopagy. 2. Reveal novel functions of TBK1 in mitophagy.</td><td class="colsep0 rowsep0" align="left">Part mechnism of mitopahgy</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></td></tr></tbody></table></div></div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Genomics Analyses</h3><div class="NLM_p last">A recent report has performed a gene network study in chronic myelogenous leukemia to explore the interplay between apoptosis and autophagy. To elucidate the crosstalk between apoptosis and autophagy, genomics analyses were applied to investigate the underlying mechanisms and explore the roles of transcription factors and microRNAs.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In addition, according to analyses of the cancer genome atlas (TCGA) and tissue microarray (TMA), a remarkable downregulated level of Unc-51-like kinase 1 (ULK1) in breast cancer tissue samples, especially in triple negative breast cancer (TNBC), is observed. These results revealed that ULK1 activation is a new strategy for future TNBC therapy.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In another study, the relationship between genetic variants in core Atgs with prognosis of breast cancer is investigated. As a result, the Atg7 variant rs8154 is validated to be a prognostic marker for patients with breast cancer.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Transcriptomics Analyses</h3><div class="NLM_p last">In silico research has been applied on transcripts encoding Atg proteins within various tissue transcriptomes of <i>Macrobrachium rosenbergii</i>. This method offers a platform for autophagy investigation in crustaceans, which led to a better understanding of reproduction- and stress-related autophagy, and may promote more efficient aquaculture practices.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> In another study, it has shown that the “division of labor” among histone modifications could facilitate the independent regulation of the autophagic gene expression level and noise. This “histone code” hypothesis can be extended to include expression noise. These results provide insight into the optimization of transcriptomes in evolution.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Proteomics Analyses</h3><div class="NLM_p last">Proteomics-based discovery and integrative bioinformatics have been utilized to identify a eukaryotic elongation factor 2 kinase (eEF2K) inhibitor, called cefatrizine, which induces endoplasmic reticulum (ER) stress in breast cancer cells.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Phosphoproteome-based kinase activity profiling has been reported to reveal that MAP2K2 and PLK1 play pivotal roles in neuronal autophagy.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Moreover, the autophagy interaction network has been investigated with a proteomic analysis in human cells under basal autophagy. As a result, a network of 751 interactions was obtained, among 409 candidate interacting proteins with extensive connectivity among subnetworks.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Recently, a global quantitative proteomics method has been used for verifying that phosphorylation RAB7A by TBK1 triggers mitophagy through PINK-PARKIN pathway.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The above-mentioned genomics, transcriptomics, and proteomics analyses of autophagy shed new light on further elucidating autophagic mechanisms and relevant diseases for potential medicinal implications.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Systems Biology Network Approaches of Autophagy</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In autophagy research, a new method with therapeutic implications lies in discovering candidate drugs, which act on autophagy-related networks and the key signaling pathways rather than their individual genes or proteins. In this section, several systems-biology network approaches are summarized to solve the problems of in silico autophagy research, such as autophagy/autophagic mechanisms and relevant human diseases (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Systems-Biology Network Approaches to Autophagy</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">title</th><th class="colsep0 rowsep0" align="center">purpose</th><th class="colsep0 rowsep0" align="center">method</th><th class="colsep0 rowsep0" align="center">feature</th><th class="colsep0 rowsep0" align="center">application</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A novel hybrid yeast-human network analysis reveals an essential role for FNBP1L in antivacterial autophagy.</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Integrated the interolog network</td><td class="colsep0 rowsep0" align="left">1. Fourteen novel mammalian genes involoved in autophagy were revealed. 2. FNBP1L appears dispensable for starvation- or rapamycin-induced autophagy. 3. FNBP1L is a differentially used molecule in specific autophagic contexts.</td><td class="colsep0 rowsep0" align="left">The role for genes in antibacterial autophagy</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Autophagy regulatory network, a systems-level bioinformatics resource for studying the mechanism and regulation of autophagy.</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Collected the literature and integrated external resources</td><td class="colsep0 rowsep0" align="left">1. 1485 proteins with 4013 interactions. 2. 413 transcription factors and 386 miRNAs which could regulate autophagy components or their protein regulators. 3. Established a user-friendly Web site.</td><td class="colsep0 rowsep0" align="left">The mechanism and regulation of autophagy</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Discovering protein–protein interactions within the programmed cell death network using a protein-fragment complementation screen.</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Modeling the PCD modules and their connectivity</td><td class="colsep0 rowsep0" align="left">1. Forty-six novel interactions were identified. 2. The interaction of 14-3-3τ and DAPK2 could inhibit DAPK2 dimerization and activity. 3. Supported the power of the Gluc PCA platform for the discovery of biochemical pathways.</td><td class="colsep0 rowsep0" align="left">The interaction of apoptosis and autophagy</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Database screening of herbal monomers regulating autophagy by constructing a “disease–gene–drug” network.</td><td class="colsep0 rowsep0" align="left">Mechanisms</td><td class="colsep0 rowsep0" align="left">Modeling a compound–gene regulatory network</td><td class="colsep0 rowsep0" align="left">1. 544 differentially expressed genes, 375 pairs of differentially expressed genes, 7 gene modeles were screened. 2. Thirty herbal monomers that can regulate 7 genes were identified.</td><td class="colsep0 rowsep0" align="left">The application of herbal medicines in cancer therapy</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Identification of microRNA-regulated autophagic pathways in plant lectin-induced cancer cell death.</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Modeling the global protein–protein interaction network</td><td class="colsep0 rowsep0" align="left">1. Nine autophagic hub proteins were identified. 2. Thirteen relevant oncogenic and tumor suppressive miTNAs were identified. Plant lectins could block the sugar-containing receptor EGFR-mediated survival pathways.</td><td class="colsep0 rowsep0" align="left">Identification of targets and pathways in breast cancer</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Identification of novel caspase/autophagy-related gene switch to cell fate decisions in breast cancers.</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Modeling the global protein–protein interaction network</td><td class="colsep0 rowsep0" align="left">1. Three genes (androgen receptor, PAK1, MAPK3) were identified as the potential targets, especially MAPK3.</td><td class="colsep0 rowsep0" align="left">Identification of targets in breast cancer</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Identification of ULK1 as a novel biomarker involved in miR-4487 and miR-595 regulation in neuroblastoma SH-SY5Y cell autophagy.</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Analysis of the RNA-mediated network</td><td class="colsep0 rowsep0" align="left">1. MiR-4487 and miR-595 could target ULK1 to regulate autophagy. 2. A nobel ULK1-p7056K autophagic pathway was identified.</td><td class="colsep0 rowsep0" align="left">Identification of targets and pathways in Parkinson’s disease</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Systems biology-based discovery of a potential Atg4B agonist (flubendazole) that induces autophagy in breast cancer.</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Modeling the global protein–protein interaction network</td><td class="colsep0 rowsep0" align="left">1. Atg4B was identified as a novel target.</td><td class="colsep0 rowsep0" align="left">Identification of targets in triple-negative breast cancer</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bioinformatic analysis of cacybp-associated proteins using human glioma databases.</td><td class="colsep0 rowsep0" align="left">Diseases</td><td class="colsep0 rowsep0" align="left">Modeling the protein–protein interaction network</td><td class="colsep0 rowsep0" align="left">1. Cacybp expressions in different grades of gliomas have no significant difference. 2. Cacybp expressions have no significant association with the prognosis of low-grade glioma and glioblastoma.</td><td class="colsep0 rowsep0" align="left">Prognosis of low-grade glioma and glioblastoma</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr></tbody></table></div></div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Network Approaches for Autophagy/Autophagic Mechanisms</h3><div class="NLM_p last">A novel network analysis strategy of hybrid yeast-human has been performed and revealed that FNBP1L is essential in antibacterial autophagy. To derive functional biological information by applying network-based biology, integrated genomics has been utilized for novel molecule discovery in autophagy.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In addition, taking regulatory network of autophagy as a systems-level bioinformatics resource has been reported for exploring the regulation and mechanism of autophagy. The known and predicted regulators may play important role in pharmacological attempts against neurodegenerative diseases and cancer.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Furthermore, protein–protein interactions within the programmed cell death network have been discovered by protein-fragment complementation screening.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The <i>Gaussia</i> luciferase protein fragment complementation assay (GLuc PCA) has been utilized to measure the global profile of these interactions. The binding status between proteins fused to the complementary fragments of a luciferase reporter was monitored. Thus, the GLuc PCA platform can be a useful approach for the discovery of biochemical pathways of the cell death network. Moreover, the disease–gene–drug network is a significant strategy for the application of natural products in disease therapy. A disease–gene–drug network was constructed based on human protein reference database and the Gene Expression Omnibus database. Then, 30 herbal monomers were screened that could regulate 7 autophagic genes simultaneously, indicating their potential role in autophagy.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Network Approaches for Autophagy-Related Diseases</h3><div class="NLM_p last">Recently, the microRNA-regulated autophagic pathways have been identified in plant lectin-induced cancer cell death. Nine hub proteins and 13 relevant targeted miRNAs were identified in human breast carcinoma MCF-7 cells.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Additionally, a novel caspase/autophagy-related gene switch to determine the fate of breast cancer cells has been identified.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Three molecular switches, including mitogen-activated protein kinase 3 (MAPK3), serine/threonine-protein kinase PAK-1 (PAK1), and the androgen receptor, were identified between Atgs and certain caspases in MCF-7 cells. In addition, ULK1 has been identified as a potential biomarker that is involved in miR-595 and miR-4487 regulation in SH-SY5Y cells, which provides a light to explore ULK1 and its target miRNAs in future PD therapy.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Recently, a potential Atg4B agonist (termed as flubendazole), which could induce autophagy in breast cancer, has been discovered based on systems biology technology,.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Moreover, a bioinformatic analysis of Cacybp-associated proteins by using human glioma databases has been reported in a recent study. It identifies 121 differentially expressed genes, including their biological processes, protein–protein interaction (PPI) network, and molecular functions.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Taken together, systems-biology network methods can help explore complicated autophagic mechanisms for further drug discovery but still require additional experimental supports.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Currently, despite the great advances that have been achieved by numerous experimental methods, many complicated problems in the mechanisms of autophagy remain unresolved. In addition to the aforementioned experimental approaches, a series of databases and Web servers, mathematical models, “omics” techniques, systems-biology networks, as well as other approaches related to autophagy have become available online, which may be collectively termed as in silico autophagy methods that are key supplements to experimental methods in medicinal research (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). However, compared with a wealth of experimental methods, the development of in silico autophagy approaches still remains in its infancy. Therefore, with the rapid progress of autophagy research, we believe that more new emerging in silico methods will be developed to solve the important problems of autophagy, which may be combined with experimental methods to decipher the rules of autophagy for the future pharmaceutical applications and therapeutics.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/medium/jm-2018-01673b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic model of in silico autophagy methods for expediting the current medicinal research. A variety of in silico autophagy approaches for expediting medicinal research are classified as follows: (A) databases and Web servers, (B) mathematics models, (C) omics approaches, and (D) systems-biology network approaches.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01673" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Liu</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory
of Biotherapy and Cancer Center and Department of Gastrointestinal
Surgery, West China Hospital, Sichuan University, and Collaborative
Innovation Center for Biotherapy, Chengdu 610041, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3900-9486" title="Orcid link">http://orcid.org/0000-0003-3900-9486</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1975706c7b762b2d292959282f2a377a7674"><span class="__cf_email__" data-cfemail="e08c8995828fd2d4d0d0a0d1d6d3ce838f8d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi Chen</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory
of Biotherapy and Cancer Center and Department of Gastrointestinal
Surgery, West China Hospital, Sichuan University, and Collaborative
Innovation Center for Biotherapy, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guan Wang</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory
of Biotherapy and Cancer Center and Department of Gastrointestinal
Surgery, West China Hospital, Sichuan University, and Collaborative
Innovation Center for Biotherapy, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haoyang Cai</span> - <span class="hlFld-Affiliation affiliation">Center
of Growth, Metabolism, and Aging, Key Laboratory of Bio-Resources
and Eco-Environment, College of Life Sciences, Sichuan University, Chengdu 610064, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yang Sun</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Pharmaceutical Biotechnology, Department of Biotechnology
and Pharmaceutical Sciences, School of Life Sciences, Nanjing University, Nanjing 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liang Ouyang</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory
of Biotherapy and Cancer Center and Department of Gastrointestinal
Surgery, West China Hospital, Sichuan University, and Collaborative
Innovation Center for Biotherapy, Chengdu 610041, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5537-8834" title="Orcid link">http://orcid.org/0000-0001-5537-8834</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.C., G.W., and H.C. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01780" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Yi Chen</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=BIO-d7e1534-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yi Chen</b> is an Associate Professor at State Key Laboratory of Biotherapy and Cancer Center of West China Hospital at Sichuan University. His research interest focuses on target identification and structure-based drug design.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Guan Wang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=BIO-d7e1539-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Guan Wang</b> is an Assistant Professor at State Key Laboratory of Biotherapy and Cancer Center of West China Hospital at Sichuan University. His research interest focuses on targeted drug design and discovery.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Haoyang Cai</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=BIO-d7e1544-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Haoyang Cai</b> is a Professor at School of Life Sciences at Sichuan University. His research interest focuses on cancer genomics, bioinformatics, and drug target identification.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Yang Sun</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=BIO-d7e1549-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yang Sun</b> is a Professor at State Key Laboratory of Pharmaceutical Biotechnology at Nanjing University. His research interest focuses on cancer immunity and molecular pharmacology.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Liang Ouyang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=BIO-d7e1554-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Liang Ouyang</b> received his Ph.D. degree in Medicinal Chemistry from West China School of Pharmacy, Sichuan University, in 2010. He began his academic career in 2012 at the State Key Lab of Biotherapy of West China Hospital at Sichuan University as a professor. His research interests include identification of novel drug targets in cell death and structure-based discovery of small-molecule and peptide drugs.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Bo Liu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=BIO-d7e1559-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Bo Liu</b> received his Ph.D. degree in Bioinformatics (drug design) from School of Life Sciences, Sichuan University, in 2010. In 2012, he joined the faculty of State Key Laboratory of Biotherapy of West China Hospital at Sichuan University as a professor. His research interests include autophagic target identification and structure-based drug design.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63752" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63752" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from National Key R&D Program of China (Grant  2017YFC0909301 and Grant 2017YFC0909302) and National Natural Science Foundation of China (Grants 91853109, 81673290, 81602130, and 81673455).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ABM</td><td class="NLM_def"><p class="first last">agent-based model</p></td></tr><tr><td class="NLM_term">ACDB</td><td class="NLM_def"><p class="first last">autophagic compound database</p></td></tr><tr><td class="NLM_term">ACTP</td><td class="NLM_def"><p class="first last">autophagic compound-target prediction</p></td></tr><tr><td class="NLM_term">Atg</td><td class="NLM_def"><p class="first last">autophagy-related</p></td></tr><tr><td class="NLM_term">AutomiRDB</td><td class="NLM_def"><p class="first last">autophagy-microRNA database</p></td></tr><tr><td class="NLM_term">AutopahgySMDB</td><td class="NLM_def"><p class="first last">autophagy small molecule database</p></td></tr><tr><td class="NLM_term">eEF2K</td><td class="NLM_def"><p class="first last">eukaryotic elongation factor 2 kinase</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">endoplasmic reticulum</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">GAMDB</td><td class="NLM_def"><p class="first last">gerontology-autophagic-microRNA database</p></td></tr><tr><td class="NLM_term">GLuc PCA</td><td class="NLM_def"><p class="first last"><i>Gaussia</i> luciferase protein fragment complementation assay</p></td></tr><tr><td class="NLM_term">HADb</td><td class="NLM_def"><p class="first last">human autophagy database</p></td></tr><tr><td class="NLM_term">HAMdb</td><td class="NLM_def"><p class="first last">human autophagy modulator database</p></td></tr><tr><td class="NLM_term">hfAIM</td><td class="NLM_def"><p class="first last">high fidelity Atg8 interacting motif</p></td></tr><tr><td class="NLM_term">miRNA</td><td class="NLM_def"><p class="first last">microRNA</p></td></tr><tr><td class="NLM_term">LC3</td><td class="NLM_def"><p class="first last">microtubule-associated protein 1 light chain 3</p></td></tr><tr><td class="NLM_term">iLIR</td><td class="NLM_def"><p class="first last">LC3-interacting region database</p></td></tr><tr><td class="NLM_term">LIRCP</td><td class="NLM_def"><p class="first last">LC3-interacting region-containing protein</p></td></tr><tr><td class="NLM_term">MAPK3</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase 3</p></td></tr><tr><td class="NLM_term">ncRNA</td><td class="NLM_def"><p class="first last">noncoding RNA</p></td></tr><tr><td class="NLM_term">PAK1</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase PAK-1</p></td></tr><tr><td class="NLM_term">PCD</td><td class="NLM_def"><p class="first last">programmed cell death</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">ProSelection</td><td class="NLM_def"><p class="first last">protein structure selection</p></td></tr><tr><td class="NLM_term">RNA</td><td class="NLM_def"><p class="first last">ribonucleic acid</p></td></tr><tr><td class="NLM_term">SNP</td><td class="NLM_def"><p class="first last">single-nucleotide polymorphism</p></td></tr><tr><td class="NLM_term">TCGA</td><td class="NLM_def"><p class="first last">the cancer genome atlas</p></td></tr><tr><td class="NLM_term">THANATOS</td><td class="NLM_def"><p class="first last">the autophagy necrosis, apoptosis orchestrators</p></td></tr><tr><td class="NLM_term">TMA</td><td class="NLM_def"><p class="first last">tissue microarray</p></td></tr><tr><td class="NLM_term">TNBC</td><td class="NLM_def"><p class="first last">triple negative breast cancer</p></td></tr><tr><td class="NLM_term">ULK1</td><td class="NLM_def"><p class="first last">unc-51-like kinase 1</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72014" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72014" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emr, S. D.</span></span> <span> </span><span class="NLM_article-title">Autophagy as a regulated pathway of cellular degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">1717</span>– <span class="NLM_lpage">1721</span>, <span class="refDoi"> DOI: 10.1126/science.290.5497.1717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1126%2Fscience.290.5497.1717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=11099404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVOitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2000&pages=1717-1721&author=D.+J.+Klionskyauthor=S.+D.+Emr&title=Autophagy+as+a+regulated+pathway+of+cellular+degradation&doi=10.1126%2Fscience.290.5497.1717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Review: Cell biology: Autophagy as a regulated pathway of cellular degradation</span></div><div class="casAuthors">Klionsky, Daniel J.; Emr, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">5497</span>),
    <span class="NLM_cas:pages">1717-1721</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review, with 83 refs.  Macroautophagy is a dynamic process involving the rearrangement of subcellular membranes to sequester cytoplasm and organelles for delivery to the lysosome or vacuole where the sequestered cargo is degraded and recycled.  This process takes place in all eukaryotic cells.  It is highly regulated through the action of various kinases, phosphatases, and GTPases.  The core protein machinery that is necessary to drive formation and consumption of intermediates in the macroautophagy pathway includes a ubiquitin-like protein conjugation system and a protein complex that directs membrane docking and fusion at the lysosome or vacuole.  Macroautophagy plays an important role in developmental processes, human disease, and cellular response to nutrient deprivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLO3Uce-OuRbVg90H21EOLACvtfcHk0lhZuBe7uuY0KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVOitbw%253D&md5=0ad887bd4edc4fb1c0cd45764ff6896d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.290.5497.1717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.290.5497.1717%26sid%3Dliteratum%253Aachs%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26aulast%3DEmr%26aufirst%3DS.%2BD.%26atitle%3DAutophagy%2520as%2520a%2520regulated%2520pathway%2520of%2520cellular%2520degradation%26jtitle%3DScience%26date%3D2000%26volume%3D290%26spage%3D1717%26epage%3D1721%26doi%3D10.1126%2Fscience.290.5497.1717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Autophagy: from phenomenology to molecular understanding in less than a decade</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">931</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1038/nrm2245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fnrm2245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=17712358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=931-937&author=D.+J.+Klionsky&title=Autophagy%3A+from+phenomenology+to+molecular+understanding+in+less+than+a+decade&doi=10.1038%2Fnrm2245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: from phenomenology to molecular understanding in less than a decade</span></div><div class="casAuthors">Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">931-937</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In 2000, it was suggested to the author that "autophagy will be the wave of the future: it will become the new apoptosis.".  Few people would have agreed at the time, but this statement has turned out to be prophetic, and this process of 'self-eating' has rapidly exploded as a research field, as scientists discovered connections to cancer, neurodegeneration, and even lifespan extension.  Amazingly, although autophagy was 1st discovered ∼40 yr ago, the mol. breakthroughs in autophagy have taken place during only the past decade.  A time-line covering the history of autophagy research is provided, with a focus on the key events that have led to its increasing prominence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSmR5Px0Qh87Vg90H21EOLACvtfcHk0lhZuBe7uuY0KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsL3I&md5=0498357d8b0c64add87fb82e89770572</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrm2245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2245%26sid%3Dliteratum%253Aachs%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DAutophagy%253A%2520from%2520phenomenology%2520to%2520molecular%2520understanding%2520in%2520less%2520than%2520a%2520decade%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D931%26epage%3D937%26doi%3D10.1038%2Fnrm2245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooze, S. A.</span></span> <span> </span><span class="NLM_article-title">The autophagosome: origins unknown, biogenesis complex</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">774</span>, <span class="refDoi"> DOI: 10.1038/nrm3696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fnrm3696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=24201109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=759-774&issue=12&author=C.+A.+Lambauthor=T.+Yoshimoriauthor=S.+A.+Tooze&title=The+autophagosome%3A+origins+unknown%2C+biogenesis+complex&doi=10.1038%2Fnrm3696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The autophagosome: origins unknown, biogenesis complex</span></div><div class="casAuthors">Lamb, Christopher A.; Yoshimori, Tamotsu; Tooze, Sharon A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">759-774</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Healthy cells use autophagy as a general 'housekeeping' mechanism and to survive stress, including stress induced by nutrient deprivation.  Autophagy is initiated at the isolation membrane (originally termed the phagophore), and the coordinated action of ATG (autophagy-related) proteins results in the expansion of this membrane to form the autophagosome.  Although the biogenesis of the isolation membrane and the autophagosome is complex and incompletely understood, insight has been gained into the mol. processes involved in initiating the isolation membrane, the source from which this originates (for example, it was recently proposed that the isolation membrane forms from the mitochondria-assocd. endoplasmic reticulum (ER) membrane (MAM)) and the role of ATG proteins and the vesicular trafficking machinery in autophagosome formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtLSmFqUjw0LVg90H21EOLACvtfcHk0lhKeH3sdzF-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCnur%252FJ&md5=1e0d82497a3d6c319c3a14ef1ffd98ad</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3696%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DC.%2BA.%26aulast%3DYoshimori%26aufirst%3DT.%26aulast%3DTooze%26aufirst%3DS.%2BA.%26atitle%3DThe%2520autophagosome%253A%2520origins%2520unknown%252C%2520biogenesis%2520complex%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26issue%3D12%26spage%3D759%26epage%3D774%26doi%3D10.1038%2Fnrm3696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Autophagy in the pathogenesis of disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.12.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.cell.2007.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=18191218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFehtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2008&pages=27-42&author=B.+Levineauthor=G.+Kroemer&title=Autophagy+in+the+pathogenesis+of+disease&doi=10.1016%2Fj.cell.2007.12.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in the pathogenesis of disease</span></div><div class="casAuthors">Levine, Beth; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-42</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The role of mTOR kinase-related multiple pathways as regulators of autophagy in diseases is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEJaHO5xPdGbVg90H21EOLACvtfcHk0lhKeH3sdzF-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFehtb8%253D&md5=330b3696429911990b51385919e2bd39</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DAutophagy%2520in%2520the%2520pathogenesis%2520of%2520disease%26jtitle%3DCell%26date%3D2008%26volume%3D132%26spage%3D27%26epage%3D42%26doi%3D10.1016%2Fj.cell.2007.12.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Core signaling pathways of survival/death in autophagy-related cancer networks</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1263</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2011.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.biocel.2011.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=21640844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1Olurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=1263-1266&author=S.+Y.+Wangauthor=Q.+J.+Yuauthor=R.+D.+Zhangauthor=B.+Liu&title=Core+signaling+pathways+of+survival%2Fdeath+in+autophagy-related+cancer+networks&doi=10.1016%2Fj.biocel.2011.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Core signaling pathways of survival/death in autophagy-related cancer networks</span></div><div class="casAuthors">Wang, Shu-ya; Yu, Qi-jia; Zhang, Ru-dian; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1263-1266</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Autophagy (macroautophagy), an evolutionarily conserved lysosomal degrdn. process, is implicated in a wide variety of pathol. processes including cancer.  Autophagy plays the Janus role in regulating several survival or death signaling pathways that may decide the fate of cancer cell.  Accumulating evidence has revealed the core mol. machinery of autophagy in tumor initiation and progression; however, the intricate relationships between autophagy and cancer are still in its infancy.  In this review, we summarize several key survival/death pathways such as mTOR subnetwork, Beclin 1 interactome, and p53 signaling that may play the crucial roles for the regulation of the autophagy-related cancer networks.  Therefore, a better understanding of the relationships between autophagy and cancer may ultimately allow cancer biologists and clinicians to harness core autophagic pathways for the discovery of potential novel drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroJKxaLZqWqLVg90H21EOLACvtfcHk0lhKeH3sdzF-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1Olurc%253D&md5=e4f3695908b438f07d555da9e9bc3e87</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2011.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2011.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%2BY.%26aulast%3DYu%26aufirst%3DQ.%2BJ.%26aulast%3DZhang%26aufirst%3DR.%2BD.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DCore%2520signaling%2520pathways%2520of%2520survival%252Fdeath%2520in%2520autophagy-related%2520cancer%2520networks%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2011%26volume%3D43%26spage%3D1263%26epage%3D1266%26doi%3D10.1016%2Fj.biocel.2011.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span> <span> </span><span class="NLM_article-title">Survival or death: disequilibrating the oncogenic and tumor suppressive autophagy in cancer</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">e892</span>, <span class="refDoi"> DOI: 10.1038/cddis.2013.422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fcddis.2013.422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=24176850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCrur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=e892&author=B.+Liuauthor=X.+Wenauthor=Y.+Cheng&title=Survival+or+death%3A+disequilibrating+the+oncogenic+and+tumor+suppressive+autophagy+in+cancer&doi=10.1038%2Fcddis.2013.422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Survival or death: disequilibrating the oncogenic and tumor suppressive autophagy in cancer</span></div><div class="casAuthors">Liu, B.; Wen, X.; Cheng, Y.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">e892</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Autophagy (macroautophagy) is an evolutionarily conserved lysosomal degrdn. process, in which a cell degrades long-lived proteins and damaged organelles.  Recently, accumulating evidence has revealed the core mol. machinery of autophagy in carcinogenesis; however, the intricate relationship between autophagy and cancer continue to remain an enigma.  Why does autophagy have either pro-survival (oncogenic) or pro-death (tumor suppressive) role at different cancer stages, including cancer stem cell, initiation and progression, invasion and metastasis, as well as dormancy.  How does autophagy modulate a series of oncogenic and/or tumor suppressive pathways, implicated in microRNA (miRNA) involvement.  Whether would targeting the oncogenic and tumor suppressive autophagic network be a novel strategy for drug discovery.  To address these problems, we focus on summarizing the dynamic oncogenic and tumor suppressive roles of autophagy and their relevant small-mol. drugs, which would provide a new clue to elucidate the oncosuppressive (survival or death) autophagic network as a potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2wUhKw1JqAbVg90H21EOLACvtfcHk0lhKeH3sdzF-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCrur3L&md5=1914543371d0618accc380fe95186e6c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.422%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DY.%26atitle%3DSurvival%2520or%2520death%253A%2520disequilibrating%2520the%2520oncogenic%2520and%2520tumor%2520suppressive%2520autophagy%2520in%2520cancer%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2013%26volume%3D4%26spage%3De892%26doi%3D10.1038%2Fcddis.2013.422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ylä-Anttila, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vihinen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jokitalo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eskelinen, E. L.</span></span> <span> </span><span class="NLM_article-title">Monitoring autophagy by electron microscopy in Mammalian cells</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>452</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(08)03610-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2FS0076-6879%2808%2903610-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=19200881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkt1eqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2009&pages=143-164&author=P.+Yl%C3%A4-Anttilaauthor=H.+Vihinenauthor=E.+Jokitaloauthor=E.+L.+Eskelinen&title=Monitoring+autophagy+by+electron+microscopy+in+Mammalian+cells&doi=10.1016%2FS0076-6879%2808%2903610-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring autophagy by electron microscopy in mammalian cells</span></div><div class="casAuthors">Yla-Anttila, Paivi; Vihinen, Helena; Jokitalo, Eija; Eskelinen, Eeva-Liisa</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">452</span>
        (<span class="NLM_cas:issue">Autophagy in Mammalian Systems, Part B</span>),
    <span class="NLM_cas:pages">143-164</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Electron microscopy remains one of the most accurate methods for the detection of autophagy and quantification of autophagic accumulation.  Compared to fluorescence microscopy, the resoln. of transmission electron microscopy is superior.  In this chapter we describe the fine structure of early and late autophagic compartments in mammalian cells.  Instructions are given for the prepn. of samples for conventional electron microscopy using three different protocols suitable for cultured cells and animal tissues.  We also introduce tomog. as a tool to study the three-dimensional morphol. of autophagic organelles and show the morphol. of a phagophore as an example.  Finally, we describe a protocol for the quantification of autophagic compartments by electron microscopy and point counting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgD9_Nlt8MRLVg90H21EOLACvtfcHk0ljJZuB30g2oyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkt1eqs78%253D&md5=727bdf41f134b23b61d3e9a9de9e35e4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2808%2903610-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252808%252903610-0%26sid%3Dliteratum%253Aachs%26aulast%3DYl%25C3%25A4-Anttila%26aufirst%3DP.%26aulast%3DVihinen%26aufirst%3DH.%26aulast%3DJokitalo%26aufirst%3DE.%26aulast%3DEskelinen%26aufirst%3DE.%2BL.%26atitle%3DMonitoring%2520autophagy%2520by%2520electron%2520microscopy%2520in%2520Mammalian%2520cells%26jtitle%3DMethods%2520Enzymol.%26date%3D2009%26volume%3D452%26spage%3D143%26epage%3D164%26doi%3D10.1016%2FS0076-6879%2808%2903610-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span> <span> </span><span class="NLM_article-title">Deconvoluting the role of reactive oxygen species and autophagy in human diseases</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">402</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2013.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.freeradbiomed.2013.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23872397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFylurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2013&pages=402-410&issue=6&author=X.+Wenauthor=J.+M.+Wuauthor=F.+T.+Wangauthor=B.+Liuauthor=C.+Huangauthor=Y.+Wei&title=Deconvoluting+the+role+of+reactive+oxygen+species+and+autophagy+in+human+diseases&doi=10.1016%2Fj.freeradbiomed.2013.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Deconvoluting the role of reactive oxygen species and autophagy in human diseases</span></div><div class="casAuthors">Wen, Xin; Wu, Jinming; Wang, Fengtian; Liu, Bo; Huang, Canhua; Wei, Yuquan</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">402-410</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Reactive oxygen species (ROS), chem. reactive mols. contg. oxygen, can form as a natural byproduct of the normal metab. of oxygen and also have their crucial roles in cell homeostasis.  Of note, the major intracellular sources including mitochondria, endoplasmic reticulum (ER), peroxisomes and the NADPH oxidase (NOX) complex have been identified in cell membranes to produce ROS.  Interestingly, autophagy, an evolutionarily conserved lysosomal degrdn. process in which a cell degrades long-lived proteins and damaged organelles, has recently been well-characterized to be regulated by different types of ROS.  Accumulating evidence has demonstrated that ROS-modulated autophagy has numerous links to a no. of pathol. processes, including cancer, aging, neurodegenerative diseases, type-II diabetes, cardiovascular diseases, muscular disorders, hepatic encephalopathy and immunity diseases.  In this review, we focus on summarizing the mol. mechanisms of ROS-regulated autophagy and their relevance to diverse diseases, which would shed new light on more ROS modulators as potential therapeutic drugs for fighting human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMLM_rtLoDZLVg90H21EOLACvtfcHk0ljJZuB30g2oyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFylurvJ&md5=bb75bd646e83e92fdd4d183e29f83329</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2013.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2013.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DF.%2BT.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DY.%26atitle%3DDeconvoluting%2520the%2520role%2520of%2520reactive%2520oxygen%2520species%2520and%2520autophagy%2520in%2520human%2520diseases%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2013%26volume%3D65%26issue%3D6%26spage%3D402%26epage%3D410%26doi%3D10.1016%2Fj.freeradbiomed.2013.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span> <span> </span><span class="NLM_article-title">Autophagy and metabolism</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span>,  <span class="NLM_fpage">1344</span>– <span class="NLM_lpage">1348</span>, <span class="refDoi"> DOI: 10.1126/science.1193497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1126%2Fscience.1193497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=21127245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=2010&pages=1344-1348&author=J.+D.+Rabinowitzauthor=E.+White&title=Autophagy+and+metabolism&doi=10.1126%2Fscience.1193497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy and metabolism</span></div><div class="casAuthors">Rabinowitz, Joshua D.; White, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6009</span>),
    <span class="NLM_cas:pages">1344-1348</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Autophagy is a process of self-cannibalization.  Cells capture their own cytoplasm and organelles and consume them in lysosomes.  The resulting breakdown products are inputs to cellular metab., through which they are used to generate energy and to build new proteins and membranes.  Autophagy preserves the health of cells and tissues by replacing outdated and damaged cellular components with fresh ones.  In starvation, it provides an internal source of nutrients for energy generation and, thus, survival.  A powerful promoter of metabolic homeostasis at both the cellular and whole-animal level, autophagy prevents degenerative diseases.  It does have a downside, however; cancer cells exploit it to survive in nutrient-poor tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREWU_wFW6GLVg90H21EOLACvtfcHk0ljJZuB30g2oyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsbbE&md5=4b6ed3a623cbef8df1241579e15632fa</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1193497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1193497%26sid%3Dliteratum%253Aachs%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DWhite%26aufirst%3DE.%26atitle%3DAutophagy%2520and%2520metabolism%26jtitle%3DScience%26date%3D2010%26volume%3D330%26spage%3D1344%26epage%3D1348%26doi%3D10.1126%2Fscience.1193497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Eaten alive: a history of macroautophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">814</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fncb0910-814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=20811353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=814-822&author=Z.+Yangauthor=D.+J.+Klionsky&title=Eaten+alive%3A+a+history+of+macroautophagy&doi=10.1038%2Fncb0910-814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Eaten alive: a history of macroautophagy</span></div><div class="casAuthors">Yang, Zhifen; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">814-822</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter autophagy), or 'self-eating', is a conserved cellular pathway that controls protein and organelle degrdn., and has essential roles in survival, development and homeostasis.  Autophagy is also integral to human health and is involved in physiol., development, lifespan and a wide range of diseases, including cancer, neurodegeneration and microbial infection.  Although research on this topic began in the late 1950s, substantial progress in the mol. study of autophagy has taken place during only the past 15 years.  This review traces the key findings that led to our current mol. understanding of this complex process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_iZJ3YUoHrVg90H21EOLACvtfcHk0lhQCJs1JHAATQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE&md5=a20abac7db3f20eac668c41052413397</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-814%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DEaten%2520alive%253A%2520a%2520history%2520of%2520macroautophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D814%26epage%3D822%26doi%3D10.1038%2Fncb0910-814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Biological functions of autophagy genes: a disease perspective</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>176</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.cell.2018.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=30633901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFyquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2019&pages=11-42&author=B.+Levineauthor=G.+Kroemer&title=Biological+functions+of+autophagy+genes%3A+a+disease+perspective&doi=10.1016%2Fj.cell.2018.09.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Biological Functions of Autophagy Genes: A Disease Perspective</span></div><div class="casAuthors">Levine, Beth; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">11-42</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The lysosomal degrdn. pathway of autophagy plays a fundamental role in cellular, tissue, and organismal homeostasis and is mediated by evolutionarily conserved autophagy-related (ATG) genes.  Definitive etiol. links exist between mutations in genes that control autophagy and human disease, esp. neurodegenerative, inflammatory disorders and cancer.  Autophagy selectively targets dysfunctional organelles, intracellular microbes, and pathogenic proteins, and deficiencies in these processes may lead to disease.  Moreover, ATG genes have diverse physiol. important roles in other membrane-trafficking and signaling pathways.  This Review discusses the biol. functions of autophagy genes from the perspective of understanding - and potentially reversing - the pathophysiol. of human disease and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUer8pL-luJrVg90H21EOLACvtfcHk0lhQCJs1JHAATQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFyquw%253D%253D&md5=65c2303ca9ea78c00cae26a3ff5a9b73</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DBiological%2520functions%2520of%2520autophagy%2520genes%253A%2520a%2520disease%2520perspective%26jtitle%3DCell%26date%3D2019%26volume%3D176%26spage%3D11%26epage%3D42%26doi%3D10.1016%2Fj.cell.2018.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of autophagy and relevant small-molecule compounds for targeted cancer therapy</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1803</span>– <span class="NLM_lpage">1826</span>, <span class="refDoi"> DOI: 10.1007/s00018-018-2759-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1007%2Fs00018-018-2759-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=29417176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2018&pages=1803-1826&issue=10&author=J.+Zhangauthor=G.+Wangauthor=Y.+Zhouauthor=Y.+Chenauthor=L.+Ouyangauthor=B.+Liu&title=Mechanisms+of+autophagy+and+relevant+small-molecule+compounds+for+targeted+cancer+therapy&doi=10.1007%2Fs00018-018-2759-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of autophagy and relevant small-molecule compounds for targeted cancer therapy</span></div><div class="casAuthors">Zhang, Jin; Wang, Guan; Zhou, Yuxin; Chen, Yi; Ouyang, Liang; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1803-1826</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Autophagy is an evolutionarily conserved, multi-step lysosomal degrdn. process for the clearance of damaged or superfluous proteins and organelles.  Accumulating studies have recently revealed that autophagy is closely related to a variety of types of cancer; however, elucidation of its Janus role of either tumor-suppressive or tumor-promoting still remains to be discovered.  In this review, we focus on summarizing the context-dependent role of autophagy and its complicated mol. mechanisms in different types of cancer.  Moreover, we discuss a series of small-mol. compds. targeting autophagy-related proteins or the autophagic process for potential cancer therapy.  Taken together, these findings would shed new light on exploiting the intricate mechanisms of autophagy and relevant small-mol. compds. as potential anti-cancer drugs to improve targeted cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolGggPdyfsHrVg90H21EOLACvtfcHk0lhQCJs1JHAATQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKjtbY%253D&md5=7efbbc0b553a8843dba97c8d341a4853</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00018-018-2759-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-018-2759-2%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DMechanisms%2520of%2520autophagy%2520and%2520relevant%2520small-molecule%2520compounds%2520for%2520targeted%2520cancer%2520therapy%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2018%26volume%3D75%26issue%3D10%26spage%3D1803%26epage%3D1826%26doi%3D10.1007%2Fs00018-018-2759-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryter, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span> <span> </span><span class="NLM_article-title">Autophagy in human health and disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>368</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1205406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1056%2FNEJMra1205406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23406030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivVWjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2013&pages=651-662&issue=7&author=A.+M.+Choiauthor=S.+W.+Ryterauthor=B.+Levine&title=Autophagy+in+human+health+and+disease&doi=10.1056%2FNEJMra1205406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in human health and disease</span></div><div class="casAuthors">Choi, Augustine M. K.; Ryter, Stefan W.; Levine, Beth</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">651-662</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  Autophagy is now widely implicated in pathophysiol. processes (e.g., cancer, metabolic and neurodegenerative disorders, and cardiovascular and pulmonary diseases) and in physiol. responses to exercise and aging.  This article discusses on the autophagy in human health and disease.  An improved understanding of the mechanism (or mechanisms) by which autophagy can prevent pathogenesis may lead to the identification of new targets for both diagnostic and therapeutic approaches.  Drug screening for agonists or antagonists of autophagic activity, including upstream regulators and downstream targets of autophagy, may yield addnl. therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP5LjXnUJzhrVg90H21EOLACvtfcHk0lhQCJs1JHAATQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivVWjsr0%253D&md5=be5f49522caa165d5bf18b4b50edfeec</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1205406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1205406%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DA.%2BM.%26aulast%3DRyter%26aufirst%3DS.%2BW.%26aulast%3DLevine%26aufirst%3DB.%26atitle%3DAutophagy%2520in%2520human%2520health%2520and%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D368%26issue%3D7%26spage%3D651%26epage%3D662%26doi%3D10.1056%2FNEJMra1205406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Homma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, H.</span></span> <span> </span><span class="NLM_article-title">The autophagy database: an all-inclusive information resource on autophagy that provides nourishment for research</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D986</span>– <span class="NLM_lpage">D990</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1093%2Fnar%2Fgkq995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=20972215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2lurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D986-D990&author=K.+Hommaauthor=K.+Suzukiauthor=H.+Sugawara&title=The+autophagy+database%3A+an+all-inclusive+information+resource+on+autophagy+that+provides+nourishment+for+research&doi=10.1093%2Fnar%2Fgkq995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Autophagy Database: an all-inclusive information resource on autophagy that provides nourishment for research</span></div><div class="casAuthors">Homma, Keiichi; Suzuki, Koji; Sugawara, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D986-D990</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Autophagy is a process of self-digestion generally obsd. in eukaryotes and has been shown to play crucial roles for survival under starvation and removal of deleterious substances.  Despite great advances that have been made, many problems in mechanisms of autophagy remain unsolved.  As a large no. of autophagy-related proteins are identified in each species, a database that collects data, identifies their homologs in other species and makes them available will contribute to research advancement.  As no such resources exist, we built the Autophagy database (http://tp-apg.genes.nig.ac.jp/autophagy) to provide basics, up-to-date information on relevant literature, and a list of autophagy-related proteins and their homologs in 41 eukaryotes.  From the database, the user can search for proteins by keywords or sequences to obtain a wealth of data including functional and structural information and find possible functional homologs of proteins whose functions have been demonstrated in other species.  As proteins that bind the phospholipid, phosphatidyl inositol 3-phosphate (PI3P) are essential for autophagy to proceed, we carried out an original anal. to identify probable PI3P-binding proteins, and made the list available from the database.  The database is expected to give impetus to further research on autophagy by providing basic and specialized data on the subject.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-YW_gAyhH7Vg90H21EOLACvtfcHk0lgoS-SG85brTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2lurc%253D&md5=8663738874465b0ea579bedfeb970da6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq995%26sid%3Dliteratum%253Aachs%26aulast%3DHomma%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DSugawara%26aufirst%3DH.%26atitle%3DThe%2520autophagy%2520database%253A%2520an%2520all-inclusive%2520information%2520resource%2520on%2520autophagy%2520that%2520provides%2520nourishment%2520for%2520research%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD986%26epage%3DD990%26doi%3D10.1093%2Fnar%2Fgkq995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacomin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samavedam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Promponas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nezis, I. P.</span></span> <span> </span><span class="NLM_article-title">iLIR database: a web resource for LIR motif-containing proteins in eukaryotes</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1945</span>– <span class="NLM_lpage">1953</span>, <span class="refDoi"> DOI: 10.1080/15548627.2016.1207016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2016.1207016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27484196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1945-1953&issue=10&author=A.+C.+Jacominauthor=S.+Samavedamauthor=V.+Promponasauthor=I.+P.+Nezis&title=iLIR+database%3A+a+web+resource+for+LIR+motif-containing+proteins+in+eukaryotes&doi=10.1080%2F15548627.2016.1207016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">iLIR database: A web resource for LIR motif-containing proteins in eukaryotes</span></div><div class="casAuthors">Jacomin, Anne-Claire; Samavedam, Siva; Promponas, Vasilis; Nezis, Ioannis P.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1945-1953</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Atg8-family proteins are the best-studied proteins of the core autophagic machinery.  They are essential for the elongation and closure of the phagophore into a proper autophagosome.  Moreover, Atg8-family proteins are assocd. with the phagophore from the initiation of the autophagic process to, or just prior to, the fusion between autophagosomes with lysosomes.  In addn. to their implication in autophagosome biogenesis, they are crucial for selective autophagy through their ability to interact with selective autophagy receptor proteins necessary for the specific targeting of substrates for autophagic degrdn.  In the past few years it has been revealed that Atg8-interacting proteins include not only receptors but also components of the core autophagic machinery, proteins assocd. with vesicles and their transport, and specific proteins that are selectively degraded by autophagy.  Atg8-interacting proteins contain a short linear LC3-interacting region/LC3 recognition sequence/Atg8-interacting motif (LIR/LRS/AIM) motif which is responsible for their interaction with Atg8-family proteins.  These proteins are referred to as LIR-contg. proteins (LIRCPs).  So far, many exptl. efforts have been carried out to identify new LIRCPs, leading to the characterization of some of them in the past 10 years.  Given the need for the identification of LIRCPs in various organisms, we developed the iLIR database (https://ilir.warwick.ac.uk) as a freely available web resource, listing all the putative canonical LIRCPs identified in silico in the proteomes of 8 model organisms using the iLIR server, combined with a Gene Ontol. (GO) term anal.  Addnl., a curated text-mining anal. of the literature permitted us to identify novel putative LICRPs in mammals that have not previously been assocd. with autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0mr_HkTMOoLVg90H21EOLACvtfcHk0lgoS-SG85brTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnur3F&md5=0fa319f294097a9d96ef2e9caa950b09</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F15548627.2016.1207016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2016.1207016%26sid%3Dliteratum%253Aachs%26aulast%3DJacomin%26aufirst%3DA.%2BC.%26aulast%3DSamavedam%26aufirst%3DS.%26aulast%3DPromponas%26aufirst%3DV.%26aulast%3DNezis%26aufirst%3DI.%2BP.%26atitle%3DiLIR%2520database%253A%2520a%2520web%2520resource%2520for%2520LIR%2520motif-containing%2520proteins%2520in%2520eukaryotes%26jtitle%3DAutophagy%26date%3D2016%26volume%3D12%26issue%3D10%26spage%3D1945%26epage%3D1953%26doi%3D10.1080%2F15548627.2016.1207016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzfadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weithorn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peled-Zehavi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Parys, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van de Peer, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galili, G.</span></span> <span> </span><span class="NLM_article-title">hfAIM: a reliable bioinformatics approach for in silico genome-wide identification of autophagy-associated Atg8-interacting motifs in vatious organisms</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">887</span>, <span class="refDoi"> DOI: 10.1080/15548627.2016.1147668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2016.1147668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27071037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1eis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=876-887&issue=5&author=Q.+Xieauthor=O.+Tzfadiaauthor=M.+Levyauthor=E.+Weithornauthor=H.+Peled-Zehaviauthor=T.+Van+Parysauthor=Y.+Van+de+Peerauthor=G.+Galili&title=hfAIM%3A+a+reliable+bioinformatics+approach+for+in+silico+genome-wide+identification+of+autophagy-associated+Atg8-interacting+motifs+in+vatious+organisms&doi=10.1080%2F15548627.2016.1147668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">hfAIM: A reliable bioinformatics approach for in silico genome-wide identification of autophagy-associated Atg8-interacting motifs in various organisms</span></div><div class="casAuthors">Xie, Qingjun; Tzfadia, Oren; Levy, Matan; Weithorn, Efrat; Peled-Zehavi, Hadas; Van Parys, Thomas; Van de Peer, Yves; Galili, Gad</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">876-887</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Most of the proteins that are specifically turned over by selective autophagy are recognized by the presence of short Atg8 interacting motifs (AIMs) that facilitate their assocn. with the autophagy app.  Such AIMs can be identified by bioinformatics methods based on their defined degenerate consensus F/W/Y-X-X-L/I/V sequences in which X represents any amino acid.  Achieving reliability and/or fidelity of the prediction of such AIMs on a genome-wide scale represents a major challenge.  Here, we present a bioinformatics approach, high fidelity AIM (hfAIM), which uses addnl. sequence requirements-the presence of acidic amino acids and the absence of pos. charged amino acids in certain positions-to reliably identify AIMs in proteins.  We demonstrate that the use of the hfAIM method allows for in silico high fidelity prediction of AIMs in AIM-contg. proteins (ACPs) on a genome-wide scale in various organisms.  Furthermore, by using hfAIM to identify putative AIMs in the Arabidopsis proteome, we illustrate a potential contribution of selective autophagy to various biol. processes.  More specifically, we identified 9 peroxisomal PEX proteins that contain hfAIM motifs, among which AtPEX1, AtPEX6 and AtPEX10 possess evolutionary-conserved AIMs.  Bimol. fluorescence complementation (BiFC) results verified that AtPEX6 and AtPEX10 indeed interact with Atg8 in planta.  In addn., we show that mutations occurring within or nearby hfAIMs in PEX1, PEX6 and PEX10 caused defects in the growth and development of various organisms.  Taken together, the above results suggest that the hfAIM tool can be used to effectively perform genome-wide in silico screens of proteins that are potentially regulated by selective autophagy.  The hfAIM system is a web tool that can be accessed at link: http://bioinformatics.psb.ugent.be/hfAIM/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2DSVeo6k6SrVg90H21EOLACvtfcHk0lgoS-SG85brTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1eis7k%253D&md5=68574d1ab076e5ccb519d87898140c72</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F15548627.2016.1147668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2016.1147668%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DTzfadia%26aufirst%3DO.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DWeithorn%26aufirst%3DE.%26aulast%3DPeled-Zehavi%26aufirst%3DH.%26aulast%3DVan%2BParys%26aufirst%3DT.%26aulast%3DVan%2Bde%2BPeer%26aufirst%3DY.%26aulast%3DGalili%26aufirst%3DG.%26atitle%3DhfAIM%253A%2520a%2520reliable%2520bioinformatics%2520approach%2520for%2520in%2520silico%2520genome-wide%2520identification%2520of%2520autophagy-associated%2520Atg8-interacting%2520motifs%2520in%2520vatious%2520organisms%26jtitle%3DAutophagy%26date%3D2016%26volume%3D12%26issue%3D5%26spage%3D876%26epage%3D887%26doi%3D10.1080%2F15548627.2016.1147668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moussay, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaoma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baginska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Moer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazarov, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chouaib, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berchem, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janji, B.</span></span> <span> </span><span class="NLM_article-title">The acquisition of resistance to TNFα in breast cancer cells is associated with constitutive activation of autophagy as revealed by a transcriptome analysis using a custom microarray</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">770</span>, <span class="refDoi"> DOI: 10.4161/auto.7.7.15454</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4161%2Fauto.7.7.15454" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=21490427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=760-770&issue=7&author=E.+Moussayauthor=T.+Kaomaauthor=J.+Baginskaauthor=A.+Mullerauthor=K.+Van+Moerauthor=N.+Nicotauthor=P.+V.+Nazarovauthor=L.+Vallarauthor=S.+Chouaibauthor=G.+Berchemauthor=B.+Janji&title=The+acquisition+of+resistance+to+TNF%CE%B1+in+breast+cancer+cells+is+associated+with+constitutive+activation+of+autophagy+as+revealed+by+a+transcriptome+analysis+using+a+custom+microarray&doi=10.4161%2Fauto.7.7.15454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The acquisition of resistance to TNFα in breast cancer cells is associated with constitutive activation of autophagy as revealed by a transcriptome analysis using a custom microarray</span></div><div class="casAuthors">Moussay, Etienne; Kaoma, Tony; Baginska, Joanna; Muller, Arnaud; Van Moer, Kris; Nicot, Nathalie; Nazarov, Petr V.; Vallar, Laurent; Chouaib, Salem; Berchem, Guy; Janji, Bassam</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">760-770</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">While the autophagic process is mainly regulated at the post-translational level, a growing body of evidence suggests that autophagy might also be regulated at the transcriptional level.  The identification of transcription factors involved in the regulation of autophagy genes has provided compelling evidence for such regulation.  In this context, a powerful high throughput anal. tool to simultaneously monitor the expression level of autophagy genes is urgently needed.  Here we describe setting up the first comprehensive human autophagy database (HADb, available at www.autophagy.lu) and the development of a companion Human Autophagy-dedicated cDNA Microarray which comprises 234 genes involved in or related to autophagy.  The autophagy microarray tool used on breast adenocarcinoma MCF-7 cell line allowed the identification of 47 differentially expressed autophagy genes assocd. with the acquisition of resistance to the cytotoxic effect of TNFα.  The autophagy-core machinery genes DRAM (Damage-Regulated Autophagy Modulator), BNIP3L (BCL2/adenovirus E1B 19 kDa interacting protein 3-like), BECN1 (Beclin 1), GABARAP (Gamma-AminoButyric Acid Receptor-Assocd. Protein) and UVRAG (UV radiation resistance assocd. gene) were found upregulated in TNF-resistant cells, suggesting a constitutive activation of the autophagy machinery in these cells.  More interestingly, we identified NPC1 as the most upregulated genes in TNF-resistant compared to TNF-sensitive MCF-7 cells, suggesting a relation between the intracellular transport of cholesterol, the regulation of autophagy and NPC1 expression in TNF-resistant tumor cells.  In conclusion, we describe here new tools that may help investigating autophagy gene regulation in various cellular models and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlFmjiGdXKrLVg90H21EOLACvtfcHk0lhP6XpnmCs-Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCisLc%253D&md5=16673c0bced2a094c4e9bde06bea7e7a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4161%2Fauto.7.7.15454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.7.7.15454%26sid%3Dliteratum%253Aachs%26aulast%3DMoussay%26aufirst%3DE.%26aulast%3DKaoma%26aufirst%3DT.%26aulast%3DBaginska%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DA.%26aulast%3DVan%2BMoer%26aufirst%3DK.%26aulast%3DNicot%26aufirst%3DN.%26aulast%3DNazarov%26aufirst%3DP.%2BV.%26aulast%3DVallar%26aufirst%3DL.%26aulast%3DChouaib%26aufirst%3DS.%26aulast%3DBerchem%26aufirst%3DG.%26aulast%3DJanji%26aufirst%3DB.%26atitle%3DThe%2520acquisition%2520of%2520resistance%2520to%2520TNF%25CE%25B1%2520in%2520breast%2520cancer%2520cells%2520is%2520associated%2520with%2520constitutive%2520activation%2520of%2520autophagy%2520as%2520revealed%2520by%2520a%2520transcriptome%2520analysis%2520using%2520a%2520custom%2520microarray%26jtitle%3DAutophagy%26date%3D2011%26volume%3D7%26issue%3D7%26spage%3D760%26epage%3D770%26doi%3D10.4161%2Fauto.7.7.15454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span> <span> </span><span class="NLM_article-title">GAMDB: a web resource to connect microRNAs with autophagy in gerontology</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.1111/cpr.12247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fcpr.12247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27037912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28XlslSqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=246-251&issue=2&author=L.+Zhangauthor=T.+Xieauthor=M.+Tianauthor=J.+Liauthor=S.+Songauthor=L.+Ouyangauthor=B.+Liuauthor=H.+Cai&title=GAMDB%3A+a+web+resource+to+connect+microRNAs+with+autophagy+in+gerontology&doi=10.1111%2Fcpr.12247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">GAMDB: a web resource to connect microRNAs with autophagy in gerontology</span></div><div class="casAuthors">Zhang, Lan; Xie, Tao; Tian, Mao; Li, Jingjing; Song, Sicheng; Ouyang, Liang; Liu, Bo; Cai, Haoyang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">246-251</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Objectives : MicroRNAs (miRNAs) are endogenous ∼23 nucleotides (nt) RNAs, regulating gene expression by pairing to the mRNAs of protein-coding genes to direct their post-transcriptional repression.  Both in normal and aberrant activities, miRNAs contribute to a recurring paradigm of cellular behaviors in pathol. settings, esp. in gerontol.  Autophagy, a multi-step lysosomal degrdn. process with function to degrade long-lived proteins and damaged organelles, has significant impact on gerontol.  Thus, elucidating how miRNAs participate in autophagy may enlarge the scope of miRNA in autophagy and facilitate researches in gerontol.  Materials and methods : Herein, based upon the published studies, predicted targets and gerontol.-related diseases, we constructed a web resource named Gerontol.-Autophagic-MicroRNA Database (GAMDB) (http://gamdb.liu-lab.com/index.php), which contained 836 autophagy-related miRNAs, 197 targeted genes/proteins and 56 aging-related diseases such as Parkinson's disease, Alzheimer's disease and Huntington's disease.  Results and Conclusion : We made use of large amts. of data to elucidate the intricate relationships between microRNA-regulated autophagic mechanisms and gerontol.  This database will facilitate better understanding of autophagy regulation network in gerontol. and thus promoting gerontol.-related therapy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr39ijB6UOplrVg90H21EOLACvtfcHk0lhP6XpnmCs-Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlslSqur0%253D&md5=1714e8200b2096b5b53802eb9b22ec0b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fcpr.12247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcpr.12247%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DTian%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DCai%26aufirst%3DH.%26atitle%3DGAMDB%253A%2520a%2520web%2520resource%2520to%2520connect%2520microRNAs%2520with%2520autophagy%2520in%2520gerontology%26jtitle%3DCell%2520Proliferation%26date%3D2016%26volume%3D49%26issue%3D2%26spage%3D246%26epage%3D251%26doi%3D10.1111%2Fcpr.12247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">AutomiRDB: a web resource connecting microRNAs and autophagy in cancer</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1016</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1007/s10495-015-1133-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1007%2Fs10495-015-1133-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25931313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2MjpsFaqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1016-1017&issue=7&author=Y.+Chenauthor=J.+Huangauthor=B.+Liu&title=AutomiRDB%3A+a+web+resource+connecting+microRNAs+and+autophagy+in+cancer&doi=10.1007%2Fs10495-015-1133-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">AutomiRDB: a web resource connecting microRNAs and autophagy in cancer</span></div><div class="casAuthors">Chen Yi; Huang Jian; Liu Bo</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis : an international journal on programmed cell death</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1016-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2PofzmVRV8lf1XRtfYM0dfW6udTcc2eYGObWKuU1cY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjpsFaqsw%253D%253D&md5=a6a11f767968493805d80e5cf3343014</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs10495-015-1133-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-015-1133-1%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DAutomiRDB%253A%2520a%2520web%2520resource%2520connecting%2520microRNAs%2520and%2520autophagy%2520in%2520cancer%26jtitle%3DApoptosis%26date%3D2015%26volume%3D20%26issue%3D7%26spage%3D1016%26epage%3D1017%26doi%3D10.1007%2Fs10495-015-1133-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. H.</span></span> <span> </span><span class="NLM_article-title">miRDearhDB: a database bridging microRNAs and the programmed cell death</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1571</span>, <span class="refDoi"> DOI: 10.1038/cdd.2012.87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fcdd.2012.87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=22743998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC38jlslOqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1571&issue=9&author=J.+Xuauthor=Y.+H.+Li&title=miRDearhDB%3A+a+database+bridging+microRNAs+and+the+programmed+cell+death&doi=10.1038%2Fcdd.2012.87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">miRDeathDB: a database bridging microRNAs and the programmed cell death</span></div><div class="casAuthors">Xu J; Li Y-H</div><div class="citationInfo"><span class="NLM_cas:title">Cell death and differentiation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1571</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTPCX-rhbLpbw5nADWVYHg1fW6udTcc2eaco8af2QGTorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38jlslOqsw%253D%253D&md5=44dd905c9e2371609c35603093e1f71b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2012.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2012.87%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BH.%26atitle%3DmiRDearhDB%253A%2520a%2520database%2520bridging%2520microRNAs%2520and%2520the%2520programmed%2520cell%2520death%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2012%26volume%3D19%26issue%3D9%26spage%3D1571%26doi%3D10.1038%2Fcdd.2012.87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span> <span> </span><span class="NLM_article-title">ncRDeathDB: a comprehensive bioinformatics resource for deciphering network organization of the ncRNA-mediated cell death system</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1917</span>– <span class="NLM_lpage">1926</span>, <span class="refDoi"> DOI: 10.1080/15548627.2015.1089375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2015.1089375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26431463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWrurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1917-1926&issue=10&author=D.+Wuauthor=Y.+Huangauthor=J.+Kangauthor=K.+Liauthor=X.+Biauthor=T.+Zhangauthor=N.+Jinauthor=Y.+Huauthor=P.+Tanauthor=L.+Zhangauthor=Y.+Yiauthor=W.+Shenauthor=J.+Huangauthor=X.+Liauthor=X.+Liauthor=J.+Xuauthor=D.+Wang&title=ncRDeathDB%3A+a+comprehensive+bioinformatics+resource+for+deciphering+network+organization+of+the+ncRNA-mediated+cell+death+system&doi=10.1080%2F15548627.2015.1089375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">ncRDeathDB: A comprehensive bioinformatics resource for deciphering network organization of the ncRNA-mediated cell death system</span></div><div class="casAuthors">Wu, Deng; Huang, Yan; Kang, Juanjuan; Li, Kongning; Bi, Xiaoman; Zhang, Ting; Jin, Nana; Hu, Yongfei; Tan, Puwen; Zhang, Lu; Yi, Ying; Shen, Wenjun; Huang, Jian; Li, Xiaobo; Li, Xia; Xu, Jianzhen; Wang, Dong</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1917-1926</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Programmed cell death (PCD) is a crit. biol. process involved in many important processes, and defects in PCD have been linked with numerous human diseases.  In recent years, the protein architecture in different PCD subroutines has been explored, but our understanding of the global network organization of the noncoding RNA (ncRNA)-mediated cell death system is limited and ambiguous.  Hence, we developed the comprehensive bioinformatics resource (ncRDeathDB, www.rna-society.org/ncrdeathdb) to archive ncRNA-assocd. cell death interactions.  The current version of ncRDeathDB documents a total of more than 4600 ncRNA-mediated PCD entries in 12 species. ncRDeathDB provides a user-friendly interface to query, browse and manipulate these ncRNA-assocd. cell death interactions.  Furthermore, this resource will help to visualize and navigate current knowledge of the noncoding RNA component of cell death and autophagy, to uncover the generic organizing principles of ncRNA-assocd. cell death systems, and to generate valuable biol. hypotheses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR5dgdyv7Lf7Vg90H21EOLACvtfcHk0lgo4eOr1C-heQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWrurnN&md5=11e72bb05e906170fd4ae8d4818400da</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F15548627.2015.1089375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2015.1089375%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DBi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DYi%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DD.%26atitle%3DncRDeathDB%253A%2520a%2520comprehensive%2520bioinformatics%2520resource%2520for%2520deciphering%2520network%2520organization%2520of%2520the%2520ncRNA-mediated%2520cell%2520death%2520system%26jtitle%3DAutophagy%26date%3D2015%26volume%3D11%26issue%3D10%26spage%3D1917%26epage%3D1926%26doi%3D10.1080%2F15548627.2015.1089375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Autophagic compound database: A resource connecting autophagy-modulating compounds, their potential targets and relevant diseases</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">3</span>), <span class="NLM_elocation-id">e12403</span> <span class="refDoi"> DOI: 10.1111/cpr.12403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fcpr.12403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=29094410" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2018&issue=3&author=Y.+Dengauthor=L.+Zhuauthor=H.+Caiauthor=G.+Wangauthor=B.+Liu&title=Autophagic+compound+database%3A+A+resource+connecting+autophagy-modulating+compounds%2C+their+potential+targets+and+relevant+diseases&doi=10.1111%2Fcpr.12403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Fcpr.12403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcpr.12403%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DAutophagic%2520compound%2520database%253A%2520A%2520resource%2520connecting%2520autophagy-modulating%2520compounds%252C%2520their%2520potential%2520targets%2520and%2520relevant%2520diseases%26jtitle%3DCell%2520Proliferation%26date%3D2018%26volume%3D51%26issue%3D3%26doi%3D10.1111%2Fcpr.12403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, X.</span></span> <span> </span><span class="NLM_article-title">TBC2health: a database of experimentally validated health-beneficial effects of tea bioactive compounds</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">830</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1093/bib/bbw055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1093%2Fbib%2Fbbw055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27387194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvFyjt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=830-836&issue=5&author=S.+Zhangauthor=H.+Xuanauthor=L.+Zhangauthor=S.+Fuauthor=Y.+Wangauthor=H.+Yangauthor=Y.+Taiauthor=Y.+Songauthor=J.+Zhangauthor=C.+T.+Hoauthor=S.+Liauthor=X.+Wan&title=TBC2health%3A+a+database+of+experimentally+validated+health-beneficial+effects+of+tea+bioactive+compounds&doi=10.1093%2Fbib%2Fbbw055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">TBC2health: a database of experimentally validated health-beneficial effects of tea bioactive compounds</span></div><div class="casAuthors">Zhang, Shihua; Xuan, Hongdong; Zhang, Liang; Fu, Sicong; Wang, Yijun; Yang, Hua; Tai, Yuling; Song, Youhong; Zhang, Jinsong; Ho, Chi-Tang; Li, Shaowen; Wan, Xiaochun</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">830-836</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1477-4054</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Tea is one of the most consumed beverages in the world.  Considerable studies show the exceptional health benefits (e.g. antioxidn., cancer prevention) of tea owing to its various bioactive components.  However, data from these extensively published papers had not been made available in a central database.  To lay a foundation in improving the understanding of healthy tea functions, we established a TBC2health database that currently documents 1338 relationships between 497 tea bioactive compds. and 206 diseases (or phenotypes) manually culled from over 300 published articles.  Each entry in TBC2health contains comprehensive information about a bioactive relationship that can be accessed in three aspects: (i) compd. information, (ii) disease (or phenotype) information and (iii) evidence and ref.  Using the curated bioactive relationships, a bipartite network was reconstructed and the corresponding network (or sub-network) visualization and topol. analyses are provided for users.  This database has a user-friendly interface for entry browse, search and download.  In addn., TBC2health provides a submission page and several useful tools (e.g. BLAST, mol. docking) to facilitate use of the database.  Consequently, TBC2health can serve as a valuable bioinformatics platform for the exploration of beneficial effects of tea on human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmjwsNAJEPq7Vg90H21EOLACvtfcHk0lgo4eOr1C-heQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvFyjt74%253D&md5=772f369129c68a7ebaf5e7b6589716ed</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbw055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbw055%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXuan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DTai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DC.%2BT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWan%26aufirst%3DX.%26atitle%3DTBC2health%253A%2520a%2520database%2520of%2520experimentally%2520validated%2520health-beneficial%2520effects%2520of%2520tea%2520bioactive%2520compounds%26jtitle%3DBriefings%2520Bioinf.%26date%3D2017%26volume%3D18%26issue%3D5%26spage%3D830%26epage%3D836%26doi%3D10.1093%2Fbib%2Fbbw055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">HAMdb: a database of human autophagy modulators with specific pathway and disease information</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">34</span>, <span class="refDoi"> DOI: 10.1186/s13321-018-0289-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2Fs13321-018-0289-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtFWhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=34&issue=1&author=N.+Wangauthor=J.+Dongauthor=L.+Zhangauthor=D.+Ouyangauthor=Y.+Chengauthor=A.+F.+Chenauthor=A.+P.+Luauthor=D.+S.+Cao&title=HAMdb%3A+a+database+of+human+autophagy+modulators+with+specific+pathway+and+disease+information&doi=10.1186%2Fs13321-018-0289-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">HAMdb: a database of human autophagy modulators with specific pathway and disease information</span></div><div class="casAuthors">Wang, Ning-Ning; Dong, Jie; Zhang, Lin; Ouyang, Defang; Cheng, Yan; Chen, Alex F.; Lu, Ai-Ping; Cao, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/8</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Autophagy is an important homeostatic cellular recycling mechanism responsible for degrading unnecessary or dysfunctional cellular organelles and proteins in all living cells.  In addn. to its vital homeostatic role, this degrdn. pathway also involves in various human disorders, including metabolic conditions, neurodegenerative diseases, cancers and infectious diseases.  Therefore, the comprehensive understanding of autophagy process, autophagyrelated modulators and corresponding pathway and disease information will be of great help for identifying the new autophagy modulators, potential drug candidates, new diagnostic and therapeutic targets.  In recent years, some autophagy databases providing structural and functional information were developed, but the specific databases covering autophagy modulator (proteins, chems. and microRNAs)-related target, pathway and disease information do not exist.  Hence, we developed an online resource, Human Autophagy Modulator Database (HAMdb, http://hamdb .scbdd .com), to provide researchers related pathway and disease information as many as possible.  HAMdb contains 796 proteins, 841 chems. and 132 microRNAs.  Their specific effects on autophagy, physicochem. information, biol. information and disease information were manually collected and compiled.  Addnl., lots of external links were available for more information covering extensive biomedical knowledge.  HAMdb provides a user-friendly interface to query, search, browse autophagy modulators and their comprehensive related information.  HAMdb will help researchers understand the whole autophagy process and provide detailed information about related diseases.  Furthermore, it can give hints for the identification of new diagnostic and therapeutic targets and the discovery of new autophagy modulators.  In a word, we hope that HAMdb has the potential to promote the autophagy research in pharmacol. and pathophysiol. area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQDp8mr_QhELVg90H21EOLACvtfcHk0livxtQ8agAcYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtFWhsrg%253D&md5=8b92387aacb3fd6e6fa156355f072d65</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs13321-018-0289-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-018-0289-4%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DN.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DOuyang%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DA.%2BF.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DHAMdb%253A%2520a%2520database%2520of%2520human%2520autophagy%2520modulators%2520with%2520specific%2520pathway%2520and%2520disease%2520information%26jtitle%3DJ.%2520Cheminf.%26date%3D2018%26volume%3D10%26issue%3D1%26spage%3D34%26doi%3D10.1186%2Fs13321-018-0289-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagyaraj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacko, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, P.</span></span> <span> </span><span class="NLM_article-title">AutoppahgySMDB: a curated database of small molecules that modulate protein targets regulating autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1080/15548627.2019.1571717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2019.1571717" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2019&pages=1-16&author=R.+Nanduriauthor=R.+Kalraauthor=E.+Bhagyarajauthor=A.+P.+Chackoauthor=N.+Ahujaauthor=D.+Tiwariauthor=S.+Kumarauthor=M.+Jainauthor=R.+Parkeshauthor=P.+Gupta&title=AutoppahgySMDB%3A+a+curated+database+of+small+molecules+that+modulate+protein+targets+regulating+autophagy&doi=10.1080%2F15548627.2019.1571717"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F15548627.2019.1571717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2019.1571717%26sid%3Dliteratum%253Aachs%26aulast%3DNanduri%26aufirst%3DR.%26aulast%3DKalra%26aufirst%3DR.%26aulast%3DBhagyaraj%26aufirst%3DE.%26aulast%3DChacko%26aufirst%3DA.%2BP.%26aulast%3DAhuja%26aufirst%3DN.%26aulast%3DTiwari%26aufirst%3DD.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DParkesh%26aufirst%3DR.%26aulast%3DGupta%26aufirst%3DP.%26atitle%3DAutoppahgySMDB%253A%2520a%2520curated%2520database%2520of%2520small%2520molecules%2520that%2520modulate%2520protein%2520targets%2520regulating%2520autophagy%26jtitle%3DAutophagy%26date%3D2019%26volume%3D22%26spage%3D1%26epage%3D16%26doi%3D10.1080%2F15548627.2019.1571717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">THANATOS: an integrative data resource of proteins and post-translational modifications in the regulation of autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">296</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1402990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2017.1402990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=29157087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1emsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=296-310&issue=2&author=W.+Dengauthor=L.+Maauthor=Y.+Zhangauthor=J.+Zhouauthor=Y.+Wangauthor=Z.+Liuauthor=Y.+Xue&title=THANATOS%3A+an+integrative+data+resource+of+proteins+and+post-translational+modifications+in+the+regulation+of+autophagy&doi=10.1080%2F15548627.2017.1402990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">THANATOS: an integrative data resource of proteins and post-translational modifications in the regulation of autophagy</span></div><div class="casAuthors">Deng, Wankun; Ma, Lili; Zhang, Ying; Zhou, Jiaqi; Wang, Yongbo; Liu, Zexian; Xue, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">296-310</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Macroautophagy/autophagy is a highly conserved process for degrading cytoplasmic contents, dets. cell survival or death, and regulates the cellular homeostasis.  Besides ATG proteins, numerous regulators together with various post-translational modifications (PTMs) are also involved in autophagy.  In this work, we collected 4,237 exptl. identified proteins regulated in autophagy and cell death pathways from the literature.  Then we computationally identified potential orthologs of known proteins, and developed a comprehensive database of The Autophagy, Necrosis, ApopTosis OrchestratorS (THANATOS, http://thanatos.biocuckoo.org), contg. 191,543 proteins potentially assocd. with autophagy and cell death pathways in 164 eukaryotes.  We performed an evolutionary anal. of ATG genes, and obsd. that ATGs required for the autophagosome formation are highly conserved across eukaryotes.  Further analyses revealed that known cancer genes and drug targets were overrepresented in human autophagy proteins, which were significantly assocd. in a no. of signaling pathways and human diseases.  By reconstructing a human kinase-substrate phosphorylation network for ATG proteins, our results confirmed that phosphorylation play a crit. role in regulating autophagy.  In total, we mapped 65,015 known sites of 11 types of PTMs to collected proteins, and revealed that all types of PTM substrates were enriched in human autophagy.  In addn., we obsd. multiple types of PTM regulators such as protein kinases and ubiquitin E3 ligases or adaptors were significantly assocd. with human autophagy, and again the results emphasized the importance of PTM regulations in autophagy.  We anticipated THANATOS can be a useful resource for further studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0TlGKc1EH7rVg90H21EOLACvtfcHk0livxtQ8agAcYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1emsbs%253D&md5=ae3c6ea6dca17b8fa819b81413341992</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1402990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1402990%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DTHANATOS%253A%2520an%2520integrative%2520data%2520resource%2520of%2520proteins%2520and%2520post-translational%2520modifications%2520in%2520the%2520regulation%2520of%2520autophagy%26jtitle%3DAutophagy%26date%3D2018%26volume%3D14%26issue%3D2%26spage%3D296%26epage%3D310%26doi%3D10.1080%2F15548627.2017.1402990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">ACTP: a webserver for predicting potential targets and relevant pathways of autophagy-modulating compounds</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">10015</span>– <span class="NLM_lpage">10022</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.7015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.18632%2Foncotarget.7015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26824420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC28nlsVOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10015-10022&issue=9&author=T.+Xieauthor=L.+Zhangauthor=S.+Zhangauthor=L.+Ouyangauthor=H.+Caiauthor=B.+Liu&title=ACTP%3A+a+webserver+for+predicting+potential+targets+and+relevant+pathways+of+autophagy-modulating+compounds&doi=10.18632%2Foncotarget.7015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ACTP: A webserver for predicting potential targets and relevant pathways of autophagy-modulating compounds</span></div><div class="casAuthors">Xie Tao; Zhang Lan; Zhang Shouyue; Ouyang Liang; Liu Bo; Cai Haoyang</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">10015-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Autophagy (macroautophagy) is well known as an evolutionarily conserved lysosomal degradation process for long-lived proteins and damaged organelles.  Recently, accumulating evidence has revealed a series of small-molecule compounds that may activate or inhibit autophagy for therapeutic potential on human diseases.  However, targeting autophagy for drug discovery still remains in its infancy.  In this study, we developed a webserver called Autophagic Compound-Target Prediction (ACTP) (http://actp.liu-lab.com/) that could predict autophagic targets and relevant pathways for a given compound.  The flexible docking of submitted small-molecule compound (s) to potential autophagic targets could be performed by backend reverse docking.  The webpage would return structure-based scores and relevant pathways for each predicted target.  Thus, these results provide a basis for the rapid prediction of potential targets/pathways of possible autophagy-activating or autophagy-inhibiting compounds without labor-intensive experiments.  Moreover, ACTP will be helpful to shed light on identifying more novel autophagy-activating or autophagy-inhibiting compounds for future therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxdDd8aHT25YYPsCkJIw00fW6udTcc2ea2kcBp7eOBarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nlsVOkug%253D%253D&md5=fb4309bd3739c07c1fac5d2717dfadda</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7015%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DACTP%253A%2520a%2520webserver%2520for%2520predicting%2520potential%2520targets%2520and%2520relevant%2520pathways%2520of%2520autophagy-modulating%2520compounds%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26issue%3D9%26spage%3D10015%26epage%3D10022%26doi%3D10.18632%2Foncotarget.7015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X. Q.</span></span> <span> </span><span class="NLM_article-title">ProSelection: A novel algorithm to select proper protein structure subsets for in silico target identification and drug discovery research</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2686</span>– <span class="NLM_lpage">2898</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aks7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2686-2898&issue=11&author=N.+Wangauthor=L.+Wangauthor=X.+Q.+Xie&title=ProSelection%3A+A+novel+algorithm+to+select+proper+protein+structure+subsets+for+in+silico+target+identification+and+drug+discovery+research&doi=10.1021%2Facs.jcim.7b00277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">ProSelection: A Novel Algorithm to Select Proper Protein Structure Subsets for in Silico Target Identification and Drug Discovery Research</span></div><div class="casAuthors">Wang, Nanyi; Wang, Lirong; Xie, Xiang-Qun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2686-2698</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is widely applied to computer-aided drug design and has become relatively mature in the recent decades.  Application of docking in modeling varies from single lead compd. optimization to large-scale virtual screening.  The performance of mol. docking is highly dependent on the protein structures selected.  It is esp. challenging for large-scale target prediction research when multiple structures are available for a single target.  Therefore, the authors have established ProSelection, a docking preferred-protein selection algorithm, in order to generate the proper structure subset(s).  By the ProSelection algorithm, protein structures of "weak selectors" are filtered out whereas structures of "strong selectors" are kept.  Specifically, the structure which has a good statistical performance of distinguishing active ligands from inactive ligands is defined as a strong selector.  In this study, 249 protein structures of 14 autophagy-related targets are investigated.  Surflex-dock was used as the docking engine to distinguish active and inactive compds. against these protein structures.  Both t test and Mann-Whitney U test were used to distinguish the strong from the weak selectors based on the normality of the docking score distribution.  The suggested docking score threshold for active ligands (SDA) was generated for each strong selector structure according to the receiver operating characteristic (ROC) curve.  The performance of ProSelection was further validated by predicting the potential off-targets of 43 U.S.  Federal Drug Administration approved small mol. antineoplastic drugs.  Overall, ProSelection will accelerate the computational work in protein structure selection and could be a useful tool for mol. docking, target prediction, and protein-chem. database establishment research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRzAFCu_t_P7Vg90H21EOLACvtfcHk0ljpHVZM-zlVrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aks7nF&md5=87d2e6a034fa96b442c0e5513f66fe32</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00277%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DX.%2BQ.%26atitle%3DProSelection%253A%2520A%2520novel%2520algorithm%2520to%2520select%2520proper%2520protein%2520structure%2520subsets%2520for%2520in%2520silico%2520target%2520identification%2520and%2520drug%2520discovery%2520research%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D11%26spage%3D2686%26epage%3D2898%26doi%3D10.1021%2Facs.jcim.7b00277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dosztányi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mészáros, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, I.</span></span> <span> </span><span class="NLM_article-title">ANCHOR: web server for predicting protein binding regions in disordered proteins</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">2745</span>– <span class="NLM_lpage">2746</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1093%2Fbioinformatics%2Fbtp518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=19717576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Glsr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2745-2746&issue=20&author=Z.+Doszt%C3%A1nyiauthor=B.+M%C3%A9sz%C3%A1rosauthor=I.+Simon&title=ANCHOR%3A+web+server+for+predicting+protein+binding+regions+in+disordered+proteins&doi=10.1093%2Fbioinformatics%2Fbtp518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">ANCHOR: web server for predicting protein binding regions in disordered proteins</span></div><div class="casAuthors">Dosztanyi, Zsuzsanna; Meszaros, Balint; Simon, Istvan</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2745-2746</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: ANCHOR is a web-based implementation of an original method that takes a single amino acid sequence as an input and predicts protein binding regions that are disordered in isolation but can undergo disorder-to-order transition upon binding.  The server incorporates the result of a general disorder prediction method, IUPred and can carry out simple motif searches as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFf_tj9V_BkbVg90H21EOLACvtfcHk0ljpHVZM-zlVrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Glsr3M&md5=db80e68870116d378ebd524cfc87a445</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp518%26sid%3Dliteratum%253Aachs%26aulast%3DDoszt%25C3%25A1nyi%26aufirst%3DZ.%26aulast%3DM%25C3%25A9sz%25C3%25A1ros%26aufirst%3DB.%26aulast%3DSimon%26aufirst%3DI.%26atitle%3DANCHOR%253A%2520web%2520server%2520for%2520predicting%2520protein%2520binding%2520regions%2520in%2520disordered%2520proteins%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26issue%3D20%26spage%3D2745%26epage%3D2746%26doi%3D10.1093%2Fbioinformatics%2Fbtp518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aslan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küçükoǧlu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arslanyolu, M.</span></span> <span> </span><span class="NLM_article-title">A comparative in-silico analysis of autophagy proteins in ciliates</span>. <i>PeerJ</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e2878</span> <span class="refDoi"> DOI: 10.7717/peerj.2878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.7717%2Fpeerj.2878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28123910" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&author=E.+Aslanauthor=N.+K%C3%BC%C3%A7%C3%BCko%C7%A7luauthor=M.+Arslanyolu&title=A+comparative+in-silico+analysis+of+autophagy+proteins+in+ciliates&doi=10.7717%2Fpeerj.2878"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.2878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.2878%26sid%3Dliteratum%253Aachs%26aulast%3DAslan%26aufirst%3DE.%26aulast%3DK%25C3%25BC%25C3%25A7%25C3%25BCko%25C7%25A7lu%26aufirst%3DN.%26aulast%3DArslanyolu%26aufirst%3DM.%26atitle%3DA%2520comparative%2520in-silico%2520analysis%2520of%2520autophagy%2520proteins%2520in%2520ciliates%26jtitle%3DPeerJ%26date%3D2017%26volume%3D5%26doi%3D10.7717%2Fpeerj.2878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheidel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amstein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, I.</span></span> <span> </span><span class="NLM_article-title">In silico knockout studies of xenopahgic capturing of salmonella</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>), <span class="NLM_elocation-id">e1005200</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1371%2Fjournal.pcbi.1005200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27906974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVekurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&issue=12&author=J.+Scheidelauthor=L.+Amsteinauthor=J.+Ackermannauthor=I.+Dikicauthor=I.+Koch&title=In+silico+knockout+studies+of+xenopahgic+capturing+of+salmonella&doi=10.1371%2Fjournal.pcbi.1005200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">In silico knockout studies of xenophagic capturing of Salmonella</span></div><div class="casAuthors">Scheidel, Jennifer; Amstein, Leonie; Ackermann, Jorg; Dikic, Ivan; Koch, Ina</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1005200/1-e1005200/18</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The degrdn. of cytosol-invading pathogens by autophagy, a process known as xenophagy, is an important mechanism of the innate immune system.  Inside the host, Salmonella Typhimurium invades epithelial cells and resides within a specialized intracellular compartment, the Salmonella-contg. vacuole.  A fraction of these bacteria does not persist inside the vacuole and enters the host cytosol.  Salmonella Typhimurium that invades the host cytosol becomes a target of the autophagy machinery for degrdn.  The xenophagy pathway has recently been discovered, and the exact mol. processes are not entirely characterized.  Complete kinetic data for each mol. process is not available, so far.  We developed a math. model of the xenophagy pathway to investigate this key defense mechanism.  In this paper, we present a Petri net model of Salmonella xenophagy in epithelial cells.  The model is based on functional information derived from literature data.  It comprises the mol. mechanism of galectin-8-dependent and ubiquitindependent autophagy, including regulatory processes, like nutrient-dependent regulation of autophagy and TBK1-dependent activation of the autophagy receptor, OPTN.  To model the activation of TBK1, we proposed a new mechanism of TBK1 activation, suggesting a spatial and temporal regulation of this process.  Using std. Petri net anal. techniques, we found basic functional modules, which describe different pathways of the autophagic capture of Salmonella and reflect the basic dynamics of the system.  To verify the model, we performed in silico knockout expts.  We introduced a new concept of knockout anal. to systematically compute and visualize the results, using an in silico knockout matrix.  The results of the in silico knockout analyses were consistent with published exptl. results and provide a basis for future investigations of the Salmonella xenophagy pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdFUjUNoHRfrVg90H21EOLACvtfcHk0lhypXDXtiHZrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVekurc%253D&md5=1a413cba7b29dfb3bb54a4de001d020c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005200%26sid%3Dliteratum%253Aachs%26aulast%3DScheidel%26aufirst%3DJ.%26aulast%3DAmstein%26aufirst%3DL.%26aulast%3DAckermann%26aufirst%3DJ.%26aulast%3DDikic%26aufirst%3DI.%26aulast%3DKoch%26aufirst%3DI.%26atitle%3DIn%2520silico%2520knockout%2520studies%2520of%2520xenopahgic%2520capturing%2520of%2520salmonella%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D12%26doi%3D10.1371%2Fjournal.pcbi.1005200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obaid, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikarm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janjua, H. A.</span></span> <span> </span><span class="NLM_article-title">Mutation-Structure-Function relationship based integrated strategy reveals the potential impact of deleterious missense mutations in autophagy related proteins on hepatocellular carcinoma (HCC): a comprehensive informatics approach</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">E139</span>, <span class="refDoi"> DOI: 10.3390/ijms18010139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.3390%2Fijms18010139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28085066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1eisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=E139&issue=1&author=F.+M.+Awanauthor=A.+Obaidauthor=A.+Ikarmauthor=H.+A.+Janjua&title=Mutation-Structure-Function+relationship+based+integrated+strategy+reveals+the+potential+impact+of+deleterious+missense+mutations+in+autophagy+related+proteins+on+hepatocellular+carcinoma+%28HCC%29%3A+a+comprehensive+informatics+approach&doi=10.3390%2Fijms18010139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Mutation-structure-function relationship based integrated strategy reveals the potential impact of deleterious missense mutations in autophagy related proteins on hepatocellular carcinoma (HCC): a comprehensive informatics approach</span></div><div class="casAuthors">Awan, Faryal Mehwish; Obaid, Ayesha; Ikram, Aqsa; Janjua, Hussnain Ahmed</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">139/1-139/26</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Autophagy, an evolutionary conserved multifaceted lysosome-mediated bulk degrdn. system, plays a vital role in liver pathologies including hepatocellular carcinoma (HCC).  Post-translational modifications (PTMs) and genetic variations in autophagy components have emerged as significant determinants of autophagy related proteins.  Identification of a comprehensive spectrum of genetic variations and PTMs of autophagy related proteins and their impact at mol. level will greatly expand our understanding of autophagy based regulation.  In this study, we attempted to identify high risk missense mutations that are highly damaging to the structure as well as function of autophagy related proteins including LC3A, LC3B, BECN1 and SCD1.  No. of putative structural and functional residues, including several sites that undergo PTMs were also identified.  In total, 16 high-risk SNPs in LC3A, 18 in LC3B, 40 in BECN1 and 43 in SCD1 were prioritized.  Out of these, 2 in LC3A (K49A, K51A), 1 in LC3B (S92C), 6 in BECN1 (S113R, R292C, R292H, Y338C, S346Y, Y352H) and 6 in SCD1 (Y41C, Y55D, R131W, R135Q, R135W, Y151C) coincide with potential PTM sites.  Our integrated anal. found LC3B Y113C, BECN1 I403T, SCD1 R126S and SCD1 Y218C as highly deleterious HCC-assocd. mutations.  This study is the first extensive in silico mutational anal. of the LC3A, LC3B, BECN1 and SCD1 proteins.  We hope that the obsd. results will be a valuable resource for in-depth mechanistic insight into future investigations of pathol. missense SNPs using an integrated computational platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp7pDq7Tquv7Vg90H21EOLACvtfcHk0lhypXDXtiHZrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1eisbo%253D&md5=cc4148da3a68cb044be36d5c2384b5ee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fijms18010139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18010139%26sid%3Dliteratum%253Aachs%26aulast%3DAwan%26aufirst%3DF.%2BM.%26aulast%3DObaid%26aufirst%3DA.%26aulast%3DIkarm%26aufirst%3DA.%26aulast%3DJanjua%26aufirst%3DH.%2BA.%26atitle%3DMutation-Structure-Function%2520relationship%2520based%2520integrated%2520strategy%2520reveals%2520the%2520potential%2520impact%2520of%2520deleterious%2520missense%2520mutations%2520in%2520autophagy%2520related%2520proteins%2520on%2520hepatocellular%2520carcinoma%2520%2528HCC%2529%253A%2520a%2520comprehensive%2520informatics%2520approach%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26issue%3D1%26spage%3DE139%26doi%3D10.3390%2Fijms18010139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barua, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westrate, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posner, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hlavacek, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackeigan, J. P.</span></span> <span> </span><span class="NLM_article-title">Computational model for autophagic vesicle dynamics in single cells</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.4161/auto.22532</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4161%2Fauto.22532" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23196898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKgs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=74-92&issue=1&author=K.+R.+Martinauthor=D.+Baruaauthor=A.+L.+Kauffmanauthor=L.+M.+Westrateauthor=R.+G.+Posnerauthor=W.+S.+Hlavacekauthor=J.+P.+Mackeigan&title=Computational+model+for+autophagic+vesicle+dynamics+in+single+cells&doi=10.4161%2Fauto.22532"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Computational model for autophagic vesicle dynamics in single cells</span></div><div class="casAuthors">Martin, Katie R.; Barua, Dipak; Kauffman, Audra L.; Westrate, Laura M.; Posner, Richard G.; Hlavacek, William S.; MacKeigan, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-92</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Macroautophagy (autophagy) is a cellular recycling program essential for homeostasis and survival during cytotoxic stress.  This process, which has an emerging role in disease etiol. and treatment, is executed in four stages through the coordinated action of more than 30 proteins.  An effective strategy for studying complicated cellular processes, such as autophagy, involves the construction and anal. of math. or computational models.  When developed and refined from exptl. knowledge, these models can be used to interrogate signaling pathways, formulate novel hypotheses about systems, and make predictions about cell signaling changes induced by specific interventions.  Here, we present the development of a computational model describing autophagic vesicle dynamics in a mammalian system.  We used time-resolved, live-cell microscopy to measure the synthesis and turnover of autophagic vesicles in single cells.  The stochastically simulated model was consistent with data acquired during conditions of both basal and chem.-induced autophagy.  The model was tested by genetic modulation of autophagic machinery and found to accurately predict vesicle dynamics obsd. exptl.  Furthermore, the model generated an unforeseen prediction about vesicle size that is consistent with both published findings and our exptl. observations.  Taken together, this model is accurate and useful and can serve as the foundation for future efforts aimed at quant. characterization of autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdPcEwLNXdjrVg90H21EOLACvtfcHk0lgx6drcN2WfOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKgs7s%253D&md5=6759efc98c0fa4d49b6c8b1dec9f8606</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.4161%2Fauto.22532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.22532%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DK.%2BR.%26aulast%3DBarua%26aufirst%3DD.%26aulast%3DKauffman%26aufirst%3DA.%2BL.%26aulast%3DWestrate%26aufirst%3DL.%2BM.%26aulast%3DPosner%26aufirst%3DR.%2BG.%26aulast%3DHlavacek%26aufirst%3DW.%2BS.%26aulast%3DMackeigan%26aufirst%3DJ.%2BP.%26atitle%3DComputational%2520model%2520for%2520autophagic%2520vesicle%2520dynamics%2520in%2520single%2520cells%26jtitle%3DAutophagy%26date%3D2013%26volume%3D9%26issue%3D1%26spage%3D74%26epage%3D92%26doi%3D10.4161%2Fauto.22532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Börlin, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamacher-Brady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, N. R.</span></span> <span> </span><span class="NLM_article-title">Agent-based modeling of autophagy reveals emergent regulatory behavior of spatio-temporal autophagy dynamics</span>. <i>Cell Commun. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">56</span>, <span class="refDoi"> DOI: 10.1186/s12964-014-0056-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2Fs12964-014-0056-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25214434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2M7gsFKltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=56&author=C.+S.+B%C3%B6rlinauthor=V.+Langauthor=A.+Hamacher-Bradyauthor=N.+R.+Brady&title=Agent-based+modeling+of+autophagy+reveals+emergent+regulatory+behavior+of+spatio-temporal+autophagy+dynamics&doi=10.1186%2Fs12964-014-0056-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Agent-based modeling of autophagy reveals emergent regulatory behavior of spatio-temporal autophagy dynamics</span></div><div class="casAuthors">Borlin Christoph S; Lang Verena; Hamacher-Brady Anne; Brady Nathan R</div><div class="citationInfo"><span class="NLM_cas:title">Cell communication and signaling : CCS</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Autophagy is a vesicle-mediated pathway for lysosomal degradation, essential under basal and stressed conditions.  Various cellular components, including specific proteins, protein aggregates, organelles and intracellular pathogens, are targets for autophagic degradation.  Thereby, autophagy controls numerous vital physiological and pathophysiological functions, including cell signaling, differentiation, turnover of cellular components and pathogen defense.  Moreover, autophagy enables the cell to recycle cellular components to metabolic substrates, thereby permitting prolonged survival under low nutrient conditions.  Due to the multi-faceted roles for autophagy in maintaining cellular and organismal homeostasis and responding to diverse stresses, malfunction of autophagy contributes to both chronic and acute pathologies.  RESULTS:  We applied a systems biology approach to improve the understanding of this complex cellular process of autophagy.  All autophagy pathway vesicle activities, i.e. creation, movement, fusion and degradation, are highly dynamic, temporally and spatially, and under various forms of regulation.  We therefore developed an agent-based model (ABM) to represent individual components of the autophagy pathway, subcellular vesicle dynamics and metabolic feedback with the cellular environment, thereby providing a framework to investigate spatio-temporal aspects of autophagy regulation and dynamic behavior.  The rules defining our ABM were derived from literature and from high-resolution images of autophagy markers under basal and activated conditions.  Key model parameters were fit with an iterative method using a genetic algorithm and a predefined fitness function.  From this approach, we found that accurate prediction of spatio-temporal behavior required increasing model complexity by implementing functional integration of autophagy with the cellular nutrient state.  The resulting model is able to reproduce short-term autophagic flux measurements (up to 3 hours) under basal and activated autophagy conditions, and to measure the degree of cell-to-cell variability.  Moreover, we experimentally confirmed two model predictions, namely (i) peri-nuclear concentration of autophagosomes and (ii) inhibitory lysosomal feedback on mTOR signaling.  CONCLUSION:  Agent-based modeling represents a novel approach to investigate autophagy dynamics, function and dysfunction with high biological realism.  Our model accurately recapitulates short-term behavior and cell-to-cell variability under basal and activated conditions of autophagy.  Further, this approach also allows investigation of long-term behaviors emerging from biologically-relevant alterations to vesicle trafficking and metabolic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRwKkEHzyB-CU2yPDyEFiigfW6udTcc2eYdmdlzQySlObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7gsFKltQ%253D%253D&md5=ae4d2c8fe3cc16c3f0e600d9d6d5ff5f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1186%2Fs12964-014-0056-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12964-014-0056-8%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6rlin%26aufirst%3DC.%2BS.%26aulast%3DLang%26aufirst%3DV.%26aulast%3DHamacher-Brady%26aufirst%3DA.%26aulast%3DBrady%26aufirst%3DN.%2BR.%26atitle%3DAgent-based%2520modeling%2520of%2520autophagy%2520reveals%2520emergent%2520regulatory%2520behavior%2520of%2520spatio-temporal%2520autophagy%2520dynamics%26jtitle%3DCell%2520Commun.%2520Signaling%26date%3D2014%26volume%3D12%26spage%3D56%26doi%3D10.1186%2Fs12964-014-0056-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanneman, W. H.</span></span> <span> </span><span class="NLM_article-title">A multimethod computational simulation approach for investigating mitochondrial dynamics and dysfunction in degenerative aging</span>. <i>Aging Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1111/acel.12644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Facel.12644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28815872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlels7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1244-1255&issue=6&author=T.+E.+Hoffmanauthor=K.+J.+Barnettauthor=L.+Wallisauthor=W.+H.+Hanneman&title=A+multimethod+computational+simulation+approach+for+investigating+mitochondrial+dynamics+and+dysfunction+in+degenerative+aging&doi=10.1111%2Facel.12644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A multimethod computational simulation approach for investigating mitochondrial dynamics and dysfunction in degenerative aging</span></div><div class="casAuthors">Hoffman, Timothy E.; Barnett, Katherine J.; Wallis, Lyle; Hanneman, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Aging Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1244-1255</span>CODEN:
                <span class="NLM_cas:coden">ACGECQ</span>;
        ISSN:<span class="NLM_cas:issn">1474-9718</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Research in biogerontol. has largely focused on the complex relationship between mitochondrial dysfunction and biol. aging.  In particular, the mitochondrial free radical theory of aging (MFRTA) has been well accepted.  However, this theory has been challenged by recent studies showing minimal increases in reactive oxygen species (ROS) as not entirely deleterious in nature, and even beneficial under the appropriate cellular circumstances.  To assess these significant and nonintuitive observations in the context of a functional system, we have taken an in silico approach to expand the focus of the MFRTA by including other key mitochondrial stress response pathways, as they have been obsd. in the nematode Caenorhabditis elegans.  These include the mitochondrial unfolded protein response (UPRmt), mitochondrial biogenesis and autophagy dynamics, the relevant DAF-16 and SKN-1 axes, and NAD+-dependent deacetylase activities.  To integrate these pathways, we have developed a multilevel hybrid-modeling paradigm, contg. agent-based elements among stochastic system-dynamics environments of logically derived ordinary differential equations, to simulate aging mitochondrial phenotypes within a population of energetically demanding cells.  The simulation expts. resulted in accurate predictions of physiol. parameters over time that accompany normal aging, such as the declines in both NAD+ and ATP and an increase in ROS.  Addnl., the in silico system was virtually perturbed using a variety of pharmacol. (e.g., rapamycin, pterostilbene, paraquat) and genetic (e.g., skn-1, daf-16, sod-2) schemes to quantitate the temporal alterations of specific mechanistic targets, supporting insights into mol. determinants of aging as well as cytoprotective agents that may improve neurol. or muscular healthspan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvTvWIQSAssLVg90H21EOLACvtfcHk0lgx6drcN2WfOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlels7rF&md5=0eaa2625915ff30004912f3534fe81d9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Facel.12644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Facel.12644%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DT.%2BE.%26aulast%3DBarnett%26aufirst%3DK.%2BJ.%26aulast%3DWallis%26aufirst%3DL.%26aulast%3DHanneman%26aufirst%3DW.%2BH.%26atitle%3DA%2520multimethod%2520computational%2520simulation%2520approach%2520for%2520investigating%2520mitochondrial%2520dynamics%2520and%2520dysfunction%2520in%2520degenerative%2520aging%26jtitle%3DAging%2520Cell%26date%3D2017%26volume%3D16%26issue%3D6%26spage%3D1244%26epage%3D1255%26doi%3D10.1111%2Facel.12644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tavassoly, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shajahan-Haq, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyson, J. J.</span></span> <span> </span><span class="NLM_article-title">Dynamic modeling of the interaction between autophagy and apoptosis in mammalian cells</span>. <i>CPT: Pharmacometrics Syst. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1002/psp4.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1002%2Fpsp4.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26225250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFSgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=263-272&issue=4&author=I.+Tavassolyauthor=J.+Parmarauthor=A.+N.+Shajahan-Haqauthor=R.+Clarkeauthor=W.+T.+Baumannauthor=J.+J.+Tyson&title=Dynamic+modeling+of+the+interaction+between+autophagy+and+apoptosis+in+mammalian+cells&doi=10.1002%2Fpsp4.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Modeling of the Interaction Between Autophagy and Apoptosis in Mammalian Cells</span></div><div class="casAuthors">Tavassoly, I.; Parmar, J.; Shajahan-Haq, A. N.; Clarke, R.; Baumann, W. T.; Tyson, J. J.</div><div class="citationInfo"><span class="NLM_cas:title">CPT: Pharmacometrics & Systems Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">CPSPBR</span>;
        ISSN:<span class="NLM_cas:issn">2163-8306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Autophagy is a conserved biol. stress response in mammalian cells that is responsible for clearing damaged proteins and organelles from the cytoplasm and recycling their contents via the lysosomal pathway.  In cases of mild stress, autophagy acts as a survival mechanism, while in cases of severe stress cells may switch to programmed cell death.  Understanding the decision process that moves a cell from autophagy to apoptosis is important since abnormal regulation of autophagy occurs in many diseases, including cancer.  To integrate existing knowledge about this decision process into a rigorous, anal. framework, we built a math. model of cell fate decisions mediated by autophagy.  Our dynamical model is consistent with existing quant. measurements of autophagy and apoptosis in rat kidney proximal tubular cells responding to cisplatin-induced stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVXs53snsbRrVg90H21EOLACvtfcHk0ljWS8l5LROqNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFSgurc%253D&md5=6858ab9b0ef3ef715c640b81adc55498</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fpsp4.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsp4.29%26sid%3Dliteratum%253Aachs%26aulast%3DTavassoly%26aufirst%3DI.%26aulast%3DParmar%26aufirst%3DJ.%26aulast%3DShajahan-Haq%26aufirst%3DA.%2BN.%26aulast%3DClarke%26aufirst%3DR.%26aulast%3DBaumann%26aufirst%3DW.%2BT.%26aulast%3DTyson%26aufirst%3DJ.%2BJ.%26atitle%3DDynamic%2520modeling%2520of%2520the%2520interaction%2520between%2520autophagy%2520and%2520apoptosis%2520in%2520mammalian%2520cells%26jtitle%3DCPT%253A%2520Pharmacometrics%2520Syst.%2520Pharmacol.%26date%3D2015%26volume%3D4%26issue%3D4%26spage%3D263%26epage%3D272%26doi%3D10.1002%2Fpsp4.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yung, B. Y.</span></span> <span> </span><span class="NLM_article-title">Gene network exploration of crosstalk between apoptosis and autophagy in chronic myelogenous leukemia</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>,  <span class="NLM_fpage">459840</span>, <span class="refDoi"> DOI: 10.1155/2015/459840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1155%2F2015%2F459840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25821802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2Mnpt1eiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&pages=459840&author=F.+Wangauthor=W.+C.+Choauthor=L.+W.+Chanauthor=S.+C.+Wongauthor=N.+B.+Tsuiauthor=P.+M.+Siuauthor=S.+P.+Yipauthor=B.+Y.+Yung&title=Gene+network+exploration+of+crosstalk+between+apoptosis+and+autophagy+in+chronic+myelogenous+leukemia&doi=10.1155%2F2015%2F459840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Gene network exploration of crosstalk between apoptosis and autophagy in chronic myelogenous leukemia</span></div><div class="casAuthors">Wang Fengfeng; Chan Lawrence W C; Wong S C Cesar; Tsui Nancy B Y; Siu Parco M; Yip S P; Yung Benjamin Y M; Cho William C S</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">459840</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Gene expression levels change to adapt the stress, such as starvation, toxin, and radiation.  The changes are signals transmitted through molecular interactions, eventually leading to two cellular fates, apoptosis and autophagy.  Due to genetic variations, the signals may not be effectively transmitted to modulate apoptotic and autophagic responses.  Such aberrant modulation may lead to carcinogenesis and drug resistance.  The balance between apoptosis and autophagy becomes very crucial in coping with the stress.  Though there have been evidences illustrating the apoptosis-autophagy interplay, the underlying mechanism and the participation of the regulators including transcription factors (TFs) and microRNAs (miRNAs) remain unclear.  RESULTS:  Gene network is a graphical illustration for exploring the functional linkages and the potential coordinate regulations of genes.  Microarray dataset for the study of chronic myeloid leukemia was obtained from Gene Expression Omnibus.  The expression profiles of those genes related to apoptosis and autophagy, including MCL1, BCL2, ATG, beclin-1, BAX, BAK, E2F, cMYC, PI3K, AKT, BAD, and LC3, were extracted from the dataset to construct the gene networks.  CONCLUSION:  The network analysis of these genes explored the underlying mechanisms and the roles of TFs and miRNAs for the crosstalk between apoptosis and autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFWtPqxa6zoFvAgqb2vdc5fW6udTcc2eZv5-NbhHGVgrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mnpt1eiug%253D%253D&md5=a91e2e3d3280ac9ebbe2c9fba634f459</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1155%2F2015%2F459840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F459840%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DCho%26aufirst%3DW.%2BC.%26aulast%3DChan%26aufirst%3DL.%2BW.%26aulast%3DWong%26aufirst%3DS.%2BC.%26aulast%3DTsui%26aufirst%3DN.%2BB.%26aulast%3DSiu%26aufirst%3DP.%2BM.%26aulast%3DYip%26aufirst%3DS.%2BP.%26aulast%3DYung%26aufirst%3DB.%2BY.%26atitle%3DGene%2520network%2520exploration%2520of%2520crosstalk%2520between%2520apoptosis%2520and%2520autophagy%2520in%2520chronic%2520myelogenous%2520leukemia%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2015%26volume%3D2015%26spage%3D459840%26doi%3D10.1155%2F2015%2F459840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small molecule targeting ULK1-modulated cell death of triple negative breast cancer <i>in vitro</i> and <i>in vivo</i></span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">2687</span>– <span class="NLM_lpage">2701</span>, <span class="refDoi"> DOI: 10.1039/C6SC05368H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1039%2FC6SC05368H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28553505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltlChtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2687-2701&issue=4&author=L.+Zhangauthor=L.+Fuauthor=S.+Zhangauthor=J.+Zhangauthor=Y.+Zhaoauthor=Y.+Zhengauthor=G.+Heauthor=S.+Yangauthor=L.+Ouyangauthor=B.+Liu&title=Discovery+of+a+small+molecule+targeting+ULK1-modulated+cell+death+of+triple+negative+breast+cancer+in+vitro+and+in+vivo&doi=10.1039%2FC6SC05368H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small molecule targeting ULK1-modulated cell death of triple negative breast cancer in vitro and in vivo</span></div><div class="casAuthors">Zhang, Lan; Fu, Leilei; Zhang, Shouyue; Zhang, Jin; Zhao, Yuqian; Zheng, Yaxin; He, Gu; Yang, Shengyong; Ouyang, Liang; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2687-2701</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">UNC-51-like kinase 1 (ULK1) is well-known to initiate autophagy, and the downregulation of ULK1 has been found in most breast cancer tissues.  Thus, the activation of ULK1-modulated autophagy could be a promising strategy for breast cancer therapy.  In this study, we found that ULK1 was remarkably downregulated in breast cancer tissue samples by The Cancer Genome Atlas (TCGA) anal. and tissue microarray (TMA) anal., esp. in triple neg. breast cancer (TNBC).  To design a ULK1 agonist, we integrated in silico screening and chem. synthesis to acquire a series of small mol. candidates.  After rounds of kinase and anti-proliferative activity screening, we discovered the small mol., LYN-1604, to be the best candidate for a ULK1 agonist.  Addnl., we identified that three amino acid residues (LYS50, LEU53, and TYR89) were key to the activation site of LYN-1604 and ULK1 by site-directed mutagenesis and biochem. assays.  Subsequently, we demonstrated that LYN-1604 could induce cell death, assocd. with autophagy by the ULK complex (ULK1-mATG13-FIP200-ATG101) in MDA-MB-231 cells.  To further explore LYN-1604-induced autophagic mechanisms, we found some potential ULK1 interactors, such as ATF3, RAD21, and caspase3, by performing comparative microarray anal.  Intriguingly, we found that LYN-1604 induced cell death involved in ATF3, RAD21, and caspase3, accompanied by autophagy and apoptosis.  Moreover, we demonstrated that LYN-1604 has potential for good therapeutic effects on TNBC by targeting ULK1-modulated cell death in vivo; thus making this ULK1 agonist a novel potential small-mol. drug candidate for future TNBC therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqilhZpiddfcbVg90H21EOLACvtfcHk0ljWS8l5LROqNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltlChtg%253D%253D&md5=6c40089843a8fbcdbb5ee60e0547dc7c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2FC6SC05368H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC05368H%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520small%2520molecule%2520targeting%2520ULK1-modulated%2520cell%2520death%2520of%2520triple%2520negative%2520breast%2520cancer%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26issue%3D4%26spage%3D2687%26epage%3D2701%26doi%3D10.1039%2FC6SC05368H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of genetic vatiants in autophagy pathway genes as prognostic biomarkers for breast cancer</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>627</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1016/j.gene.2017.06.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.gene.2017.06.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28669927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1CjtbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=627&publication_year=2017&pages=549-555&author=J.+Zhouauthor=D.+Hangauthor=Y.+Jiangauthor=J.+Chenauthor=J.+Hanauthor=W.+Zhouauthor=G.+Jinauthor=H.+Maauthor=J.+Dai&title=Evaluation+of+genetic+vatiants+in+autophagy+pathway+genes+as+prognostic+biomarkers+for+breast+cancer&doi=10.1016%2Fj.gene.2017.06.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of genetic variants in autophagy pathway genes as prognostic biomarkers for breast cancer</span></div><div class="casAuthors">Zhou, Jing; Hang, Dong; Jiang, Yue; Chen, Jiaping; Han, Jing; Zhou, Wen; Jin, Guangfu; Ma, Hongxia; Dai, Juncheng</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">627</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Autophagy-related genes (ATGs) play a crit. role in the development of various diseases including cancer.  However, the role of ATGs in breast cancer survival remains unclear.  This study aims to investigate whether genetic variants in core ATGs are correlated with the prognosis of breast cancer.  A total of 14 potentially functional variants in core ATGs were genotyped in 790 breast cancer patients.  The assocn. of each variant with breast cancer-specific survival was evaluated by log-rank test and Cox regression model.  In silico anal. was also performed to evaluate the potential function of selected variants.  We found that one variant in ATG7 rs8154 (A > G) was significantly assocd. with breast cancer-specific survival after adjusted for age and clin. stage (HR = 1.61, 95% CI: 1.12-2.31, P = 0.010).  Stratified anal. showed that the prognostic role of rs8154 was significant in subgroups of elder age, elder menarche age, and postmenopausal status (all P < 0.001).  Interaction effects were also detected between rs8154 and these grouping variables.  In silico anal. revealed that rs8154 was annotated to be expression quant. trait loci (eQTL) and methylation quant. trait loci (meQTL) based on Genotype-Tissue Expression (GTEx) and The Cancer Genome Atlas (TCGA) datasets (both P < 0.05).  In addn., upregulated expression of ATG7 in breast cancer tissues was obsd. and is significantly assocd. with poorer overall survival (log-rank P = 0.015), poorer relapse free survival (log-rank P = 0.017), and poorer distant metastasis free survival (log-rank P = 0.034) in different datasets.  Summarily, ATG7 variant rs8154 represents a novel prognostic marker for breast cancer patients, which may shed light on clin. risk stratification and therapeutic decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx-WJ2jwZyH7Vg90H21EOLACvtfcHk0lhNq6LDCS2tRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1CjtbfF&md5=15b73aadc54ff7ba7890ae281861b6ba</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2017.06.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2017.06.053%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHang%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJin%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DDai%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520genetic%2520vatiants%2520in%2520autophagy%2520pathway%2520genes%2520as%2520prognostic%2520biomarkers%2520for%2520breast%2520cancer%26jtitle%3DGene%26date%3D2017%26volume%3D627%26spage%3D549%26epage%3D555%26doi%3D10.1016%2Fj.gene.2017.06.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suwansa-Ard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kankuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongbuakaew, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saetan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornthong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruangkum, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khornchatri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isidoro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobhon, P.</span></span> <span> </span><span class="NLM_article-title">Transcriptomic analysis of the autophagy machinery in crustaceans</span>. <i>BMC Genomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">587</span>, <span class="refDoi"> DOI: 10.1186/s12864-016-2996-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2Fs12864-016-2996-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27506197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSju7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=587&author=S.+Suwansa-Ardauthor=W.+Kankuanauthor=T.+Thongbuakaewauthor=J.+Saetanauthor=N.+Kornthongauthor=T.+Kruangkumauthor=K.+Khornchatriauthor=S.+F.+Cumminsauthor=C.+Isidoroauthor=P.+Sobhon&title=Transcriptomic+analysis+of+the+autophagy+machinery+in+crustaceans&doi=10.1186%2Fs12864-016-2996-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomic analysis of the autophagy machinery in crustaceans</span></div><div class="casAuthors">Suwansa-ard, Saowaros; Kankuan, Wilairat; Thongbuakaew, Tipsuda; Saetan, Jirawat; Kornthong, Napamanee; Kruangkum, Thanapong; Khornchatri, Kanjana; Cummins, Scott F.; Isidoro, Ciro; Sobhon, Prasert</div><div class="citationInfo"><span class="NLM_cas:title">BMC Genomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">587/1-587/14</span>CODEN:
                <span class="NLM_cas:coden">BGMEET</span>;
        ISSN:<span class="NLM_cas:issn">1471-2164</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: The giant freshwater prawn, Macrobrachium rosenbergii, is a decapod crustacean that is com. important as a food source.  Farming of com. crustaceans requires an efficient management strategy because the animals are easily subjected to stress and diseases during the culture.  Autophagy, a stress response process, is well-documented and conserved in most animals, yet it is poorly studied in crustaceans.  Results: In this study, we have performed an in silico search for transcripts encoding autophagy-related (Atg) proteins within various tissue transcriptomes of M. rosenbergii.  Basic Local Alignment Search Tool (BLAST) search using previously known Atg proteins as queries revealed 41 transcripts encoding homologous M. rosenbergii Atg proteins.  Among these Atg proteins, we selected commonly used autophagy markers, including Beclin 1, vacuolar protein sorting (Vps) 34, microtubule-assocd. proteins 1A/1B light chain 3B (MAP1LC3B), p62/sequestosome 1 (SQSTM1), and lysosomal-assocd. membrane protein 1 (Lamp-1) for further sequence analyses using comparative alignment and protein structural prediction.  We found that crustacean autophagy marker proteins contain conserved motifs typical of other animal Atg proteins.  Western blotting using com. antibodies raised against human Atg marker proteins indicated their presence in various M. rosenbergii tissues, while immunohistochem. localized Atg marker proteins within ovarian tissue, specifically late stage oocytes.  Conclusions: This study demonstrates that the mol. components of autophagic process are conserved in crustaceans, which is comparable to autophagic process in mammals.  Furthermore, it provides a foundation for further studies of autophagy in crustaceans that may lead to more understanding of the reprodn.- and stress-related autophagy, which will enable the efficient aquaculture practices.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Yhee2Z7vILVg90H21EOLACvtfcHk0lhNq6LDCS2tRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSju7rM&md5=6dcc2c29d94e76de1a2b27e959cf8ffa</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs12864-016-2996-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12864-016-2996-4%26sid%3Dliteratum%253Aachs%26aulast%3DSuwansa-Ard%26aufirst%3DS.%26aulast%3DKankuan%26aufirst%3DW.%26aulast%3DThongbuakaew%26aufirst%3DT.%26aulast%3DSaetan%26aufirst%3DJ.%26aulast%3DKornthong%26aufirst%3DN.%26aulast%3DKruangkum%26aufirst%3DT.%26aulast%3DKhornchatri%26aufirst%3DK.%26aulast%3DCummins%26aufirst%3DS.%2BF.%26aulast%3DIsidoro%26aufirst%3DC.%26aulast%3DSobhon%26aufirst%3DP.%26atitle%3DTranscriptomic%2520analysis%2520of%2520the%2520autophagy%2520machinery%2520in%2520crustaceans%26jtitle%3DBMC%2520Genomics%26date%3D2016%26volume%3D17%26spage%3D587%26doi%3D10.1186%2Fs12864-016-2996-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, W.</span></span> <span> </span><span class="NLM_article-title">Independent regulation of gene expression level and noise by histone modifications</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>), <span class="NLM_elocation-id">e1005585</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1371%2Fjournal.pcbi.1005585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28665997" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&issue=6&author=S.+Wuauthor=K.+Liauthor=Y.+Liauthor=T.+Zhaoauthor=T.+Liauthor=Y.+F.+Yangauthor=W.+Qian&title=Independent+regulation+of+gene+expression+level+and+noise+by+histone+modifications&doi=10.1371%2Fjournal.pcbi.1005585"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005585%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DY.%2BF.%26aulast%3DQian%26aufirst%3DW.%26atitle%3DIndependent%2520regulation%2520of%2520gene%2520expression%2520level%2520and%2520noise%2520by%2520histone%2520modifications%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D6%26doi%3D10.1371%2Fjournal.pcbi.1005585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span> <span> </span><span class="NLM_article-title">Integrative bioinformatics and proteomics-based discovery of an eEF2K inhibitor (cefatrizine) with ER stress modulation in breast cancer cells</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1039/C5MB00848D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1039%2FC5MB00848D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26776155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVGhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=729-736&issue=3&author=Z.+Yaoauthor=J.+Liauthor=Z.+Liuauthor=L.+Zhengauthor=N.+Fanauthor=Y.+Zhangauthor=N.+Jiaauthor=J.+Lvauthor=N.+Liuauthor=X.+Zhuauthor=J.+Duauthor=C.+Lvauthor=F.+Xieauthor=Y.+Liuauthor=X.+Wangauthor=Z.+Feiauthor=C.+Gao&title=Integrative+bioinformatics+and+proteomics-based+discovery+of+an+eEF2K+inhibitor+%28cefatrizine%29+with+ER+stress+modulation+in+breast+cancer+cells&doi=10.1039%2FC5MB00848D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative bioinformatics and proteomics-based discovery of an eEF2K inhibitor (cefatrizine) with ER stress modulation in breast cancer cells</span></div><div class="casAuthors">Yao, Zhiqiang; Li, Juntang; Liu, Zhongyu; Zheng, Lu; Fan, Naijun; Zhang, Ying; Jia, Nan; Lv, Jingjing; Liu, Ningning; Zhu, Xiaoshan; Du, Jiangbo; Lv, Ci; Xie, Feng; Liu, Yigang; Wang, Xingke; Fei, Zhou; Gao, Chunfang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">729-736</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Eukaryotic elongation factor-2 kinase (eEF2K), a unique calcium/calmodulin-dependent protein kinase, is well known to regulate apoptosis, autophagy and ER stress in many types of human cancers.  Therefore, eEF2K would be regarded as a promising therapeutic target; however, the eEF2K-regulated mechanism and its targeted inhibitor still remain to be discovered in cancer.  Herein, we constructed a protein-protein interaction (PPI) network of eEF2K and achieved an eEF2K-regulated ER stress subnetwork by bioinformatics prediction.  Then, we found that the differential protein expressions involved in ER stress in the context of si-eEF2K-treated MCF-7 and MDA-MB-436 cells by iTRAQ-based analyses, resp.  Integrated into these aforementioned results, we constructed a core eEF2K-regulated ER stress subnetwork in breast cancer cells.  Subsequently, we screened a series of candidate compds. targeting eEF2K and discovered a novel eEF2K inhibitor (cefatrizine) with an anti-proliferative activity toward breast cancer cells.  Moreover, we found that cefatrizine induced ER stress in both MCF-7 and MDA-MB-436 cells.  Interestingly, we demonstrated that the mechanism of cefatrizine-induced ER stress was in good agreement with our bioinformatics and proteomics-based results.  In conclusion, these results demonstrate that a novel eEF2K inhibitor (cefatrizine) induces ER stress in breast cancer cells by integrating bioinformatics prediction, proteomics analyses and exptl. validation, which would provide a clue for exploring more mechanisms of eEF2K and its targeted inhibitors in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDQPIGrQ6947Vg90H21EOLACvtfcHk0lhNq6LDCS2tRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVGhuw%253D%253D&md5=84c2bbd60d8c71479e98f426db5fbb5b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC5MB00848D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MB00848D%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DFan%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJia%26aufirst%3DN.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DLv%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DFei%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DC.%26atitle%3DIntegrative%2520bioinformatics%2520and%2520proteomics-based%2520discovery%2520of%2520an%2520eEF2K%2520inhibitor%2520%2528cefatrizine%2529%2520with%2520ER%2520stress%2520modulation%2520in%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26issue%3D3%26spage%3D729%26epage%3D736%26doi%3D10.1039%2FC5MB00848D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Phosphoproteome-based kinase activity profiling reveals the critical role of MAP2K2 and PLK1 in neuronal autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1969</span>– <span class="NLM_lpage">1980</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1371393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2017.1371393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28933595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1SqsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1969-1980&issue=11&author=L.+L.+Chenauthor=Y.+B.+Wangauthor=J.+X.+Songauthor=W.+K.+Dengauthor=J.+H.+Luauthor=L.+L.+Maauthor=C.+B.+Yangauthor=M.+Liauthor=Y.+Xue&title=Phosphoproteome-based+kinase+activity+profiling+reveals+the+critical+role+of+MAP2K2+and+PLK1+in+neuronal+autophagy&doi=10.1080%2F15548627.2017.1371393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoproteome-based kinase activity profiling reveals the critical role of MAP2K2 and PLK1 in neuronal autophagy</span></div><div class="casAuthors">Chen, Lei-Lei; Wang, Yong-Bo; Song, Ju-Xian; Deng, Wan-Kun; Lu, Jia-Hong; Ma, Li-Li; Yang, Chuan-Bin; Li, Min; Xue, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1969-1980</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Recent studies have demonstrated that dysregulation of macroautophagy/autophagy may play a central role in the pathogenesis of neurodegenerative disorders, and the induction of autophagy protects against the toxic insults of aggregate-prone proteins by enhancing their clearance.  Thus, autophagy has become a promising therapeutic target against neurodegenerative diseases.  In this study, quant. phosphoproteomic profiling together with a computational anal. was performed to delineate the phosphorylation signaling networks regulated by 2 natural neuroprotective autophagy enhancers, corynoxine (Cory) and corynoxine B (Cory B).  To identify key regulators, namely, protein kinases, we developed a novel network-based algorithm of in silico Kinome Activity Profiling (iKAP) to computationally infer potentially important protein kinases from phosphorylation networks.  Using this algorithm, we obsd. that Cory or Cory B potentially regulated several kinases.  We predicted and validated that Cory, but not Cory B, downregulated a well-documented autophagy kinase, RPS6KB1/p70S6K (ribosomal protein S6 kinase, polypeptide 1).  We also discovered 2 kinases, MAP2K2/MEK2 (mitogen-activated protein kinase kinase 2) and PLK1 (polo-like kinase 1), to be potentially upregulated by Cory, whereas the siRNA-mediated knockdown of Map2k2 and Plk1 significantly inhibited Cory-induced autophagy.  Furthermore, Cory promoted the clearance of Alzheimer disease-assocd. APP (amyloid β [A4] precursor protein) and Parkinson disease-assocd. SNCA/α-synuclein (synuclein, α) by enhancing autophagy, and these effects were dramatically diminished by the inhibition of the kinase activities of MAP2K2 and PLK1.  As a whole, our study not only developed a powerful method for the identification of important regulators from the phosphoproteomic data but also identified the important role of MAP2K2 and PLK1 in neuronal autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcXcdC7yLfIbVg90H21EOLACvtfcHk0liUh-ALBJufDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1SqsLfF&md5=b55042a64c901efceaca0224a907a91a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1371393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1371393%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BL.%26aulast%3DWang%26aufirst%3DY.%2BB.%26aulast%3DSong%26aufirst%3DJ.%2BX.%26aulast%3DDeng%26aufirst%3DW.%2BK.%26aulast%3DLu%26aufirst%3DJ.%2BH.%26aulast%3DMa%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DC.%2BB.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DPhosphoproteome-based%2520kinase%2520activity%2520profiling%2520reveals%2520the%2520critical%2520role%2520of%2520MAP2K2%2520and%2520PLK1%2520in%2520neuronal%2520autophagy%26jtitle%3DAutophagy%26date%3D2017%26volume%3D13%26issue%3D11%26spage%3D1969%26epage%3D1980%26doi%3D10.1080%2F15548627.2017.1371393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behrends, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sowa, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span> <span> </span><span class="NLM_article-title">Network organization of the human autophagy system</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span> (<span class="NLM_issue">7302</span>),  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1038/nature09204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fnature09204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=20562859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXns1Gis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=68-76&issue=7302&author=C.+Behrendsauthor=M.+E.+Sowaauthor=S.+P.+Gygiauthor=J.+W.+Harper&title=Network+organization+of+the+human+autophagy+system&doi=10.1038%2Fnature09204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Network organization of the human autophagy system</span></div><div class="casAuthors">Behrends, Christian; Sowa, Mathew E.; Gygi, Steven P.; Harper, J. Wade</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7302</span>),
    <span class="NLM_cas:pages">68-76</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy, the process by which proteins and organelles are sequestered in autophagosomal vesicles and delivered to the lysosome/vacuole for degrdn., provides a primary route for turnover of stable and defective cellular proteins.  Defects in this system are linked with numerous human diseases.  Although conserved protein kinase, lipid kinase and ubiquitin-like protein conjugation subnetworks controlling autophagosome formation and cargo recruitment have been defined, our understanding of the global organization of this system is limited.  Here we report a proteomic anal. of the autophagy interaction network in human cells under conditions of ongoing (basal) autophagy, revealing a network of 751 interactions among 409 candidate interacting proteins with extensive connectivity among subnetworks.  Many new autophagy interaction network components have roles in vesicle trafficking, protein or lipid phosphorylation and protein ubiquitination, and affect autophagosome no. or flux when depleted by RNA interference.  The six ATG8 orthologues in humans (MAP1LC3/GABARAP proteins) interact with a cohort of 67 proteins, with extensive binding partner overlap between family members, and frequent involvement of a conserved surface on ATG8 proteins known to interact with LC3-interacting regions in partner proteins.  These studies provide a global view of the mammalian autophagy interaction landscape and a resource for mechanistic anal. of this crit. protein homeostasis pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvZw7FZHvYbLVg90H21EOLACvtfcHk0liUh-ALBJufDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXns1Gis7Y%253D&md5=5d64dbe07f7d51a962568c3e8d522991</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature09204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09204%26sid%3Dliteratum%253Aachs%26aulast%3DBehrends%26aufirst%3DC.%26aulast%3DSowa%26aufirst%3DM.%2BE.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26atitle%3DNetwork%2520organization%2520of%2520the%2520human%2520autophagy%2520system%26jtitle%3DNature%26date%3D2010%26volume%3D466%26issue%3D7302%26spage%3D68%26epage%3D76%26doi%3D10.1038%2Fnature09204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordureau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swarup, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span> <span> </span><span class="NLM_article-title">RAB7A phosphorylation by TBK1 promotes mitophagy bia the PINK-PARKIN pathway</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">eaav0443</span>, <span class="refDoi"> DOI: 10.1126/sciadv.aav0443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1126%2Fsciadv.aav0443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=30627666" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=eaav0443&issue=11&author=J.+M.+Heoauthor=A.+Ordureauauthor=S.+Swarupauthor=J.+A.+Pauloauthor=K.+Shenauthor=D.+M.+Sabatiniauthor=J.+W.+Harper&title=RAB7A+phosphorylation+by+TBK1+promotes+mitophagy+bia+the+PINK-PARKIN+pathway&doi=10.1126%2Fsciadv.aav0443"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aav0443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aav0443%26sid%3Dliteratum%253Aachs%26aulast%3DHeo%26aufirst%3DJ.%2BM.%26aulast%3DOrdureau%26aufirst%3DA.%26aulast%3DSwarup%26aufirst%3DS.%26aulast%3DPaulo%26aufirst%3DJ.%2BA.%26aulast%3DShen%26aufirst%3DK.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26atitle%3DRAB7A%2520phosphorylation%2520by%2520TBK1%2520promotes%2520mitophagy%2520bia%2520the%2520PINK-PARKIN%2520pathway%26jtitle%3DSci.%2520Adv.%26date%3D2018%26volume%3D4%26issue%3D11%26spage%3Deaav0443%26doi%3D10.1126%2Fsciadv.aav0443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuballa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolsky, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, R. J.</span></span> <span> </span><span class="NLM_article-title">A novel hybrid yeast-human network analysis reveals an essential role for FNBP1L in antibacterial autophagy</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>182</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">4917</span>– <span class="NLM_lpage">4930</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.0803050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4049%2Fjimmunol.0803050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=19342671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFOju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2009&pages=4917-4930&issue=8&author=A.+Huettauthor=A.+Ngauthor=Z.+Caoauthor=P.+Kuballaauthor=M.+Komatsuauthor=M.+J.+Dalyauthor=D.+K.+Podolskyauthor=R.+J.+Xavier&title=A+novel+hybrid+yeast-human+network+analysis+reveals+an+essential+role+for+FNBP1L+in+antibacterial+autophagy&doi=10.4049%2Fjimmunol.0803050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Hybrid Yeast-Human Network Analysis Reveals an Essential Role for FNBP1L in Antibacterial Autophagy</span></div><div class="casAuthors">Huett, Alan; Ng, Aylwin; Cao, Zhifang; Kuballa, Petric; Komatsu, Masaaki; Daly, Mark J.; Podolsky, Daniel K.; Xavier, Ramnik J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4917-4930</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Autophagy is a conserved cellular process required for the removal of defective organelles, protein aggregates, and intracellular pathogens.  We used a network anal. strategy to identify novel human autophagy components based upon the yeast interactome centered on the core yeast autophagy proteins.  This revealed the potential involvement of 14 novel mammalian genes in autophagy, several of which have known or predicted roles in membrane organization or dynamics.  We selected one of these membrane interactors, FNBP1L (formin binding protein 1-like), an F-BAR-contg. protein (also termed Toca-1), for further study based upon a predicted interaction with ATG3.  We confirmed the FNBP1L/ATG3 interaction biochem. and mapped the FNBP1L domains responsible.  Using a functional RNA interference approach, we detd. that FNBP1L is essential for autophagy of the intracellular pathogen Salmonella enterica serovar Typhimurium and show that the autophagy process serves to restrict the growth of intracellular bacteria.  However, FNBP1L appears dispensable for other forms of autophagy induced by serum starvation or rapamycin.  We present a model where FNBP1L is essential for autophagy of intracellular pathogens and identify FNBP1L as a differentially used mol. in specific autophagic contexts.  By using network biol. to derive functional biol. information, we demonstrate the utility of integrated genomics to novel mol. discovery in autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0P-EhlrQHIrVg90H21EOLACvtfcHk0ljc4NpzTgEfKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFOju7k%253D&md5=edb0aa79cd4842063f0bb9a4ae602c42</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.0803050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.0803050%26sid%3Dliteratum%253Aachs%26aulast%3DHuett%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DA.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DKuballa%26aufirst%3DP.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DDaly%26aufirst%3DM.%2BJ.%26aulast%3DPodolsky%26aufirst%3DD.%2BK.%26aulast%3DXavier%26aufirst%3DR.%2BJ.%26atitle%3DA%2520novel%2520hybrid%2520yeast-human%2520network%2520analysis%2520reveals%2520an%2520essential%2520role%2520for%2520FNBP1L%2520in%2520antibacterial%2520autophagy%26jtitle%3DJ.%2520Immunol.%26date%3D2009%26volume%3D182%26issue%3D8%26spage%3D4917%26epage%3D4930%26doi%3D10.4049%2Fjimmunol.0803050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Türei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Földvári-Nagy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazekas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Módos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubisch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadlecsik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demeter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcsmáros, T.</span></span> <span> </span><span class="NLM_article-title">Autophagy regulatory network-a systems-level bioinformatics resource for studying the mechanism and regulation of autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.4161/15548627.2014.994346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4161%2F15548627.2014.994346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25635527" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=155-165&issue=1&author=D.+T%C3%BCreiauthor=L.+F%C3%B6ldv%C3%A1ri-Nagyauthor=D.+Fazekasauthor=D.+M%C3%B3dosauthor=J.+Kubischauthor=T.+Kadlecsikauthor=A.+Demeterauthor=K.+Lentiauthor=P.+Csermelyauthor=T.+Vellaiauthor=T.+Korcsm%C3%A1ros&title=Autophagy+regulatory+network-a+systems-level+bioinformatics+resource+for+studying+the+mechanism+and+regulation+of+autophagy&doi=10.4161%2F15548627.2014.994346"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.4161%2F15548627.2014.994346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15548627.2014.994346%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25BCrei%26aufirst%3DD.%26aulast%3DF%25C3%25B6ldv%25C3%25A1ri-Nagy%26aufirst%3DL.%26aulast%3DFazekas%26aufirst%3DD.%26aulast%3DM%25C3%25B3dos%26aufirst%3DD.%26aulast%3DKubisch%26aufirst%3DJ.%26aulast%3DKadlecsik%26aufirst%3DT.%26aulast%3DDemeter%26aufirst%3DA.%26aulast%3DLenti%26aufirst%3DK.%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3DVellai%26aufirst%3DT.%26aulast%3DKorcsm%25C3%25A1ros%26aufirst%3DT.%26atitle%3DAutophagy%2520regulatory%2520network-a%2520systems-level%2520bioinformatics%2520resource%2520for%2520studying%2520the%2520mechanism%2520and%2520regulation%2520of%2520autophagy%26jtitle%3DAutophagy%26date%3D2015%26volume%3D11%26issue%3D1%26spage%3D155%26epage%3D165%26doi%3D10.4161%2F15548627.2014.994346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilad, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiloh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ber, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bialik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimchi, A.</span></span> <span> </span><span class="NLM_article-title">Discovering protein-protein interactions within the programmed cell death network using a protein-fragment complementation screen</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">909</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.celrep.2014.06.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25066129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1agsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=909-921&issue=3&author=Y.+Giladauthor=R.+Shilohauthor=Y.+Berauthor=S.+Bialikauthor=A.+Kimchi&title=Discovering+protein-protein+interactions+within+the+programmed+cell+death+network+using+a+protein-fragment+complementation+screen&doi=10.1016%2Fj.celrep.2014.06.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Protein-Protein Interactions within the Programmed Cell Death Network Using a Protein-Fragment Complementation Screen</span></div><div class="casAuthors">Gilad, Yuval; Shiloh, Ruth; Ber, Yaara; Bialik, Shani; Kimchi, Adi</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">909-921</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Apoptosis and autophagy are distinct biol. processes, each driven by a different set of protein-protein interactions, with significant crosstalk via direct interactions among apoptotic and autophagic proteins.  To measure the global profile of these interactions, we adapted the Gaussia luciferase protein-fragment complementation assay (GLuc PCA), which monitors binding between proteins fused to complementary fragments of a luciferase reporter.  A library encompassing 63 apoptotic and autophagic proteins was constructed for the anal. of ∼3,600 protein-pair combinations.  This generated a detailed landscape of the apoptotic and autophagic modules and points of interface between them, identifying 46 previously unknown interactions.  One of these interactions, between DAPK2, a Ser/Thr kinase that promotes autophagy, and 14-3-3τ, was further investigated.  We mapped the region responsible for 14-3-3τ binding and proved that this interaction inhibits DAPK2 dimerization and activity.  This proof of concept underscores the power of the GLuc PCA platform for the discovery of biochem. pathways within the cell death network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnFV3lEKHgFLVg90H21EOLACvtfcHk0ljc4NpzTgEfKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1agsL%252FL&md5=22aced64b8cc8645b686f1d0a6b08da6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.049%26sid%3Dliteratum%253Aachs%26aulast%3DGilad%26aufirst%3DY.%26aulast%3DShiloh%26aufirst%3DR.%26aulast%3DBer%26aufirst%3DY.%26aulast%3DBialik%26aufirst%3DS.%26aulast%3DKimchi%26aufirst%3DA.%26atitle%3DDiscovering%2520protein-protein%2520interactions%2520within%2520the%2520programmed%2520cell%2520death%2520network%2520using%2520a%2520protein-fragment%2520complementation%2520screen%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26issue%3D3%26spage%3D909%26epage%3D921%26doi%3D10.1016%2Fj.celrep.2014.06.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Database screening of herbal monomers regulating autophagy by constructing a “disease-gene-drug” network</span>. <i>BMC Complementary Altern. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">466</span>, <span class="refDoi"> DOI: 10.1186/1472-6882-14-466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2F1472-6882-14-466" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25475428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2MzltlOrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=466&author=C.+Haoauthor=Z.+Yangauthor=B.+Gaoauthor=M.+Luauthor=X.+Mengauthor=X.+Qiaoauthor=D.+Xueauthor=W.+Zhang&title=Database+screening+of+herbal+monomers+regulating+autophagy+by+constructing+a+%E2%80%9Cdisease-gene-drug%E2%80%9D+network&doi=10.1186%2F1472-6882-14-466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Database screening of herbal monomers regulating autophagy by constructing a "disease-gene-drug" network</span></div><div class="casAuthors">Hao Chenjun; Yang Zhengpeng; Gao Bo; Lu Ming; Meng Xianzhi; Qiao Xin; Zhang Weihui; Xue Dongbo</div><div class="citationInfo"><span class="NLM_cas:title">BMC complementary and alternative medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">466</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Studies suggest an important role of autophagy as a target for cancer therapy.  We constructed a "disease-gene-drug" network using the modular approach of bioinformatics and screened herbal monomers demonstrating functions related to autophagy regulation.  METHODS:  Based on the microarray results of the gene expression omnibus (GEO) database (GSE2435 and GSE31040, starvation-induced autophagy model), we used the human protein reference database (HPRD) to obtain the protein-protein interaction (PPI) network.  In addition, we used the CFinder software to identify several functional modules, performed gene ontology-biological process (GO-BP) functional enrichment analysis using the DAVID software, constructed a herbal monomer-module gene regulatory network using literature search and the Cytoscape software, and then analyzed the relationships between autophagy, genes, and herbal monomers.  RESULTS:  We screened 544 differentially expressed genes related to autophagy, 375 pairs of differentially expressed genes, and 7 gene modules, wherein the functions of module 3 (composed of 7 genes) were enriched in "cell death".  Using the constructed herbal monomer-module gene regulatory network, we found that 30 herbal monomers can simultaneously regulate these 7 genes, indicating a potential regulatory role in autophagy.  CONCLUSIONS:  Database screening using the disease-gene-drug network can provide new strategies and ideas for the application of herbal medicines in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT8xUK8QhIWR4vKgEt20y1XfW6udTcc2ebzgnBnQU4Sz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzltlOrtw%253D%253D&md5=d3a1edcb08b6d7e31e3e2885198c16f6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1186%2F1472-6882-14-466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6882-14-466%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DQiao%26aufirst%3DX.%26aulast%3DXue%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DDatabase%2520screening%2520of%2520herbal%2520monomers%2520regulating%2520autophagy%2520by%2520constructing%2520a%2520%25E2%2580%259Cdisease-gene-drug%25E2%2580%259D%2520network%26jtitle%3DBMC%2520Complementary%2520Altern.%2520Med.%26date%3D2014%26volume%3D14%26spage%3D466%26doi%3D10.1186%2F1472-6882-14-466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y. Q.</span></span> <span> </span><span class="NLM_article-title">Identification of microRNA-regulated autophagic pathways in plant lectin-induced ccancer cell death</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2184.2012.00840.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fj.1365-2184.2012.00840.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=22882626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GrsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=477-485&issue=5&author=L.+L.+Fuauthor=X.+Zhaoauthor=H.+L.+Xuauthor=X.+Wenauthor=S.+Y.+Wangauthor=B.+Liuauthor=J.+K.+Baoauthor=Y.+Q.+Wei&title=Identification+of+microRNA-regulated+autophagic+pathways+in+plant+lectin-induced+ccancer+cell+death&doi=10.1111%2Fj.1365-2184.2012.00840.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of microRNA-regulated autophagic pathways in plant lectin-induced cancer cell death</span></div><div class="casAuthors">Fu, L.-L.; Zhao, X.; Xu, H.-L.; Wen, X.; Wang, S.-Y.; Liu, B.; Bao, J.-K.; Wei, Y.-Q.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-485</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Plant lectins, carbohydrate-binding proteins of non-immune origin, have recently been reported to induce programmed cell death (including apoptosis and autophagy) in many types of cancer cells.  MicroRNAs (miRNAs), small, non-coding endogenous RNAs, ∼22 nucleotides (nt) in length, have been well characterized to play essential roles in regulation of the autophagy process in cancer; however, how these miRNAs regulate autophagic pathways in plant lectin-induced cancer cells, still remains an enigma.  Identification of microRNA-regulated autophagic pathways was carried out using a series of elegant systems - biol. and bioinformatics approaches, such as network construction, hub protein identification, targeted microRNA prediction, microarray analyses and mol. docking.  We computationally constructed the human autophagic protein-protein interaction (PPI) network, and further modified this network into a plant lectin-induced network.  Subsequently, we identified 9 autophagic hub proteins and 13 relevant oncogenic and tumor suppressive miRNAs, that could regulate these aforementioned targeted autophagic hub proteins, in human breast carcinoma MCF-7 cells.  In addn., we confirmed that plant lectins could block the sugar-contg. receptor EGFR-mediated survival pathways, involved in autophagic hub proteins and relevant miRNAs, thereby ultimately culminating in autophagic cell death.  These results demonstrate that network-based identification of microRNAs modulate autophagic pathways in plant lectin-treated cancer cells, which may shed new light on the discovery of plant lectins as potent autophagic inducers, for cancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr08h3aoUEwS7Vg90H21EOLACvtfcHk0lgdqDEJXPyFWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1GrsbbE&md5=834902129a211fff46450d4a82d4bdcf</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2184.2012.00840.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2184.2012.00840.x%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.%2BL.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%2BL.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%2BY.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBao%26aufirst%3DJ.%2BK.%26aulast%3DWei%26aufirst%3DY.%2BQ.%26atitle%3DIdentification%2520of%2520microRNA-regulated%2520autophagic%2520pathways%2520in%2520plant%2520lectin-induced%2520ccancer%2520cell%2520death%26jtitle%3DCell%2520Proliferation%26date%3D2012%26volume%3D45%26issue%3D5%26spage%3D477%26epage%3D485%26doi%3D10.1111%2Fj.1365-2184.2012.00840.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Identification of novel caspase/autophagy-related gene switch to cell fate decisions in breast cancers</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1111/cpr.12005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fcpr.12005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23289893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslGrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2013&pages=67-75&issue=1&author=L.+L.+Fuauthor=Y.+Yangauthor=H.+L.+Xuauthor=Y.+Chengauthor=X.+Wenauthor=L.+Ouyangauthor=J.+K.+Baoauthor=Y.+Q.+Weiauthor=B.+Liu&title=Identification+of+novel+caspase%2Fautophagy-related+gene+switch+to+cell+fate+decisions+in+breast+cancers&doi=10.1111%2Fcpr.12005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel caspase/autophagy-related gene switch to cell fate decisions in breast cancers</span></div><div class="casAuthors">Fu, L. L.; Yang, Y.; Xu, H. L.; Cheng, Y.; Wen, X.; Ouyang, L.; Bao, J. K.; Wei, Y. Q.; Liu, B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-75</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Objectives : Caspases, a family of cysteine proteases with unique substrate specificities, contribute to apoptosis, whereas autophagy-related genes (ATGs) regulate cytoprotective autophagy or autophagic cell death in cancer.  Accumulating evidence has recently revealed underlying mechanisms of apoptosis and autophagy; however, their intricate relationships still remain to be clarified.  Identification of caspase/ATG switches between apoptosis and autophagy may address this problem.  Materials and methods : Identification of caspase/ATG switches was carried out using a series of elegant systems biol. & bioinformatics approaches, such as network construction, hub protein identification, microarray analyses, targeted microRNA prediction and mol. docking.  Results : We computationally constructed the global human network from several online databases and further modified it into the basic caspase/ATG network.  On the basis of apoptotic or autophagic gene differential expressions, we identified three mol. switches [including androgen receptor, serine/threonine-protein kinase PAK-1 (PAK-1) and mitogen-activated protein kinase-3 (MAPK-3)] between certain caspases and ATGs in human breast carcinoma MCF-7 cells.  Subsequently, we identified microRNAs (miRNAs) able to target androgen receptor, PAK-1 and MAPK-3, resp.  Ultimately, we screened a range of small mol. compds. from DrugBank, able to target the three above-mentioned mol. switches in breast cancer cells.  Conclusions : We have systematically identified novel caspase/ATG switches involved in miRNA regulation, and predicted targeted anti-cancer drugs.  These findings may uncover intricate relationships between apoptosis and autophagy and thus provide further new clues towards possible cancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq66f6u6W3mv7Vg90H21EOLACvtfcHk0lgdqDEJXPyFWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslGrtw%253D%253D&md5=979ceb62ad722681d7102d2ef17ee25a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fcpr.12005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcpr.12005%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%2BL.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DBao%26aufirst%3DJ.%2BK.%26aulast%3DWei%26aufirst%3DY.%2BQ.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520novel%2520caspase%252Fautophagy-related%2520gene%2520switch%2520to%2520cell%2520fate%2520decisions%2520in%2520breast%2520cancers%26jtitle%3DCell%2520Proliferation%26date%3D2013%26volume%3D46%26issue%3D1%26spage%3D67%26epage%3D75%26doi%3D10.1111%2Fcpr.12005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Identification of ULK1 as a novel biomarker involved in miR-4487 and miR-595 regulation in neurobalstoma SH-SY5Y Cell Autophagy</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11035</span>, <span class="refDoi"> DOI: 10.1038/srep11035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fsrep11035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26183158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWgs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11035&author=Y.+Chenauthor=S.+Wangauthor=L.+Zhangauthor=T.+Xieauthor=S.+Songauthor=J.+Huangauthor=Y.+Zhangauthor=L.+Ouyangauthor=B.+Liu&title=Identification+of+ULK1+as+a+novel+biomarker+involved+in+miR-4487+and+miR-595+regulation+in+neurobalstoma+SH-SY5Y+Cell+Autophagy&doi=10.1038%2Fsrep11035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of ULK1 as a novel biomarker involved in miR-4487 and miR-595 regulation in neuroblastoma SH-SY5Y cell autophagy</span></div><div class="casAuthors">Chen, Yi; Wang, Shuya; Zhang, Lan; Xie, Tao; Song, Sicheng; Huang, Jian; Zhang, Yonghui; Ouyang, Liang; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11035</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy, referring to an evolutionarily conserved, multi-step lysosomal degrdn. process, has been well-known to be initiated by Unc-51 like kinase 1 (ULK1) with some links to Parkinson's disease (PD).  MicroRNAs (miRNAs), small and non-coding endogenous RNAs 22 ∼ 24 nucleotides (nt) in length, have been demonstrated to play an essential role for modulating autophagy.  Recently, the relationships between miRNAs and autophagy have been widely reported in PD; however, how microRNAs regulate autophagy still remains in its infancy.  Thus, in this study, we computationally constructed the ULK1-regulated autophagic kinase subnetwork in PD and further identified ULK1 able to neg. regulate p70S6K in starvation-induced autophagy of neuroblastoma SH-SY5Y cells.  Combination of in silico prediction and microarray analyses, we identified that miR-4487 and miR-595 could target ULK1 and exptl. verified they could neg. or pos. regulate ULK1-mediated autophagy.  In conclusion, these results may uncover the novel ULK1-p70S6K autophagic pathway, as well as miR-4487 and miR-595 as new ULK1 target miRNAs.  Thus, these findings would provide a clue to explore ULK1 and its target miRNAs as potential biomarkers in the future PD therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp063qhAPzS8bVg90H21EOLACvtfcHk0lgdqDEJXPyFWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWgs7vM&md5=a7633347eaacfd907d8195066af71a48</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fsrep11035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11035%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520ULK1%2520as%2520a%2520novel%2520biomarker%2520involved%2520in%2520miR-4487%2520and%2520miR-595%2520regulation%2520in%2520neurobalstoma%2520SH-SY5Y%2520Cell%2520Autophagy%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11035%26doi%3D10.1038%2Fsrep11035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Systems biology-based discovery of a potential Atg4B agonist (Flubendazole) that induces autophagy in breast cancer</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2860</span>– <span class="NLM_lpage">2866</span>, <span class="refDoi"> DOI: 10.1039/C5MB00466G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1039%2FC5MB00466G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26299935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWlt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=2860-2866&issue=11&author=L.+Zhangauthor=M.+Guoauthor=J.+Liauthor=Y.+Zhengauthor=S.+Zhangauthor=T.+Xieauthor=B.+Liu&title=Systems+biology-based+discovery+of+a+potential+Atg4B+agonist+%28Flubendazole%29+that+induces+autophagy+in+breast+cancer&doi=10.1039%2FC5MB00466G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology-based discovery of a potential Atg4B agonist (Flubendazole) that induces autophagy in breast cancer</span></div><div class="casAuthors">Zhang, Lan; Guo, Mingrui; Li, Jing; Zheng, Yaxin; Zhang, Shouyue; Xie, Tao; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2860-2866</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The aim of this study was to explore the autophagy-related protein 4B(ATG4B) and its targeted candidate agonist in triple-neg. breast cancer (TNBC) therapy.  In this study, the identification of Atg4B as a novel breast cancer target for screening candidate small mol. agonists was performed by phylogenetic anal., network construction, mol. modeling, mol. docking and mol. dynamics (MD) simulation.  In vitro, MTT assay, electron microscopy, western blot and ROS measurement were used for validating the efficacy of the candidate compds.  We used the phylogenetic anal. of Atg4B and constructed their protein-protein interaction (PPI) network.  Also, we screened target compds. of Atg4 proteins from Drugbank and ZINC.  Flubendazole was validated for its anti-proliferative efficacy in MDA-MB-231 cells.  Further MD simulation results supported the stable interaction between Flubendazole and Atg4B.  Moreover, Flubendazole induced autophagy and increased ROS prodn.  In conclusion, in silico anal. and exptl. validation together demonstrate that Flubendazole can target Atg4B in MDA-MB-231 cells and induce autophagy, which may shed light on the exploration of this compd. as a potential new Atg4B targeted drug for future TNBC therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4pY16N3ASc7Vg90H21EOLACvtfcHk0lgUq-YYhNpWNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWlt73O&md5=f1e946be8d60619445e25a70fde2f7b8</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1039%2FC5MB00466G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MB00466G%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DSystems%2520biology-based%2520discovery%2520of%2520a%2520potential%2520Atg4B%2520agonist%2520%2528Flubendazole%2529%2520that%2520induces%2520autophagy%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520BioSyst.%26date%3D2015%26volume%3D11%26issue%3D11%26spage%3D2860%26epage%3D2866%26doi%3D10.1039%2FC5MB00466G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Q.</span></span> <span> </span><span class="NLM_article-title">Bioinformatic analysis of cacybp-associated proteins using human glioma databases</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1002/liub.1999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1002%2Fliub.1999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=30762928" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=C.+Xuanauthor=Y.+Gaoauthor=M.+Jinauthor=S.+Xuauthor=L.+Wangauthor=Y.+Wangauthor=R.+Hanauthor=K.+Shiauthor=X.+Chenauthor=Q.+An&title=Bioinformatic+analysis+of+cacybp-associated+proteins+using+human+glioma+databases&doi=10.1002%2Fliub.1999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fliub.1999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fliub.1999%26sid%3Dliteratum%253Aachs%26aulast%3DXuan%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAn%26aufirst%3DQ.%26atitle%3DBioinformatic%2520analysis%2520of%2520cacybp-associated%2520proteins%2520using%2520human%2520glioma%2520databases%26jtitle%3DIUBMB%2520Life%26date%3D2019%26doi%3D10.1002%2Fliub.1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shouyue  Zhang</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yang  An</span>, <span class="hlFld-ContribAuthor ">Xiaoxi  Zeng</span>, <span class="hlFld-ContribAuthor ">Ziyi  Qin</span>, <span class="hlFld-ContribAuthor ">Yuqian  Zhao</span>, <span class="hlFld-ContribAuthor ">Heng  Xu</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-omics approaches identify SF3B3 and SIRT3 as candidate autophagic regulators and druggable targets in invasive breast carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 1227-1245. <a href="https://doi.org/10.1016/j.apsb.2020.12.013" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.12.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.12.013%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DMulti-omics%252Bapproaches%252Bidentify%252BSF3B3%252Band%252BSIRT3%252Bas%252Bcandidate%252Bautophagic%252Bregulators%252Band%252Bdruggable%252Btargets%252Bin%252Binvasive%252Bbreast%252Bcarcinoma%26aulast%3DZhang%26aufirst%3DShouyue%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D1227%26epage%3D1245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tuhina  Khan</span>, <span class="hlFld-ContribAuthor ">Nicola  Relitti</span>, <span class="hlFld-ContribAuthor ">Margherita  Brindisi</span>, <span class="hlFld-ContribAuthor ">Stefania  Magnano</span>, <span class="hlFld-ContribAuthor ">Daniela  Zisterer</span>, <span class="hlFld-ContribAuthor ">Sandra  Gemma</span>, <span class="hlFld-ContribAuthor ">Stefania  Butini</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Campiani</span>. </span><span class="cited-content_cbyCitation_article-title">Autophagy modulators for the treatment of oral and esophageal squamous cell carcinomas. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (3)
                                     , 1002-1060. <a href="https://doi.org/10.1002/med.21646" title="DOI URL">https://doi.org/10.1002/med.21646</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21646%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DAutophagy%252Bmodulators%252Bfor%252Bthe%252Btreatment%252Bof%252Boral%252Band%252Besophageal%252Bsquamous%252Bcell%252Bcarcinomas%26aulast%3DKhan%26aufirst%3DTuhina%26date%3D2020%26date%3D2019%26volume%3D40%26issue%3D3%26spage%3D1002%26epage%3D1060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/medium/jm-2018-01673b_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/medium/jm-2018-01673b_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic model of the representative small-molecule compounds targeting autophagic proteins or autophagic process in different types of diseases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/medium/jm-2018-01673b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic model of in silico autophagy methods for expediting the current medicinal research. A variety of in silico autophagy approaches for expediting medicinal research are classified as follows: (A) databases and Web servers, (B) mathematics models, (C) omics approaches, and (D) systems-biology network approaches.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.8b01673/20190801/images/large/jm-2018-01673b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01673&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77874" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77874" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emr, S. D.</span></span> <span> </span><span class="NLM_article-title">Autophagy as a regulated pathway of cellular degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">1717</span>– <span class="NLM_lpage">1721</span>, <span class="refDoi"> DOI: 10.1126/science.290.5497.1717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1126%2Fscience.290.5497.1717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=11099404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVOitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2000&pages=1717-1721&author=D.+J.+Klionskyauthor=S.+D.+Emr&title=Autophagy+as+a+regulated+pathway+of+cellular+degradation&doi=10.1126%2Fscience.290.5497.1717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Review: Cell biology: Autophagy as a regulated pathway of cellular degradation</span></div><div class="casAuthors">Klionsky, Daniel J.; Emr, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">5497</span>),
    <span class="NLM_cas:pages">1717-1721</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review, with 83 refs.  Macroautophagy is a dynamic process involving the rearrangement of subcellular membranes to sequester cytoplasm and organelles for delivery to the lysosome or vacuole where the sequestered cargo is degraded and recycled.  This process takes place in all eukaryotic cells.  It is highly regulated through the action of various kinases, phosphatases, and GTPases.  The core protein machinery that is necessary to drive formation and consumption of intermediates in the macroautophagy pathway includes a ubiquitin-like protein conjugation system and a protein complex that directs membrane docking and fusion at the lysosome or vacuole.  Macroautophagy plays an important role in developmental processes, human disease, and cellular response to nutrient deprivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLO3Uce-OuRbVg90H21EOLACvtfcHk0lhQTUUIEVwGTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVOitbw%253D&md5=0ad887bd4edc4fb1c0cd45764ff6896d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.290.5497.1717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.290.5497.1717%26sid%3Dliteratum%253Aachs%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26aulast%3DEmr%26aufirst%3DS.%2BD.%26atitle%3DAutophagy%2520as%2520a%2520regulated%2520pathway%2520of%2520cellular%2520degradation%26jtitle%3DScience%26date%3D2000%26volume%3D290%26spage%3D1717%26epage%3D1721%26doi%3D10.1126%2Fscience.290.5497.1717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Autophagy: from phenomenology to molecular understanding in less than a decade</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">931</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1038/nrm2245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fnrm2245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=17712358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=931-937&author=D.+J.+Klionsky&title=Autophagy%3A+from+phenomenology+to+molecular+understanding+in+less+than+a+decade&doi=10.1038%2Fnrm2245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: from phenomenology to molecular understanding in less than a decade</span></div><div class="casAuthors">Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">931-937</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In 2000, it was suggested to the author that "autophagy will be the wave of the future: it will become the new apoptosis.".  Few people would have agreed at the time, but this statement has turned out to be prophetic, and this process of 'self-eating' has rapidly exploded as a research field, as scientists discovered connections to cancer, neurodegeneration, and even lifespan extension.  Amazingly, although autophagy was 1st discovered ∼40 yr ago, the mol. breakthroughs in autophagy have taken place during only the past decade.  A time-line covering the history of autophagy research is provided, with a focus on the key events that have led to its increasing prominence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSmR5Px0Qh87Vg90H21EOLACvtfcHk0lhQTUUIEVwGTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsL3I&md5=0498357d8b0c64add87fb82e89770572</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrm2245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2245%26sid%3Dliteratum%253Aachs%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DAutophagy%253A%2520from%2520phenomenology%2520to%2520molecular%2520understanding%2520in%2520less%2520than%2520a%2520decade%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D931%26epage%3D937%26doi%3D10.1038%2Fnrm2245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooze, S. A.</span></span> <span> </span><span class="NLM_article-title">The autophagosome: origins unknown, biogenesis complex</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">774</span>, <span class="refDoi"> DOI: 10.1038/nrm3696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fnrm3696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=24201109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=759-774&issue=12&author=C.+A.+Lambauthor=T.+Yoshimoriauthor=S.+A.+Tooze&title=The+autophagosome%3A+origins+unknown%2C+biogenesis+complex&doi=10.1038%2Fnrm3696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The autophagosome: origins unknown, biogenesis complex</span></div><div class="casAuthors">Lamb, Christopher A.; Yoshimori, Tamotsu; Tooze, Sharon A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">759-774</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Healthy cells use autophagy as a general 'housekeeping' mechanism and to survive stress, including stress induced by nutrient deprivation.  Autophagy is initiated at the isolation membrane (originally termed the phagophore), and the coordinated action of ATG (autophagy-related) proteins results in the expansion of this membrane to form the autophagosome.  Although the biogenesis of the isolation membrane and the autophagosome is complex and incompletely understood, insight has been gained into the mol. processes involved in initiating the isolation membrane, the source from which this originates (for example, it was recently proposed that the isolation membrane forms from the mitochondria-assocd. endoplasmic reticulum (ER) membrane (MAM)) and the role of ATG proteins and the vesicular trafficking machinery in autophagosome formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtLSmFqUjw0LVg90H21EOLACvtfcHk0lg02mfk691_1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCnur%252FJ&md5=1e0d82497a3d6c319c3a14ef1ffd98ad</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3696%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DC.%2BA.%26aulast%3DYoshimori%26aufirst%3DT.%26aulast%3DTooze%26aufirst%3DS.%2BA.%26atitle%3DThe%2520autophagosome%253A%2520origins%2520unknown%252C%2520biogenesis%2520complex%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26issue%3D12%26spage%3D759%26epage%3D774%26doi%3D10.1038%2Fnrm3696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Autophagy in the pathogenesis of disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.12.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.cell.2007.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=18191218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFehtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2008&pages=27-42&author=B.+Levineauthor=G.+Kroemer&title=Autophagy+in+the+pathogenesis+of+disease&doi=10.1016%2Fj.cell.2007.12.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in the pathogenesis of disease</span></div><div class="casAuthors">Levine, Beth; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-42</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The role of mTOR kinase-related multiple pathways as regulators of autophagy in diseases is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEJaHO5xPdGbVg90H21EOLACvtfcHk0lg02mfk691_1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFehtb8%253D&md5=330b3696429911990b51385919e2bd39</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DAutophagy%2520in%2520the%2520pathogenesis%2520of%2520disease%26jtitle%3DCell%26date%3D2008%26volume%3D132%26spage%3D27%26epage%3D42%26doi%3D10.1016%2Fj.cell.2007.12.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Core signaling pathways of survival/death in autophagy-related cancer networks</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1263</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2011.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.biocel.2011.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=21640844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1Olurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=1263-1266&author=S.+Y.+Wangauthor=Q.+J.+Yuauthor=R.+D.+Zhangauthor=B.+Liu&title=Core+signaling+pathways+of+survival%2Fdeath+in+autophagy-related+cancer+networks&doi=10.1016%2Fj.biocel.2011.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Core signaling pathways of survival/death in autophagy-related cancer networks</span></div><div class="casAuthors">Wang, Shu-ya; Yu, Qi-jia; Zhang, Ru-dian; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1263-1266</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Autophagy (macroautophagy), an evolutionarily conserved lysosomal degrdn. process, is implicated in a wide variety of pathol. processes including cancer.  Autophagy plays the Janus role in regulating several survival or death signaling pathways that may decide the fate of cancer cell.  Accumulating evidence has revealed the core mol. machinery of autophagy in tumor initiation and progression; however, the intricate relationships between autophagy and cancer are still in its infancy.  In this review, we summarize several key survival/death pathways such as mTOR subnetwork, Beclin 1 interactome, and p53 signaling that may play the crucial roles for the regulation of the autophagy-related cancer networks.  Therefore, a better understanding of the relationships between autophagy and cancer may ultimately allow cancer biologists and clinicians to harness core autophagic pathways for the discovery of potential novel drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroJKxaLZqWqLVg90H21EOLACvtfcHk0lg02mfk691_1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1Olurc%253D&md5=e4f3695908b438f07d555da9e9bc3e87</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2011.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2011.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%2BY.%26aulast%3DYu%26aufirst%3DQ.%2BJ.%26aulast%3DZhang%26aufirst%3DR.%2BD.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DCore%2520signaling%2520pathways%2520of%2520survival%252Fdeath%2520in%2520autophagy-related%2520cancer%2520networks%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2011%26volume%3D43%26spage%3D1263%26epage%3D1266%26doi%3D10.1016%2Fj.biocel.2011.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span> <span> </span><span class="NLM_article-title">Survival or death: disequilibrating the oncogenic and tumor suppressive autophagy in cancer</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">e892</span>, <span class="refDoi"> DOI: 10.1038/cddis.2013.422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fcddis.2013.422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=24176850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCrur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=e892&author=B.+Liuauthor=X.+Wenauthor=Y.+Cheng&title=Survival+or+death%3A+disequilibrating+the+oncogenic+and+tumor+suppressive+autophagy+in+cancer&doi=10.1038%2Fcddis.2013.422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Survival or death: disequilibrating the oncogenic and tumor suppressive autophagy in cancer</span></div><div class="casAuthors">Liu, B.; Wen, X.; Cheng, Y.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">e892</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Autophagy (macroautophagy) is an evolutionarily conserved lysosomal degrdn. process, in which a cell degrades long-lived proteins and damaged organelles.  Recently, accumulating evidence has revealed the core mol. machinery of autophagy in carcinogenesis; however, the intricate relationship between autophagy and cancer continue to remain an enigma.  Why does autophagy have either pro-survival (oncogenic) or pro-death (tumor suppressive) role at different cancer stages, including cancer stem cell, initiation and progression, invasion and metastasis, as well as dormancy.  How does autophagy modulate a series of oncogenic and/or tumor suppressive pathways, implicated in microRNA (miRNA) involvement.  Whether would targeting the oncogenic and tumor suppressive autophagic network be a novel strategy for drug discovery.  To address these problems, we focus on summarizing the dynamic oncogenic and tumor suppressive roles of autophagy and their relevant small-mol. drugs, which would provide a new clue to elucidate the oncosuppressive (survival or death) autophagic network as a potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2wUhKw1JqAbVg90H21EOLACvtfcHk0ljWmjtm0p8hCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCrur3L&md5=1914543371d0618accc380fe95186e6c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.422%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DY.%26atitle%3DSurvival%2520or%2520death%253A%2520disequilibrating%2520the%2520oncogenic%2520and%2520tumor%2520suppressive%2520autophagy%2520in%2520cancer%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2013%26volume%3D4%26spage%3De892%26doi%3D10.1038%2Fcddis.2013.422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ylä-Anttila, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vihinen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jokitalo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eskelinen, E. L.</span></span> <span> </span><span class="NLM_article-title">Monitoring autophagy by electron microscopy in Mammalian cells</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>452</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(08)03610-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2FS0076-6879%2808%2903610-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=19200881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkt1eqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2009&pages=143-164&author=P.+Yl%C3%A4-Anttilaauthor=H.+Vihinenauthor=E.+Jokitaloauthor=E.+L.+Eskelinen&title=Monitoring+autophagy+by+electron+microscopy+in+Mammalian+cells&doi=10.1016%2FS0076-6879%2808%2903610-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring autophagy by electron microscopy in mammalian cells</span></div><div class="casAuthors">Yla-Anttila, Paivi; Vihinen, Helena; Jokitalo, Eija; Eskelinen, Eeva-Liisa</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">452</span>
        (<span class="NLM_cas:issue">Autophagy in Mammalian Systems, Part B</span>),
    <span class="NLM_cas:pages">143-164</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Electron microscopy remains one of the most accurate methods for the detection of autophagy and quantification of autophagic accumulation.  Compared to fluorescence microscopy, the resoln. of transmission electron microscopy is superior.  In this chapter we describe the fine structure of early and late autophagic compartments in mammalian cells.  Instructions are given for the prepn. of samples for conventional electron microscopy using three different protocols suitable for cultured cells and animal tissues.  We also introduce tomog. as a tool to study the three-dimensional morphol. of autophagic organelles and show the morphol. of a phagophore as an example.  Finally, we describe a protocol for the quantification of autophagic compartments by electron microscopy and point counting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgD9_Nlt8MRLVg90H21EOLACvtfcHk0ljWmjtm0p8hCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkt1eqs78%253D&md5=727bdf41f134b23b61d3e9a9de9e35e4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2808%2903610-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252808%252903610-0%26sid%3Dliteratum%253Aachs%26aulast%3DYl%25C3%25A4-Anttila%26aufirst%3DP.%26aulast%3DVihinen%26aufirst%3DH.%26aulast%3DJokitalo%26aufirst%3DE.%26aulast%3DEskelinen%26aufirst%3DE.%2BL.%26atitle%3DMonitoring%2520autophagy%2520by%2520electron%2520microscopy%2520in%2520Mammalian%2520cells%26jtitle%3DMethods%2520Enzymol.%26date%3D2009%26volume%3D452%26spage%3D143%26epage%3D164%26doi%3D10.1016%2FS0076-6879%2808%2903610-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span> <span> </span><span class="NLM_article-title">Deconvoluting the role of reactive oxygen species and autophagy in human diseases</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">402</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2013.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.freeradbiomed.2013.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23872397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFylurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2013&pages=402-410&issue=6&author=X.+Wenauthor=J.+M.+Wuauthor=F.+T.+Wangauthor=B.+Liuauthor=C.+Huangauthor=Y.+Wei&title=Deconvoluting+the+role+of+reactive+oxygen+species+and+autophagy+in+human+diseases&doi=10.1016%2Fj.freeradbiomed.2013.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Deconvoluting the role of reactive oxygen species and autophagy in human diseases</span></div><div class="casAuthors">Wen, Xin; Wu, Jinming; Wang, Fengtian; Liu, Bo; Huang, Canhua; Wei, Yuquan</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">402-410</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Reactive oxygen species (ROS), chem. reactive mols. contg. oxygen, can form as a natural byproduct of the normal metab. of oxygen and also have their crucial roles in cell homeostasis.  Of note, the major intracellular sources including mitochondria, endoplasmic reticulum (ER), peroxisomes and the NADPH oxidase (NOX) complex have been identified in cell membranes to produce ROS.  Interestingly, autophagy, an evolutionarily conserved lysosomal degrdn. process in which a cell degrades long-lived proteins and damaged organelles, has recently been well-characterized to be regulated by different types of ROS.  Accumulating evidence has demonstrated that ROS-modulated autophagy has numerous links to a no. of pathol. processes, including cancer, aging, neurodegenerative diseases, type-II diabetes, cardiovascular diseases, muscular disorders, hepatic encephalopathy and immunity diseases.  In this review, we focus on summarizing the mol. mechanisms of ROS-regulated autophagy and their relevance to diverse diseases, which would shed new light on more ROS modulators as potential therapeutic drugs for fighting human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMLM_rtLoDZLVg90H21EOLACvtfcHk0ljWmjtm0p8hCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFylurvJ&md5=bb75bd646e83e92fdd4d183e29f83329</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2013.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2013.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DF.%2BT.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DY.%26atitle%3DDeconvoluting%2520the%2520role%2520of%2520reactive%2520oxygen%2520species%2520and%2520autophagy%2520in%2520human%2520diseases%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2013%26volume%3D65%26issue%3D6%26spage%3D402%26epage%3D410%26doi%3D10.1016%2Fj.freeradbiomed.2013.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span> <span> </span><span class="NLM_article-title">Autophagy and metabolism</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span>,  <span class="NLM_fpage">1344</span>– <span class="NLM_lpage">1348</span>, <span class="refDoi"> DOI: 10.1126/science.1193497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1126%2Fscience.1193497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=21127245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=2010&pages=1344-1348&author=J.+D.+Rabinowitzauthor=E.+White&title=Autophagy+and+metabolism&doi=10.1126%2Fscience.1193497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy and metabolism</span></div><div class="casAuthors">Rabinowitz, Joshua D.; White, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6009</span>),
    <span class="NLM_cas:pages">1344-1348</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Autophagy is a process of self-cannibalization.  Cells capture their own cytoplasm and organelles and consume them in lysosomes.  The resulting breakdown products are inputs to cellular metab., through which they are used to generate energy and to build new proteins and membranes.  Autophagy preserves the health of cells and tissues by replacing outdated and damaged cellular components with fresh ones.  In starvation, it provides an internal source of nutrients for energy generation and, thus, survival.  A powerful promoter of metabolic homeostasis at both the cellular and whole-animal level, autophagy prevents degenerative diseases.  It does have a downside, however; cancer cells exploit it to survive in nutrient-poor tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREWU_wFW6GLVg90H21EOLACvtfcHk0lje6tdr7USzcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsbbE&md5=4b6ed3a623cbef8df1241579e15632fa</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1193497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1193497%26sid%3Dliteratum%253Aachs%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DWhite%26aufirst%3DE.%26atitle%3DAutophagy%2520and%2520metabolism%26jtitle%3DScience%26date%3D2010%26volume%3D330%26spage%3D1344%26epage%3D1348%26doi%3D10.1126%2Fscience.1193497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Eaten alive: a history of macroautophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">814</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fncb0910-814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=20811353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=814-822&author=Z.+Yangauthor=D.+J.+Klionsky&title=Eaten+alive%3A+a+history+of+macroautophagy&doi=10.1038%2Fncb0910-814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Eaten alive: a history of macroautophagy</span></div><div class="casAuthors">Yang, Zhifen; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">814-822</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter autophagy), or 'self-eating', is a conserved cellular pathway that controls protein and organelle degrdn., and has essential roles in survival, development and homeostasis.  Autophagy is also integral to human health and is involved in physiol., development, lifespan and a wide range of diseases, including cancer, neurodegeneration and microbial infection.  Although research on this topic began in the late 1950s, substantial progress in the mol. study of autophagy has taken place during only the past 15 years.  This review traces the key findings that led to our current mol. understanding of this complex process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_iZJ3YUoHrVg90H21EOLACvtfcHk0lje6tdr7USzcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE&md5=a20abac7db3f20eac668c41052413397</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-814%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DEaten%2520alive%253A%2520a%2520history%2520of%2520macroautophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D814%26epage%3D822%26doi%3D10.1038%2Fncb0910-814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Biological functions of autophagy genes: a disease perspective</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>176</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.cell.2018.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=30633901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFyquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2019&pages=11-42&author=B.+Levineauthor=G.+Kroemer&title=Biological+functions+of+autophagy+genes%3A+a+disease+perspective&doi=10.1016%2Fj.cell.2018.09.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Biological Functions of Autophagy Genes: A Disease Perspective</span></div><div class="casAuthors">Levine, Beth; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">11-42</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The lysosomal degrdn. pathway of autophagy plays a fundamental role in cellular, tissue, and organismal homeostasis and is mediated by evolutionarily conserved autophagy-related (ATG) genes.  Definitive etiol. links exist between mutations in genes that control autophagy and human disease, esp. neurodegenerative, inflammatory disorders and cancer.  Autophagy selectively targets dysfunctional organelles, intracellular microbes, and pathogenic proteins, and deficiencies in these processes may lead to disease.  Moreover, ATG genes have diverse physiol. important roles in other membrane-trafficking and signaling pathways.  This Review discusses the biol. functions of autophagy genes from the perspective of understanding - and potentially reversing - the pathophysiol. of human disease and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUer8pL-luJrVg90H21EOLACvtfcHk0lje6tdr7USzcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFyquw%253D%253D&md5=65c2303ca9ea78c00cae26a3ff5a9b73</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DBiological%2520functions%2520of%2520autophagy%2520genes%253A%2520a%2520disease%2520perspective%26jtitle%3DCell%26date%3D2019%26volume%3D176%26spage%3D11%26epage%3D42%26doi%3D10.1016%2Fj.cell.2018.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of autophagy and relevant small-molecule compounds for targeted cancer therapy</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1803</span>– <span class="NLM_lpage">1826</span>, <span class="refDoi"> DOI: 10.1007/s00018-018-2759-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1007%2Fs00018-018-2759-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=29417176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2018&pages=1803-1826&issue=10&author=J.+Zhangauthor=G.+Wangauthor=Y.+Zhouauthor=Y.+Chenauthor=L.+Ouyangauthor=B.+Liu&title=Mechanisms+of+autophagy+and+relevant+small-molecule+compounds+for+targeted+cancer+therapy&doi=10.1007%2Fs00018-018-2759-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of autophagy and relevant small-molecule compounds for targeted cancer therapy</span></div><div class="casAuthors">Zhang, Jin; Wang, Guan; Zhou, Yuxin; Chen, Yi; Ouyang, Liang; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1803-1826</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Autophagy is an evolutionarily conserved, multi-step lysosomal degrdn. process for the clearance of damaged or superfluous proteins and organelles.  Accumulating studies have recently revealed that autophagy is closely related to a variety of types of cancer; however, elucidation of its Janus role of either tumor-suppressive or tumor-promoting still remains to be discovered.  In this review, we focus on summarizing the context-dependent role of autophagy and its complicated mol. mechanisms in different types of cancer.  Moreover, we discuss a series of small-mol. compds. targeting autophagy-related proteins or the autophagic process for potential cancer therapy.  Taken together, these findings would shed new light on exploiting the intricate mechanisms of autophagy and relevant small-mol. compds. as potential anti-cancer drugs to improve targeted cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolGggPdyfsHrVg90H21EOLACvtfcHk0lj-3-P6ONLOUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKjtbY%253D&md5=7efbbc0b553a8843dba97c8d341a4853</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00018-018-2759-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-018-2759-2%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DMechanisms%2520of%2520autophagy%2520and%2520relevant%2520small-molecule%2520compounds%2520for%2520targeted%2520cancer%2520therapy%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2018%26volume%3D75%26issue%3D10%26spage%3D1803%26epage%3D1826%26doi%3D10.1007%2Fs00018-018-2759-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryter, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span> <span> </span><span class="NLM_article-title">Autophagy in human health and disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>368</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1205406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1056%2FNEJMra1205406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23406030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivVWjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2013&pages=651-662&issue=7&author=A.+M.+Choiauthor=S.+W.+Ryterauthor=B.+Levine&title=Autophagy+in+human+health+and+disease&doi=10.1056%2FNEJMra1205406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in human health and disease</span></div><div class="casAuthors">Choi, Augustine M. K.; Ryter, Stefan W.; Levine, Beth</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">651-662</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  Autophagy is now widely implicated in pathophysiol. processes (e.g., cancer, metabolic and neurodegenerative disorders, and cardiovascular and pulmonary diseases) and in physiol. responses to exercise and aging.  This article discusses on the autophagy in human health and disease.  An improved understanding of the mechanism (or mechanisms) by which autophagy can prevent pathogenesis may lead to the identification of new targets for both diagnostic and therapeutic approaches.  Drug screening for agonists or antagonists of autophagic activity, including upstream regulators and downstream targets of autophagy, may yield addnl. therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP5LjXnUJzhrVg90H21EOLACvtfcHk0lj-3-P6ONLOUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivVWjsr0%253D&md5=be5f49522caa165d5bf18b4b50edfeec</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1205406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1205406%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DA.%2BM.%26aulast%3DRyter%26aufirst%3DS.%2BW.%26aulast%3DLevine%26aufirst%3DB.%26atitle%3DAutophagy%2520in%2520human%2520health%2520and%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D368%26issue%3D7%26spage%3D651%26epage%3D662%26doi%3D10.1056%2FNEJMra1205406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Homma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, H.</span></span> <span> </span><span class="NLM_article-title">The autophagy database: an all-inclusive information resource on autophagy that provides nourishment for research</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D986</span>– <span class="NLM_lpage">D990</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1093%2Fnar%2Fgkq995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=20972215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2lurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D986-D990&author=K.+Hommaauthor=K.+Suzukiauthor=H.+Sugawara&title=The+autophagy+database%3A+an+all-inclusive+information+resource+on+autophagy+that+provides+nourishment+for+research&doi=10.1093%2Fnar%2Fgkq995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Autophagy Database: an all-inclusive information resource on autophagy that provides nourishment for research</span></div><div class="casAuthors">Homma, Keiichi; Suzuki, Koji; Sugawara, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D986-D990</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Autophagy is a process of self-digestion generally obsd. in eukaryotes and has been shown to play crucial roles for survival under starvation and removal of deleterious substances.  Despite great advances that have been made, many problems in mechanisms of autophagy remain unsolved.  As a large no. of autophagy-related proteins are identified in each species, a database that collects data, identifies their homologs in other species and makes them available will contribute to research advancement.  As no such resources exist, we built the Autophagy database (http://tp-apg.genes.nig.ac.jp/autophagy) to provide basics, up-to-date information on relevant literature, and a list of autophagy-related proteins and their homologs in 41 eukaryotes.  From the database, the user can search for proteins by keywords or sequences to obtain a wealth of data including functional and structural information and find possible functional homologs of proteins whose functions have been demonstrated in other species.  As proteins that bind the phospholipid, phosphatidyl inositol 3-phosphate (PI3P) are essential for autophagy to proceed, we carried out an original anal. to identify probable PI3P-binding proteins, and made the list available from the database.  The database is expected to give impetus to further research on autophagy by providing basic and specialized data on the subject.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-YW_gAyhH7Vg90H21EOLACvtfcHk0lj-3-P6ONLOUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2lurc%253D&md5=8663738874465b0ea579bedfeb970da6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq995%26sid%3Dliteratum%253Aachs%26aulast%3DHomma%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DSugawara%26aufirst%3DH.%26atitle%3DThe%2520autophagy%2520database%253A%2520an%2520all-inclusive%2520information%2520resource%2520on%2520autophagy%2520that%2520provides%2520nourishment%2520for%2520research%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD986%26epage%3DD990%26doi%3D10.1093%2Fnar%2Fgkq995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacomin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samavedam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Promponas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nezis, I. P.</span></span> <span> </span><span class="NLM_article-title">iLIR database: a web resource for LIR motif-containing proteins in eukaryotes</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1945</span>– <span class="NLM_lpage">1953</span>, <span class="refDoi"> DOI: 10.1080/15548627.2016.1207016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2016.1207016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27484196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1945-1953&issue=10&author=A.+C.+Jacominauthor=S.+Samavedamauthor=V.+Promponasauthor=I.+P.+Nezis&title=iLIR+database%3A+a+web+resource+for+LIR+motif-containing+proteins+in+eukaryotes&doi=10.1080%2F15548627.2016.1207016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">iLIR database: A web resource for LIR motif-containing proteins in eukaryotes</span></div><div class="casAuthors">Jacomin, Anne-Claire; Samavedam, Siva; Promponas, Vasilis; Nezis, Ioannis P.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1945-1953</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Atg8-family proteins are the best-studied proteins of the core autophagic machinery.  They are essential for the elongation and closure of the phagophore into a proper autophagosome.  Moreover, Atg8-family proteins are assocd. with the phagophore from the initiation of the autophagic process to, or just prior to, the fusion between autophagosomes with lysosomes.  In addn. to their implication in autophagosome biogenesis, they are crucial for selective autophagy through their ability to interact with selective autophagy receptor proteins necessary for the specific targeting of substrates for autophagic degrdn.  In the past few years it has been revealed that Atg8-interacting proteins include not only receptors but also components of the core autophagic machinery, proteins assocd. with vesicles and their transport, and specific proteins that are selectively degraded by autophagy.  Atg8-interacting proteins contain a short linear LC3-interacting region/LC3 recognition sequence/Atg8-interacting motif (LIR/LRS/AIM) motif which is responsible for their interaction with Atg8-family proteins.  These proteins are referred to as LIR-contg. proteins (LIRCPs).  So far, many exptl. efforts have been carried out to identify new LIRCPs, leading to the characterization of some of them in the past 10 years.  Given the need for the identification of LIRCPs in various organisms, we developed the iLIR database (https://ilir.warwick.ac.uk) as a freely available web resource, listing all the putative canonical LIRCPs identified in silico in the proteomes of 8 model organisms using the iLIR server, combined with a Gene Ontol. (GO) term anal.  Addnl., a curated text-mining anal. of the literature permitted us to identify novel putative LICRPs in mammals that have not previously been assocd. with autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0mr_HkTMOoLVg90H21EOLACvtfcHk0lgofzstaRY81w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnur3F&md5=0fa319f294097a9d96ef2e9caa950b09</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F15548627.2016.1207016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2016.1207016%26sid%3Dliteratum%253Aachs%26aulast%3DJacomin%26aufirst%3DA.%2BC.%26aulast%3DSamavedam%26aufirst%3DS.%26aulast%3DPromponas%26aufirst%3DV.%26aulast%3DNezis%26aufirst%3DI.%2BP.%26atitle%3DiLIR%2520database%253A%2520a%2520web%2520resource%2520for%2520LIR%2520motif-containing%2520proteins%2520in%2520eukaryotes%26jtitle%3DAutophagy%26date%3D2016%26volume%3D12%26issue%3D10%26spage%3D1945%26epage%3D1953%26doi%3D10.1080%2F15548627.2016.1207016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzfadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weithorn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peled-Zehavi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Parys, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van de Peer, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galili, G.</span></span> <span> </span><span class="NLM_article-title">hfAIM: a reliable bioinformatics approach for in silico genome-wide identification of autophagy-associated Atg8-interacting motifs in vatious organisms</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">887</span>, <span class="refDoi"> DOI: 10.1080/15548627.2016.1147668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2016.1147668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27071037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1eis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=876-887&issue=5&author=Q.+Xieauthor=O.+Tzfadiaauthor=M.+Levyauthor=E.+Weithornauthor=H.+Peled-Zehaviauthor=T.+Van+Parysauthor=Y.+Van+de+Peerauthor=G.+Galili&title=hfAIM%3A+a+reliable+bioinformatics+approach+for+in+silico+genome-wide+identification+of+autophagy-associated+Atg8-interacting+motifs+in+vatious+organisms&doi=10.1080%2F15548627.2016.1147668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">hfAIM: A reliable bioinformatics approach for in silico genome-wide identification of autophagy-associated Atg8-interacting motifs in various organisms</span></div><div class="casAuthors">Xie, Qingjun; Tzfadia, Oren; Levy, Matan; Weithorn, Efrat; Peled-Zehavi, Hadas; Van Parys, Thomas; Van de Peer, Yves; Galili, Gad</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">876-887</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Most of the proteins that are specifically turned over by selective autophagy are recognized by the presence of short Atg8 interacting motifs (AIMs) that facilitate their assocn. with the autophagy app.  Such AIMs can be identified by bioinformatics methods based on their defined degenerate consensus F/W/Y-X-X-L/I/V sequences in which X represents any amino acid.  Achieving reliability and/or fidelity of the prediction of such AIMs on a genome-wide scale represents a major challenge.  Here, we present a bioinformatics approach, high fidelity AIM (hfAIM), which uses addnl. sequence requirements-the presence of acidic amino acids and the absence of pos. charged amino acids in certain positions-to reliably identify AIMs in proteins.  We demonstrate that the use of the hfAIM method allows for in silico high fidelity prediction of AIMs in AIM-contg. proteins (ACPs) on a genome-wide scale in various organisms.  Furthermore, by using hfAIM to identify putative AIMs in the Arabidopsis proteome, we illustrate a potential contribution of selective autophagy to various biol. processes.  More specifically, we identified 9 peroxisomal PEX proteins that contain hfAIM motifs, among which AtPEX1, AtPEX6 and AtPEX10 possess evolutionary-conserved AIMs.  Bimol. fluorescence complementation (BiFC) results verified that AtPEX6 and AtPEX10 indeed interact with Atg8 in planta.  In addn., we show that mutations occurring within or nearby hfAIMs in PEX1, PEX6 and PEX10 caused defects in the growth and development of various organisms.  Taken together, the above results suggest that the hfAIM tool can be used to effectively perform genome-wide in silico screens of proteins that are potentially regulated by selective autophagy.  The hfAIM system is a web tool that can be accessed at link: http://bioinformatics.psb.ugent.be/hfAIM/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2DSVeo6k6SrVg90H21EOLACvtfcHk0lgofzstaRY81w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1eis7k%253D&md5=68574d1ab076e5ccb519d87898140c72</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F15548627.2016.1147668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2016.1147668%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DTzfadia%26aufirst%3DO.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DWeithorn%26aufirst%3DE.%26aulast%3DPeled-Zehavi%26aufirst%3DH.%26aulast%3DVan%2BParys%26aufirst%3DT.%26aulast%3DVan%2Bde%2BPeer%26aufirst%3DY.%26aulast%3DGalili%26aufirst%3DG.%26atitle%3DhfAIM%253A%2520a%2520reliable%2520bioinformatics%2520approach%2520for%2520in%2520silico%2520genome-wide%2520identification%2520of%2520autophagy-associated%2520Atg8-interacting%2520motifs%2520in%2520vatious%2520organisms%26jtitle%3DAutophagy%26date%3D2016%26volume%3D12%26issue%3D5%26spage%3D876%26epage%3D887%26doi%3D10.1080%2F15548627.2016.1147668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moussay, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaoma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baginska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Moer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazarov, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chouaib, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berchem, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janji, B.</span></span> <span> </span><span class="NLM_article-title">The acquisition of resistance to TNFα in breast cancer cells is associated with constitutive activation of autophagy as revealed by a transcriptome analysis using a custom microarray</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">770</span>, <span class="refDoi"> DOI: 10.4161/auto.7.7.15454</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4161%2Fauto.7.7.15454" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=21490427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=760-770&issue=7&author=E.+Moussayauthor=T.+Kaomaauthor=J.+Baginskaauthor=A.+Mullerauthor=K.+Van+Moerauthor=N.+Nicotauthor=P.+V.+Nazarovauthor=L.+Vallarauthor=S.+Chouaibauthor=G.+Berchemauthor=B.+Janji&title=The+acquisition+of+resistance+to+TNF%CE%B1+in+breast+cancer+cells+is+associated+with+constitutive+activation+of+autophagy+as+revealed+by+a+transcriptome+analysis+using+a+custom+microarray&doi=10.4161%2Fauto.7.7.15454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The acquisition of resistance to TNFα in breast cancer cells is associated with constitutive activation of autophagy as revealed by a transcriptome analysis using a custom microarray</span></div><div class="casAuthors">Moussay, Etienne; Kaoma, Tony; Baginska, Joanna; Muller, Arnaud; Van Moer, Kris; Nicot, Nathalie; Nazarov, Petr V.; Vallar, Laurent; Chouaib, Salem; Berchem, Guy; Janji, Bassam</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">760-770</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">While the autophagic process is mainly regulated at the post-translational level, a growing body of evidence suggests that autophagy might also be regulated at the transcriptional level.  The identification of transcription factors involved in the regulation of autophagy genes has provided compelling evidence for such regulation.  In this context, a powerful high throughput anal. tool to simultaneously monitor the expression level of autophagy genes is urgently needed.  Here we describe setting up the first comprehensive human autophagy database (HADb, available at www.autophagy.lu) and the development of a companion Human Autophagy-dedicated cDNA Microarray which comprises 234 genes involved in or related to autophagy.  The autophagy microarray tool used on breast adenocarcinoma MCF-7 cell line allowed the identification of 47 differentially expressed autophagy genes assocd. with the acquisition of resistance to the cytotoxic effect of TNFα.  The autophagy-core machinery genes DRAM (Damage-Regulated Autophagy Modulator), BNIP3L (BCL2/adenovirus E1B 19 kDa interacting protein 3-like), BECN1 (Beclin 1), GABARAP (Gamma-AminoButyric Acid Receptor-Assocd. Protein) and UVRAG (UV radiation resistance assocd. gene) were found upregulated in TNF-resistant cells, suggesting a constitutive activation of the autophagy machinery in these cells.  More interestingly, we identified NPC1 as the most upregulated genes in TNF-resistant compared to TNF-sensitive MCF-7 cells, suggesting a relation between the intracellular transport of cholesterol, the regulation of autophagy and NPC1 expression in TNF-resistant tumor cells.  In conclusion, we describe here new tools that may help investigating autophagy gene regulation in various cellular models and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlFmjiGdXKrLVg90H21EOLACvtfcHk0lgv3Ls-Lp8y1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCisLc%253D&md5=16673c0bced2a094c4e9bde06bea7e7a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4161%2Fauto.7.7.15454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.7.7.15454%26sid%3Dliteratum%253Aachs%26aulast%3DMoussay%26aufirst%3DE.%26aulast%3DKaoma%26aufirst%3DT.%26aulast%3DBaginska%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DA.%26aulast%3DVan%2BMoer%26aufirst%3DK.%26aulast%3DNicot%26aufirst%3DN.%26aulast%3DNazarov%26aufirst%3DP.%2BV.%26aulast%3DVallar%26aufirst%3DL.%26aulast%3DChouaib%26aufirst%3DS.%26aulast%3DBerchem%26aufirst%3DG.%26aulast%3DJanji%26aufirst%3DB.%26atitle%3DThe%2520acquisition%2520of%2520resistance%2520to%2520TNF%25CE%25B1%2520in%2520breast%2520cancer%2520cells%2520is%2520associated%2520with%2520constitutive%2520activation%2520of%2520autophagy%2520as%2520revealed%2520by%2520a%2520transcriptome%2520analysis%2520using%2520a%2520custom%2520microarray%26jtitle%3DAutophagy%26date%3D2011%26volume%3D7%26issue%3D7%26spage%3D760%26epage%3D770%26doi%3D10.4161%2Fauto.7.7.15454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span> <span> </span><span class="NLM_article-title">GAMDB: a web resource to connect microRNAs with autophagy in gerontology</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.1111/cpr.12247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fcpr.12247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27037912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28XlslSqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=246-251&issue=2&author=L.+Zhangauthor=T.+Xieauthor=M.+Tianauthor=J.+Liauthor=S.+Songauthor=L.+Ouyangauthor=B.+Liuauthor=H.+Cai&title=GAMDB%3A+a+web+resource+to+connect+microRNAs+with+autophagy+in+gerontology&doi=10.1111%2Fcpr.12247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">GAMDB: a web resource to connect microRNAs with autophagy in gerontology</span></div><div class="casAuthors">Zhang, Lan; Xie, Tao; Tian, Mao; Li, Jingjing; Song, Sicheng; Ouyang, Liang; Liu, Bo; Cai, Haoyang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">246-251</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Objectives : MicroRNAs (miRNAs) are endogenous ∼23 nucleotides (nt) RNAs, regulating gene expression by pairing to the mRNAs of protein-coding genes to direct their post-transcriptional repression.  Both in normal and aberrant activities, miRNAs contribute to a recurring paradigm of cellular behaviors in pathol. settings, esp. in gerontol.  Autophagy, a multi-step lysosomal degrdn. process with function to degrade long-lived proteins and damaged organelles, has significant impact on gerontol.  Thus, elucidating how miRNAs participate in autophagy may enlarge the scope of miRNA in autophagy and facilitate researches in gerontol.  Materials and methods : Herein, based upon the published studies, predicted targets and gerontol.-related diseases, we constructed a web resource named Gerontol.-Autophagic-MicroRNA Database (GAMDB) (http://gamdb.liu-lab.com/index.php), which contained 836 autophagy-related miRNAs, 197 targeted genes/proteins and 56 aging-related diseases such as Parkinson's disease, Alzheimer's disease and Huntington's disease.  Results and Conclusion : We made use of large amts. of data to elucidate the intricate relationships between microRNA-regulated autophagic mechanisms and gerontol.  This database will facilitate better understanding of autophagy regulation network in gerontol. and thus promoting gerontol.-related therapy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr39ijB6UOplrVg90H21EOLACvtfcHk0lgv3Ls-Lp8y1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlslSqur0%253D&md5=1714e8200b2096b5b53802eb9b22ec0b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fcpr.12247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcpr.12247%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DTian%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DCai%26aufirst%3DH.%26atitle%3DGAMDB%253A%2520a%2520web%2520resource%2520to%2520connect%2520microRNAs%2520with%2520autophagy%2520in%2520gerontology%26jtitle%3DCell%2520Proliferation%26date%3D2016%26volume%3D49%26issue%3D2%26spage%3D246%26epage%3D251%26doi%3D10.1111%2Fcpr.12247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">AutomiRDB: a web resource connecting microRNAs and autophagy in cancer</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1016</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1007/s10495-015-1133-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1007%2Fs10495-015-1133-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25931313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2MjpsFaqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1016-1017&issue=7&author=Y.+Chenauthor=J.+Huangauthor=B.+Liu&title=AutomiRDB%3A+a+web+resource+connecting+microRNAs+and+autophagy+in+cancer&doi=10.1007%2Fs10495-015-1133-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">AutomiRDB: a web resource connecting microRNAs and autophagy in cancer</span></div><div class="casAuthors">Chen Yi; Huang Jian; Liu Bo</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis : an international journal on programmed cell death</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1016-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2PofzmVRV8lf1XRtfYM0dfW6udTcc2eb6iOCXfM9PE7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjpsFaqsw%253D%253D&md5=a6a11f767968493805d80e5cf3343014</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs10495-015-1133-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-015-1133-1%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DAutomiRDB%253A%2520a%2520web%2520resource%2520connecting%2520microRNAs%2520and%2520autophagy%2520in%2520cancer%26jtitle%3DApoptosis%26date%3D2015%26volume%3D20%26issue%3D7%26spage%3D1016%26epage%3D1017%26doi%3D10.1007%2Fs10495-015-1133-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. H.</span></span> <span> </span><span class="NLM_article-title">miRDearhDB: a database bridging microRNAs and the programmed cell death</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1571</span>, <span class="refDoi"> DOI: 10.1038/cdd.2012.87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fcdd.2012.87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=22743998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC38jlslOqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1571&issue=9&author=J.+Xuauthor=Y.+H.+Li&title=miRDearhDB%3A+a+database+bridging+microRNAs+and+the+programmed+cell+death&doi=10.1038%2Fcdd.2012.87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">miRDeathDB: a database bridging microRNAs and the programmed cell death</span></div><div class="casAuthors">Xu J; Li Y-H</div><div class="citationInfo"><span class="NLM_cas:title">Cell death and differentiation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1571</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTPCX-rhbLpbw5nADWVYHg1fW6udTcc2eaUM0i-u0R_k7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38jlslOqsw%253D%253D&md5=44dd905c9e2371609c35603093e1f71b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2012.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2012.87%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BH.%26atitle%3DmiRDearhDB%253A%2520a%2520database%2520bridging%2520microRNAs%2520and%2520the%2520programmed%2520cell%2520death%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2012%26volume%3D19%26issue%3D9%26spage%3D1571%26doi%3D10.1038%2Fcdd.2012.87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span> <span> </span><span class="NLM_article-title">ncRDeathDB: a comprehensive bioinformatics resource for deciphering network organization of the ncRNA-mediated cell death system</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1917</span>– <span class="NLM_lpage">1926</span>, <span class="refDoi"> DOI: 10.1080/15548627.2015.1089375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2015.1089375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26431463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWrurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1917-1926&issue=10&author=D.+Wuauthor=Y.+Huangauthor=J.+Kangauthor=K.+Liauthor=X.+Biauthor=T.+Zhangauthor=N.+Jinauthor=Y.+Huauthor=P.+Tanauthor=L.+Zhangauthor=Y.+Yiauthor=W.+Shenauthor=J.+Huangauthor=X.+Liauthor=X.+Liauthor=J.+Xuauthor=D.+Wang&title=ncRDeathDB%3A+a+comprehensive+bioinformatics+resource+for+deciphering+network+organization+of+the+ncRNA-mediated+cell+death+system&doi=10.1080%2F15548627.2015.1089375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">ncRDeathDB: A comprehensive bioinformatics resource for deciphering network organization of the ncRNA-mediated cell death system</span></div><div class="casAuthors">Wu, Deng; Huang, Yan; Kang, Juanjuan; Li, Kongning; Bi, Xiaoman; Zhang, Ting; Jin, Nana; Hu, Yongfei; Tan, Puwen; Zhang, Lu; Yi, Ying; Shen, Wenjun; Huang, Jian; Li, Xiaobo; Li, Xia; Xu, Jianzhen; Wang, Dong</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1917-1926</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Programmed cell death (PCD) is a crit. biol. process involved in many important processes, and defects in PCD have been linked with numerous human diseases.  In recent years, the protein architecture in different PCD subroutines has been explored, but our understanding of the global network organization of the noncoding RNA (ncRNA)-mediated cell death system is limited and ambiguous.  Hence, we developed the comprehensive bioinformatics resource (ncRDeathDB, www.rna-society.org/ncrdeathdb) to archive ncRNA-assocd. cell death interactions.  The current version of ncRDeathDB documents a total of more than 4600 ncRNA-mediated PCD entries in 12 species. ncRDeathDB provides a user-friendly interface to query, browse and manipulate these ncRNA-assocd. cell death interactions.  Furthermore, this resource will help to visualize and navigate current knowledge of the noncoding RNA component of cell death and autophagy, to uncover the generic organizing principles of ncRNA-assocd. cell death systems, and to generate valuable biol. hypotheses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR5dgdyv7Lf7Vg90H21EOLACvtfcHk0lj82qeO07XgtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWrurnN&md5=11e72bb05e906170fd4ae8d4818400da</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F15548627.2015.1089375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2015.1089375%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DBi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DYi%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DD.%26atitle%3DncRDeathDB%253A%2520a%2520comprehensive%2520bioinformatics%2520resource%2520for%2520deciphering%2520network%2520organization%2520of%2520the%2520ncRNA-mediated%2520cell%2520death%2520system%26jtitle%3DAutophagy%26date%3D2015%26volume%3D11%26issue%3D10%26spage%3D1917%26epage%3D1926%26doi%3D10.1080%2F15548627.2015.1089375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Autophagic compound database: A resource connecting autophagy-modulating compounds, their potential targets and relevant diseases</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">3</span>), <span class="NLM_elocation-id">e12403</span> <span class="refDoi"> DOI: 10.1111/cpr.12403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fcpr.12403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=29094410" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2018&issue=3&author=Y.+Dengauthor=L.+Zhuauthor=H.+Caiauthor=G.+Wangauthor=B.+Liu&title=Autophagic+compound+database%3A+A+resource+connecting+autophagy-modulating+compounds%2C+their+potential+targets+and+relevant+diseases&doi=10.1111%2Fcpr.12403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Fcpr.12403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcpr.12403%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DAutophagic%2520compound%2520database%253A%2520A%2520resource%2520connecting%2520autophagy-modulating%2520compounds%252C%2520their%2520potential%2520targets%2520and%2520relevant%2520diseases%26jtitle%3DCell%2520Proliferation%26date%3D2018%26volume%3D51%26issue%3D3%26doi%3D10.1111%2Fcpr.12403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, X.</span></span> <span> </span><span class="NLM_article-title">TBC2health: a database of experimentally validated health-beneficial effects of tea bioactive compounds</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">830</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1093/bib/bbw055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1093%2Fbib%2Fbbw055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27387194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvFyjt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=830-836&issue=5&author=S.+Zhangauthor=H.+Xuanauthor=L.+Zhangauthor=S.+Fuauthor=Y.+Wangauthor=H.+Yangauthor=Y.+Taiauthor=Y.+Songauthor=J.+Zhangauthor=C.+T.+Hoauthor=S.+Liauthor=X.+Wan&title=TBC2health%3A+a+database+of+experimentally+validated+health-beneficial+effects+of+tea+bioactive+compounds&doi=10.1093%2Fbib%2Fbbw055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">TBC2health: a database of experimentally validated health-beneficial effects of tea bioactive compounds</span></div><div class="casAuthors">Zhang, Shihua; Xuan, Hongdong; Zhang, Liang; Fu, Sicong; Wang, Yijun; Yang, Hua; Tai, Yuling; Song, Youhong; Zhang, Jinsong; Ho, Chi-Tang; Li, Shaowen; Wan, Xiaochun</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">830-836</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1477-4054</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Tea is one of the most consumed beverages in the world.  Considerable studies show the exceptional health benefits (e.g. antioxidn., cancer prevention) of tea owing to its various bioactive components.  However, data from these extensively published papers had not been made available in a central database.  To lay a foundation in improving the understanding of healthy tea functions, we established a TBC2health database that currently documents 1338 relationships between 497 tea bioactive compds. and 206 diseases (or phenotypes) manually culled from over 300 published articles.  Each entry in TBC2health contains comprehensive information about a bioactive relationship that can be accessed in three aspects: (i) compd. information, (ii) disease (or phenotype) information and (iii) evidence and ref.  Using the curated bioactive relationships, a bipartite network was reconstructed and the corresponding network (or sub-network) visualization and topol. analyses are provided for users.  This database has a user-friendly interface for entry browse, search and download.  In addn., TBC2health provides a submission page and several useful tools (e.g. BLAST, mol. docking) to facilitate use of the database.  Consequently, TBC2health can serve as a valuable bioinformatics platform for the exploration of beneficial effects of tea on human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmjwsNAJEPq7Vg90H21EOLACvtfcHk0livCRissd5hXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvFyjt74%253D&md5=772f369129c68a7ebaf5e7b6589716ed</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbw055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbw055%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXuan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DTai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DC.%2BT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWan%26aufirst%3DX.%26atitle%3DTBC2health%253A%2520a%2520database%2520of%2520experimentally%2520validated%2520health-beneficial%2520effects%2520of%2520tea%2520bioactive%2520compounds%26jtitle%3DBriefings%2520Bioinf.%26date%3D2017%26volume%3D18%26issue%3D5%26spage%3D830%26epage%3D836%26doi%3D10.1093%2Fbib%2Fbbw055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">HAMdb: a database of human autophagy modulators with specific pathway and disease information</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">34</span>, <span class="refDoi"> DOI: 10.1186/s13321-018-0289-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2Fs13321-018-0289-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtFWhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=34&issue=1&author=N.+Wangauthor=J.+Dongauthor=L.+Zhangauthor=D.+Ouyangauthor=Y.+Chengauthor=A.+F.+Chenauthor=A.+P.+Luauthor=D.+S.+Cao&title=HAMdb%3A+a+database+of+human+autophagy+modulators+with+specific+pathway+and+disease+information&doi=10.1186%2Fs13321-018-0289-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">HAMdb: a database of human autophagy modulators with specific pathway and disease information</span></div><div class="casAuthors">Wang, Ning-Ning; Dong, Jie; Zhang, Lin; Ouyang, Defang; Cheng, Yan; Chen, Alex F.; Lu, Ai-Ping; Cao, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/8</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Autophagy is an important homeostatic cellular recycling mechanism responsible for degrading unnecessary or dysfunctional cellular organelles and proteins in all living cells.  In addn. to its vital homeostatic role, this degrdn. pathway also involves in various human disorders, including metabolic conditions, neurodegenerative diseases, cancers and infectious diseases.  Therefore, the comprehensive understanding of autophagy process, autophagyrelated modulators and corresponding pathway and disease information will be of great help for identifying the new autophagy modulators, potential drug candidates, new diagnostic and therapeutic targets.  In recent years, some autophagy databases providing structural and functional information were developed, but the specific databases covering autophagy modulator (proteins, chems. and microRNAs)-related target, pathway and disease information do not exist.  Hence, we developed an online resource, Human Autophagy Modulator Database (HAMdb, http://hamdb .scbdd .com), to provide researchers related pathway and disease information as many as possible.  HAMdb contains 796 proteins, 841 chems. and 132 microRNAs.  Their specific effects on autophagy, physicochem. information, biol. information and disease information were manually collected and compiled.  Addnl., lots of external links were available for more information covering extensive biomedical knowledge.  HAMdb provides a user-friendly interface to query, search, browse autophagy modulators and their comprehensive related information.  HAMdb will help researchers understand the whole autophagy process and provide detailed information about related diseases.  Furthermore, it can give hints for the identification of new diagnostic and therapeutic targets and the discovery of new autophagy modulators.  In a word, we hope that HAMdb has the potential to promote the autophagy research in pharmacol. and pathophysiol. area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQDp8mr_QhELVg90H21EOLACvtfcHk0livCRissd5hXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtFWhsrg%253D&md5=8b92387aacb3fd6e6fa156355f072d65</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs13321-018-0289-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-018-0289-4%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DN.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DOuyang%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DA.%2BF.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DHAMdb%253A%2520a%2520database%2520of%2520human%2520autophagy%2520modulators%2520with%2520specific%2520pathway%2520and%2520disease%2520information%26jtitle%3DJ.%2520Cheminf.%26date%3D2018%26volume%3D10%26issue%3D1%26spage%3D34%26doi%3D10.1186%2Fs13321-018-0289-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagyaraj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacko, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, P.</span></span> <span> </span><span class="NLM_article-title">AutoppahgySMDB: a curated database of small molecules that modulate protein targets regulating autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1080/15548627.2019.1571717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2019.1571717" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2019&pages=1-16&author=R.+Nanduriauthor=R.+Kalraauthor=E.+Bhagyarajauthor=A.+P.+Chackoauthor=N.+Ahujaauthor=D.+Tiwariauthor=S.+Kumarauthor=M.+Jainauthor=R.+Parkeshauthor=P.+Gupta&title=AutoppahgySMDB%3A+a+curated+database+of+small+molecules+that+modulate+protein+targets+regulating+autophagy&doi=10.1080%2F15548627.2019.1571717"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F15548627.2019.1571717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2019.1571717%26sid%3Dliteratum%253Aachs%26aulast%3DNanduri%26aufirst%3DR.%26aulast%3DKalra%26aufirst%3DR.%26aulast%3DBhagyaraj%26aufirst%3DE.%26aulast%3DChacko%26aufirst%3DA.%2BP.%26aulast%3DAhuja%26aufirst%3DN.%26aulast%3DTiwari%26aufirst%3DD.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DParkesh%26aufirst%3DR.%26aulast%3DGupta%26aufirst%3DP.%26atitle%3DAutoppahgySMDB%253A%2520a%2520curated%2520database%2520of%2520small%2520molecules%2520that%2520modulate%2520protein%2520targets%2520regulating%2520autophagy%26jtitle%3DAutophagy%26date%3D2019%26volume%3D22%26spage%3D1%26epage%3D16%26doi%3D10.1080%2F15548627.2019.1571717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">THANATOS: an integrative data resource of proteins and post-translational modifications in the regulation of autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">296</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1402990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2017.1402990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=29157087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1emsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=296-310&issue=2&author=W.+Dengauthor=L.+Maauthor=Y.+Zhangauthor=J.+Zhouauthor=Y.+Wangauthor=Z.+Liuauthor=Y.+Xue&title=THANATOS%3A+an+integrative+data+resource+of+proteins+and+post-translational+modifications+in+the+regulation+of+autophagy&doi=10.1080%2F15548627.2017.1402990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">THANATOS: an integrative data resource of proteins and post-translational modifications in the regulation of autophagy</span></div><div class="casAuthors">Deng, Wankun; Ma, Lili; Zhang, Ying; Zhou, Jiaqi; Wang, Yongbo; Liu, Zexian; Xue, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">296-310</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Macroautophagy/autophagy is a highly conserved process for degrading cytoplasmic contents, dets. cell survival or death, and regulates the cellular homeostasis.  Besides ATG proteins, numerous regulators together with various post-translational modifications (PTMs) are also involved in autophagy.  In this work, we collected 4,237 exptl. identified proteins regulated in autophagy and cell death pathways from the literature.  Then we computationally identified potential orthologs of known proteins, and developed a comprehensive database of The Autophagy, Necrosis, ApopTosis OrchestratorS (THANATOS, http://thanatos.biocuckoo.org), contg. 191,543 proteins potentially assocd. with autophagy and cell death pathways in 164 eukaryotes.  We performed an evolutionary anal. of ATG genes, and obsd. that ATGs required for the autophagosome formation are highly conserved across eukaryotes.  Further analyses revealed that known cancer genes and drug targets were overrepresented in human autophagy proteins, which were significantly assocd. in a no. of signaling pathways and human diseases.  By reconstructing a human kinase-substrate phosphorylation network for ATG proteins, our results confirmed that phosphorylation play a crit. role in regulating autophagy.  In total, we mapped 65,015 known sites of 11 types of PTMs to collected proteins, and revealed that all types of PTM substrates were enriched in human autophagy.  In addn., we obsd. multiple types of PTM regulators such as protein kinases and ubiquitin E3 ligases or adaptors were significantly assocd. with human autophagy, and again the results emphasized the importance of PTM regulations in autophagy.  We anticipated THANATOS can be a useful resource for further studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0TlGKc1EH7rVg90H21EOLACvtfcHk0liF36gORLu5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1emsbs%253D&md5=ae3c6ea6dca17b8fa819b81413341992</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1402990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1402990%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DTHANATOS%253A%2520an%2520integrative%2520data%2520resource%2520of%2520proteins%2520and%2520post-translational%2520modifications%2520in%2520the%2520regulation%2520of%2520autophagy%26jtitle%3DAutophagy%26date%3D2018%26volume%3D14%26issue%3D2%26spage%3D296%26epage%3D310%26doi%3D10.1080%2F15548627.2017.1402990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">ACTP: a webserver for predicting potential targets and relevant pathways of autophagy-modulating compounds</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">10015</span>– <span class="NLM_lpage">10022</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.7015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.18632%2Foncotarget.7015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26824420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC28nlsVOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10015-10022&issue=9&author=T.+Xieauthor=L.+Zhangauthor=S.+Zhangauthor=L.+Ouyangauthor=H.+Caiauthor=B.+Liu&title=ACTP%3A+a+webserver+for+predicting+potential+targets+and+relevant+pathways+of+autophagy-modulating+compounds&doi=10.18632%2Foncotarget.7015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ACTP: A webserver for predicting potential targets and relevant pathways of autophagy-modulating compounds</span></div><div class="casAuthors">Xie Tao; Zhang Lan; Zhang Shouyue; Ouyang Liang; Liu Bo; Cai Haoyang</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">10015-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Autophagy (macroautophagy) is well known as an evolutionarily conserved lysosomal degradation process for long-lived proteins and damaged organelles.  Recently, accumulating evidence has revealed a series of small-molecule compounds that may activate or inhibit autophagy for therapeutic potential on human diseases.  However, targeting autophagy for drug discovery still remains in its infancy.  In this study, we developed a webserver called Autophagic Compound-Target Prediction (ACTP) (http://actp.liu-lab.com/) that could predict autophagic targets and relevant pathways for a given compound.  The flexible docking of submitted small-molecule compound (s) to potential autophagic targets could be performed by backend reverse docking.  The webpage would return structure-based scores and relevant pathways for each predicted target.  Thus, these results provide a basis for the rapid prediction of potential targets/pathways of possible autophagy-activating or autophagy-inhibiting compounds without labor-intensive experiments.  Moreover, ACTP will be helpful to shed light on identifying more novel autophagy-activating or autophagy-inhibiting compounds for future therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxdDd8aHT25YYPsCkJIw00fW6udTcc2eYr4p5muFNF6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nlsVOkug%253D%253D&md5=fb4309bd3739c07c1fac5d2717dfadda</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7015%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DACTP%253A%2520a%2520webserver%2520for%2520predicting%2520potential%2520targets%2520and%2520relevant%2520pathways%2520of%2520autophagy-modulating%2520compounds%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26issue%3D9%26spage%3D10015%26epage%3D10022%26doi%3D10.18632%2Foncotarget.7015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X. Q.</span></span> <span> </span><span class="NLM_article-title">ProSelection: A novel algorithm to select proper protein structure subsets for in silico target identification and drug discovery research</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2686</span>– <span class="NLM_lpage">2898</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aks7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2686-2898&issue=11&author=N.+Wangauthor=L.+Wangauthor=X.+Q.+Xie&title=ProSelection%3A+A+novel+algorithm+to+select+proper+protein+structure+subsets+for+in+silico+target+identification+and+drug+discovery+research&doi=10.1021%2Facs.jcim.7b00277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">ProSelection: A Novel Algorithm to Select Proper Protein Structure Subsets for in Silico Target Identification and Drug Discovery Research</span></div><div class="casAuthors">Wang, Nanyi; Wang, Lirong; Xie, Xiang-Qun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2686-2698</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is widely applied to computer-aided drug design and has become relatively mature in the recent decades.  Application of docking in modeling varies from single lead compd. optimization to large-scale virtual screening.  The performance of mol. docking is highly dependent on the protein structures selected.  It is esp. challenging for large-scale target prediction research when multiple structures are available for a single target.  Therefore, the authors have established ProSelection, a docking preferred-protein selection algorithm, in order to generate the proper structure subset(s).  By the ProSelection algorithm, protein structures of "weak selectors" are filtered out whereas structures of "strong selectors" are kept.  Specifically, the structure which has a good statistical performance of distinguishing active ligands from inactive ligands is defined as a strong selector.  In this study, 249 protein structures of 14 autophagy-related targets are investigated.  Surflex-dock was used as the docking engine to distinguish active and inactive compds. against these protein structures.  Both t test and Mann-Whitney U test were used to distinguish the strong from the weak selectors based on the normality of the docking score distribution.  The suggested docking score threshold for active ligands (SDA) was generated for each strong selector structure according to the receiver operating characteristic (ROC) curve.  The performance of ProSelection was further validated by predicting the potential off-targets of 43 U.S.  Federal Drug Administration approved small mol. antineoplastic drugs.  Overall, ProSelection will accelerate the computational work in protein structure selection and could be a useful tool for mol. docking, target prediction, and protein-chem. database establishment research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRzAFCu_t_P7Vg90H21EOLACvtfcHk0liwiz2tKLc2uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aks7nF&md5=87d2e6a034fa96b442c0e5513f66fe32</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00277%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DX.%2BQ.%26atitle%3DProSelection%253A%2520A%2520novel%2520algorithm%2520to%2520select%2520proper%2520protein%2520structure%2520subsets%2520for%2520in%2520silico%2520target%2520identification%2520and%2520drug%2520discovery%2520research%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D11%26spage%3D2686%26epage%3D2898%26doi%3D10.1021%2Facs.jcim.7b00277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dosztányi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mészáros, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, I.</span></span> <span> </span><span class="NLM_article-title">ANCHOR: web server for predicting protein binding regions in disordered proteins</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">2745</span>– <span class="NLM_lpage">2746</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1093%2Fbioinformatics%2Fbtp518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=19717576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Glsr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2745-2746&issue=20&author=Z.+Doszt%C3%A1nyiauthor=B.+M%C3%A9sz%C3%A1rosauthor=I.+Simon&title=ANCHOR%3A+web+server+for+predicting+protein+binding+regions+in+disordered+proteins&doi=10.1093%2Fbioinformatics%2Fbtp518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">ANCHOR: web server for predicting protein binding regions in disordered proteins</span></div><div class="casAuthors">Dosztanyi, Zsuzsanna; Meszaros, Balint; Simon, Istvan</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2745-2746</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: ANCHOR is a web-based implementation of an original method that takes a single amino acid sequence as an input and predicts protein binding regions that are disordered in isolation but can undergo disorder-to-order transition upon binding.  The server incorporates the result of a general disorder prediction method, IUPred and can carry out simple motif searches as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFf_tj9V_BkbVg90H21EOLACvtfcHk0liwiz2tKLc2uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Glsr3M&md5=db80e68870116d378ebd524cfc87a445</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp518%26sid%3Dliteratum%253Aachs%26aulast%3DDoszt%25C3%25A1nyi%26aufirst%3DZ.%26aulast%3DM%25C3%25A9sz%25C3%25A1ros%26aufirst%3DB.%26aulast%3DSimon%26aufirst%3DI.%26atitle%3DANCHOR%253A%2520web%2520server%2520for%2520predicting%2520protein%2520binding%2520regions%2520in%2520disordered%2520proteins%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26issue%3D20%26spage%3D2745%26epage%3D2746%26doi%3D10.1093%2Fbioinformatics%2Fbtp518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aslan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küçükoǧlu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arslanyolu, M.</span></span> <span> </span><span class="NLM_article-title">A comparative in-silico analysis of autophagy proteins in ciliates</span>. <i>PeerJ</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e2878</span> <span class="refDoi"> DOI: 10.7717/peerj.2878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.7717%2Fpeerj.2878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28123910" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&author=E.+Aslanauthor=N.+K%C3%BC%C3%A7%C3%BCko%C7%A7luauthor=M.+Arslanyolu&title=A+comparative+in-silico+analysis+of+autophagy+proteins+in+ciliates&doi=10.7717%2Fpeerj.2878"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.2878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.2878%26sid%3Dliteratum%253Aachs%26aulast%3DAslan%26aufirst%3DE.%26aulast%3DK%25C3%25BC%25C3%25A7%25C3%25BCko%25C7%25A7lu%26aufirst%3DN.%26aulast%3DArslanyolu%26aufirst%3DM.%26atitle%3DA%2520comparative%2520in-silico%2520analysis%2520of%2520autophagy%2520proteins%2520in%2520ciliates%26jtitle%3DPeerJ%26date%3D2017%26volume%3D5%26doi%3D10.7717%2Fpeerj.2878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheidel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amstein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, I.</span></span> <span> </span><span class="NLM_article-title">In silico knockout studies of xenopahgic capturing of salmonella</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>), <span class="NLM_elocation-id">e1005200</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1371%2Fjournal.pcbi.1005200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27906974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVekurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&issue=12&author=J.+Scheidelauthor=L.+Amsteinauthor=J.+Ackermannauthor=I.+Dikicauthor=I.+Koch&title=In+silico+knockout+studies+of+xenopahgic+capturing+of+salmonella&doi=10.1371%2Fjournal.pcbi.1005200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">In silico knockout studies of xenophagic capturing of Salmonella</span></div><div class="casAuthors">Scheidel, Jennifer; Amstein, Leonie; Ackermann, Jorg; Dikic, Ivan; Koch, Ina</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1005200/1-e1005200/18</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The degrdn. of cytosol-invading pathogens by autophagy, a process known as xenophagy, is an important mechanism of the innate immune system.  Inside the host, Salmonella Typhimurium invades epithelial cells and resides within a specialized intracellular compartment, the Salmonella-contg. vacuole.  A fraction of these bacteria does not persist inside the vacuole and enters the host cytosol.  Salmonella Typhimurium that invades the host cytosol becomes a target of the autophagy machinery for degrdn.  The xenophagy pathway has recently been discovered, and the exact mol. processes are not entirely characterized.  Complete kinetic data for each mol. process is not available, so far.  We developed a math. model of the xenophagy pathway to investigate this key defense mechanism.  In this paper, we present a Petri net model of Salmonella xenophagy in epithelial cells.  The model is based on functional information derived from literature data.  It comprises the mol. mechanism of galectin-8-dependent and ubiquitindependent autophagy, including regulatory processes, like nutrient-dependent regulation of autophagy and TBK1-dependent activation of the autophagy receptor, OPTN.  To model the activation of TBK1, we proposed a new mechanism of TBK1 activation, suggesting a spatial and temporal regulation of this process.  Using std. Petri net anal. techniques, we found basic functional modules, which describe different pathways of the autophagic capture of Salmonella and reflect the basic dynamics of the system.  To verify the model, we performed in silico knockout expts.  We introduced a new concept of knockout anal. to systematically compute and visualize the results, using an in silico knockout matrix.  The results of the in silico knockout analyses were consistent with published exptl. results and provide a basis for future investigations of the Salmonella xenophagy pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdFUjUNoHRfrVg90H21EOLACvtfcHk0liwiz2tKLc2uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVekurc%253D&md5=1a413cba7b29dfb3bb54a4de001d020c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005200%26sid%3Dliteratum%253Aachs%26aulast%3DScheidel%26aufirst%3DJ.%26aulast%3DAmstein%26aufirst%3DL.%26aulast%3DAckermann%26aufirst%3DJ.%26aulast%3DDikic%26aufirst%3DI.%26aulast%3DKoch%26aufirst%3DI.%26atitle%3DIn%2520silico%2520knockout%2520studies%2520of%2520xenopahgic%2520capturing%2520of%2520salmonella%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D12%26doi%3D10.1371%2Fjournal.pcbi.1005200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obaid, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikarm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janjua, H. A.</span></span> <span> </span><span class="NLM_article-title">Mutation-Structure-Function relationship based integrated strategy reveals the potential impact of deleterious missense mutations in autophagy related proteins on hepatocellular carcinoma (HCC): a comprehensive informatics approach</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">E139</span>, <span class="refDoi"> DOI: 10.3390/ijms18010139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.3390%2Fijms18010139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28085066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1eisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=E139&issue=1&author=F.+M.+Awanauthor=A.+Obaidauthor=A.+Ikarmauthor=H.+A.+Janjua&title=Mutation-Structure-Function+relationship+based+integrated+strategy+reveals+the+potential+impact+of+deleterious+missense+mutations+in+autophagy+related+proteins+on+hepatocellular+carcinoma+%28HCC%29%3A+a+comprehensive+informatics+approach&doi=10.3390%2Fijms18010139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Mutation-structure-function relationship based integrated strategy reveals the potential impact of deleterious missense mutations in autophagy related proteins on hepatocellular carcinoma (HCC): a comprehensive informatics approach</span></div><div class="casAuthors">Awan, Faryal Mehwish; Obaid, Ayesha; Ikram, Aqsa; Janjua, Hussnain Ahmed</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">139/1-139/26</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Autophagy, an evolutionary conserved multifaceted lysosome-mediated bulk degrdn. system, plays a vital role in liver pathologies including hepatocellular carcinoma (HCC).  Post-translational modifications (PTMs) and genetic variations in autophagy components have emerged as significant determinants of autophagy related proteins.  Identification of a comprehensive spectrum of genetic variations and PTMs of autophagy related proteins and their impact at mol. level will greatly expand our understanding of autophagy based regulation.  In this study, we attempted to identify high risk missense mutations that are highly damaging to the structure as well as function of autophagy related proteins including LC3A, LC3B, BECN1 and SCD1.  No. of putative structural and functional residues, including several sites that undergo PTMs were also identified.  In total, 16 high-risk SNPs in LC3A, 18 in LC3B, 40 in BECN1 and 43 in SCD1 were prioritized.  Out of these, 2 in LC3A (K49A, K51A), 1 in LC3B (S92C), 6 in BECN1 (S113R, R292C, R292H, Y338C, S346Y, Y352H) and 6 in SCD1 (Y41C, Y55D, R131W, R135Q, R135W, Y151C) coincide with potential PTM sites.  Our integrated anal. found LC3B Y113C, BECN1 I403T, SCD1 R126S and SCD1 Y218C as highly deleterious HCC-assocd. mutations.  This study is the first extensive in silico mutational anal. of the LC3A, LC3B, BECN1 and SCD1 proteins.  We hope that the obsd. results will be a valuable resource for in-depth mechanistic insight into future investigations of pathol. missense SNPs using an integrated computational platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp7pDq7Tquv7Vg90H21EOLACvtfcHk0ljKl8zLh39Iww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1eisbo%253D&md5=cc4148da3a68cb044be36d5c2384b5ee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fijms18010139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18010139%26sid%3Dliteratum%253Aachs%26aulast%3DAwan%26aufirst%3DF.%2BM.%26aulast%3DObaid%26aufirst%3DA.%26aulast%3DIkarm%26aufirst%3DA.%26aulast%3DJanjua%26aufirst%3DH.%2BA.%26atitle%3DMutation-Structure-Function%2520relationship%2520based%2520integrated%2520strategy%2520reveals%2520the%2520potential%2520impact%2520of%2520deleterious%2520missense%2520mutations%2520in%2520autophagy%2520related%2520proteins%2520on%2520hepatocellular%2520carcinoma%2520%2528HCC%2529%253A%2520a%2520comprehensive%2520informatics%2520approach%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26issue%3D1%26spage%3DE139%26doi%3D10.3390%2Fijms18010139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barua, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westrate, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posner, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hlavacek, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackeigan, J. P.</span></span> <span> </span><span class="NLM_article-title">Computational model for autophagic vesicle dynamics in single cells</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.4161/auto.22532</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4161%2Fauto.22532" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23196898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKgs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=74-92&issue=1&author=K.+R.+Martinauthor=D.+Baruaauthor=A.+L.+Kauffmanauthor=L.+M.+Westrateauthor=R.+G.+Posnerauthor=W.+S.+Hlavacekauthor=J.+P.+Mackeigan&title=Computational+model+for+autophagic+vesicle+dynamics+in+single+cells&doi=10.4161%2Fauto.22532"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Computational model for autophagic vesicle dynamics in single cells</span></div><div class="casAuthors">Martin, Katie R.; Barua, Dipak; Kauffman, Audra L.; Westrate, Laura M.; Posner, Richard G.; Hlavacek, William S.; MacKeigan, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-92</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Macroautophagy (autophagy) is a cellular recycling program essential for homeostasis and survival during cytotoxic stress.  This process, which has an emerging role in disease etiol. and treatment, is executed in four stages through the coordinated action of more than 30 proteins.  An effective strategy for studying complicated cellular processes, such as autophagy, involves the construction and anal. of math. or computational models.  When developed and refined from exptl. knowledge, these models can be used to interrogate signaling pathways, formulate novel hypotheses about systems, and make predictions about cell signaling changes induced by specific interventions.  Here, we present the development of a computational model describing autophagic vesicle dynamics in a mammalian system.  We used time-resolved, live-cell microscopy to measure the synthesis and turnover of autophagic vesicles in single cells.  The stochastically simulated model was consistent with data acquired during conditions of both basal and chem.-induced autophagy.  The model was tested by genetic modulation of autophagic machinery and found to accurately predict vesicle dynamics obsd. exptl.  Furthermore, the model generated an unforeseen prediction about vesicle size that is consistent with both published findings and our exptl. observations.  Taken together, this model is accurate and useful and can serve as the foundation for future efforts aimed at quant. characterization of autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdPcEwLNXdjrVg90H21EOLACvtfcHk0ljKl8zLh39Iww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKgs7s%253D&md5=6759efc98c0fa4d49b6c8b1dec9f8606</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.4161%2Fauto.22532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.22532%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DK.%2BR.%26aulast%3DBarua%26aufirst%3DD.%26aulast%3DKauffman%26aufirst%3DA.%2BL.%26aulast%3DWestrate%26aufirst%3DL.%2BM.%26aulast%3DPosner%26aufirst%3DR.%2BG.%26aulast%3DHlavacek%26aufirst%3DW.%2BS.%26aulast%3DMackeigan%26aufirst%3DJ.%2BP.%26atitle%3DComputational%2520model%2520for%2520autophagic%2520vesicle%2520dynamics%2520in%2520single%2520cells%26jtitle%3DAutophagy%26date%3D2013%26volume%3D9%26issue%3D1%26spage%3D74%26epage%3D92%26doi%3D10.4161%2Fauto.22532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Börlin, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamacher-Brady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, N. R.</span></span> <span> </span><span class="NLM_article-title">Agent-based modeling of autophagy reveals emergent regulatory behavior of spatio-temporal autophagy dynamics</span>. <i>Cell Commun. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">56</span>, <span class="refDoi"> DOI: 10.1186/s12964-014-0056-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2Fs12964-014-0056-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25214434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2M7gsFKltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=56&author=C.+S.+B%C3%B6rlinauthor=V.+Langauthor=A.+Hamacher-Bradyauthor=N.+R.+Brady&title=Agent-based+modeling+of+autophagy+reveals+emergent+regulatory+behavior+of+spatio-temporal+autophagy+dynamics&doi=10.1186%2Fs12964-014-0056-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Agent-based modeling of autophagy reveals emergent regulatory behavior of spatio-temporal autophagy dynamics</span></div><div class="casAuthors">Borlin Christoph S; Lang Verena; Hamacher-Brady Anne; Brady Nathan R</div><div class="citationInfo"><span class="NLM_cas:title">Cell communication and signaling : CCS</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Autophagy is a vesicle-mediated pathway for lysosomal degradation, essential under basal and stressed conditions.  Various cellular components, including specific proteins, protein aggregates, organelles and intracellular pathogens, are targets for autophagic degradation.  Thereby, autophagy controls numerous vital physiological and pathophysiological functions, including cell signaling, differentiation, turnover of cellular components and pathogen defense.  Moreover, autophagy enables the cell to recycle cellular components to metabolic substrates, thereby permitting prolonged survival under low nutrient conditions.  Due to the multi-faceted roles for autophagy in maintaining cellular and organismal homeostasis and responding to diverse stresses, malfunction of autophagy contributes to both chronic and acute pathologies.  RESULTS:  We applied a systems biology approach to improve the understanding of this complex cellular process of autophagy.  All autophagy pathway vesicle activities, i.e. creation, movement, fusion and degradation, are highly dynamic, temporally and spatially, and under various forms of regulation.  We therefore developed an agent-based model (ABM) to represent individual components of the autophagy pathway, subcellular vesicle dynamics and metabolic feedback with the cellular environment, thereby providing a framework to investigate spatio-temporal aspects of autophagy regulation and dynamic behavior.  The rules defining our ABM were derived from literature and from high-resolution images of autophagy markers under basal and activated conditions.  Key model parameters were fit with an iterative method using a genetic algorithm and a predefined fitness function.  From this approach, we found that accurate prediction of spatio-temporal behavior required increasing model complexity by implementing functional integration of autophagy with the cellular nutrient state.  The resulting model is able to reproduce short-term autophagic flux measurements (up to 3 hours) under basal and activated autophagy conditions, and to measure the degree of cell-to-cell variability.  Moreover, we experimentally confirmed two model predictions, namely (i) peri-nuclear concentration of autophagosomes and (ii) inhibitory lysosomal feedback on mTOR signaling.  CONCLUSION:  Agent-based modeling represents a novel approach to investigate autophagy dynamics, function and dysfunction with high biological realism.  Our model accurately recapitulates short-term behavior and cell-to-cell variability under basal and activated conditions of autophagy.  Further, this approach also allows investigation of long-term behaviors emerging from biologically-relevant alterations to vesicle trafficking and metabolic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRwKkEHzyB-CU2yPDyEFiigfW6udTcc2eahp3p_FM9Z1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7gsFKltQ%253D%253D&md5=ae4d2c8fe3cc16c3f0e600d9d6d5ff5f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1186%2Fs12964-014-0056-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12964-014-0056-8%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6rlin%26aufirst%3DC.%2BS.%26aulast%3DLang%26aufirst%3DV.%26aulast%3DHamacher-Brady%26aufirst%3DA.%26aulast%3DBrady%26aufirst%3DN.%2BR.%26atitle%3DAgent-based%2520modeling%2520of%2520autophagy%2520reveals%2520emergent%2520regulatory%2520behavior%2520of%2520spatio-temporal%2520autophagy%2520dynamics%26jtitle%3DCell%2520Commun.%2520Signaling%26date%3D2014%26volume%3D12%26spage%3D56%26doi%3D10.1186%2Fs12964-014-0056-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanneman, W. H.</span></span> <span> </span><span class="NLM_article-title">A multimethod computational simulation approach for investigating mitochondrial dynamics and dysfunction in degenerative aging</span>. <i>Aging Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1111/acel.12644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Facel.12644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28815872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlels7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1244-1255&issue=6&author=T.+E.+Hoffmanauthor=K.+J.+Barnettauthor=L.+Wallisauthor=W.+H.+Hanneman&title=A+multimethod+computational+simulation+approach+for+investigating+mitochondrial+dynamics+and+dysfunction+in+degenerative+aging&doi=10.1111%2Facel.12644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A multimethod computational simulation approach for investigating mitochondrial dynamics and dysfunction in degenerative aging</span></div><div class="casAuthors">Hoffman, Timothy E.; Barnett, Katherine J.; Wallis, Lyle; Hanneman, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Aging Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1244-1255</span>CODEN:
                <span class="NLM_cas:coden">ACGECQ</span>;
        ISSN:<span class="NLM_cas:issn">1474-9718</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Research in biogerontol. has largely focused on the complex relationship between mitochondrial dysfunction and biol. aging.  In particular, the mitochondrial free radical theory of aging (MFRTA) has been well accepted.  However, this theory has been challenged by recent studies showing minimal increases in reactive oxygen species (ROS) as not entirely deleterious in nature, and even beneficial under the appropriate cellular circumstances.  To assess these significant and nonintuitive observations in the context of a functional system, we have taken an in silico approach to expand the focus of the MFRTA by including other key mitochondrial stress response pathways, as they have been obsd. in the nematode Caenorhabditis elegans.  These include the mitochondrial unfolded protein response (UPRmt), mitochondrial biogenesis and autophagy dynamics, the relevant DAF-16 and SKN-1 axes, and NAD+-dependent deacetylase activities.  To integrate these pathways, we have developed a multilevel hybrid-modeling paradigm, contg. agent-based elements among stochastic system-dynamics environments of logically derived ordinary differential equations, to simulate aging mitochondrial phenotypes within a population of energetically demanding cells.  The simulation expts. resulted in accurate predictions of physiol. parameters over time that accompany normal aging, such as the declines in both NAD+ and ATP and an increase in ROS.  Addnl., the in silico system was virtually perturbed using a variety of pharmacol. (e.g., rapamycin, pterostilbene, paraquat) and genetic (e.g., skn-1, daf-16, sod-2) schemes to quantitate the temporal alterations of specific mechanistic targets, supporting insights into mol. determinants of aging as well as cytoprotective agents that may improve neurol. or muscular healthspan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvTvWIQSAssLVg90H21EOLACvtfcHk0lh6ix1iWJ5o_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlels7rF&md5=0eaa2625915ff30004912f3534fe81d9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Facel.12644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Facel.12644%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DT.%2BE.%26aulast%3DBarnett%26aufirst%3DK.%2BJ.%26aulast%3DWallis%26aufirst%3DL.%26aulast%3DHanneman%26aufirst%3DW.%2BH.%26atitle%3DA%2520multimethod%2520computational%2520simulation%2520approach%2520for%2520investigating%2520mitochondrial%2520dynamics%2520and%2520dysfunction%2520in%2520degenerative%2520aging%26jtitle%3DAging%2520Cell%26date%3D2017%26volume%3D16%26issue%3D6%26spage%3D1244%26epage%3D1255%26doi%3D10.1111%2Facel.12644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tavassoly, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shajahan-Haq, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyson, J. J.</span></span> <span> </span><span class="NLM_article-title">Dynamic modeling of the interaction between autophagy and apoptosis in mammalian cells</span>. <i>CPT: Pharmacometrics Syst. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1002/psp4.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1002%2Fpsp4.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26225250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFSgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=263-272&issue=4&author=I.+Tavassolyauthor=J.+Parmarauthor=A.+N.+Shajahan-Haqauthor=R.+Clarkeauthor=W.+T.+Baumannauthor=J.+J.+Tyson&title=Dynamic+modeling+of+the+interaction+between+autophagy+and+apoptosis+in+mammalian+cells&doi=10.1002%2Fpsp4.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Modeling of the Interaction Between Autophagy and Apoptosis in Mammalian Cells</span></div><div class="casAuthors">Tavassoly, I.; Parmar, J.; Shajahan-Haq, A. N.; Clarke, R.; Baumann, W. T.; Tyson, J. J.</div><div class="citationInfo"><span class="NLM_cas:title">CPT: Pharmacometrics & Systems Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">CPSPBR</span>;
        ISSN:<span class="NLM_cas:issn">2163-8306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Autophagy is a conserved biol. stress response in mammalian cells that is responsible for clearing damaged proteins and organelles from the cytoplasm and recycling their contents via the lysosomal pathway.  In cases of mild stress, autophagy acts as a survival mechanism, while in cases of severe stress cells may switch to programmed cell death.  Understanding the decision process that moves a cell from autophagy to apoptosis is important since abnormal regulation of autophagy occurs in many diseases, including cancer.  To integrate existing knowledge about this decision process into a rigorous, anal. framework, we built a math. model of cell fate decisions mediated by autophagy.  Our dynamical model is consistent with existing quant. measurements of autophagy and apoptosis in rat kidney proximal tubular cells responding to cisplatin-induced stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVXs53snsbRrVg90H21EOLACvtfcHk0lh6ix1iWJ5o_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFSgurc%253D&md5=6858ab9b0ef3ef715c640b81adc55498</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fpsp4.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsp4.29%26sid%3Dliteratum%253Aachs%26aulast%3DTavassoly%26aufirst%3DI.%26aulast%3DParmar%26aufirst%3DJ.%26aulast%3DShajahan-Haq%26aufirst%3DA.%2BN.%26aulast%3DClarke%26aufirst%3DR.%26aulast%3DBaumann%26aufirst%3DW.%2BT.%26aulast%3DTyson%26aufirst%3DJ.%2BJ.%26atitle%3DDynamic%2520modeling%2520of%2520the%2520interaction%2520between%2520autophagy%2520and%2520apoptosis%2520in%2520mammalian%2520cells%26jtitle%3DCPT%253A%2520Pharmacometrics%2520Syst.%2520Pharmacol.%26date%3D2015%26volume%3D4%26issue%3D4%26spage%3D263%26epage%3D272%26doi%3D10.1002%2Fpsp4.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yung, B. Y.</span></span> <span> </span><span class="NLM_article-title">Gene network exploration of crosstalk between apoptosis and autophagy in chronic myelogenous leukemia</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>,  <span class="NLM_fpage">459840</span>, <span class="refDoi"> DOI: 10.1155/2015/459840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1155%2F2015%2F459840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25821802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2Mnpt1eiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&pages=459840&author=F.+Wangauthor=W.+C.+Choauthor=L.+W.+Chanauthor=S.+C.+Wongauthor=N.+B.+Tsuiauthor=P.+M.+Siuauthor=S.+P.+Yipauthor=B.+Y.+Yung&title=Gene+network+exploration+of+crosstalk+between+apoptosis+and+autophagy+in+chronic+myelogenous+leukemia&doi=10.1155%2F2015%2F459840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Gene network exploration of crosstalk between apoptosis and autophagy in chronic myelogenous leukemia</span></div><div class="casAuthors">Wang Fengfeng; Chan Lawrence W C; Wong S C Cesar; Tsui Nancy B Y; Siu Parco M; Yip S P; Yung Benjamin Y M; Cho William C S</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">459840</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Gene expression levels change to adapt the stress, such as starvation, toxin, and radiation.  The changes are signals transmitted through molecular interactions, eventually leading to two cellular fates, apoptosis and autophagy.  Due to genetic variations, the signals may not be effectively transmitted to modulate apoptotic and autophagic responses.  Such aberrant modulation may lead to carcinogenesis and drug resistance.  The balance between apoptosis and autophagy becomes very crucial in coping with the stress.  Though there have been evidences illustrating the apoptosis-autophagy interplay, the underlying mechanism and the participation of the regulators including transcription factors (TFs) and microRNAs (miRNAs) remain unclear.  RESULTS:  Gene network is a graphical illustration for exploring the functional linkages and the potential coordinate regulations of genes.  Microarray dataset for the study of chronic myeloid leukemia was obtained from Gene Expression Omnibus.  The expression profiles of those genes related to apoptosis and autophagy, including MCL1, BCL2, ATG, beclin-1, BAX, BAK, E2F, cMYC, PI3K, AKT, BAD, and LC3, were extracted from the dataset to construct the gene networks.  CONCLUSION:  The network analysis of these genes explored the underlying mechanisms and the roles of TFs and miRNAs for the crosstalk between apoptosis and autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFWtPqxa6zoFvAgqb2vdc5fW6udTcc2eYUOqzhuldbELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mnpt1eiug%253D%253D&md5=a91e2e3d3280ac9ebbe2c9fba634f459</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1155%2F2015%2F459840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F459840%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DCho%26aufirst%3DW.%2BC.%26aulast%3DChan%26aufirst%3DL.%2BW.%26aulast%3DWong%26aufirst%3DS.%2BC.%26aulast%3DTsui%26aufirst%3DN.%2BB.%26aulast%3DSiu%26aufirst%3DP.%2BM.%26aulast%3DYip%26aufirst%3DS.%2BP.%26aulast%3DYung%26aufirst%3DB.%2BY.%26atitle%3DGene%2520network%2520exploration%2520of%2520crosstalk%2520between%2520apoptosis%2520and%2520autophagy%2520in%2520chronic%2520myelogenous%2520leukemia%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2015%26volume%3D2015%26spage%3D459840%26doi%3D10.1155%2F2015%2F459840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small molecule targeting ULK1-modulated cell death of triple negative breast cancer <i>in vitro</i> and <i>in vivo</i></span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">2687</span>– <span class="NLM_lpage">2701</span>, <span class="refDoi"> DOI: 10.1039/C6SC05368H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1039%2FC6SC05368H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28553505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltlChtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2687-2701&issue=4&author=L.+Zhangauthor=L.+Fuauthor=S.+Zhangauthor=J.+Zhangauthor=Y.+Zhaoauthor=Y.+Zhengauthor=G.+Heauthor=S.+Yangauthor=L.+Ouyangauthor=B.+Liu&title=Discovery+of+a+small+molecule+targeting+ULK1-modulated+cell+death+of+triple+negative+breast+cancer+in+vitro+and+in+vivo&doi=10.1039%2FC6SC05368H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small molecule targeting ULK1-modulated cell death of triple negative breast cancer in vitro and in vivo</span></div><div class="casAuthors">Zhang, Lan; Fu, Leilei; Zhang, Shouyue; Zhang, Jin; Zhao, Yuqian; Zheng, Yaxin; He, Gu; Yang, Shengyong; Ouyang, Liang; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2687-2701</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">UNC-51-like kinase 1 (ULK1) is well-known to initiate autophagy, and the downregulation of ULK1 has been found in most breast cancer tissues.  Thus, the activation of ULK1-modulated autophagy could be a promising strategy for breast cancer therapy.  In this study, we found that ULK1 was remarkably downregulated in breast cancer tissue samples by The Cancer Genome Atlas (TCGA) anal. and tissue microarray (TMA) anal., esp. in triple neg. breast cancer (TNBC).  To design a ULK1 agonist, we integrated in silico screening and chem. synthesis to acquire a series of small mol. candidates.  After rounds of kinase and anti-proliferative activity screening, we discovered the small mol., LYN-1604, to be the best candidate for a ULK1 agonist.  Addnl., we identified that three amino acid residues (LYS50, LEU53, and TYR89) were key to the activation site of LYN-1604 and ULK1 by site-directed mutagenesis and biochem. assays.  Subsequently, we demonstrated that LYN-1604 could induce cell death, assocd. with autophagy by the ULK complex (ULK1-mATG13-FIP200-ATG101) in MDA-MB-231 cells.  To further explore LYN-1604-induced autophagic mechanisms, we found some potential ULK1 interactors, such as ATF3, RAD21, and caspase3, by performing comparative microarray anal.  Intriguingly, we found that LYN-1604 induced cell death involved in ATF3, RAD21, and caspase3, accompanied by autophagy and apoptosis.  Moreover, we demonstrated that LYN-1604 has potential for good therapeutic effects on TNBC by targeting ULK1-modulated cell death in vivo; thus making this ULK1 agonist a novel potential small-mol. drug candidate for future TNBC therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqilhZpiddfcbVg90H21EOLACvtfcHk0lj2tADomLrGdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltlChtg%253D%253D&md5=6c40089843a8fbcdbb5ee60e0547dc7c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2FC6SC05368H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC05368H%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520small%2520molecule%2520targeting%2520ULK1-modulated%2520cell%2520death%2520of%2520triple%2520negative%2520breast%2520cancer%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26issue%3D4%26spage%3D2687%26epage%3D2701%26doi%3D10.1039%2FC6SC05368H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of genetic vatiants in autophagy pathway genes as prognostic biomarkers for breast cancer</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>627</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1016/j.gene.2017.06.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.gene.2017.06.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28669927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1CjtbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=627&publication_year=2017&pages=549-555&author=J.+Zhouauthor=D.+Hangauthor=Y.+Jiangauthor=J.+Chenauthor=J.+Hanauthor=W.+Zhouauthor=G.+Jinauthor=H.+Maauthor=J.+Dai&title=Evaluation+of+genetic+vatiants+in+autophagy+pathway+genes+as+prognostic+biomarkers+for+breast+cancer&doi=10.1016%2Fj.gene.2017.06.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of genetic variants in autophagy pathway genes as prognostic biomarkers for breast cancer</span></div><div class="casAuthors">Zhou, Jing; Hang, Dong; Jiang, Yue; Chen, Jiaping; Han, Jing; Zhou, Wen; Jin, Guangfu; Ma, Hongxia; Dai, Juncheng</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">627</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Autophagy-related genes (ATGs) play a crit. role in the development of various diseases including cancer.  However, the role of ATGs in breast cancer survival remains unclear.  This study aims to investigate whether genetic variants in core ATGs are correlated with the prognosis of breast cancer.  A total of 14 potentially functional variants in core ATGs were genotyped in 790 breast cancer patients.  The assocn. of each variant with breast cancer-specific survival was evaluated by log-rank test and Cox regression model.  In silico anal. was also performed to evaluate the potential function of selected variants.  We found that one variant in ATG7 rs8154 (A > G) was significantly assocd. with breast cancer-specific survival after adjusted for age and clin. stage (HR = 1.61, 95% CI: 1.12-2.31, P = 0.010).  Stratified anal. showed that the prognostic role of rs8154 was significant in subgroups of elder age, elder menarche age, and postmenopausal status (all P < 0.001).  Interaction effects were also detected between rs8154 and these grouping variables.  In silico anal. revealed that rs8154 was annotated to be expression quant. trait loci (eQTL) and methylation quant. trait loci (meQTL) based on Genotype-Tissue Expression (GTEx) and The Cancer Genome Atlas (TCGA) datasets (both P < 0.05).  In addn., upregulated expression of ATG7 in breast cancer tissues was obsd. and is significantly assocd. with poorer overall survival (log-rank P = 0.015), poorer relapse free survival (log-rank P = 0.017), and poorer distant metastasis free survival (log-rank P = 0.034) in different datasets.  Summarily, ATG7 variant rs8154 represents a novel prognostic marker for breast cancer patients, which may shed light on clin. risk stratification and therapeutic decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx-WJ2jwZyH7Vg90H21EOLACvtfcHk0lj2tADomLrGdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1CjtbfF&md5=15b73aadc54ff7ba7890ae281861b6ba</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2017.06.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2017.06.053%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHang%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJin%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DDai%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520genetic%2520vatiants%2520in%2520autophagy%2520pathway%2520genes%2520as%2520prognostic%2520biomarkers%2520for%2520breast%2520cancer%26jtitle%3DGene%26date%3D2017%26volume%3D627%26spage%3D549%26epage%3D555%26doi%3D10.1016%2Fj.gene.2017.06.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suwansa-Ard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kankuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongbuakaew, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saetan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornthong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruangkum, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khornchatri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isidoro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobhon, P.</span></span> <span> </span><span class="NLM_article-title">Transcriptomic analysis of the autophagy machinery in crustaceans</span>. <i>BMC Genomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">587</span>, <span class="refDoi"> DOI: 10.1186/s12864-016-2996-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2Fs12864-016-2996-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=27506197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSju7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=587&author=S.+Suwansa-Ardauthor=W.+Kankuanauthor=T.+Thongbuakaewauthor=J.+Saetanauthor=N.+Kornthongauthor=T.+Kruangkumauthor=K.+Khornchatriauthor=S.+F.+Cumminsauthor=C.+Isidoroauthor=P.+Sobhon&title=Transcriptomic+analysis+of+the+autophagy+machinery+in+crustaceans&doi=10.1186%2Fs12864-016-2996-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomic analysis of the autophagy machinery in crustaceans</span></div><div class="casAuthors">Suwansa-ard, Saowaros; Kankuan, Wilairat; Thongbuakaew, Tipsuda; Saetan, Jirawat; Kornthong, Napamanee; Kruangkum, Thanapong; Khornchatri, Kanjana; Cummins, Scott F.; Isidoro, Ciro; Sobhon, Prasert</div><div class="citationInfo"><span class="NLM_cas:title">BMC Genomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">587/1-587/14</span>CODEN:
                <span class="NLM_cas:coden">BGMEET</span>;
        ISSN:<span class="NLM_cas:issn">1471-2164</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: The giant freshwater prawn, Macrobrachium rosenbergii, is a decapod crustacean that is com. important as a food source.  Farming of com. crustaceans requires an efficient management strategy because the animals are easily subjected to stress and diseases during the culture.  Autophagy, a stress response process, is well-documented and conserved in most animals, yet it is poorly studied in crustaceans.  Results: In this study, we have performed an in silico search for transcripts encoding autophagy-related (Atg) proteins within various tissue transcriptomes of M. rosenbergii.  Basic Local Alignment Search Tool (BLAST) search using previously known Atg proteins as queries revealed 41 transcripts encoding homologous M. rosenbergii Atg proteins.  Among these Atg proteins, we selected commonly used autophagy markers, including Beclin 1, vacuolar protein sorting (Vps) 34, microtubule-assocd. proteins 1A/1B light chain 3B (MAP1LC3B), p62/sequestosome 1 (SQSTM1), and lysosomal-assocd. membrane protein 1 (Lamp-1) for further sequence analyses using comparative alignment and protein structural prediction.  We found that crustacean autophagy marker proteins contain conserved motifs typical of other animal Atg proteins.  Western blotting using com. antibodies raised against human Atg marker proteins indicated their presence in various M. rosenbergii tissues, while immunohistochem. localized Atg marker proteins within ovarian tissue, specifically late stage oocytes.  Conclusions: This study demonstrates that the mol. components of autophagic process are conserved in crustaceans, which is comparable to autophagic process in mammals.  Furthermore, it provides a foundation for further studies of autophagy in crustaceans that may lead to more understanding of the reprodn.- and stress-related autophagy, which will enable the efficient aquaculture practices.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Yhee2Z7vILVg90H21EOLACvtfcHk0lj2tADomLrGdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSju7rM&md5=6dcc2c29d94e76de1a2b27e959cf8ffa</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs12864-016-2996-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12864-016-2996-4%26sid%3Dliteratum%253Aachs%26aulast%3DSuwansa-Ard%26aufirst%3DS.%26aulast%3DKankuan%26aufirst%3DW.%26aulast%3DThongbuakaew%26aufirst%3DT.%26aulast%3DSaetan%26aufirst%3DJ.%26aulast%3DKornthong%26aufirst%3DN.%26aulast%3DKruangkum%26aufirst%3DT.%26aulast%3DKhornchatri%26aufirst%3DK.%26aulast%3DCummins%26aufirst%3DS.%2BF.%26aulast%3DIsidoro%26aufirst%3DC.%26aulast%3DSobhon%26aufirst%3DP.%26atitle%3DTranscriptomic%2520analysis%2520of%2520the%2520autophagy%2520machinery%2520in%2520crustaceans%26jtitle%3DBMC%2520Genomics%26date%3D2016%26volume%3D17%26spage%3D587%26doi%3D10.1186%2Fs12864-016-2996-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, W.</span></span> <span> </span><span class="NLM_article-title">Independent regulation of gene expression level and noise by histone modifications</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>), <span class="NLM_elocation-id">e1005585</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1371%2Fjournal.pcbi.1005585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28665997" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&issue=6&author=S.+Wuauthor=K.+Liauthor=Y.+Liauthor=T.+Zhaoauthor=T.+Liauthor=Y.+F.+Yangauthor=W.+Qian&title=Independent+regulation+of+gene+expression+level+and+noise+by+histone+modifications&doi=10.1371%2Fjournal.pcbi.1005585"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005585%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DY.%2BF.%26aulast%3DQian%26aufirst%3DW.%26atitle%3DIndependent%2520regulation%2520of%2520gene%2520expression%2520level%2520and%2520noise%2520by%2520histone%2520modifications%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D6%26doi%3D10.1371%2Fjournal.pcbi.1005585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span> <span> </span><span class="NLM_article-title">Integrative bioinformatics and proteomics-based discovery of an eEF2K inhibitor (cefatrizine) with ER stress modulation in breast cancer cells</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1039/C5MB00848D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1039%2FC5MB00848D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26776155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVGhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=729-736&issue=3&author=Z.+Yaoauthor=J.+Liauthor=Z.+Liuauthor=L.+Zhengauthor=N.+Fanauthor=Y.+Zhangauthor=N.+Jiaauthor=J.+Lvauthor=N.+Liuauthor=X.+Zhuauthor=J.+Duauthor=C.+Lvauthor=F.+Xieauthor=Y.+Liuauthor=X.+Wangauthor=Z.+Feiauthor=C.+Gao&title=Integrative+bioinformatics+and+proteomics-based+discovery+of+an+eEF2K+inhibitor+%28cefatrizine%29+with+ER+stress+modulation+in+breast+cancer+cells&doi=10.1039%2FC5MB00848D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative bioinformatics and proteomics-based discovery of an eEF2K inhibitor (cefatrizine) with ER stress modulation in breast cancer cells</span></div><div class="casAuthors">Yao, Zhiqiang; Li, Juntang; Liu, Zhongyu; Zheng, Lu; Fan, Naijun; Zhang, Ying; Jia, Nan; Lv, Jingjing; Liu, Ningning; Zhu, Xiaoshan; Du, Jiangbo; Lv, Ci; Xie, Feng; Liu, Yigang; Wang, Xingke; Fei, Zhou; Gao, Chunfang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">729-736</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Eukaryotic elongation factor-2 kinase (eEF2K), a unique calcium/calmodulin-dependent protein kinase, is well known to regulate apoptosis, autophagy and ER stress in many types of human cancers.  Therefore, eEF2K would be regarded as a promising therapeutic target; however, the eEF2K-regulated mechanism and its targeted inhibitor still remain to be discovered in cancer.  Herein, we constructed a protein-protein interaction (PPI) network of eEF2K and achieved an eEF2K-regulated ER stress subnetwork by bioinformatics prediction.  Then, we found that the differential protein expressions involved in ER stress in the context of si-eEF2K-treated MCF-7 and MDA-MB-436 cells by iTRAQ-based analyses, resp.  Integrated into these aforementioned results, we constructed a core eEF2K-regulated ER stress subnetwork in breast cancer cells.  Subsequently, we screened a series of candidate compds. targeting eEF2K and discovered a novel eEF2K inhibitor (cefatrizine) with an anti-proliferative activity toward breast cancer cells.  Moreover, we found that cefatrizine induced ER stress in both MCF-7 and MDA-MB-436 cells.  Interestingly, we demonstrated that the mechanism of cefatrizine-induced ER stress was in good agreement with our bioinformatics and proteomics-based results.  In conclusion, these results demonstrate that a novel eEF2K inhibitor (cefatrizine) induces ER stress in breast cancer cells by integrating bioinformatics prediction, proteomics analyses and exptl. validation, which would provide a clue for exploring more mechanisms of eEF2K and its targeted inhibitors in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDQPIGrQ6947Vg90H21EOLACvtfcHk0ljBcjAS0S_CpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVGhuw%253D%253D&md5=84c2bbd60d8c71479e98f426db5fbb5b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC5MB00848D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MB00848D%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DFan%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJia%26aufirst%3DN.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DLv%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DFei%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DC.%26atitle%3DIntegrative%2520bioinformatics%2520and%2520proteomics-based%2520discovery%2520of%2520an%2520eEF2K%2520inhibitor%2520%2528cefatrizine%2529%2520with%2520ER%2520stress%2520modulation%2520in%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26issue%3D3%26spage%3D729%26epage%3D736%26doi%3D10.1039%2FC5MB00848D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Phosphoproteome-based kinase activity profiling reveals the critical role of MAP2K2 and PLK1 in neuronal autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1969</span>– <span class="NLM_lpage">1980</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1371393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1080%2F15548627.2017.1371393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=28933595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1SqsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1969-1980&issue=11&author=L.+L.+Chenauthor=Y.+B.+Wangauthor=J.+X.+Songauthor=W.+K.+Dengauthor=J.+H.+Luauthor=L.+L.+Maauthor=C.+B.+Yangauthor=M.+Liauthor=Y.+Xue&title=Phosphoproteome-based+kinase+activity+profiling+reveals+the+critical+role+of+MAP2K2+and+PLK1+in+neuronal+autophagy&doi=10.1080%2F15548627.2017.1371393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoproteome-based kinase activity profiling reveals the critical role of MAP2K2 and PLK1 in neuronal autophagy</span></div><div class="casAuthors">Chen, Lei-Lei; Wang, Yong-Bo; Song, Ju-Xian; Deng, Wan-Kun; Lu, Jia-Hong; Ma, Li-Li; Yang, Chuan-Bin; Li, Min; Xue, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1969-1980</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Recent studies have demonstrated that dysregulation of macroautophagy/autophagy may play a central role in the pathogenesis of neurodegenerative disorders, and the induction of autophagy protects against the toxic insults of aggregate-prone proteins by enhancing their clearance.  Thus, autophagy has become a promising therapeutic target against neurodegenerative diseases.  In this study, quant. phosphoproteomic profiling together with a computational anal. was performed to delineate the phosphorylation signaling networks regulated by 2 natural neuroprotective autophagy enhancers, corynoxine (Cory) and corynoxine B (Cory B).  To identify key regulators, namely, protein kinases, we developed a novel network-based algorithm of in silico Kinome Activity Profiling (iKAP) to computationally infer potentially important protein kinases from phosphorylation networks.  Using this algorithm, we obsd. that Cory or Cory B potentially regulated several kinases.  We predicted and validated that Cory, but not Cory B, downregulated a well-documented autophagy kinase, RPS6KB1/p70S6K (ribosomal protein S6 kinase, polypeptide 1).  We also discovered 2 kinases, MAP2K2/MEK2 (mitogen-activated protein kinase kinase 2) and PLK1 (polo-like kinase 1), to be potentially upregulated by Cory, whereas the siRNA-mediated knockdown of Map2k2 and Plk1 significantly inhibited Cory-induced autophagy.  Furthermore, Cory promoted the clearance of Alzheimer disease-assocd. APP (amyloid β [A4] precursor protein) and Parkinson disease-assocd. SNCA/α-synuclein (synuclein, α) by enhancing autophagy, and these effects were dramatically diminished by the inhibition of the kinase activities of MAP2K2 and PLK1.  As a whole, our study not only developed a powerful method for the identification of important regulators from the phosphoproteomic data but also identified the important role of MAP2K2 and PLK1 in neuronal autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcXcdC7yLfIbVg90H21EOLACvtfcHk0ljBcjAS0S_CpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1SqsLfF&md5=b55042a64c901efceaca0224a907a91a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1371393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1371393%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BL.%26aulast%3DWang%26aufirst%3DY.%2BB.%26aulast%3DSong%26aufirst%3DJ.%2BX.%26aulast%3DDeng%26aufirst%3DW.%2BK.%26aulast%3DLu%26aufirst%3DJ.%2BH.%26aulast%3DMa%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DC.%2BB.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DPhosphoproteome-based%2520kinase%2520activity%2520profiling%2520reveals%2520the%2520critical%2520role%2520of%2520MAP2K2%2520and%2520PLK1%2520in%2520neuronal%2520autophagy%26jtitle%3DAutophagy%26date%3D2017%26volume%3D13%26issue%3D11%26spage%3D1969%26epage%3D1980%26doi%3D10.1080%2F15548627.2017.1371393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behrends, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sowa, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span> <span> </span><span class="NLM_article-title">Network organization of the human autophagy system</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span> (<span class="NLM_issue">7302</span>),  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1038/nature09204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fnature09204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=20562859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXns1Gis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=68-76&issue=7302&author=C.+Behrendsauthor=M.+E.+Sowaauthor=S.+P.+Gygiauthor=J.+W.+Harper&title=Network+organization+of+the+human+autophagy+system&doi=10.1038%2Fnature09204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Network organization of the human autophagy system</span></div><div class="casAuthors">Behrends, Christian; Sowa, Mathew E.; Gygi, Steven P.; Harper, J. Wade</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7302</span>),
    <span class="NLM_cas:pages">68-76</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy, the process by which proteins and organelles are sequestered in autophagosomal vesicles and delivered to the lysosome/vacuole for degrdn., provides a primary route for turnover of stable and defective cellular proteins.  Defects in this system are linked with numerous human diseases.  Although conserved protein kinase, lipid kinase and ubiquitin-like protein conjugation subnetworks controlling autophagosome formation and cargo recruitment have been defined, our understanding of the global organization of this system is limited.  Here we report a proteomic anal. of the autophagy interaction network in human cells under conditions of ongoing (basal) autophagy, revealing a network of 751 interactions among 409 candidate interacting proteins with extensive connectivity among subnetworks.  Many new autophagy interaction network components have roles in vesicle trafficking, protein or lipid phosphorylation and protein ubiquitination, and affect autophagosome no. or flux when depleted by RNA interference.  The six ATG8 orthologues in humans (MAP1LC3/GABARAP proteins) interact with a cohort of 67 proteins, with extensive binding partner overlap between family members, and frequent involvement of a conserved surface on ATG8 proteins known to interact with LC3-interacting regions in partner proteins.  These studies provide a global view of the mammalian autophagy interaction landscape and a resource for mechanistic anal. of this crit. protein homeostasis pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvZw7FZHvYbLVg90H21EOLACvtfcHk0lgYt6qto-DP_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXns1Gis7Y%253D&md5=5d64dbe07f7d51a962568c3e8d522991</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature09204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09204%26sid%3Dliteratum%253Aachs%26aulast%3DBehrends%26aufirst%3DC.%26aulast%3DSowa%26aufirst%3DM.%2BE.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26atitle%3DNetwork%2520organization%2520of%2520the%2520human%2520autophagy%2520system%26jtitle%3DNature%26date%3D2010%26volume%3D466%26issue%3D7302%26spage%3D68%26epage%3D76%26doi%3D10.1038%2Fnature09204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordureau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swarup, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span> <span> </span><span class="NLM_article-title">RAB7A phosphorylation by TBK1 promotes mitophagy bia the PINK-PARKIN pathway</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">eaav0443</span>, <span class="refDoi"> DOI: 10.1126/sciadv.aav0443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1126%2Fsciadv.aav0443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=30627666" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=eaav0443&issue=11&author=J.+M.+Heoauthor=A.+Ordureauauthor=S.+Swarupauthor=J.+A.+Pauloauthor=K.+Shenauthor=D.+M.+Sabatiniauthor=J.+W.+Harper&title=RAB7A+phosphorylation+by+TBK1+promotes+mitophagy+bia+the+PINK-PARKIN+pathway&doi=10.1126%2Fsciadv.aav0443"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aav0443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aav0443%26sid%3Dliteratum%253Aachs%26aulast%3DHeo%26aufirst%3DJ.%2BM.%26aulast%3DOrdureau%26aufirst%3DA.%26aulast%3DSwarup%26aufirst%3DS.%26aulast%3DPaulo%26aufirst%3DJ.%2BA.%26aulast%3DShen%26aufirst%3DK.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26atitle%3DRAB7A%2520phosphorylation%2520by%2520TBK1%2520promotes%2520mitophagy%2520bia%2520the%2520PINK-PARKIN%2520pathway%26jtitle%3DSci.%2520Adv.%26date%3D2018%26volume%3D4%26issue%3D11%26spage%3Deaav0443%26doi%3D10.1126%2Fsciadv.aav0443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuballa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolsky, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, R. J.</span></span> <span> </span><span class="NLM_article-title">A novel hybrid yeast-human network analysis reveals an essential role for FNBP1L in antibacterial autophagy</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>182</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">4917</span>– <span class="NLM_lpage">4930</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.0803050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4049%2Fjimmunol.0803050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=19342671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFOju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2009&pages=4917-4930&issue=8&author=A.+Huettauthor=A.+Ngauthor=Z.+Caoauthor=P.+Kuballaauthor=M.+Komatsuauthor=M.+J.+Dalyauthor=D.+K.+Podolskyauthor=R.+J.+Xavier&title=A+novel+hybrid+yeast-human+network+analysis+reveals+an+essential+role+for+FNBP1L+in+antibacterial+autophagy&doi=10.4049%2Fjimmunol.0803050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Hybrid Yeast-Human Network Analysis Reveals an Essential Role for FNBP1L in Antibacterial Autophagy</span></div><div class="casAuthors">Huett, Alan; Ng, Aylwin; Cao, Zhifang; Kuballa, Petric; Komatsu, Masaaki; Daly, Mark J.; Podolsky, Daniel K.; Xavier, Ramnik J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4917-4930</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Autophagy is a conserved cellular process required for the removal of defective organelles, protein aggregates, and intracellular pathogens.  We used a network anal. strategy to identify novel human autophagy components based upon the yeast interactome centered on the core yeast autophagy proteins.  This revealed the potential involvement of 14 novel mammalian genes in autophagy, several of which have known or predicted roles in membrane organization or dynamics.  We selected one of these membrane interactors, FNBP1L (formin binding protein 1-like), an F-BAR-contg. protein (also termed Toca-1), for further study based upon a predicted interaction with ATG3.  We confirmed the FNBP1L/ATG3 interaction biochem. and mapped the FNBP1L domains responsible.  Using a functional RNA interference approach, we detd. that FNBP1L is essential for autophagy of the intracellular pathogen Salmonella enterica serovar Typhimurium and show that the autophagy process serves to restrict the growth of intracellular bacteria.  However, FNBP1L appears dispensable for other forms of autophagy induced by serum starvation or rapamycin.  We present a model where FNBP1L is essential for autophagy of intracellular pathogens and identify FNBP1L as a differentially used mol. in specific autophagic contexts.  By using network biol. to derive functional biol. information, we demonstrate the utility of integrated genomics to novel mol. discovery in autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0P-EhlrQHIrVg90H21EOLACvtfcHk0lgYt6qto-DP_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFOju7k%253D&md5=edb0aa79cd4842063f0bb9a4ae602c42</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.0803050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.0803050%26sid%3Dliteratum%253Aachs%26aulast%3DHuett%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DA.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DKuballa%26aufirst%3DP.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DDaly%26aufirst%3DM.%2BJ.%26aulast%3DPodolsky%26aufirst%3DD.%2BK.%26aulast%3DXavier%26aufirst%3DR.%2BJ.%26atitle%3DA%2520novel%2520hybrid%2520yeast-human%2520network%2520analysis%2520reveals%2520an%2520essential%2520role%2520for%2520FNBP1L%2520in%2520antibacterial%2520autophagy%26jtitle%3DJ.%2520Immunol.%26date%3D2009%26volume%3D182%26issue%3D8%26spage%3D4917%26epage%3D4930%26doi%3D10.4049%2Fjimmunol.0803050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Türei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Földvári-Nagy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazekas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Módos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubisch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadlecsik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demeter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcsmáros, T.</span></span> <span> </span><span class="NLM_article-title">Autophagy regulatory network-a systems-level bioinformatics resource for studying the mechanism and regulation of autophagy</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.4161/15548627.2014.994346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.4161%2F15548627.2014.994346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25635527" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=155-165&issue=1&author=D.+T%C3%BCreiauthor=L.+F%C3%B6ldv%C3%A1ri-Nagyauthor=D.+Fazekasauthor=D.+M%C3%B3dosauthor=J.+Kubischauthor=T.+Kadlecsikauthor=A.+Demeterauthor=K.+Lentiauthor=P.+Csermelyauthor=T.+Vellaiauthor=T.+Korcsm%C3%A1ros&title=Autophagy+regulatory+network-a+systems-level+bioinformatics+resource+for+studying+the+mechanism+and+regulation+of+autophagy&doi=10.4161%2F15548627.2014.994346"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.4161%2F15548627.2014.994346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15548627.2014.994346%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25BCrei%26aufirst%3DD.%26aulast%3DF%25C3%25B6ldv%25C3%25A1ri-Nagy%26aufirst%3DL.%26aulast%3DFazekas%26aufirst%3DD.%26aulast%3DM%25C3%25B3dos%26aufirst%3DD.%26aulast%3DKubisch%26aufirst%3DJ.%26aulast%3DKadlecsik%26aufirst%3DT.%26aulast%3DDemeter%26aufirst%3DA.%26aulast%3DLenti%26aufirst%3DK.%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3DVellai%26aufirst%3DT.%26aulast%3DKorcsm%25C3%25A1ros%26aufirst%3DT.%26atitle%3DAutophagy%2520regulatory%2520network-a%2520systems-level%2520bioinformatics%2520resource%2520for%2520studying%2520the%2520mechanism%2520and%2520regulation%2520of%2520autophagy%26jtitle%3DAutophagy%26date%3D2015%26volume%3D11%26issue%3D1%26spage%3D155%26epage%3D165%26doi%3D10.4161%2F15548627.2014.994346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilad, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiloh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ber, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bialik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimchi, A.</span></span> <span> </span><span class="NLM_article-title">Discovering protein-protein interactions within the programmed cell death network using a protein-fragment complementation screen</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">909</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1016%2Fj.celrep.2014.06.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25066129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1agsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=909-921&issue=3&author=Y.+Giladauthor=R.+Shilohauthor=Y.+Berauthor=S.+Bialikauthor=A.+Kimchi&title=Discovering+protein-protein+interactions+within+the+programmed+cell+death+network+using+a+protein-fragment+complementation+screen&doi=10.1016%2Fj.celrep.2014.06.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Protein-Protein Interactions within the Programmed Cell Death Network Using a Protein-Fragment Complementation Screen</span></div><div class="casAuthors">Gilad, Yuval; Shiloh, Ruth; Ber, Yaara; Bialik, Shani; Kimchi, Adi</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">909-921</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Apoptosis and autophagy are distinct biol. processes, each driven by a different set of protein-protein interactions, with significant crosstalk via direct interactions among apoptotic and autophagic proteins.  To measure the global profile of these interactions, we adapted the Gaussia luciferase protein-fragment complementation assay (GLuc PCA), which monitors binding between proteins fused to complementary fragments of a luciferase reporter.  A library encompassing 63 apoptotic and autophagic proteins was constructed for the anal. of ∼3,600 protein-pair combinations.  This generated a detailed landscape of the apoptotic and autophagic modules and points of interface between them, identifying 46 previously unknown interactions.  One of these interactions, between DAPK2, a Ser/Thr kinase that promotes autophagy, and 14-3-3τ, was further investigated.  We mapped the region responsible for 14-3-3τ binding and proved that this interaction inhibits DAPK2 dimerization and activity.  This proof of concept underscores the power of the GLuc PCA platform for the discovery of biochem. pathways within the cell death network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnFV3lEKHgFLVg90H21EOLACvtfcHk0lgYt6qto-DP_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1agsL%252FL&md5=22aced64b8cc8645b686f1d0a6b08da6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.049%26sid%3Dliteratum%253Aachs%26aulast%3DGilad%26aufirst%3DY.%26aulast%3DShiloh%26aufirst%3DR.%26aulast%3DBer%26aufirst%3DY.%26aulast%3DBialik%26aufirst%3DS.%26aulast%3DKimchi%26aufirst%3DA.%26atitle%3DDiscovering%2520protein-protein%2520interactions%2520within%2520the%2520programmed%2520cell%2520death%2520network%2520using%2520a%2520protein-fragment%2520complementation%2520screen%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26issue%3D3%26spage%3D909%26epage%3D921%26doi%3D10.1016%2Fj.celrep.2014.06.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Database screening of herbal monomers regulating autophagy by constructing a “disease-gene-drug” network</span>. <i>BMC Complementary Altern. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">466</span>, <span class="refDoi"> DOI: 10.1186/1472-6882-14-466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1186%2F1472-6882-14-466" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=25475428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A280%3ADC%252BC2MzltlOrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=466&author=C.+Haoauthor=Z.+Yangauthor=B.+Gaoauthor=M.+Luauthor=X.+Mengauthor=X.+Qiaoauthor=D.+Xueauthor=W.+Zhang&title=Database+screening+of+herbal+monomers+regulating+autophagy+by+constructing+a+%E2%80%9Cdisease-gene-drug%E2%80%9D+network&doi=10.1186%2F1472-6882-14-466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Database screening of herbal monomers regulating autophagy by constructing a "disease-gene-drug" network</span></div><div class="casAuthors">Hao Chenjun; Yang Zhengpeng; Gao Bo; Lu Ming; Meng Xianzhi; Qiao Xin; Zhang Weihui; Xue Dongbo</div><div class="citationInfo"><span class="NLM_cas:title">BMC complementary and alternative medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">466</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Studies suggest an important role of autophagy as a target for cancer therapy.  We constructed a "disease-gene-drug" network using the modular approach of bioinformatics and screened herbal monomers demonstrating functions related to autophagy regulation.  METHODS:  Based on the microarray results of the gene expression omnibus (GEO) database (GSE2435 and GSE31040, starvation-induced autophagy model), we used the human protein reference database (HPRD) to obtain the protein-protein interaction (PPI) network.  In addition, we used the CFinder software to identify several functional modules, performed gene ontology-biological process (GO-BP) functional enrichment analysis using the DAVID software, constructed a herbal monomer-module gene regulatory network using literature search and the Cytoscape software, and then analyzed the relationships between autophagy, genes, and herbal monomers.  RESULTS:  We screened 544 differentially expressed genes related to autophagy, 375 pairs of differentially expressed genes, and 7 gene modules, wherein the functions of module 3 (composed of 7 genes) were enriched in "cell death".  Using the constructed herbal monomer-module gene regulatory network, we found that 30 herbal monomers can simultaneously regulate these 7 genes, indicating a potential regulatory role in autophagy.  CONCLUSIONS:  Database screening using the disease-gene-drug network can provide new strategies and ideas for the application of herbal medicines in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT8xUK8QhIWR4vKgEt20y1XfW6udTcc2eaJFPX4fcZajrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzltlOrtw%253D%253D&md5=d3a1edcb08b6d7e31e3e2885198c16f6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1186%2F1472-6882-14-466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6882-14-466%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DQiao%26aufirst%3DX.%26aulast%3DXue%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DDatabase%2520screening%2520of%2520herbal%2520monomers%2520regulating%2520autophagy%2520by%2520constructing%2520a%2520%25E2%2580%259Cdisease-gene-drug%25E2%2580%259D%2520network%26jtitle%3DBMC%2520Complementary%2520Altern.%2520Med.%26date%3D2014%26volume%3D14%26spage%3D466%26doi%3D10.1186%2F1472-6882-14-466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y. Q.</span></span> <span> </span><span class="NLM_article-title">Identification of microRNA-regulated autophagic pathways in plant lectin-induced ccancer cell death</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2184.2012.00840.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fj.1365-2184.2012.00840.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=22882626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GrsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=477-485&issue=5&author=L.+L.+Fuauthor=X.+Zhaoauthor=H.+L.+Xuauthor=X.+Wenauthor=S.+Y.+Wangauthor=B.+Liuauthor=J.+K.+Baoauthor=Y.+Q.+Wei&title=Identification+of+microRNA-regulated+autophagic+pathways+in+plant+lectin-induced+ccancer+cell+death&doi=10.1111%2Fj.1365-2184.2012.00840.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of microRNA-regulated autophagic pathways in plant lectin-induced cancer cell death</span></div><div class="casAuthors">Fu, L.-L.; Zhao, X.; Xu, H.-L.; Wen, X.; Wang, S.-Y.; Liu, B.; Bao, J.-K.; Wei, Y.-Q.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-485</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Plant lectins, carbohydrate-binding proteins of non-immune origin, have recently been reported to induce programmed cell death (including apoptosis and autophagy) in many types of cancer cells.  MicroRNAs (miRNAs), small, non-coding endogenous RNAs, ∼22 nucleotides (nt) in length, have been well characterized to play essential roles in regulation of the autophagy process in cancer; however, how these miRNAs regulate autophagic pathways in plant lectin-induced cancer cells, still remains an enigma.  Identification of microRNA-regulated autophagic pathways was carried out using a series of elegant systems - biol. and bioinformatics approaches, such as network construction, hub protein identification, targeted microRNA prediction, microarray analyses and mol. docking.  We computationally constructed the human autophagic protein-protein interaction (PPI) network, and further modified this network into a plant lectin-induced network.  Subsequently, we identified 9 autophagic hub proteins and 13 relevant oncogenic and tumor suppressive miRNAs, that could regulate these aforementioned targeted autophagic hub proteins, in human breast carcinoma MCF-7 cells.  In addn., we confirmed that plant lectins could block the sugar-contg. receptor EGFR-mediated survival pathways, involved in autophagic hub proteins and relevant miRNAs, thereby ultimately culminating in autophagic cell death.  These results demonstrate that network-based identification of microRNAs modulate autophagic pathways in plant lectin-treated cancer cells, which may shed new light on the discovery of plant lectins as potent autophagic inducers, for cancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr08h3aoUEwS7Vg90H21EOLACvtfcHk0lh3RKUcol2MYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1GrsbbE&md5=834902129a211fff46450d4a82d4bdcf</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2184.2012.00840.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2184.2012.00840.x%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.%2BL.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%2BL.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%2BY.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBao%26aufirst%3DJ.%2BK.%26aulast%3DWei%26aufirst%3DY.%2BQ.%26atitle%3DIdentification%2520of%2520microRNA-regulated%2520autophagic%2520pathways%2520in%2520plant%2520lectin-induced%2520ccancer%2520cell%2520death%26jtitle%3DCell%2520Proliferation%26date%3D2012%26volume%3D45%26issue%3D5%26spage%3D477%26epage%3D485%26doi%3D10.1111%2Fj.1365-2184.2012.00840.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Identification of novel caspase/autophagy-related gene switch to cell fate decisions in breast cancers</span>. <i>Cell Proliferation</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1111/cpr.12005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1111%2Fcpr.12005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=23289893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslGrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2013&pages=67-75&issue=1&author=L.+L.+Fuauthor=Y.+Yangauthor=H.+L.+Xuauthor=Y.+Chengauthor=X.+Wenauthor=L.+Ouyangauthor=J.+K.+Baoauthor=Y.+Q.+Weiauthor=B.+Liu&title=Identification+of+novel+caspase%2Fautophagy-related+gene+switch+to+cell+fate+decisions+in+breast+cancers&doi=10.1111%2Fcpr.12005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel caspase/autophagy-related gene switch to cell fate decisions in breast cancers</span></div><div class="casAuthors">Fu, L. L.; Yang, Y.; Xu, H. L.; Cheng, Y.; Wen, X.; Ouyang, L.; Bao, J. K.; Wei, Y. Q.; Liu, B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-75</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Objectives : Caspases, a family of cysteine proteases with unique substrate specificities, contribute to apoptosis, whereas autophagy-related genes (ATGs) regulate cytoprotective autophagy or autophagic cell death in cancer.  Accumulating evidence has recently revealed underlying mechanisms of apoptosis and autophagy; however, their intricate relationships still remain to be clarified.  Identification of caspase/ATG switches between apoptosis and autophagy may address this problem.  Materials and methods : Identification of caspase/ATG switches was carried out using a series of elegant systems biol. & bioinformatics approaches, such as network construction, hub protein identification, microarray analyses, targeted microRNA prediction and mol. docking.  Results : We computationally constructed the global human network from several online databases and further modified it into the basic caspase/ATG network.  On the basis of apoptotic or autophagic gene differential expressions, we identified three mol. switches [including androgen receptor, serine/threonine-protein kinase PAK-1 (PAK-1) and mitogen-activated protein kinase-3 (MAPK-3)] between certain caspases and ATGs in human breast carcinoma MCF-7 cells.  Subsequently, we identified microRNAs (miRNAs) able to target androgen receptor, PAK-1 and MAPK-3, resp.  Ultimately, we screened a range of small mol. compds. from DrugBank, able to target the three above-mentioned mol. switches in breast cancer cells.  Conclusions : We have systematically identified novel caspase/ATG switches involved in miRNA regulation, and predicted targeted anti-cancer drugs.  These findings may uncover intricate relationships between apoptosis and autophagy and thus provide further new clues towards possible cancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq66f6u6W3mv7Vg90H21EOLACvtfcHk0lh3RKUcol2MYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslGrtw%253D%253D&md5=979ceb62ad722681d7102d2ef17ee25a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fcpr.12005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcpr.12005%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%2BL.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DBao%26aufirst%3DJ.%2BK.%26aulast%3DWei%26aufirst%3DY.%2BQ.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520novel%2520caspase%252Fautophagy-related%2520gene%2520switch%2520to%2520cell%2520fate%2520decisions%2520in%2520breast%2520cancers%26jtitle%3DCell%2520Proliferation%26date%3D2013%26volume%3D46%26issue%3D1%26spage%3D67%26epage%3D75%26doi%3D10.1111%2Fcpr.12005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Identification of ULK1 as a novel biomarker involved in miR-4487 and miR-595 regulation in neurobalstoma SH-SY5Y Cell Autophagy</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11035</span>, <span class="refDoi"> DOI: 10.1038/srep11035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1038%2Fsrep11035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26183158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWgs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11035&author=Y.+Chenauthor=S.+Wangauthor=L.+Zhangauthor=T.+Xieauthor=S.+Songauthor=J.+Huangauthor=Y.+Zhangauthor=L.+Ouyangauthor=B.+Liu&title=Identification+of+ULK1+as+a+novel+biomarker+involved+in+miR-4487+and+miR-595+regulation+in+neurobalstoma+SH-SY5Y+Cell+Autophagy&doi=10.1038%2Fsrep11035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of ULK1 as a novel biomarker involved in miR-4487 and miR-595 regulation in neuroblastoma SH-SY5Y cell autophagy</span></div><div class="casAuthors">Chen, Yi; Wang, Shuya; Zhang, Lan; Xie, Tao; Song, Sicheng; Huang, Jian; Zhang, Yonghui; Ouyang, Liang; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11035</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy, referring to an evolutionarily conserved, multi-step lysosomal degrdn. process, has been well-known to be initiated by Unc-51 like kinase 1 (ULK1) with some links to Parkinson's disease (PD).  MicroRNAs (miRNAs), small and non-coding endogenous RNAs 22 ∼ 24 nucleotides (nt) in length, have been demonstrated to play an essential role for modulating autophagy.  Recently, the relationships between miRNAs and autophagy have been widely reported in PD; however, how microRNAs regulate autophagy still remains in its infancy.  Thus, in this study, we computationally constructed the ULK1-regulated autophagic kinase subnetwork in PD and further identified ULK1 able to neg. regulate p70S6K in starvation-induced autophagy of neuroblastoma SH-SY5Y cells.  Combination of in silico prediction and microarray analyses, we identified that miR-4487 and miR-595 could target ULK1 and exptl. verified they could neg. or pos. regulate ULK1-mediated autophagy.  In conclusion, these results may uncover the novel ULK1-p70S6K autophagic pathway, as well as miR-4487 and miR-595 as new ULK1 target miRNAs.  Thus, these findings would provide a clue to explore ULK1 and its target miRNAs as potential biomarkers in the future PD therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp063qhAPzS8bVg90H21EOLACvtfcHk0li4PjlGRc29yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWgs7vM&md5=a7633347eaacfd907d8195066af71a48</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fsrep11035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11035%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520ULK1%2520as%2520a%2520novel%2520biomarker%2520involved%2520in%2520miR-4487%2520and%2520miR-595%2520regulation%2520in%2520neurobalstoma%2520SH-SY5Y%2520Cell%2520Autophagy%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11035%26doi%3D10.1038%2Fsrep11035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Systems biology-based discovery of a potential Atg4B agonist (Flubendazole) that induces autophagy in breast cancer</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2860</span>– <span class="NLM_lpage">2866</span>, <span class="refDoi"> DOI: 10.1039/C5MB00466G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1039%2FC5MB00466G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=26299935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWlt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=2860-2866&issue=11&author=L.+Zhangauthor=M.+Guoauthor=J.+Liauthor=Y.+Zhengauthor=S.+Zhangauthor=T.+Xieauthor=B.+Liu&title=Systems+biology-based+discovery+of+a+potential+Atg4B+agonist+%28Flubendazole%29+that+induces+autophagy+in+breast+cancer&doi=10.1039%2FC5MB00466G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology-based discovery of a potential Atg4B agonist (Flubendazole) that induces autophagy in breast cancer</span></div><div class="casAuthors">Zhang, Lan; Guo, Mingrui; Li, Jing; Zheng, Yaxin; Zhang, Shouyue; Xie, Tao; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2860-2866</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The aim of this study was to explore the autophagy-related protein 4B(ATG4B) and its targeted candidate agonist in triple-neg. breast cancer (TNBC) therapy.  In this study, the identification of Atg4B as a novel breast cancer target for screening candidate small mol. agonists was performed by phylogenetic anal., network construction, mol. modeling, mol. docking and mol. dynamics (MD) simulation.  In vitro, MTT assay, electron microscopy, western blot and ROS measurement were used for validating the efficacy of the candidate compds.  We used the phylogenetic anal. of Atg4B and constructed their protein-protein interaction (PPI) network.  Also, we screened target compds. of Atg4 proteins from Drugbank and ZINC.  Flubendazole was validated for its anti-proliferative efficacy in MDA-MB-231 cells.  Further MD simulation results supported the stable interaction between Flubendazole and Atg4B.  Moreover, Flubendazole induced autophagy and increased ROS prodn.  In conclusion, in silico anal. and exptl. validation together demonstrate that Flubendazole can target Atg4B in MDA-MB-231 cells and induce autophagy, which may shed light on the exploration of this compd. as a potential new Atg4B targeted drug for future TNBC therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4pY16N3ASc7Vg90H21EOLACvtfcHk0li4PjlGRc29yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWlt73O&md5=f1e946be8d60619445e25a70fde2f7b8</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1039%2FC5MB00466G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MB00466G%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DSystems%2520biology-based%2520discovery%2520of%2520a%2520potential%2520Atg4B%2520agonist%2520%2528Flubendazole%2529%2520that%2520induces%2520autophagy%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520BioSyst.%26date%3D2015%26volume%3D11%26issue%3D11%26spage%3D2860%26epage%3D2866%26doi%3D10.1039%2FC5MB00466G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Q.</span></span> <span> </span><span class="NLM_article-title">Bioinformatic analysis of cacybp-associated proteins using human glioma databases</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1002/liub.1999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=10.1002%2Fliub.1999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;key=30762928" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=C.+Xuanauthor=Y.+Gaoauthor=M.+Jinauthor=S.+Xuauthor=L.+Wangauthor=Y.+Wangauthor=R.+Hanauthor=K.+Shiauthor=X.+Chenauthor=Q.+An&title=Bioinformatic+analysis+of+cacybp-associated+proteins+using+human+glioma+databases&doi=10.1002%2Fliub.1999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fliub.1999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fliub.1999%26sid%3Dliteratum%253Aachs%26aulast%3DXuan%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAn%26aufirst%3DQ.%26atitle%3DBioinformatic%2520analysis%2520of%2520cacybp-associated%2520proteins%2520using%2520human%2520glioma%2520databases%26jtitle%3DIUBMB%2520Life%26date%3D2019%26doi%3D10.1002%2Fliub.1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01673&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-15%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01673" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01673" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b1084ec9e3c16","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
