
<html lang="en"     class="pb-page"  data-request-id="b9fb92ae-8f82-4f45-9930-e8c19462c184"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00227;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Structure–Activity Relationship Study of (Z)-5-Methylenethiazolidin-4-one Derivatives as Potent and Selective Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors" /></meta><meta name="dc.Creator" content="Theresa D.  Manz" /></meta><meta name="dc.Creator" content="Sindhu Carmen  Sivakumaren" /></meta><meta name="dc.Creator" content="Fleur M.  Ferguson" /></meta><meta name="dc.Creator" content="Tinghu  Zhang" /></meta><meta name="dc.Creator" content="Adam  Yasgar" /></meta><meta name="dc.Creator" content="Hyuk-Soo  Seo" /></meta><meta name="dc.Creator" content="Scott B.  Ficarro" /></meta><meta name="dc.Creator" content="Joseph D.  Card" /></meta><meta name="dc.Creator" content="Hyeseok  Shim" /></meta><meta name="dc.Creator" content="Chandrasekhar V.  Miduturu" /></meta><meta name="dc.Creator" content="Anton  Simeonov" /></meta><meta name="dc.Creator" content="Min  Shen" /></meta><meta name="dc.Creator" content="Jarrod A.  Marto" /></meta><meta name="dc.Creator" content="Sirano  Dhe-Paganon" /></meta><meta name="dc.Creator" content="Matthew D.  Hall" /></meta><meta name="dc.Creator" content="Lewis C.  Cantley" /></meta><meta name="dc.Creator" content="Nathanael S.  Gray" /></meta><meta name="dc.Description" content="Due to their role in many important signaling pathways, phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are attractive targets for the development of experimental therapeutics for cancer, meta..." /></meta><meta name="Description" content="Due to their role in many important signaling pathways, phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are attractive targets for the development of experimental therapeutics for cancer, meta..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 16, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00227" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00227" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00227" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00227" /></link>
        
    
    

<title>Discovery and Structure–Activity Relationship Study of (Z)-5-Methylenethiazolidin-4-one Derivatives as Potent and Selective Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00227" /></meta><meta property="og:title" content="Discovery and Structure–Activity Relationship Study of (Z)-5-Methylenethiazolidin-4-one Derivatives as Potent and Selective Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0011.jpeg" /></meta><meta property="og:description" content="Due to their role in many important signaling pathways, phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are attractive targets for the development of experimental therapeutics for cancer, metabolic, and immunological disorders. Recent efforts to develop small molecule inhibitors for these lipid kinases resulted in compounds with low- to sub-micromolar potencies. Here, we report the identification of CVM-05-002 using a high-throughput screen of PI5P4Kα against our in-house kinase inhibitor library. CVM-05-002 is a potent and selective inhibitor of PI5P4Ks, and a 1.7 Å X-ray structure reveals its binding interactions in the ATP-binding pocket. Further investigation of the structure–activity relationship led to the development of compound 13, replacing the rhodanine-like moiety present in CVM-05-002 with an indole, a potent pan-PI5P4K inhibitor with excellent kinome-wide selectivity. Finally, we employed isothermal cellular thermal shift assays (CETSAs) to demonstrate the effective cellular target engagement of PI5P4Kα and -β by the inhibitors in HEK 293T cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00227"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00227">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00227&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00227&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00227&amp;href=/doi/10.1021/acs.jmedchem.0c00227" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4880-4895</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00202" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00280" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Structure–Activity Relationship Study of (<i>Z</i>)-5-Methylenethiazolidin-4-one Derivatives as Potent and Selective Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Theresa D. Manz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Theresa D. Manz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical and Medicinal Chemistry, Saarland University, 66123 Saarbruecken, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Theresa+D.++Manz">Theresa D. Manz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sindhu Carmen Sivakumaren</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sindhu Carmen Sivakumaren</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sindhu+Carmen++Sivakumaren">Sindhu Carmen Sivakumaren</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fleur M. Ferguson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fleur M. Ferguson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fleur+M.++Ferguson">Fleur M. Ferguson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4091-7617" title="Orcid link">http://orcid.org/0000-0003-4091-7617</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tinghu Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tinghu Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tinghu++Zhang">Tinghu Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adam Yasgar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adam Yasgar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adam++Yasgar">Adam Yasgar</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7350-1402" title="Orcid link">http://orcid.org/0000-0001-7350-1402</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hyuk-Soo Seo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hyuk-Soo Seo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hyuk-Soo++Seo">Hyuk-Soo Seo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0646-2102" title="Orcid link">http://orcid.org/0000-0003-0646-2102</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott B. Ficarro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott B. Ficarro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology and Blais Proteomics Center, Dana-Farber Cancer Institute, Boston, Massachusetts, 02215, United States</div><div class="loa-info-affiliations-info">Department of Oncologic Pathology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+B.++Ficarro">Scott B. Ficarro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph D. Card</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph D. Card</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology and Blais Proteomics Center, Dana-Farber Cancer Institute, Boston, Massachusetts, 02215, United States</div><div class="loa-info-affiliations-info">Department of Oncologic Pathology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+D.++Card">Joseph D. Card</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hyeseok Shim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hyeseok Shim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Meyer Cancer Center, Weill Cornell Medicine and New York Presbyterian Hospital, New York, New York 10065, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hyeseok++Shim">Hyeseok Shim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chandrasekhar V. Miduturu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chandrasekhar V. Miduturu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chandrasekhar+V.++Miduturu">Chandrasekhar V. Miduturu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Simeonov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Simeonov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Simeonov">Anton Simeonov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Min Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Min Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Min++Shen">Min Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jarrod A. Marto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jarrod A. Marto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology and Blais Proteomics Center, Dana-Farber Cancer Institute, Boston, Massachusetts, 02215, United States</div><div class="loa-info-affiliations-info">Department of Oncologic Pathology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jarrod+A.++Marto">Jarrod A. Marto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sirano Dhe-Paganon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sirano Dhe-Paganon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sirano++Dhe-Paganon">Sirano Dhe-Paganon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Hall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Hall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Hall">Matthew D. Hall</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lewis C. Cantley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lewis C. Cantley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Meyer Cancer Center, Weill Cornell Medicine and New York Presbyterian Hospital, New York, New York 10065, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lewis+C.++Cantley">Lewis C. Cantley</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nathanael S. Gray</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nathanael S. Gray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f1bf908599909f90949dae96839088b195979298df99908387908395df949584"><span class="__cf_email__" data-cfemail="d896b9acb0b9b6b9bdb487bfaab9a198bcbebbb1f6b0b9aaaeb9aabcf6bdbcad">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nathanael+S.++Gray">Nathanael S. Gray</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5354-7403" title="Orcid link">http://orcid.org/0000-0001-5354-7403</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00227&amp;href=/doi/10.1021%2Facs.jmedchem.0c00227" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4880–4895</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 16, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 February 2020</li><li><span class="item_label"><b>Published</b> online</span>16 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00227" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00227</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4880%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DTheresa%2BD.%2BManz%252C%2BSindhu%2BCarmen%2BSivakumaren%252C%2BFleur%2BM.%2BFerguson%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c00227%26title%3DDiscovery%2Band%2BStructure%25E2%2580%2593Activity%2BRelationship%2BStudy%2Bof%2B%2528Z%2529-5-Methylenethiazolidin-4-one%2BDerivatives%2Bas%2BPotent%2Band%2BSelective%2BPan-phosphatidylinositol%2B5-Phosphate%2B4-Kinase%2BInhibitors%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4895%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00227"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1385</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00227" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Structure–Activity Relationship Study of (Z)-5-Methylenethiazolidin-4-one Derivatives as Potent and Selective Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Theresa&quot;,&quot;last_name&quot;:&quot;D. Manz&quot;},{&quot;first_name&quot;:&quot;Sindhu&quot;,&quot;last_name&quot;:&quot;Carmen Sivakumaren&quot;},{&quot;first_name&quot;:&quot;Fleur&quot;,&quot;last_name&quot;:&quot;M. Ferguson&quot;},{&quot;first_name&quot;:&quot;Tinghu&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Adam&quot;,&quot;last_name&quot;:&quot;Yasgar&quot;},{&quot;first_name&quot;:&quot;Hyuk-Soo&quot;,&quot;last_name&quot;:&quot;Seo&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;B. Ficarro&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;D. Card&quot;},{&quot;first_name&quot;:&quot;Hyeseok&quot;,&quot;last_name&quot;:&quot;Shim&quot;},{&quot;first_name&quot;:&quot;Chandrasekhar&quot;,&quot;last_name&quot;:&quot;V. Miduturu&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Simeonov&quot;},{&quot;first_name&quot;:&quot;Min&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Jarrod&quot;,&quot;last_name&quot;:&quot;A. Marto&quot;},{&quot;first_name&quot;:&quot;Sirano&quot;,&quot;last_name&quot;:&quot;Dhe-Paganon&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Hall&quot;},{&quot;first_name&quot;:&quot;Lewis&quot;,&quot;last_name&quot;:&quot;C. Cantley&quot;},{&quot;first_name&quot;:&quot;Nathanael&quot;,&quot;last_name&quot;:&quot;S. Gray&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4880-4895&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00227&quot;},&quot;abstract&quot;:&quot;Due to their role in many important signaling pathways, phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are attractive targets for the development of experimental therapeutics for cancer, metabolic, and immunological disorders. Recent efforts to develop small molecule inhibitors for these lipid kinases resulted in compounds with low- to sub-micromolar potencies. Here, we report the identification of CVM-05-002 using a high-throughput screen of PI5P4Kα against our in-house kinase inhibitor library. CVM-05-002 is a potent and selective inhibitor of PI5P4Ks, and a 1.7 Å X-ray structure reveals its binding interactions in the ATP-binding pocket. Further investigation of the structure–activity relationship led to the development of compound 13, replacing the rhodanine-like moiety present in CVM-05-002 with an indole, a potent pan-PI5P4K inhibitor with excellent kinome-wide selectivity. Finally, we employed isothermal cellular thermal shift assays (CETSAs) to demonstrate the effective cellular target engagemen&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00227&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00227" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00227&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00227" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00227&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00227" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00227&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00227&amp;href=/doi/10.1021/acs.jmedchem.0c00227" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00227" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00227" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00227%26sid%3Dliteratum%253Aachs%26pmid%3D32298120%26genre%3Darticle%26aulast%3DManz%26date%3D2020%26atitle%3DDiscovery%2Band%2BStructure%25E2%2580%2593Activity%2BRelationship%2BStudy%2Bof%2B%2528Z%2529-5-Methylenethiazolidin-4-one%2BDerivatives%2Bas%2BPotent%2Band%2BSelective%2BPan-phosphatidylinositol%2B5-Phosphate%2B4-Kinase%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D9%26spage%3D4880%26epage%3D4895%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291137" title="Lipids">Lipids</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/jmcmar.2020.63.issue-9/20200514/jmcmar.2020.63.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Due to their role in many important signaling pathways, phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are attractive targets for the development of experimental therapeutics for cancer, metabolic, and immunological disorders. Recent efforts to develop small molecule inhibitors for these lipid kinases resulted in compounds with low- to sub-micromolar potencies. Here, we report the identification of CVM-05-002 using a high-throughput screen of PI5P4Kα against our in-house kinase inhibitor library. CVM-05-002 is a potent and selective inhibitor of PI5P4Ks, and a 1.7 Å X-ray structure reveals its binding interactions in the ATP-binding pocket. Further investigation of the structure–activity relationship led to the development of compound <b>13</b>, replacing the rhodanine-like moiety present in CVM-05-002 with an indole, a potent pan-PI5P4K inhibitor with excellent kinome-wide selectivity. Finally, we employed isothermal cellular thermal shift assays (CETSAs) to demonstrate the effective cellular target engagement of PI5P4Kα and -β by the inhibitors in HEK 293T cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17002" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17002" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Phosphoinositides, such as phosphatidylinositol 4,5-bisphosphate (PI-4,5-P<sub>2</sub>), are key players in many cell-regulating and -signaling processes. Inositide function is controlled by lipid kinases, which alter their phosphorylation pattern, thus generating a range of signaling molecules with different downstream effects. Phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) represent a family of these lipid kinases, which have been shown to phosphorylate phosphatidylinositol 5-phosphate (PI5P) on its 4-position to produce PI-4,5-P<sub>2</sub>.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In healthy cells, PI5P4Ks play a role in regulating cellular metabolism, stress response, and immunological processes,<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7">(2−7)</a> while the abnormal function of these lipid kinases has been linked to diseases, such as diabetes and cancer.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8−11)</a> As a result, PI5P4K inhibitors pose potentially important therapeutic agents, for example, in oncology. To date, several small molecule PI5P4K inhibitors with low- or sub-micromolar potency have been reported in the literature,<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17">(12−17)</a> including our recently reported phenylamino pyrimidine-based, covalent pan-PI5P4K inhibitors, THZ-P1-2 and compound <b>32</b> (labeled compound “<b>30</b>” in the original publication; <a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 1</a>).<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a></div><div class="NLM_p">Here, we report our efforts to find chemically distinct PI5P4K inhibitors that can serve as chemical probes for further drug development efforts. We focused on developing reversible inhibitors that would not be susceptible to potentially acquired resistance through mutation of the active site cysteine residue, which is modified by irreversible PI5P4K inhibitors.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> A prominent example of this type of resistance mechanism is the EGFR C797S mutation that develops in nonsmall cell lung cancer following treatment with irreversible small molecule EGFR inhibitor AZD9291 and renders tumors resistant.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p last">To find novel, reversible PI5P4K inhibitor scaffolds, we performed a high-throughput screen of our internal kinase inhibitor library, containing approximately 6,000 structurally diverse compounds developed and synthesized in our lab against PI5P4Kα, using an ADP-Glo assay to measure the PI5P4Kα kinase activity. This screen identified potent hit compound CVM-05-002 (IC<sub>50</sub> 270 nM). CVM-05-002 contains a distinct (<i>Z</i>)-5-methylenethiazolidin-4-one moiety, which can be described as a rhodanine-like structure. Although rhodanine-type compounds can function as pan-assay interference compounds (PAINS) in AlphaScreen assays,<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a> in other settings, rhodanines and their derivatives elaborated through focused medicinal chemistry campaigns<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> have been advanced into clinical trials.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In this study, we describe the discovery, validation, and structure–activity relationship (SAR) of compounds based on this scaffold as potent, specific pan-PI5P4K inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> High-Throughput Screen</h3><div class="NLM_p">Since reliable ADP-Glo-based assays for PI5P4Kα inhibition have been previously described,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> we adapted this assay to establish a protocol for a miniaturized high-throughput screen. We optimized the assay to perform in a 384-well format with respect to PI5P4Kα enzyme concentration and ADP-Glo reagent volumes to obtain a robust signal-to-background ratio, a stable <i>Z</i>′-factor (0.85), and a low coefficient of variation (CV = 0.098) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 2</a>). Our in-house compound library, consisting of diverse kinase inhibitor scaffolds, was screened at a concentration of 66 μM. This relatively high concentration was chosen based on previous reports describing low potency inhibition of PI5P4K by known kinase inhibitor scaffolds, indicating that the identification of a highly potent inhibitor for this subfamily of lipid kinases may be more challenging than for the protein kinase family. Our screen resulted in the identification of approximately 50 hits exhibiting >90% inhibition (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figures 3 and 4</a>). With a mean of 2.7% dimethyl sulfoxide (DMSO) control, CVM-05-002 showed the strongest effects on the screen. All identified hits were analyzed regarding the diversity of the scaffolds, and historical data relating to kinase selectivity was examined. Highly promiscuous or potentially covalent scaffolds were excluded from the set. Based on this, a subset of hit compounds was tested in an 11-point concentration–response experiment, which confirmed CVM-05-002 as the most potent hit for PI5P4Kα inhibitory activity (IC<sub>50</sub> 0.27 μM; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,C). We verified that the PI5P4Kα activity was not due to compound interactions with assay reagents. This was done by counterscreening at a fixed ADP/ATP concentration ratio representing 20% conversion but excluding the PI5P4Kα enzyme, as well as the lipid substrate, confirming no assay interference with CVM-05-002 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Scatter plot of high-throughput screen for the PI5P4Kα activity of in-house small molecule library with ∼6000 compounds (shown are hits at <120% DMSO control). The results of two replicates are plotted against each other (normalized to DMSO). Negative controls (DMSO) are shown in dark gray, CVM-05-002 is depicted in green, and the 10% DSMO control cutoff (90% inhibition) is labeled. (B) Concentration–response curves of CVM-05-002, including counterscreen results. (C) Structure of identified lead compound, CVM-05-002.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interestingly, several structurally similar molecules to CVM-05-002 were present in the library, while only four analogues were found among the top hits identified in the high-throughput screen. Subsequent testing of these hits in our ADP-Glo-based PI5P4Kα assay revealed that these analogues are ∼30-fold less potent than CVM-05-002, or more (data not shown). Furthermore, the closest analogues of CVM-05-002, only differing in the 3,5-substituted pyridine moiety of the hit compound, showed strong PIM kinase off-targets (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 1</a>). This was not surprising since similar scaffolds have been shown to be potent PIM kinase inhibitors.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, since CVM-05-002 was the only compound of this scaffold without any PIM kinase off-target activity, carrying a unique 3,5-substituted pyridine moiety, while showing potent on-target activity, we were highly motivated to further optimize this screening hit and explore its SAR.</div><div class="NLM_p last">Finally, knowing that the scaffold of CVM-05-002 could potentially have nonspecific effects due to its similarity to compounds with reported reactivity,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> we performed mass spectrometry (MS) analysis of recombinant PI5P4Kα and -β, incubated with CVM-05-002. PI5P4Kα showed no detectable reaction after 2 h, while the β-isoform of the lipid kinase was modified only at a very low level (∼14%). Under the same conditions, our previously reported covalent PI5P4K inhibitor, THZ-P1-2,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> labeled ∼63–87% of both isoforms (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figures 5 and 6</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structural Basis of the CVM–PI5P4K Interaction</h3><div class="NLM_p">Recombinant PI5P4Kα was cocrystallized in complex with CVM-05-002 at a 1.7 Å resolution, confirming a noncovalent binding mode of the inhibitor in the active site of the lipid kinase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) (PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9">6UX9</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Binding site of cocrystal structure of PI5P4Kα with bound inhibitor CVM-05-002 (dark gray). Key hydrogen bond interactions are depicted by dashed lines (numbers indicate distances in Å). (B) Key hydrogen bond interactions of the hinge-binding site are shown as dashed lines (numbers indicate distances in Å). (C) Proposed hydrogen bond network of the thiazolidinone moiety of CVM-05-002 with Lys209 and Thr232 (interactions are shown as dashed lines; numbers indicate distances in Å). PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9">6UX9</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The obtained cocrystal structure revealed general hydrophobic interactions of CVM-05-002 with the ATP-binding pocket. For example, Phe178, Ile147, Ile194, and Leu361 form a nonpolar pocket encasing the methyl group of the sulfonamide (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). In addition, five key hydrogen-bonding interactions could be identified. At the hinge, the pyridine nitrogen anchors the inhibitor <i>via</i> a hydrogen bond with the backbone of Val199 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The thiazolidinone moiety interacts with the side chains of Thr232 and Lys209 on the left-hand side of the scaffold, while the sulfonamide engages in two hydrogen-bonding interactions with the backbone of Ile360 and the side chain of Asp359 on the right-hand side (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Furthermore, Asp359 and Lys145 form a salt-bridge, potentially improving the hydrogen-bonding interaction with the sulfonamide-NH. Likewise, it should be noted that the thiazolidinone moiety is positioned in a way that permits the formation of hydrogen-bonding interactions of the imine, the carbonyl, and the cyclic-NH groups with Thr232 and Lys209, together forming a fork-like hydrogen bond network on the left-hand side of the inhibitor with the binding site. While the interaction of the pyridine moiety seemed to be essential for the selectivity of the molecule (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 1</a>), the strong left- and right-hand-side hydrogen bond interactions are likely contributing to the relatively high potency of this scaffold (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,C).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Selectivity Screening</h3><div class="NLM_p">The kinome-wide binding selectivity of CVM-05-002 at a 1 μM concentration was determined, using the commercial KINOMEscan profiling platform, consisting of 468 kinases. CVM-05-002 showed an excellent selectivity profile (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>; selectivity score for hits with a percent control < 35, <i>S</i><sub>35</sub> = 0.02), providing further evidence that this inhibitor is a selective and potent pan-PI5P4K inhibitor. Taken together, these findings were encouraging to pursue further medicinal chemistry efforts to better understand the SAR around this promising hit.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinome-wide selectivity profile for compounds CVM-05-002, <b>3</b>, <b>13</b>, and <b>31</b> (excluding mutant, atypical, and pathogen panels). The inhibitors were tested at 1 μM on a panel of 468 kinases. The results are displayed as red circles with their sizes correlating with the inhibitor’s binding affinity (percent DMSO control).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Chemistry</h3><div class="NLM_p">Given the above, the presented SAR study focuses mainly on changes regarding the thiazolidinone and the <i>N</i>-phenylmethanesulfonamide groups, decorating the hinge-binding pyridine moiety and being referred to as left- and right-hand sides of the scaffold, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Hence, two main series of CVM-05-002 analogues were designed as summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, with changes made on either side of the molecule.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Evaluation of Compounds CVM-05-002, <b>3</b>–<b>15</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0006.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0007.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Average ± SD of <i>n</i> = 2 or 3 experiments, each using an 11-point titration.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> Evaluation of Compounds <b>18</b>–<b>27</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0008.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0009.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Average ± SD of <i>n</i> = 2 or 3 experiments, each using an 11-point titration.</p></div></div><div></div></div><div class="NLM_p">In general, CVM-05-002 and analogues were synthesized as summarized in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, starting from a 3-bromopyridine analogue (<b>1, 11, 16, 28a,b</b>) and by introducing different aromatic ring systems <i>via</i> Suzuki couplings (<b>2a–c</b>, <b>6–8</b>, <b>12a–c</b>, <b>13–15</b>, <b>17a–j</b>, and <b>29a,b</b>). (<i>Z</i>)-Methylenethiazolidinone and its analogues were accessed from nicotinaldehyde derivatives, which were coupled with either 2-iminothiazolidin-4-one, oxazolidine-2,4-dione, or maleimide, using different Knoevenagel condensation reaction conditions (CVM-05-002, <b>3–5</b>, <b>18–27</b>, and <b>30–31</b>). Compounds <b>9</b> and <b>10</b> were yielded <i>via</i> S<sub>N</sub>2 reactions with different 3-substituted <i>N</i>-phenylacetamide analogues.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound CVM-05-002 and Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phenylboronic acids/respective pinacol boronic acid esters, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 9–93% yield; (b) (<i>Z</i>)-methylenethiazolidinone derivative, β-alanine, AcOH, 100 °C, 0.5 h, 12–33% yield, or maleimide, PPh<sub>3</sub>, dichloromethane (DCM), rt, overnight, 21% yield, or oxazolidine-2,4-dione, piperidine, EtOH, 100 °C, overnight, 4% yield, or phenylboronic acids/respective pinacol boronic acid ester, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 1–37% yield, or <i>N</i>-(3-hydroxyphenyl)acetamide, CuI, K<sub>2</sub>CO<sub>3</sub>, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 120 °C, 2 d, 3% yield, or <i>N</i>-(3-bromophenyl)acetamide, Pd<sub>2</sub>(dba)<sub>3</sub>, X-Phos, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 80 °C, overnight, 7% yield; (c) 4-(methanesulfonylamino)phenylboronic acid pinacol ester, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 19–27% yield; (d) 2-iminothiazolidin-4-one, β-alanine, AcOH, 100 °C, 0.5 h, 4–22% yield.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Biological Evaluation</h3><div class="NLM_p">The biochemical inhibition of PI5P4Kα and -β was determined as previously described.<a onclick="showRef(event, 'ref12 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref17">(12,17)</a> Due to its low enzymatic activity,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> we were unable to determine the PI5P4Kγ activity for our compounds; thus, this discussion focuses solely on the PI5P4Kα and -β isoforms. It should be noted that the structural alignment of all three PI5P4Ks confirmed that the binding pocket is highly conserved within this lipid kinase family (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 7</a>). The amino acids, highlighted in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, are identical among all three isoforms, with the exception of Ile147 and Ile194 in PI5P4Kα, which are being replaced by valines in PI5P4Kβ or a valine and a leucine, respectively, in PI5P4Kγ. Val199 in PI5P4Kα is a methionine in PI5P4Kγ, which should not affect the hinge binding of CVM-05-002, since the interaction of the pyridine is with the backbone of the amino acid.</div><div class="NLM_p">In this biochemical evaluation, our initial hit compound, CVM-05-002, showed consistent IC<sub>50</sub> values of 270 nM and 1.69 μM for PI5P4Kα and -β, respectively, hence representing—to the best of our knowledge—the most potent reported pan-PI5P4K inhibitor to date. However, CVM-05-002 is a polar compound with a log <i>D</i> value of 0.66, which limits its use in cells and, subsequently, as a useful probe to study PI5P4Ks <i>in vivo</i>. The balance of lipophilicity and PI5P4K inhibitory activity was therefore another crucial criterion of this SAR study.</div><div class="NLM_p">With a first set of compounds, summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we introduced different substituents on the left-hand side of the scaffold, altering or replacing the methylenethiazolidinone moiety. For example, changing the imine of the thiazolidinone to a carbonyl (compound <b>3</b>) slightly improved the inhibitory activity with respect to PI5P4Kα by about 2-fold to an IC<sub>50</sub> of 133 nM, while the β-activity did not change. This is not surprising since the carbonyl-O and the imine-NH can both function as hydrogen bond acceptors. However, an oxazolidine-2,4-dione in compound <b>4</b> led to an inactive compound on both isoforms, highlighting the importance of the electrochemical properties of this five-membered ring system for the PI5P4K activity, since the electronegativity of oxygen (compound <b>4</b>) is much higher as compared to that of sulfur (compound <b>3</b>). This is further supported by the fact that introducing a carbon (compound <b>5</b>), having similar electronegativity as sulfur, in position of the oxygen (compound <b>4</b>) led to a moderately active compound with IC<sub>50</sub>s of 6.1 and 27 μM for PI5P4Kα and -β, respectively. It should be noted that this equals a 1–2-log-fold drop in the PI5P4K activity of compound <b>5</b> compared to that of compound <b>3</b>, pointing toward an essential role of the cyclic sulfur for the PI5P4K activity of the scaffold. One major difference between the five-membered ring systems of compounds <b>3</b> and <b>5</b> is the aromaticity. Whereas compound <b>3</b> shows aromatic features due to the lone electron pairs of the sulfur, compound <b>5</b> introduces an sp<sup>3</sup>-hybridized carbon atom, disrupting the aromatic π-system of the ring.</div><div class="NLM_p">All compounds mentioned above have lower log <i>D</i> values than CVM-05-002, with compounds <b>3</b> and <b>5</b> having a log <i>D</i> of around 0, while compound <b>4</b> has the lowest log <i>D</i> value of the set with −1. Consequently, we next focused on replacing the entire methylenethiazolidinone moiety to improve lipophilicity of the inhibitors. Strategies for this replacement were inspired by the cocrystal structure, which suggests that the (<i>Z</i>)-5-methylenethiazolidin-4-one could be cyclized, by linking the methylene-H with the cyclic carbonyl to form an aromatic six-membered ring. An initial version of this is compound <b>6</b>, carrying an indolinone moiety on the left-hand side. This structure is closely related to compound <b>5</b> since the five-membered ring systems of both compounds only differ in the carbonyl oxygen in the 2-position, which is missing in compound <b>6</b>. Here, an aromatic six-membered ring, directly connecting to the pyridine, was introduced instead. The compound showed IC<sub>50</sub>s of 8.9 and 15 μM for PI5P4Kα and -β, respectively, which is comparable to compound <b>5</b>’s PI5P4K activity, while improving log <i>D</i> by 1.5-log-folds to 1.5. Interestingly, the loss of the carbonyl interaction did not seem to affect the overall PI5P4K activity, which might indicate a positive effect of the aromatic six-membered ring system of compound <b>6</b> for the PI5P4K activity, potentially compensating for the loss of the carbonyl hydrogen bond interaction.</div><div class="NLM_p">Due to the overall connected π-electron system in compound <b>6</b>, the molecule is likely rather flat, as compared to CVM-05-002, which shows a slight bend at the methylene linker between the pyridine and thiazolidinone moiety in the cocrystal structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Thus, the left-hand side residues with higher flexibility, especially in the hydrogen-bond-forming groups to compensate for this, were of interest to us. Ring opening of the pyrrolidine-2-one ring, resulting in a more flexible amide in compound <b>7</b>, led to minor improvements in the PI5P4Kα activity while showing a slight drop in -β activity with IC<sub>50</sub>s of 5.3 and 36 μM, respectively. This trend was further pronounced in compound <b>8</b>, introducing a methylated urea motif in the para-position of the aromatic ring system, with IC<sub>50</sub>s of 3.7 μM for PI5P4Kα and 43 μM for PI5P4Kβ. Compound <b>8</b> offers more potential hydrogen bond interactions than the acetamide of compound <b>7</b>, which could be an explanation for its slightly higher potency on the α-isoform of the lipid kinase. It should be noted, however, that, even though there seems to be a drop in the PI5P4Kβ activity of over 2-fold from compound <b>6</b> to compound <b>8</b>, the relatively high standard deviation values are causing the differences in IC<sub>50</sub> values to be statistically nonsignificant.</div><div class="NLM_p">Introduction of an O- or NH-linker to compound <b>7</b> (compounds <b>9</b> and <b>10</b>, respectively) is further increasing the flexibility of the acetamide to form hydrogen bond interactions with the lipid kinase. Both compounds showed a slight loss in the PI5P4Kα activity by about 2–3-fold compared to compound <b>7</b>, while there was no significant difference in the PI5P4Kβ activity. In summary, comparing the overall lipophilic efficiency (LipE) of these molecules, the flatness of the SAR for compounds <b>6</b>–<b>10</b> becomes obvious. LipE is an important feature in drug discovery since it sets the lipophilicity of a molecule in relation to its activity, here, log <i>D</i> to pIC<sub>50</sub>. Since compounds <b>6</b>–<b>10</b> have very similar log <i>D</i> values, plotting against pIC<sub>50</sub> shows that the compounds depict almost identical PI5P4Kα and -β activity profiles with LipE values between 3 and 4 for PI5P4Kα and of values of about 3 for PI5P4Kβ (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 10</a>), clearly showing no real improvement of the PI5P4K activity by more flexible hydrogen-bond-offering moieties. These findings indicate a limited benefit of the hydrogen bond interactions in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C while highlighting the gained PI5P4K activity through the introduced aromatic ring system.</div><div class="NLM_p">When further simplifying the bicycle of compound <b>6</b> by introducing an indole on the left-hand side of the molecule (compound <b>13</b>), the activity on both PI5P4K isoforms could be improved to low micromolar IC<sub>50</sub>s. While the slight improvement in the PI5P4Kβ activity is not significant, the PI5P4Kα activity of compound <b>13</b> improves significantly to 2.0 μM (<i>P</i> value = 0.049 by <i>t</i>-test) as compared to that of compound <b>6</b>. This further supports our findings that the aromaticity of the bicyclic system is more important than the hydrogen-bonding interaction by the lactam. Hence, removing the carbonyl of compound <b>6</b>, while increasing the aromaticity across both rings of the bicyclic moiety, did not result in loss in the potency of the compound, but actually improved the potency on PI5P4Kα. Consequently, the higher lipophilicity with a log <i>D</i> value of 2.5, which is an improvement of over 1-log-fold as compared to that of compounds <b>6</b>–<b>10</b>, as well as the enlarged aromatic ring system of compound <b>13</b>, is beneficial for the PI5P4Kα activity, but also likely improves the properties of the compound for the use in cells.</div><div class="NLM_p">We assume that this benefit of aromatic moieties on the left-hand side could be indicating π–π interactions with either Phe200 or Phe134 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), which we had previously identified as a crucial interaction for our reported covalent pan-PI5P4K inhibitor series.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> This was supported by the fact that further changes in the attachment site of the indole from the 4-position (compound <b>13</b>) to the 5- or 6- position (compounds <b>14</b> and <b>15</b>, respectively) led to no change in the PI5P4Kα activity and only a minor 2–3-fold decrease in the β-activity. Even though compounds <b>13</b>–<b>15</b> show, thus, a very similar PI5P4K activity profile, compound <b>13</b>, the isomer closely related to compound <b>6</b>, showed the best-combined PI5P4Kα and -β activities. As already mentioned, since all three compounds have improved log <i>D</i> values of 2.5, leading to LipE values of around 3 for PI5P4Kα and values of 2–2.5 for PI5P4Kβ, the compounds should show a good cellular target engagement, considering the improved balance of lipophilicity and biochemical activity as compared to the initial hit compound, CVM-05-002 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 10A,B</a>).</div><div class="NLM_p">In a next step, the SAR on the right-hand side of CVM-05-002 was investigated by introducing aromatic ring systems, carrying different substituents ranging from polar sulfonamides, amides, and urea motifs to nonpolar and bulkier moieties (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The sulfonamide moiety in CVM-05-002 significantly contributes to the compound’s high hydrophilicity, which inspired designs that aim to create compounds of higher lipophilicity, while at the same time interrogating the necessity of the sulfonamide of the parent compound for the PI5P4K activity.</div><div class="NLM_p">Compound <b>18</b>, a reversed sulfonamide analogue of CVM-05-002, lost activity against both PI5P4K isoforms by about 1-log-fold, most likely due to the changed hydrogen bond donor/acceptor configuration. Removing the sulfonamide-NH in compound <b>19</b> kept the log <i>D</i> of the compound comparable to that in compound <b>18</b> at 1.1; however, the overall polar surface area (PSA) of the compound decreased from 112 for CVM-05-002 and compound <b>18</b> to 100 for compound <b>19</b>. Interestingly, losing this additional NH hydrogen-bonding interaction led to a complete loss of activity on PI5P4Ks. These results might indicate that the sulfonamide moiety of CVM-05-002 is crucial for the optimal PI5P4K activity.</div><div class="NLM_p">Switching from a sulfonamide to an acetamide (compound <b>20</b>) increased the lipophilicity by about 1-log-fold, compared to that of CVM-05-002, to a log <i>D</i> value of 1.5. Nevertheless, the trigonal-planar conformation of the amide, in contrast to the tetrahedral conformation of the sulfonamide, led to an activity decrease of 2-log-folds or higher on the PI5P4K isoforms. The configuration of the amide in compound <b>20</b> makes it impossible for the hydrogen of the NH and the oxygen-O to point into the same direction, which is necessary to form both hydrogen bond interactions seen for CVM-05-002 with Asp359 and Ile360. This is further supported by compound <b>21</b>, carrying a methylated urea motif and showing improved biochemical potency on both PI5P4K isoforms to levels comparable to the reversed sulfonamide analogue <b>18</b>, while having a higher log <i>D</i> of around 1.4. The additional NH-moiety in compound <b>21</b> is likely forming similar interactions to the NH group of compound <b>18</b>, which would explain the similar potency, as well as the lack in the potency of compound <b>20</b>. Changing the acetamide from <i>p</i>- to <i>m</i>-position (compound <b>22</b>) did not influence the PI5P4K activity, underlining a certain tolerance in this substituent. The meta-position could either point toward the deeper pocket, or toward the thiazolidinone moiety, which would mean that both the NH and the carbonyl-<i>O</i> could potentially form hydrogen bond interaction with Asp359 or Ile360.</div><div class="NLM_p">Interestingly, compound <b>23</b>, carrying a trifluoromethoxy substituent, exhibited the same activity profile as compound <b>20</b> with weak PI5P4Kα (11 μM) and no PI5P4Kβ activity. This points toward a tolerance for hydrogen bond donors, as well as acceptors for PI5P4Kα in this position, which is in accordance with the cocrystal structure, since Lys145 is in reach to form hydrogen bond interactions with the methoxy-<i>O</i> (2.4 Å). Removing the hydrogen bond acceptor, resulting in compound <b>24</b>, supports this hypothesis by showing the loss of PI5P4Kα activity. In addition to that, compounds <b>23</b> and <b>24</b> exhibit good lipophilicity for cellular treatments with log <i>D</i> values of 3.7 and 3.2, respectively, while the matched methoxy or methyl substituents would have had log <i>D</i> values of only 1.4 or 1.9 instead. Surprisingly, compound <b>24</b> gained back some PI5P4Kβ activity with an IC<sub>50</sub> of 26 μM. However, this β-activity does not seem to be influenced by the trifluoromethyl moiety since a simple fluoro substituent (compound <b>25</b>) led to the same activity profile. Furthermore, comparing compounds <b>23</b> and <b>25</b>, it seems that the polar NH- or O-groups adjacent to the aromatic ring are beneficial for the PI5P4Kα activity, whereas it is unfavorable in the context of PI5P4Kβ activity, which is in contrast to the high similarity of the two binding sites (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 7</a>).</div><div class="NLM_p">Similarly, compounds <b>26</b> and <b>27</b> introduce either a dimethylamine or a methoxy group, respectively, in the para-position of the phenyl moiety on the right-hand side. These substituents pose hydrogen bond acceptors, leading to the activity pattern already seen above with PI5P4Kα, but no -β activity. Interestingly, the bulky naphthalene of compound <b>27</b> was well tolerated by PI5P4Kα and even led to a slight increase in IC<sub>50</sub> (3.9 μM) as compared to that of compound <b>23</b> (11 μM). Despite the desirable lipophilicity of these two molecules with log <i>D</i> values of 2.4 and 3.1 for compounds <b>26</b> and <b>27</b>, respectively, the lack of PI5P4Kβ activity, as seen with other compounds of this batch of CVM-05-002 analogues, makes them unfit as pan-PI5P4K inhibitor probes. Taken together, it appears that trying to increase lipophilicity by changing the right-hand side of the molecule led to either poor PI5P4Kα or -β inhibitors, while even slight changes in the sulfonamide moiety of CVM-05-002 led to a drop in PI5P4K activity of at least 1-log-fold. These findings suggest that the sulfonamide of CVM-05-002 is indeed necessary to maintain both PI5P4Kα and -β activities.</div><div class="NLM_p">In summary, the SAR analysis on the left- and right-hand sides of the PI5P4K inhibitor scaffold revealed that the key interactions seen in the cocrystal structure of PI5P4Kα with CVM-05-002 seem to be essential for the potency of the inhibitor. Removal of hydrogen bond donors or acceptors on either side of the molecule generally led to a minimum of 1-log-fold decrease in potency. Interestingly, replacing the (<i>Z</i>)-methylenethiazolidinone moiety with an aromatic bicyclic indole (compound <b>13</b>) retained good potency comparable to our previously reported pan-PI5P4K inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 1</a>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> while improving the lipophilicity of the scaffold by about 2-log-folds.</div><div class="NLM_p">Although we had shown that the 3,5-substituted pyridine is likely essential for the CVM-05-002’s kinome-wide selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 1</a>), we had noticed a small pocket extending from the 2-position of the pyridine at the hinge-binding region (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 8</a>). Hence, we designed and synthesized the additional compounds <b>30</b> and <b>31</b> with an amino group at either the 2- or 6-position of the central pyridine of CVM-05-002. According to the cocrystal structure, the amino group of compound <b>30</b> should cause a steric clash with the protein surface at the hinge, whereas compound <b>31</b> should be able to fit the amino group into the adjacent pocket, potentially causing a shift in the compound’s selectivity profile. The polar amino group was chosen since the backbone carbonyl of Arg197, as well as the side-chain hydroxyl group of Thr196, is in close proximity to the pyridine with distances under 4 Å (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 8</a>). 2-Aminopyridine moieties are also well-known hinge-binding motifs of kinase inhibitors since they mimic the adenine binding interactions of ATP.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a> As expected, compound <b>30</b> lost activity on both, PI5P4Kα and -β, by almost 1.5-log-folds with IC<sub>50</sub>s of 4.0 and 25 μM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In contrast to this 15-fold decrease in potency, compound <b>31</b> showed only a very minor decrease in potency with 660 nM and 2.7 μM potency on PI5P4Kα and -β, respectively. Naturally, both compounds are more polar than the parent inhibitor, CVM-05-002, with log <i>D</i> values of 0.4.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> Evaluation of Compounds <b>30</b> and <b>31</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0010.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Average ± SD of three experiments, each using an 11-point titration.</p></div></div><div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Selectivity Profiling</h3><div class="NLM_p">Selected compounds of our series were tested on a set of 468 kinases in Ambit’s commercially available KINOMEscan platform. Besides our most potent pan-PI5P4K inhibitors (CVM-05-002, as well as compounds <b>3</b> and <b>31</b>), compound <b>13</b>, with the best balance between lipophilicity and biochemical activity, was selected as well. The compounds’ binding activity was determined at 1 μM as percent DMSO control and is summarized in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. It should be noted that PI5P4Kα was not available in this profiling panel.</div><div class="NLM_p">Our initial hit compound, CVM-05-002 (selectivity score for hits with a percent control < 35; <i>S</i><sub>35</sub> = 0.02), showed high selectivity among the kinases tested, with only 10 hits at a cutoff of 35% DMSO control, such as activin A receptor type 2A (ACVR2A) and TGFBR2, both of which are members of the TGFβ family receptors.<a onclick="showRef(event, 'ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35">(33−35)</a> Within the lipid kinases, PI5P4Kγ is the only strongly bound target, suggesting excellent selectivity of the inhibitor within the lipid kinase families. PI5P4Kβ did not show up as a hit in this assay, potentially due to the weaker activity of CMV-05-002 toward this isoform. Consequently, we do not believe that the assay is a good predictor for identifying PI5P4Kβ activity in the micromolar range at this concentration, but it is still very useful to identify potential off-targets. This is true for all of the analogues tested in this assay format.</div><div class="NLM_p">Surprisingly, even though compound <b>3</b> is similarly potent to CVM-05-002 in biochemical assays, the change from the 2-iminothiazolidin-4-one to the thiazolidine-2,4-dione led to a slight selectivity shift of the inhibitor. With a similar number of off-targets (<i>S</i><sub>35</sub> = 0.03), compound <b>3</b> was not more promiscuous than CVM-05-002; however, its activity shifted from ACVR2A toward MEK3. More importantly, among the lipid kinases, a major off-target, phosphatidylinositol 4-phosphate 5-kinase γ (PI4P5Kγ), showed strong binding affinity to compound <b>3</b>, while losing affinity for PI5P4Kγ. We further confirmed potential off-targets by commercially available biochemical activity assays (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 2</a>). This data validated that CVM-05-002 and compound <b>3</b> have potent off-target activity for activin A receptor type 2B (ACVR2B) and TGFBR2. Compound <b>3</b> showed slightly stronger PI4P5Kγ activity as compared to CVM-05-002 by almost 2-fold, which is not as pronounced as in the Ambit KINOMEscan profiling, nonetheless confirming this lipid kinase off-target. As mentioned before, there are no kinase activity assays for PI5P4Kγ; however, we have compared the binding affinity using <i>K</i><sub>D</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 2</a>). This data supports our findings that CVM-05-002 is a strong binder of PI5P4Kγ with a <i>K</i><sub>D</sub> of 1.6 nM. Interestingly, compound <b>3</b> showed a slightly stronger <i>K</i><sub>D</sub>. It should be noted that we have added our previously reported pan-PI5P4K inhibitor, THZ-P1-2, as a positive control for PI5P4Kγ binding (<i>K</i><sub>D</sub> of 4.8 nM), where we have additionally shown covalent engagement of the lipid kinase.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p last">Comparing the selectivity profile of CVM-05-002 and compound <b>13</b>, replacing the (<i>Z</i>)-methylenethiazolidinone moiety with the bicyclic indole, the change on the left-hand side of the scaffold was able to dial out all seen off-targets of CVM-05-002 and resulted in a highly selective compound with a strong affinity to PI5P4Kγ (<i>S</i><sub>35</sub> = 0.02), confirmed with a <i>K</i><sub>D</sub> of 3.4 nM. The biochemical activity assays further showed lost off-target activity for ACVR2A/B and TGFBR2 of almost 2-log-fold changes or more. Likewise, the lipid kinase off-target PI4P5Kγ activity dropped by 4-fold. In contrast to that, even though we were able to gain selectivity against protein kinases with the additional hinge-binding amino group in compound <b>31</b>, this inhibitor became a pan-lipid kinase binder (<i>S</i><sub>35</sub> = 0.03), with a strong affinity for PI4P5Kγ, 200-times more active on this lipid kinase than compound <b>13</b>. Interestingly, this compound exhibited the strongest <i>K</i><sub>D</sub> for PI5P4Kγ with 23 pM.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cellular Target Engagement</h3><div class="NLM_p">To determine the cellular target engagement of our inhibitors, we established a cellular thermal shift assay (CETSA) protocol, which was previously described.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> HEK 293T cells were treated with either DMSO or 10 μM of CVM-05-002 for 1 h. The collected cells were then aliquoted and treated at temperatures ranging from 40 to 67 °C. Finally, the treated samples were lysed and the soluble protein fraction was analyzed <i>via</i> Western blot. A representative Western blot is shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, indicating that 49 °C is a suitable temperature for further isothermal CETSA experiments since the inhibitor-treated cells show a strong signal for both PI5P4Kα and -β, while the DMSO-treated cells show a visible loss in soluble PI5P4K kinases in comparison. This temperature was, hence, selected for the following isothermal CETSA experiments.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Western blot of the melting curve determination of PI5P4Kα and -β. HEK 293T cells were either treated for 1 h with DMSO or 10 μM CVM-05-002. Representative Western blot of two independent repeats. (B) Structures of selected compounds of the presented SAR study and their biochemical activities. (C) Quantified relative band intensity (%) of the isothermal CETSA experiment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 10</a>). The mean of four independent experiments is shown. ** <i>p</i> < 0.01, *** <i>p</i> < 0.001 by <i>t</i>-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Summarized in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C are four independently performed isothermal CETSA experiments with the previously selected four PI5P4K inhibitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 4</a>). Compared to DMSO, all compounds, apart from compound <b>3</b>, show at least a slight stabilization of both PI5P4K isoforms in HEK 293T cells with increasing compound concentration. Although CVM-05-002 shows a visible stabilization of PI5P4Ks, the results are not statistically significant, which is in accordance with the compound’s low permeability in a PAMPA assay setup (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 4</a>). This was to be expected due to the compound’s low log <i>D</i> value of 0.66. It should, however, be noted, that the CETSA assay results were rather noisy, leading to relatively high error bars, as seen in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, which in turn makes it difficult to get statistically significant results, even though the increase in soluble protein is visible in the Western blot, indicating some target engagement in HEK 293T cells. This lies in the nature of the assay itself since the outcome can be influenced by variations in the cell density of each repeat or within well plates, the duration of heating and cooling periods, and other factors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Nonetheless, the CETSA assay remains a powerful tool to confirm the overall target engagement in cells.</div><div class="NLM_p">As expected, compound <b>13</b> showed excellent permeability in the PAMPA assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 4</a>), which we were able to confirm in HEK 293T cells. In contrast to CMV-05-002 and compound <b>3</b>, compound <b>13</b> was able to cause statistically significant stabilization of soluble PI5P4Kβ (<i>p</i> < 0.01 by <i>t</i>-test). The visible but only slight effect of compound <b>13</b> on PI5P4Kα underlines that the biochemical PI5P4Kα and -β assays are not necessarily translating in cells since the IC<sub>50</sub> values for PI5P4Kα were generally lower than those for the β-isoform. That said, compound <b>13</b> seemed to have a better stabilizing effect on PI5P4Kβ in HEK 293T cells.</div><div class="NLM_p last">Somewhat surprisingly, compound <b>31</b>, only differing from CVM-05-002 in an additional hinge-binding amino group, showed strong cellular stabilizing effects of PI5P4Kα (<i>p</i> < 0.001 by <i>t</i>-test). This was unexpected since the compound is even more polar than the initial hit compound and is consequently showing similarly bad permeability properties as CVM-05-002 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 4</a>). Since all compounds tested showed good microsome stability (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Table 4</a>) and PAMPA assays do not include active cellular uptake or excretion, nor do we know the effects of the off-target binding of the inhibitors on the PI5P4K activity in cells, it is hard to explain this strong cellular effect of the compound. It should also be noted that PI5P4Ks are able to heterodimerize,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> meaning the stabilization of PI5P4Kγ may stabilize PI5P4Kα through protein–protein interaction-mediated mechanism, further complicating the detailed interpretation of the CETSA results. We therefore focused on the general activity of the compounds in cells as a measure of the ability to function as a probe in the cellular context to investigate PI5P4Ks.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07331" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07331" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this study, we have discovered a novel pan-PI5P4K inhibitor, CVM-05-002, with low- to sub-micromolar potency based on a thiazolidinone scaffold, using high-throughput screening methods of our in-house kinase inhibitor library. Even though the rhodanine-like scaffold can exhibit PAINS behavior, we were able to show that the identified hit compound is nonreactive, and a highly potent and selective ATP-site-directed inhibitor of the PI5P4K kinases in biochemical and cellular assays. A cocrystal structure of the initial hit with PI5P4Kα was used for the structure-guided elucidation of the structure–activity relationships to design further analogues based on this newly discovered PI5P4K-binding scaffold and explore its SAR. As a result, we were able to design compound <b>13</b>, which lacks the thiazolidinone chemotype and is a low micromolar PI5P4K inhibitor with excellent selectivity and cellular potency, posing a useful probe to study PI5P4Ks. In summary, we present not only a valuable new scaffold of PI5P4K inhibitors but also provide another example of the significance of rhodanine-based scaffolds for medicinal chemistry campaigns, when coupled with careful validation experiments.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45121" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry Experimental</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> General Methods</h4><div class="NLM_p last">Solvents and all chemicals were purchased from commercial sources and used without further purification. Purifications of intermediates and final compounds were performed, using a preparative high-performance liquid chromatography (HPLC) with a Waters Sunfire C18 column (19 mm × 50 mm, 5 μM) and a gradient of 15–95% MeOH in water (0.05% trifluoroacetic acid (TFA)) over 22 min (28 min total run time) at a flow rate of 20 mL/min. Mass spectra data were obtained on a Waters Acquity ultrahigh-pressure liquid chromatography (UPLC), which was also used to determine the purity of final compounds and is noted for each final compound individually. Proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR spectra were recorded on a 500 MHz Bruker Avance III spectrometer (500 MHz for <sup>1</sup>H; 126 MHz for <sup>13</sup>C). Chemical shifts in ppm (δ) downfield from TMS are reported. Coupling constants (<i>J</i>) are reported in Hz, and spin multiplicities are noted as s (singlet), d (doublet), t (triplet), q (quartet), dd (doublet of doublet), or m (multiplet).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Procedure A: Suzuki Coupling</h4><div class="NLM_p last">Boronic acids or respective boronic acid esters (1.1 equiv), bromobenzenes (1.0 equiv), and Cs<sub>2</sub>CO<sub>3</sub> (5.0 equiv) were dissolved in a mixture of 1,4-dioxane and water (9:1). The mixture was purged with N<sub>2</sub> for about 10 min. Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (0.1 equiv) was added, and the reaction was stirred at 100 °C for 1.5 h. The resulting suspension was filtered through a pad of celite, and the solvent was removed <i>in vacuo</i>. The crude was further purified by preparative HPLC to yield the desired product.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure B: Knoevenagel Condensation</h4><div class="NLM_p">To a solution of thiazolidinone derivative (2.0 equiv) and aldehyde (1.0 equiv) in AcOH was added β-alanine (3.0 equiv). The resulting mixture was stirred at 100 °C for 30 min. After conversion, the precipitated product was filtered and purified by preparative HPLC to yield the desired product.</div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> <i>N</i>-(4-(5-Formylpyridin-3-yl)phenyl)methanesulfonamide (<b>2a</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>1a</b>) and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide to yield a yellow-brown oil (84%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>N<sub>2</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 277.06; found 277.26.</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> <i>N</i>-(4-(5-Bromopyridin-3-yl)phenyl)methanesulfonamide (<b>2b</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 3,5-dibromopyridine (<b>1b</b>) and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide to yield a light-ochre solid (55%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>11</sub>BrN<sub>2</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 326.98; found 327.11.</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> <i>N</i>-(4-(5-Aminopyridin-3-yl)phenyl)methanesulfonamide (<b>2c</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromopyridin-3-amine (<b>1c</b>) and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide to yield a white solid (206 mg, 100% = 187 mg). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>13</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 264.08; found 264.27.</div></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> (<i>Z</i>)-<i>N-</i>(4-(5-((2-Imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)methanesulfonamide (CVM-05-002)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>2a</b> and 2-iminothiazolidin-4-one to yield a light-yellow solid (33%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>N<sub>4</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup> 375.06; found 375.20. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.98 (s, 1H), 9.57 (s, 1H), 9.24 (s, 1H), 8.88 (s, 1H), 8.77 (s, 1H), 8.15 (s, 1H), 7.76 (d, <i>J</i> = 8.5 Hz, 2H), 7.70 (s, 1H), 7.35 (d, <i>J</i> = 8.5 Hz, 2H), 3.06 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 175.67, 149.25, 147.98, 139.45, 135.59, 133.82, 132.58, 132.02, 130.84, 128.45, 125.93, 120.41, 40.92; purity 100%.</div></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> (<i>Z</i>)-<i>N</i>-(4-(5-((2,4-Dioxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)methanesulfonamide (<b>3</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>2a</b> and thiazolidine-2,4-dione to yield a light-yellow solid (12%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>3</sub>O<sub>4</sub>S<sub>2</sub> [M + H]<sup>+</sup> 376.04; found 376.18. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.75 (s, 1H), 9.99 (s, 1H), 8.93 (d, <i>J</i> = 1.8 Hz, 1H), 8.77 (s, 1H), 8.16 (s, 1H), 7.90 (s, 1H), 7.77 (d, <i>J</i> = 8.5 Hz, 2H), 7.36 (d, <i>J</i> = 8.5 Hz, 2H), 3.06 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 167.96, 167.62, 149.44, 148.71, 139.54, 135.65, 134.25, 131.65, 129.93, 128.89, 128.52, 126.87, 120.32; purity 100%.</div></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> (<i>Z</i>)-<i>N</i>-(4-(5-((2,4-Dioxo-oxazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)methanesulfonamide (<b>4</b>)</h5><div class="NLM_p last">Intermediate <b>2a</b> (0.145 mmol, 1.0 equiv), oxazolidine-2,4-dione (0.174 mmol, 1.2 equiv), and piperidine (0.044 mmol, 0.3 equiv) were dissolved in EtOH (1 mL) and stirred at 100 °C overnight. The crude was purified by preparative HPLC to yield the title compound as a brown solid (4%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>3</sub>O<sub>5</sub>S [M + H]<sup>+</sup> 360.06; found 360.48. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.96 (s, 2H), 8.94 (s, 1H), 8.53 (s, 1H), 8.03 (s, 1H), 7.77 (d, <i>J</i> = 8.6 Hz, 2H), 7.33 (d, <i>J</i> = 8.6 Hz, 2H), 3.05 (s, 3H); purity 90%.</div></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> (<i>E</i>)-<i>N</i>-(4-(5-((2,5-Dioxopyrrolidin-3-ylidene)methyl)pyridin-3-yl)phenyl)methanesulfonamide (<b>5</b>)</h5><div class="NLM_p last">Intermediate <b>2a</b> (0.145 mmol, 1.0 equiv), maleimide (0.145 mmol, 1.0 equiv), and triphenylphosphine (0.145 mmol, 1.0 equiv) were dissolved in DCM (1 mL) and stirred at room temperature overnight. The crude was purified by preparative HPLC to yield the title compound as an ivory solid (4%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub>S [M + H]<sup>+</sup> 358.09; found 358.44. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.54 (s, 1H), 9.96 (s, 1H), 8.89 (s, 1H), 8.79 (s, 1H), 8.21 (s, 1H), 7.81 (d, <i>J</i> = 8.5 Hz, 2H), 7.49 (s, 1H), 7.35 (d, <i>J</i> = 8.5 Hz, 2H), 3.84 (d, <i>J</i> = 2.4 Hz, 2H), 3.05 (s, 3H); purity 98%.</div></div><div id="sec4_1_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> <i>N</i>-(4-(5-(2-Oxoindolin-4-yl)pyridin-3-yl)phenyl)methanesulfonamide (<b>6</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure A, starting from intermediate <b>2b</b> and (2-oxoindolin-4-yl)boronic acid pinacol ester to yield an orange-brown solid (1%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 380.11; found 380.22. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.53 (s, 1H), 9.94 (s, 1H), 8.89 (s, 1H), 8.76 (s, 1H), 8.20 (s, 1H), 7.82 (d, <i>J</i> = 8.6 Hz, 2H), 7.40–7.30 (m, 3H), 7.17 (d, <i>J</i> = 7.7 Hz, 1H), 6.90 (d, <i>J</i> = 7.5 Hz, 1H), 3.73 (s, 2H), 3.05 (s, 3H); purity 94%.</div></div><div id="sec4_1_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> <i>N</i>-(3-(5-(4-(Methylsulfonamido)phenyl)pyridin-3-yl)phenyl)acetamide (<b>7</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure A, starting from intermediate <b>2b</b> and (3-acetamidophenyl)boronic acid to yield a white solid (37%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 382.12; found 382.28. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.15 (s, 1H), 10.04 (s, 1H), 9.03 (s, 1H), 8.93 (s, 1H), 8.49 (s, 1H), 8.03 (s, 1H), 7.88 (d, <i>J</i> = 8.7 Hz, 2H), 7.67 (d, <i>J</i> = 8.1 Hz, 1H), 7.54 (d, <i>J</i> = 7.8 Hz, 1H), 7.47 (t, <i>J</i> = 7.9 Hz, 1H), 7.37 (d, <i>J</i> = 8.7 Hz, 2H), 3.07 (s, 3H), 2.08 (s, 3H); purity 100%.</div></div><div id="sec4_1_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> <i>N</i>-(4-(5-(4-(3-Methylureido)phenyl)pyridin-3-yl)phenyl)methanesulfonamide (<b>8</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure A, starting from intermediate <b>2b</b> and (4-(3-methylureido)phenyl)boronic acid to yield a light-yellow solid (37%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 397.13; found 397.21. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.97 (s, 1H), 8.88 (s, 1H), 8.86 (s, 1H), 8.73 (s, 1H), 8.38 (s, 1H), 7.86 (d, <i>J</i> = 8.5 Hz, 2H), 7.76 (d, <i>J</i> = 8.9 Hz, 2H), 7.56 (d, <i>J</i> = 8.9 Hz, 2H), 7.35 (d, <i>J</i> = 8.5 Hz, 2H), 6.10 (d, <i>J</i> = 4.0 Hz, 1H), 3.06 (s, 3H), 2.66 (d, <i>J</i> = 3.2 Hz, 3H); purity 100%.</div></div><div id="sec4_1_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> <i>N</i>-(3-((5-(4-(Methylsulfonamido)phenyl)pyridin-3-yl)oxy)phenyl)acetamide (<b>9</b>)</h5><div class="NLM_p last">Intermediate <b>2b</b> (0.122 mmol, 1.0 equiv), <i>N</i>-(3-hydroxyphenyl)acetamide (0.24 mmol, 2.0 equiv), K<sub>2</sub>CO<sub>3</sub> (0.24 mmol, 2.0 equiv), and CuI (0.0122 mmol, 0.1 equiv) were dissolved in DMF (0.5 mL) and stirred at 120 °C for 2 days. The crude was purified by preparative HPLC to yield the title compound as a brown solid (3%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>3</sub>O<sub>4</sub>S [M + H]<sup>+</sup> 398.12; found 398.29. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.04 (s, 1H), 9.95 (s, 1H), 7.98–7.68 (m, 4H), 7.42 (s, 1H), 7.38–7.29 (m, 5H), 6.78 (s, 1H), 3.03 (s, 3H), 2.01 (s, 3H); purity 92%.</div></div><div id="sec4_1_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> <i>N</i>-(3-((5-(4-(Methylsulfonamido)phenyl)pyridin-3-yl)amino)phenyl)acetamide (<b>10</b>)</h5><div class="NLM_p last">Intermediate <b>2c</b> (0.152 mmol, 1.0 equiv), <i>N</i>-(3-bromophenyl)acetamide (0.182 mmol, 1.2 equiv), K<sub>2</sub>CO<sub>3</sub> (0.38 mmol, 2.5 equiv), and X-Phos (0.023 mmol, 0.15 equiv) were dissolved in 1,4-dioxane (1.0 mL). The mixture was purged with N<sub>2</sub> for about 10 min, Pd<sub>2</sub>(dba)<sub>3</sub> (0.0076 mmol, 0.05 equiv) was added, and the reaction was stirred at 80 °C overnight. The crude was purified by preparative HPLC to yield the title compound as a yellow solid (7%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 397.13; found 397.54. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i>6) δ 9.96 (s, 1H), 9.92 (s, 1H), 8.71 (s, 1H), 7.78 (s, 1H), 7.71 (d, <i>J</i> = 8.5 Hz, 2H), 7.66 (s, 1H), 7.32 (d, <i>J</i> = 8.5 Hz, 2H), 7.24–7.18 (m, 1H), 7.11 (s, 1H), 7.05 (d, <i>J</i> = 8.2 Hz, 1H), 7.01 (s, 1H), 6.83 (d, <i>J</i> = 8.0 Hz, 1H), 3.05 (s, 3H), 2.04 (s, 3H); purity 100%.</div></div><div id="sec4_1_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 4-(5-Bromopyridin-3-yl)-1<i>H</i>-indole (<b>12a</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 4-bromo-1<i>H</i>-indole and (5-bromopyridin-3-yl)boronic acid (<b>11</b>) to yield a yellow-brown oil (18%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>9</sub>BrN<sub>2</sub> [M + H]<sup>+</sup> 273.00; found 273.32.</div></div><div id="sec4_1_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 5-(5-Bromopyridin-3-yl)-1<i>H</i>-indole (<b>12b</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromo-1<i>H</i>-indole and (5-bromopyridin-3-yl)boronic acid (<b>11</b>) to yield a yellow-brown oil (9%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>9</sub>BrN<sub>2</sub> [M + H]<sup>+</sup> 273.00; found 273.33.</div></div><div id="sec4_1_3_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 6-(5-Bromopyridin-3-yl)-1<i>H</i>-indole (<b>12c</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 6-bromo-1<i>H</i>-indole and (5-bromopyridin-3-yl)boronic acid (<b>11</b>) to yield a yellow-orange solid (14%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>9</sub>BrN<sub>2</sub> [M + H]<sup>+</sup> 273.00; found 273.34.</div></div><div id="sec4_1_3_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> <i>N</i>-(4-(5-(1<i>H</i>-Indol-4-yl)pyridin-3-yl)phenyl)methanesulfonamide (<b>13</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure A, starting from intermediate <b>12a</b> and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide to yield an ochre solid (19%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 364.11; found 364.14. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.39 (s, 1H), 9.97 (s, 1H), 8.94 (s, 1H), 8.90 (s, 1H), 8.35 (s, 1H), 7.85 (d, <i>J</i> = 8.9 Hz, 2H), 7.61 (d, <i>J</i> = 8.5 Hz, 1H), 7.51 (d, <i>J</i> = 7.3 Hz, 1H), 7.49–7.46 (m, 1H), 7.36 (d, <i>J</i> = 8.9 Hz, 2H), 7.30–7.23 (m, 1H), 6.61 (s, 1H), 3.06 (s, 3H); purity 94%.</div></div><div id="sec4_1_3_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>N</i>-(5-(5-(1<i>H</i>-Indol-4-yl)pyridin-3-yl)phenyl)methanesulfonamide (<b>14</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure A, starting from intermediate <b>12b</b> and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide to yield an orange solid (27%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 364.11; found 364.14. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.30 (s, 1H), 9.98 (s, 1H), 8.94 (s, 1H), 8.89 (s, 1H), 8.42 (s, 1H), 7.88 (d, <i>J</i> = 8.9 Hz, 2H), 7.84 (s, 1H), 7.70 (d, <i>J</i> = 8.2 Hz, 1H), 7.48 (d, <i>J</i> = 10.1 Hz, 1H), 7.46–7.41 (m, 1H), 7.37 (d, <i>J</i> = 8.5 Hz, 2H), 6.50 (s, 1H), 3.06 (s, 3H); purity 92%.</div></div><div id="sec4_1_3_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> <i>N</i>-(6-(5-(1<i>H</i>-Indol-4-yl)pyridin-3-yl)phenyl)methanesulfonamide (<b>15</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure A, starting from intermediate <b>12c</b> and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide to yield an off-white solid (21%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 364.11; found 364.13. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.28 (s, 1H), 9.99 (s, 1H), 8.97 (s, 1H), 8.89 (s, 1H), 8.50 (s, 1H), 8.07 (s, 1H), 7.90 (d, <i>J</i> = 8.5 Hz, 2H), 7.59 (d, <i>J</i> = 8.9 Hz, 1H), 7.55 (d, <i>J</i> = 8.2 Hz, 1H), 7.46–7.42 (m, 1H), 7.37 (d, <i>J</i> = 8.5 Hz, 2H), 6.54 (s, 1H), 3.07 (s, 3H); purity 95%.</div></div><div id="sec4_1_3_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 4-(5-Formylpyridin-3-yl)-<i>N</i>-methylbenzenesulfonamide (<b>17a</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-(<i>N</i>-methylsulfamoyl)phenyl)boronic acid to yield a colorless oil (76%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>N<sub>2</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 277.06; found 277.40.</div></div><div id="sec4_1_3_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 5-(4-(Methylsulfonyl)phenyl)nicotinaldehyde (<b>17b</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-(methylsulfonyl)phenyl)boronic acid to yield a colorless oil (74%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>11</sub>NO<sub>3</sub>S [M + H]<sup>+</sup> 262.05; found 262.38.</div></div><div id="sec4_1_3_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>N</i>-(4-(5-Formylpyridin-3-yl)phenyl)acetamide (<b>17c</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-acetamidophenyl)boronic acid to yield a yellow oil (88%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>12</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 241.10; found 241.27.</div></div><div id="sec4_1_3_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 1-(4-(5-Formylpyridin-3-yl)phenyl)-3-methylurea (<b>17d</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-(3-methylureido)phenyl)boronic acid pinacol ester to yield a yellow oil (63%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>13</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> 256.11; found 256.46.</div></div><div id="sec4_1_3_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> <i>N</i>-(3-(5-Formylpyridin-3-yl)phenyl)acetamide (<b>17e</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (3-acetamidophenyl)boronic acid to yield a light-yellow oil (71%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>12</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 241.10; found 241.27.</div></div><div id="sec4_1_3_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 5-(4-(Trifluoromethoxy)phenyl)nicotinaldehyde (<b>17f</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-(trifluoromethoxy)phenyl)boronic acid to yield a yellow oil (93%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>8</sub>F<sub>3</sub>NO<sub>2</sub> [M + H]<sup>+</sup> 268.06; found 268.38.</div></div><div id="sec4_1_3_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 5-(4-(Trifluoromethyl)phenyl)nicotinaldehyde (<b>17g</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-(trifluoromethyl)phenyl)boronic acid to yield a light-yellow oil (64%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>8</sub>F<sub>3</sub>NO [M + H]<sup>+</sup> 252.06; found 252.25.</div></div><div id="sec4_1_3_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 5-(4-Fluorophenyl)nicotinaldehyde (<b>17h</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-fluorophenyl)boronic acid to yield a light-yellow oil (86 mg, 100% = 85 mg). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>8</sub>FNO [M + H]<sup>+</sup> 202.07; found 202.18.</div></div><div id="sec4_1_3_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 5-(4-(Dimethylamino)phenyl)nicotinaldehyde (<b>17i</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-(dimethylamino)phenyl)boronic acid to yield a red-orange oil (114 mg, 100% = 92 mg). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>14</sub>N<sub>2</sub>O [M + H]<sup>+</sup> 227.12; found 227.42.</div></div><div id="sec4_1_3_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 5-(4-Methoxynaphthalen-1-yl)nicotinaldehyde (<b>17j</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 5-bromonicotinaldehyde (<b>16</b>) and (4-methoxynaphthalen-1-yl)boronic acid to yield a yellow-orange oil (71%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>13</sub>NO<sub>2</sub> [M + H]<sup>+</sup> 264.10; found 264.43.</div></div><div id="sec4_1_3_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (<i>Z</i>)-4-(5-((2-Imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)-<i>N</i>-methylbenzenesulfonamide (<b>18</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17a</b> and 2-iminothiazolidin-4-one to yield a white solid (19%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>N<sub>4</sub>O<sub>2</sub>S<sub>3</sub> [M + H]<sup>+</sup> 375.06; found 375.42. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.59 (s, 1H), 9.26 (s, 1H), 8.97 (s, 1H), 8.86 (s, 1H), 8.27 (t, <i>J</i> = 2.0 Hz, 1H), 8.02 (d, <i>J</i> = 8.5 Hz, 2H), 7.91 (d, <i>J</i> = 8.5 Hz, 2H), 7.73 (s, 1H), 7.56 (q, <i>J</i> = 4.9 Hz, 1H), 2.46 (d, <i>J</i> = 5.2 Hz, 3H); purity 98%.</div></div><div id="sec4_1_3_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (<i>Z</i>)-2-Imino-5-((5-(4-(methylsulfonyl)phenyl)pyridin-3-yl)methylene)thiazolidin-4-one (<b>19</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17b</b> and 2-iminothiazolidin-4-one to yield a white solid (9%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup> 360.05; found 360.38. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.59 (s, 1H), 9.27 (s, 1H), 8.99 (s, 1H), 8.88 (s, 1H), 8.28 (s, 1H), 8.11–8.01 (m, 4H), 7.74 (s, 1H), 3.29 (s, 3H); purity 97%.</div></div><div id="sec4_1_3_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (<i>Z</i>)-<i>N</i>-(4-(5-((2-Imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)acetamide (<b>20</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17c</b> and 2-iminothiazolidin-4-one to yield an off-white solid (4%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>4</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 339.09; found 339.18. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.11 (s, 1H), 9.57 (s, 1H), 9.23 (s, 1H), 8.88 (d, <i>J</i> = 2.2 Hz, 1H), 8.75 (d, <i>J</i> = 2.1 Hz, 1H), 8.15 (t, <i>J</i> = 2.2 Hz, 1H), 7.76–7.71 (m, 4H), 7.70 (s, 1H), 2.08 (s, 3H); purity 100%.</div></div><div id="sec4_1_3_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>Z</i>)-1-(4-(5-((2-Imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)-3-methylurea (<b>21</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17d</b> and 2-iminothiazolidin-4-one to yield an ivory solid (18%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>5</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 354.10; found 354.42. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.56 (s, 1H), 9.23 (s, 1H), 8.87 (s, 1H), 8.75–8.67 (m, 2H), 8.12 (d, <i>J</i> = 2.1 Hz, 1H), 7.70 (s, 1H), 7.65 (d, <i>J</i> = 8.8 Hz, 2H), 7.56 (d, <i>J</i> = 8.8 Hz, 2H), 6.09 (t, <i>J</i> = 4.7 Hz, 1H), 2.66 (d, <i>J</i> = 4.5 Hz, 3H); purity 100%.</div></div><div id="sec4_1_3_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (<i>Z</i>)-<i>N</i>-(3-(5-((2-Imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)acetamide (<b>22</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17e</b> and 2-iminothiazolidin-4-one to yield an off-white solid (4%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>4</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 339.09; found 339.16. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 9.58 (s, 1H), 9.26 (s, 1H), 8.82 (s, 2H), 8.09 (s, 1H), 7.95 (s, 1H), 7.72 (s, 1H), 7.66 (d, <i>J</i> = 8.5 Hz, 1H), 7.46 (t, <i>J</i> = 7.9 Hz, 1H), 7.41 (d, <i>J</i> = 7.6 Hz, 1H), 2.08 (s, 3H); purity 100%.</div></div><div id="sec4_1_3_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (<i>Z</i>)-2-Imino-5-((5-(4-(trifluoromethoxy)phenyl)pyridin-3-yl)methylene)thiazolidin-4-one (<b>23</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17f</b> and 2-iminothiazolidin-4-one to yield a light-orange solid (11%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>10</sub>F<sub>3</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 366.05; found 366.40. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 9.25 (s, 1H), 8.93 (s, 1H), 8.83 (s, 1H), 8.20 (t, <i>J</i> = 2.0 Hz, 1H), 7.95–7.88 (m, 2H), 7.72 (s, 1H), 7.54 (d, <i>J</i> = 7.9 Hz, 2H); purity 96%.</div></div><div id="sec4_1_3_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (<i>Z</i>)-2-Imino-5-((5-(4-(trifluoromethyl)phenyl)pyridin-3-yl)methylene)thiazolidin-4-one (<b>24</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17g</b> and 2-iminothiazolidin-4-one to yield a white solid (12%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>10</sub>F<sub>3</sub>N<sub>3</sub>OS [M + H]<sup>+</sup> 350.06; found 350.15. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.59 (s, 1H), 9.27 (s, 1H), 8.97 (s, 1H), 8.86 (s, 1H), 8.26 (t, <i>J</i> = 2.0 Hz, 1H), 8.02 (d, <i>J</i> = 8.2 Hz, 2H), 7.90 (d, <i>J</i> = 8.2 Hz, 2H), 7.73 (s, 1H); purity 100%.</div></div><div id="sec4_1_3_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (<i>Z</i>)-5-((5-(4-Fluorophenyl)pyridin-3-yl)methylene)-2-iminothiazolidin-4-one (<b>25</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17h</b> and 2-iminothiazolidin-4-one to yield a white solid (4%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>10</sub>FN<sub>3</sub>OS [M + H]<sup>+</sup> 300.06; found 300.15. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 9.25 (s, 1H), 8.89 (d, <i>J</i> = 2.1 Hz, 1H), 8.80 (s, 1H), 8.16 (t, <i>J</i> = 2.0 Hz, 1H), 7.88–7.77 (m, 2H), 7.71 (s, 1H), 7.39 (t, <i>J</i> = 8.9 Hz, 2H); purity 97%.</div></div><div id="sec4_1_3_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> (<i>Z</i>)-5-((5-(4-(Dimethylamino)phenyl)pyridin-3-yl)methylene)-2-iminothiazolidin-4-one (<b>26</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17i</b> and 2-iminothiazolidin-4-one to yield an orange-red solid (13%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>16</sub>N<sub>4</sub>OS [M + H]<sup>+</sup> 325.11; found 325.43. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.57 (s, 1H), 9.25 (s, 1H), 8.87 (s, 1H), 8.69 (s, 1H), 8.14 (t, <i>J</i> = 2.1 Hz, 1H), 7.70 (s, 1H), 7.64 (d, <i>J</i> = 8.8 Hz, 2H), 6.86 (d, <i>J</i> = 8.8 Hz, 2H), 2.98 (s, 6H); purity 96%.</div></div><div id="sec4_1_3_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (<i>Z</i>)-2-Imino-5-((5-(4-methoxynaphthalen-1-yl)pyridin-3-yl)methylene)thiazolidin-4-one (<b>27</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>17j</b> and 2-iminothiazolidin-4-one to yield an orange solid (21%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>15</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 362.10; found 362.44. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.54 (s, 1H), 9.18 (s, 1H), 8.89 (s, 1H), 8.68 (s, 1H), 8.33–8.24 (m, 1H), 7.99 (s, 1H), 7.79–7.75 (m, 1H), 7.75 (s, 1H), 7.61–7.56 (m, 2H), 7.50 (d, <i>J</i> = 7.9 Hz, 1H), 7.12 (d, <i>J</i> = 7.9 Hz, 1H), 4.05 (s, 3H); purity 100%.</div></div><div id="sec4_1_3_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> <i>N</i>-(4-(6-Amino-5-formylpyridin-3-yl)phenyl)methanesulfonamide (<b>29a</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 2-amino-5-bromonicotinaldehyde (<b>28a</b>) and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide (<b>1</b>) to yield a yellow oil (74%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 292.08; found 292.16.</div></div><div id="sec4_1_3_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> <i>N</i>-(4-(2-Amino-5-formylpyridin-3-yl)phenyl)methanesulfonamide (<b>29b</b>)</h5><div class="NLM_p last">The intermediate was synthesized after general procedure A, starting from 6-amino-5-bromonicotinaldehyde (<b>28b</b>) and <i>N</i>-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)methanesulfonamide (<b>1</b>) to yield a light-yellow solid (61%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 292.08; found 292.23.</div></div><div id="sec4_1_3_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (<i>Z</i>)-<i>N</i>-(4-(6-Amino-5-((2-imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)methanesulfonamide (<b>30</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>29a</b> and 2-iminothiazolidin-4-one to yield a yellow solid (22%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>15</sub>N<sub>5</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup> 390.07; found 390.24. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.85 (s, 1H), 9.36 (s, 1H), 9.11 (s, 1H), 8.59 (d, <i>J</i> = 2.3 Hz, 1H), 8.24 (dd, <i>J</i> = 7.5, 2.3 Hz, 1H), 7.67–7.53 (m, 4H), 7.34–7.27 (m, 3H), 3.03 (s, 3H); purity 86%.</div></div><div id="sec4_1_3_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (<i>Z</i>)-<i>N</i>-(4-(2-Amino-5-((2-imino-4-oxothiazolidin-5-ylidene)methyl)pyridin-3-yl)phenyl)methanesulfonamide (<b>31</b>)</h5><div class="NLM_p last">The title compound was synthesized after general procedure B, starting from intermediate <b>29b</b> and 2-iminothiazolidin-4-one to yield a yellow solid (10%). MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>15</sub>N<sub>5</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup> 390.07; found 390.23. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.96 (s, 1H), 9.39 (s, 1H), 9.03 (s, 1H), 8.26 (d, <i>J</i> = 2.1 Hz, 1H), 7.60 (d, <i>J</i> = 2.1 Hz, 1H), 7.53 (s, 1H), 7.49–7.42 (m, 2H), 7.35–7.32 (m, 2H), 7.07 (s, 2H), 3.06 (s, 3H); purity 100%.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Biochemical Experimental</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> High-Throughput Screen</h4><div class="NLM_p last">Screenings were performed according to previously described protocols.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> 1,2-dipalmitoyl-<i>sn</i>-glycero-3-phosphoserine, sodium salt (DPPS) and PI5P (Echelon Biosciences) (2:1) were dissolved in DMSO (333 mL/mg). Sixty-three microliters of DMSO was added to 1255 μL of buffer 1 (30 mM Hepes pH 7.4, 1 mM ethylene glycol-bis(β-aminoethyl ether)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetraacetic acid (EGTA), 0.1% 3-((3-cholamidopropyl) dimethylammonio)-1-propanesulfonate (CHAPS)) and 2868 μL of buffer 2 (46 mM Hepes pH 7.4, 0.1% CHAPS). This could be adjusted to sample numbers. PI5P4Kα enzyme was added to the buffer mixture at 32 nM concentration. Ten microliters was dispensed into white 384-well plates (Corning #3824), and 100 nL of compound solution (in DMSO) was transferred by a pintool (JANUS, PerkinElmer). The reaction was initiated with the addition of 5 μL of 15 μM ATP (Promega) and PI5P/DPPS (at 0.06 μg/μL and 0.12 μg/μL, respectively) in buffer 3 (20 mM Hepes pH 7.4, 60 mM MgCl<sub>2</sub>, and 0.1% CHAPS). The final concentration of DMSO in the reaction was less than 5%. The resulting mixture was incubated at room temperature in the dark for 1 h. Five microliters of ADP-Glo reagent 1 was added to stop the reaction and remove the remaining ATP. After 45 min, 10 μL of ADP-Glo reagent 2 was added and incubated for 30 min. The luminescence intensity (RLU) was then read on an EnVision 2104 Multilabel Plate Reader (PerkinElmer). IC<sub>50</sub> values were determined using GraphPad Prism 7.04 and nonlinear regression curve fit.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Protein Expression and Purification</h4><div class="NLM_p last">Recombinant PI5P4Kα, β, and γ were obtained as described before.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Mass Spectrometry</h4><div class="NLM_p last">Recombinant PI5P4Kα/β was incubated with 10 μM CVM-05-002, THZ-P1-2 (positive control), or DMSO and analyzed by LC/MS as previously described,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> except that all samples were incubated at room temperature for 2 h, and UniDec<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was used to deconvolute the protein mass spectra.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> X-ray Crystallography and Structure Determination</h4><div class="NLM_p">PI5P4Kα was cocrystallized with CVM-05-002 using similar conditions as previously described.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> A 2-fold excess of inhibitor was mixed with 500 μM protein and crystallized by sitting-drop vapor diffusion as described. The buffer contained 2 M NH<sub>4</sub> (pH 6.5). Crystals were transferred briefly into crystallization buffer, containing 25% glycerol prior to flash-freezing in liquid nitrogen. Diffraction data from complex crystals were collected at beamline 24ID of the NE-CAT at the Advanced Photon Source (Argonne National Laboratory). Data sets were integrated and scaled using XDS (Kabsch, 2010). Structures were solved by molecular replacement using the program Phaser. The ligand was positioned manually and refined using Buster and Rhofit. Iterative manual model building and refinement using Phenix and Coot led to a model with excellent statistics, including a maximum diffraction of 1.71 Å (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Data Collection and Refinement Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">PI5P4Kα/CVM-05-002</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.9792</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">45.37–1.71 (1.771–1.71)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i> 1 21 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell constants a, b, c, α, β, γ</td><td class="colsep0 rowsep0" align="left">44.23 Å, 88.58 Å, 105.78 Å, 90°, 92.91°, 90°</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total reflections</td><td class="colsep0 rowsep0" align="left">291561 (29776)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">85988 (8587)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multiplicity</td><td class="colsep0 rowsep0" align="left">3.4 (3.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">97.72 (97.59)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean <i>I</i>/σ(<i>I</i>)</td><td class="colsep0 rowsep0" align="left">12.42 (1.34)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson B-factor</td><td class="colsep0 rowsep0" align="left">32.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">0.04003 (1.022)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>meas</sub></td><td class="colsep0 rowsep0" align="left">0.04773 (1.207)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>pim</sub></td><td class="colsep0 rowsep0" align="left">0.02566 (0.6364)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC1/2</td><td class="colsep0 rowsep0" align="left">0.999 (0.754)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC*</td><td class="colsep0 rowsep0" align="left">1 (0.927)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections used in refinement</td><td class="colsep0 rowsep0" align="left">85916 (8579)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections used for <i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">4198 (366)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub></td><td class="colsep0 rowsep0" align="left">0.1767 (0.3611)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">0.2032 (0.3938)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC (work)</td><td class="colsep0 rowsep0" align="left">0.966 (0.878)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC (free)</td><td class="colsep0 rowsep0" align="left">0.947 (0.833)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of nonhydrogen atoms</td><td class="colsep0 rowsep0" align="left">5611</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">macromolecules</td><td class="colsep0 rowsep0" align="left">5203</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent</td><td class="colsep0 rowsep0" align="left">358</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein residues</td><td class="colsep0 rowsep0" align="left">628</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMS (bonds)</td><td class="colsep0 rowsep0" align="left">0.016</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMS (angles)</td><td class="colsep0 rowsep0" align="left">1.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran favored (%)</td><td class="colsep0 rowsep0" align="left">97.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran allowed (%)</td><td class="colsep0 rowsep0" align="left">2.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran outliers (%)</td><td class="colsep0 rowsep0" align="left">0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rotamer outliers (%)</td><td class="colsep0 rowsep0" align="left">0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clashscore</td><td class="colsep0 rowsep0" align="left">10.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average B-Factor</td><td class="colsep0 rowsep0" align="left">51.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">macromolecules</td><td class="colsep0 rowsep0" align="left">52.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">46.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent</td><td class="colsep0 rowsep0" align="left">49.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of TLS groups</td><td class="colsep0 rowsep0" align="left">15</td></tr></tbody></table></div></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Protein Structure Alignment</h4><div class="NLM_p last">The above-described cocrystal structure of PI5P4Kα with CVM-05-002 was aligned with the PDB structures of PI5P4Kβ (PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZZ">3WZZ</a>) and PI5P4Kγ (PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GK9">2GK9</a>.A) using the alignment function in PyMOL 2.0.5, Incentive Product Copyright Schrodinger, LLC.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Biological Evaluation</h4><div id="sec4_2_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> Materials and Methods</h5><div class="NLM_p">All compounds were initially sourced from the National Center for Advancing Translational Studies (NCATS)/National Institutes of Health (NIH). Shipped compounds from NCATS were subjected to quality control by LC/UV, LC/MS, or high-resolution mass spectrometry (HRMS), unless otherwise noted, and exhibited >90% purity by peak area or <i>m</i>/<i>z</i><sup>2</sup>.</div><div class="NLM_p last">Data analysis was performed as previously described.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> ADP-Glo Kinase Assay (PI5P4Kα)</h4><div id="sec4_2_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> GDP Transcreener Assay (PI5P4Kβ)</h5><div class="NLM_p last">Protocols for screens and counterscreens were previously reported.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Kinome Profiling</h4><div class="NLM_p last">Protocols for the KINOMEscan are available from DiscoverX. Tree<i>spot</i> images were generated, using TREE<i>spot</i> Software Tool, and reprinted with permission from KINOMEscan, a division of DiscoverX Corporation, DISCOVERX CORPORATION 2010. All compounds were tested at a concentration of 1 μM.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> IC<sub>50</sub> Evaluation of Off-Targets</h4><div class="NLM_p last">Biochemical IC<sub>50</sub>s of identified off-targets were determined using the commercially available LanthaScreen or Adapta offered by Invitrogen (Thermo Fisher Scientific).</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>K</i><sub>D</sub> Evaluation for PI5P4Kγ</h4><div class="NLM_p last">The <i>K</i><sub>D</sub> values for PI5P4Kγ binding were determined by the commercially available KdELECT, and a respective protocol is available from DiscoverX.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Cell Culture</h4><div class="NLM_p last">HEK 293T cells were cultured as recommended by American Type Culture Collection (ATCC). The cells were kept frozen, so only freshly thawed cells at early passage were used. Cells were maintained in Dulbecco’s modified Eagle’s medium (DMEM) media, which was supplemented with 10% fetal bovine serum (FBS) and 100 μL/mL penicillin–streptomycin and incubated at 37 °C and 5% CO<sub>2</sub>. No authentication was done by the authors.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Compounds and Reagents</h4><div class="NLM_p last">Antibodies purchased from Cell Signaling Technology (Hanover, MA) were anti-PI5P4Kα (5527) and anti-PI5P4Kβ (9694). Anti-GAPDH (MAB374) was obtained from EMD Millipore Corporation (Merck KGaA, Darmstadt, Germany).</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Cellular Thermal Shift Assay (CETSA)</h4><div class="NLM_p last">Obtaining the melting curves, as well as performing the isothermal CETSA experiments and Western blot analysis were reported previously.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Kinetic Solubility Assay</h4><div class="NLM_p last">Solubility of selected compounds was determined according to methods as previously published.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Rat Liver Microsome Stability Assay</h4><div class="NLM_p last">Single time point microsomal stability was determined in a 96-well HTS format. Sample preparation was automated using a Tecan EVO 200 robot. High-resolution LC–MS (Thermo ZExactive) instrument was used to measure the percentage of compound remaining after incubation using a previously described method.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Six standard controls were tested in each run: buspirone and propranolol (for short half-life), loperamide and diclofenac (for short to medium half-life), and carbamazepine and antipyrine (for long half-life). Briefly, the incubation consisted of a 0.5 mg/mL microsomal protein, a 1.0 μM drug concentration, and NADPH regeneration system (containing 0.650 mM NADP<sup>+</sup>, 1.65 mM glucose 6-phosphate, 1.65 mM MgCl<sub>2</sub>, and 0.2 unit/mL G6PDH) in 100 mM phosphate buffer at pH 7.4. The incubation was carried out at 37 °C for 15 min. The reaction was quenched by adding 555 μL of acetonitrile (∼1:2 ratio) containing 0.28 μM albendazole (internal standard). Sample acquisition and data analysis were done using a previously described method.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Parallel Artificial Membrane Permeability Assay (PAMPA)</h4><div class="NLM_p last">Compound permeability was determined as previously described.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00227" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20246" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20246" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00227?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00227</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Overview of CVM-05-002 analogs, full KINOMEscan data sets, lipophilicity and early ADME overview of CVM-05-002 and analogs, structures of reported covalent pan-PI5P4K inhibitors, plate-well series plot with mean %DMSO control, and complete results of high-throughput screen (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf">jm0c00227_si_001.pdf (2.46 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_002.csv">jm0c00227_si_002.csv (1.88 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB code for PI5P4Kα with bound CVM-05-002 is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9">6UX9</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00227" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nathanael S. Gray</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5354-7403" title="Orcid link">http://orcid.org/0000-0001-5354-7403</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#276946534f464946424b784055465e674341444e094f46555146554309424352"><span class="__cf_email__" data-cfemail="2c624d58444d424d4940734b5e4d556c484a4f4502444d5e5a4d5e4802494859">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Theresa D. Manz</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical and Medicinal Chemistry, Saarland University, 66123 Saarbruecken, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sindhu Carmen Sivakumaren</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fleur M. Ferguson</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4091-7617" title="Orcid link">http://orcid.org/0000-0003-4091-7617</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tinghu Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adam Yasgar</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7350-1402" title="Orcid link">http://orcid.org/0000-0001-7350-1402</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hyuk-Soo Seo</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0646-2102" title="Orcid link">http://orcid.org/0000-0003-0646-2102</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott B. Ficarro</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology and Blais Proteomics Center, Dana-Farber Cancer Institute, Boston, Massachusetts, 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Oncologic Pathology, Dana-Farber Cancer
Institute, 360 Longwood
Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph D. Card</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology and Blais Proteomics Center, Dana-Farber Cancer Institute, Boston, Massachusetts, 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Oncologic Pathology, Dana-Farber Cancer
Institute, 360 Longwood
Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hyeseok Shim</span> - <span class="hlFld-Affiliation affiliation">Meyer
Cancer Center, Weill Cornell Medicine and
New York Presbyterian Hospital, New York, New York 10065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chandrasekhar V. Miduturu</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Simeonov</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Min Shen</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jarrod A. Marto</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology and Blais Proteomics Center, Dana-Farber Cancer Institute, Boston, Massachusetts, 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Oncologic Pathology, Dana-Farber Cancer
Institute, 360 Longwood
Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sirano Dhe-Paganon</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Hall</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, Rockville, Maryland 20850 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lewis C. Cantley</span> - <span class="hlFld-Affiliation affiliation">Meyer
Cancer Center, Weill Cornell Medicine and
New York Presbyterian Hospital, New York, New York 10065, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>T.D.M. and S.C.S. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>T.D.M., S.C.S., T.Z., F.M.F., and N.S.G. conceived the project and designed the research strategy. S.C.S. performed the library screening and helped with planning experimental strategies. T.D.M., F.M.F., and T.Z. designed the compounds. C.V.M. synthesized the initial hit compound. T.D.M. synthesized the compounds presented. H.-S.S. and S.D.-P. generated the cocrystal structure. A.Y. and A.S. performed the biological evaluation of the compounds under the supervision of M.D.H. and M.S. J.D.C. and S.B.F. performed mass spectrometry analysis under the supervision of J.A.M. T.D.M. planned and performed the cellular assays. H.S. and L.C.C. were involved in the planning and provided expertise and feedback for the project. The manuscript was written by T.D.M. and the contributions of all authors. All authors have given approval to the final version of the manuscript. N.S.G. supervised the research.</p></li><li><span class="author-information-subsection-header">Funding</span><p>The authors would like to thank the National Institutes of Health for their generous financial support through grants R01 CA197329 (to N.S.G., subcontracted to S.D.P.), R01 NS089815, and R03 CA250020 (to J.A.M.), as well as R35 CA197588, R01 GM041890, and U54 CA210184 (to L.C.C.). This work was further supported by the Breast Cancer Research Foundation (to L.C.C.) and the NIH intramural research program (NCATS).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): N.S.G. is a founder, SAB member, and equity holder in Gatekeeper, Syros, Petra, C4, B2S, and Soltego. The Gray lab receives or has received research funding from Novartis, Takeda, Astellas, Taiho, Janssen, Kinogen, Voronoi, Her2llc, Deerfield, and Sanofi. L.C.C. is a founder and member of the Board of Directors (BOD) of Agios Pharmaceuticals and is a founder and receives research support from Petra Pharmaceuticals. These companies are developing novel therapies for cancer. J.A.M. serves on the SAB of 908 Devices. S.D.P. receives research support from Taiho Pharmaceuticals.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4252-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62210" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62210" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank M. Kostic for her help and expertise in writing the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ACVR2A</td><td class="NLM_def"><p class="first last">activin A receptor type 2A</p></td></tr><tr><td class="NLM_term">ACVR2B</td><td class="NLM_def"><p class="first last">activin A receptor type 2B</p></td></tr><tr><td class="NLM_term">ATCC</td><td class="NLM_def"><p class="first last">American Type Culture Collection</p></td></tr><tr><td class="NLM_term">calcd</td><td class="NLM_def"><p class="first last">calculated</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">CHAPS</td><td class="NLM_def"><p class="first last">3-((3-cholamidopropyl) dimethylammonio)-1-propanesulfonate</p></td></tr><tr><td class="NLM_term">CV</td><td class="NLM_def"><p class="first last">coefficient of variation</p></td></tr><tr><td class="NLM_term">DPPS</td><td class="NLM_def"><p class="first last">1,2-dipalmitoyl-<i>sn</i>-glycero-3-phosphoserine, sodium salt</p></td></tr><tr><td class="NLM_term">EGTA</td><td class="NLM_def"><p class="first last">ethylene glycol-bis(β-aminoethyl ether)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetraacetic acid</p></td></tr><tr><td class="NLM_term">Hepes</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">MEK3</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinase 3</p></td></tr><tr><td class="NLM_term">NCATS</td><td class="NLM_def"><p class="first last">National Center for Advancing Translational Studies</p></td></tr><tr><td class="NLM_term">NIH</td><td class="NLM_def"><p class="first last">National Institutes of Health</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">Pd<sub>2</sub>(dba)<sub>3</sub></td><td class="NLM_def"><p class="first last">tris(dibenzylideneacetone)dipalladium(0)</p></td></tr><tr><td class="NLM_term">PI-4,5-P<sub>2</sub></td><td class="NLM_def"><p class="first last">phosphatidylinositol 4,5-bisphosphate</p></td></tr><tr><td class="NLM_term">PI5P</td><td class="NLM_def"><p class="first last">phosphatidylinositol 5-phosphate</p></td></tr><tr><td class="NLM_term">PI5P4K</td><td class="NLM_def"><p class="first last">phosphatidylinositol 5-phosphate 4-kinases</p></td></tr><tr><td class="NLM_term">PI4P5K</td><td class="NLM_def"><p class="first last">phosphatidylinositol 4-phosphate 5-kinase</p></td></tr><tr><td class="NLM_term">S<sub>35</sub></td><td class="NLM_def"><p class="first last">selectivity score for hits with percent control <35</p></td></tr><tr><td class="NLM_term">RLU</td><td class="NLM_def"><p class="first last">relative light units</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">TGFβ</td><td class="NLM_def"><p class="first last">transforming growth factor β</p></td></tr><tr><td class="NLM_term">TGFBR2</td><td class="NLM_def"><p class="first last">transforming growth factor β receptor 2</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultrahigh-pressure liquid chromatography</p></td></tr><tr><td class="NLM_term">X-Phos</td><td class="NLM_def"><p class="first last">2-dicyclohexylphosphino-2′,4′,6′-triisopropylbiphenyl</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38505" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38505" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 41 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rameh, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolias, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckworth, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">A new pathway for synthesis for phosphatidylinositol-4,5-bisphosphate</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">192</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1038/36621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2F36621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=9367159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVOntL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=1997&pages=192-196&author=L.+E.+Ramehauthor=K.+F.+Toliasauthor=B.+C.+Duckworthauthor=L.+C.+Cantley&title=A+new+pathway+for+synthesis+for+phosphatidylinositol-4%2C5-bisphosphate&doi=10.1038%2F36621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A new pathway for synthesis of phosphatidylinositol-4,5-bisphosphate</span></div><div class="casAuthors">Rameh, Lucia E.; Tolias, Kimberley F.; Duckworth, Brian C.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">6656</span>),
    <span class="NLM_cas:pages">192-196</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Phosphatidylinositol-4,5-bisphosphate (PtdIns-4,5-P2), a key mol. in the phosphoinositide signaling pathway, was thought to be synthesized exclusively by phosphorylation of PtdIns-4-P at the D-5 position of the inositol ring.  The enzymes that produce PtdIns-4,5-P2 in vitro fall into two related subfamilies (type I and type II PtdInsP-5-OH kinases, or PIP(5)Ks) based on their enzymic properties and sequence similarities.  Here we have reinvestigated the substrate specificities of these enzymes.  As expected, the type I enzyme phosphorylates PtdIns-4-P at the D-5 position of the inositol ring.  Surprisingly, the type II enzyme, which is abundant in some tissues, phosphorylates PtdIns-5-P at the D-4 position, and thus should be considered as a 4-OH kinase, or PIP(4)K.  The earlier error in characterizing the activity of the type II enzyme is due to the presence of contaminating PtdIns-5-P in com. prepns. of PtdIns-4-P.  Although PtdIns-5-P was previously thought not to exist in vivo, we find evidence for the presence of this lipid in mammalian fibroblasts, establishing a new pathway for PtdIns-4,5-P2 synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5w0D7qVNp4rVg90H21EOLACvtfcHk0lhbfZoLVYJuDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVOntL0%253D&md5=8196f300c91a49209d7c47eaff979282</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F36621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F36621%26sid%3Dliteratum%253Aachs%26aulast%3DRameh%26aufirst%3DL.%2BE.%26aulast%3DTolias%26aufirst%3DK.%2BF.%26aulast%3DDuckworth%26aufirst%3DB.%2BC.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DA%2520new%2520pathway%2520for%2520synthesis%2520for%2520phosphatidylinositol-4%252C5-bisphosphate%26jtitle%3DNature%26date%3D1997%26volume%3D390%26spage%3D192%26epage%3D196%26doi%3D10.1038%2F36621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, B.-L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">PIP4K2A regulates intracellular cholesterol transport through modulating PI(4,5)<sub>2</sub> homeostasis</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">514</span>, <span class="refDoi"> DOI: 10.1194/jlr.M082149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1194%2Fjlr.M082149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29353240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1ags7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2018&pages=507-514&author=A.+Huauthor=X.-T.+Zhaoauthor=T.+Fuauthor=Y.+Wangauthor=Y.+Liuauthor=X.-J.+Shiauthor=J.+Luoauthor=B.-L.+Song&title=PIP4K2A+regulates+intracellular+cholesterol+transport+through+modulating+PI%284%2C5%292+homeostasis&doi=10.1194%2Fjlr.M082149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PIP4K2A regulates intracellular cholesterol transport through modulating PI(4,5)P2 homeostasis</span></div><div class="casAuthors">Hu, Ao; Zhao, Xue-Tong; Tu, Heng; Xiao, Ting; Fu, Ting; Wang, Yan; Liu, Yong; Shi, Xiong-Jie; Luo, Jie; Song, Bao-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">507-514</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The transport of LDL-derived cholesterol from lysosomes to peroxisomes is facilitated by membrane contacts formed between the lysosomal protein synaptotagmin VII and the peroxisomal lipid phosphatidylinositol 4, 5-bisphosphate [PI(4,5)P2].  Here, we used RNA interference to search for regulators of PI(4,5)P2 and to study the effects of altered PI(4,5)P2 homeostasis on cholesterol transport.  We found that knockdown of phosphatidylinositol 5-phosphate 4-kinase type-2 α (PIP4K2A) reduced peroxisomal PI(4,5)P2 levels, decreased lysosome-peroxisome membrane contacts, and increased accumulation of lysosomal cholesterol in human SV-589 fibroblasts.  Forced expression of peroxisome-localized, kinase-active PIP4K2A in the knockdown cells reduced cholesterol accumulation, and in vitro addn. of recombinant PIP4K2A restored membrane contacts.  These results suggest that PIP4K2A plays a crit. role in intracellular cholesterol transport by upregulating PI(4,5)P2 levels in the peroxisomal membrane.  Further research into PIP4K2A activity may inform future therapeutic interventions for managing lysosomal storage disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIj9z5zOM-vrVg90H21EOLACvtfcHk0lhbfZoLVYJuDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1ags7c%253D&md5=e5fe05aaa5f1015939df56bda9cb825e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M082149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M082149%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.-T.%26aulast%3DFu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DX.-J.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DB.-L.%26atitle%3DPIP4K2A%2520regulates%2520intracellular%2520cholesterol%2520transport%2520through%2520modulating%2520PI%25284%252C5%25292%2520homeostasis%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2018%26volume%3D59%26spage%3D507%26epage%3D514%26doi%3D10.1194%2Fjlr.M082149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamia, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peroni, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rameh, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Increased insulin sensitivity and reduced adiposity in phosphatidylinositol 5-phoshate 4-kinase β<sup>–/–</sup> mice</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">5080</span>– <span class="NLM_lpage">5087</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.11.5080-5087.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1128%2FMCB.24.11.5080-5087.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=15143198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVyhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=5080-5087&author=K.+A.+Lamiaauthor=O.+D.+Peroniauthor=Y.-B.+Kimauthor=L.+E.+Ramehauthor=B.+B.+Kahnauthor=L.+C.+Cantley&title=Increased+insulin+sensitivity+and+reduced+adiposity+in+phosphatidylinositol+5-phoshate+4-kinase+%CE%B2%E2%80%93%2F%E2%80%93+mice&doi=10.1128%2FMCB.24.11.5080-5087.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Increased insulin sensitivity and reduced adiposity in phosphatidylinositol 5-phosphate 4-kinase β-/- mice</span></div><div class="casAuthors">Lamia, Katja A.; Peroni, Odile D.; Kim, Young-Bum; Rameh, Lucia E.; Kahn, Barbara B.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5080-5087</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phosphorylated derivs. of the lipid phosphatidylinositol are known to play crit. roles in insulin response.  Phosphatidylinositol 5-phosphate 4-kinases convert phosphatidylinositol 5-phosphate to phosphatidylinositol 4,5-bis-phosphate.  To understand the physiol. role of these kinases, we generated mice that do not express phosphatidylinositol 5-phosphate 4-kinase β.  These mice are hypersensitive to insulin and have reduced body wts. compared to wild-type littermates.  While adult male mice lacking phosphatidylinositol 5-phosphate 4-kinase β have significantly less body fat than wild-type littermates, female mice lacking phosphatidylinositol 5-phosphate 4-kinase β have increased insulin sensitivity in the presence of normal adiposity.  Furthermore, in vivo insulin-induced activation of the protein kinase Akt is enhanced in skeletal muscle and liver from mice lacking phosphatidylinositol 5-phosphate 4-kinase β.  These results indicate that phosphatidylinositol 5-phosphate 4-kinase β plays a role in detg. insulin sensitivity and adiposity in vivo and suggest that inhibitors of this enzyme may be useful in the treatment of type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM5s3W3lwpLrVg90H21EOLACvtfcHk0ljO9l6TRkUcLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVyhtr0%253D&md5=1a53df7bd5b007d1c2b6f3ef4c691706</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.11.5080-5087.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.11.5080-5087.2004%26sid%3Dliteratum%253Aachs%26aulast%3DLamia%26aufirst%3DK.%2BA.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DKim%26aufirst%3DY.-B.%26aulast%3DRameh%26aufirst%3DL.%2BE.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DIncreased%2520insulin%2520sensitivity%2520and%2520reduced%2520adiposity%2520in%2520phosphatidylinositol%25205-phoshate%25204-kinase%2520%25CE%25B2%25E2%2580%2593%252F%25E2%2580%2593%2520mice%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D5080%26epage%3D5087%26doi%3D10.1128%2FMCB.24.11.5080-5087.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsamooj, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosch, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo-Wing, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulf, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Deletion of the gene <i>Pip4k2c</i>, a novel phosphatidylinositol kinase, results in hyperactivation of the immune system</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7596</span>– <span class="NLM_lpage">7601</span>, <span class="refDoi"> DOI: 10.1073/pnas.1600934113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1073%2Fpnas.1600934113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=27313209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC28XpvFeku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7596-7601&author=H.+Shimauthor=C.+Wuauthor=S.+Ramsamoojauthor=K.+N.+Boschauthor=Z.+Chenauthor=B.+M.+Emerlingauthor=J.+Yunauthor=H.+Liuauthor=R.+Choo-Wingauthor=Z.+Yangauthor=G.+M.+Wulfauthor=V.+K.+Kuchrooauthor=L.+C.+Cantley&title=Deletion+of+the+gene+Pip4k2c%2C+a+novel+phosphatidylinositol+kinase%2C+results+in+hyperactivation+of+the+immune+system&doi=10.1073%2Fpnas.1600934113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of the gene Pip4k2c, a novel phosphatidylinositol kinase, results in hyperactivation of the immune system</span></div><div class="casAuthors">Shim, Hyeseok; Wu, Chuan; Ramsamooj, Shivan; Bosch, Kaitlyn N.; Chen, Zuojia; Emerling, Brooke M.; Yun, Jihye; Liu, Hui; Choo-Wing, Rayman; Yang, Zhiwei; Wulf, Gerburg M.; Kuchroo, Vijay Kumar; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">7596-7601</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Type 2 phosphatidylinositol-5-phosphate 4-kinase (PI5P4K) converts phosphatidylinositol-5-phosphate to phosphatidylinositol-4,5-bisphosphate.  Mammals have three enzymes PI5P4Kα, PI5P4Kβ, and PI5P4Kγ, and these enzymes have been implicated in metabolic control, growth control, and a variety of stress responses.  Here, the authors show that mice with germline deletion of type 2 phosphatidylinositol-5-phosphate 4-kinase gamma (Pip4k2c), the gene encoding PI5P4Kγ, appear normal in regard to growth and viability but have increased inflammation and T-cell activation as they age.  Immune cell infiltrates increased in Pip4k2c-/- mouse tissues.  Also, there was an increase in proinflammatory cytokines, including IFNγ, interleukin 12, and interleukin 2 in plasma of Pip4k2c-/- mice.  Pip4k2c-/- mice had an increase in T-helper-cell populations and a decrease in regulatory T-cell populations with increased proliferation of T cells.  Interestingly, mammalian target of rapamycin complex 1 (mTORC1) signaling was hyperactivated in several tissues from Pip4k2c-/- mice and treating Pip4k2c-/- mice with rapamycin reduced the inflammatory phenotype, resulting in a decrease in mTORC1 signaling in tissues and a decrease in proinflammatory cytokines in plasma.  These results indicate that PI5P4Kγ plays a role in the regulation of the immune system via mTORC1 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKHtPtm6tGbrVg90H21EOLACvtfcHk0ljO9l6TRkUcLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpvFeku7w%253D&md5=97c294816991d9790a8241c8bac965cb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1600934113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1600934113%26sid%3Dliteratum%253Aachs%26aulast%3DShim%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DRamsamooj%26aufirst%3DS.%26aulast%3DBosch%26aufirst%3DK.%2BN.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26aulast%3DYun%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DChoo-Wing%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWulf%26aufirst%3DG.%2BM.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DDeletion%2520of%2520the%2520gene%2520Pip4k2c%252C%2520a%2520novel%2520phosphatidylinositol%2520kinase%252C%2520results%2520in%2520hyperactivation%2520of%2520the%2520immune%2520system%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7596%26epage%3D7601%26doi%3D10.1073%2Fpnas.1600934113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lundquist, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goncalves, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughran, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possik, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayaraghavan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, K. S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinkove, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divecha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ause, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span> <span> </span><span class="NLM_article-title">Phosphatidylinositol-5-phosphate 4-kinases regulate cellular lipid metabolism by facilitating autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.03.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.molcel.2018.03.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29727621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovFGiu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=531-543&author=M.+R.+Lundquistauthor=M.+D.+Goncalvesauthor=R.+M.+Loughranauthor=E.+Possikauthor=T.+Vijayaraghavanauthor=A.+Yangauthor=C.+Pauliauthor=A.+Raviauthor=A.+Vermaauthor=Z.+Yangauthor=J.+L.+Johnsonauthor=J.+C.+Y.+Wongauthor=Y.+Maauthor=K.+S.-K.+Hwangauthor=D.+Weinkoveauthor=N.+Divechaauthor=J.+M.+Asaraauthor=O.+Elementoauthor=M.+A.+Rubinauthor=A.+C.+Kimmelmanauthor=A.+auseauthor=L.+C.+Cantleyauthor=B.+M.+Emerling&title=Phosphatidylinositol-5-phosphate+4-kinases+regulate+cellular+lipid+metabolism+by+facilitating+autophagy&doi=10.1016%2Fj.molcel.2018.03.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol-5-Phosphate 4-Kinases Regulate Cellular Lipid Metabolism By Facilitating Autophagy</span></div><div class="casAuthors">Lundquist, Mark R.; Goncalves, Marcus D.; Loughran, Ryan M.; Possik, Elite; Vijayaraghavan, Tarika; Yang, Annan; Pauli, Chantal; Ravi, Archna; Verma, Akanksha; Yang, Zhiwei; Johnson, Jared L.; Wong, Jenny C. Y.; Ma, Yilun; Hwang, Katie Seo-Kyoung; Weinkove, David; Divecha, Nullin; Asara, John M.; Elemento, Olivier; Rubin, Mark A.; Kimmelman, Alec C.; Pause, Arnim; Cantley, Lewis C.; Emerling, Brooke M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">531-544.e9</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">While the majority of phosphatidylinositol-4, 5-bisphosphate (PI-4,5-P2) in mammalian cells is generated by the conversion of phosphatidylinositol-4-phosphate (PI-4-P) to PI-4,5-P2, a small fraction can be made by phosphorylating phosphatidylinositol-5-phosphate (PI-5-P).  The physiol. relevance of this second pathway is not clear.  Here, we show that deletion of the genes encoding the two most active enzymes in this pathway, Pip4k2a and Pip4k2b, in the liver of mice causes a large enrichment in lipid droplets and in autophagic vesicles during fasting.  These changes are due to a defect in the clearance of autophagosomes that halts autophagy and reduces the supply of nutrients salvaged through this pathway.  Similar defects in autophagy are seen in nutrient-starved Pip4k2a-/-Pip4k2b-/- mouse embryonic fibroblasts and in C. elegans lacking the PI5P4K ortholog.  These results suggest that this alternative pathway for PI-4,5-P2 synthesis evolved, in part, to enhance the ability of multicellular organisms to survive starvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3l-f4CGzkx7Vg90H21EOLACvtfcHk0ljO9l6TRkUcLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovFGiu7c%253D&md5=e68f21997c55e440747cfacfa03e20af</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.03.037%26sid%3Dliteratum%253Aachs%26aulast%3DLundquist%26aufirst%3DM.%2BR.%26aulast%3DGoncalves%26aufirst%3DM.%2BD.%26aulast%3DLoughran%26aufirst%3DR.%2BM.%26aulast%3DPossik%26aufirst%3DE.%26aulast%3DVijayaraghavan%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DPauli%26aufirst%3DC.%26aulast%3DRavi%26aufirst%3DA.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DJohnson%26aufirst%3DJ.%2BL.%26aulast%3DWong%26aufirst%3DJ.%2BC.%2BY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DHwang%26aufirst%3DK.%2BS.-K.%26aulast%3DWeinkove%26aufirst%3DD.%26aulast%3DDivecha%26aufirst%3DN.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3Dause%26aufirst%3DA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26atitle%3DPhosphatidylinositol-5-phosphate%25204-kinases%2520regulate%2520cellular%2520lipid%2520metabolism%2520by%2520facilitating%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D70%26spage%3D531%26epage%3D543%26doi%3D10.1016%2Fj.molcel.2018.03.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droubi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">In B cells, phosphatidylinositol 5-phosphate 4-kinase-α synthesizes PI(4,5)P<sub>2</sub> to impact mTORC2 and Akt signaling</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">10571</span>– <span class="NLM_lpage">10576</span>, <span class="refDoi"> DOI: 10.1073/pnas.1522478113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1073%2Fpnas.1522478113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=27601656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=10571-10576&author=S.+J.+Bulleyauthor=A.+Droubiauthor=J.+H.+Clarkeauthor=K.+E.+Andersonauthor=L.+R.+Stephensauthor=P.+T.+Hawkinsauthor=R.+F.+Irvine&title=In+B+cells%2C+phosphatidylinositol+5-phosphate+4-kinase-%CE%B1+synthesizes+PI%284%2C5%29P2+to+impact+mTORC2+and+Akt+signaling&doi=10.1073%2Fpnas.1522478113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">In B cells, phosphatidylinositol 5-phosphate 4-kinase-α synthesizes PI(4,5)P2 to impact mTORC2 and Akt signaling</span></div><div class="casAuthors">Bulley, Simon J.; Droubi, Alaa; Clarke, Jonathan H.; Anderson, Karen E.; Stephens, Len R.; Hawkins, Phillip T.; Irvine, Robin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10571-10576</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are enigmatic lipid kinases with physiol. functions that are incompletely understood, not the least because genetic deletion and cell transfection have led to contradictory data.  Here, we used the genetic tractability of DT40 cells to create cell lines in which endogenous PI5P4Kα was removed, either stably by genetic deletion or transiently (within 1 h) by tagging the endogenous protein genomically with the auxin degron.  In both cases, removal impacted Akt phosphorylation, and by leaving one PI5P4Kα allele present but mutating it to be kinase-dead or have PI4P 5-kinase activity, we show that all of the effects on Akt phosphorylation were dependent on the ability of PI5P4Kα to synthesize phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] rather than to remove PI5P.  Although stable removal of PI5P4Kα resulted in a pronounced decrease in Akt phosphorylation at Thr308 and Ser473, in part because of reduced plasma membrane PIP3, its acute removal led to an increase in Akt phosphorylation only at Ser473.  This process invokes activation primarily of mammalian target of rapamycin complex 2 (mTORC2), which was confirmed by increased phosphorylation of other mTORC2 substrates.  These findings establish PI5P4Kα as a kinase that synthesizes a physiol. relevant pool of PI(4,5)P2 and as a regulator of mTORC2, and show a phenomenon similar to the "butterfly effect" described for phosphatidylinositol 3-kinase Iα [Hart JR, et al. (2015) Proc Natl Acad Sci USA 112(4):1131-1136], whereby through apparently the same underlying mechanism, the removal of a protein's activity from a cell can have widely divergent effects depending on the time course of that removal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvPu7VOCVxFLVg90H21EOLACvtfcHk0lgxlEf27nr7Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsb3O&md5=1c385a91dca10e46324865d27fce88f4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1522478113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1522478113%26sid%3Dliteratum%253Aachs%26aulast%3DBulley%26aufirst%3DS.%2BJ.%26aulast%3DDroubi%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DAnderson%26aufirst%3DK.%2BE.%26aulast%3DStephens%26aufirst%3DL.%2BR.%26aulast%3DHawkins%26aufirst%3DP.%2BT.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DIn%2520B%2520cells%252C%2520phosphatidylinositol%25205-phosphate%25204-kinase-%25CE%25B1%2520synthesizes%2520PI%25284%252C5%2529P2%2520to%2520impact%2520mTORC2%2520and%2520Akt%2520signaling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D10571%26epage%3D10576%26doi%3D10.1073%2Fpnas.1522478113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keune, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divecha, N.</span></span> <span> </span><span class="NLM_article-title">PtdIns5<i>P</i> and Pin1 in oxidative stress signaling</span>. <i>Adv. Biol. Regul.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1016/j.jbior.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.jbior.2013.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=23602596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlGrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=179-189&author=W.-J.+Keuneauthor=D.+R.+Jonesauthor=N.+Divecha&title=PtdIns5P+and+Pin1+in+oxidative+stress+signaling&doi=10.1016%2Fj.jbior.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">PtdIns5P and Pin1 in oxidative stress signaling</span></div><div class="casAuthors">Keune, Willem-Jan; Jones, David R.; Divecha, Nullin</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Biological Regulation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-189</span>CODEN:
                <span class="NLM_cas:coden">ABRDE5</span>;
        ISSN:<span class="NLM_cas:issn">2212-4934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Oxidative signaling is important in cellular health, involved in aging and contributes to the development of several diseases such as cancer, neurodegeneration and diabetes.  Correct management of reactive oxygen species (ROS) prevents oxidative stress within cells and is imperative for cellular wellbeing.  A key pathway that is regulated by oxidative stress is the activation of proline-directed stress kinases (p38, JNK).  Phosphorylation induced by these kinases is often translated into cellular outcome through the recruitment of the prolyl-isomerase Pin1.  Pin1 binds to phosphorylated substrates using its WW-domain and can induce conformational changes in the target protein through its prolyl-isomerase activity.  We show that exposure of cells to UV irradn. or hydrogen peroxide (H2O2), induces the synthesis of the phosphoinositide second messenger PtdIns5P in part by inducing the interaction between phosphatidylinositol-5-phosphate 4-kinase (PIP4K) enzymes that remove PtdIns5P, with Pin1.  In response to H2O2 exposure, Murine Embryonic Fibroblasts (MEFs) derived from Pin1-/- mice showed increased cell viability and an increased abundance of PtdIns5P compared to wild-type MEFs.  Decreasing the levels of PtdIns5P in Pin1-/- MEFs decreased both their viability in response to H2O2 exposure and the expression of genes required for cellular ROS management.  The decrease in the expression of these genes manifested itself in the increased accumulation of cellular ROS.  These data strongly argue that PtdIns5P acts as a stress-induced second messenger that can calibrate how cells manage ROS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3lS32vZ0c47Vg90H21EOLACvtfcHk0lgxlEf27nr7Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGrtr0%253D&md5=1d64aef9a69ef29b45d257184dd68255</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.jbior.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbior.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DKeune%26aufirst%3DW.-J.%26aulast%3DJones%26aufirst%3DD.%2BR.%26aulast%3DDivecha%26aufirst%3DN.%26atitle%3DPtdIns5P%2520and%2520Pin1%2520in%2520oxidative%2520stress%2520signaling%26jtitle%3DAdv.%2520Biol.%2520Regul.%26date%3D2013%26volume%3D53%26spage%3D179%26epage%3D189%26doi%3D10.1016%2Fj.jbior.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ramahi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giridharan, S. S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safren, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Haro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner Gee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titus, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainc, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barmada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of PIP4Kγ amerliorates the pathological effects of mutant huntingtin protein</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>, <span class="NLM_elocation-id">e29123</span> <span class="refDoi"> DOI: 10.7554/eLife.29123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.7554%2FeLife.29123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29256861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1enu7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=I.+Al-Ramahiauthor=S.+S.+P.+Giridharanauthor=Y.-C.+Chenauthor=S.+Patnaikauthor=N.+Safrenauthor=J.+Hasegawaauthor=M.+de+Haroauthor=A.+K.+Wagner+Geeauthor=S.+A.+Titusauthor=H.+Jeongauthor=J.+Clarkeauthor=D.+Kraincauthor=W.+Zhengauthor=R.+F.+Irvineauthor=S.+Barmadaauthor=M.+Ferrerauthor=N.+Southallauthor=L.+S.+Weismanauthor=J.+Botasauthor=J.+J.+Marugan&title=Inhibition+of+PIP4K%CE%B3+amerliorates+the+pathological+effects+of+mutant+huntingtin+protein&doi=10.7554%2FeLife.29123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of PIP4Kγ ameliorates the pathological effects of mutant huntingtin protein</span></div><div class="casAuthors">Al-Ramahi, Ismael; Giridharan, Sai Srinivas Panapakkam; Chen, Yu-Chi; Patnaik, Samarjit; Safren, Nathaniel; Hasegawa, Junya; de Haro, Maria; Gee, Amanda K. Wagner; Titus, Steven A.; Jeong, Hyunkyung; Clarke, Jonathan; Krainc, Dimitri; Zheng, Wei; Irvine, Robin F.; Barmada, Sami; Ferrer, Marc; Southall, Noel; Weisman, Lois S.; Botas, Juan; Marugan, Juan Jose</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29123/1-e29123/25</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The discovery of the causative gene for Huntington's disease (HD) has promoted numerous efforts to uncover cellular pathways that lower levels of mutant huntingtin protein (mHtt) and potentially forestall the appearance of HD-related neurol. defects.  Using a cell-based model of pathogenic huntingtin expression, we identified a class of compds. that protect cells through selective inhibition of a lipid kinase, PIP4Kγ.  Pharmacol. inhibition or knock-down of PIP4Kγ modulates the equil. between phosphatidylinositide (PI) species within the cell and increases basal autophagy, reducing the total amt. of mHtt protein in human patient fibroblasts and aggregates in neurons.  In two Drosophila models of Huntington's disease, genetic knockdown of PIP4K ameliorated neuronal dysfunction and degeneration as assessed using motor performance and retinal degeneration assays resp.  Together, these results suggest that PIP4Kγ is a druggable target whose inhibition enhances productive autophagy and mHtt proteolysis, revealing a useful pharmacol. point of intervention for the treatment of Huntington's disease, and potentially for other neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDQBO77YRMvLVg90H21EOLACvtfcHk0livCAMrMYjD2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1enu7nK&md5=61a8a87888b3fbfc8a24b4d58f08527d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.7554%2FeLife.29123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.29123%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Ramahi%26aufirst%3DI.%26aulast%3DGiridharan%26aufirst%3DS.%2BS.%2BP.%26aulast%3DChen%26aufirst%3DY.-C.%26aulast%3DPatnaik%26aufirst%3DS.%26aulast%3DSafren%26aufirst%3DN.%26aulast%3DHasegawa%26aufirst%3DJ.%26aulast%3Dde%2BHaro%26aufirst%3DM.%26aulast%3DWagner%2BGee%26aufirst%3DA.%2BK.%26aulast%3DTitus%26aufirst%3DS.%2BA.%26aulast%3DJeong%26aufirst%3DH.%26aulast%3DClarke%26aufirst%3DJ.%26aulast%3DKrainc%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26aulast%3DBarmada%26aufirst%3DS.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DWeisman%26aufirst%3DL.%2BS.%26aulast%3DBotas%26aufirst%3DJ.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26atitle%3DInhibition%2520of%2520PIP4K%25CE%25B3%2520amerliorates%2520the%2520pathological%2520effects%2520of%2520mutant%2520huntingtin%2520protein%26jtitle%3DeLife%26date%3D2017%26doi%3D10.7554%2FeLife.29123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jude, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somerville, T. D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divecha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span> <span> </span><span class="NLM_article-title">A targeted knockdown screen of genes coding for phosphoinositide modulators identifies <i>PIP4K2A</i> as required for acute myeloid leukemia cell proliferation and survival</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1253</span>– <span class="NLM_lpage">1262</span>, <span class="refDoi"> DOI: 10.1038/onc.2014.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fonc.2014.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24681948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFOgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=1253-1262&author=J.+G.+Judeauthor=G.+J.+Spencerauthor=X.+Huangauthor=T.+D.+D.+Somervilleauthor=D.+R.+Jonesauthor=N.+Divechaauthor=T.+C.+P.+Somervaille&title=A+targeted+knockdown+screen+of+genes+coding+for+phosphoinositide+modulators+identifies+PIP4K2A+as+required+for+acute+myeloid+leukemia+cell+proliferation+and+survival&doi=10.1038%2Fonc.2014.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A targeted knockdown screen of genes coding for phosphoinositide modulators identifies PIP4K2A as required for acute myeloid leukemia cell proliferation and survival</span></div><div class="casAuthors">Jude, J. G.; Spencer, G. J.; Huang, X.; Somerville, T. D. D.; Jones, D. R.; Divecha, N.; Somervaille, T. C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1253-1262</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Given the importance of deregulated phosphoinositide (PI) signaling in leukemic hematopoiesis, genes coding for proteins that regulate PI metab. may have significant and as yet unappreciated roles in leukemia.  We performed a targeted knockdown (KD) screen of PI modulator genes in human acute myeloid leukemia (AML) cells and identified candidates required to sustain proliferation or prevent apoptosis.  One of these, the lipid kinase phosphatidylinositol-5-phosphate 4-kinase, type II, alpha (PIP4K2A) regulates cellular levels of phosphatidylinositol-5-phosphate (PtsIns5P) and phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2).  We found PIP4K2A to be essential for the clonogenic and leukemia-initiating potential of human AML cells, and for the clonogenic potential of murine MLL-AF9 AML cells.  Importantly, PIP4K2A is also required for the clonogenic potential of primary human AML cells.  Its KD results in accumulation of the cyclin-dependent kinase inhibitors CDKN1A and CDKN1B, G1 cell cycle arrest and apoptosis.  Both CDKN1A accumulation and apoptosis were partially dependent on activation of the mTOR pathway.  Critically, however, PIP4K2A KD in normal hematopoietic stem and progenitor cells, both murine and human, did not adversely impact either clonogenic or multilineage differentiation potential, indicating a selective dependency that we suggest may be the consequence of the regulation of different transcriptional programs in normal vs. malignant cells.  Thus, PIP4K2A is a novel candidate therapeutic target in myeloid malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1NosZXJPbErVg90H21EOLACvtfcHk0livCAMrMYjD2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFOgt70%253D&md5=28c5806e8ceddbd668bb7066ce183b36</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.77%26sid%3Dliteratum%253Aachs%26aulast%3DJude%26aufirst%3DJ.%2BG.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DSomerville%26aufirst%3DT.%2BD.%2BD.%26aulast%3DJones%26aufirst%3DD.%2BR.%26aulast%3DDivecha%26aufirst%3DN.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DA%2520targeted%2520knockdown%2520screen%2520of%2520genes%2520coding%2520for%2520phosphoinositide%2520modulators%2520identifies%2520PIP4K2A%2520as%2520required%2520for%2520acute%2520myeloid%2520leukemia%2520cell%2520proliferation%2520and%2520survival%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26spage%3D1253%26epage%3D1262%26doi%3D10.1038%2Fonc.2014.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luoh, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, R.</span></span> <span> </span><span class="NLM_article-title">Overexpression of the amplified <i>Pip</i>4<i>k</i>2β gene from 17q11-12 in breast cancer cells confers proliferation advantage</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1354</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1207251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fsj.onc.1207251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=14691457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlKhtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=1354-1363&author=S.-W.+Luohauthor=N.+Venkatesanauthor=R.+Tripathi&title=Overexpression+of+the+amplified+Pip4k2%CE%B2+gene+from+17q11-12+in+breast+cancer+cells+confers+proliferation+advantage&doi=10.1038%2Fsj.onc.1207251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of the amplified Pip4k2β gene from 17q11-12 in breast cancer cells confers proliferation advantage</span></div><div class="casAuthors">Luoh, Shiuh-Wen; Venkatesan, Natarajan; Tripathi, Reshimi</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1354-1363</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene amplification is common in solid tumors and is assocd. with adverse prognosis, disease progression, and development of drug resistance.  A small segment from chromosome 17q11-12 contg. the HER-2/Neu gene is amplified in about 25% of breast cancer.  HER-2/Neu amplification is assocd. with adverse prognosis and may predict response to chemotherapy and hormonal manipulation.  Moreover, HER-2/Neu amplification may select patients for anti-HER-2/Neu-based therapy with Herceptin.  The authors and others recently described a common sequence element from the HER-2/Neu region that was amplified in breast cancer cells.  In addn., most, if not all, of the amplified genes from this region display overexpression.  This raises the intriguing possibility that genes immediately adjacent to HER-2/Neu may influence the biol. behavior of breast cancer carrying HER-2/Neu amplification and serve as rational targets for therapy.  By extg. sequence information from public databases, the authors have constructed a contig in bacterial artificial chromosomes (BACs) that extends from HER-2/Neu to a phosphatidylinositol phosphate kinase (PIPK), Pip4k2β from 17q11-12.  Although a role of PI-3-kinase and AKT in cancer biol. has been previously described, PIPK has not been previously implicated.  The authors show that Pip4k2β, initially known as Pip5k2β, is amplified in a subset of breast cancer cell lines and primary breast cancer samples that carry HER-2/Neu amplification.  Out of eight breast cancer cell lines with HER-2/Neu amplification, three have concomitant amplification of the Pip4k2β gene - UACC-812, BT-474 and ZR-75-30.  Similarly, two out of four primary breast tumors with HER-2/Neu amplification carry Pip4k2β gene amplification.  Intriguingly, one tumor displays an increase in the gene copy no. of Pip4k2β that is significantly more than that of HER-2/Neu.  Moreover, dual color FISH reveals that amplified Pip4k2β gene may exist in a distinct structure from that of HER-2/Neu in ZR-75-30 cell line.  These studies suggest that Pip4k2β may reside on an amplification max. distinct from that of HER-2/Neu and serve as an independent target for amplification and selective retention.  Pip4k2β amplification is assocd. with overexpression at the RNA and protein level in breast cancer cell lines.  Stable expression of Pip4k2β in breast cancer cell lines with and without HER-2/Neu amplification increases cell proliferation and anchorage-independent growth.  The above observations implicate Pip4k2β in the development and/or progression of breast cancer.  The authors' study suggests that Pip4k2β may be a distinct target for gene amplification and selective retention from 17q11-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6BmkVQfDHk7Vg90H21EOLACvtfcHk0livCAMrMYjD2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlKhtbk%253D&md5=2fd95453b778fb45b899d4e049aa57d5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207251%26sid%3Dliteratum%253Aachs%26aulast%3DLuoh%26aufirst%3DS.-W.%26aulast%3DVenkatesan%26aufirst%3DN.%26aulast%3DTripathi%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520the%2520amplified%2520Pip4k2%25CE%25B2%2520gene%2520from%252017q11-12%2520in%2520breast%2520cancer%2520cells%2520confers%2520proliferation%2520advantage%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D1354%26epage%3D1363%26doi%3D10.1038%2Fsj.onc.1207251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulogiannis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukazawa, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo-Wing, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulf, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamia, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rameh, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Depletion of a putatively druggable class of phosphatidylinositol kinases inhibits growth of p53-null tumors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">844</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.09.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.cell.2013.09.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24209622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCls7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=844-857&author=B.+M.+Emerlingauthor=J.+B.+Hurovauthor=G.+Poulogiannisauthor=K.+S.+Tsukazawaauthor=R.+Choo-Wingauthor=G.+M.+Wulfauthor=E.+L.+Bellauthor=H.-S.+Shimauthor=K.+A.+Lamiaauthor=L.+E.+Ramehauthor=G.+Bellingerauthor=A.+T.+Sasakiauthor=J.+M.+Asaraauthor=X.+Yuanauthor=A.+Bullockauthor=G.+M.+DeNicolaauthor=J.+Songauthor=V.+Brownauthor=S.+Signorettiauthor=L.+C.+Cantley&title=Depletion+of+a+putatively+druggable+class+of+phosphatidylinositol+kinases+inhibits+growth+of+p53-null+tumors&doi=10.1016%2Fj.cell.2013.09.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of a Putatively Druggable Class of Phosphatidylinositol Kinases Inhibits Growth of p53-Null Tumors</span></div><div class="casAuthors">Emerling, Brooke M.; Hurov, Jonathan B.; Poulogiannis, George; Tsukazawa, Kazumi S.; Choo-Wing, Rayman; Wulf, Gerburg M.; Bell, Eric L.; Shim, Hye-Seok; Lamia, Katja A.; Rameh, Lucia E.; Bellinger, Gary; Sasaki, Atsuo T.; Asara, John M.; Yuan, Xin; Bullock, Andrea; DeNicola, Gina M.; Song, Jiaxi; Brown, Victoria; Signoretti, Sabina; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">844-857</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Here, we show that a subset of breast cancers express high levels of the type 2 phosphatidylinositol-5-phosphate 4-kinases α and/or β (PI5P4Kα and β) and provide evidence that these kinases are essential for growth in the absence of p53.  Knocking down PI5P4Kα and β in a breast cancer cell line bearing an amplification of the gene encoding PI5P4K β and deficient for p53 impaired growth on plastic and in xenografts.  This growth phenotype was accompanied by enhanced levels of reactive oxygen species (ROS) leading to senescence.  Mice with homozygous deletion of both TP53 and PIP4K2B were not viable, indicating a synthetic lethality for loss of these two genes.  Importantly however, PIP4K2A-/-, PIP4K2B+/-, and TP53-/- mice were viable and had a dramatic redn. in tumor formation compared to TP53-/- littermates.  These results indicate that inhibitors of PI5P4Ks could be effective in preventing or treating cancers with mutations in TP53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozvZfpTxgf9bVg90H21EOLACvtfcHk0lh_Wp9oA7HqIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCls7rK&md5=adf6f54d285108fdc10ab269488faf4c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.09.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.09.057%26sid%3Dliteratum%253Aachs%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26aulast%3DHurov%26aufirst%3DJ.%2BB.%26aulast%3DPoulogiannis%26aufirst%3DG.%26aulast%3DTsukazawa%26aufirst%3DK.%2BS.%26aulast%3DChoo-Wing%26aufirst%3DR.%26aulast%3DWulf%26aufirst%3DG.%2BM.%26aulast%3DBell%26aufirst%3DE.%2BL.%26aulast%3DShim%26aufirst%3DH.-S.%26aulast%3DLamia%26aufirst%3DK.%2BA.%26aulast%3DRameh%26aufirst%3DL.%2BE.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DBullock%26aufirst%3DA.%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DV.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DDepletion%2520of%2520a%2520putatively%2520druggable%2520class%2520of%2520phosphatidylinositol%2520kinases%2520inhibits%2520growth%2520of%2520p53-null%2520tumors%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D844%26epage%3D857%26doi%3D10.1016%2Fj.cell.2013.09.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pragani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span> <span> </span><span class="NLM_article-title">A Homogeneous, high-throughput assay for phosphatidylinositol 5-phosphate 4-kinase with a novel, rapid substrate preparation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e54127</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0054127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1371%2Fjournal.pone.0054127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=23326584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=M.+I.+Davisauthor=A.+T.+Sasakiauthor=M.+Shenauthor=B.+M.+Emerlingauthor=N.+Thorneauthor=S.+Michaelauthor=R.+Praganiauthor=M.+Boxerauthor=K.+Sumitaauthor=K.+Takeuchiauthor=D.+S.+Auldauthor=Z.+Liauthor=L.+C.+Cantleyauthor=A.+Simeonov&title=A+Homogeneous%2C+high-throughput+assay+for+phosphatidylinositol+5-phosphate+4-kinase+with+a+novel%2C+rapid+substrate+preparation&doi=10.1371%2Fjournal.pone.0054127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A homogeneous, high-throughput assay for phosphatidylinositol 5-phosphate 4-kinase with a novel, rapid substrate preparation</span></div><div class="casAuthors">Davis, Mindy I.; Sasaki, Atsuo T.; Shen, Min; Emerling, Brooke M.; Thorne, Natasha; Michael, Sam; Pragani, Rajan; Boxer, Matthew; Sumita, Kazutaka; Takeuchi, Koh; Auld, Douglas S.; Li, Zhuyin; Cantley, Lewis C.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e54127</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Phosphoinositide kinases regulate diverse cellular functions and are important targets for therapeutic development for diseases, such as diabetes and cancer.  Prepn. of the lipid substrate is crucial for the development of a robust and miniaturizable lipid kinase assay.  Enzymic assays for phosphoinositide kinases often use lipid substrates prepd. from lyophilized lipid prepns. by sonication, which result in variability in the liposome size from prepn. to prepn.  Herein, we report a homogeneous 1536-well luciferase-coupled bioluminescence assay for PI5P4Kα.  The substrate prepn. is novel and allows the rapid prodn. of a DMSO-contg. substrate soln. without the need for lengthy liposome prepn. protocols, thus enabling the scale-up of this traditionally difficult type of assay.  The Z'-factor value was greater than 0.7 for the PI5P4Kα assay, indicating its suitability for high-throughput screening applications.  Tyrphostin AG-82 had been identified as an inhibitor of PI5P4Kα by assessing the degree of phospho transfer of γ-32P-ATP to PI5P; its inhibitory activity against PI5P4Kα was confirmed in the present miniaturized assay.  From a pilot screen of a library of bioactive compds., another tyrphostin, I-OMe tyrphostin AG-538 (I-OMe-AG-538), was identified as an ATP-competitive inhibitor of PI5P4Kα with an IC50 of 1 μM, affirming the suitability of the assay for inhibitor discovery campaigns.  This homogeneous assay may apply to other lipid kinases and should help in the identification of leads for this class of enzymes by enabling high-throughput screening efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomhA8K8BJDALVg90H21EOLACvtfcHk0lh_Wp9oA7HqIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCqsbo%253D&md5=254e360d68a54fbd0aa3f61b0e41ef0d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0054127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0054127%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMichael%26aufirst%3DS.%26aulast%3DPragani%26aufirst%3DR.%26aulast%3DBoxer%26aufirst%3DM.%26aulast%3DSumita%26aufirst%3DK.%26aulast%3DTakeuchi%26aufirst%3DK.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DA%2520Homogeneous%252C%2520high-throughput%2520assay%2520for%2520phosphatidylinositol%25205-phosphate%25204-kinase%2520with%2520a%2520novel%252C%2520rapid%2520substrate%2520preparation%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0054127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, P.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampetrean, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingyun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabhu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diam, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span> <span> </span><span class="NLM_article-title">Dual blockade of the lipid kinase PIP4Ks and mitotic pathways lead to cancer-selective lethality</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">2200</span> <span class="refDoi"> DOI: 10.1038/s41467-017-02287-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fs41467-017-02287-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29259156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzjtlaluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=M.+Kitagawaauthor=P.-J.+Liaoauthor=K.+H.+Leeauthor=J.+Wongauthor=S.+C.+Shangauthor=N.+Minamiauthor=O.+Sampetreanauthor=H.+Sayaauthor=D.+Lingyunauthor=N.+Prabhuauthor=G.+K.+Diamauthor=R.+Sobataauthor=A.+Larssonauthor=P.+Nordlungauthor=F.+McCormickauthor=S.+Ghoshauthor=D.+M.+Esteinauthor=B.+W.+Dymockauthor=S.+H.+Lee&title=Dual+blockade+of+the+lipid+kinase+PIP4Ks+and+mitotic+pathways+lead+to+cancer-selective+lethality&doi=10.1038%2Fs41467-017-02287-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Dual blockade of the lipid kinase PIP4Ks and mitotic pathways leads to cancer-selective lethality</span></div><div class="casAuthors">Kitagawa Mayumi; Liao Pei-Ju; Lee Kyung Hee; Wong Jasmine; Epstein David M; Lee Sang Hyun; Shang See Cheng; Dymock Brian W; Minami Noriaki; Sampetrean Oltea; Saya Hideyuki; Lingyun Dai; Prabhu Nayana; Diam Go Ka; Larsson Andreas; Nordlund Par; Sobota Radoslaw; Nordlund Par; Nordlund Par; McCormick Frank; Ghosh Sujoy; Ghosh Sujoy; Epstein David M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2200</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Achieving robust cancer-specific lethality is the ultimate clinical goal.  Here, we identify a compound with dual-inhibitory properties, named a131, that selectively kills cancer cells, while protecting normal cells.  Through an unbiased CETSA screen, we identify the PIP4K lipid kinases as the target of a131.  Ablation of the PIP4Ks generates a phenocopy of the pharmacological effects of PIP4K inhibition by a131.  Notably, PIP4Ks inhibition by a131 causes reversible growth arrest in normal cells by transcriptionally upregulating PIK3IP1, a suppressor of the PI3K/Akt/mTOR pathway.  Strikingly, Ras activation overrides a131-induced PIK3IP1 upregulation and activates the PI3K/Akt/mTOR pathway.  Consequently, Ras-transformed cells override a131-induced growth arrest and enter mitosis where a131's ability to de-cluster supernumerary centrosomes in cancer cells eliminates Ras-activated cells through mitotic catastrophe.  Our discovery of drugs with a dual-inhibitory mechanism provides a unique pharmacological strategy against cancer and evidence of cross-activation between the Ras/Raf/MEK/ERK and PI3K/AKT/mTOR pathways via a Ras PIK3IP1 PI3K signaling network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdeIFfZgL8lhFwe7cAcrfPfW6udTcc2eY7xR9sIKVS1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzjtlaluw%253D%253D&md5=f995efabc4f2f5eae92680ea8247d7ce</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02287-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02287-5%26sid%3Dliteratum%253Aachs%26aulast%3DKitagawa%26aufirst%3DM.%26aulast%3DLiao%26aufirst%3DP.-J.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DWong%26aufirst%3DJ.%26aulast%3DShang%26aufirst%3DS.%2BC.%26aulast%3DMinami%26aufirst%3DN.%26aulast%3DSampetrean%26aufirst%3DO.%26aulast%3DSaya%26aufirst%3DH.%26aulast%3DLingyun%26aufirst%3DD.%26aulast%3DPrabhu%26aufirst%3DN.%26aulast%3DDiam%26aufirst%3DG.%2BK.%26aulast%3DSobata%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DA.%26aulast%3DNordlung%26aufirst%3DP.%26aulast%3DMcCormick%26aufirst%3DF.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DEstein%26aufirst%3DD.%2BM.%26aulast%3DDymock%26aufirst%3DB.%2BW.%26aulast%3DLee%26aufirst%3DS.%2BH.%26atitle%3DDual%2520blockade%2520of%2520the%2520lipid%2520kinase%2520PIP4Ks%2520and%2520mitotic%2520pathways%2520lead%2520to%2520cancer-selective%2520lethality%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-02287-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Voss, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petry, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummerhop, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, H.-L.</span></span> <span> </span><span class="NLM_article-title">Discovery and pharmacological characterization of a novel small molecule inhibitor of phosphatidylinositol-5-phosphate 4-kinase, type II, beta</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">331</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2014.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bbrc.2014.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24845568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslWrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2014&pages=327-331&author=M.+D.+Vossauthor=W.+Czechtizkyauthor=Z.+Liauthor=C.+Rudolphauthor=S.+Petryauthor=H.+Brummerhopauthor=T.+Langerauthor=A.+Schifferauthor=H.-L.+Schaefer&title=Discovery+and+pharmacological+characterization+of+a+novel+small+molecule+inhibitor+of+phosphatidylinositol-5-phosphate+4-kinase%2C+type+II%2C+beta&doi=10.1016%2Fj.bbrc.2014.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and pharmacological characterization of a novel small molecule inhibitor of phosphatidylinositol-5-phosphate 4-kinase, type II, beta</span></div><div class="casAuthors">Voss, Marc D.; Czechtizky, Werngard; Li, Ziyu; Rudolph, Christine; Petry, Stefan; Brummerhop, Harm; Langer, Thomas; Schiffer, Alexander; Schaefer, Hans-Ludwig</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-331</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phosphatidylinositol-5-phosphate 4-kinase, type II, beta (PIP5K2B) is linked to the pathogenesis of obesity, insulin resistance and diabetes.  Here, we describe the identification of a novel pyrimidine-2,4-diamine PIP5K2B inhibitor, designated SAR088.  The compd. was identified by high-throughput screening and subsequently characterized in vitro and in vivo.  SAR088 showed reasonable potency, selectivity and physicochem. properties in enzymic and cellular assays.  In vivo, SAR088 lowered blood glucose levels of obese and hyperglycemic male Zucker diabetic fatty rats treated for 3 wk.  Thus, SAR088 represents the first orally available and in vivo active PIP5K2B inhibitor and provides an excellent starting point for the development of potent and selective PIP5K2B inhibitors for the treatment of insulin resistance and diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolvj65tqi667Vg90H21EOLACvtfcHk0ljuOYxC0rqsHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslWrtro%253D&md5=e9e0e60ec94e0b96daaadafbdc99072e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2014.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2014.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DVoss%26aufirst%3DM.%2BD.%26aulast%3DCzechtizky%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DPetry%26aufirst%3DS.%26aulast%3DBrummerhop%26aufirst%3DH.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DSchiffer%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DH.-L.%26atitle%3DDiscovery%2520and%2520pharmacological%2520characterization%2520of%2520a%2520novel%2520small%2520molecule%2520inhibitor%2520of%2520phosphatidylinositol-5-phosphate%25204-kinase%252C%2520type%2520II%252C%2520beta%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2014%26volume%3D449%26spage%3D327%26epage%3D331%26doi%3D10.1016%2Fj.bbrc.2014.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giudici, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">The function of phosphatidylinositol 5-phosphate 4-kinase γ (PI5P4Kγ) explored using a specific inhibitor that targets the PI5P-binding site</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">359</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1042/BJ20141333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1042%2FBJ20141333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=25495341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFGjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2015&pages=359-367&author=J.+H.+Clarkeauthor=M.-L.+Giudiciauthor=J.+E.+Burkeauthor=R.+L.+Williamsauthor=D.+J.+Maloneyauthor=J.+Maruganauthor=R.+F.+Irvine&title=The+function+of+phosphatidylinositol+5-phosphate+4-kinase+%CE%B3+%28PI5P4K%CE%B3%29+explored+using+a+specific+inhibitor+that+targets+the+PI5P-binding+site&doi=10.1042%2FBJ20141333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The function of phosphatidylinositol 5-phosphate 4-kinase γ (PI5P4Kγ) explored using a specific inhibitor that targets the PI5P-binding site</span></div><div class="casAuthors">Clarke, Jonathan H.; Giudici, Maria-Luisa; Burke, John E.; Williams, Roger L.; Maloney, David J.; Marugan, Juan; Irvine, Robin F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">359-367</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">NIH-12848 (NCGC00012848-02), a putative phosphatidylinositol 5-phosphate 4-kinase γ (PI5P4Kγ) inhibitor, was explored as a tool for investigating this enigmatic, low activity, lipid kinase.  PI5P4K assays in vitro showed that NIH-12848 inhibited PI5P4Kγ with an IC50 of approx. 1 μM but did not inhibit the α and β PI5P4K isoforms at concns. up to 100 μM.  A lack of inhibition of PI5P4Kγ ATPase activity suggested that NIH-12848 does not interact with the enzyme's ATP-binding site and direct exploration of binding using hydrogen-deuterium exchange (HDX)-MS (HDX-MS) revealed the putative PI5P-binding site of PI5P4Kγ to be the likely region of interaction.  This was confirmed by a series of mutation expts. which led to the identification of a single PI5P4Kγ amino acid residue that can be mutated to its PI5P4Ks α and β homolog to render PI5P4Kγ resistant NIH-12848 inhibition.  NIH-12848 (10 μM) was applied to cultured mouse principal kidney cortical collecting duct (mpkCCD) cells which, we show, express PI5P4Kγ that increases when the cells grow to confluence and polarize.  NIH-12848 inhibited the translocation of Na+/K+-ATPase to the plasma membrane that occurs when mpkCCD cells grow to confluence and also prevented reversibly their forming of 'domes' on the culture dish.  Both these NIH-12848-induced effects were mimicked by specific RNAi knockdown of PI5P4Kγ, but not that of PI5P4Ks α or β.  Overall, the data reveal a probable contribution of PI5P4Kγ to the development and maintenance of epithelial cell functional polarity and show that NIH-12848 is a potentially powerful tool for exploring the cell physiol. of PI5P4Ks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDqfSMRVrIlbVg90H21EOLACvtfcHk0ljuOYxC0rqsHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFGjsr0%253D&md5=18babc0ecf7b25901e75277955d0d075</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1042%2FBJ20141333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141333%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DGiudici%26aufirst%3DM.-L.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DMarugan%26aufirst%3DJ.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DThe%2520function%2520of%2520phosphatidylinositol%25205-phosphate%25204-kinase%2520%25CE%25B3%2520%2528PI5P4K%25CE%25B3%2529%2520explored%2520using%2520a%2520specific%2520inhibitor%2520that%2520targets%2520the%2520PI5P-binding%2520site%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D466%26spage%3D359%26epage%3D367%26doi%3D10.1042%2FBJ20141333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sivakumaren, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundquist, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paddock, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting the PI5P4K lipid kinase family in cancer using novel covalent inhibitors</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.chembiol.2020.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=32130941" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+C.+Sivakumarenauthor=H.+Shimauthor=T.+Zhangauthor=F.+M.+Fergusonauthor=M.+R.+Lundquistauthor=C.+M.+Browneauthor=H.-S.+Seoauthor=M.+N.+Paddockauthor=T.+D.+Manzauthor=B.+Jiangauthor=M.-F.+Haoauthor=P.+Krishnanauthor=D.+G.+Wangauthor=J.+Yangauthor=N.+P.+Kwiatkowskiauthor=S.+B.+Ficarroauthor=J.+M.+Cunninghamauthor=J.+A.+Martoauthor=S.+Dhe-Paganonauthor=L.+C.+Cantleyauthor=N.+S.+Gray&title=Targeting+the+PI5P4K+lipid+kinase+family+in+cancer+using+novel+covalent+inhibitors&doi=10.1016%2Fj.chembiol.2020.02.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DSivakumaren%26aufirst%3DS.%2BC.%26aulast%3DShim%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DLundquist%26aufirst%3DM.%2BR.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DPaddock%26aufirst%3DM.%2BN.%26aulast%3DManz%26aufirst%3DT.%2BD.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DHao%26aufirst%3DM.-F.%26aulast%3DKrishnan%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DD.%2BG.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DKwiatkowski%26aufirst%3DN.%2BP.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DCunningham%26aufirst%3DJ.%2BM.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520the%2520PI5P4K%2520lipid%2520kinase%2520family%2520in%2520cancer%2520using%2520novel%2520covalent%2520inhibitors%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26doi%3D10.1016%2Fj.chembiol.2020.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivakumaren, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure-activity relationship study of covalent pan-phosphatidylinositol 5-phosphate 4-kinase inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">346</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00402</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00402" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=346-352&author=T.+D.+Manzauthor=S.+C.+Sivakumarenauthor=A.+Yasgarauthor=M.+D.+Hallauthor=M.+I.+Davisauthor=H.-S.+Seoauthor=J.+D.+Cardauthor=H.+Shimauthor=S.+B.+Ficarroauthor=S.+Dhe-Paganonauthor=A.+T.+Sasakiauthor=M.+B.+Boxerauthor=A.+Simeonovauthor=L.+C.+Cantleyauthor=M.+Shenauthor=T.+Zhangauthor=F.+M.+Fergusonauthor=N.+S.+Gray&title=Structure-activity+relationship+study+of+covalent+pan-phosphatidylinositol+5-phosphate+4-kinase+inhibitors&doi=10.1021%2Facsmedchemlett.9b00402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship Study of Covalent Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors</span></div><div class="casAuthors">Manz, Theresa D.; Sivakumaren, Sindhu C.; Yasgar, Adam; Hall, Matthew D.; Davis, Mindy I.; Seo, Hyuk-Soo; Card, Joseph D.; Ficarro, Scott B.; Shim, Hyeseok; Marto, Jarrod A.; Dhe-Paganon, Sirano; Sasaki, Atsuo T.; Boxer, Matthew B.; Simeonov, Anton; Cantley, Lewis C.; Shen, Min; Zhang, Tinghu; Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">346-352</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are important mol. players in a variety of diseases, such as cancer.  Currently available PI5P4K inhibitors are reversible small mols., which may lack selectivity and sufficient cellular on-target activity.  In this study, we present a new class of covalent pan-PI5P4K inhibitors with potent biochem. and cellular activity.  Our designs are based on THZ-P1-2, a covalent PI5P4K inhibitor previously developed in our lab.  Here, we report further structure-guided optimization and structure-activity relationship (SAR) study of this scaffold, resulting in compd. 30(I), which retained biochem. and cellular potency, while demonstrating a significantly improved selectivity profile.  Furthermore, we confirm that the inhibitors show efficient binding affinity in the context of HEK 293T cells using isothermal CETSA methods.  Taken together, I represents a highly selective pan-PI5P4K covalent lead mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMr8VvNNJeXbVg90H21EOLACvtfcHk0lh3v1YuheD6HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7bO&md5=4fcc29df1916dac233a8f048132439be</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00402%26sid%3Dliteratum%253Aachs%26aulast%3DManz%26aufirst%3DT.%2BD.%26aulast%3DSivakumaren%26aufirst%3DS.%2BC.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DCard%26aufirst%3DJ.%2BD.%26aulast%3DShim%26aufirst%3DH.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DStructure-activity%2520relationship%2520study%2520of%2520covalent%2520pan-phosphatidylinositol%25205-phosphate%25204-kinase%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D346%26epage%3D352%26doi%3D10.1021%2Facsmedchemlett.9b00402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paweletz, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stetson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivancos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantarini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oxnard, G. R.</span></span> <span> </span><span class="NLM_article-title">Acquired <i>EGFR</i> C797S mediates resistance to AZD9291 in advanced non-small cell lung cancer harboring <i>EGFR</i> T790M</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nm.3854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fnm.3854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=25939061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=560-562&author=K.+S.+Thressauthor=C.+P.+Paweletzauthor=E.+Felipauthor=B.+C.+Choauthor=D.+Stetsonauthor=B.+Doughertyauthor=Z.+Laiauthor=A.+Markovetsauthor=A.+Vivancosauthor=Y.+Kuangauthor=D.+Ercanauthor=S.+Matthewsauthor=M.+Cantariniauthor=J.+C.+Barrettauthor=P.+A.+J%C3%A4nneauthor=G.+R.+Oxnard&title=Acquired+EGFR+C797S+mediates+resistance+to+AZD9291+in+advanced+non-small+cell+lung+cancer+harboring+EGFR+T790M&doi=10.1038%2Fnm.3854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</span></div><div class="casAuthors">Thress, Kenneth S.; Paweletz, Cloud P.; Felip, Enriqueta; Cho, Byoung Chul; Stetson, Daniel; Dougherty, Brian; Lai, Zhongwu; Markovets, Aleksandra; Vivancos, Ana; Kuang, Yanan; Ercan, Dalia; Matthews, Sarah E.; Cantarini, Mireille; Barrett, J. Carl; Janne, Pasi A.; Oxnard, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">560-562</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we studied cell-free plasma DNA (cfDNA) collected from subjects with advanced lung cancer whose tumors had developed resistance to the epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) AZD9291.  We first performed next-generation sequencing of cfDNA from seven subjects and detected an acquired EGFR C797S mutation in one; expression of this mutant EGFR construct in a cell line rendered it resistant to AZD9291.  We then performed droplet digital PCR on serial cfDNA specimens collected from 15 AZD9291-treated subjects.  All were pos. for the T790M mutation before treatment, but upon developing AZD9291 resistance three mol. subtypes emerged: six cases acquired the C797S mutation, five cases maintained the T790M mutation but did not acquire the C797S mutation and four cases lost the T790M mutation despite the presence of the underlying EGFR activating mutation.  Our findings provide insight into the diversity of mechanisms through which tumors acquire resistance to AZD9291 and highlight the need for therapies that are able to overcome resistance mediated by the EGFR C797S mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTggAW5fpGrbVg90H21EOLACvtfcHk0lhqUeCHu9eN8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ&md5=adac4cb6c7057855aff61fd62890d010</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnm.3854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3854%26sid%3Dliteratum%253Aachs%26aulast%3DThress%26aufirst%3DK.%2BS.%26aulast%3DPaweletz%26aufirst%3DC.%2BP.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DCho%26aufirst%3DB.%2BC.%26aulast%3DStetson%26aufirst%3DD.%26aulast%3DDougherty%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DMarkovets%26aufirst%3DA.%26aulast%3DVivancos%26aufirst%3DA.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DMatthews%26aufirst%3DS.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DOxnard%26aufirst%3DG.%2BR.%26atitle%3DAcquired%2520EGFR%2520C797S%2520mediates%2520resistance%2520to%2520AZD9291%2520in%2520advanced%2520non-small%2520cell%2520lung%2520cancer%2520harboring%2520EGFR%2520T790M%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D560%26epage%3D562%26doi%3D10.1038%2Fnm.3854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span> <span> </span><span class="NLM_article-title">Comment on the ecstasy and agony of assay interference compounds</span>. <i>J. Chem. Inf. Model</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2640</span>– <span class="NLM_lpage">2645</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00313</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00313" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1OisbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2640-2645&author=P.+W.+Kenny&title=Comment+on+the+ecstasy+and+agony+of+assay+interference+compounds&doi=10.1021%2Facs.jcim.7b00313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Comment on The Ecstasy and Agony of Assay Interference Compounds</span></div><div class="casAuthors">Kenny, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2640-2645</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A recent editorial (Aldrich et al, The Ecstasy and Agony of Assay Interference Compds., J. Chem. Inf. Model., 2017, 57, 387-390) is examd. critically.  When assessing assay hits from screening, it is important to draw a distinction between false positives, that have no effect on target function, and compds. that affect target function through an undesirable mechanism of action.  Observation of frequent-hitter behavior for a compd. should be regarded as circumstantial evidence, rather than definitive proof, that the compd. has interfered with assay readouts or acted through an undesirable mechanism of action.  The applicability domain of published (Baell & Holloway, J. Med. Chem. 2010, 53, 2719-2740) Pan Assay INterference compds. (PAINS) filters is limited by the narrow scope of the proprietary data used to derive them.  It is suggested that journal guidelines for authors should not prescribe, as those for Journal of Medicinal Chem. appear to do, that activity in assays reported for compds. that match PAINS filters be treated any differently to that for compds. that do not match PAINS filters.  It is argued that use of models based on proprietary data in the evaluation of manuscripts would contradict the editorial policy of any journal that deemed the use of proprietary data to be unacceptable in modeling studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcp9s-M9UjILVg90H21EOLACvtfcHk0lhqUeCHu9eN8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1OisbrF&md5=3c558e79c8b4e036206656350d72c800</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00313%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DComment%2520on%2520the%2520ecstasy%2520and%2520agony%2520of%2520assay%2520interference%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model%26date%3D2017%26volume%3D57%26spage%3D2640%26epage%3D2645%26doi%3D10.1021%2Facs.jcim.7b00313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhqUeCHu9eN8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven year itch: Pan-assay interference compounds (PAINS) in 2017 – utility and limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+year+itch%3A+Pan-assay+interference+compounds+%28PAINS%29+in+2017+%E2%80%93+utility+and+limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0ljebKIfmdMqfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520year%2520itch%253A%2520Pan-assay%2520interference%2520compounds%2520%2528PAINS%2529%2520in%25202017%2520%25E2%2580%2593%2520utility%2520and%2520limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendgen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Privileged scaffolds or promiscuous binders: A comparative study on rhodanines and related heterocycles in medicinal chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">753</span>, <span class="refDoi"> DOI: 10.1021/jm201243p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201243p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=743-753&author=T.+Mendgenauthor=C.+Steuerauthor=C.+D.+Klein&title=Privileged+scaffolds+or+promiscuous+binders%3A+A+comparative+study+on+rhodanines+and+related+heterocycles+in+medicinal+chemistry&doi=10.1021%2Fjm201243p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged Scaffolds or Promiscuous Binders: A Comparative Study on Rhodanines and Related Heterocycles in Medicinal Chemistry</span></div><div class="casAuthors">Mendgen, Thomas; Steuer, Christian; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">743-753</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Rhodanines and related five-membered heterocycles with multiple heteroatoms have recently gained a reputation of being unselective compds. that appear as "frequent hitters" in screening campaigns and therefore have little value in drug discovery.  However, this judgment appears to be based mostly on anecdotal evidence.  Having identified various rhodanines and related compds. in screening campaigns, we decided to perform a systematic study on their promiscuity.  An amt. of 163 rhodanines, hydantoins, thiohydantoins, and thiazolidinediones were synthesized and tested against several targets.  The compds. were also characterized with respect to aggregation and electrophilic reactivity, and the binding modes of rhodanines and related compds. in published x-ray cocrystal structures were analyzed.  The results indicate that the exocyclic, double bonded sulfur atom in rhodanines and thiohydantoins, in addn. to other structural features, offers a particularly high d. of interaction sites for polar interactions and hydrogen bonds.  This causes a promiscuous behavior at concns. in the "screening range" but should not be regarded as a general knockout criterion that excludes such screening hits from further development.  It is suggested that special criteria for target affinity and selectivity are applied to these classes of compds. and that their exceptional and potentially valuable biomol. binding properties are consequently exploited in a useful way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLzna8Pcz1iLVg90H21EOLACvtfcHk0ljebKIfmdMqfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7nN&md5=6a94d097f44bdbaaec9a56fd6b371521</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm201243p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201243p%26sid%3Dliteratum%253Aachs%26aulast%3DMendgen%26aufirst%3DT.%26aulast%3DSteuer%26aufirst%3DC.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DPrivileged%2520scaffolds%2520or%2520promiscuous%2520binders%253A%2520A%2520comparative%2520study%2520on%2520rhodanines%2520and%2520related%2520heterocycles%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D743%26epage%3D753%26doi%3D10.1021%2Fjm201243p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camps, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardissone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rintelen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrandi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillieron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gretener, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitte, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymann, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirillo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rommel, C.</span></span> <span> </span><span class="NLM_article-title">Blockade of PI3Kγ suppresses joint inflammation and damage in mouse models of rheumatoid arthritis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">943</span>, <span class="refDoi"> DOI: 10.1038/nm1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fnm1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=16127437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslelur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=936-943&author=M.+Campsauthor=T.+Rueckleauthor=H.+Jiauthor=V.+Ardissoneauthor=F.+Rintelenauthor=J.+Shawauthor=C.+Ferrandiauthor=C.+Chabertauthor=C.+Gillieronauthor=B.+Franconauthor=T.+Martinauthor=D.+Gretenerauthor=D.+Perrinauthor=D.+Leroyauthor=P.-A.+Vitteauthor=E.+Hirschauthor=M.+P.+Wymannauthor=R.+Cirilloauthor=M.+K.+Schwarzauthor=C.+Rommel&title=Blockade+of+PI3K%CE%B3+suppresses+joint+inflammation+and+damage+in+mouse+models+of+rheumatoid+arthritis&doi=10.1038%2Fnm1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Blockade of PI3Kγ suppresses joint inflammation and damage in mouse models of rheumatoid arthritis</span></div><div class="casAuthors">Camps, Montserrat; Rueckle, Thomas; Ji, Hong; Ardissone, Vittoria; Rintelen, Felix; Shaw, Jeffrey; Ferrandi, Chiara; Chabert, Christian; Gillieron, Corine; Francon, Bernard; Martin, Thierry; Gretener, Denise; Perrin, Dominique; Leroy, Didier; Vitte, Pierre-Alain; Hirsch, Emilio; Wymann, Matthias P.; Cirillo, Rocco; Schwarz, Matthias K.; Rommel, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">936-943</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinases (PI3K) have long been considered promising drug targets for the treatment of inflammatory and autoimmune disorders as well as cancer and cardiovascular diseases.  But the lack of specificity, isoform selectivity and poor biopharmaceutical profile of PI3K inhibitors have so far hampered rigorous disease-relevant target validation.  Here we describe the identification and development of specific, selective and orally active small-mol. inhibitors of PI3Kγ (encoded by Pik3cg).  We show that Pik3cg-/- mice are largely protected in mouse models of rheumatoid arthritis; this protection correlates with defective neutrophil migration, further validating PI3Kγ as a therapeutic target.  We also describe that oral treatment with a PI3Kγ inhibitor suppresses the progression of joint inflammation and damage in two distinct mouse models of rheumatoid arthritis, reproducing the protective effects shown by Pik3cg-/- mice.  Our results identify selective PI3Kγ inhibitors as potential therapeutic mols. for the treatment of chronic inflammatory disorders such as rheumatoid arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEyzzdnR3FXLVg90H21EOLACvtfcHk0ljebKIfmdMqfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslelur8%253D&md5=1fed7126427b519a4a457a8bf31ade7d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnm1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1284%26sid%3Dliteratum%253Aachs%26aulast%3DCamps%26aufirst%3DM.%26aulast%3DRueckle%26aufirst%3DT.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DArdissone%26aufirst%3DV.%26aulast%3DRintelen%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%26aulast%3DFerrandi%26aufirst%3DC.%26aulast%3DChabert%26aufirst%3DC.%26aulast%3DGillieron%26aufirst%3DC.%26aulast%3DFrancon%26aufirst%3DB.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DGretener%26aufirst%3DD.%26aulast%3DPerrin%26aufirst%3DD.%26aulast%3DLeroy%26aufirst%3DD.%26aulast%3DVitte%26aufirst%3DP.-A.%26aulast%3DHirsch%26aufirst%3DE.%26aulast%3DWymann%26aufirst%3DM.%2BP.%26aulast%3DCirillo%26aufirst%3DR.%26aulast%3DSchwarz%26aufirst%3DM.%2BK.%26aulast%3DRommel%26aufirst%3DC.%26atitle%3DBlockade%2520of%2520PI3K%25CE%25B3%2520suppresses%2520joint%2520inflammation%2520and%2520damage%2520in%2520mouse%2520models%2520of%2520rheumatoid%2520arthritis%26jtitle%3DNat.%2520Med.%26date%3D2005%26volume%3D11%26spage%3D936%26epage%3D943%26doi%3D10.1038%2Fnm1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span> <span> </span><span class="NLM_article-title">Darbufelone, a novel anti-inflammatory drug, induces growth inhibition of lung cancer cells both in vitro and in vivo</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">285</span>, <span class="refDoi"> DOI: 10.1007/s00280-009-1161-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1007%2Fs00280-009-1161-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=20352217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1alsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=277-285&author=X.+Yeauthor=W.+Zhouauthor=Y.+Liauthor=Y.+Sunauthor=Y.+Zhangauthor=H.+Jiauthor=Y.+Lai&title=Darbufelone%2C+a+novel+anti-inflammatory+drug%2C+induces+growth+inhibition+of+lung+cancer+cells+both+in+vitro+and+in+vivo&doi=10.1007%2Fs00280-009-1161-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Darbufelone, a novel anti-inflammatory drug, induces growth inhibition of lung cancer cells both in vitro and in vivo</span></div><div class="casAuthors">Ye, Xiaolei; Zhou, Wu; Li, Yongqi; Sun, Yihua; Zhang, Yihua; Ji, Hui; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">277-285</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Inflammation plays a crucial role in the development of lung cancer.  Accumulated studies have proved that non-steroidal anti-inflammatory drugs (NSAIDs) which block inflammation by their actions on arachidonic acid (AA) metab. have a potential role in cancer chemotherapy and chemoprevention.  The aim of our study was to investigate whether darbufelone, a novel anti-inflammatory drug, has anticancer effects in lung cancer.  Human non-small cell lung cancer cell lines were treated with darbufelone at various doses and time points for anal. of cell viability, cell cycle, and apoptosis in vitro.  The in vivo effect of darbufelone was assessed in Lewis lung carcinoma mice model.  Darbufelone inhibited the proliferation of non-small cell lung cancer cell lines in a dose-dependent manner, and induced cell cycle arrest at G0/G1 phase through up-regulation of p27 expression.  Treatment with darbufelone also induced apoptosis by activating caspase-3 and caspase-8.  Lewis lung carcinoma growth was also significantly inhibited by darbufelone treatment at daily dose of 80 mg/kg.  Taken together, these studies suggested that darbufelone, an anti-inflammation drug, might represent a novel therapeutic approach for lung cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkTuHuzaKpHbVg90H21EOLACvtfcHk0lhh6ZCcJVaSZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1alsb4%253D&md5=4a6f1df926a4ff5b2c041206c9a659bc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs00280-009-1161-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-009-1161-z%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DDarbufelone%252C%2520a%2520novel%2520anti-inflammatory%2520drug%252C%2520induces%2520growth%2520inhibition%2520of%2520lung%2520cancer%2520cells%2520both%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2010%26volume%3D66%26spage%3D277%26epage%3D285%26doi%3D10.1007%2Fs00280-009-1161-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghatak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fresco, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwald, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamy, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afrasiabi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhury, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khetmalas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padhye, S.</span></span> <span> </span><span class="NLM_article-title">Novel di-tertiary-butyl phenylhydrazones as dual cyclooxygenase-2/5-lipoxygenase inhibitors: Synthesis, COX/LOX inhibition, molecular modeling, and insights into their cytotoxicities</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmcl.2013.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24295787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWku7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=317-324&author=S.+Ghatakauthor=A.+Vyasauthor=S.+Misraauthor=P.+O%E2%80%99Brienauthor=A.+Zambreauthor=V.+M.+Frescoauthor=R.+R.+Markwaldauthor=K.+V.+Swamyauthor=Z.+Afrasiabiauthor=A.+Choudhuryauthor=M.+Khetmalasauthor=S.+Padhye&title=Novel+di-tertiary-butyl+phenylhydrazones+as+dual+cyclooxygenase-2%2F5-lipoxygenase+inhibitors%3A+Synthesis%2C+COX%2FLOX+inhibition%2C+molecular+modeling%2C+and+insights+into+their+cytotoxicities&doi=10.1016%2Fj.bmcl.2013.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Novel di-tertiary-butyl phenylhydrazones as dual cyclooxygenase-2/5-lipoxygenase inhibitors: Synthesis, COX/LOX inhibition, molecular modeling, and insights into their cytotoxicities</span></div><div class="casAuthors">Ghatak, Shibnath; Vyas, Alok; Misra, Suniti; O'Brien, Paul; Zambre, Ajit; Fresco, Victor M.; Markwald, Roger R.; Swamy, K. Venkateshwara; Afrasiabi, Zahra; Choudhury, Amitava; Khetmalas, Madhukar; Padhye, Subhash</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">317-324</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Although dual inhibition of Cyclooxygenase-2 (COX-2) and 5-Lipoxygenase (5-LOX) enzymes is highly effective than targeting COX or LOX alone, there are only a few reports of examg. such compds. in case of colorectal cancers (CRC).  In the present work we report that the novel di-tert-Bu phenol-based dual inhibitors DTPSAL, DTPBHZ, DTPINH, and DTPNHZ exhibit significant cytotoxicity against human CRC cell lines.  Mol. docking studies revealed a good fit of these compds. in the COX-2 and 5-LOX protein cavities.  The inhibitors show significant inhibition of COX-2 and 5-LOX activities and are effective against a panel of human colon cancer cell lines including HCA-7, HT-29, SW480 and intestinal Apc10.1 cells as well as the hyaluronan synthase-2 (Has2) enzyme over-expressing colon cancer cells, through inhibition of the Hyaluronan/CD44v6 cell survival pathway.  Western blot anal. and qRT-PCR analyses indicated that the di-tert-Bu phenol-based dual inhibitors reduce the expression of COX-2, 5-LOX, and CD44v6 in human colon cancer HCA-7 cells, while the combination of CD44v6shRNA and DTPSAL has an addnl. inhibitory effect on CD44v6 mRNA expression.  The synergistic inhibitory effect of Celecoxib and Licofelone on CD44v6 mRNA expression suggests that the present dual inhibitors down-regulate cyclooxygenase and lipoxygenase enzymes through CD44v6.  The compds. also exhibited enhanced antiproliferative potency compared to std. dual COX/LOX inhibitor, viz. Licofelone.  Importantly, the HA/CD44v6 antagonist CD44v6shRNA in combination with synthetic compds. had a sensitizing effect on the cancer cells which enhanced their antiproliferative potency, a finding which is crucial for the anti-proliferative potency of the novel synthetic di-tert-Bu phenol based dual COX-LOX inhibitors in colon cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKAW7Zh1L2ibVg90H21EOLACvtfcHk0lhh6ZCcJVaSZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWku7fK&md5=7a8a58a76b9ada38e7b4b8ef57c88dcd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DGhatak%26aufirst%3DS.%26aulast%3DVyas%26aufirst%3DA.%26aulast%3DMisra%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%26aulast%3DZambre%26aufirst%3DA.%26aulast%3DFresco%26aufirst%3DV.%2BM.%26aulast%3DMarkwald%26aufirst%3DR.%2BR.%26aulast%3DSwamy%26aufirst%3DK.%2BV.%26aulast%3DAfrasiabi%26aufirst%3DZ.%26aulast%3DChoudhury%26aufirst%3DA.%26aulast%3DKhetmalas%26aufirst%3DM.%26aulast%3DPadhye%26aufirst%3DS.%26atitle%3DNovel%2520di-tertiary-butyl%2520phenylhydrazones%2520as%2520dual%2520cyclooxygenase-2%252F5-lipoxygenase%2520inhibitors%253A%2520Synthesis%252C%2520COX%252FLOX%2520inhibition%252C%2520molecular%2520modeling%252C%2520and%2520insights%2520into%2520their%2520cytotoxicities%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D317%26epage%3D324%26doi%3D10.1016%2Fj.bmcl.2013.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keeton, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">4599</span>– <span class="NLM_lpage">4604</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmcl.2012.05.098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=22727640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4599-4604&author=L.+A.+Dakinauthor=M.+H.+Blockauthor=H.+Chenauthor=E.+Codeauthor=J.+E.+Dowlingauthor=X.+Fengauthor=A.+D.+Fergusonauthor=I.+Greenauthor=A.+W.+Hirdauthor=T.+Howardauthor=E.+K.+Keetonauthor=M.+L.+Lambauthor=P.+D.+Lyneauthor=H.+Pollardauthor=J.+Readauthor=A.+J.+Wuauthor=T.+Zhangauthor=X.+Zheng&title=Discovery+of+novel+benzylidene-1%2C3-thiazolidine-2%2C4-diones+as+potent+and+selective+inhibitors+of+the+PIM-1%2C+PIM-2%2C+and+PIM-3+protein+kinases&doi=10.1016%2Fj.bmcl.2012.05.098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases</span></div><div class="casAuthors">Dakin, Les A.; Block, Michael H.; Chen, Huawei; Code, Erin; Dowling, James E.; Feng, Xiaomei; Ferguson, Andrew D.; Green, Isabelle; Hird, Alexander W.; Howard, Tina; Keeton, Erika K.; Lamb, Michelle L.; Lyne, Paul D.; Pollard, Hannah; Read, Jon; Wu, Allan J.; Zhang, Tao; Zheng, Xiaolan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4599-4604</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel substituted benzylidene-1,3-thiazolidine-2,4-diones (TZDs) have been identified as potent and highly selective inhibitors of the PIM kinases.  The synthesis and SAR of these compds. are described, along with x-ray crystallog. of deriv. I, anti-proliferative, and selectivity data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLer-JRufp_rVg90H21EOLACvtfcHk0lhiQgrE8ylZig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D&md5=cd1570938d48864bb739d2ccc37888f2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.098%26sid%3Dliteratum%253Aachs%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DHird%26aufirst%3DA.%2BW.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DKeeton%26aufirst%3DE.%2BK.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520novel%2520benzylidene-1%252C3-thiazolidine-2%252C4-diones%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520PIM-1%252C%2520PIM-2%252C%2520and%2520PIM-3%2520protein%2520kinases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4599%26epage%3D4604%26doi%3D10.1016%2Fj.bmcl.2012.05.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorente, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantarjian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanathan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, E.</span></span> <span> </span><span class="NLM_article-title">Phase I studies of AZD1208, a proviral integration Moloney virus kinase inhibitor in solid and haematological cancers</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1038/s41416-018-0082-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fs41416-018-0082-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29765150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps1Ggs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1425-1433&author=J.+Cortesauthor=K.+Tamuraauthor=D.+J.+DeAngeloauthor=J.+de+Bonoauthor=D.+Lorenteauthor=M.+Mindenauthor=G.+L.+Uyauthor=H.+Kantarjianauthor=L.+S.+Chenauthor=V.+Gandhiauthor=R.+Godinauthor=K.+Keatingauthor=K.+McEachernauthor=K.+Vishwanathanauthor=J.+E.+Peaseauthor=E.+Dean&title=Phase+I+studies+of+AZD1208%2C+a+proviral+integration+Moloney+virus+kinase+inhibitor+in+solid+and+haematological+cancers&doi=10.1038%2Fs41416-018-0082-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I studies of AZD1208, a proviral integration Moloney virus kinase inhibitor in solid and haematological cancers</span></div><div class="casAuthors">Cortes, Jorge; Tamura, Kenji; De Angelo, Daniel J.; de Bono, Johann; Lorente, David; Minden, Mark; Uy, Geoffrey L.; Kantarjian, Hagop; Chen, Lisa S.; Gandhi, Varsha; Godin, Robert; Keating, Karen; McEachern, Kristen; Vishwanathan, Karthick; Pease, Janet Elizabeth; Dean, Emma</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1425-1433</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Background: Proviral integration Moloney virus (PIM) kinases (PIM1, 2 and 3) are overexpressed in several tumor types and contribute to oncogenesis.  AZD1208 is a potent ATP-competitive PIM kinase inhibitor investigated in patients with recurrent or refractory acute myeloid leukemia (AML) or advanced solid tumors.  Methods: Two dose-escalation studies were performed to evaluate the safety and tolerability, and to define the max. tolerated dose (MTD), of AZD1208 in AML and solid tumors.  Secondary objectives were to evaluate the pharmacokinetics, pharmacodynamics (PD) and preliminary efficacy of AZD1208.  Results: Sixty-seven patients received treatment: 32 in the AML study over a 120-900 mg dose range, and 25 in the solid tumor study over a 120-800 mg dose range.  Nearly all patients (98.5%) in both studies experienced adverse events, mostly gastrointestinal (92.5%).  Dose-limiting toxicities included rash, fatigue and vomiting.  AZD1208 was not tolerated at 900 mg, and the protocol-defined MTD was not confirmed.  AZD1208 increased CYP3A4 activity after multiple dosing, resulting in increased drug clearance.  There were no clin. responses; PD anal. showed biol. activity of AZD1208.  Conclusions: Despite the lack of single-agent clin. efficacy with AZD1208, PIM kinase inhibition may hold potential as an anticancer treatment, perhaps in combination with other agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkB6HSFRtJRrVg90H21EOLACvtfcHk0lhiQgrE8ylZig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps1Ggs7o%253D&md5=a899b9509f95a01fa2b225c6ece82df7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41416-018-0082-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41416-018-0082-1%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DTamura%26aufirst%3DK.%26aulast%3DDeAngelo%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BBono%26aufirst%3DJ.%26aulast%3DLorente%26aufirst%3DD.%26aulast%3DMinden%26aufirst%3DM.%26aulast%3DUy%26aufirst%3DG.%2BL.%26aulast%3DKantarjian%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DL.%2BS.%26aulast%3DGandhi%26aufirst%3DV.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DKeating%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DK.%26aulast%3DVishwanathan%26aufirst%3DK.%26aulast%3DPease%26aufirst%3DJ.%2BE.%26aulast%3DDean%26aufirst%3DE.%26atitle%3DPhase%2520I%2520studies%2520of%2520AZD1208%252C%2520a%2520proviral%2520integration%2520Moloney%2520virus%2520kinase%2520inhibitor%2520in%2520solid%2520and%2520haematological%2520cancers%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2018%26volume%3D118%26spage%3D1425%26epage%3D1433%26doi%3D10.1038%2Fs41416-018-0082-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathre, S.</span>; <span class="NLM_string-name">Reddy, K. B.</span>; <span class="NLM_string-name">Ramya, V.</span>; <span class="NLM_string-name">Krishnan, H.</span>; <span class="NLM_string-name">Ghosh, A.</span>; <span class="NLM_string-name">Raghu, P.</span></span><span> </span><span class="NLM_article-title">Functional analysis of the biochemical activity of mammalian phosphatidylinositol 5 phosphate 4-kinase enzymes</span>.  <i>Biosci Rep.</i><span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume">39</span> ()<span class="refDoi"> DOI: 10.1042/BSR20182210</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1042%2FBSR20182210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=30718367" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&author=S.+Mathre&author=K.+B.+Reddy&author=V.+Ramya&author=H.+Krishnan&author=A.+Ghosh&author=P.+Raghu&title=Functional+analysis+of+the+biochemical+activity+of+mammalian+phosphatidylinositol+5+phosphate+4-kinase+enzymes&doi=10.1042%2FBSR20182210"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBSR20182210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20182210%26sid%3Dliteratum%253Aachs%26aulast%3DMathre%26aufirst%3DS.%26atitle%3DFunctional%2520analysis%2520of%2520the%2520biochemical%2520activity%2520of%2520mammalian%2520phosphatidylinositol%25205%2520phosphate%25204-kinase%2520enzymes%26jtitle%3DBiosci%2520Rep.%26date%3D2019%26volume%3D39%26doi%3D10.1042%2FBSR20182210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emson, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">Localization of phosphatidylinositol phosphate kinase II gamma in kidney to a membrane trafficking compartment within specialized cells of the nephron</span>. <i>Am. J. Physiol.: Renal Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">F1422</span>– <span class="NLM_lpage">F1430</span>, <span class="refDoi"> DOI: 10.1152/ajprenal.90310.2008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1152%2Fajprenal.90310.2008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=18753295" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2008&pages=F1422-F1430&author=J.+H.+Clarkeauthor=P.+C.+Emsonauthor=R.+F.+Irvine&title=Localization+of+phosphatidylinositol+phosphate+kinase+II+gamma+in+kidney+to+a+membrane+trafficking+compartment+within+specialized+cells+of+the+nephron&doi=10.1152%2Fajprenal.90310.2008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1152%2Fajprenal.90310.2008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajprenal.90310.2008%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DEmson%26aufirst%3DP.%2BC.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DLocalization%2520of%2520phosphatidylinositol%2520phosphate%2520kinase%2520II%2520gamma%2520in%2520kidney%2520to%2520a%2520membrane%2520trafficking%2520compartment%2520within%2520specialized%2520cells%2520of%2520the%2520nephron%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Renal%2520Physiol.%26date%3D2008%26volume%3D295%26spage%3DF1422%26epage%3DF1430%26doi%3D10.1152%2Fajprenal.90310.2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0ljwyqHR20MOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0ljwyqHR20MOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Exploration of type II binding mode: A privileged approach for kinase inhibitor focused drug discovery?</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+A+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0ljwyqHR20MOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520A%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massagué, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis, R. R.</span></span> <span> </span><span class="NLM_article-title">The logic of TGFβ signaling</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">2811</span>– <span class="NLM_lpage">2820</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2006.04.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.febslet.2006.04.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=16678165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD28XltVKntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2006&pages=2811-2820&author=J.+Massagu%C3%A9author=R.+R.+Gomis&title=The+logic+of+TGF%CE%B2+signaling&doi=10.1016%2Fj.febslet.2006.04.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The logic of TGFβ signaling</span></div><div class="casAuthors">Massague, Joan; Gomis, Roger R.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2811-2820</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The identification of the TGFβ cytokine signaling pathway, including membrane receptor serine/threonine kinases and Smad transcription factors as their substrates, has allowed the delineation of a process for conversion of these signals into programs of gene activation and repression that underlie crit. cell fate and developmental decisions.  The deconstruction of one of these responses - the cell cycle arrest response - into its elemental mol. parts has shed light into the mechanisms used by tumors to evade surveillance and cause metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE0ooHqQCng7Vg90H21EOLACvtfcHk0lgQI63CAvwJtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltVKntrg%253D&md5=ed5bb318fe01991b98c6e04e66b365e1</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2006.04.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2006.04.033%26sid%3Dliteratum%253Aachs%26aulast%3DMassagu%25C3%25A9%26aufirst%3DJ.%26aulast%3DGomis%26aufirst%3DR.%2BR.%26atitle%3DThe%2520logic%2520of%2520TGF%25CE%25B2%2520signaling%26jtitle%3DFEBS%2520Lett.%26date%3D2006%26volume%3D580%26spage%3D2811%26epage%3D2820%26doi%3D10.1016%2Fj.febslet.2006.04.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, T.-A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ream-Robinson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carethers, J. M.</span></span> <span> </span><span class="NLM_article-title">Mutation rates of TGFBR2 and ACVR2 coding microsatellites in human cells with defective DNA mismatch repair</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e3463</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0003463</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1371%2Fjournal.pone.0003463" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=18941508" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&author=H.+Chungauthor=D.+J.+Youngauthor=C.+G.+Lopezauthor=T.-A.+T.+Leauthor=J.+K.+Leeauthor=D.+Ream-Robinsonauthor=S.+C.+Huangauthor=J.+M.+Carethers&title=Mutation+rates+of+TGFBR2+and+ACVR2+coding+microsatellites+in+human+cells+with+defective+DNA+mismatch+repair&doi=10.1371%2Fjournal.pone.0003463"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0003463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0003463%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DD.%2BJ.%26aulast%3DLopez%26aufirst%3DC.%2BG.%26aulast%3DLe%26aufirst%3DT.-A.%2BT.%26aulast%3DLee%26aufirst%3DJ.%2BK.%26aulast%3DReam-Robinson%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DS.%2BC.%26aulast%3DCarethers%26aufirst%3DJ.%2BM.%26atitle%3DMutation%2520rates%2520of%2520TGFBR2%2520and%2520ACVR2%2520coding%2520microsatellites%2520in%2520human%2520cells%2520with%2520defective%2520DNA%2520mismatch%2520repair%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26doi%3D10.1371%2Fjournal.pone.0003463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrera, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiorino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocanegra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carethers, J. M.</span></span> <span> </span><span class="NLM_article-title"><i>Activin type 2 receptor</i> restoration in MSI-H colon cancer suppresses growth and enhances migration with activin</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2006.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1053%2Fj.gastro.2006.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=17258738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlKisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2007&pages=633-644&author=B.+H.+Jungauthor=S.+E.+Beckauthor=J.+Cabralauthor=E.+Chauauthor=B.+L.+Cabreraauthor=A.+Fiorinoauthor=E.+J.+Smithauthor=M.+Bocanegraauthor=J.+M.+Carethers&title=Activin+type+2+receptor+restoration+in+MSI-H+colon+cancer+suppresses+growth+and+enhances+migration+with+activin&doi=10.1053%2Fj.gastro.2006.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activin type 2 receptor restoration in MSI-H colon cancer suppresses growth and enhances migration with activin</span></div><div class="casAuthors">Jung, Barbara H.; Beck, Stayce E.; Cabral, Jennifer; Chau, Eddy; Cabrera, Betty L.; Fiorino, Antonio; Smith, E. Julieta; Bocanegra, Melanie; Carethers, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">633-644</span>CODEN:
                <span class="NLM_cas:coden">GASTAB</span>;
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Background & Aims: Colon cancers with high-frequency microsatellite instability (MSI-H) develop frameshift mutations in tumor suppressors as part of their pathogenesis.  ACVR2 is mutated at its exon 10 polyadenine tract in >80% of MSI-H colon cancers, coinciding with loss of protein.  ACVR2 transmits the growth effects of activin via phosphorylation of SMAD proteins to affect gene transcription.  The functional effect of activin in colon cancers has not been studied.  We developed and characterized a cell model in which we studied how activin signaling affects growth.  Methods: hMLH1 and ACVR2 mutant HCT116 cells were previously stably transferred with chromosome 2 (HCT116+chr2), restoring a single regulated copy of wild-type ACVR2 but not hMLH1.  Both HCT116+chr2 and parental HCT116 cells (as well as HEC59 and ACVR2 and hMSH2 complemented HEC59+chr2 cells) were assessed for genetic complementation and biol. function.  Results: HCT116+chr2 cells and HEC59+chr2 cells, but not ACVR2-mutant HCT116 or HEC59 cells, acquired wild-type ACVR2 as well as expression of ACVR2 wild-type mRNA.  Complemented ACVR2 protein complexed with ACVR1 with activin treatment, generating nuclear phosphoSMAD2 and activin-specific gene transcription.  ACVR2-restored cells showed decreased growth and reduced S phase but increased cellular migration following activin treatment.  ACVR2 small interfering RNA reversed these effects in complemented cells.  Conclusions: ACVR2-complemented MSI-H colon cancers restore activin-SMAD signaling, decrease growth, and slow their cell cycle following ligand stimulation but show increased cellular migration.  Activin is growth suppressive and enhances migration similar to transforming growth factor β in colon cancer, indicating that abrogation of the effects of activin contribute to the pathogenesis of MSI-H colon cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2J5XsKYrCrLVg90H21EOLACvtfcHk0lgQI63CAvwJtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlKisr0%253D&md5=624248c5746fd922b0e8724f15534195</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2006.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2006.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DB.%2BH.%26aulast%3DBeck%26aufirst%3DS.%2BE.%26aulast%3DCabral%26aufirst%3DJ.%26aulast%3DChau%26aufirst%3DE.%26aulast%3DCabrera%26aufirst%3DB.%2BL.%26aulast%3DFiorino%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DE.%2BJ.%26aulast%3DBocanegra%26aufirst%3DM.%26aulast%3DCarethers%26aufirst%3DJ.%2BM.%26atitle%3DActivin%2520type%25202%2520receptor%2520restoration%2520in%2520MSI-H%2520colon%2520cancer%2520suppresses%2520growth%2520and%2520enhances%2520migration%2520with%2520activin%26jtitle%3DGastroenterology%26date%3D2007%26volume%3D132%26spage%3D633%26epage%3D644%26doi%3D10.1053%2Fj.gastro.2006.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallal, L. A.</span></span> <span> </span><span class="NLM_article-title">High-throughput cellular thermal shift assays in research and drug discovery</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1177/2472555219877183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1177%2F2472555219877183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=31566060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A280%3ADC%252BB3Mnjt1Clsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=137-147&author=M.+J.+Hendersonauthor=M.+A.+Holbertauthor=A.+Simeonovauthor=L.+A.+Kallal&title=High-throughput+cellular+thermal+shift+assays+in+research+and+drug+discovery&doi=10.1177%2F2472555219877183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Cellular Thermal Shift Assays in Research and Drug Discovery</span></div><div class="casAuthors">Henderson Mark J; Simeonov Anton; Holbert Marc A; Kallal Lorena A</div><div class="citationInfo"><span class="NLM_cas:title">SLAS discovery : advancing life sciences R & D</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">137-147</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Thermal shift assays (TSAs) can reveal changes in protein structure, due to a resultant change in protein thermal stability.  Since proteins are often stabilized upon binding of ligand molecules, these assays can provide a readout for protein target engagement.  TSA has traditionally been applied using purified proteins and more recently has been extended to study target engagement in cellular environments with the emergence of cellular thermal shift assays (CETSAs).  The utility of CETSA in confirming molecular interaction with targets in a more native context, and the desire to apply this technique more broadly, has fueled the emergence of higher-throughput techniques for CETSA (HT-CETSA).  Recent studies have demonstrated that HT-CETSA can be performed in standard 96-, 384-, and 1536-well microtiter plate formats using methods such as beta-galactosidase and NanoLuciferase reporters and AlphaLISA assays.  HT-CETSA methods can be used to select and characterize compounds from high-throughput screens and to prioritize compounds in lead optimization by facilitating dose-response experiments.  In conjunction with cellular and biochemical activity assays for targets, HT-CETSA can be a valuable addition to the suite of assays available to characterize molecules of interest.  Despite the successes in implementing HT-CETSA for a diverse set of targets, caveats and challenges must also be recognized to avoid overinterpretation of results.  Here, we review the current landscape of HT-CETSA and discuss the methodologies, practical considerations, challenges, and applications of this approach in research and drug discovery.  Additionally, a perspective on potential future directions for the technology is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEx69ZUMaiTMjl5L8hwNX0fW6udTcc2ebDM0ywJJffrbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mnjt1Clsg%253D%253D&md5=39f58608a89ce9863098582c63d0a338</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1177%2F2472555219877183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555219877183%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DHolbert%26aufirst%3DM.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DKallal%26aufirst%3DL.%2BA.%26atitle%3DHigh-throughput%2520cellular%2520thermal%2520shift%2520assays%2520in%2520research%2520and%2520drug%2520discovery%26jtitle%3DSLAS%2520Discovery%26date%3D2020%26volume%3D25%26spage%3D137%26epage%3D147%26doi%3D10.1177%2F2472555219877183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droubi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giudici, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">Exploring phosphatidylinositol 5-phosphate 4-kinase function</span>. <i>Adv. Biol. Regul.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1016/j.jbior.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.jbior.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=25311266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslansr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2015&pages=193-202&author=S.+J.+Bulleyauthor=J.+H.+Clarkeauthor=A.+Droubiauthor=M.-L.+Giudiciauthor=R.+F.+Irvine&title=Exploring+phosphatidylinositol+5-phosphate+4-kinase+function&doi=10.1016%2Fj.jbior.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring phosphatidylinositol 5-phosphate 4-kinase function</span></div><div class="casAuthors">Bulley, Simon J.; Clarke, Jonathan H.; Droubi, Alaa; Giudici, Maria-Luisa; Irvine, Robin F.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Biological Regulation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">193-202</span>CODEN:
                <span class="NLM_cas:coden">ABRDE5</span>;
        ISSN:<span class="NLM_cas:issn">2212-4934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The family of phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) is emerging from a comparative backwater in inositide signalling into the mainstream, as is their substrate, phosphatidylinositol 5-phosphate (PI5P).  Here we review some of the key questions about the PI5P4Ks, their localization, interaction, and regulation and also we summarize our current understanding of how PI5P is synthesized and what its cellular functions might be.  Finally, some of the evidence for the involvement of PI5P4Ks in pathol. is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCTy9MhIW4aLVg90H21EOLACvtfcHk0lg-rzEdmUlUrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslansr7O&md5=af7bd2595850a46a98019a0d6cd8b5ab</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jbior.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbior.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DBulley%26aufirst%3DS.%2BJ.%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DDroubi%26aufirst%3DA.%26aulast%3DGiudici%26aufirst%3DM.-L.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DExploring%2520phosphatidylinositol%25205-phosphate%25204-kinase%2520function%26jtitle%3DAdv.%2520Biol.%2520Regul.%26date%3D2015%26volume%3D57%26spage%3D193%26epage%3D202%26doi%3D10.1016%2Fj.jbior.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marklund, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochberg, G. K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benesch, J. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. V.</span></span> <span> </span><span class="NLM_article-title">Bayesian deconvolution of mass and ion mobility spectra: From binary interactions to polydisperse ensembles</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">4370</span>– <span class="NLM_lpage">4376</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.5b00140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.5b00140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=4370-4376&author=M.+T.+Martyauthor=A.+J.+Baldwinauthor=E.+G.+Marklundauthor=G.+K.+A.+Hochbergauthor=J.+L.+P.+Beneschauthor=C.+V.+Robinson&title=Bayesian+deconvolution+of+mass+and+ion+mobility+spectra%3A+From+binary+interactions+to+polydisperse+ensembles&doi=10.1021%2Facs.analchem.5b00140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Bayesian Deconvolution of Mass and Ion Mobility Spectra: From Binary Interactions to Polydisperse Ensembles</span></div><div class="casAuthors">Marty, Michael T.; Baldwin, Andrew J.; Marklund, Erik G.; Hochberg, Georg K. A.; Benesch, Justin L. P.; Robinson, Carol V.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4370-4376</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interpretation of mass spectra is challenging because they report a ratio of 2 phys. quantities, mass and charge, which may each have multiple components that overlap in m/z.  Previous approaches to disentangling the 2 have focused on peak assignment or fitting.  However, the former struggle with complex spectra, and the latter are generally computationally intensive and may require substantial manual intervention.  The authors propose a new data anal. approach that employs a Bayesian framework to sep. the mass and charge dimensions.  From this approach, the authors developed UniDec (Universal Deconvolution), software that provides a rapid, robust, and flexible deconvolution of mass spectra and ion mobility-mass spectra with minimal user intervention.  Incorporation of the charge-state distribution in the Bayesian prior probabilities provides sepn. of the m/z spectrum into its phys. mass and charge components.  The authors have evaluated approach using systems of increasing complexity, enabling one to deduce lipid binding to membrane proteins, to probe the dynamics of subunit exchange reactions, and to characterize polydispersity in both protein assemblies and lipoprotein Nanodiscs.  The general utility of approach will greatly facilitate anal. of ion mobility and mass spectra.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEW75ktwO86rVg90H21EOLACvtfcHk0lg-rzEdmUlUrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D&md5=d2d1272237752ba52cdf2a976608f6bf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.5b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.5b00140%26sid%3Dliteratum%253Aachs%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DBaldwin%26aufirst%3DA.%2BJ.%26aulast%3DMarklund%26aufirst%3DE.%2BG.%26aulast%3DHochberg%26aufirst%3DG.%2BK.%2BA.%26aulast%3DBenesch%26aufirst%3DJ.%2BL.%2BP.%26aulast%3DRobinson%26aufirst%3DC.%2BV.%26atitle%3DBayesian%2520deconvolution%2520of%2520mass%2520and%2520ion%2520mobility%2520spectra%253A%2520From%2520binary%2520interactions%2520to%2520polydisperse%2520ensembles%26jtitle%3DAnal.%2520Chem.%26date%3D2015%26volume%3D87%26spage%3D4370%26epage%3D4376%26doi%3D10.1021%2Facs.analchem.5b00140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Predictive models of aqueous solubility of organic compounds built on a large dataset of high integrity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3110</span>– <span class="NLM_lpage">3114</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.05.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmc.2019.05.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=31176566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFClsrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=3110-3114&author=H.+Sunauthor=P.+Shahauthor=K.+Nguyenauthor=K.+R.+Yuauthor=E.+Kernsauthor=M.+Kabirauthor=Y.+Wangauthor=X.+Xu&title=Predictive+models+of+aqueous+solubility+of+organic+compounds+built+on+a+large+dataset+of+high+integrity&doi=10.1016%2Fj.bmc.2019.05.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive models of aqueous solubility of organic compounds built on A large dataset of high integrity</span></div><div class="casAuthors">Sun, Hongmao; Shah, Pranav; Nguyen, Kimloan; Yu, Katherine; Kerns, Ed; Kabir, Md; Wang, Yuhong; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3110-3114</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aq. soly. is one of the most important properties in drug discovery, as it has profound impact on various drug properties, including biol. activity, pharmacokinetics (PK), toxicity, and in vivo efficacy.  Both kinetic and thermodn. solubilities are detd. during different stages of drug discovery and development.  Since kinetic soly. is more relevant in preclin. drug discovery research, esp. during the structure optimization process, the authors have developed predictive models for kinetic soly. with inhouse data generated from 11,780 compds. collected from over 200 NCATS intramural research projects.  This represents one of the largest kinetic soly. datasets of high quality and integrity.  Based on the customized atom type descriptors, the support vector classification (SVC) models were trained on 80% of the whole dataset, and exhibited high predictive performance for estg. the soly. of the remaining 20% compds. within the test set.  The values of the area under the receiver operating characteristic curve (AUC-ROC) for the compds. in the test sets reached 0.93 and 0.91, when the threshold for insol. compds. was set to 10 and 50 μg/mL, resp.  The predictive models of aq. soly. can be used to identify insol. compds. in drug discovery pipeline, provide design ideas for improving soly. by analyzing the atom types assocd. with poor soly. and prioritize compd. libraries to be purchased or synthesized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfBZqnazp8KLVg90H21EOLACvtfcHk0lg-rzEdmUlUrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFClsrnN&md5=8981617a93473ab5bc1112fc96e2c9fa</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.05.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.05.037%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DKabir%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DPredictive%2520models%2520of%2520aqueous%2520solubility%2520of%2520organic%2520compounds%2520built%2520on%2520a%2520large%2520dataset%2520of%2520high%2520integrity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D3110%26epage%3D3114%26doi%3D10.1016%2Fj.bmc.2019.05.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">An automated high-throughput metabolic stability assay using an integrated high-resolution accurate mass method and automated data analysis software</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1653</span>– <span class="NLM_lpage">1661</span>, <span class="refDoi"> DOI: 10.1124/dmd.116.072017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1124%2Fdmd.116.072017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=27417180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1653-1661&author=P.+Shahauthor=E.+Kernsauthor=D.-T.+Nguyenauthor=S.+Obachauthor=A.+Q.+Wangauthor=A.+Zakharovauthor=J.+McKewauthor=A.+Simeonovauthor=C.+E.+C.+A.+Hopauthor=X.+Xu&title=An+automated+high-throughput+metabolic+stability+assay+using+an+integrated+high-resolution+accurate+mass+method+and+automated+data+analysis+software&doi=10.1124%2Fdmd.116.072017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An automated high-throughput metabolic stability assay using an integrated high-resolution accurate mass method and automated data analysis software</span></div><div class="casAuthors">Shah, Pranav; Kerns, Edward; Nguyen, Dac-Trung; Scott, Obach R.; Wang, Amy Q.; Zakharov, Alexey; McKew, John; Simeonov, Anton; Hop, Cornelis E. C. A.; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1653-1661</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Advancement of in silico tools would be enabled by the availability of data for metabolic reaction rates and intrinsic clearance (CLint) of a diverse compd. structure data set by specific metabolic enzymes.  Our goal is to measure CLint for a large set of compds. with each major human cytochrome P 450 (P 450) isoenzyme.  To achieve our goal, it is of utmost importance to develop an automated, robust, sensitive, high-throughput metabolic stability assay that can efficiently handle a large vol. of compd. sets.  The substrate depletion method [in vitro half-life (t1/2) method] was chosen to det. CLint.  The assay (384-well format) consisted of three parts: (1) a robotic system for incubation and sample cleanup; (2) two different integrated, ultraperformance liq. chromatog./ mass spectrometry (UPLC/MS) platforms to det. the percent remaining of parent compd., and (3) an automated data anal. system.  The CYP3A4 assay was evaluated using two long t1/2 compds., carbamazepine and antipyrine (t1/2 > 30 min); one moderate t1/2 compd., ketoconazole (10 < t1/2 < 30 min); and two short t1/2 compds., loperamide and buspirone (t < 10 min).  Interday and intraday precision and accuracy of the assay were within acceptable range (∼12%) for the linear range obsd.  Using this assay, CYP3A4 CLint and t1/2 values for more than 3000 compds. were measured.  This high-throughput, automated, and robust assay allows for rapid metabolic stability screening of large compd. sets and enables advanced computational modeling for individual human P 450 isoenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos6dcRqLvacbVg90H21EOLACvtfcHk0lidRr0E0P4BJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK&md5=92b4f23376d1cffa70c6fdc209b5569c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.072017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.072017%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DP.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DD.-T.%26aulast%3DObach%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DMcKew%26aufirst%3DJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DAn%2520automated%2520high-throughput%2520metabolic%2520stability%2520assay%2520using%2520an%2520integrated%2520high-resolution%2520accurate%2520mass%2520method%2520and%2520automated%2520data%2520analysis%2520software%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1653%26epage%3D1661%26doi%3D10.1124%2Fdmd.116.072017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly predictive and interpretable models for PAMPA permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+predictive+and+interpretable+models+for+PAMPA+permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0lidRr0E0P4BJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520predictive%2520and%2520interpretable%2520models%2520for%2520PAMPA%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Gangling Liao, Wenjuan Ye, Tyler Heitmann, Glen Ernst, Michael DePasquale, Laiyi Xu, Michael Wormald, Xin Hu, Marc Ferrer, Robert K. Harmel, Dorothea Fiedler, James Barrow, <span class="NLM_string-name hlFld-ContribAuthor">Huijun Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Small-Molecule Inhibitors of Human Inositol Hexakisphosphate Kinases by High-Throughput Screening. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2021,</strong> <em>4 </em>
                                    (2)
                                     , 780-789. <a href="https://doi.org/10.1021/acsptsci.0c00218" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00218</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00218%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DIdentification%252Bof%252BSmall-Molecule%252BInhibitors%252Bof%252BHuman%252BInositol%252BHexakisphosphate%252BKinases%252Bby%252BHigh-Throughput%252BScreening%26aulast%3DLiao%26aufirst%3DGangling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D20122020%26date%3D03032021%26volume%3D4%26issue%3D2%26spage%3D780%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Poli</span>, <span class="hlFld-ContribAuthor ">Shidqiyyah  Abdul-Hamid</span>, <span class="hlFld-ContribAuthor ">Antonio Enrico  Zaurito</span>, <span class="hlFld-ContribAuthor ">Francesca  Campagnoli</span>, <span class="hlFld-ContribAuthor ">Valeria  Bevilacqua</span>, <span class="hlFld-ContribAuthor ">Bhavwanti  Sheth</span>, <span class="hlFld-ContribAuthor ">Roberta  Fiume</span>, <span class="hlFld-ContribAuthor ">Massimiliano  Pagani</span>, <span class="hlFld-ContribAuthor ">Sergio  Abrignani</span>, <span class="hlFld-ContribAuthor ">Nullin  Divecha</span>. </span><span class="cited-content_cbyCitation_article-title">PIP4Ks impact on PI3K, FOXP3, and UHRF1 signaling and modulate human regulatory T cell proliferation and immunosuppressive activity. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2021,</strong> <em>118 </em>
                                    (31)
                                     , e2010053118. <a href="https://doi.org/10.1073/pnas.2010053118" title="DOI URL">https://doi.org/10.1073/pnas.2010053118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2010053118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2010053118%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DPIP4Ks%252Bimpact%252Bon%252BPI3K%25252C%252BFOXP3%25252C%252Band%252BUHRF1%252Bsignaling%252Band%252Bmodulate%252Bhuman%252Bregulatory%252BT%252Bcell%252Bproliferation%252Band%252Bimmunosuppressive%252Bactivity%26aulast%3DPoli%26aufirst%3DAlessandro%26date%3D2021%26date%3D2021%26volume%3D118%26issue%3D31%26spage%3De2010053118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Song  Chen</span>, <span class="hlFld-ContribAuthor ">Caroline  Chandra Tjin</span>, <span class="hlFld-ContribAuthor ">Xiang  Gao</span>, <span class="hlFld-ContribAuthor ">Yi  Xue</span>, <span class="hlFld-ContribAuthor ">Haoyan  Jiao</span>, <span class="hlFld-ContribAuthor ">Ruilin  Zhang</span>, <span class="hlFld-ContribAuthor ">Mengnan  Wu</span>, <span class="hlFld-ContribAuthor ">Zunyu  He</span>, <span class="hlFld-ContribAuthor ">Jonathan  Ellman</span>, <span class="hlFld-ContribAuthor ">Ya  Ha</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological inhibition of PI5P4Kα/β disrupts cell energy metabolism and selectively kills p53-null tumor cells. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2021,</strong> <em>118 </em>
                                    (21)
                                     , e2002486118. <a href="https://doi.org/10.1073/pnas.2002486118" title="DOI URL">https://doi.org/10.1073/pnas.2002486118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2002486118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2002486118%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DPharmacological%252Binhibition%252Bof%252BPI5P4K%2525CE%2525B1%25252F%2525CE%2525B2%252Bdisrupts%252Bcell%252Benergy%252Bmetabolism%252Band%252Bselectively%252Bkills%252Bp53-null%252Btumor%252Bcells%26aulast%3DChen%26aufirst%3DSong%26date%3D2021%26date%3D2021%26volume%3D118%26issue%3D21%26spage%3De2002486118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Scatter plot of high-throughput screen for the PI5P4Kα activity of in-house small molecule library with ∼6000 compounds (shown are hits at <120% DMSO control). The results of two replicates are plotted against each other (normalized to DMSO). Negative controls (DMSO) are shown in dark gray, CVM-05-002 is depicted in green, and the 10% DSMO control cutoff (90% inhibition) is labeled. (B) Concentration–response curves of CVM-05-002, including counterscreen results. (C) Structure of identified lead compound, CVM-05-002.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Binding site of cocrystal structure of PI5P4Kα with bound inhibitor CVM-05-002 (dark gray). Key hydrogen bond interactions are depicted by dashed lines (numbers indicate distances in Å). (B) Key hydrogen bond interactions of the hinge-binding site are shown as dashed lines (numbers indicate distances in Å). (C) Proposed hydrogen bond network of the thiazolidinone moiety of CVM-05-002 with Lys209 and Thr232 (interactions are shown as dashed lines; numbers indicate distances in Å). PDB-ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9">6UX9</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinome-wide selectivity profile for compounds CVM-05-002, <b>3</b>, <b>13</b>, and <b>31</b> (excluding mutant, atypical, and pathogen panels). The inhibitors were tested at 1 μM on a panel of 468 kinases. The results are displayed as red circles with their sizes correlating with the inhibitor’s binding affinity (percent DMSO control).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound CVM-05-002 and Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phenylboronic acids/respective pinacol boronic acid esters, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 9–93% yield; (b) (<i>Z</i>)-methylenethiazolidinone derivative, β-alanine, AcOH, 100 °C, 0.5 h, 12–33% yield, or maleimide, PPh<sub>3</sub>, dichloromethane (DCM), rt, overnight, 21% yield, or oxazolidine-2,4-dione, piperidine, EtOH, 100 °C, overnight, 4% yield, or phenylboronic acids/respective pinacol boronic acid ester, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 1–37% yield, or <i>N</i>-(3-hydroxyphenyl)acetamide, CuI, K<sub>2</sub>CO<sub>3</sub>, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 120 °C, 2 d, 3% yield, or <i>N</i>-(3-bromophenyl)acetamide, Pd<sub>2</sub>(dba)<sub>3</sub>, X-Phos, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 80 °C, overnight, 7% yield; (c) 4-(methanesulfonylamino)phenylboronic acid pinacol ester, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 19–27% yield; (d) 2-iminothiazolidin-4-one, β-alanine, AcOH, 100 °C, 0.5 h, 4–22% yield.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/medium/jm0c00227_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Western blot of the melting curve determination of PI5P4Kα and -β. HEK 293T cells were either treated for 1 h with DMSO or 10 μM CVM-05-002. Representative Western blot of two independent repeats. (B) Structures of selected compounds of the presented SAR study and their biochemical activities. (C) Quantified relative band intensity (%) of the isothermal CETSA experiment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf" class="ext-link">Supporting Figure 10</a>). The mean of four independent experiments is shown. ** <i>p</i> < 0.01, *** <i>p</i> < 0.001 by <i>t</i>-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00227/20200507/images/large/jm0c00227_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00227&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 41 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rameh, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolias, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckworth, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">A new pathway for synthesis for phosphatidylinositol-4,5-bisphosphate</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">192</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1038/36621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2F36621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=9367159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVOntL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=1997&pages=192-196&author=L.+E.+Ramehauthor=K.+F.+Toliasauthor=B.+C.+Duckworthauthor=L.+C.+Cantley&title=A+new+pathway+for+synthesis+for+phosphatidylinositol-4%2C5-bisphosphate&doi=10.1038%2F36621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A new pathway for synthesis of phosphatidylinositol-4,5-bisphosphate</span></div><div class="casAuthors">Rameh, Lucia E.; Tolias, Kimberley F.; Duckworth, Brian C.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">6656</span>),
    <span class="NLM_cas:pages">192-196</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Phosphatidylinositol-4,5-bisphosphate (PtdIns-4,5-P2), a key mol. in the phosphoinositide signaling pathway, was thought to be synthesized exclusively by phosphorylation of PtdIns-4-P at the D-5 position of the inositol ring.  The enzymes that produce PtdIns-4,5-P2 in vitro fall into two related subfamilies (type I and type II PtdInsP-5-OH kinases, or PIP(5)Ks) based on their enzymic properties and sequence similarities.  Here we have reinvestigated the substrate specificities of these enzymes.  As expected, the type I enzyme phosphorylates PtdIns-4-P at the D-5 position of the inositol ring.  Surprisingly, the type II enzyme, which is abundant in some tissues, phosphorylates PtdIns-5-P at the D-4 position, and thus should be considered as a 4-OH kinase, or PIP(4)K.  The earlier error in characterizing the activity of the type II enzyme is due to the presence of contaminating PtdIns-5-P in com. prepns. of PtdIns-4-P.  Although PtdIns-5-P was previously thought not to exist in vivo, we find evidence for the presence of this lipid in mammalian fibroblasts, establishing a new pathway for PtdIns-4,5-P2 synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5w0D7qVNp4rVg90H21EOLACvtfcHk0lgjiF1w44DrZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVOntL0%253D&md5=8196f300c91a49209d7c47eaff979282</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F36621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F36621%26sid%3Dliteratum%253Aachs%26aulast%3DRameh%26aufirst%3DL.%2BE.%26aulast%3DTolias%26aufirst%3DK.%2BF.%26aulast%3DDuckworth%26aufirst%3DB.%2BC.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DA%2520new%2520pathway%2520for%2520synthesis%2520for%2520phosphatidylinositol-4%252C5-bisphosphate%26jtitle%3DNature%26date%3D1997%26volume%3D390%26spage%3D192%26epage%3D196%26doi%3D10.1038%2F36621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, B.-L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">PIP4K2A regulates intracellular cholesterol transport through modulating PI(4,5)<sub>2</sub> homeostasis</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">514</span>, <span class="refDoi"> DOI: 10.1194/jlr.M082149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1194%2Fjlr.M082149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29353240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1ags7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2018&pages=507-514&author=A.+Huauthor=X.-T.+Zhaoauthor=T.+Fuauthor=Y.+Wangauthor=Y.+Liuauthor=X.-J.+Shiauthor=J.+Luoauthor=B.-L.+Song&title=PIP4K2A+regulates+intracellular+cholesterol+transport+through+modulating+PI%284%2C5%292+homeostasis&doi=10.1194%2Fjlr.M082149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PIP4K2A regulates intracellular cholesterol transport through modulating PI(4,5)P2 homeostasis</span></div><div class="casAuthors">Hu, Ao; Zhao, Xue-Tong; Tu, Heng; Xiao, Ting; Fu, Ting; Wang, Yan; Liu, Yong; Shi, Xiong-Jie; Luo, Jie; Song, Bao-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">507-514</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The transport of LDL-derived cholesterol from lysosomes to peroxisomes is facilitated by membrane contacts formed between the lysosomal protein synaptotagmin VII and the peroxisomal lipid phosphatidylinositol 4, 5-bisphosphate [PI(4,5)P2].  Here, we used RNA interference to search for regulators of PI(4,5)P2 and to study the effects of altered PI(4,5)P2 homeostasis on cholesterol transport.  We found that knockdown of phosphatidylinositol 5-phosphate 4-kinase type-2 α (PIP4K2A) reduced peroxisomal PI(4,5)P2 levels, decreased lysosome-peroxisome membrane contacts, and increased accumulation of lysosomal cholesterol in human SV-589 fibroblasts.  Forced expression of peroxisome-localized, kinase-active PIP4K2A in the knockdown cells reduced cholesterol accumulation, and in vitro addn. of recombinant PIP4K2A restored membrane contacts.  These results suggest that PIP4K2A plays a crit. role in intracellular cholesterol transport by upregulating PI(4,5)P2 levels in the peroxisomal membrane.  Further research into PIP4K2A activity may inform future therapeutic interventions for managing lysosomal storage disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIj9z5zOM-vrVg90H21EOLACvtfcHk0lgjiF1w44DrZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1ags7c%253D&md5=e5fe05aaa5f1015939df56bda9cb825e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M082149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M082149%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.-T.%26aulast%3DFu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DX.-J.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DB.-L.%26atitle%3DPIP4K2A%2520regulates%2520intracellular%2520cholesterol%2520transport%2520through%2520modulating%2520PI%25284%252C5%25292%2520homeostasis%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2018%26volume%3D59%26spage%3D507%26epage%3D514%26doi%3D10.1194%2Fjlr.M082149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamia, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peroni, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rameh, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Increased insulin sensitivity and reduced adiposity in phosphatidylinositol 5-phoshate 4-kinase β<sup>–/–</sup> mice</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">5080</span>– <span class="NLM_lpage">5087</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.11.5080-5087.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1128%2FMCB.24.11.5080-5087.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=15143198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVyhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=5080-5087&author=K.+A.+Lamiaauthor=O.+D.+Peroniauthor=Y.-B.+Kimauthor=L.+E.+Ramehauthor=B.+B.+Kahnauthor=L.+C.+Cantley&title=Increased+insulin+sensitivity+and+reduced+adiposity+in+phosphatidylinositol+5-phoshate+4-kinase+%CE%B2%E2%80%93%2F%E2%80%93+mice&doi=10.1128%2FMCB.24.11.5080-5087.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Increased insulin sensitivity and reduced adiposity in phosphatidylinositol 5-phosphate 4-kinase β-/- mice</span></div><div class="casAuthors">Lamia, Katja A.; Peroni, Odile D.; Kim, Young-Bum; Rameh, Lucia E.; Kahn, Barbara B.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5080-5087</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phosphorylated derivs. of the lipid phosphatidylinositol are known to play crit. roles in insulin response.  Phosphatidylinositol 5-phosphate 4-kinases convert phosphatidylinositol 5-phosphate to phosphatidylinositol 4,5-bis-phosphate.  To understand the physiol. role of these kinases, we generated mice that do not express phosphatidylinositol 5-phosphate 4-kinase β.  These mice are hypersensitive to insulin and have reduced body wts. compared to wild-type littermates.  While adult male mice lacking phosphatidylinositol 5-phosphate 4-kinase β have significantly less body fat than wild-type littermates, female mice lacking phosphatidylinositol 5-phosphate 4-kinase β have increased insulin sensitivity in the presence of normal adiposity.  Furthermore, in vivo insulin-induced activation of the protein kinase Akt is enhanced in skeletal muscle and liver from mice lacking phosphatidylinositol 5-phosphate 4-kinase β.  These results indicate that phosphatidylinositol 5-phosphate 4-kinase β plays a role in detg. insulin sensitivity and adiposity in vivo and suggest that inhibitors of this enzyme may be useful in the treatment of type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM5s3W3lwpLrVg90H21EOLACvtfcHk0liCW8-qY1NK8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVyhtr0%253D&md5=1a53df7bd5b007d1c2b6f3ef4c691706</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.11.5080-5087.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.11.5080-5087.2004%26sid%3Dliteratum%253Aachs%26aulast%3DLamia%26aufirst%3DK.%2BA.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DKim%26aufirst%3DY.-B.%26aulast%3DRameh%26aufirst%3DL.%2BE.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DIncreased%2520insulin%2520sensitivity%2520and%2520reduced%2520adiposity%2520in%2520phosphatidylinositol%25205-phoshate%25204-kinase%2520%25CE%25B2%25E2%2580%2593%252F%25E2%2580%2593%2520mice%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D5080%26epage%3D5087%26doi%3D10.1128%2FMCB.24.11.5080-5087.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsamooj, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosch, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo-Wing, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulf, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Deletion of the gene <i>Pip4k2c</i>, a novel phosphatidylinositol kinase, results in hyperactivation of the immune system</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7596</span>– <span class="NLM_lpage">7601</span>, <span class="refDoi"> DOI: 10.1073/pnas.1600934113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1073%2Fpnas.1600934113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=27313209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC28XpvFeku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7596-7601&author=H.+Shimauthor=C.+Wuauthor=S.+Ramsamoojauthor=K.+N.+Boschauthor=Z.+Chenauthor=B.+M.+Emerlingauthor=J.+Yunauthor=H.+Liuauthor=R.+Choo-Wingauthor=Z.+Yangauthor=G.+M.+Wulfauthor=V.+K.+Kuchrooauthor=L.+C.+Cantley&title=Deletion+of+the+gene+Pip4k2c%2C+a+novel+phosphatidylinositol+kinase%2C+results+in+hyperactivation+of+the+immune+system&doi=10.1073%2Fpnas.1600934113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of the gene Pip4k2c, a novel phosphatidylinositol kinase, results in hyperactivation of the immune system</span></div><div class="casAuthors">Shim, Hyeseok; Wu, Chuan; Ramsamooj, Shivan; Bosch, Kaitlyn N.; Chen, Zuojia; Emerling, Brooke M.; Yun, Jihye; Liu, Hui; Choo-Wing, Rayman; Yang, Zhiwei; Wulf, Gerburg M.; Kuchroo, Vijay Kumar; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">7596-7601</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Type 2 phosphatidylinositol-5-phosphate 4-kinase (PI5P4K) converts phosphatidylinositol-5-phosphate to phosphatidylinositol-4,5-bisphosphate.  Mammals have three enzymes PI5P4Kα, PI5P4Kβ, and PI5P4Kγ, and these enzymes have been implicated in metabolic control, growth control, and a variety of stress responses.  Here, the authors show that mice with germline deletion of type 2 phosphatidylinositol-5-phosphate 4-kinase gamma (Pip4k2c), the gene encoding PI5P4Kγ, appear normal in regard to growth and viability but have increased inflammation and T-cell activation as they age.  Immune cell infiltrates increased in Pip4k2c-/- mouse tissues.  Also, there was an increase in proinflammatory cytokines, including IFNγ, interleukin 12, and interleukin 2 in plasma of Pip4k2c-/- mice.  Pip4k2c-/- mice had an increase in T-helper-cell populations and a decrease in regulatory T-cell populations with increased proliferation of T cells.  Interestingly, mammalian target of rapamycin complex 1 (mTORC1) signaling was hyperactivated in several tissues from Pip4k2c-/- mice and treating Pip4k2c-/- mice with rapamycin reduced the inflammatory phenotype, resulting in a decrease in mTORC1 signaling in tissues and a decrease in proinflammatory cytokines in plasma.  These results indicate that PI5P4Kγ plays a role in the regulation of the immune system via mTORC1 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKHtPtm6tGbrVg90H21EOLACvtfcHk0liCW8-qY1NK8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpvFeku7w%253D&md5=97c294816991d9790a8241c8bac965cb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1600934113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1600934113%26sid%3Dliteratum%253Aachs%26aulast%3DShim%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DRamsamooj%26aufirst%3DS.%26aulast%3DBosch%26aufirst%3DK.%2BN.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26aulast%3DYun%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DChoo-Wing%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWulf%26aufirst%3DG.%2BM.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DDeletion%2520of%2520the%2520gene%2520Pip4k2c%252C%2520a%2520novel%2520phosphatidylinositol%2520kinase%252C%2520results%2520in%2520hyperactivation%2520of%2520the%2520immune%2520system%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7596%26epage%3D7601%26doi%3D10.1073%2Fpnas.1600934113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lundquist, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goncalves, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughran, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possik, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayaraghavan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, K. S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinkove, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divecha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ause, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span> <span> </span><span class="NLM_article-title">Phosphatidylinositol-5-phosphate 4-kinases regulate cellular lipid metabolism by facilitating autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.03.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.molcel.2018.03.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29727621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovFGiu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=531-543&author=M.+R.+Lundquistauthor=M.+D.+Goncalvesauthor=R.+M.+Loughranauthor=E.+Possikauthor=T.+Vijayaraghavanauthor=A.+Yangauthor=C.+Pauliauthor=A.+Raviauthor=A.+Vermaauthor=Z.+Yangauthor=J.+L.+Johnsonauthor=J.+C.+Y.+Wongauthor=Y.+Maauthor=K.+S.-K.+Hwangauthor=D.+Weinkoveauthor=N.+Divechaauthor=J.+M.+Asaraauthor=O.+Elementoauthor=M.+A.+Rubinauthor=A.+C.+Kimmelmanauthor=A.+auseauthor=L.+C.+Cantleyauthor=B.+M.+Emerling&title=Phosphatidylinositol-5-phosphate+4-kinases+regulate+cellular+lipid+metabolism+by+facilitating+autophagy&doi=10.1016%2Fj.molcel.2018.03.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol-5-Phosphate 4-Kinases Regulate Cellular Lipid Metabolism By Facilitating Autophagy</span></div><div class="casAuthors">Lundquist, Mark R.; Goncalves, Marcus D.; Loughran, Ryan M.; Possik, Elite; Vijayaraghavan, Tarika; Yang, Annan; Pauli, Chantal; Ravi, Archna; Verma, Akanksha; Yang, Zhiwei; Johnson, Jared L.; Wong, Jenny C. Y.; Ma, Yilun; Hwang, Katie Seo-Kyoung; Weinkove, David; Divecha, Nullin; Asara, John M.; Elemento, Olivier; Rubin, Mark A.; Kimmelman, Alec C.; Pause, Arnim; Cantley, Lewis C.; Emerling, Brooke M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">531-544.e9</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">While the majority of phosphatidylinositol-4, 5-bisphosphate (PI-4,5-P2) in mammalian cells is generated by the conversion of phosphatidylinositol-4-phosphate (PI-4-P) to PI-4,5-P2, a small fraction can be made by phosphorylating phosphatidylinositol-5-phosphate (PI-5-P).  The physiol. relevance of this second pathway is not clear.  Here, we show that deletion of the genes encoding the two most active enzymes in this pathway, Pip4k2a and Pip4k2b, in the liver of mice causes a large enrichment in lipid droplets and in autophagic vesicles during fasting.  These changes are due to a defect in the clearance of autophagosomes that halts autophagy and reduces the supply of nutrients salvaged through this pathway.  Similar defects in autophagy are seen in nutrient-starved Pip4k2a-/-Pip4k2b-/- mouse embryonic fibroblasts and in C. elegans lacking the PI5P4K ortholog.  These results suggest that this alternative pathway for PI-4,5-P2 synthesis evolved, in part, to enhance the ability of multicellular organisms to survive starvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3l-f4CGzkx7Vg90H21EOLACvtfcHk0ligzpDqbsPOoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovFGiu7c%253D&md5=e68f21997c55e440747cfacfa03e20af</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.03.037%26sid%3Dliteratum%253Aachs%26aulast%3DLundquist%26aufirst%3DM.%2BR.%26aulast%3DGoncalves%26aufirst%3DM.%2BD.%26aulast%3DLoughran%26aufirst%3DR.%2BM.%26aulast%3DPossik%26aufirst%3DE.%26aulast%3DVijayaraghavan%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DPauli%26aufirst%3DC.%26aulast%3DRavi%26aufirst%3DA.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DJohnson%26aufirst%3DJ.%2BL.%26aulast%3DWong%26aufirst%3DJ.%2BC.%2BY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DHwang%26aufirst%3DK.%2BS.-K.%26aulast%3DWeinkove%26aufirst%3DD.%26aulast%3DDivecha%26aufirst%3DN.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3Dause%26aufirst%3DA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26atitle%3DPhosphatidylinositol-5-phosphate%25204-kinases%2520regulate%2520cellular%2520lipid%2520metabolism%2520by%2520facilitating%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D70%26spage%3D531%26epage%3D543%26doi%3D10.1016%2Fj.molcel.2018.03.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droubi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">In B cells, phosphatidylinositol 5-phosphate 4-kinase-α synthesizes PI(4,5)P<sub>2</sub> to impact mTORC2 and Akt signaling</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">10571</span>– <span class="NLM_lpage">10576</span>, <span class="refDoi"> DOI: 10.1073/pnas.1522478113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1073%2Fpnas.1522478113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=27601656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=10571-10576&author=S.+J.+Bulleyauthor=A.+Droubiauthor=J.+H.+Clarkeauthor=K.+E.+Andersonauthor=L.+R.+Stephensauthor=P.+T.+Hawkinsauthor=R.+F.+Irvine&title=In+B+cells%2C+phosphatidylinositol+5-phosphate+4-kinase-%CE%B1+synthesizes+PI%284%2C5%29P2+to+impact+mTORC2+and+Akt+signaling&doi=10.1073%2Fpnas.1522478113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">In B cells, phosphatidylinositol 5-phosphate 4-kinase-α synthesizes PI(4,5)P2 to impact mTORC2 and Akt signaling</span></div><div class="casAuthors">Bulley, Simon J.; Droubi, Alaa; Clarke, Jonathan H.; Anderson, Karen E.; Stephens, Len R.; Hawkins, Phillip T.; Irvine, Robin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10571-10576</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are enigmatic lipid kinases with physiol. functions that are incompletely understood, not the least because genetic deletion and cell transfection have led to contradictory data.  Here, we used the genetic tractability of DT40 cells to create cell lines in which endogenous PI5P4Kα was removed, either stably by genetic deletion or transiently (within 1 h) by tagging the endogenous protein genomically with the auxin degron.  In both cases, removal impacted Akt phosphorylation, and by leaving one PI5P4Kα allele present but mutating it to be kinase-dead or have PI4P 5-kinase activity, we show that all of the effects on Akt phosphorylation were dependent on the ability of PI5P4Kα to synthesize phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] rather than to remove PI5P.  Although stable removal of PI5P4Kα resulted in a pronounced decrease in Akt phosphorylation at Thr308 and Ser473, in part because of reduced plasma membrane PIP3, its acute removal led to an increase in Akt phosphorylation only at Ser473.  This process invokes activation primarily of mammalian target of rapamycin complex 2 (mTORC2), which was confirmed by increased phosphorylation of other mTORC2 substrates.  These findings establish PI5P4Kα as a kinase that synthesizes a physiol. relevant pool of PI(4,5)P2 and as a regulator of mTORC2, and show a phenomenon similar to the "butterfly effect" described for phosphatidylinositol 3-kinase Iα [Hart JR, et al. (2015) Proc Natl Acad Sci USA 112(4):1131-1136], whereby through apparently the same underlying mechanism, the removal of a protein's activity from a cell can have widely divergent effects depending on the time course of that removal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvPu7VOCVxFLVg90H21EOLACvtfcHk0ligzpDqbsPOoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsb3O&md5=1c385a91dca10e46324865d27fce88f4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1522478113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1522478113%26sid%3Dliteratum%253Aachs%26aulast%3DBulley%26aufirst%3DS.%2BJ.%26aulast%3DDroubi%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DAnderson%26aufirst%3DK.%2BE.%26aulast%3DStephens%26aufirst%3DL.%2BR.%26aulast%3DHawkins%26aufirst%3DP.%2BT.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DIn%2520B%2520cells%252C%2520phosphatidylinositol%25205-phosphate%25204-kinase-%25CE%25B1%2520synthesizes%2520PI%25284%252C5%2529P2%2520to%2520impact%2520mTORC2%2520and%2520Akt%2520signaling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D10571%26epage%3D10576%26doi%3D10.1073%2Fpnas.1522478113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keune, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divecha, N.</span></span> <span> </span><span class="NLM_article-title">PtdIns5<i>P</i> and Pin1 in oxidative stress signaling</span>. <i>Adv. Biol. Regul.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1016/j.jbior.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.jbior.2013.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=23602596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlGrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=179-189&author=W.-J.+Keuneauthor=D.+R.+Jonesauthor=N.+Divecha&title=PtdIns5P+and+Pin1+in+oxidative+stress+signaling&doi=10.1016%2Fj.jbior.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">PtdIns5P and Pin1 in oxidative stress signaling</span></div><div class="casAuthors">Keune, Willem-Jan; Jones, David R.; Divecha, Nullin</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Biological Regulation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-189</span>CODEN:
                <span class="NLM_cas:coden">ABRDE5</span>;
        ISSN:<span class="NLM_cas:issn">2212-4934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Oxidative signaling is important in cellular health, involved in aging and contributes to the development of several diseases such as cancer, neurodegeneration and diabetes.  Correct management of reactive oxygen species (ROS) prevents oxidative stress within cells and is imperative for cellular wellbeing.  A key pathway that is regulated by oxidative stress is the activation of proline-directed stress kinases (p38, JNK).  Phosphorylation induced by these kinases is often translated into cellular outcome through the recruitment of the prolyl-isomerase Pin1.  Pin1 binds to phosphorylated substrates using its WW-domain and can induce conformational changes in the target protein through its prolyl-isomerase activity.  We show that exposure of cells to UV irradn. or hydrogen peroxide (H2O2), induces the synthesis of the phosphoinositide second messenger PtdIns5P in part by inducing the interaction between phosphatidylinositol-5-phosphate 4-kinase (PIP4K) enzymes that remove PtdIns5P, with Pin1.  In response to H2O2 exposure, Murine Embryonic Fibroblasts (MEFs) derived from Pin1-/- mice showed increased cell viability and an increased abundance of PtdIns5P compared to wild-type MEFs.  Decreasing the levels of PtdIns5P in Pin1-/- MEFs decreased both their viability in response to H2O2 exposure and the expression of genes required for cellular ROS management.  The decrease in the expression of these genes manifested itself in the increased accumulation of cellular ROS.  These data strongly argue that PtdIns5P acts as a stress-induced second messenger that can calibrate how cells manage ROS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3lS32vZ0c47Vg90H21EOLACvtfcHk0ligzpDqbsPOoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGrtr0%253D&md5=1d64aef9a69ef29b45d257184dd68255</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.jbior.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbior.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DKeune%26aufirst%3DW.-J.%26aulast%3DJones%26aufirst%3DD.%2BR.%26aulast%3DDivecha%26aufirst%3DN.%26atitle%3DPtdIns5P%2520and%2520Pin1%2520in%2520oxidative%2520stress%2520signaling%26jtitle%3DAdv.%2520Biol.%2520Regul.%26date%3D2013%26volume%3D53%26spage%3D179%26epage%3D189%26doi%3D10.1016%2Fj.jbior.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ramahi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giridharan, S. S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safren, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Haro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner Gee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titus, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainc, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barmada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of PIP4Kγ amerliorates the pathological effects of mutant huntingtin protein</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>, <span class="NLM_elocation-id">e29123</span> <span class="refDoi"> DOI: 10.7554/eLife.29123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.7554%2FeLife.29123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29256861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1enu7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=I.+Al-Ramahiauthor=S.+S.+P.+Giridharanauthor=Y.-C.+Chenauthor=S.+Patnaikauthor=N.+Safrenauthor=J.+Hasegawaauthor=M.+de+Haroauthor=A.+K.+Wagner+Geeauthor=S.+A.+Titusauthor=H.+Jeongauthor=J.+Clarkeauthor=D.+Kraincauthor=W.+Zhengauthor=R.+F.+Irvineauthor=S.+Barmadaauthor=M.+Ferrerauthor=N.+Southallauthor=L.+S.+Weismanauthor=J.+Botasauthor=J.+J.+Marugan&title=Inhibition+of+PIP4K%CE%B3+amerliorates+the+pathological+effects+of+mutant+huntingtin+protein&doi=10.7554%2FeLife.29123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of PIP4Kγ ameliorates the pathological effects of mutant huntingtin protein</span></div><div class="casAuthors">Al-Ramahi, Ismael; Giridharan, Sai Srinivas Panapakkam; Chen, Yu-Chi; Patnaik, Samarjit; Safren, Nathaniel; Hasegawa, Junya; de Haro, Maria; Gee, Amanda K. Wagner; Titus, Steven A.; Jeong, Hyunkyung; Clarke, Jonathan; Krainc, Dimitri; Zheng, Wei; Irvine, Robin F.; Barmada, Sami; Ferrer, Marc; Southall, Noel; Weisman, Lois S.; Botas, Juan; Marugan, Juan Jose</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29123/1-e29123/25</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The discovery of the causative gene for Huntington's disease (HD) has promoted numerous efforts to uncover cellular pathways that lower levels of mutant huntingtin protein (mHtt) and potentially forestall the appearance of HD-related neurol. defects.  Using a cell-based model of pathogenic huntingtin expression, we identified a class of compds. that protect cells through selective inhibition of a lipid kinase, PIP4Kγ.  Pharmacol. inhibition or knock-down of PIP4Kγ modulates the equil. between phosphatidylinositide (PI) species within the cell and increases basal autophagy, reducing the total amt. of mHtt protein in human patient fibroblasts and aggregates in neurons.  In two Drosophila models of Huntington's disease, genetic knockdown of PIP4K ameliorated neuronal dysfunction and degeneration as assessed using motor performance and retinal degeneration assays resp.  Together, these results suggest that PIP4Kγ is a druggable target whose inhibition enhances productive autophagy and mHtt proteolysis, revealing a useful pharmacol. point of intervention for the treatment of Huntington's disease, and potentially for other neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDQBO77YRMvLVg90H21EOLACvtfcHk0ljlOHnd_jeg5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1enu7nK&md5=61a8a87888b3fbfc8a24b4d58f08527d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.7554%2FeLife.29123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.29123%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Ramahi%26aufirst%3DI.%26aulast%3DGiridharan%26aufirst%3DS.%2BS.%2BP.%26aulast%3DChen%26aufirst%3DY.-C.%26aulast%3DPatnaik%26aufirst%3DS.%26aulast%3DSafren%26aufirst%3DN.%26aulast%3DHasegawa%26aufirst%3DJ.%26aulast%3Dde%2BHaro%26aufirst%3DM.%26aulast%3DWagner%2BGee%26aufirst%3DA.%2BK.%26aulast%3DTitus%26aufirst%3DS.%2BA.%26aulast%3DJeong%26aufirst%3DH.%26aulast%3DClarke%26aufirst%3DJ.%26aulast%3DKrainc%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26aulast%3DBarmada%26aufirst%3DS.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DWeisman%26aufirst%3DL.%2BS.%26aulast%3DBotas%26aufirst%3DJ.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26atitle%3DInhibition%2520of%2520PIP4K%25CE%25B3%2520amerliorates%2520the%2520pathological%2520effects%2520of%2520mutant%2520huntingtin%2520protein%26jtitle%3DeLife%26date%3D2017%26doi%3D10.7554%2FeLife.29123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jude, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somerville, T. D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divecha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span> <span> </span><span class="NLM_article-title">A targeted knockdown screen of genes coding for phosphoinositide modulators identifies <i>PIP4K2A</i> as required for acute myeloid leukemia cell proliferation and survival</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1253</span>– <span class="NLM_lpage">1262</span>, <span class="refDoi"> DOI: 10.1038/onc.2014.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fonc.2014.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24681948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFOgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=1253-1262&author=J.+G.+Judeauthor=G.+J.+Spencerauthor=X.+Huangauthor=T.+D.+D.+Somervilleauthor=D.+R.+Jonesauthor=N.+Divechaauthor=T.+C.+P.+Somervaille&title=A+targeted+knockdown+screen+of+genes+coding+for+phosphoinositide+modulators+identifies+PIP4K2A+as+required+for+acute+myeloid+leukemia+cell+proliferation+and+survival&doi=10.1038%2Fonc.2014.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A targeted knockdown screen of genes coding for phosphoinositide modulators identifies PIP4K2A as required for acute myeloid leukemia cell proliferation and survival</span></div><div class="casAuthors">Jude, J. G.; Spencer, G. J.; Huang, X.; Somerville, T. D. D.; Jones, D. R.; Divecha, N.; Somervaille, T. C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1253-1262</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Given the importance of deregulated phosphoinositide (PI) signaling in leukemic hematopoiesis, genes coding for proteins that regulate PI metab. may have significant and as yet unappreciated roles in leukemia.  We performed a targeted knockdown (KD) screen of PI modulator genes in human acute myeloid leukemia (AML) cells and identified candidates required to sustain proliferation or prevent apoptosis.  One of these, the lipid kinase phosphatidylinositol-5-phosphate 4-kinase, type II, alpha (PIP4K2A) regulates cellular levels of phosphatidylinositol-5-phosphate (PtsIns5P) and phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2).  We found PIP4K2A to be essential for the clonogenic and leukemia-initiating potential of human AML cells, and for the clonogenic potential of murine MLL-AF9 AML cells.  Importantly, PIP4K2A is also required for the clonogenic potential of primary human AML cells.  Its KD results in accumulation of the cyclin-dependent kinase inhibitors CDKN1A and CDKN1B, G1 cell cycle arrest and apoptosis.  Both CDKN1A accumulation and apoptosis were partially dependent on activation of the mTOR pathway.  Critically, however, PIP4K2A KD in normal hematopoietic stem and progenitor cells, both murine and human, did not adversely impact either clonogenic or multilineage differentiation potential, indicating a selective dependency that we suggest may be the consequence of the regulation of different transcriptional programs in normal vs. malignant cells.  Thus, PIP4K2A is a novel candidate therapeutic target in myeloid malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1NosZXJPbErVg90H21EOLACvtfcHk0ljlOHnd_jeg5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFOgt70%253D&md5=28c5806e8ceddbd668bb7066ce183b36</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.77%26sid%3Dliteratum%253Aachs%26aulast%3DJude%26aufirst%3DJ.%2BG.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DSomerville%26aufirst%3DT.%2BD.%2BD.%26aulast%3DJones%26aufirst%3DD.%2BR.%26aulast%3DDivecha%26aufirst%3DN.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DA%2520targeted%2520knockdown%2520screen%2520of%2520genes%2520coding%2520for%2520phosphoinositide%2520modulators%2520identifies%2520PIP4K2A%2520as%2520required%2520for%2520acute%2520myeloid%2520leukemia%2520cell%2520proliferation%2520and%2520survival%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26spage%3D1253%26epage%3D1262%26doi%3D10.1038%2Fonc.2014.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luoh, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, R.</span></span> <span> </span><span class="NLM_article-title">Overexpression of the amplified <i>Pip</i>4<i>k</i>2β gene from 17q11-12 in breast cancer cells confers proliferation advantage</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1354</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1207251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fsj.onc.1207251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=14691457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlKhtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=1354-1363&author=S.-W.+Luohauthor=N.+Venkatesanauthor=R.+Tripathi&title=Overexpression+of+the+amplified+Pip4k2%CE%B2+gene+from+17q11-12+in+breast+cancer+cells+confers+proliferation+advantage&doi=10.1038%2Fsj.onc.1207251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of the amplified Pip4k2β gene from 17q11-12 in breast cancer cells confers proliferation advantage</span></div><div class="casAuthors">Luoh, Shiuh-Wen; Venkatesan, Natarajan; Tripathi, Reshimi</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1354-1363</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene amplification is common in solid tumors and is assocd. with adverse prognosis, disease progression, and development of drug resistance.  A small segment from chromosome 17q11-12 contg. the HER-2/Neu gene is amplified in about 25% of breast cancer.  HER-2/Neu amplification is assocd. with adverse prognosis and may predict response to chemotherapy and hormonal manipulation.  Moreover, HER-2/Neu amplification may select patients for anti-HER-2/Neu-based therapy with Herceptin.  The authors and others recently described a common sequence element from the HER-2/Neu region that was amplified in breast cancer cells.  In addn., most, if not all, of the amplified genes from this region display overexpression.  This raises the intriguing possibility that genes immediately adjacent to HER-2/Neu may influence the biol. behavior of breast cancer carrying HER-2/Neu amplification and serve as rational targets for therapy.  By extg. sequence information from public databases, the authors have constructed a contig in bacterial artificial chromosomes (BACs) that extends from HER-2/Neu to a phosphatidylinositol phosphate kinase (PIPK), Pip4k2β from 17q11-12.  Although a role of PI-3-kinase and AKT in cancer biol. has been previously described, PIPK has not been previously implicated.  The authors show that Pip4k2β, initially known as Pip5k2β, is amplified in a subset of breast cancer cell lines and primary breast cancer samples that carry HER-2/Neu amplification.  Out of eight breast cancer cell lines with HER-2/Neu amplification, three have concomitant amplification of the Pip4k2β gene - UACC-812, BT-474 and ZR-75-30.  Similarly, two out of four primary breast tumors with HER-2/Neu amplification carry Pip4k2β gene amplification.  Intriguingly, one tumor displays an increase in the gene copy no. of Pip4k2β that is significantly more than that of HER-2/Neu.  Moreover, dual color FISH reveals that amplified Pip4k2β gene may exist in a distinct structure from that of HER-2/Neu in ZR-75-30 cell line.  These studies suggest that Pip4k2β may reside on an amplification max. distinct from that of HER-2/Neu and serve as an independent target for amplification and selective retention.  Pip4k2β amplification is assocd. with overexpression at the RNA and protein level in breast cancer cell lines.  Stable expression of Pip4k2β in breast cancer cell lines with and without HER-2/Neu amplification increases cell proliferation and anchorage-independent growth.  The above observations implicate Pip4k2β in the development and/or progression of breast cancer.  The authors' study suggests that Pip4k2β may be a distinct target for gene amplification and selective retention from 17q11-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6BmkVQfDHk7Vg90H21EOLACvtfcHk0liZgkUbShY-7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlKhtbk%253D&md5=2fd95453b778fb45b899d4e049aa57d5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207251%26sid%3Dliteratum%253Aachs%26aulast%3DLuoh%26aufirst%3DS.-W.%26aulast%3DVenkatesan%26aufirst%3DN.%26aulast%3DTripathi%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520the%2520amplified%2520Pip4k2%25CE%25B2%2520gene%2520from%252017q11-12%2520in%2520breast%2520cancer%2520cells%2520confers%2520proliferation%2520advantage%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D1354%26epage%3D1363%26doi%3D10.1038%2Fsj.onc.1207251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulogiannis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukazawa, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo-Wing, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulf, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamia, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rameh, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Depletion of a putatively druggable class of phosphatidylinositol kinases inhibits growth of p53-null tumors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">844</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.09.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.cell.2013.09.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24209622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCls7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=844-857&author=B.+M.+Emerlingauthor=J.+B.+Hurovauthor=G.+Poulogiannisauthor=K.+S.+Tsukazawaauthor=R.+Choo-Wingauthor=G.+M.+Wulfauthor=E.+L.+Bellauthor=H.-S.+Shimauthor=K.+A.+Lamiaauthor=L.+E.+Ramehauthor=G.+Bellingerauthor=A.+T.+Sasakiauthor=J.+M.+Asaraauthor=X.+Yuanauthor=A.+Bullockauthor=G.+M.+DeNicolaauthor=J.+Songauthor=V.+Brownauthor=S.+Signorettiauthor=L.+C.+Cantley&title=Depletion+of+a+putatively+druggable+class+of+phosphatidylinositol+kinases+inhibits+growth+of+p53-null+tumors&doi=10.1016%2Fj.cell.2013.09.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of a Putatively Druggable Class of Phosphatidylinositol Kinases Inhibits Growth of p53-Null Tumors</span></div><div class="casAuthors">Emerling, Brooke M.; Hurov, Jonathan B.; Poulogiannis, George; Tsukazawa, Kazumi S.; Choo-Wing, Rayman; Wulf, Gerburg M.; Bell, Eric L.; Shim, Hye-Seok; Lamia, Katja A.; Rameh, Lucia E.; Bellinger, Gary; Sasaki, Atsuo T.; Asara, John M.; Yuan, Xin; Bullock, Andrea; DeNicola, Gina M.; Song, Jiaxi; Brown, Victoria; Signoretti, Sabina; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">844-857</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Here, we show that a subset of breast cancers express high levels of the type 2 phosphatidylinositol-5-phosphate 4-kinases α and/or β (PI5P4Kα and β) and provide evidence that these kinases are essential for growth in the absence of p53.  Knocking down PI5P4Kα and β in a breast cancer cell line bearing an amplification of the gene encoding PI5P4K β and deficient for p53 impaired growth on plastic and in xenografts.  This growth phenotype was accompanied by enhanced levels of reactive oxygen species (ROS) leading to senescence.  Mice with homozygous deletion of both TP53 and PIP4K2B were not viable, indicating a synthetic lethality for loss of these two genes.  Importantly however, PIP4K2A-/-, PIP4K2B+/-, and TP53-/- mice were viable and had a dramatic redn. in tumor formation compared to TP53-/- littermates.  These results indicate that inhibitors of PI5P4Ks could be effective in preventing or treating cancers with mutations in TP53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozvZfpTxgf9bVg90H21EOLACvtfcHk0liZgkUbShY-7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCls7rK&md5=adf6f54d285108fdc10ab269488faf4c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.09.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.09.057%26sid%3Dliteratum%253Aachs%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26aulast%3DHurov%26aufirst%3DJ.%2BB.%26aulast%3DPoulogiannis%26aufirst%3DG.%26aulast%3DTsukazawa%26aufirst%3DK.%2BS.%26aulast%3DChoo-Wing%26aufirst%3DR.%26aulast%3DWulf%26aufirst%3DG.%2BM.%26aulast%3DBell%26aufirst%3DE.%2BL.%26aulast%3DShim%26aufirst%3DH.-S.%26aulast%3DLamia%26aufirst%3DK.%2BA.%26aulast%3DRameh%26aufirst%3DL.%2BE.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DBullock%26aufirst%3DA.%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DV.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DDepletion%2520of%2520a%2520putatively%2520druggable%2520class%2520of%2520phosphatidylinositol%2520kinases%2520inhibits%2520growth%2520of%2520p53-null%2520tumors%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D844%26epage%3D857%26doi%3D10.1016%2Fj.cell.2013.09.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerling, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pragani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span> <span> </span><span class="NLM_article-title">A Homogeneous, high-throughput assay for phosphatidylinositol 5-phosphate 4-kinase with a novel, rapid substrate preparation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e54127</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0054127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1371%2Fjournal.pone.0054127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=23326584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=M.+I.+Davisauthor=A.+T.+Sasakiauthor=M.+Shenauthor=B.+M.+Emerlingauthor=N.+Thorneauthor=S.+Michaelauthor=R.+Praganiauthor=M.+Boxerauthor=K.+Sumitaauthor=K.+Takeuchiauthor=D.+S.+Auldauthor=Z.+Liauthor=L.+C.+Cantleyauthor=A.+Simeonov&title=A+Homogeneous%2C+high-throughput+assay+for+phosphatidylinositol+5-phosphate+4-kinase+with+a+novel%2C+rapid+substrate+preparation&doi=10.1371%2Fjournal.pone.0054127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A homogeneous, high-throughput assay for phosphatidylinositol 5-phosphate 4-kinase with a novel, rapid substrate preparation</span></div><div class="casAuthors">Davis, Mindy I.; Sasaki, Atsuo T.; Shen, Min; Emerling, Brooke M.; Thorne, Natasha; Michael, Sam; Pragani, Rajan; Boxer, Matthew; Sumita, Kazutaka; Takeuchi, Koh; Auld, Douglas S.; Li, Zhuyin; Cantley, Lewis C.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e54127</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Phosphoinositide kinases regulate diverse cellular functions and are important targets for therapeutic development for diseases, such as diabetes and cancer.  Prepn. of the lipid substrate is crucial for the development of a robust and miniaturizable lipid kinase assay.  Enzymic assays for phosphoinositide kinases often use lipid substrates prepd. from lyophilized lipid prepns. by sonication, which result in variability in the liposome size from prepn. to prepn.  Herein, we report a homogeneous 1536-well luciferase-coupled bioluminescence assay for PI5P4Kα.  The substrate prepn. is novel and allows the rapid prodn. of a DMSO-contg. substrate soln. without the need for lengthy liposome prepn. protocols, thus enabling the scale-up of this traditionally difficult type of assay.  The Z'-factor value was greater than 0.7 for the PI5P4Kα assay, indicating its suitability for high-throughput screening applications.  Tyrphostin AG-82 had been identified as an inhibitor of PI5P4Kα by assessing the degree of phospho transfer of γ-32P-ATP to PI5P; its inhibitory activity against PI5P4Kα was confirmed in the present miniaturized assay.  From a pilot screen of a library of bioactive compds., another tyrphostin, I-OMe tyrphostin AG-538 (I-OMe-AG-538), was identified as an ATP-competitive inhibitor of PI5P4Kα with an IC50 of 1 μM, affirming the suitability of the assay for inhibitor discovery campaigns.  This homogeneous assay may apply to other lipid kinases and should help in the identification of leads for this class of enzymes by enabling high-throughput screening efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomhA8K8BJDALVg90H21EOLACvtfcHk0lhBM8GVO_ooOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCqsbo%253D&md5=254e360d68a54fbd0aa3f61b0e41ef0d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0054127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0054127%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DEmerling%26aufirst%3DB.%2BM.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMichael%26aufirst%3DS.%26aulast%3DPragani%26aufirst%3DR.%26aulast%3DBoxer%26aufirst%3DM.%26aulast%3DSumita%26aufirst%3DK.%26aulast%3DTakeuchi%26aufirst%3DK.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DA%2520Homogeneous%252C%2520high-throughput%2520assay%2520for%2520phosphatidylinositol%25205-phosphate%25204-kinase%2520with%2520a%2520novel%252C%2520rapid%2520substrate%2520preparation%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0054127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, P.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampetrean, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingyun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabhu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diam, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span> <span> </span><span class="NLM_article-title">Dual blockade of the lipid kinase PIP4Ks and mitotic pathways lead to cancer-selective lethality</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">2200</span> <span class="refDoi"> DOI: 10.1038/s41467-017-02287-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fs41467-017-02287-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29259156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzjtlaluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=M.+Kitagawaauthor=P.-J.+Liaoauthor=K.+H.+Leeauthor=J.+Wongauthor=S.+C.+Shangauthor=N.+Minamiauthor=O.+Sampetreanauthor=H.+Sayaauthor=D.+Lingyunauthor=N.+Prabhuauthor=G.+K.+Diamauthor=R.+Sobataauthor=A.+Larssonauthor=P.+Nordlungauthor=F.+McCormickauthor=S.+Ghoshauthor=D.+M.+Esteinauthor=B.+W.+Dymockauthor=S.+H.+Lee&title=Dual+blockade+of+the+lipid+kinase+PIP4Ks+and+mitotic+pathways+lead+to+cancer-selective+lethality&doi=10.1038%2Fs41467-017-02287-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Dual blockade of the lipid kinase PIP4Ks and mitotic pathways leads to cancer-selective lethality</span></div><div class="casAuthors">Kitagawa Mayumi; Liao Pei-Ju; Lee Kyung Hee; Wong Jasmine; Epstein David M; Lee Sang Hyun; Shang See Cheng; Dymock Brian W; Minami Noriaki; Sampetrean Oltea; Saya Hideyuki; Lingyun Dai; Prabhu Nayana; Diam Go Ka; Larsson Andreas; Nordlund Par; Sobota Radoslaw; Nordlund Par; Nordlund Par; McCormick Frank; Ghosh Sujoy; Ghosh Sujoy; Epstein David M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2200</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Achieving robust cancer-specific lethality is the ultimate clinical goal.  Here, we identify a compound with dual-inhibitory properties, named a131, that selectively kills cancer cells, while protecting normal cells.  Through an unbiased CETSA screen, we identify the PIP4K lipid kinases as the target of a131.  Ablation of the PIP4Ks generates a phenocopy of the pharmacological effects of PIP4K inhibition by a131.  Notably, PIP4Ks inhibition by a131 causes reversible growth arrest in normal cells by transcriptionally upregulating PIK3IP1, a suppressor of the PI3K/Akt/mTOR pathway.  Strikingly, Ras activation overrides a131-induced PIK3IP1 upregulation and activates the PI3K/Akt/mTOR pathway.  Consequently, Ras-transformed cells override a131-induced growth arrest and enter mitosis where a131's ability to de-cluster supernumerary centrosomes in cancer cells eliminates Ras-activated cells through mitotic catastrophe.  Our discovery of drugs with a dual-inhibitory mechanism provides a unique pharmacological strategy against cancer and evidence of cross-activation between the Ras/Raf/MEK/ERK and PI3K/AKT/mTOR pathways via a Ras PIK3IP1 PI3K signaling network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdeIFfZgL8lhFwe7cAcrfPfW6udTcc2eaPsY0UtH5ePbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzjtlaluw%253D%253D&md5=f995efabc4f2f5eae92680ea8247d7ce</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02287-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02287-5%26sid%3Dliteratum%253Aachs%26aulast%3DKitagawa%26aufirst%3DM.%26aulast%3DLiao%26aufirst%3DP.-J.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DWong%26aufirst%3DJ.%26aulast%3DShang%26aufirst%3DS.%2BC.%26aulast%3DMinami%26aufirst%3DN.%26aulast%3DSampetrean%26aufirst%3DO.%26aulast%3DSaya%26aufirst%3DH.%26aulast%3DLingyun%26aufirst%3DD.%26aulast%3DPrabhu%26aufirst%3DN.%26aulast%3DDiam%26aufirst%3DG.%2BK.%26aulast%3DSobata%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DA.%26aulast%3DNordlung%26aufirst%3DP.%26aulast%3DMcCormick%26aufirst%3DF.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DEstein%26aufirst%3DD.%2BM.%26aulast%3DDymock%26aufirst%3DB.%2BW.%26aulast%3DLee%26aufirst%3DS.%2BH.%26atitle%3DDual%2520blockade%2520of%2520the%2520lipid%2520kinase%2520PIP4Ks%2520and%2520mitotic%2520pathways%2520lead%2520to%2520cancer-selective%2520lethality%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-02287-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Voss, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petry, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummerhop, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, H.-L.</span></span> <span> </span><span class="NLM_article-title">Discovery and pharmacological characterization of a novel small molecule inhibitor of phosphatidylinositol-5-phosphate 4-kinase, type II, beta</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">331</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2014.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bbrc.2014.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24845568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslWrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2014&pages=327-331&author=M.+D.+Vossauthor=W.+Czechtizkyauthor=Z.+Liauthor=C.+Rudolphauthor=S.+Petryauthor=H.+Brummerhopauthor=T.+Langerauthor=A.+Schifferauthor=H.-L.+Schaefer&title=Discovery+and+pharmacological+characterization+of+a+novel+small+molecule+inhibitor+of+phosphatidylinositol-5-phosphate+4-kinase%2C+type+II%2C+beta&doi=10.1016%2Fj.bbrc.2014.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and pharmacological characterization of a novel small molecule inhibitor of phosphatidylinositol-5-phosphate 4-kinase, type II, beta</span></div><div class="casAuthors">Voss, Marc D.; Czechtizky, Werngard; Li, Ziyu; Rudolph, Christine; Petry, Stefan; Brummerhop, Harm; Langer, Thomas; Schiffer, Alexander; Schaefer, Hans-Ludwig</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-331</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phosphatidylinositol-5-phosphate 4-kinase, type II, beta (PIP5K2B) is linked to the pathogenesis of obesity, insulin resistance and diabetes.  Here, we describe the identification of a novel pyrimidine-2,4-diamine PIP5K2B inhibitor, designated SAR088.  The compd. was identified by high-throughput screening and subsequently characterized in vitro and in vivo.  SAR088 showed reasonable potency, selectivity and physicochem. properties in enzymic and cellular assays.  In vivo, SAR088 lowered blood glucose levels of obese and hyperglycemic male Zucker diabetic fatty rats treated for 3 wk.  Thus, SAR088 represents the first orally available and in vivo active PIP5K2B inhibitor and provides an excellent starting point for the development of potent and selective PIP5K2B inhibitors for the treatment of insulin resistance and diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolvj65tqi667Vg90H21EOLACvtfcHk0li9CNV24FV5dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslWrtro%253D&md5=e9e0e60ec94e0b96daaadafbdc99072e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2014.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2014.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DVoss%26aufirst%3DM.%2BD.%26aulast%3DCzechtizky%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DPetry%26aufirst%3DS.%26aulast%3DBrummerhop%26aufirst%3DH.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DSchiffer%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DH.-L.%26atitle%3DDiscovery%2520and%2520pharmacological%2520characterization%2520of%2520a%2520novel%2520small%2520molecule%2520inhibitor%2520of%2520phosphatidylinositol-5-phosphate%25204-kinase%252C%2520type%2520II%252C%2520beta%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2014%26volume%3D449%26spage%3D327%26epage%3D331%26doi%3D10.1016%2Fj.bbrc.2014.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giudici, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">The function of phosphatidylinositol 5-phosphate 4-kinase γ (PI5P4Kγ) explored using a specific inhibitor that targets the PI5P-binding site</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">359</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1042/BJ20141333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1042%2FBJ20141333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=25495341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFGjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2015&pages=359-367&author=J.+H.+Clarkeauthor=M.-L.+Giudiciauthor=J.+E.+Burkeauthor=R.+L.+Williamsauthor=D.+J.+Maloneyauthor=J.+Maruganauthor=R.+F.+Irvine&title=The+function+of+phosphatidylinositol+5-phosphate+4-kinase+%CE%B3+%28PI5P4K%CE%B3%29+explored+using+a+specific+inhibitor+that+targets+the+PI5P-binding+site&doi=10.1042%2FBJ20141333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The function of phosphatidylinositol 5-phosphate 4-kinase γ (PI5P4Kγ) explored using a specific inhibitor that targets the PI5P-binding site</span></div><div class="casAuthors">Clarke, Jonathan H.; Giudici, Maria-Luisa; Burke, John E.; Williams, Roger L.; Maloney, David J.; Marugan, Juan; Irvine, Robin F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">359-367</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">NIH-12848 (NCGC00012848-02), a putative phosphatidylinositol 5-phosphate 4-kinase γ (PI5P4Kγ) inhibitor, was explored as a tool for investigating this enigmatic, low activity, lipid kinase.  PI5P4K assays in vitro showed that NIH-12848 inhibited PI5P4Kγ with an IC50 of approx. 1 μM but did not inhibit the α and β PI5P4K isoforms at concns. up to 100 μM.  A lack of inhibition of PI5P4Kγ ATPase activity suggested that NIH-12848 does not interact with the enzyme's ATP-binding site and direct exploration of binding using hydrogen-deuterium exchange (HDX)-MS (HDX-MS) revealed the putative PI5P-binding site of PI5P4Kγ to be the likely region of interaction.  This was confirmed by a series of mutation expts. which led to the identification of a single PI5P4Kγ amino acid residue that can be mutated to its PI5P4Ks α and β homolog to render PI5P4Kγ resistant NIH-12848 inhibition.  NIH-12848 (10 μM) was applied to cultured mouse principal kidney cortical collecting duct (mpkCCD) cells which, we show, express PI5P4Kγ that increases when the cells grow to confluence and polarize.  NIH-12848 inhibited the translocation of Na+/K+-ATPase to the plasma membrane that occurs when mpkCCD cells grow to confluence and also prevented reversibly their forming of 'domes' on the culture dish.  Both these NIH-12848-induced effects were mimicked by specific RNAi knockdown of PI5P4Kγ, but not that of PI5P4Ks α or β.  Overall, the data reveal a probable contribution of PI5P4Kγ to the development and maintenance of epithelial cell functional polarity and show that NIH-12848 is a potentially powerful tool for exploring the cell physiol. of PI5P4Ks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDqfSMRVrIlbVg90H21EOLACvtfcHk0li9CNV24FV5dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFGjsr0%253D&md5=18babc0ecf7b25901e75277955d0d075</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1042%2FBJ20141333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141333%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DGiudici%26aufirst%3DM.-L.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DMarugan%26aufirst%3DJ.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DThe%2520function%2520of%2520phosphatidylinositol%25205-phosphate%25204-kinase%2520%25CE%25B3%2520%2528PI5P4K%25CE%25B3%2529%2520explored%2520using%2520a%2520specific%2520inhibitor%2520that%2520targets%2520the%2520PI5P-binding%2520site%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D466%26spage%3D359%26epage%3D367%26doi%3D10.1042%2FBJ20141333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sivakumaren, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundquist, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paddock, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting the PI5P4K lipid kinase family in cancer using novel covalent inhibitors</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.chembiol.2020.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=32130941" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+C.+Sivakumarenauthor=H.+Shimauthor=T.+Zhangauthor=F.+M.+Fergusonauthor=M.+R.+Lundquistauthor=C.+M.+Browneauthor=H.-S.+Seoauthor=M.+N.+Paddockauthor=T.+D.+Manzauthor=B.+Jiangauthor=M.-F.+Haoauthor=P.+Krishnanauthor=D.+G.+Wangauthor=J.+Yangauthor=N.+P.+Kwiatkowskiauthor=S.+B.+Ficarroauthor=J.+M.+Cunninghamauthor=J.+A.+Martoauthor=S.+Dhe-Paganonauthor=L.+C.+Cantleyauthor=N.+S.+Gray&title=Targeting+the+PI5P4K+lipid+kinase+family+in+cancer+using+novel+covalent+inhibitors&doi=10.1016%2Fj.chembiol.2020.02.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DSivakumaren%26aufirst%3DS.%2BC.%26aulast%3DShim%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DLundquist%26aufirst%3DM.%2BR.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DPaddock%26aufirst%3DM.%2BN.%26aulast%3DManz%26aufirst%3DT.%2BD.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DHao%26aufirst%3DM.-F.%26aulast%3DKrishnan%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DD.%2BG.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DKwiatkowski%26aufirst%3DN.%2BP.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DCunningham%26aufirst%3DJ.%2BM.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520the%2520PI5P4K%2520lipid%2520kinase%2520family%2520in%2520cancer%2520using%2520novel%2520covalent%2520inhibitors%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26doi%3D10.1016%2Fj.chembiol.2020.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivakumaren, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure-activity relationship study of covalent pan-phosphatidylinositol 5-phosphate 4-kinase inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">346</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00402</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00402" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=346-352&author=T.+D.+Manzauthor=S.+C.+Sivakumarenauthor=A.+Yasgarauthor=M.+D.+Hallauthor=M.+I.+Davisauthor=H.-S.+Seoauthor=J.+D.+Cardauthor=H.+Shimauthor=S.+B.+Ficarroauthor=S.+Dhe-Paganonauthor=A.+T.+Sasakiauthor=M.+B.+Boxerauthor=A.+Simeonovauthor=L.+C.+Cantleyauthor=M.+Shenauthor=T.+Zhangauthor=F.+M.+Fergusonauthor=N.+S.+Gray&title=Structure-activity+relationship+study+of+covalent+pan-phosphatidylinositol+5-phosphate+4-kinase+inhibitors&doi=10.1021%2Facsmedchemlett.9b00402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship Study of Covalent Pan-phosphatidylinositol 5-Phosphate 4-Kinase Inhibitors</span></div><div class="casAuthors">Manz, Theresa D.; Sivakumaren, Sindhu C.; Yasgar, Adam; Hall, Matthew D.; Davis, Mindy I.; Seo, Hyuk-Soo; Card, Joseph D.; Ficarro, Scott B.; Shim, Hyeseok; Marto, Jarrod A.; Dhe-Paganon, Sirano; Sasaki, Atsuo T.; Boxer, Matthew B.; Simeonov, Anton; Cantley, Lewis C.; Shen, Min; Zhang, Tinghu; Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">346-352</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) are important mol. players in a variety of diseases, such as cancer.  Currently available PI5P4K inhibitors are reversible small mols., which may lack selectivity and sufficient cellular on-target activity.  In this study, we present a new class of covalent pan-PI5P4K inhibitors with potent biochem. and cellular activity.  Our designs are based on THZ-P1-2, a covalent PI5P4K inhibitor previously developed in our lab.  Here, we report further structure-guided optimization and structure-activity relationship (SAR) study of this scaffold, resulting in compd. 30(I), which retained biochem. and cellular potency, while demonstrating a significantly improved selectivity profile.  Furthermore, we confirm that the inhibitors show efficient binding affinity in the context of HEK 293T cells using isothermal CETSA methods.  Taken together, I represents a highly selective pan-PI5P4K covalent lead mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMr8VvNNJeXbVg90H21EOLACvtfcHk0lhuaajG4y4v9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7bO&md5=4fcc29df1916dac233a8f048132439be</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00402%26sid%3Dliteratum%253Aachs%26aulast%3DManz%26aufirst%3DT.%2BD.%26aulast%3DSivakumaren%26aufirst%3DS.%2BC.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DCard%26aufirst%3DJ.%2BD.%26aulast%3DShim%26aufirst%3DH.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DStructure-activity%2520relationship%2520study%2520of%2520covalent%2520pan-phosphatidylinositol%25205-phosphate%25204-kinase%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D346%26epage%3D352%26doi%3D10.1021%2Facsmedchemlett.9b00402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paweletz, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stetson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivancos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantarini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oxnard, G. R.</span></span> <span> </span><span class="NLM_article-title">Acquired <i>EGFR</i> C797S mediates resistance to AZD9291 in advanced non-small cell lung cancer harboring <i>EGFR</i> T790M</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nm.3854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fnm.3854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=25939061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=560-562&author=K.+S.+Thressauthor=C.+P.+Paweletzauthor=E.+Felipauthor=B.+C.+Choauthor=D.+Stetsonauthor=B.+Doughertyauthor=Z.+Laiauthor=A.+Markovetsauthor=A.+Vivancosauthor=Y.+Kuangauthor=D.+Ercanauthor=S.+Matthewsauthor=M.+Cantariniauthor=J.+C.+Barrettauthor=P.+A.+J%C3%A4nneauthor=G.+R.+Oxnard&title=Acquired+EGFR+C797S+mediates+resistance+to+AZD9291+in+advanced+non-small+cell+lung+cancer+harboring+EGFR+T790M&doi=10.1038%2Fnm.3854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</span></div><div class="casAuthors">Thress, Kenneth S.; Paweletz, Cloud P.; Felip, Enriqueta; Cho, Byoung Chul; Stetson, Daniel; Dougherty, Brian; Lai, Zhongwu; Markovets, Aleksandra; Vivancos, Ana; Kuang, Yanan; Ercan, Dalia; Matthews, Sarah E.; Cantarini, Mireille; Barrett, J. Carl; Janne, Pasi A.; Oxnard, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">560-562</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we studied cell-free plasma DNA (cfDNA) collected from subjects with advanced lung cancer whose tumors had developed resistance to the epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) AZD9291.  We first performed next-generation sequencing of cfDNA from seven subjects and detected an acquired EGFR C797S mutation in one; expression of this mutant EGFR construct in a cell line rendered it resistant to AZD9291.  We then performed droplet digital PCR on serial cfDNA specimens collected from 15 AZD9291-treated subjects.  All were pos. for the T790M mutation before treatment, but upon developing AZD9291 resistance three mol. subtypes emerged: six cases acquired the C797S mutation, five cases maintained the T790M mutation but did not acquire the C797S mutation and four cases lost the T790M mutation despite the presence of the underlying EGFR activating mutation.  Our findings provide insight into the diversity of mechanisms through which tumors acquire resistance to AZD9291 and highlight the need for therapies that are able to overcome resistance mediated by the EGFR C797S mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTggAW5fpGrbVg90H21EOLACvtfcHk0liwbSLkcjmIfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ&md5=adac4cb6c7057855aff61fd62890d010</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnm.3854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3854%26sid%3Dliteratum%253Aachs%26aulast%3DThress%26aufirst%3DK.%2BS.%26aulast%3DPaweletz%26aufirst%3DC.%2BP.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DCho%26aufirst%3DB.%2BC.%26aulast%3DStetson%26aufirst%3DD.%26aulast%3DDougherty%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DMarkovets%26aufirst%3DA.%26aulast%3DVivancos%26aufirst%3DA.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DMatthews%26aufirst%3DS.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DOxnard%26aufirst%3DG.%2BR.%26atitle%3DAcquired%2520EGFR%2520C797S%2520mediates%2520resistance%2520to%2520AZD9291%2520in%2520advanced%2520non-small%2520cell%2520lung%2520cancer%2520harboring%2520EGFR%2520T790M%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D560%26epage%3D562%26doi%3D10.1038%2Fnm.3854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span> <span> </span><span class="NLM_article-title">Comment on the ecstasy and agony of assay interference compounds</span>. <i>J. Chem. Inf. Model</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2640</span>– <span class="NLM_lpage">2645</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00313</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00313" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1OisbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2640-2645&author=P.+W.+Kenny&title=Comment+on+the+ecstasy+and+agony+of+assay+interference+compounds&doi=10.1021%2Facs.jcim.7b00313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Comment on The Ecstasy and Agony of Assay Interference Compounds</span></div><div class="casAuthors">Kenny, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2640-2645</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A recent editorial (Aldrich et al, The Ecstasy and Agony of Assay Interference Compds., J. Chem. Inf. Model., 2017, 57, 387-390) is examd. critically.  When assessing assay hits from screening, it is important to draw a distinction between false positives, that have no effect on target function, and compds. that affect target function through an undesirable mechanism of action.  Observation of frequent-hitter behavior for a compd. should be regarded as circumstantial evidence, rather than definitive proof, that the compd. has interfered with assay readouts or acted through an undesirable mechanism of action.  The applicability domain of published (Baell & Holloway, J. Med. Chem. 2010, 53, 2719-2740) Pan Assay INterference compds. (PAINS) filters is limited by the narrow scope of the proprietary data used to derive them.  It is suggested that journal guidelines for authors should not prescribe, as those for Journal of Medicinal Chem. appear to do, that activity in assays reported for compds. that match PAINS filters be treated any differently to that for compds. that do not match PAINS filters.  It is argued that use of models based on proprietary data in the evaluation of manuscripts would contradict the editorial policy of any journal that deemed the use of proprietary data to be unacceptable in modeling studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcp9s-M9UjILVg90H21EOLACvtfcHk0liwbSLkcjmIfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1OisbrF&md5=3c558e79c8b4e036206656350d72c800</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00313%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DComment%2520on%2520the%2520ecstasy%2520and%2520agony%2520of%2520assay%2520interference%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model%26date%3D2017%26volume%3D57%26spage%3D2640%26epage%3D2645%26doi%3D10.1021%2Facs.jcim.7b00313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgu3F32OvoMqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven year itch: Pan-assay interference compounds (PAINS) in 2017 – utility and limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+year+itch%3A+Pan-assay+interference+compounds+%28PAINS%29+in+2017+%E2%80%93+utility+and+limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0lgpb9E8wQdziA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520year%2520itch%253A%2520Pan-assay%2520interference%2520compounds%2520%2528PAINS%2529%2520in%25202017%2520%25E2%2580%2593%2520utility%2520and%2520limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendgen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Privileged scaffolds or promiscuous binders: A comparative study on rhodanines and related heterocycles in medicinal chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">753</span>, <span class="refDoi"> DOI: 10.1021/jm201243p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201243p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=743-753&author=T.+Mendgenauthor=C.+Steuerauthor=C.+D.+Klein&title=Privileged+scaffolds+or+promiscuous+binders%3A+A+comparative+study+on+rhodanines+and+related+heterocycles+in+medicinal+chemistry&doi=10.1021%2Fjm201243p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged Scaffolds or Promiscuous Binders: A Comparative Study on Rhodanines and Related Heterocycles in Medicinal Chemistry</span></div><div class="casAuthors">Mendgen, Thomas; Steuer, Christian; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">743-753</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Rhodanines and related five-membered heterocycles with multiple heteroatoms have recently gained a reputation of being unselective compds. that appear as "frequent hitters" in screening campaigns and therefore have little value in drug discovery.  However, this judgment appears to be based mostly on anecdotal evidence.  Having identified various rhodanines and related compds. in screening campaigns, we decided to perform a systematic study on their promiscuity.  An amt. of 163 rhodanines, hydantoins, thiohydantoins, and thiazolidinediones were synthesized and tested against several targets.  The compds. were also characterized with respect to aggregation and electrophilic reactivity, and the binding modes of rhodanines and related compds. in published x-ray cocrystal structures were analyzed.  The results indicate that the exocyclic, double bonded sulfur atom in rhodanines and thiohydantoins, in addn. to other structural features, offers a particularly high d. of interaction sites for polar interactions and hydrogen bonds.  This causes a promiscuous behavior at concns. in the "screening range" but should not be regarded as a general knockout criterion that excludes such screening hits from further development.  It is suggested that special criteria for target affinity and selectivity are applied to these classes of compds. and that their exceptional and potentially valuable biomol. binding properties are consequently exploited in a useful way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLzna8Pcz1iLVg90H21EOLACvtfcHk0lgpb9E8wQdziA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGkt7nN&md5=6a94d097f44bdbaaec9a56fd6b371521</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm201243p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201243p%26sid%3Dliteratum%253Aachs%26aulast%3DMendgen%26aufirst%3DT.%26aulast%3DSteuer%26aufirst%3DC.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DPrivileged%2520scaffolds%2520or%2520promiscuous%2520binders%253A%2520A%2520comparative%2520study%2520on%2520rhodanines%2520and%2520related%2520heterocycles%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D743%26epage%3D753%26doi%3D10.1021%2Fjm201243p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camps, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardissone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rintelen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrandi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillieron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gretener, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitte, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymann, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirillo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rommel, C.</span></span> <span> </span><span class="NLM_article-title">Blockade of PI3Kγ suppresses joint inflammation and damage in mouse models of rheumatoid arthritis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">943</span>, <span class="refDoi"> DOI: 10.1038/nm1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fnm1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=16127437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslelur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=936-943&author=M.+Campsauthor=T.+Rueckleauthor=H.+Jiauthor=V.+Ardissoneauthor=F.+Rintelenauthor=J.+Shawauthor=C.+Ferrandiauthor=C.+Chabertauthor=C.+Gillieronauthor=B.+Franconauthor=T.+Martinauthor=D.+Gretenerauthor=D.+Perrinauthor=D.+Leroyauthor=P.-A.+Vitteauthor=E.+Hirschauthor=M.+P.+Wymannauthor=R.+Cirilloauthor=M.+K.+Schwarzauthor=C.+Rommel&title=Blockade+of+PI3K%CE%B3+suppresses+joint+inflammation+and+damage+in+mouse+models+of+rheumatoid+arthritis&doi=10.1038%2Fnm1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Blockade of PI3Kγ suppresses joint inflammation and damage in mouse models of rheumatoid arthritis</span></div><div class="casAuthors">Camps, Montserrat; Rueckle, Thomas; Ji, Hong; Ardissone, Vittoria; Rintelen, Felix; Shaw, Jeffrey; Ferrandi, Chiara; Chabert, Christian; Gillieron, Corine; Francon, Bernard; Martin, Thierry; Gretener, Denise; Perrin, Dominique; Leroy, Didier; Vitte, Pierre-Alain; Hirsch, Emilio; Wymann, Matthias P.; Cirillo, Rocco; Schwarz, Matthias K.; Rommel, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">936-943</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinases (PI3K) have long been considered promising drug targets for the treatment of inflammatory and autoimmune disorders as well as cancer and cardiovascular diseases.  But the lack of specificity, isoform selectivity and poor biopharmaceutical profile of PI3K inhibitors have so far hampered rigorous disease-relevant target validation.  Here we describe the identification and development of specific, selective and orally active small-mol. inhibitors of PI3Kγ (encoded by Pik3cg).  We show that Pik3cg-/- mice are largely protected in mouse models of rheumatoid arthritis; this protection correlates with defective neutrophil migration, further validating PI3Kγ as a therapeutic target.  We also describe that oral treatment with a PI3Kγ inhibitor suppresses the progression of joint inflammation and damage in two distinct mouse models of rheumatoid arthritis, reproducing the protective effects shown by Pik3cg-/- mice.  Our results identify selective PI3Kγ inhibitors as potential therapeutic mols. for the treatment of chronic inflammatory disorders such as rheumatoid arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEyzzdnR3FXLVg90H21EOLACvtfcHk0lgpb9E8wQdziA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslelur8%253D&md5=1fed7126427b519a4a457a8bf31ade7d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnm1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1284%26sid%3Dliteratum%253Aachs%26aulast%3DCamps%26aufirst%3DM.%26aulast%3DRueckle%26aufirst%3DT.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DArdissone%26aufirst%3DV.%26aulast%3DRintelen%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%26aulast%3DFerrandi%26aufirst%3DC.%26aulast%3DChabert%26aufirst%3DC.%26aulast%3DGillieron%26aufirst%3DC.%26aulast%3DFrancon%26aufirst%3DB.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DGretener%26aufirst%3DD.%26aulast%3DPerrin%26aufirst%3DD.%26aulast%3DLeroy%26aufirst%3DD.%26aulast%3DVitte%26aufirst%3DP.-A.%26aulast%3DHirsch%26aufirst%3DE.%26aulast%3DWymann%26aufirst%3DM.%2BP.%26aulast%3DCirillo%26aufirst%3DR.%26aulast%3DSchwarz%26aufirst%3DM.%2BK.%26aulast%3DRommel%26aufirst%3DC.%26atitle%3DBlockade%2520of%2520PI3K%25CE%25B3%2520suppresses%2520joint%2520inflammation%2520and%2520damage%2520in%2520mouse%2520models%2520of%2520rheumatoid%2520arthritis%26jtitle%3DNat.%2520Med.%26date%3D2005%26volume%3D11%26spage%3D936%26epage%3D943%26doi%3D10.1038%2Fnm1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span> <span> </span><span class="NLM_article-title">Darbufelone, a novel anti-inflammatory drug, induces growth inhibition of lung cancer cells both in vitro and in vivo</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">285</span>, <span class="refDoi"> DOI: 10.1007/s00280-009-1161-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1007%2Fs00280-009-1161-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=20352217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1alsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=277-285&author=X.+Yeauthor=W.+Zhouauthor=Y.+Liauthor=Y.+Sunauthor=Y.+Zhangauthor=H.+Jiauthor=Y.+Lai&title=Darbufelone%2C+a+novel+anti-inflammatory+drug%2C+induces+growth+inhibition+of+lung+cancer+cells+both+in+vitro+and+in+vivo&doi=10.1007%2Fs00280-009-1161-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Darbufelone, a novel anti-inflammatory drug, induces growth inhibition of lung cancer cells both in vitro and in vivo</span></div><div class="casAuthors">Ye, Xiaolei; Zhou, Wu; Li, Yongqi; Sun, Yihua; Zhang, Yihua; Ji, Hui; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">277-285</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Inflammation plays a crucial role in the development of lung cancer.  Accumulated studies have proved that non-steroidal anti-inflammatory drugs (NSAIDs) which block inflammation by their actions on arachidonic acid (AA) metab. have a potential role in cancer chemotherapy and chemoprevention.  The aim of our study was to investigate whether darbufelone, a novel anti-inflammatory drug, has anticancer effects in lung cancer.  Human non-small cell lung cancer cell lines were treated with darbufelone at various doses and time points for anal. of cell viability, cell cycle, and apoptosis in vitro.  The in vivo effect of darbufelone was assessed in Lewis lung carcinoma mice model.  Darbufelone inhibited the proliferation of non-small cell lung cancer cell lines in a dose-dependent manner, and induced cell cycle arrest at G0/G1 phase through up-regulation of p27 expression.  Treatment with darbufelone also induced apoptosis by activating caspase-3 and caspase-8.  Lewis lung carcinoma growth was also significantly inhibited by darbufelone treatment at daily dose of 80 mg/kg.  Taken together, these studies suggested that darbufelone, an anti-inflammation drug, might represent a novel therapeutic approach for lung cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkTuHuzaKpHbVg90H21EOLACvtfcHk0ljh_bBypCvKpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1alsb4%253D&md5=4a6f1df926a4ff5b2c041206c9a659bc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs00280-009-1161-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-009-1161-z%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DDarbufelone%252C%2520a%2520novel%2520anti-inflammatory%2520drug%252C%2520induces%2520growth%2520inhibition%2520of%2520lung%2520cancer%2520cells%2520both%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2010%26volume%3D66%26spage%3D277%26epage%3D285%26doi%3D10.1007%2Fs00280-009-1161-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghatak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fresco, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwald, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamy, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afrasiabi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhury, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khetmalas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padhye, S.</span></span> <span> </span><span class="NLM_article-title">Novel di-tertiary-butyl phenylhydrazones as dual cyclooxygenase-2/5-lipoxygenase inhibitors: Synthesis, COX/LOX inhibition, molecular modeling, and insights into their cytotoxicities</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmcl.2013.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=24295787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWku7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=317-324&author=S.+Ghatakauthor=A.+Vyasauthor=S.+Misraauthor=P.+O%E2%80%99Brienauthor=A.+Zambreauthor=V.+M.+Frescoauthor=R.+R.+Markwaldauthor=K.+V.+Swamyauthor=Z.+Afrasiabiauthor=A.+Choudhuryauthor=M.+Khetmalasauthor=S.+Padhye&title=Novel+di-tertiary-butyl+phenylhydrazones+as+dual+cyclooxygenase-2%2F5-lipoxygenase+inhibitors%3A+Synthesis%2C+COX%2FLOX+inhibition%2C+molecular+modeling%2C+and+insights+into+their+cytotoxicities&doi=10.1016%2Fj.bmcl.2013.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Novel di-tertiary-butyl phenylhydrazones as dual cyclooxygenase-2/5-lipoxygenase inhibitors: Synthesis, COX/LOX inhibition, molecular modeling, and insights into their cytotoxicities</span></div><div class="casAuthors">Ghatak, Shibnath; Vyas, Alok; Misra, Suniti; O'Brien, Paul; Zambre, Ajit; Fresco, Victor M.; Markwald, Roger R.; Swamy, K. Venkateshwara; Afrasiabi, Zahra; Choudhury, Amitava; Khetmalas, Madhukar; Padhye, Subhash</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">317-324</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Although dual inhibition of Cyclooxygenase-2 (COX-2) and 5-Lipoxygenase (5-LOX) enzymes is highly effective than targeting COX or LOX alone, there are only a few reports of examg. such compds. in case of colorectal cancers (CRC).  In the present work we report that the novel di-tert-Bu phenol-based dual inhibitors DTPSAL, DTPBHZ, DTPINH, and DTPNHZ exhibit significant cytotoxicity against human CRC cell lines.  Mol. docking studies revealed a good fit of these compds. in the COX-2 and 5-LOX protein cavities.  The inhibitors show significant inhibition of COX-2 and 5-LOX activities and are effective against a panel of human colon cancer cell lines including HCA-7, HT-29, SW480 and intestinal Apc10.1 cells as well as the hyaluronan synthase-2 (Has2) enzyme over-expressing colon cancer cells, through inhibition of the Hyaluronan/CD44v6 cell survival pathway.  Western blot anal. and qRT-PCR analyses indicated that the di-tert-Bu phenol-based dual inhibitors reduce the expression of COX-2, 5-LOX, and CD44v6 in human colon cancer HCA-7 cells, while the combination of CD44v6shRNA and DTPSAL has an addnl. inhibitory effect on CD44v6 mRNA expression.  The synergistic inhibitory effect of Celecoxib and Licofelone on CD44v6 mRNA expression suggests that the present dual inhibitors down-regulate cyclooxygenase and lipoxygenase enzymes through CD44v6.  The compds. also exhibited enhanced antiproliferative potency compared to std. dual COX/LOX inhibitor, viz. Licofelone.  Importantly, the HA/CD44v6 antagonist CD44v6shRNA in combination with synthetic compds. had a sensitizing effect on the cancer cells which enhanced their antiproliferative potency, a finding which is crucial for the anti-proliferative potency of the novel synthetic di-tert-Bu phenol based dual COX-LOX inhibitors in colon cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKAW7Zh1L2ibVg90H21EOLACvtfcHk0ljh_bBypCvKpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWku7fK&md5=7a8a58a76b9ada38e7b4b8ef57c88dcd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DGhatak%26aufirst%3DS.%26aulast%3DVyas%26aufirst%3DA.%26aulast%3DMisra%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%26aulast%3DZambre%26aufirst%3DA.%26aulast%3DFresco%26aufirst%3DV.%2BM.%26aulast%3DMarkwald%26aufirst%3DR.%2BR.%26aulast%3DSwamy%26aufirst%3DK.%2BV.%26aulast%3DAfrasiabi%26aufirst%3DZ.%26aulast%3DChoudhury%26aufirst%3DA.%26aulast%3DKhetmalas%26aufirst%3DM.%26aulast%3DPadhye%26aufirst%3DS.%26atitle%3DNovel%2520di-tertiary-butyl%2520phenylhydrazones%2520as%2520dual%2520cyclooxygenase-2%252F5-lipoxygenase%2520inhibitors%253A%2520Synthesis%252C%2520COX%252FLOX%2520inhibition%252C%2520molecular%2520modeling%252C%2520and%2520insights%2520into%2520their%2520cytotoxicities%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D317%26epage%3D324%26doi%3D10.1016%2Fj.bmcl.2013.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keeton, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">4599</span>– <span class="NLM_lpage">4604</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmcl.2012.05.098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=22727640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4599-4604&author=L.+A.+Dakinauthor=M.+H.+Blockauthor=H.+Chenauthor=E.+Codeauthor=J.+E.+Dowlingauthor=X.+Fengauthor=A.+D.+Fergusonauthor=I.+Greenauthor=A.+W.+Hirdauthor=T.+Howardauthor=E.+K.+Keetonauthor=M.+L.+Lambauthor=P.+D.+Lyneauthor=H.+Pollardauthor=J.+Readauthor=A.+J.+Wuauthor=T.+Zhangauthor=X.+Zheng&title=Discovery+of+novel+benzylidene-1%2C3-thiazolidine-2%2C4-diones+as+potent+and+selective+inhibitors+of+the+PIM-1%2C+PIM-2%2C+and+PIM-3+protein+kinases&doi=10.1016%2Fj.bmcl.2012.05.098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases</span></div><div class="casAuthors">Dakin, Les A.; Block, Michael H.; Chen, Huawei; Code, Erin; Dowling, James E.; Feng, Xiaomei; Ferguson, Andrew D.; Green, Isabelle; Hird, Alexander W.; Howard, Tina; Keeton, Erika K.; Lamb, Michelle L.; Lyne, Paul D.; Pollard, Hannah; Read, Jon; Wu, Allan J.; Zhang, Tao; Zheng, Xiaolan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4599-4604</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel substituted benzylidene-1,3-thiazolidine-2,4-diones (TZDs) have been identified as potent and highly selective inhibitors of the PIM kinases.  The synthesis and SAR of these compds. are described, along with x-ray crystallog. of deriv. I, anti-proliferative, and selectivity data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLer-JRufp_rVg90H21EOLACvtfcHk0lhlgUGa6R5VwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D&md5=cd1570938d48864bb739d2ccc37888f2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.098%26sid%3Dliteratum%253Aachs%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DHird%26aufirst%3DA.%2BW.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DKeeton%26aufirst%3DE.%2BK.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520novel%2520benzylidene-1%252C3-thiazolidine-2%252C4-diones%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520PIM-1%252C%2520PIM-2%252C%2520and%2520PIM-3%2520protein%2520kinases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4599%26epage%3D4604%26doi%3D10.1016%2Fj.bmcl.2012.05.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorente, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantarjian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanathan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, E.</span></span> <span> </span><span class="NLM_article-title">Phase I studies of AZD1208, a proviral integration Moloney virus kinase inhibitor in solid and haematological cancers</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1038/s41416-018-0082-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fs41416-018-0082-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=29765150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps1Ggs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1425-1433&author=J.+Cortesauthor=K.+Tamuraauthor=D.+J.+DeAngeloauthor=J.+de+Bonoauthor=D.+Lorenteauthor=M.+Mindenauthor=G.+L.+Uyauthor=H.+Kantarjianauthor=L.+S.+Chenauthor=V.+Gandhiauthor=R.+Godinauthor=K.+Keatingauthor=K.+McEachernauthor=K.+Vishwanathanauthor=J.+E.+Peaseauthor=E.+Dean&title=Phase+I+studies+of+AZD1208%2C+a+proviral+integration+Moloney+virus+kinase+inhibitor+in+solid+and+haematological+cancers&doi=10.1038%2Fs41416-018-0082-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I studies of AZD1208, a proviral integration Moloney virus kinase inhibitor in solid and haematological cancers</span></div><div class="casAuthors">Cortes, Jorge; Tamura, Kenji; De Angelo, Daniel J.; de Bono, Johann; Lorente, David; Minden, Mark; Uy, Geoffrey L.; Kantarjian, Hagop; Chen, Lisa S.; Gandhi, Varsha; Godin, Robert; Keating, Karen; McEachern, Kristen; Vishwanathan, Karthick; Pease, Janet Elizabeth; Dean, Emma</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1425-1433</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Background: Proviral integration Moloney virus (PIM) kinases (PIM1, 2 and 3) are overexpressed in several tumor types and contribute to oncogenesis.  AZD1208 is a potent ATP-competitive PIM kinase inhibitor investigated in patients with recurrent or refractory acute myeloid leukemia (AML) or advanced solid tumors.  Methods: Two dose-escalation studies were performed to evaluate the safety and tolerability, and to define the max. tolerated dose (MTD), of AZD1208 in AML and solid tumors.  Secondary objectives were to evaluate the pharmacokinetics, pharmacodynamics (PD) and preliminary efficacy of AZD1208.  Results: Sixty-seven patients received treatment: 32 in the AML study over a 120-900 mg dose range, and 25 in the solid tumor study over a 120-800 mg dose range.  Nearly all patients (98.5%) in both studies experienced adverse events, mostly gastrointestinal (92.5%).  Dose-limiting toxicities included rash, fatigue and vomiting.  AZD1208 was not tolerated at 900 mg, and the protocol-defined MTD was not confirmed.  AZD1208 increased CYP3A4 activity after multiple dosing, resulting in increased drug clearance.  There were no clin. responses; PD anal. showed biol. activity of AZD1208.  Conclusions: Despite the lack of single-agent clin. efficacy with AZD1208, PIM kinase inhibition may hold potential as an anticancer treatment, perhaps in combination with other agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkB6HSFRtJRrVg90H21EOLACvtfcHk0lhlgUGa6R5VwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps1Ggs7o%253D&md5=a899b9509f95a01fa2b225c6ece82df7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41416-018-0082-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41416-018-0082-1%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DTamura%26aufirst%3DK.%26aulast%3DDeAngelo%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BBono%26aufirst%3DJ.%26aulast%3DLorente%26aufirst%3DD.%26aulast%3DMinden%26aufirst%3DM.%26aulast%3DUy%26aufirst%3DG.%2BL.%26aulast%3DKantarjian%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DL.%2BS.%26aulast%3DGandhi%26aufirst%3DV.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DKeating%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DK.%26aulast%3DVishwanathan%26aufirst%3DK.%26aulast%3DPease%26aufirst%3DJ.%2BE.%26aulast%3DDean%26aufirst%3DE.%26atitle%3DPhase%2520I%2520studies%2520of%2520AZD1208%252C%2520a%2520proviral%2520integration%2520Moloney%2520virus%2520kinase%2520inhibitor%2520in%2520solid%2520and%2520haematological%2520cancers%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2018%26volume%3D118%26spage%3D1425%26epage%3D1433%26doi%3D10.1038%2Fs41416-018-0082-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathre, S.</span>; <span class="NLM_string-name">Reddy, K. B.</span>; <span class="NLM_string-name">Ramya, V.</span>; <span class="NLM_string-name">Krishnan, H.</span>; <span class="NLM_string-name">Ghosh, A.</span>; <span class="NLM_string-name">Raghu, P.</span></span><span> </span><span class="NLM_article-title">Functional analysis of the biochemical activity of mammalian phosphatidylinositol 5 phosphate 4-kinase enzymes</span>.  <i>Biosci Rep.</i><span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume">39</span> ()<span class="refDoi"> DOI: 10.1042/BSR20182210</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1042%2FBSR20182210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=30718367" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&author=S.+Mathre&author=K.+B.+Reddy&author=V.+Ramya&author=H.+Krishnan&author=A.+Ghosh&author=P.+Raghu&title=Functional+analysis+of+the+biochemical+activity+of+mammalian+phosphatidylinositol+5+phosphate+4-kinase+enzymes&doi=10.1042%2FBSR20182210"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBSR20182210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20182210%26sid%3Dliteratum%253Aachs%26aulast%3DMathre%26aufirst%3DS.%26atitle%3DFunctional%2520analysis%2520of%2520the%2520biochemical%2520activity%2520of%2520mammalian%2520phosphatidylinositol%25205%2520phosphate%25204-kinase%2520enzymes%26jtitle%3DBiosci%2520Rep.%26date%3D2019%26volume%3D39%26doi%3D10.1042%2FBSR20182210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emson, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">Localization of phosphatidylinositol phosphate kinase II gamma in kidney to a membrane trafficking compartment within specialized cells of the nephron</span>. <i>Am. J. Physiol.: Renal Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">F1422</span>– <span class="NLM_lpage">F1430</span>, <span class="refDoi"> DOI: 10.1152/ajprenal.90310.2008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1152%2Fajprenal.90310.2008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=18753295" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2008&pages=F1422-F1430&author=J.+H.+Clarkeauthor=P.+C.+Emsonauthor=R.+F.+Irvine&title=Localization+of+phosphatidylinositol+phosphate+kinase+II+gamma+in+kidney+to+a+membrane+trafficking+compartment+within+specialized+cells+of+the+nephron&doi=10.1152%2Fajprenal.90310.2008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1152%2Fajprenal.90310.2008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajprenal.90310.2008%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DEmson%26aufirst%3DP.%2BC.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DLocalization%2520of%2520phosphatidylinositol%2520phosphate%2520kinase%2520II%2520gamma%2520in%2520kidney%2520to%2520a%2520membrane%2520trafficking%2520compartment%2520within%2520specialized%2520cells%2520of%2520the%2520nephron%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Renal%2520Physiol.%26date%3D2008%26volume%3D295%26spage%3DF1422%26epage%3DF1430%26doi%3D10.1152%2Fajprenal.90310.2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0liDR-Cjs0VIEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lhjiqJEC7qOHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Exploration of type II binding mode: A privileged approach for kinase inhibitor focused drug discovery?</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+A+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lhjiqJEC7qOHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520A%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massagué, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis, R. R.</span></span> <span> </span><span class="NLM_article-title">The logic of TGFβ signaling</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">2811</span>– <span class="NLM_lpage">2820</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2006.04.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.febslet.2006.04.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=16678165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD28XltVKntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2006&pages=2811-2820&author=J.+Massagu%C3%A9author=R.+R.+Gomis&title=The+logic+of+TGF%CE%B2+signaling&doi=10.1016%2Fj.febslet.2006.04.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The logic of TGFβ signaling</span></div><div class="casAuthors">Massague, Joan; Gomis, Roger R.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2811-2820</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The identification of the TGFβ cytokine signaling pathway, including membrane receptor serine/threonine kinases and Smad transcription factors as their substrates, has allowed the delineation of a process for conversion of these signals into programs of gene activation and repression that underlie crit. cell fate and developmental decisions.  The deconstruction of one of these responses - the cell cycle arrest response - into its elemental mol. parts has shed light into the mechanisms used by tumors to evade surveillance and cause metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE0ooHqQCng7Vg90H21EOLACvtfcHk0lgn47P0ah-Otg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltVKntrg%253D&md5=ed5bb318fe01991b98c6e04e66b365e1</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2006.04.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2006.04.033%26sid%3Dliteratum%253Aachs%26aulast%3DMassagu%25C3%25A9%26aufirst%3DJ.%26aulast%3DGomis%26aufirst%3DR.%2BR.%26atitle%3DThe%2520logic%2520of%2520TGF%25CE%25B2%2520signaling%26jtitle%3DFEBS%2520Lett.%26date%3D2006%26volume%3D580%26spage%3D2811%26epage%3D2820%26doi%3D10.1016%2Fj.febslet.2006.04.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, T.-A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ream-Robinson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carethers, J. M.</span></span> <span> </span><span class="NLM_article-title">Mutation rates of TGFBR2 and ACVR2 coding microsatellites in human cells with defective DNA mismatch repair</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e3463</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0003463</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1371%2Fjournal.pone.0003463" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=18941508" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&author=H.+Chungauthor=D.+J.+Youngauthor=C.+G.+Lopezauthor=T.-A.+T.+Leauthor=J.+K.+Leeauthor=D.+Ream-Robinsonauthor=S.+C.+Huangauthor=J.+M.+Carethers&title=Mutation+rates+of+TGFBR2+and+ACVR2+coding+microsatellites+in+human+cells+with+defective+DNA+mismatch+repair&doi=10.1371%2Fjournal.pone.0003463"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0003463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0003463%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DD.%2BJ.%26aulast%3DLopez%26aufirst%3DC.%2BG.%26aulast%3DLe%26aufirst%3DT.-A.%2BT.%26aulast%3DLee%26aufirst%3DJ.%2BK.%26aulast%3DReam-Robinson%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DS.%2BC.%26aulast%3DCarethers%26aufirst%3DJ.%2BM.%26atitle%3DMutation%2520rates%2520of%2520TGFBR2%2520and%2520ACVR2%2520coding%2520microsatellites%2520in%2520human%2520cells%2520with%2520defective%2520DNA%2520mismatch%2520repair%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26doi%3D10.1371%2Fjournal.pone.0003463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrera, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiorino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocanegra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carethers, J. M.</span></span> <span> </span><span class="NLM_article-title"><i>Activin type 2 receptor</i> restoration in MSI-H colon cancer suppresses growth and enhances migration with activin</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2006.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1053%2Fj.gastro.2006.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=17258738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlKisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2007&pages=633-644&author=B.+H.+Jungauthor=S.+E.+Beckauthor=J.+Cabralauthor=E.+Chauauthor=B.+L.+Cabreraauthor=A.+Fiorinoauthor=E.+J.+Smithauthor=M.+Bocanegraauthor=J.+M.+Carethers&title=Activin+type+2+receptor+restoration+in+MSI-H+colon+cancer+suppresses+growth+and+enhances+migration+with+activin&doi=10.1053%2Fj.gastro.2006.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activin type 2 receptor restoration in MSI-H colon cancer suppresses growth and enhances migration with activin</span></div><div class="casAuthors">Jung, Barbara H.; Beck, Stayce E.; Cabral, Jennifer; Chau, Eddy; Cabrera, Betty L.; Fiorino, Antonio; Smith, E. Julieta; Bocanegra, Melanie; Carethers, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">633-644</span>CODEN:
                <span class="NLM_cas:coden">GASTAB</span>;
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Background & Aims: Colon cancers with high-frequency microsatellite instability (MSI-H) develop frameshift mutations in tumor suppressors as part of their pathogenesis.  ACVR2 is mutated at its exon 10 polyadenine tract in >80% of MSI-H colon cancers, coinciding with loss of protein.  ACVR2 transmits the growth effects of activin via phosphorylation of SMAD proteins to affect gene transcription.  The functional effect of activin in colon cancers has not been studied.  We developed and characterized a cell model in which we studied how activin signaling affects growth.  Methods: hMLH1 and ACVR2 mutant HCT116 cells were previously stably transferred with chromosome 2 (HCT116+chr2), restoring a single regulated copy of wild-type ACVR2 but not hMLH1.  Both HCT116+chr2 and parental HCT116 cells (as well as HEC59 and ACVR2 and hMSH2 complemented HEC59+chr2 cells) were assessed for genetic complementation and biol. function.  Results: HCT116+chr2 cells and HEC59+chr2 cells, but not ACVR2-mutant HCT116 or HEC59 cells, acquired wild-type ACVR2 as well as expression of ACVR2 wild-type mRNA.  Complemented ACVR2 protein complexed with ACVR1 with activin treatment, generating nuclear phosphoSMAD2 and activin-specific gene transcription.  ACVR2-restored cells showed decreased growth and reduced S phase but increased cellular migration following activin treatment.  ACVR2 small interfering RNA reversed these effects in complemented cells.  Conclusions: ACVR2-complemented MSI-H colon cancers restore activin-SMAD signaling, decrease growth, and slow their cell cycle following ligand stimulation but show increased cellular migration.  Activin is growth suppressive and enhances migration similar to transforming growth factor β in colon cancer, indicating that abrogation of the effects of activin contribute to the pathogenesis of MSI-H colon cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2J5XsKYrCrLVg90H21EOLACvtfcHk0lgn47P0ah-Otg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlKisr0%253D&md5=624248c5746fd922b0e8724f15534195</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2006.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2006.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DB.%2BH.%26aulast%3DBeck%26aufirst%3DS.%2BE.%26aulast%3DCabral%26aufirst%3DJ.%26aulast%3DChau%26aufirst%3DE.%26aulast%3DCabrera%26aufirst%3DB.%2BL.%26aulast%3DFiorino%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DE.%2BJ.%26aulast%3DBocanegra%26aufirst%3DM.%26aulast%3DCarethers%26aufirst%3DJ.%2BM.%26atitle%3DActivin%2520type%25202%2520receptor%2520restoration%2520in%2520MSI-H%2520colon%2520cancer%2520suppresses%2520growth%2520and%2520enhances%2520migration%2520with%2520activin%26jtitle%3DGastroenterology%26date%3D2007%26volume%3D132%26spage%3D633%26epage%3D644%26doi%3D10.1053%2Fj.gastro.2006.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallal, L. A.</span></span> <span> </span><span class="NLM_article-title">High-throughput cellular thermal shift assays in research and drug discovery</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1177/2472555219877183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1177%2F2472555219877183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=31566060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A280%3ADC%252BB3Mnjt1Clsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=137-147&author=M.+J.+Hendersonauthor=M.+A.+Holbertauthor=A.+Simeonovauthor=L.+A.+Kallal&title=High-throughput+cellular+thermal+shift+assays+in+research+and+drug+discovery&doi=10.1177%2F2472555219877183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Cellular Thermal Shift Assays in Research and Drug Discovery</span></div><div class="casAuthors">Henderson Mark J; Simeonov Anton; Holbert Marc A; Kallal Lorena A</div><div class="citationInfo"><span class="NLM_cas:title">SLAS discovery : advancing life sciences R & D</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">137-147</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Thermal shift assays (TSAs) can reveal changes in protein structure, due to a resultant change in protein thermal stability.  Since proteins are often stabilized upon binding of ligand molecules, these assays can provide a readout for protein target engagement.  TSA has traditionally been applied using purified proteins and more recently has been extended to study target engagement in cellular environments with the emergence of cellular thermal shift assays (CETSAs).  The utility of CETSA in confirming molecular interaction with targets in a more native context, and the desire to apply this technique more broadly, has fueled the emergence of higher-throughput techniques for CETSA (HT-CETSA).  Recent studies have demonstrated that HT-CETSA can be performed in standard 96-, 384-, and 1536-well microtiter plate formats using methods such as beta-galactosidase and NanoLuciferase reporters and AlphaLISA assays.  HT-CETSA methods can be used to select and characterize compounds from high-throughput screens and to prioritize compounds in lead optimization by facilitating dose-response experiments.  In conjunction with cellular and biochemical activity assays for targets, HT-CETSA can be a valuable addition to the suite of assays available to characterize molecules of interest.  Despite the successes in implementing HT-CETSA for a diverse set of targets, caveats and challenges must also be recognized to avoid overinterpretation of results.  Here, we review the current landscape of HT-CETSA and discuss the methodologies, practical considerations, challenges, and applications of this approach in research and drug discovery.  Additionally, a perspective on potential future directions for the technology is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEx69ZUMaiTMjl5L8hwNX0fW6udTcc2eaZNLuQMJiOcrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mnjt1Clsg%253D%253D&md5=39f58608a89ce9863098582c63d0a338</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1177%2F2472555219877183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555219877183%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DHolbert%26aufirst%3DM.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DKallal%26aufirst%3DL.%2BA.%26atitle%3DHigh-throughput%2520cellular%2520thermal%2520shift%2520assays%2520in%2520research%2520and%2520drug%2520discovery%26jtitle%3DSLAS%2520Discovery%26date%3D2020%26volume%3D25%26spage%3D137%26epage%3D147%26doi%3D10.1177%2F2472555219877183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droubi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giudici, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. F.</span></span> <span> </span><span class="NLM_article-title">Exploring phosphatidylinositol 5-phosphate 4-kinase function</span>. <i>Adv. Biol. Regul.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1016/j.jbior.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.jbior.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=25311266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslansr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2015&pages=193-202&author=S.+J.+Bulleyauthor=J.+H.+Clarkeauthor=A.+Droubiauthor=M.-L.+Giudiciauthor=R.+F.+Irvine&title=Exploring+phosphatidylinositol+5-phosphate+4-kinase+function&doi=10.1016%2Fj.jbior.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring phosphatidylinositol 5-phosphate 4-kinase function</span></div><div class="casAuthors">Bulley, Simon J.; Clarke, Jonathan H.; Droubi, Alaa; Giudici, Maria-Luisa; Irvine, Robin F.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Biological Regulation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">193-202</span>CODEN:
                <span class="NLM_cas:coden">ABRDE5</span>;
        ISSN:<span class="NLM_cas:issn">2212-4934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The family of phosphatidylinositol 5-phosphate 4-kinases (PI5P4Ks) is emerging from a comparative backwater in inositide signalling into the mainstream, as is their substrate, phosphatidylinositol 5-phosphate (PI5P).  Here we review some of the key questions about the PI5P4Ks, their localization, interaction, and regulation and also we summarize our current understanding of how PI5P is synthesized and what its cellular functions might be.  Finally, some of the evidence for the involvement of PI5P4Ks in pathol. is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCTy9MhIW4aLVg90H21EOLACvtfcHk0lg1I89q3q_2Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslansr7O&md5=af7bd2595850a46a98019a0d6cd8b5ab</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jbior.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbior.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DBulley%26aufirst%3DS.%2BJ.%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DDroubi%26aufirst%3DA.%26aulast%3DGiudici%26aufirst%3DM.-L.%26aulast%3DIrvine%26aufirst%3DR.%2BF.%26atitle%3DExploring%2520phosphatidylinositol%25205-phosphate%25204-kinase%2520function%26jtitle%3DAdv.%2520Biol.%2520Regul.%26date%3D2015%26volume%3D57%26spage%3D193%26epage%3D202%26doi%3D10.1016%2Fj.jbior.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marklund, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochberg, G. K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benesch, J. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. V.</span></span> <span> </span><span class="NLM_article-title">Bayesian deconvolution of mass and ion mobility spectra: From binary interactions to polydisperse ensembles</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">4370</span>– <span class="NLM_lpage">4376</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.5b00140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.5b00140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=4370-4376&author=M.+T.+Martyauthor=A.+J.+Baldwinauthor=E.+G.+Marklundauthor=G.+K.+A.+Hochbergauthor=J.+L.+P.+Beneschauthor=C.+V.+Robinson&title=Bayesian+deconvolution+of+mass+and+ion+mobility+spectra%3A+From+binary+interactions+to+polydisperse+ensembles&doi=10.1021%2Facs.analchem.5b00140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Bayesian Deconvolution of Mass and Ion Mobility Spectra: From Binary Interactions to Polydisperse Ensembles</span></div><div class="casAuthors">Marty, Michael T.; Baldwin, Andrew J.; Marklund, Erik G.; Hochberg, Georg K. A.; Benesch, Justin L. P.; Robinson, Carol V.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4370-4376</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interpretation of mass spectra is challenging because they report a ratio of 2 phys. quantities, mass and charge, which may each have multiple components that overlap in m/z.  Previous approaches to disentangling the 2 have focused on peak assignment or fitting.  However, the former struggle with complex spectra, and the latter are generally computationally intensive and may require substantial manual intervention.  The authors propose a new data anal. approach that employs a Bayesian framework to sep. the mass and charge dimensions.  From this approach, the authors developed UniDec (Universal Deconvolution), software that provides a rapid, robust, and flexible deconvolution of mass spectra and ion mobility-mass spectra with minimal user intervention.  Incorporation of the charge-state distribution in the Bayesian prior probabilities provides sepn. of the m/z spectrum into its phys. mass and charge components.  The authors have evaluated approach using systems of increasing complexity, enabling one to deduce lipid binding to membrane proteins, to probe the dynamics of subunit exchange reactions, and to characterize polydispersity in both protein assemblies and lipoprotein Nanodiscs.  The general utility of approach will greatly facilitate anal. of ion mobility and mass spectra.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEW75ktwO86rVg90H21EOLACvtfcHk0lh4xH1SaekmMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D&md5=d2d1272237752ba52cdf2a976608f6bf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.5b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.5b00140%26sid%3Dliteratum%253Aachs%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DBaldwin%26aufirst%3DA.%2BJ.%26aulast%3DMarklund%26aufirst%3DE.%2BG.%26aulast%3DHochberg%26aufirst%3DG.%2BK.%2BA.%26aulast%3DBenesch%26aufirst%3DJ.%2BL.%2BP.%26aulast%3DRobinson%26aufirst%3DC.%2BV.%26atitle%3DBayesian%2520deconvolution%2520of%2520mass%2520and%2520ion%2520mobility%2520spectra%253A%2520From%2520binary%2520interactions%2520to%2520polydisperse%2520ensembles%26jtitle%3DAnal.%2520Chem.%26date%3D2015%26volume%3D87%26spage%3D4370%26epage%3D4376%26doi%3D10.1021%2Facs.analchem.5b00140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Predictive models of aqueous solubility of organic compounds built on a large dataset of high integrity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3110</span>– <span class="NLM_lpage">3114</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.05.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmc.2019.05.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=31176566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFClsrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=3110-3114&author=H.+Sunauthor=P.+Shahauthor=K.+Nguyenauthor=K.+R.+Yuauthor=E.+Kernsauthor=M.+Kabirauthor=Y.+Wangauthor=X.+Xu&title=Predictive+models+of+aqueous+solubility+of+organic+compounds+built+on+a+large+dataset+of+high+integrity&doi=10.1016%2Fj.bmc.2019.05.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive models of aqueous solubility of organic compounds built on A large dataset of high integrity</span></div><div class="casAuthors">Sun, Hongmao; Shah, Pranav; Nguyen, Kimloan; Yu, Katherine; Kerns, Ed; Kabir, Md; Wang, Yuhong; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3110-3114</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aq. soly. is one of the most important properties in drug discovery, as it has profound impact on various drug properties, including biol. activity, pharmacokinetics (PK), toxicity, and in vivo efficacy.  Both kinetic and thermodn. solubilities are detd. during different stages of drug discovery and development.  Since kinetic soly. is more relevant in preclin. drug discovery research, esp. during the structure optimization process, the authors have developed predictive models for kinetic soly. with inhouse data generated from 11,780 compds. collected from over 200 NCATS intramural research projects.  This represents one of the largest kinetic soly. datasets of high quality and integrity.  Based on the customized atom type descriptors, the support vector classification (SVC) models were trained on 80% of the whole dataset, and exhibited high predictive performance for estg. the soly. of the remaining 20% compds. within the test set.  The values of the area under the receiver operating characteristic curve (AUC-ROC) for the compds. in the test sets reached 0.93 and 0.91, when the threshold for insol. compds. was set to 10 and 50 μg/mL, resp.  The predictive models of aq. soly. can be used to identify insol. compds. in drug discovery pipeline, provide design ideas for improving soly. by analyzing the atom types assocd. with poor soly. and prioritize compd. libraries to be purchased or synthesized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfBZqnazp8KLVg90H21EOLACvtfcHk0lh4xH1SaekmMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFClsrnN&md5=8981617a93473ab5bc1112fc96e2c9fa</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.05.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.05.037%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DKabir%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DPredictive%2520models%2520of%2520aqueous%2520solubility%2520of%2520organic%2520compounds%2520built%2520on%2520a%2520large%2520dataset%2520of%2520high%2520integrity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D3110%26epage%3D3114%26doi%3D10.1016%2Fj.bmc.2019.05.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">An automated high-throughput metabolic stability assay using an integrated high-resolution accurate mass method and automated data analysis software</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1653</span>– <span class="NLM_lpage">1661</span>, <span class="refDoi"> DOI: 10.1124/dmd.116.072017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1124%2Fdmd.116.072017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=27417180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1653-1661&author=P.+Shahauthor=E.+Kernsauthor=D.-T.+Nguyenauthor=S.+Obachauthor=A.+Q.+Wangauthor=A.+Zakharovauthor=J.+McKewauthor=A.+Simeonovauthor=C.+E.+C.+A.+Hopauthor=X.+Xu&title=An+automated+high-throughput+metabolic+stability+assay+using+an+integrated+high-resolution+accurate+mass+method+and+automated+data+analysis+software&doi=10.1124%2Fdmd.116.072017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An automated high-throughput metabolic stability assay using an integrated high-resolution accurate mass method and automated data analysis software</span></div><div class="casAuthors">Shah, Pranav; Kerns, Edward; Nguyen, Dac-Trung; Scott, Obach R.; Wang, Amy Q.; Zakharov, Alexey; McKew, John; Simeonov, Anton; Hop, Cornelis E. C. A.; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1653-1661</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Advancement of in silico tools would be enabled by the availability of data for metabolic reaction rates and intrinsic clearance (CLint) of a diverse compd. structure data set by specific metabolic enzymes.  Our goal is to measure CLint for a large set of compds. with each major human cytochrome P 450 (P 450) isoenzyme.  To achieve our goal, it is of utmost importance to develop an automated, robust, sensitive, high-throughput metabolic stability assay that can efficiently handle a large vol. of compd. sets.  The substrate depletion method [in vitro half-life (t1/2) method] was chosen to det. CLint.  The assay (384-well format) consisted of three parts: (1) a robotic system for incubation and sample cleanup; (2) two different integrated, ultraperformance liq. chromatog./ mass spectrometry (UPLC/MS) platforms to det. the percent remaining of parent compd., and (3) an automated data anal. system.  The CYP3A4 assay was evaluated using two long t1/2 compds., carbamazepine and antipyrine (t1/2 > 30 min); one moderate t1/2 compd., ketoconazole (10 < t1/2 < 30 min); and two short t1/2 compds., loperamide and buspirone (t < 10 min).  Interday and intraday precision and accuracy of the assay were within acceptable range (∼12%) for the linear range obsd.  Using this assay, CYP3A4 CLint and t1/2 values for more than 3000 compds. were measured.  This high-throughput, automated, and robust assay allows for rapid metabolic stability screening of large compd. sets and enables advanced computational modeling for individual human P 450 isoenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos6dcRqLvacbVg90H21EOLACvtfcHk0lh4xH1SaekmMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK&md5=92b4f23376d1cffa70c6fdc209b5569c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.072017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.072017%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DP.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DD.-T.%26aulast%3DObach%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DMcKew%26aufirst%3DJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DAn%2520automated%2520high-throughput%2520metabolic%2520stability%2520assay%2520using%2520an%2520integrated%2520high-resolution%2520accurate%2520mass%2520method%2520and%2520automated%2520data%2520analysis%2520software%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1653%26epage%3D1661%26doi%3D10.1124%2Fdmd.116.072017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly predictive and interpretable models for PAMPA permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+predictive+and+interpretable+models+for+PAMPA+permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0li_nDpXG1BnXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520predictive%2520and%2520interpretable%2520models%2520for%2520PAMPA%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9','PDB','6UX9'); return false;">PDB: 6UX9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZZ','PDB','3WZZ'); return false;">PDB: 3WZZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GK9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GK9','PDB','2GK9'); return false;">PDB: 2GK9</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i83"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00227">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_25138"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00227?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00227</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Overview of CVM-05-002 analogs, full KINOMEscan data sets, lipophilicity and early ADME overview of CVM-05-002 and analogs, structures of reported covalent pan-PI5P4K inhibitors, plate-well series plot with mean %DMSO control, and complete results of high-throughput screen (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_001.pdf">jm0c00227_si_001.pdf (2.46 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00227/suppl_file/jm0c00227_si_002.csv">jm0c00227_si_002.csv (1.88 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB code for PI5P4Kα with bound CVM-05-002 is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UX9">6UX9</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00227&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00227%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00227" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677af624e8a83cb2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
