
<html lang="en"     class="pb-page"  data-request-id="f5174a47-2a59-4aa5-abb7-26f7e3be8e3f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.7b00816;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2017.60.issue-18;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Small Molecules Targeting the Synergy of Cardiac Transcription Factors GATA4 and NKX2-5" /></meta><meta name="dc.Creator" content="Mika J.  Välimäki" /></meta><meta name="dc.Creator" content="Marja A.  Tölli" /></meta><meta name="dc.Creator" content="Sini M.  Kinnunen" /></meta><meta name="dc.Creator" content="Jani  Aro" /></meta><meta name="dc.Creator" content="Raisa  Serpi" /></meta><meta name="dc.Creator" content="Lotta  Pohjolainen" /></meta><meta name="dc.Creator" content="Virpi  Talman" /></meta><meta name="dc.Creator" content="Antti  Poso" /></meta><meta name="dc.Creator" content="Heikki J.  Ruskoaho" /></meta><meta name="dc.Description" content="Transcription factors are pivotal regulators of gene transcription, and many diseases are associated with the deregulation of transcriptional networks. In the heart, the transcription factors GATA4..." /></meta><meta name="Description" content="Transcription factors are pivotal regulators of gene transcription, and many diseases are associated with the deregulation of transcriptional networks. In the heart, the transcription factors GATA4..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 31, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00816" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00816" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00816" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00816" /></link>
        
    
    

<title>Discovery of Small Molecules Targeting the Synergy of Cardiac Transcription Factors GATA4 and NKX2-5 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00816" /></meta><meta property="og:title" content="Discovery of Small Molecules Targeting the Synergy of Cardiac Transcription Factors GATA4 and NKX2-5" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0018.jpeg" /></meta><meta property="og:description" content="Transcription factors are pivotal regulators of gene transcription, and many diseases are associated with the deregulation of transcriptional networks. In the heart, the transcription factors GATA4 and NKX2-5 are required for cardiogenesis. GATA4 and NKX2-5 interact physically, and the activation of GATA4, in cooperation with NKX2-5, is essential for stretch-induced cardiomyocyte hypertrophy. Here, we report the identification of four small molecule families that either inhibit or enhance the GATA4–NKX2-5 transcriptional synergy. A fragment-based screening, reporter gene assay, and pharmacophore search were utilized for the small molecule screening, identification, and optimization. The compounds modulated the hypertrophic agonist-induced cardiac gene expression. The most potent hit compound, N-[4-(diethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide (3, IC50 = 3 μM), exhibited no activity on the protein kinases involved in the regulation of GATA4 phosphorylation. The identified and chemically and biologically characterized active compound, and its derivatives may provide a novel class of small molecules for modulating heart regeneration." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00816"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00816">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00816&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00816&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00816&amp;href=/doi/10.1021/acs.jmedchem.7b00816" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 7781-7798</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00604" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00825" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Small Molecules Targeting the Synergy of Cardiac Transcription Factors GATA4 and NKX2-5</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Mika J. Välimäki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mika J. Välimäki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</div><div class="loa-info-affiliations-info">Research Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mika+J.++V%C3%A4lim%C3%A4ki">Mika J. Välimäki</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9687-8197" title="Orcid link">http://orcid.org/0000-0002-9687-8197</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marja A. Tölli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marja A. Tölli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marja+A.++T%C3%B6lli">Marja A. Tölli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sini M. Kinnunen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sini M. Kinnunen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</div><div class="loa-info-affiliations-info">Research Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sini+M.++Kinnunen">Sini M. Kinnunen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jani Aro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jani Aro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jani++Aro">Jani Aro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Raisa Serpi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Raisa Serpi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Raisa++Serpi">Raisa Serpi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lotta Pohjolainen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lotta Pohjolainen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lotta++Pohjolainen">Lotta Pohjolainen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Virpi Talman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Virpi Talman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Virpi++Talman">Virpi Talman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antti Poso</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antti Poso</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Health Sciences, School of Pharmacy, University of Eastern Finland, Kuopio FI-70211, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antti++Poso">Antti Poso</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Heikki J. Ruskoaho</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heikki J. Ruskoaho</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</div><div class="loa-info-affiliations-info">Research Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +358-50-4480722, E-mail: <a href="/cdn-cgi/l/email-protection#c0a8a5a9ababa9eeb2b5b3abafa1a8af80a8a5acb3a9aeaba9eea6a9"><span class="__cf_email__" data-cfemail="f39b969a98989add818680989c929b9cb39b969f809a9d989add959a">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heikki+J.++Ruskoaho">Heikki J. Ruskoaho</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00816&amp;href=/doi/10.1021%2Facs.jmedchem.7b00816" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 7781–7798</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 31, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 June 2017</li><li><span class="item_label"><b>Published</b> online</span>31 August 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 September 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00816" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00816</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00816"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3077</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">27</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00816" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Small Molecules Targeting the Synergy of Cardiac Transcription Factors GATA4 and NKX2-5&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mika&quot;,&quot;last_name&quot;:&quot;J. Välimäki&quot;},{&quot;first_name&quot;:&quot;Marja&quot;,&quot;last_name&quot;:&quot;A. Tölli&quot;},{&quot;first_name&quot;:&quot;Sini&quot;,&quot;last_name&quot;:&quot;M. Kinnunen&quot;},{&quot;first_name&quot;:&quot;Jani&quot;,&quot;last_name&quot;:&quot;Aro&quot;},{&quot;first_name&quot;:&quot;Raisa&quot;,&quot;last_name&quot;:&quot;Serpi&quot;},{&quot;first_name&quot;:&quot;Lotta&quot;,&quot;last_name&quot;:&quot;Pohjolainen&quot;},{&quot;first_name&quot;:&quot;Virpi&quot;,&quot;last_name&quot;:&quot;Talman&quot;},{&quot;first_name&quot;:&quot;Antti&quot;,&quot;last_name&quot;:&quot;Poso&quot;},{&quot;first_name&quot;:&quot;Heikki&quot;,&quot;last_name&quot;:&quot;J. Ruskoaho&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;7781-7798&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00816&quot;},&quot;abstract&quot;:&quot;Transcription factors are pivotal regulators of gene transcription, and many diseases are associated with the deregulation of transcriptional networks. In the heart, the transcription factors GATA4 and NKX2-5 are required for cardiogenesis. GATA4 and NKX2-5 interact physically, and the activation of GATA4, in cooperation with NKX2-5, is essential for stretch-induced cardiomyocyte hypertrophy. Here, we report the identification of four small molecule families that either inhibit or enhance the GATA4–NKX2-5 transcriptional synergy. A fragment-based screening, reporter gene assay, and pharmacophore search were utilized for the small molecule screening, identification, and optimization. The compounds modulated the hypertrophic agonist-induced cardiac gene expression. The most potent hit compound, N-[4-(diethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide (3, IC50 = 3 μM), exhibited no activity on the protein kinases involved in the regulation of GATA4 phosphorylation. The identified and chemically and bi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00816&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00816" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00816&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00816" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00816&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00816" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00816&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00816&amp;href=/doi/10.1021/acs.jmedchem.7b00816" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00816" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00816" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00816%26sid%3Dliteratum%253Aachs%26pmid%3D28858485%26genre%3Darticle%26aulast%3DV%25C3%25A4lim%25C3%25A4ki%26date%3D2017%26atitle%3DDiscovery%2Bof%2BSmall%2BMolecules%2BTargeting%2Bthe%2BSynergy%2Bof%2BCardiac%2BTranscription%2BFactors%2BGATA4%2Band%2BNKX2-5%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D18%26spage%3D7781%26epage%3D7798%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290677" title="Toxicological synergy">Toxicological synergy</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/jmcmar.2017.60.issue-18/20170928/jmcmar.2017.60.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Transcription factors are pivotal regulators of gene transcription, and many diseases are associated with the deregulation of transcriptional networks. In the heart, the transcription factors GATA4 and NKX2-5 are required for cardiogenesis. GATA4 and NKX2-5 interact physically, and the activation of GATA4, in cooperation with NKX2-5, is essential for stretch-induced cardiomyocyte hypertrophy. Here, we report the identification of four small molecule families that either inhibit or enhance the GATA4–NKX2-5 transcriptional synergy. A fragment-based screening, reporter gene assay, and pharmacophore search were utilized for the small molecule screening, identification, and optimization. The compounds modulated the hypertrophic agonist-induced cardiac gene expression. The most potent hit compound, <i>N</i>-[4-(diethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide (<b>3</b>, IC<sub>50</sub> = 3 μM), exhibited no activity on the protein kinases involved in the regulation of GATA4 phosphorylation. The identified and chemically and biologically characterized active compound, and its derivatives may provide a novel class of small molecules for modulating heart regeneration.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22421" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22421" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The regulation of gene expression depends on specific <i>cis</i>-regulatory sequences that are located in gene promoter regions. These DNA sequences are recognized by tissue-specific transcription factors (TFs) in a sequence-specific manner, and their combinatorial interactions underlie physiological responses to developmental programs.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> In the heart, developmental processes are controlled by an interplay among the master cardiac TFs, such as GATA4, NKX2-5, and TBX5.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> These TFs bind to the promoters/enhancers of downstream targets as homo- or heteromultimeric complexes to physically interact with and synergistically regulate their target genes. GATA4 is one of the earliest-expressed TFs during cardiac development, and its up-regulation is sufficient to induce cardiogenesis in embryonic stem cells and promote a cardiac cell fate in noncardiogenic cells.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In the adult heart, GATA4 is a critical regulator of cardiac regeneration<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and acts as a key transcriptional controller of numerous cardiac genes, including those encoding atrial natriuretic peptide (ANP),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> B-type natriuretic peptide (BNP),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> α-myosin heavy chain (α-MHC),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and β-MHC.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> GATA4 binds to specific DNA response elements with or without cofactors, e.g., NKX2-5, TBX5, myocyte-specific enhancer factor 2 (MEF2), and serum response factor (SRF).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Mutational studies have shown that the GATA consensus sites in combination with an NKX2-5 binding element are important for the mechanical stretch-activated BNP transcription and sarcomere reorganization,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> thus implicating the GATA4–NKX2-5 interaction for the response of cardiomyocytes to external stimuli, such as a hemodynamic overload due to high blood pressure and myocardial infarction.</div><div class="NLM_p last">GATA4 belongs to the GATA protein family (GATA1–6) and is characterized by <i>WGATAR</i>-recognizing zinc-finger domains that mediate DNA binding and protein–protein interactions. Our previous work illustrated that the protein assembly between GATA4 and NKX2-5, which is a critical GATA4 cofactor, imposes a structural fold that is similar to that of the evolutionally conserved DNA binding domains of nuclear receptors.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The significance of this particular protein–protein complex has been further validated by the co-occurrence of GATA4 and NKX2-5 in the binding site locations.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In addition to protein–protein interactions, the function of GATA4 is modulated through post-translational modifications. An increased GATA4 transcriptional activity has been shown in luciferase reporter assays with phosphorylated,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> acetylated,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> or sumoylated GATA4 protein.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Therefore, the functional modulation of the GATA4 protein or the GATA4–NKX2-5 interaction could represent an innovative therapeutic approach for cardiac regeneration<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> or other pathophysiological conditions.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> However, currently, there are no small molecule inhibitors or activators of GATA4 function except for one report describing inhibitors of GATA4 binding to DNA.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Here, we utilized a fragment-based screening, reporter gene assay based optimization, and pharmacophore search to identify small molecules that modify the synergy between the transcription factors GATA4 and NKX2-5 in vitro.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37070" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37070" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Primary Screening</h3><div class="NLM_p">Previously, we conducted a mutagenic analysis of the second zinc finger domain to elucidate the structural basis of the GATA4–NKX2-5 interaction.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> These studies identified five highly conserved amino acids in the second zinc finger (N272, R283, Q274, K299) and its C-terminal extension (R319) that are critical for the physical and functional interaction with the third α helix of the NKX2-5 homeodomain. An integration of the experimental data with computational modeling suggested that the topology of the GATA4–NKX2-5 interaction resembles that of nuclear receptors.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In agreement with those observations, GATA4 has a synergistic effect with NKX2-5 on promoters that contain multiple NKX2-5 high affinity response elements (NKEs).<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> To evaluate the effect of the compounds on the functional interaction between GATA4 and NKX2-5, we developed a luciferase reporter gene assay specifically for the GATA4–NKX2-5 transcriptional synergy.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In this assay, mammalian COS-1 cells were transfected with the protein expression vectors pMT2-GATA4, pMT2-NKX2-5, both pMT2-GATA4 and pMT2-NKX2-5, or the empty control vector pMT2. The transfection efficiencies of the vectors in the COS-1 cells were verified by defining the protein levels of GATA4 and NKX2-5 by Western blotting.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Typically, several limitations are associated with the use of luciferase reporter assays.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The tested compounds may interfere with the luciferases and decrease the assay signal, resulting in false positives. In addition, the compounds may aggregate, have redox behavior, enzymatically inhibit the reporter, or form complexes that interfere with the fluorescence. Therefore, the positive findings obtained here with the primary screening assay were further confirmed using bioassays. As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, the NKX2-5 protein activates the construct containing three high affinity NKX2-5 binding sites alone and synergistically with GATA4, while the GATA4 protein alone had only a negligible effect.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Cell-based reporter gene assay used for the screening of compound libraries targeting the GATA4–NKX2-5 transcriptional synergy. COS-1 cells were transfected with the 3xNKE-luciferase reporter construct and GATA4 and NKX2-5 expression plasmids and treated with small molecules for 24 h. The cells were lysed, and the luciferase reporter gene activity was measured by a luminometer. (A) Compounds <b>3</b>–<b>6</b> had a significant inhibitory effect on the GATA4–NKX2-5 transcriptional synergy, while compound <b>7</b> enhanced the GATA4–NKX2-5 transcriptional synergy. (B) Compound <b>3</b> dose-dependently inhibited the GATA4–NKX2-5 transcriptional synergy in the COS-1 cells. The data are shown as the mean ± SD, <i>n</i> = 2–9: (∗∗) <i>p</i> < 0.01, (∗∗∗) <i>p</i> < 0.001 vs vehicle treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Initially, the compounds selected by the virtual screening for the modulation of protein–protein interaction were tested using an immunoprecipitation screening method, which revealed 15 weak inhibitory compounds (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). However, the identified compounds were shown to be false positives in the luciferase reporter assay and confirmatory bioassays, and subsequently, this approach was discontinued.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Overall Flowchart Illustrating the Process of Screening and Validating the GATA4–NKX2-5 Synergy Modulators<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>BNP, B-type natriuretic peptide; GPCR, G-protein-coupled receptor.</p></p></figure><div class="NLM_p">An in silico analysis focusing on the DNA binding site in the C-terminal zinc finger of GATA4 was carried out using a collection of 39 randomly placed molecular fragments, followed by energy minimization to reveal and rank the interaction potentials of the fragments inside the implied binding cavity. The compiled virtual assignments at the DNA binding site with multiple copies of simultaneous chemical probes<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> suggested a pronounced electrostatic interaction among the rigid fragment compounds with a negative charge/acceptor and aromatic features. On the basis of these theoretical annotations, a cherry-picked collection of 40 fragment compounds was purchased and tested in vitro using the GATA4–NKX2-5 luciferase reporter gene assay. These experiments identified two compounds that inhibited the transcriptional synergy, i.e., 3,4-diphenyl-1,2,5-oxadiazole (compound <b>1</b>) and 2-perfluorophenyl-3-phenylacrylic acid (compound <b>2</b>), at a concentration of 50 μM (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Chronological Workflow of the Hit Compound Search and Optimization Process Initiated from the Active Fragments<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Scheme illustrates the molecular structure, compound code numbers, and corresponding IC<sub>50</sub> values in the GATA4–NKX2-5 luciferase reporter gene assay.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Hit Optimization</h3><div class="NLM_p">A luciferase reporter gene assay based optimization of the initial fragment compounds was achieved by introducing sequential structural modifications and fragment growth to uncover compounds with activity on the GATA4–NKX2-5 transcriptional synergy. First, variable substituents were inserted into the three-ring core of fragment <b>1</b>. The compounds with the para-substituted six-member ring were the most effective. Second, different linkers with variable bond lengths and chemical compositions were inserted between the six- and five-member rings. The number of heavy atoms between the aromatic rings of the compounds ranged from zero to three. The most potent compound according to the luciferase reporter gene assay, at a concentration of 10 μM, was <i>N</i>-[4-(diethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide (<b>3</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), which dose-dependently inhibited the GATA4–NKX2-5 transcriptional synergy (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> A relatively rigid two heavy atom linker and the hydrogen acceptor feature next to the five-member ring were the essential structural features in successful fragment-to-hit growth (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). During the primary screening and hit optimization, 800 small molecule entities were tested, including at least 15 structurally related compounds in each of the three active series to confirm the original findings and the structure–activity relationships (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>).<named-content content-type="anchor" rid="tbl2" type="simple"></named-content><named-content content-type="anchor" rid="tbl3" type="simple"></named-content></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structural Variants of Hit Compound <b>3</b> in the Luciferase Reporter Gene Assay<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0010.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0011.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>a</sub> is equal to the synergy activation.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structural Variants of Hit Compound <b>4</b> in the Luciferase Reporter Gene Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0012.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0013.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structural Variants of Hit Compound <b>5</b> in the Luciferase Reporter Gene Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0014.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0015.gif" alt="" id="fx6" /></img><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Pharmacophore Search</h3><div class="NLM_p">To further evaluate the active molecular scaffold space, a pharmacophore model was built based on the following three active compounds: <b>3</b>, 4-[(4-propoxybenzylidene)amino]-5-(pyridin-4-yl)-4<i>H</i>-1,2,4-triazole-3-thiol (<b>4</b>), and isobutyl 4-(isonicotinamido)benzoate (<b>5</b>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). All three active compounds have a relatively rigid core structure, and therefore, the low energy conformations of these compounds were used as a basis for the compound superimposition and pharmacophore model. Subsequently, the pharmacophore model was completed with excluding volumes derived from the homology model of the GATA4-DNA binding site. Furthermore, 15 inactive and structurally variable compounds were selected from the parallel screening path targeting the GATA4–NKX2-5 protein–protein interface. Here, 14 of the 15 compounds screened was not detected by the pharmacophore model. Finally, an in silico screening of the commercial compound library was carried out using the pharmacophore model. Consequently, 20 compounds were purchased and tested in vitro using the luciferase reporter gene assay, leading to the identification of a new active compound, 5-(4-ethoxybenzylidene)-3-ethylthiazolidine-2,4-dione (<b>6</b>) at a concentration of 5 μM, and its variants (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Structural Variants of Hit Compound <b>6</b> in the Luciferase Reporter Gene Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0016.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0017.gif" alt="" id="fx8" /></img><div></div></div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) C-terminal zinc finger of GATA4 mediates the DNA binding and GATA4–NKX2-5 protein–protein interaction. Computational methods were applied to analyze and screen two separate binding sites. A pharmacophore model with excluded volumes was built for the DNA binding cavity of GATA4. (B) The low energy conformations of compounds <b>3</b>, <b>4</b>, and <b>5</b> were utilized as a basis for the pharmacophore model. All three compounds inhibited the GATA4–NKX2-5 synergy in the luciferase reporter gene assay. Atom colors are the following: white = hydrogen, gray = carbon, red = oxygen, blue = nitrogen, and yellow = sulfur. Pharmacophore features are the following: blue sphere = acceptor, orange sphere = aromatic, and green sphere = hydrophobic.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">All four compound families found during the in vitro screening inhibited the GATA4–NKX2-5-induced synergistic activation of the promoter containing the three high affinity NKX2-5 binding sites. Interestingly, the reporter assay screening also detected several weak hit compounds with the potential to increase the GATA4–NKX2-5 transcriptional synergy. These compounds have variable scaffolds but the structural elements were similar to the structures that inhibited the activation. Subsequently, during the small molecule optimization of the inhibitory compound <b>3</b>, surprisingly, we detected a compound that strongly augmented the GATA4–NKX2-5 synergy in the reporter gene assay (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The structure of this activator compound, <i>N</i>-(4-chlorophenyl)-5-methyl-<i>N</i>-(4-methyl-4,5-dihydrothiazol-2-yl)-3-phenylisoxazole-4-carboxamide (<b>7</b>), partially overlaps the pharmacophore model, which was developed based on inhibitory compounds with an additional five-member ring moiety attached. Due to the complexity of the compound structure, the synthesis of its derivatives was not pursued in this study, and only one related compound was tested to confirm this finding. Furthermore, the screening of a small natural compound library identified the flavonoids apigenin and luteolin as weak inhibitors of the GATA4–NKX2-5 transcriptional synergy with IC<sub>50</sub> values of 24 and 22 μM, respectively.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Aggregation of the Compounds</h3><div class="NLM_p last">Aggregation of a compound may result in artificial activity in luciferase enzyme based assays.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> To evaluate the potential aggregation of the compounds <b>3</b>, <b>4</b>, and <b>7</b>, the testing conditions were mimicked, and the light scattering of any particles/aggregates in the mixture was measured using the nephelometric method. Aggregation was only observed in the samples at a concentration of 100 μM. At lower concentrations, the values were close to those observed in the blanks, indicating that there is no detectable aggregation, except for compound <b>7</b>, which shows minor signs of aggregation at 30 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S1 in Supporting Information</a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemical Stability of Compound <b>3</b></h3><div class="NLM_p last">In addition, the stability of compound <b>3</b> was studied for 5 and 10 days in mouse embryonic stem cells (mESCs) in vitro at two different concentrations (3 and 5 μM) with two blank treatments (DMSO and embryoid body differentiation medium (EBDM)) added to the cell culture media. The intra- and extracellular concentrations of compound <b>3</b> and metabolite <b>31</b> were measured by HPLC/MS after performing specific sample pretreatment and extraction procedures. The results demonstrate that compound <b>3</b> is only modestly degraded during the 10 days in the cellular assay (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Table S1</a>). Furthermore, the storage of compound <b>3</b> in DMSO at room temperature for 6 months did not result in degradation (data not shown).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> GATA4 Transcriptional Activity and GATA4–NKX2-5 Interaction</h3><div class="NLM_p last">Compounds <b>3</b>, <b>4</b>, and <b>7</b> were evaluated in a reporter assay in COS-1 cells by using BNP promoter reporter constructs containing either the first 60 bp in the BNP promoter including the proximal TATA-box (BNP minimal promoter) (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S2A</a>) or the first 90 bp that include also the tandem GATA-site (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S2B</a>). TATA-box in BNP promoter acts as a functional GATA4 binding site.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Compound <b>3</b> significantly inhibited GATA4 driven transactivation of luciferase reporter constructs containing either BNP minimal promoter (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S2A</a>) or BNP promoter containing minimal promoter and tandem GATA-site on −90 bp (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S2B</a>). Compound <b>4</b> showed similar tendency, yet a statistically significant inhibition of gene transactivation was seen only with construct containing both minimal promoter and tandem GATA-sites (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S2B</a>). Overall, the effects of all compounds parallel those of primary screening assay with promoter containing the three high affinity NKX2-5 binding sites and are in agreement with the compound’s hypothetical mechanism of action. In addition, we used an immunoprecipitation assay to characterize further GATA4–NKX2-5 interaction. GATA4 and NKX2-5 FLAG protein lysates were produced in COS-1 cells, and FLAG-antibody bind agarose beads were used to pull down NKX2-5 together with interacting GATA4. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S3</a>, compound <b>3</b> modestly decreased the intensity of GATA4–NKX2-5 interaction, although this change was not statistically significant (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Figure S3</a>).</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> GATA4 and NKX2-5 DNA Binding</h3><div class="NLM_p">Previously, El-Hachem and Nemer have identified molecules that prevent GATA4 activity by inhibiting its interaction with DNA and blocking the activation of the downstream target genes of GATA4.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> These effects were reported to be due to a decreased GATA4-DNA interaction as evaluated by an electrophoretic mobility shift assay (EMSA). To determine whether the active compounds identified in the present study influence GATA4 or NKX2-5 DNA binding, we performed EMSA using nuclear proteins from both COS-1 cells and neonatal rat cardiomyocytes. The compounds were incubated with the COS-1 nuclear proteins, and then a <sup>32</sup>P-labeled oligonucleotide targeting GATA4 or NKX2-5 was added. Compound <b>3</b> did not alter the GATA4 or NKX2-5 DNA binding. Compounds <b>4</b> and <b>5</b> had a minor inhibitory effect on the NKX2-5 and GATA4 DNA binding, respectively; however, these changes were not statistically significant (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Moreover, neonatal rat cardiomyocytes were treated with the compounds for 24 h, the nuclear proteins were extracted, and EMSA was performed. In the cardiomyocytes, compound <b>3</b> inhibited the GATA4 DNA binding by 29%, and compound <b>4</b> inhibited the GATA4 DNA binding by 37% and the NKX2-5 DNA binding by 29% (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). However, these changes were not statistically significant, suggesting that the effects of the small molecules on the GATA4- and NKX2-5-driven transcriptional synergy are not due to the inhibition or activation of the GATA4 or NKX2-5 DNA binding.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of compounds on GATA4 and NKX2-5 protein DNA binding. (A, B) Nuclear proteins from COS-1 cells were extracted, and binding reactions and EMSA were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. The concentration of the compounds <b>3</b>–<b>5</b> was 50 μM. (C, D) Neonatal rat cardiomyocytes were treated with the compounds <b>3</b>, <b>4</b>, and <b>7</b> for 24 h. Nuclear proteins were extracted, and binding reactions and EMSA were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Specific GATA4 and NKX2-5 bands are indicated. The concentration of the compounds was 50 μM. Supershift reactions were performed by incubating the reaction mixtures with anti-GATA4 and anti-NKX2-5 antibodies. Asterisks denote the lanes with the supershifted samples. The dashed line indicates where the EMSA image has been cut. The data are shown relative to the vehicle treatment as the mean ± SD, <i>n</i> = 3–5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Biological Activity</h3><div class="NLM_p">The role of GATA4 in the heart ranges from the differentiation, survival, and proliferation of cardiac progenitors to the adaptive stress response, angiogenesis, and myocardial repair of the postnatal heart.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> These diverse GATA4 functions are thought to involve differential interactions with cell-specific and signal-inducible cofactors. GATA4 and NKX2-5 are critical transcription factors that cooperatively regulate the mechanical stretch-responsive genes ANP<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and BNP.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Both ANP and BNP are expressed in the developing and adult heart, and their gene expression is markedly up-regulated in response to myocyte hypertrophy and hemodynamic overload due to myocardial infarction and heart failure.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> Therefore, we tested whether our newly discovered active compounds could influence ANP and BNP gene expression in vitro. Neonatal rat ventricular myocytes were treated with endothelin-1 (ET-1), which is a well-established hypertrophic agonist that activates the gene expression of both ANP and BNP,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and the compounds were added to the cells 1 h prior to ET-1. The compounds either augmented or inhibited the ET-1-induced gene expression of ANP and BNP (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Consistently with the luciferase reporter gene assay finding in which compound <b>3</b> inhibited the GATA4 and NKX2-5 transcriptional synergy, compound <b>3</b> also inhibited the ET-1-stimulated BNP gene expression (68%, <i>p</i> < 0.05). There was also a tendency for compound <b>3</b> to decrease the basal mRNA levels of ANP (81%) and BNP (75%), and ET-1 stimulated the gene expression of ANP (32%), but these changes were not statistically significant (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Compounds <b>8</b>, <b>9</b>, <b>10</b>, and <b>62</b> inhibited the ET-1-induced increase in the mRNA levels of ANP, and compound <b>8</b> also inhibited the mRNA levels of BNP (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In contrast, compound <b>7</b>, which is an activator of the GATA4–NKX2-5 transcriptional synergy, enhanced the baseline mRNA levels of ANP (4.6-fold, <i>p</i> < 0.001) and BNP (6.4-fold, <i>p</i> < 0.001) and ET-1-stimulated gene expression of BNP (2.1-fold, <i>p</i> < 0.001) in the cardiomyocytes (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). Finally, compound <b>4</b> had a mixed action, acting as an inhibitor in the transcriptional synergy assay but enhancing the basal ANP (8.6-fold, <i>p</i> < 0.001) and BNP (4.9-fold, <i>p</i> < 0.001) gene expression levels in cardiomyocytes (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of different compounds in the bioassays. (A–D) Neonatal rat cardiomyocytes (CMs) were treated with 100 nM endothelin-1 (ET-1) for 23 h. Compounds were added to the cells at concentrations of 10–100 μM 1 h prior to ET-1. The results are expressed as the ratio of ANP and BNP mRNA to ribosomal 18 S RNA, which were all determined by RT-PCR at 24 h. The data are shown relative to the vehicle sample as the mean ± SD: vehicle, <i>n</i> = 24; compounds, <i>n</i> = 3–6. (E) The effect of compound <b>3</b> on the hypertrophic response was investigated by exposing the CMs to 10 μM for 48 h with or without stretching. After the stretching, the CMs were fixed, stained for cardiac α-actinin, F-actin, and nuclei, and imaged under a fluorescence microscope. The surface area of the α-actinin expressing cells was determined from the images using CellProfiler software. The results are expressed as the mean of three independent experiments ± SEM: (∗) <i>p</i> < 0.05, (∗∗) <i>p</i> < 0.01, (∗∗∗) <i>p</i> < 0.001 vs vehicle treatment (A–D) or appropriate group (E).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The activity of the most potent compound, i.e., compound <b>3</b>, was further validated in a cardiomyocyte hypertrophy model by utilizing cyclical stretching of ventricular myocytes. By use of this experimental model, the co-operation of GATA4 and NKX2-5 has been reported to be essential for mediating the stretch response via the NKE.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Myocytes were exposed to the compound at a concentration of 10 μM, and after 48 h of mechanical stretching, the myocyte area was measured. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E, compound <b>3</b> significantly inhibited the increase in the area of the myocytes in response to the stretching, indicating that the compound inhibits hypertrophic growth. Overall, the results show that the compounds can modulate the hypertrophic response in myocytes in vitro, which is consistent with the inhibition or activation of the GATA4–NKX2-5 interaction (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>).</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Cytotoxicity</h3><div class="NLM_p">The cytotoxicity of the selected biologically active compounds was evaluated by determining necrotic and apoptotic cell death by analyzing the release of adenylate kinase (AK) and TUNEL-positive cells, respectively, in neonatal rat cardiomyocytes. The cells were treated with each compound at a concentration of 50 μM for 24 h. Compounds <b>3</b>, <b>4</b>, and <b>7</b> alone or together with ET-1 did not affect necrotic cardiomyocyte death (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Compounds <b>3</b>, <b>4</b>, and <b>7</b> had no effect on apoptotic cell death (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cytotoxicity of compounds <b>3</b>, <b>4</b>, and <b>7</b> in neonatal rat ventricular myocytes. (A) Adenylate kinase (AK) release was determined after the cells were treated for 24 h with each compound at a concentration of 50 μM. The data are shown relative to the vehicle treatment as the mean ± SD, <i>n</i> = 3. (B) To detect apoptotic cells, a TUNEL assay was performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. TUNEL-positive cells were determined after the cells were treated for 24 h with each compound at a concentration of 50 μM. The data are shown as the mean ± SD, <i>n</i> = 2 (for compound <b>7</b> in TUNEL assay, <i>n</i> = 1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Protein Kinase Profiling</h3><div class="NLM_p last">To characterize the effect of the active compounds on the protein kinases, Cerep ExpresS Diversity kinase profiling was performed. Of the 48 kinases analyzed, compound <b>3</b>, at a concentration of 30 μM, inhibited epidermal growth factor receptor kinase (EGFR) by 54% and vascular endothelial growth factor receptor 2 kinase/kinase insert domain receptor (VEGFR2/KDR) by 64% (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Table S2</a>). In addition, compound <b>4</b> inhibited fibroblast growth factor receptor 3 kinase (FGFR3) by 55% (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Table S3</a>). It is noteworthy that compound <b>3</b> was inactive against p38α mitogen-activated protein kinase (MAPK), glycogen synthase kinase-3β (GSK-3β), extracellular signal-regulated kinase 2 (ERK2), protein kinase A (PKA), and Rho kinase 1 (ROCK1), which all have been reported to be involved in the regulation of GATA4 phosphorylation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Table S2</a>).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell-Based G-Protein Coupled Receptor Profiling</h3><div class="NLM_p last">To test the effect of the biologically active compound <b>3</b> on 165 different G-protein-coupled receptors (GPCRs), we used Millipore GPCR profiler, which monitors calcium flux in cells expressing specific GPCR and promiscuous GPCR-coupling Gα<sub>15/16</sub> proteins. Compound <b>3</b> inhibited cannabinoid receptor type 2 (CB2), parathyroid hormone 2 receptor (PTH2), and niacin receptor 1/G-protein-coupled receptor 109A (GPR109A) with mean percentage inhibition values of 91.8, 59.5, and 58.5, respectively, but no significant agonistic effect was found (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">Table S4</a>). In addition, the previously reported noncardiac inhibitory effect on ghrelin receptor was identified in this study (65.2%).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Molecular Modeling</h3><div class="NLM_p">A computational analysis based on a model of GATA4–NKX2-5 bound to DNA suggests that the structural arrangement of the C-terminal extension in the second zinc finger of GATA4 plays a critical role in the transcription factor networking in the cardiac context.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The functional preference of GATA4 is dependent on a distinct trade-off between DNA binding and interacting with NKX2-5. Both structural arrangements are dependent on and amplified by post-translational modifications, such as phosphorylation and acetylation.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The evaluation of the protein structure indicates that the DNA binding of a single GATA4 partially depends on the extensive contact between the DNA and the C-terminal extension in the second zinc finger. However, our proposed GATA4–NKX2-5 interaction requires a dislocation of the loop extension, resulting in an inability of GATA4 to drive the synergy through the GATA binding site (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) C-terminal extension in the second zinc finger domain (yellow ribbon) contributes to the DNA binding of GATA4. (B) For the activation of the GATA4–NKX2-5 interaction, the loop extension needs to relocate and (C) enforce the structural reorganization of the GATA4 complex in the tandem GATA binding site. (C) The GATA4 and NKX2-5 proteins were overexpressed in the COS-1 cells. Nuclear proteins were extracted, and EMSA was performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. GATA4 alone (four parallel samples) or together with NKX2-5 (four parallel samples) was incubated in binding reactions with DNA. GATA4 binding to the rat BNP promoter (−90 bp tandem GATA site) resulted in two separate bands, and their intensities depended on the presence of NKX2-5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28270" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28270" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Transcription factors are the fundamental regulators of gene transcription, and many diseases, such as cancer, neurodegenerative disorders, and heart disease, are associated with the deregulation of transcriptional networks.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The understanding of these complex networks and pharmacological strategies that modulate the activity of distinct transcription factors will, therefore, be essential for the development of novel therapeutic approaches. A proof for the relevance of directly modulating transcription factors has been provided by targeting nuclear receptors that contain a clearly defined ligand-binding pocket and are activated by natural ligands, such as retinoic acid, glucocorticoid, estrogen or androgen receptors. Moreover, the design of small molecule drugs that interfere with transcription factor/cofactor interactions or homo-/heterodimerization between cooperating transcription factor molecules has yielded various drugs that are capable of counteracting specific DNA-regulatory proteins.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Thus, protein–protein interactions of key transcription factors are likely to present one of the next major classes of innovative therapeutic targets.</div><div class="NLM_p">A substantial body of evidence suggests that a limited number of regulatory transcription factors (e.g., GATA4, HAND2, MEF2, NKX2-5, and TBX5) are needed for the initiation of cardiac-like gene expression.<a onclick="showRef(event, 'ref20 ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref20 ref39 ref40 ref41">(20,39−41)</a> Investigations of heterotypical interactions involving GATA4 have revealed that transcription factors that are critical for cardiac reprogramming, such as MEF2,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> HAND2,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and TBX5,<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> tend to synergistically activate the GATA binding site (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). In contrast, the overexpression of either NKX2-5<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> or SRF<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> leads to protein complex formation and, consequently, the activation of their corresponding sites with a diminished ability to drive cardiac transdifferentiation.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In addition to the GATA4–TBX5 interaction, other heterotypic GATA4 protein interactions require only a single binding element at the promoter. Overall, previous studies suggest that NKX2-5 plays both a structural<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and functional<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> role as a dominant cofactor of GATA4. Since an orchestrated and sequential expression of transcription factors is essential for the development of cardiac cells, theoretically, the inhibition of the GATA4–NKX2-5 protein complex would result in other GATA4-associated TF combinations that would more efficiently program cells toward a cardiomyocyte fate (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Interestingly, the increased research efforts targeting nuclear receptors have revealed that the interaction between GATA4 and NKX2-5 shares the same architecture as the two zinc fingers in the DNA binding domain of nuclear receptors,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> implicating that small molecules have ability to modulate the GATA4 and NKX2-5 protein–protein interactions.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Transcriptional Activity of GATA4 Is Regulated by Interactions with Several Other TFs and Post-Translational Modifications<a onclick="showRef(event, 'ref13 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref13 ref17 ref18 ref19">(13,17−19)</a><sup>,</sup><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Most of the protein–protein interactions of GATA factors are mediated by the C-terminal zinc finger, while the N-terminal zinc finger interacts with Friend of GATA (FOG) transcription factors. Selected heterotypic interactions and DNA occupation modes for GATA4 are classified based on experimental measurements of the binding preferences.<a onclick="showRef(event, 'ref25 ref42 ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref25 ref42 ref43 ref44 ref45 ref46">(25,42−46)</a> Heterotypic GATA4 complex subpopulations have distinct context dependent effects on the commitment of stem cells toward the cardiac fate or the enhancement of hypertrophic gene expression in mature cardiac cells. SRF, serum response factor; SRE, serum response element; PTM, post-translational modification; Phos, phosphorylation; Acet, acetylation; Sumo/Ubi, sumoylation/ubiquitination.</p></p></figure><div class="NLM_p">In the present study, we identified four small molecule families that produce either an agonistic or antagonist effect on the GATA4–NKX2-5 transcriptional synergy. Here, we used the NKE site to study the GATA4–NKX2-5 cooperative interaction. The 3xNKE reporter is an artificial tool used to detect the more pronounced signal from the GATA4–NKX2-5 synergy for screening purposes. Moreover, it has been shown that the NKE site is essential for the GATA4–NKX2-5 cooperation in stretching-induced hypertrophy.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> A fragment-based screening, transcriptional synergy measuring reporter gene assay, and pharmacophore search were utilized for the small molecule screening, identification, and optimization. The primary screening using a reporter gene assay and the in vitro cell culture studies revealed several biologically active compounds, i.e., <b>3</b>, <b>4</b>, <b>5</b>, <b>6</b>, and <b>7</b>, with distinct profiles. The structure–activity relationship studies of inhibitory compounds <b>3</b>, <b>4</b>, and <b>5</b> demonstrated the indispensable role of the two heavy atom linker together with a hydrogen acceptor feature next to the five-member ring, while a completely opposite agonistic effect was achieved by introducing a substituted five-member ring into the amide bond (<b>7</b>). In the cardiomyocyte cultures, the compounds either augmented or inhibited the hypertrophic agonist-induced ANP and BNP expression. In addition, the GPCR and kinase screenings with the most potent compound, i.e., <b>3</b>, <i>N</i>-[4-(diethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide, did not detect cross-linking pathways with the major cardiac signaling cascades. Compound <b>3</b> and its derivatives may provide a new class of small molecules for modulating cardiogenesis and heart regeneration (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Even though the precise binding site of the tool compounds in the GATA4–NKX2-5 interaction remains to be established, the structural similarity and spatial appearance suggest the existence of a common binding site that mediates the actions of the active compounds. Of note, the results concerning particular compounds may include unspecific effects, i.e., the high concentration of compound <b>4</b> in the ET-1 assay, the context-dependent degradation of the ester structure with compound <b>5</b>, and the frequent binder characteristic of compound <b>6</b>.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a></div><div class="NLM_p">Previously, numerous small molecule-based approaches have been shown to enhance cardiac reprogramming, including the activation of Wnt/β-catenin signaling (GSK3β inhibitors), inhibition of Wnt-signaling (porcupine and tankyrase inhibitors), or modulation of transforming growth factor (TGF) β-superfamily signaling (bone morphogenetic protein and TGFβ inhibitors).<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> These signaling pathways are essential during the early phases of cardiac differentiation (mesoderm induction).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The combined application of Wnt- and TGFβ-modulators requires strict time-dependent dosing in cellular assays to yield optimal conditions for cardiac reprogramming. Nevertheless, systematic protocol optimizations have led to a powerful platform for cardiac differentiation from human pluripotent stem cells using small molecule modulators of Wnt-signaling.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Additionally, very recently, Cao et al. and Mohamed et al. presented pharmacological lineage-specific approaches for the efficient conversion of human fibroblasts into functional cardiomyocytes using either a combination of nine small molecules<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> or TGFβ- and Wnt-inhibitors jointly with three cardiac transcription factors, i.e., GATA4, MEF2c, and TBX5.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> However, the translation of the in vitro tool compounds to in vivo candidates for cardiac regeneration is challenging, e.g., requires the optimization of the physicochemical or ADME properties that is necessary for efficacy in vivo. Moreover, although the in vivo dosing of tool compounds that regulate Wnt- or TGFβ-signaling may interfere with the pathways beneficial for cardiac regeneration, these compounds are potentially pro-oncogenic.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">Regarding the cardiac transcription factors, Sadek et al. performed a small molecule library screening for chemical activators of <i>NKX2-5</i>, which is one of the earliest lineage-restricted genes to be expressed in cardiovascular progenitor cells.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> A chemical library of 147 000 compounds was screened in P19CL6 cells harboring an NKX2-5 luciferase reporter construct, which led to the identification of 10 potential molecular scaffolds with cardiogenic activity, including lead compounds, such as sulfonylhydrazones (Shz-1) and isoxazoles (Isx-9). Moreover, mechanistic follow-up studies suggest that both Shz-1 (activation of Brachyury T together with early cardiogenic program genes, such as NKX2-5 and myocardin<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a>) and Isx-9 (agonist of orphan GPR68, which is an extracellular proton/pH sensing GPCR<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a>) act independently of the known procardiogenic pathways, such as Wnt and the TGFβ superfamily. Of note, although various active molecular scaffolds identified by Sadek et al. share similarities with the compounds in the present study, including 3,5-disubstituted isoxazoles vs 3,4,5-trisubstituted isoxazoles, thiazolidinediones, and flavonoids, Isx-9 did not inhibit the luciferase reporter gene activity in our GATA4–NKX2-5 transcriptional synergy assay (data not shown).</div><div class="NLM_p last">In conclusion, adult cardiomyocytes are terminally differentiated, and their capacity for renewal is limited in response to cardiac injury.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> A core set of transcription factors control the direct reprogramming of fibroblasts into induced cardiomyocytes<a onclick="showRef(event, 'ref41 ref59 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref41 ref59 ref60 ref61">(41,59−61)</a> and directed differentiation of stem cells<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> and are, therefore, implicated as potential novel drug targets for heart regeneration.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> One attractive strategy is to inhibit the protein–protein interaction since many transcription factors act as homo- or heterodimers and depend on cofactors to function appropriately.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Our results (3xNKE reporter assay, BNP transcription, endothelin-1 stimulated gene expression of ANP and BNP, hypertrophic cell surface growth in cardiomyocytes, and immunoprecipitation assay) provide evidence of the small molecule-mediated modulation of the transcriptional synergy between GATA4 and NKX2-5, which plays essential roles in programming cells toward the cardiomyocyte fate and mediating the stretching-induced hypertrophic response. GATA4 is also a critical regulator of biological processes in myocardial remodeling and repair.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> However, the inhibitory compounds could alter the interaction between GATA4 and other binding partners, and thus a more detailed assessment and validation of the molecular mechanism(s) and the optimization of small molecules are needed to fully evaluate the potential of GATA4–NKX2-5 interaction modulators in chemically mediated cardiogenesis and heart repair and regeneration.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31303" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31303" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Homology Modeling and Sequence Alignment</h3><div class="NLM_p last">A homology model of GATA4 was constructed using the highly conserved C-terminal zinc finger domain of GATA3 as a template structure (Protein Data Bank code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DFX">3DFX</a>, sequence identity 76%) for the C-terminal zinc finger of GATA4. A homology model for the homeodomain of NKX2-5 was built using the homeodomain of the related thyroid transcription factor 1 as a template structure (Protein Data Bank code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTT">1FTT</a>, sequence identity 61%). Other than the C- and N-terminal domains, template structures are lacking and were, therefore, excluded from the modeling. The sequence alignments were constructed without any insertions or deletions using PSI-BLAST. The commercial modeling package Molecular Operating Environment (MOE 2014.09; Chemical Computing Group Inc., Montreal, Canada; <a href="http://www.chemcomp.com" class="extLink">www.chemcomp.com</a>) was used to construct the homology models of the protein domains. An AMBER99 force field was applied as the source of the atom parameters for the protein minimizations and scoring during the homology modeling. The side chain data parameters were assembled from the extensive MOE-integrated rotamer library. During the homology modeling, 10 protein domain specific intermediate models were generated using the Boltzmann-weighted randomized modeling procedure. All intermediate models were submitted to an electrostatic-enabled energy minimization calculation until the root mean square (rms) gradient fell below 1. Subsequently, the refined protein models were scored using the generalized Born/volume integral (GB/VI) methodology.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> The highest scoring intermediate model was further minimized until the rms fell below 0.5 and, subsequently, was selected as the final model. The refined final models were inspected (bond lengths, bond angles, atom clashes, and contact energies) and confirmed to have satisfactory stereochemical quality using Ramachandran plots (φ and ψ angles).<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> In Silico Fragment Screening</h3><div class="NLM_p last">Binding site assessments of the homology models of GATA4 and NKX2-5 were carried out using the MOE-integrated Alpha Site Finder calculation using the default settings. Multiple copy simultaneous chemical probe analyses<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> were applied to the highest ranked binding pocket of the rigid GATA4 protein. MMFF94 was selected as the appropriate force field for the receptor–ligand interactions. The DNA binding site of the GATA4 protein model was exposed to a collection of 39 randomly placed molecular fragments and, subsequently, an energy minimization to rank the interaction potential (with and without the solvent) of the fragments with diverse functional groups inside the proposed binding cavity. The default settings and protocols were applied to the fragment library, number of fragment copies, energy minimization termination gradient, and protein flexibility parameters.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Pharmacophore Preparation and Search</h3><div class="NLM_p last">A flexible molecular alignment was implemented to study the 3D molecular similarity to elucidate the common pharmacophoric query of the three active compounds (<b>3</b>, <b>4</b>, and <b>5</b>). The stochastic search procedure aligns the conformational space of the molecules by maximizing the molecules’ chemical similarity with the least amount of internal strain. An Amber12:EHT force field was applied for the molecule parametrizations and energy minimizations. Furthermore, the default settings were employed to score and rank the collection of compound superpositions.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> The best-ranking alignment of active compounds was utilized to generate the pharmacophore annotation points (four point pharmacophore with two acceptor, aromatic, and hydrophobic features). Excluded volume constraints derived from the GATA4 protein model were utilized as a shape restrictor in the pharmacophore query. The MOE conformation database (650 000 leadlike compounds from 44 chemical suppliers) was used as a ligand reservoir for the pharmacophore search.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Screening of Commercial Small Molecules</h3><div class="NLM_p last">Small molecules for the primary screening and luciferase reporter gene assay based optimization were selected and purchased from established suppliers, including ChemBridge (San Diego, CA, USA), Enamine (Kiev, Ukraine), Sigma-Aldrich (St. Louis, MO, USA), and Specs (Zoetermeer, The Netherlands). All compounds purchased for the study were tested by HPLC–MS and <sup>1</sup>H NMR by the vendors to confirm the sample identity and ensure a minimum purity of 90%.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Chemistry</h3><div class="NLM_p">The synthesis of selected compounds (i.e., <b>3</b>, <b>4</b>, <b>7</b>, <b>31</b>, and <b>32</b>) were outsourced to Pharmatory (Oulu, Finland). All reactions were carried out with commercially available solvents and chemicals. All chemicals, solvents, and anhydrous solvents were acquired from Sigma-Aldrich (Schnelldorf, Germany), Fluka (Buchs, Switzerland), and Alfa Aesar (Ward Hill, MA, USA). The purity of the final compounds was determined by HPLC/MS and <sup>1</sup>H NMR (minimum purity >95%). The <sup>1</sup>H NMR spectra were observed at the University of Oulu using a Bruker DPX 200 instrument (Rheinstetten, Germany) in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub>. The reported chemical shifts (δ) are expressed in parts per million (ppm), and the coupling constants (<i>J</i>) are expressed in hertz (Hz). The spin multiplicities are reported as s = singlet, br s = broad singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad.</div><div class="NLM_p last">The HPLC–MS analysis was performed on an Agilent Technologies 1100 HPLC fitted with Hewlett-Packard series 1100 MSD with Zorbax Extend-C18 150 mm × 4.6 mm, 5 μm, 30 °C, and water + 0.20% v/v trifluoroacetic acid and acetonitrile as the eluent. The HPLC analyses were performed on Zorbax Extend-C18 150 mm × 4.6 mm, 5 μm at an ambient temperature using water + 0.20% v/v trifluoroacetic acid and acetonitrile as the eluent.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Synthetic Procedure for Manufacturing Compound <b>3</b> and Other Relative Compounds</h3><div class="NLM_p">One equivalent of a suitably 3,5-disubstituted isoxazole-4-carboxylic acid is dissolved in a solvent, such as DMF, and the obtained solution is cooled to 0–10 °C. One equivalent of a suitably substituted benzene-amine and an excess of <i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium tetrafluoroborate (TBTU) are added, and the mixture was stirred at 0–10 °C for 5–30 min. An excess of a non-nucleophilic base, such as diisopropylethylamine, is slowly added, and the mixture is stirred at 0–10 °C for 2–5 h. Depending on the substituents used, either isoxazole-carboxylic acid or benzene-amine, or both, can be modified by introducing suitable protective groups to the selected substituents prior to their reaction to yield the product compound.</div><div class="NLM_p">An organic solvent of a suitable polarity, such as ethyl acetate and an aq NaHCO<sub>3</sub> solution, is added to the reaction mixture to dissolve the compound, and the mixture is stirred at an ambient temperature until dissolution occurs. The layers are separated, and the organic phase is washed with water (e.g., 3 times) and brine (e.g., once). The organic layer is then dried using a drying agent, such as sodium sulfate. The drying agent is then filtered, and the solution is evaporated to dryness to yield the crude compound.</div><div class="NLM_p last">The crude compound can be purified by dissolving it at a temperature of 25–70 °C in a solvent of a suitable polarity, such as ethyl acetate, and stirring the solution. An activated carbon can optionally be added, and the mixture stirred and filtered through Celite, whereby some impurities can be removed. Finally, precipitation of the product can be achieved by cooling the solution to a point at which it begins to crystallize or adding a second solvent with a polarity opposite that of the first solvent, such as <i>n</i>-heptane, to the previous solution and slowly cooling the mixture to a point at which the product begins to crystallize. The pure precipitated product can then be filtered, washed with the second solvent (e.g., twice), and dried, optionally under vacuum.</div></div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-[4-(Diethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide (<b>3</b>)</h3><div class="NLM_p last">5-Methyl-3-phenylisoxazole-4-carboxylic acid (8.00 g, 39.4 mmol) was dissolved in DMF (200 mL), and the solution cooled to 0–10 °C. <i>N</i>′<i>,N</i>′-Diethylbenzene-1,4-diamine (6.47 g, 39.4 mmol) and TBTU (15.17 g, 47.3 mmol) were added, and the mixture was stirred at 0–10 °C for 10 min. Diisopropylethylamine (6.11 g, 47.3 mmol) was slowly added, and the mixture was stirred at 0–10 °C for 2.5 h. Ethyl acetate (400 mL) and a 5% aq NaHCO<sub>3</sub> solution (160 mL) were added to the reaction mixture, and the mixture was stirred at an ambient temperature for 15 min. The layers were separated, and the organic phase was washed with water (3 × 200 mL) and brine (180 mL). The organic layer was dried with sodium sulfate, the drying agent was filtered, and the solution evaporated to dryness to yield 12.8 g of crude compound <b>3</b>. The crude compound was dissolved in warm ethyl acetate (300 mL), and activated carbon (0.5 g) was added. The mixture was stirred for 20 min and filtered through Celite. <i>n</i>-Heptane (120 mL) was added to the previous solution at 55 °C, and the mixture was slowly cooled to 0–5 °C. The precipitated product was filtered, washed with <i>n</i>-heptane (2 × 50 mL), and dried under vacuum at 45 °C overnight to yield 8.52 g (62%) of compound <b>3</b> as an off-white powder. NMR: 10.12 ppm (s, 1H, NH), 7.71 ppm (m, 2H, arom), 7.35–7.50 ppm (m, 5H arom), 6.63 ppm (d, 2H, arom), 3.30 ppm (q, 4H, 2 × CH<sub>2</sub>), 2.55 ppm (s, 3H, CH<sub>3</sub>), 1.05 ppm (t, 6H, 2 × CH<sub>3</sub>). MS: <i>m</i>/<i>z</i> 350.2 (100%, M + 1), 351.2 (25%). Melting point range: 139.3–140.2 °C.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-[(4-Propoxybenzylidene)amino]-5-(pyridin-4-yl)-4<i>H</i>-1,2,4-triazole-3-thiol (<b>4</b>)</h3><div class="NLM_p last">A mixture of 4-propoxybenzaldehyde (10.00 g, 60.9 mmol) and 4-amino-5-(pyridin-4-yl)-4<i>H</i>-1,2,4-triazole-3-thiol (11.8 g, 60.9 mmol) in glacial acetic acid (100 mL) was heated to reflux for 2.5 h, and the solution was filtered hot. The filtrate was allowed to cool to an ambient temperature and was then cooled further to 5 °C with an ice bath. The precipitated product was filtered, washed with acetic acid (3 × 20 mL), and dried with suction to yield 17.8 g of crude compound <b>4</b>. Crude compound <b>4</b> was dissolved in boiling glacial acetic acid (110 mL), and the solution was allowed to slowly cool to an ambient temperature. The precipitated product was filtered, washed with acetic acid (2 × 10 mL) and water (2 × 10 mL), and dried under vacuum at 40 °C for 8 h to yield 12.0 g (58%) of compound <b>4</b>. NMR (DMSO-<i>d</i><sub>6</sub>): 9.51 ppm (s, 1H, SH), 8.75 ppm (m, 2H, arom), 7.88 ppm (m, 4H arom), 7.11 ppm (d, 2H, arom), 4.04 ppm (t, 2H, CH<sub>2</sub>), 1.76 ppm (m, 2H, CH<sub>2</sub>), 0.98 ppm (t, 3H, CH<sub>3</sub>). MS: <i>m</i>/<i>z</i> 340.1 (100%, M + 1), 341.1 (22%), 362.0 (5%, M + Na). Melting point: decomposition.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-(4-Chlorophenyl)-5-methyl-<i>N</i>-(4-methyl-4,5-dihydrothiazol-2-yl)-3-phenylisoxazole-4-carboxamide (<b>7</b>)</h3><div class="NLM_p last">5-Methyl-3-phenylisoxazole-4-carboxylic acid (1.01 g, 4.97 mmol) was dissolved in DMF (25 mL), and the solution was cooled to 0–10 °C. <i>N</i>-(4-Chlorophenyl)-4-methyl-4,5-dihydrothiazol-2-amine (1.12 g, 4.94 mmol) and TBTU (1.90 g, 5.92 mmol) were added, and the mixture was stirred at 0–10 °C for 10 min. Diisopropylethylamine (0.78 g, 6.04 mmol) was slowly added, and the mixture was stirred at 0–10 °C for 4 h. The cooling bath was removed, and the mixture was stirred for an additional 50 min, allowing it to warm to an ambient temperature. Ethyl acetate (40 mL) and 10% aq NaHCO<sub>3</sub> solution (35 mL) were added to the reaction mixture, and the mixture was stirred at an ambient temperature for 5 min. The layers were separated, and the organic layer was washed with water (3 × 30 mL) and brine (15 mL). The organic layer was dried with sodium sulfate, the drying agent was filtered, and the solution was evaporated to dryness to yield 2.20 g of crude compound <b>7</b>. The crude material was dissolved in ethyl acetate (10–12 mL) at an ambient temperature, and the solution was stirred at an ambient temperature for 15 min; during this time, the product began to crystallize out of the solution. <i>n</i>-Heptane (20 mL) was added, and the mixture was stirred at an ambient temperature for 15 min; during this period, more of the product precipitated. The precipitated product was filtered, washed with <i>n</i>-heptane, and dried under vacuum at 50 °C overnight to yield 1.53 g of compound <b>7</b> as a white powder. NMR (CDCl<sub>3</sub>): 7.35–7.60 ppm (m, 5H, arom), 7.08–7.18 ppm (m, 2H, arom), 6.31–6.43 ppm (m, 2H arom), 3.50–4.20 ppm (broad m, 2H thiazole CH<sub>2</sub>), 3.57 ppm (m, 1H, thiazole CH), 2.59 ppm (s, 3H, CH<sub>3</sub>), 1.36 ppm (d, 3H, thiazole CH<sub>3</sub>). Melting point range: 145.6–147.3 °C.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-[4-(Ethylamino)phenyl]-5-methyl-3-phenylisoxazole-4-carboxamide (<b>31</b>)</h3><div class="NLM_p last">5-Methyl-3-phenylisoxazole-4-carboxylic acid (300 mg, 1.48 mmol) was dissolved in DMF (7.5 mL), and the solution was cooled to 0–10 °C. 1-<i>N</i>-Ethylbenzene-1,4-diamine (201 mg, 1.48 mmol) and TBTU (569 mg, 1.77 mmol) were added, and the mixture was stirred at 0–10 °C for 10 min. Diisopropylethylamine (0.30 mL, 1.77 mmol) was slowly added, and the mixture was stirred at 0–10 °C for 1.5 h. Ethyl acetate (30 mL) and 5% aq NaHCO<sub>3</sub> solution (30 mL) were added to the reaction mixture, and the mixture was stirred at an ambient temperature for 15 min. The layers were separated, and the aqueous layer was washed with ethyl acetate (15 mL). The combined organic solutions were washed with water (30 mL) and brine (20 mL). The precipitated product was filtered and dried under vacuum at 30 °C for 20 h to yield 282 mg of compound <b>31</b> as a pinkish powder. NMR (DMSO-<i>d</i><sub>6</sub>): 10.05 ppm (s, 1H, NH), 7.71 ppm (m, 2H, arom), 7.50 ppm (m, 2H arom), 7.32 ppm (d, 2H arom), 6.52 ppm (d, 2H, arom), 5.43 ppm (t, 1H, NH), 3.00 ppm (m, 2H, CH<sub>2</sub>), 2.55 ppm (s, 3H, CH<sub>3</sub>), 1.14 ppm (t, 3H, CH<sub>3</sub>). MS: <i>m</i>/<i>z</i> 322.1 (100%, M + 1), 344.2 (25%, M + Na). Melting point range: 203.1–205.9 °C.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-[4-(Ethylamino)phenyl]-5-(hydroxymethyl)-3-phenylisoxazole-4-carboxamide (<b>32</b>)</h3><div class="NLM_p"><i>Step 1</i>. Magnesium ethoxide (4.85 g, 42.4 mmol) was suspended in dry toluene (9 mL), and ethyl benzoyl acetate (7.40 g, 38.5 mmol) was added. After stirring the reaction mixture for 1 h at room temperature, dry acetonitrile (9 mL) was added, and the reaction mixture was cooled to −10 °C. Chloroacetyl chloride (4.78 g, 3.37 mL, 42.4 mmol) was added for 35 min. The reaction mixture was allowed to warm to room temperature and stirred for 1 h. Sulfuric acid (1 mL) in ice–water (35 mL) was added to the mixture, and the layers were separated. The aqueous layer was washed with diethyl ether (20 mL), and the combined organic layers were dried with magnesium sulfate. The drying agent was filtered, and the filtrate was cooled to 0 °C. Triethylamine (5.4 mL) was dissolved in diethyl ether (3 mL), and the mixture was added to the filtrate. The reaction mixture was then stirred at room temperature for 40 min. The precipitated triethylamine hydrochloride was filtered, and the solution was evaporated to dryness to yield 9.74 g (109%) of crude ethyl 2-oxo-4-phenyl-2,5-dihydrofuran-3-carboxylate (intermediate <b>1</b>), which was used in the next step without further purification.</div><div class="NLM_p"><i>Step 2</i>. The crude intermediate <b>1</b>, which was prepared as described above (9.74 g), was dissolved in ethanol (39 mL), and hydroxylamine hydrochloride (2.68 g, 38.5 mmol) and potassium acetate (3.78 g, 38.5 mmol) were added. The reaction mixture was heated to reflux for 40 min and then cooled to room temperature. Ethanol was removed by evaporation under reduced pressure, and water (77 mL) was added to the evaporation residue. The product was extracted with diethyl ether (2 × 20 mL), and the combined organic layers were washed with brine and dried with magnesium sulfate. The drying agent was filtered, and the filtrate was evaporated to dryness under reduced pressure to yield 9.07 g (97% over two steps) of crude ethyl 5-(hydroxymethyl)-3-phenylisoxazole-4-carboxylate (intermediate <b>2</b>). NMR (CDCl<sub>3</sub>): 7.35–7.65 ppm (m, 5 H, arom), 4.95 ppm (s, 2H, CH<sub>2</sub>), 4.26 ppm (q, 2 H, CH<sub>2</sub>), 1.20 ppm (t, 3H, CH<sub>3</sub>).</div><div class="NLM_p"><i>Step 3</i>. Crude intermediate <b>2</b> (1.0 g, 4.04 mmol) was dissolved in dichloromethane (28 mL) and 3,4-dihydro-2<i>H</i>-pyran (0.51 g, 6.07 mmol), and pyridinium <i>p</i>-toluenesulfonate (0.10 g, 0.4 mmol) was added. The reaction mixture was stirred at room temperature for 6 h. The reaction mixture was diluted with dichloromethane (20 mL) and washed with half-diluted brine (40 mL). The organic layer was dried with magnesium sulfate, the drying agent was filtered, and the filtrate was evaporated to dryness to yield 1.38 (103%) of crude ethyl 3-phenyl-5-(((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)methyl)isoxazole-4-carboxylate (intermediate <b>3</b>), which was used without further purification.</div><div class="NLM_p"><i>Step 4</i>. The crude intermediate <b>3</b> (1.30 g, 3.92 mmol), which was prepared as described above, was dissolved in methanol (39 mL), and cesium carbonate (2.56 g, 7.85 mmol) was added. The reaction mixture was stirred at an ambient temperature over the weekend. The solution was evaporated to dryness under reduced pressure, and the residue was divided between ethyl acetate (20 mL) and water (20 mL). The pH of the aqueous layer was adjusted to pH 2 with 37% aq HCl, and the layers were separated. The aqueous layer was washed with ethyl acetate (20 mL), the combined organic layers were dried with magnesium sulfate, and the organic solution was evaporated to dryness to yield 1.03 g (87%) of crude 3-phenyl-5-(((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)methyl)isoxazole-4-carboxylic acid (intermediate <b>4</b>). NMR (CDCl<sub>3</sub>): 8.19 ppm (broad s, 1H, COOH), 7.30–7.75 ppm (m, 5 H, arom), 5.09 ppm (q, 2H, CH<sub>2</sub>), 4.49 ppm (m, 1H, CH), 3.91 ppm (m, 1H, CH<sub>2</sub>), 3.61 ppm (m, 1H, CH<sub>2</sub>), 1.35–1.95 ppm (m, 6H, CH<sub>2</sub>).</div><div class="NLM_p"><i>Step 5</i>. Intermediate <b>4</b> (450 mg, 1.48 mmol) was dissolved in DMF (12 mL), and the solution cooled to 0–10 °C. 1-<i>N</i>-Ethylbenzene-1,4-diamine (220 mg, 1.62 mmol) and TBTU (577 mg, 1.80 mmol) were added, and the mixture was stirred at 0–10 °C for 10 min. Diisopropylethylamine (0.30 mL, 1.77 mmol) was slowly added, and the mixture was stirred at 0–10 °C for 1.5 h. Ethyl acetate (40 mL) and 5% aq NaHCO<sub>3</sub> solution (30 mL) were added to the reaction mixture, and the mixture was stirred at an ambient temperature for 15 min. The precipitated product was filtered, washed with ethyl acetate, and dried to yield 560 mg (89%) of <i>N</i>-(4-(ethylamino)phenyl)-3-phenyl-5-(((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)methyl)isoxazole-4-carboxamide (intermediate <b>5</b>), which was used in the final deprotection step. NMR (DMSO-<i>d</i><sub>6</sub>): 10.15 ppm (s, 1H, NH), 7.74 ppm (m, 2H, arom), 7.52 ppm (m, 3H arom), 7.32 ppm (d, 2H arom), 6.52 ppm (d, 2H, arom), 5.44 ppm (t, 1H, NH), 4.82 ppm (q, 2H, CH<sub>2</sub>), 4.7 ppm (m, 1H, CH), 3.72 ppm (m, 1H, CH<sub>2</sub>), 3.45 ppm (m, 1H, CH<sub>2</sub>), 3.00 ppm (q, 2H, CH<sub>2</sub>), 1.30–1.80 ppm (m, 6H, CH<sub>2</sub>), 1.00 ppm (t, 3H, CH<sub>3</sub>).</div><div class="NLM_p last"><i>Compound <b>32</b></i>. Intermediate <b>5</b> (70 mg, 0.17 mmol) was dissolved in THF (1.4 mL), the pH of the solution was adjusted to pH 3 with 2 M aq HCl solution, and the reaction mixture was stirred at an ambient temperature overnight. The temperature of the solution was then adjusted to 50 °C, and the stirring continued for 2 h. The pH was adjusted to 9–10 with 5 M aq NaOH solution, and water (5 mL) was added. The mixture was extracted with ethyl acetate (3 × 5 mL), and the combined organic layers were washed with water (3 × 3 mL), dried over sodium sulfate, and evaporated to dryness to yield 57 mg (102%) of crude compound <b>32</b>. The crude product was purified by chromatography using ethyl acetate–heptane 1:1 as the eluent to yield 39 mg (71%) of compound <b>32</b>. NMR (DMSO-<i>d</i><sub>6</sub>): 10.07 ppm (s, 1H, NH), 7.71 ppm (m, 2H, arom), 7.51 ppm (m, 3H arom), 7.31 ppm (d, 2H arom), 6.52 ppm (d, 2H, arom), 5.88 ppm (t, 1H, OH), 5.43 ppm (t, 1H, NH), 4.71 ppm (d, 2H, CH<sub>2</sub>), 3.00 ppm (q, 2H, CH<sub>2</sub>), 1.14 ppm (t, 3H, CH<sub>3</sub>). MS: <i>m</i>/<i>z</i> 338.1 (100%, M + 1), 360.2 (25%, M + Na).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Technique for the Detection of Aggregation</h3><div class="NLM_p last">The compounds of interest (i.e., compounds <b>3</b>, <b>4</b>, and <b>7</b>) were diluted in DMSO to obtain a stock of 100 mM, which was further diluted with DMSO to obtain four different concentrations (100, 30, 10, and 3 mM). From these concentrations 1 μL of solution was diluted to 1 mL of Dulbecco’s modified Eagle’s medium (DMEM, Sigma-Aldrich/Gibco) containing 10% fetal bovine serum (FBS, Gibco) and 1% penicillin–streptomycin (PS, Sigma-Aldrich) (100 U/mL and 0.1 mg/mL, respectively) referring the same conditions as in cell culture. Finally, an amount of 100 μL of this solution was transferred to a 96-well microplate (BRANDplates pureGrade). The final concentrations (i.e., 100, 30, 10, and 3 μM) were in 0.1% DMSO. For the measurements, DMEM with FBS and PS and 0.1% DMSO were used as a blank to account for any signals due to the assay conditions. The blank and all four concentrations of the samples were measured as triplicates at three different voltages (300, 400, and 500 V) using a Nepheloskan Ascent by Labsystems (Helsinki, Finland).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> In Vitro Chemical Stability of Compound <b>3</b></h3><div class="NLM_p last">HPLC–MS CHROMASOLV grade chloroform, acetonitrile, formic acid, and sodium chloride were obtained from Sigma (Darmstadt, Germany). Water was freshly prepared in-house with a Milli-Q (Millipore Oy, Espoo, Finland) purification system and was ultrapure grade (18.2 MΩ).</div></div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Extraction Method for the Extracellular (Cell Culture Media) Metabolites</h3><div class="NLM_p last">Eight samples with no cells were thawed at room temperature, and compound <b>3</b> was extracted from the cell culture media with 500 μL of chloroform. The samples were vortexed briefly and centrifuged at 16 000 rpm at 4 °C for 10 min. The bottom chloroform layer was transferred to a glass vial and analyzed by UPLC–ESI(+)-QTOF/MS in the sensitivity mode.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Extraction Method for the Intracellular Metabolites</h3><div class="NLM_p last">Eight samples with cell culture media and mouse embryonic stem cells were thawed on ice and centrifuged at 16 000 rpm at 4 °C for 10 min. The supernatants were removed, and cells were washed twice with 200 μL of 0.9% saline (H<sub>2</sub>O), followed by centrifugation at 16 000 rpm at 4 °C for 10 min. The washed cells were disrupted, and compound <b>3</b> was extracted with 200 μL of chloroform in an ultrasonic bath for 10 min, followed by centrifugation at 16 000 rpm at 4 °C for 10 min. The bottom chloroform phase was transferred to a glass vial and analyzed by UPLC–ESI(+)-QTOF/MS in the sensitivity mode.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Analytical Method</h3><div class="NLM_p last">A Waters Acquity ultraperformance liquid chromatographic (UPLC) system (Waters Corp., Milford, MA, USA) with an autosampler, vacuum degasser, and column oven was used. The analytical column used was a Waters Acquity BEH C18 (2.1 mm × 50 mm, 1.7 μm, Waters Corp, Milford, MA, USA), together with an online filter. The used eluents were 0.1% formic acid in H<sub>2</sub>O (A) and 0.1% formic acid in acetonitrile (B). A linear gradient elution of 7% B–80% B was applied for 5 min, followed by 1 min of column equilibration. The flow rate was 0.6 mL/min, and the column oven temperature was set to +40 °C. The LC–MS data were acquired using a Waters Synapt G2 quadrupole-time-of-flight (QTOF) high definition mass spectrometer (Waters Corp., Milford, MA, USA) equipped with a LockSpray electrospray ionization source. A positive ionization mode of electrospray was used with a cone voltage of 40 V and a mass range of <i>m</i>/<i>z</i> 100–600. The mass spectrometer and UPLC system were operated using MassLynx 4.1 software. Leucine enkephalin was used as a lock mass compound ([M + H]<sup>+</sup>: 556.2771).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Plasmids</h3><div class="NLM_p last">Rat BNP minimal promoter containing luciferase reporter vector and minimal promoter together with −90 tandem GATA-site have been described previously.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The plasmid expressing the mouse GATA4 (pMT2-GATA4) and the empty pMT2 plasmid were gifts from D. B. Wilson (Department of Pediatrics, St. Louis Children’s Hospital),<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> and the plasmid expressing the mouse NKX2-5 (pEF-FLAG-NKX2-5) was provided by R. P. Harvey (the Victor Chang Cardiac Research Institute, Darlinghurst, Australia).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The mouse NKX2-5 cDNA sequence was cloned from the pEF-plasmid to the pMT2-plasmid as previously described.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The p3xNKE-luc plasmid was inserted into the pGL3 basic reporter vector containing a rat albumin minimal promoter with a TATA-box (−40 to +28), which was a kind gift from J. Hakkola (Department of Pharmacology and Toxicology, University of Oulu, Finland)<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> as previously described.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The internal control vector pRL-TKd238 was a gift from J. F. Strauss III (University of Pennsylvania School of Medicine, Philadelphia).<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> In Vitro Screening</h3><div class="NLM_p last">In total, 800 small molecules were analyzed in vitro using an immunoprecipitation assay or a luciferase reporter gene assay specifically developed for the GATA4–NKX2-5 transcriptional synergy. For the luciferase reporter assay, mammalian COS-1 cells were cultured on 48-well plates (Greiner Bio One) in Dulbecco’s modified Eagle’s medium (Sigma-Aldrich/Gibco) containing 10% fetal bovine serum (Gibco) and 1% penicillin–streptomycin (Sigma-Aldrich) (100 U/mL and 0.1 mg/mL, respectively). In addition to co-transfecting all wells with the firefly luciferase reporter vector p3xNKE-luc and the internal control Renilla luciferase vector pRL-TKd238, one set of cells was transfected with the protein expression vector pMT2-GATA4, one set of cells was transfected with pMT2-NKX2-5, one set of cells was transfected with both pMT2-GATA4 and pMT2-NKX2-5, and the final set of cells was transfected with the empty control vector pMT2. For the transfections, the ratio of DNA/transfection reagent (Fugene 6, Roche Applied Science) was 1:3. The total plasmid DNA concentration was normalized across all wells by adding the empty vector pMT2. The compounds were added to the cells 6 h after the transfections, and each well contained 0.1% DMSO. The medium containing the compounds was changed to a fresh one 24 h after the transfection. Thirty hours after the transfection, the cells were lysed with 1× passive lysis buffer (E194A, Promega). The samples were processed on a dual-luciferase assay system (E1960, Promega) according to manufacturer’s protocol and measured with a luminometer (Luminoskan RS, Labsystems, Helsinki, Finland). Each compound was tested in two parallel samples. For the analysis of the results, the Firefly values were divided by the Renilla values and then normalized to the vehicle-treated control values.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> COS-1 Cell Culture</h3><div class="NLM_p last">COS-1 cells were cultured as previously described.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> For the nuclear protein extractions, the cells were plated in 25 cm<sup>2</sup> cell culture bottles 24 h before the transfection. The pMT2-GATA4 or pMT2-NKX2-5 plasmids were transfected as described above. After 48 h, the transfected cells were collected.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> BNP Reporter Assay</h3><div class="NLM_p last">COS-1 cells were cultured on 48-well plates (Greiner Bio One) in Dulbecco’s modified Eagle’s medium (Sigma-Aldrich/Gibco) containing 10% fetal bovine serum (Gibco) and 1% penicillin–streptomycin (Sigma-Aldrich) (100 U/mL and 0.1 mg/mL, respectively). In addition to co-transfecting all wells with the firefly luciferase reporter vector containing either BNP minimal promoter or −90 bp BNP promoter and the internal control Renilla luciferase vector pRL-TKd238, one set of cells was transfected with the protein expression vector pMT2-GATA4 and another set of cells was transfected with the empty vector pMT2. The compounds were added to the cells 6 h after the transfections; each well contained 0.1% DMSO. The medium was changed to a fresh one 24 h after the transfection, and also fresh compounds were added to the cells. Thirty hours after the transfection, the cells were lysed with 1× passive lysis buffer (E194A, Promega). The samples were processed on a dual-luciferase assay system (E1960, Promega) according to the manufacturer’s protocol and measured with a luminometer (Luminoskan RS, Labsystems, Helsinki, Finland). Each compound was tested in three parallel samples. For the analysis of the results, the firefly values were divided by the Renilla values and then normalized to the vehicle-treated control values.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Neonatal Cardiomyocyte Culture</h3><div class="NLM_p last">The cell culture reagents were obtained from Sigma-Aldrich unless otherwise stated. Primary cultures of neonatal rat ventricular cardiomyocytes were prepared from 2- to 4-day-old Sprague-Dawley or Wistar rats.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The animals were sacrificed by decapitation, and the ventricles were excised and cut into small pieces. The ventricle pieces were incubated at 37 °C by shaking for 2 h in a solution containing 100 mM NaCl, 10 mM KCl, 1.2 mM KH<sub>2</sub>PO<sub>4</sub>, 4.0 mM MgSO<sub>4</sub>, 50 mM taurine, 20 mM glucose, 10 mM Hepes, 2 mg/mL collagenase type 2 (Worthington, Lakewood), 2 mg/mL pancreatin, and 1% PS (Gibco). After the incubation, the cell suspension was collected into 15 mL tubes and centrifuged for 5 min at 160<i>g</i>. The supernatant and the top layer of the pellet containing the damaged cells were discarded, and the isolated cardiomyocytes were suspended in Dulbecco’s modified Eagle medium (DMEM)/F12 culture medium containing 2.5 mM <span class="smallcaps smallerCapital">l</span>-glutamine, supplemented with 1% PS and 10% fetal bovine serum (Gibco). The cells were preplated for 30–90 min on cell culture plates with a 10 cm diameter, and the unattached cardiomyocytes were collected with the medium. The number of viable cells was determined, and the cells were seeded onto six-well plates (Corning) at a density of (1.5–2) × 10<sup>5</sup> cells/well according to the assay used. The following day, the medium was replaced with complete serum free medium (CSFM; DMEM/F12, 2.5 mg/mL bovine serum albumin, 1 μM insulin, 2.5 mM <span class="smallcaps smallerCapital">l</span>-glutamine, 32 nM selenium, 2.8 mM sodium pyruvate, 5.64 μg/mL transferrin, 1 nM T3, and 100 IU/mL PS). The cells were cultured for another 24 h before adding the compounds. For the DNA binding assay, the cells were treated for 24 h with the compounds at a concentration of 50 μM. For the analysis of the hypertrophic gene expression, the cells were treated with 100 nM endothelin 1 (ET-1, Sigma-Aldrich) for 24 h, and compounds were added to the cells 1 h prior to addition of ET-1. The temperature in the culture chamber was 37 °C with 5% CO<sub>2</sub> and 95% air atmosphere.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Nuclear Protein Extraction</h3><div class="NLM_p last">After the appropriate treatments, the cultured cells were washed, and the protein extracts were prepared as previously described.<a onclick="showRef(event, 'ref14 ref71'); return false;" href="javascript:void(0);" class="ref ref14 ref71">(14,71)</a></div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Coimmunoprecipitation Assay</h3><div class="NLM_p last">GATA4 and NKX2-5 FLAG proteins as well the cell lysate without transfection were produced in COS-1 cells. Agarose bound anti-FLAG M2 antibody (Sigma) was incubated with proteins in lysis buffer (20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% Triton X-100, 2.5 mM sodium pyrophosphate) supplemented with protease and phosphatase inhibitors (200 μM sodium orthovanadate, 20 μg/mL leupeptin, 2 μg/mL pepstatin, 20 μg/mL aprotinin, 1 mM phenylmethanesulfonyl fluoride, 50 mM sodium fluoride, 6 μg/mL <i>N</i>-tosyl-<span class="smallcaps smallerCapital">l</span>-phenylalaninyl-chloromethyl ketone, 6 μg/mL <i>N</i>-α-tosyl-<span class="smallcaps smallerCapital">l</span>-lysinyl-chloromethyl ketone) overnight at +4 °C with gentle end-over-end mixing. The beads were collected by quick spin with a table top microcentrifuge and washed three times with lysis buffer. The immunoprecipitated proteins were eluted from the agarose beads by boiling the samples in SDS-loading buffer, resolved by SDS–PAGE, transferred onto Optitran BA-S 85 reinforced nitrocellulose membrane (Schleicher & Schuell) and immunoblotted with anti-GATA4 polyclonal antibody (sc-9053, Santa Cruz Biotechnology) and anti-NKX2-5 polyclonal antibody (sc-8697, Santa Cruz Biotechnology). Horseradish peroxidase conjugated anti-rabbit IgG or anti-goat IgG was used as secondary antibody. The immune complex was visualized by using an ECL detection kit (Amersham Pharmacia Biotech) followed by exposure to film (Hyperfilm ECL, Amersham Biosciences) or digitalization of chemiluminescence with luminescent imager analyzer LAS-3000 (Fujifilm). All results were quantitated by using Quantitiy One software (Bio-Rad).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Isolation and Analysis of RNA</h3><div class="NLM_p">Total RNA from the cultured cardiomyocytes was isolated with TRIzol reagent (Invitrogen) following the manufacturer’s protocol using the phase lock gel system (Eppendorf AG). For the quantitative real-time polymerase chain reaction (RT-PCR) analyses, cDNA was synthesized from the total RNA using a first-strand cDNA synthesis kit (GE Healthcare Life Sciences) following the manufacturer’s protocol. The RNA was analyzed by RT-PCR on an ABI 7300 sequence detection system (Applied Biosystems) using TaqMan chemistry. The sequences of the forward and reverse primers and fluorogenic probes for the RNA detection are shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. The results were normalized to 18S RNA quantified from the same samples.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Forward and Reverse Primers and Fluorogenic Probe Sequences Used for the Quantitative RT-PCR Analysis</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">gene</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">sequence</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ANP</td><td class="colsep0 rowsep0" align="left">forward</td><td class="colsep0 rowsep0" align="left">GAAAAGCAAACTGAGGGCTCTG</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">reverse</td><td class="colsep0 rowsep0" align="left">CCTACCCCCGAAGCAGCT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">fluorogenic probe</td><td class="colsep0 rowsep0" align="left">TCGCTGGCCCTCGGAGCCT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BNP</td><td class="colsep0 rowsep0" align="left">forward</td><td class="colsep0 rowsep0" align="left">TGGGCAGAAGATAGACCGGA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">reverse</td><td class="colsep0 rowsep0" align="left">ACAACCTCAGCCCGTCACAG</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">fluorogenic probe</td><td class="colsep0 rowsep0" align="left">CGGCGCAGTCAGTCGCTTGG</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18S</td><td class="colsep0 rowsep0" align="left">forward</td><td class="colsep0 rowsep0" align="left">TGGTTGCAAAGCTGAAACTTAAAG</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">reverse</td><td class="colsep0 rowsep0" align="left">AGTCAAATTAAGCCGCAGGC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">fluorogenic probe</td><td class="colsep0 rowsep0" align="left">CCTGGTGGTGCCCTTCCGTCA</td></tr></tbody></table></div></div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Mechanical Stretching of Cardiomyocytes</h3><div class="NLM_p last">After the cell isolation, the neonatal rat ventricular cardiomyocytes were seeded at a density of 10<sup>6</sup> cells/well on collagen I coated flexible-bottomed six-well plates (BioFlex collagen I, Flexcell International Corp., Burlington, NC) and cultured for 48 h before the treatments. The cardiomyocytes were exposed to the compounds or vehicle in CSFM and subjected to equibiaxial cyclic mechanical stretching (0.5 Hz, 10–20% elongation) for 48 h using a FlexCell tension system FX-5000 (Flexcell International Corp., Burlington, NC). Nonstretched cardiomyocytes seeded on similar BioFlex plates served as controls.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Immunofluorescence Staining</h3><div class="NLM_p last">To investigate the effect of the stretching on the cardiomyocyte size, the cells were washed twice with PBS and fixed with 4% paraformaldehyde for 20 min at room temperature (rt) immediately after the 48 h stretching. The cells were then washed with 0.2% bovine serum albumin in Dulbecco’s PBS (DB) and permeabilized with 0.1% Triton X-100 in PBS for 10 min at rt, followed by two 10 min washes in DB. Square pieces (10–15 mm × 10–15 mm) of the silicone membranes were then cut from the centers of the wells, incubated with a primary antibody against sarcomeric α-actinin (at 1:800 in DB, Sigma-Aldrich A7811) for 60 min at rt, washed three times for 10 min with DB, and incubated with the Alexa Fluor 488 anti-mouse secondary antibody (at 1:200 in DB, Life Technologies A-11029) and Alexa Fluor 647 phalloidin (at 1:140 in DB, Life Technologies A22287) for 45 min at rt. After three 10 min washes with DB, the specimens were mounted on microscopy slides using ProLong Gold Antifade Mountant with DAPI (Molecular Probes) and cured overnight at RT, after which they were stored at 4–8 °C.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Analysis of the Cell Surface Area</h3><div class="NLM_p last">For the cell surface area analysis, a minimum of six images of each stretched or nonstretched specimen were captured with a Hamamatsu Orca-Flash4.0 V2 camera (Hamamatsu Photonics, Hamamatsu, Japan) attached to a Leica DM6000B fluorescence microscope using 20×/0.7 HC PL APO CS2 objective and Leica Application Suite X software (Leica Microsystems, Wetzlar, Germany). The cell surface areas of the cardiomyocytes were determined from the images using CellProfiler software.<a onclick="showRef(event, 'ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref72 ref73">(72,73)</a> In brief, the nuclei were identified based on DAPI staining, followed by the detection of cell edges using an overlay of the fluorescence images of α-actinin and phalloidin. The cells were then filtered based on the α-actinin staining, and the surface areas of the α-actinin<sup>+</sup> cells (cardiomyocytes) were determined. The mean cell surface areas were calculated for each specimen (>200 cells/specimen) and normalized to the mean cell surface area of the untreated, nonstretched cardiomyocytes from the same experiment.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> DNA Binding Properties</h3><div class="NLM_p last">The effect of the compounds on the DNA binding abilities of GATA4 and NKX2-5 was studied by performing an electrophoretic mobility shift assay (EMSA). The GATA4 or NKX2-5 proteins used in the binding reactions were overexpressed proteins from the COS-1 cells or endogenous proteins from neonatal rat cardiomyocytes treated with the compounds for 24 h before the nuclear protein extraction. The nuclear protein extracts from the cells were incubated with a radioactively labeled double stranded oligonucleotide, which contained either the GATA4 or NKX2-5 specific binding site. For GATA4, the double-stranded oligonucleotide corresponded to the GATA binding region −90, i.e., Δ−68/–97 of the rat BNP promoter (GenBank code <a href="https://www.ncbi.nlm.nih.gov/gquery?Term=M60266">M60266</a>) as follows: rBNP −90 tandem GATA (5′-TGTGTCTGATAAATCAGAGATAACCCCACC-3′). For NKX2-5, the double-stranded oligonucleotide corresponded to the NKX2-5 binding element in the ANP promoter region (GenBank code <a href="https://www.ncbi.nlm.nih.gov/gquery?Term=M27498">M27498</a>) as follows: rANP −240 NKE-like element (5′-AGAGACCTTTGAAGTGGGGGCCTCTTGAGGCCCCG-3′). The probes were sticky-end-labeled with [α-<sup>32</sup>P]dCTP by Klenow enzyme. For each binding reaction mixture, the nuclear/total GATA4 or NKX2-5 protein was used in a final concentration of 16 mM HEPES, 120 mM NaCl, 0.67 mM EDTA, 0.3 mM EGTA, 8% glycerol, 0.02% NP-40, 40 mM KCl, 1 mM MgCl<sub>2</sub>, 0.1 μg/μL poly(dIxdC)<sub>2</sub>, 0.5 mM Tris-HCl (pH 7.5), 1 mM PMSF, 40 μg/mL aprotinin, 40 μg/mL leupeptin, and 4 μg/mL pepstatin. The final volume of the binding reaction was equalized to 20 μL with a high salt buffer. Reaction mixtures containing the nuclear proteins from the COS-1 cells were incubated with the compounds for 10 min before adding the labeled probes. After 20 min of incubating the nuclear protein with the probe, the protein–DNA mixtures were separated by nondenaturating gel electrophoresis on 5% polyacrylamide gel. For the supershift assay, either the GATA4 antibody (sc-1237X, Santa Cruz Biotechnology) or NKX2-5 antibody (sc-8697X, Santa Cruz Biotechnology) was incubated for 10 min with the protein sample before adding the compounds. Subsequently, the gels were dried and exposed to PhosphorImager screens (Molecular Dynamics, Sunnyvale, CA, USA). The screens were scanned using a Bio-Rad molecular imager FX Pro Plus and analyzed with Quantity One software (Bio-Rad Laboratories).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> In Vitro Cytotoxicity</h3><div class="NLM_p last">The release of adenylate kinase (AK) from the ruptured rat cardiac cells into the cell culture medium was analyzed using a ToxiLight bioassay kit (nondestructive cytotoxicity assay) from Lonza Rockland Inc. (Rockland, ME, USA) according to the manufacturer’s instructions. A luminometer (Luminoskan RS, Labsystems, Helsinki, Finland) was used to measure the bioluminescence of AK in the medium after 24 h of treatment with the compounds at a concentration of 50 μM. To detect the apoptotic myocytes, a terminal deoxynucleotidyl transferase-mediated dUTP nick end-labeling (TUNEL) assay was performed. The rat cardiac myocytes were plated onto Nunc Lab-Tek chamber slide system eight-well slides (Thermo Fisher Scientific Inc.). The cells were treated with the compounds at a concentration of 50 μM for 24 h. The apoptotic cells were detected using the ApopTag peroxidase in situ apoptosis detection kit (Millipore).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> In Vitro Kinase Profiling</h3><div class="NLM_p last">The effect of the compounds <b>3</b> and <b>4</b> on the activity of 48 kinases was assessed by Cerep’s ExpresS Diversity kinase profile service (Cerep, France). The compounds were tested at a concentration 30 μM in duplicate.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> GPCR Profiling</h3><div class="NLM_p last">The effect of the compound <b>3</b> in terms of GPCR agonism and antagonism was tested by GPCRProfiler (Millipore, Germany), which comprises cell-based functional assays for 165 GPCRs. The compound was used to test agonistic activities at a concentration of 12.5 μM and antagonistic activities at a concentration of 10 μM in duplicate.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Statistics</h3><div class="NLM_p last">The results are expressed as the mean ± standard deviation (SD). The statistical analyses were performed using SPSS version 22 (SPSS Inc., Chicago, IL, USA). To determine the significant difference between two groups, independent samples of Student’s <i>t</i>-tests were used. For multiple comparisons, the statistical significance was evaluated by one-way analysis of variance (ANOVA), followed by Bonferroni post hoc tests. The probability values of (∗) <i>p</i> < 0.05, (∗∗) <i>p</i> < 0.01, and (∗∗∗) <i>p</i> < 0.001 were considered statistically significant.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i58"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00816" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73186" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73186" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00816" class="ext-link">10.1021/acs.jmedchem.7b00816</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures and tables illustrating compounds aggregation, chemical stability, effects on GATA4 transcriptional activity, the protein kinases and the GPCRs, coimmunoprecipitation assay results, and the original EMSA gel samples (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf">jm7b00816_si_001.pdf (28.59 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_002.pdb">jm7b00816_si_002.pdb (94.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_003.csv">jm7b00816_si_003.csv (5.43 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00816" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83545" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83545" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heikki J. Ruskoaho</span> - <span class="hlFld-Affiliation affiliation">Division
of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Research
Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#09616c60626260277b7c7a626668616649616c657a60676260276f60"><span class="__cf_email__" data-cfemail="1d757874767674336f686e76727c75725d7578716e74737674337b74">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mika J. Välimäki</span> - <span class="hlFld-Affiliation affiliation">Division
of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Research
Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9687-8197" title="Orcid link">http://orcid.org/0000-0002-9687-8197</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marja A. Tölli</span> - <span class="hlFld-Affiliation affiliation">Research
Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sini M. Kinnunen</span> - <span class="hlFld-Affiliation affiliation">Division
of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Research
Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jani Aro</span> - <span class="hlFld-Affiliation affiliation">Research
Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raisa Serpi</span> - <span class="hlFld-Affiliation affiliation">Research
Unit of Biomedicine, Department of Pharmacology and Toxicology, University of Oulu, Oulu FI-90014, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lotta Pohjolainen</span> - <span class="hlFld-Affiliation affiliation">Division
of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Virpi Talman</span> - <span class="hlFld-Affiliation affiliation">Division
of Pharmacology and Pharmacotherapy, Faculty of Pharmacy, University of Helsinki, Helsinki FI-00014, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antti Poso</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Health Sciences, School of Pharmacy, University of Eastern Finland, Kuopio FI-70211, Finland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.J.V. and M.A.T. contributed equally to the work and are considered co-first-authors. H.J.R, M.J.V, M.A.T, S.M.K, and A.P. conceived and designed the experiments. M.J.V, M.A.T, S.M.K, J.A., R.S., L.P, and V.T. performed the experiments. M.J.V, M.A.T, S.M.K., J.A., R.S., H.J.R., A.P., L.P., and V.T. analyzed the data. The manuscript was written with contributions from all authors. All authors have approved the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ACK-d275e2061-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Marja Arbelius, Kirsi Salo, Nina Sipari, Niklas Johansson, and Gustav Boije af Gennäs for their expert technical assistance. This work was supported by the Finnish Funding Agency for Innovation (Tekes, 3iRegeneration, Project 40395/13), the Academy of Finland (Project 2666621), the Finnish Foundation for Cardiovascular Research, and the Sigrid Jusélius Foundation.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AK</td><td class="NLM_def"><p class="first last">adenylate kinase</p></td></tr><tr><td class="NLM_term">ANP</td><td class="NLM_def"><p class="first last">atrial natriuretic peptide</p></td></tr><tr><td class="NLM_term">BNP</td><td class="NLM_def"><p class="first last">B-type natriuretic peptide</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">FGFR3</td><td class="NLM_def"><p class="first last">fibroblast growth factor receptor 3 kinase</p></td></tr><tr><td class="NLM_term">FOG</td><td class="NLM_def"><p class="first last">Friend of GATA</p></td></tr><tr><td class="NLM_term">EBDM</td><td class="NLM_def"><p class="first last">embryoid body differentiation medium</p></td></tr><tr><td class="NLM_term">EMSA</td><td class="NLM_def"><p class="first last">electrophoretic mobility shift assay</p></td></tr><tr><td class="NLM_term">ET-1</td><td class="NLM_def"><p class="first last">endothelin-1</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>a</sub></td><td class="NLM_def"><p class="first last">synergy activation</p></td></tr><tr><td class="NLM_term">KDR</td><td class="NLM_def"><p class="first last">kinase insert domain receptor</p></td></tr><tr><td class="NLM_term">MEF2</td><td class="NLM_def"><p class="first last">myocyte-specific enhancer factor 2</p></td></tr><tr><td class="NLM_term">mESC</td><td class="NLM_def"><p class="first last">mouse embryonic stem cell</p></td></tr><tr><td class="NLM_term">MHC</td><td class="NLM_def"><p class="first last">myosin heavy chain</p></td></tr><tr><td class="NLM_term">NKE</td><td class="NLM_def"><p class="first last">NKX2-5 high affinity response element</p></td></tr><tr><td class="NLM_term">PMSF</td><td class="NLM_def"><p class="first last">phenylmethanesulfonyl fluoride</p></td></tr><tr><td class="NLM_term">PS</td><td class="NLM_def"><p class="first last">penicillin–streptomycin</p></td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">QTOF</td><td class="NLM_def"><p class="first last">quadrupole-time-of-flight</p></td></tr><tr><td class="NLM_term">ROCK1</td><td class="NLM_def"><p class="first last">Rho-associated protein kinase 1</p></td></tr><tr><td class="NLM_term">SRE</td><td class="NLM_def"><p class="first last">serum response element</p></td></tr><tr><td class="NLM_term">SRF</td><td class="NLM_def"><p class="first last">serum response factor</p></td></tr><tr><td class="NLM_term">TBTU</td><td class="NLM_def"><p class="first last"><i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium tetrafluoroborate</p></td></tr><tr><td class="NLM_term">TF</td><td class="NLM_def"><p class="first last">transcription factor</p></td></tr><tr><td class="NLM_term">TUNEL</td><td class="NLM_def"><p class="first last">terminal deoxynucleotidyl transferase dUTP nick end-labeling</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultraperformance liquid chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76664" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76664" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 73 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjian, R.</span></span> <span> </span><span class="NLM_article-title">Transcription Regulation and Animal Diversity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1038/nature01763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnature01763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12853946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlt1CktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2003&pages=147-151&author=M.+Levineauthor=R.+Tjian&title=Transcription+Regulation+and+Animal+Diversity&doi=10.1038%2Fnature01763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription regulation and animal diversity</span></div><div class="casAuthors">Levine, Michael; Tjian, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">6945</span>),
    <span class="NLM_cas:pages">147-151</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Whole-genome sequence assemblies are now available for seven different animals, including nematode worms, mice and humans.  Comparative genome analyses reveal a surprising constancy in genetic content: vertebrate genomes have only about twice the no. of genes that invertebrate genomes have, and the increase is primarily due to the duplication of existing genes rather than the invention of new ones.  How, then, has evolutionary diversity arisen Emerging evidence suggests that organismal complexity arises from progressively more elaborate regulation of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkOXHnchkm1rVg90H21EOLACvtfcHk0ljxvWsN5dY49A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlt1CktLw%253D&md5=7fa0d905a78a07135ef6610d9c6b1e07</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature01763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01763%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DM.%26aulast%3DTjian%26aufirst%3DR.%26atitle%3DTranscription%2520Regulation%2520and%2520Animal%2520Diversity%26jtitle%3DNature%26date%3D2003%26volume%3D424%26spage%3D147%26epage%3D151%26doi%3D10.1038%2Fnature01763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spitz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlong, E. E.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors: From Enhancer Binding to Developmental Control</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1038/nrg3207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnrg3207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=22868264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCrtrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=613-626&author=F.+Spitzauthor=E.+E.+Furlong&title=Transcription+Factors%3A+From+Enhancer+Binding+to+Developmental+Control&doi=10.1038%2Fnrg3207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors: from enhancer binding to developmental control</span></div><div class="casAuthors">Spitz, Francois; Furlong, Eileen E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">613-626</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Developmental progression is driven by specific spatiotemporal domains of gene expression, which give rise to stereotypically patterned embryos even in the presence of environmental and genetic variation.  Views of how transcription factors regulate gene expression are changing owing to recent genome-wide studies of transcription factor binding and RNA expression.  Such studies reveal patterns that, at first glance, seem to contrast with the robustness of the developmental processes they encode.  This review discusses the current knowledge of transcription factor function from genomic and genetic studies and how different strategies, including extensive cooperative regulation (both direct and indirect), progressive priming of regulatory elements, and the integration of activities from multiple enhancers, confer specificity and robustness to transcriptional regulation during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNokxJil_DoLVg90H21EOLACvtfcHk0lj-_N4ITfYe2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCrtrbM&md5=43cf42f36512a874d1c3c8f062fbd151</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg3207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3207%26sid%3Dliteratum%253Aachs%26aulast%3DSpitz%26aufirst%3DF.%26aulast%3DFurlong%26aufirst%3DE.%2BE.%26atitle%3DTranscription%2520Factors%253A%2520From%2520Enhancer%2520Binding%2520to%2520Developmental%2520Control%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D613%26epage%3D626%26doi%3D10.1038%2Fnrg3207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flores, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oviedo, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sage, J.</span></span> <span> </span><span class="NLM_article-title">Essential Role for the Planarian Intestinal GATA Transcription Factor in Stem Cells and Regeneration</span>. <i>Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>418</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1016/j.ydbio.2016.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.ydbio.2016.08.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=418&publication_year=2016&pages=179-188&author=N.+M.+Floresauthor=N.+J.+Oviedoauthor=J.+Sage&title=Essential+Role+for+the+Planarian+Intestinal+GATA+Transcription+Factor+in+Stem+Cells+and+Regeneration&doi=10.1016%2Fj.ydbio.2016.08.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ydbio.2016.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ydbio.2016.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DFlores%26aufirst%3DN.%2BM.%26aulast%3DOviedo%26aufirst%3DN.%2BJ.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DEssential%2520Role%2520for%2520the%2520Planarian%2520Intestinal%2520GATA%2520Transcription%2520Factor%2520in%2520Stem%2520Cells%2520and%2520Regeneration%26jtitle%3DDev.%2520Biol.%26date%3D2016%26volume%3D418%26spage%3D179%26epage%3D188%26doi%3D10.1016%2Fj.ydbio.2016.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">Requirement of the Transcription Factor GATA4 for Heart Tube Formation and Ventral Morphogenesis</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1072</span>, <span class="refDoi"> DOI: 10.1101/gad.11.8.1061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.11.8.1061" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=1061-1072&author=J.+D.+Molkentinauthor=Q.+Linauthor=S.+A.+Duncanauthor=E.+N.+Olson&title=Requirement+of+the+Transcription+Factor+GATA4+for+Heart+Tube+Formation+and+Ventral+Morphogenesis&doi=10.1101%2Fgad.11.8.1061"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fgad.11.8.1061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.11.8.1061%26sid%3Dliteratum%253Aachs%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DDuncan%26aufirst%3DS.%2BA.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DRequirement%2520of%2520the%2520Transcription%2520Factor%2520GATA4%2520for%2520Heart%2520Tube%2520Formation%2520and%2520Ventral%2520Morphogenesis%26jtitle%3DGenes%2520Dev.%26date%3D1997%26volume%3D11%26spage%3D1061%26epage%3D1072%26doi%3D10.1101%2Fgad.11.8.1061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R. P.</span></span> <span> </span><span class="NLM_article-title">Myogenic and Morphogenetic Defects in the Heart Tubes of Murine Embryos Lacking the Homeo Box Gene Nkx2-5</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1654</span>– <span class="NLM_lpage">1666</span>, <span class="refDoi"> DOI: 10.1101/gad.9.13.1654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.9.13.1654" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=1654-1666&author=I.+Lyonsauthor=L.+M.+Parsonsauthor=L.+Hartleyauthor=R.+Liauthor=J.+E.+Andrewsauthor=L.+Robbauthor=R.+P.+Harvey&title=Myogenic+and+Morphogenetic+Defects+in+the+Heart+Tubes+of+Murine+Embryos+Lacking+the+Homeo+Box+Gene+Nkx2-5&doi=10.1101%2Fgad.9.13.1654"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.9.13.1654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.9.13.1654%26sid%3Dliteratum%253Aachs%26aulast%3DLyons%26aufirst%3DI.%26aulast%3DParsons%26aufirst%3DL.%2BM.%26aulast%3DHartley%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DAndrews%26aufirst%3DJ.%2BE.%26aulast%3DRobb%26aufirst%3DL.%26aulast%3DHarvey%26aufirst%3DR.%2BP.%26atitle%3DMyogenic%2520and%2520Morphogenetic%2520Defects%2520in%2520the%2520Heart%2520Tubes%2520of%2520Murine%2520Embryos%2520Lacking%2520the%2520Homeo%2520Box%2520Gene%2520Nkx2-5%26jtitle%3DGenes%2520Dev.%26date%3D1995%26volume%3D9%26spage%3D1654%26epage%3D1666%26doi%3D10.1101%2Fgad.9.13.1654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horb, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, G. H.</span></span> <span> </span><span class="NLM_article-title">Tbx5 Is Essential for Heart Development</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1751</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=1999&pages=1739-1751&author=M.+E.+Horbauthor=G.+H.+Thomsen&title=Tbx5+Is+Essential+for+Heart+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHorb%26aufirst%3DM.%2BE.%26aulast%3DThomsen%26aufirst%3DG.%2BH.%26atitle%3DTbx5%2520Is%2520Essential%2520for%2520Heart%2520Development%26jtitle%3DDevelopment%26date%3D1999%26volume%3D126%26spage%3D1739%26epage%3D1751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grepin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">Enhanced Cardiogenesis in Embryonic Stem Cells Overexpressing the GATA-4 Transcription Factor</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">2387</span>– <span class="NLM_lpage">2395</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=1997&pages=2387-2395&author=C.+Grepinauthor=G.+Nemerauthor=M.+Nemer&title=Enhanced+Cardiogenesis+in+Embryonic+Stem+Cells+Overexpressing+the+GATA-4+Transcription+Factor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGrepin%26aufirst%3DC.%26aulast%3DNemer%26aufirst%3DG.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DEnhanced%2520Cardiogenesis%2520in%2520Embryonic%2520Stem%2520Cells%2520Overexpressing%2520the%2520GATA-4%2520Transcription%2520Factor%26jtitle%3DDevelopment%26date%3D1997%26volume%3D124%26spage%3D2387%26epage%3D2395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Lozano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M. D.</span></span> <span> </span><span class="NLM_article-title">Cardiac Muscle Regeneration: Lessons from Development</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1101/gad.2018411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.2018411" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=299-309&author=M.+Mercolaauthor=P.+Ruiz-Lozanoauthor=M.+D.+Schneider&title=Cardiac+Muscle+Regeneration%3A+Lessons+from+Development&doi=10.1101%2Fgad.2018411"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fgad.2018411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2018411%26sid%3Dliteratum%253Aachs%26aulast%3DMercola%26aufirst%3DM.%26aulast%3DRuiz-Lozano%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DM.%2BD.%26atitle%3DCardiac%2520Muscle%2520Regeneration%253A%2520Lessons%2520from%2520Development%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D299%26epage%3D309%26doi%3D10.1101%2Fgad.2018411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grepin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dagnino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haberstroh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antakly, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">A Hormone-Encoding Gene Identifies a Pathway for Cardiac but Not Skeletal Muscle Gene Transcription</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3115</span>– <span class="NLM_lpage">3129</span>, <span class="refDoi"> DOI: 10.1128/MCB.14.5.3115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.14.5.3115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8164667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK2cXltVWjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=3115-3129&author=C.+Grepinauthor=L.+Dagninoauthor=L.+Robitailleauthor=L.+Haberstrohauthor=T.+Antaklyauthor=M.+Nemer&title=A+Hormone-Encoding+Gene+Identifies+a+Pathway+for+Cardiac+but+Not+Skeletal+Muscle+Gene+Transcription&doi=10.1128%2FMCB.14.5.3115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A hormone-encoding gene identifies a pathway for cardiac but not skeletal muscle gene transcription</span></div><div class="casAuthors">Grepin, Claudine; Dagnino, Lina; Robitaille, Lynda; Haberstroh, Linda; Antakly, Tony; Nemer, Mona</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3115-29</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">In contrast to skeletal muscle, the mechanisms responsible for activation and maintenance of tissue-specific transcription in cardiac muscle remain poorly understood.  A family of hormone-encoding genes is expressed in a highly specific manner in cardiac but not skeletal myocytes.  This includes the A- and B-type natriuretic peptide (ANP and BNP) genes, which encode peptide hormones with crucial roles in the regulation of blood vol. and pressure.  Since these genes are markers of cardiac cells, they were used to probe the mechanisms for cardiac muscle-specific transcription.  Cloning and functional anal. of the rat BNP upstream sequences revealed unexpected structural resemblance to erythroid but not to muscle-specific promoters and enhancers, including a requirement for regulatory elements contg. GATA motifs.  A cDNA clone corresponding to a member of the GATA family of transcription factors was isolated from a cardiomyocyte cDNA library.  Transcription of this GATA gene is restricted mostly to the heart and is undetectable in skeletal muscle.  Within the heart, GATA transcripts are localized in ANP- and BNP-expressing myocytes, and forced expression of the GATA protein in heterologous cells markedly activates transcription from the natural cardiac muscle-specific ANP and BNP promoters.  This GATA-dependent pathway defines the first mechanism for cardiac muscle-specific transcription.  Moreover, the present findings reveal striking similarities between the mechanisms controlling gene expression in hematopoietic and cardiac cells and may have important implications for studies of cardiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJ_V6zoGTlrVg90H21EOLACvtfcHk0ljXV6QQRCFzNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltVWjtr8%253D&md5=cfbafcf23285c31cd7f1503f6e8ba60f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.5.3115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.5.3115%26sid%3Dliteratum%253Aachs%26aulast%3DGrepin%26aufirst%3DC.%26aulast%3DDagnino%26aufirst%3DL.%26aulast%3DRobitaille%26aufirst%3DL.%26aulast%3DHaberstroh%26aufirst%3DL.%26aulast%3DAntakly%26aufirst%3DT.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DA%2520Hormone-Encoding%2520Gene%2520Identifies%2520a%2520Pathway%2520for%2520Cardiac%2520but%2520Not%2520Skeletal%2520Muscle%2520Gene%2520Transcription%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D3115%26epage%3D3129%26doi%3D10.1128%2FMCB.14.5.3115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thuerauf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanford, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glembotski, C. C.</span></span> <span> </span><span class="NLM_article-title">Regulation of Rat Brain Natriuretic Peptide Transcription. A Potential Role for GATA-Related Transcription Factors in Myocardial Cell Gene Expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">17772</span>– <span class="NLM_lpage">17775</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=17772-17775&author=D.+J.+Thueraufauthor=D.+S.+Hanfordauthor=C.+C.+Glembotski&title=Regulation+of+Rat+Brain+Natriuretic+Peptide+Transcription.+A+Potential+Role+for+GATA-Related+Transcription+Factors+in+Myocardial+Cell+Gene+Expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThuerauf%26aufirst%3DD.%2BJ.%26aulast%3DHanford%26aufirst%3DD.%2BS.%26aulast%3DGlembotski%26aufirst%3DC.%2BC.%26atitle%3DRegulation%2520of%2520Rat%2520Brain%2520Natriuretic%2520Peptide%2520Transcription.%2520A%2520Potential%2520Role%2520for%2520GATA-Related%2520Transcription%2520Factors%2520in%2520Myocardial%2520Cell%2520Gene%2520Expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D17772%26epage%3D17775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalvakolanu, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span> <span> </span><span class="NLM_article-title">Transcription Factor GATA-4 Regulates Cardiac Muscle-Specific Expression of the Alpha-Myosin Heavy-Chain Gene</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">4947</span>– <span class="NLM_lpage">4957</span>, <span class="refDoi"> DOI: 10.1128/MCB.14.7.4947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.14.7.4947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8007990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK2cXlslajsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=4947-4957&author=J.+D.+Molkentinauthor=D.+V.+Kalvakolanuauthor=B.+E.+Markham&title=Transcription+Factor+GATA-4+Regulates+Cardiac+Muscle-Specific+Expression+of+the+Alpha-Myosin+Heavy-Chain+Gene&doi=10.1128%2FMCB.14.7.4947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor GATA-4 regulates cardiac muscle-specific expression of the α-myosin heavy-chain gene</span></div><div class="casAuthors">Molkentin, Jeffery D.; Kalvakolanu, Dhananjaya V.; Markham, Bruce E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4947-57</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">The α-myosin heavy-chain (α-MHC) gene is the major structural protein in the adult rodent myocardium.  Its expression is restricted to the heart by a complex interplay of trans-acting factors and their cis-acting sites.  However, to date, the factors that have been shown to regulate expression of this gene have also been found in skeletal muscle cells.  Recently, transcription factor GATA-4, which has a tissue distribution limited to the heart and endodermally derived tissues, was identified.  The authors recently found two putative GATA-binding sites within the proximal enhancer of the α-MHC gene, suggesting that GATA-4 might regulate its expression.  In this study, the authors establish that GATA-4 interacts with the α-MHC GATA sites to stimulate cardiac muscle-specific expression.  Mutation of the GATA-4-binding sites either individually or together decreased activity by 50 and 88% in the adult myocardium, resp.  GATA-4-dependent enhancement of activity from a heterologous promoter was mediated through the α-MHC GATA sites.  Co-injection of an α-MHC promoter construct with a GATA-4 expression vector permitted ectopic expression in skeletal muscle but not in fibroblasts.  Thus, the lack of α-MHC expression in skeletal muscle correlates with a lack of GATA-4.  GATA-4 DNA binding activity was significantly up-regulated in triiodothyronine- or retinoic acid-treated cardiomyocytes.  Putative GATA-4-binding sites are also found in the regulatory regions of other cardiac muscle-expressed structural genes.  This indicates a mechanism whereby triiodothyronine and retinoic acid can exert coordinate control of the cardiac phenotype through a trans-acting regulatory factor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-pweq57zFLVg90H21EOLACvtfcHk0lgttZhKl2vgIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlslajsLg%253D&md5=0b717840bb2aa8503fd9341900e352ef</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.7.4947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.7.4947%26sid%3Dliteratum%253Aachs%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DKalvakolanu%26aufirst%3DD.%2BV.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26atitle%3DTranscription%2520Factor%2520GATA-4%2520Regulates%2520Cardiac%2520Muscle-Specific%2520Expression%2520of%2520the%2520Alpha-Myosin%2520Heavy-Chain%2520Gene%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D4947%26epage%3D4957%26doi%3D10.1128%2FMCB.14.7.4947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jobe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitsis, R. N.</span></span> <span> </span><span class="NLM_article-title">Cis-Acting Sequences That Mediate Induction of Beta-Myosin Heavy Chain Gene Expression during Left Ventricular Hypertrophy due to Aortic Constriction</span>. <i>Circulation</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">3943</span>– <span class="NLM_lpage">3953</span>, <span class="refDoi"> DOI: 10.1161/01.CIR.96.11.3943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2F01.CIR.96.11.3943" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1997&pages=3943-3953&author=K.+Hasegawaauthor=S.+J.+Leeauthor=S.+M.+Jobeauthor=B.+E.+Markhamauthor=R.+N.+Kitsis&title=Cis-Acting+Sequences+That+Mediate+Induction+of+Beta-Myosin+Heavy+Chain+Gene+Expression+during+Left+Ventricular+Hypertrophy+due+to+Aortic+Constriction&doi=10.1161%2F01.CIR.96.11.3943"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1161%2F01.CIR.96.11.3943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.CIR.96.11.3943%26sid%3Dliteratum%253Aachs%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DJobe%26aufirst%3DS.%2BM.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26aulast%3DKitsis%26aufirst%3DR.%2BN.%26atitle%3DCis-Acting%2520Sequences%2520That%2520Mediate%2520Induction%2520of%2520Beta-Myosin%2520Heavy%2520Chain%2520Gene%2520Expression%2520during%2520Left%2520Ventricular%2520Hypertrophy%2520due%2520to%2520Aortic%2520Constriction%26jtitle%3DCirculation%26date%3D1997%26volume%3D96%26spage%3D3943%26epage%3D3953%26doi%3D10.1161%2F01.CIR.96.11.3943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pikkarainen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokola, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">GATA Transcription Factors in the Developing and Adult Heart</span>. <i>Cardiovasc. Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">196</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1016/j.cardiores.2004.03.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cardiores.2004.03.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2004&pages=196-207&author=S.+Pikkarainenauthor=H.+Tokolaauthor=R.+Kerkelaauthor=H.+Ruskoaho&title=GATA+Transcription+Factors+in+the+Developing+and+Adult+Heart&doi=10.1016%2Fj.cardiores.2004.03.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cardiores.2004.03.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cardiores.2004.03.025%26sid%3Dliteratum%253Aachs%26aulast%3DPikkarainen%26aufirst%3DS.%26aulast%3DTokola%26aufirst%3DH.%26aulast%3DKerkela%26aufirst%3DR.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DGATA%2520Transcription%2520Factors%2520in%2520the%2520Developing%2520and%2520Adult%2520Heart%26jtitle%3DCardiovasc.%2520Res.%26date%3D2004%26volume%3D63%26spage%3D196%26epage%3D207%26doi%3D10.1016%2Fj.cardiores.2004.03.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pikkarainen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokola, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majalahti-Palviainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hautala, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuolteenaho, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">GATA-4 Is a Nuclear Mediator of Mechanical Stretch-Activated Hypertrophic Program</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">23807</span>– <span class="NLM_lpage">23816</span>, <span class="refDoi"> DOI: 10.1074/jbc.M302719200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M302719200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=23807-23816&author=S.+Pikkarainenauthor=H.+Tokolaauthor=T.+Majalahti-Palviainenauthor=R.+Kerkelaauthor=N.+Hautalaauthor=S.+S.+Bhallaauthor=F.+Charronauthor=M.+Nemerauthor=O.+Vuolteenahoauthor=H.+Ruskoaho&title=GATA-4+Is+a+Nuclear+Mediator+of+Mechanical+Stretch-Activated+Hypertrophic+Program&doi=10.1074%2Fjbc.M302719200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M302719200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M302719200%26sid%3Dliteratum%253Aachs%26aulast%3DPikkarainen%26aufirst%3DS.%26aulast%3DTokola%26aufirst%3DH.%26aulast%3DMajalahti-Palviainen%26aufirst%3DT.%26aulast%3DKerkela%26aufirst%3DR.%26aulast%3DHautala%26aufirst%3DN.%26aulast%3DBhalla%26aufirst%3DS.%2BS.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DVuolteenaho%26aufirst%3DO.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DGATA-4%2520Is%2520a%2520Nuclear%2520Mediator%2520of%2520Mechanical%2520Stretch-Activated%2520Hypertrophic%2520Program%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D23807%26epage%3D23816%26doi%3D10.1074%2Fjbc.M302719200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Välimäki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tölli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohlfahrt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darwich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor-like Structure and Interaction of Congenital Heart Disease-Associated Factors GATA4 and NKX2-5</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0144145</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0144145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1371%2Fjournal.pone.0144145" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0144145&author=S.+Kinnunenauthor=M.+V%C3%A4lim%C3%A4kiauthor=M.+T%C3%B6lliauthor=G.+Wohlfahrtauthor=R.+Darwichauthor=H.+Komatiauthor=M.+Nemerauthor=H.+Ruskoaho&title=Nuclear+Receptor-like+Structure+and+Interaction+of+Congenital+Heart+Disease-Associated+Factors+GATA4+and+NKX2-5&doi=10.1371%2Fjournal.pone.0144145"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0144145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0144145%26sid%3Dliteratum%253Aachs%26aulast%3DKinnunen%26aufirst%3DS.%26aulast%3DV%25C3%25A4lim%25C3%25A4ki%26aufirst%3DM.%26aulast%3DT%25C3%25B6lli%26aufirst%3DM.%26aulast%3DWohlfahrt%26aufirst%3DG.%26aulast%3DDarwich%26aufirst%3DR.%26aulast%3DKomati%26aufirst%3DH.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DNuclear%2520Receptor-like%2520Structure%2520and%2520Interaction%2520of%2520Congenital%2520Heart%2520Disease-Associated%2520Factors%2520GATA4%2520and%2520NKX2-5%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0144145%26doi%3D10.1371%2Fjournal.pone.0144145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schueler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tönjes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperling, S. R.</span></span> <span> </span><span class="NLM_article-title">The Cardiac Transcription Network Modulated by Gata4, Mef2a, Nkx2.5, Srf, Histone Modifications, and microRNAs</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e1001313</span>, <span class="refDoi"> DOI: 10.1371/journal.pgen.1001313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1371%2Fjournal.pgen.1001313" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1001313&author=J.+Schlesingerauthor=M.+Schuelerauthor=M.+Grunertauthor=J.+J.+Fischerauthor=Q.+Zhangauthor=T.+Kruegerauthor=M.+Langeauthor=M.+T%C3%B6njesauthor=I.+Dunkelauthor=S.+R.+Sperling&title=The+Cardiac+Transcription+Network+Modulated+by+Gata4%2C+Mef2a%2C+Nkx2.5%2C+Srf%2C+Histone+Modifications%2C+and+microRNAs&doi=10.1371%2Fjournal.pgen.1001313"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1001313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1001313%26sid%3Dliteratum%253Aachs%26aulast%3DSchlesinger%26aufirst%3DJ.%26aulast%3DSchueler%26aufirst%3DM.%26aulast%3DGrunert%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DJ.%2BJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKrueger%26aufirst%3DT.%26aulast%3DLange%26aufirst%3DM.%26aulast%3DT%25C3%25B6njes%26aufirst%3DM.%26aulast%3DDunkel%26aufirst%3DI.%26aulast%3DSperling%26aufirst%3DS.%2BR.%26atitle%3DThe%2520Cardiac%2520Transcription%2520Network%2520Modulated%2520by%2520Gata4%252C%2520Mef2a%252C%2520Nkx2.5%252C%2520Srf%252C%2520Histone%2520Modifications%252C%2520and%2520microRNAs%26jtitle%3DPLoS%2520Genet.%26date%3D2011%26volume%3D7%26spage%3De1001313%26doi%3D10.1371%2Fjournal.pgen.1001313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsimiklis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">Tissue-Specific GATA Factors Are Transcriptional Effectors of the Small GTPase RhoA</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2702</span>– <span class="NLM_lpage">2719</span>, <span class="refDoi"> DOI: 10.1101/gad.915701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.915701" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=2702-2719&author=F.+Charronauthor=G.+Tsimiklisauthor=M.+Arcandauthor=L.+Robitailleauthor=Q.+Liangauthor=J.+D.+Molkentinauthor=S.+Melocheauthor=M.+Nemer&title=Tissue-Specific+GATA+Factors+Are+Transcriptional+Effectors+of+the+Small+GTPase+RhoA&doi=10.1101%2Fgad.915701"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1101%2Fgad.915701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.915701%26sid%3Dliteratum%253Aachs%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DTsimiklis%26aufirst%3DG.%26aulast%3DArcand%26aufirst%3DM.%26aulast%3DRobitaille%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DQ.%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DMeloche%26aufirst%3DS.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DTissue-Specific%2520GATA%2520Factors%2520Are%2520Transcriptional%2520Effectors%2520of%2520the%2520Small%2520GTPase%2520RhoA%26jtitle%3DGenes%2520Dev.%26date%3D2001%26volume%3D15%26spage%3D2702%26epage%3D2719%26doi%3D10.1101%2Fgad.915701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yanazume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, T.</span></span> <span> </span><span class="NLM_article-title">Cardiac p300 Is Involved in Myocyte Growth with Decompensated Heart Failure</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3593</span>– <span class="NLM_lpage">3606</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.10.3593-3606.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.23.10.3593-3606.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12724418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs1yhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=3593-3606&author=T.+Yanazumeauthor=K.+Hasegawaauthor=T.+Morimotoauthor=T.+Kawamuraauthor=H.+Wadaauthor=A.+Matsumoriauthor=Y.+Kawaseauthor=M.+Hiraiauthor=T.+Kita&title=Cardiac+p300+Is+Involved+in+Myocyte+Growth+with+Decompensated+Heart+Failure&doi=10.1128%2FMCB.23.10.3593-3606.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cardiac p300 is involved in myocyte growth with decompensated heart failure</span></div><div class="casAuthors">Yanazume, Tetsuhiko; Hasegawa, Koji; Morimoto, Tatsuya; Kawamura, Teruhisa; Wada, Hiromichi; Matsumori, Akira; Kawase, Yosuke; Hirai, Maretoshi; Kita, Toru</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3593-3606</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A variety of stresses on the heart initiate a no. of subcellular signaling pathways, which finally reach the nuclei of cardiac myocytes and cause myocyte hypertrophy with heart failure.  However, common nuclear pathways that lead to this state are unknown.  A zinc finger protein, GATA-4, is one of the transcription factors that mediate changes in gene expression during myocardial-cell hypertrophy.  P300 not only acts as a transcriptional coactivator of GATA-4, but also possesses an intrinsic histone acetyltransferase activity.  In primary cardiac myocytes derived from neonatal rats, we show that stimulation with phenylephrine increased an acetylated form of GATA-4 and its DNA-binding activity, as well as expression of p300.  A dominant-neg. mutant of p300 suppressed phenylephrine-induced nuclear acetylation, activation of GATA-4-dependent endothelin-1 promoters, and hypertrophic responses, such as increase in cell size and sarcomere organization.  In sharp contrast to the activation of cardiac MEK-1, which phosphorylates GATA-4 and causes compensated hypertrophy in vivo, p300-mediated acetylation of mouse cardiac nuclear proteins, including GATA-4, results in marked eccentric dilatation and systolic dysfunction.  These findings suggest that p300-mediated nuclear acetylation plays a crit. role in the development of myocyte hypertrophy and represents a pathway that leads to decompensated heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6-ZHbwghCMrVg90H21EOLACvtfcHk0lh4QAFLyZDxTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs1yhtrY%253D&md5=ec98d97f4ade96679377d95563ea6075</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.10.3593-3606.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.10.3593-3606.2003%26sid%3Dliteratum%253Aachs%26aulast%3DYanazume%26aufirst%3DT.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DMorimoto%26aufirst%3DT.%26aulast%3DKawamura%26aufirst%3DT.%26aulast%3DWada%26aufirst%3DH.%26aulast%3DMatsumori%26aufirst%3DA.%26aulast%3DKawase%26aufirst%3DY.%26aulast%3DHirai%26aufirst%3DM.%26aulast%3DKita%26aufirst%3DT.%26atitle%3DCardiac%2520p300%2520Is%2520Involved%2520in%2520Myocyte%2520Growth%2520with%2520Decompensated%2520Heart%2520Failure%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D3593%26epage%3D3606%26doi%3D10.1128%2FMCB.23.10.3593-3606.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span> <span> </span><span class="NLM_article-title">SUMO-1 Modification Activated GATA4-Dependent Cardiogenic Gene Activity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">49091</span>– <span class="NLM_lpage">49098</span>, <span class="refDoi"> DOI: 10.1074/jbc.M407494200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M407494200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=49091-49098&author=J.+Wangauthor=X.+H.+Fengauthor=R.+J.+Schwartz&title=SUMO-1+Modification+Activated+GATA4-Dependent+Cardiogenic+Gene+Activity&doi=10.1074%2Fjbc.M407494200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M407494200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M407494200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%2BH.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26atitle%3DSUMO-1%2520Modification%2520Activated%2520GATA4-Dependent%2520Cardiogenic%2520Gene%2520Activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D49091%26epage%3D49098%26doi%3D10.1074%2Fjbc.M407494200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado-Olguin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedantham, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruneau, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Direct Reprogramming of Fibroblasts into Functional Cardiomyocytes by Defined Factors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cell.2010.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=20691899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvFKrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2010&pages=375-386&author=M.+Iedaauthor=J.+D.+Fuauthor=P.+Delgado-Olguinauthor=V.+Vedanthamauthor=Y.+Hayashiauthor=B.+G.+Bruneauauthor=D.+Srivastava&title=Direct+Reprogramming+of+Fibroblasts+into+Functional+Cardiomyocytes+by+Defined+Factors&doi=10.1016%2Fj.cell.2010.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Direct reprogramming of fibroblasts into functional cardiomyocytes by defined factors</span></div><div class="casAuthors">Ieda, Masaki; Fu, Ji-Dong; Delgado-Olguin, Paul; Vedantham, Vasanth; Hayashi, Yohei; Bruneau, Benoit G.; Srivastava, Deepak</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-386</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The reprogramming of fibroblasts to induced pluripotent stem cells (iPSCs) raises the possibility that a somatic cell could be reprogrammed to an alternative differentiated fate without 1st becoming a stem/progenitor cell.  A large pool of fibroblasts exists in the postnatal heart, yet no single "master regulator" of direct cardiac reprogramming has been identified.  Here, the authors report that a combination of 3 developmental transcription factors (i.e., Gata4, Mef2c, and Tbx5) rapidly and efficiently reprogrammed murine postnatal cardiac or dermal fibroblasts directly into differentiated cardiomyocyte-like cells.  The induced cardiomyocytes expressed cardiac-specific markers, had a global gene expression profile similar to cardiomyocytes, and contracted spontaneously.  Fibroblasts transplanted into mouse hearts one day after transduction of the 3 factors also differentiated into cardiomyocyte-like cells.  The authors believe these findings demonstrate that functional cardiomyocytes can be directly reprogrammed from differentiated somatic cells by defined factors.  Reprogramming of endogenous or explanted fibroblasts might provide a source of cardiomyocytes for regenerative approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruiY4LjlUFW7Vg90H21EOLACvtfcHk0lh4QAFLyZDxTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvFKrsbY%253D&md5=9f3647f4dfc98eb0df6aa28bd2ebdc30</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DIeda%26aufirst%3DM.%26aulast%3DFu%26aufirst%3DJ.%2BD.%26aulast%3DDelgado-Olguin%26aufirst%3DP.%26aulast%3DVedantham%26aufirst%3DV.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DBruneau%26aufirst%3DB.%2BG.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DDirect%2520Reprogramming%2520of%2520Fibroblasts%2520into%2520Functional%2520Cardiomyocytes%2520by%2520Defined%2520Factors%26jtitle%3DCell%26date%3D2010%26volume%3D142%26spage%3D375%26epage%3D386%26doi%3D10.1016%2Fj.cell.2010.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallquist, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neilson, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassel-Duby, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">Heart Repair by Reprogramming Non-Myocytes with Cardiac Transcription Factors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>485</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1038/nature11139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnature11139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=22660318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVyqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=485&publication_year=2012&pages=599-604&author=K.+Songauthor=Y.+J.+Namauthor=X.+Luoauthor=X.+Qiauthor=W.+Tanauthor=G.+N.+Huangauthor=A.+Acharyaauthor=C.+L.+Smithauthor=M.+D.+Tallquistauthor=E.+G.+Neilsonauthor=J.+A.+Hillauthor=R.+Bassel-Dubyauthor=E.+N.+Olson&title=Heart+Repair+by+Reprogramming+Non-Myocytes+with+Cardiac+Transcription+Factors&doi=10.1038%2Fnature11139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Heart repair by reprogramming non-myocytes with cardiac transcription factors</span></div><div class="casAuthors">Song, Kunhua; Nam, Young-Jae; Luo, Xiang; Qi, Xiaoxia; Tan, Wei; Huang, Guo N.; Acharya, Asha; Smith, Christopher L.; Tallquist, Michelle D.; Neilson, Eric G.; Hill, Joseph A.; Bassel-Duby, Rhonda; Olson, Eric N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">485</span>
        (<span class="NLM_cas:issue">7400</span>),
    <span class="NLM_cas:pages">599-604</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The adult mammalian heart possesses little regenerative potential following injury.  Fibrosis due to activation of cardiac fibroblasts impedes cardiac regeneration and contributes to loss of contractile function, pathol. remodelling and susceptibility to arrhythmias.  Cardiac fibroblasts account for a majority of cells in the heart and represent a potential cellular source for restoration of cardiac function following injury through phenotypic reprogramming to a myocardial cell fate.  Four transcription factors, GATA4, HAND2, MEF2C and TBX5, can cooperatively reprogram adult mouse tail-tip and cardiac fibroblasts into beating cardiac-like myocytes in vitro.  Forced expression of these factors in dividing non-cardiomyocytes in mice reprograms these cells into functional cardiac-like myocytes, improves cardiac function and reduces adverse ventricular remodelling following myocardial infarction.  The authors' results suggest a strategy for cardiac repair through reprogramming fibroblasts resident in the heart with cardiogenic transcription factors or other mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrcg4ogoAAbrVg90H21EOLACvtfcHk0lh4QAFLyZDxTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvVyqsbg%253D&md5=8e1285e15f1274a19d26964291fa9f46</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature11139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11139%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DK.%26aulast%3DNam%26aufirst%3DY.%2BJ.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DQi%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DG.%2BN.%26aulast%3DAcharya%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DC.%2BL.%26aulast%3DTallquist%26aufirst%3DM.%2BD.%26aulast%3DNeilson%26aufirst%3DE.%2BG.%26aulast%3DHill%26aufirst%3DJ.%2BA.%26aulast%3DBassel-Duby%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DHeart%2520Repair%2520by%2520Reprogramming%2520Non-Myocytes%2520with%2520Cardiac%2520Transcription%2520Factors%26jtitle%3DNature%26date%3D2012%26volume%3D485%26spage%3D599%26epage%3D604%26doi%3D10.1038%2Fnature11139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demaria, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yankner, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campisi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span> <span> </span><span class="NLM_article-title">The DNA Damage Response Induces Inflammation and Senescence by Inhibiting Autophagy of GATA4</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>349</i></span>,  <span class="NLM_fpage">aaa5612</span>, <span class="refDoi"> DOI: 10.1126/science.aaa5612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1126%2Fscience.aaa5612" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2015&pages=aaa5612&author=C.+Kangauthor=Q.+Xuauthor=T.+D.+Martinauthor=M.+Z.+Liauthor=M.+Demariaauthor=L.+Aronauthor=T.+Luauthor=B.+A.+Yanknerauthor=J.+Campisiauthor=S.+J.+Elledge&title=The+DNA+Damage+Response+Induces+Inflammation+and+Senescence+by+Inhibiting+Autophagy+of+GATA4&doi=10.1126%2Fscience.aaa5612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa5612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa5612%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMartin%26aufirst%3DT.%2BD.%26aulast%3DLi%26aufirst%3DM.%2BZ.%26aulast%3DDemaria%26aufirst%3DM.%26aulast%3DAron%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DYankner%26aufirst%3DB.%2BA.%26aulast%3DCampisi%26aufirst%3DJ.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520DNA%2520Damage%2520Response%2520Induces%2520Inflammation%2520and%2520Senescence%2520by%2520Inhibiting%2520Autophagy%2520of%2520GATA4%26jtitle%3DScience%26date%3D2015%26volume%3D349%26spage%3Daaa5612%26doi%3D10.1126%2Fscience.aaa5612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Hachem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, G.</span></span> <span> </span><span class="NLM_article-title">Identification of New GATA4-Small Molecule Inhibitors by Structure-Based Virtual Screening</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1734</span>– <span class="NLM_lpage">1742</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.bmc.2011.01.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1734-1742&author=N.+El-Hachemauthor=G.+Nemer&title=Identification+of+New+GATA4-Small+Molecule+Inhibitors+by+Structure-Based+Virtual+Screening&doi=10.1016%2Fj.bmc.2011.01.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Hachem%26aufirst%3DN.%26aulast%3DNemer%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520New%2520GATA4-Small%2520Molecule%2520Inhibitors%2520by%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D1734%26epage%3D1742%26doi%3D10.1016%2Fj.bmc.2011.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">Combinatorial Interactions Regulating Cardiac Transcription</span>. <i>Dev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1520-6408(1998)22:3<250::AID-DVG7>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=9621432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK1cXjvVCit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1998&pages=250-262&author=D.+Durocherauthor=M.+Nemer&title=Combinatorial+Interactions+Regulating+Cardiac+Transcription&doi=10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial interactions regulating cardiac transcription</span></div><div class="casAuthors">Durocher, Daniel; Nemer, Mona</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Genetics (New York)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">250-262</span>CODEN:
                <span class="NLM_cas:coden">DGNTDW</span>;
        ISSN:<span class="NLM_cas:issn">0192-253X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">In vertebrates, heart development is a multistep process that starts with formation and patterning of the primitive heart tube and is followed by complex morphol. events to give rise to the mature four-chambered heart.  These various stages are characterized by distinct patterns of gene expression.  Although chamber specificity and developmental regulation can be demonstrated in transgenic mice using short promoter fragments, the mechanism underlying spatial and temporal specificity within the heart remains largely unclear.  Combinatorial interaction between a limited no. of cardiac-specific and ubiquitous transcription factors may account for the diverse genetic inputs required to generate the complex transcriptional patterns that characterize the developing myocardium.  We have used the cardiac atrial natriuretic peptide (ANP) promoter to test this hypothesis.  The ANP gene is transcribed in a spatial-and temporal-specific manner in the heart, and a 500 bp promoter fragment is sufficient to recapitulate both chamber and developmental specificity.  This promoter is composed of three modules, a "basal" cardiac promoter that is essential for transcription in embryonic and postnatal atrial and ventricular myocytes and two other independent modules that behave as chamber-specific enhancers.  The basal cardiac promoter is the target of two cardiac-specific transcription factors, the zinc finger GATA-4 protein and the Nkx2-5 homeodomain, which bind to contiguous elements within this region.  At low concns., a situation that likely occurs during the very first stages of cardiac cell fate detn., the two proteins synergistically activate transcription from the ANP promoter.  This functional synergy requires phys. interaction between the GATA-4 protein and an extended C-terminal homeodomain on Nkx2-5.  This interaction, which unmasks an activation domain present just N-terminal of the homeodomain, is specific for GATA-4 and -5, but is not obsd. with the other cardiac GATA factor, GATA-6.  Optimal synergy requires binding of both proteins to their cognate sites, although modest synergy also could be obsd. on heterologous promoters contg. only multimerized Nkx binding sites, suggesting that Nkx2-5 is able to recruit GATA-4 into a transcriptionally active complex.  The GATA/Nkx interaction, which appears to have been evolutionary conserved in nematode, fly, and mammals, provides a paradigm for analyzing transcription factor interaction during organogenesis.  The data are also discussed in the context of our present knowledge of the roles of GATA and NK2 proteins in cardiac development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgaAVlHUEwzbVg90H21EOLACvtfcHk0lilf7wgXhupdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvVCit7g%253D&md5=09c172bc8995035f548288ee0baae7d2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291520-6408%25281998%252922%253A3%253C250%253A%253AAID-DVG7%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DCombinatorial%2520Interactions%2520Regulating%2520Cardiac%2520Transcription%26jtitle%3DDev.%2520Genet.%26date%3D1998%26volume%3D22%26spage%3D250%26epage%3D262%26doi%3D10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sepulveda, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belaguli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span> <span> </span><span class="NLM_article-title">GATA-4 and Nkx-2.5 Coactivate Nkx-2 DNA Binding Targets: Role for Regulating Early Cardiac Gene Expression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3405</span>– <span class="NLM_lpage">3415</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.6.3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.18.6.3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=9584181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK1cXjsFSrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=3405-3415&author=J.+L.+Sepulvedaauthor=N.+Belaguliauthor=V.+Nigamauthor=C.+Y.+Chenauthor=M.+Nemerauthor=R.+J.+Schwartz&title=GATA-4+and+Nkx-2.5+Coactivate+Nkx-2+DNA+Binding+Targets%3A+Role+for+Regulating+Early+Cardiac+Gene+Expression&doi=10.1128%2FMCB.18.6.3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GATA-4 and Nkx-2.5 coactivate Nkx-2 DNA binding targets: role for regulating early cardiac gene expression</span></div><div class="casAuthors">Sepulveda, Jorge L.; Belaguli, Narashimaswamy; Nigam, Vishal; Chen, Ching-Yi; Nemer, Mona; Schwartz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3405-3415</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The cardiogenic homeodomain factor Nkx-2.5 and serum response factor (SRF) provide strong transcriptional coactivation of the cardiac α-actin (αCA) promoter in fibroblasts (C. Y. Chen and R. J. Schwartz, Mol. Cell. Biol. 16:6372-6384, 1996).  We demonstrate here that Nkx-2.5 also cooperates with GATA-4, a dual C-4 zinc finger transcription factor expressed in early cardiac progenitor cells, to activate the αCA promoter and a minimal promoter, contg. only multimerized Nkx-2.5 DNA bindings sites (NKEs), in heterologous CV-1 fibroblasts.  Transcriptional activity requires the N-terminal activation domain of Nkx-2.5 and Nkx-2.5 binding activity through its homeodomain but does not require GATA-4's activation domain.  The minimal interactive regions were mapped to the homeodomain of Nkx-2.5 and the second zinc finger of GATA-4.  Removal of Nkx-2.5's C-terminal inhibitory domain stimulated robust transcriptional activity, comparable to the effects of GATA-4 on wild-type Nkx-2.5, which in part facilitated Nkx-2.5 DNA binding activity.  We postulate the following simple model: GATA-4 induces a conformational change in Nkx-2.5 that displaces the C-terminal inhibitory domain, thus eliciting transcriptional activation of promoters contg. Nkx-2.5 DNA binding targets.  Therefore, αCa promoter activity appears to be regulated through the combinatorial interactions of at least three cardiac tissue-enriched transcription factors, Nkx-2.5, GATA-4, and SRF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQzwtdzWGPbLVg90H21EOLACvtfcHk0lilf7wgXhupdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjsFSrsrc%253D&md5=c64eb458510a7c00ad89088a7bd81172</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.6.3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.6.3405%26sid%3Dliteratum%253Aachs%26aulast%3DSepulveda%26aufirst%3DJ.%2BL.%26aulast%3DBelaguli%26aufirst%3DN.%26aulast%3DNigam%26aufirst%3DV.%26aulast%3DChen%26aufirst%3DC.%2BY.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26atitle%3DGATA-4%2520and%2520Nkx-2.5%2520Coactivate%2520Nkx-2%2520DNA%2520Binding%2520Targets%253A%2520Role%2520for%2520Regulating%2520Early%2520Cardiac%2520Gene%2520Expression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D3405%26epage%3D3415%26doi%3D10.1128%2FMCB.18.6.3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span>; <span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Interferences with Luciferase Reporter Enzymes</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, G.</span>, <span class="NLM_string-name">Coussens, N.</span>, <span class="NLM_string-name">Brimacombe, K.</span>, <span class="NLM_string-name">Grossman, A.</span>, <span class="NLM_string-name">Arkin, M.</span>, <span class="NLM_string-name">Auld, D.</span>, <span class="NLM_string-name">Austin, C.</span>, <span class="NLM_string-name">Baell, J.</span>, <span class="NLM_string-name">Bejcek, B.</span>, <span class="NLM_string-name">Chung, T.</span>, <span class="NLM_string-name">Dahlin, J.</span>, <span class="NLM_string-name">Devanaryan, V.</span>, <span class="NLM_string-name">Foley, T.</span>, <span class="NLM_string-name">Glicksman, M.</span>, <span class="NLM_string-name">Hall, M.</span>, <span class="NLM_string-name">Hass, J.</span>, <span class="NLM_string-name">Inglese, J.</span>, <span class="NLM_string-name">Iversen, P.</span>, <span class="NLM_string-name">Kahl, S.</span>, <span class="NLM_string-name">Kales, S.</span>, <span class="NLM_string-name">Lal-Nag, M.</span>, <span class="NLM_string-name">Li, Z.</span>, <span class="NLM_string-name">McGee, J.</span>, <span class="NLM_string-name">McManus, O.</span>, <span class="NLM_string-name">Xu, X.</span></span>, Eds.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2016</span>; Vol.  <span class="NLM_volume">1</span>, pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <a href="http://www.ncbi.nlm.nih.gov/books/NBK374281/" class="extLink">http://www.ncbi.nlm.nih.gov/books/NBK374281/</a> (accessed August 10, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1-14&author=D.+S.+Auld&author=J.+Ingleseauthor=G.+Sittampalam&author=N.+Coussens&author=K.+Brimacombe&author=A.+Grossman&author=M.+Arkin&author=D.+Auld&author=C.+Austin&author=J.+Baell&author=B.+Bejcek&author=T.+Chung&author=J.+Dahlin&author=V.+Devanaryan&author=T.+Foley&author=M.+Glicksman&author=M.+Hall&author=J.+Hass&author=J.+Inglese&author=P.+Iversen&author=S.+Kahl&author=S.+Kales&author=M.+Lal-Nag&author=Z.+Li&author=J.+McGee&author=O.+McManus&author=X.+Xu&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAuld%26aufirst%3DD.%2BS.%26atitle%3DInterferences%2520with%2520Luciferase%2520Reporter%2520Enzymes%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DG.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%26date%3D2016%26volume%3D1%26spage%3D1%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, M.</span></span> <span> </span><span class="NLM_article-title">Functionality Maps of Binding Sites: A Multiple Copy Simultaneous Search Method</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1002/prot.340110104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2Fprot.340110104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1961699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=29-34&author=A.+Mirankerauthor=M.+Karplus&title=Functionality+Maps+of+Binding+Sites%3A+A+Multiple+Copy+Simultaneous+Search+Method&doi=10.1002%2Fprot.340110104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Functionality maps of binding sites:  a multiple copy simultaneous search method</span></div><div class="casAuthors">Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-34</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">A new method is proposed for detg. energetically favorable positions and orientations for functional groups on the surface of proteins with known 3-dimensional structure.  Some 1000-5000 copies of a functional group are randomly placed in the site and subjected to simultaneous energy minimization and/or quenched mol. dynamics.  The resulting functionality maps of a protein receptor site, which can take account of its flexibility, can be used for the anal. of protein ligand interactions and rational drug design.  Application of the method to the sialic acid binding site of the influenza coat protein, hemagglutinin, yields functional group min. that correspond with those of the ligand in a cocrystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5im8BqJfF37Vg90H21EOLACvtfcHk0li06ODr4hXxlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D&md5=34e0a0c36899bfeabcbdfd4790f97a1e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fprot.340110104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340110104%26sid%3Dliteratum%253Aachs%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DFunctionality%2520Maps%2520of%2520Binding%2520Sites%253A%2520A%2520Multiple%2520Copy%2520Simultaneous%2520Search%2520Method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1991%26volume%3D11%26spage%3D29%26epage%3D34%26doi%3D10.1002%2Fprot.340110104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, S.</span>; <span class="NLM_string-name">Tölli, M.</span>; <span class="NLM_string-name">Välimäki, M.</span>; <span class="NLM_string-name">Jumppanen, M.</span>; <span class="NLM_string-name">Boije af Gennäs, G.</span>; <span class="NLM_string-name">Yli-Kauhaluoma, J.</span>; <span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical Compounds</span>. Patent Application 20165712, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=S.+Kinnunen&author=M.+T%C3%B6lli&author=M.+V%C3%A4lim%C3%A4ki&author=M.+Jumppanen&author=G.+Boije+af+Genn%C3%A4s&author=J.+Yli-Kauhaluoma&author=H.+Ruskoaho&title=Pharmaceutical+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKinnunen%26aufirst%3DS.%26atitle%3DPharmaceutical%2520Compounds%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shioi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasahara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jobe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiese, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumo, S.</span></span> <span> </span><span class="NLM_article-title">The Cardiac Tissue-Restricted Homeobox Protein Csx/Nkx2.5 Physically Associates with the Zinc Finger Protein GATA4 and Cooperatively Activates Atrial Natriuretic Factor Gene Expression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3120</span>– <span class="NLM_lpage">3129</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.6.3120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.18.6.3120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=9584153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK1cXjsFSkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=3120-3129&author=Y.+Leeauthor=T.+Shioiauthor=H.+Kasaharaauthor=S.+M.+Jobeauthor=R.+J.+Wieseauthor=B.+E.+Markhamauthor=S.+Izumo&title=The+Cardiac+Tissue-Restricted+Homeobox+Protein+Csx%2FNkx2.5+Physically+Associates+with+the+Zinc+Finger+Protein+GATA4+and+Cooperatively+Activates+Atrial+Natriuretic+Factor+Gene+Expression&doi=10.1128%2FMCB.18.6.3120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The cardiac tissue-restricted homeobox protein Csx/Nkx2.5 physically associates with the zinc finger protein GATA4 and cooperatively activates atrial natriuretic factor gene expression</span></div><div class="casAuthors">Lee, Youngsook; Shioi, Tetsuo; Kasahara, Hideko; Jobe, Shawn M.; Wiese, Russell J.; Markham, Bruce E.; Izumo, Seigo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3120-3129</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Specification and differentiation of the cardiac muscle lineage appear to require a combinatorial network of many factors.  The cardiac muscle-restricted homeobox protein Csx/Nkx2.5 (Csx) is expressed in the precardiac mesoderm as well as the embryonic and adult heart.  Targeted disruption of Csx causes embryonic lethality due to abnormal heart morphogenesis.  The zinc finger transcription factor GATA4 is also expressed in the heart and has been shown to be essential for heart tube formation.  GATA4 is known to activate many cardiac tissue-restricted genes.  In this study, the authors tested whether Csx and GATA4 phys. assoc. and cooperatively activate transcription of a target gene.  Coimmunopptn. expts. demonstrate that Csx and GATA4 assoc. intracellularly.  Interestingly, in vitro protein-protein interaction studies indicate that helix III of the homeodomain of Csx is required to interact with GATA4 and that the carboxy-terminal zinc finger of GATA4 is necessary to assoc. with Csx.  Both regions are known to directly contact the cognate DNA sequences.  The promoter-enhancer region of the atrial natriuretic factor (ANF) contains several putative Csx binding sites and consensus GATA4 binding sites.  Transient-transfection assays indicate that Csx can activate ANF reporter gene expression to the same extent that GATA4 does in a DNA binding site-dependent manner.  Coexpression of Csx and GATA4 synergistically activates ANF reporter gene expression.  Mutational analyses suggest that this synergy requires both factors to fully retain their transcriptional activities, including the cofactor binding activity.  These results demonstrate the first example of homeoprotein and zinc finger protein interaction in vertebrates to cooperatively regulate target gene expression.  Such synergistic interaction among tissue-restricted transcription factors may be an important mechanism to reinforce tissue-specific developmental pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGSPRokpTxqbVg90H21EOLACvtfcHk0li06ODr4hXxlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjsFSkur8%253D&md5=dbf87dd868e0988f72ceb04834e8a50c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.6.3120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.6.3120%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%26aulast%3DShioi%26aufirst%3DT.%26aulast%3DKasahara%26aufirst%3DH.%26aulast%3DJobe%26aufirst%3DS.%2BM.%26aulast%3DWiese%26aufirst%3DR.%2BJ.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26aulast%3DIzumo%26aufirst%3DS.%26atitle%3DThe%2520Cardiac%2520Tissue-Restricted%2520Homeobox%2520Protein%2520Csx%252FNkx2.5%2520Physically%2520Associates%2520with%2520the%2520Zinc%2520Finger%2520Protein%2520GATA4%2520and%2520Cooperatively%2520Activates%2520Atrial%2520Natriuretic%2520Factor%2520Gene%2520Expression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D3120%26epage%3D3129%26doi%3D10.1128%2FMCB.18.6.3120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">The Cardiac Transcription Factors Nkx2-5 and GATA-4 Are Mutual Cofactors</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">5687</span>– <span class="NLM_lpage">5696</span>, <span class="refDoi"> DOI: 10.1093/emboj/16.18.5687</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1093%2Femboj%2F16.18.5687" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=5687-5696&author=D.+Durocherauthor=F.+Charronauthor=R.+Warrenauthor=R.+J.+Schwartzauthor=M.+Nemer&title=The+Cardiac+Transcription+Factors+Nkx2-5+and+GATA-4+Are+Mutual+Cofactors&doi=10.1093%2Femboj%2F16.18.5687"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F16.18.5687&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F16.18.5687%26sid%3Dliteratum%253Aachs%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DWarren%26aufirst%3DR.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DThe%2520Cardiac%2520Transcription%2520Factors%2520Nkx2-5%2520and%2520GATA-4%2520Are%2520Mutual%2520Cofactors%26jtitle%3DEMBO%2520J.%26date%3D1997%26volume%3D16%26spage%3D5687%26epage%3D5696%26doi%3D10.1093%2Femboj%2F16.18.5687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuolteenaho, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uusimaa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Passive Mechanical Stretch Releases Atrial Natriuretic Peptide from Rat Ventricular Myocardium</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1253</span>, <span class="refDoi"> DOI: 10.1161/01.RES.70.6.1244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2F01.RES.70.6.1244" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1992&pages=1244-1253&author=P.+Kinnunenauthor=O.+Vuolteenahoauthor=P.+Uusimaaauthor=H.+Ruskoaho&title=Passive+Mechanical+Stretch+Releases+Atrial+Natriuretic+Peptide+from+Rat+Ventricular+Myocardium&doi=10.1161%2F01.RES.70.6.1244"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1161%2F01.RES.70.6.1244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.RES.70.6.1244%26sid%3Dliteratum%253Aachs%26aulast%3DKinnunen%26aufirst%3DP.%26aulast%3DVuolteenaho%26aufirst%3DO.%26aulast%3DUusimaa%26aufirst%3DP.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DPassive%2520Mechanical%2520Stretch%2520Releases%2520Atrial%2520Natriuretic%2520Peptide%2520from%2520Rat%2520Ventricular%2520Myocardium%26jtitle%3DCirc.%2520Res.%26date%3D1992%26volume%3D70%26spage%3D1244%26epage%3D1253%26doi%3D10.1161%2F01.RES.70.6.1244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaPointe, M. C.</span></span> <span> </span><span class="NLM_article-title">Molecular Regulation of the Brain Natriuretic Peptide Gene</span>. <i>Peptides</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">944</span>– <span class="NLM_lpage">956</span>, <span class="refDoi"> DOI: 10.1016/j.peptides.2004.08.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.peptides.2004.08.028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=944-956&author=M.+C.+LaPointe&title=Molecular+Regulation+of+the+Brain+Natriuretic+Peptide+Gene&doi=10.1016%2Fj.peptides.2004.08.028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.peptides.2004.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.peptides.2004.08.028%26sid%3Dliteratum%253Aachs%26aulast%3DLaPointe%26aufirst%3DM.%2BC.%26atitle%3DMolecular%2520Regulation%2520of%2520the%2520Brain%2520Natriuretic%2520Peptide%2520Gene%26jtitle%3DPeptides%26date%3D2005%26volume%3D26%26spage%3D944%26epage%3D956%26doi%3D10.1016%2Fj.peptides.2004.08.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimasa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, K.</span></span> <span> </span><span class="NLM_article-title">Rapid Transcriptional Activation and Early mRNA Turnover of Brain Natriuretic Peptide in Cardiocyte Hypertrophy. Evidence for Brain Natriuretic Peptide as an “Emergency” Cardiac Hormone against Ventricular Overload</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">1280</span>– <span class="NLM_lpage">1287</span>, <span class="refDoi"> DOI: 10.1172/JCI118162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1172%2FJCI118162" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1995&pages=1280-1287&author=O.+Nakagawaauthor=Y.+Ogawaauthor=H.+Itohauthor=S.+Sugaauthor=Y.+Komatsuauthor=I.+Kishimotoauthor=K.+Nishinoauthor=T.+Yoshimasaauthor=K.+Nakao&title=Rapid+Transcriptional+Activation+and+Early+mRNA+Turnover+of+Brain+Natriuretic+Peptide+in+Cardiocyte+Hypertrophy.+Evidence+for+Brain+Natriuretic+Peptide+as+an+%E2%80%9CEmergency%E2%80%9D+Cardiac+Hormone+against+Ventricular+Overload&doi=10.1172%2FJCI118162"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1172%2FJCI118162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI118162%26sid%3Dliteratum%253Aachs%26aulast%3DNakagawa%26aufirst%3DO.%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DH.%26aulast%3DSuga%26aufirst%3DS.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DKishimoto%26aufirst%3DI.%26aulast%3DNishino%26aufirst%3DK.%26aulast%3DYoshimasa%26aufirst%3DT.%26aulast%3DNakao%26aufirst%3DK.%26atitle%3DRapid%2520Transcriptional%2520Activation%2520and%2520Early%2520mRNA%2520Turnover%2520of%2520Brain%2520Natriuretic%2520Peptide%2520in%2520Cardiocyte%2520Hypertrophy.%2520Evidence%2520for%2520Brain%2520Natriuretic%2520Peptide%2520as%2520an%2520%25E2%2580%259CEmergency%25E2%2580%259D%2520Cardiac%2520Hormone%2520against%2520Ventricular%2520Overload%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1995%26volume%3D96%26spage%3D1280%26epage%3D1287%26doi%3D10.1172%2FJCI118162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serby, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falls, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaszubska, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colins, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Structure–activity Relationships of Isoxazole Carboxamides as Growth Hormone Secretagogue Receptor Antagonists</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1201</span>– <span class="NLM_lpage">1204</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2004.11.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.bmcl.2004.11.075" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1201-1204&author=Z.+Xinauthor=H.+Zhaoauthor=M.+D.+Serbyauthor=B.+Liuauthor=V.+G.+Schaeferauthor=D.+H.+Fallsauthor=W.+Kaszubskaauthor=C.+A.+Colinsauthor=H.+L.+Shamauthor=G.+Liu&title=Synthesis+and+Structure%E2%80%93activity+Relationships+of+Isoxazole+Carboxamides+as+Growth+Hormone+Secretagogue+Receptor+Antagonists&doi=10.1016%2Fj.bmcl.2004.11.075"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.11.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.11.075%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DSerby%26aufirst%3DM.%2BD.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSchaefer%26aufirst%3DV.%2BG.%26aulast%3DFalls%26aufirst%3DD.%2BH.%26aulast%3DKaszubska%26aufirst%3DW.%26aulast%3DColins%26aufirst%3DC.%2BA.%26aulast%3DSham%26aufirst%3DH.%2BL.%26aulast%3DLiu%26aufirst%3DG.%26atitle%3DSynthesis%2520and%2520Structure%25E2%2580%2593activity%2520Relationships%2520of%2520Isoxazole%2520Carboxamides%2520as%2520Growth%2520Hormone%2520Secretagogue%2520Receptor%2520Antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D1201%26epage%3D1204%26doi%3D10.1016%2Fj.bmcl.2004.11.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Omichinski, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaad, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsenfeld, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appella, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronenborn, A. M.</span></span> <span> </span><span class="NLM_article-title">NMR Structure of a Specific DNA Complex of Zn-Containing DNA Binding Domain of GATA-1</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1126/science.8332909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1126%2Fscience.8332909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8332909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK3sXltlartrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1993&pages=438-446&author=J.+G.+Omichinskiauthor=G.+M.+Cloreauthor=O.+Schaadauthor=G.+Felsenfeldauthor=C.+Trainorauthor=E.+Appellaauthor=S.+J.+Stahlauthor=A.+M.+Gronenborn&title=NMR+Structure+of+a+Specific+DNA+Complex+of+Zn-Containing+DNA+Binding+Domain+of+GATA-1&doi=10.1126%2Fscience.8332909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">NMR structure of a specific DNA complex of zinc-containing DNA binding domain of GATA-1</span></div><div class="casAuthors">Omichinski, James G.; Clore, G. Marius; Schaad, Olivier; Felsenfeld, Gary; Trainor, Cecelia; Appella, Ettore; Stahl, Stephen J.; Gronenborn, Angela M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">5120</span>),
    <span class="NLM_cas:pages">438-46</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The three-dimensional soln. structure of a complex between the DNA binding domain of the chicken erythroid transcription factor GATA-1 and its cognate DNA site has been detd. with multidimensional heteronuclear magnetic resonance spectroscopy.  The DNA binding domain consists of a core which contains a zinc coordinated by four cysteines and a carboxyl-terminal tail.  The core is composed of two irregular antiparallel β sheets and an α helix, followed by a long loop that leads into the carboxyl-terminal tail.  The amino-terminal part of the core including the helix, is similar in structure, although not in sequence, to the amino-terminal zinc module of the glucocorticoid receptor DNA binding domain.  In the other regions, the structures of these two DNA binding domains are entirely different.  The DNA target site in contact with the protein spans eight base pairs.  The helix and the loop connecting the two antiparallel β sheets interact with the major groove of the DNA.  The carboxyl-terminal tail, which is an essential determinant of specific binding, wraps around into the minor groove.  The complex resembles a hand holding a rope with the palm and fingers representing the protein core and the thumb, the carboxyl-terminal tail.  The specific interactions between GATA-1 and DNA in the major groove are mainly hydrophobic in nature, which accounts for the preponderance of thymines in the target site.  A large no. of interactions are obsd. with the phosphate backbone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol5cF60Zbls7Vg90H21EOLACvtfcHk0lgj6My7xu_YLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltlartrs%253D&md5=ac5de6179e0f29eff55075d563af58be</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscience.8332909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8332909%26sid%3Dliteratum%253Aachs%26aulast%3DOmichinski%26aufirst%3DJ.%2BG.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DSchaad%26aufirst%3DO.%26aulast%3DFelsenfeld%26aufirst%3DG.%26aulast%3DTrainor%26aufirst%3DC.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DStahl%26aufirst%3DS.%2BJ.%26aulast%3DGronenborn%26aufirst%3DA.%2BM.%26atitle%3DNMR%2520Structure%2520of%2520a%2520Specific%2520DNA%2520Complex%2520of%2520Zn-Containing%2520DNA%2520Binding%2520Domain%2520of%2520GATA-1%26jtitle%3DScience%26date%3D1993%26volume%3D261%26spage%3D438%26epage%3D446%26doi%3D10.1126%2Fscience.8332909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katanasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunagawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Cardiac Transcription Factor GATA4 by Post-Translational Modification in Cardiomyocyte Hypertrophy and Heart Failure</span>. <i>Int. Heart J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1536/ihj.16-404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1536%2Fihj.16-404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=27818483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsVyju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2016&pages=672-675&author=Y.+Katanasakaauthor=H.+Suzukiauthor=Y.+Sunagawaauthor=K.+Hasegawaauthor=T.+Morimoto&title=Regulation+of+Cardiac+Transcription+Factor+GATA4+by+Post-Translational+Modification+in+Cardiomyocyte+Hypertrophy+and+Heart+Failure&doi=10.1536%2Fihj.16-404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cardiac transcription factor GATA4 by post-translational modification in cardiomyocyte hypertrophy and heart failure</span></div><div class="casAuthors">Katanasaka, Yasufumi; Suzuki, Hidetoshi; Sunagawa, Yoichi; Hasegawa, Koji; Morimoto, Tatsuya</div><div class="citationInfo"><span class="NLM_cas:title">International Heart Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">672-675</span>CODEN:
                <span class="NLM_cas:coden">IHJNAJ</span>;
        ISSN:<span class="NLM_cas:issn">1349-2365</span>.
    
            (<span class="NLM_cas:orgname">International Heart Journal Association</span>)
        </div><div class="casAbstract">Heart failure is a leading cause of cardiovascular mortality in industrialized countries.  During development and deterioration of heart failure, cardiomyocytes undergo maladaptive hypertrophy, and changes in the cellular phenotype are accompanied by reinduction of the fetal gene program.  Gene expression in cardiomyocytes is regulated by various nuclear transcription factors, co-activators, and co-repressors.  The zinc finger protein GATA4 is one such transcription factor involved in the regulation of cardiomyocyte hypertrophy.  In response to hypertrophic stimuli such as those involving the sympathetic nervous and renin-angiotensin systems, changes in protein interaction and/or post-translational modifications of GATA4 cause hypertrophic gene transcription in cardiomyocytes.  In this article, we focus on cardiac nuclear signaling mols., esp. GATA4, that are promising as potential targets for heart failure therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp09rLSa4euy7Vg90H21EOLACvtfcHk0lg8NlHdFbjsrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsVyju7Y%253D&md5=994dbecc38c3d59cb4432f0d59b558dc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1536%2Fihj.16-404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1536%252Fihj.16-404%26sid%3Dliteratum%253Aachs%26aulast%3DKatanasaka%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DSunagawa%26aufirst%3DY.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DMorimoto%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Cardiac%2520Transcription%2520Factor%2520GATA4%2520by%2520Post-Translational%2520Modification%2520in%2520Cardiomyocyte%2520Hypertrophy%2520and%2520Heart%2520Failure%26jtitle%3DInt.%2520Heart%2520J.%26date%3D2016%26volume%3D57%26spage%3D672%26epage%3D675%26doi%3D10.1536%2Fihj.16-404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Regulation and Its Misregulation in Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1251</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cell.2013.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23498934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2013&pages=1237-1251&author=T.+I.+Leeauthor=R.+A.+Young&title=Transcriptional+Regulation+and+Its+Misregulation+in+Disease&doi=10.1016%2Fj.cell.2013.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation and its misregulation in disease</span></div><div class="casAuthors">Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1237-1251</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The gene expression programs that establish and maintain specific cell states in humans are controlled by thousands of transcription factors, cofactors, and chromatin regulators.  Misregulation of these gene expression programs can cause a broad range of diseases.  Here, we review recent advances in our understanding of transcriptional regulation and discuss how these have provided new insights into transcriptional misregulation in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRZI_3gsWJi7Vg90H21EOLACvtfcHk0lg8NlHdFbjsrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D&md5=2eabd761a3721c8c509f0fa1a00ceab6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Regulation%2520and%2520Its%2520Misregulation%2520in%2520Disease%26jtitle%3DCell%26date%3D2013%26volume%3D152%26spage%3D1237%26epage%3D1251%26doi%3D10.1016%2Fj.cell.2013.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuchner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausserlechner, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Factors by Small Compounds--Current Strategies and Future Implications</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2015.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.bcp.2015.12.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1-13&author=J.+Hagenbuchnerauthor=M.+J.+Ausserlechner&title=Targeting+Transcription+Factors+by+Small+Compounds%2D%2DCurrent+Strategies+and+Future+Implications&doi=10.1016%2Fj.bcp.2015.12.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuchner%26aufirst%3DJ.%26aulast%3DAusserlechner%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520Transcription%2520Factors%2520by%2520Small%2520Compounds--Current%2520Strategies%2520and%2520Future%2520Implications%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2016%26volume%3D107%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.bcp.2015.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Addis, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ifkovits, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellam, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rentschler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christoforou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epstein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gearhart, J. D.</span></span> <span> </span><span class="NLM_article-title">Optimization of Direct Fibroblast Reprogramming to Cardiomyocytes Using Calcium Activity as a Functional Measure of Success</span>. <i>J. Mol. Cell. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.yjmcc.2013.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23591016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptVyqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2013&pages=97-106&author=R.+C.+Addisauthor=J.+L.+Ifkovitsauthor=F.+Pintoauthor=L.+D.+Kellamauthor=P.+Estesoauthor=S.+Rentschlerauthor=N.+Christoforouauthor=J.+A.+Epsteinauthor=J.+D.+Gearhart&title=Optimization+of+Direct+Fibroblast+Reprogramming+to+Cardiomyocytes+Using+Calcium+Activity+as+a+Functional+Measure+of+Success&doi=10.1016%2Fj.yjmcc.2013.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of direct fibroblast reprogramming to cardiomyocytes using calcium activity as a functional measure of success</span></div><div class="casAuthors">Addis, Russell C.; Ifkovits, Jamie L.; Pinto, Filipa; Kellam, Lori D.; Esteso, Paul; Rentschler, Stacey; Christoforou, Nicolas; Epstein, Jonathan A.; Gearhart, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97-106</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Direct conversion of fibroblasts to induced cardiomyocytes (iCMs) has great potential for regenerative medicine.  Recent publications have reported significant progress, but the evaluation of reprogramming has relied upon non-functional measures such as flow cytometry for cardiomyocyte markers or GFP expression driven by a cardiomyocyte-specific promoter.  The issue is one of practicality: the most stringent measures - electrophysiol. to detect cell excitation and the presence of spontaneously contracting myocytes - are not readily quantifiable in the large nos. of cells screened in reprogramming expts.  However, excitation and contraction are linked by a third functional characteristic of cardiomyocytes: the rhythmic oscillation of intracellular calcium levels.  We set out to optimize direct conversion of fibroblasts to iCMs with a quantifiable calcium reporter to rapidly assess functional transdifferentiation.  We constructed a reporter system in which the calcium indicator GCaMP is driven by the cardiomyocyte-specific Troponin T promoter.  Using calcium activity as our primary outcome measure, we compared several published combinations of transcription factors along with novel combinations in mouse embryonic fibroblasts.  The most effective combination consisted of Hand2, Nkx2.5, Gata4, Mef2c, and Tbx5 (HNGMT).  This combination is > 50-fold more efficient than GMT alone and produces iCMs with cardiomyocyte marker expression, robust calcium oscillation, and spontaneous beating that persist for weeks following inactivation of reprogramming factors.  HNGMT is also significantly more effective than previously published factor combinations for the transdifferentiation of adult mouse cardiac fibroblasts to iCMs.  Quantification of calcium function is a convenient and effective means for the identification and evaluation of cardiomyocytes generated by direct reprogramming.  Using this stringent outcome measure, we conclude that HNGMT produces iCMs more efficiently than previously published methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou3fsRrjAOgrVg90H21EOLACvtfcHk0lg8NlHdFbjsrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptVyqtLg%253D&md5=5b99b6dd16a2f234d38dc3c5131e1d9b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DAddis%26aufirst%3DR.%2BC.%26aulast%3DIfkovits%26aufirst%3DJ.%2BL.%26aulast%3DPinto%26aufirst%3DF.%26aulast%3DKellam%26aufirst%3DL.%2BD.%26aulast%3DEsteso%26aufirst%3DP.%26aulast%3DRentschler%26aufirst%3DS.%26aulast%3DChristoforou%26aufirst%3DN.%26aulast%3DEpstein%26aufirst%3DJ.%2BA.%26aulast%3DGearhart%26aufirst%3DJ.%2BD.%26atitle%3DOptimization%2520of%2520Direct%2520Fibroblast%2520Reprogramming%2520to%2520Cardiomyocytes%2520Using%2520Calcium%2520Activity%2520as%2520a%2520Functional%2520Measure%2520of%2520Success%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D60%26spage%3D97%26epage%3D106%26doi%3D10.1016%2Fj.yjmcc.2013.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, M.</span></span> <span> </span><span class="NLM_article-title">Critical Factors for Cardiac Reprogramming</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.112.271452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2FCIRCRESAHA.112.271452" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2012&pages=5-8&author=D.+Srivastavaauthor=M.+Ieda&title=Critical+Factors+for+Cardiac+Reprogramming&doi=10.1161%2FCIRCRESAHA.112.271452"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.112.271452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.112.271452%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DD.%26aulast%3DIeda%26aufirst%3DM.%26atitle%3DCritical%2520Factors%2520for%2520Cardiac%2520Reprogramming%26jtitle%3DCirc.%2520Res.%26date%3D2012%26volume%3D111%26spage%3D5%26epage%3D8%26doi%3D10.1161%2FCIRCRESAHA.112.271452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassel-Duby, R.</span></span> <span> </span><span class="NLM_article-title">Mending Broken Hearts: Cardiac Development as a Basis for Adult Heart Regeneration and Repair</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nrm3619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnrm3619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23839576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOrtL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=529-541&author=M.+Xinauthor=E.+N.+Olsonauthor=R.+Bassel-Duby&title=Mending+Broken+Hearts%3A+Cardiac+Development+as+a+Basis+for+Adult+Heart+Regeneration+and+Repair&doi=10.1038%2Fnrm3619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Mending broken hearts: cardiac development as a basis for adult heart regeneration and repair</span></div><div class="casAuthors">Xin, Mei; Olson, Eric N.; Bassel-Duby, Rhonda</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">529-541</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  As the adult mammalian heart has limited potential for regeneration and repair, the loss of cardiomyocytes during injury and disease can result in heart failure and death.  The cellular processes and regulatory mechanisms involved in heart growth and development can be exploited to repair the injured adult heart through 'reawakening' pathways that are active during embryogenesis.  Heart function has been restored in rodents by reprogramming non-myocytes into cardiomyocytes, by expressing transcription factors (GATA4, HAND2, myocyte-specific enhancer factor 2C (MEF2C) and T-box 5 (TBX5)) and microRNAs (miR-1, miR-133, miR-208 and miR-499) that control cardiomyocyte identity.  Stimulating cardiomyocyte dedifferentiation and proliferation by activating mitotic signalling pathways involved in embryonic heart growth represents a complementary approach for heart regeneration and repair.  Recent advances in understanding the mechanistic basis of heart development offer exciting opportunities for effective therapies for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQi6P5whO9QLVg90H21EOLACvtfcHk0li_reFsToHJdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOrtL3M&md5=124d33b33df1d632864d065890f8f142</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrm3619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3619%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DM.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26aulast%3DBassel-Duby%26aufirst%3DR.%26atitle%3DMending%2520Broken%2520Hearts%253A%2520Cardiac%2520Development%2520as%2520a%2520Basis%2520for%2520Adult%2520Heart%2520Regeneration%2520and%2520Repair%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26spage%3D529%26epage%3D541%26doi%3D10.1038%2Fnrm3619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">GATA-Dependent Recruitment of MEF2 Proteins to Target Promoters</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2046</span>– <span class="NLM_lpage">2055</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.9.2046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1093%2Femboj%2F19.9.2046" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=2046-2055&author=S.+Morinauthor=F.+Charronauthor=L.+Robitailleauthor=M.+Nemer&title=GATA-Dependent+Recruitment+of+MEF2+Proteins+to+Target+Promoters&doi=10.1093%2Femboj%2F19.9.2046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.9.2046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.9.2046%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DS.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DRobitaille%26aufirst%3DL.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DGATA-Dependent%2520Recruitment%2520of%2520MEF2%2520Proteins%2520to%2520Target%2520Promoters%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D2046%26epage%3D2055%26doi%3D10.1093%2Femboj%2F19.9.2046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cserjesi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span> <span> </span><span class="NLM_article-title">The Transcription Factors GATA4 and dHAND Physically Interact to Synergistically Activate Cardiac Gene Expression through a p300-Dependent Mechanism</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">24390</span>– <span class="NLM_lpage">24398</span>, <span class="refDoi"> DOI: 10.1074/jbc.M202490200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M202490200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=24390-24398&author=Y.+S.+Daiauthor=P.+Cserjesiauthor=B.+E.+Markhamauthor=J.+D.+Molkentin&title=The+Transcription+Factors+GATA4+and+dHAND+Physically+Interact+to+Synergistically+Activate+Cardiac+Gene+Expression+through+a+p300-Dependent+Mechanism&doi=10.1074%2Fjbc.M202490200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M202490200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M202490200%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%2BS.%26aulast%3DCserjesi%26aufirst%3DP.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520Transcription%2520Factors%2520GATA4%2520and%2520dHAND%2520Physically%2520Interact%2520to%2520Synergistically%2520Activate%2520Cardiac%2520Gene%2520Expression%2520through%2520a%2520p300-Dependent%2520Mechanism%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D24390%26epage%3D24398%26doi%3D10.1074%2Fjbc.M202490200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kathiriya, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schluterman, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothrock, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eapen, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama-Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">GATA4 Mutations Cause Human Congenital Heart Defects and Reveal an Interaction with TBX5</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">443</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/nature01827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnature01827" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2003&pages=443-447&author=V.+Gargauthor=I.+S.+Kathiriyaauthor=R.+Barnesauthor=M.+K.+Schlutermanauthor=I.+N.+Kingauthor=C.+A.+Butlerauthor=C.+R.+Rothrockauthor=R.+S.+Eapenauthor=K.+Hirayama-Yamadaauthor=K.+Jooauthor=R.+Matsuokaauthor=J.+C.+Cohenauthor=D.+Srivastava&title=GATA4+Mutations+Cause+Human+Congenital+Heart+Defects+and+Reveal+an+Interaction+with+TBX5&doi=10.1038%2Fnature01827"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature01827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01827%26sid%3Dliteratum%253Aachs%26aulast%3DGarg%26aufirst%3DV.%26aulast%3DKathiriya%26aufirst%3DI.%2BS.%26aulast%3DBarnes%26aufirst%3DR.%26aulast%3DSchluterman%26aufirst%3DM.%2BK.%26aulast%3DKing%26aufirst%3DI.%2BN.%26aulast%3DButler%26aufirst%3DC.%2BA.%26aulast%3DRothrock%26aufirst%3DC.%2BR.%26aulast%3DEapen%26aufirst%3DR.%2BS.%26aulast%3DHirayama-Yamada%26aufirst%3DK.%26aulast%3DJoo%26aufirst%3DK.%26aulast%3DMatsuoka%26aufirst%3DR.%26aulast%3DCohen%26aufirst%3DJ.%2BC.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DGATA4%2520Mutations%2520Cause%2520Human%2520Congenital%2520Heart%2520Defects%2520and%2520Reveal%2520an%2520Interaction%2520with%2520TBX5%26jtitle%3DNature%26date%3D2003%26volume%3D424%26spage%3D443%26epage%3D447%26doi%3D10.1038%2Fnature01827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ang, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippens, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radzinsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon-Grady, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pruitt, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Disease Model of GATA4 Mutation Reveals Transcription Factor Cooperativity in Human Cardiogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1734</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.11.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cell.2016.11.033" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=1734-1749&author=Y.-S.+Angauthor=R.+N.+Rivasauthor=A.+J.+S.+Ribeiroauthor=R.+Srivasauthor=J.+Riveraauthor=N.+R.+Stoneauthor=K.+Prattauthor=T.+M.+A.+Mohamedauthor=J.-D.+Fuauthor=C.+I.+Spencerauthor=N.+D.+Tippensauthor=M.+Liauthor=A.+Narasimhaauthor=E.+Radzinskyauthor=A.+J.+Moon-Gradyauthor=H.+Yuauthor=B.+L.+Pruittauthor=M.+P.+Snyderauthor=D.+Srivastava&title=Disease+Model+of+GATA4+Mutation+Reveals+Transcription+Factor+Cooperativity+in+Human+Cardiogenesis&doi=10.1016%2Fj.cell.2016.11.033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.11.033%26sid%3Dliteratum%253Aachs%26aulast%3DAng%26aufirst%3DY.-S.%26aulast%3DRivas%26aufirst%3DR.%2BN.%26aulast%3DRibeiro%26aufirst%3DA.%2BJ.%2BS.%26aulast%3DSrivas%26aufirst%3DR.%26aulast%3DRivera%26aufirst%3DJ.%26aulast%3DStone%26aufirst%3DN.%2BR.%26aulast%3DPratt%26aufirst%3DK.%26aulast%3DMohamed%26aufirst%3DT.%2BM.%2BA.%26aulast%3DFu%26aufirst%3DJ.-D.%26aulast%3DSpencer%26aufirst%3DC.%2BI.%26aulast%3DTippens%26aufirst%3DN.%2BD.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DNarasimha%26aufirst%3DA.%26aulast%3DRadzinsky%26aufirst%3DE.%26aulast%3DMoon-Grady%26aufirst%3DA.%2BJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DPruitt%26aufirst%3DB.%2BL.%26aulast%3DSnyder%26aufirst%3DM.%2BP.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DDisease%2520Model%2520of%2520GATA4%2520Mutation%2520Reveals%2520Transcription%2520Factor%2520Cooperativity%2520in%2520Human%2520Cardiogenesis%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D1734%26epage%3D1749%26doi%3D10.1016%2Fj.cell.2016.11.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belaguli, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepulveda, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span> <span> </span><span class="NLM_article-title">Cardiac Tissue Enriched Factors Serum Response Factor and GATA-4 Are Mutual Coregulators</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">7550</span>– <span class="NLM_lpage">7558</span>, <span class="refDoi"> DOI: 10.1128/MCB.20.20.7550-7558.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.20.20.7550-7558.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=11003651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnt1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=7550-7558&author=N.+S.+Belaguliauthor=J.+L.+Sepulvedaauthor=V.+Nigamauthor=F.+Charronauthor=M.+Nemerauthor=R.+J.+Schwartz&title=Cardiac+Tissue+Enriched+Factors+Serum+Response+Factor+and+GATA-4+Are+Mutual+Coregulators&doi=10.1128%2FMCB.20.20.7550-7558.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Cardiac tissue enriched factors serum response factor and GATA-4 are mutual coregulators</span></div><div class="casAuthors">Belaguli, Narasimhaswamy S.; Sepulveda, Jorge L.; Nigam, Vishal; Charron, Frederic; Nemer, Mona; Schwartz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7550-7558</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Combinatorial interaction among cardiac tissue-restricted enriched transcription factors may facilitate the expression of cardiac tissue-restricted genes.  Here we show that the MADS box factor serum response factor (SRF) cooperates with the zinc finger protein GATA-4 to synergistically activate numerous myogenic and nonmyogenic serum response element (SRE)-dependent promoters in CV1 fibroblasts.  In the absence of GATA binding sites, synergistic activation depends on binding of SRF to the proximal CArG box sequence in the cardiac and skeletal α-actin promoter.  GATA-4's C-terminal activation domain is obligatory for synergistic coactivation with SRF, and its N-terminal domain and first zinc finger are inhibitory.  SRF and GATA-4 phys. assoc. both in vivo and in vitro through their MADS box and the second zinc finger domains as detd. by protein A pullout assays and by in vivo one-hybrid transfection assays using Gal4 fusion proteins.  Other cardiovascular tissue-restricted GATA factors, such as GATA-5 and GATA-6, were equiv. to GATA-4 in coactivating SRE-dependent targets.  Thus, interaction between the MADS box and C4 zinc finger proteins, a novel regulatory paradigm, mediates activation of SRF-dependent gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKAZGYZLYBWbVg90H21EOLACvtfcHk0lhe3L3P0gXZLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnt1aitrc%253D&md5=ebc350bbcda08c4671e25ca441650de2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1128%2FMCB.20.20.7550-7558.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.20.20.7550-7558.2000%26sid%3Dliteratum%253Aachs%26aulast%3DBelaguli%26aufirst%3DN.%2BS.%26aulast%3DSepulveda%26aufirst%3DJ.%2BL.%26aulast%3DNigam%26aufirst%3DV.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26atitle%3DCardiac%2520Tissue%2520Enriched%2520Factors%2520Serum%2520Response%2520Factor%2520and%2520GATA-4%2520Are%2520Mutual%2520Coregulators%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D7550%26epage%3D7558%26doi%3D10.1128%2FMCB.20.20.7550-7558.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nam, Y. J.</span>; <span class="NLM_string-name">Song, K.</span>; <span class="NLM_string-name">Olson, E.
N.</span></span> <span> </span><span class="NLM_article-title">Cardiac Repair by Reprogramming of Cardiac Fibroblasts into Cardiomyocytes</span>. Patent <span class="NLM_patent">WO2012/116064 A1</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=Y.+J.+Nam&author=K.+Song&author=E.%0AN.+Olson&title=Cardiac+Repair+by+Reprogramming+of+Cardiac+Fibroblasts+into+Cardiomyocytes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DNam%26aufirst%3DY.%2BJ.%26atitle%3DCardiac%2520Repair%2520by%2520Reprogramming%2520of%2520Cardiac%2520Fibroblasts%2520into%2520Cardiomyocytes%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schorpp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothenaigner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinshagen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gul, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadian, K.</span></span> <span> </span><span class="NLM_article-title">Identification of Small-Molecule Frequent Hitters from AlphaScreen High-Throughput Screens</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1177/1087057113516861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1177%2F1087057113516861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=24371213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=715-726&author=K.+Schorppauthor=I.+Rothenaignerauthor=E.+Salminaauthor=J.+Reinshagenauthor=T.+Lowauthor=J.+K.+Brenkeauthor=J.+Gopalakrishnanauthor=I.+V.+Tetkoauthor=S.+Gulauthor=K.+Hadian&title=Identification+of+Small-Molecule+Frequent+Hitters+from+AlphaScreen+High-Throughput+Screens&doi=10.1177%2F1087057113516861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small-molecule frequent hitters from Alphascreen high-throughput screens</span></div><div class="casAuthors">Schorpp, Kenji; Rothenaigner, Ina; Salmina, Elena; Reinshagen, Jeanette; Low, Terence; Brenke, Jara K.; Gopalakrishnan, Jay; Tetko, Igor V.; Gul, Sheraz; Hadian, Kamyar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">715-726</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Although small-mol. drug discovery efforts have focused largely on enzyme, receptor, and ion-channel targets, there has been an increase in such activities to search for protein-protein interaction (PPI) disruptors by applying high-throughout screening (HTS)-compatible protein-binding assays.  However, a disadvantage of these assays is that many primary hits are frequent hitters regardless of the PPI being investigated.  We have used the AlphaScreen technol. to screen four different robust PPI assays each against 25,000 compds.  These activities led to the identification of 137 compds. that demonstrated repeated activity in all PPI assays.  These compds. were subsequently evaluated in two AlphaScreen counter assays, leading to classification of compds. that either interfered with the AlphaScreen chem. (60 compds.) or prevented the binding of the protein His-tag moiety to nickel chelate (Ni2+-NTA) beads of the AlphaScreen detection system (77 compds.).  To further triage the 137 frequent hitters, we subsequently confirmed by a time-resolved fluorescence resonance energy transfer assay that most of these compds. were only frequent hitters in AlphaScreen assays.  A chemoinformatics anal. of the apparent hits provided details of the compds. that can be flagged as frequent hitters of the AlphaScreen technol., and these data have broad applicability for users of these detection technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQj9hCbRqv7LVg90H21EOLACvtfcHk0lhe3L3P0gXZLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM&md5=ea65e879f093202deb2266bad94f0208</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1177%2F1087057113516861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113516861%26sid%3Dliteratum%253Aachs%26aulast%3DSchorpp%26aufirst%3DK.%26aulast%3DRothenaigner%26aufirst%3DI.%26aulast%3DSalmina%26aufirst%3DE.%26aulast%3DReinshagen%26aufirst%3DJ.%26aulast%3DLow%26aufirst%3DT.%26aulast%3DBrenke%26aufirst%3DJ.%2BK.%26aulast%3DGopalakrishnan%26aufirst%3DJ.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DGul%26aufirst%3DS.%26aulast%3DHadian%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520Small-Molecule%2520Frequent%2520Hitters%2520from%2520AlphaScreen%2520High-Throughput%2520Screens%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D715%26epage%3D726%26doi%3D10.1177%2F1087057113516861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomašić, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin Mašič, L.</span></span> <span> </span><span class="NLM_article-title">Rhodanine as a Scaffold in Drug Discovery: A Critical Review of Its Biological Activities and Mechanisms of Target Modulation</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1517/17460441.2012.688743</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1517%2F17460441.2012.688743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=22607309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWmtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=549-560&author=T.+Toma%C5%A1i%C4%87author=L.+Peterlin+Ma%C5%A1i%C4%8D&title=Rhodanine+as+a+Scaffold+in+Drug+Discovery%3A+A+Critical+Review+of+Its+Biological+Activities+and+Mechanisms+of+Target+Modulation&doi=10.1517%2F17460441.2012.688743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Rhodanine as a scaffold in drug discovery: a critical review of its biological activities and mechanisms of target modulation</span></div><div class="casAuthors">Tomasic, Tihomir; Peterlin Masic, Lucija</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Rhodanine-based compds. have been assocd. with numerous biol. activities.  After many years of research in drug discovery, they have gained a reputation as being pan assay interference compds. (PAINS) and frequent hitters in screening campaigns.  Rhodanine-based compds. are also aggregators that can non-specifically interact with target proteins as well as Michael acceptors and interfere photometrically in biol. assays due to their color.Areas covered: The authors review the recently reported biol. activities of rhodanine-based compds.  Furthermore, the article provides details of their synthesis and occurrence in compd. libraries through high-throughput screening (HTS) and virtual high-throughput screening (VHTS).  Addnl., the authors provide the reader with possible mechanisms of non-specific target modulation, anal. of the crystal structures of enzyme-rhodanine complexes and a comparison of rhodanine and thiazolidine-2,4-dione moieties.Expert opinion: The biol. activity of compds. possessing a rhodanine moiety should be considered very critically despite the convincing data obtained in biol. assays.  In addn. to the lack of selectivity, unusual structure-activity relationship profiles and safety and specificity problems mean that rhodanines are generally not optimizable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorolVH32PdC7Vg90H21EOLACvtfcHk0lg66k4NlNysfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWmtLw%253D&md5=a02e78c0c7788aaeb044f3753c901117</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.688743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.688743%26sid%3Dliteratum%253Aachs%26aulast%3DToma%25C5%25A1i%25C4%2587%26aufirst%3DT.%26aulast%3DPeterlin%2BMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%26atitle%3DRhodanine%2520as%2520a%2520Scaffold%2520in%2520Drug%2520Discovery%253A%2520A%2520Critical%2520Review%2520of%2520Its%2520Biological%2520Activities%2520and%2520Mechanisms%2520of%2520Target%2520Modulation%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D549%26epage%3D560%26doi%3D10.1517%2F17460441.2012.688743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schade, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">Medicinal Chemistry Approaches to Heart Regeneration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9451</span>– <span class="NLM_lpage">9479</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00446</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00446" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9451-9479&author=D.+Schadeauthor=A.+T.+Plowright&title=Medicinal+Chemistry+Approaches+to+Heart+Regeneration&doi=10.1021%2Facs.jmedchem.5b00446"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00446%26sid%3Dliteratum%253Aachs%26aulast%3DSchade%26aufirst%3DD.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DMedicinal%2520Chemistry%2520Approaches%2520to%2520Heart%2520Regeneration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9451%26epage%3D9479%26doi%3D10.1021%2Facs.jmedchem.5b00446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennewell, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seden, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, G. M.</span></span> <span> </span><span class="NLM_article-title">Stemistry: The Control of Stem Cells in Situ Using Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2863</span>– <span class="NLM_lpage">2894</span>, <span class="refDoi"> DOI: 10.1021/jm500838d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500838d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVyjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2863-2894&author=S.+G.+Daviesauthor=P.+D.+Kennewellauthor=A.+J.+Russellauthor=P.+T.+Sedenauthor=R.+Westwoodauthor=G.+M.+Wynne&title=Stemistry%3A+The+Control+of+Stem+Cells+in+Situ+Using+Chemistry&doi=10.1021%2Fjm500838d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Stemistry: The Control of Stem Cells in Situ Using Chemistry</span></div><div class="casAuthors">Davies, Stephen G.; Kennewell, Peter D.; Russell, Angela J.; Seden, Peter T.; Westwood, Robert; Wynne, Graham M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2863-2894</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A new paradigm for drug research has emerged, namely the deliberate search for mols. able to selectively affect the proliferation, differentiation, and migration of adult stem cells within the tissues in which they exist.  Recently, there has been significant interest in medicinal chem. toward the discovery and design of low mol. wt. mols. that affect stem cells and thus have novel therapeutic activity.  We believe that a successful agent from such a discover program would have profound effects on the treatment of many long-term degenerative disorders.  Among these conditions are examples such as cardiovascular decay, neurol. disorders including Alzheimer's disease, and macular degeneration, all of which have significant unmet medical needs.  This perspective will review evidence from the literature that indicates that discovery of such agents is achievable and represents a worthwhile pursuit for the skills of the medicinal chemist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZPeNZi3O2o7Vg90H21EOLACvtfcHk0lg66k4NlNysfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVyjuw%253D%253D&md5=f270b737affe419ca1405ba79b2a38e3</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm500838d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500838d%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DS.%2BG.%26aulast%3DKennewell%26aufirst%3DP.%2BD.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DSeden%26aufirst%3DP.%2BT.%26aulast%3DWestwood%26aufirst%3DR.%26aulast%3DWynne%26aufirst%3DG.%2BM.%26atitle%3DStemistry%253A%2520The%2520Control%2520of%2520Stem%2520Cells%2520in%2520Situ%2520Using%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2863%26epage%3D2894%26doi%3D10.1021%2Fjm500838d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimelman, D.</span></span> <span> </span><span class="NLM_article-title">Mesoderm Induction: From Caps to Chips</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nrg1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnrg1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=16619051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFSqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=360-372&author=D.+Kimelman&title=Mesoderm+Induction%3A+From+Caps+to+Chips&doi=10.1038%2Fnrg1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mesoderm induction: from caps to chips</span></div><div class="casAuthors">Kimelman, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">360-372</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Vertebrate mesoderm induction is one of the classical problems in developmental biol.  Various developmental biol. approaches, particularly in Xenopus and zebrafish, have identified many of the key factors that are involved in this process and have provided major insights into how these factors interact as part of a signaling and transcription-factor network.  These data are beginning to be refined by high-throughput approaches such as microarray assays.  Future challenges include understanding how the prospective mesodermal cells integrate the various signals they receive and how they resolve this information to regulate their morphogenetic behaviors and cell-fate decisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkEur7SDJVLVg90H21EOLACvtfcHk0ljNLNRq7KIvOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFSqt7o%253D&md5=3d7499f0d4dbac429911ac1ed67e6d4d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnrg1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1837%26sid%3Dliteratum%253Aachs%26aulast%3DKimelman%26aufirst%3DD.%26atitle%3DMesoderm%2520Induction%253A%2520From%2520Caps%2520to%2520Chips%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2006%26volume%3D7%26spage%3D360%26epage%3D372%26doi%3D10.1038%2Fnrg1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witty, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikryukov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, G.</span></span> <span> </span><span class="NLM_article-title">Generation of the Epicardial Lineage from Human Pluripotent Stem Cells</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1026</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1038/nbt.3002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnbt.3002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1026-1035&author=A.+D.+Wittyauthor=A.+Mihicauthor=R.+Y.+Tamauthor=S.+A.+Fisherauthor=A.+Mikryukovauthor=M.+S.+Shoichetauthor=R.+K.+Liauthor=S.+J.+Kattmanauthor=G.+Keller&title=Generation+of+the+Epicardial+Lineage+from+Human+Pluripotent+Stem+Cells&doi=10.1038%2Fnbt.3002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3002%26sid%3Dliteratum%253Aachs%26aulast%3DWitty%26aufirst%3DA.%2BD.%26aulast%3DMihic%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DR.%2BY.%26aulast%3DFisher%26aufirst%3DS.%2BA.%26aulast%3DMikryukov%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DR.%2BK.%26aulast%3DKattman%26aufirst%3DS.%2BJ.%26aulast%3DKeller%26aufirst%3DG.%26atitle%3DGeneration%2520of%2520the%2520Epicardial%2520Lineage%2520from%2520Human%2520Pluripotent%2520Stem%2520Cells%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D1026%26epage%3D1035%26doi%3D10.1038%2Fnbt.3002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span> <span> </span><span class="NLM_article-title">Conversion of Human Fibroblasts into Functional Cardiomyocytes by Small Molecules</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1220</span>, <span class="refDoi"> DOI: 10.1126/science.aaf1502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1126%2Fscience.aaf1502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=27127239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslOqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=1216-1220&author=N.+Caoauthor=Y.+Huangauthor=J.+Zhengauthor=C.+I.+Spencerauthor=Y.+Zhangauthor=J.-D.+Fuauthor=B.+Nieauthor=M.+Xieauthor=M.+Zhangauthor=H.+Wangauthor=T.+Maauthor=T.+Xuauthor=G.+Shiauthor=D.+Srivastavaauthor=S.+Ding&title=Conversion+of+Human+Fibroblasts+into+Functional+Cardiomyocytes+by+Small+Molecules&doi=10.1126%2Fscience.aaf1502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Conversion of human fibroblasts into functional cardiomyocytes by small molecules</span></div><div class="casAuthors">Cao, Nan; Huang, Yu; Zheng, Jiashun; Spencer, C. Ian; Zhang, Yu; Fu, Ji-Dong; Nie, Baoming; Xie, Min; Zhang, Mingliang; Wang, Haixia; Ma, Tianhua; Xu, Tao; Shi, Guilai; Srivastava, Deepak; Ding, Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">6290</span>),
    <span class="NLM_cas:pages">1216-1220</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Reprogramming somatic fibroblasts into alternative lineages would provide a promising source of cells for regenerative therapy.  However, transdifferentiating human cells into specific homogeneous, functional cell types is challenging.  Here we show that cardiomyocyte-like cells can be generated by treating human fibroblasts with a combination of nine compds. that we term 9C.  The chem. induced cardiomyocyte-like cells uniformly contracted and resembled human cardiomyocytes in their transcriptome, epigenetic, and electrophysiol. properties.  9C treatment of human fibroblasts resulted in a more open-chromatin conformation at key heart developmental genes, enabling their promoters and enhancers to bind effectors of major cardiogenic signals.  When transplanted into infarcted mouse hearts, 9C-treated fibroblasts were efficiently converted to chem. induced cardiomyocyte-like cells.  This pharmacol. approach to lineage-specific reprogramming may have many important therapeutic implications after further optimization to generate mature cardiac cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshOttrRImeLVg90H21EOLACvtfcHk0ljNLNRq7KIvOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslOqs7Y%253D&md5=e42d122563059ffe9f59fc4e889c35e5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf1502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf1502%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DSpencer%26aufirst%3DC.%2BI.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DJ.-D.%26aulast%3DNie%26aufirst%3DB.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DG.%26aulast%3DSrivastava%26aufirst%3DD.%26aulast%3DDing%26aufirst%3DS.%26atitle%3DConversion%2520of%2520Human%2520Fibroblasts%2520into%2520Functional%2520Cardiomyocytes%2520by%2520Small%2520Molecules%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D1216%26epage%3D1220%26doi%3D10.1126%2Fscience.aaf1502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radzinsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnitsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivey, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Chemical Enhancement of In Vitro and In Vivo Direct Cardiac Reprogramming</span>. <i>Circulation</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">995</span>, <span class="refDoi"> DOI: 10.1161/CIRCULATIONAHA.116.024692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2FCIRCULATIONAHA.116.024692" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2017&pages=978-995&author=T.+M.+A.+Mohamedauthor=N.+R.+Stoneauthor=E.+C.+Berryauthor=E.+Radzinskyauthor=Y.+Huangauthor=K.+Prattauthor=Y.+S.+Angauthor=P.+Yuauthor=H.+Wangauthor=S.+Tangauthor=S.+Magnitskyauthor=S.+Dingauthor=K.+N.+Iveyauthor=D.+Srivastava&title=Chemical+Enhancement+of+In+Vitro+and+In+Vivo+Direct+Cardiac+Reprogramming&doi=10.1161%2FCIRCULATIONAHA.116.024692"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1161%2FCIRCULATIONAHA.116.024692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCULATIONAHA.116.024692%26sid%3Dliteratum%253Aachs%26aulast%3DMohamed%26aufirst%3DT.%2BM.%2BA.%26aulast%3DStone%26aufirst%3DN.%2BR.%26aulast%3DBerry%26aufirst%3DE.%2BC.%26aulast%3DRadzinsky%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DPratt%26aufirst%3DK.%26aulast%3DAng%26aufirst%3DY.%2BS.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DMagnitsky%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DIvey%26aufirst%3DK.%2BN.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DChemical%2520Enhancement%2520of%2520In%2520Vitro%2520and%2520In%2520Vivo%2520Direct%2520Cardiac%2520Reprogramming%26jtitle%3DCirculation%26date%3D2017%26volume%3D135%26spage%3D978%26epage%3D995%26doi%3D10.1161%2FCIRCULATIONAHA.116.024692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadek, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannack, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garry, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longgood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. W.</span></span> <span> </span><span class="NLM_article-title">Cardiogenic Small Molecules That Enhance Myocardial Repair by Stem Cells</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">6063</span>– <span class="NLM_lpage">6068</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711507105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1073%2Fpnas.0711507105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6063-6068&author=H.+Sadekauthor=B.+Hannackauthor=E.+Choeauthor=J.+Wangauthor=S.+Latifauthor=M.+G.+Garryauthor=D.+J.+Garryauthor=J.+Longgoodauthor=D.+E.+Frantzauthor=E.+N.+Olsonauthor=J.+Hsiehauthor=J.+W.+Schneider&title=Cardiogenic+Small+Molecules+That+Enhance+Myocardial+Repair+by+Stem+Cells&doi=10.1073%2Fpnas.0711507105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711507105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711507105%26sid%3Dliteratum%253Aachs%26aulast%3DSadek%26aufirst%3DH.%26aulast%3DHannack%26aufirst%3DB.%26aulast%3DChoe%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLatif%26aufirst%3DS.%26aulast%3DGarry%26aufirst%3DM.%2BG.%26aulast%3DGarry%26aufirst%3DD.%2BJ.%26aulast%3DLonggood%26aufirst%3DJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26aulast%3DHsieh%26aufirst%3DJ.%26aulast%3DSchneider%26aufirst%3DJ.%2BW.%26atitle%3DCardiogenic%2520Small%2520Molecules%2520That%2520Enhance%2520Myocardial%2520Repair%2520by%2520Stem%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D6063%26epage%3D6068%26doi%3D10.1073%2Fpnas.0711507105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rooij, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. W.</span></span> <span> </span><span class="NLM_article-title">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1083</span>, <span class="refDoi"> DOI: 10.1021/cb300001m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300001m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1077-1083&author=J.+L.+Russellauthor=S.+C.+Goetschauthor=H.+R.+Aguilarauthor=H.+Coeauthor=X.+Luoauthor=N.+Liuauthor=E.+van+Rooijauthor=D.+E.+Frantzauthor=J.+W.+Schneider&title=Regulated+Expression+of+pH+Sensing+G+Protein-Coupled+Receptor-68+Identified+through+Chemical+Biology+Defines+a+New+Drug+Target+for+Ischemic+Heart+Disease&doi=10.1021%2Fcb300001m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span></div><div class="casAuthors">Russell, Jamie L.; Goetsch, Sean C.; Aguilar, Hector R.; Coe, Helen; Luo, Xiang; Liu, Ning; van Rooij, Eva; Frantz, Doug E.; Schneider, Jay W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1077-1083</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. biol. promises discovery of new and unexpected mechanistic pathways, protein functions and disease targets.  Here, we probed the mechanism-of-action and protein targets of 3,5-disubstituted isoxazoles (Isx), cardiomyogenic small mols. that target Notch-activated epicardium-derived cells (NECs) in vivo and promote functional recovery after myocardial infarction (MI).  Mechanistic studies in NECs led to an Isx-activated Gq protein-coupled receptor (GqPCR) hypothesis tested in a cell-based functional target screen for GPCRs regulated by Isx.  This screen identified one agonist hit, the extracellular proton/pH-sensing GPCR GPR68, confirmed through genetic gain- and loss-of-function.  Overlooked until now, GPR68 expression and localization were highly regulated in early post-natal and adult post-infarct mouse heart, where GPR68-expressing cells accumulated subepicardially.  Remarkably, GPR68-expressing cardiomyocytes established a proton-sensing cellular buffer zone surrounding the MI.  Isx pharmacol. regulated gene expression (mRNAs and miRs) in this GPR68-enriched border zone, driving cardiomyogenic and pro-survival transcriptional programs in vivo.  In conclusion, we tracked a (micromolar) bioactive small mol. mechanism-of-action to a candidate target protein, GPR68, and validated this target as a previously unrecognized regulator of myocardial cellular responses to tissue acidosis, setting the stage for future (nanomolar) target-based drug lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQLtpNAH6aiLVg90H21EOLACvtfcHk0lhUYnB4dCT1kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D&md5=751e2e1833518df9da9253f0516ba4e5</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fcb300001m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300001m%26sid%3Dliteratum%253Aachs%26aulast%3DRussell%26aufirst%3DJ.%2BL.%26aulast%3DGoetsch%26aufirst%3DS.%2BC.%26aulast%3DAguilar%26aufirst%3DH.%2BR.%26aulast%3DCoe%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3Dvan%2BRooij%26aufirst%3DE.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DSchneider%26aufirst%3DJ.%2BW.%26atitle%3DRegulated%2520Expression%2520of%2520pH%2520Sensing%2520G%2520Protein-Coupled%2520Receptor-68%2520Identified%2520through%2520Chemical%2520Biology%2520Defines%2520a%2520New%2520Drug%2520Target%2520for%2520Ischemic%2520Heart%2520Disease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1077%26epage%3D1083%26doi%3D10.1021%2Fcb300001m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sdek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLellan, W. R.</span></span> <span> </span><span class="NLM_article-title">Cardiac Myocyte Cell Cycle Control in Development, Disease, and Regeneration</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">544</span>, <span class="refDoi"> DOI: 10.1152/physrev.00032.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1152%2Fphysrev.00032.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=17429040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD2sXoslGmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2007&pages=521-544&author=P.+Ahujaauthor=P.+Sdekauthor=W.+R.+MacLellan&title=Cardiac+Myocyte+Cell+Cycle+Control+in+Development%2C+Disease%2C+and+Regeneration&doi=10.1152%2Fphysrev.00032.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Cardiac myocyte cell cycle control in development, disease, and regeneration</span></div><div class="casAuthors">Ahuja, Preeti; Sdek, Patima; Maclellan, W. Robb</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">521-544</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  Cardiac myocytes rapidly proliferate during fetal life but exit the cell cycle soon after birth in mammals.  Although the extent to which adult cardiac myocytes are capable of cell cycle reentry is controversial and species-specific differences may exist, it appears that for the vast majority of adult cardiac myocytes the predominant form of growth postnatally is an increase in cell size (hypertrophy) not no.  Unfortunately, this limits the ability of the heart to restore function after any significant injury.  Interest in novel regenerative therapies has led to the accumulation of much information on the mechanisms that regulate the rapid proliferation of cardiac myocytes in utero, their cell cycle exit in the perinatal period, and the permanent arrest (terminal differentiation) in adult myocytes.  The recent identification of cardiac progenitor cells capable of giving rise to cardiac myocyte-like cells has challenged the dogma that the heart is a terminally differentiated organ and opened new prospects for cardiac regeneration.  In this review, we summarize the current understanding of cardiomyocyte cell cycle control in normal development and disease.  In addn., we also discuss the potential usefulness of cardiomyocyte self-renewal as well as feasibility of therapeutic manipulation of the cardiac myocyte cell cycle for cardiac regeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxO6H0TKcdbVg90H21EOLACvtfcHk0lhUYnB4dCT1kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXoslGmu7c%253D&md5=956e63a025795a6e45177af98e7550e2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00032.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00032.2006%26sid%3Dliteratum%253Aachs%26aulast%3DAhuja%26aufirst%3DP.%26aulast%3DSdek%26aufirst%3DP.%26aulast%3DMacLellan%26aufirst%3DW.%2BR.%26atitle%3DCardiac%2520Myocyte%2520Cell%2520Cycle%2520Control%2520in%2520Development%252C%2520Disease%252C%2520and%2520Regeneration%26jtitle%3DPhysiol.%2520Rev.%26date%3D2007%26volume%3D87%26spage%3D521%26epage%3D544%26doi%3D10.1152%2Fphysrev.00032.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Addis, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epstein, J. A.</span></span> <span> </span><span class="NLM_article-title">Induced Regeneration--the Progress and Promise of Direct Reprogramming for Heart Repair</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1038/nm.3225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnm.3225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23836233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=829-836&author=R.+C.+Addisauthor=J.+A.+Epstein&title=Induced+Regeneration%2D%2Dthe+Progress+and+Promise+of+Direct+Reprogramming+for+Heart+Repair&doi=10.1038%2Fnm.3225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Induced regeneration-the progress and promise of direct reprogramming for heart repair</span></div><div class="casAuthors">Addis, Russell C.; Epstein, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">829-836</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Regeneration of cardiac tissue has the potential to transform cardiovascular medicine.  Recent advances in stem cell biol. and direct reprogramming, or transdifferentiation, have produced powerful new tools to advance this goal.  In this Review we examine key developments in the generation of new cardiomyocytes in vitro as well as the exciting progress that has been made toward in vivo reprogramming of cardiac tissue.  We also address controversies and hurdles that challenge the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr34PLjZ02oU7Vg90H21EOLACvtfcHk0lj-xNiDU_sdUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsr%252FJ&md5=0e23361ce4ed6a03f5342bf9818294e5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnm.3225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3225%26sid%3Dliteratum%253Aachs%26aulast%3DAddis%26aufirst%3DR.%2BC.%26aulast%3DEpstein%26aufirst%3DJ.%2BA.%26atitle%3DInduced%2520Regeneration--the%2520Progress%2520and%2520Promise%2520of%2520Direct%2520Reprogramming%2520for%2520Heart%2520Repair%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D829%26epage%3D836%26doi%3D10.1038%2Fnm.3225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muraoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, M.</span></span> <span> </span><span class="NLM_article-title">Direct Reprogramming of Fibroblasts into Myocytes to Reverse Fibrosis</span>. <i>Annu. Rev. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1146/annurev-physiol-021113-170301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1146%2Fannurev-physiol-021113-170301" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2014&pages=21-37&author=N.+Muraokaauthor=M.+Ieda&title=Direct+Reprogramming+of+Fibroblasts+into+Myocytes+to+Reverse+Fibrosis&doi=10.1146%2Fannurev-physiol-021113-170301"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1146%2Fannurev-physiol-021113-170301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-physiol-021113-170301%26sid%3Dliteratum%253Aachs%26aulast%3DMuraoka%26aufirst%3DN.%26aulast%3DIeda%26aufirst%3DM.%26atitle%3DDirect%2520Reprogramming%2520of%2520Fibroblasts%2520into%2520Myocytes%2520to%2520Reverse%2520Fibrosis%26jtitle%3DAnnu.%2520Rev.%2520Physiol.%26date%3D2014%26volume%3D76%26spage%3D21%26epage%3D37%26doi%3D10.1146%2Fannurev-physiol-021113-170301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, L.</span></span> <span> </span><span class="NLM_article-title">Direct Cardiac Reprogramming as a Novel Therapeutic Strategy for Treatment of Myocardial Infarction</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1521</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-6588-5_5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1007%2F978-1-4939-6588-5_5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=27910042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWit7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1521&publication_year=2017&pages=69-88&author=H.+Maauthor=L.+Wangauthor=J.+Liuauthor=L.+Qian&title=Direct+Cardiac+Reprogramming+as+a+Novel+Therapeutic+Strategy+for+Treatment+of+Myocardial+Infarction&doi=10.1007%2F978-1-4939-6588-5_5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Direct cardiac reprogramming as a novel therapeutic strategy for treatment of myocardial infarction</span></div><div class="casAuthors">Ma, Hong; Wang, Li; Liu, Jiandong; Qian, Li</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1521</span>
        (<span class="NLM_cas:issue">Cardiac Gene Therapy</span>),
    <span class="NLM_cas:pages">69-88</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Direct reprogramming of fibroblasts into induced cardiomyocytes (iCMs) holds great promise as a novel therapy for the treatment of heart failure, a common and morbid disease that is usually caused by irreversible loss of functional cardiomyocytes (CMs).  Recently, we and others showed that in a murine model of acute myocardial infarction, delivery of three transcription factors, Gata4, Mef2c, and Tbx5 converted endogenous cardiac fibroblasts into functional iCMs.  These iCMs integrated elec. and mech. with surrounding myocardium, resulting in a redn. in scar size and an improvement in heart function.  Our findings suggest that iCM reprogramming may be a means of regenerating functional CMs in vivo for patients with heart disease.  However, because relatively little is known about the factors that regulate iCM reprogramming, the applicability of iCM reprogramming is currently limited to the exptl. settings in which it has been attempted.  Specific hurdles include the relatively low conversion rate of iCMs and the need for reprogramming to occur in the context of acute injury.  Therefore, before this treatment can become a viable therapy for human heart disease, the optimal condition for efficient iCM generation must be detd.  Here, we provide a detailed protocol for both in vitro and in vivo iCM generation that has been optimized so far in our lab.  We hope that this protocol will lay a foundation for future further improvement of iCM generation and provide a platform for mechanistic studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeuYiCGsA7r7Vg90H21EOLACvtfcHk0lj-xNiDU_sdUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWit7vP&md5=6f47686a504ba7bd8b458c354ff49d77</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-6588-5_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-6588-5_5%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DL.%26atitle%3DDirect%2520Cardiac%2520Reprogramming%2520as%2520a%2520Novel%2520Therapeutic%2520Strategy%2520for%2520Treatment%2520of%2520Myocardial%2520Infarction%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2017%26volume%3D1521%26spage%3D69%26epage%3D88%26doi%3D10.1007%2F978-1-4939-6588-5_5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazeltine, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azarin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raval, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamp, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palecek, S. P.</span></span> <span> </span><span class="NLM_article-title">Robust Cardiomyocyte Differentiation from Human Pluripotent Stem Cells via Temporal Modulation of Canonical Wnt Signaling</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1848</span>– <span class="NLM_lpage">1857</span>, <span class="refDoi"> DOI: 10.1073/pnas.1200250109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1073%2Fpnas.1200250109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=1848-1857&author=X.+Lianauthor=C.+Hsiaoauthor=G.+Wilsonauthor=K.+Zhuauthor=L.+B.+Hazeltineauthor=S.+M.+Azarinauthor=K.+K.+Ravalauthor=J.+Zhangauthor=T.+J.+Kampauthor=S.+P.+Palecek&title=Robust+Cardiomyocyte+Differentiation+from+Human+Pluripotent+Stem+Cells+via+Temporal+Modulation+of+Canonical+Wnt+Signaling&doi=10.1073%2Fpnas.1200250109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1200250109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1200250109%26sid%3Dliteratum%253Aachs%26aulast%3DLian%26aufirst%3DX.%26aulast%3DHsiao%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DHazeltine%26aufirst%3DL.%2BB.%26aulast%3DAzarin%26aufirst%3DS.%2BM.%26aulast%3DRaval%26aufirst%3DK.%2BK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKamp%26aufirst%3DT.%2BJ.%26aulast%3DPalecek%26aufirst%3DS.%2BP.%26atitle%3DRobust%2520Cardiomyocyte%2520Differentiation%2520from%2520Human%2520Pluripotent%2520Stem%2520Cells%2520via%2520Temporal%2520Modulation%2520of%2520Canonical%2520Wnt%2520Signaling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D1848%26epage%3D1857%26doi%3D10.1073%2Fpnas.1200250109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azarin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazeltine, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamp, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palecek, S. P.</span></span> <span> </span><span class="NLM_article-title">Directed Cardiomyocyte Differentiation from Human Pluripotent Stem Cells by Modulating Wnt/β-Catenin Signaling under Fully Defined Conditions</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1038/nprot.2012.150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnprot.2012.150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23257984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1KisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=162-175&author=X.+Lianauthor=J.+Zhangauthor=S.+M.+Azarinauthor=K.+Zhuauthor=L.+B.+Hazeltineauthor=X.+Baoauthor=C.+Hsiaoauthor=T.+J.+Kampauthor=S.+P.+Palecek&title=Directed+Cardiomyocyte+Differentiation+from+Human+Pluripotent+Stem+Cells+by+Modulating+Wnt%2F%CE%B2-Catenin+Signaling+under+Fully+Defined+Conditions&doi=10.1038%2Fnprot.2012.150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/β-catenin signaling under fully defined conditions</span></div><div class="casAuthors">Lian, Xiaojun; Zhang, Jianhua; Azarin, Samira M.; Zhu, Kexian; Hazeltine, Laurie B.; Bao, Xiaoping; Hsiao, Cheston; Kamp, Timothy J.; Palecek, Sean P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">162-175</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The protocol described here efficiently directs human pluripotent stem cells (hPSCs) to functional cardiomyocytes in a completely defined, growth factor- and serum-free system by temporal modulation of regulators of canonical Wnt signaling.  Appropriate temporal application of a glycogen synthase kinase 3 (GSK3) inhibitor combined with the expression of β-catenin shRNA or a chem. Wnt inhibitor is sufficient to produce a high yield (0.8-1.3 million cardiomyocytes per cm2) of virtually pure (80-98%) functional cardiomyocytes in 14 d from multiple hPSC lines without cell sorting or selection.  Qual. (immunostaining) and quant. (flow cytometry) characterization of differentiated cells is described to assess the expression of cardiac transcription factors and myofilament proteins.  Flow cytometry of BrdU incorporation or Ki67 expression in conjunction with cardiac sarcomere myosin protein expression can be used to det. the proliferative capacity of hPSC-derived cardiomyocytes.  Functional human cardiomyocytes differentiated via these protocols may constitute a potential cell source for heart disease modeling, drug screening and cell-based therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKfgKIeSpJybVg90H21EOLACvtfcHk0lj-xNiDU_sdUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1KisLk%253D&md5=e8a8e7f0ab41ce99452cb8c4562129e7</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2012.150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2012.150%26sid%3Dliteratum%253Aachs%26aulast%3DLian%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAzarin%26aufirst%3DS.%2BM.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DHazeltine%26aufirst%3DL.%2BB.%26aulast%3DBao%26aufirst%3DX.%26aulast%3DHsiao%26aufirst%3DC.%26aulast%3DKamp%26aufirst%3DT.%2BJ.%26aulast%3DPalecek%26aufirst%3DS.%2BP.%26atitle%3DDirected%2520Cardiomyocyte%2520Differentiation%2520from%2520Human%2520Pluripotent%2520Stem%2520Cells%2520by%2520Modulating%2520Wnt%252F%25CE%25B2-Catenin%2520Signaling%2520under%2520Fully%2520Defined%2520Conditions%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D162%26epage%3D175%26doi%3D10.1038%2Fnprot.2012.150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">The Generalized Born/volume Integral Implicit Solvent Model: Estimation of the Free Energy of Hydration Using London Dispersion instead of Atomic Surface Area</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1698</span>, <span class="refDoi"> DOI: 10.1002/jcc.20933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2Fjcc.20933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=18307169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1693-1698&author=P.+Labute&title=The+Generalized+Born%2Fvolume+Integral+Implicit+Solvent+Model%3A+Estimation+of+the+Free+Energy+of+Hydration+Using+London+Dispersion+instead+of+Atomic+Surface+Area&doi=10.1002%2Fjcc.20933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The generalized Born/volume integral implicit solvent model: estimation of the free energy of hydration using London dispersion instead of atomic surface area</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1693-1698</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A new generalized Born model for estg. the free energy of hydration is presented.  The new generalized Born/vol. integral (GB/VI) ests. the free energy of hydration as a classical electrostatic energy plus a cavitation energy that is not based upon at. surface area (SA) used in GB/SA hydration models but on a VI London dispersion energy estd. from quantities already calcd. in the classical electrostatic energy.  The (relatively few) GB/VI model parameters are fitted to exptl. data, and parameterizations for two different at. partial charge models are presented.  Comparison of the calcd. and exptl. free energies of hydration for 560 small mols. (both neutral and charged) shows good agreement (r2 = 0.94).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCY3g7kWWRo7Vg90H21EOLACvtfcHk0lhBCKd8xgRd7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D&md5=59420cf1dbe689836873df8e802d06cd</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20933%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DThe%2520Generalized%2520Born%252Fvolume%2520Integral%2520Implicit%2520Solvent%2520Model%253A%2520Estimation%2520of%2520the%2520Free%2520Energy%2520of%2520Hydration%2520Using%2520London%2520Dispersion%2520instead%2520of%2520Atomic%2520Surface%2520Area%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26spage%3D1693%26epage%3D1698%26doi%3D10.1002%2Fjcc.20933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bakker, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Word, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure Validation by Calpha Geometry: Phi, Psi and Cbeta Deviation</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">450</span>, <span class="refDoi"> DOI: 10.1002/prot.10286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2Fprot.10286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12557186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2003&pages=437-450&author=S.+C.+Lovellauthor=I.+W.+Davisauthor=W.+B.+Arendallauthor=P.+I.+de+Bakkerauthor=J.+M.+Wordauthor=M.+G.+Prisantauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=Structure+Validation+by+Calpha+Geometry%3A+Phi%2C+Psi+and+Cbeta+Deviation&doi=10.1002%2Fprot.10286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structure validation by Cα geometry: φ, ψ and Cβ deviation</span></div><div class="casAuthors">Lovell, Simon C.; Davis, Ian W.; Arendall, W. Bryan, III; de Bakker, Paul I. W.; Word, J. Michael; Prisant, Michael G.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-450</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Geometrical validation around the Cα is described, with a new Cβ measure and updated Ramachandran plot.  Deviation of the obsd. Cβ atom from ideal position provides a single measure encapsulating the major structure-validation information contained in bond angle distortions.  Cβ deviation is sensitive to incompatibilities between sidechain and backbone caused by misfit conformations or inappropriate refinement restraints.  A new .vphi.,ψ plot using d.-dependent smoothing for 81,234 non-Gly, non-Pro, and non-prePro residues with B < 30 from 500 high-resoln. proteins shows sharp boundaries at crit. edges and clear delineation between large empty areas and regions that are allowed but disfavored.  One such region is the γ-turn conformation near +75°, -60°, counted as forbidden by common structure-validation programs; however, it occurs in well-ordered parts of good structures, it is overrepresented near functional sites, and strain is partly compensated by the γ-turn H-bond.  Favored and allowed .vphi.,ψ regions are also defined for Pro, pre-Pro, and Gly (important because Gly .vphi.,ψ angles are more permissive but less accurately detd.).  Details of these accurate empirical distributions are poorly predicted by previous theor. calcns., including a region left of α-helix, which rates as favorable in energy yet rarely occurs.  A proposed factor explaining this discrepancy is that crowding of the two-peptide NHs permits donating only a single H-bond.  New calcns. by H. Hu et al. (2002) for Ala and Gly dipeptides, using mixed quantum mechanics and mol. mechanics, fit our nonrepetitive data in excellent detail.  To run our geometrical evaluations on a user-uploaded file, see MOLPROBITY (http://kinemage.biochem.duke.edu) or RAMPAGE (http://www-cryst.bioc.cam.ac.uk/rampage).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFFimJUYSPl7Vg90H21EOLACvtfcHk0lhBCKd8xgRd7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D&md5=e9d41070d9e8832ef9aedf6a523dfd73</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fprot.10286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10286%26sid%3Dliteratum%253Aachs%26aulast%3DLovell%26aufirst%3DS.%2BC.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3Dde%2BBakker%26aufirst%3DP.%2BI.%26aulast%3DWord%26aufirst%3DJ.%2BM.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DStructure%2520Validation%2520by%2520Calpha%2520Geometry%253A%2520Phi%252C%2520Psi%2520and%2520Cbeta%2520Deviation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D50%26spage%3D437%26epage%3D450%26doi%3D10.1002%2Fprot.10286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">Training a Scoring Function for the Alignment of Small Molecules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1724</span>– <span class="NLM_lpage">1735</span>, <span class="refDoi"> DOI: 10.1021/ci100227h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100227h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFChs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1724-1735&author=S.+L.+Chanauthor=P.+Labute&title=Training+a+Scoring+Function+for+the+Alignment+of+Small+Molecules&doi=10.1021%2Fci100227h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Training a Scoring Function for the Alignment of Small Molecules</span></div><div class="casAuthors">Chan, Shek Ling; Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1724-1735</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A comprehensive data set of aligned ligands with highly similar binding pockets from the Protein Data Bank has been built.  Based on this data set, a scoring function for recognizing good alignment poses for small mols. has been developed.  This function is based on atoms and hydrogen-bond projected features.  The concept is simply that atoms and features of a similar type (hydrogen-bond acceptors/donors and hydrophobic) tend to occupy the same space in a binding pocket and atoms of incompatible types often tend to avoid the same space.  Comparison with some recently published results of small mol. alignments shows that the current scoring function can lead to performance better than those of several existing methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1T7YQyE40fLVg90H21EOLACvtfcHk0lhBCKd8xgRd7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFChs7nL&md5=97a3611b454a580b0fbc2db797344348</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fci100227h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100227h%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BL.%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DTraining%2520a%2520Scoring%2520Function%2520for%2520the%2520Alignment%2520of%2520Small%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1724%26epage%3D1735%26doi%3D10.1021%2Fci100227h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arceci, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. B.</span></span> <span> </span><span class="NLM_article-title">Mouse GATA-4: A Retinoic Acid-Inducible GATA-Binding Transcription Factor Expressed in Endodermally Derived Tissues and Heart</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2235</span>– <span class="NLM_lpage">2246</span>, <span class="refDoi"> DOI: 10.1128/MCB.13.4.2235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.13.4.2235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8455608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Wmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1993&pages=2235-2246&author=R.+J.+Arceciauthor=A.+A.+Kingauthor=M.+C.+Simonauthor=S.+H.+Orkinauthor=D.+B.+Wilson&title=Mouse+GATA-4%3A+A+Retinoic+Acid-Inducible+GATA-Binding+Transcription+Factor+Expressed+in+Endodermally+Derived+Tissues+and+Heart&doi=10.1128%2FMCB.13.4.2235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse GATA-4: A retinoic acid-inducible GATA-binding transcription factor expressed in endodermally derived tissues and heart</span></div><div class="casAuthors">Arceci, Robert J.; King, Alexandra A. J.; Simon, M. Celeste; Orkin, Stuart H.; Wilson, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2235-46</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">The authors report the cDNA cloning and characterization of mouse GATA-4, a new member of the family of zinc finger transcription factors that bind a core GATA motif.  GATA-4 cDNA was identified by screening a 6.5-day mouse embryo library with oligonucleotide probes corresponding to a highly conserved region of the finger domains.  Like other proteins of the family, GATA-4 is approx. 50 kDa in size and contains two zinc finger domains of the form C-X-N-C-(X17)-C-N-X-C.  Cotransfection assays in heterologous cells demonstrate that GATA-4 trans activates reporter constructs contg. GATA promoter elements.  Northern (RNA) anal. and in situ hybridization show that GATA-4 mRNA is expressed in the heart, intestinal epithelium, primitive endoderm, and gonads.  Retinoic acid-induced differentiation of mouse F9 cells into visceral or parietal endoderm is accompanied by increased expression of GATA-4 mRNA and protein.  In vitro differentiation of embryonic stem cells into embryoid bodies is also assocd. with increased GATA-4 expression.  The authors conclude that GATA-4 is a tissue-specific, retinoic acid-inducible, and developmentally regulated transcription factor.  On the basis of its tissue distribution, it is speculated that GATA-4 plays a role in gene expression in the heart, intestinal epithelium, primitive endoderm, and gonads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo00Zg1oUuGGrVg90H21EOLACvtfcHk0lh4O47vJtQzcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1Wmurc%253D&md5=d7bfb22717e981fa25b0d004f620d8d7</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1128%2FMCB.13.4.2235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.13.4.2235%26sid%3Dliteratum%253Aachs%26aulast%3DArceci%26aufirst%3DR.%2BJ.%26aulast%3DKing%26aufirst%3DA.%2BA.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DWilson%26aufirst%3DD.%2BB.%26atitle%3DMouse%2520GATA-4%253A%2520A%2520Retinoic%2520Acid-Inducible%2520GATA-Binding%2520Transcription%2520Factor%2520Expressed%2520in%2520Endodermally%2520Derived%2520Tissues%2520and%2520Heart%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1993%26volume%3D13%26spage%3D2235%26epage%3D2246%26doi%3D10.1128%2FMCB.13.4.2235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ranganayakulu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">Divergent Roles for NK-2 Class Homeobox Genes in Cardiogenesis in Flies and Mice</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">3037</span>– <span class="NLM_lpage">3048</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1998&pages=3037-3048&author=G.+Ranganayakuluauthor=D.+A.+Elliottauthor=R.+P.+Harveyauthor=E.+N.+Olson&title=Divergent+Roles+for+NK-2+Class+Homeobox+Genes+in+Cardiogenesis+in+Flies+and+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRanganayakulu%26aufirst%3DG.%26aulast%3DElliott%26aufirst%3DD.%2BA.%26aulast%3DHarvey%26aufirst%3DR.%2BP.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DDivergent%2520Roles%2520for%2520NK-2%2520Class%2520Homeobox%2520Genes%2520in%2520Cardiogenesis%2520in%2520Flies%2520and%2520Mice%26jtitle%3DDevelopment%26date%3D1998%26volume%3D125%26spage%3D3037%26epage%3D3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hakkola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingelman-Sundberg, M.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Down-Regulation of CYP2E1 Expression by Inflammatory Cytokines in Rat Hepatoma Cells</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1048</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1124/jpet.102.041582</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1124%2Fjpet.102.041582" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12604681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1eisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2003&pages=1048-1054&author=J.+Hakkolaauthor=Y.+Huauthor=M.+Ingelman-Sundberg&title=Mechanisms+of+Down-Regulation+of+CYP2E1+Expression+by+Inflammatory+Cytokines+in+Rat+Hepatoma+Cells&doi=10.1124%2Fjpet.102.041582"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of down-regulation of CYP2E1 expression by inflammatory cytokines in rat hepatoma cells</span></div><div class="casAuthors">Hakkola, Jukka; Hu, Yin; Ingelman-Sundberg, Magnus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1048-1055</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">CYP2E1 is one of the major cytochrome P 450 forms whose expression is strongly inhibited by inflammatory cytokines in humans and rodents.  In the present study, the authors have used the Fao rat hepatoma cell line that constitutively expresses CYP2E1 enzyme to investigate mechanisms of cytokine action.  The cells were treated with interleukin (IL)-1β, tumor necrosis factor-α (TNFα), or IL-6 for 24 or 72 h, and the expression of CYP2E1 was monitored at the transcriptional, mRNA, and protein levels.  All three cytokines decreased the CYP2E1 mRNA levels after 24 h, and the effect was even stronger after 72 h.  In contrast, significant inhibition of CYP2E1 protein was seen only after 72 h.  In transfection assays using a CYP2E1 5' -3685 to +29-luciferase construct, it was found that IL-6 inhibited gene transcription after 24 h, but a similar effect by IL-1β and TNFα was registered only after 72 h.  Using 5' deletions of the CYP2E1 5'-reporter construct a responsive region for the IL-6 effect was located to -669 to -507 base pairs in the CYP2E1 5'-flanking region.  Interestingly, IL-1β, but not TNFα, was found to reduce hepatocyte nuclear factor (HNF)-1α binding to the CYP2E1 promotor.  However, the transactivation function of HNF-1α was impaired in Fao cells.  In mouse primary hepatocytes, IL-1β decreased HNF-1α-mediated transactivation.  In conclusion, the authors' data indicate that inflammatory cytokines inhibit CYP2E1 expression by multiple mechanisms, including control of HNF-1α function and regulation of other transcriptional factors acting on the CYP2E1 5'-upstream regulatory region.  In addn., regulation of factors of importance for the CYP2E1 mRNA stability may be involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrShCTi2lN2V7Vg90H21EOLACvtfcHk0lh4O47vJtQzcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1eisr8%253D&md5=135375e135301578945a5836e83ad640</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.041582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.041582%26sid%3Dliteratum%253Aachs%26aulast%3DHakkola%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DIngelman-Sundberg%26aufirst%3DM.%26atitle%3DMechanisms%2520of%2520Down-Regulation%2520of%2520CYP2E1%2520Expression%2520by%2520Inflammatory%2520Cytokines%2520in%2520Rat%2520Hepatoma%2520Cells%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2003%26volume%3D304%26spage%3D1048%26epage%3D1054%26doi%3D10.1124%2Fjpet.102.041582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strauss, J. F.</span></span> <span> </span><span class="NLM_article-title">Activation of the Control Reporter Plasmids pRL-TK and pRL-SV40 by Multiple GATA Transcription Factors Can Lead to Aberrant Normalization of Transfection Efficiency</span>. <i>BMC Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.1186/1472-6750-4-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1186%2F1472-6750-4-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=15119957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A280%3ADC%252BD2c3mtFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=10&author=C.+K.+M.+Hoauthor=J.+F.+Strauss&title=Activation+of+the+Control+Reporter+Plasmids+pRL-TK+and+pRL-SV40+by+Multiple+GATA+Transcription+Factors+Can+Lead+to+Aberrant+Normalization+of+Transfection+Efficiency&doi=10.1186%2F1472-6750-4-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the control reporter plasmids pRL-TK and pRL-SV40 by multiple GATA transcription factors can lead to aberrant normalization of transfection efficiency</span></div><div class="casAuthors">Ho Clement K M; Strauss Jerome F 3rd</div><div class="citationInfo"><span class="NLM_cas:title">BMC biotechnology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Members of the GATA transcription factor family have been used in many transfection studies to investigate their roles in the regulation of gene expression.  To correct for variations in transfection efficiency, the Renilla luciferase encoding plasmids pRL-TK and pRL-SV40 are commonly used as normalization controls.  RESULTS:  We report here that plasmids expressing GATA-4 or GATA-6 transcription factor increased Renilla luciferase gene expression by 2 to 8 fold when co-transfected with pRL-TK or pRL-SV40.  This alteration of the control reporter gene activity was shown to cause erroneous normalization of transfection efficiency and thus misinterpretation of results in a transactivation assay.  To circumvent the problem, we generated two mutant control plasmids from which putative GATA response elements were deleted.  These deletions rendered pRL-SV40 unresponsive to GATA transcription factor stimulation and reduced the response of pRL-TK.  A database search also indicates that consensus GATA binding sequences are present in other commercially available Renilla luciferase encoding plasmids; therefore, the latter can potentially be transactivated by GATA transcription factors.  CONCLUSION:  Taken together, these findings highlight the importance of the selection of an appropriate control reporter plasmid for the normalization of transfection efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwZIWRsIIvYxjv2lLSLOxFfW6udTcc2eZ1AKg0WO85b7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3mtFKmsA%253D%253D&md5=e5eb0001ac6ffa79b777131644af3fb1</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1186%2F1472-6750-4-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6750-4-10%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DC.%2BK.%2BM.%26aulast%3DStrauss%26aufirst%3DJ.%2BF.%26atitle%3DActivation%2520of%2520the%2520Control%2520Reporter%2520Plasmids%2520pRL-TK%2520and%2520pRL-SV40%2520by%2520Multiple%2520GATA%2520Transcription%2520Factors%2520Can%2520Lead%2520to%2520Aberrant%2520Normalization%2520of%2520Transfection%2520Efficiency%26jtitle%3DBMC%2520Biotechnol.%26date%3D2004%26volume%3D4%26spage%3D10%26doi%3D10.1186%2F1472-6750-4-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerkela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikkarainen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majalahti-Palviainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokola, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Distinct Roles of Mitogen-Activated Protein Kinase Pathways in GATA-4 Transcription Factor-Mediated Regulation of B-Type Natriuretic Peptide Gene</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">13752</span>– <span class="NLM_lpage">13760</span>, <span class="refDoi"> DOI: 10.1074/jbc.M105736200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M105736200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=13752-13760&author=R.+Kerkelaauthor=S.+Pikkarainenauthor=T.+Majalahti-Palviainenauthor=H.+Tokolaauthor=H.+Ruskoaho&title=Distinct+Roles+of+Mitogen-Activated+Protein+Kinase+Pathways+in+GATA-4+Transcription+Factor-Mediated+Regulation+of+B-Type+Natriuretic+Peptide+Gene&doi=10.1074%2Fjbc.M105736200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105736200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105736200%26sid%3Dliteratum%253Aachs%26aulast%3DKerkela%26aufirst%3DR.%26aulast%3DPikkarainen%26aufirst%3DS.%26aulast%3DMajalahti-Palviainen%26aufirst%3DT.%26aulast%3DTokola%26aufirst%3DH.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DDistinct%2520Roles%2520of%2520Mitogen-Activated%2520Protein%2520Kinase%2520Pathways%2520in%2520GATA-4%2520Transcription%2520Factor-Mediated%2520Regulation%2520of%2520B-Type%2520Natriuretic%2520Peptide%2520Gene%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D13752%26epage%3D13760%26doi%3D10.1074%2Fjbc.M105736200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamprecht, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guertin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">CellProfiler: Image Analysis Software for Identifying and Quantifying Cell Phenotypes</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">R100</span>, <span class="refDoi"> DOI: 10.1186/gb-2006-7-10-r100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1186%2Fgb-2006-7-10-r100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=17076895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=R100&author=A.+E.+Carpenterauthor=T.+R.+Jonesauthor=M.+R.+Lamprechtauthor=C.+Clarkeauthor=I.+H.+Kangauthor=O.+Frimanauthor=D.+A.+Guertinauthor=J.+H.+Changauthor=R.+A.+Lindquistauthor=J.+Moffatauthor=P.+Gollandauthor=D.+M.+Sabatini&title=CellProfiler%3A+Image+Analysis+Software+for+Identifying+and+Quantifying+Cell+Phenotypes&doi=10.1186%2Fgb-2006-7-10-r100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">CellProfiler: image analysis software for identifying and quantifying cell phenotypes</span></div><div class="casAuthors">Carpenter Anne E; Jones Thouis R; Lamprecht Michael R; Clarke Colin; Kang In Han; Friman Ola; Guertin David A; Chang Joo Han; Lindquist Robert A; Moffat Jason; Golland Polina; Sabatini David M</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">R100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Biologists can now prepare and image thousands of samples per day using automation, enabling chemical screens and functional genomics (for example, using RNA interference).  Here we describe the first free, open-source system designed for flexible, high-throughput cell image analysis, CellProfiler.  CellProfiler can address a variety of biological questions quantitatively, including standard assays (for example, cell count, size, per-cell protein levels) and complex morphological assays (for example, cell/organelle shape or subcellular patterns of DNA or protein staining).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkZz-phUrfiVm8bhe6roZYfW6udTcc2eaWfClPOl8a3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D&md5=79d0e947559e12792f864d764657d3f4</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1186%2Fgb-2006-7-10-r100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2006-7-10-r100%26sid%3Dliteratum%253Aachs%26aulast%3DCarpenter%26aufirst%3DA.%2BE.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DLamprecht%26aufirst%3DM.%2BR.%26aulast%3DClarke%26aufirst%3DC.%26aulast%3DKang%26aufirst%3DI.%2BH.%26aulast%3DFriman%26aufirst%3DO.%26aulast%3DGuertin%26aufirst%3DD.%2BA.%26aulast%3DChang%26aufirst%3DJ.%2BH.%26aulast%3DLindquist%26aufirst%3DR.%2BA.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DGolland%26aufirst%3DP.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DCellProfiler%253A%2520Image%2520Analysis%2520Software%2520for%2520Identifying%2520and%2520Quantifying%2520Cell%2520Phenotypes%26jtitle%3DGenome%2520Biol.%26date%3D2006%26volume%3D7%26spage%3DR100%26doi%3D10.1186%2Fgb-2006-7-10-r100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamentsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljosa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A. E.</span></span> <span> </span><span class="NLM_article-title">Improved Structure, Function and Compatibility for CellProfiler: Modular High-Throughput Image Analysis Software</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1179</span>– <span class="NLM_lpage">1180</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1093%2Fbioinformatics%2Fbtr095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=21349861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1179-1180&author=L.+Kamentskyauthor=T.+R.+Jonesauthor=A.+Fraserauthor=M.-A.+Brayauthor=D.+J.+Loganauthor=K.+L.+Maddenauthor=V.+Ljosaauthor=C.+Ruedenauthor=K.+W.+Eliceiriauthor=A.+E.+Carpenter&title=Improved+Structure%2C+Function+and+Compatibility+for+CellProfiler%3A+Modular+High-Throughput+Image+Analysis+Software&doi=10.1093%2Fbioinformatics%2Fbtr095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Improved structure, function and compatibility for CellProfiler: modular high-throughput image analysis software</span></div><div class="casAuthors">Kamentsky, Lee; Jones, Thouis R.; Fraser, Adam; Bray, Mark-Anthony; Logan, David J.; Madden, Katherine L.; Ljosa, Vebjorn; Rueden, Curtis; Eliceiri, Kevin W.; Carpenter, Anne E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1179-1180</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: There is a strong and growing need in the biol. research community for accurate, automated image anal.  Here, we describe CellProfiler 2.0, which has been engineered to meet the needs of its growing user base.  It is more robust and user friendly, with new algorithms and features to facilitate high-throughput work.  ImageJ plugins can now be run within a CellProfiler pipeline.  Availability and Implementation: CellProfiler 2.0 is free and open source, available at http://www.cellprofiler.org under the GPL v. 2 license.  It is available as a packaged application for Macintosh OS X and Microsoft Windows and can be compiled for Linux.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="7d1c1313183d1f0f121c1914130e09140908091853120f1a">[email protected]</a> Supplementary information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa5_31IFp6crVg90H21EOLACvtfcHk0lgiX-RgdDVsDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKlu78%253D&md5=5f99adf6bdae740ed203dfe4adbb6396</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr095%26sid%3Dliteratum%253Aachs%26aulast%3DKamentsky%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DFraser%26aufirst%3DA.%26aulast%3DBray%26aufirst%3DM.-A.%26aulast%3DLogan%26aufirst%3DD.%2BJ.%26aulast%3DMadden%26aufirst%3DK.%2BL.%26aulast%3DLjosa%26aufirst%3DV.%26aulast%3DRueden%26aufirst%3DC.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26aulast%3DCarpenter%26aufirst%3DA.%2BE.%26atitle%3DImproved%2520Structure%252C%2520Function%2520and%2520Compatibility%2520for%2520CellProfiler%253A%2520Modular%2520High-Throughput%2520Image%2520Analysis%2520Software%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D1179%26epage%3D1180%26doi%3D10.1093%2Fbioinformatics%2Fbtr095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 27 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Nazanin Zanjanizadeh Ezazi, Rubina Ajdary, Alexandra Correia, Ermei Mäkilä, Jarno Salonen, Marianna Kemell, Jouni Hirvonen, Orlando J. Rojas, Heikki J. Ruskoaho, <span class="NLM_string-name hlFld-ContribAuthor">Hélder A. Santos</span>. </span><span class="cited-content_cbyCitation_article-title">Fabrication and Characterization of Drug-Loaded Conductive Poly(glycerol sebacate)/Nanoparticle-Based Composite Patch for Myocardial Infarction Applications. </span><span class="cited-content_cbyCitation_journal-name">ACS Applied Materials & Interfaces</span><span> <strong>2020,</strong> <em>12 </em>
                                    (6)
                                     , 6899-6909. <a href="https://doi.org/10.1021/acsami.9b21066" title="DOI URL">https://doi.org/10.1021/acsami.9b21066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsami.9b21066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsami.9b21066%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Applied%2520Materials%2520%2526%2520Interfaces%26atitle%3DFabrication%252Band%252BCharacterization%252Bof%252BDrug-Loaded%252BConductive%252BPoly%252528glycerol%252Bsebacate%252529%25252FNanoparticle-Based%252BComposite%252BPatch%252Bfor%252BMyocardial%252BInfarction%252BApplications%26aulast%3DZanjanizadeh%2BEzazi%26aufirst%3DNazanin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D19112019%26date%3D22012020%26date%3D03022020%26date%3D22012020%26volume%3D12%26issue%3D6%26spage%3D6899%26epage%3D6909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mikael Jumppanen, Sini M. Kinnunen, Mika J. Välimäki, Virpi Talman, Samuli Auno, Tanja Bruun, Gustav Boije af Gennäs, Henri Xhaard, Ingo B. Aumüller, Heikki Ruskoaho, <span class="NLM_string-name hlFld-ContribAuthor">Jari Yli-Kauhaluoma</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Identification, and Structure–Activity Relationship Analysis of GATA4 and NKX2-5 Protein–Protein Interaction Modulators. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 8284-8310. <a href="https://doi.org/10.1021/acs.jmedchem.9b01086" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01086</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01086%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252BIdentification%25252C%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BAnalysis%252Bof%252BGATA4%252Band%252BNKX2-5%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BModulators%26aulast%3DJumppanen%26aufirst%3DMikael%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D05072019%26date%3D03092019%26date%3D20082019%26volume%3D62%26issue%3D17%26spage%3D8284%26epage%3D8310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mika J.  Välimäki</span>, <span class="hlFld-ContribAuthor ">Robert S.  Leigh</span>, <span class="hlFld-ContribAuthor ">Sini M.  Kinnunen</span>, <span class="hlFld-ContribAuthor ">Alexander R.  March</span>, <span class="hlFld-ContribAuthor ">Ana Hernández  de Sande</span>, <span class="hlFld-ContribAuthor ">Matias  Kinnunen</span>, <span class="hlFld-ContribAuthor ">Markku  Varjosalo</span>, <span class="hlFld-ContribAuthor ">Merja  Heinäniemi</span>, <span class="hlFld-ContribAuthor ">Bogac L.  Kaynak</span>, <span class="hlFld-ContribAuthor ">Heikki  Ruskoaho</span>. </span><span class="cited-content_cbyCitation_article-title">GATA-targeted compounds modulate cardiac subtype cell differentiation in dual reporter stem cell line. </span><span class="cited-content_cbyCitation_journal-name">Stem Cell Research & Therapy</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13287-021-02259-z" title="DOI URL">https://doi.org/10.1186/s13287-021-02259-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13287-021-02259-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13287-021-02259-z%26sid%3Dliteratum%253Aachs%26jtitle%3DStem%2520Cell%2520Research%2520%2526%2520Therapy%26atitle%3DGATA-targeted%252Bcompounds%252Bmodulate%252Bcardiac%252Bsubtype%252Bcell%252Bdifferentiation%252Bin%252Bdual%252Breporter%252Bstem%252Bcell%252Bline%26aulast%3DV%25C3%25A4lim%25C3%25A4ki%26aufirst%3DMika%2BJ.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manman  Duan</span>, <span class="hlFld-ContribAuthor ">Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Jia  Liu</span>, <span class="hlFld-ContribAuthor ">Le  Qian</span>, <span class="hlFld-ContribAuthor ">Xiangguang  Chen</span>, <span class="hlFld-ContribAuthor ">Feng  Zhao</span>, <span class="hlFld-ContribAuthor ">Wentian  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhaomin  Zhong</span>, <span class="hlFld-ContribAuthor ">Yang  Yang</span>, <span class="hlFld-ContribAuthor ">Chengju  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Toxic effects of broflanilide exposure on development of zebrafish (Danio rerio) embryos and its potential cardiotoxicity mechanism. </span><span class="cited-content_cbyCitation_journal-name">Environmental Pollution</span><span> <strong>2021,</strong> <em>286 </em>, 117481. <a href="https://doi.org/10.1016/j.envpol.2021.117481" title="DOI URL">https://doi.org/10.1016/j.envpol.2021.117481</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.envpol.2021.117481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.envpol.2021.117481%26sid%3Dliteratum%253Aachs%26jtitle%3DEnvironmental%2520Pollution%26atitle%3DToxic%252Beffects%252Bof%252Bbroflanilide%252Bexposure%252Bon%252Bdevelopment%252Bof%252Bzebrafish%252B%252528Danio%252Brerio%252529%252Bembryos%252Band%252Bits%252Bpotential%252Bcardiotoxicity%252Bmechanism%26aulast%3DDuan%26aufirst%3DManman%26date%3D2021%26volume%3D286%26spage%3D117481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mengying  Yang</span>, <span class="hlFld-ContribAuthor ">Jiaxing  Huang</span>, <span class="hlFld-ContribAuthor ">Shuhui  Zhang</span>, <span class="hlFld-ContribAuthor ">Xin  Zhao</span>, <span class="hlFld-ContribAuthor ">Daofu  Feng</span>, <span class="hlFld-ContribAuthor ">Xizeng  Feng</span>. </span><span class="cited-content_cbyCitation_article-title">Melatonin mitigated circadian disruption and cardiovascular toxicity caused by 6-benzylaminopurine exposure in zebrafish. </span><span class="cited-content_cbyCitation_journal-name">Ecotoxicology and Environmental Safety</span><span> <strong>2021,</strong> <em>223 </em>, 112555. <a href="https://doi.org/10.1016/j.ecoenv.2021.112555" title="DOI URL">https://doi.org/10.1016/j.ecoenv.2021.112555</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ecoenv.2021.112555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ecoenv.2021.112555%26sid%3Dliteratum%253Aachs%26jtitle%3DEcotoxicology%2520and%2520Environmental%2520Safety%26atitle%3DMelatonin%252Bmitigated%252Bcircadian%252Bdisruption%252Band%252Bcardiovascular%252Btoxicity%252Bcaused%252Bby%252B6-benzylaminopurine%252Bexposure%252Bin%252Bzebrafish%26aulast%3DYang%26aufirst%3DMengying%26date%3D2021%26volume%3D223%26spage%3D112555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sung-Moo  Lee</span>, <span class="hlFld-ContribAuthor ">Eul-Bee  Ko</span>, <span class="hlFld-ContribAuthor ">Ryeo-Eun  Go</span>, <span class="hlFld-ContribAuthor ">Hong Kyu  Lee</span>, <span class="hlFld-ContribAuthor ">Kyung-Chul  Choi</span>. </span><span class="cited-content_cbyCitation_article-title">Effect of the phenylpyrrole fungicide fludioxonil on cell proliferation and cardiac differentiation in mouse embryonic stem cells. </span><span class="cited-content_cbyCitation_journal-name">Reproductive Toxicology</span><span> <strong>2021,</strong> <em>104 </em>, 76-84. <a href="https://doi.org/10.1016/j.reprotox.2021.07.006" title="DOI URL">https://doi.org/10.1016/j.reprotox.2021.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.reprotox.2021.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.reprotox.2021.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DReproductive%2520Toxicology%26atitle%3DEffect%252Bof%252Bthe%252Bphenylpyrrole%252Bfungicide%252Bfludioxonil%252Bon%252Bcell%252Bproliferation%252Band%252Bcardiac%252Bdifferentiation%252Bin%252Bmouse%252Bembryonic%252Bstem%252Bcells%26aulast%3DLee%26aufirst%3DSung-Moo%26date%3D2021%26volume%3D104%26spage%3D76%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keyuan  Zhong</span>, <span class="hlFld-ContribAuthor ">Yunlong  Meng</span>, <span class="hlFld-ContribAuthor ">Juan  Wu</span>, <span class="hlFld-ContribAuthor ">You  Wei</span>, <span class="hlFld-ContribAuthor ">Yong  Huang</span>, <span class="hlFld-ContribAuthor ">Jinze  Ma</span>, <span class="hlFld-ContribAuthor ">Huiqiang  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Effect of Flupyradifurone on Zebrafish Embryonic Development. </span><span class="cited-content_cbyCitation_journal-name">Environmental Pollution</span><span> <strong>2021,</strong> <em>57 </em>, 117323. <a href="https://doi.org/10.1016/j.envpol.2021.117323" title="DOI URL">https://doi.org/10.1016/j.envpol.2021.117323</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.envpol.2021.117323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.envpol.2021.117323%26sid%3Dliteratum%253Aachs%26jtitle%3DEnvironmental%2520Pollution%26atitle%3DEffect%252Bof%252BFlupyradifurone%252Bon%252BZebrafish%252BEmbryonic%252BDevelopment%26aulast%3DZhong%26aufirst%3DKeyuan%26date%3D2021%26volume%3D57%26spage%3D117323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emma Claire  James</span>, <span class="hlFld-ContribAuthor ">Eva  Tomaskovic-Crook</span>, <span class="hlFld-ContribAuthor ">Jeremy Micah  Crook</span>. </span><span class="cited-content_cbyCitation_article-title">Bioengineering Clinically Relevant Cardiomyocytes and Cardiac Tissues from Pluripotent Stem Cells. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (6)
                                     , 3005. <a href="https://doi.org/10.3390/ijms22063005" title="DOI URL">https://doi.org/10.3390/ijms22063005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22063005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22063005%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBioengineering%252BClinically%252BRelevant%252BCardiomyocytes%252Band%252BCardiac%252BTissues%252Bfrom%252BPluripotent%252BStem%252BCells%26aulast%3DJames%26aufirst%3DEmma%2BClaire%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D6%26spage%3D3005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie L.  Padula</span>, <span class="hlFld-ContribAuthor ">Nivedhitha  Velayutham</span>, <span class="hlFld-ContribAuthor ">Katherine E.  Yutzey</span>. </span><span class="cited-content_cbyCitation_article-title">Transcriptional Regulation of Postnatal Cardiomyocyte Maturation and Regeneration. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (6)
                                     , 3288. <a href="https://doi.org/10.3390/ijms22063288" title="DOI URL">https://doi.org/10.3390/ijms22063288</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22063288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22063288%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DTranscriptional%252BRegulation%252Bof%252BPostnatal%252BCardiomyocyte%252BMaturation%252Band%252BRegeneration%26aulast%3DPadula%26aufirst%3DStephanie%2BL.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D6%26spage%3D3288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tingting  Chai</span>, <span class="hlFld-ContribAuthor ">Feng  Cui</span>, <span class="hlFld-ContribAuthor ">Shanshan  Di</span>, <span class="hlFld-ContribAuthor ">Shenggan  Wu</span>, <span class="hlFld-ContribAuthor ">Yiming  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinquan  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">New insights into cardiotoxicity induced by chiral fluoxetine at environmental-level: Enantioselective arrhythmia in developmental zebrafish (Danio rerio). </span><span class="cited-content_cbyCitation_journal-name">Environmental Pollution</span><span> <strong>2021,</strong> <em>270 </em>, 116182. <a href="https://doi.org/10.1016/j.envpol.2020.116182" title="DOI URL">https://doi.org/10.1016/j.envpol.2020.116182</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.envpol.2020.116182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.envpol.2020.116182%26sid%3Dliteratum%253Aachs%26jtitle%3DEnvironmental%2520Pollution%26atitle%3DNew%252Binsights%252Binto%252Bcardiotoxicity%252Binduced%252Bby%252Bchiral%252Bfluoxetine%252Bat%252Benvironmental-level%25253A%252BEnantioselective%252Barrhythmia%252Bin%252Bdevelopmental%252Bzebrafish%252B%252528Danio%252Brerio%252529%26aulast%3DChai%26aufirst%3DTingting%26date%3D2021%26volume%3D270%26spage%3D116182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naying  Zheng</span>, <span class="hlFld-ContribAuthor ">Jinhui  Yan</span>, <span class="hlFld-ContribAuthor ">Wang  Qian</span>, <span class="hlFld-ContribAuthor ">Chao  Song</span>, <span class="hlFld-ContribAuthor ">Zhenghong  Zuo</span>, <span class="hlFld-ContribAuthor ">Chengyong  He</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of developmental toxicity of different surface modified CdSe/ZnS QDs in zebrafish embryos. </span><span class="cited-content_cbyCitation_journal-name">Journal of Environmental Sciences</span><span> <strong>2021,</strong> <em>100 </em>, 240-249. <a href="https://doi.org/10.1016/j.jes.2020.07.019" title="DOI URL">https://doi.org/10.1016/j.jes.2020.07.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jes.2020.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jes.2020.07.019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Environmental%2520Sciences%26atitle%3DComparison%252Bof%252Bdevelopmental%252Btoxicity%252Bof%252Bdifferent%252Bsurface%252Bmodified%252BCdSe%25252FZnS%252BQDs%252Bin%252Bzebrafish%252Bembryos%26aulast%3DZheng%26aufirst%3DNaying%26date%3D2021%26volume%3D100%26spage%3D240%26epage%3D249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert S.  Leigh</span>, <span class="hlFld-ContribAuthor ">Heikki J.  Ruskoaho</span>, <span class="hlFld-ContribAuthor ">Bogac L.  Kaynak</span>. </span><span class="cited-content_cbyCitation_article-title">Cholecystokinin peptide signaling is regulated by a TBX5-MEF2 axis in the heart. </span><span class="cited-content_cbyCitation_journal-name">Peptides</span><span> <strong>2021,</strong> <em>136 </em>, 170459. <a href="https://doi.org/10.1016/j.peptides.2020.170459" title="DOI URL">https://doi.org/10.1016/j.peptides.2020.170459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.peptides.2020.170459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.peptides.2020.170459%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptides%26atitle%3DCholecystokinin%252Bpeptide%252Bsignaling%252Bis%252Bregulated%252Bby%252Ba%252BTBX5-MEF2%252Baxis%252Bin%252Bthe%252Bheart%26aulast%3DLeigh%26aufirst%3DRobert%2BS.%26date%3D2021%26volume%3D136%26spage%3D170459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lotta  Pohjolainen</span>, <span class="hlFld-ContribAuthor ">Julia  Easton</span>, <span class="hlFld-ContribAuthor ">Reesha  Solanki</span>, <span class="hlFld-ContribAuthor ">Heikki  Ruskoaho</span>, <span class="hlFld-ContribAuthor ">Virpi  Talman</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological Protein Kinase C Modulators Reveal a Pro-hypertrophic Role for Novel Protein Kinase C Isoforms in Human Induced Pluripotent Stem Cell-Derived Cardiomyocytes. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.553852" title="DOI URL">https://doi.org/10.3389/fphar.2020.553852</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.553852&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.553852%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DPharmacological%252BProtein%252BKinase%252BC%252BModulators%252BReveal%252Ba%252BPro-hypertrophic%252BRole%252Bfor%252BNovel%252BProtein%252BKinase%252BC%252BIsoforms%252Bin%252BHuman%252BInduced%252BPluripotent%252BStem%252BCell-Derived%252BCardiomyocytes%26aulast%3DPohjolainen%26aufirst%3DLotta%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ping  Mi</span>, <span class="hlFld-ContribAuthor ">Ya-Qiu  Tang</span>, <span class="hlFld-ContribAuthor ">Xi-Zeng  Feng</span>. </span><span class="cited-content_cbyCitation_article-title">Acute fluorene-9-bisphenol exposure damages early development and induces cardiotoxicity in zebrafish (Danio rerio). </span><span class="cited-content_cbyCitation_journal-name">Ecotoxicology and Environmental Safety</span><span> <strong>2020,</strong> <em>202 </em>, 110922. <a href="https://doi.org/10.1016/j.ecoenv.2020.110922" title="DOI URL">https://doi.org/10.1016/j.ecoenv.2020.110922</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ecoenv.2020.110922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ecoenv.2020.110922%26sid%3Dliteratum%253Aachs%26jtitle%3DEcotoxicology%2520and%2520Environmental%2520Safety%26atitle%3DAcute%252Bfluorene-9-bisphenol%252Bexposure%252Bdamages%252Bearly%252Bdevelopment%252Band%252Binduces%252Bcardiotoxicity%252Bin%252Bzebrafish%252B%252528Danio%252Brerio%252529%26aulast%3DMi%26aufirst%3DPing%26date%3D2020%26volume%3D202%26spage%3D110922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yunlong  Meng</span>, <span class="hlFld-ContribAuthor ">Keyuan  Zhong</span>, <span class="hlFld-ContribAuthor ">Juhua  Xiao</span>, <span class="hlFld-ContribAuthor ">Yong  Huang</span>, <span class="hlFld-ContribAuthor ">You  Wei</span>, <span class="hlFld-ContribAuthor ">Lin  Tang</span>, <span class="hlFld-ContribAuthor ">Suping  Chen</span>, <span class="hlFld-ContribAuthor ">Juan  Wu</span>, <span class="hlFld-ContribAuthor ">Jinze  Ma</span>, <span class="hlFld-ContribAuthor ">Zigang  Cao</span>, <span class="hlFld-ContribAuthor ">Xinjun  Liao</span>, <span class="hlFld-ContribAuthor ">Huiqiang  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Exposure to pyrimethanil induces developmental toxicity and cardiotoxicity in zebrafish. </span><span class="cited-content_cbyCitation_journal-name">Chemosphere</span><span> <strong>2020,</strong> <em>255 </em>, 126889. <a href="https://doi.org/10.1016/j.chemosphere.2020.126889" title="DOI URL">https://doi.org/10.1016/j.chemosphere.2020.126889</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemosphere.2020.126889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemosphere.2020.126889%26sid%3Dliteratum%253Aachs%26jtitle%3DChemosphere%26atitle%3DExposure%252Bto%252Bpyrimethanil%252Binduces%252Bdevelopmental%252Btoxicity%252Band%252Bcardiotoxicity%252Bin%252Bzebrafish%26aulast%3DMeng%26aufirst%3DYunlong%26date%3D2020%26volume%3D255%26spage%3D126889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chen  Tang</span>, <span class="hlFld-ContribAuthor ">Chao  Shen</span>, <span class="hlFld-ContribAuthor ">Kongyang  Zhu</span>, <span class="hlFld-ContribAuthor ">Yixi  Zhou</span>, <span class="hlFld-ContribAuthor ">Yung-Jen  Chuang</span>, <span class="hlFld-ContribAuthor ">Chengyong  He</span>, <span class="hlFld-ContribAuthor ">Zhenghong  Zuo</span>. </span><span class="cited-content_cbyCitation_article-title">Exposure to the AhR agonist cyprodinil impacts the cardiac development and function of zebrafish larvae. </span><span class="cited-content_cbyCitation_journal-name">Ecotoxicology and Environmental Safety</span><span> <strong>2020,</strong> <em>201 </em>, 110808. <a href="https://doi.org/10.1016/j.ecoenv.2020.110808" title="DOI URL">https://doi.org/10.1016/j.ecoenv.2020.110808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ecoenv.2020.110808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ecoenv.2020.110808%26sid%3Dliteratum%253Aachs%26jtitle%3DEcotoxicology%2520and%2520Environmental%2520Safety%26atitle%3DExposure%252Bto%252Bthe%252BAhR%252Bagonist%252Bcyprodinil%252Bimpacts%252Bthe%252Bcardiac%252Bdevelopment%252Band%252Bfunction%252Bof%252Bzebrafish%252Blarvae%26aulast%3DTang%26aufirst%3DChen%26date%3D2020%26volume%3D201%26spage%3D110808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rubina  Ajdary</span>, <span class="hlFld-ContribAuthor ">Nazanin Zanjanizadeh  Ezazi</span>, <span class="hlFld-ContribAuthor ">Alexandra  Correia</span>, <span class="hlFld-ContribAuthor ">Marianna  Kemell</span>, <span class="hlFld-ContribAuthor ">Siqi  Huan</span>, <span class="hlFld-ContribAuthor ">Heikki J.  Ruskoaho</span>, <span class="hlFld-ContribAuthor ">Jouni  Hirvonen</span>, <span class="hlFld-ContribAuthor ">Hélder A.  Santos</span>, <span class="hlFld-ContribAuthor ">Orlando J.  Rojas</span>. </span><span class="cited-content_cbyCitation_article-title">Multifunctional 3D‐Printed Patches for Long‐Term Drug Release Therapies after Myocardial Infarction. </span><span class="cited-content_cbyCitation_journal-name">Advanced Functional Materials</span><span> <strong>2020,</strong> <em>30 </em>
                                    (34)
                                     , 2003440. <a href="https://doi.org/10.1002/adfm.202003440" title="DOI URL">https://doi.org/10.1002/adfm.202003440</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adfm.202003440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadfm.202003440%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Functional%2520Materials%26atitle%3DMultifunctional%252B3D%2525E2%252580%252590Printed%252BPatches%252Bfor%252BLong%2525E2%252580%252590Term%252BDrug%252BRelease%252BTherapies%252Bafter%252BMyocardial%252BInfarction%26aulast%3DAjdary%26aufirst%3DRubina%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D34%26spage%3D2003440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. Tuuli  Karhu</span>, <span class="hlFld-ContribAuthor ">Sini M.  Kinnunen</span>, <span class="hlFld-ContribAuthor ">Marja  Tölli</span>, <span class="hlFld-ContribAuthor ">Mika J.  Välimäki</span>, <span class="hlFld-ContribAuthor ">Zoltán  Szabó</span>, <span class="hlFld-ContribAuthor ">Virpi  Talman</span>, <span class="hlFld-ContribAuthor ">Heikki  Ruskoaho</span>. </span><span class="cited-content_cbyCitation_article-title">GATA4-targeted compound exhibits cardioprotective actions against doxorubicin-induced toxicity in vitro and in vivo: establishment of a chronic cardiotoxicity model using human iPSC-derived cardiomyocytes. </span><span class="cited-content_cbyCitation_journal-name">Archives of Toxicology</span><span> <strong>2020,</strong> <em>94 </em>
                                    (6)
                                     , 2113-2130. <a href="https://doi.org/10.1007/s00204-020-02711-8" title="DOI URL">https://doi.org/10.1007/s00204-020-02711-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00204-020-02711-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00204-020-02711-8%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Toxicology%26atitle%3DGATA4-targeted%252Bcompound%252Bexhibits%252Bcardioprotective%252Bactions%252Bagainst%252Bdoxorubicin-induced%252Btoxicity%252Bin%252Bvitro%252Band%252Bin%252Bvivo%25253A%252Bestablishment%252Bof%252Ba%252Bchronic%252Bcardiotoxicity%252Bmodel%252Busing%252Bhuman%252BiPSC-derived%252Bcardiomyocytes%26aulast%3DKarhu%26aufirst%3DS.%2BTuuli%26date%3D2020%26date%3D2020%26volume%3D94%26issue%3D6%26spage%3D2113%26epage%3D2130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mika J.  Välimäki</span>, <span class="hlFld-ContribAuthor ">Heikki J.  Ruskoaho</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting GATA4 for cardiac repair. </span><span class="cited-content_cbyCitation_journal-name">IUBMB Life</span><span> <strong>2020,</strong> <em>72 </em>
                                    (1)
                                     , 68-79. <a href="https://doi.org/10.1002/iub.2150" title="DOI URL">https://doi.org/10.1002/iub.2150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/iub.2150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fiub.2150%26sid%3Dliteratum%253Aachs%26jtitle%3DIUBMB%2520Life%26atitle%3DTargeting%252BGATA4%252Bfor%252Bcardiac%252Brepair%26aulast%3DV%25C3%25A4lim%25C3%25A4ki%26aufirst%3DMika%2BJ.%26date%3D2020%26date%3D2019%26volume%3D72%26issue%3D1%26spage%3D68%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuqiong  Wu</span>, <span class="hlFld-ContribAuthor ">Ying  Zhang</span>, <span class="hlFld-ContribAuthor ">Meng  Chen</span>, <span class="hlFld-ContribAuthor ">Qihong  Yang</span>, <span class="hlFld-ContribAuthor ">Shanshan  Zhuang</span>, <span class="hlFld-ContribAuthor ">Liangju  Lv</span>, <span class="hlFld-ContribAuthor ">Zhenghong  Zuo</span>, <span class="hlFld-ContribAuthor ">Chonggang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Exposure to low-level metalaxyl impacts the cardiac development and function of zebrafish embryos. </span><span class="cited-content_cbyCitation_journal-name">Journal of Environmental Sciences</span><span> <strong>2019,</strong> <em>85 </em>, 1-8. <a href="https://doi.org/10.1016/j.jes.2019.03.019" title="DOI URL">https://doi.org/10.1016/j.jes.2019.03.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jes.2019.03.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jes.2019.03.019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Environmental%2520Sciences%26atitle%3DExposure%252Bto%252Blow-level%252Bmetalaxyl%252Bimpacts%252Bthe%252Bcardiac%252Bdevelopment%252Band%252Bfunction%252Bof%252Bzebrafish%252Bembryos%26aulast%3DWu%26aufirst%3DYuqiong%26date%3D2019%26volume%3D85%26spage%3D1%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing-Jing  HUANG</span>, <span class="hlFld-ContribAuthor ">Yong  XIE</span>, <span class="hlFld-ContribAuthor ">He  LI</span>, <span class="hlFld-ContribAuthor ">Xiao-Xiao  ZHANG</span>, <span class="hlFld-ContribAuthor ">Qing  HUANG</span>, <span class="hlFld-ContribAuthor ">Yao  ZHU</span>, <span class="hlFld-ContribAuthor ">Ping  GU</span>, <span class="hlFld-ContribAuthor ">Wei-Min  JIANG</span>. </span><span class="cited-content_cbyCitation_article-title">YQWY decoction reverses cardiac hypertrophy induced by TAC through inhibiting GATA4 phosphorylation and MAPKs. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Natural Medicines</span><span> <strong>2019,</strong> <em>17 </em>
                                    (10)
                                     , 746-755. <a href="https://doi.org/10.1016/S1875-5364(19)30091-3" title="DOI URL">https://doi.org/10.1016/S1875-5364(19)30091-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1875-5364(19)30091-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1875-5364%2819%2930091-3%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Natural%2520Medicines%26atitle%3DYQWY%252Bdecoction%252Breverses%252Bcardiac%252Bhypertrophy%252Binduced%252Bby%252BTAC%252Bthrough%252Binhibiting%252BGATA4%252Bphosphorylation%252Band%252BMAPKs%26aulast%3DHUANG%26aufirst%3DJing-Jing%26date%3D2019%26volume%3D17%26issue%3D10%26spage%3D746%26epage%3D755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sini M.  Kinnunen</span>, <span class="hlFld-ContribAuthor ">Marja  Tölli</span>, <span class="hlFld-ContribAuthor ">Mika J.  Välimäki</span>, <span class="hlFld-ContribAuthor ">Erhe  Gao</span>, <span class="hlFld-ContribAuthor ">Zoltan  Szabo</span>, <span class="hlFld-ContribAuthor ">Jaana  Rysä</span>, <span class="hlFld-ContribAuthor ">Mónica P. A.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Pauli  Ohukainen</span>, <span class="hlFld-ContribAuthor ">Raisa  Serpi</span>, <span class="hlFld-ContribAuthor ">Alexandra  Correia</span>, <span class="hlFld-ContribAuthor ">Ermei  Mäkilä</span>, <span class="hlFld-ContribAuthor ">Jarno  Salonen</span>, <span class="hlFld-ContribAuthor ">Jouni  Hirvonen</span>, <span class="hlFld-ContribAuthor ">Hélder A.  Santos</span>, <span class="hlFld-ContribAuthor ">Heikki  Ruskoaho</span>. </span><span class="cited-content_cbyCitation_article-title">Cardiac Actions of a Small Molecule Inhibitor Targeting GATA4–NKX2-5 Interaction. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-22830-8" title="DOI URL">https://doi.org/10.1038/s41598-018-22830-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-22830-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-22830-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCardiac%252BActions%252Bof%252Ba%252BSmall%252BMolecule%252BInhibitor%252BTargeting%252BGATA4%2525E2%252580%252593NKX2-5%252BInteraction%26aulast%3DKinnunen%26aufirst%3DSini%2BM.%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adel  Abuzenadah</span>, <span class="hlFld-ContribAuthor ">Saad  Alsaedi</span>, <span class="hlFld-ContribAuthor ">Sajjad  Karim</span>, <span class="hlFld-ContribAuthor ">Mohammed  Al-Qahtani</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Overexpressed Transcription Factor FOXO1 in Fatal Cardiovascular Septal Defects in Patau Syndrome: Molecular and Therapeutic Strategies. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (11)
                                     , 3547. <a href="https://doi.org/10.3390/ijms19113547" title="DOI URL">https://doi.org/10.3390/ijms19113547</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19113547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19113547%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DRole%252Bof%252BOverexpressed%252BTranscription%252BFactor%252BFOXO1%252Bin%252BFatal%252BCardiovascular%252BSeptal%252BDefects%252Bin%252BPatau%252BSyndrome%25253A%252BMolecular%252Band%252BTherapeutic%252BStrategies%26aulast%3DAbuzenadah%26aufirst%3DAdel%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D11%26spage%3D3547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. Tuuli  Karhu</span>, <span class="hlFld-ContribAuthor ">Mika J.  Välimäki</span>, <span class="hlFld-ContribAuthor ">Mikael  Jumppanen</span>, <span class="hlFld-ContribAuthor ">Sini M.  Kinnunen</span>, <span class="hlFld-ContribAuthor ">Lotta  Pohjolainen</span>, <span class="hlFld-ContribAuthor ">Robert S.  Leigh</span>, <span class="hlFld-ContribAuthor ">Samuli  Auno</span>, <span class="hlFld-ContribAuthor ">Gábor  Földes</span>, <span class="hlFld-ContribAuthor ">Gustav  Boije af Gennäs</span>, <span class="hlFld-ContribAuthor ">Jari  Yli-Kauhaluoma</span>, <span class="hlFld-ContribAuthor ">Heikki  Ruskoaho</span>, <span class="hlFld-ContribAuthor ">Virpi  Talman</span>. </span><span class="cited-content_cbyCitation_article-title">Stem cells are the most sensitive screening tool to identify toxicity of GATA4-targeted novel small-molecule compounds. </span><span class="cited-content_cbyCitation_journal-name">Archives of Toxicology</span><span> <strong>2018,</strong> <em>92 </em>
                                    (9)
                                     , 2897-2911. <a href="https://doi.org/10.1007/s00204-018-2257-1" title="DOI URL">https://doi.org/10.1007/s00204-018-2257-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00204-018-2257-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00204-018-2257-1%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Toxicology%26atitle%3DStem%252Bcells%252Bare%252Bthe%252Bmost%252Bsensitive%252Bscreening%252Btool%252Bto%252Bidentify%252Btoxicity%252Bof%252BGATA4-targeted%252Bnovel%252Bsmall-molecule%252Bcompounds%26aulast%3DKarhu%26aufirst%3DS.%2BTuuli%26date%3D2018%26date%3D2018%26volume%3D92%26issue%3D9%26spage%3D2897%26epage%3D2911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Virpi  Talman</span>, <span class="hlFld-ContribAuthor ">Riikka  Kivelä</span>. </span><span class="cited-content_cbyCitation_article-title">Cardiomyocyte—Endothelial Cell Interactions in Cardiac Remodeling and Regeneration. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cardiovascular Medicine</span><span> <strong>2018,</strong> <em>5 </em><a href="https://doi.org/10.3389/fcvm.2018.00101" title="DOI URL">https://doi.org/10.3389/fcvm.2018.00101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcvm.2018.00101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcvm.2018.00101%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cardiovascular%2520Medicine%26atitle%3DCardiomyocyte%2525E2%252580%252594Endothelial%252BCell%252BInteractions%252Bin%252BCardiac%252BRemodeling%252Band%252BRegeneration%26aulast%3DTalman%26aufirst%3DVirpi%26date%3D2018%26date%3D2018%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hermann AM  Mucke</span>. </span><span class="cited-content_cbyCitation_article-title">Patent Highlights February–March 2018. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Patent Analyst</span><span> <strong>2018,</strong> <em>7 </em>
                                    (4)
                                     , 147-154. <a href="https://doi.org/10.4155/ppa-2018-0014" title="DOI URL">https://doi.org/10.4155/ppa-2018-0014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/ppa-2018-0014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Fppa-2018-0014%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Patent%2520Analyst%26atitle%3DPatent%252BHighlights%252BFebruary%2525E2%252580%252593March%252B2018%26aulast%3DMucke%26aufirst%3DHermann%2BAM%26date%3D2018%26volume%3D7%26issue%3D4%26spage%3D147%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuqiong  Wu</span>, <span class="hlFld-ContribAuthor ">Zhenghong  Zuo</span>, <span class="hlFld-ContribAuthor ">Meng  Chen</span>, <span class="hlFld-ContribAuthor ">Yixi  Zhou</span>, <span class="hlFld-ContribAuthor ">Qihong  Yang</span>, <span class="hlFld-ContribAuthor ">Shanshan  Zhuang</span>, <span class="hlFld-ContribAuthor ">Chonggang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">The developmental effects of low-level procymidone towards zebrafish embryos and involved mechanism. </span><span class="cited-content_cbyCitation_journal-name">Chemosphere</span><span> <strong>2018,</strong> <em>193 </em>, 928-935. <a href="https://doi.org/10.1016/j.chemosphere.2017.11.105" title="DOI URL">https://doi.org/10.1016/j.chemosphere.2017.11.105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemosphere.2017.11.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemosphere.2017.11.105%26sid%3Dliteratum%253Aachs%26jtitle%3DChemosphere%26atitle%3DThe%252Bdevelopmental%252Beffects%252Bof%252Blow-level%252Bprocymidone%252Btowards%252Bzebrafish%252Bembryos%252Band%252Binvolved%252Bmechanism%26aulast%3DWu%26aufirst%3DYuqiong%26date%3D2018%26volume%3D193%26spage%3D928%26epage%3D935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Cell-based reporter gene assay used for the screening of compound libraries targeting the GATA4–NKX2-5 transcriptional synergy. COS-1 cells were transfected with the 3xNKE-luciferase reporter construct and GATA4 and NKX2-5 expression plasmids and treated with small molecules for 24 h. The cells were lysed, and the luciferase reporter gene activity was measured by a luminometer. (A) Compounds <b>3</b>–<b>6</b> had a significant inhibitory effect on the GATA4–NKX2-5 transcriptional synergy, while compound <b>7</b> enhanced the GATA4–NKX2-5 transcriptional synergy. (B) Compound <b>3</b> dose-dependently inhibited the GATA4–NKX2-5 transcriptional synergy in the COS-1 cells. The data are shown as the mean ± SD, <i>n</i> = 2–9: (∗∗) <i>p</i> < 0.01, (∗∗∗) <i>p</i> < 0.001 vs vehicle treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Overall Flowchart Illustrating the Process of Screening and Validating the GATA4–NKX2-5 Synergy Modulators<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>BNP, B-type natriuretic peptide; GPCR, G-protein-coupled receptor.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Chronological Workflow of the Hit Compound Search and Optimization Process Initiated from the Active Fragments<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Scheme illustrates the molecular structure, compound code numbers, and corresponding IC<sub>50</sub> values in the GATA4–NKX2-5 luciferase reporter gene assay.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) C-terminal zinc finger of GATA4 mediates the DNA binding and GATA4–NKX2-5 protein–protein interaction. Computational methods were applied to analyze and screen two separate binding sites. A pharmacophore model with excluded volumes was built for the DNA binding cavity of GATA4. (B) The low energy conformations of compounds <b>3</b>, <b>4</b>, and <b>5</b> were utilized as a basis for the pharmacophore model. All three compounds inhibited the GATA4–NKX2-5 synergy in the luciferase reporter gene assay. Atom colors are the following: white = hydrogen, gray = carbon, red = oxygen, blue = nitrogen, and yellow = sulfur. Pharmacophore features are the following: blue sphere = acceptor, orange sphere = aromatic, and green sphere = hydrophobic.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of compounds on GATA4 and NKX2-5 protein DNA binding. (A, B) Nuclear proteins from COS-1 cells were extracted, and binding reactions and EMSA were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. The concentration of the compounds <b>3</b>–<b>5</b> was 50 μM. (C, D) Neonatal rat cardiomyocytes were treated with the compounds <b>3</b>, <b>4</b>, and <b>7</b> for 24 h. Nuclear proteins were extracted, and binding reactions and EMSA were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Specific GATA4 and NKX2-5 bands are indicated. The concentration of the compounds was 50 μM. Supershift reactions were performed by incubating the reaction mixtures with anti-GATA4 and anti-NKX2-5 antibodies. Asterisks denote the lanes with the supershifted samples. The dashed line indicates where the EMSA image has been cut. The data are shown relative to the vehicle treatment as the mean ± SD, <i>n</i> = 3–5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of different compounds in the bioassays. (A–D) Neonatal rat cardiomyocytes (CMs) were treated with 100 nM endothelin-1 (ET-1) for 23 h. Compounds were added to the cells at concentrations of 10–100 μM 1 h prior to ET-1. The results are expressed as the ratio of ANP and BNP mRNA to ribosomal 18 S RNA, which were all determined by RT-PCR at 24 h. The data are shown relative to the vehicle sample as the mean ± SD: vehicle, <i>n</i> = 24; compounds, <i>n</i> = 3–6. (E) The effect of compound <b>3</b> on the hypertrophic response was investigated by exposing the CMs to 10 μM for 48 h with or without stretching. After the stretching, the CMs were fixed, stained for cardiac α-actinin, F-actin, and nuclei, and imaged under a fluorescence microscope. The surface area of the α-actinin expressing cells was determined from the images using CellProfiler software. The results are expressed as the mean of three independent experiments ± SEM: (∗) <i>p</i> < 0.05, (∗∗) <i>p</i> < 0.01, (∗∗∗) <i>p</i> < 0.001 vs vehicle treatment (A–D) or appropriate group (E).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cytotoxicity of compounds <b>3</b>, <b>4</b>, and <b>7</b> in neonatal rat ventricular myocytes. (A) Adenylate kinase (AK) release was determined after the cells were treated for 24 h with each compound at a concentration of 50 μM. The data are shown relative to the vehicle treatment as the mean ± SD, <i>n</i> = 3. (B) To detect apoptotic cells, a TUNEL assay was performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. TUNEL-positive cells were determined after the cells were treated for 24 h with each compound at a concentration of 50 μM. The data are shown as the mean ± SD, <i>n</i> = 2 (for compound <b>7</b> in TUNEL assay, <i>n</i> = 1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) C-terminal extension in the second zinc finger domain (yellow ribbon) contributes to the DNA binding of GATA4. (B) For the activation of the GATA4–NKX2-5 interaction, the loop extension needs to relocate and (C) enforce the structural reorganization of the GATA4 complex in the tandem GATA binding site. (C) The GATA4 and NKX2-5 proteins were overexpressed in the COS-1 cells. Nuclear proteins were extracted, and EMSA was performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. GATA4 alone (four parallel samples) or together with NKX2-5 (four parallel samples) was incubated in binding reactions with DNA. GATA4 binding to the rat BNP promoter (−90 bp tandem GATA site) resulted in two separate bands, and their intensities depended on the presence of NKX2-5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/medium/jm-2017-008164_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Transcriptional Activity of GATA4 Is Regulated by Interactions with Several Other TFs and Post-Translational Modifications<a onclick="showRef(event, 'ref13 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref13 ref17 ref18 ref19">(13,17−19)</a><sup>,</sup><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-18/acs.jmedchem.7b00816/20180831/images/large/jm-2017-008164_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00816&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Most of the protein–protein interactions of GATA factors are mediated by the C-terminal zinc finger, while the N-terminal zinc finger interacts with Friend of GATA (FOG) transcription factors. Selected heterotypic interactions and DNA occupation modes for GATA4 are classified based on experimental measurements of the binding preferences.<a onclick="showRef(event, 'ref25 ref42 ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref25 ref42 ref43 ref44 ref45 ref46">(25,42−46)</a> Heterotypic GATA4 complex subpopulations have distinct context dependent effects on the commitment of stem cells toward the cardiac fate or the enhancement of hypertrophic gene expression in mature cardiac cells. SRF, serum response factor; SRE, serum response element; PTM, post-translational modification; Phos, phosphorylation; Acet, acetylation; Sumo/Ubi, sumoylation/ubiquitination.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80232" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80232" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 73 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjian, R.</span></span> <span> </span><span class="NLM_article-title">Transcription Regulation and Animal Diversity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1038/nature01763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnature01763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12853946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlt1CktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2003&pages=147-151&author=M.+Levineauthor=R.+Tjian&title=Transcription+Regulation+and+Animal+Diversity&doi=10.1038%2Fnature01763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription regulation and animal diversity</span></div><div class="casAuthors">Levine, Michael; Tjian, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">6945</span>),
    <span class="NLM_cas:pages">147-151</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Whole-genome sequence assemblies are now available for seven different animals, including nematode worms, mice and humans.  Comparative genome analyses reveal a surprising constancy in genetic content: vertebrate genomes have only about twice the no. of genes that invertebrate genomes have, and the increase is primarily due to the duplication of existing genes rather than the invention of new ones.  How, then, has evolutionary diversity arisen Emerging evidence suggests that organismal complexity arises from progressively more elaborate regulation of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkOXHnchkm1rVg90H21EOLACvtfcHk0lgSZ0mEboXz1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlt1CktLw%253D&md5=7fa0d905a78a07135ef6610d9c6b1e07</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature01763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01763%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DM.%26aulast%3DTjian%26aufirst%3DR.%26atitle%3DTranscription%2520Regulation%2520and%2520Animal%2520Diversity%26jtitle%3DNature%26date%3D2003%26volume%3D424%26spage%3D147%26epage%3D151%26doi%3D10.1038%2Fnature01763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spitz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlong, E. E.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors: From Enhancer Binding to Developmental Control</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1038/nrg3207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnrg3207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=22868264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCrtrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=613-626&author=F.+Spitzauthor=E.+E.+Furlong&title=Transcription+Factors%3A+From+Enhancer+Binding+to+Developmental+Control&doi=10.1038%2Fnrg3207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors: from enhancer binding to developmental control</span></div><div class="casAuthors">Spitz, Francois; Furlong, Eileen E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">613-626</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Developmental progression is driven by specific spatiotemporal domains of gene expression, which give rise to stereotypically patterned embryos even in the presence of environmental and genetic variation.  Views of how transcription factors regulate gene expression are changing owing to recent genome-wide studies of transcription factor binding and RNA expression.  Such studies reveal patterns that, at first glance, seem to contrast with the robustness of the developmental processes they encode.  This review discusses the current knowledge of transcription factor function from genomic and genetic studies and how different strategies, including extensive cooperative regulation (both direct and indirect), progressive priming of regulatory elements, and the integration of activities from multiple enhancers, confer specificity and robustness to transcriptional regulation during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNokxJil_DoLVg90H21EOLACvtfcHk0lgSZ0mEboXz1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCrtrbM&md5=43cf42f36512a874d1c3c8f062fbd151</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg3207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3207%26sid%3Dliteratum%253Aachs%26aulast%3DSpitz%26aufirst%3DF.%26aulast%3DFurlong%26aufirst%3DE.%2BE.%26atitle%3DTranscription%2520Factors%253A%2520From%2520Enhancer%2520Binding%2520to%2520Developmental%2520Control%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D613%26epage%3D626%26doi%3D10.1038%2Fnrg3207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flores, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oviedo, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sage, J.</span></span> <span> </span><span class="NLM_article-title">Essential Role for the Planarian Intestinal GATA Transcription Factor in Stem Cells and Regeneration</span>. <i>Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>418</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1016/j.ydbio.2016.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.ydbio.2016.08.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=418&publication_year=2016&pages=179-188&author=N.+M.+Floresauthor=N.+J.+Oviedoauthor=J.+Sage&title=Essential+Role+for+the+Planarian+Intestinal+GATA+Transcription+Factor+in+Stem+Cells+and+Regeneration&doi=10.1016%2Fj.ydbio.2016.08.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ydbio.2016.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ydbio.2016.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DFlores%26aufirst%3DN.%2BM.%26aulast%3DOviedo%26aufirst%3DN.%2BJ.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DEssential%2520Role%2520for%2520the%2520Planarian%2520Intestinal%2520GATA%2520Transcription%2520Factor%2520in%2520Stem%2520Cells%2520and%2520Regeneration%26jtitle%3DDev.%2520Biol.%26date%3D2016%26volume%3D418%26spage%3D179%26epage%3D188%26doi%3D10.1016%2Fj.ydbio.2016.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">Requirement of the Transcription Factor GATA4 for Heart Tube Formation and Ventral Morphogenesis</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1072</span>, <span class="refDoi"> DOI: 10.1101/gad.11.8.1061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.11.8.1061" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=1061-1072&author=J.+D.+Molkentinauthor=Q.+Linauthor=S.+A.+Duncanauthor=E.+N.+Olson&title=Requirement+of+the+Transcription+Factor+GATA4+for+Heart+Tube+Formation+and+Ventral+Morphogenesis&doi=10.1101%2Fgad.11.8.1061"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fgad.11.8.1061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.11.8.1061%26sid%3Dliteratum%253Aachs%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DDuncan%26aufirst%3DS.%2BA.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DRequirement%2520of%2520the%2520Transcription%2520Factor%2520GATA4%2520for%2520Heart%2520Tube%2520Formation%2520and%2520Ventral%2520Morphogenesis%26jtitle%3DGenes%2520Dev.%26date%3D1997%26volume%3D11%26spage%3D1061%26epage%3D1072%26doi%3D10.1101%2Fgad.11.8.1061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R. P.</span></span> <span> </span><span class="NLM_article-title">Myogenic and Morphogenetic Defects in the Heart Tubes of Murine Embryos Lacking the Homeo Box Gene Nkx2-5</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1654</span>– <span class="NLM_lpage">1666</span>, <span class="refDoi"> DOI: 10.1101/gad.9.13.1654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.9.13.1654" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=1654-1666&author=I.+Lyonsauthor=L.+M.+Parsonsauthor=L.+Hartleyauthor=R.+Liauthor=J.+E.+Andrewsauthor=L.+Robbauthor=R.+P.+Harvey&title=Myogenic+and+Morphogenetic+Defects+in+the+Heart+Tubes+of+Murine+Embryos+Lacking+the+Homeo+Box+Gene+Nkx2-5&doi=10.1101%2Fgad.9.13.1654"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.9.13.1654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.9.13.1654%26sid%3Dliteratum%253Aachs%26aulast%3DLyons%26aufirst%3DI.%26aulast%3DParsons%26aufirst%3DL.%2BM.%26aulast%3DHartley%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DAndrews%26aufirst%3DJ.%2BE.%26aulast%3DRobb%26aufirst%3DL.%26aulast%3DHarvey%26aufirst%3DR.%2BP.%26atitle%3DMyogenic%2520and%2520Morphogenetic%2520Defects%2520in%2520the%2520Heart%2520Tubes%2520of%2520Murine%2520Embryos%2520Lacking%2520the%2520Homeo%2520Box%2520Gene%2520Nkx2-5%26jtitle%3DGenes%2520Dev.%26date%3D1995%26volume%3D9%26spage%3D1654%26epage%3D1666%26doi%3D10.1101%2Fgad.9.13.1654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horb, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, G. H.</span></span> <span> </span><span class="NLM_article-title">Tbx5 Is Essential for Heart Development</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1751</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=1999&pages=1739-1751&author=M.+E.+Horbauthor=G.+H.+Thomsen&title=Tbx5+Is+Essential+for+Heart+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHorb%26aufirst%3DM.%2BE.%26aulast%3DThomsen%26aufirst%3DG.%2BH.%26atitle%3DTbx5%2520Is%2520Essential%2520for%2520Heart%2520Development%26jtitle%3DDevelopment%26date%3D1999%26volume%3D126%26spage%3D1739%26epage%3D1751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grepin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">Enhanced Cardiogenesis in Embryonic Stem Cells Overexpressing the GATA-4 Transcription Factor</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">2387</span>– <span class="NLM_lpage">2395</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=1997&pages=2387-2395&author=C.+Grepinauthor=G.+Nemerauthor=M.+Nemer&title=Enhanced+Cardiogenesis+in+Embryonic+Stem+Cells+Overexpressing+the+GATA-4+Transcription+Factor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGrepin%26aufirst%3DC.%26aulast%3DNemer%26aufirst%3DG.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DEnhanced%2520Cardiogenesis%2520in%2520Embryonic%2520Stem%2520Cells%2520Overexpressing%2520the%2520GATA-4%2520Transcription%2520Factor%26jtitle%3DDevelopment%26date%3D1997%26volume%3D124%26spage%3D2387%26epage%3D2395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Lozano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M. D.</span></span> <span> </span><span class="NLM_article-title">Cardiac Muscle Regeneration: Lessons from Development</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1101/gad.2018411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.2018411" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=299-309&author=M.+Mercolaauthor=P.+Ruiz-Lozanoauthor=M.+D.+Schneider&title=Cardiac+Muscle+Regeneration%3A+Lessons+from+Development&doi=10.1101%2Fgad.2018411"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fgad.2018411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2018411%26sid%3Dliteratum%253Aachs%26aulast%3DMercola%26aufirst%3DM.%26aulast%3DRuiz-Lozano%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DM.%2BD.%26atitle%3DCardiac%2520Muscle%2520Regeneration%253A%2520Lessons%2520from%2520Development%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D299%26epage%3D309%26doi%3D10.1101%2Fgad.2018411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grepin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dagnino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haberstroh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antakly, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">A Hormone-Encoding Gene Identifies a Pathway for Cardiac but Not Skeletal Muscle Gene Transcription</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3115</span>– <span class="NLM_lpage">3129</span>, <span class="refDoi"> DOI: 10.1128/MCB.14.5.3115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.14.5.3115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8164667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK2cXltVWjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=3115-3129&author=C.+Grepinauthor=L.+Dagninoauthor=L.+Robitailleauthor=L.+Haberstrohauthor=T.+Antaklyauthor=M.+Nemer&title=A+Hormone-Encoding+Gene+Identifies+a+Pathway+for+Cardiac+but+Not+Skeletal+Muscle+Gene+Transcription&doi=10.1128%2FMCB.14.5.3115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A hormone-encoding gene identifies a pathway for cardiac but not skeletal muscle gene transcription</span></div><div class="casAuthors">Grepin, Claudine; Dagnino, Lina; Robitaille, Lynda; Haberstroh, Linda; Antakly, Tony; Nemer, Mona</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3115-29</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">In contrast to skeletal muscle, the mechanisms responsible for activation and maintenance of tissue-specific transcription in cardiac muscle remain poorly understood.  A family of hormone-encoding genes is expressed in a highly specific manner in cardiac but not skeletal myocytes.  This includes the A- and B-type natriuretic peptide (ANP and BNP) genes, which encode peptide hormones with crucial roles in the regulation of blood vol. and pressure.  Since these genes are markers of cardiac cells, they were used to probe the mechanisms for cardiac muscle-specific transcription.  Cloning and functional anal. of the rat BNP upstream sequences revealed unexpected structural resemblance to erythroid but not to muscle-specific promoters and enhancers, including a requirement for regulatory elements contg. GATA motifs.  A cDNA clone corresponding to a member of the GATA family of transcription factors was isolated from a cardiomyocyte cDNA library.  Transcription of this GATA gene is restricted mostly to the heart and is undetectable in skeletal muscle.  Within the heart, GATA transcripts are localized in ANP- and BNP-expressing myocytes, and forced expression of the GATA protein in heterologous cells markedly activates transcription from the natural cardiac muscle-specific ANP and BNP promoters.  This GATA-dependent pathway defines the first mechanism for cardiac muscle-specific transcription.  Moreover, the present findings reveal striking similarities between the mechanisms controlling gene expression in hematopoietic and cardiac cells and may have important implications for studies of cardiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJ_V6zoGTlrVg90H21EOLACvtfcHk0linxeR5bzUMtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltVWjtr8%253D&md5=cfbafcf23285c31cd7f1503f6e8ba60f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.5.3115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.5.3115%26sid%3Dliteratum%253Aachs%26aulast%3DGrepin%26aufirst%3DC.%26aulast%3DDagnino%26aufirst%3DL.%26aulast%3DRobitaille%26aufirst%3DL.%26aulast%3DHaberstroh%26aufirst%3DL.%26aulast%3DAntakly%26aufirst%3DT.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DA%2520Hormone-Encoding%2520Gene%2520Identifies%2520a%2520Pathway%2520for%2520Cardiac%2520but%2520Not%2520Skeletal%2520Muscle%2520Gene%2520Transcription%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D3115%26epage%3D3129%26doi%3D10.1128%2FMCB.14.5.3115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thuerauf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanford, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glembotski, C. C.</span></span> <span> </span><span class="NLM_article-title">Regulation of Rat Brain Natriuretic Peptide Transcription. A Potential Role for GATA-Related Transcription Factors in Myocardial Cell Gene Expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">17772</span>– <span class="NLM_lpage">17775</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=17772-17775&author=D.+J.+Thueraufauthor=D.+S.+Hanfordauthor=C.+C.+Glembotski&title=Regulation+of+Rat+Brain+Natriuretic+Peptide+Transcription.+A+Potential+Role+for+GATA-Related+Transcription+Factors+in+Myocardial+Cell+Gene+Expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThuerauf%26aufirst%3DD.%2BJ.%26aulast%3DHanford%26aufirst%3DD.%2BS.%26aulast%3DGlembotski%26aufirst%3DC.%2BC.%26atitle%3DRegulation%2520of%2520Rat%2520Brain%2520Natriuretic%2520Peptide%2520Transcription.%2520A%2520Potential%2520Role%2520for%2520GATA-Related%2520Transcription%2520Factors%2520in%2520Myocardial%2520Cell%2520Gene%2520Expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D17772%26epage%3D17775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalvakolanu, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span> <span> </span><span class="NLM_article-title">Transcription Factor GATA-4 Regulates Cardiac Muscle-Specific Expression of the Alpha-Myosin Heavy-Chain Gene</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">4947</span>– <span class="NLM_lpage">4957</span>, <span class="refDoi"> DOI: 10.1128/MCB.14.7.4947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.14.7.4947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8007990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK2cXlslajsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=4947-4957&author=J.+D.+Molkentinauthor=D.+V.+Kalvakolanuauthor=B.+E.+Markham&title=Transcription+Factor+GATA-4+Regulates+Cardiac+Muscle-Specific+Expression+of+the+Alpha-Myosin+Heavy-Chain+Gene&doi=10.1128%2FMCB.14.7.4947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor GATA-4 regulates cardiac muscle-specific expression of the α-myosin heavy-chain gene</span></div><div class="casAuthors">Molkentin, Jeffery D.; Kalvakolanu, Dhananjaya V.; Markham, Bruce E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4947-57</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">The α-myosin heavy-chain (α-MHC) gene is the major structural protein in the adult rodent myocardium.  Its expression is restricted to the heart by a complex interplay of trans-acting factors and their cis-acting sites.  However, to date, the factors that have been shown to regulate expression of this gene have also been found in skeletal muscle cells.  Recently, transcription factor GATA-4, which has a tissue distribution limited to the heart and endodermally derived tissues, was identified.  The authors recently found two putative GATA-binding sites within the proximal enhancer of the α-MHC gene, suggesting that GATA-4 might regulate its expression.  In this study, the authors establish that GATA-4 interacts with the α-MHC GATA sites to stimulate cardiac muscle-specific expression.  Mutation of the GATA-4-binding sites either individually or together decreased activity by 50 and 88% in the adult myocardium, resp.  GATA-4-dependent enhancement of activity from a heterologous promoter was mediated through the α-MHC GATA sites.  Co-injection of an α-MHC promoter construct with a GATA-4 expression vector permitted ectopic expression in skeletal muscle but not in fibroblasts.  Thus, the lack of α-MHC expression in skeletal muscle correlates with a lack of GATA-4.  GATA-4 DNA binding activity was significantly up-regulated in triiodothyronine- or retinoic acid-treated cardiomyocytes.  Putative GATA-4-binding sites are also found in the regulatory regions of other cardiac muscle-expressed structural genes.  This indicates a mechanism whereby triiodothyronine and retinoic acid can exert coordinate control of the cardiac phenotype through a trans-acting regulatory factor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-pweq57zFLVg90H21EOLACvtfcHk0lhocrN14NMlkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlslajsLg%253D&md5=0b717840bb2aa8503fd9341900e352ef</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.7.4947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.7.4947%26sid%3Dliteratum%253Aachs%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DKalvakolanu%26aufirst%3DD.%2BV.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26atitle%3DTranscription%2520Factor%2520GATA-4%2520Regulates%2520Cardiac%2520Muscle-Specific%2520Expression%2520of%2520the%2520Alpha-Myosin%2520Heavy-Chain%2520Gene%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D4947%26epage%3D4957%26doi%3D10.1128%2FMCB.14.7.4947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jobe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitsis, R. N.</span></span> <span> </span><span class="NLM_article-title">Cis-Acting Sequences That Mediate Induction of Beta-Myosin Heavy Chain Gene Expression during Left Ventricular Hypertrophy due to Aortic Constriction</span>. <i>Circulation</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">3943</span>– <span class="NLM_lpage">3953</span>, <span class="refDoi"> DOI: 10.1161/01.CIR.96.11.3943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2F01.CIR.96.11.3943" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1997&pages=3943-3953&author=K.+Hasegawaauthor=S.+J.+Leeauthor=S.+M.+Jobeauthor=B.+E.+Markhamauthor=R.+N.+Kitsis&title=Cis-Acting+Sequences+That+Mediate+Induction+of+Beta-Myosin+Heavy+Chain+Gene+Expression+during+Left+Ventricular+Hypertrophy+due+to+Aortic+Constriction&doi=10.1161%2F01.CIR.96.11.3943"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1161%2F01.CIR.96.11.3943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.CIR.96.11.3943%26sid%3Dliteratum%253Aachs%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DJobe%26aufirst%3DS.%2BM.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26aulast%3DKitsis%26aufirst%3DR.%2BN.%26atitle%3DCis-Acting%2520Sequences%2520That%2520Mediate%2520Induction%2520of%2520Beta-Myosin%2520Heavy%2520Chain%2520Gene%2520Expression%2520during%2520Left%2520Ventricular%2520Hypertrophy%2520due%2520to%2520Aortic%2520Constriction%26jtitle%3DCirculation%26date%3D1997%26volume%3D96%26spage%3D3943%26epage%3D3953%26doi%3D10.1161%2F01.CIR.96.11.3943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pikkarainen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokola, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">GATA Transcription Factors in the Developing and Adult Heart</span>. <i>Cardiovasc. Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">196</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1016/j.cardiores.2004.03.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cardiores.2004.03.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2004&pages=196-207&author=S.+Pikkarainenauthor=H.+Tokolaauthor=R.+Kerkelaauthor=H.+Ruskoaho&title=GATA+Transcription+Factors+in+the+Developing+and+Adult+Heart&doi=10.1016%2Fj.cardiores.2004.03.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cardiores.2004.03.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cardiores.2004.03.025%26sid%3Dliteratum%253Aachs%26aulast%3DPikkarainen%26aufirst%3DS.%26aulast%3DTokola%26aufirst%3DH.%26aulast%3DKerkela%26aufirst%3DR.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DGATA%2520Transcription%2520Factors%2520in%2520the%2520Developing%2520and%2520Adult%2520Heart%26jtitle%3DCardiovasc.%2520Res.%26date%3D2004%26volume%3D63%26spage%3D196%26epage%3D207%26doi%3D10.1016%2Fj.cardiores.2004.03.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pikkarainen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokola, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majalahti-Palviainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hautala, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuolteenaho, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">GATA-4 Is a Nuclear Mediator of Mechanical Stretch-Activated Hypertrophic Program</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">23807</span>– <span class="NLM_lpage">23816</span>, <span class="refDoi"> DOI: 10.1074/jbc.M302719200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M302719200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=23807-23816&author=S.+Pikkarainenauthor=H.+Tokolaauthor=T.+Majalahti-Palviainenauthor=R.+Kerkelaauthor=N.+Hautalaauthor=S.+S.+Bhallaauthor=F.+Charronauthor=M.+Nemerauthor=O.+Vuolteenahoauthor=H.+Ruskoaho&title=GATA-4+Is+a+Nuclear+Mediator+of+Mechanical+Stretch-Activated+Hypertrophic+Program&doi=10.1074%2Fjbc.M302719200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M302719200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M302719200%26sid%3Dliteratum%253Aachs%26aulast%3DPikkarainen%26aufirst%3DS.%26aulast%3DTokola%26aufirst%3DH.%26aulast%3DMajalahti-Palviainen%26aufirst%3DT.%26aulast%3DKerkela%26aufirst%3DR.%26aulast%3DHautala%26aufirst%3DN.%26aulast%3DBhalla%26aufirst%3DS.%2BS.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DVuolteenaho%26aufirst%3DO.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DGATA-4%2520Is%2520a%2520Nuclear%2520Mediator%2520of%2520Mechanical%2520Stretch-Activated%2520Hypertrophic%2520Program%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D23807%26epage%3D23816%26doi%3D10.1074%2Fjbc.M302719200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Välimäki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tölli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohlfahrt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darwich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor-like Structure and Interaction of Congenital Heart Disease-Associated Factors GATA4 and NKX2-5</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0144145</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0144145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1371%2Fjournal.pone.0144145" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0144145&author=S.+Kinnunenauthor=M.+V%C3%A4lim%C3%A4kiauthor=M.+T%C3%B6lliauthor=G.+Wohlfahrtauthor=R.+Darwichauthor=H.+Komatiauthor=M.+Nemerauthor=H.+Ruskoaho&title=Nuclear+Receptor-like+Structure+and+Interaction+of+Congenital+Heart+Disease-Associated+Factors+GATA4+and+NKX2-5&doi=10.1371%2Fjournal.pone.0144145"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0144145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0144145%26sid%3Dliteratum%253Aachs%26aulast%3DKinnunen%26aufirst%3DS.%26aulast%3DV%25C3%25A4lim%25C3%25A4ki%26aufirst%3DM.%26aulast%3DT%25C3%25B6lli%26aufirst%3DM.%26aulast%3DWohlfahrt%26aufirst%3DG.%26aulast%3DDarwich%26aufirst%3DR.%26aulast%3DKomati%26aufirst%3DH.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DNuclear%2520Receptor-like%2520Structure%2520and%2520Interaction%2520of%2520Congenital%2520Heart%2520Disease-Associated%2520Factors%2520GATA4%2520and%2520NKX2-5%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0144145%26doi%3D10.1371%2Fjournal.pone.0144145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schueler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tönjes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperling, S. R.</span></span> <span> </span><span class="NLM_article-title">The Cardiac Transcription Network Modulated by Gata4, Mef2a, Nkx2.5, Srf, Histone Modifications, and microRNAs</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e1001313</span>, <span class="refDoi"> DOI: 10.1371/journal.pgen.1001313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1371%2Fjournal.pgen.1001313" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1001313&author=J.+Schlesingerauthor=M.+Schuelerauthor=M.+Grunertauthor=J.+J.+Fischerauthor=Q.+Zhangauthor=T.+Kruegerauthor=M.+Langeauthor=M.+T%C3%B6njesauthor=I.+Dunkelauthor=S.+R.+Sperling&title=The+Cardiac+Transcription+Network+Modulated+by+Gata4%2C+Mef2a%2C+Nkx2.5%2C+Srf%2C+Histone+Modifications%2C+and+microRNAs&doi=10.1371%2Fjournal.pgen.1001313"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1001313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1001313%26sid%3Dliteratum%253Aachs%26aulast%3DSchlesinger%26aufirst%3DJ.%26aulast%3DSchueler%26aufirst%3DM.%26aulast%3DGrunert%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DJ.%2BJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKrueger%26aufirst%3DT.%26aulast%3DLange%26aufirst%3DM.%26aulast%3DT%25C3%25B6njes%26aufirst%3DM.%26aulast%3DDunkel%26aufirst%3DI.%26aulast%3DSperling%26aufirst%3DS.%2BR.%26atitle%3DThe%2520Cardiac%2520Transcription%2520Network%2520Modulated%2520by%2520Gata4%252C%2520Mef2a%252C%2520Nkx2.5%252C%2520Srf%252C%2520Histone%2520Modifications%252C%2520and%2520microRNAs%26jtitle%3DPLoS%2520Genet.%26date%3D2011%26volume%3D7%26spage%3De1001313%26doi%3D10.1371%2Fjournal.pgen.1001313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsimiklis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">Tissue-Specific GATA Factors Are Transcriptional Effectors of the Small GTPase RhoA</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2702</span>– <span class="NLM_lpage">2719</span>, <span class="refDoi"> DOI: 10.1101/gad.915701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1101%2Fgad.915701" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=2702-2719&author=F.+Charronauthor=G.+Tsimiklisauthor=M.+Arcandauthor=L.+Robitailleauthor=Q.+Liangauthor=J.+D.+Molkentinauthor=S.+Melocheauthor=M.+Nemer&title=Tissue-Specific+GATA+Factors+Are+Transcriptional+Effectors+of+the+Small+GTPase+RhoA&doi=10.1101%2Fgad.915701"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1101%2Fgad.915701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.915701%26sid%3Dliteratum%253Aachs%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DTsimiklis%26aufirst%3DG.%26aulast%3DArcand%26aufirst%3DM.%26aulast%3DRobitaille%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DQ.%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DMeloche%26aufirst%3DS.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DTissue-Specific%2520GATA%2520Factors%2520Are%2520Transcriptional%2520Effectors%2520of%2520the%2520Small%2520GTPase%2520RhoA%26jtitle%3DGenes%2520Dev.%26date%3D2001%26volume%3D15%26spage%3D2702%26epage%3D2719%26doi%3D10.1101%2Fgad.915701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yanazume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, T.</span></span> <span> </span><span class="NLM_article-title">Cardiac p300 Is Involved in Myocyte Growth with Decompensated Heart Failure</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3593</span>– <span class="NLM_lpage">3606</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.10.3593-3606.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.23.10.3593-3606.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12724418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs1yhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=3593-3606&author=T.+Yanazumeauthor=K.+Hasegawaauthor=T.+Morimotoauthor=T.+Kawamuraauthor=H.+Wadaauthor=A.+Matsumoriauthor=Y.+Kawaseauthor=M.+Hiraiauthor=T.+Kita&title=Cardiac+p300+Is+Involved+in+Myocyte+Growth+with+Decompensated+Heart+Failure&doi=10.1128%2FMCB.23.10.3593-3606.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cardiac p300 is involved in myocyte growth with decompensated heart failure</span></div><div class="casAuthors">Yanazume, Tetsuhiko; Hasegawa, Koji; Morimoto, Tatsuya; Kawamura, Teruhisa; Wada, Hiromichi; Matsumori, Akira; Kawase, Yosuke; Hirai, Maretoshi; Kita, Toru</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3593-3606</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A variety of stresses on the heart initiate a no. of subcellular signaling pathways, which finally reach the nuclei of cardiac myocytes and cause myocyte hypertrophy with heart failure.  However, common nuclear pathways that lead to this state are unknown.  A zinc finger protein, GATA-4, is one of the transcription factors that mediate changes in gene expression during myocardial-cell hypertrophy.  P300 not only acts as a transcriptional coactivator of GATA-4, but also possesses an intrinsic histone acetyltransferase activity.  In primary cardiac myocytes derived from neonatal rats, we show that stimulation with phenylephrine increased an acetylated form of GATA-4 and its DNA-binding activity, as well as expression of p300.  A dominant-neg. mutant of p300 suppressed phenylephrine-induced nuclear acetylation, activation of GATA-4-dependent endothelin-1 promoters, and hypertrophic responses, such as increase in cell size and sarcomere organization.  In sharp contrast to the activation of cardiac MEK-1, which phosphorylates GATA-4 and causes compensated hypertrophy in vivo, p300-mediated acetylation of mouse cardiac nuclear proteins, including GATA-4, results in marked eccentric dilatation and systolic dysfunction.  These findings suggest that p300-mediated nuclear acetylation plays a crit. role in the development of myocyte hypertrophy and represents a pathway that leads to decompensated heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6-ZHbwghCMrVg90H21EOLACvtfcHk0liFtF3XQLeXcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs1yhtrY%253D&md5=ec98d97f4ade96679377d95563ea6075</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.10.3593-3606.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.10.3593-3606.2003%26sid%3Dliteratum%253Aachs%26aulast%3DYanazume%26aufirst%3DT.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DMorimoto%26aufirst%3DT.%26aulast%3DKawamura%26aufirst%3DT.%26aulast%3DWada%26aufirst%3DH.%26aulast%3DMatsumori%26aufirst%3DA.%26aulast%3DKawase%26aufirst%3DY.%26aulast%3DHirai%26aufirst%3DM.%26aulast%3DKita%26aufirst%3DT.%26atitle%3DCardiac%2520p300%2520Is%2520Involved%2520in%2520Myocyte%2520Growth%2520with%2520Decompensated%2520Heart%2520Failure%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D3593%26epage%3D3606%26doi%3D10.1128%2FMCB.23.10.3593-3606.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span> <span> </span><span class="NLM_article-title">SUMO-1 Modification Activated GATA4-Dependent Cardiogenic Gene Activity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">49091</span>– <span class="NLM_lpage">49098</span>, <span class="refDoi"> DOI: 10.1074/jbc.M407494200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M407494200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=49091-49098&author=J.+Wangauthor=X.+H.+Fengauthor=R.+J.+Schwartz&title=SUMO-1+Modification+Activated+GATA4-Dependent+Cardiogenic+Gene+Activity&doi=10.1074%2Fjbc.M407494200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M407494200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M407494200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%2BH.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26atitle%3DSUMO-1%2520Modification%2520Activated%2520GATA4-Dependent%2520Cardiogenic%2520Gene%2520Activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D49091%26epage%3D49098%26doi%3D10.1074%2Fjbc.M407494200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado-Olguin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedantham, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruneau, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Direct Reprogramming of Fibroblasts into Functional Cardiomyocytes by Defined Factors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cell.2010.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=20691899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvFKrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2010&pages=375-386&author=M.+Iedaauthor=J.+D.+Fuauthor=P.+Delgado-Olguinauthor=V.+Vedanthamauthor=Y.+Hayashiauthor=B.+G.+Bruneauauthor=D.+Srivastava&title=Direct+Reprogramming+of+Fibroblasts+into+Functional+Cardiomyocytes+by+Defined+Factors&doi=10.1016%2Fj.cell.2010.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Direct reprogramming of fibroblasts into functional cardiomyocytes by defined factors</span></div><div class="casAuthors">Ieda, Masaki; Fu, Ji-Dong; Delgado-Olguin, Paul; Vedantham, Vasanth; Hayashi, Yohei; Bruneau, Benoit G.; Srivastava, Deepak</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-386</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The reprogramming of fibroblasts to induced pluripotent stem cells (iPSCs) raises the possibility that a somatic cell could be reprogrammed to an alternative differentiated fate without 1st becoming a stem/progenitor cell.  A large pool of fibroblasts exists in the postnatal heart, yet no single "master regulator" of direct cardiac reprogramming has been identified.  Here, the authors report that a combination of 3 developmental transcription factors (i.e., Gata4, Mef2c, and Tbx5) rapidly and efficiently reprogrammed murine postnatal cardiac or dermal fibroblasts directly into differentiated cardiomyocyte-like cells.  The induced cardiomyocytes expressed cardiac-specific markers, had a global gene expression profile similar to cardiomyocytes, and contracted spontaneously.  Fibroblasts transplanted into mouse hearts one day after transduction of the 3 factors also differentiated into cardiomyocyte-like cells.  The authors believe these findings demonstrate that functional cardiomyocytes can be directly reprogrammed from differentiated somatic cells by defined factors.  Reprogramming of endogenous or explanted fibroblasts might provide a source of cardiomyocytes for regenerative approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruiY4LjlUFW7Vg90H21EOLACvtfcHk0ljqJ7BfV1t-Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvFKrsbY%253D&md5=9f3647f4dfc98eb0df6aa28bd2ebdc30</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DIeda%26aufirst%3DM.%26aulast%3DFu%26aufirst%3DJ.%2BD.%26aulast%3DDelgado-Olguin%26aufirst%3DP.%26aulast%3DVedantham%26aufirst%3DV.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DBruneau%26aufirst%3DB.%2BG.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DDirect%2520Reprogramming%2520of%2520Fibroblasts%2520into%2520Functional%2520Cardiomyocytes%2520by%2520Defined%2520Factors%26jtitle%3DCell%26date%3D2010%26volume%3D142%26spage%3D375%26epage%3D386%26doi%3D10.1016%2Fj.cell.2010.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallquist, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neilson, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassel-Duby, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">Heart Repair by Reprogramming Non-Myocytes with Cardiac Transcription Factors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>485</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1038/nature11139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnature11139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=22660318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVyqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=485&publication_year=2012&pages=599-604&author=K.+Songauthor=Y.+J.+Namauthor=X.+Luoauthor=X.+Qiauthor=W.+Tanauthor=G.+N.+Huangauthor=A.+Acharyaauthor=C.+L.+Smithauthor=M.+D.+Tallquistauthor=E.+G.+Neilsonauthor=J.+A.+Hillauthor=R.+Bassel-Dubyauthor=E.+N.+Olson&title=Heart+Repair+by+Reprogramming+Non-Myocytes+with+Cardiac+Transcription+Factors&doi=10.1038%2Fnature11139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Heart repair by reprogramming non-myocytes with cardiac transcription factors</span></div><div class="casAuthors">Song, Kunhua; Nam, Young-Jae; Luo, Xiang; Qi, Xiaoxia; Tan, Wei; Huang, Guo N.; Acharya, Asha; Smith, Christopher L.; Tallquist, Michelle D.; Neilson, Eric G.; Hill, Joseph A.; Bassel-Duby, Rhonda; Olson, Eric N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">485</span>
        (<span class="NLM_cas:issue">7400</span>),
    <span class="NLM_cas:pages">599-604</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The adult mammalian heart possesses little regenerative potential following injury.  Fibrosis due to activation of cardiac fibroblasts impedes cardiac regeneration and contributes to loss of contractile function, pathol. remodelling and susceptibility to arrhythmias.  Cardiac fibroblasts account for a majority of cells in the heart and represent a potential cellular source for restoration of cardiac function following injury through phenotypic reprogramming to a myocardial cell fate.  Four transcription factors, GATA4, HAND2, MEF2C and TBX5, can cooperatively reprogram adult mouse tail-tip and cardiac fibroblasts into beating cardiac-like myocytes in vitro.  Forced expression of these factors in dividing non-cardiomyocytes in mice reprograms these cells into functional cardiac-like myocytes, improves cardiac function and reduces adverse ventricular remodelling following myocardial infarction.  The authors' results suggest a strategy for cardiac repair through reprogramming fibroblasts resident in the heart with cardiogenic transcription factors or other mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrcg4ogoAAbrVg90H21EOLACvtfcHk0ljqJ7BfV1t-Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvVyqsbg%253D&md5=8e1285e15f1274a19d26964291fa9f46</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature11139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11139%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DK.%26aulast%3DNam%26aufirst%3DY.%2BJ.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DQi%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DG.%2BN.%26aulast%3DAcharya%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DC.%2BL.%26aulast%3DTallquist%26aufirst%3DM.%2BD.%26aulast%3DNeilson%26aufirst%3DE.%2BG.%26aulast%3DHill%26aufirst%3DJ.%2BA.%26aulast%3DBassel-Duby%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DHeart%2520Repair%2520by%2520Reprogramming%2520Non-Myocytes%2520with%2520Cardiac%2520Transcription%2520Factors%26jtitle%3DNature%26date%3D2012%26volume%3D485%26spage%3D599%26epage%3D604%26doi%3D10.1038%2Fnature11139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demaria, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yankner, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campisi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span> <span> </span><span class="NLM_article-title">The DNA Damage Response Induces Inflammation and Senescence by Inhibiting Autophagy of GATA4</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>349</i></span>,  <span class="NLM_fpage">aaa5612</span>, <span class="refDoi"> DOI: 10.1126/science.aaa5612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1126%2Fscience.aaa5612" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2015&pages=aaa5612&author=C.+Kangauthor=Q.+Xuauthor=T.+D.+Martinauthor=M.+Z.+Liauthor=M.+Demariaauthor=L.+Aronauthor=T.+Luauthor=B.+A.+Yanknerauthor=J.+Campisiauthor=S.+J.+Elledge&title=The+DNA+Damage+Response+Induces+Inflammation+and+Senescence+by+Inhibiting+Autophagy+of+GATA4&doi=10.1126%2Fscience.aaa5612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa5612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa5612%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMartin%26aufirst%3DT.%2BD.%26aulast%3DLi%26aufirst%3DM.%2BZ.%26aulast%3DDemaria%26aufirst%3DM.%26aulast%3DAron%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DYankner%26aufirst%3DB.%2BA.%26aulast%3DCampisi%26aufirst%3DJ.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520DNA%2520Damage%2520Response%2520Induces%2520Inflammation%2520and%2520Senescence%2520by%2520Inhibiting%2520Autophagy%2520of%2520GATA4%26jtitle%3DScience%26date%3D2015%26volume%3D349%26spage%3Daaa5612%26doi%3D10.1126%2Fscience.aaa5612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Hachem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, G.</span></span> <span> </span><span class="NLM_article-title">Identification of New GATA4-Small Molecule Inhibitors by Structure-Based Virtual Screening</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1734</span>– <span class="NLM_lpage">1742</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.bmc.2011.01.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1734-1742&author=N.+El-Hachemauthor=G.+Nemer&title=Identification+of+New+GATA4-Small+Molecule+Inhibitors+by+Structure-Based+Virtual+Screening&doi=10.1016%2Fj.bmc.2011.01.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Hachem%26aufirst%3DN.%26aulast%3DNemer%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520New%2520GATA4-Small%2520Molecule%2520Inhibitors%2520by%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D1734%26epage%3D1742%26doi%3D10.1016%2Fj.bmc.2011.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">Combinatorial Interactions Regulating Cardiac Transcription</span>. <i>Dev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1520-6408(1998)22:3<250::AID-DVG7>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=9621432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK1cXjvVCit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1998&pages=250-262&author=D.+Durocherauthor=M.+Nemer&title=Combinatorial+Interactions+Regulating+Cardiac+Transcription&doi=10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial interactions regulating cardiac transcription</span></div><div class="casAuthors">Durocher, Daniel; Nemer, Mona</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Genetics (New York)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">250-262</span>CODEN:
                <span class="NLM_cas:coden">DGNTDW</span>;
        ISSN:<span class="NLM_cas:issn">0192-253X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">In vertebrates, heart development is a multistep process that starts with formation and patterning of the primitive heart tube and is followed by complex morphol. events to give rise to the mature four-chambered heart.  These various stages are characterized by distinct patterns of gene expression.  Although chamber specificity and developmental regulation can be demonstrated in transgenic mice using short promoter fragments, the mechanism underlying spatial and temporal specificity within the heart remains largely unclear.  Combinatorial interaction between a limited no. of cardiac-specific and ubiquitous transcription factors may account for the diverse genetic inputs required to generate the complex transcriptional patterns that characterize the developing myocardium.  We have used the cardiac atrial natriuretic peptide (ANP) promoter to test this hypothesis.  The ANP gene is transcribed in a spatial-and temporal-specific manner in the heart, and a 500 bp promoter fragment is sufficient to recapitulate both chamber and developmental specificity.  This promoter is composed of three modules, a "basal" cardiac promoter that is essential for transcription in embryonic and postnatal atrial and ventricular myocytes and two other independent modules that behave as chamber-specific enhancers.  The basal cardiac promoter is the target of two cardiac-specific transcription factors, the zinc finger GATA-4 protein and the Nkx2-5 homeodomain, which bind to contiguous elements within this region.  At low concns., a situation that likely occurs during the very first stages of cardiac cell fate detn., the two proteins synergistically activate transcription from the ANP promoter.  This functional synergy requires phys. interaction between the GATA-4 protein and an extended C-terminal homeodomain on Nkx2-5.  This interaction, which unmasks an activation domain present just N-terminal of the homeodomain, is specific for GATA-4 and -5, but is not obsd. with the other cardiac GATA factor, GATA-6.  Optimal synergy requires binding of both proteins to their cognate sites, although modest synergy also could be obsd. on heterologous promoters contg. only multimerized Nkx binding sites, suggesting that Nkx2-5 is able to recruit GATA-4 into a transcriptionally active complex.  The GATA/Nkx interaction, which appears to have been evolutionary conserved in nematode, fly, and mammals, provides a paradigm for analyzing transcription factor interaction during organogenesis.  The data are also discussed in the context of our present knowledge of the roles of GATA and NK2 proteins in cardiac development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgaAVlHUEwzbVg90H21EOLACvtfcHk0liYlXycnJ5Vew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvVCit7g%253D&md5=09c172bc8995035f548288ee0baae7d2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291520-6408%25281998%252922%253A3%253C250%253A%253AAID-DVG7%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DCombinatorial%2520Interactions%2520Regulating%2520Cardiac%2520Transcription%26jtitle%3DDev.%2520Genet.%26date%3D1998%26volume%3D22%26spage%3D250%26epage%3D262%26doi%3D10.1002%2F%28SICI%291520-6408%281998%2922%3A3%3C250%3A%3AAID-DVG7%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sepulveda, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belaguli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span> <span> </span><span class="NLM_article-title">GATA-4 and Nkx-2.5 Coactivate Nkx-2 DNA Binding Targets: Role for Regulating Early Cardiac Gene Expression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3405</span>– <span class="NLM_lpage">3415</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.6.3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.18.6.3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=9584181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK1cXjsFSrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=3405-3415&author=J.+L.+Sepulvedaauthor=N.+Belaguliauthor=V.+Nigamauthor=C.+Y.+Chenauthor=M.+Nemerauthor=R.+J.+Schwartz&title=GATA-4+and+Nkx-2.5+Coactivate+Nkx-2+DNA+Binding+Targets%3A+Role+for+Regulating+Early+Cardiac+Gene+Expression&doi=10.1128%2FMCB.18.6.3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GATA-4 and Nkx-2.5 coactivate Nkx-2 DNA binding targets: role for regulating early cardiac gene expression</span></div><div class="casAuthors">Sepulveda, Jorge L.; Belaguli, Narashimaswamy; Nigam, Vishal; Chen, Ching-Yi; Nemer, Mona; Schwartz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3405-3415</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The cardiogenic homeodomain factor Nkx-2.5 and serum response factor (SRF) provide strong transcriptional coactivation of the cardiac α-actin (αCA) promoter in fibroblasts (C. Y. Chen and R. J. Schwartz, Mol. Cell. Biol. 16:6372-6384, 1996).  We demonstrate here that Nkx-2.5 also cooperates with GATA-4, a dual C-4 zinc finger transcription factor expressed in early cardiac progenitor cells, to activate the αCA promoter and a minimal promoter, contg. only multimerized Nkx-2.5 DNA bindings sites (NKEs), in heterologous CV-1 fibroblasts.  Transcriptional activity requires the N-terminal activation domain of Nkx-2.5 and Nkx-2.5 binding activity through its homeodomain but does not require GATA-4's activation domain.  The minimal interactive regions were mapped to the homeodomain of Nkx-2.5 and the second zinc finger of GATA-4.  Removal of Nkx-2.5's C-terminal inhibitory domain stimulated robust transcriptional activity, comparable to the effects of GATA-4 on wild-type Nkx-2.5, which in part facilitated Nkx-2.5 DNA binding activity.  We postulate the following simple model: GATA-4 induces a conformational change in Nkx-2.5 that displaces the C-terminal inhibitory domain, thus eliciting transcriptional activation of promoters contg. Nkx-2.5 DNA binding targets.  Therefore, αCa promoter activity appears to be regulated through the combinatorial interactions of at least three cardiac tissue-enriched transcription factors, Nkx-2.5, GATA-4, and SRF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQzwtdzWGPbLVg90H21EOLACvtfcHk0liYlXycnJ5Vew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjsFSrsrc%253D&md5=c64eb458510a7c00ad89088a7bd81172</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.6.3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.6.3405%26sid%3Dliteratum%253Aachs%26aulast%3DSepulveda%26aufirst%3DJ.%2BL.%26aulast%3DBelaguli%26aufirst%3DN.%26aulast%3DNigam%26aufirst%3DV.%26aulast%3DChen%26aufirst%3DC.%2BY.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26atitle%3DGATA-4%2520and%2520Nkx-2.5%2520Coactivate%2520Nkx-2%2520DNA%2520Binding%2520Targets%253A%2520Role%2520for%2520Regulating%2520Early%2520Cardiac%2520Gene%2520Expression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D3405%26epage%3D3415%26doi%3D10.1128%2FMCB.18.6.3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span>; <span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Interferences with Luciferase Reporter Enzymes</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, G.</span>, <span class="NLM_string-name">Coussens, N.</span>, <span class="NLM_string-name">Brimacombe, K.</span>, <span class="NLM_string-name">Grossman, A.</span>, <span class="NLM_string-name">Arkin, M.</span>, <span class="NLM_string-name">Auld, D.</span>, <span class="NLM_string-name">Austin, C.</span>, <span class="NLM_string-name">Baell, J.</span>, <span class="NLM_string-name">Bejcek, B.</span>, <span class="NLM_string-name">Chung, T.</span>, <span class="NLM_string-name">Dahlin, J.</span>, <span class="NLM_string-name">Devanaryan, V.</span>, <span class="NLM_string-name">Foley, T.</span>, <span class="NLM_string-name">Glicksman, M.</span>, <span class="NLM_string-name">Hall, M.</span>, <span class="NLM_string-name">Hass, J.</span>, <span class="NLM_string-name">Inglese, J.</span>, <span class="NLM_string-name">Iversen, P.</span>, <span class="NLM_string-name">Kahl, S.</span>, <span class="NLM_string-name">Kales, S.</span>, <span class="NLM_string-name">Lal-Nag, M.</span>, <span class="NLM_string-name">Li, Z.</span>, <span class="NLM_string-name">McGee, J.</span>, <span class="NLM_string-name">McManus, O.</span>, <span class="NLM_string-name">Xu, X.</span></span>, Eds.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2016</span>; Vol.  <span class="NLM_volume">1</span>, pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <a href="http://www.ncbi.nlm.nih.gov/books/NBK374281/" class="extLink">http://www.ncbi.nlm.nih.gov/books/NBK374281/</a> (accessed August 10, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1-14&author=D.+S.+Auld&author=J.+Ingleseauthor=G.+Sittampalam&author=N.+Coussens&author=K.+Brimacombe&author=A.+Grossman&author=M.+Arkin&author=D.+Auld&author=C.+Austin&author=J.+Baell&author=B.+Bejcek&author=T.+Chung&author=J.+Dahlin&author=V.+Devanaryan&author=T.+Foley&author=M.+Glicksman&author=M.+Hall&author=J.+Hass&author=J.+Inglese&author=P.+Iversen&author=S.+Kahl&author=S.+Kales&author=M.+Lal-Nag&author=Z.+Li&author=J.+McGee&author=O.+McManus&author=X.+Xu&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAuld%26aufirst%3DD.%2BS.%26atitle%3DInterferences%2520with%2520Luciferase%2520Reporter%2520Enzymes%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DG.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%26date%3D2016%26volume%3D1%26spage%3D1%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, M.</span></span> <span> </span><span class="NLM_article-title">Functionality Maps of Binding Sites: A Multiple Copy Simultaneous Search Method</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1002/prot.340110104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2Fprot.340110104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1961699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=29-34&author=A.+Mirankerauthor=M.+Karplus&title=Functionality+Maps+of+Binding+Sites%3A+A+Multiple+Copy+Simultaneous+Search+Method&doi=10.1002%2Fprot.340110104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Functionality maps of binding sites:  a multiple copy simultaneous search method</span></div><div class="casAuthors">Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-34</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">A new method is proposed for detg. energetically favorable positions and orientations for functional groups on the surface of proteins with known 3-dimensional structure.  Some 1000-5000 copies of a functional group are randomly placed in the site and subjected to simultaneous energy minimization and/or quenched mol. dynamics.  The resulting functionality maps of a protein receptor site, which can take account of its flexibility, can be used for the anal. of protein ligand interactions and rational drug design.  Application of the method to the sialic acid binding site of the influenza coat protein, hemagglutinin, yields functional group min. that correspond with those of the ligand in a cocrystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5im8BqJfF37Vg90H21EOLACvtfcHk0ljlWmNEAMAweQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D&md5=34e0a0c36899bfeabcbdfd4790f97a1e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fprot.340110104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340110104%26sid%3Dliteratum%253Aachs%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DFunctionality%2520Maps%2520of%2520Binding%2520Sites%253A%2520A%2520Multiple%2520Copy%2520Simultaneous%2520Search%2520Method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1991%26volume%3D11%26spage%3D29%26epage%3D34%26doi%3D10.1002%2Fprot.340110104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, S.</span>; <span class="NLM_string-name">Tölli, M.</span>; <span class="NLM_string-name">Välimäki, M.</span>; <span class="NLM_string-name">Jumppanen, M.</span>; <span class="NLM_string-name">Boije af Gennäs, G.</span>; <span class="NLM_string-name">Yli-Kauhaluoma, J.</span>; <span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical Compounds</span>. Patent Application 20165712, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=S.+Kinnunen&author=M.+T%C3%B6lli&author=M.+V%C3%A4lim%C3%A4ki&author=M.+Jumppanen&author=G.+Boije+af+Genn%C3%A4s&author=J.+Yli-Kauhaluoma&author=H.+Ruskoaho&title=Pharmaceutical+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKinnunen%26aufirst%3DS.%26atitle%3DPharmaceutical%2520Compounds%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shioi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasahara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jobe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiese, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumo, S.</span></span> <span> </span><span class="NLM_article-title">The Cardiac Tissue-Restricted Homeobox Protein Csx/Nkx2.5 Physically Associates with the Zinc Finger Protein GATA4 and Cooperatively Activates Atrial Natriuretic Factor Gene Expression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3120</span>– <span class="NLM_lpage">3129</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.6.3120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.18.6.3120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=9584153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK1cXjsFSkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=3120-3129&author=Y.+Leeauthor=T.+Shioiauthor=H.+Kasaharaauthor=S.+M.+Jobeauthor=R.+J.+Wieseauthor=B.+E.+Markhamauthor=S.+Izumo&title=The+Cardiac+Tissue-Restricted+Homeobox+Protein+Csx%2FNkx2.5+Physically+Associates+with+the+Zinc+Finger+Protein+GATA4+and+Cooperatively+Activates+Atrial+Natriuretic+Factor+Gene+Expression&doi=10.1128%2FMCB.18.6.3120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The cardiac tissue-restricted homeobox protein Csx/Nkx2.5 physically associates with the zinc finger protein GATA4 and cooperatively activates atrial natriuretic factor gene expression</span></div><div class="casAuthors">Lee, Youngsook; Shioi, Tetsuo; Kasahara, Hideko; Jobe, Shawn M.; Wiese, Russell J.; Markham, Bruce E.; Izumo, Seigo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3120-3129</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Specification and differentiation of the cardiac muscle lineage appear to require a combinatorial network of many factors.  The cardiac muscle-restricted homeobox protein Csx/Nkx2.5 (Csx) is expressed in the precardiac mesoderm as well as the embryonic and adult heart.  Targeted disruption of Csx causes embryonic lethality due to abnormal heart morphogenesis.  The zinc finger transcription factor GATA4 is also expressed in the heart and has been shown to be essential for heart tube formation.  GATA4 is known to activate many cardiac tissue-restricted genes.  In this study, the authors tested whether Csx and GATA4 phys. assoc. and cooperatively activate transcription of a target gene.  Coimmunopptn. expts. demonstrate that Csx and GATA4 assoc. intracellularly.  Interestingly, in vitro protein-protein interaction studies indicate that helix III of the homeodomain of Csx is required to interact with GATA4 and that the carboxy-terminal zinc finger of GATA4 is necessary to assoc. with Csx.  Both regions are known to directly contact the cognate DNA sequences.  The promoter-enhancer region of the atrial natriuretic factor (ANF) contains several putative Csx binding sites and consensus GATA4 binding sites.  Transient-transfection assays indicate that Csx can activate ANF reporter gene expression to the same extent that GATA4 does in a DNA binding site-dependent manner.  Coexpression of Csx and GATA4 synergistically activates ANF reporter gene expression.  Mutational analyses suggest that this synergy requires both factors to fully retain their transcriptional activities, including the cofactor binding activity.  These results demonstrate the first example of homeoprotein and zinc finger protein interaction in vertebrates to cooperatively regulate target gene expression.  Such synergistic interaction among tissue-restricted transcription factors may be an important mechanism to reinforce tissue-specific developmental pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGSPRokpTxqbVg90H21EOLACvtfcHk0ljlWmNEAMAweQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjsFSkur8%253D&md5=dbf87dd868e0988f72ceb04834e8a50c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.6.3120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.6.3120%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%26aulast%3DShioi%26aufirst%3DT.%26aulast%3DKasahara%26aufirst%3DH.%26aulast%3DJobe%26aufirst%3DS.%2BM.%26aulast%3DWiese%26aufirst%3DR.%2BJ.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26aulast%3DIzumo%26aufirst%3DS.%26atitle%3DThe%2520Cardiac%2520Tissue-Restricted%2520Homeobox%2520Protein%2520Csx%252FNkx2.5%2520Physically%2520Associates%2520with%2520the%2520Zinc%2520Finger%2520Protein%2520GATA4%2520and%2520Cooperatively%2520Activates%2520Atrial%2520Natriuretic%2520Factor%2520Gene%2520Expression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D3120%26epage%3D3129%26doi%3D10.1128%2FMCB.18.6.3120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">The Cardiac Transcription Factors Nkx2-5 and GATA-4 Are Mutual Cofactors</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">5687</span>– <span class="NLM_lpage">5696</span>, <span class="refDoi"> DOI: 10.1093/emboj/16.18.5687</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1093%2Femboj%2F16.18.5687" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=5687-5696&author=D.+Durocherauthor=F.+Charronauthor=R.+Warrenauthor=R.+J.+Schwartzauthor=M.+Nemer&title=The+Cardiac+Transcription+Factors+Nkx2-5+and+GATA-4+Are+Mutual+Cofactors&doi=10.1093%2Femboj%2F16.18.5687"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F16.18.5687&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F16.18.5687%26sid%3Dliteratum%253Aachs%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DWarren%26aufirst%3DR.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DThe%2520Cardiac%2520Transcription%2520Factors%2520Nkx2-5%2520and%2520GATA-4%2520Are%2520Mutual%2520Cofactors%26jtitle%3DEMBO%2520J.%26date%3D1997%26volume%3D16%26spage%3D5687%26epage%3D5696%26doi%3D10.1093%2Femboj%2F16.18.5687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuolteenaho, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uusimaa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Passive Mechanical Stretch Releases Atrial Natriuretic Peptide from Rat Ventricular Myocardium</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1253</span>, <span class="refDoi"> DOI: 10.1161/01.RES.70.6.1244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2F01.RES.70.6.1244" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1992&pages=1244-1253&author=P.+Kinnunenauthor=O.+Vuolteenahoauthor=P.+Uusimaaauthor=H.+Ruskoaho&title=Passive+Mechanical+Stretch+Releases+Atrial+Natriuretic+Peptide+from+Rat+Ventricular+Myocardium&doi=10.1161%2F01.RES.70.6.1244"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1161%2F01.RES.70.6.1244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.RES.70.6.1244%26sid%3Dliteratum%253Aachs%26aulast%3DKinnunen%26aufirst%3DP.%26aulast%3DVuolteenaho%26aufirst%3DO.%26aulast%3DUusimaa%26aufirst%3DP.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DPassive%2520Mechanical%2520Stretch%2520Releases%2520Atrial%2520Natriuretic%2520Peptide%2520from%2520Rat%2520Ventricular%2520Myocardium%26jtitle%3DCirc.%2520Res.%26date%3D1992%26volume%3D70%26spage%3D1244%26epage%3D1253%26doi%3D10.1161%2F01.RES.70.6.1244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaPointe, M. C.</span></span> <span> </span><span class="NLM_article-title">Molecular Regulation of the Brain Natriuretic Peptide Gene</span>. <i>Peptides</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">944</span>– <span class="NLM_lpage">956</span>, <span class="refDoi"> DOI: 10.1016/j.peptides.2004.08.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.peptides.2004.08.028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=944-956&author=M.+C.+LaPointe&title=Molecular+Regulation+of+the+Brain+Natriuretic+Peptide+Gene&doi=10.1016%2Fj.peptides.2004.08.028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.peptides.2004.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.peptides.2004.08.028%26sid%3Dliteratum%253Aachs%26aulast%3DLaPointe%26aufirst%3DM.%2BC.%26atitle%3DMolecular%2520Regulation%2520of%2520the%2520Brain%2520Natriuretic%2520Peptide%2520Gene%26jtitle%3DPeptides%26date%3D2005%26volume%3D26%26spage%3D944%26epage%3D956%26doi%3D10.1016%2Fj.peptides.2004.08.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimasa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, K.</span></span> <span> </span><span class="NLM_article-title">Rapid Transcriptional Activation and Early mRNA Turnover of Brain Natriuretic Peptide in Cardiocyte Hypertrophy. Evidence for Brain Natriuretic Peptide as an “Emergency” Cardiac Hormone against Ventricular Overload</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">1280</span>– <span class="NLM_lpage">1287</span>, <span class="refDoi"> DOI: 10.1172/JCI118162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1172%2FJCI118162" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1995&pages=1280-1287&author=O.+Nakagawaauthor=Y.+Ogawaauthor=H.+Itohauthor=S.+Sugaauthor=Y.+Komatsuauthor=I.+Kishimotoauthor=K.+Nishinoauthor=T.+Yoshimasaauthor=K.+Nakao&title=Rapid+Transcriptional+Activation+and+Early+mRNA+Turnover+of+Brain+Natriuretic+Peptide+in+Cardiocyte+Hypertrophy.+Evidence+for+Brain+Natriuretic+Peptide+as+an+%E2%80%9CEmergency%E2%80%9D+Cardiac+Hormone+against+Ventricular+Overload&doi=10.1172%2FJCI118162"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1172%2FJCI118162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI118162%26sid%3Dliteratum%253Aachs%26aulast%3DNakagawa%26aufirst%3DO.%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DH.%26aulast%3DSuga%26aufirst%3DS.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DKishimoto%26aufirst%3DI.%26aulast%3DNishino%26aufirst%3DK.%26aulast%3DYoshimasa%26aufirst%3DT.%26aulast%3DNakao%26aufirst%3DK.%26atitle%3DRapid%2520Transcriptional%2520Activation%2520and%2520Early%2520mRNA%2520Turnover%2520of%2520Brain%2520Natriuretic%2520Peptide%2520in%2520Cardiocyte%2520Hypertrophy.%2520Evidence%2520for%2520Brain%2520Natriuretic%2520Peptide%2520as%2520an%2520%25E2%2580%259CEmergency%25E2%2580%259D%2520Cardiac%2520Hormone%2520against%2520Ventricular%2520Overload%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1995%26volume%3D96%26spage%3D1280%26epage%3D1287%26doi%3D10.1172%2FJCI118162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serby, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falls, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaszubska, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colins, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Structure–activity Relationships of Isoxazole Carboxamides as Growth Hormone Secretagogue Receptor Antagonists</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1201</span>– <span class="NLM_lpage">1204</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2004.11.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.bmcl.2004.11.075" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1201-1204&author=Z.+Xinauthor=H.+Zhaoauthor=M.+D.+Serbyauthor=B.+Liuauthor=V.+G.+Schaeferauthor=D.+H.+Fallsauthor=W.+Kaszubskaauthor=C.+A.+Colinsauthor=H.+L.+Shamauthor=G.+Liu&title=Synthesis+and+Structure%E2%80%93activity+Relationships+of+Isoxazole+Carboxamides+as+Growth+Hormone+Secretagogue+Receptor+Antagonists&doi=10.1016%2Fj.bmcl.2004.11.075"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.11.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.11.075%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DSerby%26aufirst%3DM.%2BD.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSchaefer%26aufirst%3DV.%2BG.%26aulast%3DFalls%26aufirst%3DD.%2BH.%26aulast%3DKaszubska%26aufirst%3DW.%26aulast%3DColins%26aufirst%3DC.%2BA.%26aulast%3DSham%26aufirst%3DH.%2BL.%26aulast%3DLiu%26aufirst%3DG.%26atitle%3DSynthesis%2520and%2520Structure%25E2%2580%2593activity%2520Relationships%2520of%2520Isoxazole%2520Carboxamides%2520as%2520Growth%2520Hormone%2520Secretagogue%2520Receptor%2520Antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D1201%26epage%3D1204%26doi%3D10.1016%2Fj.bmcl.2004.11.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Omichinski, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaad, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsenfeld, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appella, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronenborn, A. M.</span></span> <span> </span><span class="NLM_article-title">NMR Structure of a Specific DNA Complex of Zn-Containing DNA Binding Domain of GATA-1</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1126/science.8332909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1126%2Fscience.8332909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8332909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK3sXltlartrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1993&pages=438-446&author=J.+G.+Omichinskiauthor=G.+M.+Cloreauthor=O.+Schaadauthor=G.+Felsenfeldauthor=C.+Trainorauthor=E.+Appellaauthor=S.+J.+Stahlauthor=A.+M.+Gronenborn&title=NMR+Structure+of+a+Specific+DNA+Complex+of+Zn-Containing+DNA+Binding+Domain+of+GATA-1&doi=10.1126%2Fscience.8332909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">NMR structure of a specific DNA complex of zinc-containing DNA binding domain of GATA-1</span></div><div class="casAuthors">Omichinski, James G.; Clore, G. Marius; Schaad, Olivier; Felsenfeld, Gary; Trainor, Cecelia; Appella, Ettore; Stahl, Stephen J.; Gronenborn, Angela M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">5120</span>),
    <span class="NLM_cas:pages">438-46</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The three-dimensional soln. structure of a complex between the DNA binding domain of the chicken erythroid transcription factor GATA-1 and its cognate DNA site has been detd. with multidimensional heteronuclear magnetic resonance spectroscopy.  The DNA binding domain consists of a core which contains a zinc coordinated by four cysteines and a carboxyl-terminal tail.  The core is composed of two irregular antiparallel β sheets and an α helix, followed by a long loop that leads into the carboxyl-terminal tail.  The amino-terminal part of the core including the helix, is similar in structure, although not in sequence, to the amino-terminal zinc module of the glucocorticoid receptor DNA binding domain.  In the other regions, the structures of these two DNA binding domains are entirely different.  The DNA target site in contact with the protein spans eight base pairs.  The helix and the loop connecting the two antiparallel β sheets interact with the major groove of the DNA.  The carboxyl-terminal tail, which is an essential determinant of specific binding, wraps around into the minor groove.  The complex resembles a hand holding a rope with the palm and fingers representing the protein core and the thumb, the carboxyl-terminal tail.  The specific interactions between GATA-1 and DNA in the major groove are mainly hydrophobic in nature, which accounts for the preponderance of thymines in the target site.  A large no. of interactions are obsd. with the phosphate backbone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol5cF60Zbls7Vg90H21EOLACvtfcHk0ljlGWgpF35bOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltlartrs%253D&md5=ac5de6179e0f29eff55075d563af58be</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscience.8332909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8332909%26sid%3Dliteratum%253Aachs%26aulast%3DOmichinski%26aufirst%3DJ.%2BG.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DSchaad%26aufirst%3DO.%26aulast%3DFelsenfeld%26aufirst%3DG.%26aulast%3DTrainor%26aufirst%3DC.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DStahl%26aufirst%3DS.%2BJ.%26aulast%3DGronenborn%26aufirst%3DA.%2BM.%26atitle%3DNMR%2520Structure%2520of%2520a%2520Specific%2520DNA%2520Complex%2520of%2520Zn-Containing%2520DNA%2520Binding%2520Domain%2520of%2520GATA-1%26jtitle%3DScience%26date%3D1993%26volume%3D261%26spage%3D438%26epage%3D446%26doi%3D10.1126%2Fscience.8332909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katanasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunagawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Cardiac Transcription Factor GATA4 by Post-Translational Modification in Cardiomyocyte Hypertrophy and Heart Failure</span>. <i>Int. Heart J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1536/ihj.16-404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1536%2Fihj.16-404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=27818483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsVyju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2016&pages=672-675&author=Y.+Katanasakaauthor=H.+Suzukiauthor=Y.+Sunagawaauthor=K.+Hasegawaauthor=T.+Morimoto&title=Regulation+of+Cardiac+Transcription+Factor+GATA4+by+Post-Translational+Modification+in+Cardiomyocyte+Hypertrophy+and+Heart+Failure&doi=10.1536%2Fihj.16-404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cardiac transcription factor GATA4 by post-translational modification in cardiomyocyte hypertrophy and heart failure</span></div><div class="casAuthors">Katanasaka, Yasufumi; Suzuki, Hidetoshi; Sunagawa, Yoichi; Hasegawa, Koji; Morimoto, Tatsuya</div><div class="citationInfo"><span class="NLM_cas:title">International Heart Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">672-675</span>CODEN:
                <span class="NLM_cas:coden">IHJNAJ</span>;
        ISSN:<span class="NLM_cas:issn">1349-2365</span>.
    
            (<span class="NLM_cas:orgname">International Heart Journal Association</span>)
        </div><div class="casAbstract">Heart failure is a leading cause of cardiovascular mortality in industrialized countries.  During development and deterioration of heart failure, cardiomyocytes undergo maladaptive hypertrophy, and changes in the cellular phenotype are accompanied by reinduction of the fetal gene program.  Gene expression in cardiomyocytes is regulated by various nuclear transcription factors, co-activators, and co-repressors.  The zinc finger protein GATA4 is one such transcription factor involved in the regulation of cardiomyocyte hypertrophy.  In response to hypertrophic stimuli such as those involving the sympathetic nervous and renin-angiotensin systems, changes in protein interaction and/or post-translational modifications of GATA4 cause hypertrophic gene transcription in cardiomyocytes.  In this article, we focus on cardiac nuclear signaling mols., esp. GATA4, that are promising as potential targets for heart failure therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp09rLSa4euy7Vg90H21EOLACvtfcHk0ljlGWgpF35bOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsVyju7Y%253D&md5=994dbecc38c3d59cb4432f0d59b558dc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1536%2Fihj.16-404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1536%252Fihj.16-404%26sid%3Dliteratum%253Aachs%26aulast%3DKatanasaka%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DSunagawa%26aufirst%3DY.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DMorimoto%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Cardiac%2520Transcription%2520Factor%2520GATA4%2520by%2520Post-Translational%2520Modification%2520in%2520Cardiomyocyte%2520Hypertrophy%2520and%2520Heart%2520Failure%26jtitle%3DInt.%2520Heart%2520J.%26date%3D2016%26volume%3D57%26spage%3D672%26epage%3D675%26doi%3D10.1536%2Fihj.16-404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Regulation and Its Misregulation in Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1251</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cell.2013.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23498934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2013&pages=1237-1251&author=T.+I.+Leeauthor=R.+A.+Young&title=Transcriptional+Regulation+and+Its+Misregulation+in+Disease&doi=10.1016%2Fj.cell.2013.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation and its misregulation in disease</span></div><div class="casAuthors">Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1237-1251</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The gene expression programs that establish and maintain specific cell states in humans are controlled by thousands of transcription factors, cofactors, and chromatin regulators.  Misregulation of these gene expression programs can cause a broad range of diseases.  Here, we review recent advances in our understanding of transcriptional regulation and discuss how these have provided new insights into transcriptional misregulation in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRZI_3gsWJi7Vg90H21EOLACvtfcHk0ljf-Abdm2hwdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D&md5=2eabd761a3721c8c509f0fa1a00ceab6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Regulation%2520and%2520Its%2520Misregulation%2520in%2520Disease%26jtitle%3DCell%26date%3D2013%26volume%3D152%26spage%3D1237%26epage%3D1251%26doi%3D10.1016%2Fj.cell.2013.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuchner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausserlechner, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Factors by Small Compounds--Current Strategies and Future Implications</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2015.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.bcp.2015.12.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1-13&author=J.+Hagenbuchnerauthor=M.+J.+Ausserlechner&title=Targeting+Transcription+Factors+by+Small+Compounds%2D%2DCurrent+Strategies+and+Future+Implications&doi=10.1016%2Fj.bcp.2015.12.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuchner%26aufirst%3DJ.%26aulast%3DAusserlechner%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520Transcription%2520Factors%2520by%2520Small%2520Compounds--Current%2520Strategies%2520and%2520Future%2520Implications%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2016%26volume%3D107%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.bcp.2015.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Addis, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ifkovits, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellam, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rentschler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christoforou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epstein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gearhart, J. D.</span></span> <span> </span><span class="NLM_article-title">Optimization of Direct Fibroblast Reprogramming to Cardiomyocytes Using Calcium Activity as a Functional Measure of Success</span>. <i>J. Mol. Cell. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.yjmcc.2013.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23591016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptVyqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2013&pages=97-106&author=R.+C.+Addisauthor=J.+L.+Ifkovitsauthor=F.+Pintoauthor=L.+D.+Kellamauthor=P.+Estesoauthor=S.+Rentschlerauthor=N.+Christoforouauthor=J.+A.+Epsteinauthor=J.+D.+Gearhart&title=Optimization+of+Direct+Fibroblast+Reprogramming+to+Cardiomyocytes+Using+Calcium+Activity+as+a+Functional+Measure+of+Success&doi=10.1016%2Fj.yjmcc.2013.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of direct fibroblast reprogramming to cardiomyocytes using calcium activity as a functional measure of success</span></div><div class="casAuthors">Addis, Russell C.; Ifkovits, Jamie L.; Pinto, Filipa; Kellam, Lori D.; Esteso, Paul; Rentschler, Stacey; Christoforou, Nicolas; Epstein, Jonathan A.; Gearhart, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97-106</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Direct conversion of fibroblasts to induced cardiomyocytes (iCMs) has great potential for regenerative medicine.  Recent publications have reported significant progress, but the evaluation of reprogramming has relied upon non-functional measures such as flow cytometry for cardiomyocyte markers or GFP expression driven by a cardiomyocyte-specific promoter.  The issue is one of practicality: the most stringent measures - electrophysiol. to detect cell excitation and the presence of spontaneously contracting myocytes - are not readily quantifiable in the large nos. of cells screened in reprogramming expts.  However, excitation and contraction are linked by a third functional characteristic of cardiomyocytes: the rhythmic oscillation of intracellular calcium levels.  We set out to optimize direct conversion of fibroblasts to iCMs with a quantifiable calcium reporter to rapidly assess functional transdifferentiation.  We constructed a reporter system in which the calcium indicator GCaMP is driven by the cardiomyocyte-specific Troponin T promoter.  Using calcium activity as our primary outcome measure, we compared several published combinations of transcription factors along with novel combinations in mouse embryonic fibroblasts.  The most effective combination consisted of Hand2, Nkx2.5, Gata4, Mef2c, and Tbx5 (HNGMT).  This combination is > 50-fold more efficient than GMT alone and produces iCMs with cardiomyocyte marker expression, robust calcium oscillation, and spontaneous beating that persist for weeks following inactivation of reprogramming factors.  HNGMT is also significantly more effective than previously published factor combinations for the transdifferentiation of adult mouse cardiac fibroblasts to iCMs.  Quantification of calcium function is a convenient and effective means for the identification and evaluation of cardiomyocytes generated by direct reprogramming.  Using this stringent outcome measure, we conclude that HNGMT produces iCMs more efficiently than previously published methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou3fsRrjAOgrVg90H21EOLACvtfcHk0ljf-Abdm2hwdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptVyqtLg%253D&md5=5b99b6dd16a2f234d38dc3c5131e1d9b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DAddis%26aufirst%3DR.%2BC.%26aulast%3DIfkovits%26aufirst%3DJ.%2BL.%26aulast%3DPinto%26aufirst%3DF.%26aulast%3DKellam%26aufirst%3DL.%2BD.%26aulast%3DEsteso%26aufirst%3DP.%26aulast%3DRentschler%26aufirst%3DS.%26aulast%3DChristoforou%26aufirst%3DN.%26aulast%3DEpstein%26aufirst%3DJ.%2BA.%26aulast%3DGearhart%26aufirst%3DJ.%2BD.%26atitle%3DOptimization%2520of%2520Direct%2520Fibroblast%2520Reprogramming%2520to%2520Cardiomyocytes%2520Using%2520Calcium%2520Activity%2520as%2520a%2520Functional%2520Measure%2520of%2520Success%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D60%26spage%3D97%26epage%3D106%26doi%3D10.1016%2Fj.yjmcc.2013.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, M.</span></span> <span> </span><span class="NLM_article-title">Critical Factors for Cardiac Reprogramming</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.112.271452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2FCIRCRESAHA.112.271452" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2012&pages=5-8&author=D.+Srivastavaauthor=M.+Ieda&title=Critical+Factors+for+Cardiac+Reprogramming&doi=10.1161%2FCIRCRESAHA.112.271452"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.112.271452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.112.271452%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DD.%26aulast%3DIeda%26aufirst%3DM.%26atitle%3DCritical%2520Factors%2520for%2520Cardiac%2520Reprogramming%26jtitle%3DCirc.%2520Res.%26date%3D2012%26volume%3D111%26spage%3D5%26epage%3D8%26doi%3D10.1161%2FCIRCRESAHA.112.271452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassel-Duby, R.</span></span> <span> </span><span class="NLM_article-title">Mending Broken Hearts: Cardiac Development as a Basis for Adult Heart Regeneration and Repair</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nrm3619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnrm3619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23839576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOrtL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=529-541&author=M.+Xinauthor=E.+N.+Olsonauthor=R.+Bassel-Duby&title=Mending+Broken+Hearts%3A+Cardiac+Development+as+a+Basis+for+Adult+Heart+Regeneration+and+Repair&doi=10.1038%2Fnrm3619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Mending broken hearts: cardiac development as a basis for adult heart regeneration and repair</span></div><div class="casAuthors">Xin, Mei; Olson, Eric N.; Bassel-Duby, Rhonda</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">529-541</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  As the adult mammalian heart has limited potential for regeneration and repair, the loss of cardiomyocytes during injury and disease can result in heart failure and death.  The cellular processes and regulatory mechanisms involved in heart growth and development can be exploited to repair the injured adult heart through 'reawakening' pathways that are active during embryogenesis.  Heart function has been restored in rodents by reprogramming non-myocytes into cardiomyocytes, by expressing transcription factors (GATA4, HAND2, myocyte-specific enhancer factor 2C (MEF2C) and T-box 5 (TBX5)) and microRNAs (miR-1, miR-133, miR-208 and miR-499) that control cardiomyocyte identity.  Stimulating cardiomyocyte dedifferentiation and proliferation by activating mitotic signalling pathways involved in embryonic heart growth represents a complementary approach for heart regeneration and repair.  Recent advances in understanding the mechanistic basis of heart development offer exciting opportunities for effective therapies for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQi6P5whO9QLVg90H21EOLACvtfcHk0ljf-Abdm2hwdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOrtL3M&md5=124d33b33df1d632864d065890f8f142</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrm3619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3619%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DM.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26aulast%3DBassel-Duby%26aufirst%3DR.%26atitle%3DMending%2520Broken%2520Hearts%253A%2520Cardiac%2520Development%2520as%2520a%2520Basis%2520for%2520Adult%2520Heart%2520Regeneration%2520and%2520Repair%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26spage%3D529%26epage%3D541%26doi%3D10.1038%2Fnrm3619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span> <span> </span><span class="NLM_article-title">GATA-Dependent Recruitment of MEF2 Proteins to Target Promoters</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2046</span>– <span class="NLM_lpage">2055</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.9.2046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1093%2Femboj%2F19.9.2046" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=2046-2055&author=S.+Morinauthor=F.+Charronauthor=L.+Robitailleauthor=M.+Nemer&title=GATA-Dependent+Recruitment+of+MEF2+Proteins+to+Target+Promoters&doi=10.1093%2Femboj%2F19.9.2046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.9.2046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.9.2046%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DS.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DRobitaille%26aufirst%3DL.%26aulast%3DNemer%26aufirst%3DM.%26atitle%3DGATA-Dependent%2520Recruitment%2520of%2520MEF2%2520Proteins%2520to%2520Target%2520Promoters%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D2046%26epage%3D2055%26doi%3D10.1093%2Femboj%2F19.9.2046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cserjesi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span> <span> </span><span class="NLM_article-title">The Transcription Factors GATA4 and dHAND Physically Interact to Synergistically Activate Cardiac Gene Expression through a p300-Dependent Mechanism</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">24390</span>– <span class="NLM_lpage">24398</span>, <span class="refDoi"> DOI: 10.1074/jbc.M202490200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M202490200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=24390-24398&author=Y.+S.+Daiauthor=P.+Cserjesiauthor=B.+E.+Markhamauthor=J.+D.+Molkentin&title=The+Transcription+Factors+GATA4+and+dHAND+Physically+Interact+to+Synergistically+Activate+Cardiac+Gene+Expression+through+a+p300-Dependent+Mechanism&doi=10.1074%2Fjbc.M202490200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M202490200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M202490200%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%2BS.%26aulast%3DCserjesi%26aufirst%3DP.%26aulast%3DMarkham%26aufirst%3DB.%2BE.%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520Transcription%2520Factors%2520GATA4%2520and%2520dHAND%2520Physically%2520Interact%2520to%2520Synergistically%2520Activate%2520Cardiac%2520Gene%2520Expression%2520through%2520a%2520p300-Dependent%2520Mechanism%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D24390%26epage%3D24398%26doi%3D10.1074%2Fjbc.M202490200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kathiriya, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schluterman, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothrock, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eapen, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama-Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">GATA4 Mutations Cause Human Congenital Heart Defects and Reveal an Interaction with TBX5</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">443</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/nature01827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnature01827" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2003&pages=443-447&author=V.+Gargauthor=I.+S.+Kathiriyaauthor=R.+Barnesauthor=M.+K.+Schlutermanauthor=I.+N.+Kingauthor=C.+A.+Butlerauthor=C.+R.+Rothrockauthor=R.+S.+Eapenauthor=K.+Hirayama-Yamadaauthor=K.+Jooauthor=R.+Matsuokaauthor=J.+C.+Cohenauthor=D.+Srivastava&title=GATA4+Mutations+Cause+Human+Congenital+Heart+Defects+and+Reveal+an+Interaction+with+TBX5&doi=10.1038%2Fnature01827"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature01827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01827%26sid%3Dliteratum%253Aachs%26aulast%3DGarg%26aufirst%3DV.%26aulast%3DKathiriya%26aufirst%3DI.%2BS.%26aulast%3DBarnes%26aufirst%3DR.%26aulast%3DSchluterman%26aufirst%3DM.%2BK.%26aulast%3DKing%26aufirst%3DI.%2BN.%26aulast%3DButler%26aufirst%3DC.%2BA.%26aulast%3DRothrock%26aufirst%3DC.%2BR.%26aulast%3DEapen%26aufirst%3DR.%2BS.%26aulast%3DHirayama-Yamada%26aufirst%3DK.%26aulast%3DJoo%26aufirst%3DK.%26aulast%3DMatsuoka%26aufirst%3DR.%26aulast%3DCohen%26aufirst%3DJ.%2BC.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DGATA4%2520Mutations%2520Cause%2520Human%2520Congenital%2520Heart%2520Defects%2520and%2520Reveal%2520an%2520Interaction%2520with%2520TBX5%26jtitle%3DNature%26date%3D2003%26volume%3D424%26spage%3D443%26epage%3D447%26doi%3D10.1038%2Fnature01827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ang, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippens, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radzinsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon-Grady, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pruitt, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Disease Model of GATA4 Mutation Reveals Transcription Factor Cooperativity in Human Cardiogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1734</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.11.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1016%2Fj.cell.2016.11.033" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=1734-1749&author=Y.-S.+Angauthor=R.+N.+Rivasauthor=A.+J.+S.+Ribeiroauthor=R.+Srivasauthor=J.+Riveraauthor=N.+R.+Stoneauthor=K.+Prattauthor=T.+M.+A.+Mohamedauthor=J.-D.+Fuauthor=C.+I.+Spencerauthor=N.+D.+Tippensauthor=M.+Liauthor=A.+Narasimhaauthor=E.+Radzinskyauthor=A.+J.+Moon-Gradyauthor=H.+Yuauthor=B.+L.+Pruittauthor=M.+P.+Snyderauthor=D.+Srivastava&title=Disease+Model+of+GATA4+Mutation+Reveals+Transcription+Factor+Cooperativity+in+Human+Cardiogenesis&doi=10.1016%2Fj.cell.2016.11.033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.11.033%26sid%3Dliteratum%253Aachs%26aulast%3DAng%26aufirst%3DY.-S.%26aulast%3DRivas%26aufirst%3DR.%2BN.%26aulast%3DRibeiro%26aufirst%3DA.%2BJ.%2BS.%26aulast%3DSrivas%26aufirst%3DR.%26aulast%3DRivera%26aufirst%3DJ.%26aulast%3DStone%26aufirst%3DN.%2BR.%26aulast%3DPratt%26aufirst%3DK.%26aulast%3DMohamed%26aufirst%3DT.%2BM.%2BA.%26aulast%3DFu%26aufirst%3DJ.-D.%26aulast%3DSpencer%26aufirst%3DC.%2BI.%26aulast%3DTippens%26aufirst%3DN.%2BD.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DNarasimha%26aufirst%3DA.%26aulast%3DRadzinsky%26aufirst%3DE.%26aulast%3DMoon-Grady%26aufirst%3DA.%2BJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DPruitt%26aufirst%3DB.%2BL.%26aulast%3DSnyder%26aufirst%3DM.%2BP.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DDisease%2520Model%2520of%2520GATA4%2520Mutation%2520Reveals%2520Transcription%2520Factor%2520Cooperativity%2520in%2520Human%2520Cardiogenesis%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D1734%26epage%3D1749%26doi%3D10.1016%2Fj.cell.2016.11.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belaguli, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepulveda, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, R. J.</span></span> <span> </span><span class="NLM_article-title">Cardiac Tissue Enriched Factors Serum Response Factor and GATA-4 Are Mutual Coregulators</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">7550</span>– <span class="NLM_lpage">7558</span>, <span class="refDoi"> DOI: 10.1128/MCB.20.20.7550-7558.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.20.20.7550-7558.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=11003651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnt1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=7550-7558&author=N.+S.+Belaguliauthor=J.+L.+Sepulvedaauthor=V.+Nigamauthor=F.+Charronauthor=M.+Nemerauthor=R.+J.+Schwartz&title=Cardiac+Tissue+Enriched+Factors+Serum+Response+Factor+and+GATA-4+Are+Mutual+Coregulators&doi=10.1128%2FMCB.20.20.7550-7558.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Cardiac tissue enriched factors serum response factor and GATA-4 are mutual coregulators</span></div><div class="casAuthors">Belaguli, Narasimhaswamy S.; Sepulveda, Jorge L.; Nigam, Vishal; Charron, Frederic; Nemer, Mona; Schwartz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7550-7558</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Combinatorial interaction among cardiac tissue-restricted enriched transcription factors may facilitate the expression of cardiac tissue-restricted genes.  Here we show that the MADS box factor serum response factor (SRF) cooperates with the zinc finger protein GATA-4 to synergistically activate numerous myogenic and nonmyogenic serum response element (SRE)-dependent promoters in CV1 fibroblasts.  In the absence of GATA binding sites, synergistic activation depends on binding of SRF to the proximal CArG box sequence in the cardiac and skeletal α-actin promoter.  GATA-4's C-terminal activation domain is obligatory for synergistic coactivation with SRF, and its N-terminal domain and first zinc finger are inhibitory.  SRF and GATA-4 phys. assoc. both in vivo and in vitro through their MADS box and the second zinc finger domains as detd. by protein A pullout assays and by in vivo one-hybrid transfection assays using Gal4 fusion proteins.  Other cardiovascular tissue-restricted GATA factors, such as GATA-5 and GATA-6, were equiv. to GATA-4 in coactivating SRE-dependent targets.  Thus, interaction between the MADS box and C4 zinc finger proteins, a novel regulatory paradigm, mediates activation of SRF-dependent gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKAZGYZLYBWbVg90H21EOLACvtfcHk0ljnjjJ3HN12cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnt1aitrc%253D&md5=ebc350bbcda08c4671e25ca441650de2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1128%2FMCB.20.20.7550-7558.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.20.20.7550-7558.2000%26sid%3Dliteratum%253Aachs%26aulast%3DBelaguli%26aufirst%3DN.%2BS.%26aulast%3DSepulveda%26aufirst%3DJ.%2BL.%26aulast%3DNigam%26aufirst%3DV.%26aulast%3DCharron%26aufirst%3DF.%26aulast%3DNemer%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DR.%2BJ.%26atitle%3DCardiac%2520Tissue%2520Enriched%2520Factors%2520Serum%2520Response%2520Factor%2520and%2520GATA-4%2520Are%2520Mutual%2520Coregulators%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D7550%26epage%3D7558%26doi%3D10.1128%2FMCB.20.20.7550-7558.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nam, Y. J.</span>; <span class="NLM_string-name">Song, K.</span>; <span class="NLM_string-name">Olson, E.
N.</span></span> <span> </span><span class="NLM_article-title">Cardiac Repair by Reprogramming of Cardiac Fibroblasts into Cardiomyocytes</span>. Patent <span class="NLM_patent">WO2012/116064 A1</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=Y.+J.+Nam&author=K.+Song&author=E.%0AN.+Olson&title=Cardiac+Repair+by+Reprogramming+of+Cardiac+Fibroblasts+into+Cardiomyocytes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DNam%26aufirst%3DY.%2BJ.%26atitle%3DCardiac%2520Repair%2520by%2520Reprogramming%2520of%2520Cardiac%2520Fibroblasts%2520into%2520Cardiomyocytes%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schorpp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothenaigner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinshagen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gul, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadian, K.</span></span> <span> </span><span class="NLM_article-title">Identification of Small-Molecule Frequent Hitters from AlphaScreen High-Throughput Screens</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1177/1087057113516861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1177%2F1087057113516861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=24371213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=715-726&author=K.+Schorppauthor=I.+Rothenaignerauthor=E.+Salminaauthor=J.+Reinshagenauthor=T.+Lowauthor=J.+K.+Brenkeauthor=J.+Gopalakrishnanauthor=I.+V.+Tetkoauthor=S.+Gulauthor=K.+Hadian&title=Identification+of+Small-Molecule+Frequent+Hitters+from+AlphaScreen+High-Throughput+Screens&doi=10.1177%2F1087057113516861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small-molecule frequent hitters from Alphascreen high-throughput screens</span></div><div class="casAuthors">Schorpp, Kenji; Rothenaigner, Ina; Salmina, Elena; Reinshagen, Jeanette; Low, Terence; Brenke, Jara K.; Gopalakrishnan, Jay; Tetko, Igor V.; Gul, Sheraz; Hadian, Kamyar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">715-726</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Although small-mol. drug discovery efforts have focused largely on enzyme, receptor, and ion-channel targets, there has been an increase in such activities to search for protein-protein interaction (PPI) disruptors by applying high-throughout screening (HTS)-compatible protein-binding assays.  However, a disadvantage of these assays is that many primary hits are frequent hitters regardless of the PPI being investigated.  We have used the AlphaScreen technol. to screen four different robust PPI assays each against 25,000 compds.  These activities led to the identification of 137 compds. that demonstrated repeated activity in all PPI assays.  These compds. were subsequently evaluated in two AlphaScreen counter assays, leading to classification of compds. that either interfered with the AlphaScreen chem. (60 compds.) or prevented the binding of the protein His-tag moiety to nickel chelate (Ni2+-NTA) beads of the AlphaScreen detection system (77 compds.).  To further triage the 137 frequent hitters, we subsequently confirmed by a time-resolved fluorescence resonance energy transfer assay that most of these compds. were only frequent hitters in AlphaScreen assays.  A chemoinformatics anal. of the apparent hits provided details of the compds. that can be flagged as frequent hitters of the AlphaScreen technol., and these data have broad applicability for users of these detection technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQj9hCbRqv7LVg90H21EOLACvtfcHk0livne1g7V0Trg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWqu7bM&md5=ea65e879f093202deb2266bad94f0208</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1177%2F1087057113516861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113516861%26sid%3Dliteratum%253Aachs%26aulast%3DSchorpp%26aufirst%3DK.%26aulast%3DRothenaigner%26aufirst%3DI.%26aulast%3DSalmina%26aufirst%3DE.%26aulast%3DReinshagen%26aufirst%3DJ.%26aulast%3DLow%26aufirst%3DT.%26aulast%3DBrenke%26aufirst%3DJ.%2BK.%26aulast%3DGopalakrishnan%26aufirst%3DJ.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DGul%26aufirst%3DS.%26aulast%3DHadian%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520Small-Molecule%2520Frequent%2520Hitters%2520from%2520AlphaScreen%2520High-Throughput%2520Screens%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D715%26epage%3D726%26doi%3D10.1177%2F1087057113516861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomašić, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin Mašič, L.</span></span> <span> </span><span class="NLM_article-title">Rhodanine as a Scaffold in Drug Discovery: A Critical Review of Its Biological Activities and Mechanisms of Target Modulation</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1517/17460441.2012.688743</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1517%2F17460441.2012.688743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=22607309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWmtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=549-560&author=T.+Toma%C5%A1i%C4%87author=L.+Peterlin+Ma%C5%A1i%C4%8D&title=Rhodanine+as+a+Scaffold+in+Drug+Discovery%3A+A+Critical+Review+of+Its+Biological+Activities+and+Mechanisms+of+Target+Modulation&doi=10.1517%2F17460441.2012.688743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Rhodanine as a scaffold in drug discovery: a critical review of its biological activities and mechanisms of target modulation</span></div><div class="casAuthors">Tomasic, Tihomir; Peterlin Masic, Lucija</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Rhodanine-based compds. have been assocd. with numerous biol. activities.  After many years of research in drug discovery, they have gained a reputation as being pan assay interference compds. (PAINS) and frequent hitters in screening campaigns.  Rhodanine-based compds. are also aggregators that can non-specifically interact with target proteins as well as Michael acceptors and interfere photometrically in biol. assays due to their color.Areas covered: The authors review the recently reported biol. activities of rhodanine-based compds.  Furthermore, the article provides details of their synthesis and occurrence in compd. libraries through high-throughput screening (HTS) and virtual high-throughput screening (VHTS).  Addnl., the authors provide the reader with possible mechanisms of non-specific target modulation, anal. of the crystal structures of enzyme-rhodanine complexes and a comparison of rhodanine and thiazolidine-2,4-dione moieties.Expert opinion: The biol. activity of compds. possessing a rhodanine moiety should be considered very critically despite the convincing data obtained in biol. assays.  In addn. to the lack of selectivity, unusual structure-activity relationship profiles and safety and specificity problems mean that rhodanines are generally not optimizable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorolVH32PdC7Vg90H21EOLACvtfcHk0livne1g7V0Trg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWmtLw%253D&md5=a02e78c0c7788aaeb044f3753c901117</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.688743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.688743%26sid%3Dliteratum%253Aachs%26aulast%3DToma%25C5%25A1i%25C4%2587%26aufirst%3DT.%26aulast%3DPeterlin%2BMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%26atitle%3DRhodanine%2520as%2520a%2520Scaffold%2520in%2520Drug%2520Discovery%253A%2520A%2520Critical%2520Review%2520of%2520Its%2520Biological%2520Activities%2520and%2520Mechanisms%2520of%2520Target%2520Modulation%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D549%26epage%3D560%26doi%3D10.1517%2F17460441.2012.688743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schade, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">Medicinal Chemistry Approaches to Heart Regeneration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9451</span>– <span class="NLM_lpage">9479</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00446</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00446" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9451-9479&author=D.+Schadeauthor=A.+T.+Plowright&title=Medicinal+Chemistry+Approaches+to+Heart+Regeneration&doi=10.1021%2Facs.jmedchem.5b00446"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00446%26sid%3Dliteratum%253Aachs%26aulast%3DSchade%26aufirst%3DD.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DMedicinal%2520Chemistry%2520Approaches%2520to%2520Heart%2520Regeneration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9451%26epage%3D9479%26doi%3D10.1021%2Facs.jmedchem.5b00446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennewell, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seden, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, G. M.</span></span> <span> </span><span class="NLM_article-title">Stemistry: The Control of Stem Cells in Situ Using Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2863</span>– <span class="NLM_lpage">2894</span>, <span class="refDoi"> DOI: 10.1021/jm500838d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500838d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVyjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2863-2894&author=S.+G.+Daviesauthor=P.+D.+Kennewellauthor=A.+J.+Russellauthor=P.+T.+Sedenauthor=R.+Westwoodauthor=G.+M.+Wynne&title=Stemistry%3A+The+Control+of+Stem+Cells+in+Situ+Using+Chemistry&doi=10.1021%2Fjm500838d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Stemistry: The Control of Stem Cells in Situ Using Chemistry</span></div><div class="casAuthors">Davies, Stephen G.; Kennewell, Peter D.; Russell, Angela J.; Seden, Peter T.; Westwood, Robert; Wynne, Graham M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2863-2894</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A new paradigm for drug research has emerged, namely the deliberate search for mols. able to selectively affect the proliferation, differentiation, and migration of adult stem cells within the tissues in which they exist.  Recently, there has been significant interest in medicinal chem. toward the discovery and design of low mol. wt. mols. that affect stem cells and thus have novel therapeutic activity.  We believe that a successful agent from such a discover program would have profound effects on the treatment of many long-term degenerative disorders.  Among these conditions are examples such as cardiovascular decay, neurol. disorders including Alzheimer's disease, and macular degeneration, all of which have significant unmet medical needs.  This perspective will review evidence from the literature that indicates that discovery of such agents is achievable and represents a worthwhile pursuit for the skills of the medicinal chemist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZPeNZi3O2o7Vg90H21EOLACvtfcHk0lja3ZKIgq6pxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVyjuw%253D%253D&md5=f270b737affe419ca1405ba79b2a38e3</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm500838d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500838d%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DS.%2BG.%26aulast%3DKennewell%26aufirst%3DP.%2BD.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DSeden%26aufirst%3DP.%2BT.%26aulast%3DWestwood%26aufirst%3DR.%26aulast%3DWynne%26aufirst%3DG.%2BM.%26atitle%3DStemistry%253A%2520The%2520Control%2520of%2520Stem%2520Cells%2520in%2520Situ%2520Using%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2863%26epage%3D2894%26doi%3D10.1021%2Fjm500838d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimelman, D.</span></span> <span> </span><span class="NLM_article-title">Mesoderm Induction: From Caps to Chips</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nrg1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnrg1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=16619051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFSqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=360-372&author=D.+Kimelman&title=Mesoderm+Induction%3A+From+Caps+to+Chips&doi=10.1038%2Fnrg1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mesoderm induction: from caps to chips</span></div><div class="casAuthors">Kimelman, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">360-372</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Vertebrate mesoderm induction is one of the classical problems in developmental biol.  Various developmental biol. approaches, particularly in Xenopus and zebrafish, have identified many of the key factors that are involved in this process and have provided major insights into how these factors interact as part of a signaling and transcription-factor network.  These data are beginning to be refined by high-throughput approaches such as microarray assays.  Future challenges include understanding how the prospective mesodermal cells integrate the various signals they receive and how they resolve this information to regulate their morphogenetic behaviors and cell-fate decisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkEur7SDJVLVg90H21EOLACvtfcHk0lja3ZKIgq6pxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFSqt7o%253D&md5=3d7499f0d4dbac429911ac1ed67e6d4d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnrg1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1837%26sid%3Dliteratum%253Aachs%26aulast%3DKimelman%26aufirst%3DD.%26atitle%3DMesoderm%2520Induction%253A%2520From%2520Caps%2520to%2520Chips%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2006%26volume%3D7%26spage%3D360%26epage%3D372%26doi%3D10.1038%2Fnrg1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witty, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikryukov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, G.</span></span> <span> </span><span class="NLM_article-title">Generation of the Epicardial Lineage from Human Pluripotent Stem Cells</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1026</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1038/nbt.3002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnbt.3002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1026-1035&author=A.+D.+Wittyauthor=A.+Mihicauthor=R.+Y.+Tamauthor=S.+A.+Fisherauthor=A.+Mikryukovauthor=M.+S.+Shoichetauthor=R.+K.+Liauthor=S.+J.+Kattmanauthor=G.+Keller&title=Generation+of+the+Epicardial+Lineage+from+Human+Pluripotent+Stem+Cells&doi=10.1038%2Fnbt.3002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3002%26sid%3Dliteratum%253Aachs%26aulast%3DWitty%26aufirst%3DA.%2BD.%26aulast%3DMihic%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DR.%2BY.%26aulast%3DFisher%26aufirst%3DS.%2BA.%26aulast%3DMikryukov%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DR.%2BK.%26aulast%3DKattman%26aufirst%3DS.%2BJ.%26aulast%3DKeller%26aufirst%3DG.%26atitle%3DGeneration%2520of%2520the%2520Epicardial%2520Lineage%2520from%2520Human%2520Pluripotent%2520Stem%2520Cells%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D1026%26epage%3D1035%26doi%3D10.1038%2Fnbt.3002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span> <span> </span><span class="NLM_article-title">Conversion of Human Fibroblasts into Functional Cardiomyocytes by Small Molecules</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1220</span>, <span class="refDoi"> DOI: 10.1126/science.aaf1502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1126%2Fscience.aaf1502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=27127239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslOqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=1216-1220&author=N.+Caoauthor=Y.+Huangauthor=J.+Zhengauthor=C.+I.+Spencerauthor=Y.+Zhangauthor=J.-D.+Fuauthor=B.+Nieauthor=M.+Xieauthor=M.+Zhangauthor=H.+Wangauthor=T.+Maauthor=T.+Xuauthor=G.+Shiauthor=D.+Srivastavaauthor=S.+Ding&title=Conversion+of+Human+Fibroblasts+into+Functional+Cardiomyocytes+by+Small+Molecules&doi=10.1126%2Fscience.aaf1502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Conversion of human fibroblasts into functional cardiomyocytes by small molecules</span></div><div class="casAuthors">Cao, Nan; Huang, Yu; Zheng, Jiashun; Spencer, C. Ian; Zhang, Yu; Fu, Ji-Dong; Nie, Baoming; Xie, Min; Zhang, Mingliang; Wang, Haixia; Ma, Tianhua; Xu, Tao; Shi, Guilai; Srivastava, Deepak; Ding, Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">6290</span>),
    <span class="NLM_cas:pages">1216-1220</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Reprogramming somatic fibroblasts into alternative lineages would provide a promising source of cells for regenerative therapy.  However, transdifferentiating human cells into specific homogeneous, functional cell types is challenging.  Here we show that cardiomyocyte-like cells can be generated by treating human fibroblasts with a combination of nine compds. that we term 9C.  The chem. induced cardiomyocyte-like cells uniformly contracted and resembled human cardiomyocytes in their transcriptome, epigenetic, and electrophysiol. properties.  9C treatment of human fibroblasts resulted in a more open-chromatin conformation at key heart developmental genes, enabling their promoters and enhancers to bind effectors of major cardiogenic signals.  When transplanted into infarcted mouse hearts, 9C-treated fibroblasts were efficiently converted to chem. induced cardiomyocyte-like cells.  This pharmacol. approach to lineage-specific reprogramming may have many important therapeutic implications after further optimization to generate mature cardiac cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshOttrRImeLVg90H21EOLACvtfcHk0lja3ZKIgq6pxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslOqs7Y%253D&md5=e42d122563059ffe9f59fc4e889c35e5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf1502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf1502%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DSpencer%26aufirst%3DC.%2BI.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DJ.-D.%26aulast%3DNie%26aufirst%3DB.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DG.%26aulast%3DSrivastava%26aufirst%3DD.%26aulast%3DDing%26aufirst%3DS.%26atitle%3DConversion%2520of%2520Human%2520Fibroblasts%2520into%2520Functional%2520Cardiomyocytes%2520by%2520Small%2520Molecules%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D1216%26epage%3D1220%26doi%3D10.1126%2Fscience.aaf1502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radzinsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnitsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivey, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Chemical Enhancement of In Vitro and In Vivo Direct Cardiac Reprogramming</span>. <i>Circulation</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">995</span>, <span class="refDoi"> DOI: 10.1161/CIRCULATIONAHA.116.024692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1161%2FCIRCULATIONAHA.116.024692" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2017&pages=978-995&author=T.+M.+A.+Mohamedauthor=N.+R.+Stoneauthor=E.+C.+Berryauthor=E.+Radzinskyauthor=Y.+Huangauthor=K.+Prattauthor=Y.+S.+Angauthor=P.+Yuauthor=H.+Wangauthor=S.+Tangauthor=S.+Magnitskyauthor=S.+Dingauthor=K.+N.+Iveyauthor=D.+Srivastava&title=Chemical+Enhancement+of+In+Vitro+and+In+Vivo+Direct+Cardiac+Reprogramming&doi=10.1161%2FCIRCULATIONAHA.116.024692"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1161%2FCIRCULATIONAHA.116.024692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCULATIONAHA.116.024692%26sid%3Dliteratum%253Aachs%26aulast%3DMohamed%26aufirst%3DT.%2BM.%2BA.%26aulast%3DStone%26aufirst%3DN.%2BR.%26aulast%3DBerry%26aufirst%3DE.%2BC.%26aulast%3DRadzinsky%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DPratt%26aufirst%3DK.%26aulast%3DAng%26aufirst%3DY.%2BS.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DMagnitsky%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DIvey%26aufirst%3DK.%2BN.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DChemical%2520Enhancement%2520of%2520In%2520Vitro%2520and%2520In%2520Vivo%2520Direct%2520Cardiac%2520Reprogramming%26jtitle%3DCirculation%26date%3D2017%26volume%3D135%26spage%3D978%26epage%3D995%26doi%3D10.1161%2FCIRCULATIONAHA.116.024692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadek, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannack, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garry, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longgood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. W.</span></span> <span> </span><span class="NLM_article-title">Cardiogenic Small Molecules That Enhance Myocardial Repair by Stem Cells</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">6063</span>– <span class="NLM_lpage">6068</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711507105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1073%2Fpnas.0711507105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6063-6068&author=H.+Sadekauthor=B.+Hannackauthor=E.+Choeauthor=J.+Wangauthor=S.+Latifauthor=M.+G.+Garryauthor=D.+J.+Garryauthor=J.+Longgoodauthor=D.+E.+Frantzauthor=E.+N.+Olsonauthor=J.+Hsiehauthor=J.+W.+Schneider&title=Cardiogenic+Small+Molecules+That+Enhance+Myocardial+Repair+by+Stem+Cells&doi=10.1073%2Fpnas.0711507105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711507105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711507105%26sid%3Dliteratum%253Aachs%26aulast%3DSadek%26aufirst%3DH.%26aulast%3DHannack%26aufirst%3DB.%26aulast%3DChoe%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLatif%26aufirst%3DS.%26aulast%3DGarry%26aufirst%3DM.%2BG.%26aulast%3DGarry%26aufirst%3DD.%2BJ.%26aulast%3DLonggood%26aufirst%3DJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26aulast%3DHsieh%26aufirst%3DJ.%26aulast%3DSchneider%26aufirst%3DJ.%2BW.%26atitle%3DCardiogenic%2520Small%2520Molecules%2520That%2520Enhance%2520Myocardial%2520Repair%2520by%2520Stem%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D6063%26epage%3D6068%26doi%3D10.1073%2Fpnas.0711507105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rooij, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. W.</span></span> <span> </span><span class="NLM_article-title">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1083</span>, <span class="refDoi"> DOI: 10.1021/cb300001m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300001m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1077-1083&author=J.+L.+Russellauthor=S.+C.+Goetschauthor=H.+R.+Aguilarauthor=H.+Coeauthor=X.+Luoauthor=N.+Liuauthor=E.+van+Rooijauthor=D.+E.+Frantzauthor=J.+W.+Schneider&title=Regulated+Expression+of+pH+Sensing+G+Protein-Coupled+Receptor-68+Identified+through+Chemical+Biology+Defines+a+New+Drug+Target+for+Ischemic+Heart+Disease&doi=10.1021%2Fcb300001m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span></div><div class="casAuthors">Russell, Jamie L.; Goetsch, Sean C.; Aguilar, Hector R.; Coe, Helen; Luo, Xiang; Liu, Ning; van Rooij, Eva; Frantz, Doug E.; Schneider, Jay W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1077-1083</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. biol. promises discovery of new and unexpected mechanistic pathways, protein functions and disease targets.  Here, we probed the mechanism-of-action and protein targets of 3,5-disubstituted isoxazoles (Isx), cardiomyogenic small mols. that target Notch-activated epicardium-derived cells (NECs) in vivo and promote functional recovery after myocardial infarction (MI).  Mechanistic studies in NECs led to an Isx-activated Gq protein-coupled receptor (GqPCR) hypothesis tested in a cell-based functional target screen for GPCRs regulated by Isx.  This screen identified one agonist hit, the extracellular proton/pH-sensing GPCR GPR68, confirmed through genetic gain- and loss-of-function.  Overlooked until now, GPR68 expression and localization were highly regulated in early post-natal and adult post-infarct mouse heart, where GPR68-expressing cells accumulated subepicardially.  Remarkably, GPR68-expressing cardiomyocytes established a proton-sensing cellular buffer zone surrounding the MI.  Isx pharmacol. regulated gene expression (mRNAs and miRs) in this GPR68-enriched border zone, driving cardiomyogenic and pro-survival transcriptional programs in vivo.  In conclusion, we tracked a (micromolar) bioactive small mol. mechanism-of-action to a candidate target protein, GPR68, and validated this target as a previously unrecognized regulator of myocardial cellular responses to tissue acidosis, setting the stage for future (nanomolar) target-based drug lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQLtpNAH6aiLVg90H21EOLACvtfcHk0lhVuxd4fQx7cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D&md5=751e2e1833518df9da9253f0516ba4e5</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fcb300001m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300001m%26sid%3Dliteratum%253Aachs%26aulast%3DRussell%26aufirst%3DJ.%2BL.%26aulast%3DGoetsch%26aufirst%3DS.%2BC.%26aulast%3DAguilar%26aufirst%3DH.%2BR.%26aulast%3DCoe%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3Dvan%2BRooij%26aufirst%3DE.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DSchneider%26aufirst%3DJ.%2BW.%26atitle%3DRegulated%2520Expression%2520of%2520pH%2520Sensing%2520G%2520Protein-Coupled%2520Receptor-68%2520Identified%2520through%2520Chemical%2520Biology%2520Defines%2520a%2520New%2520Drug%2520Target%2520for%2520Ischemic%2520Heart%2520Disease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1077%26epage%3D1083%26doi%3D10.1021%2Fcb300001m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sdek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLellan, W. R.</span></span> <span> </span><span class="NLM_article-title">Cardiac Myocyte Cell Cycle Control in Development, Disease, and Regeneration</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">544</span>, <span class="refDoi"> DOI: 10.1152/physrev.00032.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1152%2Fphysrev.00032.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=17429040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD2sXoslGmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2007&pages=521-544&author=P.+Ahujaauthor=P.+Sdekauthor=W.+R.+MacLellan&title=Cardiac+Myocyte+Cell+Cycle+Control+in+Development%2C+Disease%2C+and+Regeneration&doi=10.1152%2Fphysrev.00032.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Cardiac myocyte cell cycle control in development, disease, and regeneration</span></div><div class="casAuthors">Ahuja, Preeti; Sdek, Patima; Maclellan, W. Robb</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">521-544</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  Cardiac myocytes rapidly proliferate during fetal life but exit the cell cycle soon after birth in mammals.  Although the extent to which adult cardiac myocytes are capable of cell cycle reentry is controversial and species-specific differences may exist, it appears that for the vast majority of adult cardiac myocytes the predominant form of growth postnatally is an increase in cell size (hypertrophy) not no.  Unfortunately, this limits the ability of the heart to restore function after any significant injury.  Interest in novel regenerative therapies has led to the accumulation of much information on the mechanisms that regulate the rapid proliferation of cardiac myocytes in utero, their cell cycle exit in the perinatal period, and the permanent arrest (terminal differentiation) in adult myocytes.  The recent identification of cardiac progenitor cells capable of giving rise to cardiac myocyte-like cells has challenged the dogma that the heart is a terminally differentiated organ and opened new prospects for cardiac regeneration.  In this review, we summarize the current understanding of cardiomyocyte cell cycle control in normal development and disease.  In addn., we also discuss the potential usefulness of cardiomyocyte self-renewal as well as feasibility of therapeutic manipulation of the cardiac myocyte cell cycle for cardiac regeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxO6H0TKcdbVg90H21EOLACvtfcHk0lhVuxd4fQx7cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXoslGmu7c%253D&md5=956e63a025795a6e45177af98e7550e2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00032.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00032.2006%26sid%3Dliteratum%253Aachs%26aulast%3DAhuja%26aufirst%3DP.%26aulast%3DSdek%26aufirst%3DP.%26aulast%3DMacLellan%26aufirst%3DW.%2BR.%26atitle%3DCardiac%2520Myocyte%2520Cell%2520Cycle%2520Control%2520in%2520Development%252C%2520Disease%252C%2520and%2520Regeneration%26jtitle%3DPhysiol.%2520Rev.%26date%3D2007%26volume%3D87%26spage%3D521%26epage%3D544%26doi%3D10.1152%2Fphysrev.00032.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Addis, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epstein, J. A.</span></span> <span> </span><span class="NLM_article-title">Induced Regeneration--the Progress and Promise of Direct Reprogramming for Heart Repair</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1038/nm.3225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnm.3225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23836233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=829-836&author=R.+C.+Addisauthor=J.+A.+Epstein&title=Induced+Regeneration%2D%2Dthe+Progress+and+Promise+of+Direct+Reprogramming+for+Heart+Repair&doi=10.1038%2Fnm.3225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Induced regeneration-the progress and promise of direct reprogramming for heart repair</span></div><div class="casAuthors">Addis, Russell C.; Epstein, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">829-836</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Regeneration of cardiac tissue has the potential to transform cardiovascular medicine.  Recent advances in stem cell biol. and direct reprogramming, or transdifferentiation, have produced powerful new tools to advance this goal.  In this Review we examine key developments in the generation of new cardiomyocytes in vitro as well as the exciting progress that has been made toward in vivo reprogramming of cardiac tissue.  We also address controversies and hurdles that challenge the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr34PLjZ02oU7Vg90H21EOLACvtfcHk0lgeDurpPwc7qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsr%252FJ&md5=0e23361ce4ed6a03f5342bf9818294e5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnm.3225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3225%26sid%3Dliteratum%253Aachs%26aulast%3DAddis%26aufirst%3DR.%2BC.%26aulast%3DEpstein%26aufirst%3DJ.%2BA.%26atitle%3DInduced%2520Regeneration--the%2520Progress%2520and%2520Promise%2520of%2520Direct%2520Reprogramming%2520for%2520Heart%2520Repair%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D829%26epage%3D836%26doi%3D10.1038%2Fnm.3225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muraoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, M.</span></span> <span> </span><span class="NLM_article-title">Direct Reprogramming of Fibroblasts into Myocytes to Reverse Fibrosis</span>. <i>Annu. Rev. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1146/annurev-physiol-021113-170301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1146%2Fannurev-physiol-021113-170301" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2014&pages=21-37&author=N.+Muraokaauthor=M.+Ieda&title=Direct+Reprogramming+of+Fibroblasts+into+Myocytes+to+Reverse+Fibrosis&doi=10.1146%2Fannurev-physiol-021113-170301"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1146%2Fannurev-physiol-021113-170301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-physiol-021113-170301%26sid%3Dliteratum%253Aachs%26aulast%3DMuraoka%26aufirst%3DN.%26aulast%3DIeda%26aufirst%3DM.%26atitle%3DDirect%2520Reprogramming%2520of%2520Fibroblasts%2520into%2520Myocytes%2520to%2520Reverse%2520Fibrosis%26jtitle%3DAnnu.%2520Rev.%2520Physiol.%26date%3D2014%26volume%3D76%26spage%3D21%26epage%3D37%26doi%3D10.1146%2Fannurev-physiol-021113-170301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, L.</span></span> <span> </span><span class="NLM_article-title">Direct Cardiac Reprogramming as a Novel Therapeutic Strategy for Treatment of Myocardial Infarction</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1521</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-6588-5_5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1007%2F978-1-4939-6588-5_5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=27910042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWit7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1521&publication_year=2017&pages=69-88&author=H.+Maauthor=L.+Wangauthor=J.+Liuauthor=L.+Qian&title=Direct+Cardiac+Reprogramming+as+a+Novel+Therapeutic+Strategy+for+Treatment+of+Myocardial+Infarction&doi=10.1007%2F978-1-4939-6588-5_5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Direct cardiac reprogramming as a novel therapeutic strategy for treatment of myocardial infarction</span></div><div class="casAuthors">Ma, Hong; Wang, Li; Liu, Jiandong; Qian, Li</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1521</span>
        (<span class="NLM_cas:issue">Cardiac Gene Therapy</span>),
    <span class="NLM_cas:pages">69-88</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Direct reprogramming of fibroblasts into induced cardiomyocytes (iCMs) holds great promise as a novel therapy for the treatment of heart failure, a common and morbid disease that is usually caused by irreversible loss of functional cardiomyocytes (CMs).  Recently, we and others showed that in a murine model of acute myocardial infarction, delivery of three transcription factors, Gata4, Mef2c, and Tbx5 converted endogenous cardiac fibroblasts into functional iCMs.  These iCMs integrated elec. and mech. with surrounding myocardium, resulting in a redn. in scar size and an improvement in heart function.  Our findings suggest that iCM reprogramming may be a means of regenerating functional CMs in vivo for patients with heart disease.  However, because relatively little is known about the factors that regulate iCM reprogramming, the applicability of iCM reprogramming is currently limited to the exptl. settings in which it has been attempted.  Specific hurdles include the relatively low conversion rate of iCMs and the need for reprogramming to occur in the context of acute injury.  Therefore, before this treatment can become a viable therapy for human heart disease, the optimal condition for efficient iCM generation must be detd.  Here, we provide a detailed protocol for both in vitro and in vivo iCM generation that has been optimized so far in our lab.  We hope that this protocol will lay a foundation for future further improvement of iCM generation and provide a platform for mechanistic studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeuYiCGsA7r7Vg90H21EOLACvtfcHk0lgeDurpPwc7qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWit7vP&md5=6f47686a504ba7bd8b458c354ff49d77</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-6588-5_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-6588-5_5%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DL.%26atitle%3DDirect%2520Cardiac%2520Reprogramming%2520as%2520a%2520Novel%2520Therapeutic%2520Strategy%2520for%2520Treatment%2520of%2520Myocardial%2520Infarction%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2017%26volume%3D1521%26spage%3D69%26epage%3D88%26doi%3D10.1007%2F978-1-4939-6588-5_5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazeltine, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azarin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raval, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamp, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palecek, S. P.</span></span> <span> </span><span class="NLM_article-title">Robust Cardiomyocyte Differentiation from Human Pluripotent Stem Cells via Temporal Modulation of Canonical Wnt Signaling</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1848</span>– <span class="NLM_lpage">1857</span>, <span class="refDoi"> DOI: 10.1073/pnas.1200250109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1073%2Fpnas.1200250109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=1848-1857&author=X.+Lianauthor=C.+Hsiaoauthor=G.+Wilsonauthor=K.+Zhuauthor=L.+B.+Hazeltineauthor=S.+M.+Azarinauthor=K.+K.+Ravalauthor=J.+Zhangauthor=T.+J.+Kampauthor=S.+P.+Palecek&title=Robust+Cardiomyocyte+Differentiation+from+Human+Pluripotent+Stem+Cells+via+Temporal+Modulation+of+Canonical+Wnt+Signaling&doi=10.1073%2Fpnas.1200250109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1200250109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1200250109%26sid%3Dliteratum%253Aachs%26aulast%3DLian%26aufirst%3DX.%26aulast%3DHsiao%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DHazeltine%26aufirst%3DL.%2BB.%26aulast%3DAzarin%26aufirst%3DS.%2BM.%26aulast%3DRaval%26aufirst%3DK.%2BK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKamp%26aufirst%3DT.%2BJ.%26aulast%3DPalecek%26aufirst%3DS.%2BP.%26atitle%3DRobust%2520Cardiomyocyte%2520Differentiation%2520from%2520Human%2520Pluripotent%2520Stem%2520Cells%2520via%2520Temporal%2520Modulation%2520of%2520Canonical%2520Wnt%2520Signaling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D1848%26epage%3D1857%26doi%3D10.1073%2Fpnas.1200250109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azarin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazeltine, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamp, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palecek, S. P.</span></span> <span> </span><span class="NLM_article-title">Directed Cardiomyocyte Differentiation from Human Pluripotent Stem Cells by Modulating Wnt/β-Catenin Signaling under Fully Defined Conditions</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1038/nprot.2012.150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1038%2Fnprot.2012.150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=23257984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1KisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=162-175&author=X.+Lianauthor=J.+Zhangauthor=S.+M.+Azarinauthor=K.+Zhuauthor=L.+B.+Hazeltineauthor=X.+Baoauthor=C.+Hsiaoauthor=T.+J.+Kampauthor=S.+P.+Palecek&title=Directed+Cardiomyocyte+Differentiation+from+Human+Pluripotent+Stem+Cells+by+Modulating+Wnt%2F%CE%B2-Catenin+Signaling+under+Fully+Defined+Conditions&doi=10.1038%2Fnprot.2012.150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/β-catenin signaling under fully defined conditions</span></div><div class="casAuthors">Lian, Xiaojun; Zhang, Jianhua; Azarin, Samira M.; Zhu, Kexian; Hazeltine, Laurie B.; Bao, Xiaoping; Hsiao, Cheston; Kamp, Timothy J.; Palecek, Sean P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">162-175</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The protocol described here efficiently directs human pluripotent stem cells (hPSCs) to functional cardiomyocytes in a completely defined, growth factor- and serum-free system by temporal modulation of regulators of canonical Wnt signaling.  Appropriate temporal application of a glycogen synthase kinase 3 (GSK3) inhibitor combined with the expression of β-catenin shRNA or a chem. Wnt inhibitor is sufficient to produce a high yield (0.8-1.3 million cardiomyocytes per cm2) of virtually pure (80-98%) functional cardiomyocytes in 14 d from multiple hPSC lines without cell sorting or selection.  Qual. (immunostaining) and quant. (flow cytometry) characterization of differentiated cells is described to assess the expression of cardiac transcription factors and myofilament proteins.  Flow cytometry of BrdU incorporation or Ki67 expression in conjunction with cardiac sarcomere myosin protein expression can be used to det. the proliferative capacity of hPSC-derived cardiomyocytes.  Functional human cardiomyocytes differentiated via these protocols may constitute a potential cell source for heart disease modeling, drug screening and cell-based therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKfgKIeSpJybVg90H21EOLACvtfcHk0lhUrcOxAu_rgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1KisLk%253D&md5=e8a8e7f0ab41ce99452cb8c4562129e7</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2012.150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2012.150%26sid%3Dliteratum%253Aachs%26aulast%3DLian%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAzarin%26aufirst%3DS.%2BM.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DHazeltine%26aufirst%3DL.%2BB.%26aulast%3DBao%26aufirst%3DX.%26aulast%3DHsiao%26aufirst%3DC.%26aulast%3DKamp%26aufirst%3DT.%2BJ.%26aulast%3DPalecek%26aufirst%3DS.%2BP.%26atitle%3DDirected%2520Cardiomyocyte%2520Differentiation%2520from%2520Human%2520Pluripotent%2520Stem%2520Cells%2520by%2520Modulating%2520Wnt%252F%25CE%25B2-Catenin%2520Signaling%2520under%2520Fully%2520Defined%2520Conditions%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D162%26epage%3D175%26doi%3D10.1038%2Fnprot.2012.150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">The Generalized Born/volume Integral Implicit Solvent Model: Estimation of the Free Energy of Hydration Using London Dispersion instead of Atomic Surface Area</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1698</span>, <span class="refDoi"> DOI: 10.1002/jcc.20933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2Fjcc.20933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=18307169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1693-1698&author=P.+Labute&title=The+Generalized+Born%2Fvolume+Integral+Implicit+Solvent+Model%3A+Estimation+of+the+Free+Energy+of+Hydration+Using+London+Dispersion+instead+of+Atomic+Surface+Area&doi=10.1002%2Fjcc.20933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The generalized Born/volume integral implicit solvent model: estimation of the free energy of hydration using London dispersion instead of atomic surface area</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1693-1698</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A new generalized Born model for estg. the free energy of hydration is presented.  The new generalized Born/vol. integral (GB/VI) ests. the free energy of hydration as a classical electrostatic energy plus a cavitation energy that is not based upon at. surface area (SA) used in GB/SA hydration models but on a VI London dispersion energy estd. from quantities already calcd. in the classical electrostatic energy.  The (relatively few) GB/VI model parameters are fitted to exptl. data, and parameterizations for two different at. partial charge models are presented.  Comparison of the calcd. and exptl. free energies of hydration for 560 small mols. (both neutral and charged) shows good agreement (r2 = 0.94).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCY3g7kWWRo7Vg90H21EOLACvtfcHk0lhUrcOxAu_rgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D&md5=59420cf1dbe689836873df8e802d06cd</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20933%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DThe%2520Generalized%2520Born%252Fvolume%2520Integral%2520Implicit%2520Solvent%2520Model%253A%2520Estimation%2520of%2520the%2520Free%2520Energy%2520of%2520Hydration%2520Using%2520London%2520Dispersion%2520instead%2520of%2520Atomic%2520Surface%2520Area%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26spage%3D1693%26epage%3D1698%26doi%3D10.1002%2Fjcc.20933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bakker, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Word, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure Validation by Calpha Geometry: Phi, Psi and Cbeta Deviation</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">450</span>, <span class="refDoi"> DOI: 10.1002/prot.10286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1002%2Fprot.10286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12557186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2003&pages=437-450&author=S.+C.+Lovellauthor=I.+W.+Davisauthor=W.+B.+Arendallauthor=P.+I.+de+Bakkerauthor=J.+M.+Wordauthor=M.+G.+Prisantauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=Structure+Validation+by+Calpha+Geometry%3A+Phi%2C+Psi+and+Cbeta+Deviation&doi=10.1002%2Fprot.10286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structure validation by Cα geometry: φ, ψ and Cβ deviation</span></div><div class="casAuthors">Lovell, Simon C.; Davis, Ian W.; Arendall, W. Bryan, III; de Bakker, Paul I. W.; Word, J. Michael; Prisant, Michael G.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-450</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Geometrical validation around the Cα is described, with a new Cβ measure and updated Ramachandran plot.  Deviation of the obsd. Cβ atom from ideal position provides a single measure encapsulating the major structure-validation information contained in bond angle distortions.  Cβ deviation is sensitive to incompatibilities between sidechain and backbone caused by misfit conformations or inappropriate refinement restraints.  A new .vphi.,ψ plot using d.-dependent smoothing for 81,234 non-Gly, non-Pro, and non-prePro residues with B < 30 from 500 high-resoln. proteins shows sharp boundaries at crit. edges and clear delineation between large empty areas and regions that are allowed but disfavored.  One such region is the γ-turn conformation near +75°, -60°, counted as forbidden by common structure-validation programs; however, it occurs in well-ordered parts of good structures, it is overrepresented near functional sites, and strain is partly compensated by the γ-turn H-bond.  Favored and allowed .vphi.,ψ regions are also defined for Pro, pre-Pro, and Gly (important because Gly .vphi.,ψ angles are more permissive but less accurately detd.).  Details of these accurate empirical distributions are poorly predicted by previous theor. calcns., including a region left of α-helix, which rates as favorable in energy yet rarely occurs.  A proposed factor explaining this discrepancy is that crowding of the two-peptide NHs permits donating only a single H-bond.  New calcns. by H. Hu et al. (2002) for Ala and Gly dipeptides, using mixed quantum mechanics and mol. mechanics, fit our nonrepetitive data in excellent detail.  To run our geometrical evaluations on a user-uploaded file, see MOLPROBITY (http://kinemage.biochem.duke.edu) or RAMPAGE (http://www-cryst.bioc.cam.ac.uk/rampage).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFFimJUYSPl7Vg90H21EOLACvtfcHk0lhUrcOxAu_rgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D&md5=e9d41070d9e8832ef9aedf6a523dfd73</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fprot.10286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10286%26sid%3Dliteratum%253Aachs%26aulast%3DLovell%26aufirst%3DS.%2BC.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3Dde%2BBakker%26aufirst%3DP.%2BI.%26aulast%3DWord%26aufirst%3DJ.%2BM.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DStructure%2520Validation%2520by%2520Calpha%2520Geometry%253A%2520Phi%252C%2520Psi%2520and%2520Cbeta%2520Deviation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D50%26spage%3D437%26epage%3D450%26doi%3D10.1002%2Fprot.10286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">Training a Scoring Function for the Alignment of Small Molecules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1724</span>– <span class="NLM_lpage">1735</span>, <span class="refDoi"> DOI: 10.1021/ci100227h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100227h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFChs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1724-1735&author=S.+L.+Chanauthor=P.+Labute&title=Training+a+Scoring+Function+for+the+Alignment+of+Small+Molecules&doi=10.1021%2Fci100227h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Training a Scoring Function for the Alignment of Small Molecules</span></div><div class="casAuthors">Chan, Shek Ling; Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1724-1735</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A comprehensive data set of aligned ligands with highly similar binding pockets from the Protein Data Bank has been built.  Based on this data set, a scoring function for recognizing good alignment poses for small mols. has been developed.  This function is based on atoms and hydrogen-bond projected features.  The concept is simply that atoms and features of a similar type (hydrogen-bond acceptors/donors and hydrophobic) tend to occupy the same space in a binding pocket and atoms of incompatible types often tend to avoid the same space.  Comparison with some recently published results of small mol. alignments shows that the current scoring function can lead to performance better than those of several existing methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1T7YQyE40fLVg90H21EOLACvtfcHk0lgbrio8jaoPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFChs7nL&md5=97a3611b454a580b0fbc2db797344348</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fci100227h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100227h%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BL.%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DTraining%2520a%2520Scoring%2520Function%2520for%2520the%2520Alignment%2520of%2520Small%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1724%26epage%3D1735%26doi%3D10.1021%2Fci100227h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arceci, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. B.</span></span> <span> </span><span class="NLM_article-title">Mouse GATA-4: A Retinoic Acid-Inducible GATA-Binding Transcription Factor Expressed in Endodermally Derived Tissues and Heart</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2235</span>– <span class="NLM_lpage">2246</span>, <span class="refDoi"> DOI: 10.1128/MCB.13.4.2235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1128%2FMCB.13.4.2235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=8455608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Wmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1993&pages=2235-2246&author=R.+J.+Arceciauthor=A.+A.+Kingauthor=M.+C.+Simonauthor=S.+H.+Orkinauthor=D.+B.+Wilson&title=Mouse+GATA-4%3A+A+Retinoic+Acid-Inducible+GATA-Binding+Transcription+Factor+Expressed+in+Endodermally+Derived+Tissues+and+Heart&doi=10.1128%2FMCB.13.4.2235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse GATA-4: A retinoic acid-inducible GATA-binding transcription factor expressed in endodermally derived tissues and heart</span></div><div class="casAuthors">Arceci, Robert J.; King, Alexandra A. J.; Simon, M. Celeste; Orkin, Stuart H.; Wilson, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2235-46</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">The authors report the cDNA cloning and characterization of mouse GATA-4, a new member of the family of zinc finger transcription factors that bind a core GATA motif.  GATA-4 cDNA was identified by screening a 6.5-day mouse embryo library with oligonucleotide probes corresponding to a highly conserved region of the finger domains.  Like other proteins of the family, GATA-4 is approx. 50 kDa in size and contains two zinc finger domains of the form C-X-N-C-(X17)-C-N-X-C.  Cotransfection assays in heterologous cells demonstrate that GATA-4 trans activates reporter constructs contg. GATA promoter elements.  Northern (RNA) anal. and in situ hybridization show that GATA-4 mRNA is expressed in the heart, intestinal epithelium, primitive endoderm, and gonads.  Retinoic acid-induced differentiation of mouse F9 cells into visceral or parietal endoderm is accompanied by increased expression of GATA-4 mRNA and protein.  In vitro differentiation of embryonic stem cells into embryoid bodies is also assocd. with increased GATA-4 expression.  The authors conclude that GATA-4 is a tissue-specific, retinoic acid-inducible, and developmentally regulated transcription factor.  On the basis of its tissue distribution, it is speculated that GATA-4 plays a role in gene expression in the heart, intestinal epithelium, primitive endoderm, and gonads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo00Zg1oUuGGrVg90H21EOLACvtfcHk0lgbrio8jaoPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1Wmurc%253D&md5=d7bfb22717e981fa25b0d004f620d8d7</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1128%2FMCB.13.4.2235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.13.4.2235%26sid%3Dliteratum%253Aachs%26aulast%3DArceci%26aufirst%3DR.%2BJ.%26aulast%3DKing%26aufirst%3DA.%2BA.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DWilson%26aufirst%3DD.%2BB.%26atitle%3DMouse%2520GATA-4%253A%2520A%2520Retinoic%2520Acid-Inducible%2520GATA-Binding%2520Transcription%2520Factor%2520Expressed%2520in%2520Endodermally%2520Derived%2520Tissues%2520and%2520Heart%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1993%26volume%3D13%26spage%3D2235%26epage%3D2246%26doi%3D10.1128%2FMCB.13.4.2235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ranganayakulu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">Divergent Roles for NK-2 Class Homeobox Genes in Cardiogenesis in Flies and Mice</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">3037</span>– <span class="NLM_lpage">3048</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1998&pages=3037-3048&author=G.+Ranganayakuluauthor=D.+A.+Elliottauthor=R.+P.+Harveyauthor=E.+N.+Olson&title=Divergent+Roles+for+NK-2+Class+Homeobox+Genes+in+Cardiogenesis+in+Flies+and+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRanganayakulu%26aufirst%3DG.%26aulast%3DElliott%26aufirst%3DD.%2BA.%26aulast%3DHarvey%26aufirst%3DR.%2BP.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DDivergent%2520Roles%2520for%2520NK-2%2520Class%2520Homeobox%2520Genes%2520in%2520Cardiogenesis%2520in%2520Flies%2520and%2520Mice%26jtitle%3DDevelopment%26date%3D1998%26volume%3D125%26spage%3D3037%26epage%3D3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hakkola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingelman-Sundberg, M.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Down-Regulation of CYP2E1 Expression by Inflammatory Cytokines in Rat Hepatoma Cells</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1048</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1124/jpet.102.041582</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1124%2Fjpet.102.041582" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=12604681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1eisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2003&pages=1048-1054&author=J.+Hakkolaauthor=Y.+Huauthor=M.+Ingelman-Sundberg&title=Mechanisms+of+Down-Regulation+of+CYP2E1+Expression+by+Inflammatory+Cytokines+in+Rat+Hepatoma+Cells&doi=10.1124%2Fjpet.102.041582"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of down-regulation of CYP2E1 expression by inflammatory cytokines in rat hepatoma cells</span></div><div class="casAuthors">Hakkola, Jukka; Hu, Yin; Ingelman-Sundberg, Magnus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1048-1055</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">CYP2E1 is one of the major cytochrome P 450 forms whose expression is strongly inhibited by inflammatory cytokines in humans and rodents.  In the present study, the authors have used the Fao rat hepatoma cell line that constitutively expresses CYP2E1 enzyme to investigate mechanisms of cytokine action.  The cells were treated with interleukin (IL)-1β, tumor necrosis factor-α (TNFα), or IL-6 for 24 or 72 h, and the expression of CYP2E1 was monitored at the transcriptional, mRNA, and protein levels.  All three cytokines decreased the CYP2E1 mRNA levels after 24 h, and the effect was even stronger after 72 h.  In contrast, significant inhibition of CYP2E1 protein was seen only after 72 h.  In transfection assays using a CYP2E1 5' -3685 to +29-luciferase construct, it was found that IL-6 inhibited gene transcription after 24 h, but a similar effect by IL-1β and TNFα was registered only after 72 h.  Using 5' deletions of the CYP2E1 5'-reporter construct a responsive region for the IL-6 effect was located to -669 to -507 base pairs in the CYP2E1 5'-flanking region.  Interestingly, IL-1β, but not TNFα, was found to reduce hepatocyte nuclear factor (HNF)-1α binding to the CYP2E1 promotor.  However, the transactivation function of HNF-1α was impaired in Fao cells.  In mouse primary hepatocytes, IL-1β decreased HNF-1α-mediated transactivation.  In conclusion, the authors' data indicate that inflammatory cytokines inhibit CYP2E1 expression by multiple mechanisms, including control of HNF-1α function and regulation of other transcriptional factors acting on the CYP2E1 5'-upstream regulatory region.  In addn., regulation of factors of importance for the CYP2E1 mRNA stability may be involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrShCTi2lN2V7Vg90H21EOLACvtfcHk0lgbrio8jaoPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1eisr8%253D&md5=135375e135301578945a5836e83ad640</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.041582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.041582%26sid%3Dliteratum%253Aachs%26aulast%3DHakkola%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DIngelman-Sundberg%26aufirst%3DM.%26atitle%3DMechanisms%2520of%2520Down-Regulation%2520of%2520CYP2E1%2520Expression%2520by%2520Inflammatory%2520Cytokines%2520in%2520Rat%2520Hepatoma%2520Cells%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2003%26volume%3D304%26spage%3D1048%26epage%3D1054%26doi%3D10.1124%2Fjpet.102.041582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strauss, J. F.</span></span> <span> </span><span class="NLM_article-title">Activation of the Control Reporter Plasmids pRL-TK and pRL-SV40 by Multiple GATA Transcription Factors Can Lead to Aberrant Normalization of Transfection Efficiency</span>. <i>BMC Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.1186/1472-6750-4-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1186%2F1472-6750-4-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=15119957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A280%3ADC%252BD2c3mtFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=10&author=C.+K.+M.+Hoauthor=J.+F.+Strauss&title=Activation+of+the+Control+Reporter+Plasmids+pRL-TK+and+pRL-SV40+by+Multiple+GATA+Transcription+Factors+Can+Lead+to+Aberrant+Normalization+of+Transfection+Efficiency&doi=10.1186%2F1472-6750-4-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the control reporter plasmids pRL-TK and pRL-SV40 by multiple GATA transcription factors can lead to aberrant normalization of transfection efficiency</span></div><div class="casAuthors">Ho Clement K M; Strauss Jerome F 3rd</div><div class="citationInfo"><span class="NLM_cas:title">BMC biotechnology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Members of the GATA transcription factor family have been used in many transfection studies to investigate their roles in the regulation of gene expression.  To correct for variations in transfection efficiency, the Renilla luciferase encoding plasmids pRL-TK and pRL-SV40 are commonly used as normalization controls.  RESULTS:  We report here that plasmids expressing GATA-4 or GATA-6 transcription factor increased Renilla luciferase gene expression by 2 to 8 fold when co-transfected with pRL-TK or pRL-SV40.  This alteration of the control reporter gene activity was shown to cause erroneous normalization of transfection efficiency and thus misinterpretation of results in a transactivation assay.  To circumvent the problem, we generated two mutant control plasmids from which putative GATA response elements were deleted.  These deletions rendered pRL-SV40 unresponsive to GATA transcription factor stimulation and reduced the response of pRL-TK.  A database search also indicates that consensus GATA binding sequences are present in other commercially available Renilla luciferase encoding plasmids; therefore, the latter can potentially be transactivated by GATA transcription factors.  CONCLUSION:  Taken together, these findings highlight the importance of the selection of an appropriate control reporter plasmid for the normalization of transfection efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwZIWRsIIvYxjv2lLSLOxFfW6udTcc2ebplcCH2n4xJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3mtFKmsA%253D%253D&md5=e5eb0001ac6ffa79b777131644af3fb1</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1186%2F1472-6750-4-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6750-4-10%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DC.%2BK.%2BM.%26aulast%3DStrauss%26aufirst%3DJ.%2BF.%26atitle%3DActivation%2520of%2520the%2520Control%2520Reporter%2520Plasmids%2520pRL-TK%2520and%2520pRL-SV40%2520by%2520Multiple%2520GATA%2520Transcription%2520Factors%2520Can%2520Lead%2520to%2520Aberrant%2520Normalization%2520of%2520Transfection%2520Efficiency%26jtitle%3DBMC%2520Biotechnol.%26date%3D2004%26volume%3D4%26spage%3D10%26doi%3D10.1186%2F1472-6750-4-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerkela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikkarainen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majalahti-Palviainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokola, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruskoaho, H.</span></span> <span> </span><span class="NLM_article-title">Distinct Roles of Mitogen-Activated Protein Kinase Pathways in GATA-4 Transcription Factor-Mediated Regulation of B-Type Natriuretic Peptide Gene</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">13752</span>– <span class="NLM_lpage">13760</span>, <span class="refDoi"> DOI: 10.1074/jbc.M105736200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1074%2Fjbc.M105736200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=13752-13760&author=R.+Kerkelaauthor=S.+Pikkarainenauthor=T.+Majalahti-Palviainenauthor=H.+Tokolaauthor=H.+Ruskoaho&title=Distinct+Roles+of+Mitogen-Activated+Protein+Kinase+Pathways+in+GATA-4+Transcription+Factor-Mediated+Regulation+of+B-Type+Natriuretic+Peptide+Gene&doi=10.1074%2Fjbc.M105736200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105736200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105736200%26sid%3Dliteratum%253Aachs%26aulast%3DKerkela%26aufirst%3DR.%26aulast%3DPikkarainen%26aufirst%3DS.%26aulast%3DMajalahti-Palviainen%26aufirst%3DT.%26aulast%3DTokola%26aufirst%3DH.%26aulast%3DRuskoaho%26aufirst%3DH.%26atitle%3DDistinct%2520Roles%2520of%2520Mitogen-Activated%2520Protein%2520Kinase%2520Pathways%2520in%2520GATA-4%2520Transcription%2520Factor-Mediated%2520Regulation%2520of%2520B-Type%2520Natriuretic%2520Peptide%2520Gene%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D13752%26epage%3D13760%26doi%3D10.1074%2Fjbc.M105736200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamprecht, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guertin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">CellProfiler: Image Analysis Software for Identifying and Quantifying Cell Phenotypes</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">R100</span>, <span class="refDoi"> DOI: 10.1186/gb-2006-7-10-r100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1186%2Fgb-2006-7-10-r100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=17076895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=R100&author=A.+E.+Carpenterauthor=T.+R.+Jonesauthor=M.+R.+Lamprechtauthor=C.+Clarkeauthor=I.+H.+Kangauthor=O.+Frimanauthor=D.+A.+Guertinauthor=J.+H.+Changauthor=R.+A.+Lindquistauthor=J.+Moffatauthor=P.+Gollandauthor=D.+M.+Sabatini&title=CellProfiler%3A+Image+Analysis+Software+for+Identifying+and+Quantifying+Cell+Phenotypes&doi=10.1186%2Fgb-2006-7-10-r100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">CellProfiler: image analysis software for identifying and quantifying cell phenotypes</span></div><div class="casAuthors">Carpenter Anne E; Jones Thouis R; Lamprecht Michael R; Clarke Colin; Kang In Han; Friman Ola; Guertin David A; Chang Joo Han; Lindquist Robert A; Moffat Jason; Golland Polina; Sabatini David M</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">R100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Biologists can now prepare and image thousands of samples per day using automation, enabling chemical screens and functional genomics (for example, using RNA interference).  Here we describe the first free, open-source system designed for flexible, high-throughput cell image analysis, CellProfiler.  CellProfiler can address a variety of biological questions quantitatively, including standard assays (for example, cell count, size, per-cell protein levels) and complex morphological assays (for example, cell/organelle shape or subcellular patterns of DNA or protein staining).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkZz-phUrfiVm8bhe6roZYfW6udTcc2ebplcCH2n4xJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D&md5=79d0e947559e12792f864d764657d3f4</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1186%2Fgb-2006-7-10-r100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2006-7-10-r100%26sid%3Dliteratum%253Aachs%26aulast%3DCarpenter%26aufirst%3DA.%2BE.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DLamprecht%26aufirst%3DM.%2BR.%26aulast%3DClarke%26aufirst%3DC.%26aulast%3DKang%26aufirst%3DI.%2BH.%26aulast%3DFriman%26aufirst%3DO.%26aulast%3DGuertin%26aufirst%3DD.%2BA.%26aulast%3DChang%26aufirst%3DJ.%2BH.%26aulast%3DLindquist%26aufirst%3DR.%2BA.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DGolland%26aufirst%3DP.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DCellProfiler%253A%2520Image%2520Analysis%2520Software%2520for%2520Identifying%2520and%2520Quantifying%2520Cell%2520Phenotypes%26jtitle%3DGenome%2520Biol.%26date%3D2006%26volume%3D7%26spage%3DR100%26doi%3D10.1186%2Fgb-2006-7-10-r100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamentsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljosa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A. E.</span></span> <span> </span><span class="NLM_article-title">Improved Structure, Function and Compatibility for CellProfiler: Modular High-Throughput Image Analysis Software</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1179</span>– <span class="NLM_lpage">1180</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=10.1093%2Fbioinformatics%2Fbtr095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=21349861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1179-1180&author=L.+Kamentskyauthor=T.+R.+Jonesauthor=A.+Fraserauthor=M.-A.+Brayauthor=D.+J.+Loganauthor=K.+L.+Maddenauthor=V.+Ljosaauthor=C.+Ruedenauthor=K.+W.+Eliceiriauthor=A.+E.+Carpenter&title=Improved+Structure%2C+Function+and+Compatibility+for+CellProfiler%3A+Modular+High-Throughput+Image+Analysis+Software&doi=10.1093%2Fbioinformatics%2Fbtr095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Improved structure, function and compatibility for CellProfiler: modular high-throughput image analysis software</span></div><div class="casAuthors">Kamentsky, Lee; Jones, Thouis R.; Fraser, Adam; Bray, Mark-Anthony; Logan, David J.; Madden, Katherine L.; Ljosa, Vebjorn; Rueden, Curtis; Eliceiri, Kevin W.; Carpenter, Anne E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1179-1180</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: There is a strong and growing need in the biol. research community for accurate, automated image anal.  Here, we describe CellProfiler 2.0, which has been engineered to meet the needs of its growing user base.  It is more robust and user friendly, with new algorithms and features to facilitate high-throughput work.  ImageJ plugins can now be run within a CellProfiler pipeline.  Availability and Implementation: CellProfiler 2.0 is free and open source, available at http://www.cellprofiler.org under the GPL v. 2 license.  It is available as a packaged application for Macintosh OS X and Microsoft Windows and can be compiled for Linux.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="8cede2e2e9cceefee3ede8e5e2fff8e5f8f9f8e9a2e3feeb">[email protected]</a> Supplementary information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa5_31IFp6crVg90H21EOLACvtfcHk0ljw7okbhyGhVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKlu78%253D&md5=5f99adf6bdae740ed203dfe4adbb6396</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr095%26sid%3Dliteratum%253Aachs%26aulast%3DKamentsky%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DFraser%26aufirst%3DA.%26aulast%3DBray%26aufirst%3DM.-A.%26aulast%3DLogan%26aufirst%3DD.%2BJ.%26aulast%3DMadden%26aufirst%3DK.%2BL.%26aulast%3DLjosa%26aufirst%3DV.%26aulast%3DRueden%26aufirst%3DC.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26aulast%3DCarpenter%26aufirst%3DA.%2BE.%26atitle%3DImproved%2520Structure%252C%2520Function%2520and%2520Compatibility%2520for%2520CellProfiler%253A%2520Modular%2520High-Throughput%2520Image%2520Analysis%2520Software%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D1179%26epage%3D1180%26doi%3D10.1093%2Fbioinformatics%2Fbtr095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DFX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DFX','PDB','3DFX'); return false;">PDB: 3DFX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTT','PDB','1FTT'); return false;">PDB: 1FTT</a></li><li><a href="http://www.ncbi.nlm.nih.gov/gquery?Term=M60266" onclick="trackOutboundLink('http://www.ncbi.nlm.nih.gov/gquery?Term=M60266','Genbank','M60266'); return false;">Genbank: M60266</a></li><li><a href="http://www.ncbi.nlm.nih.gov/gquery?Term=M27498" onclick="trackOutboundLink('http://www.ncbi.nlm.nih.gov/gquery?Term=M27498','Genbank','M27498'); return false;">Genbank: M27498</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i58"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00816">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_07186"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00816">10.1021/acs.jmedchem.7b00816</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures and tables illustrating compounds aggregation, chemical stability, effects on GATA4 transcriptional activity, the protein kinases and the GPCRs, coimmunoprecipitation assay results, and the original EMSA gel samples (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_001.pdf">jm7b00816_si_001.pdf (28.59 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_002.pdb">jm7b00816_si_002.pdb (94.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00816/suppl_file/jm7b00816_si_003.csv">jm7b00816_si_003.csv (5.43 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00816&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00816%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-18%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00816" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b4804bd843ca6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
