
<html lang="en"     class="pb-page"  data-request-id="6f7d3c58-d691-43aa-8e00-849f69df7aa9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2018.61.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.7b01091"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Novel Potent VEGFR-2 Inhibitors Exerting Significant Antiproliferative Activity against Cancer Cell Lines" /></meta><meta name="dc.Creator" content="Yanmin  Zhang" /></meta><meta name="dc.Creator" content="Yadong  Chen" /></meta><meta name="dc.Creator" content="Danfeng  Zhang" /></meta><meta name="dc.Creator" content="Lu  Wang" /></meta><meta name="dc.Creator" content="Tao  Lu" /></meta><meta name="dc.Creator" content="Yu  Jiao" /></meta><meta name="dc.Description" content="Computational and experimental studies were applied to the discovery of a series of novel vascular endothelial growth factor receptor 2 (VEGFR-2) inhibitors. Eight compounds exhibited nanomolar IC5..." /></meta><meta name="Description" content="Computational and experimental studies were applied to the discovery of a series of novel vascular endothelial growth factor receptor 2 (VEGFR-2) inhibitors. Eight compounds exhibited nanomolar IC5..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 19, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01091" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01091" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01091" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01091" /></link>
        
    
    

<title>Discovery of Novel Potent VEGFR-2 Inhibitors Exerting Significant Antiproliferative Activity against Cancer Cell Lines | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01091" /></meta><meta property="og:title" content="Discovery of Novel Potent VEGFR-2 Inhibitors Exerting Significant Antiproliferative Activity against Cancer Cell Lines" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0018.jpeg" /></meta><meta property="og:description" content="Computational and experimental studies were applied to the discovery of a series of novel vascular endothelial growth factor receptor 2 (VEGFR-2) inhibitors. Eight compounds exhibited nanomolar IC50 values against VEGFR-2, and compounds 6, 19, 22, and 23 showed potent antiproliferative effects against several cell lines. Particularly, compound 23 behaved better than FDA approved drugs, sorafenib and sunitinib, in antiproliferative activity against cell lines related to all nine tumor types tested (GI50 values), and it was better or comparable in safety (LC50 values). Compound 23 even demonstrated a high potency on one of the drug-resistant cell lines (NCI/ADR-RES) responsible for ovarian cancer and cell lines contributing to prostate cancer, regarded as one of the VEGF/VEGFR pathway drug-resistant tumors. This compound is likely a promising candidate for the treatment of leukemia, non-small cell lung cancer (NSCLC), colon cancer, ovarian cancer, and breast cancer with a suitable balance of both efficacy and safety." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01091"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01091">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01091&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01091&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01091&amp;href=/doi/10.1021/acs.jmedchem.7b01091" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 140-157</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01086" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01138" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Novel Potent VEGFR-2 Inhibitors Exerting Significant Antiproliferative Activity against Cancer Cell Lines</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yanmin++Zhang">Yanmin Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yadong++Chen">Yadong Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Danfeng++Zhang">Danfeng Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lu++Wang">Lu Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tao++Lu">Tao Lu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Jiao">Yu Jiao</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-1900-8523" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Laboratory of Molecular Design and Drug Discovery, School of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">School of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">State Key Laboratory of Natural Medicines, China Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, China</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#38544d4c5957785b484d165d5c4d165b56"><span class="__cf_email__" data-cfemail="2f435a5b4e406f4c5f5a014a4b5a014c41">[email protected]</span></a>. Tel: +86-25-86185180. Fax: +86-25-86185179.</div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#412b28202e3834012231346f2425346f222f"><span class="__cf_email__" data-cfemail="167c7f77796f635675666338737263387578">[email protected]</span></a>. Tel: +86-25-86185201. Fax: +86-25-86185182.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01091&amp;href=/doi/10.1021%2Facs.jmedchem.7b01091" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 140–157</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 30, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 July 2017</li><li><span class="item_label"><b>Published</b> online</span>19 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 January 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01091" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01091</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D140%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYanmin%2BZhang%252C%2BYadong%2BChen%252C%2BDanfeng%2BZhang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D1%26contentID%3Dacs.jmedchem.7b01091%26title%3DDiscovery%2Bof%2BNovel%2BPotent%2BVEGFR-2%2BInhibitors%2BExerting%2BSignificant%2BAntiproliferative%2BActivity%2Bagainst%2BCancer%2BCell%2BLines%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D157%26publicationDate%3DJanuary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01091"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3174</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">34</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01091" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Novel Potent VEGFR-2 Inhibitors Exerting Significant Antiproliferative Activity against Cancer Cell Lines&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yanmin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yadong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Danfeng&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Lu&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Tao&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Jiao&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;140-157&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01091&quot;},&quot;abstract&quot;:&quot;Computational and experimental studies were applied to the discovery of a series of novel vascular endothelial growth factor receptor 2 (VEGFR-2) inhibitors. Eight compounds exhibited nanomolar IC50 values against VEGFR-2, and compounds 6, 19, 22, and 23 showed potent antiproliferative effects against several cell lines. Particularly, compound 23 behaved better than FDA approved drugs, sorafenib and sunitinib, in antiproliferative activity against cell lines related to all nine tumor types tested (GI50 values), and it was better or comparable in safety (LC50 values). Compound 23 even demonstrated a high potency on one of the drug-resistant cell lines (NCI/ADR-RES) responsible for ovarian cancer and cell lines contributing to prostate cancer, regarded as one of the VEGF/VEGFR pathway drug-resistant tumors. This compound is likely a promising candidate for the treatment of leukemia, non-small cell lung cancer (NSCLC), colon cancer, ovarian cancer, and breast cancer with a suitable balance of both efficacy and s&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01091&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01091" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01091&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01091" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01091&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01091" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01091&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01091&amp;href=/doi/10.1021/acs.jmedchem.7b01091" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01091" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01091" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01091%26sid%3Dliteratum%253Aachs%26pmid%3D29189002%26genre%3Darticle%26aulast%3DZhang%26date%3D2018%26atitle%3DDiscovery%2Bof%2BNovel%2BPotent%2BVEGFR-2%2BInhibitors%2BExerting%2BSignificant%2BAntiproliferative%2BActivity%2Bagainst%2BCancer%2BCell%2BLines%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D1%26spage%3D140%26epage%3D157%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/jmcmar.2018.61.issue-1/20180111/jmcmar.2018.61.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Computational and experimental studies were applied to the discovery of a series of novel vascular endothelial growth factor receptor 2 (VEGFR-2) inhibitors. Eight compounds exhibited nanomolar IC<sub>50</sub> values against VEGFR-2, and compounds <b>6</b>, <b>19</b>, <b>22</b>, and <b>23</b> showed potent antiproliferative effects against several cell lines. Particularly, compound <b>23</b> behaved better than FDA approved drugs, sorafenib and sunitinib, in antiproliferative activity against cell lines related to all nine tumor types tested (GI<sub>50</sub> values), and it was better or comparable in safety (LC<sub>50</sub> values). Compound <b>23</b> even demonstrated a high potency on one of the drug-resistant cell lines (NCI/ADR-RES) responsible for ovarian cancer and cell lines contributing to prostate cancer, regarded as one of the VEGF/VEGFR pathway drug-resistant tumors. This compound is likely a promising candidate for the treatment of leukemia, non-small cell lung cancer (NSCLC), colon cancer, ovarian cancer, and breast cancer with a suitable balance of both efficacy and safety.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Angiogenesis, the formation of new blood vessels by sprouting or splitting from pre-existing vessels for cell proliferation, migration, and survival,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> plays a pivotal role in both normal embryonic and adult development.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2-4)</a> However, angiogenic abnormalities are observed in numerous pathological conditions such as cancer, psoriasis, rheumatoid arthritis, inflammation, and retinal complications.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The specific inhibition of signal transduction via the system containing vascular endothelial growth factor (VEGF) and its receptor (VEGFR) is a promising approach to starve tumor cells of nutrients and thus impede tumor growth<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and metastasis.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> The VEGFR family, VEGFR-1 (Flt-1), VEGFR-2 [Flk-1/kinase domain receptor (KDR)], and VEGFR-3 (Flt-4), which are cell surface tyrosine kinase receptors, is located on the host vascular endothelium, lymphatic, and hematopoietic systems, and these family members are major proteins modulating angiogenesis by transducing VEGF-A, VEGF-B, VEGF-C, VEGF-D, and placental growth factor (PlGF) signals to downstream pathways.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> Specifically, VEGFR-2 is regarded as one of the central targets for discovering inhibitors against tumor-associated angiogenesis, as it is responsible for the transduction of the main proangiogenic signal for vasculature.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Because most of the current VEGFR-2 therapeutics are ATP competitive inhibitors, they exert their effect by occupying the highly conserved ATP pocket.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Moreover, small-molecule kinase inhibitors have recently seen burgeoning growth with 28 drugs gaining approval by the US Food and Drug Administration (FDA) for the treatment of cancers, such as advanced renal cell carcinoma and hepatocellular carcinoma.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Among these drugs, nine (sorafenib, sunitinib, pazopanib, vandetanib, axitinib, regorafenib, cabozantinib, nintedanib, lenvatinib, and apatinib) involve VEGFR-2 inhibition, demonstrating the clinical benefits of VEGFR-2 inhibitors.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a></div><div class="NLM_p">However, both intrinsic and acquired resistance are major contributors to transient clinical benefits and failure of antiangiogenic drugs in the clinic.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> Several mechanisms have been proposed for the drug resistance to VEGF blockade seen in some cancer patients, and these mechanisms are not mutually exclusive. For example, (1) in established tumors, the inhibition of VEGF can aggravate hypoxia, resulting in upregulation of the production of other angiogenic factors or increase in tumor cell invasiveness. (2) Tumor cells with other acquired mutations can also become hypoxia tolerant. (3) Tumor vessels covered by pericytes are generally less sensitive to VEGF inhibition. (4) Tumor vascularization begins to include intussusception, vasculogenic mimicry and vessel growth, differentiation of putative cancer stem cells into endothelial cells, existing vessel co-option, etc. These changes can be rescued by recruited macrophages, proangiogenic bone-marrow-derived cells, or activated cancer-associated fibroblasts through producing proangiogenic factors. Regarding safety, VEGFR inhibitors all share mechanism-related toxicities, of which the most common are hypertension, proteinuria, thromboembolism, hemorrhage, fistula formation, bowel perforation, and reversible posterior leukoencephalopathy.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">Generally, resistance to numerous other drugs by targeting specific genetic mutations often leads to a stable, heritable, acquired drug-resistant phenotype. In contrast, to combat acquiring resistance to antiangiogenic VEGF pathway-targeting drugs, one interesting consideration is that the evasive/acquired resistance is potentially reversible (epigenetic), not necessarily stable, and hence presumably immutable with host endothelial cells lining new tumor blood vessels.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> This may help explain why some anti-VEGF therapies can be administered for long periods as maintenance therapy even beyond disease progression.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> For example, a therapeutic benefit is observed when switching VEGFR tyrosine kinase inhibitors (TKIs) to second-line alternate antiangiogenic drugs in patients with renal cell carcinoma.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Regarding potential reversible resistance, initial TKIs might still be effective and should be considered in future clinical trials assessing subsequent-line therapies. Thus, the discovery of novel VEGFR-2 inhibitors with satisfactory properties that can surmount drug resistance and decrease toxicity remain a substantial challenge.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a></div><div class="NLM_p">In this work, computational and experimental studies were applied to the discovery of a series of novel VEGFR-2 inhibitors (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). First, BREED,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> a ligand hybridization methodology belonging to fragment-based drug design (FBDD), was conducted on the ligands from X-ray crystallographic structures of VEGFR-2, selected hit fragments from previous work<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and the KLIFS database.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Then, molecular docking was used to screen the generated database. Based on of the abundance of crystal structure data available, a robust linear model for the relationship between predicted binding free energy (Δ<i>G</i>) calculated by Molecular Mechanics/Poisson–Boltzmann Surface Area (MM/PBSA) and enzyme inhibitory activity (such as IC<sub>50</sub> values) was constructed after a 20 ns molecular dynamics (MD) simulation. Then, potential compounds selected by molecular docking were submitted for MD simulation and scored by the linear model. Ten compounds were ultimately chosen for a synthesis and biological activity evaluation. Those that exhibited favorable enzymatic activity and antiproliferative effects on human umbilical vein endothelial cells (HUVECs) and melanoma cell line MDA-MB-435 were submitted to the National Cancer Institute (NCI)-60 platform. The NCI testing measured the growth percent in 58 cell lines from nine different tumor types, one of which is antiangiogenic drug-resistant prostate cancer. Finally, the kinase activity profile of the most effective compound was investigated.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Workflow for this investigation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14025" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14025" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Binding Site Analysis and SAR Summarization</h3><div class="NLM_p">Previously, we constructed a three-dimensional quantitative structure activity relationship (3D-QSAR) model and summarized a SAR map for VEGFR-2.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Combined with molecular docking and sitemap analysis,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> the binding site of VEGFR-2 can be divided into four important regions (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). Sorafenib, for example, binds to the active site cavity of VEGFR-2 in a DFG-out (inactive) conformation, which enables the appropriately substituted inhibitor to penetrate into the extended hydrophobic pocket. The <i>N</i>-methyl picolinamide/<i>N</i>-methyl-2-pyridinecarboxamide part binds to the hinge region by forming two hydrogen bonds with the backbone amide NH and carbonyl oxygen of Cys919, which are the predominant forces that maintain inhibitor activity. The hydrogen bond-rich region indicates where hydrogen acceptors or donors are favorable. An effective inhibitor (here the amide group of sorafenib) will form two to three hydrogen bonds with the side chain of Glu885 and the backbone NH of Asp1046, two residues that have also been implicated as important contributors to both hydrogen bonding and nonbonding interactions for inhibitor binding by FTMap analysis.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> However, we focused instead on the two important hydrophobic regions. Hydrophobic region I, occupied by the central phenyl ring in sorafenib, is surrounded by several hydrophobic amino acids, including Ala866, Leu840, Gly841, and Ile915. As shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, this region, probed by SiteMap (yellow dots), can accommodate monocyclic or bicyclic ring structures, which may provide space for additional modification. Hydrophobic region II is occupied by the 4-chloro-3-(trifluoromethyl)benzene moiety and is surrounded by Ile892, Ile888, Leu889, and Val898. This region extends to hydrophobic region II and bypasses the DFG moiety, which is generally a determinant of inhibitor selectivity.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding site of VEGFR-2 in complex with sorafenib (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">BREED</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40186" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40186" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Herein, in addition to the 28 crystal structures of VEGFR-2,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> selected hit fragments from previous work<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and 1100 human kinase-ligand cocrystal complexes of various targets from the KLIFS database (<a href="http://www.vu-compmedchem.nl" class="extLink">http://www.vu-compmedchem.nl</a>)<a onclick="showRef(event, 'ref24 ref29'); return false;" href="javascript:void(0);" class="ref ref24 ref29">(24, 29)</a> were aligned. Because most kinases possess similar binding sites, approaching this dilemma by combining the available data may expand the possibility of chemical space for novel inhibitor scaffolds. <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a illustrates one example, where the two known VEGFR-2 inhibitors, vandetanib (IC<sub>50</sub> 40 nM) and sorafenib (IC<sub>50</sub> 90 nM), (activity data from <a href="http://www.selleckchem.com/" class="extLink">http://www.selleckchem.com/</a>) and a hit fragment, Hit_Scaffold_4 (Bayesian Score 35.6), from our previous study<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> were overlapped and swapped to generate hybridized compound <b>6</b>, which was synthesized and had 15.3 nM enzymatic activity against VEGFR-2. As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b, the fragments of compound <b>6</b> adopt conformations remarkably similar to the original fragments extracted from crystal structures with the two known inhibitors and the hit fragment when the BREED-generated model was overlaid within the active site of the protein. The root-mean-square deviation (RMSD) values for the 6,7-dimethoxyquinazoline group, the naphthylamine portion, and the 1-(4-chloro-3-(trifluoromethyl)phenyl)-3-methylurea moiety were 0.046, 0.037, and 0.054 Å, respectively, sufficiently demonstrating the reliability of the BREED method. A total of 1531 novel compounds were obtained after duplicate removal, and all were submitted for further molecular docking analysis (in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_002.zip" class="ext-link">Supporting Information</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) The combinatorial approach of BREED. (b) Different colors were used to denote the scaffolds from different input structures: the 6,7-dimethoxyquinazoline group from vandetanib (green), the naphthamide portion of Hit_Scaffold_4 (dark pink), and the 1-(4-chloro-3-(trifluoromethyl)phenyl)-3-methylurea moiety (blue). Compound <b>6</b> (Cmpd <b>6</b>) is colored cyan, and important active site residues are labeled and colored magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> MD Linear Model and Binding Affinity Prediction</h3><div class="NLM_p">A set of 28 crystallographic structures (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Table S1</a>) from the Protein Data Bank (PDB)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> database were used to construct the binding affinity prediction model. Of the crystallized ligands, 24 have low nanomolar (nM) IC<sub>50</sub> values (less than 100 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Table S1</a>) and they contain different structural scaffolds that sample diverse chemical space. For each protein, a 20 ns MD simulation was performed to ensure that the dynamic equilibrium was attained (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). Snapshots were extracted from the last 5 ns at every 10 ps interval. A total of 500 snapshots were obtained, and each one was used to calculate a binding free energy; then an average of these 500 values was used. Next, the average binding free energy was converted to binding affinity by the following formula: IC<sub>50</sub> = exp[Δ<i>G</i>/(<i>RT</i>)], where <i>RT</i> is approximately 0.59 kcal/mol. (We used the IC<sub>50</sub> values instead.) A linear model was built between the computational and experimental binding free energies with a Pearson correlation coefficient (<i>R</i>) of 0.8 and a p-value of about 4 × 10<sup>–6</sup> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a), suggesting the robustness of the model.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding affinity prediction model (a) and the average binding affinity of both ligands from the crystal structures and the selected compounds. (b) Delta G means Δ<i>G</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a, this model (<i>y</i> = 3.3 × <i>x</i> −11, where <i>x</i> represents the experimental binding free energy (experimental Δ<i>G</i>) and <i>y</i> represents the calculated binding free energy (calculated Δ<i>G</i>)) was used to predict the 100 compounds generated by BREED that were selected after prescreening by molecular docking. Ten compounds with the highest predicted Δ<i>G</i> values (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) were submitted for synthesis and biological activity evaluation. Because pan assay interference (PAINS) compounds are often false positives reacting with numerous targets, they tend to react with numerous biological targets simultaneously rather than specifically affecting one desired target.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Thus, before further evaluation, our ten compounds were filtered through the PAINS remover (<a href="http://cbligand.org/PAINS/" class="extLink">http://cbligand.org/PAINS/</a>),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and all of them passed the filtration process. The average binding free energy values for the 10 selected compounds were better (lower) or comparable to those of the ligands from the crystal structures (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Table S1</a> and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Enzymatic Activity of Selected Compounds against VEGFR-2 as well as Glide XP Scores and Predicted Δ<i>G</i> Values Obtained by Molecular Dynamics Combined with MMPBSA<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0016.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values were the average of three replicates.</p></div></div><div></div></div><div class="NLM_p">To ensure that the simulation had reached equilibrium, RMSD and the number of protein–ligand hydrogen bonds were calculated for both the crystallized ligands and the designed compounds. The RMSD values for both the protein (<a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S1, and S3</a>, blue) and the ligands (<a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S1, and S3</a>, orange) in the 28 crystal structures and 10 target compounds were calculated, respectively. In general, the protein atoms (RMSD of 1.5–4 Å) and the ligand atoms (RMSD of 0.25–2.5 Å) reached equilibrium at approximately 2000 ps. However, among all crystal structures, protein atoms from PDB entries 1Y6A, 2OH4, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P2I">2P2I</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DTW">3DTW</a> exhibited large fluctuations (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figure S1</a>). For the ligand atoms, the crystallized ligands of PDB entries 3C7Q, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QU6">2QU6</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGD">4AGD</a> fluctuated significantly (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figure S3</a>). In <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, the RMSD fluctuation of compound <b>6</b> presented a similar trend compared to the five FDA approved VEGFR-2 inhibitors including sorafenib (IC<sub>50</sub> = 90 nM), sunitinib (IC<sub>50</sub> = 80 nM), axitinib (IC<sub>50</sub> = 0.2 nM), nintedanib (IC<sub>50</sub> = 21 nM), and lenvatinib (IC<sub>50</sub> = 4 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Table S1</a>). Moreover, the RMSD values for the protein atoms (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>) in complex with the 10 designed compounds have lower RMSD values (2–3 Å) than the protein atoms for the 28 crystal complexes, except the complex with compound <b>21</b> (<a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S3</a>), which fluctuates from 2 to 4 Å. This observation is likely because only the most favorable crystal structure PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> was adopted for docking and MD simulations for the designed compounds. The ligand RMSD values (0.25–1.5 Å) of the designed compounds fluctuated less significantly (<a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S3</a>) than the crystallized ligands (0.25–2.5 Å) (<a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S1</a>). Additionally, the distribution of all Δ<i>G</i> values calculated from the 500 snapshots (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figure S4</a> for the crystal ligands and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figure S5</a> for the designed compounds) followed a normal distribution, which further proved that the MD simulation reached equilibrium and the rationality of using the average <i>ΔG</i> value to build the prediction model.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. RMSD values during the 20 ns MD for five FDA approved drugs crystallized with the VEGFR-2 protein and the designed compound <b>6</b> docked to the VEGFR-2 protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>). Blue represents the fluctuation of protein atoms, and orange represents the fluctuation of ligand atoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Hydrogen bonding plays a vital role in protein folding, stability, and function, and it is particularly important for the protein–ligand binding affinity. We calculated the number of intermolecular hydrogen bonds between the protein and the compounds with respect to time. On average, the designed compounds formed more hydrogen bonds (ranging from 1 to 6, with an average of 5) during the MD simulation compared to the crystallized ligands (ranging from 0 to 6, with an average of 3) (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S2 and S3</a>). Taken together, these data indicated that the VEGFR-2 protein in complex with the designed compounds can reach an equilibrium that was comparable or better in terms of RMSD, and the complexes maintain more hydrogen bonds during the simulation. Taken together, these results indicated an augmented protein stability and a stronger binding of the designed compounds.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Number of hydrogen bonds formed during 20 ns MD simulations for five FDA approved drugs crystallized with VEGFR-2 as well as designed compound <b>6</b> docked to the VEGFR-2 protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Chemistry</h3><div class="NLM_p">The preparation of target compounds <b>6</b>, <b>7</b>, <b>8</b>, <b>10</b>, and <b>11</b> is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>: <b>3a</b> and <b>3b</b> were obtained by the nucleophilic substitution reaction of two different phenols (<b>2a</b> and <b>2b</b>) and 4-chloro-6,7-dimethoxyquinazoline (<b>1</b>). Reduction of <b>3a</b> and <b>3b</b> by iron powder provided amino compounds <b>4a</b> and <b>4b</b>. Finally, <b>4a</b> was reacted with different aryl isocyanates (<b>5a</b> and <b>5b</b>) to obtain target urea compounds <b>6</b>, <b>7</b>, and <b>8</b>. <b>4b</b> was reacted with phenylsulfonyl chloride (<b>9a</b>) and 1-butanesulfonyl chloride (<b>9b</b>) to obtain sulfonamides <b>10</b> and <b>11</b>, respectively.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0014.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6</b>, <b>7</b>, <b>8</b>, <b>10</b>, and <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) anhydrous DMF, NaH, 90 °C, 24 h; (b) Fe, NH<sub>4</sub>Cl, 70% i-PrOH, 85 °C, 4 h; (c) THF, rt, 5 h; (d) THF, TEA, 0 °C to rt, 1 h/pyridine, rt, 1 h.</p></p></figure><div class="NLM_p">The preparation of target compounds <b>19</b>, <b>20</b>, <b>21</b>, <b>22</b>, and <b>23</b> is depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>: <b>4a</b> was reacted with 1-isopropyl-1<i>H</i>-pyrazol-4-amine (<b>14</b>), 5-methylisoxazole-3-amine, cyclopropylamine, 1-(4-aminophenyl)cyclopentane-1-carbonitrile, and 4-(morpholinomethyl)-3-(trifluoromethyl)aniline (<b>18</b>) to obtain compounds <b>19</b>, <b>20</b>, <b>21</b>, <b>22</b>, and <b>23</b>, respectively. The key intermediate <b>14</b> was prepared by Pd/C-catalyzed reduction of <b>13</b>, where <b>13</b> was obtained from nucleophilic substitution of 4-nitro-1<i>H</i>-pyrazole and 2-bromopropane. The key amine <b>18</b> was synthesized by hydrogenation of compound <b>17</b>, which was obtained by nucleophilic substitution of 4-nitro-2-(trifluoromethyl)benzyl bromide (<b>16</b>). Compound <b>16</b> was obtained from the bromination of 1-methyl-4-nitro-2-trifluorotoluene.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0015.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>19</b>, <b>20</b>, <b>21</b>, <b>22</b>, and <b>23</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (e) 2-bromopropane, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 0.5 h; (f) 20% Pd/C, H<sub>2</sub>, ethyl acetate, rt, 5–7 h; (g) NBS, AIBN, 1,2-dichloroethane, 90 °C, 12 h; (h) morpholine, K<sub>2</sub>CO<sub>3</sub>, rt, 16 h; (i) 20% Pd/C, H<sub>2</sub>, EtOH, rt, 2 h; (j) (<b>19</b>) 4-nitrophenyl carbonochloridate, DCM, RNH<sub>2</sub> pyridine, rt, 24 h; (<b>20</b>, <b>21</b>, <b>22</b>, and <b>23</b>) triphosgene, DCM, TEA, RNH<sub>2</sub>, 0 °C to rt, 24 h.</p></p></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> VEGFR-2 Kinase Activity</h3><div class="NLM_p">The inhibitory activity of the 10 target compounds was tested at 10 μM, and for the compounds with inhibition ratios greater than 50%, the VEGFR-2 IC<sub>50</sub> values were investigated. The activity results (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> and <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) showed that the VEGFR-2 inhibitory activities of the 10 target compounds at 10 μM were excellent, where nine compounds obtained an inhibition ratio above 90%. Moreover, rescreening results showed that these nine compounds achieved nanomolar VEGFR-2 IC<sub>50</sub> values. All nine compounds except compounds <b>10</b> (IC<sub>50</sub><b>></b> 10 000 nM) and <b>11</b> (IC<sub>50</sub> = 3937 nM) exhibited better inhibitory activity than the positive control sorafenib (IC<sub>50</sub> = 46.41 nM). Moreover, the best three compounds, <b>19</b> (IC<sub>50</sub> = 0.75 nM), <b>20</b> (IC<sub>50</sub> = 0.92 nM), and <b>21</b> (IC<sub>50</sub> = 1.88 nM), were more active than the other positive control, staurosporine (IC<sub>50</sub> = 15.97 nM). Compound <b>22</b> (IC<sub>50</sub> = 8.7 nM) also showed better inhibitory activity than sorafenib via the introduction of a cyclopentanecarbonitrile group similar to that of apatinib, which was approved for clinical use in late-stage gastric carcinoma in China by the China Food and Drug Administration (CFDA) in 2016.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Enzymatic activity testing corroborated the binding mode prediction, because all compounds bound to a DFG-out conformation and formed key hydrogen bonds with Cys919 in the hinge region and Glu885 and Asp1046 in the hydrogen bond favorable region (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). In addition, the bicyclic group (naphthamide or quinoline) fit hydrophobic region I better than the monocyclic groups, and differentially substituted six-, five-, or three-membered rings matched well with hydrophobic region II (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>a).</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Dose-dependent enzymatic activity of the 10 target compounds against VEGFR-2. Results were the average of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding conformation of the 10 designed compounds. (a) All 10 compounds, (b) compound <b>19</b>, and (c) compound <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on activity data, the structure activity relationship (SAR) can be summarized as follows. First, the introduction of a naphthalene ring and a quinoline ring in hydrophobic region I has little effect on the inhibitory activity; for example, compounds <b>6</b> and <b>8</b> exhibited inhibitory activities of 15.29 and 17.85 nM, respectively. Next, hydrophobic region II can accommodate hydrophobic groups of different sizes, such as the cyclopropyl substituent of compound <b>21</b> and the substituted five-membered ring (pyrazole or oxazole) of compounds <b>19</b> (binding mode shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b) and <b>20</b>, as well as the substituted phenyl group of compounds <b>6</b>, <b>7</b>, <b>8</b>, <b>22</b>, and <b>23</b>. Third, a comparison of compounds <b>6</b> (IC<sub>50</sub> = 15.29 nM), <b>7</b> (IC<sub>50</sub> = 28.79 nM), and <b>23</b> (IC<sub>50</sub> = 33.36 nM) indicated that 4-chloro-substituted phenyl groups in hydrophobic region II did not affect inhibitory activity against VEGFR-2, whereas the removal or replacement with a hydrophilic morpholino moiety only decreased the activity about 2-fold. Finally, the replacement of the urea group in the hydrogen bond acceptor or donor substituents was favored over a sulfonamide moiety (which may reduce inhibitory activity). This trend can be seen when comparing compound <b>6</b> (IC<sub>50</sub> = 15.29 nM) with compound <b>10</b> (inhibition ratio of 44.31%) and compound <b>11</b> (IC<sub>50</sub> = 3937 nM), indicating that the hydrogen bond formation with Glu885, which is important for inhibitor binding, may have been disrupted (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>c).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p">Based on the enzymatic activity data, the protein and ligand stability during the MD simulation process for the most active compound <b>19</b> (IC<sub>50</sub> = 0.75 nM, colored black in <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>) and the second least active compound <b>11</b> (IC<sub>50</sub> = 3937 nM, colored red in <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>) was further evaluated. As illustrated in <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>, the RMSD values for the VEGFR-2 protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>) were nearly the same for the two compounds (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a), although for compound <b>19</b> the RMSF averaged about 1.75 Å and compound <b>11</b> averaged 2.4 Å (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>c). Nevertheless, compound <b>19</b> reached equilibrium around 0.5 Å, while <b>11</b> obtained stability around 1 Å (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>b). Furthermore, <b>19</b> achieved an average of 5 hydrogen bonds, while compound <b>11</b> only averaged 3 (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>d). These results may help explain the huge loss in binding affinity for compound <b>11</b> compared to compound <b>19</b>.</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. RMSD (a and b), RMSF (c), and the number of hydrogen bonds (d) of the protein and ligands in the MD simulations for the most active compound <b>19</b> (IC<sub>50</sub> = 0.75 nM, black) and the second least active <b>11</b> (IC<sub>50</sub> = 3937 nM, red). One nm is equal to 10 Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Cellular VEGFR-2 Assays</h3><div class="NLM_p">Based on their outstanding VEGFR-2 inhibitory activity, eight compounds were submitted for initial cell proliferation inhibition activity testing. Herein, using the CellTiter-Glo (CTG) cell growth inhibition test module, the growth inhibitory effect of the eight compounds on three cell lines (HUVECs,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> melanoma cell line MDA-MB-435,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and epidermal squamous cell carcinoma cell line A431<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>) was measured. VEGFR-2 overexpression was found in HUVECs and MDA-MB-435 cells in angiogenesis-related cancers.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> HUVECs, derived from the endothelium of veins from the umbilical cord, are the most commonly studied human endothelial cell type in angiogenesis.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> It has been proven that inhibiting VEGF-induced VEGFR-2 activation by small molecules in HUVECs can inhibit subsequent angiogenesis.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Here, MDA-MB-435 was used as a melanoma cell line rather than a model for human breast cancer because recent advances in gene expression analysis revealed that the pattern of gene expression for MDA-MB-435 more closely resembled that of melanoma cell lines than other breast tumor lines.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Moreover, the results from an immunofluorescence staining study showed strong VEGFR-2 staining in MDA-MB-435 tumors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The epidermal squamous cell carcinoma cell line A431 is an epidermal growth factor receptor (EGFR) overexpressing cell, but it can also lead to angiogenesis, which was used as a negative control for comparison.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p">As shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, all compounds showed good inhibitory effects against HUVECs with compounds <b>19</b>, <b>22</b>, and <b>23</b> reaching inhibition ratios of 93.58, 92.48, and 97.11%, respectively, which are superior to positive control sorafenib (84.66%). At 82.89% inhibition, compound <b>6</b> exhibited comparable inhibitory activity to sorafenib. The inhibitory effects on the MDA-MB-435 cell line for compounds <b>6</b>, <b>22</b>, and <b>23</b> were 95.06, 96.90, and 98.36%, respectively, also outperforming sorafenib (91.89%). As expected, this series of compounds did not exhibit good inhibitory effects against the A431 cell line, which has EGFR overexpression rather than VEGFR-2. For the A431 cell line, compounds <b>22</b> (49.05%) and <b>23</b> (57.71%), which obtained the highest inhibitory activity, were still weaker than sorafenib (68.94%). Compounds with cellular inhibition rates greater than 90% were further tested to obtain IC<sub>50</sub> values. As depicted in <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> and <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, the IC<sub>50</sub> values for compounds <b>6</b>, <b>19</b>, <b>22</b>, and <b>23</b> against HUVECs were 2.97, 0.63, 3.62, and 0.63 μM, respectively, which is comparable with sorafenib (1.49 μM). The IC<sub>50</sub> values for compounds <b>6</b>, <b>22</b>, and <b>23</b> against MDA-MB-435 were 3.79, 3.01, and 2.36 μM, respectively, also comparable with sorafenib (2.11 μM). These results demonstrated that the designed compounds exhibited excellent cellular inhibitory activity, which most likely resulted from the inhibition of VEGFR-2 and were promising hits for developing antitumor candidates.</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0010.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Cellular activity against two cell lines, (a) HUVECs and (b) MDA-MB-435 cell line. Results were from three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Cellular Inhibitory Rates of Eight Compounds against HUVECs, MDA-MB-435, and A431 Cell Lines and IC<sub>50</sub> Values for Four Compounds for Two Cell Lines (HUVECs and MDA-MB-435)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">cell inhibition rate at 10 μM</th><th class="rowsep1 colsep0" colspan="2" align="center">cell inhibitory activity (IC<sub>50</sub> values, μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center" char=".">HUVECs (%)</th><th class="colsep0 rowsep0" align="center" char=".">MDA-MB-435 (%)</th><th class="colsep0 rowsep0" align="center" char=".">A431 (%)</th><th class="colsep0 rowsep0" align="center">HUVECs</th><th class="colsep0 rowsep0" align="center">MDA-MB-435</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char="."><b>82.89</b></td><td class="colsep0 rowsep0" align="char" char="."><b>95.06</b></td><td class="colsep0 rowsep0" align="char" char=".">14.95</td><td class="colsep0 rowsep0" align="left"><b>2.97 ± 0.18</b></td><td class="colsep0 rowsep0" align="left"><b>3.79 ± 0.93</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">66.8</td><td class="colsep0 rowsep0" align="char" char=".">24.54</td><td class="colsep0 rowsep0" align="char" char=".">–4.51</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">51.45</td><td class="colsep0 rowsep0" align="char" char=".">44.29</td><td class="colsep0 rowsep0" align="char" char=".">20.96</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char="."><b>93.58</b></td><td class="colsep0 rowsep0" align="char" char=".">27.71</td><td class="colsep0 rowsep0" align="char" char=".">49.87</td><td class="colsep0 rowsep0" align="left"><b>0.63 ± 0.05</b></td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">61.89</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">–1.52</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">63.9</td><td class="colsep0 rowsep0" align="char" char=".">1.48</td><td class="colsep0 rowsep0" align="char" char=".">15.13</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char="."><b>92.48</b></td><td class="colsep0 rowsep0" align="char" char="."><b>96.90</b></td><td class="colsep0 rowsep0" align="char" char=".">49.05</td><td class="colsep0 rowsep0" align="left"><b>3.62 ± 0.17</b></td><td class="colsep0 rowsep0" align="left"><b>3.01 ± 0.03</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char="."><b>97.11</b></td><td class="colsep0 rowsep0" align="char" char="."><b>98.36</b></td><td class="colsep0 rowsep0" align="char" char=".">57.71</td><td class="colsep0 rowsep0" align="left"><b>0.63 ± 0.02</b></td><td class="colsep0 rowsep0" align="left"><b>2.36 ± 0.09</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sorafenib</td><td class="colsep0 rowsep0" align="char" char="."><b>84.66</b></td><td class="colsep0 rowsep0" align="char" char="."><b>91.89</b></td><td class="colsep0 rowsep0" align="char" char=".">68.94</td><td class="colsep0 rowsep0" align="left"><b>1.49 ± 0.08</b></td><td class="colsep0 rowsep0" align="left"><b>2.11 ± 0.04</b></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">The data in bold are compounds with cellular inhibition greater than 80%, which were tested for corresponding IC<sub>50</sub> values. Results were from three replicates. nd = not determined.</p></div></div></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Antitumor Spectrum of Representative Compounds</h3><div class="NLM_p">Based on cellular activity, four compounds <b>6</b>, <b>19</b>, <b>22</b>, and <b>23</b> exhibited good inhibitory activity against HUVECs and the MDA-MB-435 cell line. Thus, to expand the spectrum of tumor cells tested, these four compounds were submitted to the NCI for testing in a panel of 58 cell lines (termed NCI-60). These cell lines correspond to nine tumor types including leukemia, NSCLC, colon cancer, central nervous system (CNS) cancer, melanoma, ovarian cancer, renal cancer, prostate cancer, and breast cancer (shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figures S6–S10</a>). The cell lines within the NCI-60 panel of human tumor cell lines are some of the most extensively characterized cell lines in broad laboratory use, and the 58 cell lines tested in this work are VEGFR-2 expressing or overexpressing ones.</div><div class="NLM_p">As seen in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> and <a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>, compound <b>6</b> had moderately potent antiproliferative activity against melanoma, renal, breast, and CNS cancers. It exhibited about 50% cell growth for at least one of the cell types that may contribute to those tumors, which means that more than 50% of the cells were inhibited. Compound <b>19</b> did not show any antiproliferative activity to all tested cell lines because all cell growth proportions were higher than 70%. Compound <b>22</b> exhibited excellent antiproliferative activity against NSCLC [cell growth A549/ATCC (25.6%), HOP-92 (−39.8%), and NCI-H460 (19.3%)], CNS cancer [cell growth SF-268 (27%), SF-295 (13.7%), and U251 (−49.3%)], ovarian cancer [cell growth OVCAR-4 (−1.3%), and OVCAR-8 (0.6%)], renal cancer [cell growth 786–0 (−40.7%), ACHN (29.3%), RXF 393 (15%), SN12C (21.3%), and TK-10 (24.1%)], and breast cancer [cell growth HS 578T (29.6%)]. Compound <b>23</b> demonstrated the best antiproliferative activity among all four compounds. It exhibited very potent antiproliferative activity on 51 out of 58 cell lines, where the growth percent was less than 50% (with 11 less than 0), and the average growth percent for all 58 cell lines was approximately 19.2%, indicating that <b>23</b> can inhibit about 80% of cell growth on all 51 cell lines on average.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0011.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Dose-dependent growth inhibition curves for compound <b>23</b> on 58 cell lines conducted by NCI. A growth percent of 100 is the growth seen in untreated cells, while a growth percent of 0 corresponds to no net growth during the assay and a growth percent of −100 indicates the time when all cells are killed. Three endpoints are routinely calculated: (1) GI<sub>50</sub>, the drug concentration yielding a growth percent of 50 (i.e., 50% growth inhibition); (2) TGI (total growth inhibition), the concentration resulting in a growth percent of 0; (3) LC<sub>50</sub>, the concentration when a growth percent of −50 or lethality in 50% of the starting cells is shown. These endpoints are illustrated for one of the melanoma cell lines MALME-CM (red open diamond).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Cellular Growth Inhibition Values of Compound <b>23</b> on 58 Cell Lines<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">tumor type</th><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char=".">GI<sub>50</sub><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">TGI<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a> (μM)</th><th class="colsep0 rowsep0" align="center">LC<sub>50</sub><a class="ref internalNav" href="#t3fn4" aria-label="d">d</a> (μM)</th><th class="colsep0 rowsep0" align="center">tumor type</th><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char=".">GI<sub>50</sub><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">TGI<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a> (μM)</th><th class="colsep0 rowsep0" align="center">LC<sub>50</sub><a class="ref internalNav" href="#t3fn4" aria-label="d">d</a> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">leukemia</td><td class="colsep0 rowsep0" align="left">CCRF-CEM</td><td class="colsep0 rowsep0" align="char" char=".">1.17</td><td class="colsep0 rowsep0" align="char" char=".">40.74</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left">melanoma</td><td class="colsep0 rowsep0" align="left">M14</td><td class="colsep0 rowsep0" align="char" char="."><b>0.95</b></td><td class="colsep0 rowsep0" align="char" char=".">4.57</td><td class="colsep0 rowsep0" align="left">26.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HL-60(TB)</td><td class="colsep0 rowsep0" align="char" char="."><b>0.38</b></td><td class="colsep0 rowsep0" align="char" char=".">2.14</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MDA-MB-435</td><td class="colsep0 rowsep0" align="char" char="."><b>0.37</b></td><td class="colsep0 rowsep0" align="char" char=".">3.39</td><td class="colsep0 rowsep0" align="left">33.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">K-562</td><td class="colsep0 rowsep0" align="char" char="."><b><0.01</b></td><td class="colsep0 rowsep0" align="char" char=".">1.12</td><td class="colsep0 rowsep0" align="left"><b>95.50</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SK-MEL-2</td><td class="colsep0 rowsep0" align="char" char=".">1.48</td><td class="colsep0 rowsep0" align="char" char=".">13.80</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MOLT-4</td><td class="colsep0 rowsep0" align="char" char="."><b>0.19</b></td><td class="colsep0 rowsep0" align="char" char=".">1.82</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SK-MEL-28</td><td class="colsep0 rowsep0" align="char" char="."><b>0.89</b></td><td class="colsep0 rowsep0" align="char" char=".">4.79</td><td class="colsep0 rowsep0" align="left">42.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">RPMI-8226</td><td class="colsep0 rowsep0" align="char" char="."><b>0.39</b></td><td class="colsep0 rowsep0" align="char" char=".">2.95</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SK-MEL-5</td><td class="colsep0 rowsep0" align="char" char="."><b>0.45</b></td><td class="colsep0 rowsep0" align="char" char=".">2.69</td><td class="colsep0 rowsep0" align="left">16.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SR</td><td class="colsep0 rowsep0" align="char" char="."><b>0.69</b></td><td class="colsep0 rowsep0" align="char" char=".">5.37</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">UACC-257</td><td class="colsep0 rowsep0" align="char" char="."><b>0.79</b></td><td class="colsep0 rowsep0" align="char" char=".">3.98</td><td class="colsep0 rowsep0" align="left">47.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">non-small cell lung cancer</td><td class="colsep0 rowsep0" align="left">A549/ATCC</td><td class="colsep0 rowsep0" align="char" char="."><b>0.55</b></td><td class="colsep0 rowsep0" align="char" char=".">10.72</td><td class="colsep0 rowsep0" align="left">81.28</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">UACC-62</td><td class="colsep0 rowsep0" align="char" char="."><b>0.60</b></td><td class="colsep0 rowsep0" align="char" char=".">3.47</td><td class="colsep0 rowsep0" align="left">18.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EKVX</td><td class="colsep0 rowsep0" align="char" char="."><b>0.45</b></td><td class="colsep0 rowsep0" align="char" char=".">6.46</td><td class="colsep0 rowsep0" align="left">67.61</td><td class="colsep0 rowsep0" align="left">ovarian cancer</td><td class="colsep0 rowsep0" align="left">IGROV1</td><td class="colsep0 rowsep0" align="char" char="."><b>0.52</b></td><td class="colsep0 rowsep0" align="char" char=".">3.63</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HOP-92</td><td class="colsep0 rowsep0" align="char" char="."><b>0.32</b></td><td class="colsep0 rowsep0" align="char" char=".">7.76</td><td class="colsep0 rowsep0" align="left"><b>91.20</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">OVCAR-3</td><td class="colsep0 rowsep0" align="char" char=".">1.00</td><td class="colsep0 rowsep0" align="char" char=".">9.55</td><td class="colsep0 rowsep0" align="left">42.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCI-H226</td><td class="colsep0 rowsep0" align="char" char="."><b>0.07</b></td><td class="colsep0 rowsep0" align="char" char=".">13.80</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">OVCAR-4</td><td class="colsep0 rowsep0" align="char" char="."><b>0.12</b></td><td class="colsep0 rowsep0" align="char" char=".">10.47</td><td class="colsep0 rowsep0" align="left">35.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCI-H23</td><td class="colsep0 rowsep0" align="char" char="."><b>0.62</b></td><td class="colsep0 rowsep0" align="char" char=".">3.98</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">OVCAR-5</td><td class="colsep0 rowsep0" align="char" char=".">1.07</td><td class="colsep0 rowsep0" align="char" char=".">10.47</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCI-H322M</td><td class="colsep0 rowsep0" align="char" char="."><b>0.56</b></td><td class="colsep0 rowsep0" align="char" char=".">15.14</td><td class="colsep0 rowsep0" align="left"><b>93.33</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">OVCAR-8</td><td class="colsep0 rowsep0" align="char" char="."><b>0.40</b></td><td class="colsep0 rowsep0" align="char" char=".">9.55</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCI-H460</td><td class="colsep0 rowsep0" align="char" char=".">1.41</td><td class="colsep0 rowsep0" align="char" char=".">4.27</td><td class="colsep0 rowsep0" align="left">25.70</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCI/ADR-RES</td><td class="colsep0 rowsep0" align="char" char="."><b>0.76</b></td><td class="colsep0 rowsep0" align="char" char=".">58.88</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCI-H522</td><td class="colsep0 rowsep0" align="char" char="."><b>0.31</b></td><td class="colsep0 rowsep0" align="char" char=".">3.80</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SK-OV-3</td><td class="colsep0 rowsep0" align="char" char="."><b>0.23</b></td><td class="colsep0 rowsep0" align="char" char=".">0.95</td><td class="colsep0 rowsep0" align="left">37.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">colon cancer</td><td class="colsep0 rowsep0" align="left">COLO 205</td><td class="colsep0 rowsep0" align="char" char="."><b>0.38</b></td><td class="colsep0 rowsep0" align="char" char=".">3.39</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left">renal cancer</td><td class="colsep0 rowsep0" align="left">786-0</td><td class="colsep0 rowsep0" align="char" char="."><b>0.79</b></td><td class="colsep0 rowsep0" align="char" char=".">5.37</td><td class="colsep0 rowsep0" align="left">26.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HCC-2998</td><td class="colsep0 rowsep0" align="char" char=".">5.25</td><td class="colsep0 rowsep0" align="char" char=".">20.89</td><td class="colsep0 rowsep0" align="left">75.86</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A498</td><td class="colsep0 rowsep0" align="char" char="."><b>0.05</b></td><td class="colsep0 rowsep0" align="char" char=".">0.52</td><td class="colsep0 rowsep0" align="left">30.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HCT-116</td><td class="colsep0 rowsep0" align="char" char="."><b>0.79</b></td><td class="colsep0 rowsep0" align="char" char=".">2.57</td><td class="colsep0 rowsep0" align="left">7.08</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ACHN</td><td class="colsep0 rowsep0" align="char" char="."><b>0.32</b></td><td class="colsep0 rowsep0" align="char" char=".">1.62</td><td class="colsep0 rowsep0" align="left">14.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HCT-15</td><td class="colsep0 rowsep0" align="char" char=".">1.00</td><td class="colsep0 rowsep0" align="char" char=".">5.13</td><td class="colsep0 rowsep0" align="left">28.18</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">RXF 393</td><td class="colsep0 rowsep0" align="char" char=".">1.05</td><td class="colsep0 rowsep0" align="char" char=".">7.08</td><td class="colsep0 rowsep0" align="left">29.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HT29</td><td class="colsep0 rowsep0" align="char" char="."><b>0.45</b></td><td class="colsep0 rowsep0" align="char" char=".">13.18</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SN12C</td><td class="colsep0 rowsep0" align="char" char="."><b>0.29</b></td><td class="colsep0 rowsep0" align="char" char=".">1.86</td><td class="colsep0 rowsep0" align="left">13.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">KM12</td><td class="colsep0 rowsep0" align="char" char="."><b>0.20</b></td><td class="colsep0 rowsep0" align="char" char=".">2.51</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TK-10</td><td class="colsep0 rowsep0" align="char" char="."><b>0.38</b></td><td class="colsep0 rowsep0" align="char" char=".">5.89</td><td class="colsep0 rowsep0" align="left">35.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SW-620</td><td class="colsep0 rowsep0" align="char" char=".">1.10</td><td class="colsep0 rowsep0" align="char" char=".">11.75</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">UO-31</td><td class="colsep0 rowsep0" align="char" char="."><b>0.20</b></td><td class="colsep0 rowsep0" align="char" char=".">10.23</td><td class="colsep0 rowsep0" align="left">38.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CNS cancer</td><td class="colsep0 rowsep0" align="left">SF-268</td><td class="colsep0 rowsep0" align="char" char="."><b>0.46</b></td><td class="colsep0 rowsep0" align="char" char=".">15.85</td><td class="colsep0 rowsep0" align="left">79.43</td><td class="colsep0 rowsep0" align="left">prostate cancer</td><td class="colsep0 rowsep0" align="left">PC-3</td><td class="colsep0 rowsep0" align="char" char=".">1.78</td><td class="colsep0 rowsep0" align="char" char=".">12.59</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SF-295</td><td class="colsep0 rowsep0" align="char" char=".">1.23</td><td class="colsep0 rowsep0" align="char" char=".">6.03</td><td class="colsep0 rowsep0" align="left">30.90</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DU-145</td><td class="colsep0 rowsep0" align="char" char=".">1.55</td><td class="colsep0 rowsep0" align="char" char=".">13.18</td><td class="colsep0 rowsep0" align="left">44.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SF-539</td><td class="colsep0 rowsep0" align="char" char=".">1.58</td><td class="colsep0 rowsep0" align="char" char=".">19.50</td><td class="colsep0 rowsep0" align="left">52.48</td><td class="colsep0 rowsep0" align="left">breast cancer</td><td class="colsep0 rowsep0" align="left">MCF7</td><td class="colsep0 rowsep0" align="char" char="."><b>0.33</b></td><td class="colsep0 rowsep0" align="char" char=".">10.96</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SNB-19</td><td class="colsep0 rowsep0" align="char" char=".">2.57</td><td class="colsep0 rowsep0" align="char" char=".">14.13</td><td class="colsep0 rowsep0" align="left">43.65</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MDA-MB-231/ATCC</td><td class="colsep0 rowsep0" align="char" char=".">1.15</td><td class="colsep0 rowsep0" align="char" char=".">7.94</td><td class="colsep0 rowsep0" align="left">42.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SNB-75</td><td class="colsep0 rowsep0" align="char" char=".">1.05</td><td class="colsep0 rowsep0" align="char" char=".">13.80</td><td class="colsep0 rowsep0" align="left">38.90</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HS 578T</td><td class="colsep0 rowsep0" align="char" char=".">1.12</td><td class="colsep0 rowsep0" align="char" char=".">9.33</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">U251</td><td class="colsep0 rowsep0" align="char" char=".">1.05</td><td class="colsep0 rowsep0" align="char" char=".">3.98</td><td class="colsep0 rowsep0" align="left">27.54</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">BT-549</td><td class="colsep0 rowsep0" align="char" char=".">1.15</td><td class="colsep0 rowsep0" align="char" char=".">7.41</td><td class="colsep0 rowsep0" align="left">34.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">melanoma</td><td class="colsep0 rowsep0" align="left">LOX IMVI</td><td class="colsep0 rowsep0" align="char" char=".">1.10</td><td class="colsep0 rowsep0" align="char" char=".">2.82</td><td class="colsep0 rowsep0" align="left">7.24</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">T-47D</td><td class="colsep0 rowsep0" align="char" char="."><b>0.25</b></td><td class="colsep0 rowsep0" align="char" char=".">4.27</td><td class="colsep0 rowsep0" align="left"><b>>100</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MALME-3M</td><td class="colsep0 rowsep0" align="char" char="."><b>0.30</b></td><td class="colsep0 rowsep0" align="char" char=".">1.55</td><td class="colsep0 rowsep0" align="left">5.75</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MDA-MB-468</td><td class="colsep0 rowsep0" align="char" char="."><b>0.23</b></td><td class="colsep0 rowsep0" align="char" char=".">2.82</td><td class="colsep0 rowsep0" align="left">72.44</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Performed by NCI.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">GI<sub>50</sub>.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">LC<sub>50</sub>.</p></div><div class="footnote" id="t3fn4"><sup>d</sup><p class="last">TGI are the same as <a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>. GI<sub>50</sub> values less than 1 μM or LC<sub>50</sub> values higher than 100 μM are shown in bold.</p></div></div></div><div class="NLM_p">Therefore, compound <b>23</b> was selected to determine GI<sub>50</sub> values at five concentrations with 1 log dilutions from 10<sup>–4</sup> to 10<sup>–8</sup> M (<a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>). Three endpoint values, GI<sub>50</sub>, TGI, and LC<sub>50</sub>, were calculated (<a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>). As illustrated in <a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>, compound <b>23</b> exhibited a dose-dependent inhibition behavior against all 58 cell lines, corroborating all in vitro tests. According to <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> and <a class="ref internalNav" href="#fig11" aria-label="Figures 11">Figures 11</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">S9</a>, compound <b>23</b> displayed an excellent antiproliferative activity, with the GI<sub>50</sub> values against all 58 cell lines at less than 5.25 μM and TGI less than 58.9 μM. Specifically, compound <b>23</b> exhibited GI<sub>50</sub> values less than 200 nM against seven cancer cell lines including leukemia [cell lines K-562 (GI<sub>50</sub> < 10 nM) and MOLT-4 (GI<sub>50</sub> = 190 nM)], NSCLC [cell line NCI-H226 (GI<sub>50</sub> < 70 nM)], colon cancer [cell line KM12 (GI<sub>50</sub> < 200 nM)], ovarian cancer [cell line OVCAR-4 (GI<sub>50</sub> < 120 nM)], and renal cancer [cell lines A498 (GI<sub>50</sub> < 50 nM) and UO-31 (GI<sub>50</sub> < 200 nM)].</div><div class="NLM_p">In addition, the NCI-60 results of compound <b>23</b> were compared with the results for two FDA approved VEGFR-2 drugs, sorafenib and sunitinib, which were downloaded from <a href="https://dtp.cancer.gov/discovery_development/nci-60/" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/</a>. Sorafenib, a Raf-VEGFR-2 dual inhibitor, is now used for renal cell carcinoma, hepatocellular carcinoma, and thyroid cancer treatments.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Sunitinib<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> is used for the treatment of renal cell carcinoma, pancreatic neuroendocrine tumors, and gastrointestinal stromal tumors to complement intolerance to imatinib.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The comparison of the mean endpoint values for all cell lines to each tumor is shown in <a class="ref internalNav" href="#fig12" aria-label="Figure 12">Figure 12</a>, and the original GI<sub>50</sub>, TGI, and LC<sub>50</sub> values for each cell line are displayed in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figures S11–S13</a>, respectively. On the basis of <a class="ref internalNav" href="#fig12" aria-label="Figure 12">Figure 12</a>, although only the renal cancer cell line that is approved for both drugs was included in the NCI-60, we observed that the GI<sub>50</sub> values of compound <b>23</b> were 9.5- and 6.8-fold better (lower) than those of sorafenib and sunitinib, respectively. The TGI values were both 3.2-fold lower than the two marketed drugs. In addition, the LC<sub>50</sub> values were 2- and 1.3-fold worse (higher) than these drugs, respectively, indicating that compound <b>23</b> was more potent in the cell lines responsible for renal cancer, and its safety was comparable with that of sorafenib and sunitinib for clinical use in renal cancer. For the other eight tumor types, compound <b>23</b> exhibited a higher potency against the cell lines and was more safe or comparable to sorafenib and sunitinib. Specifically, its GI<sub>50</sub> values were at least 3.9–9.5-fold and 5–7.7-fold better than sorafenib and sunitinib, respectively, for leukemia, NSCLC, ovarian cancer, and breast cancer. For colon cancer, CNS cancer, melanoma, and prostate cancer, the GI<sub>50</sub> of compound <b>23</b> was about 1.6–2.9-fold and 1.1–3.7-fold better than that of sorafenib and sunitinib, respectively. Among all cancers, renal and ovarian cancers are VEGF/VEGFR pathway inhibitor-sensitive (such as VEGF antibodies and VEGFR-2 TKIs) tumors, while NSCLC, breast cancer, and colon cancer are only partly sensitive to VEGFR inhibitors.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Among the cell lines tested, NCI/ADR-RES is a multidrug-resistant cancer cell line for ovarian cancer.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Nevertheless, compound <b>23</b> exhibited a very strong antiproliferative activity with a GI<sub>50</sub> of about 0.76 μM, which is 3.3- and 6.9-fold better than sorafenib and sunitinib, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Figure S11</a>), demonstrating that this compound may inhibit some drug-resistant cell lines.</div><figure id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0012.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Comparison of the GI<sub>50</sub>, TGI, and LC<sub>50</sub> values for compound <b>23</b> (black), sorafenib (red), and sunitinib (blue). The <i>y</i>-axis is the log value of the endpoints, and a difference of 1 indicates a 10-fold difference in the endpoint values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Studies showed that increased expression of VEGF/VEGFR-2 in leukemic blasts and correlation with angiogenesis in the bone marrow of acute myeloid leukemia (AML) patients has been observed.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> VEGF is also a major contributor to the growth of CNS tumors, as elevated expression of VEGF is one of the factors responsible for the virulent nature of these tumors. Thus, the interruption of VEGFR signaling, specifically VEGFR-2, may effectively suppress tumor growth in leukemia and CNS cancer.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> However, the expression of VEGFR-2 in melanoma cells is controversial. Several studies have shown that high quantities (78–89%)<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a> or at least half of the melanoma cells<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> express VEGFR-2. Others indicated that relatively few (<10%) melanoma cells expressed detectable levels of VEGFR-2.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Nevertheless, sorafenib, a dual inhibitor of BRaf and VEGFR-2 with promise in other cancers driven by VEGF, was relatively ineffective when used as a single agent in melanoma. Despite these studies casting doubt on VEGFR-2 as an effective target in melanoma, the identification of VEGFR-2 positive melanoma cell lines still presents an opportunity to personalize therapy using antibodies and other TKIs that specifically target VEGFR-2-dependent signaling. All data implied that the inhibition of the cell lines by compound <b>23</b> probably stems from its inhibition of VEGFR-2 rather than an off-target effect. Nevertheless, because compound <b>23</b> was a multikinase inhibitor, which we will discuss later, the inhibition of the cell lines may also benefit from activity at other targets. For example, sorafenib is the only compound used for the treatment of hepatocellular carcinoma among the VEGFR-2-related approved drugs. It is probably because of positive clinical trial results in hepatocellular carcinoma, which may be due to its BRaf inhibitory activity as the overexpression of BRaf was indeed frequently observed in hepatocellular carcinoma. However, further efforts to understand the complete role of sorafenib in the treatment of hepatocellular carcinoma and the cell antiproliferative activity of compound <b>23</b> are still necessary.</div><div class="NLM_p last">As the second leading cause of male cancer death in developed countries, prostate cancer belongs to one of the VEGF/VEGFR pathway inhibitor resistant cancers.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The expression of VEGFRs in endothelial cells isolated from prostate tumors has confirmed this finding. However, though sunitinib and sorafenib showed similar antiangiogenic and cytotoxic effects on normal endothelial cells, sunitinib affected the proliferation, survival, and motility of prostate tumors, whereas sorafenib only showed a minor effect. Both medications inhibited VEGFR-2 phosphorylation of prostate cancer endothelial cells, while they differentially modulated Akt phosphorylation, as no inhibitory effect of sorafenib was observed on Akt activation.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The combined treatment with Casodex reverted the observed resistance to sorafenib both on cell viability and on Akt activation, whereas it did not modify the response to sunitinib. Thus, the mechanism for the activity of sunitinib on prostate cancer remains unknown. Compound <b>23</b> exhibited about 1.6- and 1.5-fold better GI<sub>50</sub> than sorafenib and sunitinib on prostate cancer, respectively (<a class="ref internalNav" href="#fig12" aria-label="Figure 12">Figure 12</a>). Compound <b>23</b> showed TGI and LC<sub>50</sub> values similar to that of those two FDA-approved drugs. Specifically, the LC<sub>50</sub> value of compound <b>23</b> was 1.7-fold of sunitinib, showing a safer effect. However, its inhibition of Akt was not obvious (less than 20% on VEGFR-2 inhibition at 10 μM). Nevertheless, the resistance of prostate cancer to VEGF inhibitors is more likely related to redundancy in angiogenic pathways. A large amount of data indicates that fibroblast growth factor (FGF) and its family of receptors (FGFRs) were observed in the transformation and angiogenesis of prostate cancer, and several early phase clinical trials are assessing FGFR TKIs in this disease.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Thus, there is a possibility that compound <b>23</b> works in a similar way, as it showed higher than 96% inhibition on FGFR-1/-2/-3/-4. Therefore, we cannot jump to any conclusions about whether this compound is a powerful prostate cancer hit, but it still holds promise for further evaluation as a potential candidate like sunitinib.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Kinome Spectrum Screening</h3><div class="NLM_p">To further investigate compound efficacy, compound <b>23</b> was evaluated at 0.5 μM against the Reaction Biology Corporation broad kinase spectrum panel containing 349 therapeutically important kinases (<a class="ref internalNav" href="#fig13" aria-label="Figure 13">Figure 13</a>). The kinome phylogenetic tree was generated using KinomeRender.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a><a class="ref internalNav" href="#fig13" aria-label="Figure 13">Figure 13</a>a illustrates that compound <b>23</b> was a potent multitargeted kinase inhibitor, particularly for the tyrosine kinase (TK) branch including targets like members of the VEGFR, Abl, DDR, EPHA, FGFR, Flt, and PDGFR families as well as the STE family including KHS, TNIK, and HCK (<a class="ref internalNav" href="#fig13" aria-label="Figure 13">Figure 13</a>b), TAK1 of the TKL family, and SLK of the CAMK family. The kinase inhibition profile of compound <b>23</b> was similar to that of FDA approved sunitinib<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (<a class="ref internalNav" href="#fig13" aria-label="Figure 13">Figure 13</a>). Through a multiple sequence alignment, we discovered that the targets such as Src, PDGFR, and FGFR for which compound <b>23</b> exhibited a high inhibition also had a higher sequence identity. About two-thirds (38 out of 56) of the targets with an inhibition greater than 90% were found in the 105 targets exhibiting a high sequence identity (greater than 30%) with VEGFR-2 (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">Table S2</a>). This thus indicated that the selectivity profile can be determined by sequence identity. This was consistent with a comparison of multitarget drugs that inhibited target pairs in the human kinome, particularly the receptor tyrosine kinase family, with combination drugs that are able to inhibit targets of a distant homology relationship.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Compound <b>23</b>, a multitargeted kinase inhibitor, is likely to inhibit multiple targets simultaneously, which may overcome any resistance mechanism that involves a bypass or restoration of persistent mitogen-activated protein kinase (MAPK) activation. However, compound <b>23</b> may also impart unwanted off-target side effects, which necessitates a further optimization and compound evaluation.</div><figure id="fig13" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0013.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Kinome phylogenetic trees of <b>23</b> and sunitinib. (a) Percent inhibition by <b>23</b> against all 349 therapeutically important kinases. (b) <i>K</i><sub>d</sub> values of sunitinib. The AGC branch contains the PKA, PKG, and PKC families. CAMK is calcium/calmodulin-dependent protein kinase. CK1 is casein kinase 1. The CMGC branch contains the CDK, MAPK, GSK3, and CLK families. STE refers to homologues of yeast sterile 7, sterile 11, and sterile 20 kinases. TK is tyrosine kinase, and TKL corresponds to tyrosine kinase-like proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Protein kinases are the second-most investigated drug target group after G-protein-coupled receptors (GPCRs), which account for 30% of drug discovery projects at many pharmaceutical companies with dozens of compounds in clinical development.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> However, due to highly similar ATP binding pockets, kinase selectivity of drugs has become one of the greatest challenges in drug discovery. Since multiple signaling pathways are involved in most tumors, many of the inhibitors in clinical development are designed to affect multiple targets. Among the 46 FDA approved drugs targeting the human kinome, 15 are multitarget drugs and 14 are approved for use in combination with other drugs.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Seven of the nine VEGFR-2 involved FDA approved drugs (sorafenib, sunitinib, vandetanib, regorafenib, cabozantinib, nintedanib, and lenvatinib) are also multitarget drugs, demonstrating the effectiveness of designing multitarget inhibitors involving VEGFR-2.</div><div class="NLM_p last">The initial effort in developing multitarget drugs stems from a belief that attacking more than one target can provide better efficacy and safety profiles compared to single target drugs. At present, there are two contrasting multitarget drug discovery strategies. The first method focuses on discovering agents that can simultaneously inhibit two or more targets like compound <b>23</b> that we discussed herein. The second strategy involves a combination of agents that are selective for a single target to achieve an additive or synergistic effect.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> However, mulitarget inhibitors from the first strategy risk side effects from unwanted compensatory mechanisms. Moreover, the multitarget inhibitors from the second strategy may be confronted with the problem of attenuated effectiveness by pharmacokinetic variations among individual ingredients as well as drug–drug interactions. Thus, there is a recent trend to develop more selective inhibitors, but highly selective inhibitors still face the problem of low efficacy for treating diseases like cancer, which might achieve compensation via redundant signaling pathways. Therefore, the development of selectively nonselective kinase inhibitors has been proposed, which will just modulate the cancer-related targets to strike the correct balance of the nonselectivity (promiscuity) that is essential for efficacy and the selectivity required for safety.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> However, a comprehensive analysis of kinase inhibitor selectivity showed that no dedicated inhibitors exist yet.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In addition, the advantages and disadvantages of multikinase inhibitors and single kinase inhibitors are related to potential resistance mechanisms, pharmacokinetics, selectivity and tumor environment.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Although it is currently difficult to intentionally design a mutitarget inhibitor with activity only on the kinases of interest, increasingly rational and elegant medicinal chemistry approaches are being implemented to solve this difficult problem.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31588" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31588" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this work, computational and experimental methods are seamlessly integrated for the discovery of a series of novel VEGFR-2 inhibitors. Specifically, a robust linear model, between the computational and experimental binding free energy (calculated by MM/PBSA) for scoring the potential hits, was constructed with a Pearson correlation coefficient (<i>R</i>) of 0.8. A total of 10 compounds with a relatively high predicted binding affinity were synthesized and evaluated their in vitro activity. Eight compounds showed nanomolar IC<sub>50</sub> values against VEGFR-2, and the best compounds were <b>19</b> (0.75 nM) and <b>20</b> (0.92 nM). Three compounds (<b>6</b>, <b>22</b>, and <b>23</b>) exhibited micromolar antiproliferative effects against HUVECs and melanoma cell line MDA-MB-435.</div><div class="NLM_p last">Through the NCI antiproliferative activity testing against 58 cancer cell lines involving nine tumor types, compounds <b>6</b> and <b>22</b> demonstrated potent antiproliferative activity against numerous cell lines that are responsible for renal, breast, and CNS cancers. In addition, comparison of the NCI screening results with two FDA approved drugs, sorafenib and sunitinib, indicated that compound <b>23</b> had a better antiproliferative activity against cell lines related to all nine tumor types tested in terms of GI<sub>50</sub> values. Regarding drug safety, the <b>23</b> LC<sub>50</sub> values were better than or comparable with those of sorafenib and sunitinib. It even demonstrated a high potency in one of the drug-resistant cell lines (NCI/ADR-RES) responsible for ovarian cancer as well as cell lines that contribute to prostate cancer, which is regarded as one of the VEFG/VEGFR pathway drug-resistant tumors. However, the intrinsic mechanism of this inhibition should be further evaluated by considering the multitarget inhibition profile of compound <b>23</b>. As a multitargeted kinase inhibitor like sunitinib, compound <b>23</b> is more likely to inhibit multiple targets simultaneously, which may overcome any resistance mechanism that involves a bypass or restoration of persistent alternative pathway activation. Nevertheless, it may also bring unwanted off-target side effects, which are difficult to predict during the drug design process. In summary, because of both its efficacy and its safety data, compound <b>23</b> may be a promising candidate for the treatment of leukemia, NSCLC, colon cancer, ovarian cancer, and breast cancer. The integration of both computational and experimental studies like those conducted herein can be expanded in the future to similar situations with other important targets. This could more efficiently drive drug discovery forward by leveraging computational power and applying existing structural data in parallel with experimental work.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22865" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22865" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Molecular Modeling</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Molecular Docking</h4><div class="NLM_p last">The X-ray crystal structure of human VEGFR-2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>) was downloaded from the Protein Data Bank (PDB) and prepared using the Protein Preparation Wizard in the Schrödinger suite.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The small molecules were initially minimized by the LigPrep module in Schrödinger. The Glide module in extra precision (XP) mode was selected for molecular docking due to its excellent performance in a self-docking analysis,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and the 10 best poses of each ligand were minimized by a postdocking program that saved the best pose for further analysis.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Binding Site Detection</h4><div class="NLM_p last">The SiteMap<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> module in the Schrödinger software suite<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> was used to detect the active site of VEGFR-2. This incorporated SiteMap program is an accurate yet fast binding site identification tool that was used herein not only to detect the binding site of the VEGFR-2 crystal structure but also to predict the druggability of those sites.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> BREED</h4><div class="NLM_p last">Ligand hybridization is a commonly employed method in FBDD. This process generates novel inhibitors that are based on structures of known ligands, which is essentially the automation of common medicinal chemistry practice. All related crystal structures complexed with ligands are aligned, and all pairs of ligands with overlapping bonds are exchanged to generate new compounds. However, crossover bonds may be restricted to bias the generated structures toward synthetic accessibility. As this method is automated, all possible combinations of known ligands are investigated and generated recursively. Therefore, this process generates four types of compounds: entirely novel scaffolds, known scaffolds with novel substituents, the same scaffolds as known inhibitors not included in the initial set, and sometimes a reconstruction of the input ligands. BREED was implemented in MOE software with all parameters set as the default configuration.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> All crystal structures as well as the compounds generated by BREED had duplicates removed.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Molecular Dynamics Simulations</h4><div class="NLM_p last">After molecular docking, the protein complexes were used for MD simulations. These MD simulations were performed using GROMACS<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> with the AMBER03 force field<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and the TIP3P water model.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The proteins were simulated in a water filled dodecahedron box with a distance of at least 12 Å between the complex and the box edges. Charges were neutralized by adding counterions (either Na<sup>+</sup> or Cl<sup>–</sup>) by the genion tool. Then, the system was relaxed through an energy minimization process with the steepest descent algorithm first, followed by a conjugate gradient algorithm. After that, an NVT simulation was performed with the temperature increasing from 100 to 300 K, followed by an NPT simulation to equilibrate the pressure. Finally, a production MD simulation was performed for 20 ns at 300 K. Bond lengths were constrained using the LINCS algorithm.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The last 5 ns of each production simulation were extracted at every 10 ps interval for calculating the MM/PBSA binding energy using the AMBER12 package.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The average energy for the 1000 snapshots, where one energy value was calculated for each one of the 5, 10 ps samples (representing a total of 50 ps), was saved for further analysis.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The RMSD, RMSF, and the number of hydrogen bonds formed were analyzed by r_rms and g_hond tools, respectively, in the GROMACS package.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemistry Experiments</h3><div class="NLM_p">General synthetic routes of the ten target compounds were provided in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. The structures of these compounds were characterized by <sup>1</sup>H NMR, IR, and MS. The structural information for both of the intermediates and target compounds is shown below. Melting points were determined by an X-4 melting-point apparatus with a microscope (Beijing Tech Instrument) and were not corrected. <sup>1</sup>H NMR spectra were determined on Bruker AV300 and Bruker AM500 spectrometers. Chemical shifts for <sup>1</sup>H NMR are reported in parts per million (ppm) and calibrated to the solvent peak set. MS were determined by a Nicolet 2000 FT-IR mass spectrometer and MAT-212 mass spectrometer. IR was recorded on a SHIMADZU FTIR-8400S infrared spectrometer. All of the final compounds were tested by HPLC, and the purity in every case was ≥95%. Reverse-phase HPLC was conducted on an Agilent Technologies 1260 Infinity, which was equipped with a C18 column (Agilent Zorbax SB-C18, 5 μm, 4.6 mm × 150 mm). Mobile phase A was water, and mobile phase B was methanol. A gradient of 20–80% B was run at a flow rate of 0.8 mL/min over 30 min.</div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 6,7-Dimethyl-4-(4-nitro-1-naphthalenyloxy)-quinazoline (<b>3a</b>)</h4><div class="NLM_p last">To a solution of 4-nitro-1-naphthol (1.36 g, 7.19 mmol) in dry <i>N</i>,<i>N</i>-dimethylformamide (6 mL) was added sodium hydride (345.2 mg, 8.63 mmol, 60% in mineral oil) at 0 °C, and the mixture was stirred for 30 min at room temperature. A solution of 4-chloro-6,7-dimethoxyquinazoline (1.6 g, 7.19 mmol) in dry <i>N</i>,<i>N</i>-dimethylformamide (4 mL) was added, and the reaction mixture was heated to 90 °C for 24 h. The reaction mixture was taken up in ethyl acetate and water. The organic phase was washed, dried, and evaporated. Recrystallization from ethyl acetate resulted in an off-white solid: yield 950 mg (34.6%); mp 213–215 °C; MS [M + H]<sup>+</sup> 378.1; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.52–8.47 (m, 3H, ArH), 8.09 (d, <i>J</i> = 8.4 Hz, 1H, ArH), 8.09 (t, <i>J</i> = 7.3 Hz, 1H, ArH), 7.77–7.69 (m, 3H, ArH), 7.46 (s, 1H, ArH), 3.17 (d, <i>J</i> = 5.3 Hz, 6H, −OCH<sub>3</sub>).</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 6,7-Dimethyl-4-((5-nitro-quinoline-8-yl)-oxy)-quinazoline (<b>3b</b>)</h4><div class="NLM_p last">To a solution of 8-hydroxy-5-nitroquinoline (500 mg, 2.63 mmol) in dry <i>N</i>,<i>N</i>-dimethylformamide (5 mL) was added sodium hydride (127 mg, 3.16 mmol, 60% in mineral oil) at 0 °C, and the mixture was stirred for 30 min at room temperature. A solution of 4-chloro-6,7-dimethoxyquinazoline (127 mg, 2.63 mmol) in dry <i>N</i>,<i>N</i>-dimethylformamide (2 mL) was added, and the reaction mixture was heated to 140 °C for 10 h. The reaction mixture was taken up in ethyl acetate and water. The organic phase was washed, dried, and concentrated. The residue was purified by column chromatography on silica gel to give the title compound as a pale yellow solid: yield 180 mg (18.9%); MS [M + H]<sup>+</sup> 347.1.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>)</h4><div class="NLM_p last">To a solution of 6,7-dimethyl-4-(4-nitro-1-naphthalenyloxy)-quinazoline (<b>3a</b>) (2 g, 5.93 mmol) in 70% isopropyl alcohol (25 mL) were added iron powder (996.7 mg, 17.8 mmol) and ammonium chloride (635 mg, 11.87 mmol) at 0 °C. Then, the reaction mixture was stirred at 85 °C for 4 h. The reaction mixture was filtered, and the filtrate was concentrated. The residue was purified by silica gel column chromatography to give a pale yellow solid: yield 1.36 g (66.06%); MS [M + H]<sup>+</sup> 348.1.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 8-((6,7-Dimethoxyquinazolin-4-yl)oxy)quinolin-5-amine (<b>4b</b>)</h4><div class="NLM_p last">To a solution of 6,7-dimethoxy-4-((5-nitroquinolin-8-yl)oxy)quinazoline (<b>3b</b>) (140 mg, 0.37 mmol) in 70% isopropyl alcohol (5 mL) were added iron powder (82.8 mg, 1.48 mmol) and ammonium chloride (59.37 mg, 1.11 mmol) at 0 °C . Then, the reaction mixture was stirred at 85 °C for 4 h. The reaction mixture was filtered, and the filtrate was concentrated. The residue was purified by silica gel column chromatography to give the title compound: yield 28.0 mg (21.8%); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.62 (d, <i>J</i> = 2.7 Hz, 1H, ArH), 8.59 (d, <i>J</i> = 8.6 Hz, 1H, ArH), 8.36 (s, 1H, ArH), 7.69 (s, 1H, ArH), 7.41–7.37 (m, 3H, ArH), 6.75 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 6.02 (s, 2H, −NH<sub>2</sub>), 4.06 (d, <i>J</i> = 3.42 Hz, 6H, −OCH<sub>3</sub>).</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)-3-(4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)urea (<b>6</b>)</h4><div class="NLM_p last">4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (60 mg, 0.086 mmol) and 4-chloro-3-trifluoromethylphenyl isocyanate (38.2 mg, 0.086 mmol) were dissolved in dry tetrahydrofuran (5 mL) and stirred at room temperature for 5 h. The reaction mixture was filtered, and the filter cake was washed with tetrahydrofuran and dichloromethane to obtain a white solid: yield 41 mg (83.7%); mp >250 °C; MS [M + H]<sup>+</sup> 569.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.55 (s, 1H, −NHCONH−), 8.99 (s, 1H, −NHCONH−), 8.45 (s, 1H, H-quinazoline), 8.18–8.16 (m, 2H, ArH), 7.97 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 7.85–7.60 (m, 5H, ArH), 7.59–7.39 (m, 3H, ArH), 4.03 (d, <i>J</i> = 3.8 Hz, 6H, −OCH<sub>3</sub>); IR (KBr) 3266, 1629, 1566, 1505, 1422, 1379, 1233, 1171, 1117 cm<sup>–1</sup>; HPLC purity 97.3%.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 1-(4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)-3-(3-(trifluoromethyl)phenyl)urea (<b>7</b>)</h4><div class="NLM_p last">4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (200 mg, 0.58 mmol) and 3-trifluoromethylphenyl isocyanate (113 mg, 0.6 mmol) were dissolved in dry dichloromethane (5 mL), and the mixture was stirred at room temperature for 5 h. The reaction mixture was filtered, and the filter cake was washed with tetrahydrofuran and dichloromethane to obtain a light brown solid: yield 236 mg (76.1%); mp 231–235 °C; MS [M + H]<sup>+</sup> 557.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.44 (s, 1H, −NHCONH−), 8.95 (s, 1H, −NHCONH−), 8.45 (s, 1H, H-quinazoline), 8.21 (d, <i>J</i> = 8.5 Hz, 1H, ArH), 8.11 (s, 1H, H-quinazoline), 8.01 (d, <i>J</i> = 8.2 Hz, 1H, ArH), 7.80–7.76 (m, 2H, ArH), 7.69–7.67 (m, 2H, ArH), 7.64–7.43 (m, 4H, ArH), 7.35 (d, <i>J</i> = 7.6 Hz, 1H, ArH), 4.03 (d, <i>J</i> = 3.8 Hz, 6H, −OCH<sub>3</sub>); IR (KBr) 3266, 1629, 1566, 1505, 1422, 1379, 1233, 1171, 1117 cm<sup>–1</sup>; HPLC purity 99.3%.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)-3-(8-((6,7-dimethoxyquinazolin-4-yl)oxy)quinolin-5-yl)urea (<b>8</b>)</h4><div class="NLM_p last">8-((6,7-Dimethoxyquinazolin-4-yl)oxy)quinolin-5-amine (<b>4b</b>) (42 mg, 0.12 mmol) and 4-chloride-3-trifluoromethylphenyl isocyanate (26.6 mg, 0.12 mmol) were dissolved in tetrahydrofuran (5 mL), and the mixture was stirred at room temperature for 5 h. The reaction mixture was filtered, and the filter cake was washed with tetrahydrofuran and dichloromethane to obtain a white solid: yield 13 mg (19.3%); mp 240–245 °C; MS [M + H]<sup>+</sup> 570.1; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H, −NHCONH−), 9.14 (s, 1H, −NHCONH−), 8.77 (d, <i>J</i> = 3.5 Hz, 1H, H-quinoline), 8.55 (d, <i>J</i> = 8.8 Hz, 1H, H-quinoline), 8.37 (s, 1H, H-quinazoline), 8.18 (s, 1H, H-benzene), 7.99 (d, <i>J</i> = 8.3 Hz, 1H, H-quinoline), 7.75–7.61 (m, 5H, ArH), 7.41 (s, 1H, H-quinazoline), 4.01 (d, <i>J</i> = 3.8 Hz, 6H, −OCH<sub>3</sub>); IR (KBr) 2835, 1677, 1614, 1564, 1500, 1266, 1121, 987, 861 cm<sup>–1</sup>. HPLC purity 98.7%.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)benzenesulfonamide (<b>10</b>)</h4><div class="NLM_p last">To a solution of 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (150 mg, 0.43 mmol) in dry tetrahydrofuran (10 mL) were added dropwise slowly at 0 °C triethylamine (179 μL, 1.29 mmol) and benzenesulfonyl chloride (56 μL, 0.43 mmol), and the mixture was stirred for 4 h at room temperature. The reaction mixture was adjusted with a saturated sodium bicarbonate solution to pH >7, extracted with ethyl acetate, dried with Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography to give a light brown solid: yield 195 mg (92.8%); mp 241–243 °C; MS [M + H]<sup>+</sup> 488.1; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.36 (s, 1H, −NHSO<sub>2</sub>−), 8.43 (s, 1H, H-quinazoline), 8.07 (d, <i>J</i> = 7.8 Hz, 1H, H-naphthalene), 7.74–7.72 (m, 4H, ArH), 7.64–7.38 (m, 7H, ArH), 7.21 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 4.01 (s, 6H, −OCH<sub>3</sub>); IR (KBr) 3390, 1617, 1578, 1506, 1454, 1421, 1375, 1221, 1156 cm<sup>–1</sup>; HPLC purity 98.0%.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)butane-1-sulfonamide (<b>11</b>)</h4><div class="NLM_p last">To a solution of 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (150 mg, 0.43 mmol) in dry pyridine (5 mL) was added 1-butanesulfonyl chloride (63 μL, 0.48 mmol), and the mixture was stirred for 1 h at room temperature. The reaction mixture was washed with 1 N hydrochloric acid, extracted with ethyl acetate, dried with Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography to give a light brown solid: yield 24 mg (12.2%); mp 215–218 °C; MS [M + H]<sup>+</sup> 468.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.46 (s, 1H, −NHSO<sub>2</sub>−), 8.34 (d, <i>J</i> = 8.6 Hz, 1H, H-quinazoline), 7.83–7.72 (m, 2H, ArH), 7.69 (t, <i>J</i> = 7.0 Hz, 1H), 7.61–7.46 (m, 3H), 7.44 (s, 1H, H-quinazoline), 4.02 (d, <i>J</i> = 1.6 Hz, 6H, −OCH<sub>3</sub>), 3.20 (t, <i>J</i> = 7.6 Hz, 2H, −CH<sub>2</sub>−), 1.80–1.70 (m, 2H, −CH<sub>2</sub>−), 1.46–1.34 (m, 2H, −CH<sub>2</sub>−), 0.87 (t, <i>J</i> = 7.3 Hz, 3H, −CH<sub>3</sub>); IR (KBr) 1580, 1506, 1460, 1422, 1372, 1317, 1240, 1216, 1142 cm<sup>–1</sup>; HPLC purity 95.8%.</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1-Isopropyl-4-nitro-1<i>H</i>-pyrazole (<b>13</b>)</h4><div class="NLM_p last">To a solution of 4-nitro-1<i>H</i>-pyrazole (200 mg, 1.77 mmol) in dry <i>N</i>,<i>N</i>-dimethylformamide (5 mL) were added 2-bromopropane (2.83 g, 23 mmol) and potassium carbonate (3.9 g, 28.3 mmol). The reaction mixture was heated to 50 °C and stirred for 30 min. After cooling to room temperature, the reaction mixture was taken up in ethyl acetate and water. The organic phase was washed, dried, and evaporated. The residue was purified by silica gel column chromatography (ethyl acetate/petroleum ether = 1/50, v/v) to give a white solid: yield 252 mg (92%); mp 64–65 °C; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.93 (s, 1H, ArH), 8.26 (s, 1H, ArH), 4.59–4.56 (m, 1H, −CH−), 1.45 (d, <i>J</i> = 6.7 Hz, 6H, −CH<sub>3</sub>).</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1-Isopropyl-1<i>H</i>-pyrazol-4-amine (<b>14</b>)</h4><div class="NLM_p last">A mixture of 1-isopropyl-4-nitro-1<i>H</i>-pyrazole (<b>13</b>) (250 mg, 1.61 mmol), 10% palladium–carbon (25 mg), and ethyl acetate (3 mL) was stirred at room temperature under a hydrogen atmosphere at normal pressure for 5 h. The catalyst was filtered off, and the filtrate was concentrated. The oily residue <b>14</b> was utilized without further purification: <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.03 (s, 1H, ArH), 6.88 (s, 1H, ArH), 4.36–4.17 (m, 1H, −CH−), 3.81 (br, 2H, −NH<sub>2</sub>), 1.32 (d, <i>J</i> = 6.7 Hz, 6H, −CH<sub>3</sub>).</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1-(Bromomethyl)-4-nitro-2-(trifluoromethyl)benzene (<b>16</b>)</h4><div class="NLM_p last">To a solution of 1-methyl-4-nitro-2-trifluorotoluene (200 mg, 0.97 mmol) in 1,2-dichloroethane (5 mL) were added <i>N</i>-bromosuccinimide (208.2 mg, 1.17 mmol) and azodiisobutyronitrile (16 mg, 0.097 mmol). The reaction mixture was heated to 90 °C and maintained for 12 h. After cooling to room temperature, the reaction mixture was extracted with dichloromethane, dried with Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (100% petroleum ether) to give the title compound: yield 127 mg (46.01%); MS [M + H]<sup>+</sup> 283.1.</div></div><div id="sec5_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-(4-Nitro-2-(trifluoromethyl)benzyl)morpholine (<b>17</b>)</h4><div class="NLM_p last">To a solution of 1-bromomethyl-4-nitro-2-trifluorotoluene (<b>16</b>) (200 mg, 0.7 mmol) and morpholine (122 mg, 1.4 mmol) dissolved in dry tetrahydrofuran (8 mL) was added potassium carbonate (97.3 mg, 0.7 mmol), and the mixture was stirred at room temperature for 16 h. The reaction mixture was concentrated, and the residue was purified by silica gel column chromatography to give a pale yellow solid: yield 194.6 mg (96%); mp 87–88 °C; MS [M + H]<sup>+</sup> 291.1; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.52 (d, <i>J</i> = 8.6 Hz, 1H, ArH), 8.42 (s, 1H, ArH), 8.12 (d, <i>J</i> = 8.6 Hz, 1H, ArH), 3.74 (s, 2H, −CH<sub>2</sub>−), 3.68–3.53 (m, 4H, −CH<sub>2</sub>−), 2.47–2.32 (m, 4H, −CH<sub>2</sub>−).</div></div><div id="sec5_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-(Morpholinomethyl)-3-(trifluoromethyl)aniline (<b>18</b>)</h4><div class="NLM_p last">A mixture of 4-(4-nitro-2-(trifluoromethyl)benzyl)morpholine (<b>17</b>) (100 mg, 0.34 mmol), 10% palladium–carbon (20 mg), and ethanol (3 mL) was stirred at room temperature under a hydrogen atmosphere at normal pressure for 5 h. The catalyst was filtered off, and the filtrate was concentrated to give the title compound: yield 88 mg (99.4%); mp 72–74 °C; MS [M + H]<sup>+</sup> 261.1; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.30 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 6.86 (s, 1H, ArH), 6.76 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 5.46 (s, 2H, −CH<sub>2</sub>−), 3.65–3.48 (m, 4H, −CH<sub>2</sub>−), 2.50 (d, <i>J</i> = 1.6 Hz, 2H, −CH<sub>2</sub>−), 2.31 (m, 2H, −CH<sub>2</sub>−).</div></div><div id="sec5_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 1-(4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)-3-(1-isopropyl-1<i>H</i>-pyrazol-4-yl)urea (<b>19</b>)</h4><div class="NLM_p last">To a solution of 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (100 mg, 0.29 mmol) in dry dichloromethane (5 mL) were added dropwise pyridine (24 μL) and 4-nitrophenyl chloroformate (58.5 mg, 0.29 mmol), and the mixture was stirred for 30 min at room temperature. Then, 1-isopropyl-1<i>H</i>-pyrazol-4-amine (<b>14</b>) (36.3 mg, 0.29 mmol) and triethylamine (32 μL) were added, and the mixture was stirred for 24 h at room temperature. The reaction mixture was filtered, and the filter cake was recrystallized with methanol and dichloromethane to give a light white solid: yield 7 mg (4.8%); mp 178–182 °C; MS [M + H]<sup>+</sup> 499.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.80 (s, 1H, −NHCONH−), 8.74 (s, 1H, −NHCONH−), 8.44 (s, 1H, H-quinazoline), 8.17 (d, <i>J</i> = 8.7 Hz, 1H, H-naphthalene), 8.03 (d, <i>J</i> = 8.4 Hz, 1H, H-naphthalene), 7.84 (s, 1H, H-quinazoline), 7.77–7.75 (m, 2H, ArH), 7.63 (t, <i>J</i> = 6.75 Hz, 1H, H-naphthalene), 7.58–7.35 (m, 4H, ArH), 4.52–4.37 (m, 1H, −CH−), 4.03 (s, 6H, −OCH<sub>3</sub>), 1.40 (d, <i>J</i> = 6.5 Hz, 6H, −CH<sub>3</sub>); IR (KBr) 3414, 3280, 1636, 1504, 1375, 1232 cm<sup>–1</sup>; HPLC purity 96.4%.</div></div><div id="sec5_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 1-(4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)-3-(5-methylisoxazol-3-yl)urea (<b>20</b>)</h4><div class="NLM_p last">To a solution of triphosgene (756 mg, 2.55 mmol) in dry dichloromethane (6 mL) was added dropwise slowly at 0 °C 5-methylisoxazole-3-amine (500 mg, 5.1 mmol) in dry dichloromethane (4 mL), and the mixture was stirred for 3 h at room temperature. The reaction mixture was filtered, and triethylamine was added to the filtrate until no white smoke was liberated. Then, 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (200 mg, 0.58 mmol) was added and stirred for 24 h at room temperature. The resulting mixture was filtered, and the filter cake was washed with dichloromethane to obtain a light white solid: yield 21 mg (7.8%); mp >250 °C; MS [M + H]<sup>+</sup> 472.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (s, 1H, −NHCONH−), 9.13 (s, 1H, −NHCONH−), 8.45 (s, 1H, H-quinazoline), 8.17 (d, <i>J</i> = 8.4 Hz, 1H, H-naphthalene), 8.04 (d, <i>J</i> = 8.3 Hz, 1H, H-naphthalene), 7.86–7.73 (m, 2H, ArH), 7.73 (t, <i>J</i> = 7.59 Hz, 1H, H-naphthalene), 7.61–7.37 (m, 3H, ArH), 6.57 (s, 1H, H-isoxazo), 4.02 (d, <i>J</i> = 3.5 Hz, 6H, −OCH<sub>3</sub>), 2.39 (s, 3H, −CH<sub>3</sub>); IR (KBr) 3277, 1676, 1645, 1564, 1502, 1422, 1379, 1255, 1231 cm<sup>–1</sup>; HPLC purity 95.2%.</div></div><div id="sec5_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 1-Cyclopropyl-3-(4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)urea (<b>21</b>)</h4><div class="NLM_p last">To a solution of 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (200 mg, 0.58 mmol) in dry dichloromethane (10 mL) were added dropwise pyridine (48 μL) and 4-nitrophenyl chloroformate (117 mg, 0.58 mmol), and the mixture was stirred for 30 min at room temperature. Then, cyclopropylamine (42 μL, 0.58 mmol) and triethylamine (48 μL) were added and stirred for 24 h at room temperature. The reaction mixture was filtered, and the filter cake was washed with dichloromethane to give a light white solid: yield 182.8 mg (73.1%); mp >250 °C; MS [M + H]<sup>+</sup> 431.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.49 (s, 1H, −NHCONH−), 8.43 (s, 1H, −NHCONH−), 8.12 (d, <i>J</i> = 8.5 Hz, 1H, H-naphthalene), 7.98 (d, <i>J</i> = 8.4 Hz, 1H, H-naphthalene), 7.74–7.72 (m, 2H, ArH), 7.65 (t, <i>J</i> = 7.26 Hz, 1H, H-naphthalene), 7.54 (m, <i>J</i> = 7.26 Hz, 1H, H-naphthalene), 7.45–7.35 (m, 2H, ArH), 6.8 (d, <i>J</i> = 2.7 Hz, 1H, H-naphthalene), 4.02 (d, <i>J</i> = 2.8 Hz, 6H, −OCH<sub>3</sub>), 2.68–2.57 (m, 1H, −CH−), 0.75–0.62 (m, 2H, −CH<sub>2</sub>−), 0.55–0.42 (m, 2H, −CH<sub>2</sub>−); IR (KBr) 3254, 1640, 1568, 1508, 1455, 1422, 1376, 1231 cm<sup>–1</sup>; HPLC purity 97.1%.</div></div><div id="sec5_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 1-(4-(1-Cyanocyclopentyl)phenyl)-3-(4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)urea (<b>22</b>)</h4><div class="NLM_p last">To a solution of triphosgene (239 mg, 0.80 mmol) in dry dichloromethane (4 mL) was added dropwise slowly at 0 °C 1-(4-aminophenyl)cyclopentane-1-carbonitrile (300 mg, 1.6 mmol) in dry dichloromethane (2 mL), and the mixture was stirred for 3 h at room temperature. The reaction mixture was filtered, and triethylamine was added to the filtrate until no white smoke was liberated. Then, 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (100 mg, 0.29 mmol) was added, and the mixture was stirred for 24 h at room temperature. The resulting mixture was filtered, and the filter cake was washed with tetrahydrofuran to obtain a light white solid: yield 42 mg (25.9%); mp 242–245 °C; MS [M + H]<sup>+</sup> 582.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.20 (s, 1H, −NHCONH−), 8.87 (s, 1H, −NHCONH−), 8.43 (s, 1H, H-quinazoline), 8.21 (d, <i>J</i> = 8.6 Hz, 1H, H-naphthalene), 8.04 (d, <i>J</i> = 8.3 Hz, 1H, H-naphthalene), 7.78–7.76 (m, 2H, ArH), 7.71 (t, <i>J</i> = 8.0 Hz, 1H, H-naphthalene), 7.61–7.36 (m, 7H, ArH), 4.03 (d, <i>J</i> = 3.9 Hz, 6H), 2.40–2.37 (m, 2H, −CH<sub>2</sub>−), 2.15–1.99 (m, 2H, −CH<sub>2</sub>−), 1.90–1.88 (m, 4H, −CH<sub>2</sub>−); IR (KBr) 3273, 1642, 1561, 1424, 1378, 1233, 1187, 831 cm<sup>–1</sup>; HPLC purity 96.5%.</div></div><div id="sec5_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-(4-((6,7-Dimethoxyquinazolin-4-yl)oxy)naphthalen-1-yl)-3-(4-(morpholinomethyl)-3-(trifluoromethyl)phenyl)urea (<b>23</b>)</h4><div class="NLM_p last">To a solution of triphosgene (1.36 g, 7.19 mmol) in dry dichloromethane (4 mL) was added dropwise slowly at 0 °C 4-(morpholinomethyl)-3-(trifluoromethyl)aniline (<b>18</b>) (300 mg, 1.2 mmol) in dry dichloromethane (2 mL), and the mixture was stirred for 24 h at room temperature. The reaction mixture was filtered, and triethylamine was added to the filtrate until no white smoke was liberated. Then, 4-((6,7-dimethoxyquinazolin-4-yl)oxy)naphthalen-1-amine (<b>4a</b>) (100 mg, 0.29 mmol) was added, and the mixture was stirred for 24 h at room temperature. The resulting mixture was filtered, and the filter cake was washed with tetrahydrofuran to obtain a light white solid: yield 28 mg (15.3%); mp 234–238 °C; MS [M + H]<sup>+</sup> 634.2; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.46 (s, 1H, −NHCONH−), 8.95 (s, 1H, −NHCONH−), 8.45 (s, 1H, H-quinazoline), 8.20 (d, <i>J</i> = 8.3 Hz, 1H, H-naphthalene), 8.06 (s, 1H, H-quinazoline), 8.02 (dd, <i>J</i><sub>1</sub> = 12.7 Hz, <i>J</i><sub>2</sub> = 20.9 Hz, 1H, ArH), 7.81–7.40 (m, 8H, ArH), 4.03 (m, 6H, −CH<sub>2</sub>−), 3.58–3.56 (m, 6H, −CH<sub>2</sub>−), 2.38 (s, 4H, −CH<sub>2</sub>−); IR (KBr) 3339, 1645, 1503, 1380, 1318, 1233, 1116 cm<sup>–1</sup>; HPLC purity 95.5%.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Biological Assay</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> In Vitro Kinase Assay</h4><div class="NLM_p last">Inhibition ratio determination and IC<sub>50</sub> testing at a single concentration (10 μM) were entrusted to Reaction Biology Corporation. Cisbio’s HTRF KinEASE Kit was used to test the enzyme inhibitory activity. This method utilizes a unique substrate containing a single phosphorylation site recognized by a europium cryptate (Eu(K))-labeled antibody to phosphotyrosine. Based on homogeneous time-resolved fluorescence (HTRF), all KinEASE assays involve two steps: the enzymatic step and the detection step with HTRF reagents. In the kinase reaction step, 2 μL of VEGFR-2 kinase solution, 2 μL of biotin substrate, and 4 μL of compound (SEB-supplemented kinase buffer) were added to each well for incubation. Then, 2 μL of ATP was added at room temperature (18–22 °C) to initiate the reaction, which was run for 1 h. In the second step, detection reagents including 5 μL of streptavidin-XL665 (SA-XL665) in EDTA and 5 μL tyrosine kinase antibody-Eu(K) in EDTA were added to each well and incubated for 1 h at room temperature. The Beckman Coulter platform HTRF detection module was used to detect the signal. The detection reagents catch the phosphorylated substrate and the resulting HTRF signal is proportional to the amount of phosphorylation.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> According to the initial screening results, compounds <b>6</b>–<b>8</b>, <b>10</b>–<b>11</b>, and <b>19</b>–<b>23</b> were diluted in 8 gradient concentrations for rescreening. GraphPad Prism 5.0 software was used to calculate the IC<sub>50</sub> values for each compound (<a href="https://www.graphpad.com/scientific-software/prism/" class="extLink">https://www.graphpad.com/scientific-software/prism/</a>). Each test was repeated three times.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Antiproliferative Assay</h4><div id="sec5_3_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Endothelial Cell Proliferation Assays</h5><div class="NLM_p last">All cell activity tests were entrusted to Crown Bioscience Inc. The luciferase in the CTG reagent uses luciferin, ATP and oxygen as substrates to produce oxidized luciferin and release energy in the form of light. The amount of light produced is proportional to the total amount of ATP, which can reflect the total number of viable cells (HUVECs, MDA-MB-435, and A431). The anticell proliferation rate can be calculated by the fluorescence intensity. This method included several steps. The first step is cell planking, where the cells in the exponential growth phase were collected and the viable cells were counted with Vi-Cell XR cell counting instrument. According to the density in the cell culture medium, the cell suspension was adjusted and 90 μL of was added to each well of a 96-well cell culture plate. The final cell concentration was approximately 2000–4000 cells per well (the specific cell density was adjusted according to cell growth). The next step was compound dispensation, where the target compound was dissolved from 10 mM stock solutions in DMSO, and then these solutions were diluted 10-fold with the medium solution. A total of 10 μL of the 10-fold compound dilution was added per well to each cell line, leading to a final drug concentration of 10 μM and a final DMSO concentration of 0.1%. The plate was placed in an incubator containing 5% CO<sub>2</sub> at 37 °C for 72 h. Nexy was the plate detection step, where according to the manufacturer instructions, 50 μL of CTG solution that was previously thawed and equilibrated to room temperature was added to each well after 72 h of drug treatment. A microplate oscillator was used to mix the solution for 2 min. After a 10 min incubation at room temperature, the fluorescence signal value was measured by an Envision2104 plate reader. Those with the cell inhibition over than 80% at 10 μM concentration were further submitted for a 10 concentrations test (from 1 nM to 100 μM). Data processing: inhibition ratio = 1 – <i>V</i><sub>sample</sub>/<i>V</i><sub>vehicle control</sub> × 100%. <i>V</i><sub>sample</sub> is for drug treatment group while <i>V</i><sub>vehicle control</sub> is for solvent control group. The GraphPad Prism 5.0 software was used to draw nonlinear regression model and S type dose survival rate curve, and then calculate IC<sub>50</sub> values. Each test was repeated three times.</div></div><div id="sec5_3_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> Growth Percent on 60 Cell Lines by NCI</h5><div class="NLM_p">All detailed growth methodology is described on the NCI Web site (<a href="https://dtp.cancer.gov/discovery_development/nci-60/methodology.htm" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/methodology.htm</a>). Only those compounds that exhibited significant growth inhibition in the One-Dose Screen were evaluated against the 60 cell line panel at five concentrations. The human tumor cell lines for the cancer screening panel were grown in RPMI 1640 medium that included 5% fetal bovine serum and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine. Cells were inoculated into 96-well microtiter plates in 100 μL aliquots at plating densities ranging from 5000 to 40 000 cells/well based on the doubling time of individual cell lines. Prior to the addition of experimental drugs, the microtiter plates were incubated at 37 °C, 5% CO<sub>2</sub>, 95% air and 100% relative humidity for 24 h after cell inoculation. Then, after 24 h, two plates of each cell line were fixed in situ with trichloroacetic acid (TCA), representing a measurement of the cell population for each cell line at the time of drug addition (Tz). Experimental drugs were solubilized in dimethyl sulfoxide (DMSO) at 400-fold greater than the desired final maximum test concentration and stored frozen prior to use. At the time of drug addition, an aliquot of frozen concentrate was thawed and diluted to twice the desired final maximum test concentration with complete medium containing 50 μg/mL gentamicin.</div><div class="NLM_p">After drug addition, the plates were incubated for an additional 48 h at 37 °C, 5% CO<sub>2</sub>, 95% air, and 100% relative humidity. For adherent cells, the assay was terminated by the addition of cold TCA. Cells were fixed in situ by the gentle addition of 50 μL of cold 50% (w/v) TCA (final concentration 10% TCA) and incubated for 60 min at 4 °C. The supernatant was discarded, and the plates were washed five times with tap water and air-dried. Sulforhodamine B (SRB) solution (100 μL) at 0.4% (w/v) in 1% acetic acid was added to each well, and the plates were incubated for 10 min at room temperature. After staining, unbound dye was removed by washing five times with 1% acetic acid and the plates were air-dried. The bound stain was subsequently solubilized with 10 mM Trizma base, and the absorbance was read by an automated plate reader at 515 nm. For cell suspensions, the methodology was the same except that the assay was terminated by fixing the settled cells at the bottom of the wells by gently adding 50 μL of 80% TCA (final concentration, 16% TCA). Using seven absorbance measurements including time zero (Tz), control growth (C), and test growth in the presence of drug (Ti), the percentage growth was calculated at each of the drug concentrations levels. The percent growth inhibition was calculated as  for concentrations for which Ti ≥ Tz, and  for concentrations for which Ti < Tz.</div><div class="NLM_p last">Two dose response parameters were calculated for each experimental agent. Growth inhibition of 50% (GI<sub>50</sub>) was calculated from , which is the drug concentration resulting in a 50% reduction in the net protein increase (as measured by SRB staining) in control cells during drug incubation. The LC<sub>50</sub> (concentration of drug resulting in a 50% reduction in the measured protein at the end of the drug treatment as compared to that at the beginning), indicating a net loss of cells following treatment. It was calculated from . Values were calculated for each of these three parameters if the desired level of activity was reached; however, if the effect was not reached or was exceeded, the value for that parameter was expressed as greater or less than the maximum or minimum concentration tested.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01091">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02361" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01091" class="ext-link">10.1021/acs.jmedchem.7b01091</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">RMSD values for the 24 crystal structures [protein (blue) and ligand (orange)] during the 20 ns MD simulation; number of hydrogen bonds for the 24 crystal ligands during the 20 ns MD simulation; RMSD values and the number of hydrogen bonds for 9 designed compounds during the 20 ns MD simulation; Δ<i>G</i> distribution of all 28 crystal ligands during the 20 ns simulation; ΔG distribution of the 10 designed compounds during the 20 ns simulation; developmental therapeutics program one dose mean graph of the growth percent of cell lines for compounds <b>6</b>, <b>19</b>, <b>22</b>, and <b>23</b>; developmental therapeutics program five dose mean graph of the growth percent as well as GI<sub>50</sub>, LC<sub>50</sub>, and TGI values against 58 cell lines for compound <b>23</b>; comparison of GI<sub>50</sub>, TGI, and LC<sub>50</sub> values, respectively, of individual the cell lines for compound <b>23</b>, sorafenib, and sunitinib; data set for building the binding affinity model; enzymatic inhibition rate of compound <b>23</b> against the targets with a relatively high sequence identity with VEGFR-2 from the kinome wide screening (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">In-house database generated by BREED and scored by molecular docking (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_002.zip" class="ext-link">ZIP</a>)</p></li><li><p class="inline">Docking complexes between VEGFR-2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>) and the compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_003.zip" class="ext-link">ZIP</a>)</p></li><li><p class="inline">SMILES format of the 10 designed compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf">jm7b01091_si_001.pdf (2.84 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_002.zip">jm7b01091_si_002.zip (944.93 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_003.zip">jm7b01091_si_003.zip (1.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_004.csv">jm7b01091_si_004.csv (0.88 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01091" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33441" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33441" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tao Lu</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Molecular Design and Drug Discovery, School of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span>; 
    <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Natural Medicines, China
Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d4b8a1a0b5bb94b7a4a1fab1b0a1fab7ba"><span class="__cf_email__" data-cfemail="c9a5bcbda8a689aab9bce7acadbce7aaa7">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Jiao</span> - <span class="hlFld-Affiliation affiliation">School
of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1900-8523" title="Orcid link">http://orcid.org/0000-0003-1900-8523</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b2d8dbd3ddcbc7f2d1c2c79cd7d6c79cd1dc"><span class="__cf_email__" data-cfemail="315b58505e4844715241441f5455441f525f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanmin Zhang</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Molecular Design and Drug Discovery, School of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yadong Chen</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Molecular Design and Drug Discovery, School of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danfeng Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lu Wang</span> - <span class="hlFld-Affiliation affiliation">School
of Science, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript. All authors contributed equally.</p></li><li><span class="author-information-subsection-header">Funding</span><p>The work was funded by National Natural Science Foundation of China (81302634 and 21302225), the Natural Science Foundation of Jiangsu Province (BK20130662), the Fundamental Research Funds for the Central Universities (PT2014LX0072), the Priority Academic Program Development of Jiangsu Higher Education Institutions (PAPD), and the Jiangsu Qinglan Project.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d413e3842-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the NCI Division of Cancer Treatment & Diagnosis for the NCI-60 screening of our compounds, and we thank Dr. C. Leigh Allen for revising the language of our manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i52" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i52"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i53" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i53"> Abbreviations Used</h2><tr><td class="NLM_term">VEGF</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor</p></td></tr><tr><td class="NLM_term">VEGFR-2</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor receptor 2</p></td></tr><tr><td class="NLM_term">TKI</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitors</p></td></tr><tr><td class="NLM_term">HUVECs</td><td class="NLM_def"><p class="first last">human umbilical vein endothelial cells</p></td></tr><tr><td class="NLM_term">CNS cancer</td><td class="NLM_def"><p class="first last">central nervous system cancer</p></td></tr><tr><td class="NLM_term">NCI</td><td class="NLM_def"><p class="first last">National Cancer Institute</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small cell lung cancer</p></td></tr><tr><td class="NLM_term">PlGF</td><td class="NLM_def"><p class="first last">placental growth factor</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">CFDA</td><td class="NLM_def"><p class="first last">China Food and Drug Administration</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug design</p></td></tr><tr><td class="NLM_term">MM/PBSA</td><td class="NLM_def"><p class="first last">molecular mechanics/Poisson–Boltzmann surface area</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">3D-QSAR</td><td class="NLM_def"><p class="first last">three-dimensional quantitative structure activity relationship</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein data bank</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">total growth inhibition</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86682" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86682" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 63 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Holmes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, O. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, M. J.</span><span> </span><span class="NLM_article-title">Vascular endothelial growth factor receptor-2: structure, function, intracellular signalling and therapeutic inhibition</span> <span class="citation_source-journal">Cell. Signalling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2003</span><span class="NLM_x">–</span> <span class="NLM_lpage">2012</span><span class="refDoi"> DOI: 10.1016/j.cellsig.2007.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2Fj.cellsig.2007.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17658244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFylsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2007&pages=2003-2012&author=K.+Holmesauthor=O.+L.+Robertsauthor=A.+M.+Thomasauthor=M.+J.+Cross&title=Vascular+endothelial+growth+factor+receptor-2%3A+structure%2C+function%2C+intracellular+signalling+and+therapeutic+inhibition&doi=10.1016%2Fj.cellsig.2007.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Vascular endothelial growth factor receptor-2: structure, function, intracellular signalling and therapeutic inhibition</span></div><div class="casAuthors">Holmes, Katherine; Roberts, Owain Ll; Thomas, Angharad M.; Cross, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2003-2012</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Vascular endothelial growth factors (VEGFs) regulate vascular development, angiogenesis and lymphangiogenesis by binding to a no. of receptors.  VEGFR-1 is required for the recruitment of hematopoietic stem cells and the migration of monocytes and macrophages, VEGFR-2 regulates vascular endothelial function and VEGFR-3 regulates lymphatic endothelial cell function.  Over the last decade, considerable progress has been made in delineating the VEGFR-2 specific intracellular signaling cascades leading to proliferation, migration, survival and increased permeability, each of which contributes to the angiogenic response.  Furthermore, therapeutic inhibition of VEGFR-2 action is now having an impact in the clinic for the treatment of a no. of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTYxPMokVV57Vg90H21EOLACvtfcHk0lhUSdGBC_EbHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFylsLs%253D&md5=44a935f36d29b000d2a9ff514709e9c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2007.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2007.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DK.%26aulast%3DRoberts%26aufirst%3DO.%2BL.%26aulast%3DThomas%26aufirst%3DA.%2BM.%26aulast%3DCross%26aufirst%3DM.%2BJ.%26atitle%3DVascular%2520endothelial%2520growth%2520factor%2520receptor-2%253A%2520structure%252C%2520function%252C%2520intracellular%2520signalling%2520and%2520therapeutic%2520inhibition%26jtitle%3DCell.%2520Signalling%26date%3D2007%26volume%3D19%26spage%3D2003%26epage%3D2012%26doi%3D10.1016%2Fj.cellsig.2007.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Folkman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klagsbrun, M.</span><span> </span><span class="NLM_article-title">Angiogenic factors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">442</span><span class="NLM_x">–</span> <span class="NLM_lpage">447</span><span class="refDoi"> DOI: 10.1126/science.2432664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1126%2Fscience.2432664" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=442-447&author=J.+Folkmanauthor=M.+Klagsbrun&title=Angiogenic+factors&doi=10.1126%2Fscience.2432664"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.2432664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2432664%26sid%3Dliteratum%253Aachs%26aulast%3DFolkman%26aufirst%3DJ.%26aulast%3DKlagsbrun%26aufirst%3DM.%26atitle%3DAngiogenic%2520factors%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D442%26epage%3D447%26doi%3D10.1126%2Fscience.2432664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kiselyov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakin, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tkachenko, S. E.</span><span> </span><span class="NLM_article-title">VEGF/VEGFR signalling as a target for inhibiting angiogenesis</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1517/13543784.16.1.83</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1517%2F13543784.16.1.83" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17155856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1yisL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2007&pages=83-107&author=A.+Kiselyovauthor=K.+V.+Balakinauthor=S.+E.+Tkachenko&title=VEGF%2FVEGFR+signalling+as+a+target+for+inhibiting+angiogenesis&doi=10.1517%2F13543784.16.1.83"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">VEGF/VEGFR signalling as a target for inhibiting angiogenesis</span></div><div class="casAuthors">Kiselyov, Alex; Balakin, Konstantin V.; Tkachenko, Sergey E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-107</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  VEGFs and a resp. family of tyrosine kinases receptors (VEGFRs) are key proteins modulating angiogenesis, the formation of new vasculature from an existing vascular network.  There has been considerable evidence in vivo, including clin. observations, that abnormal angiogenesis is implicated in a no. of disease conditions, which include rheumatoid arthritis, inflammation, cancer, psoriasis, degenerative eye conditions and others.  Antiangiogenic therapies based on inhibition of VEGF/VEGFR signalling were reported to be powerful clin. strategies in oncol. and ophthalmol.  Current efforts have yielded promising clin. data for several antiangiogenic therapeutics.  In this review, the authors elucidate key aspects of VEGFR signalling, as well as clin. relevant strategies for the inhibition of VEGF-induced angiogenesis, with an emphasis on small-mol. VEGFR inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphlleEmD9eirVg90H21EOLACvtfcHk0lhUSdGBC_EbHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1yisL7J&md5=7926ed15c751b2903bed9eefb8ef9f98</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1517%2F13543784.16.1.83&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.16.1.83%26sid%3Dliteratum%253Aachs%26aulast%3DKiselyov%26aufirst%3DA.%26aulast%3DBalakin%26aufirst%3DK.%2BV.%26aulast%3DTkachenko%26aufirst%3DS.%2BE.%26atitle%3DVEGF%252FVEGFR%2520signalling%2520as%2520a%2520target%2520for%2520inhibiting%2520angiogenesis%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2007%26volume%3D16%26spage%3D83%26epage%3D107%26doi%3D10.1517%2F13543784.16.1.83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Musumeci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brullo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenone, S.</span><span> </span><span class="NLM_article-title">Vascular endothelial growth factor (VEGF) receptors: drugs and new inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">10797</span><span class="NLM_x">–</span> <span class="NLM_lpage">10822</span><span class="refDoi"> DOI: 10.1021/jm301085w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301085w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10797-10822&author=F.+Musumeciauthor=M.+Radiauthor=C.+Brulloauthor=S.+Schenone&title=Vascular+endothelial+growth+factor+%28VEGF%29+receptors%3A+drugs+and+new+inhibitors&doi=10.1021%2Fjm301085w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm301085w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301085w%26sid%3Dliteratum%253Aachs%26aulast%3DMusumeci%26aufirst%3DF.%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DBrullo%26aufirst%3DC.%26aulast%3DSchenone%26aufirst%3DS.%26atitle%3DVascular%2520endothelial%2520growth%2520factor%2520%2528VEGF%2529%2520receptors%253A%2520drugs%2520and%2520new%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10797%26epage%3D10822%26doi%3D10.1021%2Fjm301085w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Folkman, J.</span><span> </span><span class="NLM_article-title">Angiogenesis in cancer, vascular, rheumatoid and other disease</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1038/nm0195-27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnm0195-27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=7584949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK2MXjs1KnsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1995&pages=27-30&author=J.+Folkman&title=Angiogenesis+in+cancer%2C+vascular%2C+rheumatoid+and+other+disease&doi=10.1038%2Fnm0195-27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Angiogenesis in cancer, vascular, rheumatoid and other disease</span></div><div class="casAuthors">Folkman, Judah</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-31</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">A review with ∼72 refs.  Recent discoveries of endogenous neg. regulators of angiogenesis, thrombospondin, angiostatin and glioma-derived angiogenesis inhibitory factor, all assocd. with neovascularized tumors, suggest a new paradigm of tumorigenesis.  It is now helpful to think of the switch to the angiogenic phenotype as a net balance of pos. and neg. regulators of blood vessel growth.  The extent to which the neg. regulators are decreased during this switch may dictate whether a primary tumor grows rapidly or slowly and whether metastases grow at all.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpF2nWHt2eJbVg90H21EOLACvtfcHk0lhUGrGladv1xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjs1KnsLk%253D&md5=849f4f3828ac5d78886d666787f72d4f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnm0195-27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0195-27%26sid%3Dliteratum%253Aachs%26aulast%3DFolkman%26aufirst%3DJ.%26atitle%3DAngiogenesis%2520in%2520cancer%252C%2520vascular%252C%2520rheumatoid%2520and%2520other%2520disease%26jtitle%3DNat.%2520Med.%26date%3D1995%26volume%3D1%26spage%3D27%26epage%3D30%26doi%3D10.1038%2Fnm0195-27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Liotta, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steeg, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stetler-Stevenson, W. G.</span><span> </span><span class="NLM_article-title">Cancer metastasis and angiogenesis: an imbalance of positive and negative regulation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1016/0092-8674(91)90642-C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2F0092-8674%2891%2990642-C" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1991&pages=327-336&author=L.+A.+Liottaauthor=P.+S.+Steegauthor=W.+G.+Stetler-Stevenson&title=Cancer+metastasis+and+angiogenesis%3A+an+imbalance+of+positive+and+negative+regulation&doi=10.1016%2F0092-8674%2891%2990642-C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990642-C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990642-C%26sid%3Dliteratum%253Aachs%26aulast%3DLiotta%26aufirst%3DL.%2BA.%26aulast%3DSteeg%26aufirst%3DP.%2BS.%26aulast%3DStetler-Stevenson%26aufirst%3DW.%2BG.%26atitle%3DCancer%2520metastasis%2520and%2520angiogenesis%253A%2520an%2520imbalance%2520of%2520positive%2520and%2520negative%2520regulation%26jtitle%3DCell%26date%3D1991%26volume%3D64%26spage%3D327%26epage%3D336%26doi%3D10.1016%2F0092-8674%2891%2990642-C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Folkman, J.</span><span> </span><span class="NLM_article-title">Anti-angiogenesis: new concept for therapy of solid tumors</span> <span class="citation_source-journal">Ann. Surg.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">175</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">–</span> <span class="NLM_lpage">416</span><span class="refDoi"> DOI: 10.1097/00000658-197203000-00014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1097%2F00000658-197203000-00014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=5077799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADyaE3s%252FitV2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=1972&pages=409-416&author=J.+Folkman&title=Anti-angiogenesis%3A+new+concept+for+therapy+of+solid+tumors&doi=10.1097%2F00000658-197203000-00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-angiogenesis: new concept for therapy of solid tumors</span></div><div class="casAuthors">Folkman J</div><div class="citationInfo"><span class="NLM_cas:title">Annals of surgery</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">409-16</span>
        ISSN:<span class="NLM_cas:issn">0003-4932</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBThr2iCUqj90Q2ypr7jHkfW6udTcc2eZh7PgJhCzf2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3s%252FitV2gtA%253D%253D&md5=48381e6d0d06f30c9bd7e7d7d8a53d95</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1097%2F00000658-197203000-00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00000658-197203000-00014%26sid%3Dliteratum%253Aachs%26aulast%3DFolkman%26aufirst%3DJ.%26atitle%3DAnti-angiogenesis%253A%2520new%2520concept%2520for%2520therapy%2520of%2520solid%2520tumors%26jtitle%3DAnn.%2520Surg.%26date%3D1972%26volume%3D175%26spage%3D409%26epage%3D416%26doi%3D10.1097%2F00000658-197203000-00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bold, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McSheehy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brüggen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestan, J. r.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drückes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burglin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dürler, U.</span><span> </span><span class="NLM_article-title">A novel potent oral series of VEGFR2 inhibitors abrogate tumor growth by inhibiting angiogenesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01582</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01582" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=132-146&author=G.+Boldauthor=C.+Schnellauthor=P.+Furetauthor=P.+McSheehyauthor=J.+Br%C3%BCggenauthor=J.+r.+Mestanauthor=P.+W.+Manleyauthor=P.+Dr%C3%BCckesauthor=M.+Burglinauthor=U.+D%C3%BCrler&title=A+novel+potent+oral+series+of+VEGFR2+inhibitors+abrogate+tumor+growth+by+inhibiting+angiogenesis&doi=10.1021%2Facs.jmedchem.5b01582"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01582%26sid%3Dliteratum%253Aachs%26aulast%3DBold%26aufirst%3DG.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMcSheehy%26aufirst%3DP.%26aulast%3DBr%25C3%25BCggen%26aufirst%3DJ.%26aulast%3DMestan%26aufirst%3DJ.%2Br.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DDr%25C3%25BCckes%26aufirst%3DP.%26aulast%3DBurglin%26aufirst%3DM.%26aulast%3DD%25C3%25BCrler%26aufirst%3DU.%26atitle%3DA%2520novel%2520potent%2520oral%2520series%2520of%2520VEGFR2%2520inhibitors%2520abrogate%2520tumor%2520growth%2520by%2520inhibiting%2520angiogenesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D132%26epage%3D146%26doi%3D10.1021%2Facs.jmedchem.5b01582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Plate, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weich, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risau, W.</span><span> </span><span class="NLM_article-title">Vascular endothelial growth factor is a potential tumour angiogenesis factor in human gliomas in vivo</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1038/359845a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2F359845a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=1992&pages=845-848&author=K.+H.+Plateauthor=G.+Breierauthor=H.+A.+Weichauthor=W.+Risau&title=Vascular+endothelial+growth+factor+is+a+potential+tumour+angiogenesis+factor+in+human+gliomas+in+vivo&doi=10.1038%2F359845a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F359845a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F359845a0%26sid%3Dliteratum%253Aachs%26aulast%3DPlate%26aufirst%3DK.%2BH.%26aulast%3DBreier%26aufirst%3DG.%26aulast%3DWeich%26aufirst%3DH.%2BA.%26aulast%3DRisau%26aufirst%3DW.%26atitle%3DVascular%2520endothelial%2520growth%2520factor%2520is%2520a%2520potential%2520tumour%2520angiogenesis%2520factor%2520in%2520human%2520gliomas%2520in%2520vivo%26jtitle%3DNature%26date%3D1992%26volume%3D359%26spage%3D845%26epage%3D848%26doi%3D10.1038%2F359845a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Carmeliet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhardti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Declercq, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawling, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moons, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collen, D.</span><span> </span><span class="NLM_article-title">Abnormal blood vessel development and lethality in embryos lacking a single VEGF allele</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">380</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/380435a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2F380435a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=8602241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK28XitVKqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=1996&pages=435-439&author=P.+Carmelietauthor=V.+Ferreiraauthor=C.+Eberhardtiauthor=C.+Declercqauthor=J.+Pawlingauthor=L.+Moonsauthor=D.+Collen&title=Abnormal+blood+vessel+development+and+lethality+in+embryos+lacking+a+single+VEGF+allele&doi=10.1038%2F380435a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Abnormal blood vessel development and lethality in embryos lacking a single VEGF allele</span></div><div class="casAuthors">Carmeliet, Peter; Ferreira, Valerie; Breier, Georg; Pollefeyt, Saskia; Kieckens, Lena; Gertsenstein, Marina; Fahrig, Michaela; Vandenhoeck, Ann; Harpal, Kendraprasad; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">380</span>
        (<span class="NLM_cas:issue">6573</span>),
    <span class="NLM_cas:pages">435-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The endothelial cell-specific vascular endothelial growth factor (VEGF) and its cellular receptors Flt-1 and Flk-1 have been implicated in the formation of the embryonic vasculature.  This is suggested by their colocalized expression during embryogenesis and the impaired vessel formation in Flk-1 and Flt-1 deficient embryos.  However, because Flt-1 also binds placental growth factor, a VEGF homolog, the precise role of VEGF was unknown.  Here we report that formation of blood vessels was abnormal, but not abolished, in heterozygous VEGF-deficient (VEGF+/-) embryos, generated by aggregation of embryonic stem (ES) cells with tetraploid embryos (T-ES), and even more impaired in homozygous VEGF-deficient (VEGF-/-) T-ES embryos, resulting in death at mid-gestation.  Similar phenotypes were obsd. in F1-VEGF+/- embryos, generated by germline transmission.  We believe that this heterozygous lethal phenotype, which differs from the homozygous lethality in VEGF-receptor-deficient embryos, is unprecedented for a targeted autosomal gene inactivation, and is indicative of a tight dose-dependent regulation of embryonic vessel development by VEGF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobatChefZSx7Vg90H21EOLACvtfcHk0lgL7BVViSF2LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XitVKqtLg%253D&md5=93d159aabf470be75c118dad37a729cb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F380435a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F380435a0%26sid%3Dliteratum%253Aachs%26aulast%3DCarmeliet%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DV.%26aulast%3DEberhardti%26aufirst%3DC.%26aulast%3DDeclercq%26aufirst%3DC.%26aulast%3DPawling%26aufirst%3DJ.%26aulast%3DMoons%26aufirst%3DL.%26aulast%3DCollen%26aufirst%3DD.%26atitle%3DAbnormal%2520blood%2520vessel%2520development%2520and%2520lethality%2520in%2520embryos%2520lacking%2520a%2520single%2520VEGF%2520allele%26jtitle%3DNature%26date%3D1996%26volume%3D380%26spage%3D435%26epage%3D439%26doi%3D10.1038%2F380435a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span> </span><span class="NLM_article-title">Development and strategies of VEGFR-2/KDR inhibitors</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1839</span><span class="NLM_x">–</span> <span class="NLM_lpage">1852</span><span class="refDoi"> DOI: 10.4155/fmc.12.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.4155%2Ffmc.12.121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=23043480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVOhsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1839-1852&author=L.+Huangauthor=Z.+Huangauthor=Z.+Baiauthor=R.+Xieauthor=L.+Sunauthor=K.+Lin&title=Development+and+strategies+of+VEGFR-2%2FKDR+inhibitors&doi=10.4155%2Ffmc.12.121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Development and strategies of VEGFR-2/KDR inhibitors</span></div><div class="casAuthors">Huang, Lingyi; Huang, Zhengui; Bai, Zhiqiang; Xie, Rui; Sun, Liping; Lin, Kejiang</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1839-1852</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  VEGF is an important signaling protein involved in both vasculogenesis and angiogenesis.  As an essential receptor protein tyrosine kinase propagating cellular signal transduction processes, VEGFR-2 is a central target for drug discovery against tumor-assocd. angiogenesis.  Since the autophosphorylation of VEGFR-2 represents a key step in this signal pathway that contributes to angiogenesis, the discovery of small mol. inhibitors that block this reaction has attracted great interest for novel drugs research and development.  Advances in the understanding of catalytic cleft and the conformational changes of DFG motif have resulted in the development of small mol. inhibitors known as type I and type II.  High-resoln. crystal structures of various inhibitors in complex with the receptor offer an insight into the relationship among binding modes, inhibition mechanisms, activity, selectivity and resistance.  To control selectivity, improve activity and introduce intellectual property novelty, the strategies for the further development are discussed through structural and conformational anal. in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMgCdblpbQc7Vg90H21EOLACvtfcHk0lgL7BVViSF2LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVOhsb3O&md5=96cc54622cc92d45a8549eb88d551a7f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.121%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DK.%26atitle%3DDevelopment%2520and%2520strategies%2520of%2520VEGFR-2%252FKDR%2520inhibitors%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1839%26epage%3D1852%26doi%3D10.4155%2Ffmc.12.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Boyer, S. J.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of KDR (VEGFR-2) kinase: an overview of structure activity relationships</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">973</span><span class="NLM_x">–</span> <span class="NLM_lpage">1000</span><span class="refDoi"> DOI: 10.2174/1568026023393273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.2174%2F1568026023393273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=12171566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtVOitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=973-1000&author=S.+J.+Boyer&title=Small+molecule+inhibitors+of+KDR+%28VEGFR-2%29+kinase%3A+an+overview+of+structure+activity+relationships&doi=10.2174%2F1568026023393273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of KDR (VEGFR-2) kinase: an overview of structure activity relationships</span></div><div class="casAuthors">Boyer, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Hilversum, Netherlands)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">973-1000</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Kinase insert Domain contg. Receptor (KDR), alternatively referred to as VEGFR-2, is a receptor for Vascular Endothelial Growth Factors (VEGFs) and functions as a key regulator of angiogenesis, the process by which new capillaries are created from preexisting blood vessels.  The induction of angiogenesis, or the "angiogenic switch," is a crit. step in tumor progression, and inhibitors of KDR have been demonstrated both to induce tumor regression and reduce metastatic potential in preclin. models.  In the last few years, medicinal chemists have expanded the kinase selectivity profile of known inhibitor classes to include KDR, and also identified novel classes of KDR inhibitors.  This review presents structure activity relationships (SAR) of small mol. inhibitors of KDR, with an emphasis on the pharmacophore elements of the scaffolds employed.  Binding hypotheses based on x-ray crystallog. analyses will also be described.  Addnl., the efficacy of representative compds. in in vitro and in vivo models of tumor progression and angiogenesis are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8iX5xdLgktLVg90H21EOLACvtfcHk0lgL7BVViSF2LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtVOitbw%253D&md5=4c4445eac589bded1baed3140b918be8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2174%2F1568026023393273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026023393273%26sid%3Dliteratum%253Aachs%26aulast%3DBoyer%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520KDR%2520%2528VEGFR-2%2529%2520kinase%253A%2520an%2520overview%2520of%2520structure%2520activity%2520relationships%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2002%26volume%3D2%26spage%3D973%26epage%3D1000%26doi%3D10.2174%2F1568026023393273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lhXqHfgdVz3mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span> </span><span class="NLM_article-title">An integrated virtual screening approach for VEGFR-2 inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3163</span><span class="NLM_x">–</span> <span class="NLM_lpage">3177</span><span class="refDoi"> DOI: 10.1021/ci400429g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400429g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVajtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=3163-3177&author=Y.+Zhangauthor=S.+Yangauthor=Y.+Jiaoauthor=H.+Liuauthor=H.+Yuanauthor=S.+Luauthor=T.+Ranauthor=S.+Yaoauthor=Z.+Keauthor=J.+Xuauthor=X.+Xiongauthor=Y.+Chenauthor=T.+Lu&title=An+integrated+virtual+screening+approach+for+VEGFR-2+inhibitors&doi=10.1021%2Fci400429g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Integrated Virtual Screening Approach for VEGFR-2 Inhibitors</span></div><div class="casAuthors">Zhang, Yanmin; Yang, Shangyan; Jiao, Yu; Liu, Haichun; Yuan, Haoliang; Lu, Shuai; Ran, Ting; Yao, Sihui; Ke, Zhipeng; Xu, Jinxing; Xiong, Xiao; Chen, Yadong; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3163-3177</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, various virtual screening (VS) tools have been developed, and many successful screening campaigns have been showcased.  However, whether by conventional mol. docking or pharmacophore screening, the selection of virtual hits is based on the ranking of compds. by scoring functions or fit values, which remains the bottleneck of VS due to insufficient accuracy.  As the limitations of individual methods persist, a comprehensive comparison and integration of different methods may provide insights into selecting suitable methods for VS.  Here, we evaluated the performance of mol. docking, fingerprint-based 2D similarity and multicomplex pharmacophore in an individual and a combined manner, through a retrospective VS study on VEGFR-2 inhibitors.  An integrated two-layer work-flow was developed and validated through VS of VEGFR-2 inhibitors against the DUD-E database, which demonstrated improved VS performance through a ligand-based method ECFP_4, followed by mol. docking, and then a strict multicomplex pharmacophore.  Through a retrospective comparison with six published papers, this integrated approach outperformed 43 out of 45 methods, indicating a great effectiveness.  This kind of integrated VS approach can be extended to other targets for the screening and discovery of inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl0yCcaTmfp7Vg90H21EOLACvtfcHk0lhXqHfgdVz3mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVajtb%252FE&md5=5eac47fb02a20e95d900bb7eae65f54d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci400429g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400429g%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DKe%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DAn%2520integrated%2520virtual%2520screening%2520approach%2520for%2520VEGFR-2%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D3163%26epage%3D3177%26doi%3D10.1021%2Fci400429g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ellis, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hicklin, D. J.</span><span> </span><span class="NLM_article-title">Pathways mediating resistance to vascular endothelial growth factor–targeted therapy</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">6371</span><span class="NLM_x">–</span> <span class="NLM_lpage">6375</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-5287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1158%2F1078-0432.CCR-07-5287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=18927275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1KgtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=6371-6375&author=L.+M.+Ellisauthor=D.+J.+Hicklin&title=Pathways+mediating+resistance+to+vascular+endothelial+growth+factor%E2%80%93targeted+therapy&doi=10.1158%2F1078-0432.CCR-07-5287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pathways Mediating Resistance to Vascular Endothelial Growth Factor-Targeted Therapy</span></div><div class="casAuthors">Ellis, Lee M.; Hicklin, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6371-6375</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Vascular endothelial growth factor (VEGF)-targeted therapy has become an important treatment option for the management of a no. of human malignancies.  Unfortunately, a significant no. of patients do not respond to VEGF-targeted therapy when used as a single agent or in combination with chemotherapy.  Furthermore, the duration of benefit from VEGF-targeted therapy can be relatively short (weeks to months).  Ultimately, the vast majority of patients who initially respond to therapy will develop resistance.  To date, the mol. and cellular mechanisms assocd. with resistance to VEGF-targeted agents are poorly understood.  The mechanisms of action of anti-VEGF therapy are diverse, and it is entirely possible that resistance mechanisms are similarly diverse and depend on the tumor type.  A better understanding of these mechanisms will help in the selection of those patients that are more likely to benefit from VEGF-targeted therapy and also provide for the rational development of therapies that circumvent or overcome resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof9jwBsqNaErVg90H21EOLACvtfcHk0lhXqHfgdVz3mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1KgtbrJ&md5=5b5a06a0aa43bec60a2644a67a262820</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-5287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-5287%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DL.%2BM.%26aulast%3DHicklin%26aufirst%3DD.%2BJ.%26atitle%3DPathways%2520mediating%2520resistance%2520to%2520vascular%2520endothelial%2520growth%2520factor%25E2%2580%2593targeted%2520therapy%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D6371%26epage%3D6375%26doi%3D10.1158%2F1078-0432.CCR-07-5287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Jayson, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerbel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, A. L.</span><span> </span><span class="NLM_article-title">Antiangiogenic therapy in oncology: current status and future directions</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">388</span><span class="NLM_x">, </span> <span class="NLM_fpage">518</span><span class="NLM_x">–</span> <span class="NLM_lpage">529</span><span class="refDoi"> DOI: 10.1016/S0140-6736(15)01088-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2FS0140-6736%2815%2901088-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=26853587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12rur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2016&pages=518-529&author=G.+C.+Jaysonauthor=R.+Kerbelauthor=L.+M.+Ellisauthor=A.+L.+Harris&title=Antiangiogenic+therapy+in+oncology%3A+current+status+and+future+directions&doi=10.1016%2FS0140-6736%2815%2901088-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Antiangiogenic therapy in oncology: current status and future directions</span></div><div class="casAuthors">Jayson, Gordon C.; Kerbel, Robert; Ellis, Lee M.; Harris, Adrian L.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">10043</span>),
    <span class="NLM_cas:pages">518-529</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Angiogenesis, the formation of new blood vessels from pre-existing vessels, has been validated as a target in several tumor types through randomized trials, incorporating vascular endothelial growth factor (VEGF) pathway inhibitors into the therapeutic armory.  Although some tumors such as renal cell carcinoma, ovarian and cervical cancers, and pancreatic neuroendocrine tumors are sensitive to these drugs, others such as prostate cancer, pancreatic adenocarcinoma, and melanoma are resistant.  Even when drugs have yielded significant results, improvements in progression-free survival, and, in some cases, overall survival, are modest.  Thus, a crucial issue in development of these drugs is the search for predictive biomarkers-tests that predict which patients will, and will not, benefit before initiation of therapy.  Development of biomarkers is important because of the need to balance efficacy, toxicity, and cost.  Novel combinations of these drugs with other antiangiogenics or other classes of drugs are being developed, and the appreciation that these drugs have immunomodulatory and other modes of action will lead to combination regimens that capitalize on these newly understood mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDgPkePJIM_bVg90H21EOLACvtfcHk0ljZXdEuJfmryQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12rur0%253D&md5=7cca25ce3a4ded56b11a9571c050ce7f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2901088-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252901088-0%26sid%3Dliteratum%253Aachs%26aulast%3DJayson%26aufirst%3DG.%2BC.%26aulast%3DKerbel%26aufirst%3DR.%26aulast%3DEllis%26aufirst%3DL.%2BM.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26atitle%3DAntiangiogenic%2520therapy%2520in%2520oncology%253A%2520current%2520status%2520and%2520future%2520directions%26jtitle%3DLancet%26date%3D2016%26volume%3D388%26spage%3D518%26epage%3D529%26doi%3D10.1016%2FS0140-6736%2815%2901088-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Kuczynski, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bar-Zion, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butz, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Man, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daley, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yousef, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, F. S.</span><span> </span><span class="NLM_article-title">Co-option of liver vessels and not sprouting angiogenesis drives acquired sorafenib resistance in hepatocellular carcinoma</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1093/jnci/djw030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1093%2Fjnci%2Fdjw030" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2016&pages=1-13&author=E.+A.+Kuczynskiauthor=M.+Yinauthor=A.+Bar-Zionauthor=C.+R.+Leeauthor=H.+Butzauthor=S.+Manauthor=F.+Daleyauthor=P.+B.+Vermeulenauthor=G.+M.+Yousefauthor=F.+S.+Foster&title=Co-option+of+liver+vessels+and+not+sprouting+angiogenesis+drives+acquired+sorafenib+resistance+in+hepatocellular+carcinoma&doi=10.1093%2Fjnci%2Fdjw030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjw030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjw030%26sid%3Dliteratum%253Aachs%26aulast%3DKuczynski%26aufirst%3DE.%2BA.%26aulast%3DYin%26aufirst%3DM.%26aulast%3DBar-Zion%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.%2BR.%26aulast%3DButz%26aufirst%3DH.%26aulast%3DMan%26aufirst%3DS.%26aulast%3DDaley%26aufirst%3DF.%26aulast%3DVermeulen%26aufirst%3DP.%2BB.%26aulast%3DYousef%26aufirst%3DG.%2BM.%26aulast%3DFoster%26aufirst%3DF.%2BS.%26atitle%3DCo-option%2520of%2520liver%2520vessels%2520and%2520not%2520sprouting%2520angiogenesis%2520drives%2520acquired%2520sorafenib%2520resistance%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2016%26volume%3D108%26spage%3D1%26epage%3D13%26doi%3D10.1093%2Fjnci%2Fdjw030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Grothey, A.</span><span> </span><span class="NLM_article-title">VEGF inhibition beyond tumour progression</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">2</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1016/S1470-2045(12)70516-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2FS1470-2045%2812%2970516-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=23168363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADC%252BC3s7ltVyqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=2-3&author=A.+Grothey&title=VEGF+inhibition+beyond+tumour+progression&doi=10.1016%2FS1470-2045%2812%2970516-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">VEGF inhibition beyond tumour progression</span></div><div class="casAuthors">Grothey Axel</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSL0tZAs9R_3DAEfvC7EH1LfW6udTcc2eZW_vWHzMr34bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7ltVyqug%253D%253D&md5=b54521cc459fe4366d3d3aa8cadfa315</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970516-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970516-8%26sid%3Dliteratum%253Aachs%26aulast%3DGrothey%26aufirst%3DA.%26atitle%3DVEGF%2520inhibition%2520beyond%2520tumour%2520progression%26jtitle%3DLancet%2520Oncol.%26date%3D2013%26volume%3D14%26spage%3D2%26epage%3D3%26doi%3D10.1016%2FS1470-2045%2812%2970516-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">von Minckwitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puglisi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vrdoljak, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marschner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zielinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villanueva, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romieu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciruelos, E.</span><span> </span><span class="NLM_article-title">Bevacizumab plus chemotherapy versus chemotherapy alone as second-line treatment for patients with HER2-negative locally recurrent or metastatic breast cancer after first-line treatment with bevacizumab plus chemotherapy (TANIA): an open-label, randomised phase 3 trial</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1269</span><span class="NLM_x">–</span> <span class="NLM_lpage">1278</span><span class="refDoi"> DOI: 10.1016/S1470-2045(14)70439-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2FS1470-2045%2814%2970439-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=25273342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1269-1278&author=G.+von+Minckwitzauthor=F.+Puglisiauthor=J.+Cortesauthor=E.+Vrdoljakauthor=N.+Marschnerauthor=C.+Zielinskiauthor=C.+Villanuevaauthor=G.+Romieuauthor=I.+Langauthor=E.+Ciruelos&title=Bevacizumab+plus+chemotherapy+versus+chemotherapy+alone+as+second-line+treatment+for+patients+with+HER2-negative+locally+recurrent+or+metastatic+breast+cancer+after+first-line+treatment+with+bevacizumab+plus+chemotherapy+%28TANIA%29%3A+an+open-label%2C+randomised+phase+3+trial&doi=10.1016%2FS1470-2045%2814%2970439-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Bevacizumab plus chemotherapy versus chemotherapy alone as second-line treatment for patients with HER2-negative locally recurrent or metastatic breast cancer after first-line treatment with bevacizumab plus chemotherapy (TANIA): an open-label, randomized phase 3 trial</span></div><div class="casAuthors">von Minckwitz, Gunter; Puglisi, Fabio; Cortes, Javier; Vrdoljak, Eduard; Marschner, Norbert; Zielinski, Christoph; Villanueva, Cristian; Romieu, Gilles; Lang, Istvan; Ciruelos, Eva; De Laurentiis, Michele; Veyret, Corinne; de Duda, Sabine; Freudensprung, Ulrich; Srock, Stefanie; Gligorov, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1269-1278</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Combining bevacizumab with first-line or second-line chemotherapy improves progression-free survival in HER2-neg. locally recurrent or metastatic breast cancer.  We assessed the efficacy and safety of further bevacizumab therapy in patients with locally recurrent or metastatic breast cancer whose disease had progressed after treatment with bevacizumab plus chemotherapy.  Methods: In this open-label, randomized, phase 3 trial, we recruited patients who had HER2-neg. locally recurrent or metastatic breast cancer that had progressed after receiving 12 wk or more of first-line bevacizumab plus chemotherapy from 118 centers in 12 countries.  Patients were randomly assigned (1:1) by use of a central interactive voice response system using a block randomization schedule (block size four) stratified by hormone receptor status, first-line progression-free survival, selected chemotherapy, and lactate dehydrogenase concn., to receive second-line single-agent chemotherapy either alone or with bevacizumab (15 mg/kg every 3 wk or 10 mg/kg every 2 wk).  Second-line therapy was continued until disease progression, unacceptable toxicity, or consent withdrawal.  At progression, patients randomly assigned to chemotherapy alone received third-line chemotherapy without bevacizumab; those randomly assigned to bevacizumab continued bevacizumab with third-line chemotherapy.  The primary endpoint was progression-free survival from randomization to second-line progression or death in the intention-to-treat population.  This trial is ongoing, and registered with ClinicalTrials.gov, no. NCT01250379.  Findings: Between Feb 17, 2011, and Apr. 3, 2013, 494 patients were randomly assigned to treatment (247 in each group).  The median duration of follow-up at the time of this prespecified primary progression-free survival anal. was 15·9 mo (IQR 9·1-21·7) in the chemotherapy-alone group and 16·1 mo (10·6-22·7) in the combination group.  Progression-free survival was significantly longer for those patients treated with bevacizumab plus chemotherapy than for those with chemotherapy alone (median: 6·3 mo [95% CI 5·4-7·2] vs 4·2 mo [3·9-4·7], resp., stratified hazard ratio [HR] 0·75 [95% CI 0·61-0·93], two-sided stratified log-rank p=0·0068).  The most common grade 3 or more adverse events were hypertension (33 [13%] of 245 patients receiving bevacizumab plus chemotherapy vs 17 [7%] of 238 patients receiving chemotherapy alone), neutropenia (29 [12%] vs 20 [8%]), and hand-foot syndrome (27 [11%] vs 25 [11%]).  Grade 3 proteinuria occurred in 17 (7%) of 245 patients receiving combination therapy and one (<1%) of 238 patients receiving chemotherapy alone.  Serious adverse events were reported in 61 (25%) of 245 patients receiving bevacizumab plus chemotherapy vs. 44 (18%) of 238 patients receiving chemotherapy alone.  Interpretation: These results suggest that continued VEGF inhibition with further bevacizumab is a valid treatment option for patients with locally recurrent or metastatic HER2-neg. breast cancer whose disease was stabilized or responded to first-line bevacizumab with chemotherapy.  Funding: F Hoffmann-La Roche.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbwFFFDp-M1rVg90H21EOLACvtfcHk0ljZXdEuJfmryQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt73M&md5=cfcede8b1ef1bf73999c52adc26684e7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2814%2970439-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252814%252970439-5%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BMinckwitz%26aufirst%3DG.%26aulast%3DPuglisi%26aufirst%3DF.%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DVrdoljak%26aufirst%3DE.%26aulast%3DMarschner%26aufirst%3DN.%26aulast%3DZielinski%26aufirst%3DC.%26aulast%3DVillanueva%26aufirst%3DC.%26aulast%3DRomieu%26aufirst%3DG.%26aulast%3DLang%26aufirst%3DI.%26aulast%3DCiruelos%26aufirst%3DE.%26atitle%3DBevacizumab%2520plus%2520chemotherapy%2520versus%2520chemotherapy%2520alone%2520as%2520second-line%2520treatment%2520for%2520patients%2520with%2520HER2-negative%2520locally%2520recurrent%2520or%2520metastatic%2520breast%2520cancer%2520after%2520first-line%2520treatment%2520with%2520bevacizumab%2520plus%2520chemotherapy%2520%2528TANIA%2529%253A%2520an%2520open-label%252C%2520randomised%2520phase%25203%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2014%26volume%3D15%26spage%3D1269%26epage%3D1278%26doi%3D10.1016%2FS1470-2045%2814%2970439-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Escudier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szczylik, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gore, M.</span><span> </span><span class="NLM_article-title">Treatment selection in metastatic renal cell carcinoma: expert consensus</span> <span class="citation_source-journal">Nat. Rev. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1038/nrclinonc.2012.59</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnrclinonc.2012.59" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=22473096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVOrtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=327-337&author=B.+Escudierauthor=C.+Szczylikauthor=C.+Portaauthor=M.+Gore&title=Treatment+selection+in+metastatic+renal+cell+carcinoma%3A+expert+consensus&doi=10.1038%2Fnrclinonc.2012.59"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment selection in metastatic renal cell carcinoma: expert consensus</span></div><div class="casAuthors">Escudier, Bernard; Szczylik, Cezary; Porta, Camillo; Gore, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">327-337</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The advances in the field of targeted therapy for the treatment of renal cell carcinoma have improved outcomes for patients dramatically.  In this Review article, an expert consensus opinion for the optimal use of these targeted anticancer therapies for the treatment of patients with renal cell carcinoma is outlined.  In metastatic renal cell carcinoma (mRCC), many factors influence clin. decisions, including histol., tumor burden, prognostic factors, comorbidities, and the ability of the patient to tolerate treatment.  Progression-free survival (PFS) durations reported from randomized trials of targeted therapies vary considerably, in part because of differences in patient characteristics.  For first-line therapy, an est. of PFS with sunitinib, bevacizumab plus interferon, or sorafenib in a 'general' population is 8-9 mo, but each regimen is suitable for different patient categories.  For example, sunitinib is suitable for all-prognosis groups, particularly younger, fitter patients; pazopanib for patients with a good or intermediate prognosis; bevacizumab plus interferon for good-prognosis patients or those with indolent disease; and sorafenib for patients at all prognostic risk levels, particularly the elderly and those with comorbidities.  Sequential therapy with targeted agents provides significant benefit, and should be considered in all patients who can tolerate such treatment.  Level 1 evidence supports sequential use of tyrosine kinase inhibitors, as well as these agents followed by everolimus.  We consider how patient characteristics have influenced the results of studies of first-line therapy, and we provide expert opinion on the most appropriate treatment choices for particular patient groups receiving first-line and second-line therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxiub93j40UrVg90H21EOLACvtfcHk0lhMEGYRU8kXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvVOrtLo%253D&md5=b7b16fa16ed824583278ba376cb216e9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2012.59&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2012.59%26sid%3Dliteratum%253Aachs%26aulast%3DEscudier%26aufirst%3DB.%26aulast%3DSzczylik%26aufirst%3DC.%26aulast%3DPorta%26aufirst%3DC.%26aulast%3DGore%26aufirst%3DM.%26atitle%3DTreatment%2520selection%2520in%2520metastatic%2520renal%2520cell%2520carcinoma%253A%2520expert%2520consensus%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2012%26volume%3D9%26spage%3D327%26epage%3D337%26doi%3D10.1038%2Fnrclinonc.2012.59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhMEGYRU8kXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Barouch-Bentov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, K.</span><span> </span><span class="NLM_article-title">Mechanisms of drug resistance in kinases</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span><span class="refDoi"> DOI: 10.1517/13543784.2011.546344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1517%2F13543784.2011.546344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=21235428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=153-208&author=R.+Barouch-Bentovauthor=K.+Sauer&title=Mechanisms+of+drug+resistance+in+kinases&doi=10.1517%2F13543784.2011.546344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of drug resistance in kinases</span></div><div class="casAuthors">Barouch-Bentov, Rina; Sauer, Karsten</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">153-208</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Because of their important roles in disease and excellent "druggability", kinases have become the second largest drug target family.  The great success of the BCR-ABL inhibitor imatinib in treating chronic myelogenous leukemia illustrates the high potential of kinase inhibitor (KI) therapeutics, but also unveils a major limitation: the development of drug resistance.  This is a significant concern as KIs reach large patient populations for an expanding array of indications.  We provide an up-to-date understanding of the mechanisms through which KIs function and through which cells can become KI-resistant.  We review current and future approaches to overcome KI resistance, focusing on currently approved KIs and KIs in clin. trials.  We then discuss approaches to improve KI efficacy and overcome drug resistance and novel approaches to develop less drug resistance-prone KI therapeutics.  Although drug resistance is a concern for current KI therapeutics, recent progress in our understanding of the underlying mechanisms and promising technol. advances may overcome this limitation and provide powerful new therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprJUpdpkWU6bVg90H21EOLACvtfcHk0liDHWMtuVmMsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D&md5=cccffaca08cbcfb1e0d2c1d580c9e797</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1517%2F13543784.2011.546344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2011.546344%26sid%3Dliteratum%253Aachs%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DSauer%26aufirst%3DK.%26atitle%3DMechanisms%2520of%2520drug%2520resistance%2520in%2520kinases%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2011%26volume%3D20%26spage%3D153%26epage%3D208%26doi%3D10.1517%2F13543784.2011.546344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">BREED: generating novel inhibitors through hybridization of known ligands. application to CDK2, p38, and HIV protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2768</span><span class="NLM_x">–</span> <span class="NLM_lpage">2775</span><span class="refDoi"> DOI: 10.1021/jm030543u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030543u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2768-2775&author=A.+C.+Pierceauthor=G.+Raoauthor=G.+W.+Bemis&title=BREED%3A+generating+novel+inhibitors+through+hybridization+of+known+ligands.+application+to+CDK2%2C+p38%2C+and+HIV+protease&doi=10.1021%2Fjm030543u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BREED: Generating Novel Inhibitors through Hybridization of Known Ligands. Application to CDK2, P38, and HIV Protease</span></div><div class="casAuthors">Pierce, Albert C.; Rao, Govinda; Bemis, Guy W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2768-2775</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this work we describe BREED, a method for the generation of novel inhibitors from structures of known ligands bound to a common target.  The method is essentially an automation of the common medicinal chem. practice of joining fragments of two known ligands to generate a new inhibitor.  The ligand-bound target structures are overlaid, all overlapping bonds in all pairs of ligands are found, and the fragments on each side of each matching bond are swapped to generate the new mols.  Since the method is automated, it can be applied recursively to generate all possible combinations of known ligands.  In an application of this method to HIV protease inhibitors and protein kinase inhibitors, hundreds of new mol. structures were generated.  These included known inhibitor scaffolds not included in the initial set, entirely novel scaffolds, and novel substituents on known scaffolds.  The method is fast, and since all of the ligand functional groups are known to bind the target in the precise position and orientation present in the novel ligand, the success rate of this method should be superior to more traditional de novo design techniques.  In an era of increasingly high-throughput structural biol., such methods for high-throughput utilization of structural information will become increasingly valuable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqskNA5h73CbbVg90H21EOLACvtfcHk0liDHWMtuVmMsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D&md5=e0bec9082fab05b694742498b3f41549</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm030543u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030543u%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DRao%26aufirst%3DG.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DBREED%253A%2520generating%2520novel%2520inhibitors%2520through%2520hybridization%2520of%2520known%2520ligands.%2520application%2520to%2520CDK2%252C%2520p38%252C%2520and%2520HIV%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2768%26epage%3D2775%26doi%3D10.1021%2Fjm030543u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a knowledge-based structural database to navigate kinase–ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+knowledge-based+structural+database+to+navigate+kinase%E2%80%93ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0liDHWMtuVmMsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase%25E2%2580%2593ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span> </span><span class="NLM_article-title">De novo design of N-(pyridin-4-ylmethyl) aniline derivatives as KDR inhibitors: 3D-QSAR, molecular fragment replacement, protein-ligand interaction fingerprint, and ADMET prediction</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">787</span><span class="NLM_x">–</span> <span class="NLM_lpage">802</span><span class="refDoi"> DOI: 10.1007/s11030-012-9405-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2Fs11030-012-9405-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=787-802&author=Y.+Zhangauthor=H.+Liuauthor=Y.+Jiaoauthor=H.+Yuanauthor=F.+Wangauthor=S.+Luauthor=S.+Yaoauthor=Z.+Keauthor=W.+Taiauthor=Y.+Jiang&title=De+novo+design+of+N-%28pyridin-4-ylmethyl%29+aniline+derivatives+as+KDR+inhibitors%3A+3D-QSAR%2C+molecular+fragment+replacement%2C+protein-ligand+interaction+fingerprint%2C+and+ADMET+prediction&doi=10.1007%2Fs11030-012-9405-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs11030-012-9405-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-012-9405-y%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DKe%26aufirst%3DZ.%26aulast%3DTai%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DDe%2520novo%2520design%2520of%2520N-%2528pyridin-4-ylmethyl%2529%2520aniline%2520derivatives%2520as%2520KDR%2520inhibitors%253A%25203D-QSAR%252C%2520molecular%2520fragment%2520replacement%252C%2520protein-ligand%2520interaction%2520fingerprint%252C%2520and%2520ADMET%2520prediction%26jtitle%3DMol.%2520Diversity%26date%3D2012%26volume%3D16%26spage%3D787%26epage%3D802%26doi%3D10.1007%2Fs11030-012-9405-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, L.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability prediction of protein targets for structure-based drug design</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2326</span><span class="NLM_x">–</span> <span class="NLM_lpage">2333</span><span class="refDoi"> DOI: 10.2174/1381612811319120019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.2174%2F1381612811319120019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=23082974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2326-2333&author=Y.+Yuanauthor=J.+Peiauthor=L.+Lai&title=Binding+site+detection+and+druggability+prediction+of+protein+targets+for+structure-based+drug+design&doi=10.2174%2F1381612811319120019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Binding site detection and druggability prediction of protein targets for structure-based drug design</span></div><div class="casAuthors">Yuan, Yaxia; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2326-2333</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Assessing whether a protein structure is a good target or not before actually doing structure-based drug design on it is an important step to speed up the ligand discovery process.  This is known as the "druggability" or "ligandability" assessment problem that has attracted increasing interest in recent years.  The assessment typically includes the detection of ligand-binding sites on the protein surface and the prediction of their abilities to bind drug-like small mols.  A brief summary of the established methods of binding sites detection and druggability(ligandability) prediction, as well as a detailed description of the CAVITY approach developed in the authors' group was given.  CAVITY showed good performance on ligand-binding site detection, and was successfully used to predict both the ligandabilities and druggabilities of the detected binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9UZbLJWt2l7Vg90H21EOLACvtfcHk0lizr9s8Zq0Oyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtbc%253D&md5=9d6105927b7c36665477b446c54e7cca</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.2174%2F1381612811319120019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319120019%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520prediction%2520of%2520protein%2520targets%2520for%2520structure-based%2520drug%2520design%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D2326%26epage%3D2333%26doi%3D10.2174%2F1381612811319120019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Ngan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">FTMAP: extended protein mapping with user-selected probe molecules</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1093/nar/gks441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1093%2Fnar%2Fgks441" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=271-275&author=C.+H.+Nganauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=D.+Beglovauthor=E.+A.+Villarauthor=D.+R.+Hallauthor=D.+Kozakovauthor=S.+Vajda&title=FTMAP%3A+extended+protein+mapping+with+user-selected+probe+molecules&doi=10.1093%2Fnar%2Fgks441"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks441%26sid%3Dliteratum%253Aachs%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFTMAP%253A%2520extended%2520protein%2520mapping%2520with%2520user-selected%2520probe%2520molecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D271%26epage%3D275%26doi%3D10.1093%2Fnar%2Fgks441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, X.</span><span> </span><span class="NLM_article-title">Identification of covalent binding sites targeting cysteines based on computational approaches</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3106</span><span class="NLM_x">–</span> <span class="NLM_lpage">3118</span><span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.6b00302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.6b00302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KmtrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=3106-3118&author=Y.+Zhangauthor=D.+Zhangauthor=H.+Tianauthor=Y.+Jiaoauthor=Z.+Shiauthor=T.+Ranauthor=H.+Liuauthor=S.+Luauthor=A.+Xuauthor=X.+Qiao&title=Identification+of+covalent+binding+sites+targeting+cysteines+based+on+computational+approaches&doi=10.1021%2Facs.molpharmaceut.6b00302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Covalent Binding Sites Targeting Cysteines Based on Computational Approaches</span></div><div class="casAuthors">Zhang, Yanmin; Zhang, Danfeng; Tian, Haozhong; Jiao, Yu; Shi, Zhihao; Ran, Ting; Liu, Haichun; Lu, Shuai; Xu, Anyang; Qiao, Xin; Pan, Jing; Yin, Lingfeng; Zhou, Weineng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3106-3118</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalent drugs have attracted increasing attention in recent years due to good inhibitory activity and selectivity.  Targeting noncatalytic cysteines with irreversible inhibitors is a powerful approach for enhancing pharmacol. potency and selectivity because cysteines can form covalent bonds with inhibitors through their nucleophilic thiol groups.  However, most human kinases have multiple noncatalytic cysteines within the active site; to accurately predict which cysteine is most likely to form covalent bonds is of great importance but remains a challenge when designing irreversible inhibitors.  FTMap was first applied to check its ability in predicting covalent binding site defined as the region where covalent bonds are formed between cysteines and irreversible inhibitors.  Results show that it has excellent performance in detecting the hot spots within the binding pocket, and its hydrogen bond interaction frequency anal. could give the authors some interesting instructions for identification of covalent binding cysteines.  Furthermore, the authors proposed a simple but useful covalent fragment probing approach and showed that it successfully predicted the covalent binding site of seven targets.  By adopting a distance-based method, the authors obsd. that the closer the nucleophiles of covalent warheads are to the thiol group of a cysteine, the higher the possibility that a cysteine is prone to form a covalent bond.  The authors believe that the combination of FTMap and the authors' distance-based covalent fragment probing method can become a useful tool in detecting the covalent binding site of these targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJNLjHdOWupLVg90H21EOLACvtfcHk0lizr9s8Zq0Oyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KmtrbM&md5=ce487167befaeaa8c88ef1ee8dabc863</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.6b00302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.6b00302%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DA.%26aulast%3DQiao%26aufirst%3DX.%26atitle%3DIdentification%2520of%2520covalent%2520binding%2520sites%2520targeting%2520cysteines%2520based%2520on%2520computational%2520approaches%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2016%26volume%3D13%26spage%3D3106%26epage%3D3118%26doi%3D10.1021%2Facs.molpharmaceut.6b00302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, X.</span><span> </span><span class="NLM_article-title">Fragment virtual screening based on bayesian categorization for discovering novel VEGFR-2 scaffolds</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">895</span><span class="NLM_x">–</span> <span class="NLM_lpage">913</span><span class="refDoi"> DOI: 10.1007/s11030-015-9592-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2Fs11030-015-9592-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=26022686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1aqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=895-913&author=Y.+Zhangauthor=Y.+Jiaoauthor=X.+Xiongauthor=H.+Liuauthor=T.+Ranauthor=J.+Xuauthor=S.+Luauthor=A.+Xuauthor=J.+Panauthor=X.+Qiao&title=Fragment+virtual+screening+based+on+bayesian+categorization+for+discovering+novel+VEGFR-2+scaffolds&doi=10.1007%2Fs11030-015-9592-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment virtual screening based on Bayesian categorization for discovering novel VEGFR-2 scaffolds</span></div><div class="casAuthors">Zhang, Yanmin; Jiao, Yu; Xiong, Xiao; Liu, Haichun; Ran, Ting; Xu, Jinxing; Lu, Shuai; Xu, Anyang; Pan, Jing; Qiao, Xin; Shi, Zhihao; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">895-913</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The discovery of novel scaffolds against a specific target has long been one of the most significant but challengeable goals in discovering lead compds.  A scaffold that binds in important regions of the active pocket is more favorable as a starting point because scaffolds generally possess greater optimization possibilities.  However, due to the lack of sufficient chem. space diversity of the databases and the ineffectiveness of the screening methods, it still remains a great challenge to discover novel active scaffolds.  Since the strengths and weaknesses of both fragment-based drug design and traditional virtual screening (VS), we proposed a fragment VS concept based on Bayesian categorization for the discovery of novel scaffolds.  This work investigated the proposal through an application on VEGFR-2 target.  Firstly, scaffold and structural diversity of chem. space for 10 compd. databases were explicitly evaluated.  Simultaneously, a robust Bayesian classification model was constructed for screening not only compd. databases but also their corresponding fragment databases.  Although anal. of the scaffold diversity demonstrated a very unevenly distribution of scaffolds over mols., results showed that our Bayesian model behaved better in screening fragments than mols.  Through a literature retrospective research, several generated fragments with relatively high Bayesian scores indeed exhibit VEGFR-2 biol. activity, which strongly proved the effectiveness of fragment VS based on Bayesian categorization models.  This investigation of Bayesian-based fragment VS can further emphasize the necessity for enrichment of compd. databases employed in lead discovery by amplifying the diversity of databases with novel structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm1F-dextwrrVg90H21EOLACvtfcHk0lizr9s8Zq0Oyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1aqtr0%253D&md5=16fca90b4e5746913ba187b0043768de</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs11030-015-9592-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-015-9592-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DA.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DQiao%26aufirst%3DX.%26atitle%3DFragment%2520virtual%2520screening%2520based%2520on%2520bayesian%2520categorization%2520for%2520discovering%2520novel%2520VEGFR-2%2520scaffolds%26jtitle%3DMol.%2520Diversity%26date%3D2015%26volume%3D19%26spage%3D895%26epage%3D913%26doi%3D10.1007%2Fs11030-015-9592-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Kirchmair, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Distinto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuster, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzer, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liedl, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolber, G.</span><span> </span><span class="NLM_article-title">The protein data bank (PDB), its related services and software tools as key components for in silico guided drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7021</span><span class="NLM_x">–</span> <span class="NLM_lpage">7040</span><span class="refDoi"> DOI: 10.1021/jm8005977</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8005977" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7021-7040&author=J.+Kirchmairauthor=P.+Marktauthor=S.+Distintoauthor=D.+Schusterauthor=G.+M.+Spitzerauthor=K.+R.+Liedlauthor=T.+Langerauthor=G.+Wolber&title=The+protein+data+bank+%28PDB%29%2C+its+related+services+and+software+tools+as+key+components+for+in+silico+guided+drug+discovery&doi=10.1021%2Fjm8005977"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm8005977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8005977%26sid%3Dliteratum%253Aachs%26aulast%3DKirchmair%26aufirst%3DJ.%26aulast%3DMarkt%26aufirst%3DP.%26aulast%3DDistinto%26aufirst%3DS.%26aulast%3DSchuster%26aufirst%3DD.%26aulast%3DSpitzer%26aufirst%3DG.%2BM.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DWolber%26aufirst%3DG.%26atitle%3DThe%2520protein%2520data%2520bank%2520%2528PDB%2529%252C%2520its%2520related%2520services%2520and%2520software%2520tools%2520as%2520key%2520components%2520for%2520in%2520silico%2520guided%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7021%26epage%3D7040%26doi%3D10.1021%2Fjm8005977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljspK1zRG8YXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Zhou, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.-L.</span><span> </span><span class="NLM_article-title">The changing landscape of clinical trial and approval processes in China</span> <span class="citation_source-journal">Nat. Rev. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1038/nrclinonc.2017.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnrclinonc.2017.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=28195238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADC%252BC1c3msFShuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=577-583&author=Q.+Zhouauthor=X.-Y.+Chenauthor=Z.-M.+Yangauthor=Y.-L.+Wu&title=The+changing+landscape+of+clinical+trial+and+approval+processes+in+China&doi=10.1038%2Fnrclinonc.2017.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The changing landscape of clinical trial and approval processes in China</span></div><div class="casAuthors">Zhou Qing; Wu Yi-Long; Chen Xiao-Yuan; Yang Zhi-Min</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Clinical oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">577-583</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In the past decade, the standards of clinical trials in China have moved closer to international standards, thus encouraging the development of innovative drugs.  However, a large backlog of pending applications for both drug approval and clinical trial registration has arisen owing to the complexity of the approval process, the volume of applications and a lack of staff available to process these applications, among other reasons.  To improve the drug approval process, a 'four-colour-light' strategy was introduced.  Different drugs are classified into redefined categories of innovative and generic drugs, with priority being given to approval decisions concerning innovative drugs.  Other improvement strategies are now also being implemented, including the development of a new clinical trial approval system and several measures designed to encourage greater participation of Chinese researchers and research centres in international clinical trials.  In this Perspective, the changing landscape of clinical approval in China is described, including the difficulties that drug approval authorities face in this rapidly developing nation and the novel strategies that are being used to find solutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfGZ3GFxQ08lw2ofyr07jXfW6udTcc2eZGZZFZ0v2HqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3msFShuw%253D%253D&md5=d5043cdbd883365ed4ac473ed8878504</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2017.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2017.10%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DX.-Y.%26aulast%3DYang%26aufirst%3DZ.-M.%26aulast%3DWu%26aufirst%3DY.-L.%26atitle%3DThe%2520changing%2520landscape%2520of%2520clinical%2520trial%2520and%2520approval%2520processes%2520in%2520China%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D14%26spage%3D577%26epage%3D583%26doi%3D10.1038%2Fnrclinonc.2017.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Park, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgescu, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galper, J. B.</span><span> </span><span class="NLM_article-title">Human umbilical vein endothelial cells and human dermal microvascular endothelial cells offer new insights into the relationship between lipid metabolism and angiogenesis</span> <span class="citation_source-journal">Stem Cell Rev. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">101</span><span class="refDoi"> DOI: 10.1007/s12015-006-0015-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2Fs12015-006-0015-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17237547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Siuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=93-101&author=H.-J.+Parkauthor=Y.+Zhangauthor=S.+P.+Georgescuauthor=K.+L.+Johnsonauthor=D.+Kongauthor=J.+B.+Galper&title=Human+umbilical+vein+endothelial+cells+and+human+dermal+microvascular+endothelial+cells+offer+new+insights+into+the+relationship+between+lipid+metabolism+and+angiogenesis&doi=10.1007%2Fs12015-006-0015-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Human umbilical vein endothelial cells and human dermal microvascular endothelial cells offer new insights into the relationship between lipid metabolism and angiogenesis</span></div><div class="casAuthors">Park, Ho-Jin; Zhang, Yali; Georgescu, Serban P.; Johnson, Kristin L.; Kong, Dequon; Galper, Jonas B.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">93-101</span>CODEN:
                <span class="NLM_cas:coden">SCRTCI</span>;
        ISSN:<span class="NLM_cas:issn">1550-8943</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review.  Human umbilical vein endothelial cells (HUVECs) have played a major role as a model system for the study of the regulation of endothelial cell function and the role of the endothelium in the response of the blood vessel wall to stretch, shear forces, and the development of atherosclerotic plaques and angiogenesis.  Here, we use HUVECs and human microvascular endothelial cells to study the role of the HMG-CoA reductase inhibitor, simvastatin, and the small GTP-binding protein Rho in the regulation of angiogenesis.  Simvastatin inhibited angiogenesis in response to FGF-2 in the corneal pocket assay of the mouse and in vascular endothelial growth factor (VEGF)-stimulated angiogenesis in the chick chorioallontoic membrane.  Furthermore, simvastatin inhibited VEGF-stimulated tube formation by human dermal microvascular endothelial cells and the formation of honeycomb-like structures by HUVECs.  The effect was dose-dependent and was not secondary to apoptosis.  Geranylgeranyl-pyrophosphate (GGPP), a product of the cholesterol metabolic pathway that serves as a substrate for the posttranslational lipidation of RhoA, was required for membrane localization, but not farnesylpyrophosphate(FPP), the substrate for the lipidation of Ras.  Furthermore, GGTI, a specific inhibitor of GGPP, mimicked the effect of simvastatin of tube formation and the formation of honeycombs whereas FTI, a specific inhibitor of the farnesylation of Ras, had no effect.  Adenoviral expression of a DN-RhoA mutant mimicked the effect of simvastatin on tube formation and the formation of honeycombs, whereas a dominant activating mutant of RhoA reversed the effect of simvastatin on tube formation.  Finally, simvastatin interfered with the membrane localization of RhoA with a dose-dependence similar to that for the inhibition of tube formation.  Simvastatin also inhibited the VEGF-stimulated phosphorylation of the VEGF receptor KDR, and the tyrosine kinase FAK, which plays a role in cell migration.  These data demonstrate that simvastatin interfered with angiogenesis via the inhibition of RhoA.  Data supporting a role for angiogenesis in the development and growth of atherosclerotic plaques suggest that this antiangiogenic effect of Statins might prevent the progression of atherosclerosis via the inhibition of plaque angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPCLaFKAcrhrVg90H21EOLACvtfcHk0lgSBRJZr3AE9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Siuro%253D&md5=1074e701324394d9d3f2b78ca7c1ad85</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs12015-006-0015-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12015-006-0015-x%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.-J.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGeorgescu%26aufirst%3DS.%2BP.%26aulast%3DJohnson%26aufirst%3DK.%2BL.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DGalper%26aufirst%3DJ.%2BB.%26atitle%3DHuman%2520umbilical%2520vein%2520endothelial%2520cells%2520and%2520human%2520dermal%2520microvascular%2520endothelial%2520cells%2520offer%2520new%2520insights%2520into%2520the%2520relationship%2520between%2520lipid%2520metabolism%2520and%2520angiogenesis%26jtitle%3DStem%2520Cell%2520Rev.%2520Rep.%26date%3D2006%26volume%3D2%26spage%3D93%26epage%3D101%26doi%3D10.1007%2Fs12015-006-0015-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Ross, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherf, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisen, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perou, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spellman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van de Rijn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waltham, M.</span><span> </span><span class="NLM_article-title">Systematic variation in gene expression patterns in human cancer cell lines</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span><span class="refDoi"> DOI: 10.1038/73432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2F73432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10700174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvFaqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2000&pages=227-235&author=D.+T.+Rossauthor=U.+Scherfauthor=M.+B.+Eisenauthor=C.+M.+Perouauthor=C.+Reesauthor=P.+Spellmanauthor=V.+Iyerauthor=S.+S.+Jeffreyauthor=M.+Van+de+Rijnauthor=M.+Waltham&title=Systematic+variation+in+gene+expression+patterns+in+human+cancer+cell+lines&doi=10.1038%2F73432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic variation in gene expression patterns in human cancer cell lines</span></div><div class="casAuthors">Ross, Douglas T.; Scherf, Uwe; Eisen, Michael B.; Perou, Charles M.; Rees, Christian; Spellman, Paul; Iyer, Vishwanath; Jeffrey, Stefanie S.; Van de Rijn, Matt; Waltham, Mark; Pergamenschikov, Alexander; Lee, Jeffrey C. F.; Lashkari, Deval; Shalon, Dari; Myers, Timothy G.; Weinstein, John N.; Botstein, David; Brown, Patrick O.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">227-235</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">We used cDNA microarrays to explore the variation in expression of approx. 8,000 unique genes among the 60 cell lines used in the National Cancer Institute's screen for anti-cancer drugs.  Classification of the cell lines based solely on the obsd. patterns of gene expression revealed a correspondence to the ostensible origins of the tumors from which the cell lines were derived.  The consistent relationship between the gene expression patterns and the tissue of origin allowed us to recognize outliers whose previous classification appeared incorrect.  Specific features of the gene expression patterns appeared to be related to physiol. properties of the cell lines, such as their doubling time in culture, drug metab. or the interferon response.  Comparison of gene expression patterns in the cell lines to those obsd. in normal breast tissue or in breast tumor specimens revealed features of the expression patterns in the tumors that had recognizable counterparts in specific cell lines, reflecting the tumor, stromal and inflammatory components of the tumor tissue.  These results provided a novel mol. characterization of this important group of human cell lines and their relationships to tumors in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZhocezjCTNbVg90H21EOLACvtfcHk0lgSBRJZr3AE9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvFaqtrg%253D&md5=248ffa9428fa078fff0bdf45fddaecf4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F73432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F73432%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DD.%2BT.%26aulast%3DScherf%26aufirst%3DU.%26aulast%3DEisen%26aufirst%3DM.%2BB.%26aulast%3DPerou%26aufirst%3DC.%2BM.%26aulast%3DRees%26aufirst%3DC.%26aulast%3DSpellman%26aufirst%3DP.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DJeffrey%26aufirst%3DS.%2BS.%26aulast%3DVan%2Bde%2BRijn%26aufirst%3DM.%26aulast%3DWaltham%26aufirst%3DM.%26atitle%3DSystematic%2520variation%2520in%2520gene%2520expression%2520patterns%2520in%2520human%2520cancer%2520cell%2520lines%26jtitle%3DNat.%2520Genet.%26date%3D2000%26volume%3D24%26spage%3D227%26epage%3D235%26doi%3D10.1038%2F73432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Stanton, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edington, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendler, F.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor expression by human squamous cell carcinomas of the head and neck, cell lines and xenografts</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">427</span><span class="NLM_x">–</span> <span class="NLM_lpage">433</span><span class="refDoi"> DOI: 10.1038/bjc.1994.322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fbjc.1994.322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=8080726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK2cXmsFaksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1994&pages=427-433&author=P.+Stantonauthor=S.+Richardsauthor=J.+Reevesauthor=M.+Nikolicauthor=K.+Edingtonauthor=L.+Clarkauthor=G.+Robertsonauthor=D.+Souterauthor=R.+Mitchellauthor=F.+Hendler&title=Epidermal+growth+factor+receptor+expression+by+human+squamous+cell+carcinomas+of+the+head+and+neck%2C+cell+lines+and+xenografts&doi=10.1038%2Fbjc.1994.322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor expression by human squamous cell carcinomas of the head and neck, cell lines and xenografts</span></div><div class="casAuthors">Stanton, P.; Richards, S.; Reeves, J.; Nikolic, M.; Edington, K.; Clark, L.; Robertson, G.; Souter, D.; Mitchell, R.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">427-33</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) overexpression has been assocd. frequently with squamous cell carcinomas (SCC) and SCC cell lines.  In most cases the level of EGFR on the tumors from which the cell lines were derived has not been detd., nor have EGFR levels been detd. for xenograft tumors from the cell lines.  In this study the authors detd. EGFR expression on a new series of head and neck SCC (SCCHN)-derived cell lines, which were obtained from tumors representing a spectrum of malignant progression, and two cell strains derived from erythroplakia premalignant lesions.  The level of EGFR on cell lines was detd. by [125I]EGF competitive binding assays.  EGFR levels on some of the original tumors and xenografts of the cell lines were detd. on cryosections by a competitive binding assay based on [125I]EGFR1, an EGFR-specific monoclonal antibody.  EGFR expression on the tumor cryosections was compared with expression on cryosections of skin and buccal mucosa.  Eight of the ten tumor cell lines had elevated EGFR.  Two of the tumor-derived cell lines and the two erythroplakia-derived cell strains expressed EGFR at levels similar to that detected on normal keratinocytes in tissue culture.  Only two of the tumors overexpressed EGFR when compared with normal tissue.  The other tumors had levels similar to that detected on the basal layers of skin or buccal mucosa.  The xenografts expressed EGFR, as did the original tumors, even though they were derived from cell lines that displayed significant overexpression of EGFR.  This study suggests that most tumors have a latent potential to overexpress EGFR which is realized in tissue culture.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1dvIlD-kurVg90H21EOLACvtfcHk0lgSBRJZr3AE9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsFaksr0%253D&md5=65349d81c3a8650debfe604ddadbb2f3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fbjc.1994.322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.1994.322%26sid%3Dliteratum%253Aachs%26aulast%3DStanton%26aufirst%3DP.%26aulast%3DRichards%26aufirst%3DS.%26aulast%3DReeves%26aufirst%3DJ.%26aulast%3DNikolic%26aufirst%3DM.%26aulast%3DEdington%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DRobertson%26aufirst%3DG.%26aulast%3DSouter%26aufirst%3DD.%26aulast%3DMitchell%26aufirst%3DR.%26aulast%3DHendler%26aufirst%3DF.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520expression%2520by%2520human%2520squamous%2520cell%2520carcinomas%2520of%2520the%2520head%2520and%2520neck%252C%2520cell%2520lines%2520and%2520xenografts%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1994%26volume%3D70%26spage%3D427%26epage%3D433%26doi%3D10.1038%2Fbjc.1994.322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiesewetter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span> </span><span class="NLM_article-title">Site-specific labeling of scVEGF with fluorine-18 for positron emission tomography imaging</span> <span class="citation_source-journal">Theranostics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.7150/thno.4611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.7150%2Fthno.4611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=22768028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCqsbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=607-617&author=H.+Wangauthor=H.+Gaoauthor=N.+Guoauthor=G.+Niuauthor=Y.+Maauthor=D.+Kiesewetterauthor=X.+Chen&title=Site-specific+labeling+of+scVEGF+with+fluorine-18+for+positron+emission+tomography+imaging&doi=10.7150%2Fthno.4611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Site-specific labeling of scVEGF with fluorine-18 for positron emission tomography imaging</span></div><div class="casAuthors">Wang, Hui; Gao, Haokao; Guo, Ning; Niu, Gang; Ma, Ying; Kiesewetter, Dale O.; Chen, Xiaoyuan</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">607-617</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) is one of the most important mediators of angiogenesis.  Single-chain (s.c.)-VEGF protein contg. an N-terminal Cys-tag has been designed for site-specific modification with a variety of imaging and therapeutic moieties.  Site-specific labeling of scVEGF with thiol-reactive prosthetic group, N-[2-(4-18F-fluorobenzamido) ethyl] maleimide ([18F]FBEM) for positron emission tomog. (PET) imaging of VEFGR may provide a new tracer which has great potential for clin. translation.  Methods: [18F]FBEM-scVEGF was synthesized by site-specific conjugation of 18F-FBEM to a thiol group in Cys-tag of scVEGF at room temp.  The functional activity after labeling was tested by immunofluorescence staining, cellular uptake and efflux.  The tumor targeting and in vivo properties were evaluated by biodistribution and microPET studies in tumor-bearing mice.  Results: The radiolabeling yield and specific activity of [18F]FBEM-scVEGF were 20.6 ± 15.1 % (based on starting [18F]FBEM, uncorrected, n = 5) and 58.8 ± 12.4 GBq/μmol, resp.  Noninvasive microPET and direct tissue sampling expts. demonstrated that [18F]FBEM-scVEGF had VEGFR specific tumor uptake in MDA-MB-435, U87MG and 4T1 xenograft models.  The optimal tumor uptake was achieved at 2 h p.i., which can be partially, but significantly blocked by co-injection of non-labeled scVEGF protein.  Overall, [18F]FBEM-scVEGF showed VEGFR specific tumor uptake.  Conclusion: The scVEGF was site-specifically labeled with 18F via [18F]FBEM prosthetic group and the tracer [18F]FBEM-scVEGF exhibited high receptor binding affinity and tumor targeting efficacy.  Further study of [18F] FBEM-scVEGF to evaluate angiogenesis in cancer and other disease types is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3Qpct2fPYNrVg90H21EOLACvtfcHk0ljqH7IIg-lbwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCqsbjP&md5=4b35e0e179ccead86069f2b706d515db</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7150%2Fthno.4611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.4611%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DN.%26aulast%3DNiu%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DKiesewetter%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DSite-specific%2520labeling%2520of%2520scVEGF%2520with%2520fluorine-18%2520for%2520positron%2520emission%2520tomography%2520imaging%26jtitle%3DTheranostics%26date%3D2012%26volume%3D2%26spage%3D607%26epage%3D617%26doi%3D10.7150%2Fthno.4611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Cerezo, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornedo-Ortega, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Álvarez-Fernández, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troncoso, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Parrilla, M. C.</span><span> </span><span class="NLM_article-title">Inhibition of VEGF-Induced VEGFR-2 activation and HUVEC migration by melatonin and other bioactive indolic compounds</span> <span class="citation_source-journal">Nutrients</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="refDoi"> DOI: 10.3390/nu9030249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.3390%2Fnu9030249" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=249&author=A.+B.+Cerezoauthor=R.+Hornedo-Ortegaauthor=M.+A.+%C3%81lvarez-Fern%C3%A1ndezauthor=A.+M.+Troncosoauthor=M.+C.+Garc%C3%ADa-Parrilla&title=Inhibition+of+VEGF-Induced+VEGFR-2+activation+and+HUVEC+migration+by+melatonin+and+other+bioactive+indolic+compounds&doi=10.3390%2Fnu9030249"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.3390%2Fnu9030249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fnu9030249%26sid%3Dliteratum%253Aachs%26aulast%3DCerezo%26aufirst%3DA.%2BB.%26aulast%3DHornedo-Ortega%26aufirst%3DR.%26aulast%3D%25C3%2581lvarez-Fern%25C3%25A1ndez%26aufirst%3DM.%2BA.%26aulast%3DTroncoso%26aufirst%3DA.%2BM.%26aulast%3DGarc%25C3%25ADa-Parrilla%26aufirst%3DM.%2BC.%26atitle%3DInhibition%2520of%2520VEGF-Induced%2520VEGFR-2%2520activation%2520and%2520HUVEC%2520migration%2520by%2520melatonin%2520and%2520other%2520bioactive%2520indolic%2520compounds%26jtitle%3DNutrients%26date%3D2017%26volume%3D9%26spage%3D249%26doi%3D10.3390%2Fnu9030249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acheampong, D. O.</span><span> </span><span class="NLM_article-title">A novel bispecific diabody targeting both vascular endothelial growth factor receptor 2 and epidermal growth factor receptor for enhanced antitumor activity</span> <span class="citation_source-journal">Biotechnol. Prog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">294</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1002/btpr.2231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fbtpr.2231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=26785424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFSqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=294-302&author=M.+Xuauthor=H.+Jinauthor=Z.+Chenauthor=W.+Xieauthor=Y.+Wangauthor=Y.+Wangauthor=M.+Wangauthor=J.+Zhangauthor=D.+O.+Acheampong&title=A+novel+bispecific+diabody+targeting+both+vascular+endothelial+growth+factor+receptor+2+and+epidermal+growth+factor+receptor+for+enhanced+antitumor+activity&doi=10.1002%2Fbtpr.2231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A novel bispecific diabody targeting both vascular endothelial growth factor receptor 2 and epidermal growth factor receptor for enhanced antitumor activity</span></div><div class="casAuthors">Xu, Menghuai; Jin, Haizhen; Chen, Zhiguo; Xie, Wei; Wang, Youfu; Wang, Yang; Wang, Min; Zhang, Juan; Acheampong, Desmond Omane</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Progress</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">294-302</span>CODEN:
                <span class="NLM_cas:coden">BIPRET</span>;
        ISSN:<span class="NLM_cas:issn">1520-6033</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) and vascular endothelial growth factor receptor 2 (VEGFR2) are receptor tyrosine kinases known to play crit. roles in the development and progression of tumors.  Based on the cross-talk between EGFR and VEGFR2 signal pathways, we designed and produced a bispecific diabody (bDAb) targeting both EGFR and VEGFR2 simultaneously.  The bispecific mol. (EK-02) demonstrated that it could bind to HUVEC (VEGFR2 high-expressing) and A431 (EGFR overexpressing) cells.  Addnl., similar to the parental antibodies, it was able to inhibit proliferation and migration, and induced apoptosis in these cells (HUVECs and A431), demonstrating that it had retained the functional properties of its parental antibodies.  Furthermore, the efficacy of EK-02 was evaluated using the human colon adenocarcinoma cell line HT29 (VEGFR2 and EGFR coexpressing).  In vitro assay showed that EK-02 could bind to HT29 cells, restrain cell growth and migration, and induce apoptosis with enhanced efficacy compared to both parental antibodies.  Further, it inhibited the neovascularization and tumor formation on an HT29 cell bearing chicken chorioallantoic membrane (CAM) tumor model in vivo.  In conclusion, these data suggest that the novel bDAb (EK-02) has antiangiogenesis and antitumor capacity both in vitro and in vivo, and can possibly be used as cotargeted therapy for the treatment of EGFR and VEGFR2 overexpressing tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogmVRKSshnfbVg90H21EOLACvtfcHk0ljqH7IIg-lbwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFSqsrw%253D&md5=0a90ec5010b003b73229a2a24e37dfdf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fbtpr.2231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbtpr.2231%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAcheampong%26aufirst%3DD.%2BO.%26atitle%3DA%2520novel%2520bispecific%2520diabody%2520targeting%2520both%2520vascular%2520endothelial%2520growth%2520factor%2520receptor%25202%2520and%2520epidermal%2520growth%2520factor%2520receptor%2520for%2520enhanced%2520antitumor%2520activity%26jtitle%3DBiotechnol.%2520Prog.%26date%3D2016%26volume%3D32%26spage%3D294%26epage%3D302%26doi%3D10.1002%2Fbtpr.2231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Chow, L. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhardt, S. G.</span><span> </span><span class="NLM_article-title">Sunitinib: from rational design to clinical efficacy</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">884</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1200/JCO.2006.06.3602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1200%2FJCO.2006.06.3602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17327610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvVKmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=884-896&author=L.+Q.+Chowauthor=S.+G.+Eckhardt&title=Sunitinib%3A+from+rational+design+to+clinical+efficacy&doi=10.1200%2FJCO.2006.06.3602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Sunitinib: from rational design to clinical efficacy</span></div><div class="casAuthors">Chow, Laura Q. M.; Eckhardt, S. Gail</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">884-896</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">A review.  Sunitinib (SU011248) is an oral small mol. tyrosine kinase inhibitor that exhibits potent antiangiogenic and antitumor activity.  Tyrosine kinase inhibitors such as SU6668 and SU5416 (semaxanib) demonstrated poor pharmacol. properties and limited efficacy; therefore, sunitinib was rationally designed and chosen for its high bioavailability and its nanomolar-range potency against the antiangiogenic receptor tyrosine kinases (RTKs)-vascular endothelial growth factor receptor (VEGFR) and platelet-derived growth factor receptor (PDGFR).  Sunitinib inhibits other tyrosine kinases including, KIT, FLT3, colony-stimulating factor 1 (CSF-1), and RET, which are involved in a no. of malignancies including small-cell lung cancer, GI stromal tumors (GISTs), breast cancer, acute myelogenous leukemia, multiple endocrine neoplasia types 2A and 2B, and familial medullary thyroid carcinoma.  Sunitinib demonstrated robust antitumor activity in preclin. studies resulting not only in tumor growth inhibition, but tumor regression in models of colon cancer, non-small-cell lung cancer, melanoma, renal carcinoma, and squamous cell carcinoma, which were assocd. with inhibition of VEGFR and PDGFR phosphorylation.  Clin. activity was demonstrated in neuroendocrine, colon, and breast cancers in phase II studies, whereas definitive efficacy has been demonstrated in advanced renal cell carcinoma and in imatinib-refractory GISTs, leading to US Food and Drug Administration approval of sunitinib for treatment of these two diseases.  Studies investigating sunitinib alone in various tumor types and in combination with chemotherapy are ongoing.  The clin. benchmarking of this small-mol. inhibitor of members of the split-kinase domain family of RTKs will lead to addnl. insights regarding the biol., potential biomarkers, and clin. utility of agents that target multiple signaling pathways in tumor, stromal, and endothelial compartments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPiO2cteaPj7Vg90H21EOLACvtfcHk0ljqH7IIg-lbwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvVKmtLY%253D&md5=c2b5469aff21b2f3062249c9f56c3814</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1200%2FJCO.2006.06.3602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2006.06.3602%26sid%3Dliteratum%253Aachs%26aulast%3DChow%26aufirst%3DL.%2BQ.%26aulast%3DEckhardt%26aufirst%3DS.%2BG.%26atitle%3DSunitinib%253A%2520from%2520rational%2520design%2520to%2520clinical%2520efficacy%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2007%26volume%3D25%26spage%3D884%26epage%3D896%26doi%3D10.1200%2FJCO.2006.06.3602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Norris-Cervetto, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platt, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dwek, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butters, T. D.</span><span> </span><span class="NLM_article-title">Inhibition of glucosylceramide synthase does not reverse drug resistance in cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">40412</span><span class="NLM_x">–</span> <span class="NLM_lpage">40418</span><span class="refDoi"> DOI: 10.1074/jbc.M404466200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1074%2Fjbc.M404466200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=40412-40418&author=E.+Norris-Cervettoauthor=R.+Callaghanauthor=F.+M.+Plattauthor=R.+A.+Dwekauthor=T.+D.+Butters&title=Inhibition+of+glucosylceramide+synthase+does+not+reverse+drug+resistance+in+cancer+cells&doi=10.1074%2Fjbc.M404466200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M404466200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M404466200%26sid%3Dliteratum%253Aachs%26aulast%3DNorris-Cervetto%26aufirst%3DE.%26aulast%3DCallaghan%26aufirst%3DR.%26aulast%3DPlatt%26aufirst%3DF.%2BM.%26aulast%3DDwek%26aufirst%3DR.%2BA.%26aulast%3DButters%26aufirst%3DT.%2BD.%26atitle%3DInhibition%2520of%2520glucosylceramide%2520synthase%2520does%2520not%2520reverse%2520drug%2520resistance%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D40412%26epage%3D40418%26doi%3D10.1074%2Fjbc.M404466200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Padro, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bieker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steins, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Retzlaff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bürger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Büchner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrera, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kienast, J.</span><span> </span><span class="NLM_article-title">Overexpression of vascular endothelial growth factor (VEGF) and its cellular receptor KDR (VEGFR-2) in the bone marrow of patients with acute myeloid leukemia</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1302</span><span class="NLM_x">–</span> <span class="NLM_lpage">1310</span><span class="refDoi"> DOI: 10.1038/sj.leu.2402534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fsj.leu.2402534" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1302-1310&author=T.+Padroauthor=R.+Biekerauthor=S.+Ruizauthor=M.+Steinsauthor=S.+Retzlaffauthor=H.+B%C3%BCrgerauthor=T.+B%C3%BCchnerauthor=T.+Kesslerauthor=F.+Herreraauthor=J.+Kienast&title=Overexpression+of+vascular+endothelial+growth+factor+%28VEGF%29+and+its+cellular+receptor+KDR+%28VEGFR-2%29+in+the+bone+marrow+of+patients+with+acute+myeloid+leukemia&doi=10.1038%2Fsj.leu.2402534"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2402534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2402534%26sid%3Dliteratum%253Aachs%26aulast%3DPadro%26aufirst%3DT.%26aulast%3DBieker%26aufirst%3DR.%26aulast%3DRuiz%26aufirst%3DS.%26aulast%3DSteins%26aufirst%3DM.%26aulast%3DRetzlaff%26aufirst%3DS.%26aulast%3DB%25C3%25BCrger%26aufirst%3DH.%26aulast%3DB%25C3%25BCchner%26aufirst%3DT.%26aulast%3DKessler%26aufirst%3DT.%26aulast%3DHerrera%26aufirst%3DF.%26aulast%3DKienast%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520%2528VEGF%2529%2520and%2520its%2520cellular%2520receptor%2520KDR%2520%2528VEGFR-2%2529%2520in%2520the%2520bone%2520marrow%2520of%2520patients%2520with%2520acute%2520myeloid%2520leukemia%26jtitle%3DLeukemia%26date%3D2002%26volume%3D16%26spage%3D1302%26epage%3D1310%26doi%3D10.1038%2Fsj.leu.2402534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Stefanik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmey, J. H.</span><span> </span><span class="NLM_article-title">Vascular Endothelial Growth Factor in Malignant Disease of the Central Nervous System</span> <span class="citation_source-journal">VEGF and Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1007/978-1-4419-9148-5_9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2F978-1-4419-9148-5_9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Kgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=72-82&author=D.+Stefanikauthor=J.+H.+Harmey&title=Vascular+Endothelial+Growth+Factor+in+Malignant+Disease+of+the+Central+Nervous+System&doi=10.1007%2F978-1-4419-9148-5_9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Vascular endothelial growth factor in malignant disease of the central nervous system</span></div><div class="casAuthors">Stefanik, David</div><div class="citationInfo"><span class="NLM_cas:title">VEGF and Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">69GMAZ</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Vascular Endothelial Growth Factor (VEGF) is a major contributor to the growth of malignant tumors of the central nervous system.  It stimulates tumor angiogenesis and vascular proliferation characteristic of high grade gliomas.  Elevated expression of VEGF is one the factors responsible for the virulent nature of these tumors.  The prodn. of VEGF by malignant glial cells in response to ionizing radiation contributes to treatment failure.  The rat C6 glioma is similar to human gliomas with respect to VEGF pathophysiol.  Interruption of VEGF-Receptor signaling in preclin. models effectively suppresses tumor growth and demonstrates the potential for anti-angiogenic therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfoquTOdbjfrVg90H21EOLACvtfcHk0lg8LoMBhkH-qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Kgs7k%253D&md5=07224f3d57e29325cd04ce58bf41e436</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-9148-5_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-9148-5_9%26sid%3Dliteratum%253Aachs%26aulast%3DStefanik%26aufirst%3DD.%26aulast%3DHarmey%26aufirst%3DJ.%2BH.%26atitle%3DVascular%2520Endothelial%2520Growth%2520Factor%2520in%2520Malignant%2520Disease%2520of%2520the%2520Central%2520Nervous%2520System%26jtitle%3DVEGF%2520and%2520Cancer%26date%3D2004%26spage%3D72%26epage%3D82%26doi%3D10.1007%2F978-1-4419-9148-5_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Dias, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heissig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadburn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hyjek, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, M.</span><span> </span><span class="NLM_article-title">Autocrine stimulation of VEGFR-2 activates human leukemic cell growth and migration</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">521</span><span class="refDoi"> DOI: 10.1172/JCI8978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1172%2FJCI8978" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2000&pages=511-521&author=S.+Diasauthor=K.+Hattoriauthor=Z.+Zhuauthor=B.+Heissigauthor=M.+Choyauthor=W.+Laneauthor=Y.+Wuauthor=A.+Chadburnauthor=E.+Hyjekauthor=M.+Gill&title=Autocrine+stimulation+of+VEGFR-2+activates+human+leukemic+cell+growth+and+migration&doi=10.1172%2FJCI8978"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1172%2FJCI8978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI8978%26sid%3Dliteratum%253Aachs%26aulast%3DDias%26aufirst%3DS.%26aulast%3DHattori%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DHeissig%26aufirst%3DB.%26aulast%3DChoy%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DHyjek%26aufirst%3DE.%26aulast%3DGill%26aufirst%3DM.%26atitle%3DAutocrine%2520stimulation%2520of%2520VEGFR-2%2520activates%2520human%2520leukemic%2520cell%2520growth%2520and%2520migration%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2000%26volume%3D106%26spage%3D511%26epage%3D521%26doi%3D10.1172%2FJCI8978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Salven, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heikkila, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joensuu, H.</span><span> </span><span class="NLM_article-title">Enhanced expression of vascular endothelial growth factor in metastatic melanoma</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">930</span><span class="NLM_x">–</span> <span class="NLM_lpage">934</span><span class="refDoi"> DOI: 10.1038/bjc.1997.486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fbjc.1997.486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=9328154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVGqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=1997&pages=930-934&author=P.+Salvenauthor=P.+Heikkilaauthor=H.+Joensuu&title=Enhanced+expression+of+vascular+endothelial+growth+factor+in+metastatic+melanoma&doi=10.1038%2Fbjc.1997.486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced expression of vascular endothelial growth factor in metastatic melanoma</span></div><div class="casAuthors">Salven, P.; Heikkila, P.; Joensuu, H.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">930-934</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Churchill Livingstone</span>)
        </div><div class="casAbstract">Tumor growth is dependent on angiogenesis.  Vascular endothelial growth factor (VEGF) is a secreted endothelial cell-specific cytokine.  VEGF is angiogenic in vivo and it also acts as a vascular permeability factor.  VEGF is overexpressed in many skin disorders characterized by angiogenesis and increased vascular permeability.  The authors investigated VEGF expression in 22 primary cutaneous melanomas, 33 melanoma metastases and six nevocellular nevi using immunohistochem.  VEGF accumulated on the vascular endothelia in the normal dermis, suggesting that a constitutive low level of VEGF expression may regulate skin vessel function under normal physiol. conditions.  No VEGF was detected in the cells of nevocellular nevi or normal dermis.  In contrast, 32% of the primary and 91% of the metastatic melanomas contained melanoma cells staining for VEGF.  Expression of VEGF was more frequent in metastases than in primary melanomas.  Tumor-infiltrating inflammatory cells expressed VEGF in all melanomas.  A high no. of VEGF-expressing inflammatory cells was assocd. with high VEGF expression in melanoma cells.  The authors' results suggest that VEGF is up-regulated during the course of melanoma progression and dissemination and that tumor-infiltrating cells expressing VEGF may contribute to the progression of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_PeGlbNlEeLVg90H21EOLACvtfcHk0lg8LoMBhkH-qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVGqurw%253D&md5=26c25abadec8d64fcd733978bd1b5dc3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fbjc.1997.486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.1997.486%26sid%3Dliteratum%253Aachs%26aulast%3DSalven%26aufirst%3DP.%26aulast%3DHeikkila%26aufirst%3DP.%26aulast%3DJoensuu%26aufirst%3DH.%26atitle%3DEnhanced%2520expression%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520in%2520metastatic%2520melanoma%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1997%26volume%3D76%26spage%3D930%26epage%3D934%26doi%3D10.1038%2Fbjc.1997.486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Mehnert, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCarthy, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jilaveanu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aziz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camp, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimm, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kluger, H. M.</span><span> </span><span class="NLM_article-title">Quantitative expression of VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 in melanoma tissue microarrays</span> <span class="citation_source-journal">Hum. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">384</span><span class="refDoi"> DOI: 10.1016/j.humpath.2009.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2Fj.humpath.2009.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=20004943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitV2ktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2010&pages=375-384&author=J.+M.+Mehnertauthor=M.+M.+McCarthyauthor=L.+Jilaveanuauthor=K.+T.+Flahertyauthor=S.+Azizauthor=R.+L.+Campauthor=D.+L.+Rimmauthor=H.+M.+Kluger&title=Quantitative+expression+of+VEGF%2C+VEGF-R1%2C+VEGF-R2%2C+and+VEGF-R3+in+melanoma+tissue+microarrays&doi=10.1016%2Fj.humpath.2009.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative expression of VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 in melanoma tissue microarrays</span></div><div class="casAuthors">Mehnert, Janice M.; McCarthy, Mary M.; Jilaveanu, Lucia; Flaherty, Keith T.; Aziz, Saadia; Camp, Robert L.; Rimm, David L.; Kluger, Harriet M.</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-384</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Summary: Angiogenesis is required for progression and metastasis of melanoma.  Anal. of angiogenic mols. in benign and malignant tissues may allow identification of markers useful for prediction of sensitivity to antiangiogenic agents.  We hypothesized that differential expression of vascular endothelial growth factor (VEGF) and its receptors VEGF-R1, VEGF-R2, and VEGF-R3 would be higher in melanomas than nevi and higher in advanced melanoma.  Using automated quant. anal., we quantified VEGF, -R1, -R2 and -R3 expression in melanoma tissue microarrays composed of 540 nevi and 468 melanoma specimens (198 primaries, 270 metastases).  VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 expression was significantly higher in melanomas than nevi by unpaired t tests (P < .0001).  VEGF-R2 expression was higher in metastatic specimens (P < .0001), but VEGF-R3 expression was higher in primaries (P < .0001).  VEGF was coexpressed with all 3 receptors when assessed by Spearman's rank correlation.  VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 expression is higher in melanomas than nevi.  Higher expression of VEGF-R2 was found in metastases vs. primaries, supporting the idea that selection for an angiogenic phenotype in metastatic melanoma is conferred via up-regulation of VEGF-R2.  However, higher expression of VEGF-R3 was seen on primary lesions, potentially implicating this receptor in initiation of lymphatic tumor spread.  Clin. trials using antiangiogenic agents in melanoma should include correlative assays of VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 as biomarkers of response to therapy, preferably using quant. methods such as automated quant. anal.  Such assessments could assist with evaluation of these mols. as therapeutic targets in melanoma, ultimately facilitating improved selection of patients for treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyXtBqrIqco7Vg90H21EOLACvtfcHk0liiX2xH4jBB_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitV2ktLo%253D&md5=323546819131fb9059604c900374ed58</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2009.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2009.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DMehnert%26aufirst%3DJ.%2BM.%26aulast%3DMcCarthy%26aufirst%3DM.%2BM.%26aulast%3DJilaveanu%26aufirst%3DL.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26aulast%3DAziz%26aufirst%3DS.%26aulast%3DCamp%26aufirst%3DR.%2BL.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DKluger%26aufirst%3DH.%2BM.%26atitle%3DQuantitative%2520expression%2520of%2520VEGF%252C%2520VEGF-R1%252C%2520VEGF-R2%252C%2520and%2520VEGF-R3%2520in%2520melanoma%2520tissue%2520microarrays%26jtitle%3DHum.%2520Pathol.%26date%3D2010%26volume%3D41%26spage%3D375%26epage%3D384%26doi%3D10.1016%2Fj.humpath.2009.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Molhoek, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griesemann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershenwald, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigan, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slingluff, C. L.</span><span> </span><span class="NLM_article-title">Human melanoma cytolysis by combined inhibition of mammalian target of rapamycin and vascular endothelial growth factor/vascular endothelial growth factor receptor-2</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">4392</span><span class="NLM_x">–</span> <span class="NLM_lpage">4397</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-5844</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1158%2F0008-5472.CAN-07-5844" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=4392-4397&author=K.+R.+Molhoekauthor=H.+Griesemannauthor=J.+Shuauthor=J.+E.+Gershenwaldauthor=D.+L.+Brautiganauthor=C.+L.+Slingluff&title=Human+melanoma+cytolysis+by+combined+inhibition+of+mammalian+target+of+rapamycin+and+vascular+endothelial+growth+factor%2Fvascular+endothelial+growth+factor+receptor-2&doi=10.1158%2F0008-5472.CAN-07-5844"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-5844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-5844%26sid%3Dliteratum%253Aachs%26aulast%3DMolhoek%26aufirst%3DK.%2BR.%26aulast%3DGriesemann%26aufirst%3DH.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DGershenwald%26aufirst%3DJ.%2BE.%26aulast%3DBrautigan%26aufirst%3DD.%2BL.%26aulast%3DSlingluff%26aufirst%3DC.%2BL.%26atitle%3DHuman%2520melanoma%2520cytolysis%2520by%2520combined%2520inhibition%2520of%2520mammalian%2520target%2520of%2520rapamycin%2520and%2520vascular%2520endothelial%2520growth%2520factor%252Fvascular%2520endothelial%2520growth%2520factor%2520receptor-2%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D4392%26epage%3D4397%26doi%3D10.1158%2F0008-5472.CAN-07-5844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Molhoek, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasamny, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patterson, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slingluff, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigan, D. L.</span><span> </span><span class="NLM_article-title">VEGFR-2 expression in human melanoma: Revised assessment</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">2807</span><span class="NLM_x">–</span> <span class="NLM_lpage">2815</span><span class="refDoi"> DOI: 10.1002/ijc.25963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fijc.25963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=21544800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKgurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2011&pages=2807-2815&author=K.+R.+Molhoekauthor=G.+Erdagauthor=J.+Rasamnyauthor=C.+Murphyauthor=D.+Deaconauthor=J.+W.+Pattersonauthor=C.+L.+Slingluffauthor=D.+L.+Brautigan&title=VEGFR-2+expression+in+human+melanoma%3A+Revised+assessment&doi=10.1002%2Fijc.25963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">VEGFR-2 expression in human melanoma: Revised assessment</span></div><div class="casAuthors">Molhoek, Kerrington R.; Erdag, Gulsun; Rasamny, Jk; Murphy, Cheryl; Deacon, Donna; Patterson, James W.; Slingluff, Craig L.; Brautigan, David L.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2807-2815</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) is an angiogenic factor that also functions as an autocrine growth factor for VEGF receptor (VEGFR)-2+ melanomas.  In multiple studies, VEGFR-2 was detected by immunostaining in 78-89% of human melanoma cells, suggesting that most patients with melanoma would benefit from anti-VEGF therapy.  Here, we evaluated 167 human melanoma specimens in a tissue microarray to verify the presence of VEGFR-2, but found disparities in staining with com. antibodies A-3 and 55B11.  Antibody A-3 stained melanoma cells in 79% of specimens, consistent with published results; however, we noted extensive nonspecific staining of other cells such as smooth muscle and histiocytes.  In contrast, antibody 55B11 stained melanoma cells in only 7% (95% confidence interval: 3.3-11.5) of specimens.  As an internal pos. control for VEGFR-2 detection, vascular endothelial cells were stained with antibody 55B11 in all specimens.  We compared VEGFR-2+ and VEGFR-2- melanoma cell lines by immunoblotting and immunohistochem. after small interfering RNA (siRNA) knockdown and transient overexpression of VEGFR-2 to validate antibody specificity.  Immunoblotting revealed that A-3 primarily cross-reacted with several proteins in both cell lines and these were unaffected by siRNA knockdown of VEGFR-2.  In contrast, 55B11 staining of VEGFR-2+ cells was mostly eliminated by siRNA knockdown of VEGFR-2 and increased in VEGFR-2- melanoma cell lines following transfection to express ectopic VEGFR-2.  Our results show that relatively few melanoma cells (<10%) express detectable levels of VEGFR-2, and therefore, the majority of patients with melanoma are unlikely to benefit from antiproliferative effects of anti-VEGF therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozG_W-H4LjhbVg90H21EOLACvtfcHk0liiX2xH4jBB_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKgurrM&md5=f8037cc86c023b3b09285d315ae5df56</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fijc.25963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25963%26sid%3Dliteratum%253Aachs%26aulast%3DMolhoek%26aufirst%3DK.%2BR.%26aulast%3DErdag%26aufirst%3DG.%26aulast%3DRasamny%26aufirst%3DJ.%26aulast%3DMurphy%26aufirst%3DC.%26aulast%3DDeacon%26aufirst%3DD.%26aulast%3DPatterson%26aufirst%3DJ.%2BW.%26aulast%3DSlingluff%26aufirst%3DC.%2BL.%26aulast%3DBrautigan%26aufirst%3DD.%2BL.%26atitle%3DVEGFR-2%2520expression%2520in%2520human%2520melanoma%253A%2520Revised%2520assessment%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D129%26spage%3D2807%26epage%3D2815%26doi%3D10.1002%2Fijc.25963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Pla, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brossa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grolez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroy, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prevarskaya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gkika, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussolati, B.</span><span> </span><span class="NLM_article-title">Differential sensitivity of prostate tumor derived endothelial cells to sorafenib and sunitinib</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">939</span><span class="NLM_x">–</span> <span class="NLM_lpage">951</span><span class="refDoi"> DOI: 10.1186/1471-2407-14-939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1186%2F1471-2407-14-939" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=939-951&author=A.+F.+Plaauthor=A.+Brossaauthor=M.+Bernardiniauthor=T.+Genovaauthor=G.+Grolezauthor=A.+Villersauthor=X.+Leroyauthor=N.+Prevarskayaauthor=D.+Gkikaauthor=B.+Bussolati&title=Differential+sensitivity+of+prostate+tumor+derived+endothelial+cells+to+sorafenib+and+sunitinib&doi=10.1186%2F1471-2407-14-939"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-939%26sid%3Dliteratum%253Aachs%26aulast%3DPla%26aufirst%3DA.%2BF.%26aulast%3DBrossa%26aufirst%3DA.%26aulast%3DBernardini%26aufirst%3DM.%26aulast%3DGenova%26aufirst%3DT.%26aulast%3DGrolez%26aufirst%3DG.%26aulast%3DVillers%26aufirst%3DA.%26aulast%3DLeroy%26aufirst%3DX.%26aulast%3DPrevarskaya%26aufirst%3DN.%26aulast%3DGkika%26aufirst%3DD.%26aulast%3DBussolati%26aufirst%3DB.%26atitle%3DDifferential%2520sensitivity%2520of%2520prostate%2520tumor%2520derived%2520endothelial%2520cells%2520to%2520sorafenib%2520and%2520sunitinib%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D939%26epage%3D951%26doi%3D10.1186%2F1471-2407-14-939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Chartier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chenard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R.</span><span> </span><span class="NLM_article-title">Kinome render: a stand-alone and web-accessible tool to annotate the human protein kinome tree</span> <span class="citation_source-journal">PeerJ</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">16</span><span class="refDoi"> DOI: 10.7717/peerj.126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.7717%2Fpeerj.126" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=1-16&author=M.+Chartierauthor=T.+Chenardauthor=J.+Barkerauthor=R.+Najmanovich&title=Kinome+render%3A+a+stand-alone+and+web-accessible+tool+to+annotate+the+human+protein+kinome+tree&doi=10.7717%2Fpeerj.126"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.126%26sid%3Dliteratum%253Aachs%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DChenard%26aufirst%3DT.%26aulast%3DBarker%26aufirst%3DJ.%26aulast%3DNajmanovich%26aufirst%3DR.%26atitle%3DKinome%2520render%253A%2520a%2520stand-alone%2520and%2520web-accessible%2520tool%2520to%2520annotate%2520the%2520human%2520protein%2520kinome%2520tree%26jtitle%3DPeerJ%26date%3D2013%26volume%3D1%26spage%3D1%26epage%3D16%26doi%3D10.7717%2Fpeerj.126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Li, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, F.</span><span> </span><span class="NLM_article-title">The human kinome targeted by FDA approved multi-target drugs and combination products: a comparative study from the drug-target interaction network perspective</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0165737</span><span class="refDoi"> DOI: 10.1371/journal.pone.0165737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1371%2Fjournal.pone.0165737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=27828998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFent7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0165737&author=Y.+H.+Liauthor=P.+P.+Wangauthor=X.+X.+Liauthor=C.+Y.+Yuauthor=H.+Yangauthor=J.+Zhouauthor=W.+W.+Xueauthor=J.+Tanauthor=F.+Zhu&title=The+human+kinome+targeted+by+FDA+approved+multi-target+drugs+and+combination+products%3A+a+comparative+study+from+the+drug-target+interaction+network+perspective&doi=10.1371%2Fjournal.pone.0165737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The human kinome targeted by FDA approved multi-target drugs and combination products: a comparative study from the drug-target interaction network perspective</span></div><div class="casAuthors">Li, Ying Hong; Wang, Pan Pan; Li, Xiao Xu; Yu, Chun Yan; Yang, Hong; Zhou, Jin; Xue, Wei Wei; Tan, Jun; Zhu, Feng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0165737/1-e0165737/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human kinome is one of the most productive classes of drug target, and there is emerging necessity for treating complex diseases by means of polypharmacol. (multi-target drugs and combination products).  However, the advantages of the multi-target drugs and the combination products are still under debate.  A comparative anal. between FDA approved multi-target drugs and combination products, targeting the human kinome, was conducted by mapping targets onto the phylogenetic tree of the human kinome.  The approach of network medicine illustrating the drug-target interactions was applied to identify popular targets of multi-target drugs and combination products.  As identified, the multi-target drugs tended to inhibit target pairs in the human kinome, esp. the receptor tyrosine kinase family, while the combination products were able to against targets of distant homol. relationship.  This finding asked for choosing the combination products as a better soln. for designing drugs aiming at targets of distant homol. relationship.  Moreover, subnetworks of drug-target interactions in specific disease were generated, and mechanisms shared by multi-target drugs and combination products were identified.  In conclusion, this study performed an anal. between approved multi-target drugs and combination products against the human kinome, which could assist the discovery of next generation polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7nzlXtKvALVg90H21EOLACvtfcHk0ljL0NSPIHILEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFent7w%253D&md5=cbc355a8bcdf4b81584910521ab316b7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0165737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0165737%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%2BH.%26aulast%3DWang%26aufirst%3DP.%2BP.%26aulast%3DLi%26aufirst%3DX.%2BX.%26aulast%3DYu%26aufirst%3DC.%2BY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DXue%26aufirst%3DW.%2BW.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DF.%26atitle%3DThe%2520human%2520kinome%2520targeted%2520by%2520FDA%2520approved%2520multi-target%2520drugs%2520and%2520combination%2520products%253A%2520a%2520comparative%2520study%2520from%2520the%2520drug-target%2520interaction%2520network%2520perspective%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26spage%3De0165737%26doi%3D10.1371%2Fjournal.pone.0165737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">Protein kinases--the major drug targets of the twenty-first century?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">315</span><span class="refDoi"> DOI: 10.1038/nrd773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnrd773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=12120282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=309-315&author=P.+Cohen&title=Protein+kinases%2D%2Dthe+major+drug+targets+of+the+twenty-first+century%3F&doi=10.1038%2Fnrd773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Protein kinases - the major drug targets of the twenty-first century?</span></div><div class="casAuthors">Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">309-315</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein phosphorylation regulates most aspects of cell life, whereas abnormal phosphorylation is a cause or consequence of disease.  A growing interest in developing orally active protein-kinase inhibitors has recently culminated in the approval of the first of these drugs for clin. use.  Protein kinases have now become the second most important group of drug targets, after G-protein-coupled receptors.  Here, I give a personal view of some of the most important advances that have shaped this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRd_GqiGAWaLVg90H21EOLACvtfcHk0ljL0NSPIHILEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D&md5=67d2fbc11a55b5980ee0692f7bc13cc9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnrd773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd773%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DProtein%2520kinases--the%2520major%2520drug%2520targets%2520of%2520the%2520twenty-first%2520century%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D309%26epage%3D315%26doi%3D10.1038%2Fnrd773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Morphy, R.</span><span> </span><span class="NLM_article-title">Selectively nonselective kinase inhibition: striking the right balance</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1413</span><span class="NLM_x">–</span> <span class="NLM_lpage">1437</span><span class="refDoi"> DOI: 10.1021/jm901132v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901132v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlWmtbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1413-1437&author=R.+Morphy&title=Selectively+nonselective+kinase+inhibition%3A+striking+the+right+balance&doi=10.1021%2Fjm901132v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Selectively Nonselective Kinase Inhibition: Striking the Right Balance</span></div><div class="casAuthors">Morphy, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1413-1437</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Marketed kinase inhibitors (MKIs) can deliver superior efficacy compared to inhibitors with high specificity for a single kinase, and the recent introduction of several MKIs to the market opens the door to a new era of safer and effective anticancer therapy.  The key to combining high efficacy with acceptable safety is to inhibit multiple targets in a selectively nonselective fashion.  Strategies for intentionally designing MKIs are emerging, but the field is still in its infancy and we are as medicinal chemists currently on the steepest part of the learning curve.  MTDD can be time-consuming and expensive, and we need to become more proficient first at identifying disease-relevant target combinations and second at discovering MKIs that combine optimal physicochem. and biol. properties.  Bold and innovative medicinal chem. strategies are required to tackle "difficult combinations" where the disease rationale is compelling but where it is a struggle to combine all the desired attributes of an oral MKI drug into a single mol.  At present it is unclear to what extent MKIs with highly tuned selectivity profiles can be rationally designed, particularly for targets that are unrelated by sequence.  In addn. to the well-known selectivity challenge, the physicochem. property profiles of AT P-competitive MKIs can be inherently challenging and limited scope for patentability can also be a serious hindrance.  On the plus side, the amt. of kinase-specific structural information is growing very rapidly, and ultimately this may reveal distinct features and design rules that enable a medicinal chemist to rationally modify and refine the profile of MKIs.  In addn., increasing SAR knowledge is emerging from large scale panel screening with the binding profiles starting to reveal to medicinal chemists how chem. structure affects cross-reactivity across large parts of the kinome.  The merit of MKIs compared with single kinase inhibitors is a subject of controversy in drug discovery that is unlikely to be resolved in the near future.  At the start of a new MTDD project, a rigorous debate needs to take place as to whether it makes more sense to seek a combination of highly selective agents or a DML.  Many factors need to be taken into account in this decision such as the no., similarity, and promiscuity of the targets in the profile and the disease area.  Conformational plasticity and the occurrence of multiple binding modes complicate the in silico prediction of kinase polypharmacol. based solely upon protein structure.  The use of ligand-based similarity to assess the feasibility of a given combination can add real value.  Currently, serendipity plays a significant role in MKI discovery and many, if not most, MKIs have been discovered by chance during the search for selective inhibitors.  Medicinal chemists need to be alert to the possibilities when a surprising combination is found by chance.  To exploit such serendipity, you need a good appreciation of when you have a sufficiently high quality starting compd. and then you need to be able to make and test sufficient analogs to explore your new disease-based hypothesis.  MKIs are costly to develop and are consequently priced at a premium level, so they will need to show clear improvements in order to get reimbursement.  There have already been problems with reimbursement for some MKIs in some markets due to concerns from funding bodies over insufficient efficacy.  The true value of MKIs relative to other anticancer drugs still has to be established, and the results from recent clin. trials have been mixed.  Despite the broad activity profile of many MKIs, the patient response can be inconsistent and unpredictable.  The identification of predictive biomarkers of response or resistance is a crit. step to ascertain which specific combination of targets produces a significant clin. benefit with respect to specific tumor types.  More clin. feedback is needed to facilitate the design of the next generation of inhibitors with more precisely defined profiles.  Although it might seem immeasurably distant at the present time, the ultimate goal should be to derive the prerequisite knowledge and tools so that MTDD becomes a rational endeavor rather than a black box approach that relies upon serendipity.  This will help banish claims that MKIs are merely dirty, nonspecific drugs with insufficient specificity for treating a wider range of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-L-NWmtdikbVg90H21EOLACvtfcHk0ljL0NSPIHILEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlWmtbfF&md5=3c915b5debf9d005993c8ad31c68bc2e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm901132v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901132v%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26atitle%3DSelectively%2520nonselective%2520kinase%2520inhibition%253A%2520striking%2520the%2520right%2520balance%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1413%26epage%3D1437%26doi%3D10.1021%2Fjm901132v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Broekman, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giovannetti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, G. J.</span><span> </span><span class="NLM_article-title">Tyrosine kinase inhibitors: Multi-targeted or single-targeted?</span> <span class="citation_source-journal">World J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.5306/wjco.v2.i2.80</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.5306%2Fwjco.v2.i2.80" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=21603317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADC%252BC3Mrkt1arsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=80-93&author=F.+Broekmanauthor=E.+Giovannettiauthor=G.+J.+Peters&title=Tyrosine+kinase+inhibitors%3A+Multi-targeted+or+single-targeted%3F&doi=10.5306%2Fwjco.v2.i2.80"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine kinase inhibitors: Multi-targeted or single-targeted?</span></div><div class="casAuthors">Broekman Fleur; Giovannetti Elisa; Peters Godefridus J</div><div class="citationInfo"><span class="NLM_cas:title">World journal of clinical oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">80-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since in most tumors multiple signaling pathways are involved, many of the inhibitors in clinical development are designed to affect a wide range of targeted kinases.  The most important tyrosine kinase families in the development of tyrosine kinase inhibitors are the ABL, SCR, platelet derived growth factor, vascular endothelial growth factor receptor and epidermal growth factor receptor families.  Both multi-kinase inhibitors and single-kinase inhibitors have advantages and disadvantages, which are related to potential resistance mechanisms, pharmacokinetics, selectivity and tumor environment.  In different malignancies various tyrosine kinases are mutated or overexpressed and several resistance mechanisms exist.  Pharmacokinetics is influenced by interindividual differences and differs for two single targeted inhibitors or between patients treated by the same tyrosine kinase inhibitor.  Different tyrosine kinase inhibitors have various mechanisms to achieve selectivity, while differences in gene expression exist between tumor and stromal cells.  Considering these aspects, one type of inhibitor can generally not be preferred above the other, but will depend on the specific genetic constitution of the patient and the tumor, allowing personalized therapy.  The most effective way of cancer treatment by using tyrosine kinase inhibitors is to consider each patient/tumor individually and to determine the strategy that specifically targets the consequences of altered (epi)genetics of the tumor.  This strategy might result in treatment by a single multi kinase inhibitor for one patient, but in treatment by a couple of single kinase inhibitors for other patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUsa5qqJaBY_csfMJQC1ZdfW6udTcc2eYXgtE77wvPCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mrkt1arsw%253D%253D&md5=683eaad679244b5efc5813d1ffaaaf57</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.5306%2Fwjco.v2.i2.80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5306%252Fwjco.v2.i2.80%26sid%3Dliteratum%253Aachs%26aulast%3DBroekman%26aufirst%3DF.%26aulast%3DGiovannetti%26aufirst%3DE.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26atitle%3DTyrosine%2520kinase%2520inhibitors%253A%2520Multi-targeted%2520or%2520single-targeted%253F%26jtitle%3DWorld%2520J.%2520Clin.%2520Oncol.%26date%3D2011%26volume%3D2%26spage%3D80%26epage%3D93%26doi%3D10.5306%2Fwjco.v2.i2.80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Schrodinger, L.</span>Schrodinger software suite; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=L.+Schrodinger&title=Schrodinger+software+suite"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchrodinger%26aufirst%3DL.%26jtitle%3DSchrodinger%2520software%2520suite%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Halgren, T.</span><span> </span><span class="NLM_article-title">New Method for Fast and Accurate Binding-site Identification and Analysis</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00483.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1111%2Fj.1747-0285.2007.00483.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17381729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=146-148&author=T.+Halgren&title=New+Method+for+Fast+and+Accurate+Binding-site+Identification+and+Analysis&doi=10.1111%2Fj.1747-0285.2007.00483.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">New method for fast and accurate binding-site identification and analysis</span></div><div class="casAuthors">Halgren, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Structure-based drug design seeks to exploit the structure of protein-ligand or protein-protein binding sites, but the site is not always known at the outset.  Even when the site is known, the researcher may wish to identify alternative prospective binding sites that may result in different biol. effects or new class of compds.  It is also vital in lead optimization to clearly understand the degree to which known binders or docking hits satisfy or violate complementarity to the receptor.  SiteMap is a new technique for identifying potential binding sites and for predicting their druggability in lead-discovery applications and for characterizing binding sites and critically assessing prospective ligands in lead-optimization applications.  In large-scale validation tests, SiteMap correctly identifies the known binding site in > 96% of the cases, with best results (> 98%) coming for sites that bind ligands tightly.  It also accurately distinguishes between sites that bind ligands and sites that don't.  In binding-site anal., SiteMap provides a wealth of quant. and graphical information that can help guide efforts to modify ligand structure to enhance potency or improve phys. properties.  These attributes allow SiteMap to nicely complement techniques such as docking and computational lead optimization in structure-base drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245hmZWXilrVg90H21EOLACvtfcHk0lhCWLQoq56IMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D&md5=c233276ba9580393b282f255781f9ce6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00483.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00483.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DNew%2520Method%2520for%2520Fast%2520and%2520Accurate%2520Binding-site%2520Identification%2520and%2520Analysis%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D146%26epage%3D148%26doi%3D10.1111%2Fj.1747-0285.2007.00483.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lhCWLQoq56IMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Duan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">A point-charge force field for molecular mechanicssimulations of proteins based on condensed-phase quantum mechanical calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1999</span><span class="NLM_x">–</span> <span class="NLM_lpage">2012</span><span class="refDoi"> DOI: 10.1002/jcc.10349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fjcc.10349" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=1999-2012&author=Y.+Duanauthor=S.+Chowdhuryauthor=M.+C.+Leeauthor=G.+Xiongauthor=W.+Zhangauthor=R.+Yangauthor=R.+Luoauthor=T.+Leeauthor=J.+Caldwellauthor=J.+Wangauthor=P.+Kollman&title=A+point-charge+force+field+for+molecular+mechanicssimulations+of+proteins+based+on+condensed-phase+quantum+mechanical+calculations&doi=10.1002%2Fjcc.10349"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10349%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DChowdhury%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BC.%26aulast%3DXiong%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DA%2520point-charge%2520force%2520field%2520for%2520molecular%2520mechanicssimulations%2520of%2520proteins%2520based%2520on%2520condensed-phase%2520quantum%2520mechanical%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D1999%26epage%3D2012%26doi%3D10.1002%2Fjcc.10349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Quantum and statistical mechanical studies of liquids. 10. Transferable intermolecular potential functions for water, alcohols, and ethers. Application to liquid water</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">335</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span><span class="refDoi"> DOI: 10.1021/ja00392a016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00392a016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaL3MXotlCitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1981&pages=335-340&author=W.+L.+Jorgensen&title=Quantum+and+statistical+mechanical+studies+of+liquids.+10.+Transferable+intermolecular+potential+functions+for+water%2C+alcohols%2C+and+ethers.+Application+to+liquid+water&doi=10.1021%2Fja00392a016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum and statistical mechanical studies of liquids.  10.  Transferable intermolecular potential functions for water, alcohols, and ethers.  Application to liquid water</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">335-40</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Transferable intermol. potential functions (TIPS) suitable for use in liq. simulations are reported for water, alcs., and ethers.  Interaction sites are located on oxygens, hydroxyl hydrogens, and the carbons in alkyl groups.  Each type of site has Coulomb and Lennard-Jones parameters chosen to yield reasonable structural and energetic results for both gas-phase dimers and pure liqs.  A Monte Carlo simulation of liq. water at 25° using the TIP compares favorably with expt. or results from Clementi's CI potential except that the OO radial distribution function is calcd. to be too flat beyond the first solvent shell.  Simulations of liq. methanol and ethanol have also been carried out as described in the accompanying papers.  Overall, in view of the simplicity and transferability of the potentials, the initial results are most encouraging for the treatment of fluids with even more complex monomers and for extension to other types of interaction sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6epUf9b1HcrVg90H21EOLACvtfcHk0licWhcGAORIrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXotlCitA%253D%253D&md5=6160447b696ee61208228f2f542c4a74</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja00392a016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00392a016%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DQuantum%2520and%2520statistical%2520mechanical%2520studies%2520of%2520liquids.%252010.%2520Transferable%2520intermolecular%2520potential%2520functions%2520for%2520water%252C%2520alcohols%252C%2520and%2520ethers.%2520Application%2520to%2520liquid%2520water%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1981%26volume%3D103%26spage%3D335%26epage%3D340%26doi%3D10.1021%2Fja00392a016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Hess, B.</span><span> </span><span class="NLM_article-title">P-LINCS: A parallel linear constraint solver for molecular simulation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">116</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span><span class="refDoi"> DOI: 10.1021/ct700200b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700200b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKru7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=116-122&author=B.+Hess&title=P-LINCS%3A+A+parallel+linear+constraint+solver+for+molecular+simulation&doi=10.1021%2Fct700200b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">P-LINCS: A Parallel Linear Constraint Solver for Molecular Simulation</span></div><div class="casAuthors">Hess, Berk</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">116-122</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By removing the fastest degrees of freedom, constraints allow for an increase of the time step in mol. simulations.  In the last decade parallel simulations have become commonplace.  However, up till now efficient parallel constraint algorithms have not been used with domain decompn.  In this paper the parallel linear constraint solver (P-LINCS) is presented, which allows the constraining of all bonds in macromols.  Addnl. the energy conservation properties of (P-)LINCS are assessed in view of improvements in the accuracy of uncoupled angle constraints and integration in single precision.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXg1Gz9d_OfbVg90H21EOLACvtfcHk0licWhcGAORIrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKru7zL&md5=476d5ca2eb25574d44b775996fff7b75</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fct700200b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700200b%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26atitle%3DP-LINCS%253A%2520A%2520parallel%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D116%26epage%3D122%26doi%3D10.1021%2Fct700200b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Miller, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGee, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homeyer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">MMPBSA.py: an efficient program for end-state free energy calculations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3314</span><span class="NLM_x">–</span> <span class="NLM_lpage">3321</span><span class="refDoi"> DOI: 10.1021/ct300418h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300418h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3314-3321&author=B.+R.+Millerauthor=T.+D.+McGeeauthor=J.+M.+Swailsauthor=N.+Homeyerauthor=H.+Gohlkeauthor=A.+E.+Roitberg&title=MMPBSA.py%3A+an+efficient+program+for+end-state+free+energy+calculations&doi=10.1021%2Fct300418h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">MMPBSA.py: An Efficient Program for End-State Free Energy Calculations</span></div><div class="casAuthors">Miller, Bill R., III; McGee, T. Dwight, Jr.; Swails, Jason M.; Homeyer, Nadine; Gohlke, Holger; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3314-3321</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MM-PBSA is a post-processing end-state method to calc. free energies of mols. in soln.  MMPBSA.py is a program written in Python for streamlining end-state free energy calcns. using ensembles derived from mol. dynamics (MD) or Monte Carlo (MC) simulations.  Several implicit solvation models are available with MMPBSA.py, including the Poisson-Boltzmann Model, the Generalized Born Model, and the Ref. Interaction Site Model.  Vibrational frequencies may be calcd. using normal mode or quasi-harmonic anal. to approx. the solute entropy.  Specific interactions can also be dissected using free energy decompn. or alanine scanning.  A parallel implementation significantly speeds up the calcn. by dividing frames evenly across available processors.  MMPBSA.py is an efficient, user-friendly program with the flexibility to accommodate the needs of users performing end-state free energy calcns.  The source code can be downloaded at http://ambermd.org/ with AmberTools, released under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZK3C1jqiSULVg90H21EOLACvtfcHk0licWhcGAORIrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP&md5=cc4148bd8f70c7cad94fd3ec6f580e52</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fct300418h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300418h%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DB.%2BR.%26aulast%3DMcGee%26aufirst%3DT.%2BD.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DMMPBSA.py%253A%2520an%2520efficient%2520program%2520for%2520end-state%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3314%26epage%3D3321%26doi%3D10.1021%2Fct300418h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Chéron, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jasty, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakhnovich, E. I.</span><span> </span><span class="NLM_article-title">OpenGrowth: an automated and rational algorithm for finding new protein ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4171</span><span class="NLM_x">–</span> <span class="NLM_lpage">4188</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00886</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00886" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4171-4188&author=N.+Ch%C3%A9ronauthor=N.+Jastyauthor=E.+I.+Shakhnovich&title=OpenGrowth%3A+an+automated+and+rational+algorithm+for+finding+new+protein+ligands&doi=10.1021%2Facs.jmedchem.5b00886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">OpenGrowth: An Automated and Rational Algorithm for Finding New Protein Ligands</span></div><div class="casAuthors">Cheron, Nicolas; Jasty, Naveen; Shakhnovich, Eugene I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4171-4188</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new open-source software, called OpenGrowth, which aims to create de novo ligands by connecting small org. fragments in the active site of proteins.  Mol. growth is biased to produce structures that statistically resemble drugs in an input training database.  Consequently, the produced mols. have superior synthetic accessibility and pharmacokinetic properties compared with randomly grown mols.  The growth process can take into account the flexibility of the target protein and can be started from a seed to mimic R-group strategy or fragment-based drug discovery.  Primary applications of the software on the HIV-1 protease allowed us to quickly identify new inhibitors with a predicted Kd as low as 18 nM.  We also present a graphical user interface that allows a user to select easily the fragments to include in the growth process.  OpenGrowth is released under the GNU GPL license and is available free of charge on the authors' website and at http://opengrowth.sourceforge.net/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcRSoqoxTyrVg90H21EOLACvtfcHk0lhKCC6JDWHrCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI&md5=6f8895e71ca7749cc1dbe559bda1db7c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00886%26sid%3Dliteratum%253Aachs%26aulast%3DCh%25C3%25A9ron%26aufirst%3DN.%26aulast%3DJasty%26aufirst%3DN.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DOpenGrowth%253A%2520an%2520automated%2520and%2520rational%2520algorithm%2520for%2520finding%2520new%2520protein%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4171%26epage%3D4188%26doi%3D10.1021%2Facs.jmedchem.5b00886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Swetha, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaiah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anbarasu, A.</span><span> </span><span class="NLM_article-title">Molecular dynamics studies on D835N mutation in FLT3-its impact on FLT3 protein structure</span> <span class="citation_source-journal">J. Cell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">1439</span><span class="NLM_x">–</span> <span class="NLM_lpage">1445</span><span class="refDoi"> DOI: 10.1002/jcb.25434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fjcb.25434" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2016&pages=1439-1445&author=R.+G.+Swethaauthor=S.+Ramaiahauthor=A.+Anbarasu&title=Molecular+dynamics+studies+on+D835N+mutation+in+FLT3-its+impact+on+FLT3+protein+structure&doi=10.1002%2Fjcb.25434"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fjcb.25434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.25434%26sid%3Dliteratum%253Aachs%26aulast%3DSwetha%26aufirst%3DR.%2BG.%26aulast%3DRamaiah%26aufirst%3DS.%26aulast%3DAnbarasu%26aufirst%3DA.%26atitle%3DMolecular%2520dynamics%2520studies%2520on%2520D835N%2520mutation%2520in%2520FLT3-its%2520impact%2520on%2520FLT3%2520protein%2520structure%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2016%26volume%3D117%26spage%3D1439%26epage%3D1445%26doi%3D10.1002%2Fjcb.25434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Tardieu, J.-L.</span><span> </span><span class="NLM_article-title">HTRF KinEASE TK: a new solution for tyrosine kinase screening</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">i</span><span class="NLM_x">–</span> <span class="NLM_lpage">ii</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnmeth1082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=i-ii&author=J.-L.+Tardieu&title=HTRF+KinEASE+TK%3A+a+new+solution+for+tyrosine+kinase+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnmeth1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth1082%26sid%3Dliteratum%253Aachs%26aulast%3DTardieu%26aufirst%3DJ.-L.%26atitle%3DHTRF%2520KinEASE%2520TK%253A%2520a%2520new%2520solution%2520for%2520tyrosine%2520kinase%2520screening%26jtitle%3DNat.%2520Methods%26date%3D2007%26volume%3D4%26spage%3Di%26epage%3Dii" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 34 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xingye Chen, Wuchen Xie, Yan Yang, Yi Hua, GuoMeng Xing, Li Liang, Chenglong Deng, Yuchen Wang, Yuanrong Fan, Haichun Liu, Tao Lu, Yadong Chen, <span class="NLM_string-name hlFld-ContribAuthor">Yanmin Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Dual FGFR4 and EGFR Inhibitors by Machine Learning and Biological Evaluation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4640-4652. <a href="https://doi.org/10.1021/acs.jcim.0c00652" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00652</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00652%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BDual%252BFGFR4%252Band%252BEGFR%252BInhibitors%252Bby%252BMachine%252BLearning%252Band%252BBiological%252BEvaluation%26aulast%3DChen%26aufirst%3DXingye%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09062020%26date%3D24092020%26date%3D14092020%26volume%3D60%26issue%3D10%26spage%3D4640%26epage%3D4652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xingye Chen, Haichun Liu, Wuchen Xie, Yan Yang, Yuchen Wang, Yuanrong Fan, Yi Hua, Lu Zhu, Junnan Zhao, Tao Lu, Yadong Chen, <span class="NLM_string-name hlFld-ContribAuthor">Yanmin Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation of Crystal Structures in Structure-Based Virtual Screening for Protein Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (12)
                                     , 5244-5262. <a href="https://doi.org/10.1021/acs.jcim.9b00684" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00684</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00684%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInvestigation%252Bof%252BCrystal%252BStructures%252Bin%252BStructure-Based%252BVirtual%252BScreening%252Bfor%252BProtein%252BKinase%252BInhibitors%26aulast%3DChen%26aufirst%3DXingye%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17082019%26date%3D15112019%26date%3D05112019%26volume%3D59%26issue%3D12%26spage%3D5244%26epage%3D5262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hui Xu, Yuchen Wang, Peter W. Jurutka, Shen Wu, Yadong Chen, Chongjiang Cao, Guitang Chen, Bei Tian, Suilou Wang, <span class="NLM_string-name hlFld-ContribAuthor">Shujie Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">16α-Hydroxytrametenolic Acid from Poria cocos Improves Intestinal Barrier Function Through the Glucocorticoid Receptor-Mediated PI3K/Akt/NF-κB Pathway. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2019,</strong> <em>67 </em>
                                    (39)
                                     , 10871-10879. <a href="https://doi.org/10.1021/acs.jafc.9b04613" title="DOI URL">https://doi.org/10.1021/acs.jafc.9b04613</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.9b04613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.9b04613%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3D16%2525CE%2525B1-Hydroxytrametenolic%252BAcid%252Bfrom%252BPoria%252Bcocos%252BImproves%252BIntestinal%252BBarrier%252BFunction%252BThrough%252Bthe%252BGlucocorticoid%252BReceptor-Mediated%252BPI3K%25252FAkt%25252FNF-%2525CE%2525BAB%252BPathway%26aulast%3DXu%26aufirst%3DHui%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22072019%26date%3D13092019%26date%3D06092019%26date%3D23092019%26date%3D13092019%26volume%3D67%26issue%3D39%26spage%3D10871%26epage%3D10879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adel S.  Girgis</span>, <span class="hlFld-ContribAuthor ">Siva S.  Panda</span>, <span class="hlFld-ContribAuthor ">Aladdin M.  Srour</span>, <span class="hlFld-ContribAuthor ">Anwar  Abdelnaser</span>, <span class="hlFld-ContribAuthor ">Soad  Nasr</span>, <span class="hlFld-ContribAuthor ">Yassmin  Moatasim</span>, <span class="hlFld-ContribAuthor ">Omnia  Kutkat</span>, <span class="hlFld-ContribAuthor ">Ahmed  El Taweel</span>, <span class="hlFld-ContribAuthor ">Ahmed  Kandeil</span>, <span class="hlFld-ContribAuthor ">Ahmed  Mostafa</span>, <span class="hlFld-ContribAuthor ">Mohamed A.  Ali</span>, <span class="hlFld-ContribAuthor ">Nehmedo G.  Fawzy</span>, <span class="hlFld-ContribAuthor ">Mohamed S.  Bekheit</span>, <span class="hlFld-ContribAuthor ">ElSayed M.  Shalaby</span>, <span class="hlFld-ContribAuthor ">Lara  Gigli</span>, <span class="hlFld-ContribAuthor ">Walid  Fayad</span>, <span class="hlFld-ContribAuthor ">Ahmed A.F.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">3-Alkenyl-2-oxindoles: Synthesis, antiproliferative and antiviral properties against SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>114 </em>, 105131. <a href="https://doi.org/10.1016/j.bioorg.2021.105131" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105131</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105131%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3D3-Alkenyl-2-oxindoles%25253A%252BSynthesis%25252C%252Bantiproliferative%252Band%252Bantiviral%252Bproperties%252Bagainst%252BSARS-CoV-2%26aulast%3DGirgis%26aufirst%3DAdel%2BS.%26date%3D2021%26volume%3D114%26spage%3D105131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Neha  Upadhyay</span>, <span class="hlFld-ContribAuthor ">Kalpana  Tilekar</span>, <span class="hlFld-ContribAuthor ">Sabreena  Safuan</span>, <span class="hlFld-ContribAuthor ">Alan P.  Kumar</span>, <span class="hlFld-ContribAuthor ">Jimmy  Stalin</span>, <span class="hlFld-ContribAuthor ">Curzio  Ruegg</span>, <span class="hlFld-ContribAuthor ">Ramaa C S</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Anti‐angiogenic Drug Discovery Efforts To Combat Cancer. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2021,</strong> <em>6 </em>
                                    (23)
                                     , 5689-5700. <a href="https://doi.org/10.1002/slct.202101792" title="DOI URL">https://doi.org/10.1002/slct.202101792</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202101792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202101792%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DRecent%252BAnti%2525E2%252580%252590angiogenic%252BDrug%252BDiscovery%252BEfforts%252BTo%252BCombat%252BCancer%26aulast%3DUpadhyay%26aufirst%3DNeha%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D23%26spage%3D5689%26epage%3D5700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satyaveni  Malasala</span>, <span class="hlFld-ContribAuthor ">Anusha  Polomoni</span>, <span class="hlFld-ContribAuthor ">Md. Naiyaz  Ahmad</span>, <span class="hlFld-ContribAuthor ">Manjulika  Shukla</span>, <span class="hlFld-ContribAuthor ">Grace  Kaul</span>, <span class="hlFld-ContribAuthor ">Arunav  Dasgupta</span>, <span class="hlFld-ContribAuthor ">Sidharth  Chopra</span>, <span class="hlFld-ContribAuthor ">Srinivas  Nanduri</span>. </span><span class="cited-content_cbyCitation_article-title">Structure based design, synthesis and evaluation of new thienopyrimidine derivatives as anti-bacterial agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1234 </em>, 130168. <a href="https://doi.org/10.1016/j.molstruc.2021.130168" title="DOI URL">https://doi.org/10.1016/j.molstruc.2021.130168</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2021.130168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2021.130168%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DStructure%252Bbased%252Bdesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252Bnew%252Bthienopyrimidine%252Bderivatives%252Bas%252Banti-bacterial%252Bagents%26aulast%3DMalasala%26aufirst%3DSatyaveni%26date%3D2021%26volume%3D1234%26spage%3D130168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aram  Faraji</span>, <span class="hlFld-ContribAuthor ">Rasoul  Motahari</span>, <span class="hlFld-ContribAuthor ">Zaman  Hasanvand</span>, <span class="hlFld-ContribAuthor ">Tayebeh  Oghabi Bakhshaiesh</span>, <span class="hlFld-ContribAuthor ">Mahsa  Toolabi</span>, <span class="hlFld-ContribAuthor ">Setareh  Moghimi</span>, <span class="hlFld-ContribAuthor ">Loghman  Firoozpour</span>, <span class="hlFld-ContribAuthor ">Mohammad Amin  Boshagh</span>, <span class="hlFld-ContribAuthor ">Roya  Rahmani</span>, <span class="hlFld-ContribAuthor ">Shima H.M.E.  Ketabforoosh</span>, <span class="hlFld-ContribAuthor ">Hamid Reza  Bijanzadeh</span>, <span class="hlFld-ContribAuthor ">Rezvan  Esmaeili</span>, <span class="hlFld-ContribAuthor ">Alireza  Foroumadi</span>. </span><span class="cited-content_cbyCitation_article-title">Quinazolin-4(3H)-one based agents bearing thiadiazole-urea: Synthesis and evaluation of anti-proliferative and antiangiogenic activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>108 </em>, 104553. <a href="https://doi.org/10.1016/j.bioorg.2020.104553" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104553%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DQuinazolin-4%2525283H%252529-one%252Bbased%252Bagents%252Bbearing%252Bthiadiazole-urea%25253A%252BSynthesis%252Band%252Bevaluation%252Bof%252Banti-proliferative%252Band%252Bantiangiogenic%252Bactivity%26aulast%3DFaraji%26aufirst%3DAram%26date%3D2021%26volume%3D108%26spage%3D104553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xing-Rong  Wang</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Wen-Bo  Li</span>, <span class="hlFld-ContribAuthor ">Kai-Yan  Xu</span>, <span class="hlFld-ContribAuthor ">Xue-Peng  Qiao</span>, <span class="hlFld-ContribAuthor ">Xue-Li  Jing</span>, <span class="hlFld-ContribAuthor ">Zi-Xiao  Wang</span>, <span class="hlFld-ContribAuthor ">Chang-jiang  Yang</span>, <span class="hlFld-ContribAuthor ">Shi-Wu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel 2-(4-(1H-indazol-6-yl)-1H-pyrazol-1-yl)acetamide derivatives as potent VEGFR-2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>213 </em>, 113192. <a href="https://doi.org/10.1016/j.ejmech.2021.113192" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113192</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113192%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B2-%2525284-%2525281H-indazol-6-yl%252529-1H-pyrazol-1-yl%252529acetamide%252Bderivatives%252Bas%252Bpotent%252BVEGFR-2%252Binhibitors%26aulast%3DWang%26aufirst%3DXing-Rong%26date%3D2021%26volume%3D213%26spage%3D113192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siddharth J.  Modi</span>, <span class="hlFld-ContribAuthor ">Vithal M.  Kulkarni</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of structural requirements for the inhibition of VEGFR-2 tyrosine kinase: Binding site analysis of type II, ‘DFG-out’ inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>1 </em>, 1-16. <a href="https://doi.org/10.1080/07391102.2021.1872417" title="DOI URL">https://doi.org/10.1080/07391102.2021.1872417</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2021.1872417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2021.1872417%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DExploration%252Bof%252Bstructural%252Brequirements%252Bfor%252Bthe%252Binhibition%252Bof%252BVEGFR-2%252Btyrosine%252Bkinase%25253A%252BBinding%252Bsite%252Banalysis%252Bof%252Btype%252BII%25252C%252B%2525E2%252580%252598DFG-out%2525E2%252580%252599%252Binhibitors%26aulast%3DModi%26aufirst%3DSiddharth%2BJ.%26date%3D2021%26date%3D2021%26volume%3D1%26spage%3D1%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mona M.  Abdel-Atty</span>, <span class="hlFld-ContribAuthor ">Nahla A.  Farag</span>, <span class="hlFld-ContribAuthor ">Rabah A. T.  Serya</span>, <span class="hlFld-ContribAuthor ">Khaled A. M.  Abouzid</span>, <span class="hlFld-ContribAuthor ">Samar  Mowafy</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design, synthesis and
              in vitro
              biological evaluation of thienopyrimidine–hydroxamic acids as chimeric kinase HDAC inhibitors: a challenging approach to combat cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2021,</strong> <em>36 </em>
                                    (1)
                                     , 1290-1312. <a href="https://doi.org/10.1080/14756366.2021.1933465" title="DOI URL">https://doi.org/10.1080/14756366.2021.1933465</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2021.1933465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2021.1933465%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252Bdesign%25252C%252Bsynthesis%252Band%252Bin%252Bvitro%252Bbiological%252Bevaluation%252Bof%252Bthienopyrimidine%2525E2%252580%252593hydroxamic%252Bacids%252Bas%252Bchimeric%252Bkinase%252BHDAC%252Binhibitors%25253A%252Ba%252Bchallenging%252Bapproach%252Bto%252Bcombat%252Bcancer%26aulast%3DAbdel-Atty%26aufirst%3DMona%2BM.%26date%3D2021%26date%3D2021%26volume%3D36%26issue%3D1%26spage%3D1290%26epage%3D1312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siddharth J.  Modi</span>, <span class="hlFld-ContribAuthor ">Vithal M.  Kulkarni</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of VEGFR-2 inhibitors exerting significant anticancer activity against CD44+ and CD133+ cancer stem cells (CSCs): Reversal of TGF-β induced epithelial-mesenchymal transition (EMT) in hepatocellular carcinoma. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112851. <a href="https://doi.org/10.1016/j.ejmech.2020.112851" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112851</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112851%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BVEGFR-2%252Binhibitors%252Bexerting%252Bsignificant%252Banticancer%252Bactivity%252Bagainst%252BCD44%25252B%252Band%252BCD133%25252B%252Bcancer%252Bstem%252Bcells%252B%252528CSCs%252529%25253A%252BReversal%252Bof%252BTGF-%2525CE%2525B2%252Binduced%252Bepithelial-mesenchymal%252Btransition%252B%252528EMT%252529%252Bin%252Bhepatocellular%252Bcarcinoma%26aulast%3DModi%26aufirst%3DSiddharth%2BJ.%26date%3D2020%26volume%3D207%26spage%3D112851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ankita  Srivastava</span>, <span class="hlFld-ContribAuthor ">Kaneez  Fatima</span>, <span class="hlFld-ContribAuthor ">Eram  Fatima</span>, <span class="hlFld-ContribAuthor ">Arjun  Singh</span>, <span class="hlFld-ContribAuthor ">Aastha  Singh</span>, <span class="hlFld-ContribAuthor ">Aparna  Shukla</span>, <span class="hlFld-ContribAuthor ">Suaib  Luqman</span>, <span class="hlFld-ContribAuthor ">Karuna  Shanker</span>, <span class="hlFld-ContribAuthor ">Debabrata  Chanda</span>, <span class="hlFld-ContribAuthor ">Feroz  Khan</span>, <span class="hlFld-ContribAuthor ">Arvind S.  Negi</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorinated benzylidene indanone exhibits antiproliferative activity through modulation of microtubule dynamics and antiangiogenic activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2020,</strong> <em>154 </em>, 105513. <a href="https://doi.org/10.1016/j.ejps.2020.105513" title="DOI URL">https://doi.org/10.1016/j.ejps.2020.105513</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejps.2020.105513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejps.2020.105513%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DFluorinated%252Bbenzylidene%252Bindanone%252Bexhibits%252Bantiproliferative%252Bactivity%252Bthrough%252Bmodulation%252Bof%252Bmicrotubule%252Bdynamics%252Band%252Bantiangiogenic%252Bactivity%26aulast%3DSrivastava%26aufirst%3DAnkita%26date%3D2020%26volume%3D154%26spage%3D105513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuchen  Wang</span>, <span class="hlFld-ContribAuthor ">Shaohua  Hou</span>, <span class="hlFld-ContribAuthor ">Yu  Tong</span>, <span class="hlFld-ContribAuthor ">Hongmei  Li</span>, <span class="hlFld-ContribAuthor ">Yi  Hua</span>, <span class="hlFld-ContribAuthor ">Yuanrong  Fan</span>, <span class="hlFld-ContribAuthor ">Xingye  Chen</span>, <span class="hlFld-ContribAuthor ">Yan  Yang</span>, <span class="hlFld-ContribAuthor ">Haichun  Liu</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>, <span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of potent apoptosis signal-regulating kinase 1 inhibitors via integrated computational strategy and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>38 </em>
                                    (15)
                                     , 4385-4396. <a href="https://doi.org/10.1080/07391102.2019.1680439" title="DOI URL">https://doi.org/10.1080/07391102.2019.1680439</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2019.1680439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2019.1680439%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DDiscovery%252Bof%252Bpotent%252Bapoptosis%252Bsignal-regulating%252Bkinase%252B1%252Binhibitors%252Bvia%252Bintegrated%252Bcomputational%252Bstrategy%252Band%252Bbiological%252Bevaluation%26aulast%3DWang%26aufirst%3DYuchen%26date%3D2020%26date%3D2019%26volume%3D38%26issue%3D15%26spage%3D4385%26epage%3D4396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangjian  Yang</span>, <span class="hlFld-ContribAuthor ">Haiyan  Xu</span>, <span class="hlFld-ContribAuthor ">Lu  Yang</span>, <span class="hlFld-ContribAuthor ">Fei  Xu</span>, <span class="hlFld-ContribAuthor ">Shuyang  Zhang</span>, <span class="hlFld-ContribAuthor ">Yaning  Yang</span>, <span class="hlFld-ContribAuthor ">Yan  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Apatinib in combination with pemetrexed-platinum chemotherapy for chemo-naive non-squamous non-small cell lung cancer: a phase II clinical study. </span><span class="cited-content_cbyCitation_journal-name">Lung Cancer</span><span> <strong>2020,</strong> <em>147 </em>, 229-236. <a href="https://doi.org/10.1016/j.lungcan.2020.07.024" title="DOI URL">https://doi.org/10.1016/j.lungcan.2020.07.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lungcan.2020.07.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lungcan.2020.07.024%26sid%3Dliteratum%253Aachs%26jtitle%3DLung%2520Cancer%26atitle%3DApatinib%252Bin%252Bcombination%252Bwith%252Bpemetrexed-platinum%252Bchemotherapy%252Bfor%252Bchemo-naive%252Bnon-squamous%252Bnon-small%252Bcell%252Blung%252Bcancer%25253A%252Ba%252Bphase%252BII%252Bclinical%252Bstudy%26aulast%3DYang%26aufirst%3DGuangjian%26date%3D2020%26volume%3D147%26spage%3D229%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sravani  Sana</span>, <span class="hlFld-ContribAuthor ">Velma Ganga  Reddy</span>, <span class="hlFld-ContribAuthor ">Sonal  Bhandari</span>, <span class="hlFld-ContribAuthor ">T. Srinivasa  Reddy</span>, <span class="hlFld-ContribAuthor ">Ramya  Tokala</span>, <span class="hlFld-ContribAuthor ">Akash P.  Sakla</span>, <span class="hlFld-ContribAuthor ">Suresh K.  Bhargava</span>, <span class="hlFld-ContribAuthor ">Nagula  Shankaraiah</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of carbamide derived pyrimidine-thioindole conjugates as potential VEGFR-2 inhibitors with anti-angiogenesis effect. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112457. <a href="https://doi.org/10.1016/j.ejmech.2020.112457" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112457</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112457%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExploration%252Bof%252Bcarbamide%252Bderived%252Bpyrimidine-thioindole%252Bconjugates%252Bas%252Bpotential%252BVEGFR-2%252Binhibitors%252Bwith%252Banti-angiogenesis%252Beffect%26aulast%3DSana%26aufirst%3DSravani%26date%3D2020%26volume%3D200%26spage%3D112457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kornelia  Czaja</span>, <span class="hlFld-ContribAuthor ">Jacek  Kujawski</span>, <span class="hlFld-ContribAuthor ">Paweł  Śliwa</span>, <span class="hlFld-ContribAuthor ">Rafał  Kurczab</span>, <span class="hlFld-ContribAuthor ">Radosław  Kujawski</span>, <span class="hlFld-ContribAuthor ">Anna  Stodolna</span>, <span class="hlFld-ContribAuthor ">Agnieszka  Myślińska</span>, <span class="hlFld-ContribAuthor ">Marek K.  Bernard</span>. </span><span class="cited-content_cbyCitation_article-title">Theoretical Investigations on Interactions of Arylsulphonyl Indazole Derivatives as Potential Ligands of VEGFR2 Kinase. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (13)
                                     , 4793. <a href="https://doi.org/10.3390/ijms21134793" title="DOI URL">https://doi.org/10.3390/ijms21134793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21134793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21134793%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DTheoretical%252BInvestigations%252Bon%252BInteractions%252Bof%252BArylsulphonyl%252BIndazole%252BDerivatives%252Bas%252BPotential%252BLigands%252Bof%252BVEGFR2%252BKinase%26aulast%3DCzaja%26aufirst%3DKornelia%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D13%26spage%3D4793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Mingliang  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuchen  Wang</span>, <span class="hlFld-ContribAuthor ">Yuanrong  Fan</span>, <span class="hlFld-ContribAuthor ">Xingye  Chen</span>, <span class="hlFld-ContribAuthor ">Yan  Yang</span>, <span class="hlFld-ContribAuthor ">Yi  Hua</span>, <span class="hlFld-ContribAuthor ">Wuchen  Xie</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Weifang  Tang</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>, <span class="hlFld-ContribAuthor ">Haichun  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Protein–ligand interaction-guided discovery of novel VEGFR-2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>38 </em>
                                    (9)
                                     , 2559-2574. <a href="https://doi.org/10.1080/07391102.2019.1635915" title="DOI URL">https://doi.org/10.1080/07391102.2019.1635915</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2019.1635915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2019.1635915%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DProtein%2525E2%252580%252593ligand%252Binteraction-guided%252Bdiscovery%252Bof%252Bnovel%252BVEGFR-2%252Binhibitors%26aulast%3DZhang%26aufirst%3DYanmin%26date%3D2020%26date%3D2019%26volume%3D38%26issue%3D9%26spage%3D2559%26epage%3D2574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingze  Qin</span>, <span class="hlFld-ContribAuthor ">Ye  Tian</span>, <span class="hlFld-ContribAuthor ">Xiao  Han</span>, <span class="hlFld-ContribAuthor ">Qi  Cao</span>, <span class="hlFld-ContribAuthor ">Shuaishuai  Zheng</span>, <span class="hlFld-ContribAuthor ">Chunyang  Liu</span>, <span class="hlFld-ContribAuthor ">Xia  Wu</span>, <span class="hlFld-ContribAuthor ">Lei  Liu</span>, <span class="hlFld-ContribAuthor ">Yangyang  Meng</span>, <span class="hlFld-ContribAuthor ">Xiaobo  Wang</span>, <span class="hlFld-ContribAuthor ">Haotian  Zhang</span>, <span class="hlFld-ContribAuthor ">Yunlei  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Structural modifications of indolinones bearing a pyrrole moiety and discovery of a multi-kinase inhibitor with potent antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (11)
                                     , 115486. <a href="https://doi.org/10.1016/j.bmc.2020.115486" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115486</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115486%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructural%252Bmodifications%252Bof%252Bindolinones%252Bbearing%252Ba%252Bpyrrole%252Bmoiety%252Band%252Bdiscovery%252Bof%252Ba%252Bmulti-kinase%252Binhibitor%252Bwith%252Bpotent%252Bantitumor%252Bactivity%26aulast%3DQin%26aufirst%3DMingze%26date%3D2020%26volume%3D28%26issue%3D11%26spage%3D115486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hairui  Wu</span>, <span class="hlFld-ContribAuthor ">Xi  Gu</span>, <span class="hlFld-ContribAuthor ">Jinling  Li</span>, <span class="hlFld-ContribAuthor ">Mingxing  Wang</span>, <span class="hlFld-ContribAuthor ">Yanchun  Li</span>, <span class="hlFld-ContribAuthor ">Lei  Yuan</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Enlong  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of potential platelet-derived growth factor receptor α inhibitors by computational screening and binding simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2020,</strong> <em>96 </em>, 107527. <a href="https://doi.org/10.1016/j.jmgm.2019.107527" title="DOI URL">https://doi.org/10.1016/j.jmgm.2019.107527</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2019.107527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2019.107527%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DIdentification%252Bof%252Bpotential%252Bplatelet-derived%252Bgrowth%252Bfactor%252Breceptor%252B%2525CE%2525B1%252Binhibitors%252Bby%252Bcomputational%252Bscreening%252Band%252Bbinding%252Bsimulations%26aulast%3DWu%26aufirst%3DHairui%26date%3D2020%26volume%3D96%26spage%3D107527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Charlaine A  Aventurado</span>, <span class="hlFld-ContribAuthor ">Junie B  Billones</span>, <span class="hlFld-ContribAuthor ">Ross D  Vasquez</span>, <span class="hlFld-ContribAuthor ">Agnes L  Castillo</span>. </span><span class="cited-content_cbyCitation_article-title"><p>In Ovo and In Silico Evaluation of the Anti-Angiogenic Potential of Syringin</p>. </span><span class="cited-content_cbyCitation_journal-name">Drug Design, Development and Therapy</span><span> <strong>2020,</strong> <em>Volume 14 </em>, 5189-5204. <a href="https://doi.org/10.2147/DDDT.S271952" title="DOI URL">https://doi.org/10.2147/DDDT.S271952</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2147/DDDT.S271952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2147%2FDDDT.S271952%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Design%252C%2520Development%2520and%2520Therapy%26atitle%3D%25253Cp%25253EIn%252BOvo%252Band%252BIn%252BSilico%252BEvaluation%252Bof%252Bthe%252BAnti-Angiogenic%252BPotential%252Bof%252BSyringin%25253C%25252Fp%25253E%26aulast%3DAventurado%26aufirst%3DCharlaine%2BA%26date%3D2020%26date%3D2020%26volume%3DVolume%252014%26spage%3D5189%26epage%3D5204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sirpu Natesh  Nagabhishek</span>, <span class="hlFld-ContribAuthor ">Arumugam  Madan Kumar</span>, <span class="hlFld-ContribAuthor ">Sambhavi  B.</span>, <span class="hlFld-ContribAuthor ">Anandan  Balakrishnan</span>, <span class="hlFld-ContribAuthor ">Yash T.  Katakia</span>, <span class="hlFld-ContribAuthor ">Suvro  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Nagarajan  Nagasundaram</span>. </span><span class="cited-content_cbyCitation_article-title">A marine sponge associated fungal metabolite monacolin X suppresses angiogenesis by down regulating VEGFR2 signaling. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2019,</strong> <em>9 </em>
                                    (46)
                                     , 26646-26667. <a href="https://doi.org/10.1039/C9RA05262C" title="DOI URL">https://doi.org/10.1039/C9RA05262C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA05262C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA05262C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DA%252Bmarine%252Bsponge%252Bassociated%252Bfungal%252Bmetabolite%252Bmonacolin%252BX%252Bsuppresses%252Bangiogenesis%252Bby%252Bdown%252Bregulating%252BVEGFR2%252Bsignaling%26aulast%3DNagabhishek%26aufirst%3DSirpu%2BNatesh%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D46%26spage%3D26646%26epage%3D26667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siddharth J.  Modi</span>, <span class="hlFld-ContribAuthor ">Vithal M.  Kulkarni</span>. </span><span class="cited-content_cbyCitation_article-title">Vascular Endothelial Growth Factor Receptor (VEGFR-2)/KDR Inhibitors: Medicinal Chemistry Perspective. </span><span class="cited-content_cbyCitation_journal-name">Medicine in Drug Discovery</span><span> <strong>2019,</strong> <em>2 </em>, 100009. <a href="https://doi.org/10.1016/j.medidd.2019.100009" title="DOI URL">https://doi.org/10.1016/j.medidd.2019.100009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.medidd.2019.100009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.medidd.2019.100009%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicine%2520in%2520Drug%2520Discovery%26atitle%3DVascular%252BEndothelial%252BGrowth%252BFactor%252BReceptor%252B%252528VEGFR-2%252529%25252FKDR%252BInhibitors%25253A%252BMedicinal%252BChemistry%252BPerspective%26aulast%3DModi%26aufirst%3DSiddharth%2BJ.%26date%3D2019%26volume%3D2%26spage%3D100009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qian  Jiang</span>, <span class="hlFld-ContribAuthor ">Ning-Ling  Zhang</span>, <span class="hlFld-ContribAuthor ">Dai-Yuan  Ma</span>, <span class="hlFld-ContribAuthor ">Bang-Xian  Tan</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Xiang-Dong  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Efficacy and safety of apatinib plus docetaxel as the second or above line treatment in advanced nonsquamous NSCLC. </span><span class="cited-content_cbyCitation_journal-name">Medicine</span><span> <strong>2019,</strong> <em>98 </em>
                                    (26)
                                     , e16065. <a href="https://doi.org/10.1097/MD.0000000000016065" title="DOI URL">https://doi.org/10.1097/MD.0000000000016065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1097/MD.0000000000016065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1097%2FMD.0000000000016065%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicine%26atitle%3DEfficacy%252Band%252Bsafety%252Bof%252Bapatinib%252Bplus%252Bdocetaxel%252Bas%252Bthe%252Bsecond%252Bor%252Babove%252Bline%252Btreatment%252Bin%252Badvanced%252Bnonsquamous%252BNSCLC%26aulast%3DJiang%26aufirst%3DQian%26date%3D2019%26volume%3D98%26issue%3D26%26spage%3De16065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuchen  Wang</span>, <span class="hlFld-ContribAuthor ">Weineng  Zhou</span>, <span class="hlFld-ContribAuthor ">Yuanrong  Fan</span>, <span class="hlFld-ContribAuthor ">Junnan  Zhao</span>, <span class="hlFld-ContribAuthor ">Lu  Zhu</span>, <span class="hlFld-ContribAuthor ">Shuai  Lu</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>, <span class="hlFld-ContribAuthor ">Haichun  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A combined drug discovery strategy based on machine learning and molecular docking. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2019,</strong> <em>93 </em>
                                    (5)
                                     , 685-699. <a href="https://doi.org/10.1111/cbdd.13494" title="DOI URL">https://doi.org/10.1111/cbdd.13494</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13494%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DA%252Bcombined%252Bdrug%252Bdiscovery%252Bstrategy%252Bbased%252Bon%252Bmachine%252Blearning%252Band%252Bmolecular%252Bdocking%26aulast%3DZhang%26aufirst%3DYanmin%26date%3D2019%26date%3D2019%26volume%3D93%26issue%3D5%26spage%3D685%26epage%3D699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elham  Assareh</span>, <span class="hlFld-ContribAuthor ">Faramarz  Mehrnejad</span>, <span class="hlFld-ContribAuthor ">Kamran  Mansouri</span>, <span class="hlFld-ContribAuthor ">Ahmad Reza  Esmaeili Rastaghi</span>, <span class="hlFld-ContribAuthor ">Hossein  Naderi-Manesh</span>, <span class="hlFld-ContribAuthor ">S. Mohsen  Asghari</span>. </span><span class="cited-content_cbyCitation_article-title">A cyclic peptide reproducing the α1 helix of VEGF-B binds to VEGFR-1 and VEGFR-2 and inhibits angiogenesis and tumor growth. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2019,</strong> <em>476 </em>
                                    (4)
                                     , 645-663. <a href="https://doi.org/10.1042/BCJ20180823" title="DOI URL">https://doi.org/10.1042/BCJ20180823</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20180823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20180823%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DA%252Bcyclic%252Bpeptide%252Breproducing%252Bthe%252B%2525CE%2525B11%252Bhelix%252Bof%252BVEGF-B%252Bbinds%252Bto%252BVEGFR-1%252Band%252BVEGFR-2%252Band%252Binhibits%252Bangiogenesis%252Band%252Btumor%252Bgrowth%26aulast%3DAssareh%26aufirst%3DElham%26date%3D2019%26date%3D2019%26volume%3D476%26issue%3D4%26spage%3D645%26epage%3D663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Wei</span>, <span class="hlFld-ContribAuthor ">Pingping  Liang</span>, <span class="hlFld-ContribAuthor ">Junqi  Xie</span>, <span class="hlFld-ContribAuthor ">Chuanhui  Song</span>, <span class="hlFld-ContribAuthor ">Chuanchao  Tang</span>, <span class="hlFld-ContribAuthor ">Yufeng  Wang</span>, <span class="hlFld-ContribAuthor ">Xiteng  Yin</span>, <span class="hlFld-ContribAuthor ">Yu  Cai</span>, <span class="hlFld-ContribAuthor ">Wei  Han</span>, <span class="hlFld-ContribAuthor ">Xiaochen  Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Carrier-free nano-integrated strategy for synergetic cancer anti-angiogenic therapy and phototherapy. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 2778-2784. <a href="https://doi.org/10.1039/C8SC04123G" title="DOI URL">https://doi.org/10.1039/C8SC04123G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8SC04123G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8SC04123G%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DCarrier-free%252Bnano-integrated%252Bstrategy%252Bfor%252Bsynergetic%252Bcancer%252Banti-angiogenic%252Btherapy%252Band%252Bphototherapy%26aulast%3DWei%26aufirst%3DZheng%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D9%26spage%3D2778%26epage%3D2784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vladimir S.  D’yachenko</span>, <span class="hlFld-ContribAuthor ">Dmitry V.  Danilov</span>, <span class="hlFld-ContribAuthor ">Tatyana K.  Shkineva</span>, <span class="hlFld-ContribAuthor ">Irina А.  Vatsadze</span>, <span class="hlFld-ContribAuthor ">Vladimir V.  Burmistrov</span>, <span class="hlFld-ContribAuthor ">Gennady M.  Butov</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and properties of 1-[(adamantan-1-yl)methyl]-3-pyrazolyl ureas. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2019,</strong> <em>55 </em>
                                    (2)
                                     , 129-134. <a href="https://doi.org/10.1007/s10593-019-02428-2" title="DOI URL">https://doi.org/10.1007/s10593-019-02428-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-019-02428-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-019-02428-2%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DSynthesis%252Band%252Bproperties%252Bof%252B1-%25255B%252528adamantan-1-yl%252529methyl%25255D-3-pyrazolyl%252Bureas%26aulast%3DD%25E2%2580%2599yachenko%26aufirst%3DVladimir%2BS.%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D2%26spage%3D129%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nevine M.Y.  Elsayed</span>, <span class="hlFld-ContribAuthor ">Rabah A.T.  Serya</span>, <span class="hlFld-ContribAuthor ">Mai F.  Tolba</span>, <span class="hlFld-ContribAuthor ">Marawan  Ahmed</span>, <span class="hlFld-ContribAuthor ">Khaled  Barakat</span>, <span class="hlFld-ContribAuthor ">Dalal A.  Abou El Ella</span>, <span class="hlFld-ContribAuthor ">Khaled A.M.  Abouzid</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, biological evaluation and dynamics simulation of indazole derivatives with antiangiogenic and antiproliferative anticancer activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2019,</strong> <em>82 </em>, 340-359. <a href="https://doi.org/10.1016/j.bioorg.2018.10.071" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.10.071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.10.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.10.071%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Bbiological%252Bevaluation%252Band%252Bdynamics%252Bsimulation%252Bof%252Bindazole%252Bderivatives%252Bwith%252Bantiangiogenic%252Band%252Bantiproliferative%252Banticancer%252Bactivity%26aulast%3DElsayed%26aufirst%3DNevine%2BM.Y.%26date%3D2019%26volume%3D82%26spage%3D340%26epage%3D359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lu  Wang</span>, <span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Qing  Zhang</span>, <span class="hlFld-ContribAuthor ">Gaoyuan  Zhu</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Chunqi  Duan</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Weifang  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of potent Pan-Raf inhibitors with increased solubility to overcome drug resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>163 </em>, 243-255. <a href="https://doi.org/10.1016/j.ejmech.2018.11.033" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.033%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bpotent%252BPan-Raf%252Binhibitors%252Bwith%252Bincreased%252Bsolubility%252Bto%252Bovercome%252Bdrug%252Bresistance%26aulast%3DWang%26aufirst%3DLu%26date%3D2019%26volume%3D163%26spage%3D243%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">K.  Thirumurugan</span>, <span class="hlFld-ContribAuthor ">Sivalingam  Lakshmanan</span>, <span class="hlFld-ContribAuthor ">Dharman  Govindaraj</span>, <span class="hlFld-ContribAuthor ">D. Sam  Daniel Prabu</span>, <span class="hlFld-ContribAuthor ">N.  Ramalakshmi</span>, <span class="hlFld-ContribAuthor ">S.  Arul Antony</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and anti-inflammatory activity of pyrimidine scaffold benzamide derivatives as epidermal growth factor receptor tyrosine kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2018,</strong> <em>1171 </em>, 541-550. <a href="https://doi.org/10.1016/j.molstruc.2018.06.003" title="DOI URL">https://doi.org/10.1016/j.molstruc.2018.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2018.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2018.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Banti-inflammatory%252Bactivity%252Bof%252Bpyrimidine%252Bscaffold%252Bbenzamide%252Bderivatives%252Bas%252Bepidermal%252Bgrowth%252Bfactor%252Breceptor%252Btyrosine%252Bkinase%252Binhibitors%26aulast%3DThirumurugan%26aufirst%3DK.%26date%3D2018%26volume%3D1171%26spage%3D541%26epage%3D550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanyuan  Shan</span>, <span class="hlFld-ContribAuthor ">Binghe  Wang</span>, <span class="hlFld-ContribAuthor ">Jie  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">New strategies in achieving antiangiogenic effect: Multiplex inhibitors suppressing compensatory activations of RTKs. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2018,</strong> <em>38 </em>
                                    (5)
                                     , 1674-1705. <a href="https://doi.org/10.1002/med.21517" title="DOI URL">https://doi.org/10.1002/med.21517</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21517%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DNew%252Bstrategies%252Bin%252Bachieving%252Bantiangiogenic%252Beffect%25253A%252BMultiplex%252Binhibitors%252Bsuppressing%252Bcompensatory%252Bactivations%252Bof%252BRTKs%26aulast%3DShan%26aufirst%3DYuanyuan%26date%3D2018%26date%3D2018%26volume%3D38%26issue%3D5%26spage%3D1674%26epage%3D1705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Hao  Shi</span>, <span class="hlFld-ContribAuthor ">Feng-Tao  Liu</span>, <span class="hlFld-ContribAuthor ">Hao-Zhong  Tian</span>, <span class="hlFld-ContribAuthor ">Yan-Min  Zhang</span>, <span class="hlFld-ContribAuthor ">Nian-Guang  Li</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and structure-activity relationship of diaryl-ureas with novel isoxazol[3,4-b]pyridine-3-amino-structure as multi-target inhibitors against receptor tyrosine kinase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (16)
                                     , 4735-4744. <a href="https://doi.org/10.1016/j.bmc.2018.08.013" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.08.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.08.013%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bstructure-activity%252Brelationship%252Bof%252Bdiaryl-ureas%252Bwith%252Bnovel%252Bisoxazol%25255B3%25252C4-b%25255Dpyridine-3-amino-structure%252Bas%252Bmulti-target%252Binhibitors%252Bagainst%252Breceptor%252Btyrosine%252Bkinase%26aulast%3DShi%26aufirst%3DZhi-Hao%26date%3D2018%26volume%3D26%26issue%3D16%26spage%3D4735%26epage%3D4744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Na  Wei</span>, <span class="hlFld-ContribAuthor ">Jianqing  Liang</span>, <span class="hlFld-ContribAuthor ">Shengming  Peng</span>, <span class="hlFld-ContribAuthor ">Qiang  Sun</span>, <span class="hlFld-ContribAuthor ">Qiuyun  Dai</span>, <span class="hlFld-ContribAuthor ">Mingxin  Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of Axitinib Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (4)
                                     , 747. <a href="https://doi.org/10.3390/molecules23040747" title="DOI URL">https://doi.org/10.3390/molecules23040747</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23040747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23040747%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252BAxitinib%252BDerivatives%26aulast%3DWei%26aufirst%3DNa%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D4%26spage%3D747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia  Liu</span>, <span class="hlFld-ContribAuthor ">Bian  Ming</span>, <span class="hlFld-ContribAuthor ">Guo-Hua  Gong</span>, <span class="hlFld-ContribAuthor ">Di  Wang</span>, <span class="hlFld-ContribAuthor ">Gui-Lan  Bao</span>, <span class="hlFld-ContribAuthor ">Li-Jun  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Current research on anti-breast cancer synthetic compounds. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2018,</strong> <em>8 </em>
                                    (8)
                                     , 4386-4416. <a href="https://doi.org/10.1039/C7RA12912B" title="DOI URL">https://doi.org/10.1039/C7RA12912B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA12912B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA12912B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DCurrent%252Bresearch%252Bon%252Banti-breast%252Bcancer%252Bsynthetic%252Bcompounds%26aulast%3DLiu%26aufirst%3DJia%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D8%26spage%3D4386%26epage%3D4416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Workflow for this investigation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding site of VEGFR-2 in complex with sorafenib (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) The combinatorial approach of BREED. (b) Different colors were used to denote the scaffolds from different input structures: the 6,7-dimethoxyquinazoline group from vandetanib (green), the naphthamide portion of Hit_Scaffold_4 (dark pink), and the 1-(4-chloro-3-(trifluoromethyl)phenyl)-3-methylurea moiety (blue). Compound <b>6</b> (Cmpd <b>6</b>) is colored cyan, and important active site residues are labeled and colored magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding affinity prediction model (a) and the average binding affinity of both ligands from the crystal structures and the selected compounds. (b) Delta G means Δ<i>G</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. RMSD values during the 20 ns MD for five FDA approved drugs crystallized with the VEGFR-2 protein and the designed compound <b>6</b> docked to the VEGFR-2 protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>). Blue represents the fluctuation of protein atoms, and orange represents the fluctuation of ligand atoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Number of hydrogen bonds formed during 20 ns MD simulations for five FDA approved drugs crystallized with VEGFR-2 as well as designed compound <b>6</b> docked to the VEGFR-2 protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0014.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6</b>, <b>7</b>, <b>8</b>, <b>10</b>, and <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) anhydrous DMF, NaH, 90 °C, 24 h; (b) Fe, NH<sub>4</sub>Cl, 70% i-PrOH, 85 °C, 4 h; (c) THF, rt, 5 h; (d) THF, TEA, 0 °C to rt, 1 h/pyridine, rt, 1 h.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0015.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>19</b>, <b>20</b>, <b>21</b>, <b>22</b>, and <b>23</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (e) 2-bromopropane, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 0.5 h; (f) 20% Pd/C, H<sub>2</sub>, ethyl acetate, rt, 5–7 h; (g) NBS, AIBN, 1,2-dichloroethane, 90 °C, 12 h; (h) morpholine, K<sub>2</sub>CO<sub>3</sub>, rt, 16 h; (i) 20% Pd/C, H<sub>2</sub>, EtOH, rt, 2 h; (j) (<b>19</b>) 4-nitrophenyl carbonochloridate, DCM, RNH<sub>2</sub> pyridine, rt, 24 h; (<b>20</b>, <b>21</b>, <b>22</b>, and <b>23</b>) triphosgene, DCM, TEA, RNH<sub>2</sub>, 0 °C to rt, 24 h.</p></p></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Dose-dependent enzymatic activity of the 10 target compounds against VEGFR-2. Results were the average of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding conformation of the 10 designed compounds. (a) All 10 compounds, (b) compound <b>19</b>, and (c) compound <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. RMSD (a and b), RMSF (c), and the number of hydrogen bonds (d) of the protein and ligands in the MD simulations for the most active compound <b>19</b> (IC<sub>50</sub> = 0.75 nM, black) and the second least active <b>11</b> (IC<sub>50</sub> = 3937 nM, red). One nm is equal to 10 Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0010.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Cellular activity against two cell lines, (a) HUVECs and (b) MDA-MB-435 cell line. Results were from three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0011.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Dose-dependent growth inhibition curves for compound <b>23</b> on 58 cell lines conducted by NCI. A growth percent of 100 is the growth seen in untreated cells, while a growth percent of 0 corresponds to no net growth during the assay and a growth percent of −100 indicates the time when all cells are killed. Three endpoints are routinely calculated: (1) GI<sub>50</sub>, the drug concentration yielding a growth percent of 50 (i.e., 50% growth inhibition); (2) TGI (total growth inhibition), the concentration resulting in a growth percent of 0; (3) LC<sub>50</sub>, the concentration when a growth percent of −50 or lethality in 50% of the starting cells is shown. These endpoints are illustrated for one of the melanoma cell lines MALME-CM (red open diamond).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0012.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Comparison of the GI<sub>50</sub>, TGI, and LC<sub>50</sub> values for compound <b>23</b> (black), sorafenib (red), and sunitinib (blue). The <i>y</i>-axis is the log value of the endpoints, and a difference of 1 indicates a 10-fold difference in the endpoint values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/medium/jm-2017-01091x_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0013.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Kinome phylogenetic trees of <b>23</b> and sunitinib. (a) Percent inhibition by <b>23</b> against all 349 therapeutically important kinases. (b) <i>K</i><sub>d</sub> values of sunitinib. The AGC branch contains the PKA, PKG, and PKC families. CAMK is calcium/calmodulin-dependent protein kinase. CK1 is casein kinase 1. The CMGC branch contains the CDK, MAPK, GSK3, and CLK families. STE refers to homologues of yeast sterile 7, sterile 11, and sterile 20 kinases. TK is tyrosine kinase, and TKL corresponds to tyrosine kinase-like proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01091/20180110/images/large/jm-2017-01091x_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01091&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 63 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Holmes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, O. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, M. J.</span><span> </span><span class="NLM_article-title">Vascular endothelial growth factor receptor-2: structure, function, intracellular signalling and therapeutic inhibition</span> <span class="citation_source-journal">Cell. Signalling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2003</span><span class="NLM_x">–</span> <span class="NLM_lpage">2012</span><span class="refDoi"> DOI: 10.1016/j.cellsig.2007.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2Fj.cellsig.2007.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17658244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFylsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2007&pages=2003-2012&author=K.+Holmesauthor=O.+L.+Robertsauthor=A.+M.+Thomasauthor=M.+J.+Cross&title=Vascular+endothelial+growth+factor+receptor-2%3A+structure%2C+function%2C+intracellular+signalling+and+therapeutic+inhibition&doi=10.1016%2Fj.cellsig.2007.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Vascular endothelial growth factor receptor-2: structure, function, intracellular signalling and therapeutic inhibition</span></div><div class="casAuthors">Holmes, Katherine; Roberts, Owain Ll; Thomas, Angharad M.; Cross, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2003-2012</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Vascular endothelial growth factors (VEGFs) regulate vascular development, angiogenesis and lymphangiogenesis by binding to a no. of receptors.  VEGFR-1 is required for the recruitment of hematopoietic stem cells and the migration of monocytes and macrophages, VEGFR-2 regulates vascular endothelial function and VEGFR-3 regulates lymphatic endothelial cell function.  Over the last decade, considerable progress has been made in delineating the VEGFR-2 specific intracellular signaling cascades leading to proliferation, migration, survival and increased permeability, each of which contributes to the angiogenic response.  Furthermore, therapeutic inhibition of VEGFR-2 action is now having an impact in the clinic for the treatment of a no. of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTYxPMokVV57Vg90H21EOLACvtfcHk0lhs-AnPgfUbEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFylsLs%253D&md5=44a935f36d29b000d2a9ff514709e9c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2007.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2007.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DK.%26aulast%3DRoberts%26aufirst%3DO.%2BL.%26aulast%3DThomas%26aufirst%3DA.%2BM.%26aulast%3DCross%26aufirst%3DM.%2BJ.%26atitle%3DVascular%2520endothelial%2520growth%2520factor%2520receptor-2%253A%2520structure%252C%2520function%252C%2520intracellular%2520signalling%2520and%2520therapeutic%2520inhibition%26jtitle%3DCell.%2520Signalling%26date%3D2007%26volume%3D19%26spage%3D2003%26epage%3D2012%26doi%3D10.1016%2Fj.cellsig.2007.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Folkman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klagsbrun, M.</span><span> </span><span class="NLM_article-title">Angiogenic factors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">442</span><span class="NLM_x">–</span> <span class="NLM_lpage">447</span><span class="refDoi"> DOI: 10.1126/science.2432664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1126%2Fscience.2432664" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=442-447&author=J.+Folkmanauthor=M.+Klagsbrun&title=Angiogenic+factors&doi=10.1126%2Fscience.2432664"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.2432664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2432664%26sid%3Dliteratum%253Aachs%26aulast%3DFolkman%26aufirst%3DJ.%26aulast%3DKlagsbrun%26aufirst%3DM.%26atitle%3DAngiogenic%2520factors%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D442%26epage%3D447%26doi%3D10.1126%2Fscience.2432664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kiselyov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakin, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tkachenko, S. E.</span><span> </span><span class="NLM_article-title">VEGF/VEGFR signalling as a target for inhibiting angiogenesis</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1517/13543784.16.1.83</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1517%2F13543784.16.1.83" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17155856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1yisL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2007&pages=83-107&author=A.+Kiselyovauthor=K.+V.+Balakinauthor=S.+E.+Tkachenko&title=VEGF%2FVEGFR+signalling+as+a+target+for+inhibiting+angiogenesis&doi=10.1517%2F13543784.16.1.83"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">VEGF/VEGFR signalling as a target for inhibiting angiogenesis</span></div><div class="casAuthors">Kiselyov, Alex; Balakin, Konstantin V.; Tkachenko, Sergey E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-107</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  VEGFs and a resp. family of tyrosine kinases receptors (VEGFRs) are key proteins modulating angiogenesis, the formation of new vasculature from an existing vascular network.  There has been considerable evidence in vivo, including clin. observations, that abnormal angiogenesis is implicated in a no. of disease conditions, which include rheumatoid arthritis, inflammation, cancer, psoriasis, degenerative eye conditions and others.  Antiangiogenic therapies based on inhibition of VEGF/VEGFR signalling were reported to be powerful clin. strategies in oncol. and ophthalmol.  Current efforts have yielded promising clin. data for several antiangiogenic therapeutics.  In this review, the authors elucidate key aspects of VEGFR signalling, as well as clin. relevant strategies for the inhibition of VEGF-induced angiogenesis, with an emphasis on small-mol. VEGFR inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphlleEmD9eirVg90H21EOLACvtfcHk0lgb_M1bcMt5Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1yisL7J&md5=7926ed15c751b2903bed9eefb8ef9f98</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1517%2F13543784.16.1.83&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.16.1.83%26sid%3Dliteratum%253Aachs%26aulast%3DKiselyov%26aufirst%3DA.%26aulast%3DBalakin%26aufirst%3DK.%2BV.%26aulast%3DTkachenko%26aufirst%3DS.%2BE.%26atitle%3DVEGF%252FVEGFR%2520signalling%2520as%2520a%2520target%2520for%2520inhibiting%2520angiogenesis%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2007%26volume%3D16%26spage%3D83%26epage%3D107%26doi%3D10.1517%2F13543784.16.1.83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Musumeci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brullo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenone, S.</span><span> </span><span class="NLM_article-title">Vascular endothelial growth factor (VEGF) receptors: drugs and new inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">10797</span><span class="NLM_x">–</span> <span class="NLM_lpage">10822</span><span class="refDoi"> DOI: 10.1021/jm301085w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301085w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10797-10822&author=F.+Musumeciauthor=M.+Radiauthor=C.+Brulloauthor=S.+Schenone&title=Vascular+endothelial+growth+factor+%28VEGF%29+receptors%3A+drugs+and+new+inhibitors&doi=10.1021%2Fjm301085w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm301085w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301085w%26sid%3Dliteratum%253Aachs%26aulast%3DMusumeci%26aufirst%3DF.%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DBrullo%26aufirst%3DC.%26aulast%3DSchenone%26aufirst%3DS.%26atitle%3DVascular%2520endothelial%2520growth%2520factor%2520%2528VEGF%2529%2520receptors%253A%2520drugs%2520and%2520new%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10797%26epage%3D10822%26doi%3D10.1021%2Fjm301085w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Folkman, J.</span><span> </span><span class="NLM_article-title">Angiogenesis in cancer, vascular, rheumatoid and other disease</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1038/nm0195-27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnm0195-27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=7584949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK2MXjs1KnsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1995&pages=27-30&author=J.+Folkman&title=Angiogenesis+in+cancer%2C+vascular%2C+rheumatoid+and+other+disease&doi=10.1038%2Fnm0195-27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Angiogenesis in cancer, vascular, rheumatoid and other disease</span></div><div class="casAuthors">Folkman, Judah</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-31</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">A review with ∼72 refs.  Recent discoveries of endogenous neg. regulators of angiogenesis, thrombospondin, angiostatin and glioma-derived angiogenesis inhibitory factor, all assocd. with neovascularized tumors, suggest a new paradigm of tumorigenesis.  It is now helpful to think of the switch to the angiogenic phenotype as a net balance of pos. and neg. regulators of blood vessel growth.  The extent to which the neg. regulators are decreased during this switch may dictate whether a primary tumor grows rapidly or slowly and whether metastases grow at all.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpF2nWHt2eJbVg90H21EOLACvtfcHk0lgb_M1bcMt5Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjs1KnsLk%253D&md5=849f4f3828ac5d78886d666787f72d4f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnm0195-27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0195-27%26sid%3Dliteratum%253Aachs%26aulast%3DFolkman%26aufirst%3DJ.%26atitle%3DAngiogenesis%2520in%2520cancer%252C%2520vascular%252C%2520rheumatoid%2520and%2520other%2520disease%26jtitle%3DNat.%2520Med.%26date%3D1995%26volume%3D1%26spage%3D27%26epage%3D30%26doi%3D10.1038%2Fnm0195-27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Liotta, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steeg, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stetler-Stevenson, W. G.</span><span> </span><span class="NLM_article-title">Cancer metastasis and angiogenesis: an imbalance of positive and negative regulation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1016/0092-8674(91)90642-C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2F0092-8674%2891%2990642-C" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1991&pages=327-336&author=L.+A.+Liottaauthor=P.+S.+Steegauthor=W.+G.+Stetler-Stevenson&title=Cancer+metastasis+and+angiogenesis%3A+an+imbalance+of+positive+and+negative+regulation&doi=10.1016%2F0092-8674%2891%2990642-C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990642-C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990642-C%26sid%3Dliteratum%253Aachs%26aulast%3DLiotta%26aufirst%3DL.%2BA.%26aulast%3DSteeg%26aufirst%3DP.%2BS.%26aulast%3DStetler-Stevenson%26aufirst%3DW.%2BG.%26atitle%3DCancer%2520metastasis%2520and%2520angiogenesis%253A%2520an%2520imbalance%2520of%2520positive%2520and%2520negative%2520regulation%26jtitle%3DCell%26date%3D1991%26volume%3D64%26spage%3D327%26epage%3D336%26doi%3D10.1016%2F0092-8674%2891%2990642-C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Folkman, J.</span><span> </span><span class="NLM_article-title">Anti-angiogenesis: new concept for therapy of solid tumors</span> <span class="citation_source-journal">Ann. Surg.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">175</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">–</span> <span class="NLM_lpage">416</span><span class="refDoi"> DOI: 10.1097/00000658-197203000-00014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1097%2F00000658-197203000-00014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=5077799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADyaE3s%252FitV2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=1972&pages=409-416&author=J.+Folkman&title=Anti-angiogenesis%3A+new+concept+for+therapy+of+solid+tumors&doi=10.1097%2F00000658-197203000-00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-angiogenesis: new concept for therapy of solid tumors</span></div><div class="casAuthors">Folkman J</div><div class="citationInfo"><span class="NLM_cas:title">Annals of surgery</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">409-16</span>
        ISSN:<span class="NLM_cas:issn">0003-4932</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBThr2iCUqj90Q2ypr7jHkfW6udTcc2eaOB2-xI5VPC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3s%252FitV2gtA%253D%253D&md5=48381e6d0d06f30c9bd7e7d7d8a53d95</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1097%2F00000658-197203000-00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00000658-197203000-00014%26sid%3Dliteratum%253Aachs%26aulast%3DFolkman%26aufirst%3DJ.%26atitle%3DAnti-angiogenesis%253A%2520new%2520concept%2520for%2520therapy%2520of%2520solid%2520tumors%26jtitle%3DAnn.%2520Surg.%26date%3D1972%26volume%3D175%26spage%3D409%26epage%3D416%26doi%3D10.1097%2F00000658-197203000-00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bold, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McSheehy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brüggen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestan, J. r.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drückes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burglin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dürler, U.</span><span> </span><span class="NLM_article-title">A novel potent oral series of VEGFR2 inhibitors abrogate tumor growth by inhibiting angiogenesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01582</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01582" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=132-146&author=G.+Boldauthor=C.+Schnellauthor=P.+Furetauthor=P.+McSheehyauthor=J.+Br%C3%BCggenauthor=J.+r.+Mestanauthor=P.+W.+Manleyauthor=P.+Dr%C3%BCckesauthor=M.+Burglinauthor=U.+D%C3%BCrler&title=A+novel+potent+oral+series+of+VEGFR2+inhibitors+abrogate+tumor+growth+by+inhibiting+angiogenesis&doi=10.1021%2Facs.jmedchem.5b01582"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01582%26sid%3Dliteratum%253Aachs%26aulast%3DBold%26aufirst%3DG.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMcSheehy%26aufirst%3DP.%26aulast%3DBr%25C3%25BCggen%26aufirst%3DJ.%26aulast%3DMestan%26aufirst%3DJ.%2Br.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DDr%25C3%25BCckes%26aufirst%3DP.%26aulast%3DBurglin%26aufirst%3DM.%26aulast%3DD%25C3%25BCrler%26aufirst%3DU.%26atitle%3DA%2520novel%2520potent%2520oral%2520series%2520of%2520VEGFR2%2520inhibitors%2520abrogate%2520tumor%2520growth%2520by%2520inhibiting%2520angiogenesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D132%26epage%3D146%26doi%3D10.1021%2Facs.jmedchem.5b01582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Plate, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weich, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risau, W.</span><span> </span><span class="NLM_article-title">Vascular endothelial growth factor is a potential tumour angiogenesis factor in human gliomas in vivo</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1038/359845a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2F359845a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=1992&pages=845-848&author=K.+H.+Plateauthor=G.+Breierauthor=H.+A.+Weichauthor=W.+Risau&title=Vascular+endothelial+growth+factor+is+a+potential+tumour+angiogenesis+factor+in+human+gliomas+in+vivo&doi=10.1038%2F359845a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F359845a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F359845a0%26sid%3Dliteratum%253Aachs%26aulast%3DPlate%26aufirst%3DK.%2BH.%26aulast%3DBreier%26aufirst%3DG.%26aulast%3DWeich%26aufirst%3DH.%2BA.%26aulast%3DRisau%26aufirst%3DW.%26atitle%3DVascular%2520endothelial%2520growth%2520factor%2520is%2520a%2520potential%2520tumour%2520angiogenesis%2520factor%2520in%2520human%2520gliomas%2520in%2520vivo%26jtitle%3DNature%26date%3D1992%26volume%3D359%26spage%3D845%26epage%3D848%26doi%3D10.1038%2F359845a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Carmeliet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhardti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Declercq, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawling, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moons, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collen, D.</span><span> </span><span class="NLM_article-title">Abnormal blood vessel development and lethality in embryos lacking a single VEGF allele</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">380</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/380435a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2F380435a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=8602241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK28XitVKqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=1996&pages=435-439&author=P.+Carmelietauthor=V.+Ferreiraauthor=C.+Eberhardtiauthor=C.+Declercqauthor=J.+Pawlingauthor=L.+Moonsauthor=D.+Collen&title=Abnormal+blood+vessel+development+and+lethality+in+embryos+lacking+a+single+VEGF+allele&doi=10.1038%2F380435a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Abnormal blood vessel development and lethality in embryos lacking a single VEGF allele</span></div><div class="casAuthors">Carmeliet, Peter; Ferreira, Valerie; Breier, Georg; Pollefeyt, Saskia; Kieckens, Lena; Gertsenstein, Marina; Fahrig, Michaela; Vandenhoeck, Ann; Harpal, Kendraprasad; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">380</span>
        (<span class="NLM_cas:issue">6573</span>),
    <span class="NLM_cas:pages">435-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The endothelial cell-specific vascular endothelial growth factor (VEGF) and its cellular receptors Flt-1 and Flk-1 have been implicated in the formation of the embryonic vasculature.  This is suggested by their colocalized expression during embryogenesis and the impaired vessel formation in Flk-1 and Flt-1 deficient embryos.  However, because Flt-1 also binds placental growth factor, a VEGF homolog, the precise role of VEGF was unknown.  Here we report that formation of blood vessels was abnormal, but not abolished, in heterozygous VEGF-deficient (VEGF+/-) embryos, generated by aggregation of embryonic stem (ES) cells with tetraploid embryos (T-ES), and even more impaired in homozygous VEGF-deficient (VEGF-/-) T-ES embryos, resulting in death at mid-gestation.  Similar phenotypes were obsd. in F1-VEGF+/- embryos, generated by germline transmission.  We believe that this heterozygous lethal phenotype, which differs from the homozygous lethality in VEGF-receptor-deficient embryos, is unprecedented for a targeted autosomal gene inactivation, and is indicative of a tight dose-dependent regulation of embryonic vessel development by VEGF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobatChefZSx7Vg90H21EOLACvtfcHk0lg-n3HPIltZAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XitVKqtLg%253D&md5=93d159aabf470be75c118dad37a729cb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F380435a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F380435a0%26sid%3Dliteratum%253Aachs%26aulast%3DCarmeliet%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DV.%26aulast%3DEberhardti%26aufirst%3DC.%26aulast%3DDeclercq%26aufirst%3DC.%26aulast%3DPawling%26aufirst%3DJ.%26aulast%3DMoons%26aufirst%3DL.%26aulast%3DCollen%26aufirst%3DD.%26atitle%3DAbnormal%2520blood%2520vessel%2520development%2520and%2520lethality%2520in%2520embryos%2520lacking%2520a%2520single%2520VEGF%2520allele%26jtitle%3DNature%26date%3D1996%26volume%3D380%26spage%3D435%26epage%3D439%26doi%3D10.1038%2F380435a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span> </span><span class="NLM_article-title">Development and strategies of VEGFR-2/KDR inhibitors</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1839</span><span class="NLM_x">–</span> <span class="NLM_lpage">1852</span><span class="refDoi"> DOI: 10.4155/fmc.12.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.4155%2Ffmc.12.121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=23043480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVOhsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1839-1852&author=L.+Huangauthor=Z.+Huangauthor=Z.+Baiauthor=R.+Xieauthor=L.+Sunauthor=K.+Lin&title=Development+and+strategies+of+VEGFR-2%2FKDR+inhibitors&doi=10.4155%2Ffmc.12.121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Development and strategies of VEGFR-2/KDR inhibitors</span></div><div class="casAuthors">Huang, Lingyi; Huang, Zhengui; Bai, Zhiqiang; Xie, Rui; Sun, Liping; Lin, Kejiang</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1839-1852</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  VEGF is an important signaling protein involved in both vasculogenesis and angiogenesis.  As an essential receptor protein tyrosine kinase propagating cellular signal transduction processes, VEGFR-2 is a central target for drug discovery against tumor-assocd. angiogenesis.  Since the autophosphorylation of VEGFR-2 represents a key step in this signal pathway that contributes to angiogenesis, the discovery of small mol. inhibitors that block this reaction has attracted great interest for novel drugs research and development.  Advances in the understanding of catalytic cleft and the conformational changes of DFG motif have resulted in the development of small mol. inhibitors known as type I and type II.  High-resoln. crystal structures of various inhibitors in complex with the receptor offer an insight into the relationship among binding modes, inhibition mechanisms, activity, selectivity and resistance.  To control selectivity, improve activity and introduce intellectual property novelty, the strategies for the further development are discussed through structural and conformational anal. in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMgCdblpbQc7Vg90H21EOLACvtfcHk0lg-n3HPIltZAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVOhsb3O&md5=96cc54622cc92d45a8549eb88d551a7f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.121%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DK.%26atitle%3DDevelopment%2520and%2520strategies%2520of%2520VEGFR-2%252FKDR%2520inhibitors%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1839%26epage%3D1852%26doi%3D10.4155%2Ffmc.12.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Boyer, S. J.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of KDR (VEGFR-2) kinase: an overview of structure activity relationships</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">973</span><span class="NLM_x">–</span> <span class="NLM_lpage">1000</span><span class="refDoi"> DOI: 10.2174/1568026023393273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.2174%2F1568026023393273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=12171566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtVOitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=973-1000&author=S.+J.+Boyer&title=Small+molecule+inhibitors+of+KDR+%28VEGFR-2%29+kinase%3A+an+overview+of+structure+activity+relationships&doi=10.2174%2F1568026023393273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of KDR (VEGFR-2) kinase: an overview of structure activity relationships</span></div><div class="casAuthors">Boyer, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Hilversum, Netherlands)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">973-1000</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Kinase insert Domain contg. Receptor (KDR), alternatively referred to as VEGFR-2, is a receptor for Vascular Endothelial Growth Factors (VEGFs) and functions as a key regulator of angiogenesis, the process by which new capillaries are created from preexisting blood vessels.  The induction of angiogenesis, or the "angiogenic switch," is a crit. step in tumor progression, and inhibitors of KDR have been demonstrated both to induce tumor regression and reduce metastatic potential in preclin. models.  In the last few years, medicinal chemists have expanded the kinase selectivity profile of known inhibitor classes to include KDR, and also identified novel classes of KDR inhibitors.  This review presents structure activity relationships (SAR) of small mol. inhibitors of KDR, with an emphasis on the pharmacophore elements of the scaffolds employed.  Binding hypotheses based on x-ray crystallog. analyses will also be described.  Addnl., the efficacy of representative compds. in in vitro and in vivo models of tumor progression and angiogenesis are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8iX5xdLgktLVg90H21EOLACvtfcHk0ljyujQmNuO61Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtVOitbw%253D&md5=4c4445eac589bded1baed3140b918be8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2174%2F1568026023393273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026023393273%26sid%3Dliteratum%253Aachs%26aulast%3DBoyer%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520KDR%2520%2528VEGFR-2%2529%2520kinase%253A%2520an%2520overview%2520of%2520structure%2520activity%2520relationships%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2002%26volume%3D2%26spage%3D973%26epage%3D1000%26doi%3D10.2174%2F1568026023393273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0ljyujQmNuO61Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span> </span><span class="NLM_article-title">An integrated virtual screening approach for VEGFR-2 inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3163</span><span class="NLM_x">–</span> <span class="NLM_lpage">3177</span><span class="refDoi"> DOI: 10.1021/ci400429g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400429g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVajtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=3163-3177&author=Y.+Zhangauthor=S.+Yangauthor=Y.+Jiaoauthor=H.+Liuauthor=H.+Yuanauthor=S.+Luauthor=T.+Ranauthor=S.+Yaoauthor=Z.+Keauthor=J.+Xuauthor=X.+Xiongauthor=Y.+Chenauthor=T.+Lu&title=An+integrated+virtual+screening+approach+for+VEGFR-2+inhibitors&doi=10.1021%2Fci400429g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Integrated Virtual Screening Approach for VEGFR-2 Inhibitors</span></div><div class="casAuthors">Zhang, Yanmin; Yang, Shangyan; Jiao, Yu; Liu, Haichun; Yuan, Haoliang; Lu, Shuai; Ran, Ting; Yao, Sihui; Ke, Zhipeng; Xu, Jinxing; Xiong, Xiao; Chen, Yadong; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3163-3177</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, various virtual screening (VS) tools have been developed, and many successful screening campaigns have been showcased.  However, whether by conventional mol. docking or pharmacophore screening, the selection of virtual hits is based on the ranking of compds. by scoring functions or fit values, which remains the bottleneck of VS due to insufficient accuracy.  As the limitations of individual methods persist, a comprehensive comparison and integration of different methods may provide insights into selecting suitable methods for VS.  Here, we evaluated the performance of mol. docking, fingerprint-based 2D similarity and multicomplex pharmacophore in an individual and a combined manner, through a retrospective VS study on VEGFR-2 inhibitors.  An integrated two-layer work-flow was developed and validated through VS of VEGFR-2 inhibitors against the DUD-E database, which demonstrated improved VS performance through a ligand-based method ECFP_4, followed by mol. docking, and then a strict multicomplex pharmacophore.  Through a retrospective comparison with six published papers, this integrated approach outperformed 43 out of 45 methods, indicating a great effectiveness.  This kind of integrated VS approach can be extended to other targets for the screening and discovery of inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl0yCcaTmfp7Vg90H21EOLACvtfcHk0ljyujQmNuO61Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVajtb%252FE&md5=5eac47fb02a20e95d900bb7eae65f54d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci400429g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400429g%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DKe%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DAn%2520integrated%2520virtual%2520screening%2520approach%2520for%2520VEGFR-2%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D3163%26epage%3D3177%26doi%3D10.1021%2Fci400429g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ellis, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hicklin, D. J.</span><span> </span><span class="NLM_article-title">Pathways mediating resistance to vascular endothelial growth factor–targeted therapy</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">6371</span><span class="NLM_x">–</span> <span class="NLM_lpage">6375</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-5287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1158%2F1078-0432.CCR-07-5287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=18927275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1KgtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=6371-6375&author=L.+M.+Ellisauthor=D.+J.+Hicklin&title=Pathways+mediating+resistance+to+vascular+endothelial+growth+factor%E2%80%93targeted+therapy&doi=10.1158%2F1078-0432.CCR-07-5287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pathways Mediating Resistance to Vascular Endothelial Growth Factor-Targeted Therapy</span></div><div class="casAuthors">Ellis, Lee M.; Hicklin, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6371-6375</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Vascular endothelial growth factor (VEGF)-targeted therapy has become an important treatment option for the management of a no. of human malignancies.  Unfortunately, a significant no. of patients do not respond to VEGF-targeted therapy when used as a single agent or in combination with chemotherapy.  Furthermore, the duration of benefit from VEGF-targeted therapy can be relatively short (weeks to months).  Ultimately, the vast majority of patients who initially respond to therapy will develop resistance.  To date, the mol. and cellular mechanisms assocd. with resistance to VEGF-targeted agents are poorly understood.  The mechanisms of action of anti-VEGF therapy are diverse, and it is entirely possible that resistance mechanisms are similarly diverse and depend on the tumor type.  A better understanding of these mechanisms will help in the selection of those patients that are more likely to benefit from VEGF-targeted therapy and also provide for the rational development of therapies that circumvent or overcome resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof9jwBsqNaErVg90H21EOLACvtfcHk0ljyujQmNuO61Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1KgtbrJ&md5=5b5a06a0aa43bec60a2644a67a262820</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-5287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-5287%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DL.%2BM.%26aulast%3DHicklin%26aufirst%3DD.%2BJ.%26atitle%3DPathways%2520mediating%2520resistance%2520to%2520vascular%2520endothelial%2520growth%2520factor%25E2%2580%2593targeted%2520therapy%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D6371%26epage%3D6375%26doi%3D10.1158%2F1078-0432.CCR-07-5287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Jayson, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerbel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, A. L.</span><span> </span><span class="NLM_article-title">Antiangiogenic therapy in oncology: current status and future directions</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">388</span><span class="NLM_x">, </span> <span class="NLM_fpage">518</span><span class="NLM_x">–</span> <span class="NLM_lpage">529</span><span class="refDoi"> DOI: 10.1016/S0140-6736(15)01088-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2FS0140-6736%2815%2901088-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=26853587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12rur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2016&pages=518-529&author=G.+C.+Jaysonauthor=R.+Kerbelauthor=L.+M.+Ellisauthor=A.+L.+Harris&title=Antiangiogenic+therapy+in+oncology%3A+current+status+and+future+directions&doi=10.1016%2FS0140-6736%2815%2901088-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Antiangiogenic therapy in oncology: current status and future directions</span></div><div class="casAuthors">Jayson, Gordon C.; Kerbel, Robert; Ellis, Lee M.; Harris, Adrian L.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">10043</span>),
    <span class="NLM_cas:pages">518-529</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Angiogenesis, the formation of new blood vessels from pre-existing vessels, has been validated as a target in several tumor types through randomized trials, incorporating vascular endothelial growth factor (VEGF) pathway inhibitors into the therapeutic armory.  Although some tumors such as renal cell carcinoma, ovarian and cervical cancers, and pancreatic neuroendocrine tumors are sensitive to these drugs, others such as prostate cancer, pancreatic adenocarcinoma, and melanoma are resistant.  Even when drugs have yielded significant results, improvements in progression-free survival, and, in some cases, overall survival, are modest.  Thus, a crucial issue in development of these drugs is the search for predictive biomarkers-tests that predict which patients will, and will not, benefit before initiation of therapy.  Development of biomarkers is important because of the need to balance efficacy, toxicity, and cost.  Novel combinations of these drugs with other antiangiogenics or other classes of drugs are being developed, and the appreciation that these drugs have immunomodulatory and other modes of action will lead to combination regimens that capitalize on these newly understood mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDgPkePJIM_bVg90H21EOLACvtfcHk0liE0VG3S9nWXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12rur0%253D&md5=7cca25ce3a4ded56b11a9571c050ce7f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2901088-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252901088-0%26sid%3Dliteratum%253Aachs%26aulast%3DJayson%26aufirst%3DG.%2BC.%26aulast%3DKerbel%26aufirst%3DR.%26aulast%3DEllis%26aufirst%3DL.%2BM.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26atitle%3DAntiangiogenic%2520therapy%2520in%2520oncology%253A%2520current%2520status%2520and%2520future%2520directions%26jtitle%3DLancet%26date%3D2016%26volume%3D388%26spage%3D518%26epage%3D529%26doi%3D10.1016%2FS0140-6736%2815%2901088-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Kuczynski, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bar-Zion, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butz, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Man, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daley, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yousef, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, F. S.</span><span> </span><span class="NLM_article-title">Co-option of liver vessels and not sprouting angiogenesis drives acquired sorafenib resistance in hepatocellular carcinoma</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1093/jnci/djw030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1093%2Fjnci%2Fdjw030" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2016&pages=1-13&author=E.+A.+Kuczynskiauthor=M.+Yinauthor=A.+Bar-Zionauthor=C.+R.+Leeauthor=H.+Butzauthor=S.+Manauthor=F.+Daleyauthor=P.+B.+Vermeulenauthor=G.+M.+Yousefauthor=F.+S.+Foster&title=Co-option+of+liver+vessels+and+not+sprouting+angiogenesis+drives+acquired+sorafenib+resistance+in+hepatocellular+carcinoma&doi=10.1093%2Fjnci%2Fdjw030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjw030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjw030%26sid%3Dliteratum%253Aachs%26aulast%3DKuczynski%26aufirst%3DE.%2BA.%26aulast%3DYin%26aufirst%3DM.%26aulast%3DBar-Zion%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.%2BR.%26aulast%3DButz%26aufirst%3DH.%26aulast%3DMan%26aufirst%3DS.%26aulast%3DDaley%26aufirst%3DF.%26aulast%3DVermeulen%26aufirst%3DP.%2BB.%26aulast%3DYousef%26aufirst%3DG.%2BM.%26aulast%3DFoster%26aufirst%3DF.%2BS.%26atitle%3DCo-option%2520of%2520liver%2520vessels%2520and%2520not%2520sprouting%2520angiogenesis%2520drives%2520acquired%2520sorafenib%2520resistance%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2016%26volume%3D108%26spage%3D1%26epage%3D13%26doi%3D10.1093%2Fjnci%2Fdjw030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Grothey, A.</span><span> </span><span class="NLM_article-title">VEGF inhibition beyond tumour progression</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">2</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1016/S1470-2045(12)70516-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2FS1470-2045%2812%2970516-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=23168363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADC%252BC3s7ltVyqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=2-3&author=A.+Grothey&title=VEGF+inhibition+beyond+tumour+progression&doi=10.1016%2FS1470-2045%2812%2970516-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">VEGF inhibition beyond tumour progression</span></div><div class="casAuthors">Grothey Axel</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSL0tZAs9R_3DAEfvC7EH1LfW6udTcc2eaMPXaq0_kWAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7ltVyqug%253D%253D&md5=b54521cc459fe4366d3d3aa8cadfa315</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970516-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970516-8%26sid%3Dliteratum%253Aachs%26aulast%3DGrothey%26aufirst%3DA.%26atitle%3DVEGF%2520inhibition%2520beyond%2520tumour%2520progression%26jtitle%3DLancet%2520Oncol.%26date%3D2013%26volume%3D14%26spage%3D2%26epage%3D3%26doi%3D10.1016%2FS1470-2045%2812%2970516-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">von Minckwitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puglisi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vrdoljak, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marschner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zielinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villanueva, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romieu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciruelos, E.</span><span> </span><span class="NLM_article-title">Bevacizumab plus chemotherapy versus chemotherapy alone as second-line treatment for patients with HER2-negative locally recurrent or metastatic breast cancer after first-line treatment with bevacizumab plus chemotherapy (TANIA): an open-label, randomised phase 3 trial</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1269</span><span class="NLM_x">–</span> <span class="NLM_lpage">1278</span><span class="refDoi"> DOI: 10.1016/S1470-2045(14)70439-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2FS1470-2045%2814%2970439-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=25273342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1269-1278&author=G.+von+Minckwitzauthor=F.+Puglisiauthor=J.+Cortesauthor=E.+Vrdoljakauthor=N.+Marschnerauthor=C.+Zielinskiauthor=C.+Villanuevaauthor=G.+Romieuauthor=I.+Langauthor=E.+Ciruelos&title=Bevacizumab+plus+chemotherapy+versus+chemotherapy+alone+as+second-line+treatment+for+patients+with+HER2-negative+locally+recurrent+or+metastatic+breast+cancer+after+first-line+treatment+with+bevacizumab+plus+chemotherapy+%28TANIA%29%3A+an+open-label%2C+randomised+phase+3+trial&doi=10.1016%2FS1470-2045%2814%2970439-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Bevacizumab plus chemotherapy versus chemotherapy alone as second-line treatment for patients with HER2-negative locally recurrent or metastatic breast cancer after first-line treatment with bevacizumab plus chemotherapy (TANIA): an open-label, randomized phase 3 trial</span></div><div class="casAuthors">von Minckwitz, Gunter; Puglisi, Fabio; Cortes, Javier; Vrdoljak, Eduard; Marschner, Norbert; Zielinski, Christoph; Villanueva, Cristian; Romieu, Gilles; Lang, Istvan; Ciruelos, Eva; De Laurentiis, Michele; Veyret, Corinne; de Duda, Sabine; Freudensprung, Ulrich; Srock, Stefanie; Gligorov, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1269-1278</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Combining bevacizumab with first-line or second-line chemotherapy improves progression-free survival in HER2-neg. locally recurrent or metastatic breast cancer.  We assessed the efficacy and safety of further bevacizumab therapy in patients with locally recurrent or metastatic breast cancer whose disease had progressed after treatment with bevacizumab plus chemotherapy.  Methods: In this open-label, randomized, phase 3 trial, we recruited patients who had HER2-neg. locally recurrent or metastatic breast cancer that had progressed after receiving 12 wk or more of first-line bevacizumab plus chemotherapy from 118 centers in 12 countries.  Patients were randomly assigned (1:1) by use of a central interactive voice response system using a block randomization schedule (block size four) stratified by hormone receptor status, first-line progression-free survival, selected chemotherapy, and lactate dehydrogenase concn., to receive second-line single-agent chemotherapy either alone or with bevacizumab (15 mg/kg every 3 wk or 10 mg/kg every 2 wk).  Second-line therapy was continued until disease progression, unacceptable toxicity, or consent withdrawal.  At progression, patients randomly assigned to chemotherapy alone received third-line chemotherapy without bevacizumab; those randomly assigned to bevacizumab continued bevacizumab with third-line chemotherapy.  The primary endpoint was progression-free survival from randomization to second-line progression or death in the intention-to-treat population.  This trial is ongoing, and registered with ClinicalTrials.gov, no. NCT01250379.  Findings: Between Feb 17, 2011, and Apr. 3, 2013, 494 patients were randomly assigned to treatment (247 in each group).  The median duration of follow-up at the time of this prespecified primary progression-free survival anal. was 15·9 mo (IQR 9·1-21·7) in the chemotherapy-alone group and 16·1 mo (10·6-22·7) in the combination group.  Progression-free survival was significantly longer for those patients treated with bevacizumab plus chemotherapy than for those with chemotherapy alone (median: 6·3 mo [95% CI 5·4-7·2] vs 4·2 mo [3·9-4·7], resp., stratified hazard ratio [HR] 0·75 [95% CI 0·61-0·93], two-sided stratified log-rank p=0·0068).  The most common grade 3 or more adverse events were hypertension (33 [13%] of 245 patients receiving bevacizumab plus chemotherapy vs 17 [7%] of 238 patients receiving chemotherapy alone), neutropenia (29 [12%] vs 20 [8%]), and hand-foot syndrome (27 [11%] vs 25 [11%]).  Grade 3 proteinuria occurred in 17 (7%) of 245 patients receiving combination therapy and one (<1%) of 238 patients receiving chemotherapy alone.  Serious adverse events were reported in 61 (25%) of 245 patients receiving bevacizumab plus chemotherapy vs. 44 (18%) of 238 patients receiving chemotherapy alone.  Interpretation: These results suggest that continued VEGF inhibition with further bevacizumab is a valid treatment option for patients with locally recurrent or metastatic HER2-neg. breast cancer whose disease was stabilized or responded to first-line bevacizumab with chemotherapy.  Funding: F Hoffmann-La Roche.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbwFFFDp-M1rVg90H21EOLACvtfcHk0liIUw3gmRd6_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt73M&md5=cfcede8b1ef1bf73999c52adc26684e7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2814%2970439-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252814%252970439-5%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BMinckwitz%26aufirst%3DG.%26aulast%3DPuglisi%26aufirst%3DF.%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DVrdoljak%26aufirst%3DE.%26aulast%3DMarschner%26aufirst%3DN.%26aulast%3DZielinski%26aufirst%3DC.%26aulast%3DVillanueva%26aufirst%3DC.%26aulast%3DRomieu%26aufirst%3DG.%26aulast%3DLang%26aufirst%3DI.%26aulast%3DCiruelos%26aufirst%3DE.%26atitle%3DBevacizumab%2520plus%2520chemotherapy%2520versus%2520chemotherapy%2520alone%2520as%2520second-line%2520treatment%2520for%2520patients%2520with%2520HER2-negative%2520locally%2520recurrent%2520or%2520metastatic%2520breast%2520cancer%2520after%2520first-line%2520treatment%2520with%2520bevacizumab%2520plus%2520chemotherapy%2520%2528TANIA%2529%253A%2520an%2520open-label%252C%2520randomised%2520phase%25203%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2014%26volume%3D15%26spage%3D1269%26epage%3D1278%26doi%3D10.1016%2FS1470-2045%2814%2970439-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Escudier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szczylik, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gore, M.</span><span> </span><span class="NLM_article-title">Treatment selection in metastatic renal cell carcinoma: expert consensus</span> <span class="citation_source-journal">Nat. Rev. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1038/nrclinonc.2012.59</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnrclinonc.2012.59" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=22473096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVOrtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=327-337&author=B.+Escudierauthor=C.+Szczylikauthor=C.+Portaauthor=M.+Gore&title=Treatment+selection+in+metastatic+renal+cell+carcinoma%3A+expert+consensus&doi=10.1038%2Fnrclinonc.2012.59"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment selection in metastatic renal cell carcinoma: expert consensus</span></div><div class="casAuthors">Escudier, Bernard; Szczylik, Cezary; Porta, Camillo; Gore, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">327-337</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The advances in the field of targeted therapy for the treatment of renal cell carcinoma have improved outcomes for patients dramatically.  In this Review article, an expert consensus opinion for the optimal use of these targeted anticancer therapies for the treatment of patients with renal cell carcinoma is outlined.  In metastatic renal cell carcinoma (mRCC), many factors influence clin. decisions, including histol., tumor burden, prognostic factors, comorbidities, and the ability of the patient to tolerate treatment.  Progression-free survival (PFS) durations reported from randomized trials of targeted therapies vary considerably, in part because of differences in patient characteristics.  For first-line therapy, an est. of PFS with sunitinib, bevacizumab plus interferon, or sorafenib in a 'general' population is 8-9 mo, but each regimen is suitable for different patient categories.  For example, sunitinib is suitable for all-prognosis groups, particularly younger, fitter patients; pazopanib for patients with a good or intermediate prognosis; bevacizumab plus interferon for good-prognosis patients or those with indolent disease; and sorafenib for patients at all prognostic risk levels, particularly the elderly and those with comorbidities.  Sequential therapy with targeted agents provides significant benefit, and should be considered in all patients who can tolerate such treatment.  Level 1 evidence supports sequential use of tyrosine kinase inhibitors, as well as these agents followed by everolimus.  We consider how patient characteristics have influenced the results of studies of first-line therapy, and we provide expert opinion on the most appropriate treatment choices for particular patient groups receiving first-line and second-line therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxiub93j40UrVg90H21EOLACvtfcHk0liIUw3gmRd6_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvVOrtLo%253D&md5=b7b16fa16ed824583278ba376cb216e9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2012.59&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2012.59%26sid%3Dliteratum%253Aachs%26aulast%3DEscudier%26aufirst%3DB.%26aulast%3DSzczylik%26aufirst%3DC.%26aulast%3DPorta%26aufirst%3DC.%26aulast%3DGore%26aufirst%3DM.%26atitle%3DTreatment%2520selection%2520in%2520metastatic%2520renal%2520cell%2520carcinoma%253A%2520expert%2520consensus%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2012%26volume%3D9%26spage%3D327%26epage%3D337%26doi%3D10.1038%2Fnrclinonc.2012.59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0liIUw3gmRd6_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Barouch-Bentov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, K.</span><span> </span><span class="NLM_article-title">Mechanisms of drug resistance in kinases</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span><span class="refDoi"> DOI: 10.1517/13543784.2011.546344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1517%2F13543784.2011.546344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=21235428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=153-208&author=R.+Barouch-Bentovauthor=K.+Sauer&title=Mechanisms+of+drug+resistance+in+kinases&doi=10.1517%2F13543784.2011.546344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of drug resistance in kinases</span></div><div class="casAuthors">Barouch-Bentov, Rina; Sauer, Karsten</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">153-208</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Because of their important roles in disease and excellent "druggability", kinases have become the second largest drug target family.  The great success of the BCR-ABL inhibitor imatinib in treating chronic myelogenous leukemia illustrates the high potential of kinase inhibitor (KI) therapeutics, but also unveils a major limitation: the development of drug resistance.  This is a significant concern as KIs reach large patient populations for an expanding array of indications.  We provide an up-to-date understanding of the mechanisms through which KIs function and through which cells can become KI-resistant.  We review current and future approaches to overcome KI resistance, focusing on currently approved KIs and KIs in clin. trials.  We then discuss approaches to improve KI efficacy and overcome drug resistance and novel approaches to develop less drug resistance-prone KI therapeutics.  Although drug resistance is a concern for current KI therapeutics, recent progress in our understanding of the underlying mechanisms and promising technol. advances may overcome this limitation and provide powerful new therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprJUpdpkWU6bVg90H21EOLACvtfcHk0ljii8ho7h_lqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D&md5=cccffaca08cbcfb1e0d2c1d580c9e797</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1517%2F13543784.2011.546344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2011.546344%26sid%3Dliteratum%253Aachs%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DSauer%26aufirst%3DK.%26atitle%3DMechanisms%2520of%2520drug%2520resistance%2520in%2520kinases%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2011%26volume%3D20%26spage%3D153%26epage%3D208%26doi%3D10.1517%2F13543784.2011.546344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">BREED: generating novel inhibitors through hybridization of known ligands. application to CDK2, p38, and HIV protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2768</span><span class="NLM_x">–</span> <span class="NLM_lpage">2775</span><span class="refDoi"> DOI: 10.1021/jm030543u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030543u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2768-2775&author=A.+C.+Pierceauthor=G.+Raoauthor=G.+W.+Bemis&title=BREED%3A+generating+novel+inhibitors+through+hybridization+of+known+ligands.+application+to+CDK2%2C+p38%2C+and+HIV+protease&doi=10.1021%2Fjm030543u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BREED: Generating Novel Inhibitors through Hybridization of Known Ligands. Application to CDK2, P38, and HIV Protease</span></div><div class="casAuthors">Pierce, Albert C.; Rao, Govinda; Bemis, Guy W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2768-2775</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this work we describe BREED, a method for the generation of novel inhibitors from structures of known ligands bound to a common target.  The method is essentially an automation of the common medicinal chem. practice of joining fragments of two known ligands to generate a new inhibitor.  The ligand-bound target structures are overlaid, all overlapping bonds in all pairs of ligands are found, and the fragments on each side of each matching bond are swapped to generate the new mols.  Since the method is automated, it can be applied recursively to generate all possible combinations of known ligands.  In an application of this method to HIV protease inhibitors and protein kinase inhibitors, hundreds of new mol. structures were generated.  These included known inhibitor scaffolds not included in the initial set, entirely novel scaffolds, and novel substituents on known scaffolds.  The method is fast, and since all of the ligand functional groups are known to bind the target in the precise position and orientation present in the novel ligand, the success rate of this method should be superior to more traditional de novo design techniques.  In an era of increasingly high-throughput structural biol., such methods for high-throughput utilization of structural information will become increasingly valuable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqskNA5h73CbbVg90H21EOLACvtfcHk0ljii8ho7h_lqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D&md5=e0bec9082fab05b694742498b3f41549</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm030543u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030543u%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DRao%26aufirst%3DG.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DBREED%253A%2520generating%2520novel%2520inhibitors%2520through%2520hybridization%2520of%2520known%2520ligands.%2520application%2520to%2520CDK2%252C%2520p38%252C%2520and%2520HIV%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2768%26epage%3D2775%26doi%3D10.1021%2Fjm030543u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a knowledge-based structural database to navigate kinase–ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+knowledge-based+structural+database+to+navigate+kinase%E2%80%93ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0ljii8ho7h_lqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase%25E2%2580%2593ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span> </span><span class="NLM_article-title">De novo design of N-(pyridin-4-ylmethyl) aniline derivatives as KDR inhibitors: 3D-QSAR, molecular fragment replacement, protein-ligand interaction fingerprint, and ADMET prediction</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">787</span><span class="NLM_x">–</span> <span class="NLM_lpage">802</span><span class="refDoi"> DOI: 10.1007/s11030-012-9405-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2Fs11030-012-9405-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=787-802&author=Y.+Zhangauthor=H.+Liuauthor=Y.+Jiaoauthor=H.+Yuanauthor=F.+Wangauthor=S.+Luauthor=S.+Yaoauthor=Z.+Keauthor=W.+Taiauthor=Y.+Jiang&title=De+novo+design+of+N-%28pyridin-4-ylmethyl%29+aniline+derivatives+as+KDR+inhibitors%3A+3D-QSAR%2C+molecular+fragment+replacement%2C+protein-ligand+interaction+fingerprint%2C+and+ADMET+prediction&doi=10.1007%2Fs11030-012-9405-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs11030-012-9405-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-012-9405-y%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DKe%26aufirst%3DZ.%26aulast%3DTai%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DDe%2520novo%2520design%2520of%2520N-%2528pyridin-4-ylmethyl%2529%2520aniline%2520derivatives%2520as%2520KDR%2520inhibitors%253A%25203D-QSAR%252C%2520molecular%2520fragment%2520replacement%252C%2520protein-ligand%2520interaction%2520fingerprint%252C%2520and%2520ADMET%2520prediction%26jtitle%3DMol.%2520Diversity%26date%3D2012%26volume%3D16%26spage%3D787%26epage%3D802%26doi%3D10.1007%2Fs11030-012-9405-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, L.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability prediction of protein targets for structure-based drug design</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2326</span><span class="NLM_x">–</span> <span class="NLM_lpage">2333</span><span class="refDoi"> DOI: 10.2174/1381612811319120019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.2174%2F1381612811319120019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=23082974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2326-2333&author=Y.+Yuanauthor=J.+Peiauthor=L.+Lai&title=Binding+site+detection+and+druggability+prediction+of+protein+targets+for+structure-based+drug+design&doi=10.2174%2F1381612811319120019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Binding site detection and druggability prediction of protein targets for structure-based drug design</span></div><div class="casAuthors">Yuan, Yaxia; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2326-2333</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Assessing whether a protein structure is a good target or not before actually doing structure-based drug design on it is an important step to speed up the ligand discovery process.  This is known as the "druggability" or "ligandability" assessment problem that has attracted increasing interest in recent years.  The assessment typically includes the detection of ligand-binding sites on the protein surface and the prediction of their abilities to bind drug-like small mols.  A brief summary of the established methods of binding sites detection and druggability(ligandability) prediction, as well as a detailed description of the CAVITY approach developed in the authors' group was given.  CAVITY showed good performance on ligand-binding site detection, and was successfully used to predict both the ligandabilities and druggabilities of the detected binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9UZbLJWt2l7Vg90H21EOLACvtfcHk0lgdpbw6x9VOvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtbc%253D&md5=9d6105927b7c36665477b446c54e7cca</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.2174%2F1381612811319120019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319120019%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520prediction%2520of%2520protein%2520targets%2520for%2520structure-based%2520drug%2520design%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D2326%26epage%3D2333%26doi%3D10.2174%2F1381612811319120019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Ngan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">FTMAP: extended protein mapping with user-selected probe molecules</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1093/nar/gks441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1093%2Fnar%2Fgks441" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=271-275&author=C.+H.+Nganauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=D.+Beglovauthor=E.+A.+Villarauthor=D.+R.+Hallauthor=D.+Kozakovauthor=S.+Vajda&title=FTMAP%3A+extended+protein+mapping+with+user-selected+probe+molecules&doi=10.1093%2Fnar%2Fgks441"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks441%26sid%3Dliteratum%253Aachs%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFTMAP%253A%2520extended%2520protein%2520mapping%2520with%2520user-selected%2520probe%2520molecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D271%26epage%3D275%26doi%3D10.1093%2Fnar%2Fgks441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, X.</span><span> </span><span class="NLM_article-title">Identification of covalent binding sites targeting cysteines based on computational approaches</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3106</span><span class="NLM_x">–</span> <span class="NLM_lpage">3118</span><span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.6b00302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.6b00302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KmtrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=3106-3118&author=Y.+Zhangauthor=D.+Zhangauthor=H.+Tianauthor=Y.+Jiaoauthor=Z.+Shiauthor=T.+Ranauthor=H.+Liuauthor=S.+Luauthor=A.+Xuauthor=X.+Qiao&title=Identification+of+covalent+binding+sites+targeting+cysteines+based+on+computational+approaches&doi=10.1021%2Facs.molpharmaceut.6b00302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Covalent Binding Sites Targeting Cysteines Based on Computational Approaches</span></div><div class="casAuthors">Zhang, Yanmin; Zhang, Danfeng; Tian, Haozhong; Jiao, Yu; Shi, Zhihao; Ran, Ting; Liu, Haichun; Lu, Shuai; Xu, Anyang; Qiao, Xin; Pan, Jing; Yin, Lingfeng; Zhou, Weineng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3106-3118</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalent drugs have attracted increasing attention in recent years due to good inhibitory activity and selectivity.  Targeting noncatalytic cysteines with irreversible inhibitors is a powerful approach for enhancing pharmacol. potency and selectivity because cysteines can form covalent bonds with inhibitors through their nucleophilic thiol groups.  However, most human kinases have multiple noncatalytic cysteines within the active site; to accurately predict which cysteine is most likely to form covalent bonds is of great importance but remains a challenge when designing irreversible inhibitors.  FTMap was first applied to check its ability in predicting covalent binding site defined as the region where covalent bonds are formed between cysteines and irreversible inhibitors.  Results show that it has excellent performance in detecting the hot spots within the binding pocket, and its hydrogen bond interaction frequency anal. could give the authors some interesting instructions for identification of covalent binding cysteines.  Furthermore, the authors proposed a simple but useful covalent fragment probing approach and showed that it successfully predicted the covalent binding site of seven targets.  By adopting a distance-based method, the authors obsd. that the closer the nucleophiles of covalent warheads are to the thiol group of a cysteine, the higher the possibility that a cysteine is prone to form a covalent bond.  The authors believe that the combination of FTMap and the authors' distance-based covalent fragment probing method can become a useful tool in detecting the covalent binding site of these targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJNLjHdOWupLVg90H21EOLACvtfcHk0lgdpbw6x9VOvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KmtrbM&md5=ce487167befaeaa8c88ef1ee8dabc863</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.6b00302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.6b00302%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DA.%26aulast%3DQiao%26aufirst%3DX.%26atitle%3DIdentification%2520of%2520covalent%2520binding%2520sites%2520targeting%2520cysteines%2520based%2520on%2520computational%2520approaches%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2016%26volume%3D13%26spage%3D3106%26epage%3D3118%26doi%3D10.1021%2Facs.molpharmaceut.6b00302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, X.</span><span> </span><span class="NLM_article-title">Fragment virtual screening based on bayesian categorization for discovering novel VEGFR-2 scaffolds</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">895</span><span class="NLM_x">–</span> <span class="NLM_lpage">913</span><span class="refDoi"> DOI: 10.1007/s11030-015-9592-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2Fs11030-015-9592-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=26022686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1aqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=895-913&author=Y.+Zhangauthor=Y.+Jiaoauthor=X.+Xiongauthor=H.+Liuauthor=T.+Ranauthor=J.+Xuauthor=S.+Luauthor=A.+Xuauthor=J.+Panauthor=X.+Qiao&title=Fragment+virtual+screening+based+on+bayesian+categorization+for+discovering+novel+VEGFR-2+scaffolds&doi=10.1007%2Fs11030-015-9592-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment virtual screening based on Bayesian categorization for discovering novel VEGFR-2 scaffolds</span></div><div class="casAuthors">Zhang, Yanmin; Jiao, Yu; Xiong, Xiao; Liu, Haichun; Ran, Ting; Xu, Jinxing; Lu, Shuai; Xu, Anyang; Pan, Jing; Qiao, Xin; Shi, Zhihao; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">895-913</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The discovery of novel scaffolds against a specific target has long been one of the most significant but challengeable goals in discovering lead compds.  A scaffold that binds in important regions of the active pocket is more favorable as a starting point because scaffolds generally possess greater optimization possibilities.  However, due to the lack of sufficient chem. space diversity of the databases and the ineffectiveness of the screening methods, it still remains a great challenge to discover novel active scaffolds.  Since the strengths and weaknesses of both fragment-based drug design and traditional virtual screening (VS), we proposed a fragment VS concept based on Bayesian categorization for the discovery of novel scaffolds.  This work investigated the proposal through an application on VEGFR-2 target.  Firstly, scaffold and structural diversity of chem. space for 10 compd. databases were explicitly evaluated.  Simultaneously, a robust Bayesian classification model was constructed for screening not only compd. databases but also their corresponding fragment databases.  Although anal. of the scaffold diversity demonstrated a very unevenly distribution of scaffolds over mols., results showed that our Bayesian model behaved better in screening fragments than mols.  Through a literature retrospective research, several generated fragments with relatively high Bayesian scores indeed exhibit VEGFR-2 biol. activity, which strongly proved the effectiveness of fragment VS based on Bayesian categorization models.  This investigation of Bayesian-based fragment VS can further emphasize the necessity for enrichment of compd. databases employed in lead discovery by amplifying the diversity of databases with novel structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm1F-dextwrrVg90H21EOLACvtfcHk0lgdpbw6x9VOvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1aqtr0%253D&md5=16fca90b4e5746913ba187b0043768de</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs11030-015-9592-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-015-9592-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiao%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DA.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DQiao%26aufirst%3DX.%26atitle%3DFragment%2520virtual%2520screening%2520based%2520on%2520bayesian%2520categorization%2520for%2520discovering%2520novel%2520VEGFR-2%2520scaffolds%26jtitle%3DMol.%2520Diversity%26date%3D2015%26volume%3D19%26spage%3D895%26epage%3D913%26doi%3D10.1007%2Fs11030-015-9592-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Kirchmair, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Distinto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuster, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzer, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liedl, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolber, G.</span><span> </span><span class="NLM_article-title">The protein data bank (PDB), its related services and software tools as key components for in silico guided drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7021</span><span class="NLM_x">–</span> <span class="NLM_lpage">7040</span><span class="refDoi"> DOI: 10.1021/jm8005977</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8005977" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7021-7040&author=J.+Kirchmairauthor=P.+Marktauthor=S.+Distintoauthor=D.+Schusterauthor=G.+M.+Spitzerauthor=K.+R.+Liedlauthor=T.+Langerauthor=G.+Wolber&title=The+protein+data+bank+%28PDB%29%2C+its+related+services+and+software+tools+as+key+components+for+in+silico+guided+drug+discovery&doi=10.1021%2Fjm8005977"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm8005977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8005977%26sid%3Dliteratum%253Aachs%26aulast%3DKirchmair%26aufirst%3DJ.%26aulast%3DMarkt%26aufirst%3DP.%26aulast%3DDistinto%26aufirst%3DS.%26aulast%3DSchuster%26aufirst%3DD.%26aulast%3DSpitzer%26aufirst%3DG.%2BM.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DWolber%26aufirst%3DG.%26atitle%3DThe%2520protein%2520data%2520bank%2520%2528PDB%2529%252C%2520its%2520related%2520services%2520and%2520software%2520tools%2520as%2520key%2520components%2520for%2520in%2520silico%2520guided%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7021%26epage%3D7040%26doi%3D10.1021%2Fjm8005977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljznBxR7sG3wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Zhou, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.-L.</span><span> </span><span class="NLM_article-title">The changing landscape of clinical trial and approval processes in China</span> <span class="citation_source-journal">Nat. Rev. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1038/nrclinonc.2017.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnrclinonc.2017.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=28195238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADC%252BC1c3msFShuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=577-583&author=Q.+Zhouauthor=X.-Y.+Chenauthor=Z.-M.+Yangauthor=Y.-L.+Wu&title=The+changing+landscape+of+clinical+trial+and+approval+processes+in+China&doi=10.1038%2Fnrclinonc.2017.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The changing landscape of clinical trial and approval processes in China</span></div><div class="casAuthors">Zhou Qing; Wu Yi-Long; Chen Xiao-Yuan; Yang Zhi-Min</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Clinical oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">577-583</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In the past decade, the standards of clinical trials in China have moved closer to international standards, thus encouraging the development of innovative drugs.  However, a large backlog of pending applications for both drug approval and clinical trial registration has arisen owing to the complexity of the approval process, the volume of applications and a lack of staff available to process these applications, among other reasons.  To improve the drug approval process, a 'four-colour-light' strategy was introduced.  Different drugs are classified into redefined categories of innovative and generic drugs, with priority being given to approval decisions concerning innovative drugs.  Other improvement strategies are now also being implemented, including the development of a new clinical trial approval system and several measures designed to encourage greater participation of Chinese researchers and research centres in international clinical trials.  In this Perspective, the changing landscape of clinical approval in China is described, including the difficulties that drug approval authorities face in this rapidly developing nation and the novel strategies that are being used to find solutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfGZ3GFxQ08lw2ofyr07jXfW6udTcc2eb444vmZpP657ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3msFShuw%253D%253D&md5=d5043cdbd883365ed4ac473ed8878504</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2017.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2017.10%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DX.-Y.%26aulast%3DYang%26aufirst%3DZ.-M.%26aulast%3DWu%26aufirst%3DY.-L.%26atitle%3DThe%2520changing%2520landscape%2520of%2520clinical%2520trial%2520and%2520approval%2520processes%2520in%2520China%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D14%26spage%3D577%26epage%3D583%26doi%3D10.1038%2Fnrclinonc.2017.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Park, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgescu, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galper, J. B.</span><span> </span><span class="NLM_article-title">Human umbilical vein endothelial cells and human dermal microvascular endothelial cells offer new insights into the relationship between lipid metabolism and angiogenesis</span> <span class="citation_source-journal">Stem Cell Rev. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">101</span><span class="refDoi"> DOI: 10.1007/s12015-006-0015-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2Fs12015-006-0015-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17237547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Siuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=93-101&author=H.-J.+Parkauthor=Y.+Zhangauthor=S.+P.+Georgescuauthor=K.+L.+Johnsonauthor=D.+Kongauthor=J.+B.+Galper&title=Human+umbilical+vein+endothelial+cells+and+human+dermal+microvascular+endothelial+cells+offer+new+insights+into+the+relationship+between+lipid+metabolism+and+angiogenesis&doi=10.1007%2Fs12015-006-0015-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Human umbilical vein endothelial cells and human dermal microvascular endothelial cells offer new insights into the relationship between lipid metabolism and angiogenesis</span></div><div class="casAuthors">Park, Ho-Jin; Zhang, Yali; Georgescu, Serban P.; Johnson, Kristin L.; Kong, Dequon; Galper, Jonas B.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">93-101</span>CODEN:
                <span class="NLM_cas:coden">SCRTCI</span>;
        ISSN:<span class="NLM_cas:issn">1550-8943</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review.  Human umbilical vein endothelial cells (HUVECs) have played a major role as a model system for the study of the regulation of endothelial cell function and the role of the endothelium in the response of the blood vessel wall to stretch, shear forces, and the development of atherosclerotic plaques and angiogenesis.  Here, we use HUVECs and human microvascular endothelial cells to study the role of the HMG-CoA reductase inhibitor, simvastatin, and the small GTP-binding protein Rho in the regulation of angiogenesis.  Simvastatin inhibited angiogenesis in response to FGF-2 in the corneal pocket assay of the mouse and in vascular endothelial growth factor (VEGF)-stimulated angiogenesis in the chick chorioallontoic membrane.  Furthermore, simvastatin inhibited VEGF-stimulated tube formation by human dermal microvascular endothelial cells and the formation of honeycomb-like structures by HUVECs.  The effect was dose-dependent and was not secondary to apoptosis.  Geranylgeranyl-pyrophosphate (GGPP), a product of the cholesterol metabolic pathway that serves as a substrate for the posttranslational lipidation of RhoA, was required for membrane localization, but not farnesylpyrophosphate(FPP), the substrate for the lipidation of Ras.  Furthermore, GGTI, a specific inhibitor of GGPP, mimicked the effect of simvastatin of tube formation and the formation of honeycombs whereas FTI, a specific inhibitor of the farnesylation of Ras, had no effect.  Adenoviral expression of a DN-RhoA mutant mimicked the effect of simvastatin on tube formation and the formation of honeycombs, whereas a dominant activating mutant of RhoA reversed the effect of simvastatin on tube formation.  Finally, simvastatin interfered with the membrane localization of RhoA with a dose-dependence similar to that for the inhibition of tube formation.  Simvastatin also inhibited the VEGF-stimulated phosphorylation of the VEGF receptor KDR, and the tyrosine kinase FAK, which plays a role in cell migration.  These data demonstrate that simvastatin interfered with angiogenesis via the inhibition of RhoA.  Data supporting a role for angiogenesis in the development and growth of atherosclerotic plaques suggest that this antiangiogenic effect of Statins might prevent the progression of atherosclerosis via the inhibition of plaque angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPCLaFKAcrhrVg90H21EOLACvtfcHk0lg0ZGBfOzm9Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Siuro%253D&md5=1074e701324394d9d3f2b78ca7c1ad85</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs12015-006-0015-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12015-006-0015-x%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.-J.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGeorgescu%26aufirst%3DS.%2BP.%26aulast%3DJohnson%26aufirst%3DK.%2BL.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DGalper%26aufirst%3DJ.%2BB.%26atitle%3DHuman%2520umbilical%2520vein%2520endothelial%2520cells%2520and%2520human%2520dermal%2520microvascular%2520endothelial%2520cells%2520offer%2520new%2520insights%2520into%2520the%2520relationship%2520between%2520lipid%2520metabolism%2520and%2520angiogenesis%26jtitle%3DStem%2520Cell%2520Rev.%2520Rep.%26date%3D2006%26volume%3D2%26spage%3D93%26epage%3D101%26doi%3D10.1007%2Fs12015-006-0015-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Ross, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherf, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisen, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perou, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spellman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van de Rijn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waltham, M.</span><span> </span><span class="NLM_article-title">Systematic variation in gene expression patterns in human cancer cell lines</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span><span class="refDoi"> DOI: 10.1038/73432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2F73432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10700174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvFaqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2000&pages=227-235&author=D.+T.+Rossauthor=U.+Scherfauthor=M.+B.+Eisenauthor=C.+M.+Perouauthor=C.+Reesauthor=P.+Spellmanauthor=V.+Iyerauthor=S.+S.+Jeffreyauthor=M.+Van+de+Rijnauthor=M.+Waltham&title=Systematic+variation+in+gene+expression+patterns+in+human+cancer+cell+lines&doi=10.1038%2F73432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic variation in gene expression patterns in human cancer cell lines</span></div><div class="casAuthors">Ross, Douglas T.; Scherf, Uwe; Eisen, Michael B.; Perou, Charles M.; Rees, Christian; Spellman, Paul; Iyer, Vishwanath; Jeffrey, Stefanie S.; Van de Rijn, Matt; Waltham, Mark; Pergamenschikov, Alexander; Lee, Jeffrey C. F.; Lashkari, Deval; Shalon, Dari; Myers, Timothy G.; Weinstein, John N.; Botstein, David; Brown, Patrick O.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">227-235</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">We used cDNA microarrays to explore the variation in expression of approx. 8,000 unique genes among the 60 cell lines used in the National Cancer Institute's screen for anti-cancer drugs.  Classification of the cell lines based solely on the obsd. patterns of gene expression revealed a correspondence to the ostensible origins of the tumors from which the cell lines were derived.  The consistent relationship between the gene expression patterns and the tissue of origin allowed us to recognize outliers whose previous classification appeared incorrect.  Specific features of the gene expression patterns appeared to be related to physiol. properties of the cell lines, such as their doubling time in culture, drug metab. or the interferon response.  Comparison of gene expression patterns in the cell lines to those obsd. in normal breast tissue or in breast tumor specimens revealed features of the expression patterns in the tumors that had recognizable counterparts in specific cell lines, reflecting the tumor, stromal and inflammatory components of the tumor tissue.  These results provided a novel mol. characterization of this important group of human cell lines and their relationships to tumors in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZhocezjCTNbVg90H21EOLACvtfcHk0lg0ZGBfOzm9Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvFaqtrg%253D&md5=248ffa9428fa078fff0bdf45fddaecf4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F73432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F73432%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DD.%2BT.%26aulast%3DScherf%26aufirst%3DU.%26aulast%3DEisen%26aufirst%3DM.%2BB.%26aulast%3DPerou%26aufirst%3DC.%2BM.%26aulast%3DRees%26aufirst%3DC.%26aulast%3DSpellman%26aufirst%3DP.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DJeffrey%26aufirst%3DS.%2BS.%26aulast%3DVan%2Bde%2BRijn%26aufirst%3DM.%26aulast%3DWaltham%26aufirst%3DM.%26atitle%3DSystematic%2520variation%2520in%2520gene%2520expression%2520patterns%2520in%2520human%2520cancer%2520cell%2520lines%26jtitle%3DNat.%2520Genet.%26date%3D2000%26volume%3D24%26spage%3D227%26epage%3D235%26doi%3D10.1038%2F73432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Stanton, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edington, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendler, F.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor expression by human squamous cell carcinomas of the head and neck, cell lines and xenografts</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">427</span><span class="NLM_x">–</span> <span class="NLM_lpage">433</span><span class="refDoi"> DOI: 10.1038/bjc.1994.322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fbjc.1994.322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=8080726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK2cXmsFaksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1994&pages=427-433&author=P.+Stantonauthor=S.+Richardsauthor=J.+Reevesauthor=M.+Nikolicauthor=K.+Edingtonauthor=L.+Clarkauthor=G.+Robertsonauthor=D.+Souterauthor=R.+Mitchellauthor=F.+Hendler&title=Epidermal+growth+factor+receptor+expression+by+human+squamous+cell+carcinomas+of+the+head+and+neck%2C+cell+lines+and+xenografts&doi=10.1038%2Fbjc.1994.322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor expression by human squamous cell carcinomas of the head and neck, cell lines and xenografts</span></div><div class="casAuthors">Stanton, P.; Richards, S.; Reeves, J.; Nikolic, M.; Edington, K.; Clark, L.; Robertson, G.; Souter, D.; Mitchell, R.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">427-33</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) overexpression has been assocd. frequently with squamous cell carcinomas (SCC) and SCC cell lines.  In most cases the level of EGFR on the tumors from which the cell lines were derived has not been detd., nor have EGFR levels been detd. for xenograft tumors from the cell lines.  In this study the authors detd. EGFR expression on a new series of head and neck SCC (SCCHN)-derived cell lines, which were obtained from tumors representing a spectrum of malignant progression, and two cell strains derived from erythroplakia premalignant lesions.  The level of EGFR on cell lines was detd. by [125I]EGF competitive binding assays.  EGFR levels on some of the original tumors and xenografts of the cell lines were detd. on cryosections by a competitive binding assay based on [125I]EGFR1, an EGFR-specific monoclonal antibody.  EGFR expression on the tumor cryosections was compared with expression on cryosections of skin and buccal mucosa.  Eight of the ten tumor cell lines had elevated EGFR.  Two of the tumor-derived cell lines and the two erythroplakia-derived cell strains expressed EGFR at levels similar to that detected on normal keratinocytes in tissue culture.  Only two of the tumors overexpressed EGFR when compared with normal tissue.  The other tumors had levels similar to that detected on the basal layers of skin or buccal mucosa.  The xenografts expressed EGFR, as did the original tumors, even though they were derived from cell lines that displayed significant overexpression of EGFR.  This study suggests that most tumors have a latent potential to overexpress EGFR which is realized in tissue culture.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1dvIlD-kurVg90H21EOLACvtfcHk0lg0ZGBfOzm9Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsFaksr0%253D&md5=65349d81c3a8650debfe604ddadbb2f3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fbjc.1994.322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.1994.322%26sid%3Dliteratum%253Aachs%26aulast%3DStanton%26aufirst%3DP.%26aulast%3DRichards%26aufirst%3DS.%26aulast%3DReeves%26aufirst%3DJ.%26aulast%3DNikolic%26aufirst%3DM.%26aulast%3DEdington%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DRobertson%26aufirst%3DG.%26aulast%3DSouter%26aufirst%3DD.%26aulast%3DMitchell%26aufirst%3DR.%26aulast%3DHendler%26aufirst%3DF.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520expression%2520by%2520human%2520squamous%2520cell%2520carcinomas%2520of%2520the%2520head%2520and%2520neck%252C%2520cell%2520lines%2520and%2520xenografts%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1994%26volume%3D70%26spage%3D427%26epage%3D433%26doi%3D10.1038%2Fbjc.1994.322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiesewetter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span> </span><span class="NLM_article-title">Site-specific labeling of scVEGF with fluorine-18 for positron emission tomography imaging</span> <span class="citation_source-journal">Theranostics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.7150/thno.4611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.7150%2Fthno.4611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=22768028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCqsbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=607-617&author=H.+Wangauthor=H.+Gaoauthor=N.+Guoauthor=G.+Niuauthor=Y.+Maauthor=D.+Kiesewetterauthor=X.+Chen&title=Site-specific+labeling+of+scVEGF+with+fluorine-18+for+positron+emission+tomography+imaging&doi=10.7150%2Fthno.4611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Site-specific labeling of scVEGF with fluorine-18 for positron emission tomography imaging</span></div><div class="casAuthors">Wang, Hui; Gao, Haokao; Guo, Ning; Niu, Gang; Ma, Ying; Kiesewetter, Dale O.; Chen, Xiaoyuan</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">607-617</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) is one of the most important mediators of angiogenesis.  Single-chain (s.c.)-VEGF protein contg. an N-terminal Cys-tag has been designed for site-specific modification with a variety of imaging and therapeutic moieties.  Site-specific labeling of scVEGF with thiol-reactive prosthetic group, N-[2-(4-18F-fluorobenzamido) ethyl] maleimide ([18F]FBEM) for positron emission tomog. (PET) imaging of VEFGR may provide a new tracer which has great potential for clin. translation.  Methods: [18F]FBEM-scVEGF was synthesized by site-specific conjugation of 18F-FBEM to a thiol group in Cys-tag of scVEGF at room temp.  The functional activity after labeling was tested by immunofluorescence staining, cellular uptake and efflux.  The tumor targeting and in vivo properties were evaluated by biodistribution and microPET studies in tumor-bearing mice.  Results: The radiolabeling yield and specific activity of [18F]FBEM-scVEGF were 20.6 ± 15.1 % (based on starting [18F]FBEM, uncorrected, n = 5) and 58.8 ± 12.4 GBq/μmol, resp.  Noninvasive microPET and direct tissue sampling expts. demonstrated that [18F]FBEM-scVEGF had VEGFR specific tumor uptake in MDA-MB-435, U87MG and 4T1 xenograft models.  The optimal tumor uptake was achieved at 2 h p.i., which can be partially, but significantly blocked by co-injection of non-labeled scVEGF protein.  Overall, [18F]FBEM-scVEGF showed VEGFR specific tumor uptake.  Conclusion: The scVEGF was site-specifically labeled with 18F via [18F]FBEM prosthetic group and the tracer [18F]FBEM-scVEGF exhibited high receptor binding affinity and tumor targeting efficacy.  Further study of [18F] FBEM-scVEGF to evaluate angiogenesis in cancer and other disease types is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3Qpct2fPYNrVg90H21EOLACvtfcHk0ljxQxKJQ3u9CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCqsbjP&md5=4b35e0e179ccead86069f2b706d515db</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7150%2Fthno.4611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.4611%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DN.%26aulast%3DNiu%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DKiesewetter%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DSite-specific%2520labeling%2520of%2520scVEGF%2520with%2520fluorine-18%2520for%2520positron%2520emission%2520tomography%2520imaging%26jtitle%3DTheranostics%26date%3D2012%26volume%3D2%26spage%3D607%26epage%3D617%26doi%3D10.7150%2Fthno.4611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Cerezo, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornedo-Ortega, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Álvarez-Fernández, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troncoso, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Parrilla, M. C.</span><span> </span><span class="NLM_article-title">Inhibition of VEGF-Induced VEGFR-2 activation and HUVEC migration by melatonin and other bioactive indolic compounds</span> <span class="citation_source-journal">Nutrients</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="refDoi"> DOI: 10.3390/nu9030249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.3390%2Fnu9030249" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=249&author=A.+B.+Cerezoauthor=R.+Hornedo-Ortegaauthor=M.+A.+%C3%81lvarez-Fern%C3%A1ndezauthor=A.+M.+Troncosoauthor=M.+C.+Garc%C3%ADa-Parrilla&title=Inhibition+of+VEGF-Induced+VEGFR-2+activation+and+HUVEC+migration+by+melatonin+and+other+bioactive+indolic+compounds&doi=10.3390%2Fnu9030249"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.3390%2Fnu9030249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fnu9030249%26sid%3Dliteratum%253Aachs%26aulast%3DCerezo%26aufirst%3DA.%2BB.%26aulast%3DHornedo-Ortega%26aufirst%3DR.%26aulast%3D%25C3%2581lvarez-Fern%25C3%25A1ndez%26aufirst%3DM.%2BA.%26aulast%3DTroncoso%26aufirst%3DA.%2BM.%26aulast%3DGarc%25C3%25ADa-Parrilla%26aufirst%3DM.%2BC.%26atitle%3DInhibition%2520of%2520VEGF-Induced%2520VEGFR-2%2520activation%2520and%2520HUVEC%2520migration%2520by%2520melatonin%2520and%2520other%2520bioactive%2520indolic%2520compounds%26jtitle%3DNutrients%26date%3D2017%26volume%3D9%26spage%3D249%26doi%3D10.3390%2Fnu9030249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acheampong, D. O.</span><span> </span><span class="NLM_article-title">A novel bispecific diabody targeting both vascular endothelial growth factor receptor 2 and epidermal growth factor receptor for enhanced antitumor activity</span> <span class="citation_source-journal">Biotechnol. Prog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">294</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1002/btpr.2231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fbtpr.2231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=26785424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFSqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=294-302&author=M.+Xuauthor=H.+Jinauthor=Z.+Chenauthor=W.+Xieauthor=Y.+Wangauthor=Y.+Wangauthor=M.+Wangauthor=J.+Zhangauthor=D.+O.+Acheampong&title=A+novel+bispecific+diabody+targeting+both+vascular+endothelial+growth+factor+receptor+2+and+epidermal+growth+factor+receptor+for+enhanced+antitumor+activity&doi=10.1002%2Fbtpr.2231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A novel bispecific diabody targeting both vascular endothelial growth factor receptor 2 and epidermal growth factor receptor for enhanced antitumor activity</span></div><div class="casAuthors">Xu, Menghuai; Jin, Haizhen; Chen, Zhiguo; Xie, Wei; Wang, Youfu; Wang, Yang; Wang, Min; Zhang, Juan; Acheampong, Desmond Omane</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Progress</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">294-302</span>CODEN:
                <span class="NLM_cas:coden">BIPRET</span>;
        ISSN:<span class="NLM_cas:issn">1520-6033</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) and vascular endothelial growth factor receptor 2 (VEGFR2) are receptor tyrosine kinases known to play crit. roles in the development and progression of tumors.  Based on the cross-talk between EGFR and VEGFR2 signal pathways, we designed and produced a bispecific diabody (bDAb) targeting both EGFR and VEGFR2 simultaneously.  The bispecific mol. (EK-02) demonstrated that it could bind to HUVEC (VEGFR2 high-expressing) and A431 (EGFR overexpressing) cells.  Addnl., similar to the parental antibodies, it was able to inhibit proliferation and migration, and induced apoptosis in these cells (HUVECs and A431), demonstrating that it had retained the functional properties of its parental antibodies.  Furthermore, the efficacy of EK-02 was evaluated using the human colon adenocarcinoma cell line HT29 (VEGFR2 and EGFR coexpressing).  In vitro assay showed that EK-02 could bind to HT29 cells, restrain cell growth and migration, and induce apoptosis with enhanced efficacy compared to both parental antibodies.  Further, it inhibited the neovascularization and tumor formation on an HT29 cell bearing chicken chorioallantoic membrane (CAM) tumor model in vivo.  In conclusion, these data suggest that the novel bDAb (EK-02) has antiangiogenesis and antitumor capacity both in vitro and in vivo, and can possibly be used as cotargeted therapy for the treatment of EGFR and VEGFR2 overexpressing tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogmVRKSshnfbVg90H21EOLACvtfcHk0lhcam1KYoSzPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFSqsrw%253D&md5=0a90ec5010b003b73229a2a24e37dfdf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fbtpr.2231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbtpr.2231%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAcheampong%26aufirst%3DD.%2BO.%26atitle%3DA%2520novel%2520bispecific%2520diabody%2520targeting%2520both%2520vascular%2520endothelial%2520growth%2520factor%2520receptor%25202%2520and%2520epidermal%2520growth%2520factor%2520receptor%2520for%2520enhanced%2520antitumor%2520activity%26jtitle%3DBiotechnol.%2520Prog.%26date%3D2016%26volume%3D32%26spage%3D294%26epage%3D302%26doi%3D10.1002%2Fbtpr.2231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Chow, L. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhardt, S. G.</span><span> </span><span class="NLM_article-title">Sunitinib: from rational design to clinical efficacy</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">884</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1200/JCO.2006.06.3602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1200%2FJCO.2006.06.3602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17327610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvVKmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=884-896&author=L.+Q.+Chowauthor=S.+G.+Eckhardt&title=Sunitinib%3A+from+rational+design+to+clinical+efficacy&doi=10.1200%2FJCO.2006.06.3602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Sunitinib: from rational design to clinical efficacy</span></div><div class="casAuthors">Chow, Laura Q. M.; Eckhardt, S. Gail</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">884-896</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">A review.  Sunitinib (SU011248) is an oral small mol. tyrosine kinase inhibitor that exhibits potent antiangiogenic and antitumor activity.  Tyrosine kinase inhibitors such as SU6668 and SU5416 (semaxanib) demonstrated poor pharmacol. properties and limited efficacy; therefore, sunitinib was rationally designed and chosen for its high bioavailability and its nanomolar-range potency against the antiangiogenic receptor tyrosine kinases (RTKs)-vascular endothelial growth factor receptor (VEGFR) and platelet-derived growth factor receptor (PDGFR).  Sunitinib inhibits other tyrosine kinases including, KIT, FLT3, colony-stimulating factor 1 (CSF-1), and RET, which are involved in a no. of malignancies including small-cell lung cancer, GI stromal tumors (GISTs), breast cancer, acute myelogenous leukemia, multiple endocrine neoplasia types 2A and 2B, and familial medullary thyroid carcinoma.  Sunitinib demonstrated robust antitumor activity in preclin. studies resulting not only in tumor growth inhibition, but tumor regression in models of colon cancer, non-small-cell lung cancer, melanoma, renal carcinoma, and squamous cell carcinoma, which were assocd. with inhibition of VEGFR and PDGFR phosphorylation.  Clin. activity was demonstrated in neuroendocrine, colon, and breast cancers in phase II studies, whereas definitive efficacy has been demonstrated in advanced renal cell carcinoma and in imatinib-refractory GISTs, leading to US Food and Drug Administration approval of sunitinib for treatment of these two diseases.  Studies investigating sunitinib alone in various tumor types and in combination with chemotherapy are ongoing.  The clin. benchmarking of this small-mol. inhibitor of members of the split-kinase domain family of RTKs will lead to addnl. insights regarding the biol., potential biomarkers, and clin. utility of agents that target multiple signaling pathways in tumor, stromal, and endothelial compartments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPiO2cteaPj7Vg90H21EOLACvtfcHk0lhcam1KYoSzPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvVKmtLY%253D&md5=c2b5469aff21b2f3062249c9f56c3814</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1200%2FJCO.2006.06.3602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2006.06.3602%26sid%3Dliteratum%253Aachs%26aulast%3DChow%26aufirst%3DL.%2BQ.%26aulast%3DEckhardt%26aufirst%3DS.%2BG.%26atitle%3DSunitinib%253A%2520from%2520rational%2520design%2520to%2520clinical%2520efficacy%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2007%26volume%3D25%26spage%3D884%26epage%3D896%26doi%3D10.1200%2FJCO.2006.06.3602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Norris-Cervetto, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platt, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dwek, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butters, T. D.</span><span> </span><span class="NLM_article-title">Inhibition of glucosylceramide synthase does not reverse drug resistance in cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">40412</span><span class="NLM_x">–</span> <span class="NLM_lpage">40418</span><span class="refDoi"> DOI: 10.1074/jbc.M404466200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1074%2Fjbc.M404466200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=40412-40418&author=E.+Norris-Cervettoauthor=R.+Callaghanauthor=F.+M.+Plattauthor=R.+A.+Dwekauthor=T.+D.+Butters&title=Inhibition+of+glucosylceramide+synthase+does+not+reverse+drug+resistance+in+cancer+cells&doi=10.1074%2Fjbc.M404466200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M404466200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M404466200%26sid%3Dliteratum%253Aachs%26aulast%3DNorris-Cervetto%26aufirst%3DE.%26aulast%3DCallaghan%26aufirst%3DR.%26aulast%3DPlatt%26aufirst%3DF.%2BM.%26aulast%3DDwek%26aufirst%3DR.%2BA.%26aulast%3DButters%26aufirst%3DT.%2BD.%26atitle%3DInhibition%2520of%2520glucosylceramide%2520synthase%2520does%2520not%2520reverse%2520drug%2520resistance%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D40412%26epage%3D40418%26doi%3D10.1074%2Fjbc.M404466200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Padro, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bieker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steins, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Retzlaff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bürger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Büchner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrera, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kienast, J.</span><span> </span><span class="NLM_article-title">Overexpression of vascular endothelial growth factor (VEGF) and its cellular receptor KDR (VEGFR-2) in the bone marrow of patients with acute myeloid leukemia</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1302</span><span class="NLM_x">–</span> <span class="NLM_lpage">1310</span><span class="refDoi"> DOI: 10.1038/sj.leu.2402534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fsj.leu.2402534" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1302-1310&author=T.+Padroauthor=R.+Biekerauthor=S.+Ruizauthor=M.+Steinsauthor=S.+Retzlaffauthor=H.+B%C3%BCrgerauthor=T.+B%C3%BCchnerauthor=T.+Kesslerauthor=F.+Herreraauthor=J.+Kienast&title=Overexpression+of+vascular+endothelial+growth+factor+%28VEGF%29+and+its+cellular+receptor+KDR+%28VEGFR-2%29+in+the+bone+marrow+of+patients+with+acute+myeloid+leukemia&doi=10.1038%2Fsj.leu.2402534"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2402534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2402534%26sid%3Dliteratum%253Aachs%26aulast%3DPadro%26aufirst%3DT.%26aulast%3DBieker%26aufirst%3DR.%26aulast%3DRuiz%26aufirst%3DS.%26aulast%3DSteins%26aufirst%3DM.%26aulast%3DRetzlaff%26aufirst%3DS.%26aulast%3DB%25C3%25BCrger%26aufirst%3DH.%26aulast%3DB%25C3%25BCchner%26aufirst%3DT.%26aulast%3DKessler%26aufirst%3DT.%26aulast%3DHerrera%26aufirst%3DF.%26aulast%3DKienast%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520%2528VEGF%2529%2520and%2520its%2520cellular%2520receptor%2520KDR%2520%2528VEGFR-2%2529%2520in%2520the%2520bone%2520marrow%2520of%2520patients%2520with%2520acute%2520myeloid%2520leukemia%26jtitle%3DLeukemia%26date%3D2002%26volume%3D16%26spage%3D1302%26epage%3D1310%26doi%3D10.1038%2Fsj.leu.2402534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Stefanik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmey, J. H.</span><span> </span><span class="NLM_article-title">Vascular Endothelial Growth Factor in Malignant Disease of the Central Nervous System</span> <span class="citation_source-journal">VEGF and Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1007/978-1-4419-9148-5_9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1007%2F978-1-4419-9148-5_9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Kgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=72-82&author=D.+Stefanikauthor=J.+H.+Harmey&title=Vascular+Endothelial+Growth+Factor+in+Malignant+Disease+of+the+Central+Nervous+System&doi=10.1007%2F978-1-4419-9148-5_9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Vascular endothelial growth factor in malignant disease of the central nervous system</span></div><div class="casAuthors">Stefanik, David</div><div class="citationInfo"><span class="NLM_cas:title">VEGF and Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">69GMAZ</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Vascular Endothelial Growth Factor (VEGF) is a major contributor to the growth of malignant tumors of the central nervous system.  It stimulates tumor angiogenesis and vascular proliferation characteristic of high grade gliomas.  Elevated expression of VEGF is one the factors responsible for the virulent nature of these tumors.  The prodn. of VEGF by malignant glial cells in response to ionizing radiation contributes to treatment failure.  The rat C6 glioma is similar to human gliomas with respect to VEGF pathophysiol.  Interruption of VEGF-Receptor signaling in preclin. models effectively suppresses tumor growth and demonstrates the potential for anti-angiogenic therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfoquTOdbjfrVg90H21EOLACvtfcHk0ljMMEe3vgPDRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Kgs7k%253D&md5=07224f3d57e29325cd04ce58bf41e436</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-9148-5_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-9148-5_9%26sid%3Dliteratum%253Aachs%26aulast%3DStefanik%26aufirst%3DD.%26aulast%3DHarmey%26aufirst%3DJ.%2BH.%26atitle%3DVascular%2520Endothelial%2520Growth%2520Factor%2520in%2520Malignant%2520Disease%2520of%2520the%2520Central%2520Nervous%2520System%26jtitle%3DVEGF%2520and%2520Cancer%26date%3D2004%26spage%3D72%26epage%3D82%26doi%3D10.1007%2F978-1-4419-9148-5_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Dias, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heissig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadburn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hyjek, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, M.</span><span> </span><span class="NLM_article-title">Autocrine stimulation of VEGFR-2 activates human leukemic cell growth and migration</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">521</span><span class="refDoi"> DOI: 10.1172/JCI8978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1172%2FJCI8978" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2000&pages=511-521&author=S.+Diasauthor=K.+Hattoriauthor=Z.+Zhuauthor=B.+Heissigauthor=M.+Choyauthor=W.+Laneauthor=Y.+Wuauthor=A.+Chadburnauthor=E.+Hyjekauthor=M.+Gill&title=Autocrine+stimulation+of+VEGFR-2+activates+human+leukemic+cell+growth+and+migration&doi=10.1172%2FJCI8978"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1172%2FJCI8978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI8978%26sid%3Dliteratum%253Aachs%26aulast%3DDias%26aufirst%3DS.%26aulast%3DHattori%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DHeissig%26aufirst%3DB.%26aulast%3DChoy%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DHyjek%26aufirst%3DE.%26aulast%3DGill%26aufirst%3DM.%26atitle%3DAutocrine%2520stimulation%2520of%2520VEGFR-2%2520activates%2520human%2520leukemic%2520cell%2520growth%2520and%2520migration%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2000%26volume%3D106%26spage%3D511%26epage%3D521%26doi%3D10.1172%2FJCI8978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Salven, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heikkila, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joensuu, H.</span><span> </span><span class="NLM_article-title">Enhanced expression of vascular endothelial growth factor in metastatic melanoma</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">930</span><span class="NLM_x">–</span> <span class="NLM_lpage">934</span><span class="refDoi"> DOI: 10.1038/bjc.1997.486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fbjc.1997.486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=9328154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVGqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=1997&pages=930-934&author=P.+Salvenauthor=P.+Heikkilaauthor=H.+Joensuu&title=Enhanced+expression+of+vascular+endothelial+growth+factor+in+metastatic+melanoma&doi=10.1038%2Fbjc.1997.486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced expression of vascular endothelial growth factor in metastatic melanoma</span></div><div class="casAuthors">Salven, P.; Heikkila, P.; Joensuu, H.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">930-934</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Churchill Livingstone</span>)
        </div><div class="casAbstract">Tumor growth is dependent on angiogenesis.  Vascular endothelial growth factor (VEGF) is a secreted endothelial cell-specific cytokine.  VEGF is angiogenic in vivo and it also acts as a vascular permeability factor.  VEGF is overexpressed in many skin disorders characterized by angiogenesis and increased vascular permeability.  The authors investigated VEGF expression in 22 primary cutaneous melanomas, 33 melanoma metastases and six nevocellular nevi using immunohistochem.  VEGF accumulated on the vascular endothelia in the normal dermis, suggesting that a constitutive low level of VEGF expression may regulate skin vessel function under normal physiol. conditions.  No VEGF was detected in the cells of nevocellular nevi or normal dermis.  In contrast, 32% of the primary and 91% of the metastatic melanomas contained melanoma cells staining for VEGF.  Expression of VEGF was more frequent in metastases than in primary melanomas.  Tumor-infiltrating inflammatory cells expressed VEGF in all melanomas.  A high no. of VEGF-expressing inflammatory cells was assocd. with high VEGF expression in melanoma cells.  The authors' results suggest that VEGF is up-regulated during the course of melanoma progression and dissemination and that tumor-infiltrating cells expressing VEGF may contribute to the progression of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_PeGlbNlEeLVg90H21EOLACvtfcHk0ljgAo-YrLhL-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVGqurw%253D&md5=26c25abadec8d64fcd733978bd1b5dc3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fbjc.1997.486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.1997.486%26sid%3Dliteratum%253Aachs%26aulast%3DSalven%26aufirst%3DP.%26aulast%3DHeikkila%26aufirst%3DP.%26aulast%3DJoensuu%26aufirst%3DH.%26atitle%3DEnhanced%2520expression%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520in%2520metastatic%2520melanoma%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1997%26volume%3D76%26spage%3D930%26epage%3D934%26doi%3D10.1038%2Fbjc.1997.486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Mehnert, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCarthy, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jilaveanu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aziz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camp, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimm, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kluger, H. M.</span><span> </span><span class="NLM_article-title">Quantitative expression of VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 in melanoma tissue microarrays</span> <span class="citation_source-journal">Hum. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">384</span><span class="refDoi"> DOI: 10.1016/j.humpath.2009.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1016%2Fj.humpath.2009.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=20004943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitV2ktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2010&pages=375-384&author=J.+M.+Mehnertauthor=M.+M.+McCarthyauthor=L.+Jilaveanuauthor=K.+T.+Flahertyauthor=S.+Azizauthor=R.+L.+Campauthor=D.+L.+Rimmauthor=H.+M.+Kluger&title=Quantitative+expression+of+VEGF%2C+VEGF-R1%2C+VEGF-R2%2C+and+VEGF-R3+in+melanoma+tissue+microarrays&doi=10.1016%2Fj.humpath.2009.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative expression of VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 in melanoma tissue microarrays</span></div><div class="casAuthors">Mehnert, Janice M.; McCarthy, Mary M.; Jilaveanu, Lucia; Flaherty, Keith T.; Aziz, Saadia; Camp, Robert L.; Rimm, David L.; Kluger, Harriet M.</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-384</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Summary: Angiogenesis is required for progression and metastasis of melanoma.  Anal. of angiogenic mols. in benign and malignant tissues may allow identification of markers useful for prediction of sensitivity to antiangiogenic agents.  We hypothesized that differential expression of vascular endothelial growth factor (VEGF) and its receptors VEGF-R1, VEGF-R2, and VEGF-R3 would be higher in melanomas than nevi and higher in advanced melanoma.  Using automated quant. anal., we quantified VEGF, -R1, -R2 and -R3 expression in melanoma tissue microarrays composed of 540 nevi and 468 melanoma specimens (198 primaries, 270 metastases).  VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 expression was significantly higher in melanomas than nevi by unpaired t tests (P < .0001).  VEGF-R2 expression was higher in metastatic specimens (P < .0001), but VEGF-R3 expression was higher in primaries (P < .0001).  VEGF was coexpressed with all 3 receptors when assessed by Spearman's rank correlation.  VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 expression is higher in melanomas than nevi.  Higher expression of VEGF-R2 was found in metastases vs. primaries, supporting the idea that selection for an angiogenic phenotype in metastatic melanoma is conferred via up-regulation of VEGF-R2.  However, higher expression of VEGF-R3 was seen on primary lesions, potentially implicating this receptor in initiation of lymphatic tumor spread.  Clin. trials using antiangiogenic agents in melanoma should include correlative assays of VEGF, VEGF-R1, VEGF-R2, and VEGF-R3 as biomarkers of response to therapy, preferably using quant. methods such as automated quant. anal.  Such assessments could assist with evaluation of these mols. as therapeutic targets in melanoma, ultimately facilitating improved selection of patients for treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyXtBqrIqco7Vg90H21EOLACvtfcHk0ljgAo-YrLhL-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitV2ktLo%253D&md5=323546819131fb9059604c900374ed58</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2009.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2009.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DMehnert%26aufirst%3DJ.%2BM.%26aulast%3DMcCarthy%26aufirst%3DM.%2BM.%26aulast%3DJilaveanu%26aufirst%3DL.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26aulast%3DAziz%26aufirst%3DS.%26aulast%3DCamp%26aufirst%3DR.%2BL.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DKluger%26aufirst%3DH.%2BM.%26atitle%3DQuantitative%2520expression%2520of%2520VEGF%252C%2520VEGF-R1%252C%2520VEGF-R2%252C%2520and%2520VEGF-R3%2520in%2520melanoma%2520tissue%2520microarrays%26jtitle%3DHum.%2520Pathol.%26date%3D2010%26volume%3D41%26spage%3D375%26epage%3D384%26doi%3D10.1016%2Fj.humpath.2009.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Molhoek, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griesemann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershenwald, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigan, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slingluff, C. L.</span><span> </span><span class="NLM_article-title">Human melanoma cytolysis by combined inhibition of mammalian target of rapamycin and vascular endothelial growth factor/vascular endothelial growth factor receptor-2</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">4392</span><span class="NLM_x">–</span> <span class="NLM_lpage">4397</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-5844</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1158%2F0008-5472.CAN-07-5844" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=4392-4397&author=K.+R.+Molhoekauthor=H.+Griesemannauthor=J.+Shuauthor=J.+E.+Gershenwaldauthor=D.+L.+Brautiganauthor=C.+L.+Slingluff&title=Human+melanoma+cytolysis+by+combined+inhibition+of+mammalian+target+of+rapamycin+and+vascular+endothelial+growth+factor%2Fvascular+endothelial+growth+factor+receptor-2&doi=10.1158%2F0008-5472.CAN-07-5844"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-5844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-5844%26sid%3Dliteratum%253Aachs%26aulast%3DMolhoek%26aufirst%3DK.%2BR.%26aulast%3DGriesemann%26aufirst%3DH.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DGershenwald%26aufirst%3DJ.%2BE.%26aulast%3DBrautigan%26aufirst%3DD.%2BL.%26aulast%3DSlingluff%26aufirst%3DC.%2BL.%26atitle%3DHuman%2520melanoma%2520cytolysis%2520by%2520combined%2520inhibition%2520of%2520mammalian%2520target%2520of%2520rapamycin%2520and%2520vascular%2520endothelial%2520growth%2520factor%252Fvascular%2520endothelial%2520growth%2520factor%2520receptor-2%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D4392%26epage%3D4397%26doi%3D10.1158%2F0008-5472.CAN-07-5844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Molhoek, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasamny, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patterson, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slingluff, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigan, D. L.</span><span> </span><span class="NLM_article-title">VEGFR-2 expression in human melanoma: Revised assessment</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">2807</span><span class="NLM_x">–</span> <span class="NLM_lpage">2815</span><span class="refDoi"> DOI: 10.1002/ijc.25963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fijc.25963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=21544800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKgurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2011&pages=2807-2815&author=K.+R.+Molhoekauthor=G.+Erdagauthor=J.+Rasamnyauthor=C.+Murphyauthor=D.+Deaconauthor=J.+W.+Pattersonauthor=C.+L.+Slingluffauthor=D.+L.+Brautigan&title=VEGFR-2+expression+in+human+melanoma%3A+Revised+assessment&doi=10.1002%2Fijc.25963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">VEGFR-2 expression in human melanoma: Revised assessment</span></div><div class="casAuthors">Molhoek, Kerrington R.; Erdag, Gulsun; Rasamny, Jk; Murphy, Cheryl; Deacon, Donna; Patterson, James W.; Slingluff, Craig L.; Brautigan, David L.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2807-2815</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) is an angiogenic factor that also functions as an autocrine growth factor for VEGF receptor (VEGFR)-2+ melanomas.  In multiple studies, VEGFR-2 was detected by immunostaining in 78-89% of human melanoma cells, suggesting that most patients with melanoma would benefit from anti-VEGF therapy.  Here, we evaluated 167 human melanoma specimens in a tissue microarray to verify the presence of VEGFR-2, but found disparities in staining with com. antibodies A-3 and 55B11.  Antibody A-3 stained melanoma cells in 79% of specimens, consistent with published results; however, we noted extensive nonspecific staining of other cells such as smooth muscle and histiocytes.  In contrast, antibody 55B11 stained melanoma cells in only 7% (95% confidence interval: 3.3-11.5) of specimens.  As an internal pos. control for VEGFR-2 detection, vascular endothelial cells were stained with antibody 55B11 in all specimens.  We compared VEGFR-2+ and VEGFR-2- melanoma cell lines by immunoblotting and immunohistochem. after small interfering RNA (siRNA) knockdown and transient overexpression of VEGFR-2 to validate antibody specificity.  Immunoblotting revealed that A-3 primarily cross-reacted with several proteins in both cell lines and these were unaffected by siRNA knockdown of VEGFR-2.  In contrast, 55B11 staining of VEGFR-2+ cells was mostly eliminated by siRNA knockdown of VEGFR-2 and increased in VEGFR-2- melanoma cell lines following transfection to express ectopic VEGFR-2.  Our results show that relatively few melanoma cells (<10%) express detectable levels of VEGFR-2, and therefore, the majority of patients with melanoma are unlikely to benefit from antiproliferative effects of anti-VEGF therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozG_W-H4LjhbVg90H21EOLACvtfcHk0ljgAo-YrLhL-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKgurrM&md5=f8037cc86c023b3b09285d315ae5df56</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fijc.25963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25963%26sid%3Dliteratum%253Aachs%26aulast%3DMolhoek%26aufirst%3DK.%2BR.%26aulast%3DErdag%26aufirst%3DG.%26aulast%3DRasamny%26aufirst%3DJ.%26aulast%3DMurphy%26aufirst%3DC.%26aulast%3DDeacon%26aufirst%3DD.%26aulast%3DPatterson%26aufirst%3DJ.%2BW.%26aulast%3DSlingluff%26aufirst%3DC.%2BL.%26aulast%3DBrautigan%26aufirst%3DD.%2BL.%26atitle%3DVEGFR-2%2520expression%2520in%2520human%2520melanoma%253A%2520Revised%2520assessment%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D129%26spage%3D2807%26epage%3D2815%26doi%3D10.1002%2Fijc.25963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Pla, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brossa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grolez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroy, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prevarskaya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gkika, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussolati, B.</span><span> </span><span class="NLM_article-title">Differential sensitivity of prostate tumor derived endothelial cells to sorafenib and sunitinib</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">939</span><span class="NLM_x">–</span> <span class="NLM_lpage">951</span><span class="refDoi"> DOI: 10.1186/1471-2407-14-939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1186%2F1471-2407-14-939" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=939-951&author=A.+F.+Plaauthor=A.+Brossaauthor=M.+Bernardiniauthor=T.+Genovaauthor=G.+Grolezauthor=A.+Villersauthor=X.+Leroyauthor=N.+Prevarskayaauthor=D.+Gkikaauthor=B.+Bussolati&title=Differential+sensitivity+of+prostate+tumor+derived+endothelial+cells+to+sorafenib+and+sunitinib&doi=10.1186%2F1471-2407-14-939"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-939%26sid%3Dliteratum%253Aachs%26aulast%3DPla%26aufirst%3DA.%2BF.%26aulast%3DBrossa%26aufirst%3DA.%26aulast%3DBernardini%26aufirst%3DM.%26aulast%3DGenova%26aufirst%3DT.%26aulast%3DGrolez%26aufirst%3DG.%26aulast%3DVillers%26aufirst%3DA.%26aulast%3DLeroy%26aufirst%3DX.%26aulast%3DPrevarskaya%26aufirst%3DN.%26aulast%3DGkika%26aufirst%3DD.%26aulast%3DBussolati%26aufirst%3DB.%26atitle%3DDifferential%2520sensitivity%2520of%2520prostate%2520tumor%2520derived%2520endothelial%2520cells%2520to%2520sorafenib%2520and%2520sunitinib%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D939%26epage%3D951%26doi%3D10.1186%2F1471-2407-14-939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Chartier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chenard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R.</span><span> </span><span class="NLM_article-title">Kinome render: a stand-alone and web-accessible tool to annotate the human protein kinome tree</span> <span class="citation_source-journal">PeerJ</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">16</span><span class="refDoi"> DOI: 10.7717/peerj.126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.7717%2Fpeerj.126" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=1-16&author=M.+Chartierauthor=T.+Chenardauthor=J.+Barkerauthor=R.+Najmanovich&title=Kinome+render%3A+a+stand-alone+and+web-accessible+tool+to+annotate+the+human+protein+kinome+tree&doi=10.7717%2Fpeerj.126"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.126%26sid%3Dliteratum%253Aachs%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DChenard%26aufirst%3DT.%26aulast%3DBarker%26aufirst%3DJ.%26aulast%3DNajmanovich%26aufirst%3DR.%26atitle%3DKinome%2520render%253A%2520a%2520stand-alone%2520and%2520web-accessible%2520tool%2520to%2520annotate%2520the%2520human%2520protein%2520kinome%2520tree%26jtitle%3DPeerJ%26date%3D2013%26volume%3D1%26spage%3D1%26epage%3D16%26doi%3D10.7717%2Fpeerj.126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Li, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, F.</span><span> </span><span class="NLM_article-title">The human kinome targeted by FDA approved multi-target drugs and combination products: a comparative study from the drug-target interaction network perspective</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0165737</span><span class="refDoi"> DOI: 10.1371/journal.pone.0165737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1371%2Fjournal.pone.0165737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=27828998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFent7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0165737&author=Y.+H.+Liauthor=P.+P.+Wangauthor=X.+X.+Liauthor=C.+Y.+Yuauthor=H.+Yangauthor=J.+Zhouauthor=W.+W.+Xueauthor=J.+Tanauthor=F.+Zhu&title=The+human+kinome+targeted+by+FDA+approved+multi-target+drugs+and+combination+products%3A+a+comparative+study+from+the+drug-target+interaction+network+perspective&doi=10.1371%2Fjournal.pone.0165737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The human kinome targeted by FDA approved multi-target drugs and combination products: a comparative study from the drug-target interaction network perspective</span></div><div class="casAuthors">Li, Ying Hong; Wang, Pan Pan; Li, Xiao Xu; Yu, Chun Yan; Yang, Hong; Zhou, Jin; Xue, Wei Wei; Tan, Jun; Zhu, Feng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0165737/1-e0165737/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human kinome is one of the most productive classes of drug target, and there is emerging necessity for treating complex diseases by means of polypharmacol. (multi-target drugs and combination products).  However, the advantages of the multi-target drugs and the combination products are still under debate.  A comparative anal. between FDA approved multi-target drugs and combination products, targeting the human kinome, was conducted by mapping targets onto the phylogenetic tree of the human kinome.  The approach of network medicine illustrating the drug-target interactions was applied to identify popular targets of multi-target drugs and combination products.  As identified, the multi-target drugs tended to inhibit target pairs in the human kinome, esp. the receptor tyrosine kinase family, while the combination products were able to against targets of distant homol. relationship.  This finding asked for choosing the combination products as a better soln. for designing drugs aiming at targets of distant homol. relationship.  Moreover, subnetworks of drug-target interactions in specific disease were generated, and mechanisms shared by multi-target drugs and combination products were identified.  In conclusion, this study performed an anal. between approved multi-target drugs and combination products against the human kinome, which could assist the discovery of next generation polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7nzlXtKvALVg90H21EOLACvtfcHk0ljYhG3gzk1dGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFent7w%253D&md5=cbc355a8bcdf4b81584910521ab316b7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0165737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0165737%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%2BH.%26aulast%3DWang%26aufirst%3DP.%2BP.%26aulast%3DLi%26aufirst%3DX.%2BX.%26aulast%3DYu%26aufirst%3DC.%2BY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DXue%26aufirst%3DW.%2BW.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DF.%26atitle%3DThe%2520human%2520kinome%2520targeted%2520by%2520FDA%2520approved%2520multi-target%2520drugs%2520and%2520combination%2520products%253A%2520a%2520comparative%2520study%2520from%2520the%2520drug-target%2520interaction%2520network%2520perspective%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26spage%3De0165737%26doi%3D10.1371%2Fjournal.pone.0165737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">Protein kinases--the major drug targets of the twenty-first century?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">315</span><span class="refDoi"> DOI: 10.1038/nrd773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnrd773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=12120282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=309-315&author=P.+Cohen&title=Protein+kinases%2D%2Dthe+major+drug+targets+of+the+twenty-first+century%3F&doi=10.1038%2Fnrd773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Protein kinases - the major drug targets of the twenty-first century?</span></div><div class="casAuthors">Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">309-315</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein phosphorylation regulates most aspects of cell life, whereas abnormal phosphorylation is a cause or consequence of disease.  A growing interest in developing orally active protein-kinase inhibitors has recently culminated in the approval of the first of these drugs for clin. use.  Protein kinases have now become the second most important group of drug targets, after G-protein-coupled receptors.  Here, I give a personal view of some of the most important advances that have shaped this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRd_GqiGAWaLVg90H21EOLACvtfcHk0ljYhG3gzk1dGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D&md5=67d2fbc11a55b5980ee0692f7bc13cc9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnrd773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd773%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DProtein%2520kinases--the%2520major%2520drug%2520targets%2520of%2520the%2520twenty-first%2520century%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D309%26epage%3D315%26doi%3D10.1038%2Fnrd773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Morphy, R.</span><span> </span><span class="NLM_article-title">Selectively nonselective kinase inhibition: striking the right balance</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1413</span><span class="NLM_x">–</span> <span class="NLM_lpage">1437</span><span class="refDoi"> DOI: 10.1021/jm901132v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901132v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlWmtbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1413-1437&author=R.+Morphy&title=Selectively+nonselective+kinase+inhibition%3A+striking+the+right+balance&doi=10.1021%2Fjm901132v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Selectively Nonselective Kinase Inhibition: Striking the Right Balance</span></div><div class="casAuthors">Morphy, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1413-1437</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Marketed kinase inhibitors (MKIs) can deliver superior efficacy compared to inhibitors with high specificity for a single kinase, and the recent introduction of several MKIs to the market opens the door to a new era of safer and effective anticancer therapy.  The key to combining high efficacy with acceptable safety is to inhibit multiple targets in a selectively nonselective fashion.  Strategies for intentionally designing MKIs are emerging, but the field is still in its infancy and we are as medicinal chemists currently on the steepest part of the learning curve.  MTDD can be time-consuming and expensive, and we need to become more proficient first at identifying disease-relevant target combinations and second at discovering MKIs that combine optimal physicochem. and biol. properties.  Bold and innovative medicinal chem. strategies are required to tackle "difficult combinations" where the disease rationale is compelling but where it is a struggle to combine all the desired attributes of an oral MKI drug into a single mol.  At present it is unclear to what extent MKIs with highly tuned selectivity profiles can be rationally designed, particularly for targets that are unrelated by sequence.  In addn. to the well-known selectivity challenge, the physicochem. property profiles of AT P-competitive MKIs can be inherently challenging and limited scope for patentability can also be a serious hindrance.  On the plus side, the amt. of kinase-specific structural information is growing very rapidly, and ultimately this may reveal distinct features and design rules that enable a medicinal chemist to rationally modify and refine the profile of MKIs.  In addn., increasing SAR knowledge is emerging from large scale panel screening with the binding profiles starting to reveal to medicinal chemists how chem. structure affects cross-reactivity across large parts of the kinome.  The merit of MKIs compared with single kinase inhibitors is a subject of controversy in drug discovery that is unlikely to be resolved in the near future.  At the start of a new MTDD project, a rigorous debate needs to take place as to whether it makes more sense to seek a combination of highly selective agents or a DML.  Many factors need to be taken into account in this decision such as the no., similarity, and promiscuity of the targets in the profile and the disease area.  Conformational plasticity and the occurrence of multiple binding modes complicate the in silico prediction of kinase polypharmacol. based solely upon protein structure.  The use of ligand-based similarity to assess the feasibility of a given combination can add real value.  Currently, serendipity plays a significant role in MKI discovery and many, if not most, MKIs have been discovered by chance during the search for selective inhibitors.  Medicinal chemists need to be alert to the possibilities when a surprising combination is found by chance.  To exploit such serendipity, you need a good appreciation of when you have a sufficiently high quality starting compd. and then you need to be able to make and test sufficient analogs to explore your new disease-based hypothesis.  MKIs are costly to develop and are consequently priced at a premium level, so they will need to show clear improvements in order to get reimbursement.  There have already been problems with reimbursement for some MKIs in some markets due to concerns from funding bodies over insufficient efficacy.  The true value of MKIs relative to other anticancer drugs still has to be established, and the results from recent clin. trials have been mixed.  Despite the broad activity profile of many MKIs, the patient response can be inconsistent and unpredictable.  The identification of predictive biomarkers of response or resistance is a crit. step to ascertain which specific combination of targets produces a significant clin. benefit with respect to specific tumor types.  More clin. feedback is needed to facilitate the design of the next generation of inhibitors with more precisely defined profiles.  Although it might seem immeasurably distant at the present time, the ultimate goal should be to derive the prerequisite knowledge and tools so that MTDD becomes a rational endeavor rather than a black box approach that relies upon serendipity.  This will help banish claims that MKIs are merely dirty, nonspecific drugs with insufficient specificity for treating a wider range of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-L-NWmtdikbVg90H21EOLACvtfcHk0ljlLEy3GH7cfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlWmtbfF&md5=3c915b5debf9d005993c8ad31c68bc2e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm901132v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901132v%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26atitle%3DSelectively%2520nonselective%2520kinase%2520inhibition%253A%2520striking%2520the%2520right%2520balance%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1413%26epage%3D1437%26doi%3D10.1021%2Fjm901132v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Broekman, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giovannetti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, G. J.</span><span> </span><span class="NLM_article-title">Tyrosine kinase inhibitors: Multi-targeted or single-targeted?</span> <span class="citation_source-journal">World J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.5306/wjco.v2.i2.80</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.5306%2Fwjco.v2.i2.80" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=21603317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A280%3ADC%252BC3Mrkt1arsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=80-93&author=F.+Broekmanauthor=E.+Giovannettiauthor=G.+J.+Peters&title=Tyrosine+kinase+inhibitors%3A+Multi-targeted+or+single-targeted%3F&doi=10.5306%2Fwjco.v2.i2.80"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine kinase inhibitors: Multi-targeted or single-targeted?</span></div><div class="casAuthors">Broekman Fleur; Giovannetti Elisa; Peters Godefridus J</div><div class="citationInfo"><span class="NLM_cas:title">World journal of clinical oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">80-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since in most tumors multiple signaling pathways are involved, many of the inhibitors in clinical development are designed to affect a wide range of targeted kinases.  The most important tyrosine kinase families in the development of tyrosine kinase inhibitors are the ABL, SCR, platelet derived growth factor, vascular endothelial growth factor receptor and epidermal growth factor receptor families.  Both multi-kinase inhibitors and single-kinase inhibitors have advantages and disadvantages, which are related to potential resistance mechanisms, pharmacokinetics, selectivity and tumor environment.  In different malignancies various tyrosine kinases are mutated or overexpressed and several resistance mechanisms exist.  Pharmacokinetics is influenced by interindividual differences and differs for two single targeted inhibitors or between patients treated by the same tyrosine kinase inhibitor.  Different tyrosine kinase inhibitors have various mechanisms to achieve selectivity, while differences in gene expression exist between tumor and stromal cells.  Considering these aspects, one type of inhibitor can generally not be preferred above the other, but will depend on the specific genetic constitution of the patient and the tumor, allowing personalized therapy.  The most effective way of cancer treatment by using tyrosine kinase inhibitors is to consider each patient/tumor individually and to determine the strategy that specifically targets the consequences of altered (epi)genetics of the tumor.  This strategy might result in treatment by a single multi kinase inhibitor for one patient, but in treatment by a couple of single kinase inhibitors for other patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUsa5qqJaBY_csfMJQC1ZdfW6udTcc2eZmokE-kP044Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mrkt1arsw%253D%253D&md5=683eaad679244b5efc5813d1ffaaaf57</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.5306%2Fwjco.v2.i2.80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5306%252Fwjco.v2.i2.80%26sid%3Dliteratum%253Aachs%26aulast%3DBroekman%26aufirst%3DF.%26aulast%3DGiovannetti%26aufirst%3DE.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26atitle%3DTyrosine%2520kinase%2520inhibitors%253A%2520Multi-targeted%2520or%2520single-targeted%253F%26jtitle%3DWorld%2520J.%2520Clin.%2520Oncol.%26date%3D2011%26volume%3D2%26spage%3D80%26epage%3D93%26doi%3D10.5306%2Fwjco.v2.i2.80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Schrodinger, L.</span>Schrodinger software suite; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=L.+Schrodinger&title=Schrodinger+software+suite"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchrodinger%26aufirst%3DL.%26jtitle%3DSchrodinger%2520software%2520suite%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Halgren, T.</span><span> </span><span class="NLM_article-title">New Method for Fast and Accurate Binding-site Identification and Analysis</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00483.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1111%2Fj.1747-0285.2007.00483.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=17381729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=146-148&author=T.+Halgren&title=New+Method+for+Fast+and+Accurate+Binding-site+Identification+and+Analysis&doi=10.1111%2Fj.1747-0285.2007.00483.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">New method for fast and accurate binding-site identification and analysis</span></div><div class="casAuthors">Halgren, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Structure-based drug design seeks to exploit the structure of protein-ligand or protein-protein binding sites, but the site is not always known at the outset.  Even when the site is known, the researcher may wish to identify alternative prospective binding sites that may result in different biol. effects or new class of compds.  It is also vital in lead optimization to clearly understand the degree to which known binders or docking hits satisfy or violate complementarity to the receptor.  SiteMap is a new technique for identifying potential binding sites and for predicting their druggability in lead-discovery applications and for characterizing binding sites and critically assessing prospective ligands in lead-optimization applications.  In large-scale validation tests, SiteMap correctly identifies the known binding site in > 96% of the cases, with best results (> 98%) coming for sites that bind ligands tightly.  It also accurately distinguishes between sites that bind ligands and sites that don't.  In binding-site anal., SiteMap provides a wealth of quant. and graphical information that can help guide efforts to modify ligand structure to enhance potency or improve phys. properties.  These attributes allow SiteMap to nicely complement techniques such as docking and computational lead optimization in structure-base drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245hmZWXilrVg90H21EOLACvtfcHk0ljlLEy3GH7cfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D&md5=c233276ba9580393b282f255781f9ce6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00483.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00483.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DNew%2520Method%2520for%2520Fast%2520and%2520Accurate%2520Binding-site%2520Identification%2520and%2520Analysis%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D146%26epage%3D148%26doi%3D10.1111%2Fj.1747-0285.2007.00483.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lj_QVzGcuV4zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Duan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">A point-charge force field for molecular mechanicssimulations of proteins based on condensed-phase quantum mechanical calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1999</span><span class="NLM_x">–</span> <span class="NLM_lpage">2012</span><span class="refDoi"> DOI: 10.1002/jcc.10349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fjcc.10349" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=1999-2012&author=Y.+Duanauthor=S.+Chowdhuryauthor=M.+C.+Leeauthor=G.+Xiongauthor=W.+Zhangauthor=R.+Yangauthor=R.+Luoauthor=T.+Leeauthor=J.+Caldwellauthor=J.+Wangauthor=P.+Kollman&title=A+point-charge+force+field+for+molecular+mechanicssimulations+of+proteins+based+on+condensed-phase+quantum+mechanical+calculations&doi=10.1002%2Fjcc.10349"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10349%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DChowdhury%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BC.%26aulast%3DXiong%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DA%2520point-charge%2520force%2520field%2520for%2520molecular%2520mechanicssimulations%2520of%2520proteins%2520based%2520on%2520condensed-phase%2520quantum%2520mechanical%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D1999%26epage%3D2012%26doi%3D10.1002%2Fjcc.10349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Quantum and statistical mechanical studies of liquids. 10. Transferable intermolecular potential functions for water, alcohols, and ethers. Application to liquid water</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">335</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span><span class="refDoi"> DOI: 10.1021/ja00392a016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00392a016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADyaL3MXotlCitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1981&pages=335-340&author=W.+L.+Jorgensen&title=Quantum+and+statistical+mechanical+studies+of+liquids.+10.+Transferable+intermolecular+potential+functions+for+water%2C+alcohols%2C+and+ethers.+Application+to+liquid+water&doi=10.1021%2Fja00392a016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum and statistical mechanical studies of liquids.  10.  Transferable intermolecular potential functions for water, alcohols, and ethers.  Application to liquid water</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">335-40</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Transferable intermol. potential functions (TIPS) suitable for use in liq. simulations are reported for water, alcs., and ethers.  Interaction sites are located on oxygens, hydroxyl hydrogens, and the carbons in alkyl groups.  Each type of site has Coulomb and Lennard-Jones parameters chosen to yield reasonable structural and energetic results for both gas-phase dimers and pure liqs.  A Monte Carlo simulation of liq. water at 25° using the TIP compares favorably with expt. or results from Clementi's CI potential except that the OO radial distribution function is calcd. to be too flat beyond the first solvent shell.  Simulations of liq. methanol and ethanol have also been carried out as described in the accompanying papers.  Overall, in view of the simplicity and transferability of the potentials, the initial results are most encouraging for the treatment of fluids with even more complex monomers and for extension to other types of interaction sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6epUf9b1HcrVg90H21EOLACvtfcHk0lj_QVzGcuV4zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXotlCitA%253D%253D&md5=6160447b696ee61208228f2f542c4a74</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja00392a016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00392a016%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DQuantum%2520and%2520statistical%2520mechanical%2520studies%2520of%2520liquids.%252010.%2520Transferable%2520intermolecular%2520potential%2520functions%2520for%2520water%252C%2520alcohols%252C%2520and%2520ethers.%2520Application%2520to%2520liquid%2520water%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1981%26volume%3D103%26spage%3D335%26epage%3D340%26doi%3D10.1021%2Fja00392a016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Hess, B.</span><span> </span><span class="NLM_article-title">P-LINCS: A parallel linear constraint solver for molecular simulation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">116</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span><span class="refDoi"> DOI: 10.1021/ct700200b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700200b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKru7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=116-122&author=B.+Hess&title=P-LINCS%3A+A+parallel+linear+constraint+solver+for+molecular+simulation&doi=10.1021%2Fct700200b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">P-LINCS: A Parallel Linear Constraint Solver for Molecular Simulation</span></div><div class="casAuthors">Hess, Berk</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">116-122</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By removing the fastest degrees of freedom, constraints allow for an increase of the time step in mol. simulations.  In the last decade parallel simulations have become commonplace.  However, up till now efficient parallel constraint algorithms have not been used with domain decompn.  In this paper the parallel linear constraint solver (P-LINCS) is presented, which allows the constraining of all bonds in macromols.  Addnl. the energy conservation properties of (P-)LINCS are assessed in view of improvements in the accuracy of uncoupled angle constraints and integration in single precision.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXg1Gz9d_OfbVg90H21EOLACvtfcHk0lgbzl21MHJStg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKru7zL&md5=476d5ca2eb25574d44b775996fff7b75</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fct700200b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700200b%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26atitle%3DP-LINCS%253A%2520A%2520parallel%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D116%26epage%3D122%26doi%3D10.1021%2Fct700200b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Miller, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGee, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homeyer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A. E.</span><span> </span><span class="NLM_article-title">MMPBSA.py: an efficient program for end-state free energy calculations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3314</span><span class="NLM_x">–</span> <span class="NLM_lpage">3321</span><span class="refDoi"> DOI: 10.1021/ct300418h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300418h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3314-3321&author=B.+R.+Millerauthor=T.+D.+McGeeauthor=J.+M.+Swailsauthor=N.+Homeyerauthor=H.+Gohlkeauthor=A.+E.+Roitberg&title=MMPBSA.py%3A+an+efficient+program+for+end-state+free+energy+calculations&doi=10.1021%2Fct300418h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">MMPBSA.py: An Efficient Program for End-State Free Energy Calculations</span></div><div class="casAuthors">Miller, Bill R., III; McGee, T. Dwight, Jr.; Swails, Jason M.; Homeyer, Nadine; Gohlke, Holger; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3314-3321</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MM-PBSA is a post-processing end-state method to calc. free energies of mols. in soln.  MMPBSA.py is a program written in Python for streamlining end-state free energy calcns. using ensembles derived from mol. dynamics (MD) or Monte Carlo (MC) simulations.  Several implicit solvation models are available with MMPBSA.py, including the Poisson-Boltzmann Model, the Generalized Born Model, and the Ref. Interaction Site Model.  Vibrational frequencies may be calcd. using normal mode or quasi-harmonic anal. to approx. the solute entropy.  Specific interactions can also be dissected using free energy decompn. or alanine scanning.  A parallel implementation significantly speeds up the calcn. by dividing frames evenly across available processors.  MMPBSA.py is an efficient, user-friendly program with the flexibility to accommodate the needs of users performing end-state free energy calcns.  The source code can be downloaded at http://ambermd.org/ with AmberTools, released under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZK3C1jqiSULVg90H21EOLACvtfcHk0lgbzl21MHJStg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP&md5=cc4148bd8f70c7cad94fd3ec6f580e52</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fct300418h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300418h%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DB.%2BR.%26aulast%3DMcGee%26aufirst%3DT.%2BD.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DMMPBSA.py%253A%2520an%2520efficient%2520program%2520for%2520end-state%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3314%26epage%3D3321%26doi%3D10.1021%2Fct300418h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Chéron, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jasty, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakhnovich, E. I.</span><span> </span><span class="NLM_article-title">OpenGrowth: an automated and rational algorithm for finding new protein ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4171</span><span class="NLM_x">–</span> <span class="NLM_lpage">4188</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00886</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00886" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4171-4188&author=N.+Ch%C3%A9ronauthor=N.+Jastyauthor=E.+I.+Shakhnovich&title=OpenGrowth%3A+an+automated+and+rational+algorithm+for+finding+new+protein+ligands&doi=10.1021%2Facs.jmedchem.5b00886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">OpenGrowth: An Automated and Rational Algorithm for Finding New Protein Ligands</span></div><div class="casAuthors">Cheron, Nicolas; Jasty, Naveen; Shakhnovich, Eugene I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4171-4188</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new open-source software, called OpenGrowth, which aims to create de novo ligands by connecting small org. fragments in the active site of proteins.  Mol. growth is biased to produce structures that statistically resemble drugs in an input training database.  Consequently, the produced mols. have superior synthetic accessibility and pharmacokinetic properties compared with randomly grown mols.  The growth process can take into account the flexibility of the target protein and can be started from a seed to mimic R-group strategy or fragment-based drug discovery.  Primary applications of the software on the HIV-1 protease allowed us to quickly identify new inhibitors with a predicted Kd as low as 18 nM.  We also present a graphical user interface that allows a user to select easily the fragments to include in the growth process.  OpenGrowth is released under the GNU GPL license and is available free of charge on the authors' website and at http://opengrowth.sourceforge.net/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcRSoqoxTyrVg90H21EOLACvtfcHk0lgbzl21MHJStg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI&md5=6f8895e71ca7749cc1dbe559bda1db7c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00886%26sid%3Dliteratum%253Aachs%26aulast%3DCh%25C3%25A9ron%26aufirst%3DN.%26aulast%3DJasty%26aufirst%3DN.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DOpenGrowth%253A%2520an%2520automated%2520and%2520rational%2520algorithm%2520for%2520finding%2520new%2520protein%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4171%26epage%3D4188%26doi%3D10.1021%2Facs.jmedchem.5b00886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Swetha, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaiah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anbarasu, A.</span><span> </span><span class="NLM_article-title">Molecular dynamics studies on D835N mutation in FLT3-its impact on FLT3 protein structure</span> <span class="citation_source-journal">J. Cell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">1439</span><span class="NLM_x">–</span> <span class="NLM_lpage">1445</span><span class="refDoi"> DOI: 10.1002/jcb.25434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1002%2Fjcb.25434" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2016&pages=1439-1445&author=R.+G.+Swethaauthor=S.+Ramaiahauthor=A.+Anbarasu&title=Molecular+dynamics+studies+on+D835N+mutation+in+FLT3-its+impact+on+FLT3+protein+structure&doi=10.1002%2Fjcb.25434"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fjcb.25434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.25434%26sid%3Dliteratum%253Aachs%26aulast%3DSwetha%26aufirst%3DR.%2BG.%26aulast%3DRamaiah%26aufirst%3DS.%26aulast%3DAnbarasu%26aufirst%3DA.%26atitle%3DMolecular%2520dynamics%2520studies%2520on%2520D835N%2520mutation%2520in%2520FLT3-its%2520impact%2520on%2520FLT3%2520protein%2520structure%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2016%26volume%3D117%26spage%3D1439%26epage%3D1445%26doi%3D10.1002%2Fjcb.25434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Tardieu, J.-L.</span><span> </span><span class="NLM_article-title">HTRF KinEASE TK: a new solution for tyrosine kinase screening</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">i</span><span class="NLM_x">–</span> <span class="NLM_lpage">ii</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;key=10.1038%2Fnmeth1082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=i-ii&author=J.-L.+Tardieu&title=HTRF+KinEASE+TK%3A+a+new+solution+for+tyrosine+kinase+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnmeth1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth1082%26sid%3Dliteratum%253Aachs%26aulast%3DTardieu%26aufirst%3DJ.-L.%26atitle%3DHTRF%2520KinEASE%2520TK%253A%2520a%2520new%2520solution%2520for%2520tyrosine%2520kinase%2520screening%26jtitle%3DNat.%2520Methods%26date%3D2007%26volume%3D4%26spage%3Di%26epage%3Dii" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH','PDB','3EWH'); return false;">PDB: 3EWH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P2I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P2I','PDB','2P2I'); return false;">PDB: 2P2I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DTW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DTW','PDB','3DTW'); return false;">PDB: 3DTW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QU6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QU6','PDB','2QU6'); return false;">PDB: 2QU6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGD','PDB','4AGD'); return false;">PDB: 4AGD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i48"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01091">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_22086"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01091">10.1021/acs.jmedchem.7b01091</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">RMSD values for the 24 crystal structures [protein (blue) and ligand (orange)] during the 20 ns MD simulation; number of hydrogen bonds for the 24 crystal ligands during the 20 ns MD simulation; RMSD values and the number of hydrogen bonds for 9 designed compounds during the 20 ns MD simulation; Δ<i>G</i> distribution of all 28 crystal ligands during the 20 ns simulation; ΔG distribution of the 10 designed compounds during the 20 ns simulation; developmental therapeutics program one dose mean graph of the growth percent of cell lines for compounds <b>6</b>, <b>19</b>, <b>22</b>, and <b>23</b>; developmental therapeutics program five dose mean graph of the growth percent as well as GI<sub>50</sub>, LC<sub>50</sub>, and TGI values against 58 cell lines for compound <b>23</b>; comparison of GI<sub>50</sub>, TGI, and LC<sub>50</sub> values, respectively, of individual the cell lines for compound <b>23</b>, sorafenib, and sunitinib; data set for building the binding affinity model; enzymatic inhibition rate of compound <b>23</b> against the targets with a relatively high sequence identity with VEGFR-2 from the kinome wide screening (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf">PDF</a>)</p></li><li><p class="inline">In-house database generated by BREED and scored by molecular docking (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_002.zip">ZIP</a>)</p></li><li><p class="inline">Docking complexes between VEGFR-2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EWH">3EWH</a>) and the compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_003.zip">ZIP</a>)</p></li><li><p class="inline">SMILES format of the 10 designed compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_001.pdf">jm7b01091_si_001.pdf (2.84 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_002.zip">jm7b01091_si_002.zip (944.93 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_003.zip">jm7b01091_si_003.zip (1.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01091/suppl_file/jm7b01091_si_004.csv">jm7b01091_si_004.csv (0.88 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01091&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01091" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01091" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677afcc1fca13c46","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
